<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/2/439" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/2/439.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/2/439.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/2/439.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/2/439/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/2/439.atom" />
<meta name="HW.pisa" content="ggg;9/2/439" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Statistical Procedure for Genome-Wide Detection of QTL Hotspots Using Public Databases with Application to Rice" />
<meta name="DC.Identifier" content="10.1534/g3.118.200922" />
<meta name="DC.Date" content="2019-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2019 Yang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7451393" />
<meta name="DC.Description" content="Genome-wide detection of quantitative trait loci (QTL) hotspots underlying variation in many molecular and phenotypic traits has been a key step in various biological studies since the QTL hotspots are highly informative and can be linked to the genes for the quantitative traits. Several statistical methods have been proposed to detect QTL hotspots. These hotspot detection methods rely heavily on permutation tests performed on summarized QTL data or individual-level data (with genotypes and phenotypes) from the genetical genomics experiments. In this article, we propose a statistical procedure for QTL hotspot detection by using the summarized QTL (interval) data collected in public web-accessible databases. First, a simple statistical method based on the uniform distribution is derived to convert the QTL interval data into the expected QTL frequency (EQF) matrix. And then, to account for the correlation structure among traits, the QTL for correlated traits are grouped together into the same categories to form a reduced EQF matrix. Furthermore, a permutation algorithm on the EQF elements or on the QTL intervals is developed to compute a sliding scale of EQF thresholds, ranging from strict to liberal, for assessing the significance of QTL hotspots. With grouping, much stricter thresholds can be obtained to avoid the detection of spurious hotspots. Real example analysis and simulation study are carried out to illustrate our procedure, evaluate the performances and compare with other methods. It shows that our procedure can control the genome-wide error rates at the target levels, provide appropriate thresholds for correlated data and is comparable to the methods using individual-level data in hotspot detection. Depending on the thresholds used, more than 100 hotspots are detected in GRAMENE rice database. We also perform a genome-wide comparative analysis of the detected hotspots and the known genes collected in the Rice Q-TARO database. The comparative analysis reveals that the hotspots and genes are conformable in the sense that they co-localize closely and are functionally related to relevant traits. Our statistical procedure can provide a framework for exploring the networks among QTL hotspots, genes and quantitative traits in biological studies. The R codes that produce both numerical and graphical outputs of QTL hotspot detection in the genome are available on the worldwide web &lt;http://www.stat.sinica.edu.tw/chkao/&gt;." />
<meta name="DC.Contributor" content="Man-Hsia Yang" />
<meta name="DC.Contributor" content="Dong-Hong Wu" />
<meta name="DC.Contributor" content="Chen-Hung Kao" />
<meta name="article:published_time" content="2019-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Statistical Procedure for Genome-Wide Detection of QTL Hotspots Using Public Databases with Application to Rice" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide detection of quantitative trait loci (QTL) hotspots underlying variation in many molecular and phenotypic traits has been a key step in various biological studies since the QTL hotspots are highly informative and can be linked to the genes for the quantitative traits. Several statistical methods have been proposed to detect QTL hotspots. These hotspot detection methods rely heavily on permutation tests performed on summarized QTL data or individual-level data (with genotypes and phenotypes) from the genetical genomics experiments. In this article, we propose a statistical procedure for QTL hotspot detection by using the summarized QTL (interval) data collected in public web-accessible databases. First, a simple statistical method based on the uniform distribution is derived to convert the QTL interval data into the expected QTL frequency (EQF) matrix. And then, to account for the correlation structure among traits, the QTL for correlated traits are grouped together into the same categories to form a reduced EQF matrix. Furthermore, a permutation algorithm on the EQF elements or on the QTL intervals is developed to compute a sliding scale of EQF thresholds, ranging from strict to liberal, for assessing the significance of QTL hotspots. With grouping, much stricter thresholds can be obtained to avoid the detection of spurious hotspots. Real example analysis and simulation study are carried out to illustrate our procedure, evaluate the performances and compare with other methods. It shows that our procedure can control the genome-wide error rates at the target levels, provide appropriate thresholds for correlated data and is comparable to the methods using individual-level data in hotspot detection. Depending on the thresholds used, more than 100 hotspots are detected in GRAMENE rice database. We also perform a genome-wide comparative analysis of the detected hotspots and the known genes collected in the Rice Q-TARO database. The comparative analysis reveals that the hotspots and genes are conformable in the sense that they co-localize closely and are functionally related to relevant traits. Our statistical procedure can provide a framework for exploring the networks among QTL hotspots, genes and quantitative traits in biological studies. The R codes that produce both numerical and graphical outputs of QTL hotspot detection in the genome are available on the worldwide web http://www.stat.sinica.edu.tw/chkao/.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/02/01" />
<meta name="citation_mjid" content="ggg;9/2/439" />
<meta name="citation_id" content="9/2/439" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/2/439" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/2/439.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/2/439.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/2/439.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200922" />
<meta name="citation_pmid" content="30541929" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="439" />
<meta name="citation_lastpage" content="452" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Man-Hsia Yang" />
<meta name="citation_author_institution" content="Department of Agronomy, National Taiwan University, Taipei 10617, Taiwan, Republic of China" />
<meta name="citation_author_institution" content="Crop Science Division, Taiwan Agricultural Research Institute, Council of Agriculture, Taichung 41362, Taiwan, Republic of China" />
<meta name="citation_author" content="Dong-Hong Wu" />
<meta name="citation_author_institution" content="Crop Science Division, Taiwan Agricultural Research Institute, Council of Agriculture, Taichung 41362, Taiwan, Republic of China" />
<meta name="citation_author" content="Chen-Hung Kao" />
<meta name="citation_author_institution" content="Department of Agronomy, National Taiwan University, Taipei 10617, Taiwan, Republic of China" />
<meta name="citation_author_institution" content="Institute of Statistical Science, Academia Sinica, Taipei 11529, Taiwan, Republic of China" />
<meta name="citation_author_email" content="chkao@stat.sinica.edu.tw" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdiscip. Rev. Comput. Stat.;citation_author=H. Abdi;citation_author=L. J. Williams;citation_title=Principal component analysis.;citation_pages=433-459;citation_volume=2;citation_year=2010;citation_doi=10.1002/wics.101" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=F. Ali;citation_author=Q. Pan;citation_author=G. Chen;citation_author=K. R. Zahid;citation_author=J. Yan;citation_title=Evidence of Multiple Disease Resistance (MDR) and implication of meta-analysis in marker assisted selection.;citation_pages=e68150;citation_volume=8;citation_year=2013;citation_doi=10.1371/journal.pone.0068150" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=R. K. Basnet;citation_author=A. Duwal;citation_author=D. N. Tiwari;citation_author=D. Xiao;citation_author=S. Monakhos;citation_title=Quantitative Trait Locus Analysis of Seed Germination and Seedling Vigor in Brassica rapa Reveals OTL Hotspots and Epistatic Interactions.;citation_pages=1032;citation_volume=6;citation_year=2015;citation_doi=10.3389/fpls.2015.01032" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Breitling;citation_author=Y. Li;citation_author=B. M. Tesson;citation_author=J. Fu;citation_author=C. Wu;citation_title=Genetical genomics: spotlight on QTL hotspots.;citation_pages=e1000232-e1000232;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18949031;citation_doi=10.1371/journal.pgen.1000232" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. W. Broman;citation_title=Mapping Quantitative Trait Loci in the Case of a Spike in the Phenotype Distribution;citation_pages=1169-1175;citation_volume=163;citation_year=2003;citation_issue=3;citation_pmid=12663553" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. P. Cabrera;citation_author=P. Navarro;citation_author=J. E. Huffman;citation_author=A. F. Wright;citation_author=C. Hayward;citation_title=Uncovering networks from genome-wide association studies via circular genomic permutation. G3: Genes, Genomes;citation_pages=1067-1075;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.112.002618" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Chardon;citation_author=B. Virlon;citation_author=L. Moreau;citation_author=M. Falque;citation_author=J. Joets;citation_title=Genetic Architecture of Flowering Time in Maize As Inferred From Quantitative Trait Loci Meta-analysis and Synteny Conservation With the Rice Genome;citation_pages=2169-2185;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611184;citation_doi=10.1534/genetics.104.032375" />
<meta name="citation_reference" content="citation_title=Cluster analysis: Wiley series in probability and statistics, pp;citation_author=B. S. Everitt;citation_author=S. Landau;citation_author=M. Leese;citation_author=D. Stahl;citation_year=2011;citation_doi=10.1002/9780470977811" />
<meta name="citation_reference" content="citation_title=Introduction to quantitative genetics. Longman, Essex, UK. Introduction to quantitative genetics;citation_author=D. Falconer;citation_author=T. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. Frary;citation_author=A. Frary;citation_author=M. C. Daunay;citation_author=K. Huvenaars;citation_author=R. Mank;citation_title=QTL hotspots in eggplant (Solanum melongena) detected with a high resolution map and CIM analysis.;citation_pages=211-228;citation_volume=197;citation_year=2014;citation_doi=10.1007/s10681-013-1060-6" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Fu;citation_author=J. J. B. Keurentjes;citation_author=H. Bouwmeester;citation_author=T. America;citation_author=F. W. A. Verstappen;citation_title=System-wide molecular evidence for phenotypic buffering in Arabidopsis.;citation_pages=166-167;citation_volume=41;citation_year=2009;citation_issue=2;citation_pmid=19169256;citation_doi=10.1038/ng.308" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Fujita;citation_author=K. R. Trijatmiko;citation_author=A. G. Tagle;citation_author=M. V. Sapasap;citation_author=Y. Koide;citation_title=NAL1 allele from a rice landrace greatly increases yield in modern indica cultivars.;citation_volume=110;citation_year=2013;citation_pmid=24297875;citation_doi=10.1073/pnas.1310790110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. Goffinet;citation_author=S. Gerber;citation_title=Quantitative Trait Loci: A Meta-analysis;citation_pages=463-473;citation_volume=155;citation_year=2000;citation_issue=1;citation_pmid=10790417" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Haley;citation_author=S. A. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_issue=4;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Harushima;citation_author=M. Yano;citation_author=A. Shomura;citation_author=M. Sato;citation_author=T. Shimano;citation_title=A High-Density Rice Genetic Linkage Map with 2275 Markers Using a Single F2 Population;citation_pages=479-494;citation_volume=148;citation_year=1998;citation_issue=1;citation_pmid=9475757" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Jansen;citation_title=Interval mapping of multiple quantitative trait loci.;citation_pages=205-211;citation_volume=135;citation_year=1993;citation_issue=1;citation_pmid=8224820" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Kao;citation_title=Mapping quantitative trait loci using the experimental designs of recombinant inbred populations.;citation_volume=174;citation_year=2006;citation_pmid=17121967;citation_doi=10.1534/genetics.106.056416" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. H. Kao;citation_author=Z. B. Zeng;citation_author=R. D. Teasdale;citation_title=Multiple Interval Mapping for Quantitative Trait Loci;citation_pages=1203-1216;citation_volume=152;citation_year=1999;citation_issue=3;citation_pmid=10388834" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. I. Lee;citation_author=H. A. Ho;citation_author=C. H. Kao;citation_title=A new simple method for improving QTL mapping under selective genotyping.;citation_volume=198;citation_year=2014;citation_pmid=25245793;citation_doi=10.1534/genetics.114.168385" />
<meta name="citation_reference" content="citation_journal_title=Journal of biopharmaceutical statistics;citation_journal_abbrev=J Biopharm Stat;citation_author=S. Li;citation_author=Q. Lu;citation_author=Y. Cui;citation_title=A systems biology approach for identifying novel pathway regulators in eQTL mapping.;citation_pages=373-400;citation_volume=20;citation_year=2010;citation_issue=2;citation_pmid=20309764;citation_doi=10.1080/10543400903572803" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The map-based sequence of the rice genome.;citation_pages=793-800;citation_volume=436;citation_year=2005;citation_issue=7052;citation_pmid=16100779;citation_doi=10.1038/nature03895" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=D. K. Mengistu;citation_author=Y. G. Kidane;citation_author=M. Catellani;citation_author=E. Frascaroli;citation_author=C. Fadda;citation_title=High-density molecular characterization and association mapping in Ethiopian durum wheat landraces reveals high diversity and potential for wheat breeding.;citation_pages=1800-1812;citation_volume=14;citation_year=2016;citation_doi=10.1111/pbi.12538" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. C. Neto;citation_author=M. P. Keller;citation_author=A. F. Broman;citation_author=A. D. Attie;citation_author=R. C. Jansen;citation_title=Quantile-Based Permutation Thresholds for Quantitative Trait Loci Hotspots.;citation_volume=191;citation_year=2012;citation_pmid=22661325;citation_doi=10.1534/genetics.112.139451" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=T. Ookawa;citation_author=T. Hobo;citation_author=M. Yano;citation_author=K. Murata;citation_author=T. Ando;citation_title=New approach for rice improvement using a pleiotropic QTL gene for lodging resistance and yield.;citation_pages=132-132;citation_volume=1;citation_year=2010;citation_pmid=21119645;citation_doi=10.1038/ncomms1132" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=G. Peng;citation_author=L. Luo;citation_author=H. Siu;citation_author=Y. Zhu;citation_author=P. Hu;citation_title=Gene and pathway-based second-wave analysis of genome-wide association studies.;citation_pages=111-117;citation_volume=18;citation_year=2010;citation_issue=1;citation_pmid=19584899;citation_doi=10.1038/ejhg.2009.115" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. E. Schadt;citation_author=S. A. Monks;citation_author=T. A. Drake;citation_author=A. J. Lusis;citation_author=N. Che;citation_title=Genetics of gene expression surveyed in maize, mouse and man.;citation_pages=297-302;citation_volume=422;citation_year=2003;citation_issue=6929;citation_pmid=12646919;citation_doi=10.1038/nature01434" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=A Statistical Framework for Quantitative Trait Mapping;citation_pages=371-387;citation_volume=159;citation_year=2001;citation_issue=1;citation_pmid=11560912" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Studer;citation_author=J. F. Doebley;citation_title=Do Large Effect QTL Fractionate? A Case Study at the Maize Domestication QTL teosinte branched1.;citation_volume=188;citation_year=2011;citation_pmid=21515578;citation_doi=10.1534/genetics.111.126508" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. P. M. Swamy;citation_author=P. Vikram;citation_author=S. Dixit;citation_author=H. U. Ahmed;citation_author=A. Kumar;citation_title=Meta-analysis of grain yield QTL identified during agricultural drought in grasses showed consensus.;citation_pages=319-319;citation_volume=12;citation_year=2011;citation_pmid=21679437;citation_doi=10.1186/1471-2164-12-319" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=K. R. Trijatmiko;citation_author=J. Supriyanta;citation_author=J. Prasetiyono;citation_author=M. J. Thomson;citation_author=C. M. Vera Cruz;citation_title=Meta-analysis of quantitative trait loci for grain yield and component traits under reproductive-stage drought stress in an upland rice population.;citation_pages=283-295;citation_volume=34;citation_year=2014;citation_doi=10.1007/s11032-013-0012-0" />
<meta name="citation_reference" content="citation_title=Information retrieval;citation_author=C. J. Van Rijsbergen;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=T. D. Vuong;citation_author=H. Sonah;citation_author=C. G. Meinhardt;citation_author=R. Deshmukh;citation_author=S. Kadam;citation_title=Genetic architecture of cyst nematode resistance revealed by genome-wide association study in soybean.;citation_pages=593;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1811-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. L. Wei;citation_author=S. Z. Xu;citation_title=A Random-Model Approach to QTL Mapping in Multiparent Advanced Generation Intercross (MAGIC) Populations.;citation_volume=202;citation_year=2016;citation_pmid=26715662;citation_doi=10.1534/genetics.115.179945" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. A. L. West;citation_author=K. Kim;citation_author=D. J. Kliebenstein;citation_author=H. van Leeuwen;citation_author=R. W. Michelmore;citation_title=Global eQTL mapping reveals the complex genetic architecture of transcript-level variation in Arabidopsis.;citation_volume=175;citation_year=2007;citation_pmid=17179097;citation_doi=10.1534/genetics.106.064972" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. J. Wisser;citation_author=J. M. Kolkman;citation_author=M. E. Patzoldt;citation_author=J. B. Holland;citation_author=J. M. Yu;citation_title=Multivariate analysis of maize disease resistances suggests a pleiotropic genetic basis and implicates a GST gene.;citation_volume=108;citation_year=2011;citation_pmid=21490302;citation_doi=10.1073/pnas.1011739108" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=B. Wu;citation_author=D. H. Mao;citation_author=T. M. Liu;citation_author=Z. X. Li;citation_author=Y. Z. Xing;citation_title=Two quantitative trait loci for grain yield and plant height on chromosome 3 are tightly linked in coupling phase in rice.;citation_pages=1-13;citation_volume=35;citation_year=2015;citation_doi=10.1007/s11032-015-0202-z" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. L. Wu;citation_author=D. L. Delano;citation_author=N. Mitro;citation_author=S. V. Su;citation_author=J. Janes;citation_title=Gene set enrichment in eQTL data identifies novel annotations and pathway regulators.;citation_pages=e1000070-e1000070;citation_volume=4;citation_year=2008;citation_issue=5;citation_pmid=18464898;citation_doi=10.1371/journal.pgen.1000070" />
<meta name="citation_reference" content="citation_journal_title=Mol. Genet. Genomics;citation_author=Y. H. Wu;citation_author=M. Huang;citation_author=X. X. Tao;citation_author=T. Guo;citation_author=Z. Q. Chen;citation_title=Quantitative trait loci identification and meta-analysis for rice panicle-related traits.;citation_pages=1927-1940;citation_volume=291;citation_year=2016;citation_doi=10.1007/s00438-016-1227-7" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomics;citation_author=F. Xu;citation_author=Y. Huang;citation_author=J. Bao;citation_title=Identification of QTLs for agronomic traits in indica rice using an RIL population.;citation_pages=809-817;citation_volume=37;citation_year=2015;citation_doi=10.1007/s13258-015-0312-3" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=E. Yamamoto;citation_author=J. Yonemaru;citation_author=T. Yamamoto;citation_author=M. Yano;citation_title=OGRO: The Overview of functionally characterized Genes in Rice online database.;citation_pages=26;citation_volume=5;citation_year=2012;citation_pmid=27234245;citation_doi=10.1186/1939-8433-5-26" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. B. Zeng;citation_title=Precision mapping of quantitative trait loci.;citation_pages=1457-1468;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013918" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=K. Zhao;citation_author=C. W. Tung;citation_author=G. C. Eizenga;citation_author=M. H. Wright;citation_author=M. L. Ali;citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.;citation_pages=467-467;citation_volume=2;citation_year=2011;citation_pmid=21915109;citation_doi=10.1038/ncomms1467" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Statistical Procedure for Genome-Wide Detection of QTL Hotspots Using Public Databases with Application to Rice" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide detection of quantitative trait loci (QTL) hotspots underlying variation in many molecular and phenotypic traits has been a key step in various biological studies since the QTL hotspots are highly informative and can be linked to the genes for the quantitative traits. Several statistical methods have been proposed to detect QTL hotspots. These hotspot detection methods rely heavily on permutation tests performed on summarized QTL data or individual-level data (with genotypes and phenotypes) from the genetical genomics experiments. In this article, we propose a statistical procedure for QTL hotspot detection by using the summarized QTL (interval) data collected in public web-accessible databases. First, a simple statistical method based on the uniform distribution is derived to convert the QTL interval data into the expected QTL frequency (EQF) matrix. And then, to account for the correlation structure among traits, the QTL for correlated traits are grouped together into the same categories to form a reduced EQF matrix. Furthermore, a permutation algorithm on the EQF elements or on the QTL intervals is developed to compute a sliding scale of EQF thresholds, ranging from strict to liberal, for assessing the significance of QTL hotspots. With grouping, much stricter thresholds can be obtained to avoid the detection of spurious hotspots. Real example analysis and simulation study are carried out to illustrate our procedure, evaluate the performances and compare with other methods. It shows that our procedure can control the genome-wide error rates at the target levels, provide appropriate thresholds for correlated data and is comparable to the methods using individual-level data in hotspot detection. Depending on the thresholds used, more than 100 hotspots are detected in GRAMENE rice database. We also perform a genome-wide comparative analysis of the detected hotspots and the known genes collected in the Rice Q-TARO database. The comparative analysis reveals that the hotspots and genes are conformable in the sense that they co-localize closely and are functionally related to relevant traits. Our statistical procedure can provide a framework for exploring the networks among QTL hotspots, genes and quantitative traits in biological studies. The R codes that produce both numerical and graphical outputs of QTL hotspot detection in the genome are available on the worldwide web &lt;http://www.stat.sinica.edu.tw/chkao/&gt;." />
<meta name="og-title" property="og:title" content="A Statistical Procedure for Genome-Wide Detection of QTL Hotspots Using Public Databases with Application to Rice" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/2/439" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide detection of quantitative trait loci (QTL) hotspots underlying variation in many molecular and phenotypic traits has been a key step in various biological studies since the QTL hotspots are highly informative and can be linked to the genes for the quantitative traits. Several statistical methods have been proposed to detect QTL hotspots. These hotspot detection methods rely heavily on permutation tests performed on summarized QTL data or individual-level data (with genotypes and phenotypes) from the genetical genomics experiments. In this article, we propose a statistical procedure for QTL hotspot detection by using the summarized QTL (interval) data collected in public web-accessible databases. First, a simple statistical method based on the uniform distribution is derived to convert the QTL interval data into the expected QTL frequency (EQF) matrix. And then, to account for the correlation structure among traits, the QTL for correlated traits are grouped together into the same categories to form a reduced EQF matrix. Furthermore, a permutation algorithm on the EQF elements or on the QTL intervals is developed to compute a sliding scale of EQF thresholds, ranging from strict to liberal, for assessing the significance of QTL hotspots. With grouping, much stricter thresholds can be obtained to avoid the detection of spurious hotspots. Real example analysis and simulation study are carried out to illustrate our procedure, evaluate the performances and compare with other methods. It shows that our procedure can control the genome-wide error rates at the target levels, provide appropriate thresholds for correlated data and is comparable to the methods using individual-level data in hotspot detection. Depending on the thresholds used, more than 100 hotspots are detected in GRAMENE rice database. We also perform a genome-wide comparative analysis of the detected hotspots and the known genes collected in the Rice Q-TARO database. The comparative analysis reveals that the hotspots and genes are conformable in the sense that they co-localize closely and are functionally related to relevant traits. Our statistical procedure can provide a framework for exploring the networks among QTL hotspots, genes and quantitative traits in biological studies. The R codes that produce both numerical and graphical outputs of QTL hotspot detection in the genome are available on the worldwide web &lt;http://www.stat.sinica.edu.tw/chkao/&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
    <title>A Statistical Procedure for Genome-Wide Detection of QTL Hotspots Using Public Databases with Application to Rice | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"S_2POz-PXibQU4l9pqiByC9GuH6QxZKIrflO0ai42M4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492213","apath":"\/ggg\/9\/2\/439.atom","pisa":"ggg;9\/2\/439","ac":{"\/ggg\/9\/2\/439.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/2\/439.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/2\/439"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/2\/439"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/2\/439":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rFGCzJ\\\\\\\/DM5XuTnbZa6OTG8fHgKZRf886AH1NoetgmXVJHME2Tuw20LDlKZiGjsMPkfnd1kp\\\\\\\/sYLxOgIoFPjNZNfc4MqXJRHJAd85u1OP7SLkACp8pzseIUGWgWSfUd6x1IYKY6VRZYBAoMBCTzCd+hiWsfEVnrauAGc\\\\\\\/pbBIXgUzpuXXQnWilLL9l5lVmRZ0MJRRGGC415WDbyCbYa5Uh4n6B04F2CEqkWmveOYsm0JC3qBWjtFI0vBpKUUoxCc6XF78nATnONclXp0R00CRK+ug82AiAqGREs8GOXnG2HKHO\\\\\\\/4BKX0R7H68nb0GeoxpiXZ5sky2PZKFx8\\\\\\\/Or2wDKbMYGloVwOh5JP1TnMaHQEz2fDqluXmO8Rt9+nkhji5xwTI47wuxsFn2pJ8tOmR90RHMUyjOKdpFpWdqYaJSTANwuGl1bss9Q02BRBOaj47eiMrJ9KsTGI0R0TW6NrwcJFz\\\\\\\/rbfEB1Wtamk50knd8jhmnlENvbkZOwMgsvgOrB5YCZoPh03ZLVeEPsrfA7l7F79YtBtrg1imXijfMK0FWOYYzcneHw\\\\\\\/xpxPb0pm2G3kDj7NJbstE5YlVWn570IX67ZAH5CNpVL8nTFo4bvBeK3+9eZZpW2JCMGkJNVjkqe9kB4fvS\\\\\\\/EoKHCyvratYOGhflHHitsbLhF8H70EAOMp2ld9bM70i9RiYnKapGVPUdj7N9FXpaz9EKrtkAOj69x7PY\\\\\\\/tP5m2DjAdyO8uAcA6bv4eToS2XfXJn+j6x6WeR6c20mm13zUe1rvFSc79HNPW42DnQDm+SQprTyZWZ8lh5CNM87k1Xh2RRs91f3fjJGIlsOXKQtqsQSuhppf\\\\\\\/erkaid0Y4ZbodHGFRywjBXqLvYhtRY19wqtejZro76q6kyl4aGL6gGeJ4fCu5zvXN3Mzc4BymNyvDJ3SIKJnzeYdk4Kzy7Of9nMP9zL7Y8hdWQLF6+1CbddNOfxDSkjbWx9RCou8mYFLiYZ\\\\\\\/PhhrNz0nEc6o8LUQz9ML8zJodHtEg3G6h2OtgAHhz3IRHygHnmViZeIcu3FMGSe46lAcCPTlVLXxElot9X00ZX3F679Tyg6DkjJ9oDIGdbYAmUlU3jcqdg4kLlYjJZSmjSPYRNrtYFvv4t+0C1Q19yJBISbpIgY8\\\\\\\/TB9aQub4aPdL2vcmiprGVpJSXnLIFwR00PeRUkL+8NMbqssTyB3AbK5tURMmTwa4tgETeDi5q2NjPZjaMhYk5NkXzONbwLWoImPXFYSVMRihCROSZbdlxc2e9v9zKA+\\\u0022,\\\u0022iv\\\u0022:\\\u0022Hztmk8zDedP7hDl60VEdnA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022baebe61a030bae369372efacae65946e496a877dbcc63f6304ca8c700896ce2c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zIlMOc9grcMnHq+A1A+V679N53eu1htoykNTKbHxZoo9Aa1RyEPsYYGLpAkn7ifA1cyxbxJ1laDg7qE+mINd4KSna4jFKyeQuk5yoseiQjIZfVsxnMuvwE6BOw\\\\\\\/+q6jgJ80X\\\\\\\/NAUPeRW0WNX\\\\\\\/JgZhifJ4VV7H+2neRg8qiWrT\\\\\\\/YbGwQtV0VHkLMz+9ddvm+plKzwSMUVpEuWDfX19Pof8tmXSA0leYxLhPrh1AqAPvppxJTbJbDcgOhGvebdW\\\\\\\/AFOXIbECuBI34c03LAOgCkdUw0hBnHwU5vRzZ\\\\\\\/MzS7KUjlLQaT9vn\\\\\\\/21besL+1KLxslTe884NxMybBNNVFx1KqM\\\\\\\/tkAMwIfcYo2j9tVfAFMD5pmANoz1pEN9gy8oCkmUsCaeV8f1Q8t90q9FsBqSqfE3cpLopwGz8yWv845UuKwXlNHThIUsvhSUXYFlgexFcak+RqPdRMne8NF9UaRXpI8cNNwXympMbKVmACfl0FVy6ef1X49nx3gwlC5oDURx\\\\\\\/Jl6T+EUgA6fXiGPRZMpc8sWccq8aPvALFbOam0vVf5JSKK\\\\\\\/vBVKkX8jMyaRDs9tk44JdwuRGVthI0xl92w+2YhzwBi1luCQhyeDcyXXmP03XGRc8sBPMvQohPTZdIwAyLGm3hqq1T8yqf56CpiH4o\\\\\\\/m\\\\\\\/LszeDG73f\\\\\\\/oDEGyvK47Fif3AAguhUJv2OoWNDBc8OcV1PANrfBTabuWNa8q7qbyy\\\\\\\/3vcX24WLM+FijZzAiXVo86fwkd8oZcTRc54NWQuoER343NKK3B3VZm1Xxt\\\\\\\/Hf8y7qyOF\\\\\\\/+bbBl2F3kDkx4rk6NzjASz2WXzCHizHmL0v6\\\\\\\/\\\\\\\/+kId+40bR0ge57SC3WIpNeuiSIekNvHO5ryW1i6Oh8XDIlFZO0wN+1vTGSZfSbfPHhZLKm5YtJz1nwmca7YQIXqV+zL1eRcgS3X3eVUng2SMataKvbtee\\\\\\\/hRdZIdI40vxiwuwf1oEqlveioJ0VSgzu0ZPfOVs2oyEE8u9+utUqxYgKzj+udS5xIkDCnje9hctO64s9yimt2i01BrN9ytTxnr0oFMvEU+NryrCUgcYvic+EpBBjenxqs5XKsIm3rFJVeizX0UgKAuJerMdE4zeZlmeod9EdHuiv6NKdk7I48vS1vP42sFk2PrTt0+oPVcAtSyN2EX7dNFmm391KewySvgNZukkZum6j7ONwWyRuaC4yEJgKqW1o8bVgjRLcZABTTByOhwSP\\\\\\\/P8TiCuBmq+WGiAOCKJQcmgIy3KCJbVmz9eGu92jSDovQ+qBBMsjk1Nc05plp7iZrP9iCossbcC5azrWXemCDd9eRxMsr7KqewBZPgJ49MrrKOP1wO7GsIPxGtMUnMlJEp2JZwg2DlGHWkRVHcxN0b1oYiB1QZMu+so\\\\\\\/CwOMlSmldb\\\\\\\/V0EobYPE7p+TqXoBoxrcbzvIKzPsMoyMx+5yquDGXsE88rb3zehlkdvevSFFJ2Ot\\\\\\\/9vZVL2AKu8OyWh\\\\\\\/stZBI31C9g+eXPZ4MnWWBUX4d634WhIepVOr4t9RkvlNDNwkRB6z8tUCPaBzFUjKaRy773KxmhhRMJa+NYxsVMjPKsWTm3wTKmoNQcs+YX\\\\\\\/\\\\\\\/RvMhFiNWKu629P0WNzhiu0XxmBwZZ7znPLwuZ\\\\\\\/bXLkTlnJ6iSv3zvfNwxoArFRb+Ap68xuLZvd8o0HGqYWl0tQJS1jmzUcWx8ZTopfqLuzfuzj4mX3GaFw9FzE\\\\\\\/CjRk5k\\\\\\\/qmpoMtHmkPpmHyj0iHM0bumUMWgqTGw9\\\\\\\/acq\\\\\\\/t3CAT0b1W+BpF+8heBXfc3iCRC+D35nCnjYYNg1DIGKGBquBlDRrXbg8azHb1gaDYnw4yObixwxGNXxmRuOnl1xw3q6HH8lmEsp9zIlUb\\\\\\\/o4YulkDTZuRoGbG1QKXRFAvYfPgiPKlSEbaZ70m9h8xjWbZCHjdgbsQHtHUGB\\\\\\\/RNj1oO4GHPbBOw8hlXzGT4ejibiYTxTjlxWhs85thR210OU2nIfCs2M7SUW5u3M5RHTWAVLW6\\\\\\\/4k4pNVZkBkwE66tp0QfSeifobGxHzH57NtL4OA79+rNtI+WAw\\\\\\\/UhmEikLOVr+f9VnHK57LOPo8SNrawgKMfMHYNO2hrMu0fzfQpJnja29ddrjRKljJkj\\\\\\\/Y9+\\\\\\\/tL2qNKZTQjQkbd798UnNPPnz9IL1non7yaPObBiVd0mz2\\\\\\\/vN7oXotnEw1\\\\\\\/npCH5XOnJ7oiEvScrGGa2UG55X9IZF3ib1N+QJYjC0BItWxUt8HIrQiS+PCulfnJWjRPk9Sdwsk8GywSkMMdRvcGKKMtvPKYViJURJO\\\\\\\/z7+QVGPVrqhJ9G2hpSgfpegE382uEM6fXvHqTSHSXsRaf4wIlIEVkeiJkkNaeb7n0raKDD3UknimH9HVn7+lbgzed0JbttKK1ebnzdVzNvgLTr8aN\\\\\\\/0p2URF+grNQqdhxagg2O4Sjk4+9TuHyVQ961z0aqO6Pp9WOqhGHVX1\\\\\\\/RjAkRcE+iKtUvSyHEgY3IJsflm1u+rWKvAzM\\\\\\\/OwzrrtinI4q5b3+nGXulibgSq+OSkkkiWh0bSh3wV3MX1g4fIW9pcHjkF\\\\\\\/spcGydZTLsy711k6brizM2PqR+wUDwvUnVuw5wzZl0HkHeQtM+B\\\\\\\/PDw8Ak0UAIi1H8G\\\\\\\/m0\\\\\\\/CZugyQtAN1NBhDgY+mM4cnFfHwpBpmn+ZRHLbHS0TXuB5B\\\\\\\/nJV8uTv9j82BWMFhTIO1ZzxEEFVsVbqYtHDZ78fYf+3d\\\\\\\/MUS7v5EMik2fhClwu4\\\\\\\/dcKn3VOpnAclRZOO7wBkIWJ888Cph+ZuCRqRUaLRoMY9uANOuiaREIaKsk5DrHatkAS80gPAPbIirxVD40KjcAuAjcGmnOST+x3ZUKBYGzdLZvqQ6bOMK0RjCjnr5fXksbUEAhLtge1wUX+hPV+En8qmQ8123TT\\\\\\\/\\\\\\\/jlFqbCCs\\\\\\\/bqCyWmP9jMRbdkAoTM5EM8umudClUcrDnUCXd0n2oUggMaibmNqNqmIeyViTZkbVgWlUVJG+XDpC0JphUTscf6CuKP0gXSDzC2bJrVjVKjb60nLHKMeEDC1Z6WEIHLHZvXRneglNfV1dm2LSi6cl61a\\\\\\\/2bC\\\\\\\/vS+jFa\\\\\\\/IKcRMosaF3n2lu9BFvJLYDolM\\\\\\\/9GSZj44sFFB0RCPihgT7tp36Y4ks63kZAaiYUS34p0GUParhSEjOUtomnluLxho5n9bEeESNa\\\\\\\/vpKNZh5BRyBTY+D5FMaHrbSqHr3gqFx\\\\\\\/mvKWvL4eQwUNS3\\\\\\\/A2nSG2zL1MDkpiX8Tzgd8DBPPpvLargcENyPFjv+ufY0K2aMp4eyE9IqUBTcihpaqjgS15xkNgQHgqMPK1UYkAczY5IhxWnK4583qnmQg3EabwKudatLgaQURO551bZi3VOi1L+NKniw9zZZlTBttCSHpddksmwhySzK\\\\\\\/B+WN5Ts2EngaRADg0w+urFBc4qpd1JX7kxDjZJD2e3brSZRjEb3j8On0ZPsQVL4gvEI2BztazO+H9L9RtwPvcaSFl2eNcQ24D32QHYpyfwubPjkQ0troZ7xxf9S6nQMiCFP8ZemwlTg3O\\\\\\\/J+Nuo6DnOoZh9R1m87UYHB4vuAX5U3g9NTWmwcrRnc7kmzIdB8EDY8olHz0N8S4+jXnygpvt4e5\\\\\\\/\\\\\\\/ddFynJX8+6ufdVfB4htYyh5gCgqghLVdAfXrbPMkk4Ww3QkY6areqBaLwAM8yKZpYasfPfXcKjyQYxt4ixFSvkLpwvfsL6s\\\\\\\/cJ2\\\\\\\/vCVQzNYodJj\\\\\\\/fDcb6142JWfueSQtBrYV6k3Rg1Ru3nCIJyDfK+F+LVqSOjM3yUPQ6o23MSCNM8LvvF47VXs5Y5zDfxjoeTwN673q5r+FNFGYhzJ\\\\\\\/wm9FXuE61WbsKkSodj6+uFaBb+FcO65JSsi2PAKMgq7Q0QEUzW4Pn6fxqyoI9cBBTlqx9fZ8e7hbzvH33\\\\\\\/4NwWW97XRSncxhet+eMv3CEDptXJecR18kLvU25ZyhQqh4eT78xPj8EPq7nay3C81W7NB7Td21DqJuv1\\\\\\\/3WDmJy9ibBLCxO4C4Zv1Bj2OmP0reuEmp653JTd1qA1hJiWfcfYU8O2TW4QNgkn9WhghnIQMbMHw7j3Jn4qhXmRG4WKWF\\\\\\\/wx6VAEqpPmISTynecVey1d9wvojre8I6w+pj1NJVkPEIuafbBEHHz9oF4DT2Fht0fb+MkAL0kJqeFFsmqzOPmG4zYlX2MnYP8s5mjNcCw+28nV0wZJkeJnqfJarIXGHJU0ZQJT5PTsKvFlmOKo\\\\\\\/ThHCVUTNqsMw2CiXXGk12uaK\\\\\\\/E\\\\\\\/Nt75CCfh41s8GZ534zr4e\\\\\\\/oUuCEpPnJMNX2TQ+NvjvQLPibMhdUeepL+cCYihIgJaAc0IsBViuRgpEom7diO83z0aUJvxBKQuXUo2WL4Q\\\\\\\/yDF1xVvlN0Tzb9p1UJgVoCKHDglFhvD6uATHbDf5044x+ZrDr9Nl8CwbXvdvSO+y7Dg+c0DpHE7GxhylAp36WMOzvvpJp+1DwoDuJYEtkcxMHVpW5W7660nPV78DIeJg9H19lHAiudXfIVkTYaSkozfKoDEQOf0iQbDRTOWvOB5X3udGasO2DSmj35cb6ChWtL44UQBbXiZRt2COp6R2LJTseyu+I8TQoVnsO1QJNKS8xfZ0JL2NHZwdOSMMB23VIHS\\\\\\\/DlYvPIiC9oI7YdmnWREC2JjjmBcByxVlmZDoGVSyLWVTh8KS2Rs\\\\\\\/pdJXeaVSLor3dTfgufN1M2pDauFYHH2Mj22GO94\\\\\\\/w61N6KmoT6xrXrUNphPzSOizAniB3CDCYTTE254VneTHl3s5DxCgmr6kRzpvPGk+OXE\\\\\\\/oVhSyE+v7fi26SWTX25QCV6kSPO+hHXzVaHUXuNZcbAZ7MHccfkpKevMmVR5StEEYBQcx\\\\\\\/4dudjmujVrtlzr6ZlYpqEbpuXZn2VN8cI53IpeF+kFGjnYVNFYjn9vsIuImQVH9KisLF4dSdMn1B1luFpDY46\\\\\\\/8SqxqZXSblo3qtpUHNTA6I7BDix2NbxaF9p5sOohTSmDZCrozpcbVF+9vUUex\\\\\\\/K7t2sxhxrncR\\\\\\\/Djqwap4p4WfMgtPB+HnevvNghqs8bBY0XMl2mtX+xiiBWWErcgIuTbJmpMWNLtbnBYo2cPujERpP1DHrCJNgm4qikgJWyONgxt3LshpdUFMKIBcgyEX1xbWPR60yis2YGCTmGUPdaj+TMnUmdOZ8\\\\\\\/H+9BRw6btsSd4ErSX5H7xImmZHSogD\\\\\\\/bBHh8zdl1qJiDHg87rLwC8HApUmiEay79eKG+OF2oYqErdbe+DTr2ui04LhkORrLBFyY2G1cUugm+pokKvJ\\\\\\\/Qv35viUybfsABgbh4P5Gfl5w89dmNKRbySbZVn+seu8QtNn0yS+rVZdtNtIwp7t\\\\\\\/qYw6\\\\\\\/igFbP5LnGATta1KocUJ7V9k9p7EQ829H7pslAGD3dwiLfwACACSH0ZaPIN5AJPJSTCOFQ6iHcCDO0wCCou32epNL4rQvFfv67KgwJOmPhmdeSmKCjJCYVmF9UL1yIuxYdR2pKRLEdo9qUyh4qBcdox3PQh0Fy2QlU5T0D9sR+qsjilQ49iw8\\\\\\\/oziem0Rl4yfMezthAxXEacXKIY1AaWeJRa2KNb+akAkUjZjF45Ef4JyKvD7HTazafv6eP9d+BdOkSX6MZR4xF+7ykuVqFeMbkYIO4jXTuV7EbTwKVCDNXv7e2yL6E22TR0h1g3gRdtWtSFCKDkRjI76WLoJE6mkFQaVhE4uq9kxi2uQk8S0FFzPNwtXmnLY1fU8RccFQMaNptrZMxIJt3edCMk2c26NYX6TeqP5hjA0INjysCW\\\\\\\/5rACVHBAwipZLYPIl6XJqo2bluX2eT84eLAaZjSlFZYouUlLWf6\\\\\\\/bP+NX2+o9bJCJ3CHody+xPMPpFbGJtGdM\\\\\\\/K72PgaFEp4UrY3sDJ0PZaIkOzW2JOs3FddECrFS6QX4pbkWxCR6I5hAQHCMJkeQl\\\\\\\/F6Vy+xP1wN3vs\\\\\\\/9oeH0bIBOt2QreBylvkpuq3cmxvmRASOBmneNkTow9OPs46khDn0Fo+8ji6HtuKVDC01dh2BqWaHZsk8xn2B5tRAi+AsTlLVO02n1HnuhLk63heaqHojEa5+JXIeSt0rpoaB\\\\\\\/G6rxwk3gjz+dGyocizB4XwhQEwCrCIrqEm+dbA\\\\\\\/LNC3oxzhP2KoN6Ad\\\\\\\/QgsE7RvGE09A03tRZw7u8pzIf4V8dJoRGxoSsLS1X6keBmOCQnRajd+Ehc+qXQJ2yJgCj0OXbwTL+QMvaLEwQGsmGhvNa5HuJGnb3zdavLOhZ3vHbvXevdMO8wf\\\\\\\/FR+1JVLUBdweFr1\\\\\\\/1UT+OtCkdH9qJnXTYcmh6bjVHXULl2kKI9ZuTM1gc3APYiUWMkNawTxCqpq3PVgSwVLllTqO5oW1VTecLuxnVCFTL0xw+0jxPWkKxCBxjxRAQ3\\\\\\\/lO+Kjo0jScToA0FwTdVLW4M6OHZ7Bw1qVSYJBISU1\\\\\\\/l4nBVOCqoDVshS7+JCBne64yRB0q52v9wfas+RicL29mXhJxGIHjTsk95wEsEL+6\\\\\\\/neStSxafbpuPnEZESKxw8wqJ6uRQ4\\\\\\\/e1baGrunbhWoDeS+YPt\\\\\\\/EWCvV0zzP+gsHtR4S5tFS98fHrQBlqnkFcyHZbTlImnNBmUuQ4En0vt0zVJr9bj8twwVICjrYwUv0QMJnlM2K0tbdkXdvBoju\\\\\\\/WcHJAvNwuLBKKH9MeVq+K6HpPr97so5RAIfVjOjLC71jZK+HNp05wEroOCQDaSTa\\\\\\\/DQhpezIK1cOKYa5NYrHLhON5NnOQWAnCpr47GslwuWoB64ysq+vPJBuZALugVQ1tRexzyzpNdoCPODZfGLHybgETR1rNnX6S22iwz61wqgdLL021VBa5A\\\\\\\/yoZKC7qodOzre1fio9pLvxoBLYL\\\\\\\/CSXjyhO+GFV5fAApet\\\\\\\/VR1snUhwqFgsLOby67dp9o7Q633s4Hg9b8ZnP3bCORkn4bLv+oHagruf+n5K7o2bNBpz2tjGOzxfSE20\\\\\\\/6Y36BGZMlBVl6sln3+qU+iESbvO8S+nBjsXJrr7b7KKqQxBNAJ7l10s83\\\\\\\/4YXr90xW\\\\\\\/A8jc4qml6Dh4Rr4JQJ\\\\\\\/a+NmD+G7HDIhV7SFzoq7SNhzh2t9FbO+pC2YJpo20wnzpKJFVKmBFJi4bRB+\\\\\\\/bbizDKxnW5Wt\\\\\\\/xFGLz\\\\\\\/4\\\\\\\/pOhLeM+0uKNKWPuVAbKg2Wr1OwVmpN48a0GF20j5NBUM028JAU86qJTPYF3cHAVeqOiqkO3XrUDHCxX6zofIaQp5OcF5gGKNjDGiKD8m1W\\\\\\\/YP0JjbwBHh5DAGgGOvNomPRNBPgjDO3gzs6eYk0yDmFJURGUTsLy3I9ElzabS\\\\\\\/F49jAVmOCm7jsRCOPwlblSHQeQiMmxmxhlPkcSAQQbaU6VavvuYbAhHhTc9R\\\\\\\/mh9OY4yfU3pUbvqno6hVdJclFGqM8VzAuBvvJ\\\\\\\/ccc4jpPb+INn3DKjOg1B+pewTG5XRkfjRX9L3NHPyELaDehU5fvm\\\\\\\/bgtQO\\\\\\\/ZerytNWnXOXKJsay+VWtKDkKIaYfvgTLaj93LH2AZRDHok\\\\\\\/EYlO5P9w2MDgptPlarZBQe5\\\\\\\/Q1HkT+NOe5kBgPOEIwfWyh5GD+qy6n5vFqHlIdcqpsok6IalY1fzgUjP7muYMb+SyogJpwOrjhs0nRSWdqL1LKa9XOSlg4UtecvrGNM+a3M+hZsWZXjonOcx5z+kqVuyqCAJKb3IW6zqs8XwlXThXocKo5JgS3qQUWqGa6GX87NY21YsMhmV3vmdM2ja87WUV8uhxs9G+sfn7S\\\\\\\/4uAiUic4UeKIyomMq9j2MkfX+mN7Ay2RKf3DK1qc\\\\\\\/nMPpnc41bHUHvb0CyKzatP2i5Ahc\\\\\\\/n8BMtmBHKt8M8oqr\\\\\\\/SdDowGC\\\\\\\/F4t4H8QbP43i25FBC9htSdUrOsFYdWhuLkUB078nuH3S9PAw1subjZxtbtUaWsbI9ZKlKFgq+YFypGVvz9DwL4NFjc6xBrizAlSy4WbOQNCVCJdEZlBiumtXyt1X46OZ29hwiNyz1emnhXNxIf\\\\\\\/2bBeQ4yykjb2T\\\\\\\/sQoGJTT5KW2OszidnVYjiWEIOQj\\\\\\\/64pvwP6JCx7Fw4VztU3uKZmM+DxptMkbTLlgCm3CkFOLL9nhN\\\\\\\/VVkXIRzUCnHmV3vSFJ4BS4FKfk88HnsZqk2sRlkqq+rp+YBWljENc\\\\\\\/cHrfiZVSp+TrMPp0hGlxN60zBOf99QtHdUX3RWZs8OlvdpDdV3+asDcvKJ30cSGYLwErcV7Ax5hRQon\\\\\\\/tkcfquA5Ff2D7F3SY93TfK50CvRg59rCN1GrPdIFRrVowN0TgoL7so4WIl3fERnUMwYFPiM78Kb1Zj+nb\\\\\\\/jE42ko5ABjzmmZzeoxzzHtYYDL1fzkG2+lpROomnnwzyzehEfqndHRs+OX2Y4xGImOqx4hBWkmrWY4ndzvpU3lYeM5WrlstBr18dQuHR8ZF8zh\\\\\\\/HwJy1ZkQiQzIjZcCpWj2QgANKkBq1IaeQkM+FhLURc0DHDpPJbjs8ktZ5yLatqoq62TJk4dD7K0R91p\\\\\\\/RmX37P2QoDdyDp58znsffzF9glD1skCjaGs5zew3lsvtLdIMjnqt8KOjl\\\\\\\/fjBpOnedrK9bHBr4WeC6gXrfGyBrJqA1fkHTSRxJzBaukUm6OBOaCtFdVzckzAKxD5Rqjqbegto2M6ekl8CQAO3gDaQUdiOMUSgidau5RFk5qn8\\\\\\\/1Tq0Bn+wj9jlQ3SkAv3wu+P1Gbwx7wvUejAUzHY9cfM3Laj+8IUaze2Lg+9\\\\\\\/IrKLhiLBTfU3DoSG\\\\\\\/x8r\\\\\\\/CvQ+wgmVyxZvDGNS5Y6kRoR12iJNDhUWn8+Fnut\\\\\\\/S0KO0sCUQZI4eCclhQU+aUoj7b6cFvx8QvoVoDkC4wcZIQv7fE46KBc8dkr7XUmyfvRm2fDUhI8yJJdBHnLOJEhDxhrZ1ZvsOc8Y43F2Vp6GG2e1OoAyd2CKvtGgTGYlca9weQ0M0G0C3x8GPll\\\\\\\/O6KEjvQ1UmqfvkCwFk\\\\\\\/ap7JXA9Hx7iRWYOq+F\\\\\\\/c31CxKPTjMF6jcWX6ag0FJGUQ25H+V2Gnr\\\\\\\/6PMwmt+hMVNyoNKvkkq9y7y86GeVibYqPTGCdyLHdhR4ZbpsrBFpyf+uLpQNTRpJarGHA0ZqUELsPhlKurgKY+wmhnIV8H9FqK1gWw4\\\\\\\/Qn6Htdy5sfb3cGpo9jKV+g\\\\\\\/ealdAstcegRX2LJUfwvOvw19R+NgeTTvIIuWEAXvHi3i2z35EWYIF\\\\\\\/SWdlq4s1o4mQvliTzRQ7XDRhY+OX+mawv5z0IuTbcM+YVMrh5nXShYM5UF5zrvCEoaYW+lB4Jdw2\\\\\\\/XrmkXyt26bXRk\\\\\\\/pS5G8ZjrXUm8WhnM7woGWkgd+gcg9Cs6cTwirjllWK046qZt7unqY7Ag6BxYZjWCDewMirHstDHU1d3Dycgt0Cv8Iau4kZECwTR3gWREZ7QuexvB7UUXJHoAhWwAu+oipODRl74FzKPXFB8dZLtEcJnyGkC65ZfzdwteRjyzNFAwwI8lzQX2Bb7CSwbwtxw4G2TlGHdoy\\\\\\\/DKkuJq0n2UNln5BL3toz1OXq\\\\\\\/Hawa4D+PwUPmXkhPcCnWuVMx5VpW\\\\\\\/wUuwPFm8yG5RKqFyQgnBDStlJt7x71E+NijI7XrEIoNic1jRJ5XyP4Q1H5sVKezSumf9UWFK93I7L8brKW+8OWRHwsIB22ppeZLHeGg8vap\\\\\\\/yLs+3nbHMCnhB10+1bGzViMYiLGc\\\\\\\/2hmJZNzn5+3917ceAUuwx9dNBowLiD+\\\\\\\/aow+SNEYdui5f2uX4L6I3GvzXx0SR1chrtF+QiJZFH0laxP6Jr9N451ATJWRQMzU64eV+5wEfSG4pxUzWrMs8bKB1csV4xSNMXCQcryv11JmqP4a1YKk32bSJwrWY26iN5I5Bpk1YA3HPXd2r5QqBWRKPMW\\\\\\\/AwKQgdXB6oRn\\\\\\\/nCUpdbwT1bgPEGyqbyonZYe5xlvNZTT1TOHuO2D1Dyf1+7kSviJ0hjnfd4AMCrGBNbL5TdNBs3YG8\\\\\\\/7EyBzDXs0Gud02rusFIzWLqiipj08QgwE2xCI7DILlLItLpFFIUZZ9nDL1qTAYtM+cL\\\\\\\/ZY\\\\\\\/dRmPSKoWFm1ifBBOpSVZOQzkWPu+N5uP0aPjHVne3Yx9ujaMKM5wwwQH1M\\\\\\\/aVLkTVEFpy6ikAmHI43ZDck8wQfmhv5Ix+BckqOh8N77F5T1kqCwG60Q795jEtC2D+u+vpK5wY39mxBu1BrUdNpe+iqI2I6SjCSHiq7tPVpQjnVQjiRTLaByb+MVPHchC0JwMW+O+J\\\\\\\/U8d1JEw6NEVLAq5kPnl9WdNq3BzJGxRliCC5jCT0c9B6Xd9ddWOB6Ib2N+Bn8iCoYtls+WFGPdeMUPd\\\\\\\/wUbHjoUGsMth3kyBTm7lJ8tH+fypwW69GPeRA3FLTGZIMdPje4AWcwJrjpAoAl7\\\\\\\/R2Uz4CR+Q6lOaSj2Y20ch87Tc5IGpbJE4BMe2bh8faPFQrP1yIqm6BzP1BuJELUt5zR8EQ3TqK9idZaClGrYIrKao8O1w98nFMOyyTvVKnrWdEE14ba0emsw3WruADDF6wd4KatEe2bUYeka+0N4gXbWGHaN2kOAxyjzSss6Wg8aCrgrf9at0lnRfCcEUUrvzclbuUqQCRargBYcc4OFYyXpJw5ba5TNxNhwjmUC4gPO5DktQb\\\\\\\/OVbuCPJbs9DpUcZr8FhVQ35qzQbF6VBZKoU9dIty3r5VrXrKCyxwN2uDfJAvAIVE445weQ8zXcZ7PIAO3ZzJcin0WQVzxFCn2d4nCuKZ+oTsxEmibPBND52bxBP01kOO81qtccDyydwWFQRdK92mSJ36C6rzjPIvYv01yaa9RAl\\\\\\\/0e8Ha39hRp76FvfZSQr6XtkhOf3F5bGo7cVSYm8kKy+cZoLLQZB+vphIZOjZ4ZaY4y2IHLOgHcBDg8lhLI\\\\\\\/2zBQTDjiTJ+ACSEUBCMM7EZ8pSGHJpAWjNxwUGMTSG5qNYe1BOBPPmeywdo\\\\\\\/AkJHyCvJaXQWa7HTaqZTlfpshdwrP4mFzMbTxt+km0vCgN4c\\\\\\\/U89ZzQ0a+0RkPj80YjDwV0ZN8EkpGH5n86AEGmiyZotVj+9IKvPSfCEkGOHcfWUHhKulMV6MN4wUKFiTMYiXWLwNzeij775bGfYVf+jyMYq1qH4pcFWVyfoIkEB0XTbRORhN90yFcfjAaGNScl88NF3XeLe0JdENRFrhw\\\\\\\/yN15vS8NPAo7UbmVDik\\\\\\\/ofXNMfNjMDrQmicbLxoYEtHqQm1H0IN0R\\\\\\\/aRmoXLjOHNnfK2JJXYGCWU6rna6fUYpmt+FBa4F2uIW3sWatT3pU3N3onymuZRm47YnwH3DSKgdelJlt+iDLRaGEPRRe24yzvG4DKRJ+foGxFv7RB4zYPGmfiQT9mwchmqw187RUAJD\\\\\\\/l\\\\\\\/VtylcNk1BBQjMFFhZP+HD1ruSLzWCrGhCZ7Cu9O4abSxSCzNcodDuhm7L6WX8sAm3FZDeees5irr5o9XsMGsdDSSUtEZ8Fqmj5n7CGEiriqy5vDJn35ujj6UuiiWg3NDZ3FnT+UaMVq\\\\\\\/+17BpF\\\\\\\/9T+db8jyutGZedNX+rXB6j01R1E3T0Tp\\\\\\\/37OEEdIk1xw53Iwy7ITM6D7Bo76lImDKKd2AAni4M+xrBRJ3NL\\\\\\\/2CgquMA7h9UhyxEE9g7EwaOHLjuWtk\\\\\\\/s72MMC+RlWTdlvUsx50jCDiAgazh7Fg4QRk\\\\\\\/xBhKq1DJUW7xCKVg4MFlIgz\\\\\\\/7xR5kHyBXLDwybJHf7FoKws5E5ODx4Uipz5E2\\\\\\\/TTRHzDOwOUOvqvsod\\\\\\\/LLPJ8oL6MGUK7ddMTWcCICHQjZ4HetFgxfhxeuq4iQh59NQFrIS8wjCqt4hEWj4nCFHbxQTpNUbAzTzgYq8qq5HCq0B\\\\\\\/DUWfejZgqLcToi1P4hEpE4STiTPgMm7xELGjezuj27yoAtsLnxvAtraKQCzxqT56WR8f9BByZgQ04HFl8gn5g46afKN6PacKMAkExTVGXSEMeENPMRkLPwGE5hUay+\\\\\\\/4cikYUclr7LucUm1goJJiX0pyOji6UYSFG+GU1MFKC6xJKNj5ohlrd7ZVm7\\\\\\\/yKg1gPaEDMfk45rR09ibSxhfV+kMw2N3LhFRKogBf8nbcdKO8QhP30BxW\\\\\\\/vLlkuzF3lANtWwuEdOZ+lnk1aO737sq9Qw5r4IiuXruydqHkqimowhWHBlcum8VDouCeH2v69diR5YSHeMlJyvtr9HpVSHRimEBKNO25jJVc6SsRQYm5lrVySyrzpTXIIytQPDUspbHw2jsv96mFvqvJ3Vehe70FaMse+LLu+dLV3xbHKwn0dJMG38NGPBg6clpRdBLwbhYONohNWx6Ikp61bW7ybXWnG6t+KyvyQ1pHTGFintS5uAh44sk\\\\\\\/8IDXJw1dqpOnDDUqRlFy9RyHrf5f3vnFex2sji2sXTZSJCCGZxsRCVqp8r3ffvLJkUYCRSWzl5r1LYFDqy5ud6jSjQ\\\\\\\/2DqS4LPRIMOdeuFKaCUSAwVYynQ7Se+Ri\\\\\\\/iRvntaIoGTSS2te0CUhiOGOJjKmClSlsbTpaLzthFnpdlKEbtnZp9\\\\\\\/Nm69iWKgzu7UtqQS+zA\\\\\\\/DBy+PZ\\\\\\\/K3sqxujlntmxpyhcT1XndQo4LPHKSLaZpSwTe6HZjyuTIhVXWRcGMPSlpiEfxV6aOqBFDGd1UL9A28pmzwIKaQKT12+Tguyny5Yq+8oKxNnUXKREzX\\\\\\\/p7O+dnc8HU2lSthwMszO\\\\\\\/RzB+3CD0zXBWoH8BFG\\\\\\\/QOOIIzwcruQ2+Xx7MZFMVkBrV+ISU2X67s6mIFDHqxIdEwSDcsosVCXs8rOmIzjtrheJq7x4qZuyrWAWZrxdGfGO19Mn64zfQdVywBI4vtAMwqbybPWRemXDDlK0ji2zdKloHNUqDtV6o61rb9bklx2VKc9oY4InzstX331H3pKzPDKFVtLlO8YfB4NUxVwJ9ePzXIIcsV3yPEOQYudbZwyqdCDYC4M5C89pv5EviWIY+dc3r183qp6m\\\\\\\/GV6e2kv1kpnb0qPPB2i0fMLQOxN0UVmryEtiVcgF3gpbWUsGy\\\\\\\/PMeolH5UCsEeVq59QCC2gycskwjPoUPs1t2yi4vl4JlrD\\\\\\\/hXHTIjoxh1IfDdX2fmg+5qMocqAi\\\\\\\/0amw0OHNhaQnTFk643maZRyNNlEAnh0j4iaeifFdZ7nhwF674pTrys34VzYGBUBVBSXMjSWFzJbQ1uDA+l+9GaCgcQ7bB3rOjwX8aZcHmfzPY7Apcpiqt++xpiCwStZJHxIOop1nweQIeKihwBtMSac3uCRW7l9y1fyCj+YNZW4BlCmVo4uecgH8apMor6wvoxv5SCsLbr4I1B5+oCrMji0Xa\\\\\\\/4In3T0F5h\\\\\\\/j1gbfdoxCfJSfQE7bOo3cPmLjJIDHrxwebBFHarXr4sKF5QI0F3S+76UiJQm5Ut3L6LvFsh6431uAM6E56sIFoNRGaHY8ay6RtCuCBfMqVoEshAyK5VFGLrV7H+63tys8kSiMbFg3akrMs0t\\\\\\\/mVsO2UtJiS1\\\\\\\/CvZkjjqEHR4DKFc2FSNKxgK\\\\\\\/+DsGof1KDf9GuRt1RKUGGOOm+dM6dkMJmKD3aYwCLnPV8yEhAGFv2k0wbAzYy1ZNLgrU7U2d7a2+lllHPmZ8SsVhXxRZ9IXramnKhbeXeS7u2Eo4b4BOV6F0S6M9topcgSc38rE8qM\\\\\\\/ZKgLNDL9UnwfpAD+jiGTto0myUrchlloWp7yHAAHpfau2\\\\\\\/hvNEaXn8VK6Fk8WwSLDD8QifhA4OH+cJJ0iw6eGKFZE3BjWzEysV8kbuABy\\\\\\\/geOXebVjAD76DDJDljMlqE+EoxE2VVDbNU05zjKi3qW3HayyRArQPLmY5auFdibp\\\\\\\/ejTjsk16kaxcgOF5x98QLnp6eCkiOgay\\\\\\\/DWzC2iCvZ1QuF2xQYB5iAf+dtzdBvLgAZHlmpnscuLYdvJH7bI8oLgBzdVdTjdXTxye3ZeBqDR5K1mQFl8DB3I32ZV1+BVxCqfS\\\\\\\/46LxWmFDfTpTdl46BAdH1IbUrByJ801lz20qfOKOyEp0qOCuCZwH6q1vXo9BuKdf+3dBGEmDeaLGuIBQbABYZOz15+Icw1iyOmvjvsMxi6ZRA8OH0YWbCHjGrj39CdQTw7KlhWeABTH67sQCDyFVPaUIz6O6KEnyIjX4JHB2DW2rzI8KfeMp6zXq0Ml0tML\\\\\\\/QDVoJSa\\\\\\\/n55XENQcuEscYoychb5ngJSuQ\\\\\\\/3VSZ4vJMXu830xWFJI25PdUz\\\\\\\/P0ipcJeBAFvqwQ9RlShqOF5NIlx3Oymlx6zCPef3y1n\\\\\\\/xB53lhmzTYRuGpOlXUKzPxuTib1DD0zlnCh\\\\\\\/reTTiH0CvZLYVTiy9\\\\\\\/v3Ohl\\\\\\\/1QDWeKOyAUtIrzX3Ogm7F0XJSbBRQFpm9md4qA8O8ku5GcqHdf\\\\\\\/XyM4oFSX2inCUoNrHnuXx4Ky6xOg3qk4RoxjAS5xKdtOUnjOxr6nFlFsXp3Cz2uXiaNf02EKQfZnGYfqJEQLtkjORg5soVelcHWZRtVnIy\\\\\\\/sNDtScfp0VngZrXAwgsLLhKY6jxat1\\\\\\\/48g+b6oySRInVU9Tf3Z0jQ0yvaJy2HDFJ0z9iLOLcfVzR9RzDqVpQXAmvxSv+oLxh9rrC\\\\\\\/yix+N3g\\\\\\\/ra5JfivCpygEnUbD21wrj9Z\\\\\\\/CL6rAxQjK8XNpy05nf2\\\\\\\/XH4FoYio\\\\\\\/IRnabDzh98Z9XNcAgB1ovf5CB3MTM8b26MQV4yvQ4whkEzzZyqBBXwtVg+rjoCtXdaFI6e++DnENCBtNTB8AHZSbDxmhJeE8scb+ccrAk6uliWbjLa0QC7NZbFMAPUpeIFkNu1UGRR5FEA6DX3R5Oa3vs9SH8EHTe56Bii3oFbnPTalg3Iy7UFrngq99z1hXK6wberUp7N7PvL1fUmINNcjzZ50zp5zx\\\\\\\/g6dSt\\\\\\\/F06oZaP1ZXd1FrRiMSm9bCQgItFsIcJZ0NgBSV2P8FYrV6xIRAz+N1D7ydVMvAoF\\\\\\\/yCs18NsLHE5TB2UWY64Wwq8hzgbzoE1hljqtNzHKDHoZziWXZa2xHWHyuP1wwOMp2N4euOvmsULbhM9qOPSO\\\\\\\/QDmeY+iJYgZVK3fpnbTf6A4aps7pF98eGBmNhzNRR5EIcspzsTHNIYV1cEbY8hZp\\\\\\\/MC2acPXJo7VCSfREmMw8ccjAtL04OAL76VbWE\\\\\\\/Q3pIh\\\\\\\/y69cg5E874WkLSKViggIOAsMeOYVTzBWgVglbrvLes7dGRfXBo3sdwYB+Q3+eplv7XtTlo6lx5ggAS76X66aE7jf4AsEdLOiEiXjAROiPkkZJye9tpGpniWlx8C\\\\\\\/+moUGdGsV9YdhoO69WbKiHWmzgAkej1v2VfeA13XrsSTtgWjoU+fv2uc3uU1YpcawF4ig5Z5EkteFRlqJi3HwG5wvjN9WXqJwJdccWX4ENClhekWS0vyyEr3kVbzueFfrd5XPZOJGfgEJ7lih4gO92yGNkdH5ktwpQFCR95+sg3UpY9AJ\\\\\\\/G5nR8q66qoKPDLrvbTVt8SLVXUs+mkDed1YBtACABUt0JAKVeywJzA8eQFZKvpcZOoz9JbjkOg6s2k6vTeEwanuyoCftiuOuO\\\\\\\/trcH8nntSrGiSj357lYQ\\\\\\\/yYF786mMluU+46aEgB3YexZC7l+vU7IaTa3Ai3KWHaWI6m5wwV9A0g5D5dQKxzah5ivJuBqOGRUqEtK\\\\\\\/mm\\\\\\\/EybDhldaXsvbQXIdwzipKE6ALtTNmWB8LNJjMlyx4Ol1ZQSUceImfFg4ijb16oE8UiXdlMkyV+6rInfFkV8n9M1vE1XkdBFU8BK1hS2VjCwKzW0QcsOoc2Zr3qHSXlqFJY0d7g+w38lIlrBFyyl+Cq2lvd29BiMSTp1XHpjUp+0U5PZt1cAKNf2wFcaVyAx1sGSLjwK8lSfl\\\\\\\/8SmiMHYO452faSZAlsXiqcj8+MueAPiqnsJSRInIJAbSYz0LZcs+mhHhgJ4E\\\\\\\/TtZgM1NdGxfdIJc40zx5jw+uVV4vMHdsOBvqI+NHxvi9ZnFzvinkYILpPE6jZCNs\\\\\\\/75dm9Sz7Pl9w589uZTa5A9q\\\\\\\/SGnbNmIUEeNXdiKQ0zX5ijeHXn++4jDmd7+EAr0Lt\\\\\\\/aWlVpMGjrs9GSqiuwJDbv3ZwCfSsirHvc51gTJnwMOxKRJirjQ71ok+PlhZYlY\\\\\\\/42lAanSadND2MO8H8F33ooBfEgBv9tPwXG\\\\\\\/MzZO\\\\\\\/\\\\\\\/t0PWzg+7HXLh4ZknBl4sM+fxtFCX+rD3SAJDmyHHhoiMc6a\\\\\\\/Z7tNitbUkTZlC7wiANKjDbyqBlSmxK2TuU49ORH09PhHAbpzBrYnCUoTRW+uRS\\\\\\\/LOcLKYz2CVyYQiCiwLeIAKmG4xBHYc+aQx4OeErsX0nOprL1UVZnn0\\\\\\\/W93gvrz8pe1vY\\\\\\\/JfYnQL7FTbQHlAx22j47GRt5+Ute4wt9wQXKfkVbScfOp8hy5QSSgoVGKAb1Ty2FMx4LLy3m\\\\\\\/lu5HE+HgiqL2nIRGWx1LvEj6h4c4xdeQYZcGJSRO7ab5GwF6dMOQzGwUIed6oE1T3y1AWzBy2bMdGHmZpjVpoHam6zf3qVUbe\\\\\\\/Seyr42GNIaO4XDc3yS+rhH7JrN2A+sDikT4qV7jC4zs3mCQAiXQA8JHGoyZ7I68d7nuVB3cx\\\\\\\/xBP6ic92yKj1lFsHq5pzZhV\\\\\\\/FCoTM6y+QfHM8ebQe5nrZbhK+79QLRwnxK1ty9bqBjguklcdWtw4KIke83ZV6oEwh\\\\\\\/5Cleqx9yVe5pgimeLBdNUVqnT1G5\\\\\\\/DbkFnhKnk1SNMFs+mWUWO0G8zu762kQR7NiLZwXiqATr6Ux51R+gBto0BNukNn5rn0klvnVn1IN5rGdLXYJY5lf0zERGkzuatmvqM\\\\\\\/62NPGZdQuhg6ZEWRHxnt80GT1qlXMwFQo46gHC2gqQqQSvvhmDthQO7uXGYQGm88Pd2ZvCZ0yVpUovP4zl42L28NAOg1ZRO4215kD8qSWF+rdZ7REOLuy8Ba8EddBscLUF13fGJ1WIlCAcmctopnywZGQW2CDtcfyBE9F4nvBqEyZt7oHhAT3GODO1WgCRGf7z89HBaf74cZ2e4GemObaSOH0PxXXx0ZghDR4VWcRMeBGvz07srrkfMl+kyq8c\\\\\\\/PVx9wYtFMOqXfwssxR8el5m1vvAZbRrQnL5Y6kq1tFduIgc1wYw8GgIT8IwL8RpRWVhrOPUjNZEAb1a\\\\\\\/pA04eMKURoxJsoqBwIcrsGFYyoCbOYnHYwyH7dGb+1kXruhCyUf4PCxq+NDR0eaqqxG+v8cHKyb5K+JkImar1r4lCdbF0ddN28LOGhitnIxoJ33qQS\\\\\\\/LOsz4+viPJ\\\\\\\/bdK1hkKcmcxxTKxTJGzBkdWtEAzfyBmtjJn55hdQjKPX2wLMpQWjshwT+FceKL+Y1dSoGNWauqXM4KldWW3rXVEl1XSyQKuze1aWb6quSL\\\\\\\/BCcDqHydhg+MmYlBK9bJBy4R1oM4pOganhhqAIirb08PElLcN286YLg7UwzMwNsQm\\\\\\\/HLWi69V8n8bGlmb0bwN12bBg2HWkn2SUEHd5nf0cM9KcSLldsItl55YOZQPsRbKc0Ey6lGzC\\\\\\\/Cflj7ptbFdWy\\\\\\\/YXi+mtTxUo3tcX90P0YaNF5N0bOSHplgniGMd5vwfSbc5Ckectu4r4FmLUtd6TZ35AITZfOhlf7\\\\\\\/hRY0NH\\\\\\\/8ieu7nlGGmoArCiC\\\\\\\/g5JrV712aC9uuWJaa3ubDt1QOuV5CuBATLDDFwF5k8Zm5OVGVzvCLvKSUR0RSdcgJuQk9K2+vSqcm9SPD0UmJsFe7AKVUqsFWJgmwCtrWVA56jyq\\\\\\\/D\\\\\\\/PJphUdsUXKQmDvXy0U\\\\\\\/uy3duNrvPkwL7lvQWVSf3t3VylsEn652fcpTPbFGrLG+6imtxSeD6wA44snz3utX\\\\\\\/ju+r1vLX019vvGjcrRTa+hxk2FDHSJNmeL3L3bbtOOacpV45NrCcbucaJ+hZ9qMz1pGHsoxnialr9e2h0\\\\\\\/Yhv2Th2g8whcmMvy2UiuRjDFEAzEFfv8yHaZjQelVSZOLoFgjm4r661S0v814OpFfn49sRw+q\\\\\\\/Df8zaLtA9sPB4ziO0TYK7A\\\\\\\/apQrDF38aDqfsiUZ8jbBN7oA2lgUulB2S6xOBOIofUz7nSfbmMFC8InAMWo9d3jKP1\\\\\\\/rms41rnVYvUh\\\\\\\/ajtuA7kpakPat7py6o\\\\\\\/XyqZT9C4yLFIjXMrZhGpf1XqJZI57Y8fB5p5dqymX38uisZgn0+xMoM9VQ7xlXLYzIntsgedp5u14YCkXXw+KZ0prrA2zb\\\\\\\/ToujhruVpG++6wYSx0am59qf\\\\\\\/CAnZ7Noy27QV6dWLvgFT+FEA+AP1cBlJ6vEHMvK9CdUqtruBK5DzsPUWyQOkwmbEj6zOjR5YKuN3p2psZcv8DA86T33Tas7X0Q5ZVY5NdINF1wz9G2Do0URJaJpUIu1hR08artk0nM56O1psAC467mN5eGTpdqGRDhGnZkmAj2sZrF0rUcLf4uXPQkgOqqMsyT4CbEPJuAvz+F78hMwgxiBtbty3PHQSZSkhHkci12RlorbDTaqjaDI5pmbGfT\\\\\\\/1YLSXQ1st30GMr1jJjJJZmlyM4ZTIJmPuRfTW47BwDA+IXNFmpy0g2JvV5\\\\\\\/WcjmBTDl+U1UHBSjRoxM9Dansemt3jxCqmLeqZ1PLQoiLxtNei1Zrc8ObMovD9kH2mhef9u\\\\\\\/mA9Qqf\\\\\\\/rXUZqd8h8RBYQKteozZwNGpUmeUzQQjzoSZKtmh5TXiij0k06QdOEbiFL\\\\\\\/P5pou1lQGB4DXpBblxS3gR7zLcW\\\\\\\/JLrDm2O03mxMaDgzfp6SkgeZXLi+O\\\\\\\/dgOm5G5cUunST0cpKUAKwVexCiFex5cU0nPuJ5m5BxjEgPLHmhhVC+A0F1fZ1XW1fDYYO4IPGXXPIZ4Jlwgn7pColWHDXN+7Ol0s9xDxr\\\\\\\/OMpadPdsuFW\\\\\\\/yiMwS5WvRnnbrSLkxv9Qy27ZvHD0wUsbtfM5HlQ3dyAWpzki4497fXbmLrQBoWnDNk3WPUA4SVP3RaOFEpFktK\\\\\\\/7lGFbb0bHmWlnmDH2DF5mWqLXAXh75Km9q1\\\\\\\/yXXlY\\\\\\\/0qmaSsJd51Lx4dHsMCLEU8S+fjbg\\\\\\\/DUab+IPvHAddaP8LB6ca8maIzCm19nwZ9kSLFlt8FojbqSRsOKznpR26XjbDR+MCaVaffUn\\\\\\\/p0oprmY5j18ns7XX8NTZI5kT8tT4vr+yCNXUvGyV6m9XMDwT7sSt65bk+Fr2\\\\\\\/L4K2hegK8SX2Uiu4a89QSE5I3bKDaqhETX6IZ6Nt1VCUxX2afdTxAof0DU7pfkpfkSFP\\\\\\\/wlmZeFRsO8Ru3spP5WySPtZ0SUjtcKYtNR\\\\\\\/ZyHZGp2DTF1oCXb69W\\\\\\\/4KMXKvtZxmXl+dKdYMiKcOjAaGVyFsn5+0tyFOHrYX+m\\\\\\\/Xm\\\\\\\/htIEP8d28C4+vAl72qERnAjZpCY\\\\\\\/77i7T6WY4PML4tL+KSMplfcKTKqxU+SGGW7QervcfyqvfAMlez1NZ1T7KPtaMc2WO6sqX0aJstS0qiq41Xt6HkOSktxXTmVtn9dAZ\\\\\\\/09eqIgOXQLg6AfzazBhNapYc06IFKZaIAomwzmlCDFb6gqpCibCQGrds2Wrt\\\\\\\/qfxlvRfGWtikSjrxWGu8MWaeZJDd7qa6QfqmjLkPKluqFkLVJdSWuEm5O+9xVfFc5POPJcb3Kq\\\\\\\/8zyG8cqjegdHuItBxyWpc1jkqvbHHvYD9xzlLFeaC38yprni6kU\\\\\\\/jhBAgR4hJISH2F\\\\\\\/YoRzfgFgAI7BboGfDmxVtonSVMOWlTQcGjHROQWPC7k6G01IAr\\\\\\\/AEQsR3wSN+DyeCTkMcPseKXkVZUZC1SvfoYis4+BC5GSyvkejTtgDHVIKqEOQz0VrV+LLWblIerbvlJbHl7BpqQhK0T6c0nKXk6KK\\\\\\\/Nmgm09hN1jMLdmuoY+7TSmrkwtzUa7qivufTC4saTRp2fAJ4WbRJkqynQ\\\\\\\/HUf8+3I8hJKzPwG1SCksVkAzl4jJeEj6wSOGR8ILutHov3oZ+ax7WJ6ICy6VTBWTUf7rkKKBaq5Ty2Blm6btjsuUQE5S\\\\\\\/0R7tSyEGF8UbXffT19Qnbq\\\\\\\/jsaeHRBeWINcelBNhh9GLSp7m9pjTmH54FwkfeDbaSr\\\\\\\/mkqA5nVHG9bgmrHGPOJjH3esyukNlAx2DP\\\\\\\/cWPe9aVGbBDS2IpINQToaD6djyjGQjBqYtTJUoF3JSWGoCW\\\\\\\/w\\\\\\\/67cCtknvSv1\\\\\\\/5xgQZF6pSybL8co6hAYfSfP0XM1Tp9Z0zlufgEPlxx78q9tmQ9Y8STKczA7KNj+3djAYJeNDnxlFzHXji2ndwFB3AyE5TNjKW3T1Uv8Tf0K593k65zlGQSHjzUJUQMM65sjQTf505vBIeL\\\\\\\/PU4SoPTiGhz77OVhMGuZ4Qf2Vy7Y3apWbf3i3ACsuvCgiTgnF3tCIuRmcLIUDBU7V\\\\\\\/MR6vDEFGB4+pdfqymks5pOpi68rY0o6mPx8eM43byqMkhWgw+hSjLNLr2BY9Bi5ZqC\\\\\\\/FelLap33bInFdZQ7Keq\\\\\\\/gFObCd1YVV0AcXibZeBleBCluhW\\\\\\\/TZpZGb9R+DHBCqBLu7GxyTZ3Hvm5Lo+VdiTMaA3T8\\\\\\\/4fukDR0TniAwaSEUDu0celp9Wc63Gt1B04BlY7P4PuML1O1MMswbdMz5IJmHGIzz1Sx1eadzrWkOvGr1BwoqV36RSLhqCc80jIN1Pfa0F3DX9pyixQqKs7qQ+lTXsvKl+oEY2IgkD1BGXVaMjkcrOg7GC4qKicEliQKaYkeFlbomL+5ufNG1WY1wamQK78l201ng4rLOVhuty\\\\\\\/K+ysMWYPg\\\\\\\/1y4jNSiSMVrun1m3nupEBFKeO0pWAsN+i89trdFfjKhZAe+PxcgPFAKY2\\\\\\\/w4kfn465juPfHEFgZ5qr3BKTplf8jNqtT6YIrd5G9\\\\\\\/zfWsJsPG0dCuVx\\\\\\\/T5Ki83Il55s39mvQ5Cgx1i5Da9fuzx9wyEchQMzXu0vNacxO1\\\\\\\/lMo9tHriaOvzB9tgOzyqxbkAmA59bIDpjwYQNDY95MgLaXkUB21NFWJEBlOR9yxAzhitTd+o09DdBY8tmEz22EsEmLAilQh5RJFNJM3VmRjC5PFtzbmHVOijPSYlzaNn3SroUK44SZ39wsKiXR9tTPspg3oqLRKgWEX6nOlikqz\\\\\\\/QFCMU0CFVamfGwgDWHBclMpMWcCNFvh1h\\\\\\\/ioqtOYCtBRBCA6JjV4rwpB82klT7WCCY9eYdIULljFHUXV8KGtPF8sbnZ0MaLTAs3NNEcnOPhznMWGsKp5DrmAx2ShKLfkNuHWfLBZqSivbg43egCgLCf1D0SCJIRNvnWvEHJLOiq37S2CvqC2+NejMO2NMjPRfI4+8d+CtIjo60Yo8xcIRuioH0TQsVk++KkFJoXvOxd0ah77rbMKFtSDmWlfCuDjfDP1mGQnQefNlA08JGqAd+\\\\\\\/4cHfV6GHR\\\\\\\/Rm6s7cAsU4Dj9z9d\\\\\\\/GM1cC9bzz5ziOqOUzsJYU4LEaKqYeQ5GQLE9lvQyWvdR0vFnP6mJEdkovS0fcEQS1Hx30uYUQC7g4NKsEec8qUYYPQmHsXD1rCAl8Ov8gSHej6uCehroB05ZDHcGTny953O\\\\\\\/PTX2S5\\\\\\\/SEe7Bbz8Mf8u4fClLF9qZQXdPk9RhuL9HqYnqHHf6Cma+TlXZRH50DjVRaWzfvD3wNkHxaBTWWZF3HpkZ5AZtWevtPj4x9zJUVtYfZ9kyIkRH8egQfv1PwydvCW5e3Wpym0saUsXRxVxv8vm1j2rQXCFh0kPRkiVawnAE6Qr2nx263RQ2rWMhrd\\\\\\\/mhOeA9Gucj0njGFR5N\\\\\\\/pDD\\\\\\\/lR1eC8l9bYidtrUry4TSgJ5pZXj9BKHpyMb59otUF86ZsHTeXFvCcJwqcSavLckiFgoO1LjYD9XlNfCZVxzguDJST88Kyiz+tGM1uq3KStzxRoKbxlHp6d0N3jfwQmMQeX3t1DNtv4GkttGRdfvt45OuYiaoT7u5puS0WR442zhmmWS1icEiwW1s21x68vc2yVwkSHrjLLjRROS4QxHx7g1529APjuoe3Sws1CkUNTC1twnbngdftdtLMAfFWl3Jqk9hgmy98FTuaDzEAWHKrziUwQKPke6v4nZtOD9yTDTNk3177fA0KnRCGYNNOS44BE95Sl\\\\\\\/9aeEwKKH5v0pTk1VqgvU3r3D3qbdItEgHUmH4NS0\\\\\\\/NpVcz9S7AilAZPTC88R4nIJoZlj+tdDIzVBRD3739SaW8bakM602F4gArIYMwbsAgms6fHtzrkQeMvLpF3NV\\\\\\\/vOCHAIAUM+3QZZHORaXUsEPJGBLGCDoZjJocF3jTTWF0Iz5RA64J6K5IlQwKNKLkyfDOasbBVLrBXZgjAhNzdRdzIEjpw0Jae+RKkW0DlnrF5+cVu2LBhO4prkbVelPPzMOMBetpQoLAytSF3rFcBcgawUJTI6T4dATNInfuW+5E0k78HHyfz0fTsZU0LgCPmFY86PdonCm1yl9OR+XL5f6iDF3HiGPWj\\\\\\\/cKgi\\\\\\\/W4ts1eyp+3j0fKX8FfJpLBlLJBUApPe2UhEjakW3Zqb1CQBPtKMoG4RvNOPHQ5S+tpPVKjIr8t+Np7V6p7NFvhh7QdnEEC3IMoSSXrw\\\\\\\/EeYHbIaeWJgOx33fEAuvhVxNJ7YMX7XXRmyoO34RAP4yeCh6SBYj4wh3Dz2lsTYk05Pv5XFauaPSbWkgpoMAZfz7rb7f8wVryM5RwTy5nP8+g0yx0jAyJkuQ1wIind8Lp9FeO7neH4SToxMy5FZFwzxsXVcwCV3FGNKKVPEhGls98yKswlc69wkflLerggw8SaeKQWBqRsTl6PjnT\\\\\\\/a6re24f5GoIQZFTejez4rIAu8dED+1zVTnCwEjMdwkElflUfZpvY2srJqDQBc\\\\\\\/tZya5XlckkhWFhXAyJejelor3nNN++xLohD1XmkEs8joEzt4vPvc23\\\\\\\/VHKtz3TKVpNIbKASlRQXYy1TFZsJhzvFnwSHsCxGzfb7CkOb0nqHpars6yORTAj+Y5nE5c15c+DH42LVTzdAw8P8+feRcmh2iQVWhyUfh1E+HYGH5\\\\\\\/9VEUTQKjoELY7v1yPFVaxv2jghQ9J6b04PqG9PJ46A0zoD3PNlmS5t0hSxkacdQdo271O32xkZ+3w9ePgRKv4PaAlX5+d81fj03AnaNZGpfB4Dwwtbbyh0RF\\\\\\\/U6vu68Y8rMi\\\\\\\/qTHJ40Qqcd8vSbqIVFgaBmyD5jS+pIZKLEt7VQVI9S9HaQ3V86h2HTtnxBd6LxO0QB0JZOBmWxWkZt+EXlGM2WaS+1bVq6B9d5etDbGt83oBfhDg2SVMZWiLi\\\\\\\/qGfqC0Wcjd6pTSK55YveqLsBxOIhzmK9CSPJBsWboQFAdCPfnbnGCdI3PaTwyXfa7EpNt8TajsWJzrQbI20pr0rQyHZHL7iA1ZiQLNc6YocWVTDTOC8PLNP0H02QbJD1\\\\\\\/gr83DUKrsp7EfuEpZxjnxeDsns+5bFxWZ9pEsvU7+LkRHV0Db952xawaVXUDXU6Rxngf5U4bWsCyJjOWP0LHO\\\\\\\/eOy5+qLKJMTHnEGOUm+WMYAN66AdrCDz2QTVmfYch1lDwrCjl2E2D\\\\\\\/YcNTnH5HhfzpH0kycqwZayz5\\\\\\\/F+OybFd8IVYiSOZVpf9Vris0eMOUBVBUfzyjD6CK692PuG49Wq\\\\\\\/Jm9yjCBlF0a5yN\\\\\\\/hJiyBszPjjKGyOPLgetKAmJ11Bsh6mGxa1GgeHm+4hxRF9dpNCmHSSZdZYzGv4Cjyy\\\\\\\/CR4KPEZux1vd1e4K5yRrIJAgcDgT36AwA08A5IMKD+qTCeFjwWhu1eqxeE6hlU80cwpWiPDFrpjJVkZgJCuvZ5qOzcN9b1TyGX7rn3maf7k8ubYjNq4Z9wMnm5i\\\\\\\/nJn34QMm0mOai8aYqhC19XcI1fKz9L9mY9svFwRTDwqSnTfH+rL6YH7\\\\\\\/YCPqufRWlVvmYcfFELu7+n7U5DgsCjQCFhNf68jVUr1PjB8klzdJGFQK9h1FF\\\\\\\/BCTxeRo4uS4GXhgsyRKmbPyYwfeRD\\\\\\\/vF6rW4gYRIgJpVyGQfbkF3gwSfJxU6+IGvT9PfVKHXKh+Ve3mso4NtNdpyAKO\\\\\\\/jI\\\\\\\/JYglJXoDv8c1vo6D5D+iKfo+brN+Yg\\\\\\\/n5+7LcuhSOqFVgpUXVA\\\\\\\/uXh5O1mRJ+XeJzSDDGRCPxwJahy7gI1lXXWhIuCctDSleFokoRVr+h7Cb0xksacywceTzPsmJq0nM3vFwiNPV5Bn\\\\\\\/lnsqTNVUT\\\\\\\/ew0s7ziYwqs6zn\\\\\\\/hBde8XM9FCbfr1DEFjdmpNS+EZC0L2PnEse2HC\\\\\\\/FUfDBcUN3U4Vydeg80y9Op3teo9hmEcjcafwLHrFco+omlr6K8zyJ8pM+IoBKNFIO\\\\\\\/B3UFXO3xVgTdbOcu6YYtpPtgrn7BdP0lU0+D29egh+aLbq7eAk+Knz2Gw79a2B05BG8NUSKbQjulsmXYb9ibzBzlqhXUlBR2\\\\\\\/6T\\\\\\\/LQuotwqEyskJnX4ZYCg4c3eFRAG8NxDiQ7AqqT3UxYFYU2YI1Zqxcd6ggltJZIld8TKSQ245U4XumOPBb\\\\\\\/6B\\\\\\\/bhRqwghaHT83cXb2oxG8qS4c6lTWiX\\\\\\\/+AH9ZEuhgPFw\\\\\\\/jt6FoZx6W4juhgpQpCIcCkzQ8gzMl53yhTTqaUaC58CSlfu0fhay07d3bAyxMqvpEZsbscUWDHUExU\\\\\\\/1oUN4qo1Mmv3Jg22bY4XVVHztj5Qb4ZXnGwQWOUeYTSCr7AszAWagueLQ4N3N+vh9fmWqFqSo4nCTUp0n2Ovtz7rvBph+4OITAUnBpbHIaRcuaXZSNK76tz3PWyVZv5yFrvG3mMEGMqN5FdjS3IcMcORRUveVvjfrfbDyr6jT9plh\\\\\\\/5d+0gYl7Maii\\\\\\\/7ZobKOKZYytqO126U4Dd6YxwIQpDtU7eQMbt64j2hulZD+\\\\\\\/kHKSHDeiW1c6TMXRQpSaAf55tgjD\\\\\\\/Hn2U\\\\\\\/RxKkwhwFegBEGOykv5S0wSFQ09oEIotGdqPObZI2sbtuGVnbMvih8ErAVYYVmaCJPJyjmA0jw9+Ejn86TKzABvLcHCiG6bOs9peq1HXzvs78qiZnMCioHo7bogcEryHyEAEcWBUxEqw1e5gtS7Cz1CNgrkgpV7rKbIZPTN1AVJJkkWDBGum+I27ZqrOk+nc2WUXLX5oc3lXGZUnEoUyOYXib16ZgMCU5+rMN3nNq6hgvhzzqJRdxYOTXJ6fmstMbvAUft4WrHXiQFKgA343yP\\\\\\\/3Av2clmho89jVNb8N4Zf0mLshRUJ7yeY6RmFY4mRbim8kfgXxRbYqnC\\\\\\\/T5nm2v1\\\\\\\/d4GqOyt3JscFY5Od15FchgLDopDvq5K44jrknKyY2+z9m\\\\\\\/i8YKyG7qLOZbTdxrU7SozO2HvQy41FPheLDvB+bWfzViiDhuii\\\\\\\/i3Gau1r2XeoWo0DFeNLFs\\\\\\\/qdHpM4+rwmKCHGMm2gtvdSDbyyn2g\\\\\\\/Pm4BjV+49vyYL5TBrY5ZH40YN0hJXD+qVzu64GWoZaFUmjZ04\\\\\\\/gJ0sgDoeMefEhPA8drEbSBUufJEQEnJ+38sOQWhAfpm\\\\\\\/kzMZMSMP1CE3RdNRnQvPVlQ3LekUexfEIJmW1hMaLEqMuIeMFbtVAle6bvdWkQ75au1xykRcAQwhNedeC8oSX15Oxx8CkIr094toqX9YYBQlIjn3+SzG5hyAkD86Dsq6jzXT9n1EhLoyCucNMgkkRvGinaCEhazS0eJmJuWFEBvaUjjMMHNCEkNL8NN46U+FPzMOZUi\\\\\\\/yMs0NZF67NzHhobyf\\\\\\\/+Ofa2DNWePbGrL6TVyboxmgJzDVrJgekNCacQsls\\\\\\\/wwdZCTJroGU8AxWVwJTxZ5ONKXW0FCF5GPEbg0YIcWSSS9meYz\\\\\\\/h5ap\\\\\\\/\\\\\\\/IQlqt0QTZCS2mpjVE97mQxjFZ7Dl+NpulVydPE58lYB6z\\\\\\\/TVV98A+6RTn9\\\\\\\/nGQuff50ueiJuczaUGcpDtJcbM5LXLFNXZ\\\\\\\/mVRL4qwaOzMpnD3tDJpazINwwsPy5kj+1VNSYMJ2C7TvABPuYNKqJ\\\\\\\/WXEdvNUIBYH9x7QD6+Sa0HC7EN8VmDPdrFj6ZcWF0s4\\\\\\\/KuyPenQ4F\\\\\\\/62CS4PAtaS\\\\\\\/wfOF5sXq6kNk\\\\\\\/q1JEACnW0q47xyll1yYP2Q179DuO+OJkAU0y0lpaYGelI4hT9jmyy2oTUx2rG59icXwXK2syzj55RE95\\\\\\\/y\\\\\\\/aQmPjqA2xVROr2FN8b1XVBuUddXLvT2FKFvkcbjUG03rkYTBVMJfT2G1aVlYDn4QWKdrwo6XoSPwW4JdWhhWGnU1lKveJKbO9TFKsfDR+ZpvyfX9JkvMlVa3dd3QGArgBKqTOq4W8pjN8mpaAYu6PW+aFg\\\\\\\/l4jJYv+wjRnTcZOM7E+pl6cloeHC2IZm1dP\\\\\\\/lVpiuGmo45PPR\\\\\\\/yT0P2C6XSTj+70stidQq4FltHCBPeTcQWKF1ru9WMwd+\\\\\\\/ayJ1DDWdMixRZG+ZvCGMLm4xij6xVs0tcOgeoEA4c6w+RcPltjnctrVfwSw194tn0zV79dXe4DpXOfXh8Fz5iG8KRc8pROWdwZ78ZK\\\\\\\/zO+9VmoBjNd5DKxROELGsUfFUcPJalEHfwsxB62N3zZm+3CSlkMBN127zy413ZIV1bKE2dE0t+K1RKflLhtcF9j0JLLazsrSD28UKeKSYsoqbQeHFt9\\\\\\\/OxqWa6xMqDWFUD2YTjLjviKfJD\\\\\\\/MUmMjNn5Te4HQZo39q5J8WSPRu1BbF8QSYDq2odWGqBJRXmWHuy+wgWXrV5GEBilut4xmfK2PaswEpyFF08jQ2tLEz3q8EeownxqHw5+erh69gWjXiaGZfqsfbhyHkDmDQgSpm+g7G2D9d7EormRTWb8EQguizzeQiUZrqgpq9BrNzb9ZADV8fVy\\\\\\\/tQqZAuQaFDSx\\\\\\\/GNCtwo2vDIM\\\\\\\/y\\\\\\\/\\\\\\\/kboJj3byT\\\\\\\/hTYW8LdrgTVXyNW7JEYFyqRZAZOvoKbdr9W1RhYQ5PuOqmu05Gn9SejrnJZ1CdtU+\\\\\\\/h92N5eVIp0WiPP5M7I7jcjVtAJ8Ez5xoWsHOE9y\\\\\\\/1iILCUpR8QVNls2QW\\\\\\\/ORDyRDSjjA2jivPuaUgkyS9SYvXdP\\\\\\\/yXEMg9YJplUAAssSWEBRaMClvAYNwGitfVvjjE32KZspLfAfbz4mXmfwAEH0pBoxLozrO8xWPlPg5Lj3qSx3NlTsYhKYqC9nNyFAFeYoxSdRAUy0eJGw6V26bcBS6zPKyDmiHwZe5VVjtOtJIR7XrLTmz2Eg+Nfnc\\\\\\\/V1N\\\\\\\/sDUtg4SE+JAXIK8iaizEp2cf1EciB\\\\\\\/b9mK1S2DWZGP\\\\\\\/+ryrXHdqEeLTxPKrNrBV4pyihX0l1QCONULFyi2jabkjRahgSe6HwdD9B4tBllD1+24Tj4xG2yrI3M+zE0\\\\\\\/yd1JPTk4oZ7jYZ7atrW2fM5pjP7R8yVxuSCqB7l4lmzudLlPWtjltNE0l5FvSEQqgA6BDysswNVRFbw9uWIxW1k+6UsKb6vw8ZNgbQ4cr5lq9NSARV23g8RucokbVlJlDnnxoWqLAc4p09JH9v1ei0wGhTf05mxYmf+wuTU\\\\\\\/vmGWvtCnxW2iKFrct\\\\\\\/LUXirAhBJVdKz7ASZHGmNQj9N\\\\\\\/F\\\\\\\/Zox0t7IuAtInqPX\\\\\\\/\\\\\\\/HGMx5BOwC4B1CgWNZOXXz8pmEvWI90I4jk1kNaDXjVFGjoH+FsN1J4GVHE\\\\\\\/nXDmC6laK+0ux2QdEOwmcnBto76\\\\\\\/7Db0a+1nXb1R\\\\\\\/f4QQZNJ39AZxKoGjtj173OtF0UNfeA9tSNwxj9C7LdU4qEKQ+EdjQwimGFg+mz\\\\\\\/Otq8C4nvxcfz7tOyrrbT1xonbRi2x9dqXYaQBrH04G4sy8dqsrzziMqLi4vyg\\\\\\\/vwi9HycztyDUTGCt9bJZUkitbfiik01JTQR+Fw3CLLAYeHKzZtZdr3Cp2701WO66ZwCyZvtHWvOeMj4sATiarGyHJ3\\\\\\\/FNkfDl9lNEQlNZlifRsTqWhr+B+BJHDQC7ba+vYHvXrwEO9GMc9ddzJFvmtk\\\\\\\/F9Hhc3Z1T83cMguPCRdF87AE5CY+HQET45OUksX42XFrpv53apRRsMLjmhdnIBe64GlznXDF7mo3Mbo7rzPN8CTJb9EKoeec9D\\\\\\\/dnz1nVtEt+8ntmREetFGdDdaUkPG0ZXCZE2TBTOAB5RIOf\\\\\\\/N5RwGxtI7QuNfz5BX0i4OSsas0xT1zPwq4V0r3eZct9MUtvOsl5ZG4UKsDVgpKvZWRF9lUaPTtKKbRRczMo2eMvWEqgc+33rGmiNN6cGY3HhNPVsEikh04qrDWqvfgHsxfQDgBQTouwI+pFg6Uq5GDmKSZeA0QYf\\\u0022,\\\u0022iv\\\u0022:\\\u0022CurX5gDe6M\\\\\\\/o9Wp\\\\\\\/Eg\\\\\\\/mgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c6e90f244bb8886e91379bbd8ce570a477ef57a71ddafda554e555691e71ede2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rrGID6xrlGXWDgS9ac+YlVSKRL8E6Ck2ESTPvZdYLrmaP4uh4AutKQSFXxYzHr6dWZF2KLwn6PbuIE56Rn+IYlXhmhg5K2oQd\\\\\\\/y0f7A0eLJpXVUemc6IwpS9HlJKrYnK3i7dZWZcTo3TY88tByqLGRKCEW8jEBmXFUtxiH7BNUzOQIILQASjLTHVJIqafosH9VqJhDWnTuvCgOzzs834NYs7Y7wjiiJfUj6JXiuvWM2Kq1GBUPFRTcwQlMcih8uShGQ1OVkFeFWyJE0geeB04DUDzIHIJuF44Tmb4TV3e6cL3FkPpMG9\\\\\\\/KkPqOSski8SDgXT0pfxR7Lt6evajI5M4ThFRlDiL4BujKFCxHYYxbBZ93pPS808aEfiHMI5xDgRo3vyoS7RxShzGglk9SXU84nPuIGZ0HbpB7nLuRVf7SiXI9SWHy6e6odmBLUq9lX4C6BTUh9u7cPz9qIekfdm998imgT8Dk2d\\\\\\\/EDavKy9Y9eEm5SS4VE8rJMRsGakxa3H+XdXLkJmdiFF8TGpW6OYnp16aZ4T03y4+8ynJOFam9TLhNfsxIuRilE4IqH9dYgiUW39MAU10FEnSShbQbFJyFjvtKBhm\\\\\\\/OKjxd0pmzy6OydVNAMik9m11Ks2EBlBs9erbjEsr8fL3yAkRUeq+YecrhfQspRFIcQD3GnV7Fe\\\\\\\/lTdWcXAFgx4tPRWU0ldm8qtzkISgJJx+lW8PkU4WVolBxjVQGwp0JpUaazM3Jgxnz35SCveBl8MVX0KYCDcXvINv0GwTj7aMpNQ9prKCqCImLVOHMenI6lLjQnEXgtdPDBue89m5POot6b\\\\\\\/4k91u4aJeRy+PBzip9okVOckR1\\\\\\\/LLKjlExdO4JvUK7kcLEkAEF3ggHTjRdbDMCphbFk2qKKybZu6GlWg8AEHTYX+Xf4YB0Ix1+fMunu3rLYapuqN7RV63pfFhq0lV9ZCiz48fCJvqILUFjlA+3HHwLdfrhujc8kocM0MK0qTkjdVRBZnoPta2ba2H7tKhz0fJKN+qgqoJdc+g8aPVxrt\\\\\\\/dErrDIj1wCDa9je1NVgtKqemEnQzbNiceAzVok\\\\\\\/5zgW\\\\\\\/0aLwevs5EVFF43Oz\\\\\\\/OtN7GRHjTuNfVvDE9HZvq2pSlh3ipS2mja+vjYFY5BOWoilucWAB505gL\\\\\\\/AW27COfKsGrqSaGl+N\\\\\\\/ek2kAsGpIKFUSo+IQCULEPyPvOGfneV1cBHHijJ0RYfDm+7m8Grt8u\\\\\\\/5nglmV8uvTXG8nDctRJScRdhXyEqB2eBllVsxomh+jvMBQnYNJcEJCtCUzcPgtV3bZjaJ76e99lwFSDvNsakxOHBJGhsjbJM3SutGg3L5U5tpUWPO4tR3nW5Cu9DF10CL55JsyjoacOk91W29zfoV5Fgu600bUjVlNs6wXlSi+tR78eymCskUSbOmf8FZQOUHmt66Xuz\\\\\\\/YDdG1rsDhogcDI3IyVUwY6++liTAaqGl09X3qI4957LYL+aPffjBabcWaijzj\\\\\\\/zUg2P3K\\\\\\\/Wu2bkJAc825BNiBfIHk4r7C09Uh7P8YnDx5bPatygyombH2DBNTIdwqGPYy2B++S25fwKobnLpylExO9j30uUIOi6LBM6ikD8p\\\\\\\/ojSuVyxISSj2u3gzwEA81AAQYGTAHW54LzjFGQUy\\\\\\\/kte9BlgeprjgsVrBbSI4qjkWHyCMyKIfPPhUegKIb8ke5HmJDaaUocBeFgt\\\\\\\/0Puwg8cHugbv2TysYXKV2VspICssi0M63pP\\\\\\\/HAvyymKKZ4ZtNycpXZUm2LAeY3iFQBhsCJuCxwm8skvNBB5sF4z11s9LZnbxJ5qxhF8ghy56FUwwH9tetH7K2PDkjuWKl+O+K2UtPh\\\\\\\/eluop83HkMbEwRr4+L1mnIQwQ\\\\\\\/+rHUjAHNnrNLIM2kPOV4TUVbyAzq1sr7zz6tTYY0ylVS7eCVpFSIGIUX+CIXnfc76IdZ6btDL1dt7VqxSrsSbAsHM6wC\\\\\\\/auKHpbd+cAIC5aY+emd6b8gRlUoQUrAEyOjWCsQuBnaA2qLHzYbkLy1OrzEvjynumpEM=\\\u0022,\\\u0022iv\\\u0022:\\\u0022kX5G3SlyJd+M\\\\\\\/iOOm94pdw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b68c3134c634b314d051ceea79a57bd5b9da2a31d06a1fcbe6d2ce5b02f831d1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492213 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/439" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TD1fL0wJWyfZvHJm5wyVf1frKLU-HfoVpppw2oHQzxk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/439" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ABSxTF6hWqme0DKAGJEkziUQHTEMq4LYG4m1qNOYfAk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/2/425" title="Complex Structural PPT1 Variant Associated with Non-syndromic Canine Retinal Degeneration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/2/453" title="GAL4 Drivers Specific for Type Ib and Type Is Motor Neurons in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492213" id="node492213" data-pisa="ggg;9/2/439" data-pisa-master="ggg;g3.118.200922" data-apath="/ggg/9/2/439.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Statistical Procedure for Genome-Wide Detection of QTL Hotspots Using Public Databases with Application to Rice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Man-Hsia Yang</span>, <span class="highwire-citation-author" data-delta="1">Dong-Hong Wu</span> and <span class="highwire-citation-author" data-delta="2">Chen-Hung Kao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">439-452;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200922 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492213" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Man-Hsia Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, National Taiwan University, Taipei 10617, Taiwan, Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Crop Science Division, Taiwan Agricultural Research Institute, Council of Agriculture, Taichung 41362, Taiwan, Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Man-Hsia%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMan-Hsia%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dong-Hong Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Crop Science Division, Taiwan Agricultural Research Institute, Council of Agriculture, Taichung 41362, Taiwan, Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dong-Hong%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADong-Hong%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chen-Hung Kao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, National Taiwan University, Taipei 10617, Taiwan, Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Institute of Statistical Science, Academia Sinica, Taipei 11529, Taiwan, Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chen-Hung%2BKao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kao%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChen-Hung%2BKao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chkao@stat.sinica.edu.tw" class="" data-icon-position="" data-hide-link-title="0">chkao@stat.sinica.edu.tw</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/2/439" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492213" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492213/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/439.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492213" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492213/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/439.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492213" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492213/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/439.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492213" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492213/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/2/439.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492213" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492213/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genome-wide detection of quantitative trait loci (QTL) hotspots underlying variation in many molecular and phenotypic traits has been a key step in various biological studies since the QTL hotspots are highly informative and can be linked to the genes for the quantitative traits. Several statistical methods have been proposed to detect QTL hotspots. These hotspot detection methods rely heavily on permutation tests performed on summarized QTL data or individual-level data (with genotypes and phenotypes) from the genetical genomics experiments. In this article, we propose a statistical procedure for QTL hotspot detection by using the summarized QTL (interval) data collected in public web-accessible databases. First, a simple statistical method based on the uniform distribution is derived to convert the QTL interval data into the expected QTL frequency (EQF) matrix. And then, to account for the correlation structure among traits, the QTL for correlated traits are grouped together into the same categories to form a reduced EQF matrix. Furthermore, a permutation algorithm on the EQF elements or on the QTL intervals is developed to compute a sliding scale of EQF thresholds, ranging from strict to liberal, for assessing the significance of QTL hotspots. With grouping, much stricter thresholds can be obtained to avoid the detection of spurious hotspots. Real example analysis and simulation study are carried out to illustrate our procedure, evaluate the performances and compare with other methods. It shows that our procedure can control the genome-wide error rates at the target levels, provide appropriate thresholds for correlated data and is comparable to the methods using individual-level data in hotspot detection. Depending on the thresholds used, more than 100 hotspots are detected in GRAMENE rice database. We also perform a genome-wide comparative analysis of the detected hotspots and the known genes collected in the Rice Q-TARO database. The comparative analysis reveals that the hotspots and genes are conformable in the sense that they co-localize closely and are functionally related to relevant traits. Our statistical procedure can provide a framework for exploring the networks among QTL hotspots, genes and quantitative traits in biological studies. The R codes that produce both numerical and graphical outputs of QTL hotspot detection in the genome are available on the worldwide web <a href="http://www.stat.sinica.edu.tw/chkao/">http://www.stat.sinica.edu.tw/chkao/</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/qtl-hotspots" class="hw-term hw-article-keyword hw-article-keyword-qtl-hotspots" rel="nofollow">QTL hotspots</a></li><li class="kwd"><a href="/keyword/uniform-distribution" class="hw-term hw-article-keyword hw-article-keyword-uniform-distribution" rel="nofollow">uniform distribution</a></li><li class="kwd"><a href="/keyword/permutation-tests" class="hw-term hw-article-keyword hw-article-keyword-permutation-tests" rel="nofollow">permutation tests</a></li><li class="kwd"><a href="/keyword/trait-correlations" class="hw-term hw-article-keyword hw-article-keyword-trait-correlations" rel="nofollow">trait correlations</a></li><li class="kwd"><a href="/keyword/known-genes" class="hw-term hw-article-keyword hw-article-keyword-known-genes" rel="nofollow">known genes</a></li></ul><p id="p-3">Quantitative trait loci (QTL) detection has been a key step to provide deeper insight into the genetic mechanism of quantitative traits in many areas of biological researches, including crops, evolution, ecology and genetical genomics studies etc. (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lander and Botstein 1989</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Haley and Knott 1992</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jansen 1993</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zeng 1994</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kao <em>et al.</em> 1999</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Sen and Churchill 2001</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman 2003</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kao 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lee <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wei and Xu 2016</a>). In QTL detection, it is often found that some of the genomic regions are relatively enriched in QTL as compared to other regions, and that QTL responsible for correlated traits frequently co-localize in some specific genetic regions (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Goffinet and Gerber 2000</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Schadt <em>et al.</em> 2003</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chardon <em>et al.</em> 2004</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">West <em>et al.</em> 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Breitling <em>et al.</em> 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wu <em>et al.</em> 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2010</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ali <em>et al.</em> 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Basnet <em>et al.</em> 2015</a>). The regions enriched in QTL are usually called QTL hotspots, and, statistically, they harbor a significantly higher number of QTL than expected by random chance. There are several possible reasons for the phenomenon of QTL hotspots: First, QTL explaining most trait variations can be effectively and consistently detected and mapped to similar regions across different experiments in various studies. Second, QTL with higher allelic diversity have a greater chance of being detected in different crosses and environments (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zhao <em>et al.</em> 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Vuong <em>et al.</em> 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mengistu <em>et al.</em> 2016</a>). Third, pleiotropic or closely linked QTL for correlated traits (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Falconer and Mackay 1996</a>) will be frequently detected at the same regions in different experiments. For example, a gene called <em>SCM2/ APO1</em> in rice exhibits effects on panicle structure, culm strength and lodging resistance (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ookawa <em>et al.</em> 2010</a>), and a gene called <em>GST</em> in maize shows the resistance to northern leaf blight, southern leaf blight and gray leaf spot diseases (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Wisser <em>et al.</em> 2011</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Ali <em>et al.</em> 2013</a>). A genetical genomics study of <em>Arabidopsis</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fu <em>et al.</em> (2009)</a> found that the detected hotspots can be linked to several well-studied genes with pleiotropic effects on plant metabolism, physiology and morphology and development. Therefore, hotspot detection can lead to identifying genes that affect the relevant traits (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chardon <em>et al.</em> 2004</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fu <em>et al.</em> 2009</a>). As the QTL hotspots are highly informative and may harbor genes for the target traits, the detection of QTL hotspots at the genome-wide level has been an important task in broad areas of biological studies (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Breitling <em>et al.</em> 2008</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Fu <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Frary <em>et al.</em> 2014</a>).</p><p id="p-4">Genome-wide detection of QTL hotspots requires the collection of many QTL for numerous and widespread traits in the genome to enable the detection analysis. Genetical genomics experiments and public QTL databases are two feasible ways to provide such data with numerous QTL for genome-wide QTL hotspot detection. A single genetical genomics experiment can produce abundant individual-level data containing the original genotypes (genetic markers) and thousands of molecular traits (phenotypic traits), such as gene expressions or protein contents, in a single segregation population. By treating the molecular traits as quantitative traits, the QTL mapping procedure can be performed to detect QTL by providing the LOD scores at every genomic position for each trait. The LOD scores at every position for all traits can be recorded into a LOD score matrix, and given a predetermined LOD threshold, the LOD score matrix can then be converted into a QTL matrix by assigning 1 to the detected QTL positions and 0 otherwise. Using a genetical genomics experiment, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">West <em>et al.</em> (2007)</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Wu <em>et al.</em> (2008)</a> and <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Li <em>et al.</em> (2010)</a> permuted the QTL matrix across the genomic positions separately by traits to generate null distribution of hotspot sizes and compute the thresholds for assessing the significance of QTL hotspots. As these methods do not account for the correlation structure among traits, the hotspot size thresholds are severely underestimated, leading to the detection of too many spurious hotspots (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Breitling <em>et al.</em> 2008</a>). To consider the correlation structure among traits, <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Breitling <em>et al.</em> (2008)</a> permuted the individual-level data by swapping the phenotypes between individuals and keeping the genotypes intact to generate the permuted data sets, and then performed QTL mapping on all the permuted data sets to obtain the QTL matrices for determining the thresholds. The method by <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Breitling <em>et al.</em> (2008)</a> can overcome the underestimation of thresholds in the hotspot detection, but may neglect small and moderate hotspots with strong LOD scores as the magnitude of LOD score is not considered (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> 2012</a>). To consider the magnitude of LOD score, <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> (2012)</a> adopted the same permutation and QTL mapping schemes as in <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Breitling <em>et al.</em> (2008)</a> to obtain LOD score matrices. The LOD score matrices are then used to determine a sliding scale of empirical LOD thresholds given a range of possible spurious hotspot sizes in assessing the significance of QTL hotspots. In this way, the approach of <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> (2012)</a> can effectively discover small and moderate hotspots with strong LOD scores.</p><p id="p-5">Besides using genetical genomics experiments, using public databases is also an effective and convenient way to obtain genome-wide detection of QTL hotspots. Several public databases, such as GRAMENE (<a href="http://www.GRAMENE.org/">http://www.GRAMENE.org/</a>), Q-TARO (<a href="http://qtaro.abr.affrc.go.jp/">http://qtaro.abr.affrc.go.jp/</a>), Rice TOGO browser (<a href="http://agri-trait.dna.affrc.go.jp/index.html">http://agri-trait.dna.affrc.go.jp/index.html</a>), PeanutBase (<a href="http://peanutbase.org">http://peanutbase.org</a>) and MaizeGDB, contain diversity in traits and wide-ranging distribution in the genome from numerous independent QTL mapping experiments, thus they can serve as an alternative source of genome-wide QTL hotspot detection. In these public databases, only the flanking markers of the detected QTL (the QTL intervals), trait names and reference sources are curated, and no individual-level data are available to allow the hotspot analyses of <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Breitling <em>et al.</em> (2008)</a> and <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> (2012)</a>. <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Chardon <em>et al.</em> (2004)</a> self-collected QTL data from the literature and developed a statistical method based on the normal distribution for hotspot detection. As the method of <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Chardon <em>et al.</em> (2004)</a> requires the estimates of QTL positions and their variances in computation, it is not applicable to the above mentioned public databases, either. In this article, by using public databases, we combine and modify the key ideas of the above-mentioned methods to propose a statistical procedure for QTL hotspot detection at the genome-wide level. We first develop a statistical method based on the uniform distribution to convert the QTL intervals into the expected QTL frequency (EQF) matrix. Then, to cope with the correlation structure among the traits, we group the correlated traits into the same trait categories to form a reduced EQF matrix. Furthermore, inspired by the works of <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> (2012)</a> and <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cabrera <em>et al.</em> (2012)</a>, we devise a permutation algorithm on the EQF elements (bins) as well as on the QTL intervals to compute a sliding scale of hotspot size thresholds given a range of possible spurious hotspot numbers for assessing the QTL hotspots. In this way, our statistical procedure can effectively correct the underestimation of threshold and result in less spurious hotspots. Simulation study shows that our statistical procedure can control the genome-wide error rates at the target levels, can provide appropriate thresholds for correlated data, and is comparable to the methods using individual-level data in hotspot detection. In real example analysis, the 8216 QTL responsible for 236 different traits from 230 independent worldwide studies in GRAMENE rice database are analyzed for hotspot detection. The detected hotspots are compared with the 122 known genes collected in Q-TARO rice database to explore the interplays among QTL hotspots, genes and quantitative traits. The R codes that produce both numerical and graphical summaries of the QTL hotspot detection in the genomes are available on the worldwide web <a href="http://www.stat.sinica.edu.tw/chkao/">http://www.stat.sinica.edu.tw/chkao/</a>. Our analyses can establish a framework for exploring the networks among QTL, genes and traits in biological studies.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-6">Our proposed procedure operates on the QTL intervals and can take the correlation structure among traits into account for QTL hotspot detection. In the following, the QTL intervals in the GRAMENE database that can be integrated into a QTL matrix is first described. Then we propose a simple method based on the uniform distribution to convert the QTL matrix into the EQF matrix and further into the reduced EQF matrix by grouping of correlated traits into the same trait categories. After that, we develop a permutation algorithm that can work on the EQF elements or on the QTL intervals to compute a range of EQF thresholds that vary from strict to liberal for assessing the significance of QTL hotspots. For better understanding, we use a simple example in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> to illustrate the scheme of our statistical procedure from using the QTL intervals (matrix) to obtaining the permutation EQF thresholds. Real example and simulation study are followed to demonstrate the capability of our procedure, investigate the detection properties, and also compare with the methods using individual-level data in QTL hotspot detection.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/439/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The schemes of the proposed statistical procedure and the Q-method in obtaining the permutation thresholds. The QTL data from the public database are first recorded into a QTL matrix, where the QTL intervals take a value of one and the remaining elements will be treated as zeros at the corresponding positions. The output of the analysis is a QTL matrix, where columns represent the genomic positions (Si&#x2019;s) and rows represent the traits (tij denotes the jth trait of the ith trait category). Then, the QTL matrix is converted into the expected QTL frequency (EQF) matrix by using the uniform distribution method. The Q-method permutes the EQF matrix for each trait (the column cells) to obtain the permutation EQF threshold, &#x3B2; (see text). The proposed procedure groups together the related traits into the same trait categories and pools their EQF values to form a reduced EQF matrix, and the reduced EQF matrix for each trait category (the column cells) are permuted to obtain a series of EQF thresholds, &#x3B3;n,&#x3B1;&#x2019;s (see text)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2081177879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The schemes of the proposed statistical procedure and the Q-method in obtaining the permutation thresholds. The QTL data from the public database are first recorded into a QTL matrix, where the QTL intervals take a value of one and the remaining elements will be treated as zeros at the corresponding positions. The output of the analysis is a QTL matrix, where columns represent the genomic positions (S&lt;sub&gt;&lt;em&gt;i&lt;/em&gt;&lt;/sub&gt;&#x2019;s) and rows represent the traits (&lt;em&gt;t&lt;/em&gt;&lt;sub&gt;&lt;em&gt;ij&lt;/em&gt;&lt;/sub&gt; denotes the &lt;em&gt;j&lt;/em&gt;th trait of the &lt;em&gt;i&lt;/em&gt;th trait category). Then, the QTL matrix is converted into the expected QTL frequency (EQF) matrix by using the uniform distribution method. The Q-method permutes the EQF matrix for each trait (the column cells) to obtain the permutation EQF threshold, &lt;em&gt;&#x3B2;&lt;/em&gt; (see text). The proposed procedure groups together the related traits into the same trait categories and pools their EQF values to form a reduced EQF matrix, and the reduced EQF matrix for each trait category (the column cells) are permuted to obtain a series of EQF thresholds, &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;n,&#x3B1;&lt;/em&gt;&lt;/sub&gt;&#x2019;s (see text).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/439/F1.medium.gif" width="351" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/2/439/F1.medium.gif" width="351" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/439/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/439/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492263" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">The schemes of the proposed statistical procedure and the Q-method in obtaining the permutation thresholds. The QTL data from the public database are first recorded into a QTL matrix, where the QTL intervals take a value of one and the remaining elements will be treated as zeros at the corresponding positions. The output of the analysis is a QTL matrix, where columns represent the genomic positions (S<sub><em>i</em></sub>âs) and rows represent the traits (<em>t</em><sub><em>ij</em></sub> denotes the <em>j</em>th trait of the <em>i</em>th trait category). Then, the QTL matrix is converted into the expected QTL frequency (EQF) matrix by using the uniform distribution method. The Q-method permutes the EQF matrix for each trait (the column cells) to obtain the permutation EQF threshold, <em>Î²</em> (see text). The proposed procedure groups together the related traits into the same trait categories and pools their EQF values to form a reduced EQF matrix, and the reduced EQF matrix for each trait category (the column cells) are permuted to obtain a series of EQF thresholds, <em>Î³</em><sub><em>n,Î±</em></sub>âs (see text).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>The QTL matrix</h3><p id="p-8">The QTL intervals contain complete information about the QTL that need for hotspot detection. We use a row array of the same length as the genome size to summarize the QTL intervals for each trait. For each trait, we regularize the QTL intervals into the elements of a row array as follows: Each QTL interval corresponds to an element of the length as its width at the corresponding position, and a value of one is assigned to the element. The remaining elements will be treated as zeros. In this way, the elements in the row array are either one or zero with unequal lengths (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> for graphical illustration). Combining the arrays for all traits will form a QTL matrix with different element sizes. The QTL matrix (an atypical matrix) will be used to construct the EQF matrix for permutation as described below.</p></div><div id="sec-3" class="subsection"><h3>The expected QTL frequency</h3><p id="p-9">Consider that a QTL matrix has been constructed from the database. We assume that there are T traits mapped for <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-1.gif"/></noscript></span></span> QTL, respectively, in the experiments (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-2.gif"/></noscript></span></span>), and that the genome is divided into <em>S</em> sequential equally spaced bins, each with size â³ (say â³=0.5 cM), for investigation. We use the uniform distribution to compute the EQF value of each bin over the total experiments for hotspot detection. For a bin <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-3.gif"/></noscript></span></span> and a QTL interval <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-4.gif"/></noscript></span></span>, where x, a and b denote the genome positions, there are two possible relationships between them: (1) they have an overlap, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-5.gif"/></noscript></span></span>, or (2) they have no overlap, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-6.gif"/></noscript></span></span>. Only the QTL intervals having overlaps with a bin contribute to the EQF value of this bin, and such a QTL is called a contributive QTL of a bin. By assuming that the QTL position is uniformly distributed over its own QTL interval, the probability of a contributive QTL localized in a bin is the ratio between the lengths of the overlap and the interval. Summing over the probabilities of all the contributive QTL in a bin will give the EQF value of this bin over all traits. Explicitly, we define <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-7.gif"/></noscript></span></span> as the EQF value of the <em>s</em>th (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-8.gif"/></noscript></span> </span>bin between x and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-9.gif"/></noscript></span></span> for all traits over all experiments by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where C is the number of the contributive QTL of the bin <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-11.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-12.gif"/></noscript></span></span> is the overlap region, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-13.gif"/></noscript></span></span> denotes the <em>i</em>th QTL interval, and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-14.gif"/></noscript></span></span> is the uniform distribution density function over the interval. A ratio <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-15.gif"/></noscript></span></span> will be added to the EQF of the bin despite of the effect size of a contributive QTL. If a QTL interval fully covers the bin, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-16.gif"/></noscript></span></span> and the ratio (probability) is <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-17.gif"/></noscript></span></span>. If a QTL interval is fully covered by the bin, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-18.gif"/></noscript></span></span> and the probability is 1.</p><p id="p-10">The computation of the EQF values in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">equation (1)</a> can be illustrated by <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and the example in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. In <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, we use a set of selected 196 QTL (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-19.gif"/></noscript></span></span>) in the 1<sup>st</sup> chromosome from GRAMENE databases for illustration. These 196 QTL are responsible for 95 different traits belonging to four of the nine trait categories. Depending on the bin position, the number C involved in computing <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-20.gif"/></noscript></span></span> is different for different bins. For example, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-21.gif"/></noscript></span></span>for a bin between 80â¼90 cM as the region is only covered by the four same QTL intervals, and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-22.gif"/></noscript></span></span> for a bin between the 140â¼160 cM as the within bins may overlap with different QTL intervals. By <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">equation (1)</a>, the EQF value of a bin in the 80â¼90-cM region is 0.037, and the EQF value of a bin in the 140â¼160-cM region is 0.91â¼7.38. The EQF value can be calculated at every bin to produce an EQF hotspot architecture along the chromosomes as shown in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">3</a>. A higher EQF value reflects a greater expectation of localizing a QTL in a bin. A hotspot detection is claimed in a bin if its EQF value is higher than a specified threshold that will be determined by permutation tests as given below.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/439/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An illustration of the QTL data structure and the uniform method of computing the expected QTL frequency (EQF) in hotspot detection. The 196 QTL in the rice 1st chromosome from Gramene Rice database (http://www.gramene.org/) are used for illustration. The green ticks on the x-axis denote the positions of the 163 markers. The dotted lines denote the lengths of the marker intervals flanking the QTL responsible for yield, vigor, sterility and quality traits (denoted as &#xD7;, +, &#x2206; and &#x25CB;, respectively). The EQF architecture (the black line) are constructed by the uniform method with bin size of 0.5 cM. The black lines on the right and left borders represent the EQF. &#x25B2;denotes the position of centromere." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2081177879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An illustration of the QTL data structure and the uniform method of computing the expected QTL frequency (EQF) in hotspot detection. The 196 QTL in the rice 1&lt;sup&gt;st&lt;/sup&gt; chromosome from Gramene Rice database (http://www.gramene.org/) are used for illustration. The green ticks on the x-axis denote the positions of the 163 markers. The dotted lines denote the lengths of the marker intervals flanking the QTL responsible for yield, vigor, sterility and quality traits (denoted as &#xD7;, +, &#x2206; and &#x25CB;, respectively). The EQF architecture (the black line) are constructed by the uniform method with bin size of 0.5 cM. The black lines on the right and left borders represent the EQF. &#x25B2;denotes the position of centromere.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/439/F2.medium.gif" width="440" height="239"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/2/439/F2.medium.gif" width="440" height="239"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/439/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/439/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492187" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-11" class="first-child">An illustration of the QTL data structure and the uniform method of computing the expected QTL frequency (EQF) in hotspot detection. The 196 QTL in the rice 1<sup>st</sup> chromosome from Gramene Rice database (<a href="http://www.gramene.org/">http://www.gramene.org/</a>) are used for illustration. The green ticks on the x-axis denote the positions of the 163 markers. The dotted lines denote the lengths of the marker intervals flanking the QTL responsible for yield, vigor, sterility and quality traits (denoted as Ã, +, â and â, respectively). The EQF architecture (the black line) are constructed by the uniform method with bin size of 0.5 cM. The black lines on the right and left borders represent the EQF. â²denotes the position of centromere.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/439/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The EQF architectures along the 12 chromosomes and the hotspots detected under different EQF thresholds (&#x3B3;n,0.05) associated with their qFreq(n) statistics at GWER of 5%. The thresholds &#x3B3;n,0.05 are coordinately represented by the left and right axes. The left axis denotes the values of EQF, and the right axis denotes the values of n. The blue line corresponds to the EQF threshold &#x3B3;1,0.05 = 47.19 for the qFreq(1) statistic of detecting at least one hotspot, and there four significant hotspots with &#x3B3;1,0.05. The red line shows &#x3B3;100,0.05 = 9.77 for the qFreq(100) statistic of detecting at least 100 hotspots, which approximately corresponds to &#x3B2; (the EQF threshold of the Q-method), and there are 179 significant hotspots with &#x3B3;100,0.05&#x2033;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2081177879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The EQF architectures along the 12 chromosomes and the hotspots detected under different EQF thresholds (&lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;n&lt;/em&gt;,0.05&lt;/sub&gt;) associated with their qFreq(&lt;em&gt;n&lt;/em&gt;) statistics at GWER of 5%. The thresholds &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;n&lt;/em&gt;,0.05&lt;/sub&gt; are coordinately represented by the left and right axes. The left axis denotes the values of EQF, and the right axis denotes the values of &lt;em&gt;n&lt;/em&gt;. The blue line corresponds to the EQF threshold &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;1,0.05&lt;/sub&gt; = 47.19 for the qFreq(1) statistic of detecting at least one hotspot, and there four significant hotspots with &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;1,0.05&lt;/sub&gt;. The red line shows &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;100,0.05&lt;/sub&gt; = 9.77 for the qFreq(100) statistic of detecting at least 100 hotspots, which approximately corresponds to &lt;em&gt;&#x3B2;&lt;/em&gt; (the EQF threshold of the Q-method), and there are 179 significant hotspots with &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;100,0.05&lt;/sub&gt;&#x2033;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/439/F3.medium.gif" width="440" height="187"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/2/439/F3.medium.gif" width="440" height="187"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/439/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/439/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492077" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-12" class="first-child">The EQF architectures along the 12 chromosomes and the hotspots detected under different EQF thresholds (<em>Î³</em><sub><em>n</em>,0.05</sub>) associated with their qFreq(<em>n</em>) statistics at GWER of 5%. The thresholds <em>Î³</em><sub><em>n</em>,0.05</sub> are coordinately represented by the left and right axes. The left axis denotes the values of EQF, and the right axis denotes the values of <em>n</em>. The blue line corresponds to the EQF threshold <em>Î³</em><sub>1,0.05</sub> = 47.19 for the qFreq(1) statistic of detecting at least one hotspot, and there four significant hotspots with <em>Î³</em><sub>1,0.05</sub>. The red line shows <em>Î³</em><sub>100,0.05</sub> = 9.77 for the qFreq(100) statistic of detecting at least 100 hotspots, which approximately corresponds to <em>Î²</em> (the EQF threshold of the Q-method), and there are 179 significant hotspots with <em>Î³</em><sub>100,0.05</sub>â³.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>The EQF matrix</h3><p id="p-13"><a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">Equations (1)</a> is to compute the EQF value of a bin for all traits over all experiments in the genome. It is also desirable to compute the EQF value of a bin for each single trait, and further to construct the EQF matrix for permutation to determine the thresholds. Let <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-23.gif"/></noscript></span></span> denote the EQF value of the <em>s</em>th bin for the <em>t</em>th trait. It is straightforward to obtain <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-24.gif"/></noscript></span></span> using <a id="xref-disp-formula-1-4" class="xref-disp-formula" href="#disp-formula-1">equation (1)</a> by simply replacing the number of contributive QTL for all traits, C, by the number of contributive QTL for the <em>t</em>th trait, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-25.gif"/></noscript></span></span>. That is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>We now define the EQF matrix as <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-27.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-28.gif"/></noscript></span></span> is the index for row dimension, and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-29.gif"/></noscript></span></span> is the index for column dimension. We have <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-30.gif"/></noscript></span></span> is the number of QTL for the <em>t</em>th trait, and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-31.gif"/></noscript></span></span> is the EQF value of the <em>s</em>th bin for all traits. By pooling the EQF values of genetically correlated traits, the F matrix will be further converted into a reduced F matrix (row dimension less than T) for permutation to determine the EQF thresholds for assessing the significance of hotspots.</p><p id="p-14">The current methods of <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">West <em>et al.</em> (2007)</a>, <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Wu <em>et al.</em> (2008)</a> and <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Li <em>et al.</em> (2010)</a> permute the QTL matrix for hotspot detection in the genetical genomics study. Their methods can be directly implemented to permute the EQF matrix for QTL hotspot detection using public databases. We follow <a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> (2012)</a> to still call these methods implemented to the EQF matrix âQ-methodâ. The Q-method will permute the row elements of the EQF matrix separately by traits and then obtain the EQF sums over all traits for every bin in the genome, and the maxima of EQF sums in all permutations are used to compute the EQF threshold for assessing the QTL hotspots (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Similarly, the Q-method does not account for the correlation structure among traits, and hence it will severely underestimate the null distribution of hotspot sizes (thresholds) and detect too many spurious hotspots (<a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> 2012</a>; also see below). To consider the correlation structure among traits, we group together the correlated traits into the same trait categories and pool together their EQF values to form a reduced EQF matrix for permutation. For example, the panicle numbers, grains per panicle and grain weight are all related to yield production in rice, and their EQF values can be summed and become the EQF value of the yield trait category to form a reduced EQF matrix (the row dimension reduced by 2 to become <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-32.gif"/></noscript></span></span>). As pleiotropy and linkage of genes are the genetic causes of correlations between traits (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Falconer and Mackay 1996</a>), the trait grouping intends to pool and permute these QTL together, so that to cope with the underestimation of threshold. Other grouping strategies can be also considered (see CONCLUSION AND DISCUSSION). Permutation with trait grouping can effectively obtain much stricter thresholds to prevent detecting spurious hotspots, as will be shown below. Furthermore, inspired by the work of <a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> (2012)</a>, rather than examine the possible spurious hotspot sizes, we consider the possible spurious hotspot numbers in the genome to present a permutation algorithm for computing the thresholds. Our permutation algorithm shuffles the reduced EQF matrix to compute a series of EQF thresholds ranging from strict to liberal for assessing the QTL hotspots.</p></div><div id="sec-5" class="subsection"><h3>The permutation algorithm</h3><p>For a fixed hotspot number <em>n</em> = 1, ..., <em>k</em>, with <em>k</em> the hotspot number delivered by the Q-method (see below), across the genome, we first define <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-33.gif"/></noscript></span></span> as the <em>n</em>th EQF sum of the <em>S</em> ordered observed EQF sums (namely: <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-34.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-35.gif"/></noscript></span></span>, ..., <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-36.gif"/></noscript></span></span>, ordered from highest to lowest) in the reduced F matrix and use it as a test statistic for at least <em>n</em> spurious hotspots under the null hypothesis that the QTL are randomly distributed in the genome. We describe the permutation algorithm that can control the GWER of detecting at least <em>n</em> hotspots at a fixed Î± level as follows:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-16">For each trait (category), the EQF values of the <em>S</em> locations (bins) in the reduced F matrix (with row dimension <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-37.gif"/></noscript></span></span>) are permuted to generate a new permuted EQF matrix <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-38.gif"/></noscript></span></span>. That is, the elements in each row of the reduced F matrix are swapped to produce the <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-39.gif"/></noscript></span></span> matrix (see <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). As every row element can be only assigned to one of the <em>S</em> bins, the row sums of the permuted and observed matrices are the same and are equivalent to <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-40.gif"/></noscript></span></span> for the <em>t</em>th trait (category), <em>i.e.</em>,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-41.gif"/></noscript></span></span></p><p id="p-17">where <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-42.gif"/></noscript></span></span> is the element of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-43.gif"/></noscript></span></span>.</p></li><li id="list-item-2"><p id="p-18">Compute the total EQF sums over all traits (categories) for the <em>S</em> locations, <em>i.e. </em><span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-44.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-45.gif"/></noscript></span></span> and order them (<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-46.gif"/></noscript></span></span> from highest to lowest as<span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-47.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-48.gif"/></noscript></span></span>, ...,<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-49.gif"/></noscript></span></span>.</p></li><li id="list-item-3"><p id="p-19">For a fixed hotspot number <em>n</em>, obtain and store <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-50.gif"/></noscript></span></span> corresponding to the <em>n</em>th EQF sum of the <em>S</em> ordered EQF sums for <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-51.gif"/></noscript></span></span>.</p></li><li id="list-item-4"><p id="p-20">Repeat steps 1â3 B times so that there are B new permuted matrices (namely, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-52.gif"/></noscript></span></span>,<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-53.gif"/></noscript></span></span>,â...,<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-54.gif"/></noscript></span></span>) for obtaining the associated <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-55.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-56.gif"/></noscript></span></span>,...,<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-57.gif"/></noscript></span></span>. The B-permutation samples of <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-58.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-59.gif"/></noscript></span></span> is an estimate of the null distribution of the test statistic <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-60.gif"/></noscript></span></span>for at least <em>n</em> spurious hotspots anywhere in the genome, given that the QTL are randomly distributed along the genome.</p></li><li id="list-item-5"><p id="p-21">The upper (1-Î±)-quantile of the <em>B</em>-permutation samples generated in step 4 is the EQF threshold, denoted by <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-61.gif"/></noscript></span></span>, of the test statistic <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-62.gif"/></noscript></span></span> for assessing QTL hotspots.</p></li></ol><p id="p-22">The EQF threshold <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-63.gif"/></noscript></span></span> can control GWER of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-64.gif"/></noscript></span></span> at levelÂ Î±Â for detecting at least <em>n</em> spurious hotspots somewhere in the genome under the null. That is<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-65.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>The quantity Î± is the probability of detecting exactly <em>n</em> spurious hotspots somewhere in the genome under the null. However, similar to the argument in <a id="xref-ref-24-10" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> (2012)</a>, the threshold <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-66.gif"/></noscript></span></span> can also control the full-null GWER for more than <em>n</em> hotspots, as detecting more than <em>n</em> hotspots is less likely than detecting <em>n</em> hotspots given the threshold <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-67.gif"/></noscript></span></span>. Therefore, by adopting <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-68.gif"/></noscript></span></span> we can control GWER of <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-69.gif"/></noscript></span></span> at level Î± of detecting at least <em>n</em> spurious hotspots under the null.</p></div><div id="sec-6" class="subsection"><h3>Circular genomic permutation</h3><p id="p-23">The basic idea of the above algorithm is to randomly shift the EQF bins along the genome to obtain the thresholds for QTL hotspot detection. When doing this, a QTL interval will be broken into several bins for permutation. To keep the QTL intervals intact in permutation, we can consider the genome to be circular (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cabrera <em>et al.</em> 2012</a>) and directly implement the proposed algorithm to randomly swap the QTL intervals of correlated traits together in the circular genome. <a id="xref-disp-formula-1-5" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> is then used to compute the EQF sums of all bins in each permutation for obtaining the EQF thresholds. We call this circular permutation framework the QTL-interval permutation in our statistical procedure. In this way, the proposed algorithm can deploy both the EQF-bin permutation and the QTL-interval permutation to compute a series of thresholds, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-70.gif"/></noscript></span></span>âs, for <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-71.gif"/></noscript></span></span>âs to assess the significance of QTL hotspots.</p><p id="p-24">In general, performing permutation on the EQF bins or the QTL intervals for the hotspot detection is not computationally demanding at all. On the contrary, performing permutation on individual-level data, such as in the approaches of the <a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Breitling <em>et al.</em> (2008)</a> and <a id="xref-ref-24-11" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> (2012)</a>, is computationally expensive and requires parallel computations on a cluster (<a id="xref-ref-24-12" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> 2012</a>), mainly because it involves repeated QTL mapping analysis for thousands of traits in each permutation. Briefly, the proposed statistical procedure for genome-wide hotspot detection includes four steps: (1) obtaining the QTL intervals (matrix) from the database; (2) constructing the EQF matrix from the QTL matrix using the uniform distribution method; (3) building the reduced EQF matrix by grouping of correlated traits into the same trait categories; (4) computing the thresholds by implementing the permutation algorithm on the reduced EQF matrix or on the QTL intervals (with trait grouping). Our statistical procedure actually generalizes and combines the ideas of the previous works on QTL hotspot detection from using genetical genomics experiments to using public databases. The value of <em>n</em> is allowed to vary from 1 to <em>k</em>. Given <em>Î²</em> as the threshold value of the Q-method, <em>k</em> is determined by <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-72.gif"/></noscript></span></span>. The <em>k</em> thresholds, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-73.gif"/></noscript></span></span>,<span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-74.gif"/></noscript></span></span>,..., <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-75.gif"/></noscript></span></span>, range from the most conservative to the most liberal. Then, our statistical procedure of using <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-76.gif"/></noscript></span></span> as the threshold is equivalent to the Q-method deployed for permuting the original EQF matrix, which will also deliver the most liberal EQF threshold and suffer from excessive spurious hotspots. With the grouping strategy and permutation algorithm, our statistical procedure can provide a range of more conservative thresholds to prevent the detection of spurious QTL hotspots as will be validated by the GRAMENE rice dataset (see the S1_DataQTL file in the supplementary material or <a href="http://www.stat.sinica.edu.tw/chkao/">http://www.stat.sinica.edu.tw/chkao/</a>) example and simulation study in the next section.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-25">The authors affirm that all data necessary for confirming the conclusions of this article are represented fully within the article and its tables and figures. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7451393">https://doi.org/10.25387/g3.7451393</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-26">In this section, real example with comparative analysis and simulation study are conducted-to illustrate the proposed statistical procedure, investigate the property of the proposed procedure, evaluate the performance of the proposed procedure, and compare with other methods in QTL hotspot detection. In real example analysis, the QTL data collected in GRAMENE rice database were analyzed to detect QTL hotspots, and then a genome-wide comparative analysis of the detected QTL hotspots and the known genes collected in Q-TARO rice database (see the S2_DataGene(NV) file in supplementary material or <a href="http://www.stat.sinica.edu.tw/chkao/">http://www.stat.sinica.edu.tw/chkao/</a>) was performed for cross validation and practical use. In simulation study, we investigate the GWERs of the proposed statistical procedure and the Q-method under different levels of correlation among traits, and assess their performances in the detection of QTL hotspots. The proposed statistical procedure operates on summarized QTL data instead of original individual-level data for hotspot detection. There must be some information loss between the two types of data during the detection process. We implement our procedure, Q-method, Breitlingâs method (2008) and Netoâs approach (2012) to analyze a simulated genetical genomics data set, and compare their differences for examining such information loss in hotspot detection.</p><div id="sec-9" class="subsection"><h3>The GRAMENE rice dataset example</h3><div id="sec-10" class="subsection"><h4>The GRAMENE database and Q-TARO database:</h4><p id="p-27">Both the GRAMENE database and Q-TARO database are web-accessible and common reference databases for rice research. The GRAMENE database collects 8216 QTL (<em>N</em> = 8216) responsible for 236 different traits (<em>T</em> = 236) from 230 published studies (experiments). These 236 traits are further classified (grouped) into the nine trait categories (<em>T</em> = 9) according to the general agronomic consideration (see Tables S1 and S2 in supplementary material). The nine trait categories including yield, vigor, anatomy, development, abiotic stress, quality, sterility or fertility, biotic stress, and biochemical traits. For example, the yield category includes traits, such as the panicle numbers, grains per panicle and grain weight, etc. The nine trait categories include 26, 15, 47, 12, 52, 44, 11, 8 and 21 component traits, respectively, and they contain 1956, 1767, 1267, 901, 767, 555, 470, 378 and 155 QTL, respectively. The total length of the rice 12 chromosomes is â¼1536.9 cM (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Harushima <em>et al.</em> 1998</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">International Rice Genome Sequencing Project 2005</a>). The 1<sup>st</sup> chromosome has the highest QTL density with â¼6.77 QTL per cM, and the 12<sup>th</sup> chromosome has the lowest QTL density with â¼3.67 QTL per cM. The QTL density is â¼5.35 QTL per cM across the 12 chromosomes.</p><p id="p-28">The flanking marker pairs of the 8216 QTL (8216 QTL intervals) are recorded and used for detecting QTL hotspots. The detection results will be investigated and compared with the 122 known genes identified through natural variation methods (such as using the experiments of rice cultivars, landraces, or wild relatives) in Q-TARO database (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Yamamoto <em>et al.</em> 2012</a>). We first projected the two physical maps of the GRAMENE rice and Q-TARO databases into a consensus genetic map by homothetic function (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Chardon <em>et al.</em> 2004</a>) with their common markers (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Harushima <em>et al.</em> 1998</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">International Rice Genome Sequencing Project 2005</a>). There are 1914 common markers (see the S1_DataQTL file) on the consensus map with an average marker density of one marker every 0.81 cM. By using <a id="xref-disp-formula-1-6" class="xref-disp-formula" href="#disp-formula-1">equations (1)</a> with bin size of 0.5 cM (â³=0.5 cM), the EQF architectures (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) and the EQF matrix can be obtained. The EQF matrix has a dimension 236Ã3070, and the reduced EQF matrix has a dimension 9Ã3070 after grouping of the 236 traits into the nine trait categories. Both the EQF-bin permutation and QTL-interval permutation are considered in the analysis. The detection results are summarized and presented below and in the supplementary material.</p></div><div id="sec-11" class="subsection"><h4>QTL hotspots:</h4><p id="p-29">The thresholds <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-77.gif"/></noscript></span></span>, n = 1, 2,â..., k, obtained by permuting the EQF bins and the QTL intervals are very similar (see Figure S1). <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> presents the EQF architectures of the 12 chromosomes and the hotspots detected under different EQF thresholds (<span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-78.gif"/></noscript></span></span>) obtained using the EQF-bin permutation. In <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, the threshold values <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-79.gif"/></noscript></span></span>âs for the test statistic qFreq(<em>n</em>)âs are coordinately represented by the left and right axes. For example, the 1<sup>st</sup> highest peak (on the 4<sup>th</sup> chromosome) has an EQF value 71.97, and therefore qFreq(1) for detecting at least one hotspot in the genome is 71.97 (qFreq(1) =71.97). The threshold for qFreq(1) is <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-80.gif"/></noscript></span></span>, which is 47.67 <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-81.gif"/></noscript></span></span>. Since qFreq(1) &gt; <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-82.gif"/></noscript></span></span>, it means that there exists at least one hotspot in the genome, and in practice there are four significant hotspots (on the 3<sup>rd</sup>, 4<sup>th</sup> and 8<sup>th</sup> chromosomes). Similarly, qFreq(3) = 56.30 (the 3<sup>rd</sup> highest peak on the 8<sup>th</sup> chromosome) is the statistic for detecting at least three hotspots, and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-83.gif"/></noscript></span></span>35.29 is the threshold for qFreq(3). As qFreq(3) &gt; <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-84.gif"/></noscript></span></span>, it means that there are at least three hotspots in the genome, and in practice there are ten significant hotspots (on the 1<sup>st</sup>, 3<sup>rd</sup>, 4<sup>th</sup>, 6<sup>th</sup>, 8<sup>th</sup>, 9<sup>th</sup> and 11<sup>th</sup> chromosomes). The highest peak of the 1<sup>st</sup> chromosome has an EQF value 35.89 and was significant under <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-85.gif"/></noscript></span></span>35.29, but not significant under <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-86.gif"/></noscript></span></span>46.70. The EQF value of the 6<sup>th</sup> highest peak (on the 3<sup>rd</sup> chromosome) is 38.52, <em>i.e.</em> qFreq(6)=38.52, and <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-87.gif"/></noscript></span></span> is 30.66. Since qFreq(6) &gt; <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-88.gif"/></noscript></span></span>, it means that there are at least six hotspots in the genome, and in practice there are 16 hotspots detected. Likewise, qFreq(9) &gt; <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-89.gif"/></noscript></span></span>, meaning that there are at least nine hotspots in the genome, and in practice there are 25 significant hotspots. The top 100 hotspots are significant under <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-90.gif"/></noscript></span></span>14.21. <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Chardon <em>et al.</em> (2004)</a> empirically suggested 5 times of the average EQF value per bin (5.35Ã·2<span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-91.gif"/></noscript></span></span> as the threshold, which roughly corresponds to <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-92.gif"/></noscript></span></span>13.29, and there are 116 significant hotspots detected under <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-93.gif"/></noscript></span></span>. The EQF threshold obtained by the Q-method is about 9.77 (corresponding to <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-94.gif"/></noscript></span></span>, where 100 is the upper bound of n, <em>i.e.</em>, k = 100), leading to the detection of 179 QTL hotspots, among which many of them are believed to be spurious since the correlations among traits are not considered by the Q-method.</p></div><div id="sec-12" class="subsection"><h4>Effects of bin sizes and QTL interval sizes:</h4><p id="p-30">The above analyses use a bin size of 0.5 cM (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Chardon <em>et al.</em> 2004</a>) and all the 8216 QTL intervals for QTL hotspot detection. It is of interest to assess the effects of bin size and removing large QTL intervals on the hotspot detection. The thresholds of <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-95.gif"/></noscript></span></span>, <em>n</em> = 1, 2,â..., 100, by using bin size of 1 cM and removing the QTL intervals larger than 10 cM (10+ cM intervals) in the analyses are shown in Figure S1. Using a bin size of 1 cM, it is found that the thresholds are higher, and the EQF architectures show similar profiles to those of using a bin size of 0.5 cM with higher EQF values at the same or very nearly the same positions (not shown). For example, using a bin size of 1 cM (0.5 cM) with all intervals, the thresholds <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-96.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-97.gif"/></noscript></span></span> are 53.77 (47.67) and 15.36 (11.89), and there are 4 (4) and 124 (131) detected hotspots. Under <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-98.gif"/></noscript></span></span>, the same four hotspots are detected for the both bin sizes. Under <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-99.gif"/></noscript></span></span>, most of the detected hotspots are the same. Other bin sizes, such as &gt;1 cM, can be also considered. We suggest to use a bin size about equal to the marker density for hotspot detection (the average marker density is 0.81 cM in our case). In general, as bin sizes increase, the hotspot resolution will decrease, and the EQF values for every bin and the permutation thresholds will increase.</p><p id="p-31">The medium, mean and SD of the interval sizes are 0.56, 9.82 and 16.82 cM, respectively, showing that the QTL intervals vary greatly in size. Among the 8216 QTL collected in GRAMENE database, 309 (3.76%) QTL are localized at markers, 3791 (46.14%) QTL are localized in the marker intervals with sizes between 0 and 0.5 cM, 274 (3.33%) QTL are localized in the 0.5-2 cM intervals, 509 (6.20%) QTL are in the 2-5 cM intervals, 744 (9.06%) QTL are in the 5-10 cM intervals, and 1023 (12.45%) QTL are in the 10-20 cM intervals. Therefore, there are 2589 (about 31.15%) QTL with interval sizes &gt;10 cM. Researchers may only consider those QTL intervals with small sizes, say â¦10 cM, in the analysis. If the 10+ cM QTL intervals are not considered in the analysis, the EQF values at very bin are smaller (not shown) and the permutation thresholds decrease slightly (see Figure S1), mainly because the 10+ cM QTL intervals contribute little to the EQF value of a relatively small size, say 0.5 cM, bin. For example, if the 10+ cM QTL intervals are excluded, <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-100.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-101.gif"/></noscript></span></span> become 46.14 and 11.05, respectively, and there are 4 and 132 hotspots detected. Under <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-102.gif"/></noscript></span></span>, the four detected hotspots are the same. Under <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-103.gif"/></noscript></span></span>, an extra hotspot is detected by removing the 10+ QTL intervals from the analysis.</p><p id="p-32">For all thresholds <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-104.gif"/></noscript></span></span>, <em>n</em> = 1, 2, ..., <em>k</em>, the observed hotspot numbers are found to exceed the expected hotspot numbers, very likely because the traits in different categories are still correlated to some extent after grouping (see CONCLUSION AND DISCUSSION). The proposed statistical procedure allows to use the <em>k</em> statistics associated with their respective thresholds ranging from high to low for broad consideration in assessing the significance of QTL hotspots. The results indicate that there are many significant hotspots in the GRAMENE rice database. It would be interesting to further explore the genes underlying these hotspots. We then perform a genome-wide comparative analysis of the detected QTL hotspots and the 122 known genes in Q-TARO databases about their locations and functions.</p></div><div id="sec-13" class="subsection"><h4>QTL hotspots, gene locations and gene functions:</h4><p id="p-33">The comparative analysis reveals that many of the detected hotspots are localized in the vicinity of the genes for their associated traits (see Figures S2). Taking the top 131 hotspots significant under <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-105.gif"/></noscript></span></span> 11.89 (obtained by the EQF-bin permutation) as an example, among the 122 genes, 17 genes overlap with the hotspots (the distances between them are zeros), 48 genes are localized within a distance of 1 cM or less from their nearest hotspots with an average distance of 0.23 cM (standard deviation 0.28 cM), 78 genes are localized within a distance of 2.5 cM or less from the their nearest hotspots with an average distance of 0.83 cM (SD 0.84 cM), and 22 genes have distances of 2.5 to 5 cM to their closest hotspots with an average distance of 3.31 cM (SD 0.68 cM). To perform a statistical test for the conformity in locations between genes and hotspots, we assume that the number of overlap, <em>y</em>, follows hypergeometric distribution (<em>N,r,n</em>)<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-106.gif"/></noscript></span></span>(<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Peng <em>et al.</em> 2010</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Cabrera <em>et al.</em> 2012</a>), where <em>N</em> is the number of bins, <em>n</em> is the number of detected hotspots and <em>r</em> is the number of known genes. In this case, the <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-107.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-108.gif"/></noscript></span></span>. Note that r is assigned to the number of 102 instead of 122 because of gene overlaps. For example, six genes <em>Pik-p, Pik-1, Pik-2, Pik-m, Xa3</em> and <em>Xa26</em> overlap at the end of the 11<sup>th</sup> chromosome (see Figure S2). The p-value is <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-109.gif"/></noscript></span></span>=<span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-110.gif"/></noscript></span></span>, which indicates that QTL hotspots have the tendency to co-localize with genes (known genes and detected hotspots are conformable in location). If the QTL-interval permutation is used, the value of <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-111.gif"/></noscript></span></span> is 12.08, and there are 129 QTL hotspots detected. Two less QTL hotspots are detected due to the use of higher threshold, and the p-value is more extreme.</p><p id="p-34">These genes are also functionally related to the QTL components in the hotspots. For example, the <em>PSR</em> gene responsible for shoot regeneration is located in the hotspot [1,73-73.5] (at 73â¼73.5 cM of the 1<sup>st</sup> chromosome) with EQF value 23.47 for all traits and EQF value 11.14 for vigor traits. The <em>dth3</em> gene for days to heading is located in the hotspot [3,6.5-7] with EQF value 29.41 for all traits and EQF value 13.92 for development traits. The <em>Hd16</em> gene acting as an inhibitor in the rice flowering pathway is located in the hotspot [3,152-152.5] with EQF value 38.52 for all traits and EQF value 28.23 for sterility traits. The <em>WFP</em> gene for regulating panicle branching and grain yield is located very close the hotspot [8,105.5-106] with EQF value 56.30 for all traits and EQF value 29.18 for vigor traits. The <em>GPS</em> gene for leaf photosynthesis rate and yields is located in the hotspot [4,110.5-111] with EQF value 34.13 for all traits, EQF value 6.83 for vigor traits and EQF value 7.11 for yield traits. The well-known pleiotropic gene <em>SCM2</em> responsible for lodging resistance and yield is located very close to the detected hotspot [6,115.5-116], which is mostly composed of the QTL for yield, vigor and development traits. The <em>Xa3</em> gene conferring resistance to bacteria blight is located at the hotspot [11,118-118.5] with EQF value 35.11 for all traits, EQF value 13.00 for development traits and EQF value 12.04 for vigor traits. The analysis also recognizes several hotspot regions that do not have any known gene nearby so far (for example, in the 40â¼110-cM region of the 5<sup>th</sup> chromosome). These regions can be considered as potential regions of new genes that are functionally related to the QTL in the hotspots. In general, the QTL in the hotspots and their nearby genes are related in functions and locations to their associated traits.</p></div></div><div id="sec-14" class="subsection"><h3>Simulation study</h3><p id="p-35">The simulation study includes three parts. The first part aims to assess and compare the GWERs of the proposed statistical procedure and the Q-method in hotspot detection using the QTL data with correlation structure under the null hypothesis of no QTL hotspots. The second part focuses on assessing the performances of the two methods in detecting QTL hotspots when QTL hotspots are present. These two parts consider a 100-cM chromosome. The â¼1900 marker intervals and the 8216 QTL intervals in GRAMENE rice database (see the S1_DataQTL file) are served as the sample populations to generate the markers and QTL onto the simulated chromosomes. The third part attempts to evaluate the information loss when using summarized data instead of individual-level data by comparing the performances of the different methods in analyzing a simulated genetical genomics data set. The bin size in detection analysis is 0.5 cM (â³=0.5) for the first two parts and is 2 cM (â³=2.0) for the third part.</p><div id="sec-15" class="subsection"><h4>Genome-wide error rates:</h4><p id="p-36">In the first part, the QTL are generated to be randomly distributed but with different levels of correlation. We first assume the QTL belong to two trait categories and each trait category contains 150 QTL for the two different traits. Then, we deploy a hierarchical two-stage process to generate the QTL data in a trait category: (1) 105 QTL are randomly placed to the 200 possible positions (bins) without coincidence to determine their positions; (2) The remaining 45 QTL are then randomly assigned to the 105 determined QTL positions (in the first stage) by allowing coincidence. The hierarchical two-stage process can generate randomly distributed QTL (traits) with correlation structure in the genome. The first stage is to generate randomness for QTL, and the second stage allows to create correlations among QTL. We denote the above process as a (105,45) process for generating the QTL in a trait category. Then, the QTL data with different strengths of correlation under the null can be generated using different process, such as the (60,90) or (150,0) process. It follows that the QTL generated by the (60,90) process will be more correlated as compared to those by the (105,45) process, and those generated by the (150,0) process are uncorrelated under the null. We consider four kinds of processes, (A) (150,0) and (150,0), (B) (135,15) and (105,45), (C) (105,45) and (60,90), (D) (60,90) and (15,135), to generate the 300 QTL in the two trait categories. After the QTL positions are determined, the lengths of their flanking markers are randomly sampled from the 8216 QTL intervals in the GRAMENE database. Under such settings, the QTL in the different trait categories are uncorrelated, and those in the same trait category have different strengths of correlation. The four QTL data sets from the processes (A), (B), (C) and (D) are uncorrelated, weakly correlated, moderately correlated and highly correlated, respectively, and the data sets are analyzed by the proposed statistical procedure (<em>T</em> = 2) and Q-method (<em>T</em> = 4) to assess their GWERs. Both the EQF-bin and QTL-interval permutations are considered in the analysis. In our statistical procedure, we set <em>k</em> = 10 to investigate the GWERs of qFreq(<em>n</em>), <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-112.gif"/></noscript></span></span> here. The number of simulated replicates is 1000.</p><p id="p-37">The EQF-bin and QTL-interval permutations produce the similar permutation thresholds and same results. The results of the EQF-bin permutation are presented here. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> displays the observed GWERs of the proposed procedure and the Q-method at the <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-113.gif"/></noscript></span></span>0.02, 0.04, 0.06, 0.08 and 0.10 levels for the four QTL data sets. <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figures 4A</a> shows that for uncorrelated QTL the observed GWERs of the two approaches are about the right target levels. When the QTL are correlated, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figures 4, B-D</a> show that the GWERs of the Q-method are higher than the target levels and increase with correlation strength. Most strikingly, for the highly correlated data, the observed GWERs of the Q-method are closed to 1 at all the levels (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>), which implies that the detected hotspots by the Q-method are very likely to be spurious without accounting for the correlation features. On the other hand, the proposed procedure can cope with the correlations among QTL (by trait grouping) and control the GWERs close to the target levels.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/439/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Observed GWERs for the qFreq(n), n = 1,2, ..., 10, statistics of the proposed statistical procedure and for the Q-method at the &#x3B1; =0.02, 0.04, 0.06, 0.08 and 0.10 levels under varying strengths of QTL correlation: (A) uncorrelated, (B) weakly correlated, (C) moderately correlated, and (D) highly correlated, respectively. Black lines show the targeted error rates. Red lines show the observed GWERs of the Q-method. Green lines show the observed GWERs of the qFreq(n), n = 1,2, ..., 10, statistics." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2081177879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Observed GWERs for the qFreq(&lt;em&gt;n&lt;/em&gt;), &lt;em&gt;n&lt;/em&gt; = 1,2, ..., 10, statistics of the proposed statistical procedure and for the Q-method at the &#x3B1; =0.02, 0.04, 0.06, 0.08 and 0.10 levels under varying strengths of QTL correlation: (A) uncorrelated, (B) weakly correlated, (C) moderately correlated, and (D) highly correlated, respectively. Black lines show the targeted error rates. Red lines show the observed GWERs of the Q-method. Green lines show the observed GWERs of the qFreq(&lt;em&gt;n&lt;/em&gt;), &lt;em&gt;n&lt;/em&gt; = 1,2, ..., 10, statistics.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/439/F4.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/2/439/F4.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/439/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/439/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492234" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Observed GWERs for the qFreq(<em>n</em>), <em>n</em> = 1,2, ..., 10, statistics of the proposed statistical procedure and for the Q-method at the Î± =0.02, 0.04, 0.06, 0.08 and 0.10 levels under varying strengths of QTL correlation: (A) uncorrelated, (B) weakly correlated, (C) moderately correlated, and (D) highly correlated, respectively. Black lines show the targeted error rates. Red lines show the observed GWERs of the Q-method. Green lines show the observed GWERs of the qFreq(<em>n</em>), <em>n</em> = 1,2, ..., 10, statistics.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Performance in QTL hotspot detection:</h4><p id="p-39">For the second part, we assume there are six hotspots in the same 100-cM chromosome, and each hotspot is caused by a single gene. The six hotspots are assumed to be located at 10.25, 20.35, 31.15, 47.25, 56.40, and 67.20 cM of the chromosome, respectively. To generate QTL onto the chromosome, the contributive QTL of the top 100 hotspots in GRAMENE database are served as the sample populations of the six hotspots. The number of contributive QTL, C, ranges from 29 to 330 in the top 100 hotspots. For each simulated replicate, the contributive QTL in the top 100, top 20, top 50 to 100, top 30 to 60, top 50, and top 10 hotspots are served as the sample populations of the 1<sup>st</sup>, 2<sup>nd</sup>, 3<sup>rd</sup>, 4<sup>th</sup>, 5<sup>th</sup> and 6<sup>th</sup> hotspots, respectively. Such settings imply that the 2<sup>nd</sup> and 6<sup>th</sup> hotspots are stronger hotspots, and the 3<sup>rd</sup> hotspot is a weaker hotspot. Once the number C is determined, the relative positions and interval lengths of the <em>C</em> QTL are then randomly chosen from the respective sample population. The trait names and categories of the sampled QTL are still used, and hence correlation structure between QTL is similar to the GRAMENE database. The number of simulated replicate is 1000. For each simulated data set, the EQF matrix (<em>T</em>â236) and reduced EQF matrix (<em>T</em>â9) are constructed for detecting the QTL hotspots.</p><p id="p-40">To assess the performances of the two methods in detecting QTL hotspots, we use the true positive rate (TPR) and positive predictive value (PPV) to jointly measure the correct detection rate, and use the false discovery rate (FDR) to measure the incorrect detection rate. The TPR defines the proportion of the six hotspots that are correctly detected, and the PPV (FDR) defines the proportion of true (false) positive detection among all positive detections in the chromosome over the 1000 replicates (FDR = 1â PPV). To combine both PPV and TPR, we use the F1 score (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Van Rijsbergen 1979</a>)<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-114.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>which is the harmonic mean of PPV and TPR, to measure the correct detection rate. A high F1 score implies that PPV and TPR are both high and balanced. The average detected hotspot number, F1 score and FDR are together used to assess the performance of methods, and a quality method should have the ability to provide the result with correct hotspot numbers, high F1 score and low FDR in hotspot detection.</p><p id="p-41"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> depicts the F1 scores (<em>y</em>-axis), FDRs (<em>x</em>-axis) and average detected hotspot numbers (the numbers in the brackets) with the different thresholds over the 1000 simulation replicates. The average threshold of the Q-method is 7.58 (<em>Î²</em>=7.58). Using this threshold, the average number of detected hotspot is 9.94 (the true number is 6), indicating greater possibility of detecting extra false hotspots, and the associated F1 score and FDR are 0.749 and 0.400, respectively. With the proposed statistical procedure using the EQF-bin permutation, the average values of the thresholds <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-115.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-116.gif"/></noscript></span></span>,â..., <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-117.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-118.gif"/></noscript></span></span>for qFreq(1), qFreq(2),â..., qFreq(7) and qFreq(8) are 25.63, 18.45, 14.05, 11.53, 10.07, 9.11, 8.40 and 7.84 (â<em>Î²</em>), respectively, and the average numbers of hotspots detected under these thresholds are 1.59, 3.11, 4.46, 5.60, 6.57, 7.53, 8.52 and 9.46, respectively. The average hotspot number detected by using <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-119.gif"/></noscript></span></span> is 5.60, which is closest to the true number 6. The associated F1 scores are 0.419, 0.681, 0.836, 0.896, 0.891, 0.859, 0.814 and 0.770, respectively, and the associated FDRs are 0.000, 0.001, 0.020, 0.071, 0.148, 0.228, 0.306 and 0.371, respectively. In <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, the closer a result with a threshold is to the upper left corner, the better it performs, simply because the F1 scores are higher and the FDRs are lower. Apparently, as compared to the result of the Q-method with the threshold value <em>Î²</em>=7.58, the results with <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-120.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-121.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-122.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-123.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-124.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-125.gif"/></noscript></span></span> are closer to the upper left corner, and those with <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-126.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-127.gif"/></noscript></span></span> are farther from the upper left corner. The results with <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-128.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-129.gif"/></noscript></span></span> are better as they provide closer average hotspot numbers (5.60 and 6.57) to the true number 6, lower FDRs (0.071 and 0.148) and higher F1 scores (0.896 and 0.891), and the best result is obtained with <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-130.gif"/></noscript></span></span>, not with <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-131.gif"/></noscript></span></span>, in this specific setting (see CONCLUSION AND DISCUSSION for the reason). As expected, the Q-method produces a liberal threshold, which serves as the lower bound for the series thresholds <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-132.gif"/></noscript></span></span>âs, and detects more spurious hotspots. The proposed procedure can provide much stricter thresholds, some of which may yield better results with correct hotspot numbers, higher F1 scores and less spurious hotspots (lower FDRs), for the assessment. The same results are also obtained with the QTL-interval permutation (not shown).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/439/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The F1 scores (y-axis), FDRs (x-axis) and average detected hotspot numbers (the numbers in the brackets) of the hotspot detection with the different thresholds over the 1000 simulation replicates. The dots denote the average thresholds &#x3B3;n,0.05, n = 1,2, ..., 8, obtained by the proposed statistical procedure and are 25.63, 18.45, 14.05, 11.53, 10.07, 9.11, 8.40 and 7.84, respectively. The triangle denotes the average threshold &#x3B2; obtained by the Q-method and is 7.58. The true hotspot number is 6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2081177879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The F1 scores (y-axis), FDRs (x-axis) and average detected hotspot numbers (the numbers in the brackets) of the hotspot detection with the different thresholds over the 1000 simulation replicates. The dots denote the average thresholds &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;n&lt;/em&gt;,0.05&lt;/sub&gt;, &lt;em&gt;n&lt;/em&gt; = 1,2, ..., 8, obtained by the proposed statistical procedure and are 25.63, 18.45, 14.05, 11.53, 10.07, 9.11, 8.40 and 7.84, respectively. The triangle denotes the average threshold &lt;em&gt;&#x3B2;&lt;/em&gt; obtained by the Q-method and is 7.58. The true hotspot number is 6.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/439/F5.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/2/439/F5.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/439/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/439/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492144" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">The F1 scores (y-axis), FDRs (x-axis) and average detected hotspot numbers (the numbers in the brackets) of the hotspot detection with the different thresholds over the 1000 simulation replicates. The dots denote the average thresholds <em>Î³</em><sub><em>n</em>,0.05</sub>, <em>n</em> = 1,2, ..., 8, obtained by the proposed statistical procedure and are 25.63, 18.45, 14.05, 11.53, 10.07, 9.11, 8.40 and 7.84, respectively. The triangle denotes the average threshold <em>Î²</em> obtained by the Q-method and is 7.58. The true hotspot number is 6.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h4>Genetical genomics experiments:</h4><p id="p-43">For the third part, concerning the information loss, we mimic the simulation study in <a id="xref-ref-24-13" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> (2012)</a> with modification to simulate a small-scale genetical genomics data set, and then apply the proposed statistical procedure, Q-method, Breitlingâs method (the N-method) and Netoâs method (the NL-method) to analyze the data set for evaluation. The data set contains 100 backcross progeny with 5 chromosomes of length 100 cM and 600 molecular traits. Each chromosome contains 50 equally spaced markers. The 600 traits are assumed to belong to three different trait categories. Three hotspots are considered: (1) A small hotspot A is cause by a gene at 50 cM on the 1<sup>st</sup> chromosome. The gene controls 100 traits with heritabilities 0.3â¼0.4 showing moderate to high LOD scores (see Figure S3A) in QTL mapping. The 100 pleiotropic traits are assigned to the first trait category; (2) A big hotspot B is caused by a gene located at 50 cM on the 3<sup>rd</sup> chromosome. The gene influences 300 traits belonging to the second category. Among the 300 pleiotropic traits, half have heritabilities 0.1â¼0.45 showing small to high LOD scores (see Figure S3B), and half have heritaibilities 0.3â¼0.45 showing moderate to high LOD scores (see Figure S3B); (3) A big hotspot C is caused by a gene at 50 cM on the 5<sup>th</sup> chromosome. The gene controls 200 traits that belong to the third category. The heritabilities of the 200 pleiotropic traits are 0.1â¼0.2 showing small LOD scores (see Figure S3C). The pairwise correlations between traits are shown in Figure S3D. The bin size of 2 cM (â³=2 cM) is used in the analysis (similar to that in Neto <em>et al.</em>). The bin containing the estimated QTL position will be given to 1 (and 0 otherwise) to construct the QTL matrix. Both the proposed statistical procedure and the Q-method operate on the QTL matrix for detection analysis. The trait grouping in the proposed procedure considers that all pleiotropic traits are correctly allocated to the same categories (perfect trait grouping). For the N-method and NL-method, we follow <a id="xref-ref-24-14" class="xref-bibr" href="#ref-24">Neto <em>et al.</em> (2012)</a> to adopt 1.5-LOD support intervals for the backcross to decrease the spread of the hotspots.</p><p id="p-44"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the results of the four methods for the simulated data set. <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a> presents the hotspot architecture constructed using a single-trait LOD threshold of 2.47 and the 5% significance hotspot size thresholds obtained by the Q-method, N-method, and the proposed procedure. The permutation thresholds delivered by the Q-method and N-method are 3 and 11, respectively. Under the thresholds, there are 18 and 38 significant bins (detected hotspots), respectively (see <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A</a>), showing that, in addition to detecting the three true hotspots, several spurious hotspots are also detected near the true hotspots. The permutation thresholds obtained by the proposed procedure for assessing the significance of at least 1, 2, 3 and 4 spurious hotspots are <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-133.gif"/></noscript></span></span> 208, <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-134.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-135.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-136.gif"/></noscript></span></span>, respectively. Under these thresholds, there are 1, 1, 4 and 5 detected hotspots, respectively, indicating that less spurious hotspots are detected due to higher thresholds. Using <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/439/embed/mml-math-137.gif"/></noscript></span></span> as a threshold, the three true hotspots (the highest bins on chromosomes 1, 3, and 5) and one spurious hotspot (right next to the true hotspot on chromosome 3) are significant. <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figures 6, B-F</a>, present the hotspot architectures inferred using the NL-method LOD thresholds of 5.36, 3.04, 2.03, 1.24, and 1.07 that aim to control GWER of 5% for spurious hotspots of sizes 1, 5, 20, 77, and 100, respectively. Under these thresholds, there are 31, 22, 11, 3 and 1 significant bins-(hotspots). Using LOD thresholds of 5.36, 3.04, 2.03, not only the three true hotspots but also some spurious hotspots (the secondary peaks around the true hotspots) are detected. Using a LOD threshold of 1.24, only the three true hotspots (on chromosomes 1, 3, and 5) are significant. Using a LOD threshold of 1.07, only the true hotspot on chromosome 3 is detected as significant. The above shows that, if trait grouping is perfect, the proposed statistical procedure is applicable and can obtain comparable results in QTL hotspot detection in the genetical genomics experiments. But note that if trait grouping is not perfect, the hotspot thresholds decrease and the possibility of detecting more spurious hotspots increases (not shown).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/439/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Panels (A-F) The proposed statistical procedure, Q-method, N-method and NL- method analyses for simulated example. Panels (A) Inferred hotspot architecture using a single-trait permutation LOD threshold of 2.47 corresponding to a GWER of 5% of falsely detecting at least one QTL somewhere in the genome. The hotspots on chromosomes 1, 3 and 5 have sizes 50, 210, and 50, respectively. The blue line at count 11 corresponds to the hotspot size threshold at a GWER of 5% according to the N- method. The red line at count 3 corresponds to the Q-method&#x2019;s 5% significance threshold. The thresholds &#x3B3;1,0.05, &#x3B3;2,0.05, &#x3B3;3,0.05 and &#x3B3;4,0.05 obtained by the proposed procedure are 208, 58, 47 and 34, respectively. Panels (B, C, D, E and F) Hotspot architecture inferred using different permutation thresholds by the NLmethod; Hotspot architectures computed using QTL mapping LOD thresholds of 5.36 (B), 3.04 (C), 2.03 (D), 1.24 (E), and 1.07 (F) that aim to control GWER at a 5% error rate for spurious QTL hotspots of sizes 1, 5, 20, 77, and 100, respectively. The number in the bracket is the number of detected hotspots. Results are based on 1000 permutations. Q: The Q-method; N: The N-method." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2081177879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Panels (A-F) The proposed statistical procedure, Q-method, N-method and NL- method analyses for simulated example. Panels (A) Inferred hotspot architecture using a single-trait permutation LOD threshold of 2.47 corresponding to a GWER of 5% of falsely detecting at least one QTL somewhere in the genome. The hotspots on chromosomes 1, 3 and 5 have sizes 50, 210, and 50, respectively. The blue line at count 11 corresponds to the hotspot size threshold at a GWER of 5% according to the N- method. The red line at count 3 corresponds to the Q-method&#x2019;s 5% significance threshold. The thresholds &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;1,0.05&lt;/sub&gt;, &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;2,0.05&lt;/sub&gt;, &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;3,0.05&lt;/sub&gt; and &lt;em&gt;&#x3B3;&lt;/em&gt;&lt;sub&gt;4,0.05&lt;/sub&gt; obtained by the proposed procedure are 208, 58, 47 and 34, respectively. Panels (B, C, D, E and F) Hotspot architecture inferred using different permutation thresholds by the NLmethod; Hotspot architectures computed using QTL mapping LOD thresholds of 5.36 (B), 3.04 (C), 2.03 (D), 1.24 (E), and 1.07 (F) that aim to control GWER at a 5% error rate for spurious QTL hotspots of sizes 1, 5, 20, 77, and 100, respectively. The number in the bracket is the number of detected hotspots. Results are based on 1000 permutations. Q: The Q-method; N: The N-method.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/439/F6.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/2/439/F6.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/439/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/439/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492223" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-45" class="first-child">Panels (A-F) The proposed statistical procedure, Q-method, N-method and NL- method analyses for simulated example. Panels (A) Inferred hotspot architecture using a single-trait permutation LOD threshold of 2.47 corresponding to a GWER of 5% of falsely detecting at least one QTL somewhere in the genome. The hotspots on chromosomes 1, 3 and 5 have sizes 50, 210, and 50, respectively. The blue line at count 11 corresponds to the hotspot size threshold at a GWER of 5% according to the N- method. The red line at count 3 corresponds to the Q-methodâs 5% significance threshold. The thresholds <em>Î³</em><sub>1,0.05</sub>, <em>Î³</em><sub>2,0.05</sub>, <em>Î³</em><sub>3,0.05</sub> and <em>Î³</em><sub>4,0.05</sub> obtained by the proposed procedure are 208, 58, 47 and 34, respectively. Panels (B, C, D, E and F) Hotspot architecture inferred using different permutation thresholds by the NLmethod; Hotspot architectures computed using QTL mapping LOD thresholds of 5.36 (B), 3.04 (C), 2.03 (D), 1.24 (E), and 1.07 (F) that aim to control GWER at a 5% error rate for spurious QTL hotspots of sizes 1, 5, 20, 77, and 100, respectively. The number in the bracket is the number of detected hotspots. Results are based on 1000 permutations. Q: The Q-method; N: The N-method.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">To sum up, the simulation study shows that the proposed statistical procedure can control GWERs at the target levels for the QTL data with correlation structure, has the ability to produce quality results by offering a sliding scale of thresholds from high to low for QTL hotspot detection, and is applicable to the hotspot analysis in genetical genomics studies.</p></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-47">Both genetical genomics experiments and public QTL databases can provide abundant QTL for genome-wide detection of QTL hotspots to explore the genetic mechanism of quantitative traits in biological studies. A single genetical genomics experiment can produce an adequate individual-level data set that contains the genotypes and a large enough number of phenotypes for QTL mapping and further for QTL hotspot detection. On the contrary, public QTL databases consistently collect summarized QTL data for many phenotypic traits from numerous independent experiments that allows for detection of QTL hotspots. Several methods, mainly for using individual-level data, have been proposed to detect QTL hotspots (see introduction). In this article, we develop a statistical procedure for detecting QTL hotspots at the genome-wide level by using summarized QTL data in public databases. We first obtain the QTL intervals from public databases and use them to compute the EQF matrix for operation. We then derive a permutation algorithm on the EQF elements or the QTL intervals to compute a sliding scale of EQF thresholds that range from conservative to liberal for assessing the significance of QTL hotspots. To consider the correlation structure among traits, the correlated traits are grouped together as a trunk or unit of permutation to obtain stricter thresholds. As shown in simulation study, with the grouping strategy, our statistical procedure can control the GWERs of the test statistic at the target levels under varying strengths of correlation among QTL and can provide much more rigorous thresholds for hotspot detection with higher correct (lower incorrect) detection rates. Besides, to evaluate the information loss between the two types of data in hotspot detection, we apply the proposed procedure, Q-method, Breitlingâs method and Netoâs approach to analyze a simulated genetical genomics data set and compare their results. It shows that our procedure is comparable to the methods using individual-level data if the pleiotropic (correlated) traits can be all grouped together into the same trait categories. In the GRAMENE rice database, more than 100 QTL hotspots were detected in the genome. We also conduct a genome-wide comparative analysis of the detected hotspots and the 122 known genes in the Rice Q-TARO database. The comparative analysis shows that the QTL hotspots and genes are conformable in the sense that they co-localize closely and are functionally related to the associated traits. An R package of our proposed statistical procedure called QHOT is available on <a href="http://www.stat.sinica.edu.tw/chkao/">http://www.stat.sinica.edu.tw/chkao/</a> and is being submitted to Comprehensive R Archive Network (CRAN). The R codes can readily produce both numerical and graphical outputs that would allow visualization of several features, including the EQF architecture, the QTL components of the hotspots, and nearby known genes, at the genome-wide level (see <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figures 2</a>, <a id="xref-fig-3-5" class="xref-fig" href="#F3">3</a> and S2) for exploring the interplay among QTL, genes and traits.</p><p id="p-48">The QTL hotspot detection using public databases relies on the great number of QTL collected from numerous independent QTL mapping studies. The collected QTL are usually mapped for various traits and detected by different statistical tools under a wide range of experimental parameter settings. Quite often some biologically interesting and popular important traits are more frequently investigated and mapped for QTL in the studies, resulting in a group of traits strongly mapping to the same or closely linked QTL mostly due to the genetic correlations among them. Take rice as an example, the agronomic traits, such as panicle numbers, grains per panicle and grain weight, plant height, heading time, grain quality, insect and disease resistant, sterility, etc., are often investigated and highly correlated to each other (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Swamy <em>et al.</em> 2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Trijatmiko <em>et al.</em> 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wu <em>et al.</em> 2016</a>). To account for the correlation structure among traits, we group and permute these traits together to determine a series of thresholds for hotspot detection. Other grouping, such as by the already known correlations between traits or biological prior knowledge on traits, is also applicable. The simulation study shows that grouping of highly correlated traits is effective to control the error rates of falsely detecting a hotspot. If genetically correlated traits are not grouped together and used as control, the error rates may inflate greatly and will be higher than the target levels. The error rates are not sensitive to the situation in which uncorrelated traits are falsely grouped together. Ideally, we would like to have a perfect grouping, in which traits are correlated in the same categories and uncorrelated in different categories (the first and third parts of simulation), or the pleiotropic traits are all grouped into the same trait categories (the third part of simulation), to cope with their correlation features. However, in practice, a perfect grouping is not always possible because correlations among traits are likely to be common, and the pleiotropic traits may be allocated to different categories. For example, the yield and disease resistant are correlated and assigned into different categories in GRAMENE database. The grain weight and plant architecture are pleiotropic traits and classified into different categories (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fujita <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Xu <em>et al.</em> 2015</a>). The GRAMENE rice database classifies all traits into nine major trait categories based on the general agronomic consideration, which is imprecise in the sense that traits in the same categories may have different strengths of genetic correlation, and traits in different categories still preserve certain degrees of genetic correlation. This allows us to argue that underestimation of the threshold and the problem of information loss are still very likely to occur to some extent in the real example analysis, resulting in excesses of observed over expected hotspot numbers (as validated in the real example analysis and simulation study).</p><p id="p-49">In the QTL hotspot analysis using genetical genomics experiments, the proposed statistical procedure can be extended as follows: First, QTL mapping is performed to obtain the QTL matrix; Second, the correlation coefficients among the traits are computed for grouping reference; Next, the QTL for the (highly) correlated traits are grouped and permuted together to obtain a series of thresholds for assessing hotspot significance. Note that the trait grouping can be also done in different ways, such as by principal component analysis or cluster analysis (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdi and Williams 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Everitt <em>et al.</em> 2011</a>). Such a procedure is easy to implement and very cheap in computation as compared to the methods by permuting the original individual-level data in the hotspot analysis (see <em>The permutation algorithm</em><strong>)</strong>. It has been observed that QTL or genes for genetically correlated traits have a tendency to cluster on the same or adjacent regions of chromosomes in several organisms, which may be due to linkage, pleiotropy or natural selection for co-adapted traits (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Studer and Doebley 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wu <em>et al.</em> 2015</a>). The grouping strategy attempts to take the clustering phenomenon into account in the detection of QTL hotspots. In data collection, it would be important to collect as many QTL and genes for various traits as possible of to identify the clustering phenomenon and explore the interplays among QTL, genes and traits. Our real example analysis considers the 8216 QTL for 236 component traits in GRAMENE rice database for hotspot detection and compares with the 122 known genes in QTARO database. The results display the clustering phenomenon of QTL and genes around the detected hotspots. For example, the well-known pleiotropic gene <em>SCM2</em> responsible for lodging resistance and yield is very close to the QTL hotspot [6,115.5-116] (see Figure S2), which contains QTL for yield, vigor and development. Also, five closely linked genes, <em>Rc</em>, <em>qSD7-1/qPC7</em>, <em>OsHMA3</em>, <em>qCDT7</em> and <em>Ghd7</em>, with similar functions are localized around the detected hotspots [7,44.5-45] and [7,50-50.5]. Understanding the genetic architectures of quantitative traits at the genome-wide level has been a key and challenging issue in various areas of genetics, gene and genomics studies. This would rely heavily on effectively integrating and analyzing the information on the QTL and genes from the published literature. Currently, several well-known databases of important organisms (see INTRODUCTION section) have consistently collected the information and allow researchers access to their well-curated datasets, and, therefore, increasing numbers of QTL and genes for a variety of traits are available and ready for further application. There is a need to develop statistical methods for mining the useful information and knowledge from these databases. By using public databases, we develop a statistical procedure for detecting QTL hotspots, perform comparative analysis with the known genes for validation, and also identify several potential QTL regions of genes (not shown). The R codes can present the EQF architectures for hotspots (QTL), genes and quantitative traits region by region along the genome to overview their connections. Our approach can provide a way to explore networks among QTL hotspots, genes and traits for dissecting the genetic architecture of complex traits in broad areas of biological studies.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">We are grateful to the associate editor and the three referees for their valuable comments and suggestions. This study was supported partly by grants NSC107-2118-M-001-008 from Ministry of Science and Technology, Taiwan, Republic of China.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-51">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7451393">https://doi.org/10.25387/g3.7451393</a>.</p></li><li class="fn" id="fn-2"><p id="p-52"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-04-25"><span class="received-label">Received </span>April 25, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-12-05"><span class="accepted-label">Accepted </span>December 5, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2019 Yang <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.1" data-doi="10.1002/wics.101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Principal component analysis.</span> <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. Comput. Stat.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">433</span>â<span class="cit-lpage">459</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/wics.101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.%2BRev.%2BComput.%2BStat.%26rft.volume%253D2%26rft.spage%253D433%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwics.101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wics.101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.2" data-doi="10.1371/journal.pone.0068150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zahid</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Evidence of Multiple Disease Resistance (MDR) and implication of meta-analysis in marker assisted selection.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e68150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0068150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De68150%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0068150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0068150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.3" data-doi="10.3389/fpls.2015.01032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basnet</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duwal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monakhos</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Quantitative Trait Locus Analysis of Seed Germination and Seedling Vigor in Brassica rapa Reveals OTL Hotspots and Epistatic Interactions.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1032</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2015.01032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D1032%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2015.01032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2015.01032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.4" data-doi="10.1371/journal.pgen.1000232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breitling</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesson</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genetical genomics: spotlight on QTL hotspots.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBreitling%26rft.auinit1%253DR.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000232%26rft.epage%253De1000232%26rft.atitle%253DGenetical%2Bgenomics%253A%2Bspotlight%2Bon%2BQTL%2Bhotspots.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000232%26rft_id%253Dinfo%253Apmid%252F18949031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18949031&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Mapping quantitative trait loci in the case of a spike in the phenotype distribution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1169</span>â<span class="cit-lpage">1175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D163%26rft.issue%253D3%26rft.spage%253D1169%26rft.epage%253D1175%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2Bin%2Bthe%2BCase%2Bof%2Ba%2BSpike%2Bin%2Bthe%2BPhenotype%2BDistribution%26rft_id%253Dinfo%253Apmid%252F12663553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2My8zLzExNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvNDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.6" data-doi="10.1534/g3.112.002618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabrera</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huffman</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Uncovering networks from genome-wide association studies via circular genomic permutation. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1067</span>â<span class="cit-lpage">1075</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.002618</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D2%26rft.spage%253D1067%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.002618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.002618&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.7" data-doi="10.1534/genetics.104.032375"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chardon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virlon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joets</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Genetic architecture of flowering time in maize as inferred from quantitative trait loci meta-analysis and synteny conservation with the rice genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">2169</span>â<span class="cit-lpage">2185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.032375</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChardon%26rft.auinit1%253DF.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D2169%26rft.epage%253D2185%26rft.atitle%253DGenetic%2BArchitecture%2Bof%2BFlowering%2BTime%2Bin%2BMaize%2BAs%2BInferred%2BFrom%2BQuantitative%2BTrait%2BLoci%2BMeta-analysis%2Band%2BSynteny%2BConservation%2BWith%2Bthe%2BRice%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.032375%26rft_id%253Dinfo%253Apmid%252F15611184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzIxNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvNDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-book" id="cit-9.2.439.8" data-doi="10.1002/9780470977811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Everitt</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landau</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leese</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-source"><em>Cluster analysis: Wiley series in probability and statistics</em>, <em>pp</em></span>, <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">Chichester</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/9780470977811</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCluster%2Banalysis%253A%2BWiley%2Bseries%2Bin%2Bprobability%2Band%2Bstatistics%252C%2Bpp%26rft_id%253Dinfo%253Adoi%252F10.1002%252F9780470977811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/9780470977811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-book" id="cit-9.2.439.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-source"><em>Introduction to quantitative genetics. Longman</em>, <em>Essex</em>, <em>UK. Introduction to quantitative genetics</em></span>, <span class="cit-edition">Ed. 4th</span>. <span class="cit-publ-name">Longman</span>, <span class="cit-publ-loc">Essex, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.10" data-doi="10.1007/s10681-013-1060-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daunay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huvenaars</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">QTL hotspots in eggplant (Solanum melongena) detected with a high resolution map and CIM analysis.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">211</span>â<span class="cit-lpage">228</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-013-1060-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D197%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-013-1060-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-013-1060-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.11" data-doi="10.1038/ng.308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keurentjes</span>  <span class="cit-name-given-names">J. J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouwmeester</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">America</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstappen</span>  <span class="cit-name-given-names">F. W. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">System-wide molecular evidence for phenotypic bufferingin Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">166</span>â<span class="cit-lpage">167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DFu%26rft.auinit1%253DJ.%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D166%26rft.epage%253D167%26rft.atitle%253DSystem-wide%2Bmolecular%2Bevidence%2Bfor%2Bphenotypic%2Bbuffering%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.308%26rft_id%253Dinfo%253Apmid%252F19169256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19169256&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263091300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.12" data-doi="10.1073/pnas.1310790110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujita</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trijatmiko</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tagle</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sapasap</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koide</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">NAL1 allele from a rice landrace greatly increases yield in modern indica cultivars.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">20431</span>â<span class="cit-lpage">20436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1310790110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1310790110%26rft_id%253Dinfo%253Apmid%252F24297875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzUxLzIwNDMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzQzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerber</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Quantitative trait loci: a meta-analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">463</span>â<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGoffinet%26rft.auinit1%253DB.%26rft.volume%253D155%26rft.issue%253D1%26rft.spage%253D463%26rft.epage%253D473%26rft.atitle%253DQuantitative%2BTrait%2BLoci%253A%2BA%2BMeta-analysis%26rft_id%253Dinfo%253Apmid%252F10790417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzEvNDYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzQzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.14" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>â<span class="cit-lpage">324</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.1992.131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DA%2Bsimple%2Bregression%2Bmethod%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Bline%2Bcrosses%2Busing%2Bflanking%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harushima</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shomura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">A high-density rice genetic linkage map with 2275 markers using a single F2 population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">479</span>â<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHarushima%26rft.auinit1%253DY.%26rft.volume%253D148%26rft.issue%253D1%26rft.spage%253D479%26rft.epage%253D494%26rft.atitle%253DA%2BHigh-Density%2BRice%2BGenetic%2BLinkage%2BMap%2Bwith%2B2275%2BMarkers%2BUsing%2Ba%2BSingle%2BF2%2BPopulation%26rft_id%253Dinfo%253Apmid%252F9475757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ4LzEvNDc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzQzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Interval mapping of multiple quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">205</span>â<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJansen%26rft.auinit1%253DR.%2BC.%26rft.volume%253D135%26rft.issue%253D1%26rft.spage%253D205%26rft.epage%253D211%26rft.atitle%253DInterval%2Bmapping%2Bof%2Bmultiple%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F8224820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM1LzEvMjA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzQzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.17" data-doi="10.1534/genetics.106.056416"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Mapping quantitative trait loci using the experimental designs of recombinant inbred populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1373</span>â<span class="cit-lpage">1386</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.056416</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.056416%26rft_id%253Dinfo%253Apmid%252F17121967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8zLzEzNzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvNDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teasdale</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Multiple interval mapping for quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1203</span>â<span class="cit-lpage">1216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKao%26rft.auinit1%253DC.-H.%26rft.volume%253D152%26rft.issue%253D3%26rft.spage%253D1203%26rft.epage%253D1216%26rft.atitle%253DMultiple%2BInterval%2BMapping%2Bfor%2BQuantitative%2BTrait%2BLoci%26rft_id%253Dinfo%253Apmid%252F10388834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi8zLzEyMDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvNDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>â<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzQzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.20" data-doi="10.1534/genetics.114.168385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">A new simple method for improving QTL mapping under selective genotyping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1685</span>â<span class="cit-lpage">1698</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.168385</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.168385%26rft_id%253Dinfo%253Apmid%252F25245793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE2ODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvNDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.21" data-doi="10.1080/10543400903572803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">A systems biology approach for identifying novel pathway regulators in eQTL mapping.</span> <abbr class="cit-jnl-abbrev">J. Biopharm. Stat.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">373</span>â<span class="cit-lpage">400</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/10543400903572803</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiopharmaceutical%2Bstatistics%26rft.stitle%253DJ%2BBiopharm%2BStat%26rft.aulast%253DLi%26rft.auinit1%253DS.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D373%26rft.epage%253D400%26rft.atitle%253DA%2Bsystems%2Bbiology%2Bapproach%2Bfor%2Bidentifying%2Bnovel%2Bpathway%2Bregulators%2Bin%2BeQTL%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10543400903572803%26rft_id%253Dinfo%253Apmid%252F20309764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10543400903572803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20309764&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.22" data-doi="10.1038/nature03895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Rice Genome Sequencing Project</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The map-based sequence of the rice genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">436</span>: <span class="cit-fpage">793</span>â<span class="cit-lpage">800</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature03895</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D436%26rft.issue%253D7052%26rft.spage%253D793%26rft.epage%253D800%26rft.atitle%253DThe%2Bmap-based%2Bsequence%2Bof%2Bthe%2Brice%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03895%26rft_id%253Dinfo%253Apmid%252F16100779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16100779&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231116500034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.23" data-doi="10.1111/pbi.12538"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mengistu</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidane</span>  <span class="cit-name-given-names">Y. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catellani</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frascaroli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fadda</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">High-density molecular characterization and association mapping in Ethiopian durum wheat landraces reveals high diversity and potential for wheat breeding.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1800</span>â<span class="cit-lpage">1812</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12538</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D14%26rft.spage%253D1800%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.24" data-doi="10.1534/genetics.112.139451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neto</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Quantile-Based Permutation Thresholds for Quantitative Trait Loci Hotspots.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">1355</span>â<span class="cit-lpage">1365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.139451</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139451%26rft_id%253Dinfo%253Apmid%252F22661325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MS80LzEzNTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvNDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.25" data-doi="10.1038/ncomms1132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ookawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murata</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ando</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">New approach for rice improvement using a pleiotropic QTL gene for lodging resistance and yield.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DOokawa%26rft.auinit1%253DT.%26rft.volume%253D1%26rft.spage%253D132%26rft.epage%253D132%26rft.atitle%253DNew%2Bapproach%2Bfor%2Brice%2Bimprovement%2Busing%2Ba%2Bpleiotropic%2BQTL%2Bgene%2Bfor%2Blodging%2Bresistance%2Band%2Byield.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1132%26rft_id%253Dinfo%253Apmid%252F21119645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21119645&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.26" data-doi="10.1038/ejhg.2009.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Gene and pathway-based second-wave analysis of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">111</span>â<span class="cit-lpage">117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ejhg.2009.115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DPeng%26rft.auinit1%253DG.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D117%26rft.atitle%253DGene%2Band%2Bpathway-based%2Bsecond-wave%2Banalysis%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2009.115%26rft_id%253Dinfo%253Apmid%252F19584899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2009.115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19584899&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272609900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.27" data-doi="10.1038/nature01434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schadt</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monks</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lusis</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Che</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Genetics of gene expression surveyed in maize, mouse and man.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>: <span class="cit-fpage">297</span>â<span class="cit-lpage">302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature01434</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchadt%26rft.auinit1%253DE.%2BE.%26rft.volume%253D422%26rft.issue%253D6929%26rft.spage%253D297%26rft.epage%253D302%26rft.atitle%253DGenetics%2Bof%2Bgene%2Bexpression%2Bsurveyed%2Bin%2Bmaize%252C%2Bmouse%2Band%2Bman.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01434%26rft_id%253Dinfo%253Apmid%252F12646919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12646919&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181637300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">A statistical framework for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">371</span>â<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSen%26rft.auinit1%253DS.%26rft.volume%253D159%26rft.issue%253D1%26rft.spage%253D371%26rft.epage%253D387%26rft.atitle%253DA%2BStatistical%2BFramework%2Bfor%2BQuantitative%2BTrait%2BMapping%26rft_id%253Dinfo%253Apmid%252F11560912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzEvMzcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzQzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.29" data-doi="10.1534/genetics.111.126508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Studer</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Do Large Effect QTL Fractionate? A Case Study at the Maize Domestication QTL teosinte branched1.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">673</span>â<span class="cit-lpage">681</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.126508</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.126508%26rft_id%253Dinfo%253Apmid%252F21515578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzMvNjczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzQzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.30" data-doi="10.1186/1471-2164-12-319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swamy</span>  <span class="cit-name-given-names">B. P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vikram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">H. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Meta-analysis of grain yield QTL identified during agricultural drought in grasses showed consensus.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMallikarjuna%2BSwamy%26rft.auinit1%253DB.%2BP.%26rft.volume%253D12%26rft.spage%253D319%26rft.epage%253D319%26rft.atitle%253DMeta-analysis%2Bof%2Bgrain%2Byield%2BQTL%2Bidentified%2Bduring%2Bagricultural%2Bdrought%2Bin%2Bgrasses%2Bshowed%2Bconsensus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-319%26rft_id%253Dinfo%253Apmid%252F21679437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21679437&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.31" data-doi="10.1007/s11032-013-0012-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trijatmiko</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Supriyanta</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prasetiyono</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vera Cruz</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Meta-analysis of quantitative trait loci for grain yield and component traits under reproductive-stage drought stress in an upland rice population.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">283</span>â<span class="cit-lpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-013-0012-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D34%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-013-0012-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-013-0012-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-book" id="cit-9.2.439.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Rijsbergen</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span>â<span class="cit-source">Information retrieval</span>, <span class="cit-publ-name">Butterworths</span>, <span class="cit-publ-loc">London, Boston</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.33" data-doi="10.1186/s12864-015-1811-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vuong</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshmukh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Genetic architecture of cyst nematode resistance revealed by genome-wide association study in soybean.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">593</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1811-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D593%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1811-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1811-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.34" data-doi="10.1534/genetics.115.179945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">A Random-Model Approach to QTL Mapping in Multiparent Advanced Generation Intercross (MAGIC) Populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">471</span>â<span class="cit-lpage">486</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.179945</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179945%26rft_id%253Dinfo%253Apmid%252F26715662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvNDcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzQzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.35" data-doi="10.1534/genetics.106.064972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">M. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliebenstein</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Global eQTL mapping reveals the complex genetic architecture of transcript-level variation in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1441</span>â<span class="cit-lpage">1450</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.064972</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.064972%26rft_id%253Dinfo%253Apmid%252F17179097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzE0NDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvNDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.36" data-doi="10.1073/pnas.1011739108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolkman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patzoldt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Multivariate analysis of maize disease resistances suggests a pleiotropic genetic basis and implicates a GST gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">7339</span>â<span class="cit-lpage">7344</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1011739108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1011739108%26rft_id%253Dinfo%253Apmid%252F21490302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE4LzczMzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvNDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.37" data-doi="10.1007/s11032-015-0202-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Two quantitative trait loci for grain yield and plant height on chromosome 3 are tightly linked in coupling phase in rice.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D35%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-015-0202-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-015-0202-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.38" data-doi="10.1371/journal.pgen.1000070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delano</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Gene set enrichment in eQTL data identifies novel annotations and pathway regulators.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000070</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DC.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De1000070%26rft.epage%253De1000070%26rft.atitle%253DGene%2Bset%2Benrichment%2Bin%2BeQTL%2Bdata%2Bidentifies%2Bnovel%2Bannotations%2Band%2Bpathway%2Bregulators.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000070%26rft_id%253Dinfo%253Apmid%252F18464898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18464898&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.39" data-doi="10.1007/s00438-016-1227-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">X. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Quantitative trait loci identification and meta-analysis for rice panicle-related traits.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">291</span>: <span class="cit-fpage">1927</span>â<span class="cit-lpage">1940</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-016-1227-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGenet.%2BGenomics%26rft.volume%253D291%26rft.spage%253D1927%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-016-1227-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-016-1227-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.40" data-doi="10.1007/s13258-015-0312-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Identification of QTLs for agronomic traits in indica rice using an RIL population.</span> <abbr class="cit-jnl-abbrev">Genes Genomics</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">809</span>â<span class="cit-lpage">817</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s13258-015-0312-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomics%26rft.volume%253D37%26rft.spage%253D809%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13258-015-0312-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13258-015-0312-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.41" data-doi="10.1186/1939-8433-5-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yonemaru</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">OGRO: The Overview of functionally characterized Genes in Rice online database.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1939-8433-5-26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D5%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1939-8433-5-26%26rft_id%253Dinfo%253Apmid%252F27234245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1939-8433-5-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27234245&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Precision mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1457</span>â<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1457%26rft.epage%253D1468%26rft.atitle%253DPrecision%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F8013918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzE0NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvNDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.2.439.43" data-doi="10.1038/ncomms1467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms1467</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.spage%253D467%26rft.epage%253D467%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Breveals%2Ba%2Brich%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1467%26rft_id%253Dinfo%253Apmid%252F21915109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915109&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F439.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/2/439.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/2/425" title="Complex Structural PPT1 Variant Associated with Non-syndromic Canine Retinal Degeneration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/2/453" title="GAL4 Drivers Specific for Type Ib and Type Is Motor Neurons in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/2"><span>Volume 9 Issue 2, February 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/2.cover-source.jpg?itok=Pgd0L52l" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/439" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DSC8RVXLg8RafPRPdR_LJ8C_1qEEce3r1dkhlYZDieU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/2/439" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/2/439" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/2/439" class="active" data-icon-position="" data-hide-link-title="0">A Statistical Procedure for Genome-Wide Detection of QTL Hotspots Using Public Databases with Application to Rice</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492213" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-48zlNE70_7e4OO3OBBhRfeLggvq04-CZhSrHQ_HyX88" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/2/439.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/2/439" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_l1uNuZnxiZU6Z4a5BuzwaOfejHd2rtGbNKBzIk07fg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/2/439" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/2/439.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492213" id="node492213--2" data-pisa="ggg;9/2/439" data-pisa-master="ggg;g3.118.200922" data-apath="/ggg/9/2/439.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Statistical Procedure for Genome-Wide Detection of QTL Hotspots Using Public Databases with Application to Rice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Man-Hsia Yang</span>, <span class="highwire-citation-author" data-delta="1">Dong-Hong Wu</span> and <span class="highwire-citation-author" data-delta="2">Chen-Hung Kao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">439-452;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200922 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492213--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Man-Hsia Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, National Taiwan University, Taipei 10617, Taiwan, Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Crop Science Division, Taiwan Agricultural Research Institute, Council of Agriculture, Taichung 41362, Taiwan, Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Man-Hsia%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMan-Hsia%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dong-Hong Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Crop Science Division, Taiwan Agricultural Research Institute, Council of Agriculture, Taichung 41362, Taiwan, Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dong-Hong%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADong-Hong%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chen-Hung Kao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, National Taiwan University, Taipei 10617, Taiwan, Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Institute of Statistical Science, Academia Sinica, Taipei 11529, Taiwan, Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chen-Hung%2BKao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kao%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChen-Hung%2BKao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chkao@stat.sinica.edu.tw" class="" data-icon-position="" data-hide-link-title="0">chkao@stat.sinica.edu.tw</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/2/439" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/2/439&amp;title=A%20Statistical%20Procedure%20for%20Genome-Wide%20Detection%20of%20QTL%20Hotspots%20Using%20Public%20Databases%20with%20Application%20to%20Rice" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/2/439&amp;title=A%20Statistical%20Procedure%20for%20Genome-Wide%20Detection%20of%20QTL%20Hotspots%20Using%20Public%20Databases%20with%20Application%20to%20Rice" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/2/439&amp;title=A%20Statistical%20Procedure%20for%20Genome-Wide%20Detection%20of%20QTL%20Hotspots%20Using%20Public%20Databases%20with%20Application%20to%20Rice" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/2/439&amp;text=A%20Statistical%20Procedure%20for%20Genome-Wide%20Detection%20of%20QTL%20Hotspots%20Using%20Public%20Databases%20with%20Application%20to%20Rice" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/2/439&amp;title=A%20Statistical%20Procedure%20for%20Genome-Wide%20Detection%20of%20QTL%20Hotspots%20Using%20Public%20Databases%20with%20Application%20to%20Rice" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/2/439&amp;t=A%20Statistical%20Procedure%20for%20Genome-Wide%20Detection%20of%20QTL%20Hotspots%20Using%20Public%20Databases%20with%20Application%20to%20Rice" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/2/439&amp;title=A%20Statistical%20Procedure%20for%20Genome-Wide%20Detection%20of%20QTL%20Hotspots%20Using%20Public%20Databases%20with%20Application%20to%20Rice" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/2/439&amp;title=A%20Statistical%20Procedure%20for%20Genome-Wide%20Detection%20of%20QTL%20Hotspots%20Using%20Public%20Databases%20with%20Application%20to%20Rice" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492213/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492213" id="node492213--3" data-pisa="ggg;9/2/439" data-pisa-master="ggg;g3.118.200922" data-apath="/ggg/9/2/439.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Statistical Procedure for Genome-Wide Detection of QTL Hotspots Using Public Databases with Application to Rice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Man-Hsia Yang</span>, <span class="highwire-citation-author" data-delta="1">Dong-Hong Wu</span> and <span class="highwire-citation-author" data-delta="2">Chen-Hung Kao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">439-452;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200922 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492213--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Man-Hsia Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, National Taiwan University, Taipei 10617, Taiwan, Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Crop Science Division, Taiwan Agricultural Research Institute, Council of Agriculture, Taichung 41362, Taiwan, Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Man-Hsia%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMan-Hsia%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dong-Hong Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Crop Science Division, Taiwan Agricultural Research Institute, Council of Agriculture, Taichung 41362, Taiwan, Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dong-Hong%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADong-Hong%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chen-Hung Kao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, National Taiwan University, Taipei 10617, Taiwan, Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Institute of Statistical Science, Academia Sinica, Taipei 11529, Taiwan, Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chen-Hung%2BKao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kao%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChen-Hung%2BKao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chkao@stat.sinica.edu.tw" class="" data-icon-position="" data-hide-link-title="0">chkao@stat.sinica.edu.tw</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=439&amp;rft.epage=452&amp;rft.atitle=A%20Statistical%20Procedure%20for%20Genome-Wide%20Detection%20of%20QTL%20Hotspots%20Using%20Public%20Databases%20with%20Application%20to%20Rice&amp;rft.volume=9&amp;rft.issue=2&amp;rft.date=2019-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yang%2C+Man-Hsia&amp;rft.au=Wu%2C+Dong-Hong&amp;rft.au=Kao%2C+Chen-Hung"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492213/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492213/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492213/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492213/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492213/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492213/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492213/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492213/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492213/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492213/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492213/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492213/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/2/439" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/2/439.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/2/439.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/2/439.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/2/439.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
