<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/2/375" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/2/375.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/2/375.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/2/375.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/2/375/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/2/375.atom" />
<meta name="HW.pisa" content="ggg;9/2/375" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Very Oil Yellow1 Modifier of the Oil Yellow1-N1989 Allele Uncovers a Cryptic Phenotypic Impact of Cis-regulatory Variation in Maize" />
<meta name="DC.Identifier" content="10.1534/g3.118.200798" />
<meta name="DC.Date" content="2019-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Khangura et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7370948" />
<meta name="DC.Description" content="Forward genetics determines the function of genes underlying trait variation by identifying the change in DNA responsible for changes in phenotype. Detecting phenotypically-relevant variation outside protein coding sequences and distinguishing this from neutral variants is not trivial; partly because the mechanisms by which DNA polymorphisms in the intergenic regions affect gene regulation are poorly understood. Here we utilized a dominant genetic reporter to investigate the effect of cis and trans -acting regulatory variation. We performed a forward genetic screen for natural variation that suppressed or enhanced the semi-dominant mutant allele Oy1-N1989 , encoding the magnesium chelatase subunit I of maize. This mutant permits rapid phenotyping of leaf color as a reporter for chlorophyll accumulation, and mapping of natural variation in maize affecting chlorophyll metabolism. We identified a single modifier locus segregating between B73 and Mo17 that was linked to the reporter gene itself, which we call very oil yellow1 ( vey1 ). Based on the variation in OY1 transcript abundance and genome-wide association data, vey1 is predicted to consist of multiple cis -acting regulatory sequence polymorphisms encoded at the wild-type oy1 alleles. The vey1 locus appears to be a common polymorphism in the maize germplasm that alters the expression level of a key gene in chlorophyll biosynthesis. These vey1 alleles have no discernable impact on leaf chlorophyll in the absence of the Oy1-N1989 reporter. Thus, the use of a mutant as a reporter for magnesium chelatase activity resulted in the detection of expression-level polymorphisms not readily visible in the laboratory." />
<meta name="DC.Contributor" content="Rajdeep S. Khangura" />
<meta name="DC.Contributor" content="Sandeep Marla" />
<meta name="DC.Contributor" content="Bala P. Venkata" />
<meta name="DC.Contributor" content="Nicholas J. Heller" />
<meta name="DC.Contributor" content="Gurmukh S. Johal" />
<meta name="DC.Contributor" content="Brian P. Dilkes" />
<meta name="article:published_time" content="2019-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Very Oil Yellow1 Modifier of the Oil Yellow1-N1989 Allele Uncovers a Cryptic Phenotypic Impact of Cis-regulatory Variation in Maize" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Forward genetics determines the function of genes underlying trait variation by identifying the change in DNA responsible for changes in phenotype. Detecting phenotypically-relevant variation outside protein coding sequences and distinguishing this from neutral variants is not trivial; partly because the mechanisms by which DNA polymorphisms in the intergenic regions affect gene regulation are poorly understood. Here we utilized a dominant genetic reporter to investigate the effect of cis and &lt;i&gt;trans&lt;/i&gt;-acting regulatory variation. We performed a forward genetic screen for natural variation that suppressed or enhanced the semi-dominant mutant allele &lt;i&gt;Oy1-N1989&lt;/i&gt;, encoding the magnesium chelatase subunit I of maize. This mutant permits rapid phenotyping of leaf color as a reporter for chlorophyll accumulation, and mapping of natural variation in maize affecting chlorophyll metabolism. We identified a single modifier locus segregating between B73 and Mo17 that was linked to the reporter gene itself, which we call &lt;i&gt;very oil yellow1&lt;/i&gt; (&lt;i&gt;vey1&lt;/i&gt;). Based on the variation in OY1 transcript abundance and genome-wide association data, &lt;i&gt;vey1&lt;/i&gt; is predicted to consist of multiple &lt;i&gt;cis&lt;/i&gt;-acting regulatory sequence polymorphisms encoded at the wild-type &lt;i&gt;oy1&lt;/i&gt; alleles. The &lt;i&gt;vey1&lt;/i&gt; locus appears to be a common polymorphism in the maize germplasm that alters the expression level of a key gene in chlorophyll biosynthesis. These &lt;i&gt;vey1&lt;/i&gt; alleles have no discernable impact on leaf chlorophyll in the absence of the &lt;i&gt;Oy1-N1989&lt;/i&gt; reporter. Thus, the use of a mutant as a reporter for magnesium chelatase activity resulted in the detection of expression-level polymorphisms not readily visible in the laboratory.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/02/01" />
<meta name="citation_mjid" content="ggg;9/2/375" />
<meta name="citation_id" content="9/2/375" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/2/375" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/2/375.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/2/375.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/2/375.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200798" />
<meta name="citation_pmid" content="30518539" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="375" />
<meta name="citation_lastpage" content="390" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Rajdeep S. Khangura" />
<meta name="citation_author_institution" content="Department of Botany and Plant Pathology, Purdue University, IN 47907" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5085-6240" />
<meta name="citation_author" content="Sandeep Marla" />
<meta name="citation_author_institution" content="Department of Botany and Plant Pathology, Purdue University, IN 47907" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5778-7850" />
<meta name="citation_author" content="Bala P. Venkata" />
<meta name="citation_author_institution" content="Department of Botany and Plant Pathology, Purdue University, IN 47907" />
<meta name="citation_author" content="Nicholas J. Heller" />
<meta name="citation_author_institution" content="Department of Botany and Plant Pathology, Purdue University, IN 47907" />
<meta name="citation_author" content="Gurmukh S. Johal" />
<meta name="citation_author_institution" content="Department of Botany and Plant Pathology, Purdue University, IN 47907" />
<meta name="citation_author_institution" content="Center for Plant Biology, Purdue University, IN 47907" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8101-8410" />
<meta name="citation_author" content="Brian P. Dilkes" />
<meta name="citation_author_institution" content="Center for Plant Biology, Purdue University, IN 47907" />
<meta name="citation_author_institution" content="Department of Biochemistry, Purdue University, IN 47907" />
<meta name="citation_author_email" content="bdilkes@purdue.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2799-954X" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. Adrian;citation_author=S. Farrona;citation_author=J. J. Reimer;citation_author=M. C. Albani;citation_author=G. Coupland;citation_title=Cis-Regulatory Elements and Chromatin State Coordinately Control Temporal and Spatial Expression of FLOWERING LOCUS T in Arabidopsis.;citation_volume=22;citation_year=2010;citation_pmid=20472817;citation_doi=10.1105/tpc.110.074682" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. M. Bates;citation_author=M. Machler;citation_author=B. M. Bolker;citation_author=S. C. Walker;citation_title=Fitting Linear Mixed-Effects Models using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2015;citation_doi=10.18637/jss.v067.i01" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.;citation_pages=289-300;citation_volume=57;citation_year=1995;citation_pmid=16632515;citation_doi=10.1093/biostatistics/kxj037" />
<meta name="citation_reference" content="citation_journal_title=Innov. Food Sci. Emerg. Technol.;citation_author=N. Berardo;citation_author=O. V. Brenna;citation_author=A. Amato;citation_author=P. Valoti;citation_author=V. Pisacane;citation_title=Carotenoids concentration among maize genotypes measured by near infrared reflectance spectroscopy (NIRS).;citation_pages=393-398;citation_volume=5;citation_year=2004;citation_doi=10.1016/j.ifset.2004.03.001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. A. Birchler;citation_author=K. J. Newton;citation_title=MODULATION OF PROTEIN LEVELS IN CHROMOSOMAL DOSAGE SERIES OF MAIZE: THE BIOCHEMICAL BASIS OF ANEUPLOID SYNDROMES;citation_pages=247-266;citation_volume=99;citation_year=1981;citation_issue=2;citation_pmid=17249116" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. A. Birchler;citation_author=R. A. Veitia;citation_title=Gene balance hypothesis: Connecting issues of dosage sensitivity across biological disciplines.;citation_volume=109;citation_year=2012;citation_pmid=22908297;citation_doi=10.1073/pnas.1207726109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=R. B. Brem;citation_author=G. Yvert;citation_author=R. Clinton;citation_author=L. Kruglyak;citation_title=Genetic dissection of transcriptional regulation in budding yeast.;citation_volume=296;citation_year=2002;citation_pmid=11923494;citation_doi=10.1126/science.1069516" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=E. M. Buescher;citation_author=J. Moon;citation_author=A. Runkel;citation_author=S. Hake;citation_author=B. P. Dilkes;citation_title=Natural Variation at sympathy for the ligule Controls Penetrance of the Semidominant Liguleless narrow-R Mutation in Zea mays.;citation_volume=4;citation_year=2014;citation_pmid=25344411;citation_doi=10.1534/g3.114.014183" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Bukowski;citation_author=X. Guo;citation_author=Y. Lu;citation_author=C. Zou;citation_author=B. He;citation_title=Construction of the third-generation Zea mays haplotype map.;citation_pages=1-12;citation_volume=7;citation_year=2018;citation_pmid=29300887;citation_doi=10.1093/gigascience/gix120" />
<meta name="citation_reference" content="Bushnell, B., 2014 BBMap: A Fast, Accurate, Splice-Aware Aligner. Technical report." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Chintamanani;citation_author=S. H. Hulbert;citation_author=G. S. Johal;citation_author=P. J. Balint-Kurti;citation_title=Identification of a maize locus that modulates the hypersensitive defense response, using mutant-assisted gene identification and characterization.;citation_volume=184;citation_year=2010;citation_pmid=20176981;citation_doi=10.1534/genetics.109.111880" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=C. A. Christensen;citation_author=S. W. Gorsich;citation_author=R. H. Brown;citation_author=L. G. Jones;citation_author=J. Brown;citation_title=Mitochondrial GFA2 Is Required for Synergid Cell Death in Arabidopsis;citation_pages=2215-2232;citation_volume=14;citation_year=2002;citation_issue=9;citation_pmid=12215516;citation_doi=10.1105/tpc.002170" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=C. A. Christensen;citation_author=S. Subramanian;citation_author=G. N. Drews;citation_title=Identification of gametophytic mutations affecting female gametophyte development in Arabidopsis.;citation_pages=136-151;citation_volume=202;citation_year=1998;citation_issue=1;citation_pmid=9758709;citation_doi=10.1006/dbio.1998.8980" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. M. Clark;citation_author=T. N. Wagler;citation_author=P. Quijada;citation_author=J. Doebley;citation_title=A distant upstream enhancer at the maize domestication gene tb1 has pleiotropic effects on plant and inflorescent architecture.;citation_pages=594-597;citation_volume=38;citation_year=2006;citation_issue=5;citation_pmid=16642024;citation_doi=10.1038/ng1784" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=B. P. Dilkes;citation_author=M. Spielman;citation_author=R. Weizbauer;citation_author=B. Watson;citation_author=D. Burkart-Waco;citation_title=The maternally expressed WRKY transcription factor TTG2 controls lethality in interploidy crosses of Arabidopsis.;citation_pages=2707-2720;citation_volume=6;citation_year=2008;citation_issue=12;citation_pmid=19071961;citation_doi=10.1371/journal.pbio.0060308" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Report.;citation_author=K. Eggermont;citation_author=I. J. Goderis;citation_author=W. F. Broekaert;citation_title=High-throughput RNA Extraction from Plant Samples Based on Homogenisation by Reciprocal Shaking in the Presence of a Mixture of Sand and Glass Beads.;citation_pages=273-279;citation_volume=14;citation_year=1996;citation_doi=10.1007/BF02671663" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=S. R. Eichten;citation_author=J. M. Foerster;citation_author=N. de Leon;citation_author=Y. Kai;citation_author=C.-T. Yeh;citation_title=B73-Mo17 near-isogenic lines demonstrate dispersed structural variation in maize.;citation_volume=156;citation_year=2011;citation_pmid=21705654;citation_doi=10.1104/pp.111.174748" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Metron;citation_author=R. Fisher;citation_title=On the ”probable error” of a coefficient of correlation deduced from a small sample.;citation_pages=3-32;citation_volume=1;citation_year=1921" />
<meta name="citation_reference" content="citation_title=The genetical theory of natural selection;citation_author=R. A. Fisher;citation_year=1930;citation_doi=10.5962/bhl.title.27468" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. Flint-Garcia;citation_author=A.-C. Thuillet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=S. M. Romero;citation_title=Maize association population: a high-resolution platform for quantitative trait locus dissection.;citation_pages=1054-1064;citation_volume=44;citation_year=2005;citation_issue=6;citation_pmid=16359397;citation_doi=10.1111/j.1365-313X.2005.02591.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. N. Fodje;citation_author=A. Hansson;citation_author=M. Hansson;citation_author=J. G. Olsen;citation_author=S. Gough;citation_title=Interplay between an AAA module and an integrin I domain may regulate the function of magnesium chelatase.;citation_pages=111-122;citation_volume=311;citation_year=2001;citation_issue=1;citation_pmid=11469861;citation_doi=10.1006/jmbi.2001.4834" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=G. Gibson;citation_author=B. Weir;citation_title=The quantitative genetics of transcription.;citation_pages=616-623;citation_volume=21;citation_year=2005;citation_issue=11;citation_pmid=16154229;citation_doi=10.1016/j.tig.2005.08.010" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Gray;citation_author=P. S. Close;citation_author=S. P. Briggs;citation_author=G. S. Johal;citation_title=A novel suppressor of cell death in plants encoded by the Lls1 gene of maize.;citation_pages=25-31;citation_volume=89;citation_year=1997;citation_issue=1;citation_pmid=9094711;citation_doi=10.1016/S0092-8674(00)80179-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. Gray;citation_author=D. Janick-Buckner;citation_author=B. Buckner;citation_author=P. S. Close;citation_author=G. S. Johal;citation_title=Light-dependent death of maize lls1 cells is mediated by mature chloroplasts.;citation_volume=130;citation_year=2002;citation_pmid=12481072;citation_doi=10.1104/pp.008441" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=U. Grossniklaus;citation_author=M. S. Madhusudhan;citation_author=V. Nanjundiah;citation_title=Nonlinear enzyme kinetics can lead to high metabolic flux control coefficients: implications for the evolution of dominance.;citation_pages=299-302;citation_volume=182;citation_year=1996;citation_issue=3;citation_pmid=8944161;citation_doi=10.1006/jtbi.1996.0167" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Hansson;citation_author=C. G. Kannangara;citation_author=D. V. Wettstein;citation_author=M. Hansson;citation_title=Molecular basis for semidominance of missense mutations in the XANTHA-H (42-kDa) subunit of magnesium chelatase;citation_pages=1744-1749;citation_volume=96;citation_year=1999;citation_issue=4;citation_pmid=9990095;citation_doi=10.1073/pnas.96.4.1744" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Hansson;citation_author=R. D. Willows;citation_author=T. H. Roberts;citation_author=M. Hansson;citation_title=Three semidominant barley mutants with single amino acid substitutions in the smallest magnesium chelatase subunit form defective AAA+ hexamers.;citation_volume=99;citation_year=2002;citation_pmid=12357035;citation_doi=10.1073/pnas.212504499" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. E. Harjes;citation_author=T. R. Rocheford;citation_author=L. Bai;citation_author=T. P. Brutnell;citation_author=C. B. Kandianis;citation_title=Natural Genetic Variation in Lycopene Epsilon Cyclase Tapped for Maize Biofortification;citation_pages=330-333;citation_volume=319;citation_year=2008;citation_issue=5861;citation_pmid=18202289;citation_doi=10.1126/science.1150255" />
<meta name="citation_reference" content="citation_journal_title=Scand. J. Stat.;citation_author=S. Holm;citation_title=A SIMPLE SEQUENTIALLY REJECTIVE MULTIPLE TEST PROCEDURE.;citation_pages=65-70;citation_volume=6;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=G. Hu;citation_author=N. Yalpani;citation_author=S. P. Briggs;citation_author=G. S. Johal;citation_title=A Porphyrin Pathway Impairment Is Responsible for the Phenotype of a Dominant Disease Lesion Mimic Mutant of Maize;citation_pages=1095-1106;citation_volume=10;citation_year=1998;citation_issue=7;citation_pmid=9668130;citation_doi=10.1105/tpc.10.7.1095" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=M. Huang;citation_author=T. L. Slewinski;citation_author=R. F. Baker;citation_author=D. Janick-Buckner;citation_author=B. Buckner;citation_title=Camouflage patterning in maize leaves results from a defect in porphobilinogen deaminase.;citation_volume=2;citation_year=2009;citation_pmid=19825655;citation_doi=10.1093/mp/ssp029" />
<meta name="citation_reference" content="citation_journal_title=Journal of Plant Registrations;citation_journal_abbrev=Journal of Plant Registrations;citation_author=T. Hussain;citation_author=P. Tausend;citation_author=G. Graham;citation_author=J. Ho;citation_title=Registration of IBM2 SYN10 Doubled Haploid Mapping Population of Maize;citation_pages=81-81;citation_volume=1;citation_year=2007;citation_issue=1;citation_doi=10.3198/jpr2005.11.0414crs" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=F. Jacob;citation_author=J. Monod;citation_title=Genetic regulatory mechanisms in the synthesis of proteins.;citation_pages=318-356;citation_volume=3;citation_year=1961;citation_pmid=13718526;citation_doi=10.1016/S0022-2836(61)80072-7" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=G. S. Johal;citation_author=P. Balint-Kurti;citation_author=C. F. Weil;citation_title=Mining and Harnessing Natural Variation: A Little MAGIC;citation_pages=2066-2073;citation_volume=48;citation_year=2008;citation_issue=6;citation_doi=10.2135/cropsci2008.03.0150" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M.-C. King;citation_author=A. C. Wilson;citation_title=Evolution at two levels in humans and chimpanzees;citation_pages=107-116;citation_volume=188;citation_year=1975;citation_issue=4184;citation_pmid=1090005;citation_doi=10.1126/science.1090005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=K. A. G. Kremling;citation_author=S.-Y. Chen;citation_author=M.-H. Su;citation_author=N. K. Lepak;citation_author=M. C. Romay;citation_title=Dysregulation of expression correlates with rare-allele burden and fitness loss in maize.;citation_pages=520-523;citation_volume=555;citation_year=2018;citation_doi=10.1038/nature25966" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=A. N. M. Lauter;citation_author=M. G. Muszynski;citation_author=R. D. Huffman;citation_author=M. P. Scott;citation_title=A Pectin Metethylesterase ZmPme3 Is Expressed in Gametophyte factor1-s (Ga1-s) Silks and Maps to that Locus in Maize (Zea mays L.).;citation_pages=1-11;citation_volume=8;citation_year=2017;citation_pmid=28220127;citation_doi=10.3389/fpls.2017.00023" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=M. Lee;citation_author=N. Sharopova;citation_author=W. D. Beavis;citation_author=D. Grant;citation_author=M. Katt;citation_title=Expanding the genetic map of maize with the intermated B73 x Mo17 (IBM) population.;citation_pages=453-461;citation_volume=48;citation_year=2002;citation_issue=5-6;citation_pmid=11999829;citation_doi=10.1023/A:1014893521186" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Z. H. Lemmon;citation_author=R. Bukowski;citation_author=Q. Sun;citation_author=J. F. Doebley;citation_title=The Role of cis Regulatory Evolution in Maize Domestication.;citation_pages=e1004745;citation_volume=10;citation_year=2014;citation_pmid=25375861;citation_doi=10.1371/journal.pgen.1004745" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. Li;citation_author=K. Petsch;citation_author=R. Shimizu;citation_author=S. Liu;citation_author=W. W. Xu;citation_title=Correction: Mendelian and Non-Mendelian Regulation of Gene Expression in Maize.;citation_pages=e1007234;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pgen.1007234" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. Li;citation_author=K. Petsch;citation_author=R. Shimizu;citation_author=S. Liu;citation_author=W. W. Xu;citation_title=Mendelian and non-Mendelian regulation of gene expression in maize.;citation_pages=e1003202;citation_volume=9;citation_year=2013;citation_pmid=23341782;citation_doi=10.1371/journal.pgen.1003202" />
<meta name="citation_reference" content="citation_title=Curr. Protoc. Food Anal. Chem.;citation_author=H. K. Lichtenthaler;citation_author=C. Buschmann;citation_title=Chlorophylls and Carotenoids: measurement and characterization by UV-VIS spectroscopy.;citation_pages=F431-F4.3.8;citation_volume=55;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Y. Lin;citation_author=F. R. Allaire;citation_title=Heritability of a linear combination of traits.;citation_pages=1-3;citation_volume=51;citation_year=1977;citation_doi=10.1007/BF00306054" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. E. Lipka;citation_author=F. Tian;citation_author=Q. Wang;citation_author=J. Peiffer;citation_author=M. Li;citation_title=GAPIT: genome association and prediction integrated tool.;citation_volume=28;citation_year=2012;citation_pmid=22796960;citation_doi=10.1093/bioinformatics/bts444" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H. Liu;citation_author=Y. Niu;citation_author=P. J. Gonzalez-Portilla;citation_author=H. Zhou;citation_author=L. Wang;citation_title=An ultra-high-density map as a community resource for discerning the genetic basis of quantitative traits in maize.;citation_pages=1078;citation_volume=16;citation_year=2015;citation_pmid=26691201;citation_doi=10.1186/s12864-015-2242-5" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=J. Lundqvist;citation_author=I. Braumann;citation_author=M. Kurowska;citation_author=A. H. Müller;citation_author=M. Hansson;citation_title=Catalytic turnover triggers exchange of subunits of the magnesium chelatase AAA+ motor unit.;citation_volume=288;citation_year=2013;citation_pmid=23836887;citation_doi=10.1074/jbc.M113.480012" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. M. Mach;citation_author=A. R. Castillo;citation_author=R. Hoogstraten;citation_author=J. T. Greenberg;citation_title=The Arabidopsis-accelerated cell death gene ACD2 encodes red chlorophyll catabolite reductase and suppresses the spread of disease symptoms;citation_pages=771-776;citation_volume=98;citation_year=2001;citation_issue=2;citation_pmid=11149948;citation_doi=10.1073/pnas.98.2.771" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. Mangelsdorf;citation_author=D. Jones;citation_title=THE EXPRESSION OF MENDELIAN FACTORS IN THE GAMETOPHYTE OF MAIZE;citation_pages=423-455;citation_volume=11;citation_year=1926;citation_issue=5;citation_pmid=17246465" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. McClintock;citation_title=The origin and behavior of mutable loci in maize;citation_pages=344-355;citation_volume=36;citation_year=1950;citation_issue=6;citation_pmid=15430309;citation_doi=10.1073/pnas.36.6.344" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=B. McClintock;citation_title=Controlling Elements and the Gene;citation_pages=197-216;citation_volume=21;citation_year=1956;citation_issue=0;citation_pmid=13433592;citation_doi=10.1101/SQB.1956.021.01.017" />
<meta name="citation_reference" content="citation_journal_title=Brookhaven Symp. Biol.;citation_author=B. McClintock;citation_title=Intranuclear systems controlling gene action and mutation.;citation_pages=58-74;citation_volume=8;citation_year=1956;citation_pmid=13293421" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=B. McClintock;citation_title=Some parallels between gene control systems in maize and in bacteria.;citation_pages=265-277;citation_volume=95;citation_year=1961;citation_doi=10.1086/282188" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Meskauskiene;citation_author=M. Nater;citation_author=D. Goslings;citation_author=F. Kessler;citation_author=R. op den Camp;citation_title=FLU: A negative regulator of chlorophyll biosynthesis in Arabidopsis thaliana.;citation_volume=98;citation_year=2001;citation_pmid=11606728;citation_doi=10.1073/pnas.221252798" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. M. Neff;citation_author=E. Turk;citation_author=M. Kalishman;citation_title=Web-based primer design for single nucleotide polymorphism analysis.;citation_pages=613-615;citation_volume=18;citation_year=2002;citation_issue=12;citation_pmid=12446140;citation_doi=10.1016/S0168-9525(02)02820-2" />
<meta name="citation_reference" content="citation_journal_title=J. Hum. Evol.;citation_author=S. Ohno;citation_title=An argument for the genetic simplicity of man and other mammals.;citation_pages=651-662;citation_volume=1;citation_year=1972;citation_doi=10.1016/0047-2484(72)90011-5" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=B. A. Olukolu;citation_author=Y. Bian;citation_author=B. De Vries;citation_author=W. F. Tracy;citation_author=R. J. Wisser;citation_title=The Genetics of Leaf Flecking in Maize and Its Relationship to Plant Defense and Disease Resistance.;citation_volume=172;citation_year=2016;citation_pmid=27670817;citation_doi=10.1104/pp.15.01870" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. A. Olukolu;citation_author=A. Negeri;citation_author=R. Dhawan;citation_author=B. P. Venkata;citation_author=P. Sharma;citation_title=A connected set of genes associated with programmed cell death implicated in controlling the hypersensitive response in maize.;citation_volume=193;citation_year=2013;citation_pmid=23222653;citation_doi=10.1534/genetics.112.147595" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. A. Olukolu;citation_author=G.-F. Wang;citation_author=V. Vontimitta;citation_author=B. P. Venkata;citation_author=S. Marla;citation_title=A genome-wide association study of the maize hypersensitive defense response identifies genes that cluster in related pathways.;citation_pages=e1004562;citation_volume=10;citation_year=2014;citation_pmid=25166276;citation_doi=10.1371/journal.pgen.1004562" />
<meta name="citation_reference" content="citation_journal_title=Evolution (N. Y.);citation_author=H. A. Orr;citation_title=The Population Genetics of Adaptation: The Distribution of Factors Fixed during Adaptive Evolution.;citation_pages=935-949;citation_volume=52;citation_year=1998;citation_doi=10.1111/j.1558-5646.1998.tb01823.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. A. Orr;citation_title=The genetic theory of adaptation: a brief history.;citation_pages=119-127;citation_volume=6;citation_year=2005;citation_issue=2;citation_pmid=15716908;citation_doi=10.1038/nrg1523" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=B. W. Penning;citation_author=G. S. Johal;citation_author=M. D. Mcmullen;citation_title=A major suppressor of cell death, slm1, modifies the expression of the maize (Zea mays L.) lesion mimic mutation les23.;citation_pages=961-969;citation_volume=47;citation_year=2004;citation_issue=5;citation_pmid=15499410;citation_doi=10.1139/g04-046" />
<meta name="citation_reference" content="Peterson, P. A., 1953 A study of mutable pale green locus in maize. Dissertation, University of Illinois." />
<meta name="citation_reference" content="citation_journal_title=Functional &amp; integrative genomics;citation_journal_abbrev=Funct Integr Genomics;citation_author=A. Pribat;citation_author=I. K. Blaby;citation_author=A. Lara-Núñez;citation_author=L. Jeanguenin;citation_author=R. Fouquet;citation_title=A 5-formyltetrahydrofolate cycloligase paralog from all domains of life: comparative genomic and experimental evidence for a cryptic role in thiamin metabolism.;citation_pages=467-478;citation_volume=11;citation_year=2011;citation_issue=3;citation_pmid=21538139;citation_doi=10.1007/s10142-011-0224-5" />
<meta name="citation_reference" content="R Core, T., 2014 R: A language and environment for statistical computing. r foundation for statistical computing, vienna, austria. 2013." />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdóttir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. C. Romay;citation_author=M. J. Millard;citation_author=J. C. Glaubitz;citation_author=J. Peiffer;citation_author=K. L. Swarts;citation_title=Comprehensive genotyping of the USA national maize inbred seed bank.;citation_pages=R55;citation_volume=14;citation_year=2013;citation_pmid=23759205;citation_doi=10.1186/gb-2013-14-6-r55" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Salvi;citation_author=G. Sponza;citation_author=M. Morgante;citation_author=D. Tomes;citation_author=X. Niu;citation_title=Conserved noncoding genomic sequences associated with a flowering-time quantitative trait locus in maize.;citation_volume=104;citation_year=2007;citation_pmid=17595297;citation_doi=10.1073/pnas.0704145104" />
<meta name="citation_reference" content="citation_journal_title=Annals of Botany;citation_journal_abbrev=Ann Bot;citation_author=P. SanMiguel;citation_author=J. L. Bennetzen;citation_title=Evidence that a Recent Increase in Maize Genome Size was Caused by the Massive Amplification of Intergene Retrotransposons;citation_pages=37-44;citation_volume=82;citation_year=1998;citation_issue=suppl_1;citation_doi=10.1006/anbo.1998.0746" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. SanMiguel;citation_author=A. Tikhonov;citation_author=Y.-K. Jin;citation_author=N. Motchoulskaia;citation_author=D. Zakharov;citation_title=Nested Retrotransposons in the Intergenic Regions of the Maize Genome;citation_pages=765-768;citation_volume=274;citation_year=1996;citation_issue=5288;citation_pmid=8864112;citation_doi=10.1126/science.274.5288.765" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=R. J. H. Sawers;citation_author=J. Viney;citation_author=P. R. Farmer;citation_author=R. R. Bussey;citation_author=G. Olsefski;citation_title=The maize Oil yellow1 (Oy1) gene encodes the I subunit of magnesium chelatase.;citation_pages=95-106;citation_volume=60;citation_year=2006;citation_issue=1;citation_pmid=16463102;citation_doi=10.1007/s11103-005-2880-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Schwartz;citation_author=S. Balasubramanian;citation_author=N. Warthmann;citation_author=T. P. Michael;citation_author=J. Lempe;citation_title=Cis-regulatory changes at Flowering Locus T mediate natural variation in flowering responses of Arabidopsis thaliana.;citation_volume=183;citation_year=2009;citation_pmid=19652183;citation_doi=10.1534/genetics.109.104984" />
<meta name="citation_reference" content="citation_journal_title=Database (Oxford);citation_author=T. Z. Sen;citation_author=L. C. Harper;citation_author=M. L. Schaeffer;citation_author=C. M. Andorf;citation_author=T. E. Seigfried;citation_title=Choosing a genome browser for a Model Organism Database: surveying the Maize community.;citation_pages=1-9;citation_volume=2010;citation_year=2010;citation_doi=10.1093/database/baq007" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Sindhu;citation_author=D. Janick-Buckner;citation_author=B. Buckner;citation_author=J. Gray;citation_author=U. Zehr;citation_title=Propagation of cell death in dropdead1, a sorghum ortholog of the maize lls1 mutant.;citation_pages=e0201359;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0201359" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Studer;citation_author=Q. Zhao;citation_author=J. Ross-Ibarra;citation_author=J. Doebley;citation_title=Identification of a functional transposon insertion in the maize domestication gene tb1.;citation_pages=1160-1163;citation_volume=43;citation_year=2011;citation_issue=11;citation_pmid=21946354;citation_doi=10.1038/ng.942" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res. Acids Res;citation_author=J. D. Thompson;citation_author=D. G. Higgins;citation_author=T. J. Gibson;citation_title=ClustalW: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position specific gap penalties and weight matrix choice.;citation_volume=22;citation_year=1994;citation_pmid=7984417;citation_doi=10.1093/nar/22.22.4673" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=R. D. Vale;citation_title=AAA proteins. Lords of the ring.;citation_volume=150;citation_year=2000;citation_pmid=10893253;citation_doi=10.1083/jcb.150.1.F13" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=R. A. Veitia;citation_title=Nonlinear effects in macromolecular assembly and dosage sensitivity.;citation_pages=19-25;citation_volume=220;citation_year=2003;citation_issue=1;citation_pmid=12453447;citation_doi=10.1006/jtbi.2003.3105" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=A. J. Waters;citation_author=I. Makarevitch;citation_author=J. Noshay;citation_author=L. T. Burghardt;citation_author=C. N. Hirsch;citation_title=Natural variation for gene expression responses to abiotic stress in maize.;citation_pages=706-717;citation_volume=89;citation_year=2017;citation_doi=10.1111/tpj.13414" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. A. West;citation_author=K. Kim;citation_author=D. J. Kliebenstein;citation_author=H. Van Leeuwen;citation_author=R. W. Michelmore;citation_title=Global eQTL mapping reveals the complex genetic architecture of transcript-level variation in Arabidopsis.;citation_volume=175;citation_year=2007;citation_pmid=17179097;citation_doi=10.1534/genetics.106.064972" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=D. Von Wettstein;citation_author=S. Gough;citation_author=C. G. Kannangara;citation_title=Chlorophyll Biosynthesis;citation_pages=1039-1057;citation_volume=7;citation_year=1995;citation_issue=7;citation_pmid=12242396;citation_doi=10.1105/tpc.7.7.1039" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. C. Willing;citation_author=S. P. Deschenes;citation_author=R. L. Slayton;citation_author=E. J. Roberts;citation_title=Premature chain termination is a unifying mechanism for COL1A1 null alleles in osteogenesis imperfecta type I cell strains.;citation_pages=799-809;citation_volume=59;citation_year=1996;citation_issue=4;citation_pmid=8808594" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=G. A. Wray;citation_author=M. W. Hahn;citation_author=E. Abouheif;citation_author=J. P. Balhoff;citation_author=M. Pizer;citation_title=The evolution of transcriptional regulation in eukaryotes.;citation_volume=20;citation_year=2003;citation_pmid=12777501;citation_doi=10.1093/molbev/msg140" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=M. Yang;citation_author=E. Wardzala;citation_author=G. S. Johal;citation_author=J. Gray;citation_title=The wound-inducible Lls1 gene from maize is an orthologue of the Arabidopsis Acd1 gene, and the LLS1 protein is present in non-photosynthetic tissues.;citation_pages=175-191;citation_volume=54;citation_year=2004;citation_issue=2;citation_pmid=15159621;citation_doi=10.1023/B:PLAN.0000028789.51807.6a" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. Vroh Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Zhang;citation_author=E. Ersoz;citation_author=C.-Q. Lai;citation_author=R. J. Todhunter;citation_author=H. K. Tiwari;citation_title=Mixed linear model approach adapted for genome-wide association studies.;citation_pages=355-360;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208535;citation_doi=10.1038/ng.546" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Very Oil Yellow1 Modifier of the Oil Yellow1-N1989 Allele Uncovers a Cryptic Phenotypic Impact of Cis-regulatory Variation in Maize" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
<meta name="twitter:description" content="Forward genetics determines the function of genes underlying trait variation by identifying the change in DNA responsible for changes in phenotype. Detecting phenotypically-relevant variation outside protein coding sequences and distinguishing this from neutral variants is not trivial; partly because the mechanisms by which DNA polymorphisms in the intergenic regions affect gene regulation are poorly understood. Here we utilized a dominant genetic reporter to investigate the effect of cis and trans -acting regulatory variation. We performed a forward genetic screen for natural variation that suppressed or enhanced the semi-dominant mutant allele Oy1-N1989 , encoding the magnesium chelatase subunit I of maize. This mutant permits rapid phenotyping of leaf color as a reporter for chlorophyll accumulation, and mapping of natural variation in maize affecting chlorophyll metabolism. We identified a single modifier locus segregating between B73 and Mo17 that was linked to the reporter gene itself, which we call very oil yellow1 ( vey1 ). Based on the variation in OY1 transcript abundance and genome-wide association data, vey1 is predicted to consist of multiple cis -acting regulatory sequence polymorphisms encoded at the wild-type oy1 alleles. The vey1 locus appears to be a common polymorphism in the maize germplasm that alters the expression level of a key gene in chlorophyll biosynthesis. These vey1 alleles have no discernable impact on leaf chlorophyll in the absence of the Oy1-N1989 reporter. Thus, the use of a mutant as a reporter for magnesium chelatase activity resulted in the detection of expression-level polymorphisms not readily visible in the laboratory." />
<meta name="og-title" property="og:title" content="A Very Oil Yellow1 Modifier of the Oil Yellow1-N1989 Allele Uncovers a Cryptic Phenotypic Impact of Cis-regulatory Variation in Maize" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/2/375" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Forward genetics determines the function of genes underlying trait variation by identifying the change in DNA responsible for changes in phenotype. Detecting phenotypically-relevant variation outside protein coding sequences and distinguishing this from neutral variants is not trivial; partly because the mechanisms by which DNA polymorphisms in the intergenic regions affect gene regulation are poorly understood. Here we utilized a dominant genetic reporter to investigate the effect of cis and trans -acting regulatory variation. We performed a forward genetic screen for natural variation that suppressed or enhanced the semi-dominant mutant allele Oy1-N1989 , encoding the magnesium chelatase subunit I of maize. This mutant permits rapid phenotyping of leaf color as a reporter for chlorophyll accumulation, and mapping of natural variation in maize affecting chlorophyll metabolism. We identified a single modifier locus segregating between B73 and Mo17 that was linked to the reporter gene itself, which we call very oil yellow1 ( vey1 ). Based on the variation in OY1 transcript abundance and genome-wide association data, vey1 is predicted to consist of multiple cis -acting regulatory sequence polymorphisms encoded at the wild-type oy1 alleles. The vey1 locus appears to be a common polymorphism in the maize germplasm that alters the expression level of a key gene in chlorophyll biosynthesis. These vey1 alleles have no discernable impact on leaf chlorophyll in the absence of the Oy1-N1989 reporter. Thus, the use of a mutant as a reporter for magnesium chelatase activity resulted in the detection of expression-level polymorphisms not readily visible in the laboratory." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
    <title>A Very Oil Yellow1 Modifier of the Oil Yellow1-N1989 Allele Uncovers a Cryptic Phenotypic Impact of Cis-regulatory Variation in Maize | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"d7QfQqzLC8QVAjqmG7-yJIbkQxJfbehZ0dRgDe441Ds","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492211","apath":"\/ggg\/9\/2\/375.atom","pisa":"ggg;9\/2\/375","ac":{"\/ggg\/9\/2\/375.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/2\/375.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/2\/375"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/2\/375"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/2\/375":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xem2a3Ue7+eFrN4GArKKOIZD6YGedeAuJREPd2wrQwelwRVTXqk2nljqK8rWBcY52BrdwmKMGWByZgQpFzXlCcEtnGDkJo6xLBNSE0mlAYVgZYjCqO6neyso44MhANmRq6jA0PHfbyZ3+7lVB8aVQ+OWR9oqGimBfLKkCeb5F5wUefWVt5SzG0G94skPftzEqssLDxccSPq2PkRBK5j0YYd02QGPD4AiovfMXT9jFSQdwekDyEu37V5NbNlJgnbYCMc95eNHTehVA0bxk2+XSRBlPEG+s\\\\\\\/9yLR72OrmpFVRBAoI5lw5d\\\\\\\/Isl3eZb58hXxouRiiqXS14U+Bsngyxe1OJB0f9p\\\\\\\/97zYwFhgKFR2pXeJrd2LJVBY2Z1DmetpQZ3uEiyb2+taSxv50lY2P+ijbHdWS0pNjcf\\\\\\\/LnK8eqH1b7lANUxCsZdPsbUWNoYXs2aONEsFEuOEhrhcoavuCEPBFpUi8YT0hnWYKmAKxSYUqmp1wvNcNz4Py5JhWoetnmQCJ+K8606yz\\\\\\\/5mdqd6JoEgcfxrYhRPnt1PGOz1X9Izy5F2DOXYaypDe9MRVY4Ly+W2ExXK1utAiQssooyt9JMe0bO7QBP0SmVztnUr9Lv0BjuGA7VyWob499FNwPk\\\\\\\/tbodIL4loEnfwG\\\\\\\/drz\\\\\\\/FOvBt80l7t+Xs1rYJolVL2Hmpue8S4MaIX+mZgr5AWIPa4Qy3PrQu8tcn12+QCkAC\\\\\\\/73n\\\\\\\/UencGBA++8dOPE05HrxTLSHdjMOdrJtfLAvpZih2yPBzHqq9hNeKuhk9EqPdAADEKlzMKSvQjc9Rw\\\\\\\/fS4GTYVb+LCXNOHPLODg2kmxLRRqi+ABTB7gnF5g2\\\\\\\/wTXRu+3cLkM6zKZMDbw4eyfZzz24bnf+K9KXV75N7thucgkJA4FYfHEei4o8r5XsuDLemCosMxozOVHglfFuRz\\\\\\\/qYay0kK6Co83WAFBof1Bd9kNdV4mYvXPcggKNLS+sTnI4GPOZzSivefo8EQMTNzbviZVEatwE+IpfsYgdh56\\\\\\\/OkTwuli1mUGsUSzFjxKGV6gkrbpZgmPdsSJXY1KUl2ztKHs4dyySQop0oJL1aUZ9zO4xWxB95phnMI9RwGZXViYca611XTpAmsRhfUZkw+p3OzJnbdnwv2ARvOFJ6UegBTyZsy\\\\\\\/fU1v41FY6S2s\\\\\\\/Q57L1AJiFvyrMXtuOEFKT6q9vaZjPDQ7frd5n70iz1E0RR1JkwocwB4k8Zr\\\\\\\/WHVX9l0quCfGWkjhv9Hwj\\\\\\\/2f\\\\\\\/N5IfaZRaR\\\\\\\/WczdMAacRn\\\\\\\/var2v4Bg\\\u0022,\\\u0022iv\\\u0022:\\\u0022PRB8gRJ5QDVM9rNK4GsNFQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002278baa87f5222c4a076a11ba1c52b7eeb1484316570104cb730cae9aa7e8af578\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227rPGn2sKKRVnG3KHTV1bQ8CF9xPU9xl3I+k68snQAIvVk0pGlPAiKIf2CrNc4g0fJVmGi3DrWeFEgivv0M7ZTK63C50jw2Q4qQUPfMcKWVnfLuQp99mG6AutCqSyZu8rQt6n+Y0Ds6nB4WwJ417PwazqoDwFs6jrLLAyhCRk8j5bJoYVgPeFb+Pw89yddKlN8Y43HMpl3ZRwLAuQjRNNfhuIu+GGWVqiDCncPHiBjAls94brKgwhE7hiMWL8fTy+5hKrd1KFRmRhli9Dh7iG27P\\\\\\\/G3VggunsdGvY1lE9OxJ1DWxXDMdqU3j6olwrpskFzeRzkE0F\\\\\\\/sWtYH7Z8iw7TagV2V3QvCaxBjcXePbHX8PsAThmBxE7bvrVGFY6O0Pzpv4inQY3CvjtaEvTuI7mkTC2BpoOu9dcB1AaT8XYAQg+eZAG9ldgjoIBe67xJ\\\\\\\/Tt4MjikhCbEQ0BYScgpwBb3wZPbNeq\\\\\\\/KTDXv3ToV+NSBsEL8hcPAKzsK7n47dbLD5\\\\\\\/v8bYtBR2DLBa+S6dYspVbWnR5IAkMhKd8davQ8oI0cgwZNAxY4tQxwdj5EvFBtW9M2+YEKjDu+fLEwH6z0UMS5tBeuhVhAcoEoB0J9GYmrdGMSt+eqk28hA8nqL7MFDkyA9LMwY5zA8RdCggWAJO6VzN0+DiCU0Vlfyzixjp6QYY5Nb4BZvFm9et4obpHb6+FaRrkqXlH8WvAnlbGC07vNNbtuGOfKdZxLI+U0K6Rx9WcR29LcaK\\\\\\\/VSq1w1DE\\\\\\\/zL5gwmfQC3t1yDUQAjhIIyY5F6p4lEhM7h1Rg7vE16jdxl5v+LhF71NZDpe+i7Xkdq3n1ZZgZjk5SkNx6t+v0OGS2u\\\\\\\/\\\\\\\/6fBDnfAbqzaLmFEB7tRUSaRvV\\\\\\\/D36vdK\\\\\\\/ZUJG7PkYn1JLOv32pbzfWCv\\\\\\\/UqoN9\\\\\\\/UR8mvENnp9+vT1BTcDkmWFpG4kJ1gyn+Ivl936v2hiLZVQCT2Gqc\\\\\\\/faajB1LVEowc+dO7T6K8vsXiVV7y9WWMZfeQKBuzrvQGGiE9P1pRsu6MewKLmCgRpzt8WSLYJCVpx3UQs0Zd3tDPC3qtVWaOo85c1dr4bH6GVjwRcvazzGv8afSXKiQkTGQfnkz1MTYoo0nkyktWh+Uda0M4fsV8werKb5KuLXgjZBkEm7st1K+H49Y7YEe5kM4zwwdMKLcWaIA29j0W1Sh3\\\\\\\/DNQfq1yzxlQHjqfVcXXVaSvvD55jDnCX0y97riQI6UNEqpiEDgQU61NXyA3sTP5LHhVidNy9OOpeOp\\\\\\\/pQbzwnrhCO49hpn4WCWJAuT\\\\\\\/FwOsV1QYrP09xGl9THqKTukaCiB0VZjIm9AeVIsi9Zav\\\\\\\/3JnaimDyQ5mSp8MjV79LxXRQD\\\\\\\/kaIZkqXeuCvXAd1G8StiliWHEFtgzJcWj7MCHj0lWJambFbLi7aU+7XeGJEzrFv9xHNegozBlNsAr9zAkO5sWfa80m\\\\\\\/9DhKy89633eNpxpalkV0XpHyMMcFwa7Tgg9bAMnuexVHAGpwV07mZ+Ys6Gs8W6obRyruJkZA29HWT40GYAW7UhQBWn3htndq2bfwWXs3vha4AnuRkAwDF0ovTGSZmrGdkrXy46uppPeN+rp4ahe1k96RVeTjJWONh56Dd\\\\\\\/s9S6MKkZyKo0NTwRsxrqjFfOu4Zm6oXP4GSpYECFSH4INz2iypND5s1Bd\\\\\\\/7Tx4eFl3kChydYk78mGPDM3UvG+r1PBX+XQXl1\\\\\\\/Y77jNdgPv3bT4YUMeB\\\\\\\/AxfQ5638ozncFYacwI1bhYa+4rBUNRExHSFz6hNh9jrR3e4YBPXMvaNYFhE4XD3s9su3\\\\\\\/4la0D4uBu7UsD773UJ96cyyn2VdPNA7gLGxMDwNDJTpYKR0GoDPJSJsly6xy\\\\\\\/ilXUy58ay3eWG\\\\\\\/mTsg3s1iD\\\\\\\/V0WSIpWOHfNWlFcY35UnDLpzXHdMvg5FleKssxeNzSMAvHePxI7QulAGQ1bxWwsnTPRbCsrnXXCjGHZkm\\\\\\\/5nPbgHkJ3TgZ5IfKycoUDNCvQh4R289ZDqyLQtOr7SQt6oNNx2XLLrNC8xuxjWLshtk1gX9yTnKqABQ8\\\\\\\/p7N3dwxLT1yuBUjpR\\\\\\\/9DlqYpmO1o7SCI7higbP6eFn61fuGEma\\\\\\\/bfWy\\\\\\\/Of3Hy43fjsAttvqFi07v7r4AOIeRBuF0OLTwrx7MofDr8HdOalF7o3Uny3AnLf1jZ\\\\\\\/0lwQhsVc2ES\\\\\\\/IDI4ufRXu9V7pnwXT3w+pLZRgiFS\\\\\\\/AtLoi73qG+XhPck6GfHM1L8Wc5n40VZ9SR\\\\\\\/HZbnGEMznnEBQFvQMMwVFg43dtT0WgaDohEABbrAeL\\\\\\\/enQmtBlKMkZrbQ9xrhSWLH77SJjFV+YjKTURCUAjxooMh1pl+PV38TUxjQjaFuthQQAgYyQtI+8TB0BKA4asChr5UkMxf+3RXIbaahTVtCQmjqGDViZRXisyHSVgDp9D5fqq9SyqRluVN+BhYNxr7Fxe\\\\\\\/BmdMPw\\\\\\\/WNXUR5wQdjXIcFbuUG5z\\\\\\\/sAuM+gFMXlNmLlljK3fXO0wdW1S0YSRDPiixKWQNlsU33gP2SnHJnxFQsTrUBr6g5HA9kQEGRFXA0zhZHVDKl+JqCKaZQV1l3vIrpFUCIDpvYZ20yu6qdeBShCerhG79D0zpOV2ZVQxqIU0WN2eEjcZ1vtcDceiLV7uj3r9PyEGTG9+MUZRZJCDi0mYQDhxOzz6UZpnNfZeRcajacxu9g6kHesSQWUIjjrhUYc7r5J1hZTq3ngIhKlVsZ3ijeuk57imfiRFNmkPP04j\\\\\\\/XUq8B8fcXSXIv5h+W3RqFVqpTfs4YbM2H\\\\\\\/G6zWYCV40zjJDLOpE2fZx0YdEtfje0U0SDLRhrCMkM9TzjEmc8U2PhWyjp5x2SbaWT6kjWKG0uGJtK1wts8cwZbHn++JErMJ9XIrlx6\\\\\\\/g+Wba3j5yEDPGU2q57hZ3t7DQcSOWK9rV0JDmprw0wUWEwFVoUrvNanqifrIA5IWIfiyeOajdAIbZ6SgbBywf8w3+4dd0+MBjuCpIMhVXjKZ1W3bhLR\\\\\\\/uyubHCPFh9taVnqkZ6ujR\\\\\\\/WlYaNAi8uVpMRulKG+zBlb36LHKnlPt1n5O4YXH\\\\\\\/xWAyAe6qPOjjIL2NGxyuWT3Ic\\\\\\\/9GMPRWPFnMmY4rAxF9B\\\\\\\/GFjhJn3Lv6r\\\\\\\/gHarvT8ITePYwiR0MZAjon67ZhKyCG4hWzGVmn1ZtEKPCy03b7p7nosqhSABt+mHBW81BwsQOJEPSyNSC1WQsSCNI8truT6M\\\\\\\/FnJS72Xv\\\\\\\/mpGLd73SHDhig6J5ggILhM+P0BWMutyqsaRf7uzYJzzVhgDPSuszAwLmX6pWvGgNJFVKTDLBfjF6Rh2Vb84WxR1N26Lw+QB1mah\\\\\\\/u7kTAiK79JHlz9zs\\\\\\\/mTTI60FTIzdLWXOIRllxjQorm+d5+vLPbfcbX3W4gnFXpkXwAUq4duEjZUxogQeK\\\\\\\/DB5il0WLwZG2CIGv\\\\\\\/GbmksEAeycHU\\\\\\\/HCxL32FVcOTXcJbTyCe0w\\\\\\\/ejJ+KLUdRTCffn2oPzqfH3I5qSKIuNZQOK\\\\\\\/amONmlrou0tGh1z\\\\\\\/d5ulf9eMGRZETiwSoBoSRnzPdXtri9hzEalWXreROmgxZHkoRx7m5sup0sjCjrNXtg4Uts3HsImLFqp6+ybC\\\\\\\/k9DvUbmll2AzITKGpAeuf6IOTpPsmljoqgt0KWGrSaocVTsVArDDhTuEPliF1\\\\\\\/y05V5fsgwGIMrjPbIoL+sOOL8mK6VTWc1ghNnRw35yf8skPUsHwpI3dERVfSH\\\\\\\/Qj04Tdss72shHyFLmHRW6\\\\\\\/h2bJdNyv80H2CcwUKvp8+1iz95b9S5khUX3CRPWjW135Iu5Z5quGrl6cElqNio4J3UdbjFe8OnEYXEek9aro3bj4xzs9Arr3sIIJb9c0RgjfXTJxlp0AqBW8JY6rHwGA3Xr13KeVAsLruONg3XvsyNHUvfLi4Dd2cyHl+tzQV5fr\\\\\\\/4vMOgPPEBEJ2j\\\\\\\/2cci2fCQZO42rec5clv2ndMZFcXRs8y0LaUVShCcCT2f8SUi+t00NygjsZGTnYx3\\\\\\\/HI3GESvRNZid+a7YxwYLCIrNhNTmwWt+2qJY1jP6r1BUT7o66sQWAYiuoBrRWPW1WgIIwczCgLPvGi1DWzasp+ASXEMFaw3pC3Npja1yyF6bkHO3gboIQxTCwoIRXvR+QTAYCNZ2O\\\\\\\/lqXUyL99aP0w7xYPBp4TE0pV4rD3PDzv4O7WcvLwUJb2MbCpzM4KZSKiyXdug9f8hMKtYKWzL5udLDSp1i7cndRrGonRpJ0iNvRQj5Exf3LRDfz1ltRFLswVPlb1tvPKhRs5iBFgQt\\\\\\\/cwCUZlnZGWpsZZMD9vvVmSVFVeN6AL3Z0oVbsY00QRI7W4kKsi+OOIe3ssl5AItaCy3jpjMZids8zzV4OKbK9DmP61H8Y2wdVhuV7iYDUIb8SnNW0pqSY7yRrblhhJv\\\\\\\/PnVToZJD31r38rr3ElYYfwwdajTIU8z1vPToI+y1NkTwjzEp23gSAaXZDT2n8VQxj7WPS7JBz73oDQ9upFF8+sXObM\\\\\\\/UQ6Pknvk6XIw95fcxT800RN47NJ8ssKHJ18zZLA7I43LKSO3YQztBVJ4aRuQJVHHRohXGSlk4GSrgDEq9HCyKmakiomxjFd0ZTBNBux8UeTAopISToL\\\\\\\/ai\\\\\\\/0eqJPO3ZzV3E1mZyc0LJAlNgpIRC7HFqMSECAad7KyIf+MPTGJoWA15wHtIZIFEgpwSHMxymcpUUhL07e5fz\\\\\\\/5LxwMt4lgPXx\\\\\\\/aTYTRq4inuCZYfKL9qP7Y\\\\\\\/G1FF2\\\\\\\/r1VZoSHaJjvSBzFIW8CWsKTPa17lBoaTGWijn+fNkxVDjp9F8RNyuu2Bzf+aK9T3uU8ZNaB1Sny6L1AO4nOatRDlMojcMSR6iDGWq0cWBcTWYFRua9Yy5WTC3RtQLr2itYKoCYpVkES+jgcfYbPTBDf0SUcSC2zmvS\\\\\\\/kxPQNPVngMIlcIiEIGdCFRqjPoXU4HRfswcJ0Z\\\\\\\/WJ732pdDzJbQ+vT1+ZFxchTaHkX0FKLWeEg3fSc9cHmkKYi5\\\\\\\/jP+WJi6Pht2wQmezpBvgRV0mjOw\\\\\\\/0zDPHm+G2SEqRuduvDBtAKiLDEc7cLybmV2x8xuHdbwNat+Q9Y90QS\\\\\\\/1\\\\\\\/lUPNrqvslQkUTLGO9Ei5WVCz\\\\\\\/3JULR5V7YN1TrMFUCK\\\\\\\/K\\\\\\\/ojQ7N5eOQPEO7Idge5PLoylq37QPvQCBUd9HFUyV81G72DpUPpPtW+s18M3yD55ijPWO9eHwYkkl2p7GPHTNyk2tE351CKHtCwAQB9i1iVleq0LgXhTQqcfq1GXvC3F\\\\\\\/qT1DK9bjS193Ev3eBEjAKdiesDjic7RsuYmuJt4JWpNeVe1znDG669vztPe88dZODuuddoEkMAV3EsRdkRNNlPOBda+U5mUnc\\\\\\\/pljqye1bvTVcRwuJOO4WZOi8xL3yh7aVev\\\\\\\/Rn0UeGmmuJQ7q9U3X6vMpYKkyNz4VECdo6QidZxJDVC9TP8LD2\\\\\\\/L8MQ8vTGYKi2lvlJrzkUibrnPeI+LKGCeFA1TCSUR3JKSQXItk2EETZXGsUrpTl7OJDF6NL\\\\\\\/loX6hDrWwFHx6R4U8sCe28c7Q7PBy5GVh9kiiGRw72XfvRf8+hvQ7D2xXzmnnW3nPPbg0Ou4ziN8qeKEdzOMeezWQiIbAJBpE\\\\\\\/kZxtSTYiAeHv+1ggLCxVKXsgazhO1TTA0LgDwIEe766omBldPggEfPNxFcibQ9e251h8u24CwsW7sRnBorqWE7cxVhIWYCb+9rxFgkuvrlw6gHDQlwFLo7s435FZJcb+t1tkHvXJ4Sq5ehqdCPTZQxnWsrvi8+DGa6k+96XC0Mie8IcpydW3UNI9yQ0a1K7osWK1bj\\\\\\\/oWOOq7UClFMvW947QBuR8U9oEwFoappILNVWjgvXoDRjHw63WDfaH1VIEkKCfalEwaWJ8PR\\\\\\\/mGI\\\\\\\/\\\\\\\/MPp3E02MBjZ30zkP8M368JhCBjcL2FKaoNj1s9rWcuhp9KYwod1r871zM4Irxl0GvaWHIWoSHoJaQEAPPhGVTdSdUeX2kiYzXb1J7DMZP8DW\\\\\\\/R8iUkKMkt4mjeRQ96rxxdcDgd6LvUGCRcwafUGtrvro\\\\\\\/fUGoEg1wHj++VpbsRQY20KnnWVzDXQPmGzmJhIA6ItUEdErPEv8ROReq98NRoTQqk8qPPLP8QPK2+i59D87estMq\\\\\\\/0VAFrqxt5M+heLg5TIjepE+LPOTggvIcVW\\\\\\\/ADT8CjuEGmAtpofeDclFQI5iTcIgThD6ZMqTPX1sO\\\\\\\/zBjbRZqJ7R2MQBIP6FjvwZj2eNT26DgQu66+Ma1DZfBg6QTLproPzmQkTqhtv7OsvEXR9yHmDg6aeR98NLlHnpl5wk1tQV55BXemrc3o3xuW46PXnKNYMFcb\\\\\\\/II2HGkiV9rt8uobyJsGLh1xnYFw9GSO9XI6Q+7OHUeRpUD8AE1isWHE0x0IXPvZcwhOHnKMGOEvaLra+yIY8QlIzm4d6r5pe155PDxtr\\\\\\\/8JsxJMmyiIZshZ6lfvhBNEDsRYGnpP64IvzwXKYstWNtpESnsYS408LlBBvbjlGbszVddYALveWJGdZ0YhM1FR4coZwM7RtXa5Iubia7kwcRH8hzwgrnD1Y8ZkgTML3z7AK2tfhX9iik3LDHtMcuyV8Z4vxgnz0w777UanTRcrxz2yLvaHGwUdxq9ZS5ViqxZKicbPhFDeti0GSERZrkk07A2N0map2CA5mKUyDcoWqASSWq41sWCBHX8eKso4YaXpyEq0s+8lUeCdsYeuMVospgfCv5POPKmSA4ad98zznsYrQ0v6aOLCcQdPHIdfOiG7RLdsoRTLjLkwmzKUViX9G5xfzj9+ACjwu7eZdGoJGm1o5dc3YHFMiWSrkHcSxpzV\\\\\\\/ZmBgfylk9sPaFJhCradicTV5fWYD0OvBF9W7ZsZcjNF+uvXu7ERrWzNgz6EjCQmR8Pgk+mbXBQBt8AinARFjE4MtDzPTQBXQotjXVn33XaGt6\\\\\\\/kEzRWC4s7+0+iLT30Q6Bu9n87g6e+93o9neOagHDeheI+vH2hQ5G4ys0TKQhnvR13iTvwCSJtpcQPIG1XqRBd26vkFR59PNSFIZ277YXE82eBJbNgIdg8K2X6rowvfPYq9zTPaFmg3owz1AfdnmlXPlkMLzs+EaaNDJqe9TMr9IeZpn0pLA+XIIw3i0T\\\\\\\/fHWf0pWAN6FxY9etGg6aUJrn284iNzv+ybbi34POFHrtssTLK9N4nqbxfbMXwrGBYtcFnS+ILIxlevopyZCS\\\\\\\/SPdwa3cMVN2UR70LbskjPaYuqwVA5oLqlumPS21yAGrWKffZJdE+aUIeJk3kqT2SAoY+h9VHa3Guxe8uBAiFogGiaAX2Uz0tKyguarpJrPHfr\\\\\\\/pmhah+8oyIv+7H0qRGCsLpyx457TPsxkofF7d4XJm8nZe+jcWqm3LwluYEleAAdcDQ3TWV4R+QPQ0+w0harS1+piyzGhMkm8tAQi4wEIVRuXF9ydogBs4nSTCljHpUrM1UbttuG8cwR+GdedQMToXKgbR2qWdQl+bdGxcUKPiOagyTg+2akqXWoyorjO5qNfEjoF0utMc0oKEU6ksaUVwNMIi5c0JTzRXqymFolXi4A45DUUkl+25tbjhEArpyk8HnJ62GTMs3xdH83Stp0nBDBM7areFJHxDTWeK9JQAL4CYq1mJXSwBJOKti3jQwo9KXO5OpNtU1EV5n9vgrIo5c0i7Absy9NM4VSAYfJjo8eAN37GV9IAVV97e4x79pDEV82aMt6aQu9BY\\\\\\\/98IcmlNTQ1kiCktg6PWVdesdkTni3RxLH11No4dwRqU+hiWq3wcgUa0Qhf68RKrk4yoj\\\\\\\/y+vToyOHGXm0NG4B5PbIrZHYVyDIL5ZGYOiGnmWMr9xyU253FmFl8+9HPop3ziQQZ\\\\\\\/wIDXhsBFXEtsdxzS150tZ5VRFcjM5bBZLxx6zGACr5CKi\\\\\\\/0IlaZF8PX1FRfW2fHOvZ3AesEqBzIhHK8IAZu\\\\\\\/byFJzSn7bcizg7TQ91FbeRK9vISLu9ct\\\\\\\/YIFjFaHjUkmIJN+YpS4jilMCS86euutkAmgRkB0wMrppHHnWMVRIpJXWVKgoCPGI7q5HXWM+aMydwopfxdeH48GBhFuIcMCUNtI72Gr+jEMcmOdurBIq3DlB1x3s7DuukX8Zz3DWudOfMF2FNeAUMQZul5uBX+fZnP\\\\\\\/EGFOwR2zF3xQuI25LUOPWNQGZzaeeD1frG4f5Ld91suRTFJI95naCsIvC9fxiXZZbzGhQGqgfwpWr02aOjBF1I4nVNIJhY1tUgwj1FT4EWcPjAhXwk+B+NNc9nkUNj7GycF6u7DpaqnssCvNGcQXRts0BMzuR9hQhXkMpBu9XWZgPfVeF1Yhl+xmcz6\\\\\\\/CeXpKQwyuRLCCsGoS7d8uBpk0J6Cj4hEHvjX1cU71YxaU1Kc9CnOYYnFtY7sJwi5GPClWehyjYer43\\\\\\\/RMu4cteG5nlyIFGGACO6odh2JxkIlsAUUi55b2fRP5sQAtAjAphtrpNZmBTI2PNnHEl6lldOfo\\\\\\\/CtpjC6SXjL38UcvQH0DYIHtK+ynBWggNR51sYaYUvgYkWI3VefEpYZgOymS8UT1vNxmmEgoXT09NvVNr0tDvlPctdOhWcEkCkOcUROgCkF8plkPNZpxwmBqujk16ASHBAaO1hzOvtQmaxSpJIPGFTX5wxCX6QaVrlfKc1ruzhqgy9TLU41lAMTZylsAhTKW+ZDa+56d23UFpcnwsuIlwI\\\\\\\/Q0zQUWCSpVFu59plpf7TLShyF4PxWkT9ViA2dBMeaWH5Z+HfvvtdfLD5AA099xmPO0ykpBs64tvmgn\\\\\\\/XL2la9WkH8Hd2AfuLkkwhOfUAfPESZXDeuC1GinoUdX4+PGVYEwkco64zc\\\\\\\/2fXG7Pr4skvHflbFJs8T\\\\\\\/rTrGisLeqzz16GW2se2\\\\\\\/10pNJvzX5aSGNl4Kibiuj6XUjqeOoSOo9kGjelt\\\\\\\/rJ1EZ3GF9ixwZg33mE354uKWmBwx75RNFucZuxfAADcalaztciGEiglotAmq2EOuVFu8DUmr7Gs+HUL+neBdSRMDXNGkqSc+Z5Jk4BKwvRg14Dml69mu+XlZWHRpjD5pTZRilWVNuHcOlRpROQ5d9bWOsLIvjEbGIz8FEeyGnMiVhmt8EC+QnhS1DuRO31a9t\\\\\\\/9alxjyJRW\\\\\\\/oQxWRB6KUhhp8nWLjBN4Cm8UtbQfqldq0YnCPlSI4piiR5Yv5IgEM3xF5zwPKFB8FQE\\\\\\\/xUZQarePwFj0SJ22zCdk7p07bANTRjc1m5W74PkZJhG6II+KLgcMm83ZcQo05xVKjc+508i\\\\\\\/GnBwglpgfMtxgDNJ1KF4L\\\\\\\/BvpwW2RwEVp7Ts3F27RuN\\\\\\\/q23xDsOxBbiGSMDHjK2QQBpTnXjqN2z8wyvcC\\\\\\\/+O5ugcZfE5MTf5IgrT1K\\\\\\\/kOhLsgwQskFd1UODOC0o8nc\\\\\\\/3WWkVHfPuvbnY72lNfDe1Mm+fZ9wGqsCKN0Gkrz+gFrAYOGr2oQx\\\\\\\/fOsT61\\\\\\\/2Jcj7r0D34J7AXHd0IQoT1PlcpW3R4CUfMo5R\\\\\\\/hv9Ddua8vLUhx8y6zQUT1E9u24wt3cU+tJ5PE4J+Gor5eeQfvRrKJyYwdOx1Dk\\\\\\\/Qz0VMLwcqCP5j0A8twZvgN1EKzMmhkT2J51brgnmAOWP7aeojvKwApQ6mn7tc23fQhlw19ENEuB2DlQg3aMffPCZfl3U56Ygzgi6yKtuSf5whM\\\\\\\/NotaQjgkZDY8W7SyKKS0UArDJailrM07zaXqpr6fjrPB8Em2wFVZVWPWa4yRivcN4knNCm1MD841hyqa1rbnsLOrLaVk58dfF9Us5kSCsNqyFK0Mn\\\\\\\/\\\\\\\/FaGQU6kvmHZmg6V+5koMTzlDXitAYu9ita4GLZ480VnMAsfvQcbA6InfRcHAUHLSLhRM\\\\\\\/rOmnG9PWCieWrHsbYyRfcgSuyJM8bYH9wuqDEZWrwurcLNpNskyPqagCirupWL9rUZE7eaIupZJBrYXJcKYDDWbqQinJ3SyDcFk5yICPcVYUzjHtdFvEdjaM2mxtqNCpUDOVn4NSbNtQl6GHO+McLzuDEuQfN8pJpAbf9n0e7z\\\\\\\/tqev6DDhJ5a0qPJ8RKw1TVCZq4F36uvyDhCFlRxXYY7Bb351q24LrfeJ3RLSTHk4kXXX5efBgl2njRgVfex1T75Za4Zx9uLOzu4q2DcOd03sjJfjxemDOkKfLZ0VKnHHVNeiMGh7QQG1KePs+gtoHJXDTvLpvuj7JbEJD8N4C3EUt\\\\\\\/\\\\\\\/sRIk+KTtNlu+E+VKa0X1GaII4tMt\\\\\\\/pUuKoCe0tfbL69pxiFP8w+yCsCPn02Mbo8Rh9iYhuD6trXk+FUpRizoDg\\\\\\\/WoUT79A5wVNaKDDILEzYiTqS4c9p29vpqsnWSbREMb37LkC24ty3JUJd04P9GfoBh7SfBow4xWazkhBl5aqrLr6rZxa61nHnZyROF8fii9FU+TBZDyzleJXouVQtxidtTB+geysiUg5ERZjlvHW0zBAv5b7btmWmiMl\\\\\\\/5eRSrYPEVJojeArr6Uak51KG9fVntbv4eCb5FXbuTEFk0OOJ\\\\\\\/+wOd25QugcwNAuyhXYkm2dtOvAXEkVO9urM83Y64qm075Vgp+qtNIv876BflD2uQQEBvZZyV9m9k3BKGvJt2JEZUrS0op9CiulPdHpvVgUupgOpG3MMSGRjNzXX20AHnBAujVGDUDs9n8OgE8Utic5uYZ7iicSVK228hIV1JC4lAqd2zeDdoAmEUsKMDKzab+FzbSMLOtMGROFIYmYd10Z4CJMqikMwfGuIZ8Yn5on8CCLlRicQCvhZvOWyIHJ9E3gehpZACi9KSZB6NX14nDGkjjP3jz0ZjLoXkxXLlG\\\\\\\/SG5Ez9NTRzvVsUvgxgw+OmP7ZwBW8\\\\\\\/LXclp\\\\\\\/wYDjZZgI5F2ah5fIvTbvhVdIlNj2Kr7RZc6Wy5GccSfSz+6pmcnnmphKrmt+BAF86Hg4YZWO5M\\\\\\\/3oVImrVlx+kXtlpaH++UGv3Y2doxjaD4HFzHzHwBSE5KvBmYauyEBvNu5VysG\\\\\\\/\\\\\\\/GsWcwO8OqBvPEkdx8k6IZ3Ixfof8cpdRCaxARPy5zMNy+fI+oSgU+sNKwq86giKuD0YzYX\\\\\\\/Virub6VoZuRgqvXlVweXasqmuEsdZrAyJeJ9bg0tmLG+MCrlGQz+dKOmkLUNKMyOXjDwsmNjFqkdoYeh6ZiGl4k9mx8OcIVTtqzQ4+15EvsZkGkYwyVyHqGbxYnGXgUMrKNeXtKp\\\\\\\/ogj\\\\\\\/tDwAZmEa+Qz4MSxSxvCKlXCeOT5GR3UPRjyjWSHQOei79Eqr\\\\\\\/qApQxbfsxjdzV14\\\\\\\/Z0\\\\\\\/0xZqBqT6jKSm8YcYhefYXm5QF8ijB\\\\\\\/tA6G8e5gjgwVUjjm8emoUeDU\\\\\\\/C\\\\\\\/LngJottpYIhKKQ+LluxZRPvjnlGTT+Ts8XeLibzQHzs\\\\\\\/gsfy6WpkBm4UOcyslhvqCIDrODJxYnfH+XPQXJ9KhLRgbI1VxisSi8Bd8qk2T1GaEssZBeJqazR0OYNFkN59Q2aL4kdsVN7jZTbLe+UPL3640ydSUXKtrJQdZqV43uj7+J7CoerxGDRMCzZZRYTSQNTj\\\\\\\/d9VsauuHK+zj7TCx8EZkvT0COPuYAkzBMHaIHu2o91u7jQ31hJNWPhcXMSneceXRmSQhV9RhsFIrQC+ZXPu4c5EfEzYsMK6SxDzQwLIejUW8YPcEzmp+WsL+6aSbcXiEDgiJ47fxms2wCZbHPCSLT4hwLU++iRULetrKNZsNJ3\\\\\\\/PrkMeKqCYfhy2cpZGHhZ8a4Qf0rNG+6WDGJA3+RdkyVZqDzk8el2siotdS4TDYcf2xFLYwYQSq+bFxm\\\\\\\/ujNnb3wwiU4YWNyFqJ8JlGl1Dy6do6QXc0raj+aXdHL7o1QEOTKdlTd6Au2MRn9s3GGeiuFi905Tuv2RRqj\\\\\\\/V5jcyZsTrF5qTVm064XJyNf2keockm0qfVtLxB0j9jXbWutR14mbKL3zPwy+HalUFOKu56NiZg8vmxYonBygMljevdIW61OTdwMcJRvP8W4MAfHollY8PP2KzgE77GtepxoIa3\\\\\\\/fkskDB3C1+4qZKIots\\\\\\\/U8VZeU9ltTwqDvCO1tYv9BdElT1YvYcBpscK+e0FreSbeBW9JMl3yvpxrpJDmS1hOlq40LDsaUF4PydEgStYSxrZ7kXWG93jmxUa0xAzmfY56RAjy9eiZtgMkhAcNTcvJnCfuVvM+wrU4MfVPFAHznT1Naj5aZu3qXQ7VcZFH95LKMfh7F+bGHS\\\\\\\/tHK0Si5jw1vjQhDj4q+dmxpnMMNsYSeJzEemDhcJi4ZG1wWO3W2biDowhoBZcjXYbxlv4BBGFy4dlFUu3UPIHDKCuBvPdq58EAPLC+l4l0Of6VnLiN5R\\\\\\\/uRpxfrgYYssPUBvwtbZinZKu+4QoVoKZRsxrQ8+\\\\\\\/l6B0McCIR5i7ha1axPf2XbHNq9aNA4U5XMXegPGrFf2d7nfW3m9mEUKj9ZVnAfHwsGJzSXt75EaxmofqCrcdGOUmYLZWIPdBW8+ECr\\\\\\\/RyvM7CWfOsoSPfT7y\\\\\\\/TQhkj9hQ9Csjbko+5Nj30bIglqbChygPaT3mrsPj3X0jZA8Fi7xgiVRsa6ruVFTElMPL4fcd8LH1bDR9QKjYLKXcsFOquUZ+T1w+nTP44zV+o7Lw76z6gw43XE9\\\\\\\/RFyGIg7JdnBScTEA2Fsne7P+kSx4FwEER6pay+hzJ\\\\\\\/wDV5Iyz1ak5MJA4OnBgqfhZ7RwLbziaw337SG6vUh7y\\\\\\\/ys2vtqU7JINBMAMuIy9DGFX6HsuP4XSGOkoCmbpfIo90morPliYE\\\\\\\/KntfzJHzKhQmQ6fMEzyUjpo+omWt9Tba3C68eHRMy3nxGwmSj5ExBtBOhUnPEnTap4oPEoyid8\\\\\\\/7ByXi6bJz08ZE7KuIi7XFXENv5GDh3qeWyHOrZitlTws39BSMJVKPEB0moBD6+5iKdr6RB8YL2opxVleEUydyqNSMtXELeETOk3A1R8pzsVMACMEWq5cN2MYPEF657Px6kypjDLIrnH\\\\\\\/NXPRO4CKhYDSy3HT00W7q9LD3uDs7SU56R2UvEIGLz+2AxsNGFVSzsYD7D+oOEFSiH\\\\\\\/MTGayTCwVF9NbOOqCHSrgH4+Amqs1capzOp7zkOuEqBEDddT+JaoDtdMiTDqLMkc9WyXvYxTMBFOM3sk6wHixj9aIhlheoPZeTtMZqs3czxRl3kC7l0EYYafjx9wiDZJ+IJYkBYmLHW9DWzhZUBTnSBqWZhMNWLk89gebHOb8H1VFJ35SP\\\\\\\/YCqKE347yQQFVc5y72AbbRmGEi1NZn+iv0n709rY0ICXm8LZ93H96k4DHuXM5Xau2Fgx56oXBoabfVwp7BMUGmwcdpKkrls395k8BfWkZ9TRY1MMKSzj5w\\\\\\\/xWK\\\\\\\/LWD4cN9UZMzJK0KKspLl6XWN1uuCzU9AU6wofmfXaVjLkkbdxsHS1Ks+MQb7KtQvCpdgOLdbInJC3pMKvkKBaCVSjb6EGgqljT++BDZxDzMwnXmPazGDQ0YozS5dzjA9KwdbzY9pKuTxIOmfI2+fQemtF1N1IMHnR86A5c6H0YJ5LvI8rF4fQCiWx83G\\\\\\\/L+uXm\\\\\\\/dkeTz0EWU7vNE3HKcp\\\\\\\/GewkqT3ZrkYh4M2mCKAH0HQUM1qq\\\\\\\/8bvwrHH8Qk4a2T88YXQiPx7v3XcrOME5Ep5OWxqSbUNwW2g7WWluEyiZNkxaa0j0BB26D9QQy+qN1RaJoPTeb+Tl87VHxRcwdj5LZ6l2IFqkD\\\\\\\/cTRfxZrPlAPUiIhsINRiuh14Hjn6djck+rXA93ezGlZM6ufXt39F0yMeN2gFyizfQS3dZ5ylMirkVECJT9cippDd\\\\\\\/AnGs00oA9v0QwnZlbW4FxNitvDp8S\\\\\\\/H5g9woyAVi\\\\\\\/1q873xEtpwn+ICADOwu3y0lWlGIzRClfyx2OhqVWkZI1tG7BtYH0VJAHSRSUXyf3BoRaV057nsTvyiISyamPdhq\\\\\\\/Cj+McWhAdh3sk8Mwp7CLUtM7I2WCk\\\\\\\/J5rtRYPLEQofdqhD33Axguq9WqzlRHFGYf0L2sjCMe74FbeY0Dhir7cs6GRAc0xe63h0fFZKP0RFyipG0REQX9OXTAWfhekEzoPpGg27UhZ7ymz+04g\\\\\\\/sU55El1UHn2Aeg2TOHzGSJE6jj\\\\\\\/U2gGhd3NDLAA6Zlwa0NepdA8umUavDZ8ySd1+JPLCqOeqx1byeTXetaOrTAxuz0LUj\\\\\\\/dKoOMA0UcgWJb95GPfTJBqf7e5Us6M9VnU\\\\\\\/uDQNgyHAkDMQLcXTVTvQrKKDHef+WlgTEqsHpHG0YeJIF9hhGEykrw1Wu\\\\\\\/C94I1k9YPGj4AA3RViV9YT5brUQQC1b9LQMrhFNzqAcHeZqt6WWW\\\\\\\/c5vi7D\\\\\\\/FIGSoV6rk+UaSAZR4s5ob+\\\\\\\/cxdpmjFtMnJH1XwgyzdVzZVx2OTd\\\\\\\/IR+Oay9GBkjPQSq+fnG\\\\\\\/wrVUTl7OLP6hiMCntqUtSZ5\\\\\\\/ISfNwE9sGiKSEBVJ7uV0rTSyet544qGJePioZchEWDY68hjkMoc\\\\\\\/Qn9Q7DAtxpIO3m+PnrTWrHAdvT+T5fRH6b6UI2ob3WP6T4ZvD66JM\\\\\\\/\\\\\\\/7DNsic1+Yfkq7gIP4NP7ACbD+SkDWFZtEeGaRACJcBtY1OrhRjLm28WNYtt8FZ\\\\\\\/Ukn74T67RDnK8w2vOssBl9ezzs6DO0LAYQfzMwZkLgGaE6ktpIS9RHw2ApM8iGLBDxAMmigLVUdpSLPL8DCtvmalFxd51HPFSfE9mL5LvmxS0Xgt+a3iuBnTfcSz0htw2CXMcE+8SwlpU8piMsQl5U6AMz0ptvtnulSdZTgkfY4IpKJdYj1LYq4at5erGBvE4fIifbqe4zIpQF6Qlww5maV6W6ceiX+fhBvQ+l771kvuj08Rydr73U4uiLVGCbSeFxn0G\\\\\\\/gekFxJY2HM7yssWnoY1zOafhWufA1OoFU3QVkQJZ7jG4lYLDs159aENjIAaTOPSC4v9p6Nu9ZOzc93D4z8mISVSKvB43K1Wp\\\\\\\/m7rj19wrgwCvtbzTBYCaa8Hjom6VOh7Kbp6Fiw+wVi0cokdkYMLcVQLSGT+3bzXL\\\\\\\/vdg79izhgShnwnYyJH9uQgXVe\\\\\\\/cWBsyNa8CIRmJxh7VLHxRMHgeAD5EXtOlnbM72pY1ZWjEK779YXsk2s9Z4U3V0WJuxbV\\\\\\\/mKE\\\\\\\/r9j1LAlxJ9imFXjUNDhbHHYOQNr0tJY+PdYF858MaoiEDC6I5da+EHEYQtdqHfCYABr4l\\\\\\\/EPJWDdOr0tQhb51vBjsHjHe5B8lb8E31OLZx1PnpTplbG\\\\\\\/XAjqIBRJPxJfCsayuD6L1QSC6CiEE\\\\\\\/HmheO027My4Tz6cc\\\\\\\/pHFYOm1nWbmtySpWNi\\\\\\\/ax6PZosfrZPUPAZN8GM59Bp9sH2ubR3OHBAvGVjXHUJ0VP5v9Acj1beswq\\\\\\\/UKWKg2CcFVvbvVlsRqplTWfwj2P\\\\\\\/b3PGfn64b5LNRjmiNc+St55CNtXe\\\\\\\/AWUgCXTo3N7Akw0hlSI5ixb2fFdGCSVUNwXSBm9UGxzgNtFtaZ8d60hexpP3VoM9rWbMUXgGdh858k9U4niIp5K2kFqUPePMCYLzm3qHK6eC\\\\\\\/f6OdklbepW1xKF9uaxi7+5vjUj3Ec483aY\\\\\\\/UTqc2GjtPACVE17RF9WL+tF9M27FlmxwrrW0d0QoUBooX9CqyJcZHnTs+S7Lo+mASQb\\\\\\\/T39z0dYpGFguUbujSNjX2QLM\\\\\\\/pDiXHAVE0ar2MaZdX+KtXIHEfqRF8YAzlf4C9Ybe4hHe775UlDp7ETCSLVjYkK\\\\\\\/ONqPbDO5oQrJtqwfWq2L\\\\\\\/Ij808PZgTYVXdNkfhCKQ5pBVZQ9UMWlqq6n0KdpcAtrBAWLdFSyKQOSEUO0MO3l80sBzTRy6j+T1kDpBy6L+UwBXP1yDlgvdgV0dNUIh6OZXZhE4qrSdXlt6SLOFwvSfbJt8HOawyt2q1hesI0W6y95WqrDRro7Im0WTmIScU4hp29hD5IbMYVskV\\\\\\\/7UrAa7erkwzfXFBwUrhmQf3T9luZbrgglgCobpKADKdsPANPQxl3NfGWVqyhZ9OGdEF+f5swqSw+m5UoWuDk0REvrcNdGAPeboyWUep+nePn5ywAw7Qkf5f1G+cfl8TTXfUqBCfgibAsGKcUfxjoUWcBwdICmvvSEbgezqqZUqAmuaz7pIHn2NMaDegZLwTmBRjY9qz7QT1AgWQZDPoAxxwWGM28pDpu6P7f0fmusuKlZ\\\\\\\/5sp0GunsDYiN9FnmAr7Tby0X\\\\\\\/3kmG66ZvqOiLFpHcaXeMkLrwQIhDO8AJ9UUXHZSBKauogtYxQJHG4jxgPsK7obVz5hcrX0xrkqEO2NgnspqQqvs30JqZNtdYMSpLCPq4YHAuXhKg03HwVG1me12+szB4clTCBBI7AUZd9FQLZ2e\\\\\\\/vEtkjXZBn0TZ7Hdj4QcQBDHGzHemKlJhfURECt1uWeN46wROz9ZtQvirD4ubjoV6nSGR2KiGFiCheXS9uEn6dnjhHPCq11Xil4aj167SXlDLUjXBQ\\\\\\\/WzUMqgcUrZ+M\\\\\\\/vwdGyux3hHRL6xEDvTJqR3uAWdxBHkc1ee9N7vHNyG3kHX222IhcVzoSOLmTMm6X3JsuAoTv7CjlQgkKCo0HIKJj77wtNlLPFcG3aoGeXZ2pwUJyPRJ+YKiO2qpBAW8AnR4KpDYHKoS9RIRNzAbCXlz4zlgK8ouXFbGdxkb0ToQRz67mrTimUIo39NUmKYSD7W53tK\\\\\\\/z6VwcVLlYKVx+qiUPxgmInp5yhoqHLHb1PzosBhEloxuaiYaYoqg3Lnp4VqdsQlEHK4SnP3M\\\\\\\/hsF5esLc0TrsAHT2k+ekX+21dnHszp3KC9TZ3uXzKuJ5EhFlnZMts3q5S91Fwc05R06Sz8\\\\\\\/IYital0km0AWjC5o2xIrS9fQq6kk+ahSROoixCm3ALgjkG3REWvT\\\\\\\/8QGdqkNkdccYClnCU6lv1TkiDhBmQGWb7CuB\\\\\\\/2snxVh100kcnTgwzBG115+q\\\\\\\/NNfN5Ex88dPvb0NvC5Ze+F7M5hY+6RWthK72z\\\\\\\/14qJkXGSUQyWqE9FK91NviK9HmfEq1TAhUbp5LsZe\\\\\\\/VK1o5\\\\\\\/v0N9ESIxDOLX7+bbgspBx8huVqWgIp88IccqMHtjy2ynQr7a8LQcFJXNUatg5a4VniLwdHeWBYOVpLVNElUKeyUJ7mWyJtgvd8fmRDXDO1E0PiMb\\\\\\\/Uw+XPTYDjMFoTQEEy64EyrfIJbAt5g+BoZNH1\\\\\\\/ejUW8LBqIm19SwQIX4QqM6DqQZRr\\\\\\\/PqiZ+70c3\\\\\\\/B\\\\\\\/M9tDY\\\\\\\/4DmddexL\\\\\\\/l0pVkoN+TdAsdG9M+LORaXdb1ER4v5zI8yh2EVyXEiSqT40QtIr39XrZ2azbsRUG2lIMXNxLt5E8RD4TWgj0vYgNkjezj50IuhxSIeBkcwauTjBjGmYjZ6XrNCiP64mCA6qgI9+02vZ9ApDnOLD+FSd6uUNougqJfYcrEf\\\\\\\/WvSaW7Jt5\\\\\\\/acYNwEDmDNbpBIdXAvpMrnqN\\\\\\\/fO34lDYwAr4\\\\\\\/x\\\\\\\/OkzNmMsnzi6pW7EXF8Dyu6j9cYLcHrj8rT\\\\\\\/M4Lo\\\\\\\/3EQFIuB92EQkJ4XZqBhoUXSEYQBu\\\\\\\/1Ws5lqx8aOwUqIh81HRsjDx5cm9xCq\\\\\\\/wmqNsNozzCq9lkRkJWeLnYEgbuJsRvjGt+MOxDx0BcTueXTtQFJZerLb0mi7uRSyyfe7owcjUxd2fctQEBcY0v3mPNWPx\\\\\\\/R0IJ0+bBXNBPUV\\\\\\\/uTbh8JP6IgsoX5787hbgsMTZxPEGMUtEB4UOZaA\\\\\\\/CzGXCqql7E1ufmUwdwXGbTtfHMzTemyPbiset226mPNPyuKBFY+GPXBP5PGnlWYUnxo257rUoeSaa+NH5VlxzgQg00s20Evh6GhkUtA0O5iE4B2fmz5CgVd\\\\\\\/jxombQ2wjvRnrkENPYWUHU0op51RAljjAfjcC6AEPlX+QWB9x9AtTLibFJv7W7SJMZ9JD20Mkiy6XcJfHhdkQoHvZFUUzyBzdL0NgK4X\\\\\\\/qdNQ41yYyr3vYu7KNuOZOsF6DcdsJ6bcvbuEGQMmH4ko6\\\\\\\/1WWdIvx9TYrVESrEELnR72\\\\\\\/C+2jRHJKhT8koKQ8rngHMLOvqD01rnLN\\\\\\\/GxDxWx54FPoTumqOyulpDxHeoDXzLD96DNdvT3Y6ABjiBgMKjbgZC6ga\\\\\\\/zyKYqHQNebBJdCKf2M1tE+\\\\\\\/41h7rkWGy58+lau2wk2eKcK51CZGKSj0qUEbmulo+0i\\\\\\\/zQsmech5U4KfqoA+iYDOUDh34A9am0nLziMPEIBFhb05slplFp2ubZB6cvNSFCbkA9IAbJWK9EclCzXgF0+SxIQqVKvkvbsmOajzac1iVwIn1yY9uxxbnWlwIcoO0cZsRwD6jF3oY6A2lcNgMIN+gXASs9LjtW6sm9vmaN3NIFKNDOJiwMC6XNv3hinRCDJV7IjfBvDoDXtyV8s1xQVGPB17rwo9Na5cNoJHz\\\\\\\/AJ1m2VC5coFOXAE2KXgh1Ww7KGi9QJFpopAorlhFOZsKqQ3naXF7yVkgD0pP2ivFikXkyQxTlW3DEQl8aqJnF01sXF6qf1CJxR+pqei683RAwzpg6RY0BtS67CZ1i1qXBO6NiQDBUawQivpPHUqJVY4960lmw+EpsWh5ZMf5QHb9F5JXb1IPnLoJKJbPha53TEwyBdjgduMcqNtxodhdbnTyGYBnDMEFepqpMM4Z\\\\\\\/rs6yJEP0EWtp+chf0\\\\\\\/0dPTRa+UO7Lza8la73kSEom5qiAcwdPma+PaUfFMQ\\\\\\\/7e+1ktB3bufPL\\\\\\\/rHq+Y9MjtqG5IjpieH26uyMu7uFpaVMrnSNPlL2+E\\\\\\\/HYu1SPQD48eo0e4pyObnEZjPQojN7RV6PWPrxPM9mAlwG13RUj2s2gUv0kjqg67b3xYCAvKLW1StSvpgabvWuc3AsopH+Wis7Cv1Rtp8AX7K9af5HytuS9Kz56crC5IvFLc32Avk5CYbGTT0Id6gwTZfDCUjcHhQZHLZNaONJ\\\\\\\/g9wD8F\\\\\\\/cv3EXmz\\\\\\\/DG2hVy4In5pso\\\\\\\/Mivgk5DK8OvUM5e4KzwMv9RQgp4OqCZvxQwnYTPktVx3iSY2k4ZkcE62bQxVjqvAgOGO9XAZm+4lIOiuBv+\\\\\\\/\\\\\\\/5jahVQ\\\\\\\/zyPFGtvRUn2T5CBCHy70xYc9Rvr0SZ5FdhmSEdCriyAba8qS44GQ\\\\\\\/jJvRgeNuv+z7NgdsLkuIVuS+fNkB+RMY+68\\\\\\\/aA\\\\\\\/KudRVxJvB1\\\\\\\/s4bi4SDkSeUZfgmgdPoac9uns8fLxukguzB95Yq0nIqDhQh9CNA+IxcLCoU36yXNJroG3riewDk34+mp8Jt7tVmdZrkXwBZiq+kCwCyJ9lk6Yyte6WuE5nu9U2Fv\\\\\\\/vPpBs3+iUJqNKRjwZ9tGDTX0iVQHJHm2xCjaAVpNOEuVPPTHUzeusVfVOQEhxLj9Ki2MeHIFtjJaI1g9aJVgWyBqPW+PIhQc26h63UJWjrqqn5zRyDvk94yIXe\\\\\\\/+6WK9Fasl3mc9\\\\\\\/EkoClzJkio+3iP\\\\\\\/VbWYcAswAuLujbpBjwqZCK5+8JcuwYqbYHgXdMskh\\\\\\\/OXLkK7UzTv5RlH+kmmBkGfrZcIOZdDYtwSzbWmXxz29AgLGwN7KGjdHvgwvmXCIWjxNvTspmAlicmoJV5+HjdnOArgd8JVuxewfOD28h4b68i74lmXNtd\\\\\\\/rfFXh6KBvCGwvJBkge63\\\\\\\/cCy6QYcJqfQ1fspcKWcJIJX\\\\\\\/i19RTqvvttjHCe\\\\\\\/2gmRchNnABxF\\\\\\\/yOX8tRMF\\\\\\\/tjr5Ai5DPP9mZQXGI5mvkC5MEo\\\\\\\/Xsu0nI4IC16LglT8TLxmN7AizLkLS+kF8frAhhYDZhzQ2slar3NdkugLIXSQJexuNo4vrMk3SrYQZvg3E1NZdazxg+Uxf5n5IF6iCk2l6YaK9umOugIpMQdKrR0gf7nMVywsrI+4KHf1DftCrfdrrlILSsGTthAZHUhHnwfD1kb7heU9qwn3ZyW\\\\\\\/tZvt2C4TAtQ3CVxpzvvbtTQLh0HUOt6aWrk5iIfWpXfFUGZGOBKNkFJibnqTXXZS+cKQCC53eCpgwE0B8o78s2mVqof\\\\\\\/NZB2VKinWFoooJlLS0QspwDW+aTZQU4PXjb0y3I1zMi31n3M1E+tGjwcn\\\\\\\/krJv9owEfJ\\\\\\\/OTVDOP43wTQd4ACXH\\\\\\\/QtPYRtv4Y7G8kIiVPawelfh1vQuYV7fGD2lovsp99XXC\\\\\\\/SL2pEioas0iQNcAC+C9anWBK3wEZ7yChMBaHdOPvfGS6z7jvh+VTARfB8dblNzj0dVFMvyv\\\\\\\/0HU\\\\\\\/BaefDl4nOnIXgAvkgbl4nXVg94+9D\\\\\\\/4iSPxw8zD7r+QKzFJzlTa6tiJqeekJeOAymMberYnRD8wb03hsatPnUlcLrkXJoBQia3RPgh9uySBheVisvNw01tFZ7RQIyyzTx0ZJ1KrPQWeQhsiY7kqnxeSQwmNET9NVf4Mjp85YRrqA3OqmhJIdNrIITrzDk8qUpWdv\\\\\\\/5N5THOY8kZsb\\\\\\\/Zrxt9teYBc+3A49\\\\\\\/9ma99aH6ERqOZD8yyczbFeNVK0cPta21U0DywS2fW4uYfwGhKwpzlBg8E7tWJI+SyzqIZ+4bOZ3ZOHHzUYqv41QXhqikNCNcVOIk64Vs1uWItxYRUgeX1HzUB2GkU\\\\\\\/+EgwxUi+Ki9vjK18+VKoweYbcGoRdvDgkTwmjZhGlfY5ugAgmg1ds4m\\\\\\\/G0ZBcUUekJOKI4zSNRKJ8hzjFs3Auxt4WkgvPlujeGoN5uq\\\\\\\/hIGI9iHyF2U1I6EH9bEJmxbxxwmCfAC7nMo6EMEEWaFoAtMEupluW\\\\\\\/bSs1L09zgEmRJ7oyEWkbg3MaYDT6+YZ9HRZ0UtcEW57zHoFBGPVm4s5cSE+ObbEF6TjObQNpUvVpLsxb09AvqNOO7mVlacy0lpP\\\\\\\/wApy1V5mk6ewox4RPR7WxBxZBFWmJS7IzsykSTqgujU7jDNUIOrmgkZ\\\\\\\/D1b3fUsCXKEyOEcMt+In6\\\\\\\/gzSPpjcR\\\\\\\/1lpGTo+qVsvyzUIo4xL+VQyAHQG0pXspA4smbgDZwdZ+\\\\\\\/HkG7KPE5CEE9X\\\\\\\/gvUs16b1zMwl7oaWe3XQLwofBkz8VnurnvDT9xlIi97rsTO7IbZoOISRGgW0pReZa2kuJ8UUbELV5iH0wgZrzuI3zvSMGJTNqDAZJKaFQOokXjhzLusYeJRkYddplOOl5U+IcGcHDNI1qmJRFPfmvZuO29NFys+f\\\\\\\/jhGamIJ5c1TQ0Agr82XTEbFG8NoSyfsEkiuWcWptG\\\\\\\/VQQJizfoubQgMGiKNcZupZtnXF4Y9KNNUqC+XqSMTzlOFlpmVSqoeIuR9MAawCZ1G6CTvHM1N7rUq3tN9FpYqbRxHMeBBU1SKVWSqSFmrYlUrlWakKx3oTS4fYCL1gYF2bmA5pzcV+iGKu4Te\\\\\\\/B9Ib1\\\\\\\/BBYufgnceagSXmAKnrqO4AnNV4Uxg4G+h6uavIZkkUz8caq2V+vhHLbdt8UfVKjGg8oKvl0PaR0StZIKc8tP8vkVtMbVSZGJXnsTCEaibuGE35qGiOkPL50t2Q9FaK44MmCAfXwH1KIKar67tQF7Psdty5pJYAbSDmo75RaDLqmmOgLhM2ABAtbYGGF6Tj8omIB3HyLSUH3E0qhqKMXmBUiGrdTK6HzfwjZBsMYpXWIWNPThwgpNfXKyX2Nl9fL2bZH0IB4dAeRJHK0LZLTTN37Ag2YpvYL8AEASDn0S6w4aJh7cd5UAFGh7Ly3XOXYLvaNZ4NfSNR\\\\\\\/ljA+xxPtNh78i981eDvvNkOdjEMzlrvJ6mnjBz3gh0CbafmRj3tp5eyglTcqFV8aXpRo5co0CK246OfXHZgjZWu+rSYQSMRaAiDdjnwtiij7f8xQe9\\\\\\\/hbkcPG+Q8twaZOFm7pZGCd8nY\\\\\\\/Cq1udPeIVP1QvCXZG4bvxFad33C9uO\\\\\\\/51toSj++AGxRLr\\\\\\\/DknokDqQzrTCbqjXxSZ+OdAajLHNkZ\\\\\\\/tbbZOTgzRhatsErUmh3JjBHU28GmPUN1fPEHrTwm0443i\\\\\\\/SsLRebvSzVEI0ZquI6FKbarL3KloCq1F1+t8vhaX8kkiz1qBsrdKwqjmZgYsstCtHfd5sRCF9oeCUCZA3m4Koy4wIwbZhyMHBmb1dSYveBhgWZf+07NS\\\\\\\/OdGOo2GcIuHzxmRpVgCAEF9dIuJ2L3lz8lH0RCvOC5RrncNJue8eEgPWZWSk41UwLNqxW0ph\\\\\\\/jAFajDtraTPB+m3nGJVrYd\\\\\\\/ayzmKnNhnVVj3tm3ILZ7V6ABlxie2JvI\\\\\\\/syen1TuCSlR9wWq5PdkSZ3iwmWVUVRWK92+2zyvsq5bBQ2nWjGhtnLuK5YWgAIKXhRwBg+3qX7ANjRAwiIJ7ECq0HMJz19EPGmMfh1VhTu65kGVLir3yDuvfFNbkgnBTsGhcSSe6qEoiXiMqy57BKYCRXTEuN1d8EXUpICBfMK3i29TkqGN1337iHl5vm6lWhwZat4xzQTRZHsfDgxAxobcRGRJoqljllVC\\\\\\\/2+ODj7dc18zt5S1V1L\\\\\\\/bxuNHABjzlJHe5mgFEiOjw37ROVgrML+JwFqCzU15BXOFl4MFUHky4I6ISE\\\\\\\/B\\\\\\\/GG65uzNr8uoIoScC7otgEDLaxgjPcZA0Z6x+mZXW8EFQZ8HfM47O+mAsI3buxv5w3URmEIHjMNt8n6GTsQIwZAy6l\\\\\\\/UDaiPCqFZa7ip7XqTfuVQiwDDv1RgGzyDpnbDwUrEyZdbh6rCyO+dRWwZr4CAS1WRycRju1gyk3je5zH905ozie1BYi3\\\\\\\/w3UW\\\\\\\/Mg7efEAGPU9R9peXNYRa3BRnn4vcM0Qs4oGy5R4JJ49mu5jfLXl17hnxc5JtMDdijNW1StMxA8nUjoes9NKgvKF9oZeRBcgNF3OxAXh+WnclusET87b3\\\\\\\/KYHU5z5bkNhYUjC9E6q4QwZSlboERPQeqbrwzjyJ5cweMoxT9LYbOx\\\\\\\/VLNWAW7RIqJN6Sf4rpHIzRxFCPXtwYnfJ\\\\\\\/2shxJY2y2mRTL7HAnCKKLBGzH2AqrhnKGO0VmeIlXLIiIbj2ae85z23r08NPyD59b+PRxP\\\\\\\/LUKtZTHSZtaltKnGiWEIlDtNi5400ZviDS80HuziQsorEdIw4oV+bvT7LFqDuZbezSwMTpkZ8fCkjiFF+DYb\\\\\\\/K41kl4ns7qwAG6TkvXfF8EvRCPx6gfcSoYWzu\\\\\\\/gZvpeiQGfbuDL5sMQlv4M0\\\\\\\/GoHtHum9k8MmZaa1PErxnJaS\\\\\\\/aPqO6Z1wcwL+bHtC0IvdBRS0GxacbKKVdXXOHo56NLCwLUROii5J5mBhpnpm2nyBzaBbsobNYsOeY6vwuMqiRou9DPkZDkhQZakWnzstx92R+mXmu8X3T\\\\\\\/Wml7guz+ng7i7uHX9bKCdHrVPM74MP5A8WFBYgCmfxh6YlO1hOvuT0cGOwj5GplqlTEkrmRTfHgwHk5RsYMqdsDycIHhDPHKNpec\\\\\\\/4VyeB4\\\\\\\/9za\\\\\\\/480TjOxlBwcT6d5jEI280js\\\\\\\/NLAbJmiNrs7NCD2KRZuwg7L\\\\\\\/VpLj6IHvLH2sftcriFK2CD\\\\\\\/hu\\\\\\\/pdcfYMIspm8Y5tbYqJazGB5zFVnn3HoVXaYVJ5nSbqYz7XviM5K8zFzZHo5HENTJR2o20kMXF8aLhZ9VG5y\\\\\\\/2cB+0AOYe3f6Crqq7l+wMm0VRS4tlUIW9rm7r4K8iBZ1ywYuPrJG9R89TbCuu3e5fJxg4TFiqcbX3ZFRP16L6S9kP1DpzykbuacypRw9pHphIot\\\\\\\/BEgzBwZ5pdM03xD7RkjJvChyUalRrRnhUrYOZ9c6YPYIEzXuQU5Ks5cAwx1MtHS4HKfYSoQ\\\\\\\/kFNFalhYILT8xSc1hkFCt2\\\\\\\/DL4ICle3wo1WsMPrtAMPIviZ6H2D3uno3bXs4bCjF\\\\\\\/AgsZQCZyiB3jTjZagguTJO1NWjR20qO\\\\\\\/k2dilrYeN4sFKZhv7vI\\\\\\\/sWj5gXQ1r1+uAGXISH4SVA54+Gv9uD0RWc7m6VfucuuDPU6aaMkqtnkNLNOz98TYQxsqQC2y\\\\\\\/hI5EWYfYNZz9cvAp7fb92GLj1o1bm1+hABFtDJTE8EI7Suh8nsJzOq51RluFM5IPxC0tySIzk7ba6G\\\\\\\/ouQqSR9hs0hoYimxJddQZ+ItQefnex5KMBmj\\\\\\\/qMRGGecBLMKt\\\\\\\/kU7W\\\\\\\/JJye+jwvNsfSOIOHg8DRPyvPNkJ1mJ1NMsSKUOouRDZT1M55PrQvVog1fZ6Q+CsLwJJkbfo307BwDGBcoeAEnTSzsiSRwWdFmU55cL24FYcgCXr1qK4kKltQJ9BA11q3BpqJlTOuTzxO6M8V3XABY4mK+5rDCFA\\\\\\\/xSgC6rtx9QQjQm82A9sP+1kxSHXFt9xAXTzZjjIwy1LIBzT77eFxRu2PyUEfGRBsREuxLtmptwndfcK4unmbEkQUF+rjzjflYAYBYR72LuGfbh0oZ3yl2I1oaZilVLxJ5k2PDF8xwyOiceXzRsr0SICQFf3fcZmBDIvfLBS06717px9EO47wikT2nz4vU1FU9SHCKxjlRLFVBGgkABYlhlJ106HgXabj8swIyJKJWsI9U5U8C3J6pibQQZuLJTwodfudPwGPwbjZ51ynmGaQKDyv9aGWd1LTUpx32Q6zkwP4owwgNkRFg\\\\\\\/tlyUVXH\\\\\\\/3O5ohRbQOJz4MXy0\\\\\\\/u7wf1wiTpaKTGDxaUQBZVEnXYU8JAoM\\\\\\\/qCIzVmFmurT79cgN1ZKQPmETw1INec\\\\\\\/x1K\\\\\\\/3ULwGaTzn7s8+f3lJaZdn6sDEY5HLPDlY9pRGQb2qPaqMTD1sSBo1OeBl7MhM84MrC8jIt6CpGGeGkxhYsMfVZhLbcjB1TyAJvEOuh+L6GWOWhm7Hx9fKpGgPGx5XMYwjFXMnmdreYD2OMzrUm296QgpyYZnbkC2rdn+9V0t5j2jXR+HaBBqF1gJyE8tNOLLvtMVm8FsAKy8LJUOjKibCeKilOSLtiMfGZLttkQjxd8EwqszSAificCmkh6AdC\\\\\\\/ptBK2t6nUWBJL\\\\\\\/GKWKC6rxjNEjcTtdl08\\\\\\\/CwMpEO10K8ZNBj5eZXE5eywdskvwymuyPb6H1khNcjXmyGRFt59Emxn7qoYdBsYGXHynUIswpjeC6OXK\\\\\\\/+ytjo3sRHJ415EjvTxwEiOmOdcsvFfbijWEGRymlkEa3D+lBQ60tKBPjfbzb1ABCjxHEQgXqH5k3UoUHhjpthMBaR+ztzvv8bLun\\\\\\\/AFyAXZwcVo83y\\\\\\\/4lZbjJChAyQnL35bYUpGUjl9uCKTnEsDgJgAuhtvKqTlRI7iI2PIzh\\\\\\\/i2qdRVJyyDt7Oybue+jHo6CPxdGNWSJ5sVOQgdzlZUzdsbjMlqZo4wXb+RX0LosB02g9l+IyPrAFwxOrz91fwvQJN6oISCx8+gXNHOqCi96Z622uoVDI6kx9XJoF2OJWOiFKbOAb0m01GXFdtamF\\\\\\\/veA3tfL40nzpVmmgOQDYtUKZnr1cVPZa5U1PTIqO\\\\\\\/VrH+\\\\\\\/+0NR5vQtmVl+5+FMwikA8SGfQWCMs8GPfjvWhmMu9mUDJnN7W258\\\\\\\/1Ehg+c+wfnzMN4vPCB7x41v+61sIVDPq+waxfxRiKdDPIcJdUR0xru++XnXmrEEv694hNaSPdYWDuyg+90\\\\\\\/CdCqDqEB2UYnBKQjfKAEyTiDP7U6nTpLYVaO6\\\\\\\/QrNOXVuD29ma4VDF8jKiIZrEw2urzNmlLioAgCzeQ2h0nQFxmm+jExx1jNdtHPjehWN9TpOvxNL+BpcTV8Ph7P\\\\\\\/TiGLpI1neK6YW10aFu4\\\\\\\/+PPEqypn4tyGcohiGaaonCwJqf+c6X6Z5VXCswx55hYkhllduiE41L3sDIm5MH4DOPMFANTqNKqkrASpcMa4va3+QuNEnlaLTz\\\\\\\/ZiCSmIYiUsJlXPkLFAsnJjNVVcYISROGYJzWFVOvbEJClDQFyceQYumxtUKNhDTkqTvtRMA8B\\\\\\\/xToBpFHPoYwRtLlT0CnCRv852pLo0nWBgtxBrW0yJQ5N7q0l91z36gZG19OjNHA01revAxSjGbH+\\\\\\\/+0zGGJnO3spJOrY9vL0Vx\\\\\\\/ajo3Pyj+Qkq\\\\\\\/YJXpd1EFEGOwff9FCxZitI8FJbDEUtXU7R3mq90qGkp7LtVU4N7qG8NaZK\\\\\\\/vXdZuTGM23ZOhV\\\\\\\/+fVlsKk1ARyzL8jTAWV6rkMdkywroXez64fl3+EGXctFtKJ+973tWNzOU6K9YaEdguLgYiD8IBYFfY90SlaCKO4rYsD45Cc0rztHSSWL9qdDdcKEBgnP1eZnxKepyImpNWQzVBs83hpS6xyBWtmJQ8D3bXPaXJv37mGSgrYa\\\\\\\/Uie3lS0DzE12zblCjDM3J3id5n+AG8QrJoCMTQC54AiCh9MazNG7urbfp8UU5mhVDD5IJjJlfjyX+wXbKyas7AZsJS8sRjrwgl+7RDS\\\\\\\/A0UuIEFAT9oiUOIyYlweNby\\\\\\\/pwCSg8aFYUCEdrhibGMteF0LAVzGPpVMuihTJP1NG5mXCPvrkyBmeS7GhlQbd5yZ3kAFTNtWTJTPYu8yPdRtHgC8MuEJiyTzX2zk4hGN\\\\\\\/oZX1Eh33dF35POufya9RtEFxPR3VC9W1xbL7gVh7LR8VxhREspQsfu27Ll1QRmRGKUBbuGdHTpH6mwouAkTo9z8mSovMDTHOPbwnsg1tV\\\\\\\/irT5Sg+PRBS8UHb1XsBlx3aIVqPEYIgy0Cgbc\\\\\\\/86N4iAiyjFkGquE0Vvxja3BkHMxlwHbV8Fq8i4JwgmAMGlNPIAHWZRLCq6uRkF6i4HW9TQD6nxA13y14iynToatwyLZ9B2iC64FQ4hiOtvQuyTc16NJ4\\\\\\\/gDkVDD\\\\\\\/9r4HOaYHtqkhRqKEtpYV2MsjJo+H+L37Vxu+qwzpAdXToWHIoPEvmoVOnpOkHG4JmTu08Ka9mz9DjJjPd1393UUDZz\\\\\\\/LMcop6GetMW+mC7q9wqMEemurVuDWBd9\\\\\\\/YebNZeojzE6nIZI3VZosE3vBzrmbfUwRRIEb6Ftxpwr7RckGs0OREDidKbYz2LxCxyVE7V4pkKvFxWKs8M\\\\\\\/PgolzkzZl4srNjOpN8a4oNzMVrisCGFfBHGwC6fd1m2iZCTOO9XEuvpswZjI3ffdDB8OVCw4IzXz9TABsoa1slil5zZo2WEjB2U19cjjpbe82t57WBtaH\\\\\\\/o45TcYQDs+VHCKxSYnrueuFFfBqDVcjd+nWNXqrIoZfmD3aiF5VbLgOqiiF61Ohj2Pz6DePERSYU9\\\\\\\/mDFCfR+ZreleTpqvmSoKI11fLV4SFA85Ml8neM9MTeV4qWkxOE9cpRZP4jCLBnexoZ7l+4vgTcEOjCo1G+hWNYeMnKsuzOu0XuuGplDF8Bnr3KbPnMo9DDguMpIcZF59MVhwyrVp7dkjlYGFatVwMP7quYKQ\\\\\\\/T3tFmwyKfC8FysljILmZkpRAmV47auOSz03jxc2lYTxEQEQ+gKpDutnURzBecQoZVFHCxawzVGcVYAW2hJWYLQfMvR+NpktzkjT\\\\\\\/y6VGKlyiPnow0IzbcYUiQuh3Z12gFo0oX1rBOKdA1HtcHs\\\\\\\/e6xXTi2ULm2O8MCzvGdAmNRPB22KHJDJGoOoPpjPjriQ2bUmdZ2WQVRmR5ojCWF9OzclE3\\\\\\\/efOeUuHVsLhWxTylf\\\\\\\/sYTPHIMOb7XXw806YapNdzvel92AqbiD6nHUrLSlPni5fYnhheY9BqkveiO0RrYkk4gF\\\\\\\/iplnzyvkG0dmyd\\\\\\\/J5AhRKSyjZRS+DgYg8TwqOsz5WEDMmJkFNd73x\\\\\\\/TtXUxGl7Uss2ewRYVHrCBJWjm3ZODUOmzraSSvNnQi+OhgvlrAIGGKa3KLwfPdml2n6GsvYmnMhfryQ6UI1x5sFDkjMQPIzAyeYH4SDbHU+P8ojC6GxKcO\\\\\\\/haUNDm4568P9yZzYnkvMmv0Ac5uMXVYHrHBP53\\\\\\\/yA+eR2uz9mkPcE0+UutD+FBMln3hH7HwW93LUj2aoPpAmRNokVm25lv7TxH0rRYR4bZ\\\\\\\/ASMUdtYc+P9LJ69vbTeq7EbkfeXZ+ESxBB6O6s9r2w5kMS45\\\\\\\/xKrDG1MUlYPf+jmcgIiUiD3P8nX5AV9yxO+SkefysiQvBF5YqmZwx8B4Jhb5Z9coX\\\\\\\/kQ2BgAGsH4OksgDXy4qn8f2bAkkRsdIAgOjRagsXHatlaraidr7kUwODRaJs9YEKXzJFMX9DTzVaXNe0h7MzwXFfTKoCqFw7bC4L+AFcOguMKpKzv4TPGi\\\\\\\/2p83E2v4t3q17xLYgAPh54w+2AFZ3iP+llMXtkbfLqqjFwG0nA==\\\u0022,\\\u0022iv\\\u0022:\\\u00227IfMZSl2gOML0TTTQot6gg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228331e004b5360bdc1c243c6de9a3f020037556be327c69b11f4dac6286eddf00\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022T1UQcwaGzrwFVvvetK794KOOtYspIVrQWraZ0Dvc6GzAx1xMtVp6EN5cbSCPYUcNb40Rgyrd64LCuPsI6He1igrRLBJFV+31G+T78KI7qSGjrToX8Y20SQeVyl4toqKZ5cT1ng3NUqGxoBJsmQNJ7Q3i1+YStUQmSMhVENswofNDF18Myg6AXaw3CURVCuhercRSMUMlGMuG6EOWI5p53vlIIhW+skAKHxJIQKgKBmeJyesku3HdAMAPMwc3Zbpk2savTxk8e1yDZS36fAj32ArWbAFYJHx0WVakbjqi+EkKjG0KvZOf\\\\\\\/Qyg4ebOCedwYo3eJHK7gTROq+vIr4BnPWsE0QE3XqYKA\\\\\\\/UljdWSRPMxhRUpAaLG8ntLJFZu7bEieA8aWwDwc2kAlkfQN6fBdqcKda5ifXFEiK0aBD9Zv0vQ\\\\\\\/BjQ2WZeX74yXbmCwbBfdzHkg2qUk2d\\\\\\\/C7YfxwsOUK0uRlICi1\\\\\\\/RbH+IGJJReD7dP5im6dyIXZmfPuSQXgrinGewOS31xKeKnyRFsdXidueTSr2l60A58PKBKpEPtllYzhZP5UJIlRxvyfS+R685V7rKH6lPvYsZRUxFwehe6BJK0Me7LK0TTX4ZfPjgtI9HnCyYmjT4duKv4R\\\\\\\/HGtX+VWQET2fDhJOEl6SJP28QRsCgsrghvujdzdN0oYHEBZecY4MrKnBup28KpyRlVGOzyVG7I9Z\\\\\\\/\\\\\\\/LybO5CYpEpychluEorpmGIfYynxMSCFsye+LVLpC2AQUlSIsfUSh\\\\\\\/R+YjTy35O3z4NwchKTilb+tQRjYTmgWcVsv38cBc+ORSVDXmZrJ0uipWI3Zs2X5364jSnyCRTsOEstoG2CA9nwiq1nPqBGvJ8pNk5KTT1jZhtJRohAZkTeXrwnzA9HxbmVtMVCEM9QGKsoySEozqbB8RxcVPYJ3hHgFVEutsYWVHnOxeUZxZQUNk1moQuqIvqRdWp7Z2\\\\\\\/fKguVd2NDYAPyW+m6AmaYcaHEgpIFKW77lImanJnAJduB7JPgSU5rVv4+muB9EuKkBHD8eYbXdblJCiXhqz9GUtGdDjRpSmV97oufxYy0DmcRych7uInBF569aX46atJHHN4uL1w08r4upTX4Gsba7tBFfZiismhuhUxqXZCMJHE8u0QSLuYL4iAThSzQH5Kd6j7MuA2ESRD8V4s6\\\\\\\/fSBuNUrMcUm+LSp5B9KHI1d566aXEmcemRy0jhkBgrQ\\\\\\\/++m\\\\\\\/Kbp\\\\\\\/2bD8EPPDrCWIamS\\\\\\\/X9p7U\\\\\\\/v77VyDBnzUTa6R56GXuiK5yZxoX7DfpGU4bLxiNkhrTZ3Uu1MUj8YLGhQhCkJE7hd+6EkLMTN2dBPB0hD3t5OrjaDV24Rgqsb5bBxKAfHJ51aFtu3HUfKtkThQBIAdm1vke\\\\\\\/Hsyt1WTa6MUTgD1S0FveowamnQtVqNZv+lwwxs\\\\\\\/8vZu9EhM7Mm0OefkgyRNQcIN4US+Vx0XnEOTQodkMCcu3407BkcUnoA+f7tBN2VBVnwrE\\\\\\\/k\\\\\\\/bEqW5sPoGrfMSHDgYgnBMXhaqvV1VbN3EdsEOlJb6wxjALXoNz1fIZz1DBqKEx7QpV\\\\\\\/oR6aJOXInrlXtRdwVD53n4rImJfvA7+McPX4Juw9Cy3Bx2YK\\\\\\\/xlRcIKqM2uVIT0zNtAB2zlEmnrKFz3r7IBYAU3KqcQBf2Cg4F9m6WeXw6JCQCAti5Y5NQMSNpiNcyzcCBtT\\\\\\\/lCnJGCnMtH1DQBupwL8XREyxcGb07OLx5Joya+l9Rh7qdRL9qgqcztjN7ljcol3XhZPCxgiEkcP2AwOpXf59NNUoRyy06aJ8y6cvsak052WE9rpr8nHFRtY3V+xobR4gMmcqXjoL6Q3ZMH8uPyNF9TJEXecoVvYzGjUid+g4Sk\\\\\\\/ZjVGiw3vTUQAW8TQvXDaO9UTgNmvko3WPT3IzurwlKjiCxJARnQLImSU3iU1kWARzXh1ipaXPRLlPYCCVBy+FzphuD45W5dNxAAIZMzOx5QnbCHIZ6jz3ivL2YYmKRkA97DqmQteUYSVA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022v8FPDcIBXTjll12vrXCcsA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228fcf7b9fc4530a84bc3c5f07777194470c0234133d657a624c17798e66464fdd\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492211 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/375" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QpiMbxDZexea1MU4wFZqjBfYPIZwidUbopKq9SMpdUE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/375" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-h1_qdJF-vVkF4WE-oBP6SvOBxoz6Rul8GvckuonfrlQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/2/359" title="Comprehensive Transcriptional Profiling of the Gastrointestinal Tract of Ruminants from Birth to Adulthood Reveals Strong Developmental Stage Specific Gene Expression" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/2/391" title="Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492211" id="node492211" data-pisa="ggg;9/2/375" data-pisa-master="ggg;g3.118.200798" data-apath="/ggg/9/2/375.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A <em>Very Oil Yellow1</em> Modifier of the <em>Oil Yellow1-N1989</em> Allele Uncovers a Cryptic Phenotypic Impact of <em>Cis</em>-regulatory Variation in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5085-6240" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rajdeep S. Khangura</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5778-7850" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sandeep Marla</span>, <span class="highwire-citation-author" data-delta="2">Bala P. Venkata</span>, <span class="highwire-citation-author" data-delta="3">Nicholas J. Heller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-8101-8410" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gurmukh S. Johal</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian P. Dilkes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">375-390;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200798 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492211" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rajdeep S. Khangura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajdeep%2BS.%2BKhangura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khangura%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARajdeep%2BS.%2BKhangura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5085-6240" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rajdeep S. Khangura</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sandeep Marla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandeep%2BMarla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marla%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASandeep%2BMarla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5778-7850" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sandeep Marla</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bala P. Venkata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bala%2BP.%2BVenkata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Venkata%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABala%2BP.%2BVenkata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicholas J. Heller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BJ.%2BHeller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heller%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BJ.%2BHeller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gurmukh S. Johal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Biology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gurmukh%2BS.%2BJohal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johal%20GS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGurmukh%2BS.%2BJohal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8101-8410" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gurmukh S. Johal</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian P. Dilkes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Biology, Purdue University, IN 47907</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biochemistry, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BDilkes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilkes%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BP.%2BDilkes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian P. Dilkes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdilkes@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">bdilkes@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/2/375" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492211" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492211/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/375.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492211" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492211/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/375.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492211" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492211/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/375.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492211" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492211/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/2/375.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492211" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492211/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-6">Forward genetics determines the function of genes underlying trait variation by identifying the change in DNA responsible for changes in phenotype. Detecting phenotypically-relevant variation outside protein coding sequences and distinguishing this from neutral variants is not trivial; partly because the mechanisms by which DNA polymorphisms in the intergenic regions affect gene regulation are poorly understood. Here we utilized a dominant genetic reporter to investigate the effect of cis and <em>trans</em>-acting regulatory variation. We performed a forward genetic screen for natural variation that suppressed or enhanced the semi-dominant mutant allele <em>Oy1-N1989</em>, encoding the magnesium chelatase subunit I of maize. This mutant permits rapid phenotyping of leaf color as a reporter for chlorophyll accumulation, and mapping of natural variation in maize affecting chlorophyll metabolism. We identified a single modifier locus segregating between B73 and Mo17 that was linked to the reporter gene itself, which we call <em>very oil yellow1</em> (<em>vey1</em>). Based on the variation in OY1 transcript abundance and genome-wide association data, <em>vey1</em> is predicted to consist of multiple <em>cis</em>-acting regulatory sequence polymorphisms encoded at the wild-type <em>oy1</em> alleles. The <em>vey1</em> locus appears to be a common polymorphism in the maize germplasm that alters the expression level of a key gene in chlorophyll biosynthesis. These <em>vey1</em> alleles have no discernable impact on leaf chlorophyll in the absence of the <em>Oy1-N1989</em> reporter. Thus, the use of a mutant as a reporter for magnesium chelatase activity resulted in the detection of expression-level polymorphisms not readily visible in the laboratory.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chlorophyll-biosynthesis" class="hw-term hw-article-keyword hw-article-keyword-chlorophyll-biosynthesis" rel="nofollow">Chlorophyll biosynthesis</a></li><li class="kwd"><a href="/keyword/cryptic-variation" class="hw-term hw-article-keyword hw-article-keyword-cryptic-variation" rel="nofollow">cryptic variation</a></li><li class="kwd"><a href="/keyword/cis-acting" class="hw-term hw-article-keyword hw-article-keyword-cis-acting" rel="nofollow">cis-acting</a></li><li class="kwd"><a href="/keyword/complex-traits" class="hw-term hw-article-keyword hw-article-keyword-complex-traits" rel="nofollow">complex traits</a></li><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li></ul><p id="p-7">Gene function discovery via mutant analyses focuses on linking phenotype alterations to gene variants. Forward genetics has been of great value to understand biological systems but is predominantly useful for determining functions for genes with alleles that cause large phenotypic impacts. Natural variants, including those that encode alleles of relevance to adaptation and fitness of the organism, are often of small effect (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fisher 1930</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Orr 1998</a>, <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">2005</a>). Mutant alleles with conditional impacts on phenotypes through genetic interactions or modifiers, as well as alleles of small individual effect are difficult to study. Further, when studies of natural variation identify loci that have not been previously associated with a biological process, it can be difficult to definitively associate such genetic variants with physiological and biochemical mechanisms.</p><p id="p-8">A forward genetics approach that uses a mutant phenotype as a reporter for genetic interactions can be used to detect modifiers in natural populations and expose cryptic variation affecting traits of interest (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Johal <em>et al.</em> 2008</a>). This Mutant-Assisted Gene Identification and Characterization (MAGIC) approach is particularly efficient in species where outcrossing is easy, such as maize (<em>Zea mays</em>). Although this approach can be applied to any mutant with a quantifiable phenotype, it is convenient when a dominant mutant allele is used as a reporter because natural variants that encode enhancers or suppressors of a given mutant phenotype can be detected in F1 crosses. Any germplasm collection, diversity panel, or line-cross population can serve as the variable parents in these crosses to provide segregating alleles for mapping loci that alter mutant phenotype expression. Natural variants discovered by this type of genetic screen have an experimental link (<em>e.g.</em>, genetic modifiers) of the processes affected by the mutant reporter allele. Thus, this approach speeds the assignment of a mechanism to natural variation in the germplasm. Indeed, one can consider this as a screen for epistatic or contingent gene action. MAGIC screens have identified loci from maize involved in the hypersensitive response (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Penning <em>et al.</em> 2004</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chintamanani <em>et al.</em> 2010</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Olukolu <em>et al.</em> 2013</a>, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">2014</a>, <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">2016</a>) and plant development (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Buescher <em>et al.</em> 2014</a>), among others. In these cases, in the absence of a mutant allele, no phenotype was previously associated to the modifying alleles demonstrating the remarkable efficiency of this genetic screen to detect epistatic interactions Thus, this approach is a powerful way to both characterize cryptic variation in genomes and construct genetic pathways affecting phenotypes of interest.</p><p id="p-9">The easy visual scoring and simplicity of quantifying chlorophyll make chlorophyll biosynthetic mutants an excellent reporter for MAGIC screens. Chlorophyll is a major component of central metabolism in plants, which can produce phototoxic intermediates during both synthesis (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hu <em>et al.</em> 1998</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Huang <em>et al.</em> 2009</a>) and breakdown (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gray <em>et al.</em> 1997</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Mach <em>et al.</em> 2001</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gray <em>et al.</em> 2002</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Yang <em>et al.</em> 2004</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Sindhu <em>et al.</em> 2018</a>), and its levels are carefully regulated in plants (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Meskauskiene <em>et al.</em> 2001</a>). Enzymatic conversion of protoporphyrin IX into magnesium protoporphyrin IX by Magnesium Chelatase (MgChl) is the first committed step in chlorophyll biosynthesis (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Von Wettstein <em>et al.</em> 1995</a>). MgChl is a hetero-oligomeric enzyme consisting of three subunits (I, D, and H) that are conserved from prokaryotes to plants. The MgChl subunit I is encoded by the <em>oil yellow1</em> (<em>oy1</em>; GRMZM2G419806) gene in maize and encodes the AAA+-type ATPase subunit that energizes the protein complex (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Fodje <em>et al.</em> 2001</a>). Weak loss-of-function alleles of <em>oy1</em> result in recessive yellow-green plants while complete loss-of-function alleles result in a recessive yellow seedling-lethal phenotype (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> 2006</a>). The semi-dominant <em>Oy1-N1989</em> allele carries a leucine (L) to phenylalanine (F) change at amino acid position 176 (L176F). Heterozygous plants carrying one <em>Oy1-N1989</em> allele and one wild-type allele are oil-yellow, but <em>Oy1-N1989</em> homozygotes lack any MgChl activity and die as yellow seedlings with no chlorophyll accumulation (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> 2006</a>). The orthologous L-&gt;F mutation (encoded by <em>Oy1-N1989</em>) was identified in barley (L161F) and also results in a pale-green phenotype as a heterozygote and yellow seedling-lethal phenotype as a homozygote with no detectable chlorophyll (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hansson <em>et al.</em> 1999</a>). The biochemical basis of this semi-dominant mutant allele was studied by creating a mutant MgChl subunit I in <em>Rhodobacter</em> (BCHI), with the orthologous amino acid change at position 111 of wild-type BCHI (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hansson <em>et al.</em> 2002</a>). The L111F mutation converted BCHI into a competitive inhibitor of MgChl that reduced enzyme activity by fourfold when mixed 1:1 with wild-type BCHI (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Hansson <em>et al.</em> 2002</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Lundqvist <em>et al.</em> 2013</a>). This conserved leucine residue is between the ATP-binding fold created by the Walker A and B motifs of MgChl subunit I (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Hansson <em>et al.</em> 2002</a>; <a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> 2006</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Lundqvist <em>et al.</em> 2013</a>), and its substitution with phenylalanine was deleterious to dephosphorylation activity. The ATPase activity of wild-type MgChl complex is directly proportional to the magnesium chelation reaction. However, complexes assembled from MgChl subunit I with the L-&gt;F change exhibit reduced ATPase activity and no ability to chelate Mg<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-1.gif"/></noscript></span></span> ions into protoporphyrin IX (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hansson <em>et al.</em> 1999</a>, <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">2002</a>; <a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> 2006</a>). The absence of MgChl activity displayed by the mutant BCHI carrying L111F substitution (BCHI<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-2.gif"/></noscript></span></span>) demonstrates that this amino acid change results in a dominant-negative subunit which uncoupled ATP hydrolysis and magnesium chelation (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Hansson <em>et al.</em> 2002</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Lundqvist <em>et al.</em> 2013</a>).</p><p id="p-10">We screened maize germplasm for cryptic variation using the <em>Oy1-N1989</em> mutant as a dominant reporter for chlorophyll biosynthesis. We hypothesized that alteration in the quantity of biochemically active MgChl complex should change chlorophyll content and plant color in heterozygous <em>Oy1-N1989</em> mutants. For instance, in a population of heterozygous <em>Oy1-N1989/oy1</em> F1 plants, an amino acid change in the wild-type OY1 protein that alters the dissociation constant (kD) of OY1 for the other protein subunits of MgChl should contribute to variance in chlorophyll biogenesis. Similarly, a <em>cis</em>-acting expression QTL (eQTL) that increases expression of the wild-type <em>oy1</em> allele should result in assembly of more active MgChl complexes and increase chlorophyll content in F1 mutant plants. Thus, chlorophyll content of <em>Oy1-N1989</em> mutants should be modulated by the stoichiometry of the wild-type and mutant OY1 proteins present in the MgChl complex in heterozygous <em>Oy1-N1989/oy1</em> plants affected either by protein structure variation or abundance changes.</p><p id="p-11">We introgressed the maize <em>Oy1-N1989</em> mutant allele into the B73 inbred background and maintained it as a heterozygote (<em>Oy1-N1989/oy1</em>:B73). While crossing this mutant to multiple backgrounds, we detected genetic variation in the <em>Oy1-N1989/oy1</em> mutant phenotype expression between the maize inbred lines B73 and Mo17. The phenotype of the <em>Oy1-N1989</em> mutant heterozygotes was suppressed in the B73 background. However, F1 hybrids with Mo17 dramatically enhanced it. We carried out genetic mapping experiments using five F1 populations. In each of these mapping experiments, we identified a quantitative trait locus (QTL) of large effect on chromosome 10 linked to the <em>oy1</em> locus itself. The inheritance of the traits, proposed allele expression bias at <em>oy1</em> due to a putative <em>cis</em>-acting regulatory element, implications of the discovered cryptic variation, and the utility of this study, in general, are discussed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant Materials</h3><p id="p-12">The <em>Oy1-N1989</em> mutant allele was acquired from the Maize Genetics Cooperation Stock Center (University of Illinois, Urbana-Champaign, IL). The original allele of the <em>Oy1-N1989</em> mutation was isolated from a <em>r1 c1</em> colorless synthetic stock of mixed parentage (Myron Gerald Neuffer, personal communication). This allele was introgressed into B73 for eight generations by repeated backcrossing of B73 ear-parents with <em>Oy1-N1989/oy1</em> pollen-parents and is maintained as a heterozygote (<em>Oy1-N1989/oy1</em>:B73) by crossing to wild-type siblings.</p><p id="p-13">Line-cross QTL mapping: For these experiments, <em>Oy1-N1989/oy1</em>:B73 plants were crossed as pollen-parents to 216 Intermated B73 x Mo17 recombinant inbred lines (IBM) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lee <em>et al.</em> 2002</a>) and 251 Syn10 doubled haploid lines (Syn10) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hussain <em>et al.</em> 2007</a>) to generate F1 progenies. The QTL validation was done using F1 progenies derived from the cross of 35 B73-Mo17 Near-Isogenic lines (BM-NILs) as ear-parents crossed with pollen from <em>Oy1-N1989/oy1</em>:B73 plants. These BM-NILs consisted of 22 B73-like NILs and 13 Mo17-like NILs with introgression of the reciprocal parental genome (B73 or Mo17) and were developed by three repeated backcrosses into recurrent parent followed by four to six generations of self-pollination (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Eichten <em>et al.</em> 2011</a>).</p><p id="p-14">Genome-wide association (GWA) mapping: For this experiment, <em>Oy1-N1989/oy1</em>:B73 plants were crossed to 343 inbred lines that included 249 inbreds from maize association panel (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Flint-Garcia <em>et al.</em> 2005</a>), and 94 inbred lines that included 82 Expired Plant Variety Protections (ExPVP) lines from the Ames panel (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Romay <em>et al.</em> 2013</a>). Pollen from <em>Oy1-N1989/oy1</em>:B73 plants were used for these crosses except for the popcorn lines in the maize association panel, where <em>Oy1-N1989/oy1</em>:B73 plants were used as an ear-parent to avoid the crossing barrier due to gametophytic factor GA1-S (first described by Correns in 1902) (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Mangelsdorf and Jones 1926</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lauter <em>et al.</em> 2017</a>). This panel of 343 inbred lines is referred to as maize diversity lines (MDL). The full list of IBM, Syn10, BM-NILs, and MDL used to develop F1 hybrid populations are provided in Tables S1-S4.</p></div><div id="sec-3" class="subsection"><h3>Field Trials</h3><p id="p-15">All field experiments were performed at the Purdue Agronomy Center for Research and Education (ACRE) in West Lafayette, Indiana. All F1 populations described below were planted as a single plot of 12-16 plants that segregated for both mutant and wild-type siblings. Plots were sown in a 3.84 m long row with the inter-row spacing of 0.79 meters and an alley space of 0.79 meters. No irrigation was applied during the entire crop season as rainfalls were uniformly distributed for satisfactory plant growth. Conventional fertilizer, pest and weed control practices for growing field maize in Indiana were followed. Progenies of <em>Oy1-N1989/oy1</em>:B73 pollen-parents crossed with B73 and Mo17 were planted as parental checks in each block of every experiment. The testcross F1 populations with IBM were evaluated in a single replication in the summer of 2013 with each range treated as a block. In 2016, the testcross F1 populations with Syn10 lines were evaluated as two replications in a randomized complete block design (RCBD) with each range divided into two blocks. The testcross F1 progenies with BM-NILs and parents (B73 and Mo17) were planted in a RCBD with five replications in 2016. In the same year, F1 populations with MDL were also evaluated with three replications planted in a RCBD. Each replication of MDL F1 population was divided into ten blocks of the same size, and parental checks were randomized within each block.</p></div><div id="sec-4" class="subsection"><h3>Phenotyping and Data Collection</h3><p id="p-16">Maize seedlings used for destructive and non-destructive chlorophyll quantification were grown under greenhouse conditions using mogul base high-pressure sodium lamps (1000 Watts) as the supplemental light source for L:D cycle of 16:8 and temperature around 28° (day-time) and 20° (night-time). Destructive chlorophyll measurements were performed on the fresh weight basis in 80% acetone solution using a UV-VIS spectroscopic method (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lichtenthaler and Buschmann 2001</a>). Non-destructive chlorophyll measurements were performed in the middle of the same leaf avoiding the midribs using the procedure described below.</p><p id="p-17">For the field-grown experiments, mutant siblings in the suppressing genetic backgrounds were tagged at knee height stage with plastic tags so that they can be easily distinguished from the wild-type siblings at later developmental stages. All the F1 families segregated for the mutant (<em>Oy1-N1989/oy1</em>) and wild-type (<em>oy1/oy1</em>) siblings in approximately 1:1 fashion. For each F1 family, two to four plants of each phenotypic class were picked at random for trait measurements. Non-destructive chlorophyll content in the maize leaves was approximated using a chlorophyll content meter model CCM-200 plus (Opti-Sciences, Inc., Hudson, NH). The measurements were expressed as chlorophyll content index (CCM). Measurements were taken on the leaf lamina of the top fully expanded leaf at two time points. First CCM measurements were taken at 25-30 days after sowing (expressed as CCMI) and the second at 45-50 days after sowing (expressed as CCMII). For each trait, measurements were performed on both mutant (reported with a prefix MT) and wild-type (reported with a prefix WT) siblings. Besides using primary trait measurements of CCMI and CCMII on mutant and wild-type siblings, indirect CCM measurements were also calculated and expressed as ratios (MT/WT) and differences (WT-MT) of CCMI and CCMII. Phenotypic data of all the CCM traits in the F1 populations with both bi-parental populations, BM-NILs and MDL are provided in Tables S1-S4.</p></div><div id="sec-5" class="subsection"><h3>Public Genotypic and Gene Expression Datasets</h3><p id="p-18">Public marker data for the IBM was obtained from <a href="http://www.maizegdb.org">www.maizegdb.org</a> (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Sen <em>et al.</em> 2010</a>). A total of 2,178 retrieved markers were reduced to 2,156 after the removal duplicates. Approximately 13.3% of the marker data were missing. The reads per kilobase of transcript per million mapped reads (RPKM) for the <em>oy1</em> locus were obtained from a public repository of the National Science Foundation grant (GEPR: Genomic Analyses of shoot meristem function in maize; NSF DBI-0820610; <a href="https://ftp.maizegdb.org/MaizeGDB/FTP/shoot_apical_meristem_data_scanlon_lab/">https://ftp.maizegdb.org/MaizeGDB/FTP/shoot_apical_meristem_data_scanlon_lab/</a>). These data consist of normalized read counts (expressed as RPKM) of the maize genes from the transcriptome of shoot apex of 14 days old IBM seedlings.</p><p id="p-19">Marker data for Syn10 lines was obtained from <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Liu <em>et al.</em> (2015)</a>. The Syn10 lines were genotyped at 6611 positions (B73 RefGenv2) with SNP markers covering all ten chromosomes of maize. The entire set of markers were used for linkage analysis as there was no missing data. All B73-Mo17 NILs in both the B73 and Mo17 recurrent backgrounds that had introgression of the critical region from the opposite parent were selected for QTL validation. Genotyping data of the BM-NILs to choose informative lines and perform QTL validation was obtained from <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Eichten <em>et al.</em> (2011)</a>.</p><p id="p-20">Genotypic data for the MDL used in this study to perform GWA were obtained from third generation maize haplotypes (HapMap3) described in <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bukowski <em>et al.</em> (2018)</a>. Out of the 343 inbred lines that were used to develop F1 hybrids for phenotyping, only 305 lines were genotyped as part of HapMap3. The HapMap3 consists of over 83 million variant sites across ten chromosomes of maize that are anchored to B73 version 3 assembly. After obtaining these genotypic data, variant sites were filtered for ≤10% missing data and minor allele frequency of ≥0.05 using VCFtools (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Danecek <em>et al.</em> 2011</a>). This filtered SNP dataset was used for GWA analyses. A summary of the variant sites before and after the filtering procedure is in Table S5. The genotypic data for the set of 305 accessions were obtained from <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Romay <em>et al.</em> (2013)</a>. These genotypes consist of 681 257 SNPs (physical positions from B73 RefGenv2) obtained using a GBS protocol (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Elshire <em>et al.</em> 2011</a>) covering all ten chromosomes of maize. This marker dataset was filtered for ≤10% missing data and minor allele frequency of ≥0.05 using TASSEL (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bradbury <em>et al.</em> 2007</a>), reducing the marker number to 150 920 SNPs. This genotypic dataset was solely used to compute principal components (PC) and a kinship matrix to control for population structure and familial relatedness in a unified mixed linear model, respectively (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Yu <em>et al.</em> 2006</a>). To test for <em>cis</em>-eQTL at the <em>oy1</em> locus in the maize diversity lines used for GWA mapping, normalized count of OY1 expression derived from the germinating seedling shoots was obtained from <a href="http://www.cyverse.org">http://www.cyverse.org</a> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kremling <em>et al.</em> 2018</a>).</p></div><div id="sec-6" class="subsection"><h3>Statistical Analyses</h3><p id="p-21">QTL mapping: Line-cross phenotypes and markers were used to detect and localize QTL using the R/QTL package version 1.40-8 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Broman <em>et al.</em> 2003</a>). Trait means were used for the QTL analyses. Single interval mapping (SIM) was used for all traits, although composite interval mapping (CIM) was carried out with remarkably similar results (data not shown). Statistical thresholds were computed by 1000 permutations for each trait (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Churchill and Doerge 1994</a>).</p><p id="p-22">Genome-wide association study (GWAS): Preliminary data analysis was done using JMP 13.0. Statistical corrections on the raw phenotypic data were performed by determining the most significant terms in the model using analysis of variance (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fisher 1921</a>). Genotype and replication were used as a random effect in a linear mixed-effects model built in the lme4 package (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bates <em>et al.</em> 2015</a>) implemented in R (R Core 2014) to calculate the best linear unbiased predictor (BLUP) for each trait. Broad-sense heritability (line mean basis) were calculated using BLUP values, using the method described by <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lin and Allaire (1977)</a>. BLUP estimates for each trait were used to perform GWAS. GWAS was done using a compressed mixed linear model implemented in the R package GAPIT (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Zhang <em>et al.</em> 2010</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Lipka <em>et al.</em> 2012</a>). HapMap3 SNPs were used to calculate genotype to phenotype associations. As explained before, kinship and population structure estimates were obtained for the same population using the second subset of 150 920 SNPs to correct for spurious associations. The Bonferroni correction and false discovery rate (FDR) adjustments were used to compute a statistical threshold for the positive association to further control for false positive assessment of associations (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Holm 1979</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Benjamini and Hochberg 1995</a>).</p></div><div id="sec-7" class="subsection"><h3>Molecular Analyses</h3><p id="p-23">Genotyping: The recombinants in selected Syn10 lines and the BC1F1 population were detected using three PCR-based markers. Two markers detecting insertion polymorphisms flanking the <em>oy1</em> locus and one dCAPS marker at <em>oy1</em> locus were designed for this purpose. Genotyping at insertion-deletion (indel) marker <em>ftcl1</em> (flanking an indel polymorphism in intron 4 of <em>ftcl1</em>; GRMZM2G001904) was performed with forward primer 5’-GCAGAGCTGGAATATGGAATGC-3’ and reverse primer 5’-GATGACCTGAGTAGGGGTGC-3’. Genotyping at indel marker <em>gfa2</em> (flanking an indel polymorphism in the intron of <em>gfa2</em>; GRMZM2G118316) was performed with forward primer 5’-ACGGCTCCAAAAGTCGTGTA-3’ and reverse primer 5’-ATGGATGGGGTCAGGAAAGC-3’. A polymorphic SNP in the second intron at <em>oy1</em> was used to design a dCAPS forward primer 5’-CGCCCCCGTTCTCCAATCCTGC-3’ and a gene-specific reverse primer 5′-GACCTCGGGGCCCATGACCT-3′ using a web user interface at <a href="http://helix.wustl.edu/dcaps/dcaps.html">http://helix.wustl.edu/dcaps/dcaps.html</a> (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Neff <em>et al.</em> 2002</a>). The PCR products from polymerization reactions with the dCAPS oligonucleotide at <em>oy1</em> were digested by <em>Pst</em>I restriction endonuclease (New England Biolabs, MA, USA) and resolved on 3.5% agarose gel.</p><p id="p-24">Allele-specific expression analyses: Allelic bias at transcriptional level was quantified using the third leaf of maize seedlings at the V3 developmental stage. Total RNA was extracted using a modified Phenol/Lithium chloride protocol (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Eggermont <em>et al.</em> 1996</a>). Total RNA was subjected to DNase I treatment using Invitrogen Turbo DNA-free kit (Catalog# AM1907, Life Technologies, Carlsbad, CA) and 1 µg of DNase treated RNA from each sample was converted to cDNA using oligo dT primers and a recombinant M-MLV reverse transcriptase provided in iScriptTM Select cDNA synthesis kit (catalog# 170-8896, Bio-Rad, Hercules, CA) according to the manufacturer’s recommendations. Besides the cDNA samples, genomic DNA samples were also prepared as a control to test the sensitivity of the assay. Genomic DNA controls included a 1 1 (F1 hybrid), 1 2, and 2 1 mixture of B73 and Mo17. PCR was conducted using gene-specific forward primer 5’-CAACGTCATCGACCCCAAGA-3’ and reverse primer 5’-GGTTACCAGAGCCGATGAGG-3’ for 30 cycles (94° for 30s, 56° for 30s, 72° for 30s and final extension for 2 min) to amplify the OY1 gene product. These primers flank SNP252 (C-&gt;T), which is the causative mutation of <em>Oy1-N1989</em>, and SNP317 (C-&gt;T) which is polymorphic between B73 and Mo17 but monomorphic between the B73 and <em>Oy1-N1989</em> genetic backgrounds. Corresponding PCR products were used to generate sequencing libraries using transposon-mediated library construction with the Illumina Nextera DNA library preparation kit, and sequence data were generated on a MiSeq instrument (Illumina, San Diego, CA) at the Purdue Genomics Core Facility. The SNP variation and read counts were decoded from the sequenced PCR amplicons by alignment of the quality controlled reads to the <em>oy1</em> reference allele from B73 using the BBMap (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bushnell 2014</a>) and the GATK packages (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">DePristo <em>et al.</em> 2011</a>). Additional analysis was performed using IGV (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Robinson <em>et al.</em> 2011</a>) to manually quality-check the alignments and SNP calls. Read counts at polymorphic sites obtained from GATK was used to calculate allele-specific expression. Genomic DNA control samples showed bias in the read counts in a dosage-dependent manner. DNA from F1 hybrids between B73 and Mo17 resulted in 1:1 reads at <em>oy1</em> demonstrating no bias in the assay to quantify expression.</p><p id="p-25">OY1 sequencing: The coding sequence of the <em>oy1</em> locus from B73, Mo17, W22, and <em>Oy1-N1989</em> homozygous seedlings were obtained from the genomic DNA using PCR amplification. For the rest of the maize inbred lines, the <em>oy1</em> locus was amplified from cDNA synthesized from total RNA derived from the shoot tissue of 14 days old maize seedlings. PCR amplification of the <em>oy1</em> locus from genomic DNA was performed using four primer pairs: (a) forward primer Oy1-FP1 5’-GCAAGCATGTTGGGCACAGCG-3’ and reverse primer Oy1-RP12 5’-GGGCGGCGGGATTGGAGAAC-3’, (b) forward primer Oy1-FP5 5’-GGTGGAGAGGGAGGGTATCT-3’ and reverse primer Oy1-RP6 5’-GGACCGAGGAAATACTTCCG-3’, (c) forward primer Oy1-F8 5’-ATGCCCCTTCTTCCTCTCCT-3’ and reverse primer Oy1-R8 5’-CGCCTTCTCGATGTCAATGG-3’, (d) forward primer Oy1-F9 5’-GGCACCATTGACATCGAGAA-3’ and reverse primer Oy1-R9 5’-GCTGTCCCTTCCTTTCAACG-3’. PCR amplification of OY1 transcripts from cDNA was performed using all primer pairs except Oy1-FP1/RP12. The PCR products from these samples were sequenced either using Sanger or Illumina sequencing. For Sanger sequencing, cleaned PCR products were used to perform a cycle reaction using Big Dye version 3.1 chemistry (Applied Biosystems, Waltham, MA) and run on ABI 3730XL sequencer by Purdue genomics core facility. Read with high-quality base pairs from Sanger sequencing were aligned using ClustalW (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Thompson <em>et al.</em> 1994</a>). Illumina sequencing was performed as described above except in this case paired-end reads were aligned to the B73 reference of <em>oy1</em> gene using bwa version 0.7.12 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Li and Durbin 2009</a>) and variant calling was done using Samtools (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Li <em>et al.</em> 2009</a>).</p></div><div id="sec-8" class="subsection"><h3>Data Availability</h3><p id="p-26">The public genotypic and expression datasets leveraged in this study are open access at the data repositories mentioned above. The input files containing genotypes and phenotypes of both line-cross populations that were used for QTL detection are provided in the supplemental section for re-analyses. All of the supplemental data (Figures S1-S9, Tables S1-S23, and File S1-S2) was deposited at figshare. The seed stocks described in this study are available upon request. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7370948">https://doi.org/10.25387/g3.7370948</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Mo17 encodes an enhancer of the semi-dominant mutant allele <em>Oy1-N1989</em> of maize</h3><p id="p-27">The <em>Oy1-N1989</em> allele was recovered from a nitrosoguanidine mutant population in mixed genetic background. The molecular nature of the mutation is a single non-synonymous base pair change (<a id="xref-ref-78-5" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> 2006</a>). Heterozygous <em>Oy1-N1989</em> plants have the eponymous oil-yellow color but are reasonably vigorous and produce both ears and tassels. During introgression of the semi-dominant <em>Oy1-N1989</em> allele into B73 and Mo17 inbred backgrounds, we observed a dramatic suppression of the mutant phenotype in F1 crosses of the mutant stock (obtained from Maize COOP) to the B73 background. In contrast, crosses to Mo17 enhanced the mutant phenotype. The difference in phenotype expression was stable and persisted in both genetic backgrounds through all six backcross generations observed to date. To further explore and quantify this suppression, B73, Mo17, as well as <em>Oy1-N1989/oy1</em>:B73, crossed with each of these inbred lines were grown to the V3 stage in the greenhouse. To improve upon our visual assessment of leaf color and provide quantitation, optical absorbance was measured using a Chlorophyll Content Meter-200 plus (CCM; Opti-Sciences, Inc), a hand-held LED-based instrument. CCM is predicted to strongly correlate with chlorophyll and carotenoid contents. To confirm that our rapid phenotyping with the CCM would accurately assess chlorophyll levels, we measured the absolute pigment levels using a UV-VIS spectrophotometer (destructive sampling) on the same leaves used for CCM measurements. The non-destructive CCM measurements and the absolute pigment contents displayed a strong positive correlation with a R2 value of 0.94 for chl<em>a</em>, chl<em>b</em>, and total chlorophyll (Figure S1). Given this high correlation of maize leaf greenness between the rapid measurement using CCM-200 plus instrument and absolute pigment contents quantified using UV-VIS spectrophotometer, we performed all chlorophyll measurements of <em>Oy1-N1989</em> enhancement discussed in later results using CCM values.</p><p id="p-28">In the greenhouse grown seedlings, chlorophyll accumulation was below the level of detection in the <em>Oy1-N1989</em> homozygotes using CCM and spectrophotometric method. In wild-type plants, CCM measurements were slightly higher in B73 than Mo17, but the spectrophotometric method did not identify any significant difference in the amount of chlorophyll <em>a</em> (chl<em>a</em>), chlorophyll <em>b</em> (chl<em>b</em>), total chlorophyll, or carotenoids between these two genotypes (Table S6). We detected a mild parent-of-origin-effect for both CCM and absolute amounts of chl<em>a</em>, chl<em>b</em>, total chlorophyll, and carotenoids in the wild-type siblings of our F1 crosses. These plants had slightly higher chlorophyll accumulation when B73 was used as the pollen parent (Table S6). However, no such effect of parent-of-origin was observed for the mutant heterozygotes (<em>Oy1-N1989/oy1</em>) and reciprocal hybrid combinations of crosses between <em>Oy1-N1989/oy1</em>:B73 and Mo17 were indistinguishable. Further, both CCM and absolute chlorophyll contents were higher in the <em>Oy1-N1989/oy1</em>:B73 plants compared to the mutants in B73 x Mo17 hybrid background. Thus, there was a strong effect of genetics on chlorophyll pigment variation in mutants, that went opposite to predictions for hybrid vigor.</p><p id="p-29">We tested the progenies from the crosses of <em>Oy1-N1989/oy1</em>:B73 with B73 and Mo17 inbred lines in the field. Consistent with our previous observation, mutant heterozygotes in the B73 inbred background were substantially greener than heterozygotes in the Mo17 x B73 F1 hybrid background (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, Table S7). The increased severity of <em>Oy1-N1989</em> heterozygotes in the Mo17 genetic background was stable across generations and observed even after six backcrosses (Figure S2). No suppressed mutant plants were observed during any generation of backcrossing into Mo17 (data not shown). Thus, the expected negative impact of the <em>Oy1-N1989</em> allele on chlorophyll pigment accumulation was dramatically suppressed by the B73 background compared to Mo17 genotype.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/375/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The chlorophyll pigment accumulation differs in the Oy1-N1989/oy1 heterozygotes in the B73 and B73 x Mo17 hybrid backgrounds. Representative wild-type and mutant siblings from (A) B73 x Oy1-N1989/oy1:B73, and (B) Mo17 x Oy1-N1989/oy1:B73 F1 crosses in the field-grown plants. Measuring stick in panels A and B is 243 cm. (C) Non-destructive chlorophyll approximation in mutant and wild-type siblings at an &#x223C;3 weeks (CCMI) and &#x223C;6 weeks (CCMII) after planting; data for each class of genotype is derived from 39 replications planted in RCBD. Asterisks indicate statistical significance between the means in each genotypic category at P &lt; 0.01 using student&#x2019;s t-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349837589" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The chlorophyll pigment accumulation differs in the &lt;em&gt;Oy1-N1989/oy1&lt;/em&gt; heterozygotes in the B73 and B73 x Mo17 hybrid backgrounds. Representative wild-type and mutant siblings from (A) B73 x &lt;em&gt;Oy1-N1989/oy1&lt;/em&gt;:B73, and (B) Mo17 x &lt;em&gt;Oy1-N1989/oy1&lt;/em&gt;:B73 F1 crosses in the field-grown plants. Measuring stick in panels A and B is 243 cm. (C) Non-destructive chlorophyll approximation in mutant and wild-type siblings at an &#x223C;3 weeks (CCMI) and &#x223C;6 weeks (CCMII) after planting; data for each class of genotype is derived from 39 replications planted in RCBD. Asterisks indicate statistical significance between the means in each genotypic category at &lt;em&gt;P&lt;/em&gt; &lt; 0.01 using student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/375/F1.medium.gif" width="440" height="136"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/2/375/F1.medium.gif" width="440" height="136"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/375/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/375/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492224" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">The chlorophyll pigment accumulation differs in the <em>Oy1-N1989/oy1</em> heterozygotes in the B73 and B73 x Mo17 hybrid backgrounds. Representative wild-type and mutant siblings from (A) B73 x <em>Oy1-N1989/oy1</em>:B73, and (B) Mo17 x <em>Oy1-N1989/oy1</em>:B73 F1 crosses in the field-grown plants. Measuring stick in panels A and B is 243 cm. (C) Non-destructive chlorophyll approximation in mutant and wild-type siblings at an ∼3 weeks (CCMI) and ∼6 weeks (CCMII) after planting; data for each class of genotype is derived from 39 replications planted in RCBD. Asterisks indicate statistical significance between the means in each genotypic category at <em>P</em> &lt; 0.01 using student’s <em>t</em>-test.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>vey1 maps to a single locus that co-segregates With the oy1 allele of Mo17 in DH, RIL, BC1F1 and NIL families derived From B73 and Mo17</h3><p id="p-31">To identify the genetic basis of the suppression of <em>Oy1-N1989</em> allele in B73, we performed a series of crosses to four mapping populations. In each case, we crossed pollen from heterozygous <em>Oy1-N1989</em> plants to a population of recombinant lines as ear-parents (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). We chose two public populations, IBM and Syn10, to map all modifiers altering the severity of the <em>Oy1-N1989</em> phenotypes. IBM and Syn10 differ in the number of rounds of intermating, and therefore vary in the number of recombinants captured and genetic resolution of trait localization (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lee <em>et al.</em> 2002</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Hussain <em>et al.</em> 2007</a>). Each F1 progeny of the testcross segregated approximately 1:1 for wild-type (<em>oy1/oy1</em>) and mutant heterozygote (<em>Oy1-N1989/oy1</em>) in the hybrid genetic background with B73. Mutant heterozygote siblings in both (IBM and Syn10) F1 populations, chlorophyll approximation using CCM measured at an early (CCMI) and late (CCMII) developmental stages displayed bimodal trait distributions (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3A</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">3B</a>; Figures S3 and S4), suggesting the presence of an allele of strong effect. The CCM distributions of the wild-type and mutant siblings did not overlap (Figures S5A and S5B). CCM values collected at both time points in the wild-type F1 siblings showed positive correlations in both F1 populations (Tables S8 and S9; Figures S3 and S4). A similar trend was also observed for the CCMI and CCMII in the mutant F1 siblings. The CCM measurements in the wild-type and mutant siblings did not display any significant correlation. To control for variation in CCM observed due to the genetic potential of each line that was independent of the <em>Oy1-N1989</em> modification, we divided the mutant CCM trait values by the congenic wild-type sibling CCM values to derive ratio for both time points. We also calculated differences between congenic wild-type and mutant CCM values. Each of the direct measurements, as well as the ratio-metric and difference values, were used as phenotypes to detect and localize QTL.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/375/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The crossing scheme used to map Oy1-N1989 enhancer/suppressor loci in IBM and Syn10 populations. Red, blue and white colors indicate B73, Mo17, and missing genotypes, respectively. The heterozygous tester shows chromosome 10 with a black spot indicating Oy1-N1989 mutant allele. F1 progenies depicted here shows hypothetical genotypes of chromosome 10 for each F1 wild-type and mutant siblings." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349837589" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The crossing scheme used to map &lt;em&gt;Oy1-N1989&lt;/em&gt; enhancer/suppressor loci in IBM and Syn10 populations. Red, blue and white colors indicate B73, Mo17, and missing genotypes, respectively. The heterozygous tester shows chromosome 10 with a black spot indicating &lt;em&gt;Oy1-N1989&lt;/em&gt; mutant allele. F1 progenies depicted here shows hypothetical genotypes of chromosome 10 for each F1 wild-type and mutant siblings.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/375/F2.medium.gif" width="440" height="195"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/2/375/F2.medium.gif" width="440" height="195"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/375/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/375/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492227" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">The crossing scheme used to map <em>Oy1-N1989</em> enhancer/suppressor loci in IBM and Syn10 populations. Red, blue and white colors indicate B73, Mo17, and missing genotypes, respectively. The heterozygous tester shows chromosome 10 with a black spot indicating <em>Oy1-N1989</em> mutant allele. F1 progenies depicted here shows hypothetical genotypes of chromosome 10 for each F1 wild-type and mutant siblings.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/375/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The phenotypic distribution, QTL analysis, and fine mapping results of MT_CCMII. Distribution of MT_CCMII in (A) Syn10 x Oy1-N1989/oy1:B73, and (B) IBM x Oy1-N1989/oy1:B73 F1 populations. (C) Genome wide QTL plot of MT_CCMII in Syn10 x Oy1-N1989/oy1:B73 F1 population. (D) Close-up view of the vey1 locus on chromosome 10. Black horizontal bar in panels C-D indicate permutation testing based threshold to declare statistical significance of the QTL. Recombinants detected at vey1 in F1 crosses of Oy1-N1989/oy1:B73 with (E) Syn10, B73 x Mo17 F1 (BC1F1), and (F) IBM. Number at a given marker and population intersection in (E) and (F) indicates the total number of recombinants between the respective marker genotype and observed MT_CCMII phenotype; hyphen denotes no genotyping. dCAPS marker at oy1 is highlighted in bold." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349837589" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The phenotypic distribution, QTL analysis, and fine mapping results of MT_CCMII. Distribution of MT_CCMII in (A) Syn10 x &lt;em&gt;Oy1-N1989/oy1&lt;/em&gt;:B73, and (B) IBM x &lt;em&gt;Oy1-N1989/oy1&lt;/em&gt;:B73 F1 populations. (C) Genome wide QTL plot of MT_CCMII in Syn10 x &lt;em&gt;Oy1-N1989/oy1&lt;/em&gt;:B73 F1 population. (D) Close-up view of the &lt;em&gt;vey1&lt;/em&gt; locus on chromosome 10. Black horizontal bar in panels C-D indicate permutation testing based threshold to declare statistical significance of the QTL. Recombinants detected at &lt;em&gt;vey1&lt;/em&gt; in F1 crosses of &lt;em&gt;Oy1-N1989/oy1&lt;/em&gt;:B73 with (E) Syn10, B73 x Mo17 F1 (BC1F1), and (F) IBM. Number at a given marker and population intersection in (E) and (F) indicates the total number of recombinants between the respective marker genotype and observed MT_CCMII phenotype; hyphen denotes no genotyping. dCAPS marker at &lt;em&gt;oy1&lt;/em&gt; is highlighted in bold.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/375/F3.medium.gif" width="419" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/2/375/F3.medium.gif" width="419" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/375/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/375/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492166" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">The phenotypic distribution, QTL analysis, and fine mapping results of MT_CCMII. Distribution of MT_CCMII in (A) Syn10 x <em>Oy1-N1989/oy1</em>:B73, and (B) IBM x <em>Oy1-N1989/oy1</em>:B73 F1 populations. (C) Genome wide QTL plot of MT_CCMII in Syn10 x <em>Oy1-N1989/oy1</em>:B73 F1 population. (D) Close-up view of the <em>vey1</em> locus on chromosome 10. Black horizontal bar in panels C-D indicate permutation testing based threshold to declare statistical significance of the QTL. Recombinants detected at <em>vey1</em> in F1 crosses of <em>Oy1-N1989/oy1</em>:B73 with (E) Syn10, B73 x Mo17 F1 (BC1F1), and (F) IBM. Number at a given marker and population intersection in (E) and (F) indicates the total number of recombinants between the respective marker genotype and observed MT_CCMII phenotype; hyphen denotes no genotyping. dCAPS marker at <em>oy1</em> is highlighted in bold.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Summary of the peak positions of all QTL passing a permutation computed threshold are presented in Table S10 for the IBM F1 populations and Table S11 for the Syn10 F1 populations. All mutant CCM traits, all mutant to wild-type CCM ratios, and all differences between mutant and wild-type CCM measurements identified a single QTL of large effect on chromosome 10. We name this suppressor locus <em>very oil yellow1</em> (<em>vey1</em>). A plot of the log10 of odds (LOD) score with permutation calculated threshold for CCMII from the mutant siblings in the Syn10 F1 population is plotted in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3C</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">3D</a>. Other mutant-related traits in Syn10 and IBM F1 populations produced similar plots (data not shown). One additional minor effect QTL was identified from these data, and it only influenced the early chlorophyll measurements in wild-type siblings of the IBM F1 population (Table S10). Interestingly, <em>vey1</em> mapped to the same genetic position as <em>oy1</em> locus itself (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figures 3E</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">3F</a>) and there was no influence of this locus on CCM traits in wild-type siblings. These analyses indicate that the <em>vey1</em> QTL encodes a single locus with an effect contingent upon the presence of the <em>Oy1-N1989</em> allele.</p><p id="p-35">The high penetrance of the MT_CCMII trait allowed us to interpret a discordance between the marker genotype and F1 mutant phenotype of high or low CCM categorization as recombination between <em>vey1</em> and the given marker. We classified each F1 mutant hybrid in the Syn10 and IBM F1 populations as high or low CCMII using the bimodal distribution. We then compared the marker genotypes at each marker linked to <em>vey1</em> with the phenotypic categories. Table S12 shows the mutant trait values, marker genotypes, and phenotypic categories for the nine F1 Syn10 lines with recombinants within the <em>vey1</em> region (between the flanking markers <em>10.90.5</em> and <em>10.95.5</em>). A summary of the recombinants between the markers and the phenotypic class at each marker position across the <em>vey1</em> region is presented in <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figures 3E</a> and <a id="xref-fig-3-8" class="xref-fig" href="#F3">3F</a>. Genotypes at marker <em>10.93</em> perfectly predicted CCMII trait expression in Syn10 F1 mutant siblings and recombinants placed this QTL within the ∼227 kb interval between markers <em>10.94.5</em> and <em>10.90.5</em> (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3E</a>). This <em>vey1</em> region includes the <em>oy1</em> locus itself, suggesting that the Mo17 allele of <em>oy1</em> may enhance the impact of the <em>Oy1-N1989</em> allele. Three Syn10 DH lines recombinant within this critical region were genotyped with additional markers developed at <em>ftcl1</em> (GRMZM2G001904), and <em>gfa2</em> (GRMZM2G118316). Three recombinants were detected between <em>vey1</em> and <em>ftcl1</em> but no recombinants were recovered between the <em>gfa2</em> indel marker and <em>vey1</em>. A SNP marker used in the Syn10 map construction (10.94.5) at the distal end of the <em>gfa2</em> locus identified one recombinant (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3E</a>). Thus, <em>vey1</em> is localized between <em>ftcl1</em> and <em>gfa2</em>. This region includes <em>oy1</em>, <em>ereb28</em>, a small region of <em>gfa2</em> (from marker <em>10.94.5</em>), and <em>ftcl1</em> proximate to <em>oy1</em>. A similar fine-mapping attempt with the IBM F1 population provided no additional resolution (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3F</a>). The genotyping of 576 BC1F1 individuals from (B73 x Mo17) x <em>Oy1-N1989/oy1</em>:B73 crosses failed to recover any recombinants between <em>vey1</em> and <em>gfa2</em> (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3E</a>).</p><p id="p-36">We further explored background influence on the <em>vey1</em> alleles from B73 and Mo17 using a series of BM-NILs that contained the <em>vey1</em> region introgressed into a homogeneous background of either B73 or Mo17. Progeny from crosses of BM-NILs with <em>Oy1-N1989/oy1</em>:B73 displayed the expected mutant CCM phenotype based on the <em>vey1</em> introgression regardless of the recurrent parent (Figure S6 and Table S3). These data confirm the expectation of the QTL mapping but offered no additional recombinants within the identified <em>vey1</em> QTL.</p><p id="p-37">Thus, the formal list of candidate genes for the <em>vey1</em> QTL is the <em>oy1</em> gene itself and the three most closely linked loci: <em>ftcl1</em>; <em>gfa2</em>; <em>ereb28</em>. The ortholog of <em>Zmftcl1</em> (∼62% protein identity) from <em>Arabidopsis thaliana</em> is required for folate metabolism (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Pribat <em>et al.</em> 2011</a>). The maize gene <em>gfa2</em> is uncharacterized, but mutation of the <em>Arabidopsis</em> ortholog caused defects in megagametogenesis including failures of polar nuclear fusion in the female gametophyte and synergid cell-death at fertilization (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Christensen <em>et al.</em> 1998</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">2002</a>). The third linked gene, <em>ereb28</em> (<em>Apetela2-Ethylene Responsive Element Binding Protein-transcription factor 28</em>) has a highly conserved AP2/EREB domain. This gene has a very low expression level and is localized only to the root tissue of maize (<a href="https://www.maizegdb.org/gene_center/gene?id=GRMZM2G544539#rnaseq">https://www.maizegdb.org/gene_center/gene?id=GRMZM2G544539#rnaseq</a>).</p></div><div id="sec-12" class="subsection"><h3>Controlling for the vey1 QTL did not reveal additional modifiers of Oy1-N1989 phenotype expression</h3><p id="p-38">The non-normality of some of the trait distributions and apparent thresholds prompted us to explore additional QTL models. No additional QTL were recovered by implementing two-part threshold models (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Broman <em>et al.</em> 2003</a>) for any of the traits (data not shown). Similarly, two-way genome scans also failed to detect any statistically significant genetic interactions with <em>vey1</em> or between other loci. In both the IBM and the Syn10, the region encoding <em>vey1</em> exhibited substantial segregation distortion with the B73:Mo17 alleles present at 120:72 in the IBM and 175:76 in the Syn10 population. This uneven sample size will reduce the power to detect epistasis with <em>vey1</em> but would not limit the detection of additional unlinked epistatic modifiers of <em>Oy1-N1989</em>.</p><p id="p-39">We used the top marker at <em>vey1</em> as a covariate to control for the contribution of this allele to phenotypic variation and performed a one-dimensional scan of the genome (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Broman <em>et al.</em> 2003</a>). In our previous naïve one-dimensional scans, the large effect of <em>vey1</em> partitioned into the error term and might reduce our power to detect additional unlinked QTL(s). By adding a marker linked to <em>vey1</em> as a covariate, this term will capture the variance explained by <em>vey1</em> and could improve detection of additional QTL(s) of presumably smaller effect. In both populations, use of a <em>vey1</em> linked marker as a covariate did not identify any additional QTL for any trait (data not shown). Thus, modification of the <em>Oy1-N1989</em> phenotype by <em>vey1</em> was inherited as a single QTL, acting alone.</p></div><div id="sec-13" class="subsection"><h3>GWAS for chlorophyll content in maize diversity lines (MDL) and Oy1-N1989/oy1 F1 genotypes identifies vey1</h3><p id="p-40">We undertook GWA mapping of <em>Oy1-N1989</em> severity to search for additional loci and potentially identify recombinants at <em>vey1</em>. A population of 343 lines was crossed to <em>Oy1-N1989/oy1</em>:B73. This generated MDL x <em>Oy1-N1989/oy</em>1:B73 F1 populations segregating 1:1 for mutant and wild-type siblings in hybrid genetic background. There was total separation between mutant and wild-type siblings for the CCMI and CCMII traits (Figure S5C). The mutant severity of some F1 families was similar to the F1 progeny of Mo17 x <em>Oy1-N1989/oy1</em>:B73 (Figure S7). Pairwise correlations of the CCM trait measurements at two time points (CCMI and CCMII) in the wild-type siblings displayed statistically significant positive relationship (Table S13). CCM traits were much more strongly correlated in the mutant F1 siblings, similar to the B73 x Mo17 F1 populations. However, weak positive correlations were also observed between mutant and wild-type CCM measurements in the MDL F1 populations. Broad-sense heritability estimates for CCMI and CCMII were high for the mutant and ratio traits, whereas wild-type siblings had much lower repeatability (Table S14).</p><p id="p-41">GWA was performed using the 305 inbred lines from MDL crosses that were present in the HapMap3 SNP data set (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Bukowski <em>et al.</em> 2018</a>). The variation in MT_CCMI, MT_CCMII, and their ratios identified a region encoding multiple SNPs that passed a multiple test correction (see Methods) on chromosome 10 at the site of the <em>oy1</em> gene. No other loci passed the corrected threshold in the GWA analysis. No statistically significant SNPs were detected for the CCM traits of wild-type siblings. The plot showing the negative log10 of the p-values from GWA tests for all the SNPs for MT_CCMII trait is graphed in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>. A closer view of the SNPs within the region encoding the <em>vey1</em> locus on chromosome 10 are plotted in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>. A summary of the GWAS results for mutant CCM and ratio traits is presented in Table S15. The top association for the mutant CCM traits was a SNP at position 9161643 on chromosome 10 (S10_9161643) located just 3’ of the <em>oy1</em> protein-coding sequence. S10_9161643 displays high allelic frequency in our population (f = 0.49). Thus, it appears that variation at the <em>oy1</em> locus may be responsible for the suppression of the <em>Oy1-N1989</em> mutant allele in the diverse panel of maize inbred lines analyzed in this experiment. Analysis of the LD between S10_9161643 and the other variants in this region identified no other variants with r2 greater than 0.5 despite the relatively strong associations between many SNPs and the CCM traits (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4E</a>). LD was substantially higher for SNPs encoded toward the telomere from <em>oy1</em> than toward the centromere, with a strong discontinuity of LD at the 3’-end of <em>oy1</em>. Given the relatively low p-values calculated for multiple SNPs in the area, this raises the possibility that multiple alleles contribute to the suppression of the <em>Oy1-N1989</em> phenotype. To test this hypothesis, the genotypes at SNP S10_9161643 were used as a covariate, and the genetic associations were recalculated. If SNPs segregate independently of S10_9161643 and contribute to <em>Oy1-N1989</em> suppression, the p-values of association test statistics should stay significant. On the contrary, those SNPs that have relatively low p-values due to linkage with S10_9161643 should become statistically insignificant in the covariate model. When these analyses were done, low-frequency variants at the 5’ end of the <em>oy1</em> locus were identified as the most significant SNPs and passed a chromosome-wide multiple test correction (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a> and <a id="xref-fig-4-5" class="xref-fig" href="#F4">4D</a>). This result suggests that there are multiple alleles capable of modifying the <em>Oy1-N1989</em> mutant phenotype in the MDL panel. The top SNP on chromosome 10 in the covariate model of MLM was detected at position 9179932 (S10_9179932), and the allele associated with <em>Oy1-N1989</em> suppression is a relatively low-frequency variant (f = 0.08). It remains formally possible that the SNP S10_9179932 is not causative and merely in LD with a causative polymorphism, and the second locus is fortuitously present in recombinant haplotypes. Analysis of LD of S10_9179932 with other SNPs in ∼500 kb window detected multiple SNPs of low allelic frequency that were in high LD (r2 ∼0.85) toward the 5’ end of <em>oy1</em> (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4F</a>). Consistent with the strong discontinuity of LD at 3’ end of <em>oy1</em> with S10_9161643, we observed discontinuity of LD with S10_9179932 at 5’ end of the <em>oy1</em> locus. The analyses suggest that S10_9161643 and S10_9179932 are not in LD with each other and can act independently. The MLM model that considered only these two SNPs accounted for ∼29% of the variation in CCMII measurements of the mutant siblings (Table S15).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/375/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Manhattan plots of SNP associations with MT_CCMII in MDL x Oy1-N1989/oy1:B73 F1 population. Genome-wide association of (A) MT_CCMII with no marker covariate, (B) MT_CCMII using S10_9161643 as a covariate. Close-up views of the SNP associations in the vey1 region on chromosome 10 for (C) Panel A, (D) Panel B. Arrows in panels A-D identify SNPs S10_9161643 and S10_9179932. Horizontal solid red and hashed red lines in panels A-D indicate the genome-wide Bonferroni cut-off at P &lt; 0.05, and hashed golden line in panels C-D is a chromosome-wide FDR-adjusted threshold at P &lt; 0.05. Linkage disequilibrium of all SNPs in a 450 kb region flanking oy1 with SNPs (E) S10_9161643, and (F) S10_9179932. Vertical lines in panels C-F from left to right represent genomic position of ftcl1, ereb28, oy1 (green), and gfa2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349837589" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The Manhattan plots of SNP associations with MT_CCMII in MDL x &lt;em&gt;Oy1-N1989/oy1&lt;/em&gt;:B73 F1 population. Genome-wide association of (A) MT_CCMII with no marker covariate, (B) MT_CCMII using S10_9161643 as a covariate. Close-up views of the SNP associations in the &lt;em&gt;vey1&lt;/em&gt; region on chromosome 10 for (C) Panel A, (D) Panel B. Arrows in panels A-D identify SNPs S10_9161643 and S10_9179932. Horizontal solid red and hashed red lines in panels A-D indicate the genome-wide Bonferroni cut-off at &lt;em&gt;P&lt;/em&gt; &lt; 0.05, and hashed golden line in panels C-D is a chromosome-wide FDR-adjusted threshold at &lt;em&gt;P&lt;/em&gt; &lt; 0.05. Linkage disequilibrium of all SNPs in a 450 kb region flanking &lt;em&gt;oy1&lt;/em&gt; with SNPs (E) S10_9161643, and (F) S10_9179932. Vertical lines in panels C-F from left to right represent genomic position of &lt;em&gt;ftcl1&lt;/em&gt;, &lt;em&gt;ereb28&lt;/em&gt;, &lt;em&gt;oy1&lt;/em&gt; (green), and &lt;em&gt;gfa2&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/375/F4.medium.gif" width="440" height="249"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/2/375/F4.medium.gif" width="440" height="249"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/375/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/375/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492264" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">The Manhattan plots of SNP associations with MT_CCMII in MDL x <em>Oy1-N1989/oy1</em>:B73 F1 population. Genome-wide association of (A) MT_CCMII with no marker covariate, (B) MT_CCMII using S10_9161643 as a covariate. Close-up views of the SNP associations in the <em>vey1</em> region on chromosome 10 for (C) Panel A, (D) Panel B. Arrows in panels A-D identify SNPs S10_9161643 and S10_9179932. Horizontal solid red and hashed red lines in panels A-D indicate the genome-wide Bonferroni cut-off at <em>P</em> &lt; 0.05, and hashed golden line in panels C-D is a chromosome-wide FDR-adjusted threshold at <em>P</em> &lt; 0.05. Linkage disequilibrium of all SNPs in a 450 kb region flanking <em>oy1</em> with SNPs (E) S10_9161643, and (F) S10_9179932. Vertical lines in panels C-F from left to right represent genomic position of <em>ftcl1</em>, <em>ereb28</em>, <em>oy1</em> (green), and <em>gfa2</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Given that the MLM model using S10_9161643 as a covariate detected S10_9179932 as the most significant association, we tested the phenotypic outcome of the four possible haplotypes at these two SNPs. We observed that the four haplotypes varied only for mutant CCM traits, with haplotypes AG and CA being the most favorable (highest CCM mean) and least favorable (lowest CCM mean), respectively (Table S16). Alleles at these two SNPs affected CCMI and CCMII in the mutant plants, consistent with additive inheritance for two polymorphisms. The additive effect of the SNPs is consistent with independent <em>cis</em>-regulatory alleles at <em>oy1</em> modifying the <em>Oy1-N1989</em> mutant phenotype. Consistent with the strong enhancement caused by crossing Mo17 to <em>Oy1-N1989/oy1</em>:B73, the Mo17 <em>oy1</em> locus encodes the most severe, and relatively rare, CA allele combination, and B73 encodes the most suppressing AG allele combination (Table S16). Thus, the line-cross mapping was performed with inbred lines that carry phenotypically extreme allele combinations of these two SNPs in the vicinity of the <em>oy1</em> locus.</p></div><div id="sec-14" class="subsection"><h3>Oy1-N1989 is a semi-dominant chlorophyll mutant and enhanced by reduced function at oy1</h3><p id="p-44">Maize seedlings that are double heterozygotes for <em>Oy1-N1989</em> and hypomorphic <em>oy1</em> allele (<em>chlI-MTM1</em>) are more severe than isogenic <em>Oy1-N1989/oy1</em> siblings (<a id="xref-ref-78-6" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> 2006</a>). To confirm that the reduced <em>oy1</em> function could determine the differential sensitivity to <em>Oy1-N1989</em>, we crossed dominant and recessive mutant alleles of <em>oy1</em> to each other. The recessive weak hypomorphic allele <em>oy1-yg</em> was obtained from the Maize COOP in the unknown genetic background. The homozygous <em>oy1-yg</em> plants were crossed as a pollen-parent with both B73 and Mo17 to develop F1 material that would segregate the mutation. The F1 plants were then crossed to <em>Oy1-N1989/oy1</em>:B73 as well as backcrossed to the <em>oy1-yg</em> homozygotes in the original mixed background. These crosses allowed us to recover plants that had <em>Oy1-N1989</em> in combination with the wild-type <em>oy1</em>-B73, wild-type <em>oy1</em>-Mo17, and mutant <em>oy1-yg</em> alleles. Chlorophyll contents were determined using CCM at 21 and 40 days after sowing in the field. The <em>Oy1-N1989</em> allele was substantially enhanced when combined with the <em>oy1-yg</em> allele, demonstrating that reduced function of the <em>oy1</em> allele in Mo17 could be the genetic basis of <em>vey1</em> QTL (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figures 5A</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">5B</a>). A summary of these data are presented in Table S17. This result is similar to the one described by <a id="xref-ref-78-7" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> (2006)</a>, where a reduction in chlorophyll content was observed when <em>Oy1-N1989</em> allele was combined with a recessive allele of <em>oy1</em>. We also noticed a similar drop in chlorophyll accumulation in the <em>oy1-yg</em> homozygotes as oppose to <em>oy1-yg</em> heterozygotes with wild-type <em>oy1</em> allele from both B73 and Mo17 in the BC1F1 progenies (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a> and Table S17). However, we did not observe any significant difference in the <em>oy1-yg</em> heterozygotes with B73 and Mo17 wild-type <em>oy1</em> allele. Selfed progeny from <em>Oy1-N1989</em> heterozygotes segregated for yellow-seedling lethal <em>Oy1-N1989</em> homozygotes with no detectable chlorophyll by either CCM or spectrophotometer quantification (Table S6). Therefore, consistent with previous work (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Hansson <em>et al.</em> 2002</a>; <a id="xref-ref-78-8" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> 2006</a>), the <em>Oy1-N1989</em> is a dominant-negative neomorphic mutant allele with no evident MgChl activity under the tested conditions. Based on these genetic data, any QTL resulting in decreased expression of <em>oy1</em> or an increased proportion of mutant to wild-type gene product in the <em>Oy1-N1989/oy1</em> heterozygotes can increase the severity of the mutant phenotype.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/375/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The single locus test of oy1 showing the interaction between wild-type alleles of oy1 from B73 and Mo17 with the semi-dominant (Oy1-N1989), and recessive mutant allele (oy1-yg) at oy1. (A) Mutant (two severity groups) and wild-type individuals segregating in a cross (B73 x oy1-yg/oy1-yg) x Oy1-N1989/+:B73. White-fill arrows indicate Oy1-N1989/+ plants (pale-green and suppressed), whereas yellow-fill arrows indicate Oy1-N1989/oy1-yg (yellow-green and severe) plants. CCM measurements of the test crosses at 21 and 40 days after planting in the (B) Mutant siblings (Oy1-N1989/oy1-yg and Oy1-N1989/+) of (Mo17 x oy1-yg/oy1-yg) x Oy1-N1989/+:B73, and (B73 x oy1-yg/oy1-yg) x Oy1-N1989/+:B73 crosses, (C) Mutant (oy1-yg/oy1-yg) and wild-type (oy1-yg/+) siblings of (Mo17 x oy1-yg/oy1-yg) x oy1-yg/ oy1-yg and (B73 x oy1-yg/oy1-yg) x oy1-yg/ oy1-yg crosses. Asterisk in panels B-C indicate statistically significant difference between the genotypes in a given cross at P &lt; 0.01 using student&#x2019;s t-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349837589" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The single locus test of &lt;em&gt;oy1&lt;/em&gt; showing the interaction between wild-type alleles of &lt;em&gt;oy1&lt;/em&gt; from B73 and Mo17 with the semi-dominant (&lt;em&gt;Oy1-N1989&lt;/em&gt;), and recessive mutant allele (&lt;em&gt;oy1-yg&lt;/em&gt;) at &lt;em&gt;oy1&lt;/em&gt;. (A) Mutant (two severity groups) and wild-type individuals segregating in a cross (B73 x &lt;em&gt;oy1-yg/oy1-yg&lt;/em&gt;) x &lt;em&gt;Oy1-N1989/+&lt;/em&gt;:B73. White-fill arrows indicate &lt;em&gt;Oy1-N1989/+&lt;/em&gt; plants (pale-green and suppressed), whereas yellow-fill arrows indicate &lt;em&gt;Oy1-N1989/oy1-yg&lt;/em&gt; (yellow-green and severe) plants. CCM measurements of the test crosses at 21 and 40 days after planting in the (B) Mutant siblings (&lt;em&gt;Oy1-N1989/oy1-yg&lt;/em&gt; and &lt;em&gt;Oy1-N1989/+&lt;/em&gt;) of (Mo17 x &lt;em&gt;oy1-yg/oy1-yg&lt;/em&gt;) x &lt;em&gt;Oy1-N1989/+&lt;/em&gt;:B73, and (B73 x &lt;em&gt;oy1-yg/oy1-yg&lt;/em&gt;) x &lt;em&gt;Oy1-N1989/+&lt;/em&gt;:B73 crosses, (C) Mutant (&lt;em&gt;oy1-yg/oy1-yg&lt;/em&gt;) and wild-type (&lt;em&gt;oy1-yg/+&lt;/em&gt;) siblings of (Mo17 x &lt;em&gt;oy1-yg/oy1-yg&lt;/em&gt;) x &lt;em&gt;oy1-yg/ oy1-yg&lt;/em&gt; and (B73 x &lt;em&gt;oy1-yg/oy1-yg&lt;/em&gt;) x &lt;em&gt;oy1-yg/ oy1-yg&lt;/em&gt; crosses. Asterisk in panels B-C indicate statistically significant difference between the genotypes in a given cross at &lt;em&gt;P&lt;/em&gt; &lt; 0.01 using student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/375/F5.medium.gif" width="440" height="103"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/2/375/F5.medium.gif" width="440" height="103"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/375/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/375/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492231" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-45" class="first-child">The single locus test of <em>oy1</em> showing the interaction between wild-type alleles of <em>oy1</em> from B73 and Mo17 with the semi-dominant (<em>Oy1-N1989</em>), and recessive mutant allele (<em>oy1-yg</em>) at <em>oy1</em>. (A) Mutant (two severity groups) and wild-type individuals segregating in a cross (B73 x <em>oy1-yg/oy1-yg</em>) x <em>Oy1-N1989/+</em>:B73. White-fill arrows indicate <em>Oy1-N1989/+</em> plants (pale-green and suppressed), whereas yellow-fill arrows indicate <em>Oy1-N1989/oy1-yg</em> (yellow-green and severe) plants. CCM measurements of the test crosses at 21 and 40 days after planting in the (B) Mutant siblings (<em>Oy1-N1989/oy1-yg</em> and <em>Oy1-N1989/+</em>) of (Mo17 x <em>oy1-yg/oy1-yg</em>) x <em>Oy1-N1989/+</em>:B73, and (B73 x <em>oy1-yg/oy1-yg</em>) x <em>Oy1-N1989/+</em>:B73 crosses, (C) Mutant (<em>oy1-yg/oy1-yg</em>) and wild-type (<em>oy1-yg/+</em>) siblings of (Mo17 x <em>oy1-yg/oy1-yg</em>) x <em>oy1-yg/ oy1-yg</em> and (B73 x <em>oy1-yg/oy1-yg</em>) x <em>oy1-yg/ oy1-yg</em> crosses. Asterisk in panels B-C indicate statistically significant difference between the genotypes in a given cross at <em>P</em> &lt; 0.01 using student’s <em>t</em>-test.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>No coding sequence difference in OY1 accounts for vey1 inheritance</h3><p id="p-46">Our reliance on SNP variation leaves us open to the problem that linked, but the unknown non-SNP variation can be responsible for <em>vey1</em>. Given that reduced <em>oy1</em> activity enhanced the phenotype of <em>Oy1-N1989</em>, we sequenced the <em>oy1</em> locus from Mo17 and B73 to determine if coding sequence differences could encode the <em>vey1</em> modifier. The only non-synonymous changes that distinguish these two alleles is at the site of the previously reported in-frame 6 bp insertion (<a id="xref-ref-78-9" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> 2006</a>), which adds alanine (A) and threonine (T) amino acid residues to the OY1 protein. PCR amplification of <em>oy1</em> locus in 18 maize inbred lines, as well as the <em>Oy1-N1989</em> allele, was performed. Sequencing of the PCR products confirmed the absence of the 6 bp insertion in <em>Oy1-N1989</em> allele reported by <a id="xref-ref-78-10" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> (2006)</a>. In addition, multiple inbred lines including B73, CML103, and CML322 also carried this 6 bp in-frame deletion. A polymorphism within the 6 bp insertion was also found that resulted in an alternative in-frame insertion encoding an alanine and serine (S) codon in Mo17 and five other inbred lines. Thus, three alleles at this site were found to be a common variant in OY1 gene product. These allelic states of <em>oy1</em> did not explain the phenotypic severity of CCM trait value in the F1 mutant siblings (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The allelic state of <em>oy1</em> at this polymorphic site in 18 maize inbred lines and the average CCM trait values in the wild-type and mutant siblings of their respective F1 progenies with <em>Oy1-N1989/oy1</em>:B73 are summarized in Table S18. Five inbred lines, including Mo17, resulted in dramatic enhancement of the mutant CCMI and ratio of CCMI phenotypes of F1 plants crossed to <em>Oy1-N1989/oy1</em>:B73. These enhanced genotypes encoded all three possible alleles at <em>oy1</em>. In addition, the suppressing inbred lines also encoded all three possible alleles. Besides this 6 bp indel, three inbred lines had few more variants in OY1 protein. An enhancing inbred line CML322 had two missense mutations that lead to amino acid change at position 321 (D-&gt;E), and 374 (S-&gt;I). A suppressing inbred line NC358 had one amino acid change at position 336 (D-&gt;G) and the enhancing inbred Tzi8 had a 15 bp in-frame deletion leading to the removal of five amino acids (VMGPE) in the third exon of the coding sequence. Even considering the additional alleles at <em>oy1</em> found in few maize inbred lines, these results suggest that the only coding sequence polymorphism at <em>oy1</em> between B73 and Mo17 could not be the genetic basis of <em>vey1</em>. This result leaves the two additive top SNPs <em>in cis</em> with <em>oy1</em> as the most likely cause of <em>cis</em>-acting regulatory variation.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/375/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distributions of CCM trait measurements in the F1 progenies of a sub-set of maize inbred lines crossed with Oy1-N1989/oy1:B73 divided using three allelic variants in the oy1 coding sequence of the respective inbred line. Phenotypic distribution of (A) MT_CCMI, (B) WT_CCMI, (C) MT_CCMII, and (D) WT_CCMII. Symbols &#x201C;-&#x201C;, &#x201C;AS&#x201D;, and &#x201C;AT&#x201D; on the X-axis of each panel denote deletion of 6 base pairs (bp), insertion of amino acid residues Alanine-Serine (AS), and Alanine-Threonine (AT), respectively, in a given inbred line. Three inbred lines including B73 carried &#x201C;-&#x201C; allele, six inbred lines carried &#x201C;AS&#x201D; insertion, nine inbred lines carried &#x201C;AT&#x201D; insertion. No statistically significant differences were found among three categories using ANOVA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349837589" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distributions of CCM trait measurements in the F1 progenies of a sub-set of maize inbred lines crossed with &lt;em&gt;Oy1-N1989/oy1&lt;/em&gt;:B73 divided using three allelic variants in the &lt;em&gt;oy1&lt;/em&gt; coding sequence of the respective inbred line. Phenotypic distribution of (A) MT_CCMI, (B) WT_CCMI, (C) MT_CCMII, and (D) WT_CCMII. Symbols &#x201C;-&#x201C;, &#x201C;AS&#x201D;, and &#x201C;AT&#x201D; on the X-axis of each panel denote deletion of 6 base pairs (bp), insertion of amino acid residues Alanine-Serine (AS), and Alanine-Threonine (AT), respectively, in a given inbred line. Three inbred lines including B73 carried &#x201C;-&#x201C; allele, six inbred lines carried &#x201C;AS&#x201D; insertion, nine inbred lines carried &#x201C;AT&#x201D; insertion. No statistically significant differences were found among three categories using ANOVA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/375/F6.medium.gif" width="440" height="368"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/2/375/F6.medium.gif" width="440" height="368"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/375/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/375/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492162" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-47" class="first-child">The distributions of CCM trait measurements in the F1 progenies of a sub-set of maize inbred lines crossed with <em>Oy1-N1989/oy1</em>:B73 divided using three allelic variants in the <em>oy1</em> coding sequence of the respective inbred line. Phenotypic distribution of (A) MT_CCMI, (B) WT_CCMI, (C) MT_CCMII, and (D) WT_CCMII. Symbols “-“, “AS”, and “AT” on the X-axis of each panel denote deletion of 6 base pairs (bp), insertion of amino acid residues Alanine-Serine (AS), and Alanine-Threonine (AT), respectively, in a given inbred line. Three inbred lines including B73 carried “-“ allele, six inbred lines carried “AS” insertion, nine inbred lines carried “AT” insertion. No statistically significant differences were found among three categories using ANOVA.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Expression level polymorphism at oy1 co-segregates with suppression of Oy1-N1989 mutant phenotype</h3><p id="p-48">Measurements of mRNA accumulation from <em>oy1</em> in the IBM was publicly available in a previously published study (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2013</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">2018</a>). Out of 105 IBM lines that were assessed for expression level in the past study, 74 were among those tested for chlorophyll accumulation in the <em>Oy1-N1989</em> F1 hybrid populations. Using a genetic marker linked to the <em>oy1</em> locus, we determined that a <em>cis</em>-acting eQTL controlled the accumulation of OY1 transcripts in IBM shoot apex (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). A summary of these data are presented in Table S19. This <em>cis</em>-acting eQTL conditioned greater expression of the B73 allele and explained 19% variation in OY1 transcript abundance in the IBM. Given the enhancement of <em>Oy1-N1989</em> by the <em>oy1-yg</em> allele, a lower expression of the wild-type <em>oy1</em> allele from Mo17 is expected to enhance the phenotype of <em>Oy1-N1989</em> (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). In addition, OY1 RPKM values obtained from the shoot apex of IBM were able to predict the CCM trait values in the mutant but not the wild-type siblings in IBM F1 population with <em>Oy1-N1989/oy1</em>:B73 (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). This result suggests that inbred lines with increased MgChl subunit I transcripts available for protein production and MgChl complex assembly could overcome chlorophyll accumulation defects caused by the <em>Oy1-N1989/oy1</em> genotype in IBM F1 hybrids. Consistent with this, a full linear regression model that included both the isu085b marker (<em>cis</em>-eQTL) genotypes and the residual variation in RPKM at OY1 did a better job in predicting CCMI and CCMII in the IBM mutant F1 hybrids than the isu085b marker by itself. If the <em>cis</em>-eQTL at <em>oy1</em>, which results in differential accumulation of OY1 transcripts in the IBM inbred lines can affect allele-specific expression in the F1 hybrids, it could explain the better performance of the IBM mutant F1 hybrids with the B73 allele at <em>vey1</em>.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/375/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of OY1 in the shoot apices of 14 days old IBM seedlings co-segregates with marker linked to vey1. (A) Distribution of OY1 RPKM values (X-axis) in the IBM RILs using the marker genotype at isu085b (linked to oy1 and vey1). Asterisk indicate significant difference in the mean between two groups using student&#x2019;s t-test at P &lt; 0.01. Linear regression of OY1 expression in IBM RILs on CCMII in the (B) Wild-type and (C) Mutant siblings derived from IBM x Oy1-N1989/oy1:B73 crosses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349837589" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression of OY1 in the shoot apices of 14 days old IBM seedlings co-segregates with marker linked to &lt;em&gt;vey1&lt;/em&gt;. (A) Distribution of OY1 RPKM values (X-axis) in the IBM RILs using the marker genotype at &lt;em&gt;isu085b&lt;/em&gt; (linked to &lt;em&gt;oy1&lt;/em&gt; and &lt;em&gt;vey1&lt;/em&gt;). Asterisk indicate significant difference in the mean between two groups using student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test at &lt;em&gt;P&lt;/em&gt; &lt; 0.01. Linear regression of OY1 expression in IBM RILs on CCMII in the (B) Wild-type and (C) Mutant siblings derived from IBM x &lt;em&gt;Oy1-N1989/oy1&lt;/em&gt;:B73 crosses.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/375/F7.medium.gif" width="440" height="135"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/2/375/F7.medium.gif" width="440" height="135"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/375/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/375/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492159" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-49" class="first-child">Expression of OY1 in the shoot apices of 14 days old IBM seedlings co-segregates with marker linked to <em>vey1</em>. (A) Distribution of OY1 RPKM values (X-axis) in the IBM RILs using the marker genotype at <em>isu085b</em> (linked to <em>oy1</em> and <em>vey1</em>). Asterisk indicate significant difference in the mean between two groups using student’s <em>t</em>-test at <em>P</em> &lt; 0.01. Linear regression of OY1 expression in IBM RILs on CCMII in the (B) Wild-type and (C) Mutant siblings derived from IBM x <em>Oy1-N1989/oy1</em>:B73 crosses.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">A previous study of allele-specific expression in the F1 hybrid maize seedlings identified expression bias at <em>oy1</em> toward B73 in the hybrid combinations of B73 inbred line with PH207 and Mo17 but not Oh43 (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Waters <em>et al.</em> 2017</a>). We used two SNP positions, SNP_252 and SNP_317, to explore the allele-specific expression of OY1 in our materials. SNP_252 is the causative polymorphism for the <em>Oy1-N1989</em> missense allele while SNP_317 is polymorphic between B73 and Mo17, but monomorphic between <em>Oy1-N1989</em> and B73. As the original allele of the <em>Oy1-N1989</em> mutation was isolated from a <em>r1 c1</em> colorless synthetic stock of mixed parentage, this raises the possibility that the same <em>cis</em>-acting regulatory variation that lowered expression of OY1 from PH207 and Mo17 when combined with the B73 allele might also be present in the <em>oy1</em> allele that was the progenitor of <em>Oy1-N1989</em>. We tested this possibility by using the SNPs that distinguish B73, Mo17, and the <em>Oy1-N1989</em> alleles to measure allele-specific expression in each of the hybrids. Consistent with the previous data (<a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Waters <em>et al.</em> 2017</a>), we observed a biased expression at <em>oy1</em> toward the B73 allele in the B73 x Mo17 F1 wild-type hybrids (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Extended data from this experiment is provided in Table S20. In the B73 isogenic crosses, transcripts from the <em>Oy1-N1989</em> and B73 wild-type alleles accumulated to equal levels in the heterozygotes, indicating that the suppressed phenotype of the mutants in B73 background was not due to a lowered expression of <em>Oy1-N1989</em> relative to the wild-type allele. Remarkably, mutant siblings from the reciprocal crosses between <em>Oy1-N1989/oy1</em>:B73 and Mo17 resulted in greater expression from the <em>Oy1-N1989</em> allele than the wild-type <em>oy1</em> allele of Mo17. Allele-specific bias at <em>oy1</em> was significantly higher toward the <em>Oy1-N1989</em> allele in the <em>Oy1-N1989</em> mutant heterozygotes in the B73 x Mo17 hybrid background compared to B73 isogenic material. Thus, in mutant hybrids, overexpression of <em>Oy1-N1989</em> relative to the wild-type <em>oy1</em> allele in Mo17 could account for increased phenotypic severity.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492078/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492078/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The allele-specific expression at <em>oy1</em> in the top fully-expanded leaf at the V3 developmental stage of B73 x Mo17 F1 wild-type and <em>Oy1-N1989/oy1</em> mutant siblings, and inbred <em>Oy1-N1989/oy1</em>:B73 mutants</span><div class="sb-div caption-clear"></div></div></div><p id="p-53">If <em>vey1</em> is encoded by an eQTL, then PH207 should encode an enhancing allele and Oh43 should encode a suppressing allele of <em>vey1</em>. We tested this genetically by producing F1 progenies in crosses of PH207 and Oh43 by <em>Oy1-N1989/oy1</em>:B73 pollen. Oh43 was evaluated in our initial screening and also as part of the MDL panel used for GWAS. In both experiments, the F1 hybrids between Oh43 and <em>Oy1-N1989/oy1</em>:B73 suppressed the mutant phenotype, suggesting that Oh43 is a suppressing inbred line (CCM values in Tables S4 and S18). B73 x PH207 F1 hybrids were missing from our previous datasets. We crossed PH207 ears with pollen from <em>Oy1-N1989/oy1</em>:B73 plants. The F1 hybrids from this cross were analyzed in the greenhouse at seedlings stage along with F1 hybrids of B73 x <em>Oy1-N1989/oy1</em>:B73 and Mo17 x <em>Oy1-N1989/oy1</em>:B73 F1 progenies as controls. PH207 was an enhancing inbred genotype as mutant heterozygotes in a PH207 x B73 F1 genetic background accumulated less chlorophyll than mutants in the B73 isogenic background (Figure S8 and Table S21).</p><p id="p-54">We further leveraged the normalized expression data of OY1 in the emerging shoot tissue of the maize diversity lines (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Kremling <em>et al.</em> 2018</a>) and used the top two additive SNPs (S10_9161643 and S10_9179932) at <em>vey1</em> from GWAS to test if these <em>cis</em>-variants of <em>oy1</em> affect its expression. Alleles that suppress <em>Oy1-N1989</em> linked to either SNP were associated with the greater abundance of OY1 transcripts. Plants carrying the B73-like allele combination (A and G at marker S10_9161643 and S10_9179932, respectively) showed the highest OY1 abundance in the emerging shoots of diverse maize inbred lines, whereas, plants with the Mo17-like allele combination (C and A at S10_9161643 and S10_9179932, respectively) showed lowest OY1 count (Table S22). Consistent with the additive suppression of leaf greenness in <em>Oy1-N1989</em> mutants by the alleles at S10_9161643 and S10_9179932 as discussed previously (Table S16), these alleles were also additive for their impacts on OY1 transcript abundance (Table S22). This observation is consistent with the hypothesis that increased OY1 abundance can overcome the negative effect of the <em>Oy1-N1989</em> allele. It is likely that multiple phenotypically-affective <em>cis</em>-acting regulatory polymorphisms at <em>oy1</em> are responsible for the <em>vey1</em> locus.</p><p id="p-55">The effect sizes of the gene expression changes observed in the IBM, diverse maize inbred lines, and allele-specific expression in hybrids are quite modest, resulting in ∼10% of differences in <em>oy1</em> accumulation. If these changes in wild-type OY1 transcript accumulation are responsible for suppression of <em>Oy1-N1989</em>, then the severity of the mutant phenotype (as indicated by CCM) in the MDL F1 population should correlate with OY1 expression level. As expected, we observed a statistically significant positive correlation between the mutant derived CCM traits and OY1 counts (Table S23). Wild-type CCM in the MDL F1 population did not show any significant correlation with OY1 abundance in the emerging shoot tissue of maize inbred lines. These correlations are in agreement with the lack of any QTL at this locus controlling wild-type chlorophyll levels, and the epistatic relationship between <em>vey1</em> and <em>Oy1-N1989</em>.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-56">The semi-dominant mutant allele <em>Oy1-N1989</em> encodes a dominant-negative allele at the <em>oy1</em> locus (<a id="xref-ref-78-11" class="xref-bibr" href="#ref-78">Sawers <em>et al.</em> 2006</a>). In a heterozygous condition, the strength of the negative effect of this allele on the MgChl enzyme complex depends on the wild-type <em>oy1</em> allele. Thus, the <em>Oy1-N1989</em> allele can sensitize maize plants to variation in MgChl and expose a phenotypic consequence for genetic variants that are otherwise invisible. Similar methodology has been adopted previously in maize to gain the genetic understanding of various traits (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chintamanani <em>et al.</em> 2010</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Olukolu <em>et al.</em> 2013</a>, <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">2014</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Buescher <em>et al.</em> 2014</a>). Thus, genetic screens based upon semi-dominant mutant alleles as reporters offer a cost-effective and robust approach to map QTL(s) for metabolic pathways of interest by leveraging the publicly available germplasm.</p><p id="p-57">Alleles with dramatic fitness consequences may be visible to researchers working with population sizes in the thousands, such as in GWAS. By contrast, evolutionarily-relevant genetic variation may have minimal phenotypic effects in laboratory and agronomic field conditions. The cryptic variation observed by these mutant-contingent QTL approaches may not be fitness-affecting, and interpreting mutant-conditioned phenotypes as non-neutral variation would be a mistake. It is, of course, possible that neutral variants may result in increased severity of a mutant phenotype due to changes not physiologically relevant for all alleles of that reporter gene present in a species. Nevertheless, it can inform us about the allelic variation in the species and pathway topology via gene discovery.</p><p id="p-58">In the current study, use of bi-parental mapping populations derived from the same inbred lines but developed using different breeding schemes provided the opportunity to compare the effect of additional rounds of random interbreeding in the development of mapping population on the genetic resolution. The comparative fine mapping of <em>vey1</em> in the Syn10 population that employed ten rounds of random mating provided far better localization of the <em>vey1</em> QTL than the IBM populations that was derived from four rounds of random mating. This observation demonstrates the benefits of increased recombination during random intermating of early generations in QTL localization. Based on these results, as future RILs are generated, we recommend increased intermating in the early generations followed by DH induction rather than relying on further recombination during the self-pollination cycles of RIL development.</p><p id="p-59">This study illustrated a complementary use of line-cross QTL mapping and GWAS to explore trait genetics. As expected, GWAS provided a fine-scale genetic resolution and corroborated the mapping of <em>vey1</em>. GWAS identified two SNPs, one in the 5’ and another in the 3’ intergenic DNA, proximate to <em>oy1</em> that represent candidate quantitative trait nucleotides (QTN). The signal detection by multiple unlinked SNPs in GWAS may indicate a complex set of phenotypically-affective alleles at the <em>oy1</em> locus. Alternatively, it could be an artifact of strong associations with markers that are tightly linked to an unmeasured causative variant but unlinked or in repulsion to each other. Consistent with two QTN, rather than fortuitous linkage of tag SNPs with a single causative polymorphism, alleles at the two SNPs additively influenced chlorophyll contents in the mutant siblings. Future work will be required to identify the nucleotide changes sufficient to reproduce the <em>vey1</em> phenotype.</p><p id="p-60">We used a non-destructive, inexpensive, and rapid phenotyping method to approximate leaf chlorophyll with high accuracy. Previous studies have highlighted the importance of benchmarking indirect measurements or proxies for traits of interest. For instance, near infrared reflectance spectroscopy (NIRS) that estimates major and total carotenoids in maize kernels could replace sensitive, accurate, cumbersome, expensive, slow, and destructive measurements by HPLC (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Berardo <em>et al.</em> 2004</a>). Comparisons of HPLC and NIRS measurements resulted in a correlation of 0.85 for total carotenoids (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Berardo <em>et al.</em> 2004</a>). A subjective visual score for yellow color in maize kernels was not adequate, yielding a correlation of 0.12 with HPLC measurements of total carotenoids (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Harjes <em>et al.</em> 2008</a>). Thus, using mutant alleles as reporters to develop methodologies that rely on non-invasive multispectral data as proxies for absolute quantification of biochemical compounds will accelerate studies on gene function and allele discovery. This can enable genetic studies that are currently deemed unfeasible due to the arduous task of phenotyping large populations for traits only visible in the laboratory.</p><div id="sec-18" class="subsection"><h3>How could a 10% change in wild-type OY1 expression affect chlorophyll biosynthesis in the Oy1-N1989/oy1 mutant heterozygotes?</h3><p id="p-61">Magnesium chelatase (MgChl) is formed by a trimer of dimers of MgChl subunit I interacting with the other subunits of MgChl complex. A previous study found that addition of mutant or wild-type BCHI protein to pre-assembled MgChl complexes resulted in altered reaction rates due to differences in subunit turnover, which occurred on a minutes time-scale (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Lundqvist <em>et al.</em> 2013</a>). This subunit turnover and reformation of the complex dynamically exchange mutant and wild-type BCHI subunits over time. Therefore, any net increase in the amount of wild-type OY1 in the reaction pool, for instance, due to higher transcription of wild-type <em>oy1</em> allele will allow a higher rate of magnesium chelatase activity and result in more chlorophyll biosynthesis. The observation of stronger affinity and greater dissociation rate of BCHI<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-3.gif"/></noscript></span></span> subunits (orthologous to the L176F change encoded by <em>Oy1-N1989</em>) for the wild-type subunits (<a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Hansson <em>et al.</em> 2002</a>) suggests that exchange of BCHI monomeric units in the magnesium chelatase complex might also differ based on the structure of BCHI protein (<a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Lundqvist <em>et al.</em> 2013</a>). In the AAA+ protein family, the ATP-binding site is located at the interface of two neighboring subunits in the oligomeric complex (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Vale 2000</a>). Since a dimer of functional MgChl subunit I proteins are required for the complex to carry out the MgChl activity (<a id="xref-ref-54-6" class="xref-bibr" href="#ref-54">Lundqvist <em>et al.</em> 2013</a>), approximately 1 in 3 dimers of assembled MgChl subunit I will be active in a 1:1 mixture of wild-type and BCHI<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-4.gif"/></noscript></span></span> Indeed, complexes made from reaction mixtures with equal proportions of wild-type and BCHI<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-5.gif"/></noscript></span></span> subunits resulted in ∼26% of the enzyme activity of an equivalent all-wildtype mix (<a id="xref-ref-54-7" class="xref-bibr" href="#ref-54">Lundqvist <em>et al.</em> 2013</a>). Therefore, we expect that decreasing expression of the wild-type <em>oy1</em> subunit by 10% and creating a 0.9:1.1 mixture of wild-type OY1 and mutant OY1-N1989 protein, respectively, would result in ∼21% activity compared to the activity of all-wildtype mixture. Likewise, increasing the wild-type <em>oy1</em> expression by 10% would result in ∼30% activity of MgChl compared to the all-wildtype mixture. This dosage-sensitivity is a general feature of protein complexes (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Birchler and Newton 1981</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Grossniklaus <em>et al.</em> 1996</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Veitia 2003</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Birchler and Veitia 2012</a>), and the semi-dominant nature of <em>Oy1-N1989</em> is dosage sensitive. Taking these observations and proposed models on the dynamics of molecular interaction between the wild-type and mutant BCHI protein subunits (especially BCHI<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/375/embed/mml-math-6.gif"/></noscript></span></span>) into account, it is formally possible that a small change in the expression of wild-type OY1 can have a significant impact on the magnesium chelatase activity of heterozygous <em>Oy1-N1989/oy1</em> plants. The increase in magnesium chelatase activity due to the even small relative increase in the proportion of wild-type OY1 transcripts over the mutant OY1-N1989 transcripts will read out as a proportional, presumably non-linear, increase in chlorophyll accumulation.</p></div><div id="sec-19" class="subsection"><h3>What is vey1?</h3><p id="p-62">Variation at the <em>vey1</em> locus appears to be the result of allelic diversity linked to the <em>oy1</em> locus. The only remaining possibilities are regulatory polymorphisms within the <em>cis</em>-acting control regions of <em>oy1</em>. Previous studies have utilized reciprocal test-crosses to loss-of-function alleles in multiple genetic backgrounds to provide single-locus tests of additive QTL alleles in an otherwise identical hybrid background (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dilkes <em>et al.</em> 2008</a>). Protein-null alleles of <em>oy1</em> isolated directly from the B73 and Mo17 backgrounds could be used to carry out a similar test. The intergenic genomic region in maize is spanned by transposable elements and can be highly divergent between different inbred lines due to large insertion/deletion polymorphisms (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">SanMiguel and Bennetzen 1998</a>). Consistent with this, inbreds B73 and Mo17 are polymorphic at the region between <em>oy1</em> and <em>gfa2</em>. These two maize inbred lines share ∼12 kb of sequence interspersed with numerous large insertions and deletions that add ∼139 kb of DNA sequence to Mo17 as compared to B73 (data not shown). However, we did not find any conserved non-coding sequence (CNS) in this region (data not shown). It is conceivable that recombinants at <em>oy1</em> between B73 and Mo17 themselves could be identified and used to test the effects of upstream or downstream regulatory sequences. The recombinant haplotypes encoding all four possible alleles at the top two SNPs identified in the GWAS indicate that Mo17 may be a strong enhancer due to more than one causative polymorphism. Allele-specific expression at <em>oy1</em> locus in the <em>Oy1-N1989</em> heterozygous mutants demonstrated the existence of functional <em>cis</em>-acting regulatory polymorphism between <em>Oy1-N1989</em> and both wild-type <em>oy1</em> alleles in B73 and Mo17. In addition, <em>oy1</em> is affected by a <em>cis</em>-eQTL in the IBM and MDL. Together with our other data, the allele-specific expression at OY1 that was visible when we re-analyzed the data from <a id="xref-ref-86-4" class="xref-bibr" href="#ref-86">Waters <em>et al.</em> (2017)</a> led us to propose that <em>vey1</em> encodes a <em>cis</em>-acting regulatory DNA sequence variation (Figure S9). Sequence comparisons outside the protein coding sequence of the gene can be quite challenging, especially in maize, as it exhibits limited to poor sequence conservation between different inbred lines (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">SanMiguel <em>et al.</em> 1996</a>). Thus, distinguishing phenotypically-affective polymorphisms from the neutral variants, <em>e.g.</em>, via transgenic testing, is not trivial. As a result, biochemical and <em>in vitro</em> studies are the best tool for functional validation of these polymorphisms (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Wray <em>et al.</em> 2003</a>). Similar experiments have been done to characterize the role of DNA sequence polymorphisms <em>in cis</em> on the expression of downstream genes in the case of <em>flowering locus T</em> in <em>Arabidopsis</em> and <em>teosinte branched1</em> in maize (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adrian <em>et al.</em> 2010</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Studer <em>et al.</em> 2011</a>).</p></div><div id="sec-20" class="subsection"><h3>Relevance to research on transcriptional regulation</h3><p id="p-63">Since their discovery, the role of regulatory elements in gene function has been recognized as vital to our understanding of biological systems (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">McClintock 1950</a>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">1956a</a>,<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">b</a>, <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">1961</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Peterson 1953</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Jacob and Monod 1961</a>). Gene regulation and gene product dosage are at the forefront of evolutionary theories about sources of novelty and diversification (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Ohno 1972</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">King and Wilson 1975</a>). Transcriptional regulation of a gene can be as important as the protein coding sequence (<a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">Wray <em>et al.</em> 2003</a>). For instance, complete knock-down of expression of a gene by a regulatory polymorphism will have the same phenotypic consequence as the non-sense mediated decay of a transcript harboring an early stop codon (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Willing <em>et al.</em> 1996</a>). We do not have a set of rules, analogous to codon tables, for functional polymorphisms outside the coding sequence of a gene. Thus, detecting expression variation and tying it to phenotypic consequence, especially in the absence of CNS, remains a challenge to this day.</p><p id="p-64">Several eQTL studies in eukaryotes have found abundant cis and <em>trans</em>-acting genomic regions that affect gene expression (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brem <em>et al.</em> 2002</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">West <em>et al.</em> 2007</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2013</a>, <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">2018</a>). Expression polymorphisms are a potential source of phenotypic variation (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gibson and Weir 2005</a>), and multiple studies detected expression polymorphisms co-segregating with phenotypic changes, including contributions to species domestication (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Clark <em>et al.</em> 2006</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Salvi <em>et al.</em> 2007</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Schwartz <em>et al.</em> 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lemmon <em>et al.</em> 2014</a>). However, the majority of the <em>cis</em>-eQTLs only exhibit a moderate difference in gene expression. Detecting such variants and linking them to visible phenotypes may require detailed study using approaches focused on the specific biological process affected by the gene product. We do not yet have a standardized experimental tool for these purposes. As such, we cannot simultaneously identify and characterize the phenotypic impact of most <em>cis</em>-acting eQTLs. The <em>vey1</em> polymorphism detected in the current study co-segregates with a <em>cis</em>-eQTL at <em>oy1</em> in IBM (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2013</a>, <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">2018</a>) and diverse maize inbred lines (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Kremling <em>et al.</em> 2018</a>). The high heritability of the alternative and direct CCM phenotype allowed us to scan large populations at a rapid rate to identify the genomic regions underlying the <em>cis</em>-acting regulatory elements and study allelic diversity in the natural population of maize. We propose that the approach we have taken is not likely to be unique to <em>oy1</em>. Therefore, we propose that all semi-dominant mutant alleles can be used as reporters to not only detect novel <em>cis</em>-acting gene regulatory elements but also functionally validate previously-detected <em>cis</em>-eQTL(s) from genome-wide eQTL studies.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-65">Authors acknowledge the help of the staff at Purdue ACRE farm for assistance with planting and crop management of all the field experiments described in this study. R.S.K was supported by USAID Feed the Future CGIAR-US Initiative grant “Heat stress resilient maize for South Asia through a public-private partnership” to G.S.J.. This work was supported by NSF grant 1444503 awarded to G.S.J. and B.P.D. We especially thank the Muehlbauer lab for making their data open to the public, and the USDA for the MaizeGDB website, which allowed us to reanalyze the eQTL data.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-66">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7370948">https://doi.org/10.25387/g3.7370948</a>.</p></li><li class="fn" id="fn-8"><p id="p-67"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-08-28"><span class="received-label">Received </span>August 28, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-12-02"><span class="accepted-label">Accepted </span>December 2, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Khangura <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-5">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.1" data-doi="10.1105/tpc.110.074682"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adrian</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrona</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reimer</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albani</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cis-Regulatory Elements and Chromatin State Coordinately Control Temporal and Spatial Expression of FLOWERING LOCUS T in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1425</span>–<span class="cit-lpage">1440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.110.074682</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.074682%26rft_id%253Dinfo%253Apmid%252F20472817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzUvMTQyNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.2" data-doi="10.18637/jss.v067.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Machler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fitting Linear Mixed-Effects Models using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v067.i01</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v067.i01%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v067.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.3" data-doi="10.1093/biostatistics/kxj037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxj037%26rft_id%253Dinfo%253Apmid%252F16632515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxj037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16632515&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.4" data-doi="10.1016/j.ifset.2004.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berardo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenna</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valoti</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pisacane</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Carotenoids concentration among maize genotypes measured by near infrared reflectance spectroscopy (NIRS).</span> <abbr class="cit-jnl-abbrev">Innov. Food Sci. Emerg. Technol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">398</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ifset.2004.03.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInnov.%2BFood%2BSci.%2BEmerg.%2BTechnol.%26rft.volume%253D5%26rft.spage%253D393%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ifset.2004.03.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ifset.2004.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newton</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Modulation of protein levels in chromosomal dosage series of maize: the biochemical basis of aneuploid syndromes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBirchler%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D99%26rft.issue%253D2%26rft.spage%253D247%26rft.epage%253D266%26rft.atitle%253DMODULATION%2BOF%2BPROTEIN%2BLEVELS%2BIN%2BCHROMOSOMAL%2BDOSAGE%2BSERIES%2BOF%2BMAIZE%253A%2BTHE%2BBIOCHEMICAL%2BBASIS%2BOF%2BANEUPLOID%2BSYNDROMES%26rft_id%253Dinfo%253Apmid%252F17249116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiOTkvMi8yNDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.6" data-doi="10.1073/pnas.1207726109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene balance hypothesis: Connecting issues of dosage sensitivity across biological disciplines.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">14746</span>–<span class="cit-lpage">14753</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1207726109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1207726109%26rft_id%253Dinfo%253Apmid%252F22908297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM3LzE0NzQ2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzM3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.7" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.8" data-doi="10.1126/science.1069516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yvert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clinton</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic dissection of transcriptional regulation in budding yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">752</span>–<span class="cit-lpage">755</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1069516</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1069516%26rft_id%253Dinfo%253Apmid%252F11923494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTYvNTU2OC83NTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.9" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.10" data-doi="10.1534/g3.114.014183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buescher</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Runkel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hake</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dilkes</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural Variation at sympathy for the ligule Controls Penetrance of the Semidominant Liguleless narrow-R Mutation in Zea mays.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2297</span>–<span class="cit-lpage">2306</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.014183</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.014183%26rft_id%253Dinfo%253Apmid%252F25344411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTIvMjI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.11" data-doi="10.1093/gigascience/gix120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bukowski</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Construction of the third-generation Zea mays haplotype map.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/gix134</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgix120%26rft_id%253Dinfo%253Apmid%252F29300887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/gix120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29300887&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-other" id="cit-9.2.375.12"><div class="cit-metadata unstructured">Bushnell, B., 2014 BBMap: A Fast, Accurate, Splice-Aware Aligner. Technical report.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.13" data-doi="10.1534/genetics.109.111880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chintamanani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulbert</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johal</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balint-Kurti</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of a maize locus that modulates the hypersensitive defense response, using mutant-assisted gene identification and characterization.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">813</span>–<span class="cit-lpage">825</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.111880</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.111880%26rft_id%253Dinfo%253Apmid%252F20176981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzMvODEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzM3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.14" data-doi="10.1105/tpc.002170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorsich</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mitochondrial GFA2 Is Required for Synergid Cell Death in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2215</span>–<span class="cit-lpage">2232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.002170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DChristensen%26rft.auinit1%253DC.%2BA.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D2215%26rft.epage%253D2232%26rft.atitle%253DMitochondrial%2BGFA2%2BIs%2BRequired%2Bfor%2BSynergid%2BCell%2BDeath%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.002170%26rft_id%253Dinfo%253Apmid%252F12215516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE0LzkvMjIxNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.15" data-doi="10.1006/dbio.1998.8980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drews</span>  <span class="cit-name-given-names">G. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Identification of Gametophytic Mutations Affecting Female Gametophyte Development in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">151</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/dbio.1998.8980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DChristensen%26rft.auinit1%253DC.%2BA.%26rft.volume%253D202%26rft.issue%253D1%26rft.spage%253D136%26rft.epage%253D151%26rft.atitle%253DIdentification%2Bof%2Bgametophytic%2Bmutations%2Baffecting%2Bfemale%2Bgametophyte%2Bdevelopment%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.1998.8980%26rft_id%253Dinfo%253Apmid%252F9758709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.1998.8980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9758709&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076396200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.16" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzM3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.17" data-doi="10.1038/ng1784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagler</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quijada</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A distant upstream enhancer at the maize domestication gene tb1 has pleiotropic effects on plant and inflorescent architecture.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">594</span>–<span class="cit-lpage">597</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1784</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DClark%26rft.auinit1%253DR.%2BM.%26rft.volume%253D38%26rft.issue%253D5%26rft.spage%253D594%26rft.epage%253D597%26rft.atitle%253DA%2Bdistant%2Bupstream%2Benhancer%2Bat%2Bthe%2Bmaize%2Bdomestication%2Bgene%2Btb1%2Bhas%2Bpleiotropic%2Beffects%2Bon%2Bplant%2Band%2Binflorescent%2Barchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1784%26rft_id%253Dinfo%253Apmid%252F16642024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1784&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16642024&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237147500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.18" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.19" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.20" data-doi="10.1371/journal.pbio.0060308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dilkes</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spielman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weizbauer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burkart-Waco</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Maternally Expressed WRKY Transcription Factor TTG2 Controls Lethality in Interploidy Crosses of Arabidopsis.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2707</span>–<span class="cit-lpage">2720</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0060308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDilkes%26rft.auinit1%253DB.%2BP.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D2707%26rft.epage%253D2720%26rft.atitle%253DThe%2Bmaternally%2Bexpressed%2BWRKY%2Btranscription%2Bfactor%2BTTG2%2Bcontrols%2Blethality%2Bin%2Binterploidy%2Bcrosses%2Bof%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060308%26rft_id%253Dinfo%253Apmid%252F19071961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19071961&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261913700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.21" data-doi="10.1007/BF02671663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eggermont</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goderis</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broekaert</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">High-throughput RNA Extraction from Plant Samples Based on Homogenisation by Reciprocal Shaking in the Presence of a Mixture of Sand and Glass Beads.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Report.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02671663</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BReport.%26rft.volume%253D14%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02671663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02671663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VN71300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.22" data-doi="10.1104/pp.111.174748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foerster</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">B73-Mo17 near-isogenic lines demonstrate dispersed structural variation in maize.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1679</span>–<span class="cit-lpage">1690</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.111.174748</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.174748%26rft_id%253Dinfo%253Apmid%252F21705654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU2LzQvMTY3OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.23" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1921</span> <span class="cit-article-title">On the ”probable error” of a coefficient of correlation deduced from a small sample.</span> <abbr class="cit-jnl-abbrev">Metron</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMetron%26rft.volume%253D1%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.375.25" data-doi="10.5962/bhl.title.27468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-source">The genetical theory of natural selection</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford, UK</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5962/bhl.title.27468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Bgenetical%2Btheory%2Bof%2Bnatural%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.5962%252Fbhl.title.27468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5962/bhl.title.27468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.26" data-doi="10.1111/j.1365-313X.2005.02591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thuillet</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Maize association population: a high-resolution platform for quantitative trait locus dissection.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1054</span>–<span class="cit-lpage">1064</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.2005.02591.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFlint-Garcia%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1054%26rft.epage%253D1064%26rft.atitle%253DMaize%2Bassociation%2Bpopulation%253A%2Ba%2Bhigh-resolution%2Bplatform%2Bfor%2Bquantitative%2Btrait%2Blocus%2Bdissection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02591.x%26rft_id%253Dinfo%253Apmid%252F16359397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16359397&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233776900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.27" data-doi="10.1006/jmbi.2001.4834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fodje</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gough</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Interplay between an AAA module and an integrin I domain may regulate the function of magnesium chelatase.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">311</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jmbi.2001.4834</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DFodje%26rft.auinit1%253DM.%2BN.%26rft.volume%253D311%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D122%26rft.atitle%253DInterplay%2Bbetween%2Ban%2BAAA%2Bmodule%2Band%2Ban%2Bintegrin%2BI%2Bdomain%2Bmay%2Bregulate%2Bthe%2Bfunction%2Bof%2Bmagnesium%2Bchelatase.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2001.4834%26rft_id%253Dinfo%253Apmid%252F11469861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2001.4834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11469861&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170326500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.28" data-doi="10.1016/j.tig.2005.08.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The quantitative genetics of transcription.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">616</span>–<span class="cit-lpage">623</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2005.08.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DGibson%26rft.auinit1%253DG.%26rft.volume%253D21%26rft.issue%253D11%26rft.spage%253D616%26rft.epage%253D623%26rft.atitle%253DThe%2Bquantitative%2Bgenetics%2Bof%2Btranscription.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2005.08.010%26rft_id%253Dinfo%253Apmid%252F16154229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2005.08.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16154229&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233021600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.29" data-doi="10.1016/S0092-8674(00)80179-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johal</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A novel suppressor of cell death in plants encoded by the Lls1 gene of maize.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)80179-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGray%26rft.auinit1%253DJ.%26rft.volume%253D89%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D31%26rft.atitle%253DA%2Bnovel%2Bsuppressor%2Bof%2Bcell%2Bdeath%2Bin%2Bplants%2Bencoded%2Bby%2Bthe%2BLls1%2Bgene%2Bof%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980179-8%26rft_id%253Dinfo%253Apmid%252F9094711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80179-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9094711&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WR68500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.30" data-doi="10.1104/pp.008441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janick-Buckner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johal</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Light-dependent death of maize lls1 cells is mediated by mature chloroplasts.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">1894</span>–<span class="cit-lpage">1907</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.008441</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.008441%26rft_id%253Dinfo%253Apmid%252F12481072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTMwLzQvMTg5NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.31" data-doi="10.1006/jtbi.1996.0167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grossniklaus</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madhusudhan</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nanjundiah</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Nonlinear enzyme kinetics can lead to high metabolic flux control coefficients: Implications for the evolution of dominance.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jtbi.1996.0167</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DGrossniklaus%26rft.auinit1%253DU.%26rft.volume%253D182%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D302%26rft.atitle%253DNonlinear%2Benzyme%2Bkinetics%2Bcan%2Blead%2Bto%2Bhigh%2Bmetabolic%2Bflux%2Bcontrol%2Bcoefficients%253A%2Bimplications%2Bfor%2Bthe%2Bevolution%2Bof%2Bdominance.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjtbi.1996.0167%26rft_id%253Dinfo%253Apmid%252F8944161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jtbi.1996.0167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8944161&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.32" data-doi="10.1073/pnas.96.4.1744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kannangara</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wettstein</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Molecular basis for semidominance of missense mutations in the XANTHA-H (42-kDa) subunit of magnesium chelatase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1744</span>–<span class="cit-lpage">1749</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.96.4.1744</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHansson%26rft.auinit1%253DA.%26rft.volume%253D96%26rft.issue%253D4%26rft.spage%253D1744%26rft.epage%253D1749%26rft.atitle%253DMolecular%2Bbasis%2Bfor%2Bsemidominance%2Bof%2Bmissense%2Bmutations%2Bin%2Bthe%2BXANTHA-H%2B%252842-kDa%2529%2Bsubunit%2Bof%2Bmagnesium%2Bchelatase%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.4.1744%26rft_id%253Dinfo%253Apmid%252F9990095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni80LzE3NDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.33" data-doi="10.1073/pnas.212504499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willows</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Three semidominant barley mutants with single amino acid substitutions in the smallest magnesium chelatase subunit form defective AAA+ hexamers.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">13944</span>–<span class="cit-lpage">13949</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.212504499</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.212504499%26rft_id%253Dinfo%253Apmid%252F12357035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjEvMTM5NDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.34" data-doi="10.1126/science.1150255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harjes</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocheford</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brutnell</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kandianis</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Natural Genetic Variation in Lycopene Epsilon Cyclase Tapped for Maize Biofortification.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1150255</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHarjes%26rft.auinit1%253DC.%2BE.%26rft.volume%253D319%26rft.issue%253D5861%26rft.spage%253D330%26rft.epage%253D333%26rft.atitle%253DNatural%2BGenetic%2BVariation%2Bin%2BLycopene%2BEpsilon%2BCyclase%2BTapped%2Bfor%2BMaize%2BBiofortification%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150255%26rft_id%253Dinfo%253Apmid%252F18202289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2MS8zMzAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">A simple sequentially rejective multiple test procedure.</span> <abbr class="cit-jnl-abbrev">Scand. J. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScand.%2BJ.%2BStat.%26rft.volume%253D6%26rft.spage%253D65%26rft.atitle%253DA%2BSIMPLE%2BSEQUENTIALLY%2BREJECTIVE%2BMULTIPLE%2BTEST%2BPROCEDURE.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1979JY78700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.36" data-doi="10.1105/tpc.10.7.1095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalpani</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johal</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A porphyrin pathway impairment is responsible for the phenotype of a dominant disease lesion mimic mutant of maize.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1095</span>–<span class="cit-lpage">1105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.10.7.1095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DHu%26rft.auinit1%253DG.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D1095%26rft.epage%253D1106%26rft.atitle%253DA%2BPorphyrin%2BPathway%2BImpairment%2BIs%2BResponsible%2Bfor%2Bthe%2BPhenotype%2Bof%2Ba%2BDominant%2BDisease%2BLesion%2BMimic%2BMutant%2Bof%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.10.7.1095%26rft_id%253Dinfo%253Apmid%252F9668130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEwLzcvMTA5NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.37" data-doi="10.1093/mp/ssp029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slewinski</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janick-Buckner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Camouflage patterning in maize leaves results from a defect in porphobilinogen deaminase.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">789</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/mp/ssp029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fssp029%26rft_id%253Dinfo%253Apmid%252F19825655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/ssp029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19825655&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268331700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.38" data-doi="10.3198/jpr2005.11.0414crs"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hussain</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tausend</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Registration of IBM2 SYN10 Doubled Haploid Mapping Population of Maize.</span> <abbr class="cit-jnl-abbrev">J. Plant Regist.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">81</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3198/jpr2005.11.0414crs</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPlant%2BRegistrations%26rft.stitle%253DJournal%2Bof%2BPlant%2BRegistrations%26rft.aulast%253DHussain%26rft.auinit1%253DT.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D81%26rft.atitle%253DRegistration%2Bof%2BIBM2%2BSYN10%2BDoubled%2BHaploid%2BMapping%2BPopulation%2Bof%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.3198%252Fjpr2005.11.0414crs%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3198/jpr2005.11.0414crs&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.39" data-doi="10.1016/S0022-2836(61)80072-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacob</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monod</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">Genetic regulatory mechanisms in the synthesis of proteins.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">318</span>–<span class="cit-lpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(61)80072-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DJacob%26rft.auinit1%253DF.%26rft.volume%253D3%26rft.spage%253D318%26rft.epage%253D356%26rft.atitle%253DGenetic%2Bregulatory%2Bmechanisms%2Bin%2Bthe%2Bsynthesis%2Bof%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252861%252980072-7%26rft_id%253Dinfo%253Apmid%252F13718526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(61)80072-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13718526&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19615792B00007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.40" data-doi="10.2135/cropsci2008.03.0150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johal</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balint-Kurti</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weil</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mining and Harnessing Natural Variation: A Little MAGIC.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">2066</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.03.0150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DJohal%26rft.auinit1%253DG.%2BS.%26rft.volume%253D48%26rft.issue%253D6%26rft.spage%253D2066%26rft.epage%253D2073%26rft.atitle%253DMining%2Band%2BHarnessing%2BNatural%2BVariation%253A%2BA%2BLittle%2BMAGIC%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.03.0150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.03.0150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000261599000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.41" data-doi="10.1126/science.1090005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Evolution at two levels in Humans and Chimpanzees.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1090005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKing%26rft.auinit1%253DM.%26rft.volume%253D188%26rft.issue%253D4184%26rft.spage%253D107%26rft.epage%253D116%26rft.atitle%253DEvolution%2Bat%2Btwo%2Blevels%2Bin%2Bhumans%2Band%2Bchimpanzees%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1090005%26rft_id%253Dinfo%253Apmid%252F1090005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjE4OC80MTg0LzEwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.42" data-doi="10.1038/nature25966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kremling</span>  <span class="cit-name-given-names">K. A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">M.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lepak</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Dysregulation of expression correlates with rare-allele burden and fitness loss in maize.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">555</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">523</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature25966</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D555%26rft.spage%253D520%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature25966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature25966&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.43" data-doi="10.3389/fpls.2017.00023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lauter</span>  <span class="cit-name-given-names">A. N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muszynski</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huffman</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A Pectin Metethylesterase ZmPme3 Is Expressed in Gametophyte factor1-s (Ga1-s) Silks and Maps to that Locus in Maize (Zea mays L.).</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01926</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.00023%26rft_id%253Dinfo%253Apmid%252F28220127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.00023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28220127&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.44" data-doi="10.1023/A:1014893521186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharopova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Expanding the genetic map of maize with the intermated B73 x Mo17 (IBM) population.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">461</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1014893521186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DLee%26rft.auinit1%253DM.%26rft.volume%253D48%26rft.issue%253D5-6%26rft.spage%253D453%26rft.epage%253D461%26rft.atitle%253DExpanding%2Bthe%2Bgenetic%2Bmap%2Bof%2Bmaize%2Bwith%2Bthe%2Bintermated%2BB73%2Bx%2BMo17%2B%2528IBM%2529%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1014893521186%26rft_id%253Dinfo%253Apmid%252F11999829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1014893521186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11999829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174606400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.45" data-doi="10.1371/journal.pgen.1004745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lemmon</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bukowski</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Role of cis Regulatory Evolution in Maize Domestication.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004745</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004745</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004745%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004745%26rft_id%253Dinfo%253Apmid%252F25375861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25375861&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.46" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.47" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.48" data-doi="10.1371/journal.pgen.1007234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petsch</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Correction: Mendelian and Non-Mendelian Regulation of Gene Expression in Maize.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1007234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007234</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D14%26rft.spage%253De1007234%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.49" data-doi="10.1371/journal.pgen.1003202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petsch</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mendelian and non-Mendelian regulation of gene expression in maize.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003202</span>. [erratum: PLos Genet 14: e1007234.]<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003202%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003202%26rft_id%253Dinfo%253Apmid%252F23341782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23341782&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.375.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lichtenthaler</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buschmann</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chlorophylls and Carotenoids: measurement and characterization by UV-VIS spectroscopy.</span> <span class="cit-source">Curr. Protoc. Food Anal. Chem.</span> <span class="cit-vol">55</span>: <span class="cit-fpage">F4.3.1</span>–<span class="cit-lpage">F4.3.8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BFood%2BAnal.%2BChem.%26rft.volume%253D55%26rft.spage%253DF431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.51" data-doi="10.1007/BF00306054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allaire</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Heritability of a linear combination of traits.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00306054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D51%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00306054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00306054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1977EA58200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.52" data-doi="10.1093/bioinformatics/bts444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GAPIT: genome association and prediction integrated tool.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2397</span>–<span class="cit-lpage">2399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts444</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts444%26rft_id%253Dinfo%253Apmid%252F22796960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22796960&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308532300066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.53" data-doi="10.1186/s12864-015-2242-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Portilla</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">An ultra-high-density map as a community resource for discerning the genetic basis of quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1078</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2242-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1078%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2242-5%26rft_id%253Dinfo%253Apmid%252F26691201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2242-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26691201&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.54" data-doi="10.1074/jbc.M113.480012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lundqvist</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braumann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurowska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Catalytic turnover triggers exchange of subunits of the magnesium chelatase AAA+ motor unit.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">24012</span>–<span class="cit-lpage">24019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M113.480012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M113.480012%26rft_id%253Dinfo%253Apmid%252F23836887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvMzMvMjQwMTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.55" data-doi="10.1073/pnas.98.2.771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mach</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castillo</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoogstraten</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenberg</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The Arabidopsis-accelerated cell death gene ACD2 encodes red chlorophyll catabolite reductase and suppresses the spread of disease symptoms.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.98.2.771</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMach%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D771%26rft.epage%253D776%26rft.atitle%253DThe%2BArabidopsis-accelerated%2Bcell%2Bdeath%2Bgene%2BACD2%2Bencodes%2Bred%2Bchlorophyll%2Bcatabolite%2Breductase%2Band%2Bsuppresses%2Bthe%2Bspread%2Bof%2Bdisease%2Bsymptoms%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.98.2.771%26rft_id%253Dinfo%253Apmid%252F11149948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5OC8yLzc3MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mangelsdorf</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1926</span> <span class="cit-article-title">The expression of mendelian factors in the gametophyte of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMangelsdorf%26rft.auinit1%253DP.%2BC.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D423%26rft.epage%253D455%26rft.atitle%253DTHE%2BEXPRESSION%2BOF%2BMENDELIAN%2BFACTORS%2BIN%2BTHE%2BGAMETOPHYTE%2BOF%2BMAIZE%26rft_id%253Dinfo%253Apmid%252F17246465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxMS81LzQyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.57" data-doi="10.1073/pnas.36.6.344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClintock</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">The origin and behavior of mutable loci in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.36.6.344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMcClintock%26rft.auinit1%253DB.%26rft.volume%253D36%26rft.issue%253D6%26rft.spage%253D344%26rft.epage%253D355%26rft.atitle%253DThe%2Borigin%2Band%2Bbehavior%2Bof%2Bmutable%2Bloci%2Bin%2Bmaize%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.36.6.344%26rft_id%253Dinfo%253Apmid%252F15430309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiIzNi82LzM0NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.58" data-doi="10.1101/SQB.1956.021.01.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClintock</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span><span class="cit-comment">a</span> <span class="cit-article-title">Controlling elements and the gene.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">216</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/SQB.1956.021.01.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DMcClintock%26rft.auinit1%253DB.%26rft.volume%253D21%26rft.issue%253D0%26rft.spage%253D197%26rft.epage%253D216%26rft.atitle%253DControlling%2BElements%2Band%2Bthe%2BGene%26rft_id%253Dinfo%253Adoi%252F10.1101%252FSQB.1956.021.01.017%26rft_id%253Dinfo%253Apmid%252F13433592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjg6IjIxLzAvMTk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzM3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClintock</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span><span class="cit-comment">b</span> <span class="cit-article-title">Intranuclear systems controlling gene action and mutation.</span> <abbr class="cit-jnl-abbrev">Brookhaven Symp. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrookhaven%2BSymp.%2BBiol.%26rft.volume%253D8%26rft.spage%253D58%26rft_id%253Dinfo%253Apmid%252F13293421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=13293421&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.60" data-doi="10.1086/282188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClintock</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">Some parallels between gene control systems in maize and in bacteria.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/282188</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D95%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.61" data-doi="10.1073/pnas.221252798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meskauskiene</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nater</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goslings</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessler</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">op den Camp</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">FLU: A negative regulator of chlorophyll biosynthesis in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">12826</span>–<span class="cit-lpage">12831</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.221252798</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.221252798%26rft_id%253Dinfo%253Apmid%252F11606728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjIvMTI4MjYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.62" data-doi="10.1016/S0168-9525(02)02820-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neff</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turk</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalishman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Web-based primer design for single nucleotide polymorphism analysis.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">615</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0168-9525(02)02820-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DNeff%26rft.auinit1%253DM.%2BM.%26rft.volume%253D18%26rft.issue%253D12%26rft.spage%253D613%26rft.epage%253D615%26rft.atitle%253DWeb-based%2Bprimer%2Bdesign%2Bfor%2Bsingle%2Bnucleotide%2Bpolymorphism%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902820-2%26rft_id%253Dinfo%253Apmid%252F12446140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02820-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12446140&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179433300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.63" data-doi="10.1016/0047-2484(72)90011-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">An argument for the genetic simplicity of man and other mammals.</span> <abbr class="cit-jnl-abbrev">J. Hum. Evol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">651</span>–<span class="cit-lpage">662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0047-2484(72)90011-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHum.%2BEvol.%26rft.volume%253D1%26rft.spage%253D651%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0047-2484%252872%252990011-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0047-2484(72)90011-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972Z012900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.64" data-doi="10.1104/pp.15.01870"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olukolu</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bian</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Vries</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tracy</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Genetics of Leaf Flecking in Maize and Its Relationship to Plant Defense and Disease Resistance.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1787</span>–<span class="cit-lpage">1803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.15.01870</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.01870%26rft_id%253Dinfo%253Apmid%252F27670817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTcyLzMvMTc4NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.65" data-doi="10.1534/genetics.112.147595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olukolu</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Negeri</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhawan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkata</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A connected set of genes associated with programmed cell death implicated in controlling the hypersensitive response in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147595</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147595%26rft_id%253Dinfo%253Apmid%252F23222653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNjA5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzM3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.66" data-doi="10.1371/journal.pgen.1004562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olukolu</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vontimitta</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkata</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marla</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A genome-wide association study of the maize hypersensitive defense response identifies genes that cluster in related pathways.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004562</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004562</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004562%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004562%26rft_id%253Dinfo%253Apmid%252F25166276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25166276&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.67" data-doi="10.1111/j.1558-5646.1998.tb01823.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The Population Genetics of Adaptation: The Distribution of Factors Fixed during Adaptive Evolution.</span> <abbr class="cit-jnl-abbrev">Evolution (N. Y.)</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">949</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1998.tb01823.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2528N.%2BY.%2529%26rft.volume%253D52%26rft.spage%253D935%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.1998.tb01823.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.1998.tb01823.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.68" data-doi="10.1038/nrg1523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The genetic theory of adaptation: A brief history.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1523</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D119%26rft.epage%253D127%26rft.atitle%253DThe%2Bgenetic%2Btheory%2Bof%2Badaptation%253A%2Ba%2Bbrief%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1523%26rft_id%253Dinfo%253Apmid%252F15716908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716908&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226721300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.69" data-doi="10.1139/g04-046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Penning</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johal</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mcmullen</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A major suppressor of cell death, slm1, modifies the expression of the maize (Zea mays L. ) lesion mimic mutation les23.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">969</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g04-046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DPenning%26rft.auinit1%253DB.%2BW.%26rft.volume%253D47%26rft.issue%253D5%26rft.spage%253D961%26rft.epage%253D969%26rft.atitle%253DA%2Bmajor%2Bsuppressor%2Bof%2Bcell%2Bdeath%252C%2Bslm1%252C%2Bmodifies%2Bthe%2Bexpression%2Bof%2Bthe%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2Blesion%2Bmimic%2Bmutation%2Bles23.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg04-046%26rft_id%253Dinfo%253Apmid%252F15499410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g04-046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15499410&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-thesis" id="cit-9.2.375.70"><div class="cit-metadata unstructured">Peterson, P. A., 1953 <em>A study of mutable pale green locus in maize</em>. Dissertation, University of Illinois.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.71" data-doi="10.1007/s10142-011-0224-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pribat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaby</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lara-Núñez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeanguenin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fouquet</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A 5-formyltetrahydrofolate cycloligase paralog from all domains of life: Comparative genomic and experimental evidence for a cryptic role in thiamin metabolism.</span> <abbr class="cit-jnl-abbrev">Funct. Integr. Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">478</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10142-011-0224-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunctional%2B%2526%2Bintegrative%2Bgenomics%26rft.stitle%253DFunct%2BIntegr%2BGenomics%26rft.aulast%253DPribat%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D467%26rft.epage%253D478%26rft.atitle%253DA%2B5-formyltetrahydrofolate%2Bcycloligase%2Bparalog%2Bfrom%2Ball%2Bdomains%2Bof%2Blife%253A%2Bcomparative%2Bgenomic%2Band%2Bexperimental%2Bevidence%2Bfor%2Ba%2Bcryptic%2Brole%2Bin%2Bthiamin%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10142-011-0224-5%26rft_id%253Dinfo%253Apmid%252F21538139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10142-011-0224-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21538139&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-9.2.375.72"><div class="cit-metadata unstructured">R Core, T., 2014 R: A language and environment for statistical computing. r foundation for statistical computing, vienna, austria. 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.73" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative Genome Viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1754</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.74" data-doi="10.1186/gb-2013-14-6-r55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millard</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comprehensive genotyping of the USA national maize inbred seed bank.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-6-r55</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR55%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r55%26rft_id%253Dinfo%253Apmid%252F23759205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23759205&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.75" data-doi="10.1073/pnas.0704145104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salvi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sponza</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgante</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomes</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Conserved noncoding genomic sequences associated with a flowering-time quantitative trait locus in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">11376</span>–<span class="cit-lpage">11381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0704145104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0704145104%26rft_id%253Dinfo%253Apmid%252F17595297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzI3LzExMzc2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzM3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.76" data-doi="10.1006/anbo.1998.0746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">SanMiguel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Evidence that a Recent Increase in Maize Genome Size was Caused by the Massive Amplification of Intergene Retrotransposons.</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/anbo.1998.0746</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft.stitle%253DAnn%2BBot%26rft.aulast%253DSanmiguel%26rft.auinit1%253DP.%26rft.volume%253D82%26rft.issue%253Dsuppl_1%26rft.spage%253D37%26rft.epage%253D44%26rft.atitle%253DEvidence%2Bthat%2Ba%2BRecent%2BIncrease%2Bin%2BMaize%2BGenome%2BSize%2Bwas%2BCaused%2Bby%2Bthe%2BMassive%2BAmplification%2Bof%2BIntergene%2BRetrotransposons%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fanbo.1998.0746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/anbo.1998.0746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.77" data-doi="10.1126/science.274.5288.765"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">SanMiguel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tikhonov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motchoulskaia</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakharov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Nested Retrotransposons in the Intergenic regions of the maize genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">765</span>–<span class="cit-lpage">768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.274.5288.765</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSanMiguel%26rft.auinit1%253DP.%26rft.volume%253D274%26rft.issue%253D5288%26rft.spage%253D765%26rft.epage%253D768%26rft.atitle%253DNested%2BRetrotransposons%2Bin%2Bthe%2BIntergenic%2BRegions%2Bof%2Bthe%2BMaize%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5288.765%26rft_id%253Dinfo%253Apmid%252F8864112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTI4OC83NjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.78" data-doi="10.1007/s11103-005-2880-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawers</span>  <span class="cit-name-given-names">R. J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viney</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsefski</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The maize Oil yellow1 (Oy1) gene encodes the I subunit of magnesium chelatase.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11103-005-2880-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DHasegawa%26rft.auinit1%253DT.%26rft.volume%253D60%26rft.issue%253D1%26rft.spage%253D95%26rft.epage%253D106%26rft.atitle%253DThe%2Bmaize%2BOil%2Byellow1%2B%2528Oy1%2529%2Bgene%2Bencodes%2Bthe%2BI%2Bsubunit%2Bof%2Bmagnesium%2Bchelatase.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-005-2880-0%26rft_id%253Dinfo%253Apmid%252F16463102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-005-2880-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16463102&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235202000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.79" data-doi="10.1534/genetics.109.104984"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balasubramanian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warthmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempe</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Cis-regulatory changes at Flowering Locus T mediate natural variation in flowering responses of Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">732</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.104984</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104984%26rft_id%253Dinfo%253Apmid%252F19652183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzIvNzIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzM3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.80" data-doi="10.1093/database/baq007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andorf</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seigfried</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Choosing a genome browser for a Model Organism Database: surveying the Maize community.</span> <abbr class="cit-jnl-abbrev">Database (Oxford)</abbr> <span class="cit-vol">2010</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/database/baq007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDatabase%2B%2528Oxford%2529%26rft.volume%253D2010%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdatabase%252Fbaq007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/database/baq007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.81" data-doi="10.1371/journal.pone.0201359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sindhu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janick-Buckner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zehr</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Propagation of cell death in dropdead1, a sorghum ortholog of the maize lls1 mutant.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0201359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0201359</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0201359%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0201359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0201359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.82" data-doi="10.1038/ng.942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Studer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of a functional transposon insertion in the maize domestication gene tb1.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1160</span>–<span class="cit-lpage">1163</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.942</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DStuder%26rft.auinit1%253DA.%26rft.volume%253D43%26rft.issue%253D11%26rft.spage%253D1160%26rft.epage%253D1163%26rft.atitle%253DIdentification%2Bof%2Ba%2Bfunctional%2Btransposon%2Binsertion%2Bin%2Bthe%2Bmaize%2Bdomestication%2Bgene%2Btb1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.942%26rft_id%253Dinfo%253Apmid%252F21946354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21946354&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.83" data-doi="10.1093/nar/22.22.4673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">ClustalW: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position specific gap penalties and weight matrix choice.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res. Acids Res</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">4673</span>–<span class="cit-lpage">4680</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/22.22.4673</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.22.4673%26rft_id%253Dinfo%253Apmid%252F7984417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/22.22.4673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7984417&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PU19900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.84" data-doi="10.1083/jcb.150.1.F13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vale</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">AAA proteins. Lords of the ring.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">F13</span>–<span class="cit-lpage">F19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1083/jcb.150.1.F13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.150.1.F13%26rft_id%253Dinfo%253Apmid%252F10893253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE1MC8xL0YxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi8zNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.85" data-doi="10.1006/jtbi.2003.3105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Nonlinear effects in macromolecular assembly and dosage sensitivity.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">220</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jtbi.2003.3105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DVeitia%26rft.auinit1%253DR.%2BA.%26rft.volume%253D220%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D25%26rft.atitle%253DNonlinear%2Beffects%2Bin%2Bmacromolecular%2Bassembly%2Band%2Bdosage%2Bsensitivity.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjtbi.2003.3105%26rft_id%253Dinfo%253Apmid%252F12453447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jtbi.2003.3105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12453447&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180313000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.86" data-doi="10.1111/tpj.13414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarevitch</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noshay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burghardt</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Natural variation for gene expression responses to abiotic stress in maize.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">717</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13414</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D89%26rft.spage%253D706%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.87" data-doi="10.1534/genetics.106.064972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliebenstein</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Global eQTL mapping reveals the complex genetic architecture of transcript-level variation in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1441</span>–<span class="cit-lpage">1450</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.064972</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.064972%26rft_id%253Dinfo%253Apmid%252F17179097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzE0NDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.88" data-doi="10.1105/tpc.7.7.1039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Von Wettstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gough</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kannangara</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Chlorophyll Biosynthesis.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1039</span>–<span class="cit-lpage">1057</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.7.7.1039</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253Dvon%2BWettstein%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D1039%26rft.epage%253D1057%26rft.atitle%253DChlorophyll%2BBiosynthesis%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.7.7.1039%26rft_id%253Dinfo%253Apmid%252F12242396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo5OiJwbGFudGNlbGwiO3M6NToicmVzaWQiO3M6ODoiNy83LzEwMzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willing</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deschenes</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slayton</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Premature chain termination is a unifying mechanism for COL1A1 null alleles in osteogenesis imperfecta type I cell strains.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">809</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWilling%26rft.auinit1%253DM.%2BC.%26rft.volume%253D59%26rft.issue%253D4%26rft.spage%253D799%26rft.epage%253D809%26rft.atitle%253DPremature%2Bchain%2Btermination%2Bis%2Ba%2Bunifying%2Bmechanism%2Bfor%2BCOL1A1%2Bnull%2Balleles%2Bin%2Bosteogenesis%2Bimperfecta%2Btype%2BI%2Bcell%2Bstrains.%26rft_id%253Dinfo%253Apmid%252F8808594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8808594&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VL28300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.90" data-doi="10.1093/molbev/msg140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouheif</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balhoff</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pizer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The evolution of transcriptional regulation in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1377</span>–<span class="cit-lpage">1419</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msg140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg140%26rft_id%253Dinfo%253Apmid%252F12777501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msg140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12777501&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185175500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.91" data-doi="10.1023/B:PLAN.0000028789.51807.6a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wardzala</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johal</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The wound-inducible Lls1 gene from maize is an orthologue of the Arabidopsis Acd1 gene, and the LLS1 protein is present in non-photosynthetic tissues.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DYang%26rft.auinit1%253DM.%26rft.volume%253D54%26rft.issue%253D2%26rft.spage%253D175%26rft.epage%253D191%26rft.atitle%253DThe%2Bwound-inducible%2BLls1%2Bgene%2Bfrom%2Bmaize%2Bis%2Ban%2Borthologue%2Bof%2Bthe%2BArabidopsis%2BAcd1%2Bgene%252C%2Band%2Bthe%2BLLS1%2Bprotein%2Bis%2Bpresent%2Bin%2Bnon-photosynthetic%2Btissues.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253APLAN.0000028789.51807.6a%26rft_id%253Dinfo%253Apmid%252F15159621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:PLAN.0000028789.51807.6a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15159621&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221591100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.92" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vroh Bi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1702</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.375.93" data-doi="10.1038/ng.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todhunter</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mixed linear model approach adapted for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.546</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F375.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/2/375.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/2/359" title="Comprehensive Transcriptional Profiling of the Gastrointestinal Tract of Ruminants from Birth to Adulthood Reveals Strong Developmental Stage Specific Gene Expression" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/2/391" title="Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/2"><span>Volume 9 Issue 2, February 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/2.cover-source.jpg?itok=Pgd0L52l" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/375" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8yFffl8A_zUoKmMFfPknHZqw9GQM_LTqQqciPDhDl0o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/2/375" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/2/375" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/2/375" class="active" data-icon-position="" data-hide-link-title="0">A Very Oil Yellow1 Modifier of the Oil Yellow1-N1989 Allele Uncovers a Cryptic Phenotypic Impact of Cis-regulatory Variation in Maize</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492211" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Sfd6Pp-91G4Q1sgiTwHDiezWKJ5NGvlvB27UdtnWcN4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/2/375.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/2/375" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-oaDhVWVsHb9n-ld8fmdVAsuVis_8cON7cnUob0pri64" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/2/375" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/2/375.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492211" id="node492211--2" data-pisa="ggg;9/2/375" data-pisa-master="ggg;g3.118.200798" data-apath="/ggg/9/2/375.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A <em>Very Oil Yellow1</em> Modifier of the <em>Oil Yellow1-N1989</em> Allele Uncovers a Cryptic Phenotypic Impact of <em>Cis</em>-regulatory Variation in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5085-6240" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rajdeep S. Khangura</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5778-7850" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sandeep Marla</span>, <span class="highwire-citation-author" data-delta="2">Bala P. Venkata</span>, <span class="highwire-citation-author" data-delta="3">Nicholas J. Heller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-8101-8410" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gurmukh S. Johal</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian P. Dilkes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">375-390;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200798 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492211--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rajdeep S. Khangura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajdeep%2BS.%2BKhangura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khangura%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARajdeep%2BS.%2BKhangura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5085-6240" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rajdeep S. Khangura</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sandeep Marla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandeep%2BMarla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marla%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASandeep%2BMarla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5778-7850" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sandeep Marla</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bala P. Venkata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bala%2BP.%2BVenkata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Venkata%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABala%2BP.%2BVenkata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicholas J. Heller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BJ.%2BHeller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heller%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BJ.%2BHeller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gurmukh S. Johal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Biology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gurmukh%2BS.%2BJohal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johal%20GS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGurmukh%2BS.%2BJohal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8101-8410" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gurmukh S. Johal</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian P. Dilkes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Biology, Purdue University, IN 47907</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biochemistry, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BDilkes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilkes%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BP.%2BDilkes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian P. Dilkes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdilkes@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">bdilkes@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/2/375" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/2/375&amp;title=A%20Very%20Oil%20Yellow1%20Modifier%20of%20the%20Oil%20Yellow1-N1989%20Allele%20Uncovers%20a%20Cryptic%20Phenotypic%20Impact%20of%20Cis-regulatory%20Variation%20in%20Maize" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/2/375&amp;title=A%20Very%20Oil%20Yellow1%20Modifier%20of%20the%20Oil%20Yellow1-N1989%20Allele%20Uncovers%20a%20Cryptic%20Phenotypic%20Impact%20of%20Cis-regulatory%20Variation%20in%20Maize" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/2/375&amp;title=A%20Very%20Oil%20Yellow1%20Modifier%20of%20the%20Oil%20Yellow1-N1989%20Allele%20Uncovers%20a%20Cryptic%20Phenotypic%20Impact%20of%20Cis-regulatory%20Variation%20in%20Maize" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/2/375&amp;text=A%20Very%20Oil%20Yellow1%20Modifier%20of%20the%20Oil%20Yellow1-N1989%20Allele%20Uncovers%20a%20Cryptic%20Phenotypic%20Impact%20of%20Cis-regulatory%20Variation%20in%20Maize" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/2/375&amp;title=A%20Very%20Oil%20Yellow1%20Modifier%20of%20the%20Oil%20Yellow1-N1989%20Allele%20Uncovers%20a%20Cryptic%20Phenotypic%20Impact%20of%20Cis-regulatory%20Variation%20in%20Maize" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/2/375&amp;t=A%20Very%20Oil%20Yellow1%20Modifier%20of%20the%20Oil%20Yellow1-N1989%20Allele%20Uncovers%20a%20Cryptic%20Phenotypic%20Impact%20of%20Cis-regulatory%20Variation%20in%20Maize" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/2/375&amp;title=A%20Very%20Oil%20Yellow1%20Modifier%20of%20the%20Oil%20Yellow1-N1989%20Allele%20Uncovers%20a%20Cryptic%20Phenotypic%20Impact%20of%20Cis-regulatory%20Variation%20in%20Maize" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/2/375&amp;title=A%20Very%20Oil%20Yellow1%20Modifier%20of%20the%20Oil%20Yellow1-N1989%20Allele%20Uncovers%20a%20Cryptic%20Phenotypic%20Impact%20of%20Cis-regulatory%20Variation%20in%20Maize" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492211/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492211" id="node492211--3" data-pisa="ggg;9/2/375" data-pisa-master="ggg;g3.118.200798" data-apath="/ggg/9/2/375.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A <em>Very Oil Yellow1</em> Modifier of the <em>Oil Yellow1-N1989</em> Allele Uncovers a Cryptic Phenotypic Impact of <em>Cis</em>-regulatory Variation in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5085-6240" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rajdeep S. Khangura</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5778-7850" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sandeep Marla</span>, <span class="highwire-citation-author" data-delta="2">Bala P. Venkata</span>, <span class="highwire-citation-author" data-delta="3">Nicholas J. Heller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-8101-8410" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gurmukh S. Johal</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian P. Dilkes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">375-390;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200798 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492211--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rajdeep S. Khangura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajdeep%2BS.%2BKhangura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khangura%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARajdeep%2BS.%2BKhangura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5085-6240" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rajdeep S. Khangura</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sandeep Marla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandeep%2BMarla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marla%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASandeep%2BMarla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5778-7850" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sandeep Marla</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bala P. Venkata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bala%2BP.%2BVenkata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Venkata%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABala%2BP.%2BVenkata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicholas J. Heller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BJ.%2BHeller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heller%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BJ.%2BHeller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gurmukh S. Johal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany and Plant Pathology, Purdue University, IN 47907</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Biology, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gurmukh%2BS.%2BJohal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johal%20GS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGurmukh%2BS.%2BJohal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8101-8410" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gurmukh S. Johal</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian P. Dilkes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Biology, Purdue University, IN 47907</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biochemistry, Purdue University, IN 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BDilkes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilkes%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BP.%2BDilkes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian P. Dilkes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdilkes@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">bdilkes@purdue.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=375&amp;rft.epage=390&amp;rft.atitle=A%20Very%20Oil%20Yellow1%20Modifier%20of%20the%20Oil%20Yellow1-N1989%20Allele%20Uncovers%20a%20Cryptic%20Phenotypic%20Impact%20of%20Cis-regulatory%20Variation%20in%20Maize&amp;rft.volume=9&amp;rft.issue=2&amp;rft.date=2019-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Khangura%2C+Rajdeep+S.&amp;rft.au=Marla%2C+Sandeep&amp;rft.au=Venkata%2C+Bala+P.&amp;rft.au=Heller%2C+Nicholas+J.&amp;rft.au=Johal%2C+Gurmukh+S.&amp;rft.au=Dilkes%2C+Brian+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492211/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492211/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492211/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492211/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492211/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492211/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492211/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492211/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492211/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492211/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492211/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492211/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/2/375" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/2/375.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/2/375.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/2/375.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/2/375.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
