<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/2/391" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/2/391.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/2/391.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/2/391.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/2/391/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/2/391.atom" />
<meta name="HW.pisa" content="ggg;9/2/391" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study" />
<meta name="DC.Identifier" content="10.1534/g3.118.200932" />
<meta name="DC.Date" content="2019-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Muleta et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7396781" />
<meta name="DC.Description" content="Young breeding programs in developing countries, like the Chibas sorghum breeding program in Haiti, face the challenge of increasing genetic gain with limited resources. Implementing genomic selection (GS) could increase genetic gain, but optimization of GS is needed to account for these programs’ unique challenges and advantages. Here, we used simulations to identify conditions under which genomic-assisted recurrent selection (GARS) would be more effective than phenotypic recurrent selection (PRS) in small new breeding programs. We compared genetic gain, cost per unit gain, genetic variance, and prediction accuracy of GARS (two or three cycles per year) vs. PRS (one cycle per year) assuming various breeding population sizes and trait genetic architectures. For oligogenic architecture, the maximum relative genetic gain advantage of GARS over PRS was 12–88%, which was observed only during the first few cycles. For the polygenic architecture, GARS provided maximum relative genetic gain advantage of 26–165%, and was always superior to PRS. Average prediction accuracy declines substantially after several cycles of selection, suggesting the prediction models should be updated regularly. Updating prediction models every year increased the genetic gain by up to 33–39% compared to no-update scenarios. For small populations and oligogenic traits, cost per unit gain was lower in PRS than GARS. However, with larger populations and polygenic traits cost per unit gain was up to 67% lower in GARS than PRS. Collectively, the simulations suggest that GARS could increase the genetic gain in small young breeding programs by accelerating the breeding cycles and enabling evaluation of larger populations." />
<meta name="DC.Contributor" content="Kebede T. Muleta" />
<meta name="DC.Contributor" content="Gael Pressoir" />
<meta name="DC.Contributor" content="Geoffrey P. Morris" />
<meta name="article:published_time" content="2019-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Young breeding programs in developing countries, like the Chibas sorghum breeding program in Haiti, face the challenge of increasing genetic gain with limited resources. Implementing genomic selection (GS) could increase genetic gain, but optimization of GS is needed to account for these programs’ unique challenges and advantages. Here, we used simulations to identify conditions under which genomic-assisted recurrent selection (GARS) would be more effective than phenotypic recurrent selection (PRS) in small new breeding programs. We compared genetic gain, cost per unit gain, genetic variance, and prediction accuracy of GARS (two or three cycles per year) &lt;i&gt;vs.&lt;/i&gt; PRS (one cycle per year) assuming various breeding population sizes and trait genetic architectures. For oligogenic architecture, the maximum relative genetic gain advantage of GARS over PRS was 12–88%, which was observed only during the first few cycles. For the polygenic architecture, GARS provided maximum relative genetic gain advantage of 26–165%, and was always superior to PRS. Average prediction accuracy declines substantially after several cycles of selection, suggesting the prediction models should be updated regularly. Updating prediction models every year increased the genetic gain by up to 33–39% compared to no-update scenarios. For small populations and oligogenic traits, cost per unit gain was lower in PRS than GARS. However, with larger populations and polygenic traits cost per unit gain was up to 67% lower in GARS than PRS. Collectively, the simulations suggest that GARS could increase the genetic gain in small young breeding programs by accelerating the breeding cycles and enabling evaluation of larger populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/02/01" />
<meta name="citation_mjid" content="ggg;9/2/391" />
<meta name="citation_id" content="9/2/391" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/2/391" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/2/391.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/2/391.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/2/391.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200932" />
<meta name="citation_pmid" content="30530641" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="391" />
<meta name="citation_lastpage" content="401" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kebede T. Muleta" />
<meta name="citation_author_institution" content="Department of Agronomy, Kansas State University, Manhattan, Kansas" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5403-5098" />
<meta name="citation_author" content="Gael Pressoir" />
<meta name="citation_author_institution" content="Chibas and Faculty of Agriculture and Environmental Sciences, Quisqueya University, Port-au-Prince, Haiti" />
<meta name="citation_author" content="Geoffrey P. Morris" />
<meta name="citation_author_institution" content="Department of Agronomy, Kansas State University, Manhattan, Kansas" />
<meta name="citation_author_email" content="gpmorris@ksu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3067-3359" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=F. M. Bassi;citation_author=A. R. Bentley;citation_author=G. Charmet;citation_author=R. Ortiz;citation_author=J. Crossa;citation_title=Breeding schemes for the implementation of genomic selection in wheat (Triticum spp.).;citation_pages=23-36;citation_volume=242;citation_year=2016;citation_doi=10.1016/j.plantsci.2015.08.021" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. W. M. Bastiaansen;citation_author=A. Coster;citation_author=M. P. L. Calus;citation_author=J. A. M. van Arendonk;citation_author=H. Bovenhuis;citation_title=Long-term response to genomic selection: effects of estimation method and reference population structure for different genetic architectures.;citation_pages=3-3;citation_volume=44;citation_year=2012;citation_pmid=22273519;citation_doi=10.1186/1297-9686-44-3" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=W. A. Bekele;citation_author=S. Wieckhorst;citation_author=W. Friedt;citation_author=R. J. Snowdon;citation_title=High-throughput genomics in sorghum: From whole-genome resequencing to a SNP screening array.;citation_pages=1112-1125;citation_volume=11;citation_year=2013;citation_doi=10.1111/pbi.12106" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. K. Chen;citation_author=P. Marjoram;citation_author=J. D. . Wall;citation_title=Fast and flexible simulation of DNA sequence data.;citation_volume=19;citation_year=2009;citation_pmid=19029539;citation_doi=10.1101/gr.083634.108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. De Beukelaer;citation_author=Y. Badke;citation_author=V. Fack;citation_author=G. De Meyer;citation_title=Moving beyond managing realized genomic relationship in long-term genomic selection.;citation_volume=206;citation_year=2017;citation_pmid=28381589;citation_doi=10.1534/genetics.116.194449" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. A. Bhat;citation_author=S. Ali;citation_author=R. K. Salgotra;citation_author=Z. A. Mir;citation_author=S. Dutta;citation_title=Genomic selection in the era of next generation sequencing for complex traits in plant breeding.;citation_pages=1-11;citation_volume=7;citation_year=2016;citation_doi=10.3389/fgene.2016.00096" />
<meta name="citation_reference" content="citation_journal_title=Cah. Agric.;citation_author=K. Brocke;citation_author=G. Trouche;citation_author=S. Zongo;citation_author=C. P. Barro-kondombo;citation_author=E. Weltzien;citation_title=Création et amélioration de populations de sorgho à base large avec les agriculteurs au Burkina Faso.;citation_pages=146-153;citation_volume=17;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=M. Cooper;citation_author=F. A. van Eeuwijk;citation_author=G. L. Hammer;citation_author=D. W. Podlich;citation_author=C. Messina;citation_title=Modeling QTL for complex traits: detection and context for plant breeding.;citation_pages=231-240;citation_volume=12;citation_year=2009;citation_issue=2;citation_pmid=19282235;citation_doi=10.1016/j.pbi.2009.01.006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. De Los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Cuevas;citation_author=O. Montesinos-López;citation_author=D. Jarquín;citation_title=Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.;citation_pages=961-975;citation_volume=22;citation_year=2017;citation_doi=10.1016/j.tplants.2017.08.011" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=J. Crossa;citation_author=P. Pérez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=X. Delannay;citation_author=G. McLaren;citation_author=J. M. Ribaut;citation_title=Fostering molecular breeding in developing countries.;citation_pages=857-873;citation_volume=29;citation_year=2012;citation_doi=10.1007/s11032-011-9611-9" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.;citation_pages=250;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A.-M. Faux;citation_author=G. Gorjanc;citation_author=R. C. Gaynor;citation_author=M. Battagin;citation_author=S. M. Edwards;citation_title=AlphaSim: Software for Breeding Program Simulation.;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.02.0013" />
<meta name="citation_reference" content="Fischer, T., D. Byerlee, G. Edmeades, D. Byerlee, G. Edmeades et al., 2014 Crop yields and global food security. Aust. Cent. Int. Agric. Res. 1–660." />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=G. Gorjanc;citation_author=M. Battagin;citation_author=J. F. Dumasy;citation_author=R. Antolin;citation_author=R. C. Gaynor;citation_title=Prospects for cost-effective genomic selection via accurate within-family imputation.;citation_pages=216-228;citation_volume=57;citation_year=2017;citation_doi=10.2135/cropsci2016.06.0526" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Grenier;citation_author=T. V. Cao;citation_author=Y. Ospina;citation_author=C. Quintero;citation_author=M. H. Châtel;citation_title=Accuracy of genomic selection in a rice synthetic population developed for recurrent selection breeding.;citation_pages=1-25;citation_volume=11;citation_year=2015" />
<meta name="citation_reference" content="Guzman, P. S., 1998 Effective population size, genetic variability, and gains from recurrent selection in the BS11 maize population. Iowa State Univ. Digit. Repos. Retrosp. theses Diss." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=Genomic Selection Using Low-Density Marker Panels.;citation_volume=182;citation_year=2009;citation_pmid=19299339;citation_doi=10.1534/genetics.108.100289" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=B. J. Hayes;citation_author=N. O. I. Cogan;citation_author=L. W. Pembleton;citation_author=M. E. Goddard;citation_author=J. Wang;citation_title=Prospects for genomic selection in forage plant species.;citation_pages=133-143;citation_volume=132;citation_year=2013;citation_doi=10.1111/pbr.12037" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with Genomic selection: Gain per unit time and cost.;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Heslot;citation_author=D. Akdemir;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.;citation_pages=463-480;citation_volume=127;citation_year=2014;citation_pmid=24264761;citation_doi=10.1007/s00122-013-2231-5" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection in Plant Breeding: A Comparison of Models.;citation_pages=146;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. M. Hickey;citation_author=T. Chiurugwi;citation_author=I. Mackay;citation_author=W. Powell;citation_title=Genomic prediction unifies animal and plant breeding programs to form platforms for biological discovery.;citation_pages=1297-1303;citation_volume=49;citation_year=2017;citation_pmid=28854179;citation_doi=10.1038/ng.3920" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. L. Jannink;citation_title=Dynamics of long-term genomic selection.;citation_pages=35-35;citation_volume=42;citation_year=2010;citation_pmid=20712894;citation_doi=10.1186/1297-9686-42-35" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J.-L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=S. Kumar;citation_author=R. D. Burdon;citation_author=G. T. Stovold;citation_author=L. D. Gea;citation_title=Implications of selection history on genetic architecture of growth, form, and wood-quality traits in Pinus radiata.;citation_pages=2372-2381;citation_volume=38;citation_year=2008;citation_doi=10.1139/X08-086" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Plant Sci.;citation_author=A. A. Kumar;citation_author=B. V. S. Reddy;citation_author=H. C. Sharma;citation_author=C. T. Hash;citation_author=P. S. Rao;citation_title=Recent Advances in Sorghum Genetic Enhancement Research at ICRISAT.;citation_pages=589-600;citation_volume=02;citation_year=2011;citation_doi=10.4236/ajps.2011.24070" />
<meta name="citation_reference" content="citation_author=M. Lopez-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_year=2015" />
<meta name="citation_reference" content="citation_author=A. J. Lorenz;citation_year=2013" />
<meta name="citation_reference" content="citation_title=Adv. Agron.;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_title=Genomic selection in plant breeding, knowledge and prospects.;citation_pages=77-123;citation_volume=110;citation_year=2011;citation_doi=10.1016/B978-0-12-385531-2.00002-5" />
<meta name="citation_reference" content="citation_title=Genomic Selection for Crop Improvement;citation_author=A. Lorenz;citation_author=L. Nice;citation_title=Training Population Design and Resource Allocation for Genomic Selection in Plant Breeding;citation_year=2017;citation_doi=10.1007/978-3-319-63170-7_2" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=O. P. Morais Júnior;citation_author=J. B. Duarte;citation_author=F. Breseghello;citation_author=A. S. G. Coelho;citation_author=A. M. Magalhães Júnior;citation_title=Single-step reaction norm models for genomic prediction in multienvironment recurrent selection trials.;citation_pages=592-607;citation_volume=58;citation_year=2018;citation_doi=10.2135/cropsci2017.06.0366" />
<meta name="citation_reference" content="citation_author=D. Müller;citation_author=P. Schopp;citation_author=A. E. Melchinger;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=H. Muranty;citation_author=M. Troggio;citation_author=I. B. Sadok;citation_author=M. A. Rifai;citation_author=A. Auwerkerken;citation_title=Accuracy and responses of genomic selection on key traits in apple breeding.;citation_pages=15060;citation_volume=2;citation_year=2015;citation_doi=10.1038/hortres.2015.60" />
<meta name="citation_reference" content="citation_author=J. L. Neyhart;citation_author=T. Tiede;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. A. Orr;citation_title=Fitness and its role in evolutionary genetics.;citation_pages=531-539;citation_volume=10;citation_year=2009;citation_issue=8;citation_pmid=19546856;citation_doi=10.1038/nrg2603" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=R. Bruggmann;citation_author=I. Dubchak;citation_author=J. Grimwood;citation_title=The Sorghum bicolor genome and the diversification of grasses.;citation_pages=551-556;citation_volume=457;citation_year=2009;citation_issue=7229;citation_pmid=19189423;citation_doi=10.1038/nature07723" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=D. A. Roff;citation_author=T. A. Mousseau;citation_title=Does natural selection alter genetic architecture? An evaluation of quantitative genetic variation among populations of Allonemobius socius and A. fasciatus.;citation_pages=361-369;citation_volume=12;citation_year=1999;citation_doi=10.1046/j.1420-9101.1999.00036.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=S. Bhavani;citation_author=J. Poland;citation_title=Genetic Gain from Phenotypic and Genomic Selection for Quantitative Resistance to Stem Rust of Wheat.;citation_pages=1-10;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.06.0025" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. T. Slater;citation_author=N. O. I. Cogan;citation_author=J. W. Forster;citation_author=B. J. Hayes;citation_author=H. D. Daetwyler;citation_title=Improving Genetic Gain with Genomic Selection in Autotetraploid Potato.;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.02.0021" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. Solberg;citation_author=A. Sonesson;citation_author=J. Woolliams;citation_author=J. Degard;citation_author=T. Meuwissen;citation_title=Persistence of accuracy of genome-wide breeding values over generations when including a polygenic effect.;citation_pages=53-53;citation_volume=41;citation_year=2009;citation_pmid=20040081;citation_doi=10.1186/1297-9686-41-53" />
<meta name="citation_reference" content="Tester, M., and P. Langridge, 2010 Breeding Technologies to Increase Crop Production in a Changing World. Science 80: 818–822. 10.1126/science.1183700" />
<meta name="citation_reference" content="citation_journal_title=F. Crop. Res.;citation_author=T. K. Tovignan;citation_author=D. Fonceka;citation_author=I. Ndoye;citation_author=N. Cisse;citation_author=D. Luquet;citation_title=The sowing date and post-flowering water status affect the sugar and grain production of photoperiodic, sweet sorghum through the regulation of sink size and leaf area dynamics.;citation_pages=67-77;citation_volume=192;citation_year=2016;citation_doi=10.1016/j.fcr.2016.04.015" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=R. K. Varshney;citation_author=H. Kudapa;citation_author=L. Pazhamala;citation_author=A. Chitikineni;citation_author=M. Thudi;citation_title=Translational Genomics in Agriculture: Some Examples in Grain Legumes. CRC.;citation_pages=169-194;citation_volume=34;citation_year=2015;citation_doi=10.1080/07352689.2014.897909" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=R. K. Varshney;citation_author=R. Terauchi;citation_author=S. R. McCouch;citation_title=Harvesting the Promising Fruits of Genomics: Applying Genome Sequencing Technologies to Crop Breeding.;citation_pages=e1001883;citation_volume=12;citation_year=2014;citation_pmid=24914810;citation_doi=10.1371/journal.pbio.1001883" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. A. Weyhrich;citation_author=K. R. Lamkey;citation_author=A. R. Hallauer;citation_title=Responses to Seven Methods of Recurrent Selection in the BS11 Maize Population;citation_pages=308-321;citation_volume=38;citation_year=1998;citation_issue=2;citation_doi=10.2135/cropsci1998.0011183X003800020005x" />
<meta name="citation_reference" content="citation_author=V. S. Windhausen;citation_author=G. N. Atlin;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. K. Wong;citation_author=R. Bernardo;citation_title=Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations.;citation_pages=815-824;citation_volume=116;citation_year=2008;citation_issue=6;citation_pmid=18219476;citation_doi=10.1007/s00122-008-0715-5" />
<meta name="citation_reference" content="citation_author=S. Wright;citation_pages=356-366;citation_year=1932" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=Y. Xu;citation_author=P. Li;citation_author=C. Zou;citation_author=Y. Lu;citation_author=C. Xie;citation_title=Enhancing genetic gain in the era of molecular breeding.;citation_pages=2641-2666;citation_volume=68;citation_year=2017;citation_doi=10.1093/jxb/erx135" />
<meta name="citation_reference" content="citation_author=X. Zhang;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_author=M. Olsen;citation_author=E. Buckler;citation_year=2017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
<meta name="twitter:description" content="Young breeding programs in developing countries, like the Chibas sorghum breeding program in Haiti, face the challenge of increasing genetic gain with limited resources. Implementing genomic selection (GS) could increase genetic gain, but optimization of GS is needed to account for these programs’ unique challenges and advantages. Here, we used simulations to identify conditions under which genomic-assisted recurrent selection (GARS) would be more effective than phenotypic recurrent selection (PRS) in small new breeding programs. We compared genetic gain, cost per unit gain, genetic variance, and prediction accuracy of GARS (two or three cycles per year) vs. PRS (one cycle per year) assuming various breeding population sizes and trait genetic architectures. For oligogenic architecture, the maximum relative genetic gain advantage of GARS over PRS was 12–88%, which was observed only during the first few cycles. For the polygenic architecture, GARS provided maximum relative genetic gain advantage of 26–165%, and was always superior to PRS. Average prediction accuracy declines substantially after several cycles of selection, suggesting the prediction models should be updated regularly. Updating prediction models every year increased the genetic gain by up to 33–39% compared to no-update scenarios. For small populations and oligogenic traits, cost per unit gain was lower in PRS than GARS. However, with larger populations and polygenic traits cost per unit gain was up to 67% lower in GARS than PRS. Collectively, the simulations suggest that GARS could increase the genetic gain in small young breeding programs by accelerating the breeding cycles and enabling evaluation of larger populations." />
<meta name="og-title" property="og:title" content="Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/2/391" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Young breeding programs in developing countries, like the Chibas sorghum breeding program in Haiti, face the challenge of increasing genetic gain with limited resources. Implementing genomic selection (GS) could increase genetic gain, but optimization of GS is needed to account for these programs’ unique challenges and advantages. Here, we used simulations to identify conditions under which genomic-assisted recurrent selection (GARS) would be more effective than phenotypic recurrent selection (PRS) in small new breeding programs. We compared genetic gain, cost per unit gain, genetic variance, and prediction accuracy of GARS (two or three cycles per year) vs. PRS (one cycle per year) assuming various breeding population sizes and trait genetic architectures. For oligogenic architecture, the maximum relative genetic gain advantage of GARS over PRS was 12–88%, which was observed only during the first few cycles. For the polygenic architecture, GARS provided maximum relative genetic gain advantage of 26–165%, and was always superior to PRS. Average prediction accuracy declines substantially after several cycles of selection, suggesting the prediction models should be updated regularly. Updating prediction models every year increased the genetic gain by up to 33–39% compared to no-update scenarios. For small populations and oligogenic traits, cost per unit gain was lower in PRS than GARS. However, with larger populations and polygenic traits cost per unit gain was up to 67% lower in GARS than PRS. Collectively, the simulations suggest that GARS could increase the genetic gain in small young breeding programs by accelerating the breeding cycles and enabling evaluation of larger populations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
    <title>Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"GorkbJ-F6eYq-Iv_RgwIy0uozShvba2ICq-SIrVcz98","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492134","apath":"\/ggg\/9\/2\/391.atom","pisa":"ggg;9\/2\/391","ac":{"\/ggg\/9\/2\/391.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/2\/391.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/2\/391"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/2\/391"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/2\/391":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022u5kjHVWjLVqWhThSz6GDq5v7AFAJiU9wL80yHLD4jwtEZ7MkqIGpxIEhnWn1ttZi82vPa181V7zeZoaYyVA5lcZaQgAP38+\\\\\\\/EiAnYHozNY+Yq3mXQ\\\\\\\/nhgn+EXgbhM27r6QVRLCyad1XeAXBxs+8k74bVBjJXG2T\\\\\\\/FaV+5B4idCUT62mnAsCGLC\\\\\\\/aAA2txCgTjzd1XfYCuyRQOTJmmE1oY8P8vZuuwBF8wMUZmdfkksuotxy0jjvzIV0OFNUWPj9QMlMGPspiVFlse0\\\\\\\/sBWvnyChbC7RslChzmposP4CKBsHrmryOAw+EwKZUijLtl3lDjCRCpiiJoZF8SxPNYkdwlDKs7RKC0wuYdxOVOr0Ta7LDAqBLwTSXJMkOf\\\\\\\/\\\\\\\/+frzGonYKHevqdEzzX2ZGFtLT260CPJrd2jf7qv5esI5LQOPm2jqHJDrB9lrGx56YiPxnxXZp3I4Wuup5p6Idh+QcE\\\\\\\/nOe4yIxM\\\\\\\/Hj+c8WoCKGozLUmfYY37jbBCie89xqAIwGhkDUaLF+MxJOS\\\\\\\/Rvj5X7sDENLPsGboMzARCkEyj4TBzRiHB2blfx41ms2rYa0Kg5lEi4pvHLhCeZWi\\\\\\\/AUI7ULv1WO7uz0lcTWN+x95t1CnPxl02pASoJuKSe5i1+wia2BNxO31LgVdOg2WdrbC+lLV7nR0No9uSm7FlqTr68nwAQ+jowoy\\\\\\\/WpSsN2KsN0HFslXjjgfr338riTJ8khZCXs2JqVFTfjOLlVNCEw\\\\\\\/h3FVMbCYcUMgaSvp27revEvb9tn+fOlbYYhQhIFgENDKt3h\\\\\\\/y2isdhbhD2r9aaHlbvMcV4\\\\\\\/ZDVuTSFZ6SQLC\\\\\\\/AV9SUFzhzbjB5ViV\\\\\\\/RLroIKGL73YA4Pt4801557tBbZVJnI\\\\\\\/yr2JZ\\\\\\\/+EFexJfSrf14amNlFvnuydfvgRSOViG9aMhH+elKj8y3LkwGmrDvnNrxYcsCATAvho67DiQsDkqMSIIPlCsih8+j5kU6XxOKMv3iFRY93g1NgGcSLTrwKlKW3FCh4W4eS3DkElSmF7fwNclAgLsXfZ6xMJX1pjIw+2Dmk1Xc4\\\\\\\/\\\\\\\/tHFMqfqTWiUMgAFtwVyeKhG7elsXv20X9taFyF5mMjtrMJ02azR7L0c8efYzc8ZedoSwq+pvIuDqDjvgLZTgTEROnO6eaB878TO5NEtgfA2OJdQlDh54RYw2AkkXLjk+67TUTM3VnYcfj5cwdDINdOt63u00TtD5MJvZrK8Odw1h87IJx1R3St8yiEtkNkJacSLdzBxTVSfiTjkSKMcfEO7clQWJlXhxy0a\\\u0022,\\\u0022iv\\\u0022:\\\u0022oMPK5llmRcdbvSdw4YiVyQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224c2a6287477bb5e9e88811ef51459a5f7d6ca9438b53fe2175450da0a7400f48\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LFY9FKCOLXdycgvMb81jeT3lZGKlzKT2RisinrLC3ekzzT4PAsIofhZkvR4NXDjWXfLMzUfrgpF\\\\\\\/SrAKNyifXVvVGKl\\\\\\\/bpxc4a10f11lVSKtKtGOMVWAQigbRKZxGjNC6Q9xRt65s5x\\\\\\\/YqnnDW2HoJCz7r7p2n3YFgpR0s\\\\\\\/1Tw0tqTJl7Oe6AdrYYV8syp6D2s0HfxUxTKmBDA4+\\\\\\\/VT6NjPVPHBezRF3fCnbp9ALg7KVR9cJKZRaRVcjT9Z9QaSDpUelD8hjQIHZ8hDNg4s7DFNDPMVOz+1330QI59o9bd0BHEipOBL+IdNvHTkEy5aviUXMK05EQgsUo+onPyCdG+trS77dlm8g9rNc\\\\\\\/n4dgR\\\\\\\/76SygiD1aRsN31MDWYgkBAb+ApoRY6wb35Lrh7DRmhAWkH0tzMcZ+o8Lim3fClrzWXfcHt5c\\\\\\\/kRXAJ7h0Upeodr+2LVFY3kwq+ArhCEnq7RQA8o46nYDVirhdfDBshXu2J0cVSDycuhTL8OL9ASeI6YjzM72jSpxYD4bVShajCW15OnVeWZMsjgkXHRUhDV3w94QnzjlhYgBRPbLo\\\\\\\/oszlAKt2JS6PB6DrDddxlEQfCumB7QvtmUbCT7Aq8YDLb7dfAUpvf1iVHglPOPPwIsoXe+Armk2O5z\\\\\\\/WOWUvGF7Tn9uTpi0cHYC55+nER9+gaqk6ycCQI1Wcapr4lNx2oPbVBqToAEObPtIZZpGnP4EmXcxdffXXUdJ65LjpugLF7g6H6bS0oUGD4le+no\\\\\\\/89ZuxYWNHj2uUxxlLMnGQyQazjWFFzoNONvRSVl3KQ7zpUNIvySU9kSYDblflrStM8ul45M+yODLKh8sLqEEaQ7CnoJG\\\\\\\/DzqJ7U0\\\\\\\/h6pvmPdvxLKLlxG3RuS00vlWpITPElE5LUhkM7nd1NczafO4dr9FNieU2xz9xpZ+Me7vjH5NuBMBD5UJpgToxq9x+gF1y9eL9Tvio+ho7DDLhxYUA6rPkPOdQN0bVtl4Fh8rTkFmvTnys525GfHVpi4PPUSpxnTHywp7s8WpN7\\\\\\\/Yd14q00xLNfqutfWIA\\\\\\\/EMJGySYM7dAJ6zeOQBsMrKA2zSHTnK2XgDR+JBK0A80oYY3kcjqVaYus53vThDOItiItaCLKwA34Da\\\\\\\/cL6uq6CBzX0SwQ0U46LR6d0U6QwnHYlshvPUVTZ9ZkKrha2esbuUUJJO02S1\\\\\\\/Cp8GwEQPQgu6DSpP8rBbk0mr89mAS7lbsZrzjvn8+R8rM7a+KH8tHGzdH2veSeqqLbzGkACEWqXYXDULnMtSsUgbkXzX+RR4dpE3+d5+hg1Voqz\\\\\\\/fWwWmPEyRz8xDBe12iciyNOWcQjTESZFRZ3dawcVxBdTaG6fOwqiHZWtidgBusQk8\\\\\\\/xBNptYi\\\\\\\/t3GmWkZWe4OoPEWFQAvcP7SAaucBAOex2TRWjbELOqGJjlxfP9PXA\\\\\\\/9dYpoc\\\\\\\/glb3P\\\\\\\/KxsZ\\\\\\\/ewa3laTo7hIlSEWp5Fuc0wuHKLRhrCMrW3G45UpqRVOIJLRYRqQfSRMEe1uN8Q8TdnCp1aZdpf5rcDsxkhNJ3fJ2M70TdGDPuiRo0nUfOjjLUEbFIkaObc3+jyANJhyZPnfd4QJI2ozoooj5LOACSWVifoHUmynM2XAnh+hwV0p1BUxn8fs1MB62ZQI3oDxgRQDsLv1N2VpYRzS1owNHtNZ7m3rTyu9qj7OYGNfszDqXIdPE75qQQwwl6O+yIWoHS9fUwqzuvz7Ej1cSSHASnhWZZH2Lb4sZJKMTaaufWnTXthb+wdsbg6skaHvwxaS7moyz8BH91CvPUEIcclsEWgHKww3UxI73Ehh90LWOpJmo+y6KrHEhhC14eQYRtJOLhVnhCjN6vKRJQ5helMfB5bHEO8x6apRWa+PNc3gMGyEP5mQpfx\\\\\\\/b0YszkHtf6WpQv7zv\\\\\\\/YbK\\\\\\\/1Y39tlt0DQQsFWMmJiKdvvlCqMAEpfEdEbebZuSrA2Eecn6ktIr9EfP\\\\\\\/B1mRB4NpzrhKKxDfDOR\\\\\\\/CWnkAZD6XPXi+3OAAtNfhuQJFa0\\\\\\\/ZkC3qm6veKZg2aVbZtkZU5MjY50vhGXoiUiigp4G\\\\\\\/4nGqCMV80joav2F5Y2Df8WS5LBgaPAaYpWOzYeAMXCyvzM4FXmiI5WBsg7ddZFCtC7rilUhUgmMdNs\\\\\\\/px0izd9Aicq+fiohjBgIzJrAgwUusUfW+w8gpy2YFi4gYFOF+mDpe2JY8Im7cUKjpCxLak1KW+osvi\\\\\\\/0LFU3jXXb30OXyk8LgJht0ehLqGt+mVh6cNLMJgIDUTbErEj62AlQ\\\\\\\/nhfbgeI7l9Q+Farzu0cG4EdOwzRie2E6myIpGa4skt8A3HKazQTyGuTvgnFG79sI6gZPhn3dQCwxhWXKXIJNVr83FQO97cN2OspEPEIu2AVuhL4dsrwT\\\\\\\/3OnYBq800qXBqPFGcl1XqYiYmySzVuesltr5A5sRt9MW2LE6ROGEHtjTlAuA094L2BimsTC55bB909OTywyCQUGAa2+hPbh+L\\\\\\\/T+2bRyN5l++Ndg1j39wN4tJG20meISdn+6gzIJ6dG4LHm4Tel3MXQMi\\\\\\\/wmFgTC8gGcmF1rQa0wC+3byOciFB0+eA4Dyx2OkAMFK8ZotCle0F+xN0QHjrhJva\\\\\\\/AZwTOOxaskmfuh4nvtBbCIDuaC+imZAt5QzGlQCS07Jcz1Go5gwSKVGCXknflAKtLr+0fO7a\\\\\\\/qi6C979A2ETqVBqMKt7d1Pw9927UkO3zB8Lk5YDnXNXmfHPeozYQ0tNV6ySwfEQNm1Pvyy9IwjeuOLsaXkL5q5TIOzi7hKNzTcWNWMBU\\\\\\\/se09Nb35KnNCvNFEWBXSHF9+sdjOvj54ONkCUIgYnn60M7\\\\\\\/WuQyF\\\\\\\/UdiYc4Vvh8w8T2+gzIRWgzi6vqrfWjJdmhaPtgIyV8sa1KaMss0VpwCeS5sVVTB7ebSdACxYUOIv8AOuj8gS\\\\\\\/rcrLSn9nU3oCx+gUO4OhMPwC3FtIahoTDP\\\\\\\/g27dbHM\\\\\\\/AA3KhHwWOaOB5PcrXR2X5UHIs7yXFoC55L8q0nPfENdT6P7uk3kHAdYOuQBiKYKk1gT\\\\\\\/uJEBC6Bpytp8Sm4mDvbzvU1TFeYIPhuS58GKnaZ\\\\\\\/23otHy\\\\\\\/Xu+8cuzYEShWW4Pg+\\\\\\\/m5R0MJ5MBTzQg3qX2UMCxZWCUoQF8LQx1VnX0H7g6rWOJrxRcMsTRc0VhXiO8jtmEqQ2WOdbtKNs\\\\\\\/GVnhFINDWIZJqpXZ2izHS+uHIgZzINOu+TGeschpRpMajjGgMkfnFy6idjqI7uwczOIV6mu5kWDvY0BeVL1RwAmqgjk9yJmBT3zWUmXB4gLG3Q1eeNIK+ZmJOZxC3SPlEjOZPmlfGlnmALKYXCgsA4qV2ecsoLXqQ1NwV1Q+7rHDjAzpNqd5O532cjmmo\\\\\\\/jLPo\\\\\\\/qHws+Czb8JafIq5ScTcjCyCjMRvd7nKYhvsnqC4CS\\\\\\\/YrSQgS8zIPdU9Pzw3EdT1f8kv5tR1Z\\\\\\\/3BknjSTdpCaSKzxGYvp9togZ\\\\\\\/6FETzAmRhVkavZaNuwsEj+UtXyzMC+eqC3JJAJWE8pi41uQOeBAxOAfYnTO8jdQG+arkjvMEyJMVt96z3yWpXb4u8X+onX79uHnHbEcbkvmr0ZTEt2ix3rp8vtkodSnIDDaaZcDy81kOv1qDeTOUleGdsJbTYpxHpELeXJr4S9URG5yqQ1qm9369ZqIYkvcCB8K5MU8akVttX3p5rZddrnsZSz7cAcvu2qjZDwpb6++qPmmUPSyMYezgcHi8NghsbPnZp1Vs3BIN4KmGc\\\\\\\/EONL53BUz+FuDnHC+JsEKkf+2+Axlm3Gw0QGgMESr67Fe\\\\\\\/i93emNmFG8i6g1sAy\\\\\\\/EKWwBc7agLOVVB5t2PwDdtGO7wrR9znP266hvvCRqGSBukeOaRKybdGFW3rL6DPY5X37XMluiXqy97TdxqY0xMnv8QRYYKKQuw8ta\\\\\\\/WfxRkwXA+AfOneMOV2f56mPlL3IVqQbqTtnaTbWVwaI6gHZE8EppXzRu5FDzv67xWCy1Ohkx\\\\\\\/kqEMewpp6NRhPG6jFn9FTmAr9JYCY4Ivd87DHBzK2GqVGhJhdQvtHj1bGaEirtg+cQJCLoyPGFUaqnF79LXZ\\\\\\\/jo65t5YlJJ2X2ntxKdPutCaK9CebNkEBLJ29Qfst13ObMXn0ScqFT1WpnhMbwTM0PSwN94AM0jnqEsB2l3R8HDck8lnvfF1EKCCUPoJmV7TLDoljsYZujHUpXHqhlRaOt30KFVOMoKvKoRwdSFPRbt8T13q8ibSn5R8r2UCSqDdsi3dZYhWbkW6v7uwiXAsWdlRmY5qKFb\\\\\\\/0AiwxCwQvZ99GQ\\\\\\\/DVg18bIIfxOaIjI11o2rwCjwHAKUDQtLNOuSjEm48SPaVUo020yu+Kq93VjYV5W\\\\\\\/5xcntE51wN7DG6c1JdqvVg2ujwNUQZFBt8YdBK7zBgJyMsHn8iDFCVJLcRaPJs6YTub4SgchrYq6gqn6aFblvvN\\\\\\\/jp55n9UoYLMsEHxzJTSlacShibPPgLG4oppM93JfGgQpRHfY3FHDnZvNTgHLOnnqa47DT3u3t5otE8v5Hdvv1Bm7TUAgC2wCaplO2AJK5E\\\\\\\/kzlLgeprWJpEt6bRMqjNE+MghnBbsBZ6VgLLA25qBwCAnfp5u\\\\\\\/dNhgLmTIvrBlJkEzSpFf+q+r41jM24LzHpGT7t5i2FjyUMTp8iRiCwFuZoDkF98CokzM1u7XbUzrOn1hCP0QWlSMdF9NqmdzrJ15sEvwBvtOcGEf6R+6JT4+2Kc0TXrU5KW6I1Pabb1z2vkVtFhnB7ZppA5Gy7qKl+c3vI+4f0H+AVC4\\\\\\\/+VZnj9bDWxzlIuO6GZpJAkSmwcDzi\\\\\\\/CDFSCj5auMtEtT6CT6WK\\\\\\\/IQSwGstMrL2fm5SuKVjyWpDBguDMcfqPNxzixFOlL9T3Hb1214vAlbhUKyzPOCLN\\\\\\\/GetSJ3yuIP7YsrjH++GiK1L+4WXwbds2FiJJ5q9NQBN4qQqFxfNPueYNGirfj1Z7hJLOpdJoUImP3QuWRP1ItpRWn30dfURdUL5xUJaG5e\\\\\\\/UUi3mdqtxehXxghjoxMemdY3Zo2W3XBnmK60UZedt0mwWwK21RWgC2Hkrx+6rZFZW+GbpflbSqZduteN78UH+VqH5fspiznQK3W2zq1TarzXYa19U15s2p1CZ\\\\\\\/LI1nfmF4q6czL1AyyDnIlR67WXEmg17HK\\\\\\\/FieoqBXi23kJkBmzxGDVRs75tJFVF2dEtecSrkbwFrJqiqWKki2nhOv6BOUiriTjShhugtwAdn6seAauWHxdtO4jxVnXGUQ3N0vknaci9eKdE6E7zD7M0Y8ZmpUXBT8kEi0OKzGyb2hSnMICxAGzLL8ddi1GJk6jDlUxRkfmlW2jIUYZfvZFxHZUkuOkgbj74VFKpvskUdrRRhSwnyDVHDqbu5RVTUktP8eFtECBdtgP5RdhFCW1r+kbPGa4qIutqaB\\\\\\\/4x\\\\\\\/QJ9mJNW5+iIfqqayQgYwnlRcP9nu+tiLX\\\\\\\/yDNJzzM9rF7FnLSsn7EU\\\\\\\/Pk6iH5b\\\\\\\/7+BK4J3\\\\\\\/mQEWUPo9oeXTlAYdnuaTy8Ie8YKurc3fnTZIsAqxX6q1JoQE0lJwmi218UWNjKZmBOGT6feTzukM546f\\\\\\\/HoVTiGW1Ux5V7qskpGMRtw4Hxfr7gKeyUehSgjo+\\\\\\\/OKEOcEhhFxpRx8XYGuBmR4lIk9njnNjH8YU8ugFdv1EExFvSgnuZVPPs\\\\\\\/nAEQkh0M6ySQ25ICZGV1wMhZG8b+AJSqHtApoAwuHJKyDZQKbOFKjQcW3NMqgYyLd\\\\\\\/ebiVG8+q+V\\\\\\\/xCiG8vraP8TUxfDrOsDIPge86z2yJOolASH3ez4TLi2+ePpXF7\\\\\\\/iWuy+FoHRlmf+GQgTiBfW0hF7NSdIjAygiD79zCoYMMeCWfnu\\\\\\\/PpjHVzz6oWdlgpj7wpYdMFRaCgRdzAIjnXmozQDFbzwpEzTgf6YJPbpvTivmDHAtPe2EtuTM5D3I1Ulo7t7vAP\\\\\\\/fy5GlUZE\\\\\\\/kH5qcKPvH2pOWdQxfRlg+yUEkoLcvyR51FFS3B8hh7u3uqUSjFiNjhIygXUzgGikbSeywNwmKsTE5yCLDYemGdVrg9\\\\\\\/XtMFgjzPIEXqKG5k5MS11OGLFlpjfyio07prFUcI0QZ+p76BLbk9eRlcxXF\\\\\\\/Xkk3Mqk97vERJMOaoCWmWpBCk9znAxDzAnWqOu16vOwbUGjjUrQGP\\\\\\\/YcckUwcFhx27rr1c588\\\\\\\/vrM3zrVSbU1q8TPCFDPAOcBlg9iFBywSF6r11i+nov5H0CtRutE4HhDabi833c5WeRecGBf+TV98j\\\\\\\/Ih2IKHDsR2pocfsblT9FAO+H46yt97KZHH9L\\\\\\\/CrNU2Rslde7EISmJck0PMS75LmyMMv2jFJPIfHQm\\\\\\\/+LyM\\\\\\\/9+HEhHemfFa0Tu8aabtXv\\\\\\\/+GczTTeUXQtLs3nBWB6Bhh8Cf1aq8HdIcvENu\\\\\\\/37YJO\\\\\\\/leyZ4Ish+W2j6Lmgqi3Sl3NHxI4lS+8\\\\\\\/NeJ3Jyk9hJjo19Bzlg1ECCv5k+pDqMF0FP77Uiz9FF80Zttyys9v2v9ug76csar7vFPygJ5XTYJhOyF9fXNakB2SwrpdPGG4yTS3eX8OKBnsm1+LPQV8jkNelI5thbO5Dsc1\\\\\\\/Prcvogeg8KpZliUQIC8Euf1AziIelVobLRi1LxWsOw25ytGEKXlnCPdG8VoXcR43t\\\\\\\/9TzS1wOw\\\\\\\/UTcENpHELure9q0I46sbGNeqgLejRKQeY\\\\\\\/+KZkohB5PH8ueCZzw6nKkB8eKGgHS8NjeSG3G\\\\\\\/A8KcNvAZ7op11DdtXkJc++S8rWDakUq2Ld\\\\\\\/9m4OGEb54IXF\\\\\\\/968mXbcRDuESgcexKPYTau+8jCbqFg7rjHqrKfppYOcwiAEf0Rk\\\\\\\/BqiZtwztI5TBlyoKXcFwNjqPxzh4vdXkJZqRBDtL\\\\\\\/0BG9xDSNEGFJ8++u98unkJpRRsp+wo2YPfH2HVg8J4\\\\\\\/gez1JYex\\\\\\\/vkodoO8ckeG8zq0vmEuCFrkf0SM6W8okuzdLTFIrq2umibNKsK\\\\\\\/vhJQNb9InAS2lqXeMOraLV8zOXr0COsWZilgxDvJ\\\\\\\/aZoDr13WBFKNEr4YmaL6o1hbMI3itD9wCK7HgNtT7HqiTip1Hafe4Pie8khKX0BIWElXhEsKtQrB4AewjBlqnTh+9cP+du6RWKiq89gCxAXonIbXgX1TaZQktQQG03S6uzVkir3LS9Ko5hb7\\\\\\\/+\\\\\\\/SWF68DIo8XNHroumy3J\\\\\\\/zLA+K5jfgvop9cl\\\\\\\/zlbTKPrkdB4z\\\\\\\/mXrAB4UWUNcpKMjTp4wDZiOqjfDBV0EUSWUnwUxA1iLVm2FUy+PMIFU5sgMJUB7NyITeqGzHFm4biNiUhelvUPXYoYeXl7lHNszZeSDN0YzsJ3ITISutwex7X9RYYck7CffnINb\\\\\\\/H55nD7ML7uL3V8HU+H8lrq5Kbxsgcup\\\\\\\/ZV0YY3v+3nPelp2Gvgieu3Y6s0IBMHNE7i6OQJp9d9x6Pi5GuVKNp2LlBZZbqrhdWqFYjkycO0o13bEwJMF2l13gs2xGxhzyUm6txOUVdtzbBTjitvA5GlVy8hQEeTOPwZ2mKAqEkCBOIse8otpPxqiLeP2rpOiDeQl8jrGtOMi7xF5ChoBcC9VYOlaHCWpLf0dMqbVvGxRw\\\\\\\/RVAbLQJiOkVYI+62QOdcMLr5Opl5zjCltVR0kXXnjUUr6KH1D6DAJzMda5cY4m7Y0KRQcD69a\\\\\\\/5xCPXHDfPcCn6vqTpiGHvnsgKrFwfQQsyr7kqjnHiqm9Gz9Gx8w\\\\\\\/B6AH\\\\\\\/HXU8gwisV3dPMjFpzUtYObhoYVg3OIKy+v7Umw3R2S\\\\\\\/tF3qdsBywOtbfjlpKH1R6rfNd4u4Dh5eucOV2bcmE3tB6oqAPhVfW\\\\\\\/jraEn3bEYeoUfIc3bw66XZypUGn0YU8uu1enGcMCSUyEiprMOq6enHvOql4Z2BPtWZD9KEJ0USIT7cpr274FpJQmIzpSz6OZo9ciGfBJjo3oq7AdpEsMj01zXrZWi27iINJrriyAVVbVz39H4sEwFA+VySXmsQkjH9C0zrNQ5KHYf9tjEy+v13jCKkV0fQb2ApPvgqCp3RxTpbeGC0mVaM1lP9pm1WOwN1nsx\\\\\\\/GCWKdAW7K3\\\\\\\/1ILcN\\\\\\\/5ZITFJpPgY63Poa1CiGOtAVaFjHADJlpeAqW3LdFncJFdVdzRNLXun2uicGKd2GMcgsOuMcT\\\\\\\/dWbuL1XEX2MU\\\\\\\/dB9\\\\\\\/ZnuvluaTh5FFApV6DdO3cuz3aFOb6VkXHGFZ\\\\\\\/uhSdW\\\\\\\/AcKiMDkwoLADVLcCValsE7NojnSQwzIy5Hl2ElmryxwWfx1DOYmYE476nKhSEWoH2ikT733zIU3dtBKD2985SqjAeWi\\\\\\\/6S6bk3OaWkqVr2JA3JJFKWhZBxEnVIcBFbDUQQUiH2xDQqs+tXaqdx9ZnCoJCcL0Q0iBguTTdx8sYomTXq7L6KEEk8LhLjao\\\\\\\/IudVJF3AjahCwUHmaHTLAEIepeZzT7x1PX76QcZFvNALrLB6PgctrA01B45tW4bGSGQs0SdsuzBAlJur6mjgDPkY70Bxi3dQwxGs3ULtEOiCbMp5uuZ3ldy\\\\\\\/9kydjGttC1JB5sO4Xmaona6GoSmW\\\\\\\/2bSVf9N\\\\\\\/TNEIUtikrSW\\\\\\\/Xp+oCmYedEr3gb9PjisLkHVFTrJCKLtUAYsEp37gIiMe+yQqo2fuWuP4wxlZQVkBuWFYPFdDo5WPlnetxyHBJlNBNniuDwlbTvxkC5\\\\\\\/yqHYSK6eZpIGaKa9+FGF7Sz9W7lGph3brBNlNr3KB1aXWFenVBRvWDcOL71rgtXisC5MnB1y1i8WGE0T8nQETjifbLag0dM6vMbC00VPJfmreh6XJr5rm96fBIIPPDg0L3b0vtwjL2mpNqM1awWHr\\\\\\\/BNzTF\\\\\\\/fybk65uAvCQ5xDxhsM7zcDHMa6+Oyom8jb6CMhXPgAmr7mr300H\\\\\\\/KwvDJJQiySUArT\\\\\\\/uNEkVqBD92ZTdL\\\\\\\/PLEEM3XTttqaICPsYW794wDNTju1VsOThgW1fihSfPuK\\\\\\\/KRA5e1taJsZOZ5s3\\\\\\\/Wt9JADelUHKscupYHczILJB3N69rE25adsoYslZ8dI2XU\\\\\\\/mOIqBQTvM00PCbzwu+52dZi9R2LAaEyJWDXCLYZ\\\\\\\/y+895A+p7h70eMd3n6+URHnTTEpJPjSMAAHFTz1HXS9vfxy0TxR7bnk5HaYUQTQcL3ekOF1inKbkmJvnJG83QJIJQRBZrE3Flx5Gb1In4kVMkCPnFOAq4sjjG+2pgjIoDrnXBV\\\\\\\/7nGN4v\\\\\\\/Zs+bGfppWaVkH4S5ZHop4MdXNesDyJyQ7PACJ1fnVT\\\\\\\/fiMwjQN3NDGFw45Piu19p6tCF72bgOhxw1QLrYt3ef7JBcFVgmyrruirLAlR\\\\\\\/VVvOjd\\\\\\\/BNSDeAuhHIrIavsCJAiII\\\\\\\/as3ytKuA22PW+Wb2cU2aOz\\\\\\\/SVXynXz\\\\\\\/NwWHyHFEixFh07NJ0jb9MU3TRFFLu\\\\\\\/kObYgCnwowAzEq1eXmrnnxVq+otiD5\\\\\\\/R7gJvMRWYwI5QwWESIpejNTfBlA7Ieqrf+uaSvww7yyUrh5iLv3IRuwBUHLNWXeQYViB3aSjD85MUkpN\\\\\\\/i+fop3hqbTFtujROLO\\\\\\\/6gymSmzffBnAd5I8qAoxhLQwEHbr7ZJg81Lb41DM69EEbBnf0tVpL\\\\\\\/HksuGKDsnZHjLUn3EQ+aJqXU4zxDLEXTFv9k\\\\\\\/lMqY9EjqXiwYBbwa73ZCIvRMByQalgYJCsojbO3bIpJVJg81F\\\\\\\/EgoMLEWAy2KfHRCv79LLAJ6qyXH1B8Wdh+oaFt5HJguTxm8sR51EbTwmBNJKdSB7vs1FCJmPftskg6px9bmDWA0IdF5pTnCxSTby8ZVTKYkn70S06iGxmR7vEMx92+D3EJ8lOwiq3v4k2qhjkdU1eNeiATW7Wr67nGRS93eKd1lD5mKFHc59b0x7nAvbI1ZPOJqkrhvvVS41KL8Xq08wuX8efQ1zYJGvDlvTYUD5ThBilaCMy3fr8OVfN9g3MgCSZ1NEtWwe4mqwJBx2ALyqSBBJdk\\\\\\\/Pyb2De+9Jbv4yN6qazZUMBjTkaRpf47Iy715WH8WIyeScAQPivr0\\\\\\\/lLiMhtSDl9z+FVAkSxKVwCav6D7d0O43N5K0DkDe617R7Y65wygfuA8cCtzw+tLICrj0yPEtTiBvtaHekLO8k8G7Bp43zPttTpzYrFCMyue\\\\\\\/IKuoctun\\\\\\\/d4DdYjOHdXTBdDKNh4+xkqoNUbcShNmONEBa2MpfTjdd9gWilsLrDKpnWkoPtyAH1cVN2MBO5fYB222tJV6fBgiEgfT0FpS5NNor+uaPEr8ButHe59kT5X3L4OB0o3ypyoh08xCOpTO+xD\\\\\\\/3AQpx3Bla9kJxKn8NxBR71IcxP9CJuEtn+iDJJRxk91\\\\\\\/LKWBwzlLrIA2+W58lYzDaL38EOdaD2fnTyDz2+EXmwmbED32R4\\\\\\\/d4Qy80RqmsZu7cnY0l8G7ePJrrsze99PVHucTkxrWn5vpWt9dDFXAmZ0SKe82wM4UDeotTlGZBVDZUs07IynMmnys4CW1T2tNsaoU6ca5ENGKfrv\\\\\\\/tXdf9jdHXNsNsFmFo6o\\\\\\\/C5ioDNeM1So5Ajib8p81I9r70zISHyYcYBug7tb7MetcqAEm3AYBHCs+eh5wLFtCfHmYk1r4CgXlNi5Yj\\\\\\\/H+fYnu4IS50exQYJcZVBMvIqHG\\\\\\\/QWrqjSUjkekhMhlbVAQxipCP3eQvf8ZTrXiUY4YRlFI8TmIDcWGOucSNgsvoG5qgozQy5yF53\\\\\\\/7h0YdvtzObWumsZb5Z6QpLUMrYXNK2SgYx2Fn+SsQYIUaYI7WR6S85f\\\\\\\/ZgMtVmErtE4BHXM67\\\\\\\/7Q8pc0fsNBgGbxK7C7LgC2xhtIT9VdqWCHLlyE2qiLZcBmC5HlFXkDtYFfTcutzKyUWLS0X0FdCjc2GwjoMfqLP1DXYvpwpoqFsWeGQf5EdafUTjTfLmiUTrpCSyAw5L8BZzHJnmw6m68WsWOz40BlM4thzoo57LpVJfUykvC\\\\\\\/Xq+bqPvmKHnqCQM4LyDgsJ9sd6KEgIskyJRxDtmifGj\\\\\\\/NOjCx2F77FXz8gBf6EeNRGaylx9UBlAaNN7fi59bg1l5mugAqVWq6JQRsLLfnemZbT9w5uowC7TYj0cF\\\\\\\/HRaJlJzdeCC8dFd4ad4kQpgXUePLzc4\\\\\\\/88fJ8ltJ5YXDzFyt1H1I5jgjr1bf7SwohrB\\\\\\\/0xKvSHggWdhng5D\\\\\\\/BUiLXLUW7wUqmOjyb6YCzg5uBTAE8zlYkNXrVNfjJatL7MwaGmryf5lLQ4jDaONY7Q3PhkJCvsOVOeeNfQKxMKDehZxvjaXIetR14m6zC+\\\\\\\/PH1VYhrvHpc8JUQngR6Ie3CaCyS7LccYqvVYNWilm\\\\\\\/tcTTaYLQ5044LylOId\\\\\\\/fnxEekmUopZRc4a3p+M8RwW0MzrWzJIsvl3OIttN43n3UZPL2lxdT7+MosPMyT4RsfqvYlENoBRzd0Hg8iNDCNtEgywLI7YDlMb215dksy0VnGmYrSYkdMsDq9FMX2DZvYV9KFrMbRSG9CKoqb188wWQ+frmThVHlyGNyNzzXw+PhjEf3eWEk9K5z12mCAMMFYhN2YLYbbaZA54G3wpvYMF0pszJTWT+d\\\\\\\/uy48js\\\\\\\/M\\\\\\\/yyRafFDBS5eBNlZy4o75yiXaPDwnuw6hVODpWjyraYzaBckkTU18DlOi\\\\\\\/wFjNRNDwHIPYXgoToCQcm7wp+eoonrQv0fLsRx9sPP7dq3x14qkj2UlFfrpCNEHnQ1SAhor\\\\\\\/ovbv2BPf7ebgEE0ewstN6IjFzAnNH8rtEg9VsVeyTbHWN3csrEvCbWWPOrKAoqmO7E5e5ACJFKrY27Sq\\\\\\\/J6LLwrjeRxHfv4q\\\\\\\/7tWpZ+6mF9eM4G7+HryUlTKvkBPxOvlFeM0qMR+SGMtyqaH40aFNnIuXwXkGFVjzFX6SAPD\\\\\\\/hbxI9jboNagMxtFFmX1rR5FrL6uKCquc0XWsaVA9vGACUe5ZExFkRW7p9abW9BkPETilXyruyImi3UincRtH+eTyzbi+fE8q06dWeWxuyiWPDraP6pA+gHty3GrBtUQFcbtRAGAJQdbVH7WeIq1QF3DE7tECHJfgQ\\\\\\\/WYvJKNaySANg1bPmfz+YaNf49QJI0+HevYAcSUpc1HvZCpQUWbIZsHgUOtc5aaJNthkoVDZEouPnYbVMJu\\\\\\\/slnvdC9oQc5MPUAKW9RAipmeet9\\\\\\\/sUiLjthhwfv+OQ9Qmrf8JBJQU5E6PWQVtfiNbFXBVibJABwsWBcEZXrB0H7JNm5Cva2XenNxAnEgr2urK+cYl4diToy1TnyPATyQPsG9AfEIzRjxHDN09mXE4YtTat0oNlTSYjEnjKYY0hpqk7MbkLvdQ29EvmYG2eOdV3g5qw2tKkeHS+b3UF35R+jz1RsijqsLbGXl0n3k9WwIUbspv5w+kAhOOJH+6d3Mx3nfCdd7qoWiAaw\\\\\\\/Xid1PR\\\\\\\/JJVcdSiReVV6ONQF6nzByj5QEE8RRMci89vaP1aziQZq9xZkaDvBEbuqzMtoHoRZRPvFWk1Iitg73JZLmNIEq\\\\\\\/KBaReqeX8B6w9OC10j9LDZvGjG1dKyU2l\\\\\\\/ylhnYmds\\\\\\\/0R0LvIAdDi+EeifbEXH0RrkrK62QETH9CgN\\\\\\\/aXdcuK1+dv6oHFHQIjrMvp9rEz6sxZg8o5UQrSMLvI8Q35TMD+kvQZYEtnlKGao0zI9EvyXmXTQ1LQfvMpkjQ6kSiUFiXtE1sJGtB4KqVbbu8NCvLf1daVDBvCf5d+4gPKhiS\\\\\\\/8pxcOlZwBPYLM2Gw7owIC6faUZ4LZeYehXE7Tyx0\\\\\\\/PmsSm9152EQLlg3sFIhD1KagKyns3Jdu6CNkyn9+ecs8jeyXLXDDKtVsZ13chfBp9x7jCFu5LXN9znO0wYIsn4aSM0qjLaAm5aeCo6YlmtEtEEA26cH2fr7Gu7T4ti7KLMrV7U5uDZLj9a\\\\\\\/ml2o+ZUZ2aXdTRzmMDSBBZ0kJqs9saL4JE1\\\\\\\/zyrmsikyPOTWr7swGE3JriEDTQjhGGyoGo0bYc0gOUnZk+QEFyX6Puaa\\\\\\\/jD1MCSrsTOWjZGvQXp9kkN2vZKV87Fw0CyEGXpI2PxicVVNWgy7Qc9KI2A\\\\\\\/DaRSo4kWk4Eds5wUpo8rw3VRlXLl3Tf+QuJ4kBgoQHtZijuHSU5SDDnqbMgzbku5g5gzK\\\\\\\/ocdbqCLY5kUQFlo3KOBJ\\\\\\\/fQka6YWy2zyR8MNakV8ah2mw3kBXBXCTSDvrcuZSy+scSOdlPkmoydFSL4of5PxgkxoSsDU3RyhD8T706LNHWQx1HU9Dr8I4LtQd9aazP9H5S6jaWwD2Eg6pXKN0EQuK5uWconJIgb6mAOsD12Ob+Sm21IC+dotfDGx8iv\\\\\\\/1WRINSvjICPsqK88+1A7oZewdsdCdfgwwPwnQpBNViNu0BGTez3HyvNLW+VCJ3gnK9ai4bUz9JbZTfqYu\\\\\\\/2Jv7fEw0M7aniJIM76y7\\\\\\\/GJktz4MRi4R\\\\\\\/f3HYpIN7rqmOGVVPmKrzPuYeiPwylBJo+7qTh8b5r78A7mbYRvsWnL9+7YK+9hZ1iRzG+QopECd7DjA524OM7\\\\\\\/thy9af+qi4oW81HkIOXTpcIt6EBMWW2Abqm99f7NN\\\\\\\/u2Z+HLTbzxzTeXDJiiF4EXipjgrFVkP5+QEqbImU7y0f5LZ5qReDo5tI0DuAgZEcPK+btqMeEtTrjEx7SEkmEXeX8QT1VUaLOxIf6Gx8Rs\\\\\\\/mrBFnkuEYrihI0kvuBZhwZY8NaBT7LkC7swTDSmkkRs\\\\\\\/4D6bqmpVvg8ML0vYaNiOjc1Oyt73Dcd6M5wWp\\\\\\\/5aFnm6U\\\\\\\/isDl5zlvfHbpvijjoOi4UdX74XzviUnZ7jKJOYM4yPc4ajZS6hBZZNywD3lJ7mkOoM5d6FVG7B0+v8A5YGz8gIgVO8OKTdGIyQPn+Qef6FopYrtah6qjYtK9wj1CwsQoSh2FoCdLWcqeHxiqYuITjewK2lDVu4N2Ne+0az6JsTPJQzQcNUrkOwcgqZMCQjsHyKPDTpdsYLydKNA4wPeuCsDy7YxY567iYTCLZnYkIkfzBBZqg0+6rti61\\\\\\\/g9SZQVlxjoz1+fJblPaKRkad9yMDZ5l4hX2a\\\\\\\/SUmLG8Be2Z3W9BSkgGKTOHE3UHX3ifpki2aVyjjivK4UC2HD9v4XSK8sQtzp5vcI42avfMmAQQ32Za3AVLbQtwlC\\\\\\\/cWAQmG3zG8FCG536w5WMwwjxhlynzNum4U5gBorqTWHrNi+GgEWZW5ApdYzdS4hV8RhtOPTmHUaaFSB6Ugk27XB51ZN4Rz50v6xfyk3xrjf+nKQaX8\\\\\\\/xdxdVk1AITqSNrATK2sNozkMXzDDeaeyB8rdVPTKYtU\\\\\\\/xNnUnskztIzh\\\\\\\/FM00H5rd5OjgFeqaIhe71WjDMhFOi0ri6cK9MRc7\\\\\\\/dXKOG34R12sd86BkamWsLkbTHCrmlxg5iuY1k\\\\\\\/f68NBshHtDx+3sDmgLn1FhPSUald\\\\\\\/qArGV7MnXJbMKOlli39koU\\\\\\\/ovKUjCHi+aH8D8EUwqx8ff0Nndx7y+xsdLRwVz1armtWNr68MjYIiBsqOm42z4JzSodc6WgA1tzWvGfSKXhQNjlVJ0+f581JBcl71ldeVdejyvUzlzDCXjp1KjSBCpGBFlnIZfEiEARQyZ4THy1HRYK5eq0S8tOUR90zSppY+Vb2PWILVOej+eQqhjMtWeEbdc89ZX9M+zz9fDDugTbSrev23q+BSk55hxqe4v\\\\\\\/YOsuPa5v3tImW6WFlLuvFXj4DxhvR\\\\\\\/36fJVWH5uhxv98BjbA\\\\\\\/I9Tbk1CfzBuIwkzBOduc0E1Kf36oZvlIcDpQKZ+2e1PKzj7oStOPZVwDWXkfhGJn6Kh89t\\\\\\\/Kx94CZ\\\\\\\/0At7x1cKnhw6g9cOgH3bwlRRWpdF2Of9PndmifitilijRDzMX7BKyu73LDjM5kVnd4ZknGzufHlApJ\\\\\\\/89XH96pEnFH3QWmFQ3ExCcxJIPQwUf\\\\\\\/mJKzK9RijX7P\\\\\\\/c5TJsWEteXSUQETiMKV7gXYs\\\\\\\/m3qf4NmjF3hB6mlUwUZ1LVVkap1gjnK9qUwrUKuZcOWdwOlD1b+sYqBcoeOpFSuw0PuXCN4RL+BL4yITdr4Pklx8NUz0O7Ckcp+pzVMlsLwANjJF2LMTvlXTFKAyTmB7k747SI5zxO\\\\\\\/bKP\\\\\\\/S5X01ZJ5H7K5rdm86NV+KOet4ZoUQvenSVwHpZUUGXW2Q1909+xyXzusRMb5nGcsQbqgBRjXjW7MEF5iDlhJKpmizvI8mKTy2plOeRmvIL9P6qJ0qlZ\\\\\\\/pi7ZhY+WEQLm2Oi2QtUj31eYKgelhIhBatcW2tdOyec9rXhspQdTPUes27uTfCVvylNPCQ4e0yqMsmLpvwimq6Zn+ZYcGy4djoeDYj290zyGmpvCBTrXJEZrjnq5rLhlgMu8kpW36HNdJhg65YgIGuEgAgbPaQNWt1SedQ5\\\\\\\/FmT9ZI8dgI4SAxaxh+MKvJMllc18\\\\\\\/ByDIQqxlCV4Gj2rBuJJWCCv3WHf8xnfqvdI6CE7lqoHCr267Fq+\\\\\\\/cLC7ec+366S3Qs5cIy3+yVpwqYM+q+tC4vufMWXe1MmddaCqaTvQWYkOu3Jz05ILjIH6UW6GsuGOf5xcgaqm2vSM+4i1Pe4\\\\\\\/ebOzsd57GM2kFPrPVdjb1mxqThFYP\\\\\\\/cNCBtsLVp7AOT8iAS5znkfyRwG8zlfui\\\\\\\/0UqyEhLMlEqJ4QjTA2uKcXC9SQoJWp+U4BiXs7Oh+NMAAtFtg6gXe6coNGzP9MfJfrcmWhW0NPMJWRvS0PCBGGqeILlrx1yqjPn6DCSilBlQT9NmhA+Oy6BkONkY5lAVb2xlOBJBPhLWhhF7RcmuwVX5O\\\\\\\/o67rKLUbj1seCVEKo+0JWpWYFVilIhKNaMlnkP8003Q5mF+Yop3LxaTy5iQt+1QfZb3memzYoOAtIFR+WzMk\\\\\\\/qFBd2j7nx8IpybWGREw\\\\\\\/Md4NE1yKNXGm+Vrqn8uEelW\\\\\\\/4DVBp257dbi5tL9DjRBAbErFIIkWmf6S4OjrVbSSzh+eetombnU5BVWkM95idjX8C0ozAgVUya5agMRBFeg4qTMRpeuJf78ZPK2AsE+Tg67n0cLd7\\\\\\\/FCF1\\\\\\\/FMBZ8nn5Zz6XvTliGWCQ\\\\\\\/zn6ITiJdhF9jXYYU+rDWvcdLJFVIs\\\\\\\/5ZGGFcYLcopPV9gINDcZ+zVvEjbyjEZF6ta6nPRtbd5COH8gC\\\\\\\/h7eZ1BNHRY4pVpg5ruK5ogvOXrjoEFyvVXF6SoXsHQW9ljrKDeWGpJUuIYjLW5mY79qT6R5zZo2iEzU5HR+zQgItIWiSP3a9Tj9PeNu2i67DgWuFwEGo2lvCIES8eTKFtV\\\\\\\/Gj\\\\\\\/LqNB9vf5BMbJybQ3EL\\\\\\\/eaVHKHajVJtsFxpFGwJpPg5FtFaDOK5vMDBOANO4Nbht8vsy7za99kIz4rRj\\\\\\\/BaUWzuYzP04NnXNByeW0qrVaxlgGt2imA\\\\\\\/WbfQmjHI7w+yu3GOZ5syLL\\\\\\\/GVTS8YGC0acJYwG19HnZ1faDt39Lul\\\\\\\/v50Ij4TP5nsAKU33ks5yJtmtsm2KMFwq49eUNagrCIG+GztMGy0f75ceDJRnF4wWg3ArE35LZ30At0jVcRj8sWgxFag8JaE+6VzuK22Mk557rAYglrfVF0ESaYsPiNMxQdCy6dcY2VQ4HqL9w3ZzErGhvri\\\\\\\/T5ppNIt4pRjjWop8zojcgM4WV\\\\\\\/xGbJwjX56qlGpic9FngY0iNvS7MgrnldPkPEGWdoPZeMoo0tdnZ52jlQ9LjfPt7qgHOSTIvJxPPIHNo1R3syh7\\\\\\\/QWK9PCBapx60w\\\\\\\/hZiZufG5linQAfljkBI4BNfifxZIxq5ChrCSryX9dTzVhdvte2D7CY1AmiLV7p\\\\\\\/gtYA5VmBcyTgF0KLWLNgoXL0kwydqDq5OaysvTRaOCCYnlczmSttItpW8V0vIs+0u3WbuHJnulFmMl2UxO8ARsUJLL74hXZjZTtVanNVvXV24cxH7LHeTkzTXrNunvimqyzLVit7Y8U+dCiacA7G8gojjZ1ndmviMzs4TuqLxkYkYzV+EzYcE5MKnCH+oWqrbuTeHSStfZqtYzDtCnadNTPJhEt8icNGz6seVFsdo\\\\\\\/Kys83Ck3GofPhS4GAn714lVCLE1jvmsNGbRrUqQVl6O085Zn0iNBgzVYgqM9oaq2jjoTm3Iq5AphJuqc4sSrO2AFARX8TaL2x2pB3jTHFr05h3kqvQncRZKDoqnyjF9MSKMYsgyJKS96ChRs5\\\\\\\/Ca\\\\\\\/ABZ\\\\\\\/EcyKQPj4eYouMatQRa\\\\\\\/biFoL0jEM5EJNDxjYbt0wI\\\\\\\/1imuPLdbJCqa\\\\\\\/PHi98D9055n4RtAr43raUY1tGJclnUAOm+wHqdhOTfLmRsAgJM9UzmPLnLnwINN++TcQFwJkSCidUtJUHOxe1pxBP8oZGuGPdI7wXw33\\\\\\\/sI0HI507oNNA7iMW45pAOehvHzwTfVlWaMtbhNduzl6j0SUfDv3OxEjHoJXXE29p\\\\\\\/ispMGIFbYql\\\\\\\/n1CzIeI0dh2T9Ekr+5qlMS6E8wA6NcsryFFCSBz76FBFzNSnykCCsqsE5Kd4ksN6U3H4FTVsubhnWUzi+Vuw+ef3n4tyf2bROI\\\\\\\/ggw9YiBlrBryYDkRuPHKsIEXgl0iJZE3UbLHjZeNLfxhT\\\\\\\/amQ3pudMBEYvW5uZ2x3jtOmonn2yeNoGC6bMFYYfweXYDqiUuuj1jPTyRqJ8A6Pj8b8NI9bX9Lv0j41hIe40d6fqUG+JiyG522iD3uI7RNHwObk9njTXEiEk2mMSLiWoB8fzXRxfexu2AMQI8c45RTLmSi4q5fqsN2x+yuGYRYkPnBuGoYZ5Q+x3ZFYvyBWZnBVcEa+r\\\\\\\/4KxTIE3+x8s+7c+NLTX0UKUbQAV00n++XH62IWWfC7ConqHiKS3DbqitNwGMiyqi0LMJo\\\\\\\/jFyO5l3g6ppGGiX8lhd12OysWil8VM1RuhSjYmZvy5FDAgjtLTb0jcJsKHFidUZEcjyDx6FnP8YZSTuiJw0yUrAnpU5mD4y6yCDcDNPlo7x7mRGbCSOT5f1LiZbAZHdbeUuoG\\\\\\\/sRyVt7rLTXlK2ZhGufXTheN+CTUVOX39fnoETr2jbXZpYUYBDXMZExgJpPHmf9lJxdEejjGzqGKH+LaK+YXRSxwQOHqV0HQPPA3kF\\\\\\\/DyeNQW4dFdkCUpMyxXKgzlEAuE3VBfeJWfag7eUCWTMHAJ9kjB8+IpuC1sqda6yzfg+1RV3Qnry\\\\\\\/76l\\\\\\\/+mghHEWhWHANImBwypTWcHd4qNBAEDig2yDi8NCew7oMzj94drsQQBX9hl4JECPrHWMfMUgKTb8fyo7G5c\\\\\\\/l+p59KNSWemaOhKNgokkRV6NB8EXBw6caiSBMMJ+YljjNP1J17FR8VcKSjuP5XVu\\\\\\\/fF1UywBebD9KMzur0xU9AOa+rGFc\\\\\\\/0C3Rimxnx3A5RxHYUjpiQRqGSTnQIX7QwSeJp595OabBISjtU5bjAyH9cC7dhLRPWohnwgfelsuZ8r2VE68CzhnM8RcR0HIRpJINB2URgIzZUVnI3\\\\\\\/Xaznddl9VCtp9vZv1XGqF4YOe8wQ1LozAx8vBHgQkCzX1FqxuOxg6I2ksTIuGb\\\\\\\/VG7B+JPeHt5pDF5pEA57Hi\\\\\\\/Ft7y\\\\\\\/WpEueJ3T\\\\\\\/f5Xz4Dus96yGlWeCOvHe32WgbuLYTrz2iUc4ZyQNu02Q\\\\\\\/irppXmPcW+aLT58SsdA9A17+jcTCtLUMieEAG80DY+XwFF\\\\\\\/wp8TxukyzJjKG7VXcrk0TvVSCGwcBJwC1Yu1dSLHAINBdVIZ9A13dCQOdxXNIVqrzr60l80m9QH2CNp7B8jDzIEXpAFTv0iM4N7mZG8Lf8NPzJZZQePtusvK2wiPfE\\\\\\\/OH\\\\\\\/kMpZFINOk6dnRuphp80dfMWxnEWTXzBCEuUufsWBtJ2517CywWK579TY4pBem2At2fejHEjQ2bcegYFQPunDj403Kj9nwIGtmPMwiziuTZqQkku5yHLaqjidjkRFHtrfECTzF1zYz4M0HSK8DofHdg4fE7RSq+kEffIIiPqk8HwRscgSAnBcSxpvihc3N3oZCgcW0y0SQk270OmFnqIkorRxQ1+jsae0gsiALw3bxeW07KudMKOs8tccYOyfo7ixeIU2P3O7U2pJYC6L2Y21Au4b1IVH1LMu9+hHU\\\\\\\/Y\\\\\\\/J89Xrasw1Vr12XmvtJFI+qufGarrTDhRGzBBnXNXsEvCzrUW7dO1LTy13APNV20s\\\\\\\/z28+x27pzZ8OwO3ZomVg6cr1Ek6uFMDShnO+Q397vKRsJEAUEnLtBUdCZxzIE8Bwi+gVx+bOHms+CYuDPwOCv9TT4+p8wNAeqlL1HrIRsWdn5yb6LcZDkIkwIoZ6z76AJeWtYM8S1AIs+BTzuIVY\\\\\\\/TL0BN7i1DK9tT2bSLWc\\\\\\\/3i\\\\\\\/fcMyroNLhHL4tdFHzQUpxtp9JChZLybUYPRiZJodQ1DL\\\\\\\/P6TafORh+YXOsX5RGkc7You5bJU+NQ0XJFKRLB\\\\\\\/ylI5uxDQdoP7Gd8VSIUPHpLWSfZpoas4v3S4XjCzmLjeZgw7LuK9T6cwYcce+Bw9iVFKKKaSs7vAYiyP03ITmK1uqN2bWnzq91xSUr\\\\\\\/4p\\\\\\\/qVhughhYZ4I51KKp15HTXu+doVyE3cuXTfoLGITG4fnSjtuewRRsi5Ez6fQGPf2KqiQbBrX5GqizFo1y\\\\\\\/zz8YymvXGHq\\\\\\\/aRRKK5hty3RVpqdtRMhel513BaYPVGjp7QshiVsH+7KyaeXEeh+\\\\\\\/qv6\\\\\\\/03qoOln4I+1dJtqX47j2JqxS\\\\\\\/A3cVrrJVHZagZUrcHBzhHYP2cKUx+bvjVb7CtsWNQYuE4NHIq+Sw\\\\\\\/OY+udQOby\\\\\\\/2Zkf7cn5CS0LO3vFfAv\\\\\\\/hIj7JBdPlxfT5s\\\\\\\/SiLpOVbF262FaG3HOpnzkPSaJXDRJ5kw11ZtkoJ5gsf+GJn\\\\\\\/djf9BJoH61qN677euMVQEIwZG18hg303n+C4Fe159WTIovDrDx1sELoUTJACAiX9Lfs6iZwR2LvhKpBcJYzxCd9hvnNeQ4EqHnoiDznY0WE\\\\\\\/iZ3pFtUGm+j0zc59Vg\\\\\\\/v4cwC0rXEtbH\\\\\\\/nossbrCOzKven4f4HrY1\\\\\\\/T9lIb3l6fmk0H99DUxoH8K2DfWWMftbH75aJtfQWSTVeTKtV51nvLIWmtZfWiAiBv4d\\\\\\\/M4tfbTKoll54tvJjd91WK7Z2AYOQHJupdt7G9U94mKAPdbO8M8E6ipnZrie3\\\\\\\/Go0cgO93UCwcDnEUPfRV9Rtr3KNXIb5fVo6C1KouHH0\\\\\\\/2z+4ABN6U122O6WOMWFPcyN600wEj8+kqLmAnE+x3rfgb5LuNv9RT2wRExRfwYX8a24aKQdvnxJdXDzSYl9RbSMAUwWjZ+VQgQJHK3paj3rCtL8ktZj+7MRqpnAWEMBPh1f6qZafgZCGMav0Ta1\\\\\\\/eBGOC4ynHZcgo89zjDIP0TAAZ9tLzLvf0ujLDiPh8iiBIc98r2Aoh7ZYEFkbIDoZfg5MJgfSUzR9C2gFv4eVZuynU2uENF7q3rPSw1pX7AP3AFsCYuazYP8mmKfosn5mzlI1UC4cgZFFokAn2r9FySjxJvwb+yVdjkKgBzDGq5y6XDDeHfNpxiFyrfCITg+dAENxETrDFH10JEuGlyfWHlEdoESSt2mtsnbEkJXuNagJeUksey4KoWpLkaVkXDfSuFzy2Ixl+f0dyLMiioFfBT0y\\\\\\\/b0sadDOyxBLh8w83PZghdTcp4qMyYHVvyAY56goJVxpnkpxSbWf0ZafvZtjdZBSURq2gc7wKCEA4vdxrEDA2orhrb3mYh4\\\\\\\/80rBoBjjv0mE3Dd5GTX7ML0f2XS9O3i8tWA4\\\\\\\/17TRHt7H2+Tp8L51RJWv+h0NnbzsvMV4NxwqUO8r1v9iIvFOV1UviZMrtiTkkfME69jJPuXuuIwvxqedJaH1LIU3mJAwoDMbPH0HNSBMl2MSnfrVkm+i4r3EaJWhwVx1Gt7d+PMtY7Xazc+rblCLYjGfzHS\\\\\\\/Ei9WuOoWIX0n3lYR67c443Ay6YJIo1hkmUelZvCVD+I5giuI+g07mcBE4Xv5aee8d9WA2UE7CIsyAu6k1pMlZHk6RZ4cSI+Sn68qvZS8HrN5\\\\\\\/LYnan7Z\\\\\\\/NphORO5KrQFplnjDiVap++0vf8kwwZgdzbw8FAmryr9rGb2UXb0q3KQH+nEOIAofkFWy60sLV6zy0re2k9MPoFyXXH+cYybhQILauhtOhpOThaVHZ2YA1jSjAOSQbyLqW\\\\\\\/PiktdId7uAV4PPVibrXl7oVBZXz2TViHUdZXf7m8FEZbCQgfJEvchgXgnFRDYN8LSxWxBS2UsmBW+eMMKN2Q6Pf33ciFmc2Vj6tKdgGH\\\\\\\/CwhibuQY9P\\\\\\\/kWaEJXE4eEONmeDwM6zUYa9x\\\\\\\/KPjWOBa2k5ujcM4iYDHge4tEmx6zZvRTmk\\\\\\\/N9bMMYm4XvRQh+wqs4allh7BjBJclh0glQ6Q+9o\\\\\\\/1\\\\\\\/pyw5250ojJkn\\\\\\\/kiPJYbgGZHZWG1cKle8m5PbW+nrl3xxy+2Xuuvioa\\\\\\\/hUsUJSB9ivrCVkBdSJrH02EzXwyVP1wcJ0359xzM3cVkoKwF8N0mr\\\\\\\/Mu2TmnHSFcU0o\\\\\\\/rZ2ZmbgcOhJJ8V21W8JNEFmLcvm7kdUctAXwU6IR7znM\\\\\\\/MBBFZiLLFVwZg+EeRQOd2xKCcjgUTfTtKst\\\\\\\/6IJx35pYLzn6k9bLFJTVriN6as\\\\\\\/7xGkPbIjyP3gWCcOPrv6lydNTjV6LqawppgVkOzJZaGANVezKXIt9mhkudf+L720T1TtYrivHjYcRTiqK0WEU4IeLoujYCPKXnN4EnTonD7jmoMWu94gK0rR7juaBPDSO4V4jZhwgQ7iQ8MHhk442XImFHwsYOTcuQXDARo8Y7+hMt3IRiVYZp8hNXerdLEj3MaMCD3J3Xli3jnlzKsUcyR2tFbgH6NzXrr4RxENNLlldpLCqE8sklR0iN3FqnE5W4in0p9p8PsSSwwE1cXZ3APL2SJ0A93Rawo1I1MOxgcrrP9XPMEwlb1jlssBKhX8rB2nv4Qj3BjNZrYl07zdA7SirxoCKGP3twc0ZGbF8GPfYOp\\\\\\\/WZT5zjfMhf6onNhpxz\\\\\\\/1W1gGZiyYreBYX+oqRf8l73c0cz1vc9QncpKmbUFDzUXZBFKToaMWgnzavvK3dwvN4fXNU03OLpaOPE2O3ctDwAPpjhY\\\\\\\/4Bw4I7jxyGm+4PssFb08O+F1lmWjFjA5+V+APUXcvg9sQClcmyL6t7sEhG\\\\\\\/g85IUcBmorQleNI9yt6DulDwEQqCWDi2h3e+RpFRP+owvoqvl91BZjjRGMGsxglAVUCoiKXTA+ueQ3AjYblyUqyenzeAcDP1EstcOM7sf6rUJTbBoZRQO7+\\\\\\\/0ABWLnHFNf4EtYYQ3je5KQpLRl2TcsOQXVN+BriRPGefuTADyGdkM+QDcFvMaDKDskH3hZjzzJl8faptsjA+HKDcu5J08U0XcB3RQvJfbiG7a8IVtK36yCeC6KGpFfpZvG5xghvudj4n3oVuU7tdmHtf19zDT2PR\\\\\\\/e3EjwgoMFrd1cPNdC9DXbA4ptrObGyWdEJ5omqsEU62OgjNCKkxoFDkLeSGGklM+fT7QcjyaM7zAWs4+PNIsrSEAWaZIAjlXqFiWlMYrmVb\\\\\\\/RC7gqums0fMdWoXUDMR\\\\\\\/1DPFbCKHRncm6C658UbtweF+yMcPbv0AXyDD6doRLqbaIjC3UPMaL54MfrD5O7mb6syr3TOvY6zPpMlizH+f6WGacDSvb5YpKhHhx83IrxSJyMLgeuwUZXFwitAiSQhAFF5s5qwNs2+s2dEHipjC4pcxstPeRlq24JPyGerMXJ6lH\\\\\\\/7JcofwvpyCyPpng6QFy5bR9id7lzJQ\\\\\\\/Sy8nArAo6+8H6izomV4ZE76YIxB5yPdNiv3XX8oYSarhet8XCktLV7LwPiAQBQnElw68iobA\\\\\\\/ardRJQNsiC9SZStPgM2\\\\\\\/XWbVCZkN5z8J+xjfa0KUhIIkFZ+aYTtVeCmZ5\\\\\\\/iLegnxSNgYWGInOHmXQV3dFqIqBejwT1i\\\\\\\/N+leaGpb8TSI7iqy9URu3LIAGwJjXtr\\\\\\\/wtdGNrGiDr6CIpymdBx4KhUcOPcERC\\\\\\\/9Gyzoxyyga8llowSrHSxP2FHSC\\\\\\\/Q5bmH+DZoLZSth7XvEyIH2mHjJ357zDuW3Ln8ocG9UPKpwjzTRbB9EShqPi42HP+AsGoVbo0qWr1cBkapNY9IppEr91rD9JTVY\\\\\\\/wjKV+JFTbsnsgqIlSKO5twg4LbNLH4tHAh166d4W8bk6S0b98dx9lzu94RhboHV9DiJN503Se03B9dgxaHWKqbPOxfJLcy6JeabC2+iAg0yGa1ptRMTRK3qFJL5uUnJXcRszl0oCGHzwtA8LpaXOwgK74Zw9Q5OJTdaK5etVks9QkSMHOdb93cJd97uZRXGE6MM2+xrGKYiRIK2o62NOb0D80x9zEqGf5ym1yUhmcMTTcof5oiheY2AWOgbBSlY87lHlAb\\\\\\\/nU5e7AgkuIA08j1+08sHwhjJzRjaB3mgpDAwBodwuPVrZ5po658kXPYKGdyWwH0efoEltzGGmp1OO\\\\\\\/pkyhfZbrZkjB91U9JPEdogOJ6OvvQb7Qa7OtY9QRVBB8igKTaZDPBDa0gtUkLGYyvtZdcGJX6bR1vkyF84AJqj0Gme1v1oJUfluUUZdcrUxlSOYFQGksY0cx5nUHFr0mFJW\\\\\\\/c\\\\\\\/H4G5NrsBLdHVeazA+FQHwVVCyf3Eto655TYxd8LyG8WAAG7+eYH5IzXLxzCtXbVkJyMf\\\\\\\/4Nv1MNKJnYVFR3511b2oxvIWZ0HFQTsvNSSW5MIjATC5hxmNVCvYU9VCS+M5BkdoqsCgaSYCZ8ZbNw4WQPmzriGbUkkHbDgCi29jBdUe1DtI\\\\\\\/p7IafwNv\\\\\\\/2x4Ykca5eiios0hK4XyDEiUw3Ojt8P1Iwe2iEF\\\\\\\/npoP2zJFZRe2Ag87+JhkRYBRyoeOLn9ey+O3CZPD+OHBbr4zkS6BdbGdKdGIZndk2SQYKE3MgyJ0nrEle+e7kxGj6lyBBB5q1WNxGm8+2tzLGU1QdgqPLtKXR+FAMFJ6LBdLHaIYC4miUk\\\\\\\/++5mn6hQmr7AMUB5qZGbzdgZxkOfi2m0qAdWrGKKPZ0KyDiWQrZe00JkJIyhiDxS4c+t83IJfOzGJsBwi1GbmmdvGA4UiJZe14dQY6ASMQ4b7aw6laQtSvSRfdFcn6AAFerOOGUu9AX47njhJTyug+1ZnL951bxz34gdWhX5GnQTvHLj8M73iwxKc4NUw3gYlZVAOQ\\\\\\\/+NUR6plP5hznDDjpCmataUrU\\\\\\\/+u8t7W2Kd\\\\\\\/DAkenfXNrXL3Wa7csBQ4VomwK0Btjd56JkkHlPpOJ0DJ5ou+z8kpf90m3BoXaN7ZmblFI42hKVPenEYogFp2\\\\\\\/L1+GiMdDEP9Y8nUXkM8KT\\\\\\\/TetlmKfgC0wElOmfOmrRzW06dOI0h3LygawpaURNAI7FNFRw5NywnqU799HTBkmMceo65r1TTqZewNZkv3MOU8VO29ufKaM70VwDokqaESB3gcywmj5V\\\\\\\/zzv7txXLH\\\\\\\/YXAXINAdt1wsjBHg4PRO2SiibzURFwIgGfm4+IwwW\\\\\\\/t1jp0\\\\\\\/yN4mKwzPuTfYSKbkf\\\\\\\/Lr7C3+jvSE543iV4poFUmQt7qWVB+G48gBP9Pypon54KYYH2g8oWQkYI5\\\\\\\/+OxskeIJSbTPZkkpiKcSXD5fvZCHhME\\\\\\\/58pslwnmP3+gS2dEh4BchHYcISfLsywbKXX65\\\\\\\/s334zGEDCJGcJKHlhxqjr5uunevnp8nqK7Nvl4d+JHr3zt2Ts88ygGNS20erJTJ9wET5jOSQUHWx9kPZSSsGQmx6RTvYEm0VkJBaHaykDZya0D4+gVrEon+DsL\\\\\\\/vAfoH43gavZ5kmx5FA\\\\\\\/JvahGGSnJ\\\\\\\/ueEgSQJnqQSeGfp4fGMxrPvh9eEKIs1Z2ilP6vTHU1wM93QDIOiLTVWt8vzsjCBRVgyZQd+R2MSQPOJEbrx7qZO2XRCe9ZJ5ynwLeVQz5xRFaj36+ht5mXgiAlzJ9HZc7lvBDUfhBXXgafSr\\\\\\\/NNlEjoabKLux12KxMFHjCiFz8cxYifI2FKEguCzEwPdR+3Rxr4MFmIOATW2Qr6JaZ9mSCquPf75IjlxJ0K4sPk4SLgVJkoL3NK0HDw\\\\\\\/lBEFY24LH2jEqb+RJL3vd6b2aLDLLzAPBbnVQp+4CSNnm8UenpsfU6Kzaigo\\\\\\\/jjXqNTvvXtIR\\\\\\\/dSjFooWIoidfKr30M6lMD9av05YlpssbpfijUPvVgmclzev+lyy3NqC5eIoVqC9QZdYZ2dNMbYWT39+FzeQn46zt9RTnT2fDSE\\\\\\\/LD+Thyn2Keb8AuEsSKc3IGmH8STDoSLI0\\\\\\\/pWsm\\\\\\\/0aR40MUj2p8p\\\\\\\/OmGw6o7aPFHNFJHD7+03nWCfFJxoB5veBPhbu+W\\\\\\\/SuKfMWV1p9IWpuiRTEkv1+pUsxCBN+apCVlAKv4FGU4M+xqiZrvSj2PJe\\\\\\\/RERgjbSwwMI8G+EQjqTIoLMYHBit0t4z8mR8GUcH61GTM0CdFC05bvAs2efJ5gQFKtC\\\\\\\/6ksg08Tl\\\\\\\/EzfKbAWaED9D0r0VZgbjyq0aIhoDMWflh1+EJLynNHZj9V1CG92YAHR1suONy2vrtgeFlMzvr5FK64ev+EWF6uB0+2NxuWH1Mj11GAPT3PMDnIoF5RHlfKWfY3ikDLfC9f0tl+KILdcg9v9\\\\\\\/6KrLrjRR3\\\\\\\/51Y1Bk5AYFDNIiLH6X8fMU2OC2wIxXFTmP2mM3+Wv6+Yn+1R38J2lnjgTebClh8xGhjuvD8ruTplPvHsIu9epQWApPc6Y3kcwzGrmPU+rvY9xMDDdpBS+Pyn\\\\\\\/a6q\\\\\\\/twdJcXwIPFQVu2KtLKYQsprcn+C8jz1Mn1Gxd21s7MnYPyTk87pbTJW2eH\\\\\\\/Yi1i9UX2IqZCzXBRM6xcB3rXlPDUcp4TTiqEUxUHDxCUofHedffJsCpcGi31HdVtIjul2aiFEwTUNwDCO\\\\\\\/RtV6V188b1eAQvAAlsBzxuIIxU7pqHp0KWbhdC3wEAuyKZQl1hFVDvnvfA24SaLduDafG3dT7GDiuq7uMStVfPlSjlPNeHSqI26dSY\\\\\\\/zSgxanuDSZM75A5uNk2tGNu4HXqrwDNYdnk4ot3R23gCIqg\\\\\\\/WG1FaSVO3WooC1WSliw+8cRgeejb1c0PVi12FnAvkbmauVjZ+O6\\\\\\\/1M2mcKbQiRuIDy1ESRVa3hNg4RhTbaC\\\\\\\/kJHx2MSqyah+t8paCDUIRO7GFY3jcIdEwRBW\\\\\\\/S8k69On5la\\\\\\\/iZVK+aQcEwJZX2E8SlCXbrxd68WzWQBUmqx1XLBcCfV622D8IoZm7KKnQ+lbI6JkBBX1xGcbdwQYrBAoOX5KK5A5HC8o\\\\\\\/GZOdEtDNmGv\\\\\\\/rohiTS9nGCdfrkpGvaeseAWi6uHTWX5Xy2DLuspb83lfHFatBC7T0zT18c1TvFU031phrGKoybmSRoiTcZk0QjPeJ+63lArAPWVBelNR0lLm674RFS1gDhUZ\\\\\\\/HOG8TuhL94BI3to8l3p3zoogt8vtQT+zeIV5x3P+gNDFGvDhDIl4TYun+2uIQSuTI3Gzi56ZQ79aBYcxF2o9jp2WCLltq56EDahqL1uza4vdfqTSlD6vS5kTGszO2Yl5tlpB+063VOF8gz82\\\\\\\/uSHfguJ9WTebUIwYe9h2PNL\\\\\\\/rMXtGzOIxVQzb2V\\\\\\\/9pBm3PQLefMkhj0OLFerLYxlhKOOLtxluqmhFrFEdRMUMHljr5ujYDBkPpGWAauUXBSZSb90ZzykYnbgstv0aJjbNXV3jWw\\\\\\\/BUTK2B3h3scKZ6NM0qkeOjN\\\\\\\/X8C8M+3uj+LUo+jlFgghI0wEzKE6f9MYmp0VTK+5Fxq3wzd278SBOjfW7N\\\\\\\/XObgbHjvr0fzaakVPWZ1EpJeDXQ3O0F\\\\\\\/muCj5y0S6Pw4NB7h4p0WnnnZYT7kRd2nJI+SyLlJiD9c\\\\\\\/GXr07UxZo+uLGXljjXsVmaAL4a4QmtdsBR3CQh4frcAvmWomxFHNUALDaCp9ZctuOWdUocp6kaO4uVWP9zVHg5XIOgsurtOP6hrKttQWnHbRz4uU+wa3+X9NLwicJIYzFRsrtZHhsnZJabWvdmQG1astlPeJlV6BQwZuwNIAX35Wm0suEqY+hmted2fo5Zdw6\\\\\\\/K\\\\\\\/PCV3D2KRT80Sg12hJX3OicFb+fntRns7KAIWX+jYLgrI1VnHYNxEMbb0QDHWWMkitnS0pxH4ZsHdCBzZqu2gija9kfRK1oD90iiWkeJvSbwCuO9gribNuxIU8IM0M1gNxGSNHL2QopayHLPLzSagXHdLARK8BmJidVSanvuSPc1kDUUTSj3WNQVpb1QqdADpZ6zJ+QVUkOtuJN72E\\\\\\\/EkqF08ktImsBxXI\\\\\\\/ka8OwVVHMEEK5gDZmCa8VA+HwQuEPgFJR62rAIbgOp88cND46ejujSdFpKEXzot9fGYdwZGEE6yEDfD0AuefP2CBocUGF\\\\\\\/dK+Om7oPdxkP1U1eFQWwutYKZSGsgFmORK8jJMPPi\\\\\\\/BtnqFRgx5naZypicJfDpOmbwv4aik3iKhoE6Z3ytzxody9AqUjsP++FQQ1xYDqX\\\\\\\/PhRTC2SfLxd5VEe3Plk3zt4bPrtvFTrsAIsxsu7ZYKQKFekWFpwqs0MKEBy1sosdsEVvekRnOST3B+5o7Yq17LJlH3OFJ8J+a6VfXJGYwtLibSkP+0yWUDCK1F6V2Rtbb4Akjo39WtT53m+\\\\\\\/WhH7zMqqugb7FJ7rGDCBUuq5r6F8+UHmae5xKI7+Fnr+aBc8re3fz+aU8dMXGPUEoBqa4k2gIHN5HuISLdH\\\\\\\/Kg6qLKroEzdOvAJ81BegUUx3N\\\\\\\/vhV+l5ahXL9a3c3bbdZ31xjaRSIBp9yf\\\\\\\/wMNqWDztv9Vn\\\\\\\/o9dhjbUgjJHw2DAW+89k3QdKIxcJ6LHwXaSmkgg6YoxYZY3qmxXcvHkrf7rt1becX+c6T7Ueo8pFd6apYckcvhJAbniYFelheo6Ye6fx5K02axtDDiWer62Vd8IyYvuVRz56acA0+O+w5eHEV6CpT0d3QPNK8zQtbCm1v66d5NIflNKi3Cap3L8VFCc+zpz5t3Nhv+ga2kinH9GMYVW2CRCROem11Mq9\\\\\\\/UYWQ5\\\\\\\/M7lZYjE\\\\\\\/cSwlewggsBPTgg5UrvAbdxYdPv0fnPo9WRxhUUGT9h+nQM\\\\\\\/zeUMA470elSnaGNi4tm8+9cc7dfh3jS7RLvHvzF6KgDZtWf5Z+iFeA3jrrUqSh1BD41dsYwUcsX9w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FnWoQQUOtE5uTlKCqhVfeA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226f29b339968a9fedfdafbd2a4f8023f2718a0dcc785bc915ea3e38b4236475c4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jqKshZ19xx8auRzd66htRzT94WAvwfduLdPlWAE3ZvnKWuAILSmOVgKt6BzXrvIn5Rnigp2ECCUzZTWRC+3ClPUEYGRYSzwhi3XIxBURehUAfogbcU4lplP0lzsGQI6eXxpnfU+tfIp+vqmZlMrpsd2AuwxQEqbPKCE3wU6KPzcw5aa1sjo3IPE6fyrNU5AOFVcU8DPl\\\\\\\/G7D5\\\\\\\/qUjiftRBId8H1vYIf3U7DQUdF1\\\\\\\/XMc2J7c7\\\\\\\/N8XatwwogpyJtsfUGmrIOmoprB4i4hwffRVOUjfwlVMKBqQsJxkWl2aoLggpT1UN4COm+KlOrTrfc0ZqvmokgKVa5ygkkyVqpDcTW2i5e5lHwZBQRTYY92nb9f6pnb+mR8Elwdk0NKm2JMuGi8B9STMQoA2KgAHG+IFJCY2VkI+U+fkuc+CN\\\\\\\/4i2xDSlBhYnFEH8eQOUEHwL7BlonYq6qlbQYVjESBsSVPZXkxYWFlgdnyp32qiJbEDhDQ\\\\\\\/O0WUT2FP3hVIHHJ\\\\\\\/icQOMVDaHkgPJBu1KGuerxgO2XwK9SV8vo4n7kDPvuFwa\\\\\\\/pP84jbyFZkfArLio2OwLcBWUMPAyXjTXrBU\\\\\\\/eMHn8FpsFVZeA4OjCeCCh4KvPTg6lIiyQmpWOB5CnGecBoXIP6hA\\\\\\\/pjiTQEUM9pinzXKuTQqWkDcrAqCM+gXy46ttAb10YLbAufOpryooet6mJaroD\\\\\\\/+G5t4\\\\\\\/WcN4RNwMWGbKjfaQsdKWHT+c43VOONCob5Akla9GwcLnm0wGQLYDpzlGgz2VC0uIPDH\\\\\\\/177qnMQURL+eMLDbKUtMoia8sIyPfviC7dEILtPTgrLP6Dqm7ZcGQA2yZDoObw8ClGjiCGgizMDkRsWASKGiLsL6YQnStGwVo67s0a7mOXzmCeKiArqlqsksGcONAOwsNiT7tbeTboahGY1PWfx7Dc7PtVyoXCYuAtS1IiPPFfYDcfzLSdjxKniZakszju1A\\\\\\\/qhCRsYkiUY7+kXSpB7QGOZEe2SudSWN5Jhk5gYTHMeG1cP8TIDg6k9zgL\\\\\\\/z4cMdt9QnNa361ysuNbXrUgadOcSd7SYk4oMctZR6bTjqNT786DbIaAMI6s1HO7doEuMrK7vyv5ZcU+JC2fJe3MiKpn8yCDEL3zHZSe4RoSA+ILOOACRyvahq75Ry+G0hB\\\\\\\/lTce3FS64ThZsTxC4hI9vTDYrHuU9Xqq1VXhSI3rCdgikCMkXJNSRPlOlowm9rKWAZNDciREzDxdYoK1ehCKPhlYu8JfHZY\\\\\\\/HtUBVDZhUFnIMFJ7FsslOaUQ+BiLe6Jtwe9gxiNn3piKjOu9qC9\\\\\\\/OaKS5hLcW7VJetBqsPfH65rkTg+S1LbTMQQ9SggHyEQtzYFriRYvOnjIX4vst+m8JMYLlWlAUg6T4VD02Qqlxaf+XiR6fFDoJ\\\\\\\/UmAr8NP9BVYnD4bde8\\\\\\\/NLg7eCy7GCzYp+iaHG77M47SlZeizaa0+kiOmXdF7H0xrRZjKIZCAHxJkXnRNXF77j5\\\\\\\/kcSWMn\\\\\\\/1julVf++hUxUlhwMBLIk6hUGuHSHEM\\\\\\\/0td\\\\\\\/XP8DxNWzgaRYb4SdmNcSNhhGMgXqMdxhzt5ZJjzylY330rIu5Vf60EgEFlmM7GUFN4lg0wyi459SOwGvst9wzhw0xzA9s6XsBqIjI8vw0J5uIgjuQatXgMN+nY13aVOhlAsNOk59Pxi3a\\\\\\\/dHYpNkCtcSohINb2o4ISjwLnfFirewDDEJJ+GlHmmEPNvo2o8dPhsoPtzcD8vYGhZBhSN1hcWoR6j9R1cHPsEnk4II7b9kbxeNi8bWwMu1qU+WFj1F9VriBQ4KIzDhH78Hbq\\\\\\\/InCzGODXOVrfwnhdiHcWhWKruFFNcf5xKkb\\\\\\\/DdR8VfLMS6ZOHpixwU\\\\\\\/Psa7ft1bJftGxr9kh5sUN4pBbv14VP1cC3AVZvmSSIn6kBUxnEeaz8vCOQ0eDIM5dC\\\\\\\/xq0m2DLX9d18MtoWN+E9yr+QjhYev8QSIX5YT0Y75hDkLdNZInnbRfdgtIxATcc8sFOrC4dA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022pIf+ksovlXJa2R1Z4P0AbA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002237dda12bcf91551dc7e48abaf214efab90a872c057df4067f4d5a591de2f4bce\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492134 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/391" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yhc686EfWF6g4DacrG_4I4sCCrdvX5sWGTIj6z5i2aU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/391" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KgJW0ecvBr49R1uI-ykU3i047P5E2LMjMlj7Z1bplLU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/2/375" title="A Very Oil Yellow1 Modifier of the Oil Yellow1-N1989 Allele Uncovers a Cryptic Phenotypic Impact of Cis-regulatory Variation in Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/2/403" title="A Transgenic Flock House Virus Replicon Reveals an RNAi Independent Antiviral Mechanism Acting in Drosophila Follicular Somatic Cells" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492134" id="node492134" data-pisa="ggg;9/2/391" data-pisa-master="ggg;g3.118.200932" data-apath="/ggg/9/2/391.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5403-5098" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kebede T. Muleta</span>, <span class="highwire-citation-author" data-delta="1">Gael Pressoir</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3067-3359" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Geoffrey P. Morris</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">391-401;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200932 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492134" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kebede T. Muleta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Kansas State University, Manhattan, Kansas</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kebede%2BT.%2BMuleta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muleta%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKebede%2BT.%2BMuleta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5403-5098" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kebede T. Muleta</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gael Pressoir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Chibas and Faculty of Agriculture and Environmental Sciences, Quisqueya University, Port-au-Prince, Haiti</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gael%2BPressoir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pressoir%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGael%2BPressoir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Geoffrey P. Morris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Kansas State University, Manhattan, Kansas</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Geoffrey%2BP.%2BMorris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morris%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeoffrey%2BP.%2BMorris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3067-3359" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Geoffrey P. Morris</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gpmorris@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">gpmorris@ksu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/2/391" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492134" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492134/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/391.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492134" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492134/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/391.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492134" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492134/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/391.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492134" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492134/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/2/391.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492134" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492134/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Young breeding programs in developing countries, like the Chibas sorghum breeding program in Haiti, face the challenge of increasing genetic gain with limited resources. Implementing genomic selection (GS) could increase genetic gain, but optimization of GS is needed to account for these programs’ unique challenges and advantages. Here, we used simulations to identify conditions under which genomic-assisted recurrent selection (GARS) would be more effective than phenotypic recurrent selection (PRS) in small new breeding programs. We compared genetic gain, cost per unit gain, genetic variance, and prediction accuracy of GARS (two or three cycles per year) <em>vs.</em> PRS (one cycle per year) assuming various breeding population sizes and trait genetic architectures. For oligogenic architecture, the maximum relative genetic gain advantage of GARS over PRS was 12–88%, which was observed only during the first few cycles. For the polygenic architecture, GARS provided maximum relative genetic gain advantage of 26–165%, and was always superior to PRS. Average prediction accuracy declines substantially after several cycles of selection, suggesting the prediction models should be updated regularly. Updating prediction models every year increased the genetic gain by up to 33–39% compared to no-update scenarios. For small populations and oligogenic traits, cost per unit gain was lower in PRS than GARS. However, with larger populations and polygenic traits cost per unit gain was up to 67% lower in GARS than PRS. Collectively, the simulations suggest that GARS could increase the genetic gain in small young breeding programs by accelerating the breeding cycles and enabling evaluation of larger populations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/molecular-breeding" class="hw-term hw-article-keyword hw-article-keyword-molecular-breeding" rel="nofollow">Molecular breeding</a></li><li class="kwd"><a href="/keyword/genetic-gain" class="hw-term hw-article-keyword hw-article-keyword-genetic-gain" rel="nofollow">genetic gain</a></li><li class="kwd"><a href="/keyword/genetic-architecture" class="hw-term hw-article-keyword hw-article-keyword-genetic-architecture" rel="nofollow">genetic architecture</a></li><li class="kwd"><a href="/keyword/genome-wide-prediction" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-prediction" rel="nofollow">genome-wide prediction</a></li><li class="kwd"><a href="/keyword/simulation" class="hw-term hw-article-keyword hw-article-keyword-simulation" rel="nofollow">simulation</a></li><li class="kwd"><a href="/keyword/smallholder-agriculture" class="hw-term hw-article-keyword hw-article-keyword-smallholder-agriculture" rel="nofollow">smallholder agriculture</a></li></ul><p id="p-3">High-yielding and climate-resilient crop varieties are needed to meet increasing demand for food and feed in the developing world (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tester and Langridge 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> 2014</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Varshney <em>et al.</em> 2014</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bhat <em>et al.</em> 2016</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mausch <em>et al.</em> 2017</a>). Young small breeding programs in developing countries face many constraints in genetic improvement of crop varieties, which include limited resources, multipurpose yield targets, and highly heterogeneous production environments. Due to slow breeding cycles, phenotypic selection approaches may not provide adequate genetic gain (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fischer <em>et al.</em> 2014</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bhat <em>et al.</em> 2016</a>). This limitation is especially true for polygenic quantitative traits, which are controlled by large numbers of small effect loci and generally show low heritability (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Varshney <em>et al.</em> 2014</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Bhat <em>et al.</em> 2016</a>).</p><p id="p-4">Genomic selection (GS) has been evaluated in many crop improvement programs as an alternative to phenotypic selection (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Habier <em>et al.</em>, 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Heffner <em>et al.</em>, 2009</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em>, 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jannink <em>et al.</em>, 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lorenz <em>et al.</em>, 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hayes <em>et al.</em>, 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lorenz 2013</a>). In GS approaches, genome-wide prediction models are developed using markers that are in linkage disequilibrium (LD) with quantitative trait loci (QTL); the models of marker effects are used to calculate genomic-estimated breeding values (GEBVs) for genotyped lines and the GEBVs are used for selection (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Meuwissen <em>et al.</em> 2001</a>). Once trained with field phenotypes, GS allows selection without field evaluation, so genetic gain can be increased through increased population size and selection intensity. GS can accelerate population improvement and lower the cost of identifying genotypes with improved breeding or varietal value (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> 2014</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2017</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bassi <em>et al.</em> 2016</a>; <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Bhat <em>et al.</em> 2016</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zhang <em>et al.</em> 2017</a>).</p><p id="p-5">GS is generally applied to facilitate recurrent selection (RS) schemes (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grenier <em>et al.</em> 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hickey <em>et al.</em> 2017</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Müller <em>et al.</em> 2017</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Morais Júnior <em>et al.</em> 2018</a>). RS is a cyclical breeding approach for improvement of quantitative traits in breeding populations. RS produces novel gene combinations and increases the frequency of favorable alleles (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Windhausen <em>et al.</em>, 2012</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Müller <em>et al.</em>, 2017</a>). Phenotypic recurrent selection (PRS) involves three steps conducted in a cyclical manner: (i) development of progeny (S0) by crossing selected parents, (ii) evaluation of progeny (S1 or S2) to select new parents, and (iii) recombination of superior progeny to create the next cycle (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Weyhrich <em>et al.</em> 1998</a>). Accurate evaluation of progeny performance is limited by lack of seed for replicated trials. Genomic-assisted recurrent selection (GARS) would allow prediction of S0 breeding values without evaluation of S1 or S2 families in replicated trials.</p><p id="p-6">There is great interest and potential to apply GS in developing country breeding programs (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Varshney <em>et al.</em> 2014</a>; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Bhat <em>et al.</em> 2016</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em> 2017</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Xu <em>et al.</em> 2017</a>). However, most studies intended to guide GS implementation were not parameterized to reflect small developing country breeding programs (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lorenz 2013</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Müller <em>et al.</em> 2017</a>). GS implementation should take into account these programs’ unique challenges/advantages and selection histories. For instance, the Fisher-Orr model of adaptive walks would predict that young breeding programs have more large effect variants segregating, while established breeding programs (which have fixed large effect variants) should approach an infinitesimal model (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Orr 2009</a>). Simulation studies can been used to investigate genetic gain, prediction accuracy, cost-effectiveness of GS under various scenarios (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Heffner <em>et al.</em> 2009</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Solberg <em>et al.</em> 2009</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jannink 2010</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bastiaansen <em>et al.</em> 2012</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Müller <em>et al.</em> 2017</a>). However, to our knowledge, no simulation study has evaluated GS <em>vs.</em> phenotypic selection considering breeding population size, genetic architecture and costs of small breeding programs in developing countries.</p><p id="p-7">In most developing countries of the semi-arid tropics, including Haiti and countries in sub-Saharan Africa, sorghum is grown primarily under low-input rain-fed conditions with heterogeneous climate and soil conditions (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kumar <em>et al.</em>, 2011</a>). Here, our goal was to evaluate GS strategies for the Chibas sorghum breeding program in Haiti and similar breeding programs. The Chibas sorghum program develops pure line varieties for low-input smallholder agriculture in Haiti using RS with a <em>ms3</em> genetic male sterility (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brocke <em>et al.</em> 2008</a>). The program was launched in 2011 with two rounds of intermating of founders (selections from global tropical germplasm) to establish a base population (Cycle 0). PRS based on evaluation of S1 and S2 families began in 2013. The PRS program proceeded to cycle 3 by 2016 when population improvement was paused to develop sugarcane aphid (<em>Melanaphis sacchari</em>) tolerant varieties. A shift to GARS would allow the prediction of breeding value on S0 progenies, potentially shortening the breeding cycle to one generation. Further, GARS could take advantage of the tropical environment to rapidly cycle two to three times per year. The objective of this simulation study was to compare genetic gain, prediction accuracy, genetic variance, and cost under GARS <em>vs.</em> PRS across various genetic architectures, breeding population sizes, and model updating scenarios. We found that by accelerating the breeding cycle and enabling evaluation of larger populations GARS could increase genetic gain and lower cost per unit gain in developing country breeding programs.</p><div class="section materials-methods" id="sec-1"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-2" class="subsection"><h3>Overview of breeding program simulation</h3><p id="p-8">The Chibas sorghum breeding program’s RS scheme is based on the evaluation of S1 progenies, which involves the selection of best parents followed by randomly intermating the selected parents to generate the next selection cycle. Each cycles of RS followed by multiple generations of selfing to generate inbred line. The inbred families will be evaluated in replicated trials in multiple locations toward variety release. The Chibas sorghum breeding program was simulated using AlphaSimR package (<a href="https://cran.r-project.org/web/packages/AlphaSimR/index.html">https://cran.r-project.org/web/packages/AlphaSimR/index.html</a>) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Faux <em>et al.</em> 2016</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). All results are based on 100 simulated breeding programs, each from an independently simulated founder population. Haplotype sequences of founder populations were simulated with the coalescent simulation program MaCS (Chen <em>et al.</em>, 2009) implemented in AlphaSimR. The population parameters used to simulate the founders include effective population size (Ne) of 20 (the approximate number of founders of the Chibas program), number of base pairs = 2x10<sup>8</sup>, and mutation rate = 2x10<sup>−8</sup> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Genomes were defined with 10 chromosomes, as in sorghum (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Paterson <em>et al.</em> 2009</a>), and 1,000 segregating sites per chromosome. Segregating sites in the founder population were assigned randomly to become either markers (n = 9,000) or QTL (n = 1,000). The actual number of QTL with effect size greater than zero varied according the genetic architecture of the trait simulated.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/391/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Breeding program simulation. (A) Schematic representation of the simulated breeding programs. In the population establishment phase, the base population was created by randomly intercrossing the founder population for several generations, which was then phenotyped and genotyped to be used as a training population. For the breeding cycles, the training population was used as a cycle 0 generation from where best parents were selected to initiate PRS and GARS. GARS was conducted three cycles per year (GARS3, shown) or two cycles per year (GARS2, not shown), compared to a single cycle of PRS per year. (B) The density plot of the distribution of allelic substitution effects of the quantitative trait loci (QTL) simulated in two trait genetic architectures (oligogenic and polygenic architectures)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-295089270" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Breeding program simulation. (A) Schematic representation of the simulated breeding programs. In the population establishment phase, the base population was created by randomly intercrossing the founder population for several generations, which was then phenotyped and genotyped to be used as a training population. For the breeding cycles, the training population was used as a cycle 0 generation from where best parents were selected to initiate PRS and GARS. GARS was conducted three cycles per year (GARS3, shown) or two cycles per year (GARS2, not shown), compared to a single cycle of PRS per year. (B) The density plot of the distribution of allelic substitution effects of the quantitative trait loci (QTL) simulated in two trait genetic architectures (oligogenic and polygenic architectures).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/391/F1.medium.gif" width="380" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/2/391/F1.medium.gif" width="380" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/391/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/391/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492209" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Breeding program simulation. (A) Schematic representation of the simulated breeding programs. In the population establishment phase, the base population was created by randomly intercrossing the founder population for several generations, which was then phenotyped and genotyped to be used as a training population. For the breeding cycles, the training population was used as a cycle 0 generation from where best parents were selected to initiate PRS and GARS. GARS was conducted three cycles per year (GARS3, shown) or two cycles per year (GARS2, not shown), compared to a single cycle of PRS per year. (B) The density plot of the distribution of allelic substitution effects of the quantitative trait loci (QTL) simulated in two trait genetic architectures (oligogenic and polygenic architectures).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Defining trait genetic architectures</h3><p id="p-10">Traits were simulated with (i) oligogenic or (ii) polygenic architecture by varying the number and effect size of QTL (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). The oligogenic and polygenic traits were assumed to have high and low heritability, respectively, to represent contrasting extremes of genetic architecture. Oligogenic architecture was simulated by randomly sampling 50 QTL (five per chromosome) with effects sizes drawn from a gamma distribution with scale and shape parameter of 0.5 and 1.8, respectively. Components of phenotypic variance (additive genetic variance, environmental variance, genotype by environment interaction [GxE] variance, and residual variance) were chosen to make heritability <em>h</em><sup>2</sup> = 0.7. For the oligogenic trait, we are referring to a quantitative trait that is controlled by relatively small number of genes, in which most of the trait variation is attributed to the top few genes. In this case, only five loci are responsible for 45% of the trait variation.</p><p id="p-11">Polygenic architecture was simulated by sampling 500 QTL (50 per chromosome) with effect sizes drawn from a gamma distribution with scale and shape parameter of 1.0 and 0.5, respectively. Variance components were chosen to make heritability <em>h</em><sup>2</sup> = 0.3. Intra- and inter-locus interactions (dominance and epistasis effects) were not modeled. For each of the two genetic architectures, various population sizes and selection schemes were compared for genetic gain, genetic variance, and prediction accuracies. True breeding value for each individual was calculated by summing effects across all QTL. Genetic gain (assuming true breeding value of zero at <em>t</em> = 0) was defined and plotted as the true breeding value expressed in standard deviation units of the phenotypic values of cycle 0 (C0) generation.</p></div><div id="sec-4" class="subsection"><h3>Defining base/training populations</h3><p id="p-12">For each simulation, a base population (C0 generation) of 400 inbred lines was simulated by randomly mating the founder population, followed by five generations of selfing to create inbred lines. Phenotypic values for each inbred line in the base population was simulated by adding normally-distributed random deviates to each line’s true breeding value. The base population was used as a source of parental lines that were used to create the initial recurrent selection population. The base population is also used as the training population for estimation of markers effects for the genome-wide prediction model.</p></div><div id="sec-5" class="subsection"><h3>Conducting phenotypic recurrent selection</h3><p id="p-13">PRS was performed by selecting the top 5, 20, 100, or 200 parental lines from the base population based on their phenotypic values. Selected parents were randomly intermated to generate S0 individuals. The S0 individuals were then selfed to generate S1 families. Following the Chibas breeding program’s current PRS method, the S1 families were phenotyped and S1 families with the top phenotypic values were randomly intermated to generate the next cycle. Intermating of 5, 20, 100, or 200 parents generated 50, 200, 1000, or 2000 S0 individuals, respectively (designated par<sub>5</sub>_pop<sub>50</sub>, par<sub>20</sub>_pop<sub>200</sub>, par<sub>100</sub>_pop<sub>1000</sub> and par<sub>200</sub>_pop<sub>2000</sub>, respectively) (File S1A). In all of these scenarios, the top 10% of S1 families were retained each cycle. Since developing countries breeding programs generally have limited field phenotyping capacity, scenarios were also simulated where the number of phenotyped S1 families was capped at 500. In these scenarios (designated as par<sub>100</sub>_pop<sub>500</sub>, par<sub>200</sub>_pop<sub>500</sub>), populations were generated by randomly intermating 100 or 200 selected parents to generate 500 S0 individuals, which meant the top 20% or 40% of S1 families were retained, respectively. Assuming that the first selfing generation (S0→S1) was performed during an off-season nursery, a single cycle of PRS was performed each year, for a total of ten cycles of PRS in ten years. For each cycle of PRS, genetic gain, genetic variance, and prediction accuracy were calculated. Prediction accuracy in PRS was calculated as the Pearson correlation between the phenotypic value and true breeding value of the S1 families.</p></div><div id="sec-6" class="subsection"><h3>Conducting genomic-assisted recurrent selection</h3><p id="p-14">Similarly, GARS was initiated by selecting the top 5, 20, 100, or 200 parental lines from the base population based on their phenotypic values and randomly intermating to generate 50, 200, 1000, or 2000 S0 populations, respectively. Ridge regression best linear unbiased prediction (RR-BLUP) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Endelman 2011</a>) was used to estimate genome-wide marker effects (using the phenotypic and genotypic data from the training population) and calculate GEBVs of the S0 individuals according to the following model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/391/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/391/embed/mml-math-1.gif"/></noscript></span></span>where <em>y<sub>i</sub></em> is the phenotype for individual <em>i</em>; <em>μ</em> is the overall mean; <em>X<sub>ik</sub></em> is the genotype of individual <em>i</em> at marker <em>k</em>; <em>β<sub>k</sub></em> is the additive effect of marker <em>k</em>; <em>p</em> is the number of markers; and <em>e<sub>i</sub></em> is the residual effect. GEBVs were estimated using the following formula: <em>y<sub>v</sub></em> = <strong>M</strong><em>β</em>, where <em>y<sub>v</sub></em> is an <em>n</em>×1 vector of GEBVs for the accessions in the training population, <strong>M</strong> is an <em>n</em>×<em>m</em> matrix of genotype indicators for the training population, <em>β</em> is an <em>m</em>×1 vector of marker effects, <em>n</em> is the number of individuals, and <em>m</em> is the number of markers.</p><p id="p-15">The GEBVs were used to select the top S0 individuals of C1 population. The selected S0 individuals were randomly intermated to generate the next cycle (C2) recurrent selection population, and so on for 30 cycles. To assess the effect of number of breeding cycle per year, two or three cycles of GARS per year (designated GARS2 and GARS3, respectively) were compared to PRS. Note, the results for GARS2 and GARS3 represent the same simulation runs, plotted differently to represent the given number of GARS cycles per year. Therefore, given a 10-year time horizon, there were 20 cycles of GARS2 and 30 cycles of GARS3. To align with the 10-year selection cycles of the PRS, only the first 28 and 19 cycles were plotted. During each cycle, the number of parents and populations generated by random intermating of the parents remained constant. Genetic gain, genetic variance, and prediction accuracy were calculated for the different selection schemes, population sizes, and trait genetic architectures. Prediction accuracies in GARS was calculated using a Pearson correlation between the GEBVs and true breeding values of S1 families.</p></div><div id="sec-7" class="subsection"><h3>Updating training populations and prediction models</h3><p id="p-16">We examined the impact of updating the training populations and prediction models on the genetic gain, genetic variance, and prediction accuracy. Given that cropping-season phenotyping can only be conducted once per year, updating of the training population and prediction model was performed every two and three cycles for GARS2 and GARS3, respectively. The updated training population for the a given cycle (<em>N</em><sub>ind</sub> = 400) consisted of (i) the 200 top lines from the previous training population based on phenotype values and (ii) the 200 top lines from the previous cycle based on GEBVs. Updating of the training population was performed based on the population size of par<sub>100</sub>_pop<sub>1000</sub>, which represents an optimum breeding population size at which genetic gain has plateaued. Changes in genetic gain, genetic variance, and prediction accuracy due to updating were compared to no-update GARS scenarios and PRS.</p></div><div id="sec-8" class="subsection"><h3>Comparing costs of GARS <em>vs.</em> PRS</h3><p id="p-17">We compared the efficiency of the two selection methods in terms of cost per unit gain. One cycle of PRS in the Chibas sorghum breeding program involves the intercrossing of best parents to generate S0 individuals; planting and selfing of the S0 individuals to generate S1 families; and evaluation of the S1 families in replicated trials. The following costs (in US dollars per line calculated from 2017/18 budget estimates from Chibas sorghum breeding program) were assumed: $38 in the crossing phase, $14 in managing the S0 phase, and $33 replication<sup>-1</sup> for evaluating the S1 families (File S1B). Costs in PRS were calculated for the following activities; planting and field management, pollination, perform replicated yield trial for grain yield, disease resistance/tolerance, stay-green, stem biomass, stem sugar content and other traits. GARS does not require phenotypic evaluation of the S1 population, but estimates breeding values of the S0 individuals. The costs involved in the different activities GARS includes genotyping and phenotyping of the training population, crossing of the parents and genotyping of the S0 individuals. Different genotyping cost, which include US $5, $15, $25, and $35 were used to estimate cost per gain for GARS and compared to that of PRS. Cost per unit genetic gain for the different scenarios were calculated by dividing the cumulative cost per cycle with the corresponding genetic gain.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-18">All relevant data are within the paper and its supplemental files. Figure S1 shows the effects of updating the training population under an polygenic architecture. Figure S2 and S3 describe cumulative cost per unit gain in GARS2 and GARS3, respectively, compared to PRS. Figure S4 shows the costs per unit gain for updating the training population. Figure S5 shows the shift in the trait genetic architecture due to selection. File S1 summarizes the different populations sizes and cost estimates of activities in GARS and PRS. File S2 presents the <em>t</em>-tests for comparing the genetic gain between GARS and PRS across the different trait genetic architectures and population size. File S3 contains the script used to simulate the breeding program, while File S4 contains the script used to run the cost analysis and the output from the breeding program simulation (genetic gains) that was used to run the cost analysis. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7396781">https://doi.org/10.25387/g3.7396781</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">RESULTS</h2><div id="sec-11" class="subsection"><h3>Genetic gain differences among scenarios</h3><p id="p-19">We first compared the genetic gain across population sizes, genetic architectures, and selection schemes. The genetic gain per unit time resulting from GARS and PRS differed significantly and depended on population size and genetic architecture (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>; <em>t</em>-tests in File S2). For the oligogenic architecture, GARS3 yielded significantly higher (<em>P</em> &lt; 0.01) short-term genetic gain (year &lt; 5) than PRS for all population size (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A-D</a>). Across the years, the maximum relative short-term genetic gain from GARS3 compared to PRS ranged from 12–88% over different population sizes. By contrast, GARS2 yielded short-term genetic gain comparable to that of PRS. Long-term genetic gain (year ≥ 5–6) for the oligogenic architecture was significantly higher (<em>P</em> &lt; 0.01) for PRS compared to either GARS2 or GARS3. For the polygenic architecture, GARS3 resulted in higher genetic gain (<em>P</em> &lt; 0.01) over PRS for all population sizes across all selection cycles, except the first year (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2E-H</a>). Across years, maximum relative advantage of GARS3 over PRS was 26–165% depending on population size. Similarly, GARS2 provided higher genetic gain (<em>P</em> &lt; 0.01) than PRS across all population sizes for the polygenic architecture, with maximum relative advantage of 12–99%.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/391/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic gains from genomic assisted recurrent selection (GARS) vs. phenotypic recurrent selection (PRS). The two selection schemes were compared across five population sizes and two types of traits genetic architecture (oligogenic and high heritability vs. polygenic and low heritability). The five population sizes were designated as par5_pop50, par20_pop200, par100_pop1000, par200_pop2000, which represent selection of 5, 20, 100, or 200 best parents and randomly intermated to create 50, 200, 1000, or 2000 individuals, respectively. par100_pop500 and par200_pop500 represent the selection of 100 and 200 best parents and randomly intermated to create 500 individuals, respectively. Changes in genetic gain (Y-axis) was ploted against selection cycles in years (X-axis). The plotted values are means (+/&#x2212; 95% confidence interval) from 100 simulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-295089270" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic gains from genomic assisted recurrent selection (GARS) &lt;em&gt;vs.&lt;/em&gt; phenotypic recurrent selection (PRS). The two selection schemes were compared across five population sizes and two types of traits genetic architecture (oligogenic and high heritability &lt;em&gt;vs.&lt;/em&gt; polygenic and low heritability). The five population sizes were designated as par&lt;sub&gt;5&lt;/sub&gt;_pop&lt;sub&gt;50&lt;/sub&gt;, par&lt;sub&gt;20&lt;/sub&gt;_pop&lt;sub&gt;200&lt;/sub&gt;, par&lt;sub&gt;100&lt;/sub&gt;_pop&lt;sub&gt;1000&lt;/sub&gt;, par&lt;sub&gt;200&lt;/sub&gt;_pop&lt;sub&gt;2000&lt;/sub&gt;, which represent selection of 5, 20, 100, or 200 best parents and randomly intermated to create 50, 200, 1000, or 2000 individuals, respectively. par&lt;sub&gt;100&lt;/sub&gt;_pop&lt;sub&gt;500&lt;/sub&gt; and par&lt;sub&gt;200&lt;/sub&gt;_pop&lt;sub&gt;500&lt;/sub&gt; represent the selection of 100 and 200 best parents and randomly intermated to create 500 individuals, respectively. Changes in genetic gain (Y-axis) was ploted against selection cycles in years (X-axis). The plotted values are means (+/&#x2212; 95% confidence interval) from 100 simulations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/391/F2.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/2/391/F2.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/391/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/391/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492169" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Genetic gains from genomic assisted recurrent selection (GARS) <em>vs.</em> phenotypic recurrent selection (PRS). The two selection schemes were compared across five population sizes and two types of traits genetic architecture (oligogenic and high heritability <em>vs.</em> polygenic and low heritability). The five population sizes were designated as par<sub>5</sub>_pop<sub>50</sub>, par<sub>20</sub>_pop<sub>200</sub>, par<sub>100</sub>_pop<sub>1000</sub>, par<sub>200</sub>_pop<sub>2000</sub>, which represent selection of 5, 20, 100, or 200 best parents and randomly intermated to create 50, 200, 1000, or 2000 individuals, respectively. par<sub>100</sub>_pop<sub>500</sub> and par<sub>200</sub>_pop<sub>500</sub> represent the selection of 100 and 200 best parents and randomly intermated to create 500 individuals, respectively. Changes in genetic gain (Y-axis) was ploted against selection cycles in years (X-axis). The plotted values are means (+/− 95% confidence interval) from 100 simulations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">For both genetic architectures, increasing population size resulted in increased genetic gain with GARS and PRS. Given GARS3 and oligogenic architecture, final genetic gain (year 10) increased by 22% and 3% when population size was increased (from par<sub>5</sub>_pop<sub>50</sub> to par<sub>20</sub>_pop<sub>200</sub> and from par<sub>20</sub>_pop<sub>200</sub> to par<sub>100</sub>_pop<sub>1000</sub>, respectively; <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A-C</a>). Given PRS and oligogenic architecture, final genetic gain increased by 25% and 2%, respectively, for the same population size increases. Further increase in population size (to par<sub>200</sub>_pop<sub>2000</sub>) did not result in significant increase in genetic gain (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a> <em>vs.</em> 2D). For polygenic architecture, final genetic gain from GARS3 increased by 36% and 7%, for the same population size increases. (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2E-G</a>). With the exception of the smallest population size, GARS3 genetic gain (under polygenic architecture) continued to increase through the final year.</p></div><div id="sec-12" class="subsection"><h3>Genetic variance differences among scenarios</h3><p id="p-22">Total genetic variance was reduced over generations with both GARS and PRS (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The decline in genetic variance differed highly significantly based on population sizes and genetic architecture (<em>P</em> &lt; 0.001). For the oligogenic architecture, genetic variance during early selection years (year &lt; 5) were higher for PRS compared to GARS (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A-D</a>). In the later years (year ≥ 5), differences in genetic variance between the selection schemes became (i) non-significant (<em>P</em> &gt; 0.05) at the smallest population size or (ii) significantly higher (<em>P</em> &lt; 0.01) for GARS than PRS at the largest population size, notably for GARS2. Under the assumptions of limited phenotyping capacity in PRS (maximum population size capped at 500), genetic variance was higher than GARS at the maximum population size (<em>P</em> &lt; 0.01; <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3D</a>). Genetic variance declined to zero in PRS for all population sizes (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A-D</a>) and at the smallest population size with GARS2 and GARS3 (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/391/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Changes in genetic variance during the recurrent selection process. Genomic assisted recurrent selection (GARS) was compared to the phenotypic recurrent selection (PRS) across five population sizes and two types of traits genetic architecture (oligogenic and high heritability vs. polygenic and low heritability). The five population sizes were designated as par5_pop50, par20_pop200, par100_pop1000, par200_pop2000, which represent selection of 5, 20, 100, or 200 best parents and randomly intermated to create 50, 200, 1000, or 2000 individuals, respectively. par100_pop500 and par200_pop500 represent the selection of 100 and 200 best parents and randomly intermated to create 500 individuals, respectively. Changes in total genetic variance (Y-axis) was ploted against selection years (X-axis). GARS2 and GARS3, with two and three cycles of genomic selection per year, respectively, were compared to one cycle PRS per year. Plotted values are means (+/&#x2212; 95% confidence interval) from 100 simulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-295089270" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Changes in genetic variance during the recurrent selection process. Genomic assisted recurrent selection (GARS) was compared to the phenotypic recurrent selection (PRS) across five population sizes and two types of traits genetic architecture (oligogenic and high heritability &lt;em&gt;vs.&lt;/em&gt; polygenic and low heritability). The five population sizes were designated as par&lt;sub&gt;5&lt;/sub&gt;_pop&lt;sub&gt;50&lt;/sub&gt;, par&lt;sub&gt;20&lt;/sub&gt;_pop&lt;sub&gt;200&lt;/sub&gt;, par&lt;sub&gt;100&lt;/sub&gt;_pop&lt;sub&gt;1000&lt;/sub&gt;, par&lt;sub&gt;200&lt;/sub&gt;_pop&lt;sub&gt;2000&lt;/sub&gt;, which represent selection of 5, 20, 100, or 200 best parents and randomly intermated to create 50, 200, 1000, or 2000 individuals, respectively. par&lt;sub&gt;100&lt;/sub&gt;_pop&lt;sub&gt;500&lt;/sub&gt; and par&lt;sub&gt;200&lt;/sub&gt;_pop&lt;sub&gt;500&lt;/sub&gt; represent the selection of 100 and 200 best parents and randomly intermated to create 500 individuals, respectively. Changes in total genetic variance (Y-axis) was ploted against selection years (X-axis). GARS2 and GARS3, with two and three cycles of genomic selection per year, respectively, were compared to one cycle PRS per year. Plotted values are means (+/&#x2212; 95% confidence interval) from 100 simulations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/391/F3.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/2/391/F3.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/391/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/391/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492111" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Changes in genetic variance during the recurrent selection process. Genomic assisted recurrent selection (GARS) was compared to the phenotypic recurrent selection (PRS) across five population sizes and two types of traits genetic architecture (oligogenic and high heritability <em>vs.</em> polygenic and low heritability). The five population sizes were designated as par<sub>5</sub>_pop<sub>50</sub>, par<sub>20</sub>_pop<sub>200</sub>, par<sub>100</sub>_pop<sub>1000</sub>, par<sub>200</sub>_pop<sub>2000</sub>, which represent selection of 5, 20, 100, or 200 best parents and randomly intermated to create 50, 200, 1000, or 2000 individuals, respectively. par<sub>100</sub>_pop<sub>500</sub> and par<sub>200</sub>_pop<sub>500</sub> represent the selection of 100 and 200 best parents and randomly intermated to create 500 individuals, respectively. Changes in total genetic variance (Y-axis) was ploted against selection years (X-axis). GARS2 and GARS3, with two and three cycles of genomic selection per year, respectively, were compared to one cycle PRS per year. Plotted values are means (+/− 95% confidence interval) from 100 simulations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The decline in genetic variance over years was minimal for the polygenic architecture, particularly for the larger population sizes (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3E-H</a>). Genetic variance was higher for PRS than GARS2 and GARS3 across all population sizes and selection years (<em>P</em> &lt; 0.01). This difference was greater during the early years, but declined in later years. Similarly, the difference in genetic variance between GARS2 and GARS3 was minimal during the early years, but GARS3 lost more variance than GARS2 in later years (<em>P</em> &lt; 0.01). For the polygenic architecture, genetic variance declined to zero only for GARS3 at the smallest population size (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3E</a>).</p></div><div id="sec-13" class="subsection"><h3>Prediction accuracies differences among scenarios</h3><p id="p-25">Prediction accuracy of GARS declined over generations, at rates that varied by trait genetic architecture and population size (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). In early years, average GARS prediction accuracy was higher for the oligogenic architecture (0.64–0.76) compared to that of polygenic architecture (0.62–0.68) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A-D</a>). However, the decline in prediction accuracy given oligogenic architecture was more rapid than that given polygenic architecture, notably after the second year of RS. Accordingly, prediction accuracies were significantly higher (<em>P</em> &lt; 0.01) given polygenic architecture <em>vs.</em> oligogenic architecture in the later years (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B-D</a>). PRS prediction accuracy for the oligogenic architecture was initially high but declined over years, most notably for the smallest population size (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A-D</a>). For the polygenic architecture, PRS prediction accuracy showed strong decline only for the smallest population size (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4E-H</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/391/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of genetic architecture on prediction accuracy. Average prediction accuracy of genomic estimated breeding values (GEBV) under genomic-assisted recurrent selection and phenotypic estimated values (PEV) under phenotypic recurrent selection across years of selection for different breeding population sizes and trait genetic architectures, for an oligogenic and high heritability trait or polygenic and low heritability trait. The Y-axis represent accuracy of predicting GEBVs (estimated as the correlation between the GEBVs and true breeding values) and PEV (estimated as a correlation between the phenotypic-estimated values and true breeding values), while the X-axis represent selection years. The plotted values are means (+/&#x2212; 95% confidence interval) from 100 simulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-295089270" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effects of genetic architecture on prediction accuracy. Average prediction accuracy of genomic estimated breeding values (GEBV) under genomic-assisted recurrent selection and phenotypic estimated values (PEV) under phenotypic recurrent selection across years of selection for different breeding population sizes and trait genetic architectures, for an oligogenic and high heritability trait or polygenic and low heritability trait. The Y-axis represent accuracy of predicting GEBVs (estimated as the correlation between the GEBVs and true breeding values) and PEV (estimated as a correlation between the phenotypic-estimated values and true breeding values), while the X-axis represent selection years. The plotted values are means (+/&#x2212; 95% confidence interval) from 100 simulations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/391/F4.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/2/391/F4.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/391/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/391/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492251" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Effects of genetic architecture on prediction accuracy. Average prediction accuracy of genomic estimated breeding values (GEBV) under genomic-assisted recurrent selection and phenotypic estimated values (PEV) under phenotypic recurrent selection across years of selection for different breeding population sizes and trait genetic architectures, for an oligogenic and high heritability trait or polygenic and low heritability trait. The Y-axis represent accuracy of predicting GEBVs (estimated as the correlation between the GEBVs and true breeding values) and PEV (estimated as a correlation between the phenotypic-estimated values and true breeding values), while the X-axis represent selection years. The plotted values are means (+/− 95% confidence interval) from 100 simulations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Impact of updating the training population and prediction model</h3><p id="p-27">Given the decline of prediction accuracy with GARS, we considered the effects of updating the training population and prediction models to account for new LD generated over breeding cycles. We simulated scenarios with updates each year (every two or three cycles for GARS2 and GARS3, respectively) and compared these scenarios to no-update GARS and PRS (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and Figure S1). Updating strategies provided large increases in genetic gain in later years of GARS under both oligogenic (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) and polygenic architectures (Figure S1). For the oligogenic architecture, final genetic gain increased by 39% and 7% for updated GARS2, compared to no-update GARS2 and PRS, respectively (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). For updated GARS3, genetic gain increased by 33% and 10% over no-update GARS3 and PRS, respectively (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/391/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of updating the training population on genetic gain, genetic variance and prediction accuracy. Genetic gain (panels A and B), genetic variance (panels C and D), and prediction accuracy (panels E and F) for GARS on oligogenic and high heritability trait when the training population was updated and the model retrained every two cycle (GARS2) (left panels) or three cycles (GARS3) (right panels) and compared to no update scenario. Updating was performed by mixing 200 best performing individuals from the recurrent selection cycle with 200 best performing individuals from the training population. The recurrent selection was performed by selecting 100 best performing individuals and intermated to create 1000 recurrent selection population, which was carried out for 30 cycles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-295089270" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effects of updating the training population on genetic gain, genetic variance and prediction accuracy. Genetic gain (panels A and B), genetic variance (panels C and D), and prediction accuracy (panels E and F) for GARS on oligogenic and high heritability trait when the training population was updated and the model retrained every two cycle (GARS2) (left panels) or three cycles (GARS3) (right panels) and compared to no update scenario. Updating was performed by mixing 200 best performing individuals from the recurrent selection cycle with 200 best performing individuals from the training population. The recurrent selection was performed by selecting 100 best performing individuals and intermated to create 1000 recurrent selection population, which was carried out for 30 cycles.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/391/F5.medium.gif" width="356" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/2/391/F5.medium.gif" width="356" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/391/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/391/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492191" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">Effects of updating the training population on genetic gain, genetic variance and prediction accuracy. Genetic gain (panels A and B), genetic variance (panels C and D), and prediction accuracy (panels E and F) for GARS on oligogenic and high heritability trait when the training population was updated and the model retrained every two cycle (GARS2) (left panels) or three cycles (GARS3) (right panels) and compared to no update scenario. Updating was performed by mixing 200 best performing individuals from the recurrent selection cycle with 200 best performing individuals from the training population. The recurrent selection was performed by selecting 100 best performing individuals and intermated to create 1000 recurrent selection population, which was carried out for 30 cycles.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Given polygenic architecture, final genetic gain (year 10) was increased by 29% and 13% for updated GARS2 compared to no-update GARS2 and PRS, respectively (Figure S1A). For updated GARS3, the increase in final genetic gain was 30% and 40% compared to no-update GARS3 and PRS, respectively (Figure S1B). Updating resulted in a greater decline of genetic variance compared to no-update GARS (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C-D</a>). Prediction accuracy remained largely above 0.2 with updated GARS2 and GARS3 (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5E-F</a>).</p></div><div id="sec-15" class="subsection"><h3>Cost per unit gain differences among scenarios</h3><p id="p-30">Cost per unit genetic gain was compared for GARS <em>vs.</em> PRS across population sizes, genetic architectures, and updating scenarios. For small population size, cost per unit gain was lower than GARS regardless of trait genetic architecture and genotyping costs considered. Cost per unit gain was lower by up to 67% in GARS than PRS when larger population size was evaluated, notably at the genotyping cost of $15 or under (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>; Figure S2; Figure S3). When updating the training population, the additional cost of evaluating the training population was offset by the extra genetic gain achieved by model retraining (Figure S4B, C, D, G, and H). The exception was at the early selection years of updating GARS2 in which cost per unit gain in no-update GARS2 was slightly lower than that of updated GARS2. Given genotyping cost of $15 or lower, cost per unit gain in updated GARS2 was lower than PRS Hence, cost per unit gain in updated GARS2 and GARS3 was either comparable to or lower than no-update scenarios, except at early years of the selection cycles.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/391/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cumulative cost (in thousands of US dollars) per unit gain in GARS3 compared to PRS: Four different genotyping costs (turquoise color) were compared to that of PRS (red color) for oligogenic architecture and polygenic architecture for population sizes of par5_pop50 (panels A and B), and par20_pop200 (panels C and D). The four realistic genotyping costs include $5, $15, $25, and $35 per sample; were compared to phenotyping costs for PRS, which was calculated based the current Chibas expenses for managing field experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-295089270" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cumulative cost (in thousands of US dollars) per unit gain in GARS3 compared to PRS: Four different genotyping costs (turquoise color) were compared to that of PRS (red color) for oligogenic architecture and polygenic architecture for population sizes of par&lt;sub&gt;5&lt;/sub&gt;_pop&lt;sub&gt;50&lt;/sub&gt; (panels A and B), and par&lt;sub&gt;20&lt;/sub&gt;_pop&lt;sub&gt;200&lt;/sub&gt; (panels C and D). The four realistic genotyping costs include $5, $15, $25, and $35 per sample; were compared to phenotyping costs for PRS, which was calculated based the current Chibas expenses for managing field experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/391/F6.medium.gif" width="397" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/2/391/F6.medium.gif" width="397" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/391/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/391/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492092" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-31" class="first-child">Cumulative cost (in thousands of US dollars) per unit gain in GARS3 compared to PRS: Four different genotyping costs (turquoise color) were compared to that of PRS (red color) for oligogenic architecture and polygenic architecture for population sizes of par<sub>5</sub>_pop<sub>50</sub> (panels A and B), and par<sub>20</sub>_pop<sub>200</sub> (panels C and D). The four realistic genotyping costs include $5, $15, $25, and $35 per sample; were compared to phenotyping costs for PRS, which was calculated based the current Chibas expenses for managing field experiments.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">DISCUSSION</h2><div id="sec-17" class="subsection"><h3>Maximization of genetic gain</h3><p id="p-32">Increasing genetic gain is a top consideration for breeding program design in the Chibas sorghum program and many similar programs (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Xu <em>et al.</em> 2017</a>). Simulation and empirical studies have reported that GS increases genetic gain, but have not been specifically parameterized to reflect small breeding programs in developing countries (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Heffner <em>et al.</em> 2010</a>). Our simulation of genetic gain in GARS <em>vs.</em> PRS was designed to represent the genetic architecture and breeding schemes of the Chibas sorghum program in Haiti, but may be relevant to programs with similar genetic properties. The simulations show that GARS should, in most cases, increase genetic gain compared to PRS when at least two cycles of GARS are performed. Given oligogenic architecture and no updates of GARS models, however, the long-term genetic gain from PRS is expected to be greater than GARS. This result is likely due to new variation released by recombination that can be selected with PRS but not with no-update GARS. When the GARS prediction model is updated regularly, GARS outperforms PRS under either genetic architecture. Together, these findings highlight the likely advantage of GARS and the value of GARS model updating.</p><p id="p-33">Larger population sizes provide the opportunity to increase selection intensity while maintaining genetic variance (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guzman 1998</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Jannink 2010</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lorenz 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gorjanc <em>et al.</em> 2017</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Xu <em>et al.</em> 2017</a>). In our simulation, genetic gain was significantly greater in larger breeding populations (<em>P</em> &lt; 0.01, as described in previous simulations (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Lorenz 2013</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Slater <em>et al.</em> 2016</a>; <a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Xu <em>et al.</em> 2017</a>). In practice however, population size is limited by capacity to manage field experiments. When a realistic limit on phenotyping capacity for Chibas was imposed, the advantage of GARS over PRS increased by more than fourfold. The effect of increased populations size may also be dictated by the genetic architecture of the trait. Increasing population size increased long-term gain given polygenic architecture, but provided little benefit given oligogenic architecture.</p><p id="p-34">The advantage of GARS over PRS may vary according to the selection history of the breeding program and concomitant changes in genetic architecture (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Roff and Mousseau 1999</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kumar <em>et al.</em> 2008</a>). Breeding programs that have had few selection cycles (as in the case of Chibas sorghum breeding program) are expected to segregate large effect variants, while breeding programs that have had many selection cycles should segregate small effect loci (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Orr 2009</a>). Given oligogenic architecture in a young breeding program, GARS may provide only a short-term increase in genetic gain. However, as large-effect variants are fixed genetic architecture may become more polygenic (Figure S5), so GARS may provide both short- and long-term increase in genetic gains. That is, benefits of implementing GARS may increase as the selection cycles advance the program toward the infinitesimal model.</p></div><div id="sec-18" class="subsection"><h3>Maximization of cost effectiveness</h3><p id="p-35">Cost-efficiency of evaluating and maintaining the breeding population is also a critical aspect of implementing genomic-assisted recurrent selection (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wong and Bernardo 2008</a>; <a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Varshney <em>et al.</em> 2014</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bassi <em>et al.</em> 2016</a>; <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Bhat <em>et al.</em> 2016</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em> 2017</a>). We compared cost per unit gain in GARS <em>vs.</em> PRS across various population size and trait genetic architecture. The relative cost effectiveness of GARS <em>vs.</em> PRS was sensitive to many aspects of the breeding scheme. For instance, GARS was more cost efficient than PRS given current cost of field phenotyping for Chibas, moderate breeding population size (par<sub>20</sub>_pop<sub>200</sub> or larger), and genotyping cost &lt;$15 per sample. With higher genotyping costs, the greater cost efficiency of GARS will be due to larger breeding population (par<sub>100</sub>_pop<sub>1000</sub> and par<sub>200</sub>_pop<sub>2000</sub>). The result implied that it would be economically feasible to implement GARS to Chibas breeding program even with relatively expensive genotyping costs (US $15/sample) and moderate breeding population size. When genotyping costs are driven down to $5/sample, GARS is expected to be cost competitive even when traits are oligogenic and highly heritable.</p><p id="p-36">Despite recent genotyping advances, existing platforms may not be appropriate for GARS. For instance, genotyping-by-sequencing (GBS) has been used to rapidly generate 100,000-500,000 SNP markers for diverse sorghum germplasm (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bekele <em>et al.</em> 2013</a>), but the cost (∼$20–40 USD), turn-around time (∼6–8 weeks), and bioinformatics requirements may be prohibitive for GARS. One option would be to use GBS and resequencing data to develop an out-sourced SNP assay with 2,000–10,000 informative markers. If a turn-around of &lt;4 weeks and cost of $5–10/sample was achieved, our simulations suggest that GARS would be effective in developing country breeding programs. Given the volume of data production and turn-around times, it would be critical to provide user-friendly bioinformatics and decision-support tools as a part of a breeding management system (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Delannay <em>et al.</em> 2012</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Varshney <em>et al.</em> 2015</a>). Finally, training of developing country breeders in genetic and genomics will be essential for implementation.</p></div><div id="sec-19" class="subsection"><h3>Maintenance of genetic variance and prediction accuracy</h3><p id="p-37">Long-term recurrent selection may reduce genetic variance to the point where genetic gains are limited (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Jannink 2010</a>). Indeed, genetic variance showed substantial decline over breeding cycles. The greatest decline was observed in the smallest breeding population given oligogenic architecture and high heritability. These results corroborate previous findings that showed decline of genetic variance during long-term selection is pronounced with smaller population sizes (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Heffner <em>et al.</em> 2010</a>; <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Müller <em>et al.</em> 2017</a>). GARS resulted in greater loss of genetic diversity per unit time compared to PRS. This observation may account for plateauing genetic gain in GARS compared to PRS in final years. The greater loss of genetic variance in GARS <em>vs.</em> PRS can be attributed to greater selection intensity in GARS due to more selection cycles per year (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lorenz <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">De Beukelaer <em>et al.</em> 2017</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Neyhart <em>et al.</em> 2017</a>). Greater loss of genetic variance per unit time in GS compared to phenotypic selection has been attributed to selection and drift of alleles that are not tagged by markers in the training population, which cannot be targeted by GARS (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rutkoski <em>et al.</em> 2015</a>). The accelerated loss of genetic variance in rapid cycle GARS highlights the need for long-term selection strategies that balance genetic gain with maintenance of diversity (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">De Beukelaer <em>et al.</em> 2017</a>).</p><p id="p-38">Maintenance of prediction accuracy across selection cycles is critical for long-term genetic gain (<a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Müller <em>et al.</em> 2017</a>). Prediction accuracies for GARS were initially high, likely due to the within-family prediction scheme, high genetic variance and similarity between the selection population and training population (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lorenz and Nice 2017</a>). Average prediction accuracy after the first year of GARS showed substantial decline. This finding may be explained by the breakdown of LD between markers and QTL due to recombination (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Jannink 2010</a>; <a id="xref-ref-40-7" class="xref-bibr" href="#ref-40">Müller <em>et al.</em> 2017</a>). Phenotypic accuracy showed substantial decline only for the smallest population size, notably for the oligogenic architecture. For the polygenic architecture with low heritability, phenotypic accuracy was less affected by selection and remained proportional to the square root of narrow sense heritability (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Muranty <em>et al.</em> 2015</a>).To maintain GARS prediction accuracy over cycles, we considered a strategy in which the training population was updated and reevaluated to generate new prediction models. We found that updating the training population with the highest performing lines resulted in substantially higher genetic gain and prediction accuracy. Updating the training population has been previously shown to increase short-term prediction accuracy and genetic gain (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Neyhart <em>et al.</em>, 2017</a>). Our study indicated that updating the training population also increased long-term genetic gain. This observation can be explained by more pronounced decay in LD among QTL in long-term selection and increased genetic distance between the training population and the selection candidates, which is addressed by model updating (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Jannink 2010</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Lorenz <em>et al.</em> 2011</a>).</p></div><div id="sec-20" class="subsection"><h3>Next steps for implementation and optimization of GARS</h3><p id="p-39">Currently, the Chibas breeding program is developing multipurpose midheight (∼2 m) sorghum varieties that are simultaneously selected for a high grain yield (white hard to semihard grains primarily for human consumption) and forage quality after grain maturity. Our simulations provide guidance on several key decisions regarding resource allocation and capacity building to maximize genetic gain: (1) achieving three cycles of GARS per year (two off-season nurseries per year) will be critical to provide a higher rate of genetic gain; (2) rapid cycling GARS can provide increased genetic gain even at a modest population size (though a strategy to maintain genetic diversity will be needed if population sizes are small); (3) GARS can be cost-effective for developing countries breeding programs even at modest population size if genotyping cost under US $15; or at higher genotyping costs if larger population can be evaluated; and (4) updating the training population and prediction models using lines that capture new LD will be valuable to maintain high prediction accuracy and genetic gain. It should be noted that the simulation carried out in present study assumes that all selected parents are fertile. In an actual breeding program that employ male sterility system, however, the population segregates for male sterility. Therefore, the crossing pattern of randomly intercrossing the selected parents may differ from the breeding population in which male sterility system segregates in the population.</p><p id="p-40">Future studies will need to address other aspects of genetic architecture and resource allocation that could affect implementation of GARS in small young breeding programs in developing countries. These breeding programs, which often evaluate and cross diverse germplasm, may encounter more epistasis than established breeding programs that have fixed favorable genetic interactions (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wright 1932</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cooper <em>et al.</em> 2009</a>). In addition, changes in environment (new pests, new diseases, climate change, etc.) may shift the adaptive landscape for the breeding program and force them to restart their adaptive walk. Genome-wide prediction models with more flexible parameterization (BayesA, BayesB, random forest, etc.) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Heslot <em>et al.</em> 2012</a>) may capture genetic architecture of small young breeding programs better than the RR-BLUP method used in this study. Another challenge is to achieve dualpurpose crop improvements targets when favorable traits are negatively-correlated due to trade-offs from source-sink relationships (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tovignan <em>et al.</em> 2016</a>). Since the use of GS across heterogeneous target environments is poorly understood, further studies are also required to account for the effect of GxE (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Heslot <em>et al.</em> 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lopez-Cruz <em>et al.</em> 2015</a>).</p><p id="p-41">GS can also be applied to prioritize line advancement (<em>e.g.</em>, S0→S5) for varietal development. Genomic assisted selection on S1 and subsequent families would reduce the number of inbred families that needs to be phenotyped in replicated plots at multiple locations. It could also lower the cost of inbred line development and allow prediction for a greater number of environments and traits. While the results of the current simulation experiment can be applicable to the varietal development cycle of the recurrent selection in terms of the population size and associated costs, additional analysis may be required to test if complete phenotypic evaluation across multiple location should be done at the S2 and subsequent generations. Rigorous theoretical and empirical evaluation of genomic breeding approaches will be critical for translating technology investments into genetic gains for smallholder farmers.</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-42">This study is made possible by the support of the American People provided to the Feed the Future Innovation Lab for Collaborative Research on Sorghum and Millet through the United States Agency for International Development (USAID). The contents are the sole responsibility of the authors and do not necessarily reflect the views of USAID or the United States Government. Program activities are funded by the United States Agency for International Development (USAID) under Associate Award No. AID-OAA-LA-16-00003, “Feed the Future Innovation Lab for Genomics-Assisted Sorghum Breeding”. This study is contribution 19-146-J from the Kansas Agricultural Experiment Station. We thank Guillaume Ramstein and Edward Buckler for helpful suggestions. We thank three anonymous reviewers and the editor for suggestions that improved the study. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-43">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7396781">https://doi.org/10.25387/g3.7396781</a>.</p></li><li class="fn" id="fn-2"><p id="p-44"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-09-13"><span class="received-label">Received </span>September 13, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-12-02"><span class="accepted-label">Accepted </span>December 2, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Muleta <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.1" data-doi="10.1016/j.plantsci.2015.08.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassi</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charmet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Breeding schemes for the implementation of genomic selection in wheat (Triticum spp.).</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">242</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.plantsci.2015.08.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D242%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2015.08.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2015.08.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.2" data-doi="10.1186/1297-9686-44-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bastiaansen</span>  <span class="cit-name-given-names">J. W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coster</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovenhuis</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Long-term response to genomic selection: effects of estimation method and reference population structure for different genetic architectures.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DBastiaansen%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D44%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DLong-term%2Bresponse%2Bto%2Bgenomic%2Bselection%253A%2Beffects%2Bof%2Bestimation%2Bmethod%2Band%2Breference%2Bpopulation%2Bstructure%2Bfor%2Bdifferent%2Bgenetic%2Barchitectures.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-3%26rft_id%253Dinfo%253Apmid%252F22273519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22273519&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.3" data-doi="10.1111/pbi.12106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bekele</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wieckhorst</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedt</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snowdon</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-throughput genomics in sorghum: From whole-genome resequencing to a SNP screening array.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1125</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D11%26rft.spage%253D1112%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.4" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genomewide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2006.11.0690</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.2.391.5" data-doi="10.1101/gr.083634.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D. </span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and flexible simulation of DNA sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.083634.108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.083634.108%26rft_id%253Dinfo%253Apmid%252F19029539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMTM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzM5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.6" data-doi="10.1534/genetics.116.194449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Beukelaer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badke</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fack</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Meyer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Moving beyond managing realized genomic relationship in long-term genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1127</span>–<span class="cit-lpage">1138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.194449</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.194449%26rft_id%253Dinfo%253Apmid%252F28381589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8yLzExMjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.7" data-doi="10.3389/fgene.2016.00096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salgotra</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mir</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic selection in the era of next generation sequencing for complex traits in plant breeding.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2016.00221</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2016.00096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2016.00096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brocke</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trouche</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zongo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barro-kondombo</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weltzien</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Création et amélioration de populations de sorgho à base large avec les agriculteurs au Burkina Faso.</span> <abbr class="cit-jnl-abbrev">Cah. Agric.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCah.%2BAgric.%26rft.volume%253D17%26rft.spage%253D146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.9" data-doi="10.1016/j.pbi.2009.01.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammer</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messina</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Modeling QTL for complex traits: detection and context for plant breeding.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">240</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2009.01.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DCooper%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D231%26rft.epage%253D240%26rft.atitle%253DModeling%2BQTL%2Bfor%2Bcomplex%2Btraits%253A%2Bdetection%2Band%2Bcontext%2Bfor%2Bplant%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2009.01.006%26rft_id%253Dinfo%253Apmid%252F19282235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2009.01.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19282235&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265115400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.10" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.118521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzM5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.11" data-doi="10.1016/j.tplants.2017.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2017.08.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D22%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.08.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.12" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.13" data-doi="10.1007/s11032-011-9611-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delannay</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribaut</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fostering molecular breeding in developing countries.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">873</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-011-9611-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D29%26rft.spage%253D857%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-011-9611-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-011-9611-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.14" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.15" data-doi="10.3835/plantgenome2016.02.0013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faux</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Battagin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">AlphaSim: Software for Breeding Program Simulation.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.02.0013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.02.0013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.02.0013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-other" id="cit-9.2.391.16"><div class="cit-metadata unstructured">Fischer, T., D. Byerlee, G. Edmeades, D. Byerlee, G. Edmeades <em>et al.</em>, 2014 Crop yields and global food security. Aust. Cent. Int. Agric. Res. 1–660.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.17" data-doi="10.2135/cropsci2016.06.0526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Battagin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumasy</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antolin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Prospects for cost-effective genomic selection via accurate within-family imputation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">228</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2016.06.0526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D216%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.06.0526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.06.0526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grenier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ospina</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintero</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Châtel</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy of genomic selection in a rice synthetic population developed for recurrent selection breeding.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-thesis" id="cit-9.2.391.19"><div class="cit-metadata unstructured">Guzman, P. S., 1998 Effective population size, genetic variability, and gains from recurrent selection in the BS11 maize population. Iowa State Univ. Digit. Repos. Retrosp. theses Diss.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.20" data-doi="10.1534/genetics.108.100289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic Selection Using Low-Density Marker Panels.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">353</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.100289</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.100289%26rft_id%253Dinfo%253Apmid%252F19299339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzM5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.21" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.22" data-doi="10.1111/pbr.12037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pembleton</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Prospects for genomic selection in forage plant species.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">143</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbr.12037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D132%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.23" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with Genomic selection: Gain per unit time and cost.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.11.0662</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.24" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic Selection for Crop Improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.08.0512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.25" data-doi="10.1007/s00122-013-2231-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">480</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2231-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2231-5%26rft_id%253Dinfo%253Apmid%252F24264761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2231-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24264761&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.26" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic Selection in Plant Breeding: A Comparison of Models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.27" data-doi="10.1038/ng.3920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiurugwi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction unifies animal and plant breeding programs to form platforms for biological discovery.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3920</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D1297%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3920%26rft_id%253Dinfo%253Apmid%252F28854179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28854179&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.28" data-doi="10.1186/1297-9686-42-35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamics of long-term genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-42-35</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DJannink%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D42%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DDynamics%2Bof%2Blong-term%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-35%26rft_id%253Dinfo%253Apmid%252F20712894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20712894&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.29" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elq001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.30" data-doi="10.1139/X08-086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burdon</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stovold</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gea</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Implications of selection history on genetic architecture of growth, form, and wood-quality traits in <em>Pinus radiata</em>.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">2372</span>–<span class="cit-lpage">2381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/X08-086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D38%26rft.spage%253D2372%26rft_id%253Dinfo%253Adoi%252F10.1139%252FX08-086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/X08-086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.31" data-doi="10.4236/ajps.2011.24070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">B. V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hash</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recent Advances in Sorghum Genetic Enhancement Research at ICRISAT.</span> <abbr class="cit-jnl-abbrev">Am. J. Plant Sci.</abbr> <span class="cit-vol">02</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">600</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4236/ajps.2011.24070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPlant%2BSci.%26rft.volume%253D02%26rft.spage%253D589%26rft_id%253Dinfo%253Adoi%252F10.4236%252Fajps.2011.24070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4236/ajps.2011.24070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model. G3 Genes|Genomes|Genetics (Bethesda) 5: 569–582.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Resource allocation for maximizing prediction accuracy and genetic gain of genomic selection in plant breeding: a simulation experiment. G3 Genes|Genomes|Genetics (Bethesda) 3: 481–91.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.391.34" data-doi="10.1016/B978-0-12-385531-2.00002-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in plant breeding, knowledge and prospects.</span> <span class="cit-source">Adv. Agron.</span> <span class="cit-vol">110</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-12-385531-2.00002-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D110%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-385531-2.00002-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-385531-2.00002-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.391.35" data-doi="10.1007/978-3-319-63170-7_2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nice</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Training Population Design and Resource Allocation for Genomic Selection in Plant Breeding</span>, <span class="cit-source">Genomic Selection for Crop Improvement</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Roorkiwal</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Cham</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-63170-7_2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-63170-7_2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-63170-7_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.36" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-009-1166-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-9.2.391.37"><div class="cit-metadata"><cite>Mausch, K., A. Orr, B. P. Miller, U. N. Avenue, and R. Falls, 2017 Targeting resilience and profitability in African smallholder agriculture : Insights from ICRISAT-led research programs. FACETS 545–558.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/facets-2017-0029</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.38" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.39" data-doi="10.2135/cropsci2017.06.0366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morais Júnior</span>  <span class="cit-name-given-names">O. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duarte</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breseghello</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coelho</span>  <span class="cit-name-given-names">A. S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magalhães Júnior</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Single-step reaction norm models for genomic prediction in multienvironment recurrent selection trials.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2017.06.0366</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D58%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2017.06.0366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2017.06.0366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schopp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Persistency of Prediction Accuracy and Genetic Gain in Synthetic Populations Under Recurrent Genomic Selection. G3 Genes|Genomes|Genetics (Bethesda) 7: 801–811.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.41" data-doi="10.1038/hortres.2015.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muranty</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadok</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rifai</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auwerkerken</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy and responses of genomic selection on key traits in apple breeding.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">15060</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hortres.2015.60</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D2%26rft.spage%253D15060%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhortres.2015.60%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hortres.2015.60&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neyhart</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiede</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection. G3 Genes|Genomes|Genetics (Bethesda) 7: 1499–1510.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.43" data-doi="10.1038/nrg2603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fitness and its role in evolutionary genetics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">539</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D531%26rft.epage%253D539%26rft.atitle%253DFitness%2Band%2Bits%2Brole%2Bin%2Bevolutionary%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2603%26rft_id%253Dinfo%253Apmid%252F19546856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19546856&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268038200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.44" data-doi="10.1038/nature07723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sorghum bicolor genome and the diversification of grasses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07723</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D457%26rft.issue%253D7229%26rft.spage%253D551%26rft.epage%253D556%26rft.atitle%253DThe%2BSorghum%2Bbicolor%2Bgenome%2Band%2Bthe%2Bdiversification%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07723%26rft_id%253Dinfo%253Apmid%252F19189423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19189423&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262852200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.45" data-doi="10.1046/j.1420-9101.1999.00036.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roff</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mousseau</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Does natural selection alter genetic architecture? An evaluation of quantitative genetic variation among populations of Allonemobius socius and A. fasciatus.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">369</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1420-9101.1999.00036.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D12%26rft.spage%253D361%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1420-9101.1999.00036.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1420-9101.1999.00036.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000079104400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.46" data-doi="10.3835/plantgenome2014.06.0025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic Gain from Phenotypic and Genomic Selection for Quantitative Resistance to Stem Rust of Wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.10.0074</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.06.0025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.06.0025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.47" data-doi="10.3835/plantgenome2016.02.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improving Genetic Gain with Genomic Selection in Autotetraploid Potato.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.02.0021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.02.0021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.02.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.48" data-doi="10.1186/1297-9686-41-53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Persistence of accuracy of genome-wide breeding values over generations when including a polygenic effect.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-53</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DSolberg%26rft.auinit1%253DT.%2BR.%26rft.volume%253D41%26rft.spage%253D53%26rft.epage%253D53%26rft.atitle%253DPersistence%2Bof%2Baccuracy%2Bof%2Bgenome-wide%2Bbreeding%2Bvalues%2Bover%2Bgenerations%2Bwhen%2Bincluding%2Ba%2Bpolygenic%2Beffect.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-53%26rft_id%253Dinfo%253Apmid%252F20040081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-53&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20040081&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-other" id="cit-9.2.391.49" data-doi="10.1126/science.1183700"><div class="cit-metadata"><cite>Tester, M., and P. Langridge, 2010 Breeding Technologies to Increase Crop Production in a Changing World. Science 80: 818–822.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1183700</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTester%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5967%26rft.spage%253D818%26rft.epage%253D822%26rft.atitle%253DBreeding%2BTechnologies%2Bto%2BIncrease%2BCrop%2BProduction%2Bin%2Ba%2BChanging%2BWorld%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183700%26rft_id%253Dinfo%253Apmid%252F20150489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84MTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.50" data-doi="10.1016/j.fcr.2016.04.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tovignan</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonceka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ndoye</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cisse</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luquet</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The sowing date and post-flowering water status affect the sugar and grain production of photoperiodic, sweet sorghum through the regulation of sink size and leaf area dynamics.</span> <abbr class="cit-jnl-abbrev">F. Crop. Res.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">77</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2016.04.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF.%2BCrop.%2BRes.%26rft.volume%253D192%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2016.04.015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2016.04.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.51" data-doi="10.1080/07352689.2014.897909"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudapa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pazhamala</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitikineni</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thudi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Translational Genomics in Agriculture: Some Examples in Grain Legumes. CRC.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/07352689.2014.897909</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D34%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07352689.2014.897909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07352689.2014.897909&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.52" data-doi="10.1371/journal.pbio.1001883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terauchi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Harvesting the Promising Fruits of Genomics: Applying Genome Sequencing Technologies to Crop Breeding.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1001883</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1001883</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D12%26rft.spage%253De1001883%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001883%26rft_id%253Dinfo%253Apmid%252F24914810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24914810&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.53" data-doi="10.2135/cropsci1998.0011183X003800020005x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weyhrich</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Responses to seven methods of recurrent selection in the BS11 maize population.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">308</span>–<span class="cit-lpage">321</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1998.0011183X003800020005x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWeyhrich%26rft.auinit1%253DR.%2BA.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D308%26rft.epage%253D321%26rft.atitle%253DResponses%2Bto%2BSeven%2BMethods%2Bof%2BRecurrent%2BSelection%2Bin%2Bthe%2BBS11%2BMaize%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1998.0011183X003800020005x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1998.0011183X003800020005x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000073474200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments. G3 Genes|Genomes|Genetics (Bethesda) 2: 1427–1436.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.55" data-doi="10.1007/s00122-008-0715-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">815</span>–<span class="cit-lpage">824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-008-0715-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWong%26rft.auinit1%253DC.%2BK.%26rft.volume%253D116%26rft.issue%253D6%26rft.spage%253D815%26rft.epage%253D824%26rft.atitle%253DGenomewide%2Bselection%2Bin%2Boil%2Bpalm%253A%2Bincreasing%2Bselection%2Bgain%2Bper%2Bunit%2Btime%2Band%2Bcost%2Bwith%2Bsmall%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0715-5%26rft_id%253Dinfo%253Apmid%252F18219476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0715-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18219476&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254262600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.2.391.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1932</span> The roles of mutation, inbreeding, crossbreeding and selection in evolution. Proc. Sixth Int. Congr. Genet. 1: <span class="cit-fpage">356</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.57" data-doi="10.1093/jxb/erx135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Enhancing genetic gain in the era of molecular breeding.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">2641</span>–<span class="cit-lpage">2666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erx135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft.volume%253D68%26rft.spage%253D2641%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferx135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erx135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.391.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> Rapid Cycling Genomic Selection in a Multiparental Tropical Maize Population. G3 Genes|Genomes|Genetics 7: 2315–2326.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/2/391.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/2/375" title="A Very Oil Yellow1 Modifier of the Oil Yellow1-N1989 Allele Uncovers a Cryptic Phenotypic Impact of Cis-regulatory Variation in Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/2/403" title="A Transgenic Flock House Virus Replicon Reveals an RNAi Independent Antiviral Mechanism Acting in Drosophila Follicular Somatic Cells" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/2"><span>Volume 9 Issue 2, February 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/2.cover-source.jpg?itok=Pgd0L52l" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/391" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CeiGBWsBu6nM2bAaMhDqzH1QTy8xr9t62TWiMWlevcE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/2/391" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/2/391" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/2/391" class="active" data-icon-position="" data-hide-link-title="0">Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492134" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-V0zeQ5jYFptN4oQvxYMQvtxmwbbyQDVpvzIfE7OKqME" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/2/391.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/2/391" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Wf3m2YcO5aXzhlRPj-LRZ3XG5-cttsI2pKl4v1B-ILo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/2/391" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/2/391.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492134" id="node492134--2" data-pisa="ggg;9/2/391" data-pisa-master="ggg;g3.118.200932" data-apath="/ggg/9/2/391.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5403-5098" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kebede T. Muleta</span>, <span class="highwire-citation-author" data-delta="1">Gael Pressoir</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3067-3359" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Geoffrey P. Morris</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">391-401;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200932 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492134--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kebede T. Muleta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Kansas State University, Manhattan, Kansas</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kebede%2BT.%2BMuleta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muleta%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKebede%2BT.%2BMuleta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5403-5098" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kebede T. Muleta</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gael Pressoir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Chibas and Faculty of Agriculture and Environmental Sciences, Quisqueya University, Port-au-Prince, Haiti</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gael%2BPressoir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pressoir%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGael%2BPressoir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Geoffrey P. Morris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Kansas State University, Manhattan, Kansas</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Geoffrey%2BP.%2BMorris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morris%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeoffrey%2BP.%2BMorris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3067-3359" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Geoffrey P. Morris</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gpmorris@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">gpmorris@ksu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/2/391" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/2/391&amp;title=Optimizing%20Genomic%20Selection%20for%20a%20Sorghum%20Breeding%20Program%20in%20Haiti%3A%20A%20Simulation%20Study" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/2/391&amp;title=Optimizing%20Genomic%20Selection%20for%20a%20Sorghum%20Breeding%20Program%20in%20Haiti%3A%20A%20Simulation%20Study" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/2/391&amp;title=Optimizing%20Genomic%20Selection%20for%20a%20Sorghum%20Breeding%20Program%20in%20Haiti%3A%20A%20Simulation%20Study" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/2/391&amp;text=Optimizing%20Genomic%20Selection%20for%20a%20Sorghum%20Breeding%20Program%20in%20Haiti%3A%20A%20Simulation%20Study" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/2/391&amp;title=Optimizing%20Genomic%20Selection%20for%20a%20Sorghum%20Breeding%20Program%20in%20Haiti%3A%20A%20Simulation%20Study" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/2/391&amp;t=Optimizing%20Genomic%20Selection%20for%20a%20Sorghum%20Breeding%20Program%20in%20Haiti%3A%20A%20Simulation%20Study" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/2/391&amp;title=Optimizing%20Genomic%20Selection%20for%20a%20Sorghum%20Breeding%20Program%20in%20Haiti%3A%20A%20Simulation%20Study" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/2/391&amp;title=Optimizing%20Genomic%20Selection%20for%20a%20Sorghum%20Breeding%20Program%20in%20Haiti%3A%20A%20Simulation%20Study" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492134/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492134" id="node492134--3" data-pisa="ggg;9/2/391" data-pisa-master="ggg;g3.118.200932" data-apath="/ggg/9/2/391.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Genomic Selection for a Sorghum Breeding Program in Haiti: A Simulation Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5403-5098" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kebede T. Muleta</span>, <span class="highwire-citation-author" data-delta="1">Gael Pressoir</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3067-3359" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Geoffrey P. Morris</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">391-401;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200932 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492134--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kebede T. Muleta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Kansas State University, Manhattan, Kansas</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kebede%2BT.%2BMuleta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muleta%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKebede%2BT.%2BMuleta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5403-5098" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kebede T. Muleta</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gael Pressoir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Chibas and Faculty of Agriculture and Environmental Sciences, Quisqueya University, Port-au-Prince, Haiti</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gael%2BPressoir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pressoir%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGael%2BPressoir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Geoffrey P. Morris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Kansas State University, Manhattan, Kansas</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Geoffrey%2BP.%2BMorris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morris%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeoffrey%2BP.%2BMorris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3067-3359" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Geoffrey P. Morris</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gpmorris@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">gpmorris@ksu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=391&amp;rft.epage=401&amp;rft.atitle=Optimizing%20Genomic%20Selection%20for%20a%20Sorghum%20Breeding%20Program%20in%20Haiti%3A%20A%20Simulation%20Study&amp;rft.volume=9&amp;rft.issue=2&amp;rft.date=2019-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Muleta%2C+Kebede+T.&amp;rft.au=Pressoir%2C+Gael&amp;rft.au=Morris%2C+Geoffrey+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492134/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492134/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492134/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492134/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492134/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492134/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492134/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492134/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492134/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492134/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492134/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492134/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/2/391" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">MATERIALS AND METHODS</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/2/391.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/2/391.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/2/391.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/2/391.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
