<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/2/327" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/2/327.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/2/327.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/2/327.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/2/327/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/2/327.atom" />
<meta name="HW.pisa" content="ggg;9/2/327" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="pSBVB: A Versatile Simulation Tool To Evaluate Genomic Selection in Polyploid Species" />
<meta name="DC.Identifier" content="10.1534/g3.118.200942" />
<meta name="DC.Date" content="2019-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Zingaretti et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic Selection (GS) is the procedure whereby molecular information is used to predict complex phenotypes and it is standard in many animal and plant breeding schemes. However, only a small number of studies have been reported in horticultural crops, and in polyploid species in particular. In this paper, we have developed a versatile forward simulation tool, called polyploid Sequence Based Virtual Breeding (pSBVB), to evaluate GS strategies in polyploids; pSBVB is an efficient gene dropping software that can simulate any number of complex phenotypes, allowing a very flexible modeling of phenotypes suited to polyploids. As input, it takes genotype data from the founder population, which can vary from single nucleotide polymorphisms (SNP) chips up to sequence, a list of causal variants for every trait and their heritabilities, and the pedigree. Recombination rates between homeologous chromosomes can be specified, so that both allo- and autopolyploid species can be considered. The program outputs phenotype and genotype data for all individuals in the pedigree. Optionally, it can produce several genomic relationship matrices that consider exact or approximate genotype values. pSBVB can therefore be used to evaluate GS strategies in polyploid species (say varying SNP density, genetic architecture or population size, among other factors), or to optimize experimental designs for association studies. We illustrate pSBVB with SNP data from tetraploid potato and partial sequence data from octoploid strawberry, and we show that GS is a promising breeding strategy for polyploid species but that the actual advantage critically depends on the underlying genetic architecture. Source code, examples and a complete manual are freely available in GitHub &lt;https://github.com/lauzingaretti/pSBVB&gt;." />
<meta name="DC.Contributor" content="María L. Zingaretti" />
<meta name="DC.Contributor" content="Amparo Monfort" />
<meta name="DC.Contributor" content="Miguel Pérez-Enciso" />
<meta name="article:published_time" content="2019-02-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="pSBVB: A Versatile Simulation Tool To Evaluate Genomic Selection in Polyploid Species" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic Selection (GS) is the procedure whereby molecular information is used to predict complex phenotypes and it is standard in many animal and plant breeding schemes. However, only a small number of studies have been reported in horticultural crops, and in polyploid species in particular. In this paper, we have developed a versatile forward simulation tool, called polyploid Sequence Based Virtual Breeding (pSBVB), to evaluate GS strategies in polyploids; pSBVB is an efficient gene dropping software that can simulate any number of complex phenotypes, allowing a very flexible modeling of phenotypes suited to polyploids. As input, it takes genotype data from the founder population, which can vary from single nucleotide polymorphisms (SNP) chips up to sequence, a list of causal variants for every trait and their heritabilities, and the pedigree. Recombination rates between homeologous chromosomes can be specified, so that both allo- and autopolyploid species can be considered. The program outputs phenotype and genotype data for all individuals in the pedigree. Optionally, it can produce several genomic relationship matrices that consider exact or approximate genotype values. pSBVB can therefore be used to evaluate GS strategies in polyploid species (say varying SNP density, genetic architecture or population size, among other factors), or to optimize experimental designs for association studies. We illustrate pSBVB with SNP data from tetraploid potato and partial sequence data from octoploid strawberry, and we show that GS is a promising breeding strategy for polyploid species but that the actual advantage critically depends on the underlying genetic architecture. Source code, examples and a complete manual are freely available in GitHub https://github.com/lauzingaretti/pSBVB.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/02/01" />
<meta name="citation_mjid" content="ggg;9/2/327" />
<meta name="citation_id" content="9/2/327" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/2/327" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/2/327.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/2/327.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/2/327.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200942" />
<meta name="citation_pmid" content="30573468" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="327" />
<meta name="citation_lastpage" content="334" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="María L. Zingaretti" />
<meta name="citation_author_institution" content="Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain" />
<meta name="citation_author_email" content="laura.zingaretti@cragenomica.es" />
<meta name="citation_author_email" content="miguel.perez@uab.es" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5618-2630" />
<meta name="citation_author" content="Amparo Monfort" />
<meta name="citation_author_institution" content="Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain" />
<meta name="citation_author_institution" content="IRTA (Institut de Recerca i Tecnologia Agroalimentàries), Barcelona, Spain" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7106-7745" />
<meta name="citation_author" content="Miguel Pérez-Enciso" />
<meta name="citation_author_institution" content="Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain" />
<meta name="citation_author_institution" content="ICREA, Passeig de Lluís Companys 23, 08010 Barcelona, Spain" />
<meta name="citation_author_email" content="laura.zingaretti@cragenomica.es" />
<meta name="citation_author_email" content="miguel.perez@uab.es" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3524-995X" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=F. Balloux;citation_title=EASYPOP (Version 1.7): A Computer Program for Population Genetics Simulations;citation_pages=301-302;citation_volume=92;citation_year=2001;citation_issue=3;citation_pmid=11447253;citation_doi=10.1093/jhered/92.3.301" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=F. M. Bassi;citation_author=A. R. Bentley;citation_author=G. Charmet;citation_author=R. Ortiz;citation_author=J. Crossa;citation_title=Breeding schemes for the implementation of genomic selection in wheat (triticum spp.).;citation_pages=23-36;citation_volume=242;citation_year=2016;citation_doi=10.1016/j.plantsci.2015.08.021" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=N. V. Bassil;citation_author=T. M. Davis;citation_author=H. Zhang;citation_author=S. Ficklin;citation_author=M. Mittmann;citation_title=Development and preliminary evaluation of a 90 K Axiom SNP array for the allo-octoploid cultivated strawberry Fragaria × ananassa.;citation_pages=155;citation_volume=16;citation_year=2015;citation_pmid=25886969;citation_doi=10.1186/s12864-015-1310-1" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. D. Blischak;citation_author=L. S. Kubatko;citation_author=A. D. Wolfe;citation_title=SNP genotyping and parameter estimation in polyploids using low-coverage sequencing data;citation_pages=407-415;citation_volume=34;citation_year=2017;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=P. M. Bourke;citation_author=P. Arens;citation_author=R. E. Voorrips;citation_author=G. D. Esselink;citation_author=C. F. Koning-Boucoiran;citation_title=Partial preferential chromosome pairing is genotype dependent in tetraploid rose.;citation_pages=330-343;citation_volume=90;citation_year=2017;citation_doi=10.1111/tpj.13496" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=P. M. Bourke;citation_author=R. E. Voorrips;citation_author=R. G. F. Visser;citation_author=C. Maliepaard;citation_title=Tools for Genetic Studies in Experimental Populations of Polyploids.;citation_pages=513;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00513" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Caballero;citation_author=A. Tenesa;citation_author=P. D. Keightley;citation_title=The Nature of Genetic Variation for Complex Traits Revealed by GWAS and Regional Heritability Mapping Analyses.;citation_volume=201;citation_year=2015;citation_pmid=26482794;citation_doi=10.1534/genetics.115.177220" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Clevenger;citation_author=P. Ozias-Akins;citation_title=SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops. G3: Genes, Genomes;citation_pages=1797-1803;citation_volume=5;citation_year=2015;citation_doi=10.1534/g3.115.019703" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. e los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_author=E. Manfredi;citation_title=Predicting Quantitative Traits with Regression Models for Dense Molecular Markers and Pedigree;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=G3: Genes, Genomes, Genetics;citation_author=F. Enciso-Rodriguez;citation_author=D. Douches;citation_author=M. Lopez-Cruz;citation_author=J. Coombs;citation_author=G. de los Campos;citation_title=Genomic Selection for Late Blight and Common Scab Resistance in Tetraploid Potato (Solanum tuberosum;citation_volume=8;citation_year=2018;citation_pmid=29794167;citation_doi=10.1534/g3.118.200273" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Eyre-Walker;citation_author=P. D. Keightley;citation_title=The distribution of fitness effects of new mutations.;citation_pages=610-618;citation_volume=8;citation_year=2007;citation_issue=8;citation_pmid=17637733;citation_doi=10.1038/nrg2146" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=A. García-Dorado;citation_author=J. L. Monedero;citation_author=C. López-Fanjul;citation_title=The mutation rate and the distribution of mutational effects of viability and fitness in Drosophila melanogaster.;citation_pages=255-265;citation_volume=102–103;citation_year=1998;citation_doi=10.1023/A:1017086909282" />
<meta name="citation_reference" content="citation_author=D. Gerard;citation_author=L. F. V. Ferrão;citation_author=A. A. F. Garcia;citation_author=M. Stephens;citation_title=Genotyping polyploids from messy sequencing data;citation_year=2018;citation_pmid=30185430;citation_doi=10.1534/genetics.118.301468" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=S. A. Gezan;citation_author=L. F. Osorio;citation_author=S. Verma;citation_author=V. M. Whitaker;citation_title=An experimental validation of genomic selection in octoploid strawberry.;citation_pages=16070;citation_volume=4;citation_year=2017;citation_doi=10.1038/hortres.2016.70" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. He;citation_author=S. Saha;citation_author=R. Finkers;citation_author=L. Parida;citation_title=Efficient algorithms for polyploid haplotype phasing.;citation_pages=110;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-4464-9" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Howie;citation_author=C. Fuchsberger;citation_author=M. Stephens;citation_author=J. Marchini;citation_author=G. R. Abecasis;citation_title=Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.;citation_pages=955-959;citation_volume=44;citation_year=2012;citation_issue=8;citation_pmid=22820512;citation_doi=10.1038/ng.2354" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Kessner;citation_author=J. Novembre;citation_title=forqs: forward-in-time simulation of recombination, quantitative traits and selection.;citation_volume=30;citation_year=2014;citation_pmid=24336146;citation_doi=10.1093/bioinformatics/btt712" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=P. G. Meirmans;citation_author=S. Liu;citation_author=P. H. van Tienderen;citation_title=The analysis of polyploid genetic data.;citation_pages=283-296;citation_volume=109;citation_year=2018;citation_doi=10.1093/jhered/esy006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. W. Messer;citation_title=SLiM: simulating evolution with selection and linkage.;citation_volume=194;citation_year=2013;citation_pmid=23709637;citation_doi=10.1534/genetics.113.152181" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=S. Naithani;citation_author=C. M. Partipilo;citation_author=R. Raja;citation_author=J. L. Elser;citation_author=P. Jaiswal;citation_title=FragariaCyc: A Metabolic Pathway Database for Woodland Strawberry Fragaria vesca.;citation_pages=1-10;citation_volume=7;citation_year=2016;citation_pmid=26858731;citation_doi=10.3389/fpls.2016.01509" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Peng;citation_author=C. I. Amos;citation_title=Forward-time simulations of non-random mating populations using simuPOP.;citation_volume=24;citation_year=2008;citation_pmid=18417488;citation_doi=10.1093/bioinformatics/btn179" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Peng;citation_author=M. Kimmel;citation_title=simuPOP: A forward-time population genetics simulation environment.;citation_volume=21;citation_year=2005;citation_pmid=16020469;citation_doi=10.1093/bioinformatics/bti584" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Pérez-Enciso;citation_author=N. Forneris;citation_author=G. de los Campos;citation_author=A. Legarra;citation_title=Evaluating Sequence-Based Genomic Prediction with an Efficient New Simulator.;citation_volume=205;citation_year=2017;citation_pmid=27913617;citation_doi=10.1534/genetics.116.194878" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=E. Raineri;citation_author=L. Ferretti;citation_author=A. Esteve-Codina;citation_author=B. Nevado;citation_author=S. Heath;citation_title=SNP calling by sequencing pooled samples.;citation_pages=239-239;citation_volume=13;citation_year=2012;citation_pmid=22992255;citation_doi=10.1186/1471-2105-13-239" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=U. R. Rosyara;citation_author=W. S. De Jong;citation_author=D. S. Douches;citation_author=J. B. Endelman;citation_title=Software for genome-wide association studies in autopolyploids and its application to potato.;citation_pages=1-10;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.03.0027" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. A. Schmitz Carley;citation_author=J. J. Coombs;citation_author=D. S. Douches;citation_author=P. C. Bethke;citation_author=J. P. Palta;citation_title=Automated tetraploid genotype calling by hierarchical clustering.;citation_pages=717-726;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2845-5" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Shen;citation_author=Z. Li;citation_author=J. Chen;citation_author=Z. Song;citation_author=Z. Zhou;citation_title=Shesisplus, a toolset for genetic studies on polyploid species.;citation_pages=24095;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep24095" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. T. Slater;citation_author=N. O. Cogan;citation_author=J. W. Forster;citation_author=B. J. Hayes;citation_author=H. D. Daetwyler;citation_title=Improving genetic gain with genomic selection in autotetraploid potato.;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.02.0021" />
<meta name="citation_reference" content="citation_journal_title=Advances in genetics;citation_journal_abbrev=Adv Genet;citation_author=G. L. Stebbins;citation_title=Types of polyploids; their classification and significance.;citation_pages=403-429;citation_volume=1;citation_year=1947;citation_pmid=20259289;citation_doi=10.1016/S0065-2660(08)60490-3" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. Sverrisdóttir;citation_author=S. Byrne;citation_author=E. H. R. Sundmark;citation_author=H. Ø. Johnsen;citation_author=H. G. Kirk;citation_title=Genomic prediction of starch content and chipping quality in tetraploid potato using genotyping-by-sequencing.;citation_pages=2091-2108;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2944-y" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="Voorrips, R. E. and G. Gort, 2018 fitPoly: Genotype Calling for Bi-Allelic Marker Assays. R package version 3.0.0." />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. E. Voorrips;citation_author=G. Gort;citation_author=B. Vosman;citation_title=Genotype calling in tetraploid species from bi-allelic marker data using mixture models.;citation_pages=172-172;citation_volume=12;citation_year=2011;citation_pmid=21595880;citation_doi=10.1186/1471-2105-12-172" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. E. Voorrips;citation_author=C. A. Maliepaard;citation_title=The simulation of meiosis in diploid and tetraploid organisms using various genetic models.;citation_pages=248-248;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23013469;citation_doi=10.1186/1471-2105-13-248" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="pSBVB: A Versatile Simulation Tool To Evaluate Genomic Selection in Polyploid Species" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
<meta name="twitter:description" content="Genomic Selection (GS) is the procedure whereby molecular information is used to predict complex phenotypes and it is standard in many animal and plant breeding schemes. However, only a small number of studies have been reported in horticultural crops, and in polyploid species in particular. In this paper, we have developed a versatile forward simulation tool, called polyploid Sequence Based Virtual Breeding (pSBVB), to evaluate GS strategies in polyploids; pSBVB is an efficient gene dropping software that can simulate any number of complex phenotypes, allowing a very flexible modeling of phenotypes suited to polyploids. As input, it takes genotype data from the founder population, which can vary from single nucleotide polymorphisms (SNP) chips up to sequence, a list of causal variants for every trait and their heritabilities, and the pedigree. Recombination rates between homeologous chromosomes can be specified, so that both allo- and autopolyploid species can be considered. The program outputs phenotype and genotype data for all individuals in the pedigree. Optionally, it can produce several genomic relationship matrices that consider exact or approximate genotype values. pSBVB can therefore be used to evaluate GS strategies in polyploid species (say varying SNP density, genetic architecture or population size, among other factors), or to optimize experimental designs for association studies. We illustrate pSBVB with SNP data from tetraploid potato and partial sequence data from octoploid strawberry, and we show that GS is a promising breeding strategy for polyploid species but that the actual advantage critically depends on the underlying genetic architecture. Source code, examples and a complete manual are freely available in GitHub &lt;https://github.com/lauzingaretti/pSBVB&gt;." />
<meta name="og-title" property="og:title" content="pSBVB: A Versatile Simulation Tool To Evaluate Genomic Selection in Polyploid Species" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/2/327" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic Selection (GS) is the procedure whereby molecular information is used to predict complex phenotypes and it is standard in many animal and plant breeding schemes. However, only a small number of studies have been reported in horticultural crops, and in polyploid species in particular. In this paper, we have developed a versatile forward simulation tool, called polyploid Sequence Based Virtual Breeding (pSBVB), to evaluate GS strategies in polyploids; pSBVB is an efficient gene dropping software that can simulate any number of complex phenotypes, allowing a very flexible modeling of phenotypes suited to polyploids. As input, it takes genotype data from the founder population, which can vary from single nucleotide polymorphisms (SNP) chips up to sequence, a list of causal variants for every trait and their heritabilities, and the pedigree. Recombination rates between homeologous chromosomes can be specified, so that both allo- and autopolyploid species can be considered. The program outputs phenotype and genotype data for all individuals in the pedigree. Optionally, it can produce several genomic relationship matrices that consider exact or approximate genotype values. pSBVB can therefore be used to evaluate GS strategies in polyploid species (say varying SNP density, genetic architecture or population size, among other factors), or to optimize experimental designs for association studies. We illustrate pSBVB with SNP data from tetraploid potato and partial sequence data from octoploid strawberry, and we show that GS is a promising breeding strategy for polyploid species but that the actual advantage critically depends on the underlying genetic architecture. Source code, examples and a complete manual are freely available in GitHub &lt;https://github.com/lauzingaretti/pSBVB&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
    <title>pSBVB: A Versatile Simulation Tool To Evaluate Genomic Selection in Polyploid Species | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"nktHVr9xQguzObLp-X-X-JzFQIt--quBj1n7TcNyMFQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492183","apath":"\/ggg\/9\/2\/327.atom","pisa":"ggg;9\/2\/327","ac":{"\/ggg\/9\/2\/327.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/2\/327.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/2\/327"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/2\/327"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/2\/327":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002299o3pTvkbmSgzW9InZ083++cdTHBtU8Mo4ZijL7rqzh1aA9XNK7jEUtco2+tCDBqXW0C72dqMxIOm+UIxdG6SWXjhX5UKrR4OzlqxKKkCyLDBt4tNmMGV9rabn1lqq71nmsaT8GfvFLDR+sjX\\\\\\\/iEjGpLaJDIu5bBr6FLyTyH833CoQBx2S8RYOvoPOMVuq6CvV3PSW12IVFkkCXdaCFxQp0EE3s+Qr\\\\\\\/hmI9tSaVljEgCuEI5yFXxU6bHBKD\\\\\\\/Sl70pec6vP7i4YpGrxcbmcgAJ8IbTUVCaKCTWL6ZCsB8PLVvweq1uB4NUWvgk1jLh\\\\\\\/T8RTyRbYhdkCCWkrC+IrxXOPZ9WXJ6LMO8mmsAdahAx6XKQzTDc5K28DjexLwqap7TqWgy3s+Oyqz+cK8HOg+iE3WQ7Y051TrlXZbPNNyK7SLcA7L3z+If3JXBfyY\\\\\\\/BOylcYRXxxvQdpEU4OcMznDJLt9Xkdyn+ih4XT0DOFg4NV82P\\\\\\\/WtzuBJ53p\\\\\\\/mu9aZ91UFbXUxFAre8vaxN\\\\\\\/PJMZQQoc00DZ5RN7n2o+npUEK9km7vZyAzxyNLD2qtLqWRe3jRvMs+u2NwqD+TIikhbd37Jvccpy3cEnh0Rq8QB+HQ8vvglmIqq7MgBbfbT5\\\\\\\/4fF\\\\\\\/nsBr5BVjO5Srqe0PXsOCHIi5HsSsRoy4XK8gZOK3wqKr\\\\\\\/A6bLseHSZC23NLUJ4rzH0VreG\\\\\\\/MI4IcmtecqesG80uhyfVU69uWMhNm5tdAxlWUMRD5OsEBO6URpTh2f4yKkGRN7xS6CqlNDul2FX5bVgMkuir12BIsJhPta7D1WfPwJt+p88NzDBSb+Caxaw2M0UMb8BBblRATjE\\\\\\\/DKHjpcFDgtAvpaaZ+g30ojjPJ3WtbD\\\\\\\/Q+ort78UUNGVtFKd6AOb3KdWtNd\\\\\\\/F6CpGnvwV71xW0DZYOL2BDxASuhTCewW4jEZe7\\\\\\\/ZDuz5js1C6rrtaN\\\\\\\/VW\\\\\\\/TClqPNZI09MIjB75c5AMwz3sUeafM\\\\\\\/4IurbUSUITZoKJM3jyhze\\\\\\\/RreqL4L53X1FWpNWbFaCkDhWKI0gl1umKT8W95kGBB49ZE71CIoTeuEBt9mrKELhid90YoMUIOo0Zg5IrMf2rkSvGu3h5Y8AIv1AGDfomZwrWR1tMC4f6izTWT7uTDQxl54gAcI5GyrcaddMw1wHZc6cNK3l4Gpz5R+gGGwg1utScKoGvYWaa99w0wmQYv1qHfL\\\\\\\/5ZW17HuXqfV8pTZkmm0inYpmIYggwWwkRbMCMlOQu1cSczjNRXCU6Z9YSLLrmDhhX2vz\\\u0022,\\\u0022iv\\\u0022:\\\u0022fyvb2gn2cbyG7B3pFUQIJw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226705695893c22258cce1dd4d71f762e773fefc550d46b9a23b45fd071fec3e4c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oiNlVrxqLxTwMcc6z35Tis8RWsONcpblEUTf0VXw11hzE7T3M9TMeo\\\\\\\/gjFPZFOCjTmr+YEb4971SAgemqZ7nuGd+sSV1A+e011bb4vzceJxyYMhcoCfZEH4g8xOsnARMawgSTLKRWrO9+h8H9IDGBmAfhJ4CHBqEzrhDu933pgvOe++2m183WljVd4s3MrED6n+HuEFusiS4WosJNeo4xNst\\\\\\\/OC0dYCXSHC2vZPZ7jyxlu3pxTy0toBWjuFWV9Lz6O13\\\\\\\/YgKDngZSivwysiXgHrNbcvktztQeujgl65Z5rHMfT7SyoaO\\\\\\\/vZs+13srncPQS6REJ7rS5h5KcYQVDB1GHlRaJjw+UzaGiVQycdnDDrArXnQHdrPD33Q0f2JVQJtguOxPrN+EhfWBHrAaxRtER+WFHSuQdJ83SrmlR2S8OdpaV5heH5CoBgQCR5UjTAP9ehNffSY3sro1h6L4di1L5UsFwg9dhRqux5CWPB9RKKK42jti7W6ueE67MrlCXr+fb93YSYtAdZ7iVIg5FD38Df6OwtazuKXwzgM4aI+IdP63k99L5Ad9Jsn9v4CKzqIvDpUO1+VUIE8RgBQ2GOcW6CgF4eDpcF59LgCFRTKTN8Hmgn3H0+RldPibOfdZNpG1CIFbQoujaYmSkKsDN7N9qYCDnsa9LAZPOKneEs3DDRL\\\\\\\/OsUTEnENnDH3PwlrsfEHRcrxDUf2FTbaRdsh+TuX9\\\\\\\/OjgsMpKx68ZrQ\\\\\\\/r3IcY8Y\\\\\\\/QSjpgXeLRZ1\\\\\\\/z2ufZRluWIMrOTuCzxQAYn9acY3oV5EsyKep0Al7mP92i3qB8kK2bnaaqdATKSVnQfE4hs4Qe8EWnfMVqXSTuPE\\\\\\\/bJY\\\\\\\/tOYrgDUhE8HRKhf\\\\\\\/CQcMDWbQMdq6m3coKCzYsOW9XSctFz3RtbZN1Ls\\\\\\\/5fhs1JwyPfw7Vo2sIv6CvfY2O3ZOgMVLlmvPpiwaJjBLbu4mN8iNOsaX0ikes9sCjcvZkSwVJpmX3jEcejXNVTjBL6JVOL87HwFEqhzWhpBhvO7tOzfl62FSdtgKS1pn8hwA2ZqkeTRVvRYhHPrYq5tpmTKE7VhbxSQAea+jCzK6\\\\\\\/T\\\\\\\/Arb\\\\\\\/M6ZGBUSXfpo6skuaA3LvOKfXX2JyLJY+vuxsUAQ+noSfmx7HdU0gbwrZt5EvHWCWg5pfQxSAl02YxfKB6TJy2coCSMQZte75BgUWNBLNSDm3AqpnMSD1MpC1lEoZ5tS4D6ceZCVzXwb7Eo50EyRM743WnV1\\\\\\\/2li\\\\\\\/x3J3kRMd0kYE6LQeNDrxz9Oou3ZPyV3i6zdOYbE7mwoGtTDIIimNVex5poWSbNT1+Wd832PBCLbWw2J\\\\\\\/7XDYO7HX6lmLkoC0128hKWKJN\\\\\\\/FSp4Elmx3mdSng16DqFZA5es1Bt3ORcqSYw5uNnjWbUjNeUSE9vDMaNnmJALy78JFvenWaLhB4+I0sIOB6bPvxvfMsmMBH4Lav2h+Vd\\\\\\\/U6v6a23BzFFkfYSk\\\\\\\/Vcz6SVYcZz9zjieXLh8irHx3ogYTIihwEGctvGtjJiGQCqn7Zpacvd9OPX2VGK\\\\\\\/jRc3Z1zoDQIRQIwaFk0opKEgmi08gpcIJyzlvTI8zDlGJHdAECRXTuZ8KQB55zFkLarkQhrP5bqXNYEZ0rb21AAGlCyEkPt1gD4Yq2iuclDEftC8AxomX13sFLLtqs8VWw\\\\\\\/KIHv7WMdI1xRyz2+D8UsuLos11YXj7ciG1CRXuEhNSWJM8SywwVKi\\\\\\\/D6sePu7a5RMD5ba77fWhxEBt0bfhZ4WlZwqZJZxNxZEEgBLAojx2tBm0CLb8kbT1vqWcjgQDm+P86pmCLcrO0VWlF2u2OH5AIon8MKeX44NzDV9hSaGRIFyYPP4nQ5nLszGL\\\\\\\/zJgGegisLmVByzocy4HzG1pZSnj47EdbRdw4N3qVybYzGGbmG6oS6Hd8BDkK3dt57SuQs3InNTxOxNGk4xch9lIfujX8q\\\\\\\/aPR6Y7lWjsjI20XpEqb06ZGqvRNnzpwVh7XrEWngdcDn2H4mrUWjrlJgq\\\\\\\/n1ii49TwI74CWf\\\\\\\/bg47TAhRbhQz2uRl8MU86SgpnlJHuCbQVkZoCF\\\\\\\/Fpr1XbNYLwIcQ8LuWap1ydBnln0i12Mf7GZSZGkZ34kIlLRTrpXikgGIitia2Wu8E8C0LqoPIqCrYtOlabrIx7LvpyEZeTBu48OcF0XciPaXXIVMKjvr3qViPHbPpRyc\\\\\\\/AyF8tfjXwee1hSRE1l8wJ2uEeBidBnyoRY\\\\\\\/fJCM2heYn+40LV7uP8kWKqlBOBHvDyOIo155MtZwNMGbw4W4a6di6mEj+Z8wFhooI3SoLbCWLTQT9J6F5uzs\\\\\\\/PCIiRojcMjEiYZAxYJeqaVxazvgAXpp9Smne8gdskI5dvany6y94HDvWKWFmx4s5x5cV8DmkQhSmA3e9SAe+DOFBUsB5QvIpzGcZNKjUm4g1DOflt5fivnIIXTp\\\\\\\/8fZgwVEeRSO8Vzw5sNGp5Yo+OlyzWSM6kE0YN6AtnrFbSRaVhzAE1rJeCM5sRUI4LU0YpptIH7IAJ\\\\\\\/S9k+3nKa+Kdo3QI4xcdZSF8uhPRGtqHUax7x5FEV8E9lskPu8IZh6It1vkbZmq6r8ATsIiYwxIoZahArM55kdUIWWGyCw432yU5oFAckyarHQyv4VdKt5Db1DE3SPqK+Oa6TPBa11R\\\\\\\/1xZMeyemqu0xr99FNGhzKMhuJ2TJVehfF5PPQQVDNbgApCyUT4qq7DO9DY8tvwWkY4Ka5quNvAVHjgwhFsw04Fmhkrj8tvKJYDVCosA6s6jAYrPi4L6cHDQ+MhOM0pfC+8EnUxaFVZYKcJckhKoPnVHl6hFn8P99FPr1aRnAsTtZHyXoRvb\\\\\\\/nXvwZvfSaEZuhVQu1NVDWvvifRxILRq8A74JDqxsbdkKLC5+i6qfevvdphcWoo\\\\\\\/D9MU9HWLMFNlnVp9IfUuMBUQOhsQYbbAgEkNL4TGbOHz2dWEdszpddB+t\\\\\\\/iOkujYYId6HOiufuXGZDWEbrlyAlEr3QOWDzbIxYiyObTv7Fhkh6UBlCMQL7BlshZQ5nTjm03tAqXlPnXiM03FbGyyztmtqO+rLnHzGxcT8YYfGZ\\\\\\\/1lPLWqCEptpTfYoY\\\\\\\/ui6yYczDhFcE61QJMA5+mb9BL5eC0stYI+4eqbYnJhS9ctuiSvrRqwckTM6hB8bFX\\\\\\\/DQQ2r5W1aKKNFaxvOZLN1pFRH2gxXAtWSTuqqqfBLCraZPS3TdBryV8jMc\\\\\\\/2ksZ4KhaQPD4FajkCyGegQzM5P2IINnEkhvYj8FuZiJlxq\\\\\\\/KY1nvPCQWfJRhl9yP4Y\\\\\\\/MD267zS1uGTG5KGMigFrSZH2AmS8bBw6eu5tThHBK7mUC6r8H0wvGcS+ZW2Rcsyqoq8OTNGRseCdv1kb+Xb5zkMJfQinUTwwxQhw9YAXbYVJfb1XFTfHsAZqznJ7wVcVK6w5blhqDUm1aIRUWG5ZwxITTs192SgXfub6vhDnWZy5f4cqoVhJ3S\\\\\\\/teHiQ5DzdLOmLtpcuoj+1YjLUjXT+IfG2kp5N\\\\\\\/CSS7bSrLWh\\\\\\\/FcSAzOakLbIAOEJt5912CVVXwtxtAkGNZxYO9iMeB9BzADzbS55+GJEah0GS7DVIGznplZyBhz5pXRfCok24UAL4F7Z4d2RSB7+BNGYT8lV7m7o2dFfSl78frmTUi0304HBKJdN1JTHHdKuijnymGF\\\\\\\/FY1C59wo6VlkbG9Qz\\\\\\\/uAPfAGJgTE1VJmiby0z5lB84iOR5NSaLZVlCwWFFGXCgKUeaDrURNyKMAPfpXGAmw3\\\\\\\/oS7C\\\\\\\/TAagnFI6gxO7SoO8wHx\\\\\\\/Ikr0S3M9KZO9Z3Tiz+proouVmyFBAMZ158Y8XABpBorE0MK2I9p6ca\\\\\\\/HCgHN5i8s1i\\\\\\\/iFcEDZyJ0CJgBPU3b0pF481B9cH1GkEtHtrRZBsBj4pQe8K8vHTsOmZjus4J3yh+VBKbVZukdF2P2Nd00HKdP2gJFajT7MNGkGey61traTcK425PJM6KX7FUYXHwmd52uivluJCRI11cf7Y\\\\\\\/31PlOt3U00hRaRBiUr6pj\\\\\\\/F3uq5C70MJM6Tjt4AICWY1n27QTVjcGVvxLP+O+RzbefmwYaxF4Bxe2THODDcdcdL5v8IoSRpi5oy+motkviXiSy7TKpJFx2cfRnKUVoEWipjEPkvgzxptKKREwk0MoKHf8\\\\\\\/VnQMG7YARwr4rqjBCBQPV8ZOhvDyM+xo1TABliHD6kndWvzZpCJBN1JgmZeoKKJ8tangZhUvdUyGydYzHN18n\\\\\\\/bdzFS8JOBDPGMq1QbYmFX\\\\\\\/yNXWVHTg+62NHoEIlpM4cn6SEKgzT3ahcvV9GgayxRigGKpWieGVwb8CHSVRvcgtmvuFQtPxfI39gJ2z3YzB+23FO+kv6e8W9o\\\\\\\/6HB6ecxt69PE4VhjH3f9UKlyuchvbkE6QmWGod5iwuWn46bO6PwfgqfRa\\\\\\\/8qQ73vtiX4Sf58ayT0xYtX\\\\\\\/OjUx+QzXT6+LL+tGc8qmpVbfAROnITyqqFGuRDJLk\\\\\\\/FDB1kjPU\\\\\\\/5LPtuNivdogQeUA1olH4yBptZT9eoLFePR+up7VWCDBQQ7P4zDJAOkhx\\\\\\\/Te9hKu9zILrpN2Nbdt3kCWluZGNLOhNKgfhFOmiGKWFIC2+rAEMTm+5fPgN911wO0D3naKTvOu\\\\\\\/F22fPxbYc834t\\\\\\\/0\\\\\\\/cjDjYphG4EXFpUVpY+HfINmW3hz4aHkpH90+wdpi+OL+OB7GtnHO0cu2bN6hT62CxlsMqiczzNpPVUlqOe3k2\\\\\\\/nZyOzO4MEwHp0oqDEXuer4GJFAkHj7kkaSfeXR8PEeSFxc78BeGaF34W9hen2RfxTu+iQx2z\\\\\\\/hMA1WyGBzIo74c2nAdATcm+2IRV+ggnzncgexNG+S+GAYuXUZrNsxQIYfJZcQlOQx\\\\\\\/L7XuOFyn9ipiDv2MmSC2NgIwbdGhSjB0voYvG1N4DwDzWfYMUsmH7f9NHg0jnKneNO7E\\\\\\\/Yp\\\\\\\/CNiN99n8PcuvNM8QW\\\\\\\/fHSu1ODftfPZelHIecYZZkntQ9BcMCWet5f8o+r1ojYPbGp55kF3jrDEd5vFrLunfgeb3VyiBjprWki9ra7ywEpgWUWjQcVdglo32Ch2hQipZQtAUG\\\\\\\/s2ZtOX+6KLLYDzwRuWrio9FYkezOGuZH\\\\\\\/ZZmhpq66vyCnVg1v\\\\\\\/5p6i\\\\\\\/M7ZIq4sEkod5uOP7I2gB40T1R+69mKS5PSMJfAbEgJtE4h3ZxV+ZDawIoXlGnEQAiSZT6PDlT2853OFvgE3YXrcNIzCGjv1K4WY4rSx3Owzxzhoa4VudicMBTRZ+9K+1iqlylJQXCoBBhfVCZY3YkOsduhT7Jx+D7ELJVEzl9S\\\\\\\/J6wzvuWUPjMIjU3I4+ZPf1aEfiBhf0u4Dk22mwJ11smJ4zPuqOMaVCWYZGSnNy2KgTKkDKRJJVxFZFm8vms7MV2AnG816lzy7EBPN66Lw2m6GCI4rufamk52oUY29iCdNkmIIcPnAHLdh6lZdE9KdpJdcza6AfpP7qTWWlMu9SnysODeYzED3VKdCWg+L7lAMj7GWMocK66TBSZ90+v84DfkXGankS0pB7mwHrXzuKmJooP6bqm9um+QKbNaN1JZWpA1B8d4NY2eFyiF\\\\\\\/wmCRAXOx9HFlVpJm6luT1nKYTJD5pRTPmZ5HgLQsxA1g62bSZ2P5d\\\\\\\/PzjQBAOJltPC2kCiYZHNFsgLCLvTh6PUdguLK9NJJvSxd+yJJ9qaLTZaZh9x3y1BE799prYbTXVBNz9gdvn+xoksEmd\\\\\\\/f6NTYWREt8hZHGU\\\\\\\/nUQTcUGbs4TouTrIIUTob\\\\\\\/BU\\\\\\\/5OuL5GZD4Gt+GBRSlw1UOUsvPVHrYwg7HFTBKn+t1oG+\\\\\\\/YcAWtmMFvvbnddz1tMSHsxxT\\\\\\\/SQhtJ+0JIoo++pOcwSxzoec39hp+Obnh5PYETMkLp\\\\\\\/l\\\\\\\/g54WMd+703dMiIEG3g4u86tr4\\\\\\\/L9zk3ryG+YMUqRO6oLq27qRu1LwrsvogENhXRTH6j8tnqof6RzcECHXZv2sC3WKjX5iqFtlDPJ4RY0Kn83YT9+zPFIe6FH+8tbDsJoXFG9Ao7F2QCROcPiKQBMBCY15wMPI4f\\\\\\\/7HNkLgqnAHzy5fYUAozyHJjr+kyomlHQr2g\\\\\\\/oqUfghOaoNx7n6SmCRh88B+U1ALMGvPelPuAw2FVdjXi3Rvz5+F9LW+v\\\\\\\/bSrHMLg8VtS8xMxsy9rWfeZgZRqentrFPboHiR2+OCceK9UWwr6nDF9nSO+p7Z91+KR0ssfqBa52glH278FXcOd6cHlXutpZ\\\\\\\/jRRPk3a0Bwxl7W40J4nVjo+oRBqE7T\\\\\\\/d8dR1CO5TgEuKoBrRrDJ1qmQLFY3BLwzeSNcPDCaM+lqw9O4mzdNWROxCnwrwno\\\\\\\/\\\\\\\/GVNeHVEDflXzCzPEBRd7mDri\\\\\\\/ymQDgyEvwEvOkzjflm02ipHuFHxhNPX7WQeDDLimB9kNDsAjGMrWfPahhSMWsbbuysqLvh73rtPbZMj06DMMo3hQh1hH+CbJbKrpi2\\\\\\\/W\\\\\\\/6chQV9UHFW4edNc5dNJ61+APOF+vWbu5NBx8bZ4Wd9ZDYTQIVrOgS91pkh+bKPu6utCbMkHlPEXZd0MUPWJHg1TTdBrbWoF\\\\\\\/f7W+n\\\\\\\/yWUnwXJVfEcmI4KA6o\\\\\\\/zhTTR4TeeZY+4ngeMQ5P7Z0j4m\\\\\\\/OgluGSyYCp2tYZxA\\\\\\\/JbPK1JqV65\\\\\\\/llF1V48AlUzpZYi5J0mBXUBXOsakwco0I0QTW4pmAKt7ZqvJngVlw\\\\\\\/7y2Dobl1ztJM8oKY24\\\\\\\/ybEhQdqExTYLo5GCUadWrznFL7WH+9BXLtkMvwD6m0uSMq1MNyyhhG9R6bNChy0xayUbYJy9tD++lXBo6kbirdFLA+Q341+p3jSPcsqrXsulr7a4W89dOjjM0Wy1vvyAL1W4ognIUBtIwEfbSEqKj09KngRb1knr5QSSw+EDLZONIX22vjs5NBo3hHnsf5t6yHTvE+SevwwCMBuvxuEUqeNuuEpSVEJlJRgwn63dG8HCxVkh2Zl2ZwY88iEH9NFwpPeEahlLd5WIHOmmZDm4Wiql5ihN1yqVVGBiOTrdDgk3HJjqzgiuBMxH3NgXmWcU+lRPJzIWKZCxL26lCCPmhw04DNrhlub\\\\\\\/kel9X3Xr6m+UfubdET54f5v8z+ARfozQW3+yEiubKY08SX8sQBm2IC9\\\\\\\/DzZhslvu2anPuomtd1SyZJb7lS74y9ue3rfz6GL22dWOqKLu22QFFqEuPMt4ioDbqUnlE+o\\\\\\\/FNJMWvL2PCJjz8+G3ciFH5tYd6ezE++fG6saTNc9yKQ9MOKQzcyX9MZPhb1wONy3hYpeLEEYKCo0+lTal1VrpsVS39S+5N22Snvw2bpXS8SIUhce91tcT4UL\\\\\\\/EXc8kxqNYS1ZEGeZwJgcE1QI7HwOhcSLe8zJ6jnEY\\\\\\\/B3j9zdW3UMltWQvrCI4WSIFeWo8OWUjH0ze02+MkUrzQyUdBTusN4gkCRhHetssEeVZsNphhAlja+zRD6TB4XuSKEjacGk69c+MDUS9xOTYzQaPWAPDn3qZMREROXbPQDcExu3iitnaHgGoq6Bam5500P7CMxNzn8K94HRE1xStdzlEe7nLm6+95RQEduyreI6rWsTm00NhGIBs6Tmg\\\\\\\/vHyhiNeSkcAb0Vate9kdX6mZT7s55pUaaxXmDQ8eN4PzjmbYunBal0nxmxJPJHFQI7KBg3LIAWaw8U+A6cv3pb3WDlY8RQeITjusggu2\\\\\\\/1wspAdLLBF5b5GFAphAG92b9nG+0dSsHG0+UNuQT+AfwPW2mlDvIAxpm6CwY41WOMRmnX019GYxkfn\\\\\\\/Cg6OW2HOk8TR62AEcD\\\\\\\/tOUXCcZvQxPP8WxFYmhW6Oknv0OxJLWCjpDw1RVm9Vp1OUocPdfG6QFWk9jWFt4eZN52Dkf8bBPVDMqYW2Weu39c+GB43Mc+ieOSF0t1KFuHpe4BuCXlz+TRJA4aysOhkhIjl\\\\\\\/9F00iFe37xfiNsQFGJ+SH2YESpCmN90RCGO8JNo1o5EdrRKYrMUXy778XcfzslaS+ko31kyYJ8+wXjgBq7ypSiGZWpNTl41Ms35VG3tCsUnTxOBl8K5SRSuUjhIhEUpuX+dN9hk14CRvf8\\\\\\\/kGx+GAwaGwsc7x7AP2DnBTGR11hStTMVJkTTtE021DSj20y6btHYn4PM2toGCFoaoXt0VIaSoZwIPIgRXcdLSB8ac7l5cBg63tCo6KaFpi0KUtXO5igdjzNlkyP\\\\\\\/MmT37G5HclH4KDu1ttxLdMmbtx57nxxGWCJ2iD+SGe1fstZNjTZ3Ou1Dtb8Gk5f1HjUHcd2LVLQjg3dMGVPsXBm1t09aBUyWHmG28pZsgEDSzyYKdJ5X9efSmGFbfR3HcTZcuJKA90uD\\\\\\\/lS4EHt6smJjsNrNqBU+hh38gmaViQ4715ZrFIej92fOD+rYGtrETnuzdLRrnMS251P4JS8EzKVUdHV0XIDsP91J+ZKHU0+hC\\\\\\\/kw+EArnSgll93aAZsTIC7Jip15Vu0QayUCOtli8EaVsGJMYED3qbH612ZyvnkUwKqVXz7fIcg\\\\\\\/GkLVaVORMu5JWY+c7+k7kqcUIsaJZ2wIs0SN8KaXvyTdqWpiRd04c7khLdrF8uP1ee3iRsBTePFxAXZvTzfhg712hpRuSHcUYTiKCBht0crSSO\\\\\\\/2FX5C87wmNqje8MO\\\\\\\/yERuWPxNa7zQflt7+RXzwhjpjZ53OZqcl9Xl11VrtdiIhmbp7zcLz1g\\\\\\\/4lhQiA223A+6ozDWxA5zzUsFSS0f7kL9TkpRmYnX9GHlkBNn8T4sK\\\\\\\/hBzMHTgopop7ZHyDTWUWyFPu6X0Emp4UQLUwNfRdVlnr4gsuqicieI0YU9u6UjHv6mGMApyQpNri1od8qCZKCyg\\\\\\\/jP7IWlV5RifA3Hu5uzOxxW7324AHd5AkYBp8DPQxbz9tvP8SbwSN+7HgwnpH60tvZtNXZkW96IMSPnPnUUaz\\\\\\\/c5g0GtlS1UOVqXHO5S7QGhJJ\\\\\\\/eO2D89LQtLcrXCGmKpMEqH8O44PlqDZ2xbQKb7SM5n7F75Eegz+W3ahK5NWODtN9FOfHH+8USEm1m34sJBZAg62Rw3n857DW4y4j8pP9KkMbQtyEwycXX1EPH6BmwIYC256V2NmMIcFJlcbKvCfSPMCkxltVcSr3oRQKgoHmzVaLO3lWWuN7nmwd7eh0EDW8sk9HSx7yGxLbP2pk5SYNZ2Hym2\\\\\\\/J+GPcBBQkPKAIau3BJL9WlLapIR99iElVIHkQFDM6NItc1yVwENB8VYjwrdydYZibTGfXk9gyokVp1cw9KdZ8fIMPVVjimanWilLt1EtOUfUe14Ud4ONkKmmcqpq\\\\\\\/ZaZX2b9QtEFrWJ6iYAcwanVZ4pZP7nTZB3yd5MVg6NhECF5mWtjVpdWtwCR3o6cV26Uo+t\\\\\\\/aUBAGrhALL4J9XDIA67qfm9gNCqriXI1u1UuFEbvL3fmQE+YkTe0nhcfomzV6yyNMVxZ5PzVmPBRJrsIfMmNw2ns88yLyhhfZWa+xwvmocAhviF41GIhUS94F1Fx04iMfvlLseK3lRymJZnyU3AT5mXMFt\\\\\\\/7RoAE+NmgGCUNSWEGiopJ6F2pFEtykALEAYrPURCqS+Zg6zBYah0LR1Lm9ClIJOFBGGMTOdXajeHMlvarPEmdTWzJbnEDuR2XxwnM4Esfk9lP46tOiRakU1vQYadg8pFCpRjAcKPrRwKAleGfGABQRhG8QCwWhLcRuhyfzi4gA\\\\\\\/F21hb0549AR1KF\\\\\\\/XVQ72yIbi5tV8iznbXOLuXM6zRzp2Tr6kGwVq2OkaG7vzOgsqUXe\\\\\\\/+GnlhQ3x8oL0y1lmPhDIptQ2IVLMsJDr7mmuwLPbp\\\\\\\/eJG8mHHQ8jqCcAzEJR+D5glb4YUlJqT7de+NVbQjM7HfVzokvxYAdPre\\\\\\\/HGha\\\\\\\/houJbAnRanH87z403Sk3+geMcBa7meXiKQdP8TBWK9SPnrCI376swIY2AFfpDa1XBMYIqHcqNT2kqj82F0VFAuc+MUFpv0WRIACGrYuzQult7v\\\\\\\/8Yy3tpay4pg0SzkJNjrcQMENRTlBplQpi2sffdh2G3pyMMhvAxzMpwe\\\\\\\/3mSYgR4YnzfzjZscwAbtQp0r8TDgbyisKuLUBmiHbvaUj5z3oqIFznrvne3xQ+K5+wEWJS34UBrRq9fcDS2cnRxLWlaA6nvJit2QVbwDxwX\\\\\\\/3t8J62WInzkOxIPUlvPpCKgnx1Rfnqi6jIkh3usDJVS9LrxbIWLmiepvJFCUNvBKNFT9n9v4mbt5561u+mPupiMRruHmnUa\\\\\\\/Hb481b6FMng32\\\\\\\/tK24htgl1F9H9Qn9qFEE\\\\\\\/jVREUuqt+91xfk3SM+K8wSM2E+KgjHM7jmoIZ6WC3ype5cB8CIok0pOvg+4VKlPty\\\\\\\/3VexG5Fgitp4uBgNUMpDJYsVX2HarDR1pYngda\\\\\\\/AN8k0klQl0UcQjsMUgOxEZkY8LXv9ypGsPxBkgW8EZjx1MqWVL9+88nhREJcmsFwGfYPuIo1kxxh7ORCQedJNIvbqiYVZooeHKVPeGMq89hkP9237ye4YEPgrptwTx9L2JMTqZ1FtSaQ31NmyRyRKs\\\\\\\/FL0nmtCu1m2cmwrawH\\\\\\\/fKMc1Ct0kb4Gr+qrn\\\\\\\/4N1loyrrTgu542eJVyQc2i6pjYXQEd5OSS3zULt7XeWbCP3hr+3D0j+a6AM7d83hvRTsNWG32OdQThkMzxLzMee0MLJe8qGLI1MF\\\\\\\/uJs9UPyrtN6jbUTXiR29y4340f\\\\\\\/gEq4PJezCZzzrkv0kZs3ofYzXcnN6wuNf5lkk8iTCrq6ZRYJTXEBpFj6SJ5m5\\\\\\\/T9XfJ3p8LX+3f7iLNKq9dnD\\\\\\\/+jcGz1jPcXLw8bYrQhuUM0Lsx3p4PuGbTsVW0aFGdglNhoZaDXQ9sjcdwqH4vItsPk\\\\\\\/\\\\\\\/DD+gzeUHaszXwzOe7kSXD0lD3YxkaTcEoVNFMe2ficaI8KjQQUNmTpT2eMggUywMsEFmi9iq843\\\\\\\/aKrYRB0r1qKj53PYAyRkVPxDG9za7byvHCPDoViZJlKefvBUVAn1XLGmE4l2UwPSbDBeQ5E25sbvrkFNdFKMwuepjnvhKQ5wuIveL7VKeF0ARHV0vz4p6ptIbpTc8MXkghCThrh005IgocS30uGLbkYVWKi1BhPnHSKiW9z2iMX1D+SvhhnU5qJI7ioBz09OO1x+AToRkhySciqzfsnC1+UrzwAHe5C2m2iy1hScLAflQZAb6lTTkHCRchrGQ0\\\\\\\/D88N8YXa8wIQuSRrzfHsPASWbZ3O4tLVav0DEFOr8Yl2uUs1h83l09ego6FvPdYmTpQ5UO8nVulh4PTNnc0TLYRChedHkeCw6fR2YyVffhfAYsk4ceGOQtVEYSTyqNUh8Gm23xsW8KqheMQ+p9xabVrA4OgYK\\\\\\\/tfY0nuM2pVuwhjZa3\\\\\\\/f0G6K77OCFd+MlZHazEE+rg+uIsQZttCz\\\\\\\/UL8iqqBpvx3V61UlZBUX04DKdErW7yFPcmVORZCxAdZAx1tzrjDjbsPbaELihSITKbU\\\\\\\/wG2OK07+SqCxVWYJwa3Iz4uwQG7bIrr+v2Y4O9F\\\\\\\/DtUDUMqdKnL7WOdNla2fTJppcJm9kwbqRXkCDb6NHiGA7T43mJUeTYnFdcNElMZbn2YMtTLb8hmd1thiEwYBHt00Lgp4n5flEJ8brAEs8C9hhZkVADTDV\\\\\\\/lzA9izKI8YxpXl8qb1qHHfrS8dKAShSJ2D1i2CUQ1SKAtyTUoNEeJ7uEfxLuBu3zKGYk27\\\\\\\/XTrdbMSAs8CsvwD9OzCyY7pxnZKEsNP7F1ftBxJPdlIZjGzGcg5+J\\\\\\\/KnwTiBXzCXxzYtCdBtI4iNNej5+SOKT8w9S+DGSRL\\\\\\\/p8sJT2t9lJ2pCJ6HFk2+DGcD04eIwf1LjPFINw7Na\\\\\\\/jHSRNDLUUonjU+UCmByQLpdwn4qqZv+Ys803f59SWTS1P1id8at2eriuOvvpJc4GHfdh1uP5b\\\\\\\/7bE3MN0pWB\\\\\\\/Z8B17Cp+editmexYOYiwZswtgjvrNshD5ZCOS\\\\\\\/5SEj1WvxGgE8V44B2a4AuA5+6\\\\\\\/BA0nvwnVBZBmzjg0pSCJBytc1vKYzUZLIZnrdwEcAUnoUkVOtQDaehMIL2ItRP51\\\\\\\/1JtC9+ZhsR9\\\\\\\/VigGH2S7ldrbixjrsjdDGejKOUXnpTuDaoeapKT0O\\\\\\\/GHjX17CBzE8+DOeFPX\\\\\\\/9EHd0sezXYjANjoqnu5711EGmjcK4yYVNO7yfcyvwIibRA3Il4OOLknu4g2ysAngZJJbctwcXY51kUvrvTxNQN4D0KVmTIwm5TwDZUYa+DmTAtd5WMFT\\\\\\\/xuBobNclqXH1iyUl9MSl3nS7MSDcWyWnVVGXONybkOOsg5V8fMTimLNUSg2QOIG\\\\\\\/Ey0XmkA5LMnAOrXXPfJkzaajF2IOAPsP6wzKTBjUC\\\\\\\/RkllIm6EF5Tqxr2IxTgB8I+W4+zGAmeil6Bnmr8k90RtBq4ZzHtvi9C\\\\\\\/B9WCvfSZo\\\\\\\/SPKOhfhJVIiI4rHccd3qQdFgS8ygMLXDiApBf\\\\\\\/pOF9bIK6VbmHDllahAQU\\\\\\\/14RHqaDiGJtvSNeJbjMB+TDWquXaLhrztlvbqcoJVXAeVmhB4zafLXGGMn2ZJWt58Rk+gYx+vLfbP5cW1y5kUpVj6SEV1eIJ+V7wjbw2zjo07mneME+ap\\\\\\\/YkaKQWJ1Ecla7p1Xb5y2h\\\\\\\/kJYrewMYj\\\\\\\/7KpmMheKBk8uAHU2kNrtht\\\\\\\/g5S2Yy4sXMuhyDuANXzzB2VikkOXfpksIwlR6E3bVwfLatcAU\\\\\\\/0zf3oq3LmYi6zCHBoETQc4jOl6EaMy\\\\\\\/XA\\\\\\\/Twufj\\\\\\\/hfU6c6KSkWRQhG2U9GmaoUOuNcPMazWC+oERtwG++XewBizW8K0uTsqXK8E9uRf3PD5E1MSOMTXwtu5BxWPzdOBB3+qHcAEegUJsylCZFFlEZtTjdkqmJLMhzK3YIAkOUEAvmlRQ0kWOrkknzo2pld3B\\\\\\\/dcKqG76dt5C6maNUwg4GRpqZnJQP2dt1Ds5rrPBqkciE6VeGdMZ+U6L9aohj843j+w0PFD0J6k5qX5XZ3yTWeLMq6a45f0Vl4avFPC9UU8XS6bxUCBhvRE0NpL9XT1Nfu3NONdLuEsb+DOJs4kT+eEhxWMfLM1adCPDeti\\\\\\\/Liln7zwhEbZ233\\\\\\\/W\\\\\\\/qyn2ov4s1eFVm6N\\\\\\\/jQQzDr3GHmZ1OG7FGYxSxTcZbjGyGzFfq7PXdCRxsJ47fX74hTc8ut9gIM4q8iV3N2q2cdw\\\\\\\/pzmNeJcEVEr4fFn14YFNHUchaYJ9XPUb5v4ozwqPG6MXbhi6hjJeyc1mvDAOlZp7m5auhWhSteIfAixz4g8i+co076ozsnyRsAQ7Dz7ORzPmAgERHw9\\\\\\\/u0G2Uor4ircPOtaKefKGRss9YFaL9+1mq+F0TE76F1TAosTC6lTsUWEDrlIv0IHAnUAjaHYCO\\\\\\\/Ym\\\\\\\/uuG9xKKM32B9V10t0+6cPjeLYha75bY+JTTBXsxToq0Xf2yQTFAhC24VQVaXdh+kH7Zzu8V7hbVmthuCgCjnuAp986sZQ6gNTmYNwqzKZYKS+p9sohDcI+p1XUn\\\\\\\/Y5VHALSN6WhP7Eg9JyVBmizT5D5yPTCn6cxOytn9OLCOKZTYDghDoGvO0lSF6csgmlg\\\\\\\/4q+o+dyPy1BKxg8Ta5PNNNDcqQUqRRv+w3Ff7IPvXN+eTeAZelSkoe5YyxWiA8T2uGHez4knEcPcwrEsvqFhK1aR7KySbbHGBQgyO4MIKFJsysDxcqGlrPDokSDwp1NMnGhXGomw5cqwoeF8fbDitTwfiqzI1XgONHrWIgB7xmBoWtmtOe51uzop0ZQJ4QglbIDY+VBImX3bT3rqaLXhkZM5Z0F7S8fztJ+NmyfZRCWSw3Z7suoTTThkE3m3FEkMc1agzsuZANn8urGc1kAjPw2jcewtzzbgPUNksHIN7yQmqMCOMpo1lP2mHKQcWLgN6TogRXYOL5HDMY1dq2Amffv8Xtgw7+vfKmg3kDUOF7BBzskYq19y7ht9XBPeY1cv7ov9nbcqynxgvnrgGcjr9wzZ7p9JNZgFZx7Tam3qctlYSkQ5CnjTViFfFuK3llDbDzYR9AJsB+usyJalOi1v0rHt93U6jP0qsHYhJR6Zhx13naeapp+JIc7tlOnHcL44bqHShhK4J7VPAPE4lNtUtIs5VjLA1M3Ds+5KqaDg7XlX73yBfacqGfGjf7K0\\\\\\\/1pR9bI1gWJn+OjecYh7JhJB5RMzMSpGqsXVUuKp\\\\\\\/HhpdH\\\\\\\/8XtqFd3Hnk2\\\\\\\/J85z0KdkVcagZs8wyJ6FIQ3o88RdSMuxvmgIsHP0tZ7vE3Xsd\\\\\\\/970HOK+Ch5hdkEqIqsMEqoZFbv1tNrJtOXVgGn7BZhwD9J64RF8ilN4q+8CJ2CO3lQ4Mx6rsBJ5+N1ub4SU595kaZMzWQLjHJh+HPFSJkDyrWjx9B2pYmVQRgVsxQl3Y1lUK4TGPDMAfKNqNMReIoIIZziib8DfTH7lLTDBu5fdNXe1btKvwHf5YcksBjHIrENf59B7hCWvtEhfLxR3SvXA08TIeFQTGB07V1TLixjXmiBFGn7uU8QVpj5CYtbd573gx8wGeL1D4GpoRUgZZI7FMO3uCHf78\\\\\\\/bAqcwL6ySXMz96gttbRtw5XdczFg8QYj0vNqbo6THKZqLlnQkn4zXmWIm\\\\\\\/XxbclwMSeGgBGAQYItmZm1WQmIISICNUNQ6iC0DJabIY3r2xr8L3f++fSoX5xkj8GYwqmIo\\\\\\\/N2H5jxO2ygpa1ytgLSpdrWngHgc0Gf1EYKyu14PNVihZNjK9O1wBRvFl8SICaom0pixzwMuiOxPKwfMrgMtFHfPXUfBP0Nzq4xPDFbxwoxLdkp7cUXJjqfD0IJl4R4gF5SZvvJEZ0NAPnBKD0XkyjvZVOUQXvsuEwVlLuWIqDmr+0B\\\\\\\/C6Sq9IgNA+wxjB9zGKYXNUctoKwbKbQhkRcGWq6OBnpSgtzYZvEUEugpe+QLxLhNZKDakTr3zmPT4veeHPN0\\\\\\\/3fFf4AHm0pbHhmWNLnuR2kAN\\\\\\\/qS8PEGsyX1mrsRHgYircMZ8soqPJX35AOkYPhBx+Q8aUewmzXEqAseZseSnvWkH8MdQszjynHu\\\\\\\/eoS83mwyRJjPBAPBR32UqTdaxuaGVPEjOdY8SG6F2EG+uNPNeLmFnkM74xpOC\\\\\\\/ITFbyAUmn3tpUKoiuWxr441ZYTvje4IB9dycA93X7ifF6xZ91zpqwS6Oh9ld2+3mtMqCmAAOiu9rId8U1WH\\\\\\\/5QJvtFk3Rmugmv4sR+T6ohdYQx5LCb62ibJfWg+m4C8sMKhAHbRnxHZjCM4H9gcM+gCRyGwSmChRq4koOul33qk5ujUnlD6LxX6vM1J2WMkKqQy3PEONGfTrPf44Tg2MdVl5j1aD7OgAQ2jMIJZ0fAMPySAYZBsG0idraX+KC7Rsw2hLPBMzAMs8RlFjd9KMZa1DDqi1u5dnW4S5Wns1NtVVF8+5mnqEiXDQYSguQhhEgHUGmE9zXwvI+s9x3tVEQRYt6EkX9aKvdyAaXVsFOjp4Eebem3uNojKETGWBl0hEJQzthQDcOlpi160AzGwa3RaGIGw28Pt0h6HSv0PO2eYb6b0H2Da4zz6hFzVyBzXB3C+TMB1djMFJsmLoVnGDpH5Fp+T32huzV0q\\\\\\\/\\\\\\\/NkLFjbZd5hm0w47JUIogvnWEhmHeJdnZR3Eio3orXqRfUjPChqhvjoWp7RF\\\\\\\/uuRUyEjTw9HU+5HfZrutoSBqj0fxsnENYCXJ3FOLkx3\\\\\\\/itnpMMj1v7X+F\\\\\\\/7wOW7EnvmTtknyLuwP5j5YJMUDtCCnhEKFkG79k\\\\\\\/w8xnTUXTqOmZ1\\\\\\\/LCz5Z8I6CQQu9MsosYvfl8lSrfvNri01dHLW95uWTF7DC5h+3T9kF7Z9IQkg8FI5YO0sBqG4JnUcp4Rnt8fq18MCzPB5ksoQibzN9intfclSAYfhjJwrKWNFAafAQWdB4zH1\\\\\\\/XQLZrvmItYsZmHvJ824ESZSN2Jxl2gnXuHOWPi3\\\\\\\/YF2LYO08uIQjNDhICuFdRoZlftaiwziljuLCk\\\\\\\/\\\\\\\/pcFpBdHWPJ708MSdRKKjPuC3Obtt5Uj2\\\\\\\/lkADjqwYTNoYFQ4I2OnyZ1Hoo82txhMP2DCt5vGnIFY9AuWnw4zlOQqJUZIa8\\\\\\\/uOuC\\\\\\\/7xFxwyJz6P1+a5UQzW1fF\\\\\\\/riScKP0P2o9QTnsZ309ZoRjHnfFeg38IudwhP+Ts9QQnLSUKtjHlz2Hpn4G7fzqU79+LJlj6UBqVdSKv\\\\\\\/vJoKHnsELTefe06OIKJdora9pca2HjjmNynq4po5me+Bf3ITTl9\\\\\\\/mAuFDlvkYPCNde1GXnmA2uLDb24tUwfJ04EpKucVlz5TN\\\\\\\/g7wYVK7bwxZkOOZhRosM0zxNM2B71E3enONueWkmlQ34jpawzv5Aog2YpDrsjraG1Prm8HbPRRjaAAbC90sk1dYaqE3gn8FcwhKEh7YvqTVvX1qLVGiDkBgQvfR6XT6+Nu1PuXQyDn\\\\\\\/F9EhsJM3wyWp5VIOgp8BT\\\\\\\/tDvjck54YPmnqmQlCMqp4w+jyT5L3PzwPBmaByuKTEmfkn7qSmmjsuZpGSPWiFICkU\\\\\\\/eKt5Gh5D7mEbVgesznmahI2TmBZU6CXwWoU\\\\\\\/tRjJarJo1UFO3NYJxFjrAVKZ5ne3HbqPEQPPtIiZ07AEExN7Obm4xt7ILSUwUx3FRgbYV97R3jGnzVcjtpe\\\\\\\/k+35T6lUfRFTMpXkANcN8OyZLC+6y\\\\\\\/gVHIEjkCK3AdKZLYA0F8IObNralSE\\\\\\\/Cjf+V22zbY+MdfTI6DY5aGIa+CeFfsF5NBSPnwdI4Uz54WCirfW0DQPIy4ydSNFsxwwa1dOSlPOCmBtrpLshHFAm6yxOCL9ZRcSnjtDtQyyCi9fiCdj5n3AoxpvbpgojfdPaFGeq083qd1BH7JUJqqzg5a\\\\\\\/NRciVMI0AUGn6fQK6OCIBY+cIQkzO0k2B2qP0a1eLOQ05sx465AG51058GNH0kJdTuzw8QNfths8iTqzF9aj4\\\\\\\/f8CHS434qCTRUztt+2qnVqj1xqY7iYtgl\\\\\\\/SDpR3ZwmaFgzm7zVGfQ50H6rQjP5e6ps\\\\\\\/AraaKNafvUcLqi4AY2vFCek09Z7V4PPr1i+2UA+BL+ffzc93e0+hhfEd60TyOHYDmx7mx3wXiWDstHxcUd14XFt49WnXgts1yB+GFPFaPFoyPW+EMdJhjoRBwdPBLtfBB3mM\\\\\\\/WkQMOL3G9JJhMSaW+RCxccJh1WO4qu0T9boZD4inHVMPbpWX2qmDgG3hUGXkIuwLz51O8vJZLrv\\\\\\\/guWdx6obLHJIwmutKsfqiLp6IYA0xx1CcIbJhBBxCMRDuZXpgfrRx1cfhfBAKPhQdY+YDt8a2mzkL9wKOBWeSwrEJ2qFNaO9\\\\\\\/x5lIKS+LJMCnKXC7WX417IgLIpI0X03uighQQUqtmb5FQ+3TU8FLfPj48hBCvxw\\\\\\\/oacwFC0UQOUEW\\\\\\\/SxNBXp5RDPRNi4SR9wlky4dHPBmnlxIrX3u50PJiJ3zRXW3r5g\\\\\\\/8jldBN7DtfZJvUhBY1y3gx3vn+go3\\\\\\\/U28QsVU7bIytjHSyKTMjOu7SQ3SEZjIWABWHpPq7OBzM4L3gL1FQn7GpUKKi+aDZB2O01figz+vA8pTvwYhb3ENqy6A5LfBKDj6Zi6y7TFjpAF6YDAHn1ef2D4ZCB3qLCpILA8Eztwed80mWO2hACUu8Oe0o4pMNwGtL9KLTAyXjrsSd9DGyGumi02Hb1wfAwdXwxml+uEHZ2I8Q4icgBsW7+NByLVM2rzPWiY3NoeYl+YYvTwSgb2+rSUGXA0qs4+Tpb7MKYqjt4Pq6rl9ZcOQ4FugKsLqjQGgxbVPoRFOV+\\\\\\\/8YXPJ4shTm\\\\\\\/I\\\\\\\/bZJZSEPXARV4HnZIjl34Z1krFxrogbWPRlMpHX5Oy7W8DEMDMAWtj8wZYzNl1lwPWuy97efamxyRzR\\\\\\\/g\\\\\\\/6mywNhjtXNmauNoxy70Rj3pMdnH+q6e8d7ZKMw3vWynsa8zGYcUFzp73KCnjPkEuqXhnELvFVfGgWkqeCi6Na7n8EuUcByBdjTxB\\\\\\\/wBR1FPrU8\\\\\\\/ebCUwrmmGTWPUfI1T0ITMqMoJnKWdxcfLze\\\\\\\/FU0UUyc4qfv2AP\\\\\\\/jvD42qsDZu9a8yxGCwQcU4\\\\\\\/ccN4E960QIi1BVcOJGDW7hZy8Iw3YTodumwz81XBkeGmTAMHfSVegO9fx0yk642BOcifilROxKSni\\\\\\\/IvmxeRM7XnU\\\\\\\/O9f6usWe97BvgQH3zUOyocIDVMEk9xD341f2x1FyX3Hy82VboZ7KkI0KFxvFanj0N5K1YsOPktDaYCD35n1YcgadtLGv+RnnwpPxQgfSDCC6+YEXcxLrMYKBEwcHFPzejyjOa\\\\\\\/kubij8jhGbXSsewhLtxh0y85CRzQwE+u+6JPtfBpC8FW0ukd7fHOK7Yw1fiSpaSTwO6q6puZJcYt9iUvN9XfzEtkpFnSLIOims+50P\\\\\\\/A3w3N16voGuVfE4jSvsGiYibGNWV+\\\\\\\/mEaVT60pGXwQfkgvwpJzN81o1nF3a+EMdxrKjILY+nyltxODcjSRr3sZ2NuUWHh4zfSbdcwKyAyzaCv4RRHJeA07ic3BUu7Xt6wI3jjPHdVzQ2Vj2lcFyIha5CPL5X9+dh3j2tcwi9hkb8hklUEmOuEHyyIIgN2EKuVHZKaI2oB7amgqKClw\\\\\\\/Onh8MRbRoU5csJJ56Dksfp7tpNBujbMsXvdbRDU+5wDtHsxN\\\\\\\/dZQ5XjBtrTdMSMGXxxJFxCZGMUJex0auW3E4kLBMn\\\\\\\/OhnWc3ou9iJR9jayJcBPydEf6\\\\\\\/FbcuQoP0jYYYYXAYI0lZFgnacKg++AHGP2VCJ+3Oz9ZMLmttkl7VnStiLpAcNqu973HoPPC0njv4Tofh7EHRRIV8VnlHjKjUsIALR9pezirCqWa6n+FbnMq+kuiBVRJvxLMTZ1bF8qh2IzV2XOEL7dAzgYQBg37bIcysWsVhQRG2wl30kl+3x\\\\\\\/FueDX8WKI2nKVaaTiUQvP\\\\\\\/PuO1\\\\\\\/dZ8IJ9eIfEJX5LY3O1sE+WzeJbcB4vghm3TrQHzbI578E8Sg+GLJ12EsplQyINUdoaq9XGtXgP24VMvrOluJ44JUkl4w8Ap8mphB8TCoHWI\\\\\\\/zU\\\\\\\/PLvFhtcnYclRNc15w\\\\\\\/JM\\\\\\\/ukJ8bti+rrsrEb3W\\\\\\\/4qP7he\\\\\\\/oPfb1VeusNUo1NFEYssNvy\\\\\\\/vvs7EBHkc+3sTrDjEdQ5F5esKZioYY9iSr3uHZbsh3+GMeDaTxhcXwvuOa7GR7\\\\\\\/GIK9qJng\\\\\\\/MkZjojpOrK+m7BFNfyJCsRk1nxF2DieEj0AkQbMXKruq3sh3GKZgaHHKz7DyjBU+Jmgq+5iLwhfN2Tl0DsdzxRlRxoveTlQRBYv+H4v2ytGBotDW5GjTfig1hNzgB6cLlinx0n1hbWeOrEAXwXRRktV+SAtJCz54q8urTHLCPOhSS1w012EVbOYRoKd7IZz1TApFeJZgZ\\\\\\\/LMi02KT+M6W+NkAy8KNzLWLxOnsrGlMii2HDyPddYM9Frak4zMk\\\\\\\/+cDfCoUnq4zcvQvZjVIiQcnVozG\\\\\\\/7nSnh5y4pr5AoGqt7dRuebuJEYW43JXNSs\\\\\\\/PZM6stGsb7FnGtkiQJx7Gy8h2jzP5wYs6IPHipj5SvZ0stCcBhIt7tslvyrKJ\\\\\\\/5kVLwN7RJpfRyc6JM0neW95o05Zdi3NCyR4rXkpfSYA2dSZiC0se8mAJIhGZYGkCk6an3exvVPjdxbmVCz3ZzlneBvLXvmTAaae7Hdpg+yus5VVWfPhfioaddkXAvZUoqYo7erZc2YP++vQB4yu3EvxMKYxHo0wwA5nr\\\\\\\/VPVYb9JKJFrVONOwoC3\\\\\\\/fpyMWX3CVn+ys0WV3MDl53eZYxa9LVoWrqSs0Fkg2tVwy5mzL\\\\\\\/Vi5BGeYzAxLlq\\\\\\\/utm10jGLAOv95r4cHeVOVF7PihMdoq4IHtZfURmH7iGj4kKH1GiX9QSnH7Hp6Mgi50E1EckLM9vmW\\\\\\\/mM7hF1NhCZrDnD0mZWDtLrti6FkmPJ6NOdQuTDa+wMgdGNubxg0kWJrKXaRXunwLnD4m2pcTTQLKZ4DhlqYStNkkFHEHZjw2kctuiiiR5BuTCSOrDbkOMo6Lyc8sSF6icy129HbB86XfXelRunvSh59ipM0EM8mc\\\\\\\/44\\\\\\\/JTuer6UHfYZXxvizi3zI4kK6Xq9CCdwnuaGkYCvSbkcnMAoSQ4Os1dcA+oI7Ho8iI\\\\\\\/31OZfYgq0gMQbPxb4QCn5uG6ba2F7TrrKW17bM1q05WmFdrsVQS618EvJA4U6xevvASj5siG1tXe5otLIaBRuFiHhu8nGllOi2Nzx2Z+QV2ZhkfZWZEBK7\\\\\\\/LMySYvUdabfjkAvAZA1CVkT+oIhZ8531KsXUx5TWp3fZ+zWm6pS3a6auyruoA8wvW86AvgAEIgJE3V+dwVMl0VHAflkWXNH8zk0XVDgmOI9ODvlUn5ke\\\\\\\/tKVobaC3t9Qxit3cnVsevVmqWgOiTM5yvcs8Dv89+7HOyKpDDJ01MtRBSdKX5D1vorbvkSR+d+ZCzFGMfchk413AwJF8rmez9oL8uTNBkJ\\\\\\\/kkvNPUN9tPX0MU9A5DcV\\\\\\\/HmvxrTaVynLtSvTQ9ZD0Cmjpmf5NSmQjMvY3RZnhmtkxA4QJ1ANQX1+CWq4cOC8QcoX\\\\\\\/kEukNs2P8y7YdelIhIMAZk7NQYGVXFGQz61fo81L+2Pvr36iSXCONktG5yd0Lpj7g6Cp5Pxq76sPQDl6sc3lt2OeY5LZ1+U6uVBlWhCXVhZBdw78N1vyIlEMFXxBzdSFBlkcA8OWSrTdL84kTADj3nyY\\\\\\\/AAveIzgvbHZ3do6Yi8xPlbl+q93qOaD\\\\\\\/RhtcjRMdfFvhuQVgAxXe6udkQCFUKJJrwkj290HnXw2TSDdvOYJQCgZR0kAhqhSBBvey3tl2PahPXwRL4Ks3oZvcT43F9VM\\\\\\\/CnlRjVeeSZBc9MzIAJ\\\\\\\/vg5mo6UWK7l46OXRjSU0Tx7T+c+PxPmnXqNGi2JC9zkGAwrNIsJFdEatNpqRaaVrGYkrat+cx7rwrZgExPhZqxiXbmvvJMC7Z\\\\\\\/aq3iUSSjl84RqWLtYWLkIUXmk1SLpeps5Itobt6f8FTNYvITPW7N6TCzpzx3DNdJ4z1ylB\\\\\\\/bKE8hLaFenvXxeg1mzx72GoPeh\\\\\\\/M5IelmT59yPdoC7v2ELtJCERzH301O\\\\\\\/GYplcK07QP\\\\\\\/kNAQjwNDAJl2yP5pYk9FJeO1Fjy+qqB+Zty5VJEqdsZxSz8gYaOPQuQwL5Aome\\\\\\\/DENzqt3Ma5sRT8Rzw4Rn\\\\\\\/B46Uz\\\\\\\/GlBjyJYM5NKs4vH2r5bUrZdqWDloDkVORtJHNn2lMFC3G3MzCbLxmikdIlXr553qcjh3Odcww6JLQS+R1bb+eQfBVobmfbUm6b2d4spmM9ea2Hs47iJVwf+kNgO6xd8CPF9W8s9Q8FwL2+yabI4xqWZl6XURIqHj3GL2nA1xR4kbWTj+SPoqUkvCThd96BdntU9fIAXohbSs0NbousnW4jds+0ywiSXZJAJonkQbCVe0K4BmlThAyWo6J8VxR3eI7e+lwmI09HnQ9XxCXUQdJKNFycGAm6T69D\\\\\\\/GkhnPvSbo9f8rhNx6aSUvN6TtqykZMIrNLXuG+I67PqTUGNCmfLEVr\\\\\\\/4rcAp7\\\\\\\/hO1TQthTq5sn2bl0Ics+SK+1R8VSas0hsmA8MiKflyGl7s3heOhCVZexh355eshyAVK1b5bapLR7X+Z69t2hXxCScPBfsVaqidNzhEM\\\\\\\/Sev29sqit293zbZQSmssekbxAJCBVeXOIuHNP8hv1zwhQXgy\\\\\\\/Tg\\\\\\\/JLveTqKAvSgVKGvB1UN9xQTMIYrVYBMlUyk7tpuw2odn5eURX193Llzk6PMKibDKScEbC69AceikQdhtAtRCU\\\\\\\/Neqf81SC2o94Zr2cp+Z7PYZOvm3PtJatKpqi9x5ZJIv44Qvy9mN5m\\\\\\\/tfEKOHD5oieOuAFkkuNLNpJTj6AOl\\\\\\\/gMGyk8IMG\\\\\\\/cPpf2\\\\\\\/jCqp7+w7wHXywlXS4lk7OseV1JFTYmJwj1UjnrLngY7m9my2nREKe50cfVVnI25th2mY7E6B6g8dBDE2RxlPdDRYa0GzEm7CjvWfGzU3igsq9woKdUhpbE9AnfS1VBF7Id\\\\\\\/gjMwhNdjFSZE0gWvqvsTwhe\\\\\\\/JvN3pfwydi8NuJq8Hpp19Ym48TeRzwmvJVs3Xzi9I3k+z2VjFzWrHE3Jzp0H+hJoLk9mAw3IN0aR2nC143sMJrJXjayDsuzLFTdl+hnZyntDmJwpOGYCh\\\\\\\/8eR1kvnzwMuBiprEGaTJ5OyVDVwfldONHVe+CF5n2V6Jt\\\\\\\/pJD\\\\\\\/WRL5XFtze72UeqNs6ao7f3c9EiXs8PAigR1LaC+SA+To50h\\\\\\\/9W1RxCCLM0\\\\\\\/jIEoIUzo1OA+hZCAwHNSMgn+sCuKmv5KxFozFNRXNU3ks0WnmIn2VPMggD+1RTI60SWPpz+fR52Cc+ukTeGi9lQiHzIloLYO0i9Tp5kc7bMCRITJoh8AmdHcCwNUAfgr1wAjlZMlDFPCpyOkgMVwvmZQFvXLUodcbHB+xexdItTqICqpDXuRJRPqGCLkamdpl+UZ\\\\\\\/qgOt1\\\\\\\/yRusjVy0Wus0nTkZHl\\\\\\\/hLXkPeL2f88kc628VsdxSb6EvnqYEp7rnjamxjLRoqTlx40yJMKcClCici0xqwN7HlI5bgAn0GFYy6myzFCN9sbqgkTdsuYMBdV\\\\\\\/5kqTPScGdr8OSue9klw2oI6Eut0CnDHeG++lPGgsFJvbRCrENrXnm0p6SHqFAPbBCTyj417kmTJB7jVChz\\\\\\\/4tEh1rFEOvOQkoaJ3021gzFLNMnNraVsoW0SCuTxT12FPFUbkIjVgXXyMsW4YxXTuHniwJNVMo2whxXVaMJgew16kLmPX9KOdAGkkmbP4qYJL29mtzb6Jm1mDgz7GkQfhkaY5QAMiok8cbrx6g7UxaA\\\\\\\/BUOXTOp2MojtkWJJTsS4bBrp+SCNA1WSORmdl2bm8OJaVl6kV1SKh6Wu6uhFTr42X19vMrEMzqPriiixf7GTh+Cok2+hl+JBj4mvAu\\\\\\\/MVj9JW\\\\\\\/CS6pT24nkInNiISv6QoVqe3zZwtVYN0WRnA4+64AVswdDdE+Rq1vHytlUb5H8ptcoRLK6YuHsSek0fqJuVwRQsh4q5Nn71FfG07KOWRCl1OiBi9LjurXVfJxTa84Sz6XZhnbeu+FQP6qyhfmxIaU1UBHWOo\\\\\\\/ESD\\\\\\\/DsXjcGjhd8nrWk2bGhpvIWC5pRxmpHwEA6ZSX\\\\\\\/AHTUBpHVHcAmPmy\\\\\\\/VRXGEhzB3e7WG90Yi+SYz07tjGEjd2VKAp2ygIOR8ydxyNQHyPnBhjdB\\\\\\\/m4MVMw\\\\\\\/Q9DtzyLXDis9bXFtlwaifWDRuOfm1U6CF6n6Os0F7fPUPbFxKe29OQtTZFYQeUFbE2m3C+lTTjNmbmyrqjxxq\\\\\\\/mjR1oHeceKKKqw33LSOhbenn\\\\\\\/A7IgqwNADsu+GQCzOvqYpuwXYnWb25bUaWP8UgTXaaWXUXwJeOFTgHyILJ7qGChrqQ\\\\\\\/XxThBTKBEgfd91fwp\\\\\\\/Lc0v5iykRurXFsnIxx\\\\\\\/RqJA2k0Ox4f\\\\\\\/qMlcGvRfXyxe0p0Gml2ocRhenTTnYt3ToBtI8rs4et85GPzBu3T9SHNhwjW0HtOBEHQQUXK582OkvU4JTazLZNAQmnRjka\\\\\\\/I1xwbMiWaU5FesRQxGE7hQWyR72SseJrkjoUsR9HUIM5hquXqQ0JfAUKQB2EghzMvmHFj4pF4ajvjJ+h6s2qIkNiBsg8AcAVCtZL4HRsMfRyBrEwKiSRBGt81\\\\\\\/nVSzy\\\\\\\/Zo0LeItmuyPDBQu8Y2MxvpoFIgeL1WscmV8lHsDYR9TRpwESLERf62oTp2E9LOiIVdkUotTbsG09wI58V1lwrb9LhIxD0E6bdpTfk3\\\\\\\/niRKZ0Lz4mlU+XPmqE1NtBoCRFvq0VxsmypO0TWVtnbesGjKhtS9tlbcekwAFHS5+GTIZJ9hNqSoDJ27PfOcVGYI+ueHrpcUT62XfabhSA07c1tv1Yz2xJG8LIrGxZRwwAXU5SEp3\\\\\\\/wZf6ogs6FqQ3wEsLQjna7JMQROz57c4aWnKXoC1sJDbcx+caeYshVSqWX8FQSflS+DjbkK9wXpJNRo5pM6cEz2ZP6NiWaHXEqNIaZUZ1MUALwhpa76DAxxRGvl6vQ3y974SnnoLRmIknBktJeTqu2dpcZUaqCJfNrIhhZ7zoW931bLPuViLvtXlCqQRS7E5xWI1XP6Dt2Nqgu\\\\\\\/AXpZ9KTdxLAogTCa4qHSYtFyXadbOKX0KFE\\\\\\\/LGckpEr4Dabgc++ASvRDFQYv2jhT2nJyge6SX924iVe9SM3m0YV2VICFag2bK9rxBysFaUphWDl0m3TsB0mlVNcY9U13Ob9ZAoerQYRCLjRjPINS45VvMLUco5hq+TlL4Xepslyl5drGfV96h7yuAZBESZdqYaRG86mDhGCr83bC+QQysrSaCh6eqCyhjDp94PO9quSgdjGhDTXbH\\\\\\\/50mgohpWRN7Nu9aX5p5KbIkvND4fTQye1UzVMKCo3KyD8TFfiXV55Z9M5wnU3JUcEy8T\\\\\\\/f6Kzo+FbGK6wUzm532DoZhn6XF0l3yHGoybu6GZRAuPNRO4AD2qx2ORS5nDnuh96spyZkLTsqgomHRKbgpb+Pm4k9ozVmcH5V\\\\\\\/a2inmvfwDzvSlKiOX5nrV\\\\\\\/qBzdZf6IApAIhurK6Ea42SHW1v5QCRoTjLOm33e5whCdQZvXnVCylq\\\\\\\/+EdGjgVHrdPiI7Vnb6xpFhUmJusWIxSdb0dqNFoJ8ZWrJQUgEYdQ187EUPLTSb8wjOSX06qIstTzIDOS7UCol\\\\\\\/fwd+JHUqbKBW5x3QIE0Zg84IBJPd+qHXnG2Vtv0lCp1ABO7ceSWfuuWk7G5YLd+wc\\\\\\\/KSbgtKsjQKh93NcpZMRnaQYgodCkdAUpTmapZRkxqOPy3VViYTtmilQ2DLHXfMJQH1lsPa\\\\\\\/LjZ0+ZSv+lA8EZyiX2XBJ8WiHVSI\\\\\\\/HxTW74O6GoabJgM9Wfl\\\\\\\/WVV2rL0qo8sOdAlL5ch\\\\\\\/pgJvwNLWZ8jrZvDc1oEp3ViF58LF+Uj3aO2yHDUJQs97MymRK\\\\\\\/MSgg5uo5qjcOFWuM7Sp4oxk9TotgJ5g0KZ06o0UnHjRDD6v8Qq0RgatIC2K78tj6wkeYto4RBDiMZVDMABB1xzPuKnXHhQ8MCySjcH+4wcyp8QJ2pgcE245fRY5sI9OtnOjuacFNNkQ9M8BVWro0x6JytsALfh75FTgLqes7zvvc7szUkheNGrZ\\\\\\\/+qyul6s+TQFLAmnp\\\\\\\/WEICUF+HWEqu4MqcWteYoAWlQ79nZ9mL9YTyM2UH5gIAGoTkb2fDCnMvyHetAv23lWp7ptuJod8IiZ\\\\\\\/Sm62NWLnk7YYdxRAeKNbBL9dAf\\\\\\\/k3hHAjPvmTLOil8F+jkdkiAw3+wH+V2m69dZVqukWNMd4\\\\\\\/CaIyzgaNveDSrS1F3V8w8UNiGzY3Ie3uEdXsiZnf9vP6Op2kGaNupN7PMFgWb3rmNb88T4CeQ68kLfj8\\\\\\\/KvQP5SCqwVjesBAvI1\\\\\\\/j7xMJYvz6zZKUw6B\\\\\\\/VRKF6Cw\\\\\\\/5l81eguwdszkdyPqB9jTbVw4Zqy9VY6ttS62XdXdb6CKYu0S\\\\\\\/pRwzDOJqiT6\\\\\\\/kPNAErIMGAsbDExYZEot3VyedQQP31MDuF3rpV2XcmMSa\\\\\\\/dctSJ0HlipQHDb8LTN89UJ1NAgQvMeJ3PMcONkaLQk6FIQj+4Mgg46n2A8QYj\\\\\\\/a87RGWGMeHf91cAJ8iDWpSBNQOBO\\\\\\\/yj1FtJCfL1UUnwLAGQAq69t02pM08rI0dEtryEjoERAx2QXgz38QBsk\\\\\\\/cLMPFtQVxxTaCfrlFP\\\\\\\/3AAwd7FaiC3EljlDdymdjSiqr2zEf03q6QIz2aXFx2vr7wzS2DjWi0Eug0or5emefF7K8pNKkCmgpqMmgVFh9EStVLceqA0ly4uiKnCoVIcaDJNt3DMRUIUcr2z+l9PvXWNrB\\\\\\\/i5HytH1E8xWuAQVi3r3dRG62wA7SS1+A4+hvy\\\\\\\/7eqCMYnMlyG2XeDwPjcX1top6N2dZzaBtG+rJPdtR8NLOkKZgMdIXd0twHSaGN2hWedSw8HeWUPa6GK8k2D5GhxfAu623LnEySaCyLsBZgu\\\\\\\/lrB9mjFknllXUvmI1kTcSmESdwfYMXCmPtx882gIz2LB1LVpCc2ISuYxo9wh4X15+T+AT1YYh4sNjZrXenECV0pamIx6Zgqxo3Yqn4+7rPRTcraeQS08uTcr37DnDX2RpQ0FAEbQgBkKr\\\\\\\/b6XFXITWi6UXbFzw6qzrhDwFiO58himDDOSGGOZIPWO8GLTAMtfpZRYRdHlEmqmqTmIJ6w\\\\\\\/yaey9O2jNKINzmuiJ7G6Yha7gkpOTGKjLSvZV4mO5yELRyqCIXJcmbjRCQgFfRJgs0Y+AqVczWLwTFYRYqmUslZt9I4rxelzdIYHizIIlqqnbMzIZu+4TXKD6lTkiirB2HlF3Fs0F4Uqhm2QruT1l\\\\\\\/G2AWSA9AFnJN5dC8Bxa27CS8fa8Xn\\\\\\\/AeZN\\\\\\\/++pk2ok34mKSnAV0tKY4Gha0RDLnMYb6I6ntWHFlhf1FwaJOsOWg1UdRaOcvqnylLTs896q9JVloRJz09yuFi6WVjDGmUcXuWMs6d5pn88VAMRH8qsQLy52U\\\\\\\/pE7s\\\\\\\/tmB5pxK+fcrGX3NKvE3CURJqwS1Lv+XGVelYdR24HU7p1gJ\\\\\\\/JCNa6L8gDLBSSNre8lLKpLdjdCvszyKOHbVC\\\\\\\/bGLPpWD5sabReX+opjjii66eLTJlh00efSuOQWSj+K8X33MuBfQbY3i2waRXeCLrbAo1dAxK43x+eW4zGpIhyC4TDfe5VNHCBJ\\\\\\\/KxKdnyNERZ6Cs5WPZBBCaRtkTxk9T8yI5NIc5U6nCvnFbuf4gP95FtxsoRQrgPQftXxtrHtb8yPrNKyU2g8BkUWGYNWvy+qRysUQLs34jQSEscdXWtWY3F22rUJKNWXZQhlL6vtNbSj91SyHEZIANaw3yfo7bh2k1jjf+qBb2Y5U8HcB8iigNbEk+PbBUfvtplFaIqtBG2doxMy\\\\\\\/DROivytwLn\\\\\\\/oySSDur5vgYDhciom8xT5w0ZVzdWc7sn9Lo\\\\\\\/Pngs8TM7+qrR7ll8Dse3FO6xdZoT3o26qI1f5lApzlaCVuNfOgHzYwN6i8FrlyjhQtzmmlyORI+1A3tGlQnAblzSVz5m\\\\\\\/PV5loTQJhIJIEBqPxyi3NtevNIGnXeL41r1SIa79cJL2naGApPm1heX9s\\\\\\\/qgdyi5vCQvDlt0Tp7a\\\\\\\/635WMF0tP7xsTjvz5q\\\\\\\/YynDsZaH73w8MhW6ILOdSH\\\\\\\/vLAj0GQH6mWVhveg+ce3KBEFfS6\\\\\\\/syyo\\\\\\\/rwCaOUKCoN0X6JPiSKvtc2Ur2upVuKAu46pP13UQwbLA9SUYiKpl8htUP+zgJMmZQa5PekyLa8Zc3rNTXI2lfJLBp7wUQP7P0npJnJSxyZ7YiQBOuljrOHXdLMVPkyvG\\\\\\\/cIqGzevQc2GU7GmiaomwF\\\\\\\/jX6rCTNglc71a8p+DhNaA3NCiMrqBXRz5hfdIV9rwmcRrdts9luHdntdDJGfKj7uUgMQa2mHCPAZiplwdYZ2jFfTTy9nekAVS2OUi\\\\\\\/\\\\\\\/4a+3EilryEJtuvQpV98Bt3bsyIgtrQ3la9d2\\\\\\\/QQRuRaJrDznKQyilypnyogQq4FqPlhCHbOU0vPVa3haDCZM5xQeTyV+T9pgEoBLD+nA\\\\\\\/d4WyINGjB5DWf7IJLCrU\\\\\\\/FZdEbeUIchu9tnOy7MNdV\\\\\\\/\\\\\\\/CEL2HoBFc9zrEqgIN6f90WM3FGwCpt9rSfwBArrYNlFq8XEw44CayDCXrzwqJ5744LmLUDUNyyoImghiu3+ypYlf5Kbm1EOwXEuaT9sbaYdzujIOTiWS53fWl1\\\\\\\/P86VzQ8CkEaAYOz276dlpcl0JhI7PnuLXgzMzdJm+vJyrjUKO3P\\\\\\\/v5z8J6d5PzbgbdSxeWUsU4m6DTBOse\\\\\\\/ohGUaOzt983yPnEWzcH4wIPxWQ52iCbGfmW6hU+4ZXIlIZ9EIzeAbdBCblkzWVTqPY0lvsgYviyPMA7+1PVk0AcZcfChrWvQpjLNsfLL5Y7MKKWAfxtoFdWTdngCw83Ox773sZX06EVpRqhSTSmtq5iYebOjOzAAAafpkR7XslAg1RyDwYEcpTZFDsLkZdBXCejiTQkBZ6iL+H8oibP8quVjE\\\\\\\/1Ng6qWzrDGIuQiGwablQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022r5wZVcBYX9mHQqLHxKh58Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226473cbb7e45789611fa07f011a45a34d92791a0bb5cd08a684baf1dc4281bfb5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AGw+9n\\\\\\\/sArDDIylFjYgwydfJ3ZLvJcX2LeH1MpT6rMbvtVTIHnu2i5BFXQJUKdeH6z2Cdf1M4ZHNsaY1Bto8nyKNocnyWr9vJMkZ5VpdpG4e9\\\\\\\/guvboPySeOBVOESYE\\\\\\\/8FJjPfn3+AFQZCe34QGUxvQ4y0giiuUWDwAuvITiEvhh\\\\\\\/SVc2tWZprSYF1FvPaFlSDf54Cui7kKesKNaDGFeDIrXUz77eUw0\\\\\\\/yNPE91kgkLuVc\\\\\\\/19iefMetgNlBZarF\\\\\\\/qucNHoGYhqQFyDLJRpJLDWvbiRk8hj6AZraF9bYn7k7hIC7JAJltZG0PVbB3w\\\\\\\/a6FTfrACatezI2Tie96RxnfqrBvxFsP7z9qNTQ9oDRUrj2Z7vCCdJMLEs6wqFQM7VnGLIKAEvWxUjxAeSDvzvNNKUQzAY6\\\\\\\/xL2RDgehZ3cfnhSIYfM\\\\\\\/gsogRQe4gd4A9YWUVbUlLMA+jaMyt6UcbogycyF4Hcct\\\\\\\/QC+WvH0AuafDXjEjLPOyW4EgJWC8xcVTBf41iPHhHkU9nKqpapmVOcz5VZ9eykymIeP0KGBs2TlXwM6SEGSTDTcu844+Mx3r1Raxz70WJIBETN7VTLayTw0re0NuvEiBUiN81A5IOIWEcJUEQlYq8kAhE+NI7yYXkxx9YJU2H+iLj5N+NAqsZhcZT6zQQdRCgdokUO6eBcfLogd77VJOogKdXMxnR0IK\\\\\\\/Wn4Y1h9+Fz9v+eVjnot5Jf6O6tR1afsIKrVYlXfpE0tAAGbZZCirVi7ON9R1WvtEKF\\\\\\\/bEDFzwcYMlLI\\\\\\\/vRfID5k+PGF6G1vWfmfPShnfASfChruPeOHPBitjxNoIFDtNIsNw4bgZQX0DeioWtRUiVxYq\\\\\\\/wNBgmHn8vro4u48ILMev9PAqA0QWZLax8iEJHX3\\\\\\\/g6u7vMP0xwj5yWi7RcM7kIXJJNNGYIlX9FjTuNpo0z8mg\\\\\\\/z9XkCCG+ISoFUG25SJfw48V9GEEJPOwkunLgJgNc8SOFanzOlRl\\\\\\\/PacoiyqndzmuNk9Sq4\\\\\\\/zlE1Oypc4ioC82KxKRItsrO9yMJ9zYEqn476D0MkGwv8lreN+8Uwp2ckMw4FyoKk6e\\\\\\\/kzxv1JzJpem0TmSQ\\\\\\\/ZqS5+sTQTgKsFHGMhd0t8VV8oKTEVdi3QeuHs+Lc8\\\\\\\/17NEyZ3NJSku2w8QsS5\\\\\\\/Mbqop8PYB\\\\\\\/T\\\\\\\/+5wAN0Qy5vifZ\\\\\\\/nJgDJJ21DbQOjNpxDX6wyhqhIMUq4yM7xRRy+hayUV1SkJmIvJaZRjZA8+rv7ABBjR5VkEGMAybKdvOuBUZ2jwBGTzquYQRrCgS7vSdTJeTjQPFL+BqaRlEJkfIdPjnZbjJM9N2rVCBsd5lOW8OODfWf87LBdktx1THgQ55bupOmJNKGzwdKUFptWQNyUB1+w\\\\\\\/sXbp\\\\\\\/PcLKOFITbtl4NGT0RZy9EgWOW6HqT7VDbJcNu1X+ByOYffK8aj9xTozz39jUNPDBvJDEZcRs9rMx4j+K6q3dLlZp23MsQCWDXHgoFIZmrEFg3IRenr2QfTJs190HQvWDzQgYuBFkgu5lLzlazzjNZOB2WOf3y\\\\\\\/u8MwEft1gfoRHmZZrIN7BgP02bmwtSzRYh\\\\\\\/1KzBxWox2gp\\\\\\\/1a6ETgfHdiTxYHWO\\\\\\\/HiGckG99FAsc58VN4JbvUwNdkr3A+auT3gDqVd3Te\\\\\\\/afFunKQTHZmzt91SozKpYcQQi26wo7zWrkJrcja6RW23SY6uZq8WrLz1FEhBg64KxKv8utM\\\\\\\/3ts39o48UwSPNLv2QEQ8kwrOqj0FUyEcsZoTYB8pE61bXfzz9NuCsSEUj4yHh1RTiE\\\\\\\/Ln60JFWiU5wAKAGinGnD2z68sKJqTTnaCXmzf8W8YYsHTqh5ozQhOKnDrMyoQrbPoS97WOlAH39vttEcGeOeuva8HnQTQn7QHS7BCkhfhGyFWq\\\\\\\/gzusiEfQAhK\\\\\\\/5pOpSH9kA6YP36OK2km01msDcsiwEfWalEjy9TJtk23iM6\\\\\\\/8F0qrg79y1HASl6rHL6\\\\\\\/KqpjkiB9+F0+cdRdkKEF9U0Eg93bg==\\\u0022,\\\u0022iv\\\u0022:\\\u002299M8xyJxXYsEkQq7ypGV0g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c6481c61714c346735cbc4bdf7eb5f851cbebde6e1bc23a5b8ca4c2f4e87080c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492183 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/327" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-58yVyW2rMpiuk-TOhpCtmglY2-YTDy6D2iYMGJeHP-c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/327" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QMS55lfigTPaCrDXzCk_Jw0GpeCU6D6Etrnhlq_Aorw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/2/317" title="Genome-Wide Analysis of Mycoplasma dispar Provides Insights into Putative Virulence Factors and Phylogenetic Relationships" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/2/335" title="SNZ3 Encodes a PLP Synthase Involved in Thiamine Synthesis in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492183" id="node492183" data-pisa="ggg;9/2/327" data-pisa-master="ggg;g3.118.200942" data-apath="/ggg/9/2/327.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">pSBVB: A Versatile Simulation Tool To Evaluate Genomic Selection in Polyploid Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5618-2630" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>María L. Zingaretti</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7106-7745" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amparo Monfort</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Miguel Pérez-Enciso</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">327-334;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200942 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492183" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">María L. Zingaretti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mar%C3%ADa%2BL.%2BZingaretti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zingaretti%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMar%25C3%25ADa%2BL.%2BZingaretti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5618-2630" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for María L. Zingaretti</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.zingaretti@cragenomica.es" class="" data-icon-position="" data-hide-link-title="0">laura.zingaretti@cragenomica.es</a>
<a href="mailto:miguel.perez@uab.es" class="" data-icon-position="" data-hide-link-title="0">miguel.perez@uab.es</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amparo Monfort </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>IRTA (Institut de Recerca i Tecnologia Agroalimentàries), Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amparo%2BMonfort%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monfort%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmparo%2BMonfort%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7106-7745" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amparo Monfort</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Miguel Pérez-Enciso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>ICREA, Passeig de Lluís Companys 23, 08010 Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BP%C3%A9rez-Enciso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Enciso%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiguel%2BP%25C3%25A9rez-Enciso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Miguel Pérez-Enciso</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.zingaretti@cragenomica.es" class="" data-icon-position="" data-hide-link-title="0">laura.zingaretti@cragenomica.es</a>
<a href="mailto:miguel.perez@uab.es" class="" data-icon-position="" data-hide-link-title="0">miguel.perez@uab.es</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/2/327" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492183" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492183/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/327.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492183" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492183/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/327.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492183" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492183/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/2/327.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492183" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492183/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic Selection (GS) is the procedure whereby molecular information is used to predict complex phenotypes and it is standard in many animal and plant breeding schemes. However, only a small number of studies have been reported in horticultural crops, and in polyploid species in particular. In this paper, we have developed a versatile forward simulation tool, called polyploid Sequence Based Virtual Breeding (pSBVB), to evaluate GS strategies in polyploids; pSBVB is an efficient gene dropping software that can simulate any number of complex phenotypes, allowing a very flexible modeling of phenotypes suited to polyploids. As input, it takes genotype data from the founder population, which can vary from single nucleotide polymorphisms (SNP) chips up to sequence, a list of causal variants for every trait and their heritabilities, and the pedigree. Recombination rates between homeologous chromosomes can be specified, so that both allo- and autopolyploid species can be considered. The program outputs phenotype and genotype data for all individuals in the pedigree. Optionally, it can produce several genomic relationship matrices that consider exact or approximate genotype values. pSBVB can therefore be used to evaluate GS strategies in polyploid species (say varying SNP density, genetic architecture or population size, among other factors), or to optimize experimental designs for association studies. We illustrate pSBVB with SNP data from tetraploid potato and partial sequence data from octoploid strawberry, and we show that GS is a promising breeding strategy for polyploid species but that the actual advantage critically depends on the underlying genetic architecture. Source code, examples and a complete manual are freely available in GitHub <a href="https://github.com/lauzingaretti/pSBVB">https://github.com/lauzingaretti/pSBVB</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic selection</a></li><li class="kwd"><a href="/keyword/polyploids" class="hw-term hw-article-keyword hw-article-keyword-polyploids" rel="nofollow">Polyploids</a></li><li class="kwd"><a href="/keyword/simulation" class="hw-term hw-article-keyword hw-article-keyword-simulation" rel="nofollow">Simulation</a></li><li class="kwd"><a href="/keyword/psbvb" class="hw-term hw-article-keyword hw-article-keyword-psbvb" rel="nofollow">pSBVB</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Genomic selection (GS) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Meuwissen <em>et al.</em> 2001</a>) is the breeding strategy consisting in predicting future performance using DNA information from the whole genome, typically SNPs (single nucleotide polymorphisms). It relies on genome wide linkage disequilibrium (LD) between markers and the causal mutations, without the need to identify them. Due to dramatic reduction in genotyping costs, GS is becoming standard in many animal and plant breeding schemes, replacing or complementing traditional methods based solely on pedigree information. So far, GS has been mainly applied to diploid species. Yet, polyploidy is a very common phenomenon in evolution and include numerous species of interest (<em>e.g.</em>, strawberry, potato, wheat). Traditionally, polyploid species have been classified into autopolyploids, caused by one or more genome duplication events in a single species, and allopolyploids, the result of hybridization between closely related species (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Stebbins 1947</a>). The impact of GS on either auto- or allo-polyploid species breeding, however, remains largely unexplored.</p><p id="p-4">In principle, the application of GS in polyploid species can have a positive impact in the rates of genetic gain through improved accuracy of predicted breeding values and/or reduction of generation intervals (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Slater <em>et al.</em> 2016</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bassi <em>et al.</em> 2016</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sverrisdóttir <em>et al.</em> 2017</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gezan <em>et al.</em> 2017</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Enciso-Rodriguez <em>et al.</em> 2018</a>). However, the complex genetic structure of polyploids has delayed the availability of genome-wide genotyping SNP arrays that are needed for GS. Polyploid SNP detection can be challenging due to a high similarity between homologous and homeologous sequences, which generates complications to differentiate true SNPs from nuisance paralogous variants (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bassil <em>et al.</em> 2015</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Clevenger and Ozias-Akins 2015</a>).</p><p id="p-5">Further, accurate genotyping is also important but becomes more complex as ploidy level increases. Several tools to perform genotype estimation from SNP array platforms are already available (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Voorrips <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Voorrips and Gort 2018</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Schmitz Carley <em>et al.</em> 2017</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blischak <em>et al.</em> 2017</a>). However, the arising of Next Generation Sequencing technologies requires of new tools adapted for this type of data, which are also being developed (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bourke <em>et al.</em> 2018</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Meirmans <em>et al.</em> 2018</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gerard <em>et al.</em> 2018</a>).</p><p id="p-6">Computer simulation is a fundamental tool to evaluate alternative breeding schemes, since it allows the exploration of a wide range of hypothesis at no cost and can help to interpret the outcome of selection in complex situations. In this regard, numerous simulation tools have been developed such as easyPOP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Balloux 2001</a>), simuPOP (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Peng and Kimmel 2005</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Peng and Amos 2008</a>), forqS (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kessner and Novembre 2014</a>) Slim (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Messer 2013</a>), PedigreeSim (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Voorrips and Maliepaard 2012</a>) among others. However, simulation approaches may not be straightforward to interpret owing to unknowns on the genetic architecture, among other factors. These problems are exacerbated in polyploid species and, to the best of our knowledge, only simuPOP and PedigreeSim allow polyploids organisms. simuPOP is not developed to compare breeding schemes, whereas PedigreeSim does not directly generate phenotypes nor produces genomic relationship matrices.</p><p id="p-7">Here we present a flexible simulation tool for complex phenotypes adapted to polyploids and we propose several approaches to compute the molecular relationship matrix in polyploids. The software is an extension of Sequence-Based Virtual Breeding (SBVB, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Pérez-Enciso <em>et al.</em> 2017</a>), called pSBVB. This tool employs complete or partial genome data as input and simulates new genomes by gene dropping. We illustrate the software with data from two economically important polyploid species: potato, an autopolyploid, and strawberry, an allopolyploid.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Polyploid sequence based virtual breeding (pSBVB)</h3><p id="p-8">pSBVB is a modification of SBVB software (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Pérez-Enciso <em>et al.</em> 2017</a>) that allows simulating genotypes and phenotypes of an arbitrary genetic complexity in polyploids. Compared to SBVB designed for diploid organisms only, pSBVB enables simulating meiosis in autopolyploid or allopolyploid species (see below). It takes ploidy into account to generate the phenotypes and incorporates several options to compute the molecular relationship matrix that are pertinent to polyploids, as described below.</p></div><div id="sec-3" class="subsection"><h3>Software algorithm</h3><p id="p-9">As input, pSBVB needs genotypes in vcf format (<a href="https://samtools.github.io">https://samtools.github.io</a>) or a text file with genotypes coded to 0 up to <em>h</em> (where <em>h</em> is the ploidy level). For diploids, the vcf genotype format is of the kind 0/0, 0/1, and 1/1 for the three possible genotypes in a biallelic SNP. The polyploid vcf format is an extension of the type 0/0/0/0, 0/0/0/1 and so on in the case of an unphased tetraploid genotypes. Phased genotypes are represented by vertical bars, (<em>e.g.</em>, genotype 0|0|0|1 is different from 1|0|0|0) . No missing values are allowed. Phased genotypes are needed in pSBVB to identify which chromosomes are passed to offspring. A number of accurate phasing algorithms for diploids are available such as beagle (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Browning and Browning 2007</a>) or minimac (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Howie <em>et al.</em> 2012</a>). For polyploids, several approaches are also developed (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">He <em>et al.</em> 2018</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Shen <em>et al.</em> 2016</a>), but their accuracy has not been completely validated and seems critically dependent on ploidy level. If phase is unknown, pSBVB randomly generates a phase configuration. Further, linkage disequilibrium can be obtained by generating an individual genome out of a random pedigree starting with the founders’ genotypes. To do that, pSBVB incorporates the option ‘EXPAND_BASEPOP’, which generates additional founders’ by randomly crossing the available ones and random breeding for a pre-specified number of generations (see SBVB manual, <a href="https://lauzingaretti.GitHub.io/pSBVB/">https://lauzingaretti.GitHub.io/pSBVB/</a>). A list with QTNs (Quantitative Traits Nucleotides) positions, a list of SNP positions to be used for GS, a pedigree file and a parameter file are also necessary. The pedigree file is used to perform the gene dropping simulation, <em>i.e.</em>, genotypes’ of the descendants along the pedigree are generated following Mendelian rules and a pre specified pairing rate between homologous and homoelogous pairs; for autopolyploids, pairing is at random. While performing gene dropping, pSBVB stores only the recombination breakpoints, which results in an efficient algorithm to recover marker genotypes and phenotypes.</p><p id="p-10">pSBVB is very flexible in terms of the genetic architectures; it can simulate any number of traits with their specific QTNs and allelic effects. QTNs effects can be specified in a file or sampled from gamma, normal, or uniform distributions. In contrast to SBVB, though, pSBVB does not allow for epistasis. The <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows a general representation of the pSBVB software, as well as screen shots.</p><p id="p-11">As output, pSBVB produces phenotype and marker data of individuals obtained from the pedigree-based gene-dropping procedure. In addition, pSBVB can also compute molecular relationship matrices G using predefined marker subsets (<em>e.g.</em>, a genotyping array) or the whole sequence. For polyploids, G is computed by default from:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where M is a <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-2.gif"/></noscript></span></span> matrix with elements containing the number of copies of the alternative allele for <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-3.gif"/></noscript></span></span> individual (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-4.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-5.gif"/></noscript></span></span> SNP (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-6.gif"/></noscript></span></span>), and <em>p</em> is a <em>m</em>-dimension vector with marker allele frequencies. Note that <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> reduces to the standard formula in the case of diploidy (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-7.gif"/></noscript></span></span>) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">VanRaden 2008</a>).</p><p id="p-12">Assessing the genotype for polyploids can be inferred from fluorescence intensity in SNP arrays or from read count in sequence data (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bourke <em>et al.</em> 2018</a>) but may not be as accurate as for diploid organisms, specially at high ploidy levels. If genotyping is not accurate, a simple alternative is assuming that only one full homozygous can be distinguished for the rest of genotypes, <em>i.e.</em>, that a given marker allele behaves as fully dominant. To accommodate this, pSBVB allows computing a modified <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-8.gif"/></noscript></span></span> where element <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-9.gif"/></noscript></span></span> is coded as 0 if all alleles are 0 and 1 otherwise. This is specified with the MIMIC_HAPLOID statement in the parameter file. The software also incorporates a ’MIMIC_DIPLOID’ option, which assumes only presence or absence of the alternative allele can be ascertained for genotype values higher than 2. In summary, the software is able to generate three G matrices:</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-13"><strong>Default option:</strong> The true genotype, <em>i.e.</em>, number of copies of the alternative allele, is known without error (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-10.gif"/></noscript></span></span>). In this approach M (<a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>) has elements varying between 0 to <em>h</em>.</p></li><li id="list-item-2"><p id="p-14"><strong>MIMIC_DIPLOID option:</strong> Only 0, 1 and 2 or more copies of a given allele can be distinguished. In this case, all genotypes with values larger than 2 area assigned a value ’2’, thus M (<a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>) has elements ranging between 0 and 2 and ploidy is set to 2.</p></li><li id="list-item-3"><p id="p-15"><strong>MIMIC_HAPLOID option:</strong> It considers that only one full homozygous can be distinguished for the rest of genotypes, then M (<a id="xref-disp-formula-1-4" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>) has elements ranging between 0 and 1 and ploidy is set to 1.</p></li></ul></div><div id="sec-4" class="subsection"><h3>Modeling meiosis in polyploids</h3><p id="p-16">Autopolyploids species have polysomic inheritance where homologous and homeologous chromosomes are randomly paired during meiosis. In contrast, most of allopolyploids have disomic inheritance, resulting from preferential pairing between homologous chromosomes. However, there is a continuum between both extreme meiotic behaviors that can be modeled by preferential pairing factor (<em>θ</em>), which expresses the increased probability of pairing between homologous chromosomes (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bourke <em>et al.</em> 2017</a>). In a generic case with <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-11.gif"/></noscript></span></span> sub-genomes, where <em>h</em> is the ploidy level, there are <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-12.gif"/></noscript></span></span> possible paring combinations between all the chromosomes. pSBVB allows modeling meiotic pairing via a recombination <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-13.gif"/></noscript></span></span> matrix:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-15.gif"/></noscript></span></span> ∀ <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-16.gif"/></noscript></span></span> in <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> represents the probability of pairing between <em>i</em> and <em>j</em> chromosomes, assuming chromosomes <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-17.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-18.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-19.gif"/></noscript></span></span> are the homologous pairs.</p><p id="p-17">For example, the matrix for a strict auto-tetraploid is:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-20.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>And for a strict allopolyploid would be:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-21.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Internally, pSBVB assumes that the order of chromosomes in this matrix is the same as in the genotype alleles from the vcf file.</p></div><div id="sec-5" class="subsection"><h3>Phenotype simulation</h3><p id="p-18">In a diploid organism, the phenotype for <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-22.gif"/></noscript></span></span> individual can be simulated from:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-23.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <em>μ</em> is the general mean, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-24.gif"/></noscript></span></span> is the additive effect of <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-25.gif"/></noscript></span></span> SNP, that is, half the expected difference between homozygous genotypes, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-26.gif"/></noscript></span></span> takes values <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-27.gif"/></noscript></span></span>, 0 and 1 for homozygous, heterozygous and alternative homozygous genotypes, respectively, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-28.gif"/></noscript></span></span> is the dominance effect of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-29.gif"/></noscript></span></span> SNP, and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-30.gif"/></noscript></span></span> takes value 1 if the genotype is heterozygous, 0 otherwise, and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-31.gif"/></noscript></span></span> is a normal residual of the <em>i</em>- observation. For polyploids, the equivalent equation can be expressed as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-32.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-33.gif"/></noscript></span></span> is the number of copies of the alternative allele (coded say as 1) minus half the ploidy (<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-34.gif"/></noscript></span></span>) for <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-35.gif"/></noscript></span></span> SNP and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-36.gif"/></noscript></span></span> individual, and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-37.gif"/></noscript></span></span> is therefore the expected change in phenotype per copy of allele ’1’ in the <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-38.gif"/></noscript></span></span> SNP. In polyploids, as many dominance coefficients as ploidy level (<em>h</em>) minus 1 can technically be defined. However, this results in an over-parameterized model that is of no practical use. Here instead we define the <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-39.gif"/></noscript></span></span> parameter as the minimum number of copies of allele 1 such that the expected phenotype is <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-40.gif"/></noscript></span></span>. In our modeling, all genotypes with number of copies over <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-41.gif"/></noscript></span></span> have the same expected phenotype. See <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> for a graphical representation. By default, pSBVB takes <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-42.gif"/></noscript></span></span>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/327/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Additive and dominance modeling in polyploids used by pSBVB. The figure represents three possible genic actions in an octoploid. Under a strict additive action (&#x2218;), phenotype is expected to increase in an a unit per copy of the alternative allele, note that a can be negative or positive. Under a dominance scheme, and , phenotype is expected to be the same for any heterozygous genotype (+) (Dom 1), whereas under dominance and  (Dom 3), the genotype is expected to be the same for all heterozygous with more than 3 copies to alternative allele (&#x25C7;)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1444386730" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Additive and dominance modeling in polyploids used by pSBVB. The figure represents three possible genic actions in an octoploid. Under a strict additive action (&#x2218;), phenotype is expected to increase in an &lt;em&gt;a&lt;/em&gt; unit per copy of the alternative allele, note that a can be negative or positive. Under a dominance scheme, and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-43&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, phenotype is expected to be the same for any heterozygous genotype (+) (Dom 1), whereas under dominance and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-44&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (Dom 3), the genotype is expected to be the same for all heterozygous with more than 3 copies to alternative allele (&#x25C7;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/327/F1.medium.gif" width="440" height="325"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/2/327/F1.medium.gif" width="440" height="325"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/327/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/327/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492087" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Additive and dominance modeling in polyploids used by pSBVB. The figure represents three possible genic actions in an octoploid. Under a strict additive action (∘), phenotype is expected to increase in an <em>a</em> unit per copy of the alternative allele, note that a can be negative or positive. Under a dominance scheme, and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F1/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F1/embed/mml-math-43.gif"/></noscript></span></span>, phenotype is expected to be the same for any heterozygous genotype (+) (Dom 1), whereas under dominance and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F1/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F1/embed/mml-math-44.gif"/></noscript></span></span> (Dom 3), the genotype is expected to be the same for all heterozygous with more than 3 copies to alternative allele (◇).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/327/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="General representation of pSBVB software. As input, the software reads the vcf file containing all phased SNPs from founder haplotypes. Additional files specify the genetic architecture (it may include additive and dominant effects), the lists of SNPs (each corresponding to one genotyping array and/or complete sequence), and the recombination map for each sex and genome location (optional). pSBVB then performs gene dropping following a predetermined pedigree, generating phenotypes and true genotypes (Y), genomic relationship matrices (G, one per SNP list), and genotypes for each individual in the pedigree and for each SNP list in Plink or generic format, an optional file containing haplotype information that allows quick restart of the program, and information on QTN contribution to variance. Genomic Relationship Matrix can be computed using several options (see main text). As output, the software provides genomic relationship matrix (Y.grm.1), QTN&#x2019;s effects (y.outq) and simulated phenotypes (y.outy)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1444386730" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;General representation of pSBVB software. As input, the software reads the vcf file containing all phased SNPs from founder haplotypes. Additional files specify the genetic architecture (it may include additive and dominant effects), the lists of SNPs (each corresponding to one genotyping array and/or complete sequence), and the recombination map for each sex and genome location (optional). pSBVB then performs gene dropping following a predetermined pedigree, generating phenotypes and true genotypes (Y), genomic relationship matrices (G, one per SNP list), and genotypes for each individual in the pedigree and for each SNP list in Plink or generic format, an optional file containing haplotype information that allows quick restart of the program, and information on QTN contribution to variance. Genomic Relationship Matrix can be computed using several options (see main text). As output, the software provides genomic relationship matrix (Y.grm.1), QTN&#x2019;s effects (y.outq) and simulated phenotypes (y.outy).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/327/F2.medium.gif" width="440" height="298"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/2/327/F2.medium.gif" width="440" height="298"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/327/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/327/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492198" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">General representation of pSBVB software. As input, the software reads the vcf file containing all phased SNPs from founder haplotypes. Additional files specify the genetic architecture (it may include additive and dominant effects), the lists of SNPs (each corresponding to one genotyping array and/or complete sequence), and the recombination map for each sex and genome location (optional). pSBVB then performs gene dropping following a predetermined pedigree, generating phenotypes and true genotypes (Y), genomic relationship matrices (G, one per SNP list), and genotypes for each individual in the pedigree and for each SNP list in Plink or generic format, an optional file containing haplotype information that allows quick restart of the program, and information on QTN contribution to variance. Genomic Relationship Matrix can be computed using several options (see main text). As output, the software provides genomic relationship matrix (Y.grm.1), QTN’s effects (y.outq) and simulated phenotypes (y.outy).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Statistical model for Genomic prediction</h3><p id="p-21">There are currently numerous statistical methods that address the large p small n problem and use genome-wide markers to predict breeding values (<em>e.g.</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hayes <em>et al.</em> 2009</a>). pSBVB does not compute genomic breeding values but can produce genomic relationship matrices suitable to obtain GBLUP (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">VanRaden 2008</a>), as detailed above. Otherwise, pSBVB outputs genotypes of all or a subset of markers and any desired GS algorithm can be applied. R scripts are provided in GitHub that performs GBLUP.</p></div><div id="sec-7" class="subsection"><h3>Data Availability</h3><p id="p-22">The source codes and the documented functions are distributed from GitHub: <a href="https://github.com/lauzingaretti/pSBVB">https://github.com/lauzingaretti/pSBVB</a>. The manual includes a full tutorial of all functions at the program and a user guide with the installation guidelines and examples to simulate polyploid organisms. The software is accompanied by R scripts(<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">R Core Team 2017</a>) to generate a pedigree file, cmpute the numerator relationship matrix, perform GBLUP (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">VanRaden 2008</a>) or assess predictive ability (PA). Examples showing the software capabilities with alternative parameter options are also available.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-23">In order to illustrate the software capabilities, we have used dataset from two polyploids species: autoplyploid potato (<em>Solanum tuberosum</em>, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-45.gif"/></noscript></span></span>) and allopolyploid strawberry (<em>Fragaria</em> x <em>ananassa</em> <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-46.gif"/></noscript></span></span>).</p><div id="sec-9" class="subsection"><h3>Potato genotypes</h3><p id="p-24">The availability of an <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-47.gif"/></noscript></span></span> SNP array has allowed the development of GS studies in potato, one of the most important crops worldwide (<em>e.g.</em>, <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Sverrisdóttir <em>et al.</em> 2017</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Enciso-Rodriguez <em>et al.</em> 2018</a>). To illustrate our tool, here we used a subset of 407 SNPs and 150 individuals from <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Enciso-Rodriguez <em>et al.</em> (2018)</a>. SNP positions were obtained from <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rosyara <em>et al.</em> (2016)</a>. We used these genotypes to generate a vcf file where genotypes were coded between 0 and 4 (the potato ploidy level), phases were randomly generated.</p><p id="p-25">Next, to generate linkage disequilibrium in the randomly phased dataset, we included additional dummy founders using the “EXPAND_BASEPOP” statement in the parameter file (see reference manual, <a href="https://lauzingaretti.github.io/pSBVB/">https://lauzingaretti.github.io/pSBVB/</a>). With this option, new base population individuals are obtained via randomly generated pedigrees. A new base population with 100 founders was obtained. The total pedigree size was 700, with 250 founders was obtained. The total pedigree size was 700, including 250 founders (150 initial individuals and the 100 new base population individuals) and four generations with 100, 100, 100 and 150 observations, respectively.</p><p id="p-26">Phenotypes were simulated using 140 randomly chosen QTNs and heritability (<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-48.gif"/></noscript></span></span>) was set to 0.5. As numerous studies suggest that allele distribution is highly leptokurtic (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">García-Dorado <em>et al.</em> 1998</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eyre-Walker and Keightley 2007</a>) with many near-zero effects and a few large effects, we used a gamma <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-49.gif"/></noscript></span></span> distribution to simulate additive effects as in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Caballero <em>et al.</em> (2015)</a>. G matrix was computed assuming that all markers are known without error, since the potato chip ensures that the true genotype can be obtained. Finally, to ilustrate GS performance, which was assessed removing the 150 individuals from the last generation and computing the correlation between predicted and observed phenotypes of these 150 individuals. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> plots the observed <em>vs.</em> predicted phenotypes in training (400 individuals) and test (150 individuals) population. In this example, PA was reasonably high (<span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-50.gif"/></noscript></span></span>), and illustrates that reasonable accuracies can be obtained even with small population sizes provided linkage disequilibrium and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-51.gif"/></noscript></span></span> are relatively high.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/327/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted breeding values from GS model in the simulated potato dataset: correlations between observed and predicted values from training and testing populations were 0.91 and 0.52, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1444386730" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predicted breeding values from GS model in the simulated potato dataset: correlations between observed and predicted values from training and testing populations were 0.91 and 0.52, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/327/F3.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/2/327/F3.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/327/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/327/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492229" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Predicted breeding values from GS model in the simulated potato dataset: correlations between observed and predicted values from training and testing populations were 0.91 and 0.52, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The pedigree and the numerator relationship matrix files were generated using the pedigree.R and RelationshipMatrix.R functions, respectively; breeding values were predicted with GBLUP using GBlupFunction.R script. The whole source code and scripts to run this example are available at GibHub site.</p></div><div id="sec-10" class="subsection"><h3>Application to strawberry GBS data</h3><p id="p-29">We also applied our program to octoploid strawberry <em>F</em>. x <em>ananassa</em>. In the absence of a reasonable number of strawberry sequenced genomes, we used unpublished data obtained with GBS (Genotyping by Sequencing) from 47 strawberry cultivars. Genotype-by-Sequencing libraries were prepared by Heartland Plant Innovations (<a href="http://www.heartlandinnovations.com/">http://www.heartlandinnovations.com/</a>). Samples were multiplexed and sequenced 92 cycles on the Illumina MiSeq at the Oklahoma Medical Research Foundation. Data quality was checked by FASTQC (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). To obtain reasonably realistic genotypes based on these data, we applied the following pipeline. GBS reads were aligned against <em>Fragaria vesca</em> (diploid strawberry) reference genome (F. vesca-genome.v2.0.a1), bam files were filtered setting minimum base and mapping qualities to 37 and 20, respectively, and parsed with snape (<a href="https://github.com/EmanueleRaineri/snape-pooled">https://github.com/EmanueleRaineri/snape-pooled</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Raineri <em>et al.</em> 2012</a>), a SNP caller developed for pools.</p><p id="p-30">This software requires as input the number of diploid individuals in the pool, which was set to four. Polymorphic positions with fewer than 20 high quality reads were removed, as well as those where more than 60% of the cultivars were not covered. Logically, only allele counts 0, 1, to 8 are allowed in an octoploid genome SNP, whereas the number of reads per position follows a quasi-continuous distribution. To convert number of reads to genotype score, we computed the fraction of alternative allele reads divided by the total number of reads (<em>f</em>) and inferred its genotype from the nearest possible integer to <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-52.gif"/></noscript></span></span>. This was done for each SNP and cultivar. Missing genotypes were sampled according to the genotype frequency in the non-missing positions for that SNP. We assumed independence to perform the assignations. A total of 50,609 variant positions were obtained (5779, 7985, 7328, 6362, 8282, 9012, 5862 in linkage groups GL1, GL2, GL3, GL4, GL5, GL6 and GL7, respectively). These markers were used as genetic file input for the program. Among those SNPs, ∼ 36%, 37%, 14% and 13% variants were classified as segregating in 1, 2, 3 or all sub- genomes: <em>2x</em>, <em>4x</em>, <em>6x</em> and <em>8x</em>, respectively.</p><p id="p-31">Strawberry breeding programs are based on evaluating crosses between elite lines. Traditional crop breeding is expensive and time consuming and GS can accelerate strawberry improvement if only a subset of these crosses were fully tested in the field. To mimic this scenario, we generated a pedigree file with five generations of intercrossing starting with the 53 base population lines. Each generation was made up of 100 lines. In the last generation, 1000 crosses with unknown phenotype were generated from the 100 current parental lines. As measure of predictive accuracy, we computed the correlation between observed and predicted phenotypes of the 1000 crosses, when the phenotypes from these 1000 crosses were removed. One hundred replicates were run per case.</p><p id="p-32">To simulate the phenotypes, we considered a range of genetic architectures with a focus on sugar content:</p><ul class="list-simple " id="list-2"><li id="list-item-4"><p id="p-33">• Random QTNs in sugar associated Pathways (RQP): 100 SNPs were randomly chosen as causal among the SNPs in the sugar pathway associated genes ± 10 kb.</p></li><li id="list-item-5"><p id="p-34">• Diploid QTNs in sugar associated Pathways (DQP): 100 SNPs were randomly chosen as causal among the diploid SNPs in the sugar pathway associated genes ± 10 kb.</p></li><li id="list-item-6"><p id="p-35">• Random QTNs Genome-wide chosen (RQG): 100 SNPs were randomly chosen as causal among all detected SNPs.</p></li></ul><p id="p-36">In the first two architectures, we aimed at mimicking a trait of economic interest such as sucrose content. The gene information was obtained from FragariaCyc (<a href="http://pathways.cgrb.oregonstate.edu">http://pathways.cgrb.oregonstate.edu</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Naithani <em>et al.</em> 2016</a>). In total, there were 159 genes containing 499 SNPs associated with these pathways. Within each of the three architectures, phenotypes were simulated according to two extreme gene actions: fully additive and complete dominance (<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-53.gif"/></noscript></span></span>, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Heritability was set to 0.5.</p><p id="p-37">For each architecture, phenotypes were simulated according to two extreme gene actions: fully additive and complete dominance. In the dominant approach, we set <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-54.gif"/></noscript></span></span> (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Each phenotype was generated from its genotypic value adding an environmental effect, where was adjusted such that heritability was <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-55.gif"/></noscript></span></span>.</p><p id="p-38">Simulated PAs are in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. We estimated the PA using the following matrices:</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/327/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictive Ability () of GBPLUP and  models for each of the three genetic architectures considered in strawberry simulated dataset: random QTNs in sugar associated pathways (RQP), diploid QTNs in sugar associated pathways (DQP) and genome-wide chosen (RQG) and each of the three GBLUP models. Three GBLUP models were compared: In , genetic matrix G was computed assuming SNP allele frequencies were known without error; in , only diploid SNPs were used, and genotypes were known without error; and in , G Genomic relationship matrix is computed assuming than only presence or absence of the alternative allele could be known for the remaining, i.e., although the organism was polyploid, Genomic relationship matrix is computed assuming than only presence or absence of the alternative allele can be ascertained. (a) additive architecture; (b) dominant architecture." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1444386730" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Predictive Ability (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-56&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) of GBPLUP and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-57&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; models for each of the three genetic architectures considered in strawberry simulated dataset: random QTNs in sugar associated pathways (RQP), diploid QTNs in sugar associated pathways (DQP) and genome-wide chosen (RQG) and each of the three GBLUP models. Three GBLUP models were compared: In &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-58&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, genetic matrix G was computed assuming SNP allele frequencies were known without error; in &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-59&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, only diploid SNPs were used, and genotypes were known without error; and in &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-60&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, G Genomic relationship matrix is computed assuming than only presence or absence of the alternative allele could be known for the remaining, &lt;em&gt;i.e.&lt;/em&gt;, although the organism was polyploid, Genomic relationship matrix is computed assuming than only presence or absence of the alternative allele can be ascertained. (a) additive architecture; (b) dominant architecture.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/327/F4.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/2/327/F4.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/327/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/327/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492104" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">Predictive Ability (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F4/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F4/embed/mml-math-56.gif"/></noscript></span></span>) of GBPLUP and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F4/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F4/embed/mml-math-57.gif"/></noscript></span></span> models for each of the three genetic architectures considered in strawberry simulated dataset: random QTNs in sugar associated pathways (RQP), diploid QTNs in sugar associated pathways (DQP) and genome-wide chosen (RQG) and each of the three GBLUP models. Three GBLUP models were compared: In <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F4/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F4/embed/mml-math-58.gif"/></noscript></span></span>, genetic matrix G was computed assuming SNP allele frequencies were known without error; in <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F4/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F4/embed/mml-math-59.gif"/></noscript></span></span>, only diploid SNPs were used, and genotypes were known without error; and in <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F4/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/F4/embed/mml-math-60.gif"/></noscript></span></span>, G Genomic relationship matrix is computed assuming than only presence or absence of the alternative allele could be known for the remaining, <em>i.e.</em>, although the organism was polyploid, Genomic relationship matrix is computed assuming than only presence or absence of the alternative allele can be ascertained. (a) additive architecture; (b) dominant architecture.</p><div class="sb-div caption-clear"></div></div></div><ul class="list-unord " id="list-3"><li id="list-item-7"><p id="p-40"><span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-61.gif"/></noscript></span></span>: The true genotype, <em>i.e.</em>, number of copies of the alternative allele, was known without error and all SNPs were used. In this approach Equation M (<a id="xref-disp-formula-1-5" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>) has elements varying between 0 and 8.</p></li><li id="list-item-8"><p id="p-41"><span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-62.gif"/></noscript></span></span>: Only diploid SNPs were used, and genotypes were known without error. M (<a id="xref-disp-formula-1-6" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>) has elements ranging between 0 and 2.</p></li><li id="list-item-9"><p id="p-42"><span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-63.gif"/></noscript></span></span>: All SNPs were employed but only genotypes of diploid SNPs were known without error, whereas for the remaining, although the organism was polyploid, Genomic matrix is computed mimic diploid. M (<a id="xref-disp-formula-1-7" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>) has elements ranging between 0 and 2.</p></li><li id="list-item-10"><p id="p-43"><strong>Numerator Relationship Matrix (P-BLUP)</strong>: The breeding values were predicted using the pedigree relationship matrix.</p></li></ul><p id="p-44"><a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> shows the obtained accuracies across genetic architectures and for each evaluation method. Overall, these results indicate that performance of GS in polyploids may critically depend on the underlying genetic architecture. Unsurprisingly, accuracy also drops when dominance exists compared to the additive scenarios. Several additional observations of interest can be drawn from <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>. First, there were no differences in the ranking of methods irrespective of whether QTN were scattered throughout the genome (RQG) or localized in given segments (RQP). This was observed for both additive and dominant architectures. Second, using the true genotype values to build <strong>G</strong> (<strong>GT</strong>) did not always outperform the rest of GBLUP methods considered. In fact, this was observed only when the architecture was fully additive and the QTNs were segregating in more than one homeolog group. In these cases, <strong>GT</strong>-BLUP was <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-64.gif"/></noscript></span></span> better than G2-BLUP or <strong>G2*</strong>-BLUP. <strong>G2</strong>, which employs only diploid SNPs, should be preferred to <strong>GT</strong>-BLUP only if QTNs are exclusively diploid. A relevant result is that <strong>G2*</strong> -BLUP, which treats markers as dominant, was a quite robust strategy, in particular with complete dominance and with the exception of DQP scenario (<em>i.e.</em>, when all QTNs were diploid).</p><p id="p-45">Finally, note that the advantage of GBLUP over P-BLUP is not always guaranteed. At least in the breeding scenario analyzed here, <strong>G2</strong>-BLUP might actually perform worse than <strong>P-BLUP</strong> when QTNs segregate randomly (RQP and RQG) and genic action is additive. If true SNP genotypes could be known without error (<strong>GT</strong>), the increase in accuracy compared to <strong>P-BLUP</strong> would vary between <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-65.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-66.gif"/></noscript></span></span>. As for using <strong>G2*</strong>-BLUP, increase in accuracy was between <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-67.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/327/embed/mml-math-68.gif"/></noscript></span></span> across all cases examined here. The advantage, though, would diminish if genic action were additive and QTN would segregate in all homologous.</p><p id="p-46">The genetic file used as input includes 1500 SNPs from the whole <em>vcf</em> file. More examples combining a set of different parameters (additive and dominance effects, Genetic Matrix calculation, pedigree and Genomic Relationship Generation, among others) are available on GitHub.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-47">Certainly, polyploid sequence data will be increasingly available, which will be used to achieve a better understanding of complex trait genetics and to optimize GS strategies. To help in the latter task, here we have developed an extension of SBVB software (pSBVB) that feeds from real sequence data of polyploid organisms. It uses efficient forward algorithms and allows simulating meiosis in polyploid species, suited for both auto and allopolyploid organisms. Further, pSBVB generalizes genetic modeling in polyploids to generate phenotypes and incorporates several options to compute predefined molecular relationship matrices that are specific to polyploid organisms. Note though that, since pSBVB can print the whole SNP dataset, any custom-made <strong>G</strong> can be computed and any alternative GS method can be evaluated. There are some limitations though. An important one is that epistasis cannot be modeled in pSBVB -in contrast to the diploid version (SBVB)- but this limitation stems from the lack of realistic modeling on epistasis for polyploids rather than out of computational constraints.</p><p id="p-48">To the best of our knowledge, there are no simulation tools that allow estimate genetic matrix in polyploids organisms with a range of options like the one described here. Among the available forward-time simulation tools, only simuPOP (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Peng and Kimmel 2005</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Peng and Amos 2008</a>) and PedigreeSim (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Voorrips and Maliepaard 2012</a>) consider polyploids. Compared to simuPOP, pSBVB allows simulating both auto and allo-polyploids organisms, accepting as input a recombination matrix between homeolog groups. PedigreeSim is not specifically designed for GS and is not able to simulate complex genetic architectures and relationships matrices as pSBVB. A further outcome of our work is the proposal of several G matrices that are robust to genotype misspecification, an important problem in polyploids (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bourke <em>et al.</em> 2017</a>).</p><p id="p-49">To conclude, we have developed a flexible GS simulation tool capable of using real sequence data from polyploids. We show the tool capabilities using potato and strawberry real datasets. With potato genotypes, we illustrate how new base population individuals can be generated and show that accuracy can be relatively high even with modest population sizes. Among the molecular relationship matrices proposed, assuming that only diploid genotypes can be identified seems overall a good compromise in terms of performance, at least in strawberry data. Our study suggests that GS may increase response to selection compared to <strong>P-BLUP</strong>, but this will depend on the true genetic architecture of the trait, as also shown by <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gezan <em>et al.</em> (2017)</a> with real strawberry data. We urge advancing on the quantitative and molecular dissection of complex traits in polyploids, which should provide important parameters such as prevalent genic action or number of segregating homeolog groups, in order to design optimum GS breeding schemes for these species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">MLZ is recipient of a FPI grant to achieve the PhD research from Ministry of Economy and Science (MINECO, Spain), associated with ‘Centro de Excelencia Severo Ochoa 2016-2019’ award SEV-2015-0533 to CRAG. Work funded by MINECO grant RTA2013-00010-00-00 to AM and AGL2016-78709-R to MPE. We also acknowledge genotyping data provided by General Mills.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-51"><em>Communicating editor: G. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-10-10"><span class="received-label">Received </span>October 10, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-12-19"><span class="accepted-label">Accepted </span>December 19, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Zingaretti <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.1" data-doi="10.1093/jhered/92.3.301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">EASYPOP (Version 1.7): A Computer Program for Population Genetics Simulations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/92.3.301</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DBalloux%26rft.auinit1%253DF.%26rft.volume%253D92%26rft.issue%253D3%26rft.spage%253D301%26rft.epage%253D302%26rft.atitle%253DEASYPOP%2B%2528Version%2B1.7%2529%253A%2BA%2BComputer%2BProgram%2Bfor%2BPopulation%2BGenetics%2BSimulations%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F92.3.301%26rft_id%253Dinfo%253Apmid%252F11447253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/92.3.301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11447253&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169680200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.2" data-doi="10.1016/j.plantsci.2015.08.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassi</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charmet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Breeding schemes for the implementation of genomic selection in wheat (triticum spp.).</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">242</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.plantsci.2015.08.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D242%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2015.08.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2015.08.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.3" data-doi="10.1186/s12864-015-1310-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassil</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ficklin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mittmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Development and preliminary evaluation of a 90 K Axiom SNP array for the allo-octoploid cultivated strawberry Fragaria × ananassa.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1310-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1310-1%26rft_id%253Dinfo%253Apmid%252F25886969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1310-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25886969&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blischak</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubatko</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">SNP genotyping and parameter estimation in polyploids using low-coverage sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">407</span>–<span class="cit-lpage">415</span>. </cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.5" data-doi="10.1111/tpj.13496"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esselink</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koning-Boucoiran</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Partial preferential chromosome pairing is genotype dependent in tetraploid rose.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">343</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13496</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D90%26rft.spage%253D330%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13496&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.6" data-doi="10.3389/fpls.2018.00513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maliepaard</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Tools for Genetic Studies in Experimental Populations of Polyploids.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">513</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00513</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D513%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00513&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.7" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/521987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.8" data-doi="10.1534/genetics.115.177220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caballero</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Nature of Genetic Variation for Complex Traits Revealed by GWAS and Regional Heritability Mapping Analyses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">1601</span>–<span class="cit-lpage">1613</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177220%26rft_id%253Dinfo%253Apmid%252F26482794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS80LzE2MDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.9" data-doi="10.1534/g3.115.019703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clevenger</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozias-Akins</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1797</span>–<span class="cit-lpage">1803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.019703</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D5%26rft.spage%253D1797%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.115.019703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.10" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">e los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting Quantitative Traits with Regression Models for Dense Molecular Markers and Pedigree</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.11" data-doi="10.1534/g3.118.200273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enciso-Rodriguez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic Selection for Late Blight and Common Scab Resistance in Tetraploid Potato (Solanum tuberosum</span>). <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr>, <span class="cit-vol">8</span>(7): <span class="cit-fpage">2471</span>–<span class="cit-lpage">2481</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200273</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200273%26rft_id%253Dinfo%253Apmid%252F29794167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvNy8yNDcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.12" data-doi="10.1038/nrg2146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The distribution of fitness effects of new mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D610%26rft.epage%253D618%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bfitness%2Beffects%2Bof%2Bnew%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2146%26rft_id%253Dinfo%253Apmid%252F17637733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17637733&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248170700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.13" data-doi="10.1023/A:1017086909282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">García-Dorado</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monedero</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">López-Fanjul</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The mutation rate and the distribution of mutational effects of viability and fitness in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">102–103</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">265</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1017086909282</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D102103%26rft.spage%253D255%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1017086909282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1017086909282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.327.14" data-doi="10.1534/genetics.118.301468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrão</span>  <span class="cit-name-given-names">L. F. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genotyping polyploids from messy sequencing data</span>. Genetics, 210(3): <span class="cit-fpage">789</span>–<span class="cit-lpage">807</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301468%26rft_id%253Dinfo%253Apmid%252F30185430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEwLzMvNzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.15" data-doi="10.1038/hortres.2016.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gezan</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osorio</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitaker</span>  <span class="cit-name-given-names">V. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An experimental validation of genomic selection in octoploid strawberry.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">16070</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hortres.2016.70</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D4%26rft.spage%253D16070%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhortres.2016.70%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hortres.2016.70&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.16" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: Genomic selection in dairy cattle: Progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.17" data-doi="10.1186/s12864-018-4464-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finkers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parida</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Efficient algorithms for polyploid haplotype phasing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">110</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-4464-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D110%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-4464-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-4464-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.18" data-doi="10.1038/ng.2354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchsberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">959</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D955%26rft.epage%253D959%26rft.atitle%253DFast%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bin%2Bgenome-wide%2Bassociation%2Bstudies%2Bthrough%2Bpre-phasing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2354%26rft_id%253Dinfo%253Apmid%252F22820512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820512&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.19" data-doi="10.1093/bioinformatics/btt712"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">forqs: forward-in-time simulation of recombination, quantitative traits and selection.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">577</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt712</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt712%26rft_id%253Dinfo%253Apmid%252F24336146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt712&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24336146&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.20" data-doi="10.1093/jhered/esy006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meirmans</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Tienderen</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The analysis of polyploid genetic data.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esy006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D109%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesy006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esy006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.21" data-doi="10.1534/genetics.113.152181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SLiM: simulating evolution with selection and linkage.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152181</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152181%26rft_id%253Dinfo%253Apmid%252F23709637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NC80LzEwMzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.22" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzIvMzI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.23" data-doi="10.3389/fpls.2016.01509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naithani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Partipilo</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raja</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elser</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaiswal</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">FragariaCyc: A Metabolic Pathway Database for Woodland Strawberry Fragaria vesca.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.00242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2016.01509%26rft_id%253Dinfo%253Apmid%252F26858731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2016.01509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26858731&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.24" data-doi="10.1093/bioinformatics/btn179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Forward-time simulations of non-random mating populations using simuPOP.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1408</span>–<span class="cit-lpage">1409</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btn179</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn179%26rft_id%253Dinfo%253Apmid%252F18417488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn179&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18417488&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.25" data-doi="10.1093/bioinformatics/bti584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimmel</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">simuPOP: A forward-time population genetics simulation environment.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3686</span>–<span class="cit-lpage">3687</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti584%26rft_id%253Dinfo%253Apmid%252F16020469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16020469&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231694600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.26" data-doi="10.1534/genetics.116.194878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forneris</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evaluating Sequence-Based Genomic Prediction with an Efficient New Simulator.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">939</span>–<span class="cit-lpage">953</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.194878</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.194878%26rft_id%253Dinfo%253Apmid%252F27913617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA1LzIvOTM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-9.2.327.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2017</span> R: A Language and Environment for Statistical Computing.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.28" data-doi="10.1186/1471-2105-13-239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raineri</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferretti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esteve-Codina</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevado</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heath</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP calling by sequencing pooled samples.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-13-239</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DRaineri%26rft.auinit1%253DE.%26rft.volume%253D13%26rft.spage%253D239%26rft.epage%253D239%26rft.atitle%253DSNP%2Bcalling%2Bby%2Bsequencing%2Bpooled%2Bsamples.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-239%26rft_id%253Dinfo%253Apmid%252F22992255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22992255&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.29" data-doi="10.3835/plantgenome2016.03.0027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosyara</span>  <span class="cit-name-given-names">U. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Software for genome-wide association studies in autopolyploids and its application to potato.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.08.0073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.03.0027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.03.0027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.30" data-doi="10.1007/s00122-016-2845-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitz Carley</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombs</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bethke</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palta</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Automated tetraploid genotype calling by hierarchical clustering.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">726</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2845-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D717%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2845-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2845-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.31" data-doi="10.1038/srep24095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Shesisplus, a toolset for genetic studies on polyploid species.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">24095</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep24095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D24095%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep24095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep24095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.32" data-doi="10.3835/plantgenome2016.02.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improving genetic gain with genomic selection in autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.02.0021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.02.0021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.02.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.33" data-doi="10.1016/S0065-2660(08)60490-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stebbins</span>  <span class="cit-name-given-names">G. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1947</span> <span class="cit-article-title">Types of Polyploids: Their Classification and Significance.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">429</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0065-2660(08)60490-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bgenetics%26rft.stitle%253DAdv%2BGenet%26rft.aulast%253DStebbins%26rft.auinit1%253DG.%2BL.%26rft.volume%253D1%26rft.spage%253D403%26rft.epage%253D429%26rft.atitle%253DTypes%2Bof%2Bpolyploids%253B%2Btheir%2Bclassification%2Band%2Bsignificance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252808%252960490-3%26rft_id%253Dinfo%253Apmid%252F20259289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2660(08)60490-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20259289&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1947XT73400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.34" data-doi="10.1007/s00122-017-2944-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sverrisdóttir</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundmark</span>  <span class="cit-name-given-names">E. H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">H. Ø.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirk</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction of starch content and chipping quality in tetraploid potato using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2091</span>–<span class="cit-lpage">2108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2944-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D2091%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2944-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2944-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.35" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient Methods to Compute Genomic Predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-other" id="cit-9.2.327.36"><div class="cit-metadata unstructured">Voorrips, R. E. and G. Gort, 2018 <em>fitPoly: Genotype Calling for Bi-Allelic Marker Assays</em>. R package version 3.0.0.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.37" data-doi="10.1186/1471-2105-12-172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gort</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vosman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype calling in tetraploid species from bi-allelic marker data using mixture models.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-172</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D12%26rft.spage%253D172%26rft.epage%253D172%26rft.atitle%253DGenotype%2Bcalling%2Bin%2Btetraploid%2Bspecies%2Bfrom%2Bbi-allelic%2Bmarker%2Bdata%2Busing%2Bmixture%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-172%26rft_id%253Dinfo%253Apmid%252F21595880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21595880&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.327.38" data-doi="10.1186/1471-2105-13-248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maliepaard</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The simulation of meiosis in diploid and tetraploid organisms using various genetic models.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-13-248</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D248%26rft.epage%253D248%26rft.atitle%253DThe%2Bsimulation%2Bof%2Bmeiosis%2Bin%2Bdiploid%2Band%2Btetraploid%2Borganisms%2Busing%2Bvarious%2Bgenetic%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-248%26rft_id%253Dinfo%253Apmid%252F23013469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23013469&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/2/327.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/2/317" title="Genome-Wide Analysis of Mycoplasma dispar Provides Insights into Putative Virulence Factors and Phylogenetic Relationships" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/2/335" title="SNZ3 Encodes a PLP Synthase Involved in Thiamine Synthesis in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/2"><span>Volume 9 Issue 2, February 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/2.cover-source.jpg?itok=Pgd0L52l" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/327" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ou7T0VP5dPPGglDUZP_xsT2BdkyVI-y7MpEQsYYokpY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/2/327" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/2/327" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/2/327" class="active" data-icon-position="" data-hide-link-title="0">pSBVB: A Versatile Simulation Tool To Evaluate Genomic Selection in Polyploid Species</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492183" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ZPhM8VR-RZDdQ9cchrEBISFCWuRBRYfmlnJBi1S3lQo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/2/327.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/2/327" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-LhYzNG9936s618hIdU1PqHy6gLZw-Co7weXTn-dCzNc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/2/327" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/2/327.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492183" id="node492183--2" data-pisa="ggg;9/2/327" data-pisa-master="ggg;g3.118.200942" data-apath="/ggg/9/2/327.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">pSBVB: A Versatile Simulation Tool To Evaluate Genomic Selection in Polyploid Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5618-2630" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>María L. Zingaretti</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7106-7745" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amparo Monfort</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Miguel Pérez-Enciso</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">327-334;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200942 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492183--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">María L. Zingaretti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mar%C3%ADa%2BL.%2BZingaretti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zingaretti%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMar%25C3%25ADa%2BL.%2BZingaretti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5618-2630" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for María L. Zingaretti</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.zingaretti@cragenomica.es" class="" data-icon-position="" data-hide-link-title="0">laura.zingaretti@cragenomica.es</a>
<a href="mailto:miguel.perez@uab.es" class="" data-icon-position="" data-hide-link-title="0">miguel.perez@uab.es</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amparo Monfort </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>IRTA (Institut de Recerca i Tecnologia Agroalimentàries), Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amparo%2BMonfort%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monfort%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmparo%2BMonfort%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7106-7745" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amparo Monfort</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Miguel Pérez-Enciso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>ICREA, Passeig de Lluís Companys 23, 08010 Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BP%C3%A9rez-Enciso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Enciso%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiguel%2BP%25C3%25A9rez-Enciso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Miguel Pérez-Enciso</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.zingaretti@cragenomica.es" class="" data-icon-position="" data-hide-link-title="0">laura.zingaretti@cragenomica.es</a>
<a href="mailto:miguel.perez@uab.es" class="" data-icon-position="" data-hide-link-title="0">miguel.perez@uab.es</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/2/327" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/2/327&amp;title=pSBVB%3A%20A%20Versatile%20Simulation%20Tool%20To%20Evaluate%20Genomic%20Selection%20in%20Polyploid%20Species" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/2/327&amp;title=pSBVB%3A%20A%20Versatile%20Simulation%20Tool%20To%20Evaluate%20Genomic%20Selection%20in%20Polyploid%20Species" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/2/327&amp;title=pSBVB%3A%20A%20Versatile%20Simulation%20Tool%20To%20Evaluate%20Genomic%20Selection%20in%20Polyploid%20Species" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/2/327&amp;text=pSBVB%3A%20A%20Versatile%20Simulation%20Tool%20To%20Evaluate%20Genomic%20Selection%20in%20Polyploid%20Species" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/2/327&amp;title=pSBVB%3A%20A%20Versatile%20Simulation%20Tool%20To%20Evaluate%20Genomic%20Selection%20in%20Polyploid%20Species" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/2/327&amp;t=pSBVB%3A%20A%20Versatile%20Simulation%20Tool%20To%20Evaluate%20Genomic%20Selection%20in%20Polyploid%20Species" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/2/327&amp;title=pSBVB%3A%20A%20Versatile%20Simulation%20Tool%20To%20Evaluate%20Genomic%20Selection%20in%20Polyploid%20Species" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/2/327&amp;title=pSBVB%3A%20A%20Versatile%20Simulation%20Tool%20To%20Evaluate%20Genomic%20Selection%20in%20Polyploid%20Species" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492183/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492183" id="node492183--3" data-pisa="ggg;9/2/327" data-pisa-master="ggg;g3.118.200942" data-apath="/ggg/9/2/327.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">pSBVB: A Versatile Simulation Tool To Evaluate Genomic Selection in Polyploid Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5618-2630" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>María L. Zingaretti</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7106-7745" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amparo Monfort</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Miguel Pérez-Enciso</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">327-334;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200942 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492183--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">María L. Zingaretti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mar%C3%ADa%2BL.%2BZingaretti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zingaretti%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMar%25C3%25ADa%2BL.%2BZingaretti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5618-2630" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for María L. Zingaretti</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.zingaretti@cragenomica.es" class="" data-icon-position="" data-hide-link-title="0">laura.zingaretti@cragenomica.es</a>
<a href="mailto:miguel.perez@uab.es" class="" data-icon-position="" data-hide-link-title="0">miguel.perez@uab.es</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amparo Monfort </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>IRTA (Institut de Recerca i Tecnologia Agroalimentàries), Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amparo%2BMonfort%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monfort%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmparo%2BMonfort%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7106-7745" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amparo Monfort</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Miguel Pérez-Enciso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB Consortium, 08193 Bellaterra, Barcelona, Spain</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>ICREA, Passeig de Lluís Companys 23, 08010 Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BP%C3%A9rez-Enciso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Enciso%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiguel%2BP%25C3%25A9rez-Enciso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Miguel Pérez-Enciso</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.zingaretti@cragenomica.es" class="" data-icon-position="" data-hide-link-title="0">laura.zingaretti@cragenomica.es</a>
<a href="mailto:miguel.perez@uab.es" class="" data-icon-position="" data-hide-link-title="0">miguel.perez@uab.es</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=327&amp;rft.epage=334&amp;rft.atitle=pSBVB%3A%20A%20Versatile%20Simulation%20Tool%20To%20Evaluate%20Genomic%20Selection%20in%20Polyploid%20Species&amp;rft.volume=9&amp;rft.issue=2&amp;rft.date=2019-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zingaretti%2C+Mar%C3%ADa+L.&amp;rft.au=Monfort%2C+Amparo&amp;rft.au=P%C3%A9rez-Enciso%2C+Miguel"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492183/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492183/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492183/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492183/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492183/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492183/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492183/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492183/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492183/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492183/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492183/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492183/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495851" id="node495851" data-pisa="ggg;10/3/933" data-pisa-master="ggg;g3.119.400898" data-apath="/ggg/10/3/933.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/933" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Interactively AUDIT Your Growth Curves with a Suite of R Packages</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/2/327" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/2/327.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/2/327.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/2/327.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
