<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/2/601" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/2/601.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/2/601.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/2/601.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/2/601/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/2/601.atom" />
<meta name="HW.pisa" content="ggg;9/2/601" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Benchmarking Between Deep Learning, Support Vector Machine and Bayesian Threshold Best Linear Unbiased Prediction for Predicting Ordinal Traits in Plant Breeding" />
<meta name="DC.Identifier" content="10.1534/g3.118.200998" />
<meta name="DC.Date" content="2019-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Montesinos-López et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection is revolutionizing plant breeding. However, still lacking are better statistical models for ordinal phenotypes to improve the accuracy of the selection of candidate genotypes. For this reason, in this paper we explore the genomic based prediction performance of two popular machine learning methods: the Multi Layer Perceptron (MLP) and support vector machine (SVM) methods vs. the Bayesian threshold genomic best linear unbiased prediction (TGBLUP) model. We used the percentage of cases correctly classified (PCCC) as a metric to measure the prediction performance, and seven real data sets to evaluate the prediction accuracy, and found that the best predictions (in four out of the seven data sets) in terms of PCCC occurred under the TGLBUP model, while the worst occurred under the SVM method. Also, in general we found no statistical differences between using 1, 2 and 3 layers under the MLP models, which means that many times the conventional neuronal network model with only one layer is enough. However, although even that the TGBLUP model was better, we found that the predictions of MLP and SVM were very competitive with the advantage that the SVM was the most efficient in terms of the computational time required." />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Javier Martín-Vallejo" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Daniel Gianola" />
<meta name="DC.Contributor" content="Carlos M. Hernández-Suárez" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="Philomin Juliana" />
<meta name="DC.Contributor" content="Ravi Singh" />
<meta name="article:published_time" content="2019-02-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="A Benchmarking Between Deep Learning, Support Vector Machine and Bayesian Threshold Best Linear Unbiased Prediction for Predicting Ordinal Traits in Plant Breeding" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection is revolutionizing plant breeding. However, still lacking are better statistical models for ordinal phenotypes to improve the accuracy of the selection of candidate genotypes. For this reason, in this paper we explore the genomic based prediction performance of two popular machine learning methods: the Multi Layer Perceptron (MLP) and support vector machine (SVM) methods &lt;i&gt;vs.&lt;/i&gt; the Bayesian threshold genomic best linear unbiased prediction (TGBLUP) model. We used the percentage of cases correctly classified (PCCC) as a metric to measure the prediction performance, and seven real data sets to evaluate the prediction accuracy, and found that the best predictions (in four out of the seven data sets) in terms of PCCC occurred under the TGLBUP model, while the worst occurred under the SVM method. Also, in general we found no statistical differences between using 1, 2 and 3 layers under the MLP models, which means that many times the conventional neuronal network model with only one layer is enough. However, although even that the TGBLUP model was better, we found that the predictions of MLP and SVM were very competitive with the advantage that the SVM was the most efficient in terms of the computational time required.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/02/01" />
<meta name="citation_mjid" content="ggg;9/2/601" />
<meta name="citation_id" content="9/2/601" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/2/601" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/2/601.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/2/601.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/2/601.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200998" />
<meta name="citation_pmid" content="30593512" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="601" />
<meta name="citation_lastpage" content="618" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática" />
<meta name="citation_author" content="Javier Martín-Vallejo" />
<meta name="citation_author_institution" content="Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="aml_uach2004@hotmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Daniel Gianola" />
<meta name="citation_author_institution" content="Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Carlos M. Hernández-Suárez" />
<meta name="citation_author_institution" content="Facultad de Ciencias, Universidad de Colima, Colima, Colima, 28040, México" />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Guadalajara, Jalisco, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="aml_uach2004@hotmail.com" />
<meta name="citation_author" content="Philomin Juliana" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México" />
<meta name="citation_author" content="Ravi Singh" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Albrecht;citation_author=V. Wimmer;citation_author=H. Auinger;citation_author=M. Erbe;citation_author=C. Knaak;citation_title=Genome-based prediction of testcross values in maize.;citation_pages=339-350;citation_volume=123;citation_year=2011;citation_issue=2;citation_pmid=21505832;citation_doi=10.1007/s00122-011-1587-7" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=B. Alipanahi;citation_author=A. Delong;citation_author=M. T. Weirauch;citation_author=B. J. Frey;citation_title=Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning.;citation_pages=831-838;citation_volume=33;citation_year=2015;citation_pmid=26213851;citation_doi=10.1038/nbt.3300" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=C. Angermueller;citation_author=T. Pärnamaa;citation_author=L. Parts;citation_author=O. Stegle;citation_title=Deep learning for computational biology;citation_pages=878-878;citation_volume=12;citation_year=2016;citation_issue=7;citation_pmid=27474269;citation_doi=10.15252/msb.20156651" />
<meta name="citation_reference" content="citation_title=Data Mining for the Social Sciences: An Introduction;citation_author=P. Attewell;citation_author=D. B. Monaghan;citation_author=D. Kwong;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Bellot;citation_author=G. de los Campos;citation_author=M. Pérez-Enciso;citation_title=Can Deep Learning Improve Genomic Prediction of Complex Human Traits?;citation_volume=210;citation_year=2018;citation_pmid=30171033;citation_doi=10.1534/genetics.118.301298" />
<meta name="citation_reference" content="citation_journal_title=Front. Comput. Neurosci.;citation_author=A. S. Benjamin;citation_author=H. L. Fernandes;citation_author=T. Tomlinson;citation_author=P. Ramkumar;citation_author=C. VerSteeg;citation_title=Modern Machine Learning as a Benchmark for Fitting Neural Responses.;citation_pages=56;citation_volume=12;citation_year=2018;citation_doi=10.3389/fncom.2018.00056" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=U. Beukert;citation_author=Z. Li;citation_author=G. Liu;citation_author=Y. Zhao;citation_author=N. Ramachandra;citation_title=Genome-based identification of heterotic patterns in rice.;citation_pages=22;citation_volume=10;citation_year=2017;citation_doi=10.1186/s12284-017-0163-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: Software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_author=H. Byun;citation_author=S. W. Lee;citation_year=2002;citation_doi=10.1007/3-540-45665-1_17" />
<meta name="citation_reference" content="citation_title=Deep Learning with R. Manning Publications, Manning Early Access Program;citation_author=F. Chollet;citation_author=J. J. Allaire;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=M. A. Cleveland;citation_author=J. M. Hickey;citation_title=Practical implementation of cost-effective genomic selection in commercial pig breeding using imputation.;citation_volume=91;citation_year=2013;citation_pmid=23736050;citation_doi=10.2527/jas.2013-6270" />
<meta name="citation_reference" content="de los Campos, G., and P. Pérez-Rodríguez, 2014 Bayesian Generalized Linear Regression. R package version 1.0.4. http://CRAN.R-project.org/package=BGLR." />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_title=The State of the World’s Land and Water Resources for Food and Agriculture: Managing Systems at Risk;citation_year=2011" />
<meta name="citation_reference" content="citation_title=Crop Yields and Global Food Security;citation_author=T. Fischer;citation_author=D. Byerlee;citation_author=G. Edmeades;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. Gianola;citation_title=Theory and Analysis of Threshold Characters;citation_pages=1079-1096;citation_volume=54;citation_year=1982;citation_issue=5;citation_doi=10.2527/jas1982.5451079x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS : A High Capacity Genotyping by Sequencing Analysis Pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. M. González-Camacho;citation_author=L. Ornella;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=S. Dreisigacker;citation_title=Applications of Machine Learning Methods to Genomic Selection in Breeding Wheat for Rust Resistance.;citation_pages=1-15;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.11.0104" />
<meta name="citation_reference" content="citation_title=Deep Learning with Keras: Implementing deep learning models and neural networks with the power of Python;citation_author=A. Gulli;citation_author=P. Sujit;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with genomic selection: gain per unit time and cost.;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_title=An Introduction to Statistical Learning: With Applications in R;citation_author=G. James;citation_author=D. Witten;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_year=2013;citation_doi=10.1007/978-1-4614-7138-7" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=D. Jarquín;citation_author=C. Lemes da Silva;citation_author=R. C. Gaynor;citation_author=J. Poland;citation_author=A. R. Fritz;citation_title=Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.12.0130" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Juliana;citation_author=R. P. Singh;citation_author=J. Poland;citation_author=S. Mondal;citation_author=J. Crossa;citation_title=Prospects and challenges of applied genomic selection-a new paradigm in breeding for grain yield in bread wheat.;citation_pages=1-17;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.11.0104" />
<meta name="citation_reference" content="citation_author=D.C. Kadam;citation_author=S.M. Potts;citation_author=M.O. Bohn;citation_author=A.E. Lipka;citation_author=A.J. Lorenz;citation_year=2016;citation_pmid=27646704;citation_doi=10.1534/g3.116.031286" />
<meta name="citation_reference" content="citation_journal_title=BMC Proc;citation_author=K. Kizilkaya;citation_author=R. G. Tait;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_author=J. M. Reecy;citation_title=Whole genome analysis of infectious bovine kerato conjunctivitis in Angus cattle using Bayesian threshold models;citation_pages=S22;citation_volume=5;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=M. Lillehammer;citation_author=T. H. Meuwissen;citation_author=A. K. Sonesson;citation_title=Genomic selection for two traits in a maternal pig breeding scheme.;citation_volume=91;citation_year=2013;citation_pmid=23658351;citation_doi=10.2527/jas.2012-5113" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Makridakis;citation_author=E. Spiliotis;citation_author=V. Assimakopoulos;citation_title=Statistical and Machine Learning forecasting methods: Concerns and ways forward.;citation_pages=e0194889;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0194889" />
<meta name="citation_reference" content="citation_author=D. Money;citation_author=K. Gardner;citation_author=Z. Migicovsky;citation_author=H. Schwaninger;citation_author=G. Zhong;citation_author=S. Myles;citation_year=2015;citation_pmid=26377960;citation_doi=10.1534/g3.115.021667" />
<meta name="citation_reference" content="citation_author=A. Montesinos-López;citation_author=O.A. Montesinos-López;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=C.M. Hernández-Suárez;citation_year=2018;citation_pmid=30291107;citation_doi=10.1534/g3.118.200740" />
<meta name="citation_reference" content="Montesinos-López, O. A. Martín-Vallejo, J., Crossa, J. Gianola, D. Hernández-Suárez, C. M. Montesinos-López, A. Juliana, P. Singh, R., 2018c New deep learning genomic prediction model for multi-traits with mixed binary, ordinal, and continuous phenotypes. Manuscript submitted for publication." />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=A. Nakaya;citation_author=S. N. Isobe;citation_title=Will genomic selection be a practical method for plant breeding?;citation_volume=110;citation_year=2012;citation_pmid=22645117;citation_doi=10.1093/aob/mcs109" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=L. Ornella;citation_author=S. Singh;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_author=R. Singh;citation_title=Genomic prediction of genetic values for resistance to wheat rusts.;citation_pages=136-148;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.07.0017" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=L. Ornella;citation_author=P. Pérez-Rodríguez;citation_author=E. Tapia;citation_author=J. M. González-Camacho;citation_author=J. Burgueño;citation_title=Genomic-enabled prediction with classification algorithms.;citation_pages=616-626;citation_volume=112;citation_year=2014;citation_doi=10.1038/hdy.2013.144" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Agron.;citation_author=F.-X. Oury;citation_author=C. Godin;citation_author=A. Mailliard;citation_author=A. Chassin;citation_author=O. Gardet;citation_title=A study of genetic progress due to selection reveals a negative effect of climate change on bread wheat yield in France.;citation_pages=28-38;citation_volume=40;citation_year=2012;citation_doi=10.1016/j.eja.2012.02.007" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=X. Pan;citation_author=H.-B. Shen;citation_title=RNA-protein binding motifs mining with a new hybrid deep learning based cross-domain knowledge integration approach.;citation_pages=136;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12859-017-1561-8" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Quang;citation_author=Y. Chen;citation_author=X. Xie;citation_title=Dann: a deep learning approach for annotating the pathogenicity of genetic variants.;citation_volume=31;citation_year=2015;citation_pmid=25338716;citation_doi=10.1093/bioinformatics/btu703" />
<meta name="citation_reference" content="R Core Team, (2018). R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna. Austria. ISBN 3–900051–07–0. URL http://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=I. Simko;citation_author=H.-P. Piepho;citation_title=Combining phenotypic data from ordinal rating scales in multiple plant experiments.;citation_pages=235-237;citation_volume=16;citation_year=2011;citation_issue=5;citation_pmid=21367649;citation_doi=10.1016/j.tplants.2011.02.001" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. A. Sorensen;citation_author=S. Andersen;citation_author=D. Gianola;citation_author=I. Korsgaard;citation_title=Bayesian inference in threshold mode ls using Gibbs sampling.;citation_pages=229-249;citation_volume=27;citation_year=1995;citation_doi=10.1051/gse:19950303" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=N. Srivastava;citation_author=G. Hinton;citation_author=A. Krizhevsky;citation_author=I. Sutskever;citation_author=R. Salakhutdinov;citation_title=Dropout: A Simple Way to Prevent Neural Networks from Overfitting.;citation_pages=1929-1958;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_title=The Nature of Statistical Learning Theory;citation_author=V. Vapnik;citation_year=1995;citation_doi=10.1007/978-1-4757-2440-0" />
<meta name="citation_reference" content="citation_journal_title=Crop J.;citation_author=X. Wang;citation_author=Y. Xua;citation_author=Z. Hu;citation_author=C. Hu;citation_title=Genomic selection methods for crop improvement: Current status and prospects.;citation_pages=330-340;citation_volume=6;citation_year=2018;citation_doi=10.1016/j.cj.2018.03.001" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Zhang;citation_author=J. Zhou;citation_author=H. Hu;citation_author=H. Gong;citation_author=L. Chen;citation_title=A deep learning framework for modeling structural features of RNA-binding protein targets.;citation_volume=44;citation_year=2016;citation_pmid=26467480;citation_doi=10.1093/nar/gkv1025" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Benchmarking Between Deep Learning, Support Vector Machine and Bayesian Threshold Best Linear Unbiased Prediction for Predicting Ordinal Traits in Plant Breeding" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection is revolutionizing plant breeding. However, still lacking are better statistical models for ordinal phenotypes to improve the accuracy of the selection of candidate genotypes. For this reason, in this paper we explore the genomic based prediction performance of two popular machine learning methods: the Multi Layer Perceptron (MLP) and support vector machine (SVM) methods vs. the Bayesian threshold genomic best linear unbiased prediction (TGBLUP) model. We used the percentage of cases correctly classified (PCCC) as a metric to measure the prediction performance, and seven real data sets to evaluate the prediction accuracy, and found that the best predictions (in four out of the seven data sets) in terms of PCCC occurred under the TGLBUP model, while the worst occurred under the SVM method. Also, in general we found no statistical differences between using 1, 2 and 3 layers under the MLP models, which means that many times the conventional neuronal network model with only one layer is enough. However, although even that the TGBLUP model was better, we found that the predictions of MLP and SVM were very competitive with the advantage that the SVM was the most efficient in terms of the computational time required." />
<meta name="og-title" property="og:title" content="A Benchmarking Between Deep Learning, Support Vector Machine and Bayesian Threshold Best Linear Unbiased Prediction for Predicting Ordinal Traits in Plant Breeding" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/2/601" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection is revolutionizing plant breeding. However, still lacking are better statistical models for ordinal phenotypes to improve the accuracy of the selection of candidate genotypes. For this reason, in this paper we explore the genomic based prediction performance of two popular machine learning methods: the Multi Layer Perceptron (MLP) and support vector machine (SVM) methods vs. the Bayesian threshold genomic best linear unbiased prediction (TGBLUP) model. We used the percentage of cases correctly classified (PCCC) as a metric to measure the prediction performance, and seven real data sets to evaluate the prediction accuracy, and found that the best predictions (in four out of the seven data sets) in terms of PCCC occurred under the TGLBUP model, while the worst occurred under the SVM method. Also, in general we found no statistical differences between using 1, 2 and 3 layers under the MLP models, which means that many times the conventional neuronal network model with only one layer is enough. However, although even that the TGBLUP model was better, we found that the predictions of MLP and SVM were very competitive with the advantage that the SVM was the most efficient in terms of the computational time required." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2.cover-source.jpg" />
    <title>A Benchmarking Between Deep Learning, Support Vector Machine and Bayesian Threshold Best Linear Unbiased Prediction for Predicting Ordinal Traits in Plant Breeding | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"K924JxkWSUVv0VjpEpZFA2QD4qEupT8kkDGDAPXv02M","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492153","apath":"\/ggg\/9\/2\/601.atom","pisa":"ggg;9\/2\/601","ac":{"\/ggg\/9\/2\/601.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/2\/601.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/2\/601"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/2\/601"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/2\/601.abstract":true,"\/content\/9\/2\/601":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Tdpd9WweypYlpkUMVUkW3VSr+jbRZ+vZ2lN7VIQirsmBhUXq2NCH+Xp+CVDv8onubu8LPHdFQY3o43wwN+8qBeT62J1ub2yXvkZLFQm57nD2MGQmDaNCh45c7n\\\\\\\/\\\\\\\/WIDQIohR7ADIMeQCJSGyrMY7sLZm4HRuFXv5m5zEFVtaPkKB4bjsEyMYBuyarbBnEu0Sn8lR9qeAkfmR+vO3f+A6d5i0C7jlt189n2jXkVof1WdtoF9CQd9dZKA4uSyryYhSrJuIoW7SpamRqQ82MBsGp3jtml6UgnOOie3PYkyoXQZjI4u9iSjeiY8gAEGlozUenZD9Zjr2ZNFU44\\\\\\\/Ylaim\\\\\\\/yfhNAeGG9Obg\\\\\\\/A1U6HkV844p8vEbmc3AbZhBQm5ApEeE9LqwBKB8g9Esa6nOhKkbe13YnT01nMer7pVG9Pdc2MFnBsFV9WbhaR\\\\\\\/Es4NlX4024SKHQ6PKmbaVLKNt5CYHTsfqAPKyJOWl\\\\\\\/OVjzLjnEiXMuyu7wY9zYl3DYj\\\\\\\/ABs2YnXMUnQNApAlfLooxsMaSXR+deorsAnDl7GdmndGtujfri\\\\\\\/8kajE745XPB9dhTJuOq2rm9ISF\\\\\\\/DzgAmg2HfMtpr4zepeJIpLOVF0quzk3u8YT36XF7K1M\\\\\\\/7nebc4P4Zx8KTfello+tCXOgp680J6OrwXK11HPe0YtRMhKK1PWyZHr+VrI10lEoweiDxDfaqziKjsJnoprpkpC9r6y604XEM8HUtYHvxGzXdGZTj1MVQe33cntdCOA4ItAYXvW4jxp0mubzIix0QoqxGqbLpYb0D\\\\\\\/19zG3HyZhSp+fI0T3LXV+6XC+G447pTT8cBDGOk2m2DX6WDlRUmjTQCfmxvQggumLVfOKzkbQdezXbXd+6GRiQVCpGuoUxF3pyjihLZD4QcNIlx2yRDZ3tLcVONira8oc24v+ugiFXqJu6O6litQJeK\\\\\\\/rkMk1KEy1\\\\\\\/XcYk1oLBEGXUdVbjZYchZfBgjmVw2kJ8Ik0hzLodZiAglT7KxJrYZXn6mZwTmaWHqcwkVUlVzXJPNpmonQs6kYkF+XMk\\\\\\\/GlF78+wjTZb7lLsP06MZKwAXOndNvCBo0UuyWPL3k\\\\\\\/fFBvnal+UvrJCJkTGWlQclVmWODNWNH90KoZa4K3rITnNKy0BaKE360B7IPVwFDQKsV8Xv\\\\\\\/moBicMdkqMrE6FcDqyU6tfLQmZ9YBS4wyKxNqBvXnti\\\\\\\/YqBSO1jDjsqzqmra8UddNtobGJWUHM\\\\\\\/auueNEFz07xcSYJj6b5qBrRYy4xffZaZPbPrFn0xLuYqD\\\u0022,\\\u0022iv\\\u0022:\\\u0022atAIkSHAtmbgaXfAwTHGzw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c2bd9371735bf1d4af2dd62bfee40df705beae8f8d76b2a4f6f109dea4bef216\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BBUmzhv4vFC33h4nLwdgufWFz\\\\\\\/S9lML44Aee1lxHldRjlWUBKb496QQ2kvKRNDFWc4jXXgFjoedD0eGct702AOitl1lmsWpQIKW4UtSQSoQamd8LyhcTkWQHkAlhGNt9IVA3JIEYwHKlGw\\\\\\\/QCo3+Q9eCkZ4AEvbZIpx3wkLKDjanUTLLs40AWcoLFla2QwCB7KncPbPiAVk+g7Z46kHK5OAfNTAN5x2MHaoVZEpBZDFT7HUKEL\\\\\\\/GFGab1CZzRCkMG8PINUW6X15j7mYWDrCszBTQuTYlK1Y2menibWKrkME7jPbFZrxE7ZibaMxM2kvIWRj27pMbRZIYObJ1F1AAJpfu5ZSv8xcrwLUyj1eSm6UBpYkMk3\\\\\\\/3OXJX43MSUz82f87d6vWGEfvetIrqy8trc+10sPgwx\\\\\\\/ySdZNfMRuyDl8Qy8yauzVdpvzBYK2mSav3aam9RNz3GPp97H5isMIIhVO4KRsmq0Et2j9fqHnRYqPAmoc+m\\\\\\\/flCPVJWgFlOuL2WlymgJD2r2hiWX4jMYOKqIUb69QAQr\\\\\\\/LQX3xQfBDnHuuNhEZkTa27y\\\\\\\/0u0\\\\\\\/NE0Pc+3jHnm24yO+YqMjyCCijqmNTpakdWnheKCb55s37KF1gtam2hdzhCRO+lsfwHqMfbzI7NM4xZ37Drmip6KqpqP1ftPXVmWomVmpGncebmmkjIicQ+g7NkG2dfU0ZKTCvLJ7Jj+euNh\\\\\\\/S158sdk0RMdpPdOrXTeUet3xC\\\\\\\/6oWerpM1M7a218SBaz81m6E344hyiwsKcbU1XKY+gXA+aPMDeURwl2RnGFCNHCYTreeKzN0n0I7FEjLzYpcuzYxYyVNc4gEp8IgAJE7IABbeNo\\\\\\\/KF9BgcRdjl53aO34V68xVYZpQGsZmUaNeKOl4oN7enH0PtnVl63KQNkHGuQpUQ54m4+pvz4ppgtgSlC7b0t2X1XrB0BBydfNueTo38vp7uyOZmYLmg6GeCYoDCq1yhVFwltpkfrsIg6O\\\\\\\/7qQ6yEYgjuk1z9PHVtNLWTWzcD2V0dF7sqn\\\\\\\/Nhi4Igv\\\\\\\/Rk9YX2RxENuflUt14dzXSiVjrr7MRj31fQFUDxmbrPK3+tKv8skFmNF28UVWPGRghSO45CxxjDvKnMaH4qbkkCS76retR+eQf6A0\\\\\\\/D0jHSUOoMBzxhUt50Wbxqxb1BDme2iw4+tIeJsNorjg2mLe5Pt2celG6ry\\\\\\\/vW2og8+I5mMVBEESL9zKKsYXEWZ34iXfgH+jCWNx+1YdN+kLgZ2ER6t+8TX9EnK6ttF7Umggt4itJA6HxylleA+HxiP6JSlW5dkfAsd9P5WoF8w71Mf+7Tlbtv1EIgqnpq\\\\\\\/uM9+XaMvw1sn2aM+1PKqmrf5T9d6a8uO+0drv03tdMAlUlBaiz5muF\\\\\\\/G2sq7ElKpA9PWVrzPojwk5CD9c1WkDJavkzF5SL3sHpJFAb4OvhgJL3sP5QfRIZZZvjajTMRz7s6pfeHfQGbv67tv0J7hO4cIxAYUdAlVOwKNoa70N63fOqBJuaAKJsh2aQ5l6rb44p74BvWYOB\\\\\\\/Rk3xyVQJGmo3MR2GIFHYgQwz9utPTbo2FyIVsKN\\\\\\\/ZqO8+8wT2M7jL+ul3SAJke772U3D13LNil7vCll4pbB3gpiJ+r9oYPjXiOHxvQoNEeVQ9NCJqKtP1vqKgch5AHpg47jkvKPMeDnOVk+Uxm5JtCuNyC16nwETtJv4B2zk0IpUCUFxHVOX33dGDNyrnvsvhkGtrhVuE28PsnVrIa4HpvhwBdZ9\\\\\\\/du8FSO9y8zRcXsAfs3kx0hjFPNtAOe\\\\\\\/bRilGBelZLUBsjr7Omsi0Zhye3tRMWTbjKo+0+Yp4yIU1jEchjXtfEKkNpf+OBgmqXYYIq1j3qK9jIIGEAnMSl6tWoMAfEAQT858aP4hDC00Nu+qYRVEbF5UdJT2zwYsl7KrriRDV6nf21ZN3Em74e4N6ksec2O0RHwIWQpBGE56y+TEtU4kIkKTMLZQe5NPG+LZ3g7xYzUejvG\\\\\\\/CdnDQd3Wj6Mp+hIl4j\\\\\\\/KPLxDVq5y06fk0Yhy6L6bcBQnVrzPXj+8lqa6SEZxYqC6DOjB3zDOU0j\\\\\\\/jdYaDNFouNRhac5RAbZkZr\\\\\\\/d6lnFa8dEpQUiLuarzyu+yQvzXFxSn+gO3oYxV4JSXVRxKVtX71ZW7e60EuW4fegzO3xfEHx7\\\\\\\/WK\\\\\\\/hybyceaj6JN1PC1lWeOuthYoG4nnnw+C0yDcgzxiyvD67KpIBRRhqAkE9J7AEfeCpx8dmpjuxQr+ZJqvZhlss+pMbsjfMZTAKu65bpds\\\\\\\/BqzEIX0mgAh4UXg1kD5nNd6shV+c8DvUjWkqsHRTYiO5BUt1e8d4juKD3GwkSHPoAlSJ4D9g+aADQOzEyXJz3AVqbCyx8G9udn7D66nKeTABNA+ltqsGbmMGZhJqrjEcM9I7m7MeFMQNDTkvAtXtR4p\\\\\\\/COBqxJULTK\\\\\\\/iLvfNCwMUzH1XIlsliNSUDBCftO4VHXkP6Z1uxjmVzDNxpJVRB8ote4MqELW7f+pyLOJ\\\\\\\/Zh4PSbHBVpb45AX4H9mXycrfZOBLM3nYSBZrE0wGvSJpe0XUZfR708IGMOucRIxrgqBM6acoN6h0MeO290B4bdQ9Te0T8pySljhsZM3jL\\\\\\\/E3aq0BZiCfMMPWE4FTGJQsMs7zjQu1U4jNsfxiJqRIiWjuJlc75WNkYEbqoGDtLYyInoAFCB037qudMHnLq0d2zlmjmjVCNgLrCrgT8BFGZ5HapSdnIhO4ma6kt+5MmUfESQbHbYtZeGl7LhRG0pkW1DsSPzmO1HlAO4arZ1esoaON9glqzbwASl4C98sic2Tfx3f9tH+XRLI4LR5bXbQ3Mlx50jH9GSzO9BzuDxh472BloRz2SiSNjWhXzKUyf8Jo04GmIs2qh80PBPYmJNBZVyjqBV\\\\\\\/7GCaNuL8d51C+Ke\\\\\\\/hJEK4YjDZylYiyDhDKblG+4JMjDpd5QegvdU5lWgMTov+00fnBN87FVl34a3tOgrtqFRXMehdi1ZCyZ51j3rci3Emk1MBHwVF97c8wzO9pFo9Lgp9HiG8SOmMCpq1l5Hb+MC0vpliZAqvYcA769efI47fz6B5W8HvI3uIUbATh0EahdNPCCBC4EPQsIhfen6qO1QfDTjc9Y2s08pd4gMWWZk4DT7zk9+WEDRULD+VY5qyjTcRgez4Vj5u9II9IMLw1gpcvKgxdSfeZsZFvQ49ZnlKtfbhPvOF57zIuEKcr3xZXYSNGd5WbK2X9l5QGcFokyFB46qVwfe0yPDTEkyxyCw98qOBsvMS6Aw6URrztn2e7H9EgE6g4N1YMhDlKvY5I1PQdSUa851TSEefqeqo5Iy8zwxjAGSjQUQ+uZWwrgIjjJGpA6ixcfKdKeT\\\\\\\/ZynDTXHwgYKVtvdYQgyBEp1sQ8nNQNiIS47diAv4ldSe892D2chLgF+4NZx0qRJInLWBc3s4kE5MdYDTYTpSjmT9pys7pugPvlqS4SjsT\\\\\\\/3T8C4SqH+aPgB9F2De8cDIHDMzt7QL2NB5gESvuuuVYJ91mJ0OTxXcrgdXHghzrPU4gdfaNM6Cj6AWvDaA6vd5QI8uuMYSwHlUTbtGX1Xs87j4AqIiF2EPg4saQsx8plYfJqJqf1y8YqFxRhkktjJJ68eN4o4PHCJHu\\\\\\\/zSS75cL21OmsGBg2f1JbKzbWyp0\\\\\\\/cmEy4M1jPko+\\\\\\\/Vw1JXhthFqbBIP55cmUoi7StVWjuN1\\\\\\\/jYJ1YWyudMjtXMQ\\\\\\\/ZEynFK\\\\\\\/GBaMKSi8E3FX5oVwGOqAAZGUlVveX1y0wMl3Tg0QTgBbzhDhEnvil\\\\\\\/3wZkQ9e1PAnEUIMv1aKcN7zM+aRhbelZbaAB5xTRkzRAh8btql6HwVsMMsJyV44sNrP7f1IuKiYOsj0XSbOU+4xBXB\\\\\\\/I4FujQWTwoYab1974T3seQMueztk9jrVUkJx\\\\\\\/hnDyb+4SSs5UAD06WheryvYPjnJEIX9BnDaMiFUn\\\\\\\/M0eCYvevv68T84KbwdgE+2ZHKyhN7EKA\\\\\\\/ftzJY72wGQscI\\\\\\\/8F5XR33OLnbeEgxTx6RvaKJNjx1XH1vIyt7e2ReP5TnLkwh9jFGRr5aKunrLLEE9m2M4N6+KuWhV4UoFHeNhNmGrCz5vH6osin0USJNLrlOWuJYBCFq2f2qH2J5YPWyR3B6COikxj2cuqmwa2DFeEzRLEB9yG15waGIyCYGTJiVaChcJxpIwhxSTZbCKW4QLeJb4geZ0tmobISbkV7K5kjvpVJAg8EDbGDnyV+N2k946VhmPK4VNEo2c9iauM8vCSa+Cm9QmNcA6Xh3nsUDXMLRRuUlZQuWEV\\\\\\\/q5n406ptOrjswPm11KkxVHf\\\\\\\/gpi6ajEf+qnPcABmwlSm5o+KFkdQbR5vgGYPM9s6x5fyiTuIQzwdJ\\\\\\\/xrkXDt7S6XRP9bZSVFti8AXu\\\\\\\/TaVL6qzV\\\\\\\/c7LmB2gEtFKLM2OG7Kl6UdjhJ1W3po2yT4s4dKIvBQdjMVt2U0LlvbwkbyGa2u3mIssbIuvMS3aDqKjZhY040zMUtdbNVVZYMM34w8LUR1eUBKxLA+lwRdqZt9ukww9evkEbJ2yltV8ipnEvHI+xPfibS1d7xhLaYW2EYwSfq5BPELRNgwBxi2efjxe+4oemyGMc\\\\\\\/5+uT8FeWK0dxJkqPUnJyHlivTnPzge6gF\\\\\\\/cJNRJzOI5oTyt4U1V3fCTezZRbKUQw4zawhkSuhytCFNeTYPJpCF9hq6gCKG0iHaltmSqEJgnGndeb9hmvyd5IEXj7xoczxSuIqvqZW7E2ZT222BOAKq87eHu2NRMq83IDy4LZj\\\\\\\/rUhL5Q58GGIheei8CDSj7BXTKp99bEW7KI5GpdMaQiIgsYe\\\\\\\/S6ukJc8r\\\\\\\/iEVy9PsyzrcHLHV0SpEA7tLtxa9AkFuWdBWOMbDox070LoBeVYFrMPWPFinDp4ksverhyJXr7PTb+XdfUn7VQ4diQio7xq6M0gvogFYeMsyM06NQEbiPkHkOrqXzxS9Kd\\\\\\\/oQs+raT1dHBkZLXeTwPZR4ygpr492ymo4AV9okA98WghVtsuC78j1VZohN9OiGvv8FIxEpowCwwQo+aIjcWobvmlBRK4Ym+cN1UkpZkDpxgT2Y9oo2+Vdo+0KzNDk06rvr39xalj88+jK+OHp45Np6HL1OngxrMO0y2GqiAFuDY+Ww5oT7yaxqXxDUnM7vlA6XkxaYR7ReaGXxClFaQUlOzQt+zwuuAeA4aMBL87UmJZCSZEM4t+YwbDdtA8yiy2rFx\\\\\\\/345NJz0Q1TSUZcmaPVydYeVd5XO8nruywKtFuEpUQVnouplgjHGcoNY6XXMhFQeW\\\\\\\/R+RWaLxQhtCbwdn3433G+VyGR+695tyn3+MUsNLarOKaF7cxM5h02BTkPl7+eD5Y3kELApSxy9lAduXGa7l12W8ijQYJTwfsYej8QKCnErqmGQiweUm1nxFfLOMHnQhCwQCm+HaFKe9g0GqsKJu36s5dXSzislWXwXyR+PpK4Ju7d3FBViRNNWNdSWzsB7HyWqGrPSrbAt3RRSojiFvENt4vBSXvH\\\\\\\/cC6Uc+7MGhR9Cph\\\\\\\/AC74GFn6NNsZCQ6qb\\\\\\\/gW4pukfv99B4DwDGCMwr3J1vylVXzo35EYchfGMvz45AoU4+wZOHV+Zvw8z7+8UC0mAiUMs9\\\\\\\/k7Wyo2QBROrTmj7HkaaZ0xApLzpDehSHM54LuclYnncl6ZmymiR5aJH5RBXZY+V1WQvUZYB5aZivUkZO9HKWtra8rdx6pr5zVb3iFVtEma4UUK5Es2eZFxLBOXGk85gSlIfN1ePMib8dnRN3xc\\\\\\\/od3fyZokRxwXUZ5pg2wiyHBOLpsih3HiOQQrd9cENd3WrdAaBfg54YKzR3FNSr8OyXid\\\\\\\/x+PsRBWts1Qaah3a4\\\\\\\/3SK6Wb0aUEtVr8AEKDaAlaFNXoaaNZnmE\\\\\\\/p9hsUSIl1bdE1rRtE4k3b0iK8by5AL65\\\\\\\/HnbxyTtcJAGdzH8FQTkvLg4cHoKhqc5\\\\\\\/RwAsBcW5YJDvyvGtgsbSTNkTyR+xTyzlwSTXatIxv46lrcYIb8R1u5\\\\\\\/7ilQZ\\\\\\\/LqLEQGDuJikhDmIVdekIzx9YVoZuJna2XFGXfT2qwxdE2avjZIOwgInqmyR804gWoSqEMLMkZYTrx8\\\\\\\/fkq49\\\\\\\/pIbig+c5S45smOPVYsgrt\\\\\\\/EjyR1ptI0Cr6pMXNXanWv0bcsax+qGTLNSQKzhMXxIa8OXO9YLCJbJK2NceNEDZBycrabq1VyjdK+SNLprc3y4NGIgpI5KWX9RIbpjk7lqsvFR3\\\\\\\/CtwfOrk3N0\\\\\\\/I1TMjTa2xhP8RdWIQfMHRM9MqItEhAHUiPB+AnsHAhqUgoVRWxyML2KTOWgZAJd1sOxME2x+1yzsNvO2iu3TwdvmU\\\\\\\/DPokN0RNLVb6LnA1kPnwW8KiT3pCQYcDxxk7VxhLBycLY7vEktnKaI6jHPB2FQsCgu\\\\\\\/A0WPj4bx0Jv\\\\\\\/jmKSacfzmv4e\\\\\\\/mT2FO4JMUaHQ9u4nuzDgygshPzpxeOmUk9XfLmqrjOVtHxT8ofqaPanMsxC3PLndn8dIpJA5yPr3xNI6UbK4RAUw3c6u8xkKRC\\\\\\\/554t4M20psLMe8Tg0GHiNIBlxe\\\\\\\/v7j8UtlByTR9QlOeuAnQp6HwKamURupPszgQ3zeYsCHk5mR33ArD2v3JIJbaCwBqQifbcnOhjrbEELFB3xSjy9lFn1tAsU3alwSf1fjHKiWcCW\\\\\\\/zYIt9X\\\\\\\/+HNg0SbACMUuKqBSheEO+fqA\\\\\\\/WgOkQO9hS83DOdI6Lk67lzLlBJeWlQoH9GjAsta\\\\\\\/xxp10lyYaw2aS74z6owc4FEA8c9lBQTtYPtdbq172p4j4Ur1Dj1SGbRYVrekzh0HgM6x3AqngZDz6TCHvaBjz4qWoEsYeC1SFpBE7dYxJFVgXmY6PIEYo1EyQ2ml2H6N6GNjLYh3XXMGOy98nULZ2rlPhoqhZetu2YxHv8FLuWSD575RbNbysYPia34\\\\\\\/9t5pcx4ZZ5+Dh60keAJ8\\\\\\\/gPgj10cht8AE6koHPG9W9Su7mzFpvfyHr5PxNt9SgvrAXY2y91H5PCLB4JxpucTaMAxYP0bQVOgzIug9oYXpDDagx6dxl4HpwhcIWtctYdQ6DqbX+rvhvrBgJOoRkpytVKX3\\\\\\\/UYNbhRj5BUvEhjWn8hch9vm0kIUiSaTdxA+ae7sYTky5rlQn20QRJfmejaJoWidyzIDgyY0TUNP6thwLLWb+yChPvv6DgsOES9f+Y\\\\\\\/+oqAvu9kaL3rNzzIg9NQCX8oYHTcmeVXoEM8HbsMlhMNJZas0wNxroKdfq80rM6TH\\\\\\\/tck8Tr422accgSCQfdU5b4IcqkbxmDQzZF71J2SSW62n7VHm+N9P1I85V3re8trK8ySMIhbFEdVonlCesM0PiV96vA29heQHaULmwUSPvMghdQiZv1ZCmnJxrM22Xthdf0AagxrfnXNnQPpnZvPflHV0mngZvu0r+HPOSEQ8NQvExPNw\\\\\\\/bTp14gS2Imwrgb59omagplghquMCFka8IuDfOc+5iDH39PZicFE5igVp3udJw+NBP1ERXj7meCfnr38\\\\\\\/3QfUPR\\\\\\\/BuIdWjASHTfExECA5KC30nyHbMpKp5xtjN\\\\\\\/z\\\\\\\/\\\\\\\/k0msi39gyO3M8PIPEJ8Sv168KKGuzTvtmFMHzXLlQBfojXVY1CNmQhnHtqjdR+zO0GupMBOx3kxnf+JpNkCuGWeoqaf\\\\\\\/\\\\\\\/U4OZ79zQ0B7o5xoC6r22oxgGn5eV6DnaCzo3ePHAa8V7i7G+PeDOacnvkJ3zwDH9Tr5Waz4CBVxLeNkX0bPP\\\\\\\/I3k9AIoooToRv24EbN98Focetw10zEv4TMAR4f2ALefs1JtY29VnJHVXGX7QZW6P0QwSwmHRfF0t3C6t8sskrv5zEKx4QCxrHnoM3X5ioRnP7PLThTnxLSNeyCc9u0xIrNdknlnPS+HIZfwJC+VB2gs6V02u76X8\\\\\\\/VefN6mB\\\\\\\/p1UKgL+46TGBW\\\\\\\/OgV6aoQ9azd6Cvdqqsp3Ar0wG8mAwXyy\\\\\\\/iO4l153Y0ILEkfDeGvXBux0hcoXk7boZAbo\\\\\\\/TVsEPykmi+kad\\\\\\\/YoxEnqMXVQIzbUBlm+gpBEIocj\\\\\\\/kXrdaN+o1gtTwAcCEzboIBOx3q9G+2+GoVqrA1DMoUNbnntfChZ8wnCkZntAxJgASvuc9Sv7sEihQbmH5\\\\\\\/DG2FBqcaRNYHRYSYPMOrkhwnVENW+8Ku4qrtwmwpTw+cIw3EKKHqQ1prJgYmLCV1O94IypYQW4xNb8ZsIaSYd6XS1djZ5S3qk\\\\\\\/QEsBAJ745f80bcrCbTHtgeO0k2o\\\\\\\/g8nOG+srPZ1OIXb8P3U6zx\\\\\\\/UIS4xKbxLXLMHhgzNSdxazz0v94sLayrxDxbmtLkDc09dUW8aaBJfk6rIpHS4CJL7rKleHLbGhyOFBmWkKQByHNadX+3xbcU6P2FAU1wHa0p\\\\\\\/LY1O09pNhvsTxQeC6XFidWiGg7qIe8BMmq1SwK06pa+NMed4rfJgrSi18sGZY\\\\\\\/1Ns0ol+HbSbuFhKs3Uqy\\\\\\\/\\\\\\\/RYNLYhrXt4eRYIprKB9RUhwMM8CWb5iM4rwIsR1xOeWXAl1bPkKuzsr40lEtIKMTtaNL8kGEnJWfM6yNE7Z3AqPiM6BVftcbbTtEg9zDjH7qiro3I7\\\\\\\/EnKVdkyxf1dyRJx0XQNTM2VFYMcgxbdNh+1oNJdkBxMHWWBtsFh6Gd1tBjeoIVgcAssCesKVRboTniO\\\\\\\/dQRcTkx\\\\\\\/lNfgGt2uG3FV\\\\\\\/4t+pvWs2UoBNShUiCZ1LsN3j2NgAjzzyubqQa3qTwXPBHyus8yCUU0EjYhOyINtc+6fUqXkynsegW8MdanoESlbQaUZBnE6BZrfj54k7ZBDS17snxKQlhiiR93M0WMZNNFlhG1YoDPhSGL4dFZ0uuMA+2rskNbe5YLYeT\\\\\\\/7jJsfJvm5AbyFNGXaphFRGsy\\\\\\\/zbMsW702rDeGXA5yWxANwtk93fqK5+OFgPkgJ86r4R2jzttOJCdGfcBe7lqyUfZRjUrMkPv99L5uHLAhVHb6bpWspywRt7tQpo\\\\\\\/eyyaRrS+TXdyN5WBUeBEVH9xEu30eQ54KOPffR6IlBLqkd9Hriud0VfSpKa0+q0103riI90KLgSpOQwsFO2yanmfb\\\\\\\/9LLMAXeRuv\\\\\\\/VZIsKbIfNhVcOr7a4OnfBWacB6lVLuMFanyDYQLTqc6bL98RV8hFW7cXg\\\\\\\/MjFb6hjRQXLAk1aYVT6jsk4C2HBnyamT1o2kkbtc6Ju71CNTJE6Zr0GNRrMe9de7ScQ7r7mlN9z6xHIzo0F\\\\\\\/jq5\\\\\\\/B4HQ\\\\\\\/RKgrllEtN6hvejmvwnTN49xrfJ3TKshygAfs65iOnRb\\\\\\\/sDKfrCVArWjukeo0FITpY27\\\\\\\/FTypWnn8IU7SlqqqWPrCkFavNIqcORRxCryxqDRGOf1Foih+CV1JFvFTPuRe5Dx5MLpOXZ0MrfvMzrRhAts8OCAqhbR1\\\\\\\/FSaI42yFC0whqIkpfGVLSHREOeDhjMkEg4ahUYZnS9tIKaT8PqvTsSN0vpad+LwYQcYX9SyLL7gEa0tkiIOI3kuBEPknTG4HCWnR2Ixgt5hgZ9c0xvoTPYR2tj9wAxEFqsEeYZFDL9gaXHVmCsF3FN89hm7cCHekjnHOeCivUgL3pHvC7jSWkHWWPJ41k26nG1rwPYJrEruIfXN92mw9a\\\\\\\/2W3Nst4cquRidJQXa7PpNgVu00fNR0DhWuoSSQCXkWydR4St0kLpthNIqbHhrtkN7oA2xzOj84Eq+6SUCmSu1o0JDFlicIvKUVj1ghP57vP1SjS4fQEixJEg+R7dNLu1cBPiuUDWs6t5pp2E\\\\\\\/3SMol840ywQ9Caip0u05H4k7\\\\\\\/AzYaWnRLlvVjiv0O4EH2UvwSPsqeb+y\\\\\\\/5vuoRGADS93uLM7ibw+RDe\\\\\\\/gEg7RVp\\\\\\\/bIvSpSK6TSDu6MJk7x3+fjwotB7a4M6wARn3N09O9a06awbaaX2Cwo4o60nBHIHHYgUp4qBpzcfINp1dwVxItiqUtsueM7P6TnYin2oZYGXmZC7FH9qouudaETOYcoP\\\\\\\/ZWqcENNxC4omjQkfxVGY6Z0lR23KHAzkWovXQsuSh7bwY0q3gYVMAOGH\\\\\\\/IneS5AZdpHmTub\\\\\\\/Dyvt44ZzCRjrMa8QokZHOkKOpvZB6SypdzpNBUbOIZApsYmvt3Q16VCfWms0xv\\\\\\\/tPP7mXhuvXO1P\\\\\\\/khUgyKuny74t6+kyfSkMloon8nXHDdcXg\\\\\\\/DHvBiYqtTXuZj5Bzs9tHDDs2APrEzNPVuLULDVSaR\\\\\\\/PQpsKUAsuQsoHaX7P7vOBvtpqfjP3ePCnPIIIMYeavtezdeBf3XUuB0K8TTzRqxsO661rMNpmdXDAQbi0Wx6HAFP6RGF6Uk\\\\\\\/feVzZftjdHhsz694k5FzbH9natSIY79kNRvl\\\\\\\/qguXZyXTDmSt33c6COsYMblRTGqlaWKlkx4ZmHgM5+wr2ZcsQwtfA0sBHspRaPG9jCNEUIcWd9T7RaZGQX95CEc+BrqPz8ZRE0tTEHKA9qAzZFiZ6P63uTSu81H2Gw5yw65u9erIafTrSgyk80fw1EB4XoSh9\\\\\\\/oPBmL1a5yJ5WNSk39Euqt5v1+7H4FmAnuVwrt995dK4by1KGOcj4W85secrvNvR2b7ShucL9lWVpvZoJW959aHwmdPWXPKNiPsFQINw8sCG1jmKMODbsrAVz+ZAXZJ+8bf8xtK2k\\\\\\\/fjcNOYwN\\\\\\\/Gp2ik6SVF3RWU9+6rjNFCKPND2PiU17lG\\\\\\\/QwMJnUmSjEjF6jJu0yQiatTf8m3HHeTezFXFcmNE4pBr9kKi9tLFjuoQ\\\\\\\/tNJC7qrcy0dnZRQAO4za4oanwh6t9M4YcgKzBec8JAIGfT5u6caZSsgtQvowqDT5ii37Lrzu016NBy3fG\\\\\\\/rbLoafXDOCgftbeWmHTBDcjupjHacQ3Mtl97qilXajn5WtL4eMccZTEjbjRvb8T3cEAGYaAe2My0t\\\\\\\/n1Jsns5MO\\\\\\\/sEVVr59Xu4jwHb1FHqxVyGRqhyW2SGQt2ubdPwrcFv9CkFtwMTB8iQFAI3kIlH7HptXqfHDIsbFLXX+RfMAAJ5s+LrPx2dD\\\\\\\/4HWqjqrTvgK+Nnzan\\\\\\\/QDdqZAnppFmYwXESEia3Cury00NSM9Jx0cRA3E7e1zbvVJnhcsLcPr+1DlfrO66TLD0F2RnaS8Vb0uQskdOpMUq7Z4807kCjKLHs7GJGYfNVZF4HY5xqLAVv9SMOQUPuXhurmj56nx1fQUWbXw2TpyN7lIPmKYc+Cfi6zLsY6R1QZH7Fe4PCz9aW2t\\\\\\\/MlHeL+gxddZq8uKVuaiiI7gMyNxBCcECkzBIV6aEba7thM2f5nDlcrzKh8jqweD66FJc2Sr9NWMoBg0ArxBKLbOj4psNz083cHG0X9kjZozHK1h\\\\\\\/oILvl+rgCWqeJowDw5Z+SgsKKGNw5XVnbUF6nMx\\\\\\\/r5VrFqezM0B4QzqMoROhW6r7Q2BcpWOPyYyV1xJ4OzG6U+M5WvV2SCtzrg9gS\\\\\\\/3LE73edlZXg6o9uZNeLs308bPYpZ+rI8uGst4gEyMXZTCuw1Iff1sH1lIMsjYKNHltLy25pG2ce2\\\\\\\/kW33ZlpBL9Vc9kYOBgs85f56aW78Os+c\\\\\\\/SPcwMu5RWNDkakYnXrLtpsbk4pVUBtVQDCxYgpYkrfUch8t4z0Reu2HniD9ypfjbUYZof+It1pSnu8bgWKOyWkVZT8LMRmzLugExQREyif++zpiN8ihknYhFJgGVLW8JoS8VPsTRx47c2KTuPv95BMEFkMnYXXVcIUC2eRVwDbdVqQuOmLbPkitgdpZ0wnN58szlSxQXrj1H1nVUJKabN5FDbTm1jSBBI9iDA6IPuMQFSY3BVq8An2qdhCAu88AOCpJ22I\\\\\\\/QGObFkkCv9LS7iK0GhPszSYe73uMJeEM2Jr7sUYm7VxWS5p3G899BAZeporzt0eOTEDEiGJfQPWDwazCx81L2gxVPvgRzXzQLgQ3heT6uqssdBwWVT1AUbzlLkhotwLuvQXlvxeUqBgDPcXEVGKuFPd6bDSjlwiY5gNzuxcQufv3BOSwCsPh+2JhkLWrWPEwTx7TXhKoHQ9mPBKfpN0qix7JbA9ayrVKojS8S1CcgceVGzGUuZ8ME+NRczGaMMymX+lRLUeCHMdH\\\\\\\/GJG2s1f9PiVGA1iMU+3LaX1D\\\\\\\/yAftTCGKKLYbzV8e41ccH6r0jOM5DdjPiMjs9IPcYxe9XWQ6OTtk0KHvSqxiceaeOZNxqZo+q4qoej4BFP9kNgH2TlNOIt4gPzOENFDv9FjahAuVpf3qdbtRj4qXrGD84RWvRYuU\\\\\\\/ChZ5E95\\\\\\\/+ifQQ4ZSmSgwHWY4htaKlb4vlsvLJoRPWgT2JpBas9wV2OVVCsieIcMw5nPg21ZnMVnHKxDmMNb4sr+8ZsvH3cPmglA+KVl9Vmq\\\\\\\/UV3EAAlX748iEDJ6aKR1OUrV1t5YoR\\\\\\\/c4E7roRzney993f9elCe5iq03kuVcV3QxRfUwcKf7yoejw8QcdkWo7BLrGtU9QtMYTzMzncjJ+CDR\\\\\\\/ELsMCSK3OIaDOwjvB4AlmMwHxwLuieCgH4EC0DizJyJCiHu0PFKCFp3z8v4NGm7jeGQmjx\\\\\\\/8oI1SKUcm1RSpMyRdd\\\\\\\/5BjDzpCAItmKlBr4u0mguiJyDFUJJ0YEBUBMhkJMJaPyWzlBAO7vdV71vEjqO99yVBiJ0lngin9KHn34DMP5ITRRMBpChzG0VBc9XibaciQXWU+nqtN1LJ1BkrWeDMgNzXzdxeQDTCdMxxAL74nSrj0NHD0WH8C7xJY1aaZ9guzBx0mF2OPjeh6M8KiUGo8ivGpEahKrO5esXzRRrBmqBKADNOoFhnuOOZn6XMXEILj7CaLmqcDFQCBnZfannxRDeER8+3m9m9FhMX\\\\\\\/GFBjV7O1xdr7F4am+U4Zrp49ohcPc\\\\\\\/6vIfUJQAJB5o57OFYvBzT2njfG3srOtDZxdhv++u0c42YDsXmWL0YTT3nuX9CecPkNJzsqWuu3vBS46VcCBPYh1o4lo8EsEPVRmaHRk2P+9YsaxSjJXZajYSzCNrt4hf1xVOf0t2pZgNABaimhTpBFSb\\\\\\\/bEprEu1tHKpI1Fi3LfR\\\\\\\/VN8SgOnPVbXNuD5k9+\\\\\\\/Gfs1ik1UxafUi5sfOaZa8uFNtY6raZ2pvpwLMYcekvaERZHgA7TuUUc\\\\\\\/kU\\\\\\\/80t+10Gj0W11OZ04HTcOl1Y2uMH8OZnzZfqbT+bW+7qvpa6Bo8OUo1C0iq00VyymP8V8gZw9QYVJ9mk0uNW\\\\\\\/KUDJPg6O8UZOet9xR5UmfJ2H74AaZR1bXlxkrYpLlR2MJjsVJC4m+AdLV6\\\\\\\/r79t+ENRrqUoqpuw2\\\\\\\/j\\\\\\\/Vai+cJ\\\\\\\/W0XJJUD4wLkxWbTPNMn5bfAWwRlvA6TW07WFx2eevAV4xSk564UUVGo+DJMBOkBUTrPb57ud2gDhJFIPJ5wx8I+si8Ez40ODMIyjwS7HtOujhi0QSxH4v2gK\\\\\\\/C12gWcQmV34t+7V7RR2F7iyUQQG5\\\\\\\/E1jUI+Da\\\\\\\/DXWfYhWMrSB0Yw6Oy4Sc6jbcO62hX93Ag8bVGZbSEBYOjaaAyG0QMr6ll4z\\\\\\\/mYBBE5GqlRsurs\\\\\\\/syOo+B7T+SvsQPqc\\\\\\\/1CYmY6SEZQK4LmClDHg2uIi6AMfl4gxA9nsgPW0kOml47rlF5DK0rtdsF9bi5KtnBcKRckka5ykDx9NXiLaOb1hCF9gAjtbTUlWMazmOCxevQx0YXOt+5jeVogCd\\\\\\\/5qoJzhFs5tQN04Vgkul0R2NAX8c7thDI9ZAl+72LB+DvtGrt8V6sK6aOUapYLJ+X354Fl5eLRclObBXgl2uM1\\\\\\\/8W5O5UXy5RWdEBtVG2ipXiaojmcY1I49\\\\\\\/zlzRwsS188efu7fj5zKro5b2MVRCjZJ5f3TW4EWvTuJ7M1\\\\\\\/7epnrbszFC2VKFfCR+4tuY8qpyCf00eFgh6x5EqpIGpg0RzO0nxWNRiVx1cmWBopIUOsIgi59sO+QW9\\\\\\\/+wfIlnt6VJtlwnwC67YvQo9fnh+sx+elkE55lH7QTuPgVFtvQl2WNXSqe3lBQ7DX8KA193cKa6N\\\\\\\/JMeFKCDHUC\\\\\\\/dO139u7VGV2sI0YtEp0iNmVIIzAbnieNNJscTwVCJWVWHOFL24GZ1WlbgWqAwu+jqTeoKlzypQZVEbY4xbwjZWlEOPo54u+Fm3kVTJD\\\\\\\/YELxwceVBM\\\\\\\/O8e1aQsORjn1i0XjB0I5V2oDIEdYCSUOBMGJBmwF7fq8c7X30oQZk9gB0SAjbaULkt4\\\\\\\/N8pS+YolbwsplSPUldWwVmdvNsKRRwr+KgtMMFWWr7VqgNoB1h9cMTMeWb3+\\\\\\\/f6GJzD4lpQOLSbw26Qs2w2iBM\\\\\\\/kP876\\\\\\\/eqw8VTNmLUxkyS2YdiCIBrQqGX0rYk4WUqWNpLoTqmEOE5O5YAihrq5IHCoLmVGHJigzW+o0ibvPGsSWSxXO89Y6rbDvzRw1qVZ3Y+09khkMpQGEFw67l69CUhVSjstJ8vVdnl63d4rjUl9bLXHFqMPBeMMoXjklJSocEzqKrk\\\\\\\/DWWqFx+5K4UTL0Nb3hxxBf92B6IDc1npy5mJX9QcFNjz7QVWoZ\\\\\\\/I2Up5uGbSekdvmHUX3lfUO\\\\\\\/9uV7rR3urzOT3fsY4aadbT2tZ8nnydAygZRyhWA1GuVyEPAB1gcfKZxIju5bjfZPempcMPMu5q5uv4BKFwtDS9EuhXf9FQci5+l9QOKZQpevZ6\\\\\\\/ajL\\\\\\\/5dcbPO29BNbitGMJAt\\\\\\\/wRyD69R+sR\\\\\\\/Pyo8KXEEq6TWEIXE\\\\\\\/4XueuXbNatC5tMh\\\\\\\/yC75esS\\\\\\\/XWR+xUbcFvMpTMgMlQbPoS1wBvdC5GoZOdnWF2LsDm\\\\\\\/N4DdSaS9u7TQ2Y7v0VsewhX6Xw1P+MwdNM6Mh11jR1j6HMcBlfMrP1b7kwH+yCmHcRVb4HM6ZcM5XaoLeFviV70D1im+vWcQ+7lBT4TqtEMBex3PywSWQIX8dNMo7WszU9129NeUmyMRwJ6WpmVI4DCLbUug1HaSDDXRqGnFTLpFHnoKmxEZY2P8RPo5fWh7h4Bp\\\\\\\/YLwvKaQe6a5UltpCHHDwF4yETQebaTHcCUdxUjTe9Rt0gGMjEQxBZJLzNB0mj57ChS4urhguCy3zudNqYt8kMKIe+0GcvEmKLzQkHQ95wWL0oKgN07jCqLU3ewWAqX2+l1FIUQcdX2pX872r0abv1rAeocHYQsI4Y6VLEIDWbvW+dJOg2HNELfqzqB\\\\\\\/FiYzXPipTVymahPPvKraNhahAqfV3UMO7+okPwb7FLUf\\\\\\\/8L9KJlUV6sIusiRLBKyVc99GpOnstNYV+1j261krjTXWckvIUojz8SOmQW4zNmg9hruZ2yKW7pwtgfaTZ2uHY2FJcRM\\\\\\\/0\\\\\\\/tNUriUAMll7nXShzYHetVO2wGBlYfezKP8V7eiXfkRjEhEjNRa3392eW9yUBLgn60JSBxQY5wC8jkKagr3SMf3QRiZpGxc17g2yP+Va2t7plpSiHPi3ZX6IiJFvV4PFbItU5r5TdmTJvYD1WEowgJOB1pn2mAPtI\\\\\\\/IyEg\\\\\\\/c+g2j6YsX4S0qnDqqrVcRd9cASPZPRIRegkWku0msE3pu1vLVagv7cwkDkIHNcI2Qw73to2AYUcDt3Pg8bCnRQtBW4tPsw7psuSPU\\\\\\\/RkMdLd8op9dUlL1zlEVCntWZKqJJHkPY8bh1BNW3m9USKoSn\\\\\\\/eQDCNC9U+5mMA4jTukbmeaGC9yvqAz\\\\\\\/3+YQn7vJOZ2UTnXl3D\\\\\\\/FWlGHHDHDEvi3NfNJIVFFwhIrH7Le4f6fP9FS\\\\\\\/6+wmGCbUTUy5\\\\\\\/MZnSdYde+kgyCO45qZBAIKwaaNn3\\\\\\\/SxosJ7EziqYbQ\\\\\\\/4g9WyMKLfAHBH24wF0XBRkhcYooloR0TBaZsj4DR8+0urp+QZKKLKSNORRIcVolAQY5SpsL7iGPkqgH+amwwLeovKGryK238WmV\\\\\\\/N5ds4VA3Gjn\\\\\\\/+\\\\\\\/SvWH\\\\\\\/Ozl6w\\\\\\\/P7kul41l0F2iOugXP6aVMqAkqX62NEgwMWCREIx6SclYfKyvSYAqMtAqA8\\\\\\\/t+Is2sWNNrLThYgg2LkNllFgx7ucLP3d1YLJf6\\\\\\\/PzB41wr6aEIs\\\\\\\/0Jz9SZJfD029T+zHB1PnXk5ACwh1JL0yX2w7zNnkBmgIT6mM0LM+iNfdYsSNEgM9QHu9ApZIf50tQWCH9G+75kPsnNKzoUl5ffZNJHiVoTROKu9dTUX0RglSYASiIu0Uqnw15PZiC1CU6AAB0ODn4ycyG0pJRsk1asboVfD4hUZlr21NqK14fzz0toGKGJHrRYDfl\\\\\\\/5KHAeJqajlQ1u8Q1qxfPhEpsh88fnctSLZh97gibOXng7eDozpY9qTi6Vr7ikOtPPxsZv5eECedG4HMVwe3BjssUb8Tftt6c9Q+zUaSxAYf9r6OsWRhd6b1U5r5tEPvOwnrx0RsvJOBeZjbxfRAtBa0M15rOf6Zf0V4Oo\\\\\\\/x4vrROqoRQrFrD2MRRaYOSZ56PICtH7LSvdsKUiVv6fCkFBVhrYnKP\\\\\\\/HhlEl1CgQ0m9+5969A07T6MCX6nS+XlVLfhi0TzsYkyJ4OgZ7unFNhhrQYc0evIAnfYUbcQKzyOCe8rDWgv3TnLI3xye47ojYomxmdsp93dedXD\\\\\\\/sSNkIt9DYaoHHRs+l8wyBMSpgprei2gOYmkitJG2EyRNZ\\\\\\\/YuBEzVwPxFBOR+JTZNOrunyWP6ggI2fpISlXJMSeEAV+aDyo4iU7rTHSfWhqhOEvAEJv3VoYpIQjocUOMyWrTxQMYz8tORff0m4NB8rcFEJIeAyOMQQA7l9LQOeVFIVW7f7CKK+N3v9s44rqu4hxE+t7mKtxmRuTlkAymU12S3PMLFkyBWWJBYixYozKH3ubvUSWiTNDTTsYYplQ1Z\\\\\\\/pGEP+OQohBdMN2Hj6j1nYwQuy2ZGB6gRUpxEPXICZYZN2gUgn8h\\\\\\\/kjnxxZLsjTBWL5aYYFVOfdpZpgjDibw+OjpEqBB7XXPY89ObYxvPNRkuEv5G4ayeVHqWHLFxpMiFj9bdx+hFjAbCPtHB4QQYChajkB7YRYL5sWGd67noWn4uEpUDFdmdfHEyUe\\\\\\\/0NTlt9\\\\\\\/OoR02Qho7QQZkhwLMVXaXQsH\\\\\\\/3ExuEesqvdwvjIYWKaBitI2\\\\\\\/o2RiD8gHgr3JaJyWP6XwjeR\\\\\\\/FJK8UiGASropJFsmBBJZ2srAJ7fyJugAm4QnRs7sbbcabcm22KqnVzaQ+pFdjV2PPBykJRwyt+aRjnOY80GbSCAqY7r7OMmum7tVeeWfGdP\\\\\\\/wfytc9TsqkILOus0lswfzqTH+LDtkNHfvnodfbzzfIn3r0ekxRI0wr6wzdPDIxi8Q4bi1liTwhDGhlm7LEclknDKmjEpd932CqnsLxaIrrU9v7eNvCpY2RAfN56gV75yF7DW07Lif0SpQJxKV0gjijYSZNKLvemenBVpilC2sGto54Zj5RCP\\\\\\\/t3W3kxKrCOC7+XRqZVtjpKYHrz+nyt3S9wW6ouMdVFQKMc2OohO9IOKe6qYsceZC+EE1MR6aMCb0RwnKiZhpibGSsvUVyaJQnxniAkmlpIE\\\\\\\/cQyS55tjh8x4cdwS\\\\\\\/uojKga48lzJCuScgDdoFImvV8VzJNmP6MGxaHXSDMqxuI\\\\\\\/GEJEZMM99664R4dZl9vq14t2+KRWzn3pP2WlIDyVF9NmgYxQEoHo1jbOETn7Wf4+7a9p68ntn65QRrDdCGbqKx82UhC2ocVhhQD9CotESsow5Yyhsr9Z7mNkOqIt60rt0xUPtZA+\\\\\\\/q1TQAqmC5ofG+1EKFePea9DLJTh64OK+7i01XY0vNbzJOzUIeo825pTbka6hGyBu9pYTR4vba1kHLL9tBY4g7bisg9xQMxHRe9eykFVCmcjbx0tqjqD0f4Jk4KizaRJMX0HIPCeyhGElNJdQEoQfxphICrOJzP0OqABx0G1hBifO3bJpk2+r9CPqLFaHb1Gbhs6QRsp963VbXvnO3BWwD\\\\\\\/ehk1JJ3U4f0T8qNBe17HVQh5CdsMWXuEDoMV99Li0jSIRDb9Hnm2AywjrouqCk83vtdUREszDrU1GpLBYTuk7Ro4VChlhPjcDY5JlPVkZIZCkfTDCX\\\\\\\/tPPy8Av2HFgm13idt8rmfLupprHT0JBjNsOAStiXG8EQB0Vn1JLzQre0WB\\\\\\\/dXr2xyqJ\\\\\\\/5U\\\\\\\/decl+OLbmCoKL15uroqqO\\\\\\\/uYcB1Gg9KxBL4FsYtKSR289OqKuissV1Ky3ZX8XApmuFscWl1bh\\\\\\\/D7dWq8F6qE8hd55gfGJDI2l+F8sEetGqKEDT7PGhoCC\\\\\\\/vcqHo4EWL\\\\\\\/R7gauAFBtZqJvUhirnu6Ivou\\\\\\\/cnsv97W5TspuEBBKa9DZHQHVleHKjbXTxv+GrDRjqH\\\\\\\/ruRqvLt0WzvWDfZqGWHZii35kgQUZMIXQUC3DPChm8xWzSMEk+qXVXL7Gl7w1+N1f4d3g86kZvEtDKB9AQ60WJrrynzJcMjAoBlc9oVn65NoFKSUnn8sJQVe8HC\\\\\\\/V5ht2RV5ri6qStBgxAh+XaTaIU1vCbI2GiN2dDvvoegzFdFicERRsakfg56eA3RXJJyMTo+zsX0CA0sZUfcJPzb4Zei9YO9U6OrqUGsdFaggVb9iYIOpMM5J9rkpVUEIvCLvEhnUhsgtn9JsSrsVlg1\\\\\\\/kNUGbtyRiH319yVAPUsR6glSOV5336pnTu7eSKHCIP+OkmfOdhMycL8jsnp0MEtPz1eF4QvEsRCY4CF+qB0+8Ny3y6\\\\\\\/7xn1ouhswlEXKBA9iEE6g3BdUuPH7t87TrGQSElk9FJVdOkWP1G1M2IlAyO+odIhDiOfVya7NYfJk2ufkaw\\\\\\\/e0\\\\\\\/BQXfZGH3Zce2Nk8\\\\\\\/fsbDMcjBSYZAoA8HX4e0ddJ\\\\\\\/QASelnpD7PjT3dpQ9+04d5F9fZ6FQRXpIyJarag\\\\\\\/T+zf4Q+T\\\\\\\/5MnZP4VsjReguHHDcRIEs+E7SqZDU4mhgtxuoW4R5zZ8KzzREBObN8\\\\\\\/4vRKeIFTQ1Yfbg684ou57dT8+gOuPY4pSo1oCAKYi5hq4\\\\\\\/8RQTGeoNsrk1vDbzK9K34kW9KNQvJRQbAB0Lh6nDDFiYFihIM3qwQG\\\\\\\/lb0m4rZQJ32FYKTqM3qwjuxYhD0tA2DmlHIIS3UsifdLVA0dYtZLHK5r1w\\\\\\\/f39m+v41ta\\\\\\\/Oh65Lms0b9Lm5suk3klBRFB+XSuDWB\\\\\\\/1o9PGcKyulF+Dp4GmCj8dlAwAJx9L+cMvNLC6sEZTVnZxg1vh1L9hrsuAHsf28cR+92+5TV7um5LEJJ0dOluIwTqObEm81m4aMTysNDlnBMbzjtdNB0vjFD+gLk6OLCo5wt9q82l8pyKiwjPbqpk1HGrYAjAMVYFSMlpXLvfdQftK6fKL9kAezWL+qLK6lDz6ZrnnjPftTK\\\\\\\/e7XD3i1+UXg73jby\\\\\\\/\\\\\\\/aKv8n4P7TXpSBTilw8AdlCl0YbDaNkxulWzvK4+6j5zmXoOhfR1aqqbwR7NXMAF8cl9q9AapWA\\\\\\\/YrUUn4e4DuSadNIhJFwFJKa1KAaje084Q38ytEtLgwZF5vXIc1kZxZromh\\\\\\\/Q\\\\\\\/OEJZJHnASh7zVHMbdCZxPxxQfWlLIhlaGBrBogUKOaW1uE3QTt2dJHm6DA08fuDga+i5Hxaic5Dni+ZgvUX89NBbGDv7JSlIZ2taNZy1NDPfhYVtIg4JJQ4edHMWV4lQFcV3O3D\\\\\\\/cDw8gZAK4OoM75eJeqYxlAKI5ObGUvjTmvGblds4k1m\\\\\\\/Su7We2De7k4H42BuHBP89f+\\\\\\\/dmrRic7BpBNyiR3hX66HRkxK1w8y1IMtgi\\\\\\\/XcqyM+ZEYhDfrb5PHtlWNP3GJUxBJz5P1rhNm0TxR\\\\\\\/CcEJmPZ6KWLs\\\\\\\/nrBwTXUViuDamYPoew7Q9Z2WHLuJDjNfs7dhTO02FAD9Vf7E3D3PG5EsmnmqlaClYPbc3WINX1fz9fvrUpQL21TqyXeodzmah9Wcx5WeoPbcV9kwxFV+CHqSEcyEL6HmQSRirr3sK4vSCQD5JBMjGr8ulDl6FJAznAj+m5Jf1bmy6uaz9SuLjTwRxVF9Ms9NkP5SajbosDHW+sNoFIbVeK0FTHVJOv3Ro3dwI6IUolMmew+HHwol+4LSAU7GBTMLg3rXPMwBIikIZO16mLF+q8qpgz60q+jMjQ48vpndqx6WMm31+EMmqSCDScI8jaiZ4Q5H1lj2SjrMiNaAlEL7X5HyCsYO8x8ybylhs+k4G+8g9oAOHYwNg0Tk5Bnv0I8HwEjEj5GmG4E3K2ERXYsCXuxjTzYjM3tTOzuVN0Fm4K4yTYoEnatYMLAnqpheaM2m+rgL7eyh3LB8Dx7X3o\\\\\\\/CWjtYhWri+yMIC1UgM5lffzhySr6a3awkfXeRTMcExbchgOP6amNHTf\\\\\\\/njvCb3ktUj+7PZ68VmPeWoJx4NHJgHhScmgSY8Oeg7phGtz6SrNJUxuuj5z1oDJkuTXyH+sxJMXEL+oaruZY5jaiB8j+bk7EqbzRh6YO6U2kq6yqoXydniyu860ELC\\\\\\\/rbqMyLTbdWGaUppnL5RKNCewrYoJgNjMyXv9kV3IiC5MvtqNPk5decIO5YvAWfANxNypVJHSAQHay2QEUtNIh7ElhEbnb66299XG1bGZUEESY6Q623IOF9BjuB8yaunrg6ae5oP0A0qa3xuASeDlWb4XdhFi5QH7oHcmlrgT4Tme+7\\\\\\\/\\\\\\\/eRx8ntWgfYezDXRffJiLVrbNff91R33KXXIY+mgOc7LyjXIA3EgY78aYpnUU8CAM8nKDU\\\\\\\/eqddq5c4cRcwjNW9PgQ2IEPn00\\\\\\\/edmneUPweyGoRb7eI27S\\\\\\\/6JOCrPU7aMOJwyZDtkoNy6RSa+e7mos\\\\\\\/D\\\\\\\/FXZJ8kOh514UVsG0DIuDXA4YstBgSWXfWU1PTB71geU4TZFJETu+uAZf8kQ3tvNl1iC2h518yXYvdt24+p95ZSUMNQAvm+abfIzAmcAwhZNjzwxzE2ZKjn8Ps9Eao9\\\\\\\/32qVIaDDAUEIuMwTWmVCqLI4G2h8WeZEt+p6X\\\\\\\/iRiQhynU38iP+yeDWjqtitzU7jUorAaIYX4vj\\\\\\\/lGD2gmBF3aetaVWX5JqWlXZEXW\\\\\\\/WMMtCyY\\\\\\\/N4wfgZRCptowxcr\\\\\\\/6Vy8yDlMfTY8AWzguhUtqoAZWBY\\\\\\\/BIBiH0cbW8T4CaNj\\\\\\\/4j12B8fWmmlTPjYMzZOZfyapHuFmNelK7d1Z7uxBD8edm3QUr+wUeHlg7+lxxUeGPC4sRxwHd\\\\\\\/Dxf\\\\\\\/X\\\\\\\/i8+he24p06R6WIeET7oyCO61MxB\\\\\\\/xO69KPvYie8hozWKa\\\\\\\/k1mb297jQdEsqC7xw49UrhUpEvvR5VHBxNwWT9q2vRtWRIDb5pf2DNQaKaH+1+hzonUAhkctoM8tkz\\\\\\\/gRKDgyi0k7JejzfazTEhAQ1oY5a0MLgmc8FxUQ5dxBiVdv0M8z+Ma+TIbpK+AIrbWfnmBYojkMIoxWxeeAxzSjRoI6ZoxPQUvvLrA6F3XDN0K45F30BknqHReYHr7SMmmGXyLt8qtT3urqEIBMSxWG5cY4u2fXpP01\\\\\\\/Ex9vhRL0M37UPBSa\\\\\\\/KjF40zV97i++uQ6kJOJH\\\\\\\/55hmpvuaefnFXF1DC2+JEqoc9ATHExX9X7oBIsVjej\\\\\\\/fquObQOeNaFg2BSvZYd9pSGwfMXlUThzf8061Fl3sKtdZfqMB+SdSP2uzyyf321hsAM3zl0T+dlUguY7S8H4fsgGC1YakJvJHvFCzE7AK84vKetEKYxFWxhwGC+7mRRyp3Q\\\\\\\/0Jol+67fxgNnayno7aBDdY2i2pTYwZ6p4OScucQkFtZF8R0yZxgGszpryKxmM0M229yC\\\\\\\/02Ub6VzzcWKUrZ+GDjoZZK++upPVA\\\\\\\/jBLD7PbEZSCs+8rOdh8+uJxF0OS9ys5GE2lyuYUAIeg0QmN0rwHRzeELQIzGIVi8rtyixX5bEQ5BTFtEiMGciLyfV9xt5NZ8ubuacV7Ts\\\\\\\/lbm\\\\\\\/9Xs9xB5Qzi6QrSYjDoGUh9c00M6dh18JfAgBEtFlTCxAOrEKysJSkVMBOIs7Sa6bioeQ8XnUigc8UtB\\\\\\\/3GpLvSo8fgxB1qb6nhh2HcnVurIB8z2\\\\\\\/8UeUU8zu1+VZnugzqlAtsGIkhVs0n+qjCVqCVz0Lz6d\\\\\\\/cvwckzxjVY9PPEN6lnDkm\\\\\\\/EcM2eAGjjgy8Qr8C\\\\\\\/\\\\\\\/rjMItspwiFxx26sXvSLBGupUSdqN1HDESByaYSD7y7MFcm9woXVcDHFwIflmsYR6dorD0pGoLhJ9Kir+I26O1BKOjpdeLreKRU7rK6tc22VN7Ng+usZ45oL6wU5uwzCgzRdKuA5Uc2P31X4Ku2Gb+9gZz8MFJJOgzEVnBKQxJvpAYMKEMeUbRwe1vQIMdTy21MWLsGRb541KXc5OfTMNp7Jni5XSrENSNbC9VvOcmFnI8qgvnoY+z7EpVM5HUCCllPKHabLZGzAI5rQi21KKYiUFXlcpNShg9iN7h+RIGEWujiK1f35onwitWV8vQDzcXf2pwKzQJZrUnjaKSRwY1WaCUXObFe0qNC1FMBj8xKNBcKaMpVw3bR+0BFN5P20vySm0v9L2Re7Iytl+maz1Asx5Kr7DdQhLZO3O0OswNvMUWUjrGigpjxs8QDRQ9Jq03UYPDcBFjfPaCI21on1Vnw2hSSQLc7yhwRRD2B3angYC\\\\\\\/xErsboHRRjLc6O84t3v6HbTdEWmY2RBrrDLiYkP8IbHUKCCtkgnmHyqE3nbWX481Y7tPXOYWIkE6+BdaRIWego26AZ5ZP7GRusUKj\\\\\\\/8X1B4UCwdn68wcEQ8RulEyegXzrTZ2INRH3MAYawuQSNdCWnMxPulY2GHAXbua2I9cfdDQw0BNU1aF82tWnDTFlsGoNa\\\\\\\/pcC5JchFOGyviarhvuTIkY6BGustFAAUUaP5O72p02n2gFtlGMsnM2YuL1xFSRJPt97c0JCpkLvxKLFOq3b2tRwG3MbVKwA6AV4e+8AXOGfh\\\\\\\/qsh9WDRCmWb8OgsAF2S1vOjS28s5sAM5d06Y\\\\\\\/TycJalwYg9DMDDurs4gc9lbOF8axEkMOf484uk0pnrO91XgTchDkBzY2L\\\\\\\/pyv2rh2dBnxxZXEhkWO\\\\\\\/62O\\\\\\\/RUHUj9S0HCeNKwdm0FWz3acjPzFb\\\\\\\/CCDOpx7GdW5KF65nwYysiqOvJBewn2NfvghsRZVdbRmpuGHMrQE6lAoP41BC4Lc\\\\\\\/N5edIe\\\\\\\/MBBWi3O0lz5CnePYNwej0UbL20xkr7B6qjKmhDiLUKiyHEfKqMDJwV+aPJPTwJwMYkPGEhiM0LvF\\\\\\\/T6IrFemxth79LaC9coqMrJ9If250MwGOGnndIz2b5nqrnwkhZxv173eoEqqQ0MIhJOm3u7qX943ZSWmzIvaZ\\\\\\\/3msbAj\\\\\\\/VXjInXBnCtNzLXQR7k\\\\\\\/ioALA\\\\\\\/mLCRPzvThJWmUnqU3pTFZg1IyVsrtJmmDnseuBl6Lf9hg7xyZEKnwFAZJrpzXXgop5hsKYgG2dD+eeeesr\\\\\\\/YKgqBpOPvTiQIHSFLxi\\\\\\\/hlcoAusPWafZrksuViiv9MDNjKKAGwXdX2aHGC9XzOebZfQM32W9BVntvAGK1iRCZmjiyvroNVH28dm0gHC8dx2MiTH\\\\\\\/soZ1gFyjbqrMJc7OxJ8VfdKqy1qe09e0+Pk1KjD\\\\\\\/wCsx2EkGCjzcLyt7NCmlpUWbX9\\\\\\\/8fJ6kW+XpEMnaUTkdEVVOlGzQIBT1qFZmLlnQrhBg2icJTtQH3iGhSzIqFjRmrcyP3Ohh4V0GZ267OcBCKlPtzNAaRWYiwsoOIIdRoqGjOF+6BjA99e+IxbjTn9R3e1mUqg7HNgXvHPlu8w5faXJ6baBC4xmbPvBNa9Uma58jYHhse8qoec0ZprEj+cvTYNU2v8zEvq9OoMPa06fIleXaujLr\\\\\\\/Rt1tHVonsE3jmxD\\\\\\\/nqkglIMUqMf4L+7vi1sn8HaksyEK4lSBbYhxvmfjH\\\\\\\/Z6YqMrcGDK4TYFln1ITPa8b15AhWL+lRpArmQHV2HsBf2VQFIolH34BK7I7sexGp1Bay1OEJgVVYUq48mmraBekKUQ8tGZOImsrAXsekW1ttCYIqUlzVLZah4RI51UDH2c6TRb0gdNOXwLnTXkqm6Zjoz25M3YFjDQoTIvPlH1UVQru0808g7fob22y4LdxAwZe01oXRwm8w13UK1kUxGVSr6VB+G\\\\\\\/vJoR\\\\\\\/+cq80R8XsnSc6s7qg\\\\\\\/AxkpcJW4UJDr6PlQlhMNjuJIoUdb1Ss72W3Bqrd13JagF1dcV9YVs1w0Q3DtsrjFqfi4CubHAYRqIVKAMhrz1P0XbYJJ8nH629acZQWTZcr9i0nx3ha7yG489o4N17L5AtiLoDN7zN02p5pA0RtND9o86BBt6dC8j4aVECHO8MQitR7JYVvOCAdd79PxBgORzHfhtc93bH41VXQ66HHwimBjNesgL8wehqABp4HpsxTIFOeQ7POLOnCnE2k3kIb3BkKC1YqBTEaxLfPCA1TUgexWy5I2c38vZi5CJWOyO7XS39rrNu03qfu\\\\\\\/Brew\\\\\\\/Y5NvpFJkRYk4Zr7K6fzosOpRH4H\\\\\\\/OfjZd6RnFwZKKzsuLozDDFCOZsdBHRFDqFlyzd8HHDruc8DS7d+pjQGA\\\\\\\/OyA5gX\\\\\\\/CjcfOBP0faemojIvJ3QAcn2ipzFTsbfTkvlS\\\\\\\/vqpQsI7XAnGYQa4JGhdJwHybpU9NyFBwmkB7rQWl+m89FMjAHKuDL1nThwzejViQMrzmXCQhjxbO0PIBEq+jdfLsi2e+56FiI2pGH2ZZHxEw7jWaGYgyRJyQHwVQ0NOokHRkqr4ntxauXjpOiLe4lW5qVYt8vOh0xdvTbdtX1O1Q1vN2AVGWhsGjZm9LBxCRGBnyZIx0tEkdiE82TYLlDTUayDexDFJWYQljiLT57SKIAeSzNg0RprccKdzOxm6041mcAvYVMGlhllgiuhOlItgqoMyTUa0VVp3p\\\\\\\/uFFwjA6\\\\\\\/YYNzXdzjFFrqOoXGYby34bmMjIvayMSANQIAGJHb97i4x7eQbCG7CNLFThXI3L8I88a\\\\\\\/Qs17ETt35Ihvs5CFZEnfu5\\\\\\\/EbKb\\\\\\\/aBI99fhPjRfNigQk+2ZLaf3DXZtW5Jw1unYoP41lryZ1tGZDCEKvAQKeBuSGRJyagUyBDesTGRMGduHT+01DuuDWPGDN1dtyzJyE7udTnHSYJHhzblQHZYW8jJnJaJZpOxaF2UHrYCI88KSbPVB\\\\\\\/6l3fqBtFdZzvuq9ruhXjDuYcBAArR1zt+pmBjSetn6npBjwWE8Lm\\\\\\\/jBN9IXTV15RaSNmt0FZgjIrCeJ67G+Dpum2kacSpIphGdTf\\\\\\\/1K6FI4gnHv81wkdia7U\\\\\\\/8FD7yIoYXpJMvXVT+Vd7JyVE9GgwoOHmUS2RKE8dgiTJFd+b9rUxEOIdbulvka8PhFC6KC8ks2rdfuBH5gp\\\\\\\/FEuZtJuNfZ6LMb+\\\\\\\/I69jSARjaRf+yGocXua3PHjbLFSBExbQAMpaSZ\\\\\\\/qVhhhi5OUBeiA9eZSjVZsdkoUdU9u35\\\\\\\/RSKnPKprzt790k64wW4S+vH6zFMj6slizkwWIiR\\\\\\\/17V16yXo7alWR4SqnpCPvxaa6YMK\\\\\\\/csx0zGw5jTe2hl4TeKJpagjf5lSqkrOnpV26kQPlAYO9m2OI1K0n0xXjyXYwfGcLYjPhSOz0zIpuRfCvWT01I7koCucjV4fvgj4T5k\\\\\\\/VwKBd0RdmkY8cJdG+NK\\\\\\\/5xxQ2463BQqYiTX\\\\\\\/Mjd12gSz0sjle9T9s3TKagJ8XMALLNSqKiDJ1EvHjGd+snCrlzpNvmqQcKDubtgJWltbLa\\\\\\\/Pb97\\\\\\\/yC5i4HkzftFONi1vOOMoVvpfxqwOIXyQZaVD\\\\\\\/MDuDOTM5pE9LUXwhRybEbiOW0FP7GC0\\\\\\\/pYpSaZooczjeHQNKk6WyYgfuEw8JzW+6Oz8L1S\\\\\\\/IzgTb2\\\\\\\/tKRN4WcqajM+iiZULWlGSNIQKmXRjG0XDOhr4Zloe0oM+kgUUJ7LKWl4DOvbmh8PdD2XXgWQMnjZx02CVdb3fZ2abLFmzNgs3B3akTB9wG83EYsE0BBkV3O+DLBaT4VSlzWcM8T+Uxp3Gz+hbWqKI\\\\\\\/m4llviBVFfUbOuLn2+tUm\\\\\\\/BJ\\\\\\\/HLle4sa5WWxz3IBzw3Mi9XeRdHSNxJoNuk8cMJivD3f9ZFUEKFcMIBNcPU2vOXv25yv4Sp+JdArlRWOOYguYsFa5t9kU\\\\\\\/3dF+UsZUKKDpW28JAY4Gsf\\\\\\\/jQhuILqljrNG9r7hFGabsdYrzx1KvIqqeiWIzng7IBaWdoo+ZgcrJwEqAsdyJMDsDQRPmDLHb+H8UhZon\\\\\\\/N86utwZGT6Q7SgG7n+PNakC58JqSroQSfI8uo7fNtFoeysWvyRq6eoL4OFBPghMdyYsyK\\\\\\\/PeiIji9lA\\\\\\\/jdnCSTsKk6f837\\\\\\\/MyL+26NnqNatOfrrIaLqp54nAJ9XT8+ssTzsATDASs2uwGRsq8lcScU2eaIEBezhxQ0vuZOuM5NJBr2kmXKp81yhbnZjfmg6K6oADGFqI+8Lu6wIiX2qx7Dy\\\\\\\/4rcWsdzBR6NiXGBB6GvU4haKAh2rlJ1st8Pudz6jXsVYE68eDfefqZcIojFxlfxngjG88MSTAuhsRafyblrbecXAShsZ8E2BOhxCJSr4We3Ce2oS1ltGs7GBnc8YeNlORM81AHpjCPubtqI+0\\\\\\\/TZR1VC94As3npPgoIAzbiiIRMcL2EE\\\\\\\/xcvThgU4G7YSKwbEgVM6+Q2NPhXqOQ26rjBAcmL5M2AzS9tWkVuHjyzyz5SWiPFR\\\\\\\/Kum1wKxd68t93\\\\\\\/a2B0YHY+pNPrZb0xwWcFMO+Yrb2BjPeBtfRGgH1r8dTf97xcRLlS6d+HTj63gJcRbtH2B3rBxvbJ0sKzxmkL6+pqSGH8mWNI17ZOuUpjmLAtECDMqjI+u2z6TGWPje1eYlNcNFl+alvnE0D+5+X5eaj7AWZkOC+TrOUDIcUcC0FqXr\\\\\\\/oRcOuazCnhDhU80mwMO7ezWsJxfI1gZc2so5WCHdoy\\\\\\\/pdoo3aOjYRzA4BE9VexPMZmHiCc+2mymVlh\\\\\\\/EY3RZGHDUOOWh1nrpDIm2n\\\\\\\/lhkMMijqhiSiRXmtYPtgcLnhLhdidyayhP+KSHHh63kmjRJSkU73e2Q+VyUpHHLF\\\\\\\/rbiiDM+kJWqwk3+\\\\\\\/IJGrtVthvDwfpt\\\\\\\/H6KEcPM9XXaw3\\\\\\\/Ej9s7mn1ViisCKblDuJyrsdlKm1vL4LEf9dNU\\\\\\\/FatUB4NCR345Px1v6oI0+hADGI2JQFT0Cmkj0tCyVK2fNsVjpBRr8\\\\\\\/SlSelyTJpm7l9tYBqGzBuDstido1zRybKBIodYKoBX5uLS8VpjcACUVj4h5j\\\\\\\/WaZQdJso6aNZ9M9gIcWWU8U5HTLQ644b+RXb7BTyhjCs+H+QgrFH+fvqE6W7ObLEg74lellodvhTuMO4pE3FQEAyFQGT2ogK+J7hpQ4N2x3DvCBU5DZC3cI3+Lmtv49TkWkJj9ckbPGjg6Irg\\\\\\\/Cd3H6L5KccJBvaquwQOOExbRTim+lDCm6evQnKNqczvQubhDGGHpZpB0o6Vi9VZ+5kZ8Q6UHDTgeVPRw8Gboq04u\\\\\\\/cqtLf9u4s8KK4yxBp2t9dCzHdXRVfx5lfdVG5z8ntjRyi29+0LWBvGe+HwLwa6yWYgUDlvpHTa4i\\\\\\\/rXc6VzeULAANLykbqvUdfR9c9O3rLSzH2cZ\\\\\\\/kVL0n6rR0t6FQBD5hX+QrWiT5owmyLKqAvtQOezysZzFSIh747y4S13JUg0Z6x56oaISpY7hbou8gv80IQsXSrKNVvYUHOrjx6HeWlRjhS3xeM86TBK\\\\\\\/l96bY20B2MzpXcU6aF7JJ699e2zopP0QyfaUHsLAj65gGgenQXL5tqEp6OpCaJ04irnT3kXycvzXYIwvAYPTGGMa+NwjTxiaDTsaS7cZy\\\\\\\/WQCzW9R4Ql39sYwnj+Ix7xAMR9mLTOlxJy7wLKCvPkzdNf5jUTjUIlolXMnQyrbA6jpNAk9p+NCst4J5sASl5+2vGPPxXsAFE+sCJUiHgkSjGwsnQXkN2XPlh860girVcKODItAO7u+7pEv7Q+hSq0aICPDmqDO54inRvnaBun\\\\\\\/5GEh0x4vLccb+ZkBnrXIq9qT39FIq7T25kPxjrCiarSKBKRMZuQSci7ZsQIbFj2d7dFQIqLnpgbP0NxZ4SKmc5bxCp0UuMYOY5a+nit+2nH7wtQKWAk2sNKtYH2Ua3tOmLYTTcRnB7Jd+f7XFT7pyps31Ojw4\\\\\\\/1JfitFVpCCZKY9oj3N9tez7K2B4BXima5mwrEEbezzX12G+HApEuMhM7ui3BmpwgMD\\\\\\\/rx9qZWSlYBByrrERWYjiee0SZFL4VQxlxLW3Q6GslpqO2SYN\\\u0022,\\\u0022iv\\\u0022:\\\u0022SS\\\\\\\/isaaH6GS9qzSyC4nZDA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002239c6c3ccc626ebee3dd70c3b7c9ad5ea2a8355e5f5aaf5ecef5412c25f90e56e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O5IfCRaQol4hFMhyRZ\\\\\\\/+RDu9dXKuQ8oW7pt1znZxMxi3QEjJJPiVNslW1rAtRnUVpvibEi4qWR0AhvLydbdvB2B5eOIhV9+BVGuKR+LKAOVxf24Mhc\\\\\\\/gXTS8846sCTpktwKVZYj08IcacXjdLC3zIvQIFCX7K76TJAYqnPYaJGvjJYN25B4RgwkTNZWk2i3f7Hen1GV6Jcsv7skQzG12R2CHoVjy\\\\\\\/3xF4qmL0F2ZfPlDDolAEFjTuxAsq6dXBJjcCIDmxuSTQyGI7pEXVZ8CqV7ih3ij0DXms2NJqjE+InwmPdBMmFsm8h4V5YseqJtnysBPBB4iSI\\\\\\\/Z6iKk4spKZXjoGdqgChQTsIo\\\\\\\/Vthpm+2Cg4S7ppkRof9ygq+PVTzO3DypejtsP57LVwEUxZWXakm5ECglyG5\\\\\\\/TAl56CZkI3SZiSYG3By6Vb0I6s\\\\\\\/c6a2aqN67e2ArsWBVV5zckcdzJtUs4N7Wp5WPEynfaRK\\\\\\\/NgozJgFTNhiIdPo9G10t7ucFKU4BaCPSnKngtmydGqjNfEAL3cWGmJ+DCkVCj9i81sQ7r8eA0MEWE83H933WWjEz6k8rH5ZDBVkEg1tGnFogen4kWed7uFJlhHu228J1+MiX1j8r0jbPqwThJZVv00sjkMhOr5VywwPypRH\\\\\\\/kqmkezEizIX9nXO8Z0i9YnZrag+kkesZFU9yUGHQolPcblnjFMjFYPPMqUn1s1gfXg3bmE+gYH8UqNWhZobv64j6ao9vCLjOATpM65dTUbgAQV1Myvzx3uHe2RN+XbO0I86zDqakBqD7CuG3qPVKNdgEV6Qf4Ah84jYNRVL0d6odP8wFZ9KTLXRv1ZtvioKpt1agzGmc\\\\\\\/WwloYXk5tnNI8MxCeyQd6d9z3N0M\\\\\\\/IksnJD8mcezvVpxReaAnPfoYWkRwDOSv2VEcUI4gr07kWNC1mih5Q5ey\\\\\\\/8SIaVwMFp\\\\\\\/S5US8TaizYO7PlUTOB6DOtjqjl\\\\\\\/svdSWfTLjSCbiVan\\\\\\\/p7BEhGv0s3Zx4fhX8hDAp5ZR\\\\\\\/YtQzx4hu\\\\\\\/mkhcKl72+i84dpfJ1FWhXpZmmxUws2kLBmzA3iXFrokv6wq6w1vGCwpCDnEvlrq0Us4Cp86oVcsE0pFLmBzZWML0M\\\\\\\/j1v1cqlAzrxbw6zknywY5KpgO62d2LseAkjjL+AgbJZy6k0p9JsFzErJCr0IElLWycJsav2KjgWl8sWZ0AVDcp0C6R7cqV1XY6BJFcwXbE8Opbakk7ZJox\\\\\\\/C5ubPd1ubjMe+\\\\\\\/gBseMdWaRttYohAXI2Iyzqnk1r8x\\\\\\\/2paxklmU2L\\\\\\\/xyBK8cikwv7EpWA6Cf7EXVyP+k5ABDz8GjVkcvzd9frDw2fXDO37hKh\\\\\\\/+uDgBcrLzn5uX01QlBE2yb6UrDvKYdi3lxtNhPb4fHWd8j20P+KYXDfX1MZ9kxY\\\\\\\/jZSWOTxbUiaJV7Ypp1BRWChgSa\\\\\\\/OEQ9jDfcC9ikZHtTeReohmEGKJrGI1ByvPxuPFnoKdkPkYIWdXXkR\\\\\\\/M03VHmFCz+sLhUeNRy+30QL1QUUtBnZdgAV6cdd+jP0k4XSCT16CuVvkue6Xvigr0VsvI57TErAdTvlHJ7sRC4IwNSo4+yIB1N4wznUgjSQkfyUDzQgK4IdZ1gVa4ZzxVJULNizvDGpOGAQQ9XlvpXMzexnl6PWa4q2IgrWhSXeYTzCQrkuVfZn25+TjVePNr9AOXvoa3vWjTUk3EFzPvj1sUL4a776YOYtONzXc5aajt8eCempyyrLXTUPwktMka0w+wtpIZjSU3kZvTqMDTykbILIegOolhjYxapJsqxYXuxLhbaWq1iOkU9d3FyhyeGmlz68mmloiuFCmZ+mfsN4uUlrebYbkXXEr9Tg5Kwb\\\\\\\/vmnljJGoR4EfS85jU4SHelND447r5QxSZDw8HxUHu\\\\\\\/+uOobhUA9XnfYK49FtdqFZem6ZdmoOHopyWO5wrlCTlYxGJimTIr9Ome7u+O4sMwEefTZZjN+3pivDIqQ4+dnLGstvXB\\\\\\\/Q9jnV57+Bhv7YOuV6xj+9nuNUcL3S8emU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022EG8ryYSZ4sQofaqGwG+q2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c1e37cc80eaeccea8c1788b2a4c1f63defffe132b864fa5354b42b23545e4db6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492153 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/601" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-79qhpArIqeHolXrf4GoVCcfy52MJcC9X-rGkCFrK5vA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/601" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CzNhgZmbgEYV8k1quvB9YoS74NFnSaQApQi3uK-1jAM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/2/591" title="Generation of H11-albumin-rtTA Transgenic Mice: A Tool for Inducible Gene Expression in the Liver" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/619" title="Improved Genome Sequence of Wild Emmer Wheat Zavitan with the Aid of Optical Maps" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492153" id="node492153" data-pisa="ggg;9/2/601" data-pisa-master="ggg;g3.118.200998" data-apath="/ggg/9/2/601.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Benchmarking Between Deep Learning, Support Vector Machine and Bayesian Threshold Best Linear Unbiased Prediction for Predicting Ordinal Traits in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Javier Martín-Vallejo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="4">Carlos M. Hernández-Suárez</span>, <span class="highwire-citation-author" data-delta="5">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="6">Philomin Juliana</span> and <span class="highwire-citation-author" data-delta="7">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">601-618;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200998 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492153" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Javier Martín-Vallejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Javier%2BMart%C3%ADn-Vallejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADn-Vallejo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJavier%2BMart%25C3%25ADn-Vallejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos M. Hernández-Suárez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Facultad de Ciencias, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BM.%2BHern%C3%A1ndez-Su%C3%A1rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hern%C3%A1ndez-Su%C3%A1rez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BM.%2BHern%25C3%25A1ndez-Su%25C3%25A1rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Guadalajara, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/2/601" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492153" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/601.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492153" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/2/601.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492153" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/2/601.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492153" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection is revolutionizing plant breeding. However, still lacking are better statistical models for ordinal phenotypes to improve the accuracy of the selection of candidate genotypes. For this reason, in this paper we explore the genomic based prediction performance of two popular machine learning methods: the Multi Layer Perceptron (MLP) and support vector machine (SVM) methods <em>vs.</em> the Bayesian threshold genomic best linear unbiased prediction (TGBLUP) model. We used the percentage of cases correctly classified (PCCC) as a metric to measure the prediction performance, and seven real data sets to evaluate the prediction accuracy, and found that the best predictions (in four out of the seven data sets) in terms of PCCC occurred under the TGLBUP model, while the worst occurred under the SVM method. Also, in general we found no statistical differences between using 1, 2 and 3 layers under the MLP models, which means that many times the conventional neuronal network model with only one layer is enough. However, although even that the TGBLUP model was better, we found that the predictions of MLP and SVM were very competitive with the advantage that the SVM was the most efficient in terms of the computational time required.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/threshold" class="hw-term hw-article-keyword hw-article-keyword-threshold" rel="nofollow">threshold</a></li><li class="kwd"><a href="/keyword/gblup" class="hw-term hw-article-keyword hw-article-keyword-gblup" rel="nofollow">GBLUP</a></li><li class="kwd"><a href="/keyword/deep-learning" class="hw-term hw-article-keyword hw-article-keyword-deep-learning" rel="nofollow">deep learning</a></li><li class="kwd"><a href="/keyword/support-vector-machine" class="hw-term hw-article-keyword hw-article-keyword-support-vector-machine" rel="nofollow">support vector machine</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/plant-breeding" class="hw-term hw-article-keyword hw-article-keyword-plant-breeding" rel="nofollow">plant breeding</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Plant breeding is a key scientific area for increasing the food production required to feed the people of our planet. The key step in plant breeding is selection, and conventional breeding is based on phenotypic selection. Breeders choose good offspring using their experience and the observed phenotypes of crops, so as to achieve genetic improvement of target traits (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2018</a>). Thanks to this area (and related areas of science), the genetic gain nowadays has reached a near-linear increase of 1% in grain yield yearly (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Oury <em>et al.</em> 2012</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fischer <em>et al.</em> 2014</a>). However, a linear increase of at least 2% is required to cope with the 2% yearly increase in the world population, which relies heavily on wheat products as a source of food (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">FAO, 2011</a>). For this reason, genomic selection (GS) is now being implemented in many plant breeding programs around the world. GS consists of genotyping (markers) and phenotyping individuals in the reference (training) population and, with the help of statistical models, predicting the phenotypes or breeding values of the candidates for selection in the testing (evaluation) population that only were genotyped. GS is revolutionizing plant breeding because it is not limited to traits determined by a few major genes and allows using a statistical model to establish the associations between markers and phenotypes and also to make predictions of unphenotyped individuals that help do a more comprehensive and reliable selection of candidate individuals. In this way, it is essential for accelerating genetic progress in crop breeding.</p><p id="p-4">Evidence that GS has the potential to revolutionize plant breeding continues to grow. For example, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bernardo and Yu (2007)</a> and <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Heffner <em>et al.</em> (2010)</a> found evidence of the higher genetic gain of GS compared to marker-assisted selection. Also, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> (2011)</a> found that GS is superior in terms of genetic gain compared to conventional pedigree breeding. <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kadam <em>et al.</em> (2016)</a> and <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Beukert <em>et al.</em> (2017)</a> showed that GS is even more efficient that conventional breeding when applied to hybrid crop breeding, because hybrid genotypes can be inferred from their inbred parents, leading to lower genotyping costs (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2018</a>). Even in prolific species where not so much gain is expected when implementing GS, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cleveland and Hickey (2013)</a> and <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lillehammer <em>et al.</em> (2013)</a> showed that GS can be cost-effective in pig breeding and estimated an increase in genetic progress of about 10% when using GS to breed pigs in Norway’s national program. For these reasons, GS has met with a lot of enthusiasm and, since GS has the potential to predict the breeding values of selection candidates at birth more accurately than the classic pedigree index, some breeding companies are re-designing their breeding programs. Consequently, animals (or plants) can be selected at an early age; in some cases, this is expected to double the rate of genetic improvement per year.</p><p id="p-5">Currently, GS is a potent, attractive and valuable plant breeding approach. This method will be integrated into many practical breeding programs in the near future with further advances and the maturing of its theory (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nakaya and Isobe 2012</a>). However, one of the key elements for increasing the power and efficiency of GS are the statistical models that are used for predicting the phenotypes or breeding values of individual candidates for selection. For this reason, there is currently an extensive area of research aimed at improving existing models and developing new models in order to increase the precision of candidate selection using GS.</p><p id="p-6">However, this task is challenging since specific models are needed for each type of response variable (phenotype). In two recent pioneer articles, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Montesinos-López <em>et al.</em> (2018a</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">b</a>) evaluated the prediction performance of univariate and multivariate neural network deep learning models for continuous response variables. In general, much more research had been done for quantitative (continuous) traits than for ordinal, binary and count traits. Categorical scores for disease susceptibility or resistance often are recorded in plant breeding. For example ordinal traits are very common in plant breeding programs for measuring disease incidence and severity, for sensory evaluation, such as perceived quality of a product (<em>e.g.</em>, taste, smell, color, decay), and plant development (<em>e.g.</em>, developmental stages, maturity). These types of data are often partially subjective since the scale indicates only relative order and not absolute amounts; therefore the intervals between successive categories might not be the same (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Simko and Piepho 2011</a>).</p><p id="p-7"><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Montesinos-López <em>et al.</em> (2015a)</a> introduced genomic models for analyzing ordinal characters and to assess the genomic based for ordered categorical phenotypes using a threshold model that is the counterpart of the Genomic Best Linear Unbiased Predictor (<em>i.e.</em>, TGBLUP). The threshold model TGBLUP relates hypothetical underlying scale to the outward categorical response and it was extended to account for genomic × environment interactions. The models that included G×E achieved up to 14% more gains in prediction accuracy as compared with the main effect models. <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Montesinos-López <em>et al.</em> (2015b)</a> implemented Bayesian logistic ordinal regression in the context of genomic-enabled prediction using the Pólya-Gamma data augmentation approach that produces a Gibbs sampler with similar full conditional distributions of the Bayesian probit ordinal regression.</p><p id="p-8">Machine Learning (ML) methods have been proposed in the academic literature as alternatives to statistical methods for predicting phenotypes or breeding values in the context of GS. ML has gained considerable prominence over the last decade fueled by a number of high profile applications in Autonomous Vehicles, intelligent robots, image and speech recognition, automatic translations, medical and law usage, as well as for beating champions in games like chess, Jeopardy, GO and poker (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Makridakis <em>et al.</em> 2018</a>). ML also has high profile applications in biological science research (genomics, proteomics or metabolomics) to extract features, functions, structure and molecular dynamics from raw biological sequence data (<em>e.g.</em>, DNA, RNA, and amino acids). <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alipanahi <em>et al.</em> (2015)</a> used deep learning to predict DNA- and RNA-binding proteins. <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> (2016)</a> developed a deep neural network framework to model structural features of restricted Boltzmann machines. <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pan and Shen (2017)</a> proposed a hybrid convolutional neural network-deep belief network model to predict restricted Boltzmann machine interaction sites and motifs on RNAs. <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Quang <em>et al.</em> (2015)</a> proposed a deep neural network model to annotate and identify pathogenicity in genetic variants.</p><p id="p-9">For stem rust in wheat, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ornella <em>et al.</em> (2012)</a> analyzed and compared the performance of Bayesian Lasso, ridge regression, and support vector machine. Bayesian Lasso and ridge regression had slightly superior prediction accuracy than support vector regression.</p><p id="p-10"><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ornella <em>et al.</em> (2014)</a> evaluated six regression models, Bayesian Lasso, ridge regression, random forest regression, Reproducing Kernel Hilbert space, and two support vector regression in several wheat rust data bases; the authors found that random forest regression and Reproducing Kernel Hilbert Space were the best models. Recently, in a very comprehensive review, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">González-Camacho <em>et al.</em> (2018)</a>, presented and discussed several ML methods applied in genomic selection to predict rust resistance in wheat as well as classifications and regression methods. These authors compared results from linear models with those from ML, random forest, support vector machine (SVM) and radial basis function neural network and they found that in general the SVM with linear kernel was the best in terms of genomic based prediction performance (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">González-Camacho <em>et al.</em> 2018</a>).</p><p id="p-11">As stated above, ML methods are applied in many domains of science and technology. However, it is still not clear if ML methods outperform conventional statistical models in terms of prediction performance, since there is only weak empirical evidence of the relative performance of ML methods. Most of the time, ML methods are supported by few real data sets, which raises questions about the statistical significance of the results and their generalization (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Makridakis <em>et al.</em> 2018</a>). No benchmarks are used to compare the accuracy of ML methods <em>vs.</em> alternative ones (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Makridakis <em>et al.</em> 2018</a>). For this reason, it is of paramount importance to objectively evaluate the relative performance of ML methods in terms of prediction performance as compared to the conventional GS models in order to improve prediction accuracy and the selection of candidate genotypes early in time. For these reasons, in this paper we compare the conventional GS model TGBLUP with two popular models of the machine learning domain –MLP and SVM models − with the goal of exploring its prediction accuracy and practical implementation in the GS context for ordinal traits. We compare the prediction performance of the three models with seven real data sets and cross-validation, using the percentage of cases correctly classified (PCCC) as a metric.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Implemented models</h3><div id="sec-3" class="subsection"><h4>Bayesian threshold genomic best linear unbiased prediction (TGBLUP):</h4><p id="p-12">We used <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-1.gif"/></noscript></span></span> to represent the ordinal response, that belong to exactly one of the C mutually exclusive categories, of the jth line in the ith environment with <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-2.gif"/></noscript></span></span> and we propose the following <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-3.gif"/></noscript></span></span> linear predictors to fully specified the model related to the C response probabilities that takes into account the genotype<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-4.gif"/></noscript></span></span>environment (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-5.gif"/></noscript></span></span>) interaction term:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-8.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-10.gif"/></noscript></span></span> denotes the c<sup>th</sup> predictor (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-11.gif"/></noscript></span></span> for the fixed and random effects combination, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-12.gif"/></noscript></span></span> represents the probability in environment i, line j in category c; and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-13.gif"/></noscript></span></span> is the threshold (intercept) for the c<sup>th</sup> predictor. <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-14.gif"/></noscript></span></span> represents environment <em>i</em> and is assumed fixed, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-15.gif"/></noscript></span></span> is the marker effect of genotype <em>j</em>, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-16.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-17.gif"/></noscript></span></span> interaction term. Distributions: <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-18.gif"/></noscript></span></span>Multinomial <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-19.gif"/></noscript></span></span>. Only <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-20.gif"/></noscript></span></span> link functions and cummulative probabilities are needed (estimated) since the cumulative probability of the last category C is 1 and having the first <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-21.gif"/></noscript></span></span> probabilities we can obtain the probability of the last category. <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-22.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-23.gif"/></noscript></span></span> is the Genomic Relationship Matrix (GRM) that is calculated as <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-24.gif"/></noscript></span></span> (as proposed by <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">VanRaden 2008</a>), where W is a matrix of scaled markers alleles of dimension <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-25.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-26.gif"/></noscript></span></span> is the genotypic variance and m denotes the number of markers. The <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-27.gif"/></noscript></span></span> matrix is a covariance matrix that contains the similarity between individuals based on marker information, rather than the expected similarity based on pedigree. <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-28.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-29.gif"/></noscript></span></span> is computed as <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-30.gif"/></noscript></span></span> of order <em>IJ</em>x<em>IJ</em> and ⊗ denotes the Kronecker product, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-31.gif"/></noscript></span></span> means that we assume independence between environments (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Montesinos-López <em>et al.</em> 2015a</a>,<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">b</a>) and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-32.gif"/></noscript></span></span> is the variance corresponding to the <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-33.gif"/></noscript></span></span> interaction term. <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-34.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-35.gif"/></noscript></span></span> were assumed known. <span class="underline">Link function</span>: cumulative probit: Φ(.) is the cumulative distribution function of a standard normal distribution (probit link) and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-36.gif"/></noscript></span></span> its corresponsing inverse. For this model, the inverse link is as follows: <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-37.gif"/></noscript></span></span>. Once we have estimates of <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-38.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-39.gif"/></noscript></span></span> we can estimate <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-40.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-41.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-42.gif"/></noscript></span></span> (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Montesinos-López <em>et al.</em>, 2015a</a>). This threshold model assumes that the process that gives rise to the observed categories is an underlying continuous variable with a normal distribution <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-43.gif"/></noscript></span></span>; where <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-44.gif"/></noscript></span></span> are called “liabilities,” <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-45.gif"/></noscript></span></span> (<em>e.g.</em>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gianola, 1982</a>, and <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sorensen <em>et al.</em> 1995</a>) and the ordinal categorical phenotypes with C categories are generated from the underlying phenotypic values, <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-46.gif"/></noscript></span></span>, as <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-47.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-48.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-49.gif"/></noscript></span></span> ....<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-50.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-51.gif"/></noscript></span></span>. The implementation of the TGBLUP model was done in the BGLR package of <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos and Pérez-Rodríguez (2014)</a> in the R statistical software (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">R Core Team 2018</a>).</p></div><div id="sec-4" class="subsection"><h4>Multi Layer Perceptron (MLP) for ordinal data:</h4><p id="p-13">The architecture of the deep learning method we implemented is depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>; this architecture is called densely connected network or feedforward neural network, since it consists of an input layer, an output layer (for univariate-trait modeling) and multiple hidden layers between the input and output layers. There are many other deep learning architectures (convolutional networks, recurrent networks, etc.), which can be found in <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gulli and Sujit (2017)</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Angermueller <em>et al.</em> (2016)</a> and <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chollet and Allaire (2017)</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example of a feedforward neural network with eight input variables (x1,..,x8), one output variable (y1) and two hidden layers with three neurons each. The input variables correspond to information of environments, genotypes and genotype&#xD7;environment interaction. The marker information is included in the genotype and genotype&#xD7;environment interaction through the Cholesky decomposition of the genomic relationship matrix. The output variable is the ordinal response variable that we are interested to predict." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example of a feedforward neural network with eight input variables (x1,..,x8), one output variable (y1) and two hidden layers with three neurons each. The input variables correspond to information of environments, genotypes and genotype&#xD7;environment interaction. The marker information is included in the genotype and genotype&#xD7;environment interaction through the Cholesky decomposition of the genomic relationship matrix. The output variable is the ordinal response variable that we are interested to predict.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F1.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/2/601/F1.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492098" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Example of a feedforward neural network with eight input variables (x1,..,x8), one output variable (y1) and two hidden layers with three neurons each. The input variables correspond to information of environments, genotypes and genotype×environment interaction. The marker information is included in the genotype and genotype×environment interaction through the Cholesky decomposition of the genomic relationship matrix. The output variable is the ordinal response variable that we are interested to predict.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">The architecture shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> was applied to seven ordinal real data sets with 1, 2 and 3 hidden layers and number of neurons (from 10 to 500 with increases of 20). The input variables of the MLP model (<span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-52.gif"/></noscript></span></span> i = 1,2,..,n; <em>p</em> = 1,2,..,<span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-53.gif"/></noscript></span></span>) given in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> are the result of concatenating the information on environments, the information on markers through the Cholesky decomposition of the genomic relationship matrix and the information on the genotype × environment interaction (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-54.gif"/></noscript></span></span>). Via weights, the input variables (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-55.gif"/></noscript></span></span>) and the units (neurons) of the hidden layers are connected. The information to the neurons in the first hidden layer is simply passed by the input variables. In the first hidden layer, the net input into the jth hidden unit is <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-56.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-57.gif"/></noscript></span></span> is the total number of input variables, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-58.gif"/></noscript></span></span> is the weight of input unit p to hidden unit j in the first hidden layer, <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-59.gif"/></noscript></span></span> is the value of the pth input variable and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-60.gif"/></noscript></span></span> is a bias specific to each neuron in layer 1. Then an activation function is applied to the net input of the jth hidden unit in the first hidden layer and outputs <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-61.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-62.gif"/></noscript></span></span>. Similarly, the net input that neuron <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-63.gif"/></noscript></span></span>in the second hidden layer receives is <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-64.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-65.gif"/></noscript></span></span> is the total number of input neurons that come from hidden layer 1 to neuron <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-66.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-67.gif"/></noscript></span></span> is the weight from unit j of layer 1 that goes to unit k in hidden layer 2, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-68.gif"/></noscript></span></span> is the value of the output of unit j in layer 1 and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-69.gif"/></noscript></span></span> is a bias specific term to neuron k in layer 2. Then an activation function is applied to the net input of the kth hidden unit in the second hidden layer and outputs <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-70.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-71.gif"/></noscript></span></span>. Similarly, the unique output unit receives a net input of <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-72.gif"/></noscript></span></span>, where M is the number of hidden units from hidden layer 2, and <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-73.gif"/></noscript></span></span> is the weight from hidden unit k in layer 2 to the unique output. Finally, the prediction of individuals for the unique trait is obtained as: <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-74.gif"/></noscript></span></span>. It is important to point out that we used the sigmoid and softmax activation functions in the output layer (<span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-75.gif"/></noscript></span></span> when the response variables were binary and ordinal respectively, since we are working with binary or ordinal phenotypes. However, for the hidden activations functions we implemented the rectified linear activation unit (RELU) function.</p><p id="p-16">The successful implementation of the MLP model depends on appropriately selecting the following hyperparameters: (1) number of units (U), (2) number of layers, (3) number of epochs (E), (4) type of regularization method and (5) type of activation function. An epoch means one pass (forward and backward) of the full training set through the neural network. Regarding the number of units, we used between 10 to 500 units with increases of 20, and with regard to the number of layers we used 1, 2 and 3; we used from 1 to 100 epochs, and the type of regularization we chose was dropout regularization for training the models (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gulli and Sujit 2017</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chollet and Allaire 2017</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Srivastava <em>et al.</em> 2014</a>). For more details on model selection in MLP models, we suggest reading the papers by <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Montesinos-López <em>et al.</em> (2018a</a>, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">b</a>), where the authors evaluate the prediction performance of univariate and multivariate deep learning models for continuous response variables. All MLP models were implemented in the keras package (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Chollet and Allaire 2017</a>) in the open-source software R (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">R Core Team 2018</a>).</p></div><div id="sec-5" class="subsection"><h4>Support vector machine:</h4><p id="p-17">Support Vector Machine is one of the most popular and efficient machine learning algorithms, which was proposed to the computer science community in the 1990s by <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Vapnik (1995)</a> and used mostly for classification problems. Its versatility and the fact that it performs well in the presence of a large number of predictors, even with a small number of cases, makes SVM very appealing for tackling a wide range of problems such as speech recognition, text categorization, image recognition, face detection, faulty card detection, junk mail classification, credit rating analysis, and cancer and diabetes classification, among others (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Attewell <em>et al.</em> 2015</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Byun and Lee 2002</a>). Briefly, SVM is the solution to the optimization problem:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-76.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-77.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-78.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-79.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-80.gif"/></noscript></span></span> are the coefficients of the maximum margin hyperplane. A hyperplane is a subspace whose dimension is one less than that of its original space. For a space in 3-dimensions its hyperplanes has 2-dimensions, while for a space in 2-dimensions, its hyperplanes is a line (one dimension). T is a non-negative tuning parameter that determines the number and severity of the violations to the margin (and to the hyperplane) that we will tolerate and is seen as the total amount of errors allowed since it is the bound of the sum of <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-81.gif"/></noscript></span></span>. For T close to zero, the soft-margin SVM allows very little error and is similar to the hard-margin classifier (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">James <em>et al.</em> 2013</a>). The larger T is, the more error is allowed, which in turn allows for wider margins. In practice, T is treated as a tuning parameter that is generally chosen via cross-validation. M is the width of the margin and we seek to make this quantity as large as possible. In (7), <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-82.gif"/></noscript></span></span> are slack (error) variables that allow individual observations to be on the wrong side of the margin or the hyperplane. The slack variable <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-83.gif"/></noscript></span></span> tells us where the ith observation is located, relative to the hyperplane and relative to the margin. If <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-84.gif"/></noscript></span></span>=0, then the ith observation is on the correct side of the margin, If <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-85.gif"/></noscript></span></span>&gt;0, then the ith observation is on the wrong side of the margin, and we say that the ith observation has violated the margin. If <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-86.gif"/></noscript></span></span>&gt;1, then it is on the wrong side of the hyperplane. Once we have solved (4)–(7), we classify a test observation <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-87.gif"/></noscript></span></span> by simply determining on which side of the hyperplane it lies. That is, we classify the test observation in the training/testing sets based on the sign of <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-88.gif"/></noscript></span></span> =<span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-89.gif"/></noscript></span></span>; if <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-90.gif"/></noscript></span></span>&lt;0, then the observation is assigned to the class corresponding to -1, but if <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-91.gif"/></noscript></span></span>&gt;0, then the observation is assigned to the class corresponding to 1 (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">James <em>et al.</em> 2013</a>). We chose <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-92.gif"/></noscript></span></span> as a nonlinear function of x and implemented the radial kernel. This type of kernel is a nonlinear function of x, but with fewer parameters than quadratic, cubic, or higher order expansion of x. The SVM with radial kernel [<span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-93.gif"/></noscript></span></span> was implemented with the R package e1071 in the R statistical software (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">R Core Team 2018</a>). Also, due to the fact that we work with binary (K = 2 classes) and ordinal (K &gt; 2 classes) data for the ordinal response variables we implemented the one-<em>vs.</em>-one classification approach that construct <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-94.gif"/></noscript></span></span> binary SVMs each of which compare a pair of classes. Each SVM compare the k<em>th</em> class coded as +1, to the k′<em>th</em> class, coded as -1. At prediction time, a voting scheme is applied: all <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-95.gif"/></noscript></span></span> binary SVMs are applied to an unseen sample and the class that got the highest number of “+1” predictions gets predicted by the combined classifier (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">James <em>et al.</em> 2013</a>).</p></div></div><div id="sec-6" class="subsection"><h3>Experimental data sets</h3><div id="sec-7" class="subsection"><h4>Phenotypic data sets:</h4><p id="p-18">In this study, we used the data set of <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Juliana <em>et al.</em> (2018)</a>. The data used belong to four elite yield trial (EYT) nurseries from the Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT). The EYT nurseries were planted in mid-November. They were planted in bed and flat planting systems in optimally irrigated environments and received 500 mm of water at the Norman E. Borlaug Research Station, Ciudad Obregon, Sonora, Mexico. The nurseries were sown in 39 trials, each comprising 28 lines and two high-yielding checks (Kachu and Borlaug) that were arranged in an alpha lattice design with three replications and six blocks. The nurseries were evaluated for the following traits: number of days from germination to 50% spike emergence (days to heading, DTHD), number of days from germination to 50% physiological maturity (days to maturity, DTMT), grain yield (GY, tons per hectare) and plant height (Height, centimeters). All these nurseries were evaluated during four seasons: 2013-2014 (EYT 13-14; here called <strong>data set 1</strong>), 2014-2015 (EYT 14-15; called <strong>data set 2</strong>), 2015-2016 (EYT 15-16; called <strong>data set 3)</strong> and 2016-2017 (EYT 16-17; called <strong>data set 4</strong>). It is important to point out that the trait GY was ignored for this application.</p><p id="p-19">Data set 1 included 767 lines, data set 2,775 lines, data set 3,964 lines, and data set 4, 980 lines (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Juliana <em>et al.</em> 2018</a>). In addition, in each season we studied six environments resulting from the level of irrigation (IR) and planting system (bed or flat) which we called: Bed2IR, Bed5IR, Flat5IR, FlatDrip, EHT and LHT. However, all these environments were not evaluated in all seasons (data sets). <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> gives the environments under study in each of the first four data sets.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492247/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492247/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Environments evaluated in data sets 1, 2, 3 and 4. Environments in data set 1 are Bed5IR (bed planting and 5 Irrigation levels), EHT (early heat stress), Flat5I (flat plating system and 5 irrigation levels), LTH (late heat stress). Environments in data set 2 are Bed2IR (bed planting and 2 Irrigation levels), Bed5IR (bed planting and 5 Irrigation levels), EHT (early heat stress), Flat5IR (flat plating system and 5 irrigation level), LTH (late heat stress). Environments in data set 3 are Bed2IR (bed planting and 2 Irrigation levels), Bed5IR (bed planting and 5 Irrigation levels), Flat5IR (flat plating system and 5 irrigation level), FlatDrip (flat planting system a drip irrigation). Environments in data set 4 are Bed5IR (bed planting and 5 Irrigation levels), EHT (early heat stress), Flat5IR (flat plating system and 5 irrigation level), FlatDrip (flat planting system a drip irrigation)</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">It is important to point out that here we used the BLUEs of each of the lines obtained (as suggested by <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Juliana <em>et al.</em> 2018</a>) adjusted for trials, blocks and replications in each data set. The three traits used were discretized because the original data sets are continuous, only to illustrate the proposed models. Traits DTHD and DTMT were discretized at quantiles 33.33% and 66.67% (in <strong>data sets 1</strong> and <strong>2</strong>) to obtain three categories, while trait Height was discretized at quantile 50% to obtain 2 categories (in <strong>data sets 1</strong>, <strong>2</strong>, <strong>3</strong> and <strong>4</strong>); the discretization process was done for each environment of each data set. For <strong>data sets 3</strong> and <strong>4</strong>, traits DTHD and DTMT were discretized at quantiles 20%, 45%, 70% and 90%.</p><p id="p-21"><strong>Data set 5</strong> is part of <strong>data set 3</strong>; for this reason, the phenotypic information and genomic information were obtained in the same way as in <strong>data set 3</strong>; however, only 964 lines had complete data on the total 980 lines under study in <strong>data set 3.</strong> But now the traits measured in <strong>data set 5</strong> were grain color (GC) (1 = yes, 2 = no), leaf rust (ordinal scale with 5 points), stripe rust (ordinal scale with 3 points) and GY, which is a continuous trait (this trait was not used because it is continuous). <strong>Data set 6</strong> and <strong>data set 7</strong> are part of the wheat yield trial (YT) nurseries from CIMMYT’s Global Wheat Breeding Program. For <strong>data set 6</strong>, the number of lines used was 945 and for <strong>data set 7</strong>, 1145 wheat lines were used. A continuous trait (grain yield, GY) and an ordinal trait (lodging, ordinal scale of 5 points) were measured on both data sets. However, only lodging was used in this paper due to the fact that the other trait is continuous.</p></div><div id="sec-8" class="subsection"><h4>Genotypic data:</h4><p id="p-22">The lines used in this study were 3,486 (79.807%) out of 4,368 lines that were evaluated in the four seasons (nurseries) comprising the <strong>data sets 1</strong>, <strong>2</strong>, <strong>3</strong>, and <strong>4</strong> were genotyped using genotyping-by-sequencing (GBS) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Poland <em>et al.</em> 2012</a>) at Kansas State University, using an Illumina HiSeq2500 for obtaining genome-wide markers. Marker polymorphisms were called across all lines using the TASSEL (Trait Analysis by Association Evolution and Linkage) GBS pipeline (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Glaubitz <em>et al.</em>, 2014</a>) and anchored to the International Wheat Genome Sequencing Consortium’s (IWGSC) first version of the reference sequence (RefSeq v1.0) assembly of the bread wheat variety Chinese Spring. Markers with more than 60% missing data, less than 5% minor allele frequency and percent heterozygosity greater than 10% were removed; as a result, we obtained 2,038 markers. Missing marker data were imputed using LinkImpute (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Money <em>et al.</em> 2015</a>) implemented in TASSEL (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bradbury <em>et al.</em> 2007</a>), version 5. The lines were also filtered for more than 50% missing data and we end up with 3,486 lines (79.807%) of the total 4,368 lines originally evaluated (767 lines from <strong>data set 1</strong>, 775 lines from <strong>data set 2</strong>, 964 lines from <strong>data set 3</strong> and 980 lines from <strong>data set 4</strong>) (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Juliana <em>et al.</em> 2018</a>). The lines used in <strong>data sets 5</strong>, <strong>6</strong>, and <strong>7</strong> were genotyped with the same marker system that was used for the other data sets.</p></div></div><div id="sec-9" class="subsection"><h3>Evaluation of prediction accuracy with cross-validation</h3><p id="p-23">The prediction accuracy of the three models under study (TGBLUP, MLP and SVM) was evaluated with an outer cross-validation (CV), while the prediction accuracy of the MLP and SVM models, in addition to outer cross-validation, was also evaluated using an inner cross-validation as was done in <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Montesinos-López <em>et al.</em> (2018a</a>, <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">b</a>). The outer CV was used for evaluating the prediction accuracy of the three models, while the inner CV was used for tuning the hyperparameters in the MLP and SVM models. In the outer CV, the original data set was partitioned into five subsamples of equal size and each time four of them were used for training (TRN) and the remaining one for testing (TST), that is, we implemented a fivefold cross-validation. In the design, some lines can be evaluated in some, but not all, target environments, which mimics a prediction problem faced by breeders in incomplete field trials. Our cross-validation strategy is the same as the strategy denoted as CV2 that was proposed and implemented by <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jarquín <em>et al.</em> (2017)</a>, where a certain portion of test lines in a certain portion of test environments is predicted, since some test lines that were evaluated in some test environments are assumed to be missing in others. We used the percentage of cases correctly classified (PCCC) for evaluating the prediction performance since our response variables were binary and ordinal and it was calculated from each trait-environment combination for each of the testing sets and the average of all folds was reported as a measure of prediction performance. It is important to point out that, to avoid biased results, the tuning step was done in each fold using only the training set. As mentioned above, we implemented an inner CV for the MLP and SVM models.</p><p id="p-24">As mentioned above, for the MLP and SVM methods we implemented an inner CV using the grid search method. For the MLP model, the grid for the number of epochs and units was explained above, where the MLP was presented and each training set of the outer CV was split, with 20% of the data in the inner testing set and the remaining 80% in the inner training set. This inner CV was implemented using the validation_split argument on the fit function of the keras library to avoid implementing manual k-fold cross-validation for the inner CV, which requires more computational resources (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Chollet and Allaire 2017</a>). However, for the SVM method, the inner cross-validation was implemented with a 10-fold cross-validation with the following values for parameters T and <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-96.gif"/></noscript></span></span> T = (1,1.2,1.4,1.6,1.8,2) and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-97.gif"/></noscript></span></span>. Then the outer CV was implemented with the best combination of T and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-98.gif"/></noscript></span></span>in each training set.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-25">Details of the phenotypic and genomic data of the seven data sets used in this study can be downloaded in the link: <a href="http://hdl.handle.net/11529/10548140">http://hdl.handle.net/11529/10548140</a>. The seven data sets have also trait grain yield (not used in this study). These seven data sets were also used in <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Montesinos-López <em>et al.</em> (2018c</a>, Manuscript submitted for publication) including trait grain yield.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><p id="p-26">The results are given in seven sections, one for each data set under study. Each section gives a descriptive analysis of each data set and the prediction accuracy obtained for each of the three models that we implemented. Also <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table A1</a> (Appendix) shows the Average percentage of cases correctly classified for each data set (data sets 1-7), model (SVM, TGBLUP and MLP), layer, type of interaction terms with genotype<span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-99.gif"/></noscript></span></span>environment (I) and without genotype<span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-100.gif"/></noscript></span></span>environment (WI), and trait.</p><div id="sec-12" class="subsection"><h3>Data set 1</h3><p id="p-27"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the percentages of individuals in each category for each trait where can be observed that the number of individuals in each category are different. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> gives the prediction accuracies for the three traits under the three methods (TGBLUP, MLP and SVM) with (I) and without (WI) the genotype<span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-101.gif"/></noscript></span></span>environment interaction. The best predictions with the interaction term (I) were observed under the TGBLUP model and the worst occurred under the SVM method and the range of predictions with (I) was between 0.5206 and 0.6773. Without the interaction term (WI), we did not find statistical differences between the three methods (TGBLUP, SVM and MLP) for the three traits under study, and the range of predictions was between 0.5992 and 0.6722. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> also provides the predictions for each trait with (I) and without (WI) the interaction term using 1, 2 and 3 layers under the MLP model. The <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> shows (subpanel b) that there are no statistical differences between the number of layers used (exist overlapping of the corresponding confidence intervals), which was not expected, since with deep learning methods, using more deep hidden layers helps to capture complex interactions which many times help to increase prediction accuracies.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of individuals in each category of the ordinal response for data set 1 across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of individuals in each category of the ordinal response for &lt;em&gt;data set 1&lt;/em&gt; across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F2.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/2/601/F2.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492254" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Percentage of individuals in each category of the ordinal response for <em><span class="underline">data set 1</span></em> across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Data set 1 in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Data set 1&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F3.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/2/601/F3.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492226" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child"><em><span class="underline">Data set 1</span></em> in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G×E term (I) and without the G×E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G×E term (I) and without the G×E term (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Data set 2</h3><p id="p-30">The percentages of individuals in each category for data set 2 are given in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, were we can see that each category has a different number of individuals. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> gives the predictions in terms of PCCC for the three methods under study for data set 2, which consists of three traits. The predictions are provided with genotype × environment interaction (I) and without genotype × environment interaction (WI). The predictions obtained with (I) ranged between 0.5297 and 0.7021. The best predictions with the interaction (I) term were observed under the TGBLUP model, and the worst under the SVM method. Without the interaction term (WI), the predictions ranged between 0.6371 and 0.6970, and we found no statistical differences between the three methods (TGBLUP, SVM and MLP) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). We also did not find statistical differences between the MLP models using 1, 2 and 3 layers, which means that the conventional neural network (with one hidden layer) is as good as the deep learning models that had two or three hidden layers.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of individuals in each category of the ordinal response for data set 2 across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of individuals in each category of the ordinal response for &lt;em&gt;data set 2&lt;/em&gt; across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F4.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/2/601/F4.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492086" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Percentage of individuals in each category of the ordinal response for <em><span class="underline">data set 2</span></em> across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Data set 2 in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Data set 2&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F5.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/2/601/F5.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492148" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child"><em><span class="underline">Data set 2</span></em> in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G×E term (I) and without the G×E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G×E term (I) and without the G×E term (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Data set 3</h3><p id="p-33">We can observe that there are different percentages of individuals in each category in data set 3 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The prediction accuracies in terms of PCCC for the three traits in this data set are given for the three methods evaluated with (I) and without (WI) the genotype<span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-102.gif"/></noscript></span></span>environment interaction term (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). When the genotype<span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-103.gif"/></noscript></span></span>environment interaction term was taken into account, the predictions ranged between 0.3630 and 0.6636, and the TGBLUP model was the best. On the other hand, when the genotype<span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-104.gif"/></noscript></span></span>environment interaction term was ignored, the predictions ranged between 0.3998 and 0.6377, and the best predictions occurred under the MLP method in two out of the three traits; however, the MLP was not statistically superior to the TGBLUP model. Finally, we found no significant differences using 1, 2 and 3 layers with the deep learning methods (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of individuals in each category of the ordinal response for data set 3 across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of individuals in each category of the ordinal response for &lt;em&gt;data set 3&lt;/em&gt; across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F6.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/2/601/F6.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492199" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">Percentage of individuals in each category of the ordinal response for <em><span class="underline">data set 3</span></em> across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Data set 3 in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Data set 3&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F7.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/2/601/F7.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492075" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-35" class="first-child"><em><span class="underline">Data set 3</span></em> in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G×E term (I) and without the G×E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G×E term (I) and without the G×E term (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Data set 4</h3><p id="p-36">In <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> it is observed that there are different percentages of individuals in each category in data set 4. For this data set, <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a> gives the prediction accuracies in terms of PCCC for each of the three traits evaluated under the three models (TGBLUP, SVM and MLP) with (I) and without (WI) the genotype<span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/2/601/embed/mml-math-105.gif"/></noscript></span></span>environment interaction term. When the genotype × environment interaction term was taken into account, the predictions ranged between 0.3720 and 0.6131, and the TGBLUP model produced the best predictions. However, when the genotype × environment interaction term was ignored, the predictions ranged between 0.4247 and 0.6028, and the best predictions were observed under the SVM method. We did not find statistical differences between using 1, 2 and 3 layers under the MLP model, which means that for this data set even the simple MLP model is enough for producing competitive predictions (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of individuals in each category of the ordinal response for data set 4 across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of individuals in each category of the ordinal response for &lt;em&gt;data set 4&lt;/em&gt; across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F8.medium.gif" width="440" height="257"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/2/601/F8.medium.gif" width="440" height="257"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492192" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-37" class="first-child">Percentage of individuals in each category of the ordinal response for <em><span class="underline">data set 4</span></em> across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Data set 4 in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Data set 4&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F9.medium.gif" width="440" height="249"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/9/2/601/F9.medium.gif" width="440" height="249"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492095" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-38" class="first-child"><em><span class="underline">Data set 4</span></em> in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height. (A) Prediction accuracy of TGBLUP, MLP with one layer and SVM models with the G×E term (I) and without the G×E term (WI) for each trait; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model with the G×E term (I) and without the G×E term (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Data set 5</h3><p id="p-39">There are different percentages of individuals in each category in data set 5 (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>). <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure 11</a> gives the prediction accuracies in terms of PCCC for the three traits under study under the three models (TGBLUP, SVM and MLP). We can see that for trait grain color (GC), the best prediction occurred under the SVM method and the worst under the MLP model; however, there were no significant differences between the three methods in any of the three traits under study, and the predictions ranged between 0.5082 and 0.5273. Also, it is important to point out that no significant differences were found for the MLP model with 1, 2 and 3 hidden layers (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Figure 11</a>).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of individuals in each category of the ordinal response for data set 5 for traits grain color (GC), Leaf Rust and Stripe Rust." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of individuals in each category of the ordinal response for &lt;em&gt;data set 5&lt;/em&gt; for traits grain color (GC), Leaf Rust and Stripe Rust.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F10.medium.gif" width="440" height="311"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/9/2/601/F10.medium.gif" width="440" height="311"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492259" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-40" class="first-child">Percentage of individuals in each category of the ordinal response for <em><span class="underline">data set 5</span></em> for traits grain color (GC), Leaf Rust and Stripe Rust.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Data set 5 in terms of percentage of cases correctly classified (PCCC) for traits grain color (GC), leaf rust and stripe rust. (A) Prediction accuracy of TGBLUP, SVM and MLP with one layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Data set 5&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for traits grain color (GC), leaf rust and stripe rust. (A) Prediction accuracy of TGBLUP, SVM and MLP with one layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F11.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11" src="https://www.g3journal.org/content/ggg/9/2/601/F11.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492242" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11</span> <p id="p-41" class="first-child"><em><span class="underline">Data set 5</span></em> in terms of percentage of cases correctly classified (PCCC) for traits grain color (GC), leaf rust and stripe rust. (A) Prediction accuracy of TGBLUP, SVM and MLP with one layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Data set 6</h3><p id="p-42">In each category there are different percentages of individuals for data set 6 (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure 12</a>). For this data set, <a id="xref-fig-13-1" class="xref-fig" href="#F13">Figure 13</a> does not show statistical differences between the three methods under study, but the predictions of TGBLUP were better than the predictions of the SVM and MLP models by 12.54% and 3.47%, respectively, while the MLP model was superior to the SVM model by 9.07%. The predictions ranged between 0.3979 and 0.4550. <a id="xref-fig-13-2" class="xref-fig" href="#F13">Figure 13</a> also shows that no significant differences were found using 1, 2 and 3 layers with the MLP model (<a id="xref-fig-13-3" class="xref-fig" href="#F13">Figure 13</a>).</p><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of individuals in each category of the ordinal response (Lodging) for data set 6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of individuals in each category of the ordinal response (Lodging) for &lt;em&gt;data&lt;/em&gt; set 6.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 12" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F12.medium.gif" width="440" height="227"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 12" src="https://www.g3journal.org/content/ggg/9/2/601/F12.medium.gif" width="440" height="227"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 12" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492249" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 12</span> <p id="p-43" class="first-child">Percentage of individuals in each category of the ordinal response (Lodging) for <em><span class="underline">data</span></em> set 6.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Data set 6 in terms of percentage of cases correctly classified (PCCC) for trait Lodging. (A) Prediction accuracy of TGBLUP, SVM and MLP with one layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Data set 6&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for trait Lodging. (A) Prediction accuracy of TGBLUP, SVM and MLP with one layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 13" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F13.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 13" src="https://www.g3journal.org/content/ggg/9/2/601/F13.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 13" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492218" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 13</span> <p id="p-44" class="first-child"><em><span class="underline">Data set 6</span></em> in terms of percentage of cases correctly classified (PCCC) for trait Lodging. (A) Prediction accuracy of TGBLUP, SVM and MLP with one layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Data set 7</h3><p id="p-45">In each category there are different percentages of individuals for data set 7 (<a id="xref-fig-14-1" class="xref-fig" href="#F14">Figure 14</a>). For this data set, no statistical differences were found between the three models (TGBLUP, SVM and MLP) for the trait Lodging, but the TGBLUP model was slightly better than the SVM and MLP models by 3.28% and 3.62%, respectively (<a id="xref-fig-15-1" class="xref-fig" href="#F15">Figure 15</a>). The predictions for this trait under the three methods ranged between 0.5082 and 0.5273, respectively. Finally, also in this data set we did not find significant differences between using 1, 2 and 3 layers in the MLP model (<a id="xref-fig-15-2" class="xref-fig" href="#F15">Figure 15</a>).</p><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of individuals in each category of the ordinal response (Lodging) for data set 7." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of individuals in each category of the ordinal response (Lodging) for &lt;em&gt;data&lt;/em&gt; set 7.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 14" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F14.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 14" src="https://www.g3journal.org/content/ggg/9/2/601/F14.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 14" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492122" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 14</span> <p id="p-46" class="first-child">Percentage of individuals in each category of the ordinal response (Lodging) for <em><span class="underline">data</span></em> set 7.</p><div class="sb-div caption-clear"></div></div></div><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Data set 7 in terms of percentage of cases correctly classified (PCCC) for trait Lodging. (A) Prediction accuracy of TGBLUP, SVM and MLP with one layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Data set 7&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for trait Lodging. (A) Prediction accuracy of TGBLUP, SVM and MLP with one layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 15" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F15.medium.gif" width="440" height="268"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 15" src="https://www.g3journal.org/content/ggg/9/2/601/F15.medium.gif" width="440" height="268"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 15" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492172" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 15</span> <p id="p-47" class="first-child"><em><span class="underline">Data set 7</span></em> in terms of percentage of cases correctly classified (PCCC) for trait Lodging. (A) Prediction accuracy of TGBLUP, SVM and MLP with one layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MLP model.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Meta-comparison across environments and traits</h3><p id="p-48">Finally, <a id="xref-fig-16-1" class="xref-fig" href="#F16">Figure 16</a> provided a meta-picture across environments and traits. When the interaction term was taken into account in the four data sets (1, 2, 3, and 4) the TGBLUP presented the best prediction accuracies, but not significantly differences were observed with those of the MLP model. While when the interaction term was ignored the three methods in the first four data sets perform very similar in terms of prediction performance. Also, in the meta-picture of data sets 5, 6 and 7 we found only in data set 6 a better performance of TGBLUP model but not statistical differences were found with the other two methods (SVM and MLP).</p><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/2/601/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A meta-comparison across traits of the seven data sets in terms of percentage of cases correctly classified (PCCC) for the three methods under study (TGBLUP, SVM and MLP)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1868536675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A meta-comparison across traits of the seven data sets in terms of percentage of cases correctly classified (PCCC) for the three methods under study (TGBLUP, SVM and MLP).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 16" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/2/601/F16.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 16" src="https://www.g3journal.org/content/ggg/9/2/601/F16.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/2/601/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 16" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/2/601/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492271" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 16</span> <p id="p-49" class="first-child">A meta-comparison across traits of the seven data sets in terms of percentage of cases correctly classified (PCCC) for the three methods under study (TGBLUP, SVM and MLP).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-50">Due to the need for more powerful prediction models in the GS context to improve the selection process of candidate individuals in plant (or animal) breeding programs, we performed a benchmark study between two popular ML models (MLP and SVM) and the TGBLUP model with ordinal response variables (ordinal traits). This benchmark study is very important because many claim that ML methods outperform conventional genomic selection models in terms of prediction accuracy; however, there is not enough empirical evidence in the literature to support this claim. As stated by <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Makridakis <em>et al.</em> (2018)</a>, simply because the models are new or based on ML (or artificial intelligence) is not enough to persuade users of their practical advantages over alternative or conventional methods. For this reason, it is of paramount importance to properly evaluate the performance of ML methods using a wide range of diverse data sets and compare them to alternative or conventional models in order to obtain enough evidence of their prediction performance. For this reason, in this publication we used seven real data sets for evaluating the prediction performance of two machine learning methods, the MLP and SVM methods, against the TGBLUP model. We found that in general (4 out of 7 data sets), the best performance in terms of prediction accuracy for ordinal data using as a metric the percentage of cases correctly classified was obtained with the TGBLUP model, a conventional Bayesian method with weakly informative priors. However, it is important to point out that the predictions of both machine ML methods (MLP and SVM) were very competitive with the predictions of the TGBLUP model but not better, since the MLP method outperformed the TGBLUP model in two of the seven data sets under study, while the SVM method only outperformed it in one of the seven data sets used (across environments). For this reason, we agree with <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Makridakis <em>et al.</em> (2018)</a>, who pointed out that it should become clear that ML methods are not a panacea that would automatically improve forecasting accuracy. Their capabilities can easily generate implausible solutions, leading to exaggerated claims of their potential and must be carefully investigated before any claims can be accepted. This latter claim that ML methods can easily generate implausible solutions is supported by the fact that their successful application depends strongly on the tuning process, which is very challenging, so that when this process is not appropriate, the resulting solutions are very implausible.</p><p id="p-51">Also, it is important to point out that for convenience we discretized the response variables in four out of the 7 data sets. However, there is evidence that the process of discretization of continuous variables produces a significant loss of information and this was studied by Kizikaya <em>et al.</em> (2014). When comparing the prediction accuracy using a threshold model for ordinal data and a model for continuous response variable using a data set with an ordinal response variable, Kizikaya <em>et al.</em> (2014) showed that to reach the same level of predictive capacity, the model of continuous response variable needs a training sample 2.25 times higher than when the threshold model is implemented. Also, Kizikaya <em>et al.</em> (2014) showed that using the same size of the training sample (TRN) the model for continuous response variable produces lower predictive capacity (16% lower) than using a threshold model.</p><p id="p-52">A favorable attribute of many supervised learning algorithms is that there is no restriction on the distribution of response variables; this makes them less sensitive to the problems that arise in parametric models when ordinal scores are used to quantify diseases. Our results found using the MLP, the SVM methods, and the Bayesian threshold genomic best linear unbiased prediction (TGBLUP) model using ordinal data are similar to those found by other authors. For example, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Ornella <em>et al.</em> (2012)</a> compared the performance of ridge regression with Bayesian Lasso and two Support Vector Regression for predicting ordinal traits stem rust and yellow rust resistance in five wheat populations. The Bayesian Lasso and the Ridge Regression had similar prediction performance and with a small superiority over Support Vector Regression. <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">González-Camacho <em>et al.</em> (2018)</a> compared the performance of several regression/classification ML models against some parametric models (Bayesian LASSO, ridge regression, etc.) on stem rust and yellow rust and found similar results as those found by <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Ornella <em>et al.</em> (2012)</a>.</p><p id="p-53">Our results support the idea that the ML methods are not the panacea for prediction modeling and we invite the users of prediction models not to blindly adopt or accept any model and assume it is the best in terms of prediction performance, since more empirical evidence is needed and a lot of research supports the idea that there is no universal model or machine (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Montesinos-López <em>et al.</em>, 2018a</a>,<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">b</a>,<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">c</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bellot <em>et al.</em>, 2018</a>). However, we need to be willing to test and adopt models coming from other areas since we need to improve the prediction accuracy of existing GS models to be able to really revolutionize plant and animal breeding and produce the required amount of food that the world needs and will need in the coming years without bringing more land under cultivation.</p><p id="p-54">Also, we are convinced that more benchmark studies with a wide range of real data sets should be conducted to be able to clearly identify those models that perform better in terms of prediction accuracy (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Benjamin, <em>et al.</em>, 2018</a>). When researchers propose new models for prediction, they should compare their models to conventional models and use many data sets so as to provide enough evidence that the new proposed models actually outperform conventional models, since a lot of the new proposed models are not better than conventional models in terms of prediction accuracy. However, many times the new models are more complex to understand and to implement, which does not really help satisfy the need for more powerful prediction models and for advancing in the field of prediction modeling.</p><p id="p-55">Another disadvantage of the ML methods is that a preprocessing step is needed to be able to apply them successfully, since the results depend on the type of transformation or preprocessing that is used (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Chollet and Allaire 2017</a>). For this reason, ML methods need to be improved to perform the tuning process and choose the appropriate preprocessing step automatically to be able to implement these methods more easily and with less possibility of error due to the fact that the tuning process and preprocessing step are very challenging.</p><p id="p-56">Finally, with the seven real data sets used we found that the ML models (SVM and MLP) are very competitive with the TGBLUP model and can be implemented in the GS context since software in the ML domain has been developed that is very easy to use and works reasonably well with moderately large data sets. For this reason, we encourage other scientists to perform more benchmark studies and compare the existing GS models to ML methods, since the ML community had developed many efficient and friendly software to implement prediction models that can be used successfully in GS. We are convinced that by exploring the theory behind ML methods, we will find a lot of opportunities to improve the prediction performance of these methods and those of GS since this area had long time working in developing prediction models.</p><div id="sec-21" class="subsection"><h3>Conclusions</h3><p id="p-57">In this paper, we explored two very popular machine learning methods, the MLP and SVM models, and compared them to the TGBLUP model in terms of prediction performance of ordinal data using seven real data sets. We found that in general (four out of seven data sets), the TGBLUP model was the best in terms of prediction accuracy using the percentage of cases correctly classified as a metric, followed by the MLP and SVM models. However, the two machine learning methods were very competitive since they produced very similar predictions to those of the TGBLUP model and, in some cases, outperformed the TGBLUP model. The disadvantage of both machine learning methods is that, to produce reasonable predictions, they require a tuning process that is challenging since it is both an art and a scientific process. However, despite these difficulties, we found that both machine learning algorithms are very competitive and practical to implement in GS because they are easy to implement using the existing software and work efficiently with moderately large data sets.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">We thank all scientists, field workers, and lab assistants from National Programs and CIMMYT who collected the data used in this study. We acknowledge the financial support provided by the Foundation for Research Levy on Agricultural Products (FFL) and the Agricultural Agreement Research Fund (JA) in Norway through NFR grant 267806. We are also thankful for the financial support provided by CIMMYT CRP (maize and wheat), the Bill &amp; Melinda Gates Foundation, as well the USAID projects (Cornell University and Kansas State University) that generated the CIMMYT wheat data analyzed in this study. We are thankful to two anonymous reviewers that invested long hours revising the article and writing the suggestions that significantly improved the quality of the manuscript. We are also thankful to the Senior Editor of G3 Genes|Genomes|Genetics Prof. Dr. D.J de Koning for his rigorous scientific comments and always positive attitude toward the work done.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-22"><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492205/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492205/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A1</span> <span class="caption-title">Average percentage of cases correctly classified (PCCC) for each data set, model, layer, type of interaction terms with genotype×environment (I) and without genotype×environment (WI), and trait DTHD = days to heading; DTMT = days to maturity; Height = plant height). SE denotes standard error. For data sets 1-4, the averages are across environments</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-59"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-11-22"><span class="received-label">Received </span>November 22, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-12-27"><span class="accepted-label">Accepted </span>December 27, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Montesinos-López <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.1" data-doi="10.1007/s00122-011-1587-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-based prediction of testcross values in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1587-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAlbrecht%26rft.auinit1%253DT.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DGenome-based%2Bprediction%2Bof%2Btestcross%2Bvalues%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1587-7%26rft_id%253Dinfo%253Apmid%252F21505832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1587-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505832&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.2" data-doi="10.1038/nbt.3300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alipanahi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frey</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">838</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D831%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3300%26rft_id%253Dinfo%253Apmid%252F26213851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26213851&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.3" data-doi="10.15252/msb.20156651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angermueller</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pärnamaa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Deep learning for computational biology.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">878</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.15252/msb.20156651</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DAngermueller%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D878%26rft.epage%253D878%26rft.atitle%253DDeep%2Blearning%2Bfor%2Bcomputational%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20156651%26rft_id%253Dinfo%253Apmid%252F27474269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjg6IjEyLzcvODc4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzYwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.601.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Attewell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monaghan</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwong</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source">Data Mining for the Social Sciences: An Introduction</span>, <span class="cit-publ-name">University of California Press</span>, <span class="cit-publ-loc">Oakland, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.5" data-doi="10.1534/genetics.118.301298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bellot</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Can Deep Learning Improve Genomic Prediction of Complex Human Traits?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">819</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301298</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301298%26rft_id%253Dinfo%253Apmid%252F30171033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEwLzMvODA5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8yLzYwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.6" data-doi="10.3389/fncom.2018.00056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamin</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomlinson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramkumar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VerSteeg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Modern Machine Learning as a Benchmark for Fitting Neural Responses.</span> <abbr class="cit-jnl-abbrev">Front. Comput. Neurosci.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fncom.2018.00056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BComput.%2BNeurosci.%26rft.volume%253D12%26rft.spage%253D56%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffncom.2018.00056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fncom.2018.00056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.7" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genome-wide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2006.11.0690</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.8" data-doi="10.1186/s12284-017-0163-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beukert</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramachandra</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-based identification of heterotic patterns in rice.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12284-017-0163-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D10%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12284-017-0163-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12284-017-0163-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.9" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: Software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.2.601.10" data-doi="10.1007/3-540-45665-1_17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byun</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, (<span class="cit-pub-date">2002</span>). Applications of Support Vector Machines for Pattern Recognition: A Survey. In: SVM ’02 Proceedings of the First International Workshop on Pattern Recognition with Support Vector Machines, <span class="cit-fpage">213</span>–<span class="cit-lpage">236</span>. <span class="cit-publ-loc">London, UK</span>: <span class="cit-publ-name">Springer</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/3-540-45665-1_17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1007%252F3-540-45665-1_17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/3-540-45665-1_17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.601.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chollet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allaire</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source"><em>Deep Learning with R. Manning Publications</em>, <em>Manning Early Access Program</em></span>, <span class="cit-edition">Ed. 1st</span>. <span class="cit-publ-name">MEA</span>, <span class="cit-publ-loc">New Delhi, India</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.12" data-doi="10.2527/jas.2013-6270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Practical implementation of cost-effective genomic selection in commercial pig breeding using imputation.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">3583</span>–<span class="cit-lpage">3592</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2013-6270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2013-6270%26rft_id%253Dinfo%253Apmid%252F23736050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2013-6270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23736050&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-9.2.601.13"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2014 Bayesian Generalized Linear Regression. R package version 1.0.4. <a href="http://CRAN.R-project.org/package=BGLR">http://CRAN.R-project.org/package=BGLR</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.14" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Robust, Simple Genotyping-by-Sequencing (GBS) Approach for High Diversity Species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.601.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">FAO</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">The State of the World’s Land and Water Resources for Food and Agriculture: Managing Systems at Risk</span>, <span class="cit-publ-name">Food and Agriculture Organization of the United Nations</span>, <span class="cit-publ-loc">Rome, Italy</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.601.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byerlee</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edmeades</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Crop Yields and Global Food Security</span>, <span class="cit-publ-name">ACIAR</span>, <span class="cit-publ-loc">Canberra, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.17" data-doi="10.2527/jas1982.5451079x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Theory and analysis of threshold characters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1079</span>–<span class="cit-lpage">1096</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas1982.5451079x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D1079%26rft.epage%253D1096%26rft.atitle%253DTheory%2Band%2BAnalysis%2Bof%2BThreshold%2BCharacters%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1982.5451079x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1982.5451079x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1982NP93300025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.18" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS : A High Capacity Genotyping by Sequencing Analysis Pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.19" data-doi="10.3835/plantgenome2017.11.0104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Applications of Machine Learning Methods to Genomic Selection in Breeding Wheat for Rust Resistance.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.11.0104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.11.0104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.11.0104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.601.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gulli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sujit</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">Deep Learning with Keras: Implementing deep learning models and neural networks with the power of Python</span>, <span class="cit-publ-name">Packt Publishing Ltd.</span>, <span class="cit-publ-loc">Birmingham, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.21" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.11.0662</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.601.22" data-doi="10.1007/978-1-4614-7138-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witten</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">An Introduction to Statistical Learning: With Applications in R</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4614-7138-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAn%2BIntroduction%2Bto%2BStatistical%2BLearning%253A%2BWith%2BApplications%2Bin%2BR%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4614-7138-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4614-7138-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.23" data-doi="10.3835/plantgenome2016.12.0130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemes da Silva</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.12.0130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.12.0130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.12.0130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.24" data-doi="10.3835/plantgenome2017.11.0104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juliana</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Prospects and challenges of applied genomic selection-a new paradigm in breeding for grain yield in bread wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2018.03.0017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.11.0104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.11.0104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.25" data-doi="10.1534/g3.116.031286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadam</span>  <span class="cit-name-given-names">D.C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potts</span>  <span class="cit-name-given-names">S.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohn</span>  <span class="cit-name-given-names">M.O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A.E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A.J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Genomic prediction of single crosses in the early stages of a maize hybrid breeding pipeline. G3-Genes Genomes Genet. 6: 3443–3453.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.031286</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.031286%26rft_id%253Dinfo%253Apmid%252F27646704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjYvMTEvMzQ0MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi82MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.2.601.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tait</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reecy</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole genome analysis of infectious bovine kerato conjunctivitis in Angus cattle using Bayesian threshold models</span>. <abbr class="cit-jnl-abbrev">BMC Proc</abbr>. <span class="cit-vol">5</span>: <span class="cit-fpage">S22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc%26rft.volume%253D5%26rft.spage%253DS22%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.27" data-doi="10.2527/jas.2012-5113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lillehammer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection for two traits in a maternal pig breeding scheme.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">3079</span>–<span class="cit-lpage">3087</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2012-5113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2012-5113%26rft_id%253Dinfo%253Apmid%252F23658351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2012-5113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23658351&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.28" data-doi="10.1371/journal.pone.0194889"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makridakis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiliotis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Assimakopoulos</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Statistical and Machine Learning forecasting methods: Concerns and ways forward.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0194889</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0194889</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0194889%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0194889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0194889&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.29" data-doi="10.1534/g3.115.021667"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Money</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Migicovsky</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwaninger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> LinkImpute : Fast and Accurate Genotype Imputation for Nonmodel Organisms. G3 Genes|Genomes|Genetics 5:2383<span class="cit-pub-id-sep cit-pub-id-doi-sep">–2390.</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.021667</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.021667%26rft_id%253Dinfo%253Apmid%252F26377960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTEvMjM4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi82MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskridge</span>  <span class="cit-name-given-names">K.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> Threshold models for genome-enabled prediction of ordinal categorical traits in plant breeding. G3: Genes| Genomes| Genetics 5(1): 291–300.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.016188</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskridge</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression. G3: Genes|Genomes|Genetics 5(10): 2113–2126.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.021154</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.32" data-doi="10.1534/g3.118.200740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Suárez</span>  <span class="cit-name-given-names">C.M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> Multi-environment genomic prediction of plant traits using deep learners with a dense architecture. G3: Genes|Genomes|Genetics, 8(12): 3813–3828.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200740</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200740%26rft_id%253Dinfo%253Apmid%252F30291107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjgvMTIvMzgxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMi82MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Suárez</span>  <span class="cit-name-given-names">C.M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> Multi-trait, multi-environment deep learning modeling for genomic-enabled prediction of plant traits. G3: Genes|Genomes|Genetics, 8(12):3829–3840.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200728</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-other" id="cit-9.2.601.34"><div class="cit-metadata unstructured">Montesinos-López, O. A. Martín-Vallejo, J., Crossa, J. Gianola, D. Hernández-Suárez, C. M. Montesinos-López, A. Juliana, P. Singh, R., 2018c New deep learning genomic prediction model for multi-traits with mixed binary, ordinal, and continuous phenotypes. Manuscript submitted for publication.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.35" data-doi="10.1093/aob/mcs109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakaya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Will genomic selection be a practical method for plant breeding?</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1303</span>–<span class="cit-lpage">1316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/aob/mcs109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcs109%26rft_id%253Dinfo%253Apmid%252F22645117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcs109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22645117&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.36" data-doi="10.3835/plantgenome2012.07.0017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of genetic values for resistance to wheat rusts.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.07.0017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.07.0017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.07.0017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.37" data-doi="10.1038/hdy.2013.144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tapia</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic-enabled prediction with classification algorithms.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">616</span>–<span class="cit-lpage">626</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.144</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D616%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.38" data-doi="10.1016/j.eja.2012.02.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oury</span>  <span class="cit-name-given-names">F.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mailliard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chassin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardet</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A study of genetic progress due to selection reveals a negative effect of climate change on bread wheat yield in France.</span> <abbr class="cit-jnl-abbrev">Eur. J. Agron.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.eja.2012.02.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BAgron.%26rft.volume%253D40%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.eja.2012.02.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.eja.2012.02.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.39" data-doi="10.1186/s12859-017-1561-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">H.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">RNA-protein binding motifs mining with a new hybrid deep learning based cross-domain knowledge integration approach.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">136</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-017-1561-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D18%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-017-1561-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-017-1561-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.40" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0032253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.41" data-doi="10.1093/bioinformatics/btu703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dann: a deep learning approach for annotating the pathogenicity of genetic variants.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">763</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu703</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu703%26rft_id%253Dinfo%253Apmid%252F25338716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25338716&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-web" id="cit-9.2.601.42"><div class="cit-metadata unstructured">R Core Team, (2018). R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna. Austria. ISBN 3–900051–07–0. URL <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.43" data-doi="10.1016/j.tplants.2011.02.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Combining phenotypic data from ordinal rating scales in multiple plant experiments.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">237</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2011.02.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DSimko%26rft.auinit1%253DI.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D235%26rft.epage%253D237%26rft.atitle%253DCombining%2Bphenotypic%2Bdata%2Bfrom%2Bordinal%2Brating%2Bscales%2Bin%2Bmultiple%2Bplant%2Bexperiments.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2011.02.001%26rft_id%253Dinfo%253Apmid%252F21367649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2011.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21367649&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.44" data-doi="10.1051/gse:19950303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korsgaard</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayesian inference in threshold mode ls using Gibbs sampling.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-27-3-229</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D27%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19950303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19950303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RE69800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krizhevsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutskever</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salakhutdinov</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dropout: A Simple Way to Prevent Neural Networks from Overfitting.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1929</span>–<span class="cit-lpage">1958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D15%26rft.spage%253D1929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.46" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient method to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-9.2.601.47" data-doi="10.1007/978-1-4757-2440-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vapnik</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-source">The Nature of Statistical Learning Theory</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York, NY</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4757-2440-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BNature%2Bof%2BStatistical%2BLearning%2BTheory%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4757-2440-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4757-2440-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.48" data-doi="10.1016/j.cj.2018.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xua</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic selection methods for crop improvement: Current status and prospects.</span> <abbr class="cit-jnl-abbrev">Crop J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">340</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cj.2018.03.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BJ.%26rft.volume%253D6%26rft.spage%253D330%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cj.2018.03.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cj.2018.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.2.601.49" data-doi="10.1093/nar/gkv1025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A deep learning framework for modeling structural features of RNA-binding protein targets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">e32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1025%26rft_id%253Dinfo%253Apmid%252F26467480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26467480&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F2%2F601.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/2/601.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/2/591" title="Generation of H11-albumin-rtTA Transgenic Mice: A Tool for Inducible Gene Expression in the Liver" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/619" title="Improved Genome Sequence of Wild Emmer Wheat Zavitan with the Aid of Optical Maps" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/2"><span>Volume 9 Issue 2, February 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/2.cover-source.jpg?itok=Pgd0L52l" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/2/601.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kcByLbUBu5BraMLkDsE4d_ntIcvbeMtNuLfIBo5rriU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/2/601" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/2/601" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/2/601" class="active" data-icon-position="" data-hide-link-title="0">A Benchmarking Between Deep Learning, Support Vector Machine and Bayesian Threshold Best Linear Unbiased Prediction for Predicting Ordinal Traits in Plant Breeding</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492153" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-15hnCinOZNsoAQUV46AyWHq1lyJww9JlFmx1HQuyJN0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/2/601.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/2/601" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-rHShczkA8dttbyolM8otWyTTE_5ntVUUT1ZAAlZat5U" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/2/601" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/2/601.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492153" id="node492153--2" data-pisa="ggg;9/2/601" data-pisa-master="ggg;g3.118.200998" data-apath="/ggg/9/2/601.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Benchmarking Between Deep Learning, Support Vector Machine and Bayesian Threshold Best Linear Unbiased Prediction for Predicting Ordinal Traits in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Javier Martín-Vallejo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="4">Carlos M. Hernández-Suárez</span>, <span class="highwire-citation-author" data-delta="5">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="6">Philomin Juliana</span> and <span class="highwire-citation-author" data-delta="7">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">601-618;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200998 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492153--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Javier Martín-Vallejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Javier%2BMart%C3%ADn-Vallejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADn-Vallejo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJavier%2BMart%25C3%25ADn-Vallejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos M. Hernández-Suárez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Facultad de Ciencias, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BM.%2BHern%C3%A1ndez-Su%C3%A1rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hern%C3%A1ndez-Su%C3%A1rez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BM.%2BHern%25C3%25A1ndez-Su%25C3%25A1rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Guadalajara, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/2/601" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/2/601&amp;title=A%20Benchmarking%20Between%20Deep%20Learning%2C%20Support%20Vector%20Machine%20and%20Bayesian%20Threshold%20Best%20Linear%20Unbiased%20Prediction%20for%20Predicting%20Ordinal%20Traits%20in%20Plant%20Breeding" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/2/601&amp;title=A%20Benchmarking%20Between%20Deep%20Learning%2C%20Support%20Vector%20Machine%20and%20Bayesian%20Threshold%20Best%20Linear%20Unbiased%20Prediction%20for%20Predicting%20Ordinal%20Traits%20in%20Plant%20Breeding" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/2/601&amp;title=A%20Benchmarking%20Between%20Deep%20Learning%2C%20Support%20Vector%20Machine%20and%20Bayesian%20Threshold%20Best%20Linear%20Unbiased%20Prediction%20for%20Predicting%20Ordinal%20Traits%20in%20Plant%20Breeding" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/2/601&amp;text=A%20Benchmarking%20Between%20Deep%20Learning%2C%20Support%20Vector%20Machine%20and%20Bayesian%20Threshold%20Best%20Linear%20Unbiased%20Prediction%20for%20Predicting%20Ordinal%20Traits%20in%20Plant%20Breeding" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/2/601&amp;title=A%20Benchmarking%20Between%20Deep%20Learning%2C%20Support%20Vector%20Machine%20and%20Bayesian%20Threshold%20Best%20Linear%20Unbiased%20Prediction%20for%20Predicting%20Ordinal%20Traits%20in%20Plant%20Breeding" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/2/601&amp;t=A%20Benchmarking%20Between%20Deep%20Learning%2C%20Support%20Vector%20Machine%20and%20Bayesian%20Threshold%20Best%20Linear%20Unbiased%20Prediction%20for%20Predicting%20Ordinal%20Traits%20in%20Plant%20Breeding" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/2/601&amp;title=A%20Benchmarking%20Between%20Deep%20Learning%2C%20Support%20Vector%20Machine%20and%20Bayesian%20Threshold%20Best%20Linear%20Unbiased%20Prediction%20for%20Predicting%20Ordinal%20Traits%20in%20Plant%20Breeding" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/2/601&amp;title=A%20Benchmarking%20Between%20Deep%20Learning%2C%20Support%20Vector%20Machine%20and%20Bayesian%20Threshold%20Best%20Linear%20Unbiased%20Prediction%20for%20Predicting%20Ordinal%20Traits%20in%20Plant%20Breeding" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492153/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492153" id="node492153--3" data-pisa="ggg;9/2/601" data-pisa-master="ggg;g3.118.200998" data-apath="/ggg/9/2/601.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Benchmarking Between Deep Learning, Support Vector Machine and Bayesian Threshold Best Linear Unbiased Prediction for Predicting Ordinal Traits in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Javier Martín-Vallejo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="4">Carlos M. Hernández-Suárez</span>, <span class="highwire-citation-author" data-delta="5">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="6">Philomin Juliana</span> and <span class="highwire-citation-author" data-delta="7">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">601-618;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200998 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492153--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Javier Martín-Vallejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Javier%2BMart%C3%ADn-Vallejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADn-Vallejo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJavier%2BMart%25C3%25ADn-Vallejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos M. Hernández-Suárez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Facultad de Ciencias, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BM.%2BHern%C3%A1ndez-Su%C3%A1rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hern%C3%A1ndez-Su%C3%A1rez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BM.%2BHern%25C3%25A1ndez-Su%25C3%25A1rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Guadalajara, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=601&amp;rft.epage=618&amp;rft.atitle=A%20Benchmarking%20Between%20Deep%20Learning%2C%20Support%20Vector%20Machine%20and%20Bayesian%20Threshold%20Best%20Linear%20Unbiased%20Prediction%20for%20Predicting%20Ordinal%20Traits%20in%20Plant%20Breeding&amp;rft.volume=9&amp;rft.issue=2&amp;rft.date=2019-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Mart%C3%ADn-Vallejo%2C+Javier&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Gianola%2C+Daniel&amp;rft.au=Hern%C3%A1ndez-Su%C3%A1rez%2C+Carlos+M.&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=Juliana%2C+Philomin&amp;rft.au=Singh%2C+Ravi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492153/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492153/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492153/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492153/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492153/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492153/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492153/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492153/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492153/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492153/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492153/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492153/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/2/601" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/2/601.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/2/601.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/2/601.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
