<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/7/2171" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/7/2171.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/7/2171.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/7/2171.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/7/2171/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/7/2171.atom" />
<meta name="HW.pisa" content="ggg;7/7/2171" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia" />
<meta name="DC.Identifier" content="10.1534/g3.117.042671" />
<meta name="DC.Date" content="2017-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Leno-Colorado et al." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Ascertaining the molecular and physiological basis of domestication and breeding is an active area of research. Due to the current wide distribution of its wild ancestor, the wild boar, the pig ( Sus scrofa ) is an excellent model to study these processes, which occurred independently in East Asia and Europe ca. 9000 yr ago. Analyzing genome variability patterns in terms of metabolic pathways is attractive since it considers the impact of interrelated functions of genes, in contrast to genome-wide scans that treat genes or genome windows in isolation. To that end, we studied 40 wild boars and 123 domestic pig genomes from Asia and Europe when metabolic pathway was the unit of analysis. We computed statistical significance for differentiation (Fst) and linkage disequilibrium (nSL) statistics at the pathway level. In terms of Fst, we found 21 and 12 pathways significantly differentiated at a q -value &lt; 0.05 in Asia and Europe, respectively; five were shared across continents. In Asia, we found six significant pathways related to behavior, which involved essential neurotransmitters like dopamine and serotonin. Several significant pathways were interrelated and shared a variable percentage of genes. There were 12 genes present in &gt;10 significant pathways (in terms of Fst), comprising genes involved in the transduction of a large number of signals, like phospholipase PCLB1, which is expressed in the brain, or ITPR3, which has an important role in taste transduction. In terms of nSL, significant pathways were mainly related to reproductive performance (ovarian steroidogenesis), a similarly important target trait during domestication and modern animal breeding. Different levels of recombination cannot explain these results, since we found no correlation between Fst and recombination rate. However, we did find an increased ratio of deleterious mutations in domestic vs. wild populations, suggesting a relaxed functional constraint associated with the domestication and breeding processes. Purifying selection was, nevertheless, stronger in significantly differentiated pathways than in random pathways, mainly in Europe. We conclude that pathway analysis facilitates the biological interpretation of genome-wide studies. Notably, in the case of pig, behavior played an important role, among other physiological and developmental processes." />
<meta name="DC.Contributor" content="Jordi Leno-Colorado" />
<meta name="DC.Contributor" content="Nick J. Hudson" />
<meta name="DC.Contributor" content="Antonio Reverter" />
<meta name="DC.Contributor" content="Miguel Pérez-Enciso" />
<meta name="article:published_time" content="2017-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Ascertaining the molecular and physiological basis of domestication and breeding is an active area of research. Due to the current wide distribution of its wild ancestor, the wild boar, the pig (&lt;i&gt;Sus scrofa&lt;/i&gt;) is an excellent model to study these processes, which occurred independently in East Asia and Europe ca. 9000 yr ago. Analyzing genome variability patterns in terms of metabolic pathways is attractive since it considers the impact of interrelated functions of genes, in contrast to genome-wide scans that treat genes or genome windows in isolation. To that end, we studied 40 wild boars and 123 domestic pig genomes from Asia and Europe when metabolic pathway was the unit of analysis. We computed statistical significance for differentiation (Fst) and linkage disequilibrium (nSL) statistics at the pathway level. In terms of Fst, we found 21 and 12 pathways significantly differentiated at a &lt;i&gt;q&lt;/i&gt;-value &amp;lt; 0.05 in Asia and Europe, respectively; five were shared across continents. In Asia, we found six significant pathways related to behavior, which involved essential neurotransmitters like dopamine and serotonin. Several significant pathways were interrelated and shared a variable percentage of genes. There were 12 genes present in &amp;gt;10 significant pathways (in terms of Fst), comprising genes involved in the transduction of a large number of signals, like phospholipase PCLB1, which is expressed in the brain, or ITPR3, which has an important role in taste transduction. In terms of nSL, significant pathways were mainly related to reproductive performance (ovarian steroidogenesis), a similarly important target trait during domestication and modern animal breeding. Different levels of recombination cannot explain these results, since we found no correlation between Fst and recombination rate. However, we did find an increased ratio of deleterious mutations in domestic &lt;i&gt;vs.&lt;/i&gt; wild populations, suggesting a relaxed functional constraint associated with the domestication and breeding processes. Purifying selection was, nevertheless, stronger in significantly differentiated pathways than in random pathways, mainly in Europe. We conclude that pathway analysis facilitates the biological interpretation of genome-wide studies. Notably, in the case of pig, behavior played an important role, among other physiological and developmental processes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/07/01" />
<meta name="citation_mjid" content="ggg;7/7/2171" />
<meta name="citation_id" content="7/7/2171" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/7/2171" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/7/2171.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/7/2171.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/7/2171.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.042671" />
<meta name="citation_pmid" content="28500056" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2171" />
<meta name="citation_lastpage" content="2184" />
<meta name="citation_author" content="Jordi Leno-Colorado" />
<meta name="citation_author_institution" content="Centre for Research in Agricultural Genomics (CRAG), Consejo Superior de Investigaciones Científicas-Instituto Recerca i Tecnologia Agroalimentaries-Universidad Autonoma de Barcelona-Universidad de Barcelona (CSIC-IRTA-UAB-UB) Consortium, 08193 Bellaterra, Spain" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6049-256X" />
<meta name="citation_author" content="Nick J. Hudson" />
<meta name="citation_author_institution" content="School of Agriculture and Food Science, University of Queensland, Brisbane 4343, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3549-9396" />
<meta name="citation_author" content="Antonio Reverter" />
<meta name="citation_author_institution" content="Commonwealth Scientific and Industrial Research Organisation (CSIRO) Agriculture and Food, St. Lucia, Brisbane, Queensland 4067, Australia" />
<meta name="citation_author" content="Miguel Pérez-Enciso" />
<meta name="citation_author_institution" content="Centre for Research in Agricultural Genomics (CRAG), Consejo Superior de Investigaciones Científicas-Instituto Recerca i Tecnologia Agroalimentaries-Universidad Autonoma de Barcelona-Universidad de Barcelona (CSIC-IRTA-UAB-UB) Consortium, 08193 Bellaterra, Spain" />
<meta name="citation_author_institution" content="Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona 08010, Spain" />
<meta name="citation_author_email" content="miguel.perez@uab.es" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3524-995X" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. Ai;citation_author=X. Fang;citation_author=B. Yang;citation_author=Z. Huang;citation_author=H. Chen;citation_title=Adaptation and possible ancient interspecies introgression in pigs identified by whole-genome sequencing.;citation_pages=217-225;citation_volume=47;citation_year=2015;citation_pmid=25621459;citation_doi=10.1038/ng.3199" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. J. Amaral;citation_author=L. Ferretti;citation_author=H.-J. Megens;citation_author=R. P. Crooijmans;citation_author=H. Nie;citation_title=Genome-wide footprints of pig domestication and selection revealed through massive parallel sequencing of pooled DNA.;citation_pages=e14782-e14782;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21483733;citation_doi=10.1371/journal.pone.0014782" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=E. Bianco;citation_author=H. W. Soto;citation_author=L. Vargas;citation_author=M. Pérez-Enciso;citation_title=The chimerical genome of Isla del Coco feral pigs (Costa Rica), an isolated population since 1793 but with remarkable levels of diversity.;citation_pages=2364-2378;citation_volume=24;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. Bianco;citation_author=B. Nevado;citation_author=S. E. Ramos-Onsins;citation_author=M. Pérez-Enciso;citation_title=A deep catalog of autosomal single nucleotide variation in the pig.;citation_pages=e0118867;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Bosse;citation_author=H.-J. Megens;citation_author=O. Madsen;citation_author=L. A. Frantz;citation_author=Y. Paudel;citation_title=Untangling the hybrid nature of modern pig genomes: a mosaic derived from biogeographically distinct and highly divergent Sus scrofa populations.;citation_pages=4089-4102;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12807" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Improving the accuracy and efficiency of identity-by-descent detection in population data.;citation_volume=194;citation_year=2013;citation_pmid=23535385;citation_doi=10.1534/genetics.113.150029" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=B. Buitenhuis;citation_author=L. L. G. Janss;citation_author=N. A. Poulsen;citation_author=L. B. Larsen;citation_author=M. K. Larsen;citation_title=Genome-wide association and biological pathway analysis for milk-fat composition in Danish Holstein and Danish Jersey cattle.;citation_pages=1112;citation_volume=15;citation_year=2014;citation_pmid=25511820;citation_doi=10.1186/1471-2164-15-1112" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=W. Burgos-Paz;citation_author=C. A. Souza;citation_author=H. J. Megens;citation_author=Y. Ramayo-Caldas;citation_author=M. Melo;citation_title=Porcine colonization of the Americas: a 60k SNP story.;citation_pages=321-330;citation_volume=110;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=C. C. Chang;citation_author=C. C. Chow;citation_author=L. C. Tellier;citation_author=S. Vattikuti;citation_author=S. M. Purcell;citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets.;citation_pages=7;citation_volume=4;citation_year=2015;citation_pmid=25722852;citation_doi=10.1186/s13742-015-0047-8" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. Chen;citation_author=D. Pan;citation_author=H. Ren;citation_author=J. Fu;citation_author=J. Li;citation_title=Identification of selective sweeps reveals divergent selection between Chinese Holstein and Simmental cattle populations.;citation_pages=76;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=F. Cruz;citation_author=C. Vilà;citation_author=M. T. Webster;citation_title=The legacy of domestication: accumulation of deleterious mutations in the dog genome.;citation_volume=25;citation_year=2008;citation_pmid=18689870;citation_doi=10.1093/molbev/msn177" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=E. C. da Silva;citation_author=N. de Jager;citation_author=W. Burgos-Paz;citation_author=A. Reverter;citation_author=M. Perez-Enciso;citation_author=E. Roura;citation_title=Characterization of the porcine nutrient and taste receptor gene repertoire in domestic and wild populations across the globe;citation_pages=1057;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=G. M. Dall’Olio;citation_author=H. Laayouni;citation_author=P. Luisi;citation_author=M. Sikora;citation_author=L. Montanucci;citation_title=Distribution of events of positive selection and population differentiation in a metabolic pathway: the case of asparagine N-glycosylation.;citation_pages=98-98;citation_volume=12;citation_year=2012;citation_pmid=22731960;citation_doi=10.1186/1471-2148-12-98" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. T. Daub;citation_author=T. Hofer;citation_author=E. Cutivet;citation_author=I. Dupanloup;citation_author=L. Quintana-Murci;citation_title=Evidence for polygenic adaptation to pathogens in the human genome.;citation_volume=30;citation_year=2013;citation_pmid=23625889;citation_doi=10.1093/molbev/mst080" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Diamond;citation_title=Evolution, consequences and future of plant and animal domestication.;citation_pages=700-707;citation_volume=418;citation_year=2002;citation_issue=6898;citation_pmid=12167878;citation_doi=10.1038/nature01019" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=B. J. Dickson;citation_title=Molecular mechanisms of axon guidance.;citation_pages=1959-1964;citation_volume=298;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. Esteve-Codina;citation_author=Y. Paudel;citation_author=L. Ferretti;citation_author=E. Raineri;citation_author=H.-J. Megens;citation_title=Dissecting structural and nucleotide genome-wide variation in inbred Iberian pigs.;citation_pages=148;citation_volume=14;citation_year=2013;citation_pmid=23497037;citation_doi=10.1186/1471-2164-14-148" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Ferrer-Admetlla;citation_author=M. Liang;citation_author=T. Korneliussen;citation_author=R. Nielsen;citation_title=On detecting incomplete soft or hard selective sweeps using haplotype structure.;citation_volume=31;citation_year=2014;citation_pmid=24554778;citation_doi=10.1093/molbev/msu077" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Ferretti;citation_author=E. Raineri;citation_author=S. Ramos-Onsins;citation_title=Neutrality tests for sequences with missing data.;citation_volume=191;citation_year=2012;citation_pmid=22661328;citation_doi=10.1534/genetics.112.139949" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Anim. Biosci.;citation_author=L. Frantz;citation_author=E. Meijaard;citation_author=J. Gongora;citation_author=J. Haile;citation_author=M. A. M. Groenen;citation_title=The evolution of suidae.;citation_pages=61-85;citation_volume=4;citation_year=2016;citation_pmid=26526544;citation_doi=10.1146/annurev-animal-021815-111155" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=L. A. F. Frantz;citation_author=J. G. Schraiber;citation_author=O. Madsen;citation_author=H.-J. Megens;citation_author=A. Cagan;citation_title=Evidence of long-term gene flow and selection during domestication from analyses of Eurasian wild and domestic pig genomes.;citation_pages=1141-1148;citation_volume=47;citation_year=2015;citation_pmid=26323058;citation_doi=10.1038/ng.3394" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Fujii;citation_author=K. Otsu;citation_author=F. Zorzato;citation_author=S. de Leon;citation_author=V. K. Khanna;citation_title=Identification of a mutation in porcine ryanodine receptor associated with malignant hyperthermia;citation_pages=448-451;citation_volume=253;citation_year=1991;citation_issue=5018;citation_pmid=1862346;citation_doi=10.1126/science.1862346" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. Y. Geer;citation_author=A. Marchler-Bauer;citation_author=R. C. Geer;citation_author=L. Han;citation_author=J. He;citation_title=The NCBI BioSystems database.;citation_volume=38;citation_year=2010;citation_pmid=19854944;citation_doi=10.1093/nar/gkp858" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. A. M. Groenen;citation_title=A decade of pig genome sequencing: a window on pig domestication and evolution.;citation_pages=23;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. A. M. Groenen;citation_author=A. L. Archibald;citation_author=H. Uenishi;citation_author=C. K. Tuggle;citation_author=Y. Takeuchi;citation_title=Analyses of pig genomes provide insight into porcine demography and evolution.;citation_pages=393-398;citation_volume=491;citation_year=2012;citation_issue=7424;citation_pmid=23151582;citation_doi=10.1038/nature11622" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N.-T. Ha;citation_author=J. J. Gross;citation_author=A. van Dorland;citation_author=J. Tetens;citation_author=G. Thaller;citation_title=Gene-based mapping and pathway analysis of metabolic traits in dairy cows.;citation_pages=e0122325;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cell &amp; developmental biology;citation_journal_abbrev=Semin Cell Dev Biol;citation_author=K. D. Irvine;citation_title=Integration of intercellular signaling through the Hippo pathway.;citation_pages=812-817;citation_volume=23;citation_year=2012;citation_issue=7;citation_pmid=22554983;citation_doi=10.1016/j.semcdb.2012.04.006" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Kanehisa;citation_author=M. Araki;citation_author=S. Goto;citation_author=M. Hattori;citation_author=M. Hirakawa;citation_title=KEGG for linking genomes to life and the environment.;citation_pages=480-484;citation_volume=36;citation_year=2008;citation_doi=10.1093/nar/gkm882" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Keinan;citation_author=D. Reich;citation_title=Human population differentiation is strongly correlated with local recombination rate.;citation_pages=e1000886-e1000886;citation_volume=6;citation_year=2010;citation_issue=3;citation_pmid=20361044;citation_doi=10.1371/journal.pgen.1000886" />
<meta name="citation_reference" content="citation_journal_title=Genes Brain Behav.;citation_author=H.-Y. Koh;citation_author=D. Kim;citation_author=J. Lee;citation_author=S. Lee;citation_author=H.-S. Shin;citation_title=Deficits in social behavior and sensorimotor gating in mice lacking phospholipase C beta 1.;citation_pages=120-128;citation_volume=7;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=A. V. Kukekova;citation_author=S. V. Temnykh;citation_author=J. L. Johnson;citation_author=L. N. Trut;citation_author=G. M. Acland;citation_title=Genetics of behavior in the silver fox.;citation_pages=164-177;citation_volume=23;citation_year=2012;citation_issue=1-2;citation_pmid=22108806;citation_doi=10.1007/s00335-011-9373-z" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Larson;citation_author=K. Dobney;citation_author=U. Albarella;citation_author=M. Fang;citation_author=E. Matisoo-Smith;citation_title=Worldwide Phylogeography of Wild Boar Reveals Multiple Centers of Pig Domestication;citation_pages=1618-1621;citation_volume=307;citation_year=2005;citation_issue=5715;citation_pmid=15761152;citation_doi=10.1126/science.1106927" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. Matthews;citation_author=G. Gopinath;citation_author=M. Gillespie;citation_author=M. Caudy;citation_author=D. Croft;citation_title=Reactome knowledgebase of human biological pathways and processes.;citation_pages=619-622;citation_volume=37;citation_year=2009;citation_doi=10.1093/nar/gkn863" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. McLaren;citation_author=B. Pritchard;citation_author=D. Rios;citation_author=Y. Chen;citation_author=P. Flicek;citation_title=Deriving the consequences of genomic variants with the Ensembl API and SNP effect predictor.;citation_volume=26;citation_year=2010;citation_pmid=20562413;citation_doi=10.1093/bioinformatics/btq330" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Molnár;citation_author=T. Nagy;citation_author=V. Stéger;citation_author=G. Tóth;citation_author=F. Marincs;citation_title=Genome sequencing and analysis of Mangalica, a fatty local pig of Hungary.;citation_pages=761;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=M. a. Mooney;citation_author=J. T. Nigg;citation_author=S. K. McWeeney;citation_author=B. Wilmot;citation_title=Functional and genomic context in pathway analysis of GWAS data.;citation_pages=390-400;citation_volume=30;citation_year=2014;citation_pmid=25154796;citation_doi=10.1016/j.tig.2014.07.004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=V. Munoz-Fuentes;citation_author=M. Marcet-Ortega;citation_author=G. Alkorta-Aranburu;citation_author=C. L. Forsberg;citation_author=J. M. Morrell;citation_title=Strong artificial selection in domestic mammals did not result in an increased recombination rate.;citation_volume=32;citation_year=2015;citation_pmid=25414125;citation_doi=10.1093/molbev/msu322" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. Nevado;citation_author=S. E. Ramos-Onsins;citation_author=M. Perez-Enciso;citation_title=Resequencing studies of nonmodel organisms using closely related reference genomes: optimal experimental designs and bioinformatics approaches for population genomics.;citation_pages=1764-1779;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12693" />
<meta name="citation_reference" content="citation_journal_title=Dev. Med. Child Neurol;citation_author=A. Ngoh;citation_author=A. McTague;citation_author=I. M. Wentzensen;citation_author=E. Meyer;citation_author=C. Applegate;citation_title=Severe infantile epileptic encephalopathy due to mutations in PLCB1: expansion of the genotypic and phenotypic disease spectrum.;citation_pages=1124-1128;citation_volume=56;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=L. Ollivier;citation_title=Genetic differences in recombination frequency in the pig (Sus scrofa).;citation_pages=1048-1051;citation_volume=38;citation_year=1995;citation_issue=5;citation_pmid=8536996" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. Pavlidis;citation_author=J. D. Jensen;citation_author=W. Stephan;citation_author=A. Stamatakis;citation_title=A critical assessment of storytelling: gene ontology categories and the importance of validating genomic scans.;citation_volume=29;citation_year=2012;citation_pmid=22617950;citation_doi=10.1093/molbev/mss136" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=M. Pérez-Enciso;citation_author=G. de los Campos;citation_author=N. Hudson;citation_author=J. Kijas;citation_author=A. Reverter;citation_title=The “heritability” of domestication and its functional partitioning in the pig.;citation_pages=160-168;citation_volume=118;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=A. R. Pico;citation_author=T. Kelder;citation_author=M. P. Van Iersel;citation_author=K. Hanspers;citation_author=B. R. Conklin;citation_title=WikiPathways: pathway editing for the people.;citation_pages=1403-1407;citation_volume=6;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.;citation_author=M. Popoli;citation_author=Z. Yan;citation_author=B. S. McEwen;citation_author=G. Sanacora;citation_title=The stressed synapse: the impact of stress and glucocorticoids on glutamate transmission.;citation_pages=22;citation_volume=13;citation_year=2011;citation_pmid=22127301;citation_doi=10.1038/nrn3138" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinformatics.;citation_author=A. R. Quinlan;citation_title=BEDTools: the swiss-army tool for genome feature analysis.;citation_pages=11121-11.12.34;citation_volume=47;citation_year=2014;citation_doi=10.1002/0471250953.bi1112s47" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=O. Ramírez;citation_author=W. Burgos-Paz;citation_author=E. Casas;citation_author=M. Ballester;citation_author=E. Bianco;citation_title=Genome data from a sixteenth century pig illuminate modern breed relationships.;citation_pages=175-184;citation_volume=114;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=S. E. Ramos-Onsins;citation_author=W. Burgos-Paz;citation_author=A. Manunza;citation_author=M. Amills;citation_title=Mining the pig genome to investigate the domestication process.;citation_pages=471-484;citation_volume=113;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Renaut;citation_author=L. H. Rieseberg;citation_title=The accumulation of deleterious mutations as a consequence of domestication and improvement in sunflowers and other compositae crops.;citation_volume=32;citation_year=2015;citation_pmid=25939650;citation_doi=10.1093/molbev/msv106" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Reverter;citation_author=E. K. F. Chan;citation_title=Combining partial correlation and an information theory approach to the reversed engineering of gene co-expression networks.;citation_volume=24;citation_year=2008;citation_pmid=18784117;citation_doi=10.1093/bioinformatics/btn482" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=J. Ross-Ibarra;citation_title=The evolution of recombination under domestication: a test of two hypotheses.;citation_pages=105-112;citation_volume=163;citation_year=2004;citation_pmid=14767840;citation_doi=10.1086/380606" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C.-J. Rubin;citation_author=H.-J. Megens;citation_author=A. Martinez Barrio;citation_author=K. Maqbool;citation_author=S. Sayyab;citation_title=Strong signatures of selection in the domestic pig genome.;citation_volume=109;citation_year=2012;citation_pmid=23151514;citation_doi=10.1073/pnas.1217149109" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. C. Sabeti;citation_author=S. F. Schaffner;citation_author=B. Fry;citation_author=J. Lohmueller;citation_author=P. Varilly;citation_title=Positive natural selection in the human lineage.;citation_pages=1614-1620;citation_volume=312;citation_year=2006;citation_issue=5780;citation_pmid=16778047;citation_doi=10.1126/science.1124309;citation_georef=2006066590" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Shannon;citation_author=A. Markiel;citation_author=O. Ozier;citation_author=N. S. Baliga;citation_author=J. T. Wang;citation_title=Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks;citation_pages=2498-2504;citation_volume=13;citation_year=2003;citation_issue=11;citation_pmid=14597658;citation_doi=10.1101/gr.1239303" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N.-L. Sim;citation_author=P. Kumar;citation_author=J. Hu;citation_author=S. Henikoff;citation_author=G. Schneider;citation_title=SIFT web server: predicting effects of amino acid substitutions on proteins.;citation_volume=40;citation_year=2012;citation_pmid=22689647;citation_doi=10.1093/nar/gks539" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=EVOLUTIONARY RELATIONSHIP OF DNA SEQUENCES IN FINITE POPULATIONS;citation_pages=437-460;citation_volume=105;citation_year=1983;citation_issue=2;citation_pmid=6628982" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=F. Tortereau;citation_author=B. Servin;citation_author=L. Frantz;citation_author=H.-J. Megens;citation_author=D. Milan;citation_title=A high density recombination map of the pig reveals a correlation between sex-specific recombination and GC content.;citation_pages=586-586;citation_volume=13;citation_year=2012;citation_pmid=23152986;citation_doi=10.1186/1471-2164-13-586" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-statistics for the analysis of population structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_doi=10.2307/2408641" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. Wang;citation_author=M. Li;citation_author=H. Hakonarson;citation_title=Analysing biological pathways in genome-wide association studies.;citation_pages=843-854;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21085203;citation_doi=10.1038/nrg2884" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. A. Zeder;citation_title=Core questions in domestication research.;citation_volume=112;citation_year=2015;citation_pmid=25713127;citation_doi=10.1073/pnas.1501711112" />
<meta name="twitter:title" content="A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="twitter:description" content="Ascertaining the molecular and physiological basis of domestication and breeding is an active area of research. Due to the current wide distribution of its wild ancestor, the wild boar, the pig ( Sus scrofa ) is an excellent model to study these processes, which occurred independently in East Asia and Europe ca. 9000 yr ago. Analyzing genome variability patterns in terms of metabolic pathways is attractive since it considers the impact of interrelated functions of genes, in contrast to genome-wide scans that treat genes or genome windows in isolation. To that end, we studied 40 wild boars and 123 domestic pig genomes from Asia and Europe when metabolic pathway was the unit of analysis. We computed statistical significance for differentiation (Fst) and linkage disequilibrium (nSL) statistics at the pathway level. In terms of Fst, we found 21 and 12 pathways significantly differentiated at a q -value &lt; 0.05 in Asia and Europe, respectively; five were shared across continents. In Asia, we found six significant pathways related to behavior, which involved essential neurotransmitters like dopamine and serotonin. Several significant pathways were interrelated and shared a variable percentage of genes. There were 12 genes present in &gt;10 significant pathways (in terms of Fst), comprising genes involved in the transduction of a large number of signals, like phospholipase PCLB1, which is expressed in the brain, or ITPR3, which has an important role in taste transduction. In terms of nSL, significant pathways were mainly related to reproductive performance (ovarian steroidogenesis), a similarly important target trait during domestication and modern animal breeding. Different levels of recombination cannot explain these results, since we found no correlation between Fst and recombination rate. However, we did find an increased ratio of deleterious mutations in domestic vs. wild populations, suggesting a relaxed functional constraint associated with the domestication and breeding processes. Purifying selection was, nevertheless, stronger in significantly differentiated pathways than in random pathways, mainly in Europe. We conclude that pathway analysis facilitates the biological interpretation of genome-wide studies. Notably, in the case of pig, behavior played an important role, among other physiological and developmental processes." />
<meta name="og-title" property="og:title" content="A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/7/2171" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Ascertaining the molecular and physiological basis of domestication and breeding is an active area of research. Due to the current wide distribution of its wild ancestor, the wild boar, the pig ( Sus scrofa ) is an excellent model to study these processes, which occurred independently in East Asia and Europe ca. 9000 yr ago. Analyzing genome variability patterns in terms of metabolic pathways is attractive since it considers the impact of interrelated functions of genes, in contrast to genome-wide scans that treat genes or genome windows in isolation. To that end, we studied 40 wild boars and 123 domestic pig genomes from Asia and Europe when metabolic pathway was the unit of analysis. We computed statistical significance for differentiation (Fst) and linkage disequilibrium (nSL) statistics at the pathway level. In terms of Fst, we found 21 and 12 pathways significantly differentiated at a q -value &lt; 0.05 in Asia and Europe, respectively; five were shared across continents. In Asia, we found six significant pathways related to behavior, which involved essential neurotransmitters like dopamine and serotonin. Several significant pathways were interrelated and shared a variable percentage of genes. There were 12 genes present in &gt;10 significant pathways (in terms of Fst), comprising genes involved in the transduction of a large number of signals, like phospholipase PCLB1, which is expressed in the brain, or ITPR3, which has an important role in taste transduction. In terms of nSL, significant pathways were mainly related to reproductive performance (ovarian steroidogenesis), a similarly important target trait during domestication and modern animal breeding. Different levels of recombination cannot explain these results, since we found no correlation between Fst and recombination rate. However, we did find an increased ratio of deleterious mutations in domestic vs. wild populations, suggesting a relaxed functional constraint associated with the domestication and breeding processes. Purifying selection was, nevertheless, stronger in significantly differentiated pathways than in random pathways, mainly in Europe. We conclude that pathway analysis facilitates the biological interpretation of genome-wide studies. Notably, in the case of pig, behavior played an important role, among other physiological and developmental processes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
    <title>A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__HLkFhTtFYKr9UM35ZkeLEJf0AA-RJsk2jDIDlTr7P6E__db-Yi0lrH1NHADEOMB-20Dvr8Qs6F9pwOIMQF9BHr1w__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__EjT-6YMPvgLl8LSJ78FRw-tAFtIqvvC0dKPDt3xx2Ec__naWIXWhSBwaYF2DqFTeJJxk8gTTEkdfCwGZ2o8_HgIw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__WoUUhFHu9IZRJChlxfU_hFiUTBhKim997BgtM6rQv_0__WtsG9gDKkyL0Zcl619hpmtR4jbV5vkG5UnoAmrjf3jM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"bVL69r8j5HKuR-umcjwyDDRamysR1SAWJaNcq4Q9I3s","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485295","apath":"\/ggg\/7\/7\/2171.atom","pisa":"ggg;7\/7\/2171","ac":{"\/ggg\/7\/7\/2171.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/7\/2171.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2171"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2171"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/7\/2171":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hOp2eOjZeww3JDxlrHR\\\\\\\/nztoxFfmhVgnwXsMPrOpTR1s8UnSLIPP1zZsXGKMO4Oc4Rq\\\\\\\/EMluSXqlpvsjkkx5GIoYWbD0OkUt5HKl18HocLBxlSUjJke\\\\\\\/Jwg6d\\\\\\\/9xCO\\\\\\\/ll6k6CIx9YffKVDqeQv86ho2AbRSaSTTqXERhSoEy\\\\\\\/7FCCJxVS+Dd55LLWuL1508GHJDB4eMbnEVsRjFTvqvljtC78lMGEUvoOa9gt4cZ3RN7QpfzWpRMGAold5HTIgz8vHUrAk3jxyuuIV9E6wzNtgghLy3J6WklP3FteyEgCKovOa3lDGzkiYbbvzE0KVhL+8y6bEDTNGBUBfl6D4O6H+LX5g93sS9UcESgEVVeWqwM9LD19F29LTwM+GpCjUmW5dY\\\\\\\/ST7EBvVJv6asZqHJurlx5l7b\\\\\\\/sK+w8RQ1tww\\\\\\\/2z9KXS+uX3fMwjuB\\\\\\\/Og1cHj08fPGWOmV4vMUMwzWFZqnA74FTeE5VKotHC9LSrRUoxCZ6l99vtSfcCH4MpJuqwFR3V9\\\\\\\/JfNSyytu2upsL9TGWox6ifpjyhhFsHE+t8dWMuDk53d5LjXRJeXd4OFaetJVGLChJ18WqARi1lhNIxU7VSJVSTSN9kPL6xkqjvsJB4G4UOC\\\\\\\/wcawNmCnnmDzL5foQNalMBEApXZ88ILuPDwkGqhQ5yveXY6wvATOp5C7mrJ3pRK4wAHjC6B8qyRlf2NmZ7AOXvGGxneQ+UVTCawZ8zGUd13vPTCtOuhPXEwTh7NA4BWym1TXqg6wQiocOvevB+rLaPFEjIgdcIbrDG4jzdUMMO8syjU8wEeupMKiGUZe0GJSEKl\\\\\\\/wYh3rhY7LTV2wrxI\\\\\\\/EApIChClZGQbw2YTzZIH+DFL04xaY3Yvw\\\\\\\/CE2FrZtdXgCpM8qkKSEL0qa3aDLjJFEuG+SEtB4NqdbdtBkODVouOFw9rDLhbIG9vC0NlGe03Sktm9rKZRI0PJwCVSy89QPN9aSH8Qq0meceSgCXMPqdBfRhxKyYP5E6RMuc8H6ajni1z7xLwowi2iYaKDN5pxlPTlkrfPmlIcxyFheEOgiALrLEveta+\\\\\\\/l1\\\\\\\/0MhEPxX7thbbI4koiFBe\\\\\\\/yZdHio2kVVbL9Xf2m538ZMWKdBnU3+0+4l8Vosoa9J24Fhy7aHHt4PiRppgxJizLqYUcTM1qGfaDPq74Rv0T+NuIVZCdjgNoMYybb1Yy8vHv5xW1OVt7y77ezpqgLwRuKU\\\\\\\/Ttc\\\\\\\/SVUnjlQ2GYQrZzUu7RUkVaR3fMqAIRikIVorM3SCSbl3mwR1bMD2ZYjBRUd\\\u0022,\\\u0022iv\\\u0022:\\\u0022CC2yAMXXFIllTxaCypTXcw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022155ff83a023ba206d28c7d289763ca034a2dcda29ce512cea6f55a4edc669805\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TEA8aaFvAppW0nn7pRW0vJoB64w+\\\\\\\/EtZr4HsNNNtMWvWmYo4xMH95BdAt1bmoVruMwVCl6VlO1HuXQ2TfyBYu7EhiJ2H6SI0OSXvSA5lG9Gh5A3FUkKkwJK9DUB06lfX0vVk4jP93uLF4oyzTnQALXiQsrskczZwC9H58aRHrnAM1iJHXCo5dzqGJjwNYs9seh12giA8AmO0gpeTzcSYyChrkKRt7HQcWUL46GwAZjMOT1mzk1HrE5MXEwYGtX325b6lBYur72tGJwqldlgN1VduV\\\\\\\/ZQl9FfqquJar1y3bpgO00KuWiRfN5l9FsUeM0BFcz1nkbJcFQDdkcJyXUA2i+EmXuw31oThS9zfr5eVda8qle9e6poWnUygdclAkeZvNULLkAy8ot2oXStTN3JuHjVr05p+ZclnTeS96+jIG+GencWsD+TV2FsRyRPX0ckkojNPUYegWNwU1R4GHOWJYKGxdJy2oI4o9JaAgOF0fGjMqEA1Fe17GiLv+PljrTR4iCetig3kWImIx8x4XTwlLcRf1t8rDMpNKivjgatfsovU58siGeMVVOe8ZjQ+BrbnfFSFFPmdoMjNHOfSilocpKWHiR8EdPxaQ4iXQ5CfsyQ4EkkelSfparKylRymgXsk8tWPZNRDfbCk\\\\\\\/Zt+nqF4r\\\\\\\/CSDhdvPxAYyPn7skrDIh5dTlCfqZJwHGFlJgXsj9N\\\\\\\/djnaKMIrCBhFCMe3uakjzAV\\\\\\\/vfh9EsXcZWssZwbs1oecF59SuMorcWWmTnQBT7qyJuwNkS3hBbDdasheGJdHTPT7sd84l+Ty5IX\\\\\\\/UGWS8PTCK2vCwjhnu9ThU2Qaytv7i2+Y5CHeuOL0mjZLM\\\\\\\/qXAFZLueBwCDlpwDuKp1SNLrmEeqjZnlM+zGGNo7TSUC5UFrwk0on8V9YFoNFJ6CTHxonifaT0jHlWYOPQEYQfgJKlT4qCxcnPSv0vJ4qSwQWerIgrryZg0E1Eb+Itxy826Sr6gHruTt4s\\\\\\\/aUj3Dkvrn94U3qq89RBsl+vIeLuinkXLw+ZIEaGrgQQ5fHRP9Ks6SS+fkhe5lQX0fiOO5RDAeFuNO538xpd1Fa2KDmow41iKZF6Z6Dlw09a3W\\\\\\\/hc34LDv5uJpwNwfjCIVe5pln2nwmyOmpwYF+pqXhqP9MmojcVKsyuMabG8QiSDJXaaEbv6bgRTVqpw31r3LFvhYJ2Y8x5VjXr84pdYXpY705MrbaVsAsxeHYDulYDlb2mzGX1w+ZnMfJyp2QKxnkIPUEl6DmYwuyJ8MZHC1wvoPKxpwvFM5YsRC1oFAYoAPNTmo4DXyKgKBFP8g1BkDeRQssajqFKhzyX\\\\\\\/3AAWtropOvikAXqir5bBQdNXAgTv+ZW+9wTj65\\\\\\\/qtEakdpNzmUJPLjczuRVyQBNWNfPavTXduqSIUBwbnwy+IUUXh1aGXYj0D49W6izBoKs4jYHTs\\\\\\\/av\\\\\\\/c3TJWenDIoU2e8cqhlmkj9udAPQ+Ctg1oAIZA+jDBA853FBOrteiiQw1R5eVRG\\\\\\\/uo3z64DQQVdyFtzPkjDFv9H0kQCWdhZR69nZx0u9ME+LzlIPELijW6uhcTNVzItiC7WKL1zsWHud0L865lsm0hrYtdNi71GIwpYvil4Xl\\\\\\\/WxiYMGeOToRLLzP4ILrij0iNHW\\\\\\\/rokWCaBqoCkjA9YzWXorf\\\\\\\/yBYxKmnS3s\\\\\\\/S882VIrO9L+4KFe++hxK4U8q2NiLWJjPd78XlEZwAqBEHrJJ27\\\\\\\/SQgdm8QHOIFg1hYwsmbEqxvKy3Ml9ESs31tk8x+dLjqR55uLXzR6poWn+EfjeaxPGgpb3l3hVcOtvpapk1pwxR3aBhaArwopBIIrP6UYvbP4\\\\\\\/5z4tk2Cbnio8o0VxA7RZ\\\\\\\/N9ReADGRwYpGEK3nERkpvOFW7HCzqm4ZHJN3eXez2NrgRlaeuF\\\\\\\/8U\\\\\\\/S9u3wiJxnWAsxbrwA7o5tWCiO88iE8b1tkB4UkSOuGAqc\\\\\\\/zaRJauZTYKUHB59u5p6gbbhUIXH2bvheORx4GHrY4\\\\\\\/8Jc+nhEwN5axUvYTH1EsJxfIL8esKFpFtnIhA2g9ZnDIEGhMsfSeza9nWwdr1EvZ9W1Fm0b\\\\\\\/PhKt0+M6\\\\\\\/nW43xAoiVKuiuu\\\\\\\/CffOTWIbbr9dkaX\\\\\\\/C2nY6e6LMfukD5HC3jMGCjW5DdbkGU8SAc3rScASwbvhZyvSZpd32bgqVCIoNtsc\\\\\\\/bm8sRE1icLkf0V3KLTwzMwvplP3qW\\\\\\\/8NVSzSmdVzuBMB0QLThiDue7oUX6dVxGyj9qMNY8xdcsXPS816t9sDyKTYhnkXpRUWjs\\\\\\\/GUhLoItRYit8sA1E33QVm\\\\\\\/gu37JPQPbIXSLYLnj5sqaV0XAy6CaxP4ZmPG6HxferECbp3s3lknG8G9AzpPjX2pqWJ6+0la0+M+WQs88m\\\\\\\/rSrjd8csxuKi6ysxkMQBsj8Bgq4QeObuYaDrSo5SfDgesjN1vGElMsQBjMCbgHKeHeaUy8VRNEjTfI33w0E9mCaUiM0c5p8o1jQb25gq3LSo0d8aURVlsfUSjJZ\\\\\\\/X0FeqtkKPpxLSTsqBbgsJT4DF5IW1ULSplc\\\\\\\/phIZhpRWVUhzuvQlaREiMCjMOeNBWTmEHIsGPf54b7nk8Pc4zBMxtZiekb5rc7iDVvARCpPDcL\\\\\\\/h+D0NwilzA1JwOrTMxdXIJLyuDvXl2eVfiOd7iFebScfTF\\\\\\\/AXt+5PxdsQnyW+sU\\\\\\\/S10JZydkDv52JDKmZQqk8JOUPOnRLv1AK3oI1pj\\\\\\\/kkJQaNQ9jYmc0yxTIIm5Th8yh2AGI7yR2+bgHz3VWkTqWg1OZX7bsg9AwXEwoL6U4TRz4GMLTU+KN7KJ31IXXO8FN5fKV\\\\\\\/Ff2B3AtCph2sHf1Lvu0E27RG8A39KzAQ2UfV+Tcq4WSVOEdM3MUqR3WIBoI7fYtsmZ4Z29RKSx5FGUQ+Y2\\\\\\\/hC2jyXsvfl8W0pzJXBzJlOElkR7sAWNpDeYdOo3tt9I1JuqwruPL\\\\\\\/hVEn74dNxUYmRfMQKpHBWTIQ78YdEPp+A0oFiwYoohhj6dGmVxghxeNNosrrda7iSMgfmES3\\\\\\\/V5iNN0755Sl\\\\\\\/pOECWiUZGrpDDOy3CDp8d5SiQ+eE8OxmBK1chOAYPgYXvdc3sWSY18Y3kaM+r6kmNLCQ3hG8vkCFuPk4RDueCMiYTDMIPO9dJLBQN4RPPPcswfb+3SH+uqoEQXFpub2Id7utnIxfhcL+i1OCcbfGU+\\\\\\\/KrGmKPh3sLFX+YALwKpUEkRO28+LNfRLJAc99C1iyd9k5Y185wHOwhvejHkrmqZCDjsnqyJ0AfGYKTGv3Vxzb9C5Kf\\\\\\\/1\\\\\\\/FD3kgkhphY9CfvJazBLOaGCXyhYHFrLsBSqdTc1cMa8V\\\\\\\/8xxXCdRum+3H2lOkR2E2IobimzIosHpY1n5QEFfGsIrDQJIqq7p94XmNL5FPwXnFu1euFqYcmk08J7ta1z6J7RjcuLkKVQMUC4vMqxA2416oocNnqkksk33VE3QnnG1WQ9anAXRo6RuiqxS2EOu\\\\\\\/XKH7ZcSAzu11OYwYSBWCtP7LgkjyJc+OQjsTsljpYer2eOcgPFNnXJarKwdyse6XfDVRQ6wBdprcblQEEIkjjzacWK6S\\\\\\\/GF03Bve966jtlZkQMe+P3Olnh6+OZ1CtLT5hf9eLXMADMdn5wN2cRek8V5IUy5hbFiRDkpo2yhaq0ewQejD7apDYEoZQM5Yoosf5foY8xG7FWsZG4FU07jYl1Khg8wYuj2Cmd57fqiyOn\\\\\\\/TuXr8b0YJ8xi6+TsRpeuncvZbJP46ooMKeyW\\\\\\\/iZqGw3otg5s+Ppm+EDlcM4xGRUJulc+v\\\\\\\/YvpUMwG5vLiVstC9\\\\\\\/K\\\\\\\/q7a1slujCujXj8+A9lUgR+TeGVbMXi\\\\\\\/KZOf5pAIin6rgJnwyiukw2LkfRwOm1Czf8Qui+ubrSaOMjqDS9SDAA2KrdVEZU0XAqpeKgPWdtV\\\\\\\/1CN2Bk\\\\\\\/KCRf8fKEHCgAiY0RHlOuU62\\\\\\\/ZKVRtbW4V577hW8cKP+ysaiS0a748zf0DWpM8mEXrKqNuLlvkOe5X7KzqFQTiC47mWMvzFga5Dt70WYAFT7bX\\\\\\\/b8plWjvqcQWJ98nkatz4dqSukvTifxZVpJ+go4RjQC22AxM1oMv1LW5G4MH+N29Mmfp0Pvd27VoFEulpEl7kg3EH0x\\\\\\\/Y4jWlSKeINLJxT1n7Nlq2FNTtJyA09XtuSo1SsjHXXoDmrBy5eVsOMA9nw5sHgr9TmoIgqmqrh48R\\\\\\\/OT6mWZ+YzA\\\\\\\/GF9he80tc3GUzWHc6mkJxt3y1EirqxUkXJ2HqVn58ho+iZA\\\\\\\/LjVHWeZ6+G\\\\\\\/uT9JFiEMnSV1UcMcZ7GrRwZub+iP8mWi0lROmNor4LlMbOcDbVV3Nd21Ms4cB1NqmGAb\\\\\\\/dtFYrOsfdSE2yd9TopTiORxHRbWigcQ7qp4A9Q68\\\\\\\/C1zyXjFSDGPmOaS335AA\\\\\\\/pQvpuJtEKIPAJry9tJhlxV++7BGbauoph7XzQuzr5C10Hc4rSXXSt2kKv\\\\\\\/TTwQ2CbymIR3e+flWUYBIfFdqzyiX+EkBF2el3F2fUN4Pw9qyi0Socr+t4fPf16vWRTLNjlayfd7GQHLX1r9xR8RT63uovsPYdqGmZoerO+oJZEM9Qvr\\\\\\\/MnFrjJ8ouEfntWqDZsPmKPMfxub+n8B\\\\\\\/XN8rrjdS5eEa5IRVLDo2xP7JfJXvWxhs\\\\\\\/s9RJxjn8ptpUuI9RXTnr9RH9uMCzqkDWjx68WyU1CpqfEyE8cwagUaDlpKICczxibEcauqaRXafK9\\\\\\\/bQuXcqv7VDaktq\\\\\\\/2qwA2RHQ6SL0kjDH2e+8sc+E3LuJySjDJMDMTF2GDmdzo9a2i9iOvy9xDCIb1uwzqJPqFsjOTU94rordi0lmPQ0aqdoU6q3z27ZNEg9GA0mdtit2rFzlweww42jOIeajOX1dNqMTl6y5zfDWuOHJRQOqLx4+XdVjfzpj16+fLHijIRC1DUvgXH8BuICUbSVTWGCXMQ7\\\\\\\/BIQNxFgU8\\\\\\\/yG9nrRrCpJHnXRELzJ10zBRUCx0dzEtPWDReGumGKNUtp6MWoNzkcfb6afMetkfyWMFUdY4N1zwq0LOdSvANLC7ED9AMck4NAhDvveWKLEHJYhW1MJ9svLCqs3+mOIsOoYfPA9+129VVqGwCc0Igk0lBQM+95J\\\\\\\/1TZr5ZVHh87H4xECJvUZpjSAnbveradGtRDl7GFkvKrsw2AqOG\\\\\\\/GCGP9ul8Rc+co9Qz+oe77ERFsWkNG3Nn\\\\\\\/YJWtJI\\\\\\\/89ZaVy5WdQS2TwWSmzgHTIPl8rHAvBAF4seq83wXVxd9aH6MrrSJ8n+VzFqWlXw9pgMrV8H0Grm8DyoQfb3ZtM8ZcHSwMUlgrEHj\\\\\\\/HuU2ZjwEXPVO88l72HTIjUlCkh4D\\\\\\\/2owrXgD7SFYKsJO76VGEe0VFBixnQH38OBuYAZVpcpmtmLSIdhv03+dBctW6+sHZ8SmpCmPzIe3ONlpjIhzkaW+XVu2r\\\\\\\/a+T6OLbqgW9UZ3NCHY09HHPxb5suD5Qeof\\\\\\\/Fz4Nv3Er8FHHrdMS0CfWeZckws0IpGVIUgrlAi+49hG+tp19YwVqYL4RqBBi1FnzIyo0eUxBIAwH0uToP+uhptaeaoyhT\\\\\\\/a6IBf9OU66bVpbMlFK1Zonfm2ee7WvykL9IKgesi101lOhp\\\\\\\/5n7CK34dAS7ChQ5\\\\\\\/Atbf2dfZOgzA5wyKzcV2XgHDOa1FcF+agzVpPqofQhMr8yArgn5En5JbEiySfYPPxy51oprvkQvCcsYgt6GGRneAbr3l2e2S3D9x3g56nxYHX3U5zKhkHz1xCRvuAt5t9\\\\\\\/F051ucTI1MWmSQd+G18Sm820fdhzWXyTQc4kyU1r6pVXSQpoSN9PX08mXwCL\\\\\\\/txvtRWRXHJK9Ec2fVWmy57QXuYWHOMVywMOrtTSvqllhKX5a1wX+Kt2A\\\\\\\/yhJE3GP5KhXRpeUFXMPqENa6KvZci9INZ0KOf1puIgZdolcdSAF08HEmoR+bD9boUyy7WM4PIePkfFmrVLy2wEu0ATIGflHZC7UcwD6IrMGg56Hu9qHBmKVYv2KCbaW2KqkGo7\\\\\\\/\\\\\\\/QQDpwVFnhsyxC0lQ3JylZOIs7rcG+frvAPEm0SuBVIPhwpkeEjTwfMrbApHmr067aKdizAAlEnw9IiqGHaFx\\\\\\\/nkdmSmQqObNGS1I9\\\\\\\/J+pc\\\\\\\/BC5le2DfuvPnNbMxzxbjlOxJqMfziwM483l+qRYLipea8ymg3V\\\\\\\/SeS9hgJ4hn72YNnDRF23ac8TgPwUzi27VVP72OYT0w04fabgGy1elSI3qfmDNppGRr5ZcU3qI\\\\\\\/zuXtxzamuhMdg06dFhMQWTBEmzhaSW\\\\\\\/qlmtlfi6xAnMJhgGFg+3XTxVdHep4H8AxX3pjlLPHnixpBSPm2w5ceBOEpJF\\\\\\\/zXDxjYiOH6VvOoCB16Tdwxdtfz9tHH5S6TthG4waSmje5PXXzI4ToMxcUYKWFGJN6RdzgNln0Mewk\\\\\\\/ZHhaLFwTKppRSZRhesREA2JqCN4gB+qJRMTGA8j8CQ7vZiP93NxUiG0CZDf7sLiyslBkKbu8Z91Bzqg9ZXBl1zQFWQyQKr+yq9yjxyfa8XeG+OJT+gp\\\\\\\/poI2z9LbtdibUkm30oc0wQ8mRcAtJaAdwSUt+nFGRpiAKxW814Br7Uwch6Klk7J34wxmB0MXSzjtaMXarc\\\\\\\/lIUrAj0aVan4lWze3TJJNTStKYCNLvV3w6KtSi1kJ5y8+J4HB+jLal52EkRmCybpNgrTe6XNd0pHEOFyK1KpMrMvasdIhFkrCAq744zblc6995SUYd3I3+UGvQO46yUqDLRb\\\\\\\/Pmg1CxstI7kQfj1c1vnXPFlPy\\\\\\\/28YMiH76qXpR7TzDhxU1Cuskbxk7Lo0jLXiDOhvXi6TREPnAJRk\\\\\\\/JA3Cv2Qs5qYXJocpW1xeUHp4TmhKEAtncPCaVRkX6WJW8SfWjtmNp7kQJX+fFpxBzMGyPC87OIRaS1e1iD+5FQJscWflInTthDdy6dxjXHEDgAT2mxDRmaOoZC1iRD6ph8IcW90\\\\\\\/EkzYJHyO11wjTUJcofhfeckPYOSfowIqr95Vk1dc5tW7H9AtaCQeGGpBspIQNzcqzftBUTpRo7Wso8vnWyFmxf\\\\\\\/fS\\\\\\\/NZGApVD41Np+EWQI\\\\\\\/\\\\\\\/j1TG7hpqxBsm6P6nGCACmdpFevKbZJUbpx2gYmI9i6fcSsKKcbyp6FHjPn9fTZeameImj+QjwOd+RBrFrduRQwAe5ZnZRnWR\\\\\\\/jWOLg7vLE9L7aduzCf8Ylvm5VuumKt01talwIf134hRb2rK4RE+GS+yjQ7VnXhMbab3SVdv\\\\\\\/vgSBwFfIHNt5pnOePB3FXWWPJXgvMjhFCIiACjexC6EhPAIpRXTXMpI0v54kSAUB+tvnO2Eohh5FVHwInJElCJV5obOodChH2MYRrYyUm\\\\\\\/UhuKxuSSrYpSL98vamMEuSaTurknq6ImeJ+Cgcqdtl0QmtrDejafQ9QfMzBMFtbyDt5HEOHlFP39VC6Vg+Kr1wXHm1AaT2F7ZKD5LaunlWqlb16mNTBXvkED0qGEyYsPkmGU5CQ64slW1v1EwU6xnk3ZDziZt0vYrbbVl0F7V9Mvev\\\\\\\/46bVLoZ5iT1VEgdnifnaSJ9bW+WTr8bhpO1iXCVDP+5g0iV1Znseu62TEmEIBYl6oF2itLAftqT861IlS\\\\\\\/hNwbMlAIbUo\\\\\\\/LngG+OhqG0SCapQeziY9D63w2lDRP1xdXfo6vSTPGBN6IITL8kpnkyGXVvIx1XU97nb7DyH5LdpmGkM3gXS1vldHtxKxW\\\\\\\/dzvrAXRYwPfttizNTFmDoP\\\\\\\/k+VWzNaBKXQV3oLF4yLpJwY6AsHGD9ynOHbhC0m9vmiMj2Kb2518sbOFh2JxfaAIj7LpHxl8s4bhLzLRQ70YlV\\\\\\\/XyZeTxq9pBTy0cZyIz78i1DcBkGvOYyHIg7FKG7YdQQgmMHha+GtGt\\\\\\\/Gsc50Uu3H\\\\\\\/57hU9YYTJHL7tAW4QPQwXcW7WcNcopE1AkscLukMOg73LrfggufaXg\\\\\\\/sDa3Sof59NfDtp4I9fWFKOFDehE2VuJcVA6cCW2fGL+XVnvu4g9lo\\\\\\\/DE+9hiV\\\\\\\/pn\\\\\\\/iFxRRuLhWDRu8YrVjqcL6NeODNFjCfsgoANrW6zzFLir+OCiNUMIyi6WkzqxI\\\\\\\/FeSQY+7JlxyHaxWXJ40Antyj8yCsVZd5oN5myDjHmq0luV2EdUamSe4ZMj58RyRtTQb2dBsC2lDcTu3720n+2QfqVGpQhLLAoEw36X18mlVYLQkIxEwTLINfOq2SYCJDTHc0s1x0Q3oM6XN2c\\\\\\\/Txn1LEzlEqqVzSNnIpL+ldCzW6P9dljcBS41Opr3\\\\\\\/ZrhHzDwjjehFDEgcdHTOClaAgtvwtp4aQoFLx5HLakZ\\\\\\\/xl1f0sGXTX0PdwSFaMF3kcsa1G6wxmQun+XdNrQuS\\\\\\\/yQd9XTSme2YwrqWwWab1ODIkVg8YnKZ01O+IpYCHNJO3Y\\\\\\\/yt+RsU3R6Jn3m9z2rnj4U3Lr1vwXaAcQelx5Gyv\\\\\\\/yeMAY0p5Phnpgt2xbW+IJ2EkEl3jjpSzNNG8U73sm4+ADBeJ24FbfFN\\\\\\\/NEUsRn6bA\\\\\\\/b9dyvzqyCzF+h4s0agfnshRR\\\\\\\/lMKASxYfiMzRN7qVTtVkj+7BsdtTRK6HPryUD6+EohAGQPNiNgAbs6jqXWub3A0cBccf9Zjt7jjUd8eyYQN6p7hDldXMmPabYRUY1WOZTnYrjtTPhKc0VIN8FUV3rZwGl1y9TbaSkhiOwk8K6rtLXuErTpy8tSfhDXXZGTuR6nR85rLCthrRcyBXeD2kcOZEHL3syDj5p5JOwjcKD9JwvYXJl17i0BNKE0z4oWCFYZruR1Ff82wsglJ6QH7A7lu2xWSUcUJhzTF5U4P5XGf4u2FqOYSZhQuwJIRr1zd0r1ywHNbXLRkB+ghlAB0cgMe4TUiHnIFEaadijQ5YpEmyh0ODdV0CVQ1SVxrLUSO\\\\\\\/OORjKy683DRfW8zjdVYdxVub5wWLN0VPfuowJJ0XdE6YEUQgJTvLuYUfv8VN1Em1aeE7gGLuTMfjTpHBDEaPKXzok5U3LRBkw4VphgGc1J0svOZRvNYpgZZ6g0LnrShO1EqA85MbNnuX7pb76j03bB3F\\\\\\\/FBvTYKlqNp9h7c\\\\\\\/ezhS\\\\\\\/kPjhxzStlfXLay\\\\\\\/LBHAb5ROVLZs+DjtHFIRD+zGtNGWb9Z\\\\\\\/bb339MwcmSpb6dZ+UIwkpsyL73BglElncnFz4fa1DqEe4YXTDQTQH98GE\\\\\\\/kgquQCi2PZ4m48AKGPZ8DO+te51V0bLYYtZv3V139J8i\\\\\\\/HI+aJ7EdT+d+AoLMDBjILAIxRilbV8SKQqKSDNVrEzsgINyAuGJCYNucEpWCfA9tLwC7B7vhWUwdZrBrQ81mBsSJ6Ypx7A2k1p0yKHTEjO52JfCgJQbDLdO9vgMn266HMxl2OIS8C0aP\\\\\\\/vGxiI8\\\\\\\/qkWbhkbOyLV\\\\\\\/a7D+6qyRXKObW7ZtmGZaJEy97IlW3qHM6uovmxA9gmYwClvvsWVo9bU8A76K8zSpglWeHyw4IPnFl4RQ5U94MBkP\\\\\\\/oAQMZ8LKcngBQweEguWdQSwvbDNZ2VZ2h\\\\\\\/zPc0+TXMj+cZLuAKIIxlrzGxEGKcM\\\\\\\/+lDIOyI+gr\\\\\\\/hRDFY8+tIqYvx+TNt7Kdg\\\\\\\/H2lCYJfQjjEBrurodaxxFWr0d5Zkw6A5VR\\\\\\\/F8u6a3fKPVv1f1S3YFz41CPxirUrFaJfPYycEh1FMd31blAFivqQPj8CTmHsMjh0SXlS5zHa4pW\\\\\\\/hOXknpK2S0qBd5nURekeeu23bTAEVkpd0z4h5jvzTZQmgFF6AiRIwObRJ1PYTXu\\\\\\\/MzzIxtydwKQq\\\\\\\/opxlF1BbgpKVSguwqe1jAjyLibc7xcj8CXTTgcrFvNrdW0bhZ1kyxG1IPZWEUZ8O9X6dVVNv0Oy9hi1cFoPkzaDNe1C4LFQCubyY+q\\\\\\\/4qJVM7TIiaxJVmaJaW2CaD+FJpl7lQD55y7D0iEnqWeXxJuitP0c3Z0YvJuqqww9QvGrxEiZCajyRNY1YreegabbBLDN89sWmO667UYV53gJF+iRnuUCwtriBVqe5Ny32BvQg5VGRvZa1dqkaicgPLkYfW6q1JSjbYARZkiWrVLL473GXxw9Z4M8uwecyrqmZOS8XmxFX8LGv\\\\\\\/cyR+rc6zTIPrt6uq9BjubkDOsD6scIa1BaEOGWPcJ5Qk8zxof9DlNe5YGHqdWZPNP5s8mbfgsDcqBHjsZw6zlaPMM8lGjhQif7BTvJGsgw+sGVAYpHgJWE6z\\\\\\\/Ieqjje8jAH2o6vpx\\\\\\\/AhKwkh92WxRB+bZH4cRFXuDAOxBUkEPakYhBShjNOVJAbQGKLNkuljEugj\\\\\\\/\\\\\\\/HGN+k7KpJYAWkT5+93XVm3hkXnSmctq64QHJK35b3ATHiasPW5LRUj+qDNFy5NDwQYjF0r2ZzzAsVwYTbNPgbi+ddydS0yKZpFpRPxwguLtHegPzxa5Db8p9Sylmlo\\\\\\\/8MfJmuL36yTz\\\\\\\/SLSFEORAKfVQdmABhCTzneFLpvgheVqyQw9TWZfBVnNUf5AxveQbT5mKPcLjgvd1TULh3w4mPfnDKvHL6GwxWVG6ZirCJs8yAOOb26dS+p8b4np5j2Ig9gNLoHLdUqlAeDPJPJu\\\\\\\/VBXBYYmCNqmBbVH2sL4WZsgYhlmDSkvzf8s9hSjmf2mlpNTtlcOFOrZ2K4aj+12CatfWouPsUc+7MIQzYVvnB7\\\\\\\/agimrSYeJmNWbOoB\\\\\\\/pmTLi2gp94RI+VLcvXsoeXt57StjN80gAqtf9hHpP30rH7bBbRjAYEWXgGoavjrnUhiBV9mZ8DKZCgNL7AiIwHUKSimviHUkSivPiBSjFpzxS9poTMLkXcjJX10lxOUnZLNU1T\\\\\\\/ddtpI7DPzC5stDnDhrJ6a4a05z+JLyzbW4cq1oDL\\\\\\\/q+ASQEC7ma9BMAcxyZPtbKcvimRUJ+ZaN6F1jqjqwY6PvPedyuiLZy5mL89w61zMKxn2DbzWEt9FZrk1qr35DKwznVywLnoUNJbla9Ej\\\\\\\/LamPpm6zX4NlgFQXJBk+rCKoSJzi2MEFhz3GNmrzvCBjVi8NdRQhqDBGrNi\\\\\\\/z2wGXhHW\\\\\\\/0cQ7Mv+\\\\\\\/bMhviAoVmux13CKcrXY3LgbJGu\\\\\\\/u3yRikkiw7HMWRqZI3N6oBVG2+ofKLMeal0QeRvrctEjSfDyjEFKRBbngCvbMEt48D2KGBqU4UrW43mGbtip50AsN6ElWWkgxBUKpihyzaO7W9yJkLUz\\\\\\\/LsEjzkKbyV48dNcMDGre5D0JJwFEzUdKx187bLEugwLFTUm723h8G2xrgx2fOe0Fx\\\\\\\/GkW1YCExHxqsglj60BDU8ciu4fAqZJ6gvxpDQ8KDmn0D30OAinIdaN7hDlDu1YlUQKrSYTMa5k4Pg0uuMXzv3M8tM8og6XOwhfICdP8NmE+ri3LaBBnhnhyciZl+POFQFd46RRB7yoXU5W\\\\\\\/US33YRsXjeMlA1SkoyWZXUlQrEkI32\\\\\\\/fBmMASQJi4+xxClOxguDm9k+AWya2hwQZmdiKVOImWpyrdx7YGYB6yyePyQ7MdBTX\\\\\\\/qiO8v4OTFnq3uXd2tJP8iXo0HFr4TX7gStsc3u1pJaQQ8qe5uvgFtcQxTdxMJKB00O8R\\\\\\\/TzJ3Y+P9jglCEbS14lZFdGTxpBDN462L+I6+GtaBe7HmbEo4hqCOHuDbeHgjMatGAj8nGCyvG1kTWQsDi1IBq55Oy3LJP0ZAw\\\\\\\/Rum1soi0IrwmRYaCFQP6TY44iNpNnSBI82Bweot\\\\\\\/SYA8F4L+G8bh\\\\\\\/5Ut3vVgwM+u3FkHJlN2cvfYBccrs4K8TVeBBYbjHTNhYZLcWXgbgqUgcpm2\\\\\\\/V6a8I8VbroTjxUXva9LTxkolRc1mPrXPgjgf+m3xYneXZyZ73lDKcvXmWMxsvs7QZg0cjtftR5Hfot87qBEY\\\\\\\/6K0C+qxKrsmjR4taxsUeJs4Tt+MiIgNVdrkuVezCCfQm\\\\\\\/nFneSEEMGf3zYe16LFXJarLC3Hxk80ahEZMJXeDKRsoTNvNTeDBJY+3Egycw6FaDbO98\\\\\\\/QjOa0czXM8cLukPsyc9mJZ3zMy\\\\\\\/dQuAc7qy8Zqh1Do8enKgMuqiStdmTeUQpDAd3ry4ZQXENLYj7NEcTavipP0rzL+KXc4V7lavkPFsCpe5j+oczgOMv+kh7PBHIsQbBnXNrSo0Z\\\\\\\/\\\\\\\/1l6gn6ecjNIt\\\\\\\/SHFxEjEZO4jmskXBH7CIWb0mQvyXQe3F340gDHCB\\\\\\\/pDgL3KpQfgUE9DATAW8CtRzz6AVIvlRz9Y0xyk5eyLP0AX0hNV4jjUpxlP2xXWynTuM3gZbn\\\\\\\/dILrXZfJlGx1KpbBdS9K2cdT62Yv6VQ8DL5cSSzRxIXuvVdxiMRu03u4rZ9xqv84cr0pfAiM5YC2t\\\\\\\/KrB\\\\\\\/c+zUL4j6qwFeFc0TfijLE0mjSg0P\\\\\\\/u4ihjIHyvov9169JGZCUKBChRVn\\\\\\\/5dUw7+ZBocU7qmU3XnnxnSh8PqSWW68XMcPgnp6WkN559O0rdAgT2\\\\\\\/DuXkgW7meN\\\\\\\/CTKZtF5asAz+qAU\\\\\\\/wkCMjKJPL7fFWhbhI6aLk30wUrzicY4sdMccK1jmLuQIXwh8r8SA+fXY82A6Rw5f8CtKJ9Li8KiaVXDk+5FQb+oGovoyKdXT1NSzgX3t77HQeJFaslUmCt9gwh+pWb24SlfviflQBVv9goVjuaDCJliHD\\\\\\\/k5OaygFfvHe1i7dR\\\\\\\/jlzBGnF\\\\\\\/WJoNIYQw9OCIWZ645UVAvCHAtmmxYZKwkPiCS7s3\\\\\\\/ajsd4NhKhtEwWsxM8vWXYZ7ehmhi\\\\\\\/UJUsd2iYSVt9VjVr6NJg2ZLJs7LKZdkm7G3PRz5KTYlEitSvKKX\\\\\\\/SRUWCjbhOhCDVxL5bZyw9LH0OADkWCTV9+5H6PxrjNv9IilkWlLxG2EanwlQaXOQGUXWle1v0pk1mgNcbcqUfEsAznJnX6ypzJrT9+4pyQjnTe3sodGKaW39VLei0r2Sdpp+kN5x6XKTQo57VnuLdDDiVlbW7ZDhJ4RH97tHhaNxq9txUwVQgTP6OJmLUpwOf7h1fmq+4scY4MgzbaN+bPuLFTZ7PMovTcUFKmPaYSefXYFhsJCPnMoYd2HRl182zJWOJEuX4eCtan2+\\\\\\\/GFXEV\\\\\\\/tKfst7QobQfF43em6peJL3whJqujV8tW6q5mVPEfWxUAJDBCOkCOgISp3M6vrAe2x\\\\\\\/S4mYPlI4pFSDuf2u\\\\\\\/0uKSb0\\\\\\\/diEdfmHT3umqk2l9wLPX9mqOOYmyIE7nHyDhtxCwtPcH71qbGBM4shVdLaDib+6MD+A8iej\\\\\\\/Xw58Up1J+0yyHCXgGnBRtS7uVav0LzFJ6Q5U0z3Av1EB8bgNEE9etIMezK8fASlAwUFA81chUcJncowxROCwlHu+ZsIhotdic5RbLIWepg1yunfWcUIfYp74dZGZrFBZiFbD5cF2oIOWjvih\\\\\\\/KCf7YCGSmkQImvUrEsbzN005NewbDpV9JdIitHVDM09C4OeP4n3RECq3v01dDQhCHpCV5MxJDE0UcJm\\\\\\\/mQnfW3fjLqSGNoXVA0\\\\\\\/jisp4w71U98+9hwSYIlVkrTbJuAmpbSSz32cjvQO8ZmhSrI8awFzI5qHHZqIZb27EhMQKzKlmw3iZ26JArJUdAsWctPd3s9CPfLdTOJtzrm3WdBJOn8716xZSPcYCA\\\\\\\/e3LU\\\\\\\/EU6nJfcjhsajSNmg044bbXmJvw2ZLH1vDLxLN0FbDfmLyQw37NbfBWYA1uqmPiXarL+UWe0UEIIXV5+C4Qcw4eOR21FfAJkkvQtgkJXHPi\\\\\\\/U+RHjY9o9aPNZyM7vq+5hp1ABfi9rcmmFplBJJgIblaSJY2KaT2WiFHWxiUZtU5+e1rT4jO9q3sV2pGeH0ktzF0T0EB2eMEpIKWm1ZZgCd4gBr7bo\\\\\\\/F5OGYXh1PiRZYnDHEa2ox2gMkzQP38k7C\\\\\\\/d\\\\\\\/fZk3VXIRByN35wtbpaWtMpI9NHmfkrp+B280Tmq0OwekEgcLSh4BaJlgFVqxV+Xgc2VSiWwmoVQyXcPOC+znDjt8ZLLYSLHu5O0h1v08AXNgJ4SDm9itXsCbJSadY5m7BDPnuPaSUbWXI3Gyo6q5EOnP0874NTeHbppp2IUMFXYeKA2FFJ1fI\\\\\\\/BLtWeTW0XqQPhRPqtFneN6JDQz8yRYYYmpkuu1qt3YuX4xtS+8qmSpjZFDOZYlKes6XfNCuK+py9dRu5XQ4kA30rtD9xvKKFQ8EDdeqSGMkwoXwNhZcxkSpx5fKDASIOr9\\\\\\\/\\\\\\\/0xXMS235uIEseHtMi3U8OvZ+38q+KkeuhQRv8A6MSeYEyXPcYBaZC99\\\\\\\/KsrPb1zh8dX00BpYgxurIaCu1nmUa+lXHirTTMr0JIRb+vf0DihCL5vADBeNXhTqizIodo6Lxr72WliCmTzc38eqjFymInsipoL+bPTtTFicWPd1sB15LL2XTPWBuCF7alnctQtXeCAx2hUVyR1Kv7UEdLeEF3kp9njJs0Dj9xprkIi9VsXjRJl6Rhhp5l6ATpNYXvNsz22FTdmjbckg46q1emWEEu7e0u\\\\\\\/TJm7r8uI+n58465ci0HWp2902r7KNTylAm5sBesoTwsXLNQWyb8sY+akLJIDh1IEiSLTGEWacEB8v22hkK7Z9Rhk+G\\\\\\\/auyrNH7rRsZ+YEEy0CZm4nGnlQL3AFYRDSN0n2SfdaRmkwSDHlKk6T3+MCSrpiZXZgGHV9SY+i\\\\\\\/WJbQqpe10yrKe75exp\\\\\\\/+Ff7kCrBydLa1hU81S5Xuyskw2iXErm5qtiSTe8SQpCCOtF04BRJyg7\\\\\\\/pLB\\\\\\\/szm+iwNYdeky9ypAmJvbRd3tQ6nCODvdVdUfwsWLb\\\\\\\/T9QpKX\\\\\\\/VbS5jiJQDLxEt5VRL56Ie1I08q+W\\\\\\\/Yx+vDA2D8cwivM8E\\\\\\\/m4sbuCVmEB9dDqTiVOyrXjXjUM6\\\\\\\/02lK8eaa7EOxWbrV\\\\\\\/30SwLoHFhZUVfCS17BUEynjk88T0scFVe8OLSqPodIhUI2PyN3O4hDLDX5oaVjqNe03pwsIeQvikwBntbXMw4kisltbw89DLDmPtbDZgr\\\\\\\/caZwg2lNciqSiTx2ckIwMTo8QnGkxaTX2DsmSyrCkCLKAF3F9abBw2awiuriGpzOMny9gLv2lBgiVE6X82teMWLwfKhh3pfpmnZRqV4XgqEA2HgKU3t8eT2PwnNnfVbOACJVGGmZAT6MHg\\\\\\\/T6NO3Bi6qMG2F2WV3IeJLXKf674HWnVDkkOlGjRiWRJwZC9vhvNINL2zXdSxq\\\\\\\/GJ5fTlDqV4mzNm6ACjVVGWLjxFlRoSOJxp\\\\\\\/XqqkmOqALEQMQuO\\\\\\\/eiMQrpyB\\\\\\\/YYR7z7dS9r+i70eOpmTqpSGHBOs7EPjd\\\\\\\/WgLisKIUR6xnemnEMI0PqQzzh4elidcAfzatCyIpY4SZY85yENynR\\\\\\\/16mbabOzo4MM4PyXkEM3Lox2RBaph79jHMdvHfO0pdIgq+\\\\\\\/j8u+a\\\\\\\/QAntYsaETxfPH5r6GsXY8zXuFIDHj673quZRF1iJQslPIiZe0spO9JdYHv2nduDQRDhL39rgguft0wsPa9vMS3wNvXP7bRHVE+L\\\\\\\/\\\\\\\/C9bto8m2n1DbfHi5l1Z45gNZkZQYn5E0eJEgl+xmIpA6tEybOqIK0BNNp3eueviEk9brA+Mk5j\\\\\\\/CDiq0Ac4bh4UPanGHAd8Z4x5lM7LTLRmKa2u4Qo8b1g4zxJr\\\\\\\/92aODuzl45rPvz+yeI6LEdompfsUNRnvUIfBPxOxLqQFf7nC6uM\\\\\\\/hQsC9wHe7Usu3qkiQ8IMt5mRWs8+e+2vL7PrN5hudOZL1NV8ixK9bESBIDk7SbwYUS9+RXyw\\\\\\\/NGujIsyiiDWHUKda9qM\\\\\\\/eJjsrelFLjINq\\\\\\\/wkTfqpac8lY0mI87sabJR0abeL\\\\\\\/iveAWsA4zgDZi1cJ2jvfbk7iDwlQB8boRmkjxlJ2ULKI4zL6FXi5PW1I2qBVuZzEEPvKM1Yx5bxFEeKqhpiyMLXumSSVrQYWfEaP75y0IL\\\\\\\/C610dr3nkMNXgOOFdRMPsBNVKhwe0aUENwLdV9H6noNKf13Tz3lo0sx1EY1SohuJsc4fwyjQT5FF4HBOnG3RoYiq9+USnYl9nRcx5gC898eDaJtn+7ldo5JQCR60iSsJuo+1YM0fuijTCvjXnP6PeaahyieKHhpq3Lz6j4WJeYXh8gYt5c3M6udkBBGmM40enubAouCZCC1O4qFhneNeeSk0MuoiWLCWb4GzfrJYyo\\\\\\\/jHncMWsg18x732iojlmgAQ3oSvzLJRY1OXZjQqSZENSoSbDoRbPai2WEYVX2C\\\\\\\/weEy1i3Gw73e4Iv3ZCZlqWI3MbCYklEGwgcFxEA9mNqS8wvlRcJbMeYMyfrW1kUMxKVX217bFcKPbNVA7\\\\\\\/1BfSV5lXKWGtldNsKoawsjArpgZW7DTM9P\\\\\\\/7nOudZE\\\\\\\/nWF3tvEHVV61pxnzF\\\\\\\/elvq30dXLd9FFcdxWt8L9ktPk1yrvxmDK+8grtJWb6m2Szo\\\\\\\/QT8eBE7bqWquOJ\\\\\\\/qW2Dc6qukKXWaF3Tc3VEDg+9plL2ROybRotj2BZM2hR34UPPAyjFoJFtsX8bhi\\\\\\\/S3XyG0I30ydUeGzgwWMXU0E5l7etRTf1kBgMGVbXRxLcZoaT\\\\\\\/VfknrbynipCf3RnAis4BTLxN8dRfellyrk0QllvihpGImT2t2KDBS2reSt3My+8nZyEfrkrU+iA5oZh1znyJHcHInGB5T9GAPpwNJmIrZz+HyNfSWnA1\\\\\\\/9tsZft5y62D3I7URIR5+FLjGf\\\\\\\/WXKUO4R89jGhlg3OxDLL\\\\\\\/TlRlr59bAsxGzSBpC4OwfCdfW13wJl+PyVo1QrIy5emu6dWkoTV8pzOyPWGXyPi10kdKAh\\\\\\\/smARCs+l3CqOSsp1iZum7NlkMUCXUeeEUF7ygoOqh8TjsUwdMwZl6JQMbJH1ynuhNEhTFhPzDeHqSXQVkufM4k1I8HEGD08CPrSMXUVT+KDspwpLR6YXAXP+eFCjLh56wYXIPG984YmtcIE3XgmxO1oHyPxQ0hRffMtBAMOcfNMF\\\\\\\/rPHuSGl6KwbL+Db96X01nCdreMOA66ZoUcjK1f0uvAJTUiYq9gs+lZLT3aZRqiVyHXcbR2kvZpFcg8v+ryd+i31xTniMTcEkOwW1EIq+UmikrHjraxZKuKRRktaQyXC2CfrupQPqRVDe3Q5UcbDLChvP+oj58PNyNT7jnGsXUcCheRMc+zigrEzSpJ9gg8YPNolHglykx\\\\\\\/FEkJz36KVAApFcvl58g1xBxTRchOlOMEk4MWNzDVchdLJaZwbiKmOlSkLNRllJVJop5ix4qyidLWtzkmIS9To\\\\\\\/wF4BxvxxOlOUKY2ELVplD+kpuYDrQvau3xO6o4KrVJuPj9BC7yReLkqk2szVdc5+0t2KipvXkJi86l97ziwZpiPVf\\\\\\\/q\\\\\\\/RxgoO5Aj7wasHDLEMWFA2GJ9DcF74volhldrVdBdN+L3qeSYN8l9hnB2h9ow8iMfhyeaYVNSUoNdUBmT4gWZF2EiyhGHyOJ7q0QZ6WjufERnft8OQRKCDk\\\\\\\/RMoLs8Cm3XTg4lwbJpw\\\\\\\/CJonMWUO3q54Tb2HoJmQed\\\\\\\/+KlJDIPbBM5cjGrETJ48iO5z2Z4oKeM5qp5lA9xoSMAk9pjXeTZP6hUhYRttCzpT0dCsSp2li7fRwA+EgjjzYLkOFIe586Osk8vvIPvyGCH7r\\\\\\\/5MwE8rwEt\\\\\\\/j8EYz0cx3hFXz2j00H+wYjFneGd8qBEKIf3lkpQs8NJER0ONXPhTvzxM42dErIsaLbOJZA1ModU4uwf+2gW+tFixLdPo7xnviBBaVkzXLDNK3W7JymGC5KMwX+ONFHJ\\\\\\\/OOyvXWNG2F76CP0Rycw\\\\\\\/vve4WZM5ZIJ1MRmisIrVDVCuz3bNKAsd20mi0fec14MgZ8HnDT5gznQIWNYVmNrh5tqRo8y5pLyCL4stbvbVH3yCP8svBI2tEScpiXXTGxl7FWfWCaxf+Gcy9CnLEmjazELldGbVUqmW4ADmNQKw1kjd44qTJL9Hc5aH6PPdxjX31sy8ON9\\\\\\\/gW9ltrXSPn9GKq8+xxiRadswZ8jP6FrzxN68VuIAWXERR+\\\\\\\/99bfemqlz+oxr36vdDIJj2jlbZNZ1CUZcIWq0wDczYvf\\\\\\\/x4MC+sXxoKq52Tgm4vXz4ENzKbixH\\\\\\\/aY+ZbBoxpPUPppg4oX69jopPdipm\\\\\\\/QUAZez93u6JG4GALviamzaybISz1+XGUPdZkywm48YEeqG0xJKk6UnIpxaC8Pbtnh\\\\\\\/ErybN+WHNvCEcc2HsS4R\\\\\\\/UUpoukRQjVgOrypeb7j8FYOLYbnrYqy+llxoePTwT38D\\\\\\\/Ws\\\\\\\/xcMRuiygWjdnWRegctvjbR+SZsDXUhhzFro+0WcyW5TaRh+2oX1juzfWH6BB\\\\\\\/K3Uz6xMWSlLwetEEBeMXF7angZrBzY6iQSpU5WO3fv63Aky3aOtOL37kwfOpW6rlG4ZCnwvFcmmUsJt6vHtRiCG0c+oBkPU+q+UEqizpVMV9nvLT9LEFJ4bBXKQYZ7KZTqDga8VZi7\\\\\\\/AgoJjGUAqEnVli+en0qcyLtqXkEnB\\\\\\\/XeNAOywxNEkQZTdJ42j+xwXmzv9S5AlLqtnN+egVAYeP2ASGHWQSq22Ix93e7I0v6i3HhjaNfI0ipO2d3FwJMTTwwj\\\\\\\/sC1yb3Vkro+og7E6YUSDGPaaqJpu3qFB+e1lDRi92wxrQuvf6Elfu3GyZqZOx1KROTrqVPVws0A8gOd2alAppiyGcQgU4jdoZq6i5YlnN3mgJZglebd4ZpOzoGb95zQRvXxNwCiRmXe1a3h\\\\\\\/7UfSK6C\\\\\\\/TbcbwHEsqXmKgPuWJpxBb4nxXk7\\\\\\\/RmOStlEjxbsh5JyijzbGzDDYnLae6nJwUgbhDYezzaFYurybwLVDN0tAnRX3xKL0oqb61Nk1oCTwa6lNM+l3ALh1DtqhBsuXjVzyd+P8yaUiQ33gBCX\\\\\\\/KaCG10lsOy1QTIvEGdSj4wElHS9SZA4BcalwoqAHtHLmorS\\\\\\\/nbHz\\\\\\\/GHcDuA\\\\\\\/zZ9USY1WU++o7yyx7Fswt9Ia26lX4UvZ9+TMXsIesGd70pR3zo0SulmbjW+vtIN5SNydi7itgRvirYshEhOrL1eJENbOz3hjgUkA0pv1614J2OLRDytLlUvfql2hAV+Z825P5L1J4G7\\\\\\\/OgENU6wXtXxMEY0Dj0FZ75l\\\\\\\/MAsRq9ZHlNdzRizNmKdL99GmS9lScFjz6xt\\\\\\\/S51lsWEzSIqgB3QUOY4kxK6i8X\\\\\\\/NU+t+l7vfJqSy3X2ny7STyhOIJ8goNW1u3uEbVUENZiva5GX0AOSwLz4Dg84ZQK8VGQAaaEOC2E9HIW2fAXkYFi4zXPU\\\\\\\/n2m+cReP7ArMod1YXZ1bUiVowWAXsbEHC4MZ5CnXGoiVGpbQg+1HYXaDRk0+\\\\\\\/NHvUnfbvL1oc3al194QFG3zYZz8R7QmJSKe5RrlpPgpDGMb5+DepLRPnoIB3gOgad+x8DRDWZqdW67ZuFzTO5SpYVAHoSTY3x60fII7tP3UL6\\\\\\\/IIweggnVNtjvITvpyOwmN3UeJc45FEY1eIf+f2q6QsTwhyvqMDzXNHcVe0bW0BwaiiRYRaYLwK3w9e\\\\\\\/QjKvkpkq4TlWpocAY6O2ky8NEkQxxIjmwJdjVEmXLIlnNDmkJKLr2BWU8ljQwzNWM5Ps5EhpepuG7Ft1B2Y1fsVS9gqG0wguw1yC718sF0n\\\\\\\/Edz2sP9YSKWvFPSW5araf9qCLayhAF1kg3hX4geXsxEYQD4Ud3UoD3bqCkXsU\\\\\\\/\\\\\\\/ECwrpRz0vpyesPb09B9XzYLplKjdZKPIhAkKd8l8IjXtmzgjkCFbFuJropw1Siam6npj\\\\\\\/TBXVyeEceMtwRUfB2DyLFWrWGdz1OA0vFKkZmNU8OYEC7AHKbfDZpUlXhwx3VzqcSvb9L3bPoX7tt5M3z5U7vOsOwS9RsrpP7G98a\\\\\\\/j1deRPlzm+UUR+fIOM\\\\\\\/Y6P6ZsTvlh+kWIS+Riuwl3vVynym4vmTcO+uc0xm\\\\\\\/d1Y05qUC1TSgDGCvK7g2iabU62fAuYXJcpaJ31dXkag\\\\\\\/dR2py0XBglPvMi8tilEdxQyt4WXRm1hIrjDmicz6LkRQlrmfboUSGVtoEDQ4vq4p1RO8JjJxXO9jitDb9DsEjGWq1DsLcYanXDDq4OMQWT6dnNopcDumL0YlYU\\\\\\\/cY5YtsNXbd+gtH2j\\\\\\\/cdafzy9u+PDsW8SFbTTqXq7rrC8ZxTMx0ZiqEfVCc\\\\\\\/49VPk+I39+D7YZPAlCjAdhXOppt8QTDWqMjwQB83sh5c0A0MP8jqSFTiwOtiWj51shuaAI1VS8kJPYAc0vDnTwtxNbiodxXRJVelmZMItIr83cJrVzaP3SFGjV+9smYjaQ2ENN47fqOqGtU3\\\\\\\/KEpmS2R0V3KTAprwFJpIMx2r8eXh9O57MpxPYVp9EuiCX9eV\\\\\\\/doVH2YRVoi7RgtQRXO1VGnnes6ay2176BHOGNffNTPpObCyI6ic\\\\\\\/Zx08DH+FI3meUoxlRHSKXdttC8BxeGynId1K1DVU3+trOYm4fJC8I6liy+olZnzHrFT2CBsN5RCbPEg3+BSMs4\\\\\\\/VgKRi7HRvvJtDOaFv9zgddx\\\\\\\/8FyZ0krW6g4Tjoc5lKhivLoHca2FPKBwVDc3lEMvAKrFWZWaTihWdcC0AhxxlzihJDIMNrdS7CUqylDpRoyjq0XLM8ApJrxj73fzgpqJNfrgcAvhMQXgzx4tfPrQYxSMlBwlGt8EIfIEVNGoqMslVkKuBnPkAGd\\\\\\\/jpI7enFT7yKrqNet7k7gPFQpW5Q7K0kHl9ybP0F4o9m25ZP0q9rh8tZiFsmMxLsLAnDWwMhVnfHGWM7z\\\\\\\/G+CdaRRaFlNj4MOq5fiB9qerQDHEoUJx3vTrHbl4sekaoJOIUAAMgZzKOLhqfQ7fWjFSIObFLqgybV5ED4p9+KoUUZebifVm5ymPhtOZYGkwKWgg89h9E7C4+3HmGfxmoX5mJ8FQvAp6+Di\\\\\\\/5PhyrjQSJl3Po8III5E0PccmFqHHvvqxKN5q3cTTdiJJl7jcUDsqkRIZ\\\\\\\/BhQaZsbHLVbzsVpIQvpadUbAbftUDhat2wgLPgAaQP4qnI\\\\\\\/Z4sK1ouNwh4O9VhcGTSzndBlu5Sru4UdVj9qOYMHL4hEDL0Gn1t24LekkiNP88Pq+scL6VV3wYEcD1nvw38GRMtWLr+DJQY0Ga7BldLMaQLzKnd55Xvn16BVSNB1kBu7oLWgbUsICxnD8OvikTzvlkqy5J+fRg4QfmvMtQf1U4paESDW7BPR0OyqcROizWpww1slp6kyHRQc6JFJbxp9jlZ5tGXvx5yF3IX7nFokPUBXZZiChe32wmf+udH6n4ojhoJN8bPDihf1xHtJ\\\\\\\/Ex6KlH70kB5AmZbzFh6QY207zm984o6rT2CuSNfxiQPuqOf4BncKjKTe7VzoWCM4BoHJ+tMtIhJenCIi66TQewFpWW5TT1rIkkE8znzp26bnh2493YbMJyAU7lz26fYaAY6LLC9gb3+t++PGBZd9ABdTk30zDs+5Y0EvS9uUSRP8k9lZa6imdnDEz1L5hZUw2OBFH5WsHMtCM0OjWRakMCY+52VD21D2mTljr9P6SFPqxkccbxbY6CwB8kXxA7qw\\\\\\\/1sxpTpt1BOTm4b62LJwj7JEXEpyyigX5dr75Ke4h7TVorwjDMzLJ1hmf3huNDW0FO9NarA50PYNMkjz+sIUgi4OqDE4XECqGPiFKDXwriDQ2fg0Apo1mpERAPYoPSPLb\\\\\\\/zfRcJ5YrXCwW9uHRPyXVIFOimf\\\\\\\/XKPkHytZ\\\\\\\/BC4\\\\\\\/8q33Tn6k11QddMNzENM9uSputruitXlHch3gM+Q3W4QXM1w48WNKH6opyqRARVs2p7mxxwBdTjIQpdo7JGrTQLNCiJuLj6SIwEDtelPnvngiQFskrmbS08\\\\\\\/JjHDKeTDCTu26AIIkYB4f6VeKqyqefOj4FI2TYV0F75b9QcXOEEv8WBj\\\\\\\/VSDZzgwwRCZVf23C7LDLmSZcbAUFw49dZJMT3R7IT0O1LouassbX0s+AldV8NfSXgQMd5D0LHhtQEhmvXvyIsNM6uhoassrNLBnryUBif\\\\\\\/9n1rFUp1iICX5dMD2dWwdnnBC\\\\\\\/0QShuYGBV\\\\\\\/VagxaqtfyMNkVtREZ2IicWZUNQC9OnT6PwxjJKs9KIfCn88ePLnQb6GKp0Hjxl1dWuFEUkgHnnVrKCV1\\\\\\\/EHVFWHoAiCGxOsMiQb6LnJT0ZqkTUnxOHupBTS7hhRew96q3iVwk+RwLsRU\\\\\\\/U7imBX8TzLYPaiOHUGjAKL\\\\\\\/XgQKvoig\\\\\\\/GwinqPHylnWghVVaeFzDEAWBBt5Yp0WFMcWqnNCMUTBSqgnhwigCBtwpRw0ehcZwy3uIC2fRdw0JNPwfDJPvyxKCkUjHqq25joPsEwtQur0Uv4Y1BFZO3pPSWjpopRlvdEkxet4KlOvYyonMD9YMzbhwX75AttnvP\\\\\\\/gD3UbDYULN56b9nMWVNROkn+PXDfgp3Dctb6ZXdfH7BB5eMuYxyDKHZxtxUVyFGkxinFRw\\\\\\\/yzYHVw2yuBCQyKD0Ruto05CgnZnVzZEQDewy18\\\\\\\/csxvi6nlsizXXhL\\\\\\\/hE+fKVafCrBGpGIUuW3kOMfHO9YFA6usTLGuFa9Kqb9ilsbBAKrJKf5mTkPLh+MKrhseFBjhwxoBD0XIgcY4Wnw9tKQSYZy\\\\\\\/KaHNX5N+PRmhHvQu3Z2MstDRDy2PLnLXh+lp2iO8gWkyFjDcX3CZQ4FD5Zfudhl\\\\\\\/3vP3AV3oIHTvA8FKI8a0xm8ycPVju0NjifMSiFFaXKIGsntMnD56S4Y9zPtV+4lRX8yYeXndkNQSfmtrbDKcH6aZd3hu9tHtqcAOzgH\\\\\\\/7ampx59fDLn2B5u+fIchwN8TwVsQwrccjsG0UPjRPbI+3AhX1zqy+6LLRJOhZ4sdhuw71bFzklURcQ6FQAmb9XO7HRRqoA\\\\\\\/L4nWGXVSktbHuO7FUmSir1hvxCImvKdDzxeNI089A6AEIoXhdHnC+k2o8GR3yV0Zgob6JH0e\\\\\\\/kYdgsOLqGzMgG62Wb5Bm3UlPMwAyorT+ffpjumsDkTTQEdJEjduhmhE8aAduz73pSC8tOk+5YwZIuVqAb+0T+hKqG7lm1wKj7G\\\\\\\/GrJLCOsbov0Sg8PJ9twcaW202q+\\\\\\\/NnMBO8BtcG9HFOQRqikPAwFHgBXVhi9ao50dm3NOBpokxSc8VvZBqY0d\\\\\\\/0l\\\\\\\/vqWEQEx9XjbOSQ57ghn9d5naxtQcKdSCIOMElwyC64wAFiAofPuDVSwj0x8zJTjUK63qGSzSpBIqhy6qJMM7Hq+B3OOjIWqIkkvZlXKAFMguFe0g3WnLdsujS6cmsPJzFCU60JY3V80dAqWGXSI5kp0AEA43mGH4EVjohN+NzxXf1YflCEY0Vhm0Xe\\\\\\\/g7DavuzUW6V3pYQsYLpkyDeuW6\\\\\\\/lqF7s+XiE34Mphnp89BaoULBQJHZ2Soew74l8gnrm9TRe5pUcqQDVA4hN1565PNNCC\\\\\\\/6+KhNFtaPC\\\\\\\/u\\\\\\\/pGd1\\\\\\\/QdIoydBAKX9ld98NfVu6JwsT1Bg70\\\\\\\/ByhdrtkDnRPlyGYp2I1GjqSnaRy6q2OufCrA\\\\\\\/TFdAhndcT46zKHG3TWLMJL\\\\\\\/XYMypMkGnoWvdK82PmDMchxoCAwyQIEqKpy3\\\\\\\/Xqa6AgIHQxafqqj76U\\\\\\\/y10ND8xYQ+iKMucPcO4V9Ke3+jsJ3u9erVzA6EHh9tWpHqzTHQpnyr1kWq+oTK43rgMpp776z5JcVLEZP7\\\\\\\/OAeh1yyzHyLDLCd+afeZF2ivWC1lfnCdHxzh3ICP7vRYNxwFfALsEczOiPagP0n4BPVPrH8\\\\\\\/kIBQgWuILyz4HgCnjaX8vMX8krw3\\\\\\\/bkhveNII+dLlrzswzWToiKmPdnixNemzSp8hqLZSVm6QpKTjvBceqLYwFHSisf7cnMqHUPM4cUKEaiXkc1EecZBRTXx8nJsaOnrno\\\\\\\/koME6AdtP3u1lBr+uhZ+JDgbwmoBHxG729V\\\\\\\/k\\\\\\\/sY++IG8I9QBsBXelMgKs3rQZzN5xojrLlnmn8EBUu1B8h\\\\\\\/iHywkvCVUaSmj6hOntBu3jCvnRdSJUP0ZhnStZnRyFarvFvbjIyCGX8l5b1ZFQLKTBQdGMvf83QnubCBOP8XBOni9s9F+Dkbi0ntTZw\\\\\\\/\\\\\\\/tmGPJoAyHeloDN3qCNBxlKpoY6g3y\\\\\\\/XzJ4\\\\\\\/xs6C9gDbIJgDhD+4ypQCT4zooocfNyA7Zy5\\\\\\\/CSdgQ+lLjxWJV487th301tpS+9jwIkhFhNMibMxPg8E4f2hdBMwwj\\\\\\\/uu8FtXJBAT6NE9KbgXA+ADKgEGmXjyeggM+lrVy5CcfYUWoTS49WGcKLB9dFrl3bRuwjAGI71hn5S9YPZnLKUPzNzO9zF3EcP5tbdIHYodMuMKDBNf7x2voAFBxUsyW1YGVjLgTcJRgKAeq7ch1xiT9reJgGWdQQoS2PGa\\\\\\\/+F5CRH9oZfzO0o\\\\\\\/YO3kh7Gb6MCLCIHwKFXxtVA\\\\\\\/IyaAFiwsRFLe5Ck\\\\\\\/i1nNbwFcsTe7PJBjX1H2b+BPEKTV5cdNLYoHSzALQid5Kq23VzDS1XXulRq8xQgFWGnGhAvOv8vEFJk9YHKx2tmEPET4IA3BUT0xG9tRDR59lnZTV+TDP0EYvULAtdBwRC9jIiEVJF2qgnBjpTn7eePsD5iLYqxcYhGKjZo9Z5FkiQZfra2lh+voAxF54YW3PuA8sykPflPIfm6PIWItEQrsjzb+RAQds9uKoEDJ6usGjMQW41HJvlOoTiZVD5gQuJkomyDsWmUEPusp1hqR9eyTi+AGxmvUK\\\\\\\/wMKWdkAXHsoCxUi+7zeO9LUL0Boo\\\\\\\/e6oeNGsuarVupBmboJB37HrtOHi+w8GrtwelNzYR8wxrE1BEkrRtScAN8BlrUFvF92tXvfrMJGAQqZKe9489sY+FokVrdAX+fEYGsd17Y6lPWsqnPBaBXTnqMl0fHT4pwDY6Lccm0xJh4cm+eTpdW06NUz0OoA6\\\\\\\/RTyfSUZfr1Jb5\\\\\\\/dDbenKWg8985lBwkknKxMpGDwX2vZU\\\\\\\/JfUuKprzlYrao5cQXUJlQgM\\\\\\\/8kHjokYZ+LlivpudtG9CELmjdkJXU\\\\\\\/d03b1yuzqLunVM7Zc7n3b9kAYi6LIUyZWb0cY09Ba58RfiiGgKkZb\\\\\\\/6C\\\\\\\/IdMfdUkHcLOtyq6Cl2eN4Rjwn2CZga4AfOpg2h7wS3qiIqJdKMYvRtmYOr6vZvZ8z5VPMZbE+Jfbva0lOGxsMkAOJ0Ey1\\\\\\\/Zfk6mwCqoTO7Jq+PvUR4n\\\\\\\/h8ZjpZUwIiX9bLyPZMHZn4BJPaTY3yTO4PiDyy68j0OdVe\\\\\\\/3TOQekK5lNHK\\\\\\\/bPwzf8ML8VxNU0pJMc0kM0B8orhwdcsdrngT4fC9MN26aoUfL4\\\\\\\/OTJJguL10u23jV8QiHlrmvTR2n59On9SPiQ7xvzrxwFEWhYA1h3AwKKovBZws01PUdnSkP657esArU3kDMXafeZpkn9Gc01vEApBhisgF+\\\\\\\/uNPzC0j8nMebDl3gweV8+yw0SIkWEMsHB0\\\\\\\/ZDyrTZneLgQx4T9NWhpNvgq00ijo2KmoWLaHOLqY2jmr6HkC7g3HbYuqiqEqnOkb+xXf6nEWisT1hKV+V45WkRZ+OQySN1OEXPdx5mDxTCPnap5S4ngKknqK75oZlPOOY6qU+fcCv0\\\\\\\/CqbqnriKOxZcQPfGwipmvGAxBZKxuWcbgkMRZC+AkKn4Ms39C3omYmX\\\\\\\/uMMoVXUjHdhKz3IR+wv4YWvuIXyvITLVivQXCaVuz8LCn4xb6trcU3mrZC6kaPRxRGW9DNcYUBMq1jKFvlLINsSsoMfLwbA6WMOLa9+LuJU3EnAiXOgF5OiHDoJ7xi8K1uiPUTuFbg88WAX\\\\\\\/fcvyOn\\\\\\\/9H6wHGm\\\\\\\/++2SFj\\\\\\\/SI1lcwmnwyUEQOVxwAAbWz4\\\\\\\/PKrayJGPAjEwAEUJ8vwmNM3fHRuGscGTLzkbNfXmScuWQR139JSSppaLtwfLODE\\\\\\\/X395rs7MNmQRWmRIt26j+rF4WyZHmzoe8Bpti0pZMswfGofrD5MBMYDTwY32hFXf7tF7TDDBAMDVcQWFic6teBNYqJeKl4lqF5Ko+9hUG1S\\\\\\\/uKg6ROf25pYLgBa\\\\\\\/SjNvsQu3uhjVGDxCErgBXlB4FjORkrSLPgHvTdZ8+Mta1hB0XQNTRwJnO\\\\\\\/jj8ycl+jn3LdNAN7eWMFqekJ1tm38GGCX7xgWVho6ydu1NiWG\\\\\\\/FQ2+J4L6pUqXrFHEYpUO9S7LfVmVbKR4eKsY8Hk0b\\\\\\\/jynguwAaf5ehnEGZi+uNw3O4IUln9s5y1aeMuWWtaJJtY7c+DUcJjouebFc+kxQmqsaWTDIc8NEEgikQpc0Qwr0b\\\\\\\/tRj6y+N+dzDpOTMUnSptOOStuGhkMMlEv2AhCs3ycgcUZrzlghy8R7a+Z1Op4bxb\\\\\\\/LRRnO5q6WUih7bn6dIWuSpWHTcuPwBhGU7\\\\\\\/uLcSbtw50NdM+OdieoN5dfFZz6UC0DNt0FTM3DR3k2SfmIZS3GhYuaShtSWT5iS8Y6Q3BYFNKuAW6UBSczQsBBMJu3qMFtXx4khdwIPcYAObXfP4UAO\\\\\\\/rKwnJPWn8nFZtxmFZh5plCZHsXUdsqav60jNdFk0KnrZMQBIR5P\\\\\\\/19ZfhNfyyY0B1e\\\\\\\/v55sPgi0mvq7EC4oZ38K07bJXUp9bMGap15HJGl\\\\\\\/EEAYbT\\\\\\\/MsHdTGmDxmbMuKl0GrTOjgqJ7vHtcBPntumufxwpVvezAEhaLoBIEoYtwFoO6o5bUVuxiRuy508pkdaMKkjKSpnzdiaNTh2XBqgRVJwMpo1SQZ\\\\\\\/sgdikNEHbCgMikg\\\\\\\/aNpTw2E9UaqY1hnRPNQLYsWNMT3TAwNGIbT8e4nVwDTT9lxGbKPDxW\\\\\\\/KZWCRUDFtTshev0noN4AFv7yGzYUZoOurTCsS5ZW0UuGhE4PbYbybVvbKUuNDh8k1e8\\\\\\\/DowB1m8IVMvzGSbK3zShGax7cRlHN1OzhsC7HIXk3m1TmGvmIb3GyPLQHuQOf2K5rs5+4qpUSWX4t+VzuDelAafHXhAR7v9lm\\\\\\\/7eE8jPzKH3y0PXjYFLYawjoVDCtrbo5UK3ayy0zkwMy9xem\\\\\\\/b1A0RW1ztLwEKIhmaaC5E2LsX+0at5qLFDqZsyIBsfdKW\\\\\\\/56vTC35m0xpy+d3xbUIVdvVsKZm5eJcRyatIi8nxAa5g6JPighMn2VicWLK5FTk+Vs3xDiN3sIwoCqxfox9Ti+i2aVaNIV7C53nQ1w4jmCGGHjFVWYFYatkiYL+AzEGmtxM9wygiJSxtYB6r5PLgEN9dgOQZcav8SXMsNHkso+Kd1aLgI5eFU1HCq2yFdepeMzHn7KnV\\\\\\\/0ys5cLr0w4biC\\\\\\\/T19wgTTDCntOIbS9vZJ4AOl4ID\\\\\\\/W844QbwLUvjr2hcwyv8ikqWsZKTt\\\\\\\/vGXhJqPTz1QXhy+5il7sd4qRz5gfncgXqxgZTHmCm0zsKuiEmw1PO1Yj5wUyrHofeNxRzaNpfKWtbCAMmyeQUswVzwWnk7mxUMvBqWjjDBSuqFlMyqfPlMLkek03YfcqnHeILNdlLH5dwex3Nqw0ALVWX\\\\\\\/HvQPZ0f7IrahELdJMtlPovCdHXT2Vz\\\\\\\/gf3RIWXff6cE9yX4WHn9XUVEJtxxPlBXVoQp\\\\\\\/YzIz0CLS5s9NK2LTFF0HVGeydo7W40McsoeBH\\\\\\\/nKb6EE4XMGqpHlnMy\\\\\\\/1DQn6WTje7iVVpxjhhE5Is7tEqXgFD6oywamyif\\\\\\\/Nd6is2ttNCkobAwdOJWjIzAccLtalwi\\\\\\\/xFH04um+7UHeCmwgXheYgZ6InLgT2h5FSP\\\\\\\/WXc\\\\\\\/hBlrouE\\\\\\\/84kDdp9EZ9vBD3fPXK4GK+DJHkKRmiCOXCcwGoVMror4Xbf+Kyezj1O2G\\\\\\\/z9o2\\\\\\\/2jwBHsxkKQceU0sz8ygXqQCZssvelsbXvkfxxqGUc7y84WryVQALHJt+G9BKnJtxgdgGU22o7+o2nb4\\\\\\\/1eYf0h3zHtZbjosQLXPQl2sxowZwqdZKCIqqItpUZiin86pK7qvQr6NIlnqnerlEJMceuvweamWTILrdjVIn4zeyxPKMqUFtRQyPO6Gw4Db2NCDsYsQCKcbv\\\\\\\/00T5szVERfvVrRFsr+U\\\\\\\/sds5fMc7fCaNrfqMH9ZpXjs8I8mIZr+kCj\\\\\\\/x3xy2JKw9nN8Y1e4oDgcGA\\\\\\\/vXdoqhWxvoHfMQ9v4D6qNEKpE0JNT7BcBxgGm45G1EeKe9A6zVLxvsufQbIoR6Lvrye\\\\\\\/QQ+RnU3e8QheqvFZBVOT7\\\\\\\/nKHknaGNbqNWSP1XbTKF83rog\\\\\\\/hnL4k4nvGBGCHNAdTMg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zQEN43ZDwEry\\\\\\\/TBZdXAM+w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022de074409f0c54d86a6c6a4e879dc61e276c6ae3e221acd1370973b36f5c17da2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IsNDdkzRMIIk8oXl8lAr2vfj8Km3KqajGZa\\\\\\\/J\\\\\\\/4\\\\\\\/7U248LmhYXkj6YbVKBB798oF6RlV2AdmFMyCRKyKYJhn9ns2YpEyhcsDtEV40KL0Xy11RAqmauNtAQEn8AItIe4JiwVnIZ20Z\\\\\\\/QEmw5d95lJ2iUaTS5o57s3\\\\\\\/zwGHRx8juBEqoifqi6W2Rd4S0DCun9OCo+5w3ljvrCwvOj6Goiv48Rs54Ym412kNrkJDGrGZlyvHNUthNjS5HPR4j99UH5FZD0+PaG\\\\\\\/rNxRuByRmNEWjx0bM2zpdQUvkd5kSqlxVbQU+tEa9bQsrLU9VkkadRGffPtYN6XOVyCW5\\\\\\\/p4T63pjmylcA0m15hoCqMiOw\\\\\\\/91ibZpRfwWmsvfen6qaDHtFMwukYVbytU1jjnVrH5u1aHn4L+tKsdzNJSoi70SE9qNP\\\\\\\/ojGdiPMM0I4Rrsc6EGrD+AdF2Gq1VSmMccL7yhOX1lWEv2OFAJblLfiCDXlcIVydTBUCNHJ7JGvWkVVdnkjbAPecEFfUXsIKd3s9tBR2QolsSmp\\\\\\\/EyU7hrbetiFmHLny7YgQVvN1YPuym52yN04qMhzATC8snsyONrq9XS+Hi4TiFO\\\\\\\/SMogdLXLQrqHuO4FOq705uWUXw8xCjXnkiqOK5NypxDJ8CwzvnAUv\\\\\\\/d1rzjkkcQV3MCu2bkIGjnIf+qNIsGZ9\\\\\\\/dQwa1FhOwfDanp3hvPZXgqmUnagqePggkF9S9rrGCJ3FwGvUxz4E18i52oEP2m4XxUWddP4G5MmBJo05TdSeavxeKtpD6OvMHqSSdueOOofkz0TVE0Sn0esHFVehLTn2aiJ8YFx3xGFCCykqTkhSJJfipschdcZO7U4g8F52SPv0r80ZBwRpE0wLht+YOzXgnH1kQ9XUpiT+yt1v24tS8QhVE2GlS5S7PbeOKqIF9o\\\\\\\/\\\\\\\/XX7mS2TjKuo66M6R9+\\\\\\\/6CH6u4Bv\\\\\\\/2pO32S1c\\\\\\\/CWoiU9mpjNX0RCH18JPl89U8EN3YcafD10yacoC5UQS4IfgL1W7U+IEhEUFW5HAb25g\\\\\\\/aNvLjy7xYYPK1x9LjZpXCENQhPWJ4oPjvcl7W1EGNJ1O0XLqTAXVd5+HdGtES4FXqjuq+80pRf7OE7sGxP1yy8BmeUt+AzreDGfnlhHpkIQOPckMh\\\\\\\/zvtQ+TEpSeeSAh98PlUA7NeWmRHv2TFV\\\\\\\/+iWgS5lrQcP3g8vKqXIMVR9jkx2BlhcHXK3\\\\\\\/WeIfo4BBXcy2LaRPi5YBWuPcwhKOW\\\\\\\/pugpMwSLmg0SOE3iyEvIbdM1\\\\\\\/L+wl75MWnEdTHik6zozS9yryCA87JwcmmEzFCHFZqx3zjOeHlbqVb0nVwQdkoU+BmZuo74lzVGEvziAIQMqXNwNREWr69GLVFrMin0OC1le4\\\\\\\/3xY2y+lnKzyw+vd5EGUC2\\\\\\\/KiRiYpO0ad8dGVOlVqD52a\\\\\\\/0BTtDxXhNPtl+zKl9LgUByQLQz4bkgo81Q0sgZKAoiLkqlR8G5LNqdq3l64mYHu+FBtfWNfuOh3AA9aRwNezyfOyj3KDWy4mNnXiYhfZcS774ZNKIR1qk\\\\\\\/YqvLbg6pXvbBHcrTdrVRx+fdzX8oYDbDK+utXzrenZgkKsac2KGF7adz84bVEEej5cEc8syW+VdWfGb0rxPw\\\\\\\/C3Ws3VRLNLXSu8BsMdFInKc\\\\\\\/fEd9G1VokccAAU4tIbPsJPJrZV8QHWrdof3MWUKl9RbjhJmlVVJx1tKAdhH6imR1GOVGe41KvuQILoI2IWYwW+1w2ZBGeMyetytgVQW6aADouU5e7Z5WYB1qCmYgheU3pZSNwWncihzP+LJqR\\\\\\\/\\\\\\\/8PsDBVEEp\\\\\\\/r5XZivntdoQvwFmowC3CeMN9MvmA5YeUeGuJAsFVuWRR1VujIZyGNqHCc\\\\\\\/WCigLeZmiT2np0edO+bOiLO\\\\\\\/NOLFbk5RI0Q\\\\\\\/+MJonTpHJA+2fz3kE1RXpTjNeMbAYqrJg5RSrCYcZ+T2l7SvloVG0Y7\\\\\\\/b8p31CJStx8X7KKGxWi5X7e+14ZANdFqc9zCzJ+YCoG9kHx\\\\\\\/rKg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FZa4bv59UFYoDorPrsQ63A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225d26913f9d736e8628bd9da97d5a10b2ba85923fbfa1bba678447f69f5180894\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485295 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2171" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DMzZ3ZsOdigwNKePFqOxlwoBWjO3T6bsfI4nzyL44g8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2171" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ni7kKJFPtVRR2Hfey2OhoFp6ZSJZ2GRF0AVIXKI0Ppg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2161" title="Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2185" title="Assessment of Gene Flow Between Gossypium hirsutum and G. herbaceum: Evidence of Unreduced Gametes in the Diploid Progenitor" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485295" id="node485295" data-pisa="ggg;7/7/2171" data-pisa-master="ggg;g3.117.042671" data-apath="/ggg/7/7/2171.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6049-256X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jordi Leno-Colorado</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3549-9396" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nick J. Hudson</span>, <span class="highwire-citation-author" data-delta="2">Antonio Reverter</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Miguel Pérez-Enciso</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2171-2184;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042671 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.042671" data-atom-uri="/ggg/7/7/2171.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485295" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jordi Leno-Colorado </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), Consejo Superior de Investigaciones Científicas-Instituto Recerca i Tecnologia Agroalimentaries-Universidad Autonoma de Barcelona-Universidad de Barcelona (CSIC-IRTA-UAB-UB) Consortium, 08193 Bellaterra, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordi%2BLeno-Colorado%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leno-Colorado%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJordi%2BLeno-Colorado%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6049-256X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jordi Leno-Colorado</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nick J. Hudson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Agriculture and Food Science, University of Queensland, Brisbane 4343, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nick%2BJ.%2BHudson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hudson%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANick%2BJ.%2BHudson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3549-9396" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nick J. Hudson</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Antonio Reverter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Commonwealth Scientific and Industrial Research Organisation (CSIRO) Agriculture and Food, St. Lucia, Brisbane, Queensland 4067, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BReverter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reverter%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonio%2BReverter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Miguel Pérez-Enciso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), Consejo Superior de Investigaciones Científicas-Instituto Recerca i Tecnologia Agroalimentaries-Universidad Autonoma de Barcelona-Universidad de Barcelona (CSIC-IRTA-UAB-UB) Consortium, 08193 Bellaterra, Spain</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona 08010, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BP%C3%A9rez-Enciso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Enciso%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiguel%2BP%25C3%25A9rez-Enciso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Miguel Pérez-Enciso</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:miguel.perez@uab.es" class="" data-icon-position="" data-hide-link-title="0">miguel.perez@uab.es</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/7/2171" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485295" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2171.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485295" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2171.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485295" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2171.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485295" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/7/2171.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485295" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Ascertaining the molecular and physiological basis of domestication and breeding is an active area of research. Due to the current wide distribution of its wild ancestor, the wild boar, the pig (<em>Sus scrofa</em>) is an excellent model to study these processes, which occurred independently in East Asia and Europe ca. 9000 yr ago. Analyzing genome variability patterns in terms of metabolic pathways is attractive since it considers the impact of interrelated functions of genes, in contrast to genome-wide scans that treat genes or genome windows in isolation. To that end, we studied 40 wild boars and 123 domestic pig genomes from Asia and Europe when metabolic pathway was the unit of analysis. We computed statistical significance for differentiation (Fst) and linkage disequilibrium (nSL) statistics at the pathway level. In terms of Fst, we found 21 and 12 pathways significantly differentiated at a <em>q</em>-value &lt; 0.05 in Asia and Europe, respectively; five were shared across continents. In Asia, we found six significant pathways related to behavior, which involved essential neurotransmitters like dopamine and serotonin. Several significant pathways were interrelated and shared a variable percentage of genes. There were 12 genes present in &gt;10 significant pathways (in terms of Fst), comprising genes involved in the transduction of a large number of signals, like phospholipase PCLB1, which is expressed in the brain, or ITPR3, which has an important role in taste transduction. In terms of nSL, significant pathways were mainly related to reproductive performance (ovarian steroidogenesis), a similarly important target trait during domestication and modern animal breeding. Different levels of recombination cannot explain these results, since we found no correlation between Fst and recombination rate. However, we did find an increased ratio of deleterious mutations in domestic <em>vs.</em> wild populations, suggesting a relaxed functional constraint associated with the domestication and breeding processes. Purifying selection was, nevertheless, stronger in significantly differentiated pathways than in random pathways, mainly in Europe. We conclude that pathway analysis facilitates the biological interpretation of genome-wide studies. Notably, in the case of pig, behavior played an important role, among other physiological and developmental processes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/behavior" class="hw-term hw-article-keyword hw-article-keyword-behavior" rel="nofollow">behavior</a></li><li class="kwd"><a href="/keyword/domestication" class="hw-term hw-article-keyword hw-article-keyword-domestication" rel="nofollow">domestication</a></li><li class="kwd"><a href="/keyword/pathway-analysis" class="hw-term hw-article-keyword hw-article-keyword-pathway-analysis" rel="nofollow">pathway analysis</a></li><li class="kwd"><a href="/keyword/pig" class="hw-term hw-article-keyword hw-article-keyword-pig" rel="nofollow">pig</a></li><li class="kwd"><a href="/keyword/genome-sequence-data" class="hw-term hw-article-keyword hw-article-keyword-genome-sequence-data" rel="nofollow">genome sequence data</a></li></ul><p id="p-3">Plant and animal domestication were cornerstone events in mankind’s recent history (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Diamond 2002</a>). By ensuring a continuous and reliable supply of food, domestication allowed a steady increase in human population size that eventually resulted in the first urban societies, thereby facilitating the technological development that characterizes the human species. Although domestication has received considerable interest for many years from multiple disciplines, modern large-scale genomic technologies are shedding new light on a process where many unknowns still remain. This endeavor is largely facilitated in species, such as the pig, where a modern equivalent of the wild ancestor is available for comparison.</p><p id="p-4">There is some ambiguity in defining what is domestication (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zeder 2015</a>), given that many concurrent processes have occurred in the transition between wild specimens and the individuals bred by humans, and that domestication likely involved gradual discontinuities in gene flow between domestic and wild populations instead of a sudden stop (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Frantz <em>et al.</em> 2015</a>). Nevertheless, in animals, there are some shared characteristics among the major domestic species since they have been selected to meet human preferences; domestic animals have modified behavior, and distinctive growth and reproductive features compared to their wild ancestors. The genetic bases of these traits are clearly polygenic, as is evident for the numerous QTL that have been identified (<a href="http://www.animalgenome.org/cgi-bin/QTLdb/SS/index">www.animalgenome.org/cgi-bin/QTLdb/SS/index</a>). This complicates the discovery of genes underlying their phenotypic variability because small effect sizes are difficult to detect.</p><p id="p-5">Traditionally, studies looking for selective signals have analyzed individual SNPs or carried out a genomic scan in windows of contiguous SNPs of arbitrary size (<em>e.g.</em>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amaral <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Burgos-Paz <em>et al.</em> 2013</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rubin <em>et al.</em> 2012</a>). Since genes do not act in isolation but in concerted action with other genes, we argue, as other studies have done (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Daub <em>et al.</em> 2013</a>), that analyzing genomic variability patterns from a metabolic pathway point of view should facilitate the biological interpretation of the results. Compared to a genome window analysis, this approach could improve power when individual gene signals are weak. By adding up these individually weak signals in a pathway framework, a global significant statistic can eventually be obtained. Note that a pathway analysis differs from analyzing <em>a posteriori</em> a list of statistically significant genes using, by instance, gene ontology tools, since here we predefine a list of genes and we then study the collective behavior of genes of the whole list. The criticism by <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pavlidis <em>et al.</em> (2012)</a> is then less applicable when a prior hypothesis exists. For gene set approaches, see review in, <em>e.g.</em>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mooney <em>et al.</em> (2014)</a>.</p><p id="p-6">Previous studies do show that taking into account how genes interact along metabolic pathways is enlightening. For instance, using a pathway approach, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Daub <em>et al.</em> (2013)</a> discovered that adaptation signals in humans, measured by increased differentiation, are enriched for pathogen resistance pathways. However, none of the genes were statistically outliers so this observation would probably have been overlooked had genes been analyzed individually. In cattle, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ha <em>et al.</em> (2015)</a> identified several pathways associated with a number of key metabolites in dairy cows and <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Buitenhuis <em>et al.</em> (2014)</a> revealed pathways associated with milk production. The main difference between those studies is the criterion that they used to merge individual signals, as numerous variants have been proposed [<em>e.g.</em>, <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wang <em>et al.</em> (2010</a>)]. Here, we used Fisher’s statistics to combine several independent Fst values for each SNP into a gene <em>P</em>-value and, subsequently, those gene <em>P</em>-values were combined into a pathway <em>P</em>-value. We argue that combining signals from multiple SNPs should be less prone to false discoveries than taking the single most outlier signal for each gene.</p><p id="p-7">Despite numerous studies in pig domestication [<em>e.g.</em>, for a review see <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ramos-Onsins <em>et al.</em> (2014)</a>] so far, to our knowledge, pathway analysis has not been applied to improve our understanding of the domestication or breeding processes in this species. The fact that most phenotypic characteristics have a polygenic basis makes pathway analysis an attractive approach, provided that the pathway as a whole better explains the genetic basis of the trait than do individual genes. Here, we have used sequence data from 163 domestic and wild pigs to study how potentially selective processes associated with domestication and ensuing breeding have modeled the pig genome, when viewed from a pathway point of view. By using sequence instead of chips, we further avoid the issue of ascertainment bias, and provide a comprehensive, unbiased portray of nucleotide diversity. Note that a comparison between domestic and wild specimens necessarily confounds domestication and modern breeding signals, and truly disentangling genetic changes due to domestication from those caused by ensuing breeding requires ancient DNA studies at population scale, which is currently unrealistic despite some recent advances (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ramírez <em>et al.</em> 2014</a>). Since we were predominantly interested in the shared signal left by domestication and breeding across breeds, we tried to minimize the specific breed effects. To this end, we combined genomes from several domestic breeds, sampling evenly the number of specimens per breed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Pig samples</h3><p id="p-8">We analyzed a sample of 163 wild and domestic pig (<em>Sus scrofa</em>) genomes (Supplemental Material, Figure S1 and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>). The 163 pigs were classified into Asian domestic pigs (ASDM, <em>n</em> = 60), Asian wild boars (ASWB, <em>n</em> = 20), European domestics (EUDM, <em>n</em> = 63), and European wild boar (EUWB, <em>n</em> = 20). ASDM represented 10 Chinese breeds (Meishan, Bamaxiang, Hetao, Laiwu, Luchuan, Minzhu, Sichuan, Tibetan, Wuzhishan, and Yunnan), which were chosen to represent the different geographic locations in China, six samples from each breed. ASWB comprised 10 boars from South China and 10 from North Asia (North China, Korea, and East Russia). EUDM were all from major breeds (Duroc, Landrace, Large White, and Pietrain), plus the local breeds Iberian, Mangalica, and the American miniature pig Yucatan, of Iberian descent (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Burgos-Paz <em>et al.</em> 2013</a>); 10 genomes per breed were chosen except for Mangalica and Iberian, where only five and eight were available, respectively. The 20 EUWBs were from Spain, France, The Netherlands, Switzerland, Italy, Greece, Tunisia, and the Near East. The domestic breeds used in this study are selected for a diversity of traits. For European breeds, meat content and growth are important targets, whereas Chinese breeds tend to be more prolific and fatter than their European counterparts.</p><p id="p-9">Most of the sample sequences were available in public databases (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Groenen <em>et al.</em> 2012</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Rubin <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Esteve-Codina <em>et al.</em> 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Molnár <em>et al.</em> 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ai <em>et al.</em> 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bianco <em>et al.</em> 2015a</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Pérez-Enciso <em>et al.</em> 2016</a>) and were downloaded from the short read archive (SRA, <a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>). Two additional samples (Iberian pig IBGU1805 and a British Large White LWGB0348) were specifically sequenced for this study and have been submitted to the SRA (<a href="https://www.ncbi.nlm.nih.gov/sra">https://www.ncbi.nlm.nih.gov/sra</a>; accessions SRX2787051 and SRX2788443 within study PRJNA255085). The VCF files containing both raw and imputed SNPs are available at <a href="https://bioinformatics.cragenomica.es/numgenomics">https://bioinformatics.cragenomica.es/numgenomics</a> (under the heading “data”).</p></div><div id="sec-3" class="subsection"><h3>NGS bioinformatics</h3><p id="p-10">We downloaded and mapped raw reads against the reference assembly (Sscrofa10.2, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Groenen <em>et al.</em> 2012</a>) using the BWA mem option (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li and Durbin 2009</a>). We removed PCR duplicates using SAMtools rmdup v0.1.19 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2009</a>) and realigned around indels with the GATK IndelRealigner tool (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">McKenna <em>et al.</em> 2010</a>). We called genotypes with SAMtools mpileup and bcftools call v1.3.0 (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2009</a>) for each individual separately. To call a SNP, we set the minimum and maximum depths between 5 × and twice the average sample’s depth plus one, the minimum SNP quality was 10 in each sample, with the further requirements of minimum mapping quality and minimum base quality of 20. We also called the homozygous blocks, which are the parts of the sequence that are equal to the reference. Since SAMtools does not filter by default these homozygous blocks by depth, we filtered them fitting the same depth and quality requirements as for the SNP calling procedure using the “samtools depth” utility, BEDtools (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Quinlan 2014</a>), and custom scripts. In this way, both SNPs and homozygous blocks were filtered by the same criteria.</p><p id="p-11">We then merged individual gVCF files into a multi-individual VCF file, with all the SNPs from the 163 samples. For this purpose, we followed a two-step approach, closely resembling that in <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Pérez-Enciso <em>et al.</em> (2016)</a>. In summary, we first generated a fasta file from the gVCF file for each individual and generated a multi-individual VCF file using the individual fasta file to identify whether a position is equal to the reference, polymorphic, or missing. An alternative approach would have been to call SNPs using all samples simultaneously, but this strategy has been shown to have less power (and similar type I errors) than the one followed here, because joint SNP calling is less sensitive to rare variants than individual calling (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nevado <em>et al.</em> 2014</a>). Furthermore, Asian and European samples are highly divergent and multi-sample algorithms are optimized for single population analyses.</p><p id="p-12">Once the multiple sample file was obtained, we discarded the singletons, SNPs in sex chromosomes, and the SNPs with &gt;30% of missing data of the samples in each group (ASDM, ASWB, EUDM, and EUWB). If a given SNP was not called in at least ≥30% of samples in all groups, it was discarded from further analyses. Finally, we imputed the missing genotypes and inferred phases with Beagle 4.0 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Browning and Browning 2013</a>). We annotated SNPs with Ensembl’s Variant Effect Predictor (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">McLaren <em>et al.</em> 2010</a>). This tool also classifies nonsynonymous variants as tolerated or deleterious based on their SIFT scores (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sim <em>et al.</em> 2012</a>), which predicts whether an amino acid substitution affects protein function. For each gene, we computed the ratio of deleterious <em>vs.</em> tolerated SNPs. These statistics were computed for each population (ASDM, ASWB, EUDM, and EUWB) separately. R (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">R Development Core Team 2014</a>) was used to obtain a “heatmap” to represent Euclidean distances between samples’ genotypes.</p></div><div id="sec-4" class="subsection"><h3>Differentiation and disequilibrium metrics</h3><p id="p-13">Selection increases differentiation at positively selected loci between a control population and a population where the loci are beneficial, also causing an increase in linkage disequilibrium (LD) around selected haplotypes. These two well-known phenomena (<em>e.g.</em>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sabeti <em>et al.</em> 2006</a>) can be captured by either Fst (allele frequency differentiation) or haplotype-based tests, such as nSL (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ferrer-Admetlla <em>et al.</em> 2014</a>). Since the pig was independently domesticated in Asia and in Europe (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Larson <em>et al.</em> 2005</a>), we computed Fst (Weir–Cockerham estimate, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Weir and Cockerham 1984</a>) between wild and domestic populations in each continent, Asia and Europe, separately using VCFtools (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Danecek <em>et al.</em> 2011</a>). The nSL metrics are designed to detect the positive selection signal due to an increase in haplotype homozygosity; for this purpose, nSL measures the length of a segment of haplotype homozygosity in terms of number of mutations. We calculated the statistics with the program <em>nSL</em> (<a href="http://cteg.berkeley.edu/software.html">http://cteg.berkeley.edu/software.html</a>) within the four different populations of interest (ASDM, ASWB, EUDM, and EUWB); the statistics were normalized according to derived allele frequency in 10 bins of size 0.10. The ancestral allele is needed for the nSL statistic and was inferred from a consensus outgroup allele, as explained in <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bianco <em>et al.</em> (2015b)</a>. The consensus was obtained from several species: <em>S. barbatus</em>, <em>S. cebifrons</em>, <em>S. verrucosus</em>, <em>S. celebensis</em>, and African warthog (<em>Phacochoerus africanus</em>). The divergence between the different <em>Sus</em> species is ∼4.2 MYA, whereas that of <em>Sus</em> with warthog is ca. 10 MYA (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Frantz <em>et al.</em> 2016</a>). We removed those SNPs for which the ancestral allele could not be reliably identified or with less than two alleles. For each gene, we assessed the average recombination rate based in the linkage map by <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tortereau <em>et al.</em> (2012)</a>. This map was based on four different F2 crosses between European and Chinese breeds; total autosomal length was ∼20 M. We obtained a smoothed recombination rate using the <em>loess</em> R package, to minimize the effect of gaps in the recombination map.</p></div><div id="sec-5" class="subsection"><h3>Pathway analysis</h3><p id="p-14">We downloaded the complete dataset with pig pathways and genes from NCBI Biosystems v.20160202 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Geer <em>et al.</em> 2010</a>). The downloaded file contained 1789 pathways and 7157 genes. The median number of genes per pathway was 47 and ranged from 1 to 1519. The NCBI biosystems database contains records from different source databases, such as KEGG (<a href="http://www.genome.jp/kegg/">http://www.genome.jp/kegg/</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kanehisa <em>et al.</em> 2008</a>), REACTOME (<a href="http://www.reactome.org/">http://www.reactome.org/</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Matthews <em>et al.</em> 2009</a>), or WikiPathways (<a href="http://www.wikipathways.org/">http://www.wikipathways.org/</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pico <em>et al.</em> 2008</a>), which are often redundant. For this reason, we filtered the pathways according to their size and redundancy in two steps. First, we removed pathways with &lt;10 and &gt;150 genes (150 corresponds to two SD in the distribution of number of genes per pathway); this was aimed at discarding pathways that were either not informative or too generic and complex. For instance, among the pathways with over 150 genes we find: metabolic pathways, gene expression, metabolism, hemostasis, immune system, and neuronal system. Second, for pathways sharing &gt;50% of their genes, we selected the largest one.</p><p id="p-15">We obtained an empirical <em>P</em>-value for Fst and nSL for each pathway following <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dall’Olio <em>et al.</em> (2012)</a>. First, an empirical <em>P</em>-value for each SNP was obtained by ranking the statistics (Fst or nSL). Thus, a SNP with Fst (or nSL) ranked as the i-largest out of N SNPs, was assigned a <em>P</em>-value of i/N. Next, we obtained a gene <em>P</em>-value with Fisher’s statistics, which combines several independent <em>P</em>-values:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2171/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2171/embed/mml-math-1.gif"/></noscript></span> </span>where <em>S</em> is the number of SNPs for the gene analyzed (<em>i.e.</em>, those within the gene boundaries in the Ensembl database) and <em>P<sub>j</sub></em> each associated <em>P</em>-value; since <em>x</em> is distributed as a χ<sup>2</sup> test with 2<em>N</em> d.f., we can obtain a combined <em>P</em>-value for the gene. In a second step, we repeated the same procedure by combining the <em>P</em>-values of each gene in the pathway to obtain a pathway <em>P</em>-value. The actual significance of this <em>P</em>-value is difficult to interpret since the null hypothesis is not clearly defined; therefore, we carried out permutations to determine significance. Since each pathway differs in number of genes, we carried out 1000 permutations for random gene sets of sizes 10–150 genes and differing by increments of 10 genes. In these permutations, dummy pathways were assembled using the <em>P</em>-values of randomly sampled genes, and the actual pathway <em>P</em>-value was compared with the null distribution obtained by permutation. To account for multiple testing, we used the <em>q</em>-value (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>), computed with R-package <em>qvalue</em> (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Storey <em>et al.</em> 2015</a>), to determine significant pathways using the <em>P</em>-values obtained by permutation.</p><p id="p-16">Critically, Fisher’s statistics are based on the premise of independence between <em>P</em>-values, and this is not guaranteed with sequence data given the extreme disequilibrium between nearby SNPs. To avoid this, we pruned the SNP dataset by selecting those positions that minimized LD using the PLINK v.1.9 program (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chang <em>et al.</em> 2015</a>), setting the variant inflation factor equal to two. With this approach, the <em>P</em>-value obtained from Fisher’s statistics was independent of the number of SNPs for each gene (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>). It should be mentioned that the nSL statistics were computed using all SNPs for which the ancestral allele could be determined, since nSL measures LD in the number of SNP units, but only the values for SNPs in equilibrium were retained to obtain the gene <em>P</em>-value, as for Fst metrics.</p><p id="p-17">To investigate whether significance could be partly explained by Asian introgression in EUDM, we carried out a semisupervised ADMIXTURE (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> 2009</a>) analysis. We extracted SNPs from all genes pertaining to the given pathway and we ran ADMIXTURE with <em>K</em> = 2. We run a semisupervised analysis where all EUWB were assigned <em>K</em> = 1 and all Asian pigs were assigned <em>K</em> = 2, and we let the program compute the fraction of the EUDM genomes due to Asian origin. We did this for each significant pathway and for a random set of pathways with a similar number of genes.</p><p id="p-18">Further, we built a coassociation network to visualize pathway relationships. Significant pathway-to-pathway connections were identified using the PCIT network inference algorithm (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Reverter and Chan 2008</a>). The PCIT algorithm is a soft-thresholding method that exploits the twin concepts of Partial Correlation and Mutual Information. In brief, it explores relationships between all possible triplets of nodes (<em>i.e.</em>, pathways in our context), in an attempt to determine truly informative correlations between node pairs once the numerical influence of other nodes in the system has been accounted for. Clustering was based on 10 variables per pathway: the six pathway <em>P</em>-values for Fst (one value per continent) and nSL (one value per population) metrics, and nucleotide diversity in each of the four populations (ASWB, ASDM, EUWB, and EUDM), averaged for each gene in the pathway. We estimated Tajima’s nucleotide diversity (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Tajima 1983</a>, <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">1989</a>) per gene per population with the methods developed by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ferretti <em>et al.</em> (2012)</a>, which account for missing data, using mstatspop software (S. Ramos-Onsins, unpublished data, available at <a href="http://bioinformatics.cragenomica.es/numgenomics/people/sebas/">http://bioinformatics.cragenomica.es/numgenomics/people/sebas/</a>). We visualized the resulting network using Cytoscape (<a href="http://www.cytoscape.org">www.cytoscape.org</a>, <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Shannon <em>et al.</em> 2003</a>). In the visualization scheme, we mapped the pathways (nodes) to a series of attributes to help identify emerging properties. These included number of genes in the pathway, pathway source (KEGG or REACTOME), population with lowest Fst <em>P</em>-value (Asia or Europe), and pathway nucleotide variability.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-19">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results-discussion" id="sec-7"><h2 class="">Results and Discussion</h2><div id="sec-8" class="subsection"><h3>Genetics mirrors geography, to an extent</h3><p id="p-20">Out of the 163 genomes, we initially identified 71,458,035 autosomal SNPs. After quality filtering, removing the positions with &gt;30% of missing data, and discarding singletons, these were reduced to 48,008,185 SNPs. Of those, 31,363,201 (65%) were annotated in dbSNP (<a href="https://www.ncbi.nlm.nih.gov/SNP">https://www.ncbi.nlm.nih.gov/SNP</a>), and the ancestral allele could be determined in 44,417,146 sites. By continent, Asia had a much larger number of private SNPs than Europe, 25,258,008 <em>vs.</em> 5,726,610, as expected from the fact that the species is of Asian origin and that European populations suffered a strong bottleneck, as has been observed in previous studies (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bianco <em>et al.</em> 2015b</a>).</p><p id="p-21">A heatmap of genetic distances between all samples and SNPs showed the well-known split between Asia and Europe (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>). The European heatmap (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) shows that the main division is between wild boar and local breeds Iberian, Mangalica, and Yucatan <em>vs.</em> international pig breeds Pietrain, Landrace, Large White, and Duroc. Further, all EUWBs were clustered together except the two Near East wild boars, which were grouped in a separate branch. The Yucatan, a miniature pig developed in the USA starting with local Mexican pigs and that still retains an important percentage of ancestry from Iberian pigs (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Burgos-Paz <em>et al.</em> 2013</a>), formed a separate group but was closer to local pigs than to international breeds. Among those, Duroc was genetically more separate from the rest of the international pig breeds.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Heatmap of the European individuals using the molecular relationship matrix, computed using all available autosomal single nucleotide polymorphisms (SNPs). (B) Heatmap of the Asian pigs. In Europe, breed codes are DU, Duroc; IB, Iberian; LR, Landrace; LW, Large White; MG, Mangalitza; PI, Pietrain; and YU, Yucatan minipig. In Asia, breed codes are BX, Bamaxiang; HT, Hetao; LA, Laiwu; LU, Luchuan; MI, Minzhu; MS, Meishan; ST, Sichuan; TT, Tibet; WU, Wuzhishan; and YT, Yunnan. Colors are used to differentiate among the populations: ASDM (Asian domestic, blue), ASWB (Asian wild boar, purple), EUDM (European domestic, green), and EUWB (European wild boar, dark red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1479253965" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Heatmap of the European individuals using the molecular relationship matrix, computed using all available autosomal single nucleotide polymorphisms (SNPs). (B) Heatmap of the Asian pigs. In Europe, breed codes are DU, Duroc; IB, Iberian; LR, Landrace; LW, Large White; MG, Mangalitza; PI, Pietrain; and YU, Yucatan minipig. In Asia, breed codes are BX, Bamaxiang; HT, Hetao; LA, Laiwu; LU, Luchuan; MI, Minzhu; MS, Meishan; ST, Sichuan; TT, Tibet; WU, Wuzhishan; and YT, Yunnan. Colors are used to differentiate among the populations: ASDM (Asian domestic, blue), ASWB (Asian wild boar, purple), EUDM (European domestic, green), and EUWB (European wild boar, dark red).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2171/F1.medium.gif" width="213" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/7/2171/F1.medium.gif" width="213" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485395" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">(A) Heatmap of the European individuals using the molecular relationship matrix, computed using all available autosomal single nucleotide polymorphisms (SNPs). (B) Heatmap of the Asian pigs. In Europe, breed codes are DU, Duroc; IB, Iberian; LR, Landrace; LW, Large White; MG, Mangalitza; PI, Pietrain; and YU, Yucatan minipig. In Asia, breed codes are BX, Bamaxiang; HT, Hetao; LA, Laiwu; LU, Luchuan; MI, Minzhu; MS, Meishan; ST, Sichuan; TT, Tibet; WU, Wuzhishan; and YT, Yunnan. Colors are used to differentiate among the populations: ASDM (Asian domestic, blue), ASWB (Asian wild boar, purple), EUDM (European domestic, green), and EUWB (European wild boar, dark red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The picture was somewhat more complex in Asia (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>), although pigs were also grouped by breed. In contrast to Europe, we observed a genetic split between North and South wild boars, in agreement with previous results (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ai <em>et al.</em> 2015</a>). Nevertheless, this geographic pattern was not so evident among the domestic pigs, <em>e.g.</em>, North Asian breeds (Laiwu, Hetao, and Minzhu), which are less separated from those from the South, compared to wild populations.</p></div><div id="sec-9" class="subsection"><h3>Pathway statistics</h3><p id="p-24">We retrieved 1789 pathways comprising 7157 genes from the NCBI database, which were reduced to a final set of 442 pathways with 5713 genes after filtering (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>) by size (<em>e.g.</em>, number of genes) and redundancy. Note that only 25% of pathways but 80% of genes were retained, showing the large redundancy in terms of genes across pathways. Most discarded pathways (676) were very small and contained &lt;10 genes. The distribution of genes per pathway was highly leptokurtic (Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>).</p></div><div id="sec-10" class="subsection"><h3>Differentiation metrics (Fst)</h3><p id="p-25">Differentiation (Fst) analysis indicates that allele frequency changes occurred in pathways associated with some important biological processes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We found more significant pathways in Asia than in Europe; there were 21 pathways significantly differentiated at a <em>q</em>-value &lt; 0.05 in Asia and 12 in Europe, involving a total of 1065 and 576 genes, respectively. Pathways were predominantly continent-specific, but five pathways were differentiated in both continents: integrin cell surface interactions, insulin secretion, pancreatic secretion, ABC transporters, and glutamatergic synapse. Our results are unlikely to be an artifact caused by differential recombination rate, as we found no correlation between Fst and recombination rate (Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>). This contrasts with what has been observed in humans (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Keinan and Reich 2010</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485415/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485415/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Significant pathways (<em>q</em>-value &lt; 0.05) obtained in the Fst analysis in Asia and/or Europe</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">In Asia, we found six significant pathways related to behavior (serotonergic synapse, dopaminergic synapse, glutamatergic synapse, opioid signaling, long-term depression, and adrenergic signaling in cardiomyocytes). This is remarkable since it has long been recognized that domestication has affected behavior, yet the genetic basis for these changes has not been convincingly identified. The six pathways included a total of 264 genes, which codify for proteins involved in the metabolism of important neurotransmitters like serotonin, dopamine, and L-glutamate. Serotonin and dopamine are involved in aggression (serotonin) and reinforcement and reward (dopamine), whereas L-glutamate is the major excitatory neurotransmitter in the central nervous system. Clearly, aggression and reward must have played a role at least during the early stages of domestication and the genetic causes are likely shared between all domestic breeds. Pathway “adrenergic signaling in cardiomyocytes” involves several adrenaline receptors and calcium channels such as ryanodine receptor 2 (RYR2). While RYR2 is primarily expressed in cardiomyocytes, its isoform RYR1 is expressed in skeletal muscle and is well-known in pig genetics for being responsible for the pale, soft, and exudative syndrome (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fujii <em>et al.</em> 1991</a>).</p><p id="p-29">For the six behavior pathways, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the <em>P</em>-values of all genes that were significant at the 1% nominal level either in Europe, Asia, or both. Although behavior pathways were mainly significant only in Asia (except glutamatergic synapse, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), several individual genes were significant in both continents, foremost phospholipase C β 1 (PLCB1), which was significant in both continents and was present in all six behavior pathways. It can be suspected that these pathways were significant only because they contained the PLCB1 gene; however, PLCB1 was involved in a total of 35 pathways, and only 14 were significant (<em>q</em>-value &lt; 0.05). Furthermore, we also computed the pathway <em>P</em>-value excluding PLCB1 and found only a modest decrease in significance (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>). With our approach, it is unlikely then that a single gene is responsible for significance at the pathway level. Note that this is reasonable under a multi-cause mechanism but may prevent the researcher from identifying pathways where a single gene has main responsibility for the rate limiting effect on the whole pathway. In all, PLCB1 plays an important role in the intracellular transduction of many extracellular signals mediated by calcium. It cleaves the PIP2 molecule into IP3 and DAG. DAG, together with Ca<sup>2+</sup> (its secretion is activated by ITPR3, also significant in Europe), activates PKC, which plays a central role in activating numerous functions such as transcription, the immune response, growth, learning, and smooth muscle contraction. This explains its presence in so many different pathways. Importantly, PLCB1 is expressed in select areas of the brain, including the cerebral cortex, hippocampus, amygdala, lateral septum, and olfactory bulb (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Koh <em>et al.</em> 2007</a>). In humans, deficiencies in this gene are associated with some kinds of epilepsy (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ngoh <em>et al.</em> 2014</a>). Another interesting and significant gene in both continents was GSK3B (Glycogen synthase kinase-3), which is involved in energy metabolism, neuronal cell development, and body pattern formation.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene P-values (&#x2212;log10) of significant genes at the 1% nominal level in Europe (red bars), in Asia (blue bars,) or both continents (black bars) from the significant differentiated (Fst) pathways involved in behavior. When a gene was significant in both continents, the smallest P-value is plotted. Gene symbols are provided when available; otherwise numbers indicate ensembl ENSSSCG id." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1479253965" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene &lt;em&gt;P&lt;/em&gt;-values (&#x2212;log10) of significant genes at the 1% nominal level in Europe (red bars), in Asia (blue bars,) or both continents (black bars) from the significant differentiated (Fst) pathways involved in behavior. When a gene was significant in both continents, the smallest &lt;em&gt;P&lt;/em&gt;-value is plotted. Gene symbols are provided when available; otherwise numbers indicate ensembl ENSSSCG id.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2171/F2.medium.gif" width="419" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/7/2171/F2.medium.gif" width="419" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485472" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">Gene <em>P</em>-values (−log10) of significant genes at the 1% nominal level in Europe (red bars), in Asia (blue bars,) or both continents (black bars) from the significant differentiated (Fst) pathways involved in behavior. When a gene was significant in both continents, the smallest <em>P</em>-value is plotted. Gene symbols are provided when available; otherwise numbers indicate ensembl ENSSSCG id.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The rest of the significantly differentiated pathways comprises those related to glucose metabolism (insulin and pancreatic secretion) and development (Wnt signaling, Hippo signaling, and axon guidance) in Asia, and recombination or muscle contraction in Europe. Hippo and Wnt signaling pathways are intimately related, and half of all significant genes were shared (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>). In contrast, vascular smooth muscle contraction and muscle contraction share only 16 significant genes out of 85 and 117 genes, respectively. Significant genes in the insulin pathway include PLCB1 and RYR2, as well as potassium and calcium channels that act on insulin granules and insulin transcription (KCNN1, KCNN2, and KCNMB1). The Hippo signaling pathway controls organ size, a fundamental target during domestication and modern breeding. Wnt signaling, in turn, is one of the most relevant and highly conserved signal transduction pathways, and it has a fundamental role in embryonic development. Hippo and Wnt are tightly interconnected signaling cascades, although their mechanisms differ; Hippo is mainly sensitive to cell density, whereas Wnt responds to concentrations of specific proteins (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Irvine 2012</a>). Interestingly, there is also a direct relationship between the Wnt and insulin pathways, as Wnt signaling increases cells’ insulin sensitivity. The three most significant genes in the Wnt pathway were PLCB1 (shared with other pathways, see <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>), inversin, which contains calmodulin domains and is involved in renal development, and GSK3B, also involved in body pattern formation. GSK3B is also a negative regulator of glucose hormone control.</p><p id="p-32">It is finally worth mentioning two significant pathways involved in recombination, “DNA double strand break repair” and “nonhomologous end-joining” (Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>, which also shows the rest of the significant Fst pathways). The issue of the effect of recombination on domestication has been debated for a long time in the literature. Theoretical models have predicted that domestication should increase recombination, as rapid selection indirectly favors an increased recombination rate such that the Hill–Robertson effect is less limiting for response, and this prediction has been confirmed using chiasma data from the literature (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ross-Ibarra 2004</a>). In a classic paper, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ollivier (1995)</a> also showed that the wild boar linkage map was ∼33% shorter than domestic pig maps. Nevertheless, other recent studies in sheep, goat, and dogs have ruled out changes in recombination rates compared to their wild ancestors (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Munoz-Fuentes <em>et al.</em> 2015</a>). Therefore, the significant differentiation found here in this pathway may not be paralleled with changes in recombination rate caused by domestication.</p></div><div id="sec-11" class="subsection"><h3>LD metrics</h3><p id="p-33">We repeated the same statistical procedure as for Fst with the nSL statistics, which measures LD instead of differentiation, and that is especially powerful for identifying soft sweeps (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Ferrer-Admetlla <em>et al.</em> 2014</a>). Each of the four populations, ASDM, ASWB, EUDM, and EUWB, were analyzed separately. Overall, there were fewer significant pathways at a <em>q</em>-value &lt; 0.05 with nSL than with Fst (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> <em>vs.</em> <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). In particular, we did not find a significant value in either EUWBs nor in ASWBs, perhaps because there were fewer wild than domestic pigs. In comparison to Fst, concordance between continents with nSL was very high in domestic pigs, as we found the same six out of seven significant pathways in both Asia and Europe. The only exception was pathway “Inflammatory mediator regulation of TRP channels” involved in the immune response, which was significant only in Asia. A potential matter of concern with pathway analysis is its definition. As an example, we found that arachidonic acid metabolism pathways annotated by KEGG (NCBI id 84417) and REACTOME (NCBI id 1336691) contained 24 shared genes out of a total of 47 and 39, respectively. Nevertheless, the significant genes (<em>P</em>-value &lt; 0.01) of both pathways were the same. As noted by <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Mooney <em>et al.</em> (2014)</a>, different databases may contain different genes to represent the same biological process; this is a warning of the fact that pathway definition is not an unambiguous concept, and different criteria can legitimately be used to define a given biological process.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485496/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485496/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Significant pathways (<em>q</em>-value &lt; 0.05) obtained in the nSL analysis for the four populations, according to continent Europe (EU)/Asia (AS) and domestic (DM)/wild (WB) status</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">Two of the significant pathways are directly linked to reproductive performance (“Ovarian steroidogenesis” and “Steroid hormone biosynthesis”). Importantly, we also identified ovarian steroidogenesis in our previous work (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Pérez-Enciso <em>et al.</em> 2016</a>) in a much smaller study on domestication merging ASDM and EUDM <em>vs.</em> ASWB and EUWBs, and where a completely different analytical approach was employed. The remaining significant pathways were related to lipid metabolism, in particular to linoleic and arachidonic metabolism. Importantly, some of the final products of linoleic metabolism are THF-diols, which are converted into prostaglandins and are involved in the sexual behavior of males and the ovarian cycle in females. Therefore, most pathways identified with nSL are interrelated and linked to reproduction.</p><p id="p-37">Among the most significant genes in ovarian steroidogenesis, there appears the uncharacterized gene ENSSSCG00000003824. This gene seems to be orthologous to UGT2B (UDP glycosyltransferase), a cluster of genes involved in the glucuronidation of estrogens. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the significant gene <em>P</em>-values for the significant nSL pathways. It is interesting to note that a more coherent signal across continents emerged with nSL than with Fst metrics, since the most significant genes with nSL are shared between continents (<em>e.g.</em>, compare <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Significant genes at the 1% nominal level either in Europe, Asia or both, present in the significant pathways obtained from the nSL (linkage disequilibrium) analysis. Gene symbols are provided when available; otherwise numbers indicate ensembl ENSSSCG id." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1479253965" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Significant genes at the 1% nominal level either in Europe, Asia or both, present in the significant pathways obtained from the nSL (linkage disequilibrium) analysis. Gene symbols are provided when available; otherwise numbers indicate ensembl ENSSSCG id.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2171/F3.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/7/2171/F3.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485417" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Significant genes at the 1% nominal level either in Europe, Asia or both, present in the significant pathways obtained from the nSL (linkage disequilibrium) analysis. Gene symbols are provided when available; otherwise numbers indicate ensembl ENSSSCG id.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Pathways are interrelated</h3><p id="p-39">Much as genes do not act in isolation, neither do pathways. To represent this, a coassociation network was built with all 35 significant pathways, either with Fst or nSL analysis (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> merged). The metrics used for the clustering contained the pathway Fst and nSL <em>P</em>-values together with nucleotide variabilities (see <em>Materials and Methods</em>). The entire network of pathways contained 83 negative and 129 positive connections. Note that the interpretation of a “positive” or “negative” connection is not straightforward, as is often the case in multivariate methods. The sign would indicate that domestication and/or breeding has exerted similar or opposite effects on the variables used to build the network, conditional on the fact that pathways are significant in at least one analysis. The three most connected pathways, each with 19 connections, were arachidonic acid metabolism (NCBI id 1336691) with 45 genes, glutamatergic synapse with 122 genes, and dopaminergic synapse with 119 genes. When the minimum correlation was set to 0.80 in absolute value (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), four pathways were not sufficiently connected (nuclear signaling by ERBB4, Chagas disease, serotonergic synapse, and ABC transporters). In turn, three clusters of highly interconnected pathways are immediately apparently in the network visualization of <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>. Cluster A contains nine pathways with higher than average nucleotide diversity that show strong positive connections between them. Prominent in this cluster is the axon guidance pathway with 119 genes. Its connections with cell–cell communication, G α signaling events, and assembly of the primary cilium via the insulin secretion pathways suggest that this cluster mainly involves extracellular guidance such as growth and hormonal regulation, helping axons reach their targets (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dickson 2003</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coassociation network among the 31 interconnected significant pathways. Each node represents a pathway that is connected by an edge if partial correlation with another pathway is significant and &gt;0.8 (in absolute value). Node size is proportional to number of genes in the pathways. Node shapes represent pathway source: triangles for REACTOME and circles for KEGG. Colors indicate the population with lowest Fst (statistical significance for differentiation) P-value: pink for Asia, blue for Europe, and green for equal significance. Node line width to pathway variability: thin and thick lines for pathways with variability below and above average, respectively. Black and red edges represent positive and negative correlations between pathways, respectively. The three main pathway clusters are identified with letters A&#x2013;C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1479253965" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Coassociation network among the 31 interconnected significant pathways. Each node represents a pathway that is connected by an edge if partial correlation with another pathway is significant and &gt;0.8 (in absolute value). Node size is proportional to number of genes in the pathways. Node shapes represent pathway source: triangles for REACTOME and circles for KEGG. Colors indicate the population with lowest Fst (statistical significance for differentiation) &lt;em&gt;P&lt;/em&gt;-value: pink for Asia, blue for Europe, and green for equal significance. Node line width to pathway variability: thin and thick lines for pathways with variability below and above average, respectively. Black and red edges represent positive and negative correlations between pathways, respectively. The three main pathway clusters are identified with letters A&#x2013;C.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2171/F4.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/7/2171/F4.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2171/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485507" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-40" class="first-child">Coassociation network among the 31 interconnected significant pathways. Each node represents a pathway that is connected by an edge if partial correlation with another pathway is significant and &gt;0.8 (in absolute value). Node size is proportional to number of genes in the pathways. Node shapes represent pathway source: triangles for REACTOME and circles for KEGG. Colors indicate the population with lowest Fst (statistical significance for differentiation) <em>P</em>-value: pink for Asia, blue for Europe, and green for equal significance. Node line width to pathway variability: thin and thick lines for pathways with variability below and above average, respectively. Black and red edges represent positive and negative correlations between pathways, respectively. The three main pathway clusters are identified with letters A–C.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Most of the corresponding pathways of the other two clusters are negatively related between them. Cluster C is composed of processes related with the sympathetic nervous system, which is activated in response to stress by neurotransmitters such as dopamine (dopaminergic synapse) and glucocorticoids, which induce glutamate release (glutamatergic synapse, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Popoli <em>et al.</em> 2011</a>). Several other processes in cluster C are activated in response to the activation of the sympathetic nervous system, for instance increased heart contraction (adrenergic signaling in cardiomyocytes), blood vessels constriction in some parts of the body (vascular smooth muscle contraction and renin secretion), blood vessel dilatation in muscle and muscle contraction (muscle contraction), and energy obtainment by lipids and carbohydrate degradation (pancreatic secretion). These processes are negatively connected with pathways in cluster B, which contains hormone-controlled processes related to reproduction (steroid hormone biosynthesis, ovarian steroidogenesis, linoleic acid metabolism, and arachidonic acid metabolism), and that are inhibited in stress events, like the pathway “Inflammatory mediator regulation of TRP channels.”</p><p id="p-42">Finally, some genes also appeared repeatedly across pathways, which may indicate a central role in some biochemical routes. The list of genes present in at least 10 of the significant pathways is in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. Most of these genes are enzymes involved in general processes, such as phospholipases PLCB1 and PLCB3 involved in the transduction of many signals; PLA2G4A and PLA2G4B, which release arachidonic acid; kinases (PRKCA, PRKCG, PRKACA, and MAPK1) involved in development; and adenylate cyclases (ADCY2, ADCY3, and ADCY4), which are part of the signal transduction of G proteins, <em>e.g.</em>, affecting dopamine. In addition, we also found the receptor ITPR3, which has an important role in taste transduction and is involved in the activation process of PKC that, as explained above, acts on several processes. Some taste receptors have been shown to be affected by domestication (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">da Silva <em>et al.</em> 2014</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485426/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485426/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Genes present in 10 or more significant pathways (Fst)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Impact of deleterious mutations rate</h3><p id="p-44">We found 123,571 synonymous and 138,121 nonsynonymous SNPs, of which 75,486 were predicted by the VEP tool (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">McLaren <em>et al.</em> 2010</a>) to be tolerated and 62,635 to be deleterious. In order to investigate whether the significant pathways and their genes have a larger proportion of deleterious <em>vs.</em> tolerated variants than the rest of pathways, we classified the SNPs in three groups: (i) SNPs in nonsignificant genes of nonsignificant pathways; (ii) SNPs in nonsignificant genes (<em>P</em>-value &gt; 0.01) from significant pathways; and (iii) SNPs in significant genes (<em>P</em>-value &lt; 0.01) of significant pathways. <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the count of predicted deleterious and tolerated SNPs by group according to continent and domestic/wild status populations. In all four populations, there was a systematic trend of decreasing deleterious/tolerated rate with SNPs in significant genes of significant pathways compared to SNPs from non-significant pathways. The χ<sup>2</sup> test was significant in Europe and when all populations were jointly considered (<em>P</em> &lt; 0.01) but not in Asia. These results can be interpreted as an increased functional constraint (lower ratio of deleterious mutations) in significant genes from significant pathways rather than in genes from nonsignificant pathways.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485406/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485406/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Deleterious and tolerated SNPs grouped according to Europe (EU)/Asia (AS) continent and domestic (DM)/wild (WB) status</span><div class="sb-div caption-clear"></div></div></div><p id="p-48">Previous studies suggest that domestication has resulted in an increased accumulation of deleterious mutations (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cruz <em>et al.</em> 2008</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Renaut and Rieseberg 2015</a>; <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Pérez-Enciso <em>et al.</em> 2016</a>). In agreement with this, we observed larger ratio of deleterious <em>vs.</em> tolerated mutations in domestics than in wild boars (λ<sub>ASDM</sub>/λ<sub>ASWB</sub> and λ<sub>EUDM</sub>/λ<sub>EUWB</sub> in <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Interestingly, these ratios are higher in significant genes than in random gene SNPs, and also higher in Europe than in Asia. Therefore, these data suggest that potential purifying selection is weaker/less effective in Europe than in Asia, likely because of the well-known low effective population size of old world pigs (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Groenen <em>et al.</em> 2012</a>). Besides, even if within-population purifying selection was stronger in significant genes, it was comparatively weaker in domestic than in wild populations.</p></div><div id="sec-14" class="subsection"><h3>General discussion</h3><p id="p-49">We report a functional analysis of pig domestication and breeding using a large complete sequence dataset that consisted of 40 wild boars and 123 domestic pig genomes from Asia (mainly China) and Europe. Rather than a standard exploratory genome-wide analysis, we focused on an analysis where the unit of study was the pathway. Genes do not function in isolation but coordinately, and thus metabolic pathways provide a reasonable scaffold to accommodate this fact (<em>e.g.</em>, <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Daub <em>et al.</em> 2013</a>). In a previous study, we observed that the “heritability” of domestic status varied according to pathway and that differences were not due to the number of genes in the pathway, suggesting that pathway can be a meaningful analysis unit (<a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Pérez-Enciso <em>et al.</em> 2016</a>). In fact, one of the main advantages of this approach is that it provides a direct biological interpretation of the analyses, although an independent source of information may be required to conclude which tissue and developmental stage the perturbed pathway may act in. In contrast, standard window-based genome-wide scans may pinpoint regions devoid of annotations or where the functional relation between significant windows is unknown. We assessed two metrics, differentiation (Fst) and disequilibrium (nSL), and although some of the pathways were connected (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>), we found little concordance between the two analyses. Lack of agreement between differentiation and disequilibrium statistics have been reported previously (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2016</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Dall’Olio <em>et al.</em> 2012</a>), and this is likely because of the different timing and persistence of effects caused by selection (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Sabeti <em>et al.</em> 2006</a>). In particular, since disequilibrium erodes rapidly, our analysis suggests that reproductive changes (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) are among the most recent ones, whereas others such as development and behavior (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) were earlier targets of domestication and/or breeding. This is coherent with current knowledge, as behavioral changes must have occurred in earlier stages of domestication, as exemplified by the important experiment for tameness in foxes (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kukekova <em>et al.</em> 2012</a>), whereas an emphasis on increasing reproductive performance is a more recent target of modern breeding.</p><p id="p-50">Our approach has limitations as well. Foremost, many genes are not assigned to any pathway. In the NCBI Biosystems database v.160202 used here, 7157 genes out of a total of 21,691 annotated genes (Ensembl genes v. 83) were assigned to at least one pathway. After filtering, we further restricted the analysis to 442 pathways containing 5713 genes; these correspond to a total of 220 Mb or ∼8.5% of the whole genome. Another issue is redundancy and the definition of a pathway itself, since there are several databases (KEGG, REACTOME, and Interactome, etc.) that contain lists of functionally related genes. Definitions of the same pathways can actually be quite different between databases (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Mooney <em>et al.</em> 2014</a>), as we observed here with the arachidonic metabolism pathways (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Here, as in <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Daub <em>et al.</em> (2013</a>), we decided to initially consider all available pathways from the NCBI biosystems database, although we set a maximum redundancy between pathways of 50%. However, in contrast to <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Daub <em>et al.</em> (2013)</a> who considered the most significant SNP from each gene and removed all gene redundancy between pathways, here we combined all SNPs (after pruning for LD) from a given gene into a single statistics using Fisher’s method, and we allowed a 50% gene redundancy. It is not evident which method is best, but it seems that our approach is more conservative since outlier Fst will be smoothed out unless a general trend across SNPs in the whole gene is maintained. In turn, allowing for gene redundancy allows us to keep the original gene set instead of pathway pruning.</p><p id="p-51">The history of domestication and domestic breeds is quite complex. In addition to multiple independent domestication events, as occurred in Asia and in Europe in the case of the pig, local adaptive processes have occurred since different breeds have been selected for different traits. Therefore, it is not surprising that previous works (<em>e.g.</em>, <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Amaral <em>et al.</em> 2011</a>) reported that most of selective signals were breed-specific, although our work demonstrates that shared domestication and breeding signals across breeds can still be detected. These signals are numerous and none of them are strong enough to explain the whole process. Once more, the polygenic model prevails. We further show that Asian and European domestication/breeding processes have both distinctive and shared pathways, and that multiple processes have been involved, such as an increase in disequilibrium and in differentiation. Differentiation metrics (Fst) revealed a larger number of signals than disequilibrium (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a> <em>vs.</em> <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>), but this may be due to the experimental design: we analyzed several breeds jointly and disequilibrium is more rapidly eroded by demographic processes than differentiation (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Sabeti <em>et al.</em> 2006</a>). Nevertheless, we often found genes that were significant in both continents, such as several genes involved in behavior (PLCB1, GSK3B, and HTR4, <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), while the pathways they belong to were significant in only one continent. Given that most European breeds have been admixed with Asian pigs (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Groenen 2016</a>), it is possible that these shared signals may actually be due to introgression. To verify this, we ran a semisupervised ADMIXTURE analysis on all significant (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>) and a set of random pathways (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1/FileS1.docx">File S1</a>). The average Asian component in EUDM pigs across significant pathways was <em>q</em> = 0.11 (SD = 0.03), which is nearly identical to that observed in a random set of pathways (0.11, SD = 0.02). Similarly, we did not find differences in Asian component between shared significant pathways across continents (<em>q</em> = 0.105) and those that were continent-specific (<em>q</em> = 0.102). This suggests that Asian introgression is unlikely to have caused a shared signal between continents, which can be explained because the Asian signature in European breeds seems to be quite heterogeneous, <em>i.e.</em>, due to different Asian origins (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bosse <em>et al.</em> 2014</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bianco <em>et al.</em> 2015a</a>).</p><p id="p-52">In contrast to previous works in humans, which reported an enrichment of pathways related to pathogen response in adaptation (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Daub <em>et al.</em> 2013</a>), we did not find a strong overrepresentation of immune system-related pathways. Only three related pathways were detected with Fst (complement cascade, Chagas disease, and FCGR phagocytosis, <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). This could be due to the fact that domestication was accompanied by stronger selection for traits other than disease resistance such as behavior, reproduction, or development. Another explanation is that these disease resistance signals were breed-specific and therefore remained undetected in this experimental design.</p><p id="p-53">As in other studies (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cruz <em>et al.</em> 2008</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Renaut and Rieseberg 2015</a>; <a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Pérez-Enciso <em>et al.</em> 2016</a>), we found an increased accumulation of deleterious mutations in domestic animals. We systematically observed a higher proportion of deleterious variants in domestic groups compared to wild boars. This was observed for all genes, regardless of whether they were significant or not. On the other hand, a decreased accumulation of deleterious mutations was observed in significant genes from significant pathways, suggesting, as shown in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, that these genes perform essential and central tasks in the physiology and development of the pig. Therefore, these genes seem to be under stronger functional constraint than randomly sampled genes.</p></div><div id="sec-15" class="subsection"><h3>Conclusions</h3><p id="p-54">We have studied the functional basis of domestication and breeding in the pig. This was possible because a modern equivalent of the wild ancestor is still available for study and was facilitated by the numerous sequences in the public domain. We show that these processes predominantly involved pathways related to behavior, especially in Asia, but also others like insulin, organ size development, recombination, and female reproduction. At least in part, these results can be explained by a relaxation of purifying selection associated with the domestication and/or breeding processes. Nevertheless, this purifying selection was stronger in genes and pathways that were significant using <em>Fst</em> than in random genes, likely because these genes play central roles and are highly functionally constrained. Negative selection was also stronger in Asia than in Europe, likely due to the larger effective size of the Asian population. In all probability, this analysis is conservative since we have focused on SNPs and genes that are consistently differentiated between all domestic breeds pooled together <em>vs.</em> wild boar. Focusing on a specific breed may have increased power but also could be prone to false discovery rate, identifying signals that are breed specific rather than domestic specific.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We thank Sara Guirao-Rico, Marcel Amills, and Sebastián Ramos-Onsins for comments. Part of this work was developed while M.P.-E. was visiting CSIRO Agriculture at the St. Lucia campus (Brisbane, Australia), funded by a CSIRO McMaster Visiting Fellowship. J.L.-C. is the recipient of a Training Grant for Research Staff (FPI) grant, to achieve a PhD, from the Ministry of Economy and Science (MINECO, Spain). Work was also funded by grants AGL2013-41834-R and AGL2016-78709-R from MINECO to M.P.-E. We also acknowledge the support of MINECO for the “Centro de Excelencia Severo Ochoa 2016–2019” award SEV-2015-0533 and of the Centres de Recerca de Catalunya program (CERCA, Generalitat de Catalunya, Spain) to the Centre for Research in Agricultural Genomics.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></li><li class="fn" id="fn-10"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042671/-/DC1</a>.</p></li><li class="fn" id="fn-11"><p id="p-57"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-06"><span class="received-label">Received </span>January 6, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-05-02"><span class="accepted-label">Accepted </span>May 2, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Leno-Colorado <em>et al.</em></li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.1" data-doi="10.1038/ng.3199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Adaptation and possible ancient interspecies introgression in pigs identified by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D217%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3199%26rft_id%253Dinfo%253Apmid%252F25621459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25621459&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.2" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.3" data-doi="10.1371/journal.pone.0014782"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amaral</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferretti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooijmans</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide footprints of pig domestication and selection revealed through massive parallel sequencing of pooled DNA.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e14782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAmaral%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De14782%26rft.epage%253De14782%26rft.atitle%253DGenome-wide%2Bfootprints%2Bof%2Bpig%2Bdomestication%2Band%2Bselection%2Brevealed%2Bthrough%2Bmassive%2Bparallel%2Bsequencing%2Bof%2Bpooled%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0014782%26rft_id%253Dinfo%253Apmid%252F21483733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0014782&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21483733&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bianco</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soto</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">The chimerical genome of Isla del Coco feral pigs (Costa Rica), an isolated population since 1793 but with remarkable levels of diversity.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D2364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bianco</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevado</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">A deep catalog of autosomal single nucleotide variation in the pig.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0118867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0118867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.7" data-doi="10.1111/mec.12807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bosse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frantz</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paudel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Untangling the hybrid nature of modern pig genomes: a mosaic derived from biogeographically distinct and highly divergent <em>Sus scrofa</em> populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4089</span>–<span class="cit-lpage">4102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4089%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12807&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.8" data-doi="10.1534/genetics.113.150029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Improving the accuracy and efficiency of identity-by-descent detection in population data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150029%26rft_id%253Dinfo%253Apmid%252F23535385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNDU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.9" data-doi="10.1186/1471-2164-15-1112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buitenhuis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L. L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poulsen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsen</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsen</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association and biological pathway analysis for milk-fat composition in Danish Holstein and Danish Jersey cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1112%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1112%26rft_id%253Dinfo%253Apmid%252F25511820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25511820&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgos-Paz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramayo-Caldas</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Porcine colonization of the Americas: a 60k SNP story.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D110%26rft.spage%253D321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.11" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tellier</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vattikuti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of selective sweeps reveals divergent selection between Chinese Holstein and Simmental cattle populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D76%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.13" data-doi="10.1093/molbev/msn177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cruz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilà</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The legacy of domestication: accumulation of deleterious mutations in the dog genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2331</span>–<span class="cit-lpage">2336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn177%26rft_id%253Dinfo%253Apmid%252F18689870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMS8yMzMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">da Silva</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jager</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgos-Paz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reverter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roura</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of the porcine nutrient and taste receptor gene repertoire in domestic and wild populations across the globe</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.15" data-doi="10.1186/1471-2148-12-98"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dall’Olio</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laayouni</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luisi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sikora</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montanucci</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Distribution of events of positive selection and population differentiation in a metabolic pathway: the case of asparagine N-glycosylation.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DDall%2527olio%26rft.auinit1%253DG.%2BM.%26rft.volume%253D12%26rft.spage%253D98%26rft.epage%253D98%26rft.atitle%253DDistribution%2Bof%2Bevents%2Bof%2Bpositive%2Bselection%2Band%2Bpopulation%2Bdifferentiation%2Bin%2Ba%2Bmetabolic%2Bpathway%253A%2Bthe%2Bcase%2Bof%2Basparagine%2BN-glycosylation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-12-98%26rft_id%253Dinfo%253Apmid%252F22731960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-12-98&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22731960&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.16" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.17" data-doi="10.1093/molbev/mst080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daub</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutivet</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupanloup</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintana-Murci</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evidence for polygenic adaptation to pathogens in the human genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1544</span>–<span class="cit-lpage">1558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst080%26rft_id%253Dinfo%253Apmid%252F23625889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMwLzcvMTU0NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.18" data-doi="10.1038/nature01019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diamond</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evolution, consequences and future of plant and animal domestication.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">700</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDiamond%26rft.auinit1%253DJ.%26rft.volume%253D418%26rft.issue%253D6898%26rft.spage%253D700%26rft.epage%253D707%26rft.atitle%253DEvolution%252C%2Bconsequences%2Band%2Bfuture%2Bof%2Bplant%2Band%2Banimal%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01019%26rft_id%253Dinfo%253Apmid%252F12167878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12167878&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177305600058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Molecular mechanisms of axon guidance.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">1959</span>–<span class="cit-lpage">1964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D298%26rft.spage%253D1959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.20" data-doi="10.1186/1471-2164-14-148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esteve-Codina</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paudel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferretti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raineri</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dissecting structural and nucleotide genome-wide variation in inbred Iberian pigs.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-148%26rft_id%253Dinfo%253Apmid%252F23497037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23497037&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.21" data-doi="10.1093/molbev/msu077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrer-Admetlla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">On detecting incomplete soft or hard selective sweeps using haplotype structure.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1275</span>–<span class="cit-lpage">1291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu077%26rft_id%253Dinfo%253Apmid%252F24554778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMxLzUvMTI3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.22" data-doi="10.1534/genetics.112.139949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferretti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raineri</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Neutrality tests for sequences with missing data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">1397</span>–<span class="cit-lpage">1401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139949%26rft_id%253Dinfo%253Apmid%252F22661328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MS80LzEzOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.23" data-doi="10.1146/annurev-animal-021815-111155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frantz</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meijaard</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gongora</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haile</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The evolution of suidae.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci.%26rft.volume%253D4%26rft.spage%253D61%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-021815-111155%26rft_id%253Dinfo%253Apmid%252F26526544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-021815-111155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26526544&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.24" data-doi="10.1038/ng.3394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frantz</span>  <span class="cit-name-given-names">L. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schraiber</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cagan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evidence of long-term gene flow and selection during domestication from analyses of Eurasian wild and domestic pig genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D1141%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3394%26rft_id%253Dinfo%253Apmid%252F26323058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26323058&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.25" data-doi="10.1126/science.1862346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujii</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otsu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorzato</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khanna</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Identification of a mutation in porcine ryanodine receptor associated with malignant hyperthermia.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">253</span>: <span class="cit-fpage">448</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFujii%26rft.auinit1%253DJ%26rft.volume%253D253%26rft.issue%253D5018%26rft.spage%253D448%26rft.epage%253D451%26rft.atitle%253DIdentification%2Bof%2Ba%2Bmutation%2Bin%2Bporcine%2Bryanodine%2Breceptor%2Bassociated%2Bwith%2Bmalignant%2Bhyperthermia%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1862346%26rft_id%253Dinfo%253Apmid%252F1862346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNTMvNTAxOC80NDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.26" data-doi="10.1093/nar/gkp858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geer</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchler-Bauer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geer</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The NCBI BioSystems database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D492</span>–<span class="cit-lpage">D496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp858%26rft_id%253Dinfo%253Apmid%252F19854944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8xL0Q0OTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A decade of pig genome sequencing: a window on pig domestication and evolution.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.28" data-doi="10.1038/nature11622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archibald</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uenishi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuggle</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeuchi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Analyses of pig genomes provide insight into porcine demography and evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGroenen%26rft.auinit1%253DM.%2BA.%26rft.volume%253D491%26rft.issue%253D7424%26rft.spage%253D393%26rft.epage%253D398%26rft.atitle%253DAnalyses%2Bof%2Bpig%2Bgenomes%2Bprovide%2Binsight%2Binto%2Bporcine%2Bdemography%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11622%26rft_id%253Dinfo%253Apmid%252F23151582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23151582&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311031600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ha</span>  <span class="cit-name-given-names">N.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dorland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tetens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Gene-based mapping and pathway analysis of metabolic traits in dairy cows.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0122325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0122325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.30" data-doi="10.1016/j.semcdb.2012.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irvine</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integration of intercellular signaling through the Hippo pathway.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">812</span>–<span class="cit-lpage">817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.stitle%253DSemin%2BCell%2BDev%2BBiol%26rft.aulast%253DIrvine%26rft.auinit1%253DK.%2BD.%26rft.volume%253D23%26rft.issue%253D7%26rft.spage%253D812%26rft.epage%253D817%26rft.atitle%253DIntegration%2Bof%2Bintercellular%2Bsignaling%2Bthrough%2Bthe%2BHippo%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2012.04.006%26rft_id%253Dinfo%253Apmid%252F22554983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2012.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22554983&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.31" data-doi="10.1093/nar/gkm882"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattori</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">KEGG for linking genomes to life and the environment.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D36%26rft.spage%253D480%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm882&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.32" data-doi="10.1371/journal.pgen.1000886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keinan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Human population differentiation is strongly correlated with local recombination rate.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKeinan%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De1000886%26rft.epage%253De1000886%26rft.atitle%253DHuman%2Bpopulation%2Bdifferentiation%2Bis%2Bstrongly%2Bcorrelated%2Bwith%2Blocal%2Brecombination%2Brate.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000886%26rft_id%253Dinfo%253Apmid%252F20361044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20361044&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">H.-S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Deficits in social behavior and sensorimotor gating in mice lacking phospholipase C beta 1.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BBrain%2BBehav.%26rft.volume%253D7%26rft.spage%253D120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.34" data-doi="10.1007/s00335-011-9373-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kukekova</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Temnykh</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trut</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acland</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetics of behavior in the silver fox.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">164</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DKukekova%26rft.auinit1%253DA.%2BV.%26rft.volume%253D23%26rft.issue%253D1-2%26rft.spage%253D164%26rft.epage%253D177%26rft.atitle%253DGenetics%2Bof%2Bbehavior%2Bin%2Bthe%2Bsilver%2Bfox.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-011-9373-z%26rft_id%253Dinfo%253Apmid%252F22108806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-011-9373-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22108806&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.35" data-doi="10.1126/science.1106927"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobney</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albarella</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matisoo-Smith</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Worldwide phylogeography of wild boar reveals multiple centers of pig domestication.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">307</span>: <span class="cit-fpage">1618</span>–<span class="cit-lpage">1621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLarson%26rft.auinit1%253DG.%26rft.volume%253D307%26rft.issue%253D5715%26rft.spage%253D1618%26rft.epage%253D1621%26rft.atitle%253DWorldwide%2BPhylogeography%2Bof%2BWild%2BBoar%2BReveals%2BMultiple%2BCenters%2Bof%2BPig%2BDomestication%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1106927%26rft_id%253Dinfo%253Apmid%252F15761152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDcvNTcxNS8xNjE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.36" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.37" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.38" data-doi="10.1093/nar/gkn863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gopinath</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillespie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croft</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reactome knowledgebase of human biological pathways and processes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D37%26rft.spage%253D619%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn863&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.39" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.40" data-doi="10.1093/bioinformatics/btq330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rios</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flicek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Deriving the consequences of genomic variants with the Ensembl API and SNP effect predictor.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2069</span>–<span class="cit-lpage">2070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq330%26rft_id%253Dinfo%253Apmid%252F20562413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTYvMjA2OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Molnár</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stéger</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tóth</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marincs</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequencing and analysis of Mangalica, a fatty local pig of Hungary.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.42" data-doi="10.1016/j.tig.2014.07.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mooney</span>  <span class="cit-name-given-names">M. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nigg</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McWeeney</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilmot</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Functional and genomic context in pathway analysis of GWAS data.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D390%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.07.004%26rft_id%253Dinfo%253Apmid%252F25154796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.07.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25154796&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.43" data-doi="10.1093/molbev/msu322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Munoz-Fuentes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Ortega</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkorta-Aranburu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forsberg</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Strong artificial selection in domestic mammals did not result in an increased recombination rate.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">510</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu322%26rft_id%253Dinfo%253Apmid%252F25414125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMyLzIvNTEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.44" data-doi="10.1111/mec.12693"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nevado</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Enciso</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Resequencing studies of nonmodel organisms using closely related reference genomes: optimal experimental designs and bioinformatics approaches for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1764</span>–<span class="cit-lpage">1779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D1764%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12693&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000333333500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ngoh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McTague</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wentzensen</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Applegate</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Severe infantile epileptic encephalopathy due to mutations in PLCB1: expansion of the genotypic and phenotypic disease spectrum.</span> <abbr class="cit-jnl-abbrev">Dev. Med. Child Neurol</abbr>. <span class="cit-vol">56</span>: <span class="cit-fpage">1124</span>–<span class="cit-lpage">1128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BMed.%2BChild%2BNeurol%26rft.volume%253D56%26rft.spage%253D1124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ollivier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic differences in recombination frequency in the pig (<em>Sus scrofa</em>).</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1048</span>–<span class="cit-lpage">1051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DOllivier%26rft.auinit1%253DL.%26rft.volume%253D38%26rft.issue%253D5%26rft.spage%253D1048%26rft.epage%253D1051%26rft.atitle%253DGenetic%2Bdifferences%2Bin%2Brecombination%2Bfrequency%2Bin%2Bthe%2Bpig%2B%2528Sus%2Bscrofa%2529.%26rft_id%253Dinfo%253Apmid%252F8536996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8536996&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.47" data-doi="10.1093/molbev/mss136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A critical assessment of storytelling: gene ontology categories and the importance of validating genomic scans.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3237</span>–<span class="cit-lpage">3248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss136%26rft_id%253Dinfo%253Apmid%252F22617950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMC8zMjM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reverter</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The “heritability” of domestication and its functional partitioning in the pig.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D118%26rft.spage%253D160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pico</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelder</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Iersel</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanspers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conklin</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">WikiPathways: pathway editing for the people.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D6%26rft.spage%253D1403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.50" data-doi="10.1038/nrn3138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Popoli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEwen</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanacora</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The stressed synapse: the impact of stress and glucocorticoids on glutamate transmission.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Neurosci.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BNeurosci.%26rft.volume%253D13%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrn3138%26rft_id%253Dinfo%253Apmid%252F22127301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrn3138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22127301&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.51" data-doi="10.1002/0471250953.bi1112s47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">BEDTools: the swiss-army tool for genome feature analysis.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Bioinformatics.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">11.12.1</span>–<span class="cit-lpage">11.12.34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BBioinformatics.%26rft.volume%253D47%26rft.spage%253D11121%26rft_id%253Dinfo%253Adoi%252F10.1002%252F0471250953.bi1112s47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471250953.bi1112s47&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-book" id="cit-7.7.2171.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em></span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramírez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgos-Paz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casas</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballester</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bianco</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome data from a sixteenth century pig illuminate modern breed relationships.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D114%26rft.spage%253D175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgos-Paz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manunza</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amills</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mining the pig genome to investigate the domestication process.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D113%26rft.spage%253D471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.55" data-doi="10.1093/molbev/msv106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The accumulation of deleterious mutations as a consequence of domestication and improvement in sunflowers and other compositae crops.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2273</span>–<span class="cit-lpage">2283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv106%26rft_id%253Dinfo%253Apmid%252F25939650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMyLzkvMjI3MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.56" data-doi="10.1093/bioinformatics/btn482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reverter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">E. K. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Combining partial correlation and an information theory approach to the reversed engineering of gene co-expression networks.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2491</span>–<span class="cit-lpage">2497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn482%26rft_id%253Dinfo%253Apmid%252F18784117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjQvMjEvMjQ5MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.57" data-doi="10.1086/380606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The evolution of recombination under domestication: a test of two hypotheses.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D163%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1086%252F380606%26rft_id%253Dinfo%253Apmid%252F14767840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/380606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14767840&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188954900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.58" data-doi="10.1073/pnas.1217149109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez Barrio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maqbool</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sayyab</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Strong signatures of selection in the domestic pig genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">19529</span>–<span class="cit-lpage">19536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217149109%26rft_id%253Dinfo%253Apmid%252F23151514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ4LzE5NTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.59" data-doi="10.1126/science.1124309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varilly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Positive natural selection in the human lineage.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">312</span>: <span class="cit-fpage">1614</span>–<span class="cit-lpage">1620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D312%26rft.issue%253D5780%26rft.spage%253D1614%26rft.epage%253D1620%26rft.atitle%253DPositive%2Bnatural%2Bselection%2Bin%2Bthe%2Bhuman%2Blineage.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1124309%26rft_id%253Dinfo%253Apmid%252F16778047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTIvNTc4MC8xNjE0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.60" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markiel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baliga</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.61" data-doi="10.1093/nar/gks539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">N.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SIFT web server: predicting effects of amino acid substitutions on proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">W452</span>–<span class="cit-lpage">W457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks539%26rft_id%253Dinfo%253Apmid%252F22689647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9XMS9XNDUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-web" id="cit-7.7.2171.62"><div class="cit-metadata unstructured">Storey, J. D., A. J. Bass, A. Dabney, and D. Robinson, 2015 qvalue: Q-value Estimation for False Discovery Rate Control. Available at: <a href="http://github.com/jdstorey/qvalue">http://github.com/jdstorey/qvalue</a>. Accessed: June 1, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Evolutionary relationship of DNA sequences in finite populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D105%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D460%26rft.atitle%253DEVOLUTIONARY%2BRELATIONSHIP%2BOF%2BDNA%2BSEQUENCES%2BIN%2BFINITE%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F6628982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA1LzIvNDM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.65" data-doi="10.1186/1471-2164-13-586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tortereau</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Servin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frantz</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high density recombination map of the pig reveals a correlation between sex-specific recombination and GC content.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DTortereau%26rft.auinit1%253DF.%26rft.volume%253D13%26rft.spage%253D586%26rft.epage%253D586%26rft.atitle%253DA%2Bhigh%2Bdensity%2Brecombination%2Bmap%2Bof%2Bthe%2Bpig%2Breveals%2Ba%2Bcorrelation%2Bbetween%2Bsex-specific%2Brecombination%2Band%2BGC%2Bcontent.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-586%26rft_id%253Dinfo%253Apmid%252F23152986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-586&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23152986&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.66" data-doi="10.2307/2408641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-statistics for the analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.67" data-doi="10.1038/nrg2884"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysing biological pathways in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">843</span>–<span class="cit-lpage">854</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D843%26rft.epage%253D854%26rft.atitle%253DAnalysing%2Bbiological%2Bpathways%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2884%26rft_id%253Dinfo%253Apmid%252F21085203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2884&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085203&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284359000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2171.68" data-doi="10.1073/pnas.1501711112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeder</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Core questions in domestication research.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">3191</span>–<span class="cit-lpage">3198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1501711112%26rft_id%253Dinfo%253Apmid%252F25713127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzExLzMxOTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/7/2171.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2161" title="Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2185" title="Assessment of Gene Flow Between Gossypium hirsutum and G. herbaceum: Evidence of Unreduced Gametes in the Diploid Progenitor" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/7"><span>Volume 7 Issue 7, July 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/7.cover-source.jpg?itok=eAMfPilr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2171" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3w35NH70JYWKHaQjBYXYEa1-FkX24KpB_I4yMVuUiPc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/7/2171" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/7/2171" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/7/2171" class="active" data-icon-position="" data-hide-link-title="0">A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485295" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-FcFqAwe9V2yhBr6MaXQ2VjtIuUSGTOoGvkBeWGpAxTs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2171.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/7/2171" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-IQPO86mqaw7gzXp41-fkMw3yLZS9wFKz19Q9ZXDqxuY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/7/2171" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2171.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485295" id="node485295--2" data-pisa="ggg;7/7/2171" data-pisa-master="ggg;g3.117.042671" data-apath="/ggg/7/7/2171.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6049-256X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jordi Leno-Colorado</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3549-9396" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nick J. Hudson</span>, <span class="highwire-citation-author" data-delta="2">Antonio Reverter</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Miguel Pérez-Enciso</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2171-2184;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042671 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.042671" data-atom-uri="/ggg/7/7/2171.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485295--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jordi Leno-Colorado </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), Consejo Superior de Investigaciones Científicas-Instituto Recerca i Tecnologia Agroalimentaries-Universidad Autonoma de Barcelona-Universidad de Barcelona (CSIC-IRTA-UAB-UB) Consortium, 08193 Bellaterra, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordi%2BLeno-Colorado%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leno-Colorado%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJordi%2BLeno-Colorado%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6049-256X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jordi Leno-Colorado</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nick J. Hudson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Agriculture and Food Science, University of Queensland, Brisbane 4343, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nick%2BJ.%2BHudson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hudson%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANick%2BJ.%2BHudson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3549-9396" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nick J. Hudson</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Antonio Reverter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Commonwealth Scientific and Industrial Research Organisation (CSIRO) Agriculture and Food, St. Lucia, Brisbane, Queensland 4067, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BReverter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reverter%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonio%2BReverter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Miguel Pérez-Enciso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), Consejo Superior de Investigaciones Científicas-Instituto Recerca i Tecnologia Agroalimentaries-Universidad Autonoma de Barcelona-Universidad de Barcelona (CSIC-IRTA-UAB-UB) Consortium, 08193 Bellaterra, Spain</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona 08010, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BP%C3%A9rez-Enciso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Enciso%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiguel%2BP%25C3%25A9rez-Enciso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Miguel Pérez-Enciso</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:miguel.perez@uab.es" class="" data-icon-position="" data-hide-link-title="0">miguel.perez@uab.es</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/7/2171" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/7/2171&amp;title=A%20Pathway-Centered%20Analysis%20of%20Pig%20Domestication%20and%20Breeding%20in%20Eurasia" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/7/2171&amp;title=A%20Pathway-Centered%20Analysis%20of%20Pig%20Domestication%20and%20Breeding%20in%20Eurasia" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/7/2171&amp;title=A%20Pathway-Centered%20Analysis%20of%20Pig%20Domestication%20and%20Breeding%20in%20Eurasia" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/7/2171&amp;text=A%20Pathway-Centered%20Analysis%20of%20Pig%20Domestication%20and%20Breeding%20in%20Eurasia" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/7/2171&amp;title=A%20Pathway-Centered%20Analysis%20of%20Pig%20Domestication%20and%20Breeding%20in%20Eurasia" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/7/2171&amp;t=A%20Pathway-Centered%20Analysis%20of%20Pig%20Domestication%20and%20Breeding%20in%20Eurasia" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/7/2171&amp;title=A%20Pathway-Centered%20Analysis%20of%20Pig%20Domestication%20and%20Breeding%20in%20Eurasia" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/7/2171&amp;title=A%20Pathway-Centered%20Analysis%20of%20Pig%20Domestication%20and%20Breeding%20in%20Eurasia" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485295/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485295" id="node485295--3" data-pisa="ggg;7/7/2171" data-pisa-master="ggg;g3.117.042671" data-apath="/ggg/7/7/2171.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6049-256X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jordi Leno-Colorado</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3549-9396" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nick J. Hudson</span>, <span class="highwire-citation-author" data-delta="2">Antonio Reverter</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Miguel Pérez-Enciso</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2171-2184;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042671 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.042671" data-atom-uri="/ggg/7/7/2171.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485295--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jordi Leno-Colorado </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), Consejo Superior de Investigaciones Científicas-Instituto Recerca i Tecnologia Agroalimentaries-Universidad Autonoma de Barcelona-Universidad de Barcelona (CSIC-IRTA-UAB-UB) Consortium, 08193 Bellaterra, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordi%2BLeno-Colorado%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leno-Colorado%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJordi%2BLeno-Colorado%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6049-256X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jordi Leno-Colorado</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nick J. Hudson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Agriculture and Food Science, University of Queensland, Brisbane 4343, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nick%2BJ.%2BHudson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hudson%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANick%2BJ.%2BHudson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3549-9396" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nick J. Hudson</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Antonio Reverter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Commonwealth Scientific and Industrial Research Organisation (CSIRO) Agriculture and Food, St. Lucia, Brisbane, Queensland 4067, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BReverter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reverter%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonio%2BReverter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Miguel Pérez-Enciso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Research in Agricultural Genomics (CRAG), Consejo Superior de Investigaciones Científicas-Instituto Recerca i Tecnologia Agroalimentaries-Universidad Autonoma de Barcelona-Universidad de Barcelona (CSIC-IRTA-UAB-UB) Consortium, 08193 Bellaterra, Spain</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona 08010, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BP%C3%A9rez-Enciso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Enciso%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiguel%2BP%25C3%25A9rez-Enciso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3524-995X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Miguel Pérez-Enciso</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:miguel.perez@uab.es" class="" data-icon-position="" data-hide-link-title="0">miguel.perez@uab.es</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2171&amp;rft.epage=2184&amp;rft.atitle=A%20Pathway-Centered%20Analysis%20of%20Pig%20Domestication%20and%20Breeding%20in%20Eurasia&amp;rft.volume=7&amp;rft.issue=7&amp;rft.date=2017-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Leno-Colorado%2C+Jordi&amp;rft.au=Hudson%2C+Nick+J.&amp;rft.au=Reverter%2C+Antonio&amp;rft.au=P%C3%A9rez-Enciso%2C+Miguel"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485295/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485295/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485295/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485295/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485295/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485295/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485295/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485295/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485295/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485295/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485295/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485295/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/7/2171" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/7/2171.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/7/2171.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/7/2171.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/7/2171.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
