<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/7/2081" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/7/2081.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/7/2081.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/7/2081.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/7/2081/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/7/2081.atom" />
<meta name="HW.pisa" content="ggg;7/7/2081" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Validated Bayesian Differentiation of Causative and Passenger Mutations" />
<meta name="DC.Identifier" content="10.1534/g3.117.039016" />
<meta name="DC.Date" content="2017-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Cross et al." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="In many contexts, the problem arises of determining which of many candidate mutations is the most likely to be causative for some phenotype. It is desirable to have a way to evaluate this probability that relies as little as possible on previous knowledge, to avoid bias against discovering new genes or functions. We have isolated mutants with blocked cell cycle progression in Chlamydomonas and determined mutant genome sequences. Due to the intensity of UV mutagenesis required for efficient mutant collection, the mutants contain multiple mutations altering coding sequence. To provide a quantitative estimate of probability that each individual mutation in a given mutant is the causative one, we developed a Bayesian approach. The approach employs four independent indicators: sequence conservation of the mutated coding sequence with Arabidopsis ; severity of the mutation relative to Chlamydomonas wild-type based on Blosum62 scores; meiotic mapping information for location of the causative mutation relative to known molecular markers; and, for a subset of mutants, the transcriptional profile of the candidate wild-type genes through the mitotic cell cycle. These indicators are statistically independent, and so can be combined quantitatively into a single probability calculation. We validate this calculation: recently isolated mutations that were not in the training set for developing the indicators, with high calculated probability of causality, are confirmed in every case by additional genetic data to indeed be causative. Analysis of “best reciprocal BLAST” (BRB) relationships among Chlamydomonas and other eukaryotes indicate that the temperature sensitive-lethal (Ts-lethal) mutants that our procedure recovers are highly enriched for fundamental cell-essential functions conserved broadly across plants and other eukaryotes, accounting for the high information content of sequence alignment to Arabidopsis ." />
<meta name="DC.Contributor" content="Frederick R. Cross" />
<meta name="DC.Contributor" content="Michal Breker" />
<meta name="DC.Contributor" content="Kristi Lieberman" />
<meta name="article:published_time" content="2017-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Validated Bayesian Differentiation of Causative and Passenger Mutations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In many contexts, the problem arises of determining which of many candidate mutations is the most likely to be causative for some phenotype. It is desirable to have a way to evaluate this probability that relies as little as possible on previous knowledge, to avoid bias against discovering new genes or functions. We have isolated mutants with blocked cell cycle progression in &lt;i&gt;Chlamydomonas&lt;/i&gt; and determined mutant genome sequences. Due to the intensity of UV mutagenesis required for efficient mutant collection, the mutants contain multiple mutations altering coding sequence. To provide a quantitative estimate of probability that each individual mutation in a given mutant is the causative one, we developed a Bayesian approach. The approach employs four independent indicators: sequence conservation of the mutated coding sequence with &lt;i&gt;Arabidopsis&lt;/i&gt;; severity of the mutation relative to &lt;i&gt;Chlamydomonas&lt;/i&gt; wild-type based on Blosum62 scores; meiotic mapping information for location of the causative mutation relative to known molecular markers; and, for a subset of mutants, the transcriptional profile of the candidate wild-type genes through the mitotic cell cycle. These indicators are statistically independent, and so can be combined quantitatively into a single probability calculation. We validate this calculation: recently isolated mutations that were not in the training set for developing the indicators, with high calculated probability of causality, are confirmed in every case by additional genetic data to indeed be causative. Analysis of “best reciprocal BLAST” (BRB) relationships among &lt;i&gt;Chlamydomonas&lt;/i&gt; and other eukaryotes indicate that the temperature sensitive-lethal (Ts-lethal) mutants that our procedure recovers are highly enriched for fundamental cell-essential functions conserved broadly across plants and other eukaryotes, accounting for the high information content of sequence alignment to &lt;i&gt;Arabidopsis&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/07/01" />
<meta name="citation_mjid" content="ggg;7/7/2081" />
<meta name="citation_id" content="7/7/2081" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/7/2081" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/7/2081.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/7/2081.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/7/2081.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.039016" />
<meta name="citation_pmid" content="28526731" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2081" />
<meta name="citation_lastpage" content="2094" />
<meta name="citation_author" content="Frederick R. Cross" />
<meta name="citation_author_institution" content="The Rockefeller University, New York, 10065" />
<meta name="citation_author_email" content="fcross@mail.rockefeller.edu" />
<meta name="citation_author" content="Michal Breker" />
<meta name="citation_author_institution" content="The Rockefeller University, New York, 10065" />
<meta name="citation_author" content="Kristi Lieberman" />
<meta name="citation_author_institution" content="The Rockefeller University, New York, 10065" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=K. L. Adams;citation_author=J. F. Wendel;citation_title=Polyploidy and genome evolution in plants.;citation_pages=135-141;citation_volume=8;citation_year=2005;citation_issue=2;citation_pmid=15752992;citation_doi=10.1016/j.pbi.2005.01.001" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=I. K. Blaby;citation_author=C. E. Blaby-Haas;citation_author=N. Tourasse;citation_author=E. F. Hom;citation_author=D. Lopez;citation_title=The Chlamydomonas genome project: a decade on.;citation_pages=672-680;citation_volume=19;citation_year=2014;citation_issue=10;citation_pmid=24950814" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Botstein;citation_author=G. R. Fink;citation_title=Yeast: An Experimental Organism for 21st Century Biology;citation_pages=695-704;citation_volume=189;citation_year=2011;citation_issue=3;citation_pmid=22084421;citation_doi=10.1534/genetics.111.130765" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_title=High-throughput robotically assisted isolation of temperature-sensitive lethal mutants in Chlamydomonas reinhardtii.;citation_author=M. Breker;citation_author=K. Lieberman;citation_author=F. Tulin;citation_author=F. R. Cross;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=F. R. Cross;citation_title=Tying down loose ends in the Chlamydomonas genome: functional significance of abundant upstream open reading frames.;citation_pages=435-446;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=F. R. Cross;citation_author=N. E. Buchler;citation_author=J. M. Skotheim;citation_title=Evolution of networks and sequences in eukaryotic cell cycle control;citation_pages=3532-3544;citation_volume=366;citation_year=2011;citation_issue=1584;citation_pmid=22084380;citation_doi=10.1098/rstb.2011.0078" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Z. L. Gu;citation_author=L. M. Steinmetz;citation_author=X. Gu;citation_author=C. Scharfe;citation_author=R. W. Davis;citation_title=Role of duplicate genes in genetic robustness against null mutations.;citation_pages=63-66;citation_volume=421;citation_year=2003;citation_issue=6918;citation_pmid=12511954;citation_doi=10.1038/nature01198" />
<meta name="citation_reference" content="citation_journal_title=Proteins;citation_journal_abbrev=Proteins;citation_author=S. Henikoff;citation_author=J. G. Henikoff;citation_title=Performance evaluation of amino acid substitution matrices.;citation_pages=49-61;citation_volume=17;citation_year=1993;citation_issue=1;citation_pmid=8234244;citation_doi=10.1002/prot.340170108" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=E. M. Medina;citation_author=J. J. Turner;citation_author=R. Gordân;citation_author=J. M. Skotheim;citation_author=N. E. Buchler;citation_title=Punctuated evolution and transitional hybrid network in an ancestral cell cycle of fungi.;citation_pages=e09492;citation_volume=5;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. S. Merchant;citation_author=S. E. Prochnik;citation_author=O. Vallon;citation_author=E. H. Harris;citation_author=S. J. Karpowicz;citation_title=The Chlamydomonas Genome Reveals the Evolution of Key Animal and Plant Functions;citation_pages=245-250;citation_volume=318;citation_year=2007;citation_issue=5848;citation_pmid=17932292;citation_doi=10.1126/science.1143609" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Onishi;citation_author=J. R. Pringle;citation_author=F. R. Cross;citation_title=Evidence that an unconventional actin can provide essential F-actin function and that a surveillance system monitors F-actin integrity in Chlamydomonas.;citation_volume=202;citation_year=2016;citation_pmid=26715672;citation_doi=10.1534/genetics.115.184663" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Remm;citation_author=C. E. Storm;citation_author=E. L. Sonnhammer;citation_title=Automatic clustering of orthologs and in-paralogs from pairwise species comparisons.;citation_pages=1041-1052;citation_volume=314;citation_year=2001;citation_issue=5;citation_pmid=11743721;citation_doi=10.1006/jmbi.2000.5197" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=I. B. Rogozin;citation_author=M. K. Basu;citation_author=M. Csürös;citation_author=E. V. Koonin;citation_title=Analysis of rare genomic changes does not support the unikont-bikont phylogeny and suggests cyanobacterial symbiosis as the point of primary radiation of eukaryotes.;citation_volume=1;citation_year=2009;citation_pmid=20333181;citation_doi=10.1093/gbe/evp011" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=F. Tulin;citation_author=F. R. Cross;citation_title=A microbial avenue to cell cycle control in the plant superkingdom.;citation_volume=26;citation_year=2014;citation_issue=10;citation_pmid=25336509;citation_doi=10.1105/tpc.114.129312" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=F. Tulin;citation_author=F. R. Cross;citation_title=Cyclin-dependent kinase regulation of diurnal transcription in Chlamydomonas.;citation_volume=27;citation_year=2015;citation_issue=10;citation_pmid=26475866;citation_doi=10.1105/tpc.15.00400" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=F. Tulin;citation_author=F. R. Cross;citation_title=Patching holes in the Chlamydomonas genome.;citation_pages=1899-1910;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Y. Yampolsky;citation_author=A. Stoltzfus;citation_title=The exchangeability of amino acids in proteins.;citation_volume=170;citation_year=2005;citation_pmid=15944362;citation_doi=10.1534/genetics.104.039107" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol;citation_author=H. S. Yoon;citation_author=J. D. Hackett;citation_author=C. Ciniglia;citation_author=G. Pinto;citation_author=D. A. Bhattacharya;citation_title=Molecular timeline for the origin of photosynthetic eukaryotes.;citation_volume=21;citation_year=2004;citation_issue=5;citation_pmid=14963099;citation_doi=10.1093/molbev/msh075" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. M. Zones;citation_author=I. K. Blaby;citation_author=S. S. Merchant;citation_author=J. G. Umen;citation_title=High-resolution profiling of a synchronized diurnal transcriptome from Chlamydomonas reinhardtii reveals continuous cell and metabolic differentiation.;citation_volume=27;citation_year=2015;citation_issue=10;citation_pmid=26432862;citation_doi=10.1105/tpc.15.00498" />
<meta name="twitter:title" content="Validated Bayesian Differentiation of Causative and Passenger Mutations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="twitter:description" content="In many contexts, the problem arises of determining which of many candidate mutations is the most likely to be causative for some phenotype. It is desirable to have a way to evaluate this probability that relies as little as possible on previous knowledge, to avoid bias against discovering new genes or functions. We have isolated mutants with blocked cell cycle progression in Chlamydomonas and determined mutant genome sequences. Due to the intensity of UV mutagenesis required for efficient mutant collection, the mutants contain multiple mutations altering coding sequence. To provide a quantitative estimate of probability that each individual mutation in a given mutant is the causative one, we developed a Bayesian approach. The approach employs four independent indicators: sequence conservation of the mutated coding sequence with Arabidopsis ; severity of the mutation relative to Chlamydomonas wild-type based on Blosum62 scores; meiotic mapping information for location of the causative mutation relative to known molecular markers; and, for a subset of mutants, the transcriptional profile of the candidate wild-type genes through the mitotic cell cycle. These indicators are statistically independent, and so can be combined quantitatively into a single probability calculation. We validate this calculation: recently isolated mutations that were not in the training set for developing the indicators, with high calculated probability of causality, are confirmed in every case by additional genetic data to indeed be causative. Analysis of “best reciprocal BLAST” (BRB) relationships among Chlamydomonas and other eukaryotes indicate that the temperature sensitive-lethal (Ts-lethal) mutants that our procedure recovers are highly enriched for fundamental cell-essential functions conserved broadly across plants and other eukaryotes, accounting for the high information content of sequence alignment to Arabidopsis ." />
<meta name="og-title" property="og:title" content="Validated Bayesian Differentiation of Causative and Passenger Mutations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/7/2081" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In many contexts, the problem arises of determining which of many candidate mutations is the most likely to be causative for some phenotype. It is desirable to have a way to evaluate this probability that relies as little as possible on previous knowledge, to avoid bias against discovering new genes or functions. We have isolated mutants with blocked cell cycle progression in Chlamydomonas and determined mutant genome sequences. Due to the intensity of UV mutagenesis required for efficient mutant collection, the mutants contain multiple mutations altering coding sequence. To provide a quantitative estimate of probability that each individual mutation in a given mutant is the causative one, we developed a Bayesian approach. The approach employs four independent indicators: sequence conservation of the mutated coding sequence with Arabidopsis ; severity of the mutation relative to Chlamydomonas wild-type based on Blosum62 scores; meiotic mapping information for location of the causative mutation relative to known molecular markers; and, for a subset of mutants, the transcriptional profile of the candidate wild-type genes through the mitotic cell cycle. These indicators are statistically independent, and so can be combined quantitatively into a single probability calculation. We validate this calculation: recently isolated mutations that were not in the training set for developing the indicators, with high calculated probability of causality, are confirmed in every case by additional genetic data to indeed be causative. Analysis of “best reciprocal BLAST” (BRB) relationships among Chlamydomonas and other eukaryotes indicate that the temperature sensitive-lethal (Ts-lethal) mutants that our procedure recovers are highly enriched for fundamental cell-essential functions conserved broadly across plants and other eukaryotes, accounting for the high information content of sequence alignment to Arabidopsis ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
    <title>Validated Bayesian Differentiation of Causative and Passenger Mutations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4KZF3Z5zQFtxolB-UY7Jo_ZU1SoywdoDU4G8aIS9AOI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485298","apath":"\/ggg\/7\/7\/2081.atom","pisa":"ggg;7\/7\/2081","ac":{"\/ggg\/7\/7\/2081.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/7\/2081.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2081"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2081"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/7\/2081.figures-only":true,"\/content\/7\/7\/2081":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ndVPeJRqjMvT7R4ESJRkBPvLQnTFSjs4SaecM3KM+fEtTt+l74TFrMwi1TjNj0wq3gE7dxODGI1Ix\\\\\\\/1NRUMDbvCGduZ4v9M\\\\\\\/8+eXGgd37OWnv5EzDV9GTIcJIJtgteMiBEpfn8Epn0Yd0JxnF7RNVl0zTzVlfn+6uNG8zOA+HY3qfc6E+3jYChzLRr4cQGmdlWcFq8FmwxNijVWCOTMzc\\\\\\\/0EaGZ6VgIHqcuT\\\\\\\/jKVKUUMMWZyf4wnUAFCGVpzn+vmOkkMdpvuZrkU3odKI9GLEESL+WPLOrUFlDErOX9X7Z\\\\\\\/1AHDd5VWcgFPokquA7wLVl9SK7Lz\\\\\\\/4omMsLYGa8Ae2g0CRRp8y4GshOHTtrS2n1aZbzu88mhUau89+n\\\\\\\/RKf5qymTT3hcGtBznjU\\\\\\\/eYpw+1rdyS\\\\\\\/AhC944nSMHaMoxmmz2r5bHStuzNeQ7hnaAbadaI0ONLpQHnNSWMMCGzpbYOhjQvU5RowTmwidHxIMPsUFMbYZu9L9Rt9wdRjTgDXN\\\\\\\/AEupXRnuVoT+qjWXNw0FkZuGC6W6nY3opHSGA+zn3CjAY6BgLwdz47Vvf+CmWJ3gkBSumMrkVYsW0uDHyesIH3cRmTbNOI1L9esMfw7bpa+kL5xNwNx1xasalFZ+uQXUa9wZScgWb5wP2Os\\\\\\\/i94QLmUFlM8yq7lhrK+jC399tfRejuuI7hL82Bts9Wzj2cL3tKWHx7326kKZB+mMtA7lwN6xlxI3dkgI3AZnUZS3UjxRx4XgBoyUmdtbuSd8VB1DqKzFq+VpM4Mc7K1rTqic\\\\\\\/3U\\\\\\\/\\\\\\\/DYLJIem3XIB+cmeObrbP0Irg579M1u9b0hw3TOpL7dy67CMXank4\\\\\\\/gdAeQLeLAHJX0QQ5hzMJdHh8XAzi8glZPkEEPOpM5p0EUeuWVT6pcqm7M7TUxcMLQmv0mULFI09cgCMumEJurlPXftqcqPRPylqtsgnmB9R4q0r87RqA8q6TblLVdgZphYwPDnEsJfq5qSFv7WzMPQlCKvIvF7EGWBTJmKzN7LVsmuOGBIGQNc1VlSi15obw7lTnf3FhTuBND7oQii1pdONlIaXlv0VXxStfK8ls+h8Qaj3JTst1ErXekqt1tqx\\\\\\\/Pq27xeI4YjPKoIJO2R7D52ypkBdqka2H80Bzo1xyPCUnaHgO0qoFSwYfPdTIrjxzK9mS1HArTkP52FALujzEkxO4u0nSFcnD5gC9NfNSJvI4kdYJUCcqDFpoedLKHsauqaxLD12rbr6A9saN+t7mb2Lxi\\\\\\\/e3WmgynE4rmnff3QUgSuOqla\\\u0022,\\\u0022iv\\\u0022:\\\u0022mT3gj4ibH8e3tK1XnwXUQg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002294a61cdb2b61b96a1b6c5c3ea6a15ffe905bfad2596c183570bcadbad787a240\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ihtlCDKNajNAHPjCvb+m+LKVdb7EqZRreim2SssVhoCNTNSwYbauTAnbQ5eMj6S6Z+rhQcVMpEgl2F5JSn4XmxbJWKEbfBVyquNfpTcZYmPpjoVS0OPdIEFleUNhL9bePGQOw1vyCsRu3L5IRddXcLgCPX9zK\\\\\\\/xDq30YEl1Wz3lTg5B2ZUba88cmS8xs8gqs0pq3Vq6CIvLcKP+lFX+8YUsl1IlgyZ5Mfye8hpXWGOvEuaChjnoPT76N5Dot3kMSLaPpLwQB0+gimyb6m52w3tHcwVTE6NktUCjBq0GsoUzNEI5GmcImSxh2q5sZQ9\\\\\\\/lmONZqNhsHh9gcC4xQKehPEik1UgtAgj7o9a\\\\\\\/cvC2Ut9ivjBI4UB79edW\\\\\\\/x7Z5yde0t3w4Y1\\\\\\\/bjOlxlarGD5La3Y5yNnfYwodCkkAKvydHm9RdDDpTJ3BVoT5CLjxaRMcKPbrOGnWBLWH0IrZb\\\\\\\/he+FVyEEoXctA4efQtxYv0NaxofP\\\\\\\/SyPDLHJro5VLTmO0WmKnXpyZ5tlEm\\\\\\\/GBOW8lc3X610eyihZs70DY1+0ECrwkJ1heeP39oPKcrPI9j5AvUErrQ3f75AiGe67qjH7IUXNZvqM40joW88DAjkQb65fLAznJ+Tbet2r4W\\\\\\\/6PP0UzkHLE99+825XWuAkobv6Y4oeZOaoojacaobgdFghvKhMiViLWP9IdNJgioiGpSLrhfEkjyD424Yyq\\\\\\\/hd\\\\\\\/J0HM\\\\\\\/q7iT6MVpIAj9uQmE0P8J21Y7pDqtSPV6zsIpaeRsi3g6wvD3syZJUCCtlQ\\\\\\\/09rx7\\\\\\\/IAKPd9In86YtPDzLEFXWKblxipPk8xlrG\\\\\\\/hEfmKBOW+co24elRNld0QlCJQ71AMe6CwdLZnPTrwS5kxKmskJttzofpk+qR\\\\\\\/ySocAjFBURtV8ZIIhGaumU0qfoIjN1GN6sPxc71fp7FYPdtoxKwyoLUOce2BXk23D5c2GcAq4BIm3hOxrS8ZD7LT53\\\\\\\/aGdOJI600FDtRtvSGXWqisTbmDFghHKIqlpk0PKq6koXh+2CYvY12foSiOSdsy18NL5FzJqtZMVy8yRUKE0YBi7suFHeBifx\\\\\\\/Ln7VSkZECS0+csha5uazw3lOG4VGOsOtSL7nH77yQYHGJbo7GpqMpCKUcfZcBi91UZbE9ZVxRMhgnKBBjH6oPQwDuiO1ipLXJcaT8yMf3phI4Tl1sio6AFqJO2bplgHVJ4rlMXaLIapyWq9Bl1Fa0E6OyVwmnXHcx7MwmSLe4F6495cCXU+XD+WsQIVUrmjAUk4FzO4AqAiPd0VOipSd4NYJIb4\\\\\\\/97I38kAa\\\\\\\/Gppr+qoSrhppgM1E3aYVRw4VvRsyMZTyDJKeNWU7C46EaqeWtjXEQ\\\\\\\/BqfAFnszvrAksWzMwLWAyu\\\\\\\/bF6NqW904EK66KMUr5Nvt2io+isZhBHQpYlkK6xDtZD+dJaBH7bPl3ABsS8BbcazAGgnLyEg8FciWFMQD2DW0ahLzopGVnxSpsTb9W2A5qGSGTsEXwh0W8Vox3ATaa7599LJpRiUYs9kp4hRPch2aQtq+pqo5cKYBvyGyZ0UmQxCzIVBc5U7INNW\\\\\\\/ow+ZsO+sK+UjsPwqdlMOeAL4oziPID+lB7Jli5dTJ63ACl7dMWvkjVshKOpFowwY9eHuIaIKKnY0yeMYDBZBPlxm2IjkAubdulBl7cTjPvzTpyrDE3\\\\\\\/brD4qIUJvAGX5DkEyhC2jx7BdOesvEW1outKx1RWovFYgMrij3+3EGpDkrlQQCF8iL\\\\\\\/aIxfLldwpUQGuY2KNnoXIIYNtEBk\\\\\\\/vigLuw3WB3IDe1r+y09u4jfbZUTEEuWTWZ20t3o43220ZevRWyk\\\\\\\/cCuM6hh7Q4u3YnRSieOlWT+3KiaNjAmNFNaSdWK9E3xGi1Z8\\\\\\\/3ZgW8BRb8qnaTbN4l1psjP+FhYUsADCqHuwiRdRVjy6aBkFeizfyIFZeuFq5RkIHcjYY6Ekdchlcp\\\\\\\/6PtVr7cTnM4FBwHMC1CvZ48u+vTyHjJTwgAwzn8evU8ITYncE5+jwOg1Dkpn6CA2qbBUsXNWMxi2uu8n6+CULfr2f42Se\\\\\\\/v4ht6qzNtMNNSzpaMRmEthl9zd6bdbCfTkX2nguIorm\\\\\\\/uCNTAr7nQWHMV+1DNQDrDHxxUOODzwzUUUrz6QZ8WDUgLqkHleF4VIQIMSh\\\\\\\/tPbTvrIGJ9eUDy3Wv9Mge1hrBrnF\\\\\\\/Ng+18\\\\\\\/k\\\\\\\/ywNVH3QSLV16N1IwpU6ohzRANUJHz1oEspjI5x2iUjeQ7xDikcVZ1pnJT8NnFBO0LGHTOCzx5nj97xhM\\\\\\\/3MDHPXE+6ooiXMzx\\\\\\\/T2fZq3guoq\\\\\\\/gCmogxE2Wo8WOYdG1Y7LZs8M0KdXKBZK1ldRwDTa+0lt0h3YUNjI+X\\\\\\\/KE8N3xrtsR4Y9NVetW5W99oown4w1LTOhXDLa6gJDtnajl9eKHUjW\\\\\\\/awBwGs9rIO\\\\\\\/WtaXnc0cfnQ9rAK4F4DjisLO8FguPXOt6JpFm2QMqMwk0yQPxSRk9eGykc1jNeNMUyrWo\\\\\\\/O\\\\\\\/4cE9kli\\\\\\\/D66WEIPZMW2uQsqLLobdZVh8Ane1ubBqke\\\\\\\/MTmNA0zq+fONAjFP0UGjT1XiR4zu884jkkuEUuG3S8YmtUQ2ecrH\\\\\\\/oaccPZqrHOvfK9hIS3tbYYzlN54Pw8u5iz\\\\\\\/SjRRZkkbGFpwMmUwgNKsgfjD67AzxLiTbhVBNwtBjXLlF5EYEDwpzR4THsTsdDmkv0Ae2djN0e5YsJjU9Vk45r321aJw2tD3jkysyOupJ7DbusNMuuceklT9G4Z+zlKxmS39H6+BYIWtevY+\\\\\\\/t8zRHhu7nDqyLojc+sTeIpRNdjc6Lm6Vji\\\\\\\/tN\\\\\\\/t5TIxpUqjB3WbUPgNQcVHINiOFfwe+CcStTHnqs4Bi8SSPRK5MmJpujWzS\\\\\\\/nhFHrCvh\\\\\\\/yUdos7tX6B6NFOqJjkLPzZ5nftfXY0CXc\\\\\\\/EYxPhQwtTfsp18CgL6CuMm3VzN0+pVX0S5RPDfjVs2dKQVYxDlaifgpH88P7OH2JuAuqga+VUBoT226uSzxgu16qYTme1\\\\\\\/nsubUoe31ynzDpO4Lul7ulY8gVJPn1W1MYxn58ouslIQ8yBkb2Ek1BKL4U3dg9nPbhGq5I2UWxxvfjAh7LoXItRcKnwLDpt3FTm1nQpjs2Le8tUn\\\\\\\/evx\\\\\\\/pheB8jvO7qyhY7gGh5F0MRW1G+PT4Farw4BmvUAOiu7fOw6aLIdQgjAxl3nYnuBPXwAnkA8TSzHuHXp4SjRrJ3msCj6CKuZf4a8mMyMwzJnd\\\\\\\/pK3lY2K1\\\\\\\/ofBsExy5D\\\\\\\/mu4KwmO+33EbfyL8Xb5IUvSxUcUX8W9Zc8\\\\\\\/ZMIWWEGbPYIi8zvMcYfH+50KDE7L8gcF5iSU70ibgPn\\\\\\\/K\\\\\\\/27ExQWTCDd7Z612PwyyELm+K9LR3db1jnZfrY1A1nhUzPHgKdjOsUos1Wq7nDQXPzFSKaN6Avb3+QoHie0aS64EHQsj0yShLJrBBjpFroGcBxKliT2uQMih2rGaEMXG8OiC6x1NVXHh+hunbEA5y3+VKkAF6IfXzC8QO3zBnxM8kCSwjBRvFqFV9iBgehO6Gdj7P1hwXjrn1mYmjeoDIyOFdjoNTMgMDBfxBGgVVq3eToHw\\\\\\\/WfcN+wMjV\\\\\\\/plJWuqF8uxCqKBDJwSsMEDPZds2vo\\\\\\\/x615qv2p5pzh3vTwV8np9tSa62+jjiC5TJb6KlUzV4Iwi2G9PGpY\\\\\\\/21nmOsBBJGfiw6t0A4ZeNr8d0hI4icbandoS8H5c2n8ueFKfuL60Pwd86Vx5V6Oera2zHBU2E5gv6jN6mKRnFe899x\\\\\\\/4G6Wu4Hgl1uvIvlGzPNrbtdIwiU\\\\\\\/y5GWiUSuYXa0ROu7K4+yERueDWgyhTXO5BA5ZiIfOeBxpWLsQu8OWtfuZhXzV0E24u5w+h9dNlFMlWM6kiZIL7YCIdqfT03VAUkgC\\\\\\\/4oQFwdAK5G62MAesR21TuX5k3GpI3f7ImLUaxxCsU9CmvBS3cOVeIi98Pb3hlDW8uO9m\\\\\\\/jco2HQDA8fK4frZ8IIb5TH5ZI6luq0LkX9n3t8Az8CEeMD7nkKML2oQDaWeqeg3YV48W2JHuo3gXeBqvsMYR3lH1+JG2reiTUdwC\\\\\\\/z\\\\\\\/2esTlMrRsk9gl+PatSbI2iSv76ggf6E8eG7gEaSTjTQHIV6rUXOxZqgqhjUla\\\\\\\/N+WL42si47zBMBCWgiorBDYBPJ25a5QyGmOY5ZSrMTc8M5zrWD+DAfAuv23DWy0II+FOUg8MtnWuqs7s7NmzFY0cQN8e6ie7v7MkXAKE5884x1\\\\\\\/f0N9OEggOwirFisW0cVJ7q\\\\\\\/dqi4ryzimiLnkTXvNnjI+wAASeWsE68z88oyqqQZY5yHbBHjpL3T8bthnsQqQhWbvpZlotLfLx3+Nc8j1DBSLCaNTuC0MkC3qA20JqTXvDq31ZTEKmKETAfP02s1pgUl6NLTd6IQa4kv3owtOVcil57O22JdaQj63SpJhLXzvsXrAjJz4iJTfa5biVFAvjKbHxyiU1WRHqb+jwoeS4ORYDV++R87WbllRPmJWc+GFOdSFdm7cOB0Dsv3z29mFZlqZ2IVqKiyKdEpV\\\\\\\/CbA6hIkcu3qetw+ckyL7ZqRaZ7xSopR+jGquPlkUaQb6GhO+lhm\\\\\\\/PrLKhf5\\\\\\\/cYREXa0\\\\\\\/byLT9eWc25Jau0cm\\\\\\\/bODLVXNUNZwoITHmDVAl5hmT77\\\\\\\/67TKSHcx7eGOCf4dJUaMwdmPLzjPV+JrpH+FOJA\\\\\\\/bzutFbl2sgCuP0qVDZvlkcVuwR8ElbaYPaTCWDXIqkHULyxhaQsQqvAkSWjMr0Nx2+qyFXXduHLHzVkfoM82HdIbkWj\\\\\\\/VDIn1r5U9jqzyYh18lfeBJJgeg9SSnspfsYxuetyHRjvVpbMm52iHfx2EZrANJD786fOYyCCzdEVSMn5FPnCjbUDttJi7iCrGrLi9MwGJg24cZllX+GJOm\\\\\\\/S1e0ubEox\\\\\\\/lHacPyswBLvSg5SXnlZ5g4Qnw9jCv+\\\\\\\/Wi06sOvjWR44hfdOdl+oftmD0QiuHREUrDjM7DvutYDqhDy2tWkCb1NQz+twqLt0fIIEEcJn60sx8jr4dMneTduWLHHNiAeg92vRzWH5RCogdZo9bofZ6wzcR4OSUSzrcvn2jLCGfbJcmq5eAxzSxgNShrAfgGxeKmACXUJMCjTYiDJ\\\\\\\/0nLpbn4se84UpcNwFWl4FNzAwiC7E4zhZmLY8vVGvkx4O8uyWLwQ1Tr5rIoXbYpcWRt7rqQPwJhs8hSOeaubPxGF1Byo0CaV2Q6Vcqw+GqPEv\\\\\\\/FZMM+z7yAb9Od5KOgyaSn06+0r6SaOCScljWrNA+RoxLjmGsyuN7oI3jBEpJrLtdbDC1IbN2PEGjQxxTm9btXlC+jeWV\\\\\\\/5BVS\\\\\\\/RE12jqIFaKlfrLMPL5qfyzBLXVRCAeApd66uDkcH38AVhnzJL7o5QUbOH0+5ZRYfRkJQWy+iMjnuM\\\\\\\/GIs6CkPXk789A8Bhfe97pKhwKAglJo465w9MqaVNzTM5uzna8p7\\\\\\\/Kgm7kn8m2robcIvYLSmJ43o+CFbA4uQM8HOkWYKrOBJR\\\\\\\/E\\\\\\\/KcBHdV8GFd+mpUDsgUKafnzQWktjsE++okrPFb3J8JX0vNZSncNXzDLKvMp0WbZ2+hSwDU7D8kKDxgNri\\\\\\\/83+SYaUHQ7Puzmq3+3KLPIgsdc\\\\\\\/7X1diETgJl2F4liP8qg7IpchRDUuOlPtaq+gDyG7mEn2SDi4PzwGQMaGJQtQWRKYqvniJpehXMizUSgZGg+jishrbb3EDAQcc2fqmRZLQMVaFfbr7zUGXc4cBCy78DZEJuoobtLxayBsBKXOTolgzJEtTzJ03mQLkVjQc5yrGlDro43CTdzQBRBAM+1\\\\\\\/1S0OxaDv0wDbSjMZP+6ZOhJzY4b0KaG6hoJMNuuu8YCB9Ad2M0GKdSqedB\\\\\\\/7nHMcguQf7+2bzdcuC+BNDQ4BhDGmbW5FjLBE1Bz0EV9yvM8344Fg+LWuJ8D712\\\\\\\/PHURD\\\\\\\/Nk\\\\\\\/fMjztYbvZGkj2b2KcENO81mYLE3tcSvvXK4w5Cbk2csF8ZLo5VA4J4HobR0+az\\\\\\\/rnZzmdZ3icvfXSnAbIoimE2OThTj92IwDz1zs1WIXXxz4lnUnZzW+1ORjGIEJfkZln1RgD8kyE5OYeU42SDgr9+JDbza2cOGl+LF04iTUg56\\\\\\\/eIfxKiB5CIuqJ\\\\\\\/6v7TslkBoFcqmLXXB0dug7iUyg8TIpqY\\\\\\\/QyZDfggqcKR+RPzLYRlQH7TnJwSfidAEvGqRx3ADn9fN+i86AwST9A43yuZpqARTg4xe+KQlEMhRGvbHSxp3MWiTYmlO2NKXNz9o77KqYiko7VNeimMZQFcw6fx9ZS3sUFlGYYqVLxLhgsDaA\\\\\\\/UPe0r0LEwyUujY7ao9xWWQ7BwDBHFaE7kitSwemOqHLNTDd12ycJoEQcqYJ37eAvKlkxzrUL5SMbobj3\\\\\\\/VJQLiTb79f66QqRHPahqquJHquH6J8eSHmOjWf7du1GmE5ALTQUYEyDaL\\\\\\\/cfNGUFvw8SoD3QveJjXabKuQFm+jG+r3WgfKC0OLqyaDOKgONtlz6oLh2vuGjv7u7jNm9ILCLX1QH1wvm2lVLHwoSi8dehNU7loV5wHjlGkLDsnl6fXbWxLWpI8qAHu5Qu1rllfw3LVHDh+qR2Fo46kbezD39h3mEem4qEKaRW2wifc2bJXqOE4J7+SSoCtAg6bc7SylHzNhrYy1zwrG+EDHkPKhXoIs6fjF8\\\\\\\/BRQaYzx\\\\\\\/lkCLgj57h8G1\\\\\\\/sb8LU1Pv5I0ukfHiyJ\\\\\\\/4AxR7yvF7BUXqV7YJqgKlDCI0yOBab4d6wNlcxOt9VdTAFu7JiuHngsQ\\\\\\\/uGRrLIJa4L+p2ByKRWgjlqzryD\\\\\\\/bRxhOJ+NhbKnWMFu5q\\\\\\\/Dc1lKKSR3JQl4c3UOm9UVnjrNosLdQAQ+poLssUP9wHRgEWiyP3K7b67ORYcerfN6za2720kLODUpQEUyVENTQNGxAyr9A67GonnYwazo9ipAIvzaeGsz9pP67vvzA0rwYNK+pZavEN2h4T6Yjgl8c4Jdar8pRcqU0D6blRKkFiqliYuLK9CoTT9GHei3oHJr2S+XOM8HHh6AuRHJRbsWXraeUWAlQzUGErxW2kZPEkAwr1Nkhtn39hMjfvARyCBvFw7gOeBKX8iRO38TpWV\\\\\\\/NkuonS+UHZqB26hh7Ml3wLF1OsjWUv7oRUUFA7kG2DdVVm3rtOaIVarYq41C+LLft2vUUzi42SRRm\\\\\\\/8vI5kkN4AtyWkm2xAJ1YPBYa2bfPDIbKNnxJLlmyFL09Ribdgwi92bqNxSX8qir1vk+Mb5AjHXyki+7x9\\\\\\\/7B+k4s40kO5l9LZ2m2aXYAO2IOI7pXj24JgNvw7948qoNibwW3T6HtKjSb6JE0YfB+Uj5zrKVFMnUFHwL3bIahgQxvMAMDbTCcmt5oR3QlkJxnbNGfqm4zn\\\\\\\/gXCkWE6GP2+1Kx0oaRpFYF57plzcdQ9n1y4h\\\\\\\/nlosyTGqCJJJ+KwHSRtnHSRvOokhvEhFYjfS2MuLwv2S2SkEbHqrZfriV7CbR+OEEIOqlzCIfYPUBNOAPY8dfWfN9seRjPS9NIpquheR1dFKOLNchO5uwFZqgloYyVLG3fnqYpiCsvoE0sJAdWivTH8Xt6M0XI4nOcoRNph+\\\\\\\/4WatAx5D67nKhw8xBzSt3tRkKgCS6fiopUB9Oc4xVozikH+gzAqEdpv0QVzVLIA0nRX+mABDzuCe9VC2+617t7wt2uiOEtZSejYC3VcaMFzFtBDvIMzXPD6elXKOsAdkhGQ+M\\\\\\\/mof8eJUFpMrp4uaEsvfHKsmzCvBF+y+HUcxsXw\\\\\\\/GsNaL\\\\\\\/GgB2umMj2xz7ba8PYIq3BVKdItQYq9YPrXaZP9ttB6LsGqPH09Er9\\\\\\\/T\\\\\\\/axXDTkHyE1r4WqMS7\\\\\\\/0Tt9K80yiP9TddhW6o5JttO3MBF4fQpjK6JwBu5JHFBNrR4VUsFrZmV9D4VGTZ0pfT2o5oUoy1\\\\\\\/nUrrRbplLeP5W4A6vIKB5HYT0H1JTHxCWUF93WeXfCJIJdJRQvfqFReBo4Qr6kaLPq76aN+zv0ecBP0Vhya8bpt6ZZEGilW4B0gz3L50WFH+zKRrgWW7SU4BsD+B3MqAgCavhLSMSzSvIMGGjQjq3V35awPluzQp6oOyaMmHijdMbsTw\\\\\\\/0C25AObbrKFX5X4XtmZ4R2JhHrnYn29vbe+HPgiebjrqch3XbsI5y6396OO3YTUAlsoq7\\\\\\\/57C0xduvPWp0cGq3ebkdVEqycA5hFnC6JgKc+uXnppUBx2OCmRE6cfyvO5jJBoEIuZaj+5X6L7n1+UNDIU0Fj13ghPmxFtReHGzpSOSasWSn3wy4gky7UWDaShHmo2tricW5g3TigDoDWut8TLbeoYCCOnaOy88Riunz\\\\\\\/j+8nXodNvFOprwJYfoPYrjw7MAd9xrfWUfIaJRPcqk7XzbuJXzugXX2bbN+UVWj7DHQcNT+jiUEhbh+7140dsoM0\\\\\\\/A3C\\\\\\\/eFO\\\\\\\/aJjmYOJq2nynrbZnL0wcrBhUikiUUQkmyq7cjgy68MXUDJ0mkI0VKW2mC37F5rAUDahm+oy2Bb3yQnvSx+Yet6xWE85ERbEPFYR1DDItw08MJzO0pXrMOB9zdH4b\\\\\\\/8GbLQ9v\\\\\\\/mchaTCXlWvt7iHKpDv6Q+dpPexFPnc8lvz3svdfeTFyoSZSOM7AvkhAHWSKaFtCgW5b4z6UvlwwkJvXiHrnb6Q9dLr4f7pDQm5WT1YBu1oS9SaspvuVCUlrdrDti2A1oPtImdLgBYODFPehNzJ9mVRu4bGSVMUaeVomqxKi2p5TzKNhg\\\\\\\/1flaBklcOvBHTpiM3OR3EVJ+pVvyeVm7P1AVK6eMdY1ue025w8C6CeSp5dKMsaOWOIGGLtJgK5i73dnNzc2H\\\\\\\/CJxGqmZO4IKjfjbXKiAmkFAYW\\\\\\\/NL\\\\\\\/dQ8688WVuwkuhV+\\\\\\\/ZEpnwDGBhtU5TKg4jnJu6HWmxryW941oJ1yQFN9dgbAPkDj3khbeBl90Pu8vK9VslsXLF9NUQd81cONdaFa9D1D5U0DoFtdcXMlssYezPamw8tCvLM7kO85czCjaFBpPj+ZhI9ZqfcR\\\\\\\/0OK97oiWUm+ssPFakZr6dNlnIy7RwDcH9DltKcFxFncORAP+FsfHcE0ZR1QaKsl13K+p51zepU+0vKp3TGlpZ2qxXH5m9qRor45m1V8gfGLkgQrw1BdRDn2KAOwEmqkUY9ru0L8RqMZBltLv0+YPQlY71t96iY2ra9kdSU+xFpFREtHTkpGULtGxBHseQwEXRIeaSYaYWmasxb2xvTEdCdu94LiNDVnRmGme6IkK5OvGvaPoa1746z8M+4aX579eislwFDBURl3FqElnQEvE6cyz6T4zkBjbudwMqZWILsOhl6jdY0jbe307IwATXuX6YCHDyB8oEhumr68kr2cLq8wtTBUWIKHNgWDc0ZGVaaN+SGJOdkfDX49SrcaIDbbGI4\\\\\\\/TqeWRbwt6dvqM9l4gpUdv9Az6DTgQfTqwIpP1i\\\\\\\/7sJwqyS9ee1vojD743uD+41oEuOFtSPl5IZBYBFc+LjS7y4eImJLTHSFa5KlFqRPVnJreUNG6XALikCHFM7+GDkxLAEvb5l0M+v\\\\\\\/X0g7ZZg+RRBRG9lRbrVzh\\\\\\\/cXF3mPYmEPaXKEPBbt+EGIDV\\\\\\\/S5hFHPjNj8gpP2hNkLr7yv1eNJvAii0TSaPuhSS0t8tp09JnkmAjqjARM4muE08JMLprgcBwGHir0guU6Rl+LEx6an5xGCarCF0+FP+Ybs+fSN3bCNLjeugBH+P6p80oHGsGXb6uvwMv8p05mMmuSwYre6Y5alheWJOBRbxFhmqWHPkLkbFKdbDfTokSh8\\\\\\\/1ClxoKiauFr9zqLjVSKDYQX7cMKzHjTReS02hrOijkGuVDxIt\\\\\\\/uiQVinGwTvLGc1cFoVyWvCc3X2zQwt+o67g5sfAQYoSFXY3fLZHG5bwjSLfYHEA77Aelska8e+XyRSRQdNAxx0xZi5AAYIhhME7+FpFlPD6q49BxzOhFUVIVeENl6Q7avbYdkMcTZtkjCmWWhfE28LcF0uOAX6pilGACtHHOoFVAEExOAd+KRJHHyF796u\\\\\\\/Zk\\\\\\\/IMCxf1d\\\\\\\/wj+MCfBu\\\\\\\/Me3jY5mV5OJ0kJb4UE9MsqDaGdhhxwS76u8eg4bbFsYjU9X6EUNlhydTGCnwGafk+btcImK9qI5GPmw8caYAzQIwag78Oy8dIcoSvDV3EHpBRLJdh4K\\\\\\\/535thdIbLUh8fnIMigzkrgEwM8PgcXJyzUMOMCyJFSu3Xd9EAbanV1G\\\\\\\/15jZ0GaVgaLEJrKdW3zeqJ57zSLSnlpyOImnd1GXcrabV\\\\\\\/1QUhG+Rq9nuxuouWMy\\\\\\\/ugTUFVrubzN84Eap5kloec2sHSt0UoEktNi4DZ2EvM4Br4v+hp\\\\\\\/ERw\\\\\\\/oHWR4hzTL0Rps2RWn8MltJZFEnVN28XgY5GESqvdGvJyzzK3EzA2GEMk7TWirzdNtVpX2\\\\\\\/8iofNfxcXAEZeoGtY7qCdEWKNZD1nW1KvmtuSQo0GbLTUxUXz+LHZO4FH52zu4ZxoqO4oG8hwcyhF0q1Hl2bSEeZAT7vnbaiZicqD5V0plb2X4m5chcIGHZ3VHt0pLz3S+4WZivpr6QUdargvi8KynumRMEPd+BzOUTJ7Cpa12acs4ZxeV0S+ujzdgQXkZE35W1Y1ub6P2QxVIEamCR7ceRBZGGLHqUHdmZGcJaLpWJNUyeYEAPMzq\\\\\\\/pNZ4sh\\\\\\\/JqasEwmcKjPm8HGfiwaJ4NREIOR9DVFuvQN7gDKwV9jEyNbHrTln01q220pZ9EGQik284FuFsA+VCXCurcls9t5xUrHbEdefLkWKsXryQNWIB7+bpj6RQ1fqg3QAlMZzKj7c0c9E1DnPRcIymUlU66u50reYJDN3eTkwDNNzVg2+bErtsr\\\\\\\/pu6bB18QFsMr98L7BbL6XMaOKSItz0WpyP3gd2SxaE4F+dOhNF3ssqbXUKr3cx0ob0o+y5BopQvFOZVKsRAL2UtlGY3f4Eys476eTz5GMfaPFgoVOmouc9FXfLhG1TzS7e8S2eBGUhdy00uu8veEqNU0O9pBL0JEnDDZi+e\\\\\\\/095YH5+dLbYMo6gu9Z+ik52xkO5\\\\\\\/oJmncaJtDyCOpzDzkQL\\\\\\\/XKjIIorvqSDuHHOu61xcGw2gU9zL4bWkTKO2nfqc3VWw9HfVlG3sowkHf2OotLa6RUsy4oHSibi+a3fglbP0Ly5YOY\\\\\\\/NM2IRuPUaygrMDbMhXYYR5MBj6KpPOY3WTQWuj\\\\\\\/8xuKXm\\\\\\\/TUuO8jgqugzCXN4RNWSKOLhlag4OeclYs6WBQpKTTxEyTqCZvzmbCFuaNWwlnLfMdwHrhpJ\\\\\\\/a9taPvlBQ\\\\\\\/BaJ79WWn\\\\\\\/SiMNIle3PM8vqk16klqQ8jOpwcwMp1SmyxV1C2dE+0PnQsAyxWzyVMpDlDr5rlKbntwyOFAeZQiFd+JeZvZiVISXj4Rc5e5sej+zmHM1sslqGB+PAall6pdSgdRxs63F2Czwnb\\\\\\\/eg24HLUr\\\\\\\/lPNDai1qmwtGLBphHDd6KbR22JcoHzGtAPK00KRWZd6tyT0ZmxgwNAVKqBOsnOOp454Ruik6hAMotsgGDORY+\\\\\\\/2ErugU3oQIokWrlVAlOoHdtBDIEuPRvYeyODPspNWuYDDWdfQNlEjFB408ndy\\\\\\\/j7jO2Pt6EhV30cnSlEFxnPP06oJXXoCsdC8VlcWQtfTTOlOq6etV4KvutIsLfTV2qfUsNiZtXlNKN4wK6LQsk8\\\\\\\/iztAut\\\\\\\/AXVrq4\\\\\\\/sK07q0hy+2CceF3PiOZUCQ3m3+2IIwSjVlMdhn5W2zk7QVDT6gewFK\\\\\\\/Y2XUN\\\\\\\/EestfQOXUp1et+2fExgwm0WEHx\\\\\\\/Fvw+jXlWCz3j6Y9okFz6qtFya+723dXD50a+NFQOZtdf+WxRWyv4cMCUYAcq0heNgEVHdPbkYjyHvMTbe7219x2fsp0hIBCaWaZ5o2l7G2uf0pcxut8gNwn0DV\\\\\\\/LVv2oP0L6HgQSmZRNux2nvbR04j+ikVT2+d0QdjmD7g4GK+ZCaD4357XHslNPGFa5jM6IAMdNdI7TeTTgLD1emgNP1Wq8\\\\\\\/ayHLijv73ln2dD0KBpw2dlLQSAFDkJSu\\\\\\\/5cXCC\\\\\\\/KVZes+MgeaNJSUai2tcvoBKGtTBaOtQ7vR1Fz7qXdHmKly0CUcnoh3t7HdWCXL3ZVM+kzgW9NaToZVY4Yp0kh+Wdizwtz\\\\\\\/ZmTbhTCULcoWcTPDEnn7\\\\\\\/P2I0jreANXAdwQbJ18zKyIELa6ZsWZFqkIz3uOTC2nC+i9+5NgcbMlH3XaNiBsm\\\\\\\/hXAVm4fg4Ig5FWvRh4unJFE1S1JshS2post2H6ViCeQyjOha0GiORZmfRb4yvRdXxNeCawSsupnoy9D5njKMymbYep\\\\\\\/2IkZzKVAt+jy0VeID4kG2Kf6R\\\\\\\/cekJfejgnsdxuqv+g5BQlHfnIw0MieMesWh4wQ6xgykR2S9Cy8hs4NYPyZxKbVBxBc\\\\\\\/Edt2Ollowtu\\\\\\\/zqz2oRukC3KjdsdbLvkc+2f7GXGsOJ0kTpJMCMK15\\\\\\\/WLLbvZ6qMG0\\\\\\\/ramoRtgd957PECz0E5EfywlAsR8NbTax6CSiKop+6EdVFzOGuIUgGZcyRnUHjdQ0IXluVghz8vjBEpKNHDF6PjKWh1vf8EIQfi\\\\\\\/tEuz0Xtx9zT137qRoTAVYFBmGFCXkXZ6Gu0AD6t9xUzAGFsmccSMw2PET\\\\\\\/YPRiXsSXSAxkpypxk3jqdhC+NwBamnUzRsEQH+bBgC86wdodIaMLNtFPfC9F7fbUEaceOo182HP+GXBjBUYRJZg\\\\\\\/KKR3+XXvIT\\\\\\\/UQigWfsDceXXBK1RDsf5OWUi10thhX3OY5DHExMRM6YEKLeudDvdjq9J4JqIHheeCIjesZ30trjpj1y5u7RTs\\\\\\\/v3tEwtWT9bjT58xH\\\\\\\/oWUKO3So1Pvxbey4u9LDFPUCOLqx1qvfydc+lXGxZm9h8oGJnf0RPFVsaUUBxP241kUSB1+Y7bE8oK6vxQpellmcsX34kL8eYkQcF+yukh76u5tz4HUqpojFCrRy6QVVVr93YBuoYL4Q7kT6oeVjBTM0\\\\\\\/UhEn2t4b\\\\\\\/CM4axzPUrJZgFPwGW3rIMW1Y4sLu8h6BEgxAHwaWCnOZW\\\\\\\/MDEc6Kgu+WLIXl0YNTKuzf5MxXPMlAbcxv0V80gJ65f\\\\\\\/CrBsYr1ARbl1cJfFseYCYMJTU36zRSJIcNKkhRRSGHhP9AOC5zKGfLP7Xag12sSDgOxH7GxuwbEKPi9xGjQu+MTg+hw4sZKcewPP+PgPi0YIPEeI9OAvaROYDY34Njhtgs8Q+WXJHJ0mdIbGdGQI0\\\\\\\/RK9i1H57\\\\\\\/460gxBgGAgBB6oWEfVNW4CrJUFkx+0bR7RP0inRsRpF30UWY2TaIzKdQr5ldfMqsyBnGoTR1+m+q+HJHGHVJPlI5IFpPFa25NGiv9Pw6JK9gSrFFfuxJfU7agaZo2w54Km\\\\\\\/rCOycY+RerTgiMXVJSSQ\\\\\\\/sAlYQpftHod03844C1C9u5MqCQbxb4WZspwV\\\\\\\/D5CkpB5dilhZ9mZqBYBvZ0tk4ryT\\\\\\\/fbtqFPFDPo0pT0\\\\\\\/y\\\\\\\/JH71AHAKc0w+7EYFq0F4lZijNoj8hxmAWrwptfjWZMulPKYBvsasGySXHRkGEDwH40x9kFca0ob1v0uAQEjiEsS5tpIgGmTYoQ0TEe+5Pe795GHHlRMCvjb1r2OPjFuzBTO\\\\\\\/lE8ht8un05PM1jysTApyUe+NhqXB8DpK9Qd7\\\\\\\/+GQ\\\\\\\/+eTt8hh0Vmf77O4neO7g434dieQspxH93ZVtDn5djOueBEc6w6k7\\\\\\\/yX4alfYioLpDDSt7lTR2VBTr1Ese4Q7STugcHTzYPkrGWgWl9aRezj76LeMTvrly\\\\\\\/eCeAXZHjkSYl3zXSnyhWv6rhvS8IQtBQ4hyuhZGfE6mLY1CMDj1zSw8PzpxWOw4TxS73U3ykHJfyxhcTvz2um+IkhgF80VJskOnCXrdtnJAzhtGnZZN1S96qrb5H7QZ1bCIFW\\\\\\\/fqOReiiTmtp4tjmOkSMN3i\\\\\\\/RrD296uCIeNiS1MGMjQ0O9MacxTRLAUnS0SgHCLP1y7HkLpq0oqIQLhIbQvDPnI6db4yhOs+VmL7oEhyvmc+s0hRs8x0vdpbH27iQe18Dcd0nu+NdmuAYnvUzBdU2Jpic\\\\\\\/r4WfZtwH+6lE9HjtxfxvHYZJCaivVYyhMLQ6EdRFrKaKCoOM7sHTdvhA7Haw7JtSk3rLe+sjKYWqvzkRJazDugx1YwuP5sKCVHzOkQw9Yk\\\\\\\/ytB76+eqFnus6blaYldjhjm6JXL2H2IgPtn2wrJpmv9OTfzDR9iO5Tx3ZZ77rzEr9gDo5JK0ZyW+ZKMAM\\\\\\\/neg1WVu84EfwGxkTED4Vbp6ltFi\\\\\\\/jt8oPmWnMzSllwG2YNHLZ14fuqoxjscPAwo6WMBbvIoiIRs63smal+8cvTxbxzJAphK6er7ij3PjeRpEYp7T4PWpOGES\\\\\\\/rWwN9JvZgvgNq6DRZVTZsGcyimrNvzt\\\\\\\/hRkUq4sb8kAdqJB\\\\\\\/hjgACGaT9nVIw6StUwtSKG6UDA+F3FCtPfpXPgoABiMqBSpRo+SCw4h2aHjUVDDnZuZf3tg2UNMYHChwQU\\\\\\\/4FJZQEawRbaRVm51Sq1tEi3bLuooAFATfSPeKbq30gdz5fkyYRoxrLXNyVXzpZJbtneUJRv4fkz\\\\\\\/hLqJhajPC4McXDleU6P83TREUuzhb\\\\\\\/W6LUDXBKPSfa60nsvaJNOdb8DKuWNTATglapkvA\\\\\\\/VCJeP6yy6DhZ4O02Fr86q\\\\\\\/GvRcQKyC4peEvyzzOs0t7UOpoEwWdSWavdzeN1HatH2pRsdbym9YiLEKyl5Qs1v\\\\\\\/2XKVs8SBINVTbNi4l9HbKOK7zLuycE31FSlBLV4hOfw9TBfKAfGO+Hn6XPXoAaY+4gVD14xnlMDEssJret+FyQ3KPNSVQjPVlLiN0qo\\\\\\\/EliVqVegta3zVhpB5lX6Fy0JTlFgII8fpd4eFRXKhuSHV13hH+RH2hjirmzgmOGVNq52af2wzKltwD0ecZDJFvMOHz4J2xP9liwwukqvRICs+ymb0XM2eWoGmEvK2yqvsSS9xpLnXX7\\\\\\\/awrjbW8CgmX8sF1zF9T+4DYWI9XusqJrqCoEG5mF+g7NiCTlyr+ss90aUI9ti0C\\\\\\\/0uPQXoJpll0UJPPtMO6QQf\\\\\\\/+adXN88gr\\\\\\\/Ls1GKyuBq19kNgJsHU0mFRe4EuuC9syUcYghiFf5rod43N0Cp7LBl5Kf4OGf7RORYlElmlucMAGTbZmQHwSeKxsjQnmirHFEy6NuoTNNlJSB62SeyZnK+12Mwhr5vh8\\\\\\\/T5Sa2zmtk0Den8037hifBkEQL6OQlkfHd3Vr6964VF58MyIIBXpTrirOmSe4D49Q\\\\\\\/YQBgNp4fpZ8KmLlbm3O8LweUvoizZ0MBjMtSOAXaWaXAEBVluzR\\\\\\\/TE9I8Xy\\\\\\\/R8vpoKLeoYWf1Ee24EyzqvQ9E92QpkUsGBDOUDHWcw++L1RB2JLodqCPA710QC20s0Kgq7do47IIvNj5dyVQQ9WbnnIPfOp+1FqVCnKodrbmN3d6+aQ+Wdg\\\\\\\/UO0ClvDKqg46YsbYWQ+rWLNOQP3QngT259sPpOe2\\\\\\\/Z80wKqIOS1rsBpoIvaOvALydu\\\\\\\/+yFw3jNJvvc2fgm1JZ\\\\\\\/wTdB3x9cHYt4QfPcOzAse9yXvPWHtd1TIj2hhReUuLqQKe8L4KOBG6pYZexIozdWGAU9Rh0OkE0aO4D9pvPcTymjTqUwiwFWeOU5zUWxML0jqH+v9ify4pYuFgYMH5vERn\\\\\\\/AxPAoM4lXxoafrttHYD+rFEDqp+Bz8Cf8TlbR1fxObzGY8qPfn\\\\\\\/heT5TUSHnMQkFMohJuUGBO0eLhz3vmgrQThkMHGRVyi8w2m8JroM5Yd7zRaZc27SZ18t9jLMn8GA+6NgatcEWxAuavkIzBB+VoGkZyA607jZ\\\\\\\/W\\\\\\\/hKiQxhKN3ts4mxQ1M8RMSuE01kJ\\\\\\\/ohHHz1CtKQSw31vo0vMnPj8riufgI3fTRBac+eD9JYxq1LWLUH2haoDAg+ILfGagEEcnnmwfUDxan\\\\\\\/95qJ33pPIi\\\\\\\/fd1icAQvntqYfgqQk5G+1zowkCCO7Ye3vV7Zt0bWeP13hzf6h4PGcwpFohLUAkF5f4HLsJJFHEPF6KullQYcMZonvA+tKZrS16vo0Z71nfFzcjFAirAi9t1eBoawWmlRrmbW8g9Yq8+\\\\\\\/VzCiPkgxJUUiK9eZs4IcnCV+wVggGUTJnUxEoY0OvSpY\\\\\\\/ukZqwSdL29Gt\\\\\\\/FHSGTzo90Upn8v9fjFcF5LCycEXNWRSXoUw3fmikXdtT33mTGPKcEKFOxMUkw+nYPkiDVDf6sTAgGV0K\\\\\\\/Sm330r8x0Nbw0+EHx3O0seo2LzHauBGPYgYQ1LfHr2olCuaGbhFD9OE7KK\\\\\\\/fktfTpKHzEordcYx+sWHx7vLUVOttDP8+RS1KYKHcc7fKTLGdxsYa\\\\\\\/mLvaQGFY+TJLu25\\\\\\\/beBG1Wnark68iLKn2gM+A0p7c+hcF7CNgCgZO8CPdBFBCR8+FIBSqG17pDLvtrkyf3pMWkxSwaWK9hCVHPq\\\\\\\/a8SBG6IxU1xEFWEiXwj8wNiTALmkgg27tBoD7UrABdA6IJyjBGNOruQxNNh15BWea0tJ2DOSVYhEhME9alU0wuDtnXmAMkhvsvWiymUccfbyCygIv3a26UYRJkmBmnqOlX8qMU7nvxZq\\\\\\\/zzzJOu8V7L0QgIdRWkrjK1aHrId7rN65+ZLTgHe3dk1FCsNVNwonCGEVGUVp2ljWbhqVnsJLRsYFrNAfqASUsr82w3xpVL7wI+98PdZH8t0bzyufqf1u\\\\\\\/xbW2yO0IgFAAAM2JAFIBJ+Xip7PbancqPeX7QTlNnZ1VzTTlmH18mioNDBWqSWrs6gcs9tRXb4kWcjCr9LSZRePkzGZSymzVTw8VLk0NGheywNyx1+b69trx4d6gRrRJOoHmH\\\\\\\/0SIWDQPN3JT3DM9EYiS1NML9Z3Som0WOS73wnrB8gE4AvY63MQLCodzworznc78sfd4dGk36qwtUEb8669SKTjxM2LuaLsEr6bQELfHA0c1cZsvWHOpkBEuMMqrElErTFVQmX0uJfzhZ6s6XJ74b0EaETC6KJvF8CcHisNAMhloxn7UEjRrfAkuQtL\\\\\\\/nE0nEoCMBvhDWXvGvlc2fOMYAtBUbF2nK+bHlh5TlCxTw282zhN1MpGlZCaWGk2lcTYoDMxlHfgAzZmITqH0Zh2zPQ6TGFOpNWi0Nsj0RaQtLpZSYKHptYEnmbQ+x0JFh494DRdjoujmJWijsAQ4mwSl7t2tsvkchZzAZNaZVj38hPTphnXAiZMEVyO1H2yXXTRQQ6UfUd+Dxl6O4IeX7Y0sLwCsandqGddt6kuaETX+VA2\\\\\\\/oZd671bkTYdZFsJiURdkjaUouAIyhU3GJjY02DUJ0lY6TnuGtIOAj0LO17\\\\\\\/SxaQxOlyI6dLtOtcwW0xo1ia\\\\\\\/3in5eggF0INaVNmDqSOLDpQCPMKMno5o12CYgXWHceBx6ToLWJCjblJAacTqBPutI6fBsuj6LdQwCGOjmtjku4B+UU4eCIJ1lmw9+B7uPMPJheMEiP34C4WRo73PJPCK7bdwd82rY3\\\\\\\/GeoIttsPxlzdXufkSWHn8gC8Uvuj4CnN9ROLVrhkbr\\\\\\\/1z7W9llJ8zYIvOCfz1CYxfj5U1FUa+6ycqmheE0D7T2R2W24L9SS\\\\\\\/Sfb\\\\\\\/ySxpkCv61bFVwDX4NWonAncj1Zc60sw+s3esaxD2XQVekByZLT+mgD3\\\\\\\/Kpx7gQE7QstgYTaPETJzgnp\\\\\\\/wtpxa9ahI7zBJO8Fp5MZOAZbgwhN2AorAQk3SScB4tcE3StNxvjzkYNvNZY2J8gcthny7NSfF5KYa1ISE9anRnrPu3\\\\\\\/bTrUBha3ERT+mRCP2CRwjxfvL12Z1YIsV\\\\\\\/0zqcvzel9xvqMXpHYxiYpHC0NjFlKdZRK0yGuxuDakd6vOmcq5ggCrJOSr764jYfE5fooBTZo9gpW0ML9wfRK6IP5dTVVu7lomP11j3yA2siv9y1CwJ3KZrpPYh3fNLE4Auyw05STqlJ6KpHlr7085Hb9do9UDxfz0Em20ny5NNAD2DAgUXKTjgZ8rNfKmQMOo9fyMOFJw7LomNes8\\\\\\\/JQa7Sm0tgXxGdJO6cu2IaUkWiWvq1tFA4TaIn3dvQk4pDTKSLYnIgWnDol4NKHuEr1uDk+Tx11bg3ZsS2RKRYDERzVhC6NE8gHh4SkLeH9ragGo75i2IQWt9E2G8GpQB03SuwjImBAhRvolLF8H5x452esFfVHMGGzT54xEybmmftPDLJUXm8BfuM3kWZ2DsT7jfDReeWk9Oi6DvZNDbq\\\\\\\/YSUFcT\\\\\\\/iHW2kb\\\\\\\/PVEjEZ8NAqJ9VuxxRuU5I2FMFa1M6Kv8iXYtCxDGCihVj2aK1rzY723zNGjGFQ8isDgq9IoIp+\\\\\\\/+DIbDy407LzSfNvFhjEYDZZfV\\\\\\\/KgQrskyXy015S2FcNqXmwlILRWQXUODHp\\\\\\\/GxP7CYM5KoNqlw0ZoJNMTDIdcAiUjDvNUgyBfClYC+fEFcSgTzBJAvSJYW955B1c2NyA3qTB4rBGUon6AATyQpTwciAm0cDqQQ1Fw4cvEdRXkqGbmJIIqrpq+jRYNTXtgNgA7n7c51i3t5MoMAkz8lIP8ba3bEYganu9IqwGzvQU9qq1exxYFF5CAISOmrmE94bOeOZLwlU\\\\\\\/g4jRZCM5Ee7rjpZJQox71+5e2ezH9FIjoVhJ6NC2cobu1u04DXDwzeaLEI4TFhvP5ZPXuDCzr1qtKciIRY2gFom+NT9VyZtLu2g+vwqJsOkQ5ZbDsk\\\\\\\/kx2yiu9aY0ch1ak3l67QTktWzd1wBr3Ua1mBCVZ\\\\\\\/2At\\\\\\\/xnph5MyZDZE57lNR1+0\\\\\\\/h6IORa7jLFi4g\\\\\\\/uNbkA9Rsjv7Xf1fNvzyiPIOH2tWJBW5NaVlk1pFnsdE4ZL\\\\\\\/+ralOZoa\\\\\\\/x44bJXrQvPEih0kFvstZeX9UzVXOTecay8LuT6eGW9wqiE96TRiHPXhviKZFORKlRdpBFjQPe18d5raKGuWV+QTICJpmCOzyQd8tS5Jc8AICbLKfaQvP+hZ1JeR0MdcLXYfBIujq1kSKoUeB87DuDfADsHp2jzv1AFpkkqDXBf0tyDlrUcyIxpf\\\\\\\/oOYgj8o9iUlA63Cq6oeVK0y9aWJDKivfn6xvIF4ShauZ+ddHVOQfXUL\\\\\\\/GeJ5oFoVpZxYUykERstf3jWp1lQJ38lirCgE1memmjeSkr3B3JCZXAOIQYTj7VUjqQBcwZeEnk4Mm4ZmuccbEwkbRTakXbV6HuDRtKUTdzU+9zxKFdUYqoXJZZTxURTQqZXKJMngKzDL0veOifVX704VG0v5rQu5BF1Fqz58c3UB\\\\\\\/wmHCwplMxiJthX\\\\\\\/NBkRJOiL9IAwAuMIhaIswg41qV76xYMTI+feZH4a4zonVxb5vJaSUIlf+LzCM7l28lYii\\\\\\\/wZe2dvLQH92DacNz6LElzTkeTxEe7HEcRt1F+1hy6krAUN5\\\\\\\/mj3iuBgBw6I3arPJzCwFvBrMteYoUIVS858lTQcAQSwlq2BfPccCZJDMbLylcAjtRPL\\\\\\\/mIMe28xAKEgTgjGO7Xz53965PIo3cTMAeuFGsB0wNszXFkJ21Rzm6wTg3ZPKNpZFXsFhqovC4X9a5rAph4T2\\\\\\\/FijSBhTg2lZX3o2AaQc9gmFsSCKyV2SpZ7S8t3UFcBkWP0zldi7LR7U7FMYLan+sXGSJqS984L8qZgEIjiGa4vGxvRqOH7s3h0aLxaqOcOdk5BevbAS1120VFM7o4w3tkqsxvXRcbbblkn5pb\\\\\\\/uKzis4WTms06tgGB3asCzFCyn\\\\\\\/d6aNfm4rD0R4\\\\\\\/qfkKkzMHVJFbswjCVN8F6YEvuoggNjaxFOACqihdv6z0j+BHNQIZK11cWbXOfI7ZPbzHt8tenS5efIjDYSsyIwpqHiH68\\\\\\\/lssEC1+uORTxTro84tbH\\\\\\\/Jujb0j1TZJKHeVVSN2UVvYVf2xLFBX41n9UqhnUZn3X\\\\\\\/gvz1u8h1LubGJu0dTwhtQBFUeMDIze9Dl4ap1qvIbJIkAbxWHVO2YnyLLCxf86JzMMmJZRA6XhqJePx2CoeSJocRTGLuxuIRZJO3IIuQA1ivYStfV7cLhEXo71rRcqB\\\\\\\/KKVEurFbDW3x7AsROgH9taBwbqof8IcJPHjRT4yxemU4Yoa6NbHASQ6w2VGkixY6UpoZWcfQyZIm2icD\\\\\\\/+SCtYD9VyalctCN0uSWECKIGt\\\\\\\/gaP5mG2W\\\\\\\/gApm67v1I5d2qB\\\\\\\/bWearMEb2Kgdsqti6RHCQJR8J9zBEP9+q7XW39FZF9Nt1RZ1kMTcBauwq7YQzuZu1onKO\\\\\\\/4uzXE6gA3azHbmx5O5H6DmrH1uNDUW91LkgEyaa8rTS8r2lyh9WcvN\\\\\\\/DIkRIa81MIs0jzougud2t567W9vQkOYsL7x0sZgM6rtJCeVEJzpR\\\\\\\/iKU31wem6bxltV993OveZ\\\\\\\/cTEJ3R3b060v6v4iuYMGVZPNkf6GavqFt3Tt7rU5yd\\\\\\\/L2cTJhzkCBljQLF1ZzaNDGjZwz9FyE30HD4ww7SmoOumDJksnESAN5dghtqf3FNT76q7bpkZkGwUWH+t5Z\\\\\\\/P8w5oodBdYwMDWQftVUis3BIyHcRs0I+qwipp8aJlSc+L0wLXVC6oWocA04veoDegrq2dPAx6K7WXQtG2JWKtTaSSDOfnzp9oCG5x+1aEySOgqHY\\\\\\\/ZQv4+1dji6rpyyJcrByYOUric3JjZaeuf2csBgcqz2mSS+TaYlRm7oLx5UrrTn6kNk\\\\\\\/+eG2UyHHL3IRtrt8+kqNW5Ljo9KEgS\\\\\\\/zsUCxljWU6zl0iYG69cYFL6FAhryRAGGG\\\\\\\/jaNu5OR9784qyyaOcrrwsfh1zHcrL\\\\\\\/YtptOEU\\\\\\\/bS21Ov73bBAi5hIHrgeWvDecuKF6TFJNBpIrqZAqRPRsSBkEzjqEdOsdvecacKm3N34F20jq2AWKBEmVrmf9iR36efSpri++AJ\\\\\\\/vozfKgHWoVdzM7\\\\\\\/I5UX2EbOVfnzUQmXnF37SXF2bmh0Aw4dZHKRbX0rBYl\\\\\\\/v3oobhqxn9s\\\\\\\/00+BC7JitgSz8sOqjzsXQyiok\\\\\\\/7RjLm9IUD66DxXijjELC20rlDG1\\\\\\\/nYM\\\\\\\/S3VFXSyN1JMK5+4uJ8txxC7EEkCMd06ia6CsszSDNjMJVXDwWJjbu2sxP2WhnXn6Y65xZkIyerNnZBYsVRS1gRc7XkDtmofoT+NOoU9wsn0r2LiiEbTzjkWfRz7f+DIPJX7IkUAgH5mN+6SGJ2UKWRU+n3e5IQInelxhCJ\\\\\\\/qnTdvKZuGC+FQ6t5U4dCDXG0wgh1lL9nhupqvZ5V+JTLRt7SHd9zcBh6p93zJgSN7FYlaCREbYGICEW8xRLyKAD1zHXpVySBVsBDU29w5P\\\\\\\/QWO\\\\\\\/gjRok4P\\\\\\\/Jt75aNvCo7rsqonap0mSHceYjrNnRuLgmpfzZO2nRbkQY6bfFRkWSBmaDxhIEnSdtiJTBq6GkoWaJcLmcnOwLYvfbwBJ9qEnb5zcJ4u2u1iCZ5jp65rVC5RXApcaiTujFcZxWwBV2Lwot48U5ddIuTNdNB8spvdmgYWrLPQNVnsS6hRFcDvm61+I7GcejyGZ7Fld8ywkrp22WbUTESOUVAcZyv\\\\\\\/SYrwYFJYZzT8cudu3VkfcRINBHM5jIEBZ+ZMVFhh2KZOCy0cZpDJPqxvbn8A0WzQt4g\\\\\\\/Hbw90WNb5QF+fgwGxbB5C1o168B7Ehdbr+\\\\\\\/C2z6vnckOk\\\\\\\/qo2vrP83kSGqbHb8AysH10PptF8heFwtuSI0Z5EvXUCASIV4N16mxsHvM7c2HPMN1LZ0p80QA5p6wylxqWBcLOgLVW0z5oysarI+mFg44t5L7SojJ9vu\\\\\\\/7jaMModSItX8EC5limXbwjpfRypRyg382aDZc+AHzV+sjqi9jbRCtlIvPwJF5\\\\\\\/PMxiAEF7FkziuB\\\\\\\/5mgUA1TY4tCMZfrDf23lfPt4WWsCqkPM+9odaN8PmSVaGqEz9zrKND9HhEQzkPA7Uig1ood9ImnKF9a1Qgk3Db83XhS03yKR7ZM45uaVdEDui0l79lnH1NV9D9gbEyrm6p0fuL5yjeoBoUVzZ+\\\\\\\/edYJWGUPPcvWchd4aV+JvV9DeqkE4zQsyQ9ouCwyRyjVyN\\\\\\\/XQ0bOFkM+5yzWJUrY7JujHkUtWaMFwXIEK8J9YeUTnL5ZG6B8ejNAA17nrl0MO2\\\\\\\/TSkML4RFO6+Tj+DHbAmOmhKZTpJCv2uKAz7BOTgAc\\\\\\\/MqekXNYJ+SrfoVPvA1dGCfdro5iwg0\\\\\\\/\\\\\\\/LraJiSv7SV3X18hNbqSBdiaOR5IJWBa9y3Eq5f2CLf9zWTwINqfd6h+mAbfKlxkLfQ02M+RmDFB+B8bE++fwuKS7sJP63+5Qu\\\\\\\/po3yHN8x0vZwTx0OBHPUjZVC6dESAYvrrWhB6JnzAS7ox1R8FCDQXBI\\\\\\\/j9spKnBZaLgz9TyHQvP+lgM4sZQ+s0f\\\\\\\/sBXBsnldIwlNFkV8n1JIFigvLO5A+jHwZcdk6rX+pKb1RrgcOW0DypndHJE60y5Myj5\\\\\\\/jKbu72n83n+YR7OxJPhG4VvCgibWfZaxJm4gKhx5vmggdlhxId+wac2I86dHb1v89Ato3IljAVmkT2GkfE3qzBrDUjb2NlCtlVa1j9kDt7D6K7Y56noUY9N6BEfkMLwY+keko3zsEtJ15XOib3xcpQxNXaGNzk702xOt4idWlVSyrMyQDn3ZFlJYUY5h8WRXgZXJrSPyCI7SEniwdM36ffXkNNvLe\\\\\\\/5YDwGR71OZlui3FF5AMhkRQaKfBr8p2PcnqZUNG04Ugin7TddYuNpke0tR8m0Z2r0aXfBF9oTzZ9CG\\\\\\\/d8qxF8eiazTmve8E9GvBUuhNGu0BodwBcOd91JUz33pDPz5X+tpMmxAHKtMFrsZ0vF2+krHkjvbGdB+CFbMTQ3vBTWj4+ev7dqQsFJXGC8aeCEgjZI8mrZsfqqUBQXoqP8xVkFsMfjJvycTYR\\\\\\\/NCpCnfc7YuswqTcUk22Did7gYeXgrbNlsDym7f46pejhj37dJZr6WVRAoRRr0LgKNGALiFjdRkyPJ5m0XchKuHDwV\\\\\\\/rs1cpN\\\\\\\/9Xb5LxAcUlmnBlzP\\\\\\\/2UiAu3ShVdf2PIVaiz4Ei87idnZjFbx4vbu8lUDz8I7GtAHEF6m6xQz6mS9tvJ4U8HhuZKgXNiaeQqLEWzH8HsD9oodE5LZpzOxFXYJHoWETfBZpeOF\\\\\\\/M6k4THIN\\\\\\\/T5MAA+VyLtuBvzjkbB6dMaFZqg2g\\\\\\\/\\\\\\\/aN0LXzPIWdl7VgHjEZAF\\\\\\\/QoGtRfASHSb8eMbGJz7ngsleFJJOQ2vqAcK2KZT3plvOAVtswOlCLuBRi3a2Y2ieiULy0VtAbuUT0CE1tO49lhR6hQIedH8pfwND36LtgwVIXltpd8aQavWY7AEvNDcyI920hzdQOE7+jYKWOW2a0TrICmjmqsao3l7y33VL\\\\\\\/WLtbYGwAVHTvlD+6fVRm0xfSI11QqUUPUpzUp9JTXac8Vn4vwrAuaRBMWWc1706D+rD+bwzQUuD+NZP8gwE5bKyKeyX9XIao4tGWgh\\\\\\\/IM3wZ3lovw4pI6Os5wFzpWuNLzJZDCi42Z9OinO8IFO2S4lTBJSRHah2WJCemtMq+wmqYzysG3PIYUxQFC1GqBSAV2t+pXlb9QcQ5uRjO6f5vLsOcx83gos+fhZXsC1M4V+JxzNNgeMMY9cyzvGsKNbtFdSZleexIqf4eROp1Xp1DI+1y6OMbkLvr\\\\\\\/BHcp8LNESwn6aSrNRpy9do3JPcEWM8T1up22kMQ9sZBiV1LLQY7MEquucP8lIJ\\\\\\\/JhthjOWOHMyf5zsuqokEmfqOWXmvGUgCRlb6bJ7J5gkIhXrGJ0zX\\\\\\\/FaHF2ignMZz5u\\\\\\\/+odCRMvvy4NyTU81QHIMSJmxwYuMHRkZP3zmSmd7NeF5NvnHJ2GgVsVkazmesFWHxlA3ym7TPbuhEwfPoS1DgoxYTM+8fUjfGbVGuehY2M4zPpF3c2tU04xuQ7AGJXiHQGDhztKEtzwWwGosmYJpKjCZAjpk0bvvBw958C9QN1UYPLICjafoL4AO5SHzzz5gF6x\\\\\\\/e35tIPPV4pJgVUqiyzU9guRYjCg\\\\\\\/OGwvW1612inxtSaWdqfxHdq\\\\\\\/F\\\\\\\/hqhU0O9DxCg5+hIIdNA16ZkATRy6xGsVm3L6C\\\\\\\/X+dGXmUEhpqwJRaqXv4o1yyqTxCwcEK2vTazM6DgJL6WGLYVPQ9MimOz9\\\\\\\/OEGYVhvXDIMzquvorcXn\\\\\\\/TB6h3\\\\\\\/lANo6KP5+40V3RAKe\\\\\\\/I8HEeBXr2gXrZ658bZt+6C6ALZXZpzHjVOluvRWJFixbX1wEdlmqxT5KKohcEw0pR2HoJJrElLnfA79JktzGl93qzhyZQaN3smccn4aFUHPRKnNoxsJ4mCRwKr3Sr4S\\\\\\\/Inw1gttYCqLsd6f3ypZ6JIBkCULEjU4z2ldflIvn\\\\\\\/3Oyea2ESrdCFSzbmuzZ9s8jUZpVTAEQwO5x6+gO1FUwqMqkI3YYvklknys85\\\\\\\/kNDuMeVM03Od6Z6Yfp7Mk2CR5SbHM99WqSVfcmgMmouQFezNllrt2MAl7s8aaOL2EkocCswEjnqgbOwEMeXhfbgfXbBSF5eEDIJXJHb3iKyaw4c5GdVa7oRfEySkJuEVcEUyaaG7w2asXWodWy0IsE5W0vNFawA0wWQnin30MwBokVHXiIlTCOuU\\\\\\\/5z\\\\\\\/5BLXwlAgeC4aJgMRrTeDf8GVCq3zWRMeb4A6ma4Q2Orj9obJCysNlCNL2lmH03K+wuGeTFQ3IMaPoJ0oMAKGP3xBh9orAHZRQO2Pi6NWoIBrOO39v\\\\\\\/q4JIcbr\\\\\\\/0j83JEbScgA3x0zdnjfxTI6egFhoQfuN6oaKI+MiwiYmJpnf7QmSHdz7eo3d9JdXb+gL1\\\\\\\/xRqM3pewq8057Xp171isUV+Grsk5XLiq4iXKFnLDCBgz4ke22g0ju5qZutQ4rS6NFREfAEfkGoz79jrex2OmGm\\\\\\\/WDY\\\\\\\/Rbk0Zw8NEFOKKSfwZnaqkLXgFXqLVuDYj5W8RtN3oTHnokuPYaFhhOVooewYBq05dCHUwD7pyGnDHZbbEew3oB+Bcg0SDe4Cnkdgnsyj3OA6+NogsNYFsxFb2l\\\\\\\/\\\\\\\/HCoIm4qIefJzIIRDeW+7XBXmHO2C+IPd263IGUuMidrxK601tnxiP4gbfqvkvVtQxXQ1CvV9leJlVkdyZA0CueYTOg0jdob8IAhtgEgmi4ZiF5VsVwPXWByuRNCgFldN+Zu5NbRGLv7ixi7oRAY7z9QQRLjbgD57H1eGmsiRkCoInc7XyO5BQ8AGvFx81c6y\\\\\\\/Ya+6LHUd2RPTZiR1ypRyttxFZGMyauWyBVZ\\\\\\\/MEgXYlshO1xF3stHtD8BmtosfLY90EvrdVWsxxx5AqrvhakPpfmRDix\\\\\\\/ulVNWoBZJSea0jsmkY0\\\\\\\/8Y0R9+hJYrEqgf1DX0RAlatlH4trhBLCEHQ1RDhJ\\\\\\\/NQwmNHWgTWbHH3HKrfeNLJw11uiGjpbPPgadItcSciHGftpOk6QGtbp2FTqN6JbMn+T+z3A86aBuCZetsXhENva\\\\\\\/clcpgLwPatf\\\\\\\/lmNb2aoq2JHhFNUCu6dfwt1k3N6QMgETAt0U5fvZSKE7DwEQFqXNrOWelxHexsbmnpxv30WzjirBVoyf6sE\\\\\\\/4s090TOKoqfkAKlNEs1cAAQErYTXjnfu1RaK1LfpHoyYBXjG2MctmgIye7Ew30BnY4gwHst977\\\\\\\/1Dbs4jQd\\\\\\\/zN+RJNApdgNi4CuqybyVilC5xFrWOr4woa40Pl62QxgIFzataPIf3tqVb1gO0FnrCIcExw1Bjtnu768NYt4wLoFTdGRGivM7TYStfxK6PF9SoSkR+XCLnQ0SgKgi2cbKTx7GzR+qnq8GtQllIsIbEcBTIt+7Kwm\\\\\\\/kgb4KCbrXZPFLSaRTpNf4G1tvSDHifnDkWWnh1doO7BXyfTg0V+LOYAgOt5IPKif\\\\\\\/BjD7dS0MNSIF9Omp92O6DN2MDcP+1njCZ2QcaqVYniwhlwZktaz3V7+L3Z8tOkCvaZAdy\\\\\\\/pRdbsmsfkeB\\\\\\\/H62\\\\\\\/oqwL9JHBw6JibjixJpyLla9D7SxYrpdGyRdS3yQteW03kjLtA7rMNy5BFos+rkmeZeTOOtvMbC0y+zLA1vG8rvFw3Rme5siXmGeei4Ri8ZlLDxPuxJf92E4R0IgEpoLqBeQsNO\\\\\\\/tqM3ErYnxGH8cDSDXuQn05x0DeqJcpdVjw3ocg09VMl45pOOpphbtK2MB2mHiM4S1bUE0kxoIP7pXEE0IOC\\\\\\\/NdvN4Z8vr9mgHBxZb7Tg6bRHw3xVWfRkxQZ0ZJ5Soh2KXYBR+akWCKYolpsn3rBTWEzKrpRwl9prDBVGnishIp7VcZa87uKF8Rw8Yy36msHDy4lZ\\\\\\\/sEuj883H2eCq3WXEwCiQJTJIarXnGujXAbXZrWDzi1kB9ckNJoRhF60GW21PYGsy0von5eXIoVM1WqsNe+f2Ww38GdqjHPsMkXKcGMPkuzcDrwyoUGVdIAfrZ4eDadSMZMdQrD+mTv6EykeqiN5YBu6WsGWqyxse+tUQrlSihOZ8yixCbzThts6E8ZM0DG9j3npsd60bawKJz24Q2GNPMAUM9O58d6QUZfsfFJMyvQMRvZEj3OsSPSB4Gqa3BNL\\\\\\\/aHkiWIdw8c37T3x6i0Vh7utRoro+0fX8HL7r1lXg8tTe3m1\\\\\\\/eto4u\\\\\\\/O7O\\\\\\\/yi6dmSdXVLxnw2N8CspH2kJ89BMggCGHlV3d1bfjqKNLD+izIB2T9vzhHVV\\\\\\\/\\\\\\\/5VfzJ+OJBHIbVifs1DtRN6TnDy+uZ4JULl8jwvvB4BpXdpvs68e2sP677YZy0iiDT9BLKXzHBw1TkHN\\\\\\\/WNcW7HZJTwNoDebq+9SJXDB2SKqmbn8M+uUM4NEqpjzKAKwys8pjhQ\\\\\\\/P2quqdr5m7Go\\\\\\\/MR9injP46vT2v+YmG1d5lamszN2\\\\\\\/mj58yY6NwPMfqVQkKwo1Mo8zBb7jF3XZYsyfB8n3jT9UA1PBrFEyavcTll4p1kkBEJbV66lMKZywPU\\\\\\\/J6uTyvAbXwdzwTDor4u+9Maho5AKB28CDsdkqaCZctbvpTqcItVAOPS7HRjbXESsKht4Go5mhR0fq4OpMcLEinSgUw0nS\\\\\\\/hK\\\\\\\/\\\\\\\/gspxO9kejlvzB9YQOl7cMeBF4d31K5DWvFeGkWTFLrBvfI4Tu0onM8seI9ENoKsyWmSygPKA1wk2KXERlKPKi3BLUFLLOIjX\\\\\\\/zFfGxjHShlLyfR34s2RhFQhwfs7VOiaFkUw1JCLCKq7wNuBtlLUR3qtvy1kI2OlqzHNRunDoKsYxKRlsdEevKiJ68oEU4kVWFJ0YVgXYx8D3rwNrlXPwTpxfTnGhDrfMRJZ9m4roKe9QNdvHDrIDQPV4dG0amgZhsMyqfKTtE52wq3PYt2ZFuFrQYZg\\\\\\\/TeL9zRoFTHGqQDqnQ\\\\\\\/xNiNdWaP7JwuKntOYhzKZI2X6+Zr2pwELOMQRrSF5TS45bdDn1qhraRywwtlc8CVXqpP5iTupkH0zUewhdAa0nSIr7kwuSkHI8Ydyehz3odxKe4RoqW2p\\\\\\\/9xP+wfTHK6\\\\\\\/GZAxER04eP0d4SMJdPiOw\\\\\\\/j+RYgPD6\\\\\\\/eG2NMAKH53QvU7Vs7YOWUFdMQIrQ7UFjZyX57edK80ZxAYU\\\\\\\/DSaQUC9F0r73ExWjuxs05vxWpl9pzdR8xYvD1Wb+qmiLKHlRxLLUqc5Eef04v279yQtk130oVVrr67IWM3mek0ou+BOzsbbUHr1OM7jNjiRgU6E\\\\\\\/vECIw4+9bUlTTa\\\\\\\/HEVQ1hnPgaSdTenS0SYATpH1iV77Ck\\\\\\\/YX1r5a6Xvge+mLryYcIHDYKihx0kytZzrH9M8b69agOTJVQE83slB4Z6jqUtDCC7624TBymL40yeVCRafXnxGPurwoVeFdF5l\\\\\\\/+\\\\\\\/MktSipU1AiLOoeNJAWMY9wDuy7xIwdvGlyWjw3xT3cqDm87SkZCwb4vEErXGUfMD8j7Oozxrf\\\\\\\/z6jMWaYCTVcpKDdHb8FmCEe+tYSMjVakVJaHj1TENlaTKp3bYFybB9VVIJggS+rWyYpbetHz\\\\\\\/9IT1jSAwMdgNZjB+lChrPxJAvBd0Zmt5BjtkxnnrBBbIOUfcyWuEPIMQ7ZHLpI69ckVnqwabKWu6rkTScw98sMnj8tMLqVRQd\\\\\\\/DiVCjK2vs0aRqXA951ps5Q9lTt4eMfc6+Gt\\\\\\\/oPa1h7ri+O30J6NkCA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022TNOV9lJ87CJWVphcS3kK\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223c0e6d7b8dcb6f5b27e8a6d9ce90e4573da4acf0358a2336986a270895bae1e1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pcwYm+e7NCkmsYL3Q+vIUcIeXSSl3B880Q1qh92ElsViVioGZE4X2cVCdn3wCKY3caAkN\\\\\\\/Km1SJbWId609yeYZnOFBs9P7R0s82i2NZSXIwitjTh\\\\\\\/0uvOm4gkuCTN7LAHJ3wR\\\\\\\/kS1IU6oOfetNmx5XKaz4wKuh2c6rpRP73tNT04rjuQa73DRclO31v0whrTTjw5PsbcKwER2XGx8rPVA3i8G3q+K6nxlNiyFu8EnkNIsEMVSQ8za0xXsB5KQUcYWl32xEiFCbovA7b5Zs3GRQ0AOjxcGj1ZgQTIjEns3ewdNfWhIOpqZVFNuk5FVxHo7WsCJ4P1LlF2OTEwPg8psC5IRP7V0yjAB61bxTI9gISwUUX3ksi1O4PcL4hHAO1d0OWCSzN7kYqoBz7RUWBakZk5ZrSSyNqDl8CG4QZ7U3CKYd5OzcbWGF9JYybaFWaOSQJHMs741OSuZ8qCp6wqjjL5c52KznXNgpdzNCVXjfx\\\\\\\/pFTYMUDh4B\\\\\\\/ZyNo1F8hXxqxy0hkuACSSWZQMxnm013jvfQsBJWOBc5AhKelY39hY7FXNvfao9V10xWQBeorNEZQJt\\\\\\\/LJvxu\\\\\\\/Tj5\\\\\\\/dpeQ7o3UUdndX8F\\\\\\\/8vznLwC1cyKCW7ziDO\\\\\\\/dnbHeLH225t1HrD0+zpN9BLHoiTnVE1QY76pXNgWn9gwEzPS3HeBXhSvdz33yX2Uz1h5xuNSDVqmxr27v1bioIKoFec2CNsx4WZyLi\\\\\\\/5yGS+ix+XJ6Z6g7SYB5ykc6fx6i7+04q3aQWrrDCd9eCrTxGmAIYAoIv2HZ61oMNn6YEuJrUwiXQpznGWFiNY25n+d3g466ItqPDtkAsKC2vGODSBU+AqmGU0woX1TY1CIJ1mlErXlFYx\\\\\\\/ToNdKC65DAdEdZ2rOSFACw\\\\\\\/lBOFfFd0j+8bys59hEXSradWTvusKtgssu7\\\\\\\/mWmk6aF9EHHp+mvwkulembiTrGu2egq+9MFi0jnglLFMac89DoZUIzNd81XFHXZEV0eCC\\\\\\\/T8owQqsQg2ywIQDoax4DBXOd9rMlsborrDjE9\\\\\\\/Eq8aopUeEoGg0NPlH89Tp0Izm\\\\\\\/T9CBIhNDpKKJebB\\\\\\\/VMKjIKCh9MGO8VgM\\\\\\\/ijUpoZLphLlIOrH9O4Q1OIQbzTjTRZVHANUVLKAev65hioIsHsdZ7DYbVk76Yph\\\\\\\/bOY7GSXEAaCN1W0v0Dp4ci\\\\\\\/tSVUcm6aXQbBI8AbrgFH863kLOwr6NB7c7Jl0elF2A+SwvpQXWaY8xuY8ywVnaASu7KCMgu0UmNV8i2WIF3ypheGd+J8VWbdDHVgSa3uzS\\\\\\\/15rYWdvZnUwZ0Iu2FDFvlRT6fM+n6fv9cwGnRIM+fAYKJlq2+fwba2JKw0p9uB7XnOUDZRpez4bUxykyURqcEaSOl2zJcCMg+dvYNkTf2U\\\\\\\/Op+JX6xdO+r1f3fkWLAAnzMC74J0caW\\\\\\\/Xy8NSuPMg0TNP2y1wZOW3RBZ7fzxQWKGo8t2QYP7kh9MzjemUXxyKeJWknGFfTvboHY4g\\\\\\\/QIAf6Mih6RzAHJ6HVHbN8xb+ZDOc54r614iSU6NgJxREmKPvKxgJd96+nZ\\\\\\\/Ojdc4D\\\\\\\/\\\\\\\/ofua4HsoIJq\\\\\\\/CUXTBQ4ym\\\\\\\/7P2LI2l8S2AG786QlDk0ImO9HDLuyef8F3Acv0+JHFFAAxdQkBB+YlTErYot9+nUH\\\\\\\/SbAu22GdPCUzAJ\\\\\\\/Dg1WdpZmTRWyBBmFE8x1kZEQdLShTk1W\\\\\\\/TmqRXNi6ARk7d5wlJl7rZBpgVoinb2gibWbBRxLoXLxAraAhWJ0COyRv9hH66PMNIEgtgxrDQANUjoaW1rTOiMWBlTc044qxQsy\\\\\\\/TKTvg8xzyQ3Zl5C6ecX2AO5BdRWAoLws3sHu8oJ+UOxG47OwN1XR\\\\\\\/qH4rPMtUQpEwjVsZ6TsTRC2XxChJPGNz8yfIZRzK6VbQKfFJ+Fm22Eg7UOTQw\\\\\\\/nSD7Z3tQvT2NCuyQvccyWcK8IdyrL257omvKoIrd5eSX8q20PWFkaJfQICsleyMnxkrPKNiXvWAB7otmzMrgWjA==\\\u0022,\\\u0022iv\\\u0022:\\\u00221k8OxnwGQo\\\\\\\/tNBf+zT+G5A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d4c3521fa36b45d40185fdcf7856097e983f022c466a528a5420dd495a9668b6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485298 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2081" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eEUtqg3EFJFer8VCqjsxQLptH3FimaTjMxEaoSY5rXE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2081" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ml-RcCcS4CIYyA-un8Jc2a0ef3Y4Shjhx9w5IfBqiXs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2065" title="Manipulating the Mitochondrial Genome To Enhance Cattle Embryo Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2095" title="Effects of the Ordering of Natural Selection and Population Regulation Mechanisms on Wright-Fisher Models" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485298" id="node485298" data-pisa="ggg;7/7/2081" data-pisa-master="ggg;g3.117.039016" data-apath="/ggg/7/7/2081.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Validated Bayesian Differentiation of Causative and Passenger Mutations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frederick R. Cross</span>, <span class="highwire-citation-author" data-delta="1">Michal Breker</span> and <span class="highwire-citation-author" data-delta="2">Kristi Lieberman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2081-2094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039016 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.039016" data-atom-uri="/ggg/7/7/2081.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485298" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frederick R. Cross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BR.%2BCross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cross%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BR.%2BCross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fcross@mail.rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">fcross@mail.rockefeller.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michal Breker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michal%2BBreker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Breker%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichal%2BBreker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kristi Lieberman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristi%2BLieberman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lieberman%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristi%2BLieberman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/7/2081" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485298" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485298/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2081.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485298" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485298/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2081.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485298" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485298/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2081.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485298" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485298/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/7/2081.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485298" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485298/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In many contexts, the problem arises of determining which of many candidate mutations is the most likely to be causative for some phenotype. It is desirable to have a way to evaluate this probability that relies as little as possible on previous knowledge, to avoid bias against discovering new genes or functions. We have isolated mutants with blocked cell cycle progression in <em>Chlamydomonas</em> and determined mutant genome sequences. Due to the intensity of UV mutagenesis required for efficient mutant collection, the mutants contain multiple mutations altering coding sequence. To provide a quantitative estimate of probability that each individual mutation in a given mutant is the causative one, we developed a Bayesian approach. The approach employs four independent indicators: sequence conservation of the mutated coding sequence with <em>Arabidopsis</em>; severity of the mutation relative to <em>Chlamydomonas</em> wild-type based on Blosum62 scores; meiotic mapping information for location of the causative mutation relative to known molecular markers; and, for a subset of mutants, the transcriptional profile of the candidate wild-type genes through the mitotic cell cycle. These indicators are statistically independent, and so can be combined quantitatively into a single probability calculation. We validate this calculation: recently isolated mutations that were not in the training set for developing the indicators, with high calculated probability of causality, are confirmed in every case by additional genetic data to indeed be causative. Analysis of “best reciprocal BLAST” (BRB) relationships among <em>Chlamydomonas</em> and other eukaryotes indicate that the temperature sensitive-lethal (Ts-lethal) mutants that our procedure recovers are highly enriched for fundamental cell-essential functions conserved broadly across plants and other eukaryotes, accounting for the high information content of sequence alignment to <em>Arabidopsis</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/causative-mutations" class="hw-term hw-article-keyword hw-article-keyword-causative-mutations" rel="nofollow">causative mutations</a></li><li class="kwd"><a href="/keyword/passenger-mutations" class="hw-term hw-article-keyword hw-article-keyword-passenger-mutations" rel="nofollow">passenger mutations</a></li><li class="kwd"><a href="/keyword/genetic-screen" class="hw-term hw-article-keyword hw-article-keyword-genetic-screen" rel="nofollow">genetic screen</a></li></ul><p id="p-3">The use of model genetic systems to obtain insight into related organisms is well established; as a leading example, yeast genetics has been highly revelatory about fundamental cell biology in animals (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Botstein and Fink 2011</a>). Because fungi diverged from animals considerably after their last common ancestor diverged from the plant lineage (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Rogozin <em>et al.</em> 2009</a>), it is an open question to what extent yeast/animal paradigms will apply to the plant lineage.</p><p id="p-4">To address this question, we initiated a genetic screen for Ts-lethal mutations in the green alga <em>Chlamydomonas reinhardtii</em>, focusing on cell cycle control mutations (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>). The reasoning was that facile microbial genetics and cell culture would facilitate isolation of informative mutants, compared to carrying out a related screen directly in higher plants. A specific feature likely to make such a screen easier in green algae than in higher plants is the high degree of gene duplication in plants, largely due to multiple whole-genome duplications in the plant lineage after divergence from green algae. Loss-of-function genetics is severely hampered by the presence of duplicated sequences; the <em>Chlamydomonas</em> genome is largely (though not entirely) single-copy for protein-coding sequence.</p><p id="p-5">From the broad spectrum of Ts-lethal phenotypes, we concentrated on two classes: mutants that initiated some cytological features of the cell division program but failed to complete division, and mutants with competence at cell growth that were unable to initiate division processes. We called these mutant classes “<em>DIV</em>” and “<em>GEX</em>”, respectively. Cell cycle-related genes based on annotations (<em>e.g.</em>, DNA polymerase subunits) were mainly in the <em>DIV</em> class; <em>GEX</em> genes were more diverse in function based on annotations. Together, these classes represented a few percent of the total Ts-lethal spectrum (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>).</p><p id="p-6">The assumption behind the project, with respect to gaining insight into higher plant genomes, is that evolutionary conservation will likely preserve the function of essential genes, so that genes identified by Ts-lethal screening as essential in <em>Chlamydomonas</em> will be essential (or will be members of essential sequence families) in higher plants. This assumption is plausible but by no means certain. As a trivial example, yeast cell walls are essential, but animal cells lack walls and genes for their production. Additionally, replacement of important cell cycle regulators by entirely unrelated proteins carrying out similar functions is documented comparing yeast and animals (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cross <em>et al.</em> 2011</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Medina <em>et al.</em> 2016</a>).</p><p id="p-7">In our approach, Ts-lethal mutations were induced by UV mutagenesis; identification of likely causative mutations was by next-generation sequencing analysis of bulked segregant pools, which identifies SNPs at or linked to the causative mutation (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>). Two problems remain. First, the sequencing approach identifies the causative mutation in most but not all cases; second, the density of UV-induced mutagenesis needed for efficient screening (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Breker <em>et al.</em> 2016</a>) is such that the causative SNP is frequently very difficult to separate by meiotic recombination from a small number of linked “passenger” SNPs. These problems mean that formally, in no case can we assert with certainty that a given SNP is causative; if it is one of a number of linked candidates, <em>a priori</em> any of them could be causative, and even if it is the only candidate, it is possible that it is a passenger with an unsequenced, truly causative mutation.</p><p id="p-8">To solve this problem, we followed three routes. First, complementation and linkage analysis can demonstrate that we have multiple independent alleles in the same gene. In such a case, if independent mutations altering the same gene model are found by sequence analysis, we assume that these mutations are causative. Second, in many cases we could isolate revertants by selection at high temperature. When sequence analysis showed that the revertants altered one of the gene models hit by the original set of SNPs (typically by exact reversion, or by pseudoreversion at a nearby residue), we assume that this also constitutes definitive identification.</p><p id="p-9">Third, for mutants not covered by either method, we developed a Bayesian method, based on sequence characteristics of the candidate SNPs and linkage analysis. This method was sketched out previously (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>). Here, we specify and develop the exact model. We extend the method, substantially increasing its power, by the addition of a new indicator: transcriptional profiling of candidate genes through the cell cycle. Critically, we validate the method by analysis of new genetic and molecular data, with mutations not in the training set used to develop the indicators. Finally, we use bioinformatics tests, including analysis of BRB-defined sequence families, to show that essential genes in <em>Chlamydomonas</em> are preferentially conserved with respect to higher plant genomes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mutant isolation and mutation detection</h3><p id="p-10">Ts-lethal (“Ts−”) mutants were isolated after UV mutagenesis, and screened for proliferation-specific defects as described (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Breker <em>et al.</em> 2016</a>). Candidate causative mutations were identified by bulked segregant sequence analysis, as described (<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>), or by a new method for parallel bulked segregant sequence analysis in combinatorial pools (M. Breker, K. Lieberman, F.R. Cross, unpublished results). Candidate causative mutations were those that were uniformly mutant in pools of Ts− segregants from a cross to wild-type (or, reciprocally, uniformly wild-type in pools of Ts+ segregants.</p><p id="p-11">We call a candidate causative mutation “definitive” (<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>) if it hits a gene model that is also hit in an independent isolate and the two isolates fall in the same linkage/complementation group, or if we have isolated an intragenic revertant that alters the mutation (true or pseudorevertants). For the present analysis, we employed definitive mutations defined in <a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">Tulin and Cross (2014)</a>. “Passenger” mutations (UV-induced mutations that do not cause the Ts− phenotype) were identified either as those present at &lt; 100% prevalence in pools of Ts− segregants from crosses of various mutants to wild-type, or that were uniform but distinct from the true causative mutation, provided the latter was known “definitively.” In this way, we collected 67 independent definitive causative mutations, described in <a id="xref-ref-14-9" class="xref-bibr" href="#ref-14">Tulin and Cross (2014)</a>, and 137 passenger mutations. These mutations constitute the “training set” for finding Bayesian discriminators.</p><p id="p-12">Importantly for development of the present analysis, identification of the causative and passenger mutations in the training set is entirely based on genetic data, and is independent of annotations (beyond the essential segmentation of the genome into gene models (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blaby <em>et al.</em> 2014</a>), mutational severity, or transcriptional pattern.</p><p id="p-13">For the “test set,” we employed the 20 mutants isolated in <a id="xref-ref-14-10" class="xref-bibr" href="#ref-14">Tulin and Cross (2014)</a> as single members of their complementation groups. All of these mutants were mapped to a chromosomal location relative to physical markers (generally to within 1–2 Mb), and all had varying numbers of candidate mutations for causality within the mapped region. For seven of these mutants, additional mutant screening (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Breker <em>et al.</em> 2016</a>) yielded new alleles based on linkage and complementation testing, and causative mutations were obtained from the genomic sequences of these new mutants as well.</p></div><div id="sec-3" class="subsection"><h3>Linkage mapping</h3><p id="p-14">We carried out meiotic mapping of Ts-lethals mainly employing two methods. First, we developed allele-discriminatory PCR probes (using the competitive approach described by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Onishi <em>et al.</em> (2016)</a>) that allow determination of wild-type or mutant in a single reaction, and tested multiple meiotic segregants (Ts− or Ts+) for marker status. This procedure has the benefit of anchoring the mapping at exactly the site of interest, but the disadvantage that it is hard to test large numbers of progeny. Ts− segregants lacking the mutant marker or Ts+ segregants with the marker are counted as recombinant chromosomes. A second method is tetrad analysis of a mutant against a tester Ts-lethal with a known physical location. We have developed a “microtetrad” approach in which hundreds of tetrads can be rapidly dissected and analyzed on a single plate, by dissecting over an area of ∼1 mm and transferring the plate to restrictive temperature only when meiotic products have just germinated and formed microcolonies. It is then simple to count the number of Ts+ segregants per tetrad to determine PD, NPD, and T tetrad classes.</p><p id="p-15">For analysis, we combine mapping data obtained with independent alleles of the same gene, defined by noncomplementation and lack of recombination in meiosis since the latter property, using our current recombination assay (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Breker <em>et al.</em> 2016</a>), implies no recombination in hundreds or thousands of meioses, and thus very close linkage.</p><p id="p-16">The computation of probability requires specification of a physical region that is assumed to contain the causative mutation. For simplicity, we use as boundaries positions 2 Mb away from the leftmost or rightmost SNPs detected as uniformly mutant in bulked segregant sequence analysis.</p></div><div id="sec-4" class="subsection"><h3>Transcription data</h3><p id="p-17">The cell cycle transcriptome data of <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Zones <em>et al.</em> (2015</a>) consists of RNAseq gene counts of two replicates of a light:dark-synchronized cell cycle. We averaged the replicates and carried out three-point running-average smoothing, then determined the time point when each gene reached its smoothed maximum (“peak time”) and the ratio of the maximum to the minimum of the smoothed series for each gene (“peak-to-trough ratio,” PTR). This simple calculation obviously discards a lot of information, but these simple features are enough to make an effective discriminator.</p></div><div id="sec-5" class="subsection"><h3>BRB analysis</h3><p id="p-18">If two sequences, one from each of two genomes, find each other as their best BLAST hit in cross-genome comparisons, this provides the basis of forming a potential “orthologous” family, containing these two sequences as well as potential recent gene duplicates from within each genome (“in-paralogs”) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Remm <em>et al.</em> 2001</a>). Here, we report analysis of such orthologous families “seeded” by searching with each <em>Chlamydomonas</em> protein as query against seven genomes: <em>Arabidopsis</em>, <em>Brachypodium</em>, and <em>Physcomitrella</em> (dicot, monocot, and moss from the land plants); <em>Homo</em> and <em>Drosophila</em> (two animals); and <em>Saccharomyces</em> and <em>Aspergillus</em> (two fungi). Reference proteomes were downloaded from public-access databases Phytozome, UniProt, SGDB, and ADB.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-19">See Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FileS1.zip">File S1</a> for the MATLAB code. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS1.pptx">Figure S1</a> illustrates how BRB analysis will likely pick out sequences in different genomes originating from a single gene ancestor at the time of the species split. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS2.pptx">Figure S2</a> contains a construction of a theoretical probability distribution for location of a causative mutation based on genetic mapping, and also the assignment of candidate SNPs to either the “causative” probability density function (pdf) derived from mapping, or to a uniform “passenger” pdf. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS3.pptx">Figure S3</a> illustrates the transcriptional pattern of <em>Chlamydomonas</em> genes, segregated by BLAST scores <em>vs.</em> <em>Arabidopsis</em>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS4.pptx">Figure S4</a> is a test of the computation with synthetic data.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>The training set</h3><p id="p-20">To determine useful Bayesian indicators, it is necessary to have a training set of positive and negative examples. To obtain these, we made the assumption that causative mutations were “definitively” (<em>i.e.</em>, ground truth) detected in a specific gene model in the following cases: (1) where multiple alleles in the same gene, defined by failure of genetic complementation and recombination, were shown to have mutational lesions altering coding sequence in the same gene model; and (2) where at least a subset of selected revertants of the mutant were found to have reverted the original lesion in the gene model (either exact or pseudorevertants). In almost all cases, these assignments were also supported by meiotic mapping. These constitute positive examples of causative mutations. The bulked segregant sequence analysis to determine candidate causative mutations (<a id="xref-ref-14-11" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>) also yields sequences of noncausative mutations induced by UV (“passengers”). We consider such mutations to be definitive passengers if they are either linked to but distinct from a definitive causative mutation or if they are on a different chromosome from the causative mutation [note that we usually can definitively identify the chromosome bearing the causative mutation even when the causative mutation itself is not definitively identified, based on uniform detection in the region of mutations (frequently passengers) linked to the causative mutation]. This analysis resulted in a training set of 69 causative and 137 passenger mutations.</p><p id="p-21">The training set is only useful if it is an unbiased sample. In this case, bias appears unlikely. Identification of the causative mutations is based entirely on formal genetic criteria, and is independent of any annotation information (<em>e.g.</em>, alignment or previous information about the gene or its relatives).</p></div><div id="sec-9" class="subsection"><h3>BLAST-detected conservation with Arabidopsis</h3><p id="p-22">The <em>DIV/GEX</em> Ts-lethal mutations identify essential functions, and genes required for essential functions might be more conserved across evolution than nonessential genes. The land plants <em>Arabidopsis</em> and <em>Chlamydomonas</em> diverged ∼1 billion years ago (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Yoon <em>et al.</em> 2004</a>), and their common ancestor diverged from other eukaryotes including animals and yeast ∼1.6 billion years ago. Only 40% of <em>Chlamydomonas</em> genes have detectable BLAST homology over any part of their sequence with <em>Arabidopsis</em>, and this number drops to 30% with a requirement for a BLAST bit-score of at least 100 (data not shown). Thus, conservation with <em>Arabidopsis</em> is a strict criterion, but one that likely allows detection of core machinery conserved in the Viridiplantae lineage.</p><p id="p-23">Retaining protein function through evolution generally requires much greater conservation in some protein regions (<em>e.g.</em>, active sites) than others (<em>e.g.</em>, protein loops and N- and C-termini). The scheme used here (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) subdivides the relationship of mutational position to BLAST results as follows:</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship of mutation to BLAST alignments of Chlamydomonas gene model (C.r) to Arabidopsis (A.t.). (A) Mutation falls within a conserved segment; (B) mutation falls between conserved segments; (C) mutation is N- or C-terminal to a conserved segment; or (D) no conserved segment detected by BLAST. Note that for empirical classification, a mutation in an unconserved residue, in an unaligned residue within a high-scoring pair (conserved segment), or a mutation falling between two conserved segments are all assigned to class B. Rules for truncating mutations (stop codons and splice donor/acceptor mutations; found in a small minority of the causative temperature-sensitive-lethal mutations): if upstream of the C-terminal-most conserved segment, class A; otherwise class B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1560748519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship of mutation to BLAST alignments of &lt;em&gt;Chlamydomonas&lt;/em&gt; gene model (C.r) to &lt;em&gt;Arabidopsis&lt;/em&gt; (A.t.). (A) Mutation falls within a conserved segment; (B) mutation falls between conserved segments; (C) mutation is N- or C-terminal to a conserved segment; or (D) no conserved segment detected by BLAST. Note that for empirical classification, a mutation in an unconserved residue, in an unaligned residue within a high-scoring pair (conserved segment), or a mutation falling between two conserved segments are all assigned to class B. Rules for truncating mutations (stop codons and splice donor/acceptor mutations; found in a small minority of the causative temperature-sensitive-lethal mutations): if upstream of the C-terminal-most conserved segment, class A; otherwise class B.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2081/F1.medium.gif" width="431" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/7/2081/F1.medium.gif" width="431" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485357" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Relationship of mutation to BLAST alignments of <em>Chlamydomonas</em> gene model (C.r) to <em>Arabidopsis</em> (A.t.). (A) Mutation falls within a conserved segment; (B) mutation falls between conserved segments; (C) mutation is N- or C-terminal to a conserved segment; or (D) no conserved segment detected by BLAST. Note that for empirical classification, a mutation in an unconserved residue, in an unaligned residue within a high-scoring pair (conserved segment), or a mutation falling between two conserved segments are all assigned to class B. Rules for truncating mutations (stop codons and splice donor/acceptor mutations; found in a small minority of the causative temperature-sensitive-lethal mutations): if upstream of the C-terminal-most conserved segment, class A; otherwise class B.</p><div class="sb-div caption-clear"></div></div></div><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-25">Class A: mutation falls within a segment of BLAST alignment (high-scoring pair or HSP), and the mutation alters a conserved residue within this segment [“conserved” is defined operationally as follows: the Blosum62 score (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Henikoff and Henikoff 1993</a>) between <em>Arabidopsis</em> and <em>Chlamydomonas</em> at the position is &gt; 0 (meaning that conservation is observed more often than would be expected by chance)].</p></li><li id="list-item-2"><p id="p-26">Class B: mutation falls within an overall conserved region, but alters an unconserved residue; is BLAST-aligned across a small deletion in the <em>Arabidopsis</em> sequence in the HSP; or is found between two distinct HSPs (these three distinct possibilities are joined into one category to prevent excessive slicing of the training set, and because in all cases the mutation is to an unconserved residue but is surrounded by regions of conservation)].</p></li><li id="list-item-3"><p id="p-27">Class C: mutation is N-terminal or C-terminal to all detected HSPs.</p></li><li id="list-item-4"><p id="p-28">Class D: no <em>Arabidopsis</em> BLAST hit.</p></li></ul><p id="p-29">The order A &gt; B &gt; C &gt; D seems plausible for likely disruption of conserved protein structure/function. While finer-grained classifications are possible, it is important to keep the number small enough that the training set can provide reasonable occupancy of the bins.</p><p id="p-30">The distributions of causative and passenger mutations from the training sets among these classes showed sharp differentiation (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, top left). Most notable was the extreme enrichment of classes A and B among causative mutations, indicating a strong correlation between essential functional regions of key proteins, and conserved alignment through evolution.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Constructing Bayesian classifiers. Mutational Blosum scores, BLAST category, and BLAST/Blosum category distribution (top left, top right, and bottom left) for definitive DIV/GEX vs. coding sequence-changing passenger mutations (the training set). Lower right: BLAST/Blosum categories computed by multiplication of individual Blosum and BLAST probabilities. Near identity of the two lower graphs indicates independence of these measures. DIV, mutants that initiated some cytological features of the cell division program but failed to complete division; GEX, mutants with competence at cell growth that were unable to initiate division processes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1560748519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Constructing Bayesian classifiers. Mutational Blosum scores, BLAST category, and BLAST/Blosum category distribution (top left, top right, and bottom left) for definitive &lt;em&gt;DIV/GEX&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; coding sequence-changing passenger mutations (the training set). Lower right: BLAST/Blosum categories computed by multiplication of individual Blosum and BLAST probabilities. Near identity of the two lower graphs indicates independence of these measures. &lt;em&gt;DIV&lt;/em&gt;, mutants that initiated some cytological features of the cell division program but failed to complete division; &lt;em&gt;GEX&lt;/em&gt;, mutants with competence at cell growth that were unable to initiate division processes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2081/F2.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/7/2081/F2.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485434" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Constructing Bayesian classifiers. Mutational Blosum scores, BLAST category, and BLAST/Blosum category distribution (top left, top right, and bottom left) for definitive <em>DIV/GEX</em> <em>vs.</em> coding sequence-changing passenger mutations (the training set). Lower right: BLAST/Blosum categories computed by multiplication of individual Blosum and BLAST probabilities. Near identity of the two lower graphs indicates independence of these measures. <em>DIV</em>, mutants that initiated some cytological features of the cell division program but failed to complete division; <em>GEX</em>, mutants with competence at cell growth that were unable to initiate division processes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Best reciprocal BLAST analysis</h3><p id="p-32">BLAST alignments can reflect a range of degree of relationship. High, end-to-end similarity might reflect orthologous function. However, many alignments are due solely to conservation of a common small protein domain (<em>e.g.</em>, WD40-repeats). One way to discriminate among BLAST hits is best reciprocal BLAST analysis (BRB): when protein X from genome A finds protein Y from genome B as one of the best hits, and reciprocally, protein Y from genome B has protein X from genome A as one of the best hits (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Remm <em>et al.</em> 2001</a>). Frequently, there are other sequences in genome A, more similar to X than is protein Y from genome B; these may be derived from duplication of X after separation of lineage A from lineage B (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS1.pptx">Figure S1</a>). We have carried out reciprocal BLAST analysis along these lines between <em>Chlamydomonas</em> and a range of other eukaryotes: three representative land plants: <em>Arabidopsis</em>, a dicot; <em>Brachypodium</em>, a monocot; <em>Physcomitrella</em>, a moss; two animals, humans, and <em>Drosophila</em>; and two fungi, <em>Aspergillus</em> and <em>Saccharomyces</em>.</p><p id="p-33">To validate the BRB search, we note that <em>Chlamydomonas</em> should be approximately equally diverged from <em>Arabidopsis</em> (a dicot), <em>Brachypodium</em> (a monocot), and <em>Physcomitrella</em> (a moss), since these are all in the higher land plant lineage with a single divergence time from <em>Chlamydomonas</em>. Consistent with this idea, a Venn diagram shows very strong overlap between <em>Chlamydomonas</em> proteins identified as BRB family members using these three genomes as targets (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, top left). This set of 2818 orthologous families may be broadly distributed among Viridiplantae.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BRB analysis was carried out with the Chlamydomonas proteome as query, against Arabidopsis thaliana, Brachypodium distachyon, Physcomitrella patens, Homo sapiens, Drosophila melanogaster, Saccharomyces cerevisiae, and Aspergillus niger genomes (three plants, two animals, and two fungi). Top left: overlap of identity of Chlamydomonas genes in BRB-orthologous families with the three plant genomes. 2818 Chlamydomonas genes are in such families with proteins from all three plant proteomes. Top right: overlap of Chlamydomonas genes in BRB-orthologous families with all three plant proteomes, both animal and both fungal genomes. Below: proportion of total Chlamydomonas genes (blue) and definitive DIV/GEX genes (red; Tulin and Cross 2014) in the overlap classes shown at top. A.th, Arabidopsis thaliana; B.di, Brachypodium distachyon; BRB, best reciprocal BLAST; Cre, Chlamydomonas; DIV, mutants that initiated some cytological features of the cell division program but failed to complete division; GEX, mutants with competence at cell growth that were unable to initiate division processes; P.pa; Physcomitrella patens." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1560748519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;BRB analysis was carried out with the &lt;em&gt;Chlamydomonas&lt;/em&gt; proteome as query, against &lt;em&gt;Arabidopsis thaliana&lt;/em&gt;, &lt;em&gt;Brachypodium distachyon&lt;/em&gt;, &lt;em&gt;Physcomitrella patens&lt;/em&gt;, &lt;em&gt;Homo sapiens&lt;/em&gt;, &lt;em&gt;Drosophila melanogaster&lt;/em&gt;, &lt;em&gt;Saccharomyces cerevisiae&lt;/em&gt;, and &lt;em&gt;Aspergillus niger&lt;/em&gt; genomes (three plants, two animals, and two fungi). Top left: overlap of identity of &lt;em&gt;Chlamydomonas&lt;/em&gt; genes in BRB-orthologous families with the three plant genomes. 2818 &lt;em&gt;Chlamydomonas&lt;/em&gt; genes are in such families with proteins from all three plant proteomes. Top right: overlap of &lt;em&gt;Chlamydomonas&lt;/em&gt; genes in BRB-orthologous families with all three plant proteomes, both animal and both fungal genomes. Below: proportion of total &lt;em&gt;Chlamydomonas&lt;/em&gt; genes (blue) and definitive &lt;em&gt;DIV/GEX&lt;/em&gt; genes (red; Tulin and Cross 2014) in the overlap classes shown at top. A.th, &lt;em&gt;Arabidopsis thaliana&lt;/em&gt;; B.di, &lt;em&gt;Brachypodium distachyon&lt;/em&gt;; BRB, best reciprocal BLAST; Cre, &lt;em&gt;Chlamydomonas&lt;/em&gt;; &lt;em&gt;DIV&lt;/em&gt;, mutants that initiated some cytological features of the cell division program but failed to complete division; &lt;em&gt;GEX&lt;/em&gt;, mutants with competence at cell growth that were unable to initiate division processes; P.pa; &lt;em&gt;Physcomitrella patens&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2081/F3.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/7/2081/F3.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485362" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">BRB analysis was carried out with the <em>Chlamydomonas</em> proteome as query, against <em>Arabidopsis thaliana</em>, <em>Brachypodium distachyon</em>, <em>Physcomitrella patens</em>, <em>Homo sapiens</em>, <em>Drosophila melanogaster</em>, <em>Saccharomyces cerevisiae</em>, and <em>Aspergillus niger</em> genomes (three plants, two animals, and two fungi). Top left: overlap of identity of <em>Chlamydomonas</em> genes in BRB-orthologous families with the three plant genomes. 2818 <em>Chlamydomonas</em> genes are in such families with proteins from all three plant proteomes. Top right: overlap of <em>Chlamydomonas</em> genes in BRB-orthologous families with all three plant proteomes, both animal and both fungal genomes. Below: proportion of total <em>Chlamydomonas</em> genes (blue) and definitive <em>DIV/GEX</em> genes (red; <a id="xref-ref-14-12" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>) in the overlap classes shown at top. A.th, <em>Arabidopsis thaliana</em>; B.di, <em>Brachypodium distachyon</em>; BRB, best reciprocal BLAST; Cre, <em>Chlamydomonas</em>; <em>DIV</em>, mutants that initiated some cytological features of the cell division program but failed to complete division; <em>GEX</em>, mutants with competence at cell growth that were unable to initiate division processes; P.pa; <em>Physcomitrella patens</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">In the <em>Chlamydomonas</em> <em>vs.</em> <em>Arabidopsis</em> search, most candidate ortholog families contain single members in <em>Chlamydomonas</em>, but frequently contain multiple members in <em>Arabidopsis</em> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). This may be due to ancient gene or genome duplications in the land plant lineage (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams and Wendel 2005</a>) after the split from green algae. Similar results for gene family sizes were obtained in comparisons of <em>Chlamydomonas</em> to <em>Brachypodium</em> and <em>Physcomitrella</em> (data not shown).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene duplicates in BRB-defined orthologous families. Orthologous families containing Cre and At members were sorted by which proteome contributed more members to the family (top). Left: Cre &gt; At; middle: same numbers from each; right: At &gt; Cre. Bottom: mean and SEM of the number of family members from each proteome, in each case. A.t, Arabidopsis thaliana; BRB, best reciprocal BLAST; Cre, Chlamydomonas." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1560748519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene duplicates in BRB-defined orthologous families. Orthologous families containing Cre and At members were sorted by which proteome contributed more members to the family (top). Left: Cre &gt; At; middle: same numbers from each; right: At &gt; Cre. Bottom: mean and SEM of the number of family members from each proteome, in each case. A.t, &lt;em&gt;Arabidopsis thaliana&lt;/em&gt;; BRB, best reciprocal BLAST; Cre, &lt;em&gt;Chlamydomonas&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2081/F4.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/7/2081/F4.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485353" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Gene duplicates in BRB-defined orthologous families. Orthologous families containing Cre and At members were sorted by which proteome contributed more members to the family (top). Left: Cre &gt; At; middle: same numbers from each; right: At &gt; Cre. Bottom: mean and SEM of the number of family members from each proteome, in each case. A.t, <em>Arabidopsis thaliana</em>; BRB, best reciprocal BLAST; Cre, <em>Chlamydomonas</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Eight-hundred and nine <em>Chlamydomonas</em> genes were in orthologous families with family members in all seven genomes tested (three plant, two animal, and two fungal); these may be near universal among eukaryotes.</p></div><div id="sec-11" class="subsection"><h3>DIV/GEX gene enrichment in the BRB gene sets</h3><p id="p-38">In carrying out this mutant hunt (<a id="xref-ref-14-13" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>), the phenotypic sorting of mutants, and concentration on the “<em>DIV/GEX</em>” class, was based on the idea that genes with these mutant phenotypes might specifically identify core conserved cellular functions. Confirming this idea, while only 26% of <em>Chlamydomonas</em> genes are in a BRB-defined orthologous family with any of the seven genomes tested, 79% of <em>DIV</em> and <em>GEX</em> genes are in a BRB family with a gene from at least one of the test genomes. This enrichment is even greater in the plant-orthologous families (76% <em>vs.</em> 16%), and is especially strong in the plant–animal–fungi orthologous families: (44% <em>vs.</em> 5%) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). This finding strongly suggests that the <em>DIV/GEX</em> class is strongly enriched for deeply conserved functions, dating back to the LCA of plants and Opisthokonts [thus, near to the eukaryotic LCA (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Rogozin <em>et al.</em> 2009</a>)].</p></div><div id="sec-12" class="subsection"><h3>Mutational severity</h3><p id="p-39">Amino acid substitutions vary in their potential to disrupt protein function. To measure the likely severity of effect of substitutions caused by UV-induced mutations, we use the Blosum62 score (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Henikoff and Henikoff 1993</a>). Blosum scores have been shown to perform well compared to most other measures for the determination of mutational severity (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Yampolsky and Stoltzfus 2005</a>). Distributions of this score for causative and passenger mutations are broad, but there is a clear shift of the causative mutations to more negative (more severe) Blosum62 scores. Empirically, a cutoff of score &lt; −1 gives a good separator between most causative and most passenger mutations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, top right).</p></div><div id="sec-13" class="subsection"><h3>“BLAST/Blosum” combined index</h3><p id="p-40">To make a simple discriminator, the BLAST classes A–D were combined with the severity index (mutation has Blosum &lt; −1, or ≥ −1) to make eight classes. Notably, the occupancy of these classes by causative and passenger mutations was almost exactly that expected from multiplication of the marginals, that is, the BLAST criterion and the mutational severity criterion were almost entirely independent (compare <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> lower left and lower right). The overall discriminatory power of this index is high, with log10-likelihood ratios ranging from 0.9 for category 1 to −1.4 for category 8, which is up to a 200-fold differential.</p><p id="p-41">Blosum scores are used for both aspects of the BLAST/Blosum categorization, but in different ways. The “mutational Blosum” is for the comparison of wild-type and mutant <em>Chlamydomonas</em> genes, while the BLAST-related score is for the comparison of wild-type <em>Chlamydomonas</em> to wild-type <em>Arabidopsis</em>. These two Blosum scores can, thus, be high or low independently of each other [<em>e.g.</em>, mutation of a Leu to Ile in a residue that is Pro in <em>Arabidopsis</em> (mutation severe, conservation low) or mutation of a Leu to Pro in a residue that is also Leu in <em>Arabidopsis</em> (mutation nonsevere, conservation high)]. The observation that the theoretical distribution derived by multiplication of marginals in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> lower right is almost identical to the actual distribution is empirical proof that these criteria are indeed independent.</p><p id="p-42">The distribution of passenger mutations among these categories was similar to that of mutations randomly generated <em>in silico</em> (data not shown), suggesting that the passenger training set is largely reflective of the initially generated mutational spectrum, with little effective selection operating against most mutations over the short term of these experiments. This computational finding implies that the distribution of passenger mutations among these categories is robust, and not a statistical fluke or an artifactual consequence of the training set selection procedure. <em>A priori</em> one might expect <em>in silico</em>-generated random mutations to have a higher content of severe mutations in conserved regions, since the actual passengers were subject to <em>in vivo</em> negative selection. This probably reflects the severe population bottleneck involved in a mutant hunt, where all that is required is proliferation of a mutated cell to make a moderate-sized colony, in the essential absence of competition.</p></div><div id="sec-14" class="subsection"><h3>Construction of a formal Bayesian model</h3><p id="p-43">The full presentation is in the <em>Appendix</em>. Briefly, suppose there are <em>N</em> candidate SNPs. We assume the causative mutation is a single-gene lesion (since a large majority of Ts-lethals segregate 2:2 in tetrad analysis). Note that, in the present experimental context, <em>N</em> is generally not large, since the bulked segregant sequencing approach (<a id="xref-ref-14-14" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>) will rule out most of the mutations in the original mutant since the wild-type and mutant alleles will both be detected (typically ∼50% each) in sequence from a pool of ∼10 Ts segregants from a cross to wild-type. This reduces <em>N</em> from hundreds to single digits.</p><p id="p-44">Bulked segregant sequencing fails to detect <em>any</em> candidate causative SNPs in a minority of cases (<a id="xref-ref-14-15" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>) (discussed further below). The reason for this is presently unknown; a simple explanation would be if the mutated gene is not present in the assembled genome (which is known to have gaps), but this is unlikely to be the complete explanation (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Tulin and Cross 2016</a>). Call U the probability [which we estimate at ∼25% (<a id="xref-ref-14-16" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>)] that the causative mutation escaped detection by sequencing and therefore is none of the <em>N</em> candidates. If the causative mutation was detected by sequencing, we assume that it is exactly one of the <em>N</em> SNPs.</p><p id="p-45">From <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, define B<sub>i</sub> as the likelihood ratio for SNP<sub>i</sub>: the probability that a causative mutation is in the BLAST/Blosum class of the SNP, divided by the probability that the passenger mutation is in this class. Define Q as the likelihood ratio of unsequenceability U/(1−U).</p><p id="p-46">Then, the probability that one specific SNP<sub>i</sub> is causative, given the BLAST/Blosum characteristics of the collection of N SNPs, by Bayes’ theorem (see Supplemental Material for detailed derivation), is:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-1.gif"/></noscript></span></span>The probability that the causative mutation escaped detection by sequencing is:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-2.gif"/></noscript></span></span>Mutations with high B<sub>i</sub> (such as severe mutations in conserved residues) are more likely to be causative. Increasing numbers of candidate mutations (higher <em>N</em>) decreases the likelihood that any individual one is causative. Conversely, the likelihood that the causative mutation escaped detection by sequencing decreases with increasing numbers of SNPs with high B<sub>i</sub>.</p></div><div id="sec-15" class="subsection"><h3>Mapping information</h3><p id="p-47">In principle, meiotic mapping could reduce the interval carrying the causative mutation to an arbitrarily small size. However, meiotic distances are in centimorgans. Conversion to physical distance (needed for causative SNP identification) requires knowing the centimorgan/megabase conversion ratio. This is known on average to be ∼10 cM/Mb (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Merchant 2007</a>; <a id="xref-ref-14-17" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>), but it is well known in many organisms that this ratio is variable across the genome due to hot- and cold-spots for recombination. We have found one region of ∼0.5 Mb across which there is no detectable recombination (no recombinants in at least 1000 meioses; F. R. Cross, unpublished results); this is an extreme case, but surely not the sole such interval.</p><p id="p-48">However, in cases where candidate mutations are separated by 1 Mb or greater, meiotic mapping can provide discriminatory power, using the mapping strategies described in the <em>Materials in Methods</em>. It is useful to translate such mapping results into estimated probabilities for physical location of the causative mutation. The full development of these estimates is described in the Supplemental Material. Briefly, we assume a normal-like distribution of probable location of the causative mutation, with the mean at the best estimate (measured centimorgans from the known marker * 0.1 Mb/cM), and SD 0.5 Mb [based on the maximum nonrecombining interval detected, and on apparent measured error (deviation from the 0.1 Mb/cM average) in many such mapping experiments (<a id="xref-ref-14-18" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>)]. Passenger mutations have probability density that is uniform across the known possible region (typically a chromosome or chromosome arm). Then, if L<sub>i</sub> is the location of SNP<sub>i</sub>, g(L<sub>i</sub>) is a scaled relative probability density at this position, and Q is the likelihood ratio that the causative mutation was not sequenced [that is, if U is the probability of unsequenceability, Q = (U/(1−U)], then the probability that of <em>N</em> SNPs, SNP<sub>i</sub> is causative is:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-3.gif"/></noscript></span></span>where L<sub>i</sub> is the location of SNPi, and g is an appropriately scaled probability density function intended to conservatively represent plausible locations of the causative SNP [derivation of g (Supplemental Material) essentially assumes that a given mapping result is equivalent to an approximately normal distribution for probability of true location, with mean the best estimated location, and SD based on mapping error).</p><p id="p-49">Chromosomal location is almost surely independent of BLAST/Blosum values since, like most eukaryotes, <em>Chlamydomonas</em> exhibits broad dispersal of functionally-related genes across chromosomes. Independent probabilities multiply; therefore, BLAST/Blosum information can be integrated with mapping information to produce a single probability of causality for each candidate SNP:</p><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-4.gif"/></noscript></span></span></div><div id="sec-16" class="subsection"><h3>Transcriptional regulation</h3><p id="p-50"><em>Chlamydomonas</em> exhibits very strong differential transcription through its mitotic cycle. In particular, many genes, including many that are probably specifically required for DNA replication and cell division, are induced by huge factors (&gt; 100-fold) in S/M-phase cells compared to newborn G1 cells (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Tulin and Cross 2015</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Zones <em>et al.</em> 2015</a>). The mutations we isolated previously as blocking cell cycle progression (<a id="xref-ref-14-19" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>) were separated into two broad phenotypic categories: “<em>div</em>” mutations, that showed evidence of entry into the replicative cycle followed by arrest, and “<em>gex</em>” mutants, that showed no signs of even initial replication/division processes. It was noted (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Tulin and Cross 2015</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Zones <em>et al.</em> 2015</a>) that <em>DIV</em> genes, but not <em>GEX</em> genes, were highly likely to exhibit the transcriptional pattern noted above, of huge induction in S/M phase. Since this induction is observed with only a small proportion of genes overall, this provides another plausible Bayesian discriminator, specifically for the <em>DIV</em> class of genes.</p><p id="p-51">To set up such a discriminator, we made use of a cell cycle transcriptome derived from RNAseq of two replicates of a light:dark-synchronized cell cycle (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Zones <em>et al.</em> 2015</a>). We converted data for every gene into two numbers: PTR and time of peak (T) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Almost 80% of genes are at least twofold differentially regulated through the timecourse, as previously reported (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Zones <em>et al.</em> 2015</a>). It is also notable that peak times are mostly in three clusters: early [∼6 hr, before commitment to a replicative cell cycle (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Zones <em>et al.</em> 2015</a>)], middle (∼13 hr, in the middle of the S/M cycles), and late (20–24 hr, as newborn cells mature and hatch). <em>DIV</em> genes identified in <a id="xref-ref-14-20" class="xref-bibr" href="#ref-14">Tulin and Cross (2014)</a> are marked on the figure. Most <em>DIV</em> genes are expressed in the S/M period, at very high PTR, clearly separated from the large majority of other genes (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Tulin and Cross 2015</a>; <a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Zones <em>et al.</em> 2015</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The DIV-like transcriptional pattern. (A) Cell cycle transcriptional data of Zones et al. (2015) was analyzed: two timecourses were averaged, and a three-timepoint running average calculated. Two values were extracted: peak time (time point at which the plot was maximal), x-axis; and PTR (highest level divided by lowest level), y-axis. The heat map (scale at right) shows the complete gene set. Placement of def and non-def DIV genes on the plot is indicated by green and blue circles. (B) proportion of DIV genes and of all genes in two bins of PTR/peak expression time, the &#x201C;S/M&#x201D;-like pattern (PTR &#x2265; 4, peak time 12&#x2013;14 hr; Zones et al. 2015), and all other patterns. def, definitive; DIV, mutants that initiated some cytological features of the cell division program but failed to complete division; GEX, mutants with competence at cell growth that were unable to initiate division processes; non-def, nondefinitive; PTR, peak-to-trough ratio." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1560748519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &lt;em&gt;DIV&lt;/em&gt;-like transcriptional pattern. (A) Cell cycle transcriptional data of Zones &lt;em&gt;et al.&lt;/em&gt; (2015) was analyzed: two timecourses were averaged, and a three-timepoint running average calculated. Two values were extracted: peak time (time point at which the plot was maximal), &lt;em&gt;x&lt;/em&gt;-axis; and PTR (highest level divided by lowest level), &lt;em&gt;y&lt;/em&gt;-axis. The heat map (scale at right) shows the complete gene set. Placement of def and non-def &lt;em&gt;DIV&lt;/em&gt; genes on the plot is indicated by green and blue circles. (B) proportion of &lt;em&gt;DIV&lt;/em&gt; genes and of all genes in two bins of PTR/peak expression time, the &#x201C;S/M&#x201D;-like pattern (PTR &#x2265; 4, peak time 12&#x2013;14 hr; Zones &lt;em&gt;et al.&lt;/em&gt; 2015), and all other patterns. def, definitive; &lt;em&gt;DIV&lt;/em&gt;, mutants that initiated some cytological features of the cell division program but failed to complete division; &lt;em&gt;GEX&lt;/em&gt;, mutants with competence at cell growth that were unable to initiate division processes; non-def, nondefinitive; PTR, peak-to-trough ratio.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2081/F5.medium.gif" width="376" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/7/2081/F5.medium.gif" width="376" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2081/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485543" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-52" class="first-child">The <em>DIV</em>-like transcriptional pattern. (A) Cell cycle transcriptional data of <a id="xref-ref-19-8" class="xref-bibr" href="#ref-19">Zones <em>et al.</em> (2015)</a> was analyzed: two timecourses were averaged, and a three-timepoint running average calculated. Two values were extracted: peak time (time point at which the plot was maximal), <em>x</em>-axis; and PTR (highest level divided by lowest level), <em>y</em>-axis. The heat map (scale at right) shows the complete gene set. Placement of def and non-def <em>DIV</em> genes on the plot is indicated by green and blue circles. (B) proportion of <em>DIV</em> genes and of all genes in two bins of PTR/peak expression time, the “S/M”-like pattern (PTR ≥ 4, peak time 12–14 hr; <a id="xref-ref-19-9" class="xref-bibr" href="#ref-19">Zones <em>et al.</em> 2015</a>), and all other patterns. def, definitive; <em>DIV</em>, mutants that initiated some cytological features of the cell division program but failed to complete division; <em>GEX</em>, mutants with competence at cell growth that were unable to initiate division processes; non-def, nondefinitive; PTR, peak-to-trough ratio.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">The number of definitively identified <em>DIV</em> genes is not large, so the data will not at present support a fine-grained analysis. Around seventy percent of <em>DIV</em> genes have T between 12 and 13.5 hr and PTR ≥ 2, while this category includes only 13% of the total genes. We used this criterion as a binary discriminator to detect a “<em>DIV</em>-like” pattern (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Importantly, this criterion was established using only the definitively identified <em>DIV</em> genes from <a id="xref-ref-14-21" class="xref-bibr" href="#ref-14">Tulin and Cross (2014)</a>, although it is evident that the broader class of probable <em>DIV</em> genes follows the same pattern (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). This criterion provides another Bayesian test. Call “T<sub>i</sub>” the transcriptional likelihood ratio (<em>DIV</em>/all genes) for the gene in which SNP<sub>i</sub> is found (two classes, <em>DIV</em>-like or not):<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-5.gif"/></noscript></span></span>Perhaps surprisingly, the cell cycle transcriptional pattern is largely independent of BLAST homology to <em>Arabidopsis</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS3.pptx">Figure S3</a>), and is surely independent of mutational Blosum scores for individual SNPs. Absence of functional clustering implies that the transcriptional pattern is likely also independent of map position. Therefore, the test for transcriptional category can be combined multiplicatively with BLAST/Blosum scores and mapping information to yield an integrated probability that a given SNP is causative for a mutation in the <em>div</em> phenotypic class:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-6.gif"/></noscript></span></span>The transcriptional regulation test is more preliminary than the BLAST/Blosum and mapping tests, because it is based on only a small training set and is restricted to a single phenotypic class. This will doubtless improve as more mutants are defined, growing the training sets for <em>div</em> and for other phenotypic classes. Initial analysis supported some subclustering among the <em>gex</em> phenotypic class of genes, for example (<a id="xref-ref-19-10" class="xref-bibr" href="#ref-19">Zones <em>et al.</em> 2015</a>).</p><p id="p-54">Thus, given the results of BLAST for the candidate gene against <em>Arabidopsis</em>, the severity and location of the mutation, any available mapping data, and (for <em>DIV</em>-class mutations) the transcriptional pattern, a Bayesian probability of causality for each SNP can be calculated directly from the equation above. The values in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a> generate the likelihoods, and the computational procedure outlined in the Appendix converts mapping data for each mutant also into likelihood (see <em>Appendix</em>; MATLAB code is provided in Supplemental Material to carry out this calculation). U, the probability of unsequenceability or failure to detect a Ts-lethal lesion by sequencing, is estimated at a flat 25% (<a id="xref-ref-14-22" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>).</p></div><div id="sec-17" class="subsection"><h3>Validation of the approach</h3><p id="p-55">To test the accuracy of the derivation, we generated <em>in silico</em> “mutants” with varying numbers of candidates over a region, where the causative and passenger mutations had the empirically observed distributions of indicators, the causative mutation had 25% probability of unsequenceability, and linkage data were generated by explicit simulation of crossover generation. We then calculated the probability of being causative for each SNP in this set. The assigned probability was accurate, and the detection had high sensitivity and selectivity based on ROC curves (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS4.pptx">Figure S4</a>).</p><p id="p-56">This finding shows that the method has potential as a strong discriminator, but provides no independent validation of the approach. To accomplish this, we made use of new mutant isolation data, generated since the publication of the training set in <a id="xref-ref-14-23" class="xref-bibr" href="#ref-14">Tulin and Cross (2014)</a>. In <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, we report the calculated Bayesian probabilities for causality for candidate SNPs found in genome sequences of mutants in 20 complementation groups. These complementation groups were identified in <a id="xref-ref-14-24" class="xref-bibr" href="#ref-14">Tulin and Cross (2014)</a> by single mutant alleles, therefore causative mutation identification was not considered definitive. Importantly, for this reason, none of the mutants or SNPs in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> were used for the training set for BLAST/Blosum values or transcriptional pattern.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485370/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485370/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Bayesian testing with mutations in 20 genes not in the training set</span><div class="sb-div caption-clear"></div></div></div><p id="p-58">Tests in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> used the BLAST/Blosum score, linkage, or transcriptional pattern, alone or in all combinations, using the equations above (source data in Supplemental Material). Note that while the initial mutants contained large numbers (10–100) of coding sequence-changing mutations, the bulked segregant sequencing strategy (<a id="xref-ref-14-26" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>), and in some cases additional mapping crosses, whittles these down to a much smaller number, based on the principle that temperature-sensitive segregants WITHOUT some SNP, or temperature-resistant segregants WITH some SNP, eliminate that SNP from consideration for causality (we take this as ground truth). For each mutant background, all remaining candidate SNPs are tested in the table. The calculation is carried out as described above with input for each SNP: BLAST/Blosum category 1–8, transcriptional category (<em>DIV</em>-like or not), and linkage data (marker location and numbers of recombinant/nonrecombinant progeny).</p><p id="p-59">In ∼85% of cases, a single SNP is identified as a strongly preferred candidate from each mutant background (calculated probability 0.96 ± 0.05). In mutants with multiple “competing” SNPs, probability distributions are in general strongly bimodal: one high-probability SNP (usually <em>P</em> &gt; 0.9), with other SNPs assigned probabilities of 0.00–0.10. These two clusters most likely represent causative mutations and passengers.</p><p id="p-60">The three criteria (BLAST/Blosum, linkage, and transcriptional pattern) interact. For the presumed causative mutations (high probability), the single, double, and triple tests gave probability estimates of 0.75 ± 0.24, 0.88 ± 0.16, and 0.96 ± 0.05 (mean ± SD), respectively; thus, combining tests increased the probability estimates. For the presumed passenger mutations, the estimates for single, double, and triple tests were 0.15 ± 0.16, 0.07 ± 0.13, and 0.02 ± 0.03, respectively; combining tests decreased the probability estimates. This is expected if the presumed causative mutations are likely to share all three attributes, while random passenger mutations might fortuitously score high for one but probably not for another one. Thus, the multiple tests interacted to drive divergence in probability estimates between the presumed causative mutations and passengers, resulting in more reliable specific detection. For <em>gex</em> mutations, the transcription test is not applicable; nevertheless, the BLAST/Blosum and linkage tests interacted in the same way to give high probability identification in most cases.</p><p id="p-61">Critically, new mutant isolation (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Breker <em>et al.</em> 2016</a>; Breker <em>et al.</em>, unpublished results) has resulted in definitive determination of 13 causative mutations falling in six genes (<em>div43</em>, <em>div45</em>, <em>div46</em>, <em>div48</em>, <em>div50</em>, and <em>div72</em>) (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). This determination is based on the criterion (<a id="xref-ref-14-27" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>; see above) that lesions in the same gene model, found in multiple independent isolates in the same complementation group, definitively identify the causative target gene. We also consider the assignment of <em>DIV68</em> to the sole profilin homolog in <em>Chlamydomonas</em> to be definitive, since the <em>div68-1</em> mutation results in almost complete loss of profilin detectable by Western blotting, and also has multiple phenotypes consistent with loss of actin function, as expected for profilin inactivation (M. Onishi, personal communication). Thus, 14 mutations, newly determined to be causative, have high calculated Bayesian probabilities (0.93 ± 0.14).</p><p id="p-62">These results contrast with those for <em>DIV40</em>. <em>div40-1</em> contains a single candidate coding sequence-changing SNP; however, it is a very weak candidate (similar in probability estimates to the collection of known passenger mutations). We have now isolated a second independent allele <em>div40-2</em> (defined as allelic since <em>div40-1</em> and <em>div40-2</em> fail to complement in transheterozygous diploids, are both located on the left arm of chromosome 17, and fail to recombine with each other in hundreds of meioses.) However, bulked segregant sequencing of <em>div40-2</em> revealed no coding sequence-changing mutation that was uniformly present in Ts− segregants (data not shown), and no mutation at all within hundreds of kilobases of the <em>div40-1</em> candidate. Therefore, we suspect that <em>both</em> the <em>div40-1</em> and <em>div40-2</em> causative mutations escaped detection by sequencing, and that the single candidate for <em>div40-1</em> is indeed a passenger.</p><p id="p-63">This phenomenon, in which multiple independent alleles in a well-defined complementation group fail to share lesions in any one gene model (or have no candidate lesions at all), was observed previously and was examined with care in the cases of <em>div14</em> and <em>div16</em> (<a id="xref-ref-14-28" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>). Four alleles of <em>div14</em> were mapped in multiple crosses and by complementation testing to the same position on chromosome 4 (∼4 Mb on the physical map), and five alleles of <em>div16</em> to chromosome 10 at ∼6.5 Mb (data not shown). Bulked segregant sequencing to high coverage (&gt; 200 × in the cases of one allele each of <em>div14</em> and <em>div16</em>, and at least 50 × coverage of the other seven independent alleles) failed to reveal the causative mutations. We call this phenomenon unsequenceability, and we estimated previously that ∼25% of Ts-lethal mutations fall into unsequenceable genes (<a id="xref-ref-14-29" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>). We do not know the explanation for this problem, but it is taken into account in the Bayesian calculation (<em>Appendix</em>). The results on <em>div40</em> suggest that it may be in this class. This interpretation also supports the value of the Bayesian calculation, since a possible candidate for causation of <em>div40-1</em> had a low calculated probability, and was subsequently found to be likely a passenger to an unsequenced true causative mutation.</p><p id="p-64"><em>div53-1</em> and <em>div60-1</em> mutants are intermediate cases; they contain stronger candidates for causality than <em>div40-1</em>, but the candidates are outliers relative to the larger population of presumed causative mutations. The causative mutation in these strains may be atypical with respect to the “rules” followed by the training set. These intermediate cases quantitatively identify mutants for which identification is not strong, and further data are required before relying on the identification for additional experiments.</p><p id="p-65">The data are clearly insufficient for a full analysis of efficiency of the test, but qualitatively the empirical results follow the shape of the curves derived from simulated data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS4.pptx">Figure S4</a>): high calculated Bayesian probability correlates with high true-positive proportion (0.93 ± 0.13, 14 examples), while low calculated Bayesian probability may correlate with a false-positive (0.04 for the sole candidate in <em>div40-1</em>; effectively zero for <em>div40-2</em> because of the absence of any candidate).</p><p id="p-66">These results constitute strong validation, since the approach developed with the initial training set of definitive causative mutation generalizes to mutants not in the training set, and in most cases yields a strong preferential identification of a single highly likely causative mutation, even in backgrounds also carrying multiple passenger mutations.</p><p id="p-67">The “BL” column shows the power of the BLAST/Blosum and linkage tests alone. These tests are sufficient to identify a most-likely candidate in most cases, but it is clear that the orthogonal transcriptional information provides considerable additional resolving power.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><div id="sec-19" class="subsection"><h3>Identification of causative mutations in an unbiased screen</h3><p id="p-68">Untargeted mutagenic screens are unbiased, requiring no hypotheses as to which genes might be involved in a given system; this is a clear advantage over targeted approaches such as genome editing. However, to attain reasonable efficiency of isolation of informative mutations, it is necessary to “pack” a large number of mutations into each clone because most point mutations have little or no phenotypic effect. This means that, although high-throughput sequencing can identify (nearly) all mutations in a mutant strain of interest, the problem remains of determining which mutation is causative among hundreds or thousands of candidates. For any individual mutant, there are methods that will allow absolute certainty as to the causative SNP: high-resolution genetic mapping, sufficient screening to identify multiple independent alleles, isolation of intragenic revertants, and rescue by transformation. However, these approaches are impractical for large numbers of mutants.</p><p id="p-69">The Bayesian method converts diverse types of data to the “common currency” of probability. This then allows statement of a quantitative degree of certainty that our identifications are correct, which provides a rational basis for evaluating the need for further work to confirm the identifications.</p></div><div id="sec-20" class="subsection"><h3>Annotation-independent identification of causative mutations</h3><p id="p-70">The problem of identifying causative mutations from a set of candidates is a common one; for example in analyzing cancer genomes, or in population genetics when a QTL is known to be located somewhere in a highly polymorphic haplotype block. In those contexts, it is very difficult to proceed without relying on annotation-based information (<em>e.g.</em>, a height QTL with a SNP in a growth hormone-related gene within the haplotype block). In the experimental context discussed in this paper, genetic methods are sufficiently powerful that annotation-based information can be dispensed with altogether. This is fortunate since first, restriction to annotations largely restricts discovery to things that are already at least partially known, and second, it is not obvious how to assign a quantitative probability value to annotation-based information. The approach above is a uniform Bayesian calculation, which should integrate diverse sorts of information on a quantitatively equal basis.</p><p id="p-71">However, it is important to note that the power of meiotic mapping to eliminate most SNPs from consideration is essential for successful restriction to a single highly likely candidate in our analysis; the Bayesian discriminators are strong enough to detect one likely positive out of a small number of candidates, but cannot do so from a larger field. This aspect is less applicable in the haplotype block case, and obviously completely unavailable in the cancer genome case. In these situations, additional discriminators are clearly essential.</p><p id="p-72">One class of annotation-based information is required in our approach: parsing of the raw genome sequence into gene models (exons and coding sequence especially). Fortunately, this has been done quite carefully and effectively in the <em>Chlamydomonas</em> case (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Merchant <em>et al.</em> 2007</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Blaby <em>et al.</em> 2014</a>); our detailed examinations of specific issues with the annotation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cross 2015</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Tulin and Cross 2016</a>) have revealed problems with only a small minority of genes.</p></div><div id="sec-21" class="subsection"><h3>Conservation, divergence, gene duplication, and essentiality</h3><p id="p-73">A central aspect of this computation is based on the observation that essential genes, identified by Ts-lethal single-gene mutations blocking cell proliferation, are much more likely than the <em>Chlamydomonas</em> gene set overall to lie in proteins and specific residues conserved in higher plants, and frequently across yeast and animals as well. In contrast, a substantial majority of <em>Chlamydomonas</em> genes have either no <em>Arabidopsis</em> BLAST hit, or only a hit suggestive of a small protein domain. Most of these genes have unknown, possibly algal-specific functions; our results suggest that few of these functions are essential for cell viability, or at the least very seldom are specifically essential for cell cycle progression. Evolution of cell-essential processes is slow.</p><p id="p-74">Isolation of single-gene Ts-lethals implies that there is no effective backup in the genome; in particular, there is no gene duplicate retaining substantial functional overlap (since otherwise the lethal phenotype should require at least a double hit). Gene families with presumed orthologous members in <em>Chlamydomonas</em> and <em>Arabidopsis</em> tend to be single copy in <em>Chlamydomonas</em>. In <em>Arabidopsis</em>, multiple gene family members are common (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), a well-known observation substantially due to multiple whole-genome duplications in the higher land plant lineage (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Adams and Wendel 2005</a>). It is a commonplace observation in <em>Arabidopsis</em> genetics that strong phenotypes frequently require disruption of multiple gene family members. Gene duplication has been proposed to provide genetic “robustness” (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gu <em>et al.</em> 2003</a>). In general, the <em>Chlamydomonas</em> genome lacks this robustness mechanism: in most cases, mutation of the single <em>Chlamydomonas</em> family member has the potential to immediately expose the maximum phenotype (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Our results suggest that <em>Chlamydomonas</em> has an essential gene set substantially conserved with higher plants, and nearly free of duplicates, supporting its utility as a genetic and cell biological model for the crucially important plant kingdom.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-100">The work was supported by PHS grant GM078152.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-22"><h3>Detailed Development of Bayesian Models for BLAST/Blosum, Linkage, Transcription, and Combination Models</h3></div><div class="section" id="sec-23"><h3>BLAST/Blosum model</h3><p id="p-75">Assume there are N SNPs. Each SNP falls into one of the eight BLAST/Blosum classes. Associated with each class <em>k</em> (<em>k</em> = 1:8), there are two likelihoods: Caus(k), the probability that a causative mutation is in class <em>k</em>, and Pass(k), the probability that a passenger mutation is in class <em>k</em>. These values are exactly those plotted in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>, lower right, derived from the training set (multiplication of marginals is used to avoid zeros).</p><p id="p-76">Then, the set of N SNPs is associated with two 1 × N vectors <em>C</em> and <em>P</em>:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-7.gif"/></noscript></span></span>Call U the probability that the causative mutation is none of the <em>N</em> candidates (this corresponds to unsequenceability, we estimate U at ∼25%; <a id="xref-ref-14-30" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>).</p><p id="p-77">We assume that either the mutation is unsequenceable (probability U), in which case all of the <em>N</em> SNPs are passengers, or exactly one of the N SNPs is causative, therefore the remaining N−1 are passengers. Call model M<sub>i</sub> the model that SNP<sub>i</sub> is causative. The prior probability of M<sub>i</sub> is (1−U)/N, and the probability of M<sub>U</sub> (unsequenceability) is U. These models are mutually exclusive and exhaustive.</p><p id="p-78">Then, the relative probability of S = (S<sub>1</sub>,S<sub>2</sub>…S<sub>n</sub>) (the set of N SNPs) given M<sub>i</sub> is:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-8.gif"/></noscript></span></span>because in this model, all SNPs are passengers except for SNP<sub>i</sub> (Π<sub>j</sub> (P<sub>j</sub>) denotes the product of the P<sub>j</sub>’s, <em>j</em> = 1 to N).</p><p id="p-79">The relative probability of S given MU (unsequenceable) is:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-9.gif"/></noscript></span></span>because in this model, all SNPs are passengers.</p><p id="p-80">These terms are in relative probability units because no probability estimate is provided for having exactly the <em>N</em> SNPs found in S. This probability, if expressed, would multiply every term and therefore divides out.</p><p id="p-81">Call Q the unsequenceability likelihood ratio U/(1−U).</p><p id="p-82">Bayes’ theorem, given the assumption that exactly one of [M<sub>1</sub>, M<sub>2</sub>,..M<sub>n,</sub> M<sub>U</sub>] must be true, gives:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-10.gif"/></noscript></span></span>(where Σ<sub>k</sub> denotes the sum over <em>k</em> = 1 to <em>N</em>)</p><p id="p-83">Substituting:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-11.gif"/></noscript></span></span>Dividing top and bottom by Π<sub>j</sub> (P<sub>j</sub>), multiplying top and bottom by N/(1−U), and substituting Q for U/(1−U) gives:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-12.gif"/></noscript></span></span>Call B<sub>i</sub> the likelihood ratio C<sub>i</sub>/P<sub>i</sub> (the relative prior probability that SNP<sub>i</sub> is causative relative to the probability that it is a passenger); then<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-13.gif"/></noscript></span></span>P(M<sub>i</sub> | S) is the probability that SNP<sub>i</sub> is causative, given the BLAST/Blosum classes of all <em>N</em> candidate SNPs, and the probability U that the causative mutation escaped detection by sequencing.</p><p id="p-84">These probabilities have natural and expected properties. Mutations with high B<sub>i</sub> (such as severe mutations in conserved residues) have greater probability of being causative [high P(M<sub>i</sub> | S)]. Increasing numbers of candidate mutations (higher <em>N</em>) decreases P(M<sub>i</sub> | S), which makes sense since there are many possible candidates to choose from. M<sub>U</sub> decreases in probability given the presence of one or more SNPs with high B<sub>i</sub>. This makes sense, since high B<sub>i</sub> is unlikely; so seeing such a SNP in the collection leads to high likelihood that it is causative. In contrast, if all SNPs have negligible B<sub>i</sub>, the probability of M<sub>U</sub> approaches 1.</p></div><div class="section" id="sec-24"><h3>Linkage model</h3><p id="p-85">In the main text, we summarized two methods for meiotic mapping of a Ts-lethal: cosegregation with a SNP marker (in most cases, the SNP suspected of being causative), or segregation compared to a second Ts-lethal with known (or strongly supported) physical map location. These mapping results yield the number of recombinant/nonrecombinant chromosomes between a physical marker (the candidate SNP or the known location of the second Ts-lethal) and the Ts-lethal of interest. The aim is to translate this information into probabilities for physical location of the causative mutation, since this information can then be used as above to discriminate the different models [M<sub>1</sub>, M<sub>2</sub>,..M<sub>n,</sub> M<sub>U</sub>].</p><p id="p-86">To do this, we employed the following quantitative approach. We assume that the physical location of one marker is known (the PCR-detected SNP for the first approach, or the location of the known second mutation in the second approach). We assume an average 10 cM/Mb ratio (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Merchant <em>et al.</em> 2007</a>; <a id="xref-ref-14-31" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>), and further assume that the minimum error on the mapping is 5 cM∼ = 0.5 Mb, based on the largest cold-spot that we have detected, as well as apparent mapping errors over a large number of such experiments (<a id="xref-ref-14-32" class="xref-bibr" href="#ref-14">Tulin and Cross 2014</a>). In almost all experiments, sufficient meioses are tested that this uncertainty (rather than sampling error) is the main source of error.</p><p id="p-87">We then suppose that the probability density for mutant location is approximately normal, with mean at the exact estimated location. The SD is estimated by finding 95% C.I. limits for the true recombination rate, given the observed numbers of recombinants and nonrecombinants. For a normal distribution, these 95% C.I. limits will be separated by ∼4 SD. The theoretical SD is then set as the minimum of this distance/4, and 0.5 Mb (to account for uncertainty about the uniformity of the 10 cM/Mb genomic average; see above). Mapping is bidirectional (<em>e.g.</em>, 10 cM from a marker could be 10 cM to the left or the right), so we construct both normals, sum them, and renormalize to make the area 1. In cases where the distribution is terminated (the end of the chromosome or the end of a region of uniformity in bulked segregant sequencing analysis), we truncate the distribution and renormalize. This also means that the probability of causality on unlinked chromosomes is set to zero. Note that presence of the causative mutation on this limited region is set as ground truth in this approach. This is reasonable, since linkage to a chromosome is generally established already to extremely high probability.</p><p id="p-88">If there are multiple such mapping experiments, the probability densities are multiplied at each point and renormalized to make a single model. Thus, for example, a bimodal distribution of likely locations is converted by this multiplication to an essentially unimodal one, by finding cosegregation of Ts-lethality with a SNP near the center of one of the bimodal peaks.</p><p id="p-89">An illustration of the generation of such a probability density function is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS2.pptx">Figure S2A</a>, assuming one experiment showing 25/100 recombinants of a marker at 3 Mb with Ts−, and another showing 10/100 recombinants of a marker at 5 Mb with Ts−. The two mapping experiments (top and middle) are bimodal because distance is approximately known (using 10 cM/Mb conversion) but direction is not. The two combined (bottom) strongly favor a location of the Ts− lesion at ∼5.5 to be consistent with both mapping experiments. All curves have total area 1. Call this function <em>f</em>, where f(location) = probability of the causative mutation being at the location.</p><p id="p-90"><em>f</em> is in units of (probability/basepair), and is nonzero only over the interval known (as ground truth) to contain the Ts-lethal mutation.</p><p id="p-91">Thus, the probability density for location of the causative SNP should be just <em>f</em>.</p><p id="p-92">In contrast, the probability density for a passenger SNP should be uniform over the ground truth interval and zero elsewhere, because presence somewhere in this interval is required for it to be in consideration. Thus, this probability is identically 1/(length of ground truth interval) for all candidates (all positions equally likely). Call the length of this interval G; so probability for passengers at each basepair in the interval is 1/G (units of probability/basepair, the same units as for <em>f</em>).</p><p id="p-93">Call L<sub>i</sub> the location of SNPi (L is the vector over the <em>N</em> SNPs). Then:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-14.gif"/></noscript></span></span>Rescale probability density for passenger and causative SNPs by multiplying by <em>G</em>; then the relative pdf of each passenger is 1, and that of the causative SNP is G*f. Call G*f function g.</p><p id="p-94">Then:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-15.gif"/></noscript></span></span>where <em>p</em> is proportional to probability density.</p><p id="p-95">Prior probabilities of M<sub>i</sub>’s and M<sub>U</sub> are as before.</p><p id="p-96">Then Bayes’ theorem yields:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-16.gif"/></noscript></span></span>(scaled probability density above is converted to probability here, because all terms are in the same units of relative probability density).</p><p id="p-97"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS2.pptx">Figure S2B</a> shows placement of two SNPs on the probability curves according to models [M<sub>1</sub>, M<sub>2</sub>, M<sub>U</sub>] (only two candidate SNPs in this example). In this case, M2 is most likely (product of probabilities for the two SNPs is highest for this model).</p></div><div class="section" id="sec-25"><h3>Combined model</h3><p id="p-98">Note that location of a SNP on the chromosome is independent of the BLAST/Blosum characteristics of the SNP. This means that probabilities multiply. So:</p><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-17.gif"/></noscript></span></span></div><div class="section" id="sec-26"><h3>Transcription</h3><p id="p-99">For the <em>DIV</em> subclass, we can also integrate the probability that SNP<sub>i</sub> is causative, based on whether its containing gene follows the transcriptional pattern of most of these genes (since transcriptional pattern is essentially independent of the other classifications). Using the set of strongly identified <em>DIVs</em>, we have a simple 2 × 2 classification table: <em>DIV</em> gene <em>vs.</em> all genes and S/M transcription pattern <em>vs.</em> different pattern (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). If Ti is the likelihood ratio for the gene model containing SNP<sub>i</sub> relative to transcription pattern, based on the training set of definitive <em>DIVs</em> <em>vs.</em> all genes (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>), we can integrate this information into the calculation:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2081/embed/mml-math-18.gif"/></noscript></span></span>To confirm that the model is correctly generated and the code calculating probabilities are accurate, we generated 20,000 “mutants” <em>in silico</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1/FigureS4.pptx">Figure S4</a>), aiming for a reasonable simulation of the observed distributions, and number and scale of typical linkage experiments. High accuracy, sensitivity, and selectivity were observed, verifying the correctness of the calculations.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></li><li class="fn" id="fn-3"><p id="p-101">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039016/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-102"><em>Communicating editor: O. Troyanskaya</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-03"><span class="received-label">Received </span>January 3, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-04-27"><span class="accepted-label">Accepted </span>April 27, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Cross <em>et al</em>.</li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.1" data-doi="10.1016/j.pbi.2005.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Polyploidy and genome evolution in plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">135</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DAdams%26rft.auinit1%253DK.%2BL.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D135%26rft.epage%253D141%26rft.atitle%253DPolyploidy%2Band%2Bgenome%2Bevolution%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2005.01.001%26rft_id%253Dinfo%253Apmid%252F15752992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2005.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15752992&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227932600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blaby</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaby-Haas</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tourasse</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hom</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Chlamydomonas genome project: a decade on.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">672</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D672%26rft_id%253Dinfo%253Apmid%252F24950814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24950814&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.3" data-doi="10.1534/genetics.111.130765"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Yeast: an experimental organism for 21st Century biology.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBotstein%26rft.auinit1%253DD.%26rft.volume%253D189%26rft.issue%253D3%26rft.spage%253D695%26rft.epage%253D704%26rft.atitle%253DYeast%253A%2BAn%2BExperimental%2BOrganism%2Bfor%2B21st%2BCentury%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.130765%26rft_id%253Dinfo%253Apmid%252F22084421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzMvNjk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieberman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tulin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High-throughput robotically assisted isolation of temperature-sensitive lethal mutants in Chlamydomonas reinhardtii.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> DOI: 10.3791/54831.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Tying down loose ends in the Chlamydomonas genome: functional significance of abundant upstream open reading frames.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.6" data-doi="10.1098/rstb.2011.0078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchler</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skotheim</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution of networks and sequences in eukaryotic cell cycle control.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">366</span>(<span class="cit-issue">1584</span>): <span class="cit-fpage">3532</span>–<span class="cit-lpage">3544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DCross%26rft.auinit1%253DF.%2BR.%26rft.volume%253D366%26rft.issue%253D1584%26rft.spage%253D3532%26rft.epage%253D3544%26rft.atitle%253DEvolution%2Bof%2Bnetworks%2Band%2Bsequences%2Bin%2Beukaryotic%2Bcell%2Bcycle%2Bcontrol%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2011.0078%26rft_id%253Dinfo%253Apmid%252F22084380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjYvMTU4NC8zNTMyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.7" data-doi="10.1038/nature01198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scharfe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Role of duplicate genes in genetic robustness against null mutations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGu%26rft.auinit1%253DZ.%26rft.volume%253D421%26rft.issue%253D6918%26rft.spage%253D63%26rft.epage%253D66%26rft.atitle%253DRole%2Bof%2Bduplicate%2Bgenes%2Bin%2Bgenetic%2Brobustness%2Bagainst%2Bnull%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01198%26rft_id%253Dinfo%253Apmid%252F12511954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12511954&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180165500037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.8" data-doi="10.1002/prot.340170108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Performance evaluation of amino acid substitution matrices.</span> <abbr class="cit-jnl-abbrev">Proteins</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">49</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DHenikoff%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D61%26rft.atitle%253DPerformance%2Bevaluation%2Bof%2Bamino%2Bacid%2Bsubstitution%2Bmatrices.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.340170108%26rft_id%253Dinfo%253Apmid%252F8234244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/prot.340170108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8234244&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LT88100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Medina</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordân</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skotheim</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchler</span>  <span class="cit-name-given-names">N. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Punctuated evolution and transitional hybrid network in an ancestral cell cycle of fungi.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e09492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D5%26rft.spage%253De09492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.10" data-doi="10.1126/science.1143609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merchant</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prochnik</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpowicz</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">2007 The Chlamydomonas genome reveals the evolution of key animal and plant functions.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">318</span>(<span class="cit-issue">5848</span>): <span class="cit-fpage">245</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMerchant%26rft.auinit1%253DS.%2BS.%26rft.volume%253D318%26rft.issue%253D5848%26rft.spage%253D245%26rft.epage%253D250%26rft.atitle%253DThe%2BChlamydomonas%2BGenome%2BReveals%2Bthe%2BEvolution%2Bof%2BKey%2BAnimal%2Band%2BPlant%2BFunctions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1143609%26rft_id%253Dinfo%253Apmid%252F17932292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTgvNTg0OC8yNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjA4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.11" data-doi="10.1534/genetics.115.184663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Onishi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evidence that an unconventional actin can provide essential F-actin function and that a surveillance system monitors F-actin integrity in Chlamydomonas.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">977</span>–<span class="cit-lpage">996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.184663%26rft_id%253Dinfo%253Apmid%252F26715672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzMvOTc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.12" data-doi="10.1006/jmbi.2000.5197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storm</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Automatic clustering of orthologs and in-paralogs from pairwise species comparisons.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">314</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1041</span>–<span class="cit-lpage">1052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DRemm%26rft.auinit1%253DM.%26rft.volume%253D314%26rft.issue%253D5%26rft.spage%253D1041%26rft.epage%253D1052%26rft.atitle%253DAutomatic%2Bclustering%2Bof%2Borthologs%2Band%2Bin-paralogs%2Bfrom%2Bpairwise%2Bspecies%2Bcomparisons.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.5197%26rft_id%253Dinfo%253Apmid%252F11743721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.5197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11743721&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173365300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.13" data-doi="10.1093/gbe/evp011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogozin</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basu</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Csürös</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Analysis of rare genomic changes does not support the unikont-bikont phylogeny and suggests cyanobacterial symbiosis as the point of primary radiation of eukaryotes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevp011%26rft_id%253Dinfo%253Apmid%252F20333181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjY6IjEvMC85OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.14" data-doi="10.1105/tpc.114.129312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tulin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A microbial avenue to cell cycle control in the plant superkingdom.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">4019</span>–<span class="cit-lpage">4038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.129312%26rft_id%253Dinfo%253Apmid%252F25336509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMC80MDE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.15" data-doi="10.1105/tpc.15.00400"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tulin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cyclin-dependent kinase regulation of diurnal transcription in Chlamydomonas.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">2727</span>–<span class="cit-lpage">2742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00400%26rft_id%253Dinfo%253Apmid%252F26475866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMC8yNzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tulin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Patching holes in the Chlamydomonas genome.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1899</span>–<span class="cit-lpage">1910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D1899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.17" data-doi="10.1534/genetics.104.039107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yampolsky</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoltzfus</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The exchangeability of amino acids in proteins.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1459</span>–<span class="cit-lpage">1472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.039107%26rft_id%253Dinfo%253Apmid%252F15944362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC80LzE0NTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjA4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.18" data-doi="10.1093/molbev/msh075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciniglia</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharya</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Molecular timeline for the origin of photosynthetic eukaryotes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">21</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">809</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh075%26rft_id%253Dinfo%253Apmid%252F14963099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIxLzUvODA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2081.19" data-doi="10.1105/tpc.15.00498"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zones</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaby</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merchant</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Umen</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-resolution profiling of a synchronized diurnal transcriptome from Chlamydomonas reinhardtii reveals continuous cell and metabolic differentiation.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">2743</span>–<span class="cit-lpage">2769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00498%26rft_id%253Dinfo%253Apmid%252F26432862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMC8yNzQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/7/2081.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2065" title="Manipulating the Mitochondrial Genome To Enhance Cattle Embryo Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2095" title="Effects of the Ordering of Natural Selection and Population Regulation Mechanisms on Wright-Fisher Models" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/7"><span>Volume 7 Issue 7, July 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/7.cover-source.jpg?itok=eAMfPilr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2081.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Z4ejBzLaAxMy8oLTrqLiteR-gr9ygnpTYeqfGVzFICU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/7/2081" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/7/2081" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/7/2081" class="active" data-icon-position="" data-hide-link-title="0">Validated Bayesian Differentiation of Causative and Passenger Mutations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485298" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1nRmlCEAVO2QdVdgO0baJlpCBzL312dF5GRCEhioHUo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2081.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/7/2081" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-TnZmToAhic7X0v7aZeUXX_54ZXkjz9ipxH82sUF4Dxw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/7/2081" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2081.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485298" id="node485298--2" data-pisa="ggg;7/7/2081" data-pisa-master="ggg;g3.117.039016" data-apath="/ggg/7/7/2081.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Validated Bayesian Differentiation of Causative and Passenger Mutations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frederick R. Cross</span>, <span class="highwire-citation-author" data-delta="1">Michal Breker</span> and <span class="highwire-citation-author" data-delta="2">Kristi Lieberman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2081-2094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039016 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.039016" data-atom-uri="/ggg/7/7/2081.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485298--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frederick R. Cross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BR.%2BCross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cross%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BR.%2BCross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fcross@mail.rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">fcross@mail.rockefeller.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michal Breker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michal%2BBreker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Breker%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichal%2BBreker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kristi Lieberman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristi%2BLieberman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lieberman%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristi%2BLieberman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/7/2081" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/7/2081&amp;title=Validated%20Bayesian%20Differentiation%20of%20Causative%20and%20Passenger%20Mutations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/7/2081&amp;title=Validated%20Bayesian%20Differentiation%20of%20Causative%20and%20Passenger%20Mutations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/7/2081&amp;title=Validated%20Bayesian%20Differentiation%20of%20Causative%20and%20Passenger%20Mutations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/7/2081&amp;text=Validated%20Bayesian%20Differentiation%20of%20Causative%20and%20Passenger%20Mutations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/7/2081&amp;title=Validated%20Bayesian%20Differentiation%20of%20Causative%20and%20Passenger%20Mutations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/7/2081&amp;t=Validated%20Bayesian%20Differentiation%20of%20Causative%20and%20Passenger%20Mutations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/7/2081&amp;title=Validated%20Bayesian%20Differentiation%20of%20Causative%20and%20Passenger%20Mutations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/7/2081&amp;title=Validated%20Bayesian%20Differentiation%20of%20Causative%20and%20Passenger%20Mutations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485298/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485298" id="node485298--3" data-pisa="ggg;7/7/2081" data-pisa-master="ggg;g3.117.039016" data-apath="/ggg/7/7/2081.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Validated Bayesian Differentiation of Causative and Passenger Mutations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frederick R. Cross</span>, <span class="highwire-citation-author" data-delta="1">Michal Breker</span> and <span class="highwire-citation-author" data-delta="2">Kristi Lieberman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2081-2094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039016 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.039016" data-atom-uri="/ggg/7/7/2081.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485298--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frederick R. Cross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BR.%2BCross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cross%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BR.%2BCross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fcross@mail.rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">fcross@mail.rockefeller.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michal Breker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michal%2BBreker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Breker%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichal%2BBreker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kristi Lieberman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristi%2BLieberman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lieberman%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristi%2BLieberman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2081&amp;rft.epage=2094&amp;rft.atitle=Validated%20Bayesian%20Differentiation%20of%20Causative%20and%20Passenger%20Mutations&amp;rft.volume=7&amp;rft.issue=7&amp;rft.date=2017-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cross%2C+Frederick+R.&amp;rft.au=Breker%2C+Michal&amp;rft.au=Lieberman%2C+Kristi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485298/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485298/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485298/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485298/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485298/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485298/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485298/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485298/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485298/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485298/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485298/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485298/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/7/2081" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/7/2081.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/7/2081.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/7/2081.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/7/2081.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
