<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/7/2015" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/7/2015.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/7/2015.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/7/2015.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/7/2015.atom" />
<meta name="HW.pisa" content="ggg;7/7/2015" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Insight into the Recent Genome Duplication of the Halophilic Yeast Hortaea werneckii: Combining an Improved Genome with Gene Expression and Chromatin Structure" />
<meta name="DC.Identifier" content="10.1534/g3.117.040691" />
<meta name="DC.Date" content="2017-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Sinha et al." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Extremophilic organisms demonstrate the flexibility and adaptability of basic biological processes by highlighting how cell physiology adapts to environmental extremes. Few eukaryotic extremophiles have been well studied and only a small number are amenable to laboratory cultivation and manipulation. A detailed characterization of the genome architecture of such organisms is important to illuminate how they adapt to environmental stresses. One excellent example of a fungal extremophile is the halophile Hortaea werneckii (Pezizomycotina, Dothideomycetes, Capnodiales), a yeast-like fungus able to thrive at near-saturating concentrations of sodium chloride and which is also tolerant to both UV irradiation and desiccation. Given its unique lifestyle and its remarkably recent whole genome duplication, H. werneckii provides opportunities for testing the role of genome duplications and adaptability to extreme environments. We previously assembled the genome of H. werneckii using short-read sequencing technology and found a remarkable degree of gene duplication. Technology limitations, however, precluded high-confidence annotation of the entire genome. We therefore revisited the H. wernickii genome using long-read, single-molecule sequencing and provide an improved genome assembly which, combined with transcriptome and nucleosome analysis, provides a useful resource for fungal halophile genomics. Remarkably, the ∼50 Mb H. wernickii genome contains 15,974 genes of which 95% (7608) are duplicates formed by a recent whole genome duplication (WGD), with an average of 5% protein sequence divergence between them. We found that the WGD is extraordinarily recent, and compared to Saccharomyces cerevisiae , the majority of the genome’s ohnologs have not diverged at the level of gene expression of chromatin structure." />
<meta name="DC.Contributor" content="Sunita Sinha" />
<meta name="DC.Contributor" content="Stephane Flibotte" />
<meta name="DC.Contributor" content="Mauricio Neira" />
<meta name="DC.Contributor" content="Sean Formby" />
<meta name="DC.Contributor" content="Ana Plemenitaš" />
<meta name="DC.Contributor" content="Nina Gunde Cimerman" />
<meta name="DC.Contributor" content="Metka Lenassi" />
<meta name="DC.Contributor" content="Cene Gostinčar" />
<meta name="DC.Contributor" content="Jason E. Stajich" />
<meta name="DC.Contributor" content="Corey Nislow" />
<meta name="article:published_time" content="2017-07-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Insight into the Recent Genome Duplication of the Halophilic Yeast Hortaea werneckii: Combining an Improved Genome with Gene Expression and Chromatin Structure" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Extremophilic organisms demonstrate the flexibility and adaptability of basic biological processes by highlighting how cell physiology adapts to environmental extremes. Few eukaryotic extremophiles have been well studied and only a small number are amenable to laboratory cultivation and manipulation. A detailed characterization of the genome architecture of such organisms is important to illuminate how they adapt to environmental stresses. One excellent example of a fungal extremophile is the halophile &lt;i&gt;Hortaea werneckii&lt;/i&gt; (Pezizomycotina, Dothideomycetes, Capnodiales), a yeast-like fungus able to thrive at near-saturating concentrations of sodium chloride and which is also tolerant to both UV irradiation and desiccation. Given its unique lifestyle and its remarkably recent whole genome duplication, &lt;i&gt;H. werneckii&lt;/i&gt; provides opportunities for testing the role of genome duplications and adaptability to extreme environments. We previously assembled the genome of &lt;i&gt;H. werneckii&lt;/i&gt; using short-read sequencing technology and found a remarkable degree of gene duplication. Technology limitations, however, precluded high-confidence annotation of the entire genome. We therefore revisited the &lt;i&gt;H. wernickii&lt;/i&gt; genome using long-read, single-molecule sequencing and provide an improved genome assembly which, combined with transcriptome and nucleosome analysis, provides a useful resource for fungal halophile genomics. Remarkably, the ∼50 Mb &lt;i&gt;H. wernickii&lt;/i&gt; genome contains 15,974 genes of which 95% (7608) are duplicates formed by a recent whole genome duplication (WGD), with an average of 5% protein sequence divergence between them. We found that the WGD is extraordinarily recent, and compared to &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, the majority of the genome’s ohnologs have not diverged at the level of gene expression of chromatin structure.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/07/01" />
<meta name="citation_mjid" content="ggg;7/7/2015" />
<meta name="citation_id" content="7/7/2015" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/7/2015" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/7/2015.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/7/2015.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/7/2015.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.040691" />
<meta name="citation_pmid" content="28500048" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="2015" />
<meta name="citation_lastpage" content="2022" />
<meta name="citation_author" content="Sunita Sinha" />
<meta name="citation_author_institution" content="Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada" />
<meta name="citation_author" content="Stephane Flibotte" />
<meta name="citation_author_institution" content="Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada" />
<meta name="citation_author" content="Mauricio Neira" />
<meta name="citation_author_institution" content="Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada" />
<meta name="citation_author" content="Sean Formby" />
<meta name="citation_author_institution" content="Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada" />
<meta name="citation_author" content="Ana Plemenitaš" />
<meta name="citation_author_institution" content="Institute of Biochemistry, Faculty of Medicine, University of Ljubljana, 1000, Slovenia" />
<meta name="citation_author" content="Nina Gunde Cimerman" />
<meta name="citation_author_institution" content="Department of Biology, Biotechnical Faculty, University of Ljubljana, 1000, Slovenia" />
<meta name="citation_author" content="Metka Lenassi" />
<meta name="citation_author_institution" content="Institute of Biochemistry, Faculty of Medicine, University of Ljubljana, 1000, Slovenia" />
<meta name="citation_author" content="Cene Gostinčar" />
<meta name="citation_author_institution" content="Department of Biology, Biotechnical Faculty, University of Ljubljana, 1000, Slovenia" />
<meta name="citation_author" content="Jason E. Stajich" />
<meta name="citation_author_institution" content="Department of Plant Pathology and Microbiology, Institute for Integrative Genome Biology, University of California, Riverside, California 92521" />
<meta name="citation_author_email" content="jason.stajich@ucr.edu" />
<meta name="citation_author_email" content="corey.nislow@ubc.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7591-0020" />
<meta name="citation_author" content="Corey Nislow" />
<meta name="citation_author_institution" content="Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada" />
<meta name="citation_author_email" content="jason.stajich@ucr.edu" />
<meta name="citation_author_email" content="corey.nislow@ubc.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4016-8874" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schäffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=G. Blanc;citation_author=K. H. Wolfe;citation_title=Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes.;citation_volume=16;citation_year=2004;citation_pmid=15208399;citation_doi=10.1105/tpc.021345" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. P. Byrne;citation_author=K. H. Wolfe;citation_title=The yeast gene order browser: combining curated homology and syntenic context reveals gene fate in polyploid species.;citation_volume=15;citation_year=2005;citation_pmid=16169922;citation_doi=10.1101/gr.3672305" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. Chen;citation_author=Y. Xi;citation_author=X. Pan;citation_author=Z. Li;citation_author=K. Kaestner;citation_title=DANPOS: dynamic analysis of nucleosome position and occupancy by sequencing.;citation_volume=23;citation_year=2013;citation_pmid=23193179;citation_doi=10.1101/gr.142067.112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. F. Cliften;citation_author=R. S. Fulton;citation_author=R. K. Wilson;citation_author=M. Johnston;citation_title=After the duplication: gene loss and adaptation in Saccharomyces genomes.;citation_pages=863-872;citation_volume=172;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=L. M. Corrochano;citation_author=A. Kuo;citation_author=M. Marcet-Houben;citation_author=S. Polaino;citation_author=A. Salamov;citation_title=Expansion of signal transduction pathways in fungi by extensive genome duplication.;citation_pages=1577-1584;citation_volume=26;citation_year=2016;citation_doi=10.1016/j.cub.2016.04.038" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=J. C. del Pozo;citation_author=E. Ramirez-Parra;citation_title=Whole genome duplications in plants: an overview from Arabidopsis.;citation_volume=66;citation_year=2015;citation_pmid=26417017;citation_doi=10.1093/jxb/erv432" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. J. G. M. de Wit;citation_author=A. van der Burgt;citation_author=B. Ökmen;citation_author=I. Stergiopoulos;citation_author=K. A. Abd-Elsalam;citation_title=The Genomes of the Fungal Plant Pathogens Cladosporium fulvum and Dothistroma septosporum Reveal Adaptation to Different Hosts and Lifestyles But Also Signatures of Common Ancestry.;citation_pages=e1003088-e1003088;citation_volume=8;citation_year=2012;citation_issue=11;citation_pmid=23209441;citation_doi=10.1371/journal.pgen.1003088" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=R. Dhar;citation_author=R. Sägesser;citation_author=C. Weikert;citation_author=J. Yuan;citation_author=A. Wagner;citation_title=Adaptation of Saccharomyces cerevisiae to saline stress through laboratory evolution.;citation_pages=1135-1153;citation_volume=24;citation_year=2011;citation_issue=5;citation_pmid=21375649;citation_doi=10.1111/j.1420-9101.2011.02249.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. A. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: ultrafast universal RNA-seq aligner.;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=B. Dujon;citation_title=Basic principles of yeast genomics, a personal recollection.;citation_volume=15;citation_year=2015;citation_pmid=26071597;citation_doi=10.1093/femsyr/fov047" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Dujon;citation_author=D. Sherman;citation_author=G. Fischer;citation_author=P. Durrens;citation_author=S. Casaregola;citation_title=Genome evolution in yeasts.;citation_pages=35-44;citation_volume=430;citation_year=2004;citation_issue=6995;citation_pmid=15229592;citation_doi=10.1038/nature02579" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Microbiol.;citation_author=I. V. Ene;citation_author=R. J. Bennett;citation_title=The cryptic sexual strategies of human fungal pathogens.;citation_pages=239-251;citation_volume=12;citation_year=2014;citation_pmid=24625892;citation_doi=10.1038/nrmicro3236" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. C. Gerstein;citation_author=H.-J. E. Chun;citation_author=A. Grant;citation_author=S. P. Otto;citation_title=Genomic convergence toward diploidy in Saccharomyces cerevisiae.;citation_pages=e145-e145;citation_volume=2;citation_year=2006;citation_issue=9;citation_pmid=17002497;citation_doi=10.1371/journal.pgen.0020145" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. B. Goodwin;citation_author=S. Ben M’barek;citation_author=B. Dhillon;citation_author=A. H. J. Wittenberg;citation_author=C. F. Crane;citation_title=Finished genome of the fungal wheat pathogen Mycosphaerella graminicola reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.;citation_pages=e1002070-e1002070;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21695235;citation_doi=10.1371/journal.pgen.1002070" />
<meta name="citation_reference" content="citation_journal_title=Advances in applied microbiology;citation_journal_abbrev=Adv Appl Microbiol;citation_author=C. Gostinčar;citation_author=M. Lenassi;citation_author=N. Gunde-Cimerman;citation_author=A. Plemenitaš;citation_title=Fungal adaptation to extremely high salt concentrations.;citation_pages=71-96;citation_volume=77;citation_year=2011;citation_pmid=22050822;citation_doi=10.1016/B978-0-12-387044-5.00003-0" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=B. J. Haas;citation_author=A. Papanicolaou;citation_author=M. Yassour;citation_author=M. Grabherr;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.;citation_pages=1494-1512;citation_volume=8;citation_year=2013;citation_pmid=23845962;citation_doi=10.1038/nprot.2013.084" />
<meta name="citation_reference" content="citation_author=B. Hall;citation_author=T. De Rego;citation_author=S. Geib;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. G. Henikoff;citation_author=J. A. Belsky;citation_author=K. Krassovsky;citation_author=D. M. MacAlpine;citation_author=S. Henikoff;citation_title=Epigenome characterization at single base-pair resolution.;citation_volume=108;citation_year=2011;citation_pmid=22025700;citation_doi=10.1073/pnas.1110731108" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_author=M. Henry;citation_author=L. Debarbieux;citation_title=Tools from viruses: bacteriophage successes and beyond.;citation_pages=151-161;citation_volume=434;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Holt;citation_author=M. Yandell;citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.;citation_pages=491-491;citation_volume=12;citation_year=2011;citation_pmid=22192575;citation_doi=10.1186/1471-2105-12-491" />
<meta name="citation_reference" content="citation_journal_title=Trends in cell biology;citation_journal_abbrev=Trends Cell Biol;citation_author=V. Iyer;citation_title=Nucleosome positioning: bringing order to the eukaryotic genome.;citation_pages=250-256;citation_volume=22;citation_year=2012;citation_issue=5;citation_pmid=22421062;citation_doi=10.1016/j.tcb.2012.02.004" />
<meta name="citation_reference" content="citation_journal_title=Extremophiles;citation_author=B. Jia;citation_author=G.-W. Cheong;citation_author=S. Zhang;citation_title=Multifunctional enzymes in archaea: promiscuity and moonlight.;citation_pages=193-203;citation_volume=17;citation_year=2013;citation_pmid=23283522;citation_doi=10.1007/s00792-012-0509-1" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=K. Katoh;citation_author=H. Toh;citation_title=Improved accuracy of multiple ncRNA alignment by incorporating structural information into a MAFFT-based framework.;citation_pages=212-212;citation_volume=9;citation_year=2008;citation_pmid=18439255;citation_doi=10.1186/1471-2105-9-212" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=K. Katoh;citation_author=H. Toh;citation_title=Recent developments in the MAFFT multiple sequence alignment program.;citation_volume=9;citation_year=2008;citation_pmid=18372315;citation_doi=10.1093/bib/bbn013" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=D. Kim;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=HISAT: a fast spliced aligner with low memory requirements.;citation_pages=357-360;citation_volume=12;citation_year=2015;citation_pmid=25751142;citation_doi=10.1038/nmeth.3317" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=I. Korf;citation_title=Gene finding in novel genomes.;citation_pages=59-59;citation_volume=5;citation_year=2004;citation_pmid=15144565;citation_doi=10.1186/1471-2105-5-59" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Kurtz;citation_author=A. Phillippy;citation_author=A. L. Delcher;citation_author=M. Smoot;citation_author=M. Shumway;citation_title=Versatile and open software for comparing large genomes.;citation_pages=R12-R12;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=14759262;citation_doi=10.1186/gb-2004-5-2-r12" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Lenassi;citation_author=C. Gostinčar;citation_author=S. Jackman;citation_author=M. Turk;citation_author=I. Sadowski;citation_title=Whole genome duplication and enrichment of metal cation transporters revealed by de novo genome sequencing of extremely halotolerant black yeast Hortaea werneckii.;citation_pages=e71328;citation_volume=8;citation_year=2013;citation_pmid=23977017;citation_doi=10.1371/journal.pone.0071328" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=G. Lentzen;citation_author=T. Schwarz;citation_title=Extremolytes: Natural compounds from extremophiles for versatile applications.;citation_pages=623-634;citation_volume=72;citation_year=2006;citation_issue=4;citation_pmid=16957893;citation_doi=10.1007/s00253-006-0553-9" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Li;citation_author=Z. Yuan;citation_author=Z. Zhang;citation_title=The cellular robustness by genetic redundancy in budding yeast.;citation_pages=e1001187-e1001187;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=21079672;citation_doi=10.1371/journal.pgen.1001187" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Evolutionary Fate and Consequences of Duplicate Genes;citation_pages=1151-1155;citation_volume=290;citation_year=2000;citation_issue=5494;citation_pmid=11073452;citation_doi=10.1126/science.290.5494.1151" />
<meta name="citation_reference" content="citation_journal_title=Journal of structural and functional genomics;citation_journal_abbrev=J Struct Funct Genomics;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The evolutionary demography of duplicate genes.;citation_pages=35-44;citation_volume=3;citation_year=2003;citation_issue=1-4;citation_pmid=12836683;citation_doi=10.1023/A:1022696612931" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L.-J. Ma;citation_author=A. S. Ibrahim;citation_author=C. Skory;citation_author=M. G. Grabherr;citation_author=G. Burger;citation_title=Genomic analysis of the basal lineage fungus Rhizopus oryzae reveals a whole-genome duplication.;citation_pages=e1000549-e1000549;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19578406;citation_doi=10.1371/journal.pgen.1000549" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Musso;citation_author=M. Costanzo;citation_author=M. Huangfu;citation_author=A. M. Smith;citation_author=J. Paw;citation_title=The extensive and condition-dependent nature of epistasis among whole-genome duplicates in yeast.;citation_volume=18;citation_year=2008;citation_pmid=18463300;citation_doi=10.1101/gr.076174.108" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=R. A. Ohm;citation_author=N. Feau;citation_author=B. Henrissat;citation_author=C. L. Schoch;citation_author=B. A. Horwitz;citation_title=Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen Dothideomycetes fungi.;citation_pages=e1003037;citation_volume=8;citation_year=2012;citation_pmid=23236275;citation_doi=10.1371/journal.ppat.1003037" />
<meta name="citation_reference" content="citation_journal_title=Environmental Technology Letters;citation_journal_abbrev=Environmental Technology Letters;citation_author=A. Oren;citation_title=Industrial and environmental applications of halophilic microorganisms.;citation_pages=825-834;citation_volume=31;citation_year=2010;citation_issue=8-9;citation_pmid=20662374;citation_doi=10.1080/09593330903370026" />
<meta name="citation_reference" content="citation_journal_title=Extremophiles;citation_author=A. Oren;citation_title=Taxonomy of halophilic Archaea: current status and future challenges.;citation_pages=825-834;citation_volume=18;citation_year=2014;citation_pmid=25102811;citation_doi=10.1007/s00792-014-0654-9" />
<meta name="citation_reference" content="citation_title=Funannotate: Fungal genome annotation scripts.;citation_author=J. Palmer;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Parra;citation_author=K. Bradnam;citation_author=I. Korf;citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes.;citation_volume=23;citation_year=2007;citation_pmid=17332020;citation_doi=10.1093/bioinformatics/btm071" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=M. Pertea;citation_author=G. M. Pertea;citation_author=C. M. Antonescu;citation_author=T.-C. Chang;citation_author=J. T. Mendell;citation_title=StringTie enables improved reconstruction of a transcriptome from RNA-seq reads.;citation_pages=290-295;citation_volume=33;citation_year=2015;citation_pmid=25690850;citation_doi=10.1038/nbt.3122" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=A. Plemenitaš;citation_author=M. Lenassi;citation_author=T. Konte;citation_author=A. Kejžar;citation_author=J. Zajc;citation_title=Adaptation to high salt concentrations in halotolerant/halophilic fungi: a molecular perspective.;citation_pages=199;citation_volume=5;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdóttir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=D. R. Scannell;citation_author=G. Butler;citation_author=K. H. Wolfe;citation_title=Yeast genome evolution--the origin of the species.;citation_pages=929-942;citation_volume=24;citation_year=2007;citation_issue=11;citation_pmid=17621376;citation_doi=10.1002/yea.1515" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. E. Schoustra;citation_author=A. J. M. Debets;citation_author=M. Slakhorst;citation_author=R. F. Hoekstra;citation_title=Mitotic recombination accelerates adaptation in the fungus Aspergillus nidulans.;citation_pages=e68-e68;citation_volume=3;citation_year=2007;citation_issue=4;citation_pmid=17465683;citation_doi=10.1371/journal.pgen.0030068" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=S. Shabala;citation_author=J. Bose;citation_author=A. T. Fuglsang;citation_author=I. Pottosin;citation_title=On a quest for stress tolerance genes: membrane transporters in sensing and adapting to hostile soils.;citation_volume=67;citation_year=2016;citation_pmid=26507891;citation_doi=10.1093/jxb/erv465" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=G. S. C. Slater;citation_author=E. Birney;citation_title=Automated generation of heuristics for biological sequence comparison.;citation_pages=31-31;citation_volume=6;citation_year=2005;citation_pmid=15713233;citation_doi=10.1186/1471-2105-6-31" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. E. Stajich;citation_author=D. Block;citation_author=K. Boulez;citation_author=S. E. Brenner;citation_author=S. A. Chervitz;citation_title=The Bioperl Toolkit: Perl Modules for the Life Sciences;citation_pages=1611-1618;citation_volume=12;citation_year=2002;citation_issue=10;citation_pmid=12368254;citation_doi=10.1101/gr.361602" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. Stanke;citation_author=O. Schöffmann;citation_author=B. Morgenstern;citation_author=S. Waack;citation_title=Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources.;citation_pages=62-62;citation_volume=7;citation_year=2006;citation_pmid=16469098;citation_doi=10.1186/1471-2105-7-62" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. H. Stukenbrock;citation_author=F. B. Christiansen;citation_author=T. T. Hansen;citation_author=J. Y. Dutheil;citation_author=M. H. Schierup;citation_title=Fusion of two divergent fungal individuals led to the recent emergence of a unique widespread pathogen species.;citation_volume=109;citation_year=2012;citation_pmid=22711811;citation_doi=10.1073/pnas.1201403109" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdottir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative genomics viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=K. Tsui;citation_author=T. Durbic;citation_author=M. Gebbia;citation_author=C. Nislow;citation_title=Genomic approaches for determining nucleosome occupancy in yeast.;citation_pages=389-411;citation_volume=833;citation_year=2012;citation_pmid=22183606;citation_doi=10.1007/978-1-61779-477-3_23" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B. Biol. Sci.;citation_author=K. Vanneste;citation_author=S. Maere;citation_author=Y. Van de Peer;citation_title=Tangled up in two: a burst of genome duplications at the end of the Cretaceous and the consequences for plant evolution.;citation_volume=369;citation_year=2014;citation_pmid=24958926;citation_doi=10.1098/rstb.2013.0353" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. Vaupotic;citation_author=A. Plemenitaš;citation_title=Differential gene expression and Hog1 interaction with osmoresponsive genes in the extremely halotolerant black yeast Hortaea werneckii.;citation_pages=280-280;citation_volume=8;citation_year=2007;citation_pmid=17705830;citation_doi=10.1186/1471-2164-8-280" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Woese;citation_author=G. E. Fox;citation_title=Phylogenetic structure of the prokaryotic domain: The primary kingdoms;citation_pages=5088-5090;citation_volume=74;citation_year=1977;citation_issue=11;citation_pmid=270744;citation_doi=10.1073/pnas.74.11.5088" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=S. Nacu;citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads.;citation_volume=26;citation_year=2010;citation_pmid=20147302;citation_doi=10.1093/bioinformatics/btq057" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=Z. Yang;citation_author=R. Nielsen;citation_title=Estimating Synonymous and Nonsynonymous Substitution Rates Under Realistic Evolutionary Models;citation_pages=32-43;citation_volume=17;citation_year=2000;citation_issue=1;citation_pmid=10666704;citation_doi=10.1093/oxfordjournals.molbev.a026236" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=N. H. Youssef;citation_author=M. B. Couger;citation_author=C. G. Struchtemeyer;citation_author=A. S. Liggenstoffer;citation_author=R. A. Prade;citation_title=The genome of the anaerobic fungus Orpinomyces sp. strain C1A reveals the unique evolutionary history of a remarkable plant biomass degrader.;citation_volume=79;citation_year=2013;citation_pmid=23709508;citation_doi=10.1128/AEM.00821-13" />
<meta name="twitter:title" content="Insight into the Recent Genome Duplication of the Halophilic Yeast Hortaea werneckii: Combining an Improved Genome with Gene Expression and Chromatin Structure" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="twitter:description" content="Extremophilic organisms demonstrate the flexibility and adaptability of basic biological processes by highlighting how cell physiology adapts to environmental extremes. Few eukaryotic extremophiles have been well studied and only a small number are amenable to laboratory cultivation and manipulation. A detailed characterization of the genome architecture of such organisms is important to illuminate how they adapt to environmental stresses. One excellent example of a fungal extremophile is the halophile Hortaea werneckii (Pezizomycotina, Dothideomycetes, Capnodiales), a yeast-like fungus able to thrive at near-saturating concentrations of sodium chloride and which is also tolerant to both UV irradiation and desiccation. Given its unique lifestyle and its remarkably recent whole genome duplication, H. werneckii provides opportunities for testing the role of genome duplications and adaptability to extreme environments. We previously assembled the genome of H. werneckii using short-read sequencing technology and found a remarkable degree of gene duplication. Technology limitations, however, precluded high-confidence annotation of the entire genome. We therefore revisited the H. wernickii genome using long-read, single-molecule sequencing and provide an improved genome assembly which, combined with transcriptome and nucleosome analysis, provides a useful resource for fungal halophile genomics. Remarkably, the ∼50 Mb H. wernickii genome contains 15,974 genes of which 95% (7608) are duplicates formed by a recent whole genome duplication (WGD), with an average of 5% protein sequence divergence between them. We found that the WGD is extraordinarily recent, and compared to Saccharomyces cerevisiae , the majority of the genome’s ohnologs have not diverged at the level of gene expression of chromatin structure." />
<meta name="og-title" property="og:title" content="Insight into the Recent Genome Duplication of the Halophilic Yeast Hortaea werneckii: Combining an Improved Genome with Gene Expression and Chromatin Structure" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/7/2015" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Extremophilic organisms demonstrate the flexibility and adaptability of basic biological processes by highlighting how cell physiology adapts to environmental extremes. Few eukaryotic extremophiles have been well studied and only a small number are amenable to laboratory cultivation and manipulation. A detailed characterization of the genome architecture of such organisms is important to illuminate how they adapt to environmental stresses. One excellent example of a fungal extremophile is the halophile Hortaea werneckii (Pezizomycotina, Dothideomycetes, Capnodiales), a yeast-like fungus able to thrive at near-saturating concentrations of sodium chloride and which is also tolerant to both UV irradiation and desiccation. Given its unique lifestyle and its remarkably recent whole genome duplication, H. werneckii provides opportunities for testing the role of genome duplications and adaptability to extreme environments. We previously assembled the genome of H. werneckii using short-read sequencing technology and found a remarkable degree of gene duplication. Technology limitations, however, precluded high-confidence annotation of the entire genome. We therefore revisited the H. wernickii genome using long-read, single-molecule sequencing and provide an improved genome assembly which, combined with transcriptome and nucleosome analysis, provides a useful resource for fungal halophile genomics. Remarkably, the ∼50 Mb H. wernickii genome contains 15,974 genes of which 95% (7608) are duplicates formed by a recent whole genome duplication (WGD), with an average of 5% protein sequence divergence between them. We found that the WGD is extraordinarily recent, and compared to Saccharomyces cerevisiae , the majority of the genome’s ohnologs have not diverged at the level of gene expression of chromatin structure." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
    <title>Insight into the Recent Genome Duplication of the Halophilic Yeast Hortaea werneckii: Combining an Improved Genome with Gene Expression and Chromatin Structure | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"1cj7Mzhdn875m7uNreWtsyqvXAuHl1mFWzICqZnYh8I","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485294","apath":"\/ggg\/7\/7\/2015.atom","pisa":"ggg;7\/7\/2015","ac":{"\/ggg\/7\/7\/2015.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/7\/2015.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2015"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2015"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/7\/2015.figures-only":true,"\/content\/7\/7\/2015":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PZb2hBG6IoAmAf6jFxB3fBh2tOKyMzrFBDYshwzsEtEPlLX3L1GGp\\\\\\\/s1TRiV++Mjoj8AOM4alMy2+ivOqYngIv3duEay\\\\\\\/3fEoTsLzdXPORugTgSO4MioJxK9VhbnwXtTUCkvWBjq5S9JaaEZ103d4nLkYiJGFebSYrRgKbrQlw4oEhadraR0+evTQ14PtHqYDAWJhhE0HcmQh2pYigfCS7Z5XccksyiTwVCqsIq9aAxwFcor70IfW7WzWH\\\\\\\/i5g8fj7l3qUmcuDgVYLQUCfe9C8O\\\\\\\/RFP2FIVmOE72fyGgeGzRt86F8KdSfGdliUj306n26y6qxGJLqgNuOpDuZNONrof\\\\\\\/svty\\\\\\\/PGmou9agmDGTf8nlDfUD+s9tXdpCGWh1E\\\\\\\/xVKmA7+XDROx9ge8qA5q8zPcyOSWm31jkr6GVe09Bsc+PccIcZv1C4dwF2i7VMQJ3oa5wAM0Xyu2EY2xZEtn93Y2U\\\\\\\/lRjpE9lUFDfu+NqG9ASpRPogFiY7Silsqs\\\\\\\/7Ib6+\\\\\\\/4HYtRBVc7e922LAhRpggwQJ99QbmGWjYADlLgKzfYJY24191yvUXcCLZKfEdF3gOs9rsrFBj5yLGZC3aCtSf353CCIcGYXH04tU26lxvYbDGrX4m6ID+8ccUiJt7h89NBis2xn4xEr76NMr\\\\\\\/Mk0Ek2\\\\\\\/TaJz6k05O6oaDDQ0vP8DMJUDUekfV8\\\\\\\/1643ueb9cXb+cptpz9rHMybu3mRfetWopnnbq2RJMyCEvo9qVGX\\\\\\\/pBr2dDLDMFSUuVNoJd5RmK+d6cppEGInm\\\\\\\/o43t71fYMRIGPzLHqht3DlaVTzI2VXaEFUodrm42XXTzby9HrLbHnmtAPTmPYNYVoFMr\\\\\\\/CJMmr\\\\\\\/9lW7PgXqMSMsKBJMVp8QguhWLKn\\\\\\\/baIwv13n+dX1mgxSfZWNp\\\\\\\/VVsZIewTHwK\\\\\\\/b5w3EoDXwHrzEKXsmA129wVnpsg7Ro3vJB4fyLMuSeEyilIuQF77pdm9sFW4KMpooti9utl4LHWMxeOm5MJoJ\\\\\\\/\\\\\\\/seBwiWMeqgSwAC7FIHbh6b4f6Jb04eIOlxDSnTQFJnbppyKrrcZRkH+ti2ohEL6ZpqFFNu1hgYK3fdTtgRFrAJ7f8M8pNKUkGcgXFQ97Qus1Z7G4PKgJB2Cd2p0gGyYCYSLeaeM6ge+J9fPDpZyu1OrJiwa9\\\\\\\/XVDXooATOIBcgZmpVzWKbYXW4pUMFTqzj60kDcchYMxjdbmzIg5ih1gMgCp0nCxm9jd+nzku28etsyRitgnDGvd+f6Gq99Z1\\\\\\\/BgzEAmtba\\\\\\\/OIsIONs5EL\\\u0022,\\\u0022iv\\\u0022:\\\u0022ED6pE5luyLk6rkpKKkASeg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a334b547188511f384a5c97fc8b44ad6757a445a743dcfc1865b6373e11f823c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dM9Erd4J\\\\\\\/RHcasIqHe3QflpMiU8QN1yXRMwIr8m7V8e4NjVAaWtzcPbMPV5Lfuau7kBJJXXUe+GVQZzDVSBvs7RhnOAaKTNwofOWlGUs7cp\\\\\\\/XfkM2Dk4fov+NhkOO3j5P8sOcNgtRqPVG+W8uXckLveWhMBAkaNfPwyyFkjcl+ZRDAroF5xgNPL374yl9J300h34mk8ZVTVor8ow7Fu3byyxIma+68hKJki09Bxg4bmjMXGlfhs\\\\\\\/9RNyT\\\\\\\/k\\\\\\\/hDbK4EbhUsgw3XbHiQve6ROEWGQ8Am3tCJ1bpFguu2B1ApK5q3fYKTilLI0M2SOFgxdpy27jiEk5MlCwSQ05PHbZe9WpL3GwBm4gEEuQvDYfXw8hllTmrl94AUx8igfXOlNQQHGx6brhNfIrW2gC9Ns61J4Gqr2gyjAFYQFb\\\\\\\/HeXwt0gw3NqxK7nawdYTk69Slvm0F1fH+oHpAFv\\\\\\\/HE0H6+bdGPo929NwAlMujyBkaSaofvTOywbg6k50PeEMyYRoZOB2IYWWUA8rwAmdMxJMjO5yRIn\\\\\\\/JchD9U7xz4cF3gA5aE8DIil\\\\\\\/+EaLMOcLAM9Wl7Zcz9SJKxn3G4gFDLY+hR\\\\\\\/gSdwm0griyl\\\\\\\/OPfguKnYxQM7JPYZ1Xem\\\\\\\/R4oUsmcqfKyEg6ZIP50dkMF9xdhdWIo2bl36tIhMJk6eoTB3dVXQKc7b4sl9PXPfcEvE5neFQOm8nlFRMWcjzQwLmOgdRQ01OxSjnovLUMqIdn8E9CF+RKWqYJEorRmEp\\\\\\\/IwY8JGKFsJhIad9Vr9VivORVOihInjm2Gx+7\\\\\\\/ZxTD5rbwek9zdAVEln+l5nHS4C\\\\\\\/Jukjmi3rIQrNyz0xSAA4xl05pOsPWvgrSOcgPi3GGcwMCQYbg3JuSIaXDPe\\\\\\\/k9J1PXv4XeCKj3evbSAnjA8wsSrerns8cmyt2YcJf\\\\\\\/Ru+YMuoo0scw0YT2xwHO8M6Uq9eopM9pHcxcvkAsTj0Q5MZrUROopyHakfozDrY9OpMRJOwIeAqRIucnkcUNJibInT+wPmDX5MpQS7FxAFrr06dE9eUenciiW0T\\\\\\\/dizPtUuLqsNKyBTlZEPtYqHz\\\\\\\/621g+R\\\\\\\/lhPIblqZVF2shtVrlt2cwKIa5VSgKciJTTBa0TqMxsVMRSLmMRZLKOEmaWt0dMgN\\\\\\\/O9lLSe3TWV+O1f+c6AEx3XV0RTv2wGgC43MICsto9fHznqnTNVq\\\\\\\/YtmpX6bXdnDTKqiMZLykBXA652FVMPMiSxgMkFbZG1s2WPm4YUsqEeFGuUjDas\\\\\\\/Qh+mwLizT6sqhDpaNugzxJ3NhyOE1ST+l73eCXI8HSE4cfI3qeBxIWmc4JTPhbVUQAFKPlePsXHs9dYnmBWn5YBcMmZwRHFCxLhSb+7woSjcDP7koWpmw5HN1f62PynPRBwytTDsrOhbBxRvh5oP9RhPA1skZ55klxV21b0MQOQe3qc1\\\\\\\/5OC\\\\\\\/qQM52AqTZlxfFahzEN6geuGDdnbESBTLe5SO+2kkyiqg8RywFDpi1DCvCFxIeHIAmnnJNfowp5QC6cTfNCRNajcVOjfoh7awJptMjkoy0wnZvou16TnNaDSP4d6itQI0OKJpbcBiO9Lf8TmpJrfnGmxurQ14D0cVyovnHMzI6U4k3ElfJ3PENgcTjbxWM3jeljaY0j6k\\\\\\\/HXOQZSBrVZN+q0nUfUCaDGn6BOhr+jv1lfjT3vutnDCDWd\\\\\\\/HNAF9hL\\\\\\\/\\\\\\\/eBnOePX71FLrDqKEeURjzNogCdzB4ml16QlpgkDt+MbAvMdlZSnxz5a55ug1fWGZv7KAcG47CvPGERXhBucwtHanT0b43dSVHbdsrPb6UWoXaLHnAy\\\\\\\/n2wN8ShzP6pVbSO04GRYyjnXiU3kfgdSr0QFlcqtVYUFl8TgIyMoN\\\\\\\/HVavmOwLguO0mWNUkH3tcVhjOyO409wJv2gxs27aOyfsNKFi\\\\\\\/I890jEV+4\\\\\\\/muv9ZSn69CPDsZqHY\\\\\\\/u\\\\\\\/nZVzLwrkjD0HUFg3h69bLYytmfLfdKZXRQnHj2Cq0RCebtX5lYMbwRX8imbU6qQ0gz0uwS3UG1quHbNbS\\\\\\\/tlE9LR146Ck4CYhtSYx1GWw\\\\\\\/SfmNuC9Ca1hZS9n3XoTs5\\\\\\\/11xR1XbjxvLQ6MeS91aoFrLUBsPBaSdrB6KnfhMo3r+N3saEcwwfEa\\\\\\\/ogku+Yj5dGkUO\\\\\\\/YePaY3cp01ZbuFTnpJVKgRpA8OVepMxypdTa02\\\\\\\/AQteSK9pL0Ck+KrvLDG5zgArvyhsmS28MraUrym34amPB3BvgfV3Tq6vuj9DmWahsdm53s4LMdcl8haldNvWmGpwegbtp1p3G03Qv\\\\\\\/Fl5U3cW5uyWkgVofyaxiyGCptp4mcT2WgOME4A2X\\\\\\\/f3HO5fiZ5IU8e8+aqS7xpSsOxsnorY5d2lTXDxbwChl5nnUkdxtPB3BvcRAvaiuYYqXcCqd7pb7CLGbU0K5iGTAwUFsT2RbsqaInfI1AOPCJbomxQXCcBwRBmR5s8AyjmR44nxdNPFRjkf6Yu5+H4RZ5xEMonh7YYXe4bKFrW3OCItzxKqyKCJ7mGkAB3g4deF3\\\\\\\/bzqOTJrfeVI0jNC5zM6tqZaBDwvRV7mAn3LZdx2G0\\\\\\\/poRk4Ie544DTDcZT8V0wVPj6IgxappxP8ZaDRwjruwIjBK\\\\\\\/\\\\\\\/MItebMXSMRD53DwAMvsoM+GRyBLw+ojFarLDcWSHuqbGi5bEbVJOV\\\\\\\/V4\\\\\\\/5RE62fSt+kcQLVE6Gr0QgPJNQdPZuSssFmG3TI7Q+WPD7lp4R2kveTZK+eOugZ9XjzqRYnFJYpGqgeTL7LCmd5Cex0kt7OL+c6Xv8ldK+8WiKa9zXd0+6n5eAjPa2u9JulOq6pe2EGlUUpOsPZemLmdeTWzVa68iXDAnH6apu7hieplEzCtY8r7NIZu2MSr5U1IYQng1l2Y1lEzc5yCTzAM1TcizWANOVkOrGaHhWcFftUV5wV68UwDUYqfEmeVL7Z9HApjKkFvr86MmVu16vvba05DhpJQVc7HLUKn4hnd9EreEACGV83bJ1JwYLn+EwtQLV4Dyae8Gq7fJVgqEcb1fY7tLgSx7RUNsBHMC3jcrimKYrSyOZfUC28gOjshdY8eOSS4SGWH+wC2yUnmaAirmy4QIn3+PVDPdMHTrdSRdKENzYzr66P5N0wQd1J1B\\\\\\\/RH9VSl\\\\\\\/bWCqMsQySx4uQWlrjWAzFitgXPDAmZfB+w8vaQKZUsY7LL9VEnitiS1l3GlyMoWOji+Xd504Iy0fo9UxLq6dhdDEERHj1fli5Wlo\\\\\\\/VXQTkiFJrs6BWIViamJoSgQqa2DKAysxOhQcVxS2YJxOun4YKx5hc50sAnEI7V4nCWLmX\\\\\\\/whme1627fhH8AcGcKQkxJ8809ahbtI7iZbJFNthMK1WrCEKqrfy7gcQWbq1SiUlnnHSCba8n\\\\\\\/fxeH1j6xoelXdOUfzIGuYmpWJ\\\\\\\/dD2SXd3r5Ou81B+W8V\\\\\\\/AIGOnY5S5CgnAQGlF98JC+h5G7aG2836eWoVhCvNTHycF\\\\\\\/Gu0pa1mA0\\\\\\\/3J2MaDB2XnhIf\\\\\\\/AmPH2MyVTek\\\\\\\/\\\\\\\/DO\\\\\\\/Tr9146BLI7x731Nu2cAzsGVP\\\\\\\/VdgYcwjmZNvlAcZxT3TdtJcW8qPCd1nozgb\\\\\\\/cx0umITbBUAOwa9CrCK3yHYzAjfQG5ub1rfyKCId\\\\\\\/WWou\\\\\\\/0n1a+dbVI\\\\\\\/D8ZdAUbFJm62ayN703r3HGoi046tcvGAC6JPE3xIO9476tAjdBPhz8MuAZJvmwiQhjZ+40Fw5d1dhSTBTfbz5Ste78d+\\\\\\\/7aJeLXRtseIuzFJDVjJIW2ONLwJX+7429vdQGxSk3UQ5\\\\\\\/hCCeta1XIxxAOJW1YkB5q4MA5J4JlLsvAGwWmhdpTa\\\\\\\/Ygs9IB4PDwLw5Z65jI\\\\\\\/98\\\\\\\/f4pMW3Ag4VKUYio9Ceuufn\\\\\\\/NMr15M17fXy7ulLmPtkTffnha4i\\\\\\\/AitHYdDidc0uIrJ5RUEoooEWoYuVmGhezyUotrdwKKpxdeUnz7mjwzbYm9OGpHZdDrhReH4sZKmKsn2FLh4AkOg0shfnRYpqHXo+wKNDbXZa4LNY1I3V70JxUFoM6l3WCtpxm0VzT5eaZRlgqC4tc5Q7I6TcDZc1f2YCpWztqCnlZFIJt1JSSqRouU7RVKoyaVNHNbeV6\\\\\\\/HKv8QGNPExBdyOweTGOqJnQvGcBr2i4aW9W27n2g9Srg1Rabrmwqp3MltemDSQws24zzTQUUbNN4a9l9ttp5hP8K07n5AhmM6SGTciZMtXAJ2N3k4aS0von7wlCKDjRH21\\\\\\\/hEzIUWqp3T3aXtsR2U\\\\\\\/3q2avb1UF2pUiCRypFndMqgcNhyxe2DK\\\\\\\/qM+hYxWmliyPLROoIHcB51bxrfmIJRIDIHeCmWZMmujg8QMjNACYuMe0rGyprigbTs8kWYgv2uz9Gaqa5kNYcko1VZBOXJoAQtzcd+2dPzI+F8BwjGfAqbKPYf6ivCMV1rLFimmmGsYmwzPxj3wXsFe1WLrPc62HjXq7TDGOB5qZ1PfXJTsNpU7WS48uqp6jXOqwZxYdhuCd5znqNEKsy0\\\\\\\/GRh34jgviZ2JhPbLhQZWPOZTx2FMIlJ0rpebKaLwdvZOVsR0FQZ65R+DAUgIHyg5s73WZG2bPgdBIfK90KIkuEYTGhWwsZ5kTAKS6U7fIHN20paE1+XGStOT0DR6lMzqYPqm6izUA1DSyJiicTWZXXDKq1rcU47y9LYu\\\\\\\/RteW4\\\\\\\/+WQT8Q5G4ZGngsVD62iDvTLxNgnjY5fl87uw+7pPS5l7\\\\\\\/nfrmQ6z3J+jPwr9\\\\\\\/\\\\\\\/LztnU2sQQ9zfWY1fAqkoAMRiTDGvA\\\\\\\/DtjVFlsceoITkPqaXZ6dTUeM9dsjy\\\\\\\/vHCpfrcHhaOGq5z7narHLj75y8BXD5412jXWdnPp5StZctX4yfBCxtVBz9EM9puCuTgJBApea1G5XOwNXrG44+lb3L5FMUx70xo759nHdAw3B5ThM4utsmE8Y+XACxwQS\\\\\\\/rTX\\\\\\\/3eGAqFxQwRSnp5eSbcNZ9OKyLo0OlDrpWoKktROLX\\\\\\\/t4XLcFYhnc5xOtpewr3Ou6OA+jHzx0srNJT47JDEW3RCoilFq0D9JB\\\\\\\/KoVSz346an76vA2fl99gsOjqxoE3HNmWaFtYavh87q1MihhOGk18Wty7Syhw6UeNijyr\\\\\\\/UOOpBCVP7iNfGY\\\\\\\/\\\\\\\/nuquEpRpStauT3OqPr8iFWmQgrXvsWheckWXW\\\\\\\/1UYcQDAiNSCpuiPcsphPT8ylwtJ\\\\\\\/WF7K6mXmr\\\\\\\/Ex998auQgR9P9s1LslE\\\\\\\/oPjP7P0l0I54X0x3fcEMresnNczDDb1UK7NN+oS0gSgPoNWHi1nY5yKSDFvwPATl5pmJiGR2XhNSlYpl5f7DAtiOTojpfy0m51HBxk+jROlT4FDIXF1lsvfeuGGVZf1Ren+VvYV9Yua3o+vF6O00MNgQj6j3\\\\\\\/Qn6Ftob4nmvRSk5SVpmu6hNUMHZelzBZaNQY56eYAZN68SZqi42IC1CnjzJLutt31Y5pGjhFXNLb7Ki7tkYVr2oB3qt\\\\\\\/ADckG0PvFs92XfiW4vVgs\\\\\\\/fpsGE6JE2uVcJEk0J+8ZDXOjQGOL5i7\\\\\\\/+\\\\\\\/Kl+4axySonJxKhBk9M0OzH8V5jfFZEnl4iSt4Q04YsmyzifP8WLUklhayqNwZODGZZX7wMUuRnMIRjU5XUxw1C\\\\\\\/xKxh5eIJrtk624ePby1fFC11dtrfIWqbvRiBsAZSSxHTTJLZEI8I9KikSRMPXdZWATjuPTMEh6AWREhxw4Ij1QI8ZrPZpTlcPDydmO8R\\\\\\\/ELtie527bWJjR\\\\\\\/\\\\\\\/BQSeov8twsZeBMxhSwFL6sEqpu\\\\\\\/gAhWhOMKpHeaUZGqWlEwnUGErJR13MistCgwT6mrRPLOBbuZWprx102wFEFlJIvbbigp7+7n1n7j269VsoL7W8W3vj7D9MW5TuIBx75N\\\\\\\/QiSkeHDRxpldF2a\\\\\\\/3Miecv3DlQnRItEW5VXdgZkJsXbGFJK\\\\\\\/pGfVZy69wEXdWTdXDRtBqiN0wu9SqD37PdXM1JGpSx88MTeLSSvGwos3GdjcWwAz1qPufeIw\\\\\\\/Y9DvLxjArGR\\\\\\\/9IrYEru9JiAvtRGaHkzQwNnKl6QnYZh3Y0Rgp6ZkwUDExoz3ud\\\\\\\/7qlgGMBC+Ou9WFSM2EzMAFdFhYFjF9Z0rVJjvklMVkX3xwxOgagVHooVRuFyrMPJ\\\\\\\/4luZAUuDtvxl47pvEEtsh+TJd8yaZNAvqKmNlFf2msJXqCLx+IeYAz856HegGVUHH9LwiYHTkpB5dV39RmYHYZvi7JyXoh7WjeKqT3MQNkm0yQ0nM\\\\\\\/wPjf\\\\\\\/WhwzxBWGayYroZeZUOOhR0TR62ePc3OCONfSV4bd7Gs88G9j2WZCNHPlxywNVHeUqS2Q8iqbGzFIjSOdxNSc\\\\\\\/qk6AdMwkZrrHfSmAACdNGx0GmAF0Na\\\\\\\/5M0a2V+i3w\\\\\\\/DoZzLUd54nq1wtLBjBubYl5vO76zkgXZd5yaOlp2YY6oRJsyQhvk64a25T+w3Lpockm2wWN1xbhQ44YvYjBr3hw3ryxtZ6Yywf\\\\\\\/qr9D2da6n0HwJYlLL8UXUn5dP4bfmxuP1MKOTnhaowAHiFVOD4bI1iatB64k\\\\\\\/Obb8jFLqMJl\\\\\\\/SLgZphEm714WjSDfDrGD6pz+\\\\\\\/LpAoy7xMzJpz2YRXrUQbi5MI1sqVu5Snzgvs0hpf9440HGctKBIJSpnnMsFNpZnTuvxKiNQ2HXoUPbO3QAp4PxJ5P99A1JU6pKsLX2yZCLzLe+Bxi+rTXjyDNCtqgasPQNSL5NeokdJG940gcqTAX7qbd+rNRxn0LsSznrx6cV0UOxLsoB0CjMDXnOkB8e\\\\\\\/n7oy45oby9lXxV8fz2ZYu3IJQErZ3AZ4yJ9I\\\\\\\/XcyrTIhx0ZpIac\\\\\\\/CSS8ok7S3QL80\\\\\\\/BBuC2qc5r2Mf8jR3EuHiu9Fv2R9SJRDkgWQrza+0FaC73i5O64zgDQPqZ8T4itel+JujoH5Nk\\\\\\\/Ax8oxfPM3MfhIlsAR+NzqieE9Z6DY+bNcdtTzjYogdq7hsDPK2Y35V+3VpcD0M0lEYMIYqXamAhVzoWXR6P3rDsbiUxvxlUzY4ZOGRyyzFI1IzootLX9JQZUGO9CaVZ8qiwHUiKas\\\\\\\/af9\\\\\\\/0iMaBzy4KnpAFzuOwJXjtRCa4UWiuh3jsdUt\\\\\\\/7zRFtQp5yS\\\\\\\/PE2Gpkj4H\\\\\\\/ZQUbSQ+Zra0j385xDIQCCioFjzjhHSbPVdZa2f2K5H2NaPj87Kysaox6ES9yEvPa5LJJg2IHZ+pCvGS5ysHX6qFoBAqvoWy4x6Dxr5AcYNWsIIyh2RIjTZSqtDbaSjLn5NTJ6VFNobJjipiRhScbCHpVWF23eSVPLaxhb\\\\\\\/2GIC14VElCkENfqNpeMVfh+sEZtw0stfAXJHbmzn0PGyF7E\\\\\\\/axa2S7\\\\\\\/HTBp1BvEHvIEFkTEOtrX\\\\\\\/4xuaZTnI21uGPXCFKBd+0IChdtsKjaQuaC4Nw4bd2HXPo0HZsC2oC58u6ePrsxIKGLajCzPjIeJXWnNnwTdOFWPtg+xW+qCddvglMkiLanSOcDt6g8\\\\\\\/iGw7+Grkqyqy2vQwj3+fTZyn4GxAk6AsLdWW6lufElwyLO022JPE8IcxbTUj+zFZNg4ca31LU\\\\\\\/vfVR3aQJ0zKCUukJ3OG09Y473j6siciwBDeYAZNUXCQJGaWQrI5f28FnKC9LFHb7QbLEN+pzhne2C5HOTILcNtaqB83yDCKyPnDcuwUWlSB4eSwTUy4\\\\\\\/VnS45UWYWE5NJ4TqPLl3kIHUXH6ybQ+f0gBlcu1qMW0Wp+2szhceH95QZIno27ZkASWvebbEdmYQkmOKdzvX82v2uiZ\\\\\\\/mJg0v7b8HYc048yWEVX1eHWYWfLDVhdK508DG88gC3Gu5iIuHaOr8dqPGC\\\\\\\/\\\\\\\/nFVDrKu\\\\\\\/F\\\\\\\/TEbQsDJaHqB1pcj7272PSBTCuJQLbuqTugl8EhUfe+6bRK8i1CJ5Vsl5wBvNDNxk6aIGWnodZGIQcB8xDyDRxzAvZbUTw7fl4l2bJSUeiABBebc8eJfNlphL1nxgEluahNyz2w\\\\\\\/r\\\\\\\/DWmN7ArvZX20tbscn4SOg+QXh7m1uG1\\\\\\\/X\\\\\\\/E79Lw8VsoT76qGFSeZi3YKz0BH+4vOmlMl6ARANpc3Xck3VlCdJaPoimPM2cWGveHBjsRCvyPsi7J2E3Z28TeFoJ7MyCZU43BYWhvT0j+cl7sOa5kU4sTR1hJMNtC1\\\\\\\/0Z1p85R+GltQLxNsq2oZB3xQzUUVXOoJfT\\\\\\\/srK3b8Vs+3s7Hq6dSDZ06GmVrjmR4rdgDOIGLE9L\\\\\\\/uXyp4FoEDlz8u9YNmLK1JvPiTfghyxu9hVmLAUZinhHeTWPsFVQNjluJofayd6vh549IQgp9++S2gBnygu9FSeMUCbBT\\\\\\\/C+aeibqoT8Rfs5mEOHHyrfOleVTF8fYWS8ryS7qbRUf7yd01paTobhkhb4MhgT011vYy6edZ9z0DZ3lPcIgDj6gyXn6wnTRxqfiDOJrw+aRgfsw3j+h1UbEDlxsfO4oorq9JADqgiaFDnrDHloryqr\\\\\\\/frmdDI5tFVmmm+fHKJwxf8ZFVTYZiOlMuP9wRSxD23GbHntLBBgf2Kmrl\\\\\\\/LPT\\\\\\\/aMWad+ItAN\\\\\\\/10opJExL1Dt+dy1uSdnB1Ink9oS9J75drxdfIxAJEQ68KL3pn1uYicsnN5GWOId5G0MEe17yoRt+QvVfpJsBB19y1BqRPi6vg9IThfpcJk\\\\\\\/nKuM+dhA4lwurjja3VDkjZSDQRBgRuN5Wtmm1bwEkAqLmOYNbbn5DZKMatQlmdbkIXJ3oytyHMT5KObkL4vlHbeVmtyZYFSU4ag9rC1Uth8yUwsdxXh3qVFL0xQbvPc\\\\\\\/7Tmjf4kkxUek0u4h+wgi0uwViHDzNgzHTsLL73ImfgDtkY3exOcj46Tc1bYMXzA345DJ5V8QrbZ4GDlitJAsKQGJUYZl+ExF+aGoTRsRgEhjxrJH4tmVvdDLf0gPL1t8eBN6bcEJfLulOYt3+uU5OZ6xGjzgDscspIHILIodFzfwPrVSsOIy8RNK6WCjViThRq3COujBNSuTW6sfEoJOcEXo+vmZ3M6X5ksv84N8GMFyssBgrIwh1J5CD+4DiQn0szk1dIe5FZ1lktGQF2KDRqE+gQGMCrsJYf68+SEFCw8B9rfl\\\\\\\/V5meriQEyLXghAEcMR6NLJMWbOA+DkTbTCGuf2r0\\\\\\\/BL3O\\\\\\\/BmGnrc6scy2WIfDWKF\\\\\\\/8Kchb+0WvLvdLAaofmXF\\\\\\\/2dg3C6drMbOBivTWOkaJ8JSps7fpnEcbcdl5fil7fBhxnWaCdsgrUf5vvecz\\\\\\\/JBARvL6IhWskdySfBnT35I6z2MQsULUCHnoBcraOueFjjdGYT2UI2r9SgSHOJ+oJv1MjFzl2CZbIq62h02+h0RY8a1wD8aJ7QFJro2FTcdlmpL3KFHuGkh\\\\\\\/b1fACxbEPTKHWn67YieV9ZBXzIp7fgnSODneW9XIJ+Tj44xxxXxgOOc6MLLN17qjaP29IRMY83dzh2aheIZQjA3P8CHV\\\\\\\/eY+namN8lWPIBEwu\\\\\\\/nvK9uDcQ0BWOw1F0Jk67t7qvAtVw99b1MP0yhnRRJhAk4Ss47+tWy+O6m+FVScHXYLKiIqbx\\\\\\\/ZpoxK6esa3kRqQ0SJgLKrQT45FFlE32fCd7eVwvm4fQsGDK2cKWwsApw95\\\\\\\/pEqChHVKjp13NNgjW32P1gjm9cnQzqGydQ3UOmVqEOkcz1f5f6myzEQLWCFooNNFOddmmAVM6HJx6e2i+LYwEEouHhKAGagDCSYUxnrZo57Ypdj1wGHQMdpBikFqLSgQsU2aAL+hOKE4MJwau6Asc4CjzxulaCfFQ4P135t2Tjm\\\\\\\/EWGAV0dcqHC4GB6uHnby28+VSKQ1hHQTBRA9+iyPIavgLkMsrEry1QcNidAI+VL6U973\\\\\\\/ZiBYxle7QOh1ry6rPSaHt98ImDwlu8fsZo4lOwYRUaEwrufLJgqQGzHdj2+fRF2YszxL5d+Fq1OwQWst2f2xvGgKENaDJnZb+fWv64D6thQQiEuUIC244HMwWWa9CUgydmh\\\\\\\/PpMv8Kd06dDxS5rW4dlp7mFm4CUl4K3dcPk+pMRQ034YYTL4lVmuPrBEdFY4tLPoMb8WUP+ma1TaVFzQE5dfilvk7MXgIQf\\\\\\\/vEHsdueLIDBvYT7p360oLzf8N5TzpVUNWhp+OKABqch+E+dn6JngPGslJrpY6exPtweN6MJCa83Fa9kGwudoN\\\\\\\/r0X\\\\\\\/cwfzY2cRUYViCDYyX9IyODtJt6hwDOy0uva\\\\\\\/YMKwRysrKOcQ\\\\\\\/Yw+nW4UwsW5WYmkTAs\\\\\\\/+fcExNIJWzYZBczRxNtl1PGuvYg7F0PoZL2\\\\\\\/DwXzCZIrgx1Cy1Ej106eXvG9IvGAO4IepYHH\\\\\\\/dFo4HdqqZ3Gy3mIs7Nax60P+CCgNLg6NdpqzzbwJ+XdbGDCET3SgaqqYRC\\\\\\\/dn+h6sHObW55IHxX\\\\\\\/VEF6RB4hU3vJOT4DodzXJP49Df17pzYQO0N+0\\\\\\\/PtMjUMic2Vu3sk7aueKMKBHRvZuVT8rCdpsWazIv+hupvmiJ6loeX\\\\\\\/0poMC3FvB37uYWRSx3GJth8DTCUlB7vCA9dd3X7deey8zPnk2jBFE4ubJXefxgSwdQcH22Wv4j9H+5lu8rASRsY1TaMvH7k4vmk9GTn2IH16MU05bxoyyCotgIJwd5oLXYiWKJ++q+wWc1wiJQaKMtSCwe\\\\\\\/0paW+QvyvH6EKL\\\\\\\/DaUizOk+SMYcRiFZ6KzY8UGogaLu83xRV1MF6B8VORZuTta9dZnrEeCSav2qtbknjpkHd6b7Dnjr+RM7H\\\\\\\/7Rr0NkWXMIjLd8SRKr5CoNoToP6tGx5qj4v6HG5Lt1qtjfXYPV2oM8\\\\\\\/TqAWLbxsnwh6yBZs\\\\\\\/ieKp1B8BGCtv7KAfg5unsCkx\\\\\\\/pPbZeFCmaD0VFU5cnlbWOXLMB8pdFpNNLiGUCDHOkTjRvtiOcdYHGpPzFOa6axXayTQrg2GsAXemZmaCM8zX6yY6l2l9m7Uv1ErgAGL8T99Er6YV\\\\\\\/2wUhq+\\\\\\\/JiNnSh+eGtExgJfirG+BPrGgHpHAvlUZBY9SgAgLRGpRkS4kZ8xL1wN2HfloqwW6nT25fe5YJeUrZGRp2h97fjk9b1mTj7xy7EocyvgChHj7xL1LfNznD9OxIKIpSnqSY8DCtbEJtJBVD65uQPtcRVCcw8GkBjvbTxZa7XSApaujLGSUVf11t1bFcYTsFYKJR\\\\\\\/+YC+XrsqLrH+4l5zFAu\\\\\\\/epo7ASgIynPq1wSsAvtngqPVFBBNK4oMO24VwzUhUc1kExnBCXEzV8gb87d13bMnvkD9en4GgMrAmf46H7d+QN6UnAalrw5Guhl\\\\\\\/lgvStvf0n3YdD78zqHoJwxTELGfc+yxI89cC1G8o4UbApOuXncM8bVNCQHAuUIAchg695d85mbk3YcDBzHg55lOvt8Z5fOZ8Qo0zSbP7b6fZD6cEerfni3sH6WUEFTYi4n2zH7nT0xLDYu1cwBK6QrQDP2v6IILUjbPyNHtAGxYy1y1wnpLkoeW6g8ZAoXYrg7A7qBNEyQ5L5vPztCrZyT7H22I4Un2KsA90TUrfllA1+tjx0mfDF6X1P1UC9jKCWW+mu1KerUsN8Q5pl1y9ly5OZ2RaLUlDvupFqkW\\\\\\\/D2jp9HGIL5u+rJ3qN83amiAZC+LfadC7uw9EiDXTt5yX8BFXmqKd\\\\\\\/VsbYiPN3iixiIGOeg2Fe0xibNTudto5sDxMduZSgo9JplZgInmu9iD00bFquQN+5y4N+UsVdrbGOsKGVWhoNIGIERQz4L5ONjo4lrhy6RrKdcOGHD8RFV9sUcOB+KWwI7aOTNFhNBRD+z0ldT3Mspz6eR+WpgPUEiV+OlBw\\\\\\\/1dWJ6EvKqL+GJFMVs8bpHZ+oL0p3FNZDlo46mNVLWRyKYlXCL9Ms4K\\\\\\\/3l9rirdOMZh8pVnM03bXDB3n\\\\\\\/uSln+AgJUxLwFfqTf0d4Ch3zaeGpP0TCJ40j\\\\\\\/m\\\\\\\/6pFY0NMd0xumGIoEcsiI\\\\\\\/d0kRlRHgKrjsHqcu25O7BgtgkeygmaZV+WQWpfENftZ6Seql\\\\\\\/QaQgGuaVmZ6MpZjI\\\\\\\/CwBaeqg3LI691B7KR3zcQ1PlDenqT4MQawe5lAJE59xTytI22oXbafoF9JNrmohCzy0grji8k4Y6bT+FGHOWcoW9LfHLJ4PyQwXmLS\\\\\\\/Y\\\\\\\/nNedyiJxhm05zH\\\\\\\/JR8UfGSAVNaQxt6UpBUU+diyJzF\\\\\\\/Z0bliYiFJvDErnHDTpl9jlUprFnf1D9+GDPEjkKJd+Hula2Uh48rksH78muxSwUpiMaHo0qGYA6oN71BevVsORww3Bpxfdr62pUCwFbEmtyl949O01XIOlQboAbA4rWlCS8+VQyDnx54HQgcAwYBwbdSOLXy7l4DAM5K6mKFO3loezLpc\\\\\\\/ppn5wM5nMsIr58sapZjGJ7CH+V3fsPlM\\\\\\\/J5odrC+AiJy8u\\\\\\\/LdEUm6XuTdlCuHpq+ZCA0j8JI\\\\\\\/ZvBte2EzN+wnUTAJh2QXDj3qRD3yv3EY55pkTzYNihWEbIJb1dFqtS865aKdCHul+XC1uVziOpZl0PgaM1Sb\\\\\\\/91Md9RFSQnpXHsBZ0xKXp+AKb2Cn0Ef5j68byX+WP7IQIQxUBgyq+N1tSpSzxzuHVZYMMeWyFJ+F46\\\\\\\/iLmU5cJ\\\\\\\/S9PKPkFFmnJT6dn+oOyqjD4TrdXWYPANgHU3m3avPjj1VJvBnhEnzfRLcWN5oaL3bXt35J1fzYYt3dGDYmF24cSkl0Kq4n\\\\\\\/RQ1wXou6LhtcYrXUyeGY9AMi4rKIPGpQqZn4cjXyLsBW5ds3cQc4YYvgwMGeRDleeokW44f8C8ArsdOj2os3jnpXnqyClgjHp0hsJJdrZ5cv3uYNgmt5J26BCbX5tUbOHQVHz29hTVcw2\\\\\\\/ekXS49oJSuO0DYhgJAeVypiFuOqFb4XK8QQbh1ix7e5x3q+OF1u8SE5d0x3XbPmwUs\\\\\\\/P2Z\\\\\\\/pezwQvaa0HQdhtpUfyKn9BVJWgI2eXpLQudCKflESZj3NhzY8Sq40qwV2q4FI50VdgGSGJajMpLORb\\\\\\\/OHFVc3fUEbem82XFabavab6xzOuFohoozZMyHax8\\\\\\\/\\\\\\\/2sUAFlZ1N8glML\\\\\\\/hiWXjDloC1oOaYksM3NTI6DM7oONvqso44Rk5wXHspEntUP1aSwp6\\\\\\\/PZ6p7JxBY\\\\\\\/mRAgRSZVNuklY3wMfmfEGDWgbrd6HVK7Rfatykzj29FDN7K3AEWCUpRZBQ8tzNE103V30QQIzGEHqNR4Et6n+rxRCIrwC\\\\\\\/OG8XSAIw649HhJibH8XJK1NuQk3DlVsuDpCiK4GmEd6d2PJWFgyDLbve83rig4WHrGbXl1u4yghsrOYrEJgWVgwA88XAduDHVP7\\\\\\\/Lsk2n9xE\\\\\\\/m24V9\\\\\\\/y9I33O7xo1kke+m8aWtTmLjdsjAT9tmpqv99v7u748sFhAZhIzQeVc2Lle8RbBaMThMW8wiCaPMCY4C2sJYkhNiaAdTLUNsxa037hxhPH+JF+Jmu5z6Bp2pfyVPCbHBOk9sEQU0QJI50JlNYZrDPEZIQg2WF+jmoxD5g1IvZLz\\\\\\\/bV\\\\\\\/k64eHj96zuD8KfQB++DyNxVoZnhMbl5IvGDuBxJUbM+0u4PTGVbEJxSAc\\\\\\\/nnny\\\\\\\/iCZIErBd\\\\\\\/7EcCTrFpkfLprpQk0OuqKZV5tgqrz2+Hur7smhp5Cw+SJW40VO18XkEw+Lzbp+wwu7aXI19vzTg5lc\\\\\\\/f+dzMYvwbMphpuGxEZqn7rT4I8edCFtS9NcLJYr9RjJ9JOCQegyah\\\\\\\/mW4ob4KcfRorGEBncrBiCRYqaWjLmjL2zENAE3CBabMpg6jpdIkajQ3N4PqJ01PPAbUIbnHrFJAkZSdwmd3sRZFK36pEflVmtjokJrbbj9lxvJl5Df1xPZn1699IBOTHX4TVUlPvazgmNMxtfGZl9qnQtQWu3SK2td0VNzg9al0Yz7ejmX339qa78FUGFInajofLaUiGymy39F26T92PW75isLhm6xDUPTgqzU1VnXPFc9zDwXoE5WR0rff6jwjLmN\\\\\\\/sGf5z8rL+Xlrs8FX23VsaaVWuaAp\\\\\\\/UnO9V6Mgpz5GuqFizkbEpVFY5v0\\\\\\\/f\\\\\\\/LF1XVkpJCTxdtsXNQMEonN8A3pRzCFDI3nJcM5omkaVL0Y4slmUhb90z\\\\\\\/q3YjwE+adb5wh1xuorpqcXDDShPlbSCv+GH\\\\\\\/wlVoK6RDyYXhRfaMr+pl4IYy741gBbFtfo\\\\\\\/Pv00yhEUQxSnm5exPapG3kTWclRJBJ2jyrMf1d3kBH2FuNo81bCC3Wecj61rhtOUKWzpDX0HnY1inc5cNX\\\\\\\/NuwTWZxGgarEFDliT5LLvyix88bJksB9IyZ0vNc7F6wxZdbLY\\\\\\\/I1PAqyMgqGtYoNk6rNmsRNX+TdXpf8U6Z7PhaZTr2\\\\\\\/xpOft9XW3Rv5PJqUEWq4ZU9DKo8e2JkY8wqBsHuALomKFHOnq3gYx\\\\\\\/Cc4iqtLwDyup8e\\\\\\\/G9GLPfYGCxZfDDjl9ug12runny6\\\\\\\/whFFie2XPLvRU+1NtIoCnxsOcWrryaIDbav+JF0ckrdUKW9CRThKvRtHyS4WLbv0K6TyKSqF86LfVGHCnAWRpLBEWJtDUNXeILtCpDW21a8SaMVV6rWYSIKWx0uhFzpVIEq9M1GmwgIhi6Mt1m1SqaWF9W8epvUwoe+5Gdpui7eIeNStvGLdazkMnVYRqeSO0M2vWZVNz5B3HTlBygLx5OMyPZ8\\\\\\\/xskfisth4O\\\\\\\/M70U15e96A9IwEErVvRqR0M6\\\\\\\/StKerkAt6+yBmHhFaIke5gEudXwoxFwWVmXlNjhGchdOWOw2NpxKMgSGGCY2MhbLlqGSbUowss1HJXmMvmVXSpR9OPqKkZ8yyUAbbT3EA4xPTFDk3A89mKKK\\\\\\\/Bfcd+3Dcp9Yaq6Lxcb1p94weRyWdgGolx4vI4OdsXz5zY72O830st\\\\\\\/9c8zcrmYUnyLlXiGR73Or65uzKOTJaC67Y8d4vmYbB2Qmk1DIRR3WxcIQ+kNlNjki7EpoSXyTZd4MgIamKCyv5gfLy6ooaI3LBXFHmrFd6KIPyUflstgRnrmLB2dBp3ae3TGCyBxHuWja7r8Zk01caDHQSoc2TlL33CMAkKKigeX3I\\\\\\\/NjHcvqPilKLnJobHxZ4+nGam2NmI4XfzhncZiA\\\\\\\/0Hpzfe3UH7Aj+SBfy3vr8z\\\\\\\/yT+fqbs4QdkDWZEjgVIxh\\\\\\\/f6YBfhigYyDRQmnRes0MZeelDuB4JeC7rZmn8qg1DdxG2KuI1VDz660hh8HWmf2rJOn8oCYQdTR+5NKXIqz5UsTyBtfl6Sq5pKvh8dxRSH27MHH6D3wyDNABLzHNxTfi8Qb78dAqqYqB85D4H7J9GXwnbVYWBKXD5ukLbE+McmKF51ds\\\\\\\/W9RPq3xMVkthNZgA\\\\\\\/Y9ZpLX9F8D+wyHzrCnv5DzXLK6yC7oLnFUywIUAJSKpsiwLdr96zyGVklAL8w\\\\\\\/df97O4LWmnVLUa2uTUgfYLTmzLCqd0QLlFbA3nolWWCK0n0RIaWsfcIGuqIpeYMMDHW49qVVN50EHSJzf3Jl\\\\\\\/4sHJIiXZsirJwIzBlpp5PmnzPLmXkQjmciS6Tc3dQX1VKUUqtLfXrrtVt2qBW1oHYMRoatQtr6IzwYeJ7BPTyrzF7LsqUq8xMErEZv3GyR2RVfDYqnSsXW0w2+exAMTzwZi+9RKuSDbu+b2jdvUGXN3chw0dV8kLHylpWP7trvagU5wP5NayYJBpSDuooyOfdviaq5j7NXtJNfnx68v2eAmz4vins8kmMuH5duTYw9kJCTrZvFWy84Oy78EXrfNaiCDJZMroM1PNRw\\\\\\\/52V8Py33y9TjIs4GAGoeQX4YdfgL9KwiQ9ex8LXb5qi82O0EsGREtGUc9N\\\\\\\/ro70xkyFW4ColisOWEn\\\\\\\/YjRn50sm1P2p0CL\\\\\\\/3X11z9lCqu6Onn3z3bkjrErDOiOfcUgYijWPwNcfefl4jHSD\\\\\\\/QvL5Xrp3FL+4wgz\\\\\\\/Z3YddzocTbZhBpFfLBbYODLVVJmm3SW4bdkxfAVkvGhKmnvodPUC1puDLKpjl8oyj6DiG4Xi5YoReutzvnU274T3cdLKOHypGAH2RAhe9EVoM1GsRc+xIdGQINCw5Zm8bvnDArLDKeJinUv7nwoKE4qT7UfFy4mpwN5Do565fG\\\\\\\/KZ+2R8wT9c+IVL01VFWuVegNr93JR\\\\\\\/awcEk3kcltjxOUuMzknjtzeIHdF+c9aUP7K0tjf7xpcdZ1aI2uUIJVGHvv5C2jVZrlj6EjhyzTbQhIxUtgGXfdZ7Mw0GbeYdtfIRYrBCY479eypSa\\\\\\\/Uikpj\\\\\\\/18uNJxKHyHsSFd52NamAK804OtRxn8e0rtbcF1GXdD+2HAVWIl+j0pQxjptGxXp5bHts5I5QCzrnqu2GfgFEr1SpWAEDp6IPTyjRznt9LPpyUfnhTAlCgEesrSLIbNoJ2MDj7ZjS1diSJI8miXGC2mz6UN2dWh818QzUpJySgkWTHpNZg7siUk5fcDd3\\\\\\\/wWcDdUpCrnTTGm3fQ5bMUgGaeGPl8c2qS2gcNrizD1Ym6FGhbIMdRom5UyiHivyIGA8DUfZtGhDtrJkgGCuSTOc2m6r1kQkT\\\\\\\/kN2zVns74I2fIQPXvSCb0xT1FZYDJPSglpSU3n\\\\\\\/mN7cVmvzYT2D6UorsAt7NwETJ2WvtHvDoTRIxgfGI3o\\\\\\\/zUYDZLqWXdwhsT4AQFhC9c46bdOcepyRPV+XzGhkMfKQSK+7A8VsIPxJz9eVlv6+f9u8dJSKt1muXetrN4u+1hoLdNxDr5NSvLwn0cijDvm8q43net69rj2LYN09pcIQjOapLVjv8CYUfOcyGFL7igBOC+9XHuhnL+17AJrFjkvjPVe46LGWfQqfdeFE\\\\\\\/bvCVbAOwHVdNh5hfF+3OcKHob6amaVlH6tXo\\\\\\\/QhSS+duG7hNWD\\\\\\\/3gporEqLtCchv7nngeqLeWBA5X+SNijJWM\\\\\\\/BwXGEFho5hfbgJMyHBoICwbdcdG8krZ9uoNg3AQLBsruw7T93ingGuYxfdZr7PepW9GKdPsBXNfPH2lFT8nAXf6sGxKNn9mosnpgTFGs5jgoW78pnwhKLO3ij5D7umonE7NBFcwBny7ndNAj570UDD160Y6drT2YaiLur\\\\\\\/CeH8Y13kQVC\\\\\\\/ImayVc4caKTca+y3uROpCDHkTiq6qL6ZgoUHPoDVnJRpPxgT3pa7OndpIYVIIgGnp3I00GhTUchCXpafDSi6tSjyCV8B5JT52Cz1N\\\\\\\/WUeC\\\\\\\/7bw\\\\\\\/EAOGnhJz4VjkvUBLIN2oZBzz\\\\\\\/EgLc+mmTZuceGGFkl0qZlYEKTwiWTS\\\\\\\/24YT0U5JgbPFRY2h\\\\\\\/zgoQnW\\\\\\\/IZlYOrY5nJOI1mJ8IDJ4qkw4k7vUvoU5Kjv0TA3VV31Jl0nmBsNfpSlLuk4t\\\\\\\/\\\\\\\/2nQI9bOJ5sM5wu4+DSGwCQqX5Az3xIhS7lXWf1JqqDUGWoWJdQkydTJtDzaF9v\\\\\\\/Uv51f2OAhnvKCLCSoKcuOdMpxok1Kwi3dpcLY22ZpezKroPaUquWrpR3GqT0fxqLhjZ3hbx92\\\\\\\/gLG+YiJUYJNKRWV+TMNw8J7xefFghlGjfP\\\\\\\/Ak6lhepei0ODrq18xXBxkuJKI1mxLUbCcchdDAc7FN5AA4jWRd78TGn6u0U14zdbFCm2hyO+graE1TWBA1dOFbe03zNBu6crhQDC8jmTo6eM5SVVPQHBgm9aLM+YVKkHN16218cfy3BVTLRphHQ\\\\\\\/SEYxCWt4OvK7G5e6bVs0S3pLjcQ\\\\\\\/Vh1QYRX6Oc1zX5RP28Y6rNyQvLAcPKKHbwqszvcb6mHMwREycFIJrBXzIpf1m+md\\\\\\\/84fUyPMXO83jMczgn3Qh5QZg7By7I1KHzud\\\\\\\/i\\\\\\\/U6CxwiHlOHYv2Gvp+HSef34hycMsMR9hbe0VG6OCx4RN3pEjX\\\\\\\/zqKCXrYHkymuz4D4HiH+ynC0oMsvpKbYV3FUV3Ksk8RINkgdsk0X9qsxbvkReXQpHzdQfLD+vqCejoRPAoUYRtIpdKz9Ligz4VMTLnqslo8CnIAd43O+n5risKEnXdS07B0Miu79aaqVPhDPi6NMPM0mH8ADhdiKwGo7iFByQ\\\\\\\/jLtfJT+lJJ7hDoGWhhGokxtwnDP7tHEoJcJTXGmobkMqhMAL\\\\\\\/wKDpa39RLOglMleTPqBPQahkpoz2N84HaRQ4r5Cau4+h1rH0I3l0kIsqxwfOxGA39pNZepsrIow\\\\\\\/2SaITuKbbyEmb+\\\\\\\/lAbZd40v9tbhlHUZYJskvdzJC7lLfqYzzWdBr6KlQ\\\\\\\/LnYgPlXQyeotCyAxV+pFdoUKu\\\\\\\/tpIw1byo8AdNgNqAU4JZxn0xLh1E\\\\\\\/8KQsCUoBw4ScEUp0g4MmqBJ4u81sI1bbYvfQOFvkAjetC9kT4Dl7dOSc+tl2Gei8aw2mM+XYxD4yWFzSiqY4dKb7OmfN11tnxwABgvrZV+6zmemS+JYNIisYmXBEENkFGHVeHq2ykfZq2fnLT4APdCK6bfjm\\\\\\\/AjBl3Gtuz3lEpTiiHiDDBpTfiQrZEkXWsU3pKgo2h2Qpxf+vZKBYDtwox1LccY2SYj8rVYi\\\\\\\/THJO1Toyes3CGQVEEnSFvU+RmIfgYat+en2aabpONmtS6QY0hO\\\\\\\/Mdp4Afmusev\\\\\\\/7Rh\\\\\\\/BsMPlSVmyZkcoW\\\\\\\/VaFYE42tm2Isl0R3mWCjNAL084ZH9L+4W\\\\\\\/NEiyRO3TsYY07X0rpN+aC44zutid9wFuJMIii09tFqEU9UlPY8NqILvYC4aI7z4pqA5mTCBbXA6k6DkDi6VbGUYJnj5WdECFf5orMZjyfdpHTx0I1DLSahgQ93s909EVlbuvJaz1fWsXbE2XPx\\\\\\\/neU1yFR6zKAT\\\\\\\/EH\\\\\\\/IfOVBwX\\\\\\\/MH8rIz7sejXfIe+AfuJns4JylJN3yTB8bYxRpKqCg1Ka\\\\\\\/SODF1K\\\\\\\/gOASr0RL3LqRJZ\\\\\\\/x1lEbMMiqpq0ZR0abw8d4C3EWRY3Lfmg1tbnRTnhd6UFmpKKnk64DiPhRGD5YTno20VOgFhIRdVpHuAamP\\\\\\\/yGb6Xrjtvo6jW5CL1R0ZhkGrBL6+RgTuwUmONzKDdOa3OsEy7tC6sR4R53W5wQeWSUGyL6iPjz2Svc\\\\\\\/\\\\\\\/FpOONUFPu3VHwKWe41\\\\\\\/3RZ+KjRSTfpHVvd3jqnI9yg51wb9\\\\\\\/CcpjEXDwOeSP8OCHoDZmsATc+cEFS3td9HIx4lNovCC06GT2dotA8zIp4AHvI8oeuuoAuuCblsEy0tjxv1H78UzDTODQNCuvEZCBa6cIrZKlV\\\\\\\/YEl5GENo980ENQ\\\\\\\/jes7kUmIFxFJG3YpNNa8gJlmmMx3YEVCnnceJD7NBgfqvv9z7qcXVWStvL1kuLdrdcMmbgqKdiHVmHutOUCjprJx76\\\\\\\/bJmz7K3GwKThfYpZfstCkFBryTipxiSQp9JLkbhft3peJjGihHQYY6CE+nSC5o0qVKZvxW2oW\\\\\\\/jExDMDGRjHw3vcwiSScFs4jLXMl\\\\\\\/\\\\\\\/n2SSryRn+RnZE\\\\\\\/BMaeNrKBKBWYi0yy4RVda33dQ7qWttw\\\\\\\/pm0zCzSIp1Z1FQoRBOmOrZn3gFoYNj9YDUaReW\\\\\\\/HiqIc\\\\\\\/KyZeY8LxxNLBJIae8R1VBePh8YZ5T2pAXjoKSmRsN1cw2V2xvhHaXczkg1FDxQkkzzsKFKqd6n\\\\\\\/WjYlF7QgfpH7s7iCSyTfR9tC1s0y1UD7L0WPEbEvRJFutZEkuymeMFk5hLtviKRewACahg\\\\\\\/IKfmTF29k+31hE0qXSiXksjR2TwY63iL6A6p5tfg1154tFxXZqk\\\\\\\/z\\\\\\\/pPBcAOjL6GUHME05WYeWGBJxZggstEVqRiFCtgIz6yTGZtfofwi356eIh7yfWuZA\\\\\\\/ugmFsa4WBecumKDKx+SuumdS\\\\\\\/zVcv3S6mxDdkKWXjYKG\\\\\\\/Sv1nmYiHkZgTq0Xvx\\\\\\\/EsPmN0DoijsKmG5\\\\\\\/DjhNlY5Km2WVjVfhjgyBfRwlWcMmwA1NlrD3j1xW9eWg+hyqjIicDZylJ\\\\\\\/8lFQpYSRT6ooPUrj4y1ka+iKHmtujiQ8bQ0QO5ILorrFoIaZzGJ3elGQOCLpKHsBRa054zXqAXe9HHWuL7e+OTmgKb2j9SnSnGNhSJ+KBZWLKlr0GwQhWLEWtVaNZXGlUTdP4+9ikUJMXTlZYyYmZostJOqM1snt\\\\\\\/7ESLRC3VEXxJt9sYo5FY5XE2oXSXHq+SbibzkXZNU9W3BxFNvI7cMn5Ifppz\\\\\\\/14E6efBuvI9yGIPnyjXZILneA2LCxvDt2UiFXU4WENUohyYXCcrAT\\\\\\\/Ud7BVVV5esvQsC+LUt0SCkjNxmOoqSpN3mtFN4B+8BCt+oLTCRRiyN6QftHgGpWOUMb193GL03IdIWlAZRP7ucRgwkwSobRTABvnfq+U71onLij8DdwKm0QKLAyEylsbxyhncGvGcPDx1cHU037lalrmG8Nn0cl5JHDq\\\\\\\/GVNm\\\\\\\/zrMDmOzxnhFFRco7mlyKduUM61JSqKeoCo\\\\\\\/ZdzARHSUjZw+IsaxYjsGKzql5QgnvVl7WYHGbZPdL0lLSJkOLQhk7WpoNfkSRZYsm6JXFIwM\\\\\\\/ZlXV+\\\\\\\/CetJzdlCSuPv\\\\\\\/SyVoFZ132kIfDeInmmJqwxOkWLDiC7VZof9i2QCp\\\\\\\/Y2fGL3ap1kdk++TV+1PsG3Yoa\\\\\\\/XDdgtceZ6rRsR+lORNhgf3yRlcQ9YTL\\\\\\\/6NFdplxzxGbBWmlQMUIqLxJh74t4yb31XqjVs4y6onTqIem8bHzgawreObyHMRZUWlMLuzBylOq9bvUxUgixoTb\\\\\\\/qv5iqcEjjJtm5Lp6FhF93V+WorGClnAlDHmjTldkHqh2nQr\\\\\\\/aUI8PEI\\\\\\\/jwshbopA+KxbhwNC31kO55OBT7+qZvyv7dlsA\\\\\\\/LFSKueNiKbdeZ90buVYioGJuhRDbXlEDJIb565tj65+LywCWjygsKEScpI3XVVp7l\\\\\\\/nM5kqWC2O7x7xD3F+ORL4bSyC47iOcNj5l+n+uR2WBkAMcdXKUfPdJtBhCbbKnFNJQ3sKAmf8LjsgQOK\\\\\\\/ry2TkaU9ryFSR3e23IgVmxQDRYjOKfYRahKfJRzDNIwVTwA82Lx0YLlU80hCB\\\\\\\/FukM7KlUY2OpfM1nYtDJgtp695cxIpTkNwFK33gV5ZgvC7sLAL2H4CQLMzGQG+agIg4MFr3Tw1RWNDLkzR2dxpckowfUB65pcZyJExyNuMhRMzSTE6XkCrsxr+z6bofNkAKILVw2YXJdCqlC3UvkbR4UEHv1RJWO2TlU7THr8yWnYXXJF5Eeh1cYVk6DuvsJ9l5gQJBroRICx+royiF7X7Sc4oTW3itvD3CF07ehBPwqQDvEnBWF34lcaTnzxw0k0ylrpf1+9djA1hda7dy5pgLn5B5JWzYoxcsF8IOA9rEBmjKTV0EPpM3Bj+CjAu9ARJdCvapHoCf4gQjL4VJhH9Jn5NOJlg1k8qPkZjVgZQ3Vd8aXI96dE4\\\\\\\/0KzT5982p7U\\\\\\\/aKnGs28nz6jG43GbY7OfKWYJZtEntxOk02lGKTXZ1uAabWO3MFOmbFmb0OtWU+PHwKxalfCKTdLPnfYwKWfEQulEPR2FJHc2rAX44qmv0e5LZFrIUN\\\\\\\/NMdlP56pAZMP+xJCW7\\\\\\\/Zna61584AMS\\\\\\\/nvhVho0CkBA9qIRsKWSOuWh7RfXbFrmCvfebwSQuSVWtl7M3ID8HLMBlXD09Kkgx3L2gzqkKlrwgxvpAN\\\\\\\/HryaUhnTMRJHjVgsCWXK2IERfGqpCl\\\\\\\/vBZN82CEuLXNl3xk1\\\\\\\/gtIXrOi\\\\\\\/1pDNCFgEbVg+F981D2NYzVK5q+YyVbnJDVxcKUeUkHeqngIlKtu6dZHJgxwWUSeknxtOJc+ZGi9pgeIDvphgDs+i8OJWPRYY3zVkroHAdUOf2kBUqP7ZecU3pTWxU0hMs5e738S8q1TnoHP1wlWhwQSJYl5bsKZUDxZsjoWZvMmQj6HqFtFJEcDs2YiqY1uE21c1nB7vZYxZjeyzCLJ0FjTrLul2sCAc1CfQ4uE1SYnUHeUKdsvNzuQFaHieORdDG0EGx8BywqdSn52MTJlyKLeM4i2754HiDRUn5Kkt4LFXnn94z3vPDG5+JyPO7Vxay3iukNCMVY3UQySlkVGBWVThoPcF0h3+WMA3C5BKfLW70pNCEcwZcFjVpx2rEPbmj30HbdysAfZiEdeRw7VFiYTiJatjtyPt+lmbW6sqpXiosupsZokOTF2PE5uLFdvZ1p0NtZ4PLJ748oFLZItfUW9p1fjnHKTYQI3tz15doLt+ZQIL4TVgy0t\\\\\\\/n2BLZ0b2v10oI2zqOeKI5iBkmIzb2BecDiM3OXJ5x6mTqVjfTLw7IJbhXDz\\\\\\\/QAXrX3uKMO5Z3Ta10ClsDM0vQC2lJt34RMFTMk8osieR72sy8hmSPrpBNVs4IX0uMw4IWu\\\\\\\/JrPq+voLR8VidRubpVGSXKx4u5GC2AP18tI3nHxiTxBp7ozV1ezwosVCQIWttJ\\\\\\\/Q8RFUjA604VbCaUdL6YS94QcvpfpM590aG9zg8\\\\\\\/jKcAXg+vW+HXNOXU3mXpNH7JaDVQKNHTJHJiFCnAJzUODFcAIDNgYStVqPViy\\\\\\\/NLcWOsKTteO6D+Vge3+YuWWYMFkZrIQlpN+ZkJRwirKftRcd2dHlYHcngirNKNvxqw3Fqmnq5q6AHinnpyVpHooxCC0W1r0Ck0UPL+fgmdWBlWBrUr8JpXQkg45IeMh8VPsvABV1pxhVUbrY9kdjwUfTWPnyrCUQt7r\\\\\\\/w+hRk5ex+7NJvJQls8pBGj+3z7CV3QdtAFECa+QvCw+mWr7hFIfspf0mJ3qDTcavXDWa71GTgSdzZlnSGuTc56V4\\\\\\\/0WsNC68ahBsqgt9fHqIRoDEZfgm3WhDVglHG\\\\\\\/Y0UtvWnDmhoycTrEKk0pj7Bw\\\\\\\/VQ1dj26kVl7ALhblQjNob57lUQe67xPs26n3hl9P9SZV1ZIAVVidIoLPIq9z0VFzp4wvVODZEZ9CHvvkgNDhO7wFzs1LJVumNkdoJaTb+XMTDf55UZczdKno57qv3IbcLoTGGCZJIbxrU917tO9UKfECIUMq25MJCnMVbZ1j0+WhuZEMXtYMoea7tcNARQ1XIn42M17BYjtonPCbNivsSjhuxNLCI0MPsSrDRQ34G3xWolIf+W4Wisoe0FJWVYUnZf0YKpxv1owIHDOX8yPTJHiNrrVXQpZkwe\\\\\\\/GcI+ZwfZG5du0xrYOXsobIYmChs56fa7lr4RKKSnvOJGsa3ArZ13SfP90RM4y9BTOy0jFI6dQyNG7ezKDeo7CRHD70ORYlsKUa91M5BTPp9e9RVJApvKasomu4r6Ig\\\\\\\/fUMsi4Yv27oRA0f2\\\\\\\/5jEhNlZJx2mdSm5Ci\\\\\\\/vc4txff7XizTABDmh9c3jj9TJ2ABbD09AsJE5zkNlkgjSVOy7ry1EBGtfX7Ll49xI\\\\\\\/QMakCkyQ2yZ92YPiV\\\\\\\/J8AjWziG3kgHbCH2Y0yr8GxhoOoEacwXEzmL3RSMABGBTe9Ql+P+474Cgev1P05xy\\\\\\\/5cRJQH0wo\\\\\\\/PIQvJ1uzqRnX98BxyBHhDzQqHIewXAQkieY5jCi9Lctfq\\\\\\\/RQJ14V3yq+2EM2DFXxBDzrbmAdIhTYOGXmUlrrlx8WP+s77thZ+dpv1kmjHESe9GXwxdNX7X0pFh5m4yu5aFqwByx1lf8otyQIC8CenVWw5Fbkh35u\\\\\\\/R4KP4xCaQRQXj9Z3wt2tlrDATJPF0ywOaHGdeAOb\\\\\\\/o6j5+EpEbfdJuImKmL\\\\\\\/0+jOecA5PxA3mAaS6IadYmIP4Dy5sIGHWdWJSIIhEiwFYsT1ewYoRJ69rKWszxSYVcFXbX5u+KVQMjvLwwaE4Qnf1E2DI5MdU5trcogBOpyA++9W4F6+WcZJoOWFVNXoLhWEGOIgL34YxvETcf8Bz8PWTzD9A4+kDbj1tH+JX0TKx4C6uWwwdv\\\\\\\/Eyn\\\\\\\/kemubNLw11DsmKHW3uqy9Ps99+u8zar2MJ0NZrGJr8\\\\\\\/iRNb9xhbm1rSwY9QptQpyCeIccTveOIPA04H2PhXFp1eWVYyoJJEBoeadb1dHoAkf0JYyCwihDzBeQwftUGoyiE59ZMO8IYRC7kpvzWC6ny\\\\\\\/3W7MvdmEIit\\\\\\\/3+9w+amZFgpnUL1bt\\\\\\\/igE8RmoLu2UX7\\\\\\\/5HiJyoTkyhRPSTR7PrFH2TYHeIaRcd4wnDaNSXAeAIW4t5Rqd2SeNI6RcBCU5sZOIkomt3DtubslbySBQVCmZgtbi5fBKOh5lW6Hn1YBd1Dl5MuOeFWS5FuAwY8Ct9FSJaFEriTzW0Md0wHDYmVy4vz+I8xBDonRW54zFsB5T9bi+NJsZYajoOeWF8dXpK8eRLfcyaM6FT59r7bYHLMjNWmduPPPNFXrBUCnUlMr2m5r4d\\\\\\\/3RZduZufzi\\\\\\\/4sKx9o35Ev4Nf7LNSDg2TuQEMupO\\\\\\\/J3tCtq3vytaWBP+SBBxX+aplrUX\\\\\\\/krryxuOSGZIXLPDG6AmsgTaCbwHky8x784YryK1TohnykKKW8H4qkBx3WUDNbPdkqErZBs\\\\\\\/sL405IEgzspMgy8pK91Kmqz25OMKr7iA4gkA2gR1XHiSwzM+Xz4nTl\\\\\\\/ETmPRN9OikJKAEmLiVXeseZwL26uINgk3zkkm6GpTJJgH14mdQgzMMt0aEdmZPxSnYVAejeOmewqGRw2ZJOmiMdmQsbSqNHgEqQ8umTgn15O9KdyPepU94tcnnusXE0NgNOyAPW0OupDNyTOw4mjchWfsF8iDwm5W4RSM78fq4EpfTal7fkzPslGb6ytOAQk44Fc63psudVP49EW1HJD\\\\\\\/RY63PVX6pawJtk0r+i+qXfSLY\\\\\\\/l6BampOQeoH95L7DreqDZwPuvxrOCn0nnjbZRK\\\\\\\/3P+2i9OflgV3wftWMzfeTQtEPa5TDM6XwwhuJYVqReo10j9oOkKgKsUtTz\\\\\\\/Tsm41rqNl9YJ7nrbMEV0QtYFFdIbaBYt3SjuYv4UBmHSF1f+SYwgr\\\\\\\/ScG5wF\\\\\\\/uc8rf1k0r0Pj20na\\\\\\\/jphsNpE5L9NxLTPSfUAmZ2Q9kkz4Zk1T8k4LcvyqP6qpfeXb+0XLrtpDi4FVHLpTy1aQNr+lOybkKtjrq1w2wNh+WMW40iWeLOUJ5OPURUUJ\\\\\\\/OG1UCQTpiwNRRkALJ5j5NlSGbVWuLs3Le9jsXqAOfmfia0+gV\\\\\\\/T5EXZhPHswushNPCrfiVrbUrThLlChpWU+1crK6uEL33bPyi6eX+++Fq+rcHC8w6bxr3j20zEW+jHIf57iKNKIRjw9hbRsCz7n6oolcLMD9ywWpbxEHlZAMBg6VHPVrhVKSPCwIiDltHeuDqTvP4AGZottovJpKglG+BzM+sIZHAFteyWY6xdQUuk+TtOw8xYIn3qkk1gfLnxHNJY\\\\\\\/aqOrWVNi2pYt0\\\\\\\/O5zLCLjyvyX0GriooJ57hI1j8uHYSpzMBlYQ6R3HRCr1lQzOGAEowzlZgDrU7igBkMlqqv8krp5kQXJEua3WQ+Ry+1KwZK6ftNZK\\\\\\\/\\\\\\\/uTGkbSHkyL9neXtN57tKarMHQOOY3OJtYEA6\\\\\\\/H2tPcKXNqakjbTE3ZAdyJvrX8H4qGlxvEI7avW7GzBjHoNzCPYrw6CBtmoVmKZFOur3sr7QJKXDKG1NzRMewoY0d2a2iin2WjY3pewxZyUBe7vOxS0EfmlCHajEkZsxi0fnbbhsxyoxUfZ8m4q3+K\\\\\\\/BLKM\\\\\\\/s90Pk0GcLNkoxShUYmjdGpSK4cIyfVqCx7dT\\\\\\\/SRtui+Xlvtyp9WwXMzQ\\\\\\\/C6mJnaoQp8DCYRQAnDVsoczinKn0q0f8++8TKOkCvQVgsjrMi393hwsG3nkgXMr0E4L4N2Thage1bm06gRFw1hQ8avt8xkZRbVCmccfs4VZcN\\\\\\\/MCGov1r12H8jsSdD930oSkThlAuR\\\\\\\/CcUJFL7R\\\\\\\/uSuP09erx4r4nVvO4aeNTcpCWEyAiIGXVNk\\\\\\\/0C8LltWlEEi5EN9hEmaTH4KT+pgv3zYFxtYF1Yr3QbLpjvGpTDmdCXFpL\\\\\\\/+pnH7JDGVG2q05v\\\\\\\/yaZkzaASOMH2kmR0ttPKn7Iew8DuJ2uj3+IXxmFJFKPq8z+1iuOaJ36Q5Tm0Jo674Xl1y0F2\\\\\\\/O7sYCDhR00hK0ZvQiEmH5ueWnRfz3sDIjCq9KRnRCzsGpBSnYHElBwpDJi1Ve7y\\\\\\\/YIqd3Np+c+BU+V5H+EEj7v1H5FiGVECiiZpnyo+ubsr9xFefx4VTLNbllWJ1rbo7OHJhkzSAyE6aTA6qJmWq79nJ8y1Cwa+ppNyy\\\\\\\/h3c3nziR6plRacdk1GTySx8\\\\\\\/NDKuJ85u18dIeZebiu7jFKX0nhDHpfzASzC6mx5EyKKyKKyJ8k1f+0EUvQhAXK43q6DEGQlZ2Nx3seozTI7sCOqtEpvphcg3PkvJ9n8QXrik9D9a3O5i4cdFeHltjJEREck+AvHPpsGaNOeY0wCRKCs93h0mQLItAJzh16NpEv0pGkyOmmbsR90zIfG00BMvUAN73l1algLliprm1lb55q\\\\\\\/ght4FViFq+M7UmRyepaR3zSeljEWdAKr4I6HIYFYJH4widWHMbPWk7gNRk0XU41gpPRY8EPi6898B\\\\\\\/Es5kZw+DO0dTtcDsG6cdmiMpWLDeX1NKJsbGAuH0igwlYRSTFm3k7D1hK9HGR2wZ+cJbICES1i9zc7neBCNtp0KpDzGOUkJvS1RTG1L7X+s7oQH2jsNl84Vve6hr0kW\\\\\\\/p601GjgPAIweg4SBNEuRdDTA3A7AXFBegepnpenHPGWoJtudqCj+Jz033rkEnpydOvuM5I4yHSY9A34AMRvw4XKhuLaMv\\\\\\\/vdlnVN0WaGwAqYSnGKehnFNoJnSIgpQfxokIwAPAu+3dg0ggJJVzf8QHT2SUFGJhNkE7eql5uHqlulACK1j4ft7VxtQYnA8Ur4HSsBSiFopQPxfZh7AZecQixp5vGKWUJtiwSOqs8oWYtO5akKbpZbuetbTOxb5VIRGjJ6FMs1yLGuxjZ6BS9xajY8dycrZNnbcfwhwf+i7mfK8cfyetxnEryVW997prAM24lvX8CzIMPigbZrm8mI1zhBOBw7qeYVXchntDvbB73WLTyRj1VgC7Z\\\\\\\/qX+JWCQHCbD4AMINj3JEsrB3P5EOJQetQm4TqAWWYqllPktyUQ7JWLH3RVjWLqn9Y1xKtd5tEgZRd3nkgEmV33dsEaNGsaZLP\\\\\\\/SWZsMK\\\\\\\/9c8Yx0kcXDi1CBJj7jtBy4W3LL6R6NsNQM+d7ahsxfAfdtfrmc9ZJZJmqq47eyawKTVT\\\\\\\/x1H6PP0+PBaxqEHaTgvPvglNNyZZmhu9\\\\\\\/Fdl+dkLZOk8DXAAdzc3tsc+9qemmDEF5DvRHQHoMDtFno9nF4AYlfYaRrtGeFKiuCK3aDUx0t4HcYishraGhE\\\\\\\/SwhpTuuqUGYChx9wJ8f22RxkjKlGrbGhLPFDNPQyQ5L7wDw4EgO6ZIyLEdBrIhe+CiW3CV7Xfe4j4FCCdo+tqXz8VPrOU3GvJA7SDkBlwepjex7LRjv4UtQIgnXK0dkKrD7rgQm9XcCqdYEsdES1L2JtCKuIOH9J\\\\\\\/411lzlQPCWUWwymyVU38gVy67d833bmvGTaguTIS7GYKve+Ql5PVkGPqcxcGSARSFIc1U7yCl9Rp9Peakrqym2\\\\\\\/Vp8CLS+1bgUS5vE3gOvKEpaE7G\\\\\\\/ymoMKsnCB+hE5EFkm1v81+pOg7E9eK\\\\\\\/2spCdEB7byOMjpwKgfyeFaBR1p632PHOaLrlvVKxdLd0SDM+giXyG1I5ouuPRytqhzELD3RzFcLS1uEBfsN0sc5o7HIZjGaq2+E4zMnQaL60gmJaC1Pawg8r1na6w1GmB\\\\\\\/udb+gi\\\\\\\/Z8nOHwn7SDAyUdR3UzZRV7AAjSaTpBeBOymDH8j+PK6V0E0eo9l03nn0Fn1I9nfbWMehXu4ZzYGq5zrXkIjAic5aKLUmWskWmsubPO+8JoTKhiX5T77O+NAQD1IeNimHkCHPAPZRr+1kriCFPryqknmc9PIo1bf8dVrXvVm2ilB3S2kSNv6SxPmWqWAjCF56feSpdi89Mt8LvKwhdDHf8fOXmpXae8BW4\\\\\\\/fwIg\\\\\\\/b6hlVNVAnqbJCIMeR0VdeIr31ElBNQZgzXkBe+Z98XHhUbS\\\\\\\/vmDg05kF6ypjbwTcbcpMBkincA96hL87RyCYUPE3EUs6uxodlBNb3cE\\\\\\\/yB8ut\\\\\\\/nehR6t928gkhQUbGnTR2gtlR9CD6+N2UMb0AazUc\\\\\\\/+esUsN6WEBKJS+UwVybYpr7\\\\\\\/B\\\\\\\/X+85JWrSsyiGGGcROzfGYXqnPT2Clv8FRlJ+NKN8Cyqm+VP1M\\\\\\\/fGlumBrj839sXyERXV9OEodqQ7M75fTpJHcpDoc5WD4KpEMYBrp5wjv6tTF4d32KEeq5V+krV1ksUyA\\\\\\\/w89ymH3Y0hoTXEZ1AaqTjX2hjGkXXhZsf9lL8jsvhrVB\\\u0022,\\\u0022iv\\\u0022:\\\u0022Vxl7wZFMez3yp3qLbOKfzg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002231948f64d11710efe69b66f7b9052eb58c6bb0de0f2f918f0902db779559e299\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022seCeVswSpZYSL+\\\\\\\/3qWSflef3jWfC9gylqKvMPoWwtVVSjl\\\\\\\/KOYSjO7IZu8dPnmNLctGMShOvqC8Z9St2zlJ2kfiptbqmCNfiBb0fBBTbVQ4MGGhiT8qa9Lyxb45mO94XF7V4C3KKc+PTAyipbcJX4nvnJ08xaV\\\\\\\/PaXn+AGDBgT1Ev7SNPC2\\\\\\\/xk5BbrulxecGmn8h+B3HkqVuR0UOcfjLXG554RfMP8cmw9CmxsvmJaF5F513jryzgM41rWL43c+kNU3ItW+Nscb5UnYiHNDAfTfT5tDo\\\\\\\/r+i5EfsiCtLjr06iHqS2teoMbKdadnoHbi5bnM\\\\\\\/2ZxLasmWidwUU0y8qRv\\\\\\\/O5sUvJ+kPiET4mVp8IorEbRIzKSMw8sW0PAb6JLtB0hxf5Oh7DgH0aDqlFsD6Nxi5BPnfb5DuxEOrK0583EPgmGCbB5v1kPIt1Pnz8afS1clJTQQAxHu\\\\\\\/YXgfIrn1ySFvB77zJqju8iJGjSyYw3Gy4yqJbbWOpeNGjOGhUeU3T9PrMd3LvfS27Lafl1OqiA\\\\\\\/c7N6AoxC3rtGkefTiKv\\\\\\\/lmbg533FW4H+FhmuKk3sNBj5QvsNCfte+9SgePEj+eDa0WktlWCrVCKsoiOxT64rrV4XR+\\\\\\\/1x7Fg6\\\\\\\/w\\\\\\\/+p8dlUzcwT4vPtnkXWWmfRKf9yGrRDfa+hMamhCEzrNvM+67i9nBvp9\\\\\\\/JuLA3zNWwkCxPHGBgu9iEDQn616XKpykXyrfTPqinkl1V\\\\\\\/KMDlLMc\\\\\\\/ewHdenRECrkZii1iGVVlUESyoWxbbSeDp5vULW9VUMVqMgsCBB4lLZGcFeLhiyoJBRQ5R0gBD+2n0\\\\\\\/\\\\\\\/fK0baZYS\\\\\\\/3hd2zKk8hVm0qnZTYMlachk+bALVageCSyrTciruQNMpfe6WO1D5kIvDifZv4\\\\\\\/eCsZNlY5PajkV3dbHqzZLIMbBrWWTARHtvvQa63vZvcCEIcjuFZUDem1hAuxFTtyIn8rSzr2pVcS+mCUDa+Saq8Py82SEF\\\\\\\/B42cYV9p6bFtB\\\\\\\/0vdykf6RCCqM0WULgyJMM9QKm+QAlwttoglkDidmgnldbL+4IlnS3qf8Rzct6yxasxCySzLipndKDOtlxBsMPqdclcXSe2zKilzwS2d8y\\\\\\\/V95pSHKGW6BPc0A32OBAG2PJFIdlfZU+ADiJyCVe4MqCX2oDlZ6hN8sVFGzOEGDGDbYqfv+l5WKygnep0Fsm2VLcCiZeXTtcc4XvHH18v61bWNf1GWklxFVjFB3ynddjaiy8YntMA3sxHyQvoBfA\\\\\\\/VW9jogKdWeZrsQ4Pkq2WOMBC+uyU7nsTcmBYkbWXfmtOm80kGJyiGRGp\\\\\\\/XCwgdDCslOcFKfwWHgX7j19NvO+sG2RlS48TlOTQ3WyJE9RZTiaokIobIPoWwtuZRIA+TPjEaij566xU1QFoj46KrqsbaF8bC7sfYMTCqmbg45lBVfZ9W5WFpTuP1uHK\\\\\\\/TczAVA7wRedHb4d9NV0j+DAK1j8KoCWTqD1KGfL3HrWVkEr\\\\\\\/edpJq50AEQJ9M7F6fPwqKQlsN2ZC8vrG30DXX22klEUtONRAHzWbuBs9ojHINtyGnmolXi1EJPiYgEsmPSgR6kuKoe7Bc4bZTvZhztiwP7FsgxYROMNaxtAw5s39Z\\\\\\\/w4ZjBbQTuRqkJ1Ac6x\\\\\\\/ehiQZsyXhQh9xMPdtH8Dh6SvtTFdS15WCyKNULb82jH747VhrCx\\\\\\\/SV9mh07T31BEtJLNSdqJLVM6aussyT6r\\\\\\\/1qGIpiHV5q+ANUgxK2W4BXxUISVj16bwkk085i\\\\\\\/STtx0yQxcVeau3HG1ccKsPYu9TZpCpN135oDy24BAOmtuVAroQVNDJ9\\\\\\\/KUpY84XNoafDKftxrPwMC99JEdzfTXR3tHQ1Oy3JtSZ3ATzhXDfZVPFPKhf8EtUmdD7via7vFCvQRvs+2mcwFs2ukdA6TKa7wLP697\\\\\\\/nrpHNZlbGcJewDlpJS5DbM0DVeO6o54kUq\\\\\\\/BSGwIZHNQOOaYsT\\\\\\\/WkXP7S9J7lBXYhiCqI+9KfHbeQCGa09n8NgZtLoIfQ2UzRj51rUTZ82YsllIeFve24xniHXGXdh\\\u0022,\\\u0022iv\\\u0022:\\\u0022uFsVVdYZ0UczcNRgiMplKA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221af6c976db7fbce9f54e63540218d5f71b651230cca8bc295dbc557a3a451a80\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485294 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2015" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3NpmQZCfYWzLfMxFj77J0s5eBnnDKT-xRhEGT7qDAGA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2015" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TIPiKNtiEs8y5_xoJsA7H5HXWziTH3yp7jtNXzL8Sbc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1995" title="Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2023" title="RNAi-Mediated Reverse Genetic Screen Identified Drosophila Chaperones Regulating Eye and Neuromuscular Junction Morphology" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485294" id="node485294" data-pisa="ggg;7/7/2015" data-pisa-master="ggg;g3.117.040691" data-apath="/ggg/7/7/2015.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Insight into the Recent Genome Duplication of the Halophilic Yeast <em>Hortaea werneckii</em>: Combining an Improved Genome with Gene Expression and Chromatin Structure</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sunita Sinha</span>, <span class="highwire-citation-author" data-delta="1">Stephane Flibotte</span>, <span class="highwire-citation-author" data-delta="2">Mauricio Neira</span>, <span class="highwire-citation-author" data-delta="3">Sean Formby</span>, <span class="highwire-citation-author" data-delta="4">Ana Plemenitaš</span>, <span class="highwire-citation-author" data-delta="5">Nina Gunde Cimerman</span>, <span class="highwire-citation-author" data-delta="6">Metka Lenassi</span>, <span class="highwire-citation-author" data-delta="7">Cene Gostinčar</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jason E. Stajich</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Corey Nislow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2015-2022;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040691 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.040691" data-atom-uri="/ggg/7/7/2015.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485294" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sunita Sinha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sunita%2BSinha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sinha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASunita%2BSinha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stephane Flibotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephane%2BFlibotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Flibotte%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephane%2BFlibotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mauricio Neira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BNeira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neira%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMauricio%2BNeira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sean Formby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BFormby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Formby%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BFormby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ana Plemenitaš </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Biochemistry, Faculty of Medicine, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BPlemenita%C5%A1%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plemenita%C5%A1%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BPlemenita%25C5%25A1%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nina Gunde Cimerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Biotechnical Faculty, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nina%2BGunde%2BCimerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cimerman%20NG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANina%2BGunde%2BCimerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Metka Lenassi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Biochemistry, Faculty of Medicine, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Metka%2BLenassi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lenassi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMetka%2BLenassi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Cene Gostinčar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Biotechnical Faculty, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cene%2BGostin%C4%8Dar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gostin%C4%8Dar%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACene%2BGostin%25C4%258Dar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jason E. Stajich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Pathology and Microbiology, Institute for Integrative Genome Biology, University of California, Riverside, California 92521 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BE.%2BStajich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stajich%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BE.%2BStajich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jason E. Stajich</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jason.stajich@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">jason.stajich@ucr.edu</a>
<a href="mailto:corey.nislow@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@ubc.ca</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Corey Nislow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corey%2BNislow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nislow%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorey%2BNislow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Corey Nislow</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jason.stajich@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">jason.stajich@ucr.edu</a>
<a href="mailto:corey.nislow@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/7/2015" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485294" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2015.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485294" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2015.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485294" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2015.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485294" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/7/2015.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485294" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Extremophilic organisms demonstrate the flexibility and adaptability of basic biological processes by highlighting how cell physiology adapts to environmental extremes. Few eukaryotic extremophiles have been well studied and only a small number are amenable to laboratory cultivation and manipulation. A detailed characterization of the genome architecture of such organisms is important to illuminate how they adapt to environmental stresses. One excellent example of a fungal extremophile is the halophile <em>Hortaea werneckii</em> (Pezizomycotina, Dothideomycetes, Capnodiales), a yeast-like fungus able to thrive at near-saturating concentrations of sodium chloride and which is also tolerant to both UV irradiation and desiccation. Given its unique lifestyle and its remarkably recent whole genome duplication, <em>H. werneckii</em> provides opportunities for testing the role of genome duplications and adaptability to extreme environments. We previously assembled the genome of <em>H. werneckii</em> using short-read sequencing technology and found a remarkable degree of gene duplication. Technology limitations, however, precluded high-confidence annotation of the entire genome. We therefore revisited the <em>H. wernickii</em> genome using long-read, single-molecule sequencing and provide an improved genome assembly which, combined with transcriptome and nucleosome analysis, provides a useful resource for fungal halophile genomics. Remarkably, the ∼50 Mb <em>H. wernickii</em> genome contains 15,974 genes of which 95% (7608) are duplicates formed by a recent whole genome duplication (WGD), with an average of 5% protein sequence divergence between them. We found that the WGD is extraordinarily recent, and compared to <em>Saccharomyces cerevisiae</em>, the majority of the genome’s ohnologs have not diverged at the level of gene expression of chromatin structure.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/extremophilic-yeast" class="hw-term hw-article-keyword hw-article-keyword-extremophilic-yeast" rel="nofollow">extremophilic yeast</a></li><li class="kwd"><a href="/keyword/gene-duplication" class="hw-term hw-article-keyword hw-article-keyword-gene-duplication" rel="nofollow">gene duplication</a></li><li class="kwd"><a href="/keyword/halophile" class="hw-term hw-article-keyword hw-article-keyword-halophile" rel="nofollow">halophile</a></li><li class="kwd"><a href="/keyword/salt-tolerance" class="hw-term hw-article-keyword hw-article-keyword-salt-tolerance" rel="nofollow">salt tolerance</a></li><li class="kwd"><a href="/keyword/genome-report" class="hw-term hw-article-keyword hw-article-keyword-genome-report" rel="nofollow">Genome Report</a></li></ul><p id="p-3">The study of prokaryotes able to thrive in extreme environments has led to fundamental discoveries like the archaea (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Woese and Fox 1977</a>) and the development of breakthrough technologies (polymerase chain reaction) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Henry and Debarbieux 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jia <em>et al.</em> 2013</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Oren 2014</a>). In contrast, eukaryotic extremophiles in general, and fungal extremophiles in particular, remain largely unexplored. A detailed characterization of such extremophiles is important because it can advance our understanding of how organisms with a fundamentally different cellular organization and evolutionary background compared to prokaryotic extremophiles adapt to environmental stresses. Furthermore, such insights have the potential to expand the stress tolerance of industrially relevant organisms (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gostinčar <em>et al.</em> 2011</a>). Halotolerance is a phenotype of particular interest since it can inform modifications of fungi used in biotechnology and bioremediation, and of plant crops whose growth is compromised by the widespread salinization of agricultural land (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Oren 2010</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lentzen and Schwarz 2006</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Shabala <em>et al.</em> 2016</a>).</p><p id="p-4">One eukaryotic extremophile to direct attention to is the ascomycetous yeast <em>Hortaea werneckii</em> (Pezizomycotina, Dothideomycetes, Capnodiales), which is exceptionally adaptable to osmotic stress and can tolerate extracellular salt concentrations that either kill or inhibit the growth of most microorganisms (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gostinčar <em>et al.</em> 2011</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Plemenitaš <em>et al.</em> 2014</a>). It is commonly found in brines formed by the evaporation of sea water, where it thrives despite fluctuating salt concentrations (ranging from no salt to near-saturating concentrations), low water activity, high temperature, high UV radiation, varying nutrient availability, and near-alkaline pH (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Gostinčar <em>et al.</em> 2011</a>).</p><p id="p-5">Several studies have identified individual mechanisms that contribute to <em>H. werneckii</em>’s extreme halotolerance (reviewed in <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Gostinčar <em>et al.</em> 2011</a> and <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Plemenitaš <em>et al.</em> 2014</a>). These studies found that general strategies including cell wall melanization, changes in membrane composition, and the accumulation of glycerol (and other compatible solutes) promote resistance to salt stress. Specific adaptations include rapid changes in the expression of genes involved in salt sensing (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Vaupotic and Plemenitaš 2007</a>), and resistance to high temperature and oxidative stress (reviewed in <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Gostinčar <em>et al.</em> 2011</a> and <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Plemenitaš <em>et al.</em> 2014</a>). Finally, as seen in other organisms (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gerstein <em>et al.</em> 2006</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Schoustra <em>et al.</em> 2007</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dhar <em>et al.</em> 2011</a>), <em>H. werneckii</em> has expanded its genome and contains multiple copies of halotolerance genes (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lenassi <em>et al.</em> 2013</a>).</p><p id="p-6">Interestingly, in <em>H. werneckii</em>, the mating-type locus also exists in two copies and contains the idiomorph MAT1-1. The <em>HwMAT1-1-1</em> gene sequences are 88.7% identical and have homologous 5′ and 3′ flanking regions (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Lenassi <em>et al.</em> 2013</a>). A compatible idiomorph MAT1-2 was not found in the sequenced <em>H. werneckii</em> genome, indicating that this species is heterothallic and would require a partner strain encoding <em>MAT1-2-1</em> for sexual reproduction. <em>MAT1-2-1</em> strains have not yet been identified and neither has the existence of a <em>H. werneckii</em> teleomorph. The sexual cycle of <em>H. werneckii</em> is also unknown.</p><p id="p-7">As a first step toward a comprehensive understanding of halotolerance in <em>H. werneckii</em>, we previously sequenced the genome of strain EXF-2000 isolated from marine solar salterns in Slovenia (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Lenassi <em>et al.</em> 2013</a>). The short length of next generation sequencing (NGS) reads available at the time, combined with the unexpectedly large number of duplicated sequences, complicated the production of a high-quality genome assembly. Nonetheless, we confirmed that fungal ion transporters were present in multiple copies, and a close inspection of the sequence data led us to hypothesize that <em>H. werneckii</em> had recently duplicated its entire genome (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Lenassi <em>et al.</em> 2013</a>). If confirmed, this genome duplication offers a “natural experiment” in which the early stages of functional gene diversification (<em>e.g.</em>, neofunctionalization) can be interrogated.</p><p id="p-8">Here we leveraged improvements in NGS to resequence the genome of EXF-2000 using long-read, single-molecule technology. Our new genome assembly reduced the number of contigs 20-fold from 12,620 to 651. Analysis and annotation of our improved genome (Hw 2.0) confirmed our previous hypothesis of a recent whole genome duplication (WGD) and further demonstrated a high degree of sequence conservation between each ohnolog pair. This greatly improved genome assembly allowed us to unambiguously compare gene pairs and closely examine the evidence for duplicate gene divergence at the level of primary sequence, gene expression, and chromatin architecture. We then compared the results from <em>H. werneckii</em> to those obtained for <em>Saccharomyces cerevisiae</em>, a species with a relatively ancient genome duplication, to compare the spectrum of genome reduction following duplication. In addition to providing a baseline for exploring genomic adaptation to extreme environmental challenge, our data set provides a framework in which to address fundamental questions of gene functionalization and divergence at the single gene, gene pair, and whole genome level. By way of example, we combined RNA-seq and nucleosome profiling to demonstrate the high degree of similarity between each ohnolog gene pair.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strain information and growth conditions</h3><p id="p-9"><em>H. werneckii</em> strain EXF-2000 was isolated from marine solar salterns on the Adriatic coast in Slovenia. It is archived in the Ex Culture Collection of the Department of Biology, Biotechnical Faculty, University of Ljubljana (Infrastructural Centre Mycosmo, MRIC UL). Cells were grown at 28° in synthetic defined yeast nitrogen base (YNB) liquid medium (Sigma Aldrich), supplemented with 1.7 M NaCl and adjusted to pH 7.0.</p></div><div id="sec-3" class="subsection"><h3>Long-read sequencing</h3><p id="p-10">Genomic DNA (gDNA) for PacBio sequencing was extracted using the Gentra Puregene Yeast/Bact. Kit (Qiagen) from two independent 1 ml overnight cultures of <em>H. werneckii</em> grown in YNB containing 1.7 M NaCl, taking care to avoid manipulations that could shear the gDNA. DNA samples were quantified, pooled, and sequenced using five SMRT cells with P4/C2 chemistry at the University of Washington PacBio Sequencing Facility.</p></div><div id="sec-4" class="subsection"><h3>RNA-seq library preparation and sequencing</h3><p id="p-11"><em>H. werneckii</em> cells were grown to midexponential phase, and 50 ml of culture was pelleted, flash frozen in liquid nitrogen, and stored at −80° until RNA isolation. Pelleted cells were ground in a mortar and pestle maintained in a liquid nitrogen bath and the grindate used for total RNA isolation using TRIzol Reagent according to the manufacturer’s instructions (Thermo Fisher Scientific). Two independent RNA samples were converted to sequencing libraries using Illumina’s TruSeq RNA version 2 Library Preparation Kit, and sequenced on a HiSequation 2500 or MiSeq (Illumina) to generate paired-end 100 base reads.</p></div><div id="sec-5" class="subsection"><h3>MNase-seq library preparation and sequencing</h3><p id="p-12"><em>H. werneckii</em> cells were grown to midexponential phase, and 200 ml of culture was used for nucleosome-bound gDNA preparation, following the protocol of <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Tsui <em>et al.</em> (2012)</a> but omitting the gel extraction step as described by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Henikoff <em>et al.</em> (2011)</a>. DNA was digested with titrations (12.5–50 U) of micrococcal nuclease (Fermentas), selecting for library preparation samples that showed &gt;80% mononucleosomes based on agarose gel analysis. Two independent DNA samples were converted to sequencing libraries using NEBNext Ultra DNA Library Prep Kit for Illumina (New England Biolabs), selecting for 150 bp inserts with Agencourt AMPure XP Beads (Beckman Coulter). Libraries were sequenced on a HiSequation 2500 (Illumina), generating paired 100 base reads.</p></div><div id="sec-6" class="subsection"><h3>Genome assembly and annotation</h3><p id="p-13">Sequencing reads were <em>de novo</em> assembled using the SMRT Analysis version 2.3 suite and RS_HGAP_assembly.3 pipeline provided by Pacific Biosciences, using default parameters. Genome assembly quality was assessed with CEGMA (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Parra <em>et al.</em> 2007</a>). Protein-coding and tRNA genes were annotated with MAKER (version 2.31.8) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Holt and Yandell 2011</a>), using as evidence for gene predictions a set of Ascomycete fungal proteomes, the complete UniProt SwissProt database, and the Genome Guided (GG) Trinity transcripts obtained from our RNA-seq data. Gene prediction training of the SNAP (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Korf 2004</a>) and Augustus (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Stanke <em>et al.</em> 2006</a>) gene callers was performed initially on protein-to-genome alignments of a conserved set of proteins in CEGMA (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Parra <em>et al.</em> 2007</a>). Paired RNA-seq reads were quality trimmed with sickle (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Joshi and Fass 2011</a>) and assembled into a consensus of transcripts with Trinity. Trinity was run with GG mode, which required the RNA-seq reads to be aligned to the genome. We used GSNAP (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wu and Nacu 2010</a>) to align the reads and produce a BAM file, followed by running Trinity in GG mode with jaccard clipping enabled to improve fungal transcript calling and a maximum intron length of 1500 nt. The assembled consensus transcripts were used as transcript evidence for MAKER genome annotation. MAKER annotation was further processed with funannotate: Fungal genome annotation scripts (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Palmer 2017</a>) (<a href="https://github.com/nextgenusfs/funannotate">https://github.com/nextgenusfs/funannotate</a>) for cleanup of gene models and passed to Genome Annotation Generator (GAG) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hall <em>et al.</em> 2014</a>) (<a href="https://genomeannotation.github.io">https://genomeannotation.github.io</a>) for creation of Genbank submission files.</p></div><div id="sec-7" class="subsection"><h3>Bioinformatics analysis of genome duplication</h3><p id="p-14">To globally investigate the duplication of genomic sequences in our new assembly Hw 2.0, the genome was aligned to itself with the PROmer algorithm, as implemented in MUMmer 3.23, and plotted with the mummerplot utility (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kurtz <em>et al.</em> 2004</a>). The pairs of most similar contigs were divided into two lists, resulting in two nonoverlapping sets of contigs, each set roughly representing half of the duplicated genome (a haploid genome). To compare protein paralogs in the genome, duplicated genes were identified from the published Hw 1.0 and the improved Hw 2.0 genomes as in <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Lenassi <em>et al.</em> (2013)</a>. An all-against-all protein sequence similarity search of <em>H. werneckii</em> proteins to a <em>H. werneckii</em> protein database was performed by BLASTP, part of BLAST 2.2.28+ (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) using an <em>e</em>-value cutoff of 1<em>e</em><sup>−50</sup>. Additionally, predicted proteins were aligned back to the genomic sequence with Exonerate version 2.2.0 using the protein2genome model (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Slater and Birney 2005</a>), and the number of loci to which each protein aligned with at least 75% of the maximal score was counted. To assess sequence conservation, duplicated proteins were identified by BLASTP using a cutoff of 1<em>e</em><sup>−50</sup>. Each protein was allowed to be in only one pair (the one with the lowest <em>e</em>-value), which resulted in 7608 protein pairs. The two proteins in each pair were aligned to each other with MAFFT software in the “–auto” mode (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Katoh and Toh 2008a</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">b</a>). The alignment was split into 20 sectors of equal length, and the number of identical and mismatched amino acids for each was counted. The total number of amino acid substitutions within each protein pair was calculated by summing the substitutions in all 20 alignment sectors. For analysis of upstream regions, protein pairs were determined by BLASTP and a cutoff of 1<em>e</em><sup>−90</sup>. For each duplicated protein pair, regions extending from 1 kb upstream and the first 100 bp of each CDS were aligned with MAFFT software in the “–auto” mode (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Katoh and Toh 2008a</a>,<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">b</a>). All pairs without a start codon at the expected location, or with &lt;75% of identical nucleotides in the region between duplicated proteins were discarded. This resulted in 4441 pairs of genes, for which the conserved nucleotide positions were counted within the pair, summed, and expressed as the proportion of conserved nucleotides per position within the whole data set. Synonymous site divergence was calculated from these pairs by aligning the protein sequences with Muscle (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Edgar 2004</a>), followed by a back translation to coding sequences using bp_mrtrans (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Stajich <em>et al.</em> 2002</a>), and substitutions calculated with YN00 (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Yang and Nielsen 2000</a>) reimplemented for multi-sequence processing in the package subopt-kaks (<a href="https://github.com/hyphaltip/subopt-kaks">https://github.com/hyphaltip/subopt-kaks</a>). Substitutions per synonymous site (dS) values above dS = 2 were removed as they are unlikely to be useful estimates and represent the limit of estimation.</p></div><div id="sec-8" class="subsection"><h3>RNA-seq analysis</h3><p id="p-15">After alignment to the Hw 2.0 assembly, the number of aligned RNA-seq reads for each gene was tabulated directly with the alignment program STAR (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dobin <em>et al.</em> 2013</a>). The same procedure was also applied to RNA-seq reads for <em>S. cerevisiae</em> available at the NCBI Short Read Archive (accession numbers SRR488142 and SRR488143) using the reference genome S288C version R64 [<em>Saccharomyces</em> Genome Database (SDG); <a href="http://www.yeastgenome.org">http://www.yeastgenome.org</a>].</p></div><div id="sec-9" class="subsection"><h3>MNAse-seq analysis</h3><p id="p-16"><em>H. werneckii</em> sequencing reads were aligned to the Hw 2.0 genome using BWA version 0.7.12 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li and Durbin 2009</a>) and the resulting files sorted with the SAMtools suite version 1.2 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2009</a>). DANPOS2 (version 2.2.2) was then used on the alignments to extract nucleosome positioning and occupancy (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2013</a>). The same analysis procedure was applied to publicly available reads for <em>S. cerevisiae</em> using the reference genome S288C version R64 (SDG; <a href="http://www.yeastgenome.org">http://www.yeastgenome.org</a>).</p></div><div id="sec-10" class="subsection"><h3>Analysis of divergence in gene regulation</h3><p id="p-17">The difference in expression level between paralog pairs was calculated with custom Perl and R scripts. Briefly, for each sample, gene paralog pairs were kept for further analysis if both members were covered by at least 50 aligned reads. An expression ratio in the log<sub>2</sub> scale was then calculated for each pair, taking into account the coding size of the individual genes. Those ratios were then averaged between the two samples for each species and finally the absolute value of the resulting fold changes was used in the calculation of the distribution for each species. In addition, for each gene the output from DANPOS2 provided an average nucleosome occupancy value calculated every 10 bases for a 1 kb window centered on the transcription start site. Those nucleosome occupancy profiles were then normalized to an average of one. In other words, only the shape of the profile and not the absolute occupancy level was kept for the downstream analysis. The similarity between paralog pairs was then assessed by calculating the root mean square between the individual normalized profiles (analogous to a SD or the goodness of fit in a χ<sup>2</sup> minimization).</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-18">Sequencing reads have been deposited in Genbank under the BioProject PRJNA356640. Scripts and data are available from <a href="https://github.com/stajichlab/Hortaea_werneckii">https://github.com/stajichlab/Hortaea_werneckii</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>The Hortaea werneckii genome assembly version 2.0</h3><p id="p-19">Our previously published genome sequence (Hw 1.0) used short 75 bp Illumina reads, which limited our assembly and ability to analyze <em>H. werneckii</em>’s genomic content (<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Lenassi <em>et al.</em> 2013</a>). Although the data provided a case for the highly duplicated nature of the genome, the large number of contigs (&gt;12,000) prevented us from unambiguously distinguishing <em>bona fide</em> gene duplications from potential mis-assemblies. In this study, we took advantage of long-read, single-molecule Pacific Biosciences RS (PacBio) sequencing technology, which offers considerably longer read lengths. gDNA was extracted from <em>H. werneckii</em> strain EXF-2000 and sequenced using PacBio P4/C2 chemistry. Reads (average length of 5458 bases) were <em>de novo</em> assembled to generate genome sequence Hw 2.0 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) using the SMRT Analysis version 2.3 suite and RS_HGAP_assembly.3 protocol with default parameters (Pacific Biosciences). In generating this new assembly, we considered a hybrid short-/long-read assembly approach (<em>e.g.</em>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Youssef <em>et al.</em> 2013</a>) but found that there was little improvement based on summary statistics and empirical gene content inspection to the PacBio alone assembly (data not shown).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485485/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485485/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Assembly statistics of the Hw 2.0 genome</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">We conducted RNA-seq to aid genome annotation and to compare the expression of each ohnolog gene pair. Sequencing reads were <em>de novo</em> assembled using Trinity, an analysis recommended for <em>de novo</em> transcriptome assembly from RNA-seq data in nonmodel organisms (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Haas <em>et al.</em> 2013</a>). One caveat is that the gene models used have not been hand curated, therefore some inaccuracies are to be expected, especially for genes with multiple exons. In order to estimate the accuracy of our gene models, we compared the alignments of RNA-seq reads performed with both the STAR (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dobin <em>et al.</em> 2013</a>) and HISAT2 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kim <em>et al.</em> 2015</a>) aligners. Visual inspection of the alignments using the IGV viewer (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Robinson <em>et al.</em> 2011</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Thorvaldsdottir <em>et al.</em> 2013</a>) suggested that ∼30% of the genes with multiple exons have imperfect gene models. A more global inspection at the transcriptome level using StringTie (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Pertea <em>et al.</em> 2015</a>) and GffCompare (<a href="https://github.com/gpertea/gffcompare">https://github.com/gpertea/gffcompare</a>) agreed with this estimate.</p><p id="p-22">Our genome resequencing reduced the number of contigs from 12,620 in Hw 1.0 to 651 in Hw 2.0, yet the genome composition (<em>e.g.</em>, GC content) remained largely unchanged (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The greatest improvement in the genome assembly and annotation addressed the key questions of (1) total gene number, and (2) the number of duplicated gene pairs. The number of predicted protein-coding genes was reduced by 30% to 15,974 genes (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). This decrease likely reflects the fact that the large number of contigs in Hw 1.0 comprised a substantial number of fragmented or misassembled sequences, resulting in an overestimate of the number of unique protein-coding genes.</p><p id="p-23">At 49.9 Mb, the <em>Hortaea</em> genome is larger than many other species in the order Capnodiales (16.9–74.1 Mb, average size 35.8 Mb), which also includes the “whiskey fungus” <em>Baudoinia compniacensis</em> (21.88 Mb) and the wheat pathogen <em>Zymoseptoria tritici</em> (previously <em>Mycosphaerella graminicola</em>; 39.69 Mb) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Goodwin <em>et al.</em> 2011</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Ohm <em>et al.</em> 2012</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Stukenbrock <em>et al.</em> 2012</a>). In this order, differences in genome size tend to reflect differences in the amount of repeat sequences; large genomes have large amounts of repeated DNA content and vice versa (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de Wit <em>et al.</em> 2012</a>). In contrast, our new genome assembly shows that Hw 2.0 contains a small amount of repetitive DNA (3.2% of total genome sequence; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) and that the large genome can be attributed to a greater-than-average number of predicted protein-coding genes (∼30% higher than other sequenced Capnodiales species, average of ∼11,300). We previously hypothesized that this large number of genes results from a WGD event (<a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Lenassi <em>et al.</em> 2013</a>) and as described below, the data from Hw 2.0 supports this conclusion. In addition, the <em>H. werneckii</em> genome is more gene dense than that of its relatives: gene, protein, and exon lengths are slightly longer, while the intergenic distance is smaller (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Ohm <em>et al.</em> 2012</a>). As a consequence, over two thirds of the genome (69.9%) codes for proteins (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). While genome size varies among the Capnodiales, the proportion of genes with introns and the length of those introns are similar among species in the order.</p></div><div id="sec-14" class="subsection"><h3>WGD</h3><p id="p-24">We next used our improved genome assembly to investigate the duplication status of all proteins and protein pairs to evaluate the recent genome duplication event in greater detail. One simple approach to address this question is to divide the genome into two “haploid” single gene copies and compare the paralogs. When contigs were randomly divided into two nonoverlapping subsets, each roughly representing a haploid complement of the genome (see <em>Materials and Methods</em>), the subsets could be aligned to each other with few gaps or rearrangements (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040691/-/DC1/FigureS1.jpg">Figure S1</a>; a representative alignment using the 10 largest contigs is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). When aligning the subsets with an 85% minimum nucleotide similarity threshold and counting only the best one-to-one alignments for each locus, 69.46% of the genome was covered by alignable duplicated regions. Alignment of each subset to its pair also showed that there were no additional large-scale duplications beyond the global WGD in either of the subgenomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040691/-/DC1/FigureS2.jpg">Figure S2</a>). This confirms that <em>H. werneckii</em> contains two highly similar copies of the genome, and that the duplicated proteins arose from a large-scale genome duplication and not through duplications of individual chromosome segments.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2015/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evidence for whole genome duplication in H. werneckii. (A) Partial alignment of two genome copies of the duplicated H. werneckii genome. The 10 largest contigs of one genome copy were aligned against the corresponding contigs of the other genome copy, and the alignment was visualized with mummerplot. The gridlines separate individual contigs. (B) The share of duplicated predicted proteins in Hw 1.0 and Hw 2.0 genomic sequences. Duplications were determined by an all-against-all proteome BLASTP (left two bars; similarity threshold 1e&#x2212;50) and by aligning the proteins back to the genomic sequence using Exonerate algorithm (right two bars; similarity threshold 75% of maximal score). (C) Assessment of sequence conservation. The number of duplicated proteins with a certain share of amino acid substitutions between the proteins in the duplicated pair. (D) Distribution of synonymous site substitution rates for paralogous genes. A histogram of dS for all paralog pairs. (E) Sequence conservation along the amino acid sequence. The average share of amino acids that remain identical between the duplicated proteins after the WGD is shown along the relative length of the protein sequence. (F) The share of conserved positions in the upstream and partial CDS regions of duplicated genes. The average share of nucleic acids that remain identical between the duplicated genes after the WGD is shown relative to the start codon." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-903004618" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Evidence for whole genome duplication in &lt;em&gt;H. werneckii&lt;/em&gt;. (A) Partial alignment of two genome copies of the duplicated &lt;em&gt;H. werneckii&lt;/em&gt; genome. The 10 largest contigs of one genome copy were aligned against the corresponding contigs of the other genome copy, and the alignment was visualized with mummerplot. The gridlines separate individual contigs. (B) The share of duplicated predicted proteins in Hw 1.0 and Hw 2.0 genomic sequences. Duplications were determined by an all-against-all proteome BLASTP (left two bars; similarity threshold 1&lt;em&gt;e&lt;/em&gt;&lt;sup&gt;&#x2212;50&lt;/sup&gt;) and by aligning the proteins back to the genomic sequence using Exonerate algorithm (right two bars; similarity threshold 75% of maximal score). (C) Assessment of sequence conservation. The number of duplicated proteins with a certain share of amino acid substitutions between the proteins in the duplicated pair. (D) Distribution of synonymous site substitution rates for paralogous genes. A histogram of dS for all paralog pairs. (E) Sequence conservation along the amino acid sequence. The average share of amino acids that remain identical between the duplicated proteins after the WGD is shown along the relative length of the protein sequence. (F) The share of conserved positions in the upstream and partial CDS regions of duplicated genes. The average share of nucleic acids that remain identical between the duplicated genes after the WGD is shown relative to the start codon.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2015/F1.medium.gif" width="280" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/7/2015/F1.medium.gif" width="280" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2015/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2015/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485342" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Evidence for whole genome duplication in <em>H. werneckii</em>. (A) Partial alignment of two genome copies of the duplicated <em>H. werneckii</em> genome. The 10 largest contigs of one genome copy were aligned against the corresponding contigs of the other genome copy, and the alignment was visualized with mummerplot. The gridlines separate individual contigs. (B) The share of duplicated predicted proteins in Hw 1.0 and Hw 2.0 genomic sequences. Duplications were determined by an all-against-all proteome BLASTP (left two bars; similarity threshold 1<em>e</em><sup>−50</sup>) and by aligning the proteins back to the genomic sequence using Exonerate algorithm (right two bars; similarity threshold 75% of maximal score). (C) Assessment of sequence conservation. The number of duplicated proteins with a certain share of amino acid substitutions between the proteins in the duplicated pair. (D) Distribution of synonymous site substitution rates for paralogous genes. A histogram of dS for all paralog pairs. (E) Sequence conservation along the amino acid sequence. The average share of amino acids that remain identical between the duplicated proteins after the WGD is shown along the relative length of the protein sequence. (F) The share of conserved positions in the upstream and partial CDS regions of duplicated genes. The average share of nucleic acids that remain identical between the duplicated genes after the WGD is shown relative to the start codon.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">To comprehensively examine the predicted protein sequences (in terms of amino acid sequence similarity) for each duplicate protein pair, we used BLASTP to compare the predicted proteomes as a matrix. Over 90% of proteins were found in more than one copy (71% present as duplicates) at an <em>e</em>-value cutoff of 1<em>e</em><sup>−50</sup> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>, left panel). In a different analysis, we aligned the predicted proteins within the proteome back to the genome using Exonerate (with a cutoff of 75% similarity) and corroborated that ∼90% of protein sequences aligned to two separate genomic locations (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>, right panel). This provides unambiguous confirmation that most protein-coding genes have been duplicated. In fact, both comparisons showed that the proportion of duplicated genes is even higher than originally estimated in Hw 1.0 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-27">To investigate the degree of sequence divergence between each pair of duplicated proteins, we aligned protein pairs generated by BLASTP to each other using MAFFT. In general, the sequence identity between pairs was very high, with most proteins differing by &lt;5% (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). This high sequence identity was also apparent when analyzing the dS for each pair of duplicated sequences, which showed a discrete peak at a low dS (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>). This profile is consistent with a model of evolution where most of the duplicates were created at the same evolutionary time, followed by gradual accumulation of changes; multiple WGDs would show multiple peaks while duplicates created at random over evolutionary time would instead have a flat distribution across dS values. This further confirms <em>H. werneckii</em>’s WGD as seen in similar approaches to uncover plant WGD (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blanc and Wolfe 2004</a>). The age of the WGD can be inferred to be relatively recent based on a median dS of 0.33 (mean 0.36) and the relatively tight peak. Not unexpectedly, the conservation between duplicated proteins extended along the entire length of the protein sequence (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1E</a>). A parallel analysis restricted to the CDS of these pairs also showed high sequence identity (data not shown).</p><p id="p-28">Because the sequence of <em>H. werneckii</em>’s ohnologs is highly conserved, we next asked if their transcription, and therefore the resulting protein levels, differs between gene pairs. We first looked at gene structure on the premise that the diversity of upstream noncoding sequences could serve as a potential indicator of regulatory divergence. We examined the degree of sequence variation around the transcriptional start site, focusing on 4441 high-confidence gene pairs (generated using a BLASTP cutoff of 1<em>e</em><sup>−90</sup>). In the 400 bp upstream of the ATG codon, ∼85% of nucleotides were conserved between duplicates, with the highest conservation observed ∼150 bp upstream of the start codon (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1F</a>). As expected, sequence identity was highest in the first 100 bp of the CDS (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1F</a>); throughout this region, the third base of each codon was much less conserved than the first two bases, and its conservation was close to the region upstream of CDS. The higher divergence of sequences upstream of the CDS, where most transcriptional factors would bind, suggests that gene regulation, for example by binding of transcription factors, may have already diverged between ohnolog pairs.</p><p id="p-29">To further explore the similarities between each ohnolog pair, we used two approaches to examine potential differences between the 4441 high-confidence <em>H. werneckii</em> gene pairs. To directly measure gene expression divergence, we compared transcription levels of ohnolog pairs. Since eukaryotic gene regulation is also affected by chromatin architecture (<em>e.g.</em>, nucleosome occupancy and positioning) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Iyer 2012</a>), we also compared nucleosome occupancy between ohnolog pairs. RNA and nucleosome-bound gDNA were extracted, converted to libraries, and sequenced to generate short paired-end reads. To put these observations in context, we performed the same comparisons using comparable data available for paralog pairs of <em>S. cerevisiae</em>. The rationale behind this comparison is that while <em>H. werneckii</em> has undergone a recent large-scale gene duplication, several lines of evidence support the idea that genome duplication of <em>S. cerevisiae</em> is considerably older and the limited number of gene pairs which have persisted have undergone significant functional divergence (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Musso <em>et al.</em> 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2010</a>). Consistent with this premise, our analysis of the gene expression and nucleosome occupancy showed much greater conservation between ohnolog pairs in <em>H. werneckii</em> than <em>S. cerevisiae</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). This suggests that not only the primary sequence, but also the function of each ohnolog is highly conserved in <em>H. werneckii</em>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2015/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Similarity of gene expression level (A) and nucleosome occupancy profile (B) between paralog pairs. (A) Distribution of the absolute value of gene expression ratio between paralog pairs in log2 scale for H. werneckii (blue) and S. cerevisiae (red). (B) Distribution of nucleosome profile similarity between paralog pairs in H. werneckii (blue) and paralogs in S. cerevisiae (red) (see Materials and Methods for details)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-903004618" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Similarity of gene expression level (A) and nucleosome occupancy profile (B) between paralog pairs. (A) Distribution of the absolute value of gene expression ratio between paralog pairs in log&lt;sub&gt;2&lt;/sub&gt; scale for &lt;em&gt;H. werneckii&lt;/em&gt; (blue) and &lt;em&gt;S. cerevisiae&lt;/em&gt; (red). (B) Distribution of nucleosome profile similarity between paralog pairs in &lt;em&gt;H. werneckii&lt;/em&gt; (blue) and paralogs in &lt;em&gt;S. cerevisiae&lt;/em&gt; (red) (see &lt;em&gt;Materials and Methods&lt;/em&gt; for details).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2015/F2.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/7/2015/F2.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2015/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2015/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485497" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">Similarity of gene expression level (A) and nucleosome occupancy profile (B) between paralog pairs. (A) Distribution of the absolute value of gene expression ratio between paralog pairs in log<sub>2</sub> scale for <em>H. werneckii</em> (blue) and <em>S. cerevisiae</em> (red). (B) Distribution of nucleosome profile similarity between paralog pairs in <em>H. werneckii</em> (blue) and paralogs in <em>S. cerevisiae</em> (red) (see <em>Materials and Methods</em> for details).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-31">We present an improved genome assembly for <em>H. werneckii</em>, which is a foundational resource for future global studies of this unique extremophile. The associated Web sites (<a href="https://github.com/stajichlab/Hortaea_werneckii">https://github.com/stajichlab/Hortaea_werneckii</a>) incorporate the sequencing data, genome annotation, and associated methods in a single archive. The Gbrowse genome browser site (<a href="http://gb2.fungalgenomes.org/gb2/gbrowse/hortaea">http://gb2.fungalgenomes.org/gb2/gbrowse/hortaea</a>) provides additional interactive views of the genome and annotation.</p><p id="p-32">A careful assessment of sequence duplication and conservation enabled us to confirm our previously hypothesized WGD. This duplicated genome content likely allows <em>H. werneckii</em> to possess the genetic redundancy associated with diploidy, thereby avoiding a haploid phase, which normally predominates in the life cycle of ascomycetes. The fact that the genome is homozygous for the MAT1-1-1 gene (<a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">Lenassi <em>et al.</em> 2013</a>) suggests two possibilities: (1) the fungus is asexual, which might enable it to maintain the genome configuration that is well-adapted to extreme salinity (similar to observations in some pathogens; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ene and Bennett 2014</a>); or (2) it can mate with haploid or diploid strains of the opposite mating type, resulting in triploid or tetraploid progeny, respectively. WGDs and polyploidization events are common in plants, and tend to result in increased fitness and stress resistance (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Vanneste <em>et al.</em> 2014</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">del Pozo and Ramirez-Parra 2015</a>). However, they still appear to be relatively rare in other eukaryotic lineages. Instead, transient ploidy changes are described in <em>S. cerevisiae</em> and others in laboratory experimental evolution experiments. In these studies, genomic content increases rapidly in response to environmental stresses (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gerstein <em>et al.</em> 2006</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dhar <em>et al.</em> 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dujon 2015</a>). <em>H. werneckii</em> therefore appears unusual among fungi, as a very recent WGD has only been described in some zygomyceteous Mucoromycotina fungi, the human pathogen <em>Rhizopus oryzae</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ma <em>et al.</em> 2009</a>), and the dung fungus <em>Phycomyces blakesleeanus</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Corrochano <em>et al.</em> 2016</a>). The <em>R. oryzae</em> and <em>P. blakesleeanus</em> WGDs have been suggested to contribute to pathogenicity and the expansion of signal transduction and light sensing, while for <em>H. werneckii</em> it may confer a selective advantage in hypersaline environments. However, the relative age of the WGD in these Mucoromycotina fungi is older than that of <em>H. werneckii</em> as the duplicate pairs are much more divergent and there are many fewer sets of gene pairs. Many lineages await genomic investigations so the ever decreasing cost of sequencing and improvements in long-read and high-repeat content sequencing may help uncover additional recent WGD events in other eukaryotes.</p><p id="p-33">Postduplication events normally include large-scale gene deletions, reductions, and genome reshuffling. Indeed, only 550 pairs of ohnologs remain in <em>S. cerevisiae</em> and similar numbers are seen in relatives (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dujon <em>et al.</em> 2004</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Byrne and Wolfe 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cliften <em>et al.</em> 2005</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Scannell <em>et al.</em> 2007</a>). This does not seem to be the case in the evolution of <em>H. werneckii</em> as the WGD, for now, appears stable. Indeed, we have shown that even after long-term growth in the presence of salt stress, the genome is not reduced (C. Gostinčar, A. Kejžar, J. Zajc, C. Nislow, S. Sinha <em>et al.</em>, unpublished data). A WGD could result from either an endoreduplication of the genome, or from hybridization between two strains (intraspecific) or two species (interspecific). While further research is required to distinguish between these possibilities, the current genomic data does not appear to support the hybridization hypothesis. Hybridization would likely occur between strains of opposite mating types, resulting in a hybrid containing both MAT1-1-1 and MAT1-2-1 genes <em>vs.</em> two copies of the MAT1-1-1 gene as seen in <em>H. werneckii</em>. While this could have arisen secondarily through replacement of a MAT1-2-1 by recombination from the other idiomorph, this seems unlikely considering that the two MAT1-1-1 copies have diverged and are only 88.7% identical.</p><p id="p-34">Considering the endoreduplication hypothesis, the stability of the WGD is worth remarking on further. Usually, after gene duplication one of the copies is expected to accumulate deleterious mutations and be lost through nonfunctionalization (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lynch and Conery 2000</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">2003</a>). The number of observed substitutions between <em>H. werneckii</em> duplicates suggests that since the WGD event there has not been a substantial loss of one of the duplicate copies from the genome, nor have there been major changes in the gene expression and chromatin structure between the ohnologs. Further comparison and reconstruction of the WGD will be important future research steps, taking into account comparisons to the closest members of the genus.</p><p id="p-35">We suggest that the improved genome presented in this work, combined with the relative resistance of the genome to environmental stresses make <em>H. werneckii</em> an attractive “emerging” model organism both for understanding genetic redundancy and for developing strains for demanding biotechnological conditions.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">We thank the University of British Columbia Sequencing and Bioinformatics Consortium for sample preparation, sequencing, and analysis. This work was partially supported by the United States Department of Agriculture, Agricultural Experimental Station at the University of California (UC), Riverside; and the National Institute of Food and Agriculture Hatch Project CA-R-PPA-5062-H to J.E.S. Computational analyses were performed on high performance computing resources in the Institute for Integrative Genome Biology at UC Riverside supported by National Science Foundation DBI-1429826 and National Institutes of Health S10-OD016290.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></li><li class="fn" id="fn-3"><p id="p-37">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040691/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040691/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-38"><em>Communicating editor: J. Rine</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-02-19"><span class="received-label">Received </span>February 19, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-05-08"><span class="accepted-label">Accepted </span>May 8, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Sinha <em>et al.</em></li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schäffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.2" data-doi="10.1105/tpc.021345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1667</span>–<span class="cit-lpage">1678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.021345%26rft_id%253Dinfo%253Apmid%252F15208399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE2LzcvMTY2NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.3" data-doi="10.1101/gr.3672305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The yeast gene order browser: combining curated homology and syntenic context reveals gene fate in polyploid species.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1456</span>–<span class="cit-lpage">1461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3672305%26rft_id%253Dinfo%253Apmid%252F16169922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMC8xNDU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.4" data-doi="10.1101/gr.142067.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaestner</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DANPOS: dynamic analysis of nucleosome position and occupancy by sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.142067.112%26rft_id%253Dinfo%253Apmid%252F23193179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvMzQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cliften</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">After the duplication: gene loss and adaptation in Saccharomyces genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D172%26rft.spage%253D863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.6" data-doi="10.1016/j.cub.2016.04.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corrochano</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Houben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polaino</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salamov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Expansion of signal transduction pathways in fungi by extensive genome duplication.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1577</span>–<span class="cit-lpage">1584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D26%26rft.spage%253D1577%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2016.04.038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2016.04.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.7" data-doi="10.1093/jxb/erv432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">del Pozo</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramirez-Parra</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Whole genome duplications in plants: an overview from <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">6991</span>–<span class="cit-lpage">7003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferv432%26rft_id%253Dinfo%253Apmid%252F26417017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjEwOiI2Ni8yMi82OTkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.8" data-doi="10.1371/journal.pgen.1003088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">P. J. G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Burgt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ökmen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stergiopoulos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abd-Elsalam</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomes of the fungal plant pathogens <em>Cladosporium fulvum</em> and <em>Dothistroma septosporum</em> reveal adaptation to different hosts and lifestyles but also signatures of common ancestry.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003088</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253Dde%2BWit%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1003088%26rft.epage%253De1003088%26rft.atitle%253DThe%2BGenomes%2Bof%2Bthe%2BFungal%2BPlant%2BPathogens%2BCladosporium%2Bfulvum%2Band%2BDothistroma%2Bseptosporum%2BReveal%2BAdaptation%2Bto%2BDifferent%2BHosts%2Band%2BLifestyles%2BBut%2BAlso%2BSignatures%2Bof%2BCommon%2BAncestry.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003088%26rft_id%253Dinfo%253Apmid%252F23209441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23209441&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.9" data-doi="10.1111/j.1420-9101.2011.02249.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dhar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sägesser</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weikert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Adaptation of <em>Saccharomyces cerevisiae</em> to saline stress through laboratory evolution.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1135</span>–<span class="cit-lpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DDhar%26rft.auinit1%253DR.%26rft.volume%253D24%26rft.issue%253D5%26rft.spage%253D1135%26rft.epage%253D1153%26rft.atitle%253DAdaptation%2Bof%2BSaccharomyces%2Bcerevisiae%2Bto%2Bsaline%2Bstress%2Bthrough%2Blaboratory%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2011.02249.x%26rft_id%253Dinfo%253Apmid%252F21375649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2011.02249.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21375649&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289692700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.10" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drenkow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaleski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo3OiIyOS8xLzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.11" data-doi="10.1093/femsyr/fov047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Basic principles of yeast genomics, a personal recollection.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">fov047</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffov047%26rft_id%253Dinfo%253Apmid%252F26071597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjExOiIxNS81L2ZvdjA0NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.12" data-doi="10.1038/nature02579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casaregola</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome evolution in yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDujon%26rft.auinit1%253DB.%26rft.volume%253D430%26rft.issue%253D6995%26rft.spage%253D35%26rft.epage%253D44%26rft.atitle%253DGenome%2Bevolution%2Bin%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02579%26rft_id%253Dinfo%253Apmid%252F15229592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15229592&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222356800036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.13" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzUvMTc5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.14" data-doi="10.1038/nrmicro3236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ene</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The cryptic sexual strategies of human fungal pathogens.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol.%26rft.volume%253D12%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro3236%26rft_id%253Dinfo%253Apmid%252F24625892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro3236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24625892&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.15" data-doi="10.1371/journal.pgen.0020145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chun</span>  <span class="cit-name-given-names">H.-J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic convergence toward diploidy in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGerstein%26rft.auinit1%253DA.%2BC.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De145%26rft.epage%253De145%26rft.atitle%253DGenomic%2Bconvergence%2Btoward%2Bdiploidy%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020145%26rft_id%253Dinfo%253Apmid%252F17002497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17002497&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.16" data-doi="10.1371/journal.pgen.1002070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben M’barek</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">A. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crane</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Finished genome of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGoodwin%26rft.auinit1%253DS.%2BB.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002070%26rft.epage%253De1002070%26rft.atitle%253DFinished%2Bgenome%2Bof%2Bthe%2Bfungal%2Bwheat%2Bpathogen%2BMycosphaerella%2Bgraminicola%2Breveals%2Bdispensome%2Bstructure%252C%2Bchromosome%2Bplasticity%252C%2Band%2Bstealth%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002070%26rft_id%253Dinfo%253Apmid%252F21695235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21695235&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.17" data-doi="10.1016/B978-0-12-387044-5.00003-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gostinčar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenassi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunde-Cimerman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plemenitaš</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fungal adaptation to extremely high salt concentrations.</span> <abbr class="cit-jnl-abbrev">Adv. Appl. Microbiol.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bapplied%2Bmicrobiology%26rft.stitle%253DAdv%2BAppl%2BMicrobiol%26rft.aulast%253DGostincar%26rft.auinit1%253DC.%26rft.volume%253D77%26rft.spage%253D71%26rft.epage%253D96%26rft.atitle%253DFungal%2Badaptation%2Bto%2Bextremely%2Bhigh%2Bsalt%2Bconcentrations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-387044-5.00003-0%26rft_id%253Dinfo%253Apmid%252F22050822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-387044-5.00003-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22050822&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.18" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papanicolaou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blood</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>De novo</em> transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Rego</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geib</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-comment">GAG: the Genome Annotation Generator (Version 1.0) [Software] <a href="http://genomeannotation.github.io/GAG">http://genomeannotation.github.io/GAG</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.20" data-doi="10.1073/pnas.1110731108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belsky</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krassovsky</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacAlpine</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Epigenome characterization at single base-pair resolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">18318</span>–<span class="cit-lpage">18323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1110731108%26rft_id%253Dinfo%253Apmid%252F22025700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzQ1LzE4MzE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Debarbieux</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Tools from viruses: bacteriophage successes and beyond.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">434</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D434%26rft.spage%253D151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.22" data-doi="10.1186/1471-2105-12-491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHolt%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D491%26rft.epage%253D491%26rft.atitle%253DMAKER2%253A%2Ban%2Bannotation%2Bpipeline%2Band%2Bgenome-database%2Bmanagement%2Btool%2Bfor%2Bsecond-generation%2Bgenome%2Bprojects.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-491%26rft_id%253Dinfo%253Apmid%252F22192575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192575&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.23" data-doi="10.1016/j.tcb.2012.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Nucleosome positioning: bringing order to the eukaryotic.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DIyer%26rft.auinit1%253DV.%2BR.%26rft.volume%253D22%26rft.issue%253D5%26rft.spage%253D250%26rft.epage%253D256%26rft.atitle%253DNucleosome%2Bpositioning%253A%2Bbringing%2Border%2Bto%2Bthe%2Beukaryotic%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2012.02.004%26rft_id%253Dinfo%253Apmid%252F22421062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2012.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22421062&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304295400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.24" data-doi="10.1007/s00792-012-0509-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheong</span>  <span class="cit-name-given-names">G.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multifunctional enzymes in archaea: promiscuity and moonlight.</span> <abbr class="cit-jnl-abbrev">Extremophiles</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExtremophiles%26rft.volume%253D17%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00792-012-0509-1%26rft_id%253Dinfo%253Apmid%252F23283522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00792-012-0509-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23283522&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-web" id="cit-7.7.2015.25"><div class="cit-metadata unstructured">Joshi, N. A., and J. N. Fass, 2011 Sickle: a sliding-window, adaptive, quality-based trimming tool for FastQ files (Version 1.33) [Software]. Available at: <a href="https://github.com/najoshi/sickle">https://github.com/najoshi/sickle</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.26" data-doi="10.1186/1471-2105-9-212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toh</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">a</span> <span class="cit-article-title">Improved accuracy of multiple ncRNA alignment by incorporating structural information into a MAFFT-based framework.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKatoh%26rft.auinit1%253DK.%26rft.volume%253D9%26rft.spage%253D212%26rft.epage%253D212%26rft.atitle%253DImproved%2Baccuracy%2Bof%2Bmultiple%2BncRNA%2Balignment%2Bby%2Bincorporating%2Bstructural%2Binformation%2Binto%2Ba%2BMAFFT-based%2Bframework.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-212%26rft_id%253Dinfo%253Apmid%252F18439255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18439255&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.27" data-doi="10.1093/bib/bbn013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toh</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">b</span> <span class="cit-article-title">Recent developments in the MAFFT multiple sequence alignment program.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbn013%26rft_id%253Dinfo%253Apmid%252F18372315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjc6IjkvNC8yODYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjAxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.28" data-doi="10.1038/nmeth.3317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HISAT: a fast spliced aligner with low memory requirements.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3317%26rft_id%253Dinfo%253Apmid%252F25751142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25751142&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.29" data-doi="10.1186/1471-2105-5-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene finding in novel genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKorf%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.spage%253D59%26rft.epage%253D59%26rft.atitle%253DGene%2Bfinding%2Bin%2Bnovel%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-59%26rft_id%253Dinfo%253Apmid%252F15144565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15144565&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.30" data-doi="10.1186/gb-2004-5-2-r12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shumway</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Versatile and open software for comparing large genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKurtz%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253DR12%26rft.epage%253DR12%26rft.atitle%253DVersatile%2Band%2Bopen%2Bsoftware%2Bfor%2Bcomparing%2Blarge%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-2-r12%26rft_id%253Dinfo%253Apmid%252F14759262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-2-r12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14759262&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.31" data-doi="10.1371/journal.pone.0071328"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenassi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gostinčar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadowski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole genome duplication and enrichment of metal cation transporters revealed by <em>de novo</em> genome sequencing of extremely halotolerant black yeast <em>Hortaea werneckii</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e71328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De71328%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0071328%26rft_id%253Dinfo%253Apmid%252F23977017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0071328&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23977017&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.32" data-doi="10.1007/s00253-006-0553-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lentzen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Extremolytes: natural compounds from extremophiles for versatile applications.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DLentzen%26rft.auinit1%253DG.%26rft.volume%253D72%26rft.issue%253D4%26rft.spage%253D623%26rft.epage%253D634%26rft.atitle%253DExtremolytes%253A%2BNatural%2Bcompounds%2Bfrom%2Bextremophiles%2Bfor%2Bversatile%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-006-0553-9%26rft_id%253Dinfo%253Apmid%252F16957893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-006-0553-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16957893&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240809600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.33" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.34" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.35" data-doi="10.1371/journal.pgen.1001187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The cellular robustness by genetic redundancy in budding yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001187%26rft.epage%253De1001187%26rft.atitle%253DThe%2Bcellular%2Brobustness%2Bby%2Bgenetic%2Bredundancy%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001187%26rft_id%253Dinfo%253Apmid%252F21079672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21079672&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.36" data-doi="10.1126/science.290.5494.1151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The evolutionary fate and consequences of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D290%26rft.issue%253D5494%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DThe%2BEvolutionary%2BFate%2Band%2BConsequences%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5494.1151%26rft_id%253Dinfo%253Apmid%252F11073452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTQ5NC8xMTUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.37" data-doi="10.1023/A:1022696612931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The evolutionary demography of duplicate genes.</span> <abbr class="cit-jnl-abbrev">J. Struct. Funct. Genomics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Band%2Bfunctional%2Bgenomics%26rft.stitle%253DJ%2BStruct%2BFunct%2BGenomics%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.issue%253D1-4%26rft.spage%253D35%26rft.epage%253D44%26rft.atitle%253DThe%2Bevolutionary%2Bdemography%2Bof%2Bduplicate%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1022696612931%26rft_id%253Dinfo%253Apmid%252F12836683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1022696612931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12836683&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.38" data-doi="10.1371/journal.pgen.1000549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibrahim</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skory</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burger</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic analysis of the basal lineage fungus <em>Rhizopus oryzae</em> reveals a whole-genome duplication.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMa%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000549%26rft.epage%253De1000549%26rft.atitle%253DGenomic%2Banalysis%2Bof%2Bthe%2Bbasal%2Blineage%2Bfungus%2BRhizopus%2Boryzae%2Breveals%2Ba%2Bwhole-genome%2Bduplication.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000549%26rft_id%253Dinfo%253Apmid%252F19578406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19578406&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.39" data-doi="10.1101/gr.076174.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Musso</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huangfu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paw</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The extensive and condition-dependent nature of epistasis among whole-genome duplicates in yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1092</span>–<span class="cit-lpage">1099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.076174.108%26rft_id%253Dinfo%253Apmid%252F18463300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzcvMTA5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.40" data-doi="10.1371/journal.ppat.1003037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohm</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feau</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henrissat</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horwitz</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen <em>Dothideomycetes</em> fungi.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D8%26rft.spage%253De1003037%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003037%26rft_id%253Dinfo%253Apmid%252F23236275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23236275&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.41" data-doi="10.1080/09593330903370026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oren</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Industrial and environmental applications of halophilic microorganisms.</span> <abbr class="cit-jnl-abbrev">Environ. Technol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">825</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BTechnology%2BLetters%26rft.stitle%253DEnvironmental%2BTechnology%2BLetters%26rft.aulast%253DOren%26rft.auinit1%253DA.%26rft.volume%253D31%26rft.issue%253D8-9%26rft.spage%253D825%26rft.epage%253D834%26rft.atitle%253DIndustrial%2Band%2Benvironmental%2Bapplications%2Bof%2Bhalophilic%2Bmicroorganisms.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F09593330903370026%26rft_id%253Dinfo%253Apmid%252F20662374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/09593330903370026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20662374&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.42" data-doi="10.1007/s00792-014-0654-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oren</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Taxonomy of halophilic Archaea: current status and future challenges.</span> <abbr class="cit-jnl-abbrev">Extremophiles</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">825</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExtremophiles%26rft.volume%253D18%26rft.spage%253D825%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00792-014-0654-9%26rft_id%253Dinfo%253Apmid%252F25102811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00792-014-0654-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25102811&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Funannotate: Fungal genome annotation scripts.</span> <a href="https://github.com/nextgenusfs/funannotate">https://github.com/nextgenusfs/funannotate</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.44" data-doi="10.1093/bioinformatics/btm071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm071%26rft_id%253Dinfo%253Apmid%252F17332020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMy85LzEwNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjAxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.45" data-doi="10.1038/nbt.3122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonescu</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">T.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendell</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D290%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3122%26rft_id%253Dinfo%253Apmid%252F25690850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25690850&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plemenitaš</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenassi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kejžar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zajc</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Adaptation to high salt concentrations in halotolerant/halophilic fungi: a molecular perspective.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D5%26rft.spage%253D199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.47" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.48" data-doi="10.1002/yea.1515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Yeast genome evolution—the origin of the species.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DScannell%26rft.auinit1%253DD.%2BR.%26rft.volume%253D24%26rft.issue%253D11%26rft.spage%253D929%26rft.epage%253D942%26rft.atitle%253DYeast%2Bgenome%2Bevolution--the%2Borigin%2Bof%2Bthe%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1515%26rft_id%253Dinfo%253Apmid%252F17621376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17621376&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251549800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.49" data-doi="10.1371/journal.pgen.0030068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schoustra</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Debets</span>  <span class="cit-name-given-names">A. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slakhorst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mitotic recombination accelerates adaptation in the fungus Aspergillus nidulans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSchoustra%26rft.auinit1%253DS.%2BE.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253De68%26rft.epage%253De68%26rft.atitle%253DMitotic%2Brecombination%2Baccelerates%2Badaptation%2Bin%2Bthe%2Bfungus%2BAspergillus%2Bnidulans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030068%26rft_id%253Dinfo%253Apmid%252F17465683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17465683&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.50" data-doi="10.1093/jxb/erv465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shabala</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bose</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuglsang</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pottosin</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">On a quest for stress tolerance genes: membrane transporters in sensing and adapting to hostile soils.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1015</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferv465%26rft_id%253Dinfo%253Apmid%252F26507891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjk6IjY3LzQvMTAxNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.51" data-doi="10.1186/1471-2105-6-31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">G. S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Automated generation of heuristics for biological sequence comparison.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSlater%26rft.auinit1%253DG.%2BS.%26rft.volume%253D6%26rft.spage%253D31%26rft.epage%253D31%26rft.atitle%253DAutomated%2Bgeneration%2Bof%2Bheuristics%2Bfor%2Bbiological%2Bsequence%2Bcomparison.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-31%26rft_id%253Dinfo%253Apmid%252F15713233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15713233&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.52" data-doi="10.1101/gr.361602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Block</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulez</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chervitz</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Bioperl toolkit: perl modules for the life sciences.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1611</span>–<span class="cit-lpage">1618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DStajich%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D1611%26rft.epage%253D1618%26rft.atitle%253DThe%2BBioperl%2BToolkit%253A%2BPerl%2BModules%2Bfor%2Bthe%2BLife%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.361602%26rft_id%253Dinfo%253Apmid%252F12368254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMC8xNjExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.53" data-doi="10.1186/1471-2105-7-62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schöffmann</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgenstern</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waack</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.spage%253D62%26rft.epage%253D62%26rft.atitle%253DGene%2Bprediction%2Bin%2Beukaryotes%2Bwith%2Ba%2Bgeneralized%2Bhidden%2BMarkov%2Bmodel%2Bthat%2Buses%2Bhints%2Bfrom%2Bexternal%2Bsources.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-62%26rft_id%253Dinfo%253Apmid%252F16469098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16469098&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.54" data-doi="10.1073/pnas.1201403109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christiansen</span>  <span class="cit-name-given-names">F. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutheil</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fusion of two divergent fungal individuals led to the recent emergence of a unique widespread pathogen species.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">10954</span>–<span class="cit-lpage">10959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1201403109%26rft_id%253Dinfo%253Apmid%252F22711811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzI3LzEwOTU0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.55" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative genomics viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvMTc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.56" data-doi="10.1007/978-1-61779-477-3_23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsui</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbic</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gebbia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nislow</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic approaches for determining nucleosome occupancy in yeast.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">833</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DTsui%26rft.auinit1%253DK.%26rft.volume%253D833%26rft.spage%253D389%26rft.epage%253D411%26rft.atitle%253DGenomic%2Bapproaches%2Bfor%2Bdetermining%2Bnucleosome%2Boccupancy%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-477-3_23%26rft_id%253Dinfo%253Apmid%252F22183606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-477-3_23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22183606&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.57" data-doi="10.1098/rstb.2013.0353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vanneste</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maere</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Tangled up in two: a burst of genome duplications at the end of the Cretaceous and the consequences for plant evolution.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B. Biol. Sci.</abbr> <span class="cit-vol">369</span>: <span class="cit-fpage">20130353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2013.0353%26rft_id%253Dinfo%253Apmid%252F24958926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjE3OiIzNjkvMTY0OC8yMDEzMDM1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.58" data-doi="10.1186/1471-2164-8-280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaupotic</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plemenitaš</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Differential gene expression and Hog1 interaction with osmoresponsive genes in the extremely halotolerant black yeast <em>Hortaea werneckii</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DVaupotic%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.spage%253D280%26rft.epage%253D280%26rft.atitle%253DDifferential%2Bgene%2Bexpression%2Band%2BHog1%2Binteraction%2Bwith%2Bosmoresponsive%2Bgenes%2Bin%2Bthe%2Bextremely%2Bhalotolerant%2Bblack%2Byeast%2BHortaea%2Bwerneckii.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-280%26rft_id%253Dinfo%253Apmid%252F17705830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17705830&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2015.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.59" data-doi="10.1073/pnas.74.11.5088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woese</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Phylogenetic structure of the prokaryotic domain: the primary kingdoms.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">5088</span>–<span class="cit-lpage">5090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWoese%26rft.auinit1%253DC.%2BR.%26rft.volume%253D74%26rft.issue%253D11%26rft.spage%253D5088%26rft.epage%253D5090%26rft.atitle%253DPhylogenetic%2Bstructure%2Bof%2Bthe%2Bprokaryotic%2Bdomain%253A%2BThe%2Bprimary%2Bkingdoms%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.74.11.5088%26rft_id%253Dinfo%253Apmid%252F270744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzQvMTEvNTA4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.60" data-doi="10.1093/bioinformatics/btq057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nacu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq057%26rft_id%253Dinfo%253Apmid%252F20147302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi83Lzg3MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.61" data-doi="10.1093/oxfordjournals.molbev.a026236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Estimating synonymous and nonsynonymous substitution rates under realistic evolutionary models.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D43%26rft.atitle%253DEstimating%2BSynonymous%2Band%2BNonsynonymous%2BSubstitution%2BRates%2BUnder%2BRealistic%2BEvolutionary%2BModels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026236%26rft_id%253Dinfo%253Apmid%252F10666704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjE3LzEvMzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjAxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2015.62" data-doi="10.1128/AEM.00821-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Youssef</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couger</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struchtemeyer</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liggenstoffer</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prade</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genome of the anaerobic fungus <em>Orpinomyces</em> sp. strain C1A reveals the unique evolutionary history of a remarkable plant biomass degrader.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">4620</span>–<span class="cit-lpage">4634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00821-13%26rft_id%253Dinfo%253Apmid%252F23709508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3OS8xNS80NjIwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIwMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/7/2015.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1995" title="Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2023" title="RNAi-Mediated Reverse Genetic Screen Identified Drosophila Chaperones Regulating Eye and Neuromuscular Junction Morphology" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/7"><span>Volume 7 Issue 7, July 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/7.cover-source.jpg?itok=eAMfPilr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2015.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5z1kqMlKVfT-odIN6dXqz83Vw04_hm9EkUGCtjvj83Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/7/2015" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/7/2015" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/7/2015" class="active" data-icon-position="" data-hide-link-title="0">Insight into the Recent Genome Duplication of the Halophilic Yeast Hortaea werneckii: Combining an Improved Genome with Gene Expression and Chromatin Structure</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485294" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-p2EbnvrMgF9jr4SA29OyGyLw1Yu3DyJmheTaRwXtkeo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2015.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/7/2015" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_cB1M2I0mpqb81kZG3HbXWGpzX-K9LVXhDK2uHnCiDM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/7/2015" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2015.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485294" id="node485294--2" data-pisa="ggg;7/7/2015" data-pisa-master="ggg;g3.117.040691" data-apath="/ggg/7/7/2015.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Insight into the Recent Genome Duplication of the Halophilic Yeast <em>Hortaea werneckii</em>: Combining an Improved Genome with Gene Expression and Chromatin Structure</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sunita Sinha</span>, <span class="highwire-citation-author" data-delta="1">Stephane Flibotte</span>, <span class="highwire-citation-author" data-delta="2">Mauricio Neira</span>, <span class="highwire-citation-author" data-delta="3">Sean Formby</span>, <span class="highwire-citation-author" data-delta="4">Ana Plemenitaš</span>, <span class="highwire-citation-author" data-delta="5">Nina Gunde Cimerman</span>, <span class="highwire-citation-author" data-delta="6">Metka Lenassi</span>, <span class="highwire-citation-author" data-delta="7">Cene Gostinčar</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jason E. Stajich</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Corey Nislow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2015-2022;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040691 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.040691" data-atom-uri="/ggg/7/7/2015.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485294--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sunita Sinha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sunita%2BSinha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sinha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASunita%2BSinha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stephane Flibotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephane%2BFlibotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Flibotte%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephane%2BFlibotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mauricio Neira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BNeira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neira%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMauricio%2BNeira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sean Formby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BFormby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Formby%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BFormby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ana Plemenitaš </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Biochemistry, Faculty of Medicine, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BPlemenita%C5%A1%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plemenita%C5%A1%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BPlemenita%25C5%25A1%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nina Gunde Cimerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Biotechnical Faculty, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nina%2BGunde%2BCimerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cimerman%20NG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANina%2BGunde%2BCimerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Metka Lenassi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Biochemistry, Faculty of Medicine, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Metka%2BLenassi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lenassi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMetka%2BLenassi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Cene Gostinčar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Biotechnical Faculty, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cene%2BGostin%C4%8Dar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gostin%C4%8Dar%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACene%2BGostin%25C4%258Dar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jason E. Stajich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Pathology and Microbiology, Institute for Integrative Genome Biology, University of California, Riverside, California 92521 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BE.%2BStajich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stajich%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BE.%2BStajich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jason E. Stajich</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jason.stajich@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">jason.stajich@ucr.edu</a>
<a href="mailto:corey.nislow@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@ubc.ca</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Corey Nislow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corey%2BNislow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nislow%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorey%2BNislow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Corey Nislow</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jason.stajich@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">jason.stajich@ucr.edu</a>
<a href="mailto:corey.nislow@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/7/2015" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/7/2015&amp;title=Insight%20into%20the%20Recent%20Genome%20Duplication%20of%20the%20Halophilic%20Yeast%20Hortaea%20werneckii%3A%20Combining%20an%20Improved%20Genome%20with%20Gene%20Expression%20and%20Chromatin%20Structure" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/7/2015&amp;title=Insight%20into%20the%20Recent%20Genome%20Duplication%20of%20the%20Halophilic%20Yeast%20Hortaea%20werneckii%3A%20Combining%20an%20Improved%20Genome%20with%20Gene%20Expression%20and%20Chromatin%20Structure" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/7/2015&amp;title=Insight%20into%20the%20Recent%20Genome%20Duplication%20of%20the%20Halophilic%20Yeast%20Hortaea%20werneckii%3A%20Combining%20an%20Improved%20Genome%20with%20Gene%20Expression%20and%20Chromatin%20Structure" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/7/2015&amp;text=Insight%20into%20the%20Recent%20Genome%20Duplication%20of%20the%20Halophilic%20Yeast%20Hortaea%20werneckii%3A%20Combining%20an%20Improved%20Genome%20with%20Gene%20Expression%20and%20Chromatin%20Structure" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/7/2015&amp;title=Insight%20into%20the%20Recent%20Genome%20Duplication%20of%20the%20Halophilic%20Yeast%20Hortaea%20werneckii%3A%20Combining%20an%20Improved%20Genome%20with%20Gene%20Expression%20and%20Chromatin%20Structure" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/7/2015&amp;t=Insight%20into%20the%20Recent%20Genome%20Duplication%20of%20the%20Halophilic%20Yeast%20Hortaea%20werneckii%3A%20Combining%20an%20Improved%20Genome%20with%20Gene%20Expression%20and%20Chromatin%20Structure" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/7/2015&amp;title=Insight%20into%20the%20Recent%20Genome%20Duplication%20of%20the%20Halophilic%20Yeast%20Hortaea%20werneckii%3A%20Combining%20an%20Improved%20Genome%20with%20Gene%20Expression%20and%20Chromatin%20Structure" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/7/2015&amp;title=Insight%20into%20the%20Recent%20Genome%20Duplication%20of%20the%20Halophilic%20Yeast%20Hortaea%20werneckii%3A%20Combining%20an%20Improved%20Genome%20with%20Gene%20Expression%20and%20Chromatin%20Structure" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485294/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485294" id="node485294--3" data-pisa="ggg;7/7/2015" data-pisa-master="ggg;g3.117.040691" data-apath="/ggg/7/7/2015.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Insight into the Recent Genome Duplication of the Halophilic Yeast <em>Hortaea werneckii</em>: Combining an Improved Genome with Gene Expression and Chromatin Structure</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sunita Sinha</span>, <span class="highwire-citation-author" data-delta="1">Stephane Flibotte</span>, <span class="highwire-citation-author" data-delta="2">Mauricio Neira</span>, <span class="highwire-citation-author" data-delta="3">Sean Formby</span>, <span class="highwire-citation-author" data-delta="4">Ana Plemenitaš</span>, <span class="highwire-citation-author" data-delta="5">Nina Gunde Cimerman</span>, <span class="highwire-citation-author" data-delta="6">Metka Lenassi</span>, <span class="highwire-citation-author" data-delta="7">Cene Gostinčar</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jason E. Stajich</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Corey Nislow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2015-2022;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040691 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.040691" data-atom-uri="/ggg/7/7/2015.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485294--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sunita Sinha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sunita%2BSinha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sinha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASunita%2BSinha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stephane Flibotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephane%2BFlibotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Flibotte%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephane%2BFlibotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mauricio Neira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BNeira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neira%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMauricio%2BNeira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sean Formby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BFormby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Formby%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BFormby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ana Plemenitaš </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Biochemistry, Faculty of Medicine, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BPlemenita%C5%A1%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plemenita%C5%A1%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BPlemenita%25C5%25A1%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nina Gunde Cimerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Biotechnical Faculty, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nina%2BGunde%2BCimerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cimerman%20NG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANina%2BGunde%2BCimerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Metka Lenassi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Biochemistry, Faculty of Medicine, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Metka%2BLenassi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lenassi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMetka%2BLenassi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Cene Gostinčar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Biotechnical Faculty, University of Ljubljana, 1000, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cene%2BGostin%C4%8Dar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gostin%C4%8Dar%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACene%2BGostin%25C4%258Dar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jason E. Stajich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Pathology and Microbiology, Institute for Integrative Genome Biology, University of California, Riverside, California 92521 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BE.%2BStajich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stajich%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BE.%2BStajich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jason E. Stajich</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jason.stajich@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">jason.stajich@ucr.edu</a>
<a href="mailto:corey.nislow@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@ubc.ca</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Corey Nislow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corey%2BNislow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nislow%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorey%2BNislow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Corey Nislow</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jason.stajich@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">jason.stajich@ucr.edu</a>
<a href="mailto:corey.nislow@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@ubc.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2015&amp;rft.epage=2022&amp;rft.atitle=Insight%20into%20the%20Recent%20Genome%20Duplication%20of%20the%20Halophilic%20Yeast%20Hortaea%20werneckii%3A%20Combining%20an%20Improved%20Genome%20with%20Gene%20Expression%20and%20Chromatin%20Structure&amp;rft.volume=7&amp;rft.issue=7&amp;rft.date=2017-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sinha%2C+Sunita&amp;rft.au=Flibotte%2C+Stephane&amp;rft.au=Neira%2C+Mauricio&amp;rft.au=Formby%2C+Sean&amp;rft.au=Plemenita%C5%A1%2C+Ana&amp;rft.au=Cimerman%2C+Nina+Gunde&amp;rft.au=Lenassi%2C+Metka&amp;rft.au=Gostin%C4%8Dar%2C+Cene&amp;rft.au=Stajich%2C+Jason+E.&amp;rft.au=Nislow%2C+Corey"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485294/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485294/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485294/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485294/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485294/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485294/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485294/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485294/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485294/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485294/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485294/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485294/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/7/2015" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/7/2015.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/7/2015.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/7/2015.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/7/2015.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
