<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/7/2315" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/7/2315.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/7/2315.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/7/2315.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/7/2315/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/7/2315.atom" />
<meta name="HW.pisa" content="ggg;7/7/2315" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Rapid Cycling Genomic Selection in a Multiparental Tropical Maize Population" />
<meta name="DC.Identifier" content="10.1534/g3.117.043141" />
<meta name="DC.Date" content="2017-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Zhang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection (GS) increases genetic gain by reducing the length of the selection cycle, as has been exemplified in maize using rapid cycling recombination of biparental populations. However, no results of GS applied to maize multi-parental populations have been reported so far. This study is the first to show realized genetic gains of rapid cycling genomic selection (RCGS) for four recombination cycles in a multi-parental tropical maize population. Eighteen elite tropical maize lines were intercrossed twice, and self-pollinated once, to form the cycle 0 (C) training population. A total of 1000 ear-to-row C families was genotyped with 955,690 genotyping-by-sequencing SNP markers; their testcrosses were phenotyped at four optimal locations in Mexico to form the training population. Individuals from families with the best plant types, maturity, and grain yield were selected and intermated to form RCGS cycle 1 (C1). Predictions of the genotyped individuals forming cycle C1 were made, and the best predicted grain yielders were selected as parents of C2; this was repeated for more cycles (C2, C3, and C4), thereby achieving two cycles per year. Multi-environment trials of individuals from populations C0, C1, C2, C3, and C4, together with four benchmark checks were evaluated at two locations in Mexico. Results indicated that realized grain yield from C1 to C4 reached 0.225 ton ha−1 per cycle, which is equivalent to 0.100 ton ha−1 yr−1 over a 4.5-yr breeding period from the initial cross to the last cycle. Compared with the original 18 parents used to form cycle 0 (C), genetic diversity narrowed only slightly during the last GS cycles (C3 and C4). Results indicate that, in tropical maize multi-parental breeding populations, RCGS can be an effective breeding strategy for simultaneously conserving genetic diversity and achieving high genetic gains in a short period of time." />
<meta name="DC.Contributor" content="Xuecai Zhang" />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="Juan Burgueño" />
<meta name="DC.Contributor" content="Michael Olsen" />
<meta name="DC.Contributor" content="Edward Buckler" />
<meta name="DC.Contributor" content="Gary Atlin" />
<meta name="DC.Contributor" content="Boddupalli M. Prasanna" />
<meta name="DC.Contributor" content="Mateo Vargas" />
<meta name="DC.Contributor" content="Félix San Vicente" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="article:published_time" content="2017-07-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Rapid Cycling Genomic Selection in a Multiparental Tropical Maize Population" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection (GS) increases genetic gain by reducing the length of the selection cycle, as has been exemplified in maize using rapid cycling recombination of biparental populations. However, no results of GS applied to maize multi-parental populations have been reported so far. This study is the first to show realized genetic gains of rapid cycling genomic selection (RCGS) for four recombination cycles in a multi-parental tropical maize population. Eighteen elite tropical maize lines were intercrossed twice, and self-pollinated once, to form the cycle 0 (C&lt;sub&gt;0&lt;/sub&gt;) training population. A total of 1000 ear-to-row C&lt;sub&gt;0&lt;/sub&gt; families was genotyped with 955,690 genotyping-by-sequencing SNP markers; their testcrosses were phenotyped at four optimal locations in Mexico to form the training population. Individuals from families with the best plant types, maturity, and grain yield were selected and intermated to form RCGS cycle&amp;nbsp;1 (C&lt;sub&gt;1&lt;/sub&gt;). Predictions of the genotyped individuals forming cycle C&lt;sub&gt;1&lt;/sub&gt; were made, and the best predicted grain yielders were selected as parents of C&lt;sub&gt;2&lt;/sub&gt;; this was repeated for more cycles (C&lt;sub&gt;2&lt;/sub&gt;, C&lt;sub&gt;3&lt;/sub&gt;, and C&lt;sub&gt;4&lt;/sub&gt;), thereby achieving two cycles per year. Multi-environment trials of individuals from populations C&lt;sub&gt;0,&lt;/sub&gt; C&lt;sub&gt;1&lt;/sub&gt;, C&lt;sub&gt;2&lt;/sub&gt;, C&lt;sub&gt;3&lt;/sub&gt;, and C&lt;sub&gt;4&lt;/sub&gt;, together with four benchmark checks were evaluated at two locations in Mexico. Results indicated that realized grain yield from C&lt;sub&gt;1&lt;/sub&gt; to C&lt;sub&gt;4&lt;/sub&gt; reached 0.225&amp;nbsp;ton&amp;nbsp;ha&lt;sup&gt;−1&lt;/sup&gt; per cycle, which is equivalent to 0.100&amp;nbsp;ton&amp;nbsp;ha&lt;sup&gt;−1&lt;/sup&gt;&amp;nbsp;yr&lt;sup&gt;−1&lt;/sup&gt; over a 4.5-yr breeding period from the initial cross to the last cycle. Compared with the original 18 parents used to form cycle 0 (C&lt;sub&gt;0&lt;/sub&gt;), genetic diversity narrowed only slightly during the last GS cycles (C&lt;sub&gt;3&lt;/sub&gt; and C&lt;sub&gt;4&lt;/sub&gt;). Results indicate that, in tropical maize multi-parental breeding populations, RCGS can be an effective breeding strategy for simultaneously conserving genetic diversity and achieving high genetic gains in a short period of time.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/07/01" />
<meta name="citation_mjid" content="ggg;7/7/2315" />
<meta name="citation_id" content="7/7/2315" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/7/2315" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/7/2315.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/7/2315.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/7/2315.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.043141" />
<meta name="citation_pmid" content="28533335" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="2315" />
<meta name="citation_lastpage" content="2326" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Xuecai Zhang" />
<meta name="citation_author_institution" content="Global Maize Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, CP 56230, Montecillos, 56230 México D.F., México" />
<meta name="citation_author" content="Juan Burgueño" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México" />
<meta name="citation_author" content="Michael Olsen" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Nairobi 1041-00621, Kenya" />
<meta name="citation_author" content="Edward Buckler" />
<meta name="citation_author_institution" content="United States Department of Agriculture, Agricultural Research Service, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Gary Atlin" />
<meta name="citation_author_institution" content="Bill and Melinda Gates Foundation, Seattle, Washington 98109" />
<meta name="citation_author" content="Boddupalli M. Prasanna" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Nairobi 1041-00621, Kenya" />
<meta name="citation_author" content="Mateo Vargas" />
<meta name="citation_author_institution" content="Universidad Autónoma Chapingo, 56230 Texcoco, México" />
<meta name="citation_author" content="Félix San Vicente" />
<meta name="citation_author_institution" content="Global Maize Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México" />
<meta name="citation_author_email" content="f.sanvicente@cgiar.org" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México" />
<meta name="citation_author_email" content="f.sanvicente@cgiar.org" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Ahfock;citation_author=I. Wood;citation_author=S. Stephen;citation_author=C. R. Cavanagh;citation_author=B. E. Huang;citation_title=Characterizing Uncertainty in High-Density Maps from Multiparental Populations;citation_pages=117-128;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236453;citation_doi=10.1534/genetics.114.167577" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Molecular Markers and Selection for Complex Traits in Plants: Learning from the Last 20 Years;citation_pages=1649-1664;citation_volume=48;citation_year=2008;citation_issue=5;citation_doi=10.2135/cropsci2008.03.0131" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Bernardo;citation_title=Bandwagons I, too, have known.;citation_pages=2323-2332;citation_volume=129;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=S. Mugo;citation_author=A. Tarekegne;citation_author=R. Babu;citation_title=Genetic gains in grain yield through genomic selection in eight bi-parental maize populations under drought stress.;citation_pages=154;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.07.0460" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=C. Cavanagh;citation_author=M. Morell;citation_author=I. Mackay;citation_author=W. Powell;citation_title=From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants.;citation_pages=215-221;citation_volume=11;citation_year=2008;citation_issue=2;citation_pmid=18295532;citation_doi=10.1016/j.pbi.2008.01.002" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. de Roos;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Reliability of genomic predictions across multiple populations.;citation_volume=183;citation_year=2009;citation_pmid=19822733;citation_doi=10.1534/genetics.109.104935" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping-by-sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=A. Hoffstetter;citation_author=A. Cabrera;citation_author=M. Huang;citation_author=C. Sneller;citation_title=Optimizing training population data and validation of genomic selection for economic traits in soft winter wheat.;citation_pages=2919-2928;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=B. E. A. Huang;citation_author=W. George;citation_author=K. L. Forrest;citation_author=A. Kilian;citation_author=M. J. Hayden;citation_title=A multiparent advanced generation inter-cross population for genetic analysis in wheat.;citation_pages=826-839;citation_volume=10;citation_year=2012;citation_issue=7;citation_pmid=22594629;citation_doi=10.1111/j.1467-7652.2012.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J.-L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding from theory to practice.;citation_volume=9;citation_year=2010;citation_issue=2;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Lehermeier;citation_author=N. Kramer;citation_author=E. Bauer;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Usefulness of Multiparental Populations of Maize (Zea mays L.) for Genome-Based Prediction;citation_pages=3-16;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236445;citation_doi=10.1534/genetics.114.161943" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Massman;citation_author=H.-J. G. Jung;citation_author=R. Bernardo;citation_title=Genome-wide selection vs. marker-assisted recurrent selection to improve grain yield and stover-quality traits for cellulosic ethanol in maize.;citation_pages=58-66;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.02.0112" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=B. Masuka;citation_author=G. Atlin;citation_author=M. Olsen;citation_author=C. Magorokosho;citation_author=M. Labuschagne;citation_title=Gains in maize genetic improvement in Eastern and Southern Africa: I. CIMMYT hybrid breeding pipeline.;citation_pages=168;citation_volume=57;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=B. Masuka;citation_author=C. Magorokosho;citation_author=M. Olsen;citation_author=G. Atlin;citation_author=M. Bänziger;citation_title=Gains in maize genetic improvement in Eastern and Southern Africa: II. CIMMYT open-pollinated variety breeding pipeline.;citation_pages=180;citation_volume=57;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Rebai;citation_author=B. Goffinet;citation_title=Power of tests for QTL detection using replicated progenies derived from a diallel cross.;citation_pages=1014-1022;citation_volume=86;citation_year=1993;citation_pmid=24194011" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Genetic gain from phenotypic and genomic selection for quantitative resistance to stem rust of wheat.;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=B. Bhavani;citation_author=J. Poland;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.;citation_author=K. Swarts;citation_author=H. Li;citation_author=J. A. R. Navarro;citation_author=D. An;citation_author=M. C. Romay;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Thepot;citation_author=G. Restoux;citation_author=I. Goldringer;citation_author=F. Hospital;citation_author=D. Gouache;citation_title=Efficiently tracking selection in a multiparental population: the case of earliness in wheat.;citation_volume=199;citation_year=2015;citation_pmid=25406468;citation_doi=10.1534/genetics.114.169995" />
<meta name="citation_reference" content="citation_journal_title=Interbull Bull.;citation_author=P. M. VanRaden;citation_title=Genomic measures of relationship and inbreeding.;citation_pages=33-36;citation_volume=37;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=K. J. F. Verhoeven;citation_author=J.-L. Jannink;citation_author=L. M. McIntyre;citation_title=Using mating designs to uncover QTL and the genetic architecture of complex traits.;citation_pages=139-149;citation_volume=96;citation_year=2006;citation_issue=2;citation_pmid=16304603;citation_doi=10.1038/sj.hdy.6800763" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Use of Genomic estimated breeding values results in rapid genetic gains for drought tolerance in maize.;citation_author=B. S. Vivek;citation_author=G. K. Krishna;citation_author=V. Vengadessan;citation_author=R. Babu;citation_author=P. H. Zaidi;citation_volume=10;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Y. Wu;citation_author=F. San Vicente;citation_author=K. Huang;citation_author=T. Dhliwayo;citation_author=D. E. Costich;citation_title=Molecular characterization of CIMMYT maize inbred lines with genotyping-by-sequencing SNPs.;citation_pages=753-765;citation_volume=129;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=X. Zhang;citation_author=P. Pérez-Rodríguez;citation_author=K. Semagn;citation_author=Y. Beyene;citation_author=R. Babu;citation_title=Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS SNPs.;citation_pages=291-299;citation_volume=114;citation_year=2015;citation_pmid=25407079;citation_doi=10.1038/hdy.2014.99" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J. Dekkers;citation_author=R. L. Fernando;citation_author=J. L. Jannink;citation_title=Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study.;citation_volume=182;citation_year=2009;citation_pmid=19299342;citation_doi=10.1534/genetics.108.098277" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Rapid Cycling Genomic Selection in a Multiparental Tropical Maize Population" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection (GS) increases genetic gain by reducing the length of the selection cycle, as has been exemplified in maize using rapid cycling recombination of biparental populations. However, no results of GS applied to maize multi-parental populations have been reported so far. This study is the first to show realized genetic gains of rapid cycling genomic selection (RCGS) for four recombination cycles in a multi-parental tropical maize population. Eighteen elite tropical maize lines were intercrossed twice, and self-pollinated once, to form the cycle 0 (C) training population. A total of 1000 ear-to-row C families was genotyped with 955,690 genotyping-by-sequencing SNP markers; their testcrosses were phenotyped at four optimal locations in Mexico to form the training population. Individuals from families with the best plant types, maturity, and grain yield were selected and intermated to form RCGS cycle 1 (C1). Predictions of the genotyped individuals forming cycle C1 were made, and the best predicted grain yielders were selected as parents of C2; this was repeated for more cycles (C2, C3, and C4), thereby achieving two cycles per year. Multi-environment trials of individuals from populations C0, C1, C2, C3, and C4, together with four benchmark checks were evaluated at two locations in Mexico. Results indicated that realized grain yield from C1 to C4 reached 0.225 ton ha−1 per cycle, which is equivalent to 0.100 ton ha−1 yr−1 over a 4.5-yr breeding period from the initial cross to the last cycle. Compared with the original 18 parents used to form cycle 0 (C), genetic diversity narrowed only slightly during the last GS cycles (C3 and C4). Results indicate that, in tropical maize multi-parental breeding populations, RCGS can be an effective breeding strategy for simultaneously conserving genetic diversity and achieving high genetic gains in a short period of time." />
<meta name="og-title" property="og:title" content="Rapid Cycling Genomic Selection in a Multiparental Tropical Maize Population" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/7/2315" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection (GS) increases genetic gain by reducing the length of the selection cycle, as has been exemplified in maize using rapid cycling recombination of biparental populations. However, no results of GS applied to maize multi-parental populations have been reported so far. This study is the first to show realized genetic gains of rapid cycling genomic selection (RCGS) for four recombination cycles in a multi-parental tropical maize population. Eighteen elite tropical maize lines were intercrossed twice, and self-pollinated once, to form the cycle 0 (C) training population. A total of 1000 ear-to-row C families was genotyped with 955,690 genotyping-by-sequencing SNP markers; their testcrosses were phenotyped at four optimal locations in Mexico to form the training population. Individuals from families with the best plant types, maturity, and grain yield were selected and intermated to form RCGS cycle 1 (C1). Predictions of the genotyped individuals forming cycle C1 were made, and the best predicted grain yielders were selected as parents of C2; this was repeated for more cycles (C2, C3, and C4), thereby achieving two cycles per year. Multi-environment trials of individuals from populations C0, C1, C2, C3, and C4, together with four benchmark checks were evaluated at two locations in Mexico. Results indicated that realized grain yield from C1 to C4 reached 0.225 ton ha−1 per cycle, which is equivalent to 0.100 ton ha−1 yr−1 over a 4.5-yr breeding period from the initial cross to the last cycle. Compared with the original 18 parents used to form cycle 0 (C), genetic diversity narrowed only slightly during the last GS cycles (C3 and C4). Results indicate that, in tropical maize multi-parental breeding populations, RCGS can be an effective breeding strategy for simultaneously conserving genetic diversity and achieving high genetic gains in a short period of time." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
    <title>Rapid Cycling Genomic Selection in a Multiparental Tropical Maize Population | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"875z6ULhQJujuQzRU8ugVTQawB7c96uJ_4B6hrQDzQY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485325","apath":"\/ggg\/7\/7\/2315.atom","pisa":"ggg;7\/7\/2315","ac":{"\/ggg\/7\/7\/2315.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/7\/2315.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2315"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2315"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/7\/2315":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UqKXEhkfnJc+9FmGISjTw3hwp4R3bw+WfO2FZB2YZiz+LFbFYdiM2lULYEt\\\\\\\/DZE7T4KeEdFCc8A9A1X85jLq4R\\\\\\\/DZmG4VLHJsr2Q6QMLLQbFy8S0CAX6P\\\\\\\/6mzz4PGDl9twPhko7Szi0p199QYq9r9KpnOqLW45vBEmn3UDSzdCRPLqsUp06XNoIJRYWaUcssQMjz4qP4gxMAsBRpsotlhsrnmbWZy+ZEoJLgNlQC\\\\\\\/isC5sxwGM4ePe8YgejGh7LsXn1D4RrhewirmeDR\\\\\\\/hoVAsiuSPCGhKtZJz90bKHKiEH097BZZFOiy+GwEq18gw5esdibnzBTpXKHx1o0s885WlTh3e7CSUzETO7CzwDQ10O2lHbZX62+e9VZ5Pp2hZFbSCtivc9pU8v3qFmZKCdOdPScYfn4KPOxY1gjAU2DeqCMmFRqPy3I\\\\\\\/yonU\\\\\\\/206lN8+HrCyIyxV+Pd0ZfPV7y4gewC3H3g1kN6GKrtzDq+t\\\\\\\/+JajwNWH36e6IUX6g\\\\\\\/USdGmtWRg3EBuUQ2iU3Ficksp8SDYT24yXjXXTl5x7U2tQ6FosE4PUEsJMJdzj0CeQY0bJm6mRCK3zcsfZAdnXQ0+5FH+C6Mi\\\\\\\/CRHlKlnNocjFr1dzTKrbFqeXpna8a8WpVukAYgTdjPaeCXFw1i3UGDEWPAHt1A37qFoK97Fy0jgghdpXG\\\\\\\/3nlc4d2qNuavxsCH6FeTN+zZkJ4MOqAvVC7UVjgG4xH3fJeT4lJHAuuaeN+33G8yJ2PBSJnhmdnSpv9r7Amn8BSvSv8q11UkGJDLjcLtCbJ\\\\\\\/V3H9YfoOw8EVYOpyIBUKxANykBOW1wMgUNbSHdYQWrdiX39H0dMW5ZwMNRoRZI+cuAFFwu0AacDq2sJlppQ1YypBWF+1wo28m8o7uEwOuE9Mju4Qdb0rl0TLwbQopFsklOhEVy90ow4QGXrm4mPull3258OnkJYIAtPVGg2FsWUoBWEJ\\\\\\\/MCzlbH6L8zK1tOLsh7iei3sqsk7SN1o2ZJLCppZ7DIChTNZCsp4lb6GVfP+9RJd7+e7H\\\\\\\/nxtbORdGx\\\\\\\/Bph\\\\\\\/0IT3BTW6L1b29NXOwZirMXx+hIAswvfBoZZfMujevXdbQRcl2cLdXuLEbZV37m1zDNkxRJvNmn1et8hXUsFnCQoTOqs88yKo4OwCoPEXA+hHro\\\\\\\/UyGkbR+AsEBKAZUvFf9cPgB2TQwcT62yl4XWiUaAnq8JxrgIaMmLB4+82tbArlhzeGskMDlxn8FW0rBgL0E0UXiNZgh7gwJZ9eA4wcnz7jJ7pr2qb\\\u0022,\\\u0022iv\\\u0022:\\\u0022EKnxrdRP9gRbvmB+i1caOQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e278676d23f2452d5c91d2b139bf1e857c180bb38ad4a564f56c9f15c4323a8d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225XMV+VgEjA+eWouNuDK6GT7NzIyytuyFVEzurAHwux7IIa5gf7Tq6owVT1lJcOJi3KsHa+IWbeIgMoEKpYWI9nis73FWtZKFBmFTE8pyG0b81WewvDmqZL0Js2l3OtsGBhhQrku\\\\\\\/EL4QdIfXep1DA3PDXyGGCZG459DKdCP\\\\\\\/ty3D73Z1UqU24K4V8IviNLuPu28alc6dawewFAA6H1oUZPbHBNh\\\\\\\/SEw0YdbWdYdgyqnAu03KEWzWqdYANxktr69K\\\\\\\/NWH\\\\\\\/Mf1Zh6kOroofvIDhzWi\\\\\\\/S\\\\\\\/FoPvDofkm5zKyhxvfRCvOpBpn8ad9xG0eiRxSxjBQFOqBCVIZT\\\\\\\/O2\\\\\\\/URCPLoi34hiY3qyGpHiXJq9wNGQnS9tzGF6vc5Pyo94zXL8YDL3id5YTyUVHSxD6rPPVOl9Csxuve9b6nERHr4IBrxTUwJTfY3jj5cshUNRGLl4fwWoG9YmhVa1AhJKEsDeU0m1tlerQd9vFTPFyt9UH+lILIO0NfmO+v6PPHmY0jXVLjJZjlUSrxZqfE83zC+WSNQfY7cvxwxCcXhA8wVPSlupLf\\\\\\\/lx0Z8Ab65bTrHa3ZtdUJS3GE7yGLZshyNNQUNDhywGOPN9y9WWtLQ4OX0LiiWaRZW3FCAmaRFq0bcL6CYqLvti3mewH35V9lY\\\\\\\/BT0lFGgb1fE9LbqRPMkPRb6fBco0C6hmXkd4yL8WhMlsAgm0yKCE8FDnPx5sP+xfUA7mGf4JUjeBsCJebKE93\\\\\\\/NJ6qbI6YV7+V1h95Wv0N8vahyHpDOW0irxA4gf\\\\\\\/NxiJ9VHS1flXjbWSgSViLWGO\\\\\\\/wt4dMqUpzc9sng8wHDksCr\\\\\\\/sqJfleo6MDudMxuPXdY2mR6GtoL1qY3ysCdcUGbHc66kDv0jIEHeYerAYzE44mnGQPi1cFnB2KFQbQmEbYF9TiQFTyZeUjD4Q+edy8ynxAk0sidvOT3qzJTZImHv2R52Zggpz6P6q05KNHk6i24VKnXmdjz7JU9l1EERI5lzv1J\\\\\\\/+MJ7oM5LZaiJr\\\\\\\/k+ugK\\\\\\\/YxJjaoF0MkzSr0cteDpVGBrm8md0IZSk6O362IM9BZuSiBUIkTVRyNdQ67kM\\\\\\\/IcIJR6CbGkMtENDCCbFlAiDD+gSdUyovAhT+dsZiTt\\\\\\\/a1ymVIsrEHzECFHQMq8OL4L77ykkmHrcQ1gMebpvayDC+OSsmcZocZOTHAwFZazE6Qm9o2LeTo2HOOSV1YrtiI+DIcsdNoYWEANnmRFhZpRpYhFNGDm7OahPqQFzlSBJm4BHhM\\\\\\\/btSBXlIl9gvtLcxw495jDsCb3eZdQg2C31TgsVCtGjkvK3LQjDobZ\\\\\\\/dTzIqdCFsbpjK9Q2Lh9mMdxE+5nZ4NCWkd9odshr7qpFqtxWFECJ83bLrgEO9oaqyKbQseYdoIv8sIsfkUYKF1wLZ0K2hZF4FCzbsc4t1hykKfMfns3qVQGuGuGmD4iqOq0QCBNo5PSd17Tbu7oJRNwmfzR6HevWqxmzwdoR5XRHTrFiquAdHlVUaakiIqAcy9Qj+3V+b4Mf5HZeWGIPRIpQC9bD7IiTQLVgjDul6SZVND38fJJPkS4uSI69bzOFuDALhuo94Mc1TWSfJmE0OoQH3OPnSvlPj08ZJu8ro7uM1XWasa3RP68xPRY70p0HmtvSKpwb40hBdLq6BmSgKDvt0T3IrBc8UjB7urkt7BSYKBOXB21s7kQM08QRfNGnl6U9xPV+Ao4RJiQqI4IRacP4dVOJiB9eNK870CKOVBOlYIt12IDZi4XX\\\\\\\/ZeY5zrJZ3VUVe4scI4z8T6Jp42R6KuOMXikUAIQNB5YZ+dS7T18rtxP17wwnYE5NPQbeonkk48arFZzMcx9GTkUXgZDmelea6CBVt3O\\\\\\\/pWbAXd4ET9HUkwS+u8PnOMlmrXGHWIj1Y2sljzd8VFT7tOZX4y79FpIF+aKkEIPhctR1+ziGJlAI4PJpxWjCN5zNNVZpQ0fAADJJXvlj+J37k9obi+8Qh6D7hwwiBY33KDe0VPZLMjUfroCOci6SQhuKcXaLaQ2nTnT7MTXtU7g5n7PEu8rr9LwwV9q51QgmlF84eGj1+2LRTpBIDgYGnLW1osm4FbTjeRzclMnS7\\\\\\\/OFYwHuysucZdvLjgh9wdCvpmka3GbGeV3dYQ+c308QiHfnbUlt+ExGya0AlFrJPjo73LsgvXiFhyWoytiizHq90YDDgKsEUiOmkGDk88GN+daV9iXO1x4k3uwTCDk2vZBppbpQB61OTntUMdDP72G2E2\\\\\\\/vSAAysxAVe4j5yZd7EAwnvUmn7jNcYCINWHrofQhaRpixhbwIiHKEV4bKm\\\\\\\/73I4x1PC8erMq3w\\\\\\\/1knE31znouT3P2TcYxs4hqM7e34XZN+R6x7jQWQWLmSg+VI\\\\\\\/bV4qJF+LOwmwa1cgcA1xDMb\\\\\\\/usEsbF4l3p4Ur3WAtedAf7qzzk315orQdqzuPrwNisKq5bbGqZFGQAZ9lQoDy82x+kSRmgdqE4RnRGIpwmzmfXLIUTU2NR8AEebvpt1lcflDZFPh3L3mHnSuEq5UZ4\\\\\\\/DswFA3Mfp3suBSggwvL6NOS60scukppYYuVZrBR6tAWbVZKgacCF96wN7Vi8Qp2\\\\\\\/GLwM8VthBw+WsJv6mavH5FJz1z2JYkD5Xx7RNCp4WPnVtPMlZWTyfam20Ro6kUwKynx1puBM\\\\\\\/9O8Fld+rOUppM5xeVP9n7jxQIxw6Is24DucPlUjPi6e7UrzIQ3F2rs5bMq\\\\\\\/O8+UfarRbs3Ey0GGm6OtciRAwD7c06DLwtMoa+38tjpn1EDJNcPs8OiFaTIccMTodcZ\\\\\\\/Y3Ctwv5ydGgfVVuJUayg9yOW3O04OZqjm4LAZeqa5fcWGYCQ39rYPQy\\\\\\\/rE4V69FiN6ysZmXdCfZU+R3ny7ckc\\\\\\\/ZOgFV21Unv8aZLxPS+iwYbXuFXA2QRL0UHysWPRjzaR9iGZHQc8cikx+1H1dbrpltpsRp88zw\\\\\\\/4aG7yzheMd13L9d\\\\\\\/Z7e1rWlBLpYQVFZ6yIIAKw8T6Cs\\\\\\\/tBrhSlwQB4uZ+xULFtqcKVQu5DNe11iNx5PQJQ60FG7+AQ4fuhIOEyfPo4u6WJOXJpr8lh51ii66qeY7xbQqf3o5OUYpc27J5SraAuRefsi7IWzQyvaL5FOS3G9aCFUOhGz+dfiWaCPaP3dTa6elxPUayOSIhFdHalh+TMqA+u0qfzhF1KjSfxriC7tEjFWmMcK\\\\\\\/ocrCPHpZTXHFR4vIAWUmNza8x0QqeI4kJgt2zwUMh9zj\\\\\\\/1GoBKkBFLCsEVp4Drzta6rFfNL7hyQADlouiVVCPO+R47EwAdGObydsEwg879ter3nhmOuPp1gHEAn++Q+m0DQR5fLrwKbg4hxj0kbMBz42okkFqp9c5CiK7uF\\\\\\\/K5FIwMC4qCKQlXHCHlDdgQeIvjZ7J7Gt6NPz7zolSKSGlr0hnUtAof97ZaEKVk0tlmtjSu27c14kLp2C55qdIIeb5HRPxxc7NBME54INsi6d\\\\\\\/02T9C7DR9w2y60XQS1xlV\\\\\\\/SR4bnJ+4UA6gaZtDpDTVNI0aV0cBDHCZkamI+cWrV0iVEkPNUGTbqZRSTgxeCtyfaVh\\\\\\\/8Kfdij7Pu0UhUtBUHD+Y98dUlaY92GWcCM3TGZ5u9LuvnBQtsXg5PePFUkYYsJVrxVcQPCxt1+HxhJJ+CUZH1oExq7ID9+s5WL34LLJTab\\\\\\\/bNxWbbL60w6edOh0lj8FJlKCJdC5sJ6\\\\\\\/6qpV9w7wa7GlcKL+BaxXYY6aJHRup2sezVRmZ3NKFuINergUj+utVIGSAY2juLeltS3jNggrDzSXY1rK7BVVWZgNXEJnLOAXeqoUKZOiVLTppE9x5YPyQT1FJc0QNDVAGaCHBCY+nbf5T80hssED5wjQ3ph1Ua1nlxn2Sd4\\\\\\\/Fip1J42gINyvurnXvKt\\\\\\\/x1N3xIJeW+cUcfeyDmCt04ll6GE+F9bsLk4DT2iQrU\\\\\\\/ZjNA4EP\\\\\\\/EKnasyB8tSW79TP5xq\\\\\\\/i52ZYgeohfIlN7ilvitmZtr34c9ILgrhiV6fKgWE4hya02jol\\\\\\\/nb34l4JLNn9AE+5a1ABiHoVrgclsrZjZMHBUAABw9FxMJepqCmu9SkGRgrzd0mM41iWYqihEU44eEb7qJiWx0FVrfrdcoU+zU0ujb5038KBWzG0eIz9JSBtXutAZSa7kydeJCPg13z1j5\\\\\\\/GoDC5A9ub1jag4WBVKM7xBg3p3MoUwgFtVp3RezhVfbtqA1+zFvNQfa9Qcxu2X619obzAOOWlYMaWbRDYxCbFGIgToYg+cSEZ3MXTHA9UzY5ZQBrfVyyp\\\\\\\/17lSHvHEcewIGXMsVcR6Y\\\\\\\/peU\\\\\\\/UkhqBSmXPFY2yaBspjvBGarpCJNxdIoOJoH58AyzP0locezqvrz8yP3GKtpX\\\\\\\/9vq4eRDik72efIiq37hvwvdBqjgm9XMwv7O0R2JeUKK91XRliAFJ3SaO4oTvss76kfTGbzwwWH2uK0H\\\\\\\/l0Y71kXa8BqnNqJsludjefIU8cjyOVBdNyl7fMkmiawDn8I6B+4V5DxLtH0r8W7gA53GxjWk0spl34PprJpKXdIDtQaFudHHv4RPp+wM4nNzBuGODSwQOUiYwEb4ztmWgRlk5QvhuBZL\\\\\\\/RxC3ajTXg788+EgVLyjsNIOg39YBlPwiCgwa6cl376DbrZ51dzzjnZaskFakFthhCEhcZYf7AvFcK6x3O03JUL+hvWPhvofHX0eoXEPEJWxPsfxgecYHiIz5IIFozDpr4lNkRC1koz6enpenShdGvKAbHmrh0+M9m1AL1RxRbkrBg6wMYKAl\\\\\\\/NkBMJNZ8T1Hp\\\\\\\/CQrO0c7dymp8EIbh0zhHocCp\\\\\\\/Ot7iy2qisFJTBv75kNF4TpQ4lssrjaWCSvOGYxkwwRYvT6t6UhQGhppI37KyVtNhSKslOqZic00tFOK+mrmuTaOs9xrqxFhJYMK2Y9eAOCDgAckt6VCz1TJxnG1qmWfJbHtzATw1b\\\\\\\/\\\\\\\/6B5fY95IessNXIXZl+dsaErjEHxWlN\\\\\\\/SZavFKuWie+SyrZot80B+KIJ3zufb4Awjjssow4fKDwXrb\\\\\\\/mRWYqpHutEqcha8BSI8uGPunYmRJYLFRImgVh5+bWQbn\\\\\\\/oHlBMrGzxMOlbIljl8TYyxka\\\\\\\/5ZYieW7CvPism6Hg+Wy6DC58S7d7h7ZoqiUj485CWVZvWT0BRs0ukdt4LAQ64YzxHs5GTrhUMb2\\\\\\\/1AvFApfwrkH3XLK8XZn+3Zcqjfr\\\\\\\/dB5Anf5V+j2j\\\\\\\/X\\\\\\\/4ZMoe7dCGoHe5LUmHEIgXlK3JOTKfnzGOBvUT82rot4ijnKyh6DaTClQVKiVMuE3rvGOkGecN0nAQ4dBRFpu9c1A2sLFfaKvgAt0fcqX3SJolT2D0KeYbJV+1RXN240ABY3EIbGQkc1HKCtFwi1dDB07eUKSGMtUh3uaNg+voC8h3XXpv2bLJccqke+y3xkvpCRKnaqx0pIOxzEFM6M132LEx5K1YfJr5U8BbRRyFZvANeHIULHjrlqeYUzIzTxYLZWxXsFXAGvYMMZ+dwKBgQQkz2kT3aucTQf5RuUvrr5zqTMdUiDOB+BWuyHrFJhT9xmwTBGdyRh8x38TXNndRiYKiDlVuARQ\\\\\\\/JEpzDBYKqKfPOFU1iOFPaFFhrgr\\\\\\\/4FPt6SwTUWjeHKeMBR2N\\\\\\\/lb5U1YzumRb37ijzm0uTmruuvN1Pzn183AZXzugLYGz6Ul+n892rARhJHU0svhz9+tl6ZGkYbpf7JN4iPFGj0JLQF64TCQCdq0rpBq3s+p7wvpQkZTEIo14vflUUslSCDMp+7dFOpgqZ7dOZ6KFCd0MH0WI2MijLYv2SdMuTXpECNHhVjYYkIgiiZ\\\\\\\/gB61QkA+mlnlKWtKBpsd0liS+QlOfvQnawS8gRaH4x5Zv22oYunSILR29Z7jBgP\\\\\\\/G87SA9b4f4tfXshM7VtMOUyav8mlSZv4lN8KhGuz3RkoIB+79Zp1mkmNQFGrpmLNkaJe85SkXs5R7gn44ujEDu2UhT4KJyTe1qMsOGIZ7jly3BIpizvKvWaTl8K+1jSjg2yjojawEmtApWaWlU4qkUxKkbCNSMQoX1QdiBRW6RLF6LE9Wor+pNm+Y4a53jzGsUHOdIy1pyggXQ2CuFYrPmYKZsHE4MeBiJOozGAvYcJJ0kWDtn6eKSBLXm5ewzw3ZhNnIaFKcoMC4hine1Omi3eYa2qBdnnhtbCITWyJ8CewunB+v7Mvx3H4g9Tip\\\\\\\/SdJvnhkGRd994vOuHVaMuzHgM\\\\\\\/K4Qh1t6\\\\\\\/nXNGdAeHnYOy7j6B3SzufyiE\\\\\\\/DODxcPlX2yxBR4Q5VVsuRz3rEP\\\\\\\/p9Cj5TfamClOheD1WOhIqRHNZRstwnJ0YEMtSgnf+lkzje0pKhmi99Ce9YdxkXHtfQib\\\\\\\/cON8mpaxVvVTC\\\\\\\/b45KpbDcW9TiD+4rUKBbh7f7XqfiHMoShXYQYP0PBCPQhLCmKv7Vv3BUnBulqTsxIHS8Kkb3FUIA\\\\\\\/JA2p57HBgqqf5x4iC2rGoz0uYKg4UVlUtvID8kLXnVquvXRgKX9lV9YNmvDArGEPa2q3A6y9qCh3mdRwYSGXYD2eNRACBIEvbeZuidphCzcuNlzyCAe5PV2O1QO6kv6uGW3uynZuMmFNORNHhPI\\\\\\\/M\\\\\\\/+YDd7ZBmKGvxIG8EaF3JM4y15Oewkxl0pxWq6Zp3p36eA1W2+r42RmeJUvkXXJn+D5N0GlrepZtWdab9NdZHZYeTjQb3PX\\\\\\\/uB9evPHcb3xIgOgl4FtvxMmjAfT5Hr65qbYrMsANnmCJtqilTlc3dQavDnOYEH4FD3hla3L4+1YLL\\\\\\\/qbTo12nnR\\\\\\\/NQihpxbFLz+HmH\\\\\\\/e1fPugHdOBZrkEFvygBKXBytGSsG\\\\\\\/Hb\\\\\\\/Ox0Go9XcwWWdHoxKR5YS8E5LpWzEWI8s3WTHKrLV10+VKD+ERi\\\\\\\/ER\\\\\\\/5sZsC4aiO7sBoBWqZeu9dNkPtlkQ6BAY4PobwWPKy\\\\\\\/72VYS5RI0pCTLSDecOI7lbQajvH150hdgLXdJ0kGngz+B4Lhy3RyLdNCtRuVLMjhRHueyhUoGxHqp3CF\\\\\\\/Xx\\\\\\\/HvbJSQV0Hzcux2T+zHeeLwDiAKtgjZymlBHr+Dui55TbXof+DlDipwWcqP\\\\\\\/HuBbJIEDwQ8NK7PTZpb2Thu8r+ja80Cak\\\\\\\/ZpKO6qpJCMprjhAk6cM1C\\\\\\\/gX5bTRWyoL1RIY1yN1\\\\\\\/D1zOPzJi79AQSnlvdV0I6waiPbdAGoeN2ER+mgoCgtBLJ+TDx2YO+WUH8KV7IVcYss3Vr1cXdyRZM0AOoT4cCHkLY9SF+\\\\\\\/spBrJqDPSmMH4ojLA+CbSMK54DIdmnGkIepSZolwMYTcWSXOIc1DJph8b1nFZBv2jDjMFVB0sU4f9mh6ETjQ7SlL5suTni1Ezup1UhCjdiPfaG12hhkC3jgcQ0PaH\\\\\\\/34G5FVoD\\\\\\\/SzUngSzEbzQxjUZ+vcOzD8hreHkrzIJUkocPlFK0f5I5LZZ\\\\\\\/vZpXthTSn67ZzuCtjEQm3ntMtFDFr17cjWSo9fjEKGzT0nb4rxOzdqRbMCvDkx8laBs2qJKG9CDt+CHbS\\\\\\\/dfkv5q59v5eTheEptrF7qPHhbgiYzqdEoGe\\\\\\\/zHLmWRjxEoY3oM\\\\\\\/Vu2A7lQjovnZ\\\\\\\/nd3WPiDyRMzEXHmhIJb9JMtdlqLoIx9BKNcSa8AwCrTXHpOHxtZNj\\\\\\\/RjKO0Z6qxYIip81woJv4i0gG4zeD8Aer+BzjAACEf2SsKnnMcOVDo4I0pUr8Q5zm89rYiFTQmTjtS5zU\\\\\\\/VeUidLhPVPtjDdOl7x2eKVmGcU+D7DhiMyLBER6iLwZRBIUtdQ3iA+j4jk2lyyl7g7u+mBC+RXqKuuPTVtaoKRXfsRk4T0eCR8PfkzroLCW\\\\\\\/i0\\\\\\\/HizeOgq6BnxYdyooTfAOhSn4T6ONp50S3qhfp+5eqsJQ9JY+0CDB6Gs2PC7O5kkDR6iNs9p4RHgWLMSkSl0NmsptNWawoWkRQV1kPHkPL\\\\\\\/26\\\\\\\/SATsAt86bAJ9oQAwfxPUjn48UQJSdknMdVNURxJXTOlPDltXEZdtzv8mDnLGjE2e\\\\\\\/p7Mhtn\\\\\\\/HYobquG4HBuqdmIsrV\\\\\\\/Yxr1NXZF2ff7amu83Qg1q7DSAXrxmqyzxTYe88vAZ0xzHLDGzzqAQzI4K5UFguoiq0n16avQSKfMbDzeLMiejZSAYItryG\\\\\\\/PEkEE8zMva8kfvne3A5s3eGnqL\\\\\\\/zmxuE5a03muo3IQIkYT4aq15z6ktA\\\\\\\/xNMPutMi7Qv8k50YlGtWJtb\\\\\\\/+tB2ADena+uMb1W9SN3ZOUfKlzLQdoFTISbHDRys8uyD+MS5wjLlF\\\\\\\/rGEv19DlT+1rVRDLltxIwpi2\\\\\\\/0T2NhIlJcEqV358aq9qOytURl5rWiKBl0LeTNCbc2vPd9ORT2hdTujGiz5rEu895FjWfE5irKAkUWTvtGHZOj9GQ2Me1PpkeP\\\\\\\/\\\\\\\/pYuWkXILQLr\\\\\\\/7ojkGGcQ4QSJwVjs8wu5dJGAXg52aLNlbU+SS9sI+B+Rmm9jUwQg1AaIcaiHKNdeflv0\\\\\\\/JzOzFwh5646d49qWyjJ0Gln2276\\\\\\\/OzPhJlj+kYg8csjDtA3rKV8RylVDvWfrHAoE1EhLt1jVyPaYlEtfOCVAt4\\\\\\\/xlAXXxkHxk\\\\\\\/m5wkk8g8WJ3tweLE8oxIbiebU41PBKLy+tr5Zw5IaddluuaNAuV9hMsSV\\\\\\\/+4MrD5dZdBYmA+pR2K2cU1HpRyRMMPDL+Khjul02iOYU9GOHolUDDzn\\\\\\\/DpA08mflnOQudlCqq\\\\\\\/LS6KxQQmmZsdwAwZn0eWSGCdWXnZki1\\\\\\\/r8zXJMOR8Fy2zVSzea\\\\\\\/shvfMnguFOm4uR9+qbOcvMxnOe4YNMnMq8wPKfyBBcNwiksQ7lzlau+eNxoaz4aB8Moh2psTO6GvgzJkWSwrq7k3yNxrwRyjGJJICP2Xz5RxivEt4q7U+Xs3JRmF2Qmfc1+G7g452XLpJkGro9SQrli\\\\\\\/Kgb0JZNdjUxVtdIpj\\\\\\\/qxp1JTKuXyVk2F5UNLRA9G2P6pikHLCQC3C9lSC0Qn6lpWIqgpNEM+Q+NmBJaJvYvKdhgzoFXGaZnCnxhDOW0xB5B3vG56nHykj7dz\\\\\\\/GH\\\\\\\/AwMQaDLkLL1c1c2yz9YHWxMxvPthxG+WeeJ8hQOIGylfg5EAVCYJxY2kW6tFgt3e8Xlm2YcHwGWm2fqRlkM+HMzvJi0W07rojJNkoe455imNMqjTAaSHsOrChOXtgv0dUDiSQN1RzVLnSX9LLv9EfJCUkEgM5swmbQyIJpi3mgiwVtWH78Yu2hndGJ2Qy7Jiyvk9xwEs3lJgbnAAGrFapGF7by4ocTW3psVobMA79V24ukAuaJfFI7nYAgwjedlBAl\\\\\\\/JfvhqsyX8MEwD8HON8ZJtRwxME\\\\\\\/KfBiSNbH0qdtn1yXLcyGptc0v8cwYRqfry54iohQPJcRAZBnX0Pq\\\\\\\/GNSvKGCyvfP5OFhfMay9JxFVfWgMWt4\\\\\\\/OwGOplWVNXJjTHJs0LSkoXdegGT6vJjFRSatx\\\\\\\/2tyKaq0Krb\\\\\\\/3AyvxPtLxQ9YiZGptrGARnGyR3JEgWdnQItiPP7GzeYHzmR1e+ooHwcL192R5\\\\\\\/ldKY1RsRcNKqAneo1e\\\\\\\/xkIdxePfrR8jvf3yInaG4AEFJDfCHp2Grq0o20iVGVeZ145R2aqjUVdKlNbAh1UVXScf8UEgWTopA7FMNFjUaYonf0T2PwH\\\\\\\/msZfbb\\\\\\\/1xUbmZcKUJmJAAVncvrKo9u9kXP9W8v2bqqWmOX57HgsmUFIw5AdyVtegwj8bSwLzeedFr+a0tD5d8JVR5zJq\\\\\\\/N5PeZZgwHsc\\\\\\\/x\\\\\\\/00paBcmLXjdjdbIuMOp7DmT0LGxO9zM6wYCp3JE8gqxTsKqDy9xImfojJbx4xtnrLbmJkuo72kVoGtc8W+Z2iWdYMT0QW\\\\\\\/mqdGo0GrGpsmORF477FJZuX+l0H5QsbL4xZ13fAaLFV9Oq\\\\\\\/gvbKOWgFA8w7CsYUHKQAUbdou6sN\\\\\\\/4p+rpCGzHWj2jY5BTAj7mccxbRb\\\\\\\/v4ifYkXoMpEdg3dB\\\\\\\/wQQblagKDXP7MjfsVmL\\\\\\\/V7dl3vAGjTiNPqj3o19rrO+uig5etpqvOQRwltJV9245P1Y7QL+\\\\\\\/rfoCzpNfmOWtdr4IyVmQLE6Agzr0t9Yk6hJbPsY7C8glEWbBNnxc2vjovrkcA9br8JEmXfpsuMribaJY21fK02Q+5Co23wLhuPND3vXySfCaek1V7oXWSqmr7iY90u8ukCPcB3R4qJ+dSZ7RVluHS73MKnh96jSrurhDm9Rha7bClb\\\\\\\/W1MdhcK3\\\\\\\/f9gslBASt8vMhVtaBGV536ha5swCRFsiD88tYP9Ue0rsVW0ONdd4OYXLK6W14y07SO6e7RcFKWv0jbqKwRfvRLRzxD0RpspNFjqebUgHx1e1JJCpJPYSK+EC1xcOAy3eTqdgIgBPNPlVxgYNbrY02iGeiNAkC\\\\\\\/J8b\\\\\\\/1PvIn9PC9Dp80oSBCC+Nkr15AukVjw+vXlRr8AKGNUsikGHj7E67D\\\\\\\/k2yfTPZRfdZ3VNsoBhLb8KMlM+WiTZg5woDSfuph8pnxSrbLgsTQZVY1LdApKRak+z1vUEFfK1fg1MaVZ9yFgsWZMagIllPXIh\\\\\\\/TPwbcM\\\\\\\/hMnIjDf5FcHjruF0fGnmaniML8fzgngAJGjRb3RjwlVbCq8AzKPHIxNfNiHZU4hKjW8aaC1I9KI6vpVnwEU7mPdGGizuKWTijxnvDI8UUpD9T7ZTeWI+cFA73Fu8E8d1Rmlx1HDry17okbuvnt0mXHcZ513lSBaoOAWFD60YBIKkykj8597TZydkXzUyXIALQBa1n2KD2TDRR7Zq864ynDgORGPSyf0jbXFs2CARuUCwt0Jcd2TRQyGr88HHTvKSySppLVfIEVK5BTwhQcvhfw5vJLGWB13ift5auR+eFL7dwk4U5ywRmdSWmYTuZMhrcTlk7muhJgZn4fnHCxmgt7F0Dn0FmHL6VB0GP4f3vL+TXAcbjxtlaF0C9aa+dCrUyi9LHW8a97y0LzSK0f7HcmONHPA3I7tXz9\\\\\\\/bO38nz0Yi04HVRx\\\\\\\/\\\\\\\/2wZyovkXRQBSxwPXUejz4OdzBldRUhC\\\\\\\/fliEltKLYj560xZuSeGgfMT4ZGPf7HdR1HseJBr4QpcYze8\\\\\\\/TcjzQDuZlvX4dyn16BCFHNADa1pJMuy\\\\\\\/v2mdSV0LF2oLLszMgzmgjATLUIQs70QNy1ujsEgM5Jv4u6Wge1KJBRLs5iiGA+Tcw+UhNv8nlxTjppZ0Yo+Gv0+9aiuBkSRwnmYEwsqchRzDH\\\\\\\/p2IrO3NjYOG14I1gmhcvh40fbxxMoHmvk5pJbbgCw8B2X+ztkK2Htuf4USKsPCTlAuedtyCdYb5gUXr6rYfOTGCxEJ3Y2ijFda7EAJ1reKi5TcdeS2gomsNFvi2pjtISQVe8bT9CjdpLroqZNysbzDd4VF6oFj6uTmAZpL8iPV9VypJJjhx4cQsUn0NAe0xNJPtzpZzs+7iJqhJ1MUfhCELYmS9q7zuWJ\\\\\\\/Y9rwqotD19dLwxvhC+8MywHW0ksQUDJejkZnJbQejyalWuDZW2m3LO+cGJPVrpNRU5RQlUcV\\\\\\\/OrTMiGVP2j8M\\\\\\\/a\\\\\\\/O3e5OmoEN+oJtxktBO+kW\\\\\\\/T3LLya7SFP1Fo9UAQkGrbEi+uGgh5PPjW7mYSR92BrVwXlZv5dGiQI20wsj3xN44sz6YT2DnYkWxwD2Dg3vzaEK\\\\\\\/nH5GvAkFBdqWn32qxYNqKQmju5tEgBl+8CWxyUUqfVdETxI+NR60zs7D8\\\\\\\/7Gp+lxY6bJk66r2wSRy2IhRUhZQ9oxLplyLViSVY5dJjhjTWvnwJiFqCrem3uqyrRMA9\\\\\\\/VHJfhNncVF0gEqWbrXg98ybyqkErbIb4sl0xeFq\\\\\\\/NWmU4wQI7DqV38f3T+xz6pOf9wFcNQr9uI2fOwYoG90pF5+Z2S6LQf2hNDHUU8p4fxsUbcXtFIHx+vk78kxvlD3Vb59g4j8y+TAU8cs5nKSW902pnEO2nj\\\\\\\/Ap5hp\\\\\\\/XN2okvjZzf2BoiEg3WNypss4HHciq1TB\\\\\\\/OgRN6WldEHVhbSqV56+wLDJCCr8Z23SLlvtoozklnZ8ALBqDFSLBV+ig30w+cczvluqrSPQJW79A0cgNX\\\\\\\/RfC63zneCzoHrk4Xyi+VMLIFENcrr34cVfOTEHX2OBNABUH+I+eAgs3n8qHNh7Cfuu1gytLAWyP1nwAiA2KWsOVC1suQEvb5aVoRxjjSafiuzs4WyBmizJSX07L1z20nTLB5KHWDnFJaIp5vRxIbVRE17neihiHtBL1UMwGU\\\\\\\/lM0DDApLXOKmYg0H5PmBwbGEX7nUiPAqqY8qVSNKYumMR\\\\\\\/gRx+2adGosDi\\\\\\\/SZENEfsGO6bVQiEw9CFr81WNP1ap3Aotx37W16N4td\\\\\\\/UFM4dFNfDhuVRLlSbJ7SZJwVrvX5CRl21EakiFyq0VSP9JooEQ3nvD9UwB2I0n+vfXyqU3JbUpyBHkbBwb\\\\\\\/XuWgN1z6uUniLAmyU2wTqiCUEm36DWkbWDpZHQPY79toAhr6bnd4evqSLKTUkITvDKsY1SZU0QHtRQU8ZmF5RJHxxuBU+ZYMXY2uWwW0NEHVK4u0kjorh8t5gO8YZPU3QCtnovUsolCeRyp672TlCvInbHpEbXhWkRH0upXw57hp706xQ+1H+nQLqQ3pNduR6LCKnkeB1hxnSIQ6rsC118pn+CXvsNhQApp6tDwlvF3Eocq9Do5WeucHiELbpWZbvggFaqvxP4NdJs1ei\\\\\\\/YR\\\\\\\/krmvaYVj18v\\\\\\\/quhUJI+VR34bmRfetD7hJ5GdO\\\\\\\/34GOH8162AtTut9XrcwCHt2qn+onpLP7vVAZX6sI0doJSn7mKDHsrYKLGyYgvbxsvatiAf4PZoifTdiCEF901iHJ4A9jzbRizj4sl8MkzN\\\\\\\/fe4+6BQx9vc+IoXttbn0coCJpzt6RzS2wMKDF1JdOFPuYDgz6Gs0pVFWOVxYguh14Y\\\\\\\/o7F\\\\\\\/Cy2OD\\\\\\\/5nYb49Y518kpkO80mc95Ho6Knamu+Cj\\\\\\\/eHn6Q0k5tZ9HoEMzjmRKGk8zi6Qr7CEn7XFTNEJx6nN4DaeWhcLO9saAvzhhkVADJQRXBoUC7mtNEs0M4nI\\\\\\\/rgiN+7Q7A5y5bRR9WXLiR3SWWLvJLuH6VNVqftEF8jDTfL6dCwqy6gpb1Uf0kSluH\\\\\\\/evpdfWq6bk3asFDAeptkyy9rzA3v4ZRriMux\\\\\\\/Ewc8biOXY+3EQ3IhaWXCmniovXG7Ylb2kaWr2Y3NRQMCqSl\\\\\\\/Gq\\\\\\\/i4k\\\\\\\/atWYjKi7jYJN0BlBgpaQmqzRFka\\\\\\\/AQIyLxQSRkyK8ZsQf+1+JKO9tqyJlQIcXvSggBvTsv\\\\\\\/+n0mq4eiCaNiSmcXnbHFqpEk5Pl\\\\\\\/FWRqH+iQ6+\\\\\\\/eWEk6mlXJ45iSnqLcvrS7KKi8xe2s830G\\\\\\\/JV814fBAbNdDW0pzZJyXspp5+A\\\\\\\/0ddHnzeAW6S\\\\\\\/dDYslsqdJs00l0EyauBUxSmQiYfnxJnXyC+AwcgvfeygXmNe8G2t8LPMczg9+cpvei4ONOSq+0Hr3mIm25+40KMzgguqMeaHZVaoimUYRJDRvKg8J9q5M8zOLr7TjIKZ27epB+qXjxUmRNu8vy\\\\\\\/LAN+f4i8tpzRGJQUuZQM4ipv4NaZ1QPEjR6iaurnTmoXrHKDYMaQkpKKHKmQ4FK58bEDENWto1lLobLxRaeMDgMvPmNoNjVX1LyVGkfQl88sghpr6uMrcjt5UHTtdWKpO07ZxSApZXpMIDZiUBRVtmcDHgrEo4zrCKcMuQl8IkInW+Ecf9sMBewdjPf\\\\\\\/GRG\\\\\\\/OFxROVvT9PovqDqOLiB51HcofGs4yGNMbKebUGiGEMFAunCEQWkySk3MnRKkl1aRHtuQJVl\\\\\\\/s5S3vUqaW3mKzqKYpxk70LwsH08Pf4N6\\\\\\\/gDCubTGH6Qk35qv1TgutqQP4J0cyHi5TXvnj7p1BglzUopGIRNQHZSK9zByQ9qmpZ2bpWnVfxb2GzjgXwzp\\\\\\\/WER1PoBjPVoAdCaG30TGuHvCFWtY8SAnmQuvCBaCJ3mgtYeLUkzNKNtj9vp9Q1e4wb0WSRcIuYlyosb6zZpi4nKT\\\\\\\/ATYKfiM14P\\\\\\\/AoQ7T4DqBJNLICJ4vxknyBi6uendH\\\\\\\/4hgOwcruij83zhx9tRUAE3cfJUA\\\\\\\/oy+tbOvfV31SA8xon3e4MH7ydaCAjuKpoIwsvyn4KBIBWyey+HFN3GyFIYTfTBTXUEpLqJnpydiXEC8ZYZZITD5XcVqwFevjhI7JboOnZ+wz49WCOweAeYr5hPmDLapjMGlFyw\\\\\\\/\\\\\\\/BookdFOrqap9jCszc\\\\\\\/WPX3bXtfcxghYDySZwxmTFr3tPQkPrukt9Mu8g4kAsufOMuDiXNd6bU0Gxzx8tx9OdHQMcgJeEkgv0BIj\\\\\\\/GRmEsUq9sbiQqQ+CLUPxtEj2EqLpsMWzVGkU\\\\\\\/4OZ1FZ7ry5bF06Z0Ef2eiUAmybBN3QcOWBsKv9sW0nshwunClZf4jUR3S3i+5ziP94dfArljzZ4noPwMLUQQzxJixdMb6IsVRCk4HQkKxHHvPt9slLU1RSFKuOr18I4b5u+mAXLsdAPn7ml0Xs4d3i6Zw9431Es4QGEL9FSy298IGk7dNEq60bPKn5XdmfD3UE0rL6tpEgQ+TM5u5SytJMgBsYxfOjlMprtyTw0DbP9YUYrwirW7DeT4W4lnF4wbVIH8Odz0WPX6PWC+kHs8dRCBWWIvMraMZxIVJdqQZsk90rSPFyQDt4wqeBj1JevhnQpwBn\\\\\\\/eZxYVcpNjJMKUYKP832fgUJlTm7ysMRtkhZ52hp2p25wnZSqfkjcMW+lugQKQu32eYBaLAHXUQx\\\\\\\/3j5czok0RLohPUgnfqjd0ZNoN3b0rrlOTi2ezqiLTMnVyZT4Z\\\\\\\/YTZX9YFiD\\\\\\\/nm22P97VCBXWK4Riral5xz\\\\\\\/fMA0PpIYqk8SN8T+H7dVsHxUK1DosMAxnWWs7ctTXg1odPcWJLyzbkRjZqX92\\\\\\\/BymbWYwzJypephHIqwbAg6tEBP2jAD+KzGCxllQNcQinCN49fZlrfKqIvVMsyVvNgp64heCIsSmIOrASdiC6NrmPJZON7hwVgXPQl8s7IsHXm9xnZZrMnLTiP+XeNcyNY0pWnlz2Q3GtGT3quuRUJnmk8g\\\\\\\/DiVbjC+dnuWHNOsdX0f8AYBc3g3yeX54TS3Y\\\\\\\/1N0bkTk2RzS4pkHpMlUwCNEtWSwWgCaqMTq73bgl2vqK\\\\\\\/6wIhgzoBYMZBQ8zMOBH0ExKQUJKogeXQXZUscy0j\\\\\\\/XknvZjiTpNNcbPFIneownCF68creApXm8JiPbhvKXnKPuWY1deDdkJAQksuQp0dUyjTpgqxrGWWWrPkklX891gqMz3WTIWcfueXPeXKmJAiJA\\\\\\\/Sz7aX2iEIppvjpoPUsKdVxwWpkwG+sY3NUEBrMiLLdMEIZVJQY72QhpJ+F3VgqiOOTiBTnq2Gzdu9W21cpZOrvCG4Wt4FD1aCdsfKMNdRlyNuMjLaqYbXueQdSynbGGmDYbWugF\\\\\\\/TED7EUKldngcFmuUYmSD9dJWXKGblVcKyXt6HB361BBq\\\\\\\/\\\\\\\/F0LZRNAIHjpb5tUAoEy7dsSrD9AKeksZ\\\\\\\/B8Shrg6nR9CaNSW+cBWh4DcekQUCiSIrx6pEGVNYA95g8PGlNn94PTI8G5nOnaf4T6yGMg+VjzOCNayGG2cLeXfDXz\\\\\\\/Bw9IVOi+gonVkWDOA26mk\\\\\\\/HqifSr6StNexmU2I3G9RRqhOcbPdNBDqt+yY2Q7ZoyERLT2nCmxus\\\\\\\/r0LvArAFjpv9SNivi6uYqVl3HQMMyU8pJP6PBpYYQfy1a6XldTdEwn4\\\\\\\/F1pFe7HhA8Nz7kDPD9aSrfi\\\\\\\/W5TVpEkW5jvA8rkxsZyX12QQpGaWY+GipXAwODM6bn7psKw\\\\\\\/hy9RY0dK4X2ntfo93kjngPvCH5yrhR8kc7BMOHiJIwEhG5O5PnfjuPKUBBFo8Nya3c3Ww78OlFXGMQxpmOxJggD9BHb4LnpsQpq\\\\\\\/MGchIC2ADEDmPFBKERo4oFIBze0abeI\\\\\\\/YvWmIXh4PL\\\\\\\/CaQLB9wq9MwQu2fgqu\\\\\\\/w70XteXC63H9+f+VlWL56bkfCqOiEWOYz4NJ5WaZ4ASZ6oVTItpEueTus2wi96ov+EuUUSfB64GwFP1B7Q9i5SVZbTlsgsvSEzJfpNdUnE5nikU9tXxQ\\\\\\\/2yClG7u9Y+8JfpYRUkIL88ixG6TQ1iKcQ2nSnOztZj\\\\\\\/kP+E6pLmK0ej3t+e6xteZJiQC2BDy0dLZvuWdcxagmtP82xJD001YQDe6GlC1m762s5lVsuwuQX+ufvzN3HV5gcYPDGs\\\\\\\/lp+x5FueKDjS3Jm2KI6bLYaQydyN6l9qzWBbJie3f4SjDyUmnrQI\\\\\\\/w83MrQ\\\\\\\/QTwx0rqJGaNcvwPDhDtZ37b+Zg4JXYiPUiWg7UEpcujWOVvcp6pLjp3+RSAVTXLU1VIERLuWLVGJF1KkKv4dPd\\\\\\\/rZPuSysHUMsdhu+jS4prhZQ85ClOYmSvy98wEnNjonCaIfaOY3QXwJjSs8WTqzznynoD7Grbn6+VgYivFtKcUTtlveufcmDkb4lg38biTlcXNPkTf1qawprD7RvrmBrr2iIeKNvGxqNcXP6xfTriac+eFPTrPOZ5AC7I+PlNYnhgkXALgymQS3jxS7olvK2qw1Goy1ijFoFfEXBEcQsuQmosp8ceKbn2+Vdm1JPd+eWYhAyv8Y2XmZdxlPWuG3\\\\\\\/Ocswe9rHpK3mDv2+vTv5GY4MPC6yCWYJJ+0XrcxirO260BHVOIEdefjPPy763Ye+20JUrd6Dc\\\\\\\/808dP1gy9VMLwgg0x0uOBPNqOGJzGdydt+BJaJOalw\\\\\\\/60E6fRG0zJQVY3bInPMRd05ep6Uns0bpOJFH8z8TtMkLA4VfvLAFdrc0wHz1YvWaR4NWtjICG3B6eJo65Jfm2470AFUUzDeAwKzL1BsP3+Oil0pD+d+Xlt2Jy1c3+YlO7cQ+ACmXadf62IzcrWlpoW1nGZVBEe6ETTYS0DD1Dvwskg2vujlUb\\\\\\\/WNnk7XkYD\\\\\\\/9NtAjOPmezuc+6QKY1Zx3OEFrHG\\\\\\\/EGw9vIFMLhoNrNP9iIVGvSNME+YT35Fd9RIrtE1V5dBq+IjU7tFF6RclsLtLegjxecetGx1LtL2jaQiPm88+Pf49J9\\\\\\\/6BKGHncgPwC5Fs+\\\\\\\/la8uUK4OBU\\\\\\\/TmSJy5F642B\\\\\\\/3kWyuC4VImlhPijVD0zY5BjD8miaEnTYAOWnl1W65VqtEzMDyV5Z2NmiVECgAUS276Nu7wmYZcCKmam0mYCaEX6p\\\\\\\/0dmE8k6Xv7dCQXkdhZk69er4iUrgUXQe1OpDr3eLkQhE3Qpa4y7eisa1X\\\\\\\/Jtc69UF2lr3VwoXmFdeRB4KmJZwLh6Vpa8F3I91FiJEUFViLaFq32Q7rRQKT\\\\\\\/L7oe71SSq21FxSuPwNCcesE9apS+cHRkgaUb8f838JiaDk2kVjC\\\\\\\/J\\\\\\\/OjWr4V0o4Pn1JDhGCPscIMTBl5TSKnnGUCj\\\\\\\/jFwGeinZwsZkNfSZCs+NXcz32cglP0HOJeHdpTKZf04hOhQSlPHaIK9eG04E21XzL+F9gYC8pfvKCHWh0s3olwML\\\\\\\/Rl9clq+ab8+feAMKOh4gRTwgbUIDOZPzCMyPgF01uxG\\\\\\\/ljuIqMxbkQ\\\\\\\/ur1Uj4o5NF\\\\\\\/65xNGH0ZSTZjJZdzvx3N+a\\\\\\\/iQQ0BZ+55Mg3GgsW0zFbIA1QocXzXyAlX7UM4z6KiA4v\\\\\\\/dT5gTMzPo4E7FWiejGVxCBMGVJ\\\\\\\/R\\\\\\\/nhQWGEw9mdE9Wme3+Mw7yytxtCj+c+orzJHT8\\\\\\\/6\\\\\\\/Urxf4whJml6nbDOwNUewzGMH0kyTLY3\\\\\\\/XSiawgbUG\\\\\\\/R32jj1ygIp6f0fMoS5Laag89kU+xPXwNaBeEw0tyuCQvJtip58tAk9zo33IpDMao1GJ5OOqyVlODQrpKwMQtZzycl8MjTzYozReZiv8jxU+8WEFhcY+Ccc5Nhkd6PGfljK6it9wt3zEBRpg5Vn2n2B8\\\\\\\/VRN5KSgLl9PuwEGMRJ6V\\\\\\\/cONDSZ9+7d9AMOVPUCSn9sUjAFgi+ZGCZFMGrO\\\\\\\/FLn7uVXkilbfC+8vd+ZnIuauJjf3FNmBW9nFQ8p86NJBH5uYSmhAWAyHzSGxb2bd4S4goaciJ9vI44pI5UnmoIxlJ9g1IuiCIZ833yKtlwl9UlYKrP0vwIg4cWUFy1rbyUTEAdKKBOTYVjHjcSjkE\\\\\\\/KnaMzfmUazqQy4DwRUdDO3mzWlgX0vIl8XFH9hFK1iBEcpoxE9QHVNilLDj2lMhAxxCyPNF2jtt4CZAp8pSeCuOR1ZzTFX0mFAG02RGaHbR6yuiTqbpysWurEbhMZurvCBt4Sc6Gxd+6Nx2Q51\\\\\\\/BSX0dRZQcRI197fVFjeFWCOXASzw4608Lol+ITQB3riWF9Gt9RFksMQZEmBIjcReyDbKXUQrNrS2cbdyrGwS+pb5y3BqK6v1sRLXR1XMoI+AnDZTcrLm1H8v2ZeEgHfvTrQgZ3H0NI+aINnDCLYquHGdYB1ivT2v0AfZLUntCgvzUcoIWACoI9v+aB\\\\\\\/znhb3qj3gRRFoU7N4BUP2hbk5YxSI91VfV8BIMGrzxff9\\\\\\\/RauHWz8ceK4v17ZVzf+Df\\\\\\\/J0Wu2BYMxsKpUnn6GVMC3rCw4pLkdgUqd1xQD9BWx08cQ7phINu5S2IOKFAuDN0sXpswS9S7td2YJcOxdrB+PyXf7iFUwIEvvJuXAwQpjmMz2H2mmQJ2EJjJkayqMRPcYuArfSGrpokMDxNlwFCSULq8JtV\\\\\\\/mKQbrRos\\\\\\\/M84J5962JmiFiI2rII4VwiWifzw2OYHfgd+kuhevlhYcJ8SRPFkvVBNdAyY9iSBfHNwqzCRRGCBDmXLDXo+nLOypNyoZimnhi0lj10W6JEmbtTtUs8A\\\\\\\/Q0mJtX7eY+Twwovz\\\\\\\/danrWeSOoZKyuu5KuQzDc11ew6wULUfW3puMF7zMW20+asnJWtIPlGr6wiOvgpELkYrVI23avXa2zTxmh10XDtv9fvFp6KZl+AW48U6enisfbLFgtpxb9sEhGRvFvSDpGkvod4w5n4IwRrW4MV8MPTGWIKEQVTymO5Q\\\\\\\/YLTfHDbWm4miugh1qwUtuKe9iEQiFtFFGb7WWOZuwuBVx9uEoLIDYni2v14Uo2t3SeJZCmKv5bikaM0JSCgHA6gWyMCRW5p0kBWyTQZOScquXnQlsYzoxMb29N38UYuon8YTFV1Esw\\\\\\\/nCZaXGszoyBBFTkstTk+QCHRvYQO0eFKKMXwD2\\\\\\\/p4xePtMNCDnp9ucSnaqRduKp7GCs9294n\\\\\\\/aXkMpgQCaS9t56y0A8F4fRpzGDFCqo0xDaS+QqYxgXchAbP+5IUWfwh8UBzMLldhHU1hxadZki4jME5psQ3qnGDnDoowFNbbtTqeGXuEyckrQ2E130R4YkMZxU+gINMxd5oWYZi7wYSeaIvKuLKG+\\\\\\\/qa3oQXHP7b9RmZQpnL+8znxbBFlGqa+ENQdIsYU4BW6U1yic+G1nkq+ty4+yAU4\\\\\\\/OR2ZFciCFQffKVUF91QLecYt9pgB1BKQnZ1VOENuSGjxE15gQqqk4wFb9qUVyH48pCw5sB\\\\\\\/jwnBZ0I3FjKdU8UYEeHvFNg\\\\\\\/QP43Em8W0eOm1l3KGJQlabZLtQ0oOUeAzlPjwVhW13Zr2b4yjGW7MfpJFqIiAoxlxuggdIuOOYMcjkdwXQ7NBshFwP77PLJS+Drh2G2CHKfo60Unrna6eS0SKL5Ya5zD4snOVIi0mE5qNQiElEbrf6CDcyXD3hwYxkQBo\\\\\\\/xb98zGtG6PgHfuT5dhkpkOFKsQDGRF8s+gX5M32F+dVFfcF\\\\\\\/\\\\\\\/p1XypEtvmqkinu6THx6jBG+TSYRvvOL51oUUY3mU2exQabDVIwIksz6xuYhnVvCuLjzw4F0dQlTPu\\\\\\\/VcRDP77d+sC8Uv4kD3CfdUnm4t+iOjaM61W3kPtutnRULGLt\\\\\\\/Qj2LX7VBDixwH9T8sOyudjkYnGEp17wYRMPxDyjSt\\\\\\\/9lW6+taO41sHEIzR6rzlJfhAb0T5ZuR3FxjfYPM9Ve8rFW6TeHhOBMevf37n22eQwyqiliLl\\\\\\\/wkYKLCEtEE\\\\\\\/2fmrmBHa+lmO150X+3XAol4Hq9qvLlx80rtIToJxRptIN5V29YUa+VCSNTe7sXvokw+HbfrVX6P83bZp932cHQTREhhqkr3ba7ch5uoiKO99A\\\\\\\/XSpYUt9LtEYguZDwJtFvzOHk75SbOTNrXMKC7zIXgeSv6zRjXLwkseY4cb64Qkcz\\\\\\\/SCndiMR83eyKPYtPhy63wXtrcviAZvVVFfnie9aZNNFP+lW3nGweQxStbuxfsTq0\\\\\\\/NMar4MgP7ZYHYRj3Hqx2wf3xJ9fetSzEBwckOsFyfhNJsowOzi8EwwRp0MMNGTgDG14WF1mZN+KQjfZQ4i7Xnp\\\\\\\/+xYhCxJBoDfH88T4G+4NLKGFFxPJbnuVxP\\\\\\\/ZBxm4eOK3de6+P9e6F6+SFv9V7BOpUEE7j05TZk4UDeSUTQYbfLJ4RgxHNm7RxuAXPJmhQ+eZdUjJU6KEnwKzyK2+I6LnQKa4kQEmdoWvHNJYqSjw5FWg7PEer6dZW6\\\\\\\/fYfZktAdbkmN205yWNo0xUg\\\\\\\/KPJyqA9OF7RUGKj\\\\\\\/xtmZrSFw+QqpWmTJ0EZdlbn0soLfKjgK7SFGljM6DEkXejVbHIYxVaaMel\\\\\\\/hRt0QOrjh++mhSMlx5QC7LTMO9VMp58nKlf\\\\\\\/nzyxlsMsybj74uxlVmPIrO\\\\\\\/+J67P9enR2\\\\\\\/n8pH0aCGeiWU2GTT4zaav9IGbrOujOY6+6QvXwJxiiwpLP40HYdVJEKlnzqPxSqd51Xgtvh9VmYlI+gzCs\\\\\\\/MLfNf7+ZGmsVia07w2c+I3CavZ9ARNaAqROvPhjVtVudTMHrpmJD8TRx2R7e\\\\\\\/3OK97hrt25lOd4sCfKOdJeqmjsns7Zv+YbkoNc\\\\\\\/BvUQ5gxbgkRx4FCmLa\\\\\\\/jtU\\\\\\\/CPcKMxDHAGlqHo7zgvthPAD3dnq9tmSxEkmr6hEIuOS5axexFOiwebA9SGzumv\\\\\\\/uXf\\\\\\\/0ht012iCy3oSvZl0lb6mZt307EGmynXEtqVQyMsF8VMG071fqLAzqglfb\\\\\\\/y17p6blIRFN\\\\\\\/ADuMn7U6tq4+yDyVBv9YGjPQDBI+LRyjjjqab7AoQc\\\\\\\/M7X31EaIDm8gC7p6H6AH+cFN0J2XqqaaqyBr6aC7MIFEezLTbcuqoMXnDQu9g7i684KR84DGZTgS3TjusXLgQ5sRr67g0as\\\\\\\/79+MARFVRll06ZfuTnZV9TkXQssw3MkVVnMfT3Sh7x2gdMLcwOCxVESu+oQVsM1CvpiGwGZ9rgF\\\\\\\/T1PsiZSvGxo3U1Yfqalv2SX\\\\\\\/Q4lHX21GgnVSQVFf+Mn2ntSYmSAJ+\\\\\\\/l7bt8lUHbo\\\\\\\/okT4ZysraXrO0Wu\\\\\\\/J3BpEtyIGjmyRbaJAA0u1k1iNF0572ePpkn+tQC6qybROu7rQ\\\\\\\/OwFshbG4h5j6nedsJCor7tM21dZZl+3k7CVYn26lpFjKVn2CTKxQQ1TOY0xF8+bA4bEwkmGP0zXq8HurVu0XJRIMDFSm7bsquODe7RrbgclP6Fe4VP+jQIW5RO+b4YYkySpocZAJC3o2nhRHCAmW2Juucjf\\\\\\\/nlQUYQwdi0LO80GoLl+nj6JoWxqpQEpvMfEOWrWXPvexzDadZLDo+us7n4PZG9400zQKaAhWYEq2s4F2TjbcWm2wR8mE1Q\\\\\\\/PdPswblM+n+Ug3Ls5E\\\\\\\/If7\\\\\\\/nbz4e\\\\\\\/jUNagqXOcVjZwyrLjIs74vXJC5RJ4I3wusxujOneIQBA126JZJ6\\\\\\\/DHVuP3Jh1QCF189runm66HbeGxrs9FBMtKjKUTf8ZiWt8JViVAib7E8ar1KQ5cFkbpluqjx2JfDPE8VKz\\\\\\\/LKhxYW0mNHC8FYyHmQVdBmtGtFNIvLrEKfQjGhaKfIbfS3keIDEGlAFewBRc6D+w7X99TVyfLTvmfV0PSm6OvkmaUOfNnx86OnZ8Y2X42vQ4hrNGbLoLjn+mlmPzSxeYTjPhjFUcF6oos1We1lrcz84fD1kCfmGcu\\\\\\\/HRbr8SyUb6nzdUAQt9CP9D\\\\\\\/Nlc20d37pZMUMwNiWbyJXDiAiTX9TLpdtjxTUUDHNsqSJWTEth3iokDtzfN9+SL0DdFAidi\\\\\\\/t0BeDl6lPNapCbQ67vMrykSZg9d5XZrpaxceiYMnyUDzx8j9XcYZSGIyJ9nDWMa0QHrgIMi1IW7QwaXo6G8Bmj0ylgxB672P3byWCJwpSklX+tU\\\\\\\/TiZHvmHp4V5gang3zO8w6M3ZIxkJdfLYYvC0DB37LobiNbvos24BRdEAqE7cpQ4Nam3f7qjk0tQfqX57vwV6TcjL8QDVVwvI3rs6AY6tfobraiB2CZIEryOXJ0Y3iVV7S8wOTM9Z+tnqRi9NK1qQLO7wu2O3W1XDGnniN7WcRJ1fGsNTnHFG1vMEA1Lhzzs5yFNYJffX29LIUVJt79j0LtvYBk8R8daixFvuHMbKXrmF1AAsl7ft+Wim3bXI8dFNM2LMWmNywW98+E3XqJGYsOf+RPHpjb80Td0tOMS+tDWKwsOTGtGK9Qt7vhxj45PMQcD\\\\\\\/wFNGYYnAf+tc8\\\\\\\/Rfl1504r4UosUaB8q84q+MRdmVSmyp+ZETfqHazcUaBeEg4PK4uMubviViP7Ypqs5ZSA0KnHwy6IF42Ea+S2ludEXEvapZyh0WjX4www1Q9X6pniuptQSoy9RiqkijBH9wH1EPQhiXVT2nxoCzzkVXASIR+0AbZUvutv7KTe4c0LHYHmMzh177JX+18EeRwBdj+SfOsj6Pbbf3aelWP8nHBfx7JvN+XZOHFdj4xV0G8o6pFv9sCd1D5nFOoayTRpgUntuwDpbHAf9gUqZOprluylCUzevYrTdjO6\\\\\\\/bbYnfAGh303kKrh57YncC2OJnivPyHLoxW0kX5bn5lPefmZ9hOlzcKBvj7wMVIWEwQamh1YWPkMS4XsuFBku\\\\\\\/1gJsKIYF52SfZC4\\\\\\\/GFTuSxa1RsRybZ+6B4b5tu+Q5iLZyegnvwQUQRmiRr7hlt8YulsevI65HQjjyPKSnMbppxc4T\\\\\\\/5veuKI7VKKE6pSImQf0yNbCHfhlgUyBkoHGOBh9C7wwncNiZly2qBUZS+De2QgsDTgnvHhENZj+n6bkfvTP+rLxaBjmLEY5aFf+cJCaHlW+rmbY14Ex47DdgKwVS\\\\\\\/wYhqNYkTZzI3nWf2u2HiTKBfBTZi0v4Bo6Wn4HW+1SSEFcL47Tjv4zX7xKaxnO1LAi0u1HpU3DyoO0OZ9A1wnSEOj1+pnvvQkluFb+vuCLCWGTVRzj\\\\\\\/\\\\\\\/oIEYdEaBnHRnfiMfWwHzfDeWWCOvZSYgnl3ady2xn+k7QSNfwSR2RpD7LciRftIS+4PP5CFE2t3vIPDC6Y9XmO80QAgqRpwLbJBnlTqmb92yb0qZJO3p3YxIBB5WkZJhe7okZRTHbCeNU2U352xpvv6dVCDHLZEGxk\\\\\\\/8r\\\\\\\/glDhbZ2ZcV23doduGBkI1\\\\\\\/IGIYdPrm0Tg8e8KqqlYxmydm3deEC89D7+2DxJE\\\\\\\/yMc8t7efBGkzZDUmp6M373\\\\\\\/RNj2HAZNiVWryIXtkh+sFyjFBgP93BKyew1pMZjhUxv0fw05BYF40T8UuiT24\\\\\\\/rQihF7pIn\\\\\\\/HxqNiXOrLyENHp04TIHZA2Qv3VC0ObLBR1bUgLgfIqVASeFcc2ICUBXOL2PcjNQKi3mCO9XVbcA6Gfh5sMi9Bqxo18NYfjgjsUGiGmJBHg1TcN7E0PFhMRE2j9aYckZ3lC+TpQ\\\\\\\/vkTJ+rFnzYsDCz1DelUReElLNPtAZ+IHRdM6gM7lAVmb2U92MZEthT3dFCcutzUoZcSyoTQIfBku\\\\\\\/rhH\\\\\\\/YY9y1ir\\\\\\\/XCd5IdxXOYPkItzHogbTUWh1uOC7vNeftvjD+m\\\\\\\/reO2rUZDRe\\\\\\\/\\\\\\\/daFGdsI+QaFv+aaVqD\\\\\\\/Eu1r7mHyKeJJCSB\\\\\\\/elNcNfQko+OR1OVRUfWjI\\\\\\\/Pvn\\\\\\\/spN5BwTECZsl7+FeA3Ix5XTSKz7axqQngi5f\\\\\\\/\\\\\\\/+3OuxmtNbhySbuMW57fRfgpD1UTLKuzb9\\\\\\\/5B6\\\\\\\/6ofdhWuQbI+dxTYcRq7E0RWjrIGUQYH7jlylxp+IfP7hv8TiUtG0dSKqmGV34jHS2ivfCEDvXy+WEEMSv\\\\\\\/Z7XzMA+Gg7NXYtV5eZdJ0Ln\\\\\\\/TENwiMa1OJbg302yL5lqcm3SunAAtWMD\\\\\\\/k5g\\\\\\\/1eMy2tFLFvCEIIbqcNZ1GQ6dNvrkIqmN0jqRnUd9CLncFBKXiaqvZCToKUXH3L\\\\\\\/l5e0TorF1WBSAJFXBq5+GlXt3WoAD4sjeYWEn33m9jnnkSwdgSCen5MlrWfBfjN1eehBinsmrTuzu2v1pcstd2UGSYwdnrgaCQhu+PcTQkyojdgvrYYgCF6vWAL0lA3iS4LOhnUY5nw4wU90ftO6NMHSDpjg0YBaH46rBoRn7yT5KCbuqZ1tMvZHP8L4pcWi09hM7LX45XvEAtv4dM0M5Y1R4ycnbmcZUc19A3\\\\\\\/PPLEYXiwoDydkriWVkBJRALcknO7idT1dRKCzrQS6PLtw1XSFFdrofbzwei8SRM04BsTbVTYvDP5tFtPrzyjyUkL3WfyrOrCg5wzLfG+lcCaeKLGkSM7iaj2ABWx4je6gRTHF\\\\\\\/UoK9h7EtvP+VcUHM1FAypB8VrHhBBDzTYUFM8mA+WgBTnNXZJZsJoFJIHSCmXD7yZ3X1MvJ+MqwgrjKZfkr8SAmvMcQqvhnOPmRqo+zRw0uOQxlItBCYrgfgDzh93x7RUfsrpriCDFsVnQUBWmlY4PZViZWiGgq5e4ojk2qgcfHogtFiDc3AvDao8UWkHFDLteYj0bDjoErlBDoxgm1gJCeEQJH5nYT4JNJKiCFWtHyrT5Qc1exRaLRo+oYVzaazTaMtxcRvPXNaGXcxZUt3wT2mmt7TAhtwTIlwev0rIL5GPbOu8EzTlSySdSSnOT2mQPtzj3h+u1SFvB2LxxvDwWWjaYv81KqFH3ZVIEBDun+LfcXQr+lagt9+Y8FCplSbeW\\\\\\\/+CpvVAQ2YnlKFUi4AL3qxuDPbLU2\\\\\\\/JHOs4anzP2RzQwDPPY+WWw3YjYfuIq0AA1uihOxgAdaQWG5rrzpeYVRHSZY60DHQ0LQB6E+4oKY7eGHgsLBSgF0AeQfRfO5fI9x39Wm3ckQbmSQ6qCeArRP9Yfu2QjQH9mWmqfEcsBxRZkq5dXLv3YKgZYKtzg4F8rY3Vhx6i6syhRDstc5M6rFW4\\\\\\\/ntuzxyKzVx\\\\\\\/z7ay98Ur9RKzZ\\\\\\\/jWNWZkAVQJ2rBtCTbNfYGHwZp9Sap58LalVURBBM\\\\\\\/6ZUAXiN5wVZXBhzrGAtGzbx6dVXrL2XObPKCN0xw+4VW91hruqgJPt8wSZVIvxvKSx6FoKkXRmODZ\\\\\\\/QbaLVw91jtj7IN3Wg8hkpWquLb3kzxiU6rXNcP2IpfhCM9EL23F0B4BFdxW7TfW2FXi0iez7xVAWM2asp7851XGwnw1cn9rMwcmua3kYSRpC5sFBs5RaAZKEPzFy5EuNFx15yUT6lBuD5A06A\\\\\\\/cOBjYo0tXJuy4wNFuNxqm+QrhSxA5596dcq6K9v7LphjUUjLZS2rVyGUvN\\\\\\\/ffRHnrE2MiFsmAXdV46XG0RHgRjwoNSmkKaFBIfrQeoeLCxowA74Jl\\\\\\\/ve335MYVtzkgajuwL8qNTbl0H16ysW40KjLFQUjZAu3YpSHb+ijgFXtTtREwWZ\\\\\\\/kTqaFUSAYaSIGBtSMbAc32wn4bm8r6r7VkpCM6EXhvlRyz\\\\\\\/Ov13oKgzZI6zoHCokvPA+huYa0WJ9mun3hTpe+y6o0nnrpu9GoQAcewPa1b+ot\\\\\\\/+IBa\\\\\\\/FcQ\\\\\\\/txefW0WyRmnxNJpeyI2fCoeVeI8GXdzTvcV93tmB5q988LaOsLOuzi2lWzXzLT2zrdAZ6HgEr17UltRstUFetTYu2YgwkNfamiVB0cSCuRnyYgMzY3EzcZWErnFfEY4xu+pqsGyKZJC\\\\\\\/LW\\\\\\\/kWBybaKcvhdeZnSRYid0zHT3hm\\\\\\\/lTeHSICF8LhCt29+VYKDOBDhe3fUsfmAO32i0X5nEt2m1wz8IJ4Jy4w1nGgCX+9U+p7TAjAlhjtNjXmKpEvj5hr4zYVzq1HPhjOYhnOelHoI+O2lREKQXHbOQFF5O8iQSndquCgo\\\\\\\/SppPCE+LlQwKhOTwCKqsv8txLUVgRuMtrxDpDsu98uNv6Ih8J49k5IeqfZ\\\\\\\/WmDUhzOtXtSrvP1Ir0va0QelAjNKaqADkvz6RMeqOqCOo8tR6HbPE7\\\\\\\/AfLrU5CFYhXB\\\\\\\/cBfoHYv5pLMj3bi0mhe7K7osH\\\\\\\/s05iJTG\\\\\\\/uQnIpbM1jx1ETyWbdGkuE0EhXwakpvLRdlCm8gbeLwVFfJ\\\\\\\/vBYj7FF9CxZCFLlgDR+uEZ8jDiq8zCI6EdMOnFy+CKRoqIC78RDvxbBiDR6w\\\\\\\/Xnp\\\\\\\/7gV2BB6AQ8cixw+OzwqdLzWqhecrIo4GI7EfbHtYFC2lXiEU8w7wBBsPeBLWPEiTmVBZEkKHXznhE9Jtu3v+wogf1pMlCZ0z+08bHIzspBXWCkL7BbIKq6nZQRWbGOEeFLKgcQ1A95wEjksi0JgCtwUgcZ6xAI8Po+IjCCiaq5uWjyM0nWAMht9LgNsl2QK\\\\\\\/GdsbE5IDjKyWM0XnxWIO7v5GHkM236fBYmvQszkRZpMw\\\\\\\/O9kTXYhAQnTNMvZunYBw9fSi7b\\\\\\\/Kd8j2i7QTDf1ULkvn08Uap79lmSZyyTOvFqMmwFth\\\\\\\/hgtUFAoMFMUCaq2jYppEewAblCJDCvyDC2amsd8+j84xfsIBeEerpvirQacvzhVRvabo2x6dGsHfXWxaYiPHFrp5QhcjabzoFZ0TjIahfw2Q2xB0TDofEz2YY3P0TI9X\\\\\\\/AjUlqJsoJn39lTI\\\\\\\/9aoWkQdpdKIOgReKJw7wXgD\\\\\\\/uvSdO0lkgNSxMJyTUabMJYAvQLStas3pZYIaabKtjqryXDWlonDWspwtnYjkxGbiLxnf8a9LEBOiHjeDuGidvhvulsYysuKVcLV43j0sbk804uXsKkCEsibYLRg0eHxW+dniU53GOtOee83qNbOag13xHnm8uD7KRREgDdz1wtkV\\\\\\\/bIGAYRwKbrwnTYESQ8tCQgr34JfCjFSHIuOaSh2GatazXMWp4nC4HzVfOyUUWNYyJoLwH+jl65qr+MpVM64kc3Ed6\\\\\\\/6SyMIfhl+AJNt1cf7gFibRhwavkibaKzHIWcPWf61rub4QWKDYQOJuTjuuyplqkbTRMB7NHAu3OVUK4dyy6FHg2kLMyB8CBB8r+YRybjbXY\\\\\\\/UyEdMLv9ERtvvzUljRk0uu0Q0yoey5FBA\\\\\\\/+nDMfUr6JOxMFpXwDW6A2V3ISuZNygO8QAVRht4Hn44EbPj1gfE+VCPYJiSr8OTPGI4CDl\\\\\\\/vRhyryjfS+eF9Fea145i1\\\\\\\/mRhUnueDt73zoPz9WRnORFH\\\\\\\/hSh0Ux5smj3l+TueLAIuq\\\\\\\/RnylQN0q5WEbdaam0TRUvS5KvTn4tJLz5WnL8qtU8CEz7ci570VMtsUC2Ll+6Qdi0+ScqyDmmyeErVFIFD5btaavzgAAaQjovKyjwmrlXD9xXOPPQyGbKLRzgNEG1bWBzl6SERBWfxQorRv3WbviCRPzvtnPNVmiOsnW6OU3EHB7w0ZNV6iuLqHEZw6grl5rHCW4XCfpo6ZPdTBIPwxtF7MisapEjiKC1NdA0vBOiLWAHj588DBrWuXfuSIUVYgWS5sXCMn0he1anYRFTXBeJVCOQMmeByNRVms+34v1OV8yFuqS3h5rgYPoQSywFBrGZx1C9HWmYa1h3\\\\\\\/lab1ISK4gTDGeGEwxK2CX71aVwpyMdMEnLrM1VirHljCFU3Yz1lDHJS9oZFriFVaJcUv8ASLLBpAAmLF4RB1H1l3d8NrwPIE6\\\\\\\/7F+MdVi7tOBGEHtA46Ev0lQZPBhVsiddWJnK\\\\\\\/Xjzhto0aiQxzw20AdgoAS3HbpEuLVo\\\\\\\/SSqqNWeuK3D58HQwey8UmZTIybwA\\\\\\\/KfnYx9P7it0MBCQu6L4EIr\\\\\\\/hkX9fZEJbfCw3sO\\\\\\\/JqRCILDKjfEq6jJ4BCc\\\\\\\/SjFTXXoFzLld7fNKZwtTsuRZ8b\\\\\\\/98st1v9nmMlL9ImNxxHL8xpFXx07ATxhbUWLG3rJr6RBytvA7dCnEOVpFixLnWYlCRWC\\\\\\\/Q9lxlL72Li6x4rTJD6ac1mWbhDIbyhqvwI41MADSiJvlQu4+RDiz4FSyANrEVaU6zP4T6rgaJtBJn9b4yFqZY+Rr3fXH9len3NLJPkkQo+RnEGMQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022M+vXhquPHNSMpMkS9CYZpg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002253f8d9c854216df5f0b812a78e0a442de4d5a7d444dae0fc04cfe5f59f7f3c48\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NFJMd7wvX4ElgTh5K1atkT1HBXhpzrakmYjE6XCkM4aBmt1hodzLOpOT3EfrfGfXot67gS42TRHgt4ZLT5ArD1Zfc+kaGcujtM+BRNTXErA20kqZW3APlyl5j8P5wA2tvdSlfGuk\\\\\\\/XyCmSuWhVNoCY4HxMOXUoWmuTJLMjGk3YhYzfsVjSaMTkiAdvi8ZQ9gFxlvxwoWuGm9yeOugLlmoSGeCDSTA25fvFUI5sJNat6PIWZ97oLQPLY0f7YRgDC4m0ur85klttYVkoPZ2f16Y4N4wQ8fAuMpZUhGGOL4\\\\\\\/+fF15cr8BN7uogMwotKss+S2iSZEMx9BknznHzZYClDEooLoU16yeji1PsIWAtH2Z4T1Dx0G32kEbcXECLxePAbP9whjlF8gsJwpebAN9StbWvpTGVf2JWZpe0IAMbtIOSFR+hppaAgf7xPcQ+0u9ZaaJ5CnMT8bmyE0AHE+93fa3JrtbBewjzOA8QUoK1N2CijgeRBMZoaF1s1PgScDOQzUThoDPFQKE\\\\\\\/35n+SC\\\\\\\/wJRYE71IdirPUz9X6NPEYlpcp\\\\\\\/Pgs1YIxTHLqmcjho1mSYti4c5zB+SUJ4y6+f3KPzXgFVjBNXaRzkuzwifRgsJ6OoTbwyQdWC5VPJzN2FyywMBKNoRYNbqrw4Eu5a1QhDXbzpFnFvNe9VhxmsXaJGHbQeZNx0mBgFymrbjNLT82UdA0hyBxLH\\\\\\\/jP0MqYaVjjm786T6gIAZ3meUMsFelDu0dyHCKE6kb7rwMdpBuWeaZt6B3OdXFiMhIp3GzUJTLwddxeJFl2mUt5OSZDgZjvlnGiHsysLwXjeDUdcacXzQmKBb7FeoMR6bkBb2ieLJXNtmerfho6sz3enlOEUT23T3Wyeq\\\\\\\/WcVIn3fE78fisWVZ8MR82n3zm17Cj3X4oCYeD3MDDXvthTPFEcl1h2jUO5rD6LvQV5xaSCaHhRReDAaQkDKcVRykubKGPNeSy6\\\\\\\/qlRAlVzeArLAL1knjihRmdhyH1MJlHC44qoMi\\\\\\\/8BjUpvsrKh7xGfV70CDdpHAabmCTbd9uUihSV0z0oToyT971HBxlFAR091Zfj\\\\\\\/D6awc33K0S+v+F8NdSgsVKBhG12ALkNzpIgaeMjVTCYCxZEQ4O+dEBQkx7HC+vPaQG4D9X1ofWKIAeFUHw2yCBWZiboTjqorDT0mDS0Muf\\\\\\\/QjVWUxa2p0BAPPf\\\\\\\/LkGWyFLVn\\\\\\\/t+VtqXFAkgjd8VVlFQm9EM3uRGmBXhShnmCoGA8Q5xJzEx8P+AP\\\\\\\/WwprXd3QZjvUAcgQtEPPQxRrnO45T5vR\\\\\\\/34LLzoQ\\\\\\\/z3wZdyjgED7cnoDTEWHBDzPwpZMm7eRlZWOmftJW0bQrKUr0n2jT7jNb1iIDj4U0KPe6RTn1sLvBGcqp3tTt3cbIaRyTT7Y47EI+7OUW6n8cTuTF6k70B5EmoDEF6uptiGBfuRZXdQnZFzL63oKg4JAM3HjyUqJ2VAGKI663QpP2lR9cKL89ZyZZBbCpdxuC6oqn2Z3IXuyL4ffVCC8Q8peAaLBpHbJO12g9ulWIQTcL+rv0Py6QT+6VHtRBsNtxzy0taykt1rO4WevLSK\\\\\\\/fXQKUlztAkPbHAaDRg4nbuqmCMyFXM+k+MpXelUHWFVNgNzzbhgIAtplvl0483iAVhV2rmsj3IWcBTfjM74aNeb8CQXXzcidcOs85MvMRo6ufNjX0x5lTIW5du52CUb8SsSWlgu6r3tAPswoYXX1SEnuyjRV3S1POJRez0v2imk\\\\\\\/NMUDfzarli0ePCBa237LBdd5V1\\\\\\\/ghgbXKD7\\\\\\\/pr++HLw2CNfsHcHVstYrs+XodlZls8d2\\\\\\\/5aUhsvRyrkgaPxWfdd7Ho8hEOx0uyHSvc4zLKjOmfQKGSvAC4GHHZnEhAVerg1Y3nyk7ruj90C3Vwq0k5Inpnx3xpsSgrl\\\\\\\/21kvU4wC0xioiZAaEBASdDUUtXNHKjbMkSQssxFHsMlwiF+IUiaEjTL+mHzr1Lv8LJpreQgWuaKztjnvvkFGoohu1R+A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022GHIJZzB4jvmBJN8zxKLfcA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022302f27144193f4ca4664a3aea91198377a5aa9162f2986e7343c0c382353711f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485325 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2315" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7gIic5WlSRvqVcst3A6aBii5EdCc4WFkEDnGAm1zIz0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2315" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sFAx-6Xg0oPKehVqwV8GlNYqOYVTh88KfymgnqMppMY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2305" title="Essential Function of the Serine Hydroxymethyl Transferase (SHMT) Gene During Rapid Syncytial Cell Cycles in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2327" title="A Coding Variant in the Gene Bardet-Biedl Syndrome 4 (BBS4) Is Associated with a Novel Form of Canine Progressive Retinal Atrophy" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485325" id="node485325" data-pisa="ggg;7/7/2315" data-pisa-master="ggg;g3.117.043141" data-apath="/ggg/7/7/2315.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Cycling Genomic Selection in a Multiparental Tropical Maize Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xuecai Zhang</span>, <span class="highwire-citation-author" data-delta="1">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="2">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="3">Michael Olsen</span>, <span class="highwire-citation-author" data-delta="4">Edward Buckler</span>, <span class="highwire-citation-author" data-delta="5">Gary Atlin</span>, <span class="highwire-citation-author" data-delta="6">Boddupalli M. Prasanna</span>, <span class="highwire-citation-author" data-delta="7">Mateo Vargas</span>, <span class="highwire-citation-author" data-delta="8">Félix San Vicente</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2315-2326;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043141 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485325" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xuecai Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Maize Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuecai%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuecai%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Colegio de Postgraduados, CP 56230, Montecillos, 56230 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael Olsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Nairobi 1041-00621, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BOlsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Olsen%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BOlsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Edward Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>United States Department of Agriculture, Agricultural Research Service, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gary Atlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Bill and Melinda Gates Foundation, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BAtlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Atlin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BAtlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Boddupalli M. Prasanna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Nairobi 1041-00621, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boddupalli%2BM.%2BPrasanna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prasanna%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoddupalli%2BM.%2BPrasanna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mateo Vargas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Universidad Autónoma Chapingo, 56230 Texcoco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mateo%2BVargas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vargas%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMateo%2BVargas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Félix San Vicente </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Maize Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=F%C3%A9lix%2BSan%2BVicente%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=San%20Vicente%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AF%25C3%25A9lix%2BSan%2BVicente%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:f.sanvicente@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">f.sanvicente@cgiar.org</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:f.sanvicente@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">f.sanvicente@cgiar.org</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/7/2315" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485325" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485325/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2315.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485325" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485325/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2315.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485325" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485325/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/7/2315.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485325" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485325/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection (GS) increases genetic gain by reducing the length of the selection cycle, as has been exemplified in maize using rapid cycling recombination of biparental populations. However, no results of GS applied to maize multi-parental populations have been reported so far. This study is the first to show realized genetic gains of rapid cycling genomic selection (RCGS) for four recombination cycles in a multi-parental tropical maize population. Eighteen elite tropical maize lines were intercrossed twice, and self-pollinated once, to form the cycle 0 (C<sub>0</sub>) training population. A total of 1000 ear-to-row C<sub>0</sub> families was genotyped with 955,690 genotyping-by-sequencing SNP markers; their testcrosses were phenotyped at four optimal locations in Mexico to form the training population. Individuals from families with the best plant types, maturity, and grain yield were selected and intermated to form RCGS cycle 1 (C<sub>1</sub>). Predictions of the genotyped individuals forming cycle C<sub>1</sub> were made, and the best predicted grain yielders were selected as parents of C<sub>2</sub>; this was repeated for more cycles (C<sub>2</sub>, C<sub>3</sub>, and C<sub>4</sub>), thereby achieving two cycles per year. Multi-environment trials of individuals from populations C<sub>0,</sub> C<sub>1</sub>, C<sub>2</sub>, C<sub>3</sub>, and C<sub>4</sub>, together with four benchmark checks were evaluated at two locations in Mexico. Results indicated that realized grain yield from C<sub>1</sub> to C<sub>4</sub> reached 0.225 ton ha<sup>−1</sup> per cycle, which is equivalent to 0.100 ton ha<sup>−1</sup> yr<sup>−1</sup> over a 4.5-yr breeding period from the initial cross to the last cycle. Compared with the original 18 parents used to form cycle 0 (C<sub>0</sub>), genetic diversity narrowed only slightly during the last GS cycles (C<sub>3</sub> and C<sub>4</sub>). Results indicate that, in tropical maize multi-parental breeding populations, RCGS can be an effective breeding strategy for simultaneously conserving genetic diversity and achieving high genetic gains in a short period of time.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/tropical-maize" class="hw-term hw-article-keyword hw-article-keyword-tropical-maize" rel="nofollow">tropical maize</a></li><li class="kwd"><a href="/keyword/multiparental-population" class="hw-term hw-article-keyword hw-article-keyword-multiparental-population" rel="nofollow">multiparental population</a></li><li class="kwd"><a href="/keyword/rapid-cycling-recombination-genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-rapid-cycling-recombination-genomic-selection" rel="nofollow">rapid cycling recombination genomic selection</a></li><li class="kwd"><a href="/keyword/realized-genetic-gains" class="hw-term hw-article-keyword hw-article-keyword-realized-genetic-gains" rel="nofollow">realized genetic gains</a></li><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">genetic diversity</a></li><li class="kwd"><a href="/keyword/multiparental-populations-mpp" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations-mpp" rel="nofollow">Multiparental Populations MPP</a></li></ul><p id="p-3">In the last 20 yr, marker-assisted selection has been widely used in plant breeding where a few markers significantly associated with the phenotypic trait are employed to predict the genetic value of the candidates for selection (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bernardo 2008</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">2016</a>). On the other hand, genomic-assisted breeding (genomic selection, GS) incorporates all available marker information simultaneously into a model to predict the genetic value of the candidates for selection (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Meuwissen <em>et al.</em> 2001</a>). In plants, a computer simulation study (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>) showed that better prediction accuracy of breeding and genetic values was achieved by incorporating all markers, as compared to using a subset of markers significantly associated with QTL. This result was verified by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Massman <em>et al.</em> (2013)</a>, who used a biparental temperate maize population derived from a cross between two distinct heterotic groups (B73 and Mo17); the testcrosses were evaluated under well-watered conditions, and the population advanced using rapid cycling GS (RCGS where all markers are used for prediction) and marker-assisted recurrent selection (MARS where only significant markers are used for prediction). <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Massman <em>et al.</em> (2013)</a> reported that RCGS had a superior response for stover yield, as well as stover and grain yield indices that were 14–50% higher than those of MARS.</p><p id="p-4">In tropical maize, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beyene <em>et al.</em> (2015)</a> evaluated realized genetic gains in grain yield from RCGS in eight biparental maize populations in drought stress environments. The authors found that the average gain from RCGS per cycle across eight populations was 0.086 ton ha<sup>−1</sup> and that hybrids derived from cycle 3 produced 7.3% (0.176 ton ha<sup>−1</sup>) higher grain yield than those developed through the conventional pedigree breeding method. RCGS in biparental populations offered the advantage of significant time efficiency over conventional breeding methods, as up to three cycles of RCGS can be conducted within a year. Interestingly, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Beyene <em>et al.</em> (2015)</a> pointed out that the average genetic gain per year in tropical maize grain yield using RCGS was three times higher than that achieved by using conventional pedigree-based phenotypic selection in drought stress environments. Furthermore, Vivek <em>et al.</em> (2016) recently reported a study on realized genetic gains in grain yield using two biparental populations generated by crossing two elite Asian maize inbred lines with an African drought tolerant line. Cycle 1 (C<sub>1</sub>) was formed by recombining the top 10% of the F<sub>2:3</sub> families. Cycle 2 (C<sub>2</sub>) was derived using two different methods: (1) phenotypic selection (C<sub>2</sub>-PS); and (2) GS (C<sub>2</sub>-GS). Results showed that C<sub>2</sub>-GS top-crosses produced 4–43% higher grain yield than C<sub>2</sub>-PS top-crosses.</p><p id="p-5">For RCGS within biparental populations, prediction accuracy is achieved thanks to high linkage disequilibrium (LD), no pedigree, and no family substructure (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Zhang <em>et al.</em> 2015</a>). However, predictions across biparental populations will be poor if unrelated biparental populations with different allelic diversity are used as the training population. Furthermore, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Jannink <em>et al.</em> (2010)</a> outlined some of the disadvantages of using GS applications in biparental populations: (1) separate model training is required for each biparental cross; GS should be applied to the entire population; and (2) the first generation of progeny from a cross needs to be phenotyped and candidates cannot be selected on the basis of prior information, a practice that slows down the breeding cycle. Biparental populations have also been widely used for detecting and mapping QTL. However, QTL mapping power and resolution might be comparatively reduced in biparental mapping populations as the number of segregating causal loci is very low, because large blocks of parental chromosomes are preserved (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cavanagh <em>et al.</em> 2008</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Thepot <em>et al.</em> 2015</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Lehermeier <em>et al.</em> 2014</a>). The problem of limited allelic diversity in one genetic background which occurs in biparental populations can be overcome by the use of multi-parental populations (MPP) with greater allelic diversity and from different genetic backgrounds (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Verhoeven <em>et al.</em> 2006</a>), along with increased polymorphism and recombination as compared to biparental populations (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahfock <em>et al.</em> 2014</a>).</p><p id="p-6">Different MPPs have been constructed with the aim of increasing precision in fine mapping. For example, researchers have combined different biparental populations using factorial crosses, partial or complete diallels (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Rebai and Goffinet 1993</a>), or circular crosses (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cavanagh <em>et al.</em> 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Huang <em>et al.</em> 2012</a>). The designs and analyses of these types of populations have been extended to genomic-enabled studies using different schemes and designs for efficient training, and for testing MPPs in different species.</p><p id="p-7">Multi-parental populations are useful for mapping extensive numbers of loci. <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Thepot <em>et al.</em> (2015)</a> used a multi-parental mapping population created after 12 generations of recombination among 60 founder wheat lines. The 12 generations of recombination broke up the LD and the existing population structure of the original populations. This approach to fine mapping helped to identify 26 genomic regions, six of which carried flowering QTL, and allowed detecting loci under selection and association mapping. In a recent article, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hoffstetter <em>et al.</em> (2016)</a> used 47 bi-parental crosses, including 23 parental wheat lines as a training set, and 17 half-sib lines as a validation set, with some of the lines from the training set included in the testing set. Prediction accuracy using subsets of the training population to predict the validation sets ranged from 0 to 0.85. In maize, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Lehermeier <em>et al.</em> (2014)</a> evaluated prediction accuracy in 21 biparental doubled haploid populations, including 10 dent kernel type related populations, and 11 flint kernel type related populations. The authors found that prediction combining several half-sibs gave similar or higher prediction accuracy than predictions within biparental populations.</p><p id="p-8">However, several theoretical complexities and challenges arise when attempting to perform GS in MPP. Plant breeders usually work with sets of full-sib families generated from crosses of inbred parents that vary in size so that extensive LD exists within each family; however, different LD patterns must exist across families. Therefore, because, in a typical MPP, the breeder is faced with LD across a large set of families, the level of LD as well as the marker density are important factors to be considered when applying GS to MPP. <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Zhong <em>et al.</em> (2009)</a> using marker data on 42 two-row spring barley inbred lines simulated MPP with high and low LD populations generated from multiple inbred crosses. The authors suggested a trade-off between the model-method’s ability to capture LD between marker and QTL <em>vs.</em> its ability to exploit marker-based relatedness of individuals. Genomic relationship information (Best Linear Unbiased Predictor, GBLUP) is more valuable than LD information given by models that use marker effects. However, when markers were in strong LD with QTL with large effects, models based on marker effects were better predictors than models based on the genomic relationship between individuals.</p><p id="p-9">The reliability of GS is greatly influenced by the number of phenotypes; therefore, combining data sets from MPP should increase the reliability of GS by making it more efficient and attractive for use in breeding. However, when combining populations, allele frequencies, LD and segregating haplotypes are different in different populations. Thus, when the marker effects are different between the different combined populations, this can reduce the reliability of GS. <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de Roos <em>et al.</em> (2009)</a> studied this problem by combining two simulated cattle populations that diverged for certain number of generations; the authors found that increased genomic accuracy is achieved when all populations are combined in one training population but increasing marker density is required when the diversity of the combined population increases.</p><p id="p-10">Despite the above theoretical and simulation results, which show that genomic-enabled prediction accuracy of MPPs is higher than the accuracy achieved within a single population, studies implementing RCGS in MPPs have not been reported. The research presented in this paper was initially conceived in 2009 and started in 2010. It included an initial MPP made up of 18 elite tropical maize lines intercrossed twice and self-pollinated once to form the cycle 0 (C<sub>0</sub>) training population. A total of 1000 ear-to-row C<sub>0</sub> families were genotyped with dense GBS markers, and their testcrosses were phenotyped at four locations in Mexico to develop genomic prediction models. One cycle of phenotypic selection (C<sub>0</sub>–C<sub>1</sub>) and three cycles of RCGS (C<sub>1</sub>–C<sub>4</sub>) were carried out. The main objectives of this study were: (1) to report the realized genetic gains of four cycles (C<sub>1</sub>, C<sub>2</sub>, C<sub>3</sub>, and C<sub>4</sub>), plus the original training population (C<sub>0</sub>) in multi-environmental field trials of RCGS-assisted breeding evaluated together with four benchmark checks in two Mexican environments (locations), and (2) to investigate the genetic diversity of the families within each RCGS selection cycle to assess the level of genetic diversity after three cycles of rapid cycling GS.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Developing the training population from 18 tropical maize inbred lines</h3><p id="p-11">The RCGS experiment was designed in 2009 as part of the MasAgro project funded by Mexico’s Secretariat of Agriculture, Livestock, Rural Development, Fisheries and Food (SAGARPA, its acronym in Spanish) through the Sustainable Modernization of Traditional Agriculture program (MasAgro; <a href="http://masagro.mx">http://masagro.mx</a>).</p><p id="p-12">The steps in the breeding scheme used for RCGS are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. In total, 18 CIMMYT tropical maize inbred lines (CML247, CML264, CML448, CML494, CML498, CML531, CLRCW72, CLRCW75, CLRCW76, CLRCW93, CLRCW100, CLRCW260, CLWN201, CLWN228, CLWN229, CLWN247, CLG2312, and CLSPLW04), widely used in lowland tropical breeding environments, were crossed as parents to form the training population through twice intermated pollination and one self-pollination; the parents were selected based on their general combining ability for grain yield and <em>per se</em>, visual evaluation information for major stress tolerance and disease resistance in lowland tropical breeding environments. All 18 original parents tended to group in heterotic pattern group “B” (flint type kernel).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Breeding scheme used in the MPPs reported in this study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1642002731" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Breeding scheme used in the MPPs reported in this study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2315/F1.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/7/2315/F1.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485411" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Breeding scheme used in the MPPs reported in this study.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">In the 2010B season, half-diallel crosses were made between the 18 original parents to generate all possible F<sub>1</sub> progenies (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). In the 2011A season, all F<sub>1</sub> were planted ear-to-row and intermated to form the S<sub>1</sub> population; then, all the F<sub>1</sub> were separated into two groups of equal size. Bulk pollen from the first group was used to pollinate all plants of the other group and vice versa; three ears were harvested from each F<sub>1</sub> family, and equal amounts of seed from each selected F<sub>1</sub> ear were bulked to form the subsequent generation for planting. In the 2011B season, 4800 S<sub>1</sub> individuals were planted and self-pollinated and advanced to S<sub>2</sub>. The best 1000 S<sub>2</sub> ears were selected, and planted ear-to-row in the 2012A season (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). A single-cross tester (CML495/CML549) from the complementary heterotic group (dent type kernel) was used to generate testcrosses. All pollination activities were conducted at CIMMYT’s experiment station in Agua Fria, Puebla.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485391/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485391/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Number of families and individual plants sown, selected, and advanced in each breeding cycle and among-family, within-family, and total selection intensity</span><div class="sb-div caption-clear"></div></div></div><p id="p-15">The training population (C<sub>0</sub>) for developing genomic prediction models was formed with the best 1000 selected S2s. Testcrosses of all 1000 selected S<sub>2</sub> were planted using a partial replicated design with 25% of replicated genotypes at four optimal Mexican locations. Phenotypic data were collected at all locations for &gt;10 agronomic traits, including grain yield at 12.5% moisture content (GY), anthesis date (AD), silking date (SD), plant height (PH), ear height (EH), and moisture content (MOI). For each S<sub>2</sub> family, DNA was extracted by bulking equal amounts of leaf tissue from 15 individual plants. Genotyping-by-sequencing was performed at Cornell University Biotechnology Resource Center as described by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Wu <em>et al.</em> (2016)</a>, where 955,690 SNPs were generated for each DNA sample. In the training population, the genomic prediction model was developed by using only 331,740 filtered SNPs with minor allele frequency (&gt;0.05), and where the missing data rate was &lt;10%.</p></div><div id="sec-3" class="subsection"><h3>Cycle 0 (C<sub>0</sub>) phenotypic selection and formation of cycle 1 (C<sub>1</sub>)</h3><p id="p-16">In C<sub>0</sub>, phenotypic selection was conducted by ranking the grain yield of the 1000 S<sub>2</sub> testcrosses. The best 50 respective S<sub>2</sub> families were selected and planted ear-to-row, 25 plants per family (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Cycle 1 (C<sub>1</sub>) was formed by intermating the 50 selected S<sub>2</sub> families. The 50 families were divided into two equal groups, and bulk pollen from the first group was used to pollinate all plants in the other group and vice versa. Based on visual evaluation of flowering time, plant type, plant/ear height, well-filled ears, and reaction to naturally occurring major diseases, along with among-family and within-family selection, 157 ears (1–6 ears from each selected family) were harvested and shelled individually to form C<sub>1</sub>.</p></div><div id="sec-4" class="subsection"><h3>Rapid cycling recombination of GS cycle 1 (C<sub>1</sub>), cycle 2 (C<sub>2</sub>), and cycle 3 (C<sub>3</sub>)</h3><p id="p-17">In C<sub>1</sub>, 157 selected ears (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) were planted ear-to-row, 25 plants per family. DNA was extracted from the bulk tissue and shipped to Cornell University Biotechnology Resource Center for genotyping-by-sequencing. The Genomic Best Linear Unbiased Predictor (GBLUP) model (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">VanRaden 2007</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2008</a>) implemented in the BGLR package was used for genomic prediction. Genomic estimated breeding values were calculated for all 157 C<sub>1</sub> families; among-family selection was implemented based on genomic estimated breeding value information. The top 25 families were selected and intermated to form the C<sub>2</sub> population. The 25 families selected were divided into two equal groups. Bulk pollen from one group was used to pollinate all plants in the other group and vice versa. Within-family selection was implemented based on visual evaluation of flowering time, plant type, plant/ear height, well-filled ears, and reaction to naturally occurring major diseases. A total of 91 ears were harvested, and shelled individually to form C<sub>2</sub>. In C<sub>2</sub> and C<sub>3</sub>, the recombination protocol was repeated. The number of families and individual plants planted per cycle; number of families selected for next cycle recombination; number of ears harvested per cycle; and selection intensity information are listed in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. After C<sub>3</sub> recombination, 45 ears were harvested and shelled individually to form C<sub>4</sub>; within-family selection was implemented based on visual evaluation of agronomic traits. Families from Cycle 4 (C<sub>4</sub>) were not genotyped, as this was the last rapid cycling recombination.</p></div><div id="sec-5" class="subsection"><h3>Phenotypic evaluation of the selection cycles for assessing genetic gains—benchmark checks and experimental designs in multi-environment trials</h3><p id="p-18">A total of 233 testcrosses was tested in the field to estimate the realized genetic gains; these entries belong to different groups. One group of entries (48) represents selection cycle C<sub>0</sub>, which is a subset of the best 50 families selected from the training population. Another group of entries represents RCGS cycles C<sub>1</sub> (47 entries), C<sub>2</sub> (48 entries), C<sub>3</sub> (43 entries), and C<sub>4</sub> (43 entries), and the last group of entries comprises the four benchmark checks (two local checks, one commercial check, and one experimental baseline check formed by testcrossing all 18 original parents with a single-cross tester (CML495/CML549). All the entries were crossed with a single-cross tester (CML495/CML549). Their testcrosses were planted at two locations in Mexico (Agua Fria and Tlaltizapan) in a modified split-plot design where the selection cycles were the main plots and the entries within each selection cycle were the subplots. The experimental design within each selection cycle (main plot) was an alpha-lattice design with two replications per location. The four benchmark checks were repeated in each subplot and planted together with the entries belonging to the different GS cycle; for example, the four checks were planted in the two replicates of the subplot where entries from cycle C<sub>0</sub> were planted.</p></div><div id="sec-6" class="subsection"><h3>Statistical analyses</h3><p id="p-19">Phenotypic data were collected at the two locations for the main agronomic traits including GY, AD, SD, PH, EH, and MOI. A linear mixed model was fitted to the data considering the incomplete block within replicate as random effects, and locations, cycles (main plot), entry within cycle (subplot), cycle × location interaction, and entry within cycle × location interactions as fixed effects. Random errors are assumed to be identically and independently normally distributed with mean zero and homogeneous variance. ANOVA for GY were performed including the evaluated entries from the selection cycles and the checks. Genetic gain response was assessed by regressing mean GY values on the selection cycle means (C<sub>0</sub>, C<sub>1</sub>, C<sub>2</sub>, C<sub>3</sub>, and C<sub>4</sub>) within each location and combined across both locations. ANOVA for AD, SD, PH, EH, and MOI were performed for each location and combined across both locations.</p></div><div id="sec-7" class="subsection"><h3>SNP genotyping</h3><p id="p-20">For each cycle, bulk DNA of each planted family was sent to the Biotechnology Resource Center of Cornell University for genotyping-by-sequencing. The number of DNA samples used for genotyping were 1000, 157, 91 and 44 in C<sub>0</sub>, C<sub>1</sub>, C<sub>2</sub> and C<sub>3</sub>, respectively. Families from C<sub>4</sub> were not genotyped. Genotyping-by-sequencing, SNP calling, imputation and filtering were performed as described by <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Wu <em>et al.</em> (2016)</a>. Briefly, genomic DNA was digested with the restriction enzyme <em>Ape</em>KI. GBS libraries were constructed in 96-plex and sequenced on Illumina HiSeq2000 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Elshire <em>et al.</em> 2011</a>). SNP calling was performed using the TASSEL GBS Pipeline, and a GBS 2.7 TOPM (tags on physical map) file was used to anchor reads to the Maize B73 RefGen_v2 reference genome (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Glaubitz <em>et al.</em> 2014</a>). Imputation was carried out with the FILLIN method in TASSEL 5.0 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Swarts <em>et al.</em> 2014</a>), which anonymized GBS 2.7 haplotypes made from 8000-site windows. In total, 955,690 SNPs were generated for each sample, filtering was performed with minor allele frequency (&gt;0.05) and the missing data rate was &lt;10%.</p></div><div id="sec-8" class="subsection"><h3>Assessing the genetic diversity of the selection cycles</h3><p id="p-21">Based on genomic data, we computed two genetic diversity indices between the families of the different selection cycles as well as the 18 parents. We calculated the Shannon Diversity Index of the sample for each selection cycle as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2315/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2315/embed/mml-math-1.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2315/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2315/embed/mml-math-2.gif"/></noscript></span></span> is the frequency of the major allele in the <em>a</em>th marker over the entire sample, and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2315/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2315/embed/mml-math-3.gif"/></noscript></span></span> is the total number of markers. The expected proportion of heterozygous loci per individual was computed as the mean of heterozygosity for each marker as <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2315/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2315/embed/mml-math-4.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2315/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2315/embed/mml-math-5.gif"/></noscript></span></span> is the frequency of the major allele in the <em>a</em><sup>th</sup> marker of the <em>l</em><sup>th</sup> individual, and <em>L</em> is the number of individuals.</p><p id="p-22">Multidimensional scaling (MDS) was performed with the TASSEL software (<a href="http://www.maizegenetics.net/tassel">http://www.maizegenetics.net/tassel</a>) to assess the genetic similarity of all the materials in each selection cycle.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-23">The phenotypic and genotypic data for the training population (cycle C<sub>0</sub>) evaluated in four sites, the phenotypic and genotypic data for the evaluation of the entries from the different selection cycles (C<sub>0</sub>, C<sub>1</sub>, C<sub>2</sub>, C<sub>3</sub>, and C<sub>4</sub>), as well as a brief GUIDE can be found in the link <a href="http://hdl.handle.net/11529/10927">http://hdl.handle.net/11529/10927</a>. A marker information file and characteristics of the genetic materials are also included in the link.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Heritability and prediction accuracy of GY in the training population</h3><p id="p-24">The combined GY heritability across both locations was 0.34, while GY heritability at individual locations was 0.48, and 0.19 in Agua Fria and Tlaltizapan, respectively. Low-to-intermediate GY heritability was observed in the individual location analysis and combined analysis, mainly because GY is a complex trait. To mimic future prediction problems we will face, we implemented a fivefold random cross-validation with 100 replicates using entries in C<sub>0</sub> (training population) for GY; the mean correlation between the predicted and observed values was 0.55.</p></div><div id="sec-12" class="subsection"><h3>Realized genetic gains from rapid cycling recombination of GS for grain yield</h3><p id="p-25">A total of five groups of entries from C<sub>0</sub>, C<sub>1</sub>, C<sub>2</sub>, C<sub>3</sub>, and C<sub>4</sub> plus four checks were used for field evaluation at two Mexican locations (Agua Fria and Tlaltizapan). Mean grain yield for each cycle and average gains per cycle are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The Tlaltizapan location had the highest mean yield, with C<sub>4</sub> reaching 10.96 ton ha<sup>−1</sup>. At both individual locations, the average performance across all C<sub>4</sub> entries surpassed the grain yield performance of the other cycles; average grain yield performance across all C<sub>4</sub> entries was 7.13 ton ha<sup>−1</sup>, and 10.96 ton ha<sup>−1</sup> for Agua Fria and Tlaltizapan, respectively. Also, the combined analyses of the two locations showed an increase in mean grain yield in C<sub>4</sub> of 9.05 ton ha<sup>−1</sup> over that achieved in C<sub>3</sub> (8.92 ton ha<sup>−1</sup>) and over the other GS cycles.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485423/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485423/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean of GY (ton ha<sup>−1</sup>) for each genomic cycle C<sub>0</sub>, C<sub>1</sub>, C<sub>2</sub>, C<sub>3</sub>, and C<sub>4</sub>, broad-sense heritability (H<sup>2</sup>), and mean of the four testers at each location (Agua Fria and Tlaltizapan), and combined across the two locations</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">For each location and combined, the entries representing the first selection cycle (C<sub>1</sub>) had lower GY than entries representing the base selection cycle (C<sub>0</sub>); this was due to the fact that the parents of the C<sub>1</sub> population (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) were not selected based on the grain yield performance of the testcrosses <em>per se</em>; instead, among- and within-family selection was conducted based on visual evaluation for flowering time, plant type, plant/ear height, well-filled ears, and reaction to naturally occurring major diseases. The other important reason is that the best 50 selected families were used to represent selection cycle C<sub>0</sub> in the genetic gain evaluation study (rather than the random selected families). In C<sub>3</sub>, GY substantially increased in TL and combined across two locations with respect to previous selection cycles. For analyses in Agua Fria location, GY declined slightly to 6.88 ton ha<sup>−1</sup> for C<sub>3</sub> (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). However, in the subsequent genomic selection cycles, the increases in realized genetic gains are clear for each location and combined across locations.</p><p id="p-28">All selection cycles had higher average grain yield than the corresponding mean of the checks for individual locations and combined. Concerning broad-sense heritability, there was a decline in cycles C<sub>1</sub> and C<sub>4</sub> as compared to the base cycle (C<sub>0</sub>) in Agua Fria and Tlaltizapan; however, the opposite occurred when combining cycles C<sub>1</sub> and C<sub>3</sub> in both locations as compared to the heritability in the base cycle (C<sub>0</sub>). In the combined analyses across both locations, broad-sense heritability (H<sup>2</sup>) showed no decrease. In general, results showed that at each location and combined, while there was a decrease in GY from C<sub>0</sub> to C<sub>1</sub> there were also important increases in realized genetic gains for trait GY at the two locations from C<sub>2</sub> to C<sub>3</sub> and from C<sub>3</sub> to C<sub>4</sub>. The mean grain yield of the benchmark checks varied slightly for each cycle at Agua Fria but stayed fairly constant at Tlaltizapan and combined locations for all selection cycles. Mean performance of the checks in selection cycle C<sub>0</sub> was consistently lower than the mean performance in the other selection cycles.</p><p id="p-29">The average gains per cycle for each location and combined across the two locations ranged from 0.131 ton ha<sup>−1</sup> to 0.177 ton ha<sup>−1</sup> when considering all cycles (C<sub>0</sub>–C<sub>4</sub>), and from 0.171 ton ha<sup>−1</sup> to 0.276 ton ha<sup>−1</sup> when considering only rapid cycling GS (C<sub>1</sub>–C<sub>4</sub>). For the combined location analyses, the realized genetic gains were 0.158 ton ha<sup>−1</sup> and 0.225 ton ha<sup>−1</sup> for cycles (C<sub>0</sub>–C<sub>4</sub>) and cycles (C<sub>1</sub>–C<sub>4</sub>), respectively. The realized genetic gains due to RCGS were highest in C<sub>4</sub> at the two locations and combined across them.</p></div><div id="sec-13" class="subsection"><h3>Changes in the mean for unselected flowering, moisture, and height traits</h3><p id="p-30">The effects of genomic selection on unselected flowering, moisture and height traits are shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. On average, anthesis and silking days of the entries representing C<sub>4</sub> did not increase with respect to their averages in early GS cycles (C<sub>0,</sub> C<sub>1,</sub> C<sub>2</sub>, and C<sub>3</sub>). They ranged from 56 d for anthesis and 57 d for silking for all genomic selection cycles, and showed good general synchrony between both flowering times. However, GS produced taller plants and ear insertions during cycles C<sub>3</sub> and C<sub>4</sub> than during cycles C<sub>1</sub> and C<sub>2</sub>. Grain moisture content did not seem to have been greatly affected after the three cycles of RCGS.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485514/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485514/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Means of entry and checks for traits anthesis days (AD, days), silking days (SD, days), plant height (PH, centimeter), ear height (EH, centimeter), and moisture content (MOI, %) in each cycle across the two locations</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">The plant and ear heights for the two latest GS cycles were ∼6–10 cm higher than the plant and ear heights of the maize plants for the two earlier GS cycles (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The phenotypic variance (data not shown) of the entries for each GS cycle varied during the latest cycles (C<sub>3</sub> and C<sub>4</sub>), and tended to decline with respect to the early cycles. Similarly, trends in broad-sense heritability are smaller in the last cycle (C<sub>4</sub>) than in early cycles (C<sub>0</sub> and C<sub>1</sub>). The mean of the entries within each selection cycle, and the mean of the benchmark checks did not differ much for the different traits. For example, the check had ∼56 d to anthesis and 57 d to silking. In general, the checks had smaller plant height, taller ear height and similar grain moisture than the entries in the selection cycle.</p><p id="p-33">The phenotypic correlation between the different traits and grain yield changed from different selection cycles (data not shown). Results indicated positive correlations between grain yield and days to silk ranging from 0.3 to 0.5 for the different selection cycles. Negligible and negative correlations between grain yield and plant height, anthesis days, and moisture content were found.</p></div><div id="sec-14" class="subsection"><h3>Genetic diversity of the rapid cycle recombination of GS</h3><p id="p-34">The diversity structure pattern including the 18 parents, the C<sub>0</sub> families and the individuals selected as parents of C<sub>1</sub> is displayed in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>; the selected individuals are well spread along the three dimensions, and should capture most of the diversity in the C<sub>0</sub> families. The original parents, the C<sub>1</sub> families and the selected individuals (that form the parents of C<sub>2</sub> selection based on genomic prediction) are depicted in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>. The C<sub>1</sub> families are located between dimensions 1 and 3, close to four of the original parents located in this region of the figure.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of parents, cycle C0 entries (A) and the selected parents, and cycle C1 entries (B) and the selected parents based on rapid cycling genomic selection-assisted recombination." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1642002731" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of parents, cycle C&lt;sub&gt;0&lt;/sub&gt; entries (A) and the selected parents, and cycle C&lt;sub&gt;1&lt;/sub&gt; entries (B) and the selected parents based on rapid cycling genomic selection-assisted recombination.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2315/F2.medium.gif" width="440" height="362"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/7/2315/F2.medium.gif" width="440" height="362"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485469" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Distribution of parents, cycle C<sub>0</sub> entries (A) and the selected parents, and cycle C<sub>1</sub> entries (B) and the selected parents based on rapid cycling genomic selection-assisted recombination.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The original 18 parents, the C<sub>2</sub> families, and the parents selected to form the next selection cycle are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>. The C<sub>2</sub> families and the selected parents are located between dimensions 1 and 3, clearly heading in the direction of two of the original parents located in this region of the three-dimensional figure. Finally, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> includes the original 18 parents, the C<sub>3</sub> families and the selected parents that were intermated in RCGS to form C<sub>4</sub>. Clearly, the C<sub>3</sub> families and the selected parents that form C<sub>4</sub> are concentrated around the two original parents located in the upper region of the figure, between dimensions 1 and 3. However, a direct comparison between the genetic diversity of different populations (C<sub>0</sub>–C<sub>4</sub>) may be confounded by the differences in population size, and also by the different levels of inbreeding in the different selection cycles.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of parents, cycle C2 entries (A), cycle C3 entries (B) and the selected parents based on rapid cycling genomic selection-assisted recombination." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1642002731" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of parents, cycle C&lt;sub&gt;2&lt;/sub&gt; entries (A), cycle C&lt;sub&gt;3&lt;/sub&gt; entries (B) and the selected parents based on rapid cycling genomic selection-assisted recombination.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2315/F3.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/7/2315/F3.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485349" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-37" class="first-child">Distribution of parents, cycle C<sub>2</sub> entries (A), cycle C<sub>3</sub> entries (B) and the selected parents based on rapid cycling genomic selection-assisted recombination.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> depicts the two plot dimensions of the multidimensional scaling with the 18 parents and all families in selection cycles C<sub>0</sub>–C<sub>4</sub>. The C<sub>3</sub> families and the selected parents that form C<sub>4</sub> are located toward the upper left quadrant of the biplot in the same direction as one of the original parents.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the two dimensions of the multi-dimensional scaling analysis including all samples of the original parents and families in each cycle." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1642002731" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plot of the two dimensions of the multi-dimensional scaling analysis including all samples of the original parents and families in each cycle.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2315/F4.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/7/2315/F4.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485499" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">Plot of the two dimensions of the multi-dimensional scaling analysis including all samples of the original parents and families in each cycle.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Concerning the genetic diversity of the different C<sub>0</sub>–C<sub>4</sub> entries, <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the values of Shannon’s Diversity Index, the heterozygosity and the number of SNPs for the 18 parents, cycles C<sub>0</sub>–C<sub>4</sub>, the selected parents from C<sub>0</sub> to C<sub>4</sub>, along with all the entries. Results of Shannon’s Diversity Index and the heterozygosity for cycles C<sub>0</sub>–C<sub>4</sub>, along with the selected parents from C<sub>0</sub> to C<sub>4</sub> and all other entries, are displayed in the barplots in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. Genetic diversity did not decline in the initial cycles and the trends in Shannon’s Diversity Index and heterozygosity even showed a slight increase in cycles C1 and C2. However, there was a decrease in the three diversity measurements for the population represented by the C<sub>3</sub> entries, as well as for the parents selected to form C<sub>4</sub>. Note that entries from C<sub>4</sub> were not included because they were not genotyped.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485461/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485461/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">The Shannon Diversity Index, heterozygosity, and number of SNPS of the 18 original parents, the number of families in cycles C<sub>0</sub>–C<sub>3</sub> (in parentheses), and the selected parents in C<sub>0</sub>–C<sub>3</sub>, and including all the entries</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar-plot of the Shannon Diversity Index (blue) and heterozygosity (brown and light pink) for the original parents, individuals from cycles C0&#x2013;C3 and the selected entries in light blue and light pink." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1642002731" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bar-plot of the Shannon Diversity Index (blue) and heterozygosity (brown and light pink) for the original parents, individuals from cycles C&lt;sub&gt;0&lt;/sub&gt;&#x2013;C&lt;sub&gt;3&lt;/sub&gt; and the selected entries in light blue and light pink.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2315/F5.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/7/2315/F5.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485394" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">Bar-plot of the Shannon Diversity Index (blue) and heterozygosity (brown and light pink) for the original parents, individuals from cycles C<sub>0</sub>–C<sub>3</sub> and the selected entries in light blue and light pink.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Changes in the status of marker frequency</h3><p id="p-43">For each SNP marker, we measured the change in the allele frequency of the 18 parents’ original parents that formed C<sub>0</sub>, and the allele frequency of the C<sub>3</sub> and C<sub>4</sub> entries. Initially, allele frequency was calculated for the 18 parents, and for the entries of cycles C<sub>3</sub> and C<sub>4</sub>; SNP markers with allele frequencies between 0.05 and 0.95 were considered polymorphic markers, whereas markers with allele frequencies &lt;0.05 or &gt;0.95 were considered monomorphic markers. Using these criteria, we found that from a total of 1120 SNP markers (0.117% of all markers) that changed their polymorphic/monomorphic status, 123 markers swapped the major allele frequency, 968 markers became monomorphic in C<sub>3</sub>–C<sub>4</sub> although they were polymorphic in the 18 parents and C<sub>0</sub>, and only 29 markers were polymorphic in cycles C<sub>3</sub> and C<sub>4</sub>, although they were monomorphic in the 18 parents and cycle C<sub>0</sub> (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Chromosomes 1–4 had a low percent change in allele frequency (0.080–0.097%), whereas the rates of change in chromosomes 5 and 6 were 0.177, and 0.139%, respectively. Chromosomes 7–10 had a percent change in allele frequency ranging from 0.118 to 0.131%. Almost two thirds of the markers (614,663 markers, 64.37%) changed their allele frequencies without becoming monomorphic and 33.99% (324,575) of the markers changed their allele frequency by &lt;15%.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485576/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485576/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Number of SNP markers with allele swaps, number of polymorphic markers that became monomorphic and number of markers that were monomorphic and became polymorphic from [parents-C<sub>0</sub>] to [C<sub>3</sub>–C<sub>4</sub>]</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">Markers with changes in their frequency were clustered according to the physical distance in the map; SNPs with a physical distance &lt;1000 bp were considered a cluster (or haplotype). A total of 88 clusters (haplotypes) of different sizes were found with three or more SNPs that changed their frequency. The distribution and size of the clusters as well the type of change are shown in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>. For example, chromosome 1 had one cluster with four markers that changed their frequency; this indicated that there were at most 4000 bp units where these four markers were located. Most of the changes correspond to SNP markers that became monomorphic in cycles C<sub>3</sub> and C<sub>4</sub> although they were polymorphic in the 18 parents and cycle C<sub>0</sub>. Most of the clusters were found on chromosome 5 (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485452/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485452/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Distribution and size of clusters of SNPs with changes in their polymorphic status by chromosome</span><div class="sb-div caption-clear"></div></div></div><p id="p-45"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> depicts the location of the clusters for each chromosome in the genome. For example, chromosome 1 has one cluster with four markers that changed their frequency (green color), one cluster of four markers that changed from monomorphic in the 18 parents and cycle C<sub>0</sub> to polymorphic in cycles C<sub>3</sub> and C<sub>4</sub> (blue color), and three clusters with three markers, four clusters with four markers, and two clusters with six markers that changed from polymorphic in the 18 parents and cycle C<sub>0</sub> to monomorphic in cycles C<sub>3</sub> and C<sub>4</sub> (black color). As shown in <a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>, most of the clusters with changes in their allele frequency occurred in chromosome 5.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome location of clusters of SNPs with changes in their polymorphic status." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1642002731" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome location of clusters of SNPs with changes in their polymorphic status.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2315/F6.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/7/2315/F6.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485387" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-46" class="first-child">Genome location of clusters of SNPs with changes in their polymorphic status.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-47">Previous studies on temperate and tropical maize showed realized gains of RCGS in biparental populations (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Massman <em>et al.</em> 2013</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Beyene <em>et al.</em> 2015</a>; Vivek <em>et al.</em> 2016). In this study, our results showed realized gains of RCGS in a multi-parental tropical maize population that originated from crosses of 18 CIMMYT elite tropical maize lines. From a practical breeding perspective, multi-parental populations might not be an attractive option because the mean of 18 parental lines might be lower than the mean of the best few lines that could be used in biparental crosses; however, as diversity becomes an important issue in GS, multi-parental populations offer the opportunity to maintain diversity, while still achieving rapid cycles with high realized grain yield genetic gains achieved in a shorter period of time, as found in this study. As for the decrease in genetic diversity, this is not of much concern in a short-term selection (four to five cycles), especially if the new developed lines from C<sub>4</sub> are crossed with other lines for further breeding.</p><div id="sec-17" class="subsection"><h3>Trends in the realized genetic gains of multi-parental populations for grain yield</h3><p id="p-48">The genetic gains per unit of time are given by the breeders’ equation, which is Gain = (<em>i×r×h</em>)/I, where <em>i</em> is the selection intensity, <em>r</em> is the selection accuracy, <em>h</em> is the square root of narrow-sense heritability, and <em>I</em> is the time (in years) it takes to complete a selection cycle. In this study, the gains in GY in different selection cycles were not consistent, decreasing slightly from C<sub>0</sub> to C<sub>2</sub>, while increasing significantly from C<sub>2</sub> to C<sub>3</sub>, and from C<sub>3</sub> to C<sub>4</sub>. As for analyses combining the two sites, the gains in grain yield were 6.2 and 7.7% from C<sub>0</sub> to C<sub>4</sub> and from C<sub>1</sub> to C<sub>4</sub>, respectively. The combined realized genetic gains reached 0.158 ton ha<sup>−1</sup> per cycle for C<sub>0</sub>–C<sub>4</sub>, (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>) and 0.225 ton ha<sup>−1</sup> per cycle for RCGS C<sub>1</sub>–C<sub>4</sub> (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Response to selection for grain yield from the families of (A) rapid cycling recombination genomic selection for cycles C0, C1, C2, C3, and C4 and of (B) rapid cycling recombination genomic selection for cycles C1, C2, C3, and C4. Mean of the checks (red), and mean of the entries (blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1642002731" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Response to selection for grain yield from the families of (A) rapid cycling recombination genomic selection for cycles C&lt;sub&gt;0&lt;/sub&gt;, C&lt;sub&gt;1&lt;/sub&gt;, C&lt;sub&gt;2,&lt;/sub&gt; C&lt;sub&gt;3,&lt;/sub&gt; and C&lt;sub&gt;4&lt;/sub&gt; and of (B) rapid cycling recombination genomic selection for cycles C&lt;sub&gt;1&lt;/sub&gt;, C&lt;sub&gt;2,&lt;/sub&gt; C&lt;sub&gt;3,&lt;/sub&gt; and C&lt;sub&gt;4&lt;/sub&gt;. Mean of the checks (red), and mean of the entries (blue).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2315/F7.medium.gif" width="377" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/7/2315/F7.medium.gif" width="377" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2315/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485559" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-49" class="first-child">Response to selection for grain yield from the families of (A) rapid cycling recombination genomic selection for cycles C<sub>0</sub>, C<sub>1</sub>, C<sub>2,</sub> C<sub>3,</sub> and C<sub>4</sub> and of (B) rapid cycling recombination genomic selection for cycles C<sub>1</sub>, C<sub>2,</sub> C<sub>3,</sub> and C<sub>4</sub>. Mean of the checks (red), and mean of the entries (blue).</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">The lower GY observed in C<sub>1</sub> compared to C<sub>0</sub> is explained because the best GY entries selected from C<sub>0</sub> as parents of C<sub>1</sub> were further intermated and selected based on flowering, plant, and ear height, etc. This also helped to broaden and maintain the genetic diversity observed in C<sub>1</sub> and C<sub>2</sub>, which later declined in C<sub>3</sub>. As already mentioned, the other reason of lower GY observed in C<sub>1</sub> compared to C<sub>0</sub> is that the best 50 selected families (not the random selected families) were used to represent selection cycle C<sub>0</sub> in the genetic gain evaluation study. The GY mean of the best 50 selected families is much higher than that value of the 50 random selected families.</p><p id="p-51">In terms of the prediction models used to predict the genetic values of the entries to be selected in each genomic cycle, we used the direct genotyping-by-sequencing marker as biallelic. Since a multi-parental (not a biparental) population was used, haplotype rather than biallelic marker could have been used in order to attempt to capture the whole allelic diversity. However, the problem on how to define the length of the haplotype segment in each chromosome could impose a major drawback for using this approach; different haplotypes methods exist but none of them seems to give clear superiority in terms of genomic-enabled prediction accuracy.</p></div><div id="sec-18" class="subsection"><h3>Realized genetic gains per unit of time</h3><p id="p-52">To compute the realized genetic gains per year (ton ha<sup>−1</sup> yr<sup>−1</sup>), it is necessary to account for the number of cycles per year (two cycles per year in this study), and also for the time from the initial cross to the last cycle (4.5 yr from F<sub>1</sub> development to harvesting the C<sub>4</sub> in this study). Therefore, given that grain yield from C<sub>1</sub> (8.40 ton ha<sup>−1</sup>) to C<sub>4</sub> (9.05 ton ha<sup>−1</sup>) increased by 7.74%, the average genetic gain of 0.225 ton ha<sup>−1</sup> per cycle (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) is equivalent to 0.100 ton ha<sup>−1</sup>yr<sup>−1</sup> [<em>i.e.</em>, (2 × 0.225)/4.5] under optimal conditions.</p><p id="p-53"><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Masuka <em>et al.</em> (2017a)</a> conducted a review of genetic gain studies that used conventional pedigree selection on tropical hybrid maize germplasm under optimal conditions in Sub-Saharan Africa, which gave gains of 0.109 ton ha<sup>−1</sup> yr<sup>−1</sup>. For tropical open-pollinated maize varieties, realized genetic gains reached 0.109 ton ha<sup>−1</sup> yr<sup>−1</sup> in the early maturity group, and 0.079 ton ha<sup>−1</sup> yr<sup>−1</sup> in the intermediate-to-late group (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Masuka <em>et al.</em> 2017b</a>). Therefore, the genetic gains from the RCGS observed in the MPPs used in this study (0.100 ton ha<sup>−1</sup> yr<sup>−1</sup>) are at the same or higher level than those observed in other studies under phenotypic selection but with a shorter breeding cycle. However, the 0.070 ton ha<sup>−1</sup> yr<sup>−1</sup> achieved by <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Beyene <em>et al.</em> (2015)</a> in bi-parental populations is not comparable to the results of this study because the genetic gains from RCGS in biparental populations targeted managed drought environments (not optimal environments), and the RCGS in this MPP targeted optimal environments.</p><p id="p-54">In this study, results obtained from MPPs in optimal environments reinforce the usefulness of GS-assisted recombination for achieving high genetic gains in GY. Although only two cycles per year were completed in this study (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Beyene <em>et al.</em> 2015</a>, completed three cycles per year in biparental populations), it is still time-efficient when compared to the 1.5 yr per selection cycle required for making testcrosses, phenotyping testcrosses, and conducting selection and recombination in conventional pedigree breeding.</p></div><div id="sec-19" class="subsection"><h3>Trends in genetic diversity under rapid cycling recombination GS</h3><p id="p-55">There are not many reports on the influence of RCGS on genetic variance in plant breeding. In a simulation study, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Jannink <em>et al.</em> (2010)</a> were the first to caution about the possible decline in genetic variance due to RCGS. Genetic gains in GS for stem rust in wheat were reported by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Rutkoski <em>et al.</em> (2015)</a>; genetic gains in GS were compared with gains in phenotypic selection and no differences were found. However, GS caused faster decline in genetic variance than phenotypic selection. <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Rutkoski <em>et al.</em> (2015)</a> also found significant increases in inbreeding after one and two cycles of GS as compared with C<sub>0</sub>; this increase in inbreeding was significantly greater than the expected value under random genetic drift for all populations.</p><p id="p-56">The above results seem to be in partial agreement with the findings of this study. The decrease in genetic diversity measured by the Shannon Diversity Index and the expected heterozygosity only occurred in RCGS in C<sub>3</sub>, whereas, in previous cycles, genetic diversity was very well maintained. These results may be due to the fact that C<sub>0</sub> selection was initially based on GY, and then a second selection was conducted based on flowering, maturity, and other traits after intermating; this may be one of the reasons why the genetic diversity stayed at the same level as in C<sub>0</sub>, at least in the initial recombination cycles.</p></div><div id="sec-20" class="subsection"><h3>Changes in the frequency status of markers</h3><p id="p-57">Results of this study show that only a few SNPs changed their polymorphic status after three cycles of GS. This result indicates that marker interaction (epistasis) may play an important role in complex traits (such as grain yield), in which phenotype is the result of the sum of the small effects of many genes. Only chromosome 5 showed a total of 19 clusters, with 16 of them having markers that became monomorphic (being polymorphic in their 18 parents and in the C<sub>0</sub> training population); all the other chromosomes showed a small number of clusters with markers that became monomorphic (from their original polymorphic status) after cycles of RCGS. It is likely that some of those clusters of SNP markers with changes in their polymorphic status are related to less complex traits that were selected during C<sub>0</sub> and C<sub>1</sub>, such as flowering time, and plant and ear height.</p></div><div id="sec-21" class="subsection"><h3>Conclusions</h3><p id="p-58">Results described in this study are the first report of RCGS in MPPs. A realized genetic gain of ∼2% for GY with two rapid cycles per year saves time and produces efficient genetic gains overall. The decline in genetic gains from cycle C<sub>0</sub> to C<sub>1</sub> is because the aim of selecting parents for C<sub>1</sub> was to maintain genetic diversity and the best selected families of cycle C<sub>0</sub> were used as baseline for evaluating genetic gain among cycles. The realized gain achieved in this study was 0.100 ton ha<sup>−1</sup> yr<sup>−1</sup> when only GS cycles were considered (C<sub>1</sub>–C<sub>4</sub>). Another important finding is that genetic diversity was well controlled up to C<sub>3</sub> and then declined. Although other traits were correlated with GY, they did not show any important change after three cycles of RCGS for GY. In the end, 64.3% of the markers changed their allele frequencies but never became monomorphic, and 33.99% of the markers modified their allele frequency by &lt;15%.</p><p id="p-59">The target of this study was to perform three rapid cycles per year; however, we only achieved two rapid cycles per year due to delays in the DNA preparation and genotyping turnaround time. Therefore, further studies in maize or other crops are required to confirm the promising RCGS results obtained in MPP using tropical maize lines described in this study, and examining if the length of selection cycle could be further reduced by implementing RCGS.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">The authors gratefully acknowledge the financial support from the MasAgro project funded by Mexico's Secretariat of Agriculture, Livestock, Rural Development, Fisheries and Food (SAGARPA), and the CGIAR Research Program (CRP) MAIZE.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-61"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-04"><span class="received-label">Received </span>January 4, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-05-15"><span class="accepted-label">Accepted </span>May 15, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Zhang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.1" data-doi="10.1534/genetics.114.167577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahfock</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterizing uncertainty in high-density maps from multiparental populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAhfock%26rft.auinit1%253DD.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D117%26rft.epage%253D128%26rft.atitle%253DCharacterizing%2BUncertainty%2Bin%2BHigh-Density%2BMaps%2Bfrom%2BMultiparental%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.167577%26rft_id%253Dinfo%253Apmid%252F25236453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMTE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.2" data-doi="10.2135/cropsci2008.03.0131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular markers and selection for complex traits in plants: learning from the last 20 years.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1649</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D1649%26rft.epage%253D1664%26rft.atitle%253DMolecular%2BMarkers%2Band%2BSelection%2Bfor%2BComplex%2BTraits%2Bin%2BPlants%253A%2BLearning%2Bfrom%2Bthe%2BLast%2B20%2BYears%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.03.0131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.03.0131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000259792100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Bandwagons I, too, have known.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2323</span>–<span class="cit-lpage">2332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.4" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genome-wide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.5" data-doi="10.2135/cropsci2014.07.0460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarekegne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic gains in grain yield through genomic selection in eight bi-parental maize populations under drought stress.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.07.0460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.07.0460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.6" data-doi="10.1016/j.pbi.2008.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">From mutation to MAGIC: resources for gene discovery, validation and delivery in crop plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DBeyer%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D215%26rft.epage%253D221%26rft.atitle%253DFrom%2Bmutations%2Bto%2BMAGIC%253A%2Bresources%2Bfor%2Bgene%2Bdiscovery%252C%2Bvalidation%2Band%2Bdelivery%2Bin%2Bcrop%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2008.01.002%26rft_id%253Dinfo%253Apmid%252F18295532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2008.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18295532&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255623300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.7" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.8" data-doi="10.1534/genetics.109.104935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reliability of genomic predictions across multiple populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104935%26rft_id%253Dinfo%253Apmid%252F19822733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE1NDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.9" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.10" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping-by-sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffstetter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabrera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sneller</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Optimizing training population data and validation of genomic selection for economic traits in soft winter wheat.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2919</span>–<span class="cit-lpage">2928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D2919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.12" data-doi="10.1111/j.1467-7652.2012.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A multiparent advanced generation intercross population for genetic analyses in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">826</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DHuang%26rft.auinit1%253DB.%2BE.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D826%26rft.epage%253D839%26rft.atitle%253DA%2Bmultiparent%2Badvanced%2Bgeneration%2Binter-cross%2Bpopulation%2Bfor%2Bgenetic%2Banalysis%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00702.x%26rft_id%253Dinfo%253Apmid%252F22594629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22594629&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307051700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.13" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo3OiI5LzIvMTY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.14" data-doi="10.1534/genetics.114.161943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camisan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Usefulness of multiparental populations of maize (<em>Zea mays</em> L.) for genome-based prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLehermeier%26rft.auinit1%253DC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D16%26rft.atitle%253DUsefulness%2Bof%2BMultiparental%2BPopulations%2Bof%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bfor%2BGenome-Based%2BPrediction%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161943%26rft_id%253Dinfo%253Apmid%252F25236445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk4LzEvMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.15" data-doi="10.2135/cropsci2012.02.0112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">H.-J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide selection <em>vs.</em> marker-assisted recurrent selection to improve grain yield and stover-quality traits for cellulosic ethanol in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.02.0112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.02.0112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masuka</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magorokosho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Labuschagne</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Gains in maize genetic improvement in Eastern and Southern Africa: I. CIMMYT hybrid breeding pipeline.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masuka</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magorokosho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bänziger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Gains in maize genetic improvement in Eastern and Southern Africa: II. CIMMYT open-pollinated variety breeding pipeline.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Power of tests for QTL detection using replicated progenies derived from a diallel cross.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">1014</span>–<span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D86%26rft.spage%253D1014%26rft_id%253Dinfo%253Apmid%252F24194011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24194011&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic gain from phenotypic and genomic selection for quantitative resistance to stem rust of wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: 2.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">J. A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>: 3.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.22" data-doi="10.1534/genetics.114.169995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thepot</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Restoux</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldringer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hospital</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouache</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Efficiently tracking selection in a multiparental population: the case of earliness in wheat.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169995%26rft_id%253Dinfo%253Apmid%252F25406468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzIvNjA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic measures of relationship and inbreeding.</span> <abbr class="cit-jnl-abbrev">Interbull Bull.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterbull%2BBull.%26rft.volume%253D37%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.24" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.25" data-doi="10.1038/sj.hdy.6800763"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verhoeven</span>  <span class="cit-name-given-names">K. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Using mating designs to uncover QY+TL and the genetic architecture of complex traits.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DVerhoeven%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D96%26rft.issue%253D2%26rft.spage%253D139%26rft.epage%253D149%26rft.atitle%253DUsing%2Bmating%2Bdesigns%2Bto%2Buncover%2BQTL%2Band%2Bthe%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800763%26rft_id%253Dinfo%253Apmid%252F16304603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304603&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234843500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.7.2315.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vivek</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishna</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vengadessan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaidi</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Use of Genomic estimated breeding values results in rapid genetic gains for drought tolerance in maize.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span> (in press).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">San Vicente</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhliwayo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costich</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Molecular characterization of CIMMYT maize inbred lines with genotyping-by-sequencing SNPs.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.28" data-doi="10.1038/hdy.2014.99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS SNPs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D291%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.99%26rft_id%253Dinfo%253Apmid%252F25407079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25407079&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2315.29" data-doi="10.1534/genetics.108.098277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098277%26rft_id%253Dinfo%253Apmid%252F19299342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/7/2315.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2305" title="Essential Function of the Serine Hydroxymethyl Transferase (SHMT) Gene During Rapid Syncytial Cell Cycles in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2327" title="A Coding Variant in the Gene Bardet-Biedl Syndrome 4 (BBS4) Is Associated with a Novel Form of Canine Progressive Retinal Atrophy" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/7"><span>Volume 7 Issue 7, July 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/7.cover-source.jpg?itok=eAMfPilr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2315" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XVbuPU2cyWPSLrNrHumR57jaX-bN9U2zgYficCV6Hhs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/7/2315" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/7/2315" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/7/2315" class="active" data-icon-position="" data-hide-link-title="0">Rapid Cycling Genomic Selection in a Multiparental Tropical Maize Population</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485325" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-4DuApsYNhF_jNA7bGuV7xK5afs5UbqtUwJnbKs62VkA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2315.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/7/2315" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-MOP7LbWzqXuXai-adxuM2m5XeX8gpiRXVi91lZ8TJdI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/7/2315" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2315.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485325" id="node485325--2" data-pisa="ggg;7/7/2315" data-pisa-master="ggg;g3.117.043141" data-apath="/ggg/7/7/2315.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Cycling Genomic Selection in a Multiparental Tropical Maize Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xuecai Zhang</span>, <span class="highwire-citation-author" data-delta="1">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="2">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="3">Michael Olsen</span>, <span class="highwire-citation-author" data-delta="4">Edward Buckler</span>, <span class="highwire-citation-author" data-delta="5">Gary Atlin</span>, <span class="highwire-citation-author" data-delta="6">Boddupalli M. Prasanna</span>, <span class="highwire-citation-author" data-delta="7">Mateo Vargas</span>, <span class="highwire-citation-author" data-delta="8">Félix San Vicente</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2315-2326;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043141 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485325--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xuecai Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Maize Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuecai%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuecai%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Colegio de Postgraduados, CP 56230, Montecillos, 56230 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael Olsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Nairobi 1041-00621, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BOlsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Olsen%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BOlsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Edward Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>United States Department of Agriculture, Agricultural Research Service, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gary Atlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Bill and Melinda Gates Foundation, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BAtlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Atlin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BAtlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Boddupalli M. Prasanna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Nairobi 1041-00621, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boddupalli%2BM.%2BPrasanna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prasanna%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoddupalli%2BM.%2BPrasanna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mateo Vargas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Universidad Autónoma Chapingo, 56230 Texcoco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mateo%2BVargas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vargas%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMateo%2BVargas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Félix San Vicente </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Maize Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=F%C3%A9lix%2BSan%2BVicente%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=San%20Vicente%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AF%25C3%25A9lix%2BSan%2BVicente%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:f.sanvicente@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">f.sanvicente@cgiar.org</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:f.sanvicente@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">f.sanvicente@cgiar.org</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/7/2315" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/7/2315&amp;title=Rapid%20Cycling%20Genomic%20Selection%20in%20a%20Multiparental%20Tropical%20Maize%20Population" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/7/2315&amp;title=Rapid%20Cycling%20Genomic%20Selection%20in%20a%20Multiparental%20Tropical%20Maize%20Population" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/7/2315&amp;title=Rapid%20Cycling%20Genomic%20Selection%20in%20a%20Multiparental%20Tropical%20Maize%20Population" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/7/2315&amp;text=Rapid%20Cycling%20Genomic%20Selection%20in%20a%20Multiparental%20Tropical%20Maize%20Population" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/7/2315&amp;title=Rapid%20Cycling%20Genomic%20Selection%20in%20a%20Multiparental%20Tropical%20Maize%20Population" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/7/2315&amp;t=Rapid%20Cycling%20Genomic%20Selection%20in%20a%20Multiparental%20Tropical%20Maize%20Population" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/7/2315&amp;title=Rapid%20Cycling%20Genomic%20Selection%20in%20a%20Multiparental%20Tropical%20Maize%20Population" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/7/2315&amp;title=Rapid%20Cycling%20Genomic%20Selection%20in%20a%20Multiparental%20Tropical%20Maize%20Population" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485325/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485325" id="node485325--3" data-pisa="ggg;7/7/2315" data-pisa-master="ggg;g3.117.043141" data-apath="/ggg/7/7/2315.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Cycling Genomic Selection in a Multiparental Tropical Maize Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xuecai Zhang</span>, <span class="highwire-citation-author" data-delta="1">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="2">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="3">Michael Olsen</span>, <span class="highwire-citation-author" data-delta="4">Edward Buckler</span>, <span class="highwire-citation-author" data-delta="5">Gary Atlin</span>, <span class="highwire-citation-author" data-delta="6">Boddupalli M. Prasanna</span>, <span class="highwire-citation-author" data-delta="7">Mateo Vargas</span>, <span class="highwire-citation-author" data-delta="8">Félix San Vicente</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2315-2326;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043141 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485325--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xuecai Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Maize Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuecai%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuecai%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Colegio de Postgraduados, CP 56230, Montecillos, 56230 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael Olsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Nairobi 1041-00621, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BOlsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Olsen%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BOlsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Edward Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>United States Department of Agriculture, Agricultural Research Service, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gary Atlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Bill and Melinda Gates Foundation, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BAtlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Atlin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BAtlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Boddupalli M. Prasanna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Nairobi 1041-00621, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boddupalli%2BM.%2BPrasanna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prasanna%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoddupalli%2BM.%2BPrasanna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mateo Vargas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Universidad Autónoma Chapingo, 56230 Texcoco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mateo%2BVargas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vargas%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMateo%2BVargas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Félix San Vicente </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Maize Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=F%C3%A9lix%2BSan%2BVicente%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=San%20Vicente%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AF%25C3%25A9lix%2BSan%2BVicente%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:f.sanvicente@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">f.sanvicente@cgiar.org</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:f.sanvicente@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">f.sanvicente@cgiar.org</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2315&amp;rft.epage=2326&amp;rft.atitle=Rapid%20Cycling%20Genomic%20Selection%20in%20a%20Multiparental%20Tropical%20Maize%20Population&amp;rft.volume=7&amp;rft.issue=7&amp;rft.date=2017-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhang%2C+Xuecai&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=Burgue%C3%B1o%2C+Juan&amp;rft.au=Olsen%2C+Michael&amp;rft.au=Buckler%2C+Edward&amp;rft.au=Atlin%2C+Gary&amp;rft.au=Prasanna%2C+Boddupalli+M.&amp;rft.au=Vargas%2C+Mateo&amp;rft.au=San+Vicente%2C+F%C3%A9lix&amp;rft.au=Crossa%2C+Jos%C3%A9"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485325/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485325/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485325/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485325/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485325/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485325/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485325/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485325/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485325/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485325/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485325/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485325/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/7/2315" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/7/2315.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/7/2315.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/7/2315.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
