<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/7/2259" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/7/2259.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/7/2259.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/7/2259.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/7/2259.atom" />
<meta name="HW.pisa" content="ggg;7/7/2259" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Using RNA-Seq for Genomic Scaffold Placement, Correcting Assemblies, and Genetic Map Creation in a Common Brassica rapa Mapping Population" />
<meta name="DC.Identifier" content="10.1534/g3.117.043000" />
<meta name="DC.Date" content="2017-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Markelz et al." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Brassica rapa is a model species for agronomic, ecological, evolutionary, and translational studies. Here, we describe high-density SNP discovery and genetic map construction for a B. rapa recombinant inbred line (RIL) population derived from field collected RNA sequencing (RNA-Seq) data. This high-density genotype data enables the detection and correction of putative genome misassemblies and accurate assignment of scaffold sequences to their likely genomic locations. These assembly improvements represent 7.1–8.0% of the annotated B. rapa genome. We demonstrate how using this new resource leads to a significant improvement for QTL analysis over the current low-density genetic map. Improvements are achieved by the increased mapping resolution and by having known genomic coordinates to anchor the markers for candidate gene discovery. These new molecular resources and improvements in the genome annotation will benefit the Brassicaceae genomics community and may help guide other communities in fine-tuning genome annotations." />
<meta name="DC.Contributor" content="R. J. Cody Markelz" />
<meta name="DC.Contributor" content="Michael F. Covington" />
<meta name="DC.Contributor" content="Marcus T. Brock" />
<meta name="DC.Contributor" content="Upendra K. Devisetty" />
<meta name="DC.Contributor" content="Daniel J. Kliebenstein" />
<meta name="DC.Contributor" content="Cynthia Weinig" />
<meta name="DC.Contributor" content="Julin N. Maloof" />
<meta name="article:published_time" content="2017-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Using RNA-Seq for Genomic Scaffold Placement, Correcting Assemblies, and Genetic Map Creation in a Common Brassica rapa Mapping Population" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Brassica rapa&lt;/i&gt; is a model species for agronomic, ecological, evolutionary, and translational studies. Here, we describe high-density SNP discovery and genetic map construction for a &lt;i&gt;B. rapa&lt;/i&gt; recombinant inbred line (RIL) population derived from field collected RNA sequencing (RNA-Seq) data. This high-density genotype data enables the detection and correction of putative genome misassemblies and accurate assignment of scaffold sequences to their likely genomic locations. These assembly improvements represent 7.1–8.0% of the annotated &lt;i&gt;B. rapa&lt;/i&gt; genome. We demonstrate how using this new resource leads to a significant improvement for QTL analysis over the current low-density genetic map. Improvements are achieved by the increased mapping resolution and by having known genomic coordinates to anchor the markers for candidate gene discovery. These new molecular resources and improvements in the genome annotation will benefit the Brassicaceae genomics community and may help guide other communities in fine-tuning genome annotations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/07/01" />
<meta name="citation_mjid" content="ggg;7/7/2259" />
<meta name="citation_id" content="7/7/2259" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/7/2259" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/7/2259.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/7/2259.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/7/2259.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.043000" />
<meta name="citation_pmid" content="28546385" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2259" />
<meta name="citation_lastpage" content="2270" />
<meta name="citation_author" content="R. J. Cody Markelz" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author" content="Michael F. Covington" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0555-6409" />
<meta name="citation_author" content="Marcus T. Brock" />
<meta name="citation_author_institution" content="Department of Botany, University of Wyoming, Laramie, Wyoming 82072" />
<meta name="citation_author" content="Upendra K. Devisetty" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4686-7235" />
<meta name="citation_author" content="Daniel J. Kliebenstein" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California at Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5759-3175" />
<meta name="citation_author" content="Cynthia Weinig" />
<meta name="citation_author_institution" content="Department of Botany, University of Wyoming, Laramie, Wyoming 82072" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2833-9316" />
<meta name="citation_author" content="Julin N. Maloof" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_email" content="jnmaloof@ucdavis.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9623-2599" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=R. L. Baker;citation_author=W. F. Leong;citation_author=M. T. Brock;citation_author=R. J. C. Markelz;citation_author=M. F. Covington;citation_title=Modeling development and quantitative trait mapping reveal independent genetic modules for leaf size and shape.;citation_pages=257-268;citation_volume=208;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. T. Brock;citation_author=J. M. Dechaine;citation_author=F. L. Iniguez-Luy;citation_author=J. N. Maloof;citation_author=J. R. Stinchcombe;citation_title=Floral genetic architecture: an examination of QTL architecture underlying floral (Co)variation across environments.;citation_volume=186;citation_year=2010;citation_pmid=20837996;citation_doi=10.1534/genetics.110.119982" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Buckler;citation_author=T. L. Phelps-Durr;citation_author=C. S. K. Buckler;citation_author=R. K. Dawe;citation_author=J. F. Doebley;citation_title=Meiotic Drive of Chromosomal Knobs Reshaped the Maize Genome;citation_pages=415-426;citation_volume=153;citation_year=1999;citation_issue=1;citation_pmid=10471723" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. Chalhoub;citation_author=F. Denoeud;citation_author=S. Liu;citation_author=I. A. P. Parkin;citation_author=H. Tang;citation_title=Early allopolyploid evolution in the post-Neolithic Brassica napus oilseed genome;citation_pages=950-953;citation_volume=345;citation_year=2014;citation_issue=6199;citation_pmid=25146293;citation_doi=10.1126/science.1253435" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Plant Sci;citation_author=F. Cheng;citation_author=J. Wu;citation_author=L. Fang;citation_author=X. Wang;citation_title=Syntenic gene analysis between Brassica rapa and other Brassicaceae species.;citation_pages=198-198;citation_volume=3;citation_year=2012;citation_pmid=22969786" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=F. Cheng;citation_author=T. Mandáková;citation_author=J. Wu;citation_author=Q. Xie;citation_author=M. A. Lysak;citation_title=Deciphering the diploid ancestral genome of the Mesohexaploid Brassica rapa.;citation_volume=25;citation_year=2013;citation_pmid=23653472;citation_doi=10.1105/tpc.113.110486" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=J. M. Dechaine;citation_author=J. A. Johnston;citation_author=M. T. Brock;citation_author=C. Weinig;citation_title=Constraints on the evolution of adaptive plasticity: costs of plasticity to density are expressed in segregating progenies.;citation_pages=874-882;citation_volume=176;citation_year=2007;citation_issue=4;citation_pmid=17822398;citation_doi=10.1111/j.1469-8137.2007.02210.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=J. M. Dechaine;citation_author=M. T. Brock;citation_author=C. Weinig;citation_title=QTL architecture of reproductive fitness characters in Brassica rapa.;citation_pages=1;citation_volume=14;citation_year=2014;citation_pmid=24387633;citation_doi=10.1186/1471-2229-14-1" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=U.K. Devisetty;citation_author=M. F. Covington;citation_author=A. V. Tat;citation_author=S. Lekkala;citation_author=J. N. Maloof;citation_title=Polymorphism identification and improved genome annotation of Brassica rapa through deep RNA sequencing.;citation_pages=2065-2078;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_title=Vegetable Brassicas and Related Crucifers.;citation_author=G. Dixon;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. E. Edwards;citation_author=B. E. Ewers;citation_author=D. G. Williams;citation_author=Q. Xie;citation_author=P. Lou;citation_title=The genetic architecture of ecophysiological and circadian traits in Brassica rapa.;citation_volume=189;citation_year=2011;citation_pmid=21750258;citation_doi=10.1534/genetics.110.125112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Fang;citation_author=T. Pyhäjärvi;citation_author=A. L. Weber;citation_author=R. K. Dawe;citation_author=J. C. Glaubitz;citation_title=Megabase-scale inversion polymorphism in the wild ancestor of maize.;citation_volume=191;citation_year=2012;citation_pmid=22542971;citation_doi=10.1534/genetics.112.138578" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. Fulop;citation_author=A. Ranjan;citation_author=I. Ofner;citation_author=M. F. Covington;citation_author=D. H. Chitwood;citation_title=A new advanced backcross tomato population enables high resolution leaf QTL mapping and gene identification.;citation_pages=3169-3184;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. P. Hammond;citation_author=S. Mayes;citation_author=H. C. Bowen;citation_author=N. S. Graham;citation_author=R. M. Hayden;citation_title=Regulatory hotspots are associated with plant gene expression under varying soil phosphorus supply in Brassica rapa.;citation_volume=156;citation_year=2011;citation_pmid=21527424;citation_doi=10.1104/pp.111.175612" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Harushima;citation_author=M. Yano;citation_author=A. Shomura;citation_author=M. Sato;citation_author=T. Shimano;citation_title=A High-Density Rice Genetic Linkage Map with 2275 Markers Using a Single F2 Population;citation_pages=479-494;citation_volume=148;citation_year=1998;citation_issue=1;citation_pmid=9475757" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=W. Haupt;citation_author=T. C. Fischer;citation_author=S. Winderl;citation_author=P. Fransz;citation_author=R. A. Torres-Ruiz;citation_title=The centromere1 (CEN1) region of Arabidopsis thaliana: architecture and functional impact of chromatin.;citation_pages=285-296;citation_volume=27;citation_year=2001;citation_issue=4;citation_pmid=11532174;citation_doi=10.1046/j.1365-313x.2001.01087.x" />
<meta name="citation_reference" content="citation_journal_title=Data Brief;citation_author=Z. He;citation_author=F. Cheng;citation_author=Y. Li;citation_author=X. Wang;citation_author=I. A. P. Parkin;citation_title=Construction of Brassica A and C genome-based ordered pan-transcriptomes for use in rapeseed genomic research.;citation_pages=357-362;citation_volume=4;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. L. Iniguez-Luy;citation_author=L. Lukens;citation_author=M. W. Farnham;citation_author=R. M. Amasino;citation_author=T. C. Osborn;citation_title=Development of public immortal mapping populations, molecular markers and linkage maps for rapid cycling Brassica rapa and B. oleracea.;citation_pages=31-43;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19784615;citation_doi=10.1007/s00122-009-1157-4" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Plant Genomics;citation_author=S. Kumar;citation_author=T. W. Banks;citation_author=S. Cloutier;citation_title=SNP discovery through next-generation sequencing and its applications.;citation_pages=831460;citation_volume=2012;citation_year=2012;citation_pmid=23227038" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. Li;citation_author=C. F. Quiros;citation_title=Sequence-related amplified polymorphism (SRAP), a new marker system based on a simple PCR reaction: its application to mapping and gene tagging in Brassica.;citation_pages=455-461;citation_volume=103;citation_year=2001;citation_doi=10.1007/s001220100570" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Liu;citation_author=Y. Liu;citation_author=X. Yang;citation_author=C. Tong;citation_author=D. Edwards;citation_title=The Brassica oleracea genome reveals the asymmetrical evolution of polyploid genomes.;citation_pages=3930;citation_volume=5;citation_year=2014;citation_pmid=24852848;citation_doi=10.1038/ncomms4930" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=P. Lou;citation_author=Q. Xie;citation_author=X. Xu;citation_author=C. E. Edwards;citation_author=M. T. Brock;citation_title=Genetic architecture of the circadian clock and flowering time in Brassica rapa.;citation_pages=397-409;citation_volume=123;citation_year=2011;citation_issue=3;citation_pmid=21505830;citation_doi=10.1007/s00122-011-1592-x" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=P. Lou;citation_author=J. Wu;citation_author=F. Cheng;citation_author=L. G. Cressman;citation_author=X. Wang;citation_title=Preferential retention of circadian clock genes during diploidization following whole genome triplication in Brassica rapa.;citation_volume=24;citation_year=2012;citation_pmid=22685167;citation_doi=10.1105/tpc.112.099499" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=D. B. Lowry;citation_author=J. H. Willis;citation_title=A widespread chromosomal inversion polymorphism contributes to a major life-history transition, local adaptation, and reproductive isolation.;citation_pages=e1000500;citation_volume=8;citation_year=2010;citation_pmid=20927411;citation_doi=10.1371/journal.pbio.1000500" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=I. A. Parkin;citation_author=C. Koh;citation_author=H. Tang;citation_author=S. J. Robinson;citation_author=S. Kagale;citation_title=Transcriptome and methylome profiling reveals relics of genome dominance in the mesopolyploid Brassica oleracea.;citation_pages=R77;citation_volume=15;citation_year=2014;citation_pmid=24916971;citation_doi=10.1186/gb-2014-15-6-r77" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. D. Sherman;citation_author=S. M. Stack;citation_title=Two-dimensional spreads of synaptonemal complexes from solanaceous plants. VI. High-resolution recombination nodule map for tomato (Lycopersicon esculentum).;citation_pages=683-708;citation_volume=141;citation_year=1995;citation_issue=2;citation_pmid=8647403" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=H. Wang;citation_author=J. Wu;citation_author=S. Sun;citation_author=B. Liu;citation_author=F. Cheng;citation_title=Glucosinolate biosynthetic genes in Brassica rapa.;citation_pages=135-142;citation_volume=487;citation_year=2011;citation_pmid=21835231;citation_doi=10.1016/j.gene.2011.07.021" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Wang;citation_author=H. Wang;citation_author=J. Wang;citation_author=R. Sun;citation_author=J. Wu;citation_title=The genome of the mesopolyploid crop species Brassica rapa.;citation_pages=1035-1039;citation_volume=43;citation_year=2011;citation_issue=10;citation_pmid=21873998;citation_doi=10.1038/ng.919" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Yang;citation_author=D. Liu;citation_author=X. Wang;citation_author=C. Ji;citation_author=F. Cheng;citation_title=The genome sequence of allopolyploid Brassica juncea and analysis of differential homoeolog gene expression influencing selection.;citation_pages=1225-1232;citation_volume=48;citation_year=2016" />
<meta name="twitter:title" content="Using RNA-Seq for Genomic Scaffold Placement, Correcting Assemblies, and Genetic Map Creation in a Common Brassica rapa Mapping Population" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="twitter:description" content="Brassica rapa is a model species for agronomic, ecological, evolutionary, and translational studies. Here, we describe high-density SNP discovery and genetic map construction for a B. rapa recombinant inbred line (RIL) population derived from field collected RNA sequencing (RNA-Seq) data. This high-density genotype data enables the detection and correction of putative genome misassemblies and accurate assignment of scaffold sequences to their likely genomic locations. These assembly improvements represent 7.1–8.0% of the annotated B. rapa genome. We demonstrate how using this new resource leads to a significant improvement for QTL analysis over the current low-density genetic map. Improvements are achieved by the increased mapping resolution and by having known genomic coordinates to anchor the markers for candidate gene discovery. These new molecular resources and improvements in the genome annotation will benefit the Brassicaceae genomics community and may help guide other communities in fine-tuning genome annotations." />
<meta name="og-title" property="og:title" content="Using RNA-Seq for Genomic Scaffold Placement, Correcting Assemblies, and Genetic Map Creation in a Common Brassica rapa Mapping Population" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/7/2259" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Brassica rapa is a model species for agronomic, ecological, evolutionary, and translational studies. Here, we describe high-density SNP discovery and genetic map construction for a B. rapa recombinant inbred line (RIL) population derived from field collected RNA sequencing (RNA-Seq) data. This high-density genotype data enables the detection and correction of putative genome misassemblies and accurate assignment of scaffold sequences to their likely genomic locations. These assembly improvements represent 7.1–8.0% of the annotated B. rapa genome. We demonstrate how using this new resource leads to a significant improvement for QTL analysis over the current low-density genetic map. Improvements are achieved by the increased mapping resolution and by having known genomic coordinates to anchor the markers for candidate gene discovery. These new molecular resources and improvements in the genome annotation will benefit the Brassicaceae genomics community and may help guide other communities in fine-tuning genome annotations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
    <title>Using RNA-Seq for Genomic Scaffold Placement, Correcting Assemblies, and Genetic Map Creation in a Common Brassica rapa Mapping Population | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"V2uoBUBHtctvF0n8gBia4GG0a_f3Ju6Ikys-FPIbCzo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485324","apath":"\/ggg\/7\/7\/2259.atom","pisa":"ggg;7\/7\/2259","ac":{"\/ggg\/7\/7\/2259.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/7\/2259.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2259"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2259"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/7\/2259":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jMDZI1zZNK\\\\\\\/guGZppoXQy2rRDfkO\\\\\\\/RuSZCQgXUcz69i2r0cj9MiO0IIiCgADAbriTrt6jvthiX+cLWKS8uGe3r0ShfO553ZKf1R6g+7sv3zCgA9gjxKzwlpD5u+CUbk0+n\\\\\\\/6B88i\\\\\\\/Vez1tcr8PjYHQHlSNWxlDHf0ArKzNIjYa4AkXMw9oTLh6hqURwAeC635GSVNV\\\\\\\/f5nzstnXI0XbvIQRUGxkHddn3jQD7lgo2ysGXDsqpsC57SxjPRkT1Fb66pvBqd3SncsQZZd2dQBQPVYDjNJ1i+eFbx3n9SJPobvyRt3Eri0OwxwAO89mPWsGxRLv7jjew6QnI5ak9JOFImKbXQLeL1eXoX+eRPk5A+62IVrDqCM5ITwUPEv5+R\\\\\\\/zT5OGu40kdqx4nLkeY6F0awp+lDy3rpQ49a1hO5VzmnnYy5lhc4RsU5l\\\\\\\/aDi\\\\\\\/9WXT3d1OvmhULsmvjGONzmWNMxKVwHYPY1iAO5ElTdRDftuFJyyDCre0vYwcJgImnW\\\\\\\/5l5J\\\\\\\/1UxjaxsyqYMZ04\\\\\\\/3C6pNihFsnO1vDES3lRMuCBrBau35U\\\\\\\/kVq6ONo1f+aT\\\\\\\/zw8Q1VOWZXSpulEKN60j+5xugjAp5CZdTNhl4GbQB4mzd7Iv8iHX14Lj\\\\\\\/D3by6zgthXHYIwZeVuVv5JUsKEKmEtDpPYkpWqdIrbFVmoEdXwoDRSGiFExPcP8AVuhjuMDbKl2gm7+szU+PYPzIO1PsrZS\\\\\\\/CK\\\\\\\/6LpF1h9No53umwQqfICbKFtr3xRHTh1DrhrSTR5fo48C37RrtUlliuCZQ\\\\\\\/fd41aNJwS2cyZnqW9PzP+VifJ9CLA79ZV8nhbDH9WWt37LTaC3qHbyftQ5B8e2va6+o24QRYk+FJO6GzsbuJvOAJSqWfuSMoK7kkbTuvCvqLpEl4qJvUtm5NzQDZGQTUml6i0na2MpV2EkdkB\\\\\\\/HDszAcuYOhYs2VTGqzfDKqZxd8HnUeAZJmEscmExNo\\\\\\\/SsxIo0VOtWyecorZE9LT5SIL\\\\\\\/lgGUv+eqoY151x+qCM2knRRUnnz1IfBcg5CgTOkk2NewK0WOcBKTYXqUuVd6wkHmUJSxqiWefm5uKjpgbS5WSCJqYunc6WhmZ\\\\\\\/t8H86mB7wERKBpVU3crFQnDMAD35AinOPxhekuwdSjEwBnhjZbfMOfIM4ftj3gNJXkggDhJBKbkqMhMEAIFLKnY5\\\\\\\/mjw3PcDfWyjGZUfMgQg8dnK2YHT\\\\\\\/0qz\\\\\\\/iB0CexZg0n7kqIiCwEp7vXgjjTf+UIslkFI7qpqZz5aAAmGjZzEmrl6JyxQ\\\u0022,\\\u0022iv\\\u0022:\\\u0022iZ2xuCbmvr23RvQ4YOkVyw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229787e3d83605fef57422d51d6d4e1d307f152eee29312786e1eab4df38906027\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tImAS0RfkKOAXDkZ1zknuyUy6J3Zf1BRLBxJTeKs9vfUDZePOcL7WjzJvHQYYN7cNs78WtRjxYgivajSJRXAHB0zUFE3bykVwlzMRbhhkd3CjbrEGXge7ZThuJa3JVvogi8Wqy+tKy3cwlJIPU3rXUQ\\\\\\\/r8MRyV6HkP\\\\\\\/CgtYCNb1LGq6npcuaU7SGGLixK\\\\\\\/9dVZQQC7O8l9j55IVimlZv5kLEYiJprzMFiyMbKGQF0C\\\\\\\/JMCvluBDLc7AMOkJTuPKsdUQ6xS8ibkau6k\\\\\\\/MPgHpiaS+j2bDNWiR8DJB8J5iQ47e2B2dy1QAYRxnZqjNamcAMXfZ2ohWa6izWTU+zthmzehNOthV7UVl08FTxlr17Uy62tcdz2raK0hz3lbcp4f56hKiMlc8msbk7pXg+x1UGavkcSHCb\\\\\\\/k\\\\\\\/ck4AKlavPKCe+w0gmk3ISqL\\\\\\\/etWjqeW7PAw818akZwI6UyauSgiAGU6PCq8gM1g7rB8E1EzWsk4c0HQCj4XD4WTMfziadM\\\\\\\/P5+QTConqe3JWQKjhvkRiTbe1USbWferVQtdPTOwqPiLwe+6kbVhGksItd10\\\\\\\/jyotN+NOa8cjTLm0dCUwEx7WxCYBLhbkwj+WC5y7pevjOMr1HbxEOdPxKiMSSsbCbzeGLxCDcNofsKLbpMEJFpzj\\\\\\\/\\\\\\\/Pwr2DfXMYHVbmJCdi9+4AVHJNTQW1xkU1\\\\\\\/WFcSMTLpeZ7GG7HjThXVEhuHKyK28Xog+7jKES1YRXykh5+nODpMDeov83pkajdUvpvicGGIXHNzxSzhQKrMw0HecVhcK7xt+PU6ELGGAf2ZtDHxJ1Pbgn+lhqTqTxhJZDVsvmBv8g+v0ZavZx1yn+nZuiBpO9K0BFtuca8lqbOfKnoyIHYLv+Jq9N0i7mWUcTVVzR5NCMLSzfeOEIhwL+E1YcED\\\\\\\/ualTLAoAVrIDwok3Akkz21XAiqeQGvhurZz0irByLZqRy7vInIljtEwB8CzpOu\\\\\\\/8HWs1Pu6CRUn6wPp7ZxqAqBK\\\\\\\/6GX64zGjcYeN69xl5hQ2V7dTLJjyaVAJsg7OBl7aMtnnHICu4beG12siZjgUCKIiQjk4nLxWHqwJXUiVOapSoXhAlqPqY\\\\\\\/vGA1diXkNo2gMr6XAT72YuLnj1cRAQtNYGePsQdT+DFgbmyD\\\\\\\/vAlcLwwJ6zdHxSUQHq6n7FfipuGvBePW2lOnTC44in4QXJ4tkYRGBF+JlPVB24PfVk22yHYd2ZFM8zq1\\\\\\\/xxroa6af\\\\\\\/gq7mg+ljOOtZsNVFiF0vfqiRu1Z94SfRGbdwdpBm4PDJ+kqUYz9loudCC2wHeTz8K0ehvjrEVA1cAdjoOwdco7w2EWe0bBAzUGqPW7VHz5Eql9yz+LFlwS\\\\\\\/C+gQnsH5Eneq0U1FmBvgULOE8y+hh3mcj5mYHooTK19mPJxyeJZWb\\\\\\\/dSPZ9e5WEWCzf3NinMZ2be2zOIeW8+dP7NsVd1KeLVBhVX9baMPdDkbN8gFUvLcAyyW7SjU9s\\\\\\\/ifUNOJIwxZxmVXGhQAbDO\\\\\\\/LnwJoYx3gvHXc6P1s5aLfuLoxLZsbIFLlU9SoSErpkjERxUzn\\\\\\\/2UFBcnf+k0BhmrgP3vpKdWnYOmk89KnSCVMLQXUhtVY1F4xPCG+krrMngqlacrO5G+qg2ge6LTnlFQedRBsGEcTXThqp5uU5E4CsvH0xrALtTURV8dryocLm0CT+SIZzKJCUGsNomwEAM0nJQGP3QIIc\\\\\\\/\\\\\\\/UUReZ7RwKRoMyIVNthl8N2OeGEs1OvVOV5Sc1bbiSj\\\\\\\/oLKBTbho4S4CcOUXZCQowRGcd1nEvpyaKqYZWrSmdTmpPDDdHPnxvr1j3DznC8q+pRERLo0m\\\\\\\/540UFszP4Bs3cTq7E7a08MqQbsMx6mlgUk+BMuXeWqkTmFkkgbQUCvcFsSg0abiqiMAPak6Bg5bQS5m7qqBaNXio5B6otNF3XVcheGsjt202EztoNoy9hdy8728de\\\\\\\/asWfMJ1ZxBUiMkrTNz74aKQfSIp9IyjHtASD9Tl0V9Ai3KX4Jm2LiGE61oy\\\\\\\/WDkNNVL6AVPeKWC89RHj\\\\\\\/p\\\\\\\/xIUj8FME47KNZNXoJ8Xjo20Dwqx8dKKVbZUmhV6anyUGV6ipaCs3DMA0shz6EPCu432q9VLBgI8vxXAhKLF0d9\\\\\\\/8qztVRY4HCJwQ3nxgJ7rhH+zu2BbuKU32uFJsVW+YLYKIgOyHJKXADqib1JnYrqxH7FXZOQo8lwrZlu00o+Lej40RVGV+KiBS77BBG7W5r7W9QMaPBsE3Tqr\\\\\\\/87BoO3FJRszD50AIW8B19uuaQTFytSwlesF9BOXmqbAYt9n5ca8sxiFch0F45gT5yYHFQp\\\\\\\/DQZwt\\\\\\\/gixAOslC\\\\\\\/iL9MoiKag\\\\\\\/Sy4iyMkbcScF7OUuzrxascb2GTn1xwcnUcKCdjS9DWmEQB0GQG2oRPz+qzRwUkhFYkOqc7aHun+etDEgE3B+zNxx2dXK20JJdFWh0fod5wu2RYrLVuWHN8jOIuu+qBnEWwzBoLsNrYeXp8ojXxHrHsX6tryOSi6vT4FTAEZ4TkW0ZQ95TkvOAkhB1Dfd7yMt6AigG2+jR\\\\\\\/4VTVwR2k4TvRSQ7DueNomMud8ZNTcyWTDQUmO9dBv2G5oYCXoGIEnplmOdqzYbrYTOvbD\\\\\\\/dCqIFhwAEdY97IOmkEmGC7Me7oyxtMj1Rows8vmKPDsEDkWt7FOYgqay2Djj7vjSzUAn7Muo9JPiMuiCuM2thWDdBuU4zWoVpn\\\\\\\/R9Iq\\\\\\\/cT1N\\\\\\\/BRkYJ7J4PAtgA1wj5besiip3Cru3DwNQ2qUIFrNQ9LPgIwNuxSkhqmQLqPi7x4W32AXj+uFIFwVdRbWd\\\\\\\/xCCB7FFe1AFECxJD0lZWUepz5L0rFZq9pVbAyr7nxvAFt9a3kI4QwK19fBENWa6+XwlnixbSnDpFCb9zfj8\\\\\\\/jK9ndxv+Oz3W4SjAjzuBE5CC3Voo\\\\\\\/6K9wT9lYkvvuJaAA4dX6yfNO8\\\\\\\/9RBh5YCsNv\\\\\\\/hJcmsRPcO94xV8B7cp6XCaM4HkFqyt7yfjlBah5snptputQ2HGomMgGDapzDv42g6F35UvewWEW64NCqK4ELFrfrr3xu9PSVyV+ZVgBKQ4+EkILI0qAtzeRqZF7Q87CDeE3VjOpinImOCMWF04\\\\\\\/NZ8jEKCCZv6a1+3V57kt5w2L3xOUPjCJODqFKGoYJXL8EB11TRpgZp7nCE8lyk9Cp60UhssbsJmIAL6eA2LpMloc\\\\\\\/GVwPyp6nHEH4Ul9GOY4WOeZ25lh0KAAjc2OWKFCN0hEIBOM5drXR7y44B30K27eC6AViNyHOQSy2y\\\\\\\/7uKmD4G\\\\\\\/tBw\\\\\\\/8DwlnT6b09ZXfSQueiAY+OLYSvXI\\\\\\\/fj\\\\\\\/GuDINx21wR1OXu3sQirrxK4uirocSJjffvngsSJmVyIwNJvbtyEdINTddssEczleCZHQdq0RvcdlWchd+mdP1yket3PY+7rmcldSppbJA2dyZKC3KZchTg+gkLqpJPa+jeXhNwZgi4f6fHQQ4HunXsn5gkLb3hCr\\\\\\\/jPYk100VPMX9Rtwm0eXZqceMWf8twGBSSZEUHG2CaIK\\\\\\\/G5kxUFcm6sqriQ7ycsOJbSgYpQW6T9woZAcptxEwkVQkSQUZRu91pykgnMcLwlFbmfPFgaJQWTQF4s7I5abvN5dz7l3ZUyx5fgv2wIkI92qpve6WKtvQOSPuiKQqTtfbfogRyMpqTXvHZaJYAhKrceulg6RXufq4Ey8j+y1V6XU+umKHaQExCN\\\\\\\/3QuhsxJPcDSu2bvmmfmZqXCNsdkczGkO9ASee3WOSLKZVVgYsh8L6RCwOR14Er0U3K3fHofmKioALwsa6ODbPTZxBDmAqzsdvb9qwEmVpy31bMhHh\\\\\\\/\\\\\\\/0xo8RIp8CU42BYe3uwz89FgQjrsj9tIYASi\\\\\\\/ZOTm6H\\\\\\\/CoQ\\\\\\\/HUQURTZYFMVSzOXMpXdoQHIHftJ0hgRl5+TdVtJOmFt4Zh6a2hn6RnEAabNWXwW+9iOgEjErZmGgok0RZNpeadoTbmv6UXbY0gJ0dgW+Vc\\\\\\\/CTNGJTUvrf\\\\\\\/7eKes2sATe0ix0NnFm4ermbjkQJzrIWaK+QtGe8UUsBc1\\\\\\\/QZEzniu\\\\\\\/1z8wcr6KEMCRlckoxMuA\\\\\\\/+IzlsnVS2R7ohfB1rYi4FqAdfqlhGtlAwTWe+7pV+q2I41EVTlWySDzozVcB8VWhFKxFOQqCUoff0IVy5C5BMrgRu1YiqW5EeZVNz4uvOY1mJyGPKAIqST5TKQD12sGrMd93loxm\\\\\\\/UcqyppZ36RySlyno9i81ON4eCcwBIDvKsmgK6Zv94kQPX7GxMn6ZDOebGytv4gY27yPu1xcuvBKm6e0bocY68aKv1Rf4ytaWUpEMm04NeVFG36J7GAznhm0uXxBm0+pwsIOYriIvoAHpnLoW0TDLWrtGVVHucrBqMdn8myvaY0y5eejwlTQrGz+MyDPyXX+z1F7I+8yGiREQ7oYJJsi36ViehoQfwGy\\\\\\\/DUPFDMtI2a\\\\\\\/5ieQKoOOebLJ9ZvPpdxBSHw12X7rzUk3t0ASksxxYga5Dexb9zHIesukpWLtTtXYIanYwpCRo1CyB\\\\\\\/WzZ6yrXKQiBk8LWUXHDGdMyJN1YzLFIvY+9wqdGPTyZUOY53\\\\\\\/kflrnPwCJ+5fDZClHtfdqJ3kXcORDv0Ti8zTWtn4s5aCxLqLCmdS6Qg7Wux2OHjhF+UEab7QRTgQrlTvfk53K\\\\\\\/dW+8wn\\\\\\\/M3nHJ\\\\\\\/3jdV888PxrlF+anNonC+XvxzdUq1\\\\\\\/1CEIWK0jGglJ7O6emcG0uPSwWsCM+uRkVHDfSwgLhEhaLXoSOrdsRd5FixXGKWrBKUYa0I8OehVD1vc9QHmwe67ekZ8\\\\\\\/wcziGYDcHw\\\\\\\/HxUltFzpC1L+vsWNdr7ZNYOC415jI2BOlELAUx+edz5al3eQmw8lO4rnUXss591U8pdzAz7iHAcr2CZiBqSZoopD\\\\\\\/X4oxlMbYsvcsxos7\\\\\\\/S+gvm5eUZSV8Vg+GWhhVoPEOZSGTPv6TjXpRsrOEPCvQ9ofM+SbOLpUvD3KCBybUQM0bg9Sb6h7kGkzTLmcnE9NLu+PLUJa4rY\\\\\\\/YA7YSYN1c6ts5zTOTaJ3fYno5aZ4US\\\\\\\/u9cyPL8hhnirqYlQU\\\\\\\/hxiG4KeqaXvoiEN6QrvW2E6u7kQpzK2qPGj\\\\\\\/A1rAMQxpX6Wy6J+ChBBZDRwl3xF6+Kyc45vuVEtQADj9usWYC7EB+bh7vt8NJEbmXVFFm8CHPQLXz5LSpO83S\\\\\\\/IZX6lhnfBkThlYRlAmUUlViSCB3mFML4n6FI7e4Iuqh9BBeeSWc9nrvUzNgMM8OBFrREVW7XE+LjgsmtOCAVdUeGRMOPMsGcolETh7QTXudWblUfilbQIpzJiD41CgDEf9Y5F3fM6X6peSZAHzyILlNR4KbUuyH1sUBNCcSt4rx3\\\\\\\/oO5KpVpzH1n78Z+Gd7H\\\\\\\/JnZVjiHIB9hEmtd+8Og\\\\\\\/pJwhiqDjUCM3QzzCogKRtTgbQ\\\\\\\/JcXtI9QqwFwZaes\\\\\\\/izxruQHF3AwW0Iwi1ykd73YcHV1jgp8ytdmUBDmvte+9GBYL80sg\\\\\\\/xUjACWiohpjtCgktcv8QqTWjLugGIz\\\\\\\/R7\\\\\\\/McrN\\\\\\\/GOPQ+XvXjJXrZZTWZQc3vaiW5G6oJIEp\\\\\\\/\\\\\\\/5N\\\\\\\/T73mNlp1VidGDqEapAse0PRt2qWoB661ebfNzYlQ+1\\\\\\\/FVP+O5KRQ7IfvrPJU9\\\\\\\/c\\\\\\\/\\\\\\\/YVatMQQIbGWrZRTnsLiiI1zFYNtnGzy2qJizhcHdOamzBjImB4O\\\\\\\/OAlfOcdwXXrqG9VaqrN0A4e0epXN7qOqcpq9mxytyW3JaeMR+y7xTJPVh6cS08o019ccRR8cTAvfxGbIZ4YyeBrQBiT5BmuTJYJr5KPVbp+oTreF4Qqh7q7fcqEfs5S\\\\\\\/CXPNGAO9PnyeMgeelF1FsTpeBBR3x8OhhGV91Vpv99QO66m7McRC41Ppkg8Cs5eLbueCfdNsfuX1SSnrNXc68wgglBSfZvZ4uWrxR7zY+Qwq5IbZXnMschCjYHb4e10FseGGEbIWtEBHoNpt5oOg4x3YK+bMkUXYToY+4QdXsIlQIOplZnr2ANAvB\\\\\\\/\\\\\\\/U2WQFSTXtCC9wkO0miag\\\\\\\/b2u7U6HeyluUDSK5Db22+JWMv+IkNv81UI6J2+ZHk1jzogtKCaXUzAr0mtQVln55zAtilDD\\\\\\\/BcaTc5DdByy2xWVHq4WDgYbQqwS1K3oL\\\\\\\/ceF0KODh2Yylw25wQdDShDCl7l7KWZeq+rg3fncG\\\\\\\/gomewbW\\\\\\\/W+78XAyVauq0xmMs4tltPAoakO7Gb6mr6ng9\\\\\\\/A\\\\\\\/LvoOoUpbgII5XOkuvV+uHlqxz4iUFry7ucRpBqKP9e9es+5Y4b1LlXcFPlbmdRUCwyRr4fboqh5Fd8vEE0HaIeFvl0rFBoptaLRhZAnkoKe1FrdP8uYDp7kKij8XG2k1S3oDId2X50I3kqbGHyj5KllQybnzH4nSM7vdkkeFIedT1Lb30IxmSpwf\\\\\\\/rX8s\\\\\\\/pY7QQro89aeAo8WQj+1pE+LwY5cllqm6KHt2e3LvqAPEE9YXmcHDsEckO387s\\\\\\\/X3tKfdaPEq6z9RI2XS2WHGGaqw\\\\\\\/ZtIF1nCJmwzbD4sZPQC84tVFgApyYUZw\\\\\\\/v2bvVbaalX5yq2Z7nrNrYU9S3QRPNohcEufuHQgdW9E+5WMtae0OVlmEvLYNmETKZ9BrqjLwtDJu+SBuKtjG9KtQVhbFMxo5anOriYfkAEfv3pviRLKS4+taQ8+WCEdK3Gg0AKkfVeWTlr\\\\\\\/dX23QY+gpFuwrhfELWqLNJ91qUi\\\\\\\/LtUjNrG7lXVrOc0CLNo26PoFuYuc277B+w3E\\\\\\\/JkiXJIKzTaYUWsP6\\\\\\\/PpkNN4EWEHbSMEvS5tywx02+85asRXfJai7E2tzMvXdHf6XE9gv53yS62ZpAJl0jIKn6vr9zc3v1ucmSy1X1coWwO5f4HWJsRl10iydOjU0L0cWsKrYQ+01Q517D+xdoU74NqjaWg9+M4aeScT8f4UsdXomo\\\\\\\/7iBavKCye\\\\\\\/Qm0Oa37kGTvp1Aa9P+pNkNuUCywgs+KbaGMxXwbY8RzPQ5idjuZ4IAy94svcLW9FLJGhApbBtgu3RHDOcdash7OCPw4kO7cG1xyxaa1hIJCxbKMJgoOgtVJ+wIUC1zIqS7Aa4FOZ9ponkr0ntagTpNMygbMc+stCnAkbmqUSbjuQoJnTQp0P9zfqZHrfmy97ESkK9XATPO7iPWb9Hq5ogivtPLyUzQjjLrE2ru6YYE953VLOPXAblxMSGgbK0VmI6KQcbqVRVjKdMdzQVMdNE7rMnxL\\\\\\\/XaEmd80y0dcAeVCGk6F9KovQFlb03949cnWZ0qw2N1SaXaiiMpUuyayRNJH0HZPTTOsZtEpbZbVmWpZfXes\\\\\\\/RoR5hD4mjsgMLzTFhXMUNc8Sfia1ILcoNdN\\\\\\\/+iFvGadyLNrYIbZRDKsKMSfIPnODx7TyWC\\\\\\\/StuyJRW4w71k2UEJm8c2Jzszy9e5MxOxwWTtkl3U2A9GTOMCCnax8hgAaYfLxyv\\\\\\\/ZeI9+ZkGm7hUS05lCSFtbfIhqHm+QFVkA0vE0vbEjTq7K\\\\\\\/Bc84W4fRKKREGfgUmrG0AP82U4HGeOn5KCezJOFop1jt0XtBmIxaCMJUCkkEojX4mmGZswYXeRA5b7Pmv3BR+33un7w0PXOBgqrjcwY2nGZtn0BzGNM\\\\\\\/1AtnVdEyMOoqBnqHnwQMWu5ep8zIjnL8jflgXTg2odL3WtGPTjwP+aj4tQfAjZ0AfaIaH3lymIhaZBEPxNmZXaNpnN1U42twtwZYgreUZWDDT5FuAQgIDQzVbnS2YouS8ZrRJPsa7BmPxOAYNfvdze9wb5U7IS0S9rnOCm0EksnmjEJdw30MorDQWOGJnURMki4\\\\\\\/Xo8sE6\\\\\\\/xCvjjm9jT6Ek1A8MCKrIWy87Tnf4Mh7xwBqR7MJ0b2VTyBDn5QYlnPhzWfenxcecDW8kxyYKEyO\\\\\\\/Yj4w+ygAY43Kr6cSACwk8s4akNudsZ25YlAJTECLqLNq4qLFOVs\\\\\\\/m5CMPwbFjk3GWF\\\\\\\/d32\\\\\\\/d1+5JRC3fj5yDdpvDGB0HS\\\\\\\/LU9gr6+TqkIOBd8N7tMV7G8zLEanKIOBvt9dr7DX3diyU6kmkq826mZV1li053CIBl02XsHsaYlUQ\\\\\\\/NvANKS0wPJgKh6zp71xV0GvVekDlCQjFXdRG80UQ1+tf5OpJm7LUPc0wjwggKkiTfNjSJrmycDxaH4NTS7cG6BUhdWOCbVvw+KsqTB2JITcdmIFMGCHpxE3bqzle62nWaxFVu4hOuhyj4dFwxiR8niQlT9eN3ZPoqqC8aOzWI84sOkRuCw28f\\\\\\\/BV8pd+ZkWJz\\\\\\\/AYN7ejymOufo+JRoD4Sdtibl6FiW05C0N\\\\\\\/E2l0pnhyd0iAuZUfvnJRIiFrdLua4m\\\\\\\/crsvtwsWp7vM0YfgfeYVzB2i00jTEdHoVZ+u1d6QxGdSvAyg\\\\\\\/TMVo3bumv0M6ZCVVvVFlx\\\\\\\/C+VjlQfWDAgMA0u0CcKOWRLVxn1qFb7Tr4xqwmGYllG\\\\\\\/3dUf6HEl40tlmZvaMzIiqQV2Wckzoh+j5m+65cbT62CLZACR6tgO7wT\\\\\\\/g\\\\\\\/QUTg1ylaccK3MQXiLipaa8UWWEETlhYCIhHKyc+0eWHLf53WoUDEy\\\\\\\/CV8QkY0ZvGvN+bCRkoYTvlNAIfeEJiMBzHs2QxX\\\\\\\/e+uup7uUf4scOYzuSOHA4vHflGg1FN5XbgZUIW0+ntS8c6Q2j4GDMd9RPi1yqgotNCPv6ac3PiYvPHEBI\\\\\\\/lDFpM0BnNrxtrkS8K0lmT1JfP8BzgrnN45q0yqELJydsnCX1zuCmjykEd1az4uT3wKt5K9sW7CN61ub5msVVI0LaWbVfTNTY6P569I51zET08zbzgWsUHfYRMgqcNIh5sRgGhb7R6i3iSTyocp4Fm0L887OoPeTOdHDwWkn7DRnPO9YaEEAe\\\\\\\/OUhK81NCkstW1pv0m26QBRAT4FZJqYJo42p467a4SLbSvfNYRlp+rDPq0e575gHAOGx2etZiPFFK+v\\\\\\\/FRgDZ1gXVzIpLGIrfnJDNb27+lE5j2XC6NRlOvj8Neo9hoW1lrS3yq1FDzTN5eQvHm9VqttsS388cGgROwdDFvjJpOYbDLZ\\\\\\\/0qh9Od927hwjUti3pi8oAttvqRV1FenOvjWZakyEpfSLgQeySbmB62tHDJhpHlOXCWsZK8HkvPRo31ZStx2KFe8eGiXw6TRsy8bNE1eQNfnkTAsbskW91tRNnUTvEXIH6LpLSbpsFLdP6b6oP2HKWaevijmJ2XvtXfflD7uILFcYpC294GMD+fyIj\\\\\\\/j2d7cTmMAnIzlYgnZ+ar3MsQxnYkw6h2HBXrI2PW84knhTKgFcwWH+BgadKfilUhI8TjmVjzfPsJm3e\\\\\\\/npnIlYb22nOS8RKuzJ3yUArmJev27WSnZDKPvbPVYC01oGylXVBry\\\\\\\/zotAo4YoqumwMjaLlG1SY61D4o9MGv1JqsCl5uj296b2SebkGKg2H609g7owDDzQ+BX\\\\\\\/+FJmKUJGa2YOaDaABGEamRH+YHMA+9ctxpl9IRyggJd6cErMfoyntLouklzmPvg9ljesnudQWhADPT0xWVz75HbKfiUqc63MBT5ygquy2TZDe9hwf1FRbtfphY\\\\\\\/r+zm87EGj4WKuK1iy6dI2NSSI5jdCEITPsBj0B5ZyT+mWQpMP7mwginmI+oaqH1jsLrMLFvHplpobEvuprBNA6mHhX2aquG4T\\\\\\\/WUQt2rZR\\\\\\\/xCXAv1JSJwHl5Ak3szwHFyunhjEABKQ0WrB4PT1pNPBcDWc5ivCflUUZRDUoAJhkrzFredGsd38hKZItl5vut9QlbRIEmr1mTMowlISS++DbyHAWTyuE+EY9\\\\\\\/UwVfqxOg6W5sqBwTR9bINfKxbg5zN2YpYTOlwaE7HnWEVfAj+Swf2Lz5GtCoK5kB9zeJhhCbAa3wAoY6kQSXKuymvtZyxKZNNlN2Lbo1urhjeYfWVZRa6MpJEywF4S0MfQevlglXK2pnLcEsHqKwgV9hZuvF39zTdHEy1yh26pAYNNWdxaDB0YPuDvduGWXeD4E65vGLVvKRaHMqDGUVI\\\\\\\/IAs+E7sJwCnn4cYPZQEYiVNrVhdwtdRGwhU5IlhxDUEfyLdo\\\\\\\/aiVpinTfrhQjKnmeMdccB9JUJ06fXDV+AioxwmayG3jgucFSGf+PDrNHmp\\\\\\\/Q8d10a2yrqjr1Mjb5Nq3ArXIMvGZYkH6bQOtxZ\\\\\\\/3B9HTWkc21+Z0UyJrzAK\\\\\\\/AovzczorzDYH2LCJd9+9U9h7zab9UJDd2+ZF\\\\\\\/liRxyhx8jU2Hlmg6QWI\\\\\\\/4xtaJ+DHv7T7sT+vHu4N3gQzCboPbF3XoNEVzrzPPZT5+xl6WUa4WuzIiCBhFKKA97GGRGTPvlYCZVxevbtl1hPqYUraefq5iKmAUbRDFYvKkIyh63aBwT5mhRx+xB7bwkm3BqC1t8Ve5g5N2NWfxQfiVP04JQ\\\\\\\/mhsvK2LrZ+jxcI3D2Wj\\\\\\\/R1\\\\\\\/STI+gEmPrPlLfzwwfDIdsReMqKTeqmkzzTWOxxEKxCzcBT5VAbYgFSmc46UOQNDFonljgfMIdwuehVrAoZOfcxSFWzjKD8Wzh0I\\\\\\\/PGfhCceGQ6jxz7CNe4IGH8JqdFZuaABOqTHxAVHcEWZ23Zp7H56Dg82b5fX\\\\\\\/cH0Jo7FBG7PhcEswOfCVsuOZP6By6oMooIe23SqsLi2bTD0dCRw9KG83xQH53bJmEMlvUxKQudo77y6G4XDnbhatPXwIzdY7YMwz8m4JY3XTXwDaGBuZvt1CHJDutRd8cplM8u+sX9OoJwYvdaE9tar2LcaNxxn+H6xXgiSvgZek8vOjGuJKOMHU2v2CPqArwMRpSgNOSKAE3+nmG3vxJkKSlRq42L16JwvLjhdMEkOKI7VXOLdRY8NO5aupku0HC3WoSCpe0Ab7apawMV4KvsBnpE\\\\\\\/0m53j65EsImQwWLQKap7404eDA3fZKimlm0nfxH44Ypj9GdeNHa68h01q0x9hlGWXu45FQ3yYJU5FgqhUd8ECXBwa\\\\\\\/XeRYSkQjWz07IpvD2Z5W4K\\\\\\\/2fn+aSIeTVnv+1h6nxGq63klhKlW80bgf\\\\\\\/xdzoDYVoJjgbZHiQZh5Bp4++5k5BzgDNAQECIFixZRDBDpaNux0GvQzncsoSOl+cPGTZ8vurg5c5fpMoP56pfGcdoAV\\\\\\\/nL5z2vMG0vueWg7LLlLfbt\\\\\\\/Q17ymrBfYlE7sldNlW\\\\\\\/+YSQGE2tWdgIyNNJVcFfgQ7GcHuchFOlsgWzEhquzKwtIY7+Qs\\\\\\\/oa6VJPusKDvX+dndtq3UTk9x0d7z5ilAM8PRHm4iGciYOctuQtqRGeP0eveiHPlYnFigsDAY1OmrkgqrOgIzDL6jOh7pzmjX9IjpgoB20Do1CDYn3Yp1V\\\\\\\/99T30KPkcBhFs0OwF\\\\\\\/R86xHBFNdfsm9Rt11vwSOZ\\\\\\\/BTLnEW3XRfHWyMjXOGEzQgV+gOBTtTLeEl12UTKPY8srApW\\\\\\\/jHrY62OWGMsqrsOw48ni8HbrqJInIlYAWAPXdHLPEln3kBC6MTyngJ7j9D3ebhFbQu\\\\\\\/JkqVamHJG8Ez8iivyzfrXy7tf\\\\\\\/JDdpE8mUw2i+q+q9ikF4kpV1E7klo2EfZlYkGwbHIPoxTY14UkYhXxKHvwGlQWxm+JwJM0pGY4gk1fZArlKaUWLdFh6dtHigtheujviqsOMve67kpIjDM3+EH8GxzFFuPvw6wWLviETOWwOCoBSp\\\\\\\/FulWK7Uon9\\\\\\\/BwxdnQvqThhp6qYhgUZWOcLcahkLC66Dx8R8n+t4YVUcUzCDigLwtW14RedNueWyLel75up1bYDOL61AG0r7YXjECdwqjfroto3ldSjsr\\\\\\\/DsdSsonrmliGHr\\\\\\\/Q+f9uX7XGnp4Ehy6xNb8cEKiXpmXZufRXSC8z5ypQwupO8FJt9baLBzg7Vqi7czNDsOc2KG6swiLl\\\\\\\/285iAaylgdlmI0zXUA3FraEmWbrjvGDt16jJ4Rd194sricsEKbL9LLVq0nmZq\\\\\\\/+0NJPkUZEYvlXW7Nt1Xxd7HGJv2iEO4cf69YuDNsh3Ks7O2pbWZc3NOh\\\\\\\/cGSCIWiEFAcYsguOMwdjDll737171d0nXui8LqlnnsKLQ3w1wV6C04sxmFMo\\\\\\\/DyYvqjz3KYjg1WfAOAxaUUOski5JIvcy9Y1NsRrqRc8Nqh9AXwY6uysHRK5JZRqvfSPu5xmp7Awx+3uwz3lpUQ3FPhkFKVpVRoj5FNI8K5OpzsmtCOosv0npab5Giq85TCmN65XaSaR92xGHVTGMiz9vNqPVbODoO5WYM8MGIsEouh+TtKS\\\\\\\/zzD55w2rluPXR6U5kKet\\\\\\\/e0q+v+yxr+DyQxy30erKd12Kq3uAVST0A1X8LC2KxXX1Rqj1ahwT7meX1uD7lW0HWrd7h4HKd\\\\\\\/RuHfzhMfzzuvBfnWZ\\\\\\\/L2HWLZpm7ic5qKiJvEsN17v164D2QJHNXOHwq4GnMu\\\\\\\/6KNKctFIxzS6EmECK2HOoX+xCBZpSSUsZYurpKKw82tPx8yRjLlEX1ssW8zCsub9wTiOD2SApk2zs82tCBMI\\\\\\\/TZV30WCpNh\\\\\\\/5Ko6abdNKdacCY\\\\\\\/6quI6RlKX9F5lS6pgilk\\\\\\\/NAfzqS077nGKNl\\\\\\\/Lp7o94E3HbefOMiBBy08D7Jxw4lK+6UmH2VqgK\\\\\\\/uWboTiZaRIo+5DkQruMvD7ycbAlNRPx7f3VY6iav8dCJ18NVDSgZtQYKb1zdoNFSQBHwWbvUieEzZR\\\\\\\/mF4iwF306G9tvyYXebGofAnmQ\\\\\\\/bX2D8r5FSgWJWA4Pvosol4Bu2UFjTz06qh3shy0BUvk0cRePCU1zVgMYM4F\\\\\\\/kgKNmFDEzO2MN90winZlEsjIcDOZopAuxNgdOv4qiwLka4QxI\\\\\\\/izSkyMdIQc90fhpGtDIPR968KTMztOMdVxKdAkYlHKJLpnE6BXpiLkUaNgJJNohQgYw1VLnNJ9gufGHNiRXMWHMmMFbbkcDOAyWK4gxbvUEg9UMbBOStNyy0ULFkGcalfzdC9HPsDg88XtKsU414TlcoWolUUbvyJz843m6xh+GbvcWiu6VpPZ8AG21p5I57NhtONPNaQFY0966MDLjCcA52PI\\\\\\\/I36+ITCF5+PjkCz\\\\\\\/qzVN09aK\\\\\\\/9MhlZQ3JP9skz3fpPedtS2GlZ9FFrC+nuCL3ICCom5AEv+0SEIDR3W2DhJyJ50\\\\\\\/yVFMPM7VzZAa5vIxF6FQ7gurqit8J7M6stZDOZIntw8COx5vL6\\\\\\\/uxQ34wiRpVuSS+vzXG\\\\\\\/5waDjZ3Tyalm+DkPhGAG91RCarfJNbKltt2DScEv3hRpBiThU3fLBQ0+959mLUT2fH1fswNRTB\\\\\\\/dWPV6ad6KOb1q1\\\\\\\/bbTbH2oxQmcpyPnlpJaokvMIQgFYR7GfYVx9Xn6NiDOkzvvBIxhikkGDLVbZOvL0ElrKud2AuFVNHC+KCNGElAxVCMFKhsas18EW+8TzzUEKltg5UR20NciQIAIXyEvvYvvwMgOJNNgxzDp\\\\\\\/lq8+C8XjtgIA7LLIDRLkL\\\\\\\/bulH+7OBMWryX5UPZABQH+qwE1fjP\\\\\\\/txqpADgX\\\\\\\/CY3voUIc1VfAPbpB7Zdw5ZdRYdkdZyvGOl+BmUt3KHBwZEfR6obRwhuve4qj8Pd1OjFDZcJBW9YI3WyU8QobbTvlw8rnNiZxdjorP99\\\\\\\/TBjJrcIgqd5AZ3Y49KYWzYrNXoOigfdXBDKZzFSs28TLw6PEt3fzmt2E0uKKycfYzrLZDISwdwLijj58yZ8hq04+IW\\\\\\\/hYJlUcl08C7WaekzVUhipQlYAVAqJP\\\\\\\/skcrxgFhMsY0s8z2KRql\\\\\\\/h9QDYs3TQnc6N+fbbNKTSEFNHCoshnbxVr9Uf1cyqak6IR9bl7qYhFBY5T2k\\\\\\\/x1sfxuEWF2e0c7lwPjCp70HTllYbtLoqrh9f93YrlUPH6hTZZ\\\\\\\/ADC3wF+g\\\\\\\/TcvFg9zyt7oFWqJDMa+C7dTF9GMQD3MwOdBtwlLs7BbnCWooyEWG00OMxpqHi6PrK43+frt9UDpitBKHTVjbcilhZDgIw+nyFLnQJjq3jFkkhd1FSO+Igr86AwMZKzsozTbXs2mekZjgakSvYqrpCHYu3mR2yH6Devl9pUA0Ibo85HapiahA8xmvg4Pc3oJzqpPIGF+ocrn3JnhQ49mXlQIixrwZlpleepWO1ygB8FbRY5BsfzjBR9rmxIZu4wxcnq8mSwDlm7JihePFqd9urRbbjqx1n+ohyymMzWbbSzd2OCBGhG80284sbTttDecbJVCvUcwol4OS4RgEkCQBZf27ggbL9ivStd1IcRycs8\\\\\\\/z99yx52aYlukFhdyxAkGbW6giXJdzqwmrD+bHMRqAPDAoXfQVlVVbl2JuvE30iPqtxb122NEGmILj6zs8YUNRb+HN9aFE25pmS3xenjRlDv2fY5BJHNykbKXK4d1xLz00C0nsJ3iUTazXnySEPTGAdR8eoZOq3pvNYE37R8TzvLU6omLdUcRb0RK2ubCB\\\\\\\/60lTqkqbbNJNd\\\\\\\/n5YAKlf6oD6o30DfCpm+O2zl4riwxNScaeC0sfUB1j2sAASyKjOzHHD0UIuRNIDg+FpHqE3mRrgy4psFbQ0S2IIFbp7b5FSQ+P0S9IX4v3t6LKwosaFuqHDAq9bU6Qt\\\\\\\/jHyADniPnXRWMbGAA5FfhgfNZE9dv0vq9HY1lXfMnY83wVCgztNhxVCJOadXFYldXUTCf8XJr5\\\\\\\/d64tUCc93Bebr85SWZsOk\\\\\\\/XcuspP43aiPlKKYVI7Um9Cunn32Vp9esS9pRpuon+w0Au6ta64bkJ5Iz1uXp7fGkJiOV1gR6n3p2gA0ooRibXHzikEnqLYRFYUJoqkCoNRp8UyTpfx4Luc4+tjeLCn\\\\\\\/Xwb9VcB380JcXUzrBLnLzrAHaFf2SmMjNeRo6LwrFoVR6cYg4M1Y88eEPb5LSCMpHEoO7sm9MnQeDeiCKEXsgh7JGa618etJVwr50PEwR4Uus5w+5jp3yBrdRq7MZ4mI+84Caf5McI545rRKwt4iRwRiLvmwnsPJhmkK7utZqJXb15qKDfkHGlBZtVhCzVeIt2ndGQyfr6XgWKv4SFa+HOohp2A2Yjj6AzHt7nqMHzgL7D9pxV4tNXFCyhkp+erzYHzV9v8+BggqkNf0OYNBIMScdr9RKPu5sww4elVymn1gK2he58Aem0\\\\\\\/4K3DagPKMp\\\\\\\/ATEM\\\\\\\/7o3BD7B35M5A\\\\\\\/R9aLZ7sw\\\\\\\/YA2H6zFrzdceLIqJJwkPumaEol91ftkNHNhYeqxTrDaKPBcld40iCLtGcw1cV1yt\\\\\\\/bgKyIlpstrtmT+NpKKY0qS27oodNJKDR0u9p0o383ZmZWubhvFGaw6h9QzKjp+NStHFqWi4Ea0r9xSAVziBeAWT+qV74vgSFLQ426PDZzkHUdW5\\\\\\\/\\\\\\\/IPo\\\\\\\/whbBoQ2ELQXpL\\\\\\\/fxd9w7pmzFObNcsOFncFJ\\\\\\\/yF37Kyvj\\\\\\\/LI2KWVKGyiXt2B3rQ4xqtes9kBBfOc0EfOExZmWK9Kvh4mo15VBUO2Wauh+9EXcmho9b6DWkdQ2iQTPPvQfzBuIIZ4K\\\\\\\/5a5veCaNFlYKzI0GOsMCD6YTD50EybrzcIrKUYukMJjgaP+lJVs7\\\\\\\/9gqyyWeeTKy7s488jWVZkEogYvy2zACgQNdtoPIgyp5KeUYsQkc7lSL875FV00l1JMfdcWkXTftm2E3K0vc6ORQ90hivcpQnwZpNkIYpa3dW7Oo5d2kFlu8fb6BguCjvvUrjIL3527rgCgTqEEgqRLIoZI1FJrGroU5naIvGT+gsK4VPYEDy++t3OHv7igC4XiCfSF9jgPOBwGVYSqKTof0VY2\\\\\\\/LDUOBEO0d23OXi27OKt2OMbPY0\\\\\\\/l0YvoEPwA7tuSX0WJS\\\\\\\/HKTLU7og\\\\\\\/DWgylIbXlVTAL2UtfEBZSmyxztaERFimQXg3wG6yDnYMzXamANJBVMn6Ta4HZzMbEkMcI7TSXLYjwu\\\\\\\/ZlYQZZjniPr3SLYpznwgmYHcOuibIiWxqn2KxS64YQACT1tCkMfqhlVXuF4aaO9wVq5nw4K2Xt6iKFuAsNInPw5fYacs4W5tx2JGNFcDLOhU+a3Lk7sZIU7q\\\\\\\/wN57bPOgETUXNVq1ns0E9aAHRTKPPeu4QxIr3mjM+zl4eYGiL6INdPenXjLVl1xHUuKDek54wZ5DWCuZ9hsC2zAidRfVicaXfduoD2lP5pqDrJgY+yKe4F0Q4WhAmXqqnXTgz0Rr1StxwPkViKu\\\\\\\/qJQs2rkH6p4xdJfueul\\\\\\\/BaTk9ueSxYzlH\\\\\\\/r3lj7UaUbYbMqb6AR531RBtvM\\\\\\\/eFw2X1Y\\\\\\\/xGJysxsvso+7DtWankudesCtxx79QVDPbqxAHGpfrvyULqYxoUO8GbqryD\\\\\\\/dh6Rvv9ZEJ4yrsAoANPReqYSuO3fPBjyw4Ut8oipW5IJNgvQJLRcjOI9h062OFdDgcQxJ\\\\\\\/SmunMJGPYq\\\\\\\/UQ\\\\\\\/HGbonzRj\\\\\\\/+lOfRP6gybQsMFvupyEgc+yxIF\\\\\\\/vyoNcvk58lyg9jmiXSP7JsVy+dQkORIxleLB3SstQvszWrkLuSrIjfDNohWIb7OjCL9bq\\\\\\\/VyRuIOd8\\\\\\\/UZNRXIN0fbalQGm69cRqwO6zglvFBa\\\\\\\/1mmXEFJoTdoXIF4ele\\\\\\\/qeVWYY7WneSGJGoAUEnZIc7dDQeYrb9FabVhlPxxvgPmHW8gakW1Y5jk3f8O\\\\\\\/Bf5D4JKivO03sXT4C35mJuGFAInU2vjt4zCyOFYES4mkRvnQOacoKKBDuHZPcRmImtGgFaImcJKtg5Du+cRX7gzQNLEsB6c8JdXnfZlRl8sAakaYbmAQT5nyD9nAdeAmBpkPZf\\\\\\\/QHZpE3\\\\\\\/KicPRnHNMT9vernVbMrY82yflbl6Au5JFcmrCLJT7PkcEVETXu8u7w5xIZD1bc+5GQHLQOPfeR7mjcoQ92hZgYCggqiYx0ZcFWJoG8Kg3djfVdWxwZWO5yvKNuzfJHt7EFFIk4tp\\\\\\\/KuTBwnHGBgtLpLUW6UzgWluQfKuh9NRpbeN9bD89nF\\\\\\\/seuIgEpSMRcCpAPJm+YuArRr6hU9KbNjkAKGQgxtLsSBnCLyRQTUxmkIrv5cgIhzrSFRiytIIoIgS\\\\\\\/zluGEOSLkbMGGjukuXcT4x15eChOpL5O66v8KhQLRV4coZeDjZrydgggfUR+Zhk8596ymemv1IlI95xoI7WBK+qSpRKtLS8wbcsJZ50MgadVcf\\\\\\\/5lrrX\\\\\\\/CfgcXqUonES2NbTLOQD8e2XDuCGrLFw23AsBB8FTwJLSv9bOHJfEHCXMzqMO7dR3aNswQql7oZR0rLjUagsX8XWVnae+k6j5DCEVZeVlXRtJcl2NktRVo0u3pOjKmQQ4m2ZQ\\\\\\\/iK\\\\\\\/GAcah50lSQuKza\\\\\\\/YM++rXrdqqtA536jNUJb6UBbvIF1reJ93kTCQHYk+1GBuJCZUCm5zPqyX3NB6y\\\\\\\/6NOFPVBFRFlzxZEkzWtV8yYVGgk3iolcRGbYkgJYdGdp+39pXutojGyGrhVflljky2u29KlR5kuAc31NqYVJPXeHTUcyHhdM8jSD6pgYIoFYKVZhoGMqp0k6Ku63EwjDElXmmwMAz6eQC\\\\\\\/G9zMWtylcH+NhZRtDt9dTMU+7iREaQ5rUdI74E08s+q2iG9r2Irjqcx4BpXq+ivhtI+BxZxve+6\\\\\\\/alf56MxvUi3jCPuR8mlrV6AlXaW6WVcqnQOOgAY1BzvDyFpXBxcGoqNtR9PqSob0KonkNLNrGEdOWojJTbdtmF2iOnd3QVzivggmIfrrys6cyV5tNDsmYcWNBvJm2+9ESuq8vlwnmEsPeVICV7zClQeMOjF4cx5a8WWUP0lz88APWuQmM7shGqEBdbY0GxhgN4y5XdJNN5F53GQk3ekoPoU5nkO1P+TpZwMtp8wg3HFuEHludKOqUqGdZDGg4NkutuiV9k2+8JzDz8BLOB6go1J5tdNFD+3rwzojD9LdPBrnqggxXDT3dRqgEtm8k9WDO13tpiIKuYLxKWVgxR07t4xG2C5crNmaDj2qcYPSP5FNulNYfVGQsB0g6DaCZgLXObVR5IdZs5c4nz4PIKoAqxW1cNanr182dwkJDt09YF\\\\\\\/felReDI0UyYa128iqJoyO3lB2qMrUKDN+uyCM1+PALbC1TcjuhHKqoh\\\\\\\/dtMkf+J2f2LLI6UQ1SBZHD+WcqdGaaODJUDriLlQUtaJgbS5E17Y3\\\\\\\/zw1F9dweYvxRd8jrlHkbO92JFrCA4HJWju\\\\\\\/TI9m3IcnfW4ZJD0wg6lTphgwv\\\\\\\/X4kaSJD3PTdjJaRNfyXvQY3B0gcXFQFSDr4+rIx7QggMLmDfFNSdoSyS1sXsP+\\\\\\\/JyPNuIrmVZucAz9UcmDEAi8Dh1eF4yQDnq95GCN0Uz8dLOI5qFObr6zsOrOcR2bGbJLMtRPBcXBnK7YA06u9SmfhJvW\\\\\\\/OIagczwo0C3DfZA8N\\\\\\\/gh7Zar0LlxJkK5vXFx0kRBl906uEx08rZ1REFSLvVMqR3UpgQNlidoEy7VR+EymHA7UVi+3uZVG2ZHulfAaRMdHBt0YTUr\\\\\\\/VTu2cLV1ojA0esOF\\\\\\\/oKtovKiraV0JBV\\\\\\\/\\\\\\\/utJMZ+Y3ZIwAixovZny7SLDBe+1Mmqe0Y6qLmCgvOwNeY2dADEP8EXGRxzNN1Q5NtaNUgxjtX8z7RZz11UWHXQo3r6A\\\\\\\/UnPVW32mddSD6UPlYid8SW6abiGHXvzsinzhA2vT3e83fpUSGPC4ilcqKEeI4IHzkRqAMt6dd8QM0yWcSZJHwfNmVxnKCpMjB4oemBGBFWHnYB2IWdeQkbhbHZPpEHvLlCBqvTgJblDmNLIlJii70REKwMnQWjklyMxxpWMvqLOkutiqO1iFgXy6uGAdcbGTSxabALRWkDnmB7P\\\\\\\/JwyjyYC2KWJ7n\\\\\\\/93puV2FmGmJ6NEHSgHZpyd0JQDan34yxyj\\\\\\\/jSBNHVKy2ivN7hN5tgWCYvJgftWhC6zJXEfIn5zxx0p+AVpetIBz\\\\\\\/\\\\\\\/h9NPQpRDoxSJgJe\\\\\\\/aatOgSHhgvlB6KF76+4O5AYnzBLFPW5Mf38HdjRS7mAEuTLxTT3oCGwYx8cHQOJst3b6VEsGW3GpXp00Sb8qC4saagmSBUUManXr5yXjYEIFrna7BTV1XbGAaTa+yRKo34+0xKppKHFYSGlqkdOwmMCrENTXlzt54PS426XmRXqd3Q8gWHEqUc5EQpT8zeftYfjZpwBfqUxOcGCKkwmWnsRKoD5Yff7HViP0mm2bPD9rbX99hCn3X5M8LG6YEfqbI63IBnFmOWeYQwVY9\\\\\\\/rhdA7wYT\\\\\\\/Rw18PuqVhH4Yvz6cwUhZDUhE3ksqT6eAjeblWwUDd51Q80YJS4d1zEVCZxuXz9uiXtMBCfGOLD91qIjyScKGsDFDrqvO6mdslctaXrdQ9t9no+JYaMyuPG1pfPwIxFF+v0PAI4\\\\\\\/C8qHb\\\\\\\/h+CGPT\\\\\\\/YFa8\\\\\\\/e5uwiFBjJOSxIo7eL7nOB0ruI8JXa9vF3paUOe70EGnDMsdDr1ZU484zrU6phiqCm4T6oVQthjPJxXLrWsd4ga+26QF\\\\\\\/XgViZWI+pQS3IKkLdJKVMB0ZexhVto5\\\\\\\/oJ\\\\\\\/HtJYfzO1DIjCY3+saHMJweCe09f0sifQiDeDB4lrLxMmo1FuwDbMdYgwI6TP0Jv7CqcbKECeoC7Lyy+BZmzIYofLN8xwectlrdKSr9WicDJU39QCNIaU5Kn8vnICGeRlS2VUQEpVRUjf79g3XCx\\\\\\\/AN5gnS\\\\\\\/nKABUA7tI1lNnEef5PDueMj4cru9Iw5CIcEvx2nso8vB3Jt11GR0goZt12QfTF\\\\\\\/MkFCDOS7P419rxfG13a6L7ifO+ymXxfW3WAP5daG5wg7M6wMNoZmwnRwuoVHRrxA4+jShPFtu1aDH+MA3bfAz5tMyh5j50lk2zXAoR6GJGob\\\\\\\/ej9BGEpH8+IbNNr+IxZ8TEue+BMhau7T8aZXxqHronWfJdj1bHQONdAE8D9za\\\\\\\/gbAGRxxrtNcAqryd1A\\\\\\\/YIFSKQHOaWvEk\\\\\\\/2rFyD+LqOb5NgyU2oJbebjY5O9Ns4zO8PXBkzCEl1M0SpPxZAYWa2wgKX6\\\\\\\/UeXHUVjkoO2zYZ1\\\\\\\/3oO9xhQRU0ctSRL2fBJPYqAQ2kwF3Dj4iVddC4F\\\\\\\/u4nLfXL0TYf1wKV8f259AnmL05m2CZHvN93l9FKO\\\\\\\/zKDXQKlxr31yrohcXD42XWI30fxxReLxmtfb\\\\\\\/9HQUtxzj0LooszdkInx\\\\\\\/YRc6RAXaHCBmTmja5WBsHeWS2kZqPE8mQlCrllDIOpWCkx9VQFcXoQZoPoWKCBpRIeLhigoXy2ZIj1JAUE0czdoQ\\\\\\\/EOfRkkt+sT+lvgNZ3N1IFXywI7oeNNYnchtM4Ea9RyP+g6hLSveytXiYN9kYeDkyTvSwygEx328ZGDB15PmXJMincfjBrFheYYiXy70npfB4DnatT+Ml1NBh1yHJ0XQ22tWqF00liTBU5oSrAi5qU2ZM8y1J2hHKj53sySZ7qxfZ0pthl+Tr3UgQOmojARw7E4OMdkXkljfuAu7YzfCJNH\\\\\\\/qv4k+SWOjlJFOqWNNxGrrp2taH0vSQ5NRkh5s4kGYJcqZn1D6gCvGBIcFroOCSN7XN7NVlL3k9Cfxr+rSvbJZSp5LhEVhSMNMhpRY6sbm5PYPE0TWFG1WSglbo9chvuuPUtKSD1CuvgWIUDLjXN2RxFKyTN0JvyH6VxfNn1owHUd7V1HgEZoK\\\\\\\/+UcaPf10W\\\\\\\/RrfTdbHUJIKL841KyHokG5ogynR2clxyjOw4g0rUaQA8dmU1BQAzVYy9LST0LndDSJtQQPTKJTUUt21PLAWg0zKCZoS0V8\\\\\\\/vz3B8+xUy5ggA8AjMi7ZpAilWLyG6+piaEp0vJ3bVzlW\\\\\\\/ZgAFXB6qhe3oMIMlphDOLQGF6mibYb81XrDnYZRZYLoubLqt2HC+yEhYU6A1eS7Mebmna+fOVgZHWFvK5rMDUmt5RXy72S3gY6uM\\\\\\\/RxrDcRhj9K3EonC\\\\\\\/zLpGsMOlHQQd0FCiHym5wKfRMKa+\\\\\\\/u5mEI6iyOayXy4WRWyCEnK6U1mHa7LPlNAMAwV2LTMOzymJ3vwbLCwZW7hNcOW+oUVNoWlaTf\\\\\\\/uyxxoZVt\\\\\\\/83vNb9l3aua3pE3oWCa8Zfddq9DPEXLF7Mxj2Z+gtbDXbCo9cDfKsb84bdtYy3crugzHBj2vc76bUDMQb08e0ZBP3+XT8Ap8ROeF9dqXbYYn0+lXg7QpfAf+bjC1lLSbunaQkn0J0wAlsMhlY4AquysOlETYHE29HuJqCBU9twxly+ujZFQucMILy7ZrXgS2u\\\\\\\/bIU7iOpsn0ftYCtkiZQVT1EkxXcUMxE6F9KOu\\\\\\\/YrOrQjW6BnOeOA8xGLWdOcPPJJO9Rl7Kk3CXVCQ\\\\\\\/ibZnJb0kPYeEmOkCtH0mxyIE7tX1akCQ7bcNNqMjxCv34CFNIxhHkycE1Icxs0cL2JJ+a6Wqumnp\\\\\\\/oG3XbplHJuHr87w3u4nygv9WW4FZEc\\\\\\\/ibNIwZUyyh4062A5\\\\\\\/L2nyxxITEore0yConpWrV1i4efK\\\\\\\/WVY5UwaYjZP86rex5Sxrw6Jzn9NC51ZMBHsqHYz8R49n+jXwDpcp0lp6GDZDFHB6abW96YKaGLs6SxpjkMXdKI1YzyXUMs07v6WSsxJQV9hsZx9otLCPSCyPx8L\\\\\\\/SufNDpybyCOB777OeWxWR2aNsDbcqdMDgjnKpS8X3ERD5T7dx6vS8+apVOnhAJ9JbF1nBtw95JRw8O294E5eMQxLXjsX1QyCGHNH+y9BktNjBe8Mki1oEY4eiNebDR7bGpCTZWulAVwOEi8XGd9KpfGaH03k\\\\\\\/A9TAyRviOxtLn\\\\\\\/j2RjN4QPymt6NOvOv1Epn9wE2jiicT3m7A23xtcqr0Taqq9sC2epldGkI\\\\\\\/wxMmMDhl3yU\\\\\\\/Ss9UjIdoN5evMKr7dZSNFKq90CArEdvVr2T3ZQqhhuJyiecRzt6d5cRfayk3s2GfKTH5HlkfXKFZz76de5MGVkKQZlwHQORdx6tl6LEtFylikNzDWD\\\\\\\/oYtGBBDLVnpOPUZzNS1PDtn8XHUvohyMSiewmhuuMF1\\\\\\\/KW3fsRRVKDRq9Qa+PMVdZxTWix3dmRyNv5IEAF2bJxl4wCU0ii2cWWVw+t3g9pmL2XqsSSfUXFa53moNOjNkazVofwp8p4okFK05tB75RyS+FBzCCCmoF7Pnr7GlZb1YwseO1nrNpkQ\\\\\\\/kBVZKwwlTE70I3dmAGEuteW7uCCqZHMXl47CxdFm65hygUKfSy7zMwI4axj1tbkNVXxrmr59dPsPKGWj45k5aY\\\\\\\/tSZILHYSE83V30wTQI6jUTGFW7PBCdpJyadlKEDO1nuhFxAlUwh27unLDTtx55QdiwAvkpCCRLV6wvIxjmT6KuXrSM7lV23E+8ANQs1EGMJtOyIi8Cq62s2wKIGC3xgy5\\\\\\\/7Vzt2XXGJ\\\\\\\/WfSRUUK4PDGifvqJMkD0ZRO778eODO7423rooiIDaFDKGvoEEmlreqaCj62PoX+m15mkm5RTnuYYJlTCpJy2eiL+E63rjSNCrlRsiz7ubAI3KeFi4QnhFkyxFQ\\\\\\\/77hyStZTp0NSmq\\\\\\\/6gcTwsBKqHwPDFN4LV3vxLYui4qeDMEuNqd9rfR1ajJz22EfiYujMML7z2xc3vHidA9EYr+tPKGN5GFSXIFHlhFdUKs5ydT3ZCTCJfsY0u9\\\\\\\/ZFOFwdoVfb0bca2AL+86cBppFkvL17I2WvZvakL\\\\\\\/aprmVR2i9oGuRJ1kgJMLdUWZzDnc0Dnl4pKi\\\\\\\/j++qmMo5vAV4ArsYFd5a\\\\\\\/i5apnt1ntycg\\\\\\\/6Nzkz+MbNgtq3CrhzA1YgBqfcANb\\\\\\\/aIDSbv0O9K1LOAINWJsIHTreI+y0V7sA4OA4yHrBELhEp\\\\\\\/AbhT8pC3Z9ZSI9Voclu7l94heb8SXZUqdy176szozH2oDsbPttr0UuozEkil0+BgEPMD1Xw\\\\\\\/SWxltUdVOhX454oLhbhDiuWeNX4cG5HGUnfbq6ACdtddlAVzmMU96o4p+M5mPlf4hX4gbEYwgoHe3z+OpbbrEmhF85oZthF6Cszyvv3FT2lMgK+uAlJpCBd7t8Jnna8zszObH7HnCiukalUjfTn05e4\\\\\\\/XxFnFPF\\\\\\\/iT\\\\\\\/kLteTcn+HQ98Dy9SmMbZKBR61sH8Je8xCGFYXK3kEMoOxXROeHudK3MYKRL0Iyua\\\\\\\/7fkxMvaJD0NKbmT69ceRxcovqoD1FvwmADQm\\\\\\\/nEcL\\\\\\\/IA3Ka9MiPqEUjk7rxavxboUzAj+QfJ7Urgp0GusNOWj1BRkr9SZ8a2Xo0O7foVQ4oDvQbP263HRjnCCLxzbpfmGSF9cE\\\\\\\/WOSRfuwnmMITGYyjJSSw+ktAU9Zb8SaStnWTNYF6Ywn28RwuQa\\\\\\\/6QZmMMtw5xcL\\\\\\\/hGwR9qFCMvo8TwhYcUeLg+QnpdzTiybp9Z1+Xs+zNa9Vtbqn6ifZXKh2HtQcickJ0jxoj8zledME1QDAx9d\\\\\\\/B\\\\\\\/c2DE9jojFrIJTl+3NrUqB+QnLUp+UIsnp6AoeTbcR+hRU\\\\\\\/Ed6nn8Ne3fPi8nBIfuNu76O5D0sVHMrCBBVtJQwfuPDPldYD3s7kp2B1fRSFaaNE2h5ygWeFQ7gWUt60aXAExEoDIcx\\\\\\\/6wybeivBWu9sm4VWuzmxhZIUEjfpwzcMWmc9TUI06cC5p9Lbhsv4KrTmPMN1ELIPQpQxFvujOw\\\\\\\/RcEis\\\\\\\/Xm4mlkRvhPWhkUkNn6MB3DY15I27DZYbJwZOkq5oORUCzXX3IS9PY36ygbKT5GP2soLh5M4ubHtXBsuhDL\\\\\\\/lJtTgIu9r8xhtwYm7B5FsppPeiSXv2Xu7OIM8TTw1nov4NLqcVSf5mjJRojJTM1uRPtAklUw3dYfN9PcTLvntH+FBxPU2vi8wmpsvtRtV29HWleTStwoR7Chijja\\\\\\\/wzRHpx3s6Uji2GS0Iak1wZGx5ui2WzDSXgaqzpAumgqYpTqlLQXYJFm6C15noIk4TwAWTBvU7BEMft9ETUEOFZV8qlkVZANLKxqwmmo1prmY6ltHv+toJsiUxrIT0P1SwYDsdBYKV0X6nLQSnYb+nlqasI5b4gqmbroQPif26DUVJmOMMMUQggIXPPyb3CU\\\\\\\/xghIaXscSv\\\\\\\/BC59VF6KlogTl6aWBvjwHQUJiMpQQpdyNsLsvl1BGo7bYWmB8boHnnLyKUIWwqll6WCEoMtPdRL94HqelHlOzWZO5RInZ6Yh9vxo2p5R1V4bAyNzweE\\\\\\\/SLR599uWeaiE9qoUjI8+q2+S5SeFX6Cdtp8bJEsdl1HLV+P68mi5Ryf\\\\\\\/hbvyzmo1OXOWPVXGtX5P41if3LjGq4rrvIVdRCPrlJdLnTbL5MORjnXnuxMH3CLpFxQJo4FIqqugn5lhZijdrLlWqU2ngBtcGNp\\\\\\\/lai+\\\\\\\/e2Gt+SoaNq\\\\\\\/IoVaXCUMehrM3L01kShWAjJVAwnpPhpuGC7cCiUFA3+OPOJE6oBjcWW75X88n74aIX9WhT1QhXK7XN3rTSIyUt73qX1K\\\\\\\/lVCaZqkZpugMfKNnfAJOlmvge2gBBmCzzmh4K1iBIrwF3XzwWBJ7gUb0PA03gVofMFJMWio8KksgnpAy+9rOFaVXuAi7AQuk0vNggFPSWjnNBa3R30otwEsUeuhNl+V+UVhm0U2WkEGjQ\\\\\\\/bsX16ZRnnQF34CDQ9zAuiw3OcYsvpAsEdLDah9CWdEuv9GzvNZM3Wlw5uCC+2E+8P\\\\\\\/cx5VISBPCCgswJsoxJyf95pNEeEsNEBOsfLhQvgw9sSdLkgnuVsHb1Amd7TGc4pszZVHI1vakD5PJ1utzGw139GPMNsJEikpBSP8MmZSeIvfTkMsUWy5VQpcfoulc\\\\\\\/9r3n9J83rSt+iWp2ncXiJFdHEZearhYSxfFxd7pLKI+P3+cCn8B8ZSYHb1dBrCA3a0L5NWkcP8rF42\\\\\\\/08\\\\\\\/C3VeZklBziUB+qsQ7pvzdZxo2NzI8Q+bZa+XZ6ULosvPDFsiZDxWMDj3Wh+qYAwffGCkIq\\\\\\\/wqnPe3eDS\\\\\\\/9muGXBhDXVZ6eLVx\\\\\\\/0Lpht1o2zXDAst2dhnB4UTD+OyewEo1wVAtF9Hn8Hxz\\\\\\\/LvbR02MiwXJcSIVtYfdKQx+Wmnqc+UxGSfb+5ld1YSZEfGyjQaGHu\\\\\\\/h8BLTPcrF6kMfymYKEbJzgz1YLoMp+Nl1v8Gw9OSHICvpT6tprB7qujSvHgTMXRfZx8bMZCmIWQQB7834GRl5RddsP0oY4ezK6QGnJMCCm6Q+OdEgl238mkX7zzi+YHc2ogDQORBZBOPmoO1LKcMVNC1+nDx6vPHafCmYYXrQbmVm31lvYj2tj0J+f14nR55SuWm9PmyuZfBwiKyTxJlC7MYT+1PtAW0dlX+YhIlF84PESJCsgnjZ9+vCVbPLQCk88OjUuSzhRi7lDknw+NmWkas2XLOyxGEYs8tt47TREYZbIU0HvSBc5EGrCdibK5JDPJ5qTPbTp08Tn\\\\\\\/DdfQeYPAyzFUc8SQhCVIm29+UEUnJt3SR5QqmvOH8I+SxP47oTXbgV11EQkAZ+pEeQLePtmiGxAj4s3m6f5HhrjdB6qO4DSSBEuB4\\\\\\\/KHvTNK9sDN675Kgim7wUOjggvJm4JV3XcAOYf\\\\\\\/6fFT3m3sv+9PbBxebWuENgNL8wNcfgRFuo6uT7gXTieOk9YXCS6JZVQ7y4t41oJv3qpiWCaqi9PadyfiIRYwpxPJojqNg69FDmN551u1Q9VYkmOR\\\\\\\/kqwInKIqDdVNrYzMBddl4hXNH\\\\\\\/0xfB+L0I6TpHHQupHUXG+rsk13uHIAi5J4uJljl5ynsfHRBkKj+NFc+QpsNGw44RILe5Ik1zZLNNl6gXedH06CBn+mHN+B6834WUaj9nQkKTc00KcVScWL9Oa8ZQLaUdXOrnQDqAcbdPds7PHQ7ik7258+7OMaCeKcQJzyuDMpVBTSL1yCFfkEUZ4CiJmkIaH9rGAML6fxpXWR8karsZoj83P6H9I8fhtXW66QBrMznJ07sju\\\\\\\/tHEcihBNMwh3x5\\\\\\\/pZvUAuKlXaHugDXhwEndOb4f8O1\\\\\\\/Zfa0JHzPSX\\\\\\\/5LyUjMXp+tc+bw0Bf4kxbO0xdlvKJCkSAbbXNCf69FzT7N9DCU+seWwCAGW6rX585on62AsX9xb+LDos3w8ie6WI9B1sLefNJYuvS3z98uwXzkXUY5VAYZNhPlvsC9Ws+HekhwuEyXgrrQOoLszsPZrnlwxstnrqhNmfxAD\\\\\\\/ge4p1FMYE3MsGNSHpVdpRyecm5NRFD3x6xXFIUfqED3cheWSd710phiOrZPO6fX64\\\\\\\/wX9GTroiSHRMTelysJ8BGd1geZmwYZPYRIhkE75KAWWU1rNFCm5JDc\\\\\\\/Oa1ZbNcSsfsGEthPWF8Sds5Ke0FIT\\\\\\\/DJxwJ6LdO1FRqb5SwvfA6NSHPBz2yms2Jnu8mMOr6KgJyhoaKIBSoQBPb2dh1nNi64CG0GECmmS3kmeJV7tIH4HdLq5UR8BHyVzV0ERRM6y9m4jQtUqrcna9ntuAPyWK9+lbzd\\\\\\\/hZm+gbhXQwcNlbDjswumkrijbjoBvhLuMC9HwVo6JxP57g14MQz3WbmmqzW8zdTbROlaHHS61jtKyIJn97put0n4XBsVN74wnwZRxQa2r0qkVT5QnYjRsucZY4Y4VAjzly847NplDMbEFlaL8agAW8Uohf5CVn7azMBcr9jnuB+Z\\\\\\\/m3y\\\\\\\/reCNGcacKaNq4R31+zSYn97n38sga\\\\\\\/+\\\\\\\/RN0rQsNH+hDGtnnszvIIfnLgY1zMBqVr7C04zNulp0INqzfM0eOc+uyM+MhrDmEs9rqboo210WsKuEm0tY0IS3P+Oms+frBybcKNTqTVQ0TuX2BWq1dRQWEet9nnDerzLSgnihzGVKnHdan6euRKY4c9j+BfSBKAJbXnwzQ4grbyD2IjLZ8POLOSh47m4\\\\\\\/c++T8lUCQuKEWCFLHE6+mb39IenhQfXIKpRY\\\\\\\/RDHDqzF1xTAaUm+25Z5sgYPeD\\\\\\\/Fa4+m3YT9Pt5mIspdY57yBJEPe+ZfBG34hy17fhBOr+gWOzi8G1pIYbhbQ0FTcE42J1cLdYc+uYHL\\\\\\\/QDRTkOYdsb8xfHvFP6wDAXcMmJPUkyyhCPgT99uURkTs0pFOS1GkKSKf63kSG6maCTEASfIpto\\\\\\\/VRxbQOFZQdiCwXy\\\\\\\/IjD2nrF\\\\\\\/0ma+8rT3qBl7fYdfVUZBYA1Se+VaciU4bOLFWvhRrHW0+GtgVeeaaE1rGoJwi+HrTfsksJodoeL6ezzL1zilw\\\\\\\/SSjmTe3IDxkwJyfNjlWvwKEvc8ZkM7DSgeSqcYQrl782HMyOr0mHcGQj\\\\\\\/5Y6ay\\\\\\\/6vnwiLNWGiPOpcvq1nWpGZ0Fxaoj4+fJP8FunmhGRGWUNGl7WFAgmZKhZz\\\\\\\/caRZj7BZWkekXE1I4WNAwX1K5hsHfMV0OBNuHchdvzC1EF0fneWrlENjmqiiXHRIO+PV9PRkVxMhG2E899B1gQrGuO\\\\\\\/DeHhZmj4Iarg7gWMc67q2pQdUEJDzTkwTTQgBcJTY6raWUVeZBqCto3NXIo+pfrgGfh\\\\\\\/UXgpsgax0H6v0BaI7NZrpcLambfaIA1PwxmEUgscpLVEEKCaL7Uv89wIau2eE9Qhql5EreEqabhQzqS9\\\\\\\/gq7jo8t87wejDw69iovZmR8OPcoP5mTs3gufh5C+5HPsfemrJ6U3MtuNHRSKKpjJILRSsGyJP9roB5900zkkke2yDm5ninAe1iceiuF87U1LEsVeEw9MrgHxMAre4U5KnG28vdEG7jqknJ1oEOV7h3I9hyaQdhdN+IkO29xEISFTg\\\\\\\/BW6bC8JWupJbM4sc66pc3WKJSlqmE1yajTvkbCzTcUbNMI7L1HfyNRf5v5l+DoRMauUNLyPKYnNuVfJoYUUUYEdW0WK6FlR2d50R2VoPQxKGW+YQIYfU6bW2R2c1nTv8xxkppwymcG7bJnhYzeroz68j4jzRJhzpzsMMBsjLGqmkag6QSxLc1ZmmjqvtI53ZCcAWU\\\\\\\/41QbPvsZIL1DXYp6bK5ZWewe+LGEgGKYHmJTBBipeBce\\\\\\\/XZZi0hwbEJTp4bDA0JSwhjDOKXtybUDdzRbGR+cXonOYu1+Eb\\\\\\\/fvbO152uXjzRBiXjEadx6mpSefpGrytOFNRfjF1nKzxMLqyDe3dtSK4mNm8c5EpMEhBATfYaeBZbeTtvQdW0Qn+xDhuqBC\\\\\\\/N2Op2BLQwKVF8NuaMzjcgIn4NzX2eO5NGCXyc+rzO56C7MKR4boqxj4TRvH8HChCXz508IWXcz8QTAYXrQg7Rn37MRVrkTE7egXA1pA1ypvyrT0pfQszvqC1craoIkDi8VODr9mUZ1WNQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q8gd2uwrqHCCzMFkvvPUpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dd1e86c3b1aff434ccaf15d1329391ad5bec7f5a670f133b46602fe9a7929ec7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220cxhnJOlGsHty8BLxrNwd+YFySt2\\\\\\\/Xs45QrRak43VvlvfNy7Jq7u8ay8sEFYLBhgk3CBbkDwycg9Qhqg21NrvBP4HNWw0nFZgPQ9rlAPwKa8J4rQLIAS\\\\\\\/gwO891d5zDFpr8j0NbzaW9SL\\\\\\\/VOUvapM7rhQlb0LHv9FfPDiFG8JwRWtpCV+cK13m5T4uSoG4ATUitLJezsqJRpFFpNDRIJHhFpwzZVAmcS9Avw4yk+G5T7Lpb31NURtLmuj94UoSrz7RjrXBFxkBO19DSrwUTEepRMaHopXqbQ5xB23mmKYQyZgQhgstsU5XT0hnVriZHBNNNhcgNEIXrJusd1fHRn9hDtbFAspRLKM\\\\\\\/RVrlRtSvAJkidN3J9wlWnLMogkH+5oAmNv96NI6lqtEHG9lsBZIdDNwVJ9or0qQKs92sW1UoDtqXJFWeKAuZWuXyp2K\\\\\\\/GgJCSwV9MLap41Ra31pnkvG0uz+df4eInZhN46t2oz\\\\\\\/nqGKHRCHrg25nVfleZNud04VfA\\\\\\\/fasbryI0xVI3MPmUgYZ82dAf\\\\\\\/Jlv8WyGJyDHMEwNRFwHD1\\\\\\\/5G9tTcMTVuPQqu4A\\\\\\\/3fiPlebIaxjmIFXbn0cVk93\\\\\\\/rEVTCZ\\\\\\\/4KI+zoniEV+IB47cIY22j67wIbD0yYL\\\\\\\/CMz1ZsWMOuygHgVSIxR0iegJUoHqW3bOk0gft6b+cVQAdq31CVzfRoDwOxG4FmvoxLK9sRBvkTjO9G1Ye7XnkZFLrD+ZJUBDSf5WEu\\\\\\\/0E5BsjoGzNhOQaawyBmc\\\\\\\/igCwM+r4LGE\\\\\\\/rk8HFzk1iR6d5YzqmseZs6bgTUtIHK3r8\\\\\\\/\\\\\\\/VAqqJZelXG\\\\\\\/dV5eea+d9VfypKNivfJjWwIYkXeNm32CmoPALFRgMS2thoVaiS+x8UMk9aaZwbFz6gPvhpqr\\\\\\\/AGRZ6Xbdm5g6dnEDnYksBdw8Yrrbe88iJHT4XPI4McSEBHgy7MienHqMHWP+DftX8dYq9u\\\\\\\/47byK7rrfGJpvNfwYN8ruGJBgbvCzyZAydPW4BuvmAC7YKyQ+Ne5jSOP2MfIijSCPu+11EalqzxNCutskjoTfgXPAoSDWUDWbZFSm7bK\\\\\\\/WuL6tgGQ7yDafa6uQMbE8kiRb0zwGoP+z6dywHSuZBs7sLwJxQxN\\\\\\\/RFmngXEtiF3AaN7VxFR2C5cenrt45FP5JerdFpaTJREdHe8+pBoEO2z7w7tIJvf5MxeRxdHoTFvUL2K\\\\\\\/u0ZqpofNX5MMP+E1fuW9x4U\\\\\\\/CR1Iti24d9nkbyNa58N4v+m92Vc9b9ocr7KqANlkGdorNYk6nA0aFBBE\\\\\\\/n7qBGjUxQ2HntiS3CksgWQI4LgNPEMfaldm2AD+AgZXnoA3UBiVK1ox1kBxj96qagplyfLMO+ktTod0tS8PV3\\\\\\\/OBDB87g3vxkUwwfUt6AnX9QKp5ZnYxbnaYV+gDGfelTQKu4DTzhgsNkjWSP7IGytISuSNsA7RlJ2Vw6J73wYNrzJkqE4LVgnqZMVc5j5KbtRl5ITyqoYTl\\\\\\\/kpFkbTdJxE0f3Wi06Zb4ESdkHihAAqFjt8bZ1hlqEpgXLXaKpML+czPAXvaAP2YuaNMfAXkwRIiUqXWWV+jEFmlbkzkxuAr7dN9G\\\\\\\/\\\\\\\/M2Vw4jDKOY5rJRR92Tw47A2JLfejiAxXJccvikAk7MvgzV9frKEj12rU5LLBLjwbOYkJnRXmG4IncamHbksZ+Ahk\\\\\\\/d1Ujd+OhNBvEXQNNlWYCyUKFNcSi6moeavrIAedsWhPJ3PuqfcX+LXt\\\\\\\/8\\\\\\\/xAjbkE7zg68hfrNdDG2T7Ltn+i\\\\\\\/nXx2o45FwwUT7JVGdlQeJYbuQinkYGVUfJQVcqUXrVIRsKlsCZ8lu6cqMevvozFDVHSSDA6Nujtj8ujVq65DUepaSZv5UXaeD+bfmPEBLnTdPimUbCEeXdJudzOs0X4tm7yZNWwQB2Zge+ZQhLejNtqM47bkSLYaRRnacdvyqd3jw7GaIl48caLGQqj5Jw1gmieRaj0I7Gnn8JBr9LbyOzfHgOPSOhjrKqdNJO4k\\\\\\\/T7qA==\\\u0022,\\\u0022iv\\\u0022:\\\u00228PP61wIjPiMMVnWYSurmAw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022694835e5dd495080276305d0c35b1b3dab9ef7a9fa35086959cd804499d7cdc2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485324 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2259" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CEzOCezOcA08Qqe9W5qovK05kYsu_uYGPlYKAtrdxxQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2259" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nejEqSSYol43jv0Gs_J7f5g6uz3rPE6iJVo65vKq-OI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2249" title="Definition of a RACK1 Interaction Network in Drosophila melanogaster Using SWATH-MS" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2271" title="A Systemic Analysis of Transcriptomic and Epigenomic Data To Reveal Regulation Patterns for Complex Disease" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485324" id="node485324" data-pisa="ggg;7/7/2259" data-pisa-master="ggg;g3.117.043000" data-apath="/ggg/7/7/2259.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using RNA-Seq for Genomic Scaffold Placement, Correcting Assemblies, and Genetic Map Creation in a Common <em>Brassica rapa</em> Mapping Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">R. J. Cody Markelz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael F. Covington</span>, <span class="highwire-citation-author" data-delta="2">Marcus T. Brock</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4686-7235" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Upendra K. Devisetty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5759-3175" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel J. Kliebenstein</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2833-9316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cynthia Weinig</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Julin N. Maloof</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2259-2270;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043000 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.043000" data-atom-uri="/ggg/7/7/2259.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485324" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">R. J. Cody Markelz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BJ.%2BCody%2BMarkelz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Markelz%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BJ.%2BCody%2BMarkelz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael F. Covington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BCovington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covington%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BF.%2BCovington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael F. Covington</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marcus T. Brock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wyoming, Laramie, Wyoming 82072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcus%2BT.%2BBrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brock%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcus%2BT.%2BBrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Upendra K. Devisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Upendra%2BK.%2BDevisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devisetty%20UK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUpendra%2BK.%2BDevisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4686-7235" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Upendra K. Devisetty</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel J. Kliebenstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Sciences, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJ.%2BKliebenstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kliebenstein%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BJ.%2BKliebenstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5759-3175" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel J. Kliebenstein</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Cynthia Weinig </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wyoming, Laramie, Wyoming 82072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cynthia%2BWeinig%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinig%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACynthia%2BWeinig%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2833-9316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cynthia Weinig</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Julin N. Maloof </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julin%2BN.%2BMaloof%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maloof%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulin%2BN.%2BMaloof%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Julin N. Maloof</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/7/2259" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485324" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485324/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2259.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485324" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485324/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2259.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485324" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485324/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2259.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485324" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485324/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/7/2259.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485324" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485324/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3"><em>Brassica rapa</em> is a model species for agronomic, ecological, evolutionary, and translational studies. Here, we describe high-density SNP discovery and genetic map construction for a <em>B. rapa</em> recombinant inbred line (RIL) population derived from field collected RNA sequencing (RNA-Seq) data. This high-density genotype data enables the detection and correction of putative genome misassemblies and accurate assignment of scaffold sequences to their likely genomic locations. These assembly improvements represent 7.1–8.0% of the annotated <em>B. rapa</em> genome. We demonstrate how using this new resource leads to a significant improvement for QTL analysis over the current low-density genetic map. Improvements are achieved by the increased mapping resolution and by having known genomic coordinates to anchor the markers for candidate gene discovery. These new molecular resources and improvements in the genome annotation will benefit the Brassicaceae genomics community and may help guide other communities in fine-tuning genome annotations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-Seq</a></li><li class="kwd"><a href="/keyword/genetic-map" class="hw-term hw-article-keyword hw-article-keyword-genetic-map" rel="nofollow">genetic map</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/brassica-rapa" class="hw-term hw-article-keyword hw-article-keyword-brassica-rapa" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Brassica rapa</em></a></li><li class="kwd"><a href="/keyword/genome-assembly-correction" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly-correction" rel="nofollow">genome assembly correction</a></li></ul><p id="p-4">The <em>Brassica</em> genus is important for human diets throughout Asia, providing micronutrients, up to 12% of oil calories, and a wide diversity of agricultural products (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dixon 2007</a>; X. Wang <em>et al.</em> 2011). Within this genus, genome sequences have recently been published for <em>Brassica napus</em>, <em>B. rapa</em>, and <em>B. oleracea</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chalhoub <em>et al.</em> 2014</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Liu <em>et al.</em> 2014</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Parkin <em>et al.</em> 2014</a>; X. Wang <em>et al.</em> 2011; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Yang <em>et al.</em> 2016</a>). <em>B. rapa</em> is a physiologically and morphologically diverse diploid species that has 87% gene exon similarity to the model plant <em>Arabidopsis thaliana</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheng <em>et al.</em> 2013</a>). This makes <em>B. rapa</em> an excellent species for comparing and translating knowledge of biological processes from <em>Arabidopsis</em> to a crop species. For example, homologous <em>Arabidopsis</em> gene information has been used to infer the action of <em>B. rapa</em> genes in glucosinolate metabolism (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li and Quiros 2001</a>; H. Wang <em>et al.</em> 2011), flowering time, leaf development (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baker <em>et al.</em> 2015</a>), and seed yield (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brock <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dechaine <em>et al.</em> 2014</a>). All of these important traits contribute to our understanding of plant growth in agricultural settings and the underlying genetic understanding of these traits is made possible by a reference genome sequence (X. Wang <em>et al.</em> 2011), gene annotation information (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cheng <em>et al.</em> 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Devisetty <em>et al.</em> 2014</a>), and genetic mapping populations (<em>e.g.</em>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Iniguez-Luy <em>et al.</em> 2009</a>).</p><p id="p-5">The annotated <em>B. rapa</em> genome assembly is 283.8 Mb spread over 10 chromosomes A01–10 (X. Wang <em>et al.</em> 2011). Although the current genome is diploid, there are three ancient subgenomes derived from genome duplication events. These subgenomes are designated as least fractionated (LF), most fractionated one (MF1), and most fractionated two (MF2), corresponding to the fraction of gene loss in each subgenome (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> 2012</a>; X. Wang <em>et al.</em> 2011). These three subgenomes share many paralogous genes and contiguous regions complicating genome assembly. This has prevented ∼10.8% of the gene-containing genomic scaffolds in version 1.5 of the genome (<a href="http://brassicadb.org/brad/index.php">http://brassicadb.org/brad/index.php</a>) from being assigned to chromosomes. The lack of chromosomal assignment is largely because these scaffolds have no molecular markers that would have enabled their placement on the genetic map. This suggests that identifying more markers can help to make the <em>B. rapa</em> genome assembly more comprehensive (X. Wang <em>et al.</em> 2011).</p><p id="p-6">For this study, we utilized an existing RIL population of <em>B. rapa</em> that has been used extensively for QTL mapping of physiological, developmental, and evolutionarily important traits [BraIRRi; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Baker <em>et al.</em> 2015</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Brock <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dechaine <em>et al.</em> 2007</a>, <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">2014</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edwards <em>et al.</em> 2011</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Iniguez-Luy <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lou <em>et al.</em> 2011</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">2012</a>]. Recently, we completed deep RNA-Seq of the parents of the BraIRRi population, providing a large SNP set and improved gene annotation information (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Devisetty <em>et al.</em> 2014</a>). Using a new set of RNA-Seq data collected on the entire population, we extend these SNP discovery methods to 124 genotypes in the population for placing scaffolds, correcting assemblies, and the creation of a saturated genetic map.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant growth and tissue collection</h3><p id="p-7">The field site for plant growth was located at the University of Wyoming Agricultural Experimental Station in Laramie, Wyoming. This study focused on 124 RILs and the two parental genotypes (R500 and IMB211) of the BraIRRi population (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Iniguez-Luy <em>et al.</em> 2009</a>). The BraIRRi population is derived from the R500 yellow sarson oilseed variety and the IMB211 Wisconsin Fast Plant derivative. Individual replicates of each RIL were sown into peat pots filled with field soil and topped with 1 cm LP5 potting soil (Sun Gro Horticulture, Agawam, MA). Seeds were planted in the first week of June 2011, and pots were transplanted to the field 2.5 wk later following established protocols (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Dechaine <em>et al.</em> 2014</a>). One biological replicate of each genotype was planted into each of five fully randomized blocks. After plants were established in the field for 3 wk, apical meristem tissue was collected from individual replicate plants into 1.5 ml Eppendorf tubes, immediately flash frozen in liquid nitrogen, and stored at −80° until RNA-Seq library preparation. Apical meristem tissue was chosen as part of an overlapping RNA-Seq expression QTL project (Markelz <em>et al.</em>, personal communication).</p></div><div id="sec-3" class="subsection"><h3>RNA-Seq library preparation and sequencing</h3><p id="p-8">RNA-Seq libraries were prepared using a high-throughput Illumina RNA-Seq library extraction protocol (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kumar <em>et al.</em> 2012</a>). The enriched libraries were then quantified on an Analyst Plate Reader (LJL Biosystems) using SYBR Green I reagent (Invitrogen). Once the concentration of libraries was determined, a single pool of all the RNA-Seq libraries within each block was made. The pooled libraries were run on a Bioanalyzer (Agilent, Santa Clara) to determine the average product size for each pool. Each pool was adjusted to a final concentration of 20 nM and sequenced on seven lanes on an Illumina Hi-Sequation 2000 flow cell as 50 bp single-end reads. Any failed samples from the five blocks were run on two additional lanes.</p></div><div id="sec-4" class="subsection"><h3>RNA-Seq read processing</h3><p id="p-9">Preprocessing and mapping of Illumina RNA-Seq raw reads was done as described in detail in <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Devisetty <em>et al.</em> (2014)</a> with one exception. The raw reads were quality filtered with FASTX tool kit’s (<a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>) fastq_quality_filter with parameters [−q 20, −p 95]. The qualified demultiplexed reads were then mapped to the <em>B. rapa</em> reference genome (Chiifu version 1.5) using BWA v0.6.1-r104 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li and Durbin 2009</a>) with parameters [bwa_n 0.04] and the unmapped reads were, in turn, mapped with TopHat with parameters [splice-mismatches 1, max-multihits 1, segment-length 22, butterfly-search, max-intron-length 5000, library-type fr-unstranded]. Finally, mapped reads from both BWA and TopHat were combined for genotyping purposes and quality controlled (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS1.csv">Table S1</a>).</p></div><div id="sec-5" class="subsection"><h3>Population-based polymorphism identification</h3><p id="p-10">Variant Call Format (VCF) files were generated for each of five replicate blocks of samples using samtools and bcftools. These tools were run as “samtools mpileup -E -u -f Brapa_sequence_v1.5.fa [all alignment files for the current block] | bcftools view -bvcg - | vcfutils.pl varFilter.”</p><p id="p-11">The VCF files were summarized using “summarize-v<em>cf</em>.pl” Perl script (<a href="https://github.com/mfcovington/snps-from-rils">https://github.com/mfcovington/snps-from-rils</a>). For each block of replicates, this script (run using the parameters: “–observed_cutoff 0.3–af1_min 0.3”) ignores INDELs and variant positions with &gt; 2 alleles, ignores variants with site allele frequency values too far from 0.5 (≥ 0.7 or ≤ 0.3), and ignores variants with missing information in 30% or more of the population. For variants that passed these filters, the numbers of reads matching the reference and the number of alternate allele reads were recorded in a VCF summary file.</p><p id="p-12">These VCF summary files from the different replicate blocks were merged using the “merge-vcf-summaries.pl” (<a href="https://github.com/mfcovington/snps-from-rils">https://github.com/mfcovington/snps-from-rils</a>) Perl script. Using the default parameters (“–replicate_count_min 2–ratio_min 0.9”), this script merges the information in the VCF summaries and records a putative SNP as an actual SNP if the variant is identified as a SNP in at least two replicate blocks and if the proportion of reads matching the major allele is at least 0.9. This was done on a RIL-by-RIL basis.</p></div><div id="sec-6" class="subsection"><h3>Genotyping, plotting, and identification of genotype bins</h3><p id="p-13">The Perl script “extract+genotype_pileups.pl” (<a href="https://github.com/mfcovington/detect-boundaries">https://github.com/mfcovington/detect-boundaries</a>) was used with the “–no_nr” parameter to extract genotype information from the RNA-Seq alignments at each SNP location for each member of the RIL population. The resulting genotype files were used to detect and remove SNPs with excessive noise.</p><p id="p-14">Due to the crossing scheme used to create the RIL population, each individual is expected to be nearly homozygous for one parent or the other. The “filter-noisy-SNPs.pl” (<a href="https://github.com/mfcovington/noise-reduction-for-snps-from-pop">https://github.com/mfcovington/noise-reduction-for-snps-from-pop</a>) Perl script performs noise reduction for SNPs derived from such a population. It does this by identifying and ignoring positions that have an overrepresentation of heterozygosity in individual lines across the entire population. Using the default parameters (“–cov_min 3–homo_ratio_min 0.9–sample_ratio_min 0.9”), SNPs were discarded as noisy if &gt; 10% of the lines in the population showed evidence of heterozygosity as defined by a line having at least three reads per SNP position with a major allele with a ratio &lt; 0.9.</p><p id="p-15">After noise reduction, the “extract+genotype_pileups.pl” (<a href="https://github.com/mfcovington/SNPTools">https://github.com/mfcovington/SNPTools</a>) Perl script was rerun without the “–no_nr” parameter for each RIL. The resulting genotype files were used to create genotype plots using the “genoplot_by_id.pl” Perl script (<a href="https://github.com/mfcovington/SNPTools">https://github.com/mfcovington/SNPTools</a>) and to define genotype bins for the individual RILs.</p><p id="p-16">The “filter-snps.pl” Perl script (<a href="https://github.com/mfcovington/detect-boundaries">https://github.com/mfcovington/detect-boundaries</a>) was used to identify regions of adjacent SNPs with alleles from the same genotype. Using the default parameters (“–min_cov 10–min_momentum 10–min_ratio 0.9–offset_het 0.2”), it detects boundaries between genotype bins when there is a sliding window of ≥ 10 SNPs. Within each sliding window, a depth of at least 10 reads each exhibit major allelic ratios of at least 0.9. The major allele represents the opposite genotype from the previous bin (or exhibit major allelic ratios &lt; 0.7 for transitions from regions of homozygosity to those of heterozygosity). For each member of the RIL population, this script generates one file with a boundary between genotype bins.</p><p id="p-17">The “fine-tune-boundaries.pl” Perl script (<a href="https://github.com/mfcovington/detect-boundaries">https://github.com/mfcovington/detect-boundaries</a>) is an automated tool for rapid, fine-scale human curation of boundaries between genotype bins that we used for the RIL population. As described in <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Devisetty <em>et al.</em> (2014)</a>, “This command-line tool displays color-coded genotype data together with the currently-defined bin boundaries. Using shortcut keys, the operator can quickly and easily approve or fine-tune a boundary (at which point, the next boundary is instantly displayed for approval).”</p><p id="p-18">The “merge-boundaries.pl” Perl script (<a href="https://github.com/mfcovington/detect-boundaries">https://github.com/mfcovington/detect-boundaries</a>) was used to merge all of the boundaries in the collection of the boundaries files that were generated by “filter-snps.pl” and “fine-tune-boundaries.pl.” A comprehensive list of bins and their locations resulting from the merge are written to a file: bins.tsv. The script also prints the boundary and bin stats (count, min size, max size, and mean size) to the screen to allow visual analysis of the resulting file. This information was used for human curation of the boundaries.</p><p id="p-19">The “get-genotypes-for-bins.pl” Perl script (<a href="https://github.com/mfcovington/detect-boundaries">https://github.com/mfcovington/detect-boundaries</a>) was used to convert the comprehensive bins file and all the individual boundaries files into a summary of bins and their locations across the genome and their genotypes across the entire RIL population (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS2.txt">Table S2</a>).</p><p id="p-20">Composite genotype plots (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) were created using the “plot-composite-map.R” R script (<a href="https://github.com/mfcovington/detect-boundaries">https://github.com/mfcovington/detect-boundaries</a>).</p></div><div id="sec-7" class="subsection"><h3>Validating and reassigning genomic scaffolds</h3><p id="p-21">Using the genotypic value for each genotype bin across the RILs, we calculated the asymmetric binary distance between all central SNP pairs using the dist(method = “binary”) function in <em>R</em>. The pairwise correlation matrix was then ordered by maximal correlations to place the map in a linear order and compared to the predicted bin order based on version 1.5 of the <em>B. rapa</em> genome. Comparisons between v1.5 of the genome and binary distance plots were manually inspected to ensure proper placement or reassignment.</p></div><div id="sec-8" class="subsection"><h3>Genetic map construction</h3><p id="p-22">Because each genotype bin across the RILs represents each observed recombination breakpoint in the population, we used one SNP per genotype bin to create a saturated genetic map. Aside from the possibility of rare, unobserved double crossover events, the mapping resolution in this population is no longer limited by the number of SNPs but instead by recombination events. The genetic map was constructed using the chromosomal position of each of the SNPs as a starting point for marker ordering along the chromosomes. Each chromosome was treated as a large linkage group and each SNP was tested for linkage disequilibrium with all other SNPs using the R/QTL package (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Broman <em>et al.</em> 2003</a>) in the <em>R</em> statistical environment (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">R Core Team 2015</a>). Larger gaps in RNA-Seq information corresponding to low gene density centromeric regions were problematic when ordering markers using the ripple() R/QTL function (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Broman <em>et al.</em> 2003</a>). In chromosomes A08 and A09, after local marker order was established we used the physical position of the SNPs to connect the two arms in the correct orientation.</p></div><div id="sec-9" class="subsection"><h3>QTL comparisons</h3><p id="p-23">To test how increased marker coverage affected QTL mapping and identification for physiological traits, we remapped two traits from (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Brock <em>et al.</em> 2010</a>) that had been mapped using the previous genetic map (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Iniguez-Luy <em>et al.</em> 2009</a>). We used R/QTL (v1.39-5) to compare mapping results derived from the previous and updated genetic maps using the <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Brock <em>et al.</em> (2010)</a> flowering time phenotype data. Specifically, we used the cim() function with three marker covariates and determined LOD significance cutoffs after 1000 permutations.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-24">All the raw data has been deposited in the NCBI Sequence Read Archive (Project: SRP022220). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/FigureS1.pdf">Figure S1</a> shows SNPs, centromeric regions, and gene density across the 10 chromosomes of <em>B. rapa</em>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/FigureS2.pdf">Figure S2</a> contains the genetic map before misplaced markers were reassigned. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS1.csv">Table S1</a> contains the read mapping statistics for each RIL. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS2.txt">Table S2</a> contains the SNP genotyping and genomic position for the entire RIL population. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS3.txt">Table S3</a> contains the genome-wide allele segregation statistics. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS4.txt">Table S4</a> contains RIL population genetic bins and scaffold original and final positions. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS5.txt">Table S5</a> shows the SNP base pair calls on unplaced scaffolds. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS6.csv">Table S6</a> contains the final genetic map of the RIL population. Supporting code for genetic map construction can be found at: <a href="https://github.com/rjcmarkelz/brassica_genetic_map_paper">https://github.com/rjcmarkelz/brassica_genetic_map_paper</a>.</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>R500 <em>vs.</em> IMB211 polymorphism identification</h3><p id="p-25">We performed deep RNA sequencing of 124 individuals of a RIL population derived from a cross between the <em>B. rapa</em> accessions R500 and IMB211 (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Iniguez-Luy <em>et al.</em> 2009</a>). We sequenced five replicates of each RIL and mapped 5.26 million reads per RIL. We had previously identified SNPs and INDELs between R500 and IMB211, the parents of the population (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Devisetty <em>et al.</em> 2014</a>), using v1.2 of the <em>B. rapa</em> genome. This set of R500 <em>vs.</em> IMB211 polymorphisms was used to genotype each member of the RIL population individually. The crossing scheme used to create the RIL population should create homozygous regions of contiguous R500 alleles alternating with homozygous regions of contiguous IMB211 alleles in the different RILs. However, when using the R500 <em>vs.</em> IMB211 polymorphism set to genotype the RILs, there were multiple regions where R500 and IMB211 alleles were randomly interspersed. This suggested that the RIL population might be derived from a parent or parents different to those that we had sequenced.</p><p id="p-26">To test this hypothesis, we merged the sequence data from all RILs and then genotyped the merged data set using SNPs identified by the IMB211 <em>vs.</em> R500 comparison (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The merged data set provided a much better view of segregation of putative parental SNPs in this population. Given the size of the population and the expected recombination frequency and distribution, polymorphisms identified in the true RIL parents should be segregating with approximately equal allelic frequency in this merged data set (black dots in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Most genomic regions did display this expected distribution; however, there were several large regions that were not segregating, but instead were monomorphic for one of the putative parents of the population (indicated as orange or blue dots in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). In other words, SNPs identified as polymorphic between the R500 and IMB211 strains are not segregating in the RILs. Nearly all of these monomorphic regions matched R500 alleles, consistent with the idea that the IMB211 seed strain is not the true parent of the RIL population.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of merged data from all RILs genotyped using the parent-based SNP set (A) and the population-based SNP set (B). Each of the B. rapa 10 chromosomes are displayed (A01&#x2013;A10) with count coverage of each SNP at each physical position on the chromosome in megabases (Mb). The color indicates the relative ratio of coverage between R500 and IMB211 for each SNP. Black is equal coverage, orange is more IMB211, and blue is more R500. RIL, recombinant inbred line; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-592296463" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of merged data from all RILs genotyped using the parent-based SNP set (A) and the population-based SNP set (B). Each of the &lt;em&gt;B. rapa&lt;/em&gt; 10 chromosomes are displayed (A01&#x2013;A10) with count coverage of each SNP at each physical position on the chromosome in megabases (Mb). The color indicates the relative ratio of coverage between R500 and IMB211 for each SNP. Black is equal coverage, orange is more IMB211, and blue is more R500. RIL, recombinant inbred line; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2259/F1.medium.gif" width="287" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/7/2259/F1.medium.gif" width="287" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485448" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Plot of merged data from all RILs genotyped using the parent-based SNP set (A) and the population-based SNP set (B). Each of the <em>B. rapa</em> 10 chromosomes are displayed (A01–A10) with count coverage of each SNP at each physical position on the chromosome in megabases (Mb). The color indicates the relative ratio of coverage between R500 and IMB211 for each SNP. Black is equal coverage, orange is more IMB211, and blue is more R500. RIL, recombinant inbred line; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The primary exception to the expected Mendelian parental allele frequency in the RILs is on the bottom of chromosome A03, where there is a gradual transition from equal R500:IMB211 allelic frequency to nearly all IMB211. The A03 pattern is consistent with segregation distortion within the population, possibly caused by the centromere being located at that end of chromosome A03 (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Cheng <em>et al.</em> 2013</a>). With the lower recombination frequencies commonly observed near centromeres in plants (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Harushima <em>et al.</em> 1998</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Haupt <em>et al.</em> 2001</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sherman and Stack 1995</a>), there could be a meiotic drive allele or a local inversion in this region causing the segregation distortion and this effect could be enhanced by the proximity to the centromere. There is evidence for each of these mechanisms occurring across a wide range of plant species (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Buckler <em>et al.</em> 1999</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fang <em>et al.</em> 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lowry and Willis 2010</a>).</p></div><div id="sec-13" class="subsection"><h3>Population-based SNP discovery</h3><p id="p-29">Due to the uncertainty surrounding the identity of the IMB211 parent of the RIL population, we switched to a population-based approach for SNP discovery. This new strategy involved identifying variants within the RIL population and using the R500 data to assign parental origin for each SNP. Using this approach, we identified 146,027 SNPs across <em>B. rapa</em>’s 10 chromosomes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS2.txt">Table S2</a>). These population-based SNPs segregate at the expected allele frequencies of ∼50/50 throughout the entire genome except at the previously noted end of A03 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS3.txt">Table S3</a>). Over 80% of the genome is within 100 kb of a SNP; however, there are several regions with few or no SNPs. There are two primary reasons for these SNP-free regions. Most are likely gene-poor regions or regions of genes with insufficient expression under our experimental conditions (<em>e.g.</em>, growth conditions, age, tissue, and genotypes; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/FigureS1.pdf">Figure S1</a>). We also found a few regions where there are significant numbers of expressed genes, but no SNPs between members of the RIL population. These regions primarily correspond to the nonvariant regions of <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a> and therefore likely represent regions that are very similar between the seed stocks used to generate this RIL population.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485470/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485470/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">SNP counts at different steps of the SNP discovery pipeline</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Genotyping the RIL population</h3><p id="p-31">Using the per line transcriptomic data, each RIL was genotyped as having either the R500 or IMB211 allele at each of the 149,097 SNPs identified from the population-based SNP discovery pipeline. A representative RIL genotype plot is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An individual plot of a RIL genotyped with the population-based SNP set. Each of the B. rapa 10 chromosomes are displayed (A01&#x2013;A10) with count coverage of each SNP at each physical position on the chromosome in megabases (Mb). The color indicates the relative ratio of coverage between R500 and IMB211 for each SNP. RIL, recombinant inbred line; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-592296463" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An individual plot of a RIL genotyped with the population-based SNP set. Each of the &lt;em&gt;B. rapa&lt;/em&gt; 10 chromosomes are displayed (A01&#x2013;A10) with count coverage of each SNP at each physical position on the chromosome in megabases (Mb). The color indicates the relative ratio of coverage between R500 and IMB211 for each SNP. RIL, recombinant inbred line; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2259/F2.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/7/2259/F2.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485372" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">An individual plot of a RIL genotyped with the population-based SNP set. Each of the <em>B. rapa</em> 10 chromosomes are displayed (A01–A10) with count coverage of each SNP at each physical position on the chromosome in megabases (Mb). The color indicates the relative ratio of coverage between R500 and IMB211 for each SNP. RIL, recombinant inbred line; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Collapsing adjacent SNPs into population-wide genotype bins</h3><p id="p-33">The next step toward creating a new genetic map was to define the largest set of nonredundant SNPs. This is necessary because the 149,097 SNPs in the full data set vastly exceed the expected number of recombination breakpoints in a population of 124 individuals. We developed a method to identify and summarize the “genotype bins” in the population. First, we found all detectable recombination breakpoints for each RIL. Next, we consolidated these breakpoints for the entire population. SNPs that were not adjacent to a recombination breakpoint in any of the RILs were considered redundant and removed. This yielded bins of adjacent SNPs with genotype patterns that differed from neighboring bins for at least one RIL because of a recombination event in that specific RIL. The genotype bins for the RIL population are summarized in a composite population genotype map (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Composite population genotype map with the physical position for each of the 10 chromosomes. Each RIL is represented as a single row displaying the genomic region inherited from IMB211 (orange) or R500 (blue). Small heterozygous regions are represented in black. HET, heterozygous; RIL, recombinant inbred line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-592296463" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Composite population genotype map with the physical position for each of the 10 chromosomes. Each RIL is represented as a single row displaying the genomic region inherited from IMB211 (orange) or R500 (blue). Small heterozygous regions are represented in black. HET, heterozygous; RIL, recombinant inbred line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2259/F3.medium.gif" width="325" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/7/2259/F3.medium.gif" width="325" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485340" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Composite population genotype map with the physical position for each of the 10 chromosomes. Each RIL is represented as a single row displaying the genomic region inherited from IMB211 (orange) or R500 (blue). Small heterozygous regions are represented in black. HET, heterozygous; RIL, recombinant inbred line.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Finding and reassigning misassembled genomic regions</h3><p id="p-35">A first version of the population genetic map revealed several markers that seemed to be misplaced based on physical position, resulting in large genetic distances between them (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/FigureS2.pdf">Figure S2</a>). Given that we have corrected the parental genotyping issues, the most likely explanation for this finding is that these regions represent genome assembly errors. To test this hypothesis, the genotypes of representative SNPs from each bin were used to calculate the asymmetric binary distances between each bin across the population. If the predicted genome position of each bin is correct, the expectation is that each SNP should have the lowest distance to adjacent SNPs in genome coordinates. However, consistent with genome assembly problems, there were a subset of SNPs whose genotypes were more highly correlated with SNPs located elsewhere in the genome rather than with SNPs near their current assigned genomic position (a representative example is shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). To correct these assembly problems, 13 regions consisting of 19 genotypic bins were moved to different genomic locations, and 4 regions consisting of 66 bins were inverted in place at their original position based on asymmetric binary distance (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS4.txt">Table S4</a>). <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">He <em>et al.</em> (2015)</a> also reordered <em>B. rapa</em> scaffolds, although they took a different approach whereby gene coding sequence similarity searches were used to identify, split, and reorder chimeric scaffolds to increase collinearity with pseudomolecules originally ordered using a <em>B. napus</em> linkage map. One difference with the <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">He <em>et al.</em> (2015)</a> method compared to ours is that using a <em>B. napus</em> genetic map to order <em>B. rapa</em> chromosomes could introduce errors if there are chromosomal rearrangements between these two species. Regardless, because <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">He <em>et al.</em> (2015)</a> used version 2.0 of the <em>B. rapa</em> genome, which was not released at the time this manuscript was submitted, it is not possible to directly compare the efficiency of these two approaches.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A representative asymmetric binary distance plot for a single molecular marker, A05-8245839, indicated by the purple arrow. Markers with 90% correlation to A05-8245839 are indicated in red and occur on chromosomes A05 and A08. The group of markers on A05 were moved to A08. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-592296463" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A representative asymmetric binary distance plot for a single molecular marker, A05-8245839, indicated by the purple arrow. Markers with 90% correlation to A05-8245839 are indicated in red and occur on chromosomes A05 and A08. The group of markers on A05 were moved to A08. &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2259/F4.medium.gif" width="440" height="171"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/7/2259/F4.medium.gif" width="440" height="171"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485371" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">A representative asymmetric binary distance plot for a single molecular marker, A05-8245839, indicated by the purple arrow. Markers with 90% correlation to A05-8245839 are indicated in red and occur on chromosomes A05 and A08. The group of markers on A05 were moved to A08. </p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Incorporating scaffold sequences into the genome</h3><p id="p-37">In version 1.5 of the <em>B. rapa</em> genome annotation, there are 40,357 scaffolds that have not been incorporated into any of the 10 chromosomes. These scaffolds range in size from 100 bp to 938 kbp and represent 1411 genes spanning 27.5 Mbp. For comparison, there are 39,609 genes within the 283.8 Mbp of annotated chromosomal sequence. Given that the scaffolds contain about as many genes as would be expected on one-third of an average chromosome, we decided to extend our strategy for fixing genome misassemblies to estimate the approximate chromosomal locations of the scaffolds.</p><p id="p-38">We identified 3070 SNPs across 339 of the 40,357 scaffolds (the remaining scaffolds had no SNPs, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS5.txt">Table S5</a>). To be confident in our placement, we limited ourselves to the 47 scaffolds with 10 or more SNPs. For each of these 47 scaffolds, we were able to identify at least one genomically-defined chromosomal bin that had identical or near identical genotypes. This indicates very close genetic linkage between the unplaced scaffold and the placed genomic bin, allowing us to assign a genomic position but not an orientation to the unplaced scaffold. The incorporated scaffolds range in size from 429 to 884,746 bp and are enriched for larger scaffolds (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The addition of these 47 scaffolds allowed us to incorporate 25% (∼7 Mbp) of the unplaced genomic sequence into the genome, representing 49% (691) of the unplaced scaffold genes (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS4.txt">Table S4</a>). In comparison, <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">He <em>et al.</em> (2015)</a> did not place any orphaned scaffolds into pseudomolecules, although the need for scaffold placement is likely reduced in the genome version (2.0) that was available to them.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Number of SNPs per scaffold. (B) Density distributions of scaffold sizes. Newly incorporated scaffolds are shown in green and unincorporated scaffolds are shown in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-592296463" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Number of SNPs per scaffold. (B) Density distributions of scaffold sizes. Newly incorporated scaffolds are shown in green and unincorporated scaffolds are shown in gray.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2259/F5.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/7/2259/F5.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485376" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">(A) Number of SNPs per scaffold. (B) Density distributions of scaffold sizes. Newly incorporated scaffolds are shown in green and unincorporated scaffolds are shown in gray.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485464/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485464/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Incorporated scaffolds represent a disproportionately high amount of scaffold sequence</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">While most of the incorporated scaffolds represent a single genotype bin, seven scaffolds are comprised of multiple bins. Scaffold000164, for example, includes 65 annotated genes across six distinct genotype bins within its 313.7 kbp sequence. For six of the scaffolds with multiple bins, the bins were closely linked and allowed us to place the scaffold in a single location in the genome. However, one scaffold, Scaffold000191, contained two bins that mapped to two different chromosomes, indicating that it was misassembled. Therefore, we split its two bins and assigned them to the appropriate chromosome locations (5 genes/28.2 kbp to A01 and 24 genes/104.1 kbp to A05).</p><p id="p-42">Possible reasons for the enrichment of larger scaffolds within the set of incorporated scaffolds include: (1) larger scaffolds are more likely to include expressed genes and therefore detectable SNPs and (2) larger scaffolds may be more likely to be accurate representations of a contiguous region within the genome. This second point is based on the assumption that large scaffolds could be assembled, perhaps due to more abundant, more consistent, and/or more convincing experimental support than small scaffolds. Before (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>) and after (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>) plots of genome-wide asymmetric binary distances for each marker pair show that rearranging putative genomic misassemblies and incorporating scaffolds eliminates inconsistencies between genome position and genotypes of adjacent markers.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide asymmetric binary distance plots for each marker compared against every other marker (A01&#x2013;A10). (A) Contains the unplaced genomic scaffold sequences (see A00). Dark blue indicates high correlation (low asymmetric binary distance), while white indicates no correlation. (B) The final position of each marker and scaffold after applying our pipeline." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-592296463" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide asymmetric binary distance plots for each marker compared against every other marker (A01&#x2013;A10). (A) Contains the unplaced genomic scaffold sequences (see A00). Dark blue indicates high correlation (low asymmetric binary distance), while white indicates no correlation. (B) The final position of each marker and scaffold after applying our pipeline.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2259/F6.medium.gif" width="218" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/7/2259/F6.medium.gif" width="218" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485553" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">Genome-wide asymmetric binary distance plots for each marker compared against every other marker (A01–A10). (A) Contains the unplaced genomic scaffold sequences (see A00). Dark blue indicates high correlation (low asymmetric binary distance), while white indicates no correlation. (B) The final position of each marker and scaffold after applying our pipeline.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>High-density genetic map</h3><p id="p-44">From the available SNP data, we were able to create a genetic map with 10 linkage groups corresponding to the 10 chromosomes of <em>B. rapa</em>. The map contains 1482 genotyped markers for 124 RILs and is effectively saturated based on recombination events existing in the population (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/TableS6.csv">Table S6</a>). The new map has an average marker spacing of 0.7 cM and a total map distance of 1045.6 cM. For comparison, the original map contained 225 markers with an average spacing of 3.3 cM (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Iniguez-Luy <em>et al.</em> 2009</a>). This is also compared to a recent map created on a subset of the population, 67 RILs, that had a total of 125 markers derived from microarray probes (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hammond <em>et al.</em> 2011</a>). Having the genetic distance of markers with known genomic coordinates allowed us to fix two additional genome misassemblies resulting in large inversions on chromosomes A09 and A10 (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1/FigureS2.pdf">Figure S2</a>). All of these improvements combined allow us to more accurately map QTL for known phenotypes such as flowering time (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). Lastly, we fitted spline-based regressions for each chromosome to more accurately convert between genetic distance and physical distance (A01 example; <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7, B and C</a>). These conversion equations are helpful for finding candidate genes in significant QTL regions (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fulop <em>et al.</em> 2016</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Physical position vs. genetic position of each marker for chromosome A09 (A and B) and A10 (C and D) using genome version 1.5 (A and C) and fixed inversions using recombination information (B and D). Loess smoothing for converting between genetic and physical distance is displayed by the blue line in (B) and (D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-592296463" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Physical position &lt;em&gt;vs.&lt;/em&gt; genetic position of each marker for chromosome A09 (A and B) and A10 (C and D) using genome version 1.5 (A and C) and fixed inversions using recombination information (B and D). Loess smoothing for converting between genetic and physical distance is displayed by the blue line in (B) and (D).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2259/F7.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/7/2259/F7.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485457" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-45" class="first-child">Physical position <em>vs.</em> genetic position of each marker for chromosome A09 (A and B) and A10 (C and D) using genome version 1.5 (A and C) and fixed inversions using recombination information (B and D). Loess smoothing for converting between genetic and physical distance is displayed by the blue line in (B) and (D).</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Old and new genetic map comparisons. Genetic markers for each chromosome are displayed in centimorgan distance (cM) for the old (A) and new (B) genetic maps. Comparison of likelihood odds (LOD) scores for flowering time quantitative trait loci (QTL) on chromosome A07 using the old (C) and new (D) genetic maps." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-592296463" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Old and new genetic map comparisons. Genetic markers for each chromosome are displayed in centimorgan distance (cM) for the old (A) and new (B) genetic maps. Comparison of likelihood odds (LOD) scores for flowering time quantitative trait loci (QTL) on chromosome A07 using the old (C) and new (D) genetic maps.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2259/F8.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/7/2259/F8.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2259/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485509" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-46" class="first-child">Old and new genetic map comparisons. Genetic markers for each chromosome are displayed in centimorgan distance (cM) for the old (A) and new (B) genetic maps. Comparison of likelihood odds (LOD) scores for flowering time quantitative trait loci (QTL) on chromosome A07 using the old (C) and new (D) genetic maps.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Conclusions</h3><p id="p-47">In this study, we demonstrated the flexibility and power of thoughtfully designed RNA-Seq experiments from tissue collected from a field experiment. RNA is a rich source of biological information that can be utilized beyond expression analysis and transcriptome annotation. It is our hope that these new community resources using RNA-Seq are used to further genome annotation, assembly, and functional analysis of the emerging model crop <em>B. rapa</em>. Our scaffold rearrangement and placement of orphaned scaffolds significantly improves the <em>B. rapa</em> genome (v1.5), but perhaps more importantly we provide a new saturated genetic map for the widely used BraIRRI population with over 1400 molecular markers. These improvements combined with our population-based SNP calling method are a unique contribution to the <em>Brassica</em> community.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">The authors wish to thank members of the Maloof laboratory for helpful discussion and reading of the manuscript. R.J.C.M. was supported by a National Science Foundation (NSF) Postdoctoral Research Fellowship in Biology (IOS-1402495). This research was supported by NSF grant IOS-0923752 to C.W. and J.N.M. The authors declare no conflicts of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-49">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043000/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-50"><em>Communicating editor: A. H. Paterson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-19"><span class="received-label">Received </span>January 19, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-05-09"><span class="accepted-label">Accepted </span>May 9, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Markelz <em>et al.</em></li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markelz</span>  <span class="cit-name-given-names">R. J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covington</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Modeling development and quantitative trait mapping reveal independent genetic modules for leaf size and shape.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D208%26rft.spage%253D257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.2" data-doi="10.1534/genetics.110.119982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dechaine</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iniguez-Luy</span>  <span class="cit-name-given-names">F. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maloof</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stinchcombe</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Floral genetic architecture: an examination of QTL architecture underlying floral (Co)variation across environments.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119982%26rft_id%253Dinfo%253Apmid%252F20837996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni80LzE0NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjI1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.3" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phelps-Durr</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">C. S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawe</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Meiotic drive of chromosomal knobs reshaped the maize genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D153%26rft.issue%253D1%26rft.spage%253D415%26rft.epage%253D426%26rft.atitle%253DMeiotic%2BDrive%2Bof%2BChromosomal%2BKnobs%2BReshaped%2Bthe%2BMaize%2BGenome%26rft_id%253Dinfo%253Apmid%252F10471723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUzLzEvNDE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIyNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.5" data-doi="10.1126/science.1253435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalhoub</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denoeud</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">I. A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Early allopolyploid evolution in the post-Neolithic Brassica napus oilseed genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">950</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChalhoub%26rft.auinit1%253DB.%26rft.volume%253D345%26rft.issue%253D6199%26rft.spage%253D950%26rft.epage%253D953%26rft.atitle%253DEarly%2Ballopolyploid%2Bevolution%2Bin%2Bthe%2Bpost-Neolithic%2BBrassica%2Bnapus%2Boilseed%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1253435%26rft_id%253Dinfo%253Apmid%252F25146293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDUvNjE5OS85NTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjI1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Syntenic gene analysis between Brassica rapa and other Brassicaceae species.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BPlant%2BSci%26rft.aulast%253DCheng%26rft.auinit1%253DF.%26rft.volume%253D3%26rft.spage%253D198%26rft.epage%253D198%26rft.atitle%253DSyntenic%2Bgene%2Banalysis%2Bbetween%2BBrassica%2Brapa%2Band%2Bother%2BBrassicaceae%2Bspecies.%26rft_id%253Dinfo%253Apmid%252F22969786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22969786&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.7" data-doi="10.1105/tpc.113.110486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandáková</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lysak</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Deciphering the diploid ancestral genome of the Mesohexaploid Brassica rapa.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.110486%26rft_id%253Dinfo%253Apmid%252F23653472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI1LzUvMTU0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.8" data-doi="10.1111/j.1469-8137.2007.02210.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dechaine</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinig</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Constraints on the evolution of adaptive plasticity: costs of plasticity to density are expressed in segregating progenies.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">874</span>–<span class="cit-lpage">882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DDechaine%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D176%26rft.issue%253D4%26rft.spage%253D874%26rft.epage%253D882%26rft.atitle%253DConstraints%2Bon%2Bthe%2Bevolution%2Bof%2Badaptive%2Bplasticity%253A%2Bcosts%2Bof%2Bplasticity%2Bto%2Bdensity%2Bare%2Bexpressed%2Bin%2Bsegregating%2Bprogenies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2007.02210.x%26rft_id%253Dinfo%253Apmid%252F17822398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2007.02210.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17822398&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250819700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.9" data-doi="10.1186/1471-2229-14-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dechaine</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinig</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL architecture of reproductive fitness characters in Brassica rapa.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-14-1%26rft_id%253Dinfo%253Apmid%252F24387633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-14-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24387633&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devisetty</span>  <span class="cit-name-given-names">U.K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covington</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tat</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lekkala</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maloof</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Polymorphism identification and improved genome annotation of Brassica rapa through deep RNA sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2065</span>–<span class="cit-lpage">2078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D2065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-7.7.2259.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source"><em>Vegetable Brassicas and Related Crucifers</em>.</span> <span class="cit-publ-name">Centre for Agriculture and Biosciences International</span>, <span class="cit-publ-loc">Wallingford, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.12" data-doi="10.1534/genetics.110.125112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewers</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genetic architecture of ecophysiological and circadian traits in Brassica rapa.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.125112%26rft_id%253Dinfo%253Apmid%252F21750258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIyNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.13" data-doi="10.1534/genetics.112.138578"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawe</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Megabase-scale inversion polymorphism in the wild ancestor of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">883</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.138578%26rft_id%253Dinfo%253Apmid%252F22542971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvODgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIyNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fulop</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranjan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ofner</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covington</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A new advanced backcross tomato population enables high resolution leaf QTL mapping and gene identification.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3169</span>–<span class="cit-lpage">3184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D3169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.15" data-doi="10.1104/pp.111.175612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hammond</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayes</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Regulatory hotspots are associated with plant gene expression under varying soil phosphorus supply in Brassica rapa.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1230</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.175612%26rft_id%253Dinfo%253Apmid%252F21527424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU2LzMvMTIzMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harushima</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shomura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A high-density rice genetic linkage map with 2275 markers using a single F2 population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHarushima%26rft.auinit1%253DY.%26rft.volume%253D148%26rft.issue%253D1%26rft.spage%253D479%26rft.epage%253D494%26rft.atitle%253DA%2BHigh-Density%2BRice%2BGenetic%2BLinkage%2BMap%2Bwith%2B2275%2BMarkers%2BUsing%2Ba%2BSingle%2BF2%2BPopulation%26rft_id%253Dinfo%253Apmid%252F9475757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ4LzEvNDc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIyNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.17" data-doi="10.1046/j.1365-313x.2001.01087.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haupt</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winderl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torres-Ruiz</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The CENTROMERE1 (CEN1) region of Arabidopsis thaliana: architecture and functional impact of chromatin.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DHaupt%26rft.auinit1%253DW.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D285%26rft.epage%253D296%26rft.atitle%253DThe%2Bcentromere1%2B%2528CEN1%2529%2Bregion%2Bof%2BArabidopsis%2Bthaliana%253A%2Barchitecture%2Band%2Bfunctional%2Bimpact%2Bof%2Bchromatin.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313x.2001.01087.x%26rft_id%253Dinfo%253Apmid%252F11532174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313x.2001.01087.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11532174&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170799100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">I. A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of Brassica A and C genome-based ordered pan-transcriptomes for use in rapeseed genomic research.</span> <abbr class="cit-jnl-abbrev">Data Brief</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DData%2BBrief%26rft.volume%253D4%26rft.spage%253D357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.19" data-doi="10.1007/s00122-009-1157-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iniguez-Luy</span>  <span class="cit-name-given-names">F. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukens</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farnham</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amasino</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development of public immortal mapping populations, molecular markers and linkage maps for rapid cycling Brassica rapa and B. oleracea.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DIniguez-Luy%26rft.auinit1%253DF.%2BL.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D43%26rft.atitle%253DDevelopment%2Bof%2Bpublic%2Bimmortal%2Bmapping%2Bpopulations%252C%2Bmolecular%2Bmarkers%2Band%2Blinkage%2Bmaps%2Bfor%2Brapid%2Bcycling%2BBrassica%2Brapa%2Band%2BB.%2Boleracea.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1157-4%26rft_id%253Dinfo%253Apmid%252F19784615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1157-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19784615&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cloutier</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP discovery through next-generation sequencing and its applications.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">831460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPlant%2BGenomics%26rft.volume%253D2012%26rft.spage%253D831460%26rft_id%253Dinfo%253Apmid%252F23227038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23227038&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.21" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.22" data-doi="10.1007/s001220100570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quiros</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Sequence-related amplified polymorphism (SRAP), a new marker system based on a simple PCR reaction: its application to mapping and gene tagging in Brassica.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D103%26rft.spage%253D455%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220100570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220100570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170534600033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.23" data-doi="10.1038/ncomms4930"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Brassica oleracea genome reveals the asymmetrical evolution of polyploid genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3930%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4930%26rft_id%253Dinfo%253Apmid%252F24852848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4930&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24852848&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.24" data-doi="10.1007/s00122-011-1592-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic architecture of the circadian clock and flowering time in Brassica rapa.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLou%26rft.auinit1%253DP.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D397%26rft.epage%253D409%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bthe%2Bcircadian%2Bclock%2Band%2Bflowering%2Btime%2Bin%2BBrassica%2Brapa.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1592-x%26rft_id%253Dinfo%253Apmid%252F21505830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1592-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505830&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.25" data-doi="10.1105/tpc.112.099499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cressman</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Preferential retention of circadian clock genes during diploidization following whole genome triplication in Brassica rapa.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2415</span>–<span class="cit-lpage">2426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.112.099499%26rft_id%253Dinfo%253Apmid%252F22685167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI0LzYvMjQxNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.26" data-doi="10.1371/journal.pbio.1000500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A widespread chromosomal inversion polymorphism contributes to a major life-history transition, local adaptation, and reproductive isolation.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D8%26rft.spage%253De1000500%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000500%26rft_id%253Dinfo%253Apmid%252F20927411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20927411&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.27" data-doi="10.1186/gb-2014-15-6-r77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kagale</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcriptome and methylome profiling reveals relics of genome dominance in the mesopolyploid Brassica oleracea.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR77%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-6-r77%26rft_id%253Dinfo%253Apmid%252F24916971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-6-r77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24916971&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-web" id="cit-7.7.2259.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <em>R: A Language and Environment for Statistical Computing</em>. <a href="http://www.R-project.org">http://www.R-project.org</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Two-dimensional spreads of synaptonemal complexes from solanaceous plants. VI. High-resolution recombination nodule map for tomato (Lycopersicon esculentum).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">683</span>–<span class="cit-lpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSherman%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D683%26rft.epage%253D708%26rft.atitle%253DTwo-dimensional%2Bspreads%2Bof%2Bsynaptonemal%2Bcomplexes%2Bfrom%2Bsolanaceous%2Bplants.%2BVI.%2BHigh-resolution%2Brecombination%2Bnodule%2Bmap%2Bfor%2Btomato%2B%2528Lycopersicon%2Besculentum%2529.%26rft_id%253Dinfo%253Apmid%252F8647403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQxLzIvNjgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIyNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.7.2259.30" data-doi="10.1016/j.gene.2011.07.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Glucosinolate biosynthetic genes in Brassica rapa.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">487</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D487%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2011.07.021%26rft_id%253Dinfo%253Apmid%252F21835231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2011.07.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21835231&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296407800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.7.2259.31" data-doi="10.1038/ng.919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">The genome of the mesopolyploid crop species Brassica rapa.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D43%26rft.issue%253D10%26rft.spage%253D1035%26rft.epage%253D1039%26rft.atitle%253DThe%2Bgenome%2Bof%2Bthe%2Bmesopolyploid%2Bcrop%2Bspecies%2BBrassica%2Brapa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.919%26rft_id%253Dinfo%253Apmid%252F21873998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.919&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21873998&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2259.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genome sequence of allopolyploid Brassica juncea and analysis of differential homoeolog gene expression influencing selection.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1225</span>–<span class="cit-lpage">1232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D1225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/7/2259.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2249" title="Definition of a RACK1 Interaction Network in Drosophila melanogaster Using SWATH-MS" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2271" title="A Systemic Analysis of Transcriptomic and Epigenomic Data To Reveal Regulation Patterns for Complex Disease" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/7"><span>Volume 7 Issue 7, July 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/7.cover-source.jpg?itok=eAMfPilr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2259" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IR3R5pU8BjB9-eqe8pXKlTf1KhVSUYk_oa1n9LoBebA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/7/2259" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/7/2259" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/7/2259" class="active" data-icon-position="" data-hide-link-title="0">Using RNA-Seq for Genomic Scaffold Placement, Correcting Assemblies, and Genetic Map Creation in a Common Brassica rapa Mapping Population</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485324" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-uGoEXTN2qLkofzliCYfbagRMqmPbXiAHbagwVMRLDXg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2259.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/7/2259" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1q--G40ATyTj9qpNIIKrmT_P8LYqC36UcUJvoefOoMA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/7/2259" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2259.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485324" id="node485324--2" data-pisa="ggg;7/7/2259" data-pisa-master="ggg;g3.117.043000" data-apath="/ggg/7/7/2259.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using RNA-Seq for Genomic Scaffold Placement, Correcting Assemblies, and Genetic Map Creation in a Common <em>Brassica rapa</em> Mapping Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">R. J. Cody Markelz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael F. Covington</span>, <span class="highwire-citation-author" data-delta="2">Marcus T. Brock</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4686-7235" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Upendra K. Devisetty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5759-3175" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel J. Kliebenstein</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2833-9316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cynthia Weinig</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Julin N. Maloof</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2259-2270;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043000 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.043000" data-atom-uri="/ggg/7/7/2259.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485324--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">R. J. Cody Markelz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BJ.%2BCody%2BMarkelz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Markelz%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BJ.%2BCody%2BMarkelz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael F. Covington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BCovington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covington%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BF.%2BCovington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael F. Covington</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marcus T. Brock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wyoming, Laramie, Wyoming 82072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcus%2BT.%2BBrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brock%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcus%2BT.%2BBrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Upendra K. Devisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Upendra%2BK.%2BDevisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devisetty%20UK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUpendra%2BK.%2BDevisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4686-7235" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Upendra K. Devisetty</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel J. Kliebenstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Sciences, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJ.%2BKliebenstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kliebenstein%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BJ.%2BKliebenstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5759-3175" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel J. Kliebenstein</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Cynthia Weinig </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wyoming, Laramie, Wyoming 82072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cynthia%2BWeinig%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinig%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACynthia%2BWeinig%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2833-9316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cynthia Weinig</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Julin N. Maloof </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julin%2BN.%2BMaloof%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maloof%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulin%2BN.%2BMaloof%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Julin N. Maloof</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/7/2259" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/7/2259&amp;title=Using%20RNA-Seq%20for%20Genomic%20Scaffold%20Placement%2C%20Correcting%20Assemblies%2C%20and%20Genetic%20Map%20Creation%20in%20a%20Common%20Brassica%20rapa%20Mapping%20Population" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/7/2259&amp;title=Using%20RNA-Seq%20for%20Genomic%20Scaffold%20Placement%2C%20Correcting%20Assemblies%2C%20and%20Genetic%20Map%20Creation%20in%20a%20Common%20Brassica%20rapa%20Mapping%20Population" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/7/2259&amp;title=Using%20RNA-Seq%20for%20Genomic%20Scaffold%20Placement%2C%20Correcting%20Assemblies%2C%20and%20Genetic%20Map%20Creation%20in%20a%20Common%20Brassica%20rapa%20Mapping%20Population" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/7/2259&amp;text=Using%20RNA-Seq%20for%20Genomic%20Scaffold%20Placement%2C%20Correcting%20Assemblies%2C%20and%20Genetic%20Map%20Creation%20in%20a%20Common%20Brassica%20rapa%20Mapping%20Population" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/7/2259&amp;title=Using%20RNA-Seq%20for%20Genomic%20Scaffold%20Placement%2C%20Correcting%20Assemblies%2C%20and%20Genetic%20Map%20Creation%20in%20a%20Common%20Brassica%20rapa%20Mapping%20Population" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/7/2259&amp;t=Using%20RNA-Seq%20for%20Genomic%20Scaffold%20Placement%2C%20Correcting%20Assemblies%2C%20and%20Genetic%20Map%20Creation%20in%20a%20Common%20Brassica%20rapa%20Mapping%20Population" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/7/2259&amp;title=Using%20RNA-Seq%20for%20Genomic%20Scaffold%20Placement%2C%20Correcting%20Assemblies%2C%20and%20Genetic%20Map%20Creation%20in%20a%20Common%20Brassica%20rapa%20Mapping%20Population" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/7/2259&amp;title=Using%20RNA-Seq%20for%20Genomic%20Scaffold%20Placement%2C%20Correcting%20Assemblies%2C%20and%20Genetic%20Map%20Creation%20in%20a%20Common%20Brassica%20rapa%20Mapping%20Population" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485324/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485324" id="node485324--3" data-pisa="ggg;7/7/2259" data-pisa-master="ggg;g3.117.043000" data-apath="/ggg/7/7/2259.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using RNA-Seq for Genomic Scaffold Placement, Correcting Assemblies, and Genetic Map Creation in a Common <em>Brassica rapa</em> Mapping Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">R. J. Cody Markelz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael F. Covington</span>, <span class="highwire-citation-author" data-delta="2">Marcus T. Brock</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4686-7235" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Upendra K. Devisetty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5759-3175" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel J. Kliebenstein</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2833-9316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cynthia Weinig</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Julin N. Maloof</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2259-2270;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043000 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.043000" data-atom-uri="/ggg/7/7/2259.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485324--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">R. J. Cody Markelz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BJ.%2BCody%2BMarkelz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Markelz%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BJ.%2BCody%2BMarkelz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael F. Covington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BCovington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covington%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BF.%2BCovington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael F. Covington</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marcus T. Brock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wyoming, Laramie, Wyoming 82072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcus%2BT.%2BBrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brock%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcus%2BT.%2BBrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Upendra K. Devisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Upendra%2BK.%2BDevisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devisetty%20UK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUpendra%2BK.%2BDevisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4686-7235" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Upendra K. Devisetty</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel J. Kliebenstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Sciences, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJ.%2BKliebenstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kliebenstein%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BJ.%2BKliebenstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5759-3175" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel J. Kliebenstein</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Cynthia Weinig </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wyoming, Laramie, Wyoming 82072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cynthia%2BWeinig%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinig%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACynthia%2BWeinig%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2833-9316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cynthia Weinig</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Julin N. Maloof </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julin%2BN.%2BMaloof%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maloof%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulin%2BN.%2BMaloof%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Julin N. Maloof</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2259&amp;rft.epage=2270&amp;rft.atitle=Using%20RNA-Seq%20for%20Genomic%20Scaffold%20Placement%2C%20Correcting%20Assemblies%2C%20and%20Genetic%20Map%20Creation%20in%20a%20Common%20Brassica%20rapa%20Mapping%20Population&amp;rft.volume=7&amp;rft.issue=7&amp;rft.date=2017-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Markelz%2C+R.+J.+Cody&amp;rft.au=Covington%2C+Michael+F.&amp;rft.au=Brock%2C+Marcus+T.&amp;rft.au=Devisetty%2C+Upendra+K.&amp;rft.au=Kliebenstein%2C+Daniel+J.&amp;rft.au=Weinig%2C+Cynthia&amp;rft.au=Maloof%2C+Julin+N."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485324/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485324/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485324/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485324/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485324/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485324/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485324/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485324/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485324/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485324/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485324/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485324/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/7/2259" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/7/2259.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/7/2259.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/7/2259.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/7/2259.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
