<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/7/2161" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/7/2161.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/7/2161.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/7/2161.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/7/2161/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/7/2161.atom" />
<meta name="HW.pisa" content="ggg;7/7/2161" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize" />
<meta name="DC.Identifier" content="10.1534/g3.117.042036" />
<meta name="DC.Date" content="2017-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Manching et al." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="High-throughput sequencing (HTS) of reduced representation genomic libraries has ushered in an era of genotyping-by-sequencing (GBS), where genome-wide genotype data can be obtained for nearly any species. However, there remains a need for imputation-free GBS methods for genotyping large samples taken from heterogeneous populations of heterozygous individuals. This requires that a number of issues encountered with GBS be considered, including the sequencing of nonoverlapping sets of loci across multiple GBS libraries, a common missing data problem that results in low call rates for markers per individual, and a tendency for applicability only in inbred line samples with sufficient linkage disequilibrium for accurate imputation. We addressed these issues while developing and validating a new, comprehensive platform for GBS. This study supports the notion that GBS can be tailored to particular aims, and using Zea mays our results indicate that large samples of unknown pedigree can be genotyped to obtain complete and accurate GBS data. Optimizing size selection to sequence a high proportion of shared loci among individuals in different libraries and using simple in silico filters, a GBS procedure was established that produces high call rates per marker (&gt;85%) with accuracy exceeding 99.4%. Furthermore, by capitalizing on the sequence-read structure of GBS data (stacks of reads), a new tool for resolving local haplotypes and scoring phased genotypes was developed, a feature that is not available in many GBS pipelines. Using local haplotypes reduces the marker dimensionality of the genotype matrix while increasing the informativeness of the data. Phased GBS in maize also revealed the existence of reproducibly inaccurate (apparent accuracy) genotypes that were due to divergent copy number variants (CNVs) unobservable in the underlying single nucleotide polymorphism (SNP) data." />
<meta name="DC.Contributor" content="Heather Manching" />
<meta name="DC.Contributor" content="Subhajit Sengupta" />
<meta name="DC.Contributor" content="Keith R. Hopper" />
<meta name="DC.Contributor" content="Shawn W. Polson" />
<meta name="DC.Contributor" content="Yuan Ji" />
<meta name="DC.Contributor" content="Randall J. Wisser" />
<meta name="article:published_time" content="2017-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-throughput sequencing (HTS) of reduced representation genomic libraries has ushered in an era of genotyping-by-sequencing (GBS), where genome-wide genotype data can be obtained for nearly any species. However, there remains a need for imputation-free GBS methods for genotyping large samples taken from heterogeneous populations of heterozygous individuals. This requires that a number of issues encountered with GBS be considered, including the sequencing of nonoverlapping sets of loci across multiple GBS libraries, a common missing data problem that results in low call rates for markers per individual, and a tendency for applicability only in inbred line samples with sufficient linkage disequilibrium for accurate imputation. We addressed these issues while developing and validating a new, comprehensive platform for GBS. This study supports the notion that GBS can be tailored to particular aims, and using &lt;i&gt;Zea mays&lt;/i&gt; our results indicate that large samples of unknown pedigree can be genotyped to obtain complete and accurate GBS data. Optimizing size selection to sequence a high proportion of shared loci among individuals in different libraries and using simple &lt;i&gt;in silico&lt;/i&gt; filters, a GBS procedure was established that produces high call rates per marker (&amp;gt;85%) with accuracy exceeding 99.4%. Furthermore, by capitalizing on the sequence-read structure of GBS data (stacks of reads), a new tool for resolving local haplotypes and scoring phased genotypes was developed, a feature that is not available in many GBS pipelines. Using local haplotypes reduces the marker dimensionality of the genotype matrix while increasing the informativeness of the data. Phased GBS in maize also revealed the existence of reproducibly inaccurate (apparent accuracy) genotypes that were due to divergent copy number variants (CNVs) unobservable in the underlying single nucleotide polymorphism (SNP) data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/07/01" />
<meta name="citation_mjid" content="ggg;7/7/2161" />
<meta name="citation_id" content="7/7/2161" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/7/2161" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/7/2161.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/7/2161.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/7/2161.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.042036" />
<meta name="citation_pmid" content="28526729" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2161" />
<meta name="citation_lastpage" content="2170" />
<meta name="citation_author" content="Heather Manching" />
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware 19716" />
<meta name="citation_author" content="Subhajit Sengupta" />
<meta name="citation_author_institution" content="Program of Computational Genomics and Medicine, North Shore University Health System, Evanston, Illinois 60201" />
<meta name="citation_author" content="Keith R. Hopper" />
<meta name="citation_author_institution" content="Beneficial Insect Introductions Research Unit, United States Department of Agriculture, Agricultural Research Service, Newark, Delaware 19713" />
<meta name="citation_author" content="Shawn W. Polson" />
<meta name="citation_author_institution" content="Center for Bioinformatics and Computational Biology, Delaware Biotechnology Institute, University of Delaware, Newark, Delaware 19711" />
<meta name="citation_author" content="Yuan Ji" />
<meta name="citation_author_institution" content="Program of Computational Genomics and Medicine, North Shore University Health System, Evanston, Illinois 60201" />
<meta name="citation_author_institution" content="Department of Health Studies, University of Chicago, Illinois 60637" />
<meta name="citation_author" content="Randall J. Wisser" />
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware 19716" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. A. Ali;citation_author=S. M. O’Rourke;citation_author=S. J. Amish;citation_author=M. H. Meek;citation_author=G. Luikart;citation_title=RAD capture (rapture): flexible and efficient sequence-based genotyping.;citation_volume=202;citation_year=2016;citation_pmid=26715661;citation_doi=10.1534/genetics.115.183665" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Andolfatto;citation_author=D. Davison;citation_author=D. Erezyilmaz;citation_author=T. T. Hu;citation_author=J. Mast;citation_title=Multiplexed shotgun genotyping for rapid and efficient genetic mapping.;citation_volume=21;citation_year=2011;citation_pmid=21233398;citation_doi=10.1101/gr.115402.110" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=K. R. Andrews;citation_author=J. M. Good;citation_author=M. R. Miller;citation_author=G. Luikart;citation_author=P. A. Hohenlohe;citation_title=Harnessing the power of RADseq for ecological and evolutionary genomics.;citation_pages=81-92;citation_volume=17;citation_year=2016;citation_pmid=26729255;citation_doi=10.1038/nrg.2015.28" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_author=F. Antequera;citation_author=A. P. Bird;citation_title=Unmethylated CpG islands associated with genes in higher plant DNA.;citation_pages=2295-2299;citation_volume=7;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. Barba;citation_author=L. Cadle-Davidson;citation_author=J. Harriman;citation_author=J. C. Glaubitz;citation_author=S. Brooks;citation_title=Grapevine powdery mildew resistance and susceptibility loci identified on a high-resolution SNP map.;citation_pages=73-84;citation_volume=127;citation_year=2014;citation_pmid=24072208;citation_doi=10.1007/s00122-013-2202-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Beissinger;citation_author=C. N. Hirsch;citation_author=R. S. Sekhon;citation_author=J. M. Foerster;citation_author=J. M. Johnson;citation_title=Marker density and read depth for genotyping populations using genotyping-by-sequencing.;citation_volume=193;citation_year=2013;citation_pmid=23410831;citation_doi=10.1534/genetics.112.147710" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. M. Bowcock;citation_author=A. Ruiz-Linares;citation_author=J. Tomfohrde;citation_author=E. Minch;citation_author=J. R. Kidd;citation_title=High resolution of human evolutionary trees with polymorphic microsatellites.;citation_pages=455-457;citation_volume=368;citation_year=1994;citation_issue=6470;citation_pmid=7510853;citation_doi=10.1038/368455a0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Chia;citation_author=C. Song;citation_author=P. J. Bradbury;citation_author=D. Costich;citation_author=N. de Leon;citation_title=Maize HapMap2 identifies extant variation from a genome in flux.;citation_pages=803-807;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660545;citation_doi=10.1038/ng.2313" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. M. DaCosta;citation_author=M. D. Sorenson;citation_title=Amplification biases and consistent recovery of loci in a double-digest RAD-seq protocol.;citation_pages=e106713;citation_volume=9;citation_year=2014;citation_pmid=25188270;citation_doi=10.1371/journal.pone.0106713" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=R. W. Davies;citation_author=J. Flint;citation_author=S. Myers;citation_author=R. Mott;citation_title=Rapid genotype imputation from sequence without reference panels.;citation_pages=965-969;citation_volume=48;citation_year=2016;citation_doi=10.1038/ng.3594" />
<meta name="citation_reference" content="citation_journal_title=Journal of Statistical Software.;citation_author=S. Dray;citation_author=A. B. Dufour;citation_title=The ade4 package: implementing the duality diagram for ecologists.;citation_pages=1-20;citation_volume=22;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. H. Ferdosi;citation_author=J. Henshall;citation_author=B. Tier;citation_title=Study of the optimum haplotype length to build genomic relationship matrices.;citation_pages=75;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=Y.-B. Fu;citation_author=G. W. Peterson;citation_title=Genetic diversity analysis with 454 pyrosequencing and genomic reduction confirmed the eastern and western division in the cultivated barley gene pool.;citation_pages=226-237;citation_volume=4;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. W. Ganal;citation_author=G. Durstewitz;citation_author=A. Polley;citation_author=A. Bérard;citation_author=E. S. Buckler;citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.;citation_pages=e28334-e28334;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174790;citation_doi=10.1371/journal.pone.0028334" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=K. M. Gardner;citation_author=P. J. Brown;citation_author=T. F. Cooke;citation_author=S. Cann;citation_author=F. Costa;citation_title=Fast and cost-effective genetic mapping in apple using next-generation sequencing.;citation_pages=1681-1687;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=O. Gascuel;citation_title=BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data;citation_pages=685-695;citation_volume=14;citation_year=1997;citation_issue=7;citation_pmid=9254330;citation_doi=10.1093/oxfordjournals.molbev.a025808" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. M. Gatti;citation_author=K. L. Svenson;citation_author=A. Shabalin;citation_author=L.-Y. Wu;citation_author=W. Valdar;citation_title=Quantitative trait locus mapping methods for diversity outbred mice.;citation_pages=1623-1633;citation_volume=4;citation_year=2014;citation_doi=10.1534/g3.114.013748" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_author=S. Glémin;citation_author=P. F. Arndt;citation_author=P. W. Messer;citation_author=D. Petrov;citation_author=N. Galtier;citation_title=Quantification of GC-biased gene conversion in the human genome.;citation_volume=25;citation_year=2015;citation_pmid=25995268;citation_doi=10.1101/gr.185488.114" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. He;citation_author=Y. Zhao;citation_author=M. F. Mette;citation_author=R. Bothe;citation_author=E. Ebmeyer;citation_title=Prospects and limits of marker imputation in quantitative genetic studies in european elite wheat (triticum aestivum l.).;citation_pages=168;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Heffelfinger;citation_author=C. A. Fragoso;citation_author=M. A. Moreno;citation_author=J. D. Overton;citation_author=J. P. Mottinger;citation_title=Flexible and scalable genotyping-by-sequencing strategies for population studies.;citation_pages=979;citation_volume=15;citation_year=2014;citation_pmid=25406744;citation_doi=10.1186/1471-2164-15-979" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. E. Hyma;citation_author=P. Barba;citation_author=M. Wang;citation_author=J. P. Londo;citation_author=C. B. Acharya;citation_title=Heterozygous mapping strategy (HetMappS) for high resolution genotyping-by-sequencing markers: a case study in grapevine.;citation_pages=e0134880;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Jombart;citation_title=adegenet: a R package for the multivariate analysis of genetic markers.;citation_volume=24;citation_year=2008;citation_pmid=18397895;citation_doi=10.1093/bioinformatics/btn129" />
<meta name="citation_reference" content="citation_journal_title=Psychometrika;citation_author=J. B. Kruskal;citation_title=Multidimensional scaling by optimizing goodness of fit to a nonmetric hypothesis.;citation_pages=1-27;citation_volume=29;citation_year=1964;citation_doi=10.1007/BF02289565" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. Liu;citation_author=M. Goodman;citation_author=S. Muse;citation_author=J. S. Smith;citation_author=E. Buckler;citation_title=Genetic Structure and Diversity Among Maize Inbred Lines as Inferred From DNA Microsatellites;citation_pages=2117-2128;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704191" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. J. Lorenz;citation_author=M. T. Hamblin;citation_author=J.-L. Jannink;citation_title=Performance of single nucleotide polymorphisms versus haplotypes for genome-wide association analysis in barley.;citation_pages=e14079-e14079;citation_volume=5;citation_year=2010;citation_issue=11;citation_pmid=21124933;citation_doi=10.1371/journal.pone.0014079" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Lu;citation_author=T. Shah;citation_author=Z. Hao;citation_author=S. Taba;citation_author=S. Zhang;citation_title=Comparative SNP and haplotype analysis reveals a higher genetic diversity and rapider LD decay in tropical than temperate germplasm in maize.;citation_pages=e24861-e24861;citation_volume=6;citation_year=2011;citation_issue=9;citation_pmid=21949770;citation_doi=10.1371/journal.pone.0024861" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. H. Lund;citation_author=M. Duch;citation_author=F. S. Pedersen;citation_title=Increased cloning efficiency by temperature-cycle ligation.;citation_volume=24;citation_year=1996;citation_pmid=8604328;citation_doi=10.1093/nar/24.4.800" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=N. Mantel;citation_title=The Detection of Disease Clustering and a Generalized Regression Approach;citation_pages=209-220;citation_volume=27;citation_year=1967;citation_issue=2_Part_1;citation_pmid=6018555" />
<meta name="citation_reference" content="citation_journal_title=EMBnet.journal.;citation_author=M. Martin;citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads.;citation_pages=10-12;citation_volume=17;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. McMullen;citation_author=S. Kresovich;citation_author=H. S. Villeda;citation_author=P. Bradbury;citation_author=H. Li;citation_title=Genetic Properties of the Maize Nested Association Mapping Population;citation_pages=737-740;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661427;citation_doi=10.1126/science.1174320" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Neph;citation_author=M. S. Kuehn;citation_author=A. P. Reynolds;citation_author=E. Haugen;citation_author=R. E. Thurman;citation_title=BEDOPS: high-performance genomic feature operations.;citation_volume=28;citation_year=2012;citation_pmid=22576172;citation_doi=10.1093/bioinformatics/bts277" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=E. Paradis;citation_author=J. Claude;citation_author=K. Strimmer;citation_title=APE: Analyses of Phylogenetics and Evolution in R language;citation_pages=289-290;citation_volume=20;citation_year=2004;citation_issue=2;citation_pmid=14734327;citation_doi=10.1093/bioinformatics/btg412" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Math. Biosci.;citation_author=D. F. Robinson;citation_author=L. R. Foulds;citation_title=Comparison of phylogenetic trees.;citation_pages=131-147;citation_volume=53;citation_year=1981;citation_doi=10.1016/0025-5564(81)90043-2" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Rocher;citation_author=M. Jean;citation_author=Y. Castonguay;citation_author=F. Belzile;citation_title=Validation of genotyping-by-sequencing analysis in populations of tetraploid alfalfa by 454 sequencing.;citation_pages=1-18;citation_volume=10;citation_year=2015;citation_pmid=25919488;citation_doi=10.1371/journal.pone.0131080" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Schliep;citation_title=phangorn: phylogenetic analysis in r.;citation_volume=27;citation_year=2011;citation_pmid=21169378;citation_doi=10.1093/bioinformatics/btq706" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Sengupta;citation_author=K. Gulukota;citation_author=Y. Zhu;citation_author=C. Ober;citation_author=K. Naughton;citation_title=Ultra-fast local-haplotype variant calling using paired-end DNA-sequencing data reveals somatic mosaicism in tumor and normal blood samples.;citation_pages=e25;citation_volume=44;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Sonah;citation_author=M. Bastien;citation_author=E. Iquira;citation_author=A. Tardivel;citation_author=G. Légaré;citation_title=An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.;citation_pages=e54603;citation_volume=8;citation_year=2013;citation_pmid=23372741;citation_doi=10.1371/journal.pone.0054603" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. I. Tenaillon;citation_author=M. C. Sawkins;citation_author=A. D. Long;citation_author=R. L. Gaut;citation_author=J. F. Doebley;citation_title=Patterns of DNA sequence polymorphism along chromosome 1 of maize (Zea mays ssp. mays L.).;citation_volume=98;citation_year=2001;citation_pmid=11470895;citation_doi=10.1073/pnas.151244298" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. A. Tinker;citation_author=W. A. Bekele;citation_author=J. Hattori;citation_title=Haplotag: software for haplotype-based genotyping-by-sequencing analysis.;citation_pages=857-863;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. Torkamaneh;citation_author=J. Laroche;citation_author=F. Belzile;citation_title=Genome-wide SNP calling from genotyping by sequencing (GBS) data: a comparison of seven pipelines and two sequencing technologies.;citation_pages=e0161333;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. G. A. M. L. Uitdewilligen;citation_author=A. M. A. Wolters;citation_author=B. B. D’hoop;citation_author=T. J. A. Borm;citation_author=R. G. F. Visser;citation_title=A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.;citation_pages=e62355;citation_volume=8;citation_year=2013;citation_pmid=23667470;citation_doi=10.1371/journal.pone.0062355" />
<meta name="twitter:title" content="Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="twitter:description" content="High-throughput sequencing (HTS) of reduced representation genomic libraries has ushered in an era of genotyping-by-sequencing (GBS), where genome-wide genotype data can be obtained for nearly any species. However, there remains a need for imputation-free GBS methods for genotyping large samples taken from heterogeneous populations of heterozygous individuals. This requires that a number of issues encountered with GBS be considered, including the sequencing of nonoverlapping sets of loci across multiple GBS libraries, a common missing data problem that results in low call rates for markers per individual, and a tendency for applicability only in inbred line samples with sufficient linkage disequilibrium for accurate imputation. We addressed these issues while developing and validating a new, comprehensive platform for GBS. This study supports the notion that GBS can be tailored to particular aims, and using Zea mays our results indicate that large samples of unknown pedigree can be genotyped to obtain complete and accurate GBS data. Optimizing size selection to sequence a high proportion of shared loci among individuals in different libraries and using simple in silico filters, a GBS procedure was established that produces high call rates per marker (&gt;85%) with accuracy exceeding 99.4%. Furthermore, by capitalizing on the sequence-read structure of GBS data (stacks of reads), a new tool for resolving local haplotypes and scoring phased genotypes was developed, a feature that is not available in many GBS pipelines. Using local haplotypes reduces the marker dimensionality of the genotype matrix while increasing the informativeness of the data. Phased GBS in maize also revealed the existence of reproducibly inaccurate (apparent accuracy) genotypes that were due to divergent copy number variants (CNVs) unobservable in the underlying single nucleotide polymorphism (SNP) data." />
<meta name="og-title" property="og:title" content="Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/7/2161" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-throughput sequencing (HTS) of reduced representation genomic libraries has ushered in an era of genotyping-by-sequencing (GBS), where genome-wide genotype data can be obtained for nearly any species. However, there remains a need for imputation-free GBS methods for genotyping large samples taken from heterogeneous populations of heterozygous individuals. This requires that a number of issues encountered with GBS be considered, including the sequencing of nonoverlapping sets of loci across multiple GBS libraries, a common missing data problem that results in low call rates for markers per individual, and a tendency for applicability only in inbred line samples with sufficient linkage disequilibrium for accurate imputation. We addressed these issues while developing and validating a new, comprehensive platform for GBS. This study supports the notion that GBS can be tailored to particular aims, and using Zea mays our results indicate that large samples of unknown pedigree can be genotyped to obtain complete and accurate GBS data. Optimizing size selection to sequence a high proportion of shared loci among individuals in different libraries and using simple in silico filters, a GBS procedure was established that produces high call rates per marker (&gt;85%) with accuracy exceeding 99.4%. Furthermore, by capitalizing on the sequence-read structure of GBS data (stacks of reads), a new tool for resolving local haplotypes and scoring phased genotypes was developed, a feature that is not available in many GBS pipelines. Using local haplotypes reduces the marker dimensionality of the genotype matrix while increasing the informativeness of the data. Phased GBS in maize also revealed the existence of reproducibly inaccurate (apparent accuracy) genotypes that were due to divergent copy number variants (CNVs) unobservable in the underlying single nucleotide polymorphism (SNP) data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
    <title>Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"JZaEGTyEZdH9yMhEfTxU0CKznJfhu6yw9vAmThFsDZA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485312","apath":"\/ggg\/7\/7\/2161.atom","pisa":"ggg;7\/7\/2161","ac":{"\/ggg\/7\/7\/2161.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/7\/2161.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2161"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2161"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/7\/2161":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oQgX4anc9hVJ8AsJ2TnVo40MEQXy8MEAXdNEPNYDvIcDv34PHE1dRSN3DUioTd1Znf82sJ+nQyKEAv9JmrByFuLxdBJFtxfb9ONXtUn+YbQCd6zOUR8QFpzrmcxpemnS0XXMLZtA7sCWbLaUTSXGuz8MFx5klI1zGYiOgqgcsLTY381zsfa4NMUaQc1Yi2Sr\\\\\\\/ruVoW3PpttYw4yLqzMMc4oWN19Hltm8JrI+Rc840yfjI2fSB6cJMmWNVOFsQAT9SNR\\\\\\\/9AdoB\\\\\\\/5ACMy6plxDZoWN0B+WypGuJjxPF3PQevLy617lGFCxHgiYfoNXeRL58Sogcn7aZ4pTap0ZFaVf2G07De3AStNlTw0puugPXvDF8tLbIZjOTMbxu2hNgRJItVmxWpWwcytVYdWiH9yMCIFrbfUxGwZIA30vv2JVLmJ5+8F1vuZ3RYYMH4w1Tf8UaoebIwSvYV\\\\\\\/4Zjejmupr8WV\\\\\\\/3p\\\\\\\/zclJx1f1d\\\\\\\/sBJT3z\\\\\\\/d7UI+ypEqJwYtuva2b5pTbbVRJ2djW0PzuCNsxJqabtb5T6IdJGkMeXBrsR7eA12do5yi0CNnTlUbp1K76bjp72QWr62mVVxlXRsb1ffhkCpICAAMWlM8guFACEgo4WcHQb4Ud+XGQ1UjBX43SNX5s4dtWODe1pO1yEd5r3XzGROc3vO4Jo0Kw58u1z+nPE9re8\\\\\\\/yxUNP67pfySILv0Ewg6rxNtkGvSpLTz7DqdFq412u3JGhKQfrSf7NKTJHqKQlb1TajdQ\\\\\\\/JBIe3CSJwQ0yLnsaPg+zqV5X3QdaXDUH8qbh1BS17rXugnKZbGdGEbSZLivvvelCTXZ698zXjwp16MTsyvQ2NxgUWZgt0slI8x8lUrxrShdAyDkqAb\\\\\\\/0izOJmKs1UZ\\\\\\\/J5bWEgvxKeN9jKWugsZ93wnR2yh03IaJIuXWtsE5TJmg9n4s5ecZODIrJKE9\\\\\\\/mfKtdgWAweETLCYVz68qRvGYzQ\\\\\\\/ZLCYShyv3Ih7OpAH+awcs63b3ayfABs9Aw2014UyPUfzw3gsXnPSy6wgGWkvk0SeVb+3XvLEGhx444zd5GZvEoKDwOQ\\\\\\\/0ARR1Ckmw8dC1xgIPDHEz7WT4cjBKoRa63olxPIY1UxL3Kps7n8b5EuNkuSPCidNHoprXdwua2BOsDVpsEzjqVgooTJYnAMdfYMsEwlxZr+OdCPqPGN4STSyPVNFqz45shBORivCgb5yIaWsEVbn0FWu0n1V4fKgznkVwYCXaVge+CWHf4HxLaV+fCGc7iYo0oZyJe\\\\\\\/7mdPI2mY616uVInoD\\\u0022,\\\u0022iv\\\u0022:\\\u00228Hf2shvR9htCkOmFrl6TGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c880c7b952cf7abd2e5737832cc0d2effa77b9fd36080832e92b21fad604b22b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cSR03yfMjCuae1jZVv6q+OGuOy6TPbrGgaZDUqyCGWLCVH+qS5S9kJLiLmsY+yCYhJNjiBsQs69SBgcF1E0rYvfPtTBW2ROZN8wPL0+2Uqrh3jHwvgdNNhCbsDqV4LC88bAsJHrnXNHTE5YO1us38Zxi7kIAUEyHxEgaQKqLEgkKOEg4r9m0rsPurdKVOV82l1RjMlPIM8vqpy\\\\\\\/jb1nSkNyiRiPmsZiZIyIy2Cr360+ID2da8DuIzsvy9tZlfe+2lZko5MJwoWOtAJ9aEZwcxm8a5CzjUloa9uQvAbwRhAdu\\\\\\\/9mPW1QisY+N6G2tOhqZL+LoLYJS+aGRovKK9ghEb8RD\\\\\\\/mWBcRuIeRyGn+JKMV8P9AWbPgH06hSm\\\\\\\/J9eqfmSLY3nbLIcunEej8SY6Xg4\\\\\\\/snhbtW7uRzsuvtaaqs3WEDFYmBnED5O2gtIuh9ZzU0ONkqLfzhkruL307V5eNmERB5SLb9SdMMvO6oH4dkEAZIQspID9FnchhzbBRmZo4nHKIdwxy1zmhMayp+boTCUkK+03Fzv6wEDv8R+tBT2HXxs007uMVHkMGGZcvsi7y+416bCTUbrvDPI\\\\\\\/0qMgxR44p\\\\\\\/eJogOd5+s1\\\\\\\/3oGg+rOYxagi0Uy3HCd09siJ\\\\\\\/I4aJKpGApUfVQZ6pV822osOXJpESh1Adpibmpb83n6hxd824czibsMh+Slhjl6TxiWAGVyTPy3\\\\\\\/JOs+riwPhxA4UXBDTYTqIEjJj8GviF0En+7xaC04\\\\\\\/2YXiQnfJa6rT8VP50X0t0\\\\\\\/CenC1pTqm6vVOkHTm8m32DFUqSljl4WSeNPI\\\\\\\/iQN+gHkXK8jYPt4ShFCLDLB\\\\\\\/uPppK9qZTfj7RqCRignIoCstNHaJTnY98suqMcE7kUJpdPgijuzj61MCCI737uMidOrjuP\\\\\\\/ZG6s4qykV8j+uKtUDRL06LNh06In\\\\\\\/g8Ajkz0G3esYJwfTmxOwQ+ONJT8xDQuDs3wGOAXbawMankRYnzirPoDgKiyxoag2ITeoHEYnafUFekD2348f1n8oAelIEmS5xjH2tsxBz4\\\\\\\/W9TK\\\\\\\/0j5uo+kX5JX1k3UNktovziICN2qYmlJO5fwFhLGdMYiACiYE3KM+x6diGrXINuQ7WE1NswClZluJjCP4TnrptUl9Jw9fuYctVFfvh8mz05+WgIxpblKhZV63YgDytxIjg6MTqbTQMb3UdeZM8cb5GLtypTZjI3wI8RHZixg1ssaqtJ+Hp38Dg4JJM0dKo71Lq13\\\\\\\/KAF64jydL7ekh7A3b1lTarVgGQ6jZXfbSqYNI64TxYfTpkZ9vI5nRlwfCuFVW1bQ4oLYzE5\\\\\\\/XEbID4L6LWnHaaLWRcewtP9D9EcudeirRVswvOK2Vz+wdH3ULz8zSf+87Ss8\\\\\\\/88\\\\\\\/kkeksfVTr1gU\\\\\\\/M1jbUfzUsheJirZoscR7QprxEF4p5jTMdQ3ttODJ+KCS1x+j9GazR2E7vbEgW7f5tzg+xW0MFwIA\\\\\\\/JgtmwmcI9lz7p1YV6XgA\\\\\\\/9wLyKrXQeDX8HzBKU3Smoo0xgHhsLdjQiBzQbqQMD0YTdBVUPBirAmA\\\\\\\/voFOaZUWbvGPPquNXH+Tc4emLzzsgjMIcvsDN+zV1yzzBQIfv1CYdMT867DcwAFvEPvnhMzxwZwBeEVdoSj3drJHT87HoGwcPCeZDQn8AXAYQ98D9zODWPpbdJfHXrRO7jRx9wxcBb2xVuMIc2Nhr\\\\\\\/DOPi7h9A1JFnpWId9p+mRzcQv31UT9APrKPGIkdEGXLdYibxRADZgQ0uRfQTFvLGszazMk\\\\\\\/PE2qUIn9l1uIcChODEcMnKSo2Pndk16bmjeoc9j9I4xztwep\\\\\\\/c5TasXX5QxZ1xsnjaGn42kpfOUcdFwWAuqaLnOO2sUscAHaYK5qwnW+d\\\\\\\/4lOvv5XsgqL5tJSIgf5cndQhiNGD3SRXmTOWM36ZkQUilHSqPm4cyIBfjSq60s1lSJX0sb8T1foOpIm56L71PMPfoIkLch5F8B\\\\\\\/O4Jpd0\\\\\\\/Zm61WNEcsbYhUDoFLEc7ldqtbCPwViXOAVOyuxAyKS4txYVEFi8c+fyJHw1uxrIaZsHVZHz6QqluFkmNvVQVKaspthCrxCaNkneMEWJOfbyn9wG3yDG6Yq1Pv1mnjMqWdjVlWWo\\\\\\\/XGXZ3lYH7OvrjKBHSc2MgY4dpE\\\\\\\/GIfSPUHW6YJ4rTyjxwqVPZX8Z+s2YSa33OAcB9dYeOTn4+oZ\\\\\\\/Qr4ELaeIl1vdfoMrYePSNodgByr3T1D9izpMPS5gU5BGMpl0b3c3wvGFDsO1IMX9vXUeedqmDa58BBUdxcpKd5\\\\\\\/+fLblMVHxO7Ybi4pru4weyv04eIw07CeeSEW5uo3bECT\\\\\\\/Abjvy7kf5sWxh8Y9be7jHWVgojjjVnxmtCKN0YHKkD9DYqAKCgQxQOw559Iy2BN0YrooJy0X\\\\\\\/BWb1CKQ0Ks+jz5h46uaf9XsA1G9nY9q8UW1kGEUhEYGRjalh8tEuGQkCZfMn1q6Her5MQArp+MSohK0SLXN5TiqLtTGHwDdTjo1nq1gGkHtVjvwFEBKMgbJnSG5wYD3gP\\\\\\\/059zhlDHv35PZSIApMS1JXb8BmxVIBHyw051FefRW90pk3hzPsx8GaamqOuSF6yAaxNyY8BDxA0Jo0+nWxuUXMCE6MJqu0WKLpnNO4ZjAmu0De1B873X6saFyeODZGJ+3f8i2a1N\\\\\\\/Jzctw\\\\\\\/KamKToOI6woiAHXbIMJ37Xtruj317KZPPGLQHQpgVix\\\\\\\/GPHN8NKDyECrmYNtE7yxbgv8JxKZm\\\\\\\/jBFu6sHnE\\\\\\\/LXCmTcQm+d3WlWlwhPqYVSoKOE6wUJ5wcdoTumpKxvvemmhqVLJFVpVQ1bRiQKYdU5maZGh\\\\\\\/WtCrUduTzmE31bwETbUKbC+lutPbTRtT\\\\\\\/9IKVRhwFWgQT1UQTcbG8BjgTXIg193xdVaRpm\\\\\\\/rkBbm\\\\\\\/trh0FOPjcRuK5ew1J2USoxsdTwN1OBnay+agH0BI3C8cCgCtmENk0bXU8qCN4DBniU6gTW5Bt2np+KStwC6EaOnr\\\\\\\/PTitme3elzs28+aQLYlyBJFNN97x2uEAbyXuIVTHQqA00Vb\\\\\\\/idPMqWnrVpwfHSXn4KwJ9i1l8XMqWHUfM9n4U7mBdfUA6Y7wy\\\\\\\/xI6YHW0w4czbKx1VfAwMpqdoGIR4saCqgNdoEPfsgpHxy1ugWo9ZQcs8mafGApoJOFBTQ8kiwWRYHH1kamL\\\\\\\/kHuIY+DQdBxd8Ni1bC2UiX\\\\\\\/MSPmTe1q3tDXZAFOUorD6Wb6giyHfLC\\\\\\\/sSTdRmuV\\\\\\\/\\\\\\\/6TooE+RSPgvcxIvV+ix6IHxCeGlCM\\\\\\\/j20HI7PHs\\\\\\\/fiftcw0o2n6UeIts4uo94BC\\\\\\\/VjSUtvtEpVjk25GeYzfeJ+d0ow8Txxggh0HQYOXFSsKnN5ADhwa4cTBHxlmn5ZM7Oe+\\\\\\\/H4FU5OE21eGbrWPAf6VP9S\\\\\\\/nsMFeD7H\\\\\\\/wJTk\\\\\\\/rxr+ZsNzMSavBJPyEn5ZjubT1JeoeJS3n\\\\\\\/Y0\\\\\\\/D7oKdZGNrl9gd51BkQEPFuy1QyhEufN7SNOT5Ouo0xAuiqcDnli4RjpdpJaE+wsDDf4NODZ3BL7eYN0WsiYmDLUKQi8UqFszQ2s3glmoe4UbM0j1Jtlq+QSNXPzSrnBwGE8GfR46fpbWLzfVN0kB6trZCCXEIYk4PrqoTFU5+3G3X4ZyT+8eSEZBsQwL9RdsBbNDEkiQs8gsjuRtR7PjvfDVy\\\\\\\/\\\\\\\/j5V5eqfYOA7oIuM00u8RVtEZLh3PXQ9eTb864nkq1lgHeSnfnERekZgMN5jVejmtYYMFAOPgxqsX0synWfGxSAx0HihFBZfOk2dM6GvofvnLerrOY2ZoZIsO143cKMCu17bNWbRb0uIZW6uhrSRIAblMF76V1TgNQW\\\\\\\/jy\\\\\\\/DAnXV4R15HHGWb32lkH39lQdXbndgXCUe7DmBGp3QIiUDRjCAo33wTGi\\\\\\\/KgzWxVwv4Rn0I7N1tTIOrFO2shRcx8jU0D4avvRh9mo7ZGH+g1Och4oIAgE2h1ABSYwULG3I+POqyEI3M3JgycMB4ENowxYXwTceUkV+0oOubIh7C\\\\\\\/XD+BAzDKMLTblznQ5j9rgdc3jqIOhk88QA9fbPfojcST4VSW0ciAwlNUklhjvqg3gdbPL4gWxEwWmyvjv5lGo+S6HlEaIF59AfeHHHLT0hC7\\\\\\\/QbKR9KV8sVp9VeDQYT3XbRUimeBycRxqyLbZCIB7v2p4xWq\\\\\\\/9DGrwGeb40WpthqrXPGIfTqC3hihcjWnWD6aSNWgGvPvwY9wiFaCunxCmViBCeAi\\\\\\\/ritE1UUSqvw9UBeMx2MCDERaBmDUJpSauo9waPhkuLUgDFxzVROIng1R5QceQzuQhM7rAZ1JZpccv5UEZdih8R4Cq\\\\\\\/fdG3VsUArPdYhwZBQp1puBTYRQ2cLlKjatZquxHYsd1rZSRX6lk76+9n6Bi+mLMrcagRU7keKYrUk7UVw2NeOGIkhtIKlgW+XXmaNAhJnwQazSRnJoMeYo6abIIDpXJ81dxhgZecla+YyDMhzFJynchaanXl+f452sn4U86\\\\\\\/SnIQLOncKa21IvPqBl+f64qLJIpCcdOHnanVxqZ9TBeL6ZiZ+dcexqqruxRYSLpY2hX2JKk9L82FJcZInQcAjV0\\\\\\\/qlVQkiyQXoYDXP9HvPRMVRLojOJJzEf8yaDZVPi+w5ppz1FvEXknFtBGp0Ov8JEnnAfKWbdBAMcCHZnx8AmHy7ozX3QFu1D081rr3d0ZhrhrzLgQIlpTMU0UVEoKPs3mO8dUafeUcMtAJEURQyg1g1KqHJ7pVcQ8zXxar1okcG\\\\\\\/ehuHW9uJnDJmbB9AdXpwiYs+bP3Lf4MyxNmvBjVlEmKh+S5V9ftC4hC3ozUYyGO2dSraSI0oFTwHhSMTM+pxng+FmrPgi3fGPI1xGWEAkU59jUGRS96IXFAdhRKC+KyCqvbp9qoG6eLPb8tGVOYNZ0CqHwIIj25\\\\\\\/tsFBYifYL0FFGzHR3Eo1PTpeVG\\\\\\\/D41WQtSb7JxWQSBMCJKZo6V\\\\\\\/VLbG1M2kF73+jUGP2uD4dFcLP81l4+5pb7seo83W\\\\\\\/KO\\\\\\\/e48Ve0zNrRCDLKLLsrwmoJu4lMkbpcdYvX+zHpc\\\\\\\/LKhvNV+XJwIZ6JDdy4MFvzdZQiJmJ0miqSeVA0HtC5O0Dxq4ZV+dhrLWAgG6FK8qRvQytxhciaWfJwRhC786JUdOBWooy+DqdNymb+QT\\\\\\\/HlLph5qZIO64DSww3YUV9d0dbCOd57r7GLh5THXO6BYF+1JchdQ8nJqyJ1l11j6zHkmg0Vk9PIyqTnpsQhGX8a6KmAgrinFHThZYrx8LRNfcDuK58gmQbSWaOf8RR\\\\\\\/sjCRSXzXO1mEDETDposAwPyR6vsY0BB5mvqrEZrrXPfPHO2vi\\\\\\\/XjbHSOovvWpt5+bBQSvNx3\\\\\\\/j0IctJUaJQyBtl\\\\\\\/oU0Dlu5DMclNDxpBWhKNyj3cte0SxnGzZkIa5JTk5xKE062kS297ckliTglcFR7V6hvrPW\\\\\\\/8te6CISkeQjz5or7Dovp084YBLMgbSJ8bDjO1GeYrukTdbaEwMMydId99rpe2GaWEaqUaKjUB3LzDUehcUhiAkOodjDXCRzl+YS5NSJVDCbLCgfj8PrX1rE20qL8UacsvFGcbb26VH8kknsoc4ueZTTeV9FW4fjO7a6hAXEPihzHE1XJEV\\\\\\\/9MUmnXcPBypKre9IT\\\\\\\/WQNdVL5GiW4x0\\\\\\\/rQYHkgo6NDbMaXlNF1BYtJqNfTVqm1KTp1fwjVYloqPAEWsNLiyZdn2GfnkG861SBNY4m5PHw7xhr\\\\\\\/wQyiFGsYDJoWh3vadnvUpQ1Ag+0+kLwiUw6Y+owEK5ZPrfpd7hseWUzdseOZVWrRvZ7k5oO6L+c\\\\\\\/TgzgqiHQdbW61++2QINirV5jEx+1crKIfsDgyLZG4QcP3f\\\\\\\/PUGhFcqI3McgowWHw3RhllrvwBA3sef9XDV6dGsTK9IOXvueM+f\\\\\\\/C4qVVbZ2NZzB2k8YEalM\\\\\\\/X07FeLW1MX71ys4Wc+5uuRzAmO0K323qd48ID4\\\\\\\/2G+oWWbmk+7QfQm62lOGfufGT+0UWgaYvVQDRocHdZIi6+xQJ1pdrbiiT1SBKxB8pV5y+kNsI5oE191JW2q6u\\\\\\\/bM00zvMRJU7VGqIUGGG6URcB97wTn2BxZRF7yJh1ARdPEzl3cj\\\\\\\/Oh4yyBIG8\\\\\\\/5v17veRQQCp52pb\\\\\\\/J\\\\\\\/n9kBqJaM8eNib0kwAwNCj58FzHH0g\\\\\\\/IJIOr0n4o+u4II\\\\\\\/5uixyYFz0CqpFjq\\\\\\\/0Kn1StquxfzC8IAHA27re8Jsi9JjayTA3pkGv4up3TS\\\\\\\/dxLSyl6y\\\\\\\/Kh343llmAuDfsseivHw3P1AT+bUFAXzb5A\\\\\\\/\\\\\\\/C7Q9qPfImWrL5ybq6FlEAkqLbnNw2obDIGwEdoBombq42yIUmJprXciD0w9nMuVWp3prCK20uUtZoFtt2nsDBQkwWOakcqhFZ7bkutSyn5HyEaTL6dVA4Civ08MmM+\\\\\\\/EFbTLGTWfEKEYFSv9lqH493zkdyIK++kGZPFG2+cuKUpJwU+1SmHvvpwwM7j0UFRc9oyBcOTy5iGiKPHYEVStQ7CnetZLZmHsmKOEWxhgg0FHWpXz3RKlv5ZKfKjah68JRobxjSNDxbQNa4qdHoxDzSaM\\\\\\\/f4h+n7rCRVyyyb3l8+N9Zxy65bMrsfzJpGyUHgAS6PrTePPE2eiUzquB8bb+NDWyZuPliOG3teXgeDxWGO2rI9VV40G7JIRs7+MgjYl3dyf4ayCjObb4b3rgZB+SCAl3GCTTWMkt9vUzU+Zwo1X5sF1uZS0Sm3p9HspflAmNUKEU4hUSwbveUQ91pgGGY8hx0ysNJG8A\\\\\\\/Jur9a44nzkSbRXhB2btJno4qG3SF0t1J7xu372daDWrsWw3rF5xGqjcIKLYVo0sjdqbvTVer5jVa61oZMeMAWgO2ac7L8r1+5gaJ4rrICcNU\\\\\\\/Ch1eTfYxuPptLUX36BjyL3HKpFAM59E8hliHTOg2LHEAqKvPKUYYjD+Le\\\\\\\/xzQizy2E0sX0c9mh2c3w9MndttpKan46odyWz2DLZNsyPcDTGPvvIHxFQg+eK2dzZfIXVwKmiMHPAaowKt6rC0veYpKjHe9GrTpQGwkGjZ3DJZ8uLIMrtm82ckT3+ttHo+y4T1wxcanvbxd1FS8QMmQWg7Db3U5wYFCCjCqGkC99jYGNknnQISg48jBV1ouW8guEvRdyXUMZnR4Q0cTnVRvMHSXxyg6tlB39aiX6AlaIvt8d4fEoRFBXHkn6FMEojXquCUXGlYbpR3HL\\\\\\\/6OT2MwAASmIpueV0RmStoDTd9bCYzAgDJNlssbxd3xYhjaQXUBAt4AmiasZ4TLtAWXsMymN2cC34stw+a6x6kko\\\\\\\/MwbXje8bRlWfyUwZRThqSauP81+vJyzAxEKS6Oss+XKu87edCYeOgvdcFry1Yt\\\\\\\/fDkfhEj6kxpUel5jZNdHB+CN3G1+deMz\\\\\\\/Mm7CA4mrZRxvisp3QmmB8Ud95\\\\\\\/s606UoDhi1GTpy7VpsQd6GMHmSHMYGcI9ahhkV7KEuoS2cIW1Ey3S9NvLIPFaKo9wJDV+S62wJwm2RkJgJcHCPPioN0WD8jI\\\\\\\/efv6tDaafR9z3C1HH+dlB8DTqStZntLVdrKM\\\\\\\/imnHwI3Gg7V0mlJ9gCP\\\\\\\/XMW8s+W9AE1uwwrm6zPRPhA6HnU080XUmAdHL9WQLyFgXlZp5qad94WGektCE7WoRYgeCRB+k3X1g6OIqrjTN5JaIZGm88d6Js9YQjDcEY\\\\\\\/M6kmODmhq6M0suhGahDWjhh6tImkk5FUSv1+ELcgyK2IbrM2D8iaPgLyRMu2swz2IO+q+Be57ADU3ndL0FbRSLiWEMdRNqjR9SOI4IoEzKU3Q\\\\\\\/qVlOFxsMWGBdFsFlCZjPHogmK3wqFOlFFnOYiv8TF41P2kZ\\\\\\\/PV3fjDrckz3SIDC6XT9JmVdpHDfRLAMVU80dVjZxwe6gP7ALJCXXoWfnWZ9PxKkZrrEEIvhdqPArX8E7q2sEmdJOy4V3Nx7TOyP\\\\\\\/vnPvhBxobH5zlW91oLx6FvNcT344sBBLOyPHwFuazVat1t9qEOW3sbftvUOug20gSAmiEU895SsRSSC+FzUfncHkj\\\\\\\/t1J5rSZl7I1bZPLl\\\\\\\/JVYSEH1Q8YxEVlQCPnvxBzWRCdxVLWcKjSACCvApqz0GXW3IB5FSHNaY1lTsimOPUq8qnYUz43Y0SYS5MdxwhGZEXgrL801DjLSJ0rZXby5\\\\\\\/mTHBHwjBevpq1RpNbJZB+r\\\\\\\/Iuh1x9HKXN8LPYEW2s9rLjilWIu9FGMH5+9u+aebD3QYjR1gdpDUCtV5xZW2d5dOMHQ8Ht2mAdAx8KvWAZmmT\\\\\\\/7S2U8HaaXjYY1UrWXC\\\\\\\/dBMFm0U0xc98z6CLNFeJQVbr0N6\\\\\\\/rYt9yb4LwvPxE+zAVPvWuCfgvqU8n10Cnm\\\\\\\/9dglvsI\\\\\\\/F8lFtMxGxkVfn93F522yByPo3miSxs4hu2rwA1qoDEhDv0BCQe0q6tFiA+VxyHI7YrhFPDwZUyUciBIcyZ7KFIbjG4BcoPID+fxXw3zzgr3v+S6Xv0G3zz2Jyzis7blvmjSqabZjmjMNos562Ua8d2DGtUhb26gMqdMwlkG74JstrsEYJNbDXe5G7vcWQVUJfm966CviNhQfFFC\\\\\\\/HbIpC5s2UeeyAfAGW827ghLP3ao4xlHqpNqHnwWyixv\\\\\\\/d1w0jaW+Y8uzv4vlPr8Z6Mkfui4CS\\\\\\\/Oq18mn6yhSZQ2x5Tbz13pbXMmi+NfoqCUYdJAffI+QlwCyF6q6QI3cVPBWVeWLQaZVd+zc+AFu\\\\\\\/Yk0gNl8fY9H32uzuOLG5+NmT66lJwrzR32TeFUjNVHhKQ87rwdRTMUgFSG+EAZQZFhmrbQML698GQfGsGxVqeNLLYQ8D4HCw2J7Q7kEL+l2hhes8DAW+yNdMI6wPPptxfnrOOW86Zd0onCyndURXl6czfoKEtMa1Uce35MgHZepNYmdezTdBXZVZO0xK\\\\\\\/qwusc6AmxVm\\\\\\\/\\\\\\\/jn1gVYy57LDNGP03LYtI6jvEFz1CT\\\\\\\/ZKU5JQaLZL4lcXrXAYSkpqmNn3gaeA9JoIDdiAlNbp91t9fg+e3iBpLSMMqU2aBfv20aZLkMUYATmysHE8sRDueZ\\\\\\\/nTo866XxbFNK+lQvpMJMTetvaJAvhOiDaWtRzxS5UiAMlmG6M3CRZjjJ+0lQ\\\\\\\/9O9pHwMyIqLfMbtyxZGYIp8X1H2WvzaHC7FWmg2TJNU0gUXUB9rVdVttK7JBMyYBe3t65VELrghOJcnIf3efFXqf2l99FzH7awqmHuLlnE9z3qPwmhksCnZ9Aiz8pQ+AIpgibRmy5QpMNdN7BYQOeo4iA\\\\\\\/YUXOe7Q4vklhnJI8GnM24PwHdt6GQ4B2czcznRa1D8dQjAb\\\\\\\/jdkwqro9l\\\\\\\/t1VFt307JA8R1BqlfGwD5+HwKJTDoLLyq92U9mrpQBWxhEPT70Zuy0nZ1B3TLYPUc2fiCE9LxEvRm1ZoFDHT0eswF9izFdL+v1isBv9qrdnSYF202z+8LuK6AoP\\\\\\\/SbMjfCdYuJeUpBjzb0oOwxsS+DbvQ3YiIGKRtrRMkQwv28gx8YbuHoCvEPxeijL2UtqVcQTtmdG2P0RHFcd5B202PPiu7n1w+xLzrliFC0Hlehvk08GLhmZbTJxuyag\\\\\\\/MHuIFSodiim2VxVKQguHHDTHkdJnR\\\\\\\/t6EuO+X5XaGWTnOhEdCusO2EgDsdLE4vYsLJnQFehNeLIJ1fmU5WzOjp\\\\\\\/01R4Ggk0Dy1WRdcAi0vf1iTm\\\\\\\/+UfQ+sPkPCFxRZFqnByMLExsGhBnSG6KLW2IsnY3laSf+1f8cZsyms+y0q2oBVNWp8BoFA0iB0lOW6zgoDtNQ70RTFMxH\\\\\\\/EkeBwk0X95TCYS2BhVm9bIw7uo0Yqy26kpQqqmmxmPGR5f7j2qmsqulmYfNAdGSAaPQ1tH5FFxEcNa0upaqkruejt29KA+riP2dHijSH3bspTWNHF1Ls1STisrG2Yk9XvtbSx4jUxIjX+UxLRFqJpkRtsqn7EikiDUWCZ+fCjRb+XVeQ6uE5xMmxAPRIacNf\\\\\\\/jElUxjFEDSuN46HZAH7tJmcAwBBWgSeT6xzVbK1oZ+qHXC56HjXiFHKCyUz6y9SLnOnA2CLSI5b9RoeSf2UBk0EJ0S6TodnvHFFA6GCuG59dxA3XhTcFTIigFIV1MaZGXrt35brZZ5BxSyr\\\\\\\/R8aB3tp5ML4uVT6BM52UJRW5L2OWfh0RG+kKZJyx\\\\\\\/bdeOenh9Usbw7sxkmSxGSP64V9GaSlPOTmm7VoF+2Gq3814lvtvJ+daVHy4IN3ATwtt6bFNdmk5RJ42Ionv02PWYsj7LnsaX7W9VHx7w7oJ9k6bCQTAFDffejgiw+60qq+lPizf7bYUosxfXGebfU2CNJFvGlCsuLwBFtZtUylwYqK7bBv2\\\\\\\/LXUy+9qd9LoDElD+u7MP64KY7F10tMgSxodacuR7o5ENbtZti7Q6moZR8sdmimZruIdcZaOM3WPehGkXs2iPrqTtzVxm2Kc6uprg6UUVHFOXf7fznSTuYlhNr9NrIfFEodFgNUH5\\\\\\\/eMCKtGtTuB26Ggy0Q6Ny9oqyjqCkVJyldN7piymd1phPCvv+LvDTaep9VHCsk5u\\\\\\\/0jd2pWep8hyJnQD+10W\\\\\\\/AgCVP0R8VpXVz2ZanxNXOI2O\\\\\\\/prPuJ0plq\\\\\\\/doJFtU3+5DaW1wWLwwdSpPbMlrpstElpphcwYSlYM\\\\\\\/4Dn0+ob58Ccw5khwCaE2zXkv9Zjy1iexsDau42HinbbZBWJNgByFOrgIO9WvJU3WsnJgik3l75zzau9+aLtVSi6hexjoToZYua+KUC59obNgyRKKbLEuvjdrriaiN3JfilE0ciZjLX1nrdZ8qvyDyYdDHS1xd1OErmmbgyRZ\\\\\\\/LDmT+QMRGvAjOk5fY63G\\\\\\\/z4lz8rVFNHXvmJMj\\\\\\\/unkpNDxiiwIkEPZEqIWDhA9J6TMQ7V\\\\\\\/pQ6tG+CFjCiyW1dr\\\\\\\/RSBT0xOyamAKcdMZUprWyDIBdftzb9cshxbzgnoUd+QP\\\\\\\/ADKiVQiTue4oVcMCe6C1vNdbrjViZ8RHmdEA0BjJz9PfXI8Yu\\\\\\\/qCFWwGtuQlzPm+i+eUNcXqNe+uyjH5hiojLz1d\\\\\\\/SvO9fWmzZ\\\\\\\/4WjFa3JWkxTX78HmD2GTLEvDXYJJs0P70OEkT04t6TSIv8AQ0duOyB1054T1a3WJFqj9K4Uqxa6EdFts5YAJRopeblOYDI+k3Qc3I+NPtgCm5VxKSHkB3roPTr\\\\\\\/UpGfgDZKehSolZmKrsRNYewttKOUiY65pEqkYldH+c0\\\\\\\/HUCuxy\\\\\\\/VW8WynKgtpwi\\\\\\\/F3ucdj7mVj\\\\\\\/V2OCynXgCn0w+PAMJE4JrBhrYCxfinJ4HTrwuwNGEkUuHVVP89px6OHU5sxsw78uOO1Y\\\\\\\/iHElafqltkzpdJZG53nLzmHyzP9WDULrFitV7Fn3r4Npmd3F6laE\\\\\\\/AvdDcK8YLSBrHVtC8UdEMMOOfOJF9sba8Hf64khvpvqmO7TUcTLu8E03rpRiLBVB2r8vTBg+ozz0K\\\\\\\/j+novCCibJptJIaTDPf5QGkjOY1YxB2EuKJLs3oUT3mkqC8b48\\\\\\\/PH7rvtgJLJNUGIVsBIx\\\\\\\/+yCYJr9BIu6Av3XizZx9nBSIGj514CSpBU3DcwcKCgmDOCGgiHt6CvZRJOp\\\\\\\/qSNGqNO+TwlpfsNuPY40oLopvY4Aq\\\\\\\/SJP8bIa88d5gyrFkguL8SsJCajeKxm\\\\\\\/tcJRlVzFXStcfhQ3acFR6ZWq9j1sMLTWW0RxII5L9ht916HhIavdcahKaRYJvYQCXn4Rf9fEzdom9TMB0tbtk1kMHYtDejvvxdXm\\\\\\\/x9gIRIZ2Fz6APH8x8\\\\\\\/mNiKN4TaEhYb9wA9WuV9w9yLGFTDw9sGybOCQZQZVEoOf2xP3GLTaXJEPOwsyN0ISSkSfA6zG7w937i1tSU2oraeGElPPcnLgQPzT+N8uv50BMWyq2VsrsbCyyd\\\\\\\/TViBh1kCQ7zFevp6okSIByQ8eLLf0ahwvLqCgToktEjYnRczTAiU6RwUEhmp1muHxHZINRJ4bzrtFmELNL6y0+9VEvZdNP3X6zGCdFBCkDVf0rIknh1pVEHAsoBe0aRIPzKUNNp\\\\\\\/XfOJAqI29Sx7cjMjs0jHl\\\\\\\/1zBZR+Cb\\\\\\\/6d1UfyN32ltmqRSdMARIrgO1cg7qjhKdNRsv7WT32C7OWTBRVpcadn5at5oOguguPHzZ3hVpsE8+06UaEU9fExK2rXjQJbQyp+I6WVeBc5e1GE\\\\\\\/er77LZ+Ap7qcldUVpFNkuYY9tUB+RlqRknjKI\\\\\\\/vvtMTBa2Vq8sDkHaM7Zo5dlE6WqWKAAxW6P4AoeVXYKv1KaSMk4St2acaKRp5\\\\\\\/kmeZelGu75BH48EzjOgD2qdZtpNOAMA\\\\\\\/94Kmv1G85PMjB9Om9zceqjIM6ey5HBnTj16DC+NmbjAdV3aa0Qv8E\\\\\\\/10dUDPA6xgxnMvCKuaA5XORZZXHioz0y1TweFn1Why01SeACCjEMrG4CUe7ZdE2NryqCIjdIQpoC2y\\\\\\\/Nr6UoofCLqRjvHb7HTmsOKGMRDTepKCPAmmj8RRmj\\\\\\\/jTg5dnOGLcAB2RojqglycZ6\\\\\\\/E2y4jUvYwYM1aoyZBDgN+UHd1uWIUm7f9PcWGOv6z3KabGKMYAVfH\\\\\\\/x9YcxDeBJlTlJKIeWwhTN6YVJqfzOTHF+jK8hf+JVB0z383zk4J0p\\\\\\\/K9GGGApQYucuw+SSJT4T49Rqp\\\\\\\/nzcAbO\\\\\\\/6U6rwqynAvvqLIKjIFRzm8xjJBnqbs1ffhwUs+hdHIrrNzFy7TTUp2EIuQvXhnEtPi4XIulU9n9QcV71ChlZXRZCzm3frFakuD18r0AfXgNcvDJTUaildR30IaMZwi22pH4i1PUUk8oZsznbqkP5PnUleZskuL7jlohdWK8OrBIgiYzJ\\\\\\\/Rmx\\\\\\\/pLfAGVov5EhpBCRhGGHMaHqWrx1SOw\\\\\\\/alB\\\\\\\/Ecn\\\\\\\/Lk5thTkuF+MrFCtUiAtJ5jfxvBCl0OwyZYLugP\\\\\\\/z2XaKaYZc61ziD\\\\\\\/h3+8NjrbCR9gvFrw2XYrwByXdlhsX04tcBAViDXlUzgTeokPhgjMDN+cmxw2jZZm+3thcYUJFSzY08uieo5kc7H5n6Mbe0Jv8\\\\\\\/XHggNdQPSjgGr2mK8Q9BBI01eeCCBBfwZNL2SOur8o0iRxy4YTTiTkaYsi8wjZ4h5lnHeWzrbTpbnx3OuwhKl7Gu6COOfZb2LGrYw2VBa\\\\\\\/gXQC6yIkgNzaB5Nyr+oLd1iOgxQOeuCCdU4Y\\\\\\\/MXsOOHu6pz1iDX4kho7A\\\\\\\/zKu3tFZTuyKbWgRohTvX4JFoA7who6n4jAsYZYg7\\\\\\\/R0rB0exc8aCXkT+A3s8\\\\\\\/S4z8rZs7w+XBDQUQskLhzOzQH\\\\\\\/TF1I9iD133OV8PAxR1KhqKXfmo\\\\\\\/Zo4ojXDmSlKBLA82zfJnpwmr7cHEmuK4U+juF6W0K\\\\\\\/\\\\\\\/kaRvHsBmBF03gMLLsxOclutT+Keow2QHQ1FIpoDvUADJUbAsMRNlOMqmNC4C8YJ3Ly3IG2IUQPC+5PV3VxwOZVF+4qBh3+yg511IZVu4W0eop++MdmYyFMd42CHIVF9LfntyRQp2ub940no8cXXbK\\\\\\\/2F4QT8pWr80E+mIJqazL2Gx4nR1iQgvimtSEn0bZ6+pO7g0mz4CX9dpBLwlXSRL4319e4znb70LjV3Lk0wSvayJ8Lq9VFc8ewmn2hGVaMkyv83CsbKrUxX\\\\\\\/dKgOZVkd2L1JWDUYj9hisXUVNGaBPHQLcuOhcK\\\\\\\/KFdPAezdZg2EPH8ndxs6QpGNGPiuQpYmYqKlaYVue2Hl7pczLpUdYHzbE3\\\\\\\/POxoqgLA6BpwP5cS1Y276wDEJS0kz1GpDhpx1sV5ywxzsfef3i+Z+w\\\\\\\/G0P8qLASOovgqd3GU2C9daf3RRivjzzsVseuZC90j3b\\\\\\\/rLD4+sHd\\\\\\\/SGG37Oha2iw4jvIW+aC0wSO2A+YW\\\\\\\/GfDBPOYOihwOUiz8QI6joEac6R\\\\\\\/DZbwLC6dA\\\\\\\/JLxS0isRE2MeqB48ThGbc7aKyvHhkOlDnyH+9\\\\\\\/H8rl8a0Kxlt\\\\\\\/BnG3LZrYX5BGtMiWHS+cZkCJkbyXuCkEG+4yyzNhuDuQvXVteI4yytugynnFS9UbPg2fj18XNGvd5zZeGNhzVjs3ARGodpCwQpav4seKncJJew0ZiuHqKj\\\\\\\/Bk1QTrBCMMYpq1k4HR+isbXryzUz+Ef9yqKSbPsUbz7WYXE3G9ZWFxRuTttXbljars3i+hgQhE2JHE4Qo8PhkhmKOnppAJp5DvmaiqPLiLGzjzv9l+29YLN41I+ABnQvpTsvN8QsKfM79WIzm0Mb8DMA61FqZ9qKxMt\\\\\\\/rif\\\\\\\/JvQxjSOiCJitypNIcgB1x4FGk2Tt\\\\\\\/beAtrpirRzmwUT56Rh7OqBsLtWe51ALUfHAn5VXaBwRq10vA91wJganl6eNzCH5zmM8gCmmPn7dCRYfFlNUuRWxyg3HyOr0FbA0ekoDxl3Z3+3V0AWeyAD2OPwtrtbs1VcRPASR2dWLvWYhpSh7Syl8Nn3ww4ylHuCP3i\\\\\\\/Zdtixa85lB8yJIlu0NTMu0r5qFOaGW9aiKj7zsWYbvZ\\\\\\\/2xcz60GBGE1F7bmZ\\\\\\\/y60eoho63LbOdOm+KzZ9LegG3WCm13OcOUbqNU\\\\\\\/As7psbDP703ezip63JBlo0rswqlKast4skL09Y4IivzL1oUjUvPhb7zgakCv1BXiO3NqrSkWq5AGESPHiU1Xk2hmGYS4VWrLqm+fZPdp7NkW2WO2PoFKlsVqPshY9MwArY1bJbkI\\\\\\\/CcDPGM+43+ctp9dQc\\\\\\\/viPa3UC3dnvx4XrrbbmQQsJnsKiH+fduOyZ6dkyk01yviwQ23lqYaJ8DkfkuMDGOwA\\\\\\\/o68Q7TdhoaaaQKG0f34CSs2bdEPNGEKzx4pDVB8NQBbKrRXBkxR02OgFktq0LbEwtjp1PS0l9gQCkS9o6t4oZQ0bIsSNBAS8lF\\\\\\\/avrG6XP5a0745jSaJtalnA4d2\\\\\\\/lnSVWCzf+05S9ApPF06Jbv08k7jydfuyacmC4NmL\\\\\\\/gAn9g50MvWyiQJ5rReaawhtkBl5AguDoB13s7df7dBe+6cH+p\\\\\\\/FHHGPv+SM+PPojjnqKoFQn5Z+9PVm1z3iyzRLuJEFj57dZurt+f2PY0HzZNm8oh9r0+ugA+EBHVOKYSp6VQLUfIaonCVF7f5YFSlYzSW3uUIVoI9mnYQsiuT+0fXEtakSwocBW+LmYMLrUj9PwjUimGIocrnR5EEGpSwDjPd82PrXcI7Be5CyK3FkaLTT5kp+XybcSIp+9BAiY\\\\\\\/WApkq+myn\\\\\\\/3K4B5ss2drApnLRzKcjrIIac8eckeTvAZw72HyjNPYzYqZTRxMFmgryEqi1K4PNZcoQY4ktmII8KASJurZMeUDhl1UT9mtblr8gA1KkbS58380tr5uU46vLCYLxwWnwIbohCkUpSY+1qGb2YzOQtfyJpg1MuVUQ63HkguK7\\\\\\\/DMgYWVeX4p7l4bDTDRjDWgfoxjhdRc5UKq2iCUMvvt\\\\\\\/WFlubu3A9ZZW8HW3w7UH1x5qIcq2IZYQSy2UAXB1eyhJ98+Joq8GmAdv5sf+bJbAMDBvr3P5+aB\\\\\\\/HbU+Kw39K6eC2XnRu5\\\\\\\/bW+Xaw3i3PKyWrSM4TYbyDoToom4B3qbWPUyWk9iWMRKqsTd\\\\\\\/y+grh7A\\\\\\\/O6YbMqslVRG+7gygzU8aKAgpYuC6iPI9EKN+DTJuLZyWT0apOdxr\\\\\\\/HyD2WlR8T2D5EFK7nys28sEovuSu+DYRD5dys+7ogth01DEDLKY0zIVwMvbXxIsasKNy\\\\\\\/sCRB9Pm3RBlZR+khVm9S8HlzqBq55w2EVQ6r6mzGQJxBhKUtHoWF8+eRUz0AD4krspdWzXBEa+ZMZ7qwTdCqtkxlPoXBZ\\\\\\\/lu44o\\\\\\\/dQwF+yXURjXEeEjV+kCbgeby9YlJPGs8HSiyJUvrhmSdMLX\\\\\\\/CR0tYogEn1+uRydSWKEq+D4oNW3v3MYmJVyC395xQjT29Ohi2hXxQscESrhEviqdPqpmfM85gR29bk5rSyE\\\\\\\/zUQZ1LjatfLqJtSuId7l5+urvIwou+x1GTTkEbNSQknnBJBzaOThd56JUDkUCB9a42ErRjv1qrlwnTAV34xHWh9HN0yJ2umcDtpkI9SqDPT97qA1nfDK9FS4Zm\\\\\\\/bHNd0iPw00BoNlnykQQ2mki\\\\\\\/UcHW1ONbr1uPf31\\\\\\\/X+UZQwUCmqbAWX9PmP0IzA3HLJr8RmSP5Q4yUTghwuqky6lAAHQ81WVnW8l6BZe3hytRzBvwx7y0bm3Cp0GA1mcTzooCbruerdyzlIcKn1UiurrYrKeLHAXd2+h8gvL00rXh79HEfUJhiN4pT\\\\\\\/H8uzRdO18MDzkjJZXkTaQL6W+jVzhDLCN09JK\\\\\\\/ugDOWs7LR4At2tGChqfPMxID0WlkkPFMkfaXjcmQNyRbPoapq\\\\\\\/KZin3vTcqzGxVbPy6ddXARZnJV\\\\\\\/pJbcdNrTIcymUE5gnwCdTSfaHDdDyLDHjE3wWuWkdOZ4noJI\\\\\\\/LxNftAvAyNp3nUNbG45\\\\\\\/Ng2f3wzWNkonAKEi0329EJUyo1YXPMRckh84pLNG3dQw0fLwf\\\\\\\/R01ViW8fV19sSQ8699pcpCCKtktZuJ\\\\\\\/cwG8uw52e4vE3k92gj3jyAH+k3JnKjhiCTFRVrzIHqi4kmFlQZ8X+o+SAUIEYkFKP7AvXzFGAWKbOG3I2EUwfITZaVx4zxNdUCqg6NxHM6+p7\\\\\\\/HKxVTRRFpE59VCL2okRyVRvgHhYLC3wdyN\\\\\\\/yVbnMfEU1piyMl3Zm2Sk\\\\\\\/t8EV0989dgPDetld\\\\\\\/OWjWhrqL9Au1NAl4cyjLkViZZBOR3kX4h8YP4GNQjWLlG1XDMwX216ff8C5186iNfRvQB3ohodQpoiSDzLlI2SYERq+4UhPymGZu0nmtqPVDgQAA2AvSTGwo67+8eshXMtN8tspoc4McxZLqbv9KKX9bjefA2K2YEz\\\\\\\/KiodAuC1SCcQ4jbbYLm0mGWlnvdC7p8Clw4DJzzvcC7A75+XVDr6Sc9fZsaNKuKI\\\\\\\/DiS8OgP5meIZEDv34td4qY19GFBMzRIQGU62kmEz1jhA453LABklyd0eIL8AVVfSHFrzxyqIDaxVkMuwm\\\\\\\/It+fPBtm7moczWJbBzVmas\\\\\\\/+Jfd+v3yH4HfDUqwDU2PZ2cm+\\\\\\\/fQAQNiwCWcX3FnhLRtf5wcIcXXweOTOZ0zfCaBgisdrzvbXQxfmsXq8yEObtbB7KDlb5ucI2VOPsRokGbBH+AIZwzhbiHDNGlo7C\\\\\\\/1bTsm91GYaGcg+N0n1PYcrmL2tqCUo53OpkoU\\\\\\\/YlWPuMa01j2Rn6wrVVV6HKPvnfSj1kfXHsNiOV6YXHviPrn6GXCv33RHiNHySbvV17quXMeykoTgNZBNgYRXGqBCVUWZ9UB8yY4sR+Y0URbsctfa7+XtvnwdNCaj7Pt6PSNRSeGXyD27M0R5UbPUpyOJ6QAhJYHKwlqBWSzPDouHWPJsdTPykiUXZsOKraQpzzlbl2mSoBKEVtCiLRIFjwt4nou7iuQCFRMyvRikRYSSGsIijehWylmkyfK+x8L3f9iU4DNx3zma\\\\\\\/ushjxhID5erYIcLHJjWs6J84ReJxSOMJmsEuq+yB3nPN+JGlgrY1k5XL9qkTadrtuDMFHYn2yQSahq+EFDhq91xVFKCgJtuOSAoNn+0GqLHq+fCio6WsXM7pLi7d5\\\\\\\/QL+BKqyWEC1AFtB++7sT\\\\\\\/4Sz4LrR6Bi8dBmyMg2UzfNeSzk71KNSSAl+ejmuBBLRwHzKln9KeFfFx10FMXXmuqeaYSkvj53DJh0VbN\\\\\\\/7OEoYIOuuhAYjMEqDqdlf50Muj\\\\\\\/CMHYMKai9YdCyS\\\\\\\/Jc3fkLNLSMD2PbFbNcuZtS1HbGrHckrj3z31VdW9kL81zVtDqGDTovO1SZoZ9BjU+ozYHJxnlM5vVQA9WSOjmdJPTT9YkCNKMsRFKtZ0EyY8g\\\\\\\/1mRz5IKaRsuUjQjrfpigbBL0EWsfusHRP1gy+lRcMBx0VjqfWb29hWIoifWnk+eTJOz8CaezDPk8P8ysdN1DIDyjwplp9OEj1t5FDbxYnm2SqMckm01bxIHU952FiNlWEOVwvIQ+ioEewVxalHmkbbt3dfZ8A5e8SQ76zPtoTjJ+CKKnCR1qc3G0BsyM6s5ZX3JJIE67NzttpoRtvkcpygE4rujYkU\\\\\\\/6VTvkpvRrfFNtLK4jmJ798DfD7k\\\\\\\/Rzm2CRpZWCpisdwrQJBVeZeo7hpQV4famYn\\\\\\\/agCMdr3JjzmIU68NmL2hztAuXLCbUBEwcNbu8EVhlGaj8NiuChKXF6RWNK6ocu2VvbDr8mduvJOpZ0Cafb284Tf5GPZkkQGUwTLa1kEwnp58BXM02ydrd2jqGq8xYSZL0Z3FMUuW+Rm4mAjjdUYxX4i0gu\\\\\\\/jk55L34kB+byVPFx\\\\\\\/lHhnA3Kz7TMvMroeX7dEm\\\\\\\/IvVR2WMfmO+K7soSWjtKNMOoEoZ7J4jEFIseArzNHlBdsBjTEgunH\\\\\\\/qo6JydjYapD\\\\\\\/TrYiEpSHJbISyrJygi1i5BJwQfCH7D3i4Q7b7XzRfwtgfgdsBXpjwM8g0XIFUlYDqNTCdXUXhd8mmW7Zc+zr0jkghta1zlNMLOD9AAa96bkEkzHEVHUsHxYdJEfzxH\\\\\\\/GvhHUTSJjhCMf77O7b6CuRGB5HCXTlh3DaGI4mrYQpUKGzSa1oy10XGuTnciIL2SelaCqvjU16USqaJ92rlAraLSDZ9UJmKmV2qmI19uMOuUXigkUYLx8zJ1bxMcF0K9f18wZFWj7UxlONpMZcjSYlJB63iOD8wCeiMgFnwMrU6bIc91PROyWXeNZn85MzC3xp3nYmjcLXKJR7d9GmYZ3BQw4BUQAB\\\\\\\/JwCCjHNc4Uo6Un0y4kFIKedHc\\\\\\\/WEZrJg4TDYazy4jUNkE1Ykd8x26bSf5F9iIiewtpKwvJyHVDOxoD6gsBz8p0vE52wpWN59jgI2J3vFc0u1iCX+qsW9a7QjZiTcWBCbP2qsvb4Ym775JontZfHJ9Ni25XHGmlmFaqR6FVYwZMEVwimC97KYhuzigIeCkBHF8c+9o138IoI1JN87Jt4oF2O+0cY\\\\\\\/z4lgBpNYGjgOBKaHsGgndANSkXI+JOptOw8EE5\\\\\\\/tpskisqWBzBdh57QIyfn0UBOjNUw6TgiXWy4Bh2qMj0crWHEk6TGPFn4hvWKhetFvxUhH76Ux4o95ovlWLG4eOhTvI4d1zQGD\\\\\\\/SzYpafYNr2GiujXWNlZ\\\\\\\/Wy0AhQroNyq7n5YX00uH0OGsr1EB9V\\\\\\\/0wz3Eo8kYDyGwtzYSqgGv0Dude2r7OW0T1YH7PYtfj4K8iheBpDUK0iiV23rBG4QQq7rgOWiFp4MWAf9m8XnP2YdaD0roCc5OmWktLQByZcoa5EW8a2sNu7yp\\\\\\\/egUO4Bpk0Lut6sSR3G8EOpRNoOxZ8LTBdFk6HKGThk6cShAV687sZUOSw7kL9b8e\\\\\\\/wQRiHxfqiiSmFxgQlWWQD9zGHsnh126Q6LLnio5utfcQoum5zJ2pI52FCJtbKjNRjwUHuyGv+VWqMsapwzodDrVVhsIPbotSIwVMh0CMrPqFVLWFjeztjuEuAlyCC+oDfw4oY\\\\\\\/HkZ\\\\\\\/IksdQH1YGBODTB4lyc6lqJp+dH+a24A7b8lSZT\\\\\\\/bZyNhzhPCaTGyXDHDYQbKkesA8rC4X2taglojFkl7R\\\\\\\/niBNLgE8fyL3xBiMyTrOuipGVBrm1AnMLiksOI\\\\\\\/ip3lhqsDV9+mM8duzfQeYNrlj4o\\\\\\\/zJmYGidcqmolmnqdwWMKwarTZOFJWCz\\\\\\\/7iUZ3rNu8pthRK3W6v\\\\\\\/06M6ffS8bP6dgi79ccBZ9av2agGvhy50UoL3ZvykeevA50VyFd2wrNnHMN4K+zzN1nga+dYkwlyjTuIULWAFzfVtfqtEZNNBZt\\\\\\\/fO1Kq+p8dKwF1NFGfDm++KrvXzoi8+CI+C2EnKhfhkuJULzBDXfbSpi9ohi0bpmUfSb+F9sS2\\\\\\\/Z+FRYZs1yo1I5gRivBbK4e1PJ1l7rQN8cJOdjhduQp6NSCRuTn9E6Wp7AeUap1AiqeDS\\\\\\\/UVS35SUhyLDpIfiHb9VlZUqbpXtWNswGjxyhhXdPrvyDYN0daifL4gEiDjly9nMYoSfPQF3AptKrKkA6JSXDdFdNCNckpI6N8fBYsCmtQG6vMEqfaKsQ9ioDBb62BXpbRR5Xqx42BBdZCpZKNBqBi2s2O3pJQ9XboSnAvjtD+Bj43KaI7W2rwzek0qNyMybq9r5W12VlxPylQrrwSYSAPzUQO588U6tv0KUrU3\\\\\\\/NOP13l0GpAmYQ+DM2i0m\\\\\\\/PjbeBYkW22EkXZ1U4xVpRCSwL+lvPmsX5dyt\\\\\\\/JKVfS\\\\\\\/7BmbjCM9GJdg4vViaGi3v6LPwpMM2otv5M94Lp6wSdFBjASQ3xZJcb0LeQ5UqA6tpxTZl1B594AEZiaO+Wk5x1xnpRrz4QnMXAvXgg0pXECxh7zeRenDLRxkbzayR\\\\\\\/w7epGRQpDzs4v812rPpTUcTEeA6wtFEP7Mkcz9loQlxiy5V6+UOcciLnlEca0J50T7HqJ5hZVwfMfP9gWrmRsalHR6+sJyRusDoUHTR1wqAOuQq5j7NKPMqbpoQXAUNBjRGXZUJ2wZSiDqK8pAmeQPbdgw1qix1bTlafqGi8vnMUYyKKOGfoRyRz+baONda8Jbz0r36KF8SCazfD6CEIRWeEQrIsVGf1rgSHCbP6ZMwiyN6YMMhplVTQMawzcNxj\\\\\\\/nmpidolXuomDTiHjjf1eJCILus2LupskJU5flWB5nOSPzOamRm\\\\\\\/d5e6Gt7krDHnQSx1Q1hrjUuU++5OMoudW7vB39mgFSxuGCW3eHjoqVNGv4oK8\\\\\\\/P20afrTi2gd81TSutwLTBy4LUU\\\\\\\/IxyxO+KezHFiToss9XHvs8N3SHuB6lTwJrFMv0MVf5Z9q9C2v\\\\\\\/089gXoOXA0+3eupCariL4hhG366OfEWDMSaPWe8xbQ94rFbC+MMJy5SmEC7xUdhg9dOf8C\\\\\\\/zYvR4hYdTx7vc6TPziDzIg9cfelJA1+\\\\\\\/6OsagsvZ5vm5o+RWSkFPBjFSxtSxAfGieUhdLC\\\\\\\/yAklvMLhPA2+fZ4usFSbZGbI3k9pYeoGUE68Df\\\\\\\/OwwWzatXTYI3blMGIIhA2pT\\\\\\\/BfgvJ0cD+HJ0IpHL\\\\\\\/81nknUAy94OGU+aQtcolS88aiIMOXWEexRXWsP0qYFT3ehUKLEwyjtR6UxgnQJxe3xEYhky3ary72EAZgDIOF1jR2ZwA2ey5KfA5H+m27Cj4Bh+3FUkdffAJvbdN2ghTIFnwjoAiAUp3A+xBfZj6ufMyj4wsXxZJMHkLoOiFF6w4DVI8Nv3LZiyJy0EcyP98tWgtcEeDdoFjvB0ivrMvSUxeStP1R9gqD1mF567r+VTwxBO9tVY3BT++bv3vGkpfSd8OrRjqazFczWZ5GwJ6KaZSfu7hw17uOq2VPtbQMIaaX3aSm38HZG+292mF5n1ilqm5NcDpENOw6KO+u6Nd9o4kyIXbqCUUfULbTKSoxZaCn7udVaiut33dBrbzuIauIxkAgEKuMNs5m5ewaG3m1Dua7p4LpfovqViB8EmaV\\\\\\\/b8MrIhxvi0lVBiktOxt1qPe\\\\\\\/4UaUjtVXb6jZYOtQcz7JWr6ZRuHlV2VyqJ+cwdYz22hU7rA44ze4HLYiMiOCDBtV3t1mKZmV+zBl1U7rWF5LuFcEUDVhhrkr93VAK8HpSiniZ98FjLvA9wDZjcwtsELCpvef9lZkW0Zj6IGjGQOznMc\\\\\\\/heOZ9a1jehlG0ewhhOxcRy+QaeGaLY5k6ZSLAErqYj1py1Gy1D67i+XHKSRtuxsi\\\\\\\/c9X5nFBm+LyAAilMkozGEJlg9+ntyQcZPW29gEbPJja2RlTs1RF3Tl8PTaohd2ZG+YoYdNUYtT8NTO6dm+t0rLinmVDBym+dqher4SAomOAGaEuyXWcD50kBZfpDNeJ7sweksuH3cyjGxF4xhohNNqVTLJMpPSSouBe1rCPu\\\\\\\/hDCV6gPGSVG0sRYVtFPUEe64fkCu9TQyP0bCZHmCLEUMOQrrFkn6+870ZCoHwOvGRBZbPF6Y+FKpxoheOmaS4c7lw99YvnfFr9tZqn7bVb6a\\\\\\\/F8VvHclCcY7BEzMbU0g57ffMLZkRC9lVr+DAyUu5JixGG2q1eY72ue1CWaZ4J8yzs6xdoivrBKndWo3zYoSmEkIjZ7lCXgkta+PH2JZYZe1pjF4DKzGj7oMLxfYCtyLOtcJMUtQBZDSvYm+mTLxy86FpDwSgTqZ68fB771t8XWlOQ2RTOdeaP3bIrDUzpe7DhEtW7nCs5XVzahrAbZlNATL5bG+T51kvkxHKFYYIuUb4pdsH812jAHJMCJssoGbEDHpN57mhcbtYKLckgV4KXy3Tn48RC9W24X2nTiNUwV0z8Jr2JrUPdbvX0WUEgPyCVRwW7XOE6GFxDh8S3ZdaB+JdPcr2j\\\\\\\/29\\\\\\\/7pfaciGUi3t\\\\\\\/+1Vix3PTIeRqiS6GXBJ1uq8FESmQSwz++4k9bsImI29t7h8i9bz7KgPLfCOq4OySSGG5zCCg3XCoscVAcQWoUscJ43nJQAiTuaPqJ1wmUiPlwIdsU+t7f8tkFG6HPcbNHWzDhWWT23WwwxZ1yCrjnGmuwRbo2GW\\\\\\\/eLxnObBikitvdxavYdM1D6ErozRuPFq\\\\\\\/IRGjB2VKdK2G6GQzmJZjo1jOhKGnljP78JSVq5Mu+FlZpX9ilBFbfVG2negO7cgPX7ZURjgdUKIYnVCB\\\\\\\/5hECVfMpdZ5lSfsvCMul3jkJfvIYnOYdy9G0IqNpWj+YWmsBq39wxE+vnV\\\\\\\/+lajo+3oBc4eiMkQnOQxjQEfVFYvoFJlSqGrgcmWTwVfW60i+9Io6ZWhu9mPijTlsQPg1vQG0J8vvDVYkjjOfPTmHxztrc7QNl8\\\\\\\/CXBuECUNe1RCLL2GQNLJlDFc78lsFESZLe1XWu96q+BQFOT5ye\\\\\\\/lbCvajCkNOZZYHD55dkyrGCfBhs1DTK1YCu78t58zy5z81Wu28ZV1yTjpkklu6dkvlILSA7863v8TsIrUfKuFi7O3YRRHMy+KGSzA3\\\\\\\/VqB\\\\\\\/8LblQD+7ILTe6yO6Y91y1DsjQj8ForTESUTxGKrj8ftkXJi1XaDUHu1Ipf650GCUiwCnTK07AaCfocOhaXDv6xB4VVvHmm1s9KOeHwXS+WXge2YWRSHu7XfRMYMJ8DEYS7uYBvOjTWgpStBXFQi9E3+1fa5plQ28I+YSMMjU5+C13r9AZKmj9tETQKgu6xnC6pH5sIVS6uNJvg4lPFUNop\\\\\\\/aoJ6qwWTttVSNH+SsULJp+\\\\\\\/RLaGt6l0hybXeNalaYeKy6d60rZ8EQJMuo5X6Saoxm+ARr1hVQYAPlMgANJ1h+1o4RGeN\\\\\\\/VJ4agv39uV3IQRLjkurLzCRGRcodNr9CiFjWj1SyaJxOD0HSCeHbp3\\\\\\\/MVvxrZOCQvWp1uhJ3Eib1GV3gB09CIOuGDhrJFcUPA\\\\\\\/AQsN6qnGCtC9QJduT9CBRAPcDC+\\\\\\\/GVLVFRFwrRnNbGqwAiQmemWAZZ5PatOBTA72WAvzdpgUKGt7DkAoofqlE9upWamyu\\\\\\\/KsSxm9Hm0wKjoQSmHRsY3vUBcqR4Z8iQmyv0YDIAanTLGy7mq1oD397MuHGv6yEIBLMV5pq2YcTD9GpuRFxZjm8OJPp+NZB+T0dKJuBc\\\\\\\/yNE9c0nWbr6AIN\\\\\\\/wm138ONP0EzIWc3+OimLVYagHvSa83lgLgf4UmdpQ2w7Fxmn2PT6W9puisjfsw5zc+cF1dD5iiLX\\\\\\\/SoJ+PKL1vTe4e\\\\\\\/VueUYMh4iV2lIdp0MLOnE1btygsZlpy2izMgCse\\\\\\\/k8CcPVw4K\\\\\\\/IygNW4t\\\\\\\/aD6nnTZRfu6W0gXoj+C19qhZERK9cVjUJ94rIu87MK4yyLvlitzokXVhsnwdqmoHHdovZKuABa5JJWNZWL46C2vmbFdlfouJ42ttH9TY6WKmu9gHFN3ULiPUltfpIp24LUw9wPJOKzgKR63S7CA6ZoVle9pNKPN6VR\\\\\\\/ebKA2+x20woUfkYy9babzf2mdrs9sgvHgLF7R7yTZ7F8CfcgWibV\\\\\\\/CGUMVvKs\\\\\\\/7cOpxP3tU+tHgvcqAhXU5X0It3hgQMdlfjbZxQm\\\\\\\/rB4OP7sJ4diYf8Ii1vx24YCzPi9JRX45WqOofjTAUIOei7hWDrp0xnNMbaSYLdHMLJYGKvxqX3IKVHSbywMYYL+B\\\\\\\/SQRu8v0eRm2pdX1ShjyS8doxBLEcdvjlPV7zKO1Xf7i9z0lUbKqpFiRhNFSZsebfDqTm6YHYPnFX3OVBfBXU1q2ydTdovc\\\\\\\/r706fHqpgG2h\\\\\\\/Dez\\\\\\\/1DSYdLYrjsaMeFyl4fVuDJzEsTboRYij\\\\\\\/88u6m9AFrDRJXcmwR8N2lGRd+Sg70jox62WCv5FbXMzGFZf9mXmK8YT4+QxmhKAPJrOaseyxVfZP209mFODM\\\\\\\/sz0uTPQnPZcYfZ0u72p3jV2\\\\\\\/6xZoO2f+vrWy0sy0F6SVj2ylK1vf4kNlfY6sL6huVhkGkgaxy1gwyCcvn3oGGEwExT98QDls\\\\\\\/oi9EzZH2g\\\\\\\/+MRvzejDL61qO7nvpxmTl1peyjxSX7PCBY9GYPgUu6BUXnZ1RRQLJduoUF3h70pu1MozTS8aRVIo\\\\\\\/nrgqIz6tOc+Q3b67UIoKl2eNLq76WZ9XWYnW6tKoyO09sPs3TSIwm5sTr8uiPYe8PqODwwVa+qd9lmBJyIZEkbaYDKkPkiYUcf1Xp18xtIRIdwXkReD7FEORbr4ShdHsKDgeEFavJ40WKI4y6CVlg2o6O+Jz8SkxyB3U9M1H5r0abzo30GnSzxSujRrm\\\\\\\/KSuLNthqX4Lci8PtV\\\\\\\/HEhJkWHhTc2\\\\\\\/HPYVmOA8jygY9uQXPnizApbyzGaTSnmtx9tur97V090EM2t3+5ssfoJ0jP5c62GQ64xvFnsbSIw\\\\\\\/WUORZZtYBtP37tQouGpsGDPUCJKB27BRolbM099\\\\\\\/YXOlfsEX2a5k1d0360myDHNBmID8lIwpNCQ7Fj\\\\\\\/pt12bCP4f4o63iXA04Q5+RInys8gPtfcZc6nTtKu2UxQu6wBcg1SvkJuXaum9qHCvt0gOZwernqhug6IiRaYnt72A439sSq4CgH9dlbyalme+R7ryfFpF0eEqyYEAspfVypmow1yJisoCTF+a5lh6RZ9QNC2ScXFua1kF3evEGPYMNza2zFvuFJUfd4vMS7FcSf3X53N2j4uILOZ4FwQAoKO5qAb29YYBnYS\\\\\\\/L0Jk21o2F65d6pxtMRYgdodVLjnr7aJAVdjXCcS86dEDwHs5dqifjK\\\\\\\/vJBnKLivXWUa46alrRMzMCr4z0qeV4eun4KF5ROU59+KJWhsMaRJQ+cWJVRLLOym7MMRqKhQoy4987amLDXvYR3aB0fIAeV9Ront+Ox3G3eV+vyRgkBHsiyANhLyOY+sldEZ3qHCzBXKQdIdP9E1G2JxkDSIDaivlSTOh3M7PpzHqXT6\\\\\\\/6Solj+7WzaC7vA6bwK1k6MndmX+KHTf4KCPOUVLMeW26ECLVWZXfpDQfNSgiPyJTTBBGvqx+\\\\\\\/aiJJb0bIMN62NkyLCWTsrSq2GMefzTtzqdXz4g9Lj732QyUfgLKvasZvuvCwjVKt6N5u5UK3uvw0EV0d5Or9sVd2eKNisqI+HAcIb4zwdwOLrYObkOlm3N8wlzSI7vaZ4B5ONOSGqSwZ9nyk9lTSCNvcxPT\\\\\\\/YRhx1CXhuoa2PM3QgVfvTj23rzbF88Ob4fYYqk5Fx5PyT07jb+Bdb08890DpyuEE84SxMu5ZSxOhRS0mN3+MHJSpbbPe88t2q3ZgOg7tbvcHFaedwlVypKSPg8nCpFQALBMlj6JE2Z+xZc+9indmdD4SyaXW2SkDJyrNUff4OLtORIczZQEHQ5wCNtRDoJallXxcCYsrcMfL67ZkcxFbvdcKjn9ed7pdug4+xiAF2S6zcERmf9e+drG7oYpz6K8ZPpQOEBbAFCZTkdd2J+CWVm0nHNUqY6GjZjlewqnGjhhOKJ0a\\\\\\\/YHTvyF2ZDt9Z8bDmVDMJwEW8zcM+7kgNDnVOqxcbTxF00eQRF0fBWvpp4u2pIxtVLahg6MLr8OKder+z1U\\\\\\\/GbenBtXef5AcwaLaN9MTKS+LpDZ9YoK4RnxLbFwQ1gpUtI0DouWKqJ+nmdvmNDeT1s7mdrte4ABbdTKqEhrlnqvFK1sbAG6dAvtW54j\\\\\\\/O6tesyT1HlX0PPhTvlDSWnW8NMRZXnpUP66FZtRaBg+n1SSZNsRt6Wt7XMGClsgiLYogLHaIstbCo7t5y2Hqq7L7zCCw6mLCxQIhQ5uVAdyjnrJq32pk9dZZjWVLLnov0XnvjUZGkEVlZjh3BMpdjqTW+YitMsLHTSqiD6RWj4QAubfMlol7UX3QteVOnYIkQ0RWnOEvNVx+U+NCsgkRjC3FtUQkJNqs5JNAWt5uABSsQVht7cLkQZvcSfFF80jYpZkcqwSl0Lqz8Y6Yi8uo4f1ohcjQChavZz7pFt7GNJgt1QsNa3QefVRa74mISfG62rj8EvnK\\\\\\\/qkQHaLMN7ClcewmEHL7\\\\\\\/3o5f7Orm5Neorg2DTvW0qL+1GCSSbrdwXyXbm0sWlYUo+Fd\\\\\\\/DTizrOpa2iHYIgAvGrXH\\\\\\\/TkMGFJRuj0ybsgPoj8e0Gg2RDUo6kaIsySi6EFkYbmPBgqW7pReE04tW7V37vTKq5GTYF2jX7dYHmwBYxlXT7Sq8Jds9oj9WKuQq1AeI8i77s\\\\\\\/mwvGAs53Gra4OqU1PoTJ43efksb7CI5jC3sduEHMlzdV5qVxLj8LxOzIHntbkxMRdmOkr0N+FrEL1nOYHIkkEG7IJpofdLl0CF0Aglyl+sfHcYjavZsQHKPx4S1+LogcJl1aGI+dVpVQvBBpGiS5PorpwKQrdCXU1hIhwSLoSu9knKy9cyuvw8CvJdovKjEVZ7W5g8b6au+i4faPoXUjzVrpDNsdeUThpVCfdYjrWIf8LppD0iGVYGvd5S0zbaM\\\\\\\/XgjPbF5T\\\\\\\/clPlo0DSg5TNsbyd2oEyrw1n+aU6k2UHSNUt9pJDXYO+1pgdfY+UIgr4oujy9VvYWkcm0QgH2ys8N1T2aaXeHj3HuWXHAoNGNp\\\\\\\/DNGT01HCnpKNcI9ryAvycJBHZ8H1KK65uulHBhChX1M5RDoZbNAXWfNqnsKsj6MBfdKx3w+boo0JikKxo6zUmSX\\\\\\\/pnlkokIfkFo3g8gsUBlDDPuC6wE9msahJ6dB0m0ze1puyUvTY3IkrZT6dsTJ2l6WNfYO540OvKep9JHAj\\\\\\\/SpQILUeEgXdi0LmEX0xOTrdRtmFC1TuphFx\\\\\\\/FyUVGSL4h6g4k\\\\\\\/UuuI7JkrZt1rC9UepeHj\\\\\\\/GLcLjx9ngPTIIQqAR086b2xBUCJb\\\\\\\/7tnf3ZxvOly4dCVb2UP6zi2eqpDjVayvzrl6yp5Huf8HXRj6yAzaPK+26oDZOqIkV6BqwAwzpMWZy+hwlCawesK1lgI9IElm6kLufk3pBUiL+gdz8prfOBgwM\\\\\\\/j9h2sdBgGFpOYO+KyyC0VQeu7aUPN9qPcID5vXUHAv\\\\\\\/U0trZiNnkuf\\\\\\\/DClWYtvehTZf2TJkApjcs2tLIwhaPpPg04pyH6+c6LekU8Qu9zyuj9sjfBCCl6HdFcI1\\\\\\\/hvBAKG7tQHT0Y+OmeSF71TlZD3f2MkRGThDpSdw9KdeVJOWNERXZPEWMTgkXi1q1oVp40DBT+WdYo02laCKNVAQbX+KfBnvQJ82t0i8PimKGpGak0kZSVYGg\\\\\\\/E+zfASGm3xVKrhoRXlaj+syEf5tV\\\\\\\/YdiNnyhKPmZ4m9mdZvNBXbTijwYd2YR0JXQTk+FFg19+VHyHAlwVTFF26pQoFy6nw3+hVR2LjCsRUYgFBDS9\\\\\\\/u1qqFtCdJ\\\\\\\/GzGhufRU6tn4xLM+DcdOszTa0z0MT8je8+\\\\\\\/URnvf4NfrR\\\\\\\/JGYHcpHxhFCGDgaqel8OON\\\\\\\/DJ5Mpncn8Rchd+VXY785RH1NcYwHIHK0y1uJTgBfvOnTx8ZpAPePlaMwzWzP3\\\\\\\/IwIB5Cm4KWQA\\\\\\\/9faK9ju8Nk1vuAWsJJjw4zg89BJFsnV1ybLQ5\\\\\\\/0oJZBEuebochl8FZZqXePj65ROIYtMEiBGQOgloCF3gcSqbZjAxC+bYWKftVFJ6RWKTy+PvQuRR1xswYqs6C4UnGtafrs0ET8KABUjg73fWMaBe4+w4\\\\\\\/mPFelG+yLxsxI\\\\\\\/eSZM1GYvVSm4oAEqFYHylVG6A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Yfzzp6qNNCm+dDtjS\\\\\\\/Vtkg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223e955039fb9df15c68d7d02daa54d055b5678cdcabecad33459a1bee364f5056\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LsYioT9sU8btd+cSugcna7P4xCi+buNFauWi59o+j9fOZORYVwatT4gyE1WdRWq3GUE3gKgdaUSzl0Px0KKdHceYUZ\\\\\\\/nlWFyLzPVK2r60bmO29XG2pqHuS3w2FWXQtYkcoX\\\\\\\/6oSa56HjXnmwvVL7TDB1GQ\\\\\\\/HlwVvZEFAGb7bfSlVYY9U1Gzxs1ajPnmHFUkSz7Pf+odiwZHhAkHucaEBe\\\\\\\/ujNMmP1CAPzjJ4N85ItrvS3JLR4qPoDXnY6tYJXG9DqY88hM9zRs65nxnj3P+2R2Ebpb0HzqeSO8JokQ9GyCVwupXm9HmsxYLyxkACE2mj6J2pQ19lafrkTZsoqXjdc5nZN6Jen9Qky9iL9roDVuyV8hBwrZtUr\\\\\\\/GXs5MaUEbEGeXDlJxpLeZBQo6MeWmaDLsDHYN4FGYQlKGim8V0SpmTgVqz7Ib\\\\\\\/iaMLhitIE3jg7MOjjNqoGacbZOHjm\\\\\\\/jonNBSkopHtwnsbFHpHrXXRklfPznRF7kZxBdVsk+bkxAxq9HVmbcuhWmbAw6me7jb9JsD3jbM7oHJU0RRQEgCktrrLkKsZlvTSRVqxajf2d7x+g4wIirRJZaRQL0pCpHL3CwkNPckKxDyq\\\\\\\/D7GS0H4VALmTZOKgya6x41j9WodB7vbEFStjxIV4dJ8JSCPcccH8ezNkRQ5Uj4r6oXCUJlgtEkWNXsbe9C3DU\\\\\\\/racqJDaD4rsls1mLXUO\\\\\\\/0yCJY8xvTbwmq4fOS3OMtwW4Lr+5K4jl4ns9JZal2X9D2bjIz58L32vEZJs1MbzOkzxE4PtAKnFf9KuXRs71XomFA+tyVU3wuSjD0Pd9OJJMOmK5UG93bIRQBjgYImcrEKfanKd45lyV75oYdCktpxx1RtJR7QNxeUVlT6FtBvhlDIMU44XDMSt7Y998VOq20p30\\\\\\\/3gfIGCNLoy5y0eESi1hQdXEW0LzNeUkdl\\\\\\\/APEZxQe6q5cfCOL+mvleI1+gnw1Z021fLN26MYxURbIfHknvPJg95m71UINCeyUpPnwYFHqzRUffMhzlt3XTNats4aBl6yxoXMFDO4I8R2ZoeBj9ezRopFMXrwFjJondU07xRgXy+W48GqTpCRbeVdeBXoXTmLLTXvUR92xhnP5DXZoVu\\\\\\\/R40cF+ubGRK1OOyxkXPbqM+aRcewyrQmS+IHt6b5DLEawArUKBYhdMl+6pe5tSWESOS9+B6MzKVgAOAtdIGOm\\\\\\\/v7hzEgta1rYkEymZKCkdrl+2gxABKBnzqlFsViAFtxJyXzpDS1tRGq7aViGG\\\\\\\/tx9R3i78CGn\\\\\\\/CDAyzoQ5Efboik1INanFnaNVJKhq4Y3gDk8rZnNt1TE0RLPKJAbcsfSVcTChPrKYZp\\\\\\\/LiSvPb7Ilob81ZaWOt3hSXd1I53C0X\\\\\\\/IDZ+Gb1guhPG0MtbxLqXNNW5wu34muNFspvyJuH\\\\\\\/+wvE3rFmo9etae69jdGBkTi+pQaTtgSm8zHcIE6DnBEgelJV+0tbT131u7T1GK7NrUbUhKfC7dvxY8WKOoln47goq1rQEY3FwCeKuRMJBCiV1cZpFruR4j73YH5wYUt6WVZZ+JPygmztEGxq+L2bH+hnc0cMRt0VUDRlKmwZs1Gx\\\\\\\/AL5cA4Bq8J6UE1Dx+F7GzsKPr07O4fwTUOkRdEy2za1FSXL9OaajTOc9fF+A7VMC9liomSG7AgXW1261MypXYnfObri9WAMsrN2G7BQhYVVHwixSu8nmoDin0zWt1MZyiXUpAHlbz3ihtKOxyPJtqGOLaRQkXMJzbr+j58y+hXp6eoCOrDXnWE4owCf25JcfGw0OVpOQdJL\\\\\\\/as683Dfwn+bHMmimI2PJb\\\\\\\/GrGYlfMBLbV2QtVOEOrpWXy\\\\\\\/kLz2tbomNMrMeQNToEgO0p2RwJ4xvJ5PyrYJVEYdDDEwv9JSZQa7fgGCvKL6uNQRIWHqDHETkKtfK5+IaB0nEm+FBumwmlhf58mDg97XVEtZZWB2jtwvn8sWiVqUFR0j20UgqJ0xU9UNsk8SJUDVqVvl5m\\\\\\\/YFD1hw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zOD5KwyLWH+nzxucdS3vgg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220b7c28f276f2eb625202936201d71dc22a98c9058fcbd5059193a9a28c784af0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485312 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2161" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jN5EE_wf-Cayy7QpuTs_lw0AgNCdMaAqNikPBHl-1y4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2161" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-S8-iE4TL3tAmTkOOAPMSgUCIqSx5MgYHJ5lXVKvNnO4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2151" title="Two Loci Contribute Epistastically to Heterospecific Pollen Rejection, a Postmating Isolating Barrier Between Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2171" title="A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485312" id="node485312" data-pisa="ggg;7/7/2161" data-pisa-master="ggg;g3.117.042036" data-apath="/ggg/7/7/2161.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Heather Manching</span>, <span class="highwire-citation-author" data-delta="1">Subhajit Sengupta</span>, <span class="highwire-citation-author" data-delta="2">Keith R. Hopper</span>, <span class="highwire-citation-author" data-delta="3">Shawn W. Polson</span>, <span class="highwire-citation-author" data-delta="4">Yuan Ji</span> and <span class="highwire-citation-author" data-delta="5">Randall J. Wisser</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2161-2170;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042036 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.042036" data-atom-uri="/ggg/7/7/2161.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485312" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Heather Manching </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware 19716</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heather%2BManching%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manching%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeather%2BManching%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Subhajit Sengupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Program of Computational Genomics and Medicine, North Shore University Health System, Evanston, Illinois 60201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Subhajit%2BSengupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sengupta%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASubhajit%2BSengupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Keith R. Hopper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Beneficial Insect Introductions Research Unit, United States Department of Agriculture, Agricultural Research Service, Newark, Delaware 19713</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keith%2BR.%2BHopper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hopper%20KR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeith%2BR.%2BHopper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shawn W. Polson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Bioinformatics and Computational Biology, Delaware Biotechnology Institute, University of Delaware, Newark, Delaware 19711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BW.%2BPolson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Polson%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShawn%2BW.%2BPolson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yuan Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Program of Computational Genomics and Medicine, North Shore University Health System, Evanston, Illinois 60201</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Health Studies, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuan%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuan%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Randall J. Wisser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware 19716</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randall%2BJ.%2BWisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wisser%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARandall%2BJ.%2BWisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/7/2161" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485312" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485312/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2161.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485312" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485312/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2161.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485312" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485312/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2161.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485312" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485312/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/7/2161.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485312" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485312/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">High-throughput sequencing (HTS) of reduced representation genomic libraries has ushered in an era of genotyping-by-sequencing (GBS), where genome-wide genotype data can be obtained for nearly any species. However, there remains a need for imputation-free GBS methods for genotyping large samples taken from heterogeneous populations of heterozygous individuals. This requires that a number of issues encountered with GBS be considered, including the sequencing of nonoverlapping sets of loci across multiple GBS libraries, a common missing data problem that results in low call rates for markers per individual, and a tendency for applicability only in inbred line samples with sufficient linkage disequilibrium for accurate imputation. We addressed these issues while developing and validating a new, comprehensive platform for GBS. This study supports the notion that GBS can be tailored to particular aims, and using <em>Zea mays</em> our results indicate that large samples of unknown pedigree can be genotyped to obtain complete and accurate GBS data. Optimizing size selection to sequence a high proportion of shared loci among individuals in different libraries and using simple <em>in silico</em> filters, a GBS procedure was established that produces high call rates per marker (&gt;85%) with accuracy exceeding 99.4%. Furthermore, by capitalizing on the sequence-read structure of GBS data (stacks of reads), a new tool for resolving local haplotypes and scoring phased genotypes was developed, a feature that is not available in many GBS pipelines. Using local haplotypes reduces the marker dimensionality of the genotype matrix while increasing the informativeness of the data. Phased GBS in maize also revealed the existence of reproducibly inaccurate (apparent accuracy) genotypes that were due to divergent copy number variants (CNVs) unobservable in the underlying single nucleotide polymorphism (SNP) data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gbs" class="hw-term hw-article-keyword hw-article-keyword-gbs" rel="nofollow">GBS</a></li><li class="kwd"><a href="/keyword/haplotype" class="hw-term hw-article-keyword hw-article-keyword-haplotype" rel="nofollow">haplotype</a></li><li class="kwd"><a href="/keyword/phasing" class="hw-term hw-article-keyword hw-article-keyword-phasing" rel="nofollow">phasing</a></li><li class="kwd"><a href="/keyword/copy-number-variant" class="hw-term hw-article-keyword hw-article-keyword-copy-number-variant" rel="nofollow">copy number variant</a></li><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">imputation</a></li><li class="kwd"><a href="/keyword/maize" class="hw-term hw-article-keyword hw-article-keyword-maize" rel="nofollow">maize</a></li></ul><p id="p-3">Genome-wide genotyping of population samples is fundamental to a range of studies in genetics and genomics, and GBS of multiplexed HTS libraries has emerged as a cost-effective strategy for obtaining this information. GBS of multiple samples relies on a reduced representation sequencing strategy that restricts sequencing across what is hoped to be a common subspace of the genome in different samples. A prevalent technique used for this involves the ligation of barcoded adapters to DNA digested with restriction enzymes, followed by sequencing of fragments within a restricted size range. In principle, this leads to stacks of sequences anchored at the restriction cut sites across the genomes of different individuals. Following initial publications on restriction enzyme-mediated GBS (<em>e.g.</em>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andolfatto <em>et al.</em> 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elshire <em>et al.</em> 2011</a>), the approach has continued to be extended and optimized, such as fine-tuning the number of loci sequenced by using different restriction enzymes and size selection windows (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Peterson <em>et al.</em> 2012</a>) or selective primers (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sonah <em>et al.</em> 2013</a>), minimizing the front-end cost by using enzymes that create blunt-end fragments for universal adapters (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heffelfinger <em>et al.</em> 2014</a>), and increasing the scalability for large samples by including a sequence capture step (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ali <em>et al.</em> 2016</a>). Methods and applications of GBS continue to develop at a rapid rate.</p><p id="p-4">With the mass of data produced by HTS, even low sequencing error rates can lead to an abundance of inaccurate genotype calls. Moreover, errors accumulate along steps in the GBS pipeline from library preparation to data processing. As a reflection of the amount of noise in GBS data, it is not unusual to find publications (as well as our own experience) where as much as ≈50% of the raw HTS data are filtered after mapping (<em>e.g.</em>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beissinger <em>et al.</em> 2013</a>) and as much as ≈95% of the discovered variants are filtered after variant calling (<em>e.g.</em>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hyma <em>et al.</em> 2015</a>). Typically, statistically or biologically informed criteria for filtering are used to enrich for more accurate GBS data, or a model is fitted to the data to remove genotypes that do not conform to expectations (<em>e.g.</em>, tests for independent assortment and cosegregation in genetic mapping populations and Hardy–Weinberg equilibrium in other population samples). However, estimates of genotyping accuracy for a given protocol, pipeline, and application of GBS (under all of its flavors: <em>e.g.</em>, RAD, GBS, ddRAD, and Rapture, etc.) are rarely reported. One recent study compared different bioinformatic pipelines and found that genotyping accuracy for one GBS dataset ranged from 76 to 99%, depending on the bioinformatic pipeline used to score genotypes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Torkamaneh <em>et al.</em> 2016</a>). Accuracy estimates based on resequencing of GBS loci have ranged from 51% (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rocher <em>et al.</em> 2015</a>) to 99% (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Torkamaneh <em>et al.</em> 2016</a>). Given the wide range in accuracy and method- and study-specific nature of GBS, establishing controls or approaches that allow accuracy to be estimated is important when embarking on GBS studies.</p><p id="p-5">GBS tends to have a missing data problem due to two primary issues: (i) for libraries with low sequence coverage per sample, the amount of missing genotype data can be high, resulting in a low call rate per marker (as low as 10%; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fu and Peterson 2011</a>); and (ii) for large population samples where many libraries need to be sequenced, the number of shared loci in different libraries may be low, resulting in low call rates per sample. The latter issue has not been fully addressed in the literature on GBS. We are aware of one study in which the consistency of genotyping the same loci in different libraries with the same samples was assessed, and in this study, overlap of loci among libraries ranged from 12 to 98% (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">DaCosta and Sorenson 2014</a>). The results of that study suggested an impractical (costly) solution for genotyping large samples, where greater consistency would be achieved by pooling samples later in the protocol. This would require individual samples to be processed through size selection and PCR amplification independently, which increases the cost for reagents and time for library preparation. A solution to the GBS missing data problem has been imputation, whereby the genotype for a missing SNP is inferred from the state of nearby SNPs. However, imputation is not always possible and may not be sufficiently accurate, in which case both of the above issues compromise the effectiveness of GBS in studies where sample sizes exceed the multiplex depth of a single sequencing library. While the former issue may be addressed by reducing the sequence space (<em>e.g.</em>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Peterson <em>et al.</em> 2012</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Sonah <em>et al.</em> 2013</a>) or deeper sequencing, the latter issue requires techniques that provide reproducible enrichment of shared loci (<em>e.g.</em>, <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ali <em>et al.</em> 2016</a>).</p><p id="p-6">Despite the shortcomings mentioned above, GBS has been effective in several settings. There is now a growing interest in using GBS for a wider range of studies (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andrews <em>et al.</em> 2016</a>), many of which could benefit from or be advanced with phased genotype data. The standard approach when performing GBS involves scoring SNPs. However, in GBS data, locally phased haplotypes exist as stacks of reads for each locus. If these contain more than one SNP, then multi-nucleotide polymorphisms (MNPs) can be genotyped. The use of MNPs has not been widely exploited in GBS due to the lack of tools for scoring phased genotypes. The software STACKS (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Catchen <em>et al.</em> 2013</a>) and Haplotag (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tinker <em>et al.</em> 2016</a>) do have functions embedded within their pipelines for extracting MNP haplotypes from GBS data, but the pipeline-dependency of these functions limits their use more generally, and the algorithms rely on population-specific filtering criteria rather than statistical evaluation of the likelihood that each haplotype in an individual is real. Because HTS data are imperfect and the quality of sequenced bases and read mappings are quantitatively encoded, scoring MNPs at a locus within an individual is not straightforward if one considers this information relevant. A local haplotyping tool, LocHap, which uses community-standard file formats, offers a more generalized solution for phased genotyping. LocHap was developed under a probabilistic framework that uses the quality metrics of base calls and mapping results as well as sampling effects to phase MNPs in HTS data (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sengupta <em>et al.</em> 2015</a>). LocHap was designed to identify distinct haplotypes in heterogeneous populations of cells (somatic mosaicism) irrespective of homology, such that the outputted haplotypes at a single locus can vary in length and are not necessarily alignable within and across individuals. Consequently, LocHap does not produce data in a format that is compatible with population or quantitative genetic studies. In our study, LocHap was extended to perform phased genotyping with GBS data on individual samples, which we call LocHap-GBS.</p><p id="p-7">Phased genotyping is useful for various applications in genetics and genomics. Typing MNPs can help distinguish more than two alleles at a locus, providing greater information content for studying genetic diversity (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lu <em>et al.</em> 2011</a>). Phased genotypes can facilitate imputation in multi-parental populations where SNPs would otherwise conflate ancestral alleles (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davies <em>et al.</em> 2016</a>). Haplotype data can also increase the accuracy of estimated breeding values, thereby increasing the efficiency of plant and animal breeding (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ferdosi <em>et al.</em> 2016</a>). In our study, while validating LocHap-GBS, we found that phased genotyping can also uncover copy number polymorphisms (CNPs).</p><p id="p-8">The aims of this study were to: (i) establish a standardized and empirically optimized flex-plex GBS protocol with an accompanying informatics pipeline for genotyping; (ii) evaluate the accuracy and potential applicability of this procedure for genotyping large samples from heterogeneous populations of heterozygous individuals; and (iii) extend the use of GBS for phased genotyping. We used maize, which is an agriculturally relevant species with an ≈2.3 Gb completed reference genome sequence, but where genomic analysis is challenged by large amounts of repetitive sequence. Genetic trios of pairs of inbred parental lines and their F<sub>1</sub> progeny were used in assessing the accuracy of GBS. We tested whether simple <em>in silico</em> filters could facilitate highly accurate scoring of genotypes irrespective of zygosity and knowledge of parentage. In addition to developing and validating a method for phased-GBS, we also examined how SNP and MNP data affect inference on the relatedness among a set of inbred lines used by the maize genetics research community.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Study samples</h3><p id="p-9">Detailed information on the samples used in this study is in Supplementary Material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/FileS1.pdf">File S1</a>). Briefly, the samples included a synthetic population created from seven tropical inbred lines (used to evaluate the consistency of sequenced loci in different GBS libraries), sets of trios or pairs of inbred lines and their corresponding F<sub>1</sub> hybrids (used to evaluate the repeatability and accuracy of GBS), an F<sub>2</sub> population (used to examine the transmission of phased MNPs, along with the corresponding parental trio), and parental inbred lines of a maize nested association mapping (NAM) population (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">McMullen <em>et al.</em> 2009</a>; used to examine phased genotyping for analysis of genetic diversity).</p></div><div id="sec-3" class="subsection"><h3>GBS</h3><p id="p-10">We describe the design, protocol, and associated software for GBS based on a double digestion technique similar to protocols by <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Poland <em>et al.</em> (2012)</a> and <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Peterson <em>et al.</em> (2012)</a>. A specific interest of ours was to develop a GBS design for genotyping heterozygous samples of potentially unknown parentage with little missing data for large sample studies. The method was optimized, validated, and tested for various applications.</p><p id="p-11">Detailed protocols and other relevant information are provided in Supplemental Material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/FileS2.xlsx">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/FileS3.xlsx">File S3</a>). Briefly, two separate but related restriction-associated sequence polymorphism (RASP) adapter designs were developed: (i) RASP-1.0 was based on Illumina’s “genomic DNA” adapters (Illumina, Inc., San Diego, CA) modified with appropriate overhang sequences for ligation and a six-nucleotide inline barcode for 48-plex genotyping (RASP-1.0 and RASP-1.1 adapter oligonucleotides and primers; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/FileS3.xlsx">File S3</a>); (ii) RASP-2.0 was based on Illumina’s TruSeq adapters modified with appropriate sequence overhangs for ligation. RASP-2.0 uses variable length inline barcodes (5–10 nucleotides) for 48-plex genotyping (barcode sequences were designed using <a href="http://www.deenabio.com/gbs-adapters">http://www.deenabio.com/gbs-adapters</a>) along with standard TruSeq indices. The inline barcoded adapters and TruSeq barcoded adapters can be combined to construct plexes in multiples of 48 (RASP-2.0 adapter oligos and primers: <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/FileS3.xlsx">File S3</a>).</p><p id="p-12">The protocol used for GBS was improved over the course of our study, including changes to the adapter sets, the size selection method, and number of reactions used for PCR (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/TableS1.xlsx">Table S1</a>). The general protocol was as follows. Purified and normalized DNA (200 ng) was digested using two restriction endonucleases, <em>Ngo</em>MIV and <em>Csp</em>6I, for 30 min at 37°. The <em>Ngo</em>MIV/<em>Csp</em>6I enzyme pair was chosen because it provided the highest read output and the lowest variation between samples based on comparisons between four different enzyme pairs (<em>Ngo</em>MIV/<em>Csp</em>6I, <em>Ngo</em>MIV/<em>Mse</em>I, <em>Pst</em>I/<em>Csp</em>6I, and <em>Pst</em>I/<em>Mse</em>I; data not shown). Adapters were ligated with T4 DNA ligase using temperature-cycle ligation [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lund <em>et al.</em> 1996</a>; 300 cycles of 30 sec at 10° and 30 sec at 30°; this was determined to improve ligation efficiency by qPCR (data not shown)], followed by heat inactivation of the ligase at 65° for 30 min. An equal volume of each ligate was pooled and then purified using either AMPure (A63880; Beckman Coulter, Inc., CA) or SPRIselect (B23317; Beckman Coulter, Inc.) beads following the manufacturer’s recommended protocol for standard clean-up (exclusion of fragments &lt;100 bp). For all libraries, prior to PCR amplification, size selection was performed on the pooled ligate using a BluePippin (Sage Science, MA). Size-selected samples were PCR amplified with Phusion High Fidelity Master Mix (M0531S; NEB, Inc., MA) using the universal primer sequences from Illumina’s genomic DNA or TruSeq sample kit. To reduce PCR bias, a minimum of eight separate PCR reactions were performed on the size-selected template and the products were pooled. AMPure or SPRI beads were used to eliminate excess primers and biproducts (exclusion of fragments &lt;100 bp). The range and peak of Bioanalyzer size fragment profiles were analyzed for each library before and after PCR to determine the consistency of the size profiles. Due to variation in fragment length profiles found following pre-PCR size selection, a post-PCR size selection was performed on some of the libraries, followed by Bioanalyzer analyses to confirm size profiles. This turned out to be critical for genotyping separate libraries, and we have since included this change in our standard protocol. Multiplex libraries were quantified using a Quant-iT PicoGreen dsDNA assay kit (P7589; Thermo Fisher Scientific, Inc., MA) and sequenced (1 × 101 cycles) on an Illumina HiSequation 2500 at the Delaware Biotechnology Institute.</p></div><div id="sec-4" class="subsection"><h3>GBS data processing</h3><div id="sec-5" class="subsection"><h4>Computational pipeline:</h4><p id="p-13">Sequences were processed using a custom reduced representation “RedRep” computational pipeline with the following basic steps: (1) sequence splitting by barcode; (2) raw sequence quality control; (3) reference genome mapping; and (4) variant calling (SNPs). Briefly, sequences are deconvoluted by barcode using custom logic and the FASTX-Toolkit (<a href="http://hannonlab.cshl.edu/fastx_toolkit">http://hannonlab.cshl.edu/fastx_toolkit</a>). Novel scripts and the CutAdapt package (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Martin 2011</a>) are then used to remove adapters, trim low quality read ends, and filter out sequences that do not meet minimum length/quality standards or lack the expected restriction site and adapter sequence expectations. Quality reads are mapped to the reference genome using the BWA-MEM algorithm (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li 2013</a>) and SNPs are identified using the multi-sample discovery mode of the GATK Unified Genotyper (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">McKenna <em>et al.</em> 2010</a>). The scripts for the pipeline and documentation are available under the open source MIT license at <a href="https://github.com/UD-CBCB/RedRep">https://github.com/UD-CBCB/RedRep</a>. Version 2.0 scripts committed to the repository on December 7, 2016 were used for this manuscript.</p></div><div id="sec-6" class="subsection"><h4>Post-vcf filtering:</h4><p id="p-14">All 64–384 bp loci flanked by <em>Ngo</em>MIV (G↓CCGGC) and <em>Csp</em>6I (G↓TAC) recognition sites in the B73 v2 reference genome were identified by <em>in silico</em> digestion. BLASTn was used to perform genome-wide searches for each <em>in silico</em>-digested sequence starting from the <em>Csp</em>6I recognition site (starting point for sequence reads) up to a maximum of 96 nucleotides (longest quality trimmed read length of our actual data). Sequences that were &gt;4% distant from all other loci in the reference genome were flagged as “4PD” loci, and SNPs that occurred within these loci were maintained while the remaining SNPs were filtered. This excludes the possibility for scoring variants at loci within the reference genome that differ from one another by &lt;4 nucleotides in 100 bp. In addition, for each individual and at each SNP site, if the genotype was not based on a minimum depth of read coverage, it was set to missing: 3× read depth for scoring inbred samples and 12× read depth for scoring heterozygous samples. Samples were then removed that had genotype data for &lt; 25% of the loci for 48-plex libraries and &lt;10% for 192-plex libraries; these included problematic samples and negative controls. Finally, for each SNP, an 85% call rate threshold was used, and SNPs with more than two nucleotide variants were removed.</p></div><div id="sec-7" class="subsection"><h4>In silico characterization of GBS loci:</h4><p id="p-15">The <em>in silico</em>-digested loci, along with the SNP sites scored across all samples, were summarized according their distribution across the B73 v2 reference genome. Using the closest-features program of BEDOPS v2.4.15 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Neph <em>et al.</em> 2012</a>), genic and intergenic associations of the digestion loci and SNPs were determined using gene structures in release-5b (filtered gene set; <a href="http://ftp.maizesequence.org/release-5b/">http://ftp.maizesequence.org/release-5b/</a>).</p></div></div><div id="sec-8" class="subsection"><h3>Assessing accuracy of GBS</h3><p id="p-16">A set of parent–hybrid trios were used to assess scoring accuracy: (i) CML373, CML341, and CML373 × CML341; (ii) CML341, CML277, and CML341 × CML277; and (iii) Tzi9, CML258, and Tzi9 × CML258. One trio involving Tzi8 was excluded from accuracy assessment because of excess heterogeneity (12.8%) and relatively high residual heterozygosity (1.3%), as determined in preliminary work using the MaizeSNP50 BeadChip (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ganal <em>et al.</em> 2011</a>). The repeatability of genotyping was estimated from replicate DNA samples of each member of the trio processed in the same library (except for the hybrid Tzi9 × CML258 because of quality issues with the replicate). Using loci with complete and consistent calls (<em>i.e.</em>, between replicated DNA samples), genotyping accuracy was measured as the proportion of loci with the expected genotype in the hybrid given the genotype of the parents. Because GBS was performed on only one plant for each member of a trio, loci that were heterozygous in either parental line were excluded when estimating genotyping accuracy (according to prior data of ours based on the MaizeSNP50 chip, residual heterozygosity and heterogeneity for the parental lines was &lt;0.8%). Furthermore, because the one F<sub>1</sub> plant genotyped was taken from bulked seed of progeny from multiple crosses between the parental lines (such that the genotyped parents may not provide the exact expectation for the specific F<sub>1</sub> plant that was genotyped), estimates of accuracy are expected to be slightly downward biased.</p></div><div id="sec-9" class="subsection"><h3>Genotyping of local haplotypes</h3><p id="p-17">Because GBS can resolve multiple variants present within each position-specific stack of sequence reads, phased haplotypes can be extracted. A local haplotyping program called LocHap (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Sengupta <em>et al.</em> 2015</a>) was extended for use with GBS data, which we refer to as LocHap-GBS (<a href="http://compgenome.org/lochap/GBS/">http://compgenome.org/lochap/GBS/</a>). <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> depicts the workflow for LocHap-GBS, which requires four input files: the standard <em>bam</em>, <em>bai</em>, and <em>vcf</em> files, plus a user-specified <em>bed</em> file containing intervals where haplotypes should be searched. To develop LocHap-GBS, a collection of modules were created that provide the following functionalities: (i) a file parser that defines intervals in which to search for haplotypes, (ii) automation for running multiple samples and merging samples into a single output file, (iii) outputting of haplotypes that include bases even if they are homozygous in an individual, and (iv) additional flexibility in the format of output (either haplotype calling format [<em>hcf</em>] or <em>bed</em>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phased GBS. A RedRep pipeline is used for SNP typing. The figure shows the basic flow of RedRep, which begins with a fastq data file with barcoded sequences. For QC, the &#x201C;meta&#x201D; file contains metadata used for demultiplexing into sample-specific fastq files. A reference genome sequence file is used for mapping (refmap) and variant calling (SNPcall). LocHap-GBS is run by editing a generate.py file specifying the location of the bam files, the filtered vcf file, and a bed file of window coordinates to search for haplotypes. A LocHap-GBS run file and windows file are automatically generated. Windows are currently split into subwindows with a maximum of three heterozygous sites within any one individual in the vcf file. This situation is depicted for reads across a window that has been delineated into two subwindows where phasing is performed. The dashed connecting line between reads indicates that a contiguous sequence with five SNPs was split into two subwindows. Black-filled bars represent the reference sequence and yellow squares represent SNVs. Given stacks of reads across each subwindow, LocHap-GBS uses a probabilistic model to identify haplotypes in the presence of sequence errors (depicted as one-off instances in the stacks of reads). An hcf file is created for each sample, which is then merged into a combined hcf file for downstream analysis. GBS, genotyping-by-sequencing; MNV, multi-nucleotide variant; QC, quality control; RedRep, reduced representation; SNP, single nucleotide polymorphism; SNV, single nucleotide variant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452500219" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phased GBS. A RedRep pipeline is used for SNP typing. The figure shows the basic flow of RedRep, which begins with a &lt;em&gt;fastq&lt;/em&gt; data file with barcoded sequences. For QC, the &#x201C;meta&#x201D; file contains metadata used for demultiplexing into sample-specific &lt;em&gt;fastq&lt;/em&gt; files. A reference genome sequence file is used for mapping (refmap) and variant calling (SNPcall). LocHap-GBS is run by editing a generate.py file specifying the location of the &lt;em&gt;bam&lt;/em&gt; files, the filtered &lt;em&gt;vcf&lt;/em&gt; file, and a &lt;em&gt;bed&lt;/em&gt; file of window coordinates to search for haplotypes. A LocHap-GBS run file and windows file are automatically generated. Windows are currently split into subwindows with a maximum of three heterozygous sites within any one individual in the &lt;em&gt;vcf&lt;/em&gt; file. This situation is depicted for reads across a window that has been delineated into two subwindows where phasing is performed. The dashed connecting line between reads indicates that a contiguous sequence with five SNPs was split into two subwindows. Black-filled bars represent the reference sequence and yellow squares represent SNVs. Given stacks of reads across each subwindow, LocHap-GBS uses a probabilistic model to identify haplotypes in the presence of sequence errors (depicted as one-off instances in the stacks of reads). An &lt;em&gt;hcf&lt;/em&gt; file is created for each sample, which is then merged into a combined &lt;em&gt;hcf&lt;/em&gt; file for downstream analysis. GBS, genotyping-by-sequencing; MNV, multi-nucleotide variant; QC, quality control; RedRep, reduced representation; SNP, single nucleotide polymorphism; SNV, single nucleotide variant.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2161/F1.medium.gif" width="275" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/7/2161/F1.medium.gif" width="275" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485439" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Phased GBS. A RedRep pipeline is used for SNP typing. The figure shows the basic flow of RedRep, which begins with a <em>fastq</em> data file with barcoded sequences. For QC, the “meta” file contains metadata used for demultiplexing into sample-specific <em>fastq</em> files. A reference genome sequence file is used for mapping (refmap) and variant calling (SNPcall). LocHap-GBS is run by editing a generate.py file specifying the location of the <em>bam</em> files, the filtered <em>vcf</em> file, and a <em>bed</em> file of window coordinates to search for haplotypes. A LocHap-GBS run file and windows file are automatically generated. Windows are currently split into subwindows with a maximum of three heterozygous sites within any one individual in the <em>vcf</em> file. This situation is depicted for reads across a window that has been delineated into two subwindows where phasing is performed. The dashed connecting line between reads indicates that a contiguous sequence with five SNPs was split into two subwindows. Black-filled bars represent the reference sequence and yellow squares represent SNVs. Given stacks of reads across each subwindow, LocHap-GBS uses a probabilistic model to identify haplotypes in the presence of sequence errors (depicted as one-off instances in the stacks of reads). An <em>hcf</em> file is created for each sample, which is then merged into a combined <em>hcf</em> file for downstream analysis. GBS, genotyping-by-sequencing; MNV, multi-nucleotide variant; QC, quality control; RedRep, reduced representation; SNP, single nucleotide polymorphism; SNV, single nucleotide variant.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">While developing LocHap-GBS, some additional improvements were made to LocHap that were related to dynamic memory allocation. Presently, LocHap is limited to calling haplotypes across a maximum of three heterozygous sites. LocHap-GBS uses a bed file with predefined intervals of windows in which to search for haplotypes, but these windows may contain more than three heterozygous sites in some individuals. Therefore, LocHap-GBS parses windows into subwindows that include a maximum of three heterozygous SNPs by referencing the coordinates of each window and the genotypes of SNPs in the <em>vcf</em> file (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). In this study, the intervals specified in the bed file were the 4PD loci from the <em>in silico</em> digest (only SNPs at these loci had been maintained in our <em>vcf</em> outputs, so this was the genotyping space of interest in this study).</p><p id="p-20">We define SNPs, MNPs, and CNPs as the property of a locus (loci that have variation among individuals) and single nucleotide variants (SNVs), multi-nucleotide variants (MNVs), and CNVs as the property of an individual or in reference to a specific form of the polymorphism (a diploid individual is expected to have a maximum of two SNVs or MNVs at a locus). To prepare MNP data for analysis, MNVs were split into separate columns using the df2genind function within the adegenet package (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jombart 2008</a>) in R (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">R Core Team 2016</a>). An MNV containing one missing SNV was assigned a value of “NA,” and a genotype containing a missing MNV was assigned a value of “NA.”</p></div><div id="sec-10" class="subsection"><h3>Analysis of genetic diversity</h3><p id="p-21">To examine differences in information content when using SNP <em>vs.</em> MNP data, genetic distances of the parents of the NAM population were analyzed. When analyzing the MNP data, we encountered genotypes with &gt;2 MNVs (implying a ploidy of &gt;2, which is not possible for orthologous loci in maize). After determining these were putative CNPs within the genomes of the sequenced samples relative to the reference genome sample (see <em>Results</em>), we examined the impact of including or excluding CNPs in the analysis of genetic diversity. The original MNP genotype matrix was split into three datasets where: (i) all putative CNPs were included for analysis (MNP1); (ii) all loci that had &gt; 2 MNVs were masked (MNP2); and (iii) loci that had &gt; 2 MNVs or were heterozygous in an inbred line (these are also potentially CNPs) were masked (MNP3). Pairwise genetic distance matrices based on shared allele distances (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bowcock <em>et al.</em> 1994</a>) were computed for each dataset. The distance matrices were compared based on summary statistics and the Mantel test for correlation between matrices (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mantel 1967</a>; implemented using the ade4 package in R, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dray and Dufour 2007</a>). To visualize the relationships between the lines, multi-dimensional scaling (MDS, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kruskal 1964</a>) was performed using the cmdscale function in the R package stats. Phylogenetic trees were generated using the BIONJ algorithm (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gascuel 1997</a>) with the ape package (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Paradis <em>et al.</em> 2004</a>) in R, with 1000 bootstrap replicates performed to obtain branch support probabilities. Distances between trees were compared using symmetric Robinson–Foulds distances (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Robinson and Foulds 1981</a>) calculated using the Phangorn package in R (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Schliep 2011</a>). Cladograms were plotted using FigTree 1.4.2 (<a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>) with branches transformed to equal lengths for displaying topology.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-22">Sequence data used in this study was deposited in the NCBI Short Read Archive under two Bioprojects: PRJNA385842 and PRJNA385849. The submitted data has been demultiplexed and processed through quality control using RedRep.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>GBS of NgoMIV-Csp6I loci in maize</h3><p id="p-23"><em>In silico</em> digestion of the B73 reference genome with <em>Ngo</em>MIV and <em>Csp</em>6I identified 321,927 sequences that were 64–384 bp long and flanked by the recognition site of each enzyme, of which 78,903 were 4PD loci. A 4PD locus in maize ensures that, in most cases, the distance between loci within the reference genome is greater than the average distance within a locus across genomes [on average, SNPs occur every 28 bp in maize (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tenaillon <em>et al.</em> 2001</a>)]. This was expected to minimize genotyping errors from ambiguous mapping. A lower PD threshold or a fixed 1 bp difference threshold may be appropriate and result in the identification of more SNPs, but we did not consider different thresholds in this study.</p><p id="p-24">For GBS, short-read sequence libraries typically comprise fragments that fall within a size range of 100 bp. To determine if targeted size selection could be used to optimize the recovery of fragments from nonrepetitive loci, we looked at the relative abundance of <em>Ngo</em>MIV-<em>Csp</em>6I loci classified as 4PD <em>vs.</em> repetitive (≤4PD) in 100 bp windows (sliding every 10 bp from 64 to 384 bp). The median number of 4PD loci among windows was 22,854 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/FigureS1.tiff">Figure S1</a>), but the distribution was somewhat skewed where more 4PD loci were present in 100 bp windows of smaller fragment sizes. The ratio of 4PD:repetitive loci in 100 bp windows ranged from ≈0.3 to 0.4, suggesting size selection might be used to maximize sequencing resources by avoiding repetitive sequences. However, many restriction endonucleases, including <em>Ngo</em>MIV, are sensitive to certain types of methylation, and SNPs are not expected to be uniformly distributed across the genome, such that expectations from <em>in silico</em> analyses are only a proxy for the numbers of scorable loci for a given choice of enzyme. Moreover, the fragments enriched by size selection and PCR are not a uniform sample of the underlying distribution of digested fragments and only a fraction of those loci will have sufficient read depths for scoring genotypes across samples.</p><p id="p-25">The genic space (defined here as the gene plus 5 kb flanking sequences) comprises 27% of the B73 v2 genome (genes alone comprise 8%). Unfiltered <em>Ngo</em>MIV-<em>Csp</em>6I loci from <em>in silico</em> digestion were distributed across the genic and intergenic spaces similar to that expected by chance alone (although significantly different, coverage of the intergenic space was greater than expected by only four percentage points). In total, these unfiltered loci encompass 3% of the genome and are associated with a majority of the gene space, including 69% (27,641) of all maize genes. Filtering <em>Ngo</em>MIV-<em>Csp</em>6I loci that were &lt;4% distant from at least one alternative site in the genome removed loci associated with 7472 genes. The 4PD loci used to score SNPs were enriched within the gene space by 32 percentage points (59% for 4PD loci <em>vs.</em> an expected genome distribution of 27%; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) and associated with 51% (20,169) of all genes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic distribution of NgoMIV-Csp6I loci and SNPs relative to genes in maize. (A) Nucleotide distance from each NgoMIV-Csp6I in silico-digested locus to its nearest start codon in the genome (y-axis), relative to the genomic location of the Csp6I cut site (x-axis). Two categories of percent PD are summarized: &#x201C;unique&#x201D; loci (&gt;4%; gray points) are overplotted on &#x201C;repetitive&#x201D; loci (&#x2264;4%; red points). (B) Proportional distribution of all SNPs discovered in this study with respect to genome annotation categories (&#x2248;60% fall within genes). (C) Distribution of the distance between SNPs and the nearest start site of a gene (shown: 80% of all SNPs that were located within 5 kb of a start codon). Negative values indicate SNPs upstream of the start site or 5&#x2032;-end of the gene. The distribution of in silico-digested 4PD loci is plotted as open black bars. Proportions were computed separately for SNPs and in silico-digested loci, as a function of their respective totals. chr, chromosome; nt, nucleotide; PD, percent distance; SNP, single nucleotide polymorphism" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452500219" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic distribution of &lt;em&gt;Ngo&lt;/em&gt;MIV-&lt;em&gt;Csp&lt;/em&gt;6I loci and SNPs relative to genes in maize. (A) Nucleotide distance from each &lt;em&gt;Ngo&lt;/em&gt;MIV-&lt;em&gt;Csp&lt;/em&gt;6I &lt;em&gt;in silico&lt;/em&gt;-digested locus to its nearest start codon in the genome (&lt;em&gt;y&lt;/em&gt;-axis), relative to the genomic location of the &lt;em&gt;Csp&lt;/em&gt;6I cut site (&lt;em&gt;x&lt;/em&gt;-axis). Two categories of percent PD are summarized: &#x201C;unique&#x201D; loci (&gt;4%; gray points) are overplotted on &#x201C;repetitive&#x201D; loci (&#x2264;4%; red points). (B) Proportional distribution of all SNPs discovered in this study with respect to genome annotation categories (&#x2248;60% fall within genes). (C) Distribution of the distance between SNPs and the nearest start site of a gene (shown: 80% of all SNPs that were located within 5 kb of a start codon). Negative values indicate SNPs upstream of the start site or 5&#x2032;-end of the gene. The distribution of &lt;em&gt;in silico&lt;/em&gt;-digested 4PD loci is plotted as open black bars. Proportions were computed separately for SNPs and &lt;em&gt;in silico&lt;/em&gt;-digested loci, as a function of their respective totals. chr, chromosome; nt, nucleotide; PD, percent distance; SNP, single nucleotide polymorphism&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2161/F2.medium.gif" width="440" height="140"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/7/2161/F2.medium.gif" width="440" height="140"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485549" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Genomic distribution of <em>Ngo</em>MIV-<em>Csp</em>6I loci and SNPs relative to genes in maize. (A) Nucleotide distance from each <em>Ngo</em>MIV-<em>Csp</em>6I <em>in silico</em>-digested locus to its nearest start codon in the genome (<em>y</em>-axis), relative to the genomic location of the <em>Csp</em>6I cut site (<em>x</em>-axis). Two categories of percent PD are summarized: “unique” loci (&gt;4%; gray points) are overplotted on “repetitive” loci (≤4%; red points). (B) Proportional distribution of all SNPs discovered in this study with respect to genome annotation categories (≈60% fall within genes). (C) Distribution of the distance between SNPs and the nearest start site of a gene (shown: 80% of all SNPs that were located within 5 kb of a start codon). Negative values indicate SNPs upstream of the start site or 5′-end of the gene. The distribution of <em>in silico</em>-digested 4PD loci is plotted as open black bars. Proportions were computed separately for SNPs and <em>in silico</em>-digested loci, as a function of their respective totals. chr, chromosome; nt, nucleotide; PD, percent distance; SNP, single nucleotide polymorphism</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">A total of 159,994 postfiltered SNPs (≥3× read depth at 4PD loci) were identified across all samples genotyped in this study (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/FileS4.xlsx">File S4</a>, which also includes coordinates for SNPs identified using more recent assemblies of the maize genome). These were present in 30,239 loci with a median of 4 and mean of 5 SNPs per locus. As were the loci themselves, these SNPs were more abundant within genes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>) and showed enrichment near start codons (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). The median and mean physical distance between loci was 5.7 and 67.9 kb, respectively, and the median and mean physical distance between SNPs was 14 and 12,854 bp, respectively. Previous studies, based on surveys of diverse samples of maize, have cataloged 1230 (MaizeSNP50 BeadChip; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Ganal <em>et al.</em> 2011</a>) and 64,662 (HapMap2; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chia <em>et al.</em> 2012</a>) of the SNPs found here.</p></div><div id="sec-14" class="subsection"><h3>Empirical evaluation of the accuracy of GBS on heterozygous samples</h3><p id="p-28">The repeatability and accuracy of GBS was assessed using replicate samples and parent–hybrid trios, respectively (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). At 12× read depth, the repeatability of genotyping the same DNA was 99.8% (based on 265,664 total comparisons). The accuracy of genotyping assessed in trios (<em>i.e.</em>, observed parent genotypes serving as the expectation for predicting an F<sub>1</sub> genotype) was &gt;99.6% across all monomorphic and polymorphic loci. For expected heterozygotes specifically (<em>i.e.</em>, only sites that were polymorphic between parents of a hybrid), accuracy was &gt;99.0%. Inspection of incorrectly predicted genotypes revealed discrepancies from either putative heterogeneity of the lines or apparent mis-scoring by GATK. There were 1, 12, and 0 markers that were monomorphic between the parents of CML373 × CML341, CML341 × CML277, and Tzi9 × CML258, respectively, which were called as heterozygotes in the hybrid (putative heterogeneity). There were 26, 52, and 24 polymorphic SNPs between the corresponding parents that were called as homozygotes in the hybrid. Among these SNPs, there were 9, 41, and 10, respectively, that had a skewed SNV read depth distribution in the hybrid (the minor SNV read depth proportion was no greater than 6% of the total read count), despite a median read depth of 45× at these loci. For the other 17, 11, and 14 SNPs, where the minor SNV read depth proportion was relatively high (minimum 31%) and there were &gt;10 reads for the minor SNV, the base-quality score at the variant sites in many of the reads was too low to be considered in the genotype call. Thus, considering these latter discrepancies as <em>bona fide</em> inaccuracies in genotyping, along with the repeatability error rate, the overall accuracy of GBS as applied in this study was estimated as 99.4%.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485555/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485555/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Assessment of typing accuracy for GBS</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Surveying the same loci across GBS libraries</h3><p id="p-30">When working with population samples that are larger than the multiplex size of our GBS design, we found that capturing the same SNP loci across independently constructed libraries was a challenge. This is a critical issue for studies where imputation of missing data is not an option or has unacceptably low accuracy. Our initial protocol for GBS used a single size selection step before PCR amplification and sequencing. Automated size selection (BluePippin; Sage Science, Inc.), with the same type of gel cassette and under the same run settings, yielded differences in the peak and range of the distributions of size-selected fragments as high as 75 and 41 bp, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/TableS1.xlsx">Table S1</a>). Introducing a second size selection step after PCR minimized these differences to 38 and 36 bp, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/TableS1.xlsx">Table S1</a>). Not surprisingly, libraries with similar insert size ranges showed greater correspondence in scored SNPs than those with dissimilar insert size ranges (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/TableS2.xlsx">Table S2</a>). <a href="http://www.g3journal.org/highwire/filestream/484616/field_highwire_adjunct_files/5/FileS5.docx">File S5</a> contains detailed descriptions of all supplemental files.</p></div><div id="sec-16" class="subsection"><h3>Read-based haplotyping using LocHap-GBS</h3><p id="p-31">Variant calling was performed on 276 samples that included four test trios (each in duplicate), 23 parents of the maize NAM population, and 234 F<sub>2</sub> individuals derived from the CML373 × CML341 hybrid (associated with one of the trios). After filtering, there were 29,706 biallelic SNPs (12× read depth) across 9667 digestion loci. In LocHap-GBS, the loci where reads stack up and searches for phased MNVs occur are referred to as windows. Currently, phasing in LocHap-GBS is limited to two to three jointly heterozygous sites (phasing of more heterozygous sites is under development). Because individuals may be heterozygous across more than three sites within a window, LocHap-GBS automates the determination of subwindows that are delineated based on the nearest sets of three (or two) SNPs that are found to be jointly heterozygous in any one individual in the <em>vcf</em> file. However, subwindows and outputted MNVs can be derived from more than the SNP sites that define a subwindow. This happens for SNPs within a subwindow that are found only in the homozygous state across all individuals in the data set, in which case MNVs can exceed a length of three nucleotides. As an extreme example, subwindows would correspond to the original windows for a set of inbred lines where each individual is homozygous for every SNP, such that the length of MNVs would equal the number of SNP sites within a window. Also, the calling routine maintains all of the data in the <em>vcf</em> input such that some subwindows may capture individual SNPs and the <em>hcf</em> output may contain a mixture of MNPs and SNPs.</p><p id="p-32">In the 9667 digestion loci with SNPs, 10,693 subwindows were delineated that included 7749 MNPs and 2944 SNPs. The number of SNPs within MNP subwindows ranged from 2 to 15, with ≈90% of them having five or fewer SNPs. For MNPs, three subwindows contained the observed maximum of 14 MNVs among individuals, while 91% of the subwindows had five or fewer MNVs. LocHap-GBS led to a 3× reduction in the dimensionality of the genotype matrix and a 1.5× increase in the median number of variants per locus (an increase from two to three variants per locus).</p><p id="p-33">The trios were used to confirm whether the MNP genotypes scored by LocHap-GBS were valid. For each trio, subwindows were excluded if they contained missing data, had inconsistent calls between replicate samples, or contained heterozygous SNP genotypes in the parents. The MNVs expected to be observed in the hybrid were determined directly from the filtered SNP data on the parental lines (not by LocHap-GBS). These were then compared to MNVs called by LocHap-GBS. Among 25,404 MNVs recorded for the parents of all trios (ignoring whether MNVs were shared between trios), six were scored differently by LocHap-GBS in the hybrids. All of these were associated with the inaccurately scored genotypes noted previously.</p><p id="p-34">When evaluating unique haplotype numbers for each subwindow, we noticed some MNP loci in the hybrids that had more than two MNVs, which is not expected for maize given that it is diploid. Inspection of the raw sequence data at these loci revealed no barcode swapping errors. Examining the transmission of MNVs in the trios and F<sub>2</sub> population allowed us to rule out DNA cross-contamination and deduce that most, if not all, of these corresponded to divergent paralogs in the sequenced samples that collapsed onto a single locus from the B73 reference genome; we refer to these as CNPs. For instance, a CML373 × CML341 F<sub>1</sub> sample had 21 loci with three or four MNVs that were consistent in replicate samples. For all of these loci, one or both of the inbred parents had more than one MNV (this was determined by maintaining heterozygous SNP genotypes in the inbred parents, which were filtered when estimating accuracy above) that matched each of the MNVs found in the hybrid. These same MNVs were found in the F<sub>2</sub> population, and tests of cosegregation indicated that 11 of them were genetically linked, but some of these loci included a mixture of linked and unlinked MNVs (data not shown).</p><p id="p-35">Under the assumption that loci with more than one MNV in an inbred line represent CNPs, among the 23 inbred parents of the NAM population there was an average of 155 (2.0%) total CNPs per individual (maximum of 305 CNPs for CML52). The cumulative number of unique CNPs was recorded with the addition of each inbred line inserted in order of their estimated genetic distance from B73 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). There was a median increase of 30 unique CNP-associated loci with exactly two MNVs, culminating in a total of 1195 such loci among all of the lines (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). This is an upper-bound estimate of the number of CNPs, since CNPs are conflated with heterozygous loci in inbred lines. For a lower-bound estimate, loci with more than two MNVs per line were recorded, which showed an average of 2 (0.03%) total CNPs per line (maximum of 8), a median increase of 2 CNPs with each additional line added to the dataset, and a cumulative total of 41 unique CNPs. Taken together, the estimated proportion of CNPs typed in this study among the 23 parental lines of the NAM population lies between 0.5 and 15.4% (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Excluding the reference line B73, the correlation between genetic distance and number of unique CNPs per additional line was not significantly different from zero for both classifications of CNPs.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CNP-associated loci in the NAM inbred parents. From left-to-right, each bar indicates the cumulative number of unique CNPs (loci containing more than one MNV) in each individual compared to its preceding set of lines, with the lines ordered by their MNP1-based genetic distance from B73. The gray portion of the bar is the number of loci that had exactly two MNVs and the orange portion of the bar is the number of loci that had more than two MNVs. CNP, copy number polymorphism; MNP, multi-nucleotide polymorphism; MNV, multi-nucleotide variant; NAM, nested association mapping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452500219" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;CNP-associated loci in the NAM inbred parents. From left-to-right, each bar indicates the cumulative number of unique CNPs (loci containing more than one MNV) in each individual compared to its preceding set of lines, with the lines ordered by their MNP1-based genetic distance from B73. The gray portion of the bar is the number of loci that had exactly two MNVs and the orange portion of the bar is the number of loci that had more than two MNVs. CNP, copy number polymorphism; MNP, multi-nucleotide polymorphism; MNV, multi-nucleotide variant; NAM, nested association mapping.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2161/F3.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/7/2161/F3.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485570" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">CNP-associated loci in the NAM inbred parents. From left-to-right, each bar indicates the cumulative number of unique CNPs (loci containing more than one MNV) in each individual compared to its preceding set of lines, with the lines ordered by their MNP1-based genetic distance from B73. The gray portion of the bar is the number of loci that had exactly two MNVs and the orange portion of the bar is the number of loci that had more than two MNVs. CNP, copy number polymorphism; MNP, multi-nucleotide polymorphism; MNV, multi-nucleotide variant; NAM, nested association mapping.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Genetic diversity based on SNPs <em>vs.</em> MNPs</h3><p id="p-37">Results from the analysis of genetic diversity among parents of the NAM population were compared using SNP and MNP datasets. Excluding CNP-associated genotypes (MNP2 and MNP3 datasets) from the MNP1 dataset showed no significant difference in the distribution of pairwise genetic distances, and all three datasets were almost perfectly correlated (<em>r</em> &gt; 0.999; Mantel test: <em>P</em> = 0.001). Therefore, the following results are reported only for the SNP and MNP1 datasets. Although the distance matrices were significantly correlated (<em>r</em> = 0.890; Mantel test: <em>P</em> = 0.001), the average pairwise distance for SNPs was lower than that for MNPs (0.28 and 0.37, respectively) and the range in the distances was smaller for SNPs (0.12) than for MNPs (0.20). Although the mean bootstrap probabilities were essentially identical for the consensus trees based on SNPs (79.3%) and MNPs (78.9%), the Robinson–Foulds distance between those trees was 12 (maximum possible distance was 40). Consequently, MDS with MNPs produced greater separation among the lines (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) and resulted in differences in their topology (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>; <em>e.g.</em>, c.f. CML103, Ky21, and CML52).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of genetic diversity with SNP and MNP data. (A) MDS plots based on shared allele distance for SNP (top) and MNP1 (bottom) data on 23 maize inbred lines. (B) Corresponding BIONJ trees (branches transformed to equal lengths). Colors represent previously assigned population structure (Liu et al. 2003). MDS, multi-dimensional scaling; MNP, multi-nucleotide polymorphism; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452500219" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of genetic diversity with SNP and MNP data. (A) MDS plots based on shared allele distance for SNP (top) and MNP1 (bottom) data on 23 maize inbred lines. (B) Corresponding BIONJ trees (branches transformed to equal lengths). Colors represent previously assigned population structure (Liu &lt;em&gt;et al.&lt;/em&gt; 2003). MDS, multi-dimensional scaling; MNP, multi-nucleotide polymorphism; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2161/F4.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/7/2161/F4.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2161/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485456" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Analysis of genetic diversity with SNP and MNP data. (A) MDS plots based on shared allele distance for SNP (top) and MNP1 (bottom) data on 23 maize inbred lines. (B) Corresponding BIONJ trees (branches transformed to equal lengths). Colors represent previously assigned population structure (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Liu <em>et al.</em> 2003</a>). MDS, multi-dimensional scaling; MNP, multi-nucleotide polymorphism; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-39">Many implementations of GBS are not optimal for genotyping of heterozygous individuals or populations with unknown parentage. Imputation has proven successful on biparental families of recombinant inbred lines, but falls short when applied to populations that have low linkage disequilibrium (median imputation accuracy has been shown to be between 15 and 80% for markers with a <em>r</em><sup>2</sup> &lt; 0.3; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">He <em>et al.</em> 2015</a>). The few approaches that have been developed for genotyping heterozygous populations use population-specific filters, and depend on relatedness among individuals to score genotypes or require targeted sequencing to obtain read depths sufficient for calling heterozygotes (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Uitdewilligen <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gardner <em>et al.</em> 2014</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Barba <em>et al.</em> 2014</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hyma <em>et al.</em> 2015</a>). HetMappS (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hyma <em>et al.</em> 2015</a>) was developed specifically for dense genetic maps, and markers are filtered based on expected genotype ratios for pseudotest-cross markers. This method was successful at increasing marker density, but was designed for use in F<sub>1</sub> populations only. Here, we have presented a GBS protocol and bioinformatic pipeline that, at least for maize, produces highly accurate genotype data on heterozygous individuals without requiring information on parentage or family structure, nor imputation to obtain high call rates on the typed SNPs or MNPs.</p><p id="p-40">As discussed by <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Peterson <em>et al.</em> (2012)</a>, there is a balancing act in deciding which enzymes and size selection windows should be used when implementing GBS. In maize, which has been extensively explored for SNPs, where the enzyme <em>Ape</em>KI has been used routinely for GBS, the <em>Ngo</em>MIV-<em>Csp</em>6I enzyme combination provided an effective means for accurate genotyping that led to the discovery of many new SNPs. Assuming there is a 50% chance of sequencing each allele at a biallelic marker, a minimum read depth of 12× predicts that the binomial probability of sequencing each allele at least twice is &gt; 99.6%. In this study, accuracy was estimated experimentally using parent–hybrid trios to be &gt;99.4% (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), which fits closely to the expected sampling probability. Compared to an existing catalog of <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2161/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2161/embed/mml-math-1.gif"/></noscript></span></span> SNPs (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Ganal <em>et al.</em> 2011</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chia <em>et al.</em> 2012</a>) identified among diverse individuals of maize that included all but four of the ones used here (excluding: CML10, CML258, CML373, and Tzi9), this study discovered ≈100,000 new SNPs that comprised nearly two-thirds of the total SNPs found. These were enriched near genes, which was in part due to the distribution of 4PD loci, but may also be attributed to patterns in methylation: <em>Ngo</em>MIV is averse to the CpG methylation that is predominant in the intergenic repeat space of the maize genome (Antequera and Bird 1988). Enrichment of the typed loci around the start codon of genes (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>) might be because the starts of monocot genes are GC-rich (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Glémin <em>et al.</em> 2015</a>) and <em>Ngo</em>MIV recognizes a GC-rich recognition site.</p><p id="p-41">With the decline in cost for HTS, there is growing opportunity to apply GBS to thousands of samples for genetic studies. Sequencing the same loci within and across separate GBS libraries is required for this, but whether the same loci are sequenced has not been examined much in the literature (we are aware of one exception: <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">DaCosta and Sorenson 2014</a>), perhaps because GBS studies tend to rely on imputation of missing data. It has been reasoned that size selection would play an important role in sequencing shared loci (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Ali <em>et al.</em> 2016</a>). <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Peterson <em>et al.</em> (2012)</a> showed that narrower size selection windows increased the sequencing depth of shared loci in a library. As a cautionary note, some size selection windows may contain an abundance of repetitive loci (<em>e.g</em>., <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1/FigureS1.tiff">Figure S1</a>). Here, we addressed the issue of sampling shared loci across different libraries, reaching the conclusion that two size selection steps were needed to maximize the number of shared loci sequenced. Importantly, our results led us to the realization that the standard Y-adapters used for HTS libraries may form structures that migrate slower than fully dsDNA in dye-free agarose gels. This can lead to inconsistent size selection when using automated size selection instrumentation (S. Hoda, Sage Science, Inc., personal communication). Because of the inconsistency in the size selection of libraries with Y-adaptors, we had to reduce the pre-PCR size selection window and introduce a post-PCR size selection on the dsDNA created by PCR amplification. This minimized the variation in size selection between libraries and maximized the recovery of shared GBS loci.</p><p id="p-42">This study expands the usefulness of GBS data, allowing for phased genotyping of MNPs, which enhances the information available for genetic studies. For example, founder haplotypes that are resolved for multi-parental populations can be used to impute missing data in the progeny (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gatti <em>et al.</em> 2014</a>) or to initialize imputation (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Davies <em>et al.</em> 2016</a>). Also, the use of haplotypes may prove useful in uncovering regions associated with adaptive traits in both model and nonmodel species (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lorenz <em>et al.</em> 2010</a>). The extension of LocHap for GBS was developed as a new tool for phased genotyping using standard file formats produced by most mapping and variant calling software. LocHap-GBS may be advantageous over other current tools capable of scoring haplotypes, such as STACKS and Haplotag, in that statistically vetted MNVs can be scored by LocHap-GBS on individuals without population filters only definable for specific types of samples. However, LocHap-GBS may not be readily integrated by tools that use customized data formats, though the algorithm developed for local haplotyping could be (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Sengupta <em>et al.</em> 2015</a>). Finally, the software GATK used to score SNPs can also generate phased genotypes (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">McKenna <em>et al.</em> 2010</a>). RedRep (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) was developed using the GATK UnifiedGenotyper, which does not have MNP phasing functionality (GATK HaplotypeCaller does). We have not experimented with the current phasing tool of GATK, but we note some unique features of LocHap-GBS. It uses a distinct algorithm for identifying MNVs, can be run independently with standard input files, reports phased genotypes in a different format (which we consider more appropriate for direct downstream analysis), and can score more than two haplotypes at a locus; this led to our finding of CNPs in maize.</p><p id="p-43">Phased genotype data, while decreasing the marker dimensionality of the genotype matrix, was more informative than SNP data. Previously characterized lines of maize were more differentiated and showed some differences in relatedness when analyzed using MNPs compared to SNPs (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). In addition, phased-GBS revealed CNPs that were validated by genetic transmission yet undetectable in the underlying SNP data. These can be explained as divergent paralogs within the sample genomes that are present as only one copy in the reference genome. Because the SNP data were filtered to examine only 4PD loci, it seems unlikely that the CNPs were a result of mapping to sequences that have been collapsed in the assembly of the B73 reference genome, but this is a possibility that could result in reproducibly inaccurate CNPs. Nevertheless, having included B73 in our set of sequenced individuals, which is the same line used to sequence the reference genome, we determined that our B73 sample harbored no GBS loci with more than two MNVs, while other inbred lines did (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). However, there were nearly 100 GBS loci that were heterozygous in the B73 sample. These may be attributed to residual heterozygosity in the line, errors in the assembly, or CNPs that had arisen during seed increases of the stock. Inspecting the genomic distribution of these loci showed no pattern of clustering, as one might expect for residual heterozygosity, suggesting one or both of the latter two explanations.</p><p id="p-44">Although our study was not designed for CNP discovery, it led to the identification of at least 41 CNP-associated loci (potentially ≈1000) among 23 inbred lines of maize. The average read depth at CNP-associated loci was 77, which was about the average read depth of non-CNP associated loci. However, the read depth range for CNP-associated loci was 1480 (max: 1492 reads) compared to 755 for non-CNP associated loci (max: 767 reads), which is expected if there are multiple paralogous loci mapping to the same place in the reference genome. Moreover, despite an average depth of sequence coverage that gives a high likelihood for detecting at least 10 CNVs, no more than four CNVs at a locus were found within an individual. There was no relationship between genome-wide estimates of genetic distance and the number of unique CNP-associated loci (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>), suggesting the evolution of these putative CNPs may be different from SNPs and MNPs. This study demonstrates the potential of extending GBS for phased genotyping. As applications of GBS expand beyond biparental mapping populations, we foresee numerous benefits to typing MNPs by phased-GBS.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank Karol Miaskiewicz at the Delaware Biotechnology Institute for assistance using BIOMIX. We thank Patricia Klein at Texas A&amp;M University for suggesting we consider <em>Ngo</em>MIV. This project was supported by an Agriculture and Food Research Initiative Competitive grant (no. 2011-67003-30342) from the US Department of Agriculture National Institute of Food and Agriculture (Agriculture and Natural Resources Science for Climate Variability and Change Program) and NIH 2R01 CA132897. Other support from the University of Delaware Bioinformatics Core Facility, including use of the BIOMIX computational cluster, was made possible by the Delaware IDeA Network of Biomedical Research Excellence (National Institutes of Health grant P20 GM103446). </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></li><li class="fn" id="fn-3"><p id="p-46">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042036/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-47"><em>Communicating editor: P. J. Brown</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-04-04"><span class="received-label">Received </span>April 4, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-05-01"><span class="accepted-label">Accepted </span>May 1, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Manching <em>et al.</em></li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.1" data-doi="10.1534/genetics.115.183665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Rourke</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amish</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meek</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">RAD capture (rapture): flexible and efficient sequence-based genotyping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.183665%26rft_id%253Dinfo%253Apmid%252F26715661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.2" data-doi="10.1101/gr.115402.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erezyilmaz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mast</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multiplexed shotgun genotyping for rapid and efficient genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115402.110%26rft_id%253Dinfo%253Apmid%252F21233398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNjEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.3" data-doi="10.1038/nrg.2015.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Harnessing the power of RADseq for ecological and evolutionary genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2015.28%26rft_id%253Dinfo%253Apmid%252F26729255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2015.28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26729255&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.7.2161.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antequera</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Unmethylated CpG islands associated with genes in higher plant DNA.</span> <abbr class="cit-jnl-abbrev">The EMBO Journal</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2295</span>–<span class="cit-lpage">2299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.volume%253D7%26rft.spage%253D2295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.5" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.6" data-doi="10.1007/s00122-013-2202-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barba</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cadle-Davidson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Grapevine powdery mildew resistance and susceptibility loci identified on a high-resolution SNP map.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2202-x%26rft_id%253Dinfo%253Apmid%252F24072208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2202-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24072208&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.7" data-doi="10.1534/genetics.112.147710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekhon</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foerster</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Marker density and read depth for genotyping populations using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147710%26rft_id%253Dinfo%253Apmid%252F23410831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEwNzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjE2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.8" data-doi="10.1038/368455a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowcock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz-Linares</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomfohrde</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minch</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">High resolution of human evolutionary trees with polymorphic microsatellites.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">368</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBowcock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D368%26rft.issue%253D6470%26rft.spage%253D455%26rft.epage%253D457%26rft.atitle%253DHigh%2Bresolution%2Bof%2Bhuman%2Bevolutionary%2Btrees%2Bwith%2Bpolymorphic%2Bmicrosatellites.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F368455a0%26rft_id%253Dinfo%253Apmid%252F7510853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/368455a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7510853&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994ND12000063&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.9" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.10" data-doi="10.1038/ng.2313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maize HapMap2 identifies extant variation from a genome in flux.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChia%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D803%26rft.epage%253D807%26rft.atitle%253DMaize%2BHapMap2%2Bidentifies%2Bextant%2Bvariation%2Bfrom%2Ba%2Bgenome%2Bin%2Bflux.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2313%26rft_id%253Dinfo%253Apmid%252F22660545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660545&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.11" data-doi="10.1371/journal.pone.0106713"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DaCosta</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorenson</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Amplification biases and consistent recovery of loci in a double-digest RAD-seq protocol.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e106713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De106713%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0106713%26rft_id%253Dinfo%253Apmid%252F25188270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0106713&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25188270&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.12" data-doi="10.1038/ng.3594"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Rapid genotype imputation from sequence without reference panels.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D965%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3594&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dray</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dufour</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The ade4 package: implementing the duality diagram for ecologists.</span> <abbr class="cit-jnl-abbrev">Journal of Statistical Software.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware.%26rft.volume%253D22%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.14" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferdosi</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henshall</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Study of the optimum haplotype length to build genomic relationship matrices.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D75%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic diversity analysis with 454 pyrosequencing and genomic reduction confirmed the eastern and western division in the cultivated barley gene pool.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.17" data-doi="10.1371/journal.pone.0028334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28334%26rft.epage%253De28334%26rft.atitle%253DA%2Blarge%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2BSNP%2Bgenotyping%2Barray%253A%2Bdevelopment%2Band%2Bgermplasm%2Bgenotyping%252C%2Band%2Bgenetic%2Bmapping%2Bto%2Bcompare%2Bwith%2Bthe%2BB73%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028334%26rft_id%253Dinfo%253Apmid%252F22174790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174790&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooke</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast and cost-effective genetic mapping in apple using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.19" data-doi="10.1093/oxfordjournals.molbev.a025808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">685</span>–<span class="cit-lpage">695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DGascuel%26rft.auinit1%253DO%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D685%26rft.epage%253D695%26rft.atitle%253DBIONJ%253A%2Ban%2Bimproved%2Bversion%2Bof%2Bthe%2BNJ%2Balgorithm%2Bbased%2Bon%2Ba%2Bsimple%2Bmodel%2Bof%2Bsequence%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025808%26rft_id%253Dinfo%253Apmid%252F9254330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE0LzcvNjg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.20" data-doi="10.1534/g3.114.013748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait locus mapping methods for diversity outbred mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1623%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.114.013748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.21" data-doi="10.1101/gr.185488.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glémin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantification of GC-biased gene conversion in the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Research</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1215</span>–<span class="cit-lpage">1228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185488.114%26rft_id%253Dinfo%253Apmid%252F25995268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMTIxNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mette</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bothe</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebmeyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Prospects and limits of marker imputation in quantitative genetic studies in european elite wheat (triticum aestivum l.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.23" data-doi="10.1186/1471-2164-15-979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffelfinger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragoso</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Overton</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mottinger</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Flexible and scalable genotyping-by-sequencing strategies for population studies.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D979%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-979%26rft_id%253Dinfo%253Apmid%252F25406744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25406744&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barba</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Londo</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Heterozygous mapping strategy (HetMappS) for high resolution genotyping-by-sequencing markers: a case study in grapevine.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0134880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0134880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.25" data-doi="10.1093/bioinformatics/btn129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">adegenet: a R package for the multivariate analysis of genetic markers.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn129%26rft_id%253Dinfo%253Apmid%252F18397895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjQvMTEvMTQwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.26" data-doi="10.1007/BF02289565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruskal</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Multidimensional scaling by optimizing goodness of fit to a nonmetric hypothesis.</span> <abbr class="cit-jnl-abbrev">Psychometrika</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychometrika%26rft.volume%253D29%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02289565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02289565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1964CBT8100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-7.7.2161.27"><div class="cit-metadata unstructured">Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv: 1303.3997v1 [q-bio.GN].</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muse</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetic structure and diversity among maize inbred lines as inferred from DNA microsatellites.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2117</span>–<span class="cit-lpage">2128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLiu%26rft.auinit1%253DK.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D2117%26rft.epage%253D2128%26rft.atitle%253DGenetic%2BStructure%2Band%2BDiversity%2BAmong%2BMaize%2BInbred%2BLines%2Bas%2BInferred%2BFrom%2BDNA%2BMicrosatellites%26rft_id%253Dinfo%253Apmid%252F14704191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzIxMTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjE2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.29" data-doi="10.1371/journal.pone.0014079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Performance of single nucleotide polymorphisms <em>vs.</em> haplotypes for genome-wide association analysis in barley.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e14079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLorenz%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De14079%26rft.epage%253De14079%26rft.atitle%253DPerformance%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bversus%2Bhaplotypes%2Bfor%2Bgenome-wide%2Bassociation%2Banalysis%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0014079%26rft_id%253Dinfo%253Apmid%252F21124933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0014079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21124933&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.30" data-doi="10.1371/journal.pone.0024861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taba</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparative SNP and haplotype analysis reveals a higher genetic diversity and rapider LD decay in tropical than temperate germplasm in maize.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e24861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLu%26rft.auinit1%253DY.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De24861%26rft.epage%253De24861%26rft.atitle%253DComparative%2BSNP%2Band%2Bhaplotype%2Banalysis%2Breveals%2Ba%2Bhigher%2Bgenetic%2Bdiversity%2Band%2Brapider%2BLD%2Bdecay%2Bin%2Btropical%2Bthan%2Btemperate%2Bgermplasm%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0024861%26rft_id%253Dinfo%253Apmid%252F21949770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0024861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21949770&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.31" data-doi="10.1093/nar/24.4.800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Increased cloning efficiency by temperature-cycle ligation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F24.4.800%26rft_id%253Dinfo%253Apmid%252F8604328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjI0LzQvODAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIxNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mantel</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">The detection of disease clustering and a generalized regression approach.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DMantel%26rft.auinit1%253DN.%26rft.volume%253D27%26rft.issue%253D2_Part_1%26rft.spage%253D209%26rft.epage%253D220%26rft.atitle%253DThe%2BDetection%2Bof%2BDisease%2BClustering%2Band%2Ba%2BGeneralized%2BRegression%2BApproach%26rft_id%253Dinfo%253Apmid%252F6018555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjE1OiIyNy8yX1BhcnRfMS8yMDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjE2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads.</span> <abbr class="cit-jnl-abbrev">EMBnet.journal.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBnet.journal.%26rft.volume%253D17%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.34" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.35" data-doi="10.1126/science.1174320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeda</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic properties of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcMullen%26rft.auinit1%253DM.%2BD.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D737%26rft.epage%253D740%26rft.atitle%253DGenetic%2BProperties%2Bof%2Bthe%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174320%26rft_id%253Dinfo%253Apmid%252F19661427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjE2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.36" data-doi="10.1093/bioinformatics/bts277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neph</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuehn</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haugen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BEDOPS: high-performance genomic feature operations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1919</span>–<span class="cit-lpage">1920</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts277%26rft_id%253Dinfo%253Apmid%252F22576172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTQvMTkxOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.37" data-doi="10.1093/bioinformatics/btg412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paradis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claude</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">APE: analyses of phylogenetics and evolution in R language.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMC8yLzI4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.38" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.39" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-web" id="cit-7.7.2161.40"><div class="cit-metadata unstructured">R Core Team, 2016 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. <a href="https://www.R-project.org/">https://www.R-project.org/</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.41" data-doi="10.1016/0025-5564(81)90043-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Comparison of phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath.%2BBiosci.%26rft.volume%253D53%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0025-5564%252881%252990043-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0025-5564(81)90043-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981LB66300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.42" data-doi="10.1371/journal.pone.0131080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jean</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castonguay</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belzile</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Validation of genotyping-by-sequencing analysis in populations of tetraploid alfalfa by 454 sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0131080%26rft_id%253Dinfo%253Apmid%252F25919488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0131080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25919488&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.43" data-doi="10.1093/bioinformatics/btq706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schliep</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">phangorn: phylogenetic analysis in r.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq706%26rft_id%253Dinfo%253Apmid%252F21169378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy80LzU5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sengupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gulukota</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naughton</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ultra-fast local-haplotype variant calling using paired-end DNA-sequencing data reveals somatic mosaicism in tumor and normal blood samples.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">e25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D44%26rft.spage%253De25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.45" data-doi="10.1371/journal.pone.0054603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iquira</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardivel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Légaré</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e54603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De54603%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054603%26rft_id%253Dinfo%253Apmid%252F23372741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372741&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.46" data-doi="10.1073/pnas.151244298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sawkins</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Patterns of DNA sequence polymorphism along chromosome 1 of maize (Zea mays ssp. mays L.).</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">9161</span>–<span class="cit-lpage">9166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.151244298%26rft_id%253Dinfo%253Apmid%252F11470895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTgvMTYvOTE2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekele</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattori</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Haplotag: software for haplotype-based genotyping-by-sequencing analysis.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">863</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torkamaneh</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laroche</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belzile</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide SNP calling from genotyping by sequencing (GBS) data: a comparison of seven pipelines and two sequencing technologies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0161333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0161333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2161.49" data-doi="10.1371/journal.pone.0062355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J. G. A. M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolters</span>  <span class="cit-name-given-names">A. M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’hoop</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borm</span>  <span class="cit-name-given-names">T. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e62355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62355%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062355%26rft_id%253Dinfo%253Apmid%252F23667470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23667470&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/7/2161.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2151" title="Two Loci Contribute Epistastically to Heterospecific Pollen Rejection, a Postmating Isolating Barrier Between Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2171" title="A Pathway-Centered Analysis of Pig Domestication and Breeding in Eurasia" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/7"><span>Volume 7 Issue 7, July 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/7.cover-source.jpg?itok=eAMfPilr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2161" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-noxfMpBdLPj6a87ifRHLGyCgL3k7KqMNA4Y1tzUQHD8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/7/2161" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/7/2161" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/7/2161" class="active" data-icon-position="" data-hide-link-title="0">Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485312" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1N4aBvRws2LWdup7ktQDTzg1ph1pGFgqw3a_ZzxSD78" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2161.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/7/2161" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-dVraelIPDHlITVXbgMXs3Fqg-0jqmFnGRJP1F-ALihw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/7/2161" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2161.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485312" id="node485312--2" data-pisa="ggg;7/7/2161" data-pisa-master="ggg;g3.117.042036" data-apath="/ggg/7/7/2161.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Heather Manching</span>, <span class="highwire-citation-author" data-delta="1">Subhajit Sengupta</span>, <span class="highwire-citation-author" data-delta="2">Keith R. Hopper</span>, <span class="highwire-citation-author" data-delta="3">Shawn W. Polson</span>, <span class="highwire-citation-author" data-delta="4">Yuan Ji</span> and <span class="highwire-citation-author" data-delta="5">Randall J. Wisser</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2161-2170;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042036 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.042036" data-atom-uri="/ggg/7/7/2161.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485312--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Heather Manching </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware 19716</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heather%2BManching%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manching%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeather%2BManching%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Subhajit Sengupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Program of Computational Genomics and Medicine, North Shore University Health System, Evanston, Illinois 60201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Subhajit%2BSengupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sengupta%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASubhajit%2BSengupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Keith R. Hopper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Beneficial Insect Introductions Research Unit, United States Department of Agriculture, Agricultural Research Service, Newark, Delaware 19713</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keith%2BR.%2BHopper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hopper%20KR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeith%2BR.%2BHopper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shawn W. Polson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Bioinformatics and Computational Biology, Delaware Biotechnology Institute, University of Delaware, Newark, Delaware 19711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BW.%2BPolson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Polson%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShawn%2BW.%2BPolson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yuan Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Program of Computational Genomics and Medicine, North Shore University Health System, Evanston, Illinois 60201</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Health Studies, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuan%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuan%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Randall J. Wisser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware 19716</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randall%2BJ.%2BWisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wisser%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARandall%2BJ.%2BWisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/7/2161" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/7/2161&amp;title=Phased%20Genotyping-by-Sequencing%20Enhances%20Analysis%20of%20Genetic%20Diversity%20and%20Reveals%20Divergent%20Copy%20Number%20Variants%20in%20Maize" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/7/2161&amp;title=Phased%20Genotyping-by-Sequencing%20Enhances%20Analysis%20of%20Genetic%20Diversity%20and%20Reveals%20Divergent%20Copy%20Number%20Variants%20in%20Maize" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/7/2161&amp;title=Phased%20Genotyping-by-Sequencing%20Enhances%20Analysis%20of%20Genetic%20Diversity%20and%20Reveals%20Divergent%20Copy%20Number%20Variants%20in%20Maize" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/7/2161&amp;text=Phased%20Genotyping-by-Sequencing%20Enhances%20Analysis%20of%20Genetic%20Diversity%20and%20Reveals%20Divergent%20Copy%20Number%20Variants%20in%20Maize" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/7/2161&amp;title=Phased%20Genotyping-by-Sequencing%20Enhances%20Analysis%20of%20Genetic%20Diversity%20and%20Reveals%20Divergent%20Copy%20Number%20Variants%20in%20Maize" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/7/2161&amp;t=Phased%20Genotyping-by-Sequencing%20Enhances%20Analysis%20of%20Genetic%20Diversity%20and%20Reveals%20Divergent%20Copy%20Number%20Variants%20in%20Maize" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/7/2161&amp;title=Phased%20Genotyping-by-Sequencing%20Enhances%20Analysis%20of%20Genetic%20Diversity%20and%20Reveals%20Divergent%20Copy%20Number%20Variants%20in%20Maize" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/7/2161&amp;title=Phased%20Genotyping-by-Sequencing%20Enhances%20Analysis%20of%20Genetic%20Diversity%20and%20Reveals%20Divergent%20Copy%20Number%20Variants%20in%20Maize" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485312/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485312" id="node485312--3" data-pisa="ggg;7/7/2161" data-pisa-master="ggg;g3.117.042036" data-apath="/ggg/7/7/2161.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phased Genotyping-by-Sequencing Enhances Analysis of Genetic Diversity and Reveals Divergent Copy Number Variants in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Heather Manching</span>, <span class="highwire-citation-author" data-delta="1">Subhajit Sengupta</span>, <span class="highwire-citation-author" data-delta="2">Keith R. Hopper</span>, <span class="highwire-citation-author" data-delta="3">Shawn W. Polson</span>, <span class="highwire-citation-author" data-delta="4">Yuan Ji</span> and <span class="highwire-citation-author" data-delta="5">Randall J. Wisser</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2161-2170;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042036 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.042036" data-atom-uri="/ggg/7/7/2161.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485312--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Heather Manching </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware 19716</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heather%2BManching%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manching%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeather%2BManching%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Subhajit Sengupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Program of Computational Genomics and Medicine, North Shore University Health System, Evanston, Illinois 60201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Subhajit%2BSengupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sengupta%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASubhajit%2BSengupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Keith R. Hopper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Beneficial Insect Introductions Research Unit, United States Department of Agriculture, Agricultural Research Service, Newark, Delaware 19713</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keith%2BR.%2BHopper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hopper%20KR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeith%2BR.%2BHopper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shawn W. Polson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Bioinformatics and Computational Biology, Delaware Biotechnology Institute, University of Delaware, Newark, Delaware 19711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BW.%2BPolson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Polson%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShawn%2BW.%2BPolson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yuan Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Program of Computational Genomics and Medicine, North Shore University Health System, Evanston, Illinois 60201</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Health Studies, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuan%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuan%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Randall J. Wisser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware 19716</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randall%2BJ.%2BWisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wisser%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARandall%2BJ.%2BWisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2161&amp;rft.epage=2170&amp;rft.atitle=Phased%20Genotyping-by-Sequencing%20Enhances%20Analysis%20of%20Genetic%20Diversity%20and%20Reveals%20Divergent%20Copy%20Number%20Variants%20in%20Maize&amp;rft.volume=7&amp;rft.issue=7&amp;rft.date=2017-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Manching%2C+Heather&amp;rft.au=Sengupta%2C+Subhajit&amp;rft.au=Hopper%2C+Keith+R.&amp;rft.au=Polson%2C+Shawn+W.&amp;rft.au=Ji%2C+Yuan&amp;rft.au=Wisser%2C+Randall+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485312/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485312/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485312/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485312/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485312/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485312/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485312/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485312/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485312/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485312/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485312/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485312/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/7/2161" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/7/2161.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/7/2161.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/7/2161.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/7/2161.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
