<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/7/2363" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/7/2363.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/7/2363.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/7/2363.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/7/2363/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/7/2363.atom" />
<meta name="HW.pisa" content="ggg;7/7/2363" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Evolutionary Basis of Translational Accuracy in Plants" />
<meta name="DC.Identifier" content="10.1534/g3.117.040626" />
<meta name="DC.Date" content="2017-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Camiolo et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Mistranslation errors compromise fitness by wasting resources on nonfunctional proteins. In order to reduce the cost of mistranslations, natural selection chooses the most accurately translated codons at sites that are particularly important for protein structure and function. We investigated the determinants underlying selection for translational accuracy in several species of plants belonging to three clades: Brassicaceae, Fabidae, and Poaceae. Although signatures of translational selection were found in genes from a wide range of species, the underlying factors varied in nature and intensity. Indeed, the degree of synonymous codon bias at evolutionarily conserved sites varied among plant clades while remaining uniform within each clade. This is unlikely to solely reflect the diversity of tRNA pools because there is little correlation between synonymous codon bias and tRNA abundance, so other factors must affect codon choice and translational accuracy in plant genes. Accordingly, synonymous codon choice at a given site was affected not only by the selection pressure at that site, but also its participation in protein domains or mRNA secondary structures. Although these effects were detected in all the species we analyzed, their impact on translation accuracy was distinct in evolutionarily distant plant clades. The domain effect was found to enhance translational accuracy in dicot and monocot genes with a high GC content, but to oppose the selection of more accurate codons in monocot genes with a low GC content." />
<meta name="DC.Contributor" content="Salvatore Camiolo" />
<meta name="DC.Contributor" content="Gaurav Sablok" />
<meta name="DC.Contributor" content="Andrea Porceddu" />
<meta name="article:published_time" content="2017-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Evolutionary Basis of Translational Accuracy in Plants" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Mistranslation errors compromise fitness by wasting resources on nonfunctional proteins. In order to reduce the cost of mistranslations, natural selection chooses the most accurately translated codons at sites that are particularly important for protein structure and function. We investigated the determinants underlying selection for translational accuracy in several species of plants belonging to three clades: Brassicaceae, Fabidae, and Poaceae. Although signatures of translational selection were found in genes from a wide range of species, the underlying factors varied in nature and intensity. Indeed, the degree of synonymous codon bias at evolutionarily conserved sites varied among plant clades while remaining uniform within each clade. This is unlikely to solely reflect the diversity of tRNA pools because there is little correlation between synonymous codon bias and tRNA abundance, so other factors must affect codon choice and translational accuracy in plant genes. Accordingly, synonymous codon choice at a given site was affected not only by the selection pressure at that site, but also its participation in protein domains or mRNA secondary structures. Although these effects were detected in all the species we analyzed, their impact on translation accuracy was distinct in evolutionarily distant plant clades. The domain effect was found to enhance translational accuracy in dicot and monocot genes with a high GC content, but to oppose the selection of more accurate codons in monocot genes with a low GC content.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/07/01" />
<meta name="citation_mjid" content="ggg;7/7/2363" />
<meta name="citation_id" content="7/7/2363" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/7/2363" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/7/2363.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/7/2363.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/7/2363.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.040626" />
<meta name="citation_pmid" content="28533334" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2363" />
<meta name="citation_lastpage" content="2373" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Salvatore Camiolo" />
<meta name="citation_author_institution" content="Dipartimento di Agraria, Sezione di Agronomia, Coltivazione Erbacee e Genetica, Università degli Studi di Sassari, 07100, Italy" />
<meta name="citation_author" content="Gaurav Sablok" />
<meta name="citation_author_institution" content="Plant Functional Biology and Climate Change Cluster, University of Technology Sydney, New South Wales 2007, Australia" />
<meta name="citation_author" content="Andrea Porceddu" />
<meta name="citation_author_institution" content="Dipartimento di Agraria, Sezione di Agronomia, Coltivazione Erbacee e Genetica, Università degli Studi di Sassari, 07100, Italy" />
<meta name="citation_author_email" content="aporceddu@uniss.it" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=T. Ahmad;citation_author=G. Sablok;citation_author=T. V. Tatarinova;citation_author=Q. Xu;citation_author=X. X. Deng;citation_title=Evaluation of codon biology in citrus and Poncirus trifoliata based on genomic features and frame corrected expressed sequence tags.;citation_volume=20;citation_year=2013;citation_pmid=23315666;citation_doi=10.1093/dnares/dss039" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy.;citation_pages=927-935;citation_volume=136;citation_year=1994;citation_issue=3;citation_pmid=8005445" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=BioData Min.;citation_author=S. Camiolo;citation_author=A. Porceddu;citation_title=gff2sequence, a new user friendly tool for the generation of genomic sequences.;citation_pages=15;citation_volume=6;citation_year=2013;citation_pmid=24020993;citation_doi=10.1186/1756-0381-6-15" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Camiolo;citation_author=D. Rau;citation_author=A. Porceddu;citation_title=Mutational biases and selective forces shaping the structure of Arabidopsis genes.;citation_pages=e6356-e6356;citation_volume=4;citation_year=2009;citation_issue=7;citation_pmid=19633720;citation_doi=10.1371/journal.pone.0006356" />
<meta name="citation_reference" content="citation_journal_title=BMC Res. Notes;citation_author=S. Camiolo;citation_author=S. Melito;citation_author=G. Milia;citation_author=A. Porceddu;citation_title=Seforta, an integrated tool for detecting the signature of selection in coding sequences.;citation_pages=240;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=S. Camiolo;citation_author=S. Melito;citation_author=A. Porceddu;citation_title=New insights into the interplay between codon bias determinants in plants.;citation_volume=22;citation_year=2015;citation_pmid=26546225;citation_doi=10.1093/dnares/dsv027" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=C. Chan;citation_author=C. S. Carmack;citation_author=D. D. Long;citation_author=A. Maliyekkel;citation_author=Y. Shao;citation_title=A structural interpretation of the effect of GC-content on efficiency of RNA interference.;citation_pages=S33-S37;citation_volume=10;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=F. Cheng;citation_author=J. Wu;citation_author=L. Fang;citation_author=S. Sun;citation_author=B. Liu;citation_title=Biased gene fractionation and dominant gene expression among the subgenomes of Brassica rapa.;citation_pages=e36442-e36442;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22567157;citation_doi=10.1371/journal.pone.0036442" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=Y. Clement;citation_author=M. A. Fustier;citation_author=B. Nabholz;citation_author=S. Glemin;citation_title=The bimodal distribution of genic GC content is ancestral to monocot species.;citation_volume=7;citation_year=2015;citation_pmid=25527839;citation_doi=10.1093/gbe/evu278" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. M. Comeron;citation_author=T. B. Guthrie;citation_title=Intragenic Hill-Robertson interference influences selection intensity on synonymous mutations in Drosophila.;citation_volume=22;citation_year=2005;citation_pmid=16120803;citation_doi=10.1093/molbev/msi246" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Dash;citation_author=J. Van Hemert;citation_author=L. Hong;citation_author=R. P. Wise;citation_author=J. A. Dickerson;citation_title=PLEXdb: gene expression resources for plants and plant pathogens.;citation_volume=40;citation_year=2012;citation_pmid=22084198;citation_doi=10.1093/nar/gkr938" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Dekel;citation_author=U. Alon;citation_title=Optimality and evolutionary tuning of the expression level of a protein.;citation_pages=588-592;citation_volume=436;citation_year=2005;citation_issue=7050;citation_pmid=16049495;citation_doi=10.1038/nature03842" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. dos Reis;citation_author=R. Savva;citation_author=L. Wernisch;citation_title=Solving the riddle of codon usage preferences: a test for translational selection;citation_pages=5036-5044;citation_volume=32;citation_year=2004;citation_issue=17;citation_pmid=15448185;citation_doi=10.1093/nar/gkh834" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. A. Drummond;citation_author=C. O. Wilke;citation_title=Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.;citation_pages=341-352;citation_volume=134;citation_year=2008;citation_issue=2;citation_pmid=18662548;citation_doi=10.1016/j.cell.2008.05.042" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_author=S. R. Eddy;citation_author=J. Mistry;citation_title=The Pfam protein families database: towards a more sustainable future.;citation_volume=44;citation_year=2016;citation_pmid=26673716;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=H. Gingold;citation_author=Y. Pilpel;citation_title=Determinants of translation efficiency and accuracy.;citation_pages=1-13;citation_volume=7;citation_year=2011;citation_doi=10.1038/msb.2011.80" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Homma;citation_author=T. Noguchi;citation_author=S. Fukuchi;citation_title=Codon usage is less optimized in eukaryotic gene segments encoding intrinsically disordered regions than in those encoding structural domains.;citation_volume=44;citation_year=2016;citation_pmid=27915289;citation_doi=10.1093/nar/gkw899" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=T. Ikemura;citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system.;citation_pages=389-409;citation_volume=151;citation_year=1981;citation_issue=3;citation_pmid=6175758;citation_doi=10.1016/0022-2836(81)90003-6" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D. A. Kirby;citation_author=S. V. Muse;citation_author=W. Stephan;citation_title=Maintenance of pre-mRNA secondary structure by epistatic selection.;citation_pages=9047-9051;citation_volume=92;citation_year=1995;citation_issue=20;citation_pmid=7568070;citation_doi=10.1073/pnas.92.20.9047" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=A. A. Komar;citation_author=T. Lesnik;citation_author=C. Reiss;citation_title=Synonymous codon substitutions affect ribosome traffic and protein folding during in vitro translation.;citation_pages=387-391;citation_volume=462;citation_year=1999;citation_issue=3;citation_pmid=10622731;citation_doi=10.1016/S0014-5793(99)01566-5" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Kudla;citation_author=A. W. Murray;citation_author=D. Tollervey;citation_author=J. B. Plotkin;citation_title=Coding-Sequence Determinants of Gene Expression in Escherichia coli;citation_pages=255-258;citation_volume=324;citation_year=2009;citation_issue=5924;citation_pmid=19359587;citation_doi=10.1126/science.1170160" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Li;citation_author=C. J. Stoeckert;citation_author=D. S. Roos;citation_title=OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes;citation_pages=2178-2189;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952885;citation_doi=10.1101/gr.1224503" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=W. H. Li;citation_title=Models of nearly neutral mutations with particular implications for nonrandom usage of synonymous codons.;citation_pages=337-345;citation_volume=24;citation_year=1987;citation_issue=4;citation_pmid=3110426;citation_doi=10.1007/BF02134132" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. J. Lipman;citation_author=W. J. Wilbur;citation_title=Interaction of silent and replacement changes in eukaryotic coding sequences.;citation_pages=161-167;citation_volume=21;citation_year=1984;citation_issue=2;citation_pmid=6442990;citation_doi=10.1007/BF02100090" />
<meta name="citation_reference" content="citation_journal_title=Molecular biology reports;citation_journal_abbrev=Mol Biol Rep;citation_author=H. Liu;citation_author=R. He;citation_author=H. Zhang;citation_author=Y. Huang;citation_author=M. Tian;citation_title=Analysis of synonymous codon usage in Zea mays.;citation_pages=677-684;citation_volume=37;citation_year=2010;citation_issue=2;citation_pmid=19330534;citation_doi=10.1007/s11033-009-9521-7" />
<meta name="citation_reference" content="citation_journal_abbrev=Algorithms Mol Biol;citation_author=R. Lorenz;citation_author=S. H. Bernhart;citation_author=C. Höner Zu Siederdissen;citation_author=H. Tafer;citation_author=C. Flamm;citation_title=ViennaRNA Package 2.0.;citation_pages=26-26;citation_volume=6;citation_year=2011;citation_pmid=22115189;citation_doi=10.1186/1748-7188-6-26" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. M. Lowe;citation_author=S. R. Eddy;citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.;citation_volume=25;citation_year=1997;citation_pmid=9023104;citation_doi=10.1093/nar/25.5.0955" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=G. Marais;citation_author=L. Duret;citation_title=Synonymous codon usage, accuracy of translation, and gene length in Caenorhabditis elegans.;citation_pages=275-280;citation_volume=52;citation_year=2001;citation_issue=3;citation_pmid=11428464" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=G. Milia;citation_author=S. Camiolo;citation_author=L. Avesani;citation_author=A. Porceddu;citation_title=The dynamic loss and gain of introns during the evolution of the Brassicaceae.;citation_pages=915-924;citation_volume=82;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=P. Mukhopadhyay;citation_author=S. Basak;citation_author=T. C. Ghosh;citation_title=Differential selective constraints shaping codon usage pattern of housekeeping and tissue-specific homologous genes of rice and Arabidopsis.;citation_volume=15;citation_year=2008;citation_pmid=18827062;citation_doi=10.1093/dnares/dsn023" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. B. Plotkin;citation_author=G. Kudla;citation_title=Synonymous but not the same: the causes and consequences of codon bias.;citation_pages=32-42;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21102527;citation_doi=10.1038/nrg2899" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Porceddu;citation_author=S. Camiolo;citation_title=Spatial analyses of mono, di and trinucleotide trends in plant genes.;citation_pages=e22855-e22855;citation_volume=6;citation_year=2011;citation_issue=8;citation_pmid=21829660;citation_doi=10.1371/journal.pone.0022855" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. Porceddu;citation_author=S. Zenoni;citation_author=S. Camiolo;citation_title=The signatures of selection for translational accuracy in plant genes.;citation_volume=5;citation_year=2013;citation_pmid=23695187;citation_doi=10.1093/gbe/evt078" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. Ressayre;citation_author=S. Glémin;citation_author=P. Montalent;citation_author=L. Serre-Giardi;citation_author=C. Dillmann;citation_title=Introns structure patterns of variation in nucleotide composition in Arabidopsis thaliana and rice protein-coding genes.;citation_volume=7;citation_year=2015;citation_pmid=26450849;citation_doi=10.1093/gbe/evv189" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=M. Ringnér;citation_author=M. Krogh;citation_title=Folding free energies of 5'-UTRs impact post-transcriptional regulation on a genomic scale in yeast.;citation_pages=e72-e72;citation_volume=1;citation_year=2005;citation_issue=7;citation_pmid=16355254;citation_doi=10.1371/journal.pcbi.0010072" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications;citation_pages=1281-1295;citation_volume=15;citation_year=1987;citation_issue=3;citation_pmid=3547335;citation_doi=10.1093/nar/15.3.1281" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=T. M. Tuohy;citation_author=K. R. Mosurski;citation_title=Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes;citation_pages=5125-5143;citation_volume=14;citation_year=1986;citation_issue=13;citation_pmid=3526280;citation_doi=10.1093/nar/14.13.5125" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Slotte;citation_author=J. P. Foxe;citation_author=K. M. Hazzouri;citation_author=S. I. Wright;citation_title=Genome-wide evidence for efficient positive and purifying selection in capsella grandiflora, a plant species with a large effective population size.;citation_volume=27;citation_year=2010;citation_pmid=20194429;citation_doi=10.1093/molbev/msq062" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Šmarda;citation_author=P. Bureš;citation_author=L. Horová;citation_author=I. J. Leitch;citation_author=L. Mucina;citation_title=Ecological and evolutionary significance of genomic GC content diversity in monocots.;citation_volume=111;citation_year=2014;citation_pmid=25225383;citation_doi=10.1073/pnas.1321152111" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Staiger;citation_author=G. G. Simpson;citation_title=Enter exitrons.;citation_pages=136;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. M. Stoebel;citation_author=A. M. Dean;citation_author=D. E. Dykhuizen;citation_title=The cost of expression of Escherichia coli lac operon proteins is in the process, not in the products.;citation_volume=178;citation_year=2008;citation_pmid=18245823;citation_doi=10.1534/genetics.107.085399" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=N. Stoletzki;citation_title=Conflicting selection pressures on synonymous codon use in yeast suggest selection on mRNA secondary structures.;citation_pages=224-224;citation_volume=8;citation_year=2008;citation_pmid=18671878;citation_doi=10.1186/1471-2148-8-224" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Stoletzki;citation_author=A. Eyre-Walker;citation_title=Synonymous codon usage in Escherichia coli: selection for translational accuracy.;citation_volume=24;citation_year=2007;citation_pmid=17101719;citation_doi=10.1093/molbev/msl166" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=P. Tiffin;citation_author=M. W. Hahn;citation_title=Coding sequence divergence between two closely related plant species: Arabidopsis thaliana and Brassica rapa ssp. pekinensis.;citation_pages=746-753;citation_volume=54;citation_year=2002;citation_issue=6;citation_pmid=12029356;citation_doi=10.1007/s0023901-0074-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Signal. Behav.;citation_author=L. E. Vandivier;citation_author=F. Li;citation_author=Q. Zheng;citation_author=M. R. Willmann;citation_author=Y. Chen;citation_title=Arabidopsis mRNA secondary structure correlates with protein function and domains.;citation_pages=e24301-e24305;citation_volume=8;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=L. Wang;citation_author=M. J. Roossinck;citation_title=Comparative analysis of expressed sequences reveals a conserved pattern of optimal codon usage in plants.;citation_pages=699-710;citation_volume=61;citation_year=2006;citation_issue=4-5;citation_pmid=16897485;citation_doi=10.1007/s11103-006-0041-8" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. K.-S. Wong;citation_author=J. Wang;citation_author=L. Tao;citation_author=J. Tan;citation_author=J. Zhang;citation_title=Compositional Gradients in Gramineae Genes;citation_pages=851-856;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12045139;citation_doi=10.1101/gr.189102" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. I. Wright;citation_title=Effects of gene expression on molecular evolution in Arabidopsis thaliana and Arabidopsis lyrata.;citation_volume=21;citation_year=2004;citation_pmid=15201397;citation_doi=10.1093/molbev/msh191" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Zhou;citation_author=M. Weems;citation_author=C. O. Wilke;citation_title=Translationally optimal codons associate with structurally sensitive sites in proteins.;citation_volume=26;citation_year=2009;citation_pmid=19349643;citation_doi=10.1093/molbev/msp070" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Evolutionary Basis of Translational Accuracy in Plants" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="twitter:description" content="Mistranslation errors compromise fitness by wasting resources on nonfunctional proteins. In order to reduce the cost of mistranslations, natural selection chooses the most accurately translated codons at sites that are particularly important for protein structure and function. We investigated the determinants underlying selection for translational accuracy in several species of plants belonging to three clades: Brassicaceae, Fabidae, and Poaceae. Although signatures of translational selection were found in genes from a wide range of species, the underlying factors varied in nature and intensity. Indeed, the degree of synonymous codon bias at evolutionarily conserved sites varied among plant clades while remaining uniform within each clade. This is unlikely to solely reflect the diversity of tRNA pools because there is little correlation between synonymous codon bias and tRNA abundance, so other factors must affect codon choice and translational accuracy in plant genes. Accordingly, synonymous codon choice at a given site was affected not only by the selection pressure at that site, but also its participation in protein domains or mRNA secondary structures. Although these effects were detected in all the species we analyzed, their impact on translation accuracy was distinct in evolutionarily distant plant clades. The domain effect was found to enhance translational accuracy in dicot and monocot genes with a high GC content, but to oppose the selection of more accurate codons in monocot genes with a low GC content." />
<meta name="og-title" property="og:title" content="The Evolutionary Basis of Translational Accuracy in Plants" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/7/2363" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Mistranslation errors compromise fitness by wasting resources on nonfunctional proteins. In order to reduce the cost of mistranslations, natural selection chooses the most accurately translated codons at sites that are particularly important for protein structure and function. We investigated the determinants underlying selection for translational accuracy in several species of plants belonging to three clades: Brassicaceae, Fabidae, and Poaceae. Although signatures of translational selection were found in genes from a wide range of species, the underlying factors varied in nature and intensity. Indeed, the degree of synonymous codon bias at evolutionarily conserved sites varied among plant clades while remaining uniform within each clade. This is unlikely to solely reflect the diversity of tRNA pools because there is little correlation between synonymous codon bias and tRNA abundance, so other factors must affect codon choice and translational accuracy in plant genes. Accordingly, synonymous codon choice at a given site was affected not only by the selection pressure at that site, but also its participation in protein domains or mRNA secondary structures. Although these effects were detected in all the species we analyzed, their impact on translation accuracy was distinct in evolutionarily distant plant clades. The domain effect was found to enhance translational accuracy in dicot and monocot genes with a high GC content, but to oppose the selection of more accurate codons in monocot genes with a low GC content." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
    <title>The Evolutionary Basis of Translational Accuracy in Plants | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8r65WEVAUKMtOgFsszoUCY6O3y-vXbgadIA72EdGUuI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485315","apath":"\/ggg\/7\/7\/2363.atom","pisa":"ggg;7\/7\/2363","ac":{"\/ggg\/7\/7\/2363.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/7\/2363.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2363"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2363"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/7\/2363":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CP63SrZ9IwIsIYLDZNWZtPiydTblAjySUXzZtAmkEygEm4Tbxr+5CHk+oXjRBzmNdboBVzaqMnoGj1Ukal5R2kTuGkilxPnsff3M4u90Nr96d52J67r50eFeQ35O+ZMfIeeVqCCtN9pPX5cBRliB4geDVo6y6srKTd4bXRukgWhKvsDsBHauGKHSTirT5l3Rbh8ojVuNbqP5FdDsPE4yFnfKbxxfyrKndDes5Hc950exzgnQ+XJPWbJIzx\\\\\\\/MalRZeYV87W2uRl0g50ItzDkK7i5lF9QLvejAIImp+sX12QX3sdNNjd+ba9v4GkjkWxXg5dc0JycsEl9mP+ZEGV4i02tbH\\\\\\\/FA0qPjZuY1QxNbq1g5dbLcdiAtz+ekIVpP7ZV3lqyx2swOfgJsHEvb\\\\\\\/1XbJ6iw+mBDbnPmsK\\\\\\\/x2IrJcv9H0cxkn6zmQsOzXXFRJ4b0qDajAv7ppXV1jL1Hx\\\\\\\/P5mXA3irf9gfJ4etVngV9Lof62M\\\\\\\/cECuRJEnz14dthiTZ3PSrsJOvysKwfBeBdB3BCHcSv5DBDbF5Ef4pSrRwtsD33YOAUvfhIJPhNzktW0K5XrfmUl3Q9jfvTe6gVTN4CHtlygWtdkWDlk+vzaXHi++l1NVwAxJYqVR7cU+QOG8UldrTIKetb1i8B0tz0qaHttDVB\\\\\\\/kfckh0CcRj4o\\\\\\\/2Vv2YviAjrVnc0AUsjStyxhlriFnALjhFI6csfUwIeSylGnA0P4oH4wRzR4OgOJzpDCPA5ZlZMg78wIImbP86vD\\\\\\\/AwUJMFvVaRkjJaJolpufB4fb\\\\\\\/F5wLXj8SCOf81MgbTP6uN+KXNqs56XMj7Fu9cUBv4yN3ulFJIoTWR7zRMqNikdJsr2vz5EbYJuwafb2jbq3T3GvDvuxQHbY43\\\\\\\/5IsxC6f1hii03fCAYLQhB1mEeFw+xOipNdyxoCvH3I3zrtGcl8x3J75xFWDBFs2bX65uI2S4ys0SbWCKR+0dka+ARVpNC6bhteo0Jwfub43e6EmYPsodHbpzXclVt\\\\\\\/TOwbuXXrAE5VjR6j6L69f8OXVh9BDddFjHAel+fA5HIztaD56cFWlJHBDZKXc7AkL70feJ6Cnzaq250DBRRbJAmK2YROZn\\\\\\\/5Bt\\\\\\\/4bFjEc4MtCrsQExwTDZ3TbHIQ\\\\\\\/QdkjFKpJxaV6+OXnYHWfv3yfC9SoXMDUl26XS+4T0Fq08rdlo+Uwm8s1FTw\\\\\\\/xp23Y3DBtiQ3liNmBqOQLR4LQVZpMDnXaChOE5IFS50CZ4iD\\\\\\\/j057bajhzuPF92Gugdzcc4TlVa1b2sP\\\u0022,\\\u0022iv\\\u0022:\\\u0022AMCyuJfuOhSNDpga3G\\\\\\\/78g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002217f908da5a9c3ee9c6c05be79645999251d7c454a1c330ecccb9dc26ef81064e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Oxaz6Db4fkjOqzrMD4lRSJPwir7eQ\\\\\\\/nWhozx44w1S5LXShUEcNkJ9HpNY4PWZ+9Zvhap9MLJUpwFjO32vDM9uV64DhSHlYiZOnRXogzceb1uNWbvk3Iv28LnVBeF8QCjUJQJiz7tlReSliL7ox8fkPNfeiu8SXl8Gw4pl5vgBiLEdt16JaM8onhfx8WdPrflbKH4hlEtJ9+sMASXPqCv9m5\\\\\\\/mXWiLYu7vbvUXSmlqQvEmOtvauErJWlLQQJxZwfA8koSRUYcWzqPxnLvkZgrCd4RoI3rDogFyrPeuCPj4jDpDAWB0FIJBiznrles3h\\\\\\\/ZxiqXWz+wuXmOj0KCJNjW7weUMgwuT7iuRbVs\\\\\\\/IcYXw9Zxi6vdOFeHLChzWjVJEdMYn92N1sHNIfdaYjVjDOeK52mXkBOElpyDWHxFs\\\\\\\/j52av0ejy8\\\\\\\/HC+mfd03XCvU4Sed1MPd9sK8Uy8OwGEGDbdCOqcvnI7LRUUEFNKY7e2ZqJIELSzBNsldcncAwTKlfYdUGBMlljHz\\\\\\\/+ZKMFNpsf7EQbvwyVw5UQoKkOsiGHHJq3T+a4FJQiEi\\\\\\\/gLvKMEQKYhxQvqsB7W+dXjjZHNKpRYB+KjEJQ+YKRiwJPFdD75dXBvK7tGoq6KIaotSHLCXK9nUSlkjhyWM199\\\\\\\/xBE7nTPosSCokxeBIGU33fu29buCAPiZYMGfgRr3Kk9ksiYs3k0m3D5X0gSnW40fj6Cg7F8ZLumDnGM1riomvwcWVZeDqIk7QGndLkG4OXpLI49rhlX9jhhZHTv92ZtEiBDG4O\\\\\\\/z34fF5k556O6MA+y89kn\\\\\\\/lqOKRwf4NIMynlM\\\\\\\/rVQLiasvxtKctrERQljTaodLBFXMbaH9wU9yMsxwG452hDTP76S0yhaj2ur+EHhwzbQYb5ktgIzSulC9wt5KZj1FK6RO8Wfl99fCvRlitKVJfy1IJdnAfJDwUZ6rs6\\\\\\\/2TnLemHFSMlkiASn8ZA7fLdNYsvEL4tjUIzybEpX9rimZjUNPHvMsHFulNjgpjKJ6JHdJsADYN6nF+QUZQUyTdRagR44ykNSQuBWBy+mArxw8fqBUcGE1AoIA\\\\\\\/oiu+Ba2UpSkPc7rU\\\\\\\/+wrK9FhwKCAMjvR5zCheqS2MI6oRVR2JT7H9OlTlsNn37wECUNH5MUfLolnWizWJOklXcinvG\\\\\\\/k\\\\\\\/W+FjrpHPm54TaoV\\\\\\\/qHU\\\\\\\/xYEyLcWrsAOA7ILk804aE50Jexiaiz8EhG2FpW53nyU+dcixpTXBGs1+RltH8HgkL\\\\\\\/7DKT48pcT9u1xjpe+HM4ujFbqX9oH2d4Cw8tgYQwafEHYC0WZzAIVo+gRyhTaEsLQZEbVAWi77gFp9MjDa6bGXSgOIIHE+GGNpC9tpNLttoMbdfPMrmdUz6LGj\\\\\\\/f8Hv0sOp8fp3c2T8hzSgm0U2txuzofcNYyobufMLB7GRadB+p3SS+aW+2VE7s1lCXXAB\\\\\\\/7RydoSfR0SKqMyvyQaXbwmmf+f1\\\\\\\/dD6oOxyWqbemlMncjME\\\\\\\/BwWmZL3cEZz8g4y0h2JI0ZfmCAqgY2MDfs3TTRuos5mvl5HDgKGWwUkUBoLV55Z37NEbseh0rApueo0mrmb7OYX4p7z+JjsG63hECSV7TaQEW45biS15Ir2YS++1J6dw7s0fl0NL+gaj1t4h+S\\\\\\\/ebAH0WOqf3SLOeAxRqqeEHVnWkPC5AlM1ZVDanSTRdyD+P3kxBHg3XhIqvWHqTSmW7XHCv\\\\\\\/TlABnF4SNrbrkZbtPB4zD8tH4vw0bwtVzQYLK7uuadXBofLG2Y1lQYo6e38s3agSS3A6NDRpO1AinznT1LhVXETNFcKPJkeIO\\\\\\\/LfD8hsIy1OGerVaAVrzbw5qbKWDVy64rze2hTXdQMLASfxmsA6BsUYAbYAGj\\\\\\\/va63Qo29QNGfLs59qO1tb9qGhOnKg0ML3avKxwFI2NBvvgad7sTNsUFDgMdUY09PAmoLeXZZ2dUuERKrJRXBuhbl28gUBvpyZt+ZmJzMKqu5Zal8mxdW6kNQYDLyM7R4YtmHYrQNRDY\\\\\\\/8D0xCRd9VKWaULzrsRyfmJD8oHFsfdYgWBwC+NUYF2Z3dyEwZ\\\\\\\/K8C2WCjx2t49crK4ca4UgccH38ySIpPNLwFGhJcK32lKD6lgAizsdvzFKBh7OSkLFG3BLx7fsNqduomp6JvhoJiTR9v+tT+gk6XuwRI2exhVJFYW+ltCTIb3fzqUwCBlRV7Cg7btsXDrYCfCp2VHEL5XY8yHS9liFpPpBxBr1RwgQRzqwTK9I83+GrXu+pmJJGYd9WuiWhJ0rswkXBq9RjTaj2tUX+kv5BUz1J9aY4+8XSOv1El1\\\\\\\/YtbmGMRntS7ic7FH6M8CYBY+Op5XwbyHDb\\\\\\\/SEhtd7TDGwNaKw8Wh4ahDREjQ5o\\\\\\\/uBnX8LWSxDH1\\\\\\\/rE8gYIyp4Y+ovBoSW7JINNPpUjETF\\\\\\\/N1ZNmA7UjvP8ICa38agkM2AlkjLQzYKfc5dWZDWvuI17fxjByPRwi9ROvWWMkB2Ksa9ExowRV4ZDoAAsARhqWURyNaFd2tly1wUXfuXuSsDXlik2A8MDCHowp9+ZxkUvmUHShmVOXow5KtufPYUV7PTuhOtsLm7qWwcwHzNBpX2zYMdh8pwuMXDZDJHyu8gDjOYzF6\\\\\\\/S\\\\\\\/sqUoTCDdI705zorhz\\\\\\\/d6VEPS+\\\\\\\/Kv6+FPRcnX\\\\\\\/8UJ758Uts8mJ8HhEgeCNwX+2aPhAxF89W69kO3fPldx0ZLRGZywdqVgFxrN4It8SwetWU9mCJA5J94QVzJeT24NLYAIz0XP63ELjTLScSuar4m0IE24lFD3dCKCs1Hun8BUxSWKei1YXWqM\\\\\\\/4mST3Qag\\\\\\\/lAPnh0rBBQxpjBJgaytxFnqglEQ7zZgI6bMUcJ1eVSuBHvBZKUTLgSpX8iwubtuAmLpXJneZ1dHktfaA+4q9OxdAH41uS8moevj3tEDvoJTdDgrHt3jQVPhNcj1zffIOusyF7ttx9JWHXvee1fMiF9pTJ3da2\\\\\\\/hMU6acQmY2YoDlW6ShPravwoHkNQ1WIKNVxoTLUsinz4D76tC2C8HeIrE9etdWA\\\\\\\/MFJLl\\\\\\\/zNfWsmKhDIWie1zFv\\\\\\\/YICcJxdDuQ+k9X+xNetYMtzf6auAxSZB0tc+xIN3We5reotTIxZ+JUT9KZeV6mdD4E0QoL4zzz6puADnaQUqkCrbi8ZaAcIP5CFHt5gCcSsZJ+\\\\\\\/0+P8FeVcmU6VXe2l3W5PN+K4x0klPHKucCwJfV1iSY\\\\\\\/gt9buKFDuB5fKhk+9qNBufgxmaLe9gAZCirhFsSbt4fVHN6u2HW4XUa8Arr\\\\\\\/4VasThWPkas13vMECs4X9\\\\\\\/pFU1sQ98zL\\\\\\\/u6wSN3P9KBAIEz6XdIJHpvlx70YhGGGg4ZUs9MvtClCDqlHxlsLwKxgDx0\\\\\\\/IhVuquK21e6AaWFV9vANkQd9EP3Z57diPlHptg3TSB2A7OHZPhr1cSm4IGtBlK8MFNr7X5OSefA5zRTCikOAs4BfNOUuDqiPzO6be0PzSBhQUvodJmaP8yrG2+Y44AdaX+FwWJfPf6q5uwRYv\\\\\\\/Lx0dFKOjt1i\\\\\\\/h\\\\\\\/e8\\\\\\\/HwStToPlfCr01x6vAB7pOJSSO+7cEiM5sRXjC+WrmibNwTOIJGFWsDtOC+uO1i+00Iq6BenCOv3C8p\\\\\\\/YdPfuRPVXM5iPiWuwUhcswKWlQj6qvWgEhuTjzH89Do60jFsDmQowFLGuuG9mJTnL83tmpl7yXVSCxOdCNT0d0ep0zx\\\\\\\/Ut4lLxg+hwfeGQZ+eQuXxq6kJ33qbxNpQHzpV22JhRTID2oK5UI5EHNmpjiWQJHgzCxCkswQ1HjKKDoodWgduLu1NKDRIsLeSiK+xhHevYQvwgi7jf8fnYXs7K9XWoRSYKtZHSs9rQLyP+aZS4pV8OP674tep37YpcWFJHVG+dmxDKiDtXIcnnwcWpRI3jQ3LqYrO5Mhk279LA6olekrMkjSAUZDvJGPduTg43lMbv4or1y063IqbWkWGbqcr0FdcER6Kq0VjC6YQzJ4PU2Uc8UQSaHAjn4RUTd6wMYTx4kAwIH0OkttRglOOuhnBK8DK++NmUq3Wve+iG8e5mPDlaP6bQzM8ZHAD5I8p0LNIQNKxC8aiYzd4a9uGu1vK0+YjlWV2QNllmRtb7u6B1q4o1PE1PcOigEdlwHAkvNz+s64mTfKkwqphDpP+Nx3p+8bNnqDPn0sCvoPtACzR+ik6WFKyPudiIiWrqCawZrYmLqDlj+fKvEKhtmt4N7D+WVcJv2o3dgQHRUljn8PQWdspGVaq14\\\\\\\/zWK452fXMlaFZZRfds2qhk9xKOywICiVv90DzT53B3mgcauH3XfYRMDlg6nb56dcdrYrbaMpFE+W55rjvVxBYryvsYn2bhIdG3dcwIRFHpBssEw+o74XcW67DWBRcRbUL6WlgxR3cbtoKYgerbyruRleQDTz9XVmzhOrpTuLmiS07FJ3W4dK1gBv0Ed+7ARYeCjDfEr\\\\\\\/NELo+ReCaXwO21eAIjnYq4\\\\\\\/IxdWBN\\\\\\\/YelUIs\\\\\\\/OfHvElNw\\\\\\\/Ufl85CsV7RQE7Tyy1IAKi\\\\\\\/egn71UgG\\\\\\\/Zu1qAdx4LNcojkm+6bKEq+88BbEqGN9kO+PdmzEFyw\\\\\\\/iuEVWQZfHIgpkXzSWbpdAPtaYqETzSAj8eodBBX6MkmvKapR55GhvxKAI4F5gZRwe\\\\\\\/QMq6KqM\\\\\\\/APpPVtnPzPdWhUksPmHLKWjruJP39eSkPSEbL5BDAZbVa6XazWxjIB6s\\\\\\\/q+1BYOIDgdEwuIVDwyaA0g5QX8qQDvKGqniyyN9LhWTyYwMcmR1ngJLvnuxA0Ph0Hl3uIEnR5pO+tvgHgUp9E\\\\\\\/Kvw78L3Mgk8mZJG1xFd1spRGvC4J9YeW4CI0fbsEMQzCd7I58+Q\\\\\\\/Vr+2D6c8JunygK+84IFyX8s5JOwYebF5oKUcoDkWcZzxalduUE0msscMRQBCH7rRRQPUIjD\\\\\\\/zbt9qLoPdj+C711aa8+x+wNvIlmLp3z6PLTrKzDXb5MkzBOBhSZUi\\\\\\\/0L8X0KYFf11DD3pe60bfMH3xWaDC+ynzurWsmNy5fkAAUqUILIKyzKpEcIgTDr9sMgyRSP+rBl\\\\\\\/7yPA4VbU9FlTfwQMHsMnWvBbkkf7evrrFXYFIFCJpmVXRKTFU1q6wzNeChMZU0KA3MeiCXlbGyUPGaOEk0btCuRfq2bTAF0UniIxsCjsltzIcSw0+vslRpIrF7mht6eoyPR19Clqolu1AIqTj+SU1gRLEEChXz\\\\\\\/iaN67fDEBwos9+4830lzZpDuPpMbaU8z3nG2LGDQ8qxjQiB7ro2ywZSquDYTh009mwgvE2+3ek2S\\\\\\\/qVUgm8PuaJfm6OzffBap8daxatcUORFRIlcmG5a06+zy8XOsUlKhA8p120QPHGUYtFJKS57FFQo44udae0YfNJxeROAol83MXVfeWEFSSmUUjQzebH079Pt9zL\\\\\\\/rxkyxKorl5ESCNWTahVtCi9ew+frLfZFc5tZTm+RDjbOjfdtTs0kp2rPmVGYkTBxkURbfmMMmBWSSRo1SZEMGfi5ez\\\\\\\/WKC4ee8JkNKCKXJDqvb0V5h4Oq2Ai1uvamdhJWeS5ka7bi6itQQb4Mx8qrKOU2nNC1DaJrq1Zq3CzeA3Tr00pVH\\\\\\\/OUqhogdyGprPh23HDcXUQ4pTPlbVcE+SQ0Cf4BxLOmWT8BjzTX82rGjSliv1cjqWBMIYWPsrdXrbPekCOwkpFm+hqQ5\\\\\\\/PHOBQy4NhrBAVo9s2Lfnplks6MXIGCfJYXonfDElebymv9upvX36kWmfp7nqdec\\\\\\\/DODjQdkCMmPCJFL0Qwbxewq65fE7yJR6awmG9D7A1vAZurfqR7Xt2d\\\\\\\/NOXzAQs+ZW4xgNS\\\\\\\/SdQmmqK1ixjYYgHjVvo5yiAZav\\\\\\\/3orclE5tr7IGJkZZUud4KbphoEyWHhyRkUeYWlrRARDF4v52M2IeTd6zFU0fTE\\\\\\\/KFvbIVOJhDvlFdBM0qYWKYHaTdoQFQjboQj6qK2aeGexqgPIi4T9DWPmv13L68raJBlQb9319nLL4gyi8zzoGmD41VLU+jAyyNmMdF08hv3sXJ+dBXApYjb0zvaal6xbovr22oZgoZeyTMGAnTlxejNIMAznG8igpvTTtTXgwpAb\\\\\\\/5O3mhT26dHmRhwiAyNVBmw5DJDF8QVgqNbP1mzFPmPp+3nXsO4+I7w7mcrDhTn50nzmFkRlSgw42D5qJe4WMA\\\\\\\/Ab+JIUfWWWTq15KYIFkZ0TSD9RAYgAOvpVImZ3dCfLHpFcIkPTQlHss\\\\\\\/ulUxRxsaKksydS6jy39sS6o3aSg69d+lHe7f9H0MKFdnRBkjse4JviWmittXitOEm4EJAVCp6RppdMNpn99cm146wgdNUNPF8SCNhxUJ3ySsJZIuxLI69jxDoKx3qd3OOIhVrN4R8pzqrUS33cBqlPWOOX2g2HLl+ktwHDt7W8hcRxOVXSgcvWNUEOK4YiPqvyvd8iQ00hTJpO9vQyipzgQUz3a7OjD7GznBbUrMofr9nUIJqCuDnOPEcixwJFiu3DSlCsxFvxSMh\\\\\\\/hFR62+Ytti1Dhpf0zUsZKeGT1VddMlpM\\\\\\\/UKbadWUpY+B2KEwUs2sIqqkP245Bq0lgSGJXyPaI89Zhe0qQINwDyARZp1wEPV70umdmRuhHFHz3zF9+t1nAUpUFlBGwNpOhk+RmlPspDp9G1iUqv3lG038UKcXxD74hL4Jkeem10U62hCYMwD6qLA\\\\\\\/\\\\\\\/DDGBn3xChnbOlmPTbyWLhxpF2MPN7MKuKg9g\\\\\\\/XN9qkm6ytZrlKFuMbfRQVnJx0L8nsk7qQSlCL4dC+OoSG0dtkvQ8gcDXqtdbD9pJ+jcs6O54dBddkQ8iYFRJ8MCeYTs5Uww8aOpHlZXGT1qPxcIdjH7VGXJEGCBY2hpubYXyrZUyquX57hdnOklPOJyADyorrEd0mNphU+7FI+cmBYr6MJQzAy5wSQDaeRe47Fx5USOg2psq5eQMNYsYzKGMIjER8S8te22V8EOM02Rr0FVVT4+Pua1KoL\\\\\\\/dKH2B6WI+0SYaauLugKOZEM+jFwcPLeokUe2nhpfvcytjY40SN1+peIS\\\\\\\/Up37kyyqWEjyCNVNDKF5gKLkU+NbvMiG8bOBLamIdMoX+\\\\\\\/fyadDf9sTj1YymTTUEE5p8AKZcwv9kmAA9V7W4IzTNu9d7oCEeCFM3gmDhNY6lEvylfPLlmq+\\\\\\\/Q9YfGXgRun2LPGWeLVHDkgJ7PzwmiAzI9ZQxqE9kwlf+VbmBW37GRjk19YMJ89r2Wsonp2iQ3qXAOkaQ7DbsvHcLBdmNcMzWpO4e3UdNmbuco7lEDVONMSKuaWvaGzdTQ7iPyNgAjlsZGQCeCO4dxw3l9wdxdQ4Nicjd0Ytssj8eJVCLeOJ6jGhmemystNZlS7BNU3gMzvlaErn7\\\\\\\/tmyKhYcb4HPOjRjNTzSos3oPcGIQfxiDfsXA13mnnaUBT9oONcGhqR5a6zt3jf1mm0sGNsAQ6zdgmVED0S7Mm2SL3DziDR2m7YpX0R2DOCJ4LE\\\\\\\/0sCARGNvICVJv2kqxTenQwm069R8jSB5H3N82y9o5xJ5cG5mjBqEWL6\\\\\\\/isb\\\\\\\/OOnCZMzCwBZCovkQQZcqflTG19zYNLs0RwY5Ej2S1m1z9U6teD83vIhVUYElrzjm0YfCnh8ObGhrKYNZScoRahnsf5Ss9o3VNF40qv5DkMYflK8kIfRX5Uf3wwUvhfjJtSAzfNOHWtS6a77nYTU2k9n\\\\\\\/Fqljiw2fnoI+LSBaQ7Ef5v0cYd614EVJHkrBz2hD1DDYe6ymznj877zifyHOfmBf2beVm7vTEtiqrbqQRcctQwsRTikXllHMVpQnFOyd6aH0QHNTORD7XQ8TizDwWVrRUgYCG0rksDNHOKnlc\\\\\\\/eHVALY6cXsfKmK\\\\\\\/Z4YbdkkD7jjO7SdtN8\\\\\\\/NeVj7mXdfo6G6I\\\\\\\/18sHnkCNxKflRKtQVEnU349MtkAq6quORoWEdETkD+u+TySfDrUKUwbpwnMTE3D7mFwVTCJANyWjRGf5LHPhv2OirTV6bZDgKj6b7Kdj2WZIo4c8QQS57\\\\\\\/LduA1tAgTQuKgAdJauSPhpkE8xACeDM8UAoHuvd+PNJ7Id\\\\\\\/I9aFy020x8EhpfYj2qO\\\\\\\/o56tBNDaerBqX5G2iwfe58RCL3vQIbvsaGsuMyT9WBjKgpUFjpZGb+fuHRzuDNp+cnN1J1kHe1qp22BRDHqSuqF7\\\\\\\/gdKIe89gzhp4\\\\\\\/TZ6GtnTfw9EXtqTh13S2V1RkVTVD4oB1H5xPa9M0HVWEN1pdqANgd98PF4lAy3Sq3AJBN5AdY1AxTleedb3qJ0OdTKAFrSnzdwF6\\\\\\\/Plla9wSZ9es\\\\\\\/WLG0OEg9VHanGMzHFaa7CYRM2zw0g6pZG96S4hZ9Hbu0jfAeAlqvO0gUDixR96EqJd9wJCRNbEotLjmXWqTZ4+afZAzmON0iZTrnE79YwcK\\\\\\\/Hj\\\\\\\/xBb0a4nrHDlGb48InZDFe0oHi2Hc1hzeMqseL9kX0ZwVQ4fJPyLu78XTDexyg18uiQCq+FZOSzdnahp1XkoFUWRpkAuVTBfv3JxXTpxxZueaZmc9SBwG1uGv1BYbLSfeBFInt81+1h3qzUWNI3lOk902gZs\\\\\\\/Xq5BPeOj6UzFNyh3zA7Mfwjz9zLOZQnfdhboPQ1mU8LP5UfsQu\\\\\\\/OIXd16VqpgUKFZqsXyVtSHhaZuhdXGl+mfAifFnP\\\\\\\/UWUbpeUURpBmcThqDr6KsvNjBOpzk3IuxUvfOs7SA8UVJc+LbTNMBfTK3yCp7q8kN38Ay5FFH0hgTHRnPPuoE3TGVPghdjUyTr7U6rmdioEl1fEVn5il0Rx15coPxrCbqvwbQG71ZBdy2\\\\\\\/mkUHuTiiMqg7E1ZZzzxsf10Nm7HVvUjoF\\\\\\\/BpQEf\\\\\\\/OJt1vbnsXxstoJxpz\\\\\\\/fbbfMB7TINNKzedwljIhb4luwF+ugOd0t8a19CRSgGLNkYieTpXKKLUp9Cu6sGNhAmLUrqsvCbFeTt\\\\\\\/oD3YwPPwZ7Uo\\\\\\\/hM9aBnHCKdBSqp4+2FwrOFcWVcYMA\\\\\\\/zvuBkgDSJ3Halw2i0Y4ldkA0A4vtfzywgIWfxHqfi\\\\\\\/ZnkzBtrvnPlnfjihpIyWueDYTg8v2r7w6XGVBKFAzfoDyPfl0arBxjCk73XNbQ8XtSUbO4F9FZ1NuUd1Gb18j+t9gxT\\\\\\\/Q4LmX7GFPL\\\\\\\/N\\\\\\\/mTukAkbAzj0Yw+wx9jwKVPrKDmCJGMLE5zMMlunCsqtF+sBjh2o\\\\\\\/uDMTcru04xEOfQe3BydnwOmYt11s23r+ze75rE3P6iNX1aRyHJSmroiFEONROef+olPzUxMDuehObomJ9xOZ5mtlvSVKGTQTxaxuoWKHC590oVWrjJ6cfjBw2DbbCAjZdML9B01yPqIeeBOEjblIsMlxuK9tM5FDrq66HyUzyTxCaVPmvYAePFFqOPqiq4Rm9ENR1a4vtt1UJ\\\\\\\/wqiJXfUunVt8LfrtMma6DDxX1LPUdWvN4ByfFLvHXPJxG2ImDLsvpwoB7XNbI6sODloddgRjvGbaD1LoQXZN9Wy+w1KT1kr2kzBWYeKlLQ5cv\\\\\\\/h29aPXwSOE8iS+rsr0yvwmIhUGpCNpsHNuJq8h6+u+3cAvsZKHgTtNkat7fnt+w8DrreQHzg8jTVlKaR0rELpl4WovRdd08fQOp8CUwwlZRSqbIhm9HTjYjocpAyBerz\\\\\\\/OASpqss5E6Mhq2TQnYDhqiDMrdnPU2Em4ayA5BraBjTjcHmPjsOcxBxKT4C7tLqzwx\\\\\\\/fNJeiQW0eLVbR2lVLxAac9V043CrR3DbHjmiY\\\\\\\/Edrp6G7fN7D+8dtFR2ZITLm4Ry+FemiQ4XdjAXO4H5X\\\\\\\/DQ3ZB3WH\\\\\\\/YFxf+SfdJgVdLtyMtNi9QmOAsa7cplnyNkQdM3vPswojqpfH8JqexAYYdaAQWBwOrsP5Z5ePzEwLUteRlkzRCWmfhvGucMjIg1TGKRgkpQIVHnWJfV0qWj6PwEHMLGOgwO88AooCezwNccoKMTA3i1MsyEDiHcyvVOpew1uP48FVR+nIb9JI+VSvvrDGHHUg8zUXz381tmyGqvafQq+XfyMN+hiSxdjip8oxv35kTBsntTO0UWzJDTInBd405gps5ROAy8zta6h4QtQd8h+8owsNeJPt5a9BzuFzga+Z3Y3bnsPkCw5gmU2ixuiEPXjEwPkn9bJ7tJ\\\\\\\/R0h1yfCy84M8hNbUMNEGgNP1HMLJHUt+s+97BdG\\\\\\\/RJZ+lnQEuwK1OdBjDoFw1SiGLBILSHhQlvFVyRzwSXQ\\\\\\\/sBum+7N3joD7GIezpjVl4mzFQHR9v5HfA4VqFzW\\\\\\\/ePgo\\\\\\\/m1zI45FU1Yt5nXoTFJiv0SEeS466G5qRQQqOAG9IefZjsGjIG+6q8lHo9c4kPEwIF91IQH3q4UwPmtI6uT4wqBRyneo5YdPD\\\\\\\/NCtFaMUgn5PGuDRMcjhPXg9uYu+iR4vzXEdGdBO4O7ip287H09kzxeyPYL8srwAjTviQpd5ZQem6AU+nEoaJFkCHsGhg+IcDEZfjPycdLsdxIvB2PAF3QAvkV8mrGsrylEiSV7dmTXPlQ9Eerr71YLWIok4t8rl18NnQROnMBgigq91fDVrDSOUqLBr7YQ+0t9DTRtUzn6DKbXUJYm8LnXz2WInrtDpBLDTf2OLDTMzqX4ZO8CqGGzWhuigZlnz+wI+5pDVgP6xyAYd9fln8bdUdszqU8ZLRU0+xfQO3N4GURyo6sKgYtXPv0lvR7+ablXV+Le1sYj5I\\\\\\\/PLGr3\\\\\\\/OYwAAXF3J8obEksJznK6EnTxv2jW2svYWEHPmoCg09UzTxoQfmH2LmyYUt5go+0mzgCImXWx2aUFuFRukr5HuRmkNAlOI8MGG5SnKSvTgDL8jAAQr9O6jx3YDZo53zGGNbfn34vq1z\\\\\\\/VskHZDqtIx+SK5ghAcuSL+jrHbr\\\\\\\/2N+Z\\\\\\\/5+WUdtQTvNnFyDNJS5\\\\\\\/c0IgKNuk66ZtVnuX6EEuNtqxm2AxOoP5KGw9qnFWH6NAEZeSYVCIoMGyw8739xxV45y8cl4S0gfYA5VuGzAnQkUmIwO8Zb+n9JdYXRfPTPFJacX1yoPT2yw0Yuid2xVF6BY4SSl+\\\\\\\/S2GSnVXQ3mFv3Zbnx+xNkADDD4FSSkqxb1RZRJJ3HAL0wKs5tEbn8Y9JnT0ADxRWIf56a1qSSWIaCG2M2PlJhp7bXP4GaGeg1FpR1U44fpsPA9GL3kVKpaOtI+Na+ZBWWv2fJMSQY42k6TR+HE79sqFP+RO3KPqUgJrEcxoUBRsQPHunscRY0CZJ4FXo8RwygCL1OeTRaUxLIPBgvY0PaEnP0Dvzch5q3k+lk0G0\\\\\\\/HFsQ7FEidFSzQN0KOlLZCtxWi4+wJHs5KvN0ZY81hsJ1LAXDLCjY41X8UZhLx3cef8\\\\\\\/PfC6nuVusCbrTHFb9uDc94brSDpL4OYKkIqzCKp7PKhfduyjzjUPnxISAgqMiolqXpRbl+YYOvi0rW9gLvThVSRwlZQt4OaBoUXpGyv+\\\\\\\/zNogc+sovBvuGYVhz9bL4dqoRIB5aSqLFeftq3y8O95MqtYCIMTpN\\\\\\\/Yv41QNbt2ki78dFq4DbJe8TN0HB399FrCLNHIP81YkbIbX1tZAzAfAKK2C5SXkbNRiiMCEjaqa+FOgBCjX257FhDRrfD+zG8\\\\\\\/wp3O+PER3sEpop2KRfJzJ92+ZILNjoDzkCsqkWVegxbFKfvsPD+YMiKx9Rir3uM4sHnp2yChMZHbnzxt54NrjivMDAU2BSizKTRiH3Jl5VKhbQbqJAk0FNa7byZM9Ko4bZ894R57eIX3DXwaR5sxNoqRLB0Uvb\\\\\\\/Jo8NJA\\\\\\\/A7mk2MBkW6ExM8GwsVKxBLibwMghj3UAtLTh164kuPkjV00zULFcJ8FKB1VrSo2bqbhEPn\\\\\\\/\\\\\\\/P1vIJocHINqsspwS\\\\\\\/WWFSavfxOGb\\\\\\\/RzOCOR7DiX2Mqt\\\\\\\/jt8sQeLuR0roltxs5MjcwN31Fvk6sR42krlq+xJ+ktAOP4hgbVwjIFurt3MdWmJ9YCPzytecplmNjW9h7mDtcCRZK4OqdHG8gyp1BBLVGJd\\\\\\\/m5VrMsIL6KpE3MP88Ev7krTIspMVEnV\\\\\\\/BZxfzHQZflNh8PU+Uovk9Xx89SxVYZO8j3oDJM4a8DbFFfS\\\\\\\/IHRp+qsL37TV0ohE07af1O4Unw47Y1KTvLoI\\\\\\\/KOe21vj4J2WRy\\\\\\\/MQaXEFsz\\\\\\\/Cvcyf57RarLS9cuyB2Zb\\\\\\\/eBeKxA5fssvz7EACEdbEwiltSBu0QBOysm+KbGIdfU2nA7bb\\\\\\\/LYv23Nb10sK9u0fMBDLQtishzpBcPdmARLuCVpRYACFUn\\\\\\\/W1pdytsK9\\\\\\\/SL+p9g41Hcl88CR0WgoXSVgZApfdIsPwxdKYKCv\\\\\\\/OjS0zFQ0dJ3o6tSdkxc32H6igRfL8nvhATcHZhGxXFDdUegfUmCW39qzJa7QXODCV1gUM1EjtOpeDqnmDqheFZcQ0MOCRJWzocqbdSaSrwwHv0aDTH5UoJlD2W9V4fzu5Du9rmTe2NDtO7yaSBatkcnA2SPaF4jautYi3JZmNUZBapAR+BHd6+XWbEfyysw6+JEGS\\\\\\\/w+EMwqBmA7FDhE44SOsDWpB6RkStEr\\\\\\\/StpazF2qkqaiecaeFKJM+Ajzt\\\\\\\/gxXg8NkWUrCvNYrTQGN4Zeo6PjPcXdlhQm5q6w5sICpaV5zN\\\\\\\/cLIDXxhit88SY25NRcr4YFStkPPxOqrfU8cTaGztLPviE3BRVrVn7v\\\\\\\/8Eyt36dXtU6yHSF9zI+zM6EX5leJH1zfIZ\\\\\\\/23q0X9QmA1u6cFNeTTnif42wBb6cKRPWPkkMeU+L1JM\\\\\\\/vNyvMGD0RbttpOL3+D4kBjVmTuNJnsymuqMILXGz88MELPwyr8N+PtXryN+4xi5brLX+d0lFsHn89zVHKa1kA+Bdi47gfzBmFPvGXcCYTUaH0cMFV9SgCJvOJU5HAypDFCilwZYy\\\\\\\/95Ox3fFx5FzArYLeXPz30bPjUzbt2AMd4TFaKQdYbabNcVbfqg96UKR1puHYaQTbR7NkiblWlJeybIcOlOMu4ogq4NFaQm8WlIxJQWuJGsXbfCyA3v9nipyET+mmjVHETRdtWhp3pJPCMjVorh7ys9LFOmwrEpucJiwm7Fd56l3nggsnN5a9isk2hYkw5DqmIJWajWnXgcWGn6tjPzKDT3zVoFaftlR8FFNqBLJbwN5SB5dgPE\\\\\\\/5gWQWPsBmGE\\\\\\\/nAh5wP4JAtYAGzCZRDKsGaheKmubL8AcCy4JfSxqN1b52lmWzuwtRxf69AJCB4WlgT8+GDHdZXfZH+k3AZK8JQoFSTSaTN34jRWM3QBKp9v\\\\\\\/CcdmhInCdRg2WernuGNPc+O9U3dOMpLWQydZj34f5EnYkqRq9\\\\\\\/n5EPsqHyT2nmPIb9z0FTmZgSOFH\\\\\\\/Eh9KGbs4tl9W9h+RGn6V4O+guBjAwiSPNL9VHLmQ0FQ1JwJMaL+LFj0SmJ6u21foPztxA1lLW67kkLRXT43ooavaLkBEKxtL4ye4rpTDBlIcRUTUpeb5PmXxbcv8YHUCBHnhSaRxOJyfoCEK933kEHfOkbhNMm1EoTVnsZCTeScQc8m495\\\\\\\/1vps3eerqvUnWGsx8jMfzNT7JexiTScT1dzLzoSyHYH+WtS9J4Mo7GgMkz492HSh0ZIv2E0PyeROOPdWz91nHq5Xf90oiv7ciyxqq+tv+RYQlWbr5LiUfbuEVJFYuA4lF\\\\\\\/IQLDi4anQ3fkMmcbawHlC7Lf3KfOfH96hcy7x74OSJF1S6GMrqrvSGUxg4OOty4dHVtWHDTjS8vMpRRMxuJ4\\\\\\\/t6G3IWW0CUltT8AMw6PKoFnRCc5dqJm0Zw0Nr9OfYgyN9zeKBRlbXtI3a75lTZ6OF9+ZQ9F6Wk+hlpN5CQvl9k32HWSKeGhF5uQsZoCidgiFsgs0aAhCP7IxbrYZUp+quSf9hUFjviE78vXCP5k\\\\\\\/pu1OndoBU6B8GA1LJrz3iYoeONorPqUy2GqxOfbNSzeJ+oZZ+wGnVxqc4foWK6Ewg3oJLKDb9lZHn3dA+WFEBqRkpXIsPFoMRGMBgtkM\\\\\\\/Qqr4Eqhi7QUgnScw2D0RS1c7Wm4fAp0s5EWvMGDtVv0RXFgkQIPnOJhBlAtCxnxPFQgcjHSd2RBM4m1pf6hiRR8I3hFoSdT47iFwbODLhD2wpZl8qKfCQ7SEZRjJXQBH\\\\\\\/8zNw4hX4x0jaIBcprGYzYj\\\\\\\/XJ5MlazeJvZPxZsumNvYsteTdw3THVqDBaTvGXc7igQz10FoHDR3KG+BF6y36gx+7xz2cFu7NHgHwfAM7VtlPd7ZdvEOOoiUAUMUavczb3zdw6rqIaWSLCAkP1PvvjPphBAXK7CLl4UxSM22LCB9bB+E66Q4awwm6n7ZE0PYbSc7gSZG6CBms78IyrEW05w+fVb2VwOupUtipFV7qGkSKEDW3OY9vOTzFNqTkeXU55dWmZSqrO4T2MhrGtGiHr\\\\\\\/ATLobLdEqKyRLc84TqD\\\\\\\/nmriBxdDxL+EwZJw2pYsfSzYDUxN\\\\\\\/2czVXbLVhDBnb4DCy2jxSH5FP6b02yYHctFfxqYUXVJbe1Am4VgobvU12FYlBgLdhtknCXowAv8yPTw76PrHv619kmQ7DVDVclmhT7bDm1Lx4asm+\\\\\\\/xLJryz3DZtV91jeH\\\\\\\/QeuGRgHLfW1u0X\\\\\\\/ru6m2Q3xJ+vTXawS1nIPDc58\\\\\\\/UtqnBKvTsTKtF7aMda\\\\\\\/oqueXEqnSuUfLDW0UGPTPoyeRxzivZFOw0FhJ42qWJo8IxVTJ5SJB2zDYPcF9eBFABtFIRoMDQ7F35vlvOVg+mQvyNYLsK2UuPo1PCw2jfk94agYvKD3xTRhqDrygrwCFU5UjMr7twgfiarWQOdQWeLVDhAFEjNJ6lCJn8K2iTUKfx3S+\\\\\\\/3OfeSrQMESWmrL4mXRl8Kru2YhjlSwUO38gXjs3Lgy\\\\\\\/AQEJPkxFP5j9YYX1vNCB\\\\\\\/y7IRkpg6t0B+\\\\\\\/9GuoH8JBEatZXeNm14dluefxUZsQ8pfI\\\\\\\/CYFA5Tsqam9lOMj7eQpyog4Cdv5fUFN6sQwFqfcDCK7tEsMfPon7Fzbw0iU2bC\\\\\\\/wcVeq\\\\\\\/BkX3XppyeYm4b1HncauFrbjxcY4VgedgY+QyPGwZQz7IXbbi87eO0djX33rlfV4U1DGsNxKFnuXIuDjt5XjXh6jndxK6v+FBghNwj2kqtVdwE5oc37EoWU2Xj9pcpexC4LAaze0XCMVjg4+lm+hAOylWc8wAI7G7Ddi9fUQ6QGyPp1uoxwwoMrnNpmgkqeB6UvVMqamfzPod+lOuXRkEWRMLWpL5YE1GqKIAmPvTXfJAgB0vzvF9mFkECkaXnhS3\\\\\\\/Bnlxq9YzSmXjpqfxiYZtc2Rw5Vd34W6\\\\\\\/NcKjS6w6Dd3cB8b9f3MegAUK\\\\\\\/eSJ\\\\\\\/oSbQvLOULID7E6VsBs1kVubj6klTn+N+iW0afb4l6Zd0fsAP2VUJa41R8PFAy9oafV\\\\\\\/HH7Xta\\\\\\\/dTMZHIsbm2FHLqZCMrAEM6L+jpAKKdag30lIlNXyQPDvYNoiFfJpCMJ5kM1nQfx5k5fl58ccB4RC5mxUNFn\\\\\\\/5Htwg48B4S4DVCbufR9W3kgA0hf5uK0xu+swG2Uy1IqQIkriHlqfytUUrIaCDp4sjNccrAp\\\\\\\/YLgsHtBG8K0I0EetG88Y21dHqqG60xkrmH1J8XsbvgV4PtRsS7YE7F8KqmEEfAX3P5ToqWrBkyC13WNN89eh0EZABx3a5wHlVpCJU98AU2iFUNwuoI9qfkZrDxtQXz+NGQdyLKHbtMicdNbP14sl0fH\\\\\\\/3jNr3NW5MhQN3Y4A45RqQYGtW1NTcFnQnzhZQTdtYsYM\\\\\\\/TOOqj7mYEbFCPNi7dx7NuvdHif7RwRGe9n9bwckdXyg2o8dciPDobQP2KkeojsGfv\\\\\\\/jxUIf4RmFERDzEJTSxw+2gLqcrZe8ICghCBBF3qKVbkXzNAkrUn\\\\\\\/55T7wV2Vkgg+7hZcG2sOKy6z+Dmk8TdhBvMJf3FnYxc20lEI4eg2Z8S\\\\\\\/NOmi9qeNbOFSnkM66icylKaOL+746XqQgEtlP86jURly0CzZk+1nrnlZz51qJPHi80d5WNPCrSMLr9YED4ur4IdcQMwsNLVSwsgX3btrKyoDtKwwnbj+z7Q6JMtlWQ2XjiptYEPnDX5JDLh05tlACfJVA7\\\\\\\/xayoYXSl3AXq5Wv2yt2VJlJbjMNM56breo7fLuP+PRt18\\\\\\\/Zjj\\\\\\\/bpEEPPh+Oi5BYx4X8NuNZ8pAISSjskCJJoHB2oxdP4jg4SActvytKMNlKL50Bc\\\\\\\/3n3RptJ0s1Kie7MkKRqCM1h0WR3bqgmO3Z8WdK2dQKbZLGh6mpCmT\\\\\\\/oXcRFjIopJAABXrNDcrY4RiBwvY4DAF8uqADVKio1quFoWX7s31S5zIy7xp7tWAJTQUkNO1Bf\\\\\\\/TYZlcPrNLeJi12Vp6N7UC0ViDQXiGfF1bw+o4rUtuoVVf8SN98axhV9sDBoDd\\\\\\\/oBI9Noh57nx5pzLikFjGt6F1epm6j5+GAbA28MVW0Fzkg93EymO+bPglqujY7FYNfQPWtwlsYeDdHqO6s1sOj1wckYofAj78HmTG7chABJy5\\\\\\\/0T2CRfWUBzRclDegX6QkspZa9V6Szfra\\\\\\\/VqsOvLHM+jDjPvEhOpVJ7fd\\\\\\\/hMYKBBpX97jGKiDjE2YpVG9jiCpHwC6+qLZlO19UuFCnWaWhss9VxcbsTMRpUvD4gBdiXM5s57QfuIttEfJoI3HcI31mGMbRVSr48WByA6hmgvO44RLp2PQxtcRPTI4714bq69xjmd9BU\\\\\\\/EZgeLaSN8MrclFl4vzJtn40CDp1x3hsP01EQQ3AUwU8+XCM7c8l3wdeTfhPCS6salUE7fT8R\\\\\\\/1E8QU6pZ6Ez8ILrVXJHs1OLUgOQhGQixH2a79gL7EvmGrE7OkevcmL1YXUBrUhkvdbm8XU1EHSyAkVOMmtdXZi0SU0ZGGOfIgs1ONrZVrK6WYuuDePjF2REY1sq30ujFs4Depj5C1ySiYgg+uHAMh8TOEOQcaXb7B+VDfTZ98FDHhN3TeY77Im8qpS0rbV+ZlsRG9WS8\\\\\\\/JMBx2wbFwK2\\\\\\\/tk4Aqi8x8Vu2gdWKVFDa4i1w0jqwe\\\\\\\/H6YmxhmCr7oKrynTph\\\\\\\/zcx82SJjio6F71XxLhVtOnlBe3aOiqLNcBIphjZvgcnYa1Nqa+xiT7t3CpySlj1jvStiTvAyVB36ziB7ezNYw2EzKyYRGk6FvcUQx+eCZVtoSn2X+1XwckCMksMY73lnWWBmiNKk3b2m9P\\\\\\\/Jp3+QrEAricyN2hAV83kgU\\\\\\\/CRZ\\\\\\\/jKZspMKRjTKfPs9gm2FMRiqcYUQL4LQM\\\\\\\/7Cgw27Vv9XyEySZH26nibLq+SgVaUQXex9S0wI597BP\\\\\\\/Wi5UtV1nWsoiKP4LCGtyiyyY24AOlvayE7rv9Sevw2jG9B10owxjbCQ262FaHn6inVhOhA6BzaUkxC8apDVQHc92vkXfDmw8CUgX6QAD73+gHpIY7zMjQeUd4DvUoNvx2hLmoge2HoMaKJIKAa7Mfdoteq2jbBPxj3FV0ZraAqsAUM3OWgLJe7TJx95xks7THBumoiPhCJ0U3lNU4u1blM53dbfuYNT\\\\\\\/Zz8z7Qo3OOaiPavVg385oxetDhQIApxZy23NLf0Z6lvpdTepzX6lN3W35VD4ZM2dMRumj5E5FaQ\\\\\\\/2Mj3EZaGIzFmgwdkigfhGMcdQwsAPdatyON7bqeaC3mXJUO1zuJEYYrAVDTWeITGb3GWvzzDBK10wMEl2vIqQ86\\\\\\\/vOwLHOVNPbuONtDPRIxeNMNxWY69IRX2WWb5obbd1jWNbRkLMhBCCK36uevlQHQ7XRR6WuG3ycgnsbl+LATZqKzxdMBcFDC8QFJfjkNDfesyO4OvNGGTTgAPkbsXQGbRIYx80Ql2cOBRIZAzQfSIIaxCem6UaxaFNeR8T20h9dVWDONvdUd1X\\\\\\\/bDmjjG5knMqED\\\\\\\/2bwnP1O1HI6zRp4\\\\\\\/DuJHkWWoSZ57vvglnjzKvxA2Xj3YC6YTfTQ0y8wptnZCTuB324bObdlrUsIKMoaMsS56l7L41f3oDWhbi\\\\\\\/64cSvxLIYTiAfNtnlpaAOp3h3f3WK5FnC8Qqb0g2nxx041I\\\\\\\/dLnS4j9D1lK5xAdUE3QGpG3jaBY5rB1jvZ8n9\\\\\\\/QEwJk4SiWH7rlh524EZRp7ntObDzsY7E1hjtn9JlrLJP2XKap\\\\\\\/iONkd46d5SWIEKVu1nQnWQydJEHugDFzD52t2NcDmf8K5qDHuZrysfC7x+w+2i3jCdXKSEwtlTO3XzOIUy0o1pCtsxT1d427Wg8x4ICSTNhknA8k7W89J+JiQo8DIcEN\\\\\\\/aEaO5c5uY0KWTMnWZSTiEUYVyB1OlixvrzhxGLHm86qwXQePB9SYjKIpwdw0JZNMmYgj8QxG75ugb37WJjbv9vCNv9wAJcTck2dwHu3sNLFv4Zy0UaHcqtgRPR\\\\\\\/P0b6qOq6q\\\\\\\/8A8clYya3RRb4bjFuAb+YuJOFFDdRugNxW9nxNMNG9TlCpThLcTBDoWHYjCUclHoGDGXFJoUOsFoigLjf3ASJabRIGbjsdc5OT\\\\\\\/nL1O1fw0RjWKNmZ1aqX4HGgxCQs2v2kaL3xKlW5+AUBHEzqt35nc59RxdwGgmfSESphB32D1lpYFfHO\\\\\\\/uFfgGGbbj6gPnYLg1YWvAqSQVdSZXF0I7k3V7f0w5FL9nyN0MRp1XR7GoBimW\\\\\\\/9oEfQZbVXHN5G5wzNFEsDe7m17Ntc3XP0WzOB62TBJy0vcWw4Hppm89FNlrhzbsVDFu4Wj\\\\\\\/IfUiBsssoRmZ46iAg7Z0DfZdHkUYv0sOslRIrlbscovnulligbM0q6G3V\\\\\\\/uV2xwXjCGa2X97sHZHMbvIrU3nkh0dTuCfK7gWQqT8uWykTrtQGCPjqHEEkmcV4e4pVs4+nGXxkgXdePmx+NmV5M10wzUV\\\\\\\/YmYPOcy\\\\\\\/LWMIpQVIK+nxKwsPSiCnYZUonqENj1njcx7ggP\\\\\\\/8H2WCArtxrkbV4xHQbWK2p3FwTww6aveh9LOGEsVDAsneLpOieOwGm7hitOc1RmLgG76\\\\\\\/ZBUmrU2L4EnqL0WkJUf3Kp41TT4A73NcvSsMVkavtl8SetpUWMGgvXwxV99Fh8CYk3Gq1XtwsiQCh7yFxOAHX1RMk7tTksCcyCcR+YZn2sz3W6NWBUuyCdzbP0R5BUBvvKI3EJBeXa6BV5o9HnA23kQ6AIQ\\\\\\\/yeakODEzabk7cOcjbRF0L8Iq\\\\\\\/zX7t+yhzpp+lZydwKyrbuuwdk2mwc0ZKktlp55UIBw6HXaE\\\\\\\/WlLsWgdnExtONkilO2dJJxN5sxXMu3MSFYPc9Q6It4dA7oMC4ig8IKOnwPPHN3Oc7+uGOI\\\\\\\/FVjEp09VEHNa4JZDdrijZUMqu9z8tGhr10I0azLor52vaAZXAXlfL6hF6IinTREb9Zj\\\\\\\/ueBistc2dvoyCkcvLUlLBr5Yg1NdxY0oIqmVenjPtMSe7bddeEajhDleipzuy88fbcrukkYWTLTOeQ5D58p4tC60+\\\\\\\/R4+RIb1x\\\\\\\/b+LXh9LZYVhwWwuuiPYKbJWFhayuNj3IjInuunKD9B0ENuMyj\\\\\\\/9AGXwejxGUxaYkPUhl\\\\\\\/MRqQaAihTdkaD\\\\\\\/YmUGcZlfcIC0smwIULQjjJfzoZ69WSHTX9WE9W724wi\\\\\\\/55K1myWB77VChX4XQe8i8h35nY\\\\\\\/2WsPOKrTbWCZ81C\\\\\\\/Y\\\\\\\/AiZaB6vJx5hlsYEt\\\\\\\/KqXE2G8sXvjp7yZPOiFwMRNMloo1IsyrAQuRT8AWMj6w6hfeSxfT9qlJir+RXs9zM1yj65dbKIXDKDATiczF5bapwpguHRv09sVbQ0ryY8kmvkGfmDE8qPf+\\\\\\\/j025a2eKiQa8d45kIXW3XgphK+vkm9Y7laEbfBKqPI4nh\\\\\\\/gDPnrVoSwtLk3xG500jazXRzCGv3OClCe8gw45CIcQxSuKcZfJIdNfjqk674gO24aLhvKd+fvYAgIb7al8kI9+NHP8rhLpe9TyK18jRIwp1uO+Lndw35kglHo8TBNJpXFj\\\\\\\/ojdGIbhhowET0ZvDen3s3fER27DUZVyli5YPwuFEgeKwNIQPA+CZBMjl8CN\\\\\\\/hKAJPRGTREOdM3lJORurAkdmmkb9erCkJmxOQcSxIg+cpV4DfBCJOSq39mHhvggikPQVppNif6oi+CfVgfVK\\\\\\\/if7v040fVDdBD3arsDfNNyU0mJrgGSjLFiOriJY5lhpiMq7Np4LbKrqSYalSxJWeLqYJWzVRjJ9TedjOXhpoSQ++poSgZfbk7m3IUkR9GnwN4Lt0s93jDh15t8gWVKPhvkr1r7DRAQqV9FYM474FSv7mWd6vj+JrAKB0QfKUrv4dvfd2Xonx2s9WZ2ghfnGtWyQc8o0VG0XiUKOJN7lBw+7YUUyQEjeJgInW36ajhKcOCdlnrOeEw8DB+saxFIDmqKvlAUAtsjI9tLGrm0xHRgmAynlZAkq3QNuBRjXsl+HXprzZwdAiDh0lQ57M1z4zAVmomAC403eEU\\\\\\\/WFou4VVJx7+C5rSw4BVSDEdId5C+zJl2Ff832ca\\\\\\\/LsnKZ\\\\\\\/S6dbjsODXa7YNqjD2+ib9m5lLxdy8Y1f+tuviCE\\\\\\\/fdGbJ8pK6RsUw8ewW62510CmFMRgwhABdKf6yEEKGowRkyXnqC+de5ZkopLMsJ8EFhJabiIiJsDmMWEiudu+Aty\\\\\\\/b4Er\\\\\\\/6OZ2ljwuTs\\\\\\\/9YJS8F\\\\\\\/BJIDwsZ6ECMiM98\\\\\\\/ctbOwgSkQpGQhZKG5WJDFUCkiNlq+K48d6iP4\\\\\\\/T+dTSegUU9wQImkNsw3yNFLmP1NoL1qRt3uh7N5tJqEJ07IoftU4lsNzewXlMaAwoIPaqYBX7C6Lz1DBsKNUBv6ZSMYwsFsYauJ53l3LRyfbKfy5C+XVkZ9Lu\\\\\\\/na\\\\\\\/3WOju1SA9F1bVM7MBXNnu3McWQPuvMQ6MPatNuJGs\\\\\\\/Xz1yfehNfq6hgxrqKM1UK\\\\\\\/4dkMWT3AV3OxTY\\\\\\\/09s5nyejjKsgnV3dmwQsbgiQHiO\\\\\\\/wesIwIgI6YhHp5gPCOcPV1Qp69YatqsWfDJGA1xNbOh3ZaDiWtcdIEQdJQBMjJGfPczbCcsI\\\\\\\/H7TxRI1w\\\\\\\/dk3\\\\\\\/fKTYyQT6RIVTCjOZYY\\\\\\\/p86rxLAx+rpdtCKrZOMMkjd0F1dKeEajjVFnvnjjGuI7orO5FV50BHAj4JT5dgZWAp\\\\\\\/07+tTlHaLdmj483qzuB96S3W9XzKo835Me21qljIyNyn0VqylLibiu7pI09ZZlSDAY8l2Il54hN9fCstDxQXjb6GBTSKU3EPiUIJjkJehEPaGPQWlRN8imIUHGm\\\\\\\/iNFyY0PBgm58dDr3KMeirPYOtmHZyUxFdK5vMRgGSR08svSSj3qS0TLnVRn+T2Krp5NgV84ZAsN0eX7kdjBXuBazjCAX9mn2OJBGH0pB8khHHYZV8NKjr9YJkMYVA2dCQFpOk0Gs6x2gMOTN3QT4ZhWjMotActlj+ET0gWu0eJq8TE3fomuDVXFBqqha8aeWGR\\\\\\\/PeJZ9nY7IwuZj+6wYjvvNBdP1EDQK1D53kwzGQbrgg0Y7x7qDTpAifRX+4DOfnXiLyidLqcXtVjuPtplTy+pXcm3ZZTJvi9CmKC57+kHbKAQCznYIFZjjssovgEfwaRpIpyVXQ5oaN04OHyEGqD2+TJ0f0WLxYmeZ1BLBlKhmQDH0YAWLn2oWcbUMEkm+5bJwyUFYQUIkUsRE2NGrAWI\\\\\\\/8mZBmI+oA+BnVopJfNsYtfs92q3cOCPKh+U0QCzuLJ+zMHs9llyiWqdQXi3V8DkvZP\\\\\\\/dbU5+Hak69LUaG6gyoGjVfP1ZqfD4kgrY1mhT4aGgUTrZK8ZLpGc7fGVJEpKJn2lhagDWIm2ONEGkvQOqJMcHrhvjskuaeyr99fQo56CuicAQ\\\\\\\/2tmhalpZ9f3diA3iFz6BWdd\\\\\\\/S74ys3Rv2G7Pudh1V7l+Hi4qeNNYZaImwz7Ib45yBSqf+GARmNSOlBlsSZuWLNk2AjFgNjcTqVk1jEyX3O8DxY+r2ZVURwbLi1t6nt3tN1LnLMwukOuGqOP2HL1X9sOwusaa34nZEHDe3dnlRx0n2xqidH9sCehy6tx9VFjRhUX\\\\\\\/l0\\\\\\\/Pr6bmgGvDqHoQpuBv6JBwUynQF24uDvzGWtW4H3l4WUbAk0pOlBT53JnC5+BFgJZWjjjURmy6srYl8OV8bZtDl3wAQm9KGgMS7T\\\\\\\/k9sS2x69Y3Ne6PlL34CPfss55aO5ruuizAVt5OboPsFCyaCMPEjpZerDDow6Zds\\\\\\\/2VODsO7w+kMRd5BoECy8ZYzNQ3622\\\\\\\/WBEfpf8JJMdrM\\\\\\\/0IXP1uUKHOEe4GyHY9cRQAU7Aelw+5weTsxoFr798JF8NZKhlJ1dUupT5auubxJ5575RWWWHv6\\\\\\\/kB+HtDRY1cDxz2hhPENRUw8c5UdGCf8aJ6+coL9G73+dtjjnJrWQ82VWryTt7enVLNXU277WAYe9GNbx2fRSwf8M4lR3AeLoOlUZAwk7OetqPitF+VPFfyDspXLdJFnqm9FbhrhQfvF6YlybHNs3wBbwwInxJ7f1R1AclqaTWP4qkKUBJ5+L91oGmcCMibEgLwbNipJAseMAniC3e7waLIrh3lVzu3gCVQrWpFTRzW92tFTeLT+hwfLAJdggbAErnijg2tVt9WJIaTQfgqU3VbblmYBgI+B7r898y4Lhl7W8kDzOsLke4OpIHSDPDJVcZZ7pcogxZPYNY+JjiS1aqLFdnYGK0ISC0mn7rLVnTJQLSYhVupkIHFMg18VFbM1MNqVFg7hJQmaCV2cO+qsK3hLIy6LW1we3NWYzKTtoa1IA3kkoJYoZLJJHrvg5QVobQ\\\\\\\/BxyPkg5yyyG7lFnLFLy5OzC2D9LcdmXn0vSToQ9balOOzoeIqup\\\\\\\/Bsli9GdRV16tlI9IgJ3VposM96gYQ6xFOT1++lpCdLFC7FFxetDglDWpWUXZO\\\\\\\/tSlEyIyPzYiN6cSqBU17xtsbvs5gLcFW6hDqC5fiI37YK5whIeOkcRJnqmga\\\\\\\/JOqaABFio6xHxykuRFBNiNtHRFKMSXoj46Gi8Fi+OdR7SgWy07AKS+pzVeqesiMDmSspwUsZ0Fwb1iNqxthACiOqsQawjIMfN+iQWlgzRN\\\\\\\/3rPIgjOMLgojwVHSeMdDGFlmWcF26iFdwJyXKmoLUq9LGGAWMFjXLShZM+xpJaNZBxPtX1aNEJH70SFf8oc46HmXtP8YpnQXuYXdcOSjjtfJWb5iNhfl+LGc\\\\\\\/lqskpa9hYv\\\\\\\/QlcbC6OcHxlVUEeDqeNuNpqRGGCp8sa02q5IrmdmYldgdeLlbt\\\\\\\/RWO0znbijcf\\\\\\\/pCojlaylHkUwWLaxjxxHEpI1bCxGYCWWHT1fmQMtF3MDSILHC88eUdAfJwWBx6xKgmGtYdP3IIYiKYAg6YIBMhD58JQ43yjuB\\\\\\\/GjpPmi1lghZr1qSI1BdtxHv\\\\\\\/OUqEHfyzEk8dub8xtVCrFovNBGRiqb5LetCJ76V4dbGJWOEkFU5abDVwfkGn4C1Cj7pIEDRRIc0LCpZmOGGoK4oMrc3TCUOVyGLY9bHqXVDipM3DmlCRKETwGzraGyffWm87UQzGVN3stpoeR527oQRexaAEQngIk79IHuf0nXHlYijO8Hswb4en6WRlwz9sXeOzyJEnhkSvbmBUPeMZQ9nfc0UTkd7QQPffqie+AZXpGZ2reUphS43cLj23C+\\\\\\\/CwluocKHwikv\\\\\\\/nB+g4sqOq46hEZnP2c\\\\\\\/QK9t688Cf0u9Ga\\\\\\\/NU\\\\\\\/2CnbVy9DeJWxA1iQz2G\\\\\\\/WjiTTp4lMJlW\\\\\\\/l+9uOZ24gxScu1DeOsquz+eFnGgTgRAmOSLi50UAg+wJHOhW4lGNPusIR1kY3zEg0YhDbOHvFox5RcQpGeu61+FL\\\\\\\/3GFRkO2h2vTwWKn1axWyNVnhFzrnzzTKfUhLXrPdcLzudk3Wp2sVO1RjX1XYFRielUt25SUau8GiZivlXc2ugcPQZUEp1FtkLlUtzidLpV6EWAz8cFftN+PHBpfLks1RcIbPXeMdL5qrbRHNAmFYApE3C5RZluEpxlGhsQYgJhMGe24ZJioJFCAhK4JoA0QMffYJFTJNTIc5uskU8DOvWCElJAeQFDyGKTuBdfroSxo3O0rFk5TE5Ltrt+8e0p93YQuSSOoUjGSzw1YS9XNByPk6jilBGtFPXDrW5ibAA7nNUKzl0pNjcsSpabgfkPiznb+lY4u2eHIlzSifujqYcAjWpw3EhPWj59uojtaO9DjqM19atFjkk+jqbbF7717M0DchsfISWK59wF1RGBcZJylBUuBRZuKN6uPZdNTcF1in8MC0cnCtTETBmH6zrEU1bf8uBADcTOPmFvljFG5+1PXlFPkY71qcS1nyZLIJbjvWgpJPFFa+T4QJ8M0GPdqDFUte+UUeF4CVUULXoCzq6EhyVYEtFPMrIiM6lFOrQJyob5gY5IjMunL3wOs+zCy9ra1wYqPWcx3of\\\\\\\/64ELczqIYmMpTVkluaNClci80XsUT8BGUMuQsbuAAPs47m9\\\\\\\/R42vaPJ16tFl6sj0LLiTH9PtrZkk7qbo1ZEBlAbc+rA7fDQZruCo1MUXNlKU+4DbF1O4ZnldS9\\\\\\\/i\\\\\\\/c95mz6UuFx\\\\\\\/C0VbTCmNX0VJCpdhzzJBYPPxxrOTaHwwHoGTtEIs9qE5l0sTvt1+gDr33klBq2DPaPo8yX+2L1TQ+N4EVfdCgr\\\\\\\/KZ8YWJX+1Jrn7mmMeyc5kAkAboZKRabN+DIK9FhLGKILZRM7IR20+TpBd0mahYM+w4JS0AY75PeErDRgiRJzWxK39O3\\\\\\\/MD8sNAOyQIBZc27NIP++pGm\\\\\\\/1WwqTavpDkUV0CqRqbRfGMLiW9nssMlsIW8KZnKJ0blOkadl8okOING7bDCjIhHV01t2D8ZduHUoyTfYjxL5RcWymKMOuY4AeqcZ6GZ1mPC1h2k\\\\\\\/S4k+\\\\\\\/bj8qio4bkMymP4qpLvTYmZlTyOgjYB7YgVXHnUlfLVYdAB2l3fFVP8jfyXpmIBdfnKO\\\\\\\/ea2fT1yO5BVnGHTop0I1no21xS3A8WyoI0PSs8ii8FslRGR9VyCORj5IxD86638o6Ncbo3Xw8fnnNGeUJrl\\\\\\\/FsXNpGKw6qvAd5Nn3c2c+SGwEOB\\\\\\\/BRxAeyYiZnyFkS4gss\\\\\\\/txsmxrWiz7rL7tTqGkQt0askhPW0faTmwmi57fG5tB8xgzcWEdgNqbuNgBfn9KTQu2GQ4O3xQWvF0ZN7L82iO7H2xT9e1iSKc1Mj6nNhnTgpLKm3x8qnkUjlOk7c4yCQKPuNdY31lqprzoRqe8S8tYkEe26GaiE8Y1WvWVru\\\\\\\/7vlkDXg0FM8APUTNFWGg33DyPdFufE8FLhjprnp5kBm9NP9DmMgVXkOWdaYGpfjjw9mwjyFNk5jEOemflssYpMrV89nH7dyZua8xOz3SjyAgKngs61J5R\\\\\\\/Ax9L89+jhnWJumFzeHwAGwJTaROCerFwhoc9Vwd3JgfTqtWQz18OTgtq8vhH4rwZ\\\\\\\/\\\\\\\/BXmY7kb\\\\\\\/GLuNrkWFu46cWHawk4G3dsL3bB7gaUE+Aclr5il2kf0UGJru3IYFd9I9q\\\\\\\/PYGD1LuTi\\\\\\\/7UeCE1LUvtsPP39Qv4U2CNTMmMpP379ntAA0vBlEwmtSFnzQj\\\\\\\/d9NheSPxcc6Nj7o1rV+DOg3MTuARGK0Xnu89R\\\\\\\/j0cE9NZLuphV+gp2TSS5vRvXm\\\\\\\/ysUhMboE2S4snhhNjUC8WWDCnw97LHG0OhUBT1ncBlH77XE8lYFn4sv\\\\\\\/oFhIjSv\\\\\\\/stdsa0rzuV81kcaA1SXH669LD1ru2QtwbkvlQ2uc0iuCi\\\\\\\/Uj8uDkO8mhD96w4RYLc6WIA1vkMYbghRCPfEsutpXxFIr2tTwlzAAl01a+iFFPwDDzYRhzBIAQNW2yykZ5iuvzwLyJV+1TuhLFs2MdXrTsIdKF9hlTgMfhmcROhsI2T3Yqv2xwzZAppcdrExAHK7Od\\\\\\\/YHD6By5uO9CnD01zPVzxwn4JDCCbB+mpxFeBcgHSDn09rCSvlxZI4NxQdupJDRhyYUkuu29RmWbi53bcK9wzLnMWy0yVAEBK8L7yYSUQtnRKZRcCW547xiFbad6UsW9Ak22l1cQUzAf214+3Adf9LOCx55y1gzrwY2ockIVDbMJKuzOr6knxM+UMZoWUS\\\\\\\/a0E\\\\\\\/Tq3pyylw5u09mf4EdmXngF3MeJ2LC5fz0FESeMetnMSjBJoFwREzwrXKtVHeMLrOFxX\\\\\\\/8a+3e0ghtXzo0sDcCepi8Z97muyD0yjN7qE5ZvHmjrDhKLGRIu1Ujh2VDTpsr5z7FqgbvbkUGzNkRqzeHcEk9Q2rOQ\\\\\\\/C28MIpWQwI3lczFpJJvWVRtmO3vPQ2k4jYXPWOwmkKe6dtYRGnuVLxRzG1JKSB0LaVHOOaeN+Wl62ZObfogo7SL2WlJ8w8CY7hiQOuQFqHUAnyANKn6XkJS9Hghxk0NfB9dAahUDFVF+C81abr8P1RZOzyDwVjMOCdnKYFgmAimPPyr\\\\\\\/eK6rcsmJ3ZDctYRYDEwKFMQapsvPA+42qfLB+ofXiYA7Y4QR4xm2aKnL5uL2bAcha0zbzK5eopUxDZ4evayoBTvkpZD8j5B5uFc8MfiHCWaIs7vx2fAIX8miYoLGMs6jsDQ0NrpfNgVrxibz4U1H8j+t52CWp1Y0xwIY+d7V9j3oWPbZ1ZiJ5IHkQI6nGVgu8T7A6FBMGu9xK7R\\\\\\\/y3nxeoj35rcVfLw9G7W5IiSrtYOshOGCZcBVHGijqiOwVMND1UQXN9pLDpyDy\\\\\\\/d4LQkhQPYkpkrgA9MmXDAsPVPx7DTpCE0wE9Jeid32qPVjkKGEPCGO6Rvf2\\\\\\\/j9JIdiS\\\\\\\/b1s4TkK62imEMxS8CaN3l9z3VtbnWBPBICgMaLfkHLG5SBVFS\\\\\\\/FRA34bsL1zqR16JMAdfxgXJDDk2b8rSyzk1fxP0jRGP9yTGv9btmlKZFhD1hub1huchCTghycIeKsZfxNYn5wNZuLa0U45\\\\\\\/1R8Xa5HmfP0j4b5lpAkntt11ghKuHO7O7hfaCjuxRs65Zt85conxTUhy2G53N\\\\\\\/+8TITKiY2yt9QbKfm0MkV9oizAoUqnzMs8w+eQZxcFfAEywJysAa7FUZmvpl+STtvvauijPcY3WWEpGfGRHQVgYNh66EpaIXdWIQdaAqbWrmoK8LR21\\\\\\\/Ryr7nxUJFEHDYJ1penwTKaFKXXojX5brSVGrU1y766iAUbJVmx2gb2jLrcod1UTCHxJzkx3Ii8ADwF6OS5KkPQ6BGokPculHP2ktuhW4ALncbDvob28wlsYBIfurqpcsK0lK5uhWbhHbX0vyJWL5J6\\\\\\\/WAG5C25rmr6TDQMGlP4iGv9OXWQ0PoAM06NORbN3mubA4fWUmr4+1mlign3cw7IwdVfTy+FAkLZg0xpMyDeVC87\\\\\\\/+cLALZweJ3k0BZQSHFgW8rP461tzFtq9V4JR4WvMVSX3tiyfFgnyL6tcPgztgYIGV4175CLioTbIyyncqyTedeNt2YWonIU5M5DbiWwTvXXVjqk6\\\\\\\/uPy8J56R3lA1ZslTod+sPwgT\\\\\\\/PCqigtHLtRp2uoGDprqq\\\\\\\/Sl\\\\\\\/RArESIeFV\\\\\\\/PLtD1\\\\\\\/3wAJGV\\\\\\\/fHhn9epCbTPPTdQInHnzEV5xs6ad68nEJU7SmnLRiZOsp22oEtTrjIk5IjIMXfK9yV1ZxI8FpPkPPOqP9pEft0KBPYzttdpD4D+H4f0x\\\\\\\/jxtvc0h3k4E7DCH8vU6DNaJ1FAF2UJ0q0nwtU3LVrED0E1XJB4pGDXIGBujAQE4ZpC0BpM0YAt2HiUquYAJ8gZUCcHN8OyAe0Vp4I2Z16+j8+SeJ1+f4uEqDG9vH567LS5uEOp5ij3+5aqi9dlbDpmMUsRTiSmijsqw3yRKkV0Lgbe0VwYjCTlTLHmcsdCs24LCtcAL86\\\\\\\/HUH4qxDjgAUQuw85WRIa8Xgose5vfCF3fAd9eH2hxfIoSKQ0cp5VQpwuZ5BjAZ\\\\\\\/ZL3\\\\\\\/Fkxnyzf\\\\\\\/PLJJkLEScZ33cJ6kLa7mgabMNXuKU\\\\\\\/4IK4CKleQK\\\\\\\/NZqaKYAswEdH0+8EU3oMNuoF1FYXvo51vsyAkbHk9G+vwZLPbNzv3L+MPHUM27oPMVGsyH3BB5atMjS33Sejn1ga6jVYeVu0arEIx07pNIDecqqnjr77KckyBXOLN1WMSCYXqI23Gi31jmo5ai67VL0No9VMGqaWcm\\\\\\\/j4t5J4ykGP\\\\\\\/gsyPuHx6yu4\\\\\\\/WyVORT948SBxZXnNGU2vYL45B3\\\\\\\/cc3cpB9B0mJhP+7O7sA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022rigHujOWRIONbrQ9Rs7CqQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222b8fb9bdd40a3ec9517f121f5a53b38a4c19b9ed296b4dad7ca8631f2a757b6a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228m4NjEbiIOiUfaaILtB2835YzIb+oPAjAXN7dzdG04ynNktROKYz\\\\\\\/3wZdu6fLVjvJDxRV\\\\\\\/SEuactBjdSn0Xvpry7CbkhwABRnxDvuD1Ye5C+2qdbsrN9r5AG\\\\\\\/uTrlp123LsdpQL+AoCyCTVQgBBrwIiw0OqqNnd6P2ZU6m1E5epQZVVhl0Vp2r8BjHpMpoiMnbWgmtvflItjtNAOjyOG9d9BRS6KYHivt22v85BrcBVhsdz2yzaYrFlqA0YAY1dHFWV9fxDhHD7msjGs9qkcZjrfDXVsuZjAK8+Ox7teTGk6C\\\\\\\/MT40qax9Mje0a+\\\\\\\/pwG7K\\\\\\\/yTtugqNz9QQwwc7DbOSmnisWM+Lh6Z5bys8xRPFPIjUxon34RpvJqx94vnWZdys6kMQyKeJ+HvquxhBqXXpxZuG6PRSU1GUqwSJSsZkiIIZFpgYPBjRm1Z1sJmigr1pt7o311d4BP248NfgrVMKSXyCzBvaXtqIeAR1hd2IBwYgAi\\\\\\\/qakL8HOizV5S8uMAhvwuy\\\\\\\/Z2231QhEBmhTFEFIQYeCcmWl4Mzcl+pOpFmPk6bToqovAHtpe0h1jDxHQuYL\\\\\\\/NqLkcOw0zmYtqUDlRPy1cKfo3\\\\\\\/ooeBVx30+WrvKajxPljmd\\\\\\\/4VIeOcVRgFrZ8lfOeP\\\\\\\/cMJBs+xDxL\\\\\\\/bG5IwgqQg2AehDKNtkhd6mqKc4GN+KPs1AhILFkdOGzX1nAeqkw9tGJt1efmB71kAYjQFlVK5GtTYWUEFhEdQsf9tHQIFYzATiDzc7m0K6RmJQcDhmb\\\\\\\/y2iCASUpP1\\\\\\\/UAqigDKsT9mXYi8viFVuTC1K+u60Bf2ZgAQX0qGopE6yR0vlpIC6ifO4RI+yGvn4ABqEEsqtPbDSCpEJeMkHw9qHrf1S3tcVQwBpF9BC6BPpDrRooyzi03EBkhsGNXCyXtoQH0lqhQ8RzWBRRob6WdsogPqPHAWpwxA1Re\\\\\\\/JamxHLAN1aez6Depm1lL7AMa7Uyx18CRHJZxejwV3Vs71V3OVbUl8cT2ll5BYTvxdbULLHf9q98ii5FCaX3qhIPy++i9X8vgKpOFHXYCgL91YzX6oHfMATbHjos8CZpTUMEC2DJzIH4PUUTkjlP5eqyKuWLxMCcvFIPerJL2VI2QUhf9+7b81EJRnRtGFFi6JX99O\\\\\\\/dRCZ5n9u48P70iFem9Bn\\\\\\\/Oqs3xzKRr3YIpAMaBO1g5Yj9y6yvUr\\\\\\\/IRotVp0oy\\\\\\\/tnkhlvxgm7vVD9Ooi+qN6bxoT+vhm2EIO0fdWyR8bUcmYAnXsxzTqUFvgenEsUDPm1ONlaiuwqAhjcLRkQXl2ODkKsNqFMSEVAk2YH+Zo4KbfhZux4awCIj9a5wwSFla+XyQReTkS689TqGdcQs4OHE2gpQPyA+OSmt+utCwPAZVGqzYPa7MJib1+H5zYzS2iFGnbWfKFklL6JI\\\\\\\/G640DwiOYivSr1OnjUcxuqmXAHYLfLqRgAs1uc3UhO5wYLEbITBNijXyftpzc++ZJSpU79u7W9\\\\\\\/3a\\\\\\\/I2tbMZcjbf70AGrrVqnispj4ZZ3M5rlwYsVDKatCgRoXmN3q7imjwlltMbamFA1jVGVdrWp1K2RxJhUCAWRz1Un6iYNJt7cpQ8Aqdt9dounIAlATdY8uHDKl8A3KZunNPszXD2Ux0kgXF9n6XbjU5sH6eFgWo0QPj5kGhzyjVJj6uzHjLlHCf1PK3d7b1IAGYkeQpE3tEPiDcekSAVUxmgS6Fj8DliqADI1xjNfulPi7e4jmapKvxS\\\\\\\/8Ags4vXxLN4BjiPXkXiQD7VvMxeRG2c+BMp8JxNfS0BwEQiIl\\\\\\\/VWEYqa9ctIy9ocvdVKMkX5gt2LJRkWsMFgK8nWLgJdKpQdT91g5KpFdy+wG13l8SfeEkdAuPRrABXtYYoVJzcmcERk5on\\\\\\\/9+scHjI7WG3xYmIDFN6y4PEm38amUdbtQQ26FPTLIRLNbC32BWKlpAoUeTslpzw\\\\\\\/gCC74sosWGucBN4A4n601O58gnklJhPX+bS3Nq9FAravreL685Y4Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022G2x2ntAx3GkbXh47lNtHPw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c0fbb2c9ff122ae18ae56c99ec0e776a12a205b2f0a91e76f4efc6464d72a522\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485315 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2363" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jH-wh_7gRiJvf49FJuHg7VlNTpz5UaVGOMzcw6pvhsc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2363" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pTwICUTqaPuWvhTmE8oxcxUIPBa1TXG6oQ49qQosaYY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2353" title="A Targeted Capture Linkage Map Anchors the Genome of the Schistosomiasis Vector Snail, Biomphalaria glabrata" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2375" title="Systematic Analysis of the DNA Damage Response Network in Telomere Defective Budding Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485315" id="node485315" data-pisa="ggg;7/7/2363" data-pisa-master="ggg;g3.117.040626" data-apath="/ggg/7/7/2363.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Evolutionary Basis of Translational Accuracy in Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Salvatore Camiolo</span>, <span class="highwire-citation-author" data-delta="1">Gaurav Sablok</span> and <span class="highwire-citation-author" data-delta="2">Andrea Porceddu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2363-2373;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040626 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485315" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Salvatore Camiolo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dipartimento di Agraria, Sezione di Agronomia, Coltivazione Erbacee e Genetica, Università degli Studi di Sassari, 07100, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Salvatore%2BCamiolo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Camiolo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASalvatore%2BCamiolo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gaurav Sablok </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Functional Biology and Climate Change Cluster, University of Technology Sydney, New South Wales 2007, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gaurav%2BSablok%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sablok%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGaurav%2BSablok%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrea Porceddu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dipartimento di Agraria, Sezione di Agronomia, Coltivazione Erbacee e Genetica, Università degli Studi di Sassari, 07100, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BPorceddu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Porceddu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BPorceddu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aporceddu@uniss.it" class="" data-icon-position="" data-hide-link-title="0">aporceddu@uniss.it</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/7/2363" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485315" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2363.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485315" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2363.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485315" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2363.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485315" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/7/2363.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485315" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Mistranslation errors compromise fitness by wasting resources on nonfunctional proteins. In order to reduce the cost of mistranslations, natural selection chooses the most accurately translated codons at sites that are particularly important for protein structure and function. We investigated the determinants underlying selection for translational accuracy in several species of plants belonging to three clades: Brassicaceae, Fabidae, and Poaceae. Although signatures of translational selection were found in genes from a wide range of species, the underlying factors varied in nature and intensity. Indeed, the degree of synonymous codon bias at evolutionarily conserved sites varied among plant clades while remaining uniform within each clade. This is unlikely to solely reflect the diversity of tRNA pools because there is little correlation between synonymous codon bias and tRNA abundance, so other factors must affect codon choice and translational accuracy in plant genes. Accordingly, synonymous codon choice at a given site was affected not only by the selection pressure at that site, but also its participation in protein domains or mRNA secondary structures. Although these effects were detected in all the species we analyzed, their impact on translation accuracy was distinct in evolutionarily distant plant clades. The domain effect was found to enhance translational accuracy in dicot and monocot genes with a high GC content, but to oppose the selection of more accurate codons in monocot genes with a low GC content.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/translational-accuracy" class="hw-term hw-article-keyword hw-article-keyword-translational-accuracy" rel="nofollow">translational accuracy</a></li><li class="kwd"><a href="/keyword/codon-bias" class="hw-term hw-article-keyword hw-article-keyword-codon-bias" rel="nofollow">codon bias</a></li><li class="kwd"><a href="/keyword/protein-domains" class="hw-term hw-article-keyword hw-article-keyword-protein-domains" rel="nofollow">protein domains</a></li><li class="kwd"><a href="/keyword/rna-folding" class="hw-term hw-article-keyword hw-article-keyword-rna-folding" rel="nofollow">RNA folding</a></li></ul><p id="p-3">The translation of genes into proteins is a core activity shared by all living organisms. This molecular process requires the commitment of significant resources, including energy and raw materials (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Stoebel <em>et al.</em> 2008</a>). For this reason, many species have evolved mechanisms to optimize translational efficiency and accuracy. Translational efficiency can be defined as the capacity to produce large amounts of protein while minimizing the associated costs (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dekel and Alon 2005</a>), whereas translational accuracy can be defined as increasing the probability that the resulting protein is error free (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Drummond and Wilke 2008</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Zhou <em>et al.</em> 2009</a>). The redundancy of the genetic code means that most amino acids can be specified by more than one (synonymous) codon and this provides an evolutionary mechanism to refine the efficiency and accuracy of translation while maintaining the same amino acid sequence. Synonymous codons are not used with the same frequency in different genes and species—a phenomenon known as codon bias (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Plotkin and Kudla 2011</a>). Although this can in some cases reflect neutral genetic drift (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li 1987</a>), selection pressure may also be responsible (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Komar <em>et al.</em> 1999</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kudla <em>et al.</em> 2009</a>). The tRNAs representing synonymous codons are not equally abundant in the cytosol and some may even be absent, so the enrichment of genes for codons matching the most abundant tRNAs (known as optimal codons) can increase the speed and efficiency of translation and protein accumulation (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wright 2004</a>). The association between codon bias and gene expression level is generally considered a signature of selection for translational efficiency (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">dos Reis <em>et al.</em> 2004</a>). Similarly, optimal codons are often concentrated in gene sequences that encode structurally or functionally critical residues, which tend to be conserved during evolution, and this is considered a signature of selection for translational accuracy (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Drummond and Wilke 2008</a>). The incorporation of a suboptimal tRNA featuring a one-base mismatch relative to the codon (a near-cognate tRNA) within such sites may attract severe energy-related costs due to the increased likelihood of synthesizing a nonfunctional protein or a misfolded protein that interacts unfavorably with other cellular components to promote aggregation (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gingold and Pilpel 2011</a>). Selection for translational accuracy was first detected and quantified in <em>Drosophila melanogaster</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akashi 1994</a>) and was later confirmed in other species, such as <em>Escherichia coli</em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stoletzki and Eyre-Walker 2007</a>) and several eukaryotes (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Drummond and Wilke 2008</a>).</p><p id="p-4">Although translational selection has been widely studied in plants (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wang and Roossinck 2006</a>), few of these studies have considered signatures of translational accuracy and the impact of additional factors (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Porceddu <em>et al.</em> 2013</a>). For example, the selection of mRNA secondary structures (whose organization relies on the formation of stems and loops) may potentially be involved in translational regulation (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stoletzki 2008</a>). Indeed, stem sequences are expected to be GC-rich because G/C pairs are more stable than A/T pairs (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chan <em>et al.</em> 2009</a>), but most of the optimal codons in plants end with GC, so the accumulation of optimal codons around conserved protein sites may reflect the formation of secondary structures that provide false signals of translational accuracy. Similarly, transcript portions corresponding to protein domains may be more GC-rich because they need to form complex structures that hinder the ribosome, allowing sufficient time for the nascent protein to fold correctly (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vandivier <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Homma <em>et al.</em> 2016</a>). Again, the enrichment of conserved sites within such regions may introduce false signatures of translational accuracy.</p><p id="p-5">In order to better understand the determinants underlying selection for translational accuracy in plants, we extended our previous study (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Porceddu <em>et al.</em> 2013</a>) to several species belonging to three clades: Brassicaceae, Fabidae, and Poaceae. After investigating the conservation of translational accuracy during evolution, the underlying mechanism was investigated in detail by controlling for additional factors, such as selection for mRNA secondary structure and the potential GC enrichment of transcript portions coding for protein domains. The reported results clearly underline the influence of these factors on the precise detection of the translational accuracy signature in higher plants, resulting in effects that are evolutionary conserved within species belonging to the same clade. In this regard, our results suggest careful consideration of the general GC content increase within domains when investigating the occurrence of GC3-rich optimal codons in conserved portions of the transcripts.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sequences and expression data</h3><p id="p-6">FASTA format genomic sequences together with the corresponding gff3 format genic annotation files were downloaded from the Phytozome website (<a href="https://phytozome.jgi.doe.gov/pz/portal.html">https://phytozome.jgi.doe.gov/pz/portal.html</a>). The species included in this study were comprised of five Brassicaceae, namely <em>Arabidopsis thaliana</em> (AT; v. 167, TAIR9), <em>Arabidopsis lyrata</em> (AL; v. 107), <em>Brassica rapa</em> (BR; v. 277), <em>Capsella rubella</em> (CR; v. 183) and <em>Eutrema salsugineum</em> (ES; v. 173); five Fabidae, namely <em>Fragaria vesca</em> (FV; v. 226), <em>Glycine max</em> (GM; v. 275), <em>Medicago truncatula</em> (MT; v. 285), <em>Prunus persica</em> (PP; v. 139), and <em>Phaseolus vulgaris</em> (PV; v. 218); and four Poaceae, namely <em>Brachypodium distachyon</em> (BD; v. 283), <em>Oryza sativa</em> (OS; v. 204), <em>Sorghum bicolor</em> (SB; v. 255), and <em>Zea mays</em> (ZM; v. 284). The two-letter abbreviations for these species are used when we describe interspecies comparisons.</p><p id="p-7">Coding sequences (CDS), mature mRNA sequences, and protein sequences were generated by computing the annotation information using gff2sequence (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Camiolo and Porceddu 2013</a>). Only CDS beginning with a valid start codon (ATG), ending with a canonical stop codon (TGA, TAA, or TAG), and featuring full nucleotide triplets (<em>e.g.</em>, the sequence length was a multiple of three) were used for further analysis. For monocots, the sequence datasets were also assigned as high-GC genes (HGC; GC ≥ 60%) and low-GC genes (LGC; GC &lt; 60%) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mukhopadhyay <em>et al.</em> 2008</a>).</p><p id="p-8">Expression data were retrieved from PlexDB (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dash <em>et al.</em> 2012</a>) and the NCBI GEO website for <em>A. thaliana</em>, <em>M. truncatula</em>, <em>O. sativa</em>, and <em>Z. mays</em> (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS1.docx">Table S1</a>). Only genes with expression levels &gt;100 were considered to be expressed and the corresponding values were log-transformed to minimize the effect of outliers. The average expression level for each gene was calculated by considering only experiments in which that gene was actually expressed (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Camiolo <em>et al.</em> 2009</a>).</p></div><div id="sec-3" class="subsection"><h3>Identification of orthologs and protein alignments</h3><p id="p-9">Orthologs were identified using OrthoMCL (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2003</a>) with an inflation value of four. Orthologs with an alignment coverage achieving &gt;60% identity were retained for further analysis. Alignment files were created for each species by comparing the proteins with the orthologs in all the other species within the evolutionary clade, using MUSCLE v3.8.31 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Edgar 2004</a>) with default parameters. For each pairwise comparison, we chose the orthologous pairs based on the BLAST <em>e</em>-value (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>). In cases of one to many or many to many relationships the pair of loci with the best <em>e</em>-values were analyzed. For three species comparisons (see <em>Phylogeny-based</em> <em>dataset</em>) we used only trios involving reciprocal best orthologous loci. Amino acids that did not vary in the alignment of orthologs were considered to be conserved.</p></div><div id="sec-4" class="subsection"><h3>Identification and analysis of protein domains</h3><p id="p-10">Protein sequences were scanned against the PFAM-A database (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Finn <em>et al.</em> 2016</a>) using HMMscan (<a href="http://hmmer.org/">http://hmmer.org/</a>). The nucleotide composition of domain and nondomain segments of transcripts were analyzed using domain coordinates to extract domain sequences together with their 5′ and 3′ flanking regions from each CDS. Only regions that were at least 100 amino acids long at this stage were retained for further analysis. Any 5′ and 3′ flanking regions featuring additional domain portions were removed from the final dataset. The GC (guanine + cytosine) and GC3 (guanine + cytosine at the third codon position) values were computed for the last 100 codons of the 5′ flanking domain region, the first 100 codons of the 3′ flanking domain, and the first and last 100 codons of each domain, using a sliding window approach (window step = 3 bases, window size = 9 bases). Values from the domain flanking regions were pooled and flagged as nondomain, whereas values from the first and last portion of domains were pooled together and flagged as domain. Wilcoxon tests were applied to determine whether GC and GC3 values diverged significantly between nondomain and domain regions.</p></div><div id="sec-5" class="subsection"><h3>Analysis of mRNA folding</h3><p id="p-11">The mRNA sequences were processed using the Vienna RNAfold package (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lorenz <em>et al.</em> 2011</a>) with default parameters, and each gene was annotated at the single nucleotide level as unpaired loop regions and weakly or strongly paired stem regions. Codons with a strongly paired third base in the predicted folding model were described as S-codons and all others were described as L-codons.</p></div><div id="sec-6" class="subsection"><h3>tRNA gene copy number and identification of optimal codons</h3><p id="p-12">The tRNA gene copy number was determined using tRNAscan (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lowe and Eddy 1997</a>) with default parameters. When the organelle genome was annotated, the corresponding scaffold was removed before running the analysis. If such information was not available, a BLASTn search was performed on the entire genome <em>vs.</em> the mitochondrial and plastid sequences of <em>A. thaliana</em> and the corresponding scaffolds were discarded. Because the tRNA gene copy number is associated with genome size, the relative tRNA-RSCU value was calculated by normalizing the number of tRNA copies in a manner analogous to calculating the relative synonymous codon usage (RSCU) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sharp <em>et al.</em> 1986</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2363/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7/2363/embed/mml-math-1.gif"/></noscript></span></span>where <em>n</em> is the degeneracy of the corresponding codon family.</p><p id="p-13">Optimal codons were identified for four plant species spanning the three investigated clades (<em>A. thaliana</em>, <em>M. truncatula</em>, <em>O. sativa</em>, and <em>Z. mays</em>) using Seforta (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Camiolo <em>et al.</em> 2014</a>). For each species, the 20% most expressed and least expressed genes were extracted. Codons that proved to be statistically more frequent in strongly expressed genes were considered optimal codons. Seforta yields an odds ratio value for each codon that is &gt;1 when it is used preferentially in strongly expressed genes and ≤1 otherwise. The significance of the odds ratio was taken into account when calling the optimal codons. For the monocots species, the analysis was carried out separately for the HGC and LGC datasets.</p></div><div id="sec-7" class="subsection"><h3>Analysis of codon enrichment and translational accuracy</h3><p id="p-14">Seforta was used to investigate the enrichment of each codon compared to its synonymous peers at the constrained and unconstrained sites. We defined constrained sites as those unchanged in protein coding alignments and unconstrained sites as those showing divergence in at least one species. Briefly, Seforta treats each codon as optimal while considering others within the same synonymous family as nonoptimal. The optimal and nonoptimal codons representing conserved and nonconserved sites in each gene are computed to generate a 2 × 2 matrix. The Mantel–Haenszel (MH) test was then used to pool the matrices and to calculate a cumulative odds ratio for each codon (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Akashi 1994</a>). Computation was performed so that codons were assigned an odds ratio &gt;1 if they are preferentially used at conserved sites. We refer to this approach as the original Akashi test. Because this approach identifies unconstrained sites based on the presence of an amino acid substitution, it is possible that the codon bias at these sites is influenced by the nonsynonymous substitution rate. Two further datasets were therefore generated to control for this effect.</p><div id="sec-8" class="subsection"><h4>Phylogeny-based dataset:</h4><p id="p-15">We used an alignment of three species (<em>e.g.</em>, the principal species under analysis, a sister species, and an outgroup species within the same family) to infer the lineage in which the change had occurred. For example, in order to study the translational accuracy in AT (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), we produced alignment for the species AT (species under study), AL (sister species), and BR (outgroup within the family). We considered only sites that were inferred to have mutated in the sister species, thus ruling out the hypothesis that codon bias at these sites could be influenced by nonsynonymous changes in the principal species. We refer to this approach as the phylogeny-based Akashi test (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS2.docx">Table S2</a> for the species triplets used).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic trees corresponding to the clades we analyzed: Arabidopsis thaliana (AT), Arabidopsis lyrata (AL), Brassica rapa (BR), Capsella rubella (CR), Eutrema salsugineum (ES), Fragaria vesca (FV), Glycine max (GM), Medicago truncatula (MT), Prunus persica (PP), Phaseolus vulgaris (PV), Brachypodium distachyon (BD), Oryza sativa (OS), Sorghum bicolor (SB), and Zea mays (ZM)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1755748276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic trees corresponding to the clades we analyzed: &lt;em&gt;Arabidopsis thaliana&lt;/em&gt; (AT), &lt;em&gt;Arabidopsis lyrata&lt;/em&gt; (AL), &lt;em&gt;Brassica rapa&lt;/em&gt; (BR), &lt;em&gt;Capsella rubella&lt;/em&gt; (CR), &lt;em&gt;Eutrema salsugineum&lt;/em&gt; (ES), &lt;em&gt;Fragaria vesca&lt;/em&gt; (FV), &lt;em&gt;Glycine max&lt;/em&gt; (GM), &lt;em&gt;Medicago truncatula&lt;/em&gt; (MT), &lt;em&gt;Prunus persica&lt;/em&gt; (PP), &lt;em&gt;Phaseolus vulgaris&lt;/em&gt; (PV), &lt;em&gt;Brachypodium distachyon&lt;/em&gt; (BD), &lt;em&gt;Oryza sativa&lt;/em&gt; (OS), &lt;em&gt;Sorghum bicolor&lt;/em&gt; (SB), and &lt;em&gt;Zea mays&lt;/em&gt; (ZM).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2363/F1.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/7/2363/F1.medium.gif" width="440" height="200"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485468" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Phylogenetic trees corresponding to the clades we analyzed: <em>Arabidopsis thaliana</em> (AT), <em>Arabidopsis lyrata</em> (AL), <em>Brassica rapa</em> (BR), <em>Capsella rubella</em> (CR), <em>Eutrema salsugineum</em> (ES), <em>Fragaria vesca</em> (FV), <em>Glycine max</em> (GM), <em>Medicago truncatula</em> (MT), <em>Prunus persica</em> (PP), <em>Phaseolus vulgaris</em> (PV), <em>Brachypodium distachyon</em> (BD), <em>Oryza sativa</em> (OS), <em>Sorghum bicolor</em> (SB), and <em>Zea mays</em> (ZM).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h4>Optimality-tied dataset:</h4><p id="p-17">We used a pairwise alignment between orthologous loci, and considered only nonconserved codons that met the following criteria: (i) the same (third) silent base and (ii) the same favored base. For codon pairs that differed at two bases (first and second base), we checked that both the above criteria were met by all potential intermediate changes in all reconstructed pathways between the two codons. We refer to this approach as the optimality-tied Akashi test.</p><p id="p-18">Seforta codon enrichment analysis has been adapted to study the differential use of synonymous codons in a variety of scenarios. In this investigation, we calculated the preferential use of each codon in domain <em>vs.</em> nondomain regions and in stem <em>vs.</em> loop regions. Similarly, we investigated the differential use of synonymous codons in conserved sites within domain <em>vs.</em> nondomain regions and stem <em>vs.</em> loop regions.</p><p id="p-19">Seforta was also used to investigate signatures of translational accuracy in <em>A. thaliana</em>, <em>M. truncatula</em>, <em>O. sativa</em>, and <em>Z. mays</em>. As originally reported (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Camiolo <em>et al.</em> 2014</a>), Seforta implements the Akashi test to investigate whether optimal codons are used preferentially at conserved <em>vs.</em> nonconserved sites. Again, an odds ratio was generated, indicating selection for translational accuracy when the value is significant and &gt;1.</p></div></div><div id="sec-10" class="subsection"><h3>Statistical analysis</h3><p id="p-20">The effective number of codons was calculated using codonw (<a href="http://codonw.sourceforge.net/">http://codonw.sourceforge.net/</a>). The fraction of optimal codons (Fop) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ikemura 1981</a>) was computed using <em>in situ</em> python software. Heat maps and scatterplots, together with the reported statistical analysis (<em>e.g.</em>, Pearson correlation, Spearman correlation, and Wilcoxon and MH tests), were generated using R software (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">R Core Team, 2014</a>).</p></div><div id="sec-11" class="subsection"><h3>Positional effect analysis</h3><p id="p-21">Translational accuracy and the frequency of optimal codons were calculated along the transcript by using an adjacent windows approach. Briefly, each alignment was split into windows of five amino acids. Alignment portions for all genes within the same window were pooled in a superalignment. At the same time, a transcript supersequence was assembled for each window by retrieving the corresponding codons from the original CDS. The resulting artificial alignments and CDS were analyzed by Seforta using the Akashi test and by applying a custom script to calculate the Fop statistics.</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-22"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FileS1.docx">File S1</a> contains a detailed description of all the supplemental files. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FileS2.txt">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FileS3.txt">File S3</a> contain the ortholog genes used in this study. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FileS4.txt">File S4</a> includes the coordinates of the protein domains for all the analyzed species. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FileS5.txt">File S5</a> reports pseudoalignments resulting from the computation of stem and loop structures within the transcripts.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Codon choice and evolutionary constraints at the encoded site</h3><p id="p-23">Codons that are accurately translated should be found preferentially at conserved sites because these are more likely to affect fitness in cases of mistranslation (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Akashi 1994</a>). A common method to highlight signatures of translational accuracy is based on the computation of codon usage at evolutionarily conserved and variable protein sites. However, a number of potential confounding effects can influence this type of analysis, including the gene expression level, mutational bias, and nonsynonymous mutations. Because codon usage is evaluated separately for each CDS, the overall effects of mutational bias or gene expression level can be ruled out <em>a priori</em> (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Akashi 1994</a>). However, a similar conclusion cannot be reached for the effect of nonsynonymous substitutions. Indeed, as pointed out by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lipman and Wilbur (1984</a>), the codon bias at diverged sites may depend on the codon usage before the mutation occurred rather than synonymous codon selection. Such an effect should be particularly relevant for “young” changes for which mutation/selection had, eventually, limited opportunities to reestablish the appropriate favored/unfavored base. We therefore attempted to control for such effects by discriminating among unconstrained sites on the basis of the lineage in which the mutation occurred. In practice, the changes that occurred in this principal lineage were excluded and only those sites that were inferred to have mutated along the sister lineage were considered (phylogeny-based Akashi test, see <em>Materials and Methods</em>). The codon choice at these sites in the investigated species is expected to be influenced solely by natural selection and not by nonsynonymous substitution bias.</p><p id="p-24">We identified conserved and variable amino acid residues in orthologs from several evolutionary clades (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) among the Brassicaceae (representing the malvids: <em>A. thaliana</em>, <em>A. lyrata</em>, <em>B. rapa</em>, <em>C. rubella</em>, and <em>E. salsugineum</em>), Fabidae (representing the fabids: <em>F. vesca</em>, <em>G. max</em>, <em>M. truncatula</em>, <em>P. persica</em>, and <em>P. vulgaris</em>) and Poaceae (representing the monocots: <em>B. distachyon</em>, <em>O. sativa</em>, <em>S. bicolor</em>, and <em>Z. mays</em>). Codon enrichment analysis at the conserved sites was carried out by computing an odds ratio in accordance with the MH test.</p><p id="p-25">For each species, we calculated a mean odds ratio for each codon by averaging its values in all the comparisons we analyzed. For example, the odds ratio for codon TTC was calculated in <em>A. thaliana</em> by averaging the odds ratio values that were found for this codon when analyzing the conserved sites in the alignments [(AT-AL)-BR], [(AT-CR)-BR], [(AT-ES)-BR], and [(AT-BR)-ES], and considering as unconstrained sites those inferred to have changed in the sister species. This approach allowed us to estimate the variability of the calculated odds ratio with respect to the species used for each computation. Average values &gt;1 indicated the overrepresentation of the codon at conserved sites compared to variable sites.</p><p id="p-26">This analysis revealed several codons preferentially found in the conserved sites (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS1.jpg">Figure S1</a>), thus identifying a potential set of accurate codons. Because the evolutionary state of each site was inferred based on changes in related lineages, the effects of differences in the rates and/or direction of adaptive amino acid substitutions among lineages could be raised as a concern. We found a consistent pattern of potential accurate codons regardless of the sister species used to infer unconstrained sites. This suggests that the eventual species-dependent differences in nonsynonymous changes had a limited impact on our analysis. We also compared datasets (see <em>Materials and Methods</em>) that were obtained by the pairwise alignment of orthologous genes (original Akashi test) with those obtained with the phylogeny-based approach (phylogeny-based Akashi test). Notably, all amino acid changes are considered in the former approach regardless of the lineage in which they occurred. The odds ratios calculated using both approaches were strongly correlated (MH test, <em>r</em> = 0.82; <em>P</em> &lt; 0.0001), again confirming the robustness of our phylogeny-driven approach.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Heat map showing the odds ratios for each codon in each species based on Seforta codon enrichment analysis (the average of all pairwise comparisons within each clade). (B) Heat map showing the tRNA-RSCU values in each species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1755748276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Heat map showing the odds ratios for each codon in each species based on Seforta codon enrichment analysis (the average of all pairwise comparisons within each clade). (B) Heat map showing the tRNA-RSCU values in each species.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2363/F2.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/7/2363/F2.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485368" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">(A) Heat map showing the odds ratios for each codon in each species based on Seforta codon enrichment analysis (the average of all pairwise comparisons within each clade). (B) Heat map showing the tRNA-RSCU values in each species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">In general, codons at conserved sites proved to be strongly GC-rich at the third base position in monocots, with a similar but less extreme trend observed in dicots and even the opposite trend in some legumes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS3.docx">Table S3</a>). The preferential use of particular sets of synonymous codons at conserved sites did not emerge in all species with the same intensity. Indeed, we plotted the average deviation (absolute value) of all the calculated odds ratios from the neutral value 1 and found that the fabids were characterized by significantly lower values than the malvids, whereas the monocots featured intermediate values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS2.jpg">Figure S2</a>). The fabids were characterized by the most homogenous trends within each family whereas more variability was observed for monocots and malvids, with <em>B. rapa</em> yielding the lowest values of all the species in its clade.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485489/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485489/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Average GC3 content of optimal and accurate codons in four species belonging to the fabids, malvids, and monocots</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">Similar odds ratios were observed for certain codons in all the species we investigated, including TGC, GTG, and TCT. However, others showed clade-dependent behavior, <em>e.g.</em>, ATC and AAC were strongly overrepresented at conserved sites in the Brassicaceae whereas CGC was overrepresented at conserved sites only in monocots. Certain motifs occurred frequently in the underrepresented codons, including the dinucleotides GT and TA at the second and third codon position for all species. In general, the preference for specific codons at conserved sites proved to be evolutionarily conserved, with fabids, malvids, and monocots clustering in three distinct groups, and <em>B. rapa</em> as the only exception (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>).</p><p id="p-30">If there is a relationship between the abundance of aminoacyl-tRNAs and the time taken to occupy the acceptor site on the ribosome, the codons corresponding to abundant tRNAs are likely to be translocated faster than other codons (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sharp and Li 1987</a>). Because aminoacyl-tRNAs compete at the ribosome acceptor site until the correct one is stably installed, abundant tRNAs (and the corresponding codons) may therefore be associated with a lower mistranslation rate. We used the number of tRNA gene copies found using the tRNAscan tool as a proxy to establish the most abundant isoacceptors in the cytosol. Our results (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>) did not reveal a clear association between codons featuring the most abundant isoacceptors within each synonymous group (tRNA-RSCU) and the preference at codons at conserved sites. However, the relative number of tRNA gene copies has been conserved during evolution and the three plant families partially cluster when the tRNA-RSCU values are used as variables. We ran an MH test on the two matrices used for the cluster analyses presented in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2, A and B</a> (odds ratios and tRNA-RSCU values) to verify the degree of association, resulting in a correlation of 0.279. This value was nonsignificant (<em>P</em> = 0.058 following 100,000 completely random permutations; <em>P</em> = 0.06 when permutating only the corresponding odds ratios between species to preserve the internal correlation structure) but it revealed a potential relationship between these two sets of variables. Indeed, several species showed a significant correlation between the odds ratio and tRNA-RSCU value for each codon (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS4.docx">Table S4</a>).</p></div><div id="sec-15" class="subsection"><h3>Codon usage bias and the protein domain effect</h3><p id="p-31">If the abundance of tRNAs cannot completely account for codon usage bias at evolutionarily conserved sites, additional factors other than accuracy should be taken into consideration. The portions of mRNAs encoding protein domains are significantly more structured than those specifying interdomain regions in <em>A. thaliana</em>, possibly because selection slows down the ribosomal machinery to ensure the correct folding of functionally critical protein regions (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Vandivier <em>et al.</em> 2014</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Homma <em>et al.</em> 2016</a>). To visualize the compositional bias at domain and interdomain regions, we analyzed transcript portions representing protein domains as well as the corresponding nondomain flanking regions. Both the GC3 value and, to lesser extent, the GC content proved to be higher in the domain sequences (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). We also observed a higher number of evolutionarily constrained sites preferentially located within domain regions (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Taken together, such observations suggest that the nucleotide composition within domain regions can significantly influence the codon usage bias.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average GC3, average GC, and percentage of conserved sites within different portions of the transcripts in A. thaliana, M. truncatula, and O. sativa. Position axes refer to the number of sliding windows analyzed (window step = 3 nt/1 amino acid; window size = 9 nt/3 amino acids). Average GC3 and GC values are calculated at a nucleotide level, whereas the percentage of conserved sites refers to the protein coordinates. Red represents the domain regions and blue represents the nondomain regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1755748276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Average GC3, average GC, and percentage of conserved sites within different portions of the transcripts in &lt;em&gt;A. thaliana&lt;/em&gt;, &lt;em&gt;M. truncatula&lt;/em&gt;, and &lt;em&gt;O. sativa&lt;/em&gt;. Position axes refer to the number of sliding windows analyzed (window step = 3 nt/1 amino acid; window size = 9 nt/3 amino acids). Average GC3 and GC values are calculated at a nucleotide level, whereas the percentage of conserved sites refers to the protein coordinates. Red represents the domain regions and blue represents the nondomain regions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2363/F3.medium.gif" width="440" height="141"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/7/2363/F3.medium.gif" width="440" height="141"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485528" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Average GC3, average GC, and percentage of conserved sites within different portions of the transcripts in <em>A. thaliana</em>, <em>M. truncatula</em>, and <em>O. sativa</em>. Position axes refer to the number of sliding windows analyzed (window step = 3 nt/1 amino acid; window size = 9 nt/3 amino acids). Average GC3 and GC values are calculated at a nucleotide level, whereas the percentage of conserved sites refers to the protein coordinates. Red represents the domain regions and blue represents the nondomain regions.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485402/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485402/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Compositional statistics of four plant species belonging to the three clades (<em>A. thaliana</em> for malvids, <em>M. truncatula</em> for fabids, and <em>O. sativa</em> and <em>Z. mays</em> for monocots)</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">Codon enrichment within domain sequences (MH test of domain <em>vs.</em> nondomain regions) and at conserved sites (MH test of conserved <em>vs.</em> variable sites) were not strongly associated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS5.docx">Table S5</a>). This observation indicates that the domain effect cannot be explained simply by the prevalence of conserved sites at these transcript regions. We therefore separated the effects resulting from evolutionary conservation or inclusion within a protein domain. Codon enrichment at conserved and variable sites was studied separately in domain and nondomain regions, resulting in a strong correlation between the codon odds ratios calculated in these two datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS6.docx">Table S6</a>). This result suggests that the evolutionary forces responsible for codon bias when comparing conserved and variable sites act upon both domain and nondomain regions of the transcript. However, the extent of codon enrichment sometimes differed quantitatively between the two regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS3.jpg">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS7.docx">Table S7</a>, a and b), suggesting that codon choice at each site could also depend on its inclusion in or exclusion from a functional protein domain.</p><p id="p-35">Further insight was gained by analyzing codon enrichment at conserved sites within either domain or nondomain regions. If the domain plays a role in codon choice then there should be a difference in codon bias between these datasets, which was indeed the case. Strong differences were especially prevalent in monocots, with codons ending in GC preferentially used at conserved sites within the domain regions of HGC genes and the nondomain regions of LGC genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS4.jpg">Figure S4</a>a and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS7.docx">Table S7</a>c). Several reports have described intragenic compositional patterns within plant genes (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wong <em>et al.</em> 2002</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Porceddu and Camiolo 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ressayre <em>et al.</em> 2015</a>). For example, the GC content at the third codon base decreases along the direction of transcription in monocot genes (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Wong <em>et al.</em> 2002</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Porceddu and Camiolo 2011</a>). It is possible that this effect could influence our analysis if constrained/domain sites are not uniformly distributed along the CDS. To measure the potential impact of this phenomenon, we divided each transcript into three portions and repeated the analysis separately on each portion, using <em>A. thaliana</em> and <em>O. sativa</em> to represent dicots and monocots, respectively. As shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS5.jpg">Figure S5</a>, the results from the entire CDS were generally confirmed in each separate coding portion. The only exception were the nondomain LGC rice genes, where the observed trend was mainly concentrated in the first portions of the transcripts.</p></div><div id="sec-16" class="subsection"><h3>Codon usage bias and mRNA structural constraints</h3><p id="p-36">Secondary structures in mRNAs can also form outside the regions of the transcript corresponding to protein domains (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lorenz <em>et al.</em> 2011</a>) and regulatory structures may be present in other genic portions (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kirby <em>et al.</em> 1995</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ringnér and Krogh 2005</a>), although whether such motifs also occur in CDS remains unclear due to potential functional constraints. Recently, mature mRNA was shown to be under selection for secondary structure in yeast, and the third base of the codon is under selection to maintain mRNA packing, a structural constraint that has a minor impact compared to that observed in the untranslated regions (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Stoletzki 2008</a>). Accordingly, we used RNAfold to calculate the most probable secondary structures of transcripts in our investigation and identified codons with an inferred strongly paired third position, <em>e.g.</em>, those included in stem regions.</p><p id="p-37">Codon enrichment analysis revealed that several codons are used preferentially in stems compared to loops and can be considered as structurally preferred codons (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS5.docx">Table S5</a>). Interestingly, we found that codon preferences in stems and at evolutionarily conserved sites were only marginally associated. Indeed, codons preferentially found in stems are not always the most frequent at evolutionarily conserved sites, and as anticipated, usually end in GC (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS5.docx">Table S5</a>). Furthermore, the stem regions are only slightly enriched in evolutionarily conserved sites compared to loops, and the percentage variation in conserved sites is significant only in dicots (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). These findings suggest that synonymous codon choice at a given conserved site may also depend, albeit marginally, on the stem/loop state. To test this hypothesis, we analyzed codon preferences between conserved and variable sites, looking separately at stem and loop regions, and codon usage bias was observed in both datasets. Interestingly, the relationship between codon usage preference and the evolutionary characteristics of each site was similar in the two datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS6.jpg">Figure S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS7.docx">Table S7</a>, d and e), although codons ending in GC often had a lower odds ratio in stems than loops. We also investigated the effect of structure by studying the relationship between codon choice at conserved sites and their stem/loop state. We found that conserved sites in stems tended to be slightly enriched for codons ending in GC and more strongly enriched for codons ending in AG (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS4.jpg">Figure S4</a>b). These results suggest that structural selection has little if any impact on the detection of signatures of translational accuracy, although it cannot be ruled out altogether.</p></div><div id="sec-17" class="subsection"><h3>Balancing translational accuracy, efficiency, and nonsynonymous mutation</h3><p id="p-38">Selection can maximize the accuracy and efficiency of protein synthesis simultaneously if the synonymous codons that are most accurately translated are also those that are served more quickly by the corresponding tRNAs (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Akashi 1994</a>). Accordingly, codons that are preferred in terms of translational efficiency are also more abundant in <em>D. melanogaster</em> (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Akashi 1994</a>), plants (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Porceddu <em>et al.</em> 2013</a>), humans (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Drummond and Wilke 2008</a>), yeast (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Zhou <em>et al.</em> 2009</a>), and nematodes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Marais and Duret 2001</a>). We confirmed above that codon bias at a given site in plants is influenced not only by its evolutionary state but also by its position with respect to protein domains and mRNA secondary structures. We therefore set out to investigate the balance between translational accuracy and efficiency.</p><p id="p-39">We first determined the optimal codons by statistical analysis of the differences in synonymous codon usage between strongly and weakly expressed genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS5.docx">Table S5</a>). Monocot gene sets were split into HCG and LCG subsets to account for differential selection pressures that may influence these two groups of genes (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Mukhopadhyay <em>et al.</em> 2008</a>). Optimal codons generally proved to be GC-rich at the third base but, exceptionally, the optimal codons in <em>O. sativa</em> LGC genes and all <em>Z. mays</em> genes proved to be strongly AT-rich (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Notably, the optimal codons we determined in <em>Z. mays</em> differed significantly from those previously reported based on the analysis of correspondence (COA) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 2010</a>). Indeed, whereas we directly compared the codon usage in strongly and weakly expressed genes, the COA approach investigates codons that are used in the most biased genes, which are typically expressed at higher than average levels. A correlation analysis between the codon bias (calculated by the mean effective number of codons) and their expression in <em>Z. mays</em> revealed a significant positive association between these two parameters (Pearson <em>r</em> = 0.38, <em>P</em> &lt; 0.0001). In other words, the most biased genes are those expressed at the lowest levels. The MH test was carried out to investigate the use of optimal codons at conserved <em>vs.</em> variable sites, <em>i.e.</em>, the Akashi test (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Akashi 1994</a>), using Seforta on the <em>A. thaliana</em>, <em>M. truncatula</em>, <em>O. sativa</em>, and <em>Z. mays</em> datasets. This confirmed the low odds ratio in plants and revealed only moderate signatures of translational accuracy in these species. The highest values were observed in <em>A. thaliana</em> and <em>O. sativa</em>, and the lowest values were observed in the legume <em>M. truncatula</em>. Interestingly, <em>Z. mays</em> HGC genes yielded a significant odds ratio but the value was negative, which predicts that optimal codons are preferentially used at nonconserved sites rather than conserved sites.</p><p id="p-40">It is possible that nonsynonymous mutations significantly bias the measured effect of selection for translational accuracy. For example, the amino acid change from glycine to alanine following the single base mutation GTA → ATA, may replace the favored codon with a disfavored one or vice versa. To gain insight into the balance between selection for translational accuracy and nonsynonymous mutation, we constructed an additional dataset that included all sites that conserved their condition with regard to selection (both favored or both disfavored) despite the amino acid change. In practice, we considered only variable sites in which the two codons kept the same favored (third) base and the same optimality (optimality-tied Akashi test). Amino acid changes that involved first and second codon base mutations were considered only if all intermediate states in all reconstructed mutational pathways included the same potential silent position bases and the same favored silent bases. As shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS8.docx">Table S8</a>, the results obtained with the whole dataset and the optimality-tied dataset were very similar, suggesting that nonsynonymous mutations have a negligible impact on codon usage bias.</p><p id="p-41">As previously observed, several factors may influence codon usage, such as selection for mRNA stability in transcript regions corresponding to proteins domains (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Vandivier <em>et al.</em> 2014</a>) or mRNA secondary structures (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Stoletzki 2008</a>). To determine the effects of these factors on translational accuracy, we split the CDS into two datasets representing codons in domain and nondomain regions, and signatures for the selection of translational accuracy were sought for these new datasets by applying separate Akashi tests. In <em>A. thaliana</em>, signatures of selection for translational accuracy were found in both the domain and nondomain regions, with odds ratios that were slightly higher in the domain region dataset. A similar picture emerged in <em>M. truncatula</em>, although the odds ratio was not significant in the nondomain regions. Interestingly, signatures of translational accuracy initially detected in the complete monocot LGC datasets disappeared when the domain and nondomain regions were analyzed separately, whereas a significant odds ratio was calculated for the nondomain regions dataset of <em>O. sativa</em> HGC genes, and negative values were recorded once again for both the domain and nondomain regions datasets of <em>Z. mays</em> HGC genes.</p><p id="p-42">In order to control for the selection of mRNA secondary structures, we split the CDS into two datasets representing codons featuring a strongly paired third base (stems) or a weakly paired/nonpaired third base (loops). Again, separate Akashi tests were carried out on each dataset to find signatures for the selection of translational accuracy while controlling for the effect of stem/loop structure. We generally observed slightly higher odds ratios when only codons in loop regions were tested, but in many cases, there was no substantial difference between codons in stems and loops (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS8.docx">Table S8</a>).</p></div><div id="sec-18" class="subsection"><h3>Variation of translational accuracy along the transcript</h3><p id="p-43">Compositional gradients have been widely documented in plant mRNAs (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Porceddu and Camiolo 2011</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahmad <em>et al.</em> 2013</a>). To determine whether this phenomenon is associated with signals of selection for translational accuracy, we split each alignment using an adjacent window approach and looked for gradients of codon enrichment. Variation might also be associated with differential selective pressure for translational efficiency in the direction of the CDS, so we also calculated the Fop along the transcript. <em>A. thaliana</em>, <em>M. truncatula</em>, and <em>O. sativa</em> were selected as representatives of the malvids, fabids, and monocots, respectively.</p><p id="p-44">In <em>A. thaliana</em>, we found that signals for the selection of translational accuracy were stronger in the 5′ portion of the transcript and declined beyond the first 75 codons. Furthermore, the Fop index in this species initially increased and then declined along the CDS, suggesting there is a differential selection pressure on the first portion of the CDS (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Low values of translational accuracy along the transcript were observed in <em>M. truncatula</em>, although the Fop index showed the same peak then decline behavior observed in <em>A. thaliana</em>. A variation in the calculated odds ratio was observed in monocots, with <em>O. sativa</em> HGC genes revealing a pattern similar to <em>A. thaliana</em>, whereas <em>O. sativa</em> LGC genes showed considerable variability, with only few regions revealing a signal for the selection of translational accuracy.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation of Fop and translational accuracy along the transcript. Each value represents an adjacent window containing five amino acids. The corresponding codons were pooled from all the genes to form a supersequence, which has been used for the calculations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1755748276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Variation of Fop and translational accuracy along the transcript. Each value represents an adjacent window containing five amino acids. The corresponding codons were pooled from all the genes to form a supersequence, which has been used for the calculations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2363/F4.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/7/2363/F4.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2363/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485378" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-45" class="first-child">Variation of Fop and translational accuracy along the transcript. Each value represents an adjacent window containing five amino acids. The corresponding codons were pooled from all the genes to form a supersequence, which has been used for the calculations.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><div id="sec-20" class="subsection"><h3>Codon composition at conserved sites</h3><p id="p-46">Selection for translational accuracy is a widely studied phenomenon and identification relies on several assumptions (<a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Akashi 1994</a>) that, we believe, deserve careful investigation. Indeed, for amino acid changes due to single base substitutions (<em>e.g.</em>, the first or second codon base), the base at the third codon position is (initially) determined by the wild-type third codon position. This effect is particularly relevant for young changes for which mutation/selection had limited opportunities to reestablish the appropriate favored/unfavored third base. We controlled for such effects by limiting the analysis to unconstrained sites that changed in closely related lineages. However, the unique evolutionary histories of each species and their phylogenetic distances may influence the validity of these assumptions. For example, if protein evolution is driven mainly by adaptive selection then the simple equivalence between evolutionary conservation and structural/functional importance may be inappropriate. We attempted to account for these effects by considering each species in relation to several sister species. In addition, we demonstrated that the results obtained by considering all changed sites regardless of the lineage in which they occurred did not change the picture substantially.</p><p id="p-47">We investigated the signatures of translational accuracy in three different plant clades (fabids, malvids, and monocots) to gain insight into the evolution of this phenomenon. We started by analyzing the synonymous codon composition at conserved sites using a phylogeny-based approach within each clade, in order to estimate the variability of the results. This revealed that synonymous codon preference at conserved sites is similar within clades and can be used to assign each species to its phylogenetic group.</p><p id="p-48">Interestingly, such accurate codons are not always the same as the optimal codons, <em>i.e.</em>, those used in the most strongly expressed genes or matching the most abundant tRNAs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS5.docx">Table S5</a>). This agrees with the Akashi test results, yielding values only marginally &gt;1 for each of the plant species we investigated (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS8.docx">Table S8</a>), and contrasts with previous studies in other organisms (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Drummond and Wilke 2008</a>). Signatures of translational accuracy are evident in plants not only from the Akashi test results but also by considering the recurrence of certain dinucleotide motifs in these accurate codons. For example, codons containing the notably counterselected dinucleotides TA, GT, or CG at the second and third positions (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Porceddu and Camiolo 2011</a>) were generally underrepresented. Notably, the weak signal of translational accuracy in plants should be considered in the context of its variability along the transcript. Indeed, even the legume genomes, which tended to feature minimal signals of translational accuracy, showed evidence of such signals at the beginning of the CDS in line with observations in other eukaryotes.</p><p id="p-49">We considered what mechanisms might influence the enrichment of certain synonymous codons at conserved sites when they do not mirror the most abundant tRNAs. Similarly, the apparently anomalous odds ratio values observed in <em>Z. mays</em> also called for further investigation. Our evidence suggests that the measurement of signatures of selection for translational accuracy requires the investigation of several different selective forces, some of which are conserved at the clade or species level.</p></div><div id="sec-21" class="subsection"><h3>Structural determinants</h3><p id="p-50">We found that conserved sites are preferentially located in CDS regions corresponding to functional domains, whose GC3 enrichment may promote secondary structures that can delay the translational machinery and thus facilitate correct folding in these critical regions (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Vandivier <em>et al.</em> 2014</a>). This may influence the detection of signatures of translational accuracy because optimal codons in plants are generally GC-rich at the third codon position (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), and may therefore be used preferentially at conserved sites only because these are prevalent within protein domains. We investigated this hypothesis by splitting the datasets into domain and nondomain regions because any differences reflecting the concentration of biased conserved sites in domains should disappear when the two CDS portions are analyzed separately. In <em>A. thaliana</em> and <em>M. truncatula</em>, the signature of translational accuracy proved to be even stronger within protein domains (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS8.docx">Table S8</a>). This confirmed that selection for translational accuracy occurs in these species because the incorporation of inappropriate tRNAs is likely to be counterselected, especially in functionally or structurally critical portions such as domains. The opposite trend was observed in monocots, in which signatures of translational accuracy can be totally explained by the differential distribution of conserved sites between domain and nondomain regions, particularly in LGC genes. This evidence, combined with the independent Akashi test responses along the transcript (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), indicates the absence of a true signature of selection for translational accuracy in <em>O. sativa</em> LGC genes. The observed results may reflect the nonuniform distribution of domain regions within the transcripts, so we also repeated our analysis by splitting the CDS into three portions and performing the Akashi test separately on each. The results were generally in agreement with those based on the entire transcript.</p><p id="p-51">Although the structural constraint of domain regions influences synonymous codon choice at conserved sites in some species, we propose that selection for translational accuracy can, in most cases, have an impact on the CDS. Indeed, we found that optimal codons ending in AT tended to be overrepresented at conserved sites in domain regions compared to nondomain regions, despite the general GC enrichment in the domain regions of most of the species we investigated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS3.jpg">Figure S3</a>). Only HGC genes in monocots showed the opposing trend, <em>i.e.</em>, optimal codons ending in AT tended to be underrepresented in the domain regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS3.jpg">Figure S3</a>). We calculated the difference between the odds ratios of each codon at conserved and variable sites in domain and nondomain regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS3.jpg">Figure S3</a>), and studied its variation with the odds ratio that emerged from the computation of the optimal codons. This revealed a significant correlation between the two variables for most of the species included in this investigation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/TableS9.docx">Table S9</a>), indicating that conserved sites in domains are translated more accurately than conserved sites in nondomain regions.</p><p id="p-52">We also controlled for the enrichment of mRNA stem and loop structures at conserved sites and found significant differences in distribution among the dicots but not the monocots. These results should be considered critically because although RNAfold predicts the most likely RNA secondary structures, empirical data indicate only a small (albeit significant) correlation between the predictions and experimentally determined stabilities (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Stoletzki 2008</a>). Interestingly, the signal of translational accuracy disappeared in the legume <em>M. truncatula</em> when stem and loop regions were analyzed separately. This suggests that selection for mRNA secondary structure may prevent the correct detection of signatures of translational accuracy. However, translational accuracy in <em>M. truncatula</em> was found to be low compared to the whole dataset, and the diminished odds ratios together with the loss of significance may reflect the smaller codon sample size in the separate stem and loop datasets. The general lower odds ratios found in stem regions also suggests there is selection pressure at sites critical for correct folding, <em>i.e.</em>, selection for translational accuracy is more likely to affect loop regions that are not under structural selection.</p><p id="p-53">The structural constraints driving mRNA folding may influence codon usage at conserved sites. Indeed, conserved sites in stems were preferentially represented by codons ending in GC compared to conserved sites in loops (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS5.jpg">Figure S5</a>b). However, this effect is concomitant with the anticipated preferential use of optimal codons ending in AT in stem regions, with translational accuracy dominating the stem/loop effect. Interestingly, the codons most strongly overrepresented in stems compared to loops were those ending in AG, indicating that selection pressure may act to sequester this dinucleotide in stem regions to minimize additional splicing phenomena, as recently reported in <em>A. thaliana</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Staiger and Simpson 2015</a>).</p></div><div id="sec-22" class="subsection"><h3>Genome evolution</h3><p id="p-54">We found that the codon sets at codons at conserved sites were generally conserved within each clade, but <em>B. rapa</em> was exceptional, probably reflecting the unique evolutionary history of this species. <em>B. rapa</em> has experienced a recent whole genome triplication event and the three resulting subgenomes have evolved independently, <em>i.e.</em>, the fractionation rate differs among these three genomic portions (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Milia <em>et al.</em> 2015</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cheng <em>et al.</em> 2012</a>).</p><p id="p-55">Genome evolution may also explain the results we observed in <em>Z. mays</em>, <em>i.e.</em>, the significant negative odds ratio. Monocot genomes experienced a rapid increase in GC content following their separation from the common ancestor of monocots and dicots (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Clement <em>et al.</em> 2015</a>). This increase, together with the compositional gradients along transcripts, may have resulted in the emergence of HGC genes. However, <em>Z. mays</em> features by far the largest genome among the plants we analyzed, and processes such as replication and genome repair are more of a burden on resources when the GC content is particularly high (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Šmarda <em>et al.</em> 2014</a>). This species may therefore be under more selection pressure to reduce the GC content than the other monocots we investigated, with the erosion more evident in GC-rich genes (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Clement <em>et al.</em> 2015</a>). Indeed, <em>Z. mays</em> was the only species featuring AT3-rich optimal codons. Because conserved residues are preferentially found in domain regions that tend to be GC-rich for structural reasons, the AT3-rich optimal codons are more likely to be used in nondomain regions and variable CDS regions, which would explain the negative odds ratio we observed.</p></div><div id="sec-23" class="subsection"><h3>The impact of selection</h3><p id="p-56">We observed a difference in the strength of the translational accuracy signal between <em>A. thaliana</em> and the legume <em>M. truncatula</em>, with the latter showing lower odds ratios (potentially influenced by structural selection as described above) that may indicate more relaxed selection. We recently reported that general genomic enrichment for AT in dicots is balanced by selection-driven enrichment for GC in CDS (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Camiolo <em>et al.</em> 2015</a>). Legumes have the most AT-rich genome, further supporting the more relaxed selection pressure on this species (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Camiolo <em>et al.</em> 2015</a>). Indeed, when investigating the main differences in synonymous codon usage at conserved sites, the presence of several codons ending in TC distinguished the legumes from the other species we analyzed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1/FigureS1.jpg">Figure S1</a>). This dinucleotide is a known mutation hotspot because the deamination of a methylated cytosine would give rise to TT, and it was suppressed in the conserved sites of all the species we analyzed except the legumes.</p><p id="p-57">In contrast, the stronger effect of selection on <em>B. rapa</em> CDS may help to explain the differences between this species and the remaining Brassicaceae. A comparison of <em>B. rapa</em> and <em>A. thaliana</em> transcripts showed that a higher proportion of optimal codons are used in <em>B. rapa</em> CDS, possibly because outcrossing provides more genomic diversity for selection (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tiffin and Hahn 2002</a>). The higher population size of <em>B. rapa</em> may have also hindered the fixation of alterative alleles due to drift, and enhanced the strength of both purifying and positive selection as previously reported for other Brassicaceae species (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Slotte <em>et al.</em> 2010</a>). Moreover, the higher recombination in allogamous species may strengthen the effect of selection, possibly due to a reduced Hill–Robertson effect (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Comeron and Guthrie 2005</a>). Taken together, such evidence may result in an increased number of optimal codons along the transcript, thus flattening the compositional differences between conserved and variable sites.</p></div><div id="sec-24" class="subsection"><h3>Physiological and ecological factors</h3><p id="p-58">Clade-specific codon usage at conserved sites and similar trends in the Akashi test results may also have a physiological basis. For example, the weak signal for the selection of translational accuracy in legumes may reflect the ability of legumes to fix nitrogen, which could offset the costs of mistranslation and permit the relaxation of selection pressure. The opposite trends in <em>O. sativa</em> and <em>Z. mays</em> may be associated with the ecology of these plants. The higher GC content of monocot genomes and the emergence of HGC genes have been proposed as mechanisms to deal more effectively with biotic and abiotic stress (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Šmarda <em>et al.</em> 2014</a>). <em>O. sativa</em> is cultivated under diverse ecological conditions and can deal with more biotic stress conditions than <em>Z. mays</em>, which may explain why the erosion of the GC content in <em>Z. mays</em> HGC genes is not counterselected to maintain this class of genes.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-59">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040626/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-60"><em>Communicating editor: T. Slotte</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-02-17"><span class="received-label">Received </span>February 17, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-05-18"><span class="accepted-label">Accepted </span>May 18, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Camiolo <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.1" data-doi="10.1093/dnares/dss039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahmad</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sablok</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatarinova</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">X. X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evaluation of codon biology in citrus and <em>Poncirus trifoliata</em> based on genomic features and frame corrected expressed sequence tags.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdss039%26rft_id%253Dinfo%253Apmid%252F23315666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIwLzIvMTM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D136%26rft.issue%253D3%26rft.spage%253D927%26rft.epage%253D935%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bin%2BDrosophila%2Bmelanogaster%253A%2Bnatural%2Bselection%2Band%2Btranslational%2Baccuracy.%26rft_id%253Dinfo%253Apmid%252F8005445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM2LzMvOTI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.3" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.4" data-doi="10.1186/1756-0381-6-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camiolo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porceddu</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">gff2sequence, a new user friendly tool for the generation of genomic sequences.</span> <abbr class="cit-jnl-abbrev">BioData Min.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioData%2BMin.%26rft.volume%253D6%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0381-6-15%26rft_id%253Dinfo%253Apmid%252F24020993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0381-6-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24020993&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.5" data-doi="10.1371/journal.pone.0006356"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camiolo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rau</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porceddu</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mutational biases and selective forces shaping the structure of Arabidopsis genes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e6356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCamiolo%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De6356%26rft.epage%253De6356%26rft.atitle%253DMutational%2Bbiases%2Band%2Bselective%2Bforces%2Bshaping%2Bthe%2Bstructure%2Bof%2BArabidopsis%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006356%26rft_id%253Dinfo%253Apmid%252F19633720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006356&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19633720&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camiolo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melito</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porceddu</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Seforta, an integrated tool for detecting the signature of selection in coding sequences.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BRes.%2BNotes%26rft.volume%253D7%26rft.spage%253D240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.7" data-doi="10.1093/dnares/dsv027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camiolo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melito</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porceddu</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">New insights into the interplay between codon bias determinants in plants.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsv027%26rft_id%253Dinfo%253Apmid%252F26546225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIyLzYvNDYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmack</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maliyekkel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A structural interpretation of the effect of GC-content on efficiency of RNA interference.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">S33</span>–<span class="cit-lpage">S37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D10%26rft.spage%253DS33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.9" data-doi="10.1371/journal.pone.0036442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Biased gene fractionation and dominant gene expression among the subgenomes of <em>Brassica rapa</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCheng%26rft.auinit1%253DF.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36442%26rft.epage%253De36442%26rft.atitle%253DBiased%2Bgene%2Bfractionation%2Band%2Bdominant%2Bgene%2Bexpression%2Bamong%2Bthe%2Bsubgenomes%2Bof%2BBrassica%2Brapa.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036442%26rft_id%253Dinfo%253Apmid%252F22567157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22567157&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.10" data-doi="10.1093/gbe/evu278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clement</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fustier</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nabholz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glemin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The bimodal distribution of genic GC content is ancestral to monocot species.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu278%26rft_id%253Dinfo%253Apmid%252F25527839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjcvMS8zMzYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjM2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.11" data-doi="10.1093/molbev/msi246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guthrie</span>  <span class="cit-name-given-names">T. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Intragenic Hill-Robertson interference influences selection intensity on synonymous mutations in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2519</span>–<span class="cit-lpage">2530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi246%26rft_id%253Dinfo%253Apmid%252F16120803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMi8yNTE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.12" data-doi="10.1093/nar/gkr938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dash</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Hemert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wise</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickerson</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PLEXdb: gene expression resources for plants and plant pathogens.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1194</span>–<span class="cit-lpage">D1201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr938%26rft_id%253Dinfo%253Apmid%252F22084198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0MC9EMS9EMTE5NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.13" data-doi="10.1038/nature03842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alon</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Optimality and evolutionary tuning of the expression level of a protein.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">436</span>: <span class="cit-fpage">588</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDekel%26rft.auinit1%253DE.%26rft.volume%253D436%26rft.issue%253D7050%26rft.spage%253D588%26rft.epage%253D592%26rft.atitle%253DOptimality%2Band%2Bevolutionary%2Btuning%2Bof%2Bthe%2Bexpression%2Blevel%2Bof%2Ba%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03842%26rft_id%253Dinfo%253Apmid%252F16049495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16049495&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230788800068&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.14" data-doi="10.1093/nar/gkh834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">dos Reis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savva</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wernisch</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Solving the riddle of codon usage preferences: a test for translational selection.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">5036</span>–<span class="cit-lpage">5044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DReis%26rft.auinit1%253DM.%2Bd.%26rft.volume%253D32%26rft.issue%253D17%26rft.spage%253D5036%26rft.epage%253D5044%26rft.atitle%253DSolving%2Bthe%2Briddle%2Bof%2Bcodon%2Busage%2Bpreferences%253A%2Ba%2Btest%2Bfor%2Btranslational%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh834%26rft_id%253Dinfo%253Apmid%252F15448185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xNy81MDM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.15" data-doi="10.1016/j.cell.2008.05.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDrummond%26rft.auinit1%253DD.%2BA.%26rft.volume%253D134%26rft.issue%253D2%26rft.spage%253D341%26rft.epage%253D352%26rft.atitle%253DMistranslation-induced%2Bprotein%2Bmisfolding%2Bas%2Ba%2Bdominant%2Bconstraint%2Bon%2Bcoding-sequence%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.05.042%26rft_id%253Dinfo%253Apmid%252F18662548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.05.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18662548&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257891700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.16" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzUvMTc5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.17" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coggill</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Pfam protein families database: towards a more sustainable future.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D279</span>–<span class="cit-lpage">D285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F26673716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0NC9EMS9EMjc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.18" data-doi="10.1038/msb.2011.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gingold</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilpel</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Determinants of translation efficiency and accuracy.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.80%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2011.80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.19" data-doi="10.1093/nar/gkw899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Homma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noguchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuchi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Codon usage is less optimized in eukaryotic gene segments encoding intrinsically disordered regions than in those encoding structural domains.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">10051</span>–<span class="cit-lpage">10061</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw899%26rft_id%253Dinfo%253Apmid%252F27915289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0NC8yMS8xMDA1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.20" data-doi="10.1016/0022-2836(81)90003-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the <em>E. coli</em> translational system.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DIkemura%26rft.auinit1%253DT.%26rft.volume%253D151%26rft.issue%253D3%26rft.spage%253D389%26rft.epage%253D409%26rft.atitle%253DCorrelation%2Bbetween%2Bthe%2Babundance%2Bof%2BEscherichia%2Bcoli%2Btransfer%2BRNAs%2Band%2Bthe%2Boccurrence%2Bof%2Bthe%2Brespective%2Bcodons%2Bin%2Bits%2Bprotein%2Bgenes%253A%2Ba%2Bproposal%2Bfor%2Ba%2Bsynonymous%2Bcodon%2Bchoice%2Bthat%2Bis%2Boptimal%2Bfor%2Bthe%2BE.%2Bcoli%2Btranslational%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252881%252990003-6%26rft_id%253Dinfo%253Apmid%252F6175758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(81)90003-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6175758&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MJ92600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.21" data-doi="10.1073/pnas.92.20.9047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muse</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Maintenance of pre-mRNA secondary structure by epistatic selection.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">9047</span>–<span class="cit-lpage">9051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKirby%26rft.auinit1%253DD.%2BA.%26rft.volume%253D92%26rft.issue%253D20%26rft.spage%253D9047%26rft.epage%253D9051%26rft.atitle%253DMaintenance%2Bof%2Bpre-mRNA%2Bsecondary%2Bstructure%2Bby%2Bepistatic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.20.9047%26rft_id%253Dinfo%253Apmid%252F7568070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTIvMjAvOTA0NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.22" data-doi="10.1016/S0014-5793(99)01566-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komar</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesnik</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiss</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Synonymous codon substitutions affect ribosome traffic and protein folding during in vitro translation.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">462</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DKomar%26rft.auinit1%253DA.%2BA.%26rft.volume%253D462%26rft.issue%253D3%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DSynonymous%2Bcodon%2Bsubstitutions%2Baffect%2Bribosome%2Btraffic%2Band%2Bprotein%2Bfolding%2Bduring%2Bin%2Bvitro%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252899%252901566-5%26rft_id%253Dinfo%253Apmid%252F10622731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(99)01566-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10622731&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084151200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.23" data-doi="10.1126/science.1170160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollervey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Coding-sequence determinants of gene expression in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKudla%26rft.auinit1%253DG.%26rft.volume%253D324%26rft.issue%253D5924%26rft.spage%253D255%26rft.epage%253D258%26rft.atitle%253DCoding-Sequence%2BDeterminants%2Bof%2BGene%2BExpression%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1170160%26rft_id%253Dinfo%253Apmid%252F19359587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjM2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.24" data-doi="10.1101/gr.1224503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckert</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2178%26rft.epage%253D2189%26rft.atitle%253DOrthoMCL%253A%2BIdentification%2Bof%2BOrtholog%2BGroups%2Bfor%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1224503%26rft_id%253Dinfo%253Apmid%252F12952885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjE3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.25" data-doi="10.1007/BF02134132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Models of nearly neutral mutations with particular implications for nonrandom usage of synonymous codons.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DLi%26rft.auinit1%253DW.%2BH.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D337%26rft.epage%253D345%26rft.atitle%253DModels%2Bof%2Bnearly%2Bneutral%2Bmutations%2Bwith%2Bparticular%2Bimplications%2Bfor%2Bnonrandom%2Busage%2Bof%2Bsynonymous%2Bcodons.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02134132%26rft_id%253Dinfo%253Apmid%252F3110426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02134132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3110426&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G990300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.26" data-doi="10.1007/BF02100090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilbur</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Interaction of silent and replacement changes in eukaryotic coding sequences.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DLipman%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D161%26rft.epage%253D167%26rft.atitle%253DInteraction%2Bof%2Bsilent%2Band%2Breplacement%2Bchanges%2Bin%2Beukaryotic%2Bcoding%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02100090%26rft_id%253Dinfo%253Apmid%252F6442990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02100090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6442990&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AFE8000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.27" data-doi="10.1007/s11033-009-9521-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of synonymous codon usage in <em>Zea mays</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Rep.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Breports%26rft.stitle%253DMol%2BBiol%2BRep%26rft.aulast%253DLiu%26rft.auinit1%253DH.%26rft.volume%253D37%26rft.issue%253D2%26rft.spage%253D677%26rft.epage%253D684%26rft.atitle%253DAnalysis%2Bof%2Bsynonymous%2Bcodon%2Busage%2Bin%2BZea%2Bmays.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-009-9521-7%26rft_id%253Dinfo%253Apmid%252F19330534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-009-9521-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19330534&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.28" data-doi="10.1186/1748-7188-6-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernhart</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Höner Zu Siederdissen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tafer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flamm</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ViennaRNA Package 2.0.</span> <abbr class="cit-jnl-abbrev">Algorithms Mol. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAlgorithms%2BMol%2BBiol%26rft.aulast%253DLorenz%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DViennaRNA%2BPackage%2B2.0.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1748-7188-6-26%26rft_id%253Dinfo%253Apmid%252F22115189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1748-7188-6-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22115189&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.29" data-doi="10.1093/nar/25.5.0955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.5.0955%26rft_id%253Dinfo%253Apmid%252F9023104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjI1LzUvOTU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marais</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Synonymous codon usage, accuracy of translation, and gene length in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DMarais%26rft.auinit1%253DG.%26rft.volume%253D52%26rft.issue%253D3%26rft.spage%253D275%26rft.epage%253D280%26rft.atitle%253DSynonymous%2Bcodon%2Busage%252C%2Baccuracy%2Bof%2Btranslation%252C%2Band%2Bgene%2Blength%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F11428464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11428464&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167720900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camiolo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avesani</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porceddu</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The dynamic loss and gain of introns during the evolution of the <em>Brassicaceae</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">915</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D82%26rft.spage%253D915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.32" data-doi="10.1093/dnares/dsn023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mukhopadhyay</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Differential selective constraints shaping codon usage pattern of housekeeping and tissue-specific homologous genes of rice and Arabidopsis.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsn023%26rft_id%253Dinfo%253Apmid%252F18827062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjE1LzYvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.33" data-doi="10.1038/nrg2899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPlotkin%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D42%26rft.atitle%253DSynonymous%2Bbut%2Bnot%2Bthe%2Bsame%253A%2Bthe%2Bcauses%2Band%2Bconsequences%2Bof%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285410500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.34" data-doi="10.1371/journal.pone.0022855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porceddu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camiolo</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Spatial analyses of mono, di and trinucleotide trends in plant genes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e22855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPorceddu%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253De22855%26rft.epage%253De22855%26rft.atitle%253DSpatial%2Banalyses%2Bof%2Bmono%252C%2Bdi%2Band%2Btrinucleotide%2Btrends%2Bin%2Bplant%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0022855%26rft_id%253Dinfo%253Apmid%252F21829660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0022855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21829660&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.35" data-doi="10.1093/gbe/evt078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porceddu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camiolo</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The signatures of selection for translational accuracy in plant genes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1117</span>–<span class="cit-lpage">1126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt078%26rft_id%253Dinfo%253Apmid%252F23695187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjUvNi8xMTE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-other" id="cit-7.7.2363.36"><div class="cit-metadata"><cite><span class="cit-comment">R Core Team, 2014 <em>R: A Language and Environment for Statistical Computing</em>, R Foundation for Statistical Computing, Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.37" data-doi="10.1093/gbe/evv189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ressayre</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glémin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montalent</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serre-Giardi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dillmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Introns structure patterns of variation in nucleotide composition in <em>Arabidopsis thaliana</em> and rice protein-coding genes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2913</span>–<span class="cit-lpage">2928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv189%26rft_id%253Dinfo%253Apmid%252F26450849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjk6IjcvMTAvMjkxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.38" data-doi="10.1371/journal.pcbi.0010072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ringnér</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogh</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Folding free energies of 5′-UTRs impact post-transcriptional regulation on a genomic scale in yeast.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DRingner%26rft.auinit1%253DM.%26rft.volume%253D1%26rft.issue%253D7%26rft.spage%253De72%26rft.epage%253De72%26rft.atitle%253DFolding%2Bfree%2Benergies%2Bof%2B5%2527-UTRs%2Bimpact%2Bpost-transcriptional%2Bregulation%2Bon%2Ba%2Bgenomic%2Bscale%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0010072%26rft_id%253Dinfo%253Apmid%252F16355254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0010072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16355254&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.39" data-doi="10.1093/nar/15.3.1281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The codon adaptation index–a measure of directional synonymous codon usage bias, and its potential applications.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1281</span>–<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1281%26rft.epage%253D1295%26rft.atitle%253DThe%2Bcodon%2Badaptation%2Bindex-a%2Bmeasure%2Bof%2Bdirectional%2Bsynonymous%2Bcodon%2Busage%2Bbias%252C%2Band%2Bits%2Bpotential%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1281%26rft_id%253Dinfo%253Apmid%252F3547335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjE1LzMvMTI4MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.40" data-doi="10.1093/nar/14.13.5125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuohy</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosurski</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">5125</span>–<span class="cit-lpage">5143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D14%26rft.issue%253D13%26rft.spage%253D5125%26rft.epage%253D5143%26rft.atitle%253DCodon%2Busage%2Bin%2Byeast%253A%2Bcluster%2Banalysis%2Bclearly%2Bdifferentiates%2Bhighly%2Band%2Blowly%2Bexpressed%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F14.13.5125%26rft_id%253Dinfo%253Apmid%252F3526280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMy81MTI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.41" data-doi="10.1093/molbev/msq062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foxe</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazzouri</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide evidence for efficient positive and purifying selection in capsella grandiflora, a plant species with a large effective population size.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq062%26rft_id%253Dinfo%253Apmid%252F20194429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMTgxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.42" data-doi="10.1073/pnas.1321152111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Šmarda</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bureš</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horová</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mucina</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ecological and evolutionary significance of genomic GC content diversity in monocots.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E4096</span>–<span class="cit-lpage">E4102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1321152111%26rft_id%253Dinfo%253Apmid%252F25225383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM5L0U0MDk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staiger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">G. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Enter exitrons.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.44" data-doi="10.1534/genetics.107.085399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoebel</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dykhuizen</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The cost of expression of <em>Escherichia coli lac</em> operon proteins is in the process, not in the products.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1653</span>–<span class="cit-lpage">1660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085399%26rft_id%253Dinfo%253Apmid%252F18245823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE2NTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjM2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.45" data-doi="10.1186/1471-2148-8-224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Conflicting selection pressures on synonymous codon use in yeast suggest selection on mRNA secondary structures.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">224</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DStoletzki%26rft.auinit1%253DN.%26rft.volume%253D8%26rft.spage%253D224%26rft.epage%253D224%26rft.atitle%253DConflicting%2Bselection%2Bpressures%2Bon%2Bsynonymous%2Bcodon%2Buse%2Bin%2Byeast%2Bsuggest%2Bselection%2Bon%2BmRNA%2Bsecondary%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-224%26rft_id%253Dinfo%253Apmid%252F18671878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18671878&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.46" data-doi="10.1093/molbev/msl166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Synonymous codon usage in <em>Escherichia coli</em>: selection for translational accuracy.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl166%26rft_id%253Dinfo%253Apmid%252F17101719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvMzc0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.47" data-doi="10.1007/s0023901-0074-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Coding sequence divergence between two closely related plant species: <em>Arabidopsis thaliana</em> and <em>Brassica rapa</em> ssp. <em>pekinensis</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">746</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DTiffin%26rft.auinit1%253DP.%26rft.volume%253D54%26rft.issue%253D6%26rft.spage%253D746%26rft.epage%253D753%26rft.atitle%253DCoding%2Bsequence%2Bdivergence%2Bbetween%2Btwo%2Bclosely%2Brelated%2Bplant%2Bspecies%253A%2BArabidopsis%2Bthaliana%2Band%2BBrassica%2Brapa%2Bssp.%2Bpekinensis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs0023901-0074-1%26rft_id%253Dinfo%253Apmid%252F12029356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s0023901-0074-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12029356&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175840600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vandivier</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willmann</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title"><em>Arabidopsis</em> mRNA secondary structure correlates with protein function and domains.</span> <abbr class="cit-jnl-abbrev">Plant Signal. Behav.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e24301</span>–<span class="cit-lpage">e24305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSignal.%2BBehav.%26rft.volume%253D8%26rft.spage%253De24301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.49" data-doi="10.1007/s11103-006-0041-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roossinck</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comparative analysis of expressed sequences reveals a conserved pattern of optimal codon usage in plants.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">699</span>–<span class="cit-lpage">710</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DL.%26rft.volume%253D61%26rft.issue%253D4-5%26rft.spage%253D699%26rft.epage%253D710%26rft.atitle%253DComparative%2Banalysis%2Bof%2Bexpressed%2Bsequences%2Breveals%2Ba%2Bconserved%2Bpattern%2Bof%2Boptimal%2Bcodon%2Busage%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-006-0041-8%26rft_id%253Dinfo%253Apmid%252F16897485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-006-0041-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16897485&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2363.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239551300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.50" data-doi="10.1101/gr.189102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">G. K.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Compositional gradients in <em>Gramineae</em> genes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">856</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DWong%26rft.auinit1%253DG.%2BK.-S.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D851%26rft.epage%253D856%26rft.atitle%253DCompositional%2BGradients%2Bin%2BGramineae%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.189102%26rft_id%253Dinfo%253Apmid%252F12045139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvODUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIzNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.51" data-doi="10.1093/molbev/msh191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Effects of gene expression on molecular evolution in <em>Arabidopsis thaliana</em> and <em>Arabidopsis lyrata</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1719</span>–<span class="cit-lpage">1726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh191%26rft_id%253Dinfo%253Apmid%252F15201397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIxLzkvMTcxOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2363.52" data-doi="10.1093/molbev/msp070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weems</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Translationally optimal codons associate with structurally sensitive sites in proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1571</span>–<span class="cit-lpage">1580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp070%26rft_id%253Dinfo%253Apmid%252F19349643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMTU3MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/7/2363.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2353" title="A Targeted Capture Linkage Map Anchors the Genome of the Schistosomiasis Vector Snail, Biomphalaria glabrata" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2375" title="Systematic Analysis of the DNA Damage Response Network in Telomere Defective Budding Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/7"><span>Volume 7 Issue 7, July 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/7.cover-source.jpg?itok=eAMfPilr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2363" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AKxUdo4WUo9rs6ZpD_1H1Wc87yKRGjI-ZWheqmO_QE8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/7/2363" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/7/2363" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/7/2363" class="active" data-icon-position="" data-hide-link-title="0">The Evolutionary Basis of Translational Accuracy in Plants</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485315" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-I0_GwIfWim7yHqi8sqToqkJCrcQEFW95W_yoc-xUt0k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2363.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/7/2363" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-EIB7LYCJEbTgAjz22Y0wz_QYEceSmAiAPODNowc2wwQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/7/2363" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2363.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485315" id="node485315--2" data-pisa="ggg;7/7/2363" data-pisa-master="ggg;g3.117.040626" data-apath="/ggg/7/7/2363.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Evolutionary Basis of Translational Accuracy in Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Salvatore Camiolo</span>, <span class="highwire-citation-author" data-delta="1">Gaurav Sablok</span> and <span class="highwire-citation-author" data-delta="2">Andrea Porceddu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2363-2373;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040626 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485315--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Salvatore Camiolo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dipartimento di Agraria, Sezione di Agronomia, Coltivazione Erbacee e Genetica, Università degli Studi di Sassari, 07100, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Salvatore%2BCamiolo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Camiolo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASalvatore%2BCamiolo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gaurav Sablok </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Functional Biology and Climate Change Cluster, University of Technology Sydney, New South Wales 2007, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gaurav%2BSablok%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sablok%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGaurav%2BSablok%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrea Porceddu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dipartimento di Agraria, Sezione di Agronomia, Coltivazione Erbacee e Genetica, Università degli Studi di Sassari, 07100, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BPorceddu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Porceddu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BPorceddu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aporceddu@uniss.it" class="" data-icon-position="" data-hide-link-title="0">aporceddu@uniss.it</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/7/2363" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/7/2363&amp;title=The%20Evolutionary%20Basis%20of%20Translational%20Accuracy%20in%20Plants" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/7/2363&amp;title=The%20Evolutionary%20Basis%20of%20Translational%20Accuracy%20in%20Plants" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/7/2363&amp;title=The%20Evolutionary%20Basis%20of%20Translational%20Accuracy%20in%20Plants" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/7/2363&amp;text=The%20Evolutionary%20Basis%20of%20Translational%20Accuracy%20in%20Plants" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/7/2363&amp;title=The%20Evolutionary%20Basis%20of%20Translational%20Accuracy%20in%20Plants" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/7/2363&amp;t=The%20Evolutionary%20Basis%20of%20Translational%20Accuracy%20in%20Plants" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/7/2363&amp;title=The%20Evolutionary%20Basis%20of%20Translational%20Accuracy%20in%20Plants" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/7/2363&amp;title=The%20Evolutionary%20Basis%20of%20Translational%20Accuracy%20in%20Plants" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485315/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485315" id="node485315--3" data-pisa="ggg;7/7/2363" data-pisa-master="ggg;g3.117.040626" data-apath="/ggg/7/7/2363.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Evolutionary Basis of Translational Accuracy in Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Salvatore Camiolo</span>, <span class="highwire-citation-author" data-delta="1">Gaurav Sablok</span> and <span class="highwire-citation-author" data-delta="2">Andrea Porceddu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2363-2373;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040626 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485315--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Salvatore Camiolo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dipartimento di Agraria, Sezione di Agronomia, Coltivazione Erbacee e Genetica, Università degli Studi di Sassari, 07100, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Salvatore%2BCamiolo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Camiolo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASalvatore%2BCamiolo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gaurav Sablok </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Functional Biology and Climate Change Cluster, University of Technology Sydney, New South Wales 2007, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gaurav%2BSablok%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sablok%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGaurav%2BSablok%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrea Porceddu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dipartimento di Agraria, Sezione di Agronomia, Coltivazione Erbacee e Genetica, Università degli Studi di Sassari, 07100, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BPorceddu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Porceddu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BPorceddu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aporceddu@uniss.it" class="" data-icon-position="" data-hide-link-title="0">aporceddu@uniss.it</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2363&amp;rft.epage=2373&amp;rft.atitle=The%20Evolutionary%20Basis%20of%20Translational%20Accuracy%20in%20Plants&amp;rft.volume=7&amp;rft.issue=7&amp;rft.date=2017-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Camiolo%2C+Salvatore&amp;rft.au=Sablok%2C+Gaurav&amp;rft.au=Porceddu%2C+Andrea"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485315/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485315/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485315/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485315/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485315/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485315/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485315/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485315/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485315/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485315/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485315/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485315/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/7/2363" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/7/2363.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/7/2363.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/7/2363.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/7/2363.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
