<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/7/2209" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/7/2209.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/7/2209.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/7/2209.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/7/2209.atom" />
<meta name="HW.pisa" content="ggg;7/7/2209" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Development of a Medium Density Combined-Species SNP Array for Pacific and European Oysters (Crassostrea gigas and Ostrea edulis)" />
<meta name="DC.Identifier" content="10.1534/g3.117.041780" />
<meta name="DC.Date" content="2017-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Gutierrez et al." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="SNP arrays are enabling tools for high-resolution studies of the genetic basis of complex traits in farmed and wild animals. Oysters are of critical importance in many regions from both an ecological and economic perspective, and oyster aquaculture forms a key component of global food security. The aim of our study was to design a combined-species, medium density SNP array for Pacific oyster ( Crassostrea gigas ) and European flat oyster ( Ostrea edulis ), and to test the performance of this array on farmed and wild populations from multiple locations, with a focus on European populations. SNP discovery was carried out by whole-genome sequencing (WGS) of pooled genomic DNA samples from eight C. gigas populations, and restriction site-associated DNA sequencing (RAD-Seq) of 11 geographically diverse O. edulis populations. Nearly 12 million candidate SNPs were discovered and filtered based on several criteria, including preference for SNPs segregating in multiple populations and SNPs with monomorphic flanking regions. An Affymetrix Axiom Custom Array was created and tested on a diverse set of samples ( n = 219) showing ∼27 K high quality SNPs for C. gigas and ∼11 K high quality SNPs for O. edulis segregating in these populations. A high proportion of SNPs were segregating in each of the populations, and the array was used to detect population structure and levels of linkage disequilibrium (LD). Further testing of the array on three C. gigas nuclear families ( n = 165) revealed that the array can be used to clearly distinguish between both families based on identity-by-state (IBS) clustering parental assignment software. This medium density, combined-species array will be publicly available through Affymetrix, and will be applied for genome-wide association and evolutionary genetic studies, and for genomic selection in oyster breeding programs." />
<meta name="DC.Contributor" content="Alejandro P. Gutierrez" />
<meta name="DC.Contributor" content="Frances Turner" />
<meta name="DC.Contributor" content="Karim Gharbi" />
<meta name="DC.Contributor" content="Richard Talbot" />
<meta name="DC.Contributor" content="Natalie R. Lowe" />
<meta name="DC.Contributor" content="Carolina Peñaloza" />
<meta name="DC.Contributor" content="Mark McCullough" />
<meta name="DC.Contributor" content="Paulo A. Prodöhl" />
<meta name="DC.Contributor" content="Tim P. Bean" />
<meta name="DC.Contributor" content="Ross D. Houston" />
<meta name="article:published_time" content="2017-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Development of a Medium Density Combined-Species SNP Array for Pacific and European Oysters (Crassostrea gigas and Ostrea edulis)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;SNP arrays are enabling tools for high-resolution studies of the genetic basis of complex traits in farmed and wild animals. Oysters are of critical importance in many regions from both an ecological and economic perspective, and oyster aquaculture forms a key component of global food security. The aim of our study was to design a combined-species, medium density SNP array for Pacific oyster (&lt;i&gt;Crassostrea gigas&lt;/i&gt;) and European flat oyster (&lt;i&gt;Ostrea edulis&lt;/i&gt;), and to test the performance of this array on farmed and wild populations from multiple locations, with a focus on European populations. SNP discovery was carried out by whole-genome sequencing (WGS) of pooled genomic DNA samples from eight &lt;i&gt;C. gigas&lt;/i&gt; populations, and restriction site-associated DNA sequencing (RAD-Seq) of 11 geographically diverse &lt;i&gt;O. edulis&lt;/i&gt; populations. Nearly 12 million candidate SNPs were discovered and filtered based on several criteria, including preference for SNPs segregating in multiple populations and SNPs with monomorphic flanking regions. An Affymetrix Axiom Custom Array was created and tested on a diverse set of samples (&lt;i&gt;n&lt;/i&gt; = 219) showing ∼27 K high quality SNPs for &lt;i&gt;C. gigas&lt;/i&gt; and ∼11 K high quality SNPs for &lt;i&gt;O. edulis&lt;/i&gt; segregating in these populations. A high proportion of SNPs were segregating in each of the populations, and the array was used to detect population structure and levels of linkage disequilibrium (LD). Further testing of the array on three &lt;i&gt;C. gigas&lt;/i&gt; nuclear families (&lt;i&gt;n&lt;/i&gt; = 165) revealed that the array can be used to clearly distinguish between both families based on identity-by-state (IBS) clustering parental assignment software. This medium density, combined-species array will be publicly available through Affymetrix, and will be applied for genome-wide association and evolutionary genetic studies, and for genomic selection in oyster breeding programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/07/01" />
<meta name="citation_mjid" content="ggg;7/7/2209" />
<meta name="citation_id" content="7/7/2209" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/7/2209" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/7/2209.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/7/2209.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/7/2209.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.041780" />
<meta name="citation_pmid" content="28533337" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2209" />
<meta name="citation_lastpage" content="2218" />
<meta name="citation_author" content="Alejandro P. Gutierrez" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom" />
<meta name="citation_author" content="Frances Turner" />
<meta name="citation_author_institution" content="Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom" />
<meta name="citation_author" content="Karim Gharbi" />
<meta name="citation_author_institution" content="Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom" />
<meta name="citation_author" content="Richard Talbot" />
<meta name="citation_author_institution" content="Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom" />
<meta name="citation_author" content="Natalie R. Lowe" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom" />
<meta name="citation_author" content="Carolina Peñaloza" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom" />
<meta name="citation_author" content="Mark McCullough" />
<meta name="citation_author_institution" content="Institute for Global Food Security, School of Biological Sciences, Queen’s University Belfast, BT7 1NN, United Kingdom" />
<meta name="citation_author" content="Paulo A. Prodöhl" />
<meta name="citation_author_institution" content="Institute for Global Food Security, School of Biological Sciences, Queen’s University Belfast, BT7 1NN, United Kingdom" />
<meta name="citation_author" content="Tim P. Bean" />
<meta name="citation_author_institution" content="Centre for Environment Fisheries and Aquaculture Science, Cefas Weymouth Laboratory, Dorset DT4 8UB, United Kingdom" />
<meta name="citation_author" content="Ross D. Houston" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom" />
<meta name="citation_author_email" content="ross.houston@roslin.ed.ac.uk" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=S. A. Appleyard;citation_author=R. D. Ward;citation_title=Genetic diversity and effective population size in mass selection lines of Pacific oyster (Crassostrea gigas).;citation_pages=148-159;citation_volume=254;citation_year=2006;citation_issue=1–4;citation_doi=10.1016/j.aquaculture.2005.10.017" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Ocean Coast. Manage.;citation_author=C. Bromley;citation_author=C. McGonigle;citation_author=E. C. Ashton;citation_author=D. Roberts;citation_title=Bad moves: pros and cons of moving oysters – a case study of global translocations of Ostrea edulis Linnaeus, 1758 (Mollusca: Bivalvia).;citation_pages=103-115;citation_volume=122;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_issue=11;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_issue=2;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. Correa;citation_author=J. P. Lhorente;citation_author=M. E. López;citation_author=L. Bassini;citation_author=S. Naswa;citation_title=Genome-wide association analysis reveals loci associated with resistance against Piscirickettsia salmonis in two Atlantic salmon (Salmo salar L.) chromosomes.;citation_pages=854;citation_volume=16;citation_year=2015;citation_issue=1;citation_pmid=26499328;citation_doi=10.1186/s12864-015-2038-7" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. de Roos;citation_author=B. J. Hayes;citation_author=R. J. Spelman;citation_author=M. E. Goddard;citation_title=Linkage disequilibrium and persistence of phase in Holstein–Friesian, Jersey and Angus cattle.;citation_volume=179;citation_year=2008;citation_pmid=18622038;citation_doi=10.1534/genetics.107.084301" />
<meta name="citation_reference" content="citation_journal_title=Aust. Aquacult.;citation_author=L. J. English;citation_author=G. B. Maguire;citation_author=R. D. Ward;citation_title=Genetic variation of wild and hatchery populations of the Pacific oyster, Crassostrea gigas (Thunberg);citation_pages=283-298;citation_volume=187;citation_year=2000;citation_issue=3–4" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. Falconer;citation_author=T. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. Fleury;citation_author=A. Huvet;citation_author=C. Lelong;citation_author=J. de Lorgeril;citation_author=V. Boulo;citation_title=Generation and analysis of a 29,745 unique Expressed Sequence Tags from the Pacific oyster (Crassostrea gigas) assembled into a publicly accessible database: the GigasDatabase.;citation_pages=341-341;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19640306;citation_doi=10.1186/1471-2164-10-341" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=X. Geng;citation_author=J. Sha;citation_author=S. Liu;citation_author=L. Bao;citation_author=J. Zhang;citation_title=A genome-wide association study in catfish reveals the presence of functional hubs of related genes within QTLs for columnaris disease resistance.;citation_pages=196;citation_volume=16;citation_year=2015;citation_issue=1;citation_pmid=25888203;citation_doi=10.1186/s12864-015-1409-4" />
<meta name="citation_reference" content="citation_journal_title=J. Mar. Biol. Assoc. U. K.;citation_author=I. A. Giantsis;citation_author=N. Mucci;citation_author=E. Randi;citation_author=T. J. Abatzopoulos;citation_author=A. P. Apostolidis;citation_title=Microsatellite variation of mussels (Mytilus galloprovincialis) in central and eastern Mediterranean: genetic panmixia in the Aegean and the Ionian seas.;citation_pages=797-809;citation_volume=94;citation_year=2014;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Marine biotechnology (New York, N.Y.);citation_journal_abbrev=Mar Biotechnol (NY);citation_author=X. Guo;citation_author=Q. Li;citation_author=Q. Z. Wang;citation_author=L. F. Kong;citation_title=Genetic mapping and QTL analysis of growth-related traits in the Pacific oyster.;citation_pages=218-226;citation_volume=14;citation_year=2012;citation_issue=2;citation_pmid=21932055;citation_doi=10.1007/s10126-011-9405-4" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. Hedgecock;citation_author=G. Shin;citation_author=A. Y. Gracey;citation_author=D. V. Den Berg;citation_author=M. P. Samanta;citation_title=Second-generation linkage maps for the Pacific oyster Crassostrea gigas reveal errors in assembly of genome scaffolds.;citation_pages=2007-2019;citation_volume=5;citation_year=2015;citation_issue=10" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Hong Xia;citation_author=Z. Bai;citation_author=Z. Meng;citation_author=Y. Zhang;citation_author=L. Wang;citation_title=Signatures of selection in tilapia revealed by whole genome resequencing.;citation_pages=14168;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. D. Houston;citation_author=J. B. Taggart;citation_author=T. Cézard;citation_author=M. Bekaert;citation_author=N. R. Lowe;citation_title=Development and validation of a high density SNP genotyping array for Atlantic salmon (Salmo salar).;citation_pages=90;citation_volume=15;citation_year=2014;citation_issue=1;citation_pmid=24524230;citation_doi=10.1186/1471-2164-15-90" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Hubert;citation_author=D. Hedgecock;citation_title=Linkage Maps of Microsatellite DNA Markers for the Pacific Oyster Crassostrea gigas;citation_pages=351-362;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454548;citation_doi=10.1534/genetics.104.027342" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=W. Jiao;citation_author=X. Fu;citation_author=J. Li;citation_author=L. Li;citation_author=L. Feng;citation_title=Large-scale development of gene-associated single-nucleotide polymorphism markers for molluscan population genomic, comparative genomic, and genome-wide association studies.;citation_volume=21;citation_year=2014;citation_issue=2;citation_pmid=24277739;citation_doi=10.1093/dnares/dst048" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomics;citation_author=Y.-L. Jin;citation_author=L.-F. Kong;citation_author=H. Yu;citation_author=Q. Li;citation_title=Development, inheritance and evaluation of 55 novel single nucleotide polymorphism markers for parentage assignment in the Pacific oyster (Crassostrea gigas).;citation_pages=129-141;citation_volume=36;citation_year=2014;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Jombart;citation_author=I. Ahmed;citation_title=adegenet 1.3–1: new tools for the analysis of genome-wide SNP data.;citation_volume=27;citation_year=2011;citation_issue=21;citation_pmid=21926124;citation_doi=10.1093/bioinformatics/btr521" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. T. Kalinowski;citation_author=M. L. Taper;citation_author=T. C. Marshall;citation_title=Revising how the computer program CERVUS accommodates genotyping error increases success in paternity assignment.;citation_pages=1099-1106;citation_volume=16;citation_year=2007;citation_issue=5;citation_pmid=17305863;citation_doi=10.1111/j.1365-294X.2007.03089.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Kofler;citation_author=R. V. Pandey;citation_author=C. Schlötterer;citation_title=PoPoolation2: identifying differentiation between populations using sequencing of pooled DNA samples (Pool-Seq).;citation_volume=27;citation_year=2011;citation_issue=24;citation_pmid=22025480;citation_doi=10.1093/bioinformatics/btr589" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. Kranis;citation_author=A. A. Gheyas;citation_author=C. Boschiero;citation_author=F. Turner;citation_author=L. Yu;citation_title=Development of a high density 600K SNP genotyping array for chicken.;citation_pages=59;citation_volume=14;citation_year=2013;citation_issue=1;citation_pmid=23356797;citation_doi=10.1186/1471-2164-14-59" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=D. Lallias;citation_author=A. R. Beaumont;citation_author=C. S. Haley;citation_author=P. Boudry;citation_author=S. Heurtebise;citation_title=A first-generation genetic linkage map of the European flat oyster Ostrea edulis (L.) based on AFLP and microsatellite markers.;citation_pages=560-568;citation_volume=38;citation_year=2007;citation_issue=6;citation_pmid=17931405;citation_doi=10.1111/j.1365-2052.2007.01647.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Ecol Resour;citation_author=D. Lallias;citation_author=R. Stockdale;citation_author=P. Boudry;citation_author=A. R. Beaumont;citation_author=S. Lapègue;citation_title=Characterization of 27 microsatellite loci in the European flat oyster Ostrea edulis.;citation_pages=960-963;citation_volume=9;citation_year=2009;citation_issue=3;citation_pmid=21564805" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=D. Lallias;citation_author=P. Boudry;citation_author=S. Lapègue;citation_author=J. W. King;citation_author=A. R. Beaumont;citation_title=Strategies for the retention of high genetic variability in European flat oyster (Ostrea edulis) restoration programmes.;citation_pages=1899-1910;citation_volume=11;citation_year=2010;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=S. Lapègue;citation_author=E. Harrang;citation_author=S. Heurtebise;citation_author=E. Flahauw;citation_author=C. Donnadieu;citation_title=Development of SNP-genotyping arrays in two shellfish species.;citation_pages=820-830;citation_volume=14;citation_year=2014;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=G. Li;citation_author=S. Hubert;citation_author=K. Bucklin;citation_author=V. Ribes;citation_author=D. Hedgecock;citation_title=Characterization of 79 microsatellite DNA markers in the Pacific oyster Crassostrea gigas.;citation_pages=228-232;citation_volume=3;citation_year=2003;citation_issue=2;citation_doi=10.1046/j.1471-8286.2003.00406.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_issue=14;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_issue=16;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=R. Li;citation_author=Q. Li;citation_author=F. Cornette;citation_author=L. Dégremont;citation_author=S. Lapègue;citation_title=Development of four EST-SSR multiplex PCRs in the Pacific oyster (Crassostrea gigas) and their validation in parentage assignment.;citation_pages=234-239;citation_volume=310;citation_year=2010;citation_issue=1–2;citation_doi=10.1016/j.aquaculture.2010.09.037" />
<meta name="citation_reference" content="citation_journal_title=Fish. Sci.;citation_author=S. Li;citation_author=Q. Li;citation_author=H. Yu;citation_author=L. Kong;citation_author=S. Liu;citation_title=Genetic variation and population structure of the Pacific oyster Crassostrea gigas in the northwestern Pacific inferred from mitochondrial COI sequences.;citation_pages=1071-1082;citation_volume=81;citation_year=2015;citation_issue=6" />
<meta name="citation_reference" content="citation_journal_title=BMC Res. Notes;citation_author=S. Liu;citation_author=L. Sun;citation_author=Y. Li;citation_author=F. Sun;citation_author=Y. Jiang;citation_title=Development of the catfish 250K SNP array for genome-wide association studies.;citation_pages=135;citation_volume=7;citation_year=2014;citation_issue=1;citation_pmid=24618043;citation_doi=10.1186/1756-0500-7-135" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. L. Loosanoff;citation_title=The European Oyster in American Waters;citation_pages=119-121;citation_volume=121;citation_year=1955;citation_issue=3135;citation_pmid=13225752;citation_doi=10.1126/science.121.3135.119" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Magoč;citation_author=S. L. Salzberg;citation_title=FLASH: fast length adjustment of short reads to improve genome assemblies.;citation_volume=27;citation_year=2011;citation_issue=21;citation_pmid=21903629;citation_doi=10.1093/bioinformatics/btr507" />
<meta name="citation_reference" content="citation_journal_title=EMBnet.journal;citation_author=M. Martin;citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads.;citation_pages=10;citation_volume=17;citation_year=2011;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. K. Matukumalli;citation_author=C. T. Lawley;citation_author=R. D. Schnabel;citation_author=J. F. Taylor;citation_author=M. F. Allan;citation_title=Development and characterization of a high density SNP genotyping assay for cattle.;citation_pages=e5350-e5350;citation_volume=4;citation_year=2009;citation_issue=4;citation_pmid=19390634;citation_doi=10.1371/journal.pone.0005350" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_issue=9;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=J. Ødegård;citation_author=T. H. Meuwissen;citation_title=Identity-by-descent genomic selection using selective and sparse genotyping.;citation_pages=3;citation_volume=46;citation_year=2014;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=Y. Palti;citation_author=G. Gao;citation_author=S. Liu;citation_author=M. P. Kent;citation_author=S. Lien;citation_title=The development and characterization of a 57K single nucleotide polymorphism array for rainbow trout.;citation_pages=662-672;citation_volume=15;citation_year=2015;citation_issue=3;citation_pmid=25294387;citation_doi=10.1111/1755-0998.12337" />
<meta name="citation_reference" content="citation_journal_title=Fish Shellfish Immunol.;citation_author=B. G. Pardo;citation_author=J. A. Álvarez-Dios;citation_author=A. Cao;citation_author=A. Ramilo;citation_author=A. Gómez-Tato;citation_title=Construction of an Ostrea edulis database from genomic and expressed sequence tags (ESTs) obtained from Bonamia ostreae infected haemocytes: development of an immune-enriched oligo-microarray.;citation_pages=331-344;citation_volume=59;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Peng;citation_author=H. C. M. Leung;citation_author=S. M. Yiu;citation_author=F. Y. L. Chin;citation_title=IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth.;citation_volume=28;citation_year=2012;citation_issue=11;citation_pmid=22495754;citation_doi=10.1093/bioinformatics/bts174" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B. Biol. Sci.;citation_title=Infectious diseases in oyster aquaculture require a new integrated approach.;citation_author=F. Pernet;citation_author=C. Lupo;citation_author=C. Bacher;citation_author=R.J. Whittington;citation_volume=371;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Qi;citation_author=K. Song;citation_author=C. Li;citation_author=W. Wang;citation_author=B. Li;citation_title=Construction and evaluation of a high-density SNP array for the Pacific oyster (Crassostrea gigas).;citation_pages=e0174007;citation_volume=12;citation_year=2017;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. M. Ramos;citation_author=R. P. M. A. Crooijmans;citation_author=N. A. Affara;citation_author=A. J. Amaral;citation_author=A. L. Archibald;citation_title=Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.;citation_pages=e6524-e6524;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19654876;citation_doi=10.1371/journal.pone.0006524" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. M. Richards;citation_author=M. M. Liu;citation_author=N. Lowe;citation_author=J. W. Davey;citation_author=M. L. Blaxter;citation_title=RAD-Seq derived markers flank the shell colour and banding loci of the Cepaea nemoralis supergene.;citation_pages=3077-3089;citation_volume=22;citation_year=2013;citation_issue=11;citation_doi=10.1111/mec.12262" />
<meta name="citation_reference" content="citation_journal_title=Rev. Aquacult.;citation_author=D. Robledo;citation_author=C. Palaiokostas;citation_author=L. Bargelloni;citation_author=P. Martínez;citation_author=R. Houston;citation_title=Applications of genotyping by sequencing in aquaculture breeding and genetics.;citation_year=2017;citation_doi=10.1111/raq.12193." />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=A. Rohfritsch;citation_author=N. Bierne;citation_author=P. Boudry;citation_author=S. Heurtebise;citation_author=F. Cornette;citation_title=Population genomics shed light on the demographic and adaptive histories of European invasion in the Pacific oyster, Crassostrea gigas.;citation_pages=1064-1078;citation_volume=6;citation_year=2013;citation_issue=7;citation_pmid=24187588" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Ecol Resour;citation_author=F. Rousset;citation_title=genepop'007: a complete re-implementation of the genepop software for Windows and Linux.;citation_pages=103-106;citation_volume=8;citation_year=2008;citation_issue=1;citation_pmid=21585727;citation_doi=10.1111/j.1471-8286.2007.01931.x" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=C. Sauvage;citation_author=N. Bierne;citation_author=S. Lapègue;citation_author=P. Boudry;citation_title=Single Nucleotide polymorphisms and their relationship to codon usage bias in the Pacific oyster Crassostrea gigas.;citation_pages=13-22;citation_volume=406;citation_year=2007;citation_issue=1–2;citation_pmid=17616269;citation_doi=10.1016/j.gene.2007.05.011" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=C. Sauvage;citation_author=P. Boudry;citation_author=D. J. De Koning;citation_author=C. S. Haley;citation_author=S. Heurtebise;citation_title=QTL for resistance to summer mortality and OsHV-1 load in the Pacific oyster (Crassostrea gigas).;citation_pages=390-399;citation_volume=41;citation_year=2010;citation_issue=4;citation_pmid=20096029" />
<meta name="citation_reference" content="citation_journal_title=Marine biotechnology (New York, N.Y.);citation_journal_abbrev=Mar Biotechnol (NY);citation_author=M. Sekino;citation_author=M. Hamaguchi;citation_author=F. Aranishi;citation_author=K. Okoshi;citation_title=Development of novel microsatellite DNA markers from the Pacific oyster Crassostrea gigas.;citation_pages=227-233;citation_volume=5;citation_year=2003;citation_issue=3;citation_pmid=14502394;citation_doi=10.1007/s10126-002-0104-z" />
<meta name="citation_reference" content="citation_journal_title=Mar. Biol. Res.;citation_author=A. Shabtay;citation_author=Y. Tikochinski;citation_author=Y. Benayahu;citation_author=G. Rilov;citation_title=Preliminary data on the genetic structure of a highly successful invading population of oyster suggesting its establishment dynamics in the Levant.;citation_pages=407-415;citation_volume=10;citation_year=2014;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=J. Mar. Biol. Assoc. U. K.;citation_author=H. Sobolewska;citation_author=A. R. Beaumont;citation_title=Genetic variation at microsatellite loci in northern populations of the European flat oyster (Ostrea edulis).;citation_pages=955-960;citation_volume=85;citation_year=2005;citation_issue=04" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=J. A. Sved;citation_title=Linkage disequilibrium and homozygosity of chromosome segments in finite populations.;citation_pages=125-141;citation_volume=2;citation_year=1971;citation_issue=2;citation_pmid=5170716;citation_doi=10.1016/0040-5809(71)90011-6" />
<meta name="citation_reference" content="citation_journal_title=J. Sea Res.;citation_author=K. Troost;citation_title=Causes and effects of a highly successful marine invasion: case-study of the introduced Pacific oyster Crassostrea gigas in continental NW European estuaries.;citation_pages=145-165;citation_volume=64;citation_year=2010;citation_issue=3;citation_doi=10.1016/j.seares.2010.02.004" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H.-Y. Tsai;citation_author=A. Hamilton;citation_author=A. E. Tinch;citation_author=D. R. Guy;citation_author=K. Gharbi;citation_title=Genome wide association and genomic prediction for growth traits in juvenile farmed Atlantic salmon using a high density SNP array.;citation_pages=969;citation_volume=16;citation_year=2015;citation_issue=1;citation_pmid=26582102;citation_doi=10.1186/s12864-015-2117-9" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=H.-Y. Tsai;citation_author=A. Hamilton;citation_author=A. E. Tinch;citation_author=D. R. Guy;citation_author=J. E. Bron;citation_title=Genomic prediction of host resistance to sea lice in farmed Atlantic salmon populations.;citation_pages=47;citation_volume=48;citation_year=2016;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=R. L. Vallejo;citation_author=T. D. Leeds;citation_author=B. O. Fragomeni;citation_author=G. Gao;citation_author=A. G. Hernandez;citation_title=Evaluation of genome-enabled selection for bacterial cold water disease resistance using progeny performance data in rainbow trout: insights on genotyping methods and genomic prediction models.;citation_pages=96;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mar. Biol.;citation_author=M. Vera;citation_author=J. Carlsson;citation_author=J. E. Carlsson;citation_author=T. Cross;citation_author=S. Lynch;citation_title=Current genetic status, temporal stability and structure of the remnant wild European flat oyster populations: conservation and restoring implications.;citation_pages=239;citation_volume=163;citation_year=2016;citation_issue=12" />
<meta name="citation_reference" content="citation_journal_title=J. Shellfish Res.;citation_author=B. Vercaemer;citation_author=K. R. Spence;citation_author=C. M. Herbinger;citation_author=S. Lapègue;citation_author=E. L. Kenchington;citation_title=Genetic diversity of the European oyster (Ostrea edulis L.) in Nova Scotia: comparison with other parts of Canada, Maine and Europe and implications for broodstock management.;citation_pages=543-551;citation_volume=25;citation_year=2006;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. Wang;citation_author=H. Qi;citation_author=L. Li;citation_author=H. Que;citation_author=D. Wang;citation_title=Discovery and validation of genic single nucleotide polymorphisms in the Pacific oyster Crassostrea gigas.;citation_pages=123-135;citation_volume=15;citation_year=2015;citation_issue=1;citation_pmid=24823694;citation_doi=10.1111/1755-0998.12278" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=Y. Wang;citation_author=X. Wang;citation_author=A. Wang;citation_author=X. Guo;citation_title=A 16-microsatellite multiplex assay for parentage assignment in the eastern oyster (Crassostrea virginica Gmelin).;citation_pages=S28-S33;citation_volume=308;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Xu;citation_author=Z. Zhao;citation_author=X. Zhang;citation_author=X. Zheng;citation_author=J. Li;citation_title=Development and evaluation of the first high-throughput SNP array for common carp (Cyprinus carpio).;citation_pages=307;citation_volume=15;citation_year=2014;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. M. Yáñez;citation_author=S. Naswa;citation_author=M. E. López;citation_author=L. Bassini;citation_author=K. Correa;citation_title=Genomewide single nucleotide polymorphism discovery in Atlantic salmon (Salmo salar): validation in wild and farmed American and European populations.;citation_pages=1002-1011;citation_volume=16;citation_year=2016;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_abbrev=J Genet Genomics;citation_author=H. Yu;citation_author=Q. Li;citation_title=Genetic variation of wild and hatchery populations of the Pacific oyster Crassostrea gigas assessed by microsatellite markers.;citation_pages=1114-1122;citation_volume=34;citation_year=2007;citation_issue=12;citation_pmid=18155624" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Zhang;citation_author=X. Fang;citation_author=X. Guo;citation_author=L. Li;citation_author=R. Luo;citation_title=The oyster genome reveals stress adaptation and complexity of shell formation.;citation_pages=49-54;citation_volume=490;citation_year=2012;citation_issue=7418;citation_pmid=22992520;citation_doi=10.1038/nature11413" />
<meta name="citation_reference" content="citation_journal_title=Aquacult. Int.;citation_author=X. Zhong;citation_author=Q. Li;citation_author=X. Guo;citation_author=H. Yu;citation_author=L. Kong;citation_title=QTL mapping for glycogen content and shell pigmentation in the Pacific oyster Crassostrea gigas using microsatellites and SNPs.;citation_pages=1877-1889;citation_volume=22;citation_year=2014;citation_issue=6;citation_doi=10.1007/s10499-014-9789-z" />
<meta name="citation_reference" content="citation_journal_title=J. World. Aquac. Soc.;citation_author=X. Zhong;citation_author=Q. Li;citation_author=L. Kong;citation_author=H. Yu;citation_title=Estimates of linkage disequilibrium and effective population size in wild and selected populations of the Pacific oyster using single-nucleotide polymorphism markers.;citation_year=2017;citation_doi=10.1111/jwas.12393" />
<meta name="twitter:title" content="Development of a Medium Density Combined-Species SNP Array for Pacific and European Oysters (Crassostrea gigas and Ostrea edulis)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
<meta name="twitter:description" content="SNP arrays are enabling tools for high-resolution studies of the genetic basis of complex traits in farmed and wild animals. Oysters are of critical importance in many regions from both an ecological and economic perspective, and oyster aquaculture forms a key component of global food security. The aim of our study was to design a combined-species, medium density SNP array for Pacific oyster ( Crassostrea gigas ) and European flat oyster ( Ostrea edulis ), and to test the performance of this array on farmed and wild populations from multiple locations, with a focus on European populations. SNP discovery was carried out by whole-genome sequencing (WGS) of pooled genomic DNA samples from eight C. gigas populations, and restriction site-associated DNA sequencing (RAD-Seq) of 11 geographically diverse O. edulis populations. Nearly 12 million candidate SNPs were discovered and filtered based on several criteria, including preference for SNPs segregating in multiple populations and SNPs with monomorphic flanking regions. An Affymetrix Axiom Custom Array was created and tested on a diverse set of samples ( n = 219) showing ∼27 K high quality SNPs for C. gigas and ∼11 K high quality SNPs for O. edulis segregating in these populations. A high proportion of SNPs were segregating in each of the populations, and the array was used to detect population structure and levels of linkage disequilibrium (LD). Further testing of the array on three C. gigas nuclear families ( n = 165) revealed that the array can be used to clearly distinguish between both families based on identity-by-state (IBS) clustering parental assignment software. This medium density, combined-species array will be publicly available through Affymetrix, and will be applied for genome-wide association and evolutionary genetic studies, and for genomic selection in oyster breeding programs." />
<meta name="og-title" property="og:title" content="Development of a Medium Density Combined-Species SNP Array for Pacific and European Oysters (Crassostrea gigas and Ostrea edulis)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/7/2209" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="SNP arrays are enabling tools for high-resolution studies of the genetic basis of complex traits in farmed and wild animals. Oysters are of critical importance in many regions from both an ecological and economic perspective, and oyster aquaculture forms a key component of global food security. The aim of our study was to design a combined-species, medium density SNP array for Pacific oyster ( Crassostrea gigas ) and European flat oyster ( Ostrea edulis ), and to test the performance of this array on farmed and wild populations from multiple locations, with a focus on European populations. SNP discovery was carried out by whole-genome sequencing (WGS) of pooled genomic DNA samples from eight C. gigas populations, and restriction site-associated DNA sequencing (RAD-Seq) of 11 geographically diverse O. edulis populations. Nearly 12 million candidate SNPs were discovered and filtered based on several criteria, including preference for SNPs segregating in multiple populations and SNPs with monomorphic flanking regions. An Affymetrix Axiom Custom Array was created and tested on a diverse set of samples ( n = 219) showing ∼27 K high quality SNPs for C. gigas and ∼11 K high quality SNPs for O. edulis segregating in these populations. A high proportion of SNPs were segregating in each of the populations, and the array was used to detect population structure and levels of linkage disequilibrium (LD). Further testing of the array on three C. gigas nuclear families ( n = 165) revealed that the array can be used to clearly distinguish between both families based on identity-by-state (IBS) clustering parental assignment software. This medium density, combined-species array will be publicly available through Affymetrix, and will be applied for genome-wide association and evolutionary genetic studies, and for genomic selection in oyster breeding programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/7.cover-source.jpg" />
    <title>Development of a Medium Density Combined-Species SNP Array for Pacific and European Oysters (Crassostrea gigas and Ostrea edulis) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"sCka5psZGK2-b7k6hnljiXxjbJwmsrSBjivE9jQzY54","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485318","apath":"\/ggg\/7\/7\/2209.atom","pisa":"ggg;7\/7\/2209","ac":{"\/ggg\/7\/7\/2209.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/7\/2209.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2209"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/7\/2209"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/7\/2209":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BkOiG7ULkq2kxttPmPk\\\\\\\/9K8o9IEPREgiaKTt0uzvSFqV\\\\\\\/hHpRfpJ4vTOhJrfeB8oAaiDtAXFb8eVkeEY4It3bXFsikP6yzKjrr30SMHIZ+qY8umqeBbQ6ycvfw4aNInXObxdza7vePP6nDfylNnnFFyXFZ66ttRF3BzKxMZE+9tz5BATLSn+baFgg0ojAXUsrT6m5dmcs4sdFrRCz4e9Z6Fp\\\\\\\/6dJ4D4xSHUNjWNyWrmCqGwMyjIK+fuUtYsXy\\\\\\\/PFMzlx0RHfqw7EwDMFvadXypM7x70JbYM+OPdCwK5prkjMWqViDuUN5dvRfSdZXn3OWqpDAowYPn62EYPNYrTPPuzLf6rXl\\\\\\\/RX\\\\\\\/20nCDP9o5p4E1tfk5MDDOzQXpRgGUfVpfXz5Wq3mKotMjvUEpNDpJl7HymfelsK7oejxPFgdyvM6nRlOWn+P\\\\\\\/kI8nBRPyzyBZps9pwWiyVoksxGDPehg87IqzL3XngpICIx2HK+f02Bti4QkK3UVf4OPqKjJ7toXwTshXVn5+70BiSuERg8iKe7hmVPiGZdvGZ8JxsFTW6rhWXtJElH5t7yI6oup5LLpxHRZvkqVrD4phCfuUQqC3nZfPKWx7rDLPb9jYHHNP6uA3QeZLh7DlMp9pvvNBXTZGmz7NSaunt9F1eOTo9zLh1RpxqPy+bd0vnhwnxCQo9rvp4oicd2dH7TizKifGaukX8z81FGRVg9Im5qFfYFwNjKhLkFUPrnFZc80LGf43VsJn2v\\\\\\\/A1bISaAMozbKzjbu8wkkgRHVB7aRayE2iVWEnPLV4aV9qS\\\\\\\/PAtf30v77geZg3A6YaMVn9UlIIYe+GHaTpygpgYLj5bX6kL7u6WKOHNe5gRERk5Pov8EFfNt7KlcwnHj0EmsJEEh\\\\\\\/lXzrmB4NeiIghzvL\\\\\\\/r1F4AK9l7NoMOThDN3OAliVQtrbr5NwxpnqHZ6eOFi0YJ+MAUtdXzwpTj8hAFa5ziEQPWCPK6+mhuCJwHxvwumatZ\\\\\\\/MXZHCcv91S9XQd3hG7ApkFUCYQQ\\\\\\\/Mj5mxyZVUrcmwWzbhv2pgtTVtFiQzEdkx0UOWgMr9jwdX3AelwDcjQHh98g1qTdQmCCUBBhKSnjZKpXuyiZf+LENmBlIMUkLikKlcTuqF92zsnEissN0VDJD2\\\\\\\/v+ztdMVyCoeIHxpAoPLWNkZTdknLe3F59zvDlRZb\\\\\\\/mSpKRJSc49mXbQO4h5QWauZn3heO\\\\\\\/cjTzu4GeOZRwsYL5qYYKl4g4LQ10\\\\\\\/C0xBib8i5zkMtaTlmwOwBHuxyZZdYUb90gw\\\u0022,\\\u0022iv\\\u0022:\\\u0022vLMc1XpoXBi2mYsZl1XsBA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022786f43f8e33e50360f44ba8e7f58a9c0a96f001990333701cb1af82f9954fca8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QSxjEtUDjI9ulpQj4eZiMhmiKC8HwhiObOhlg0nP2Reqkjap\\\\\\\/BWiZEg7rGJfP2jmbDP6MqZM2r+xwdrHRRbDaKujOtIAcfNnksJ4UKOL5CgIrqGoe86u\\\\\\\/jJ2lKDowFeDOFOgrYvPQ3+tijg2F2sEAm+kfkmtoVpBEdF+8CADvEVJP0fBiWqR654Kmj0ryCul0jDA4lWr5p7e9bV9an05nlsweVJwUX6dU17X2dcK3y7dzIsqZL44iOZGcYXIqUqRlgeHLYn3kE2Fnq5gAPNpuYd1jkun0uSA9eSBByGtEiJCL64YKaVMdDNH3Fla9F\\\\\\\/XL61VoN6VWpB78UjFu4FctXULO4K5du\\\\\\\/CZPU5jyGWzzxfi00LVljWG4rvQI7Q7v5MhrXh97U+YM5jQPCWOpU58ezfkdKErJ6IM5inTzbYZZ9jT9a3EwIkpfuXprCh3A4+J0Exm\\\\\\\/LM6C2kCwxIrBh\\\\\\\/+e88NhiDhGBf7CMBfsq5s1E85nJshlf\\\\\\\/lI9fx4WhJZGe2vs+nI3VRQVGxsa7argsKf71Y2AKPG0o6vNiSNZFYp4sgQ84pqTFYU1pE8VsFkyF\\\\\\\/vQprBkzKVA9rc1gXD2XlUno1iqCroxgyDpWfhnyVqtdEnEoHEgLXHFRXoGwDPeLDh57LS3v73jF60SRjSyIXjsrRwze\\\\\\\/5rGeYXj80\\\\\\\/3swOVkfztayo4a1ypgnL2NnrdZCdM0b6uAVNorV7ShQSD1sG+F20Etz+mRU7kUB7qHPcLCbuAvBqwWtsf604netFFdENcKWsw7hALZRfdFCRMw76Yv8wIyQA35QodnkcUDQgAvT8u7bgpoiNfcSkqCft6nIx5Bpm4+meIXFsk1pUR2lOcP4A45lE04mr\\\\\\\/teqyaMHjsyYwH78QMndIZBDNlL3JCAbjOuu0m3XpP\\\\\\\/++sauCYQT74z1QGouf5rbamTaHpWlJbGrg1AuhGvo+oB08PX2G41TNTq+iKFtu7BoCQzCZTCSu25HFkmvLD2ZMv5RAJPjF7wkD+wpsyPM4shkIElkwFTPY8mDfFn34QD8GrK66txVx1Pu20T\\\\\\\/KXlmPPHPU77AVZTcsGGEw6+iUOBwZInlTU0WXz8uyziBCwvWB7PN6x0X8IET7uALsRTLieiwlZChW8lgdT9ifQzTHLDacPsdCQ\\\\\\\/XkuNF4WzrZJ57Zb6Dtz66plgja0vqZ6Jn\\\\\\\/tkH2QJjX514Va30gzasd9ycWHGtUF1neF2J4LL\\\\\\\/CoBVLMA4Wqm6Hnyn\\\\\\\/pWlrcfEnQniqeOSLWr+G0aMmhvMwuLnMjGhdEndEF4\\\\\\\/etO+h9f9Ys5zM5MSlDZpiG3g1QygrQWYCq+7gbNsdg9OrmLt64n4lShLdByxK2IZwn56e3VBr+CV+P8RCiAaOKuDB3tp510ilVyx5KWiMFlVzFw9KLzf8kVsBWS\\\\\\\/TZcM0KSgqGb1x37zDyTzXiI2z27MgMnBIaCZRn3Y4gOwrCYc9\\\\\\\/4Av1KT+\\\\\\\/Y8fnOfvfzEhKK6K3t4eMNK7LxEt0rpf3ZfvQosHVLhXkCo2G2G33CJEObgYjACokV6bMcVOOFOqbQbBqas+8ymlUtXobs07JYWEU4ZOtMDfPj8MXCUpGgotwxb7H3DFyyvFa1DJkw+rdTB8+RobmqLYcvRGGojIjs58cdBFmOWEVWIt7SR4Dat8Ifvge54QuwK1pJ1xHT4uj64hFplYhHyhGwOn8KimDa7flhhuAzHjcj2Jga0ESDjMG+y2z\\\\\\\/9H1Q5k1OlJHwknbaBB\\\\\\\/FeV3zFcuuKMVpg2xVx8v7Q1Eiml6AT08DoJ0isbPuCezCaiBbeI5m0xdKVvPM\\\\\\\/RWlaKekKJjuIgm9YcNFZbidtpclL7gdVzQd8d4Kzrz\\\\\\\/5jn87QbjArPxmedn8Y4vhvBbADJqqmxnOTj3SOuaT7PnufdriuduZEJgcmTmPAibBQm6MWz29uEwRwlU6dlr1WglDRO+j+B4NkUZ924\\\\\\\/PJh5ctECECVIYcsRE3y02riaKUcAu66fmvm42U1W8Qfc+0SkpudgtMdbjUzVrnUKQBFxAp6q7Yim1RUPRGWaBaxehWwYjJUmsRnL+W8XMEefAnbHsVIrl4f+8BqSYWp9Xk+HZsbQ0rroi5ijwNdrEBEUJXX8YnbdtcghfzWACbTYr5Wp3Wfe\\\\\\\/TYaMvh5Sgm4xslrtXBvXfy71MWXvUsqcmpB9Dd6szxFESul\\\\\\\/jRzwd9S3+ygdArNedyuR288DOUjWXVsy9C+TZI2lBVhm\\\\\\\/c\\\\\\\/XGY1DYLrb\\\\\\\/+Lqg3qX+\\\\\\\/grhY38sY\\\\\\\/WYgfoPMPVsPM7g4EqrNiPVuWcVkcb3qQzY\\\\\\\/40n44jPQuz5CYZJTITq3Y5EdPVsGqHniqZPTg+B9zKZNlWZP7VNlUqe23hrOE8LTOlwpVIKTRpyl81aEOPfqFa9FkcQiUZYcC2QyxF0JQ\\\\\\\/KyqqZze4+4aM8guKy35q4wY+TelXuIEM+LLAcMKRZQbiTxf86Y5OVifisq1h+ljF20ONlWavrRB6TdPIRTcntbFD5CwDa8ZmbCgUkhD9aJkYRvsmEvKaMbvikx4MIwYG0UuDDmGHrq8qUGm\\\\\\\/Vn3TmkPSWUdBRfxYadtmOfxgMSoxq0b6GQTAJoXeYwa8lQOEtZOVelAKVz5AXZeIntveW95RrXBulp1tBqO4iWyiA\\\\\\\/k5R5xcTlz6c0ZOSgpUv8QmWcQJ0VgBNGmWtqqfKIhW5aUBYbshoA4CcB6pjlqDuICyGHuP7OB2R8r5XUgykPaMMR8XA87GaL3udhedugcMI9UMETAILpcg3uExoKY4NJJ5fhyLHXXDt1UotGtIxbNBcqj7AitgqKB31QMzRnKEDIzsJ+6LRt\\\\\\\/ObYOKnR4LbhAoYwdnFDV42XLSUIhR5zGhOhWMewHmGtnZgWatzAkCY23HpU4nihkO75ur1Ah3lNKcX18iqjfuyRyh38+xDppl1LsBNmGodMrUqYhXEWsJS7s27tsT0gCLrILYdp+WhRirPPQs8ysJscpsxvFQZMt9FUQh6alaIWOe1O3hLC611T0lnkOudbj3aOpUKbe6S3QDFbVXMmV5eX3khU1Tgfbg5syNgdI6UjjNpdIGqP2dKhdcAozD5nKewWkjoxfj8DKrFsIqNaAM3ReFiTqTEIXLE2JK3Lw6ilA9tmontZrnbn1nwmaYtfp7CkXMGarZz63uEXgoZ6xaa4cRByMR5NHypoD65EgpwMCwoIIXqTcfGW6YaLlHhhfRR3IzNvoB7tEyfAssaRWoOUrtjA9ngoEJ\\\\\\\/AAOM6\\\\\\\/ySRPaa7dtL0pf1pzrxeJez+yYMzZGnmaUcGz2lvjK2\\\\\\\/XgK0wuXqFOJyhpQ3JCFrI5edtt1I1dMdHUWXLMNrmemk1xUPWunEjWy5Pv4GKyuMfgoTpzkCQQt\\\\\\\/lYFSlSYB5jnJDKkGaZCKX5o5qI6iPDwThygVIVpCU6YJBEhNVZzzXfYU6ZrBCez401AWg2SlyiTCncOZzeDDyjiCrkww9okYDxQjYcbb9uPrupI\\\\\\\/RryNpCRiuGSr3KUVgrxxJAP1zw9nLg9gB1aPgFNAEJ8fH4umE\\\\\\\/OfSGU9x4uw+XJYkbheFzZx9JbDF8XWePKgoSUuHK8I7ROpF4NSChLjGFzpsl68XBhL5Krnv2YgF8k3o7v6sH3g642ZUrif+wc4EcIbLyQhgn9S4XQvUGCvtpYgiSsB6ufB0n0PEMAG0w9OWrH11b6cYd5IcaVp38xQcmbHjWxxmegVxdlHcFEpORPJlCzxfZLM6l3QYxHdID5t+ICQBfXj6jRVRDEVFeVpo8nyvaMMUgl75LQFIVlwl0HDJhAmawdIXGmyiMin9gQ1iDqn83kfICqo7A2rQ5HC1op\\\\\\\/2QG+23ZMkfdelUqECbekY7zMShF+i2LGSKNk3hj4dyyaWFJMyZ2KTgmOuqObhb9PXnknb74eWleRg6tgXBZHHBXPIec\\\\\\\/sGuLibUDKriLr2bu0YsstSK+NsO52AfMTIKCQZFIbq0FdMKpBqx14\\\\\\\/ckPzujGJiPwihwRmPudSCoZB7XgM2DrDxnhFkFVQ33lZSvJ0RVxFACodrAF1hJPGbYQGHHWmAwawkSAx4oOiOl2Y93I6Bq7s5YCGp5hIWIRdvbdgEvRhcltLQ8Zh\\\\\\\/3bbDMi+ft1ANrXUAkkaqq5fVvQeBFCjbiIJLp+gXxoT0ab3c+qO2dK0O1KBFE2Y4PtPJrTbTeIe9uo\\\\\\\/7mAGxM\\\\\\\/8Z4yuqp5OE8uN85TiNVuSb3c+gEHqzoSYtakSFMXG8Lf2nvXe0ArBHqmhQm83JoKiKUMtmGJf7W6DrC+xFDCXHd5Ni4hIetHhUQ+N3tWCE\\\\\\\/oLqJzJ8q5UIGjwQuKs+3dC1zeoPb7SsskVKzT6aKPdXNtgbkAT0A6SKYmGyAJQGHSw3aQtbTzGGof5dWuJ49C6RtSv5yAzMvH1AgI93K0ef2ZJ21hRdRt2kL9nLUACGeuPDYG47ewoj9xHgtH5VQiCz36qIu4O4D6CGV7JeXyDvmZ2qjy3dDk3B086ykfOqjFYmWF8ZcEZ+O6px0Bz3nixlf7NAgJde7O6qe7coNXFRAGmpzoRlB42R3To1ZDd3WLGAZ+cUh7aK4UrJvRel4a4UqhGnqJaT5P+XMGCwBqrmKC9MNHJBnO\\\\\\\/MWSPjQMRERd5rJ4E82I88F4KByDAsx9Tlh1j9q4+gtn7JXplClpb14Ue8K83K0NDztidO9uzuexZ+ZHiKKddeJICbBqnMfDeeGvaxPsYa8\\\\\\\/UH+Jn26wyW5VizgNUA\\\\\\\/s0KWu2mkz+\\\\\\\/phSLYQKUTdrb64RgBGWSLMXtvAW6tmvqWdTOZQw\\\\\\\/vip4AnrybC+se0wYJNyrTv24dnSWV0yVDm0SYl0iOwa0CmMXURB6fPPQufIZuJdjFnlnBkosGLKacUuhEFmcroMoFmCw7UXy9Kbk96yXcJxAzhDQuoYCH2bn\\\\\\\/spMsu6MplwiPFqxKjgIGF3azZOWbLnSD1QdpOlyNfBYKWbTqE5gDUuYUjkBt0r8ZlbRQDSqoVGnOjLDOhyBBogmOZG7p1H\\\\\\\/+mxgCHWBEZ4KzrIhvKa4LhzgiixapQo3xlBsOiKG6DmZftOy0reAEk3PU6f4SmLhU76ySEftxWAhem0Vwc1uS0yCQkg8nd6p6vcU9t7Uy7P5KndX9D2u74YnVcKEimMHOX5fLcUndgvBWuiSKOc2JzYHvDEMxbpcw9ue+RhZfkhxi0TMmpFHLAEd0\\\\\\\/qGLYLB7j+eGJ0VDJyqnfjUdv\\\\\\\/jt2PXHcOn+Qdeq9oeIptc+Yzd\\\\\\\/FPZDO6DFOk6cftOsw6OBPl3MU5QybOTWxKbT9RZbRrq9LyBJwClXLVu\\\\\\\/a1VPJalntWKopcMmIN8eBCUntMVC4UkfVYCkX19DIKLzznn+ekSnQO7LEK7+qvD6HnLH9+hStKDdFRFR1uHQIB3BNZE3jJkQxE+dPwdsrJngunGHAl8ksJOlllW5KVj6k1yO+qVcHQhOlIshABMakDXaMfv42xVL13EWsA+X1lepFrXOoep\\\\\\\/ISjYk69ZHkYqqIL8Y809mHnoZf54k\\\\\\\/1sS5BkxvBy1Q1AlamNybh85IbGTPNTft0QrlSZ6KKCQUphsjvipv5HjmzbL7Gj2NHwJpqL+yAejYPe8MwWICZwR6uNBnZbY+SPNSK7\\\\\\\/dtU\\\\\\\/GZaON9UhcvRGK6j0Djx4mimYO\\\\\\\/yjq\\\\\\\/JooN5SNpsF90nhEtFmRNwEdMqqpNCVsmj0rSVT5UTO14t9GrVXbb1uhM4oy5v7oT2cnBe3D3+KW2SvquZwWPwBU7AGxvOZvfnXq8nZ3m+tRXpNCFc5gJuRIl0aW8I3udSiJnacISz877XbsOSeZ7oGzxXOlT9iwqzDubtGYCjoboDUmr4aau6uFOjEvn1vQ3iGaCbcXWflT6xWfIP8zWGO4\\\\\\\/10vhX\\\\\\\/Sm6b6BjP6c5mq9oX24V7ia7esYCYy1g+X1M614aClE3N1ShRo32gkaqrmVWd4Q+VIpN5U1\\\\\\\/ScOpsf6DQzfp7XYpM\\\\\\\/du5jt6P335JPTpAxvSwL0U7hvn7pLLCLycfwTRhu8TzTiKvl7ZVaO3Aw8yYbxvvq6aEofXGTxgEhxLEtY\\\\\\\/\\\\\\\/0gIN0g6yAUVIZdbvJHa0HhdwAYO5EjnWqfg1yHWvMM\\\\\\\/8lEyZvTBWYsNsw1Lsy9UZJ+MUHKOV1NFOea3GXov8D7FSOpSpv2MJevNyYwn8fD+bz7F5rHvuWepNsj\\\\\\\/N\\\\\\\/lhuSWVFq0Vc15V+jD3ogRm88I2oHT3uweO\\\\\\\/cb39Usg6dCHasGO9OJ2WlsVrZVevXb67u3+CZcna3uhNW4UXA4c7GGi56aJFxyWHVpnGqETW4H8X2eUz1e5lPLaach5L2oGaWelJs4D5gd8B\\\\\\\/4a0IC0SzyfC\\\\\\\/vYG6Rg5RchnBf7OsyLuhCO97H7tTFnsOjNoCMa14pExI4xXnKAHNlHsbb3aoa0oJ0yUb1wXrlEbX6phCfNJOnU5gp+5O816h102nDnulptDk7oPDcTBRATaKfO2aQWU\\\\\\\/t7+VZDEFik9lFVueUtADyVJwaNn9kHLpFTF17qakrXLFiHEoz0UeNpq\\\\\\\/utreQi34+WHIq+z0vL0ndhkcmPqY\\\\\\\/VbCpg9HnWvFUT2KfQSwnFstTBL9vWRFsnEmMzNaAcwOFAocS7h3ionpGIr4B8LCoa+BAHlIybYc7dFtolBVoJrzA3JWxOHWnKp61p+BfdzxASDvZurhCYD84uQba\\\\\\\/p\\\\\\\/9aSfUAbJlzZgPyoJNV0A5aNKs3ByEkwQvFInflzvAIdMnPG\\\\\\\/N5btp334fzBsooiwjlVzmdv6Pkel+N\\\\\\\/DWDSEk1cuwh8v0JNp29Ud3bUrtTZFO4YBLW7GkTxFeEvzIb4oNXl40tEK4B0DODEBQ3y0BSfsEYL4R1kPkRY\\\\\\\/U\\\\\\\/ZSlZrwlqXbNabuk6ES9rHgo2Hh+BPEwuTgfPfrKKSNE92l1CfbsNavV+0D94MSusVW0TeXSw14QarMwiSrAPOXYPrqzJndWCEbwa+ckabA1dHSa0jsa83+qOeJuF886ylAVTBKW1mbmgSVkQbuk4mkA9LXxuGOir52+Il8Ithy02xuKWyxSVA6MFH10LLAjFr7rZldMbApc+wVQvK58jM5UY6zUUkkvzT3hDKNFZuWty970dAC7+6mDBBWDbSnV4i7inAeSfyoUuU4fH25ShGL6YLyHV3y72VGxwRx5stxfNUjEZfEO3wFrF2laLwAEaHWKtpIDSi7vi9oX12IpNmpakizfjCL9SlL6532pCIMJaXtTPOkW6EWy8YtXN4DClSyD0ZTSuBdArYYblYnyVQUdhWTrQhGovZ7oAjrDI1aGyO3DfSdUiAso87z8LlWWSV\\\\\\\/l4OX8Q2pXOJac4yXBb4s8eVPo1HZax27t46S\\\\\\\/QriIc61Km3oV9SfyeBaCYyoatHujKKnPfYi3QDN3ozgR0AS5NctoxMvjJ\\\\\\\/OfZ7x4Jd8TyW3a7KUBfEaCZ3\\\\\\\/U6CSBi0XyPaSUx2ZaIyh6ISPAYBZTNYUP3Nrp2EihvAH1qAw3+OZtykPpVe1WA2VwTAssuDRuBjtbPnj9DTVdSuXU4Ew8Rrwq1PHuWA8plTdAdv0TC15jQGJgPVwHN0sNZL\\\\\\\/J6ILnkBKzyhSOy6ayTQ62o7RcsvUc0OPJDbTi3a1E25Lc\\\\\\\/\\\\\\\/rDi\\\\\\\/iB3t9orMBWcFGPsWzbVrlW9XGSHI6ZrAgQoKicAldG6+sISjW9348debsd05FMZVH7h6KDQlo8tzxmt4+MrKZIVv0ymfUw3si0tPV9wt9GIoHK4anB91Y2YAqSZurvBww8Dly+uXOV+gEPI49HFh1LMXDJyIo7I8FYqUlgN8IfOgdEiRse+s7v+xH1TSIKAuZ6Nbo7Uf5By1vU3RPKYp7ZBoNS4cCRBy28tuCDNnTheemu+bL3Ig47Bi2oKH+Q\\\\\\\/qWWTubgbLc2H7QyOOtN4yiy47d5tOJKbZ5fGyS8TifnLySq3uJux6\\\\\\\/KN2EoW3c68nEOmPsT22jDBEXtm9ultcq2cYSy\\\\\\\/Slf0y3tjDQUzmVp8J7q\\\\\\\/\\\\\\\/nIt8qP32\\\\\\\/m5i5pEBCkE8H1sQXhCvjgZeEAve4bLhmMWU1QS5l\\\\\\\/Q5VnXE9K5+oC\\\\\\\/NLsT7oyzfCPlTowsikTSGLQH\\\\\\\/h\\\\\\\/YJh3qsjDZKP6\\\\\\\/14+ypSIE+QrrpLT9QQHUBX1qcc4j9m65kOy\\\\\\\/\\\\\\\/5idTfPzTxKjFVYiWth8AYDK3FVihMQkZexq+LLNu3yC+BOf2Ldru58LlDPOiu4rDQB2wZ2dbqalLNhG9t27c9xyOMxEfXh6OL\\\\\\\/Gk6HE9Jwj8Vx8InyscXu2FZSspKFioqiVRTEXSIdZ91kTb97MScij8CYe6Pr0Rn50W9RbuL7G\\\\\\\/kWk3DnoAFYUkquwwdvYm43H5aDE\\\\\\\/sO3k+300po6uXSa7zXIUwy2FQ6jAbG21zGjJ6hVvIbwBX6jVeOClPKcZtoVDhHvjibw\\\\\\\/Y0PHR3O0kJBiniYfzdMzZcbVdFw0w54y7v2R\\\\\\\/Icm5ae3hXVrBFncQT8G0DWAEV2COVldfQ0dvhCVeZLyK\\\\\\\/VBf\\\\\\\/r4fwLFWe644SzSNkXrknFjJQniald+sBOyBll5M1hn\\\\\\\/ywogonOI6OCKaHrsnEypOtRaighReCu2ev3HjaF1o5tHj\\\\\\\/F2nHudrpEeasyySnWcB5lN792PU\\\\\\\/wkMf3+74CkMHkJ543pnOBRHhKr2j+1fDwTmmFaamQnWpAweKmnNgOH0JIfzbJkbO+9mHNfOgMBsu4\\\\\\\/Q2KagOLTVL6Q2t4C0P9LL\\\\\\\/WIIZYjucVd8+Yix5O6ic0cmozPRcWDkk9fcorVN0RXGiusvEFu\\\\\\\/NHoWKw8rx+nBIZpRwIF4eJ2fiWlEqJtZ74kmRFDPHxszDYM0VRKlwQHtmxysJHa4596yfze4KLFCzTtlxXV56o+a45f9uSRJKXRver37Q9VLJOdbUm0hVkBmJtcwB1AB8G+SligqiPymRj+rlrzqu5dvh4Y3YNRD+rm8G6urCFFl+z\\\\\\\/eJsoBdaYdZ7X8KzW3mE++jO+kg+W+Gwf8gV1EQKMwuoWVHSxHl9MI3EzhDSOXacpJ6yQtgd1JX9+AFGQMdlsTPlVwlHnEBe5jCCb5qdyJOnL64c+b8UgvTr8eBuEccJ1quuTl2cpmOF8sKKOD5fewjOuD0pgrV1DIlqKv46jSmWfy9oC2rorw84g8HLfKDdj+D6MemY8GTbdMBil285bctJ7bRSj+xonKxEBBwhbWivzevTVy31ymKqLYUpXbrIYzJkr8sWqyHciCmRfFZQN8PlGklc7kZQElLb2w+VIF3suVHkPtxthATkLAdDmHgO1CR\\\\\\\/vRea1e5gUmOGdo3m6d++\\\\\\\/ntBCPa4LN0MBDBUBdTf420Eg405Eniw1K+043i+dj0ZMTSiL7dTIiBzPaUTb9pWxyYlJ4QrnVs2x2FxXO1MALIRTyvGo8YYDoTh+99FTHvl91mSGWGSTB+NkcKRYmQbnKZMzHRgYksWly82S2t68b85D1NrJHb1WkLrHgG2igjB2Dh8Cp0SplPnEM\\\\\\\/gNyKEwyDfTj6YxCEqPKZoFfTF41FpuCMIbVpG\\\\\\\/JvIWX05x58ZoQ5WN9lqZobk5EiWTPGidgexYEvu7J99Qy0ircDHvFswO3rJx3q47vRbloYtc7nVeDfG2vYs3VMWconkxxCuQe+VemurX95eAVA77xfD8KHhQExhl\\\\\\\/slX2QJq\\\\\\\/e0VeBxhPID0NlsmKaHUB0OTM7iIKp\\\\\\\/Iabwak\\\\\\\/3UjgFj7tRLClsgYI6xrxgdGi722f7+kuAr4PMrPXdMTse1cU+G4KAk+7\\\\\\\/tiKWYS9G5uVxKm0xldjn3iAPRxJ5Ef7PCAUNxzC0OzyurBo8\\\\\\\/saY5YjNN9hzu+EgjBJD2ng6NURJt16He8FQHhQtsHPVTmGiwdMIDeBXdJ6uhRS6Pfwz5hYxKSix2yDnKv9vVRc2wY8w7o1pRuaBh3\\\\\\\/o9Wb9OPAoZxsC7FLFyxd3fIypj8FvqDn2qyMR\\\\\\\/5up1f4p2Qi3zWBpymcmhXDrsaHvTDQxJZgUTG4OCt37ZICo0ronMBaPuvhap81LZXMCZD+v9VnShsC4aEKYwp0ichhtuWJ2aE6kNAzYT1cyREQa+n9JrpYcQSPCeXgsvMpRbzxGH+5TFYRKiMZxxfzUkhUmSippNHTeQq7mYci45THf1DqIrAXpn\\\\\\\/94yWs6WPluWC\\\\\\\/ocPwfe6eNuOCwtP9bdLEjU7oQp+1Dwg2foS9D0YokkLLJW2G5adOWJdXTqt99b2dyfveCTBwXsU29cyxmg+YHt\\\\\\\/M3wOeCmURv6EXnZLVsAZ6YmUBydynnYndq81rBaWSRCNvX7B7ErHD9hnIBUVHHRwxAU13Vs4Y0X6sLeI1VWNdyAqv3eLkcGlo3K6aOz1ZSJpkpUiWQkqKXBV4E54w6oxJKQ8cCJQ2skW\\\\\\\/5EkjREDA6Pb46XvEpUb1ozs\\\\\\\/uLtxLuYOb9VRIMYl7WAFh9hjMf9fAXnuF0v8g48sqI9YQA7ePqiiVq1Fy6IzDvNaxKmipyTsQwwi84Skj3WCazwJk80jPERIq3WUVY7TjUPRE1mdj+yKxLjn1jF8txKQAcBei5zvMrNTg6ZTbu5\\\\\\\/I70kyP8\\\\\\\/Eg68R5sIq5e2AI0j9r6coLLjH\\\\\\\/5kcoV9rnoqs89xdJppVRz8I07l6OhO3o\\\\\\\/VY6BqDuQoWDUXDaxvd\\\\\\\/MX6m18cEdl4lm5KIutp\\\\\\\/8\\\\\\\/h0UDmLDA9SIzqwtew2LLCrHjOVulYOTqxoQcrb+f\\\\\\\/lfpI9DNgQ\\\\\\\/KG412KIwgwBHzeIaTqdU8K3BZ\\\\\\\/Urn+UUutQPB44nPj459uUGToWd1Tzm5Ptaihf6cyF+pgIu2NNywiI8GbH5+jKzEdpvlOt1u7FDB6nLkOAmp1mfZvEpuHwRljWexcLIPgYw1vCkgYJWObbXe+7rjnYTO5bOiWXzB2ZiHVZHurQCeHsEb3S+dAYqZQ\\\\\\\/vpO2zX\\\\\\\/KPwj6HZmMr6UeRthZDQ9hmVqPaQFdzvwckIT9ERA6vO4v9+iyuwN0oiYPy0m24jVZ+eGJjn7FMI9ard0XWk3A9hknSnB6kFVTBlGsqnxB5OKY+AKOFCL9ay0rlxB5CJNz\\\\\\\/2XJ49q01piPHfToeb4icIFaUFQ6jaoHZSLcR9ZNdSsker0yeMPBuOet4uZ8NQ7FQDbtMnKf++NWxpAUEGTPEw\\\\\\\/FUDWa7AOpQsjEgPKRWni66slFknMabWeHbXkr4KdXR8SP4d1c\\\\\\\/OAqlZLpQUs+FyT53xGyDnWcZJbwT2ZIWNbi7JCeww\\\\\\\/e73+jSxNUXBBtVcYkwlJBhN7wV3pKplf\\\\\\\/zwwxofEfrU0shiGQtPXgrhB5DNlj6od4p5dTPzl0Wtf0sMkRH8aW6aWtd0gMa+EAiO+Jwlj2Q39jDUwZeVM7uQT6u0ZzV+\\\\\\\/lgtbRdMPIbKzhm6gmmbGhY3DgQybBDZuKXigo9sTQnwjd3qrCg9sNM9ucLUbw6iS6cxF2\\\\\\\/lWo1rfs4grbiiTiHo37Y6wMpEbEDPIyobJUB1h44mSuosfc+x9+wCdHzo+angm0udoVPjLdPx\\\\\\\/39+nAF5jGR6+XuB\\\\\\\/GdXWQ62COP4s2h1XJfxv3FA2oDrDRA0UVAdkcdDy1WgVkBprbs8dx\\\\\\\/Ghuvtcngqs0oPKSUN76o6GfYi\\\\\\\/Q64VY9tV+AAJ9cDUmAUmikHjg7WeMg9w9QSfmibF7KOZDY72Mun1NMhDY2WmHorHNnS0dy7kBbWvFYtzC5FGurhUidYG638dGNagDbn+0xmJluBMNWfe9amfan9rAaQ1ZFkMTF3xKcamt\\\\\\\/gyYIFfBxusxjqdJ8\\\\\\\/NRSVfR0kkrMxzdyQMQRF7Mq7zGpRzUwQINw9h2rI2SXLKul8\\\\\\\/lDQlsSlk8mWd2D2t\\\\\\\/aR1FX4zbQr+icCWGqU2HvZswOF198YVkG6gvkRMDCa3e93pRa3QWmyqlDSxzTpjFT4nfrs0nw2kB0sShhUdkbdz7ZpfkEM84XEHra\\\\\\\/368JPsZt6Dx66vcr5owUMw9wq\\\\\\\/He1+kbhV\\\\\\\/QCiT+DbqMEnAzXi4t9rrcsnoSZMC3iuLedtcNRqd3t+Dt48Dv4UU1VoxLg9xZ58azCUQ3gKSagBhCvPQnJrbDQ8H1IJ68DDhTpqSS64RCnjAFJzDNwJaOtxs\\\\\\\/EW0x+FoNbvgJbotiKu8T1Hn1aOjsPpNvamlGAUmlHiYh3wcrDQfAgj9oWb+9edNBaqIvsLkvb5+DaduJSPhq7+Cd3TZgwR4ZeRlg945k\\\\\\\/TngwNw2Gy4gK4\\\\\\\/4lVDlRQRslCdKj4vX4Me9KEUHTVltGuWQr3D+Y+1owmHj2gZVovkQr3g8ah1ZPlRfZoMVs4kvauv7F70to2MTrsx65dJxrCI2MyuUYzokBcbfpIy18gPiJuGmJ7wrEiapSTpgEsIgMUpGANla2qZDTHuD2Aja7Yt\\\\\\\/zbZXg6iFhbc7lPrtUZjlQyCvzTBFAXzgsGQqyGEbJ4CarqXS84PUIuG0NruhAA4Nm+xqiBkoxQTaXfY+L5Pi62fZ7hDZx6MXZoUKCAUJylBaL4gY5gagZsXPBRH7mtN+aP7QK6Rnl4ZRDAxA6Pk4a21S3Sx+3PtL57DhRN+fL3tCeczMDCIdoagUKmNm5Wt3suHjvBNerpGl3jCjKnLmgVBCQTX2zK3zL0db1kX0LQFbbrNtcOHGVSjQQ\\\\\\\/fy4OpD2DThakqPOvj6BWog4LX9MMdpbmglX8BrQJduHQOc1D2X6qkSfquVP9vVvcylwgSctNe2mBeAoJdFll+Pt4ukwRHCm+IvmUcEjVfnpn6SboprJw\\\\\\\/lsnT3SAyr+8b1Eij2I0gdlqix3MIpMd3HAXC3KW6Wt48oSuB0kys5voOfmGOFQNcUwkS\\\\\\\/Kpn4XdhHlhlJJ4VnvfX2Rf\\\\\\\/U6YGTZEk\\\\\\\/JaIPedoh+JNHz+1mtivcy8W34PwmfH678QoC3cV8v3kWGWKzVZnhGcqWOpC9qIfSN5ciMOVO038tMvqgorXtuC4pVnQ9Dg3Ru+Dis86BTTW3gxi3BM5IQ9DXtPLDJe5ZPmqaje4wrpWwlin+b5DSrCQpkHBA3tWlHLyf019dNEsnj9KAVqJyrC+7uyy2OyXDv9hb0MWyYjmNjz6+unFeUsFQ8lWEbcqOFr0lY4SxC6OE\\\\\\\/P32OPF9N9a3\\\\\\\/1STa5Xs5CMTsFVJ8Vn\\\\\\\/w\\\\\\\/DR9W6Y20ykLR4rGn7ZOqu+bCVwVJdCfJlxM5+\\\\\\\/ajlADcw6d9U6afq4p8gH8PE\\\\\\\/PYwQmPb7zVyXqsYPGim3zyUTPxQsUMkUAg9fHyDNoBcLXu3EWczshzJdH8d0fWguB0NGBD5dJ7AWBkPiCS2KTcv+j59Q+GUm3NKvJL1dA7C\\\\\\\/q6DoUZGdpEOHx7baem1H8KrSHcNLxGBnz2HWsU3sfkFMg38NRJE34\\\\\\\/A6BeZfvED0GVLLVNqaGMISSKH4rnCOmmcMtFCKpq9huZ99HbZzbzsz7WFplSuDh7aQO5DdjPbsUXDjAQFx3hOUiBAUmVJVdpJJc0SJvOzL7vNLEgcP8TR1Fd\\\\\\\/+NmE7PuYpIgtk8C1lt3olvTPKsvDhLGwEukVTqk4yux0gmZypJgRGgeVQENVU6AzTisqsIikd5BFYDdWqlfbWagU15jsgpWOdUE1PDqQtlppcTvtK+UC51u6wzwRvK2ixJC39prmuswbGKg1zW6ZwXFUyuL\\\\\\\/u+oswLLkj6dz\\\\\\\/fE6QWzXMDyRtlr7mCJaBD889U8xQWXTspgk3faYJqs21uu2ruAnWjTwysLJnq8nBauF6X+QXbFFK9tRSrAy4+\\\\\\\/hH0IcP9ujNpGB0mzI8HSZVPbq\\\\\\\/jD1LT1\\\\\\\/56XJmqWJOcINz5rGfe+IxZ3NkuYGHI2NhVTZM2jwtQ\\\\\\\/sOumPfyTPJQJ1O81C1XG\\\\\\\/gMsnH0ychOURQJYKqbao5mewkYSfYx+YUwH6N9PBGJLHpqeP1BJ4l0zy29zcmLaIfU\\\\\\\/idyJ2rAFwU6ZK6IiJ4vdcd0xMcL11SbRDcJd6EoTnP4RQ+rp2EMoZXSYPHabQ7daYrY\\\\\\\/H9R1XNtm0F75kEqdn0WeF+5B0R0LmLhTSmjizHfIdccr2N8ALi8lWV6v7oRREflaowhc4dFGVHYhhMum5l78LPM6Hfd+rP+yXFafwWEVcU2E8yVvWqSyUp5GwHZ6NROIm6UfeAjwp6hjTReujMWccuTVPSLSR0ZMLYPzXkiQrjXlBtTp+muVdTQnQsuAFKTfnSizu1Rb3+M7jKss5dm7BhMMBLEM53G9m+ivRkFjs1mSHI28b8h9Pj11CNdZPFA88tpTp\\\\\\\/XxLZnEHyTijea4U5YONbyQBrKUuMG8MFhiJFvsiC1v9esQ+dx8YhW7Kpuf7F2NZ+ZG8LSqPExgaK6RtdtCgZfRESh3r2MOQPwQox0bjY0Ox7Fut9FQdzYRqnn+K2ku+4VTVJRNZbauLNR3DYYsV18lAP5n+2kk3hN32h\\\\\\\/wyT2COjpIKRs93EP9rm\\\\\\\/\\\\\\\/4W+H346NyycLijJ0OTyRi3OU\\\\\\\/XwPgASOH7cmxzl2tbIpoHbaFsLcSIfuObHPJcIH7bip0WcW\\\\\\\/eTs37swbzI75G0aKtk2tA8G8ycQNfMNvcjfwEgONRDiULasTmLnxeZzBYo8sz7yFjd93V4R0Z5WipWI2kndCkX4FctzqhFOhkUX9ELLVWf0BkF41FTpAtZWDzJWoWm+8Uct6essqlLrKQElBAYAyBn4nBPYD2gvD2uTsEVAVmT6JUgplxWCRjG63AQXudrjog783paZz4F2gi1OcCJjrv9kXFpsQBSMMsXDU8FSS6hzKug2Hi+T3vdGO2QyQxr\\\\\\\/JifOkIFNC1nztIeaucBzcKr8WZKMz654wF6JvRXtmaoh\\\\\\\/re2\\\\\\\/dxkj4WhFn5EZ26eJM8q8nBWH+7ewQ+\\\\\\\/J6G3\\\\\\\/lFdZ4dbe4EoPp0RjyySeZk\\\\\\\/cu7ws+qZu4PoUBk9KNT6t9mnIUip0wu4ccknEiPqsjhFRmDuDbNvNlqTXwFImKo7Ja0XjGjiG\\\\\\\/J8GiQ1e1DI44cjjj9y5Sgr2CNhHTIoqoKo6FPSErpNfsTPgFvMGbDDzeWhGyXUA3SgxXeKmRdZ8bFpmZ1IYoK676Pvwi5jOco2B+csiVDflcFG0Rp5smBogU\\\\\\\/WAyTFVxCW+YRogOuAoxQZINEb1OYnggQHjFg09BnMbmHlojYpLB6C8+YC7wgVCsKa6JbpV+zDoWzTPCegaPswv+KmAlZfEvIbwxsO+PTSfE8uuwFnfTm9wRTjcpUV31CFoYm01jSY95GNtRNiUQWgIUxIYe1azpkXi\\\\\\\/hxg6OD\\\\\\\/FR73Uq2waySq89Qwwcu8pmrCwldF+gK9Fu1Y6Jk6YrOJvqn2sGHizh0GAzSqPCl23zSDoCrsgTokIybBTpuHf67PA0pETrKFYocTwzqtnZ2eoNX9tYI8v+U6XuYUEwn6S6tUNsryoCbsok47jdeDWuMNHioNAliwLc3Uip1yTPi8g+2txSX5eSc8Z2TJ06iPuOp9eEFSZTmiWJ\\\\\\\/bHsEk68k5m0KPbWw7axs1xxx6tOzD\\\\\\\/QJVpz9jiBM9ZPQy04w8N0yraESfsx0pqM1UN+lajaZG+wxtGwl9zKyjAKNwAe4211OvnTKLAUhxtrJVbi7gLLAV\\\\\\\/RP2ggGFxrdH9m5IB8Ns25XYK1baPfg7oQ3YwzBoQeBtwShQ5TffFvZq0XWtr7XLkCsZ1EaqIkIGE5\\\\\\\/FINCVCnibDUMDTzFJKv\\\\\\\/F80tDyN+es+3FvI0TCiVSL9C\\\\\\\/k+CrkhRUYTaLpFPke9ZWMAN485XVzVPuDQiwof02xMtqvUvpOn8eDmFWsexoVOOnjZ\\\\\\\/v49O8I+prML7TxwUBnHRetqu+5LW\\\\\\\/a5gIp2It1fA\\\\\\\/BUmKX8ju0M5xsmOJhU8aj5MTB4PGlIbkmzTMBv2MiWJzSGq0XCjvl5WJjDB+3AxNuejTdcKDb2qVL12yoFyjMQl0ZfPsxJ+7flYf8ELQELvkNzuHaJ3D4bkcyzAZ4z9b7f\\\\\\\/4UmyWHPy1SqM6moVhEY0uYScrKvei4MUm3IwJJpN7UrkMLd8mrHAMm36nKMZ+2CLEyzCHCf+kU3xtugvisQ1+F6U7SYcY4QA40HiwqQZVWxh9ZrulQMUK5meklsxRyOg8YhgGxp+M9n4fAxfaLXccWKHf7WYIpJg6291H0qbdt3wmokjbfYPYeWGXu3poDaSjDkMkJrGJp3KyVZov6JMu0jsbotS9jQ\\\\\\\/nEQEAFWRG9FPWHCKaV6j8B2UK6Te9j9MYw4sKep69vhQWAZaD2JSQFrPNd71qZD0\\\\\\\/1wsFdv+kkslw8h\\\\\\\/yEkMzngS2g4cGIGprPtzZehPL9JIktuh6sQ24TFqUYjJ9OOgsHYzYRzA9QpS9ylydeWzu83vpfAzvuIN1jJyEmScbDJxnwdmEPuQOHNj51oa4UQz+JDIY33wRRDeqoKTG2f\\\\\\\/fUOyFsOG\\\\\\\/qsTAvL8Qf3Fo1kFHviekkzRdoSoub8w70lZ1WMmqyjPDqdklAE8NDtRvLEskLcnu8lARMaTam4flmJoAhaQrVjLM\\\\\\\/DU3fs1uh5wA4ZbDPY2PJX0u6mwokEp2ubMuCet2i5pQ4z56fQze1NKs0Zj9Th\\\\\\\/sQjUGEOkAR4MyDwcpns1cB+39yDfyC8qxM1Wna2Ot+pasAmPhIqJd8SR0CU2TcP2hqURirkzISvWARpBWW2AcNr+Du\\\\\\\/0hbzofdsEfvjjXBmyhU+bKjMCPo3rNFrgyfMmWbZdLyq+fNsiFWfySNsosTdt9bA\\\\\\\/7MuiCuygn5e\\\\\\\/3Dr7B9hSkbAXn0YR7wd6S7e9PmzIAjLiAghfQ1IiWFvBUspTMlkdcjMlTFU19GsDb4vpQxsEBCOVMcqaFNn0s9RdXMFTru2jMCRcvEbcmd4jyOUPHrltMn9LWK0GJzptKWnE\\\\\\\/QUfuLAQQIFJk\\\\\\\/r6d1Ie76zE3AjwFvXt7lprtEkNS418LxnOE9L6XratxFHDbZWezpQ2p6O8T1xfmKqhkE8rtXXCSH3uEvyTUJ+0+6\\\\\\\/iuMkvSawMR2zgDJWR+9cG17KusyNqgFLUF0+wf2SbinVc\\\\\\\/6Y80DbmkOpne4AtskcMZPg3wxPfQlfquS2LYYk184OygP2dGDlAFCBFmkpMkSqt0cpH58z0sBoCNcVIXsYTlJyBWj0MQQN0i6z7kggYAvXZ6Jg\\\\\\\/Z9IuJfu8emnEOgCW31bNJ1m+w7xz8viyQ5Rd57pSa4Giutspup2z3ajgXk0c1Pfvyts0a4Y1\\\\\\\/nySa3IuhF1yVVRFeATHk4PaFle00yX1RgawslE3\\\\\\\/rdwAEXDDNSf9Shw\\\\\\\/Y6+IJWt1N9rOgAaXjEgEWBJ47z84UhYUyMJa1+pZ9JWykGZ7iXlGcAXeurVZGuoC2siKo5OxIyNXYXiNytrfTYuUJm8stwLwstqej8wmDZtUq7I6xMhp94dHHklNoRf71slmIQPjru1Kcue1hDOROY05qJFEFyMVfDjFRJ8J4uk6Nkh+Ey01HJ74dN8ci50eD+MU\\\\\\\/ia1EQUFmm9C5eST7Rr82geu82T02KZouzO9ye4kYuhI+pJ1u7hOwxJtJ33+cKe7M4y1MwThf0l62r0ZU28QLFd004i7E5a5cxVubyYU5C7uIpKyK0SN7QzXYBTFeZhrUdgTZmyGa9C7eM5anw4aF42k2WvTfzCOA43\\\\\\\/rIY1UEfxOQ3+\\\\\\\/LYNdORnVnIVbgG8bpOIuSZXcc7zKBnfvxpeDMe4xNvUuT3FZe\\\\\\\/THCAUHc2KlfOilzgCdcK6PttSgMZ79+\\\\\\\/1CQyjpoRH4ctXivxh+JXZftvWi6K+ymOhLiGjOkUUTYZrc2lYiLCbuMpbyN+QAqtxs6F\\\\\\\/sWqgRLg+syhQVPHNwYnNDfl\\\\\\\/mFXwjHHBVteTyhSbc28g5DMoqumMxMfQ\\\\\\\/oerhy\\\\\\\/qBdK2HAiFhLrG8Q\\\\\\\/uK38ss28PhbBAUULP4mp\\\\\\\/hZD\\\\\\\/RJ9wEDlRUTSl1HFENM7gnpRhBi6iaqOa+dBsZ8QH0BfgYaZ58dQCiiEAy1MidY43vSwpNClwCHq4N4A9cmR0unJREmLeX0et+O2ZuSG8Ffb\\\\\\\/Njqf4Z86A29f4dC7VQLO9JfeIWnUKt5ikRTLVdpqSl+Cs0N+1DkozP+mULeAlMRts68E3Hpd9Y\\\\\\\/8\\\\\\\/x2wuEQakGnHjIwckG4G532653FEH\\\\\\\/4klQSeJpTtGnjL02Aix9FtSzV8\\\\\\\/lgOmGkXHb6bCuBKJ\\\\\\\/K39P2JaGq7iV5RCqJo2HbS1O95yizXWf2IvGnoxbFM\\\\\\\/40aZvp\\\\\\\/EoquuD5uvipjodNKjLnwYl6rYVg\\\\\\\/JcQkTc69g30JdAsScfQpVLjxE6Dh6MbNyTSZcgktlm+7h4R1kAvKlTwSLQZ43aPGh2fKD38O+MjLlbmCsHawxFqNmi2FpF+T7T6V3eFra5VeDHqFj3iYmWzWrf\\\\\\\/yNRlRAxYEyduD2A3ewKuyeTR+F1rsciObjH84hm7F8pwSyT1dx2yF496smpFdfDBa8hFoYVM79Rk3wu0FEA07sZK\\\\\\\/yE3VasE5Ez7NAujgHukJhNaDXe0b2UaTRdf9G8X+0iQvnbulMYhdYnVKDPSfAsDsXb27d9eIqeloKP3dnNsXriaasSqW\\\\\\\/2Ab+ki+Y3r6FYqv0jN4h3Ug7SDpoSgUIxZDhS0rnRlImLrnY1Q5y6zzY3OC\\\\\\\/RCL1Yzfhu1mW\\\\\\\/4e+DqiFJtrStwAycgGGil9dxSVTBzup\\\\\\\/HKaudpeXTbBj+ZSfra409WIiuUXsj4iW1Lb2RfYHNCBYGn+2AffRqnVd5EsXwIpmd2t+zkaiNndD+zWtPXqijEJ685GiqUZxOd3w7bND159FuGyyDA3pNH8D3OiZVGeSG13o1j50PNjcY6PT6O2boYQElagGIDAs7Ao5Ke2bL3dW22+7Zggl9WOVHKE3MNnuyNwsvpk5MzTVoaWZSR2V0pjVi0d4FEtPxhFV1iRJGEbN3q3dgEOhssfyLbe3nJXsxrccIA9BKlH8McDsIo6qfl9UHUEnupBenB6LKXvbxoKmdVTlh44WuwGnkYp8m+D2iuDSUxMlLFkxlA2hPD61l8GjxOfrLiypQ4jM+7XDQwszvhsECs91YYdIpyNSJs0gyHYvl0Spu9lffSIUWwkmvVWvPKuGRSeABzvLtr3V68L4uYhqrUDDM3T9ousr1pGIhzBwLhq0KQTciv1iNN+dCKnZY2aC1x3JJBf76ypZ5UgwpH8O4u8j0AhjWr+Fdm5AwswwZh5I\\\\\\\/2UhdkaVuwMqS02quC6Fx23wEv5IVGCCQhI6fkEHSm+eS6XIrCzwKyq3oxxCo0tbth8WWd94rholzDM5gIWKgwQcLqgoSSgUU2lhL2fJ1ZcZk1CE+rdQaqh6kbGt1xj6VpmIs+R06aRJXYXwQngyyTNF64tv6P6lgqIli8zvP85vJCeOKSJyv+5xN3xakyClWxyauTfmXgeDo7V2mWktllgiuy0rU1TbCtkx5zGNpl9LW3HhVnR5WkCZBjuQiBgtQsB5YkEqE0\\\\\\\/bQd1j5E9La8\\\\\\\/+vYtwxPV8EaDmQH+M61SkhW1iOrFw21kRN9HLtPKzeEmGSMZvke2YVSqmkYuhUusVSbP5Mk3ZSbaTo+lKcswr4y2CW665cSySBXw5WNSY+2xkYJ4GizBR9siSS5tqNkBBUaaFLQDNesYpy1RNlVRJfb3p6hGTq4UGR8jv0qN0909GT8fshX1SiTj7Rt17r+cUYVHWdcN0phJaLhRgvz2GA2\\\\\\\/sKyrPOlfjx1BztIaeZmMvucMPu+cfWffEER1ugJ+Ck8rCyv1o99TkvJ4vfqjFZOM\\\\\\\/MK+9thWtXxvwS6JyhOQ\\\\\\\/k7yVavtvwV0GrfVGb\\\\\\\/GFMfSCHgYky\\\\\\\/UBVVez\\\\\\\/unt1nJIV6IrI\\\\\\\/rJVfQ9HhqkEvNcnkMfnrOh3KpURoStej1hKinRtCpV9Z9kf3\\\\\\\/YYZmcjyumO++mXbY8WerAj\\\\\\\/1Phuq14K61OW5pmn7yldjqX4lar8LrduhK1TuOzipUi+WU\\\\\\\/hawy\\\\\\\/b6roo9QJzaDD8zIRuqIRxdCJxqdyL+mBLFqzKa+4eRY9r795GlMtJkYFK7IZH1WaU\\\\\\\/j+i0V5WNyUUsoUJdnjjzFSptGT3EijQb8Zg30RGvOJTHTw9faaHJj216R936umUTeFOnr0+AGXZ\\\\\\\/JZiKGFwCKsUtae7hwEa\\\\\\\/nIlqgcwbS33q++mLC8XxfSsotfO5YiVmebJr\\\\\\\/dF0CnqDvNL\\\\\\\/WskWifZ7qB677CZBDX38jkoqZNUp7WoeRLCCsbxePqP\\\\\\\/6OJlYZ4hzpEwtDoIIvINvUP5tFmnErPQHmKnfZWFMnpGgAyQ+P6GVNpg8tVgc6ZE+Z9cRsQH9bKIJjZhKK2XtiFsJj76Xte\\\\\\\/qmyHD4CDu9066X6AFcc32SmgYFxiX5CPc4rJBDzI2hh3XEcHSAeZIPXSqnwU95tCVNY2zOw4bGlD3NXpvDAy9vkizyK4wV+qtoZq4anwqkN9Yc7lRkXGiiv9JDtjC30nqXoH\\\\\\\/fDiGjwvhxbkfsnRLBGS80mR+t+UKw71xBu3erwXhRiCgQFKtc2UdowS1A8atS02mccnhHxWa\\\\\\\/E7cir7\\\\\\\/nwWbn7l2qMhwFxlZdQeOtUD3SpHk\\\\\\\/nbI8NkHG0pzCclXJ7XhkaZVni5zfJPmw9nvuh\\\\\\\/jyuksAp\\\\\\\/5hpmgv056T+WMDmnCnLsakLp3DGkYQyrxI3imh7lzhWgU\\\\\\\/WHcRKg8fC+Bzft6VrOieXv+M3vrMsqcLs4c4BusgLmzIS5OGlD\\\\\\\/UstkbyPfjhLZXqSv+oDZs7di6REcN3jPZEauLsYelQLXmoC48+b12hBhCcaoA4T1cCRqQjOiq3i5hJJicnBpQMMzeXkT1ASnUbeKE7rI3Ub2eXKMSmcTy\\\\\\\/xQGxLNjQbcD+zd\\\\\\\/75+yF2R+Jtx0WzfGXz2\\\\\\\/uPzMeNNbsogwh95IF71odY\\\\\\\/oKQMPZKhAZW1S8vlF1fY80PsRmKRpmas4qsWCSZM7haT9oHzslX\\\\\\\/wNrO4rzbR9YTMR19DmM8gOMpVqdraPxzqjTVgSQGT2YukTq7Zn22MPnkEL3rzN46NNjY8UzCfBQHamwto6GKKlzO+xMY499L2uFIgs+\\\\\\\/8iBunD8OJ23halukUG5xTlFtULzAJ\\\\\\\/i2plDZPe\\\\\\\/7n8ElGl6f2X5W0X9hbPoi5ICXeqXVrtZmwZzolIAj3L4YXYhupinzDe0UNpR6IRDksbqwG+YO+hD\\\\\\\/6TuRwnMhwzvyo\\\\\\\/Q2BYRyeGqoD1XQc15YUCMtWSoWlVzyqDXfdlrm2v8WWFdRS4UtWYV\\\\\\\/uOpRN5785RvKM+8O5KY\\\\\\\/KbJwcIN\\\\\\\/elT\\\\\\\/7oXYwfBuxbwoutRKAFAPfCXP+SJ3iknsp7bURixLRc4Djju4snI2KbZGURNVvPX2azA+xM0bDrvuYWJnYmZUOzRD5Zvr4W+cotzDKmFC94KcQq0KeyCkyTTGQY3wWNeMHfCzXqyJujD60OgBEnBnk4L80OkV2Qkk6S2S73xsKtQ4se\\\\\\\/JOkxOO3Wt98ONfY0bHxE4ygWrARpmui95KqRBtahJ4j7J5Mx460BQxOLtz6AJNd951HqkVk8mDr8A6UiLtI7\\\\\\\/WssPgz7yV4TUvyirJ7smFGf6M8g0k\\\\\\\/pAUewwYWOypQOV31eZdP6jyEKRlqVskVg49wit2HamAxYdrdA1cSkDCA0J1vGv\\\\\\\/VXl4MBzZT5V7PWC+CWxTW7DAHeWiGhveht4\\\\\\\/86\\\\\\\/pte+El58pR5xkcHNEUTegg0myH1V23d2kgEYmJhcg8dHs+DEifpKFmGGnJAR6vRRK81qpDbBPyhmOqxKEt1qtXlCoMgJOySDJznc5bSUgwOh6v82WjSAzKb3e+Z\\\\\\\/9rt6Obhdzwqj3vb6FJjfWr\\\\\\\/sH\\\\\\\/ZO6qAtz8KVhQ7P31AvDEQrASHIcMxYb3igawm2MGGFiMj9GvCLAGv4aCRUzfCNbJp18M1tK8q\\\\\\\/txp8JVNMK16BmSPENRashw\\\\\\\/su6+2Sk5pzxRM2\\\\\\\/WAU\\\\\\\/W0UUaMZ9WmM5OoDgzLSOVPiQiJ\\\\\\\/F6KHZdaF1e9AeU3wFHK4M0coiTli09KiSL1AB9RHpKNsyMVOOP8UYR0kt12SxPvmI4TkrCxrT0b2kOj3RX2dnyAVg3Qov2D+vysn+32yW2oiMw4wZ4vT2yiuqLI2RD5dvQxnCmSpxyMegEq3wdUUAgn4j+\\\\\\\/oojR1Cmaskpg4Td4tWKonLG4QmxflXCR4XkyWskTSoOV\\\\\\\/RrpI47qy8eI9SiAn6g6NpFY\\\\\\\/tQHAqmh0Iv50YHdEUk8OL6dFUAEgnAVbZN1wqzVEJjDmLxB6J0fR12fh8haeMkx+qAfbR8W427rP4\\\\\\\/q5\\\\\\\/BOy4Yw\\\\\\\/V8vB0A6Y7Bw\\\\\\\/9m+\\\\\\\/txPRhiyWo4dsiNA4gvdVCzWSL0II0+GuzLK+BcnlnFKcHxKYItVu4YN9OpFt+p+E+vtoe6CaSlgA60GT2Yjr+Lj47WtP9tVtE3S5ZpWQts\\\\\\\/fl0dTV5w7m0fhDIJQt\\\\\\\/2TtkFQGfg7Rq5Tqj1TLbTomqE2Rv\\\\\\\/2nXb6ciQOGfkEtvtT\\\\\\\/7JaCPAaEgxV+OYwpbToaRwCF2dxFnlX0lz7TkB5sZ3cCK7Z\\\\\\\/83bOoa3iNUYAvOXBU3hqzCEICfZuWx\\\\\\\/g49+3By5JiHD\\\\\\\/LRvgGUsBGJ5gvBoJRuIF3ZCAbJcv5g38BHw0Zn8UGbzRHMlw\\\\\\\/wkKf2cvZruV+qsVa4ClpUKQuem+NhksPqJN5jaOMxjscNfpNcSSm5sVuCL04cbAWXSuZcWKm9HFJAFXqwqAoT02LnuuxNgIlBzzzSoSqxqrC7gGvC0Brpk6+JrjG4WGXpHnSG7MCDJT5u94Hi\\\\\\\/ecdpFSBhXbKgss4ujh4HX8Y9Ev6ESmhhI4dFOWa+X4lAX+SCxY6wu7ppeKkVSWJvSeYEJs+mN57J9WrFOFwFPyGZdMSqRHA3yRWcPkZY4cak5oKwMqsdIWEyjhYXtuo3BnMP7y37ztgQK5AUcNHx5x\\\\\\\/Hrb9ocnhDoINpI\\\\\\\/oSbTNgGn82EO0GMiPgEbwsauJFv+rRyYtRR6zk3DjJQtqO+Rr1xCCiZJhLkmJPZEW99qPuJi0vE9Ug+9X+oeSY1Rt39HJxd8qN43jv0E7IGAKHK+w8q63Qo4BqMuxJ8f1n7Fc4nHzmBsBZWfCWYfCFFvs\\\\\\\/fYWGTszxdIruqypSw3TnuSYccOCTv1y3JHv8GuFVOAq1tIpgN03l+8AD8cTstfS7LOj6YbYC4IMGJ3ogb912dW\\\\\\\/06VJblsuOjzbGQykBXK9P8wIrPfgb+Mxf5LsSHAelh3Ezx4IdwrPWSmLHhjO5fvUQgbwjnHiRFqInYCAqSCcleeESI9pG0JA5bVijqpGwqKnvtzSJZmPTOi4hTRnnHHFF1u5HjAdcy7j\\\\\\\/Z2pvvzKu9vs+Be1ZJ0OGgM6HFItQaBClrZcr7g39AGmKdYwsXl9LL7XUully\\\\\\\/a3Q2komanb7liTqPktE0nbZV0FVs0tlQWK1Ldd3E6xOwEx8BctqfvQKgJSI414lgqUGXN09SDhXRFuztlYJKJ6CTDJJiVBTBr0sCtODaSlMCWyHggil7B0pHlJbe7Ac0y1mCnNNWebvMV7JXHHaK5u8HgHqNif78gQWj6RmTIoI0IL0OqYREVw+Dpe454X8Yqpd5auTs4SUtQKVwh4dBBZkfoKRL\\\\\\\/d6e+kITdvheUCnOAupcwlZzEWNTPwKFQIIeukFAOkIGnXKY1BVs0cKbeoLtAN58gEGiTMoxdB295ITJ7Kz4os2hu6hzUEj5yMUYk1kT+Mz\\\\\\\/hSxJnQBTUyt5bWisLSXg8RrPFA7NmhhbA\\\\\\\/oEKHNEkY7rWi1cS97L9OilQkR1y9OyhtESkA0znCexB6aO\\\\\\\/Rjtr50GZWZbWN+BQWQQjAdhfbN6BCDTLcGh7zZ2UPrrO+vlNGozoa1ox9JAVsj4OeiU44o4sEAKyOtit8pDLjXk\\\\\\\/oWljTT4tZ1Aqot\\\\\\\/HeRJmaTb4Rt5DPDQ6PRXtlSDKr7ppVlfxtTk6CBZQ7WEOCoVyrz1e63KUbY0hlfhz9mdlRu4U8B3oNaQOnl1fgbUysjBLWFL2UIVwaiF3Ukf+yrN5e5W\\\\\\\/b+VXSrO+e3z6WtqhgOOqUOgTdkh3vh\\\\\\\/78Q1XR\\\\\\\/o0lE3NcHU7pPKnhmlvDnJDHsXLLr9QuTM2J89HpwaaHGpJ\\\\\\\/VRDA57i6nAtGUJD28c15yiGxN8JaDRBFgMtsovHisjMz5yXsh8\\\\\\\/rnSlyt9P0Q1zSkc7ismbFA37qwBUNTBb57kAMxdKe2dplE0\\\\\\\/2r2bi0uQp2itd+JIWtUET9UOZZYQLclsg6Or9qGzN8AWcNRnnLm5702VfGcruPrjdM\\\\\\\/b8uEgo4TfXCvhKScqW5WaJjLcc6IEYE4lLBH3GUPnn4aKHf4CfrsGp+vSOUYW55E9joJTEZOfvdVy0hdXn\\\\\\\/BZIhOWfH4bFfuvBRh+j074h0RwgEZScP9VEsC7tfr0FaRD3qLz9bwn\\\\\\\/KRuXTOE5R\\\\\\\/1uJRKclsUQbttEYvPkGkHgPywgDgDHs4uoiY3CnD2Hu83U5eYMUzfJnYpXKWoHocPWgs4ce9wxdr2ecJgVQ7Y4e\\\\\\\/8OPyaWDkizgsONVRALEpNuAEctJqD\\\\\\\/qZwJqnUR6tOybfOFaWpJeXnyrjY3QB+3WJI4IhriQtXr2bswFFKAiPX1tFnSJLmt3hejIgCTgn1UjK6bVJkanNceHq+hft7qIxBAOKfsGuAFbZc4l178iuqMnKZne1Le3oa\\\\\\\/bmyK4xKTsOZ5t2BuQ7Dqcxyr1db+b3iTEblOab+TzqUkCqZFMG9YMRgGKsHv\\\\\\\/F\\\\\\\/\\\\\\\/sbUwxSP\\\\\\\/w0BhbDEdgIXz2Magt3aMpZM58ECkB3WudWQjEZ4GYR9WolvRRbDagn7Ck\\\\\\\/xJBH\\\\\\\/xiq2BiEmg6n2iZsSPjAYUKdGW8YZDnlQkFW24y1RmpKfA1Dep5mRXwHbe\\\\\\\/YTIlOXDSZH57WFvnU8YD8pMisF9awB5aoi2JpHEeua4tiyxmva2i7kueAiChtC39uLqEqAiHzOpuzuiwTrnU62LIZq7mne\\\\\\\/jmiGZ5Awblfkw6FI6d4h11WAX6DX5r0hm9E2Aor1IkJc2edWoVJ\\\\\\\/COZiV0zqw6X2fVsCof3rxBJHeIbruWL\\\\\\\/tDkzUNBVO2OtvXkbatrFZCx0KI72iHjbvSPeJxr0VgQQX3bEg\\\\\\\/wZhh1RvZxi0WKI3YXVn2SUYy+I3pbXYToODarR9ij9JK+h6tNCGNq9mzpM0S4O\\\\\\\/yyu9PKWYHEaPAuS5QsR54JKOmvqE9GV6QRuw9nIyX8n5hHQACv9QZ4oyRXTD1qflO\\\\\\\/+RoqoQ0bVbPQc8WsEXc\\\\\\\/dxtKhDqmxHnURlQNj0Jvo7gbMq8rMcEkCBmUTlC\\\\\\\/SIr8dzjRCOMW\\\\\\\/TSy0CwD1JEi0dw0EmF\\\\\\\/zmdJMFPlUVAcpHkU5FR9I6pBikDtt9Qg994nKIPpUTYLawfCGodoFCBgVgUO\\\\\\\/W8xlrAiB7JeI9hLeQ+vxPs8Eqekq8m77LGQkRe4lhyZb90wm7cA7SLNwbxJ5bWPd\\\\\\\/yh99IbaZj7dpws\\\\\\\/XVo3AP74IBlwC6\\\\\\\/HYD2JeTOdsF7pSaLL8YMarejjpfVF42AMbmHQOng1e8KyJj50CetcbK1ds6TIIUbmhQX1ukXRRbiD1\\\\\\\/ngIwjmqEAyoAUqXWzmuI2DvnbcI7A82e1cnE4jld\\\\\\\/omG5MjD6850Luzd+P2+gAr9mDbTaJVx4s33VEkFZ9Juxq6ftNMz061w4Zk7774bv7sVH6P2+T3nXxK38AjpYk7oars75C112nElb+lZjrXNoEVxXkCd\\\\\\\/CUZPPpQKIrkeIv909ozPe6kPxV3WeLCl+Kx6Si81wYGJiGwOWAoWyNtUG3bSS36p\\\\\\\/pSfmoIYW9ItUI4i0Kq+4BgBwh7vB55Umd34dvLXkIdsb3SP094nahQ+TrtGpQiFRpD8kATXBcLuqfCAb2otHKU4BM1uWPfr8E9jkEE68+4F051YHBlmNk\\\\\\\/xbn7imO\\\\\\\/Z9KBI9sDQh70zLGTDBTj5+nHqJtLQtHjh4ATlWvNZQSY1rEURyLTZ73AQe0eVarGsa7z5\\\\\\\/0EzUO\\\\\\\/P24DHt\\\\\\\/Jj9TLYAQUXJUXUXfwiiIs0gtMZc1EdvKBCmajBRaLaRPVwAT0GpDAGy6heO5FuLYyu1bdKL8N97lnymm91a7RHeyc0OIEpsFjdqiEt28LmQMdTmU7gl+NbERauVIWbb7KwhtkoKlY\\\\\\\/bbT+aPWxQuxqhjfjD\\\\\\\/ihwGAEZaUL3j\\\\\\\/3Vl8UZbykaKTn3QgUhE6ItCifACqRhBha42jK9ohdvha\\\\\\\/Bhxu5+oF1qIl5P+52ayipm63GKb0tVn0vuomTy8swT7mqA\\\\\\\/Myo8t2f8IOYvmW\\\\\\\/P4p0CvAQ+X7r99yN5I9Je3NE8aWvUfy0YlRmy8CxCnCprf2QUGS9fBFR1XDL7Ie27XFzbrY6ANdsfLjQ7CJUGEt\\\\\\\/ILjSvTfTmJofWLvQs3Bsv4SE5Y\\\\\\\/hsOx16oD4xvIBoLstAAFKeg9gNJkhIgfXw9\\\\\\\/S6BbQ\\\\\\\/uSjf2+2T6S+pVNI5JvJhpZLJ6lMet1rLbKvEnQb0BvXeZNxhNc3B6HQYv2q4zSovGeKbJ7ZymleqwK+ktcsGMXuhCJjcjuWaF0XdA70SpU2iUi0jxf9cbg+xvvew9tpAhbvUDrhDPo35t796rly+GOrmIjN7jldiuChFDTJN98ZBWHdJoDRI4+m+OAkqUUpZ0xAKMQQ4veXeZx5daUjD4OJzcXzagsXesRGq7ot635YDyQHBU34b565RcPJBhr4AY76cYeRHmWn6b45EfaV1oq+aa35qpaGOWcPAOoLb2af1lHmlhDgbN5zQGr\\\\\\\/IR23cTQ9j2HH95i779tn5OxjtqLoimUOAYvHdp0kk6HwyoAJ1mZUOhw0fhGv2UP\\\\\\\/dVm0aSFxxhy1ZDx2iCmEANR7OQIcLWWXMExEdX2Xd4GzNjpmuTX9teZeJXDmsZ1uF6YJ6Ni08SCw6EG6C+v7sGLNwrpjFOrHsGFmHbjtHJt\\\\\\\/c6wFLorB\\\\\\\/\\\\\\\/RAbksSs7wNvae7yisw1QKvrZgZDoYPTRj+xTd78hIK0F2cW26rkIBKDtGiHpIeEe8OgBnGiba7uHm15UChpV5vLuPBNLyptvhlLOiOb\\\\\\\/XL3m+Yp1O8n2l5GCn\\\\\\\/OBIc7\\\\\\\/LBSMjJKqpKvyZXVqc\\\\\\\/7i8+dINEm+c1B\\\\\\\/VzwaJzjkYTFxmEki3gdijVYDEKUOFD3cx5rNJTuXzZBxkKp07xXx1bQ1+andvkLwPwsSMh9YYmHpI69BuUVkFUEUbz5C5XCxu2qgXlsbLYsLfB\\\\\\\/rWicXZr6Xa\\\\\\\/NRo96SyksmLqjxazyrKwOjinXCh8fUkQwAznnUpbng5tHqLXra9rn5hnTuqYNaDB656+US0a8RmFlilNNDDDguK4neBLIE5ddyf7eow2YhUxn8GD2Rq75jVVOGNeG214AZOKp4H8LNn2IDpU8OwfCLsxnRqid+GgwSYjtSVm8jWibg0Oyk6h6jM7mhFN7CV0OQH6fmJpqbvehPLdtBZGD++mLuKunP8RkzlyWkTeIViBGW7uqJGo2f8\\\\\\\/dDNH6Ina965QOhevZtIjAv1rLDPlGte\\\\\\\/CzyKPRb6yOix\\\\\\\/eb1j+hDohX78mU8JVA4\\\\\\\/IPXKBRyKnSMENj+J62nvcNM70b9\\\\\\\/AHCzUi\\\\\\\/Itvljd9YMNQx1x1s88B3jAKD1C3XVQJxFNYEGz\\\\\\\/ovHe1oeBI4mkqes9JHPEuHfDpn8MPBpLxjgIeIunbRT7+aD+A7qd5uE\\\\\\\/sIykmuiszL3RyvnO0Gt4ck2fbSTSIx6nJfUvzIZdudY7\\\\\\\/PpFujaE+5yYETdWhKQRfFntu2o0bsO+p1jDVx2qH4NLo3o2n2A1J\\\\\\\/JDdH2fV7Wq9N5pEhS9dJudJOO3ICqSBT8oYWe4D1xPk8gZSN+rSZuMbgbTnwLZNiIG53JHsjfBrXNAhzh8QUV9iXzkLSBgbTjziEyL2GEEcq2FEmqeiJIKy7E9zv42nEffnAHL9+U7pqtWsTvvoGHB6jbcCloIC5tDnEM9emi9G1rIXCpC03c\\\\\\\/j2vECAGvNT6bLRLFC\\\\\\\/TcpkZjhDs8Y+YWB44Npvlh1BrhdqP2YDpeB2tXCLEalS1CAL6ApnJdz9jJdqj5WRUBM4yxKIbo89jDFN6qZHEBeTM6cDL3Dwn699E7rNCChMymrUDVPdzLvcFvRK++NB3wSNvyLt5CUzR1O5d1GgygIetrVyzypXkbhl6qUyVObPpg7aYhCfLdiDyipbXgDI\\\\\\\/SSU\\\\\\\/8Jg9jlmL9Wqbz5kZ+3bFolOUM8hidWKCieGobXXOpTisKB7vAXJR2I6vBPW2Opi\\\\\\\/oJSf7DPyEltaasfS7jIKuwpG0eUg==\\\u0022,\\\u0022iv\\\u0022:\\\u00225l8Jv8cWoLOj\\\\\\\/vmXF8CFGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220d0b0e300299b56f37d0ce2fac64225dc0e25dde17a94399d27954e6a5427c09\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LepctaP1uAMGZBMX7K3YsHjp+5Sco3j3+BfiplEnxqFsDc1H2X\\\\\\\/g+EKaUKiY9RG3Nc1w4y\\\\\\\/wbx8RWvaMuA9mOTiqllH\\\\\\\/XcqTVhjcJz\\\\\\\/4+nvAWeQp5BZGj0UEDVE\\\\\\\/1x\\\\\\\/0G+6\\\\\\\/aR\\\\\\\/2Y3oU5Y9DVTx7bKIZzJ1tGZcgpBcog8F94eDl9CtRkA\\\\\\\/XgRadNDsR0UgaiLneJGgsDNe12FRGQmqdcx1D66xKvnZtzZzBadeIVISdObpjpGQOZbdYL5OgoFGgq1A74BaDgNMLcGv8mnuOGKbkNrOsf324Hg+1HGPTQU9k1ySRCBifo9KVSZpUj2T0Ao4ZSMUS4b+DleM3v+DNy66DHiHrxxpAeN8K02qtcxgQ4S1eNbNXUDkX68EJ8FeD3gjiHEy4OnPvGS6kAMkxd7Pqrlsc4Vg583oWhpOoPhZyFH2y4KXlgJUDOiiXthGjf2mdl3BoECXqn+vFvYODb6xpUCOh\\\\\\\/HW3CCYy3EcCx2+FSNtO0J2BbZ1tFQZ6VBCscdjZGH47MZtLy\\\\\\\/Hf6fvxYXWAaY0yOjPTunXY\\\\\\\/demiTYv35SPvvP4wJvG1R0Mmumgd2PPqis5Q596GrKoq1X\\\\\\\/yiW5fYva6p9YSB8Z\\\\\\\/Xdu3+L0wZUiMpEi4H7N7OUP7giB3DOncKLXdcdJNXOcu7UW1x\\\\\\\/0y9J0pnHKX+ucMMu2uAEvDb5zotYogXE0Zl4qon9pIsn0h+oNHZPhMWGvxH9x4wA2Sfpa7ifFIE5H+88MZZw8w8Ys\\\\\\\/9j7kwuv8FA500Ku4WDuzCbtyVwnO169bPE5hGxlMyprEEKsv9J5kkiUk15EUftjpNMnCtzbGBNkQ9Z4ALE2ZUj7M+B3nDSfcoLUOXziAfRNlCzNGijsDLdv3emGcT8FiNztZSOHp4ZG9JFEIGqgQAZfskW+Y+D9Xn\\\\\\\/z6gsCf3dCrozv56VwybsEaCX5P1oIs3O\\\\\\\/rZ\\\\\\\/bqm5jfe\\\\\\\/cJEAQFBc+T59z23Ey7Wdzk32MsRs3zx7n\\\\\\\/lekzPeca5dY3FHjW1cHWzjNfERd2lN1V9sv9eyb\\\\\\\/W4EQzSI7T2A+Dse1hi1DmU0nM8fNPiPHoj9TRLnKFxxJNKFVY5JRW9UO9T62uzBPPGdzjWu7fxq0K4xHU3fj+rXlN\\\\\\\/z8u+advAdlrAHUq5\\\\\\\/2a+VqqlzeHSjKoWIF6RRRN6LCdBuBEGUEj2st0Mn0HrHlhTYiCPNmmUBbFx1HLrsoxeajKpzgw+yBjPx5rhgerOMaebqS+SKqqah1zr8N1aKI4v6k23iGUIqjSkzdPme9kt7q41XqhFMzmIaUaNisO2g0u8YxcaIQ58WOYagoiQTfmHYf\\\\\\\/jGjxHMldYlFwz5vpgbLMSzANoIB95v\\\\\\\/7XEYB+o+fm6TWDcqJ3kilUiwFBdRmi16kPtmTzwpXGm5HfXl0Byo+mBlX6j5AZtyhxMCmHmP3V\\\\\\\/NPdMFSgm6olZdS09ZrNS3PuMDMNR8aFXZgC3ixMdoDf+\\\\\\\/Q9EV5ChQfCtMG\\\\\\\/BZ0bILwilhVRQarET5MT2cJSxSMKWgtgbTyqX1b8DjHdU27DA8pO04zOUKSTMt4Ws0twnDQsKRvWVXIvJtgRG4gSp1\\\\\\\/2j6fPfKFLme1mybiPB7iDxalVZUV6NyzK8qx8F3J+bO\\\\\\\/PHKRuXZwvlztlfa06dMWrqandiKeS\\\\\\\/XdC5CpKzbYvTnnIkERJa7T7WdGiVtcRjSKjZbYXPI+AgSbf7TCtDiiL3JxtsnqPlA3kOX1j3uTvj0LKtkqIWCWM\\\\\\\/zpzInDYzbS7gsx4IMO34aHpOIWIuc8qw1oy4TDKYRI42yI5ZdnGtlamfN60r5NThNose5vb+ZDQzhpOYvyh08vrjwpQ3eiFvei0XlK4ZogZbhp39oqAXP3A28Zi6JrEuVqgzjIroDajniASfJzO0xgkJHuy8mW2\\\\\\\/Xv3guOeJfiFQfT2quwfRCpOQ9E9Eu38\\\\\\\/fOqnRElrwCZJ\\\\\\\/u2U4Un6Rf89KIykYEeF5ExPd0+Q3yWWAK5B9ZrUofqAnf62o4UIf9kAOJZ0j3KRhA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022rmIJl8wwY6RfTIMFke+lnw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002220e20d6092226c30f99db04768c6bff35279bf8fe41536a5f1534e975d820ce2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485318 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2209" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GDeidJeyFgr5SrbcGoF16Z9nIXeNO5b9MGxFHau1q_E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2209" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--SCONTlfCCaTGx2KeLQuDYU1FjU_RSUJrY1lFoLsckg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2195" title="Cold Fusion: Massive Karyotype Evolution in the Antarctic Bullhead Notothen Notothenia coriiceps" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2219" title="Whole-Genome Sequence and Variant Analysis of W303, a Widely-Used Strain of Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485318" id="node485318" data-pisa="ggg;7/7/2209" data-pisa-master="ggg;g3.117.041780" data-apath="/ggg/7/7/2209.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Development of a Medium Density Combined-Species SNP Array for Pacific and European Oysters (<em>Crassostrea gigas</em> and <em>Ostrea edulis</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alejandro P. Gutierrez</span>, <span class="highwire-citation-author" data-delta="1">Frances Turner</span>, <span class="highwire-citation-author" data-delta="2">Karim Gharbi</span>, <span class="highwire-citation-author" data-delta="3">Richard Talbot</span>, <span class="highwire-citation-author" data-delta="4">Natalie R. Lowe</span>, <span class="highwire-citation-author" data-delta="5">Carolina Peñaloza</span>, <span class="highwire-citation-author" data-delta="6">Mark McCullough</span>, <span class="highwire-citation-author" data-delta="7">Paulo A. Prodöhl</span>, <span class="highwire-citation-author" data-delta="8">Tim P. Bean</span> and <span class="highwire-citation-author" data-delta="9">Ross D. Houston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2209-2218;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041780 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.041780" data-atom-uri="/ggg/7/7/2209.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485318" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alejandro P. Gutierrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alejandro%2BP.%2BGutierrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gutierrez%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlejandro%2BP.%2BGutierrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Frances Turner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frances%2BTurner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Turner%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrances%2BTurner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karim Gharbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karim%2BGharbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gharbi%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarim%2BGharbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Richard Talbot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BTalbot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Talbot%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BTalbot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Natalie R. Lowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalie%2BR.%2BLowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lowe%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalie%2BR.%2BLowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carolina Peñaloza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BPe%C3%B1aloza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pe%C3%B1aloza%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BPe%25C3%25B1aloza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mark McCullough </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Global Food Security, School of Biological Sciences, Queen’s University Belfast, BT7 1NN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BMcCullough%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCullough%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BMcCullough%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paulo A. Prodöhl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Global Food Security, School of Biological Sciences, Queen’s University Belfast, BT7 1NN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulo%2BA.%2BProd%C3%B6hl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prod%C3%B6hl%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulo%2BA.%2BProd%25C3%25B6hl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Tim P. Bean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Environment Fisheries and Aquaculture Science, Cefas Weymouth Laboratory, Dorset DT4 8UB, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tim%2BP.%2BBean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bean%20TP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATim%2BP.%2BBean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ross.houston@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ross.houston@roslin.ed.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/7/2209" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485318" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2209.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485318" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2209.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485318" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/7/2209.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485318" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/7/2209.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485318" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485318/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">SNP arrays are enabling tools for high-resolution studies of the genetic basis of complex traits in farmed and wild animals. Oysters are of critical importance in many regions from both an ecological and economic perspective, and oyster aquaculture forms a key component of global food security. The aim of our study was to design a combined-species, medium density SNP array for Pacific oyster (<em>Crassostrea gigas</em>) and European flat oyster (<em>Ostrea edulis</em>), and to test the performance of this array on farmed and wild populations from multiple locations, with a focus on European populations. SNP discovery was carried out by whole-genome sequencing (WGS) of pooled genomic DNA samples from eight <em>C. gigas</em> populations, and restriction site-associated DNA sequencing (RAD-Seq) of 11 geographically diverse <em>O. edulis</em> populations. Nearly 12 million candidate SNPs were discovered and filtered based on several criteria, including preference for SNPs segregating in multiple populations and SNPs with monomorphic flanking regions. An Affymetrix Axiom Custom Array was created and tested on a diverse set of samples (<em>n</em> = 219) showing ∼27 K high quality SNPs for <em>C. gigas</em> and ∼11 K high quality SNPs for <em>O. edulis</em> segregating in these populations. A high proportion of SNPs were segregating in each of the populations, and the array was used to detect population structure and levels of linkage disequilibrium (LD). Further testing of the array on three <em>C. gigas</em> nuclear families (<em>n</em> = 165) revealed that the array can be used to clearly distinguish between both families based on identity-by-state (IBS) clustering parental assignment software. This medium density, combined-species array will be publicly available through Affymetrix, and will be applied for genome-wide association and evolutionary genetic studies, and for genomic selection in oyster breeding programs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/pacific-oyster" class="hw-term hw-article-keyword hw-article-keyword-pacific-oyster" rel="nofollow">Pacific oyster</a></li><li class="kwd"><a href="/keyword/flat-oyster" class="hw-term hw-article-keyword hw-article-keyword-flat-oyster" rel="nofollow">flat oyster</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism-snp" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism-snp" rel="nofollow">single nucleotide polymorphism (SNP)</a></li><li class="kwd"><a href="/keyword/array" class="hw-term hw-article-keyword hw-article-keyword-array" rel="nofollow">array</a></li><li class="kwd"><a href="/keyword/aquaculture" class="hw-term hw-article-keyword hw-article-keyword-aquaculture" rel="nofollow">aquaculture</a></li></ul><p id="p-3">Oyster farming is one of the most important aquaculture activities worldwide, providing a socioeconomic contribution to many coastal communities. Among the numerous farmed oyster species, the Pacific oyster (<em>Crassostrea gigas</em>) is one of the most widely cultivated, with a global annual production estimated at 583 K tons in 2015 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">FAO 2017</a>). Starting in the 1960s, <em>C. gigas</em> was successfully introduced from Japan to all continents for cultivation (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Troost 2010</a>) due to its high acclimation ability, rapid growth, and high production, and as an alternative to replace the flat oyster farms affected by persistent disease outbreaks (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Pernet <em>et al.</em> 2016</a>). Accordingly, the European flat oyster (<em>Ostrea edulis</em>), an endemic species to Europe, has suffered a decrease in global production from 30 K tons in 1960 to 3 K tons produced in 2014. <em>O. edulis</em> is now a target for conservation efforts to help restore native populations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lallias <em>et al.</em> 2010</a>), and is also a niche aquaculture product, particularly in Europe and the USA.</p><p id="p-4">In the past decade, there has been increasing interest from researchers and industry in the development of genomic resources for oysters, mainly because of the economic and ecological importance of both <em>C. gigas</em> and <em>O. edulis</em>. The genomic toolbox for <em>C. gigas</em> includes a moderate number of genetic markers, such as microsatellites (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li <em>et al.</em> 2003</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sekino <em>et al.</em> 2003</a>) and SNPs (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fleury <em>et al.</em> 2009</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sauvage <em>et al.</em> 2007</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wang <em>et al.</em> 2015</a>). Low density linkage maps have been developed, containing both microsatellites and SNPs (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hedgecock <em>et al.</em> 2015</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hubert and Hedgecock 2004</a>). In addition, quantitative trait loci (QTL) analyses have been carried out to identify genomic regions associated with desirable traits for aquaculture (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sauvage <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zhong <em>et al.</em> 2014</a>). In addition, a reference genome sequence assembly is available for <em>C. gigas</em> (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zhang <em>et al.</em> 2012</a>), although a number of putative assembly errors have been identified (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hedgecock <em>et al.</em> 2015</a>). In contrast, genomic tools and resources are scarce for <em>O. edulis</em>, and only a limited number of markers, mostly microsatellites and amplified fragment length polymorphisms (AFLPs), have been utilized for the development of a linkage map (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lallias <em>et al.</em> 2007</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">2009</a>). Recently, the generation of genomic resources led to the development of a database containing genomic and transcriptome resources for <em>O. edulis</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Pardo <em>et al.</em> 2016</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Vera <em>et al.</em> 2016</a>).</p><p id="p-5">SNPs have become the marker of choice in genetics research due to their high abundance, codominant mode of inheritance, ease of high-throughput discovery, and low cost of genotyping per locus. Next-generation sequencing technologies enable efficient identification of many thousands of SNPs in a single experiment using either WGS or reduced representation approaches such as RAD-Seq (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Davey <em>et al.</em> 2011</a>). While the medium density SNP arrays typically generated by direct genotyping-by-sequencing approaches have been widely applied in aquaculture species (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Robledo <em>et al.</em> 2017</a>), SNP arrays can offer a higher density genotyping platform that is simpler to use. SNP arrays have been developed for most terrestrial livestock species such as cattle, pig, and chicken (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Matukumalli <em>et al.</em> 2009</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ramos <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kranis <em>et al.</em> 2013</a>), and also for farmed finfish species such as Atlantic salmon, rainbow trout, catfish, and carp among others (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Houston <em>et al.</em> 2014</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Yáñez <em>et al.</em> 2016</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Palti <em>et al.</em> 2015</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Liu <em>et al.</em> 2014</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Xu <em>et al.</em> 2014</a>). These arrays have formed the basis of genome-wide association studies for traits of economic importance such as resistance to pathogens (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Geng <em>et al.</em> 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Correa <em>et al.</em> 2015</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Tsai <em>et al.</em> 2016</a>) and the application of genomic selection in aquaculture breeding (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ødegård and Meuwissen 2014</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tsai <em>et al.</em> 2015</a>, <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">2016</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Vallejo <em>et al.</em> 2016</a>).</p><p id="p-6">For oyster species, low density SNP arrays for <em>C. gigas</em> and <em>O. edulis</em> have been developed, with 384 markers per species (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lapègue <em>et al.</em> 2014</a>), and these have been applied for parentage assignment. In addition, a <em>C. gigas</em>-specific high density array was recently developed, which contains ∼134 K SNP markers shown to be polymorphic across populations sampled from China, Japan, Korea, and Canada (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Qi <em>et al.</em> 2017</a>). However, a medium density, combined-species platform is a worthy addition to the genomic toolbox for oysters because: (i) the performance of the higher density (133 K) array in farmed <em>C. gigas</em> populations from other global regions (<em>e.g.</em>, Europe) is not known, (ii) medium density arrays are adequate for many genetics and breeding studies at substantially lower cost than high density arrays, and (iii) there is not yet a medium or high density genotyping platform for <em>O. edulis</em>. The major aim of the current study was to design and test a medium density, combined-species SNP array for two key oyster species, <em>C. gigas and O. edulis</em>, and to test the performance of the array on hatchery and wild populations from multiple locations, as well as nuclear families from pair-crosses.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sample collection and sequencing</h3><p id="p-7">The DNA sequencing protocols for SNP discovery were tailored to the status of genomic tools available for the two species. Since <em>C. gigas</em> has a reference genome sequence (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Zhang <em>et al.</em> 2012</a>), a whole-genome resequencing approach was taken with reads subsequently aligned to the reference assembly as described below. There was no reference sequence available for <em>O. edulis</em>, so a RAD-Seq approach was taken since this is suitable for <em>de novo</em> assembly and discovery of SNPs within RAD loci (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> 2008</a>).</p><p id="p-8">Samples from eight <em>C. gigas</em> populations from different geographical locations (primarily from hatcheries in the UK and France) were obtained, each comprising 13–47 individuals (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). These included a population of 16 samples from lines of oysters that had been selected for resistance to Oyster Herpes Virus by Ifremer (France). Genomic DNA from all individuals was extracted via the CTAB (cetyl trimethylammonium bromide) protocol described by <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Richards <em>et al.</em> (2013)</a>. Briefly, oyster tissue was incubated at 56° in lysis solution (3% CTAB, 100 mM Tris-HCl, pH 7.5, 25 mM EDTA, and 2 mM NaCl) with 0.2 mg/ml proteinase K and 5 μl of RNase (10 mg/ml). After lysis, a chloroform extraction was performed twice and three volumes of CTAB dilution solution were added (1% CTAB, 50 mM Tris-HCl, pH 7.5, and 10 mM EDTA, pH 8). The pellet was then washed in 0.4 M NaCl in TE, resuspended in 1.42 M NaCl in TE, and finally precipitated overnight in 1 ml ethanol (99%) at −4 C. Within each population, DNA samples were then pooled in equimolar concentrations, and these pools were prepared for WGS using the TruSeq Nano DNA Library Prep kit (Illumina, San Diego). Libraries were sequenced across five lanes of Illumina Hisequation 2500 to produce 125 bp paired end reads.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485445/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485445/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Detail of populations sampled for sequencing and SNP discovery</span><div class="sb-div caption-clear"></div></div></div><p id="p-10">Samples from 11 <em>O. edulis</em> wild populations from diverse geographical locations were obtained (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Each population sample comprised 13–15 individuals, and genomic DNA had previously been extracted from these samples using a phenol-chloroform method. Equimolar pools of genomic DNA were generated for each population and the pooled genomic DNA was digested using the endonuclease <em>Pst</em>I. Standard RAD libraries were constructed in three replicates following the standard protocol described by <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> (2008)</a>. Equimolar amounts of all libraries were combined and sequenced on a single Illumina Hisequation 2500 lane to produce 125 bp paired end reads.</p></div><div id="sec-3" class="subsection"><h3>SNP identification and filtering</h3><p id="p-11"><em>C. gigas</em> WGS reads were aligned to the <em>C. gigas</em> genome (GCA_000297895.1) using BWA-mem (v0.7.10) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li and Durbin 2009</a>) with the -M flag. Potential duplicated reads originating from PCR were then removed using Picard Tools (v1.69) MarkDuplicates and SAMtools (v1.2) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li <em>et al.</em> 2009</a>). Local realignment around indels was performed using the GATK (v3.4.0) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">McKenna <em>et al.</em> 2010</a>) and alignments with a quality phred score &gt; 20 were retained. SNP calling was performed using PoPoolation2 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kofler <em>et al.</em> 2011</a>), filtering to discard bases with a call quality phred score &lt; 30.</p><p id="p-12"><em>O. edulis</em> RAD-Seq reads were trimmed with Cutadapt (v1.7.1) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Martin 2011</a>). Data from each of the three replicates described above were combined. Read 1 reads were clustered using ustacks (v1.30) with the parameters “-m 2 -M 5 -H,” followed by cstacks (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Catchen <em>et al.</em> 2013</a>) with the parameter “-n 2,” to create consensus sequences for each locus. RAD loci absent from ≥8 of the 11 pooled samples were discarded. Read 1 trimmed reads from each of the samples were then aligned to the set of RAD consensus sequences using BWA (v.0.7.9a) (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Li and Durbin 2009</a>) (step 1). Reads mapping to each separate consensus sequence were then identified, and the corresponding read 2 sequences extracted from the trimmed data. These read 2 sequences for each locus were then assembled using IBDA-UD (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Peng <em>et al.</em> 2012</a>) (step 2). The read 1 consensus sequences and the associated assembled read 2 sequences for each locus were merged using flash (v1.2.2) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Magoč and Salzberg 2011</a>). For SNP discovery, the trimmed sequences corresponding to each locus were then mapped to the merged consensus sequence using smalt (v0.7.6). Duplicate reads were marked using Picard tools (v1.115) and realignments around indels performed using GATK indel realigner (v3.4.0) (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">McKenna <em>et al.</em> 2010</a>).</p><p id="p-13">SNPs were identified and genotyped using PoPoolation2 and SAMtools (v1.3) pileup. Reads with a mapping quality phred score &lt; 20 and bases with a call quality phred score &lt; 20 were discarded.</p></div><div id="sec-4" class="subsection"><h3>SNP selection for Axiom array design</h3><p id="p-14">A list of candidate SNPs from both species (containing 1,691,005 and 117,235 priority SNPs from <em>C. gigas</em> and <em>O. edulis</em>, respectively) was provided to Affymetrix as 71-mer nucleotide sequences from the forward strand, with the alleles at the target SNP highlighted at position 36. A “p-convert” value (representing the probability of a given SNP converting to a reliable SNP assay on the Axiom array system) was computed by Affymetrix for each submitted SNP sequence. Probes are assessed for each SNP in both the forward and reverse direction, in return each strand is designated as “recommended,” “neutral,” or “not recommended” based on p-convert values.</p><p id="p-15">The list of recommended markers (1,316,870 SNPs for <em>C. gigas</em> and <em>O. edulis</em> combined) was much greater than the total capacity of the Axiom MyDesign custom array. Therefore, additional filtering steps were carried out. For <em>C. gigas</em>, starting from the 1,216,467 Affymetrix-recommended SNPs, those with evidence for a 20 bp flanking monomorphic region covered by at least 36 reads from each pooled sample were retained (<em>n</em> = 186,948). For <em>O. edulis</em>, the Affymetrix-recommended SNPs (<em>n</em> = 100,403) were filtered so that each RAD locus contained a maximum of one SNP. When a RAD locus had multiple recommended SNPs, only the best SNP (based on the p-convert scores) was included (resulting in 59,976 candidate SNPs). Subsequently, to filter the SNPs to the required number for the array, SNPs for both species were selected according to the following additional filtering criteria: (i) highest p-convert values, (ii) even distribution across the reference genome (with at least 1000 bp distance between pairs of SNPs for <em>C. gigas</em>), and (iii) preference for those with a positive hit (minimum <em>e</em>-value 10E<sup>−4</sup>) against the BLASTx NCBI NR database or against the <em>C. gigas</em> genome (for <em>O. edulis)</em>. In addition, most A/T and C/G SNP transversions were discarded since these require double the space on the Affymetrix Axiom array platform. Additionally, 463 SNPs identified and validated by <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hedgecock <em>et al.</em> (2015)</a> passed the SNP filtering and scoring process and were included in the final array design.</p></div><div id="sec-5" class="subsection"><h3>SNP array validation</h3><p id="p-16">A plate of 384 individual genomic DNA samples (274 <em>C. gigas</em> and 110 <em>O. edulis</em>) was sent to Edinburgh Genomics (Edinburgh, UK) for genotyping using the array. Of these 384 samples, 219 were used for testing and validating the array’s performance and quantifying the number of segregating SNPs in the various sampled populations. These included 109 <em>C. gigas</em> samples of individuals of unknown relatedness from eight populations (the same eight populations used for SNP discovery, plus an additional set of 28 broodstock oysters from Guernsey Sea Farms (Guernsey, UK). The validation samples also included 110 <em>O. edulis</em> samples corresponding to the 11 population samples used for SNP discovery (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), with <em>n</em> = 10 from each population. The remaining 165 samples were offspring of three nuclear families derived from parents from Guernsey Sea Farms, reared at the Centre for Environment, Fisheries and Aquaculture Science (Cefas, UK). These were analyzed separately to test parentage assignment, genetic structure, and within-family linkage LD levels (see below).</p><p id="p-17">Raw data containing the results of the intensity calculations (CEL files) was imported into the Axiom Analysis Suite (v2.0.035, Affymetrix) for quality control analysis and genotype calling. Samples with a dish quality control (DQC) value &gt; 0.82 and QC call rate &gt; 0.97 threshold (following the “Best Practices Workflow” recommended by Affymetrix), were considered to have passed the quality control assessment. The quality control analysis classifies the SNPs into categories according to their clustering performance with respect to various Axiom-generated quality control criteria: (i) “polymorphic high resolution” where the SNP passes all QC, (ii) “monomorphic high resolution” where the SNP passes all QC except the presence of a minor allele in two or more samples, (iii) “call rate below threshold” where genotype call rate is &lt; 97%, (iv) “no minor homozygote” where the SNP passes all QC but only two clusters are observed, (v) “off-target variant” where atypical cluster properties arise from variants in the SNP flanking region, and (vi) “other” where the SNP does not fall into any of the previous categories. For further analyses, only SNPs from categories (i) and (iv) were included and classified as “good quality,” as they are most likely to be reliable and informative SNPs.</p></div><div id="sec-6" class="subsection"><h3>Descriptive statistics and family assignment</h3><p id="p-18">Calculations of minor allele frequencies (MAF), levels of heterozygosity, discriminant analysis of principal components (DAPC), LD, and IBS followed by multi-dimensional scaling (MDS) were carried out using Plink (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Purcell <em>et al.</em> 2007</a>), adegenet 1.3-1 package in R (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jombart and Ahmed 2011</a>), and Genepop (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Rousset 2008</a>). Family assignment for the <em>C. gigas</em> families was performed using Cervus 3.07 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kalinowski <em>et al.</em> 2007</a>). Cervus assigns offspring to their parent pairs based on the pair-wise likelihood comparison approach generating locus-by-locus likelihood scores for each candidate parent for each offspring, and assigns parentage to a candidate parent with the highest LOD score.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-19">The Illumina sequencing data for the pooled <em>C. gigas</em> and <em>O. edulis</em> samples have been deposited into the European nucleotide archive under accession number PRJEB20253 (<a href="http://www.ebi.ac.uk/ena/data/view/PRJEB20253">http://www.ebi.ac.uk/ena/data/view/PRJEB20253</a>). The details of the SNP markers on the array are given in (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FileS1.txt">File S1</a>). <em>O. edulis</em> markers with significant alignment to the <em>C. gigas</em> genome (<em>e</em>-value 1E<sup>−4</sup>) are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FileS2.txt">File S2</a>.</p></div></div><div class="section results-discussion" id="sec-8"><h2 class="">Results and Discussion</h2><div id="sec-9" class="subsection"><h3>Sequencing and SNP selection</h3><p id="p-20">To discover and prioritize SNPs for inclusion on the combined-species oyster SNP array, species-specific DNA sequencing, SNP discovery, and filtering strategies were followed. For <em>C. gigas</em>, WGS data aligned to the oyster genome identified 12.4 million putative SNPs across all populations. The 1,216,467 putative SNPs that passed the Affymetrix evaluation were subsequently filtered using the criteria described above to 40,625 putative SNPs that were submitted for the final Axiom MyDesign array. For <em>O. edulis</em>, 588,266 putative SNPs were identified, of which 100,403 putative SNPs were recommended at least for one strand by Affymetrix. Further filtering based on the criteria described above reduced the set to 19,215 putative SNPs that were submitted for array design and production.</p><p id="p-21">The final array contained 40,625 putative SNPs from <em>C. gigas</em> and 14,950 putative SNPs from <em>O. edulis</em> to give a total of 55,575 putative SNPs assayed by a total of 111,360 probes. There were a greater number of <em>C. gigas</em> SNPs placed on the array than <em>O. edulis</em> due to the anticipated greater future use of the array for genome-wide association studies and genomic prediction for economically important traits in breeding programs in this species. This includes an ongoing project to study host resistance to Oyster Herpes Virus based on genotyping samples collected from a large challenge experiment on oysters derived from Guernsey Sea Farm stocks. Nonetheless, it is anticipated that the ∼15 K putative <em>O. edulis</em> SNPs will be widely applied for population and conservation genetics in future studies of this species.</p></div><div id="sec-10" class="subsection"><h3>Evaluation of the SNP array in C. gigas and O. edulis</h3><p id="p-22">The oyster array was evaluated in <em>C. gigas</em> by analyzing the “validation populations” of 109 samples corresponding to eight distinct populations from France and the UK (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). All but one sample passed the DQC and genotype call rate ≥97% threshold. The classification of SNPs according to their quality showed that 68.2% (<em>n</em> = 27,697) had probes classified as good quality (either “Poly High Resolution” or “No Minor Hom”), which is similar to the percentage of informative markers obtained by the recently published <em>C. gigas</em> 134 K array (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Qi <em>et al.</em> 2017</a>). The MAF of these good quality SNPs (MAF &gt; 0) in the combined 108 samples varied between 0.005 and 0.5 with a median of 0.18 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). From the 110 <em>O. edulis</em> samples genotyped (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), two samples failed the DQC and genotype call rate ≥97% threshold, resulting in genotypes for 108 samples. A total of 74.6% of SNPs (<em>n</em> = 11,151) were classified as good quality as described above. The MAF of these good quality SNPs (combining all the 108 samples and SNPs with a MAF &gt; 0) also varied between 0.005 and 0.5 with a median of 0.21 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485367/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485367/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Descriptive population genetic estimates for the sampled <em>C. gigas</em> populations included in the validation of the array</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485477/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485477/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Descriptive population genetic estimates for the sampled <em>O. edulis</em> populations included in the validation of the array</span><div class="sb-div caption-clear"></div></div></div><div id="sec-11" class="subsection"><h4>Within-population segregation of SNPs:</h4><p id="p-29">The segregation of the SNPs was evaluated within each of the eight genotyped <em>C. gigas</em> population samples. From the 27,697 high quality SNPs defined across all population samples, the majority of SNPs (MAF &gt; 0) were segregating within each of the populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS1.pdf">Figure S1</a>), with an average of 22,486 SNPs segregating within each population, ranging from 20,141 (Hatchery 2) to 26,549 (Guernsey) (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Among the UK populations (sampled from Guernsey, Maldon, and Sea Salter), 19,613 SNPs were shared, while Guernsey had the highest number of exclusive SNPs (<em>n</em> = 2373) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS2.pdf">Figure S2</a>). This is likely to be due to the fact that the Guernsey population was the most highly represented within the sequenced populations used for SNP discovery (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) and the validation samples (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>), giving a greater chance of detecting rare minor alleles. Among all the five French populations, 13,855 SNPs were shared, with few SNPs segregating exclusively in particular populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS3.pdf">Figure S3</a>). Finally, 11,997 common SNPs were segregating in all the eight populations from both France and the UK (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS4.pdf">Figure S4</a>). The average MAF (for markers showing a MAF &gt; 0) was 0.207 across all UK populations, while it was 0.214 across all French populations. Analysis of the distribution of MAF values for polymorphic SNPs (MAF &gt; 0) showed that the highest numbers of SNPs are located within a MAF value range between 0.01 and 0.2 in all populations, decreasing in frequency when the MAF approaches 0.5 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS5.pdf">Figure S5</a>). A similar situation was observed by <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lapègue <em>et al.</em> (2014)</a>, who found a high proportion of low MAF SNPs within <em>C. gigas</em> populations. Based on an additional test of the array on a small number of Australian <em>C. gigas</em> samples (data not shown), the number of segregating SNPs was similar, indicating that the array is likely to perform comparably for geographically diverse populations.</p><p id="p-30">From the 11,151 high quality SNPs segregating in the <em>O. edulis</em> populations, the average number of SNPs segregating (MAF &gt; 0) in each population was 9597. The samples from Croatia showed the lowest number of segregating SNPs (<em>n</em> = 8474), while those from Foyle (Ireland) showed the highest (<em>n</em> = 10,013) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS6.pdf">Figure S6</a> and <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). A total of 4912 SNPs were shared between all (11) populations, with no particular population showing a high number of unique segregating SNPs. The average MAF value across the populations was 0.225, with Croatia showing the highest value of 0.234. Analysis of the distribution of MAF values for polymorphic SNPs (MAF &gt; 0) showed that most populations have a large number of SNPs within a MAF value range between 0.05 and 0.2, with the exception of Croatia and Swansea, which show a greater number of SNPs with a MAF higher than 0.1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS7.pdf">Figure S7</a>).</p><p id="p-31">The levels of genetic variability in terms of observed (Ho) and expected (He) heterozygosity (according to HWE) showed that most populations (<em>C. gigas</em> and <em>O. edulis</em>) had higher observed levels of heterozygosity than expected. Overall, no strong evidence of heterozygous deficiency was detected, in contrast to some previous studies that have described heterozygous deficiency in oysters and bivalves in general, albeit typically using a much lower number of microsatellites, SNPs, and allozymes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Appleyard and Ward 2006</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">English <em>et al.</em> 2000</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Li <em>et al.</em> 2003</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Sekino <em>et al.</em> 2003</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Lapègue <em>et al.</em> 2014</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Yu and Li 2007</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Sobolewska and Beaumont 2005</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Vercaemer <em>et al.</em> 2006</a>). This discrepancy may be due to the fact that genome-wide SNP markers were used in the current study at a density not previously tested. In a larger-scale SNP assay-based evaluation of the bivalve mollusc <em>Chlamys farreri</em>, no evidence for heterozygote deficiency was detected (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jiao <em>et al.</em> 2014</a>). It is also possible that the strict filtering process led to SNPs on the array being enriched for stable genomic regions with lower levels of variation, while genomic regions with higher variability (and potentially more prone to null alleles) might have been discarded.</p></div><div id="sec-12" class="subsection"><h4>Assessing population structure using IBS:</h4><p id="p-32">The overall genetic similarity of any two samples can be evaluated by calculating average measures of IBS of the marker loci, which was then summarized using MDS to give indications of population (sub)structure (IBS clustering was also confirmed by DAPC analysis, data not shown). There was some evidence of <em>C. gigas</em> samples clustering according to their hatchery origin, and French hatchery populations tended to cluster separately to UK hatchery populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS8.pdf">Figure S8</a>). The <em>O. edulis</em> samples were typically from “wild” stocks from more diverse geographical locations than for the <em>C. gigas</em> samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS9.pdf">Figure S9</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS10.pdf">Figure S10</a>). Accordingly, certain populations did show evidence of genetic differentiation, notably Croatia, Larne (Northern Ireland), and Sveio (Norway), which are geographical outgroups (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS10.pdf">Figure S10</a>). Our results show evidence of a strong genetic similarity between Maine, Sveio (Norway), and Grevelingen (Netherlands) populations. Similarly, the origin of the Maine population has been linked to the Netherlands’ (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Loosanoff 1955</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Vercaemer <em>et al.</em> 2006</a>), Netherlands populations have been linked to Denmark’s (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Vera <em>et al.</em> 2016</a>), and the genetic similarity between the Maine, Norway, Denmark, and Netherlands samples has also been observed using microsatellite markers (M. McCullough, personal communication). A lack of population structure according to geographical original was observed in the other <em>O. edulis</em> population samples tested, for example the majority of samples from the coast of the UK and Ireland (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1/FigureS9.pdf">Figure S9</a>). This is consistent with existing evidence that suggests that marine organisms with larval stages (such as bivalves) often show low genetic differentiation (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2015</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Shabtay <em>et al.</em> 2014</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rohfritsch <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Giantsis <em>et al.</em> 2014</a>), with temporal factors rather than geographical factors often playing the major role in population structure. It is also possible that historical stock translocations might have also played an important role in the lack of genetic structure and admixture of the <em>O. edulis</em> populations (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bromley <em>et al.</em> 2016</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identity-by-state clustering of selected O. edulis populations. Neth, The Netherlands; N.Ire, Northern Ireland." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-383538402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identity-by-state clustering of selected &lt;em&gt;O. edulis&lt;/em&gt; populations. Neth, The Netherlands; N.Ire, Northern Ireland.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2209/F1.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/7/2209/F1.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485501" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-33" class="first-child">Identity-by-state clustering of selected <em>O. edulis</em> populations. Neth, The Netherlands; N.Ire, Northern Ireland.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h4>Evaluation of the SNP array in pair crosses of C. gigas:</h4><p id="p-34">Three pair crosses between Guernsey Sea Farm parents were created, reared separately, and genotyped using the SNP array. Two of these nuclear families were half-siblings sharing a dam (F29 and F30). A total of 165 samples (161 offspring and their five parents) were genotyped. These families were analyzed separately from the population samples used to validate the array described above. In part, this was due to the difficulty in obtaining high quality genomic DNA from the juvenile oysters. From the 165 samples, 139 passed the DQC and genotype call rate ≥97% threshold, resulting in a total of 25,629 SNPs that were classified as good quality in these families. The vast majority of SNPs showed stable Mendelian inheritance in all samples, although there was an average of 395 SNPs (∼2% of total informative SNPs) with evidence of a Mendelian error per individual.</p><p id="p-35">Since the offspring from each nuclear family were physically tracked throughout the experiment, such that their family structure was known <em>a priori</em>, the utility of the SNP array to differentiate between families was assessed using IBS clustering with MDS scaling. The MDS scaling plot based on IBS clustering clearly shows a clear separate cluster for each of the families, as shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Interestingly, the clustering and separation of the three nuclear families was more obvious than for the population samples, even for populations from very distant geographical locations. Four individuals were distant to any of the family clusters, which may suggest incorrect pedigree assignment according to the physical animal tracking. Family assignment successfully assigned all the individuals to their correspondent parents using 3000 randomly chosen SNPs, and confirmed that the four aforementioned individuals were not members of any of these three families. Microsatellites and SNP panels for parentage assignment have been described previously for oysters (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wang <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2010</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Lapègue <em>et al.</em> 2014</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jin <em>et al.</em> 2014</a>). However, the successful parentage assignment in these physically tracked nuclear families, and the clear IBS-based differentiation of these families, bodes well for the utility of this SNP array for high resolution genetic mapping studies and selective breeding programs for oysters.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identity-by-state-based clustering of the three nuclear C. gigas families. Samples in purple (wrong pedigree &#x201C;wp&#x201D;) were not assigned to any of the three families." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-383538402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identity-by-state-based clustering of the three nuclear &lt;em&gt;C. gigas&lt;/em&gt; families. Samples in purple (wrong pedigree &#x201C;wp&#x201D;) were not assigned to any of the three families.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2209/F2.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/7/2209/F2.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485344" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-36" class="first-child">Identity-by-state-based clustering of the three nuclear <em>C. gigas</em> families. Samples in purple (wrong pedigree “wp”) were not assigned to any of the three families.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h4>Distribution of SNPs in the pacific oyster genome:</h4><p id="p-37">To assess the distribution of SNPs in the <em>C. gigas</em> genome (<a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Zhang <em>et al.</em> 2012</a>), SNPs were annotated according to the publicly available Ensembl oyster genome assembly (NCBI accession number: GCA_000297895.1). The oyster genome contains 7658 scaffolds (N50 = 401,585), 30,459 contigs (N50 = 31,239), and a total of ∼558 Mb of assembled sequence. All 27,697 SNPs are mapped to the oyster genome according to BLAST alignment using their flanking region(s), with at least one SNP on 2007 of the scaffolds, which in total covered 501 Mb (89.6% of the total assembled genome sequence). The number of SNPs per scaffold was positively associated with scaffold length (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), with approximately one-fifth of the scaffolds containing only one SNP. Additionally, harnessing the publicly-available oyster genome annotation (GCA_000297895.133), the SNPs on the array were grouped into putative positional and functional categories using SNPeff (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cingolani <em>et al.</em> 2012</a>). A total of 14.6, 13.1, 18.7, 17.6, and 2.8% of the SNPs were located in intergenic, intron, downstream, upstream, and exon regions, respectively. The remaining SNPs (33%) were identified as transcript, splice site donor, splice site acceptor, and splice site region.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of SNPs on the C. gigas genome. Number of scaffolds containing SNPs (primary axis) and the average length of the scaffolds holding an increasing number of SNPs (secondary axis). SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-383538402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of SNPs on the &lt;em&gt;C. gigas&lt;/em&gt; genome. Number of scaffolds containing SNPs (primary axis) and the average length of the scaffolds holding an increasing number of SNPs (secondary axis). SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2209/F3.medium.gif" width="440" height="204"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/7/2209/F3.medium.gif" width="440" height="204"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485473" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Distribution of SNPs on the <em>C. gigas</em> genome. Number of scaffolds containing SNPs (primary axis) and the average length of the scaffolds holding an increasing number of SNPs (secondary axis). SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The extent of LD between SNP pairs was assessed relative to their physical distance for the <em>C. gigas</em> populations. Pairwise <em>r</em><sup>2</sup> was calculated using polymorphic SNPs with MAF ≥ 0.05 as shown in <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>. The mean <em>r</em><sup>2</sup> was calculated for every kilobase and covering up to 500 kb, according to the physical distance on the oyster genome assembly, as shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. In general, low levels of LD with slow decay with increasing physical distance were observed. The Guernsey and Ifremer populations had lower levels of LD than the other populations. Although these LD levels are low compared to other aquaculture species, such as carp or tilapia (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hong Xia <em>et al.</em> 2015</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Xu <em>et al.</em> 2014</a>), they are in accordance with recent reports describing low levels and short extents of LD in wild <em>C. gigas</em> populations (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Zhong <em>et al.</em> 2017</a>). Moreover, differences in LD levels between populations can be related to the divergence of these populations and the number of generations for which they have been bred in isolation, as observed in cattle (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de Roos <em>et al.</em> 2008</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Decay of linkage disequilibrium (LD) with physical distance between markers among all the sampled C. gigas populations. Fr, France." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-383538402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Decay of linkage disequilibrium (LD) with physical distance between markers among all the sampled &lt;em&gt;C. gigas&lt;/em&gt; populations. Fr, France.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2209/F4.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/7/2209/F4.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485345" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-40" class="first-child">Decay of linkage disequilibrium (LD) with physical distance between markers among all the sampled <em>C. gigas</em> populations. Fr, France.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">There was a higher extent and slower decay of LD in the three nuclear families, and LD levels were substantially higher than those observed in the (presumably unrelated) validation populations, as would be expected (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). A lower effective population size (Ne) brings higher levels of kinship between individuals and therefore a higher extent of LD (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sved 1971</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Falconer and Mackay 1996</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Decay of linkage disequilibrium (LD) among the three C. gigas families." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-383538402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Decay of linkage disequilibrium (LD) among the three &lt;em&gt;C. gigas&lt;/em&gt; families.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/7/2209/F5.medium.gif" width="440" height="374"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/7/2209/F5.medium.gif" width="440" height="374"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/7/2209/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485524" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">Decay of linkage disequilibrium (LD) among the three <em>C. gigas</em> families.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-15" class="subsection"><h3>Conclusions</h3><p id="p-43">This article describes the development and analysis of a high density SNP array for two oyster species. A very large database of SNP markers was developed for both <em>C. gigas</em> and <em>O. edulis</em>, using WGA and RAD-Seq, respectively. Following extensive filtering, SNP assays for these two oyster species were combined on the array with 40,625 high quality SNPs for <em>C. gigas</em> and 14,950 for <em>O. edulis</em>. Testing of the array on genomic DNA samples from diverse locations revealed that the array contains a high number of SNPs that are shared between populations, and that the array can be applied to detect population and family structure. This oyster SNP array will be publicly available and will facilitate the study of important economic and ecological traits for these two oyster species, with possible applications for genomic selection, QTL mapping, evolutionary genetics, and conservation programs.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">The authors acknowledge Pierrick Haffray, Anastasia Bestin, Florian Enez, and Anne-Sophie Tyran from the Syndicat des Selectionneurs Avicoles and Aquacoles, France for coordinating the provision of the French hatchery samples used for single nucleotide polymorphism (SNP) discovery and genotyping; Lucie Buockellyoen from Marinove, Emilie Vetois from Satmar, Adeline Lange from France Naissain, and Fiz Dacosta from Novostrea for agreeing to provide samples; Lionel Degremont from Ifremer, France for coordinating the provision of the Ifremer selection line samples; and Dennis Hedgecock from the University of Southern California for providing details of validated SNPs to include on the array. P.A.P. and M.M.C. were supported by funding from the European Union (EU) INTERREG IVA Program (project 2859 “IBIS”) managed by the Special EU Programs Body. The authors acknowledge funding for this work under the Biotechnology and Biological Sciences Research Council (BBSRC)-Natural Environment Research Council UK Aquaculture Initiative (grant numbers: BB/M026140/1 and NE/P010695/1), via BBSRC Institute Strategic Funding grants to The Roslin Institute (BB/J004235/1, BB/J004324/1, and BB/J004243/1), and via the Centre for Environment, Fisheries and Aquaculture Science (Cefas) Seedcorn fund.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></li><li class="fn" id="fn-9"><p id="p-45">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041780/-/DC1</a>.</p></li><li class="fn" id="fn-10"><p id="p-46"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-04-04"><span class="received-label">Received </span>April 4, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-05-06"><span class="accepted-label">Accepted </span>May 6, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Gutierrez <em>et al.</em></li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.1" data-doi="10.1016/j.aquaculture.2005.10.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Appleyard</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic diversity and effective population size in mass selection lines of Pacific oyster (<em>Crassostrea gigas</em>).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">254</span>(<span class="cit-issue">1–4</span>): <span class="cit-fpage">148</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D254%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2005.10.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2005.10.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000237838800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.2" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bromley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGonigle</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashton</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Bad moves: pros and cons of moving oysters – a case study of global translocations of <em>Ostrea edulis</em> Linnaeus, 1758 (Mollusca: Bivalvia).</span> <abbr class="cit-jnl-abbrev">Ocean Coast. Manage.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOcean%2BCoast.%2BManage.%26rft.volume%253D122%26rft.spage%253D103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.4" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.5" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.6" data-doi="10.1186/s12864-015-2038-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">López</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naswa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide association analysis reveals loci associated with resistance against <em>Piscirickettsia salmonis</em> in two Atlantic salmon (<em>Salmo salar</em> L.) chromosomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">854</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D854%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2038-7%26rft_id%253Dinfo%253Apmid%252F26499328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2038-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26499328&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.7" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.8" data-doi="10.1534/genetics.107.084301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spelman</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Linkage disequilibrium and persistence of phase in Holstein–Friesian, Jersey and Angus cattle.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084301%26rft_id%253Dinfo%253Apmid%252F18622038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE1MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzcvMjIwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">English</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetic variation of wild and hatchery populations of the Pacific oyster, <em>Crassostrea gigas</em> (Thunberg)</span>. <abbr class="cit-jnl-abbrev">Aust. Aquacult.</abbr> <span class="cit-vol">187</span>(<span class="cit-issue">3–4</span>): <span class="cit-fpage">283</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BAquacult.%26rft.volume%253D187%26rft.spage%253D283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-7.7.2209.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source"><em>Introduction to Quantitative Genetics</em></span>, <span class="cit-publ-name">Pearson</span>, <span class="cit-publ-loc">Harlow, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-other" id="cit-7.7.2209.11"><div class="cit-metadata unstructured">FAO, 2017 Global aquaculture production: 1950–2015. Available at: <a href="http://www.fao.org/fishery/statistics/global-aquaculture-production/query/en">http://www.fao.org/fishery/statistics/global-aquaculture-production/query/en</a>. Accessed date: May 10, 2017</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.12" data-doi="10.1186/1471-2164-10-341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fleury</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huvet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lelong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Lorgeril</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulo</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Generation and analysis of a 29,745 unique expressed sequence tags from the pacific oyster (<em>Crassostrea gigas</em>) assembled into a publicly accessible database: the GigasDatabase.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DFleury%26rft.auinit1%253DE.%26rft.volume%253D10%26rft.spage%253D341%26rft.epage%253D341%26rft.atitle%253DGeneration%2Band%2Banalysis%2Bof%2Ba%2B29%252C745%2Bunique%2BExpressed%2BSequence%2BTags%2Bfrom%2Bthe%2BPacific%2Boyster%2B%2528Crassostrea%2Bgigas%2529%2Bassembled%2Binto%2Ba%2Bpublicly%2Baccessible%2Bdatabase%253A%2Bthe%2BGigasDatabase.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-341%26rft_id%253Dinfo%253Apmid%252F19640306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19640306&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.13" data-doi="10.1186/s12864-015-1409-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sha</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A genome-wide association study in catfish reveals the presence of functional hubs of related genes within QTLs for columnaris disease resistance.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D196%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1409-4%26rft_id%253Dinfo%253Apmid%252F25888203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1409-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25888203&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giantsis</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mucci</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Randi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abatzopoulos</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apostolidis</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Microsatellite variation of mussels (<em>Mytilus galloprovincialis</em>) in central and eastern Mediterranean: genetic panmixia in the Aegean and the Ionian seas.</span> <abbr class="cit-jnl-abbrev">J. Mar. Biol. Assoc. U. K.</abbr> <span class="cit-vol">94</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">797</span>–<span class="cit-lpage">809</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMar.%2BBiol.%2BAssoc.%2BU.%2BK.%26rft.volume%253D94%26rft.spage%253D797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.15" data-doi="10.1007/s10126-011-9405-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic mapping and QTL analysis of growth-related traits in the Pacific oyster.</span> <abbr class="cit-jnl-abbrev">Mar. Biotechnol. (NY)</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">218</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2Bbiotechnology%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.stitle%253DMar%2BBiotechnol%2B%2528NY%2529%26rft.aulast%253DGuo%26rft.auinit1%253DX.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D218%26rft.epage%253D226%26rft.atitle%253DGenetic%2Bmapping%2Band%2BQTL%2Banalysis%2Bof%2Bgrowth-related%2Btraits%2Bin%2Bthe%2BPacific%2Boyster.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10126-011-9405-4%26rft_id%253Dinfo%253Apmid%252F21932055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10126-011-9405-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21932055&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gracey</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Den Berg</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samanta</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation linkage maps for the Pacific oyster <em>Crassostrea gigas</em> reveal errors in assembly of genome scaffolds.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">2007</span>–<span class="cit-lpage">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D5%26rft.spage%253D2007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hong Xia</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Signatures of selection in tilapia revealed by whole genome resequencing.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">14168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D14168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.18" data-doi="10.1186/1471-2164-15-90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cézard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and validation of a high density SNP genotyping array for Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-90%26rft_id%253Dinfo%253Apmid%252F24524230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24524230&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.19" data-doi="10.1534/genetics.104.027342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Linkage maps of microsatellite DNA markers for the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHubert%26rft.auinit1%253DS.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D351%26rft.epage%253D362%26rft.atitle%253DLinkage%2BMaps%2Bof%2BMicrosatellite%2BDNA%2BMarkers%2Bfor%2Bthe%2BPacific%2BOyster%2BCrassostrea%2Bgigas%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.027342%26rft_id%253Dinfo%253Apmid%252F15454548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvMzUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIyMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.20" data-doi="10.1093/dnares/dst048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Large-scale development of gene-associated single-nucleotide polymorphism markers for molluscan population genomic, comparative genomic, and genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">183</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdst048%26rft_id%253Dinfo%253Apmid%252F24277739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMTgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy83LzIyMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development, inheritance and evaluation of 55 novel single nucleotide polymorphism markers for parentage assignment in the Pacific oyster (<em>Crassostrea gigas</em>).</span> <abbr class="cit-jnl-abbrev">Genes Genomics</abbr> <span class="cit-vol">36</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">129</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomics%26rft.volume%253D36%26rft.spage%253D129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.22" data-doi="10.1093/bioinformatics/btr521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">adegenet 1.3–1: new tools for the analysis of genome-wide SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">3070</span>–<span class="cit-lpage">3071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr521%26rft_id%253Dinfo%253Apmid%252F21926124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjEvMzA3MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.23" data-doi="10.1111/j.1365-294X.2007.03089.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalinowski</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taper</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Revising how the computer program </span>CERVUS<span class="cit-article-title"> accommodates genotyping error increases success in paternity assignment.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1099</span>–<span class="cit-lpage">1106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DKalinowski%26rft.auinit1%253DS.%2BT.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D1099%26rft.epage%253D1106%26rft.atitle%253DRevising%2Bhow%2Bthe%2Bcomputer%2Bprogram%2BCERVUS%2Baccommodates%2Bgenotyping%2Berror%2Bincreases%2Bsuccess%2Bin%2Bpaternity%2Bassignment.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03089.x%26rft_id%253Dinfo%253Apmid%252F17305863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03089.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17305863&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244245300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.24" data-doi="10.1093/bioinformatics/btr589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PoPoolation2: identifying differentiation between populations using sequencing of pooled DNA samples (Pool-Seq).</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">24</span>): <span class="cit-fpage">3435</span>–<span class="cit-lpage">3436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr589%26rft_id%253Dinfo%253Apmid%252F22025480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjQvMzQzNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.25" data-doi="10.1186/1471-2164-14-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kranis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gheyas</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boschiero</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Development of a high density 600K SNP genotyping array for chicken.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-59%26rft_id%253Dinfo%253Apmid%252F23356797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23356797&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.26" data-doi="10.1111/j.1365-2052.2007.01647.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lallias</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heurtebise</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A first-generation genetic linkage map of the European flat oyster <em>Ostrea edulis</em> (L.) based on AFLP and microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">560</span>–<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DLallias%26rft.auinit1%253DD.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D560%26rft.epage%253D568%26rft.atitle%253DA%2Bfirst-generation%2Bgenetic%2Blinkage%2Bmap%2Bof%2Bthe%2BEuropean%2Bflat%2Boyster%2BOstrea%2Bedulis%2B%2528L.%2529%2Bbased%2Bon%2BAFLP%2Band%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2007.01647.x%26rft_id%253Dinfo%253Apmid%252F17931405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2007.01647.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17931405&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lallias</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stockdale</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapègue</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Characterization of 27 microsatellite loci in the European flat oyster <em>Ostrea edulis</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">960</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DLallias%26rft.auinit1%253DD.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D960%26rft.epage%253D963%26rft.atitle%253DCharacterization%2Bof%2B27%2Bmicrosatellite%2Bloci%2Bin%2Bthe%2BEuropean%2Bflat%2Boyster%2BOstrea%2Bedulis.%26rft_id%253Dinfo%253Apmid%252F21564805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21564805&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lallias</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapègue</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Strategies for the retention of high genetic variability in European flat oyster (<em>Ostrea edulis</em>) restoration programmes.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1899</span>–<span class="cit-lpage">1910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D11%26rft.spage%253D1899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lapègue</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrang</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heurtebise</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flahauw</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnadieu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development of SNP-genotyping arrays in two shellfish species.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">820</span>–<span class="cit-lpage">830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.30" data-doi="10.1046/j.1471-8286.2003.00406.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bucklin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Characterization of 79 microsatellite DNA markers in the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">228</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D3%26rft.spage%253D228%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-8286.2003.00406.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-8286.2003.00406.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000183450700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.31" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.32" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.33" data-doi="10.1016/j.aquaculture.2010.09.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornette</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dégremont</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapègue</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Development of four EST-SSR multiplex PCRs in the Pacific oyster (<em>Crassostrea gigas</em>) and their validation in parentage assignment.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">310</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">234</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D310%26rft.spage%253D234%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2010.09.037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2010.09.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000286161200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic variation and population structure of the Pacific oyster <em>Crassostrea gigas</em> in the northwestern Pacific inferred from mitochondrial COI sequences.</span> <abbr class="cit-jnl-abbrev">Fish. Sci.</abbr> <span class="cit-vol">81</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1071</span>–<span class="cit-lpage">1082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFish.%2BSci.%26rft.volume%253D81%26rft.spage%253D1071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.35" data-doi="10.1186/1756-0500-7-135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development of the catfish 250K SNP array for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BRes.%2BNotes%26rft.volume%253D7%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-7-135%26rft_id%253Dinfo%253Apmid%252F24618043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-7-135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24618043&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.36" data-doi="10.1126/science.121.3135.119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loosanoff</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1955</span> <span class="cit-article-title">The European oyster in American waters.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">121</span>(<span class="cit-issue">3135</span>): <span class="cit-fpage">119</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLoosanoff%26rft.auinit1%253DV.%2BL.%26rft.volume%253D121%26rft.issue%253D3135%26rft.spage%253D119%26rft.epage%253D121%26rft.atitle%253DThe%2BEuropean%2BOyster%2Bin%2BAmerican%2BWaters%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.121.3135.119%26rft_id%253Dinfo%253Apmid%252F13225752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjEyMS8zMTM1LzExOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.37" data-doi="10.1093/bioinformatics/btr507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magoč</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FLASH: fast length adjustment of short reads to improve genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">2957</span>–<span class="cit-lpage">2963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr507%26rft_id%253Dinfo%253Apmid%252F21903629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjEvMjk1NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads.</span> <abbr class="cit-jnl-abbrev">EMBnet.journal</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBnet.journal%26rft.volume%253D17%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.39" data-doi="10.1371/journal.pone.0005350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawley</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allan</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and characterization of a high density SNP genotyping assay for cattle.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e5350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMatukumalli%26rft.auinit1%253DL.%2BK.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De5350%26rft.epage%253De5350%26rft.atitle%253DDevelopment%2Band%2Bcharacterization%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bfor%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005350%26rft_id%253Dinfo%253Apmid%252F19390634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19390634&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.40" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ødegård</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identity-by-descent genomic selection using selective and sparse genotyping.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.42" data-doi="10.1111/1755-0998.12337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palti</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The development and characterization of a 57K single nucleotide polymorphism array for rainbow trout.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">662</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D662%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12337%26rft_id%253Dinfo%253Apmid%252F25294387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25294387&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pardo</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Álvarez-Dios</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramilo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gómez-Tato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Construction of an <em>Ostrea edulis</em> database from genomic and expressed sequence tags (ESTs) obtained from <em>Bonamia ostreae</em> infected haemocytes: development of an immune-enriched oligo-microarray.</span> <abbr class="cit-jnl-abbrev">Fish Shellfish Immunol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFish%2BShellfish%2BImmunol.%26rft.volume%253D59%26rft.spage%253D331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.44" data-doi="10.1093/bioinformatics/bts174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">H. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yiu</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">F. Y. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">IDBA-UD: a <em>de novo</em> assembler for single-cell and metagenomic sequencing data with highly uneven depth.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1420</span>–<span class="cit-lpage">1428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts174%26rft_id%253Dinfo%253Apmid%252F22495754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTEvMTQyMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNy8yMjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pernet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bacher</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittington</span>  <span class="cit-name-given-names">R.J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Infectious diseases in oyster aquaculture require a new integrated approach.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B. Biol. Sci.</abbr> <span class="cit-vol">371</span>: 20150213.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.46" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Construction and evaluation of a high-density SNP array for the Pacific oyster <em>(Crassostrea gigas</em>).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e0174007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0174007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.48" data-doi="10.1371/journal.pone.0006524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooijmans</span>  <span class="cit-name-given-names">R. P. M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affara</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amaral</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archibald</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e6524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRamos%26rft.auinit1%253DA.%2BM.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De6524%26rft.epage%253De6524%26rft.atitle%253DDesign%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bin%2Bthe%2Bpig%2Busing%2BSNPs%2Bidentified%2Band%2Bcharacterized%2Bby%2Bnext%2Bgeneration%2Bsequencing%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006524%26rft_id%253Dinfo%253Apmid%252F19654876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19654876&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.49" data-doi="10.1111/mec.12262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaxter</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RAD-Seq derived markers flank the shell colour and banding loci of the <em>Cepaea nemoralis</em> supergene.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">3077</span>–<span class="cit-lpage">3089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3077%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.50" data-doi="10.1111/raq.12193."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargelloni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Applications of genotyping by sequencing in aquaculture breeding and genetics.</span> <abbr class="cit-jnl-abbrev">Rev. Aquacult.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/raq.12193.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRev.%2BAquacult.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fraq.12193.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/raq.12193.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rohfritsch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bierne</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heurtebise</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornette</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomics shed light on the demographic and adaptive histories of European invasion in the Pacific oyster, <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1064</span>–<span class="cit-lpage">1078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D6%26rft.spage%253D1064%26rft_id%253Dinfo%253Apmid%252F24187588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24187588&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.52" data-doi="10.1111/j.1471-8286.2007.01931.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rousset</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> GENEPOP<span class="cit-article-title">’007: a complete re-implementation of the </span>GENEPOP<span class="cit-article-title"> software for Windows and Linux.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">103</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DRousset%26rft.auinit1%253DF.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D106%26rft.atitle%253Dgenepop%2527007%253A%2Ba%2Bcomplete%2Bre-implementation%2Bof%2Bthe%2Bgenepop%2Bsoftware%2Bfor%2BWindows%2Band%2BLinux.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2007.01931.x%26rft_id%253Dinfo%253Apmid%252F21585727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2007.01931.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21585727&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253827100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.53" data-doi="10.1016/j.gene.2007.05.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bierne</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapègue</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Single nucleotide polymorphisms and their relationship to codon usage bias in the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">406</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">13</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DSauvage%26rft.auinit1%253DC.%26rft.volume%253D406%26rft.issue%253D1-2%26rft.spage%253D13%26rft.epage%253D22%26rft.atitle%253DSingle%2BNucleotide%2Bpolymorphisms%2Band%2Btheir%2Brelationship%2Bto%2Bcodon%2Busage%2Bbias%2Bin%2Bthe%2BPacific%2Boyster%2BCrassostrea%2Bgigas.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2007.05.011%26rft_id%253Dinfo%253Apmid%252F17616269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2007.05.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17616269&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252500500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heurtebise</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">QTL for resistance to summer mortality and OsHV-1 load in the Pacific oyster (<em>Crassostrea gigas</em>).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">390</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DSauvage%26rft.auinit1%253DC.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D390%26rft.epage%253D399%26rft.atitle%253DQTL%2Bfor%2Bresistance%2Bto%2Bsummer%2Bmortality%2Band%2BOsHV-1%2Bload%2Bin%2Bthe%2BPacific%2Boyster%2B%2528Crassostrea%2Bgigas%2529.%26rft_id%253Dinfo%253Apmid%252F20096029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20096029&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.55" data-doi="10.1007/s10126-002-0104-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sekino</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamaguchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aranishi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okoshi</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Development of novel microsatellite DNA markers from the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Mar. Biotechnol. (NY)</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">227</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2Bbiotechnology%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.stitle%253DMar%2BBiotechnol%2B%2528NY%2529%26rft.aulast%253DSekino%26rft.auinit1%253DM.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D227%26rft.epage%253D233%26rft.atitle%253DDevelopment%2Bof%2Bnovel%2Bmicrosatellite%2BDNA%2Bmarkers%2Bfrom%2Bthe%2BPacific%2Boyster%2BCrassostrea%2Bgigas.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10126-002-0104-z%26rft_id%253Dinfo%253Apmid%252F14502394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10126-002-0104-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14502394&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shabtay</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tikochinski</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benayahu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rilov</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Preliminary data on the genetic structure of a highly successful invading population of oyster suggesting its establishment dynamics in the Levant.</span> <abbr class="cit-jnl-abbrev">Mar. Biol. Res.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">407</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BBiol.%2BRes.%26rft.volume%253D10%26rft.spage%253D407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sobolewska</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic variation at microsatellite loci in northern populations of the European flat oyster (<em>Ostrea edulis</em>).</span> <abbr class="cit-jnl-abbrev">J. Mar. Biol. Assoc. U. K.</abbr> <span class="cit-vol">85</span>(<span class="cit-issue">04</span>): <span class="cit-fpage">955</span>–<span class="cit-lpage">960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMar.%2BBiol.%2BAssoc.%2BU.%2BK.%26rft.volume%253D85%26rft.spage%253D955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.58" data-doi="10.1016/0040-5809(71)90011-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sved</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Linkage disequilibrium and homozygosity of chromosome segments in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">125</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DSved%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D125%26rft.epage%253D141%26rft.atitle%253DLinkage%2Bdisequilibrium%2Band%2Bhomozygosity%2Bof%2Bchromosome%2Bsegments%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252871%252990011-6%26rft_id%253Dinfo%253Apmid%252F5170716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(71)90011-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5170716&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.59" data-doi="10.1016/j.seares.2010.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Troost</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Causes and effects of a highly successful marine invasion: case-study of the introduced Pacific oyster <em>Crassostrea gigas</em> in continental NW European estuaries.</span> <abbr class="cit-jnl-abbrev">J. Sea Res.</abbr> <span class="cit-vol">64</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">145</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BSea%2BRes.%26rft.volume%253D64%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.seares.2010.02.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.seares.2010.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000280976500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.60" data-doi="10.1186/s12864-015-2117-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome wide association and genomic prediction for growth traits in juvenile farmed Atlantic salmon using a high density SNP array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D969%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2117-9%26rft_id%253Dinfo%253Apmid%252F26582102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2117-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26582102&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bron</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of host resistance to sea lice in farmed Atlantic salmon populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leeds</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragomeni</span>  <span class="cit-name-given-names">B. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluation of genome-enabled selection for bacterial cold water disease resistance using progeny performance data in rainbow trout: insights on genotyping methods and genomic prediction models.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D7%26rft.spage%253D96%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vera</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlsson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlsson</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Current genetic status, temporal stability and structure of the remnant wild European flat oyster populations: conservation and restoring implications.</span> <abbr class="cit-jnl-abbrev">Mar. Biol.</abbr> <span class="cit-vol">163</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BBiol.%26rft.volume%253D163%26rft.spage%253D239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vercaemer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spence</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herbinger</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapègue</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenchington</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic diversity of the European oyster (<em>Ostrea edulis</em> L.) in Nova Scotia: comparison with other parts of Canada, Maine and Europe and implications for broodstock management.</span> <abbr class="cit-jnl-abbrev">J. Shellfish Res.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">543</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BShellfish%2BRes.%26rft.volume%253D25%26rft.spage%253D543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.65" data-doi="10.1111/1755-0998.12278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Que</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Discovery and validation of genic single nucleotide polymorphisms in the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">123</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12278%26rft_id%253Dinfo%253Apmid%252F24823694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24823694&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A 16-microsatellite multiplex assay for parentage assignment in the eastern oyster (<em>Crassostrea virginica</em> Gmelin).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">308</span>(<span class="cit-supplement">Suppl. 1</span>): <span class="cit-fpage">S28</span>–<span class="cit-lpage">S33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D308%26rft.spage%253DS28%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and evaluation of the first high-throughput SNP array for common carp (<em>Cyprinus carpio</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naswa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">López</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomewide single nucleotide polymorphism discovery in Atlantic salmon (<em>Salmo salar</em>): validation in wild and farmed American and European populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1002</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D1002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic variation of wild and hatchery populations of the Pacific oyster <em>Crassostrea gigas</em> assessed by microsatellite markers.</span> <abbr class="cit-jnl-abbrev">J. Genet. Genomics</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1114</span>–<span class="cit-lpage">1122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BGenet%2BGenomics%26rft.aulast%253DYu%26rft.auinit1%253DH.%26rft.volume%253D34%26rft.issue%253D12%26rft.spage%253D1114%26rft.epage%253D1122%26rft.atitle%253DGenetic%2Bvariation%2Bof%2Bwild%2Band%2Bhatchery%2Bpopulations%2Bof%2Bthe%2BPacific%2Boyster%2BCrassostrea%2Bgigas%2Bassessed%2Bby%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F18155624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18155624&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.70" data-doi="10.1038/nature11413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The oyster genome reveals stress adaptation and complexity of shell formation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">490</span>(<span class="cit-issue">7418</span>): <span class="cit-fpage">49</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZhang%26rft.auinit1%253DG.%26rft.volume%253D490%26rft.issue%253D7418%26rft.spage%253D49%26rft.epage%253D54%26rft.atitle%253DThe%2Boyster%2Bgenome%2Breveals%2Bstress%2Badaptation%2Band%2Bcomplexity%2Bof%2Bshell%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11413%26rft_id%253Dinfo%253Apmid%252F22992520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22992520&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F7%2F2209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309446800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.71" data-doi="10.1007/s10499-014-9789-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL mapping for glycogen content and shell pigmentation in the Pacific oyster <em>Crassostrea gigas</em> using microsatellites and SNPs.</span> <abbr class="cit-jnl-abbrev">Aquacult. Int.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1877</span>–<span class="cit-lpage">1889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquacult.%2BInt.%26rft.volume%253D22%26rft.spage%253D1877%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10499-014-9789-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10499-014-9789-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.7.2209.72" data-doi="10.1111/jwas.12393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Estimates of linkage disequilibrium and effective population size in wild and selected populations of the Pacific oyster using single-nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">J. World. Aquac. Soc.</abbr> DOI: .<span class="cit-pub-id cit-pub-id-doi">10.1111/jwas.12393</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BWorld.%2BAquac.%2BSoc.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjwas.12393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jwas.12393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/7/2209.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/7/2195" title="Cold Fusion: Massive Karyotype Evolution in the Antarctic Bullhead Notothen Notothenia coriiceps" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2219" title="Whole-Genome Sequence and Variant Analysis of W303, a Widely-Used Strain of Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/7"><span>Volume 7 Issue 7, July 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/7.cover-source.jpg?itok=eAMfPilr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/7/2209" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KL8VHQChpleSIFOkR0mObk0uXKZrldFHn3dRQ37Urm0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/7/2209" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/7/2209" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/7/2209" class="active" data-icon-position="" data-hide-link-title="0">Development of a Medium Density Combined-Species SNP Array for Pacific and European Oysters (Crassostrea gigas and Ostrea edulis)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485318" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-A6JWBpvvUAlrJU-mGhxyga7z4P8JaIs0zYzkBpQ9qzA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2209.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/7/2209" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-64STJH5SZ3Qh7y7s9_r7ludWzHQCVH4Nmk_aMhtHrTQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/7/2209" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/7/2209.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485318" id="node485318--2" data-pisa="ggg;7/7/2209" data-pisa-master="ggg;g3.117.041780" data-apath="/ggg/7/7/2209.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Development of a Medium Density Combined-Species SNP Array for Pacific and European Oysters (<em>Crassostrea gigas</em> and <em>Ostrea edulis</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alejandro P. Gutierrez</span>, <span class="highwire-citation-author" data-delta="1">Frances Turner</span>, <span class="highwire-citation-author" data-delta="2">Karim Gharbi</span>, <span class="highwire-citation-author" data-delta="3">Richard Talbot</span>, <span class="highwire-citation-author" data-delta="4">Natalie R. Lowe</span>, <span class="highwire-citation-author" data-delta="5">Carolina Peñaloza</span>, <span class="highwire-citation-author" data-delta="6">Mark McCullough</span>, <span class="highwire-citation-author" data-delta="7">Paulo A. Prodöhl</span>, <span class="highwire-citation-author" data-delta="8">Tim P. Bean</span> and <span class="highwire-citation-author" data-delta="9">Ross D. Houston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2209-2218;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041780 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.041780" data-atom-uri="/ggg/7/7/2209.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485318--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alejandro P. Gutierrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alejandro%2BP.%2BGutierrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gutierrez%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlejandro%2BP.%2BGutierrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Frances Turner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frances%2BTurner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Turner%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrances%2BTurner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karim Gharbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karim%2BGharbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gharbi%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarim%2BGharbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Richard Talbot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BTalbot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Talbot%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BTalbot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Natalie R. Lowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalie%2BR.%2BLowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lowe%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalie%2BR.%2BLowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carolina Peñaloza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BPe%C3%B1aloza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pe%C3%B1aloza%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BPe%25C3%25B1aloza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mark McCullough </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Global Food Security, School of Biological Sciences, Queen’s University Belfast, BT7 1NN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BMcCullough%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCullough%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BMcCullough%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paulo A. Prodöhl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Global Food Security, School of Biological Sciences, Queen’s University Belfast, BT7 1NN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulo%2BA.%2BProd%C3%B6hl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prod%C3%B6hl%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulo%2BA.%2BProd%25C3%25B6hl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Tim P. Bean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Environment Fisheries and Aquaculture Science, Cefas Weymouth Laboratory, Dorset DT4 8UB, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tim%2BP.%2BBean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bean%20TP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATim%2BP.%2BBean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ross.houston@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ross.houston@roslin.ed.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/7/2209" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/7/2209&amp;title=Development%20of%20a%20Medium%20Density%20Combined-Species%20SNP%20Array%20for%20Pacific%20and%20European%20Oysters%20%28Crassostrea%20gigas%20and%20Ostrea%20edulis%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/7/2209&amp;title=Development%20of%20a%20Medium%20Density%20Combined-Species%20SNP%20Array%20for%20Pacific%20and%20European%20Oysters%20%28Crassostrea%20gigas%20and%20Ostrea%20edulis%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/7/2209&amp;title=Development%20of%20a%20Medium%20Density%20Combined-Species%20SNP%20Array%20for%20Pacific%20and%20European%20Oysters%20%28Crassostrea%20gigas%20and%20Ostrea%20edulis%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/7/2209&amp;text=Development%20of%20a%20Medium%20Density%20Combined-Species%20SNP%20Array%20for%20Pacific%20and%20European%20Oysters%20%28Crassostrea%20gigas%20and%20Ostrea%20edulis%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/7/2209&amp;title=Development%20of%20a%20Medium%20Density%20Combined-Species%20SNP%20Array%20for%20Pacific%20and%20European%20Oysters%20%28Crassostrea%20gigas%20and%20Ostrea%20edulis%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/7/2209&amp;t=Development%20of%20a%20Medium%20Density%20Combined-Species%20SNP%20Array%20for%20Pacific%20and%20European%20Oysters%20%28Crassostrea%20gigas%20and%20Ostrea%20edulis%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/7/2209&amp;title=Development%20of%20a%20Medium%20Density%20Combined-Species%20SNP%20Array%20for%20Pacific%20and%20European%20Oysters%20%28Crassostrea%20gigas%20and%20Ostrea%20edulis%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/7/2209&amp;title=Development%20of%20a%20Medium%20Density%20Combined-Species%20SNP%20Array%20for%20Pacific%20and%20European%20Oysters%20%28Crassostrea%20gigas%20and%20Ostrea%20edulis%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485318/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485318" id="node485318--3" data-pisa="ggg;7/7/2209" data-pisa-master="ggg;g3.117.041780" data-apath="/ggg/7/7/2209.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Development of a Medium Density Combined-Species SNP Array for Pacific and European Oysters (<em>Crassostrea gigas</em> and <em>Ostrea edulis</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alejandro P. Gutierrez</span>, <span class="highwire-citation-author" data-delta="1">Frances Turner</span>, <span class="highwire-citation-author" data-delta="2">Karim Gharbi</span>, <span class="highwire-citation-author" data-delta="3">Richard Talbot</span>, <span class="highwire-citation-author" data-delta="4">Natalie R. Lowe</span>, <span class="highwire-citation-author" data-delta="5">Carolina Peñaloza</span>, <span class="highwire-citation-author" data-delta="6">Mark McCullough</span>, <span class="highwire-citation-author" data-delta="7">Paulo A. Prodöhl</span>, <span class="highwire-citation-author" data-delta="8">Tim P. Bean</span> and <span class="highwire-citation-author" data-delta="9">Ross D. Houston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2209-2218;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041780 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.041780" data-atom-uri="/ggg/7/7/2209.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485318--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alejandro P. Gutierrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alejandro%2BP.%2BGutierrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gutierrez%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlejandro%2BP.%2BGutierrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Frances Turner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frances%2BTurner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Turner%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrances%2BTurner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karim Gharbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karim%2BGharbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gharbi%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarim%2BGharbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Richard Talbot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BTalbot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Talbot%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BTalbot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Natalie R. Lowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalie%2BR.%2BLowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lowe%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalie%2BR.%2BLowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carolina Peñaloza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BPe%C3%B1aloza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pe%C3%B1aloza%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BPe%25C3%25B1aloza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mark McCullough </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Global Food Security, School of Biological Sciences, Queen’s University Belfast, BT7 1NN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BMcCullough%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCullough%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BMcCullough%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paulo A. Prodöhl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Global Food Security, School of Biological Sciences, Queen’s University Belfast, BT7 1NN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulo%2BA.%2BProd%C3%B6hl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prod%C3%B6hl%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulo%2BA.%2BProd%25C3%25B6hl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Tim P. Bean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Environment Fisheries and Aquaculture Science, Cefas Weymouth Laboratory, Dorset DT4 8UB, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tim%2BP.%2BBean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bean%20TP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATim%2BP.%2BBean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ross.houston@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ross.houston@roslin.ed.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2209&amp;rft.epage=2218&amp;rft.atitle=Development%20of%20a%20Medium%20Density%20Combined-Species%20SNP%20Array%20for%20Pacific%20and%20European%20Oysters%20%28Crassostrea%20gigas%20and%20Ostrea%20edulis%29&amp;rft.volume=7&amp;rft.issue=7&amp;rft.date=2017-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Gutierrez%2C+Alejandro+P.&amp;rft.au=Turner%2C+Frances&amp;rft.au=Gharbi%2C+Karim&amp;rft.au=Talbot%2C+Richard&amp;rft.au=Lowe%2C+Natalie+R.&amp;rft.au=Pe%C3%B1aloza%2C+Carolina&amp;rft.au=McCullough%2C+Mark&amp;rft.au=Prod%C3%B6hl%2C+Paulo+A.&amp;rft.au=Bean%2C+Tim+P.&amp;rft.au=Houston%2C+Ross+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485318/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485318/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485318/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485318/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485318/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485318/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485318/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485318/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485318/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485318/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485318/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485318/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/7/2209" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/7/2209.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/7/2209.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/7/2209.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/7/2209.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
