<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/9/2717.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/9/2717.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/9/2717.atom" />
<meta name="HW.pisa" content="ggg;6/9/2717" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton" />
<meta name="DC.Identifier" content="10.1534/g3.116.031302" />
<meta name="DC.Date" content="2016-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Shang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Cotton fiber, a raw natural fiber material, is widely used in the textile industry. Understanding the genetic mechanism of fiber traits is helpful for fiber quality improvement. In the present study, the genetic basis of fiber quality traits was explored using two recombinant inbred lines (RILs) and corresponding backcross (BC) populations under multiple environments in Upland cotton based on marker analysis. In backcross populations, no significant correlation was observed between marker heterozygosity and fiber quality performance and it suggested that heterozygosity was not always necessarily advantageous for the high fiber quality. In two hybrids, 111 quantitative trait loci (QTL) for fiber quality were detected using composite interval mapping, in which 62 new stable QTL were simultaneously identified in more than one environment or population. QTL detected at the single-locus level mainly showed additive effect. In addition, a total of 286 digenic interactions (E-QTL) and their environmental interactions [QTL × environment interactions (QEs)] were detected for fiber quality traits by inclusive composite interval mapping. QE effects should be considered in molecular marker-assisted selection breeding. On average, the E-QTL explained a larger proportion of the phenotypic variation than the main-effect QTL did. It is concluded that the additive effect of single-locus and epistasis with few detectable main effects play an important role in controlling fiber quality traits in Upland cotton." />
<meta name="DC.Contributor" content="Lianguang Shang" />
<meta name="DC.Contributor" content="Yumei Wang" />
<meta name="DC.Contributor" content="Xiaocui Wang" />
<meta name="DC.Contributor" content="Fang Liu" />
<meta name="DC.Contributor" content="Abdugheni Abduweli" />
<meta name="DC.Contributor" content="Shihu Cai" />
<meta name="DC.Contributor" content="Yuhua Li" />
<meta name="DC.Contributor" content="Lingling Ma" />
<meta name="DC.Contributor" content="Kunbo Wang" />
<meta name="DC.Contributor" content="Jinping Hua" />
<meta name="article:published_time" content="2016-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Cotton fiber, a raw natural fiber material, is widely used in the textile industry. Understanding the genetic mechanism of fiber traits is helpful for fiber quality improvement. In the present study, the genetic basis of fiber quality traits was explored using two recombinant inbred lines (RILs) and corresponding backcross (BC) populations under multiple environments in Upland cotton based on marker analysis. In backcross populations, no significant correlation was observed between marker heterozygosity and fiber quality performance and it suggested that heterozygosity was not always necessarily advantageous for the high fiber quality. In two hybrids, 111 quantitative trait loci (QTL) for fiber quality were detected using composite interval mapping, in which 62 new stable QTL were simultaneously identified in more than one environment or population. QTL detected at the single-locus level mainly showed additive effect. In addition, a total of 286 digenic interactions (E-QTL) and their environmental interactions [QTL × environment interactions (QEs)] were detected for fiber quality traits by inclusive composite interval mapping. QE effects should be considered in molecular marker-assisted selection breeding. On average, the E-QTL explained a larger proportion of the phenotypic variation than the main-effect QTL did. It is concluded that the additive effect of single-locus and epistasis with few detectable main effects play an important role in controlling fiber quality traits in Upland cotton.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/09/01" />
<meta name="citation_mjid" content="ggg;6/9/2717" />
<meta name="citation_id" content="6/9/2717" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/9/2717" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/9/2717.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/9/2717.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/9/2717.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.031302" />
<meta name="citation_pmid" content="27342735" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2717" />
<meta name="citation_lastpage" content="2724" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Lianguang Shang" />
<meta name="citation_author_institution" content="Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Yumei Wang" />
<meta name="citation_author_institution" content="Institute of Cash Crops, Hubei Academy of Agricultural Sciences, Wuhan 430064, China" />
<meta name="citation_author" content="Xiaocui Wang" />
<meta name="citation_author_institution" content="Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Fang Liu" />
<meta name="citation_author_institution" content="Institute of Cotton Research, Chinese Academy of Agricultural Sciences/State Key Laboratory of Cotton Biology, Anyang 455000, Henan, China" />
<meta name="citation_author" content="Abdugheni Abduweli" />
<meta name="citation_author_institution" content="Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Shihu Cai" />
<meta name="citation_author_institution" content="Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Yuhua Li" />
<meta name="citation_author_institution" content="Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Lingling Ma" />
<meta name="citation_author_institution" content="Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Kunbo Wang" />
<meta name="citation_author_institution" content="Institute of Cotton Research, Chinese Academy of Agricultural Sciences/State Key Laboratory of Cotton Biology, Anyang 455000, Henan, China" />
<meta name="citation_author" content="Jinping Hua" />
<meta name="citation_author_institution" content="Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author_email" content="jinping_hua@cau.edu.cn" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. Blanc;citation_author=A. Charcosset;citation_author=B. Mangin;citation_author=L. Moreau;citation_title=Connected populations for detecting quantitative trait loci and testing for epistasis: an application in maize.;citation_pages=206-224;citation_volume=113;citation_year=2006;citation_issue=2;citation_pmid=16791688;citation_doi=10.1007/s00122-006-0287-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=Z. Chen;citation_author=B. Scheffler;citation_author=E. Dennis;citation_author=B. Triplett;citation_author=T. Zhang;citation_title=Toward Sequencing Cotton (Gossypium) Genomes;citation_pages=1303-1310;citation_volume=145;citation_year=2007;citation_issue=4;citation_pmid=18056866;citation_doi=10.1104/pp.107.107672" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Islam;citation_author=L. Zeng;citation_author=G. Thyssen;citation_author=C. Delhom;citation_author=H. Kim;citation_title=Mapping by sequencing in cotton (Gossypium hirsutum) line MD52ne identified candidate genes for fiber strength and its related quality attributes.;citation_pages=1071-1086;citation_volume=129;citation_year=2016;citation_pmid=26883043;citation_doi=10.1007/s00122-016-2684-4" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=G. Jiang;citation_author=J. Zeng;citation_author=Y. He;citation_title=Analysis of quantitative trait loci affecting chlorophyll content of rice leaves in a double haploid population and two backcross populations.;citation_pages=287-295;citation_volume=536;citation_year=2014;citation_pmid=24361205;citation_doi=10.1016/j.gene.2013.12.010" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=R. Kohel;citation_author=J. Yu;citation_author=Y. Park;citation_author=G. Lazo;citation_title=Molecular mapping and characterization of traits controlling fiber quality in cotton.;citation_pages=163-172;citation_volume=121;citation_year=2001;citation_doi=10.1023/A:1012263413418" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=E. S. Lander;citation_author=P. Green;citation_author=J. Abrahamson;citation_author=A. Barlow;citation_author=M. J. Daly;citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.;citation_pages=174-181;citation_volume=1;citation_year=1987;citation_issue=2;citation_pmid=3692487;citation_doi=10.1016/0888-7543(87)90010-3" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=F. Li;citation_author=G. Fan;citation_author=K. Wang;citation_author=F. Sun;citation_author=Y. Yuan;citation_title=Genome sequence of the cultivated cotton Gossypium arboreum.;citation_pages=567-572;citation_volume=46;citation_year=2014;citation_pmid=24836287;citation_doi=10.1038/ng.2987" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=F. Li;citation_author=G. Fan;citation_author=C. Lu;citation_author=G. Xiao;citation_author=C. Zou;citation_title=Genome sequence of cultivated Upland cotton (Gossypium hirsutum TM-1) provides insights into genome evolution.;citation_pages=524-530;citation_volume=33;citation_year=2015;citation_pmid=25893780;citation_doi=10.1038/nbt.3208" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Li;citation_author=J. Wang;citation_author=L. Zhang;citation_title=Inclusive composite interval mapping of QTL by environment interactions in biparental populations.;citation_pages=e0132414;citation_volume=10;citation_year=2015;citation_pmid=26161656;citation_doi=10.1371/journal.pone.0132414" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. K. Li;citation_author=L. J. Luo;citation_author=H. W. Mei;citation_author=D. L. Wang;citation_author=Q. Y. Shu;citation_title=Overdominant Epistatic Loci Are the Primary Genetic Basis of Inbreeding Depression and Heterosis in Rice. I. Biomass and Grain Yield;citation_pages=1737-1753;citation_volume=158;citation_year=2001;citation_issue=4;citation_pmid=11514459" />
<meta name="citation_reference" content="citation_journal_title=Chin. Sci. Bull.;citation_author=Q. Liang;citation_author=C. Hu;citation_author=H. Hua;citation_author=Z. Li;citation_author=J. Hua;citation_title=Construction of a linkage map and QTL mapping for fiber quality traits in upland cotton (Gossypium hirsutum L.).;citation_pages=3233-3243;citation_volume=58;citation_year=2013;citation_doi=10.1007/s11434-013-5807-1" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Q. Liang;citation_author=L. Shang;citation_author=Y. Wang;citation_author=J. Hua;citation_title=Partial dominance, overdominance and epistasis as the genetic basis of heterosis in Upland Cotton.;citation_pages=e0143548;citation_volume=10;citation_year=2015;citation_pmid=26618635;citation_doi=10.1371/journal.pone.0143548" />
<meta name="citation_reference" content="citation_journal_title=Rice Genet. Newsl.;citation_author=S. R. McCouch;citation_author=Y. G. Cho;citation_author=M. Yano;citation_author=E. Paul;citation_author=M. Blinstrub;citation_title=Report on QTL nomenclature.;citation_pages=11-13;citation_volume=14;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Mei;citation_author=N. Syed;citation_author=W. Gao;citation_author=P. M. Thaxton;citation_author=C. W. Smith;citation_title=Genetic mapping and QTL analysis of fiber-related traits in cotton ( Gossypium).;citation_pages=280-291;citation_volume=108;citation_year=2004;citation_issue=2;citation_pmid=14513220;citation_doi=10.1007/s00122-003-1433-7" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. W. Mei;citation_author=Z. K. Li;citation_author=Q. Y. Shu;citation_author=L. B. Guo;citation_author=Y. P. Wang;citation_title=Gene actions of QTLs affecting several agronomic traits resolved in a recombinant inbred rice population and two backcross populations.;citation_pages=649-659;citation_volume=110;citation_year=2005;citation_issue=4;citation_pmid=15647921;citation_doi=10.1007/s00122-004-1890-7" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Paterson;citation_author=E. Lander;citation_author=J. Hewitt;citation_author=S. Peterson;citation_author=S. Lincoln;citation_title=Resolution of quantitative traits into Mendelian factors by using a complete linkage map of restriction fragment length polymorphisms.;citation_pages=721-726;citation_volume=335;citation_year=1988;citation_issue=6192;citation_pmid=2902517;citation_doi=10.1038/335721a0" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Paterson;citation_author=Y. Saranga;citation_author=M. Menz;citation_author=C. Jiang;citation_author=R. Wright;citation_title=QTL analysis of genotype x environment interactions affecting cotton fiber quality.;citation_pages=384-396;citation_volume=106;citation_year=2003;citation_issue=3;citation_pmid=12589538" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_author=H. Gundlach;citation_author=H. Guo;citation_author=J. Jenkins;citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres.;citation_pages=423-427;citation_volume=492;citation_year=2012;citation_pmid=23257886;citation_doi=10.1038/nature11798" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Radoev;citation_author=H. C. Becker;citation_author=W. Ecke;citation_title=Genetic analysis of heterosis for yield and yield components in rapeseed (Brassica napus L.) by quantitative trait locus mapping.;citation_volume=179;citation_year=2008;citation_pmid=18562665;citation_doi=10.1534/genetics.108.089680" />
<meta name="citation_reference" content="citation_journal_title=Mol. Genet. Genomics;citation_author=J. I. Said;citation_author=J. A. Knapka;citation_author=M. Song;citation_author=J. F. Zhang;citation_title=Cotton QTLdb: a cotton QTL database for QTL analysis, visualization, and comparison between Gossypium hirsutum and G. hirsutum × G. barbadense populations.;citation_pages=1615-1625;citation_volume=290;citation_year=2015;citation_pmid=25758743;citation_doi=10.1007/s00438-015-1021-y" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=L. G. Shang;citation_author=Q. Z. Liang;citation_author=Y. M. Wang;citation_author=X. C. Wang;citation_author=K. B. Wang;citation_title=Identification of stable QTLs controlling fiber traits properties in multi-environment using recombinant inbred lines in Upland cotton (Gossypium hirsutum L.).;citation_pages=877-888;citation_volume=205;citation_year=2015a;citation_doi=10.1007/s10681-015-1434-z" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=L. Shang;citation_author=F. Liu;citation_author=Y. Wang;citation_author=A. Abduweli;citation_author=S. Cai;citation_title=Dynamic QTL mapping for plant height in Upland cotton (Gossypium hirsutum).;citation_pages=703-712;citation_volume=134;citation_year=2015b;citation_doi=10.1111/pbr.12316" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=L. Shang;citation_author=A. Abduweli;citation_author=Y. Wang;citation_author=J. Hua;citation_title=Genetic analysis and QTL mapping of oil content and seed index using two recombinant inbred lines and two backcross populations in Upland cotton.;citation_pages=224-231;citation_volume=135;citation_year=2016a;citation_doi=10.1111/pbr.12352" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=L. Shang;citation_author=Y. Wang;citation_author=S. Cai;citation_author=X. Wang;citation_author=Y. Li;citation_title=Partial dominance, overdominance, epistasis and QTL by environment interactions contribute to the heterosis in two Upland cotton hybrids;citation_volume=6;citation_year=2016b;citation_pmid=26715091;citation_doi=10.1534/g3.115.025809" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=L. Shang;citation_author=S. Cai;citation_author=L. Ma;citation_author=Y. Wang;citation_author=A. Abduweli;citation_title=Seedling root QTLs analysis on dynamic development and upon nitrogen deficiency stress in Upland cotton.;citation_pages=645-663;citation_volume=207;citation_year=2016c;citation_doi=10.1007/s10681-015-1564-3" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. Shang;citation_author=Q. Liang;citation_author=Y. Wang;citation_author=Y. Zhao;citation_author=K. Wang;citation_title=Epistasis together with partial dominance, over-dominance and QTL by environment interactions contribute to yield heterosis in upland cotton.;citation_pages=1429-1446;citation_volume=129;citation_year=2016d;citation_pmid=27138784;citation_doi=10.1007/s00122-016-2714-2" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=X. Shen;citation_author=T. Guo;citation_author=W. Zhu;citation_author=X. Zhang;citation_title=Mapping fiber and yield QTLs with main epistatic and QTL × environment interaction effects in recombinant inbred lines of upland cotton.;citation_pages=61-66;citation_volume=46;citation_year=2006;citation_doi=10.2135/cropsci2005.0056" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Wang;citation_author=C. Huang;citation_author=H. Guo;citation_author=X. Li;citation_author=W. Zhao;citation_title=QTL Mapping for fiber and yield traits in upland cotton under multiple environments.;citation_pages=e0130742;citation_volume=10;citation_year=2015a;citation_pmid=26110526;citation_doi=10.1371/journal.pone.0130742" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=H. Wang;citation_author=X. Jin;citation_author=B. Zhang;citation_author=C. Shen;citation_author=Z. Lin;citation_title=Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing.;citation_volume=22;citation_year=2015b;citation_pmid=25656006;citation_doi=10.1093/dnares/dsu047" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Wang;citation_author=Z. Wang;citation_author=F. Li;citation_author=W. Ye;citation_author=J. Wang;citation_title=The draft genome of a diploid cotton Gossypium raimondii.;citation_pages=1098-1103;citation_volume=44;citation_year=2012;citation_issue=10;citation_pmid=22922876;citation_doi=10.1038/ng.2371" />
<meta name="citation_reference" content="citation_title=Windows QTL Cartographer 2.5;citation_author=S. Wang;citation_author=C. Basten;citation_author=Z. Zeng;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. Xing;citation_author=Y. Tan;citation_author=J. Hua;citation_author=X. Sun;citation_author=C. Xu;citation_title=Characterization of the main effects, epistatic effects and their environmental interactions of QTLs on the genetic basis of yield traits in rice.;citation_pages=248-257;citation_volume=105;citation_year=2002;citation_issue=2-3;citation_pmid=12582526;citation_doi=10.1007/s00122-002-0952-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. You;citation_author=X. Lu;citation_author=H. Jin;citation_author=X. Ren;citation_author=K. Liu;citation_title=Identification of quantitative trait loci across recombinant inbred lines and testcross populations for traits of agronomic importance in rice.;citation_volume=172;citation_year=2006;citation_pmid=16322522;citation_doi=10.1534/genetics.105.047209" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Yu;citation_author=S. Jung;citation_author=C. Cheng;citation_author=S. Ficklin;citation_author=T. Lee;citation_title=CottonGen: a genomics, genetics and breeding database for cotton research.;citation_pages=1229-1236;citation_volume=42;citation_year=2014;citation_doi=10.1093/nar/gkt1064" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=D. Yuan;citation_author=Z. Tang;citation_author=M. Wang;citation_author=W. Gao;citation_author=L. Tu;citation_title=The genome sequence of Sea-Island cotton (Gossypium barbadense) provides insights into the allopolyploidization and development of superior spinnable fibres.;citation_pages=17662;citation_volume=5;citation_year=2015;citation_pmid=26634818;citation_doi=10.1038/srep17662" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. Zeng;citation_title=Precision mapping of quantitative trait loci.;citation_pages=1457-1468;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013918" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=T. Zhang;citation_author=Y. Hu;citation_author=W. Jiang;citation_author=L. Fang;citation_author=X. Guan;citation_title=Sequencing of allotetraploid cotton (Gossypium hirsutum L. acc. TM-1) provides a resource for fiber improvement.;citation_pages=531-537;citation_volume=33;citation_year=2015;citation_pmid=25893781;citation_doi=10.1038/nbt.3207" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Zhang;citation_author=J. Li;citation_author=J. Muhammad;citation_author=J. Cai;citation_author=F. Jia;citation_title=High resolution consensus mapping of quantitative trait loci for fiber strength, length and micronaire on chromosome 25 of the Upland cotton (Gossypium hirsutum L.).;citation_pages=e0135430;citation_volume=10;citation_year=2015;citation_pmid=26262992;citation_doi=10.1371/journal.pone.0135430" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="twitter:description" content="Cotton fiber, a raw natural fiber material, is widely used in the textile industry. Understanding the genetic mechanism of fiber traits is helpful for fiber quality improvement. In the present study, the genetic basis of fiber quality traits was explored using two recombinant inbred lines (RILs) and corresponding backcross (BC) populations under multiple environments in Upland cotton based on marker analysis. In backcross populations, no significant correlation was observed between marker heterozygosity and fiber quality performance and it suggested that heterozygosity was not always necessarily advantageous for the high fiber quality. In two hybrids, 111 quantitative trait loci (QTL) for fiber quality were detected using composite interval mapping, in which 62 new stable QTL were simultaneously identified in more than one environment or population. QTL detected at the single-locus level mainly showed additive effect. In addition, a total of 286 digenic interactions (E-QTL) and their environmental interactions [QTL × environment interactions (QEs)] were detected for fiber quality traits by inclusive composite interval mapping. QE effects should be considered in molecular marker-assisted selection breeding. On average, the E-QTL explained a larger proportion of the phenotypic variation than the main-effect QTL did. It is concluded that the additive effect of single-locus and epistasis with few detectable main effects play an important role in controlling fiber quality traits in Upland cotton." />
<meta name="og-title" property="og:title" content="Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/9/2717" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Cotton fiber, a raw natural fiber material, is widely used in the textile industry. Understanding the genetic mechanism of fiber traits is helpful for fiber quality improvement. In the present study, the genetic basis of fiber quality traits was explored using two recombinant inbred lines (RILs) and corresponding backcross (BC) populations under multiple environments in Upland cotton based on marker analysis. In backcross populations, no significant correlation was observed between marker heterozygosity and fiber quality performance and it suggested that heterozygosity was not always necessarily advantageous for the high fiber quality. In two hybrids, 111 quantitative trait loci (QTL) for fiber quality were detected using composite interval mapping, in which 62 new stable QTL were simultaneously identified in more than one environment or population. QTL detected at the single-locus level mainly showed additive effect. In addition, a total of 286 digenic interactions (E-QTL) and their environmental interactions [QTL × environment interactions (QEs)] were detected for fiber quality traits by inclusive composite interval mapping. QE effects should be considered in molecular marker-assisted selection breeding. On average, the E-QTL explained a larger proportion of the phenotypic variation than the main-effect QTL did. It is concluded that the additive effect of single-locus and epistasis with few detectable main effects play an important role in controlling fiber quality traits in Upland cotton." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/6/9/2717" />
    <title>Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"BT8ZVtDa0NdG4Go0weHSdWXZNiRHyZsx6Hh0mven0sA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460296","apath":"\/ggg\/6\/9\/2717.atom","pisa":"ggg;6\/9\/2717","ac":{"\/ggg\/6\/9\/2717.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/9\/2717.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2717"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2717"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/9\/2717":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EVQBk4PNZ2CxdHgeTkrod5I+nClyKW43y\\\\\\\/\\\\\\\/7IjXLlTsiL7fEuV6EXNADJcxrAUUusMMYMNTPDql7wfmAXBi86EGipFxgYu7RXdDu6QQLOePSc9rjLB+8D3GG9+LHyT0dYowa3b65MtYGbLHojH7lYl2ePPLzX+AiC3l7ERd9XZwGqKumPxgbFg2F8EhtZkQU4a1SqQzB7JGTUB1s6S2lZiym0pTu+GNhApLoRuG8ToOdGpeGmwDykIBbk9r94GVfjFrREzdgG+8eTNkWLklIp5JhZ7d7Bb7OD7\\\\\\\/sd\\\\\\\/mRu\\\\\\\/EWj5Dib8jhD7bQ0mGTRw+DfIyEfZ1+ez7Ihj6CdVMz+6xJB7y7LmX7\\\\\\\/qOowm\\\\\\\/CK98N\\\\\\\/+vfFK5E1+5TfecaA+6u3tY0NwfizhTm4JBP+zNZSfDGdXkhI94Ov25xyXgW3Dn763jsa8N7Yie2zOUEYYJdy+xx6r0WDmuOj1vm1Fan6TZB+A6FQ+nVidSvqufbaQJqCQdVqhJdOV5ypfyvXnuoyz\\\\\\\/1fjILt+OQmBHoaCRIRuYKonLZfHcyN4ioPbRi\\\\\\\/uzgI4KPyIjTcApb2KMspqULuxYcT+bzXEVjCA4iFAdV0w7x4YRrK8A+zyDf7+8hYytBFbulKdQY0lSHYwgVA+yz+QlEHU5ORtrAeH54xInxWwZUHYhytr1jYldoelk5znVr8sMWr4jOC4rxaXevDDGlIEVQTaBlth+r9C2YgqSXWb\\\\\\\/\\\\\\\/sYiepBXnu+8GOJn2sUkSxQUnWdN3N8mPyV+SLB4tsDXyLmrDIKi3JEP3MPZD97URbnJRK8CEc1738ls60C2Wok\\\\\\\/K1XGy0PFY0FgEhL6g8HgMwVYXd2tAflp3ETWFPEDbqCHlwjwr43Pi90OFV054V8VDyIMdQR6rlPEoFsDX2nnRsnG0TeUErViKS939LeRMoFSv0ZtNn6pq0MIAmUWlumelLREbo8PbKnAyLfx5nCDq4E1gmLENZ1wyC2qVEJv1SjZAq+K8wZ4WHGaWDjJSS4iBEoreP6uosJgW7qSKL6lZUEhIl4p564CYXKU8AHouPc63J1Dbp1jyDgc6JViyCYRPgqcbK2gJ8iLqwag+446cMXOrjC7Y91LbQuy76KZu5W2zO7\\\\\\\/n\\\\\\\/SuiPm2u86r7RZ9rTdnorCzrQP4f+oqElf+R9mUCVtWSVjlzhqD38lLipDoYC0dRV1Z9oFzf2t\\\\\\\/Gw1HwVwKz7fEBCdyjj6nuum0KRo2AKo9XREDdGfJ6MbRCLTGD2b2NSJjDqqtFL\\\\\\\/W7k5Em2M1XbZ+5I0zWKcbz\\\u0022,\\\u0022iv\\\u0022:\\\u0022w5WDaZSWiWq6bQhU\\\\\\\/Dwvkg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a062c7a7b6ddb884280b52d8e4f7b60d189a9af7685275f4c81e2c158fa0aea6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MHbfaJpGUzoj6jqVSLAuTUAz3GSBxuNluiUDEz\\\\\\\/u0\\\\\\\/127jsvP7jrnBmehNCyqPS1K0SASQJJqRfrWyDIRPTAnwvD4QJraV12yfcdMtYdRYsAgidxKE4qIBqBSLYyb40lKjtILVqKO32O79eSMDwLhy9FJ5\\\\\\\/IeDW4UZAhZQDW0RrYNjfWjQ7uRyvGR+rWIJoN6lFwMjUWSCqJYvXoXyPUii9bVz27hT7j55E8MErQtdUtnWVPKiV6tFQKJVBetbaixgeCtYTlzqoExN1YCFuUQnlgJA3FMY5Pyyr55CdILBRe1i0S0tXRqX78WyejTmmtq1z0SK07C1ZS1PsE9pOnIoAD8Z+6Z0o0jhN08N5PUa17MbVEedZ\\\\\\\/wgxrm2pkLPYAU0C2B9SsZquIw1vesr31Nvfo1uYgQq3MeKcmCWrpdn6pQPXlcnynWOmfethmkdEV63z9k78h6JVW9h\\\\\\\/x0eUMQ+jKusjA\\\\\\\/vglfBBn8N7sVyrRY4TUDn67AR+VHoyoe6sBCveNhzmrHjsT9Eh\\\\\\\/xFAJO6wYE2uTlyygar3d0KytVa4THmKFOZm5kbRXlA3WbKfKH633zA86p+EqXP0FLOM5J+P2ibwi9kJhrMKV18qjh8kXXSlB4Ctq3atROYnphOVreI0fuTaua+HTGk+FEu057rNHQLl1nzQjNNfEoa1uT3soXsjuE4cCf7leGuzj6hhGcOWZpTHb\\\\\\\/LhoZEQ6rrkXCKo9ZrIzyedGPCdfynoSovy6N5E9HfmTZMI8ytuSLHpiRultoy8uXoQ3Mfbs9x1IyhVJIUSyoFPFjB1vdM0NkXT2eg+Lys1mTc\\\\\\\/r2k3et9ZY7XjUDGGqaYjb\\\\\\\/k7Dk70ScZeq3IKnF9Gp8MI+JhZx5HW9YWQCp+tHaENg+Rj2SfroiOzeQxfrfIpn8DrMeYwMRqPeMC5iUSEOycMQwiBzTBKyew\\\\\\\/2uM4Pq6NgtS+LgOneFVKNiwRcn8CEQgJhvBFntI1Tyr4TlwxPLnOqMJhFCkJIJ6r9kyESFehNAcGTiJBBDm72W1TiuD+nhDgVn5aHef6WpvHIWsxLGtYu1TFNML+XLkDJ7vi4idbdSiSxO3eyxeWoqY1HIfJCbZQi0y37e7g4bVZLb5atzR4HH25KrjCSPkHjdVhrNIiugPgNONXkZQFgplBuLPUAhhEXPZbVhSopBT87+63f07lmg86SCvJNXgz2A2Z90W3T0UVgFm2TnR5a2uZ3ZmWLtfdZstX3x8+Vb7RjpYDa8kweZlQA1ha9qVvrEASN2kia5B2Oh+EqTiltTtfaW9fXnEH1jB6X31+Z3chmYRibANWU2nVqDsueWsb8JGI03sqXX3VVOWL2dZkCAWAnhtu86H0p9YW77nSd0SDEWohi5M7nJwereE40foIB85vQ7bXrQZnj6tIH0718TCq1R2InKCBkxCaP9c4I1u70YM83Ry2CRvo2Rn8hTXtHHmJERSwv7D1lPTDhWikK275dEgV2an\\\\\\\/yXbErVhXlWIPJkK55m0mKFhAak5p2\\\\\\\/EoGgV0TA3Sf0wbIypPVh3cZHDjOQS2D2ZJIMk2EjwD6n3ZNNsecNK+ERGAgq2+5NzEpBSprNnKOevGFN97vVRz2k+8Byw+3NuLrcDmZlvHkWWU\\\\\\\/L3UCTQyzq0YHNBSokFxKgv4WFpf3w9r11mZt8eVqSGmEiVjcaEqDvvZXx4qKFnZme7aXeBN4kJuMydFjU08kIGPi+gXe04Y8Cauzzu\\\\\\\/I76O+AWR4C37QeIZtAd3T4VsoqjD6nUwdqjCSbLV\\\\\\\/JUqAcix2Gy6YmfgWawqR\\\\\\\/qEPmxxQlPcJKsdTyFvTjFWEYhOceDf0S\\\\\\\/ppvwsYn3v\\\\\\\/eOP\\\\\\\/wlxLu\\\\\\\/caAF0\\\\\\\/Mg7XdWFnN\\\\\\\/RugroIGmwDBOjHz06QRg+n8xueMKCAzqXdjj2VbdFV7ljQ9Uc01x6Svmv8kXdUtbzdVGawGiUTQkbpbwUTqW6Qe9BpIVGKNEq9lsoP+gL5oKIzQeZrung9W+3BhLPQBj1CwGpu3sivFwrTkfOwhNgvyLIsWWPrtfHFHrK4fPXlRu9R1hCm07cNRL6MruJwfMpoe\\\\\\\/0l7UPJAlXUmvQUjPJy7uADpm4HApmc4k1ofAKg+FFsKEuWCKPuGmVuvgoYPeCizjoE4Y120X5xPQk4Q6qYZZbpZbCCEltqjC+tdRSEYZO1FhM73AItvu4Lp0UYgULRx7bzTRn1xePDx44QuixrX+eIWCSDbiD+Z2IgEecRFJLNEu8qBbdelfUA6lPoeODIlQ93jGsKWzIaf1ErkZQN3LkLKHFtDJ48poYPyzG3emoN21Smrx7ZmQ5iQpt+LpO0CtsM1wKCi2IP9SVCDZ2YbxuqLuWncaaadCrmX7TqXHuLjg2\\\\\\\/mf0RqFosG2gn+kIZWyxFdbYtvb2zBRqY4Zy\\\\\\\/vVN9McKhGV9t8BeHArWAGQDLwaGcfDLVEsX8eQrnr9FZ1CdnKco\\\\\\\/xsW7gIAcF5MBw3Xh8pEPqLuSWnh7mG9dCAxD2B0MR9MqJHzuvheH6G6A2HOXSjcwn2v2wG8JSW30ASNMQPtwARgus\\\\\\\/hkmf8IzvRJN6icm+g2H+GS7SgT1fBnAz\\\\\\\/FP5qKCHIPELh0E5EAm+5I3Jx1UEYSgwhTPqY4bmunJXFMlJ6PMLqCpJxFXhtranryr8i9zSr6fTvkOmNXyC1Qv7M2Io7ANsqzoLaIW1SSdNILAFFjw2CeqDHBZIXaq\\\\\\\/02fLgUuXu6wKLUv+I+NOFkG9vofAn0Jy37Esl6tPgl9JdFxBx+Jvxbdskr43qknJQ2xxRF8klnKtTCryiqydbBEMOtOpem\\\\\\\/kWp4xSlFCWuoPD28gPAEWcqZZsrupvgz6VrGuedpKvRESrWM5iuKXkQuOEbW9GfcNC6Kv37znyqwyQi2JyUR2DVD0jQPUvwGkmWaYvJ1p8aTP7IXeLeHEg0mNTUfZUQlZZVEaNyU3WZG17xgCiypERH24lUPjJ6smNA8Njow+uC8rQ\\\\\\\/BlRLR0pgqX25YuFjp0+VfsPr+EPlMYxaJJobFnjMgz50P7TJv1PhZ+NNmJSxDhBX+RgZx58\\\\\\\/FXGVNWf4w\\\\\\\/AvhkJCvkdagiNJlrLS64OpUAZ287WvVmA7O1BcJ4pZJawQsGuK1PPIUp2MS0pim50eFsTsgahir3b728YJ4vA5VH964dMazQMrg6ipjsi6PD5jH3vzYayKSMVFpY52sRJUhfxwlY7U16wyQn3zs+t4RFMltEJKwdToZ5QdeJ5djksPIUxWomvD9x9\\\\\\\/iZgad\\\\\\\/Xgw1hxsJ72W78ICrG1QlLi1CDnSYtbD7ypkWoIRxKn9wtic8AtDyD1X\\\\\\\/9u\\\\\\\/XuLp4uJ90m1Xbgw5kMtj4Q\\\\\\\/ozNfonlQLNpqvt45QPLtGmih+XFVKLw\\\\\\\/h80cAutjc0PsTyE8YpNbTekSS0cASX9UbXnH0hAB5Nb8tquclq0\\\\\\\/WvRzhG6f1TN5gSCP6V3vIlrb9dxmyuVuIGg8caY6SZiYfThZUqu02WK9Y3efzB1LdXLXn4nPQyYD3fHfdX\\\\\\\/Evupa+NZ4eF6PeceBdZSFeLlhWAM5M5ZqUaS\\\\\\\/nZYZs2fSdb30aNya63kPrBbE2JAUMYfZeKZNesr5CxoIfEYAGAOyI4oJyveECLhR3ZnAbl5TJt87bYlBr7Eda2lnaml8tvMnIPp8yX9M5WUFeha+NOyfztuwqeufEYBiFkFF66QOcpuvc0KoxIzJFlq3FrWyPZyNsvziKIT3a4GHOrVaHZlhUcuxS\\\\\\\/4ZHHfTTtr1mcuIy2s+SyEmMb1k3j09yrX4\\\\\\\/Z7OGWjoFb0x8ovnu2Ve81GFyNHM0KFwJZ7jWntwMpVCAS6AwIefE8eEJjVo9H62efSeZGgBnob6qCs5MSp9OEB2nPC0LWwin0bcKT61g3QyLZDOv\\\\\\\/18DC\\\\\\\/lpi1muEVI4vR2uK3qwBNrMy2XzyflZVkPryK8WsGJ5xPo8tVipzSYwxKww6rtpzHYONlSoafMg7l0MvMi2m\\\\\\\/9g8hfchRFqg25ZE\\\\\\\/oEhAOuk4uiVMO5fcEV5Ax0CFUff\\\\\\\/\\\\\\\/zdlraP6UoeuHQibqyQBsum8qg5uz3l+lDPNNomz8xIN6TNcD\\\\\\\/d1vKL6GulBy7Molm2jfB4M0YXCo1J5iqO50pjKeYCWQQhP904JswHndNxYtSelOSuvN1oTUC\\\\\\\/Jp5ERv+nw+B1LYznJKcXE3w7YL6fka3cR4iU0f5lEEdI7++wRXfUeV6tbuipIUPA\\\\\\\/vyFDbEIEJgxX8mZmSIiqx+Xyx6aiPRZGS8ip6ITyzEqffRY4UnNLFLyfWcuLc9r7AONWW0xciOl1+y0WlqR3J+FpbZY7gurDdfbCnpFu\\\\\\\/jTpBOXJ5x5xnIB9+wnaGo87nm8iHIopGnkUCgO099VWfu\\\\\\\/id8TgmBAvv+M43TjmP0KdVec2M\\\\\\\/Qe7Xyw9yt3mDQon\\\\\\\/7LyNyGYE8j8baDbAiQr2aaIU0s4yhRL+3MO\\\\\\\/pDBBWZ0idQ38z05okV16rFDR\\\\\\\/q9L7XfM95Hoh\\\\\\\/ff8n1Yn5jUYR0pL2YuaEkPsr00dKKlfkrQx8OlgVZNxCclIjP5ig5vEF9gxhHVmD9Wwa6uKPBp1rc\\\\\\\/wuch1v3KSwyEideeSetUoJP9win3aWwTwloGTod9wghjx5Mv6nz6xwlN2RU1zdVi0FgmTbhNxvyD93ONBxpcJzmzAAuku4Oel0TYmOpMeAVW4ejz4O8jAPA6RFnpkwYUwjzDesGG\\\\\\\/fzmBi5qFY6MzcQ1\\\\\\\/tyfZJ8T472wYXtQooIDUNMvc+OFAY+XPS618qsjSwtC2V1jbm0BejRPWRnM3nZZescn0eondyAOgWIdfnd2wd+GNGTR8jmjiWQq4yR0ia\\\\\\\/WLMkf37FI51XudihVO614+J5jrRHXI7A6bG86V8VJVBeAYoIaAuOmA5JgxYTN39BsE3vCHx6sH1RwM+n2cmsCOGROu7V7uwl7OEzDmy\\\\\\\/PUzIFlu75zdQy+9f2l5HQzLJi\\\\\\\/Qo348pxqYMf1xJ\\\\\\\/a\\\\\\\/C\\\\\\\/hkVUbU1qbCe8z5\\\\\\\/dLtIPN7NlBMA+nTn+vfzkZtuuLLeUVLv3Tg8HWoIDr\\\\\\\/M\\\\\\\/tjhHjeC5Xy1fEMqr4\\\\\\\/jmsr3yQ2jayi858SrjjEPAgFJllDvRzGowiJuCgQKuFl\\\\\\\/CYx4DGbRTxYQc646i9Hr+1abx1c6F925I4VlKmmaDdE7aHJ3X\\\\\\\/IT6R22zQhBVlHknKMMvUemYFZbv0FIZJYKf2pap7jWeV6MaACsFBAEmNjkr58MTn+h0ryx0IVx3wvIiwjhZgsFGegyQQwHZHTAAZT+H7WdFFJM0B7eISDD+7dq47Owx3gh9VfwlstIIdfnzV6uu2JNyzEtQfTLFPv+Yht7IPkLL5bm5mVqFVTRmyNKScGTM1T2KiN+duorx+xQ+64IWSWPVUumFBxyMus6M\\\\\\\/itpQRyIOaYkbbqUzjou36rxFso7OQPGpFWO6sbnexq6bH+pFH6HFL8m2xYsccB3SahkCp8\\\\\\\/h7LjNBq\\\\\\\/2xAbjMlAcJePLVeCHOk2jEIED\\\\\\\/uNte3JhMwaMxrkZqh4kDqPDe9SiJ1UTtd9qw2Q5SsenCgXc8a7mDL9y1ORx7xg+PIaPpypyk04soiFIIJXD5EtQc2XWHrsSVHBEs2yhRdoxxN8e5Rsj20c6i+PmAk2zs\\\\\\\/SCANHtsfl8EzWpaOwxiXYAaDfF9rUcQL+AftU9FCtxcOKWv4YWfnhKX9JthSMgs8bqwNQYxJvAOU\\\\\\\/qVtXytkbWH9vfFw1mI7lknPrubbqQmSUzrEMvPgXOWrn2s4QnnQVJjjt7\\\\\\\/SJpOPgD2DXwRiiFccOZXT3fOMa1h69bEw+YlAxvJk2zFhGpI6ytJJni5IgsJdryyhwOIAC6qD8kbOf8Ebg0P99xisA5S+hET1cpt4Pa4gvxsydE4whEY9iuGX79+bs9iAbg78eNkdsUDkfsqTIwbchw3\\\\\\\/jySFSiSezMFrWa6laHUwsuwtayK4Z3V9Uys+ieeEScTPLNwbAdo2jW7wjShVqXj5yTqHBgNad5oxUG3te7hcS4oqthE5Sl0XNhKsvWuNwv6TfzwaQ5\\\\\\\/GaSB\\\\\\\/HamwIhsRUBe3VPCZJYCLnbQyK3ir6R2up6iS9XaKz+DzkE8dCQDinq\\\\\\\/qjJKGhGlg+Ebhrx3vby3IgiNvXP3izMEADVWLaa1DUkwUKHdQ3H6UU3EYV5FghCWt4\\\\\\\/+CAR0fedvDG8XkhQYjoTbENegg84NfyiWs1UnDKRDKRxXI0OqfdoUOItnM8INqWxD02qGc+mT39sbxclKP9sI\\\\\\\/OATuaSblXlSwmbLHpL9hs011N47Qgl9oYRoef1ziAPYhb86Dk1kM95oX6M0YtEqgcNXOzF6lGpT4kkfgIlJOM5QGvttzjc+8TS0S9m5ry7Nb0sEOrKBhTNkgMBwrd\\\\\\\/nRdssgRhCn6lbRkc0\\\\\\\/EkyzvMSgASft3wJKGGqFu3kBCD720q\\\\\\\/XTHxtbvApIZ21EkquoZtSjVxKmc4PeGX7OoqJh4gYkCqIWWzWRE1TXV6Vb+DfnuCWsqrYBJVG9LekGDnTxO1DGJZnojahLQpE0XEUBW6ZCp9udmK4XC907\\\\\\\/KIPWN9+P30\\\\\\\/aP6yq\\\\\\\/lAgQD4o9BoWiKLtxSLKuLm6ehy2K0VeXrMNtXB0DjlAz2AeXnSDim7VsejcGtWZZ6rn8ober7M72J4xT60JVoDgSut5fevhKWvD+6ZNyMPbZX1oaMuwHBw6p9iYGEy+a6CwxNiQwyM90bPxMCuFmLgNfBK1oESBY5ymfHE5TrkjwrI5NJEl2pWLJuMI+fu2XdBU7E6gch+dnI8LkLMkBQHB0bvI5Kruz+vyYwVacv1Fm9QtPxVeYHLO5htu5NShxCTtESkJQZ9mPcafex802ePpDrZhcDx27D4w\\\\\\\/F3lRTDGqYKjq0BkIMFPzs6e6k\\\\\\\/oDrVrGV3dMCXXGe1VPgQvwrbr6\\\\\\\/\\\\\\\/rE5k0gKpWF3BPgb3rTrMWsdI7Jsw+tLNCj7CwgrldHvFDPJ7EOqnpCGQyfDj2xAn+yKJEw3TRiM0zzBWrs\\\\\\\/P9r5e4PMfkhMCJyyEgySFb5PseV0av+Xw9vbDDxroqgVCrL7YZYnfadTyTsjX\\\\\\\/FbsiOCUpGd+yGoshFQB9BkJ8\\\\\\\/p6uIbjdIc75hDUKNOTs+6NMIZ3v+8Eel+dDc8mVSvMVS03oVGeYPbckZk0Lsi8vvpw4mZqHzHqLVWPGnv1FYCzVaKQUFoc5+eGEMX8lDtGH2xGoY\\\\\\\/1BJpNRLEKpKHebMNzQBMPX51y540tOIr0UK95hrVhl4iXIxJqb\\\\\\\/zgMp+s+U9H6UPt3aNVHz5I5uLJAiBo7RsiqfKt3nbH\\\\\\\/5fcxQxtuguYUqFM+Vx9pIQfi+KhHH1IluJ4gbc2zWRhKnaGjdH9O93f3cXImDuVUC28lh2GM8Z3NeU6nBqH+4DOMPkzd92\\\\\\\/8nq\\\\\\\/fcxhqZH7Jys10Mb0331eKuytuN6Yeh16kWUfKAfyJ0cL72iBuMfmYkRSoojCYG4JxegiKFMLzoZAkAd9Ua1jAteWJye875UdV1RxzlTvBivoK0Gf68ODQK7aRJFDbvauyUMBg0lVxAKscPr1E42ZLvn0TBoH6Q8wlfQzs\\\\\\\/q4dKrWe3j3wGD1T0Ghm4Bzp3bpzAnIrL4QHfK8vZETT1HTV8yEKPA8g86rXk24HnJzU5hZc+Hdz45sVZoiFX3om5TrCw5EOBBtNnIXJdnD4gCK6zHHyMvSPIJGOex6f8FNYVLyOjmrO0tCQV42481pA2vbHF+j8uHzTmC4E\\\\\\\/liXoceWp0\\\\\\\/lm5upAAZqj0I8Hp+K5VkzL9Mwdsb3lbYoJHf+YRNh+1tk\\\\\\\/hzI5pQD6KPxar0+pvi153whKkutDi1p1hHN6Zof5dzMvqe2vujmgcCkbTm8pURsuUGsIOAyhXBk\\\\\\\/f35J26RpIkmVqQE9YmHdfVaYcvmOp8F+6QOIrVofy+fO9mGrdOVwucsbCgW5LCZ9iUaKLWlEuTbzfVTx+uPSU2UBRtHgq1w1lUigcK8s\\\\\\\/i2MmU+8W43QQwhrDHqhSSRigEv2EH4Pu70+inxZkeCfDirv2D\\\\\\\/lQUm5B2jRJWqsfxwlJCqjH6IQgUmQdKOX2KfHo8IR65gvZdlfIY4GLrGVw\\\\\\\/j3KlB7U8qHOOd8SuTfG3aguuVM2ooowrJ6CIUcluUDCgYEK2Okh\\\\\\\/RzRDsWk2KGnlqc87+zK+M3+eWOJP1LQyiEg\\\\\\\/aaJneY12kAmhNuMsryeU8lbcE0zealAuMnettTfgoNrwEK5e1JoHqlf3A2KckuuJYq8HQCw04TSFT4tnQ+\\\\\\\/bPYuNMI5j0J7mu\\\\\\\/aHAdv16M7AJCO34Ei13Uvs1nGxnoEVwzt2x0RnjreYBKHHfi9Zt\\\\\\\/ZvMWyVoLzwFk1FMeZ7y7W0+Qy1SxkJMS3s6JpFV\\\\\\\/zajKwvS9SMiPa+WH7mdrYbxTVvZZYZUGsw7NX\\\\\\\/HvbY\\\\\\\/dL1fsVMgvB6wxeOxY7aLDZNrGIpjpXIzxDVUgK+aetXS9k+kTxZAAkta9pNAaX6bCURnneuf\\\\\\\/3M5CYBLcApyztdPAwh+X222QGdu\\\\\\\/qfQ3ZFlwy52RnloINNMsDWjB1QXl0LM+hKCQ8CvDbO5VwmXUo4CKdZAD2CCAxcCu16Amcv98RSNtSVXeWdRt9UPtpbFEsH8+TkKHcQiS\\\\\\\/t6DS3rH8WBcosI0x0I8avsU3zC62qrp2\\\\\\\/58qDs7sgVIhtqus4jqkyTRZcLXt4CrkbVLNTab5If4wVWE03CsFY3YtUrfzux\\\\\\\/9j2rxHQs3EEOxfNrOeVZWtPz75FSaykbg\\\\\\\/DvutoUbAMVbRMtEv2T1RqSGq7MGptU3yfBh8WGb0YJHyx2gz8z0+D3+7z7dR2TFB\\\\\\\/wr19W14M0Nbbnm+k06h2yu\\\\\\\/C2yU\\\\\\\/ixK1W4MTyfk+hvNkOdG4BT41QzPmn6P59h4ULc2U5dYilWBIVpRoqkgSiH4ybFCAkSo\\\\\\\/Rfqg738HPVYQZG9kblLOGw8faSgx5L05xdUD3NPehylbzSWa\\\\\\\/qADr6N5OsSMrEetnFToS5rRnPvkdo8OE7uwyvZV7PxwAHRDTMrKGIU1ZQclhL98swxThQHaL8qktxYPUdwii6IywKpZIMBwLYMnIC0xgwh4W9qEjoTd5L7MsNJU8mhLWAyrxjyKOHtW1sTRbbTrHlYgyLubKZ4eh+rcJzyPcFXIoRJ\\\\\\\/cGV2AYSMiFgm9oYTQhg83iY1fVastLsohGJ48inNj3BAhSZTolP1QBnpBRLv8I7A+\\\\\\\/5LFKnuLe44iwlzMTmefXl4CzuDPNCazmypiegtvlHM8TDEJvVAnPUsc22CyPnhAdGIvuRAVVmj1PWkVzoG7jv40Vb\\\\\\\/XnWHva+zSlm1T6Bk0iM2WbnBUlPy0tIhhWF5BOu\\\\\\\/FiKTDHEB2IPG37qSrld787hagi7NGA+3AXwqrHI9hOs9X3meClut5I64ruAgbWcbK4azvb0HYXU7EPkUfVpexm\\\\\\\/m3zhBpwoyVEHc4dR+1RHUvmM9Gx3vmY4+2Z9sE6e\\\\\\\/1hJrnmh\\\\\\\/NuPBnkQyWNu1F+dyHK7FAzdPIK+cS1hMb3iW5LgUIOjJslkDkX9uX3ZLR\\\\\\\/JpicctNOlR4kxswVGo8nGSGYcwAGNQA3kLNWG6RXfvFkeqdIzzi\\\\\\\/shnXECby5+lHmySE4rtkbeD9WV4LJxhfBjtmPQTbk2peTh\\\\\\\/AjSOBscjeuZbIDMeWQ\\\\\\\/yX6M6\\\\\\\/RHmHLdVyYKrAQdJB78PaIZcAHf4UwkbdzT136ensSS0otfQ5JpE\\\\\\\/5\\\\\\\/Wuvn+Cf49Ynd\\\\\\\/qqVzcGLObkrFdPqths2FK5zmA0N5dg6Ayh7mIvMgRd8Sa1RyFcRY7MsWr2gilK6r\\\\\\\/8T0OaK\\\\\\\/\\\\\\\/YrE0mWzeY8vN4wHeFrdJ2BmEAsgZ8DgdfTwdGuCnkAvkalq9DTgQOaqD677kO+AUT0cAPLwYVlWvFU1JLYwwgN9yxYzoPXwqtabdAxuTCLdCu1ULCN0vOkbF1ioYlLh5JHTKIRU08YTd0E\\\\\\\/uOGO5oqF7fKx2lIAOZB+llDFOmKBPbQ1S7cWUYEquQYR8w1+SFwwth2fbpr7oBhgW4IvOA6DC3rhUP2LbJhbDBKsWEBdkZjh11vSFILAp2cylSbvGJKCuwGE2N9sFcP7j4DgMVuNzrQJ+Oqp9A4R54+sJ8c3r11zMi7BzIN8mrfjW1FtdCpbiefEmd3ZFWRhbibDnR\\\\\\\/JretafL7sFkY2\\\\\\\/N1rKHA0F47NEMtLHmo2\\\\\\\/2y3OPPq6tVtOp96GnaKwuZlR1gb2f8+X49\\\\\\\/JVdJJOoAAow135CKms2RgzgYadZLT+Rs6C7tUx5\\\\\\\/yfHjFqVm35J6o\\\\\\\/CcP5W92+\\\\\\\/ENJjccMIivxSpemXOBigbKZsKpB0W9cRbiwm+ahaRZKtLRW14pzgrULU4S6tZUKYEEUpTt3sJZdVnxXjOpHX0etE8I0B3J6SXqenBtuHXam3o2aOkGhA9nHYYBBzpL3fRrXR6euY5i4mLvhA7gY6vAb1XyEiy1VeRnm1m+27EBHTABRillKPQFOsgb4YanO6lJJMmbBzP1zejY5Bag61jQR7TkViY09b4nIlNbIMdISuUIptHdmREYFO7QaDPZaIPCGlusgKcext0YTXqhLMeJenJng8vYawZsljGnPnqAsayPCBJ7LfaGxoS5NVuVV4OkiFkFc+Jqx1x1YMXwzb\\\\\\\/\\\\\\\/M80fec\\\\\\\/Ii7FFPOFx2y5dffxkcmC+\\\\\\\/LIWyEj6yzZZ1tTEdJ\\\\\\\/23F5NpRhMoLPneMc4z1GkjDswOLbgeret2wnlAwVz6LsGBrA\\\\\\\/LDeAtXlOkULRokwrNNEVVTVBCCpq5\\\\\\\/EPwScf\\\\\\\/V60LVfEMpziV6DLxciHjbd644LxK+6mrnqP0PnHkq9pd+mZMqiWqkHig0v3if9SbSWkxyd9q4ouI1+M4bAtbzUasQQbNFGeMKhhC0ZxJqXi3Th52h2D6vD904PTHC9rdF26eB77obrrWK+krdlTEoGbHwiG3QyLUPjnfcK0MtcGDgC0h0wpP+xreYRKqLCioeYRP9m5VdINPJdMIQYk+TbCI+m42rnK\\\\\\\/rkdpT4Qn+2en4pAZhxJgufH9k8wi70QuxlLw+dyr1LMsuCpPo0Heqw3gEh6jPyPkySv9ryi4MMoPdRcSBARyj0EtTBW\\\\\\\/4LrIhX9iD3FRB4Y1gmhqjpR3RlbIjvUs4TmStj3\\\\\\\/KK\\\\\\\/0uqE0HQEgeeEegNNu06iUPRHK104ZyaPpdXWeaRZeAkL6MxVwgLCtZea20rkTT5w4wkQK4qq8nLVU7SZ4gA\\\\\\\/ETgT\\\\\\\/WjFOS2fYA4xE9yAcMUSGijtTJRYt3eA\\\\\\\/buqmzqnbMjeNY4utI547dM73lqSKZ1drGslvEEzoYGOLIN+MTohH9nNA4sqFDuQYpM+nAS7w75XMdm81Tt3WYgQtGeveTl2KskNXMFZ9TsG3UpgkVzmu2EGzMiI4CLVRAqc+wikXSOIGcaEu5J2w+vYhpvORZ1H54qAmqDIkmdLTFhCcowxZzszrzH92YQEUHoTTxFtl0JDOsSlECcTbt8yVqQtIWnZp5O68lmaM5u\\\\\\\/DWw7KUyeZer3T+oLRM4eiCy0j5+vvrkoKqOHwmZCu9Rh+eupLTZIAOLUCwmoqWDXBLk0VIP48h7oCA\\\\\\\/M0yZbW2jydCOYqDjnN6qBhPfTlIELMhWHCZm0dT02cLg8p47ekZgnKaJ9TTrDved9opIyC3O+M8\\\\\\\/CpsZ32CxPgzC9A0a9vk3diPT\\\\\\\/4OCesY60QGv9gRIoY\\\\\\\/hB72Tj3E7ziwmg65qu5nU37PX0YEcr6ehDqO\\\\\\\/lsgdROmj0Ok6itjHwcUDucw3++L6gOIANtTFswEkmcCMA6smhgEnu+z9MXE2pIUPqEvMXbGmYpbWeDJu6DxZ7xrszzuMekkY141QkSsGdlpWHW34spO1t1qXnISdg10LWX3TJ\\\\\\\/IugJ+Tj+DpSIQj3yIacvyQh8D7xr6UumqaIXhZ6Gw9Mbnyf79tTLzdt5IHckp4VFKSSBCnDYrw3jMHepXymegkGl698oDrO4rdRyZ+h9GO\\\\\\\/XWQgPOxtVyV8QA7VVi6Q+hBInDci\\\\\\\/tNQ7dSOWATJkGzjuyyRyAPN9CKn0IQ4bmCvj4dO2a+RSaPA2WS0tcgzpWQRYV6Vc89k81LVXOn58WINr8lR4onl0wgGwqubQ5b6nGW3nBwYsTM4+qrZlI8ruhTXu1NYXuCBUTXmpk7eumWK88FBOWkKQyXu7UjpXxyaJIXB621wnCk\\\\\\\/Q7TsiNT696x3edfOVMx0kBcVJbzp4Wx6R5ZLkpuhIruXQBEHmDcaIUiUksd3iOGHHSxfG\\\\\\\/PZrTvtFkiu5lLM2sD5GyKp3qiBbwt+aXY92bCt225zfSJFxbdtRX1YgLN7+BvCXW71N8cCa4pG4GaOm744fKZHJosw5MLNNyBGVp+DdyGwJKBAWmDm8WmVAhdTOo3t7n11gAmvszahSrjn7A2LyfEg\\\\\\\/dIrX3UNFEEtb16gJQcssPjqxP39FuJwoFzE5z60zWflf7rR+QRsDxGej8BqvvK6JqFgexaUKTeoU19MAly\\\\\\\/GZGRZOhWZlPfxIm55K6vugGpHviNfG9vsuMLQzfNH5mEMzjgEcu9A2EIQSrx+no7D3szslxcoLdQWHdy9+4DNQoNThCIk+nXlzqdjkcAUjbaEfZu3qLQC2gAr2yHhpppFPc7ZAwrrUOMS34u2D17\\\\\\\/oGb3IZxg9tXKrB4wh6muXsCkFYxQpKCozh0oMA07fbJKGmtIndR50YfsUvIwP5KrWesREcmV1kHITN+tuaxxjnzW2X6fuFB\\\\\\\/pTqzjwtWLNzELUlIGM7TiDHggzIucNZBHbaVGgTHogtZaYc4ajEuZFkwHvDZk3RgiHfPflmY0+N+d03foeq+7AL93W4y7JOlIaG9DIhmirXOdhl2OuNI2boodVq3Q3E64Uuw1kO2IoqhEfgXU43eoMXqZwu0OSL4Tx4HH6FX\\\\\\\/5xFW+c24q4sj9kN2n3RvUcCLsIcfXyVEFsFKkJ2KRtVjToZzcDq4kfJ8U7pLzaElavmBKJl0fd6x2MNspXKcUeoVa0Ke7JnbhorduKtYPsbxxrA5WhSmKccZsNpoY3PnYJXTgHGTAl7vvFtIUuZ7OsY2\\\\\\\/e9EnAe\\\\\\\/U+bEszvvwwPwEqiBL5M0AP340HUtZLCYzesxO107s7rtvGDVGVAXZ4YZktLNLteXtRDEL1XirQRjZrxH5mWJ0mLo4H4DruRlhWHjZcPNHf+BYxdMEtKwQ6nBq3JzqZ48nm3xk\\\\\\\/+x\\\\\\\/DwVomMiSGUu6FlNeE3ftId5bqJGN6MRJQfi6VxPWT3KOelAr2lrL9aTkRu7hvqJ6OL74DTfE\\\\\\\/21WyFSytke4bERGyC1vUotEFwGbnUSWMlOZJqUAEacDleT\\\\\\\/wTo+2YLwbLy45cXqu3B8tGSFUkcwKP7vOzd59gYcBi7Q+QyvetpaEGd0obFUwDAmvfGoVEKM3GhZqJPgXAh7moZmuZnMIAH0dhJHwseY\\\\\\\/B+f3VXJBiyCYjkieq50jGxtPpfzgwW4tf0GrFSypRDCu8T7Okl365AicwsClBquSVcBls9zBVdKRn1cHT5hQcmTbBEgediuA+mAgRxe7pFjPMtPWwMnUHiGU\\\\\\\/Le2qSW3e7pCk1hIeRIes32I17oOCvQznxhxjESJhnj+OUp+rXfUBS\\\\\\\/MGMoLowytq6kuNuaPbsdfayj7M7vbALBvpm0U7OJTA0DnHxzs3+LX5U4nw4w6IR8lw4yG2A1hRkgN\\\\\\\/xEOQLrncdoM1iaMq4DJeGg+to0Ja1T2tF+VtnBTTrfxelPHTtDzdMskHprycPSNcP\\\\\\\/El0MPd0biGHtET6p0M1X+y3ZRxaE8iaHl0z\\\\\\\/Nxy+zZiy252HBH5LiuN\\\\\\\/N1bmHEkCAng\\\\\\\/AhX8xA9mV29q5HpUPJGT9IvfTchnK0s5cLhSSGUaHPoUBsFvN6zEV54LE7Y\\\\\\\/hx4Z2LHx7j+jfIU66t4ZNw3cKPrBlCEiHFC4aL0jvKL+6cAOzObAbHMVB\\\\\\\/88+Kxj1zKNPsHR05JZ2+Ymb9pomxbUo1fBMW9KtSgxiXIt1DjHMeCUh7rFfvOwYQYN\\\\\\\/XPcYpIoamXwxIDfh9Q7PMuvWTPIhghilQ4ZChgccLdN5OQ4zbN+RdZ\\\\\\\/I4+pdVOSlspemvlB9qete+hK7bSHz\\\\\\\/4\\\\\\\/+J0yqzZJ7atCoApHC6rrWlkJYKmQm+u\\\\\\\/N29VAnb3IeSp\\\\\\\/cBeqg+D9OsBuKspy3BuV5JQDii9JwchY7morp5+uDr0G8U5fFaKfuRSuq6BVukETTtoQqzlWcfkxat8ue77FTw+snZa9LwNZ7526v+p0cGKXm6oODXVPo0I9f8tZeig80yzJF1kY8ACJMPhWGUIA14KwR1nEaGmt4jtXV0Gdc4WSsAG6fncmJLeU0XaZYT37GPf0wRXESrpHP4TDoS5Dt4r2h5ESoJp8\\\\\\\/dORzKGjHLNAtXRQUBIppd+WehRU42n+Ec7bfjoNx8gV4qy0G6XUXYJx7QLIj8PhTHbXyqJFhhoGbcqv0NLx3HWVAGafnVOQgC4qUYmd1WGxJek4iDoIj1ys5k5oA0hN5pW9xSHNp21yGgpnGlgqT79yGbYdPozYP26ETi773qk7hX+RcmBPwVoJ+0s8cWISRBQaqPOIlc02nZbFJ1Wq7fHJecxE0YWY4MNmuKdeh0w278ktGrTHYy6sXYS7bEVFS0mId1c9iVnIQbvHcFgBMUXp9v2JoU5z31Yri3+qJBS+XvHScwrpff1ooc79h4+YHBTPrYsyVdY3iww8BSbIWy6RkURYK1yZ5IlSj2iwbJF3l6XYQ3JbrCN2CO5irYlZjQAdIyKC25XHLL8JEKSzwJ++0CncTDxGfhexp+HBmVRmrAfQQNH5zQ1p39wle2JQtvtYPsiugMbLOQQfrR5uyIlY8xZKgGjwwlGMiGxAP6ce4PvpY6wLBNjxpMv3VhjiXT\\\\\\\/koRHyEroaYQlQnzgDev8R7xDVhk7rhdocaebpvW+8u1Be2v5wEV6iyjbZbiGHHD4B3F2h4+hnYDyHGZQqRJ\\\\\\\/9S9nLConKMQS8UCmT5giXPDiNDNHrViQ7hMLZBeXjWBDx1ThlkwEO+mjRKoaBy5au8bilXLsLfOfeYbUmCGewIGhJN8GixJvboZ02QDl2P5cNLx9qIq\\\\\\\/upPRpueC5UroJuh9bMFoe0Ze7\\\\\\\/5VOrpJI+nFYEGuWwY5TtiQMXpLVWa+v7zwt03WvrrBRwGgEiUTlPfBw1kOZygrs\\\\\\\/tYjf0Q\\\\\\\/uxenztyquPVeCZcA6nl6l74YJEyLYFSHXsCIbMktgMvcGr\\\\\\\/rY2wOxzlypBsnrl1noOB69jTvef8rqqpmkGhucDwtqcar0MZrKQfXrSjHVRnKFchJgGCRmGvzpcIblHg4gb1YfjjBuZng\\\\\\\/BBzX1HAS8Ubm25LHguIDuVqXH\\\\\\\/\\\\\\\/BP4eOqcLF5RerHE60y3CbbnkaVXn1V9pabq0DLg\\\\\\\/jefzXi+Zdffsu09xU3FpBUDfUGpXTEd6K6m5I7+orHv6SfUaG6+scEfIOdePHwa4Bw9VUafdzncy1S6rleh\\\\\\\/JBQXTENgsMLFxRNtu7JKDvdbHFuR9CqVBlXShTdCL3B6QDw7hE0OKQkGghsCiYJnQLffuEWgCKiVJySunA1Co\\\\\\\/vsGEDiQvb7gPhdZ4fpUaEBLYudw5UcCQL9uCIYsVw7fP4ToGuZ7LPmepm5auNr71WvKZEPZD\\\\\\\/wd8lQgli1WGFAqlJGZfYnqsmaGON4q3SVqju+R0IKdTQvyWCYZTE2V\\\\\\\/MUxeBTpk\\\\\\\/nUAM+oUyvRhqYh4IJ3bX0WUWVvhGGfv7RZOemoeDnWk0O3dlfpxI8S1uxXYoMd8rpL7sNfMTpOYcFT7f6uVuYhujW3n2HJ\\\\\\\/s62et9EwHqffrPjnu9kccMVq5oNJNi6Ge+Faz2fuvJG8WxF7bPoBVtN5J7HPnCD50k7t9wjI+vCRLMWy9YsywPMyovTBymqeSuididcUGnxei6toNasjJFaN9ctTrjcDo8aN4IEp5L6yh2bvy1kcH5FykDbn3IOzqOcnv5LzLXZXl8utL4ggzhzf4oTfHfPs32AngN\\\\\\\/nuuYnIH1LWM2Nv62MJJKClkXacaa2YFjG5qc13kykdJEVtoMAakZu\\\\\\\/Q4QL\\\\\\\/E2IBXqNqgaKazD9tBKjJFgipuNtYZFE3g01YhHiOzfUzWMpsFu9oJpC\\\\\\\/sRkNGOdPd0gJZFurfZ6n\\\\\\\/\\\\\\\/cVlPZL0CnL9FhxBowWKNnXSnMZbzffQKL5qZYmz9RLtrYxEXxABqb+lbeD\\\\\\\/J3KRBM503zIlffYhOcuMMHyxrwCF3tQHqbIenUxF3mtYIaekj5H3eM+23In+ovPLFin2trsrVGVPsqUKFZQ9su6uf9nQ+p\\\\\\\/bH0hidE+NzWrlex17+DfZ3rHXfzUUo9GgxInfx5Irbn70Fz2BClcWvYY9+UnBF8a1YrqFK1qdeDFTlbJEhwqayykjWphPYj6wq7gM+evI8lZpkp9P5t1Al8G\\\\\\\/TPOawpZAAUV3hC3ZATbpzUqq1LZDdh71zz8j+w5h3ZeWUSCHclBzPEPrFPcAN2IqIAIXPfT5Zgw0bmsiwszx8iDv0IG1ku1UQs\\\\\\\/Qh4O6VbWqKMAsm+C0Oh9ViGcqhUClXb2SgNaqrFqTTLao3tIjJSwCQrs3tROejD9Yo0OX3IdW9DT+iX1nk8OR1ZFyoFAAqBIN5mjXoyPg3AmkP3Hf\\\\\\\/RdN3XwzjuZVfbL0zr5d7Fhmlq2TrOcPjZQANGitW6einwBBAnSFw1ajo7fQb5O2TtY+4j+e7XDn0LZt30XuAZBI3YzJJR+D8e9nQRoyxRTx++Ra0D6O+o1oJbqKz+wiVQhI04ly5g3PZQXxPPDmcb1trjoeiyOXenwQZVWew11XITX6kdYLc3j7kn4scGYfaVY4d8OBtbT+waIi+AzAfBePJNy1eBDKCZfQXimhISp1jE86\\\\\\\/AzNeu0fSrfP3BZL\\\\\\\/Pf5dG9SH+\\\\\\\/t1KjLMChlnF6VGNyPAOmlDx8kUUO5Aacdv2vJwyWKJuBY+ZI9qbFgEm3K7SbS4DqpXbmt5ZYufIY7XGO0VcC2msmZxRuG84tAofkrVzMh9nMcx0t1bzFtvc63kMIAYOlZp6agleNoVvGTiSh1rS4J74Xr2ejCWdsqWUAeE9\\\\\\\/0tD1fSGRY5UxVEMq8du4OunGTbF+fFDYQUFM1QFdZm+7dQ61olH\\\\\\\/7i\\\\\\\/2u\\\\\\\/M333B5dJkT3GQRftL2U7QE7kEmfzKN1aGHeuekorQrU1yqTYRgrbitH1m354bPutKIuC6Ax8urAbJySLHXc+noh9ds5X5wTo6Y9dnYXsiw\\\\\\\/Qx1xzwKHpEZwu9yBFK9i5s8rlLO36LBJLbR\\\\\\\/dxyCtFvUns7qXmQoGGH4m48LNuyp10Q7++j5ym9cpi5UNOqmly4U+OiDzUwb1bDoxaBUTAuj80+X3x6AOYEcPDs\\\\\\\/S5qLx9qeS7Y+SF6olY+TZgi5CxfNukhYi5Pt63D6ihsCsF7vmOgnN\\\\\\\/3Y0KohCKpwwjhjS9t6QdAa\\\\\\\/A0uZK7cY0HzXKGF\\\\\\\/OPcXKV7+4ZJ0+Ctbf3p59JOuwDLaf9TpX8UbNsA6R4CwugeFGnUPjY3WzkozVUu155ctGvLRK\\\\\\\/l4b222o3lTHunyD4HfSQizWxa74Gi\\\\\\\/WOQNgdgwfNiG6hN2jKcSCAAwm9hzsLOeYZcv8nQPlTYCvRkNU8WyUxbNHMpUO\\\\\\\/DnMw4tmFqstcWRBU\\\\\\\/UuDsAenHzfoIziXChnIRVxV\\\\\\\/qcPje+5MXPIMdc2YZoNyLTTPRRTyc23GeRHF5fyK5lzwcYsFkYuxPOSWc5u5Zm4dA3w4lFQ2PfYrWmj93I9QA6MMH8xOFnsxqw7TJFI0jZxX643o6cI0w14vEJIjdwWwfr5eeZC7sazWlZYA77bV5Cs+h0zUzSLUhtCks74AydRAm6KWtgKZlzsV2TC98MtxVQ1qoBDnd\\\\\\\/HcQ62QF9UDC55DrJKcQR06dbaiRDq4rXiBQt2ITFVfEBnHU1OVRiGvovB0b\\\\\\\/cDkHHjoQIT0LYUYuigB8R3\\\\\\\/YChzoPU5NpHpqKYZyp2MeRO9He009qmeSksEK6ebbwYSM4Y6C+bfpQfBUwTANY6qK2aLTL0TonHDYBzHhuWxyqU0KmRvaimn6fPvT0m016FzULN9eTV5ZoCeDwALDN3kgvUhdY+HKf0fJpqYgwCjTa8OZtE7n\\\\\\\/6pHaM7L1AliebVgSovWC9sKegvT5s7ixvohY8WcDnyMJrnNqyYw9ooUHykdinYHUmoPWSoDoo8HnCRUFrLLM10bScvOMx6j0iT8HnddbLKfexJ3EmLgfNrRTrT+8NiBWk7\\\\\\\/ahAtpe8s5QN588RkBWFE8wG5iOaZIXK3ke2zswl0naaM2vxp8lvdqaipV7XcXQZF7BTtKvHsN5Ulvy70sA9nLlt13p0pWkgBbtdk4KFzo67CBS++TNCrue+gmkxsZjsskaSioiEpnfU8JGtIlICZiEbR0i4wJnotMTiI0FXCCTvXKFziS0neg4SbchCCR0v5lPqrz9ThyELcFrhoGPX5xk0N8N0fqbaNEwwBx4\\\\\\\/gWxQV4DtO7QPVbM4Uxp2zHHjjdqJKe0MMaaW40jh7eDcHnSW3xjhmgVnnS8udsYkoBXgnkK7if5XNLqtRHqSkKpAnwKhY1o5M7DYJOy9ZBMEmbVQins2Pvb3fISak9q\\\\\\\/uVaNxw+YyxlwsejdFO5LlR5R2WlDVD9ogGAvRB6vZhofHlAWA67TB\\\\\\\/NI6TZcZ4MIh9NOrHdjG5JYYSFOzXMn5xtH+zSE2ckho0yFqErjy5z7W\\\\\\\/2h+FIzzLwmvinmwda7vvQYEc8SueHN39\\\\\\\/QnSwYbCibk\\\\\\\/uVkdhjZDJK5OO4wf\\\\\\\/4ckfTTDH6TKsDTpIywTbHm480zSZkx2LAePekjvX+Pvnv7okOCu3dS3tkhGW+F3Tg8GWI8f6YxmRnJ4IJcXr\\\\\\\/VDO0lpNbRvevFBCBnVHO23GIGJO4ObwAv7iwKcPNpyeckV9elqsE7iKj+03xchXYfWxosczktqkuqEQAhePaGbm4fqYUu2VXgx2hEI2BPeh8EsQUUjLRUwz9DFZ4GsPEdF7XcCgJXLiaKrUP3RpsiuLQfsSNoDhl2RL+NvEYSO2vasTnEkyQq\\\\\\\/MgsNK5LPgSExnsJpcemTKAH6vf2aQa4hLJDfP4pW4oGxby1FUAys0S8Jkqoy\\\\\\\/nD0M2N9ESwDJErXUcpbfikoXR1jL7Ow0YRPmUgDSY\\\\\\\/qbDUaB0GT8iE\\\\\\\/nKURG+qz1qJ5HUm3XHwuYFZEDpyu\\\\\\\/hYvvXXta2Gzp77k7YoSZGh7InH8yr6s7ELzVWi9sUxUnuaUjGzYc+iRQpShrinH2rP9RR8ZKmyDPcsQ8T2QnsveESXowRdxqaX045ZR1TGSCx6S7eI7BsIB0Dne\\\\\\\/i04HtvlGrZlH77DW945bwXLM6ddZGbNqT1gr4l122smbNYoGl3dPbUHaiTrOrxFWH8pBp8dyK47UWvBxMXrcztHlbb9t1XVlWsrKfEVhtjM6Ypri6lTeBYhkTOqo4AoN+zSnPqzAtGM57U9OqaIpCmn0rxMPKa1KFDg1ND\\\\\\\/Pfr+sZkxkNtk2oOyjNuwSOndJo3SwkiPGLH+OeSOH\\\\\\\/YPWGNjj8nIGCfp6e3Ayil0MXKVK7tOSpLNp7TCz3ytSlL0PEpMgrcHJf953Q5EhIL+tGQC82CTn3Iys+h2yQX3AQGdZOuD7UBXUZGSyUpfftasRl2HQJ\\\\\\\/C6f0z07880gql\\\\\\\/DQhJK9aAMLpUg3DoRcZfMBM2C6bdu1OxbzS4VCi3Aypd2VazaNOy4cc+59\\\\\\\/ZD2uitD1TtJep9xFpSKZQ+MKnbe47ryvsrlBvaBmKzXQoYhTKRk9Q7G36q\\\\\\\/eL7FsQK6RDSE50YODpZVVglfhjVdcuWpOb07mCcR8p3pVm0tW3Lip8lkBfvHR4PHrCajRC\\\\\\\/Ojzi0ejbnTNJwzuNMqxbDTVSI2HSq+R79blSVHFMOi\\\\\\\/ONs23w1VVAEvDM3V7Eu9N7K6oVM7dS1BKf8nMn+qBZla8Q4rg8u6vTOiqGRPUr4WtvLgGTmPPwVrLVbqkHOxUBCs1QpsIMT0YxSQRqRXCQztmAVskQsGFvsnO2g6Z5KgeIjkM0M99S2MDXxJf0Z+5K3IJPEQUC0C\\\\\\\/D329JJlEw\\\\\\\/oBfdSMiEANWhqZpEzXR9bgk57Dw5GjVoD2CUGxe+E1KWKBH4YTI8o4HyFWtqpo4OEIqWcu0+AXsBdukOBVvXOZ4phPPuXIeSIcBpnC84w++q2ovEN6r+H0kcCYYCW2QPe4bOF5PN23q8I7lN8Hqm1jBfR9kHYlb+yHvsvKhVOZ3x2bQ\\\\\\\/0ft8PhOOlACXadjokU6m4jBeGK3mbXR60FlRSBiJQtqKzdK+j+YDoaUMSaTvKaTjhcdIwr8hS4AjyoFmQ+hBa073ZovdknbxWr3MScQp5CYrN4KSeMccBfqdBj7WcH\\\\\\\/rry8iIOCcLVfPMGiv9NzXR3BMazakn4HmgCo8j8XV2XgAcFgR1FswdGI1piKR84m3mjtwPeCjPZuVBEk07WRUR6Jo3XIhOFIxjBONtl2nTFunltQKNf6L7iNIt\\\\\\\/eq5J9fdzcV7kLIEQZVVcMUKH0\\\\\\\/5HimhlOcV7areH+rKScGEDeDxMcXVpA4ITNjWvfN+nI41FTkDyMKixD3F6KDwfXCpdm2gXDBb2i0om\\\\\\\/nsfXxGOfPdggXnKfrq8f5mJRFfnjiKAskBMTJbMPgqUW7OnHCo1sfbd1pDDj9SxuNTZiwnmV2bNpLpv3VolfLNTwzevSSuLe7ur\\\\\\\/cnKk05I9hTZW8Kg2Yr4fdsLEWcmWo8b5zM1DhwqHdhJqaIzMNmEvjGSBvfuuUz+ZQVgqv9j+P3pNcGH\\\\\\\/7kx72KspocGTWsiRu20zKmF3rGe5Ijwrz\\\\\\\/BHa1o4J93rQ4v5j2ZJCeKwFLrVi+KOpyy58HARt+FjPTel9mHxEnwASRNz6nJlCnkkZ7OwbMaWBnLikY36E5hFICKOD1kyZgAydJdS0O\\\\\\\/d+KJ1ZvB\\\\\\\/QafcpiY9pIKvvgXCfsUgv\\\\\\\/J2FxwNAd8Xl7+4Q9a3HEB\\\\\\\/RkHYUAu6hjYqkYLlNr8bKIe0Psf9ums5f2GaZKHEpH7T16C3HsWFgLrthfhofGBE70+OGwe8ewah1ihHaNcH7xHlA7NuP61uZ+wDcV5RtIarartiprG8fjc7HmdOETnutBZ6ZAz6QNAzpZlHgyLAHDjVJ09tm8hivsgMMWnJeHHiceNJvIhEIfMZ43T9STDzRF2uX1R1Z9xkf6jOIAiNSSoXD+pqPfvFSZtGpX0ImUKmCgEk\\\\\\\/Vg2jY1\\\\\\\/ultEoGqqByN638JMWUhiYX13r9bxiKIvt+SV\\\\\\\/jR4A09Eh4oba9ic1DOfRIa4zJ0\\\\\\\/9Z9wa0+cdXaXlunOy\\\\\\\/\\\\\\\/3wZwk4\\\\\\\/Jn8kX1T8QBp5WL7YnH21L7P2nA2uR1tKbjLIceFyjHvzZeV5icznrKMe+SlUgsyiKb23dCFUe1OlIYln3fRDz88DTRLlgmsfwZDsDNXODgTpiGKs+1y0pbXdNdHLVIruz6KCoIhDPtgiFzqfEMypOXpi2kmMdsX3Asza0VCYv3Hk2vG73TIfst5o9vp\\\\\\\/9brYb+UZ48OzwEl0Ex4GF7zNWxSgVnUOhuuSY0uofRJaaFmg2kRty20\\\\\\\/nJvxHxGyMNW6T+DjC6RDruuExMpER2dPudK0EO7TapdONbFxq5XFY0FF\\\\\\\/h4x1UImkju6hqfKoByzTJnSdgAx0ION9D84yKaXYSLCzsM+SuTa6qEnamd8yK4qUe+tq7otLoEG\\\\\\\/CIhTV9E6mrmWbI8ogJ6u59nt2AwhbcEp+\\\\\\\/yQHmsjh85Qxd62hhOtBbCcZB2FnMTDe15DKlkGScJcNQSPDUfhKAno3oSU35Ws44hh2Nrfm48N2PlYopdp8Mc4cnocaBQ2gLaIV4ABoKiINo7xK0xH\\\\\\\/1XG78krcHxzOhQgv4wbW0BDpMXF2igjYdyCJbnK90tnJKSkXpIaMXTUJXrD91DwmGf7QGHThHW6SADUz1xQGUUh3pRZ3yXwFE5viSNBbjsRjbIY02fsokCAgQohdahwN52P8TkpizBzy4z8FP+3Yx6I7hhRSmZoPg1I4AdtGekvxYV8ujBY1cAR8qjyU0YNMVEtKr6sX7jDpydYYZzP+pn6lwQe1rNz288wSNpvzWKRmub8Y\\\\\\\/jH2CLx079hWsQ2AMgyvCanpHfkU8IRZlVYWdNbFXO6od8zNKS8aG+DDWyEQaZq1lVjdT5DPZfE\\\\\\\/1HucwcxQYTlC+Z3uFDLjhkk8w61MhWVcGN9eScHXWC7e7gOcsdBLvBlQFV19hLkgp5INhFOwJXpkniXt5+wod6Je1dXQyLWziK4o+oOYC+AFEONwbcBH5tMvo5ppgz4Lcavi9eyTz85U5\\\\\\\/tWL798s64fKqzgPdBHuWE7ZrZanlA4jSwMrKxaBXEL6dTqLiL5A1UHp4CO9cllPuLGYn\\\\\\\/5Yn8OTGWgyMZCl7re8y34Y2mihfvdmhlneuA3evRLKJ5Sx2uqma7keznzOBR3Qf+LpIj3saG3FzsgE1m845ABi+FGr0kUm4BFk+4XkrByk2GRYYibke500Z8L2nlGkThnCu7MKfRakaBz0mYPj4TXHXQX8bd\\\\\\\/+OYa1ZQe4ou5HIG2+JWGylvg1rT2rNs7+Fh9GmcPIjxomRhA5VfE7OkppasfBXz+CONptCZ3U+lvtQz7jtnGavTIwN22AXCKC8GAIQC2xk6Llljix8o2wHUjdeCXDLgiP7bQZTJ0xV7FAxmJF7BtXwuvLFqd7fPN98J3yt+5real0RIwHqxE1O6tPPxe7nHYHf7x4XzfattYlAupfIW8D3Tuyr5ejiU4yJT3w9V2I2WPHCQXKYZyEapMgWY76G3jxRiwWJe553gwhEDhYgYd3VU\\\\\\\/7YjnasPVLGTCex7HPMwS4nLA+WgRXhRCXTaQLmut5Jerz+owhj\\\\\\\/lGUWHf1MpkYYH1ZU7PnXrRX6PPVlc6+Venmd\\\\\\\/E1Mmbk8Q+94fMEOPjwP86h3C12EjE4tMtYZbdxy+gL2C+ItsXOCp86YiG\\\\\\\/yAVbJ+SP5Xe38ZwQPOEpTyxghw3FCaiuoH8IWIZaVrwTAI4xqN88nv4KMAljf1rsKJGrbTcVuJNbDLHj6lu4zWD744tfvrMAa+Aq3x3HKkxk4YKAl1VaS0hEiFtmfHqqH05G3vyMqzP3F80Fb7T1dz3lZY0+sKMGSVFKz4DjeHvdnvuDW7g+OmbPazK9PzIDao1xxn5afIuGFaVFQNCDwheCFSNeP5sWjpOqJuiUGpi0ThYlarksN\\\\\\\/So8uwK1YSrKnZfcsu7muuEF\\\\\\\/\\\\\\\/ITfIoGXIpsl8SY9V04ygQz9yUF1LoQnMEQGnjF8KtnjQBbooo\\\\\\\/vJSeh5Z4BbOfxtecGymkBqnDaIw9Aqq7EZBIFyZoHlrR2aicewR1gx+tiF2pfTE1UDAdfOD2OCgIumfBcEGfp6X0JJWJm8PGAiYLWQ7MmOLsW2E2hkiVP7Wu3UK+gfJQ4SjWud5bxOF4PiMXhC1MFoNm1Odj+pv5isdJpusfnyfYTeR3f90CwWZLfXwnD3pGiIYB2MMJFuqn65cM67umrSuOmT9mUViZiu8bPuW6j53lbJUtmiD2QvRAOFSytlUxvoFZ+a\\\\\\\/ROTVSuRBLDfHoKH+LewzrkxEFKFfpKrcebUOKBvoS1wOU+lIPLyXzNzV3MB3TQoy3ATB\\\\\\\/miNItUgiBfkjCW33MbfRcf+W3fw+E6dUtwmwShPe14m6eB5qYdABV\\\\\\\/i1uKU41bc+D4rgphRARJpHrsDHjuEeI5RA4q17rJfQ4keA8gUpaVBha1V7fCUgGyanUdiryuKCt258nQ9mSWwDRZZQA2NjslBvWxrHgVR8JmZAI5Uk1tv4i0itbntBQuK\\\\\\\/NwQOCrwkR5tTPyAPfCmyPYc\\\\\\\/GEnUF26y2R3AbfIXWLZeX9CujYH50+TraM0mY0JMwGtKMXLJxFK0vV06Zq3Oe9V+FGxh\\\\\\\/5qQiqih\\\\\\\/8svChiAoQWxtk+nIvhtC2WFS7Mp0aepBcyR3+esNgIIqhNDNTFgQ6mG3wEoP77THOyyYIasb4gQwox0gMWtbXbn6jUb2qEb455X7QwCEH6k7UYE69lhSr3ribfFkUPrWJ0EsVDx+jtoncvr0lUaTq0QXfNQPwOYl+LUIcSjwewQ8AO23oYwA\\\\\\\/xSI\\\\\\\/sWR7KsJ6EijvZtzIjCCF12iie3\\\\\\\/0mI5XrHKXoWqfMFc3Y87Qvcvfk7rTp0Be+7g7H7UzJsMmXA1X2Fv4sZmMervqYp47AD+nmS3dFmf5hARE2LLyhYU9kZs3c3R9svlNjQvuqLzniff3UjpoBEG5mLBZiuVfCfR15UD63BsbtMu8K+HccK2fX+Z\\\\\\\/dsYmigTBVJcNsMG2qiGYGF0HlxbtH7xtY5Ah3nSpZE8iz9R4jj3V+OSQH0e6o6PEx55xxgU6rNuU3YZrD7YEHzVKBE0fiADl8ObLSdFA\\\\\\\/EPtrbS\\\\\\\/ANuuNVdnNyAg9MBgteOdgnhzSJJmEfS3sCDsY0tkp53BNW5NekLqrKbiFZuJRC7ddSYtj2fAi78LG+1CPS55z2AzmNMgqJmpS3ltos+BOIYEuZ+++\\\\\\\/gly\\\\\\\/RanV7YrWed8eqAB+6HNzMDLRzNZgv2cpM8NDg2x1d\\\\\\\/jtL2dZsgXREr+rT5MWhk9+KFdavwTxCskOSPeh7WYZ0zn9lRkJmqT97HAVhFbQlCcmutqTwFHzrZwItR64abHq7oYr61okqgUmMajDUOT5\\\\\\\/KD8i5pxgnh\\\\\\\/U8mC0KU1\\\\\\\/igg3OS4o06AjiI1eMhtD8u8CyRUmwsdFMZ+Zr8kMp\\\\\\\/PMfWk8bVjOGv\\\\\\\/5XbTrTR3r9G1W6bEnttHy6hIS0y3I4oPR4Db6PwJIeOAjP6QU9sFkjtBCLENXBlssFxhnhA9jLO7iRzh9o5CGdGGDP\\\\\\\/3H7LzNOzuLfhy8u1M9nCKmliuaxYrGjQybomzzGtsuBHyg4w8Yw1lhvDFakeRn3SJahdv7+jTeeXWWg7\\\\\\\/+YUyvSk\\\\\\\/e5FGPHFR\\\\\\\/Oyy3NahKw3d8+ScMqX5r5mulZ+kO1xOEmGkr+lhof\\\\\\\/GvGRjV1YNqmzhGeFOafBMpjg4V3JAZss+XpC6VYaeXSflbMWHIR5ZW1tBHOsDulkuMY03ZSHrHcnTXEcmv7a80itGc8bYQVA5Te07b2FQH7SJSNoMxZ0y6N2Ym12hfVfMr7bSW8bvM7\\\\\\\/HOi5Ct9Zp7Bz4t4qToUTIVVRBOvTzCPmqmvUNdZLr9r\\\\\\\/cbkDnNJdRUAJDazfcqPzYNXjOkpcnSTijqYX4rbIb3GwKTlWr+FCJMCDqmb96PKDgDURp0ZvSHOwKhylmsod+r4nsecoHukuIBhvMDfOkjYdG2kguRgyBigQaj6Jh5MQHZp5svOUl2Qh44lozzbWStVPE4hZ3fzdBiJTrlpslkLPVU\\\\\\\/e7M\\\\\\\/XG5Kmeo4sXj5gnTzyErPCQmfLiIenq4JwiiZvi3IfZBZqMejdplNoF7rixc\\\\\\\/1od09rONExN21OPB5BHrSc3\\\\\\\/0dBr1th3UfBThzEyhVgQcQQgtzrnomNCIwk+AzhxH460nrlN+I1t6Iws2yI+fOjs5oJvVD2pj5v2\\\\\\\/CZchPZvAIXTEaFvEo3VWYOPU03LSjp7CryikkikSV+ROTjZYvL0o2t+KptX8d\\\\\\\/2DhqfPoqDw1aPcBmucTgfNEVHi7fxdsto7V1NMLl2Fi3USbhFAynUbcjisiE3g0ZSFKR9lHSe0dLU2sxV5x+DstlaUhE6wQ5uwgP6kzC8b6fbKrAiPSYru0BBZ7t+7EXs9+wKZvuV6aoPjqZyxRs5r3f+PeZcOa88vboG7d6UJ\\\\\\\/80RjJasDuVyGPJzgGVOpWrIIEY40iP050Hj6uon4ZL2bC0+boKsYSA6NqIEJXfSy7hoPalP40DC+jTLGrLQcTLY5fKEWA2ItE0dGmeji3HR33oBibK75iHSsQkFADgTWlwF0y5hoQ9cQrVJhaHiF\\\\\\\/jprqa6toWrHyGAs3Ir4EIEnF40CR2eNa9rylOQqBNJPR+12zLNrCpWmOTmmSoh\\\\\\\/Z4d9uev4ZzkdFc4eQtTLVLourmTRtGSzhvny1SUPg1LQvTvwGb0aZMkWwsS6O3m+yrDFQHaPM0cmAqqs+40KXJQEB\\\\\\\/eY\\\\\\\/5GQvvsj+ig4Qrw4ANv\\\\\\\/Gh1OZ9tInnDdks8GIYxwrWQYlj7B1o5FzDXHQoWC\\\\\\\/jc5CJYGCDhw9ThXP5ovxvKZYZ2qVhnKxFKVL4T9zPN49qK4zVB2mn43zQlVSO98m4dIPL5r+KK4GlKv7pTYF2gUoJwWTHWaUGiDvsr098NUG6PZbK2HedWreH1whzLtFY3stPQpaEaAFbke8h32VMKQjgPAipk5iZ26+c6m01wqBqGctfGDnt5KLBt5\\\\\\\/2mv2b7Yf1r4PoyvBEKvdX+dnklENz6r\\\\\\\/AIQsQqs6S+g98wGIDIWXtN6EvmdHCaY7ixdQ3YmvfPUKvSKuVtyQRmwogdpdePAqSBpCpkDF7m3wi3xsnkZKBSF2iA5C0\\\\\\\/eZJTAk3dUzeJgwlQS\\\\\\\/kvZjgqSxYSN89yp5IJEY\\\\\\\/ZS+MJuEM\\\\\\\/G6+3Z6GERlAxwo\\\\\\\/o4LxPBTmT1ZyziqYPrzhbTXLbfuuBR9z28TDm41PtJE7hGnTO24nCeMkj\\\\\\\/rBZInbeV\\\\\\\/0zVCVdJ4DP0bUTkawzi5o\\\\\\\/RWf9Y58NRoQ9cr7BjfBunx0uD2nwXI\\\\\\\/0u6Bi3d\\\\\\\/963aa5y6sInfpKhR0Pvpxxr4nGl5IQYYxlvwNYQbNX1rwxDY8BZIscdzokFpZVKSDL03S8qz8pfrZk8s78mMPPPBLpbF91ht9rmvMhJtyft7PLEMhIABFytvPgBp6MKDad7dqjl9Bg4zoVbscwI2xF415ODMUhS+JMnoWiUTfsh6hMKfPm02OSGtRIZfepAnGgQwfAD9NCVSYBiVIFEcNNbE83fIx67lFuHYXkcsrzyKzTHoIagxXb68JBmlOU2sokj3wZuO\\\\\\\/alZKZm\\\\\\\/BE91DMtcz9lk4AC5xUxi7dEONKu3hxEMSakpXMSH6SiM\\\\\\\/T4Tz3y64\\\\\\\/But1x4T2q09J6Cvk3Z1o1mu4Ltb9rZ4CjAzL1lzBsqkZMafhDEvGUTfWgNmUoMDZU0Prv7rxUjjMhkegbIRUpRf9e5DakVPQdoxC\\\\\\\/bQCFDOxi+CRBmynEEcWYZLZle4RmSvpjPAV5t9DppWIIZE2SJAUVJYXFwGmIeiFfm9jBfJqggEjmzx9xT2n7GVDHhNOH+tCw3N6WIrX9Sk8hk84IttLbY0derZFIf43NE8raf9i0bpocpqTwc56XIossO6YwQ3FYdqdjvDV5zASTfeAeXbjqG5lRgkTtSX9HfFwZ8L7jnlbbwwEW6F\\\\\\\/i5QFvipaypBdjFkehV95+kp7V307DV+2EWoxdeIY3SLsTnSdz6FPXj+QgB6o+F3fuoCedytC2DRsK52WW+I8XAsefYr1pak9mlYVDHowO1xRz1F0ig2fpcb\\\\\\\/7zN106Ekfgr4cWfwu1YT8e8nBebw0jg8OzcAmWJSahVqRZuzZvh6dQXO3MJO2EQrmlaHlNImIzm7RB1ItoprFtO0OA4EfwaHWhHaxLYMgKJ5eBgwhcK+KICV+EFXVUJlrOh9wUfES09yzPmWlTs1aVbvqU71U+Oqzsz2SrXWIjUxJHwzaNAz6kE7qUhkI6kS7QwNSDnEv+jXw5GgSqU5RhWIZ\\\\\\\/sXX5p7F2iArM7RjWhwZUv1cTJheNAyEIil5AcTF5\\\\\\\/CaqRCQT2ZtaL82m\\\\\\\/zHNZQIYZEtq\\\\\\\/\\\\\\\/dZD5+JdNe5eu2fTyBCOSKgyjfZfoMn9tDczJaamUhTmWNJMFhDSBMuCODUOdw12rWpPYdgU1\\\\\\\/Hmjj4n\\\\\\\/N0HMUMniO0V5piuqg5CJvBWWmBniXmLHiXv4ZBXxCv7sjdQkkyYbVd83PRGCsY96rk4KmfOSow1eK+EklvtX9Ypx1eFOldxi1q+9fqte44gDH95iP0Ubhk4Z1j9bq0ltF698XRZFiWWf0zz\\\\\\\/wzzDUVmO3sXwlvDLA9FrzdgXs+2eJehz\\\\\\\/1jUVTQzeLyyTGTVtoZA\\\\\\\/ILt8AfZas5LNZxxQ2QZUKBNTquvfVCWnfsGFd8E02Bec98pApeQriKJFN5wF9Dm0N+1NEZUV6e+sw1eNZ4wzcRXkpXowWntMfHifAoEYgsyN5DPfO4wJtrap\\\\\\\/5D5HUwA9pC5oULQxGkosNkeE0WZTVEiLF8EpRYFJF1ocP6J63HX1AysSKImIUA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022te4TlkdkPEVCWqHw6kDhNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022686056db96c0b631372e18a2312be9a18498e541636262dcd2949e7d8f836e27\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wibM3nmyQQNWFpL\\\\\\\/xGGwEYTNEcg+JPiJjzID9ZE3krFay\\\\\\\/31\\\\\\\/W1Avehwn4lBDn8syVk\\\\\\\/KBviilfJtgH0\\\\\\\/WiGmD411nP0kdpf4c6Lt6GBZsBhdXx5LBCidjQcxSqrXdZuQIqqvX41fmgIeToK5MSsxV4oBkXpN5iAZeu7xcIur\\\\\\\/Ji13CXONUQPuv0j+QbJfuzd6x5GONeKobaJgJyM\\\\\\\/9AhQq5JzqHgnIVUe1paITYavSVZR+MrIUqGdOjJuJPyMuIty6isPe\\\\\\\/7HLuxqosOM+HEKts50cJXO705lLqI+dWYT7buk87cqo+IRnWrXnLNJWLxwTsXu0KgkhnjunqZ3CGDqD6uMSAgtvcRVEavQPAKc9zvisaTr2ljV8picKYGoSbcC4cZp\\\\\\\/BZT3h7lNs8TwNwZY\\\\\\\/7OMAH6PIPA4DXG8aGf9xgwQRnScEuFWW2qq2SNNkX4bARyludj3OUPgOdfr7kM\\\\\\\/SnQfw3Ss1YGvhvuQx4ms+fzqBAlA1KTdE9\\\\\\\/tyH3pJwRsA7CmJJ3q8oY\\\\\\\/wjThecGTQI4B9bGPl62IF9\\\\\\\/hNBH95d7jJCZ5Fr3UEOapRjLe9mhOP99ubNd0sh42vhwR46mVWzrw+JVdUwzL5Ms0qLridHaxa2bYe9dnVXjs+deWE7Vw4KMe4r48E1SAfHJsreyE6cc8ZwzaZdEu+N1eCD49Wk5eRI\\\\\\\/rHeiibN\\\\\\\/w6skjCOsTiWcZ2tetjKhE1azmgPg+EwO9TpZScHkwr847AR8BWZcrj5S8ppRUouluOIuBz0qcHtg5dmhehHB1cQj5Y1XMymuMG0dW9JnddpYJJckGCTQToHsgCc3KlRb0SFTl99ytFH0HKmqpVP7El2o16ymKRukk\\\\\\\/CIep6BmU9SRiZ7ET+UmTOTrcOkLkH6ebuC3i1y\\\\\\\/awdQUzhCW2oedqte65vYAwHx9l5O4jysin42vPMc7r\\\\\\\/my7Kq8nr6+0O+7ZerPlXio1y\\\\\\\/N\\\\\\\/Qa2jpNjKYRGTz1di3FAduPQviPy1H\\\\\\\/zFK++BXAZFkMAjY9+dJ50T+FGN+NDuOHhgnHilypx8DnQX+ZyXI\\\\\\\/Q6WwlEZKmUSIBBh5OxGiDmwkxfZNfxZE3w7E+9UY81ShIr4gbEq6pqhg14GDt8n7glYmC5VqV3WQjApVzKwGb5SVAva+EYuiqUU4SUl9Cy4DVT3LoQ\\\\\\\/mYNlPJ+xvVdIjb3ngsxqqblx0CJ7IuinVeUbhtEeAV8VLF4\\\\\\\/YO+P35xwhkVljzWsg4qgmiy+jKSFe98Ec5TeuMeqOZsyR8GRNMfISTDADY09AM0uNEneFTdlSs490cxj+bx+LfXW9DiYzNuZkMLtRU5+Y9X1PPKN6FlRvewBG5eCLIoNGmpMLKcsFayvSnBjy+xEbnwvogHtsVMg7nzNZlUd+ok2NX2jD7meCG8hov1kLJ4Cq4TVFEY4fH9MED\\\\\\\/b2PCtyVLt4+JuQUroicUhq88URtyJyk6XKfvZTyD7RPB2vQ3gY8Gs5e7CuL2NoGCDwppnrbajR1N9eF4nXbOhqC36wXfzuHfKobIThhCfdJaWi3x5VZZuUq+JHRKBidIPG\\\\\\\/scqwI86dI4fsQxxf\\\\\\\/jvBYcVt0\\\\\\\/Y51lA9cJ0OhOyN4Nfu43D+ivgYXdAxTSlBIqS9Kbvw8ZJcSbC6oBer2KSX1OhZXRhsuOg5BCx4pRyopd2v1lHXQoqr0TDQvEar3K0pGbcs7xgZATiIxqnJZHVOEVpSNgSC4BCqKrEKy6\\\\\\\/YzJgZ0KrebwSiBeGjjfX+G9tAhklOIh0xhj\\\\\\\/aHCXwefRuGHI5esVnm8+84UZSUVEFeJzbqTbU6ELnmhvQTbM1KEM1l2M42wB0eU08UOnyh4PpJb84qz+XYbS1QO\\\\\\\/8NWIWJMDrfyMF2Ds9tDi+WWFtpsRn4niP14VBKFIgBZhQdtiVHCYdb7JKCXWXCHLqDK5nu21664Lrq3wDijIC8VEqnCVL\\\\\\\/Zg00ZNVTFWCqdyazj8WZ6sTvXFGvxlkH9cLR\\\\\\\/PsrcpYtAENi3HRWQ\\\\\\\/puUIZr6e7C4sZYQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BCUxiDkt2zc9oE0exfpeLg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a25e2c82f01b1bc3daccf08ec3f82d47f7b079f126b30d088d3485096d66017d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460296 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2717" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-p8CkHxN9b7sEShdeKZvsWUe0I2Xo1EAkQMYh-AzCxMo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2717" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8Id4M58y_ePcAE9frfeosO8u9GZI8zfV2dHXDX5MlgA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2707" title="Microsporidia Intracellular Development Relies on Myc Interaction Network Transcription Factors in the Host" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2725" title="A Genomic Bayesian Multi-trait and Multi-environment Model" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460296" id="node460296" data-pisa="ggg;6/9/2717" data-pisa-master="ggg;g3.116.031302" data-apath="/ggg/6/9/2717.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lianguang Shang</span>, <span class="highwire-citation-author" data-delta="1">Yumei Wang</span>, <span class="highwire-citation-author" data-delta="2">Xiaocui Wang</span>, <span class="highwire-citation-author" data-delta="3">Fang Liu</span>, <span class="highwire-citation-author" data-delta="4">Abdugheni Abduweli</span>, <span class="highwire-citation-author" data-delta="5">Shihu Cai</span>, <span class="highwire-citation-author" data-delta="6">Yuhua Li</span>, <span class="highwire-citation-author" data-delta="7">Lingling Ma</span>, <span class="highwire-citation-author" data-delta="8">Kunbo Wang</span> and <span class="highwire-citation-author" data-delta="9">Jinping Hua</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2717-2724;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031302 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460296" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lianguang Shang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lianguang%2BShang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALianguang%2BShang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yumei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Cash Crops, Hubei Academy of Agricultural Sciences, Wuhan 430064, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yumei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYumei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiaocui Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaocui%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaocui%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Cotton Research, Chinese Academy of Agricultural Sciences/State Key Laboratory of Cotton Biology, Anyang 455000, Henan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Abdugheni Abduweli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abdugheni%2BAbduweli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abduweli%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbdugheni%2BAbduweli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Shihu Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shihu%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShihu%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yuhua Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuhua%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuhua%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lingling Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lingling%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALingling%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kunbo Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Cotton Research, Chinese Academy of Agricultural Sciences/State Key Laboratory of Cotton Biology, Anyang 455000, Henan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kunbo%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKunbo%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jinping Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinping%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJinping%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jinping_hua@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jinping_hua@cau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/9/2717" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460296" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2717.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460296" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2717.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460296" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2717.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460296" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/9/2717.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460296" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Cotton fiber, a raw natural fiber material, is widely used in the textile industry. Understanding the genetic mechanism of fiber traits is helpful for fiber quality improvement. In the present study, the genetic basis of fiber quality traits was explored using two recombinant inbred lines (RILs) and corresponding backcross (BC) populations under multiple environments in Upland cotton based on marker analysis. In backcross populations, no significant correlation was observed between marker heterozygosity and fiber quality performance and it suggested that heterozygosity was not always necessarily advantageous for the high fiber quality. In two hybrids, 111 quantitative trait loci (QTL) for fiber quality were detected using composite interval mapping, in which 62 new stable QTL were simultaneously identified in more than one environment or population. QTL detected at the single-locus level mainly showed additive effect. In addition, a total of 286 digenic interactions (E-QTL) and their environmental interactions [QTL × environment interactions (QEs)] were detected for fiber quality traits by inclusive composite interval mapping. QE effects should be considered in molecular marker-assisted selection breeding. On average, the E-QTL explained a larger proportion of the phenotypic variation than the main-effect QTL did. It is concluded that the additive effect of single-locus and epistasis with few detectable main effects play an important role in controlling fiber quality traits in Upland cotton.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/fiber-quality" class="hw-term hw-article-keyword hw-article-keyword-fiber-quality" rel="nofollow">fiber quality</a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/recombinant-inbred-line" class="hw-term hw-article-keyword hw-article-keyword-recombinant-inbred-line" rel="nofollow">recombinant inbred line</a></li><li class="kwd"><a href="/keyword/backcross-population" class="hw-term hw-article-keyword hw-article-keyword-backcross-population" rel="nofollow">backcross population</a></li><li class="kwd"><a href="/keyword/upland-cotton" class="hw-term hw-article-keyword hw-article-keyword-upland-cotton" rel="nofollow">Upland cotton</a></li></ul><p id="p-3">Cotton is an important cash crop providing most of the natural fiber for the textile industry. Upland cotton (<em>Gossypium hirsutum</em> L.) accounts for 95% of the total production of cotton fiber in the world (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chen <em>et al.</em> 2007</a>). Cotton fiber is widely employed in the textile industry. With the development of spinning technology, the improvement of fiber quality is becoming highly crucial in Upland cotton (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Kohel <em>et al.</em> 2001</a>). However, a major problem for cotton breeding is that fiber quality has a negative genetic correlation with cotton yield. The advent of molecular markers made it possible for cotton breeders to improve yield and fiber quality traits in cotton (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Paterson <em>et al.</em> 1988</a>). Previous studies have shown that fiber quality traits were quantitative traits and were affected by genetic background and the specific growing environment. These studies also observed that epistatic effects and genotype × environment (GE) interaction effects played an important role in the genetic basis of fiber traits (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Paterson <em>et al.</em> 2003</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Shen <em>et al.</em> 2006</a>). Understanding of the genetic basis of both yield and fiber quality traits is vital for cotton breeding.</p><p id="p-4">Detection of stable quantitative trait loci (QTL) for fiber quality is essential for developing cotton cultivars with superior fiber quality using molecular marker-assisted selection (MAS) strategy. A large number of studies on mapping main agronomic characters can be found in the CottonGen database (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Yu <em>et al.</em> 2014</a>). Hundreds of QTL conditioning fiber traits were obtained based on intraspecific genetic populations in Upland cotton (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Said <em>et al.</em> 2015</a>). These fiber QTL identified using low density genetic maps have very limited value in MAS programs. Fortunately, publically available sequences of the D<sub>5</sub> genome of <em>G. raimondii</em>, A<sub>2</sub> genome of <em>G. arboreum</em>, AD<sub>1</sub> genome of <em>G. hirsutum</em>, and AD<sub>2</sub> genome of <em>G. barbadense</em> provide an opportunity to improve the density of intraspecific genetic maps (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Paterson <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">F. Li <em>et al.</em> 2014</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2015</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Yuan <em>et al.</em> 2015</a>; T. <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zhang <em>et al.</em> 2015</a>). Recently, a population of 178 recombinant inbred lines (RILs) was developed from a cross between <em>G. hirsutum</em>. acc ‘DH962’ and <em>G. hirsutum</em>. cv ‘Jimian5′. A total of 644 polymorphic loci were employed to construct a genetic linkage map, and a total of 64 QTL associated with fiber qualities were identified in seven environments (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Wang <em>et al.</em> 2015a</a>). <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Z. Zhang <em>et al.</em> (2015)</a> developed a population of 196 RILs from a cross between ‘0-153’ and ‘sGK9708’ and detected a total of 37 QTL for fiber quality traits on chromosome 25, of which 17 were stably identified under at least two environments. <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Wang <em>et al.</em> (2015b)</a> constructed an intraspecific genetic map of Upland cotton containing 1013 loci by developing markers using parental restriction site-associated DNA sequencing. And 27 new QTL for yield and fiber quality were identified, suggesting that the efficiency of QTL identification is greatly improved by the increase in genetic map density. <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Islam <em>et al.</em> (2016)</a> validated three QTL regions controlling three fiber quality traits and further fine-mapped with 27 new single nucleotide polymorphism markers. The limitations of traditional breeding can be alleviated using an Upland cotton intraspecific high density genetic map to identify fiber QTL under multiple environments and further to conduct molecular marker-assisted breeding.</p><p id="p-5">In our previous studies, two RIL populations and two corresponding backcross populations were applied to elucidate the genetic basis of oil content, seed index, and yield heterosis in Upland cotton (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Shang <em>et al.</em> 2016a</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">b</a>). In our laboratory, 39 QTL for fiber quality were identified in three generations (four environments) of F<sub>2</sub>, F<sub>2: 3</sub> and F<sub>2: 4</sub> populations derived from cross ‘GX1135’ × ‘GX100-2’ (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Liang <em>et al.</em> 2013</a>). Recently, 20 QTL associated with fiber quality traits were detected using 581 loci and a separate RIL population derived from a cross ‘GX1135’ × ‘GX100-2’. Among 20 QTL, four QTL were again detected, verifying the previous results in F<sub>2</sub>, F<sub>2:3</sub>, and F<sub>2:4</sub> populations (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Shang <em>et al.</em> 2015a</a>). In the present research, two RIL populations and their backcross progeny derived from two hybrids were used to further detect the stable QTL for fiber quality traits in Upland cotton.</p><p id="p-6">Previous reported results showed that most of the QTL controlling fiber quality traits were identified only in single mapping populations, such as F<sub>2: 3</sub> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Mei <em>et al.</em> 2004</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Liang <em>et al.</em> 2013</a>) and RIL (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Wang <em>et al.</em> 2015a</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Shang <em>et al.</em> 2015a</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Z. Zhang <em>et al.</em> 2015</a>). There are few reports available of fiber quality QTL using multiple genetic populations in multiple environments. The objective of the present study is to detect stable QTL and conduct the genetic analysis of fiber quality traits by the single-locus and two-locus analysis using two RIL populations and their backcross progeny in Upland cotton. This study will provide new insights into the genetic basis of fiber quality and make contributions to improve fiber quality in Upland cotton.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and population construction</h3><p id="p-7">As described previously (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Shang <em>et al.</em> 2016a</a>,<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">b</a>), two hybrids were used in the present research. One is ‘Xinza 1’ (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Liang <em>et al.</em> 2013</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">2015</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Shang <em>et al.</em> 2015a</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">b</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">2016c</a>; hereinafter referred to as the ’XZ hybrid’), derived from ‘GX1135’ × ‘GX100-2’. The other one has a common female parent with ‘Xinza 1’, derived from ‘GX1135’ × ‘VGX100-2’ (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Shang <em>et al.</em> 2016a</a>,<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">b</a>; hereinafter referred to as the ‘XZV hybrid’).</p><p id="p-8">In total, four populations were employed: (1) the RIL population of the XZ hybrid; (2) another RIL population (RILV) from the XZV hybrid; (3) a backcross population (BC) of the XZ hybrid (the BC population included 177 BCF<sub>1</sub> hybrids, and each BCF<sub>1</sub> hybrid was from a cross where one F<sub>9</sub> RIL was used as the female parent and the common parent GX1135 was used as the male parent, respectively); and (4) another backcross population (BCV) of the XZV hybrid. One hundred and eighty BCVF<sub>1</sub> hybrids were developed from crosses between RILs from the F<sub>9</sub> RILV population used as the female parent and the common parent GX1135 used as the male parent, respectively (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Shang <em>et al.</em> 2016a</a>,<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">b</a>).</p><p id="p-9">For ease of description, we refer to the RIL(V)s in BC(V) population as the RIL(V)′ population, respectively. In the BC(V) population, six-row plots were set which included the BCF<sub>1</sub> hybrids RIL(V)′ × GX1135 in the middle, and its corresponding female RIL(V)′ and the recurrent parent, GX1135. Each line in the RIL(V)′ population was used as the female parent in the BC(V) population and was the same as that in the RIL(V) population. In BCF<sub>1</sub> population experiments of population 3 (BC) and population 4 (BCV), each plot consisted of two rows of the female RIL(V)′, BCF<sub>1</sub> hybrids and GX1135, respectively.</p><p id="p-10">In addition, two special plots, each consisting of two rows of the XZ hybrid F<sub>1</sub> and its parents, respectively, were used as controls in the experiment of population 1 and 3. Similar controls were set for experiments of population 2 and 4, and each plot consisted of the XZV hybrid F<sub>1</sub> and its parents.</p></div><div id="sec-3" class="subsection"><h3>Field trials and phenotypic evaluation</h3><p id="p-11">The parents together with the four populations were evaluated in three environments in China (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Shang <em>et al.</em> 2015a</a>): E1 – Quzhou Experimental Station of the China Agricultural University, Handan, Hebei Province; E2 – Guoxin Seed Company, Limited, Cangzhou, Hebei Province; and E3 – Xiangyang Academy of Agricultural Sciences, Xiangyang, Hubei Province. The field planting followed a randomized complete block design with duplicate at each location in 2012. Two-row plots were 80 cm and 50 cm row spacing alternately for the experiment in E1 and E2, and two-row plots in E3 were 100 cm and 80 cm row spacing alternately. The length of plots was 4 m in E1, and 3 m in E2 and E3. In the experiment for populations 1 and 2, two-row plots with each line were used. In the experiment for populations 3 and 4, six-row plots with each plot consisting of two rows of BCF<sub>1</sub> hybrid [RIL(V) × GX1135], and two for each of the corresponding parents: the female RIL(V)′ and GX1135. Field management followed the local conventional standard field practices (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Shang <em>et al.</em> 2016a</a>,<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">b</a>).</p><p id="p-12">Thirty open bolls from each plot were sampled by hand in three sites, respectively. Fiber quality traits were measured with an HVI 900 instrument (USTER HVISPECTRUM, SPINLAB) at the Cotton Fiber Quality Inspection and Test Center of the Ministry of Agriculture (Anyang, China). The fiber quality traits included 2.5% fiber span length (mm), fiber uniformity (%), fiber strength (cN/tex), fiber elongation, and micronaire.</p></div><div id="sec-4" class="subsection"><h3>Genotype analysis</h3><p id="p-13">Extraction of individual genomic DNA and population genotype analysis were carried out following the methods of <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Liang <em>et al.</em> (2013)</a>. A total of 48,836 pairs of SSR primer were employed to screen polymorphic loci between three parents (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Shang <em>et al.</em> 2016a</a>). The SSR primers newly added in this study included the EST-SSR primers, named as CAU primers, developed from the salt-tolerance EST sequences, SWU and PGML primers developed from the <em>G. raimondii</em> genome sequence, and ICR primers developed from the <em>G. arboreum</em> genome sequence. In total, 653 polymorphic loci for the XZ hybrid and 400 polymorphic loci for the XZV hybrid were used to conduct genotype analysis of two RIL populations (<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Shang <em>et al.</em> 2016a</a>,<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">b</a>). The genotype for each BCF<sub>1</sub> was deduced on the basis of the RIL genotype used as the parent for the cross.</p></div><div id="sec-5" class="subsection"><h3>Data analysis</h3><p id="p-14">A basic statistical analysis was implemented using the software SPSS version 19.0 (SPSS, Chicago). The software MAPMAKER 3.0 was employed to construct a genetic linkage map with the <em>Kosambi</em> function (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Lander <em>et al.</em> 1987</a>). For the RIL(V) and RIL(V)′ populations, the means from two replications were used as raw data at each location. For each of the BC(V)F<sub>1</sub> populations, the means of the BC(V)F<sub>1</sub>s were used independently as raw data at three locations. Single-locus QTL was conducted using composite interval mapping by the software WinQTL Cartographer 2.5 in RIL(V)′, RIL(V), and BC(V)F<sub>1</sub> data (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Zeng 1994</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wang <em>et al.</em> 2005</a>). A LOD threshold of 3.0 was used to declare suggestive QTL, whereas the same QTL in another environment or population with LOD of at least 2.0 was considered to be a common QTL (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Shang <em>et al.</em> 2015a</a>). The graphic representation of the linkage group was created using the software Map Chart 2.2 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Voorrips 2002</a>). QTL nomenclature used in rice was employed in the present study (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">McCouch <em>et al.</em> 1997</a>). Two-locus analysis that tests the main-effect QTL (M-QTL), and digenic epistatic QTL (E-QTL) and their environmental interactions (QTL × environment, QE), was conducted using the software ICIMapping 4.0 (<a href="http://www.isbreeding.net">www.isbreeding.net</a>). LOD thresholds were respectively set at 2.5 and 5.0 for declaring the presence of M-QTL, E-QTL, and their QEs (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">S. Li <em>et al.</em> 2015</a>).</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-15">All raw data are available as Supporting Information <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS9.xls">Table S9</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS10.xls">Table S10</a>, which include genotypes and traits of two hybrids.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Performance of fiber quality traits</h3><p id="p-16">The means of phenotypic data for fiber quality traits of two hybrids in three environments are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Not all the fiber quality traits possessed higher phenotype values in heterozygotes (BCF<sub>1</sub>s) than in respective homozygotes (RILs). Most of the extreme lines in the RIL(V) populations exceeded those of BC(V)F<sub>1</sub> populations under different environments. The analysis of variance in RIL(V) and BC(V) populations was conducted and significant genotypic variances and environment variances for most of the fiber quality traits were found in four populations (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Significant variations for fiber quality traits are observed in two hybrids. Skewness and kurtosis values were calculated, and results showed that all fiber quality traits fit a normal distribution in two RIL and two backcross populations of two hybrids (data not shown). The BCF<sub>1</sub> population had higher means for most of the fiber quality traits than the RIL population in the two hybrids. These results suggested that fiber quality traits of RIL(V) and BC(V) populations were highly variable and conducive for QTL analysis.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469031/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469031/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary statistics on fiber quality traits in two hybrids</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469175/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469175/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The results of analysis of variance (ANOVA) of fiber quality traits</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Correlation analysis among fiber quality traits</h3><p id="p-18">Correlation analysis was carried out using the mean values of three environments, respectively (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The majority of fiber quality traits were significantly associated with each other in two hybrids. Fiber length is significantly positively correlated with fiber strength and fiber elongation, but was negatively correlated with micronaire. Fiber strength was significantly positively correlated with fiber elongation, but it was negatively correlated with micronaire. Fiber uniformity was significantly positively correlated with fiber elongation. These results were consistent with a previous report (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Shang <em>et al.</em> 2015a</a>). The correlation coefficients among the mean values of RILs and their BCF<sub>1</sub>s for fiber quality traits are shown in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS1.pdf">Table S1</a>. Most of the fiber trait values of the RILs and that of their BCF<sub>1</sub>s showed significant positive correlation. Population performance of the BCF<sub>1</sub> for most of the fiber traits was largely determined by performance of the parental RIL.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469040/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469040/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Correlations between fiber quality traits of RIL and backcross populations in two hybrids</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>QTL analysis for fiber quality traits at the single-locus level</h3><p id="p-20">Two genetic linkage maps were previously constructed based on the polymorphic loci identified in two hybrids (<a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Shang <em>et al.</em> 2016a</a>,<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">b</a>). For the XZ hybrid, the genetic map with 623 loci spanned 3889.9 cM. For the XZV hybrid, the genetic map with 308 loci spanned 3048.4 cM. QTL detected using composite interval mapping for fiber quality traits in XZ and XZV hybrids are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS2.pdf">Table S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/FigureS1.pdf">Figure S1</a>. A total of 71 and 40 QTL were detected for fiber quality traits in RIL(V), RIL(V)′, and BC(V)F<sub>1</sub> data sets of XZ and XZV hybrids in three environments, respectively. The genetic effect identified in the RIL population is generally larger than in the backcross population. These results suggested that heterozygosity was not always necessarily advantageous for the expression of the fiber quality traits.</p><p id="p-21">In the XZ hybrid, a total of 17 QTL were detected for fiber length (FL) in three data sets, among which 13, 12, and 9 QTL were identified for FL in the RIL′s, RILs, and BCF<sub>1</sub> hybrids data sets, respectively. Thirteen QTL were identified in more than two environments or populations. Two QTL, <em>qFL-Chr5-2</em> and <em>qFL-Chr21-1</em>, were identified simultaneously in three environments and populations, which were different genetic effects. In the XZV hybrid, a total of 10 QTL were detected for FL in three data sets, among which seven, seven, and five QTL were respectively identified for FL in the RILV′s, RILVs, and BCVF<sub>1</sub> data sets. Eight QTL were identified in more than two environments or populations. Two QTL, <em>qFL-Chr2-2</em> and <em>qFL-Chr14-1</em>, with the same genetic direction were identified simultaneously in three environments and populations.</p><p id="p-22">For fiber uniformity (FU), in the XZ hybrid, a total of six QTL were resolved explaining from 4.33% to 14.18% of phenotypic variance (PV). Two QTL were identified in more than two environments or populations. In the XZV hybrid, three QTL were detected and one QTL was identified in two populations.</p><p id="p-23">For fiber strength (FS), in the XZ hybrid, a total of 15 QTL were detected in three data sets, among which 11, eight, and six QTL were respectively identified in the RIL′s, RILs, and BCF<sub>1</sub> hybrids data. Eleven QTL were identified in more than two environments or populations. In the XZV hybrid, a total of eight QTL were detected, among which two, six, and five QTL were identified in the RILV′s, RILVs, and BCVF<sub>1</sub> hybrids data, respectively. Four QTL were identified in more than two environments or populations, in which <em>qFS-Chr26-1</em> with negative genetic effect was stably expressed in multitude environments and populations.</p><p id="p-24">For fiber elongation (FE), in the XZ hybrid, a total of 11 QTL were detected in three data sets, among which five, four, and six QTL were identified in the RIL′s, RILs, and BCF<sub>1</sub> data, respectively. Three QTL were identified in more than two environments or populations. In the XZV hybrid, a total of six QTL were detected, among which three, five, and one QTL were respectively identified in the RILV′s, RILVs, and BCVF<sub>1</sub> data. Two QTL were identified in more than two environments or populations.</p><p id="p-25">For fiber micronaire (FM), in the XZ hybrid, a total of 21 QTL were resolved explaining from 3.76% to 25.93% of PV. Eleven QTL were identified in more than two environments or populations, among which two stable, QTL <em>qFM-Chr2-1</em> and <em>qFM-Chr19-1</em>, were simultaneously observed in two environments and three populations. In the XZV hybrid, 13 QTL were detected and seven QTL were identified in more than two environments or populations, among which one major QTL, <em>qFM-Chr14-1</em>, was observed in all of the environments and populations explaining from 3.97% to 14.06% of PV.</p></div><div id="sec-11" class="subsection"><h3>QTL and QE interactions resolved by two-locus analyses</h3><p id="p-26">A total of 98 and 88 M-QTL and QEs were respectively detected by inclusive composite interval mapping (ICIM) in five fiber quality traits of XZ and XZV hybrids (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS3.pdf">Table S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS4.pdf">Table S4</a>). In the XZ hybrid, a total of 65 and 33 M-QTL and QEs were detected in the RILs and BCF<sub>1</sub> hybrids data, respectively. On average, M-QTL explained 2.35% and 2.14% of the phenotype variation, and the QE explained 0.74% and 0.86% of the phenotype variation in the RILs and BCF<sub>1</sub> hybrids data, respectively. In the XZV hybrid, a total of 56 and 32 M-QTL and QEs were detected in the RILVs and BCVF<sub>1</sub> hybrids, respectively. On average, M-QTL explained 2.73% and 1.77% of the phenotype variation, and the QE explained 0.43% and 0.92% of the phenotype variation in the RILVs and BCVF<sub>1</sub> hybrids data, respectively.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469099/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469099/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Summary of M-QTL and E-QTL detected controlling fiber quality traits by ICIM in two hybrids</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">In total, 157 and 129 E-QTL and QEs were detected by ICIM in five fiber quality data sets of XZ and XZV hybrids, respectively (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS5.pdf">Table S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS6.pdf">Table S6</a>). In the XZ hybrid, a total of 90 and 67 E-QTL and QEs were detected in the RILs and BCF<sub>1</sub> hybrids data, respectively. On average, E-QTL explained 3.40% and 2.25% of the phenotype variation, and the QE explained 0.61% and 1.90% of the phenotype variation in the RILs and BCF<sub>1</sub> hybrids data, respectively. In the XZV hybrid, a total of 101 and 28 E-QTL and QEs were detected in the RILVs and BCVF<sub>1</sub> hybrids data, respectively. On average, E-QTL explained 4.04% and 2.85% of the phenotype variation, and the QE explained 0.38% and 0.64% of the phenotype variation in the RILVs and BCVF<sub>1</sub> hybrids data, respectively.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Advantage of permanent RIL and BC<sub>1</sub> populations design</h3><p id="p-29">Fiber quality traits are quantitatively inherited, and the QTL detected tend to vary under different environments. In order to identify more stable and convincing QTL in more than one environment, permanent populations such as RILs are required. Permanent populations possessing heterozygotes could be applied to study the genetic basis of complex traits using QTL mapping strategy. For example, BC<sub>1</sub> populations based on RIL population were previously constructed and used to conduct QTL analysis referring to complex agronomic traits (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Mei <em>et al.</em> 2005</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">You <em>et al.</em> 2006</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Jiang <em>et al.</em> 2014</a>; <a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Shang <em>et al.</em> 2016b</a>). In this study, we constructed the BC<sub>1</sub> population using the RIL population to identify QTL contributing to fiber quality traits and estimate their genetic effects in Upland cotton. Several QTL that were not identified in the RIL population can be detected in the BCF<sub>1</sub> population, for example, <em>qFL-Chr2-2</em> was identified only in the BCF<sub>1</sub> population in the XZ hybrid. Moreover, the QTL identified using the RIL population could be verified using the BCF<sub>1</sub> population in the current study. For instance, the QTL <em>qFM-Chr19-1</em> identified in RIL populations was confirmed in the BCF<sub>1</sub> population in both environments again in the XZ hybrid. Furthermore, the QTL detected in the XZ hybrid could be verified in the XZV hybrid in that two hybrids shared one parent (<a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">Shang <em>et al.</em> 2016b</a>). The advantages of the permanent BC<sub>1</sub> population are that it can be repeatedly made by RILs and the genotype of the BC<sub>1</sub> population can be easily deduced by the genotype of the RIL population. In addition, the QTL actions can be inferred by comparing the genetic effects of RIL, BC<sub>1</sub> performance, and midparental heterosis (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Mei <em>et al.</em> 2005</a>). However, the disadvantage of the BC<sub>1</sub> population is that only half of the possible heterozygous loci are available (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Radoev <em>et al.</em> 2008</a>). It may be the reason why some QTL with dominant effect are omitted using BCF<sub>1</sub> data rather than F<sub>2</sub> population data.</p></div><div id="sec-14" class="subsection"><h3>Consensus QTL and improvement of fiber quality</h3><p id="p-30">Two RIL populations and two corresponding backcross populations were developed and used to detect stable QTL in the present research. In a previous study, 19 QTL including eight for FL, three for FS, four for FE, and four for FM were detected in the RIL population derived from Upland cotton cross ‘GX1135’ × ‘GX100-2’ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS7.pdf">Table S7</a>). These 19 QTL for fiber traits detected previously were once again identified in the current study. In XZ and XZV hybrids, 62 stable QTL were identified in more than one environment or population. A stable QTL, <em>qFS-Chr26-1</em>, flanked by HAU1571 and PGML2562 was detected in RILV′, RILV, and BCV populations in three environments, and this QTL could contribute to 6.07–15.03% of the phenotypic variation. Another stable QTL, <em>qFM-Chr14-1</em>, was detected in RILV′, RILV, and BCV populations, explaining the phenotypic variation by 3.97–14.06%, respectively. The single-locus QTL identified by composite interval mapping and the M-QTL from ICIM were compared. A total of 59 common QTL were found, and common QTL are shown as blue figures in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS2.pdf">Table S2</a>. In addition, QTL mapping for fiber quality using the overall means across three environments (joint analysis) were conducted, and a total of 53 QTL for fiber quality traits were identified (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1/TableS8.pdf">Table S8</a>). Of 53 QTL, 44 QTL detected using joint analysis were the same as the QTL with a single environment. These novel stable QTL for fiber quality traits identified using multiple populations and environments will be helpful to improve fiber quality in the future. We compared the QTL detected in the current study with QTL identified in other studies, however the same QTL were not found. It is difficult to search for common QTL, because the genetic map, population types, population structure, and environmental conditions vary and affect the comparison of common QTL (<a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">Shang <em>et al.</em> 2015a</a>).</p><p id="p-31">The result of two hybrids can be compared and verified using the common markers located in two connected genetic maps. Two consistent QTL were acquired in both hybrid populations. A good example is the QTL <em>qFL-Chr2-1</em> for FL on chromosome 2 in the XZV hybrid; furthermore, we also identified another two QTL, <em>qFL-Chr2-1</em> and <em>qFL-Chr2-2</em>, for FL in the XZ hybrid. These two QTL identified in the XZ hybrid were located in the same marker region as mentioned above and had a narrower interval of flanking markers than that in the XZV hybrid. Therefore, using connected genetic populations may have increased the number of common QTL identified and improved the accuracy of QTL location (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Blanc <em>et al.</em> 2006</a>).</p><p id="p-32">It is difficult to simultaneously improve yield and fiber quality in Upland cotton breeding programs (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Liang <em>et al.</em> 2013</a>). In the present study, QTL that were located on chromosome five possessed multiple effects on yield and fiber quality. Two stable QTL with flanking markers between SWU20917 and NAU6240 associated with FL and FS shared a common marker interval with the QTL referring to the boll weight and lint percentage (<a id="xref-ref-24-10" class="xref-bibr" href="#ref-24">Shang <em>et al.</em> 2016b</a>). These stable QTL simultaneously controlled yield and fiber quality with a large contribution to the phenotypic variation and provided valuable information for pyramiding elite genes of yield and fiber quality traits. Further study should be conducted to prove the effects of these QTL in the improvement of yield and fiber quality in Upland cotton.</p></div><div id="sec-15" class="subsection"><h3>Genetic basis of fiber quality traits and QTL × environment interaction</h3><p id="p-33">The genetic basis of fiber quality traits is explored using two connected RIL populations and two corresponding BCF<sub>1</sub> populations at the single-locus and two-locus levels in Upland cotton. The number and phenotypic variation of QTL identified using composite interval mapping in the RIL population were collectively larger than those QTL detected in the BCF<sub>1</sub> population in the XZ and XZV hybrids. This suggested that the QTL controlling fiber quality traits mainly showed an additive effect at the single-locus level.</p><p id="p-34">At the two-locus level, lots of digenic interactions and QEs resolved by ICIM were acquired in two hybrids. The epistasis detected had been previously classified into three types: (I) two loci with significant M-QTL, (II) one locus with M-QTL and another locus without significant M-QTL, and (III) two loci without significant M-QTL (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Li <em>et al.</em> 2001</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Shang <em>et al.</em> 2016d</a>). In the XZ and XZV hybrids, we found that 39 of 301 (12.96%) epistatic interactions were type II, and the remaining 262 (87.04%) were type III for fiber quality traits. No type I interactions were acquired (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). The fact that the digenic interaction mainly included loci without detectable M-QTL further confirmed the importance of epistasis in Upland cotton breeding (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Shen <em>et al.</em> 2006</a>). In addition, the number of E-QTL and the mean of PVs explained by E-QTL for most of the fiber traits are much greater than that for M-QTL in two hybrids (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). These results revealed that epistasis played an important role not only in the variation of the performance of the RIL(V) population but also in the expression of hybrids in the BC(V) population. These results are in agreement with previous results which indicated that additive effect and epistatic interaction are common and important for fiber quality traits in Upland cotton. Selection for improving trait values should pay attention to the best multilocus combinations and major loci (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Shen <em>et al.</em> 2006</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469170/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469170/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Type of epistatic interactions detected for fiber quality traits in two hybrids</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">The genotype × environment interactions accounted for a small proportion of the mean phenotypic variation for fiber traits (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). The QTL for fiber quality in the BC(V) population was more sensitive to the environment than that in the RIL(V) population as shown by the mean phenotypic variation explained by QEs. Particular environmental conditions were important in the expression of fiber quality, especially for Upland cotton hybrid. The results suggest that trials in multiple environments are a prerequisite for evaluating fiber quality traits (<a id="xref-ref-23-9" class="xref-bibr" href="#ref-23">Shang <em>et al.</em> 2016a</a>). The environmental factors in QTL associated with fiber quality and epistasis, and effect of QEs should be considered in MAS breeding (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Xing <em>et al.</em> 2002</a>; <a id="xref-ref-24-11" class="xref-bibr" href="#ref-24">Shang <em>et al.</em> 2016b</a>).</p><p id="p-37">Overall, the single-locus with additive effect and epistasis with few detectable main effects play an important role in controlling expression of fiber quality in Upland cotton.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We thank Dr. Zhengsheng Zhang (Southwest University) and Dr. Youlu Yuan (Institute of Cotton Research, Chinese Academy of Agricultural Sciences) for providing SWU and ICR SSR primers. We thank Qingzhi Liang (Chinese Academy of Tropical Agricultural Sciences), and Yihua Rong and Xianhui Yan (Xiangyang Academy of Agricultural Science, Hubei Provinces) for the contributions on data acquisition. This research was supported by a grant from the National Natural Science Foundation of China (31171591), and a grant from the National High Technology Research and Development Program (2011AA10A102) to J.H.</p><p id="p-39">Author contributions: J.H. conceived the experiments, provided the experimental platform, and revised the manuscript. L.S., X.W., A.A., S.C., Y.L., and L.M. performed the experiments. Y.W. maintained the experimental platform, and carried out the bench work, and F.L. and K.W. supplied some of the markers. L.S. performed the data analysis, prepared the draft manuscript, and revised the manuscript. All authors approved the final manuscript. The authors declare that they have no conflict of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-40">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031302/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-41"><em>Communicating editor: J. A Birchler</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-05-17"><span class="received-label">Received </span>May 17, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-06-20"><span class="accepted-label">Accepted </span>June 20, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Shang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.1" data-doi="10.1007/s00122-006-0287-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Connected populations for detecting quantitative trait loci and testing for epistasis: an application in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBlanc%26rft.auinit1%253DG.%26rft.volume%253D113%26rft.issue%253D2%26rft.spage%253D206%26rft.epage%253D224%26rft.atitle%253DConnected%2Bpopulations%2Bfor%2Bdetecting%2Bquantitative%2Btrait%2Bloci%2Band%2Btesting%2Bfor%2Bepistasis%253A%2Ban%2Bapplication%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0287-1%26rft_id%253Dinfo%253Apmid%252F16791688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0287-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16791688&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238503200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.2" data-doi="10.1104/pp.107.107672"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dennis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Triplett</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Toward sequencing cotton (<em>Gossypium</em>) genomes.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">1303</span>–<span class="cit-lpage">1310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DChen%26rft.auinit1%253DZ.%2BJ.%26rft.volume%253D145%26rft.issue%253D4%26rft.spage%253D1303%26rft.epage%253D1310%26rft.atitle%253DToward%2BSequencing%2BCotton%2B%2528Gossypium%2529%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.107.107672%26rft_id%253Dinfo%253Apmid%252F18056866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQ1LzQvMTMwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yNzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.3" data-doi="10.1007/s00122-016-2684-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Islam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thyssen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delhom</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mapping by sequencing in cotton (<em>Gossypium hirsutum</em>) line MD52ne identified candidate genes for fiber strength and its related quality attributes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1071</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1071%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2684-4%26rft_id%253Dinfo%253Apmid%252F26883043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2684-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26883043&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.4" data-doi="10.1016/j.gene.2013.12.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Analysis of quantitative trait loci affecting chlorophyll content of rice leaves in a double haploid population and two backcross populations.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">536</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D536%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2013.12.010%26rft_id%253Dinfo%253Apmid%252F24361205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2013.12.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24361205&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.5" data-doi="10.1023/A:1012263413418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazo</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular mapping and characterization of traits controlling fiber quality in cotton.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D121%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1012263413418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1012263413418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000171398600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.6" data-doi="10.1016/0888-7543(87)90010-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrahamson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barlow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Mapmaker an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D181%26rft.atitle%253DMAPMAKER%253A%2Ban%2Binteractive%2Bcomputer%2Bpackage%2Bfor%2Bconstructing%2Bprimary%2Bgenetic%2Blinkage%2Bmaps%2Bof%2Bexperimental%2Band%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252887%252990010-3%26rft_id%253Dinfo%253Apmid%252F3692487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(87)90010-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3692487&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.7" data-doi="10.1038/ng.2987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequence of the cultivated cotton <em>Gossypium arboreum</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2987%26rft_id%253Dinfo%253Apmid%252F24836287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24836287&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.8" data-doi="10.1038/nbt.3208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome sequence of cultivated Upland cotton (<em>Gossypium hirsutum</em> TM-1) provides insights into genome evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3208%26rft_id%253Dinfo%253Apmid%252F25893780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25893780&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.9" data-doi="10.1371/journal.pone.0132414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Inclusive composite interval mapping of QTL by environment interactions in biparental populations.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0132414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0132414%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0132414%26rft_id%253Dinfo%253Apmid%252F26161656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0132414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26161656&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mei</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">Q. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Overdominant epistatic loci are the primary genetic basis of inbreeding depression and heterosis in rice. I. Biomass and grain yield.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1737</span>–<span class="cit-lpage">1753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLi%26rft.auinit1%253DZ.-K.%26rft.volume%253D158%26rft.issue%253D4%26rft.spage%253D1737%26rft.epage%253D1753%26rft.atitle%253DOverdominant%2BEpistatic%2BLoci%2BAre%2Bthe%2BPrimary%2BGenetic%2BBasis%2Bof%2BInbreeding%2BDepression%2Band%2BHeterosis%2Bin%2BRice.%2BI.%2BBiomass%2Band%2BGrain%2BYield%26rft_id%253Dinfo%253Apmid%252F11514459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OC80LzE3MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.11" data-doi="10.1007/s11434-013-5807-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Construction of a linkage map and QTL mapping for fiber quality traits in upland cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Chin. Sci. Bull.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">3233</span>–<span class="cit-lpage">3243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChin.%2BSci.%2BBull.%26rft.volume%253D58%26rft.spage%253D3233%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11434-013-5807-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11434-013-5807-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.12" data-doi="10.1371/journal.pone.0143548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Partial dominance, overdominance and epistasis as the genetic basis of heterosis in Upland Cotton.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0143548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0143548%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0143548%26rft_id%253Dinfo%253Apmid%252F26618635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0143548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26618635&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">Y. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blinstrub</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Report on QTL nomenclature.</span> <abbr class="cit-jnl-abbrev">Rice Genet. Newsl.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2BGenet.%2BNewsl.%26rft.volume%253D14%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.14" data-doi="10.1007/s00122-003-1433-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syed</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaxton</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic mapping and QTL analysis of fiber-related traits in cotton (<em>Gossypium</em>).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMei%26rft.auinit1%253DM.%26rft.volume%253D108%26rft.issue%253D2%26rft.spage%253D280%26rft.epage%253D291%26rft.atitle%253DGenetic%2Bmapping%2Band%2BQTL%2Banalysis%2Bof%2Bfiber-related%2Btraits%2Bin%2Bcotton%2B%2528%2BGossypium%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1433-7%26rft_id%253Dinfo%253Apmid%252F14513220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1433-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14513220&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188079400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.15" data-doi="10.1007/s00122-004-1890-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mei</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">Q. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene actions of QTLs affecting several agronomic traits resolved in a recombinant inbred rice population and two backcross populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMei%26rft.auinit1%253DH.%2BW.%26rft.volume%253D110%26rft.issue%253D4%26rft.spage%253D649%26rft.epage%253D659%26rft.atitle%253DGene%2Bactions%2Bof%2BQTLs%2Baffecting%2Bseveral%2Bagronomic%2Btraits%2Bresolved%2Bin%2Ba%2Brecombinant%2Binbred%2Brice%2Bpopulation%2Band%2Btwo%2Bbackcross%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1890-7%26rft_id%253Dinfo%253Apmid%252F15647921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1890-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15647921&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227714400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.16" data-doi="10.1038/335721a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hewitt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lincoln</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Resolution of quantitative traits into Mendelian factors by using a complete linkage map of restriction length polymorphisms.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D335%26rft.issue%253D6192%26rft.spage%253D721%26rft.epage%253D726%26rft.atitle%253DResolution%2Bof%2Bquantitative%2Btraits%2Binto%2BMendelian%2Bfactors%2Bby%2Busing%2Ba%2Bcomplete%2Blinkage%2Bmap%2Bof%2Brestriction%2Bfragment%2Blength%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F335721a0%26rft_id%253Dinfo%253Apmid%252F2902517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/335721a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2902517&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q532400055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saranga</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">QTL analysis of genotype × environment interaction affecting cotton fiber quality.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">384</span>–<span class="cit-lpage">396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D106%26rft.issue%253D3%26rft.spage%253D384%26rft.epage%253D396%26rft.atitle%253DQTL%2Banalysis%2Bof%2Bgenotype%2Bx%2Benvironment%2Binteractions%2Baffecting%2Bcotton%2Bfiber%2Bquality.%26rft_id%253Dinfo%253Apmid%252F12589538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12589538&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181521500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.18" data-doi="10.1038/nature11798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundlach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibres.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">492</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D492%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11798%26rft_id%253Dinfo%253Apmid%252F23257886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23257886&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312488200055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.19" data-doi="10.1534/genetics.108.089680"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Radoev</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecke</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic analysis of heterosis for yield and yield components in rapeseed (<em>Brassica napus</em> L.) by quantitative trait locus mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1547</span>–<span class="cit-lpage">1558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089680%26rft_id%253Dinfo%253Apmid%252F18562665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE1NDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.20" data-doi="10.1007/s00438-015-1021-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Said</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapka</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cotton QTLdb: a cotton QTL database for QTL analysis, visualization, and comparison between <em>Gossypium hirsutum</em> and <em>G. hirsutum × G. barbadense</em> populations.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1615</span>–<span class="cit-lpage">1625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGenet.%2BGenomics%26rft.volume%253D290%26rft.spage%253D1615%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-015-1021-y%26rft_id%253Dinfo%253Apmid%252F25758743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-015-1021-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25758743&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.21" data-doi="10.1007/s10681-015-1434-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Q. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015a</span> <span class="cit-article-title">Identification of stable QTLs controlling fiber traits properties in multi-environment using recombinant inbred lines in Upland cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">877</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D205%26rft.spage%253D877%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-015-1434-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-015-1434-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.22" data-doi="10.1111/pbr.12316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abduweli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015b</span> <span class="cit-article-title">Dynamic QTL mapping for plant height in Upland cotton (<em>Gossypium hirsutum</em>).</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D134%26rft.spage%253D703%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.23" data-doi="10.1111/pbr.12352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abduweli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016a</span> <span class="cit-article-title">Genetic analysis and QTL mapping of oil content and seed index using two recombinant inbred lines and two backcross populations in Upland cotton.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">224</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D135%26rft.spage%253D224%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.24" data-doi="10.1534/g3.115.025809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016b</span> <span class="cit-article-title">Partial dominance, overdominance, epistasis and QTL by environment interactions contribute to the heterosis in two Upland cotton hybrids</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.025809%26rft_id%253Dinfo%253Apmid%252F26715091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvMy80OTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.25" data-doi="10.1007/s10681-015-1564-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abduweli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016c</span> <span class="cit-article-title">Seedling root QTLs analysis on dynamic development and upon nitrogen deficiency stress in Upland cotton.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D207%26rft.spage%253D645%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-015-1564-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-015-1564-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.26" data-doi="10.1007/s00122-016-2714-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016d</span> <span class="cit-article-title">Epistasis together with partial dominance, over-dominance and QTL by environment interactions contribute to yield heterosis in upland cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1429</span>–<span class="cit-lpage">1446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1429%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2714-2%26rft_id%253Dinfo%253Apmid%252F27138784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2714-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27138784&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.27" data-doi="10.2135/cropsci2005.0056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping fiber and yield QTLs with main epistatic and QTL × environment interaction effects in recombinant inbred lines of upland cotton.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D46%26rft.spage%253D61%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.0056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.0056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.28" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: Software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.29" data-doi="10.1371/journal.pone.0130742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015a</span> <span class="cit-article-title">QTL Mapping for fiber and yield traits in upland cotton under multiple environments.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0130742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0130742%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0130742%26rft_id%253Dinfo%253Apmid%252F26110526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0130742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26110526&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.30" data-doi="10.1093/dnares/dsu047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2015b</span> <span class="cit-article-title">Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsu047%26rft_id%253Dinfo%253Apmid%252F25656006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIyLzIvMTQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.31" data-doi="10.1038/ng.2371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The draft genome of a diploid cotton <em>Gossypium raimondii</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1098</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D44%26rft.issue%253D10%26rft.spage%253D1098%26rft.epage%253D1103%26rft.atitle%253DThe%2Bdraft%2Bgenome%2Bof%2Ba%2Bdiploid%2Bcotton%2BGossypium%2Braimondii.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2371%26rft_id%253Dinfo%253Apmid%252F22922876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22922876&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2717.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Windows QTL Cartographer 2.5</span>. <span class="cit-publ-name">Department of Statistics, North Carolina State University</span>, <span class="cit-publ-loc">Raleigh, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.33" data-doi="10.1007/s00122-002-0952-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Characterization of the main effects, epistatic effects and their environmental interactions of QTLs on the genetic basis of yield traits in rice.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">248</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DXing%26rft.auinit1%253DZ.%26rft.volume%253D105%26rft.issue%253D2-3%26rft.spage%253D248%26rft.epage%253D257%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Bmain%2Beffects%252C%2Bepistatic%2Beffects%2Band%2Btheir%2Benvironmental%2Binteractions%2Bof%2BQTLs%2Bon%2Bthe%2Bgenetic%2Bbasis%2Bof%2Byield%2Btraits%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-002-0952-y%26rft_id%253Dinfo%253Apmid%252F12582526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-002-0952-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582526&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177739800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.34" data-doi="10.1534/genetics.105.047209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Identification of quantitative trait loci across recombinant inbred lines and testcross populations for traits of agronomic importance in rice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1287</span>–<span class="cit-lpage">1300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047209%26rft_id%253Dinfo%253Apmid%252F16322522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8yLzEyODciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.35" data-doi="10.1093/nar/gkt1064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ficklin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CottonGen: a genomics, genetics and breeding database for cotton research.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1229</span>–<span class="cit-lpage">1236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D42%26rft.spage%253D1229%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.36" data-doi="10.1038/srep17662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genome sequence of Sea-Island cotton (<em>Gossypium barbadense</em>) provides insights into the allopolyploidization and development of superior spinnable fibres.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">17662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D17662%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep17662%26rft_id%253Dinfo%253Apmid%252F26634818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep17662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26634818&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Precision mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1457%26rft.epage%253D1468%26rft.atitle%253DPrecision%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F8013918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzE0NTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2717.38" data-doi="10.1038/nbt.3207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sequencing of allotetraploid cotton (<em>Gossypium hirsutum</em> L. acc. TM-1) provides a resource for fiber improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D531%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3207%26rft_id%253Dinfo%253Apmid%252F25893781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25893781&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.9.2717.39" data-doi="10.1371/journal.pone.0135430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muhammad</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High resolution consensus mapping of quantitative trait loci for fiber strength, length and micronaire on chromosome 25 of the Upland cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0135430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0135430%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0135430%26rft_id%253Dinfo%253Apmid%252F26262992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0135430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26262992&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/9/2717.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2707" title="Microsporidia Intracellular Development Relies on Myc Interaction Network Transcription Factors in the Host" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2725" title="A Genomic Bayesian Multi-trait and Multi-environment Model" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/9"><span>Volume 6 Issue 9, September 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/9.cover-source.jpg?itok=nund1hRf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2717" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-V23aPG9BT5vp7ZiBTJiRM-Q5re9RYBQza0B7DU7C2MI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/9/2717" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/9/2717" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/9/2717" class="active" data-icon-position="" data-hide-link-title="0">Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460296" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-n2IknVkcChmXvrw-dOllUNGDJzalwfY1gM2wCuccbBA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2717.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/9/2717" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-EfbPgADaDrnZMZVceW-gPx6NxIFCKxm6Nnn8l89QGxk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/9/2717" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2717.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460296" id="node460296--2" data-pisa="ggg;6/9/2717" data-pisa-master="ggg;g3.116.031302" data-apath="/ggg/6/9/2717.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lianguang Shang</span>, <span class="highwire-citation-author" data-delta="1">Yumei Wang</span>, <span class="highwire-citation-author" data-delta="2">Xiaocui Wang</span>, <span class="highwire-citation-author" data-delta="3">Fang Liu</span>, <span class="highwire-citation-author" data-delta="4">Abdugheni Abduweli</span>, <span class="highwire-citation-author" data-delta="5">Shihu Cai</span>, <span class="highwire-citation-author" data-delta="6">Yuhua Li</span>, <span class="highwire-citation-author" data-delta="7">Lingling Ma</span>, <span class="highwire-citation-author" data-delta="8">Kunbo Wang</span> and <span class="highwire-citation-author" data-delta="9">Jinping Hua</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2717-2724;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031302 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460296--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lianguang Shang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lianguang%2BShang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALianguang%2BShang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yumei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Cash Crops, Hubei Academy of Agricultural Sciences, Wuhan 430064, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yumei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYumei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiaocui Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaocui%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaocui%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Cotton Research, Chinese Academy of Agricultural Sciences/State Key Laboratory of Cotton Biology, Anyang 455000, Henan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Abdugheni Abduweli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abdugheni%2BAbduweli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abduweli%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbdugheni%2BAbduweli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Shihu Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shihu%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShihu%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yuhua Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuhua%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuhua%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lingling Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lingling%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALingling%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kunbo Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Cotton Research, Chinese Academy of Agricultural Sciences/State Key Laboratory of Cotton Biology, Anyang 455000, Henan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kunbo%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKunbo%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jinping Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinping%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJinping%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jinping_hua@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jinping_hua@cau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/9/2717" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/9/2717&amp;title=Genetic%20Analysis%20and%20QTL%20Detection%20on%20Fiber%20Traits%20Using%20Two%20Recombinant%20Inbred%20Lines%20and%20Their%20Backcross%20Populations%20in%20Upland%20Cotton" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/9/2717&amp;title=Genetic%20Analysis%20and%20QTL%20Detection%20on%20Fiber%20Traits%20Using%20Two%20Recombinant%20Inbred%20Lines%20and%20Their%20Backcross%20Populations%20in%20Upland%20Cotton" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/9/2717&amp;title=Genetic%20Analysis%20and%20QTL%20Detection%20on%20Fiber%20Traits%20Using%20Two%20Recombinant%20Inbred%20Lines%20and%20Their%20Backcross%20Populations%20in%20Upland%20Cotton" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/9/2717&amp;text=Genetic%20Analysis%20and%20QTL%20Detection%20on%20Fiber%20Traits%20Using%20Two%20Recombinant%20Inbred%20Lines%20and%20Their%20Backcross%20Populations%20in%20Upland%20Cotton" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/9/2717&amp;title=Genetic%20Analysis%20and%20QTL%20Detection%20on%20Fiber%20Traits%20Using%20Two%20Recombinant%20Inbred%20Lines%20and%20Their%20Backcross%20Populations%20in%20Upland%20Cotton" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/9/2717&amp;t=Genetic%20Analysis%20and%20QTL%20Detection%20on%20Fiber%20Traits%20Using%20Two%20Recombinant%20Inbred%20Lines%20and%20Their%20Backcross%20Populations%20in%20Upland%20Cotton" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/9/2717&amp;title=Genetic%20Analysis%20and%20QTL%20Detection%20on%20Fiber%20Traits%20Using%20Two%20Recombinant%20Inbred%20Lines%20and%20Their%20Backcross%20Populations%20in%20Upland%20Cotton" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/9/2717&amp;title=Genetic%20Analysis%20and%20QTL%20Detection%20on%20Fiber%20Traits%20Using%20Two%20Recombinant%20Inbred%20Lines%20and%20Their%20Backcross%20Populations%20in%20Upland%20Cotton" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460296/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460296" id="node460296--3" data-pisa="ggg;6/9/2717" data-pisa-master="ggg;g3.116.031302" data-apath="/ggg/6/9/2717.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lianguang Shang</span>, <span class="highwire-citation-author" data-delta="1">Yumei Wang</span>, <span class="highwire-citation-author" data-delta="2">Xiaocui Wang</span>, <span class="highwire-citation-author" data-delta="3">Fang Liu</span>, <span class="highwire-citation-author" data-delta="4">Abdugheni Abduweli</span>, <span class="highwire-citation-author" data-delta="5">Shihu Cai</span>, <span class="highwire-citation-author" data-delta="6">Yuhua Li</span>, <span class="highwire-citation-author" data-delta="7">Lingling Ma</span>, <span class="highwire-citation-author" data-delta="8">Kunbo Wang</span> and <span class="highwire-citation-author" data-delta="9">Jinping Hua</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2717-2724;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031302 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460296--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lianguang Shang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lianguang%2BShang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALianguang%2BShang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yumei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Cash Crops, Hubei Academy of Agricultural Sciences, Wuhan 430064, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yumei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYumei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiaocui Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaocui%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaocui%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Cotton Research, Chinese Academy of Agricultural Sciences/State Key Laboratory of Cotton Biology, Anyang 455000, Henan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Abdugheni Abduweli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abdugheni%2BAbduweli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abduweli%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbdugheni%2BAbduweli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Shihu Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shihu%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShihu%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yuhua Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuhua%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuhua%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lingling Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lingling%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALingling%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kunbo Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Cotton Research, Chinese Academy of Agricultural Sciences/State Key Laboratory of Cotton Biology, Anyang 455000, Henan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kunbo%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKunbo%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jinping Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Genetics and Breeding/Key Laboratory of Crop Heterosis and Utilization of Ministry of Education/Beijing Key Laboratory of Crop Genetic Improvement, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinping%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJinping%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jinping_hua@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jinping_hua@cau.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2717&amp;rft.epage=2724&amp;rft.atitle=Genetic%20Analysis%20and%20QTL%20Detection%20on%20Fiber%20Traits%20Using%20Two%20Recombinant%20Inbred%20Lines%20and%20Their%20Backcross%20Populations%20in%20Upland%20Cotton&amp;rft.volume=6&amp;rft.issue=9&amp;rft.date=2016-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Shang%2C+Lianguang&amp;rft.au=Wang%2C+Yumei&amp;rft.au=Wang%2C+Xiaocui&amp;rft.au=Liu%2C+Fang&amp;rft.au=Abduweli%2C+Abdugheni&amp;rft.au=Cai%2C+Shihu&amp;rft.au=Li%2C+Yuhua&amp;rft.au=Ma%2C+Lingling&amp;rft.au=Wang%2C+Kunbo&amp;rft.au=Hua%2C+Jinping"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460296/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460296/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460296/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460296/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460296/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460296/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460296/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460296/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460296/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460296/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460296/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460296/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/9/2717" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/9/2717.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/9/2717.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/9/2717.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/9/2717.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DNYVmlR6RlVWGCviGjgljNup52yhnZXIncP3-_bmyQA__xJKgaB9J7btTrapa3xItcGD9KGApkh4nzPcIVGC2naU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
