<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/9/2725.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/9/2725.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/9/2725/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/9/2725.atom" />
<meta name="HW.pisa" content="ggg;6/9/2725" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Genomic Bayesian Multi-trait and Multi-environment Model" />
<meta name="DC.Identifier" content="10.1534/g3.116.032359" />
<meta name="DC.Date" content="2016-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Montesinos-López et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="When information on multiple genotypes evaluated in multiple environments is recorded, a multi-environment single trait model for assessing genotype × environment interaction (G × E) is usually employed. Comprehensive models that simultaneously take into account the correlated traits and trait × genotype × environment interaction (T × G × E) are lacking. In this research, we propose a Bayesian model for analyzing multiple traits and multiple environments for whole-genome prediction (WGP) model. For this model, we used Half-![Formula][1]&lt;/img&gt; priors on each standard deviation term and uniform priors on each correlation of the covariance matrix. These priors were not informative and led to posterior inferences that were insensitive to the choice of hyper-parameters. We also developed a computationally efficient Markov Chain Monte Carlo (MCMC) under the above priors, which allowed us to obtain all required full conditional distributions of the parameters leading to an exact Gibbs sampling for the posterior distribution. We used two real data sets to implement and evaluate the proposed Bayesian method and found that when the correlation between traits was high (&gt;0.5), the proposed model (with unstructured variance–covariance) improved prediction accuracy compared to the model with diagonal and standard variance–covariance structures. The R-software package Bayesian Multi-Trait and Multi-Environment (BMTME) offers optimized C++ routines to efficiently perform the analyses.

 [1]: /embed/mml-math-1.gif" />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Fernando H. Toledo" />
<meta name="DC.Contributor" content="Oscar Pérez-Hernández" />
<meta name="DC.Contributor" content="Kent M. Eskridge" />
<meta name="DC.Contributor" content="Jessica Rutkoski" />
<meta name="article:published_time" content="2016-09-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="A Genomic Bayesian Multi-trait and Multi-environment Model" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;When information on multiple genotypes evaluated in multiple environments is recorded, a multi-environment single trait model for assessing genotype × environment interaction (G × E) is usually employed. Comprehensive models that simultaneously take into account the correlated traits and trait × genotype × environment interaction (T × G × E) are lacking. In this research, we propose a Bayesian model for analyzing multiple traits and multiple environments for whole-genome prediction (WGP) model. For this model, we used Half-&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me1&quot; overflow=&quot;scroll&quot;&gt;&lt;mi mathvariant=&quot;italic&quot;&gt;t&lt;/mi&gt;&lt;/math&gt; priors on each standard deviation term and uniform priors on each correlation of the covariance matrix. These priors were not informative and led to posterior inferences that were insensitive to the choice of hyper-parameters. We also developed a computationally efficient Markov Chain Monte Carlo (MCMC) under the above priors, which allowed us to obtain all required full conditional distributions of the parameters leading to an exact Gibbs sampling for the posterior distribution. We used two real data sets to implement and evaluate the proposed Bayesian method and found that when the correlation between traits was high (&amp;gt;0.5), the proposed model (with unstructured variance–covariance) improved prediction accuracy compared to the model with diagonal and standard variance–covariance structures. The R-software package Bayesian Multi-Trait and Multi-Environment (BMTME) offers optimized C++ routines to efficiently perform the analyses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/09/01" />
<meta name="citation_mjid" content="ggg;6/9/2725" />
<meta name="citation_id" content="6/9/2725" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/9/2725" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/9/2725.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/9/2725.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/9/2725.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.032359" />
<meta name="citation_pmid" content="27342738" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2725" />
<meta name="citation_lastpage" content="2744" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México" />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), Guanajuato, Guanajuato 36240, México" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author" content="Fernando H. Toledo" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México" />
<meta name="citation_author" content="Oscar Pérez-Hernández" />
<meta name="citation_author_institution" content="Department of Biology and Agriculture, University of Central Missouri, Warrensburg, Missouri 64093" />
<meta name="citation_author" content="Kent M. Eskridge" />
<meta name="citation_author_institution" content="Statistics Department, University of Nebraska, Lincoln, Nebraska, 68583" />
<meta name="citation_author" content="Jessica Rutkoski" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype×environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_issue=2;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. P. Calus;citation_author=R. F. Veerkamp;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=43;citation_year=2011;citation_issue=1;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_issue=2;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_author=G. Mahuku;citation_author=S. Dreisigacker;citation_title=Genomic selection and prediction in plant breeding.;citation_pages=239-261;citation_volume=25;citation_year=2011;citation_issue=3;citation_doi=10.1080/15427528.2011.558767" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Pérez-Rodríguez;citation_author=J. M. Hickey;citation_title=Genomic prediction in maize breeding populations with genotyping-by-sequencing.;citation_volume=3;citation_year=2013;citation_issue=11;citation_pmid=24022750;citation_doi=10.1534/g3.113.008227" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=G. de los Campos;citation_author=D. Gianola;citation_title=Factor analysis models for structuring covariance matrices of additive genetic effects: a Bayesian implementation.;citation_pages=481-494;citation_volume=39;citation_year=2007;citation_issue=5;citation_pmid=17897592;citation_doi=10.1051/gse:20070016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_issue=1;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=04;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_title=Seamless R and C++ Integration with Rcpp;citation_author=D. Eddelbuettel;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Comput. Stat. Data Anal.;citation_author=D. Eddelbuettel;citation_author=C. Sanderson;citation_title=Rcpparmadillo: Accelerating R with high-performance C++ linear algebra.;citation_pages=1054-1063;citation_volume=71;citation_year=2014;citation_doi=10.1016/j.csda.2013.02.005" />
<meta name="citation_reference" content="citation_journal_title=Bayesian Anal.;citation_author=A. Gelman;citation_title=Prior distributions for variance parameters in hierarchical models (comment on article by Browne and Draper).;citation_pages=515-534;citation_volume=1;citation_year=2006;citation_issue=3;citation_doi=10.1214/06-BA117A" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=C. J. Geyer;citation_title=Practical Markov Chain Monte Carlo.;citation_pages=473-483;citation_volume=7;citation_year=1992;citation_issue=4;citation_doi=10.1214/ss/1177011137" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: the Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_issue=3;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=G. Guo;citation_author=F. Zhao;citation_author=Y. Wang;citation_author=Y. Zhang;citation_author=L. Du;citation_title=Comparison of single-trait and multiple-trait genomic prediction models.;citation_pages=30;citation_volume=15;citation_year=2014;citation_issue=1;citation_pmid=24593261" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. R. Henderson;citation_author=R. L. Quaas;citation_title=Multiple Trait Evaluation Using Relatives' Records;citation_pages=1188-1197;citation_volume=43;citation_year=1976;citation_issue=6;citation_doi=10.2527/jas1976.4361188x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H. P. Yang;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic selection in plant breeding: A comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_issue=1;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Heslot;citation_author=D. Akdemir;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.;citation_pages=463-480;citation_volume=127;citation_year=2014;citation_issue=2;citation_pmid=24264761;citation_doi=10.1007/s00122-013-2231-5" />
<meta name="citation_reference" content="citation_journal_title=Bayesian Anal.;citation_author=A. Huang;citation_author=M. P. Wand;citation_title=Simple marginally noninformative prior distributions for covariance matrices.;citation_pages=439-452;citation_volume=8;citation_year=2013;citation_issue=2;citation_doi=10.1214/13-BA815" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. D. Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_issue=3;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J. L. Jannink;citation_title=Multiple-trait genomic selection methods increase genetic value prediction accuracy.;citation_volume=192;citation_year=2012;citation_issue=4;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Jiang;citation_author=Q. Zhang;citation_author=L. Ma;citation_author=J. Li;citation_author=Z. Wang;citation_title=Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.;citation_pages=29-36;citation_volume=115;citation_year=2015;citation_issue=1;citation_pmid=25873147;citation_doi=10.1038/hdy.2015.9" />
<meta name="citation_reference" content="citation_title=Applied Multivariate Statistical Analysis;citation_author=R. A. Johnson;citation_author=D. W. Wichern;citation_volume=4;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=W. A. Link;citation_author=M. J. Eaton;citation_title=On thinning of chains in MCMC.;citation_pages=112-115;citation_volume=3;citation_year=2012;citation_issue=1;citation_doi=10.1111/j.2041-210X.2011.00131.x" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. A. López-Cruz;citation_author=J. Crossa;citation_author=D. Bonnet;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_title=Increased prediction accuracy in wheat breeding trials using a markers × environment interaction genomic selection model.;citation_volume=5;citation_year=2015;citation_issue=4;citation_pmid=25660166;citation_doi=10.1534/g3.114.016097" />
<meta name="citation_reference" content="citation_journal_title=Am. Stat.;citation_author=S. N. MacEachern;citation_author=L. M. Berliner;citation_title=Subsampling the Gibbs sampler.;citation_pages=188-190;citation_volume=48;citation_year=1994;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_author=K. M. Eskridge;citation_title=Threshold models for genome-enabled prediction of ordinal categorical traits in plant breeding.;citation_volume=5;citation_year=2015;citation_issue=1;citation_pmid=25538102;citation_doi=10.1534/g3.114.016188" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. M. González-Camacho;citation_author=J. Crossa;citation_author=Y. Manes;citation_title=A comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.;citation_pages=1595-1605;citation_volume=2;citation_year=2012;citation_issue=12" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=S. O. Peters;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_author=J. M. Reecy;citation_title=Bayesian genome-wide association analysis of growth and yearling ultrasound measures of carcass traits in Brangus heifers.;citation_volume=90;citation_year=2012;citation_issue=10;citation_pmid=23038745;citation_doi=10.2527/jas.2011-4507" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=E. J. Pollak;citation_author=J. Van der Werf;citation_author=R. L. Quaas;citation_title=Selection Bias and Multiple Trait Evaluation;citation_pages=1590-1595;citation_volume=67;citation_year=1984;citation_issue=7;citation_doi=10.3168/jds.S0022-0302(84)81481-2" />
<meta name="citation_reference" content="citation_journal_title=Animal;citation_author=M. Pszczola;citation_author=R. F. Veerkamp;citation_author=Y. de Haas;citation_author=E. Wall;citation_author=T. Strabel;citation_title=Effect of predictor traits on accuracy of genomic breeding values for feed intake based on a limited cow reference population.;citation_pages=1759-1768;citation_volume=7;citation_year=2013;citation_issue=11;citation_pmid=23915541;citation_doi=10.1017/S175173111300150X" />
<meta name="citation_reference" content="citation_title=Testing of a Structured Covariance Matrix for Three-level Repeated Measures Data;citation_author=A. Roy;citation_author=R. A. Leiva;citation_year=2008" />
<meta name="citation_reference" content="citation_author=J. Rutkoski;citation_author=J. Poland;citation_author=S. Mondal;citation_author=E. Autrique;citation_author=J. Crossa;citation_author=M. Reynolds;citation_author=R. Singh;citation_year=2016" />
<meta name="citation_reference" content="citation_title=Armadillo: An Open Source C++ Algebra Library for Fast Prototyping and Computationally Intensive Experiments. Tech. Rep;citation_author=C. Sanderson;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=L. R. Schaeffer;citation_title=Sire and Cow Evaluation Under Multiple Trait Models;citation_pages=1567-1580;citation_volume=67;citation_year=1984;citation_issue=7;citation_doi=10.3168/jds.S0022-0302(84)81479-4" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. L. Simpson;citation_author=L. J. Edwards;citation_author=M. A. Styner;citation_author=K. E. Muller;citation_title=Kronecker Product Linear Exponent AR(1) Correlation Structures for Multivariate Repeated Measures.;citation_pages=e88864;citation_volume=9;citation_year=2014;citation_issue=2;citation_pmid=24586419;citation_doi=10.1371/journal.pone.0088864" />
<meta name="citation_reference" content="citation_title=The C++ Programming Language;citation_author=B. Stroustrup;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Genomic Bayesian Multi-trait and Multi-environment Model" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="twitter:description" content="When information on multiple genotypes evaluated in multiple environments is recorded, a multi-environment single trait model for assessing genotype × environment interaction (G × E) is usually employed. Comprehensive models that simultaneously take into account the correlated traits and trait × genotype × environment interaction (T × G × E) are lacking. In this research, we propose a Bayesian model for analyzing multiple traits and multiple environments for whole-genome prediction (WGP) model. For this model, we used Half-![Formula][1]&lt;/img&gt; priors on each standard deviation term and uniform priors on each correlation of the covariance matrix. These priors were not informative and led to posterior inferences that were insensitive to the choice of hyper-parameters. We also developed a computationally efficient Markov Chain Monte Carlo (MCMC) under the above priors, which allowed us to obtain all required full conditional distributions of the parameters leading to an exact Gibbs sampling for the posterior distribution. We used two real data sets to implement and evaluate the proposed Bayesian method and found that when the correlation between traits was high (&gt;0.5), the proposed model (with unstructured variance–covariance) improved prediction accuracy compared to the model with diagonal and standard variance–covariance structures. The R-software package Bayesian Multi-Trait and Multi-Environment (BMTME) offers optimized C++ routines to efficiently perform the analyses.

 [1]: /embed/mml-math-1.gif" />
<meta name="og-title" property="og:title" content="A Genomic Bayesian Multi-trait and Multi-environment Model" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/9/2725" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="When information on multiple genotypes evaluated in multiple environments is recorded, a multi-environment single trait model for assessing genotype × environment interaction (G × E) is usually employed. Comprehensive models that simultaneously take into account the correlated traits and trait × genotype × environment interaction (T × G × E) are lacking. In this research, we propose a Bayesian model for analyzing multiple traits and multiple environments for whole-genome prediction (WGP) model. For this model, we used Half-![Formula][1]&lt;/img&gt; priors on each standard deviation term and uniform priors on each correlation of the covariance matrix. These priors were not informative and led to posterior inferences that were insensitive to the choice of hyper-parameters. We also developed a computationally efficient Markov Chain Monte Carlo (MCMC) under the above priors, which allowed us to obtain all required full conditional distributions of the parameters leading to an exact Gibbs sampling for the posterior distribution. We used two real data sets to implement and evaluate the proposed Bayesian method and found that when the correlation between traits was high (&gt;0.5), the proposed model (with unstructured variance–covariance) improved prediction accuracy compared to the model with diagonal and standard variance–covariance structures. The R-software package Bayesian Multi-Trait and Multi-Environment (BMTME) offers optimized C++ routines to efficiently perform the analyses.

 [1]: /embed/mml-math-1.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/6/9/2725" />
    <title>A Genomic Bayesian Multi-trait and Multi-environment Model | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RXbMv18GewIEH-ML1nRq0SyetP7KLbpjhhbqfRk_vYc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460305","apath":"\/ggg\/6\/9\/2725.atom","pisa":"ggg;6\/9\/2725","ac":{"\/ggg\/6\/9\/2725.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/9\/2725.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2725"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2725"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/9\/2725":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DmkwE\\\\\\\/F0GM96wPk05uSc9Ts7vdGccpjmjxL9RfvsJOc0zE73p0FS66TqUEuPs5T5eCO1qKIAJ\\\\\\\/i2hUuqMuSNPJoj4Czr9Wg5hRnubwrWgkVIpefp1J\\\\\\\/ovgRMOO8Bl+959WQJ\\\\\\\/jJgMoYQYxigqht\\\\\\\/D5Bo3cmMMS5azy\\\\\\\/AslzyrUzUknLFCBZfeucJ0EXGsDO4rTEQDeVEzsVsuhdee9ctvibrm350CvCclgTrh231nCaBpPRS\\\\\\\/ezHcM9ySHw8eSPTo+mIkqaEzC\\\\\\\/IBC5aMGTldqqWqejfqJc2A4KLax9Vi57rbBQ30abosyecp4nGLDlNxH0BJ1pS5L18HpSb1j3+HznbWdWPfaWYpl7Xz\\\\\\\/DuWxG+6Xuj+ESLiPG9zQgvsgWiOexunM2BJpzLLQ+BtQQaSx+TcE2yMiAPzo8ZqYbCHEZaOZ0oNnnoEDpKZD2tqg2nB8ErKm7Z7eeXPO2tQxQctbAGFscGUPsFjAqkE\\\\\\\/b7vSBhioCtmtY\\\\\\\/gaTFCSVFdrKySglyVhjws\\\\\\\/hK+E3yvZkJ03rfk54Ljv4j7cO+cs4bQvR8Yk6jMlJnbmXD+hmp1JRlUtcVybLHhIlg8GnOWatBhCFom5Aopz83LhsB\\\\\\\/sl4e7b6Z7+Fr5NobgcMiXTCB10OXXnaVwExLVPo801QFCkEFdzjQAdRlO95ZnllNP4tYNKYl0jVcuc1Klp31otEVX6byHL5e+0D1wiNKlm4DgPGtBjlfuLR9QCFYwaDCAp4yqCoaEO2HMh+rqJr\\\\\\\/JhSq9AucA4QBzAytix5B8CXTUZDjQCYYPlgeXXEbRBSEeMgnbosz\\\\\\\/A\\\\\\\/8eibVncMDKGg2D\\\\\\\/Bdx5QxscKZHF9+58xtmKuGGt1JRyT7eKoi2S+dl5icKSIohIbkyzxS38XKYpJE6rQ\\\\\\\/nkr6qinUOPpnOK+EaFFmwBjle6zv02k0o1B\\\\\\\/wQfgFS8s8g2\\\\\\\/MSM5\\\\\\\/tny8uiAmeUBBdy4vghQ8Og+6F4RJj5AT1Y3q+CRQh1B1epYxQm5Qu9\\\\\\\/9AvEquso+PlWJoOd\\\\\\\/Qps7FH0HYqI1p1QNEFMzogRoabSEp752Bi+goF1XrDO1c1tMQ6uoyCW9Ml7wp84o1PMZ0zuTyBN7mPugyfW\\\\\\\/bih1Xigw8+ACRvIfKqZCMMJOIf8hM0RxFs9YbmMRV77Qs63TzO5JqPNOmULdz+6WQTMv3YlETFE0bPUwT7ZUBEhBg7ZSk0\\\\\\\/Zgz6EQpXV9arv\\\\\\\/d36YqipCBmDY\\\\\\\/nlkHCRpq+Umscr3ep5vE3ojVGVAv\\\\\\\/0Xqdq6SC8+2VH48inyIA6q6\\\u0022,\\\u0022iv\\\u0022:\\\u0022e3oMmvGqDF+oJqpTswOz4w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002211abda64e6a1f6bed345bce3c4d26dc581e6a1b7373c86dd310a0657716db89f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CE3hYpv58r+RmX4D07Xb2kFgAMaRzS5G2U\\\\\\\/QukeB8gS2yqyH3wKbxwrWQtQsYiL3ku3IDTHue8ih0k\\\\\\\/An6JNaK4YaScvLLxQzt0uA0WDU1CfDyufgXLv\\\\\\\/ZVMVYF8jQyO9W+0zXHhB\\\\\\\/CzN5TmGR2f7pMTb9R8Hm5+CQMUbMXnD3e5MrVZRdo3XW4i3FJVHyYEy3A9hRv2+OxLJl6OdFR0XStv+uJ1wgixqIN0s3isYdwe3FW3puuNV0FRStC8HHjE1D+QkASBQN\\\\\\\/4TAMFbA2ECmjdmLOXSiWwHDHb633uw56UGX3xs2gYOEahYwAc5DocoOMWTg\\\\\\\/o+YeXECYQZ6KJ73k3xZQvFm00A0xgERDoN8yn3DXaTz2RGqgCDLlJiv5uRaZm5Bw\\\\\\\/2MaKCswWHuMb51bU2OUzu6E+k1i+7n9Ce5yp4Ebw3aXqOK2YexrPvTnhDnnvjeaQLjJGbLcDqjzVlAGJ97zCwsFtOuTSLCbojYUgrnOLiJrLiW92jD0mngHaDJZ\\\\\\\/PxmQ9DGxs\\\\\\\/ZgWFP7MkXLfR0mGPHHL8Pl4\\\\\\\/4u\\\\\\\/DNULyhTuNunpkp5WMEjuNxioKGKohlaRm\\\\\\\/\\\\\\\/mWBzk8hy3E06R73Wgqi6NC7Culgw3aZTwusRfJ6gWHfXUqViJ0sb5FQI1RtCyQdirHbLhSf3wxE5YeMPBLX9ABcJs0pGvJErMRBVEKHV74HjzT4O5bk5E6Qtj5kZhomB\\\\\\\/XgXs59vgDZNYaW3qcBLRLie6GvBbk6AI2T3OpOoN17Xoeqow4vGZGMKy6ruwkswd0\\\\\\\/990c0lH0Now42O77VIB7ghpiELHvIMY\\\\\\\/dw3hesrOecGfxNm8J8IsRboF7rD9LdBei0vwjbItknOu6WEgYgyNEDJNv0RMccgjrmHarziMuYVAwdusag0rh1hhM5AoyjeYrJJnb27nwSrZ\\\\\\\/0koCwGPDwkE5X1Ej1FfG9r6r5qnVu+YLOHtQAFAjeeBE6jrf7IMXC5xQXqO9VtX\\\\\\\/GCL\\\\\\\/grt6RCOIRLd1UGHEwPbDeAN5r4QFRAW7GM2bDqDYm+Pvk1NA+5oJNsfESo1Pj\\\\\\\/4qMb7JmFaOXNf0F2JwpjwTB\\\\\\\/AnGAj7c+m1XoIi0DRIGIbFzp0mjx\\\\\\\/S6fkv2GWV+f2uysYlJkRdpsO1ahcBOXi2h\\\\\\\/hfotKGGerKCxFEim5bZAsi8Kz4gzKqBGVhFp2X5egfCvhZiCqVtHv6npnf3oqD3+d8Lw2VfLYDomZgk1mZgHDo1ItECyPhXqN9aM3pPFPlduyny1mjfg3ZE04+L5X+5wpuzLVq\\\\\\\/6NTtIw10H21PEny0HdoeaTP8vUi3vOl55yROBaK6dZqdQk3oeUvCPrbAvqgeTjxBSkCcxfVPIazHX8L9UIAQ+EMp+i7Oujc\\\\\\\/DOelVmpIfHxlUQkgYoK9J7g9DCs12S6hGSL56++\\\\\\\/ssxm3lwIfwZ+jFr4AVntKcYqqpuXeozWQKVYCWiLZLSmmCPBoEkM7TFXm+L5cTBP5fap63NqtdtA9aGpvsfKIUMNyLhkDcaZFTbAnPeT8ObceNzuSnHeVzUrUJrAc+zVkaRRQLrZPvXRyZ+aZy9TOT1rfa7jZLzbcw3r4yOeSyGeIheesDxpR9uBEGliIIO0KU97G492hzWPiHloXh3wQM8q++N1uY7quDKzrHXZFa0erpMRWV8UicYEhcIkhPmgmJzCgCusH9iQVBKfUSGzjefUk7HbCQMkY6s8YPp\\\\\\\/g8eEhG5+kpRLoYaH7DFMjS7N5NpfFz6JTptSc01AzRwCBvqyjaoNpe6JS1VufjyNTVuhzRyLlvbjiLD0f\\\\\\\/qucSa43ejchqvOGESzJ1Qi5RQue7Mwus0Xc62rqdJIKX0T\\\\\\\/\\\\\\\/\\\\\\\/cXKHKKvwoVmUdk5TmGCWWMSG3a03JhKLNcqFrQ4Nsid3Hk2\\\\\\\/Ys\\\\\\\/4tEJpeFhdmqxGV\\\\\\\/KSrcimCvFOSVfJkUitlJUSd8kq2giOP09fDrvSa5pk1DWs14\\\\\\\/PZ7W1vxKggSYhLul2Mg7Kfy1Os9iFv4VaOIp5SZRnl295vUDElFlDP\\\\\\\/aa5XbryAOqianRjGvrEjYmM2\\\\\\\/K0HcCfZnLG9xsXlDgT1spc\\\\\\\/36Ccz17IM0dLG07pcea6fm6ko388OwyMNs84LjtDeDARaIFyV5kY\\\\\\\/uhJ9KIKKnDPsOCkH7s0953Pnl4xDAE+H1FP3Xdp26IWmJ6xDBvgxHs\\\\\\\/IB8oCv07Y1rflT+v7JO9QF1sEtvwxIcY9ZsvRMQ\\\\\\\/2EZq5gRkHLc78T9yBV63JN2qBUFr3RzG558x6wCwBhpSG18nwhEJYM6\\\\\\\/hFbWcBr7iikRcAv\\\\\\\/KEHij4O70Amd\\\\\\\/4DUbPcFSSNd5esL6A\\\\\\\/4K0XfdQ\\\\\\\/TSIfFmJOWDSVt4exTuiJDtWGSb320ATH9Utd79wq+FMm3uqZZtDQGBiBcptxVSwir3G+TPwZWT0eiuJAkQ7gYia+MVNWb2uIu2sk5QyHM7fTvLBa2x5OLF5yDgHlh6vdUH\\\\\\\/+O8ObUYLI5bExaUnIb7dLRMHfpzXfsIB2rNeh9BkPSDiRx0uOOoHr1HZ0x9bEQzPVhl7h2LFIDu517VTK0RL57D9MAihmhm8HmjrjZa1DxdhWQsTEQa9intgrXFBxRxsfeivJznK\\\\\\\/Ml6dtxvS2FnKyN4yNgmNcmSPIqLfwih0SXScsp4WzZ1nUTc61a6R5TReVqUpnef1\\\\\\\/6bWfnmqpn708obsUhqT3gU2PYcLZxJ7OzcQCqCDZkgFR+uSGvozvnx\\\\\\\/FFX\\\\\\\/LhSXuuRlYuC6KS1ZP5XCYtrZtCZKX3yk4RQzdFJr+q1qL+MUhi4c8DH4hfpq1XgjVsTpXK5TcnT\\\\\\\/KMR46Akk9+8YNXukd9zxT3SG7yLe1aWxZzAAyQ4LqTQlplmfCOEiOEh0G+rm6FprVYbnnTJ4UknOsrhV70Pjd\\\\\\\/R2ZG0JBfxPFWmbRH3nq\\\\\\\/JUnf6eXr3G+mzjLoSNXdLb8lcryKC3KSbaEMjuSALC8q\\\\\\\/LP4G1u4\\\\\\\/dce8e1bL2Ai6OCMmrKyfhUdWuPkG6Xisjy\\\\\\\/rzeIw2qTHJwKclffZDUH6JtZzdNhRVxXM7GGviT4unIOi7uUuTCAKgQr3otd1Z1D2szaN+NTeHr+g09kO\\\\\\\/OuSdOew2uEjB3vxW2NLV0cSTqfiUGMK31hUm8o+daupES986mM9USSax5Z056sbZFcMfmf0K01fQS2qRvFxcnegfPbYLTecIagv7eUP\\\\\\\/J0PsZln2Y338Qa1BdRGpMtH1YdeF2ih23urpVW2Nk6wA2zSXO+EzkA0JSYLzjCmwp49EiZz7V3\\\\\\\/H7pgS\\\\\\\/pSA2lm\\\\\\\/19XhN\\\\\\\/+XPYzltkgXfaBPk0AbIyGeWI3PCosKGZdjP6s4LRbu8xaXf\\\\\\\/avJoafjvmkDLieGh5J7pGGoaxxyKQaEef1A+ItFKy+o01IgMO8A9qihq6RZZOq1iqmNJ4LXHHEVsuz7A\\\\\\\/kCWqr7qi4hgqtDAz8W0Lw9gobV6SdtAuVpA5RKZfRV71Yuzxl3GkaZCBp\\\\\\\/3KqkUku5ijRlM8C3OIJpfwzwQAvUAL5XMUTQOKppjvo0U3oSXtbIdhZrWuPnH54thqaYemJQw7r9WB8C0v1z6WRrVrIW6ZY+g0ktuC1LsyVYPoV1He0a3M2rMtg6pLZkI7dX4MJ7j1XnBYlOPtBoSq4C5cPZyNhBfYZem5PzaQpZWLs+YbDVdjevWGUvRTJN9tbJqdGO8zYbmTRwrAfxyv5Pkj\\\\\\\/1DJaoBOzaaMEvryPd+9ul9K5uUJiNO3eX8q\\\\\\\/iuhZgo+vHp61jQ6iKmrhYzBogrUoy2v1UoV7SG\\\\\\\/h\\\\\\\/SEkIzjvNWFqaplDwD9LUMy2fW5MfhRbldJCchgnQgRz1UsuG2t8KpsK\\\\\\\/YAJoo1vCT+Ly3shRbwUZmNTYcPDPZZ47ekkPP4bKNuxoDdaOnSd37oLg42cSoZd0o12sPQvZSfCsIEjOOqOn4uZpNg4xJM669vTS90kvdxP2Wou\\\\\\\/vEqrNLzFl\\\\\\\/SgyJHUmIE4VsyPBkv7PUic0TTSsn\\\\\\\/li2DdJsjoVv1RhOyVRQ7JerNn4DsmGEkWP8cMaY6H3j9nLWpufWeYQ6lmyaWCZcR7Ky0CoFE3RnLQodV1NoSszygIQZyh\\\\\\\/SRA0KyzNH9Iw5F6XG07Kk5cmmQ7GcpQDLe7TzHHLFCQA523zpvVu4ieIux9hPz5M\\\\\\\/lU55f4aMCMt6k8M69m8LB8JO0mvfQbivEgxxE0Rfmkc9uFIeSNTNcin0xgPSs\\\\\\\/n3u5njUDoUr8UCOxiJq730S8e+6vj8FoGH\\\\\\\/3GYeClufTozg842WlQN76Ak83QHcBwxcM9zFiBjQ7EQFiftIsBxow3yJNJ4cEuoLkiJoH\\\\\\\/Z0gh5HKHJIxBynluPtFXWi31akdaw\\\\\\\/vQgZVxiYiN4RVrZUWncvcleJnSA0Bushp244Qz7SdRH+FRGVTIgRWnFC3lxRWvHq9tPv+Bb9VWG91vIL2Aag5fwfZbzqzPzeVTEzHsmxfysDwjTjGqcLp5k5vaJ6vhkahIWQ\\\\\\\/vsKtoDOVIMRSefoiEnXhTMhX\\\\\\\/TSd6FtlsFF\\\\\\\/kLnaPpfzanr4hBtc6QuwkG+jc7ysBETo6eysv3t2uUu60Y0j9sGeaPFAsbkx9pZM8iiunLryarw1Sx36o\\\\\\\/XESboqe4zkbt52Gt6JPs+lg03SYmY+fdgsqDy6HOkjfeoWlzHMklArT6MVRg8oPMicUDx8jhleQXDz9QyTxWCAA7MMUXT5ZyCkEXd7jA+E3EhNah6bSkWHNaxJ5yaxpY\\\\\\\/9UBJG70HQWP45ppXgRGF01p5nmHmXHNxgEObwjFLhtTyCbXVPBEztNm7\\\\\\\/wu\\\\\\\/z7TJ+7Vfks8qKKLUUWJdZ5VakSrXaJ1bLJSghFrFQyPy\\\\\\\/P67S4BI944IrVAXIDs2nS78NuMFPw5Lg43Z5lPOgxsainvpHZbpu68DrBqJ\\\\\\\/6P8MqgmnWGcZoI8SikhkLnSGIkBH9iZZpY8Mzb\\\\\\\/oN3mUk6qO4DCYW3nIfYlF5hJwXRmGUUBk6twi+NX7f5BRT1mt1EoMI4Bi+YVWLP8q4fduWYvn1wMsoN66JJYg2nucQBV9mF8rXNgRWGXn44Q0e2zUuYHo87EdxjMeJp4C1HGTJiWB\\\\\\\/2KHoTpngjeCHUJEJyCXLw\\\\\\\/D+6lOXTZdIthajAxWsO2KfZW3UXWzwT8LtCcaJG2v1jOGatke0j+KQlE95TTpF7KCS5LSfVUDL9ecKIT3c4JhqSexb3yGeTy3MVE+Aih+pa3IPWrnEOfkCIiPEgYL3XDpn8lWayaOLiw8NDdCIs7bJX\\\\\\\/TFUsj9wLD0jRBrliQzbR1qXmc1Kd7DfhjFnVTKHgzx3OA3vTqYehymCOkBHjJZHd1wRvmH4fYSq+0N\\\\\\\/MAKiHSewuz2acRfKfa7odc7g5kgd3bIdo0d76eOEtJZv2r7SVG0VkhJgrmNgwZNDnVQJzj+DLTQu62M2DYUI0WIbmk2pcgkssVGlZtNwWhjeoPk2F2Shs4NpqR4IsPFY5kIwuBPyAvPVK42Guj7olBym6X\\\\\\\/ss0\\\\\\\/xU4pFgDnxN\\\\\\\/\\\\\\\/4ybDHtu6Be4CIlvbt+FhHh6Vegrtu9CcmuZ3ARAWKDkCYh0hU3+InTunMkBJKW6HJ2QaSWuh6rIy\\\\\\\/gBnA\\\\\\\/pWW+ACp23p9FQkzubgobbSFXzu0i9OkIu5PpVwVpfGaXPL\\\\\\\/xUu5I9yKHttI6biX8GG\\\\\\\/sXxelFmrPZ4w4a48nN3RTJFfXvBYn+NDX2R5ErRHUih9R5DaoF4aXT0Q7adGkUVUL\\\\\\\/fGGjZcdhtNLYYc2uwhrFjgtzdKdLsPwXR1zYeEBBQwE08\\\\\\\/ElZNWFh\\\\\\\/meW0LQFhtYS8hx24mTlj99tHo5wpDs2l4Rg+o1roeOxMtlcP9+LTfMNfwiCN3du9mskcoyEsuAGCFscDC0L3ZDYzZ1jLhLeg+\\\\\\\/\\\\\\\/mNrITQthIzypxC+zQ9mfnzPchYG9JGun8ruk470868IMcq7gMitguv4jgXnPa1gjsuduOSokxYXj8jlKLrthn07tgYGbMzdV0U1yrjQTxTerOPswCWt1\\\\\\\/T4vxiYWrlvzrrLz19KgymANq\\\\\\\/7oN3ZMmwzlJ+n\\\\\\\/RhLDHRuxrdJwHqm+E7OZs+xuE9KsVIPe8gnkpN5NKlACKKlDCyzIydQAN35s02BnRAORUYUC8SmZhwuKtf088N58wF66opCFvrzNzxezzfxjH0x\\\\\\\/dsMjFOHy8rJq4nNIzfNKdzvfUbgHJtlaJ0tDgOj6clTbqYp4Rf\\\\\\\/ixwfj50oO4lVQZcIQ8hja32LgnNMKKk5ilwqRzu+GDVOFyEGnJaFizhtcKbFF0hl93hMONSsF1ln+qCEGYJewIdkeCmy4c46INbEyBuRCYOS5ws3LDlWGv2nANr4PehC\\\\\\\/FEEkL8Ik20yimy0MuFPcekx30ds6fouqSb8xGKm5mNP91+P8\\\\\\\/tqqvA5k3SW3wZobsnlCGTAEK6IvYuE7Hb4q4RUuoftaB0NLYHRkbeqi0Xt8dXpSs\\\\\\\/PrruODqCgXeuXg7wHSiO+9AV3j0sYGgeeGPZgTR13F9Zjck0yuLwBSk5TGngwb8jJiRjLV+2UGMxKVCcEaddnYfNSoqg+9O0QC4IHOkTy7by97QUesCsZEnftx81tLz1cM9YcyoNhDxNt7uIw50Gw3Y\\\\\\\/+vh6l8gbGG5UHnqrYhqwvR8Zz5qNMMFxZA9fILR66eBPgHlhSM4Zs1wNrXczGy8Z\\\\\\\/OLF80Bh736mNKbzWQdqh9\\\\\\\/KTAeHlh33Y+YLi7NVaZ1NL4hbRK\\\\\\\/nMRjB\\\\\\\/7JYHZYHeVu7Y74B8qme8txKpJdB1fuMCK8ONFuYtxU0ACiMYqEO5WWKfbgqcaVJDSHvqO8nHr+bzIClYIDqiLT7E0ILWSXJNXYxIcwtgOoB3t6pHxrry\\\\\\\/1wBiYlI0gnzmLdmOppoeq41E62W8JE21ZM4Og3pSuMpAA5VIVJmyKj+6Llwv13AlpOF+EaNwFQHMKDYuPxqsAT5zLtY86+XC6sdEZSghoFi0aW4nOyjCJLBiKbT4iYP0Y9LnTxtURN8cYddjuqdSHS1qvER0+b3bbtSAT7AtLYfjZKC1C12Bqhpx9MT0o4aDNYeK1XMuZfaRdoH6e0DWzNJeUl\\\\\\\/k5LZuB1EBZU7le8LAC2EZhNeXjVo4g2Zcr9z9S7CC8qqPgd9PYfAyAECBgJMvNSkposHajB1AgikpiHyntbiGz2BfK39DWclBwMR4VqH5kFYrk16tBz0dTYfy\\\\\\\/K02wz3QBYOvED\\\\\\\/xjOWL6Jc435KxJtyMpxqBorh00unhGXFsmv3ULgdurtHts42emYNi9sdL9w7vZdXHDP8A4kKtiZjEQPUoFgLwXu4ZyPr6djMQuQH0sRdqWDE4zwjp690y9vKUcAg9hDaqSwLNBna6nRslIJsVErho4OAnlFNI7q\\\\\\\/iOh0RxtNlg9BZrNJxCg866DqOLLH4G4Bd724g08ZhV4vHkJpullWebHAfFbl\\\\\\\/KV60e2H7vxYfqNOPxrfCSo2Qn5I1w9WNSalQCO66pGzNHguKl\\\\\\\/7BiYQ3QhNZQp2wIoymLS34vTZjXNUWTRkjRPE2mTm+rCYgJHAKhYtOIlBmJUdpSpdKMskQZW14VWO\\\\\\\/AYbAOPOxsXA2ypFQzyLW4XASO80FvMODWzUe7znChezO4NGhjaEmjQ4uWRsg1Zmeowe9rHpujDqM+ZZcIRiLPhXgUjs5\\\\\\\/nTtRIov3i4u4PhJ4q8WmDNvUu81ReRYfuO32D8lwFpy14w9MM3wmbqCam9uiVQhRYY3z6k\\\\\\\/HYhYyJM7lUPCIc2dehlAlyudvX5D6QJzh9wZBCoGRv3LugEClUqq4zrrJ++AADx2+P4t2u+w70jpUNrGoEQibbKbiitrRktUZC4ZQwLKR607TpvjD4pqt3kuo3xqdzCQFa6pTXRZyH\\\\\\\/WXB5f7qLXWOSjlm1usep3LEgWx8Ti9TwIjowHvNQUuPywZxJfI83R2WzNlX4tnPkJbe7kVzC3\\\\\\\/KsfSMzuYMVj3f4DL+cCIyT9z2vPcHk8265IjjTYlLZSNUJumVZaymwM1AwwX2G6HIVQdOMNu26Rh3SAcpOKgB+\\\\\\\/ZGbZzSvGSW0Q1Ul7LiWwPFGlqTm9IZQWyiZEAqARMUX7Tm\\\\\\\/ecHFBgF+S7J8m2jwPcNlHw4r35j\\\\\\\/A2eapKieTtpxO1V51o9\\\\\\\/69PVGT6GZbh4HPg7m3xZFZxe\\\\\\\/hXAvVxlPD3iGMBINrQXKvoYmyYrIvK+bzqtryIhXkzyQzDQLARvBlDlfGpcnOfQyqWxTUyqSAUG+FBRxkOEgUCCW7sjHqZSpxM9fzigDioc6jyksofa\\\\\\\/DTnDO3pK8nBdN74iYXneRt7iQennGHz9zZo+DtgpSU0eoU7NGlzfln51GaXnlPcGSzdvt6ITxRubo2bbw5Fv8qZs\\\\\\\/DRizxRkFev\\\\\\\/ypTFndsIQMy0BkntuNHw\\\\\\\/CQoNORCbqZQ5PCPQlrwJzrH1dg0Lz4h5PAGfut2\\\\\\\/rwGONy0YDQTXDzYsRcK4BKIkAKt1V2A0fuXh36+fmE690fLQ0OfOpvvIedeUvJx\\\\\\\/pmCdE4Ik\\\\\\\/YrFRNHBK\\\\\\\/kjMWqktrZRqtR1nZeb279Oc81qnq3GehGyyI\\\\\\\/AWdZnm\\\\\\\/\\\\\\\/S258rnFNXFZBLYVPY\\\\\\\/4cL5tO\\\\\\\/xLXCd2EjnFEum+1hqOeYs5tSD3CCw5c+A4xvuUc6qnrshzuspmaf2Z8Xjt6Z8behbK7deFpKRk8yLQD5ymrHJQHvl\\\\\\\/iCX5H2yqVnumoHU89tqpZ2DSMajRhfRUxTRow253+scuvjqreptc\\\\\\\/saZ3M9OEhz6v5+3PMw5caEj3nrz3Z\\\\\\\/B0nW6oKwRrS1v+R1R59QAhikrzi1R6aD0Af7nAwkj5Hn1OXTG1Em1JDvsPKvxxX\\\\\\\/NzuKBggffEVElIloc\\\\\\\/oWDWnLboeM3PjZvAzsEFUCQVH\\\\\\\/7JyMUKTS1\\\\\\\/Q9\\\\\\\/P4t5opBrN1PkiYyURXLHgDsuOuJp7GnlcyXQbGrWNMsrw0pxWfvyu1V16lzEOTpx3hOhj\\\\\\\/r0vuvqVDdp+83oJqiAK22YJVPTYlXrvzucjm88lymcTvQvPTPKpCHP0WyPT2iqvpBFZV0eKiKsuJfIhLIe14jdD8hhhLYA4+L3Cd59CrbV8BSwpLkLhz2uGNbR\\\\\\\/o1o6dZ+pXiwZGl2EeJrRjKy0\\\\\\\/QtLo3gDRubPr4X9RHpUaF7kNQm2f\\\\\\\/P2jdq5UnbAN0UtMJlLUOF8q0U6CUnmGb48YtjKxPpwfgBhfIf0v7PCstEOo14hXHbqxOWchpeTWy6xTsBSO6Srb0bIfh9exTKMZtedqA4sWd+DChPB\\\\\\\/8uZPgh\\\\\\\/GMGzdqCLvkMZhrGdX9v6H0G16ZUTjva3FMDrSQgdG2B05fBryN06thadhZFpr+kB3X6bC2UMI+qtCjJ1ZwrL46EfPZPIxR6fvXiadVJDpjvcdvJqnlvGBXY7zM2qhmZ4D0hlDhgthGn94g3EIQMKENO5xvg+sq9MSdiritK97T0XqOuoFTG\\\\\\\/XQBiBlttut9mY6TUqgpPZVWh4vVwRonmkIo6PX\\\\\\\/UaP3jcvKfdbPR88C9afpjtgWszykur7GvLJ1n4tU+gilIHa2rd+vRwn4IMX9D+cSFUDkS55pHuxaeynSO64eg51bJhhd0WeEkRKDp1T1LaHD0foLdTHw7uqFDu6+NJCC7t57rYDDLz2sDAVz2+XrJzQwlw+TiOA3KKEe6Bc+4jeSv+4SENlUNXcT5btSnArExfhhJ49owuOvgBXxaqnCq7W5ujNTjIv7a\\\\\\\/k\\\\\\\/6hefzBxbmTE8GUfNc1j5xi1sfAG8TebSskEloUhOJhcW8R9qIlexR8lbzlbgkEphn9t6lrElt\\\\\\\/xiZ1eBZEJuW\\\\\\\/fMQzYWOIZhQwpYw1wjpaBLHSRwQZmxltoEwI+SqUeV\\\\\\\/b3NFymxJ0OoAsLEARaKVpeh77RDmaos7CTmQKASX55LP\\\\\\\/jjdY3I2V7zLeA59vJrEUo62kv+xiKOiOlJJ8ybsMXImZH6wqpzd7unCH0mS7VfmjNy5SRT5rGhMz1rSrzau79xmt9SLrpcd24R5L4R3qv7Teksg0uLi6Ut5toMZTmxa5zv36l6C0ENrvA+fVPAhIGUUkhMxAYwuLSOR7mi+1QTLSLwNLsF9D9gQhIFOSXXO9pKMtUehyjH97yvapqj38d3BYKvyXekP+KKh4Su3nI5teLxZTsVSBMpnr1YBk884zSWE3Knsofil1rAUe1RMd76q5Bps94hlVFDOl27dbJJz5RVf7EPh9nheQnUpdZpLwg72089naXK0jOfe\\\\\\\/3FlJB1FdKLkAwkcVH9DJFDcplUfyUS+DLf8yUt3BSmVOzFvffkc2qr4ccrB\\\\\\\/YvV1Ri73dnB1540QxTolBAppavdSUIzEOFTAskqGJijliLS6MhgdCJTzQPEUAtdOXoVDa7p0SUGMf+ZUaHilDF0IJ4zWcV69ljuLj4nQMM19k3EaVwGcm5TYwgNrwAqKsFfBHNCBCLeAaeUhqPGQzJ53q33W82TR3VEMmmvHJCPuuER7prMnhBE2Zudjz41rzZW8mc23Yklg7RW4u3aYUq\\\\\\\/V16VLIn41V1ODo+cPDxW8w56Ew8+FLDOcUwByFdrcZToEALKRefPanPBBsJnmr983th86tkYASPUbXDB7Fu73X6OYGpKBGA0pSfsGl952LX7\\\\\\\/8DV1hjGvGUqUG3gv8EqS4bf\\\\\\\/rJRRV9gy7N5sCHN47FTPlp5u8bAS8+EyPXDilUpwBM8GPTDHZRVvBU3S\\\\\\\/QIpsHs0UuSw7UyTBLQJOdM3zAJEBxlePfK2lbWx0FM+A8LbiOZLQMaGjLkpeQciT2VU0C+KLZ+P6uyvCMOL6cGKjFOL97msHkRiDOG9cKXGmlFhpWTHH8d74GJ39ocO6jWWGkvE5LmZDpFERAKgDY02ztxzUxA6WglkZYLSlaeewF6d6h\\\\\\\/EUvyHcIzT4q1Rwc9Rk8SJCMeFVAdToJSSUqar4C\\\\\\\/0zl9BljrUs4gCCDNhYYWa9O5D0NH6S5Jmjgzzfyg0W\\\\\\\/WedmFPT2fMNvNHYVB7UaCWA9iL5DvunT9x\\\\\\\/GP6\\\\\\\/LdeeBZ+CgkZq1KWdPiLnj+iFOOG0W3+n3klLKxxZfVQl7O3Jmysdfm4+8qEEPH3U0vpHgVoThv53KR6UlkgKIK+eqjPhgXChKJ2E+CjiJwvFbioTSkfMZiY2I1YI1jfR+emFfbnK3R\\\\\\\/P0xuwuDNhHy2yKW3vrsowEym4mccwlU\\\\\\\/vUQO2BwBC6EWNopScG4dM\\\\\\\/NledZpgdGBRAtGEgfjBN8RNpp8K8pxhGct9k01TN5TPObETbcVzHCn4ptPV\\\\\\\/foPPY5Up694cVk9o8yDjCC+L4hFz6KnpXUQlTvzsg2VJ0p7NBRkNoqFuCXMbZBqkSETwJ653uMbiFuos4bfVfnNCTUBlUGevIUwxXNIkT4Qw+qrWFePv8utpBpj2nV0EKImvrLM3ap8kssNp\\\\\\\/lf7xengc004m3gRU3ka3i0YxxeVPywe54ik5CNfUUo0adJQB5qh+KL0OkDLiOHC5fm22sB8GwdeQR02aIYNvJ6OH\\\\\\\/Zl4+H4\\\\\\\/pk5kD2S8SaeESAfxseJlU\\\\\\\/0lFAoiF+\\\\\\\/NCLlIm4vvs4N2tWRGuFblEuwW+PwUhse7ep1N75Xj3\\\\\\\/mNgFDyLsJreiYzONLqXeAJSVpESJeGDBOzZCLFc5xEXVTUBqX40G9Dpu4i\\\\\\\/3o8qErqHh4hTlTxTLPM1JlMJo7tqoKkPTTmX71nSc9sqlBV90grdK4FfVfNYK2sx2fiVGCP5Ay5q5ruvgC9ClEbg9al6+LA6Rq2vmHExwxKqLATiTOISqBOXrNLO430bIw3bmqzkm+4KnsLh05KNr11mPCwQC23nXMnqLslc0V\\\\\\\/wns\\\\\\\/6unh1PqzS7VOFB+MNS99+tnNyY5DnMFyXo6PbY6CflZ7u15S5FBf3kLnrzl0lUzEOPsixNb\\\\\\\/BoIJSEGvY\\\\\\\/cGLruLESFi78TJuJA+6T4ZCMX4jiDcOVPOz3dMiX2Jhxw\\\\\\\/hZ\\\\\\\/i1VDKZ+xG92bY\\\\\\\/2uIpu8e4QEWoG8GMum5QCpDauOAfyBbFeUcmAkpOvSLcJpxpBRnGkGzm9EbrmJNxgrMXA3VDVppZJAkjhCKZWM8tOj0w7oPO7Q7bB2S4Dfg4HvTOTdV+X5ji7affQ1SGDXeTRXPjPu3iRM\\\\\\\/G+A1oWnkxqzItAH2EKf5wzZ\\\\\\\/I24vQ7nsD49XgaRaKm2xLyJ9vpwTUtCDwzS7xkcx2mXCWDbYWpHFnAIMBQegl0asN7GyxVJho69Ww8\\\\\\\/VLua2Fesi4TBN\\\\\\\/sKUszVDVPRyLOhA4zGblhYtNCE0AeSKJHCXh+lciWPNq8GTp8ruHzObByvcqUfsaY73UE7W3EkwCJf8PU1FF2W9xJTN8WPdc1l8jHDTQ+nYVYBsMtwKU6x+8V\\\\\\\/GWLl5ZqvBqdcNr24Op2Qv04wblRwDERKz3sJKKbMLwUmrtUd9N2CloVSiWKAfocRKmPUjwvKzwsg57OgKUy0RIBtFR2rcOtczFXTY9RYdmR1t8VucD3uBdiI+zHvrzHh2rxqgFldiOAsjLBdP7WN2b8GJhflBZHRp4chZSW8XQ6YLTCAgnlqypsQIlQNYE8z5\\\\\\\/5HYqsj6enU0j3yvnsnaQLTkAKTfEOYKsTHv2YEOEMelx9YfAZosnpSARmLi5k9uGz+vjojeP06BhIn5zTT1ejzjF1nxezEK73VQSKSuexlUNFEMdDz\\\\\\\/5NaCNl896oK47wR1vZx75NAwALEntzFGiHkU3uGEhLgPGhUYlPyKXm2WPKu+qNIOKDxF8wo3DihMxGYuHRGHLk7r1woHG4n+RmB+HENO7Y7Ka2Xnf8x5r3OqYDSkK7nysllZsrHBB4Ywe\\\\\\\/FZaRGIw4VAb0aPL\\\\\\\/2wO1YwF4zjXejFZyJ47dTLYXGzxOo6\\\\\\\/sSBdILkZKdTj2R+hKcekkuA0yhJ+u3s9VPGoCO3BA5uOFwPlUoivFWoWdnjP2zBlwEF4H0ofGoyUvPehOqxkBCDpOw4KLM\\\\\\\/rKGOMDEo863zxOisupl3Jl4AVkYx\\\\\\\/ZnwvqkzgNDPinuTHtbW4x1DY1l01MS9tZOZfhXyiZOD9Gyo1pw0fGD5LoOvs3RH3TuzE8xsN1QE33fkG8RN+8rfuBZ1AszqgUtmBoGmgGx\\\\\\\/INoOSwdtndi7zaBSJIDNQlBodD2NTTON7+vs3\\\\\\\/8KcTvWmoaopAlL6+zHXgcnmJjRSWK6z0oEvxpPIdzAN3gUJkmd1kYiFMD279EqNNKFc1Sh1THpHrnrqkhgwDpuoQRl+QNebSZv+jXr1fqHxKf0evms01bFnpkhf\\\\\\\/qHc6EcgDHB7BylnbtJb9pNV5lqz8PvR0zDgj9Ih7O2Pf\\\\\\\/Ohuo\\\\\\\/gpt5dManVHSsWnFFGcpOq94Q1pkiYiO5L7GfUJoHwzWU9YqEoVCb4ljcmkm5sEMpDaoMUn4UZEUtaPwiZ+Squ3XOCGsRlgNXCeLiZKmYxNAf9TyYaRnjlbyLjvwuwoFAWf0cfVnhfaZkmAg\\\\\\\/JNHikhWy2cSzxo+fVrMmC07hyQp6fUVijdlc7RJY2pL+ZUJgtZGK8Nb3p5dKJCurNkIbqnsvstCkEmAOBTyRvvV8nNFcvM8x4xUZaeOI\\\\\\\/CGhiIPf+8pDNWPl32+duYodoFM6V2zTxQqw21Cq6mL93Wlc1TcxPPsCHWxfcuDT8\\\\\\\/Lny0eiw0zbeuWgzfT+kWH\\\\\\\/DAfij6vAGqIj778GOMVi8qE8zM\\\\\\\/N7CkikJqRzvUgzqTDD17vdqxfuUjZVWJCrGvktussurrSAAWwmHITvaPj97vR8DPeRDap1MufluH95L3ssRtclQdzHjopFDwKH319S0BVxdy7RzrhCDGMkITiKXanAEzcyuo94Lt97NX1zaGVpbZzHrRBz+S\\\\\\\/WdbzkR3peRP8tRmc9dUbCwLS0X5IXR5qZ02oxU3MZ8WyTxjWjZugC\\\\\\\/HIzisl0fN\\\\\\\/SmjNwL9gwaGMaWblGAJlj4hn2aqqB42mpo7E3mmDIumBsk187Kqzmyhxczm20QFmrfQ2XwFqqhQgg6PNlPg95YViYcWY8hngeHW3ZddJJPyLFBGJGv41NMcxKhSCX\\\\\\\/RHI3RIi3u5GtbvFKUM3Ahei\\\\\\\/\\\\\\\/oDqfvEXW9MZ5wHtU1pUZrWC8ZOLCB4zGd+S2ZkzEL+Li\\\\\\\/qMMOuKhngNfIqm4tTtZGOtrloODlEgncz4gY3iflxBnT8ijPKLX6XNnHAyt5t2VSXdh\\\\\\\/Zeirv21J7JF+pU+SbeV0pRCpD7Fi8If+bfOyiDYhbwMjhRDDQQKds3qUzkOlTT1\\\\\\\/fgjKYiwGO8fWF3n+ei8shvFRk+Kr+oDtzxq3DY41ghRcTRE+06YOi7kA+aK4KpukSksABA7ZNYz52p3ivFakad659QNTaDxb7vcMSkb+uI49\\\\\\\/wDnY3sTuhqpAEvladvAIy\\\\\\\/Ik0AHdwGelBgM6hUJwVRa766yy2rEfd5rQwzssUEd5Rn4lgs1\\\\\\\/zRuDlWAU2YKaB1dPuLcjXkQrbuKdVY\\\\\\\/hER11fg7dQHrNjsl2EbejdmfC1M\\\\\\\/ddHWBKGY5ScIFQ1mGGuSXovSrYxr4AIrtKzpEGJKZ9j8q3fum2BOA2GQ2BYc6JXZaSp8EYeTk8HhKB555oEv\\\\\\\/Ucx9AldRntaAiuTSbdlELaT6XGYjUGkRDRMJ4irMsmzIBnPiYCt84gjyWzgywQqIpepLkok7WeNEPyf31nAwgnaIGQMMxlVtyj+IBZIyvB3Xo4GTp0duS4xUmHqfSqw8\\\\\\\/L\\\\\\\/BX5j2FmQ3jrba4r3pSjcxCQFGF6L\\\\\\\/zqyXCIK9++9lBjyemaxK\\\\\\\/heYRNilUWaUTq\\\\\\\/06LwRnkaBPEuux\\\\\\\/1sMizEOBtL7oelczy3Ju0YMTf48B+W+xIBEzEyMggVZkwz34DKYSBHxAdLm+6VIquSUOYHjkVkC4w3R5AETdSWt1TGXLIjwB0oYW1KCSts9svnqwDoMN4yi3Yfn9gun97VIDAgL55oCnisW1PQSCHH+jATjpi8LWvJbo0lV2GMYYZp70DJgW9QO2K9C4dViqLOUXOpey6X5MLWYxP8R566sl1jKoiXSzGqDPYjtGwtOxmQ8szBxfBe3pm4rEv3\\\\\\\/pys+wkHA9xAzcoe6iId3WcghGxn3xY\\\\\\\/hCtMj7b5xfowttMT1ky6FwPgSx1nTD2FHkmmNz2BHIMK2wo+vg2tmEWptINBAPvxwgYJ2A\\\\\\\/LxHjf\\\\\\\/KGmO+o6SU7X9sBzuvVhcNjTJv5bjaVmPJnaMKfFikIH2ahliHsPy4SmoKS1oyByEGx0j0n0wQYF\\\\\\\/T9pRxrRUF8SeMncPeekukTpkXBLgfxBLaGUAjMc1LXt2eJHJJ8nUBf3\\\\\\\/XxDRMzcHOf7jA0QKeltNHoHon1MTHFNWX25RQdPvbGSPe6xniSZhRYQb+V\\\\\\\/OHn5S23CV70YScsr4VcuvbwTc2cdFJ3M2PAhpcCbq7NLt9K9wev+Z4dcwF2A3ApOuTGq5mKSh0gTmxzDsdelAThTbBNDn0FH0wn1csU16qzCz\\\\\\\/80FAal2VTER9TjJjDjm8XxrSwCk6R1uoaY8hL74wm9TVFz5GHyTrnyVqOXVvd8uWt71QTYwfmJYTO\\\\\\\/seCWbcZFusV+\\\\\\\/Vjw4TPKl9b3BrY2U9nOQszsts58x97CJsYACMYEY\\\\\\\/8Z+B3bNmu6OaahCscjJJOpm3fQKbIxsnXyo1tGOzantBT2jMz3sdsHrPOiojtEoDlFdFkj676Kj8zbDZWoA6bLQTu3WqO8n70j9latD9jH8sHFUmFS\\\\\\\/tO3HC5G6tMeEbVI5yyvFPdo3uiL59ieL5kWsspBthqq+uRlnHrtSqz44YLi\\\\\\\/f0L8oshpl18ERamyaG2+FHjceOYuMNTOV3sKy9Z7jdVzofu9kEJ5x0Z6xStzUC6fF2+Q2vN06oGyojOkXoq49CqZIIEEAOLu+HkZuH48kK8Xoe1Ed4BreUYVx\\\\\\\/GZ2T1xQTPnhj\\\\\\\/+yHj9GGOW2XRzO7ESPAPBtMCp+Osm7uJNAh24JHyAy+trF1SC+gMHHJIHxgXeMD3lirnxjeqsHJca3j2zjFvu5kV7oDgDEGL1M8V\\\\\\\/o6HFKxvnbBO1RwQXRAz4H9a6bYbfTMuEN7MURQSKHPf8BkfEANy6AdIVxqnZZ1WOtrnMkmzV4kkpbiKNmSQZeRBOMRFvoiWWYOMDNyWrhPYzvCE7Wi2rv976iYFdvZOZwJU3jOXW\\\\\\\/upQf0L9ez6qi8ObgRxMANdBOVbtwPESvSXksLKKquDf1+DnXjcmac\\\\\\\/RVaGY7lwhB6l3H05ylip0kejigE3v2KH7sj6Ti1mPuIpH8JhxBDzMvC7gA\\\\\\\/gA7Ec+\\\\\\\/va7aX9++rChJg19sBXacrHKONKc3YzlP3ZqmyZaR\\\\\\\/Osaj6yw3BO+hfj1EMcGhXT4HSoCupub5JvvzVP1rnQE0k8P16NfT1KpHtMsCaDXfea6gKhsOcrMlxV\\\\\\\/l3Abdy+VCslFA0LQe0x4A6aGTxJhpAE0Z4qIgTOLeQuK0IDUlTzO0ju9OnhGoMHfQxpRwrVWBR3il8GadzIbsoPwVF6EU8kgYsHuRUQjYEd0T75\\\\\\\/4GgCO6i5XMYjpSCTR4GTWrnCk2wISmz+rsfTUYAeYzIMz6iHAY9lu3F+OFDFhL8nT+jzrMaWQkxtTiIjK08taieIVXHj7sofGn3UJ9BdXt6kDOQZWaLCbmFS9K+kIMo0fHasxSRc8BJD+tTY310XatkCif6ITq3qPW8k7r96ueoWEtqpU3K3Rhw2zCNeGvK2QjD+uNrhoObzz0D\\\\\\\/sp0youbkOjY0oBZQxEkngq1yp\\\\\\\/bis5I5jZVTvKHG28ZGqkpy03\\\\\\\/ua8wCcxAkC0Mwky5HXfXF+ez8fUAct5fGLY4Nei6fpOFcmZK3eOe7W2u8vyGMaWO395ey9k0FUQtM6rN\\\\\\\/vhJK2Wsn3cyBR2HRxeeroqiwsFSi2IYUetJxoJPuNbCejswRYCoc\\\\\\\/rghIKtcDUv3NurOxu0FgVzO+Un74CeKMWu49PEqFQowM0TcwFv51vWC2MiejT4UJhyjfxmK1MCjTURfZcAoFE4RxHn0dLfbQjuhG3vzlwg1qO4YfTQiNIAZ5kb7j30NVvKDPR\\\\\\\/T94y7qEyWXtrcyUgA45DKx5HS2UyEbJcXJ4auv9b38lOvwZ4xQ4xWT\\\\\\\/PLcavJf\\\\\\\/8Wi7JFZF0WWQlqjLEYuy2oeaqqIPrRq1XXPgdfswt+vdjJZTguIbCW4s+UK41cu8Le6z+x030Eta\\\\\\\/ZzafI6ZPN0tog45j4SxkOPAOdPfoDnKgaZVQ7SEtBrCk2pmvivG4jJYealA2EVxK82Ahqdl4\\\\\\\/zicgzp20pe7zzm2bWO3O6cp6tl222dIEgfZ+L+eB7fFnInZTrspYpeRq9o+EDAXdiMsbw13EnPEFU60pLfOFGtxhvUBSOy6RIRArTNaWQfyWAz+4EmPmL2DNBLG2r9W5bxj5hzE6KQwPb2SMrXwcJO9AJ8ZTE9fRjfNutKtvFBvB20RIBYBPWP\\\\\\\/IkdcxOCidvA2UqhcUN6eWxSmIArPIfkkK1HKYXV3eDHiU4LNJKFh9YiusOCOnPoQd7Wtr+\\\\\\\/vyi8HFyoYKiZCOjBghUxmW87hJA\\\\\\\/ZnFW\\\\\\\/fiSCtQQb1MHl95FwEXeJwOaRgkYEiYmsNjcvp24hyox7q\\\\\\\/h6hpmZamp+rtUU8d3F+k2U8edvTUL2WJx7Ea\\\\\\\/jLoBQMnWZ0h3vQ6qCB\\\\\\\/7E+Fp+w6eIBIiH+rtGQQW9WvbkUP7rhJt30epscNd\\\\\\\/I6D8UmnqnkCtLBdRo77aZdoQuLqTKVhoxSj47uwNyHlE1PIn0pWI4KTS4yuUeFzW3EowGbEI0OCGHx7Qn2+BuevZ+ySZPP6jRWzfsYuXX0RZ5dg3idTNseI6ROg3B2jlTQ\\\\\\\/MlzeQt4Y1C6c3YwHioa2J9PqYbWJN1HNfUBWbp+aCcbqLuWqyz5hDJSsvc0bYtYUkNX1QBbY2bLXHdY3nKU4Ko1YQOWZeoOl+F4N\\\\\\\/Y8HlWl6Vm+PH9HAFxtTIvRVlzF5n2dlp7uINM84WGsnsJHdiviiOdxEF4FtkFIvFwC6HTg7k0dbx9IYMXhGquz0tvGMBwxD2nL5SjadV\\\\\\\/7mq2OvD0nWgl5\\\\\\\/FcBI5smbuHyrsXdOKMqFs9QOlNIoyD0HMk7mYEmYP8AH2JLJC6NpbBNr8LeAiIToktSUYhVuWtlTOhPwkYZo+C1H8rO54hGMk+gLn96nWYfTfeJRBZG\\\\\\\/xwUdcoCrEr\\\\\\\/x3q6xKpkfVr985e0Gzvi8mfofuJbc9wcZPuJN+Pf8+E+GukVwDqpRhcfbrs9\\\\\\\/3vlk\\\\\\\/uq2GFZzJxxY9g63MZnptACp5Q+lJpTb+asbC5RJv6eetNFlcJFLawiFt+SKL8sXo2Wa\\\\\\\/i1dU5oiJewcvekEmiCqkAWydeKau8wiyKub0bs3Sa5+lxNoEULZo5qpUFd5tbkTMM\\\\\\\/8kigxOaIy56e5Oy4tOupmkMWBU0XnodhVtZVk9jAwowgpVhZxmtL3W00K9N3Qj0A0nwGIlpcb\\\\\\\/JWJu\\\\\\\/Sy5S\\\\\\\/F+EumuJcqDOU21xwL2c0llSxSgB9Sw4Sb\\\\\\\/sfMToTA38j+9QkBX4Unn+l6rqXObZ7JoSvQ5lhpGPtQSLdRUd0u8s+5oVKbp0BeRXwflxQhOyQInovtwvgSkrn6C0XfCY2iQKM3ura9WXzeZlhh6ZNo58q68oOOGrWIia6UyZMItgt6L6XBkyjPfSSMJAT9M6DoFkXaxiZQip0X3Y2B0OGaXXSXZsSJEO5WMK9dCMFK3PbT61ItpW8geBm2UO4p+mE2\\\\\\\/zHNSjpmvDzIaLPVkEH7sIdTus0bBQ3Zoc3R97JzbiO5XUALqxUblnVTioSqiV3Mr5qI9Er903nudCUTLjnKylTaH7xhLqap5vw6QM2Wm2F6Nu\\\\\\\/PsIWdGVaBD2KPbodSP57v5+m4umWxqn5KHEwAOl5estQBDaDqSHJVOJF36KFx8OwTHZsx2N4tMMEXS1Wu8OYTJsjYanDOtzBN5dI0fYwjkZxxiW1wKdtovj93lYfvQUpM3wyrMDAK+CSnZlkxMYI8kyC+7qwFvH8fG5FH+gLHR3pqOVA4b5O\\\\\\\/Ougr4luZ0Gk0CjxMutLJvH0UDnkB5ztAUdTkJIMBhOOCeH+tELsElHg1v\\\\\\\/0BtxxjCO3tEYnNaGI0zh+Yk2D+BnsXjh9\\\\\\\/wRfW0uFabqINUY+HXjCf\\\\\\\/wjiEVVUMKHxflZAsLWxBe1OINM0Kfgxfr0dpMqeLBD+dJZIDYf+FyT\\\\\\\/RHVb73+I8VVw6xgB2yYBxJ3r4hS0OkdmrSvGxGt0NhGrvO7nhO7dNL0HTpGJ\\\\\\\/+a3x2hQrhaAIBCmvA7i8mRmYfgsf0AS3g8aW3zQaC4TLij1DHFhS2ZiJJgYMoTD0WtNOPSgMRq3pB1ggEcG39OlEIz+V2+3YifX4d7lZow5lgZjmYHOeq+QAOLYlmFZQML8ZAWsPmUQmBmCmuzwk0Ijep4bOyzlXrippxrRofBQOoAdER1RGspFeebYSANQ7Gi2QZ0dwQe1DqDnux\\\\\\\/Bs8PbZRrXx4AncjEKEdkgxsZd\\\\\\\/3XBlG9xn2IG2jF+An\\\\\\\/5MTtN0jF3vlVelZ0CgnYkGiY+roErNTrgZYFlWmtgRoFWWJpnm7m\\\\\\\/HfkhP5VJMPvclCiftnacpNMJB9FNVEA2ttgx5wserwKjLNP6tIQPUOeR5oCCELzOBQZqQjlI3feDWen7L1zsz8qxFhOeADAbZyutUwXFSRUlaJ3Bri6\\\\\\\/QTJPxZEYNcUVIzTF6GQrtINsNTT9eUro+WUFbNjr6SelNN+cpyLNxPbXWkdt+RgiZFNeAPT2B9prRfpZcD4pLjm+tfqw6VmgGOKWQfIptNH5aT896C+nE370cptEwzysgAex8ltQkc+J3swGhH5+BoGVKhVCBl4BPAVcKWiwnIKGcV+yFE6ysWmZ4bq+JBSRU6Lr1bW6ES9YpSizopt+ZB7q0YHBu6V493eEjSuASBnEA9kBp0NxJukBKLGgJoOuXs0UaCNGwNU07sdVy9RE\\\\\\\/eoHBoGobOilo56moBN1B4vVWWdKCSuJt3vWAXPvxPETM3lE7xQvRD2EWG+hvJovcN2ImTShTSQ+WvZBGgkXMK9smMSyATguXyQRLMWhFqq6NQQoeUrMfQKgjktbHtxL6nT6RHVWKp48hXt15VcUUCKsda7p6ofzSM20sC4N\\\\\\\/1NiaFHQECanAaLXWFyzNlFCO5gJ5OK0AsNVDeSjvxfcsCtO8Rkt6arQJ0TD6XvdbFG0vG1CDYs8erKTehqNcZesKduJUEEVcZzyY0L\\\\\\\/NbOxRNv0Zq3pCA7WyKGXzV7wtvdqJtZpCV2IIzlZkzA+cvrSXTLEnQPUnDy\\\\\\\/X7zlx7Nil7uM0y7Tqe71p2gq28wPqoaB9sOfvYiUBNFKhA2VeIxXRqcVJORKHddfOvKJh8yilzUoY9wq8wtdeG564uQLU4VDbzTUrqTsVb4q5Oy+MQ\\\\\\\/1AOPESqn6ofETHD\\\\\\\/kq7vV6jxrTGldHDIl56Ew0UTqVdBLWFLFszVi2civCVtW04zEt7VvNpnhkZbL7GAbfx0OsyEregxBaKGX8IL5MCiyHujy5SJzge5GMBexZeGFTp+XbTcPAFk3Sg73rqp6kXLfTSO+8NRGpzch0i+WmpITN6FkEslZQh\\\\\\\/Ktrq1QhMOSHPLbNsJTdnVupeh1vqk7QW95iPEZu7EPf\\\\\\\/nlMDRy8MIM1f8kQEEw6P44xiKBN13HYRPAKznjemATlQ5JU0dKvn3m\\\\\\\/dadQlN7uJ9cHi9hFhNMKJNyMXc2kVPhdjNjyeBF40m5L90HajlAny+G0LsArf0En2Sj2l0dxqboY8c7MFVPaqi225FYqDrHzMimLAYtSpiYx9YCwc2QkuEOvgJ6zu5GUy4RSNREpqhV5NGYjMD8CeMIu1Q1vhU9DTlOnUAZv4LN83C9zDtkyLBoU5rtK2HwieQyhI8uAwFtSsmA3ydCAhJRzc2wrIyHLLAIbsLWzF4aJd0j1kcyaebrNsYUj77xKRtzcpgZtlzFxSIN0VqelKzOtxUzna67EUvKDmg+iC7te2ei6XrRK46E1ZL1CHtOSqIe5eW9zBp7qrdy7m068AIyvuy9HhFg1+6KOyiEaN4xEBlDbPEHkxRJcytjfJTCBkangUo8SwDwcE1QZykkWmZxYlXbOdkGAWbODCJY5kRfozpLM0srFLa+3vXChk6aVQqy3RJTNblzFz+PIPPgOUKWG1OKxLADmO1hD\\\\\\\/aw8wmBIlpzLNbwvPw59vV0TU\\\\\\\/w7mlTqJLyAVX9PeKRL1wckQewvSkIWbMOs5oKiSh8q3yI0J+GCtuTil3ZAWLOWkWZglu1B5LMhV3GbIQuqCQV7LgX1SACfqyZW+5xbulAZxzQf6eyyJLYf9a6Juoc9kPeuiuGzxSJd1Iieo6rnkMuP6fCBLYTkDRVF87s1DUdRE6MOocYTHudIaJQXrYB2VmxjQ2WxWU8cXLSF6AWJ+ck+jkG89VBQ33bbpEcpXuvE8Fq+ZvwbLbCVlelLW\\\\\\\/gM6M24M2EEKfvYW5UuPFCQ5s0wR36JwUNCTanTf\\\\\\\/qnaB0BIIblXLENA88uquaTrRUgyRz9VP79LJd8kw3nSlJjjE6A028z3Nl8uFSQtILjvPPHQsrj20LMgNj53Q6P3hjsGrHaxBo87ojvIRDdWp0Q05oZiFhMI8bsHBfV54p1trR1\\\\\\\/fW0cvzyNR7hj1NityMEfbRRDd7JFrpebi2SPITWichGKekWflrtnMODr5m1W+6iOV6T54tcdP5abXnLGt3c\\\\\\\/GX6NCAydSeJssz1jrICfWmejFeYS\\\\\\\/czrQZFuE4H\\\\\\\/Ke0FtvQ8+Cwb7GZBvMyZEV1OdcVTK+cPR1k0nECuIrpMpYHl36rXl\\\\\\\/ePHxzSvVhKgTLbVXfzan\\\\\\\/6yAGrVqyFTv5rtC189gRvlc7tkta\\\\\\\/cnTLOT8crlajQnd\\\\\\\/EDOkO0FkMMLdqWywef5DbMftIliMWwtNR53kFMpHeyoUBL7\\\\\\\/qqzHNDR6czcXDaijDO3zzygCcDtmQ4+uRgHdDbMIBLr6Fv3vJ9yHnc\\\\\\\/Qzm1vQjXdoH3AK7pN3XpYzSBv2d2ALgJ6FcxAezv8LE69SqmdAnka4F8vKpxNnt61SmTNfiJKgUqQPXn91iCJGXBdnhz0Dn8z3uj9x04znhijMb7Wns\\\\\\\/9FXjihZW1u7p2VKCbOYhM5Ru1nENq2qzmstWO1zFVK8vS6kCnbeCgZy+Vqd4T6A4wkgR3IWREjNhnGQHCJD3PvZLV+YvX84E2NXBD++Et9nvWj0Yeh1NGaS0bjSGI9ZS+8SKue7P2Y9y+qgmQvSH0OjxlutY5ME+0uOFKTCQ6nie\\\\\\\/ve4gpH4Gd4SHzo69TW2a0W0zkiHFRaM0KsJxQIgardEeGr+IqAPsTXWsfSi7PE\\\\\\\/1421YKxQW2KjmLQW6Buiuzp0kj7JQ35AIxZug11TiwwuDTKmiyU0bxR5wLdJr2q9AV\\\\\\\/fnks+T9BWVzxyA3JKtdLTeVuQDVRiOd3DVl8GVunIHW1fgkGlEAeMTVtNjRaLMflXx+TPXd5NIGU\\\\\\\/ZdKQAFm+9eylqOXBtuurcUvn957+NpRSFL1NYCPUHIoa6nYsUgzTG\\\\\\\/2+SmJTg7MU9PWzpwtJDCz+QfeWFJWG9hkDP44wyjGn\\\\\\\/5lF5MxbB\\\\\\\/Nej8XTkroJ72aTAfcoaDdf4z7m8bf6dS\\\\\\\/PF2IIScgSFoIVusqU2sMCav1s3d2gcK7L9hUmQp7Mw18plnNq\\\\\\\/Gmc3DTaXDZwTM0rp0uyAvpF2u\\\\\\\/Qne4UkTfDMRI3z15jWlal2XLGjzazONobrta4X0+9lgNB2DZFWyU+0472hdJqeH\\\\\\\/KZGHrHykLS++FLENJDgk5zBfLFawv5fZhbrzRLaIeT4mJVwUeC05TzzUEK1wx\\\\\\\/IU8P1XEpXXw2jxG7A3lh0XzRCcB3DELWI7Zl9u2LgwYpjcWqJMCaksGtjLfVrD\\\\\\\/sAnxJNqJsLN36fJUyi7YjpNA72poR8APUM5GIHt3JQoo6qkwvqjjXzjBNUa9WNMthyRPnspTggXKDo5QQUoXAjt7U5yeGiiufU\\\\\\\/VDpjw7ty58PtfOv9Ck48kO5UBZallpJ4OapqbcB42VVGZ8iGqBqd6AgyDM28SzQYdVmkyeYPB+ffeNfU\\\\\\\/4JjtFsPZ3cW\\\\\\\/\\\\\\\/2aX3fYNCaas+rQ+8ZV4WKibuwU7AfV+4qqbjOM+1\\\\\\\/33HKU7zdmartKoLcqkiF8RJ4qojsvGqb+7ClH4ci8ftFFUO5HxmHU+bqawu3Ug4SqsC4lQeFcirpuzhunHdWa\\\\\\\/Pl\\\\\\\/L0iXXwvMlJYTBszUQ5n7Fa7d8WIDkL8oBOt6EXuL4jewbzTuQcDwqWRkygjCOSypCBT0suyqjYFCfXvsmUZul3UwDlKiFkIkWPYyY4zEFLdgVttxQ5zpZtsvzUS61vj8HpiE43hm9XBqCvf\\\\\\\/p+tXS8hmFs5zFMU7KKi8\\\\\\\/IHXx8KXN213d76T065PbvKK2PKwGwy1W\\\\\\\/VAiev\\\\\\\/WVFYAhehParPww7K+oFM+Rq1uTfw+f2z0yKJkl+HiYKPmoPb\\\\\\\/U1mG+AwUxAmqHUABQ4GknsfIc05OX\\\\\\\/VwXEwOldkTEwUlRXaz7\\\\\\\/EsXosaI5b0rTre\\\\\\\/T9EvHn5QC8BABAMQV99lDARvAQ\\\\\\\/FvujG9nJJxr55cpo6jzz70hT1pgtvaIIIh0xaje3v41G4CyOk6j9NbtS2xLp9G8ff8ROtf6y8tnk8hhubbVHhzf2bI7tYtVdbt6J\\\\\\\/V0Ew4mKrLDQ7+07MDwpA+59LEUkyzFhku9xmSnnYroUWLmBgP8p0TnLru45mSif9awaEuIWMEpnF\\\\\\\/+109MrgkzWfdyXc7EkEpX0sQ0BKvx4rJPfgENehXTqFW9M37jfOYmA7rkTv05hJfNKu3SJWgACqeuWdGSjmpVA52W2jiOdo8fZS6ZWWwsqL+6wCAEsnZa49btHVjbt4ESNQ6TTRvVt5bllKJyL+KMuttw2NzFO1zEXmEcB8s+CxPjpDxIdWIJnohg04TnBhHpfQqHO3QHgkuTJ8a1zJ\\\\\\\/SGgeC\\\\\\\/72FJ7iwPqlDSWbcl+q\\\\\\\/u\\\\\\\/oRM+JIiqkSKPYwOzsziZi59+SfYSfYtrFqhINk9HctwoUGEs08ulzyPI4zAf8i+L+zpoxwmx689Fgp6BeNPBpE+y8S1ij6BPlsXvVO2qDOAv+V5k59wseeeu0FLMbyskZWEwryMhonEXGAdICvEhsRjjRNUHVmfUmIfAO0faTNQtVbJu9cqI71lKQzNyKqZFWfVs1F+qND8QZNBZY4uVRBeaHZ4FL2x2LMSPLtuLCxh7bBVQX6QCVUTn5w2yEd\\\\\\\/vFhPyN+H8fdIz3Z9w4Hz8Eoe2zI0z9ijln01LEzB8ah1mH0B60RlgKRUQ6r692nfUKjAHlvrEUrKnRDs8YSHErx6Vp\\\\\\\/QfvvHl9+KgaWtMYr2dUZ5YV8cXg0abNCTHCs3xAZw9wzaR7GOKBc7FQ\\\\\\\/\\\\\\\/\\\\\\\/7xES6puy2qIi9qSXtkH0SU+HMGrm20fEv29xcTLarbNsYtA462igHVYbwHQceQmEKjTvrkjDtsEDPahH5zSpNabnKAdPT53Gz1GR\\\\\\\/zE0ymZY1fX4R6EeG+xn7AyracKZq\\\\\\\/e5yl1y5kUl9nbhU6P5rA9LppLoX3MDbK4pdxGSQWSjeyw4XIwfpVIuwcY7a7JRQ+cIbaX6n9nAa7JjyPnQRW4gu60XeEN5zt+ReiH0\\\\\\\/6c9iMAcf9ZxYi76\\\\\\\/uEf0anuEjLRAeL71lBrHl3LtL6UUQ5dozJNYGnezurfzFXCW\\\\\\\/lYCO0LlRklA51oQv3oNyqdz8iD2+0tcSK6J8ty6cKuVimXZBdZu87oDLzy3RjF1nefhVx02iaC+wJ2dmDCbw2yI10Y8RXb64iL599OnWbXetqaRmZmLVoMYKWnu9S4JiK1BeOdQ\\\\\\\/tuasDhTtpRD\\\\\\\/qO57LTB\\\\\\\/FptiTmu\\\\\\\/44hi\\\\\\\/4+l0fXxWKaToAo4Rcaz9455bYTS33nwbviyu5k6FGEk\\\\\\\/IKJ4rKZK9UNCSRDeeVUJRWKRABhDRDud349U9kvmmV9OlQsCvKyAtgg2nAAMk4t0kLGTJ2mce+OWRQnajZSswwc6VQzrTZebgkNEjo0jMfs98mca8P4HMT6\\\\\\\/eEJI4k9rLZqP+vrkdEDohlIDmrZo4MiILCoV8oQiWMOzCDiuY9xOqX6GHArDiEmz3CZ6OKKt\\\\\\\/4U\\\\\\\/TDV0XUF2z7A2Bhez23aYuV\\\\\\\/3+Cil5fm9SA+vAslilHopMqTPRnZhXfNa6icgpZ6WilFjIqf8Q51trHFKU0qrmz8BYR4qfza7tS7vYEOT3yQozxn2QRWp3C1gO4\\\\\\\/x3unKeMNcWtpTXT4ng65a2blW0oBTe2rCijs38bhuysQ0LtiAZjElYJejAJrDhw1TIdL1YMZ53tloToiH6i5ITGzAzYI5ouFmlRAVMAMaUmKAzEss6dK5V6U6Rgn3cErbFLTqC0xN3eQfjbAeBODQpcl7l2V8ihXbh1bLWs3CT\\\\\\\/5+gow2obFRG637IO2Kx0FZ\\\\\\\/opETIwnEpbcifUtvlcqYwrPTGCyfWaVobfz6qkXHLgZNFElRbIGls1Fbc+pM+dlI2iQNltBByIFPk98fydQhqWzOSHLAjs9PguBZ0eJTIPfRAQ1OetnC3LnJUcx6Zw5AdVoC9HMNVP15dCAsURfi7B1HZgBCu7OU+4fgIx6D6U6cx6i\\\\\\\/EZwzzdLC\\\\\\\/RFMDeVn6TAPA0M14TVzY6B8eYXYxSCoi7ExAgAl8cMHONXQQ0bfOXYa0UpeKJctgualoul\\\\\\\/JQz4vxZp4CQ5UDVsl8ORGdq8Sf2OFr\\\\\\\/f3u1DgpS6irnO6WS7ogl8IyTEPiKL1ZYTGV3mSUWwlcQGTOzaTgs\\\\\\\/kTNYgc5r6+rkWpMaIBPjsF+laT7ANv54AuFn87ugjHzhlu1m20pLHrgFM9LOSh3oN7NBmGNXG\\\\\\\/xfjgkfQQk84GwBGwbk5QBawBLvgh3\\\\\\\/rk0TUltD9rK4NmJfGmBkbdHpNOJ5ox3eXri7Tdv3YY1zSrKRzW2sz\\\\\\\/DAvSmytcvfFCbLcIj2o4zBsJjW\\\\\\\/W6RWKPDgriVqVrW3VrVe5Dk6cQdiVl8o6TZOAXcFYWsjbFPzNAXW9yCFRJk0\\\\\\\/T9mZmWKl1UxqSjuR8Ya3K8NHzzSPxM4DctTak\\\\\\\/5lBlqXJApFn6YYbUkE6pMONvgspJN\\\\\\\/UEq9z9hcWVQiUIExez6Im3a3CZRdLX59o\\\\\\\/kt7cyD2ZSd8+LSXBYc5VftkqRHmJSW0vCdFMOju\\\\\\\/inFbxL1fa4p5kXoNN77+31CcqefTqLGIAipg2\\\\\\\/15kBCr2zozKpVgwKJ5dxths\\\\\\\/hMxYw\\\\\\\/Kj56gpPC\\\\\\\/2\\\\\\\/c0+JjTuicmcHigpFFhGviPsg684KzHJ9pg11RcERjKzwtedjDG7XkTIo9jrHk8jfljOpmSsrx+Y\\\\\\\/GlxzlV3DZSg2UO6tyv0H62OUVBIbT8cMdmq62skz\\\\\\\/i6TsgZuz2HV+txB9rqXONY\\\\\\\/4PIbUD\\\\\\\/aOvZTXbdODjE1zqdvB6j3bbqFJONnUu6k2HYRH5uNDhf0sUbeCu8Zk76g8NAjS+Mr\\\\\\\/MNMz\\\\\\\/TZO3LkjMS4o9L7LnBVkuxB0hlXkPfA9LIhLCgGMrLABPDsVJG06uMM87JDO0z6bG4FOkks3UdN7ymBhFo6fllF4gcEvCuh+OqxR5Ny4iVH7Jm7vTodFzyk8krms+yzfXMQS1sqKEv1SslcEJftYB1Mv8UjmMwJyVcrI0QekhFUyO2q+MBKkHMOKSt8dT6DUvBSJ+if8lNIAhiLxBhoOsuk6HV\\\\\\\/n2Fn9UKk\\\\\\\/jUTnr+tcI\\\\\\\/7lCPVHjeC612QJd6K2md8bJISUqwfKxwz2KVApeoz538Sy57oTl6r9eoF76xFECECsgQ72j4iWrFLWt+QwhrxY3gNju2ClbyGBBFj3EYhi6dIG731mKz5fvLYOdsJJRfT0KT83sWQp22L0CsM44ocK3zBr9R4RkB2OMVMkYmg\\\\\\\/bgSKmaObDypi58NOv8jyuYyc9u\\\\\\\/A8S7lRfNrFNTyehn8BL8xhFYTy81wuBj2Y9XNsz7Ei4fKiQnxo4jh8AEwvfNPhcFooT6E4MZJbeQi6398Vs3\\\\\\\/cua5WSuPS0tJedwo8\\\\\\\/xZSDPBv\\\\\\\/UEY7zpGFiAdwDPKhHrPn4fUPqdDQmldmaYK6lHj7ktlT0ktVu+cbbmWMmaL\\\\\\\/6WIsBjd7A+Jll6wyS4Uzyvv3nPLzji3UKRxazdHnZzYDHvlPI0ya5Ay5omMdDgG74vc+IclR75NLSPk6dodEdGexIXv4VNDbrshYEZFuJwiil7b\\\\\\\/50zShEEAC6pD44EtgQPm0Nq7WWOc2yi7nMNJVFl7osAfJI39bPuLTBp\\\\\\\/sguE6KtAj6GYpoUjWjN9628mB8gYP4aY9Kuygemma7NOldRiuKrCgUFoHUF+Kh\\\\\\\/pYMvW\\\\\\\/ekmB4Ad5UxoqKu1IlMi94GEdJc+vjTFJG9ueRswDPBtg6VvxE7Niav7i5oEICvF8dSaaal+0gCHx2GL17hfWZZRMopvgmmOyxFR1mILKE9aDAq8oyjAhhHyaHKsR2bcdgDVnizQHl1vpn+xM2D8R9v9TVh\\\\\\\/Un9jgd6xaGlwkRf\\\\\\\/XjpXFsDqt4sZSlPTk8O9JxHHW5sw7vNvohnbYc3gNKCoRdhOElTR8cRKhX41nmyEzLuvaD7+PrCf1L5xL7L4mGN+bFUfSrMVJswFLnM0z6aO\\\\\\\/6YJLCEmSEe6YDKgmvBKQQXzayDCgOkMHRUtY5qeZaDSLxF6gXNMm7guEZJbfAgQj+C+Uz18MJkjVZG0owYpl8KSPywRtNt89As1sOWaJD1hMg6ZMvpk5SNXJad0GhafXdCT91FkXjeR\\\\\\\/WXZz3uBzl5kvgaf9F3X3h0Nolu6+xmc2o039ldLpgDObmdNujS+FdP7JtADV\\\\\\\/mPqSGuzRbbhEJP0C6TXvQPCGZ9cxkcYsH0HpvHLu8MRbvC6TSqgnp2OvC93d6611uAmjwbkC02Q5YdZxQNvS+b+zfzw5xftAyr4FzcFRuI9Nf4DLCj7ZMCLcVIIapamMpugZ2ItAYZbHj9vluyHAsWDP81BqFeMjI221m77msxzwkOyz7gu8hpjJgSgSuNbc482dF1kkg\\\\\\\/BK0F6hfOO+7UjS8BbSSMYyjJ9x68Lg3rXjW\\\\\\\/2G8E1voSOoEYnqtDElOmTL60wWhtMYVoWfdC3gZCc9tNIRPgXURFBNx\\\\\\\/\\\\\\\/jLbRXjz7ETdSPL8U\\\\\\\/f6lKfIXHc1Y3qe1\\\\\\\/4+3Oq67g7Frfq3lIuZcSkRvlwjcsUfpCtsoywsMkvtqcOQx+4OIp9iPultr0yr82FPg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022IKtZ1AGHtwyDAvyA9WsNYg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b7dfa7c8fd739a361b762de78e6156e98e7afd24f3f803575051aa4411be491d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VLQP9rpFfi9kidzPGRGovEq\\\\\\\/53wT8TEBpN\\\\\\\/DGUbW7lI3k7CL0Y\\\\\\\/pt7X8NuA1ne3AIPY\\\\\\\/CKRF6m0\\\\\\\/TgYl2MPsHyQgXI\\\\\\\/AJLZCG\\\\\\\/Vn8j9Z\\\\\\\/fCJ7oohJttLdzsQ4OGsB\\\\\\\/pfLNt0uim65bUSN89+5Gs78vznsHP9DGUHA1oiD1\\\\\\\/DkrPk\\\\\\\/rAnjRHw29TRHHAlsCBHDUIxJE3zTcpoAKq4CzdR8vSRiCaFpj3xKp\\\\\\\/8hf433O6loamNMg4cMG\\\\\\\/uf2TAFfECu70hXyAfgYTvIzwd3IC2az5OOxD5QAnakw9VPtbw8bMdRGbxzyC4PpiK6admgWYLVZuRjK2YtkxxFVWKcX9weunq0\\\\\\\/KIYHXjPYyyzzrLoz\\\\\\\/3hkDPsIywtSdhF0f2rPUj5PT05i95GT9SohPM46IwHX+tXWEeGcl1nUVUcpudy6zSJrF6QsP8k5a0\\\\\\\/\\\\\\\/IK\\\\\\\/oY+YANjxxPEBmerFccnrXsnj2p0qbnXUN2pC3hoEf+m7\\\\\\\/9COcvcjlUUeLXhUJ6Ohe7dJHKpNL2A75sUVs4cwS09qqmh4Yioo\\\\\\\/CQ65Dfuo5RrJQhsYhOzNJ418R1zhYEjFjZkNfArMHFjcOpD1lfDCcRbU95abFj2faoUmVPfSr9pEJKBDUeHsjIGvJ+87MskSFll3ITil436qbM\\\\\\\/5vESHIFH5njF+wzXKscKTRkc7G1RZqJfAf\\\\\\\/HXbvj5UpyztKG9DsjCPNH1NvUE4L8xjGAQBWuUHh0PGb3cdlHXU6ziMI\\\\\\\/keJPstX4VVVpXLZxrDYNr+lOAZZl4LDiboOkmMz3TgqGHJKl9gUH7pZw70x9sv\\\\\\\/wN9EiOBuJMkk2TWpuYp1ipcdJBvaux3ZmHtPdKx4hXCsbVOdYE54pHVnR3Rv2Q7444r\\\\\\\/hhkJG7bptsNVJx7x6csiBRW9OsAtEHE\\\\\\\/P8Le25mpUaE\\\\\\\/6EIK\\\\\\\/vIqTMWLqiOQglTdJMG+7HSpio1URBmt+xQS2+QBGopArfsr+mHN0hHhCU4k7dR1qiABtnQpeodki4exRkX3hBDKXHU\\\\\\\/HHv3wouwwhq2As5uC2Taoc1qgWKY0l5FutEmbHfPBOSmEVKRzweiL1p42ibk9CBC\\\\\\\/ugdyCVjQCcsttetqu8JYb+GvZjJanPgfjHYkUDbiRZOqoEl31\\\\\\\/k07Xq9t2V\\\\\\\/G\\\\\\\/bRNCrofJHWvIrTkLbl0NXat5gD09Q0Is0OCt6VUGn0MaRrpGBjhkTfDmM+ZX1FTRR\\\\\\\/kuHvN6oy5DbHvCPudY1QekvWhQihKhgnQe\\\\\\\/C\\\\\\\/b+32kLWV1IebbMZSqwhqLLiiFGCEUAetqv9gG4K8aQaGeAlLmkublZrz8utLAv3XUV5xTUrjrs9ThYbtACRNnQZ7d2VCSpRqtJQbf4qBeaiFTnt8qlT96vxJVZVBM6doW2IbWOY5VyV7b1deKk1\\\\\\\/1zJfJcEaeBKHiNhtfC6AYDQQUegCiFevEqAtDmOwbSfE0lbfqwbdPMEAyGsCcuKXK1JY2E3IcOmG+3Us6I4vme5eqX\\\\\\\/jcL46a2jmklVEXqtEYf4ywE6PfO7pRquTLB5UQ3Sfots4xfnw7PJc\\\\\\\/CebLva1cgUMYetka0prF2+ChrN+buMf8Id2IY2pYKZrrx92rQrylTotRk13DxrcUXGOvVIM8gqgep0btEysbQ9biPBn8iWaq7ZNyJlTAJfk+FeM5ecGQxN+Dwd9FMawlSSKfvy+CcnL\\\\\\\/FbYf2gBvKbjmN\\\\\\\/+rZ+dsDO4KX3gYe8P7dZkshqhXrWJ9EXcP3I8Fx+kIShMHRmMaFawRnLIH\\\\\\\/HqIuG6CmJfV4yJvZ7H4TFEhlZGmBdue3IK3STPb3mP1uLrVP\\\\\\\/E4mmtckFLXtRP62PbjA7hjK2N0FZCiAdvOiIMfTVNYIY5D1XnVObT4lVP0EfBXVurifFooE18PX9MhKu\\\\\\\/bYHvEWCoikqWIqqim7WD8FbPds9aB6Q+AOSrVrAHfOPzYbnsFvisg7TeHA7dckE23sW10f65K9\\\\\\\/2sLaBtZ2F0Kqm78j4KBONkvrs3g95owbDjMtYpssvRTizQpgQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022MOcycfcZFOhB8C0J02dkuA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d04916d727711291c31d41a2465d6b010e083e2a70a0a7c1cd275f47917b0fde\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460305 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2725" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-chw-trBi2-WRdwgihMPaWoKZuH4cHyWkkNkNl5QtBPA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2725" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kCw4rGY6uhumrp1oF3_gMiX0Lu_ewf0O6FIpr-Rw140" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2717" title="Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2745" title="Identification of QTLs Associated with Virulence Related Traits and Drug Resistance in Cryptococcus neoformans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460305" id="node460305" data-pisa="ggg;6/9/2725" data-pisa-master="ggg;g3.116.032359" data-apath="/ggg/6/9/2725.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Bayesian Multi-trait and Multi-environment Model</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Fernando H. Toledo</span>, <span class="highwire-citation-author" data-delta="4">Oscar Pérez-Hernández</span>, <span class="highwire-citation-author" data-delta="5">Kent M. Eskridge</span> and <span class="highwire-citation-author" data-delta="6">Jessica Rutkoski</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2725-2744;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032359 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460305" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), Guanajuato, Guanajuato 36240, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando H. Toledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BH.%2BToledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toledo%20FH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BH.%2BToledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Oscar Pérez-Hernández </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Agriculture, University of Central Missouri, Warrensburg, Missouri 64093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oscar%2BP%C3%A9rez-Hern%C3%A1ndez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Hern%C3%A1ndez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOscar%2BP%25C3%25A9rez-Hern%25C3%25A1ndez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kent M. Eskridge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Statistics Department, University of Nebraska, Lincoln, Nebraska, 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BM.%2BEskridge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskridge%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BM.%2BEskridge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/9/2725" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460305" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460305/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2725.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460305" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460305/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2725.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460305" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460305/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/9/2725.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460305" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460305/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">When information on multiple genotypes evaluated in multiple environments is recorded, a multi-environment single trait model for assessing genotype × environment interaction (G × E) is usually employed. Comprehensive models that simultaneously take into account the correlated traits and trait × genotype × environment interaction (T × G × E) are lacking. In this research, we propose a Bayesian model for analyzing multiple traits and multiple environments for whole-genome prediction (WGP) model. For this model, we used Half-<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-1.gif"/></noscript></span></span> priors on each standard deviation term and uniform priors on each correlation of the covariance matrix. These priors were not informative and led to posterior inferences that were insensitive to the choice of hyper-parameters. We also developed a computationally efficient Markov Chain Monte Carlo (MCMC) under the above priors, which allowed us to obtain all required full conditional distributions of the parameters leading to an exact Gibbs sampling for the posterior distribution. We used two real data sets to implement and evaluate the proposed Bayesian method and found that when the correlation between traits was high (&gt;0.5), the proposed model (with unstructured variance–covariance) improved prediction accuracy compared to the model with diagonal and standard variance–covariance structures. The R-software package Bayesian Multi-Trait and Multi-Environment (BMTME) offers optimized C++ routines to efficiently perform the analyses.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/multi-trait" class="hw-term hw-article-keyword hw-article-keyword-multi-trait" rel="nofollow">multi-trait</a></li><li class="kwd"><a href="/keyword/multi-environment" class="hw-term hw-article-keyword hw-article-keyword-multi-environment" rel="nofollow">multi-environment</a></li><li class="kwd"><a href="/keyword/bayesian-estimation" class="hw-term hw-article-keyword hw-article-keyword-bayesian-estimation" rel="nofollow">Bayesian estimation</a></li><li class="kwd"><a href="/keyword/genome-enabled-prediction" class="hw-term hw-article-keyword hw-article-keyword-genome-enabled-prediction" rel="nofollow">genome-enabled prediction</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">Since the whole-genome prediction (WGP) model of <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Meuwissen <em>et al.</em> (2001)</a>, practical results have shown that genomic selection (GS) using Bayesian and non-Bayesian linear regression models improves prediction accuracy compared to conventional and pedigree selection (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pérez-Rodríguez <em>et al.</em> 2012</a>). With GS, genomic breeding values are estimated as the sum of marker effects for genotyped individuals in the testing or prediction population. The marker effects are estimated simultaneously using a training population that contains phenotyped and genotyped individuals.</p><p id="p-4">In plant breeding, most of the available methods for WGP are useful for analyzing a single trait measured either in a single environment or in multi-environments with the incorporation of genotype × environment interaction (G × E) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Heslot <em>et al.</em> 2014</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> 2014</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em> 2015</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">López-Cruz <em>et al.</em> 2015</a>). However, researchers often face situations in which multiple traits are measured across multiple environments. For example, crop breeders record phenotypic data for multiple traits such as grain yield and its components (<em>e.g.</em>, grain type, grain weight, biomass, <em>etc</em>.), grain quality (<em>e.g.</em>, taste, shape, color, nutrient content), and tolerance to biotic and abiotic stresses. They often aim to improve all these multiple correlated traits simultaneously or to predict the ones that are difficult to measure with those that are easy to measure. However, it is common practice to perform an independent analysis and genomic prediction on a single phenotypic trait.</p><p id="p-5">The advantage of jointly modeling multiple traits compared to analyzing each trait separately is that the inference process appropriately accounts for the correlation among the traits, which helps to increase prediction accuracy, statistical power, parameter estimation accuracy, and reduce trait selection bias (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Henderson and Quaas 1976</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pollak <em>et al.</em> 1984</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schaeffer 1984</a>). In the context of WGP, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jia and Jannink (2012)</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Guo <em>et al.</em> (2014)</a>, and <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jiang <em>et al.</em> (2015)</a> found that joint prediction of multiple traits benefits from genetic correlation between traits and significantly improves prediction accuracy compared to single trait methods, specifically for low-heritability traits that are genetically correlated with a high-heritability trait. <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Jia and Jannink (2012)</a> also found better prediction accuracy for multiple traits than for single traits when phenotypes are not available for all individuals and traits. Therefore, there is evidence that multiple trait analysis is useful to predict yet-to-be observed phenotypes in plant and animal breeding when selecting unphenotyped candidates early through the prediction of their genomic breeding values. Multi-trait analysis has also been found to substantially increase prediction accuracy when some traits are observed in all individuals but the trait of interest is not observed in the individuals in the test set (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pszczola <em>et al.</em> 2013</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rutkoski <em>et al.</em> 2016</a>).</p><p id="p-6">Multivariate analysis of continuous outcomes is well established in statistical literature (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Johnson and Wichern 1992</a>). However, the available methods cannot be applied in a straightforward manner for WGP, since the number of independent variables (<em>p</em>) is usually larger than the available sample size (<em>n</em>). The genomic best linear unbiased predictor (GBLUP) WGP model can be implemented in standard software for multiple traits and multiple environments by taking into account two-way interaction terms and estimating separable unstructured covariance matrices of the form <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-2.gif"/></noscript></span></span> (where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-4.gif"/></noscript></span></span> are the corresponding covariance matrices of factors A and B, respectively). However, these software programs are unable to estimate separable unstructured variance–covariance matrices of the form <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-5.gif"/></noscript></span></span> for three-way interaction terms. For this reason, in this situation, at least one of the variance–covariance components is assumed to be identity or a new variable is created by merging two factors and estimating a covariance matrix with only two components as <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-6.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-7.gif"/></noscript></span></span> contains the variance–covariance of two factors, but each component cannot be separated. Also, univariate Bayesian inference has been proposed and extensively implemented in WGP models (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gianola 2013</a>). The Bayesian alphabet methods (Bayes A, Bayes C, and Bayes <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-8.gif"/></noscript></span></span>) have been extended for multiple trait analysis (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">de los Campos and Gianola 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Calus and Veerkamp 2011</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Jia and Jannink 2012</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Guo <em>et al.</em> 2014</a>) and most recently, <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Jiang <em>et al.</em> (2015)</a> proposed a Bayesian multivariate antedependence model.</p><p id="p-7">Despite evidence of the increased prediction accuracy of WGP models incorporating G × E (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> 2014</a>, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em> 2015</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">López-Cruz <em>et al.</em> 2015</a>) and of WGP models for multi-trait data, statistical models for analyzing continuous data for simultaneously assessing multi-traits and multi-environments are lacking. Thus, the integration of these two approaches in one unified WGP model is required (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Jiang <em>et al.</em> 2015</a>). This unified WGP model would be useful in two cases: (i) when individuals are measured for all traits in one environment, but only some traits in other environments; and (ii) when some traits are recorded in only a subset of individuals in all environments. This model would be useful not only in plant breeding but also in animal breeding, where genetic evaluation of many traits is performed on a weekly basis by many breeding programs globally. It is also possible to integrate other advantageous strategies such as the antedependence model to incorporate dominant and epistatic effects.</p><p id="p-8">All the Bayesian methods developed so far for multiple trait analysis use the Inverse-Wishart (IW) conjugate family of distributions as priors for the covariance matrices between traits. However, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gelman (2006)</a> and <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang and Wand (2013)</a> argued against using IW priors for covariance matrices because they impose a degree of informativity and the posterior inferences are sensitive to the choice of hyper-parameters. Recently, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Huang and Wand (2013)</a> proposed a scale mixture approach involving an IW distribution and independent Inverse-Gamma (IG) distributions for each dimension as priors for the covariance matrix parameters. The ensuing covariance matrix distribution is such that all standard deviation parameters have Half-<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-9.gif"/></noscript></span></span> distributions and the correlation parameters have uniform distributions on (−1,1) for a particular choice of the IW shape parameter. The advantage of this approach is that it is possible to choose shape and scale parameters that achieve arbitrary high noninformativity of all standard deviations and correlation parameters (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Huang and Wand 2013</a>). However, the model proposed by <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Huang and Wand (2013)</a> is a standard mixed model with correlated errors that does not include interaction terms of any kind and does not consider three-way interaction.</p><p id="p-9">In this study, we propose a Bayesian method that integrates the analysis of multi-traits and multi-environments and takes into account trait × genotype × environment interaction (T × G × E) in a unified WGP model. We used Half-<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-10.gif"/></noscript></span></span> priors on each standard deviation term and uniform priors on each correlation to achieve high noninformativity and posterior inferences that are not sensitive to the choice of hyper-parameters. We illustrate the use of the unified Bayesian Multi-Trait and Multi-Environment (BMTME) method in simulated data sets and two real data sets (one maize and one wheat) including multiple traits measured on wheat and maize lines evaluated in multiple environments and genotyped with dense molecular markers. We also provide an R package called BMTME that can be used to fit the proposed methods.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Statistical model</h3><p id="p-10">We use <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-11.gif"/></noscript></span></span> to represent the normal response from the <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-12.gif"/></noscript></span></span> th replication of the <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-13.gif"/></noscript></span></span> th line in the <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-14.gif"/></noscript></span></span> th environment for the <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-15.gif"/></noscript></span></span> th trait (<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-16.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-17.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-18.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-19.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-20.gif"/></noscript></span></span> represents the number of replicates of each line in each environment and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-21.gif"/></noscript></span></span> denotes the number of traits under study. To present the theory in a simple manner, we will use <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-22.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-23.gif"/></noscript></span></span> Therefore, the total number of observations for the <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-24.gif"/></noscript></span></span> th trait is <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-25.gif"/></noscript></span></span> We propose the following linear mixed model for each trait:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-27.gif"/></noscript></span></span> represents the <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-28.gif"/></noscript></span></span> th environment for the <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-29.gif"/></noscript></span></span> th trait and is assumed as a fixed effect, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-30.gif"/></noscript></span></span> represents the genomic effect of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-31.gif"/></noscript></span></span> th line in the <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-32.gif"/></noscript></span></span> th trait and is assumed as random effect, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-33.gif"/></noscript></span></span> is the interaction between the genomic effect of the <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-34.gif"/></noscript></span></span> th line and the <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-35.gif"/></noscript></span></span> th environment for the <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-36.gif"/></noscript></span></span> th trait and is assumed a random effect, and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-37.gif"/></noscript></span></span> is a random error term associated with the <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-38.gif"/></noscript></span></span> th replication of the <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-39.gif"/></noscript></span></span> th line in the <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-40.gif"/></noscript></span></span> th environment for the <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-41.gif"/></noscript></span></span> th trait. To take into account the correlation between traits, one could use the following <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-42.gif"/></noscript></span></span> variate linear mixed model:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-43.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-44.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-45.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-46.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-47.gif"/></noscript></span></span> is the genetic covariance matrix between traits and is assumed unstructured, <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-48.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-49.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-50.gif"/></noscript></span></span> is the residual covariance matrix between traits and is assumed unstructured. <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-51.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-52.gif"/></noscript></span></span> if the environment <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-53.gif"/></noscript></span></span> is observed and 0 otherwise for the <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-54.gif"/></noscript></span></span> th trait, for <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-55.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-56.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-57.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-58.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-59.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-60.gif"/></noscript></span></span>. With model (1), we can perform a separate analysis for each trait, with the inconvenience that independence between the <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-61.gif"/></noscript></span></span> traits is assumed. Model (2) can take into account and exploit the correlation between traits.</p><p id="p-11">In matrix notation, the model given in equation (2) including all the information is expressed as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-62.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-63.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-64.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-65.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-66.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-67.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-68.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-69.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-70.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-71.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-72.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-73.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-74.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-75.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-76.gif"/></noscript></span></span><sub>,</sub> and <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-77.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-78.gif"/></noscript></span></span>. Then <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-79.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-80.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-81.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-82.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-83.gif"/></noscript></span></span> denotes a Kronecker product, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-84.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-85.gif"/></noscript></span></span> is assumed a diagonal matrix of order <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-86.gif"/></noscript></span></span>, which indicates that we are assuming independence between environments. It is important to point out that the trait × environment (T × E) interaction term is included in the fixed effect <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-87.gif"/></noscript></span></span>, while the trait × genotype (T × G) interaction term is included in the random effect <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-88.gif"/></noscript></span></span> and the three-way (T × G × E) interaction term is included in <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-89.gif"/></noscript></span></span>. The errors are assumed to be correlated with the covariance defined as <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-90.gif"/></noscript></span></span>. More flexible variance–covariances as diagonal or identity are straightforward. Also note that <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-91.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-92.gif"/></noscript></span></span>; therefore, <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-93.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-94.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-95.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-96.gif"/></noscript></span></span>. The matrix of the genomic relationship between lines <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-97.gif"/></noscript></span></span>, also known as Genomic Relationship Matrix (GRM), was calculated using the method of <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">VanRaden (2008)</a>.</p></div><div id="sec-3" class="subsection"><h3>Joint posterior density and prior specification</h3><p id="p-12">In this section, we provide the joint posterior density and prior specification for the Bayesian WGP Multiple Trait and Multiple Environment (BMTME) model. The joint posterior density of the parameter vector becomes:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-98.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-99.gif"/></noscript></span></span>.</p><p id="p-13">The notation <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-100.gif"/></noscript></span></span> Inverse-Wishart <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-101.gif"/></noscript></span></span> indicates that the density function of <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-102.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-103.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-104.gif"/></noscript></span></span> both are positive definite matrices. We assume that <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-105.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-106.gif"/></noscript></span></span> denotes an Inverse-Wishart distribution with shape <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-107.gif"/></noscript></span></span> and scale parameters <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-108.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-109.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-110.gif"/></noscript></span></span> denote an IG distribution with shape <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-111.gif"/></noscript></span></span> and scale parameters <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-112.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-113.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-114.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-115.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-116.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-117.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-118.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-119.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-120.gif"/></noscript></span></span>. Since <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-121.gif"/></noscript></span></span>, the prior for <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-122.gif"/></noscript></span></span> with the prior for <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-123.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-124.gif"/></noscript></span></span>.</p><p id="p-14">Next we combine the joint posterior density of the parameter vector (4) with the priors to obtain the full conditional distribution for parameters <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-125.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-126.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-127.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-128.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-129.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-130.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-131.gif"/></noscript></span></span>. All full conditionals, as well as details of their derivations, are given in Appendix A.</p></div><div id="sec-4" class="subsection"><h3>Gibbs sampler</h3><p id="p-15">In order to produce posterior means for all relevant model parameters, below we outline the exact Gibbs sampler procedure that we propose for estimating the parameters of interest. As is the case with Markov Chain Monte Carlo (MCMC) techniques, the ordering of draws is somewhat arbitrary; however, we suggest the following order:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-16">Step 1. Simulate <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-132.gif"/></noscript></span></span> according to the normal distribution given in Appendix A (A.1).</p></li><li id="list-item-2"><p id="p-17">Step 2. Simulate <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-133.gif"/></noscript></span></span> according to the IW distribution given in Appendix A (A.2).</p></li><li id="list-item-3"><p id="p-18">Step 3. Simulate <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-134.gif"/></noscript></span></span> according to the IG distribution given in Appendix A (A.3).</p></li><li id="list-item-4"><p id="p-19">Step 4. Simulate <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-135.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-136.gif"/></noscript></span></span> according to the normal distribution given in Appendix A (A.4 and A.5).</p></li><li id="list-item-5"><p id="p-20">Step 5. Simulate <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-137.gif"/></noscript></span></span> according to the IW distribution given in Appendix A (A.6).</p></li><li id="list-item-6"><p id="p-21">Step 6. Simulate <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-138.gif"/></noscript></span></span> according to the IG distribution given in Appendix A (A.7).</p></li><li id="list-item-7"><p id="p-22">Step 7. Simulate <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-139.gif"/></noscript></span></span> according to the IW distribution given in Appendix A (A.8).</p></li><li id="list-item-8"><p id="p-23">Step 8. Simulate <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-140.gif"/></noscript></span></span> according to the IG distribution given in Appendix A (A.9).</p></li><li id="list-item-9"><p id="p-24">Step 9. Simulate <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-141.gif"/></noscript></span></span> according to the IW distribution given in Appendix A (A.10).</p></li><li id="list-item-10"><p id="p-25">Step 10. Simulate <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-142.gif"/></noscript></span></span> according to the IG distribution given in Appendix (A.11).</p></li><li id="list-item-11"><p id="p-26">Step 11. Return to step 1 or terminate when chain length is adequate to meet convergence diagnostics.</p></li></ul></div><div id="sec-5" class="subsection"><h3>Model implementation</h3><p id="p-27">The Gibbs sampler described above for the BMTME model was implemented as an R-software package. We performed a total of 60,000 iterations; 30,000 samples were used for inference because the first 30,000 were used as burn-in to decrease the MCMC errors in prediction accuracy. We did not apply thinning of the chains following the suggestions of <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Geyer (1992)</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">MacEachern and Berliner (1994)</a>, and <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Link and Eaton (2012)</a>, who provide justification of the ban on subsampling MCMC output for approximating simple features of the target distribution (<em>e.g.</em>, means, variances, and percentiles).</p><p id="p-28">We implemented the prior specification given in the previous section where the BMTME model was defined. The hyper-parameters used were for <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-143.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-144.gif"/></noscript></span></span> we used <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-145.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-146.gif"/></noscript></span></span> we used <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-147.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-148.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-149.gif"/></noscript></span></span> we used <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-150.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-151.gif"/></noscript></span></span> we used <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-152.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-153.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-154.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-155.gif"/></noscript></span></span> we used <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-156.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-157.gif"/></noscript></span></span> we used <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-158.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-159.gif"/></noscript></span></span>. All these hyper-parameters were chosen to lead weakly informative priors.</p></div><div id="sec-6" class="subsection"><h3>Assessing prediction accuracy</h3><p id="p-29">We used two cross-validation schemes for generating training and validation sets that mimic two real situations a breeder might face. Cross-validation 1 (CV1) mimics a situation where lines were evaluated in some environments for all traits but some lines are missing in other environments; this is similar to cross-validation 2 of <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Burgueño <em>et al.</em> (2012)</a>. The other cross-validation scheme is CV2, which mimics a situation where a trait is lacking in all lines in one environment but present in the remaining environments (see Table D1 in Appendix D). In this case, information from relatives is used, and prediction assessment can benefit from borrowing information between lines across environments, and among correlated traits.</p><p id="p-30">We implemented a 10-fold cross-validation with 80% of the observations in the training set and 20% in the testing set. Of the variety of methods for comparing the predictive posterior distribution to the observed data (generally termed “posterior predictive checks”), we used two criteria: the mean square error of prediction (MSEP) and the Pearson correlation. Models with small MSEP indicate better predictions, and higher correlation values indicate better predictions. The predicted observations were calculated with <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-160.gif"/></noscript></span></span> collected Gibbs samplers as: <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-161.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-162.gif"/></noscript></span></span> are estimates of <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-163.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-164.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-165.gif"/></noscript></span></span> in the s<em>th</em> collected sample.</p></div><div id="sec-7" class="subsection"><h3>Simulation data</h3><p id="p-31">To illustrate the parameter estimation of the proposed BMTME method, a small simulation experiment was conducted. The data were simulated based on model (3) with three environments, three traits, 80 genotypes, and 20 replications. We assumed that <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-166.gif"/></noscript></span></span>15,8,7,12,6,7,14,9,8], where the first three β coefficients belong to traits 1, 2, and 3 in environment 1, the second three values for the three traits in environment 2 and the last three for environment 3,<span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-167.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-168.gif"/></noscript></span></span>. These two variance–covariance matrices gave rise to a matrix of correlation between traits with each correlation between pairs of traits equal to 0.85. Also, we assume that the GRM is known, <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-169.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-170.gif"/></noscript></span></span> is an identity matrix of order 80 and <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-171.gif"/></noscript></span></span> is a matrix of order <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-172.gif"/></noscript></span></span> of ones. The relationship between environments is assumed as <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-173.gif"/></noscript></span></span>. Therefore, the total number of observations was <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-174.gif"/></noscript></span></span> <em>i.e.</em>, <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-175.gif"/></noscript></span></span> for each trait. With these parameters, 50 data sets were simulated according to model (3) and for each data set, parameters <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-176.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-177.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-178.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-179.gif"/></noscript></span></span> were estimated with the BMTME model using the Gibbs sampler given above. We used the priors given in the section on model implementation, which were also used for the applications with real data sets. For this simulated data set, we computed 20,000 MCMC samples, and Bayes estimates were computed with 10,000 samples, since the first 10,000 were discarded as burn-in. In <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, we report average estimates along with standard deviations (SD).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469123/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469123/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Simulated data with three traits and three environments</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">Also, with the proposed BMTME model, we simulated two data sets similar to the simulation study explained above, except that the environmental covariance matrix we used was an identity matrix. The first data set assumes that the genetic and residual correlation between traits was 0.85 for all pairs of traits under study, while the second data set assumes that the correlation between all pairs of traits was 0.2 for both covariance matrices (<span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-192.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-193.gif"/></noscript></span></span>). We implemented a 10-fold cross-validation (CV1). The training data set has 80% of the lines (64 lines), while the testing data set has the remaining 20% (16 lines). We assessed the prediction performance using the simulated data set under three conditions: (1) unstructured (Appendix A): assuming both variance–covariances are unstructured (<span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-194.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-195.gif"/></noscript></span></span>); (2) diagonal (Appendix B): assuming both variance–covariances are diagonal; and (3) standard (Appendix C): assuming both variance–covariances are identity multiplied by the scale parameters <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-196.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-197.gif"/></noscript></span></span>, respectively.</p></div><div id="sec-8" class="subsection"><h3>Real data sets</h3><div id="sec-9" class="subsection"><h4>Maize data set:</h4><p id="p-34">A total of 309 double-haploid maize lines were phenotyped and genotyped; this is part of the data set used by <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> (2013)</a> that comprised a total of 504 doubled haploid lines derived by crossing and backcrossing eight inbred lines to form several full-sib families. Traits available in this data set include grain yield (Yield), anthesis-silking interval (ASI), and plant height (PH); each of these traits was evaluated in three optimum rainfed environments (E1, E2, and E3). The experimental field design in each of the three environments was an α-lattice incomplete block design with two replicates. Data were preadjusted using estimates of block and environmental effects derived from a linear model that accounted for the incomplete block design within environment and for environmental effects.</p><p id="p-35">Information about genotyping-by-sequencing (GBS) data for each maize chromosome, the number of markers after initial filtering, and the number of markers after imputation, was summarized in <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> (2013)</a>. Filtering was first done by removing markers that had &gt;80% of the maize lines with missing values, and then markers with minor allele frequency lower than or equal to 0.05 were deleted. The total number of GBS data was 681,257 single nucleotide polymorphisms (SNPs) and, after filtering for missing values and minor allele frequency, 158,281 SNPs were used for the analyses. About 20% of cells were missing in the filtered GBS information used for prediction; these missing values were replaced by their expected values before doing the prediction.</p></div><div id="sec-10" class="subsection"><h4>Wheat data set:</h4><p id="p-36">A total of 250 wheat lines were extracted from a large set of 39 yield trials grown during the 2013–2014 crop season in Ciudad Obregon, Sonora, Mexico (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Rutkoski <em>et al.</em> 2016</a>). The trials were sown in mid-November and grown on beds with 5 and 2 irrigations plus drip irrigation. Days to heading (DTHD) were recorded as the number of days from germination until 50% of spikes had emerged in each plot, in the first replicate of each trial. Grain yield (GRYLD) was the total plot grain yield measured after maturity, and plant height (PTHT) was recorded in centimeters.</p><p id="p-37">Image data of the yield trials were collected using a hyper-spectral camera (A-series, Mirco-Hyperspec VNIR, Headwall Photonics, Fitchburg, Massachusetts) mounted on a manned aircraft. From this data, vegetative indices for each plot were calculated. The green normalized difference vegetation index (GNDVI) was one of the traits used in this study. Trait GNDVI is considered a good predictor when used with pedigree and/or genomic prediction of GRYLD in wheat due to its high heritability and genetic correlation with GRYLD. Also, trait GNDVI can be measured remotely in large numbers of candidates for selection.</p><p id="p-38">Genotyping-by-sequencing was used for genome-wide genotyping. Single nucleotide polymorphisms were called across all lines using the TASSEL GBS pipeline anchored to the genome assembly of Chinese Spring. Single nucleotide polymorphism calls were extracted and markers were filtered so that percent missing data did not exceed 80% and 20%, respectively. Individuals with &gt;80% missing marker data were removed, and markers were recorded as −1, 0, and 1, indicating homozygous for the minor allele, heterozygous, and homozygous for the major allele, respectively. Next, markers with &lt;0.01 minor allele frequency were removed, and missing data were imputed with the marker mean. A total of 12,083 markers remained after marker editing.</p></div></div><div id="sec-11" class="subsection"><h3>Data and codes repository</h3><p id="p-39">The phenotypic and genotypic information of the two data sets included in this study as well as the R package for performing the analyses can be downloaded from the link: <a href="http://hdl.handle.net/11529/10646">http://hdl.handle.net/11529/10646</a>. This link contains the phenotypic data on maize (Data.maize) and wheat (Data.trigo), as well as genomic data on maize (G.maize) and wheat (G.trigo). Also, the link includes the BMTME.zip with the R package used to perform the analyses under the BMTME model.</p></div><div id="sec-12" class="subsection"><h3>The BMTME R package</h3><p id="p-40">Nowadays the R programming language is a popular tool in statistical science for analyzing and visualizing data (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">R Core Team, 2015</a>). However, in the context of big data with complex models, the speed of R is slow. For this reason, many times R is combined with C++ codes to produce high-performance programs that considerably increase the speed of programs (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Stroustrup 2000</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Eddelbuettel and Sanderson 2014</a>). The R package we developed for fitting the BMTME models merges R and C++ through the use of Rcpp together with Armadillo C++ library (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sanderson 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Eddelbuettel 2013</a>). Appendix E describes how the three-way data should be arranged and Appendix F explains the basic input needed to run the routines built in the R package for fitting the BMTME.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-41">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><p id="p-42">Results for the simulated data sets and for the real data sets (maize and wheat) are shown below.</p><div id="sec-15" class="subsection"><h3>Simulated data set</h3><p id="p-43"><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> gives the posterior mean and posterior SD for β coefficients (<span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-198.gif"/></noscript></span></span>) for each trait and for the variance–covariance matrices (<span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-199.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-200.gif"/></noscript></span></span>). The estimates of the posterior means for the β coefficients (<span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-201.gif"/></noscript></span></span>) and for the variance–covariance matrices (<span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-202.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-203.gif"/></noscript></span></span>) are very close to the true values, while the estimates of the diagonal covariance matrix (<span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-204.gif"/></noscript></span></span>) are slightly overestimated. Although the diagonal covariance matrix of <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-205.gif"/></noscript></span></span> is slightly overestimated according to the performed simulation study, we have evidence that the proposed BMTME model does reasonably well in terms of parameter estimation. We also tested the proposed BMTME model with another set of parameters and our results agree with the above mentioned results.</p><p id="p-44"><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows the resulting prediction accuracy (Correlation and MSEP) for each environment–trait combination for the two simulated data sets; we also present the ranking of the BMTME model under the three conditions (unstructured, diagonal, and standard) for each environment–trait combination. Based on the ranking given in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, the best prediction accuracy for both data sets with low and high correlation between traits (using both criteria) was achieved when the model assumed an unstructured variance–covariance matrix for both <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-206.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-207.gif"/></noscript></span></span>, followed by the second condition, which assumes a diagonal matrix for <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-208.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-209.gif"/></noscript></span></span> in terms of MSEP, but for the standard condition in terms of the Pearson correlation. In terms of the Pearson correlation for both data sets (low and high correlation between traits), in five of the nine environment–trait combinations, the unstructured condition performed better in terms of prediction accuracy, while the standard condition performed better in three of nine environment–trait combinations, and the diagonal condition performed better in only one of nine.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469187/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469187/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Simulated data with three traits and three environments</span><div class="sb-div caption-clear"></div></div></div><p id="p-47">In terms of MSEP, the unstructured BMTME performed better in five (low correlation between traits) and six (high correlation between traits) of the nine environment–trait combinations, the diagonal BMTME in only four (low correlation between traits) and two (high correlation between traits) of nine combinations, and the standard BMTME in zero (low correlation between traits) and one (high correlation between traits) of nine combinations. Regarding the average of the nine groups (environment–trait combinations) for both prediction criteria (correlation and MSEP), the unstructured BMTME gave the best prediction, correlation = 0.57 (low correlation between traits), and correlation = 0.67 (high correlation between traits) and MSEP = 1.06 (low correlation between traits) and MSEP = 1.07 (high correlation between traits). In both data sets, the unstructured BMTME model had the best prediction accuracy; however, the higher the correlation between traits, the higher the prediction accuracies observed, since the average correlation between traits under the unstructured BMTME was 17.5% higher when the correlation between traits was 0.85 compared to when it was 0.2.</p></div><div id="sec-16" class="subsection"><h3>Maize data set</h3><p id="p-48"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows that for each trait there are moderate differences between the β coefficients between environments. For Yield and PH, the largest and smallest β coefficients were observed in environments E1 and E2, respectively, while for trait ASI, the largest β coefficient was observed in E3 and the smallest in E2. The genetic estimates of the variance–covariance components of traits are given in <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-210.gif"/></noscript></span></span>, where the correlation between traits is moderate. Yield and ASI have a negative correlation (−0.27), and the correlation between ASI and PH is also negative (−0.25), while the correlation between Yield and PH is 0.41. The same tendency is observed in the residual correlation between traits but with smaller correlation between traits.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469122/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469122/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Maize data</span><div class="sb-div caption-clear"></div></div></div><p id="p-50"><a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the prediction accuracies (Correlation and MSEP) for each environment–trait combination and the ranking of the three conditions studied for each criterion in the maize testing data set for cross-validation CV1. From the ranking, the best condition is the standard model, since it was the best in five of nine environment–trait combinations in terms of correlation, while in terms of MSEP, the diagonal model was the best in three of nine environment–trait combinations. As for the averages of the environment–trait combinations, the standard model was also the best in terms of both criteria. The second-best model was the diagonal, and the unstructured model was the worst in terms of both criteria. This can be explained by the low correlation between traits that exists for this maize data set.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469117/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469117/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Maize data</span><div class="sb-div caption-clear"></div></div></div><p id="p-53"><a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> provides the results of cross-validation CV2 for the maize testing data set. The trait yield is unobserved in only one environment (for example, E1) for all lines, but data on the other two traits are available for this environment (E1), as well as for the other two environments (E2 and E3). The best model in terms of correlation and MSEP for predicting Yield for all lines in E1 was the standard model (0.215, 41.714), followed by the diagonal (0.168, 43.691) and the unstructured model (0.163, 46.407). In E2-Yield and E3-Yield, the best BMTME model was the unstructured model with Pearson correlation. In terms of MSEP, the BMTME unstructured model was the best model for predicting the Yield of the unobserved lines in E2, followed by the other two models (diagonal and standard). For E3-Yield, the best predictive model in terms of MSEP was the BMTME standard, followed by the unstructured model.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469164/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469164/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Maize data</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Wheat data set</h3><p id="p-55"><a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> shows that β coefficients are very different between traits and environments for the wheat data set. In environment Bed2IR, the largest β coefficients were observed in traits DTHD and GNDVI, respectively, while in environment Drip, the largest β coefficients were observed in traits GNDVI and GRYLD, respectively. The genetic estimates of the variance–covariance components of traits are given in <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-226.gif"/></noscript></span></span>,where the largest correlations were observed between trait DTHD <em>vs.</em> GNDVI, GRYLD, and PTHT; the same is true for the residual correlation between traits (<span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-227.gif"/></noscript></span></span>). In terms of prediction accuracies for the entire data set, they are high in terms of correlation and less precise in terms of MSEP mostly for trait PTHT in the three environments.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469088/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469088/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Wheat data</span><div class="sb-div caption-clear"></div></div></div><p id="p-57"><a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a> gives the prediction accuracy of the wheat data set for the testing data set for each environment–trait combination; it also gives the ranking of the three conditions studied under both criteria for cross-validation CV1. The best case is when the BMTME model assumes an unstructured variance–covariance matrix for both <span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-241.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-242.gif"/></noscript></span></span> and a diagonal matrix for the variance–covariance for <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-243.gif"/></noscript></span></span> followed by BMTME with a diagonal matrix for <span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-244.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-245.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-246.gif"/></noscript></span></span> As for the ranking in terms of Pearson correlation, in 6 of 12 groups the BMTME unstructured model performed better in terms of prediction accuracy, while the BMTME diagonal model was the second-best model since it was the best model in 3 of 12 cases; the BMTME standard model was the worst model in terms of prediction accuracy since it was the best in only 1 of 12 cases. In terms of MSEP, the BMTME unstructured model performed better in 5 of 12 cases, the BMTME diagonal model was the best model in only 3 of 12 cases, and the BMTME standard model was the best in 1 of 12 cases. The BMTME unstructured model also had the best average prediction accuracy of the 12 groups.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469202/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469202/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Wheat data</span><div class="sb-div caption-clear"></div></div></div><p id="p-60"><a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a> gives the results of cross-validation CV2 which assumes that trait GRYLD is lacking in one environment for all lines but not in the other environments. The results given are only for the testing data set (trait GRYLD missing for all lines in one environment). The best model for predicting GRYLD for all lines in environment Bed2I with Pearson correlation was the BMTME unstructured model, followed by the BMTME standard and, in the last position, the BMTME diagonal model. In terms of MSEP, the results are exactly the opposite. While in environment Bed5I the best predictive model in terms of Pearson correlation was the BMTME standard, then the BMTME diagonal, and, in the last position, the BMTME unstructured model. In terms of MSEP, the best model was the unstructured model, then the standard, and, at the end, the diagonal model. In environment Drip, the ranking of models based on both criteria was as follows: BMTME unstructured, BMTME diagonal, and BMTME standard.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469155/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469155/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Wheat data</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-62">To our knowledge, this is the first statistical three-way genomic model for assessing the prediction accuracy of trait × genotype × environment. Other models for assessing multi-traits or multi-environments have been extensively studied in the related literature (see, for example, <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em> 2015</a>); however, none of them have simultaneously assessed and modeled the three-way variance–covariance structure. The BMTME model does this task simultaneously using Bayesian estimation and the package for performing such a task is given in this article.</p><div id="sec-19" class="subsection"><h3>Performance of the BMTME model in simulated and real data sets</h3><p id="p-63">In the simulated data sets, the best prediction accuracies were achieved with the BMTME model (which assumes an unstructured variance–covariance matrix for the genetic and residual components) even when the correlation between traits was low, followed by the model that assumed a diagonal variance–covariance matrix for both matrices (in terms of the Pearson correlation) of traits, and then by the standard model, which was formed by an identity matrix multiplied by <span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-247.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-248.gif"/></noscript></span></span> for the genetic and residual variance–covariance matrices, respectively. The simulation study provides evidence that when the correlation between traits is high, it is really important to use a multivariate model that takes into account this correlation to improve prediction accuracies.</p><p id="p-64">This evidence is also supported by the results obtained with the wheat data set, where the BMTME unstructured model was the most accurate model, followed by the diagonal and finally by the standard model. However, with the maize data set, we did not observe any gain using the unstructured variance–covariance matrix in comparison to the other two variance–covariances used (diagonal and standard), maybe because in this data set the genetic and residual correlations between traits were low. Therefore, the important message is that when the correlation between traits is high (&gt;0.5), it is really important to estimate the unstructured variance–covariance matrix; when this correlation is low, it is enough to use the BMTME standard model because with the unstructured model, the results could be worse than those of the standard model. These suggestions are not new; they were also made by <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Calus and Veerkamp (2011)</a>, <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Jia and Jannink (2012)</a>, <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Guo <em>et al.</em> (2014)</a>, and <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Jiang <em>et al.</em> (2015)</a> in the context of multi-trait analysis. Here we only point out that they are also valid in the multi-trait, multi-environment context, taking into account the T × G × E interaction term.</p><p id="p-65">Our contribution added to the traditional multi-trait model (proposed by <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Calus and Veerkamp 2011</a>; <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Jia and Jannink 2012</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Guo <em>et al.</em> 2014</a>; <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Jiang <em>et al.</em> 2015</a>) is that our model also is valid for the multi-environment and the three-way (T × G × E) interaction term, which more realistically mimic the type of data that are very common in plant breeding programs, where genotypes are evaluated for multi-traits in multi-environments. We are also aware that normally distributed traits are not the only traits commonly measured in plant breeding programs. For this reason, models for multiple categorical ordinal traits, multiple count traits, or a mixture of types of traits are also needed to help breeders improve the process of selecting candidate genotypes.</p></div><div id="sec-20" class="subsection"><h3>Prediction assessment of the BMTME model</h3><p id="p-66">We introduced a Gibbs sampler for Bayesian analysis of multi-traits and multi-environments that takes into account the three-way (T × G × E) interaction term that uses simple conditional distribution to simulate the joint posterior distribution of all required unknown parameters in the WGP model. This model has the advantage that it uses Half-<span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-249.gif"/></noscript></span></span> priors on each standard deviation term and uniform priors between −1 and 1 on each correlation of the covariance matrix of traits in order to achieve noninformativity and posterior inferences with low sensitivity to the choice of hyper-parameters for the variance–covariance matrices.</p><p id="p-67">Since we modeled the correlation patterns separately for each factor as <span class="inline-formula" id="inline-formula-246"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-250.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-247"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-251.gif"/></noscript></span></span>, this facilitates the interpretation of the contribution of every factor to the overall correlation structure. It also allows choosing specific covariance structures for each factor, which improves accuracy and makes model fitting easier. In addition, fewer parameters than an unstructured model are required. For example, for modeling <span class="inline-formula" id="inline-formula-248"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-252.gif"/></noscript></span></span> under an unstructured model, we need to estimate <span class="inline-formula" id="inline-formula-249"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-253.gif"/></noscript></span></span> unknown parameters; this number of parameters is larger than the number of parameters required to be estimated using Kronecker products for a three-factor separable model that only needs <span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-254.gif"/></noscript></span></span> parameters. In our context, the number of parameters is lower, since we assumed a diagonal matrix for the variance–covariance matrix of environments and the matrix <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-255.gif"/></noscript></span></span> is given. Also, if needed, partial derivatives, inverse computation, and Cholesky decomposition of the overall covariance matrix are performed more easily on the factor-specific covariances because they have smaller dimensions. Therefore, the use of separable covariance matrices with Kronecker products has substantial computational advantages, besides improving interpretation and model fitting (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Simpson <em>et al.</em> 2014</a>). However, care needs to be exercised with the assumption of a Kronecker product structured variance–covariance matrix, especially in three-way multivariate data, because incorrect assumptions may lead to invalid conclusions (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Roy and Leiva 2008</a>).</p></div><div id="sec-21" class="subsection"><h3>Contributions and limitations of the BMTME model</h3><p id="p-68">This study clearly described the full conditional distributions for modeling the three-way (T × G × E) interaction term with multi-traits and multi-environments, which is of paramount importance for evaluating genotypic performance in target environments and for predicting yet-to-be observed phenotypes when the relative performance of genotypes varies across environments. Because the proposed model takes into account the correlation between traits and includes the three-way (T × G × E) interaction term, the BMTME can be a useful tool for efficiently selecting superior genotypes. The proposed BMTME model can be considered a Bayesian GBLUP for multiple traits and multiple environments since the marker information is taken into account in the GRM (<span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-256.gif"/></noscript></span></span>). Some of the advantages of our model over standard software are: (a) it is able to estimate separable covariance matrices of the form <span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-257.gif"/></noscript></span></span>, which is not possible with other software; (b) the estimation of three-way terms with covariance matrices of the form <span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-258.gif"/></noscript></span></span> is more parsimonious since fewer parameters are needed than when two factors are joined and the estimation process is performed using two separable covariance structures as <span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-259.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-260.gif"/></noscript></span></span> contains the covariance of the two factors <strong><em>B</em></strong> and <span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-261.gif"/></noscript></span></span>; (c) the convergence of our model is not a big deal compared to the convergence problems of other software for complex data; and (d) our model facilitates the interpretation of the covariance matrices because we can estimate the three covariance matrices.</p><p id="p-69">On the other hand, as expected, the disadvantage of the BMTME model is its high computational cost even under the optimized C++ developed and made available in this research article. Large numbers of lines might indeed cause some delays in the computation of such large numbers of parameters in the full conditionals. However, constant developments in computing science will soon reduce the computing time of the three-way BMTME model.</p><p id="p-70">Finally, our proposed BMTME model can also be useful (a) in QTL-mapping studies, since some WGP methods are also commonly used for GWAS (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Peters <em>et al.</em> 2012</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Garrick and Fernando, 2013</a>; <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Jiang <em>et al.</em> 2015</a>), and (b) to include spatial information in the residual <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-262.gif"/></noscript></span></span> matrix of the proposed model. This information is often available from breeding programs since they measure geographical information of the plots where genotypes are tested in each environment; this could help improve prediction accuracy.</p></div><div id="sec-22" class="subsection"><h3>Conclusions</h3><p id="p-71">In this paper, we extended the multi-trait WGP model to the multi-trait and multi-environment WGP model. This unified WGP model takes into account the correlation between traits and the three-way interaction term (T × G × E). Additionally, a transparent derivation of all full conditional distributions required is given that allows us to propose an efficient Gibbs sampler that is easy to implement and produces precise parameter estimates with high noninformativity and posterior inferences with low sensitivity to the choice of hyper-parameters for the variance–covariance matrices. Finally, we successfully applied the proposed method to simulated and real data and found that when the correlation between the traits is high (&gt;0.5), the proposed BMTME model with an unstructured covariance matrix should be preferred over the diagonal and standard methods to help improve prediction accuracy. However, when correlations are low, it is enough to use the BMTME standard model because if we use the unstructured model, the results could be worse than those of the standard model. The R-software package BMTME offers specialized and optimized C++ routines to efficiently perform the analyses under the proposed model.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-114">We very much appreciate the International Maize and Wheat Improvement Center (CIMMYT) field collaborators, laboratory assistants, and technicians who collected the phenotypic and genotypic data used in this study.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-23"><h3>Derivation of full conditional distributions for the BMTME unstructured model</h3><div id="sec-24" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-263.gif"/></noscript></span></span></h4><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-264.gif"/></noscript></span><span class="disp-formula-label">(A.1)</span></span><p id="p-72">where <span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-265.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-266.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-262"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-267.gif"/></noscript></span></span>. Also, if we had assumed <span class="inline-formula" id="inline-formula-263"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-268.gif"/></noscript></span></span> as prior for <span class="inline-formula" id="inline-formula-264"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-269.gif"/></noscript></span></span>, we would have maintained a multivariate Normal posterior distribution due to the multivariate Normal distribution’s conjugacy. However, the mean vector and covariance matrix would be slightly modified.</p></div><div id="sec-25" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-265"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-270.gif"/></noscript></span></span></h4><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-271.gif"/></noscript></span><span class="disp-formula-label">(A.2)</span></span></div><div id="sec-26" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-266"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-272.gif"/></noscript></span></span></h4><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-273.gif"/></noscript></span><span class="disp-formula-label">(A.3)</span></span></div><div id="sec-27" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-267"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-274.gif"/></noscript></span></span></h4><p id="p-73">Defining <span class="inline-formula" id="inline-formula-268"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-275.gif"/></noscript></span></span> the conditional distribution of <span class="inline-formula" id="inline-formula-269"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-276.gif"/></noscript></span></span> is given as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-277.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-278.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-279.gif"/></noscript></span><span class="disp-formula-label">(A.4)</span></span>where <span class="inline-formula" id="inline-formula-270"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-280.gif"/></noscript></span></span>. In a similar way, by defining <span class="inline-formula" id="inline-formula-271"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-281.gif"/></noscript></span></span>, we arrive at the full conditional of <span class="inline-formula" id="inline-formula-272"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-282.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-282.gif"/></noscript></span></span> as<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-283.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-283.gif"/></noscript></span><span class="disp-formula-label">(A.5)</span></span>where <span class="inline-formula" id="inline-formula-273"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-284.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-284.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-274"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-285.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-285.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-275"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-286.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-286.gif"/></noscript></span></span>.</p></div><div id="sec-28" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-276"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-287.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-287.gif"/></noscript></span></span></h4><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-288.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-288.gif"/></noscript></span></span><p id="p-74">with <span class="inline-formula" id="inline-formula-277"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-289.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-289.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-290.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-290.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-291.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-291.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-292.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-292.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-278"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-293.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-293.gif"/></noscript></span></span>]) (A.6) where <span class="inline-formula" id="inline-formula-279"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-294.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-294.gif"/></noscript></span></span>. Note that <span class="inline-formula" id="inline-formula-280"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-295.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-295.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-281"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-296.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-296.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-282"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-297.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-297.gif"/></noscript></span></span>. From here, <span class="inline-formula" id="inline-formula-283"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-298.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-298.gif"/></noscript></span></span>, and so <span class="inline-formula" id="inline-formula-284"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-299.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-299.gif"/></noscript></span></span> Also note that <span class="inline-formula" id="inline-formula-285"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-300.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-300.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-286"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-301.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-301.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-287"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-302.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-302.gif"/></noscript></span></span>. Obtained using <span class="inline-formula" id="inline-formula-288"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-303.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-303.gif"/></noscript></span></span>.</p></div><div id="sec-29" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-289"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-304.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-304.gif"/></noscript></span></span></h4><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-305.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-305.gif"/></noscript></span><span class="disp-formula-label">(A.7)</span></span><p id="p-75">with <span class="inline-formula" id="inline-formula-290"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-306.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-306.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-291"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-307.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-307.gif"/></noscript></span></span> denotes the <span class="inline-formula" id="inline-formula-292"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-308.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-308.gif"/></noscript></span></span> entry of <span class="inline-formula" id="inline-formula-293"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-309.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-309.gif"/></noscript></span></span>.</p></div><div id="sec-30" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-294"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-310.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-310.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-295"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-311.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-311.gif"/></noscript></span></span></h4><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-312.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-312.gif"/></noscript></span><span class="disp-formula-label">(A.8)</span></span><p id="p-76">Since <span class="inline-formula" id="inline-formula-296"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-313.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-313.gif"/></noscript></span></span>. With <span class="inline-formula" id="inline-formula-297"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-314.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-314.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-298"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-315.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-315.gif"/></noscript></span></span>.</p></div><div id="sec-31" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-299"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-316.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-316.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-300"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-317.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-317.gif"/></noscript></span></span></h4><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-318.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-318.gif"/></noscript></span><span class="disp-formula-label">(A.9)</span></span></div><div id="sec-32" class="subsection"><h4>Full conditional for <em><span class="inline-formula" id="inline-formula-301"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-319.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-319.gif"/></noscript></span></span></em> with <span class="inline-formula" id="inline-formula-302"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-320.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-320.gif"/></noscript></span></span></h4><p id="p-77"><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-321.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-321.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-322.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-322.gif"/></noscript></span><span class="disp-formula-label">(A.10)</span></span>with <span class="inline-formula" id="inline-formula-303"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-323.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-323.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-304"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-324.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-324.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-305"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-325.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-325.gif"/></noscript></span></span></p></div><div id="sec-33" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-306"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-326.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-326.gif"/></noscript></span></span></h4><span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-327.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-327.gif"/></noscript></span><span class="disp-formula-label">(A.11)</span></span><p id="p-78">with <span class="inline-formula" id="inline-formula-307"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-328.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-328.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-308"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-329.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-329.gif"/></noscript></span></span> denotes the <span class="inline-formula" id="inline-formula-309"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-330.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-330.gif"/></noscript></span></span> entry of <span class="inline-formula" id="inline-formula-310"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-331.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-331.gif"/></noscript></span></span>.</p></div></div><div class="section" id="sec-34"><h3>Appendix B</h3><div id="sec-35" class="subsection"><h4>Derivation of full conditional distributions for the BMTME diagonal model</h4><p id="p-79">All full conditional distributions of the BMTME diagonal model are the same as those of the BMTME unstructured model, except those needed for the variance–covariance (<span class="inline-formula" id="inline-formula-311"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-332.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-332.gif"/></noscript></span></span> which are now diagonal. For this reason, here we provide the variances of the diagonal elements of <span class="inline-formula" id="inline-formula-312"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-333.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-333.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-313"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-334.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-334.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-314"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-335.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-335.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-315"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-336.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-336.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-316"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-337.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-337.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-317"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-338.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-338.gif"/></noscript></span></span>) and the required elements of <span class="inline-formula" id="inline-formula-318"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-339.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-339.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-319"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-340.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-340.gif"/></noscript></span></span>.</p><div id="sec-36" class="subsection"><h5>Full conditional for <span class="inline-formula" id="inline-formula-320"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-341.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-341.gif"/></noscript></span></span></h5><p id="p-80"><span class="inline-formula" id="inline-formula-321"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-342.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-342.gif"/></noscript></span></span>]) where <span class="inline-formula" id="inline-formula-322"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-343.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-343.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-323"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-344.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-344.gif"/></noscript></span></span> denote the <span class="inline-formula" id="inline-formula-324"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-345.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-345.gif"/></noscript></span></span> entry of the matrix <span class="inline-formula" id="inline-formula-325"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-346.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-346.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-326"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-347.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-347.gif"/></noscript></span></span>, respectively.</p></div><div id="sec-37" class="subsection"><h5>Full conditional for <span class="inline-formula" id="inline-formula-327"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-348.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-348.gif"/></noscript></span></span></h5><span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-349.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-349.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-350.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-350.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-351.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-351.gif"/></noscript></span></span></div><div id="sec-38" class="subsection"><h5>Full conditional for <span class="inline-formula" id="inline-formula-328"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-352.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-352.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-329"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-353.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-353.gif"/></noscript></span></span></h5><span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-354.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-354.gif"/></noscript></span></span><p id="p-81">where <span class="inline-formula" id="inline-formula-330"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-355.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-355.gif"/></noscript></span></span> denotes the <span class="inline-formula" id="inline-formula-331"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-356.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-356.gif"/></noscript></span></span> entry of the matrix <span class="inline-formula" id="inline-formula-332"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-357.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-357.gif"/></noscript></span></span>.</p></div><div id="sec-39" class="subsection"><h5>Full conditional for <span class="inline-formula" id="inline-formula-333"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-358.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-358.gif"/></noscript></span></span></h5><span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-359.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-359.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-360.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-360.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-361.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-361.gif"/></noscript></span></span></div></div></div><div class="section" id="sec-40"><h3>Appendix C</h3><div id="sec-41" class="subsection"><h4>Derivation of full conditional distributions for the BMTME standard model</h4><p id="p-82">All full conditional distributions of the BMTME standard model are the same as those of the BMTME unstructured model, except those needed for the variance–covariance (<span class="inline-formula" id="inline-formula-334"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-362.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-362.gif"/></noscript></span></span> which are now equal to an identity multiplied by <span class="inline-formula" id="inline-formula-335"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-363.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-363.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-336"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-364.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-364.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-337"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-365.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-365.gif"/></noscript></span></span>, respectively. For this reason, here we provide the full conditional of <span class="inline-formula" id="inline-formula-338"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-366.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-366.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-339"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-367.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-367.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-340"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-368.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-368.gif"/></noscript></span></span> and the required elements of <span class="inline-formula" id="inline-formula-341"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-369.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-369.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-342"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-370.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-370.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-343"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-371.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-371.gif"/></noscript></span></span>.</p><div id="sec-42" class="subsection"><h5>Full conditional for <span class="inline-formula" id="inline-formula-344"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-372.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-372.gif"/></noscript></span></span></h5><span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-373.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-373.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-374.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-374.gif"/></noscript></span></span></div><div id="sec-43" class="subsection"><h5>Full conditional for <span class="inline-formula" id="inline-formula-345"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-375.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-375.gif"/></noscript></span></span></h5><span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-376.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-376.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-377.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-377.gif"/></noscript></span></span></div><div id="sec-44" class="subsection"><h5>Full conditional for <span class="inline-formula" id="inline-formula-346"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-378.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-378.gif"/></noscript></span></span></h5><span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-379.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-379.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-380.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-380.gif"/></noscript></span></span></div><div id="sec-45" class="subsection"><h5>Full conditional for <span class="inline-formula" id="inline-formula-347"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-381.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-381.gif"/></noscript></span></span></h5><span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-382.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-382.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-383.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-383.gif"/></noscript></span></span></div><div id="sec-46" class="subsection"><h5>Full conditional for <span class="inline-formula" id="inline-formula-348"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-384.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-384.gif"/></noscript></span></span></h5><span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-385.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-385.gif"/></noscript></span></span><p id="p-83">where <span class="inline-formula" id="inline-formula-349"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-386.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-386.gif"/></noscript></span></span>.</p></div><div id="sec-47" class="subsection"><h5>Full conditional for <span class="inline-formula" id="inline-formula-350"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-387.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-387.gif"/></noscript></span></span></h5><span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-388.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-388.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-389.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-389.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-390.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2725/embed/mml-math-390.gif"/></noscript></span></span></div></div></div><div class="section" id="sec-48"><h3>Appendix D</h3><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469090/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469090/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table D1</span> <span class="caption-title">Cross-validation schemes</span><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-49"><h3>Appendix E</h3><p id="p-85">Data preparation for analysis with I environments, J lines, K replications, and L traits is shown in the table below. gid denotes unique lines name, env are the environments, rep denotes the replications, and resp represents the response variables.</p><div id="T10" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469169/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469169/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">ThreeWay</span> <div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-50"><h3>Appendix F</h3><div id="sec-51" class="subsection"><h4>How to install and use the BMTME package</h4><p id="p-86">The BMTME package performs the proposed models (unstructured, diagonal, and standard).</p><p id="p-87">Step 1. Install R-software version 3.2.4.</p><p id="p-88">Step 2. Manually install the BMTME package that is available at the link: <a href="http://hdl.handle.net/11529/10646">http://hdl.handle.net/11529/10646</a></p><p id="p-89">Step 3. Use the package. Open R and copy and page Example 1.</p><p id="p-90">################################ Example 1 #################################</p><p id="p-91">rm(list = objects()); ls() </p><p id="p-92">library(BMTME)</p><p id="p-93">&gt; data(ThreeWay) # load the built in package data file</p><p id="p-94">&gt; # to run with other data: load your data</p><p id="p-95">## Transforming to the data to be used. Here you do not need to modify anything.</p><p id="p-96">ThreeWay = transform(ThreeWay,</p><p id="p-97">trait = factor(trait),</p><p id="p-98">gid = factor(gid),</p><p id="p-99">env = factor(env),</p><p id="p-100">rep = factor(rep)</p><p id="p-101">## Creating the GRM matrix for this example (here you need to upload your own genomic relationship matrix ###########################################################</p><p id="p-102">K_x &lt;- matrix(.7, ncol = 10, nrow = 10)</p><p id="p-103">diag(K_x) &lt;- 1</p><p id="p-104">K &lt;- diag(8) %x% K_x</p><p id="p-105">ISigmaG &lt;- solve(K)</p><p id="p-106">####Here the model is fitted. You are only allowed to change model (“un” for unstructured #covariance matrix, “bd” for diagonal, and “st” for standard), nChain, nIter, and the working #directory (getwd ()) where you want to save your output. The output will be the β #coefficients, random effects b1 and b2, the three variance–covariances matrices (Sigma #Trait, Sigma Environments, and Sigma Residual of traits). The order of the call should be as follows considering the corresponding names in the dataframe ###########################</p><p id="p-107">fit1&lt;- fit(formula = resp ∼ trait + gid + env + rep,</p><p id="p-108">data = ThreeWay,</p><p id="p-109">K = ISigmaG,</p><p id="p-110">model = ’un’,</p><p id="p-111">nChain = 1,</p><p id="p-112">nIter = 100,</p><p id="p-113">saveAt = getwd(C:\\Osval\\))</p></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-13"><p id="p-115"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-06-02"><span class="received-label">Received </span>June 2, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-06-22"><span class="accepted-label">Accepted </span>June 22, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Montesinos-López <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.1" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype×environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.2" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.3" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.4" data-doi="10.1080/15427528.2011.558767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection and prediction in plant breeding.</span> <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">239</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D25%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.558767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.558767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.5" data-doi="10.1534/g3.113.008227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in maize breeding populations with genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1903</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008227%26rft_id%253Dinfo%253Apmid%252F24022750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMTkwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yNzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.6" data-doi="10.1051/gse:20070016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Factor analysis models for structuring covariance matrices of additive genetic effects: a Bayesian implementation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">39</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">481</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253Dde%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D481%26rft.epage%253D494%26rft.atitle%253DFactor%2Banalysis%2Bmodels%2Bfor%2Bstructuring%2Bcovariance%2Bmatrices%2Bof%2Badditive%2Bgenetic%2Beffects%253A%2Ba%2BBayesian%2Bimplementation.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A20070016%26rft_id%253Dinfo%253Apmid%252F17897592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:20070016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17897592&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.7" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.8" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>(<span class="cit-issue">04</span>): <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2725.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddelbuettel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">Seamless R and C++ Integration with Rcpp</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.10" data-doi="10.1016/j.csda.2013.02.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddelbuettel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rcpparmadillo: Accelerating R with high-performance C++ linear algebra.</span> <abbr class="cit-jnl-abbrev">Comput. Stat. Data Anal.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1054</span>–<span class="cit-lpage">1063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStat.%2BData%2BAnal.%26rft.volume%253D71%26rft.spage%253D1054%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.csda.2013.02.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.csda.2013.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2725.11"><div class="cit-metadata unstructured">Garrick, D. J., and R. L. Fernando, 2013 Implementing a QTL detection study (GWAS) using genomic prediction methodology, pp. 275–298 in Genome-Wide Association Studies and Genomic Prediction edited by John M. Walker. Humana Press, Hertfordshire, UK.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.12" data-doi="10.1214/06-BA117A"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Prior distributions for variance parameters in hierarchical models (comment on article by Browne and Draper).</span> <abbr class="cit-jnl-abbrev">Bayesian Anal.</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">515</span>–<span class="cit-lpage">534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnal.%26rft.volume%253D1%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1214%252F06-BA117A%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/06-BA117A&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000207447000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.13" data-doi="10.1214/ss/1177011137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geyer</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Practical Markov Chain Monte Carlo.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">473</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D7%26rft.spage%253D473%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1177011137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/ss/1177011137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.14" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: the Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparison of single-trait and multiple-trait genomic prediction models.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D30%26rft_id%253Dinfo%253Apmid%252F24593261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24593261&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.16" data-doi="10.2527/jas1976.4361188x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quaas</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Multiple trait evaluation using relatives’ records.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1188</span>–<span class="cit-lpage">1197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1197%26rft.atitle%253DMultiple%2BTrait%2BEvaluation%2BUsing%2BRelatives%2527%2BRecords%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1976.4361188x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1976.4361188x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976CP41700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.17" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: A comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.18" data-doi="10.1007/s00122-013-2231-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">463</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2231-5%26rft_id%253Dinfo%253Apmid%252F24264761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2231-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24264761&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.19" data-doi="10.1214/13-BA815"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wand</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Simple marginally noninformative prior distributions for covariance matrices.</span> <abbr class="cit-jnl-abbrev">Bayesian Anal.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">439</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnal.%26rft.volume%253D8%26rft.spage%253D439%26rft_id%253Dinfo%253Adoi%252F10.1214%252F13-BA815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/13-BA815&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.20" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheyron</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.21" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-trait genomic selection methods increase genetic value prediction accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjcyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.22" data-doi="10.1038/hdy.2015.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.9%26rft_id%253Dinfo%253Apmid%252F25873147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25873147&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2725.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wichern</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">Applied Multivariate Statistical Analysis</span>, Vol. <span class="cit-vol">4</span>. <span class="cit-publ-name">Prentice Hall</span>, <span class="cit-publ-loc">Englewood Cliffs, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.24" data-doi="10.1111/j.2041-210X.2011.00131.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Link</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eaton</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">On thinning of chains in MCMC.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">112</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D3%26rft.spage%253D112%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2041-210X.2011.00131.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2041-210X.2011.00131.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.25" data-doi="10.1534/g3.114.016097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnet</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Increased prediction accuracy in wheat breeding trials using a markers × environment interaction genomic selection model.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">569</span>–<span class="cit-lpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016097%26rft_id%253Dinfo%253Apmid%252F25660166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvNC81NjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjcyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacEachern</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berliner</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Subsampling the Gibbs sampler.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">48</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">188</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BStat.%26rft.volume%253D48%26rft.spage%253D188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1994PQ61600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjcyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.28" data-doi="10.1534/g3.114.016188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskridge</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Threshold models for genome-enabled prediction of ordinal categorical traits in plant breeding.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">291</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016188%26rft_id%253Dinfo%253Apmid%252F25538102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvMi8yOTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjcyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manes</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.30" data-doi="10.2527/jas.2011-4507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reecy</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bayesian genome-wide association analysis of growth and yearling ultrasound measures of carcass traits in Brangus heifers.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">90</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">3398</span>–<span class="cit-lpage">3409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2011-4507%26rft_id%253Dinfo%253Apmid%252F23038745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2011-4507&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23038745&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309590700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.31" data-doi="10.3168/jds.S0022-0302(84)81481-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pollak</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van der Werf</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quaas</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Selection bias and multiple trait evaluation.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">67</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1590</span>–<span class="cit-lpage">1595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DPollak%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D67%26rft.issue%253D7%26rft.spage%253D1590%26rft.epage%253D1595%26rft.atitle%253DSelection%2BBias%2Band%2BMultiple%2BTrait%2BEvaluation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252884%252981481-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(84)81481-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.32" data-doi="10.1017/S175173111300150X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pszczola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Haas</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strabel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Effect of predictor traits on accuracy of genomic breeding values for feed intake based on a limited cow reference population.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%26rft.volume%253D7%26rft.spage%253D1759%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS175173111300150X%26rft_id%253Dinfo%253Apmid%252F23915541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S175173111300150X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23915541&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-6.9.2725.33"><div class="cit-metadata unstructured">R Core Team, 2015 R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2725.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leiva</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">Testing of a Structured Covariance Matrix for Three-level Repeated Measures Data</span>. <span class="cit-publ-name">UTSA, College of Business</span> <span class="cit-publ-loc">San Antonio, Texas</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrique</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Predictor traits from high-throughput phenotyping improve accuracy of pedigree and genomic selection for yield in wheat. G3 (<span class="cit-comment">in press</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2725.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Armadillo: An Open Source C++ Algebra Library for Fast Prototyping and Computationally Intensive Experiments. Tech. Rep</span>. <span class="cit-publ-name">NICTA</span>, <span class="cit-publ-loc">QLD, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.37" data-doi="10.3168/jds.S0022-0302(84)81479-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Sire and cow evaluation under multiple trait models.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">67</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1567</span>–<span class="cit-lpage">1580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DSchaeffer%26rft.auinit1%253DL.%2BR.%26rft.volume%253D67%26rft.issue%253D7%26rft.spage%253D1567%26rft.epage%253D1580%26rft.atitle%253DSire%2Band%2BCow%2BEvaluation%2BUnder%2BMultiple%2BTrait%2BModels%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252884%252981479-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(84)81479-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.38" data-doi="10.1371/journal.pone.0088864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styner</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Kronecker Product Linear Exponent AR(1) Correlation Structures for Multivariate Repeated Measures.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e88864</span>. DOI:.<span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0088864</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De88864%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0088864%26rft_id%253Dinfo%253Apmid%252F24586419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0088864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586419&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2725.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stroustrup</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">The C++ Programming Language</span>, <span class="cit-edition">Ed. 4</span>. <span class="cit-publ-name">Addison-Wesley</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2725.40" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2725.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/9/2725.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2717" title="Genetic Analysis and QTL Detection on Fiber Traits Using Two Recombinant Inbred Lines and Their Backcross Populations in Upland Cotton" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2745" title="Identification of QTLs Associated with Virulence Related Traits and Drug Resistance in Cryptococcus neoformans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/9"><span>Volume 6 Issue 9, September 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/9.cover-source.jpg?itok=nund1hRf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2725" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iMLDt1v2hZNALJlIcOtNHMLdTyTp9sK5ssb5pAOpZ8I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/9/2725" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/9/2725" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/9/2725" class="active" data-icon-position="" data-hide-link-title="0">A Genomic Bayesian Multi-trait and Multi-environment Model</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460305" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zz8Cj9UWi9A_DQi8VTEw1VSiQhkzeHlpwXFN-aHcAdw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2725.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/9/2725" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-i56yPgWnUFAqbQejoqcyGvxvyUk5rITs_MszdPwHrvk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/9/2725" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2725.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460305" id="node460305--2" data-pisa="ggg;6/9/2725" data-pisa-master="ggg;g3.116.032359" data-apath="/ggg/6/9/2725.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Bayesian Multi-trait and Multi-environment Model</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Fernando H. Toledo</span>, <span class="highwire-citation-author" data-delta="4">Oscar Pérez-Hernández</span>, <span class="highwire-citation-author" data-delta="5">Kent M. Eskridge</span> and <span class="highwire-citation-author" data-delta="6">Jessica Rutkoski</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2725-2744;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032359 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460305--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), Guanajuato, Guanajuato 36240, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando H. Toledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BH.%2BToledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toledo%20FH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BH.%2BToledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Oscar Pérez-Hernández </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Agriculture, University of Central Missouri, Warrensburg, Missouri 64093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oscar%2BP%C3%A9rez-Hern%C3%A1ndez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Hern%C3%A1ndez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOscar%2BP%25C3%25A9rez-Hern%25C3%25A1ndez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kent M. Eskridge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Statistics Department, University of Nebraska, Lincoln, Nebraska, 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BM.%2BEskridge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskridge%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BM.%2BEskridge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/9/2725" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/9/2725&amp;title=A%20Genomic%20Bayesian%20Multi-trait%20and%20Multi-environment%20Model" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/9/2725&amp;title=A%20Genomic%20Bayesian%20Multi-trait%20and%20Multi-environment%20Model" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/9/2725&amp;title=A%20Genomic%20Bayesian%20Multi-trait%20and%20Multi-environment%20Model" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/9/2725&amp;text=A%20Genomic%20Bayesian%20Multi-trait%20and%20Multi-environment%20Model" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/9/2725&amp;title=A%20Genomic%20Bayesian%20Multi-trait%20and%20Multi-environment%20Model" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/9/2725&amp;t=A%20Genomic%20Bayesian%20Multi-trait%20and%20Multi-environment%20Model" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/9/2725&amp;title=A%20Genomic%20Bayesian%20Multi-trait%20and%20Multi-environment%20Model" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/9/2725&amp;title=A%20Genomic%20Bayesian%20Multi-trait%20and%20Multi-environment%20Model" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460305/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460305" id="node460305--3" data-pisa="ggg;6/9/2725" data-pisa-master="ggg;g3.116.032359" data-apath="/ggg/6/9/2725.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Bayesian Multi-trait and Multi-environment Model</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Fernando H. Toledo</span>, <span class="highwire-citation-author" data-delta="4">Oscar Pérez-Hernández</span>, <span class="highwire-citation-author" data-delta="5">Kent M. Eskridge</span> and <span class="highwire-citation-author" data-delta="6">Jessica Rutkoski</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2725-2744;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032359 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460305--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), Guanajuato, Guanajuato 36240, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando H. Toledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BH.%2BToledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toledo%20FH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BH.%2BToledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Oscar Pérez-Hernández </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Agriculture, University of Central Missouri, Warrensburg, Missouri 64093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oscar%2BP%C3%A9rez-Hern%C3%A1ndez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Hern%C3%A1ndez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOscar%2BP%25C3%25A9rez-Hern%25C3%25A1ndez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kent M. Eskridge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Statistics Department, University of Nebraska, Lincoln, Nebraska, 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BM.%2BEskridge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskridge%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BM.%2BEskridge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), México, D.F. 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2725&amp;rft.epage=2744&amp;rft.atitle=A%20Genomic%20Bayesian%20Multi-trait%20and%20Multi-environment%20Model&amp;rft.volume=6&amp;rft.issue=9&amp;rft.date=2016-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Toledo%2C+Fernando+H.&amp;rft.au=P%C3%A9rez-Hern%C3%A1ndez%2C+Oscar&amp;rft.au=Eskridge%2C+Kent+M.&amp;rft.au=Rutkoski%2C+Jessica"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460305/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460305/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460305/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460305/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460305/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460305/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460305/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460305/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460305/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460305/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460305/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460305/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/9/2725" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/9/2725.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/9/2725.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/6/9/2725.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
