<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/9/2781" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/9/2781.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/9/2781.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/9/2781.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/9/2781.atom" />
<meta name="HW.pisa" content="ggg;6/9/2781" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Sources of Error in Mammalian Genetic Screens" />
<meta name="DC.Identifier" content="10.1534/g3.116.030973" />
<meta name="DC.Date" content="2016-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Sack et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic screens are invaluable tools for dissection of biological phenomena. Optimization of such screens to enhance discovery of candidate genes and minimize false positives is thus a critical aim. Here, we report several sources of error common to pooled genetic screening techniques used in mammalian cell culture systems, and demonstrate methods to eliminate these errors. We find that reverse transcriptase-mediated recombination during retroviral replication can lead to uncoupling of molecular tags, such as DNA barcodes (BCs), from their associated library elements, leading to chimeric proviral genomes in which BCs are paired to incorrect ORFs, shRNAs, etc . This effect depends on the length of homologous sequence between unique elements, and can be minimized with careful vector design. Furthermore, we report that residual plasmid DNA from viral packaging procedures can contaminate transduced cells. These plasmids serve as additional copies of the PCR template during library amplification, resulting in substantial inaccuracies in measurement of initial reference populations for screen normalization. The overabundance of template in some samples causes an imbalance between PCR cycles of contaminated and uncontaminated samples, which results in a systematic artifactual depletion of GC-rich library elements. Elimination of contaminating plasmid DNA using the bacterial endonuclease Benzonase can restore faithful measurements of template abundance and minimize GC bias." />
<meta name="DC.Contributor" content="Laura Magill Sack" />
<meta name="DC.Contributor" content="Teresa Davoli" />
<meta name="DC.Contributor" content="Qikai Xu" />
<meta name="DC.Contributor" content="Mamie Z. Li" />
<meta name="DC.Contributor" content="Stephen J. Elledge" />
<meta name="article:published_time" content="2016-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Sources of Error in Mammalian Genetic Screens" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic screens are invaluable tools for dissection of biological phenomena. Optimization of such screens to enhance discovery of candidate genes and minimize false positives is thus a critical aim. Here, we report several sources of error common to pooled genetic screening techniques used in mammalian cell culture systems, and demonstrate methods to eliminate these errors. We find that reverse transcriptase-mediated recombination during retroviral replication can lead to uncoupling of molecular tags, such as DNA barcodes (BCs), from their associated library elements, leading to chimeric proviral genomes in which BCs are paired to incorrect ORFs, shRNAs, &lt;i&gt;etc&lt;/i&gt;. This effect depends on the length of homologous sequence between unique elements, and can be minimized with careful vector design. Furthermore, we report that residual plasmid DNA from viral packaging procedures can contaminate transduced cells. These plasmids serve as additional copies of the PCR template during library amplification, resulting in substantial inaccuracies in measurement of initial reference populations for screen normalization. The overabundance of template in some samples causes an imbalance between PCR cycles of contaminated and uncontaminated samples, which results in a systematic artifactual depletion of GC-rich library elements. Elimination of contaminating plasmid DNA using the bacterial endonuclease Benzonase can restore faithful measurements of template abundance and minimize GC bias.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/09/01" />
<meta name="citation_mjid" content="ggg;6/9/2781" />
<meta name="citation_id" content="6/9/2781" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/9/2781" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/9/2781.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/9/2781.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/9/2781.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.030973" />
<meta name="citation_pmid" content="27402361" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2781" />
<meta name="citation_lastpage" content="2790" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Laura Magill Sack" />
<meta name="citation_author_institution" content="Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Teresa Davoli" />
<meta name="citation_author_institution" content="Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Qikai Xu" />
<meta name="citation_author_institution" content="Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Mamie Z. Li" />
<meta name="citation_author_institution" content="Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Stephen J. Elledge" />
<meta name="citation_author_institution" content="Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author_email" content="selledge@genetics.med.harvard.edu" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=A. Ashworth;citation_author=R. Bernards;citation_title=Using functional genetics to understand breast cancer biology.;citation_volume=2;citation_year=2010;citation_pmid=20519343;citation_doi=10.1101/cshperspect.a003327" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. H. Brakenhoff;citation_author=J. G. Schoenmakers;citation_author=N. H. Lubsen;citation_title=Chimeric cDNA clones: a novel PCR artifact;citation_pages=1949-1949;citation_volume=19;citation_year=1991;citation_issue=8;citation_pmid=2030976;citation_doi=10.1093/nar/19.8.1949" />
<meta name="citation_reference" content="citation_journal_title=Cancer gene therapy;citation_journal_abbrev=Cancer Gene Ther;citation_author=T. L. Cheng;citation_author=C. F. Teng;citation_author=W. H. Tsai;citation_author=C. W. Yeh;citation_author=M. P. Wu;citation_title=Multitarget therapy of malignant cancers by the head-to-tail tandem array multiple shRNAs expression system.;citation_pages=516-531;citation_volume=16;citation_year=2009;citation_issue=6;citation_pmid=19165234;citation_doi=10.1038/cgt.2008.102" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. W. Cheung;citation_author=G. S. Cowley;citation_author=B. A. Weir;citation_author=J. S. Boehm;citation_author=S. Rusin;citation_title=Systematic investigation of genetic vulnerabilities across cancer cell lines reveals lineage-specific dependencies in ovarian cancer.;citation_volume=108;citation_year=2011;citation_pmid=21746896;citation_doi=10.1073/pnas.1109363108" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=S. Cooray;citation_author=S. J. Howe;citation_author=A. J. Thrasher;citation_title=Retrovirus and lentivirus vector design and methods of cell conditioning.;citation_pages=29-57;citation_volume=507;citation_year=2012;citation_pmid=22365768;citation_doi=10.1016/B978-0-12-386509-0.00003-X" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=K. A. Delviks;citation_author=V. K. Pathak;citation_title=Effect of Distance between Homologous Sequences and 3' Homology on the Frequency of Retroviral Reverse Transcriptase Template Switching;citation_pages=7923-7932;citation_volume=73;citation_year=1999;citation_issue=10;citation_pmid=10482539" />
<meta name="citation_reference" content="citation_journal_title=Drug Discov. Today. Technol.;citation_author=P. Diehl;citation_author=D. Tedesco;citation_author=A. Chenchik;citation_title=Use of RNAi screens to uncover resistance mechanisms in cancer cells and identify synthetic lethal interactions.;citation_pages=11-18;citation_volume=11;citation_year=2014;citation_pmid=24847648;citation_doi=10.1016/j.ddtec.2013.12.002" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. J. Echeverri;citation_author=P. A. Beachy;citation_author=B. Baum;citation_author=M. Boutros;citation_author=F. Buchholz;citation_title=Minimizing the risk of reporting false positives in large-scale RNAi screens.;citation_pages=777-779;citation_volume=3;citation_year=2006;citation_issue=10;citation_pmid=16990807;citation_doi=10.1038/nmeth1006-777" />
<meta name="citation_reference" content="citation_journal_title=Molecular therapy :  the journal of the American Society of Gene Therapy;citation_journal_abbrev=Mol Ther;citation_author=D. Grimm;citation_author=M. A. Kay;citation_title=Combinatorial RNAi: a winning strategy for the race against evolving targets?;citation_pages=878-888;citation_volume=15;citation_year=2007;citation_issue=5;citation_pmid=17311009" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. H. Ho;citation_author=L. Magtanong;citation_author=S. L. Barker;citation_author=D. Gresham;citation_author=S. Nishimura;citation_title=A molecular barcoded yeast ORF library enables mode-of-action analysis of bioactive compounds.;citation_pages=369-377;citation_volume=27;citation_year=2009;citation_issue=4;citation_pmid=19349972;citation_doi=10.1038/nbt.1534" />
<meta name="citation_reference" content="citation_journal_title=J. Biomol. Screen.;citation_author=H. Hoshiyama;citation_author=J. Tang;citation_author=K. Batten;citation_author=G. Xiao;citation_author=J.-M. Rouillard;citation_title=Development of methods for quantitative comparison of pooled shRNAs by mass sequencing.;citation_volume=17;citation_year=2012;citation_pmid=21956173;citation_doi=10.1177/1087057111423101" />
<meta name="citation_reference" content="citation_journal_title=Acta Biochim. Biophys. Sin. (Shanghai);citation_author=G. Hu;citation_author=J. Luo;citation_title=A primer on using pooled shRNA libraries for functional genomic screens.;citation_volume=44;citation_year=2012;citation_pmid=22271906;citation_doi=10.1093/abbs/gmr116" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. S. Hu;citation_author=H. M. Temin;citation_title=Retroviral recombination and reverse transcription;citation_pages=1227-1233;citation_volume=250;citation_year=1990;citation_issue=4985;citation_pmid=1700865;citation_doi=10.1126/science.1700865" />
<meta name="citation_reference" content="citation_journal_title=Frontiers in bioscience :  a journal and virtual library;citation_journal_abbrev=Front Biosci;citation_author=W.-S. Hu;citation_author=T. Rhodes;citation_author=Q. Dang;citation_author=V. Pathak;citation_title=Retroviral recombination: review of genetic analyses.;citation_pages=d143-d155;citation_volume=8;citation_year=2003;citation_pmid=12456341;citation_doi=10.2741/940" />
<meta name="citation_reference" content="citation_journal_title=Biomol. Detect. Quantif.;citation_author=E. Kalle;citation_author=M. Kubista;citation_author=C. Rensing;citation_title=Multi-template polymerase chain reaction.;citation_pages=11-29;citation_volume=2;citation_year=2014;citation_doi=10.1016/j.bdq.2014.11.002" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biosci Bioeng;citation_author=T. Kanagawa;citation_title=Bias and artifacts in multitemplate polymerase chain reactions (PCR).;citation_pages=317-323;citation_volume=96;citation_year=2003;citation_issue=4;citation_pmid=16233530;citation_doi=10.1016/S1389-1723(03)90130-7" />
<meta name="citation_reference" content="citation_journal_title=Human gene therapy;citation_journal_abbrev=Hum Gene Ther;citation_author=M. Kumar;citation_author=B. Keller;citation_author=N. Makalou;citation_author=R. E. Sutton;citation_title=Systematic determination of the packaging limit of lentiviral vectors.;citation_pages=1893-1905;citation_volume=12;citation_year=2001;citation_issue=15;citation_pmid=11589831;citation_doi=10.1089/104303401753153947" />
<meta name="citation_reference" content="citation_journal_title=BMC molecular biology [electronic resource];citation_journal_abbrev=BMC Mol Biol;citation_author=L. Lambeth;citation_author=N. Van Hateren;citation_author=S. Wilson;citation_author=V. Nair;citation_title=A direct comparison of strategies for combinatorial RNA interference.;citation_pages=77-77;citation_volume=11;citation_year=2010;citation_pmid=20937117;citation_doi=10.1186/1471-2199-11-77" />
<meta name="citation_reference" content="citation_journal_abbrev=Virol J;citation_author=G. J. Mcintyre;citation_author=Y.-H. Yu;citation_author=A. Tran;citation_author=A. B. Jaramillo;citation_author=A. J. Arndt;citation_title=Cassette deletion in multiple shRNA lentiviral vectors for HIV-1 and its impact on treatment success.;citation_pages=184-184;citation_volume=6;citation_year=2009;citation_pmid=19878571;citation_doi=10.1186/1743-422X-6-184" />
<meta name="citation_reference" content="citation_journal_abbrev=Genet Vaccines Ther;citation_author=G. J. Mcintyre;citation_author=A. J. Arndt;citation_author=K. M. Gillespie;citation_author=W. M. Mak;citation_author=G. C. Fanning;citation_title=A comparison of multiple shRNA expression methods for combinatorial RNAi.;citation_pages=9-9;citation_volume=9;citation_year=2011;citation_issue=1;citation_pmid=21496330;citation_doi=10.1186/1479-0556-9-9" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Protoc.;citation_title=Illumina sequencing library preparation for highly multiplexed target capture and sequencing.;citation_author=M. Meyer;citation_author=M. Kircher;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=S. Mohr;citation_author=C. Bakal;citation_author=N. Perrimon;citation_title=Genomic screening with RNAi: results and challenges.;citation_pages=37-64;citation_volume=79;citation_year=2010;citation_pmid=20367032;citation_doi=10.1146/annurev-biochem-060408-092949" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=J.-C. Paillart;citation_author=M. Shehu-Xhilaga;citation_author=R. Marquet;citation_author=J. Mak;citation_title=Dimerization of retroviral RNA genomes: an inseparable pair.;citation_pages=461-472;citation_volume=2;citation_year=2004;citation_issue=6;citation_pmid=15152202;citation_doi=10.1038/nrmicro903" />
<meta name="citation_reference" content="citation_journal_title=Molecular therapy :  the journal of the American Society of Gene Therapy;citation_journal_abbrev=Mol Ther;citation_author=M. Sano;citation_author=H. Li;citation_author=M. Nakanishi;citation_author=J. J. Rossi;citation_title=Expression of long anti-HIV-1 hairpin RNAs for the generation of multiple siRNAs: advantages and limitations.;citation_pages=170-177;citation_volume=16;citation_year=2008;citation_issue=1;citation_pmid=17726454;citation_doi=10.1038/sj.mt.6300298" />
<meta name="citation_reference" content="citation_journal_title=Human gene therapy;citation_journal_abbrev=Hum Gene Ther;citation_author=L. Sastry;citation_author=Y. Xu;citation_author=R. Cooper;citation_author=K. Pollok;citation_author=K. Cornetta;citation_title=Evaluation of plasmid DNA removal from lentiviral vectors by benzonase treatment.;citation_pages=221-226;citation_volume=15;citation_year=2004;citation_issue=2;citation_pmid=14975194;citation_doi=10.1089/104303404772680029" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. R. Schlabach;citation_author=J. Luo;citation_author=N. L. Solimini;citation_author=G. Hu;citation_author=Q. Xu;citation_title=Cancer Proliferation Gene Discovery Through Functional Genomics;citation_pages=620-624;citation_volume=319;citation_year=2008;citation_issue=5863;citation_pmid=18239126;citation_doi=10.1126/science.1149200" />
<meta name="citation_reference" content="citation_journal_title=Expert Opin. Biol. Ther.;citation_author=M. M. Segura;citation_author=M. Mangion;citation_author=B. Gaillet;citation_author=A. Garnier;citation_title=New developments in lentiviral vector design, production and purification.;citation_pages=987-1011;citation_volume=13;citation_year=2013;citation_pmid=23590247;citation_doi=10.1517/14712598.2013.779249" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=D. A. Shagin;citation_author=K. A. Lukyanov;citation_author=L. L. Vagner;citation_author=M. V. Matz;citation_title=Regulation of average length of complex PCR product;citation_pages=e23-e23;citation_volume=27;citation_year=1999;citation_issue=18;citation_pmid=10471753;citation_doi=10.1093/nar/27.18.e23" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. Sims;citation_author=A. M. Mendes-Pereira;citation_author=J. Frankum;citation_author=D. Burgess;citation_author=M.-A. Cerone;citation_title=High-throughput RNA interference screening using pooled shRNA libraries and next generation sequencing.;citation_pages=R104-R104;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=22018332;citation_doi=10.1186/gb-2011-12-10-r104" />
<meta name="citation_reference" content="citation_journal_title=Virus research;citation_journal_abbrev=Virus Res;citation_author=R. P. Smyth;citation_author=M. P. Davenport;citation_author=J. Mak;citation_title=The origin of genetic diversity in HIV-1.;citation_pages=415-429;citation_volume=169;citation_year=2012;citation_issue=2;citation_pmid=22728444;citation_doi=10.1016/j.virusres.2012.06.015" />
<meta name="citation_reference" content="citation_journal_title=Cell cycle (Georgetown, Tex.);citation_journal_abbrev=Cell Cycle;citation_author=D. J. Stone;citation_author=S. Marine;citation_author=J. Majercak;citation_author=W. J. Ray;citation_author=A. Espeseth;citation_title=High-throughput screening by RNA interference: control of two distinct types of variance.;citation_pages=898-901;citation_volume=6;citation_year=2007;citation_issue=8;citation_pmid=17438372;citation_doi=10.4161/cc.6.8.4184" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Ž. Strezoska;citation_author=A. Licon;citation_author=J. Haimes;citation_author=K. J. Spayd;citation_author=K. M. Patel;citation_title=Optimized PCR conditions and increased shRNA fold representation improve reproducibility of pooled shRNA screens.;citation_pages=e42341-e42341;citation_volume=7;citation_year=2012;citation_issue=8;citation_pmid=22870320;citation_doi=10.1371/journal.pone.0042341" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=M. T. Suzuki;citation_author=S. J. Giovannoni;citation_title=Bias caused by template annealing in the amplification of mixtures of 16S rRNA genes by PCR;citation_pages=625-630;citation_volume=62;citation_year=1996;citation_issue=2;citation_pmid=8593063" />
<meta name="citation_reference" content="citation_journal_title=Molecular therapy :  the journal of the American Society of Gene Therapy;citation_journal_abbrev=Mol Ther;citation_author=O. ter Brake;citation_author=K. ’t Hooft;citation_author=Y. P. Liu;citation_author=M. Centlivre;citation_author=K. J. von Eije;citation_title=Lentiviral vector design for multiple shRNA expression and durable HIV-1 inhibition.;citation_pages=557-564;citation_volume=16;citation_year=2008;citation_issue=3;citation_pmid=18180777;citation_doi=10.1038/sj.mt.6300382" />
<meta name="citation_reference" content="citation_journal_abbrev=Hum Gene Ther Methods;citation_author=S. Yang;citation_author=N. K. Karne;citation_author=S. L. Goff;citation_author=M. A. Black;citation_author=H. Xu;citation_title=A simple and effective method to generate lentiviral vectors for ex vivo gene delivery to mature human peripheral blood lymphocytes.;citation_pages=73-83;citation_volume=23;citation_year=2012;citation_issue=2;citation_pmid=22515320;citation_doi=10.1089/hgtb.2011.199" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=X. Yang;citation_author=J. S. Boehm;citation_author=X. Yang;citation_author=K. Salehi-Ashtiani;citation_author=T. Hao;citation_title=A public genome-scale lentiviral expression library of human ORFs.;citation_pages=659-661;citation_volume=8;citation_year=2011;citation_issue=8;citation_pmid=21706014;citation_doi=10.1038/nmeth.1638" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Sources of Error in Mammalian Genetic Screens" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="twitter:description" content="Genetic screens are invaluable tools for dissection of biological phenomena. Optimization of such screens to enhance discovery of candidate genes and minimize false positives is thus a critical aim. Here, we report several sources of error common to pooled genetic screening techniques used in mammalian cell culture systems, and demonstrate methods to eliminate these errors. We find that reverse transcriptase-mediated recombination during retroviral replication can lead to uncoupling of molecular tags, such as DNA barcodes (BCs), from their associated library elements, leading to chimeric proviral genomes in which BCs are paired to incorrect ORFs, shRNAs, etc . This effect depends on the length of homologous sequence between unique elements, and can be minimized with careful vector design. Furthermore, we report that residual plasmid DNA from viral packaging procedures can contaminate transduced cells. These plasmids serve as additional copies of the PCR template during library amplification, resulting in substantial inaccuracies in measurement of initial reference populations for screen normalization. The overabundance of template in some samples causes an imbalance between PCR cycles of contaminated and uncontaminated samples, which results in a systematic artifactual depletion of GC-rich library elements. Elimination of contaminating plasmid DNA using the bacterial endonuclease Benzonase can restore faithful measurements of template abundance and minimize GC bias." />
<meta name="og-title" property="og:title" content="Sources of Error in Mammalian Genetic Screens" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/9/2781" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic screens are invaluable tools for dissection of biological phenomena. Optimization of such screens to enhance discovery of candidate genes and minimize false positives is thus a critical aim. Here, we report several sources of error common to pooled genetic screening techniques used in mammalian cell culture systems, and demonstrate methods to eliminate these errors. We find that reverse transcriptase-mediated recombination during retroviral replication can lead to uncoupling of molecular tags, such as DNA barcodes (BCs), from their associated library elements, leading to chimeric proviral genomes in which BCs are paired to incorrect ORFs, shRNAs, etc . This effect depends on the length of homologous sequence between unique elements, and can be minimized with careful vector design. Furthermore, we report that residual plasmid DNA from viral packaging procedures can contaminate transduced cells. These plasmids serve as additional copies of the PCR template during library amplification, resulting in substantial inaccuracies in measurement of initial reference populations for screen normalization. The overabundance of template in some samples causes an imbalance between PCR cycles of contaminated and uncontaminated samples, which results in a systematic artifactual depletion of GC-rich library elements. Elimination of contaminating plasmid DNA using the bacterial endonuclease Benzonase can restore faithful measurements of template abundance and minimize GC bias." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
    <title>Sources of Error in Mammalian Genetic Screens | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"0pjj36Mq14BU5yn4MBMXNk5xKESGq46XQnCQ4UJojyw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460306","apath":"\/ggg\/6\/9\/2781.atom","pisa":"ggg;6\/9\/2781","ac":{"\/ggg\/6\/9\/2781.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/9\/2781.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2781"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2781"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/9\/2781":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227ytJphPqyJlxc2fcuGnZ917ZOs7q12dYDtMc9xTuuuG\\\\\\\/9ye1u5aPANT3yYHQlzeK42jemAgJLx0ZEyV4F32pgEAyXl7UZ6meSUCJ34GyusaGA7yVQ6GXhk2HQClP\\\\\\\/mYca7971bmZF+AtI9hDIkkXGzy1z4SjhWVs4J3msY9JfEzL+kmenvtgag73r4Ds86XLCcY\\\\\\\/8CbtitqM1TMV0gi7az5aPfu2V9KmLK45doG0QhXDS2Fr3yBP1nicOXLFJHNNzR7hLO4aK\\\\\\\/UgguBvLiEsUpOjOxgq6dWQpmTTh2kan2mtdM2\\\\\\\/S\\\\\\\/t3Mg3lreAFoj\\\\\\\/CQhylRQ+TcXf6kUq+HAAk90mMzRxoYG5agGAV6eRJrXTpmofF03M0NvOj9+8mpo2eUtSRkjonnQfUqcfFdnw3HBznHso+KkNU8UTE9nKpYEh9Zqatf+8\\\\\\\/MfXSBwdM6Bp7mP6JvBcBwBtjPMugNTb4TJ6P+iMAB\\\\\\\/26jwlmuqag+t5DEpCNop0T1nyHIialjZcxqMoV\\\\\\\/YkzAOV91YhUnl9BuvGNTsVqKIGQjMmdbaGfIw0e9YC6unojSjcaprEeaqsTYX3qN8V0zJeU4a1E2pPTNEgTyTka4Uo6U6IS14n5ny4faxcbgxm6NPbfwD3VyDAecfQgeHq3bvUjhs1D7g68sxkTixAm63UbZe2La9xHS\\\\\\\/2Fh+FMhSfH29T65QtXhw19lSEdjxxSSjI+tlPT\\\\\\\/2cj\\\\\\\/k2R2XYc+U9OM21Y5JQqYlhYaRiZRZ2WSfQmYNzuOtcQIBZnzUAd5RHYLrEcmu9OrGYfEq+IHmWokKT7u3Kd9gcRUTQRMTUGWXfvMnQIOua1+Zms5SPaNFJb\\\\\\\/Saj1rwir3qimYTT5viOOfEsvsPxHt94cIVzj49yKfln7u6W1mvBxhKuhQFmGzp7OCtAy5LllMw4V\\\\\\\/tscn+FESqCyrvcZHxN4yp\\\\\\\/RASFm0RhzaNJlF3XIHx4XB61Av2fHgOFTzRqbKq0XuBMMeIw2pRyvF1c87j9ES6Cq8UekTM2BAGQrEZpnh+3PtV1D\\\\\\\/9XQJWcgQds3ZmdwU9GV8XW45hWiKHF1m1lEV\\\\\\\/+DxtrcjLM\\\\\\\/sk\\\\\\\/s+Adnwkcu\\\\\\\/kGtCp9UC0GLOt2D9IZ24FpCnJOX+5I+lfZ7Xx9p8D5NMNvbeXzD3w6O5vxBto+3ykXyYWlQl2OBn5uy7buDBOeIH4guK\\\\\\\/R1Kmp2oZhZAo+lp5c\\\\\\\/kAOifU\\\\\\\/AE6KKoU5NfQ1zEbsXP90ejNkQUB\\\\\\\/dQDX\\\\\\\/nqN3uJOGmuzNPBnlA3vrdnieozJiO2IUUma\\\u0022,\\\u0022iv\\\u0022:\\\u0022IBYgr4xYWy\\\\\\\/UpBM2BBQqYw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e64001069716099b1a7aed9122ff9c28beb369856fabdecf4301f54f3d4efcb3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mdQsKILKD1oDyXBbs9C+J9d+E8hrjS\\\\\\\/dKLoH7trVwSB6SN\\\\\\\/6aXNHN4sATvFSmDLNarnEGa2+S33S2FwfaztoEueAdHAQVCGaXij+EUUff3sA3rxno9jLndw35G2iS6DEBhDpcQQo3YRxjpEDiEztgpEyHSuoK+Ew2\\\\\\\/zm157XEpkY7UZC8byh2idHLvn1Ma\\\\\\\/W1BKSWk6rVmQXRjGiLBCg6yXTKppB6q3zyAGmCkFipzM3pwLNsrPppeTsX01bFWto04LSxZF33G+j5T1wydcDKGXe1liMlxw3uzbXQCOS9tpRKBjSWMhn4ITm+UP8J2YrD6DdS21EM1nlJ4aM0YkyUgNFodAA1v3GumXUrNXEW8fOqoeUJ8TrcvXCVhI0CLLQXSfFcBMF0VB5Fr7MVCxvfvzuIUGfXU3\\\\\\\/lczq\\\\\\\/hOOnYbsbgQ1OwWhap\\\\\\\/xLPJI3UziDe+CnrfTDJo9ulSctzkXJdQjgnobCOFXxBjGMmfrDXR8css2mSfme9zlkyZGQZUOff\\\\\\\/1adz9pQ8Cyu8TFypQn2Rv+UnsjKuggM+M\\\\\\\/6VPHaDDTFDp1wZ88b\\\\\\\/\\\\\\\/pMzaU2LSJMJjF5Qb3cS9\\\\\\\/+ItAZaZn3ES\\\\\\\/25pR94ApBNHr3qHggKLVc2JP\\\\\\\/K7q2QVGU4gEb0jLr73bAkVF+xBO5c+5uzffCSWmFdDCOLLD5cN4JzHO5ib1CkrpqPXdj81LYd7L1fl4p9WnxZe5lUzvlcXigKY06ZiZ6BLjDrNB7HtBBmKY2nYl9qvnTGCQuXSQSLLPZlG+PYzgqLTuvgJ1oAo0Uf74PTWFMl1AXirBw4zg6pjCt1Iw7COMeJX0gYD00LjmJnrIb2CuuLXrKu38wqvA0BH8tJ5Hf0ZvJssCq425BOjejh3gutPmrv1Aw\\\\\\\/665GpeynavKVsA16PsFEyX10axJoNABfwG+8jZj9uU82G9fUncIHh8\\\\\\\/rD6K7\\\\\\\/RX+YxUL0kBTpyBvrJburqIUoYvlv5xUxzcnCJJ22ykUzvL83VItUad63Tqe6fjlrP8cOm2IsBGWqfiwzqesW3SQ4W7XsivLo9y5tXIkFnyHeuFmL2nPXjIP\\\\\\\/WFLESXBz1vHLKSNC2iHTDOSR5\\\\\\\/q0dcYfXdYEtlZgi0vMxWzRSIliMfCKO6+4jHr7pXSsnbokeWkuwRfsSofit+JGNDpMwb0cR4Ky7KQoR+mbEpzTME+IdqfCWohMPblz9SzF5nlxNBooHEZHwgPycuHSMT9ycqZjTELhF9TUgFCitzS+tLlIafH4MI5v1BiUyOa7+jbMjsoerqBOOLGoBQr5vlTLnOLTO8jNVmjy0X4RHuYy2BKarKasNtFxdUmdIJUDCepj8QWLn6Y0hl\\\\\\\/tbP4MQyJOLHErmCEHNAN+zADxs0oSK3SpGLcs8gAFcg8rp6LNZWK5um4KwQvI3pcC1lXldm2On5EhaSKuMDSj7RtNGdL6Wuh6aoK4o+VRjFWs0vHMEFBAqgWKjD2tg1042IlMO3YsJGC\\\\\\\/sQd92SqldLuxrseA4PYabHWQDGYBBgxvc6P9+oLoQb\\\\\\\/rkhc5Yx1ZPkTQKXDksJMq2JgvIwxtPeovhqZsumaM8QKz8TnyB+bKn9VOhuqgqQR6juQGL+cXrWVX0vQzmfmNIG37lBIlcVDf8yZH8mCVOOK\\\\\\\/uBdgVWHdojTphYMyz6sHacF7BbNziMwa0wO2txZM2Bb2FKXeWXftkiPoi9IHC9eO9KlOb3yy3QRhnCHiyqrC+vTnkhvDrcCvnERfmgQWbn2qd9RFVcW8+wrdYm5h34GFwXm8OQWiEliV7W\\\\\\\/VcZ98JUj3+CPg7Fl1r1nW7FtORte3pbta7TA0nfUU8Njno\\\\\\\/Wm9MZw3HTYw5TvQKrVCx0atdrFHpW6UvfKJfxebrbXbAU2kMnboGxau\\\\\\\/XJLUcJ9gkhIMRVaDeuNCbfF+D02pm+uEXbetn6CcRGXf1jb6duM7X3IWoqGXdCFQFh1eNw+Q9sSiB3lytVn\\\\\\\/Y6qFOKs33tEQZ66IaOoPL+JIuDW3JJNmVMTSwrjR1WDZc3w4YtItCulzVwACqf80nGHmeVxTn\\\\\\\/JzhT2iA2gNHL1t2Yoa0hnX9L0tLEs9zHlbtxZgjc2VpHZ6It9MIBFBdOZ4bf6itM174ikm4zxJDNr7F57pGny8HpphhjnYs8H7UI9k\\\\\\\/IF\\\\\\\/YTQoVo0Tfr+HnzpmU\\\\\\\/1prgTTBRN2cY8Q\\\\\\\/tm5KudB2+lHH6jAepYCoieh5IKOvDy30RzJw2z+VjLgkSqNXjDsMDeC7fOTViKu4NEkicqN99+MC7aEZicSpm6nSaD+eM2h\\\\\\\/Gnq9LJm5o0NqSGaUM\\\\\\\/Ud1Cy\\\\\\\/p2GMCmHKB2o4hOdNev9X9Zd41qHaTYt9\\\\\\\/1yrHAl70vAK7fsubTC8WETKtk9TZMl957OAvgSRHyOyWmZ0JDW0vOlTY245fTGF9RBpT6YBPv6GBO5m3xM1pZUv8hAcjOnuZ\\\\\\\/n4JGk\\\\\\\/2xQD8f3a8zT0HabeBpO4QT62qhsAqAKMKIX1HPG32eoS5GArCAnRCWtcGj8\\\\\\\/Bnwx8tLiq1TqGLX6HEdod7MnODRrzFd3rdFu+027ruEaFEuIFiXJ6p4Jl0pNo3s\\\\\\\/gwBOsR8nc0EBCoBcbo1b7pgw0YFhmxIUqLrE22b47O0svOTT5lApc2ckyfDeVR6CRRh8F+494WGVmzhQdt1Lr4Vx55s2HnvhENA4TlTXcvnV+DHCK598H0GosHQTBY\\\\\\\/eThUtlQU91p6drNn1Xsj1v6pfzOAoppsPt6fiXDkqy\\\\\\\/83bSO1DmFEKT6RLok1dU\\\\\\\/3k95Z\\\\\\\/mzzLCrwl\\\\\\\/7kDMlJaYaM215iUzoUi2V1J6iI34zAtXfx7vOAXafrXhBoUlUMd+U1tSsvCemtGpu8TJbFCMzaPbGH6dhyKrBvqK52NSLeXFuNFe7gsrmYdZReCXnW\\\\\\\/dlV2y7HDqiAKpbUCkAY0ZNPijtm5\\\\\\\/kfntrb97tLKe3PhFR4XqKi+NyNVKDgA3bILyXt5g8y4W09pZX+Gza7Vs9eEtb+r9Ocp34UV5OqviC6ZlfCE7N0ChCD0P3CAPbq3n35JXqCraqPdnPZ4I1eWKYOx6c4NTWB2UhXyoBThVlsdzrwROAjc0KHsP9KIALtAiooxW4pes5wzRQYFHEtYHArIX9KpkY\\\\\\\/TDI28bD+D31P2EXmt1TdIam88Bo9VXanMX7N\\\\\\\/cv4DSM\\\\\\\/NfZwRSQvO4qczn0QfeDr+AwOo9au\\\\\\\/bOk8TtxX3yqMURXycgLza\\\\\\\/2K22PLS\\\\\\\/OkBek+2ZVcUHIPg71b1aAcQAAeutBAUEFGBc6XHS\\\\\\\/pUQerFT85rjToQokIw3ryQm1NGz16gMbkreFwcWQZ4QIm4Pgw3YMzj04vcARci1J+ITUkSogTUgaKuTKVefdfDrCoYK\\\\\\\/JEXsEA8DpXst1jmNYpV8o6opRy21FrjIEhDEQdQUpmK3J30IgaiGekAw7UTQ09rVP\\\\\\\/7V861FTsU47WBp1tDNC0AyslyAZoB2M6pwqtRtyyICb\\\\\\\/agRql5\\\\\\\/zu+9iHURyPPn8+m7avOQP9CMnl\\\\\\\/q5M0o1h2vxTypmGGBe4PUeHFjGU54NlBfJoHj5rqGbmEFOTYAfZIRqJB2FirdF+0wRKIrV6kX56Ly29yfCyRgs04w2L9V2G4qBLBvPnh5+gp05btAWYzLXiuV4cPJJX0kJySyyaEF1exSE2ChnOdxxSDq06SFfs\\\\\\\/pOor\\\\\\\/95Q5K\\\\\\\/4q3FIpSceOV21DddkUBSsgFEHc1OHdW1qBtQmepfvfPVyAHjFFZPzempDCZ9r57cXM4v+Ff+EKJ7PcEvk9AdRX4o2LKuhaM2TPvI\\\\\\\/RO5DWYqtrmXnKEYOP37kdxGNj14BwJLR5QkMhMlkPOZvxv\\\\\\\/CJ1I2cg05Fe2mdTH1GEttZYhxqw5GufEIOIMvsldjDtvyyEpPsILvST\\\\\\\/5gK94Uug0rbNz8liSKA85geZKBJI4geebAsGaY4hx9laa5\\\\\\\/taiptqcGkPIYBSF9wkWZCIsAqgPk1K3m51fc8nlT3GXvZvA+nUrHVylzwR5P5wP\\\\\\\/xVt+2DzH+wn49fuJ4Xaq0JP7Z0pdGkePwG0nUYznMyz1XA6RW+QvfcJR+nybSZCt8dvQrAo8EYS0lpRFBz5wIc2ezsLSdefCNVdJ1k+rKXAkDJelkp5YoUG1a+oUrnwMoGLsGdvKuBUP0iJ8ZjHS4FDjwMPY4infCMpuuv39L2dSYcIIkFPCCMS4DoeoehjPt0MKVg1peux88HDQFGMPqg\\\\\\\/9IoXwkE1lfZEM7HQID4svI2b8ME4rqIUcxJlljd16UAoV\\\\\\\/d5Q+4he7B5qAtulUYx9z2ghg9+VdUAf1taqdBKdfDFOiO5B8nJ4AWlQv9KPrkz53CPXOvjuvHmnstJ9ds5puJI1aSLq6+B4KR2albGNME7RQ0jAShFDl9r4G5b8Ml\\\\\\\/2lGmnhf3xO1XKW8BBRjXaFVIdJg\\\\\\\/4vNZKUJq0BxVZ1OQt5xhD7laEOVKSGUVvDdSfNbImRzv4v4mQDMqrmtdSIfNpc46rvy\\\\\\\/qowo7iuHIiv19ZJCZH+51pmv2v96ZpZSMaR2zbF3XWfspOGK9gfrJvPlr7WLOWv6zgXsuxe+sY3auMjfrs3PeP9jzNblgZp6MqARqC\\\\\\\/Vknu2M03TX2eNUZIwMDm1vqVCfLIuYTNwrA48JuV\\\\\\\/XjQmRY+eqykA3zX3kwxVEcIThnuI9BfWOeaUrW8R7PQqwR7nOb9QO29GqLev7UxIB1Pj9K6h301fugZndvmR4hTqQY0HY39Vz9XwvtX4CtgKGzDHDtQO\\\\\\\/qD1XtO3cMiDVVjbK0Swi3uyheUyNAfGqp0nki1pGMM\\\\\\\/6kCh+n8Hm5u\\\\\\\/GNnbPIvBHRfdWgj2kV8+BF69idlcE7xrfQybHKqg55ImfwC96aFW6lAgv\\\\\\\/XSqMdHINQRg7XTsvuOf+A7MDlBstZDY1tub3ry9sWA4dPkgPc0wC0PkLYqblxcAqXt4HiJcXa79qyLuWgCuqqS0b5Xwk2285SG6fyNKayZcvdKGbnzig8+K+V4A53oq5GFmnRiZfaG1N7KhzJcclmRi3tkOZTvl87zM2vi2IjkK4LXtxzmedd2hUveALoUhO1QrmelN4du9hj0caLiaIewCv5mJv\\\\\\\/nwF1oxJVy0NjD8HpuEHy3K0OZWNgRqVXmbO3YBLkLF+SC1k10GaqIZu1uci4AECjZ+pam+b\\\\\\\/E0hJCSs\\\\\\\/kA4r2bSBf+rtev06rgUU0EzznonLd4VX26RzMH7NQR7HlOU3q0d1SE4FZtRcDgJs+FRUhBuBwobJhClEuw\\\\\\\/NxYXWSm2yIwZKRodi6LlWQ3Kxzxk\\\\\\\/oYr5PZWG2isVF4N3TR9oNQ5dwA3FMribl5dIAisCJBwMdWtY46+dYdWo877o4DbpfLvV3wRMnkC1FiX465TBxd4vweiwSsXpieP9VDvHQ\\\\\\\/Q\\\\\\\/grC2WZLFp9UbkcQufij869OH+wfyeO+xodaMAxEyNfZbQfdpHWwRHpKE5Jz+XLDMscaTQqqYO7FLspfqk47VukfTinsA8KwJk6V8JqYQMXZ\\\\\\\/\\\\\\\/Vi4bibdXyihusumLOBgnCnJBleWRfdLpfBM\\\\\\\/Pc35x+SnO1XeJ4i9bUgnbJhbSaOCQyL9NKOhJYwYgi2FyTvnjXCxprcYiCltDrNZz1QmNCTJXLcY9H18Ex4Xu62oGXre1Vgumd\\\\\\\/GmxwsSfcb0nqxTAbPEbjdBAZ4AEowesMbA31uqGYdnn+w9lEhG7J4x662cs8GUUAhFl0Ep3PqASf++cvi\\\\\\\/4f3CTiojoZNzoazHhLnQFczYGiGfiI2D+a5TCgsdn1wy0UYQ1F5RzFW4EUPIZdJpWnEHXLeYxErRNJRe8sbFR+Ve65CLXUVLttFvXIdcrBwkmQyN7AU0oOx0YNvT\\\\\\\/CSBDy1zWjr5jQ4azUG7kjWENcAHOPTyRcVrFL7mWQqU1\\\\\\\/B8eZE3kwnneYzub9BiFXDFjNyiCxIGR6OssPCxY6Ns7VgHBOzllZjKk18Sm5VQ9tci1pADWNkkjc6TKGX9EfD5ObiiSA\\\\\\\/uw9FN0RlyulPHS0EXP0H9wpzN7puQnEtzh8Zqs\\\\\\\/HYVOe1vFrUmcf2qjO3Butb1R1YBrmJXuozbjVP8V5O1d08Q1aZu9nhWUPCeLCUh3CLJnO9DgO54TeTBFHot2ofFLqsMC4r7+8yBQ0KYFhnkK\\\\\\\/xAExzgepScOiMCtxnQHhk1D+6YlhPVSqnF\\\\\\\/uiqTqZ3qFdpyY4BecmL2U6TFb9h8GKKlsg43U4q6ZWTc8a+kpCeAmKayqmQBrBjnMpHXxgT6clBL+HSpp4Z50Oj0bl1o1GFVIX4csxlzQp5hSV0FMGaIzgG65NKdYtv6NIe454LIjIJlHjn8vgXjnMA21viPHtrJFda5j7qr06j776b3a9PobqVHexg1zuwRDBBR5VO12zZvQHvzmaL7pydPC1r+zrb\\\\\\\/AIexGwJKdts2y0o70dh6LWijTpRgFU4JR1AG4Zjfbj9m0iW8HAcrBDDjK0GA0QuDT0pGLlvdJY87DynAxngs5jRKkPMR5fEK8NuR4W9jezEu8s+bZdAxnBK3\\\\\\\/Wr4gkVj0+uDqjLFfCqjiJlknGyMJBMsKDpqk6vlkbVktf1IbC\\\\\\\/8fXiUh4\\\\\\\/Ss\\\\\\\/6R3lXB4rQk6nIOdtKUZU8Vm7M3+GVraJYLkx9EV5\\\\\\\/gmjAIDXVsEWVQWU8F9cLguG5+T5ll+NeaP983scZiqeIRtquyG7akqbEh1qUpnhoixg5+MnzpkQ\\\\\\\/2y\\\\\\\/kZk9CXqQXLxRGXIJurvQeYSiuMQ6wg9W4LU01QS3x563zrRR4+YPUHapm6B+dwjTZq7W8DBZUSSShnhKIda+UK6Vhnyb9wTbFvE6Mq5NA4IltH5\\\\\\\/5gznSz\\\\\\\/0hP6gxDluo+gMLqPq1rN8u2UqNZjthVg9mb800prZdoZLo3KQefBmtV4uSe+XtD1vTINQbT+Ep\\\\\\\/YO9cyxIO9u0mcUZ7MCZobpPgaWY+jiHOk2diL9DGTgFfF4l7DaofWPAL4gjvhs2ZuyY8Q\\\\\\\/2tp87Bg9k35oSwqW5ANuQ542TYXehlUlM08yv2b8\\\\\\\/P7MZF7NanZc7INdLShWeuEz0deB1RolP8udeffwlj9GXVGB6yfSo1JqZ\\\\\\\/bUL4d9krokMzqHlaCxh935EkiH+dV7vtUv2BvVdzN2AyOpUBvsogmacTTwBaemDe1QN4NRXSS+gXqN5orjlULBb8\\\\\\\/r+84+os4wzUKBRS5kglR5yvO7G0\\\\\\\/ms2zWwwiMiwpb\\\\\\\/JpMbTRke9Fvc+Brd\\\\\\\/oJ6Mgl+M2LR30LQsodAO1Qjk67auY6UMTvRj\\\\\\\/HRAVz8XUyRAh34hA94E5F\\\\\\\/0XOEn1t5rzd4NZfFGZk\\\\\\\/czvCASYBrqX3HhWmsH3E\\\\\\\/gaKOLQHlcJwzLlDvibR4zEEeBnsQ6Qh1RIDsqj7rYWHoouI0yNLwDlxtoByNyvMa3J63z6XJQ+vlyQcPgwYzPU3o+ClhrCGpv8OlCNzh7+dAyj1lhbIq\\\\\\\/lADqKx2oimI1NkDWPVDuyQnkoyYV42aKvMCBxaQ5fu9NodvrRz2EV9qUPUHKaBuJF6N1FGgbuuYhTeWeeO\\\\\\\/MeFqLT82qsBlN3l2Lzu6r4T9UubnQS5SCwHS4PatGzq93IFVyXI+jW6HW2KHrDBcGkvCztpbFFuNjpb3rUqJE2xWdGYpZ4NRE5lNXuuAnMRMx8YZhDmlHZydXONIKBLmyQR+D5lMX4dQ02r+J55p1vp47KqzOi4ZXWkKEHPJ\\\\\\\/8Wz\\\\\\\/Pkvl2DtCJ3MsmOlkfRAkXO4Pfsndi6FehkR\\\\\\\/rr2yDbH7RY7aIHWFMLfooCS9qQCml5c9JIHvroXbMuJDawUxWPDU0tYp0jeCyf2rMy5flki8GzE58ULGcbRoj28qgr2idb7xuYSy7ZsWyvBeKf8kBfzeAP\\\\\\\/QJcNGMEOAu+36\\\\\\\/DcO4O+iXq36CYq4B7IZ6tZRhtLWsLgQP8zrEZR3+Ei1cSv7j12m6+tQZip2FALfpf109Ni6s0pGcebnJexsrwe4OGjVrh9mXp2LxyKKVCpISDVhOpuOhYqonLl6zP5NaFY4R90Gdl7rxawsAd3YzKs6tZndearxILS\\\\\\\/RGiLldUCjd7kIHbUTqIG3fQQTARroqakKjFahrWLl\\\\\\\/TKHQ+bU9a\\\\\\\/8rlvFqVYGYypgogv8xt2U9RNtjPu9g7G9j5ltJDDnpqpAXGjPU1KjtIwi34CRWQcc5iTt6jKRccMbGjrCkS8srY8W0h+A8\\\\\\\/4sQ4uL5\\\\\\\/vDS0r+golMVhCP5IC8tHH3CBhjrQ8L++axOaIKEzZhfCCN3zbNErGgMOU2BwjcgZ6NwuaN4wyFHSn+IY+\\\\\\\/KK3gggMb0YPejm7uSdeZzC4\\\\\\\/5dHrEd7HLlgIlHQf7sRBhraYcD9jpuF+LX+v+zGXQFyM9QQX7Rxoq9DfwImZfNj7Hp9gaAQq7el7AbtB0r7oizNBp6YJQl4WsaLaY3z0gfkNx1sV+k0YiNBmckNg6g83v3a1B9bh8Cy0\\\\\\\/sdRbC7x1xrAJXCIPMvcJ6wsRrUS9ADDUJuQrThlpBzcRKhKoYTEHlxocelLJbFxOC0DfPioHKB+u1LVwpQDcDMpxiatCyGjusqesAQEmY+xvHM8KLl02uPNIM6i4UL5BN2JIrPw+Rv4H1NjAHRPHCim7nu503E5hLtIRfot8ph7kcJRL31nu6JieDsG954GCAc5yxt4c4XmahTyez9Q4K3V2GMfOpxvrqBfdgB5PV1HDH82uNcxBgNSKX4u\\\\\\\/RjjspittsNd2NtHSOQ2TRZcinNGDssAIX+DPx0YxLi3CTiCbPkaFwp\\\\\\\/Bgs8Qpy6WLwMlvf6hcdS5JxF8vRVIPCPnwB+1\\\\\\\/gnB5C7iNRBTNM7tfft\\\\\\\/XIKC2qvyJ6\\\\\\\/qtYYEG+FDRLDXAxIRvuC9MyOVt\\\\\\\/P05xMZK676rYqUBMTr7+0Qt9nRcbYrMFGlx1\\\\\\\/H2az\\\\\\\/+hHKhU30OcK9B1aDLWjCIkdfn7Re0AfOuHB+3W1TzT9m7eB4IYQiCRE\\\\\\\/7qyQ4tq21HzhuryY3Au9LHbNECc14\\\\\\\/4Olh2g1qG1MP6ytV+kyNmGyVU\\\\\\\/AyfL4YFHQi0LM+jcMi7wiHj59Gy4rEUGkQgBpiDF6jUiTHgm2sf+6v3jQDFT+Jf4UyfX930Cc0AO6q5jpf7qFZbQDbl17VLrzq1faaBXN3dBTdwHI0HEFylvhZ\\\\\\\/PVfIXzr1AtGJMuIcVLjOGoc0R6kE9aob3mv\\\\\\\/nrvz7eGR87xAo4FpM7uB3g8KdMakWiPbRdiBBcBxe9TLduBu8Xoqpz+tKt2K+AE8W8qUIdRtujDZ9yR6H40qbyCTiHDaK8p+0Y\\\\\\\/EyOyGm1D+bgN2LrjswkfZ+cWbrcBbhcPARnmkqsGRsGd4Ds8\\\\\\\/CTA2ANgtbQk9BZ8fSH3wgVgAK4\\\\\\\/LJAU+mHkP8vJk9I1b3fpVzH+jtnnjzysWMxBzWFPTKd6Sdg+7sc6oIe6+f78Xh+AY\\\\\\\/84IB8OgnBnPKv6N3NAP8n4fJVOEIYkQ8o\\\\\\\/W3U2B\\\\\\\/g\\\\\\\/x\\\\\\\/27SLYI9M79BL46Ugzjcio\\\\\\\/lcsk1xHIJGkdPPEwGdFfwtKYbDQVC\\\\\\\/X5NT2b5dgBv48D9id7h0zzw3J49sNdMSzUbT43JyM9vDjv7YYzpghiCJt0XNXJAMRUIxg5C88XvsC7pZEYG4E0RswKlBltXEuTPOMeitWOs5QkV5fRztWhlCF7eysi2kRMwSlUzFq0s4dWvyPA6I2+FQa1GWYVAauCwABGhnmARJjKeEnoSOL3zv9ZW6psD2pjfNQe+XZQvhd+uUfa0pu\\\\\\\/w8\\\\\\\/UwWJ+GEPL9jWfSoPSGpK7lkEORvHoPhOe7ChmQruooBV2zDfsLqqpvqQnqHlZM+8\\\\\\\/XVUN3P9p5q0zCQqRFnLzQE45+eoC+tNaIkLS6jQiJf5BG0l91XZ1ctjO0IP4+Nbe12AeJZZIlfsh3XuPC7NxbdJMAw8wnH9GcCLbGXiyg8rMuYC5Bs7dwejtUVM0pUqH7F3HDKy8s2QolPMH1LDjrADuRhrKwM0DylGdWO+8Ikglu5ZTPJLW5XBrMglXPbeupwRHGCkz+57VNPnk8jLubMe+We7LeVzh\\\\\\\/A2n76gXO1l1u51K\\\\\\\/b07rYXFUi2K4JwlJhTox\\\\\\\/OBwAN8Z+EwCgZO51048623iPVsZt1G7hfLQvdy20GLXCz+Tl7zk0adoF+bPtdVpGnxOZqqR+eZe7FjMsXoFmN9v849xjJTduzsVXaC42nH4RquWbp3StfAeBZU75jRb7alyyHT3smy2ecWM0EmqLgTvOnBdtiwRlglcu8qvoXe2g4IiOwAzremu+hCIKt9xpJiz4akUldc1Mn\\\\\\\/krJvjCCgHtMygrvJKcDZVsowPLF28IkcXYw8gai2gdPIKXiqR65ERZYUElBn20\\\\\\\/+QCDMCD37uQV8r\\\\\\\/mAQPrua5ZCmE76t\\\\\\\/lmvBWcwfWjHb9GmlVisK4jrpUJqDH0gtRjxCdnyHP1GuzT2OcBVqthmw9cy\\\\\\\/5RQ6xyQ2q6Jbg89rkchPwfnH9yXQ7No8Z\\\\\\\/LMQGFSf\\\\\\\/MwSGCUy38GxTcwDcm3piODlcGwOK5MInaKkLpAuyOziYS1KfzV29lC5OWlpePbexbD30cy7x299GHaQwYo8aUF99NwQNyyFp8hlrmjpGo4usNW+CP0y7g\\\\\\\/DLfhiHoqUNNu6Md7EOBssPv45iXlZu\\\\\\\/HIvrqNHPWpNo2BEd2j581lGw52J1\\\\\\\/y5aCtKOLh03QY0WzzO7+kKxSrmeiJN5uzL7YVB88\\\\\\\/+\\\\\\\/F30SAx5e9cRTy\\\\\\\/6BrjytgMpOI3QpuA0RkcnPJNSb6kI0nU0WPyW6MQsTQGU3lOTzuEPhXMCxdOAQdxu9vSf54APDbcIsnly6KXBOsUp6CR5ShaUr2CpcpIf12W2E1d2Knt06o5NNVubRTcNL5bQ8ybfG\\\\\\\/g8Dfw3SEptTSL6XeRvQbJW+G7g99BoOmwQPAS+rfCPHMsMdYCyCDzbyxw+hjKjYEBMexHLRb0D4QXlk2C2rCAREalGShEqpSQddyXm02Cnh8mta8OzNf0PGCHB+KUVYfn0M0ADvrn3ouucj6doxAlSpA8bfP+uEKgxn3Ra5X0\\\\\\\/CmHbuZvkuTcB2RcJHEjbiw7yA9e3nh2Ue6niz2fE6PXWPZxeQM9yr25Yi\\\\\\\/lJvrGOAjMD8odzrVKzaBHMCK4BHRf2T7EW2TR3gtMxxS1JQ77ssth07mAd\\\\\\\/4y7C5nsBWQy3ukm5ndvlVIl4H8TWPtJSMMSsEcPABV4BkZcwM8437ttFLOCfUpXlsqStzz4WFEc9y1\\\\\\\/oxO4eh9RNF0whpUvZLCQ8z+bgQw9KWSYTZx0fx4\\\\\\\/nBxV7yiDB76P0m8qKxiU16WRlgRQY2Q4UDqA4U6rHcaU453MHDXdryAkTCGf\\\\\\\/28qUE24twTRXjYFpDR\\\\\\\/1XVZc4NbJdta2Q0G5ozpXv9PLTbkQY4\\\\\\\/vu3LKBBuEaZsnZ2bGsVDRLQ8gRsls10AsBXXXGkJO5fsZ19s2p28eslIvF86S9ObYzUW6V0SDcv8IM+rifqCPVy6CnH\\\\\\\/jfs1NDa+zzBaG0VLwCOEXvht+Zpy0kRImtRhvh466QwOvTVcdY0n8uavthlzMMYMp2ILrALbhcZHMKGTX6TkWQ07edulZi0rsOO5N9d+FM03tyGkitYW6eP4Z6eIe+C4uD+GyAn4l9sPf5NmfY+OYNbQT1KJEmIxpeVdpM2IbR0RjOvUmzC4YsX9CzHBPHhjbC1JdefuKxjFs5xc\\\\\\\/JXPo4UItHNp72jjKnwZ4tsfR1AMTT2ACiSY2WPlwH73DWfNqOkbmcra3nHKA0OO88VQa5x3fN7+jeaFICRRGXNukw7wI\\\\\\\/XiKckciRguWavkNhfnY95VmuJaL011WoHWkgXoH5ZG6b9PuTscihqzzWxd8ppn0RoKIThzd2bylKcnx9etcXVqh3lS1HwV1LYzDRC5EMbuq6wwAif3qYUamcK5tLtJRxkU4CbOjIrOhsF4XrN2KlJ38I39PFl7JOTcC3Bqk8x\\\\\\\/cIY6bLCTS2yYB4\\\\\\\/5JTcDyB1DwtakKx+PtbHJSR+nbfYVMkcumF1FpzKXPQorFuLD8QZwRVoDZBWls0Bovt9QJvQrqpxQKHCl6y5trv6hrnj7yG0i722Ytc5zx3b\\\\\\\/rs4U5mEViA8jaDxjA\\\\\\\/n0fbHrxtUInC3gEz3k9sykEuE1Haw6WjKONaov+GjSSxdny8ouxjk5XsgMsHwvO9+Yov8vd\\\\\\\/SoYwrIa6dXCljnWiJB9wPdM0HR2VF0jBlsB\\\\\\\/8XS5ijyKaT6gBG2lXLm1R25sCDL0wNbGDU63iGvul+hTguLDoPGBfi9jEfwT9uDcgjnlGFDgXgGCRAAqkCpWM\\\\\\\/IPd3ycj5CBclWzQxVp9KxGtdg0Bv9eUIjLUuoEDe6z4568\\\\\\\/GYe\\\\\\\/1vr8d9fU18zXXtKV5X2HN0Am5HLP5zKQhx3TFmmqP8hocebL4BONDZzc9W+M187ZfUdIXCtPM+DjVxG3BBWDBz\\\\\\\/EIFFLYBApkIHgDSOi5BbUFRURYCRJIi79RI++BhkyQRLbU0AKW1bjlzY857369WiuiLtjovOV7FOvqQohXGSnhnMPAdKQIL87eR4HZBfw4ugHqHePsQVz2oYY531N6YUn4UOhC9mKCBLb+tWVaUHMLluRDO07qYpsLXddF8thBo0NuuAwEs5CYK\\\\\\\/6BUstx34MzbzpvV+a0zFPEq1RGW3d9WKPtSFonseUL07OAKCKz5sN6wqQno8I6p2YoXW\\\\\\\/3kWOK\\\\\\\/tuasFNRP1N28dtUjlxJyiuzQConcvigSWZLP+y6SGfhqx3VGlWNeOFD2RUYlgXDOT59RgxksYwYUiRydJEe73antaYi5DIwdpSaioBb4SDVA2OIHwTID74asbsDypBobw8kPuBVTF05EruodxLfcGJ3Bs7BfGkHjBfPuRS8o3IH+DNmnIJWsY22V4IpC7ThZ1dSFEo1vcbMFuyKb345O29l2Df7OBM490bDdoK3y06eiJz6n4wdCYeC5vgLMB\\\\\\\/D+EZi6K1wIVfDbjs90mD92am01OZcMeKmRH4zjCp3GqNoNjy7j5lR2ru0VC3NODUpaa1cP9Z8yhkM5fjqeN0+7ctgU0g5zD3iieFT1\\\\\\\/kYHSLvqiRy5W+OtErxNmMrUVBnJVZJKt8VjoPP\\\\\\\/N0ExZT0lBQ\\\\\\\/GIA9MxJULktjp0M4svXTw\\\\\\\/9eEaReQkBzp4JCSu0LClvE\\\\\\\/\\\\\\\/xi0giFk165UUDNPDa\\\\\\\/jydYEddPg8qlz0sA2nB\\\\\\\/\\\\\\\/rUO+TdSz1BnrCvFkyRzgteJunWbRw8zviMGAj0F07N1Wm\\\\\\\/hZNOZw6CWxXqVCwW4DWyJxVjBLfZi\\\\\\\/VdTa1nAsbI7FriotoZsyoYApNNJi8ZZjQ31jIqqQDyEsNZwYke1uqwskwhnqvA\\\\\\\/\\\\\\\/Yane3WywXZaZIduEm9joEJzzWFGHB0+wtg0hGRJeG1xHprHfT5+qW3gW55gQMtGhNAJ+IKLcbwqy3DdUtGPPFaH2BHfZfdMapA3XYYV4OhuIglHxAjnxqQCyctqchLdu9jcg7nc496AFY0hZe8wUimX+FT9Dd8xqZd6Lxw73h4RH\\\\\\\/L0PsAFNidst7pLi6mCDyTzmOKIQacZlm4b6aTXYCq\\\\\\\/jerxX9jE4SVaKr5BeQUnO25NAwe6qmwZimM9mfeonH15RmzT+9Jg4Z5j8\\\\\\\/RH0I+O6Ve+8Lo1IimiHkJRgjm8hgUTIfpa4Dz7hw2gI1Z6DFJXKzNNbyw5OgpacUl3yQwAkSmRZU3sJZsSbW+D7DF3QtT0yTwsTGl6DLLAHWjig4UpYqPvmAQF3Dm8pmqGYCuPBdgt5LgBEE9vW0wxIpdaE81cPyWlTrqRPeTKYn6irisv\\\\\\\/HLP5TFyzupDG1P8cqsk5VUuP5YvY4oVlgawXfU6\\\\\\\/z8uItftBgS4x49JOXxE+qlCDletH\\\\\\\/bYVfxYrHYZEnZPnCYtX4yz92RS+izGQIYhS4U1BiEN1ID4NZr\\\\\\\/LHhhNgVMbhTGL9qfF2GrSU7lkabF+H2+KGxTjwLOUXgKlrrbibwvtsyMfJ83uQEVKqPYlhOrQPWc8KDwUxN2SGWMsiG9CAJpVslVctWOXWNrbuqfn9\\\\\\\/ZeuWBUiiGUdYiv4BmArk66xcEs1uCRw58CRk8k2L++C+rJXqD\\\\\\\/kCb0a5SXNFE2nI8hP0uaiNcBv7LwymkCE\\\\\\\/BUg3Cs6+OEqqrg+lXr74KapBTsh\\\\\\\/ZtzaSN\\\\\\\/j\\\\\\\/q1dmtceL7dIwHC\\\\\\\/xqW6yzmXFMcGrHLfZR5\\\\\\\/lrq4keLyOZlCjh3uuuTVCFUblvniN\\\\\\\/yFKi+8gd8L7VYmcfCKqTLpZTJGQLyvh2H4\\\\\\\/LsZPbyv9IftWpeqgt6T+Y1MiTvnCnvzLolazaMOUe9xxCFBTL7uetpoIxZjET+bGkcfJNZLnvHxO6Lkk+qNR\\\\\\\/TLEkkWXZIkK+Blk0wTtnIiA+yuDICgh3WHbEQHlH4rXWSzQsYhps8fGfCD1UWolsE+hFHKwIXw2hIVHD0si+B7kTPZVor5wCcruh0eRJT+lr04ZEKJPZ3AQC\\\\\\\/Zk6uSB8n5dZMlpsGBuPSxo6HDnO5ej9EhxZHjkQmeAvWMznbnJ9pFzH+HTUcDA0OkrpBfLisz0YZP0LOp0iKQoOFZvGEVZRyj7iFlQyQ1VJIIzROHO2SIQf2ATTR0w\\\\\\\/xjI+f98WQltPZn2FSGV7hYdDqiFkXVCnxThWjkykHvqJ2AOkSFeYZGqDqeUjZHKbfovZ3EPc41axuBYr\\\\\\\/4BBkDr3ZFybS7gDm9kuXgu7qZ9a5\\\\\\\/mFkvWC+outoPmMsdtQzaCeRSBI1W1iEoHQkhc9ZSb5L5wjUCsRxGo36T8ALI8oh5HEC4lGgHcbWMmLs1SZzejgy2ELq8ZAdYPfOO5nXDtkA7oXWjYo4xojfQiJdZWFjfo+J3RuDGvjDcIx+UBny+ec66pdb36aMeSJJHDckD62+raBwuVYNErrEj266RvRUk5YfN77m5X40jdWv4JRwwVATdsbbhuMpfp6sGqKX4zQZWNnpZSLuVANR2wX+rnxYph6VQMbA\\\\\\\/bdvhTWN1kmfZLAHceakkkS1FvqahYA4a6JDJhztu+Tb0+vdMtTlUGHxcUmYY3ATiKjTjG9Kgv6PPtayJrATz\\\\\\\/6kt9uMOagdKP2gUE04ABKXNqiyo7bXqX+K6FQpaIlkXhq6IflmGj\\\\\\\/a4a\\\\\\\/Tji0U9w3s0K3kQPEfJZDmWaGC+6ZXu470YKwnGM48Rp1mU5mNjpx1jYzvMK93Vuwsm+MLmJ7Rah5Fc1T2y+r8KbRh1W4PKVglbOxP8fbfYgzEP0EFqESKshHCcFaEA95DjLHNY9ukFjGHiX98jB9bfdXOsMZKLFsyoiUQvf6MS8ocwy2MEfoJxb\\\\\\\/CeGZerGLDBCFhJHslUNLc6Vs6TPOBXWpXH4N19MBRBWULY4eR84jJ82VpTmv2sasK+ixEqoViMnd4UozZSwt7zQz\\\\\\\/x+Aaz0gjKfSvfDRr6DeXCdKq+0lNrNwEE4uCG5Gzu4zIyQZn41u7pVMxuLBu3c4h0jB4FrMaiOK8h9Ta9NxpAPLnNBydK5JydbW7uijGxjCmEeqTndbDfupyo9a4kcP6j\\\\\\\/C5kPWoD6b9cQUn+k1NG4P9WNVNJC8PVT+7bgs3WfdqLrelzYRsyBYzoVq++A6Jei3zGggufF37swmBD4E6nx1KwLaaTyKuya4ikC0lLaAqEFdVDmLJNeFGYhFb3RH\\\\\\\/6iVv8ZF6d9J+jUhSpIhI\\\\\\\/jjtQCfNA\\\\\\\/zet8ZZl3QdM6bie2i+2nRA6yCd09WP+Js9liaOu9nBTJeyBAZuVnDyfIVLMb4T7N30b8l+kycQODJt8TlUYeABpRp3Hw\\\\\\\/5\\\\\\\/BE+jQ3eyx9lBC8O5I9p4a8JNujWo3r3Q4MpFp4lO4\\\\\\\/q\\\\\\\/rDWDrNufKiKptrUG3s3pgf1K3vqSUkr5yIoV+RwuIhrdIqQ2oV8OWFlf3vKPNfGP8m+HNzDR\\\\\\\/CmQFeo+qEn0Nq0axGJa60ouaPLbE3\\\\\\\/y+4QKdj1YwaxzXDuKCV2Jj5h9x8q6oggYp+Lkp9zDlXUJExOzSiJdlFn\\\\\\\/p9qUDrXgRiVtf9Vsw07OGycjHJy+UCaim3dKIKIeBPs7WlygXPyyw2L31nf0DPCjEpO0haReAEAJgiTBZN5JmQyIgTqtIdk4J5oZkBGEJvMRHbHOcWwxi3QWP4oIIGsECbC9BXOhpTkM6JrnlrxOdHA40yrR9fewAdnDWFuxPf+GQ3El1Rc5z4s4MVgc0mkoqZT0YApzvpJ2cFlTycvUfGU+xvXglb7cyVrMShAe\\\\\\\/B4LDNOnCjSDWu+zoTLZJgM7krQ+bGSBiqAb5yCGbwvXLv8Ef2+FUuAKrLJMTxa8ROSc2RD3JwdB3LFjDdqcTJEc8dXpwtQz9zWdAMn+LDqTWTH4kYQOc7VhDDZcKCcXKsIK8Tgojlvg+SeKTarHz4GJPtJ\\\\\\\/AOJFlMP05TZ\\\\\\\/vC6osoGTONHJZFuZ2K\\\\\\\/i0HxOwgxIwJUrT2SApguWLJLLuw9Gx7LFPQcZOZTw+TA6DDMIOh6pnIhsrLnUJT7hRouKZmNU5sxASZ2mnIIWzoznzhJ4y0j\\\\\\\/Ko7qV8rqT12u2JruKwSvFn6vIhWuWtDev\\\\\\\/TizAa1PYGeToqZLno94fWVgK+qvAud12GlJ8eNytXKU4qpCozcg1uJWbPP7xcLegg3xVz3a7qpp78IilEaqTh6OXfEB7OXo0KqmbFqK4ZgtCF2\\\\\\\/Xn1TZaXZR4CRd6n791+BSU\\\\\\\/je4Tk3ivWUYK8y7KLOcRRAw73HAE5k+Ev9iSPN8s5k6TGTT4U46bvJtnhl5ryrgxKtPTeIvA51ReggMr2e24r\\\\\\\/Z7xqtlan2kbqy3y5rW1ILjfePnsOo2wKpWxIHQ0A+PisH2ywvoyZtmdYi3hd4ng6Q6xzWKuteX7ILYuGZwGHYzwLVYIgFcLaM2F0MTR5Ujg7OsRP\\\\\\\/hhQ\\\\\\\/TH2xBmdiCh1p19Ud7ngwYWEP1K05+8PIyma90yWkBcfVMA0B6woj+cFOv9bogxXtuDwCcDwV5eFnNmBiD2PP3Jq4lDFhJgftGUy5voykGga7a+IJGnKQI\\\\\\\/mK3hbsVAfG4eUyE+h2RWaQqO0rIuLwbCIR20efiCHaWSbQnBB1az8VuSITE0TjKNgFpeIVWNV6Nuh\\\\\\\/j1xvn\\\\\\\/2OJKH7zWD+YzdY9PwjjiIYgDoizjqz4ugb\\\\\\\/bZ+h9tmNyh7TAws046Vbz5hDRqQHumPyyfbSh6HsvY2gjX6YH6b9WUSlUORDknHB+LmU\\\\\\\/\\\\\\\/MiSxV+59PPCwda6tgwUW9TXg\\\\\\\/Vve+OVfvAWo\\\\\\\/6UOaIF3V8Y5kDOLznPc4OuakXldipl4Ih463d14AGvu2GJwULVSXKEBSsdxeNjIqymTO4BN+1cJSZgVjXXLYjRFfCr1vtb2eGhOgRrGHeDgnlkqmn8PnW7NUPDYbXKISxsUaayidyC+aNZhk95VPKkgssSNZpInjG7TqQmtTJVx5CTqvcEbyVeyGANFmWn4qQh3U2T\\\\\\\/aRZhN56oWzN7NHiUyXCEoFqAactyYBU02w0raPzqYRIxUOAMPDVBN\\\\\\\/xU7\\\\\\\/vbqSIaguwMLRPbJo81jFQ3gbrw11ca\\\\\\\/f47W053LjOutvc6\\\\\\\/Mrm1C1QKwbtNlMoFH9hr4JoQ2h1Pa6weCdZ2IVRuipgzOQ7sL9YHM6AuHEUDezOdDC2xktluJC777bIAoB6VAHBMo3bcHd\\\\\\\/uTyFFX98Ks4tb2ltIzAqlN9IdnR2lVthowK9ErHaHHMkc0r+xZqF0EJaTMV\\\\\\\/SKySO0b5Sl0VT8ilTZ7sQrOQlMtsS7s\\\\\\\/U1QPct36HK\\\\\\\/TJ8o40YXxZe00GMSi0RKq0pCLrPJv7y+HhUcEsjlcI6\\\\\\\/TMGBG56vTk4HMxjNSZBrlD06HQh3JjM6coAvq8PiuUO7yVXVVlVrAvBlKVNjsnr5PcIRHQceOpTIrGmX9ugwlbffBjdN\\\\\\\/XD9viH2GVFwhROoBfMIo1Ql4v+JF6byxp90agnSuNmiNC\\\\\\\/s\\\\\\\/BHuxAOxi0ykPO\\\\\\\/MlfLjgW\\\\\\\/csAd33GcCSW1KBlLJ1lJBvwYIW+W8tJDH7jyNPSYyU4P0GXm65oiLBNpKepN1WFicyR1E66ofpYC1XlF0C8eaM6U9x9jPsdqBy07z1UYPFPVaWpEHg+vaeAZ3M0stm8Hq0bGdOllpSfLoLSXWNAtz+0V5KxGvv3+2K1Q1cl0J7JgFZ3BX2oc3GCU838aqp\\\\\\\/eSe8kLXiZ617iKoBEc9QMLb3R2clQzPu8z1Qrc+zvQ\\\\\\\/m87th0ovYs6Etw\\\\\\\/fvqEYXNfkKEQ8grL\\\\\\\/YWRfjnIjmZSoX2abBXlR1OIYbx8oviwgualg59Zm129rJu6x4B5yCe62n4dvAyEVpbiUFvV+1jb8L354R0gspsT3aDd5m+iI3E7NMEt589QJahPTHeeIdJQzqxJtFemx2hHNtZwvj8+\\\\\\\/SIpIpiYGu3atfxHG2MNsdIhRIE7y\\\\\\\/rxpq7pEPdR+dpQ5MWBlkgquJrGHDJfKzK46a76AgfKTZEvxbPZIUYc2NTtFUqJoCACTEbKybcIfdNarvqeiWDw8svuPyZFewuSNF4LoZ458Jc5CetsaywV6acvmoXOExg1nzbF9X3cnk51DHKg+cZNLiHXJ\\\\\\\/kiSVc5ZmilZ5jPjcihr5rt0dCxYA+zCR\\\\\\\/k4kdiZ+zDetEDVci9Xm6lSH2+N4u08zripw5yXXiOCTdLGrVHmfVQuabo96O5FUfOzxLvnqCFPuvdef9wQ3y+w5sksoPpuSTEqDIGBp2rAEece9zlofOFrBZWGHYDt1Od1dCXb94T01oK5nEbt8ljnXzLkCn1GRTqvzhu01VVvKLlHqVFlpAvPba93FB7\\\\\\\/bwZ8VL7GLiMFrlOUYlW9fDD4NDvlRnV5+ZacKPSRwDoKAwmG5ylBD2b\\\\\\\/VPRCvooofNC1t5XcT2J5ac6Nyud4s0raqJ8dPPzHGazJGyUW2bqJpU\\\\\\\/UratgbVqNKR1NeRWq8uQew7E1y7DTVxF1HYXwPfBRzRWUcROdeB5\\\\\\\/+SEawcnHss2p1geRIaA22fmodGRdtqJtU5k+SdJoIdu9IFGjivbcMz7iQwI6zyCVpCPBJ4jrL13DKwP1UByN\\\\\\\/+mo4ay4jOWWr4zulqs0RXr+88PkFI6lx6+P0DY+tfsf3kKFAGaloa9ZmF+fM2\\\\\\\/2x5A5st2zqCwreuQ75C2Osj8ifQt8ND1A\\\\\\\/Y8z6cnm9pQoPDjT9g0XvhtrMJfr1jESTsKb3pCU5Lh+Zmfl8KwEXGCJQYUbP387IIk7Unjj1Z\\\\\\\/qAqZ5vYVQmmKoXZ+QnnWEDmd9VNiCYubrsnr3HiYR78U5fh9ZQfbLWM3TqElgpYLN74mbRihRXp27oXn11KjoebTrJmSYxHUU0aVl1+lKMkKDK1gJTh\\\\\\\/LdW8tOtnQrkAT2A74wKVv7mB7YOO0yXBHXvEtxFjZxk7XKYikY8iNLGJgBCQTSakWr5JLnjJA6K8eQtdwtxrO9YVcOFl274n9Tz9IdPM3dIAppiaZQnXf49Vxbaw0rJQfypUbg+mC8DpQbVdp3zNimTfh95K9JyrL+HLv7qRT+7xn8eT4fiiKJJBaIV+d8sd\\\\\\\/hGSny2iHUVPh2yQNncZq5nQ8G8SJp09jrOeaORPuKJnQDVTJg+XgE3bwnWNaFeah8SqULx+vLsJelEfmT3mbd85DKpEaJFkDDtZ+dEL3p2gskvNd4c+60oq\\\\\\\/fNMJAaSrq+s+DE+KtEwHd8Id\\\\\\\/03rd3CGSaxfAckuVLfcIaU076mUnqXjbK7DnkKWOVxyBJ+0GLK0B9vhuzdliCCU6SO+\\\\\\\/n58Zu0HXmDlUj8CEk1v8U\\\\\\\/HdJIxClPbkEGZzjpUmZ4uVktM1XSDD4U7N2g4IBlT68\\\\\\\/2Aepiz2XFveU8zWfb5Ui5N\\\\\\\/mgkoAf7kFtPv0qGhrg8EXIRdnDGinumpFu0LuUb4h0NyPtI\\\\\\\/w8gtGUIWSK94iK51eXLRzCn\\\\\\\/cH5awysEBgCaWQZzYsqUfvQ0bT1g721tbEkVZ2tpn+PUNZ\\\\\\\/tXrP8UCA01JWeJYPXbSGB9RPV0j2zi1qk6eyaPQxjB2rFGezvtrR6BYGN\\\\\\\/qrzFqaPVrbIPAcm0ZNdOML7FENuwboQnu6i5GM3gRegPVkwQeY846eUQO6GRSLvzXPuu50k0QaIklb1L6pqKNHp23BooyKPRczT8I+1+MfDdMH+hEI2HSgHQxPsh80Psnb7LoLWAuqJPl7B3AReB6t5BP4PiRQTiw3rJ0AQABdTplMLRk7pK8BRd2DpmWY\\\\\\\/NU13IlKHXJS3tsT1RV2OmxrAPYzvzTvj3h3pF3ei7kN2dNqLowuPyuQ+2L99YsCJ5zDsd8T+juXKLc14NSh8w6RcGgbqd06q72+yMPsaAb6NFj05gdJVmUZhDmeXzXZ7SgFZSMkwd9FSX9Cktpxk8Ij6PYVR6NsXO4YO\\\\\\\/ruq0vwajv0Q8Zrjoh6OIzJsLrvAFXfSIWG21EcFz3nLN+V84ZTzFLp\\\\\\\/vwNHB3YNtWmPGJcycqedFKuxYRzuu+qKcYIcrctQNsuIrhLLovXv\\\\\\\/lFyDBhsN8badqMggXt1awRvdKGY2ClCcoavpQLXHzYWFxuAK2\\\\\\\/Y+XROLL+MYbIvFpBKIQUMp\\\\\\\/js+1WhYXtFS3Jsa41Hc\\\\\\\/RahY2bSO4KRLi\\\\\\\/Axs39AQF2\\\\\\\/1AfgG28X2igLN5nLnZyC9EBa0kqbPAuDzRS5FX0Va3W24DmNluesaOHFX9YC83ICqgYpN4g8z2q1oIAZP6x5HmX2b0wSLzXCepDj8WmCTRX9VLUVmnPFXeQpnZeGxO6zSsqTShwuKV\\\\\\\/DPpeor28jGoNXh0LGXbVnaZ7MEjGYcjEVTLQcUqgTxaqCxu+oFpCbcHLIMAt4cy1Z1YNcok9wm37VHPqnyZcQDkgYzuEzyZHqol3prs8ofdc88hl9ptzjFobsthzjokeVaXPFYcg\\\\\\\/wUxdctwNKEZGYWBGQWaSv2C8bdAsBm2eK2adIsn4DqUf04VSKJ+z2HQ4YDJ6f+BUtzq1Sv0yZspTzDorohq0gNxuK\\\\\\\/WFq0mhNgzdsNqL3pkms6VQDr6IA68iE7nFg6AhTEk73SN89vK6h\\\\\\\/tgvOdMXXy2X0DYB42FfgLKVa+9FEtXvZTJ\\\\\\\/tghkGe1jTaPIT7udAKJqUpvvrmEc96EOTv63zu7\\\\\\\/4TxsaoqwMJzMu5FC\\\\\\\/ESoLWWgQLso6Jzmo3r6Qfi9I\\\\\\\/JvsIOTAqFjppv0Jya32JNiAv1UZrQ61TrcWXrSNeMDZRbCfDuVAP9DL8xOiCJqcHEfIdCdddyVSP\\\\\\\/X5XtnQNp1Gcvlgnx\\\\\\\/lzc7y35rAl08bO4GOUCtwe5oeLJICgPi6i\\\\\\\/LDUFA\\\\\\\/Ymxez1sV6QT7LMuL1dXWe9Xq3PtcOPQARB\\\\\\\/vDPPCgmSpGOrEmsLzNHZUbFAhms5DbM+EWsU7Tr1VQzRawuddMcMM\\\\\\\/yV\\\\\\\/S53A1ta4lKxsgORJSnqhJVPlh0\\\\\\\/NO4aw1lf7NNY\\\\\\\/hh4N17vo8Bz+zR\\\\\\\/aEcfYjpWGinEOTVXk+xCT6UXruS\\\\\\\/a5\\\\\\\/kxzgbpjqzwSuzy4I2RKz+wGE3N1D0Kjpg1GSVWHWBgohOndusN9ctkgjq6GyZgLSKYIvkw2O6FhNLktjZj34UK6JeafiQbHu4IR+cg1I5v\\\\\\\/kR7w2uI8LdZY6tSdRCnebtb+hd0gluXK0se3t40WT8+7gc7QjjRdueBBH5b1uer5pFsztobtqU3ab8zSQ60znIiOhhztP+S97\\\\\\\/bTy0TiTf++iKjor2yK6Jy4TeKAI23fxSS1WkrsK0xLD+WrXPIcfd\\\\\\\/nB7z5pdT4m6JSCU8CinrZ8FucbN+IvjZAfEKZhS1ltd0VS5j5dI\\\\\\\/RDdHo5cI+PqNlW4TgZCzY+M6H+cFO4l4MR0Z7w9T09BkZCvsFvhuRPbDqOiSWUVAUeljJJq7f3y76o7YciVSIk6gPJss+JVerQxOVFZoJuvWBcQt\\\\\\\/ENkUmstjCusFwH3igyJV\\\\\\\/WDZz331Rb4QUtDt05SBmArqthllb8hk+UNDzFAU8UUGom0tEJS9Orjm0f2hVpmNOb2M5YD6FoYb54VTt2WuJHF0V2olXm5dGh+Y4EQDjJgBZB0WD6LrJFGEZ1EBCYAusNqpdhcVG4V20t84f5FeG0DaLsTechbot\\\\\\\/Qo1xZqUg619lwftLtLn36ex6CEHk06usTnHfHl1jlfFicocPOYTOmPsR8vtgxsGwqVy2EtnmZ7XL1FLCDtBhJ2ZNXu\\\\\\\/LdVMC6Aze4mS\\\\\\\/KrvrY3K\\\\\\\/wPWn9XtXc1IqITR6SLBSn8UPQyUpK1\\\\\\\/xMONsmCMZTavxypx4dERoUPpxQITgxUaWeVqfzRzv+n2dluDDP8ABVKBpM5cSHquzAW1okPArNO7NapFhu57Euhyh3wAQUKyht9L2QVeW6eajXbw80gHXRoddCCno48xdPXK2Cu6SS31i2WyypimtXHyc2r9c0iMNRZfLG0Yc1iJ+7hFUjE0S\\\\\\\/HrKY0YC1ZiMuxLXGv1qxrk\\\\\\\/fspGwaLT8mZVds74iVmTlbmJYXXFmC\\\\\\\/p7FQOP1h+kzt7dBnWABu2tvKk7bH2\\\\\\\/mC2yEjiAJCZTmw\\\\\\\/NrCrmIHmKJqzdkCmFz8Q8u0yQn5+VhTDccDZYaMkCm5WC6yPLXrGziInrr63sMraeUUV2E3ljQl86NJ7LWrxSEe0+ymuZdvactG298mj7dd0yNw8y598MaOmFprxXTSj9e9+u1YTHDuN2mBnpQmRslVgpT3TG\\\\\\\/uTM1Ey1B26VSzt8ZBatP+7408g9ZuPqUOWLMrBR+3A51ZgHRJ0wOC52i9xQpYVvx2RAdaj06KP5eSB\\\\\\\/uvQvLGG09HLJTInWYjRMSwPENoo0i66MqtJSZPdbOB8GdYbv6BLggHMHJS2uM2XIzDkVFWh5UCY4zBCVKTQ2h7sgGgTNw11TVSr+y4JM2KVOJ8FdAF4YYprFWc+caFpGDgfYT\\\\\\\/FsvpzpGJjTXWccqMqlKNDsjlOSO5+yPfMC2Udk0FKW2CvqFRb5G4RpcO2+VNBo\\\\\\\/+mXbqze3dFly0mGCGr9uXx99q5FVE0EmZCodPUTABKYz9CmtT\\\\\\\/kxm3mOQPARTFynYQpXcVhMWgNkOUNc4rDomaJuJyQkWu1jTylBCBjhzd4GPkY6Rntsum1hpKYtT6VZ01x0tuQABg0Gc6q5Ph4ekyMhBGZBRamAkNhuYkW7FK\\\\\\\/JXnVyPWUjRi4bLbDc\\\\\\\/o01+OwCy2SFeOTbMxZQ6B\\\\\\\/XdUxt6szIsdV8m7q5s8jk\\\\\\\/wXkLYmd+bvkEiBQiSOt0ABViDanvJdNdHIgy7mKPnquEbieaWASGRZWVb2771ViqQwxQgl+UgXZ+pL6GCXSm\\\\\\\/aCuBwiJzzb1ZLj2Rz4JtA7+V3CRF\\\\\\\/qEbV1BfGW0R7DuPnM6rMi2XiDdsmcWiSGV2gKY4DA\\\\\\\/dCV6JmRjV5PYi6Qvx\\\\\\\/+0A0MwQlxTQ3iDw4YSEJ+1o0im9+kGJu5BBr8r4vKbRg2PpRmPhf1XJs5kNyMzn5hLpb67QwBwGJ3nVbYNgbNfVaeitluMl3Dw3wWNs5YyD2bZ7UC5WbzRngTlBO+APU2SIJ1csa3ZUBzUqc1nH+ipdmgR\\\\\\\/jpTLMCYtzFtwvbYUbVhb940KCCqqFwFs4AwgTLC4VfKtlHaTwXAe\\\\\\\/9IUTJYofZde9YD4J9AEDo+5rxpwktkmJxaPFqz2LvewD5cANC2VU9y0LN1CES\\\\\\\/hMGQIVTHOpiU5DWFh20dwVMlFe\\\\\\\/AFevF8YTGSoJmdndwRAXMLgGmpwrM6SQQCaFRC1NgmnqyLlJUeGn4Sjga1GMT5u0gMIpoKbiqv8HWRSbZ\\\\\\\/paaVbR4h6CR1P4\\\\\\\/Vp3UOh6i2WYz8wlfK2NmMeX\\\\\\\/XXdq4b1AJwGQgEsHC7JpmAWufZUL0A\\\\\\\/D\\\\\\\/7mBsIWqevdSphbIhei5fWH4HTgOqUMUBTM06OY0NRDgOsjz9I\\\\\\\/jG\\\\\\\/jKl5Jx66FNqRxZ4CFn25Zt36ATKzGZe0xawLeuchxaGSVblqhAztBH++sfLpZFnaZwKyEtwPPn0yzYK3bzUZzpucjEgNle4D8ZQ6FvDbFkA96FaRTTvWx+baDK+a5CAC\\\\\\\/SWAnsF3SQeBT4Moj26p12DFvy65UWC4K2pw4z7+ezii3XTOtGJo8hPzDVIYoWUnHzNWTMFhkCodFVdym2rP+4ClhkIFkT4wpDsGY4S24C80kFapgPzqkB3Y1l2YBo+97I9MtiEa1DlcDRldyig8RiXOSfrt2MUDm8\\\\\\\/dIP8KDfDI6fTcKF0KDQlnuY2\\\\\\\/2SUgOy8e9OABFZSyWBGhYSpArIzrD4Xoqi9kcaWX9srwHfrH42AKCkZVMhc8D\\\\\\\/Ank1F5cs\\\\\\\/1fCthfMukusrjdMsdCj3QH0raFg7qh6bKLKz\\\\\\\/SJ8f+pj8SH1uGIS+kw4jUzmXvdSKq85LwLeXTqoSPR6HfFba9Hi8D+hVVKc1ysPWRp4url8P5fCUQUYuDEy5Tc5ygNw7yTZE0MuBTMwb6xaWuFqAAwODqtnM2W2mfGLkJNXho6oc7N\\\\\\\/i\\\\\\\/z0pJ2CJFI5BJJvb87XlzTUiVk73OOuJSraUn5IMTYCOycp8yu7hquR\\\\\\\/vXdMiKzdNy4WUBEccuvm8h62Ece5I2TaAapDfo6xSMLSTpQ2WL3Dxk1a4nqXwYNsVHkjgwOouZSzbCRX9MhIyN7XfQs6Rd5HpDkmzsTHEbR54jsYOfLbYunGG6Yi8gfevZolfrutx3GPtAroR9eB5uM5maBNreH9QEO6e\\\\\\\/0vyJLAKcdUr6lawqKS+PCalwYmPMjiDNN5sR4GQGgJKLBdDAZ9Bc1i\\\\\\\/G0hEu3kulxhlcbsSR9JMs3c+3PX2cEu143VQZKQVflIrBleLk9wf87eoB1b6EUv52GegKGc2X9H+lHEPVWSzFPKxoVcWSS5+jscwEDwuK56MqMJFblrmH+rJTkxqAtLcxUAUWcr7sxM1mq3fsosmad8iYnV24gcQcfILCktuz8ixfu5mE14mW2WubwQxh7ELnN\\\\\\\/PJvV742CzgvZdmd8ljYa6kbPL2lUKG9tabi2l2fBDmuVBoX6hTBqWg8G3ZgFLwgcBSZfwLoN6yk4B3YP3UihkJfct\\\\\\\/1Meon7HxYCeOlZHkrzzM+rAm5Kb\\\\\\\/4z30XDzCq4yy\\\\\\\/tBgyIGECUr1krID6LfAB5YSm\\\\\\\/fJaAeiIIj0g+KXeB9m6Hof2w\\\\\\\/QVt4\\\\\\\/f1ch19OQ8bgCycXzVXW5OTuf+2xeHTgTJEUfELvh02K2jV6sLm+JD5apr1tjyqtKKWVnMyWPuvjfcLAi4jgMgGKT0aIh8hNdcgnHZ8fzjBOiL7miRc7X7lmDS0vJQdhPPsM6Gmuh9Z4w\\\\\\\/51SayYyLCABgl08lHKS3NEyeMKqs+oJfwkDrjA0Rt96r8ZBXwjIsATMl+EYmHpR36yEKRgTzL\\\\\\\/4lgg31oqcc3PKAANqb4JxDKiCsuuhJf908qWbA4t9qEXOvEXU4gMpxGYXDQoBgPpUvHe89jGhAJpMBmFs3BGTHCTa3Lh4iur6ssDgW3jkvcx+g0c+b\\\\\\\/ssQsW1nbn3DFcVC12tlMm3SAa8eTBjfBL1HhqLFp\\\\\\\/Mx74pQrhT8+zoA3rnvRtMMIBR99A+0XcF9OGR+6eAm85BmwnB1lKhkHPLTUGPFlBjUKRH+aQ0HLpPRfhS6+4X\\\\\\\/tsmxn1rJ0yDN58gzKc8llnZNp5X7XJUtgrvYKEauheuDaHt1rtaGqxxzT5Ygv5UA\\\\\\\/8ubYPtPHnqx2StOg9oTb07s0V1+K9peuv\\\\\\\/B2VBpN9ZgSC1ELhbJod7wg+8vD5SHUjYXUMj71HjcbMUFYAsvWnNvvdH3Hv\\\\\\\/c0VLQEEkPgNMuuKF3SPRTZWCDzOkH834qQ4gAzGmOKrpJJmiMebznX59fmoPSgCM1v27hasmnhttV\\\\\\\/ILwfY05j9RADVwIszQ1cw28I5qMX\\\\\\\/Y9OBRJI5oEgPaLKOiPrUe6\\\\\\\/MZmMUqMyvTiUk9PcR\\\\\\\/XDAyw9BWclkDPZavppiGhjcy6iDnX7vES7VsZnyJWGAPMBK6lgC5pXrCCpt4jmU7VjvvowMrgRFMwH0ypdxEtcm7Duxk89TsJyC38uMlRFGV6KejGt1gohAYYqiDNEKdyT0ndK3JpqWJE6P4LuzI8JP\\\\\\\/peFb\\\\\\\/9HVog2VPLhx8OZ6bk19YAQ\\\\\\\/d7ZPXJeTSRq2VeucD5dnXaOQo24jCBJrWwNNrvDJOts6+qimqVqmQOlXKc0BeEEwWXCIqP6Mujgz3ODozN3awQMCNkdGbk7Z+wbAJEMTXDUUEjxHtd3BbJ5nibBtgP6vnQCNmk\\\\\\\/o5YDw28ZFe1Hy9Zpxw\\\\\\\/xCKDnDI5\\\\\\\/lPkMQTpndR2EIj9W+rugPjXGeD0cDkI9PTry0ZEzsO0OQvEwjEZjGC9kbFlpcpk7gnjr3qSkN5VX3q1aIe4\\\\\\\/qH3p1NplvTwTxp+sOo+McKcw+uaRrZtin3jMlis\\\\\\\/A7OsAAkdMj27aqUntsnBwJEs3KhuN6WbaCqx1KYBcQ4BHARwsL39\\\\\\\/bkYNV2JZYEZ0VN+hAf0lyTS7+ComuGv7473yP\\\\\\\/kGA7lYEOBfoO2iTAUDmeeIPgOCtpGC2epOiL+BVu5jdE7+aJdL37lFZPYv2k86TArCJOHiJ24T0EW6mQ9HtHEll6MHT8ppFNU2\\\\\\\/L3pjJB2IUR2MQwfnXCoD6yYnNwdgY5hcVwwk1EOP+ym4GE1Zq42mCgzNe22SNYV4RwcNeINUS0vPs0tzJ9YXT76E2szjkZqLqofUKYDYDqftvhPzKFqZnM6LtfsEjPq5IWWk2lLcr\\\\\\\/stZGX3QpSPvOgwT2rktcdga7cjM7JDjGLs+zFQ0hUFtoOSwwreCoxifcN6U9pUmHxKxldkAnppCgoLZ74OlEnBrjhiTPIFGh8xyxNRO5SKlZmIS5Zq0jJNKSJnCWkg02pjg5xVDEZkLH0EdSkXz4tLXpMxiLqRN9r\\\\\\\/CsYN1jYqumw0daD7VGnqyim79uIvldzNbqy8+aTMPH4wErc8IdTsORR+qsVnLucCthpyK6LxXyCnwW8Mqe46fG\\\\\\\/oEFYmnzl72+rmgtxpZwxHWZx4IzBHHUC3GBbwJJSjxeYYJ6cD67kgHxxBz4P5srP5o+GRxBmTe2WGfg7TMZmGHaJz\\\\\\\/LJ4ZuFrEBVzcBC1DzKdA3d4j+XfuKNajjfMZNykQXb3UoCXHe4FTNrP7TxY7hDlg6ICmNWmtQpLfDSgyqa85h0O4UKXpKma2uFiI4XeLv8vmic79eGClpSmd1UcGYzymw6VFMICmYZhFD\\\\\\\/kMCLb7oRgL8c+DZMPYfGyB\\\\\\\/fy0l+fL\\\\\\\/5dqVBf9mI+24r9B2i28q7Ky6DPzYZ8YM7occ8i6+JLuhEbpyt0PbIk7eKgS+0udC3NZv8lp5Q8m3Fop8a\\\\\\\/diKjvSQy\\\\\\\/6oHZ1DmpEX\\\\\\\/1HoE45GZbGsXvObS4TmiqP6m3WEs3WaeMft8FGN9yjoUwA8QG\\\\\\\/EEicVfZNx3bGVDB1Mkk18ZhyRrMFlEK264kMnjR+gLwRtxgj\\\\\\\/XGwL0V9tLS1i9TmmyZOAM8b+\\\\\\\/ufu6ZgThOR1oedI2ymadKWhB0wKuO3l2PYS+q7a0bj\\\\\\\/\\\\\\\/TTaLuwxweSFLlSQGRBIZLbD36Ld8357iNPLLr5sUE\\\\\\\/Uf74+hbsOi14hwmmg81DwkQNBOJbHZvnrgEqbYPhVHkI4FpyFKmfzV+M46YvMaZN6agYivbbHpwCBG\\\\\\\/C9K\\\\\\\/0ia83XAofFtyTEegJ6RgXcKLvqRSmUV3IhicY0FKZHsosD2inDWElRgsXK\\\\\\\/Rr8kvpSAgdE9vjSWdv4StpPTowi1U12b332tL5u\\\\\\\/Iqpb8Y4cBaijy8JnZC+Po2ZFMcgCAHR7DBGSBjsOArXGodHDUiVEKfd4fLRAf6mlQ6ybR8a3eOKLjWEYvfG3tBLSxeDp+Bvr7G5ueA0C+wsXjUda+RzLG8ch\\\\\\\/IPbsyIKuobg16RNaIIv5+q+pII3aGSW3smRZWtlpH+S2xTurMpsRWRsmYbWokUCLicK0mMT6dwlbNQEpAHqQD6WRt4qyepXFZtUGRWSmLhT6MJ96OYaqg6Bn0qyLpqpC+a+soYviTPTtA5fHoLUj3ENbICfcEtS1NKQXGSjua4cIL+jzOUI++DJEq7183doXXP4ZHQU6AhdLDBk5WUezW+gFilIghPwmjpvaqYTZQGDMXUXGxw\\\\\\\/CV0X4Nw45TQyi+x9fUqniU1Z6+Ctlg6naLAIdyuNBiPznX3R0yuzQVSp7YkaNDg7Eu4WDbn+bECgB9KbxOusz+hz6Tx9cPzFseiTZoftDivmCMlewHNyRu3vkB+dsrpmX8QGOzxmqI1B7KCgbREF0tsA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022iMLsT9VAPe07RAR3gpzdiA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220b4beaaed3944269d04637c9e6ae98f05e8d11bf80f271a8b9beb9412a950ee9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zqwoNp2NbkNytkEfvyWqCpgUJsnDRddh0zdHSQ7NWawAvphKOuKjULh3b6NlcNxXKa5qMJoqRDEIwXa16G\\\\\\\/Sc9wh+H\\\\\\\/9WSh8QPs4PcWhp9YKWhX26VmKcTfJMFoVMW1q6IneMk2JfS7UPkDxLYLovrotY0sY8AaduSJ\\\\\\\/yChoBcHRIRvnha1sqlGaJaNiptC9EO7rivVvRAje1uPp6UyfyQVu5Y5646YXUUUo3G0d8PUiH8ckp\\\\\\\/p6EcDaowHIDDRlz3PBwtbzLag628v+PxYz3XQpN5WWosnyAgXRNRHkWnOp7sXAOw8c+Sv7nm7NsprCi2E6dmypn6qaylrJ1OqcLe\\\\\\\/\\\\\\\/OvkHJF092NTjv1XjoFqUXrzauGRjLMl45eJE2V8ITLydT9NxneMPTmo1O+xA5ClB9W4nsnryOt66xgm1OCEmNvP\\\\\\\/nj8dpNuFxGdIJLuCzvgIMXOypAo+U9yKv5J1vBIzHgOq78QW5GcAlSq7qEXc9ULFeSv0eFZgM5lE+gzMF5VmscJxCKvTjWgEn9TQzvCXTjrf5QjEgpS6ZlJjLl89fUYkI1mz1uiU1x87lOOTT\\\\\\\/BnQV+7AyxkFH1Rqs795sd4lqGsu4KTBHW6\\\\\\\/qGLnG4pc184LTx3wBLaCnPL8JYgv5ENdCzQwPn1Rmw6XnINGc8yuZG9yDrgonG\\\\\\\/gzcg2LXfqXwaIgWnfxoggwRusGkDrQ4VlW\\\\\\\/geW5gOiMTBrUkq6tGreGOrZrNpkv0fH+LBdZs7vEpko\\\\\\\/nJFixh46hLOoO3Fxx9ARoNgagl\\\\\\\/k7\\\\\\\/tD09Fo4EfCnHllPmvtcv+J3cWHZZbAWTCGBTZq2KKRkHWFCAhMam5fDNaq6m6Q4WdwQG0XiveyeeBUBhzkIIonGysDU8VT04tbIFCo7iN3Z7qIpYo5ihRj0l10v\\\\\\\/jKDfpw7JWvBl3mKwnYu8dxqExQ7D\\\\\\\/hbT6LG2Y8nLh2WoJzEfyU5dtreNAgtypnZvUGJtbIT7wdhzAptg87nMEwITEjFMAaG9SeFQzMNrVxD+YXgdQWNLmODBnSoDhlDiEAq6e0l20mBX0hgzpmHR1jMjrycxl4JNepJ4GValM+MbvLjrOMZg5g9AcBb5x7BOWKnRZcCPVLgljW1FdAR0YxTCXyYVJBvdflapLTRachh4qg7ca3+UnvoYa6kmndPzgISN2Ax\\\\\\\/AOTEztyux5pjicDASzO1m3gjsokfZUURAvg8Yq6iFqLY869S03BU8BmUg2XbABFx\\\\\\\/TCWwH3tbdtwfXoHO8Cpq2pWcGwPUxF+YqtAWjM0e6Nk7d3n85GegQ9BU2nU7\\\\\\\/neUVT3lRdVwi0Ly3JvqsTkel5cUnoIdWVd\\\\\\\/QQgnLYOu6XKTRpJaqJzONKJN6kaSQZoOUKGtE\\\\\\\/7ZhQD14H84rYXoBRrwJ1Xf7nZHIi9x8jNNbuxFX2K2RVWzyAXOdwDs10V6lZPzl+vvKe6jdMbA+IJWZhsAubPwgL3J\\\\\\\/Hw9duYmEO8t5vAhGK5h45IAME6BMNbm1n3x\\\\\\\/5HAVatW\\\\\\\/Kz3p5KGf\\\\\\\/MsfdkDjjfq+M90aAcNkQMKiCoFpGQrsadqo63JabcArs5h5\\\\\\\/alM\\\\\\\/2aabXBmtnoMvlL\\\\\\\/QpCJF0qdeRtEKuq8q7qTgwylzs+RI5+DsHMdcRILLyzgIFvfjwCL12Kgct5LxOj0QENrDxeUDGddWc8f+APe7DJ5gtXLRtFDCppby\\\\\\\/esionBvMBOCUe2JUNPGtIxXzgh8LPGq7Se++fNo4o06yxbT\\\\\\\/wNwBob3YCO2b6Rb9J5zOKdyVbR5FDQWedOQznYZPJ+9M+oENVHedAl1\\\\\\\/yJ265HPkTMsMonzEZLRD7qecoLerxFphaHdWbSDhmAIoVON8NwVTELcxpplhcf48\\\\\\\/pnmHxff\\\\\\\/zac3jDexCUYxv6od\\\\\\\/igD8h4j\\\\\\\/D7JjtVAulLmsox8bWd\\\\\\\/8eAqx8Jf7eMPDjDJ3WF4D+Ay8iY9AuzA6Qal8DWKUYcTzdoUosm6d7PsSla45nJ3lWCvkaVctvzDMmaaR0HGi8Q8\\\\\\\/2fHZiD3gK7w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022yo99Khp6vpmftDtMeJBNFQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022711be5e59170e49cce8f22767039de0ca59f5f319791d4df4e5a6dcecd70467f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460306 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2781" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-l1eAUwrBq3z92uIhkrqMT1CIq6GSarYBPiJMGPuaoD4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2781" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xPzig2XRnNOS0MMJEyxmz0PyLBEwYpNGHjvHSRawli0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2773" title="A Comparative Analysis of 5-Azacytidine- and Zebularine-Induced DNA Demethylation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2791" title="Whole Genome Comparison of Thermus sp. NMX2.A1 Reveals Principal Carbon Metabolism Differences with Closest Relation Thermus scotoductus SA-01" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460306" id="node460306" data-pisa="ggg;6/9/2781" data-pisa-master="ggg;g3.116.030973" data-apath="/ggg/6/9/2781.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sources of Error in Mammalian Genetic Screens</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Laura Magill Sack</span>, <span class="highwire-citation-author" data-delta="1">Teresa Davoli</span>, <span class="highwire-citation-author" data-delta="2">Qikai Xu</span>, <span class="highwire-citation-author" data-delta="3">Mamie Z. Li</span> and <span class="highwire-citation-author" data-delta="4">Stephen J. Elledge</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2781-2790;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030973 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460306" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Laura Magill Sack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BMagill%2BSack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sack%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaura%2BMagill%2BSack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Teresa Davoli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Teresa%2BDavoli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davoli%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATeresa%2BDavoli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qikai Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qikai%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQikai%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mamie Z. Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mamie%2BZ.%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20MZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMamie%2BZ.%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Stephen J. Elledge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BJ.%2BElledge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Elledge%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BJ.%2BElledge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:selledge@genetics.med.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">selledge@genetics.med.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/9/2781" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460306" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460306/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2781.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460306" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460306/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2781.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460306" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460306/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2781.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460306" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460306/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/9/2781.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460306" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460306/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genetic screens are invaluable tools for dissection of biological phenomena. Optimization of such screens to enhance discovery of candidate genes and minimize false positives is thus a critical aim. Here, we report several sources of error common to pooled genetic screening techniques used in mammalian cell culture systems, and demonstrate methods to eliminate these errors. We find that reverse transcriptase-mediated recombination during retroviral replication can lead to uncoupling of molecular tags, such as DNA barcodes (BCs), from their associated library elements, leading to chimeric proviral genomes in which BCs are paired to incorrect ORFs, shRNAs, <em>etc</em>. This effect depends on the length of homologous sequence between unique elements, and can be minimized with careful vector design. Furthermore, we report that residual plasmid DNA from viral packaging procedures can contaminate transduced cells. These plasmids serve as additional copies of the PCR template during library amplification, resulting in substantial inaccuracies in measurement of initial reference populations for screen normalization. The overabundance of template in some samples causes an imbalance between PCR cycles of contaminated and uncontaminated samples, which results in a systematic artifactual depletion of GC-rich library elements. Elimination of contaminating plasmid DNA using the bacterial endonuclease Benzonase can restore faithful measurements of template abundance and minimize GC bias.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/lentivirus" class="hw-term hw-article-keyword hw-article-keyword-lentivirus" rel="nofollow">lentivirus</a></li><li class="kwd"><a href="/keyword/genetic-libraries" class="hw-term hw-article-keyword hw-article-keyword-genetic-libraries" rel="nofollow">genetic libraries</a></li><li class="kwd"><a href="/keyword/pooled-shrna-or-crispr-screens" class="hw-term hw-article-keyword hw-article-keyword-pooled-shrna-or-crispr-screens" rel="nofollow">pooled shRNA or CRISPR screens</a></li><li class="kwd"><a href="/keyword/barcode-screening" class="hw-term hw-article-keyword hw-article-keyword-barcode-screening" rel="nofollow">barcode screening</a></li><li class="kwd"><a href="/keyword/gc-bias" class="hw-term hw-article-keyword hw-article-keyword-gc-bias" rel="nofollow">GC bias</a></li></ul><p id="p-3">Genetic screening in cultured mammalian cells is a robust and effective means for discovering novel regulators of biological processes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Schlabach <em>et al.</em> 2008</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ashworth and Bernards 2010</a>). Pooled genetic libraries are efficient tools for this purpose, and are more cost-effective for screening than arrayed libraries. Several streamlined, optimized protocols for pooled library screens have been published (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sims <em>et al.</em> 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hoshiyama <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Strezoska <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hu and Luo 2012</a>); however, such experiments are typically plagued by high proportions of false positives (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Stone <em>et al.</em> 2007</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mohr <em>et al.</em> 2010</a>). This observation suggests that a ceiling of detection exists, perhaps owing to noise from systematic biases inherent in screen workflows. Here, we describe two sources of such noise and suggest methods to diminish their effects.</p><p id="p-4">Pooled genetic screens typically involve transduction of cultured cells with retroviruses or lentiviruses encoding a genetic element of interest (shRNA, gRNA, ORF, <em>etc</em>.) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cooray <em>et al.</em> 2012</a>). These viral genomes become integrated into the host cell genomic DNA, enabling continuous, consistent expression of genetic elements over the course of many generations (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cooray <em>et al.</em> 2012</a>). These stably transduced cells are then exposed to selective pressures, resulting in enrichment or depletion of the specific library elements that affect relevant biological pathways. These elements (screen hits) are identified by PCR-recovery of their encoding sequences or linked DNA barcodes (BCs) from the genomic DNA, and measurement of their abundance relative to a control population by microarray hybridization (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Schlabach <em>et al.</em> 2008</a>) or next-generation sequencing (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Meyer and Kircher 2010</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hoshiyama <em>et al.</em> 2012</a>).</p><p id="p-5">Each step in this process represents an opportunity for the introduction of noise or error. For instance, retrovirus and lentivirus (a type of retrovirus) virions each carry two viral genomes (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Paillart <em>et al.</em> 2004</a>). During reverse transcription, the viral reverse transcriptase (RT) can stall or dissociate from one of these RNA genomes and reassociate with the other, resulting in template switching, which has the potential to generate a recombinant DNA provirus (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hu and Temin 1990</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Smyth <em>et al.</em> 2012</a>). For viruses encoding a single shRNA, gRNA, or ORF, such recombination will not be detectable in the final integrated proviral DNA. However, in the case of pooled genetic libraries containing thousands of unique elements, the two genomes inside a single virion will nearly always encode distinct library elements. We show that, in libraries using unique DNA BCs as reporters for library elements, RT-mediated recombination can result in chimeric proviral genomes that uncouple these library elements from their associated BCs. This renders BCs unfaithful and ineffective as reporters. We demonstrate that minimizing the distance between library elements and their associated BCs can substantially diminish this effect.</p><p id="p-6">PCR recovery of library elements is also a critical step in pooled screening approaches, enabling sensitive and quantitative measurement of element abundance. However, PCR is known to be a source of bias in measuring abundance of multi-template populations (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kanagawa 2003</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kalle <em>et al.</em> 2014</a>). Accurate, representative amplification of such populations requires careful optimization of cycle number and reaction conditions (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Hoshiyama <em>et al.</em> 2012</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Strezoska <em>et al.</em> 2012</a>). We demonstrate that variation in template abundance between screen and control populations results in inaccuracies in fold-change measurements. A common source of such variation is the virus production method, in which excess plasmid is delivered to packaging cells during transfection, and is then carried over to target cells with viral supernatant during transduction (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Segura <em>et al.</em> 2013</a>). This excess plasmid dilutes out over time, but can remain intact in early samples of transduced cell populations used as the initial screen reference. This plasmid DNA will then be copurified with cellular genomic DNA, and can act as a template for PCR, sometimes leading to large differences in template abundance when compared with cell populations from the screen end-point, usually collected much later, after the plasmid has diluted away. A further consequence of these measurement inaccuracies is a bias toward enhanced recovery of GC-rich library elements from plasmid-contaminated samples, relative to their recovery from later time-point samples, due to a difference in effective number of PCR cycles. This difference leads to a perceived depletion of GC-rich templates. We demonstrate that the bacterial nuclease Benzonase can be used to degrade contaminating plasmid DNA from viral supernatant prior to transduction, minimizing plasmid carry-over, and eliminating GC-bias (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Sastry <em>et al.</em> 2004</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Cell culture</h3><p id="p-7">U2OS osteosarcoma cells (obtained directly from American Type Culture Collection, HTB-96) were maintained in McCoy’s 5A medium supplemented with 10% (v/v) FBS (Hyclone), 100 units/ml of penicillin, and 0.1 mg/ml streptomycin. 293T cells were maintained in DMEM supplemented with 10% (v/v) FBS (Hyclone), 100 units/ml penicillin, and 0.1 mg/ml streptomycin. hTERT-immortalized Human Mammary Epithelial Cells (HMECs) from a reduction mammoplasty were purchased directly from Lonza (CC-2551), immortalized with human telomerase, and maintained in MEGM (Lonza).</p></div><div id="sec-3" class="subsection"><h3>Virus production</h3><p id="p-8">In our standard lentivirus production protocol, 293T cells are seeded 24 hr before transfection at 1 × 10<sup>7</sup> cells per 150 mm dish. For transfection, plasmid DNA is diluted into reduced-serum medium (Opti-MEM, Life Technologies) for a final volume of 3 ml; 24 µg each of target lentiviral plasmid and a lentiviral packaging plasmid mixture (1:1:1:1 of SV40 VSVg, Gag/pol, TAT, and Rev) are added to Opti-MEM, then 108 µl of TransIT-293 reagent (Mirus) is added to diluted DNA, mixed and incubated at room temperature for &gt; 20 min. Where indicated, mass of plasmid DNA used for transfection was reduced to 10% or 20% relative to the standard protocol. The volume of TransIT-293 is always equal to 3× the total mass of plasmid DNA in the mixture. Following incubation, resulting lipid complexes are added dropwise to cells. After 14–18 hr, transfection medium is removed and replaced with 15–20 ml of normal culture medium. At 24–36 hr following medium replacement, viral supernatants are collected, aliquoted and stored at –80° until further use. See Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1/FileS1.docx">File S1</a> for further information.</p></div><div id="sec-4" class="subsection"><h3>Benzonase treatment</h3><p id="p-9">Lentiviral supernatants were treated with Benzonase as described (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Sastry <em>et al.</em> 2004</a>), using 100–1000 units/ml of viral supernatant, and 10× Benzonase Buffer (500 mM Tris-HCl, pH 8.0, 10 mM MgCl<sub>2</sub>, and 1 mg/ml bovine serum albumin) at a final concentration of 1×. The mixture was incubated for 30 min at 37° following addition of Benzonase. Lentivirus was then aliquoted and stored at –80° until use. For data in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, a 3′BC library virus was treated with 50 units/ml Benzonase for 15 min at 37°. The Benzonase was inactivated by treatment with 10 mg/ml proteinase K for 1 hr at 55°. Proteinase K was inactivated by incubating samples at 98° for 20 min. See Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1/FileS1.docx">File S1</a> for further information.</p></div><div id="sec-5" class="subsection"><h3>Virus titration</h3><p id="p-10">Lentiviral titers were determined by transducing U2OS cells with serial dilutions of virus by adding diluted viral supernatant directly to culture medium supplemented with 4 μg/ml polybrene. At 16 hr after transduction, culture medium was replaced. At 2 d after transduction, 1.5 µg/ml Puromycin (Clontech) was added to culture medium and replaced every 3 d. Cells were allowed to grow until visible colonies formed. Medium was then removed and cells were stained with 0.5% Methylene Blue in 50% Ethanol for 30–60 sec. Colonies were counted and used to determine titer based on the volume of virus used.</p></div><div id="sec-6" class="subsection"><h3>RT-qPCR</h3><p id="p-11">Quantitative RT-PCR (RT-qPCR) was performed in triplicate using the Platinum Sybr Green Kit (Invitrogen) on an Applied Biosystems Fast 7500 machine. Detection of <em>CCND1</em> ORF abundance was performed using Relative Quantitation (RQ) settings, and was normalized to the total integrated ORF library using primers targeting the PGK promoter as the endogenous control (<em>CCND1</em> Forward: 5′ GCTGTGCATCTACACCGACA 3′; Reverse: 5′ TTGAGCTTGTTCACCAGGAG 3′. PGK Forward: 5′ CAA CCG GCT CCG TTC TTT GG 3′; Reverse: 5′ CAC GAG ACT AGT GAG ACG TGC TA 3′). Detection of combined genomic DNA (gDNA) and plasmid signal using CMV or PGK primers was performed using RQ settings, and was normalized to total gDNA using primers targeting the <em>GAPDH</em> promoter as the endogenous control (CMV Forward: 5′ TGGCATTATGCCCAGTACATGACC 3′; Reverse: 5′ CCATTGATGTACTGCCAAAACCGC 3′. <em>GAPDH</em> Forward: 5′ ATC CAA GCG TGT AAG GGT CC 3′; Reverse: 5′ GGA CTG AGA TTG GCC CGA TG 3′). Detection of shRNA half-hairpins was performed in this same manner, using primers JH353F (5′ TAGTGAAGCCACAGATGTA 3′) and HHR2L (5′ ATGTATCAAAGAGATAGCAAGGTATTCAG 3′), with <em>GAPDH</em> promoter as the endogenous normalization control.</p></div><div id="sec-7" class="subsection"><h3>Screens</h3><p id="p-12">HMECs were transduced with library lentivirus in triplicate with 8 µg/ml polybrene (Sigma), with an average representation of 1000 cells per ORF using a multiplicity of infection (MOI) of 0.5. Medium was changed the following day. Cells were split 2 d following transduction, and a portion equivalent to ≥ 1000× representation was collected from each replicate and stored at –80° to use as a reference. Sufficient cells to maintain ≥ 1000× representation were then cultured for 10 population doublings (PDs) [following the first split, stably transduced cells were selected with 2 µg/ml Puromycin (Clontech) for 3–4 d], and final cell pellets containing ≥ 1000× library coverage were collected.</p><p id="p-13">For screens performed using the inducible tetracycline responsive element (TRE) BC library, transduced cells were fully selected with Puromycin before induction. At the time of induction, a portion of cells equivalent to ≥ 1000× representation was collected from each replicate and stored at –80° to use as a reference. Sufficient cells to maintain ≥ 1000× representation were then cultured in the presence or absence of doxycycline (dox) at 100 ng/ml for 10 PDs, when final cell pellets were collected. The shRNA screen depicted in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> was performed as described (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Schlabach <em>et al.</em> 2008</a>). The mock screen described in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, C–D</a> was performed using U2OS cells, infected at MOI &lt; 0.5 in duplicate with 4 µg/ml polybrene (Sigma). Medium was replaced 24 hr after transduction. Initial normalization samples were harvested 3 d after transduction, and cells were then selected with Puromycin (1.5 µg/ml). Following selection, cells were seeded in the presence or absence of 1 µg/ml dox. Library representation of an average of 200 copies of each shRNA was maintained at every passage. Final screen samples were harvested after 10 PDs in the presence (or absence) of dox.</p></div><div id="sec-8" class="subsection"><h3>Genomic DNA preparation and PCR</h3><p id="p-14">Screen cell pellets were thawed and lysed in 10 mM Tris, pH 8.0, 10 mM EDTA, 0.5% SDS, 0.75 mg/ml Proteinase K at 55° overnight. gDNA was extracted using Phase Lock tubes (5 PRIME) with phenol:chloroform and then chloroform. RNase A was added to a final concentration of 25 μg/ml and, following incubation overnight at 37°, gDNA was again extracted using Phase Lock tubes with phenol:chloroform, followed by two chloroform extractions. DNA was ethanol precipitated, recovered by centrifugation, and washed three times with 70% ethanol. Dried pellets were resuspended in 10 mM Tris-Cl, pH 8.5, and BCs were PCR-amplified with Phusion High-Fidelity DNA Polymerase (NEB Cat # M0530S) in three PCR steps for BC recovery, addition of Illumina adaptors, and sample indexing (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Meyer and Kircher 2010</a>). The first round of PCR was performed using common primers flanking the unique BC region (forward primer ORF.BC1.for: 5′ CCAGTAGGTCCACTATGAGT 3′; reverse primer ORF.BC.1.rev: 5′ CTAGTTCCGCTTACACAGCT 3′). The reaction contained a total mass of DNA equal to gDNA from cells covering 1000× representation, with individual 100 µl reactions containing 10 µg gDNA, 1× Phusion HF Buffer, 200 µM dNTPs, 1 µM each of ORF.BC1.for and ORF.BC1.rev, and 4 units of Phusion Polymerase. Reactions for each replicate were pooled, and 5 µl of pooled PCR1 was used as the template for a 100 µl reaction containing 1× Phusion HF Buffer, 250 µM dNTPs, 2 µM each of primers ISP-ORF.BC1.for (5′ GTGACTGGAGTTCAGACGTGTGCTCT TCCGATCTCCAGTAGGTCCACTATGAGT 3′) and P5-ORF.BC1.rev (5′ AATGATACGGCGACCACCGACTAGTTCCGCTTACACAGCT 3′), and 4 units of Phusion Polymerase. A third PCR reaction was performed to add indices and allow sample multiplexing. The 100 µl PCR3 reaction contained 2 µl of PCR2 product, 1× Phusion HF Buffer, 250 µM dNTPs, 2 µM each of primers P7-Index-ISP (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Meyer and Kircher 2010</a>) and P5-ORF.BC1.rev <span class="sc">(</span>5′ AATGATACGGCGACCACCGACTAGTTCCGCTTACACAGCT 3′), and 4 units of Phusion Polymerase. PCR3 products were gel-purified using QiaQuick Gel Extraction columns (Qiagen). Samples were sequenced on two Illumina HiSequation 2500 lanes with the primer heyME19 (5′ GCGACCACCGACTAGTTCCGCTTACACAGCT 3′).</p><p id="p-15">For the mock shRNA screens, half-hairpin amplicons were recovered by PCR using primers JH353F (5′ TAGTGAAGCCACAGATGTA 3′) and HHR2L (5′ ATGTATCAAAGAGATAGCAAGGTATTCAG 3′). Amplified half-hairpins were gel-purified, and amplified by two additional rounds of PCR to add Ion Torrent adapter sequences, and sample index sequences. Final PCR products were gel purified and prepared for Ion PGM sequencing using the Ion PGM Template OT2 200 Kit (Life Technologies) according to manufacturer’s instructions. Sequencing was performed using the Ion PGM Sequencing 200 Kit v2, and the Ion 318 Chip Kit v2, according to manufacturer’s instructions.</p></div><div id="sec-9" class="subsection"><h3>PCR assay to detect ORF-BC recombination</h3><p id="p-16">For PCR assays to measure the frequency of ORF-BC recombination, 11 clones were picked at random from each ORF-BC library, and sequenced to identify individual ORF-BC pairs. Clone plasmid DNA was transiently transfected into 293Ts individually, or pooled together for transfection, to produce lentivirus. hTERT-HMECs were transduced with individual or pooled lentivirus, and stable integrants were selected with Puromycin (2 µg/ml). gDNA was isolated from each stable cell line. PCR using primers targeting one of the 11 ORFs and a common region of the vector (5′ BC Recombination Test: 5pBC.KRTAP19-7.ORF: 5′ CCACATCCACAGCTATAGC 3′; ORF.BC1.for: 5′ CCAGTAGGTCCACTATGAGT 3′. 3′ BC Recombination Test: 3pBC.MAX.ORF: 5′ CAACCGAGGTTTCAATCTGC 3′; ORF.BC1.rev: 5′ CTAGTTCCGCTTACACAGCT 3′) were used to amplify BCs from either the pooled lentivirus-transduced gDNA, or the combined gDNA from each of the individual lentivirus-transduced cells. PCR products were cloned using the pENTR/D-TOPO Cloning Kit (Life Tech K2400-20), and clones were sequenced to determine BC identity.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-17">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Barcode recombination causes reporter infidelity</h3><p id="p-18">To enable high-throughput gain-of-function screens, we recently designed human ORF libraries in which each ORF is paired with unique DNA BCs (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ho <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Yang <em>et al.</em> 2011</a>; manuscript in preparation). The uniform length of the BCs (30 bp) enables more quantitative PCR recovery, as it alleviates the inherent bias introduced by amplification of templates that vary in length (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Shagin <em>et al.</em> 1999</a>). Furthermore, our barcoding strategy facilitates the pairing of each ORF with several independent BCs providing a measure of internal reproducibility. We designed a flexible, modular lentiviral vector for library expression and tested several iterations in which we varied the position of the BC and the promoter (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). We performed identical proliferation screens in HMECs with each version of this library, identifying BCs that shifted in abundance over 10 PDs.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Barcode agreement in 5&#x2032; BC and TRE BC libraries. (A) Vector schematics of three library vectors constructed in the pHAGE lentiviral backbone. Note that the TRE BC Library was used in cells that had previously been transduced with a lentivirus carrying the reverse tetracycline transactivator (rtTA). LTR, long terminal repeat; BC, barcode; CMV, cytomegalovirus immediate-early promoter; PGK, phosphoglycerate kinase 1 promoter; TRE, tetracycline responsive element promoter. (B, C) Distribution of normalized read counts in initial (0&#xA0;PD) and final (10&#xA0;PD) screen samples (top panel) and Log2FC (bottom panel) for a representative set of BCs mapped to CCND1 (B) or CDKN1A (C) in the 5&#x2032;BC library. (D) RT-qPCR measurement of CCND1 abundance in gDNA from three replicates measured at initial (PD0) and final (PD10) time points of the 5&#x2032;BC library screen. Assays were performed in triplicate (error bars &#xB1;&#xA0;SD). (E, F) Distribution of normalized read counts in initial (10&#xA0;PD &#x2013;Dox) and final (10 PD +Dox) screen samples (top panel) and Log2FC (bottom panel) for a representative set of BCs mapped to CCND1 (E) or CDKN1A (F) in the TRE BC library." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1509422084" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Barcode agreement in 5&#x2032; BC and TRE BC libraries. (A) Vector schematics of three library vectors constructed in the pHAGE lentiviral backbone. Note that the TRE BC Library was used in cells that had previously been transduced with a lentivirus carrying the reverse tetracycline transactivator (rtTA). LTR, long terminal repeat; BC, barcode; CMV, cytomegalovirus immediate-early promoter; PGK, phosphoglycerate kinase 1 promoter; TRE, tetracycline responsive element promoter. (B, C) Distribution of normalized read counts in initial (0&#xA0;PD) and final (10&#xA0;PD) screen samples (top panel) and Log2FC (bottom panel) for a representative set of BCs mapped to &lt;em&gt;CCND1&lt;/em&gt; (B) or &lt;em&gt;CDKN1A&lt;/em&gt; (C) in the 5&#x2032;BC library. (D) RT-qPCR measurement of &lt;em&gt;CCND1&lt;/em&gt; abundance in gDNA from three replicates measured at initial (PD0) and final (PD10) time points of the 5&#x2032;BC library screen. Assays were performed in triplicate (error bars &#xB1;&#xA0;SD). (E, F) Distribution of normalized read counts in initial (10&#xA0;PD &#x2013;Dox) and final (10 PD +Dox) screen samples (top panel) and Log2FC (bottom panel) for a representative set of BCs mapped to &lt;em&gt;CCND1&lt;/em&gt; (E) or &lt;em&gt;CDKN1A&lt;/em&gt; (F) in the TRE BC library.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2781/F1.medium.gif" width="356" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/9/2781/F1.medium.gif" width="356" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469171" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Barcode agreement in 5′ BC and TRE BC libraries. (A) Vector schematics of three library vectors constructed in the pHAGE lentiviral backbone. Note that the TRE BC Library was used in cells that had previously been transduced with a lentivirus carrying the reverse tetracycline transactivator (rtTA). LTR, long terminal repeat; BC, barcode; CMV, cytomegalovirus immediate-early promoter; PGK, phosphoglycerate kinase 1 promoter; TRE, tetracycline responsive element promoter. (B, C) Distribution of normalized read counts in initial (0 PD) and final (10 PD) screen samples (top panel) and Log2FC (bottom panel) for a representative set of BCs mapped to <em>CCND1</em> (B) or <em>CDKN1A</em> (C) in the 5′BC library. (D) RT-qPCR measurement of <em>CCND1</em> abundance in gDNA from three replicates measured at initial (PD0) and final (PD10) time points of the 5′BC library screen. Assays were performed in triplicate (error bars ± SD). (E, F) Distribution of normalized read counts in initial (10 PD –Dox) and final (10 PD +Dox) screen samples (top panel) and Log2FC (bottom panel) for a representative set of BCs mapped to <em>CCND1</em> (E) or <em>CDKN1A</em> (F) in the TRE BC library.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Surprisingly, we frequently observed discordance in the log<sub>2</sub> of the fold-change ratios (Log<sub>2</sub>FC) among the sets of BCs reporting for the same ORF in the 5′BC library. While neutral genes might be subject to random drift, causing some BCs to become slightly enriched, and others slightly depleted, we observed marked divergence of BC abundance changes among BCs paired with ORFs known to potently regulate proliferation, such as <em>CCND1</em> and <em>CDKN1A</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B and C</a>). Furthermore, when we measured the abundance of the <em>CCND1</em> ORF in the 5′BC screen gDNA by qPCR, we saw that the ORF was becoming enriched over time in at least two of the three replicates (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>), though this was not reflected by the average behavior of the paired BCs (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). We sought to investigate and correct the source of this variation, as the fidelity of our BCs as specific and precise reporters for ORF behavior is critical to the success of our library design.</p><p id="p-21">One possible explanation for this observation was retrovirus-mediated recombination between BC-ORF pairs causing shuffling of these library elements, because retroviruses carry two copies of their genome and reverse transcribe them with a recombination-prone RT enzyme (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Paillart <em>et al.</em> 2004</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Smyth <em>et al.</em> 2012</a>). To test whether retroviral-mediated recombination occurred in our libraries, and at what frequency, we assembled small sublibraries containing 11 ORF clones from the 5′BC or the 3′BC libraries, each associated with a single, unique, BC. Plasmid DNA containing a mixture of all 11 clones was used to package lentivirus for HMEC transduction. We isolated gDNA from stably transduced cells, and then designed PCR primers to amplify the BCs attached to one of the 11 ORFs using an ORF-specific primer, and a common primer that anneals to the vector. The final PCR product contains the associated BC. We then sequenced these PCR products to identify the associated BCs.</p><p id="p-22">One concern in using PCR to amplify proviral integrants for measurement of recombination frequency is the possibility that PCR artifacts could distort this measurement. Specifically, the PCR reaction itself can generate chimeric products, which, for our purposes, could lead to perceived recombination between ORFs and BCs (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brakenhoff <em>et al.</em> 1991</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Suzuki and Giovannoni 1996</a>). To control for this possibility, we also prepared lentiviruses from each of the 11 BC-ORF clones from the two sublibraries individually. We transduced cells with these individual lentiviruses, selected stable integrants, and isolated gDNA. We then pooled gDNA from each cell line infected with a single individual lentivirus together into a single PCR reaction. Thus, both our PCR control and recombination test conditions consisted of a population of heterogeneously infected cells, each carrying a single integrant of one of 11 possible ORF clones. However, only in the recombination test condition could retrovirus-mediated recombination contribute to any observed shuffling of BCs between ORFs.</p><p id="p-23">We observed recombination events in &gt; 25% of the sequences tested from the 5′BC library (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Conversely, in the 3′BC library, recombination occurred at a measured frequency of only 5.5%. This substantial difference in recombination frequency (<em>P</em> = 0.00015, binomial test) correlates with the length of homologous sequence residing between the ORF and the BC (720 bp in the 5′BC library, and 96 bp in the 3′BC library), consistent with previous reports (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Delviks and Pathak 1999</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hu <em>et al.</em> 2003</a>). Importantly, no recombination was observed in our PCR control conditions, indicating that the retroviral reverse transcription step is the likely source of this BC-ORF uncoupling. It should be noted that our measurements will underestimate the true recombination frequency, because we are unable to detect recombination events that occur between identical viruses within our pool. As this is expected to occur in 9% of cases for a sublibrary of 11 ORFs, we estimate the true recombination frequency to be greater than 28% in the 5′BC library, and ∼ 6% in the 3′BC library (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Screens performed using the TRE BC library, in which the ORF-BC pairs are identical to those in the 3′BC library and only the promoter is varied, exhibit strong concordance among BCs paired with ORFs known to promote strong proliferation phenotypes, such as <em>CCND1</em> and <em>CDKN1A</em> (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, E and F</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469134/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469134/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Frequency of retroviral-mediated recombination in BC-ORF libraries</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Plasmid contamination of viral supernatants distorts distributions of BC abundance</h3><p id="p-26">The 3′ and 5′BC libraries were constructed of five or 13 distinct pools of ORF clones, respectively. Each pool was packaged into lentivirus and titered individually, and these were then combined for transduction and pooled screening of the complete collection. In the context of constitutive expression, we sought to maximize the dynamic range of these screens by collecting initial reference samples of cells 2 d after transduction with library virus. Additional samples of cells were collected at each passage, and a final sample was taken after 10 PDs. In both of these screens, deep sequencing of initial reference samples revealed a nonrandom distribution of read density across the library BCs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1/TableS1.xlsx">Table S1</a>). Subpopulations of BCs were present at substantially (2- to 10-fold) different abundances. Furthermore, these differences did not persist at 10 PDs, suggesting a transient effect (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). The subpopulations corresponded to our library lentivirus pools, and the signal abundance of each pool inversely correlated with its viral titer (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). This suggested that the source of overrepresentation of individual pools may have been the lentivirus itself, since a larger volume of the lower titer viral supernatants was added to cells to normalize the representation of different pools.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plasmid DNA contaminates lentiviral supernatants and is detected in screen gDNA. (A) Distribution of Illumina reads per BC from the 3&#x2032;BC Library screen at 0&#xA0;PD (top panel) and 10&#xA0;PD (bottom panel). (B) Barplot distribution of reads per BC at 0&#xA0;PDs vs. viral titer in colony forming units per milliliter (CFU/ml) for each of five subpools of lentivirus that were coinfected at equal viral representation for full 3&#x2032;BC library coverage. (C) Agarose gel showing PCR product from indicated screen gDNA sample using primers specific for the bacterial origin of replication (ORI). (D) RT-qPCR measurements of relative CMV abundance at indicated screen time-points in 5&#x2032;BC library cells as a measure of the combined signal of library virus-infected cells and contaminating plasmid within gDNA. Assays were performed in triplicate (error bars &#xB1;&#xA0;SD)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1509422084" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plasmid DNA contaminates lentiviral supernatants and is detected in screen gDNA. (A) Distribution of Illumina reads per BC from the 3&#x2032;BC Library screen at 0&#xA0;PD (top panel) and 10&#xA0;PD (bottom panel). (B) Barplot distribution of reads per BC at 0&#xA0;PDs &lt;em&gt;vs.&lt;/em&gt; viral titer in colony forming units per milliliter (CFU/ml) for each of five subpools of lentivirus that were coinfected at equal viral representation for full 3&#x2032;BC library coverage. (C) Agarose gel showing PCR product from indicated screen gDNA sample using primers specific for the bacterial origin of replication (ORI). (D) RT-qPCR measurements of relative CMV abundance at indicated screen time-points in 5&#x2032;BC library cells as a measure of the combined signal of library virus-infected cells and contaminating plasmid within gDNA. Assays were performed in triplicate (error bars &#xB1;&#xA0;SD).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2781/F2.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/9/2781/F2.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469222" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Plasmid DNA contaminates lentiviral supernatants and is detected in screen gDNA. (A) Distribution of Illumina reads per BC from the 3′BC Library screen at 0 PD (top panel) and 10 PD (bottom panel). (B) Barplot distribution of reads per BC at 0 PDs <em>vs.</em> viral titer in colony forming units per milliliter (CFU/ml) for each of five subpools of lentivirus that were coinfected at equal viral representation for full 3′BC library coverage. (C) Agarose gel showing PCR product from indicated screen gDNA sample using primers specific for the bacterial origin of replication (ORI). (D) RT-qPCR measurements of relative CMV abundance at indicated screen time-points in 5′BC library cells as a measure of the combined signal of library virus-infected cells and contaminating plasmid within gDNA. Assays were performed in triplicate (error bars ± SD).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We hypothesized that, following transfection for virus production, residual plasmid DNA may remain in the viral supernatant, and be carried over to the target cells at the time of transduction. To test this hypothesis, we asked whether we could detect evidence of plasmid DNA in the gDNA isolated from our screen cell samples. We designed primers annealing to the bacterial origin of replication (ORI), which is absent in the integrated viral DNA, and observed a PCR product of the correct size (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). In samples from both the 3′ and 5′BC libraries, band strength in each sample roughly corresponded to the observed pattern of distortion in Illumina reads for that time-point. We reasoned that, using primers targeting a portion of the lentiviral genome that integrates into cells, we could detect the combined signal from both plasmid and proviral integrations in each screen sample. Using RT-qPCR primers to amplify the CMV promoter region of our vector, we confirmed the presence of a large overabundance of signal at the initial time-point of the 5′BC screen, most of which disappeared after 2 PDs, leaving a smaller population of plasmid which appears to dilute out completely by 6 PDs (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>). Presumably, the remaining signal is generated only by proviral integrants, as the contaminating plasmid is no longer present. We estimate the amount of plasmid DNA in the initial 5′ BC screen samples to be ∼ 5.8 pg per 50 ng of total extracted DNA, corresponding to &gt; 70 times more copies of the viral genome than are present integrated into the genomic DNA.</p></div><div id="sec-14" class="subsection"><h3>Elimination of plasmid DNA from viral supernatant</h3><p id="p-29">We observed that decreasing the amount of plasmid DNA used in our transfection protocol reduced levels of contaminating plasmid, without substantially diminishing viral titers (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). However, even a 10-fold reduction in input DNA failed to generate transduced cell populations completely free of contamination. Previous work has demonstrated that the bacterial nuclease Benzonase is effective for eliminating plasmid contamination from clinical preparations of lentivirus (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Sastry <em>et al.</em> 2004</a>). This endonuclease can degrade both DNA and RNA, so contaminating plasmid DNA is accessible for degradation, while enveloped viruses are protected. Indeed, Benzonase treatment is commonly included in the workflow for large-scale lentivirus vector production for gene therapy applications to avoid the immunogenic consequences of naked DNA (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Segura <em>et al.</em> 2013</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Elimination of plasmid DNA from lentivirus by treatment with Benzonase. (A) Agarose gel showing PCR products from bacterial-derived portion of plasmid DNA in library viral supernatants. Lane labels denote the volume of indicated template used in each PCR reaction in microliters. Benzonase Virus, viral supernatant treated with Benzonase; Proteinase K Virus, viral supernatant treated with Proteinase K; Benz&#xA0;+&#xA0;PK Virus, viral supernatant treated with Benzonase, followed by treatment with Proteinase K. (B) RT-qPCR measurements of relative CMV abundance as a measure of the combined signal of library-virus-infected cells, and contaminating plasmid within gDNA. Cells were infected at MOI =&#xA0;0.5 with virus prepared according to our standard lentiviral production protocol (Std DNA), or by reducing the amount of transfected plasmid DNA to 10% of that in our standard protocol (10% DNA). These viruses were then either treated with Benzonase (Benz) or left untreated (Unt), and were used to transduce HMECs. Genomic DNA was isolated 3&#xA0;d after transduction, and CMV abundance was compared to gDNA from two of three replicates (A and B) of the 3&#x2032;BC screen at 3&#xA0;PD and 13&#xA0;PD. The top panel shows full data range, and the bottom panel shows an expanded view of lower abundance samples. Assays were performed in triplicate (error bars &#xB1;&#xA0;SD). (C) Distribution of Illumina reads per BC from the inducible TRE BC library at 0&#xA0;PD (top panel) and 10&#xA0;PD (bottom panel). Library virus was treated with 500&#xA0;units/ml Benzonase before transduction. (D) RT-qPCR measurements of relative PGK abundance as a measure of the combined signal of library-virus-infected cells and contaminating plasmid within gDNA. The TRE BC library lentivirus supernatant was treated with Benzonase and transduced into HMECs which were passaged for 6&#xA0;PDs in the absence of doxycycline before collecting initial PD0 time point. PGK abundance from the inducible TRE library samples was compared to samples from the 5&#x2032;BC Library screen. Assays were performed in triplicate (error bars &#xB1;&#xA0;SD)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1509422084" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Elimination of plasmid DNA from lentivirus by treatment with Benzonase. (A) Agarose gel showing PCR products from bacterial-derived portion of plasmid DNA in library viral supernatants. Lane labels denote the volume of indicated template used in each PCR reaction in microliters. Benzonase Virus, viral supernatant treated with Benzonase; Proteinase K Virus, viral supernatant treated with Proteinase K; Benz&#xA0;+&#xA0;PK Virus, viral supernatant treated with Benzonase, followed by treatment with Proteinase K. (B) RT-qPCR measurements of relative CMV abundance as a measure of the combined signal of library-virus-infected cells, and contaminating plasmid within gDNA. Cells were infected at MOI =&#xA0;0.5 with virus prepared according to our standard lentiviral production protocol (Std DNA), or by reducing the amount of transfected plasmid DNA to 10% of that in our standard protocol (10% DNA). These viruses were then either treated with Benzonase (Benz) or left untreated (Unt), and were used to transduce HMECs. Genomic DNA was isolated 3&#xA0;d after transduction, and CMV abundance was compared to gDNA from two of three replicates (A and B) of the 3&#x2032;BC screen at 3&#xA0;PD and 13&#xA0;PD. The top panel shows full data range, and the bottom panel shows an expanded view of lower abundance samples. Assays were performed in triplicate (error bars &#xB1;&#xA0;SD). (C) Distribution of Illumina reads per BC from the inducible TRE BC library at 0&#xA0;PD (top panel) and 10&#xA0;PD (bottom panel). Library virus was treated with 500&#xA0;units/ml Benzonase before transduction. (D) RT-qPCR measurements of relative PGK abundance as a measure of the combined signal of library-virus-infected cells and contaminating plasmid within gDNA. The TRE BC library lentivirus supernatant was treated with Benzonase and transduced into HMECs which were passaged for 6&#xA0;PDs in the absence of doxycycline before collecting initial PD0 time point. PGK abundance from the inducible TRE library samples was compared to samples from the 5&#x2032;BC Library screen. Assays were performed in triplicate (error bars &#xB1;&#xA0;SD).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2781/F3.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/9/2781/F3.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469112" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Elimination of plasmid DNA from lentivirus by treatment with Benzonase. (A) Agarose gel showing PCR products from bacterial-derived portion of plasmid DNA in library viral supernatants. Lane labels denote the volume of indicated template used in each PCR reaction in microliters. Benzonase Virus, viral supernatant treated with Benzonase; Proteinase K Virus, viral supernatant treated with Proteinase K; Benz + PK Virus, viral supernatant treated with Benzonase, followed by treatment with Proteinase K. (B) RT-qPCR measurements of relative CMV abundance as a measure of the combined signal of library-virus-infected cells, and contaminating plasmid within gDNA. Cells were infected at MOI = 0.5 with virus prepared according to our standard lentiviral production protocol (Std DNA), or by reducing the amount of transfected plasmid DNA to 10% of that in our standard protocol (10% DNA). These viruses were then either treated with Benzonase (Benz) or left untreated (Unt), and were used to transduce HMECs. Genomic DNA was isolated 3 d after transduction, and CMV abundance was compared to gDNA from two of three replicates (A and B) of the 3′BC screen at 3 PD and 13 PD. The top panel shows full data range, and the bottom panel shows an expanded view of lower abundance samples. Assays were performed in triplicate (error bars ± SD). (C) Distribution of Illumina reads per BC from the inducible TRE BC library at 0 PD (top panel) and 10 PD (bottom panel). Library virus was treated with 500 units/ml Benzonase before transduction. (D) RT-qPCR measurements of relative PGK abundance as a measure of the combined signal of library-virus-infected cells and contaminating plasmid within gDNA. The TRE BC library lentivirus supernatant was treated with Benzonase and transduced into HMECs which were passaged for 6 PDs in the absence of doxycycline before collecting initial PD0 time point. PGK abundance from the inducible TRE library samples was compared to samples from the 5′BC Library screen. Assays were performed in triplicate (error bars ± SD).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">We thus tested whether Benzonase treatment could eliminate detectable levels of plasmid DNA from viral supernatants. As Benzonase is highly stable, and retains activity even after storage at –80°, any PCR-based method for detection of residual plasmid in Benzonase-treated virus is masked by degradation of PCR products (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). To visualize PCR products and assess plasmid levels in our virus, we needed to inactivate Benzonase without affecting the subsequent PCR reaction. We found that the nonspecific serine protease Proteinase K could completely digest Benzonase, so that no nuclease activity remains (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>). Conveniently, Proteinase K can be heat-inactivated, allowing the PCR reaction to proceed uninhibited. While Benzonase-treated virus without subsequent inactivation masks the signal of spiked-in untreated virus (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>, lane 5), use of Proteinase K to inactivate Benzonase following virus treatment yields apparent elimination of plasmid (lanes 9–10) but no longer suppresses the signal of spiked-in untreated virus (lane 11).</p><p id="p-32">In the context of genetic screens, our greatest concern was to ensure that plasmid contamination was no longer present at the time of collecting samples of transduced cells for screen reference samples. To test this, we packaged lentivirus using our standard transfection protocol or using one-tenth of the amount of plasmid in our standard protocol. These lentiviruses were then treated with Benzonase or left untreated. We infected HMECs at MOI &lt; 0.5, as we had done for our screens. Cells were harvested 3 d after transduction, gDNA isolated, and residual plasmid levels were detected by qPCR of the CMV promoter. In comparing our test samples to gDNA from 3′BC library screen cells that had undergone 13 PDs, and no longer exhibited any distortion of sequence reads due to plasmid contamination, we identified a baseline abundance of CMV that corresponds to a single proviral integration per diploid genome (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>). As expected, gDNA from 3′BC library screen cells that had undergone only 3 PDs after transduction exhibited about twice as much CMV signal, half of which must derive from plasmid DNA. While our untreated viruses exhibit contamination levels that correspond to the relative amount of transfected DNA, Benzonase treatment substantially reduces detection of residual plasmid. Notably, only when input DNA was reduced to 10% of standard concentrations were we able to see CMV abundance reflective of the MOI of 0.5, suggesting complete elimination of contaminating plasmid (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>) (the screen samples shown for comparison are all derived from cells that had been selected for integrants, thus would correspond to an MOI ≥ 1).</p><p id="p-33">For screens using the TRE BC libraries, library virus was generated by transfecting a reduced amount of input plasmid DNA (20%) relative to our standard protocols and viral supernatants were treated with Benzonase. Library-infected cells were then passaged for several PDs in the absence of doxycycline (dox), before initial reference samples were collected, and dox was added to induce ORF expression. We compared contamination levels of our TRE screen gDNA samples to samples from the 5′BC library, using primers amplifying the PGK promoter that drives expression of the Puromycin resistance gene in both vectors (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>). We observe nearly complete elimination of plasmid at PD0 (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3D</a>). As expected, in the absence of plasmid contamination, the distributions of Illumina reads at 0 PD and 10 PD are nearly identical (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1/TableS2.xlsx">Table S2</a>), with the persistent visible variations in abundance between viral subpools likely corresponding to real differences in MOI resulting from imperfect estimates of viral titer.</p></div><div id="sec-15" class="subsection"><h3>GC bias upon PCR amplification</h3><p id="p-34">Several aspects of multi-template PCR reactions can contribute to variation in amplification efficiencies based on the GC richness of each template. While we generally observe less efficient recovery of high-GC elements, this effect in itself should not impact the computed fold-change values, as differences in efficiency should be uniform across screen samples. However, contaminating plasmid DNA present in screen genomic DNA may lead to different degrees of GC bias in different screen samples due to a different effective number of cycles of PCR in samples where the template is over-represented. Contaminated initial reference samples will experience fewer effective PCR cycles before the PCR reaction is depleted of primer or nucleotides than uncontaminated samples from later time-points. Thus, the overabundance of target DNA in normalizing initial reference samples would cause higher GC-containing elements to erroneously appear to be depleting from the screens, although this effect would be due to artifact rather than phenotype.</p><p id="p-35">We examined the relationship between GC content of BCs and their Log<sub>2</sub>FC in our 5′BC and 3′BC library screens, and indeed observed a correlation (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1/TableS3.xlsx">Table S3</a>). We then asked whether this effect extended to other types of library elements, and examined a set of nontargeting shRNAs carried through a viability screen for 10 PDs as internal negative controls. Again, a correlation was observed between the GC content of each hairpin and its dropout during the screen leading to a two- to three-fold depletion of GC rich sequences (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1/TableS4.xlsx">Table S4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in template abundance causes artificial depletion of high GC content library elements during screens. (A, B) Boxplot representation of the distribution of BC Log2FC (A) or shRNA Log2FC (B) binned by GC content. Median, first quartile, third quartile, and outliers (points) are shown. Data in (A) are from 5&#x2032;BC library screen (Table S3). Data in (B) are from a viability screen performed using an shRNA library (Table S4). Only nontargeting negative control shRNAs are included. Pearson&#x2019;s product-moment correlation coefficient (r) and P-value for correlation are indicated below each plot. (C) Relative shRNA abundance of reference samples used for mock screen normalization. shRNA abundance was measured by RT-qPCR and normalized to total gDNA. Assays were performed in triplicate (error bars &#xB1;&#xA0;SD). Std LV, lentivirus prepared according to standard protocol; Std LV&#xA0;+&#xA0;B, lentivirus prepared according to standard protocol and treated with Benzonase; 10% LV&#xA0;+&#xA0;B, lentivirus prepared using 10% of standard input plasmid DNA and treated with Benzonase; Std PD10&#xA0;&#x2013;&#xA0;Dox, cells infected with Std LV that have been passaged for 10&#xA0;PDs in the absence of doxycycline; Lin. Plas.&#xA0;+&#xA0;gDNA, library plasmid DNA was linearized and mixed into genomic DNA from uninfected U2OS cells at a molar ratio equivalent to one virus insertion per diploid genome. (D) Boxplot representation of the distribution of mock screen shRNA Log2FC binned by GC content. Median, first quartile, third quartile and outliers (points) are shown. Plot colors correspond to screen and normalization sample combinations indicated. Pearson&#x2019;s product-moment correlation coefficient (r) and P-value for correlation are indicated below each plot. Std LV, lentivirus prepared according to standard protocol; Std LV&#xA0;+&#xA0;B, lentivirus prepared according to standard protocol and treated with Benzonase; 10% LV&#xA0;+&#xA0;B, lentivirus prepared using 10% of standard input plasmid DNA and treated with Benzonase; Std PD10&#xA0;&#x2013;&#xA0;Dox, cells infected with Std LV that have been passaged for 10&#xA0;PDs in the absence of doxycycline; Plasmid, library plasmid DNA used at an equivalent mass to gDNA samples; Lin. Plas.&#xA0;+&#xA0;gDNA, library plasmid DNA was linearized and mixed into genomic DNA from uninfected U2OS cells at a molar ratio equivalent to one virus insertion per diploid genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1509422084" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variation in template abundance causes artificial depletion of high GC content library elements during screens. (A, B) Boxplot representation of the distribution of BC Log2FC (A) or shRNA Log2FC (B) binned by GC content. Median, first quartile, third quartile, and outliers (points) are shown. Data in (A) are from 5&#x2032;BC library screen (Table S3). Data in (B) are from a viability screen performed using an shRNA library (Table S4). Only nontargeting negative control shRNAs are included. Pearson&#x2019;s product-moment correlation coefficient (&lt;em&gt;r&lt;/em&gt;) and &lt;em&gt;P&lt;/em&gt;-value for correlation are indicated below each plot. (C) Relative shRNA abundance of reference samples used for mock screen normalization. shRNA abundance was measured by RT-qPCR and normalized to total gDNA. Assays were performed in triplicate (error bars &#xB1;&#xA0;SD). Std LV, lentivirus prepared according to standard protocol; Std LV&#xA0;+&#xA0;B, lentivirus prepared according to standard protocol and treated with Benzonase; 10% LV&#xA0;+&#xA0;B, lentivirus prepared using 10% of standard input plasmid DNA and treated with Benzonase; Std PD10&#xA0;&#x2013;&#xA0;Dox, cells infected with Std LV that have been passaged for 10&#xA0;PDs in the absence of doxycycline; Lin. Plas.&#xA0;+&#xA0;gDNA, library plasmid DNA was linearized and mixed into genomic DNA from uninfected U2OS cells at a molar ratio equivalent to one virus insertion per diploid genome. (D) Boxplot representation of the distribution of mock screen shRNA Log2FC binned by GC content. Median, first quartile, third quartile and outliers (points) are shown. Plot colors correspond to screen and normalization sample combinations indicated. Pearson&#x2019;s product-moment correlation coefficient (&lt;em&gt;r&lt;/em&gt;) and &lt;em&gt;P&lt;/em&gt;-value for correlation are indicated below each plot. Std LV, lentivirus prepared according to standard protocol; Std LV&#xA0;+&#xA0;B, lentivirus prepared according to standard protocol and treated with Benzonase; 10% LV&#xA0;+&#xA0;B, lentivirus prepared using 10% of standard input plasmid DNA and treated with Benzonase; Std PD10&#xA0;&#x2013;&#xA0;Dox, cells infected with Std LV that have been passaged for 10&#xA0;PDs in the absence of doxycycline; Plasmid, library plasmid DNA used at an equivalent mass to gDNA samples; Lin. Plas.&#xA0;+&#xA0;gDNA, library plasmid DNA was linearized and mixed into genomic DNA from uninfected U2OS cells at a molar ratio equivalent to one virus insertion per diploid genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2781/F4.medium.gif" width="347" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/9/2781/F4.medium.gif" width="347" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2781/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469173" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Variation in template abundance causes artificial depletion of high GC content library elements during screens. (A, B) Boxplot representation of the distribution of BC Log2FC (A) or shRNA Log2FC (B) binned by GC content. Median, first quartile, third quartile, and outliers (points) are shown. Data in (A) are from 5′BC library screen (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1/TableS3.xlsx">Table S3</a>). Data in (B) are from a viability screen performed using an shRNA library (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1/TableS4.xlsx">Table S4</a>). Only nontargeting negative control shRNAs are included. Pearson’s product-moment correlation coefficient (<em>r</em>) and <em>P</em>-value for correlation are indicated below each plot. (C) Relative shRNA abundance of reference samples used for mock screen normalization. shRNA abundance was measured by RT-qPCR and normalized to total gDNA. Assays were performed in triplicate (error bars ± SD). Std LV, lentivirus prepared according to standard protocol; Std LV + B, lentivirus prepared according to standard protocol and treated with Benzonase; 10% LV + B, lentivirus prepared using 10% of standard input plasmid DNA and treated with Benzonase; Std PD10 – Dox, cells infected with Std LV that have been passaged for 10 PDs in the absence of doxycycline; Lin. Plas. + gDNA, library plasmid DNA was linearized and mixed into genomic DNA from uninfected U2OS cells at a molar ratio equivalent to one virus insertion per diploid genome. (D) Boxplot representation of the distribution of mock screen shRNA Log2FC binned by GC content. Median, first quartile, third quartile and outliers (points) are shown. Plot colors correspond to screen and normalization sample combinations indicated. Pearson’s product-moment correlation coefficient (<em>r</em>) and <em>P</em>-value for correlation are indicated below each plot. Std LV, lentivirus prepared according to standard protocol; Std LV + B, lentivirus prepared according to standard protocol and treated with Benzonase; 10% LV + B, lentivirus prepared using 10% of standard input plasmid DNA and treated with Benzonase; Std PD10 – Dox, cells infected with Std LV that have been passaged for 10 PDs in the absence of doxycycline; Plasmid, library plasmid DNA used at an equivalent mass to gDNA samples; Lin. Plas. + gDNA, library plasmid DNA was linearized and mixed into genomic DNA from uninfected U2OS cells at a molar ratio equivalent to one virus insertion per diploid genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">To test whether elimination of plasmid contamination could abolish the observed GC bias, we performed a mock screen using a library of 1000 nontargeting shRNAs. The library contains five pools, each with 200 shRNAs of uniform GC content: 40%, 45%, 50%, 55%, or 60% GC bases in the hairpin sequence. The shRNAs were expressed from a Tetracycline-inducible lentiviral vector. We pooled these plasmid libraries together, and packaged them into viruses, either according to our standard protocol or following modified protocols for plasmid elimination, utilizing reduced plasmid DNA input and/or treating lentiviral supernatants with Benzonase. U2OS cells were infected with these viruses at an MOI &lt; 0.5, and cells were collected 3 d posttransduction, and again after 10 PDs in the presence or absence of dox. We isolated gDNA, and recovered half-hairpin amplicons by PCR (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Schlabach <em>et al.</em> 2008</a>). The abundance of each shRNA in each of our screen samples and reference samples was quantified by deep sequencing of the half-hairpin amplicons.</p><p id="p-38">To further assess the impact of plasmid DNA on relative PCR efficiency of GC-rich templates, we designed several additional reference samples. Rather than computing the fold-change of a hairpin relative to its abundance in infected cells, we assessed the direct quantification of plasmid DNA for this purpose. Some laboratories use library plasmid DNA, rather than infected cells to normalize screen data (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Sims <em>et al.</em> 2011</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cheung <em>et al.</em> 2011</a>). We found that the amount and structure of the plasmid DNA used as reference is important. It is critical to match the total number of template molecules in the gDNA and plasmid samples. In order to try to more precisely recreate the conditions of PCR in our screen samples, we also linearized library plasmid DNA, and mixed it with untransduced gDNA from U2OS cells, so that the number of template copies was equivalent to library-infected cells, and both contain equivalent amounts of nontemplate, burden DNA. This reference condition also mimics the structure of the template DNA molecules in screen sample PCRs, which are the relaxed rather than the supercoiled form of plasmid DNA.</p><p id="p-39">We measured the abundance of the shRNA half-hairpin template in each of our reference conditions by qPCR (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a>). Though the baseline level of contamination was modest, we observed a statistically significant correlation between GC content and Log<sub>2</sub>FC when cells infected with lentivirus produced under standard conditions were used as the reference sample for normalization (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4D</a>, top left panel, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1/TableS5.xlsx">Table S5</a>). In contrast, reducing the amount of DNA used for transfection, or treating lentiviral supernatants with Benzonase, eliminated this correlation. Additionally, normalizing screen samples to cells which had been carried for 10 PD in the absence of dox induction also successfully eliminated observed GC bias. Though infected with contaminated lentivirus, these cells underwent sufficient doublings for dilution of the plasmid to occur, allowing the population to reach equilibrium. While we observed a significant negative correlation between GC richness and Log<sub>2</sub>FC when plasmid DNA was used as the reference (without matching the number of template molecules to the gDNA screen sample) (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4D</a>, bottom center panel), we found that normalizing our screen samples to a reference sample consisting of diluted linearized plasmid DNA was effective in eliminating GC bias. Importantly, shRNA half-hairpin template was over 200-fold more abundant in the plasmid sample relative to the linearized-plasmid in gDNA condition. This suggests that the total number of copies of the template present in the PCR reaction can dramatically influence the perceived relative abundance of GC-rich targets.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-40">In this study, we report several sources of error intrinsic to common genetic screening techniques in mammalian tissue culture systems. For each, we suggest methods to eliminate or minimize them, which should result in improved fidelity of screen data. Most genetic screens in human cells generate sets of candidate genes which putatively play a role in the phenotype of interest, but these are plagued by a high proportion of false positives (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Stone <em>et al.</em> 2007</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Mohr <em>et al.</em> 2010</a>). This ceiling of detection of true hits is often attributed to quality of the genetic element; for instance, variation in reagent knock-down efficiency and off-target effects are thought to account for many of the false-positives in candidate gene lists derived from RNAi screens (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Echeverri <em>et al.</em> 2006</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Hu and Luo 2012</a>). We find that additional noise is introduced due to unrepresentative abundance measurements and GC bias attributable to variable library template concentration between reference and experimental samples during PCR amplification.</p><p id="p-41">We find a common cause of variable library template representation to be the presence of contaminating plasmid DNA in cell populations shortly after transduction. Plasmid DNA remains in the viral supernatant following the transient transfection step required to produce lentivirus, and is transferred to infected cells along with the viruses. To minimize the effects of this contamination, we treated viral supernatants with the endonuclease Benzonase to degrade residual plasmid DNA. This method is well established in preparation of clinical-grade lentivirus for gene therapy applications (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Sastry <em>et al.</em> 2004</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Yang <em>et al.</em> 2012</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Segura <em>et al.</em> 2013</a>); however, to our knowledge, it has not been previously reported to be used in preparation of lentivirus for genetic screens or other tissue culture applications. We demonstrate that Benzonase treatment can significantly increase the fidelity of specific detection of integrated viral genomes from the genomic DNA of a population of infected cells. Furthermore, Benzonase treatment, or use of an inducible expression system passaged to equilibrium minimize the GC bias caused by plasmid contamination. This fidelity is critical to measurements of changes in abundance within the cell population through the course of a genetic screen for identification of screen hits.</p><p id="p-42">Many genetic screens are normalized to the distribution of genetic elements present in the plasmid DNA library, rather than using a prescreen sample of the transduced cell population as a reference. In fact, a recent analysis suggested that plasmid DNA was the optimal reference point for shRNA screens, compared with viral cDNA or early time points immediately following viral integration (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Sims <em>et al.</em> 2011</a>). However, for the use of ORFs, this option would fail to report information about significant differences in lentiviral packaging efficiency correlated with transgene size, and thus would lead to the potential for overestimating the abundance of long ORFs, and underestimating the abundance of short ORFs (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kumar <em>et al.</em> 2001</a>). Pooled shRNA or CRISPR-based libraries are uniform in size, and thus may be more amenable to use of plasmid as the reference for normalization. However, we find that care must be taken when using plasmid DNA as described below.</p><p id="p-43">An additional effect of plasmid contamination is a persistent correlation between GC-richness of individual library PCR templates, and the dropout of those sequences under screening conditions. Two factors contribute to this effect. The first is the bias against amplifying GC rich templates in each PCR cycle (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4, A and B</a>). In principle, this should cancel out when two samples (<em>i.e.</em>, start and finish) are experiencing the same number of PCR cycles. The second is the possible difference in the total number of template molecules in the initial and end-point samples, which can be influenced by plasmid contamination during the initial transduction process. With a large increase in template molecules in the initial reference sample, those samples undergo fewer actual cycles of PCR before reaching saturation relative to the end-point experimental samples lacking the contamination, resulting in an exaggeration of the PCR bias between samples. While excess contaminating plasmid causes a bias, elimination of this contamination abolishes the relationship between GC content and Log<sub>2</sub>FC (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4D</a>). The abundance of contaminating plasmid in our mock screen was lower than we had seen in previous experiments, and, correspondingly, we observed only a modest, though statistically significant, correlation between GC content and Log<sub>2</sub>FC. Nevertheless, we were able to abolish that correlation by normalizing our screen sample to reference conditions that did not exhibit contamination, and thus contained the same number of copies of template as the screen sample. Accordingly, using excess copies of plasmid DNA to normalize screen samples resulted in pronounced bias, while no bias was observed when using linearized plasmid diluted into gDNA such that template copy number was equivalent to screen samples. These observations confirm that variations in template abundance between experimental and normalization samples cause artifactual GC bias, and indicate that, unless plasmid input is diluted such that the number copies of the template present is equivalent to the number in the transduced cell population, bias against GC-rich templates will be observed. To our knowledge, considerations of the impact on PCR biases of varying template abundance in different time points or samples from screens have not previously been explored.</p><p id="p-44">We also discovered that recombination can occur between unique library elements in retroviral libraries, resulting in uncoupling and reshuffling of BC-ORF pairs. The rate of recombination depends on the length of the homologous region between unique elements. We observed a recombination rate of nearly 30% in our 5′ BC library in which the ORF and BC are separated by 720 bp, while the rate was reduced to ∼ 6% in our 3′BC library where a 96-bp homology region separates the ORF and BC. These observations are consistent with a large body of work describing recombination in wild-type HIV strains during the RT stage of the viral life cycle (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hu <em>et al.</em> 2003</a>). Recombination in retroviral vectors resulting in deletion of one or more elements from a single virus construct has been demonstrated previously (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">ter Brake <em>et al.</em> 2008</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Mcintyre <em>et al.</em> 2009</a>). However, while retroviral recombination has been studied extensively in the field of virology, because genetic screening using multiple genetic elements in a library format is a relatively recent development, intermolecular retroviral recombination resulting in shuffling of elements in a polyclonal population of viral vectors has not yet been reported.</p><p id="p-45">While a 6% recombination frequency is not negligible, this small proportion of the total number of BCs is unlikely to be problematic, as 94% of the signal still accurately reflects the behavior of the proper ORF. In theory, a recombination event which pairs a phenotypically potent ORF, such as <em>CCND1</em>, with a neutral ORF, such as a housekeeping gene, may dampen the enrichment signal, but not reverse it. In the rare cases where a BC recombines to an ORF that confers a very strong and opposing phenotype relative to the ORF it was originally paired with, the effect is still likely to be masked by the combined effects of the additional BCs for that ORF. Thus, the ability to measure internal, independent events and assess their reproducibility within the experiment provides additional confidence in the measurements made with this library.</p><p id="p-46">Retroviral recombination has important implications for experimental design when using constructs with more than one unique element. For example, combinatorial RNAi (co-RNAi) is being explored as both a tool for gene discovery and a therapeutic opportunity for combating resistance-prone diseases like viral infections and cancer (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Grimm and Kay 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cheng <em>et al.</em> 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Diehl <em>et al.</em> 2014</a>). Efforts at co-RNAi optimization have illuminated one downstream effect of lentiviral recombination tendency, in the observation that one or more elements can be deleted through intramolecular recombination when a promoter sequence is present multiple times throughout a construct, driving the expression of several distinct elements (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">ter Brake <em>et al.</em> 2008</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Mcintyre <em>et al.</em> 2009</a>). Thus, efforts have moved toward design of constructs containing multiple promoter/shRNA cassettes that provide efficient and consistent expression (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Sano <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lambeth <em>et al.</em> 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mcintyre <em>et al.</em> 2011</a>). As pooled co-RNAi screening is emerging as a tool for discovery of genetic relationships, the effects of intermolecular recombination will present challenges for library design. One study has shown that the orientation of the second promoter driving hairpin expression does not impact knockdown efficiency (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Lambeth <em>et al.</em> 2010</a>). This could enable library designs in which two unique shRNAs or guide RNAs are positioned nearly adjacent to one another, with flanking promoter sequences driving expression of each in opposite directions. Such a design would overcome the challenges of recombination in dual expression systems. However, the extensive perfect hairpins present in shRNAs could act to enhance reverse transcriptase stalling and template switching <em>in vivo</em>. Thus, the structure of the elements may play a role in addition to spacing, and such effects should be examined. We have described several sources of error that can contribute to biased results of genetic screens performed in cultured mammalian cells. In addition, we have suggested strategies to mitigate these errors, and demonstrated that the application of these strategies to screens performed in our laboratory substantially diminished bias, and will therefore be of use to the screening community.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">We thank Anthony Liang for help with library virus preparation, Eric Wooten for help with sequence alignment and bioinformatic analysis, and members of the Elledge lab for helpful discussions. We thank Andrea Bredemeyer, Anna Mazzuco, and Nicole Solimini for comments on the manuscript. This work was supported by grants from the National Institutes of Health and Department of Defense to S.J.E. S.J.E. is an Investigator with the Howard Hughes Medical Institute.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-48">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030973/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-49"><em>Communicating editor: M. Tyers</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-05-07"><span class="received-label">Received </span>May 7, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-06-27"><span class="accepted-label">Accepted </span>June 27, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Sack <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.1" data-doi="10.1101/cshperspect.a003327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashworth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernards</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Using functional genetics to understand breast cancer biology.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">a003327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a003327%26rft_id%253Dinfo%253Apmid%252F20519343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiIyLzcvYTAwMzMyNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yNzgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.2" data-doi="10.1093/nar/19.8.1949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brakenhoff</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoenmakers</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubsen</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Chimeric cDNA clones: a novel PCR artifact.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBrakenhoff%26rft.auinit1%253DR.%2BH.%26rft.volume%253D19%26rft.issue%253D8%26rft.spage%253D1949%26rft.epage%253D1949%26rft.atitle%253DChimeric%2BcDNA%2Bclones%253A%2Ba%2Bnovel%2BPCR%2Bartifact%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F19.8.1949%26rft_id%253Dinfo%253Apmid%252F2030976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJuYXIiO3M6NToicmVzaWQiO3M6OToiMTkvOC8xOTQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.3" data-doi="10.1038/cgt.2008.102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Multitarget therapy of malignant cancers by the head-to-tail tandem array multiple shRNAs expression system.</span> <abbr class="cit-jnl-abbrev">Cancer Gene Ther.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2Bgene%2Btherapy%26rft.stitle%253DCancer%2BGene%2BTher%26rft.aulast%253DCheng%26rft.auinit1%253DT.%2BL.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D516%26rft.epage%253D531%26rft.atitle%253DMultitarget%2Btherapy%2Bof%2Bmalignant%2Bcancers%2Bby%2Bthe%2Bhead-to-tail%2Btandem%2Barray%2Bmultiple%2BshRNAs%2Bexpression%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcgt.2008.102%26rft_id%253Dinfo%253Apmid%252F19165234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cgt.2008.102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19165234&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266036900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.4" data-doi="10.1073/pnas.1109363108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowley</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehm</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rusin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Systematic investigation of genetic vulnerabilities across cancer cell lines reveals lineage-specific dependencies in ovarian cancer.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">12372</span>–<span class="cit-lpage">12377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1109363108%26rft_id%253Dinfo%253Apmid%252F21746896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzMwLzEyMzcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.5" data-doi="10.1016/B978-0-12-386509-0.00003-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooray</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thrasher</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Retrovirus and lentivirus vector design and methods of cell conditioning.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">507</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DCooray%26rft.auinit1%253DS.%26rft.volume%253D507%26rft.spage%253D29%26rft.epage%253D57%26rft.atitle%253DRetrovirus%2Band%2Blentivirus%2Bvector%2Bdesign%2Band%2Bmethods%2Bof%2Bcell%2Bconditioning.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-386509-0.00003-X%26rft_id%253Dinfo%253Apmid%252F22365768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-386509-0.00003-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22365768&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delviks</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pathak</span>  <span class="cit-name-given-names">V. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Effect of distance between homologous sequences and 3′ homology on the frequency of retroviral reverse transcriptase template switching.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">7923</span>–<span class="cit-lpage">7932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DDelviks%26rft.auinit1%253DK.%2BA.%26rft.volume%253D73%26rft.issue%253D10%26rft.spage%253D7923%26rft.epage%253D7932%26rft.atitle%253DEffect%2Bof%2BDistance%2Bbetween%2BHomologous%2BSequences%2Band%2B3%2527%2BHomology%2Bon%2Bthe%2BFrequency%2Bof%2BRetroviral%2BReverse%2BTranscriptase%2BTemplate%2BSwitching%26rft_id%253Dinfo%253Apmid%252F10482539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3My8xMC83OTIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.7" data-doi="10.1016/j.ddtec.2013.12.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diehl</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tedesco</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenchik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Use of RNAi screens to uncover resistance mechanisms in cancer cells and identify synthetic lethal interactions.</span> <abbr class="cit-jnl-abbrev">Drug Discov. Today. Technol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDrug%2BDiscov.%2BToday.%2BTechnol.%26rft.volume%253D11%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ddtec.2013.12.002%26rft_id%253Dinfo%253Apmid%252F24847648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ddtec.2013.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24847648&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.8" data-doi="10.1038/nmeth1006-777"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Echeverri</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beachy</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baum</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boutros</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchholz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Minimizing the risk of reporting false positives in large-scale RNAi screens.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DEcheverri%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253D777%26rft.epage%253D779%26rft.atitle%253DMinimizing%2Bthe%2Brisk%2Bof%2Breporting%2Bfalse%2Bpositives%2Bin%2Blarge-scale%2BRNAi%2Bscreens.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1006-777%26rft_id%253Dinfo%253Apmid%252F16990807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1006-777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16990807&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240942600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grimm</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Combinatorial RNAi: a winning strategy for the race against evolving targets?</span> <abbr class="cit-jnl-abbrev">Mol. Ther.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">878</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Btherapy%2B%253A%2B%2Bthe%2Bjournal%2Bof%2Bthe%2BAmerican%2BSociety%2Bof%2BGene%2BTherapy%26rft.stitle%253DMol%2BTher%26rft.aulast%253DGrimm%26rft.auinit1%253DD.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D878%26rft.epage%253D888%26rft.atitle%253DCombinatorial%2BRNAi%253A%2Ba%2Bwinning%2Bstrategy%2Bfor%2Bthe%2Brace%2Bagainst%2Bevolving%2Btargets%253F%26rft_id%253Dinfo%253Apmid%252F17311009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17311009&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245999500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.10" data-doi="10.1038/nbt.1534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magtanong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A molecular barcoded yeast ORF library enables mode-of-action analysis of bioactive compounds.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHo%26rft.auinit1%253DC.%2BH.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D369%26rft.epage%253D377%26rft.atitle%253DA%2Bmolecular%2Bbarcoded%2Byeast%2BORF%2Blibrary%2Benables%2Bmode-of-action%2Banalysis%2Bof%2Bbioactive%2Bcompounds.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1534%26rft_id%253Dinfo%253Apmid%252F19349972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19349972&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264971800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.11" data-doi="10.1177/1087057111423101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoshiyama</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batten</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouillard</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of methods for quantitative comparison of pooled shRNAs by mass sequencing.</span> <abbr class="cit-jnl-abbrev">J. Biomol. Screen.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiomol.%2BScreen.%26rft_id%253Dinfo%253Adoi%252F10.1177%252F1087057111423101%26rft_id%253Dinfo%253Apmid%252F21956173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToic3BqYngiO3M6NToicmVzaWQiO3M6ODoiMTcvMi8yNTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjc4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.12" data-doi="10.1093/abbs/gmr116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A primer on using pooled shRNA libraries for functional genomic screens.</span> <abbr class="cit-jnl-abbrev">Acta Biochim. Biophys. Sin. (Shanghai)</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BBiochim.%2BBiophys.%2BSin.%2B%2528Shanghai%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fabbs%252Fgmr116%26rft_id%253Dinfo%253Apmid%252F22271906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWJicyI7czo1OiJyZXNpZCI7czo4OiI0NC8yLzEwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yNzgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.13" data-doi="10.1126/science.1700865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Temin</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Retroviral recombination and reverse transcription.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">250</span>: <span class="cit-fpage">1227</span>–<span class="cit-lpage">1233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHu%26rft.auinit1%253DW.%26rft.volume%253D250%26rft.issue%253D4985%26rft.spage%253D1227%26rft.epage%253D1233%26rft.atitle%253DRetroviral%2Brecombination%2Band%2Breverse%2Btranscription%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1700865%26rft_id%253Dinfo%253Apmid%252F1700865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNTAvNDk4NS8xMjI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.14" data-doi="10.2741/940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">W.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhodes</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pathak</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Retroviral recombination: review of genetic analyses.</span> <abbr class="cit-jnl-abbrev">Front. Biosci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">d143</span>–<span class="cit-lpage">d155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bbioscience%2B%253A%2B%2Ba%2Bjournal%2Band%2Bvirtual%2Blibrary%26rft.stitle%253DFront%2BBiosci%26rft.aulast%253DHu%26rft.auinit1%253DW.%2BS.%26rft.volume%253D8%26rft.spage%253Dd143%26rft.epage%253Dd155%26rft.atitle%253DRetroviral%2Brecombination%253A%2Breview%2Bof%2Bgenetic%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.2741%252F940%26rft_id%253Dinfo%253Apmid%252F12456341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2741/940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12456341&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180266400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.15" data-doi="10.1016/j.bdq.2014.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rensing</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multi-template polymerase chain reaction.</span> <abbr class="cit-jnl-abbrev">Biomol. Detect. Quantif.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiomol.%2BDetect.%2BQuantif.%26rft.volume%253D2%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bdq.2014.11.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bdq.2014.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.16" data-doi="10.1016/S1389-1723(03)90130-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanagawa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Bias and artifacts in multitemplate polymerase chain reactions (PCR).</span> <abbr class="cit-jnl-abbrev">J. Biosci. Bioeng.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiosci%2BBioeng%26rft.aulast%253DKanagawa%26rft.auinit1%253DT.%26rft.volume%253D96%26rft.issue%253D4%26rft.spage%253D317%26rft.epage%253D323%26rft.atitle%253DBias%2Band%2Bartifacts%2Bin%2Bmultitemplate%2Bpolymerase%2Bchain%2Breactions%2B%2528PCR%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1389-1723%252803%252990130-7%26rft_id%253Dinfo%253Apmid%252F16233530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1389-1723(03)90130-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16233530&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187530500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.17" data-doi="10.1089/104303401753153947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makalou</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic determination of the packaging limit of lentiviral vectors.</span> <abbr class="cit-jnl-abbrev">Hum. Gene Ther.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1893</span>–<span class="cit-lpage">1905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgene%2Btherapy%26rft.stitle%253DHum%2BGene%2BTher%26rft.aulast%253DKumar%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D15%26rft.spage%253D1893%26rft.epage%253D1905%26rft.atitle%253DSystematic%2Bdetermination%2Bof%2Bthe%2Bpackaging%2Blimit%2Bof%2Blentiviral%2Bvectors.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F104303401753153947%26rft_id%253Dinfo%253Apmid%252F11589831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/104303401753153947&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11589831&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171398700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.18" data-doi="10.1186/1471-2199-11-77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lambeth</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Hateren</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nair</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A direct comparison of strategies for combinatorial RNA interference.</span> <abbr class="cit-jnl-abbrev">BMC Mol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bmolecular%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BMol%2BBiol%26rft.aulast%253DLambeth%26rft.auinit1%253DL.%2BS.%26rft.volume%253D11%26rft.spage%253D77%26rft.epage%253D77%26rft.atitle%253DA%2Bdirect%2Bcomparison%2Bof%2Bstrategies%2Bfor%2Bcombinatorial%2BRNA%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2199-11-77%26rft_id%253Dinfo%253Apmid%252F20937117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2199-11-77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20937117&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.19" data-doi="10.1186/1743-422X-6-184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mcintyre</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Cassette deletion in multiple shRNA lentiviral vectors for HIV-1 and its impact on treatment success.</span> <abbr class="cit-jnl-abbrev">Virol. J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirol%2BJ%26rft.aulast%253DMcIntyre%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D6%26rft.spage%253D184%26rft.epage%253D184%26rft.atitle%253DCassette%2Bdeletion%2Bin%2Bmultiple%2BshRNA%2Blentiviral%2Bvectors%2Bfor%2BHIV-1%2Band%2Bits%2Bimpact%2Bon%2Btreatment%2Bsuccess.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-6-184%26rft_id%253Dinfo%253Apmid%252F19878571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-6-184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19878571&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.20" data-doi="10.1186/1479-0556-9-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mcintyre</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillespie</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mak</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fanning</span>  <span class="cit-name-given-names">G. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A comparison of multiple shRNA expression methods for combinatorial RNAi.</span> <abbr class="cit-jnl-abbrev">Genet. Vaccines Ther.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenet%2BVaccines%2BTher%26rft.aulast%253DMcIntyre%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D9%26rft.atitle%253DA%2Bcomparison%2Bof%2Bmultiple%2BshRNA%2Bexpression%2Bmethods%2Bfor%2Bcombinatorial%2BRNAi.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1479-0556-9-9%26rft_id%253Dinfo%253Apmid%252F21496330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1479-0556-9-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21496330&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Illumina sequencing library preparation for highly multiplexed target capture and sequencing.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Protoc.</abbr> 2010: pdb.prot5448.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.22" data-doi="10.1146/annurev-biochem-060408-092949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohr</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakal</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrimon</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic screening with RNAi: results and challenges.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DMohr%26rft.auinit1%253DS.%26rft.volume%253D79%26rft.spage%253D37%26rft.epage%253D64%26rft.atitle%253DGenomic%2Bscreening%2Bwith%2BRNAi%253A%2Bresults%2Band%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biochem-060408-092949%26rft_id%253Dinfo%253Apmid%252F20367032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biochem-060408-092949&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20367032&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280225300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.23" data-doi="10.1038/nrmicro903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paillart</span>  <span class="cit-name-given-names">J.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shehu-Xhilaga</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marquet</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mak</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Dimerization of retroviral RNA genomes: an inseparable pair.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DPaillart%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253D461%26rft.epage%253D472%26rft.atitle%253DDimerization%2Bof%2Bretroviral%2BRNA%2Bgenomes%253A%2Ban%2Binseparable%2Bpair.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro903%26rft_id%253Dinfo%253Apmid%252F15152202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15152202&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221586200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.24" data-doi="10.1038/sj.mt.6300298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakanishi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossi</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Expression of long anti-HIV-1 hairpin RNAs for the generation of multiple siRNAs: advantages and limitations.</span> <abbr class="cit-jnl-abbrev">Mol. Ther.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Btherapy%2B%253A%2B%2Bthe%2Bjournal%2Bof%2Bthe%2BAmerican%2BSociety%2Bof%2BGene%2BTherapy%26rft.stitle%253DMol%2BTher%26rft.aulast%253DSano%26rft.auinit1%253DM.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D170%26rft.epage%253D177%26rft.atitle%253DExpression%2Bof%2Blong%2Banti-HIV-1%2Bhairpin%2BRNAs%2Bfor%2Bthe%2Bgeneration%2Bof%2Bmultiple%2BsiRNAs%253A%2Badvantages%2Band%2Blimitations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.mt.6300298%26rft_id%253Dinfo%253Apmid%252F17726454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.mt.6300298&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17726454&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251821000027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.25" data-doi="10.1089/104303404772680029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sastry</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollok</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornetta</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evaluation of plasmid DNA removal from lentiviral vectors by benzonase treatment.</span> <abbr class="cit-jnl-abbrev">Hum. Gene Ther.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgene%2Btherapy%26rft.stitle%253DHum%2BGene%2BTher%26rft.aulast%253DSastry%26rft.auinit1%253DL.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D221%26rft.epage%253D226%26rft.atitle%253DEvaluation%2Bof%2Bplasmid%2BDNA%2Bremoval%2Bfrom%2Blentiviral%2Bvectors%2Bby%2Bbenzonase%2Btreatment.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F104303404772680029%26rft_id%253Dinfo%253Apmid%252F14975194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/104303404772680029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14975194&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220213000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.26" data-doi="10.1126/science.1149200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlabach</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solimini</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cancer proliferation gene discovery through functional genomics.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">620</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchlabach%26rft.auinit1%253DM.%2BR.%26rft.volume%253D319%26rft.issue%253D5863%26rft.spage%253D620%26rft.epage%253D624%26rft.atitle%253DCancer%2BProliferation%2BGene%2BDiscovery%2BThrough%2BFunctional%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1149200%26rft_id%253Dinfo%253Apmid%252F18239126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My82MjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjc4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.27" data-doi="10.1517/14712598.2013.779249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangion</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaillet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garnier</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">New developments in lentiviral vector design, production and purification.</span> <abbr class="cit-jnl-abbrev">Expert Opin. Biol. Ther.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">987</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2BOpin.%2BBiol.%2BTher.%26rft.volume%253D13%26rft.spage%253D987%26rft_id%253Dinfo%253Adoi%252F10.1517%252F14712598.2013.779249%26rft_id%253Dinfo%253Apmid%252F23590247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1517/14712598.2013.779249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23590247&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.28" data-doi="10.1093/nar/27.18.e23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shagin</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukyanov</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vagner</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Regulation of average length of complex PCR product.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">e23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D27%26rft.issue%253D18%26rft.spage%253De23%26rft.epage%253De23%26rft.atitle%253DRegulation%2Bof%2Baverage%2Blength%2Bof%2Bcomplex%2BPCR%2Bproduct%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.18.e23%26rft_id%253Dinfo%253Apmid%252F10471753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.18.e23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10471753&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.29" data-doi="10.1186/gb-2011-12-10-r104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendes-Pereira</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankum</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cerone</span>  <span class="cit-name-given-names">M.-A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-throughput RNA interference screening using pooled shRNA libraries and next generation sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DSims%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253DR104%26rft.epage%253DR104%26rft.atitle%253DHigh-throughput%2BRNA%2Binterference%2Bscreening%2Busing%2Bpooled%2BshRNA%2Blibraries%2Band%2Bnext%2Bgeneration%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-10-r104%26rft_id%253Dinfo%253Apmid%252F22018332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-10-r104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22018332&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.30" data-doi="10.1016/j.virusres.2012.06.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davenport</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mak</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The origin of genetic diversity in HIV-1.</span> <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DSmyth%26rft.auinit1%253DR.%2BP.%26rft.volume%253D169%26rft.issue%253D2%26rft.spage%253D415%26rft.epage%253D429%26rft.atitle%253DThe%2Borigin%2Bof%2Bgenetic%2Bdiversity%2Bin%2BHIV-1.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2012.06.015%26rft_id%253Dinfo%253Apmid%252F22728444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2012.06.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728444&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311652900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.31" data-doi="10.4161/cc.6.8.4184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marine</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Majercak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espeseth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">High-throughput screening by RNA interference: control of two distinct types of variance.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">898</span>–<span class="cit-lpage">901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DStone%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D898%26rft.epage%253D901%26rft.atitle%253DHigh-throughput%2Bscreening%2Bby%2BRNA%2Binterference%253A%2Bcontrol%2Bof%2Btwo%2Bdistinct%2Btypes%2Bof%2Bvariance.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.6.8.4184%26rft_id%253Dinfo%253Apmid%252F17438372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.6.8.4184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17438372&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245948300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.32" data-doi="10.1371/journal.pone.0042341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strezoska</span>  <span class="cit-name-given-names">Ž.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Licon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haimes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spayd</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Optimized PCR conditions and increased shRNA fold representation improve reproducibility of pooled shRNA screens.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e42341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DStrezoska%26rft.auinit1%253DZ.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De42341%26rft.epage%253De42341%26rft.atitle%253DOptimized%2BPCR%2Bconditions%2Band%2Bincreased%2BshRNA%2Bfold%2Brepresentation%2Bimprove%2Breproducibility%2Bof%2Bpooled%2BshRNA%2Bscreens.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0042341%26rft_id%253Dinfo%253Apmid%252F22870320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0042341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22870320&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giovannoni</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Bias caused by template annealing in the amplification of mixtures of 16S rRNA genes by PCR.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DSuzuki%26rft.auinit1%253DM.%2BT.%26rft.volume%253D62%26rft.issue%253D2%26rft.spage%253D625%26rft.epage%253D630%26rft.atitle%253DBias%2Bcaused%2Bby%2Btemplate%2Bannealing%2Bin%2Bthe%2Bamplification%2Bof%2Bmixtures%2Bof%2B16S%2BrRNA%2Bgenes%2Bby%2BPCR%26rft_id%253Dinfo%253Apmid%252F8593063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjg6IjYyLzIvNjI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.34" data-doi="10.1038/sj.mt.6300382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">ter Brake</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">’t Hooft</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Centlivre</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Eije</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Lentiviral vector design for multiple shRNA expression and durable HIV-1 inhibition.</span> <abbr class="cit-jnl-abbrev">Mol. Ther.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Btherapy%2B%253A%2B%2Bthe%2Bjournal%2Bof%2Bthe%2BAmerican%2BSociety%2Bof%2BGene%2BTherapy%26rft.stitle%253DMol%2BTher%26rft.aulast%253DBrake%26rft.auinit1%253DO.%2BT.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D557%26rft.epage%253D564%26rft.atitle%253DLentiviral%2Bvector%2Bdesign%2Bfor%2Bmultiple%2BshRNA%2Bexpression%2Band%2Bdurable%2BHIV-1%2Binhibition.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.mt.6300382%26rft_id%253Dinfo%253Apmid%252F18180777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.mt.6300382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18180777&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253548100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.35" data-doi="10.1089/hgtb.2011.199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karne</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Black</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A simple and effective method to generate lentiviral vectors for ex vivo gene delivery to mature human peripheral blood lymphocytes.</span> <abbr class="cit-jnl-abbrev">Hum. Gene Ther. Methods</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DHum%2BGene%2BTher%2BMethods%26rft.aulast%253DYang%26rft.auinit1%253DS.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D73%26rft.epage%253D83%26rft.atitle%253DA%2Bsimple%2Band%2Beffective%2Bmethod%2Bto%2Bgenerate%2Blentiviral%2Bvectors%2Bfor%2Bex%2Bvivo%2Bgene%2Bdelivery%2Bto%2Bmature%2Bhuman%2Bperipheral%2Bblood%2Blymphocytes.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fhgtb.2011.199%26rft_id%253Dinfo%253Apmid%252F22515320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/hgtb.2011.199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22515320&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2781.36" data-doi="10.1038/nmeth.1638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehm</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salehi-Ashtiani</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A public genome-scale lentiviral expression library of human ORFs.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DYang%26rft.auinit1%253DX.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D659%26rft.epage%253D661%26rft.atitle%253DA%2Bpublic%2Bgenome-scale%2Blentiviral%2Bexpression%2Blibrary%2Bof%2Bhuman%2BORFs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1638%26rft_id%253Dinfo%253Apmid%252F21706014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21706014&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2781.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293220600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/9/2781.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2773" title="A Comparative Analysis of 5-Azacytidine- and Zebularine-Induced DNA Demethylation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2791" title="Whole Genome Comparison of Thermus sp. NMX2.A1 Reveals Principal Carbon Metabolism Differences with Closest Relation Thermus scotoductus SA-01" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/9"><span>Volume 6 Issue 9, September 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/9.cover-source.jpg?itok=nund1hRf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2781" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6gj3ayuVyYwIDDcAIcdadcmehSlOz-lQgkcD-va88Yg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/9/2781" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/9/2781" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/9/2781" class="active" data-icon-position="" data-hide-link-title="0">Sources of Error in Mammalian Genetic Screens</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460306" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Jp3sWvQpUTuEWGqpdNGFWezmc1onVpbIDjRI-d6mhkA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2781.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/9/2781" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-QSqr2x7QeWL9JesQoLYYhjThIiO7A46N7KXHFxdc7Rk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/9/2781" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2781.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460306" id="node460306--2" data-pisa="ggg;6/9/2781" data-pisa-master="ggg;g3.116.030973" data-apath="/ggg/6/9/2781.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sources of Error in Mammalian Genetic Screens</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Laura Magill Sack</span>, <span class="highwire-citation-author" data-delta="1">Teresa Davoli</span>, <span class="highwire-citation-author" data-delta="2">Qikai Xu</span>, <span class="highwire-citation-author" data-delta="3">Mamie Z. Li</span> and <span class="highwire-citation-author" data-delta="4">Stephen J. Elledge</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2781-2790;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030973 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460306--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Laura Magill Sack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BMagill%2BSack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sack%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaura%2BMagill%2BSack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Teresa Davoli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Teresa%2BDavoli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davoli%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATeresa%2BDavoli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qikai Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qikai%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQikai%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mamie Z. Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mamie%2BZ.%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20MZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMamie%2BZ.%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Stephen J. Elledge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BJ.%2BElledge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Elledge%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BJ.%2BElledge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:selledge@genetics.med.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">selledge@genetics.med.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/9/2781" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/9/2781&amp;title=Sources%20of%20Error%20in%20Mammalian%20Genetic%20Screens" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/9/2781&amp;title=Sources%20of%20Error%20in%20Mammalian%20Genetic%20Screens" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/9/2781&amp;title=Sources%20of%20Error%20in%20Mammalian%20Genetic%20Screens" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/9/2781&amp;text=Sources%20of%20Error%20in%20Mammalian%20Genetic%20Screens" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/9/2781&amp;title=Sources%20of%20Error%20in%20Mammalian%20Genetic%20Screens" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/9/2781&amp;t=Sources%20of%20Error%20in%20Mammalian%20Genetic%20Screens" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/9/2781&amp;title=Sources%20of%20Error%20in%20Mammalian%20Genetic%20Screens" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/9/2781&amp;title=Sources%20of%20Error%20in%20Mammalian%20Genetic%20Screens" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460306/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460306" id="node460306--3" data-pisa="ggg;6/9/2781" data-pisa-master="ggg;g3.116.030973" data-apath="/ggg/6/9/2781.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sources of Error in Mammalian Genetic Screens</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Laura Magill Sack</span>, <span class="highwire-citation-author" data-delta="1">Teresa Davoli</span>, <span class="highwire-citation-author" data-delta="2">Qikai Xu</span>, <span class="highwire-citation-author" data-delta="3">Mamie Z. Li</span> and <span class="highwire-citation-author" data-delta="4">Stephen J. Elledge</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2781-2790;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030973 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460306--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Laura Magill Sack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BMagill%2BSack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sack%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaura%2BMagill%2BSack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Teresa Davoli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Teresa%2BDavoli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davoli%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATeresa%2BDavoli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qikai Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qikai%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQikai%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mamie Z. Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mamie%2BZ.%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20MZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMamie%2BZ.%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Stephen J. Elledge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BJ.%2BElledge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Elledge%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BJ.%2BElledge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:selledge@genetics.med.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">selledge@genetics.med.harvard.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2781&amp;rft.epage=2790&amp;rft.atitle=Sources%20of%20Error%20in%20Mammalian%20Genetic%20Screens&amp;rft.volume=6&amp;rft.issue=9&amp;rft.date=2016-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sack%2C+Laura+Magill&amp;rft.au=Davoli%2C+Teresa&amp;rft.au=Xu%2C+Qikai&amp;rft.au=Li%2C+Mamie+Z.&amp;rft.au=Elledge%2C+Stephen+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460306/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460306/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460306/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460306/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460306/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460306/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460306/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460306/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460306/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460306/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460306/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460306/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/9/2781" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/9/2781.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/9/2781.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/9/2781.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/9/2781.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
