<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/9/3003" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/9/3003.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/9/3003.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/9/3003.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/9/3003.atom" />
<meta name="HW.pisa" content="ggg;6/9/3003" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale" />
<meta name="DC.Identifier" content="10.1534/g3.116.032342" />
<meta name="DC.Date" content="2016-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Zackrisson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The capacity to map traits over large cohorts of individuals—phenomics—lags far behind the explosive development in genomics. For microbes, the estimation of growth is the key phenotype because of its link to fitness. We introduce an automated microbial phenomics framework that delivers accurate, precise, and highly resolved growth phenotypes at an unprecedented scale. Advancements were achieved through the introduction of transmissive scanning hardware and software technology, frequent acquisition of exact colony population size measurements, extraction of population growth rates from growth curves, and removal of spatial bias by reference-surface normalization. Our prototype arrangement automatically records and analyzes close to 100,000 growth curves in parallel. We demonstrate the power of the approach by extending and nuancing the known salt-defense biology in baker’s yeast. The introduced framework represents a major advance in microbial phenomics by providing high-quality data for extensive cohorts of individuals and generating well-populated and standardized phenomics databases" />
<meta name="DC.Contributor" content="Martin Zackrisson" />
<meta name="DC.Contributor" content="Johan Hallin" />
<meta name="DC.Contributor" content="Lars-Göran Ottosson" />
<meta name="DC.Contributor" content="Peter Dahl" />
<meta name="DC.Contributor" content="Esteban Fernandez-Parada" />
<meta name="DC.Contributor" content="Erik Ländström" />
<meta name="DC.Contributor" content="Luciano Fernandez-Ricaud" />
<meta name="DC.Contributor" content="Petra Kaferle" />
<meta name="DC.Contributor" content="Andreas Skyman" />
<meta name="DC.Contributor" content="Simon Stenberg" />
<meta name="DC.Contributor" content="Stig Omholt" />
<meta name="DC.Contributor" content="Uroš Petrovič" />
<meta name="DC.Contributor" content="Jonas Warringer" />
<meta name="DC.Contributor" content="Anders Blomberg" />
<meta name="article:published_time" content="2016-09-01" />
<meta name="article:section" content="Mutant Screen Report" />
<meta name="citation_title" content="Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The capacity to map traits over large cohorts of individuals—phenomics—lags far behind the explosive development in genomics. For microbes, the estimation of growth is the key phenotype because of its link to fitness. We introduce an automated microbial phenomics framework that delivers accurate, precise, and highly resolved growth phenotypes at an unprecedented scale. Advancements were achieved through the introduction of transmissive scanning hardware and software technology, frequent acquisition of exact colony population size measurements, extraction of population growth rates from growth curves, and removal of spatial bias by reference-surface normalization. Our prototype arrangement automatically records and analyzes close to 100,000 growth curves in parallel. We demonstrate the power of the approach by extending and nuancing the known salt-defense biology in baker’s yeast. The introduced framework represents a major advance in microbial phenomics by providing high-quality data for extensive cohorts of individuals and generating well-populated and standardized phenomics databases&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/09/01" />
<meta name="citation_mjid" content="ggg;6/9/3003" />
<meta name="citation_id" content="6/9/3003" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/9/3003" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/9/3003.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/9/3003.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/9/3003.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.032342" />
<meta name="citation_pmid" content="27371952" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Mutant Screen Report" />
<meta name="citation_firstpage" content="3003" />
<meta name="citation_lastpage" content="3014" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Martin Zackrisson" />
<meta name="citation_author_institution" content="Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden" />
<meta name="citation_author" content="Johan Hallin" />
<meta name="citation_author_institution" content="IRCAN, CNRS UMR 6267, INSERM U998, University of Nice, 06107, France" />
<meta name="citation_author" content="Lars-Göran Ottosson" />
<meta name="citation_author_institution" content="Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden" />
<meta name="citation_author" content="Peter Dahl" />
<meta name="citation_author_institution" content="Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden" />
<meta name="citation_author" content="Esteban Fernandez-Parada" />
<meta name="citation_author_institution" content="Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden" />
<meta name="citation_author" content="Erik Ländström" />
<meta name="citation_author_institution" content="Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden" />
<meta name="citation_author" content="Luciano Fernandez-Ricaud" />
<meta name="citation_author_institution" content="Department of Marine Sciences, University of Gothenburg, 40530, Sweden" />
<meta name="citation_author" content="Petra Kaferle" />
<meta name="citation_author_institution" content="Department of Molecular and Biomedical Sciences, Jožef Stefan Institute, SI-1000 Ljubljana, Slovenia" />
<meta name="citation_author" content="Andreas Skyman" />
<meta name="citation_author_institution" content="Department of Earth and Space Sciences, Chalmers University of Technology, SE-412 96 Gothenburg, Sweden" />
<meta name="citation_author" content="Simon Stenberg" />
<meta name="citation_author_institution" content="Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden" />
<meta name="citation_author_institution" content="Department of Animal and Aquacultural Sciences, Centre for Integrative Genetics (CIGENE), Norwegian University of Life Sciences (UMB), 1432 Ås, Norway" />
<meta name="citation_author" content="Stig Omholt" />
<meta name="citation_author_institution" content="Department of Biotechnology, Faculty of Natural Sciences and Technology, NTNU Norwegian University of Science and Technology, N-7491 Trondheim, Norway" />
<meta name="citation_author" content="Uroš Petrovič" />
<meta name="citation_author_institution" content="Department of Molecular and Biomedical Sciences, Jožef Stefan Institute, SI-1000 Ljubljana, Slovenia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9743-9145" />
<meta name="citation_author" content="Jonas Warringer" />
<meta name="citation_author_institution" content="Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden" />
<meta name="citation_author_institution" content="Department of Animal and Aquacultural Sciences, Centre for Integrative Genetics (CIGENE), Norwegian University of Life Sciences (UMB), 1432 Ås, Norway" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5269-971X" />
<meta name="citation_author" content="Anders Blomberg" />
<meta name="citation_author_institution" content="Department of Marine Sciences, University of Gothenburg, 40530, Sweden" />
<meta name="citation_author_email" content="anders.blomberg@marine.gu.se" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=A. Balguerie;citation_author=M. Bagnat;citation_author=M. Bonneu;citation_author=M. Aigle;citation_author=A. M. Breton;citation_title=Rvs161p and Sphingolipids Are Required for Actin Repolarization following Salt Stress;citation_pages=1021-1031;citation_volume=1;citation_year=2002;citation_issue=6;citation_pmid=12477802;citation_doi=10.1128/EC.1.6.1021-1031.2002" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_title=A quantitative fitness analysis workflow.;citation_author=A. P. Banks;citation_author=C. Lawless;citation_author=D. A. Lydall;citation_volume=66;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Baryshnikova;citation_author=M. Costanzo;citation_author=Y. Kim;citation_author=H. Ding;citation_author=J. Koh;citation_title=Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.;citation_pages=1017-1024;citation_volume=7;citation_year=2010;citation_issue=12;citation_pmid=21076421;citation_doi=10.1038/nmeth.1534" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. J. Bean;citation_author=P. A. Jaeger;citation_author=S. Bahr;citation_author=T. Ideker;citation_title=Development of ultra-high-density screening tools for microbial “omics”.;citation_pages=e85177;citation_volume=9;citation_year=2014;citation_issue=1;citation_pmid=24465499;citation_doi=10.1371/journal.pone.0085177" />
<meta name="citation_reference" content="citation_journal_title=Current Opinion in Biotechnology;citation_journal_abbrev=Current Opinion in Biotechnology;citation_author=A. Blomberg;citation_title=Measuring growth rate in high-throughput growth phenotyping.;citation_pages=94-102;citation_volume=22;citation_year=2011;citation_issue=1;citation_pmid=21095113;citation_doi=10.1016/j.copbio.2010.10.013" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. S. Bloom;citation_author=I. M. Ehrenreich;citation_author=W. T. Loo;citation_author=T. L. Lite;citation_author=L. Kruglyak;citation_title=Finding the sources of missing heritability in a yeast cross.;citation_pages=234-237;citation_volume=494;citation_year=2013;citation_issue=7436;citation_pmid=23376951;citation_doi=10.1038/nature11867" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=C. B. Brachmann;citation_author=A. Davies;citation_author=G. J. Cost;citation_author=E. Caputo;citation_author=J. Li;citation_title=Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.;citation_pages=115-132;citation_volume=14;citation_year=1998;citation_issue=2;citation_pmid=9483801;citation_doi=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. R. Collins;citation_author=M. Schuldiner;citation_author=N. J. Krogan;citation_author=J. S. Weissman;citation_title=A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.;citation_pages=R63-R63;citation_volume=7;citation_year=2006;citation_issue=7;citation_pmid=16859555;citation_doi=10.1186/gb-2006-7-7-r63" />
<meta name="citation_reference" content="citation_title=Bacterial Growth and Division;citation_author=S. Cooper;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. A. Cubillos;citation_author=E. Billi;citation_author=E. Zorgo;citation_author=L. Parts;citation_author=P. Fargier;citation_title=Assessing the complex architecture of polygenic traits in diverged yeast populations.;citation_pages=1401-1413;citation_volume=20;citation_year=2011;citation_issue=7;citation_pmid=21261765;citation_doi=10.1111/j.1365-294X.2011.05005.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. A. Cubillos;citation_author=L. Parts;citation_author=F. Salinas;citation_author=A. Bergstrom;citation_author=E. Scovacricchi;citation_title=High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.;citation_volume=195;citation_year=2013;citation_issue=3;citation_pmid=24037264;citation_doi=10.1534/genetics.113.155515" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=Progress in nucleic acid research and molecular biology;citation_journal_abbrev=Prog Nucleic Acid Res Mol Biol;citation_author=C. V. Glover;citation_title=On the physiological role of casein kinase II in Saccharomyces cerevisiae.;citation_pages=95-133;citation_volume=59;citation_year=1998;citation_pmid=9427841;citation_doi=10.1016/S0079-6603(08)61030-2" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=Powerful decomposition of complex traits in a diploid model using Phased Outbred Lines.;citation_author=J. Hallin;citation_author=K. Märtens;citation_author=A. Young;citation_author=M. Zackrisson;citation_author=F. Salinas;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genes (Basel);citation_author=J. L. t. Hartman;citation_author=C. Stisher;citation_author=D. A. Outlaw;citation_author=J. Guo;citation_author=N. A. Shah;citation_title=Yeast Phenomics: An Experimental Approach for Modeling Gene Interaction Networks that Buffer Disease.;citation_pages=24-45;citation_volume=6;citation_year=2015;citation_issue=1;citation_pmid=25668739;citation_doi=10.3390/genes6010024" />
<meta name="citation_reference" content="citation_journal_title=Appl. Microbiol. Biotechnol.;citation_author=M. C. M. Hensing;citation_author=K. A. Bangma;citation_author=L. M. Raamsdonk;citation_author=E. de Hulster;citation_author=J. P. van Dijken;citation_title=Effects of cultivation conditions on the production of heterologous α-galactosidase by Kluyveromyces lactis.;citation_pages=58-64;citation_volume=43;citation_year=1995;citation_issue=1;citation_doi=10.1007/BF00170623" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=S. Hohmann;citation_title=Osmotic Stress Signaling and Osmoadaptation in Yeasts;citation_pages=300-372;citation_volume=66;citation_year=2002;citation_issue=2;citation_pmid=12040128;citation_doi=10.1128/MMBR.66.2.300-372.2002" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. Houle;citation_author=D. R. Govindaraju;citation_author=S. Omholt;citation_title=Phenomics: the next challenge.;citation_pages=855-866;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21085204;citation_doi=10.1038/nrg2897" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Ibstedt;citation_author=S. Stenberg;citation_author=S. Bages;citation_author=A. B. Gjuvsland;citation_author=F. Salinas;citation_title=Concerted evolution of life stage performances signals recent selection on yeast nitrogen use.;citation_volume=32;citation_year=2015;citation_issue=1;citation_pmid=25349282;citation_doi=10.1093/molbev/msu285" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=D. C. Koboldt;citation_author=K. M. Steinberg;citation_author=D. E. Larson;citation_author=R. K. Wilson;citation_author=E. R. Mardis;citation_title=The next-generation sequencing revolution and its impact on genomics.;citation_pages=27-38;citation_volume=155;citation_year=2013;citation_issue=1;citation_pmid=24074859;citation_doi=10.1016/j.cell.2013.09.006" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. J. Kvitek;citation_author=J. L. Will;citation_author=A. P. Gasch;citation_title=Variations in stress sensitivity and genomic expression in diverse S. cerevisiae isolates.;citation_pages=e1000223-e1000223;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18927628;citation_doi=10.1371/journal.pgen.1000223" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_title=Colonyzer: automated quantification of micro-organism growth characteristics on solid agar.;citation_author=C. Lawless;citation_author=D. J. Wilkinson;citation_author=A. Young;citation_author=S. G. Addinall;citation_author=D. A. Lydall;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. F. Levy;citation_author=N. Ziv;citation_author=M. L. Siegal;citation_title=Bet hedging in yeast by heterogeneous, age-correlated expression of a stress protectant.;citation_pages=e1001325-e1001325;citation_volume=10;citation_year=2012;citation_issue=5;citation_pmid=22589700;citation_doi=10.1371/journal.pbio.1001325" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=Z. Li;citation_author=F. J. Vizeacoumar;citation_author=S. Bahr;citation_author=J. Li;citation_author=J. Warringer;citation_title=Systematic exploration of essential yeast gene function with temperature-sensitive mutants.;citation_pages=361-367;citation_volume=29;citation_year=2011;citation_issue=4;citation_pmid=21441928;citation_doi=10.1038/nbt.1832" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_journal_abbrev=elife;citation_author=G. Liti;citation_title=The fascinating and secret wild life of the budding yeast S. cerevisiae;citation_pages=e05835-e05835;citation_volume=4;citation_year=2015;citation_issue=0;citation_doi=10.7554/eLife.05835" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=K. Logg;citation_author=K. Bodvard;citation_author=A. Blomberg;citation_author=M. Kall;citation_title=Investigations on light-induced stress in fluorescence microscopy using nuclear localization of the transcription factor Msn2p as a reporter.;citation_volume=9;citation_year=2009;citation_issue=6;citation_pmid=19566686;citation_doi=10.1111/j.1567-1364.2009.00536.x" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=R. Lombardi;citation_author=H. Riezman;citation_title=Rvs161p and Rvs167p, the two yeast amphiphysin homologs, function together in vivo.;citation_volume=276;citation_year=2001;citation_issue=8;citation_pmid=11096097;citation_doi=10.1074/jbc.M008735200" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Anal. Chem. (Palo Alto, Calif.);citation_author=E. R. Mardis;citation_title=Next-generation sequencing platforms.;citation_pages=287-303;citation_volume=6;citation_year=2013;citation_pmid=23560931;citation_doi=10.1146/annurev-anchem-062012-092628" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=K. Martens;citation_author=J. Hallin;citation_author=J. Warringer;citation_author=G. Liti;citation_author=L. Parts;citation_title=Predicting quantitative traits from genome and phenome with near perfect accuracy.;citation_pages=11512;citation_volume=7;citation_year=2016;citation_pmid=27160605;citation_doi=10.1038/ncomms11512" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. M. Mulet;citation_author=M. P. Leube;citation_author=S. J. Kron;citation_author=G. Rios;citation_author=G. R. Fink;citation_title=A Novel Mechanism of Ion Homeostasis and Salt Tolerance in Yeast: the Hal4 and Hal5 Protein Kinases Modulate the Trk1-Trk2 Potassium Transporter;citation_pages=3328-3337;citation_volume=19;citation_year=1999;citation_issue=5;citation_pmid=10207057;citation_doi=10.1128/MCB.19.5.3328" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Narayanan;citation_author=M. Dubarry;citation_author=C. Lawless;citation_author=A. P. Banks;citation_author=D. J. Wilkinson;citation_title=Quantitative Fitness Analysis Identifies exo1 and Other Suppressors or Enhancers of Telomere Defects in Schizosaccharomyces pombe.;citation_pages=e0132240;citation_volume=10;citation_year=2015;citation_issue=7;citation_pmid=26168240;citation_doi=10.1371/journal.pone.0132240" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Parts;citation_author=F. A. Cubillos;citation_author=J. Warringer;citation_author=K. Jain;citation_author=F. Salinas;citation_title=Revealing the genetic structure of a trait by sequencing a population under selection.;citation_volume=21;citation_year=2011;citation_issue=7;citation_pmid=21422276;citation_doi=10.1101/gr.116731.110" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Biosyst;citation_author=L. Z. Pipe;citation_author=M. J. Grimson;citation_title=Spatial-temporal modelling of bacterial colony growth on solid media.;citation_pages=192-198;citation_volume=4;citation_year=2008;citation_issue=3;citation_pmid=18437261;citation_doi=10.1039/b708241j" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. Sopko;citation_author=D. Huang;citation_author=N. Preston;citation_author=G. Chua;citation_author=B. Papp;citation_title=Mapping pathways and phenotypes by systematic gene overexpression.;citation_pages=319-330;citation_volume=21;citation_year=2006;citation_issue=3;citation_pmid=16455487;citation_doi=10.1016/j.molcel.2005.12.011" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. M. Stathopoulos;citation_author=M. S. Cyert;citation_title=Calcineurin acts through the CRZ1/TCN1-encoded transcription factor to regulate gene expression in yeast;citation_pages=3432-3444;citation_volume=11;citation_year=1997;citation_issue=24;citation_pmid=9407035;citation_doi=10.1101/gad.11.24.3432" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. W. Thatcher;citation_author=J. M. Shaw;citation_author=W. J. Dickinson;citation_title=Marginal fitness contributions of nonessential genes in yeast;citation_pages=253-257;citation_volume=95;citation_year=1998;citation_issue=1;citation_pmid=9419362;citation_doi=10.1073/pnas.95.1.253" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=Comput. Sci. Eng.;citation_author=S. van der Walt;citation_author=S. C. Colbert;citation_author=G. Varoquaux;citation_title=The NumPy Array: A Structure for Efficient Numerical Computation.;citation_pages=22-30;citation_volume=13;citation_year=2011;citation_issue=2;citation_doi=10.1109/MCSE.2010.145" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=S. van der Walt;citation_author=J. L. Schonberger;citation_author=J. Nunez-Iglesias;citation_author=F. Boulogne;citation_author=J. D. Warner;citation_title=scikit-image: image processing in Python.;citation_pages=e453;citation_volume=2;citation_year=2014;citation_pmid=25024921;citation_doi=10.7717/peerj.453" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Wagner;citation_title=The Role of Population Size, Pleiotropy and Fitness Effects of Mutations in the Evolution of Overlapping Gene Functions;citation_pages=1389-1401;citation_volume=154;citation_year=2000;citation_issue=3;citation_pmid=10757778" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. E. Walker;citation_author=T. D. Nguyen;citation_author=T. Liccioli;citation_author=F. Schmid;citation_author=N. Kalatzis;citation_title=Genome-wide identification of the Fermentome; genes required for successful and timely completion of wine-like fermentation by Saccharomyces cerevisiae.;citation_pages=552;citation_volume=15;citation_year=2014;citation_pmid=24993029;citation_doi=10.1186/1471-2164-15-552" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=J. Warringer;citation_author=A. Blomberg;citation_title=Automated screening in environmental arrays allows analysis of quantitative phenotypic profiles in Saccharomyces cerevisiae.;citation_pages=53-67;citation_volume=20;citation_year=2003;citation_issue=1;citation_pmid=12489126;citation_doi=10.1002/yea.931" />
<meta name="citation_reference" content="citation_title=Phenomics;citation_title=Yeast Phenomics-Large scale mapping of the genetic basis for organismal traits;citation_author=J. Warringer;citation_author=A. Blomberg;citation_pages=172-207;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Warringer;citation_author=E. Ericson;citation_author=L. Fernandez;citation_author=O. Nerman;citation_author=A. Blomberg;citation_title=High-resolution yeast phenomics resolves different physiological features in the saline response.;citation_volume=100;citation_year=2003;citation_issue=26;citation_pmid=14676322;citation_doi=10.1073/pnas.2435976100" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Chem Biol;citation_author=J. Warringer;citation_author=D. Anevski;citation_author=B. Liu;citation_author=A. Blomberg;citation_title=Chemogenetic fingerprinting by analysis of cellular growth dynamics.;citation_pages=3-3;citation_volume=8;citation_year=2008;citation_pmid=18721464;citation_doi=10.1186/1472-6769-8-3" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Warringer;citation_author=E. Zorgo;citation_author=F. A. Cubillos;citation_author=A. Zia;citation_author=A. Gjuvsland;citation_title=Trait variation in yeast is defined by population history.;citation_pages=e1002111-e1002111;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21698134;citation_doi=10.1371/journal.pgen.1002111" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. D. Shoemaker;citation_author=A. Astromoff;citation_author=H. Liang;citation_author=K. Anderson;citation_title=Functional Characterization of the S. cerevisiae Genome by Gene Deletion and Parallel Analysis;citation_pages=901-906;citation_volume=285;citation_year=1999;citation_issue=5429;citation_pmid=10436161;citation_doi=10.1126/science.285.5429.901" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="twitter:description" content="The capacity to map traits over large cohorts of individuals—phenomics—lags far behind the explosive development in genomics. For microbes, the estimation of growth is the key phenotype because of its link to fitness. We introduce an automated microbial phenomics framework that delivers accurate, precise, and highly resolved growth phenotypes at an unprecedented scale. Advancements were achieved through the introduction of transmissive scanning hardware and software technology, frequent acquisition of exact colony population size measurements, extraction of population growth rates from growth curves, and removal of spatial bias by reference-surface normalization. Our prototype arrangement automatically records and analyzes close to 100,000 growth curves in parallel. We demonstrate the power of the approach by extending and nuancing the known salt-defense biology in baker’s yeast. The introduced framework represents a major advance in microbial phenomics by providing high-quality data for extensive cohorts of individuals and generating well-populated and standardized phenomics databases" />
<meta name="og-title" property="og:title" content="Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/9/3003" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The capacity to map traits over large cohorts of individuals—phenomics—lags far behind the explosive development in genomics. For microbes, the estimation of growth is the key phenotype because of its link to fitness. We introduce an automated microbial phenomics framework that delivers accurate, precise, and highly resolved growth phenotypes at an unprecedented scale. Advancements were achieved through the introduction of transmissive scanning hardware and software technology, frequent acquisition of exact colony population size measurements, extraction of population growth rates from growth curves, and removal of spatial bias by reference-surface normalization. Our prototype arrangement automatically records and analyzes close to 100,000 growth curves in parallel. We demonstrate the power of the approach by extending and nuancing the known salt-defense biology in baker’s yeast. The introduced framework represents a major advance in microbial phenomics by providing high-quality data for extensive cohorts of individuals and generating well-populated and standardized phenomics databases" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
    <title>Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"P5TaPkqh4rIYflHGPUpI2ZYOKdZ8foo8rvppMiMgGd8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460300","apath":"\/ggg\/6\/9\/3003.atom","pisa":"ggg;6\/9\/3003","ac":{"\/ggg\/6\/9\/3003.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/9\/3003.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/9\/3003"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/9\/3003"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/9\/3003.full":true,"\/content\/6\/9\/3003":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lUMu10xHKKdLgx+MmHH1FkRm2xk7WsGQzkmpPzUIauqVpjppM7LZefee8xXf1x7luSykRJYrrrfLotI6yt2YBt66rZsyO2DVE2r0G9hIYm7QyqD8uFakDZwv6rVjVq2ZFNQ\\\\\\\/3\\\\\\\/dEmqP0s1jqOJoJVppzOa6MT3FmCCEXgKNgskmpvHsgiUuykOQboSB6a4KUutf7iY5VxZyzG7LQA28pqxnm2ohV5T1P3+2PR6pngWzAuqlqPozLE2UNsXQ+fgauiRZo55LD5vIrKlrcnIE8brBwondaOJSjeGaGe2QRVZq4qvQUYlWh3USKxo1\\\\\\\/Du9CZ2FjMSCa2ASTHHwjkFMhgYHwGVYo37vVtAkccSoJPuk7Ka+oJde74Xjkelpl\\\\\\\/\\\\\\\/2+5+629gEmAIjm4mIkjXI5ME0+8KzLIs1PZIgf6FWEaq33zt9U4l+kCSxYphZnAtIqyPmHS8n9I5pdTLvj3pukauXZsmH1Zgks0E3JJS3MWwZD4VPhtLU15WwB8kNX44tFYrKzu4s2eK\\\\\\\/ddAlgg6qvF\\\\\\\/0RoxFEWkgmk7MQJxM4lrY4df2CJetUmv1e8BmXyvWZG5kqf+twjTblQWNOVV1miDmGQ\\\\\\\/paT8einxBwUp2aTMkDiKdAEAdkG11YrF6lhhmhYUPXHTHcVEOBEhrijQdnWRaxVAXwJ0+drrdnnzdbYnL3JkSKRsp\\\\\\\/p\\\\\\\/sbBkXacLn5hm1\\\\\\\/o9bunG5watDOFUhtdSPIkijHT9SRv3Vo5TlBUIMJezC\\\\\\\/qluHczusxri9l34TAZPyVYrBt3TMbpcp+Y3m+7oRJD5jMeWq4WDN3HkvGgzRKMImW2AsEaeVKSKH2+AvxQ8cktlcjyfqC\\\\\\\/xDw\\\\\\\/2EEe+kilbkIiImDideLwzfvfMF2yOLSU2zpS738vG6SlEdlvvd74buge+vG6Mwauf638kx+3zjj+5hEsHRsB1tAjwweMRjzKs963ukmIYAxG3K+yWJrYsgnFciDT4jP\\\\\\\/3j+fnMujNU\\\\\\\/83v\\\\\\\/oAyqsLRJc4KWdrB1yRB\\\\\\\/HD30uxmCfAKxBloA6JqJVjLXtfshnUhLi9I3yFs2hf0nOfUMSA5AS2Xo0rDDXZwufLq3uQp937vnsBfDUMVdzpU1DAx757exnkG+wzK8lIf0laH48NGTHOXdWaAqgu3arMSXO9F+JB1+dlzcesK0INbLi2DEPasSdvYaAatCAMMx8pKX6E36i2viJxDTp8JW7jqVRstv16EnoKJYPqy8P6kRvEabJ5mPzhVSj6Ov0z1t+5yKpN9auuTMNd8SkJyqw8pQy+YNKoV\\\u0022,\\\u0022iv\\\u0022:\\\u0022HFQqoWOxshMbg1RF0X5JlA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226b1dcab1cbe2e2c46289758c17ae5eb0c93f776da39d3fef14e46f6be6855e78\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cs7o6T+zmzXRUINe1aZXz8cAXIJxotoKCZkDaxBvbJE6gLX5icYFTAJOOTZRlME\\\\\\\/OkErkxudftrimWrFx4S3vXqHAgeUK8nZV8lGrg9s8GzB5ucuXF87q2aVNiqU2cwIvt4n2HK8VFxhEf\\\\\\\/gTDe813h70DArv\\\\\\\/OF6pGvLfNwQjJyxv6CjySCr5O6mtueak4ZV5rWBGWkvNR0LVAEPM9HVJSXdLpS4VbddBj9EWL7mGjE6mPLXf66oKYEp8u5SHXPW4cGAPF4IaawzeufmmUymig9B64TwcrMBX\\\\\\\/usTgwImqbLfz1hrtTInqnbNaFO2WTk6nvhofR52dxitXoTYco7H0R84rpjku0SELkX7Ik\\\\\\\/g4VuYK3uSKozvMBiNWdsqyEXD1D\\\\\\\/M2jzmLK0CE1H4ROIS\\\\\\\/sCDhdrPs2UHr4cp9j9Blq2DyQfnM7RoPthmTw1FjhuRKJsokhS\\\\\\\/LXMhpvmuThZ3GXsj4CkMxTzAgRIw\\\\\\\/jbWd+oq4WUZdEORl44pP025QqplwFg43EbcmkQc2TMBgBI3dbiAHW2t7nJ8jqVu1etIO7zi57D3D0cfAjSSiaKWE+AoDpWo6D3r9yumH4p5dRA6jxAHfsJOcYDkZiSGLPbuhwsqKLx9tMHWrsrLetjxqMAnV4zy+S\\\\\\\/jzm5Gk2A8Ys21YL1yRziFe8GCbB8Mr8EEOxNpINPbhVaWYGaJOAarXsOKdBk7mibIn+TVAMOyq\\\\\\\/hM8+NcL\\\\\\\/Sjmt4K+dlf2AfFkhl0lVlvw0IfysVgxo4Pv8w5WzeIUdBxj6EQX8\\\\\\\/NYFjDKMaGCs\\\\\\\/vtMNeo5BKcPfC+9cXKc82YhXBzymqS4k64CJhVCISpXTdtQosEAhXO\\\\\\\/jBa7y\\\\\\\/WDaQyJsi+8n7TiRkQC8Wq0MyrQftijWOzZW2bjCyF7gb2vkhZnjeBWlcY+9DQ9JsWXXGM7RYa04axM2hasv+CUftci149XPHYl1HPPey7c2lkHn6Ac1C2FvADfLTgk5l3AVGpcYJnLikPNy\\\\\\\/iB53xRZJsRcLJDUjsN613xqMwR7RmG5x8J5xim3iXupj1NOFJW4oXDwo6g9UZdmiNAMNh6pQwwqAFZN\\\\\\\/lqdFG+nlhgKl8zv\\\\\\\/u126jD\\\\\\\/OuQ4KTtd5yzJYStse\\\\\\\/X3BUt769fjsTtXgthuDZCQQLe1yIqz7GFy2WawEteWlBuH2UPJS8ErI4zXR+fJIC+xLj9HMwDUM9dDT87jiuZJl7jC62cyKVtMeF6GCGdpzIk0TzIubUQqDKbVLnys1Axlo3OVpd0EV015YifGIyYoYTUWWt8OSPZqOrteYzaRg6rpvxtxhiPKbQfya3u5G+94cmlFSybuxbfLJYx0pXiYcOkEQLr1QpCm\\\\\\\/JOnpyxMki76fMhyNHvp39CzkP1PVKNLCCg1\\\\\\\/uxQCVcoejqj4v93WvZwCblCrurcnTxmdv9yRxP\\\\\\\/25h4RCZ2jntuiDVCtGwfUZ2OXAZkSoxmaBPZ8a62Ko4C7LJIz4UnkGB8h+hOnfJ3j9V2JKs3rIjyYNbYSTGFKHdJC57OCrUoZcFppKdiUEHquraMfOLkEq6Hw+gpW+9kXKR6B2uzoxMEin7Df0S4fC2XdSFiYNVlLlA+L2ykCGuNAeR5Gu1GWsdNeJ8XHGEw6G5T7LMwgVSEoPgJ6UyHBGqWQzpN9Gjf7rLFtOVGDcZPd\\\\\\\/Z94K7bs5N6ycyy28ntX49Md7MaMeQ5RCeEB2O28ObryRVqr5AoGNzUHZZqVraA9DQAeWX75EDyvyBs0JPlaRoRQEZ09dyhATsnzck5i6Yh+29GCGLuGXVF\\\\\\\/BSzbkYtPjS2X5huR3YAmZpPpAjuP1axB5ws3IgqJmsW+FAbJVFzZ1DW41zkeVkdCF+L6ITCkcdzVGRKzGLJyvT0bXMJ63\\\\\\\/LluehHyrSQfbAUuYDsuoQI6xpngHk8MogMjtUf373C38Ous6F76wMBwlVpYfzh1Sn6sHanx2BrrAK3F9YTY+sWCJJftP8+8IbzIErFZ19AX\\\\\\\/azd1uTeoO0Tm94pxpHLy2RXrT4Fa+tm0J0Tz6\\\\\\\/c68zK6YsTW8xSiKiP2igaOom94vCCLYvL7CywzqtqNO9MwA3RHrIpE5EZxHZAYqKQlkv\\\\\\\/k1W8vFH\\\\\\\/aTrQ+pd+lMqa3H+rmO0FGYk1bhG5d9qN7+Kq9nhbUvFGY8x0+ZMOWFxD3r1IDzndSCFTMs2pqm7HQuUFsEBaD9xj+ynU5Yn\\\\\\\/PtsOEzmzMOe6+PZQUcwee1dSZjiFFMlgQKRoIVh2dHLMS1+aAIH4U3oEIRQOuOMAoPaZbmgMgvsHCvkTt3d98LkCu0c+yegri\\\\\\\/zW\\\\\\\/+HfTXKyfhyOnZ67G1Xp50u69jUm8E28rPXUoj8\\\\\\\/AImhljo9IPNT2Rb4YpNMSDixz0L6OiODJLAlUoHwTn70QL07CHfse85Uqtb0eBFS+8Tu6tsyvN0Eg\\\\\\\/dA+mUVRrEF4QWFwGTpJz9942gloZ\\\\\\\/EdbM5U0IWi0uKABkZXKIZc+zQLz1E9+a\\\\\\\/MCzeq93B52Y2mjW\\\\\\\/RERLQHEfkKRWrDKN1lh8o1D5Gv0JEBAJL7ChfEpLaLYvrcBjwe1UnltlvEWcmQcs\\\\\\\/yKpXpoHFt0Dp0v2bC\\\\\\\/iyA\\\\\\\/RHcyDRvUuNPDITZPOjXfuLWY88WGcYpWqUmEM1DsnWIUMZPEily30dIKoaCG6F2yDRsoj6UL5nBs6Yxz+3kzjZt50uq7FS7P0Fh8cBe8rheDnQRvAaCMR7Wkhow209JMRq4ghtDXocWlmVx5+s4h2Z4\\\\\\\/GsaVNJtZQPJhySB4GCPgVTYDoEjSTkkvG6MEbSUprbjQ\\\\\\\/1zqIq8kwt\\\\\\\/f6NKIcBiO8ow3of96Q2wLLhiQOGNnQuubGzGKqkRH3CMApRteIr7NW8HdIl8mzd5F4PIzF9NqdanRmGb0hANneQ1iDHk7RD6I5Q9tdVpiI3MntxD6K4yWrZlXi2va4zGwJTS42xRPvgGWnDMxMBgR3ccyld8Dvs1ae5ryL\\\\\\\/mkM944Rx9QKmqClMDPeCxe6um+HSujfVS5safUpqXfNYi\\\\\\\/aWnTVBkhg9nNHsp\\\\\\\/uV8hBJqqg\\\\\\\/ytssYl5h1YqXi7SHKNjH8BaPjSJZK2sDCALpjDVdAJpnqLWe9k5JybUR5J50awkJCeNL\\\\\\\/KaFnvhmm1cbNC9lNO03m2WvhlE\\\\\\\/Fhv6CCnc51UIqEq2zUXI4TzepKx0xQeDcr0ezTP1qhWR7zhd4OiePqKWjTjsZeOpyiF1dxYqDVi9pBqW+KaVpVeso2MK3kvnECIoGndg1UW4jwM7y3pVSfPq\\\\\\\/8boZ4KwNCR1Wlon6UB5iucg3U8owYgqHz\\\\\\\/DP2TTn7t9AdJDDlFJFpf5cVNo1NFxQZB4+Vm988YVGg+Ei1o5RtShDM1CTe53HIZh7syoah3uystLIaOascvpwcsyndD5x5Z7FVTAQosshE1lLfCsrktJZq5snKY6XLid5\\\\\\\/zsk6y5SqzoevOYlyERWa6XyS9QWBLnFqI0zuLq+e1CE+8dH1OLPm5D2NCPnIknFg3Jcd5OmdC3FLZRHPjm0nyzNW5SFyRlDyo+B1AQaQ7ZAlgRiW61pL68+MIzKZRpX0TSqiMe2+hNNy5M6+wxI+LRQBb4F13vBgHTN7m+GvV1Uld0L9F4Kcl0EcMPO7bYPCBCHI6Uq5qc8O\\\\\\\/4xZfZ4J5byjRRjdlCgNyRiUFktOGqBesSM4oQxvP3AREe3H6rKkmjroD1pIK\\\\\\\/OEN3X8MnPaVpw82LzJqOgRYpfwOV9lG3KpyrkLQl3TUQsxIcbVrvR3hqNjz39iC1QbXH8Gwd+lujcJ1DnZwEDRlwT7pnUyFmAy6tZiEV\\\\\\\/1c30uipMD7n4LmNNU38DeWG2A5eg39NgLT2H0UzDPAgo9EQo1XPvaa2QdJo3efw1fct9MEOXzak76ru0QmnxbM0CIZ3UfbG3kVbmd6ovxe4oo8u9At3jD7FqCESRdR+QHBP9R+eGFx\\\\\\\/ArrFJgaN7fnWdBxzFlTCX8DDz1ebb61u6CEVz4VcoPpJjrVLXGm1OaDECemtWSSIzbWCWTpvL\\\\\\\/lp50aybcVhZuX1IMeHeXM6R5+Jit3ETHc4\\\\\\\/8OQLRLr46lYZ1A\\\\\\\/nLflO+8hcVPMcAmgjqjvPueSzc9xZXQC3ZwrcKbIvOzwVCVaPTtp20svEMzRV1XQwk7gjXVLURnatlII9SDeP+zwtEtMMQwyIxmkbRUIXEOt1maJ07FiC8vKoRn4engQawWK9gmX44n5rN8qmWR807KcTF2ore76VxWsHq6lMHAbkZLMVR00w6Ntjhug8FRtWBSbEc8e4c3zsHpqPT3clQsvS6Sqes+SNw0afe0T5xJfe9qwIcqH81HAHq2RwFQLsGxr7TAkUAtT1\\\\\\\/oeKynfNOpIvxBsg+88nTM1yRzmhvTJXxiqYbtEmIKOuzuKA+2bcTDi7R4GU0QXzPS4hvDvDQgLxiA4+DDOiMNTwNzKJ4lNNnKITf\\\\\\\/CFzn+xjAHifswXupPPkRYnYNoP2bxUe2eveMM5olMGgeijLRXVOKCa5KkL\\\\\\\/YPkcLY2eZFCyvOh6B2mCBnyHwtq1YQsYhWsr8hLY7IEl2\\\\\\\/wCi32DGxFSzIRXEdlHgxbGMCvlLMMInadbblL2UIXNEGnn61\\\\\\\/7YqRA3RaR076El+ele9yopMVze80Q1zk+suI0q1YsOjuRwUDKIyr5fiuosATvEwsiSe7OLYvtg9rlY+opdW23qfrzmOcYOpSvWqUYBt8RmYxbNcIynN0EJOg6lwV4k4cH9qR\\\\\\\/gQ7OdRygr80TfvyCCUmR4288k1o8TRdDQSzG0lzyf8xFEE6T9Si1iKn7IIpOeCh62TJpplVyQOiX82u\\\\\\\/UOAt7aIOafq2SM5zQTIsa\\\\\\\/hu18UIoYFmgiOkvcfVy3gfcsb1V0rmZgNpr4F9NPCjaaBNCksIx3LUuOvnRK13V0X4lPKDi88yP19XJhQWjFqGvXmokwvxEdZbBRauKGxPX1uqaaA23UtRubtb68r3T6ABauvDFhRpUNjNtFpEbbarJN8Yh28ej+shNHkJiDQDe5ws3TRWQJcSU+55MxW\\\\\\\/EOqrBbFgpiOh70imdaVbog2Y\\\\\\\/iefEOXBt3430YTRtiDZaO2lacqi34CfBtSbADO3anfS\\\\\\\/2gXFbUZkxs3t168O8OKY0IKUqIhxa49LMx1\\\\\\\/qAFz7Tg1pjWD4RZSGdLkij0AVOimd0FjKp8XjXV5SZrgGN2jHushUww8a+lJL982jTofzUh9\\\\\\\/0unyVf\\\\\\\/2vXdpsc1zPePJz7CE7TgFtR26FwcKoyCX0TtOjAGLPJL4qNwIv+ovWvaG6TX9j3SSN9OqbpspU2eFDoa4ICbjBoakIJAyll91eL+LQcA\\\\\\\/nqEvpjtG1hhMZqu5p5MvEIIOHr4iSwKD9n+K3FuBvBC+t9yOyP+x3b2cCkjo7zl4ChKdHZT0uvBCW24EmBuEMGPC7hipy\\\\\\\/ezvxx\\\\\\\/L6xB1s4dNasqsxzR9xUp2DRkG4ykiYBBLYz9uEOhsmyZ+Hy+cYOipIAtffdCJThgaE1MXbTAQi2WPKWMS40ziT1hNyXRTHxHTqlNB1LMvrpEXprShShjpIrDITYqpnpw9P5LmcNZ+K\\\\\\\/hNANY+MFhOfO4C6w1k4uOg1fdvExhfgTMJ2DGfrjJ84qg4ZcsajJnnOKxhVRaz1pY90Ppdg08bl9Gk36wGoygcsdcvlHnzfFdZ96qAs5P9oDW\\\\\\\/8BhLNJyuUUJ9YMjH9yaCA9E1KihUyMsJqJSLWo60FvSuWK3mHqHcNrsHpaqfD4pDZwYHdgeq0RyFZvMMhnd2qOcKPSQeIFrNGaSn8Mrghr2fsSIvny7Lc0Q2R6nlDOeWsC7x0HohEBbZPsXcmO5QM4vfu7RB+U+5ROpE6MXawHctROtkHDY+yh0RcTuLPYkBCY4I0P9P2QJ45r6Y33HIz6GxCOSxb1gaOL7abvoYycfF1sj+9dXGXJFBObSpkCz4bgEictuEPmEAyNb3+5xny7lSGIDpq+d9AWMhNbeqACm8XCWLKUsCXrxpEwCLL54eNZ9kugBHRXFMsar8LnciImMLnklfKO0yBESLa+tfdBI+UOB5Vff2M10k5kJCqR4p1TS5tfZVnmyKSBoDq0IUeO5vOUcOHnf0vKzXZGDAhO+A5xodayKkUe\\\\\\\/J1JGRNFS9gaUFrO+gY++EAv3\\\\\\\/Yp8GHvWemvn70hLCnxoa28S8BxbKFRLL2ncKFK+a0EA4cg6VZa1vS56Mi1ixhulcwlhqhE6iDEVWaJhP0AMLEZmCsP1WtuCkPhX\\\\\\\/bebR18HOXAgzG4O1YJH8lR3kj1Wygiqo2fxBpP1NOikLrsE40Z\\\\\\\/z\\\\\\\/1ZFZjfjffBErrq7c8wBhPckqIxm\\\\\\\/MK\\\\\\\/nBfUnuGUO8cROTY3kuKlPrxU1\\\\\\\/y9VYVOgPBbKXtmak6o0U\\\\\\\/ycJtjV+gW86Xebt782iW0zOvEzT6Bt7CPq6lA+rzab46ojzU6z68mH3yFsI4ONIi+qogjjYBS6RdUKXgBusCHh1jqLt2jxJICJwszjYHEG0iYPFnW0ez9byz8zYfFzLLi0Dc3jWSnJbPqwjDNMOyJ64Y6vkqjoOrJJRNCzaY384aULJA\\\\\\\/x9+nY7dAAhvhOxJynJ7L6OJ0tugF+2GnhbypW\\\\\\\/E\\\\\\\/h74AXHHjUe6oUhCb3nCBeWBZNdabD1r8G7NGgUSUR5tfmX1OsUGxhqRgGTiAGFNbv5p2DK3h0TLMwP4ucZO1FDDWWMtcXNfkWtxL8xx0KoU62cAYwryTYHveYR+4Ql4ZLFpyhfjdKzHEo5fvRZqOvr2jaV+Z\\\\\\\/6mIP5OJD9T9oqcMo2mJ0nxgCoO9LsFQdrYkUpjv\\\\\\\/x7oavQRv818nReft5huIa6k7ZVkGzfwvphPMcdBwYYQYxJzAKXlu12DL5\\\\\\\/tAiMKNmZOfs1wI5c4XPWbDDCcT2UCzupbkrYYSdCUcll72aKTHtUZlNUEpXSMrsy0mfEjrdqJap09WVVY6Cy0NyYeKOBZQir3RUHTmGw1L3uukwMjPFsTWkKMce9KVSmw973iR6mMS6OfXIu8Sv2T9ueAOvdZawtI+MX+NVcruSliwhiYBAXhaA2YXH8RXDuQ87WHG0TymVulLle1gy5I1FMd1RI2ynPAwwzB2B8RBhlp6ezejG1me6jBTwjTiNpjTl9Iia+AXjZbGe13Zeu\\\\\\\/wGYY3EGdolt26QotK363\\\\\\\/zyGDyGIyh0ZWMgMXIeubHm4lp3vTwHcirOjL1DhkNWl8QVlwgYxWb1Ep\\\\\\\/rT8xFlDxBgoyGOCWibvvbHgj64UGVCMEfOXYlfQViPGV9WxMg5RF6xzdvb\\\\\\\/2lQvLHKtaG+ufUJvsK5jj+sl7yE2RFXL4aC3D2CnlIg8GIs9XaAiPmGSZHMv4HUr3pszjRmpyP2DYYEQvrZ455wPH\\\\\\\/TgxloBZoXBO5aSe8ERJIM+Mni5IwHDqi5LzwAk9nWL5NGCkUkLpDMB3PDnVfgaEr4KLpygoKELSK917TI1n\\\\\\\/OVOdtG68IO+\\\\\\\/FuQbo1N+2yMTAFNFiKQmvolyVf4JJtm4PWseQDkYLFx+xVEgKjaOhkdbL1920fu7V4tOm2YPXdP\\\\\\\/Eek4cHu6Zh6XpcBAGyrWGxrITgH2vvlg6to5LHSkRg4nzZYXow6jRnacrDCGcD6L6hqoW+dOBQjWZMOH3Ti1ShRSFmDE4e+J\\\\\\\/lke+tUQ2ahtPaIk9IJARxVY7mKipDeo7Vj4Bp6OEeLYa0jraYcJhCIEvHQphDd1qmFUJCZ5WlOIq7dnkpOyGIrrKOebd7jGL5PV9hEOpD3j6sQ8WEVdsFpF0Ro\\\\\\\/RSSigGUSokK1xekILe3FrzdFf85LL9Zivjq5etugQQcK0ZlU8ZJdMT+W1o9XO4u0TzfZ4mtAMo5UqH+lAd1ITJx5mZS+PEBmbZni9gak2caQVgoEae2cFXF6\\\\\\\/QtXRYRgjBLcozREsRa898TZml\\\\\\\/ZMhrkbqDK0ExMReMVyCTVEmEFGLgZnuT3kcf+tGfkPz4Xf8hXPdRDEn0Y7szw8o5VDvyFroajvZqdR9whamQep1D9PpNxBPJayIE1kt7xEhl+1dW61A7EfXJNo4sJiSqhtoAru62X4e73Lc8FPs93fWIi4sx6dI7rgxonqo1UuounK0I8wyeTUOhimI2otUlVafb7lDfH3dV48KT4pBch6QAQeYWLgQIOSHvX7QfLIgzXQXS7cAtojBE6ATxRAQdg2tsw73lQN8+1lbRuaIB3BUZsplVtJph\\\\\\\/+3T01F3AM1JLY4KMMvi484YEeNSUXeOq+3ginDFM1W6XbO4DbrnZwkm\\\\\\\/ABRCNprMwwrzMoK92NYs57ahAOE2bKIS8ApaM1Yfwupmx7kWV1V6HTIiPKZ7ocrjKdXQydQeAHKX0U6aza3d\\\\\\\/8QcIuewA\\\\\\\/j+SDEfXuo4omE5WYjJJD1oqHDwB+\\\\\\\/mEjwAL2mkxyVsm3YyBvljQgHjb1AJ8fe8O7PuihlbrD7Z\\\\\\\/TNmDLtlF2uJB7g8JXTKKRrUJmzrVN4BozcjUtGeobNONSNQuVCBg4P9kk3ewwuAK5qZtoUlUVUMbNKXX9EZXbXcHH8b4VkFCq0C0HWMR8OFoDlmK6m8wiyl1a6t9Fx9zS0xjdh5Ar8+SPuP44YZomVyYUsC4r38jfIpUFNJiFKScnRHBD\\\\\\\/zAK7RnXjefTTwTRGW2TTUk2mFUu5vH1jG0IUrXkRj1VafI\\\\\\\/gkqNiNmyu+txStqoVwiv0NO88zdaAex2CABEvvMb3RKmwr+48UagcD8BhPXtQdZJO8jFC\\\\\\\/3GZ+r5nfCydrkzKAwT3c81xaSfEuopUsI+\\\\\\\/TfJlnQozv2cfvgfpM51dh8yH4T8SEjbpcmC+yxnc2zlUW64YPtSwwykKaOze0U+WO+VG0A23rQn9hLDcHHJZLad\\\\\\\/f+qTgwlVNeUlXtqJo15e+ZFcgmJsyUZuEPRJSYVjaUjHVkAZXbsiuhmGsAPLhJb0Euj\\\\\\\/f3lpTrFFrheVsHfpWaBvwPTuvcu42ft6uIf5HtKiUvOfBwJNQU+bA2NantTDy1YUnkbbyu28g7wOxH0kbaH8g6RiLmLcK4Lsr0KKZTL\\\\\\\/SOTCLytJmK\\\\\\\/32Z4\\\\\\\/qEPHkpL\\\\\\\/nPGFbCqzkTumnYktzVJTdN6C\\\\\\\/fjYs2Rqd3eovVFpLwH8oVtFNnsSWd9RBqcfuMA+tcu3uu9RUxOgUJVViKgHARGmoCt5cgm8ALZTCyaoL9jquFFCGWH9rD6o+gQmB1VcThs4ZPuWFutXVesEZA8LEoNB\\\\\\\/kOuOygkauWkjNuzQElv\\\\\\\/e2V2R32GuBDfBCy2g52rFMIU4Gw4NiEa\\\\\\\/2kTHoSz+4TL03Vw1ExFLWowzJ9WeoMehP9qN4EFjan1P1bsGUQ6TqDXjuyzDw1BK9KPcx5GHel5J6GBPYT1Z8t3uWPQFmJKheaBC\\\\\\\/Dze62FF0Y6p2\\\\\\\/1aDFJeEVKQJlSN8N8X9bbrxeG0Zj4mJg5IIQKsvd05ZxpEJ9M3yx1xMoAL7wlVQwLKNPmqPglVzP8rP+OYSWV4dODahpkLYF5+oprUa9vcKRciQUqMjsFKgGiYYm6SCzItBcXgmPbBDgr0LesOXKfe\\\\\\\/XjDw6mmKn2b4CstdWMaUchCd4H6\\\\\\\/FuKzTao+imzaqg83V42Tb+ebhd5r9m57MB36FygvK4hegPGzQQkRwGIE2m0GGMOjvUbxxl2IDTmhUEWznQdrG+qSKTgj6eQBebwI1KOXnaX5qs0sAyCgcEi+vOCARmgDhXdq6OPOC5x7szOIAEvScJSOqi\\\\\\\/kpbqHxeSB0nStBD0h5JopVcxou0d69YDoqU70J9wXi312KCBM0vCnIRdj6Zpm5zr8HoSmjxpt2havh24IdUPbixFrbWliGVU\\\\\\\/l2MBvP9k3y34gxVQ9BwPZkHADEbb2E3W3CqOBZoLXQ3jY6sDNrE+VAfT0ipf8Ec2FF5UfNUJdDPaX74RaFkipw6zTIGitVIL0rj5nOxH\\\\\\\/yxmgB2KzzWszU7mzC3FOC\\\\\\\/N8Fd4qijKGWhwi6wl72ayHklPJJlzMMOb5EC69zAWGg7QgMKYWzAh3zn9IBaqcazmMxcYPafXn5Ik8U0AQE0rcmHBNHqhNunncRugP6pNS2lhqEIzDGCjW3TuZvc+Wzh52ZX3rZYsAQe+6L2ip5Z\\\\\\\/13c\\\\\\\/uoFTmPAYt+qC7Hoc3Md6\\\\\\\/LnCuP892zB41720ycSMwrr+dt6uZC9iSEsbF5XHdRNVaSSu0wtbmMQ+gq9BLXas\\\\\\\/lHLKtdQMo0qnGmQeF+3ToNYZ79i3zXQ7zrza7Lu5oMPTV79wYItdSZABhh1MWFUvLicTZSIJm5zRAsnwgS3fBU9B4T6yM5tJ5Z6bNzactxpy3gzqk9Z\\\\\\\/ZKgItqh7Xo9CcWNE6PXFWSCvM1Q+L1zWOpnRFeEYoKqoytqW5hmAeEUTwthDVOKbJrzcFR4NOlJaBw51yrb5\\\\\\\/4MYhJ\\\\\\\/A5KugopZgcBJabxkOp4m56hpyzCsSbdHkGuQOF5eTW4RZ4PmqHzFh+Zx+ji3PhDXd07ccvRojz3oxePRat0vtGjialEUxUMdwpjMLYiINmUAbHCDlGMHZoNgnR2sEcpmPZv4HIRglp4Pxj\\\\\\\/WIvFZz8DCZW5CQQj8UteN6ss\\\\\\\/Rg4IACo++E2bsQeQokn5zpykLVfDJmRAuPSYCwkCGIRu0ITpaidngTzh7WRsIbkmeYxwU2J4o\\\\\\\/dATKub36eODLGJ3bprIx6PoP+MiRhEIZqUUKHyTSLv9yrc7Zd9MKxHChd5MpnJrcCQ0YnMTARImESqbVt4EqfN67vPcKtXE2WMD0WEuNtkYqe2o3GSr5lvySUFVyMCSDmmYB9QLiex19\\\\\\\/CJJHMFWkZRZnixdVNAsfvYE4us4XyimC3Lv7iYyNYtN9aZF9cU8UTf6gRBVYvQaGS5W+i483oyCF2kJWYwqHefi04zjKeUC9zSwbJ+9ohJ22huimHCf2hIY6qgeKO7Cyed1zlZUAkVfBttoQGkduydN3l7q\\\\\\\/jcc6vBzt7urW1nX5r\\\\\\\/S6W0dGtVO9xZRJVTR0k5hqNiDsQiQTvDjLa0iDQJys2tuGzO0Kdfj+3fLzZiu1yvoyxZXVLDhSv5XlI5FJnUvq9fXjWUgbqFWdmZ5wWyZKn0ha2sN+lbU29UjCS6H68SRkxbdquNGCArMb+dw7SUJZ0S6kv8PbFUIEA+fs\\\\\\\/eXvlqLGof9NP8pQyVgTPdgXDzImG9YJbnwlqRgqsGFz8lCO0Gs5bDc5A\\\\\\\/0zyD3FeGR932ePr34GP9Fae7TullmKF5TFI6R3o8\\\\\\\/vz848J+8PDkX4KHmcffeIc+D\\\\\\\/pO5QWW\\\\\\\/oaMbDkVIbyGE0QTWtO8wsKC58bo2Cdvkk2SZ17yE04+NrM8XBWdfrSsJ2hjIEI+fsItpgeG+WMI44rf6TEnn59Lvq2F3duQYLFGdf\\\\\\\/YgDYvIiudoFZ70VoqBVmUKB6jeXChb84PZnvKKEPaKlVFRjuFoyVmlHjmzudkl2iFCtPyixhlZg5A6q0SziI+N49R\\\\\\\/0gW+EylOTCtmZbpkP9I7CYt7wMtGXkEqI\\\\\\\/vxlm3WfUQL4TIi9oDtaupEHvrkzpx4PpQ96W07DRKAhThgjfI19AX5l0rBsqCbJNZDz3GidJuNgFvlgc3vCTo6GUi9amhHaQgeYDP24VseT\\\\\\\/oJxtl7loUr0R3DUadNLI4JKNEuq6xJCdB1tGIjnN3OCfSq++ePBaV5t7GUdo39R93oFY0c59bwScTLmUiXsvRIhTY38BVJARFBlP6zbTUYfgiMpcCM0vSd6bbqFqVc4ifpKJ7exSL4kxSCWR54fU2k2ofuXIePHG3t3oIBuPJwdU\\\\\\\/Sm08R4h0i3y0Jm501aaDeggkGZcwpCZcsTQsA8QyKS4OP+C6\\\\\\\/ZXlUFB0mEaI1vrkC8CZOY36u40mxQa\\\\\\\/GnHnhx6whbU+1Uw1jEu3OHEvRWdw04sWmKaBf+G523V+BvKWihbb6Ur\\\\\\\/0pSj98WE9p+DuQTj9AzlzpXZHRyUVjrLaGVEEh1wx6O4eoaA53238rS\\\\\\\/rYV8iXk0zpRX2sRhx1uf5Z9abclfzZbNMgrVeTLfsn3whQYrfwgt5zvWWT0oSqihtiJHqmy4KD0MRuWi6PFMdoZBJ68oXNnlTIsUKxhPoCLoPo6Csk9m0N9SotaTaDqEUXPI2rsfE1UfAZ\\\\\\\/OfPZszfNlLT1S0e5pRY2O9Wpcq6mwUXru67rLs30Sr5IR6FqCBft+OIdD4NP1T9YoGdbcLl8mdAwoeO02gnAelaAnfxTNi5D6aZ5jPRb4PrBI0oDf7eRVoR\\\\\\\/yQi\\\\\\\/wOcYz4qTw6kU5xYSqy32hNJVfti1VK9RkWEnHizl+3MO6m7ebr7ZPP\\\\\\\/qdcnuvJxeiMfcUqJAhiZykg9Rg7XJt95ZtIk0Atd6\\\\\\\/uC9mF\\\\\\\/Pw6cNjRSh55eaXkllcx4ITr0ShjWZDr9O1GF6f2LeN61\\\\\\\/kkxe6P\\\\\\\/MQrUxh\\\\\\\/pIgA015tS46+NhWj17Jb4dFY50fOM+InnWsT9QLv4HCeX1OARyUOy0zfNSIPm2jBQO3agK1A7KIjX9pOoNn9ObIWWYi3wSliO9qwzA8wdGgUQ\\\\\\\/6ItW3Fp0dkKMXZmOdxJfrM7OGayXA0fOrb5a6+OwRaYzj\\\\\\\/rleivX713yuoesj82wJEPqbKwZzBJVZyieug5PfJKFL0uv4nvdpbWngGcUT\\\\\\\/UY+tqui8ITr8uDEt9eeziiv6qy\\\\\\\/fvPolLuYM\\\\\\\/Kh2lhOX2IGYL\\\\\\\/Sb8yxDN1Y38cjBF7rcpDcg9gfN9\\\\\\\/3v2ADwHlYNEf8wGBoDG6m15i0fUaTRF7jMYexm8Snw7A2YqtiT+WF2VWyZFlAFRIo60pVFB6yrqjeKGnsoV2jmnn9Y+UUwKJSeE6s2W3HAh0sWXUOhTmjn7scuDvsnno4AJ3Z2XYxg5Tjj5+EgEkiNz\\\\\\\/\\\\\\\/JE7ohGd03Qmnq0RSeWUUOc30xu+tyCAIHPioxNDn1KvaS29a870WBSMagRY5HN462D7IM8i7illeAAam2EskHsGe93yyZG+eIuyUA62CasdpMMapxslzVcJwFqur3Bn1jJemAqXf4MTtxcW6WCODurFvsoWR0nWA\\\\\\\/niHQUKsl9Qes3dEISfThQN5kCUJHPbGP0BwcisJYMznYs+KD0QsuXWTaUBYVDzHuiDHN3rGo7MTOeLAIbNJ4SQwpyW3B5bPbdp\\\\\\\/99H4gVXXM\\\\\\\/IgAk4ufnSN8H0GFtRmkFZqA2LQCzmMer54cC0Ti\\\\\\\/H9D1ofrmkEMcDkBDRQk21EIQRsTDtTJQq6CnAM8MAysW9TYTHFhWckigLDcdl9\\\\\\\/EDyOFiXuNCaY4nmaggqIQzQR1N1tjyiRubKeITuW7fhsRKYEmibNthDSzJYKEFMslre1RNKsBiinA8SkV55qva264jOx\\\\\\\/sPTSqRMONIB8OdiABnQzb7Tf+XDXBS3yCWX2IxEj7sJBHB+5ALYgO2vB2yW435Ep3b+shwQ+w+4hre01t5xR8r7Gvd4oLsJpqntuUZWA0jSniNX7Sl+VBQ2elP7\\\\\\\/56IrDC\\\\\\\/QSmTotHcggO0ZAkA0CkxRO\\\\\\\/UMGeWwLS9BaA2S4eiWBd3Eym96NMmUYZXe6EIJt1S5HIG0hAu7VwgxylhrQ90vcbwP6oXryr8quJ8QsR6Vz30flSmqtKtHvC2wBT2wfY6f6uVC5FyJBYtvS2L780uDKh1zh6DWBtTf5b6iUsRxYzqXmF6+iSIR8JsTvFtCLlsKOrPsQuInaizeeAHMMQbPhucE69H3t9SS3XMiw6BHgdWXaGnck5al7BjAmv\\\\\\\/e8sGmGuwDEpL1XdKmCnLUQgKk0LenjsG9DP\\\\\\\/bqogcbv9WKQKzGoT5ADNarAt6oUHZOQJO6vVratPRqQj6bgA3pnzaoVIqq5BvvNLfVfwRWZjJXYQF455Zx4gdwO0IxcdC8bgRSIqvSF8UN\\\\\\\/I8m40sWw+VsNasgMc31PwZcqTf11D33kIFq2\\\\\\\/FkUQk84VDvlaW3ACoVtFh6Tf+277l+eJN\\\\\\\/rc7sqSX6wV0Q3gtT5ymNi\\\\\\\/Eq3mVK1aSXP25lmCX348MDEYtMMtGIlzSonnpA0zi4wnh0Zp4NRwwLNsI9\\\\\\\/Kvz2tCfc3Pe5yAYJMdJz+pnrv0qftobFlOU3CyYbBPoCrpRJIyWyNUCKk9z7CoPRS356f\\\\\\\/4yD8FPC4IKk+5ZD8NyxntONMhtSKwyMDd4ZNz3LHWIoZLdplnlVudYYLzPlx8K6SzVte4mFhkX2lBb+qN4g7EV3nX3KSGHjMAAoQ4jH6Gmk52D9S+f7CASVCSk+6ay\\\\\\\/nKfa\\\\\\\/9sFIbVj41gIWpqbBfkor1gAmX8AURpd8w+ZpAzqQdSu2jkQqLptRAMiXjcVooDt+d4J+Cg0fulI7LPjC0f8fy4dZLmXuplx08NKA4kiwihrSbDYPt9kpCiO0SMbPnykcKjDuzDv28YFR\\\\\\\/JmqbCgUb6rn0xrne0VPAuEC8F1qNzHmBGMGN2LH8IVt+dGcOtcBIG\\\\\\\/KBe64SJjjyVdWyjxhs6rugkPUonaaBgU9MoXPNUCld5J9Dhe0GfnAdm+NMJo93reL5Ae\\\\\\\/6oB52gHkNgeMFnYL8w46Qou3ROeYdirGAaJSPAhs7fAd9KEQoLa+yry\\\\\\\/2rnq1FHoJZVnrMFAheahTgYswO3nO36TTybgy+1SaFVb3ZtueSskDFn9Xo3Z7dSOt6BFmIhz3gVPb\\\\\\\/OcAM77Tt8MHLVf6EUpBP3DFv5eIXvd4dSAkLKm0ctm4tb5OV741inJa9yDkCrrxM8RCqOkEWciB4tSdG7SZK8nS5bYopOofBfCtS+rQNXkBa13Ckd0eoSRUBKPcCZH21J5FQ+p9pCh1kWYTp8q+O4n9U4jK\\\\\\\/VH5tdghjuMINkGBmm4W3Xf+2jKZhXIifo1fmYR110Vjp+dYJGIhES8YQhg8OgUtjcyPXyW8CHwQUVW91OFigCk4FF6r1BFLP4j+A+HAvf9qnyOb541wkDMyMPguGCsHi\\\\\\\/cwdSEll\\\\\\\/117M3x2lfLOq\\\\\\\/Py0THSogYfo4oeQdt8eWqHgD\\\\\\\/VfKHCZVHQS4W5PQdPnhRHOXlSUoDpVgoE\\\\\\\/EVwuAPJNJzqETr0xzLPEK8nFnB4I02cMC9zY9x\\\\\\\/GuF9EKzIkqXZfRDDXpH26ve\\\\\\\/XdjEaA9Mt8GKYkUp+f4JYQDPrtguZPy+EXJtkp8kJI57YbwuRwG08mhBqDbbvX7hy2mAz66PV99O32q1628Ksbh5rQq+B+lPWTHsE9sWR\\\\\\\/B+Xa4EPHBpRVS0LoGGl2LkUM5dGPsSdWusBqRGsugrVPy9CdubAhTtXuHFXTZI6xrwNQf7Gv0ORw3763OFvKo7FMAphEzjgcbSF\\\\\\\/KPU0qxA4WluaGVC1iPKOnivRdRIJrqqjxH6zlRMETEcAsYfKg6lq4LfJG\\\\\\\/7VEgJ3I4111VUdCBHdJqZk\\\\\\\/TT7WWHEOLPaFwip8Vls\\\\\\\/OWUvF5B7ww7PVYKwZ1GM6MXhMeaNaREMereD+0F3il8qRh5hujZK4nT+nOAXeRu4FB7X1bL3JCECnSSGHrmKSpKVcHHglRNUM4mZQX5guV2l+Mh3rIu\\\\\\\/lpctWOVX+8BzzSwZYvE20Llm7PMEmf5+D9XEuFhoIdDLNew80Yqz2wO5OkF7akJz+VvTKWSnD6X\\\\\\\/\\\\\\\/p0OdXeZjNsY7txtl4kuyiw8wW91mqpHcgowXEqzEgPF4p4rlTQymbxVJsFS1M4tnJfmjYO4qqZTYbR\\\\\\\/lHvb6MWHucNoZhm59twlO6R9JSBnt1nREj\\\\\\\/PwabPjmZWCUwhVhYfM1Q4gbFiIrptJ\\\\\\\/Tix9CdYBGHhxOmb+rRxEuK+LoOgmvmo8lApOBaF1wAHD6nwgjjA0PG910NeFfVwbqy621XXwlVomcifjob4bZVCMk8Ows++SLA0Vl8cChwYecwIFBvb5k1K6+4N+QhJ\\\\\\\/D5WDStQY+oSCqjXoX\\\\\\\/m6IsHjkmAAQb5G0Q\\\\\\\/oUzjA7\\\\\\\/FqSe\\\\\\\/YVCmKO0lkLIoR5KDYpYe3y7iYXfXoJislU2VCRqWA\\\\\\\/YFXLd7wXbYfYrpbrcdQnOaYXZaVKqtB22ctWogN78MXAuKKgLntTvBIr3zVxuuh1C3M5Iji8lLDETnmvROpasDoWhrwsXeQHerrg\\\\\\\/S4h9OS4CbSh9LYpqEMaB7r82lrWMrsX7ZFv3RJoiSq2zMrWdr5KTsr7nhHkMxqHBzx0iBvkBOLmCRgR1+zjiOKpdFDKpc2lcC2d2Kf\\\\\\\/1psQOvPsUig2GLIY8DqvdPpdsn3UhlLjakaRoJMJqDSSfQIHRr5cuRGxIiN5uKuKyQT\\\\\\\/aHX2Nwi9z4pYcYdGmgGhAtzUVwJRMHMoSE5t7TBi6xt2vgcKJ8Xx63qswRewbsHfqvpmHM2c8Bh5T+VG5srh+09iZpKoO+yD74aYP3tRmJHDBinUsXyzO\\\\\\\/YrENbzVasLmDUWzuZsfskvNUriOkAAuNMBXxesDpKQ04ZDcLtjQOUPZDFWE4bgbIR+yIOmNHbDudgcnSyRUgq\\\\\\\/zs7+hy\\\\\\\/7lzyvST+vt0128FCx\\\\\\\/Oh7kod8xp7Fyhrcf3dnpd\\\\\\\/+pY9ZaB5bGY8QXk3wF7sNaxAJDqdofh743Cm6NUiJxANRQUoy6O\\\\\\\/TNBbrK0DO636qQ2rOKw6+3AaeptLp1Qe2aZx8UK3YI7RC32xVqRbCsobtZyOQ8MbbCuzxvlmk1l5se0XaxqF1Db2F084ds5LTgrbAoa78m2DwPN8637QFkfHwRNmFKcocm2MP2FqviSWPvP2x++ivlwtZ5gDeKqsGrPx4d5yRfXTdQr3JXdv0Qj72TD9TKAZ7TKrT\\\\\\\/sQk6VnpJ6D\\\\\\\/NHAvr\\\\\\\/pVjGqYpkg3GQYJ4TzwY42ZloxZ\\\\\\\/q0Ycn6kVHWgrmUOCIk\\\\\\\/vi\\\\\\\/pMlG2wBAcRSL9Egh3szdGIA7kZwzLMwB61ycnik7QupF7EYv+YLjoY9ZjydC83jAur2EpZ+veimp4\\\\\\\/MYcTuMaWzaLJV492FqXEN0TFBUjXJKWYr0a0digLEMvxl95K50jFHQ9P2DWkVWJ7dYfiqpDlOANabbws3n8jhaNBeoc8WF1dU7WmTBVSqy35BYTDIB1S\\\\\\\/\\\\\\\/Fy8rOILuuGNzIr5FEsNYBHhdwUmvfH0TkSmjvFoQwXHCKpdHrRYkQl2aHWl8PFTmBmCP67dQgHvdoGlOkzd+eBpCCL3LvFPdaZImuvr\\\\\\\/utl8t3xE9sKTO2pWrdZbreQSLyLiEir1w99Wf6BjFL3PQP6eX8SANsAMylYo7M4qDDPIf2f8DA1dGV4wR8nVSPEPS6IBLxGBNYyRck1W6ko9AJTRM+srtvLSLlGqwaSUqym8DRlCtyi+zy\\\\\\\/GPY7wa1KigtUoRALxXnmTLFCeyOfmuwlSHwOAJl7FR1QlmbmGImiEVKOB7OZrQSyNmdhNGSwHjljKo5cGMgczZT4s1xDqsn\\\\\\\/L\\\\\\\/cfxgHL7glWQXfbc6PcvfWyE16mHGe+Rm\\\\\\\/BjyeKsfD0lWaIfo5SY7mjPpA+lUK6+V85TFV1ez3Liv5BHdJHYgwCfv2RLfZrJMdYMRNZ2ybMGV0T4JghpUJ5CwSFrVIGKOsx\\\\\\\/4KlKd1W8qPDhCUJ7mcZGu8YhWklJ9lj+Y0Fh3RtcW\\\\\\\/Hc6taHma\\\\\\\/ydNc3t1jIy7le0xrnj0skdeqPabJV8IT2uF7ODUA83nps2riMvEpwJqaI89MDoqw\\\\\\\/KU1AZUGCvBrIIg6ECvO5CCLZxMG5C8GXtpWozojksXPynwLhNqbl7IU+UEJoMurKE7v7d+TvGDy3U2J4ySrNOUU5EHPux6OWUGgv9j8taY5\\\\\\\/9Hcx\\\\\\\/OedzhBFtWrhrZqqkRmpZp8EQ1+0HiOjn850Z9x5fxlHQzACZgyRk4VPnA3dq1cZDKOkhCk5PN5155SnAKvjyyXgOHN7v2uiQB+0YFq5F9JsmtCXqUqGkSw8TOpxCxW+iVRl\\\\\\\/H40TrijOuvAa1wMlCbrghAewsebLHQkHsSdCpl34PJqBC3n2Kh0yNNmbidUQksnOuHwyjNq+ktrMiuqikgu7Y7a0FJFJVgpN80R657QzGM6LdqY5NjEuhPx0Q\\\\\\\/0UxVLWA\\\\\\\/iQSFACGP1ZdY4V7wAaum9NQHOlC2cS7cfUws+M6jGBqyHNpyL4gAfd6xAWKPZQ4jvSdPNTEyA3bsgOakCQGCYNJZm1oQopOKO1Wu9eWZjTgLp2BD3jqA25L2kVRmGQvsTrwPC1HuvJ3vs1xOGGAXa20U7DMrza2S+FOnmSNparpIBYW5mGe0t44ySmYny5fg6eu63WoWx\\\\\\\/oKdGT\\\\\\\/LcVhrbVvmUEmY8vQxv2sapOXun3vGE+A50EDViwvgvwSIm1Pl8QwdVfbljpvUtsuvoCsc2KDen3QjY0pC4d2weH2k9kkstlJHAbRIRwWxU42SDluLHP4nBP7j81xTMEbknQlj\\\\\\\/jH84vYYM4tWh0XUwZxXRcXkVdwJdX7L7unceSA\\\\\\\/mh18YCi5U2rcW+wUyHQDGN57yhpWTB5cQ20SMkgNBIjbj+ESroFnDMm3x8tmvcDpX+7gHfR0RQe8Lt\\\\\\\/yhw60t5h4\\\\\\\/KUP0+OuakXTd+rgXwyVPGiyiM2BbeqkKpX5tkkV1iCNWyWhmyI+i\\\\\\\/Lesk9zzBmVe7Mcgc1lOI\\\\\\\/ke8ft1eS6B\\\\\\\/+YC6CS+Q\\\\\\\/bLashabn3drS9coXQYHro0PSIfXCXNicY8gDua\\\\\\\/kooWHZudtsgR00NQ2wlSgnMLUva3yZfS9cizXZ5rpuLnSwkmi\\\\\\\/7jzSJ\\\\\\\/ZS0EfF\\\\\\\/z1Q81M0Wstwe382XI0ljaW\\\\\\\/PvujDu7USXVsMIjL1d3zwJaXgzUhMOmtaenTJ+4fUueWQ5pwIWUtDeABY+qy3E0OvWlHPOxewJPrt0K3esvLOlw5CQwYf3FiRq7wVyJSLxyWaBmakU1szXbIWhq6yZwKaZH+ntBlIhsTDU5WTcjzT05cmLN48YU1mAHNQsi3Pwp+xhrsmSjJlaHLF\\\\\\\/uv7okkxFC8g21cY8zNgmvViI\\\\\\\/PQE3z+OvtxAsRSQGwY2nrE3D\\\\\\\/rrDaEiCUQwjzBEjrXcHlznBG6xp3FDZiaZ\\\\\\\/YSiuw+QeBseWYrpBRm5RBLpfxcVEfT6CDE99PPuZrTINenmo3irFEDALN2yMejH\\\\\\\/jGXLBVss2bXg0wKMcmpp9VKDeiESqTyDC+1VXclxkgdM4AVL2WtcpWdvGP3aeidCAMMmLkcO3GqiM9VvPjjFRWdDwkEwH\\\\\\\/d8kdTlMeqo0UW70m8MS9sgZ6pEg\\\\\\\/yxiYiNkgK50phwb5S0xreznRYOwxL\\\\\\\/oaaGdlVjpq15dcVvAZOM\\\\\\\/ne\\\\\\\/vzx\\\\\\\/fMkT2OQgbT2MK4CW2v\\\\\\\/gA9myjPrXtBZAjd0JttRTydmIOmlFAZl46\\\\\\\/khbNXhW8rSbA\\\\\\\/5cqH7kMWfhZML\\\\\\\/V\\\\\\\/lFfzY5FZvpcukCjFsYoM8vLT\\\\\\\/9mKH3BEpg7wBb6kINcJL6DqTxurVxG\\\\\\\/urvDOqOggOz88qK8YyLBv9c0t8nLD43DNy4VOsJB4jySvlVK463+683Meck+nvV2GZru\\\\\\\/byYoF+fdTmCaNlfee\\\\\\\/DYhFEUkfb0eCdalWZAkZAwbuN24ZWIuGXRRV6+61B3sRs2rgWK+Liz2YOj+WoOwILIuaOIqlP0r62Uc3g6fkamtV46qk6UNcwHnPbMHKBpzgcp6dy0QFWBpGKtcYytwHKzkRmtDu2zeecWBo0GO9jZYqincQmNpoetEBB6ZI\\\\\\\/RzCgLfZgXiX\\\\\\\/6KpDa74BQhVVFX\\\\\\\/iHo0xZMU0f6xfO\\\\\\\/NcGtzPi5y0cbeETlhCAbj+hbUbXa+2879WzfqTn+PdUmtTGFHfzFgWR77d0IxI34dpM6YWL\\\\\\\/IRxG7\\\\\\\/qOpHjG1yjHe9s68p44AscX\\\\\\\/0RD6mIdnCZiRybg3t3fSif4MGIum8wPbu2orxIVt+ewO6v6AnKKhs8LbML\\\\\\\/zLV4R3uy7CxRpiAc9WQOskouVKGUMfZw9dEf87saAaG3R\\\\\\\/GhYttHBkxuSkq0hp20qG\\\\\\\/SSydRSTL7dmxFr+eNk\\\\\\\/WhmHTHK1lJOJp+VojcYyD1kqLZJuEkAdDG81aZIChuu03Da6vHofEeCnj6s1wD6CjJj3GagRSugI\\\\\\\/2ABP3uF3ExyWc6riKWe6JKFLhJvphH1ZNPrbEPkbj34Oj7t66\\\\\\\/okbto6hiscWolAZG0cBIRBuS7ci2Lj+SNOH0s3KCWADU6R5WSxtZSOMVYCiWdaK1XfeSjRvUt8wBxNtJpRt3TMQoHl2A7DAvofzbF6qlett8fwj7OfmZ7KjRcHUURrGCl\\\\\\\/60cnIM2DFvgcXRn+gqutfw+AMC\\\\\\\/IMYHDtRy+5ihrsWlYhmIYOp4srLDwQBQ2emvk1OKrFI0mEfXk+Z0I2sT8z4N5tAxMSgU4dcqTn05dl6D6H2y0MZlro1U8GO7wE9PpBvySif7qbhyX8qD3q4EDgPENz4tc+4hcNqXFbetCbI40YKYAzj+jRRrrChg4wnAl17lyTPhs+RZpMljm8FXW\\\\\\\/DBD7HCkecJ9Ozf+0w2jxm8KxJIRoNn2Iop2TnUcvDkh5zHdcbjcS2MlM5PlLxnNTlMS28zSR3vcabKzaTisvXHKPxBEVaP1KngpOQHOzglCM\\\\\\\/sPX5gD3CHScmdyz7Ptck0QUD9ZIH3o+1QdylH3vYpl+fgb\\\\\\\/U6BVxeOZxmTtCJJ3HuG+VCt8Kb4TXqxz309AHLIS9FZd1pd+Zl5+XspVWReA9hfOwIov6Gfh71bDRJpoJejwvU59WxhPtLbtU\\\\\\\/FFXfRBX3Jj6vzYEXz7B8ZX3EBwybGzkYyUHM1lPvGzJfCy6d6QklPVjSjmHjFFbHk251o7Ph6RkjsBa5fLQblsk\\\\\\\/mo85zrBpIeHgaI19z+BUD7OJeDZp6Y9rLw6Snk1ZGhS56ao1nZ2eDHbEzNXOSdYSE8LuBQ\\\\\\\/\\\\\\\/HwbupeQvTfy7tf9PwArTKA0qmqJ8JsN+NW9PojpZArmjHFbNXQCgvqlXUbbyK2WFhoW27P5faKV+x8+G81PdvXRA2TMRBJGR78lbKMhy20JDNu5am2FY8VqlJw+5TjEpgnDTDRzbDOvhmM8uAn\\\\\\\/nILXdRbf5A7acrJeTxVr2DIGBaIZX3cpnNaQA775Jp1vba8\\\\\\\/jMrCcpdvXqpHdwN9Ape2bNk1xcdTRnUvQ7sYcUpWQUFW+QpkgXPLeYCSNNv76B9YOujwzDTJB6t3X2POjITMtautFYifo1Gj0aXowS4MF3612bMSVJJnZajIxWKnCN7ObN1Typ7tQKQ8jB0wnkAns9vTBVKYSlT4J7JU9265FpwE4aPQyt2mvZWPZgddaCfFj61rVIle+ERQ5bDnIDDWyKdp\\\\\\\/xSKk4tVqxNC+dNDR8SoJbEo8yWJAW+8ZLfrfikpK1LLtiwNVWSQjZHZzA+9Cz6agkyxrsm0fGWb5OraxSyW3Aypkp4cq+jVm4T\\\\\\\/wXIKa\\\\\\\/MlRPErpUEiOLRMzUCjJCQ8CN736AZIyn5lACUEZybRAu9b\\\\\\\/SNE3aBdMquXS9o1rXz+r\\\\\\\/vJbWf6EvU4b23g886v+LYeaYSFMyFeLg0lRo8VW7aGFLwkwHG782XUq70RtsA9rGO9Cj5Md280AfDVSG9f5z1LDc26ZXU8e5gS7QsBCUaAHA4SPVugm\\\\\\\/4PlHenaJRbEDzNK9nMl8A9L1HTbjR6O2EFYQs+S5RRm8k8S8OAyFKFUpTVVHEjK0Sgbk5PNBgpRz2SeerPdV3ot\\\\\\\/W24mSxdg0cGnr9vw2lATlJYU9Y2gWGI9otdHltUweRTmgdtN6u1wGh0x38gXbJukThrjikql6IILgrPWUoL6dLTQ3wh4CHufHJzuJEHFTPCVLcMj3WsXu7UGhd7umncQ02MPJh69JDbGwTGpfIvoKOTnNfNZfgtrw17\\\\\\\/X7Lc5+eJEX9GHkixSixY46KQ5KguuwYUHyt\\\\\\\/Z3TLU0UsEfqmzs1e0aRVdYfd8Kvam2eNM+Y4kfifnK3bbcYZnqNa8acqG5FI5LChFXD1sEqdswtraRjjiCWQX6+SOD0sCnSyXGJXFODeSlrjTnQu6OC+x3Ov9uyCzR3N8E2JqHVqPBurJAOLAWuAVnQx3S8G4GXjDQ\\\\\\\/55YhdJB0PeDPdt3blNM32T1CnFi7Rk6CW5ZTDWkNKBm0wz6ZKQRcPrKgmbgaKH4PtOdILKs9KtsDN6rnuSdlH0GhVObO2zB\\\\\\\/0qXF2pxNyePaD5tOoLpNNgl5djC4FyVHqXaMQSVlFO8+PI8yxpTfhFVowLFDyErteIa27C9+Q4Ib+w4IJNz56f8Q9XYJ5TQNnKgNiPSv2hXCWwjSOg2Ks0D29iflH1gR4rR\\\\\\\/GuYVULy5mWCyprK88\\\\\\\/MjoyPR+HXg1LNNKibgVSPzPuyVfeJx9FQy+kHklzMMQHstbu0gk\\\\\\\/IRI0hDHXb81\\\\\\\/AqNeyLlr+mGH57wqhLFirCCN3I60+R5zPWKrX2dE5lU8k2wmmIPE67eSShsaFX0llF8EnOmn96clVWktZMh6CAwVvb74UAw4WyJYUy+oC2W18a9+LBvSkCl\\\\\\\/N5YG9k9vhpeVyXqMeIjqSmBjleQdT3vsB8owZYFWj7mfXtLJx+n36hqYyRxiKCuXLX0wQsDLHgQNjoX5wti3vWtBCIKwq7fwmOeRscEEO\\\\\\\/9tLRXxC1suOOm23Y0MIMUPSLM95KDp8RKx2RQ3KdqDriWa9M0v6cSy5xN\\\\\\\/m9P\\\\\\\/tbd24GOdwaZdWaMZSRW\\\\\\\/PaRQVQ2buFTtrpU+BZmPYLqGVs825k8JPCKFwE8C+atHmpdfGtYx3OyVOFWk4m3ymM2XjZbPOvGGOXJwoW7FcXKEEd\\\\\\\/5dAZqVL+7OZ4kNNv1+Lt25loS0B8qVKbXDNBWHyF54f8Y3WeoKYkSNV+G1SI7z\\\\\\\/YbzXkEBtEi9L0Aimt3wmdlSoxfJC6vsnJvI+fHQ7tVyZt7uTqxSHvupo4Sxs2PjNiyCd8YQjqAag+nnCHKhNpSLNMJP1ltoj9lfhCr59vtb7OQVixWtWq3BRMSGesRai+3sQHOa5mQvGBplpMB4g4AmPGViweY16GNGa92et00dEmmfW8KzQQ1MGmp7SbE5AwwF7LU7SYfyIBwhD+emroNDrfjMSkNUhfpOD0yI4AYw5Mtq0jPLRidyGrD4src3IKzEsLo4bBmBtBw2vVZleaELlHxiWKZ9Uvl3WR4mOTBTWH7ZlYNnImfyWOeBl85YxuB+ueOYYv+5Fb7tMrxH7H33VF8Y+\\\\\\\/bQNICD\\\\\\\/1Y8m8xedARGwlOeptDdlcGgh638c4l3ctw5Au5Cd8nBzix4Rf8ZIZtwYDNHaPp1pN6h\\\\\\\/0ChZNWUeZqX+kWkiFh2FflPZySrvFOoqyM9Zb1sGagn7VovhC6EbQKJTfG2AumdXCn7Z295ruKyp0sewEYG5qtqk+KwdaO7fkmIDk4kGoQKRk6ikVz27XZ6uabZRsE0HTrvyoWAEGMYX+vmFs+WD3euzniRktHYZhtQhy4ogw1BiqLmliRc+fHq+2ceSMGXsyTsscXtP36UTxShXBnrl+KxHEuz9Vr7tFHYLVJK\\\\\\\/UsJbeBz4Pl5zT\\\\\\\/yoRtierQ0Yv83jy2\\\\\\\/KjvhzQ5NTAC6MhV2t3Ln7vcsnzrVbXVqdYaNr4Ny7JWDOFvZJsxF+TaMA\\\\\\\/UzH4QIIcrQtS2UsF22gpdZ3wtB7Uo5OiZ9j0BSs9HhcoDioShaB7mu\\\\\\\/2MCDnFgelf7m2wEDtw8k\\\\\\\/bFwRF8\\\\\\\/WO1fTSQvn+AORde7hWcmgqZ6hqIr07M4PZjVEyxWcgWToDxg8ORhLIdrwaQJ9ERDMbtOfrMCJ6jSHy6tYcCS+pLEOasA+vyYHNV7w5nzyEzoJ36dNvlBjDzlIK9zzmM+IzhYiSdA38Rr6Y+wEVGeUPhRUJndEUWRiBybfAHsxyvsircV+2T7UP21lf00dt24ndtu9ABDHCR9yD\\\\\\\/JC4ICYvE++4FRQ0uETrlJM3r2PmPveEN9SN38QvRZyA5HLTi2vyB8QHTDIxuugW3moNrKVRnWYyWiO8LFBYuK7xbcHaATlnjZQTb51Lr6\\\\\\\/vKCVXEj3K\\\\\\\/O\\\\\\\/n\\\\\\\/HXFZyvoZPfy2zsBpr50YalCxlfys5ressl07CmThS7o8W1osI\\\\\\\/lG11Y04OacokFsqawbuQB2VqT\\\\\\\/tBd65+UU6YOKVh3hpA+9XiWc6xSyBIYbPE\\\\\\\/b+knNbd1VxPSp8ThfbXKPVW1nselDeUp\\\\\\\/c8AsYg0qsuOlp4w53Z\\\\\\\/nGKj2UFmNrKb8f6vddmENcw+lYVs4akNeRaQIyNOUgixps2T6M3Ig0WdkjpDUaJEeVroRw4S9ohSm\\\\\\\/lhCsopIRehOrZWeZ0vdmPBlbBDW8dDOv7y+tsdspJPZK7RdXYOqLlsdaTzd0LV94ewU3v+A5Qp+LysRgTEdMaRkw6Q6rj1iEh+rDdxhFEUuxsWry8vhFbLZqPmDPZ+6oCGqwDc9jkm7laUXAwS5tKrz1GtXCDC3Km+uLSv+Qm8adp57Cq5HR+INEKG0i9jpyz0sR2erB3hPGYbNfACcV47ZbcOBbJCk4oReOjjpLnP45oyLEv43Kv2SwoNJo4PnhHnzP38mearOy8OZhq7\\\\\\\/WSOFvMuVAwSkTMbScN\\\\\\\/EJK+EpTDQeuN6cS1EXAAu2RBstdGX0BFu0OUlLe+rWek8saCHGW9fW3HSCIPa1aOg+7N0dZm5o4FFtDY5+Z9Hn0l4YKd+JhMR179\\\\\\\/zPeznevu6dP4\\\\\\\/PsaCyWUZhq4lJMIDL3enpnaq7qtiz4VPP7z9kGycVk0EY0mJJwGcOkO3+IUmzubNJm\\\\\\\/81c69emZ+8dZqAK3k0Z3vvZaiSKsGcv+x2b7KQ1nxv7RHCot\\\\\\\/ucmAH71BYtt1jghKTsYUso0EvjJkjBoAZIpWhxmdEaGBja9ljWBwjhq5XmxvJa1i+xiiPTFD6ebZ13pQzdiV\\\\\\\/ZmoOOWJfIR1VZCU51GzTdDmmVm3XBiQ7HcwKNu\\\\\\\/xZLLyMzr8MR\\\\\\\/2q3+Zt7e4xtTq\\\\\\\/KdxAo65VFhRf\\\\\\\/0TwzWN9L7G+SNmPu+tyIPiyWm4PHz2Y2HYYfjgJHJkS5a0gGPh6g\\\\\\\/OnESg+9esW5iBpPyfeZUPCl8W3TmNdpae7SLSupwJ1MuW5zoZ68VQxwEMoPOoV3Ja+Ou4wpGNEfHq\\\\\\\/CLyOPl\\\\\\\/RCZi3\\\\\\\/VHIm08NQDpw7xeXnVY0W4ToN6PjJ68eXhDzZYRwZpTaeD51E0smWCqZCmfBneCcdz+RkQGJyEziORThaij8JuVGswV1ABAwbCVxtM8nC6ki348Iwsze2ZztDiQHt+vdC1v1LkhJqCmotSOTmyV9UPKZIT7e7oWXIT9OqpCFlka6ilNcn+sA1La+ze2NAU5o3FqFZ+jW8PnP+F6tn5Oeoz5e82rEznmRjCfpHtrD7yuAu95xCNhU3t3+RtysMN0f3x6lxA2Jbzc1AcLMu2VkI7dEjR+YO\\\\\\\/YSEL9TzLlBzzR8JEl13owsCEL9J02q1FIswN4tsoJfJr2ihdPDRcBNrK4lgFKSN+gAsO\\\\\\\/ftPv29NSsnhxRPekbtaCLG\\\\\\\/RyxUeuQXWCHs8ny7\\\\\\\/pYru5qmbqB7nP3GekyCiE\\\\\\\/KUOZD7KIfnZNou8OlBQAB61rBsh9cRi983zQDB85vtbYMcxEweC6U7u6z6iJnsm+cAD48khRsb00TxgoK9bClWVXVybzQ9bmG1rGCKCZqJ\\\\\\\/lUrXcDGAyfV0nMCw56gLFtDQzVL8pfuAJpjM7albSmg\\\\\\\/nxne5CVZwRpxvjawHj\\\\\\\/lSzxiiCNusvDUJDwIjKmLo4TXRlhkVaXn+\\\\\\\/0ZfbEqa09XEcfx9Q6GUzBc+a1ej2pgvpsH9+8vod9gGkpHDEIGpfbtWDCsg2REXKq0sTuyDnMvrC1WP2iRwGtzwThCZnUvshpMFj6EajTF4iNnADrKlRoglKIh0RWI2tkT5vqVYiOPEi9QM9reb\\\\\\\/aJLSwPawn1jj7eWDHIosxYuiJfy1b4F4ssOvugKaXqDBBjbC\\\\\\\/YNboB497CdVnJjC5ar8CwqArGsIKL69nFvpEbY\\\\\\\/YtM7nC\\\\\\\/6YazBeZ7r5YyM3zPcmlVNAWOUjpIfGczeHEWdoHU5fnLHWo0OUSDIMObMRUISx\\\\\\\/8rdDc4BLLGdkX7vORTKJ7a\\\\\\\/1joUphJUzvX40op11H70voa7xGYMa8CtCgUpxw2mXpaDq41t6pgMea1rnOy65a0uac5+An9U5PUWLfanCnPXVzu1HhcuLiiOJU9DDQWVL8o5lWbIVKPP0acRzYf45xzh7mO4\\\\\\\/uNG\\\\\\\/lswLtdZQ+CH46XTuatTp1oF5BdnYgqYRS0gn\\\\\\\/5u1xZ6i8rcWphSkiPobKp1vHsB9NptHuOzC3aKZ42sxtvfiekt5szMJIZYX8iLqXmndH0RafmIYtSlp1XO+KUQ0a3QN7kuq3gXv9D3zspbVs3aAC9mXCFVl7\\\\\\\/Mua+lAxeJfnfr8U\\\\\\\/oiQCTS0ICaqeKHG8+3xWvwl1CytxM1xwIchABVsDlAlT9gAIEXkeMLHkZ7cWNV\\\\\\\/suhrwspMQYAQcyCLQRFNibiMeOGbb2oxRG+j3z1vwVSja1yDl\\\\\\\/+yUH+dRPigqKWSc\\\\\\\/12r2o\\\\\\\/8emrr22gJH0rbNZzKr4Df1EA2zABAQ7ezpTQpcgTq4ige84csljEm05YKPZ2AvGDbl6Su3DTlAXEC+pgqmNPFUcmeYSSY1f2aiORILdoSTSaV7sLuNpetBBINUCjiDoftXyxbOqiQ63Hw3DZcp6yTUmubqPHTB6GNKLSaUiWMQT\\\\\\\/8mK+vriXR6L2LC3hzBkjKGn6JxaQhROavJ\\\\\\\/0Ey72\\\\\\\/bitkhFVi5ipSIjHDKwhIdfvU1ydZBXRVkKtSOiWBuryNRRncYF5lhFBIJopWkBdKd2HgHmO238afwG2DChfE2Wv+XqG05uy4L52uEciIWxKW+swyFUOvWmL+4U68zAF3nfNPs8d5e2\\\\\\\/1PXjbVROMJfMoze+VbPPjdC+REqIucBby7q\\\\\\\/7s5HxkL6b+aHhCnBMn3G\\\\\\\/DtRPJ8bj1WnRt+Uq7Q3\\\\\\\/oGhKwTG0VeLa1eNprpPMRcaDCB+UJjm2RjPhYB876ElqjkLJhePe1LTkTymrAju6DEKYgBrjTT+30\\\\\\\/5Do7VZHxa\\\\\\\/GyVDDUe70Sbv\\\\\\\/ET85ifvLmTv+ghyum3gcx+UWIJn4\\\\\\\/thHHfHzejzLaNYeFEcJLVujajjQY53x9jab1pJoX\\\\\\\/sggyW0VnQeTQojUjS\\\\\\\/EwRiNPmSIbjziDQ5BB7WjDIm6JEy\\\\\\\/XXSQ57ZGE3GshwmBpZ+fxZS01hBsWwu8HekIBquQ2S3Dmm45X0eeZKmR5X9t4xGH5rD0OfvuJKAR6kI5nNYPjc2DlHNr9HfTXJmCz2sr\\\\\\\/sKjwp84Q3\\\\\\\/POjBKmO1lLzMCuwyOGTznFB9xODN4z7mxNr8wrxZnZxhcJQcK+zA4ww\\\\\\\/vKBeRo99J\\\\\\\/ngFc7RJxh61YZINQPlWIFB2iW\\\\\\\/CMlXoOR9ZnvVkfjAF+wv\\\\\\\/\\\\\\\/5XtcIT93SWMD+XLav1KiV12wZ83YUYAMMFvnX8hXU+WFkDZzK\\\\\\\/ADo0Gv1w1uHGqQw+yDucwRg++wj\\\\\\\/D6zI6aSCNguFbNiCA8YLd9bj3YuclCDolmAxZztVeJXpq+rJIuCUf7x3DhbVTXgH+DDf0fqAln91C8VYTqH4memHcfNtCxDr2FnRFSM0UZxDLoV6FxjWBOao22+x3KJ7ggBtDgqJio0PAn\\\\\\\/xprtfRL11+rfKtg1hh\\\\\\\/\\\\\\\/nkpEi6mRg5zgi2mZv\\\\\\\/LHc1GVTlU1iBisjpER+h4ps7Zm6DJlyY6qrrji5uqY8hNQng4Gdub4IhJUiCC+nvc5gu2GPNbolAl0Lp1IBamvvVBMkrXvOk8lUltgehMmsY80\\\\\\\/4ByfutD7PWFp8\\\\\\\/tcBe3pNM36WiL4fPh1bL0FaFj5nYTl42OZEoiu3umumtVB50tjFOXvxCuRNPrTb\\\\\\\/9209xLlfvKEplVm6Ojqu3X2lCDEsB7cTxDLeA6sytS8wSTKc5o\\\\\\\/lHP8KylnMQsmvLHW5wb1IRvkS+zMmnn+8QBNAEPZPHfSI9ipzoS9XR+hAEKiSpWbE7x73hkQp3GjvtGpFvVbfEjMB+m2mJTTTsoidAi374G1ILdmQHTj4HxG9I\\\\\\\/FvJU3M01ERKzTps1W6RuCoF7ZA0Yq8wKAozr5sLVtcDEyzx9r5C20ddCze7o6BymO9xW5nHz03o0EeXCTBTh\\\\\\\/Beq+PVGRncLds\\\\\\\/SlTW2Q84NeiEbMwCiw\\\\\\\/RXw9rFQjHAYvZqJFNJ25F\\\\\\\/FDUL0fI9C6qIgn0DZrFUJ2VaetEZqFOb4\\\\\\\/aFP6i\\\\\\\/CjbSbIrE6kCKV0BhtQYi2NpzftWTfK8GLvYLZ+smAya5pADbHTi20QRfibqu9+WabN1Osrf2wHG8nNbxWGz1\\\\\\\/L1b9Tx3bgEokTHlkxZ5gF8Zw6qeGQ2xmksx9v7NxPDfA27DSmx3LNLTnimgfKXRz5Ftk\\\u0022,\\\u0022iv\\\u0022:\\\u00226y2U+a8TgXyH\\\\\\\/CxkctOsHQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b3751190173a91e5a1decf482211f7fa118fd34f03898b9e0f03827e4822f269\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uTNBQxwUb2V2EQIm4xYd1GR8\\\\\\\/xtHr1In1vVLDwd4IQOK3uYtiREVG9DTmtJcFzfxsEno6kAvJJ1zj4V9bi4WSlCYb7\\\\\\\/+W2PRNKWXqFSU+pAG8ZRq7NFlTDZOtLwVDBUaxlXVmeDRZQIuTdH\\\\\\\/Fn8AaK1EaXYl7dA2YWoEh4GtgjWYHYTEsPdYULcgrmOBC5nZ2Vuy5siQi5ZkSiafvFTxYrTE2als8eqbAwvNoKI2hg\\\\\\\/1hDAKo2kDK5DVvUDbyfTWtf3s\\\\\\\/YtMuW4WbOlcc09zr\\\\\\\/pi4mODUXhsPIY9mdOPDUguTg24Vns6QNV2XLru+n9cR\\\\\\\/jGxf6ztzyLPu9Gkg6f0jt6Bg0II1J73JkMtMn6gKWK0ssvpBXvYOkgmH2XphJoil8lfKtHqlDMQsjEel5atMXGvh2q6kjry5KfkYTl4bLw6kVZi3fNDmJwHCQTGL3+auwUGjP5cxwOS1BmFp2Jq3m\\\\\\\/+FO9PKydx2hlNP3wqRtYB4FKTiYpzKen6xYlyuBC73UT64rEGlQ0kIw6vbmWSxhLTIY6wOSUzF3Lr5vQzt2VJaFwdVDYxnqEYp19YFR17Yt0qLPRUagMeWnmjCS1g3v\\\\\\\/E2KHlqEKPMgp7knSHEduXcgxblaG12mY5wqWzFBoovkqUmAIyhdgDeX\\\\\\\/YhftF7LHeZw2nOKZCes48i+1zL\\\\\\\/c2hnw7IHUAK76w7PcEcAoy91MzRs2qsZXVEEa2NDebtDVvdNiRs9DcGnhW13TMIghAYmDEjH9hTQ\\\\\\\/h7k7kHtKmEwJRtSMk5AJuwys8HWLvDhxvwm5UcadwnMsJXZ0ScpYxZJ2UAOQhZ65roRBvtFARVO5JenQkR6VnZjyI+kNm\\\\\\\/nukJ7eF3U1C45fHSDkxRQ1hIhk5LM+K9iz5NDiXFDHkbJhw45rZXmkWgatOj\\\\\\\/T0dXGm9xXEcoymhTlUdAtTKW2CCQcNHaIcKjHRkbaN+YaNxphgd4+9Ev7XsXzkF39aSGFStfbMoRVa0SagdVEjeZGb\\\\\\\/I6vitnq9fUo3d1Ej07g9GPBCPYZ6E4cTWML2\\\\\\\/uqXZdJdv9ztHotPJ8MV44SdwOfqKRDF7Ww7g\\\\\\\/02OJtZGsTCbISoqPl8WYr3MXNLe0\\\\\\\/B6vSZbHgAP7BQmRQ21mWi0K17bm5S8cygMPuQHJ7MQVIdDMjKjx2G7FIqa3NM10QnnC56FljrJvruO6a\\\\\\\/RwsyPsXAIkfEsYP\\\\\\\/TiPzMww8TtPo\\\\\\\/quNUmYCqg6pD7K\\\\\\\/VYNSGkMjjnpH0ikW5UmkyeP7BOwI2UnjKk8NFrEo\\\\\\\/cUwDNUk7fmRou37WKl5I3HCiUv7BUC3WK4VgqWey\\\\\\\/U7Uhs8XLlVpQjX9WdJ4VCYYesC5i2Ft3A9\\\\\\\/mTUrlgfCda6iPp5PJEWhlMJPgcQ1lY4g6hbXks3BVnh3447Mq8bCLH6\\\\\\\/1tQPekyN\\\\\\\/Xf7GfoxZUiyoSWP5l1OFE9SJAKIMFGDUiyt7QKAgYpECHsA4TOBlZ+DU5GMUM5t84+wBKtHOF7eZ+9FvZHPXXX4ttkKXShST5JtAFZm\\\\\\\/zpj\\\\\\\/mODKkzwiQf+u9HseYgFqxUuO6Mz3oYbBje2LXFipVS2vUQgGNgUhOVKY2rU3EyVzJnoN360Hi43d4uGhmUNZwNLkcv3Kc9mooZg1fxwJPhNtb\\\\\\\/t3O2xfahJewaTIGTyupaVEvw967fA0iqARsmyoHVLXPo399+mX\\\\\\\/2nwpl6W4y6jQQMt5QE006U90+JW8hzeM18MqkJbqqLAgqz0acO6h8VoTrcCgKeTSUZzk7fh7RW8LE9JGjpNZmtUjj5aJj44SaZeXj\\\\\\\/WeeWXZDFhZtmxNlsJgWk4vBjQ0mjPpQzW09FnjtTJvipFdMF\\\\\\\/GvluKIOtpLH18p86j8hNAFP9mBaOyP1rbM9s7wJnb0lghKTzDNz2XdXx5JdXfszUqSFhV+sEyQlZ5r1UYugbigaF3CHUtfd5kKBiPZ+FC53NEoz\\\\\\\/gwyxXtIdLXAQHLpey69T0qFOyuovMxxEYIrhx0RDxnQzwXxPeOxt\\\\\\\/GXjN1IzLHNJ\\\\\\\/Hj525UHJBnikRs=\\\u0022,\\\u0022iv\\\u0022:\\\u00228NpEEYb5R2VHFneShqqohg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002204400d00b74ceb95dfef9a039b86f18d062dd5fbd539f1702b5c5cc1bd14ac5d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460300 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-mutant-screen-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/3003" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4GAQcUkwLBYWQ1gQNYr4Y9v05kBeHx0raeL6PFTC40c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/3003" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZsGb0VctrQYEGXN9D4QHmI6dFFVf3LMeWkUOfDUReCY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2983" title="Aspergillus fumigatus MADS-Box Transcription Factor rlmA Is Required for Regulation of the Cell Wall Integrity and Virulence" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/10/3015" title="Multiparental Populations: A Call for Papers" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460300" id="node460300" data-pisa="ggg;6/9/3003" data-pisa-master="ggg;g3.116.032342" data-apath="/ggg/6/9/3003.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Martin Zackrisson</span>, <span class="highwire-citation-author" data-delta="1">Johan Hallin</span>, <span class="highwire-citation-author" data-delta="2">Lars-Göran Ottosson</span>, <span class="highwire-citation-author" data-delta="3">Peter Dahl</span>, <span class="highwire-citation-author" data-delta="4">Esteban Fernandez-Parada</span>, <span class="highwire-citation-author" data-delta="5">Erik Ländström</span>, <span class="highwire-citation-author" data-delta="6">Luciano Fernandez-Ricaud</span>, <span class="highwire-citation-author" data-delta="7">Petra Kaferle</span>, <span class="highwire-citation-author" data-delta="8">Andreas Skyman</span>, <span class="highwire-citation-author" data-delta="9">Simon Stenberg</span>, <span class="highwire-citation-author" data-delta="10">Stig Omholt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-9743-9145" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Uroš Petrovič</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-5269-971X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonas Warringer</span> and <span class="highwire-citation-author" data-delta="13">Anders Blomberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3003-3014;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032342 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460300" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Martin Zackrisson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BZackrisson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zackrisson%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BZackrisson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Johan Hallin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>IRCAN, CNRS UMR 6267, INSERM U998, University of Nice, 06107, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johan%2BHallin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hallin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohan%2BHallin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lars-Göran Ottosson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars-G%C3%B6ran%2BOttosson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ottosson%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars-G%25C3%25B6ran%2BOttosson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Peter Dahl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BDahl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dahl%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BDahl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Esteban Fernandez-Parada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Esteban%2BFernandez-Parada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandez-Parada%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEsteban%2BFernandez-Parada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Erik Ländström </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BL%C3%A4ndstr%C3%B6m%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=L%C3%A4ndstr%C3%B6m%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErik%2BL%25C3%25A4ndstr%25C3%25B6m%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Luciano Fernandez-Ricaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Marine Sciences, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luciano%2BFernandez-Ricaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandez-Ricaud%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuciano%2BFernandez-Ricaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Petra Kaferle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular and Biomedical Sciences, Jožef Stefan Institute, SI-1000 Ljubljana, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petra%2BKaferle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaferle%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetra%2BKaferle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Andreas Skyman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Earth and Space Sciences, Chalmers University of Technology, SE-412 96 Gothenburg, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BSkyman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Skyman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndreas%2BSkyman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Simon Stenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Animal and Aquacultural Sciences, Centre for Integrative Genetics (CIGENE), Norwegian University of Life Sciences (UMB), 1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BStenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stenberg%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BStenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Stig Omholt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biotechnology, Faculty of Natural Sciences and Technology, NTNU Norwegian University of Science and Technology, N-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stig%2BOmholt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Omholt%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStig%2BOmholt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Uroš Petrovič </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular and Biomedical Sciences, Jožef Stefan Institute, SI-1000 Ljubljana, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Uro%C5%A1%2BPetrovi%C4%8D%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrovi%C4%8D%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUro%25C5%25A1%2BPetrovi%25C4%258D%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9743-9145" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Uroš Petrovič</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jonas Warringer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Animal and Aquacultural Sciences, Centre for Integrative Genetics (CIGENE), Norwegian University of Life Sciences (UMB), 1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonas%2BWarringer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Warringer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonas%2BWarringer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5269-971X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonas Warringer</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Anders Blomberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Marine Sciences, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BBlomberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blomberg%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnders%2BBlomberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anders.blomberg@marine.gu.se" class="" data-icon-position="" data-hide-link-title="0">anders.blomberg@marine.gu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/9/3003" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460300" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460300/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/3003.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460300" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460300/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/3003.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460300" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460300/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/3003.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460300" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460300/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/9/3003.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460300" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460300/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The capacity to map traits over large cohorts of individuals—phenomics—lags far behind the explosive development in genomics. For microbes, the estimation of growth is the key phenotype because of its link to fitness. We introduce an automated microbial phenomics framework that delivers accurate, precise, and highly resolved growth phenotypes at an unprecedented scale. Advancements were achieved through the introduction of transmissive scanning hardware and software technology, frequent acquisition of exact colony population size measurements, extraction of population growth rates from growth curves, and removal of spatial bias by reference-surface normalization. Our prototype arrangement automatically records and analyzes close to 100,000 growth curves in parallel. We demonstrate the power of the approach by extending and nuancing the known salt-defense biology in baker’s yeast. The introduced framework represents a major advance in microbial phenomics by providing high-quality data for extensive cohorts of individuals and generating well-populated and standardized phenomics databases</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/phenomics" class="hw-term hw-article-keyword hw-article-keyword-phenomics" rel="nofollow">phenomics</a></li><li class="kwd"><a href="/keyword/microbiology" class="hw-term hw-article-keyword hw-article-keyword-microbiology" rel="nofollow">microbiology</a></li><li class="kwd"><a href="/keyword/genetics" class="hw-term hw-article-keyword hw-article-keyword-genetics" rel="nofollow">genetics</a></li><li class="kwd"><a href="/keyword/high-throughput" class="hw-term hw-article-keyword hw-article-keyword-high-throughput" rel="nofollow">high throughput</a></li><li class="kwd"><a href="/keyword/screening" class="hw-term hw-article-keyword hw-article-keyword-screening" rel="nofollow">screening</a></li></ul><p id="p-4">While our ability to detect and manipulate genetic variation has improved tremendously (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Koboldt <em>et al.</em> 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mardis 2013</a>), our capacity to rapidly and accurately map the phenotypic effects of this variation—phenomics—has not made comparable gains (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Houle <em>et al.</em> 2010</a>). This is certainly true for microbes, where estimates of fitness components define the key entrance point when searching for the concerted effects of causative genetic variation. Phenomics requires the availability of large cohorts of populations of genetically diverse strains, with the model yeast <em>Saccharomyces cerevisiae</em> leading the way. Collections of artificial gene constructs, <em>e.g.</em>, single gene knockout or overexpression strains, have facilitated gene function analysis in lab strains (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Giaever <em>et al.</em> 2002</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sopko <em>et al.</em> 2006</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2011</a>). Large crosses of natural, industrial, or clinical isolates allow dissection of the genotype–phenotype relationship in wider perspectives (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cubillos <em>et al.</em> 2011</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Parts <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bloom <em>et al.</em> 2013</a>). As the majority of genetic or environmental effects on net fitness are modest in size (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Thatcher <em>et al.</em> 1998</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wagner 2000</a>), this demands access to a highly accurate measurement methodology capable of capturing subtle differences in growth phenotypes.</p><p id="p-5">An array of technologies has been applied in microbial growth phenomics (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blomberg 2011</a>). Time-resolved analysis of microcolonies estimates within-population variations in growth (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Levy <em>et al.</em> 2012</a>) but is low throughput for studies on differences between strains. Miniaturization of well-mixed liquid cultures allows automated recording of high frequency time series and very accurate measurement of population growth curves from which growth variables can be extracted (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Warringer and Blomberg 2003</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">2014</a>). However, its associated costs in time and manpower are too high to allow massive scale-up in the number of strains analyzed. Tagging of strains with DNA sequence barcodes and monitoring tag frequencies in complex strain mixtures allows very high throughput, but at the cost of less accurate and precise measurements, the potential for scoring strain-strain interactions and a costly initial investment in the tagging of strains to be phenotyped (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Giaever <em>et al.</em> 2002</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Winzeler <em>et al.</em> 1999</a>). Better cost-efficiency can be achieved using ordered arrays of microbes that are cultivated on solid media as colonies whose area is estimated using cameras (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Costanzo <em>et al.</em> 2010</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kvitek <em>et al.</em> 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bean <em>et al.</em> 2014</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lawless <em>et al.</em> 2010</a>). However, the environment can rarely be maintained constant across plates. This leads to large spatial biases, especially at plate margins. Spatial bias correction is a formidable challenge (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baryshnikova <em>et al.</em> 2010</a>) because colonies exchange nutrients and toxins with the local environment and thereby affect the growth of neighboring colonies. Accentuating problems, 3-dimensional (3D) colony population size is typically approximated by measuring the often poorly correlated 2-dimensional (2D) colony area (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pipe and Grimson 2008</a>). Furthermore, growth is mostly estimated from a single endpoint measure of colony area (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tong <em>et al.</em> 2001</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baryshnikova <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Collins <em>et al.</em> 2006</a>). As any particular end point can be reached via an endless number of very different growth paths, this often confounds conclusions by comparing colonies in different physiological states (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>; conceptual figure). Cultivation on agar has also involved the recording of low frequency time series, followed by image analysis and the fitting of relatively sparse data to models with preconceived definitions of what growth should look like (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hartman <em>et al.</em> 2015</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Banks <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Narayanan <em>et al.</em> 2015</a>). Estimation of cell densities can be enhanced after serial dilutions (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hartman <em>et al.</em> 2015</a>) but at the cost of extra robotic steps that inevitably will increase errors and decrease throughput.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A novel framework for high-resolution microbial phenomics. (A) Simulated growth data to exemplify the problem with endpoint analysis given different growth paths. Single time point measures at different stages of growth (vertical broken lines) can provide wildly diverging views of the relative growth performance of strains, alternatively scoring all genotypes as identical (t2), genotype 1 as superior (t1), or genotype 4 as superior (t3). This illustrates the conceptual importance of having access to highly resolved growth curve data. (B) Illustration of the conversion from raw pixel intensities to total population size in cell numbers for a randomly chosen example colony growing on 2% glucose in basal medium. Upper graph: Colony growth curve based on raw pixel intensities (left axis; lilac) and calibrated pixel opacity values (right axis; orange). Lower graph: Colony growth curve based on background subtracted pixel opacity values (left y-axis; red) and cell counts/population size (right y-axis; blue). (C) Colony population growth curves obtained by cultivating genetically identical WT (BY4743) colonies in four environmental contexts (conditions indicated) after having shared identical preculture media and measuring colony population size in 20 min intervals. y-axis is on log2 scale. (D) Colony population size is extracted from raw images in a multi-step procedure, proceeding from 1. raw image, to 2. probable colony (blob; blue) detection and segmentation, 3. local background definition (blue) with a safety margin to colony, and 4. estimation of cells as pixel intensity as compared to background. Color intensity = 0 (dark blue), 1500 (turquoise) cells per pixel. NaCl, sodium chloride; WT, wild type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2133002037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A novel framework for high-resolution microbial phenomics. (A) Simulated growth data to exemplify the problem with endpoint analysis given different growth paths. Single time point measures at different stages of growth (vertical broken lines) can provide wildly diverging views of the relative growth performance of strains, alternatively scoring all genotypes as identical (&lt;em&gt;t&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt;), genotype 1 as superior (&lt;em&gt;t&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt;), or genotype 4 as superior (&lt;em&gt;t&lt;sub&gt;3&lt;/sub&gt;&lt;/em&gt;). This illustrates the conceptual importance of having access to highly resolved growth curve data. (B) Illustration of the conversion from raw pixel intensities to total population size in cell numbers for a randomly chosen example colony growing on 2% glucose in basal medium. Upper graph: Colony growth curve based on raw pixel intensities (left axis; lilac) and calibrated pixel opacity values (right axis; orange). Lower graph: Colony growth curve based on background subtracted pixel opacity values (left &lt;em&gt;y&lt;/em&gt;-axis; red) and cell counts/population size (right &lt;em&gt;y&lt;/em&gt;-axis; blue). (C) Colony population growth curves obtained by cultivating genetically identical WT (BY4743) colonies in four environmental contexts (conditions indicated) after having shared identical preculture media and measuring colony population size in 20 min intervals. &lt;em&gt;y&lt;/em&gt;-axis is on log&lt;sub&gt;2&lt;/sub&gt; scale. (D) Colony population size is extracted from raw images in a multi-step procedure, proceeding from 1. raw image, to 2. probable colony (blob; blue) detection and segmentation, 3. local background definition (blue) with a safety margin to colony, and 4. estimation of cells as pixel intensity as compared to background. Color intensity = 0 (dark blue), 1500 (turquoise) cells per pixel. NaCl, sodium chloride; WT, wild type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/3003/F1.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/9/3003/F1.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469194" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">A novel framework for high-resolution microbial phenomics. (A) Simulated growth data to exemplify the problem with endpoint analysis given different growth paths. Single time point measures at different stages of growth (vertical broken lines) can provide wildly diverging views of the relative growth performance of strains, alternatively scoring all genotypes as identical (<em>t<sub>2</sub></em>), genotype 1 as superior (<em>t<sub>1</sub></em>), or genotype 4 as superior (<em>t<sub>3</sub></em>). This illustrates the conceptual importance of having access to highly resolved growth curve data. (B) Illustration of the conversion from raw pixel intensities to total population size in cell numbers for a randomly chosen example colony growing on 2% glucose in basal medium. Upper graph: Colony growth curve based on raw pixel intensities (left axis; lilac) and calibrated pixel opacity values (right axis; orange). Lower graph: Colony growth curve based on background subtracted pixel opacity values (left <em>y</em>-axis; red) and cell counts/population size (right <em>y</em>-axis; blue). (C) Colony population growth curves obtained by cultivating genetically identical WT (BY4743) colonies in four environmental contexts (conditions indicated) after having shared identical preculture media and measuring colony population size in 20 min intervals. <em>y</em>-axis is on log<sub>2</sub> scale. (D) Colony population size is extracted from raw images in a multi-step procedure, proceeding from 1. raw image, to 2. probable colony (blob; blue) detection and segmentation, 3. local background definition (blue) with a safety margin to colony, and 4. estimation of cells as pixel intensity as compared to background. Color intensity = 0 (dark blue), 1500 (turquoise) cells per pixel. NaCl, sodium chloride; WT, wild type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">The near-term goal of microbial phenomics is therefore to achieve the technical precision (low random error, noise) and accuracy (low systematic error, bias) of liquid microcultivation in a high-throughput solid media cultivation mode. To reach this goal, the 3D topology of microbial colonies must be captured at high spatial and temporal resolution and reduced to precise and accurate estimates of colony population cell densities, while comprehensively accounting for the spatial and temporal bias that makes different experimental positions and time-points nonequivalent. We present a novel microbial phenomics platform, Scan-o-matic, capable of overcoming these hurdles at a very low cost. We demonstrate its utility on the eukaryotic model organism baker’s yeast, <em>S. cerevisiae</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-8">For a complete description of the experimental procedures, please consult Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS1.pdf">File S1</a>.</p><div id="sec-2" class="subsection"><h3>Physical arrangement of Scan-o-matic</h3><p id="p-9">Scan-o-matic is based on high-quality, mass-produced desktop scanners, Epson Perfection V700 PHOTO scanners (Epson Corporation, UK), which are controlled by power-managers, GEMBIRD EnerGenie PowerManager LAN (Gembird Ltd, the Netherlands) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS1.pdf">Figure S1, A and B</a>). Images are acquired using SANE (Scanner Access Now Easy) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mosberger 1998</a>) using transmissive scanning at 600 dpi, 8-bit gray-scale and a scan area extension that captures four plates per image. Plates, without lids, are fixed in custom-made acrylic glass fixtures with orientation markers ensuring software recognition of fixture position (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS2.pdf">Figure S2</a>). Fixtures are calibrated to each scanner prior to use by a fixture calibration model. Pixel intensities are standardized across instruments using transmissive gray-scale calibration strips, Kodak Professional Q-60 Color Input Target (Kodak Company), each consisting of 20–25 (depending on producer) calibration segments with fixed gray-scale intensities (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS3.pdf">Figure S3</a>). Converting scanner-generated pixel values to standardized gray-scale values makes calibrated data independent of variation in scanner properties. Such variation may be caused by production-linked scanner-to-scanner variability or deterioration of properties over time (aging of lamps, scratches etc.) or differences between scanner types or brands. Here, scanners were maintained in a 30°, high humidity environment in a thermostat controlled room and kept covered by boxes, preventing inflow of light. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS1.pdf">File S1</a>, sections 1 and 8, for a complete description of the physical arrangement.</p></div><div id="sec-3" class="subsection"><h3>Scan-o-matic software, image acquisition, and analysis</h3><p id="p-10">Scan-o-matic is written in Python 2.7 and can be installed from <a href="https://github.com/local-minimum/scanomatic/wiki">https://github.com/local-minimum/scanomatic/wiki</a>. Matplotlib is used for graph production. Numpy, Scipy, and Scikits-Image are used for computation and analysis (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">van der Walt <em>et al.</em> 2011</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">2014</a>). Experiments are initiated from a web-interface, 7 min being the minimum time interval between scans (20 min as default) and 96, 384, or 1536 pinned plates being the allowed formats. Each series of scans is analyzed in a two-pass process. The first-pass analysis of images is performed during image acquisition, as outlined in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS1.pdf">Figure S1C</a>. Using a fixture calibration model and orientation markers, the plate and transmissive gray-scale calibration strip positions are identified (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS2.pdf">Figure S2</a>). The transmissive gray-scale calibration strip area is trimmed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS3.pdf">Figure S3A, B</a>), and segment pixel intensities are compared to manufacturer’s supplied opacity values such that a polynomial relating the two sets of values is established (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS3.pdf">Figure S3C</a>). This polynomial is called upon to transform all pixels in the image to standardized gray-scale values. In the second-pass analysis (which is performed after the experiment is completed; schematically outlined in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS1.pdf">Figure S1D</a>), a virtual grid is first established across each plate, with grid intersections matching the centers of likely colonies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS4.pdf">Figure S4</a>). At grid intersections, the local area is segmented, colonies defined relative to the local background, and pixel intensities of both are determined, compared, and converted into actual cell number estimates based on a cell calibration function. These cell number estimates reflect the 3D colony population size (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS5.pdf">Figure S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS6.pdf">Figure S6</a>). For the purpose of comparison, we also extracted the horizontal 2D area covered by each colony as the number of pixels included in each colony definition. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS1.pdf">File S1</a>, sections 2 and 3, for complete descriptions.</p></div><div id="sec-4" class="subsection"><h3>Extracting, evaluating, and normalizing population growth rates from smoothed growth curves</h3><p id="p-11">Raw growth curves are smoothed to reduce the influence of noise, first using a median filter that removes local spikes and then using a Gaussian filter that reduces remaining noise (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS7.pdf">Figure S7</a> and supplemental movies <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS2.avi">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS3.avi">File S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS4.avi">File S4</a>). The minimum population doubling time is then extracted from smoothed growth curves based on a local regression over five time points in the region around the steepest slope (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS8.pdf">Figure S8</a>). The time of extraction of the minimum population size doubling time, the error of the linear regression that underlies extraction of the minimum doubling time, and the growth curve fit to an initial value extended version of the classical Chapman-Richard model are also extracted from smoothed growth curves as auxiliary growth curve parameters. These auxiliary parameters serve as quality indices upon which the growth curves are quality ranked, offering the operator easy manual evaluation of growth curves for potential rejection due to too low-quality (∼0.3% were here rejected). Here, we also extracted the initial colony population size to understand the origins of spatial bias (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS10.pdf">Figure S10</a>); however, the initial population size is not a part of the Scan-o-matic analysis pipeline.</p><p id="p-12">Every fourth experimental position was reserved for internal controls, creating a reference grid of isogenic wild-type (WT) populations. Colony population doubling times of these controls were used to establish a reference surface. First, controls with extreme values were removed. Remaining control positions were then used to interpolate a normalization surface. For each experimental colony, the log<sub>2</sub> difference between its observed doubling time and the doubling time of the corresponding position in the normalization surface was calculated. When multiple plates were included in an experimental series (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), we first normalized for between-plates bias by shifting the mean of all colonies on a plate by a factor that transforms the mean of the nonspatially normalized controls to the mean of the controls over all plates. Because of the reference grid spatial normalization, this has a very small effect on data, with the effect representing bias that affects experiments but not controls. To call gene-by-salt interactions to exclude just general growth defects (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) we finally normalized growth rates in NaCl by subtracting growth rates in a basal (no added NaCl) environment. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS1.pdf">File S1</a>, sections 4 to 7, for complete descriptions.</p></div><div id="sec-5" class="subsection"><h3>Wet-lab experimental procedure</h3><p id="p-13">Solid media plates were cast with 50 ml of Synthetic Complete (SC) medium: 0.14% Yeast Nitrogen Base (YNB, CYN2210, ForMedium), 0.50% ammonium sulfate, 0.077% Complete Supplement Mixture (CSM, DCS0019, ForMedium), 2.0% (w/v) glucose and pH buffered to 5.8 with 1.0% (w/v) succinic acid, and 0.6% (w/v) NaOH. Media was supplemented with 20 g/L of agar. Where indicated, 2.0% glucose was replaced by 2.0% galactose and/or supplemented with salt to 0.85 M NaCl. Three strain layouts were used: 1) all colonies being diploid BY4743 reference strain (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brachmann <em>et al.</em> 1998</a>) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>); 2) colonies corresponding to single yeast gene knockouts of the haploid BY4742 deletion collection (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Giaever <em>et al.</em> 2002</a>), with WT control colonies interleaved in every fourth position and <em>n</em> = 3 replicates of each strain in juxtaposition (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, A–D</a>); 3) for the confirmation experiment (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4, E and F</a>) the same procedure was employed, but at high number of replicates (<em>n</em> = 24). The reference liquid media experiments (<em>n</em> = <em>6</em>) (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4, E and F</a>) were performed as previously described (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Warringer <em>et al.</em> 2011</a>). See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS1.pdf">File S1</a>, section 8, for a complete description of the wet-lab procedures.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dramatically enhanced measurement precision in microbial phenomics. (A and B) Comparing random noise in growth curves based on either colony area or colony population size. (A) Growth curves of two colonies, labeled 1 and 2, based on either colony area or population size (cell counts). y-axes are on log2 scale. (B) Estimating growth curve noise in the critical section of the curve when growth is maximal. Noise was measured as the standard error of the regression corresponding to the highest slope. Mean of 1536 genetically identical WT growth curves in an unstressed environment is shown. Error bars = SEM. (C and D) The sum of random noise and systematic bias over a plate, measured as the coefficient of variation across 1536 genetically identical WT colonies. (C) Precision as a function of time, for colony area size and colony population size growth rate. A single plate of unstressed populations is depicted. y-axis is on log2 scale. Green area shows the time period in which 96% (within 2&#x3C3; of the mean time point: 4.15 hr) of experiments have their maximum growth rate. (D) Precision for single measure of colony area at end of growth (48 hr), colony area-based growth rate, and colony population size-based growth rate. The mean of four plates with different conditions (2% glucose and 2% galactose, with and without 0.85 M NaCl) is shown. Error bars = SEM. (E) Left panel: visual representation of the edge effect over a plate for colony area at the end of growth (48 hr). Each square corresponds to one of 1536 genetically identical WT colonies grown in 2% glucose, basal medium. Color intensity shows colony area, dark blue = 500 and red = 1800 pixels. Red arrows indicate the three colonies highlighted in the right panel. Right panel: population size growth curves for colonies indicated by red arrows in the left panel. Curve color matches the color of the squares in the left panel. Broken line: 48 hr time point that is typically used for endpoint growth measures. Green area: time region in which 96% (within 2&#x3C3; of the mean time point, 4.15 hr) of the curves have their maximum growth rate. (F) Raw image of a corner section of a 1536 plate with genetically identical (WT) colonies growing in 2% glucose, basal medium, at 48 hr and at the time of maximal growth rate (4.33 hr). NaCl, sodium chloride; WT, wild type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2133002037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dramatically enhanced measurement precision in microbial phenomics. (A and B) Comparing random noise in growth curves based on either colony area or colony population size. (A) Growth curves of two colonies, labeled 1 and 2, based on either colony area or population size (cell counts). &lt;em&gt;y&lt;/em&gt;-axes are on log&lt;sub&gt;2&lt;/sub&gt; scale. (B) Estimating growth curve noise in the critical section of the curve when growth is maximal. Noise was measured as the standard error of the regression corresponding to the highest slope. Mean of 1536 genetically identical WT growth curves in an unstressed environment is shown. Error bars = SEM. (C and D) The sum of random noise and systematic bias over a plate, measured as the coefficient of variation across 1536 genetically identical WT colonies. (C) Precision as a function of time, for colony area size and colony population size growth rate. A single plate of unstressed populations is depicted. &lt;em&gt;y&lt;/em&gt;-axis is on log&lt;sub&gt;2&lt;/sub&gt; scale. Green area shows the time period in which 96% (within 2&#x3C3; of the mean time point: 4.15 hr) of experiments have their maximum growth rate. (D) Precision for single measure of colony area at end of growth (48 hr), colony area-based growth rate, and colony population size-based growth rate. The mean of four plates with different conditions (2% glucose and 2% galactose, with and without 0.85 M NaCl) is shown. Error bars = SEM. (E) Left panel: visual representation of the edge effect over a plate for colony area at the end of growth (48 hr). Each square corresponds to one of 1536 genetically identical WT colonies grown in 2% glucose, basal medium. Color intensity shows colony area, dark blue = 500 and red = 1800 pixels. Red arrows indicate the three colonies highlighted in the right panel. Right panel: population size growth curves for colonies indicated by red arrows in the left panel. Curve color matches the color of the squares in the left panel. Broken line: 48 hr time point that is typically used for endpoint growth measures. Green area: time region in which 96% (within 2&#x3C3; of the mean time point, 4.15 hr) of the curves have their maximum growth rate. (F) Raw image of a corner section of a 1536 plate with genetically identical (WT) colonies growing in 2% glucose, basal medium, at 48 hr and at the time of maximal growth rate (4.33 hr). NaCl, sodium chloride; WT, wild type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/3003/F2.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/9/3003/F2.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469091" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">Dramatically enhanced measurement precision in microbial phenomics. (A and B) Comparing random noise in growth curves based on either colony area or colony population size. (A) Growth curves of two colonies, labeled 1 and 2, based on either colony area or population size (cell counts). <em>y</em>-axes are on log<sub>2</sub> scale. (B) Estimating growth curve noise in the critical section of the curve when growth is maximal. Noise was measured as the standard error of the regression corresponding to the highest slope. Mean of 1536 genetically identical WT growth curves in an unstressed environment is shown. Error bars = SEM. (C and D) The sum of random noise and systematic bias over a plate, measured as the coefficient of variation across 1536 genetically identical WT colonies. (C) Precision as a function of time, for colony area size and colony population size growth rate. A single plate of unstressed populations is depicted. <em>y</em>-axis is on log<sub>2</sub> scale. Green area shows the time period in which 96% (within 2σ of the mean time point: 4.15 hr) of experiments have their maximum growth rate. (D) Precision for single measure of colony area at end of growth (48 hr), colony area-based growth rate, and colony population size-based growth rate. The mean of four plates with different conditions (2% glucose and 2% galactose, with and without 0.85 M NaCl) is shown. Error bars = SEM. (E) Left panel: visual representation of the edge effect over a plate for colony area at the end of growth (48 hr). Each square corresponds to one of 1536 genetically identical WT colonies grown in 2% glucose, basal medium. Color intensity shows colony area, dark blue = 500 and red = 1800 pixels. Red arrows indicate the three colonies highlighted in the right panel. Right panel: population size growth curves for colonies indicated by red arrows in the left panel. Curve color matches the color of the squares in the left panel. Broken line: 48 hr time point that is typically used for endpoint growth measures. Green area: time region in which 96% (within 2σ of the mean time point, 4.15 hr) of the curves have their maximum growth rate. (F) Raw image of a corner section of a 1536 plate with genetically identical (WT) colonies growing in 2% glucose, basal medium, at 48 hr and at the time of maximal growth rate (4.33 hr). NaCl, sodium chloride; WT, wild type.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comprehensive removal of spatial bias in microbial phenomics. (A) Fraction of false positives due to spatial bias within plates with genetically identical colonies (WT). Each plate corresponds to one distinct environmental challenge: 2% glucose or galactose, with or without 0.85 M NaCl. Superscript 1 = glucose or galactose plates stored (cold, dark, and enclosed in plastic) for 1 mo before use. On each plate, population size doubling times of immediately adjacent colonies (excluding every fourth position that was used as a control position) were statistically compared to those of nonadjacent colonies using a one-sample Student&#x2019;s t-test (H0 = 0 difference, &#x3B1; = 0.05). Assuming all variation to be random, i.e., no spatial bias, the random expectation is 5% false positives (broken line) at this significance cut-off. Any systematic excess of false positives corresponds to spatial bias. Pink bars = before normalization, light blue bars = after normalization to initial population size, dark blue bars = after reference grid normalization. &#x201C;All&#x201D; indicates the mean of false positives over all six plates with error bars = SEM. (B) Population size doubling time as a function of initial population size. Left panel = before normalization, right panel = after reference grid normalization. All individual estimates over four of the six genetically homogeneous 1536 plates with different environmental challenges (2% glucose and 2% galactose, with and without 0.85 M NaCl) are shown. (C) Spatial bias is removed by reference grid normalization. Genetically identical reference colonies are pinned into every fourth colony position (lower right position in every tetrad of positions), creating a matrix of 384 control colonies on which a normalization surface of population doubling times is based. The local normalization surface is subtracted from each observation. Upper panel: distribution of population size doubling times of 1536 genetically identical colonies across a plate, before normalization. Each square corresponds to a colony position. Color indicates population size doubling time. Lower panel: as upper panel, but color represents population size doubling times after reference grid normalization. See also Figure S10. (D) Frequency distribution of population size doubling times, before and after reference grid normalization, in a sample plate (blue experiments in B, 2% galactose + 0.85 M NaCl. (E) Box plot showing variation in population doubling time estimates (y-axis, CV between adjacent colonies) after normalization within each of the six genetically identical (as in A) but environmentally distinct experiments, as a function of plate mean population doubling times (x-axis). Red line = median CV for all groups of adjacent colonies on plate, box = inter quartile range (mid 50%) of CVs, whiskers = complete range of CVs. CV, coefficient of variation; Gal, galactose; Glu, glucose; NaCl, sodium chloride; WT, wild type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2133002037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comprehensive removal of spatial bias in microbial phenomics. (A) Fraction of false positives due to spatial bias within plates with genetically identical colonies (WT). Each plate corresponds to one distinct environmental challenge: 2% glucose or galactose, with or without 0.85 M NaCl. Superscript 1 = glucose or galactose plates stored (cold, dark, and enclosed in plastic) for 1 mo before use. On each plate, population size doubling times of immediately adjacent colonies (excluding every fourth position that was used as a control position) were statistically compared to those of nonadjacent colonies using a one-sample Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test (&lt;em&gt;H&lt;sub&gt;0&lt;/sub&gt;&lt;/em&gt; = 0 difference, &#x3B1; = 0.05). Assuming all variation to be random, &lt;em&gt;i.e.&lt;/em&gt;, no spatial bias, the random expectation is 5% false positives (broken line) at this significance cut-off. Any systematic excess of false positives corresponds to spatial bias. Pink bars = before normalization, light blue bars = after normalization to initial population size, dark blue bars = after reference grid normalization. &#x201C;All&#x201D; indicates the mean of false positives over all six plates with error bars = SEM. (B) Population size doubling time as a function of initial population size. Left panel = before normalization, right panel = after reference grid normalization. All individual estimates over four of the six genetically homogeneous 1536 plates with different environmental challenges (2% glucose and 2% galactose, with and without 0.85 M NaCl) are shown. (C) Spatial bias is removed by reference grid normalization. Genetically identical reference colonies are pinned into every fourth colony position (lower right position in every tetrad of positions), creating a matrix of 384 control colonies on which a normalization surface of population doubling times is based. The local normalization surface is subtracted from each observation. Upper panel: distribution of population size doubling times of 1536 genetically identical colonies across a plate, before normalization. Each square corresponds to a colony position. Color indicates population size doubling time. Lower panel: as upper panel, but color represents population size doubling times after reference grid normalization. See also Figure S10. (D) Frequency distribution of population size doubling times, before and after reference grid normalization, in a sample plate (blue experiments in B, 2% galactose + 0.85 M NaCl. (E) Box plot showing variation in population doubling time estimates (&lt;em&gt;y&lt;/em&gt;-axis, CV between adjacent colonies) after normalization within each of the six genetically identical (as in A) but environmentally distinct experiments, as a function of plate mean population doubling times (&lt;em&gt;x&lt;/em&gt;-axis). Red line = median CV for all groups of adjacent colonies on plate, box = inter quartile range (mid 50%) of CVs, whiskers = complete range of CVs. CV, coefficient of variation; Gal, galactose; Glu, glucose; NaCl, sodium chloride; WT, wild type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/3003/F3.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/9/3003/F3.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469118" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-15" class="first-child">Comprehensive removal of spatial bias in microbial phenomics. (A) Fraction of false positives due to spatial bias within plates with genetically identical colonies (WT). Each plate corresponds to one distinct environmental challenge: 2% glucose or galactose, with or without 0.85 M NaCl. Superscript 1 = glucose or galactose plates stored (cold, dark, and enclosed in plastic) for 1 mo before use. On each plate, population size doubling times of immediately adjacent colonies (excluding every fourth position that was used as a control position) were statistically compared to those of nonadjacent colonies using a one-sample Student’s <em>t</em>-test (<em>H<sub>0</sub></em> = 0 difference, α = 0.05). Assuming all variation to be random, <em>i.e.</em>, no spatial bias, the random expectation is 5% false positives (broken line) at this significance cut-off. Any systematic excess of false positives corresponds to spatial bias. Pink bars = before normalization, light blue bars = after normalization to initial population size, dark blue bars = after reference grid normalization. “All” indicates the mean of false positives over all six plates with error bars = SEM. (B) Population size doubling time as a function of initial population size. Left panel = before normalization, right panel = after reference grid normalization. All individual estimates over four of the six genetically homogeneous 1536 plates with different environmental challenges (2% glucose and 2% galactose, with and without 0.85 M NaCl) are shown. (C) Spatial bias is removed by reference grid normalization. Genetically identical reference colonies are pinned into every fourth colony position (lower right position in every tetrad of positions), creating a matrix of 384 control colonies on which a normalization surface of population doubling times is based. The local normalization surface is subtracted from each observation. Upper panel: distribution of population size doubling times of 1536 genetically identical colonies across a plate, before normalization. Each square corresponds to a colony position. Color indicates population size doubling time. Lower panel: as upper panel, but color represents population size doubling times after reference grid normalization. See also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS10.pdf">Figure S10</a>. (D) Frequency distribution of population size doubling times, before and after reference grid normalization, in a sample plate (blue experiments in B, 2% galactose + 0.85 M NaCl. (E) Box plot showing variation in population doubling time estimates (<em>y</em>-axis, CV between adjacent colonies) after normalization within each of the six genetically identical (as in A) but environmentally distinct experiments, as a function of plate mean population doubling times (<em>x</em>-axis). Red line = median CV for all groups of adjacent colonies on plate, box = inter quartile range (mid 50%) of CVs, whiskers = complete range of CVs. CV, coefficient of variation; Gal, galactose; Glu, glucose; NaCl, sodium chloride; WT, wild type.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recapitulating and extending the known salt biology with Scan-o-matic. The haploid MAT&#x3B1; yeast deletion collection (BY4742) was cultivated in 2% glucose, with and without 0.85 M NaCl. Log2 population doubling times relative the control surface of WT controls were extracted. Negative values represent growth defects. (A) Upper panel: comparison of relative population size doubling times of the yeast deletion collection in presence and absence of NaCl. Lower panel: colony population doubling times in NaCl were normalized to corresponding measures in absence of NaCl, estimating NaCl-specific growth effects. These were plotted as a function of relative population doubling times in the absence of NaCl. Little correlation remains. (B) Growth dynamics of three sample deletion strains (n = 2) with NaCl-specific growth defects, in Scan-o-matic and during liquid microcultivation. (C) Functions enriched (Fisher&#x2019;s exact test, false discovery rate, q &lt; 0.05) among the top 100, 200, and 400 most salt sensitive deletion strains in Scan-o-matic. Cut-offs approximately correspond to relative growth defects larger than &#x2212;0.27, &#x2212;020, and &#x2212;0.15. (D) Frequency distributions of salt-specific deletion strain growth effects, obtained by solid substrate cultivation in Scan-o-matic and by liquid microcultivation in a Bioscreen C (Warringer and Blomberg 2003). (E and F) A subset of 70 deletion strains, half of which were chosen to represent the most salt sensitive deletion strains detected in the global screen and half of which were chosen randomly, were recultivated in the absence and presence of 0.85 M NaCl at high replication, using Scan-o-matic (n = 24) and liquid (n = 6) microcultivation, respectively. Recultivations were performed in parallel (the same precultures) and in identical conditions, removing all conceivable systematic variation beside cultivation method. (E) Salt-specific growth defects in Scan-o-matic and liquid microcultivation regimes. Regression (black, Pearson R2 is indicated) and 1:1 lines (red) are shown. (F) Comparison of growth defects in mutants between agar and liquid growth. Deletion strains were ranked based on salt-specific growth effects during solid substrate cultivation and salt-specific growth effects were plotted for both conditions (solid and liquid). Error bars = SEM. NaCl, sodium chloride; WT, wild type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2133002037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Recapitulating and extending the known salt biology with Scan-o-matic. The haploid &lt;em&gt;MAT&#x3B1;&lt;/em&gt; yeast deletion collection (BY4742) was cultivated in 2% glucose, with and without 0.85 M NaCl. Log&lt;sub&gt;2&lt;/sub&gt; population doubling times relative the control surface of WT controls were extracted. Negative values represent growth defects. (A) Upper panel: comparison of relative population size doubling times of the yeast deletion collection in presence and absence of NaCl. Lower panel: colony population doubling times in NaCl were normalized to corresponding measures in absence of NaCl, estimating NaCl-specific growth effects. These were plotted as a function of relative population doubling times in the absence of NaCl. Little correlation remains. (B) Growth dynamics of three sample deletion strains (&lt;em&gt;n&lt;/em&gt; = 2) with NaCl-specific growth defects, in Scan-o-matic and during liquid microcultivation. (C) Functions enriched (Fisher&#x2019;s exact test, false discovery rate, &lt;em&gt;q&lt;/em&gt; &lt; 0.05) among the top 100, 200, and 400 most salt sensitive deletion strains in Scan-o-matic. Cut-offs approximately correspond to relative growth defects larger than &#x2212;0.27, &#x2212;020, and &#x2212;0.15. (D) Frequency distributions of salt-specific deletion strain growth effects, obtained by solid substrate cultivation in Scan-o-matic and by liquid microcultivation in a Bioscreen C (Warringer and Blomberg 2003). (E and F) A subset of 70 deletion strains, half of which were chosen to represent the most salt sensitive deletion strains detected in the global screen and half of which were chosen randomly, were recultivated in the absence and presence of 0.85 M NaCl at high replication, using Scan-o-matic (&lt;em&gt;n&lt;/em&gt; = 24) and liquid (&lt;em&gt;n&lt;/em&gt; = 6) microcultivation, respectively. Recultivations were performed in parallel (the same precultures) and in identical conditions, removing all conceivable systematic variation beside cultivation method. (E) Salt-specific growth defects in Scan-o-matic and liquid microcultivation regimes. Regression (black, Pearson &lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; is indicated) and 1:1 lines (red) are shown. (F) Comparison of growth defects in mutants between agar and liquid growth. Deletion strains were ranked based on salt-specific growth effects during solid substrate cultivation and salt-specific growth effects were plotted for both conditions (solid and liquid). Error bars = SEM. NaCl, sodium chloride; WT, wild type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/3003/F4.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/9/3003/F4.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/3003/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469087" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-16" class="first-child">Recapitulating and extending the known salt biology with Scan-o-matic. The haploid <em>MATα</em> yeast deletion collection (BY4742) was cultivated in 2% glucose, with and without 0.85 M NaCl. Log<sub>2</sub> population doubling times relative the control surface of WT controls were extracted. Negative values represent growth defects. (A) Upper panel: comparison of relative population size doubling times of the yeast deletion collection in presence and absence of NaCl. Lower panel: colony population doubling times in NaCl were normalized to corresponding measures in absence of NaCl, estimating NaCl-specific growth effects. These were plotted as a function of relative population doubling times in the absence of NaCl. Little correlation remains. (B) Growth dynamics of three sample deletion strains (<em>n</em> = 2) with NaCl-specific growth defects, in Scan-o-matic and during liquid microcultivation. (C) Functions enriched (Fisher’s exact test, false discovery rate, <em>q</em> &lt; 0.05) among the top 100, 200, and 400 most salt sensitive deletion strains in Scan-o-matic. Cut-offs approximately correspond to relative growth defects larger than −0.27, −020, and −0.15. (D) Frequency distributions of salt-specific deletion strain growth effects, obtained by solid substrate cultivation in Scan-o-matic and by liquid microcultivation in a Bioscreen C (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Warringer and Blomberg 2003</a>). (E and F) A subset of 70 deletion strains, half of which were chosen to represent the most salt sensitive deletion strains detected in the global screen and half of which were chosen randomly, were recultivated in the absence and presence of 0.85 M NaCl at high replication, using Scan-o-matic (<em>n</em> = 24) and liquid (<em>n</em> = 6) microcultivation, respectively. Recultivations were performed in parallel (the same precultures) and in identical conditions, removing all conceivable systematic variation beside cultivation method. (E) Salt-specific growth defects in Scan-o-matic and liquid microcultivation regimes. Regression (black, Pearson <em>R</em><sup>2</sup> is indicated) and 1:1 lines (red) are shown. (F) Comparison of growth defects in mutants between agar and liquid growth. Deletion strains were ranked based on salt-specific growth effects during solid substrate cultivation and salt-specific growth effects were plotted for both conditions (solid and liquid). Error bars = SEM. NaCl, sodium chloride; WT, wild type.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-17">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>A novel framework for high-resolution microbial phenomics on solid media</h3><p id="p-18">To develop a microbial growth phenomics framework that is both high-resolution, high throughput, and accessible to the average microbiology lab, we established an analysis pipeline based on transmissive scans of colonies expanding on the surface of a solid agar medium. This is achieved using high-quality, mass-produced, desktop scanner technology (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS1.pdf">Figure S1A</a>; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS1.pdf">File S1</a> for more detailed description of the procedure). Scans are initiated at preprogrammed intervals by means of a power manager that switches off scanner lamps immediately after scanning (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS1.pdf">Figure S1B</a>). This precaution reduces confounding spatial bias from exposure of colonies closer to the lamp parking position to excessive light and temperature. Four solid media microcultivation plates, each containing from 96 to 1536 colonies, fit in each scanner. Given initial instructions, Scan-o-matic autonomously completes the entire pipeline from data acquisition to growth feature extraction.</p><p id="p-19">The computational pipeline is divided into two separate processes, first and second pass analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS1.pdf">Figure S1, C and D</a>, respectively). The computationally less intense first pass analysis is run during image acquisition, and registers images as well as calibrates pixel intensities to the opacity values of a fixed standard. Plates are fixated within an acrylic glass fixture where their precise position is automatically detected during the first pass analysis using orientation markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS2.pdf">Figure S2, A and B</a>). Each fixture also contains a transmissive gray-scale calibration strip. During the 1<sup>st</sup> pass analysis, the observed pixel intensities for the segments of this calibration strip are related to fixed standard values given for each segment (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS3.pdf">Figure S3, A–D</a>). The established gray-scale calibration function is then applied to all pixels on the whole image. This procedure effectively accounts for variations in pixel intensities in a scanner over time, differences in properties between scanners, and also compensates for much of the random noise that arises from digitalization effects (via interpolation: see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS1.pdf">File S1</a>, section 3.4.1.3). The calibration converts pixel intensities into what is here termed pixel opacity values (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>; upper panel).</p><p id="p-20">The more computer intense secondpass analysis is performed after all images in an experimental series have been acquired. It is performed in reverse chronological order, starting with the last registered image. This allows the more challenging analysis of early images, where colony population sizes are small, to utilize information about colony position from easier to analyze later images, where population sizes are larger. First, Scan-o-matic segments images and robustly localizes the center point of colonies, placing a virtual grid across the plate where each intersection corresponds to the predicted center point of one colony (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS4.pdf">Figure S4</a>). Second, once the predicted center points have been established, Scan-o-matic precisely identifies colonies and defines pixels in each local colony area as colony pixels, local background pixels, or trash that cannot confidently be assigned to either (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS5.pdf">Figure S5</a>). The local background pixel opacity values are then subtracted from the pixel opacity values of each colony pixel, producing background-subtracted pixel opacity values. Background-subtracted pixel opacity values are finally converted via a cell calibration function into cell counts per pixel (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>; lower panel). The cell calibration function was established by comparing background-subtracted pixel opacity values of &gt; 40 colonies, covering the complete range of population sizes, to both FACS and diluted OD established cell counts (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS6.pdf">Figure S6, A–D</a>). Cell counts per pixel are summed over the colony to produce precise estimates of colony population size and finally colony population size growth curves. The key steps of the second pass analysis are shown for one example colony in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>.</p><p id="p-21">Because of the frequent recording of images and the high accuracy of the data, obtained growth curves require only very light smoothing. This is achieved using median and mean filters that remove most of the remaining noise (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS7.pdf">Figure S7</a>). Animations connecting colony images to growth curves are shown in supplemental movies, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS2.avi">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS3.avi">File S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS4.avi">File S4</a>. Suspected low-quality curves are flagged for visual inspection by the operator using various quality indices, with the operator typically having to reject &lt; 0.5% of curves. Finally, population doubling times are extracted by local regression over the steepest part of each growth curve. Typically, growth is not exponential over substantial time periods (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS8.pdf">Figure S8</a>). The whole process is performed with no need for fitting population size data to models based on prior assumptions of what growth curves should look like, because of the high frequency and the high quality of data.</p><p id="p-22">Cultivation over a range of environmental contexts showed the vast majority of growth curves to be sigmoid in shape, to be minimally affected by noise and artifacts, and to have distinct, environment-specific properties typical also in liquid cultivations (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>). Following a lag phase of variable length, population doubling times reached a peak after around 4.15 hr in basal medium that corresponded to a raw population size doubling time of 2.0 hr (Coefficient of Variation, CV = 4.9%, <em>n</em> = 1536). Thereafter, growth rates declined and cultures finally reached the stationary phase, which in carbon-limited populations growing in basal medium corresponded to 4.5–5.5 population doublings over the whole growth period. Curve shapes were found to be independent of pinning density (384 or 1536 colonies) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS9.pdf">Figure S9A</a>). Final cell number in each colony increased at lower pinning density. This was expected, given the lesser competition for carbon and energy when the number of colonies on a plate is reduced. However, initial population sizes are smaller in the denser formats, because the smaller pins of the 1536 format pin-heads deposit fewer cells. The net effect was a longer growth span for each colony in the 1536 format relative to the 384 format (mean of 5.1 <em>vs.</em> 4.9 doublings, <em>P</em> &lt; 10<sup>−8</sup>), marginally faster growth, and more precise estimates of the minimum population doubling time (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS9.pdf">Figure S9B</a>).</p></div><div id="sec-9" class="subsection"><h3>Enhanced measurement precision and accuracy of colony population size and growth rate</h3><p id="p-23">Highly time-resolved growth data offers conceptual advantages by allowing unbiased comparisons of populations with very different growth dynamics, with no need of fitting data to a preconceived model (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a>). To test whether Scan-o-matic also offers other advancements, we compared its precision and accuracy to that of the current standard in microbial high-throughput phenomics: measures of the horizontal, 2D area occupied by a colony at a single endpoint. We considered plates containing 1536 genetically identical cultures (WT) and found growth curves based on colony population sizes to be considerably less affected by random noise than their colony area counter parts (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). We quantified this improvement in precision using the standard error of the regression at the time of maximal growth rate. Overall, we found the colony area-based growth curves to be about twice as noisy as population size-based growth curves (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The improvement in precision becomes even more dramatic when considering the relative noise, <em>i.e.</em>, the standard error as a fraction of the signal strength (CV), because the signal strength of 3D growth vastly exceeds that of 2D growth. The difference in 3D relative to 2D colony information content increases as growth proceeds to reach a maximum at entry into the stationary phase. At this point, the 2D colony area has on average only doubled, whereas the 3D population size growth has increased by a factor 4.5–5.5 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>, compare left and right <em>y</em>-axes).</p><p id="p-24">Diminished random noise provides an incomplete picture of technical achievement because it overlooks systematic bias. We therefore evaluated the total measurement error across plates, <em>i.e.</em>, including both random and systematic variation between the 1536 genetically identical colonies. Following the total measurement error over time for 2D colony areas and 3D population size growth rates, respectively, we found it to shift dramatically depending on growth state (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). Precision was initially low for both 2D area and 3D population size. This followed from the relatively large variations in the robotic delivery of cells and from the larger influence of randomness in defining colony borders when pixels representing each colony are few (initially ∼100 pixels) and when true borders pass through pixels. The total error for 2D colony areas then slowly decreased, the CV reaching a minimum late in the growth phase. CV then increased again, as competition for resources intensified and only outer frame colonies with fewer neighbors continued to grow. Precision for 3D population size growth rates began similarly low, but increased dramatically, with CV reaching a minimum at around 4.5 hr. At this point, it greatly exceeded the precision of 2D colony area measurements. The time of maximum measurement precision coincided with the time period at which the population size growth rate was maximal. We therefore extracted maximal growth rates, <em>i.e.</em>, minimum population size doubling times, as not only the conceptually most relevant feature of growth curves, but also both the most stable and precisely estimated feature.</p><p id="p-25">Overall, considering six diverse environments, 3D population size doubling times were estimated with about half the error of single endpoint measures of 2D colony area (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>). This gain was not only due to the more precise measurement of 3D population sizes, it also followed from the much lower competition for resources between colonies at the time at which growth is maximal and thus from lower spatial bias (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2, E and F</a>). This avoidance of colony competition effects via maximal growth rate estimates is also conceptually important, because growth rate estimates avert confounding comparisons of cell populations that are in different physiological states at the time at which growth is estimated (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1A</a>).</p></div><div id="sec-10" class="subsection"><h3>Comprehensive removal of spatial bias by control-surface normalization</h3><p id="p-26">Careful attention to experimental standardization can reduce, but never completely remove, systematic bias between colonies, because much of that bias emerges as a consequence of design parameters that cannot be altered. The growth advantage of populations in close proximity to the plate edge is one of several such biasing effects that are intrinsic to the experimental design. Randomization of replicates across biasing factors can potentially account for this bias, but requires that biasing factors have been identified and is logistically extremely challenging when ≈ 100,000 experiments are run in parallel. The most realistic approach to account for bias is therefore by normalization, <em>i.e.</em>, to identify error patterns in the acquired data and treat them such that these error patterns are compensated for. In solid medium microbial phenomics screens, the most evident bias that remains after careful technical standardization is the bias between experimental positions within a plate. Assuming the common, but naïve, null hypothesis that all systematic bias has been removed by standardization, we would expect ≈ 5% false positives for each of the above single genotype (WT) 1536-format plates (α = 0.05, Student’s <em>t</em>-tests). However, even after very extensive standardization (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS1.pdf">File S1</a>, sections 1.1 and 8) and using the most accurate measure of growth, minimum population size doubling times, to minimize error, we scored many more (∼10 ×) false positives than chance expectation (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). Thus, substantial systematic bias remained. The change in error-to-signal ratio over time (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C</a>) indicated two distinct types of errors: an initial bias from systematic variations in the number of cells deposited at different positions on plates, and a later bias from increasing competition for resources between neighboring colonies (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2E</a>). Of these, the former is the most important at the time when the maximum growth rate is extracted, as seen from the strong correlation between the minimum population size doubling time and the initial population size (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>; left panel). However, after normalizing minimum population doubling times to the initial population size, we still found that about half of the excess of false positives remained (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS10.pdf">Figure S10</a>). Therefore, we rejected initial population size normalization as an unsatisfying approach. Further scrutinizing the topology and strength of spatial bias, we found both to vary dramatically between plates and environments, producing unpredictable and very complex patterns (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS11.pdf">Figure S11</a>; left panels). The major tendency was a positive correlation between neighbors. To meet this challenge of unpredictable and highly shifting bias, we developed a normalization approach that does not require any prior assumptions of what causes the spatial bias and is independent of its topology. We replaced every fourth position on the plates with isogenic controls, creating an array of 384 evenly distributed reference positions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS12.pdf">Figure S12</a>). Assuming that spatial bias is independent of genotype, this reference grid of control colonies should well capture the spatial bias affecting colonies in the remaining 1152 experimental positions. By interpolation from the population doubling times of control positions, after the removal of control outliers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS13.pdf">Figure S13</a>), we constructed a spatial bias control-surface that estimates the spatial bias affecting each individual position. Subtracting this spatial control-surface of population doubling times from the actual estimates, in each experimental colony position, we obtained normalized population doubling times of experiments’ relative controls. Performing this normalization for the plates containing 1536 genetically identical populations (see above), we found the visually detectable bias to be vastly reduced (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS11.pdf">Figure S11</a>; right panels) and the correlation of population doubling times to initial population size to be almost gone (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3B</a>; right panel). Critically, it also resulted in a false positive rate that approached random expectations and that was an order of magnitude lower than the one obtained without spatial normalization (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3A</a>). Spatial normalization resulted in population doubling times that were approximately normally distributed (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3D</a>) and removed the correlation between relative error (CV) and signal strength (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3E</a>). These are both fundamental requirements for the application of standard parametric statistics. After spatial normalization, the total measurement error in an unstressed environment was around 2% (CV) of the signal, approaching the 1.5% achieved with state-of-the-art microcultivation in liquid media (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Warringer <em>et al.</em> 2003</a>, <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">2011</a>). Thus, although some remnants of spatial bias did remain, this control-surface normalization resulted in a measurement precision that almost matched the best that can be achieved with lower throughput approaches.</p></div><div id="sec-11" class="subsection"><h3>Scan-o-matic extends and nuances the salt biology of baker’s yeast</h3><p id="p-27">Gene-by-salt interactions have previously been called by scoring NaCl-sensitive gene deletion strains with state-of-the-art microcultivation, and the underlying biology has been extensively studied (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Warringer <em>et al.</em> 2003</a>). To evaluate the capacity of Scan-o-matic to recapitulate established knowledge, we compared our new method with previous microcultivation experiments calling salt-sensitive mutants in the complete yeast gene deletion collection.</p><p id="p-28">By comparing growth in the presence and absence of added NaCl (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a>), we identified a large number of salt-sensitive gene deletions. Many of these deleted genes, such as those encoding Hog1 and Pbs1, which are part of a signaling cascade that activates the osmo-response, or the salinity-responsive transcription factor Crz1, corresponded to proteins that are well known to control salt tolerance (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hohmann 2002</a>). Growth curves on solid and liquid media in these deletion mutants were similar overall (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4, A and B</a>). Among the salt-sensitive mutants identified in this genome-wide screen, 14 biological processes were enriched. Many of these, <em>e.g.</em>, ion homeostasis, ion transport, response to osmotic stress, and endocytosis and vacuolar transport (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4C</a>), have previously been shown to be of importance during salt exposure (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Warringer <em>et al.</em> 2003</a>). Thus, prominent parts of the salt protection system appeared to be shared between liquid and solid growth and to be well captured by Scan-o-matic.</p><p id="p-29">Nevertheless, the distribution of gene-by-salt interactions was substantially wider using Scan-o-matic, with a distinct shoulder toward salt sensitivity (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4D</a>). Thus, at any given signal strength and at any given significance stringency, more and stronger interactions were identified on agar compared to liquid microcultivation. The amplification of signal strengths of growth phenotypes was also evident in the absence of NaCl. This suggests the increase in phenotypic variation on solid media to be a general phenomenon (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS14.pdf">Figure S14A</a>). More disconcertingly, some genes that were required for salt tolerance during liquid microcultivation were irrelevant for salt tolerance when scored on agar, and vice versa. Overall, the correlation between the here reported results and earlier published data sets from other laboratories was modest (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS15.pdf">Figure S15</a>). This implied that cultivation method could have a substantial impact on growth phenotypes in general, and salt sensitivity in particular. Unfortunately, our two screens were separated by almost a decade and many variables other than cultivation method, including strain marker background and nutrient medium composition, differed between them, precluding firm conclusions. Other published data sets provided no help in this regard, because of the vast number of additional parameters differing between these and the current screen (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS15.pdf">Figure S15</a>).</p><p id="p-30">To exclude all sources of variation other than cultivation method, we therefore rescreened a subset of salt sensitive and non-salt sensitive deletion strains with both cultivation methods. We kept all factors constant, except cultivation method, and employed a large number of replicates in randomized positions, thereby maximizing both precision and accuracy. We found correlation between cultivation methods to be only intermediate (<em>r</em><sup>2</sup> = 0.5–0.6), both in the presence and absence of salt (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS14.pdf">Figure S14, B and C</a>). As roughly 10% of the total variation could be explained by technical noise, the remaining 30–40% derived from phenotypic effects imposed by differences in cultivation method.</p><p id="p-31">Both cultivation methods called the most important salt-defense regulators. This included the signaling components Hog1 and Pbs1, Rvs161 and Rvs167 (which reorganize the action cytoskeleton during Na<sup>+</sup> stress) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Balguerie <em>et al.</em> 2002</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lombardi and Riezman 2001</a>), the regulatory subunits Ckb1 and Ckb2 of the casein kinase that regulates Na<sup>+</sup> extrusion (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Glover 1998</a>), the regulatory subunit of calcineurin Cnb1 plus its downstream transcription factor Crz1 that controls Na<sup>+</sup> efflux at current pH (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Stathopoulos and Cyert 1997</a>), and the cation extrusion regulator Sat4 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mulet <em>et al.</em> 1999</a>)(<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4E</a>). However, many mutants were salt sensitive in only a single cultivation regime. Thus, despite an abundance of amino acids in the media, the absence of either Aro2 or Aro7, required for synthesis of aromatic amino acids, was critical to salt tolerance only during colony agar-growth, as were the removal of the RNA pol II component Med1 or the inhibitor of pseudohyphae formation, Sok2 (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4F</a>). In contrast, the presence of the clathrin coated vesicle component Apl4 and the actin cytoskeleton associated Sla1 were important to salt tolerance only for cells reproducing dispersed in a solution but not on agar. Overall, 81% of retested gene deletions differed significantly (Student’s <em>t</em>-test, FDR <em>q</em> &lt; 1%) in salt tolerance between solid and liquid media growth. The majority (68%) of the mutants displayed more severe salt sensitive phenotypes on solid media. Thus, whereas our new approach recapitulated the essence of established yeast salt biology, it also highlighted the large influence of cultivation approach on both the quantitative and qualitative aspects of phenotypes.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-32">Scan-o-matic sets a new standard in microbial growth phenomics by simultaneously allowing high-resolution, high precision, accuracy, and throughput analysis. Even in our prototype arrangement, with five computers and 15 scanners, we are capable of running more than 92,000 individual growth experiments in parallel, with automated extraction of minimum population doubling times. This throughput is orders of magnitude better than what can be achieved by liquid microcultivation. In recent proof-of-principle studies, we have illustrated the importance of simultaneous high-resolution and high throughput analysis, using Scan-o-matic to completely decompose trait variation in diploids into its dominant, epistatic, and additive components (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hallin <em>et al.</em> 2016</a>), and to predict traits of individuals with near perfect accuracy from their genome and the genome and phenome of relatives (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Martens <em>et al.</em> 2016</a>). Here, we showed that Scan-o-matic nicely captures the known salt biology of <em>S. cerevisiae</em>, largely uncovering the same cellular functions as earlier found to be important by microcultivation (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Warringer <em>et al.</em> 2003</a>). However, we also firmly established the context-dependence of the salinity response with several deletion mutants displaying rather distinct effects between agar- or liquid-growth cultivation. This provides stark arguments for taking the growth regime into account when comparing phenotypes obtained using different platforms and when annotating genes in phenotype databases. Surprisingly, more and stronger phenotypic interactions were identified on agar compared to liquid microcultivation. This raises the question of what is the most natural lifestyle for <em>S. cerevisiae</em>: being dispersed in liquid media or grown as microcolonies on solid substrates? <em>S. cerevisiae</em> in nature is typically found on the surface of tree bark, in the top layer of soil in forests, the intestinal content of insects, and on damaged or very ripe fruits (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liti 2015</a>). Thus, homogeneous liquid cultivations where all individuals have equal access to nutrients, are equally exposed to detrimental factors, and are perfectly genetically mixed such that mothers and daughters are randomly dispersed in the population (as in liquid growth), may not be a good representation of how natural yeast populations grow and have been selected during evolution. Solid media cultivation might, thus, provide a more natural context for testing and revealing gene function in this species.</p><p id="p-33">Despite the high performance of Scan-o-matic using our current protocols and experimental set-up, we have not exhausted all avenues for optimization. The validity of the data can be improved either by increasing precision, <em>i.e.</em>, reducing random noise, or by reducing bias. Reductions in random noise would be achievable by changing from 8- to 16-bit image depth. However, 16-bit image depth increases file size considerably, and given the vast amount of hard-disc space taken by the files currently generated from one single experiment (≈ 5–10 Gb) this would pose a serious challenge to the logistics of data storage and analysis. Analyses of 16-bit image depth data are also poorly supported in the Python packages upon which Scan-o-matic is based. This is not a trivial challenge to overcome. Second, precision could be improved by increasing image resolution. This would ensure that each colony is represented by a larger number of pixels, reducing both the stochastic noise originating in errors in assigning pixels to colonies and the noise deriving from true colony edges going through, rather than following the boundaries of, pixels. However, increasing image resolution necessarily reduces scanning speed and thereby increases exposure to radiation, heat, and dehydration from light. This is a serious concern because exposure to intense light severely impedes yeast growth (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Logg <em>et al.</em> 2009</a>). Here, the scanning speed-to-image resolution trade-off was set such that the growth of the extremely light-sensitive <em>hog1</em> and <em>pbs2</em> mutants were unaffected by the scanning, <em>i.e.</em>, we ensured that the confounding effect of light exposure would be effectively zero under basal growth conditions (data not shown). Third, precision could be improved by increasing scanning frequency (we currently use 20 min between scans, as default). This would result in growth curves represented by a larger number of measures, allowing more exact curve smoothing to filter out noise. Again, however, this increases light stress and associated impact on cells, and creates challenges in the logistics of handling and storing data, because of the increase in data amount.</p><p id="p-34">Systematic bias can be accounted for by standardization to avoid error sources, by randomization of replicates over error sources or by normalization to remove the effects of error sources, <em>a posteriori</em>. Here, we have meticulously standardized experimental procedures as described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FileS1.pdf">File S1</a> (sections 1.1 and 8) to minimize systematic variations in the robotic transfer of cells, precultivation regimes, cultivation parameters, and data acquisition. Such careful attention to standardization is essential; normalization is complementary to, not a replacement for, standardization. Nevertheless, standardization can never completely remove the spatial bias that causes systematic variations between experimental positions on a plate, because a plate due to its edges constitutes a heterogeneous environment. We found about half of the excess of false positives that persisted after standardization to derive from spatial differences in initial population size (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3A</a>). Differences emerge from variation both in the number of cells collected from each preculture colony, and the fraction of collected cells that are deposited on the cultivation plates. The size of the preculture colonies plays a role in the number of cells deposited, with fewer or no cells collected from very small precultures. It is also possible that the length of plastic pins varies slightly across a pin pad. In addition, the variations in deposition emerge due to slight differences in medium thickness across the plate which, in turn, depend on where, how, and when the medium is poured into plates and solidifies, how much the agar creeps up along the plastic walls, and the patterns of water loss across the plate. The correlation between initial population size and population doubling time is negative, <em>i.e.</em>, the higher the initial population size, the slower the estimated growth. We believe this to be caused by the absence of an extensive exponential phase: at higher initial population sizes resource limitation sets in before the theoretical maximum growth rate has been reached. Decreasing the mean initial population size by shifting to thinner pins, <em>e.g.</em>, to a 6144 pinning format, would potentially further reduce the spatial bias derived from variation in population size. Unfortunately, the current image resolution is not sufficient for the accurate detection and definition of 6144 format colonies. An increase in image resolution would, in that case, be required, which would in turn increase the image acquisition time and bias associated with radiation and light exposure. A reduced initial population size also translates into an increased risk of sampling errors, such that nonrepresentative cells are transferred from precultures to found experimental populations.</p><p id="p-35">The spatial bias deriving from sources other than variations in initial population size has diverse origins. At the point of maximum growth rate, competition for resources, and the associated influence of the local density of cells, is small. Nevertheless, when the diffusion of nutrients through the solid matrix is slow relative the growth rate of populations, <em>e.g.</em>, in basal environments, there might be some competition. In addition, the competition for resources is influenced both by the number and the size of colonies in each region, and the variations in resource abundance that derive from any variations in the thickness of the medium. Designing experiments such that genotypes with similar growth rates are cultivated on the same plate may reduce the spatial bias from competition effects, but at the cost of running the risk of systematic differences between plates. Conceivably, the concentration of the growth limiting resource could be fine-tuned such that competition for this resource has less effect on the maximum growth rate, but given the unavoidable differences in distance to the edge, it seems implausible that the effect could be removed altogether. The differences in the local density of cells across a plate not only manifests as competition for resources, but also as a faster build-up of compounds produced and secreted as a function of growth in regions where the local density of cells is high. Some of these compounds can have adverse impacts on growth. Notably, this is true for many of the acidic metabolites that are generated by carbon catabolism and that reduce the external pH, creating spatial variations in pH across plates that depend on both distance to edge, genotype, and medium thickness (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS16.pdf">Figure S16</a>). Note that the spatial variation in pH is exacerbated by the common use of ammonium as a nitrogen source: ammonium is basic, meaning that the initial pH is high, and the pH drops rather drastically when ammonium is consumed as a function of growth (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hensing <em>et al.</em> 1995</a>). The acidification in unbuffered media can reach down to pH 2.5. At this point, growth is severely affected, with the degree of effect depending on both genotype and environment (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Walker <em>et al.</em> 2014</a>). Therefore, spatial variations in pH may not just be consequences of spatial bias in growth, but drive such bias. This underscores the importance of buffering the medium. Some spatial bias is also likely to derive from variations in the physiological state of the founding cells. Colonies are structured genetically, with mothers and daughters staying in close proximity, but also physiologically, with cells toward the periphery of the colony horizontal area having access to more nutrients and for a longer time period than their starved relatives in the colony center. When cells are collected from precultures, systematic spatial differences in whether edge or central cells of a colony are collected do emerge. These depend on the precision of the <em>x-y</em> positioning of the robotic pin-head holder, size differences of colonies, and medium surface unevenness. This means that some experimental colonies are founded by predominantly starved, or even autophagic, cells. Others are founded by cells with nutrient storages intact. This, as well, is difficult to address purely by standardization.</p><p id="p-36">These and other sources of spatial bias combine to create the highly variable and unpredictable patterns that are seen in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS11.pdf">Figure S11</a>. Avoiding these patterns completely, by standardization, is not feasible because many of the error sources, including edge effects, are intrinsic features of the experimental design. Perfect randomization of replicates across experimental positions would reduce the false positives that follow as a consequence of the spatial bias. However, randomization does not remove error. Rather, it transforms systematic errors into random errors. The number of false positives therefore decreases but false negatives increases. Perfect randomization is also effectively impossible to achieve when 100,000 experiments are run in parallel. We developed an unbiased normalization approach that sacrifices 25% of the experimental positions, dedicating these positions to internal controls whose growth rates serves as the basis for estimating and adjusting for the spatial bias <em>a posteriori</em>. The basic requirement for the normalization surface to adjust for bias is that the same spatial bias is attached to both control and experimental positions. Therefore, it is essential that controls and experiments are treated identically: they must have as similar genetic and environmental histories at the initiation of the experiments as possible. Most importantly, controls must be interleaved with experiments on the same plate at the preculture stage, at the latest. Otherwise, both initial population size-linked and other types of bias will differ between experiments and controls as evident from the discussion above. It should be noted that, because of how the 1536 colony format is constructed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS12.pdf">Figure S12</a>), no control colonies can be placed in two of the four outer edges of the colony arrays. This makes the spatial bias correction slightly weaker in the vicinity of these two edges, in particular in fast growth environments. The associated error remaining after normalization will therefore tend to be slightly higher. Alternative, more complex pinning schemes could potentially address this issue, but at the cost of very challenging logistics.</p><p id="p-37">Earlier published posterior correction procedures have used spatial variations in the performance of experimental colonies themselves (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Baryshnikova <em>et al.</em> 2010</a>). However, the use of experimental data, both as correction input and as final output, risks creating unsound statistical dependencies and associated artifacts. In general, it is not recommended. These approaches also assumed spatial bias to manifest primarily as an edge effect and that this bias can be compensated for in a row/column-wise manner. This is a reasonably valid assumption for data obtained from colonies that have ceased to expand because of nutrient depletion. However, this edge effect is not the major driver of bias at the time of maximal growth rates. Consequently, the bias remaining after control-surface normalization does not have a pronounced edge character to it. Adding a posterior correction layer of row/column based normalization would risk introducing, rather than removing, artifacts. It is also debatable how local the correction for spatial bias should be, <em>i.e.</em>, what weight should be given to controls at different distances from the specific colony position to be normalized. In our basic settings, we have elected to be conservative; <em>i.e.</em>, we assume a softly undulating rather than a highly rugged spatial bias landscape. Mostly, this agrees with the observed spatial bias topology, as can be seen in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS11.pdf">Figure S11</a>. However, when fault lines are sharp, such that the bias changes dramatically between neighboring positions, we typically fail to fully compensate for this bias. It is challenging to envision a normalization approach that is capable of accommodating these sharp fault lines without also overcompensating in regions where bias is less local.</p><p id="p-38">Beyond addressing noise and bias, there are also opportunities for conceptual advances. Condensation of population growth curves into estimates of maximal growth rates, while taking advantage of the superior precision at these time points, makes poor use of the vast trove of accumulated data. A standard 72 hr experiment encompasses 217 data points, only five of which are currently used for the extraction of growth data that are retained in downstream analysis. Time before net growth commences (growth lag) and the total gain in population size (growth efficiency) are additional fitness components that have traditionally been extracted from growth curves (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cooper 1991</a>). Given that improvements of these phases are capable of driving adaptation independently (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ibstedt <em>et al.</em> 2015</a>), they should certainly be the target of efforts to improve the analysis pipeline. However, much higher noise in the beginning and end of the net growth phase means that more attention to the experimental design and analysis procedure is needed before sufficient robustness can be achieved for these fitness components. Beyond lag and efficiency, colonies rarely maintain maximal growth rates over extended periods of time. In addition, growth often becomes multiphasic (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Warringer <em>et al.</em> 2008</a>), and this constitutes a serious challenge both for the standardized extraction of growth variables and normalization to account for bias. Finally, it is clear that the microbial phenomics field would benefit substantially from standardization. As seen in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1/FigureS15.pdf">Figure S15</a>, the overlap between studies that are supposedly rather similar is at the most moderate, presumably due to the use of a variety of nutrient media, cultivation methods, and genetic backgrounds. This low reproducibility of experimental results does not necessarily indicate that results obtained in any individual study are spurious, but rather suggests that they are only valid in a very limited genetic or environmental context, and should therefore not serve as the basis for generalized conclusions. Standardization will certainly improve reproducibility between laboratories but risks restricting microbial phenomics discoveries to the genetic and environmental context chosen for generalization. Future expansions aiming to exploit the entire growth curve and to further optimize precision and accuracy will ensure that the introduced Scan-o-matic framework provides truly high-quality microbial phenomics data for extensive cohorts of individuals to generate well-populated, highly resolved, and standardized phenotype databases.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">We thank Olle Nerman and Mats Kvarnström for much appreciated statistical and analytical support, and Charlie Boone for access to strains. Financial support from the Swedish Research Council (325-2014-6547 and 621-2014-4605) and from the Carl Trygger Foundation (CTS 12:521) to J.W., from the Swedish Research Council Forskningsrådet för miljö, areella näringar och samhällsbyggande (FORMAS) to A.B., from the Slovenian Research Agency (P1-207 and L2-1112) to U.P. and P.K., from Research Council of Norway (222364/F20) to S.W.O., and from Labex SIGNALIFE (ANR-11-LABX-0028-01) to J.H., is acknowledged.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-40">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032342/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-41"><em>Communicating editor: C. Nislow</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-11-01"><span class="received-label">Received </span>November 1, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-06-27"><span class="accepted-label">Accepted </span>June 27, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Zackrisson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.1" data-doi="10.1128/EC.1.6.1021-1031.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balguerie</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bagnat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonneu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aigle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breton</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Rvs161p and Sphingolipids Are Required for Actin Repolarization following Salt Stress.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DBalguerie%26rft.auinit1%253DA.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253D1021%26rft.epage%253D1031%26rft.atitle%253DRvs161p%2Band%2BSphingolipids%2BAre%2BRequired%2Bfor%2BActin%2BRepolarization%2Bfollowing%2BSalt%2BStress%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.1.6.1021-1031.2002%26rft_id%253Dinfo%253Apmid%252F12477802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxLzYvMTAyMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8zMDAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A quantitative fitness analysis workflow.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> <span class="cit-vol">66</span>. Available at: <a href="http://www.jove.com/video/4018/a-quantitative-fitness-analysis-workflow">http://www.jove.com/video/4018/a-quantitative-fitness-analysis-workflow</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.3" data-doi="10.1038/nmeth.1534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1017</span>–<span class="cit-lpage">1024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBaryshnikova%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D1017%26rft.epage%253D1024%26rft.atitle%253DQuantitative%2Banalysis%2Bof%2Bfitness%2Band%2Bgenetic%2Binteractions%2Bin%2Byeast%2Bon%2Ba%2Bgenome%2Bscale.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1534%26rft_id%253Dinfo%253Apmid%252F21076421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076421&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284686300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.4" data-doi="10.1371/journal.pone.0085177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bean</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaeger</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahr</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ideker</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development of ultra-high-density screening tools for microbial “omics”.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e85177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De85177%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0085177%26rft_id%253Dinfo%253Apmid%252F24465499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0085177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24465499&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.5" data-doi="10.1016/j.copbio.2010.10.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Measuring growth rate in high-throughput growth phenotyping.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">94</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.stitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.aulast%253DBlomberg%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D94%26rft.epage%253D102%26rft.atitle%253DMeasuring%2Bgrowth%2Brate%2Bin%2Bhigh-throughput%2Bgrowth%2Bphenotyping.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2010.10.013%26rft_id%253Dinfo%253Apmid%252F21095113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2010.10.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21095113&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287840700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.6" data-doi="10.1038/nature11867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loo</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lite</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Finding the sources of missing heritability in a yeast cross.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">494</span>(<span class="cit-issue">7436</span>): <span class="cit-fpage">234</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D494%26rft.spage%253D234%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11867%26rft_id%253Dinfo%253Apmid%252F23376951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23376951&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315137700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.7" data-doi="10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachmann</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cost</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caputo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">115</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DBrachmann%26rft.auinit1%253DC.%2BB.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D132%26rft.atitle%253DDesigner%2Bdeletion%2Bstrains%2Bderived%2Bfrom%2BSaccharomyces%2Bcerevisiae%2BS288C%253A%2Ba%2Buseful%2Bset%2Bof%2Bstrains%2Band%2Bplasmids%2Bfor%2BPCR-mediated%2Bgene%2Bdisruption%2Band%2Bother%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%252819980130%252914%253A2%253C115%253A%253AAID-YEA204%253E3.0.CO%253B2-2%26rft_id%253Dinfo%253Apmid%252F9483801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9483801&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071817800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.8" data-doi="10.1186/gb-2006-7-7-r63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">R63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCollins%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253DR63%26rft.epage%253DR63%26rft.atitle%253DA%2Bstrategy%2Bfor%2Bextracting%2Band%2Banalyzing%2Blarge-scale%2Bquantitative%2Bepistatic%2Binteraction%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-7-r63%26rft_id%253Dinfo%253Apmid%252F16859555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-7-r63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16859555&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.3003.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-source">Bacterial Growth and Division</span>, <span class="cit-publ-name">Elsevier</span>, <span class="cit-publ-loc">Amsterdam</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.10" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>(<span class="cit-issue">5964</span>): <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMzAwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.11" data-doi="10.1111/j.1365-294X.2011.05005.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fargier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Assessing the complex architecture of polygenic traits in diverged yeast populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1401</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCubillos%26rft.auinit1%253DF.%2BA.%26rft.volume%253D20%26rft.issue%253D7%26rft.spage%253D1401%26rft.epage%253D1413%26rft.atitle%253DAssessing%2Bthe%2Bcomplex%2Barchitecture%2Bof%2Bpolygenic%2Btraits%2Bin%2Bdiverged%2Byeast%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05005.x%26rft_id%253Dinfo%253Apmid%252F21261765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05005.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21261765&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288705300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.12" data-doi="10.1534/genetics.113.155515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scovacricchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1141</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155515%26rft_id%253Dinfo%253Apmid%252F24037264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMzAwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.13" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the Saccharomyces cerevisiae genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>(<span class="cit-issue">6896</span>): <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.14" data-doi="10.1016/S0079-6603(08)61030-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glover</span>  <span class="cit-name-given-names">C. V.</span>  <span class="cit-name-suffix">3rd</span>.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">On the physiological role of casein kinase II in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Prog. Nucleic Acid Res. Mol. Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2Bnucleic%2Bacid%2Bresearch%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DProg%2BNucleic%2BAcid%2BRes%2BMol%2BBiol%26rft.aulast%253DGlover%26rft.auinit1%253DC.%2BV.%26rft.volume%253D59%26rft.spage%253D95%26rft.epage%253D133%26rft.atitle%253DOn%2Bthe%2Bphysiological%2Brole%2Bof%2Bcasein%2Bkinase%2BII%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0079-6603%252808%252961030-2%26rft_id%253Dinfo%253Apmid%252F9427841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0079-6603(08)61030-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9427841&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071719100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Märtens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zackrisson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Powerful decomposition of complex traits in a diploid model using Phased Outbred Lines.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> DOI: <a href="http://dx.doi.org/10.1101/042176">http://dx.doi.org/10.1101/042176</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.16" data-doi="10.3390/genes6010024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">J. L. t.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stisher</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Outlaw</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Yeast Phenomics: An Experimental Approach for Modeling Gene Interaction Networks that Buffer Disease.</span> <abbr class="cit-jnl-abbrev">Genes (Basel)</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">24</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2528Basel%2529%26rft.volume%253D6%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes6010024%26rft_id%253Dinfo%253Apmid%252F25668739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes6010024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25668739&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.17" data-doi="10.1007/BF00170623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hensing</span>  <span class="cit-name-given-names">M. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bangma</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raamsdonk</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Hulster</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dijken</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Effects of cultivation conditions on the production of heterologous α-galactosidase by Kluyveromyces lactis.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">58</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BMicrobiol.%2BBiotechnol.%26rft.volume%253D43%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00170623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00170623&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.18" data-doi="10.1128/MMBR.66.2.300-372.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hohmann</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Osmotic stress signaling and osmoadaptation in yeasts.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">66</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">300</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DHohmann%26rft.auinit1%253DS.%26rft.volume%253D66%26rft.issue%253D2%26rft.spage%253D300%26rft.epage%253D372%26rft.atitle%253DOsmotic%2BStress%2BSignaling%2Band%2BOsmoadaptation%2Bin%2BYeasts%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.66.2.300-372.2002%26rft_id%253Dinfo%253Apmid%252F12040128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2Ni8yLzMwMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8zMDAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.19" data-doi="10.1038/nrg2897"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govindaraju</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omholt</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Phenomics: the next challenge.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">855</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHoule%26rft.auinit1%253DD.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D855%26rft.epage%253D866%26rft.atitle%253DPhenomics%253A%2Bthe%2Bnext%2Bchallenge.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2897%26rft_id%253Dinfo%253Apmid%252F21085204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2897&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085204&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284359000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.20" data-doi="10.1093/molbev/msu285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ibstedt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stenberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bages</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Concerted evolution of life stage performances signals recent selection on yeast nitrogen use.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">153</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu285%26rft_id%253Dinfo%253Apmid%252F25349282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMyLzEvMTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzMwMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.21" data-doi="10.1016/j.cell.2013.09.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinberg</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mardis</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The next-generation sequencing revolution and its impact on genomics.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">155</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">27</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D155%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.09.006%26rft_id%253Dinfo%253Apmid%252F24074859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.09.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24074859&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324916700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.22" data-doi="10.1371/journal.pgen.1000223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kvitek</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Will</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Variations in stress sensitivity and genomic expression in diverse S. cerevisiae isolates.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e1000223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKvitek%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000223%26rft.epage%253De1000223%26rft.atitle%253DVariations%2Bin%2Bstress%2Bsensitivity%2Band%2Bgenomic%2Bexpression%2Bin%2Bdiverse%2BS.%2Bcerevisiae%2Bisolates.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000223%26rft_id%253Dinfo%253Apmid%252F18927628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18927628&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Colonyzer: automated quantification of micro-organism growth characteristics on solid agar.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> 11: 287.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.24" data-doi="10.1371/journal.pbio.1001325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziv</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegal</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bet hedging in yeast by heterogeneous, age-correlated expression of a stress protectant.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e1001325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DLevy%26rft.auinit1%253DS.%2BF.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253De1001325%26rft.epage%253De1001325%26rft.atitle%253DBet%2Bhedging%2Bin%2Byeast%2Bby%2Bheterogeneous%252C%2Bage-correlated%2Bexpression%2Bof%2Ba%2Bstress%2Bprotectant.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001325%26rft_id%253Dinfo%253Apmid%252F22589700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22589700&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.25" data-doi="10.1038/nbt.1832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vizeacoumar</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahr</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Systematic exploration of essential yeast gene function with temperature-sensitive mutants.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">361</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLi%26rft.auinit1%253DZ.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D361%26rft.epage%253D367%26rft.atitle%253DSystematic%2Bexploration%2Bof%2Bessential%2Byeast%2Bgene%2Bfunction%2Bwith%2Btemperature-sensitive%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1832%26rft_id%253Dinfo%253Apmid%252F21441928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21441928&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289284900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.26" data-doi="10.7554/eLife.05835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The fascinating and secret wild life of the budding yeast S. cerevisiae.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e05835</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D4%26rft.issue%253D0%26rft.spage%253De05835%26rft.epage%253De05835%26rft.atitle%253DThe%2Bfascinating%2Band%2Bsecret%2Bwild%2Blife%2Bof%2Bthe%2Bbudding%2Byeast%2BS.%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.05835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.05835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.27" data-doi="10.1111/j.1567-1364.2009.00536.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Logg</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bodvard</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kall</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Investigations on light-induced stress in fluorescence microscopy using nuclear localization of the transcription factor Msn2p as a reporter.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">875</span>–<span class="cit-lpage">884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00536.x%26rft_id%253Dinfo%253Apmid%252F19566686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjkvNi84NzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMzAwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.28" data-doi="10.1074/jbc.M008735200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lombardi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riezman</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Rvs161p and Rvs167p, the two yeast amphiphysin homologs, function together in vivo.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">276</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">6016</span>–<span class="cit-lpage">6022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M008735200%26rft_id%253Dinfo%253Apmid%252F11096097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzYvOC82MDE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzMwMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.29" data-doi="10.1146/annurev-anchem-062012-092628"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mardis</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Next-generation sequencing platforms.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anal. Chem. (Palo Alto, Calif.)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnal.%2BChem.%2B%2528Palo%2BAlto%252C%2BCalif.%2529%26rft.volume%253D6%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-anchem-062012-092628%26rft_id%253Dinfo%253Apmid%252F23560931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-anchem-062012-092628&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23560931&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.30" data-doi="10.1038/ncomms11512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Predicting quantitative traits from genome and phenome with near perfect accuracy.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">11512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D11512%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11512%26rft_id%253Dinfo%253Apmid%252F27160605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27160605&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.3003.31"><div class="cit-metadata unstructured">Mosberger, D., 1998<em> </em>The SANE Scanner Interface. Linux J. 47. Available at: <a href="http://www.linuxjournal.com/article/2395">http://www.linuxjournal.com/article/2395</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.32" data-doi="10.1128/MCB.19.5.3328"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mulet</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leube</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kron</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rios</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A novel mechanism of ion homeostasis and salt tolerance in yeast: the Hal4 and Hal5 protein kinases modulate the Trk1-Trk2 potassium transporter.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">3328</span>–<span class="cit-lpage">3337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMulet%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D3328%26rft.epage%253D3337%26rft.atitle%253DA%2BNovel%2BMechanism%2Bof%2BIon%2BHomeostasis%2Band%2BSalt%2BTolerance%2Bin%2BYeast%253A%2Bthe%2BHal4%2Band%2BHal5%2BProtein%2BKinases%2BModulate%2Bthe%2BTrk1-Trk2%2BPotassium%2BTransporter%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.5.3328%26rft_id%253Dinfo%253Apmid%252F10207057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE5LzUvMzMyOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8zMDAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.33" data-doi="10.1371/journal.pone.0132240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narayanan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubarry</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantitative Fitness Analysis Identifies exo1 and Other Suppressors or Enhancers of Telomere Defects in Schizosaccharomyces pombe.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e0132240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0132240%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0132240%26rft_id%253Dinfo%253Apmid%252F26168240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0132240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26168240&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.34" data-doi="10.1101/gr.116731.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Revealing the genetic structure of a trait by sequencing a population under selection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1131</span>–<span class="cit-lpage">1138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.116731.110%26rft_id%253Dinfo%253Apmid%252F21422276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTEzMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8zMDAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.35" data-doi="10.1039/b708241j"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pipe</span>  <span class="cit-name-given-names">L. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimson</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Spatial-temporal modelling of bacterial colony growth on solid media.</span> <abbr class="cit-jnl-abbrev">Mol. Biosyst.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">192</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BBiosyst%26rft.aulast%253DPipe%26rft.auinit1%253DL.%2BZ.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D192%26rft.epage%253D198%26rft.atitle%253DSpatial-temporal%2Bmodelling%2Bof%2Bbacterial%2Bcolony%2Bgrowth%2Bon%2Bsolid%2Bmedia.%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fb708241j%26rft_id%253Dinfo%253Apmid%252F18437261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/b708241j&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18437261&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253492200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.36" data-doi="10.1016/j.molcel.2005.12.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sopko</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping pathways and phenotypes by systematic gene overexpression.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">319</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSopko%26rft.auinit1%253DR.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D319%26rft.epage%253D330%26rft.atitle%253DMapping%2Bpathways%2Band%2Bphenotypes%2Bby%2Bsystematic%2Bgene%2Boverexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2005.12.011%26rft_id%253Dinfo%253Apmid%252F16455487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2005.12.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16455487&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235436100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.37" data-doi="10.1101/gad.11.24.3432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stathopoulos</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cyert</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Calcineurin acts through the CRZ1/TCN1-encoded transcription factor to regulate gene expression in yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">24</span>): <span class="cit-fpage">3432</span>–<span class="cit-lpage">3444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DStathopoulos%26rft.auinit1%253DA.%2BM.%26rft.volume%253D11%26rft.issue%253D24%26rft.spage%253D3432%26rft.epage%253D3444%26rft.atitle%253DCalcineurin%2Bacts%2Bthrough%2Bthe%2BCRZ1%252FTCN1-encoded%2Btranscription%2Bfactor%2Bto%2Bregulate%2Bgene%2Bexpression%2Bin%25A0yeast%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.24.3432%26rft_id%253Dinfo%253Apmid%252F9407035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzI0LzM0MzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMzAwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.38" data-doi="10.1073/pnas.95.1.253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thatcher</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Marginal fitness contributions of nonessential genes in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">253</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DThatcher%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D95%26rft.issue%253D1%26rft.spage%253D253%26rft.epage%253D257%26rft.atitle%253DMarginal%2Bfitness%2Bcontributions%2Bof%2Bnonessential%2Bgenes%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.1.253%26rft_id%253Dinfo%253Apmid%252F9419362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5NS8xLzI1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8zMDAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.39" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>(<span class="cit-issue">5550</span>): <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzMwMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.40" data-doi="10.1109/MCSE.2010.145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van der Walt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colbert</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varoquaux</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The NumPy Array: A Structure for Efficient Numerical Computation.</span> <abbr class="cit-jnl-abbrev">Comput. Sci. Eng.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">22</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BSci.%2BEng.%26rft.volume%253D13%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2010.145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MCSE.2010.145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.41" data-doi="10.7717/peerj.453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van der Walt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schonberger</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunez-Iglesias</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulogne</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warner</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">scikit-image: image processing in Python.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D2%26rft.spage%253De453%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.453%26rft_id%253Dinfo%253Apmid%252F25024921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.453&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25024921&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The role of population size, pleiotropy and fitness effects of mutations in the evolution of overlapping gene functions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1389</span>–<span class="cit-lpage">1401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWagner%26rft.auinit1%253DA.%26rft.volume%253D154%26rft.issue%253D3%26rft.spage%253D1389%26rft.epage%253D1401%26rft.atitle%253DThe%2BRole%2Bof%2BPopulation%2BSize%252C%2BPleiotropy%2Band%2BFitness%2BEffects%2Bof%2BMutations%2Bin%2Bthe%2BEvolution%2Bof%2BOverlapping%2BGene%2BFunctions%26rft_id%253Dinfo%253Apmid%252F10757778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC8zLzEzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMzAwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.43" data-doi="10.1186/1471-2164-15-552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liccioli</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalatzis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide identification of the Fermentome; genes required for successful and timely completion of wine-like fermentation by Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D552%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-552%26rft_id%253Dinfo%253Apmid%252F24993029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24993029&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.44" data-doi="10.1002/yea.931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Automated screening in environmental arrays allows analysis of quantitative phenotypic profiles in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">53</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D67%26rft.atitle%253DAutomated%2Bscreening%2Bin%2Benvironmental%2Barrays%2Ballows%2Banalysis%2Bof%2Bquantitative%2Bphenotypic%2Bprofiles%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.931%26rft_id%253Dinfo%253Apmid%252F12489126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12489126&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180219900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.3003.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Yeast Phenomics-Large scale mapping of the genetic basis for organismal traits</span>, pp. <span class="cit-fpage">172</span>–<span class="cit-lpage">207</span> in <span class="cit-source">Phenomics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.46" data-doi="10.1073/pnas.2435976100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ericson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nerman</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">High-resolution yeast phenomics resolves different physiological features in the saline response.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>(<span class="cit-issue">26</span>): <span class="cit-fpage">15724</span>–<span class="cit-lpage">15729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2435976100%26rft_id%253Dinfo%253Apmid%252F14676322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzI2LzE1NzI0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzMwMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.47" data-doi="10.1186/1472-6769-8-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anevski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Chemogenetic fingerprinting by analysis of cellular growth dynamics.</span> <abbr class="cit-jnl-abbrev">BMC Chem. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BChem%2BBiol%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DChemogenetic%2Bfingerprinting%2Bby%2Banalysis%2Bof%2Bcellular%2Bgrowth%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1472-6769-8-3%26rft_id%253Dinfo%253Apmid%252F18721464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1472-6769-8-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18721464&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.48" data-doi="10.1371/journal.pgen.1002111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Trait variation in yeast is defined by population history.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e1002111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002111%26rft.epage%253De1002111%26rft.atitle%253DTrait%2Bvariation%2Bin%2Byeast%2Bis%2Bdefined%2Bby%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002111%26rft_id%253Dinfo%253Apmid%252F21698134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21698134&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F3003.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.3003.49" data-doi="10.1126/science.285.5429.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astromoff</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional characterization of the S. cerevisiae genome by gene deletion and parallel analysis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>(<span class="cit-issue">5429</span>): <span class="cit-fpage">901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D285%26rft.issue%253D5429%26rft.spage%253D901%26rft.epage%253D906%26rft.atitle%253DFunctional%2BCharacterization%2Bof%2Bthe%2BS.%2Bcerevisiae%2BGenome%2Bby%2BGene%2BDeletion%2Band%2BParallel%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5429.901%26rft_id%253Dinfo%253Apmid%252F10436161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyOS85MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMzAwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/9/3003.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2983" title="Aspergillus fumigatus MADS-Box Transcription Factor rlmA Is Required for Regulation of the Cell Wall Integrity and Virulence" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/10/3015" title="Multiparental Populations: A Call for Papers" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/9"><span>Volume 6 Issue 9, September 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/9.cover-source.jpg?itok=nund1hRf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/3003.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zDvvijClmivCyY8mFuvt6omOfYiFLAZMwkOVU4B3WSA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/mutant-screen-report" class="" data-icon-position="" data-hide-link-title="0">Mutant Screen Report</a></div><div class="field-item odd"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/9/3003" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/9/3003" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/9/3003" class="active" data-icon-position="" data-hide-link-title="0">Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460300" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kaNdnfPSFPg8WfJqqF8_CC6RQVoumgpHY2B68_qPmYM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/3003.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/9/3003" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-rAusd5Sl2vcntW0oZg9wNvJdeFHL3wWv2hjZOZICYmo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/9/3003" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/3003.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460300" id="node460300--2" data-pisa="ggg;6/9/3003" data-pisa-master="ggg;g3.116.032342" data-apath="/ggg/6/9/3003.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Martin Zackrisson</span>, <span class="highwire-citation-author" data-delta="1">Johan Hallin</span>, <span class="highwire-citation-author" data-delta="2">Lars-Göran Ottosson</span>, <span class="highwire-citation-author" data-delta="3">Peter Dahl</span>, <span class="highwire-citation-author" data-delta="4">Esteban Fernandez-Parada</span>, <span class="highwire-citation-author" data-delta="5">Erik Ländström</span>, <span class="highwire-citation-author" data-delta="6">Luciano Fernandez-Ricaud</span>, <span class="highwire-citation-author" data-delta="7">Petra Kaferle</span>, <span class="highwire-citation-author" data-delta="8">Andreas Skyman</span>, <span class="highwire-citation-author" data-delta="9">Simon Stenberg</span>, <span class="highwire-citation-author" data-delta="10">Stig Omholt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-9743-9145" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Uroš Petrovič</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-5269-971X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonas Warringer</span> and <span class="highwire-citation-author" data-delta="13">Anders Blomberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3003-3014;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032342 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460300--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Martin Zackrisson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BZackrisson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zackrisson%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BZackrisson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Johan Hallin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>IRCAN, CNRS UMR 6267, INSERM U998, University of Nice, 06107, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johan%2BHallin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hallin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohan%2BHallin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lars-Göran Ottosson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars-G%C3%B6ran%2BOttosson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ottosson%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars-G%25C3%25B6ran%2BOttosson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Peter Dahl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BDahl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dahl%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BDahl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Esteban Fernandez-Parada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Esteban%2BFernandez-Parada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandez-Parada%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEsteban%2BFernandez-Parada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Erik Ländström </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BL%C3%A4ndstr%C3%B6m%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=L%C3%A4ndstr%C3%B6m%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErik%2BL%25C3%25A4ndstr%25C3%25B6m%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Luciano Fernandez-Ricaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Marine Sciences, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luciano%2BFernandez-Ricaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandez-Ricaud%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuciano%2BFernandez-Ricaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Petra Kaferle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular and Biomedical Sciences, Jožef Stefan Institute, SI-1000 Ljubljana, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petra%2BKaferle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaferle%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetra%2BKaferle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Andreas Skyman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Earth and Space Sciences, Chalmers University of Technology, SE-412 96 Gothenburg, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BSkyman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Skyman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndreas%2BSkyman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Simon Stenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Animal and Aquacultural Sciences, Centre for Integrative Genetics (CIGENE), Norwegian University of Life Sciences (UMB), 1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BStenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stenberg%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BStenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Stig Omholt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biotechnology, Faculty of Natural Sciences and Technology, NTNU Norwegian University of Science and Technology, N-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stig%2BOmholt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Omholt%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStig%2BOmholt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Uroš Petrovič </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular and Biomedical Sciences, Jožef Stefan Institute, SI-1000 Ljubljana, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Uro%C5%A1%2BPetrovi%C4%8D%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrovi%C4%8D%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUro%25C5%25A1%2BPetrovi%25C4%258D%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9743-9145" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Uroš Petrovič</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jonas Warringer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Animal and Aquacultural Sciences, Centre for Integrative Genetics (CIGENE), Norwegian University of Life Sciences (UMB), 1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonas%2BWarringer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Warringer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonas%2BWarringer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5269-971X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonas Warringer</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Anders Blomberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Marine Sciences, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BBlomberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blomberg%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnders%2BBlomberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anders.blomberg@marine.gu.se" class="" data-icon-position="" data-hide-link-title="0">anders.blomberg@marine.gu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/9/3003" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/9/3003&amp;title=Scan-o-matic%3A%20High-Resolution%20Microbial%20Phenomics%20at%20a%20Massive%20Scale" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/9/3003&amp;title=Scan-o-matic%3A%20High-Resolution%20Microbial%20Phenomics%20at%20a%20Massive%20Scale" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/9/3003&amp;title=Scan-o-matic%3A%20High-Resolution%20Microbial%20Phenomics%20at%20a%20Massive%20Scale" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/9/3003&amp;text=Scan-o-matic%3A%20High-Resolution%20Microbial%20Phenomics%20at%20a%20Massive%20Scale" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/9/3003&amp;title=Scan-o-matic%3A%20High-Resolution%20Microbial%20Phenomics%20at%20a%20Massive%20Scale" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/9/3003&amp;t=Scan-o-matic%3A%20High-Resolution%20Microbial%20Phenomics%20at%20a%20Massive%20Scale" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/9/3003&amp;title=Scan-o-matic%3A%20High-Resolution%20Microbial%20Phenomics%20at%20a%20Massive%20Scale" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/9/3003&amp;title=Scan-o-matic%3A%20High-Resolution%20Microbial%20Phenomics%20at%20a%20Massive%20Scale" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460300/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460300" id="node460300--3" data-pisa="ggg;6/9/3003" data-pisa-master="ggg;g3.116.032342" data-apath="/ggg/6/9/3003.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Martin Zackrisson</span>, <span class="highwire-citation-author" data-delta="1">Johan Hallin</span>, <span class="highwire-citation-author" data-delta="2">Lars-Göran Ottosson</span>, <span class="highwire-citation-author" data-delta="3">Peter Dahl</span>, <span class="highwire-citation-author" data-delta="4">Esteban Fernandez-Parada</span>, <span class="highwire-citation-author" data-delta="5">Erik Ländström</span>, <span class="highwire-citation-author" data-delta="6">Luciano Fernandez-Ricaud</span>, <span class="highwire-citation-author" data-delta="7">Petra Kaferle</span>, <span class="highwire-citation-author" data-delta="8">Andreas Skyman</span>, <span class="highwire-citation-author" data-delta="9">Simon Stenberg</span>, <span class="highwire-citation-author" data-delta="10">Stig Omholt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-9743-9145" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Uroš Petrovič</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-5269-971X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonas Warringer</span> and <span class="highwire-citation-author" data-delta="13">Anders Blomberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3003-3014;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032342 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460300--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Martin Zackrisson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BZackrisson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zackrisson%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BZackrisson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Johan Hallin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>IRCAN, CNRS UMR 6267, INSERM U998, University of Nice, 06107, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johan%2BHallin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hallin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohan%2BHallin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lars-Göran Ottosson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars-G%C3%B6ran%2BOttosson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ottosson%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars-G%25C3%25B6ran%2BOttosson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Peter Dahl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BDahl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dahl%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BDahl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Esteban Fernandez-Parada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Esteban%2BFernandez-Parada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandez-Parada%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEsteban%2BFernandez-Parada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Erik Ländström </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BL%C3%A4ndstr%C3%B6m%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=L%C3%A4ndstr%C3%B6m%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErik%2BL%25C3%25A4ndstr%25C3%25B6m%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Luciano Fernandez-Ricaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Marine Sciences, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luciano%2BFernandez-Ricaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandez-Ricaud%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuciano%2BFernandez-Ricaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Petra Kaferle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular and Biomedical Sciences, Jožef Stefan Institute, SI-1000 Ljubljana, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petra%2BKaferle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaferle%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetra%2BKaferle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Andreas Skyman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Earth and Space Sciences, Chalmers University of Technology, SE-412 96 Gothenburg, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BSkyman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Skyman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndreas%2BSkyman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Simon Stenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Animal and Aquacultural Sciences, Centre for Integrative Genetics (CIGENE), Norwegian University of Life Sciences (UMB), 1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BStenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stenberg%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BStenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Stig Omholt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biotechnology, Faculty of Natural Sciences and Technology, NTNU Norwegian University of Science and Technology, N-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stig%2BOmholt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Omholt%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStig%2BOmholt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Uroš Petrovič </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular and Biomedical Sciences, Jožef Stefan Institute, SI-1000 Ljubljana, Slovenia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Uro%C5%A1%2BPetrovi%C4%8D%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrovi%C4%8D%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUro%25C5%25A1%2BPetrovi%25C4%258D%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9743-9145" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Uroš Petrovič</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jonas Warringer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Molecular Biology, University of Gothenburg, 40530, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Animal and Aquacultural Sciences, Centre for Integrative Genetics (CIGENE), Norwegian University of Life Sciences (UMB), 1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonas%2BWarringer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Warringer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonas%2BWarringer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5269-971X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonas Warringer</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Anders Blomberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Marine Sciences, University of Gothenburg, 40530, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BBlomberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blomberg%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnders%2BBlomberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anders.blomberg@marine.gu.se" class="" data-icon-position="" data-hide-link-title="0">anders.blomberg@marine.gu.se</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3003&amp;rft.epage=3014&amp;rft.atitle=Scan-o-matic%3A%20High-Resolution%20Microbial%20Phenomics%20at%20a%20Massive%20Scale&amp;rft.volume=6&amp;rft.issue=9&amp;rft.date=2016-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zackrisson%2C+Martin&amp;rft.au=Hallin%2C+Johan&amp;rft.au=Ottosson%2C+Lars-G%C3%B6ran&amp;rft.au=Dahl%2C+Peter&amp;rft.au=Fernandez-Parada%2C+Esteban&amp;rft.au=L%C3%A4ndstr%C3%B6m%2C+Erik&amp;rft.au=Fernandez-Ricaud%2C+Luciano&amp;rft.au=Kaferle%2C+Petra&amp;rft.au=Skyman%2C+Andreas&amp;rft.au=Stenberg%2C+Simon&amp;rft.au=Omholt%2C+Stig&amp;rft.au=Petrovi%C4%8D%2C+Uro%C5%A1&amp;rft.au=Warringer%2C+Jonas&amp;rft.au=Blomberg%2C+Anders"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460300/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460300/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460300/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460300/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460300/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460300/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460300/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460300/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460300/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460300/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460300/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460300/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list"><div class="highwire-articles-in-toc mutant-screen-report"><h3 class="highwire-toc-heading">Mutant Screen Report</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495856" id="node495856" data-pisa="ggg;10/3/917" data-pisa-master="ggg;g3.119.400936" data-apath="/ggg/10/3/917.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/917" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Mutational Analysis of N-Ethyl-N-Nitrosourea (ENU) in the Fission Yeast <em>Schizosaccharomyces pombe</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495835" id="node495835" data-pisa="ggg;g3.119.400927v1" data-pisa-master="ggg;g3.119.400927" data-apath="/ggg/early/2020/02/21/g3.119.400927.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/02/21/g3.119.400927" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of Suppressors of <em>top-2 </em>Embryonic Lethality in <em>Caenorhabditis elegans</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495548" id="node495548" data-pisa="ggg;10/2/479" data-pisa-master="ggg;g3.119.400933" data-apath="/ggg/10/2/479.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/479" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of the Genetic Requirements for Zinc Tolerance and Toxicity in <em>Saccharomyces cerevisiae</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Mutant%20Screen%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Mutant Screen Report</span>
</a></div></div><div class="highwire-articles-in-toc multiparental-populations"><h3 class="highwire-toc-heading">Multiparental Populations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span>
</a></div></div></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/9/3003" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/9/3003.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/9/3003.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/9/3003.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/9/3003.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
