<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/9/2761" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/9/2761.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/9/2761.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/9/2761.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/9/2761/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/9/2761.atom" />
<meta name="HW.pisa" content="ggg;6/9/2761" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Covariance Between Genotypic Effects and its Use for Genomic Inference in Half-Sib Families" />
<meta name="DC.Identifier" content="10.1534/g3.116.032409" />
<meta name="DC.Date" content="2016-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Wittenburg et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In livestock, current statistical approaches utilize extensive molecular data, e.g. , single nucleotide polymorphisms (SNPs), to improve the genetic evaluation of individuals. The number of model parameters increases with the number of SNPs, so the multicollinearity between covariates can affect the results obtained using whole genome regression methods. In this study, dependencies between SNPs due to linkage and linkage disequilibrium among the chromosome segments were explicitly considered in methods used to estimate the effects of SNPs. The population structure affects the extent of such dependencies, so the covariance among SNP genotypes was derived for half-sib families, which are typical in livestock populations. Conditional on the SNP haplotypes of the common parent (sire), the theoretical covariance was determined using the haplotype frequencies of the population from which the individual parent (dam) was derived. The resulting covariance matrix was included in a statistical model for a trait of interest, and this covariance matrix was then used to specify prior assumptions for SNP effects in a Bayesian framework. The approach was applied to one family in simulated scenarios (few and many quantitative trait loci) and using semireal data obtained from dairy cattle to identify genome segments that affect performance traits, as well as to investigate the impact on predictive ability. Compared with a method that does not explicitly consider any of the relationship among predictor variables, the accuracy of genetic value prediction was improved by 10–22%. The results show that the inclusion of dependence is particularly important for genomic inference based on small sample sizes." />
<meta name="DC.Contributor" content="Dörte Wittenburg" />
<meta name="DC.Contributor" content="Friedrich Teuscher" />
<meta name="DC.Contributor" content="Jan Klosa" />
<meta name="DC.Contributor" content="Norbert Reinsch" />
<meta name="article:published_time" content="2016-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Covariance Between Genotypic Effects and its Use for Genomic Inference in Half-Sib Families" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In livestock, current statistical approaches utilize extensive molecular data, &lt;i&gt;e.g.&lt;/i&gt;, single nucleotide polymorphisms (SNPs), to improve the genetic evaluation of individuals. The number of model parameters increases with the number of SNPs, so the multicollinearity between covariates can affect the results obtained using whole genome regression methods. In this study, dependencies between SNPs due to linkage and linkage disequilibrium among the chromosome segments were explicitly considered in methods used to estimate the effects of SNPs. The population structure affects the extent of such dependencies, so the covariance among SNP genotypes was derived for half-sib families, which are typical in livestock populations. Conditional on the SNP haplotypes of the common parent (sire), the theoretical covariance was determined using the haplotype frequencies of the population from which the individual parent (dam) was derived. The resulting covariance matrix was included in a statistical model for a trait of interest, and this covariance matrix was then used to specify prior assumptions for SNP effects in a Bayesian framework. The approach was applied to one family in simulated scenarios (few and many quantitative trait loci) and using semireal data obtained from dairy cattle to identify genome segments that affect performance traits, as well as to investigate the impact on predictive ability. Compared with a method that does not explicitly consider any of the relationship among predictor variables, the accuracy of genetic value prediction was improved by 10–22%. The results show that the inclusion of dependence is particularly important for genomic inference based on small sample sizes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/09/01" />
<meta name="citation_mjid" content="ggg;6/9/2761" />
<meta name="citation_id" content="6/9/2761" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/9/2761" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/9/2761.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/9/2761.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/9/2761.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.032409" />
<meta name="citation_pmid" content="27402363" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2761" />
<meta name="citation_lastpage" content="2772" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Dörte Wittenburg" />
<meta name="citation_author_institution" content="Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany" />
<meta name="citation_author_email" content="wittenburg@fbn-dummerstorf.de" />
<meta name="citation_author" content="Friedrich Teuscher" />
<meta name="citation_author_institution" content="Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany" />
<meta name="citation_author" content="Jan Klosa" />
<meta name="citation_author_institution" content="Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany" />
<meta name="citation_author" content="Norbert Reinsch" />
<meta name="citation_author_institution" content="Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=J. Besag;citation_author=P. Green;citation_author=D. Higdon;citation_author=K. Mengersen;citation_title=Bayesian computation and stochastic systems.;citation_pages=3-66;citation_volume=10;citation_year=1995;citation_doi=10.1214/ss/1177010123" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Bonk;citation_author=M. Reichelt;citation_author=F. Teuscher;citation_author=D. Segelke;citation_author=N. Reinsch;citation_title=Mendelian sampling covariability of marker effects and genetic values.;citation_pages=1-11;citation_volume=48;citation_year=2016;citation_issue=1;citation_pmid=26743767;citation_doi=10.1186/s12711-015-0181-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. P. L. Calus;citation_author=T. H. E. Meuwissen;citation_author=A. P. W. de Roos;citation_author=R. F. Veerkamp;citation_title=Accuracy of Genomic Selection Using Different Methods to Define Haplotypes;citation_pages=553-561;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202394;citation_doi=10.1534/genetics.107.080838" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=B. C. D. Cuyabano;citation_author=G. Su;citation_author=M. S. Lund;citation_title=Genomic prediction of genetic merit using LD-based haplotypes in the Nordic Holstein population.;citation_pages=1171;citation_volume=15;citation_year=2014;citation_pmid=25539631;citation_doi=10.1186/1471-2164-15-1171" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. D. Daetwyler;citation_author=A. Capitan;citation_author=H. Pausch;citation_author=P. Stothard;citation_author=R. van Binsbergen;citation_title=Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle.;citation_pages=858-865;citation_volume=46;citation_year=2014;citation_issue=8;citation_pmid=25017103;citation_doi=10.1038/ng.3034" />
<meta name="citation_reference" content="citation_journal_title=Entropy (Basel);citation_author=C. A. De Braganca Pereira;citation_author=J. M. Stern;citation_title=Evidence and credibility: full Bayesian significance test for precise hypotheses.;citation_pages=99-110;citation_volume=1;citation_year=1999;citation_issue=4;citation_doi=10.3390/e1040099" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_issue=2;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=A. Dehman;citation_author=C. Ambroise;citation_author=P. Neuvial;citation_title=Performance of a blockwise approach in variable selection using linkage disequilibrium information.;citation_pages=148;citation_volume=16;citation_year=2015;citation_pmid=25951947;citation_doi=10.1186/s12859-015-0556-6" />
<meta name="citation_reference" content="citation_journal_title=Appl. Stat.;citation_author=L. Fahrmeir;citation_author=S. Lang;citation_title=Bayesian inference for generalized additive mixed models based on Markov random field priors.;citation_pages=201-220;citation_volume=50;citation_year=2001;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. Ferdosi;citation_author=B. Kinghorn;citation_author=J. van der Werf;citation_author=S. Lee;citation_author=C. Gondro;citation_title=hsphase: an R package for pedigree reconstruction, detection of recombination events, phasing and imputation of half-sib family groups.;citation_pages=172;citation_volume=15;citation_year=2014;citation_issue=1;citation_pmid=24906803;citation_doi=10.1186/1471-2105-15-172" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=E. I. George;citation_author=R. E. McCulloch;citation_title=Variable selection via Gibbs sampling.;citation_pages=881-889;citation_volume=88;citation_year=1993;citation_issue=423;citation_doi=10.2307/2290777" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: the Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_issue=3;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=M. Perez-Enciso;citation_author=M. A. Toro;citation_title=On Marker-Assisted Prediction of Genetic Value: Beyond the Ridge;citation_pages=347-365;citation_volume=163;citation_year=2003;citation_issue=1;citation_pmid=12586721" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_author=A. Stella;citation_title=Genomic-assisted prediction of genetic value with semiparametric procedures.;citation_volume=173;citation_year=2006;citation_issue=3;citation_pmid=16648593;citation_doi=10.1534/genetics.105.049510" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Gianola;citation_author=F. Hospital;citation_author=E. Verrier;citation_title=Contribution of an additive locus to genetic variance when inheritance is multi-factorial with implications on interpretation of GWAS.;citation_pages=1457-1472;citation_volume=126;citation_year=2013;citation_issue=6;citation_pmid=23508282;citation_doi=10.1007/s00122-013-2064-2" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci. Biotechnol.;citation_author=L. Gomez-Raya;citation_author=A. M. Hulse;citation_author=D. Thain;citation_author=W. M. Rauw;citation_title=Haplotype phasing after joint estimation of recombination and linkage disequilibrium in breeding populations.;citation_pages=30;citation_volume=4;citation_year=2013;citation_issue=1;citation_pmid=23916349;citation_doi=10.1186/2049-1891-4-30" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=H. Jorjani;citation_author=L. Klei;citation_author=U. Emanuelson;citation_title=A Simple Method for Weighted Bending of Genetic (Co)variance Matrices;citation_pages=677-679;citation_volume=86;citation_year=2003;citation_issue=2;citation_pmid=12647974;citation_doi=10.3168/jds.S0022-0302(03)73646-7" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=T. Knürr;citation_author=E. Läärä;citation_author=M. J. Sillanpää;citation_title=Impact of prior specifications in a shrinkage-inducing Bayesian model for quantitative trait mapping and genomic prediction.;citation_pages=24;citation_volume=45;citation_year=2013;citation_issue=1;citation_pmid=23834140;citation_doi=10.1186/1297-9686-45-24" />
<meta name="citation_reference" content="citation_journal_title=Comput. Stat.;citation_author=S. Lang;citation_author=E.-M. Fronk;citation_author=L. Fahrmeir;citation_title=Function estimation with locally adaptive dynamic models.;citation_pages=479-500;citation_volume=17;citation_year=2002;citation_issue=4;citation_doi=10.1007/s001800200121" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=A. Karlsen;citation_author=S. Lien;citation_author=I. Olsaker;citation_author=M. E. Goddard;citation_title=Fine Mapping of a Quantitative Trait Locus for Twinning Rate Using Combined Linkage and Linkage Disequilibrium Mapping;citation_pages=373-379;citation_volume=161;citation_year=2002;citation_issue=1;citation_pmid=12019251" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H.-P. Piepho;citation_title=Ridge Regression and Extensions for Genomewide Selection in Maize;citation_pages=1165-1176;citation_volume=49;citation_year=2009;citation_issue=4;citation_doi=10.2135/cropsci2008.10.0595" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=M. Pourahmadi;citation_title=Cholesky Decompositions and Estimation of A Covariance Matrix: Orthogonality of Variance Correlation Parameters;citation_pages=1006-1013;citation_volume=94;citation_year=2007;citation_issue=4;citation_doi=10.1093/biomet/asm073" />
<meta name="citation_reference" content="citation_title=Likelihood, Bayesian and MCMC Methods in Quantitative Genetics;citation_author=D. Sorensen;citation_author=D. Gianola;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=I. Stranden;citation_author=O. F. Christensen;citation_title=Allele coding in genomic evaluation.;citation_pages=25-25;citation_volume=43;citation_year=2011;citation_pmid=21703021;citation_doi=10.1186/1297-9686-43-25" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=R. Tibshirani;citation_author=M. Saunders;citation_author=S. Rosset;citation_author=J. Zhu;citation_author=K. Knight;citation_title=Sparsity and smoothness via the fused lasso.;citation_pages=91-108;citation_volume=67;citation_year=2005;citation_issue=1;citation_doi=10.1111/j.1467-9868.2005.00490.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=L. Tierney;citation_title=Markov chains for exploring posterior distributions.;citation_pages=1701-1728;citation_volume=22;citation_year=1994;citation_doi=10.1214/aos/1176325750" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=C. Wang;citation_author=J. Rutledge;citation_author=D. Gianola;citation_title=Bayesian analysis of mixed linear models via Gibbs sampling with an application to litter size in Iberian pigs.;citation_pages=91-115;citation_volume=26;citation_year=1994;citation_issue=2;citation_doi=10.1051/gse:19940202" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=D. Wittenburg;citation_author=N. Melzer;citation_author=L. Willmitzer;citation_author=J. Lisec;citation_author=U. Kesting;citation_title=Milk metabolites and their genetic variability.;citation_pages=2557-2569;citation_volume=96;citation_year=2013;citation_pmid=23403187;citation_doi=10.3168/jds.2012-5635" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=W. B. Wu;citation_author=M. Pourahmadi;citation_title=Nonparametric estimation of large covariance matrices of longitudinal data;citation_pages=831-844;citation_volume=90;citation_year=2003;citation_issue=4;citation_doi=10.1093/biomet/90.4.831" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_title=Estimating Polygenic Effects Using Markers of the Entire Genome;citation_pages=789-801;citation_volume=163;citation_year=2003;citation_issue=2;citation_pmid=12618414" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Yang;citation_author=R. J. Tempelman;citation_title=A Bayesian antedependence model for whole genome prediction.;citation_volume=190;citation_year=2012;citation_issue=4;citation_pmid=22135352;citation_doi=10.1534/genetics.111.131540" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=W. Yang;citation_author=C. Chen;citation_author=R. Tempelman;citation_title=Improving the computational efficiency of fully Bayes inference and assessing the effect of misspecification of hyperparameters in whole-genome prediction models.;citation_pages=13;citation_volume=47;citation_year=2015;citation_issue=1;citation_pmid=25885894;citation_doi=10.1186/s12711-015-0092-x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Covariance Between Genotypic Effects and its Use for Genomic Inference in Half-Sib Families" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="twitter:description" content="In livestock, current statistical approaches utilize extensive molecular data, e.g. , single nucleotide polymorphisms (SNPs), to improve the genetic evaluation of individuals. The number of model parameters increases with the number of SNPs, so the multicollinearity between covariates can affect the results obtained using whole genome regression methods. In this study, dependencies between SNPs due to linkage and linkage disequilibrium among the chromosome segments were explicitly considered in methods used to estimate the effects of SNPs. The population structure affects the extent of such dependencies, so the covariance among SNP genotypes was derived for half-sib families, which are typical in livestock populations. Conditional on the SNP haplotypes of the common parent (sire), the theoretical covariance was determined using the haplotype frequencies of the population from which the individual parent (dam) was derived. The resulting covariance matrix was included in a statistical model for a trait of interest, and this covariance matrix was then used to specify prior assumptions for SNP effects in a Bayesian framework. The approach was applied to one family in simulated scenarios (few and many quantitative trait loci) and using semireal data obtained from dairy cattle to identify genome segments that affect performance traits, as well as to investigate the impact on predictive ability. Compared with a method that does not explicitly consider any of the relationship among predictor variables, the accuracy of genetic value prediction was improved by 10–22%. The results show that the inclusion of dependence is particularly important for genomic inference based on small sample sizes." />
<meta name="og-title" property="og:title" content="Covariance Between Genotypic Effects and its Use for Genomic Inference in Half-Sib Families" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/9/2761" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In livestock, current statistical approaches utilize extensive molecular data, e.g. , single nucleotide polymorphisms (SNPs), to improve the genetic evaluation of individuals. The number of model parameters increases with the number of SNPs, so the multicollinearity between covariates can affect the results obtained using whole genome regression methods. In this study, dependencies between SNPs due to linkage and linkage disequilibrium among the chromosome segments were explicitly considered in methods used to estimate the effects of SNPs. The population structure affects the extent of such dependencies, so the covariance among SNP genotypes was derived for half-sib families, which are typical in livestock populations. Conditional on the SNP haplotypes of the common parent (sire), the theoretical covariance was determined using the haplotype frequencies of the population from which the individual parent (dam) was derived. The resulting covariance matrix was included in a statistical model for a trait of interest, and this covariance matrix was then used to specify prior assumptions for SNP effects in a Bayesian framework. The approach was applied to one family in simulated scenarios (few and many quantitative trait loci) and using semireal data obtained from dairy cattle to identify genome segments that affect performance traits, as well as to investigate the impact on predictive ability. Compared with a method that does not explicitly consider any of the relationship among predictor variables, the accuracy of genetic value prediction was improved by 10–22%. The results show that the inclusion of dependence is particularly important for genomic inference based on small sample sizes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
    <title>Covariance Between Genotypic Effects and its Use for Genomic Inference in Half-Sib Families | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"evhDDwJomC1v5rSNG9HMyaTEDCQVWyhHe4oZ0RPRtEY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460278","apath":"\/ggg\/6\/9\/2761.atom","pisa":"ggg;6\/9\/2761","ac":{"\/ggg\/6\/9\/2761.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/9\/2761.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2761"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2761"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/9\/2761":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KghXgFfb7uRWKLV85nzk3GzSN7KkTh\\\\\\\/bhlHhU9eNEBkcLZ34TaPbYLpZxE6hEiLyYTA8A14q65li8Sf9IDgLmTqnUpTioraf28tIBFJo3wTKmtOWP6n40e4ZPdVZTW\\\\\\\/B7XICJJAAbf44kDkP\\\\\\\/4obeewzdF1cghTAH7VHbV9BISbMcjU07VA83Z6ucTM158i0V0lAtngjxTzonTms\\\\\\\/3LJW7BFgAuJfR73zWb8646L26Hdv1w0AL4UhffW0FrHX6AP2PuzHMd1aSYyBg8q9Ml4wyDwgzjOM6sdUHbMWHIicqroQULLQDgZY2QZykP8S3l+VDel\\\\\\\/2iQZD4eWx4InO+ibS0V5wmZ9OHiA9gkiPWfbI4Ulw7H36fU7lN5u4+RKgopG1tcOs6pDvwyp31HVhOANonUL+B4gvkR0matm9QZPB5Obvwcu6w3NZenYlcLZ7wVNOhp7FbJYrQqO0yBC32dHgBwOnzOopf6lUf\\\\\\\/lPTweQdaNVzq0PAV4nt8lqpKyb9hePtGpKpeQ0lvxAsVS0kXEySoByGKWvjgZSXMIzyR6icjc\\\\\\\/a9LZne8vcmZ9XAAQS3B7s4KBd8hwLG4UDNx8I0vkKGUpedbo17vO1qLp0o57i6CyFLOewFgwLZ53z3UGDLpq9UddsihkXSj\\\\\\\/9ZkPtkHWIHhmPFX+Au062N77wRui4Igea+K91wdcXi9UohRlckJqROtQAA6X0UCKp9ZZg5rL+ULElAbmX2jgt6UrAAtO+J4HcnpTP03X8v85tbTUx1q6jzV8KLqqoy\\\\\\\/E+x7Ps\\\\\\\/XvKW4pVp\\\\\\\/kQmo9dlt1wrtysQThXP2E+p+2ycIfTdc+Jof\\\\\\\/v0TFLZF6sJ5hlzG\\\\\\\/2TQfpDivbX0BhIoy\\\\\\\/Tcal+Ig6gIp+DsQM+PWuU8+LXakoHS\\\\\\\/pzEY3gEGzx\\\\\\\/ydYGdPdWH8z8pI9lsSLIoWUVOp6FhLFFRT1IZDVLpzu0RD\\\\\\\/78GT4o64SGO6rte4a2XCjqJfjKgrof7JTGllQaZ+HelHEIhE4eeCgp0POPMTW644K\\\\\\\/WykFOdVM5Ub2pdk1M2FcyVPurrwEv8U18NjuZTATxhM+X\\\\\\\/SdUShrW6bhvzCfvRt+1HNDl9EDUmbnoUE11jdZXcXrHxLubkfTFfSeDvLbZaiYt+kggP0chyy0H4MWordJIWxxR16e5JzE8TSzGofFDEJw5AHyiPaZbr2uGBqwZcSL0GLw6MvusvZXW+lR\\\\\\\/xNyru89azFVrtLOA5VoBz9VQxdJAORCj3DijZO\\\\\\\/Klna\\\\\\\/l0sQe+hVwj3Zi0nAyl2MmFwYx\\\u0022,\\\u0022iv\\\u0022:\\\u0022inmSWDMFBkiJvzgbZOSdpg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226035b517016e2607960efd9a37637b76c05e0c35c3bedcf5d07efdb9891fb299\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224PVNui3bfWNo6ufEyfpFdQinYpPYmFdWiJu8uKZ97rKD2rke\\\\\\\/EY+AXNFcIZWu2AJfUk21FwjasNOH\\\\\\\/yMKQGGa1sdFD+OB5NRaMr7u2IBNeKUWZ4AgJ8JpRe7zLy1\\\\\\\/QSfBoZfAR0PjxMEEs\\\\\\\/PQ3CfOixx7dyerTnn02t8JtY7zjxRfUbsUYwEjYzR9QjrYXygek\\\\\\\/308TQna4GUImx1M1LjSoiRXC5E8+XhxVZFelcaYrtjO+Gk6zrBPmK8GXMYnrsNz+dtZVXuurPtQfbOqbrG0bLG1ZuWAkysgJxjVaUfhVWYVcTGwEzLzy4OWpsFnGf60pf+Jx4RZq7VgMlWkmX2lZdMNeLCcXxAzkkdtEPt+CBddOxWYkItVcVU44TIzuK\\\\\\\/TcHV9ihvz6rXLTQhwEqOUf2iCRZ5zwwKSZljTQHv3+PzFS3zhVZdG0xmdqUurhnznFzphjui7rkiQoAqglcj\\\\\\\/zbCmHN25qfpzC1SjuLRjYr6OwKKmiEkPD6YEksGcgAzhDcEpnXh53cTpzWuNnufyvk52jFNMSJPQivrDR6a23d8q4vP05TS6sJ4MnqXwk+amU2uUBfesL6tfwTVGYN5MDHSAK9mO+kc02hM1hveuJzIHoXFFlSxwVDkhaMBcoSdiHryFKTlU1IOXokdbOniuwkD0Y1fV2w64pyHE4OM9OgKd1NgLBrPqMEXystEDidnszRNB7BLLVwlErMBxEPZ0DbSJ+ufMsL9dyXe1PHnvMyk9DMFj60xhVRjywBbQ2due2inimYxshAA2IqYEtK0CvNh0SPFeHjWSBTpc8o3rZGr9b5PB3C683Lq9+xSYFtMyWYeWOLPklwuMfv5aF3SaPoGEeH7PlBXbp4nAznENuTiqwnUZqR2SCK4XXKdmaqsqnFD8gYv6WyqN3rSa+g6Ay7uN5wZdw\\\\\\\/Rfrz5yXt3G2LxDLZYXVL1A88Ti4TaDI7SncYqYXGnXB\\\\\\\/g7gaGtwLXHGqoHcrNA8uDKIRH6+DuAc0dNQqBjH3OHhqA5JHrCg26DMNsKeFvcGSK\\\\\\\/eHBSBAsE44sXDtwXTroFv1Rs9vsHFFyqx+hpaVT4egzhjUvJy56EppKjiW1mZ\\\\\\\/uMVmZV\\\\\\\/OVdiRhgDTH8Kbz759BKdsovf9jzgGZBYzH0Z9\\\\\\\/L4Nksa9X31WjdLsInvSgHY5gzMKzRS6r7igV9aKhabfShZxysYqupvZfkwbs8Pi1cmYb7CLwOWXKuXahr3OJOVQs2aSH8xVlDptKDTOfUfRy8eL5DMkaZPb60h4IyQCtrqN1mgtW1PmP4xAJvT+ORFp+eO0uZBvehGuHsJih4N6EgQh10om4bNHenLl57uZmqszoGsBmUzE725ymSxjosWwNLHDBJ1W4ED4q37XY3kFntvnhwhEK8AhqlHU4hz4toLNr1x48ncQ+YY8IrRZa0XHkrtwqzIYMpNj5VjalELhDyBSlBQe0j7\\\\\\\/0OlgdVMwo+0B\\\\\\\/uQIR7tszQecnj2Ec03aza5F2e7nES73nTvRgSqBOmUQR973WuL6xIbQz+k3Xnz5LWXTQqiyf5z\\\\\\\/DW0ah0JK8PFvQVMw23QJWSTncDLur\\\\\\\/qmOtNkEuTk9XK9jHEgfF5gwWND4hV8bgJYRSL5xS7mG0pu9y1O6bjG4mI7VsxDwwAuor6GOr87m9OG5deLbqnP+IYO+OLG3\\\\\\\/nxnGYbbx5V2nvtY69r+3QWQZVyF0D3qLSOmncAEB5Qh\\\\\\\/DXD45hgIDeH5GWqTPxdc9yzOxx2WfBIwwXpkvmCFiBJbvd0zD59GxXZeo0hwrbSF3zLrtL1irtrSryj8cS8VYQp3GnVewQXbUH6Nd9EpN7H3S9Nd+cRWldwo8xLy3j0N98BC93wq8Dzn81LItgZtYt+VwZfnaHEow5E5pi334ALREfkgiXciUia2DlJTe0W+t03d21xxAxoy0j90\\\\\\\/af\\\\\\\/ZELZj4cKlVUZM9ESvhdDv+F3jyPqPY+XXWaohGs1Sv29h8KRgss+SASLfnjJA27ngQEZRyeH3P5bnzh+afoi\\\\\\\/TveMJfCIFw0KXdzuFonUVcd7W4jRQOuRgPZ\\\\\\\/z\\\\\\\/vrr\\\\\\\/Kvl6h9Eh8lvZKfxlkI0hqnN19XZ9SKvIoBZLWOyFb4WDd12pvHnNKeW1ThIj287+e3FgHkHK+QG6PKoBHghvEVEQ6TkLdnMgrDao\\\\\\\/FVnum+fpd+xlt40t2d4GrzYZMMm7WbReTjy72RUHACM6XjJQpRkktakKiDK19ksdg2i\\\\\\\/jutYvVx5itrUpW6mD2\\\\\\\/1WsuAb\\\\\\\/i8EzB3EwUQmCJw21QK0JFLkA0OX+oqdGruT+RUGlGEpMLyyoH2xCY0Po4MDpg8whMT2ZfBESKUa6Li2e6TKuPJ3xLVoCIOveEGDBIuSh4tRSAiyAZwTmIoPX7URRcM\\\\\\\/cJmEaRwXKBtm1e8Q299RL9xVawCtyJ38qoVEMHjL+eWujLcsWF7TE4yLt4iojxOJjLpuPSsk6guwJ8uh4O97NzW2fYRU+qhaf8U+xlE8jtAk+1MEoVvdcnABYFVUmHCrKOKnCHaRtIydP0zg2HDqTx7q4XZKoasZF81Gr\\\\\\\/hrvdx+xd5DpYbI0qHVqSFsQWGmPervXQmHfakbvWdTqTqFMvLG0pW\\\\\\\/FBdU1E4+OZ2zbxyBOGpHWslHyMV\\\\\\\/Ge5slg1p633a5QsZ0ca+Pm\\\\\\\/5Tbh64kz4ixRtnHoVLsgBUeYS5FDfQwQ0CYug+VgtJsZvp8Pj0mmgQQiaMtqe5coaXjMZQ4xQiUAmqPUEb8Ks9p4BsRBElTfHJ+xRIqHo03BEB+UcjYbsnSxuUAzb4QZ\\\\\\\/bJ76C+IQyBZsnFZHfUJ5W3nMoZ4B5cgn2RgUohCNdJf4\\\\\\\/I0PJSaW6AWwAzmJv13L02r0MihJJIW1yhoUq7geiCfyAkUW73Wffi2sX4VjK01+ZI0pzLjHkicVCeO7Rr4oCbsRslVUetplgjRmbOqA1anPp01\\\\\\\/fsCW3bLUHIwXBP\\\\\\\/QFfNsjCLo2wtybBtaNEjJ7T\\\\\\\/A\\\\\\\/jIqwtjOba1oua\\\\\\\/oVi058JrwS7Dft26Bfu9xKr2dP\\\\\\\/emI\\\\\\\/BuVN3m+cWPMKqNZ4csPXZoPkukGWXacVEAmgZjBZQHOVEQGg7XKJfxIlHIiq\\\\\\\/8Lv8Z9p8\\\\\\\/XDiqgu\\\\\\\/U48vDH+7\\\\\\\/bzEaMh4UlwbcuNJ5E\\\\\\\/lqaTRzC7Wr24f974sUAySwPyy9UQ8kO05iY7NzDZ1oafnPL3jCQ3Dl+IZT4xz\\\\\\\/WvjXa\\\\\\\/42kQm4mlq0kJF8+KjitpXJZ8uklaAFg1ooTnpdU\\\\\\\/Tv0hC5fV6GW8lC\\\\\\\/CUgSvdySHEbBN9vOjKyiIlsZlyM9sI7QdDr4EvaqpZalRk1Yhp6uRYDAjQ6fUWyijYhcy8obLq0D8LQh2ndUmwpUuUxegHJWPsy9Wfy3\\\\\\\/7VgvUWeEx6cAzr4hCdKIA7pidWtRDYuBKb78z\\\\\\\/qFbOppgjIv07yqY6bAKCTdYZ7AEq9HsHk\\\\\\\/zcppmfC0LqhKuloYG15lt9FGldYZccwmHobaGaPEqjTFPTy\\\\\\\/UCCeszzE8iiGQQ\\\\\\\/FZPNPhIzROIAQ7DJtfQ1J7vmILf4WXRtEE6s95OIsFD3K+P1Z2sjoL3sW0h905MtZrDEZe4gEP\\\\\\\/z4DkYxG2KAi2ZTxcIHf2AUGUkYT1vZPMX\\\\\\\/o33P0gGYWRCD6EGaPo0akKKqwNdHPf981ksMrR1QgT7KMDf66oTFLcirqn6ZdM801GDTSdGo46N8vsawuZscXzHc3eudhshcuWQWFGszq4Xb7jpbLTs04vRXbRJEp1\\\\\\\/R7g3NI1ZT\\\\\\\/0z7PuenlxUDVOVxLUoIURJF560CK\\\\\\\/kGn7ghQvlJ42xyjfoJz530MFqETsUmjpE3tSO+J5YczA+36dELu2INvmTIcM59HV3Tul0\\\\\\\/qD4IR8SnkKbjaH8Hnyky3tkVrKPdQoXy5TwlTqhykeo5ghNW48Yca40Bz7d\\\\\\\/i30E\\\\\\\/eFsvUPNm8bLSDnfCZ+WbWb00fuiBNe5Hr8Xz45lWe4IF2x2+NGNq+VFDXScRlYjI2fEpQ9m2QnTdtMfXQBhd1roh+O5YXiOrUAPj0ckxsB2+NztNPoJR7BHmDDAhugxyktPKbLYGy2u98MMi23kIA0gyQ7cE\\\\\\\/6zHG6VECXEjoGmtd2QJeGHUa7Nwf8giobiBCQA9uvg5ZRsY5FOsobiH\\\\\\\/ZBVGBnD4eYVl8CqT8Iggvz7c71Bafh2aPENRuUkznIiC9O01GGDzhlJmrUA+b6umN3X22o4Az7kSpmbkcJODi6pXPUcTyBp3vKpTDFGT0ROpBsn10r\\\\\\\/p28cJ6QDpUZbC9roENCovZLvUVIatF0FaN9d\\\\\\\/NJxOElHwsb6j3GJ734BWpZ6714nzbqa+tcYV4hRY+hx1Ub504jsy5+\\\\\\\/vpmcO9\\\\\\\/pZuP0NLyCTiXGiorH9WwRsmEc5+b98ig5oHA87FGfpvNNenGpqzLgFhyPIPLd6l0DYYwVkgWXbMcMt9nwxsIh0+7rhu+0czfn7u9yazKAQr2DVygrRA065clhOIKIjRRFlPllOmrdUkzS2Puds5wdj13IL5pLYjIy+Pr7eniTi1nJQfMnfjYYqz99MnwIDcPvikdnokj\\\\\\\/TrpAoFWHudEi1su2gIUgnDTrnZrQ2KJ\\\\\\\/APIsIrKfWzE13KAc9gXDV5WaqEU9Ek4ZrS6PMex6SPWAx4bOMLAP6aay8XeVGIg6tmqB1CHcAVn+W+ztlAEpAcGuXC3NBQNO5YBRN6eRrrvjuxKb1rQBx0oRTJMraPFjTT1a9fpIO5+\\\\\\\/pNcbk8SiOK2yZW51phPp3IdOuTc\\\\\\\/+kkqMGzpY\\\\\\\/erXP7tzWcBtbSDw\\\\\\\/wL\\\\\\\/IitHvxNNnLeMzp4ioh\\\\\\\/E62aBZz\\\\\\\/WpP24j8h8u1Ji9x8pfkvwEEY1i3j97eCx8jD8VrCOcUF6\\\\\\\/4+rRhrIAKkMObsvS3Pq2H5UVXnyZMpBHpe1rM4HyRNRTwikKNne2ILJmcfPaMtogGpw5wv9K17ZPUdZ+kIu+LZ0rQP+J1Kv+tMsX19yLXl7Jktuzw6ZqNBBEENZzrwzYB6ZATFDlRhYmGAwqWd4NV7u\\\\\\\/kVgehqbRqcb8xYwndj9e0hTgsN7yCD9r4wh2mG9ND3XP8C9MEBPwy3TEmXOiy56XbJw7flac5Z3Dqb7ng09Bxh9\\\\\\\/MD8A6TrMN8ngMdO1JHsQavo5NTaAiWrDUO+QnwEwKmQHUlNlmIO1rbywdR850RanfWy4P04OvUe+RcJuGP6JZYOfZa7eCrWRy71e\\\\\\\/40c0lHk\\\\\\\/sm2U92tOlpatgEur3tyW0wyxFShqE+HWJpJiLL6QS4fwBBnWcKayBU3JjsDuzSc3pTsFRUGST31Xbh1ssr05PBb3zMHg6PJ\\\\\\\/3+C5oUJK5zcFhk9tGY5KnfmmuB4fq9BZGrV9PlePYo0yv6oIxX28nZBcvGmZSdFSeEzBqqjcjFXu3vDKdA8ASheazCg64r73zCwxtswDDtDc8VHu2EXduTcAsgQxrEH02SopvtNjGIjnLbYI7wr+rqHY39gSYvbOtbND2UaIBGLR7UZ7bPG89CsWtfrmiE7zQpo1+K8YpkLDsFU7y9tOlPTLREugtKzci+eWRVYVbnbGjvoJEvOyIS\\\\\\\/HqoK4kePU\\\\\\\/Wc0HT77VFR4CYeBxAQzFZ3Lc5KxLWXSzIDwagqu+zHY7fg9Qr8v7GAMuLEuxvFg80qOPhePTjRkd9mhwJNMugyNZ8F8Bo3vJvNDTOa5TeBLWpLxN0jD\\\\\\\/RXX55npDqff5MH1ZjH48LGFb3oDPS5kuxi5HdxzHz5crhd\\\\\\\/PNCZq0tS4BIvy3XnxnfN00i3gqMriTjPNm4GS7p+fdzxKJe0u6Ycp04CU1QC7H6GnLotKmlQNW0xy+Bz4EPgXNmljvgEVFH70pliByG1wQ9M4zl\\\\\\\/X7cFPTASGjq2rY3c8G36TKtezlZ+ste20nNmhcwOWxVftmjbVkOdwmz2Q3JSiwcA6qjfHoo+mROpRvauY4uO4nupWSt7FjjhU2LtdhxGtTlgtnjZ\\\\\\\/fOjnBIRGFCmVqe\\\\\\\/8E3xizkNNMWddaUWNrmjyRhycSFVpYa1TcfGqXf2hLQhgQGobO6fpAGdmJgXqQdpb+LubSlduMh7BtuaD1aPUKP9I3Ip+LpwEBQGxPJlkMmBD9S8+UAy\\\\\\\/I2rrzT7VeTQafGf80qrawmMyaKx0X5vby4j\\\\\\\/zU1kBMWhcfizevNUyORp+AtqSHtRk88c05jG3NdZG+M9iUQ1nj7GUzWv4ANqpPUVBBLNSMjYsOeemlkqizuSLJhVd84Rxrdln9AuGuoDtj2sfAzV+zVSsOBnx+43Z16ahKH4fJZKVOTRrV+sCDN5nA2vPWrVA0yXA+pBBDJr5a+TqFtVl64zed31i6FSMqhUfd+YZbKAjTQ3mZ8ys2jRnrBG+tixseKcvgtKbf1BLarYtH4Ae8B\\\\\\\/\\\\\\\/\\\\\\\/qqgh6Z3RYioADbYyzRxhVNm1m5sO5mZflkcV8QblmhOgKGti\\\\\\\/OH58Iz7I6XaCZkq4I3DrH5O98tVe6gqHKhtlJd55wFKBKm6iSv1K0C8SPvWzZJq9bW5+xFicPgcu0aa4Lz16fcthsrvCvCTE8D17N0rs6A+F09KOGByMBUJS9v3Fu7S\\\\\\\/zsd3IFYcVysWOZAK5MAwtmqTHo3cEPRODLdoyA21tMuIedO0gAiyHvGmfsJh86vmwjFji8ZfvNrFEHSyfhBUSfrNSfVYsyTbGkEhynqwfgyktMQdeJNLtBQ9yQKfglru0iSgwp3KHqg4cSdKioXQWPd4kaIR4Wfwlo3pDSw0L1RaASG7Wd8\\\\\\\/4NS+VNvDWgruBsDNaJ74jFXBVyvp+BdVDFrr5bzFmYDtJXkbPjcYJI9qtWaJU1TEUYrkZXfM2qkHsiHoEheI9ZIMTtSsxZ8ejxavQQXQ1PQlWQUPPvNdlJhc\\\\\\\/GiYpEKuTgjsZYGrWSc1vAmJBISBuPwevXheyppS7JJ2FmxKBaGMGicsrFqY2gsSwnU8+EEjn9yrJwuv1OQn3P04LEWZ7wHiAlXVHPiuQTmmqHeJdHfox8BE3zx89\\\\\\\/h7KYdiTOPOCvOP\\\\\\\/2\\\\\\\/xipFU4YXAN4dOBcdHWpcK0LIGobZCj4sl8EAS+MHg8tR4M7V3LU4BtzKtopiIzNXl8j7AECYsLRMnul4dgQYDOxWK90OngjPOLnBtKA1902fgNh8hHd\\\\\\\/HpWlKEitgxg\\\\\\\/3KeXdvKITVJPyaoF6+0MGChj8ap4ADmoLlOiju4u\\\\\\\/pOW0XTLwqsAHHIo4TQUsEqBlhqVvpsqWXuZsg7K9oig81jLAxSYBWmlLHsOyTnHA\\\\\\\/hMqC5v+M5O+C49djGQt3WldUdoQicCdKEkJgSfZiY73vITqg0rYIoJVzE9JlFtdzmU2JH9yHruI80kaUKhy\\\\\\\/CVLIBF8jG2W2fp\\\\\\\/cxeAkecowd16+HCuz+UJxgdweb329wF4Ne\\\\\\\/r2EpH\\\\\\\/fBV\\\\\\\/PiZfIrPQ1yjqTGto1ptPfIkXVBPOXX3uBZFs05aefbG6p3M7NnfBEBEKqhr4Ka5zHKshJsk+GCI1bSnwmE\\\\\\\/ujVdaR9j1j2tPRvZ2AUOoW998hqwFnv37ASZqr4EpuCIRoOGvijwUlmXo5jrqVsERLlLnGXTMJbwWfZBaSom8a\\\\\\\/kN1IzaUL4JzaNzJJZECWqjgaoJ1qxVrvqsSBDFgVOwfOAA9890zBa+4Kkmyj0xYHm5+gpYXZGsub41Z8bqqeA9Za5zSV275IGWMx\\\\\\\/jqlmmzPsvLLuT85SkwYZdknUU8SVQDBlH0V56wUk7zk1W\\\\\\\/YJpUPJntZ1L5Ins7jMYb56dcZ0dqYw3EThz9LbkdOK7h6xqM0xdzSn4DYRjdKTgY0GDMFp4\\\\\\\/5jng0t4REHzQsntsR4iMwYMqgcMQTiL+vIGJpbFrPrtvMH\\\\\\\/UowwVzSxUf3Aea8l1279C+C2xzoTfBzVQM\\\\\\\/38mYj+vKNd+uxXzrfbpEArKk8aXWQ6JhMDdWDJvqbsDZI4PQ+waJfJPV6FFPNX1cndNkYseVBMNkHt7pvbi\\\\\\\/LLV3lhzI3FPAjhOghZi7b\\\\\\\/JTpGmXGfYIpokxUYAeVdPVDou30PEH3yVOvRN6flwWDFmEkUs9G0L7a5RgsYIqPpIxW\\\\\\\/2lfa5X+1si\\\\\\\/C7qxS3noOR23fAf7EK3L\\\\\\\/5GONQn8ROt6rllQizj2sytMnO2W3FMLCMacgeuu5g+oh8ki6Juw4zc0etAERi2XBwZCTDoyN\\\\\\\/5cT7g2h\\\\\\\/o\\\\\\\/DQtMy+RvWN5OJyE69AkeySQ2KndIqwLblFTBN1iiHqM2KqM8d+eFQiRMQvoTzJQJcOh6du5PWJrv0yHtIewtwpPUsC24ysE92p3ieovUaRCYDBvPf62+IIH2oAywYnyIRCV1dUqbO8GpfBI86AHIccWs0oV8HJwehI+R\\\\\\\/jX+LOIc\\\\\\\/cl9IV3hxvqp4CFhEHUyYlHVvkTzBwy9f6USu5naOD\\\\\\\/Zf7WZbjBO9xLzIAgdAtiSBqlmNVd1npzNSSOW9PO6P1IIF5kU2tPDONbCstfNlqnIak4rmmYtiBaSYKXiTV0ssRfToL8eKj9EmN2xIwq42f\\\\\\\/rjparDXtWrWeRiY2Vn8S6nufdouPvp\\\\\\\/hNUCutcOc+SwVlAwXqm\\\\\\\/GguBvuNy6xZQrrTc5rOqf34u9l3wfudhlFoQtDLuUjj7JwjJJx+VW1gDbivFerYfukLlsWqE9bcsaxxFXaSdV1RuGmyOMpudv2BZYI1RzYEdycjXRejU\\\\\\\/Vl3E3KKM3vsxDqj90CX5d5+L1mxPDIcGoyEH3Bva+TBB5KubKkPudprevTZf4WWXWyD43G8QyaxlrZE8BFl4kKsRBIQyT72lc76CQ+HzXdA62GsydWF7\\\\\\\/Yk2xUNMw+nG035wWm4sCTCh23ryqID5hyvR3TpPhx\\\\\\\/eTTEmu6HmO5mwW7YmZfrrPswQcIHerY49JH62Z8ZVrG+UK96Zan9efGkIhE\\\\\\\/k0FUf73xkoRepCIWWTDgzALlDBfRRcynAOfOMM3HDZ4S\\\\\\\/c89y0BqtF+lcxxFhTxXx4RGhISnD+B4w3lC7j+Qqvix6tgquu1nthk2tQft5cvQLlp+Sfk90wbXeOEaJ0uDmXXrjxbqhLurWYTjX7v6v2Fr14otBWPcP6deczCihSooqvAbTStD9jOw4JQIfvimRMgb5vqVN1vN81TamNowGIAoGC8UkDk0yNoWM7IKwFZpKjPPpjU5MFpVrUBwCl9dB04kguLml0WhrkeWq1rvrsH\\\\\\\/hM037npeyEyOzwcot603gCMhh+rVKid3TVU2X9icgfb0V2KXJ82bOLEQUguplI0Wi6qvcADCysXiOol9NEM6Zf0P+dfEtaby5ZuuO8rN5gKkdOCIFMhxgGA+ZI\\\\\\\/do7r2lJusqklOp4NPEKEAAZmBfLMc6uADTnZuW6XEnPGtTSoaRh+\\\\\\\/CLbEUNvj2i4Xs5F+amUZbUNpoOBlfQdc2j7Pnv4xvEl4DWcrGm+HiiiA0eGkQUvduoC860qjtYP2lO1F5b\\\\\\\/2noXoJs9bc7yjo2pvXHLoZHCFrFt1dbiBYa0+1roS61kohOumNoCv0UdJDHyBRpE+7MyFlWFRc8AoIbaJXo6nAMb0VNvOun0FeT38fZsq4sGlZRpmpOguE9mQXhzrDvlBzqlThjfAqCVqWJNVSId\\\\\\\/0QZgbuvW+7V+NsmhlMnQBSHmTGxSYLkNsdlQb5LNmqlk49u14uGfTV98D4BCRjbMcDS4AlScqas8z8ZYtdzl74sU80AwMmZGYAgMhPkrWHMFFhNsrzblsRydYUHbRVLWV5qeh2DxdeMJk12aIl9HSxoPHF8MEMXk\\\\\\\/Kcvqqqzacz1yUOTRIQp3qApuwYpbaiD3gJK8vq5TpVxXiAZounkdnYaYGtH8\\\\\\\/PCIsjkoFHSD\\\\\\\/tlrXGprqukmpt1hIyDtPdfWImcDlWZ7YkkGRzSCvlBdHVPllXhFF83oCGA72uu\\\\\\\/2gQaNXmGt+unDNJ6VifdENn26HGbIDnQ8Td+wrVlev3ABKgwXFQkkGXOFfYtRLhX1yzEyIvLcHVfgIsnnd1bpiz6PJXiaFx8hW3D1uv27Bk0hiwWPWpYjaFB8oMFD\\\\\\\/9s2gHu5dvwNQ\\\\\\\/opvGFmEeV+CAUzM4knc8Cj77PjQVfW8jxCLqW7SVk7U7hHJJVQHlhhvUUGwfDZ71m1IhtRD\\\\\\\/rDttwJAlgnglWGPqpVIJGLjVsAcofMXEQ0b9z5gkGgimAtdEVkYHn7yowOiNdw5HQEFzMYuEceppBb3m8S4wGlVVI3f1bZPUpjnxaIJQbl2du1z8shKFcHlvlAx2ZCZUtrDrT9sloCrIwMJ8OjBYfTZ+T22AQclGa3MSwIyDU44mN0UTQo1oxqPmJm4FgUjW7LzbnbKADo8KbYch372HPk34xz20yIVpqTySZ+1hF42rGf\\\\\\\/BOoRB5Tn5tiHdqcWcvZUPrmkRIrgSY4Y0jBZv8YpJn8YWxbC8nId57xFbw0v6tj8Z3OtPIXtqRqAbrcAifk2\\\\\\\/BQd4phEoQ6okonKmCj6O2Eu0nmNdvhgkCobHLdOgc+TFCwlQBcllpU0\\\\\\\/w1y0wzlD4dSMuxMjYS7SWiPEERG4UCow5W4KjGmO+BV1tJmtQPPZf6+G96fpLlGjimlS4Ztqt1NTBD6BQuOHCrvd1kaFXzoq6NLizkvgkDinEDpfN7y7fjKlZoHNYd6\\\\\\\/EWpGNCr3aorLDHzjeF4CJQFjiwmsKqb\\\\\\\/lQSgI\\\\\\\/CMa3zT92D4Z3UkTdslw5UJju9UJcSHnTTunhJHPAAUXeT2cG\\\\\\\/X0N+BvvhiW4b\\\\\\\/uxnyT9v4+vRbFjKhS9cCDmUaqQDzsjem1S8N+Wik9E7iLFmkTPjqh4llnpBgngNvcg7tyuZF70Gi4KdH49+sVyilHOZq6qeW92S9n2enAsmsgAdoqceojrf0bHcxUdI3Rv554tqTuEgbDXNhxEOsUfDd05HQK1otRJ+SNrCWrco\\\\\\\/\\\\\\\/\\\\\\\/VRgO3BijpAod+vjb5RRl62bsD+BCoS5+uZ05FitbDvlnb\\\\\\\/joI8I31zFZoo8rSYVhuDiZH9iBQTGBVq2GKgyl0DuSUOpWyMu88iGZDsNWPafMV8t5mJbzmlwRSQQTjpKCVjVs0Pk2UAEtluoywYa1xdGNXa50o87O+85eBFTJA6KOpSlfje5AbG598VnhUsQOassWTtRHLIEV3sSpi+\\\\\\\/v\\\\\\\/gL8St5mKDD+ho3fdGzNkUFV9xVI\\\\\\\/lgNuEbafd2psa1OzbZQ28GaSnhYbuNSjGGeh8M1+LyDmxLHi4PwFwxn\\\\\\\/fOshiWhrST\\\\\\\/1KfAcRENRo6bGFh\\\\\\\/cf\\\\\\\/thgyA1i7OetDS\\\\\\\/0jW7T1GH6FkJa179zmaI2GtLdE+tDB8Q15\\\\\\\/oiP52zbJGitC\\\\\\\/9a+R574CALcKkUogKsm8kSkbOpjPAqVUS6TxqoEhWuhVSf\\\\\\\/4LHgcEh9opBg59fTKGRcNPCfUaO7KI45VOo38c8kCuW5Y6uUfNipxCGLXg0oqhBxcvG7Fp23oMcz6KMYFzWcXxXh1HM9FlIXac9ND6mwMdhvuDP6xLiNWjTYy8T6TvHUQc5L4gYSmfuLH4KrPvTtxgVzcgPXDNHp+64ESFKriOJIDdbZNFCoWNYHVOsd44VDAmpz2RwuvM3fBChjkgN59F4t3izuUc5Xrh0WTfkZpxy+5ZuIAXw0lSL1Xu53+SDaU6eN38hZCsGNZtAm63XgwAqzGy0725pGRnPlXopqu5AFsDoe3j5+Mfm4xmpmE6MnhKMAcmV+lf\\\\\\\/VB3EnW02rvcXiwP5ps0Iu5CtLqiiEK4rKHphssODqCljuWv8YlpusWuvutmSrT968alCbPQ6iAAg9FI2Bom3q\\\\\\\/uChxhc28Nv4P66CPvVVIypNkh\\\\\\\/p3tUleM6TYgXMfyjp+kQwKpUv0gmAYp667dPImpm7iNT7KeeuSCGVpDYOR9\\\\\\\/Fb6lG4hHeXbuJq9uC+rCmqY6Mtj2zxTHplsKnjPhRt+UPzgsN6pZCkgVv6gnLOHoDCG++YNZe2rEvFCtOYgipg5pwxrcleGuQJZtRwwui\\\\\\\/MaJTo2mNZGaY750YxvHFD\\\\\\\/T2eIW9dvuU0JMk9VKY9DasKveBdJMH\\\\\\\/\\\\\\\/puxTwnPa47nB6tn1UFXGe4f6ijmb9gL1BeJhDIi+xyHE9Ebh3WMSP25hZA4W\\\\\\\/cLFtc8ZBzy3NTbXabbQI+qnimsH4IJHmWIQ4kiZo3dJnN\\\\\\\/dnc2oKGHp5jIvIz7eN\\\\\\\/ca0u8mDdYRMWKwXJCwhs55ag9SaVFxeBk8V0V7CLpz4GOohK9vOcRZ+FzdKG0KxwI3bWudeh1lTgFnPNVEuNecXfyTfcHkpyUaeW6zD3bCpPcCOsugnvhsvSKSwaX7qGZ+5UeBsgCgyOB+vUKCCxhFyR97MuteOsua\\\\\\\/ROju2eN6LGInYfhm1TWzomyLcQqrvYh4yx5zIXPAgKSYxkOWZDHVZTKkjVD1jAsekPNaKm4vx5B4ti1NXiUTg3Vt\\\\\\\/XF0QL9f9Fnk3dtspbfr34iQd7SMvW291MrTEAqY7ncd6Z7EgicSpFdSAcpohmgxzjFeMSSjVMjlJzd\\\\\\\/mp1bdRrciRsOHlv3savL\\\\\\\/fDLya\\\\\\\/lJq5xZQhYa1hyl76GH0AzcGYj8nJrG6DeEiMd92JR6PgMGsO32xO8HMwjnXEXrQLTSrFxg6vqKS+DY7F2VlgmUVTde5AmnNYoY+pXpzXP+LavI1wQ1027+Y6jUbfKPEYZFVKUOqkpnb370KL7cU2jSLdX0JoNA3U9EFzO79mVgyhbxRwouqGXLNMUFzKpFacBwUD0\\\\\\\/Kwx4HWzsnn3va2+9OrAOKBAJW6oUbyFwjTfADexWHDB5RNCojjdjlUAVGg5RNrbxkzlnOBjRNKbhVVQJvDajrYL5liOMT8MUtV1p2wdyuSaX1fWQsgnQEH2AA6TpR8KvV2NHMNd9Xdp94SHYzE4PX\\\\\\\/Lk\\\\\\\/Skrl6cRpEhCljvwH64ffaGG7JCg+Z7wbX7lAfKbfaehwvch7eIz1X6PD4OWy0pYiD\\\\\\\/fL+CJZyqaoAitQLaSeq1FDLXu0stYlMnnJpChrfHX+R4vC9ZAeWDlrCW4uASRb4JdK2lBe59xvyNltLcQlWd6ej4fFQsWN0RUaJ4WYogGT\\\\\\\/00zgnuCPP41bnj9uejS+hvBE2W9lwa6d7IKjF4W9zHYsU8Zja7fAxgn6LNlsY5x9qlbThsx7t\\\\\\\/d9o0jLkc+CHjSR5HCmt\\\\\\\/gxTCv05D7SR0ON3vbqJFPqT9Vi6SFA3rRis\\\\\\\/7vfSXrSVH3X8lQHupaV5W9W8z1GaqlgZLVZRAg3sXrB\\\\\\\/QX9Qw0taiS4l2TyuFn+9s+JPwjl5zkgKqpc2luak\\\\\\\/5EtfoBb7aLefVssqfwb27eG5911ZaNN2wgh7rHJKs+T12g3sqQ04wNF6v04qwBdcxEOP9+s8dCMxQz2taqBD49\\\\\\\/KqLfDbD75+QmP2RPcCGHKG5Q7MQusx2+WoJYpIYYM4g+QDYjA15Bg9UzGX4yiPoYEidQnyjI3rbDItZjwSQKoohZxOQ++TZkZphu01xGkK2Kn+ETFfb\\\\\\\/Yw3FtG3Cr3ZCkEWWJ2JOw7L4AZt7ZfNYjgfRFu2I\\\\\\\/CD+q19KdPa3JfEkx4H2uoB+MOAw4EUVrNgfeXC8ZFgFRgkPJg+ElnCuYK6opsNWEkwSI1ZuPUzOQm+15QQ3mX6Hzo41wu\\\\\\\/IWkoIAe4s3Yw2KZAC3RUMRlO+c0LooZd+ymRMEPwkwDJrwS+ZfIi7lBuWq4m2q3Tt8LkDBxXYemufUe8G+Sl93zvGyAZbD5\\\\\\\/Cft9ERAcbmFbwMoDzPe3+xP4LQ+ZPmUIdGIp61EJUn55bJA9QfdeTYIMzu5SaXkqX0xXoGSFRM\\\\\\\/wnWvJmJ6ChgEqw3+pRzw7VPGjR1dkdzVLi4FM\\\\\\\/E4GVDOhbMP7h2ltLSe50Vmrexb6TJU5eM\\\\\\\/oRO\\\\\\\/fwaO9pXDnU\\\\\\\/pkKXcQqBtkHEuA0R5i3TzUbVIPQMpyaKgwyKuIN4jb+MrVSjCO4c2ckv5znX4uPUygD88kGE4nhZm02yD3xkh66XXfhe39x3cHFsNl1ZuHRpcam+qAYmFJopnezeWLLeBy\\\\\\\/m3XVa36s1m55WiOSCzMjSjp3INgTSdQ1ub\\\\\\\/O\\\\\\\/WYoEcsA9gR8nFB7daK7W\\\\\\\/2fbroGE0xjwU3Ap9TpPKfueYxMk9yPwYo1K+Gm3dupw\\\\\\\/tqnK7HfyzRRTIxBtgcLE5m3iDmHWA8GJtIJDq\\\\\\\/D+DE3QH8DyOA9W6dcaU18U26dGCfR4CxPpBxv1XYgSglPE\\\\\\\/mm0pthNgGOEBw6MxqBusDrB5kMCmi+0ZdI3nRdxcMd8+qk0Qe3p7AO1i+iZrE4uCySidl41SdJGQWir\\\\\\\/cpxe+rGMAHv\\\\\\\/33sEdX8AWutL+xqW8aFg0J19ieANiHGSnlKSWzbIXKKiiw7HaJOlaoeyGJ2Zy88+aBdkFKWB4IETqVyZz8xviLazu1YCxcyLYEP0cMzvq9HkeHc+aoP1AIMUN4rRtvDV8R8iN7CeCS93DEvtDO98jenY7N7Df4uKLR5Ydn8Jdgv+JvnVKu5tpZHxVlSydmOj5+vHiTKe8FIqp\\\\\\\/6GYL54o7uZhSbsY\\\\\\\/\\\\\\\/WLrMsc9lnP3VTyuvmil5aWUkapold1c1b0sLqyND4uppWYkdPPapxxl6YkFr3Foe2IaQpklwsxZSA6MsM4QWA3yno2fPzB1pU40JKpXZpSC31AvLTEuruSFOgh5UdDJpsIufPJsfn6ELrF5KbVJqu3nmdMxOjjUI8C\\\\\\\/Va\\\\\\\/qSESGPkLOf5x9fgO28I4Ygf\\\\\\\/BDdkALYsHlxnFCte6D24dHpcrkH\\\\\\\/wgR8RtQtZAWoiLsJrBaW1x3\\\\\\\/uHTtvhuzG1vNHa4b\\\\\\\/AC1HXwbM4xIBX+ts4fviZ3cBXUt0xbSIVoaDsryXus6webjlqgoK0Vl61LR2y8P+dixl7FQ3RRdL00fV958+YHuPJIdbK8VXjkUdsaQhLUmVbrNErMQP\\\\\\\/fGudFxTxWy8daW0PqVVffriZUqkoCucbHNGcMyY14RgsIHqNBpDcOcv6nYbPfuadDoBFOU52rYXjVC2oKxpPzPM03Sv66fm1Q\\\\\\\/ZJ7tZXbFOVavUapTzyvZ3Ug0TDmRLmQSuW5qz+lrjIMZ2yqhERb1FzA5oq1PeMUkN+Go1IRgMEIV7UPLytIRxpTwkLqFTHCwXmNp26YlLO9uzaaMRXgAl2gq5KIjHRiNpj4ZhFugPrIZIdUqZcXJ7E64U5JWvblxHMoqpT9+CTHfAFnqPhumSsTfrlToZVbpSt3gu+b9sCgsWOKjhRQ5zEhYc2Azs5PtU6US8NSn\\\\\\\/5YzuK6AISq\\\\\\\/KfTg44WLXwNAgCKTTtxJYy8+H5nSTfbVz9keN2f8TYtH+9d1wZJygI90LbQ\\\\\\\/nz8MmQpC83WMeljpEdWZfpi4eijFTGVINhBITsHIw8G+8I1Ntp+zJlcDJQ0xeDF1fFWCoAGPVD92XQiN8u\\\\\\\/ZIE0nm01T9RYoL51ws2yWkURwTNab9n9GHtNcfikvmqrXdKsfXKHL4HZzljogI1KFek9gJTVQ00XIi73ORyOWd4SWZiwUq8TCs4j3EJO2M9HDILGC+\\\\\\\/aqPNHCAcJ7u6ZcahHeN0CrHjzsMgOKJw2LCNVZAoGFioTt1Lu0TGqoqfxgvLgbobmuf2AbFBCDqX6Ojdeq1nK1bqAlCnr39orsNZgqEUwgL4Qifr5qWEVAiq5jgw6BC2xXg\\\\\\\/rntuRS9zkieZpjYqfs+OZVwLIhTEDfnBl4UrbPqtCAWsZejRViXGPoXb04iN0YjrHe4q6794kIa2ygVFiAGjjdMTKcjEHzZNpWG0dEajwUHS0R5q9Q8i\\\\\\\/h97lAi5wXE3sBf9MIWaMKlP8NsdV+llM58nSfsp74A2g2pbm0orysw6WquMEypwiy6k8ZY1LohlfGlmy2uEkjSn+1IO1sCzl7z+gN\\\\\\\/ZY++OqwQGjJpkxaHURLd1q+W7Hq31Jm9mdnM\\\\\\\/wmfIHm1q3pU5bRdNePg86J6rewfDZEnQIN0ZdtH5C4EOB++NkE9fr6xhQPTlQpzHbVottrdH4sClxUfca0l4X\\\\\\\/7+CpEl4YPTu8mkNWpn5hXDDm9mfdL5JvQ8zSAArsGH+KYsimDHviYAPJ8SYiPH7h9W6NgWQMmVpXzwCwe\\\\\\\/J4Jcti72NzaqfgGzGORwLgDWo3aeSnX1gv69vW84IbIpXic92gqpDH0Ay10texQjjqDzIPbR8Hoqs3zgm25j5WWDGxEb93jjMubAknfnwXOsAxBXEF91WKwFr2qKgOr3dWY2AzTm8gEuICKZqspcdB8egHfwulYDGmNz1ZxnyVza\\\\\\\/lHFk1d+FHsftbUZt92QDPuEilI1qXYOQblmcR4dEcPBHUi5\\\\\\\/g\\\\\\\/l3u5iivDLTE4EQI0oUnoX7EF4zArDpfPHKAUee+Kb0i+MHTDsF6vHbJK6E1v3pXrWbI3NCXuMprZ5pyztQcx7u3E51GtL14k4bYnjX6cTjyDPVLVqlIx7c5abDUyQpfeqI9uD91kgbZ4+3k1BHt\\\\\\\/evpuUvxXHD9QpaWKWA7cxZfVkri5tFW23EWPu2Dt0pVEt1jP8K9P5Ei7cNZEwLKR6AZeB4BKhRCweKjDZREyo3k9x2DT+ncJNRIuJpBK4ZQH6uj+RH5HpYiy++ofd6UrU4+nFp32yDUo0JQNWrkU+rs4yACOpRFZeUf4CL+vasqhwo7gIbof0\\\\\\\/crndJL5xdEqQzDT+AG\\\\\\\/VsLCmQbG9vNe7TxTsPCcss4fqb4VMTwUu\\\\\\\/6PYbHpzNBG9UveDyMNc0bswtWMPlP6ajdNeZ2flzwLtjAs73zRVhm40VTmrnB\\\\\\\/WFqId8HjDM+zy3N0tdvABvt9Xat4Turt4qOvn1yb4UCRbuNsjm0tqRA3rPD7+QjYPq+h5mL6y+rV8U7OGeH24IKPnDmNp5dpo0T\\\\\\\/4xlvotemThf3c4mPe7vGo+gDZlbv+fP0FcVMGtu\\\\\\\/G3gDHutPToYiJrbXbEGFo7S7cXOROvUMXkZuzinpXUz1M4UOyED2rvChR99BFsmrvwb4GpkXf90D1rpksF0fl\\\\\\\/CDjuBvOXFFJFG9LCUOPhT9YJ1xbPcz23\\\\\\\/cixODdRVqgSPluvQfrYiCjY1dpryX978y2kQPg9PfrcNSWDgwxgEudd2ltnHB06iDRsS6t9xQVmKL8mxl2ltmT2KxtNBc6HB7oImzv+PszowfNGJuJMiszouSs\\\\\\\/B8wYQ\\\\\\\/vcKjkJbbweRX1Ih5lG8UsPShWWD4mEVnLwMdFHRFLZ0BTPDrDA9z8N4nbOWYc5hDRxOegnEecl0v+Gj+0xGbQeqQH\\\\\\\/A9c57gc37a4GCAa7dSs+7hARyb5AH6gLmJ6clujA2byGhop8hkeWsxORcq5F92lP1IawLozuwctc7\\\\\\\/FXki3mjYV\\\\\\\/UYaEMeOy8GGU+p7O2uM2t4FTLS\\\\\\\/16fRK+89JorfwUo\\\\\\\/v4+CwF89eJEhyQM1EVp+rbia20tWOrkXFwMx92i7rBf0t1qCebA2Kw3t3C\\\\\\\/nTovDSyAJ+nw5G9wtXy8cN0C+UFeDoaqXSQuxUSA1sk1BARNwk7Fk+E9XTQS4vo0ciG883rEs\\\\\\\/t6N9g9Z8eZyhz9Ju63BFWHsapF1zBRIAFdqmEQX392IWarCSS2W34nwEZDhqhlQCj3N7ikZEbPbkc9BfazCeAWN3eVVZLBWvXbvI7vqsX4\\\\\\\/MQ+6ko2jxqe1aIcCaiV1L4R0ZeBDIHXhDbwZMfj8glJNFbzNdbectCHgfc3ud3bCKq2fEHl4jHlyiwmOcd0Pw0KBSIyHOenA8\\\\\\\/Wy\\\\\\\/Cp3HiLnfOE5dmP08lnPAuB8eAepsl19ZNrIue6N5w2f4xLc4uLCbq1ZxsCE6up7pZVPfGI2REUI3ugM9uPv8\\\\\\\/ClKVZoDhT\\\\\\\/ij\\\\\\\/wqeNVpiaQRGrU95vuReUgSy8p185igDW7TlHudyr9F7jCnLJbeZQrzGvmwndZvT\\\\\\\/L6Jfg\\\\\\\/f+tJ2H0AgTgiVETT\\\\\\\/GRoLguzeUCYan+VNPMCMBU\\\\\\\/PlngvGu+5saMf1RK1JytLFCSZY9eRijcYtD8iPLSfPc9Xi0NMntbwT+KT1fQWpUyHeWYXfoHjH1Ciq62zpXiilCy45AWzSw61EL52BQqa\\\\\\\/naoQ+fsBb6UZBZxxgRBpaf5OXqHHpfNAVFhO05jk307McaR12tmMOqLPymp0Cveofr0wRyrXQswB83yGQ0Y5dhlx7j011vNBfH\\\\\\\/lbcjRPH5vytyHBwylV7xI3u+Ij0HsWqFCLKnrrDMFlWnhrmaW1QEGl0gp+7Y9u45Xa0CNLZIwqAdkvt4g5FkYS36NgFYOBuyi7lqBu+VG2KCR0kTB6K2R4jJXfKhd9BSOkJX1ywjArsyb+0yOg67uBt\\\\\\\/jmcZEBt\\\\\\\/vF\\\\\\\/nklPkFyQLg08H47yQ1bCymS\\\\\\\/768dPzx\\\\\\\/Unw3qjsI1cB\\\\\\\/+T2OWovw03fukltv6QXnxDyL4haQRQJboAhtoRqk2kRikZ64wdWhdzbxjjx\\\\\\\/v\\\\\\\/mqnAkiFbdl5iRujHxYjNNot8T2R7q6ZDTprh2+QM9ZfviSvstaaAmak1rcX3DrwjFSNjG1t4uA16UUVhnMMT0VYp086uu3qcLnHuBx\\\\\\\/kAxumPsoKm0zXlwfgZsTlZ4ZsExkQu1ckWDJTV60mNPRRg5TH2A5P1RdhBfKHQDdBK\\\\\\\/L+mClo92wXC5BuJh4r\\\\\\\/hFEzl1BA+DxslHDmuWn\\\\\\\/Pc4kjuGI0Yi\\\\\\\/JAzzVQxI283qme5+RZ+ti+b6Fc3\\\\\\\/x28cAT+KORcabZhTg7beGezdEhEuRjRVlbAbAEa8hXxmFpsM9FCouuoLbPqmKXbGt\\\\\\\/PtRZL+b\\\\\\\/1RTB3wADDT2KTxeXR9LsqxEe9SkB\\\\\\\/YOL0znB+zwdOORvz7jsn30eaMzt6dkJzl1Ut7ci\\\\\\\/N8THNZdHKHy2VKg0HirLmvjTwxjuk3UpB\\\\\\\/9CXMzdbq45yfEXdeDS1TMhYDEVDYV7+zxGzaGHfRTnnYHJUsAojvagdo8w4xE79ATMbpOTBQYY5itxm6Gd\\\\\\\/lZjQkLLwKvpqGupLZaqHY+DimKMCwjIMQiphOOEVh9kGPbNIysiDuCMBaT3bQBGoQZwP2Yh2fJRUJDPEMiZDUYdoQEnS5cjC\\\\\\\/NtBRnXEVWm9J1ERL\\\\\\\/MS3itbTcX0z30JpHTWk86AZ8yV4YlZuXXYaeTygt6moqjndQhiVpD0AMEs1m8YJBoeurDJsWyxJ5SiJf5YvTpXtVKIyYMNhPyIcG\\\\\\\/Vglx5mNjgUVdZA7gVDchSGLNu0+T6wrl2ieVlxtw4zy3sipluckGkNGq0ykwhmwghj6AT67Pdxj6fmnAmv7Gx5VFgp93JN+wIHMhVL2OTosxvIKkHaBzZlISyu3S0g7dcjFDjU5sAwUtQSNZVL1i76mf9xd8eaMmR2TVov3NnbUtzOPkFXg9u+H+C2LnTMkuFBGKN2lyq3i5a+PV2\\\\\\\/psUbAGs\\\\\\\/GQ+qyNZYeGVnUJAWC+XBQXkTljI9fcWXeXhobEbaqLh8sdcpme4JcUEVvBX3L+T8yOA9+iRti0Ak6zaUKkzmfdkTFTOn9FqCHYG3IeKVsPblnwHhxUWRyL9YQrSmI\\\\\\\/OkY21RJjxT1yrMkbsSJYxpHp1gQbMoUeR3ZDGagh\\\\\\\/L8rf5ITarQVlOND5Ge1BOIIuwtzVTBxTZGFgpjOA1T4Bd2IeMXYVG4HqRwH9y\\\\\\\/Dn5V\\\\\\\/HS62LwMaysqYfdxbHz8wrBCG3HQqzw89Gb1UMJdLBo+uZbZtu5LUz+icT2Z7pxLrnH9wJni+F4fFg7XngdmCmHnw0MumdMLUUsJCUU7eeakRZKzqkfyzNbTBUuopZF4WkVEsNZWPgQReM4Sk4wRQkcoDPSHh4Cn4SNL+tyQn0IqIeq0fDEIfljf1\\\\\\\/SoO\\\\\\\/UEITqFIMlu6bQ0aPQqbMlxHXCfpawcy8Skt0JAbXes+5dAPnVnWyBtq7yWYBv+QAH9f2qAjrV3NSpcal\\\\\\\/vEqxmk\\\\\\\/adP09ea6cUWfO6SPxDJigA9Db7ndvD\\\\\\\/bUo0YA\\\\\\\/RWwIyHEYOGDETTb3wFOPOv28MC5uIteXajZPEOfpYMnkW1q0Gd0VO6eSlmjv9HZQcKquSI6BBg5ipgZwCZuQlHRd2gNs4Bu6aNyOdvi1t8V4uFss1i3J2qBAAedMz2MDL0XAdeOGzBhBIF461mEOCUpWL\\\\\\\/mc57hTlf\\\\\\\/IxWyK3w8QCmDApluBMSIutQ\\\\\\\/BpfzFqIMmTwgR1PpuStz45ruLvfjPrdjFesbI4VHYrNK9qJsGrqu0VOL3rAsivveZxYLTfqGRP4vcZK15+NuRA+Foq2VVDriVNX5CuVMbMGoYn2Rly9Sqf6FnikrgPi+gs9O1WugN\\\\\\\/r6jWoN\\\\\\\/5x3CN\\\\\\\/OP0NoixrwST7RMQmrizGGf\\\\\\\/b1YpWgZvXZtHxRxiATFVK0fe9AV2g+JSm4ljYKiun1O1ssLq06mUX\\\\\\\/ZD1\\\\\\\/zPNk588yKYUjCzueEY6ruJTAHvcRIiHh3CTVeFSF4DadEAjzFP6Mx1je2at5kaSNdirHLtQxuRHGwWETuFVU+eAGDV2hw6r9H3Ld5nOSDB4IEzVP6GEawrQzbCkSflnLkSTO\\\\\\\/EWEeW5va\\\\\\\/Q4D34buMW5RkCLwUyxVG6uOvs06jStDbdFzc\\\\\\\/0l+BGP0UCtqHWljCYp34A+gq+ZiIzU5u+Mrl3N+vy3vC0IfmiEx\\\\\\\/4S7NGrzpy39HJoz1EL4jGFRSgXypxvPOx\\\\\\\/eK2kxnfakKXZqbyw20Cyt61VB+tojqIxqyj+k5DwmgT+IzUvGhYj0hlaXb4y9goREh+igIybS57Qc6Gz1bbF\\\\\\\/3KFrmgGbqX+T0Wkk2hJfQuMqaePaEUPE\\\\\\\/J804rq9fmKMUX3bN61S9mKRX3AzevwfznOYQdYkjOZtybQFB3swqmpkI1E73\\\\\\\/1wOVNuU9lBtjGdwAbJBIuyyJ14ZqcJDLNZJUkAD8NOZJMzawwT1QGRLLYa4zepzhdQ\\\\\\\/\\\\\\\/N0HGa\\\\\\\/6eOB0qTdj87BSXTVipID0G2pWoB+SeiscsVIVULQR8S1OfoXDl3TyxK4IzbKzvXZkz3wnuP0HFd\\\\\\\/RY+gYzTpZHEqCs+8aBgIj\\\\\\\/CYlRODHxE1OVZJAYQGVx8S9Fr1CuqhIQd+++S4fsZBR\\\\\\\/mNDoJQRtWMS7NlZRtmCRN5uOKP0eV7kayG96RBcD6mDkvyzJM6OjMXVck2qkU+BQ\\\\\\\/YAJlwbyeTj00UkznlXNtRrP7rpD1yYW+oXg5kNRzL0OYVBdlneMvyjLLbuNsJgyQOs\\\\\\\/GXpRyJ4xDQsyggNNwOzfE9vm+q7P98w2rjrlfmx5wyM7fNwZ8LBQJyhrPm9w4oBwTP\\\\\\\/FPM+JYxVZW9XzptdkUxFUm2x+n3VXq549UheHU0sXwStAnfnyg5sT0Ft1XUuGqYTzvTRvgguY2wJeRfAPtEqWQDhjw2weolvmwDsoBJFI13f+53tIgeyk5SeQRXICUgTOXgjBFZ+K1+f0tcvDfMrCuGpnADfYKpZSNZbY2q4VDvTRax++ch19RKL6vDTXL559oQlDjrL4uIoWlp5uEnuUAh3GhDjytDSfL+H2S8c3qUSMm67JZP6KfCrOyX0yo5ERzBJftNZvyCA4Qj4umZuB\\\\\\\/gXEyDKzR7Cqa+RaEUCESEn2FQPS3Uu9R9uOaVIF0xtjNaEbaNSzERUQyWOR8EDrGGiJG0YQzeSEfsVJp5XMXdif\\\\\\\/s1HddeMQpgFcKwpb9h3lSwLP+KLlmYa6lEg+sZWgVT2h7nFVsSrhq2dui6pACKZUv8UR18eV+18UmbRwCOHhK5UPchrbqgmfgTaUqZWRnkQXfhIKWLwqbwUGajbz\\\\\\\/vVVefpzxkng+znFzQLIrFEuuX8zstvd1tYxDqqh\\\\\\\/OBVC3sBmXFUR9HL3E4K2OAr+6mSVgF0472+Q9ubH2olzNJYMry407LufOpzw1h6KZMxhU3I2bfKBcuJ3qgwvz8A0s+yX9FcrAz6XmWOvRRwZUCxn46ZhOXi72rkDtR3FoLgMcYotDeqhBD74GrcxP9q5ZXFhxEgrhYZdoJcrg9Wd2XSdaMPu2RMmKU0UC\\\\\\\/brFigAH765jz020foZ5VHNnqKaIxLgQrCzWbor70PouutvkXFYZUnp2Y6dvrMsFSVwPaUxD0adJP1NF4zfYS4AMzYRHbQtVJFd5wV3Uo8o0YLcxrhslR6gWUO1NEIT2ftlK9C6RZo05SXHyTlWeASpKDZvEAnHly+p\\\\\\\/cTACmyLjIll7Sq0wfoL2l34dnd+Mp0ifKfMUIbKg7LIpLhEYuzxPncehZym48dTpk+elAwBDwUjQYhvxi8EgV6KnJmm7Dg4E8jPOTuqINgcQHEneTRWdysgKQlvlHQoph5Dj5\\\\\\\/yChHBZ85hr6PLE8h8dRhd2kuiDG+E1ni8CljIAuMl0cWraIutGi1plCs6kDFsayP65okD1tPMRQXCmScL5\\\\\\\/ym8ydFD2Wv2QoMheF0inXY+3D5uJI6UNDZViyDiYN2EdMA7k43YQmMfJ5RgpcCtMvUEMAAuukBjSL+V\\\\\\\/CmoDrGipOgp2hLSkhMyslEbS7ci6zMtnByKXG2qOMq3JYrfWTYgrfNdwrF5uyWSWMaWhEZXsuNTtDioMCcYZmrMYCNOllUeysZ+gOYDfeJAwlyQTCBX2Hjjv+fiYskO5Jdn12shf6N\\\\\\\/KzUGUkCSyS4tO67czDSpezQ5Y1pFnNiGDBQpzTF3K1ZG1nGubAYmp7t0dJQoqv0HECW4qypX7hnWj9J+0PjoKl9DvqGoBjUTQHAX67AQvEhZjq4R4BAnn+wyMBwzINB6u1oZ9Vaqf1PUXhNqa9nTUCDuiCefzAvJMj8ztlR+ZcnsNTcl7VoaESOQBDGoJDD6LeyVeCpiSIKU9TjrlEX+T2ICXfpqAmml4xgHLBjRYSxlv5QPtc7F0axdyPvpSZLTYjQhlf5qBG\\\\\\\/b7OUtURWk0p1fbbHMoBTwaPYRGwx\\\\\\\/7Ws4JH4aaYr3JVoeiigJVFTxivGb+PvTL2444FaKVdg0L+FCYF\\\\\\\/aIPXH9u0RRexiTduQMjJbJOTsdYeHJFPYG6GqnFdTodFvQ7y5BQYH3id6F0hjMTM6qHBsZxh7qaoQaCEnMKbjgNP8oQzc7b5tAD4GDAqWjz5safoH\\\\\\\/bteUifZG7fnubq+EwMXXywocYR6i7T6J85HKlLVVxl+rcLVqoXwGcfRQWm6BZnre8soCjnrnsi+gKnveeMMvdTIQo7Zo5nVPfyB\\\\\\\/R9HugRAx6lpGNIj67Wrc8H5uS4+Cz958tHLDJjkmfYLojpl7uhXWeRb8oiS+vVv5y70idzf+kGDMNX44is0F0R8EhFW6IL7PIN8YY4xPY5TyDTvqxfmK++foLNbyKDjaanspJ\\\\\\\/hScw53WIwVHimUMS2kNwxoy4N7OPYyTq8Zn3UCIE2M3hnHPE2IiSaU4XXD+AG6ULzq9dSJ0f1lVakcZngD+lKF4F3s6od04xffjJIH1LKLpPePUdLSAM1BIM7itiAZZJBruXSiqIK79spUbxbKbOw7CXkVDVCSqMInx\\\\\\\/uHkizSLmsqXQGb1d\\\\\\\/re5l6r2blQqL0Met602FeCYBr8XvrcV0y9UoF21cXI8GgUMtr8rJviOlpt9BbWaknQ0WNt4maakOmHl+up\\\\\\\/BkoAn4GHvidEpRBrbnVqvCrgJxRnQEEULSYF+iYtQ47RFTfc76rJMQMw7LRVASLMPcOes1qOOrtCqn1Uk4bZh5GpuvBIgGpKp5DWL7yYOKr0FSYGCnAi9bd\\\\\\\/V7Z5idZtvxJLm1TAlnYHHmLGANuZPz5Lalt+ejNNwKLo9NIdmk0KY8Pl10WQ\\\\\\\/luMxaGGL3BcyD5sOrrM0gIqGDR+kxZ4PN5TfIZ+\\\\\\\/1+0T\\\\\\\/bC1wWkfRlSfMm6c1FAq4IAZtWrWfyrf6SbCXKWF0LUn1Sfu5S8e3u6fpr2EdpxfQ\\\\\\\/6oGxe0bjXQf+GolqdQbe6n7ohjrJBaRGryT1VLzqcLvs\\\\\\\/54kATQs8MGC+244\\\\\\\/BahLiMMctzkogUAmmXqgl34XJhLbRbc2\\\\\\\/7\\\\\\\/mURb+e44hFCOc6QI5xJ6JOzc6tzicexZM3nXYH4lX5+Qjw+UKwbd60CAIyfCecS3npEOpEgDKPRi6TxEqB5QCsAGt8ffdYRFsWF\\\\\\\/VYa\\\\\\\/Dxw5xsk+\\\\\\\/dcgNXEgXdhCVGH0mFTB\\\\\\\/dzH7BB6y\\\\\\\/rwzBm3mTJV38RCqWXbqDWwf90L1k\\\\\\\/nEqnxUDOX+n+OLsZq9bH7rxo7qUBNebvX\\\\\\\/RmsWWLr9S6QK1zDZVHUa\\\\\\\/5x63I3cd+AwmxVa3PrU9YMoQTThUV5\\\\\\\/5q4VgcaWHj9TwEV2lf+B9stoQK9CV6itsG7zjD+ix\\\\\\\/tDeHehvdPcXOVfbJICRYWMae8Yn65E+0NxL3r5c\\\\\\\/D\\\\\\\/w\\\\\\\/L5Wd11l4Mz\\\\\\\/LNOkcJL5you7TZ344IfdxDUTRAlU4gocs2qNEkpspqp6U8+g3PYm808LAOSpXaUrxqGco6EMrNAtY0XbkjHMAiODrx5tTKdrp1\\\\\\\/zAYU+OZehfp1A5hM5xoHCHMhMWAipdtQ+fmw\\\\\\\/+gHMmsGg68gowl9PpSoqTeKaSb94UrpIp230KQpHo9eEiaLWKRYL4TxUvSoVpd0KIVV\\\\\\\/eVgbsFyb5Rk0nP8jMPIFC2TBmRchj+nQatCpOIcSINc16AQE0rIejl4gdjdBRvcLCVe6LZVn6rNrzUDx8rLQU6OtJ7tZUSLYErW05Xm2EO5GGmgH9gzktVvHfLR2AkXESywf6e1VZM0Xhfu3ga+gY5W8vUHj1l92yXzUu0+2RXGANe3VwC0OlSnw8sY3OZJfEA37OE294hsfvtBq1pT1saSCzdJOBbX5O9lz3KITgEKCoZE5paeC7JaHdUm7qHm8IWa7eXKloZ1on9aXvHXwiRnPvNFkmhklzk8W27ECAhTugrjpUuWmFK6D5WBZtSaCyvbsMz\\\\\\\/YgMnHKLZQLQjKov+ATY0nqkLyS0gVA8RdumCQZW2R2P9AmX9TXq8ZCbFttWvFd6efLr9o9D1csPYypWorabYRxHFtkcRXoXkyTqoLTvLxiDqOCqNDe6oVMdj2+z1OxXJ56DYpHIDBN50JOMmk\\\\\\\/EUNOsy57R8CgZ3+23pyn3myQPFZiH+AqOJM\\\\\\\/aZ9Dvk+HfGRAPSRcJXFtOzdlM2itCj\\\\\\\/IKZM2985p0qq\\\\\\\/fUKzfe77KoIcKa7LNufZvbo9vE99jLOVGpru0NhZIvVmGMZRv\\\\\\\/wo6hbhQKsgNnPR9Wc2irqAgs6dbRy1tM+Hl1X8LSJn1TBWe3+NCoVu3Bm8gkc3hGM50trdIiN10+83H1QTuhfJijIp6464cFaI63KpW5ttAKq7hBlGLQVP6SmWrzV5pzufL+eaJ3rWgXMPuDlZPpkyC64aX+yeSaVwUycH57BmJQEbiEHc9zk+k\\\\\\\/qYtPAOvYcHE3mB\\\\\\\/s0fS7piku0AoJZPfOSJkCELr92n4Kc\\\\\\\/gP+A6Xsl\\\\\\\/8mo5aq11k3FyZ0meW\\\\\\\/oDzhBSPjvB6dIHpvL8WbtnsKh16SfSgX6UiWkMXwPpt9+hyCs7yRK9vviocB5KWKtBxhPPhyj+t0cviOQ74WnAg1Qx7LLV29awKEYAjj4GGSupuyqFO5bpSnZDfu+m90GwwCvgCanX\\\\\\\/amhWlTQZm2MmCYIzhPb4Eneji\\\\\\\/dAwmWGA7JUH4yFPj6OQKSNbAX0TYB5yxxc0gmyuP5xrMYuL4bONifyo3WtBfR25YkTpWgIQjqg4KevhUO1HdFl5AcnsW9xzSOW5W5gn11uUS8r505uuT456nudFucm0RK5uac5hiymtP+SaW7eYZ2WD7jA0ONeyucIhypIIus9zOVcmY\\\\\\\/Yn5dzIzPo8KxE9gsXezkI2Pu1fyvxemlD0kxV0xqz7Wr\\\\\\\/5R42Mv2YjkXSoEhYapEvw8v8jbLYNGDCm1x3EXNUUMxv+rV5684chv8hZ8FDUMJOEfClSYPJFvmYYbEjiAAS2ToA\\\\\\\/h23ozXVyXeZDiOnJ7XLH1gh\\\\\\\/W7E5SZvoLvhF7SBjUIR9Un45FF0yYHGskOwLWENS\\\\\\\/Ji1VdIQL7Tg5hqqDN53aJNGsrfmo+d05aULH3\\\\\\\/EBB\\\\\\\/nErl0U2pi27MIhtN+s4tvwrC0InZsZEWsZAdEQzaYZXXH6TStFjIxM9R+J13IvnucjyZQyImA6411qHH\\\\\\\/JhUuKxxFIdjpB5ER9fwANwumJaKL6eS1xf\\\\\\\/Yt+QtZpxmepzxZWDRHK8Pms4hhuoPa1Ir7eyDhEnrcR\\\\\\\/pAdgHa524uaMhRUI2APEHzaJ7lvqXH4IeSBNsQPdyYf\\\\\\\/AXqwc6qaq6vwOL4bJZUx+hYZhHi\\\\\\\/HnaCtETcizgqZkyBlHBRqsikvRxzHCu2naIKK6sNjRch9RL1cLoOicdt3bOlcxS9OtKFCje6f97HO6i3YtiNDXy3p5iDMTD6H7ant52vrjcLq52LzXS3pFxEQDrR8DSIJ8tFVKTLll97H\\\\\\\/+0X5Vn6zwbVms3LpM7GTe3Z\\\\\\\/VK3lM6XvWmDu9okJebrTMBbK7QLlqhH8yebGODOgx3fGS6PD+jCa61yR3ox9VmlxvqbLFLGOYRy8irebYXlAitlXXn8LAM+wNBDLbhkwPxLfSu9JjLiMEDdez+EtayDq8eWUP63bqgvX3ZWtP7q0rPNi2xCL\\\\\\\/3EhSoLzzfVa+U\\\\\\\/ziEjZobOKqvNAmaxYsc\\\\\\\/lV0Cl+CBNjy94oI9R6nBA3p1zHpycTYrKvo+Jwf0FHeQTABH\\\\\\\/BPBwJUPrq9go2JOUClCOsHIIUQOCyTpx8tI\\\\\\\/57i5De+i6UQQEcCw+V3Ncc4YMKlIcUZEwVH9KR\\\\\\\/S+Z+xIIVQOw2DXdjJHWbjmR0uZgESF1S\\\\\\\/4UdYS37AVVoXyWE5l2kaZeumt3izTDV+dKLdeyseTWZhc7WIeKTQpw58H2FfeM7Uk\\\\\\\/qu2LvyuNxhCPVABkK1joAodkJ8HLfWS7pD2IAjcJsoBfLS8n9ZOr+fBodBLflZ0nA11\\\\\\\/lTILE21DgeSYccWbDt3i3bHYZRX6qHhuXQNhI6RWfAjeED6459qcZaKit+MCpDvgWrFx\\\\\\\/HVK6wxvOWdrqvU+3Z2aq0r6R12PZioQ8mCrADognKOHOIP3ieSzQwajRWWFTRkvlzJLLqkGAqYlGUopzkRMv4k9n39AhbW29+A5kM8CSNn5nV+9y9H07wbwxUsWRlb+yjyRcZKLln8Z0g7rVa5zkLwjwJVOApapZ7o\\\\\\\/EdhrUrAvq8nYP0sLmvnUOgH1PTaIM3WljIaw+edbaCXdutGOCZ8ABbScarqe4OYfJrJYKBLIdkKUF9xfwD+ZDIsYPeZKlqxBEdh5sEw9tP\\\\\\\/wuWgmgFycLd0DXbBFQOOrw3WLUkNVQgkf01ev9zA9l9tc1fYbvMfrgf9A+h6X6TD4eSeaDPWdW5LJ2S9JS91OnCTTCSeeewbk86kfw\\\\\\\/V0Zea\\\\\\\/6zdPGblfcyGHzUdMm636VLHrNKCJltVtLxhxhj9MMW7UYpwhdwHWM86cAB2GLwFKAr\\\\\\\/eoGrKg7egOyZ6ulXw3WJQ7xw7XEG3pffde+zJtl4IhkCphvJutJ1Af7bY3jL4o7EtWzyhntVFdeHjc1by7oHCtyI2zXUvFaUCFxd7uG70zsB3k5WJTaXlLyHYgYp8OseRgqBJMzBM\\\\\\\/iOqUV4wf6RQS9QrK7xdTvyuw3Ahkaaehr8m9m2l9XJAILCuy3Dgq\\\\\\\/TlS5JTFWP+nscMrCm1ifKTery5cHLXDQJFW1EFydTb6YT3pOrvc+pNUOVM2xDUT8yshevk7y3JYe5weZeX1R6ece2NEbwTf1k1PGAv+KCQaP7VERYDZV2pnbiz1NlMnnYeKkm6IMq9SuSbxeRfTGCyMbRx6oUBI\\\\\\\/J3NNJgybUxR9O6WQrfY+i3SyBH3jg1rdgNH2XwGcaaVRJGNThRfkFwe5qQ9kCbZDLMHBkRAvX6Z0tBjilX19qkyPe824eAVbNfShi6fZoWu71FqZ8zQTLm2uWkwpoopWhW+ZLL3T1klYcyT50k8g8G3WPS+slILPNDSGezuRlEgwINf9I3UK6gmjdIDvQqG7HnSQBJbHf1NoLY1MJ5MLNoDB+LZYjRBdn0YPLnaTKTb2JrwKHZz+t2ZZTEmkfnuVIs5J4GSERS1b6oHFFBlJ4ALPQn9CO8eQJNfo85ZnZhA1jGuABLI9jWtDTaKEmvbIPQNSYFh\\\\\\\/jcT3TNzYcWzqOQFY3LnGz+e0WZ+AgdXBoC6kBGNS1tWCPFWrcqACCO3xDwXGW4hm2HlNAa8lMC83TZ9tGdquSWlIVJSWMyKkButTuTEasDkG9SzhUHa+IX2BXOPkg6cCeuVrkZOMueqlIWJtu2eOIgp4mLo2jwghh96G10RvcYZmjrEJwZpKniWBlwoFaPdTh5A==\\\u0022,\\\u0022iv\\\u0022:\\\u00220ojPAbv68hIbyDZ4XbaOLw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227ddb94e8a51338c9278b4d318d5a88719e0afd792c7829e723931e32e8dc2089\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022leArxPwj2sQEkM5Euz6gYWR3TndlpCrm+mCsWla4PAqvQmfbB5siCWMYK3Qi4qivTZLXfvXHeOjQW09BQzCNltBaUCs2WVaxcBH4QvHY9SYBP65yZxJWxOKf+Rwj0Ha8RGgW+erwGmeYChkJv1yk6PFyIyPzM5h1gTFnC++W+x1mMofhVJ8w3vzZPajNwjKjSNlpD6anvMlZ0QLl\\\\\\\/KhvJe8MWmaeZJJNh6\\\\\\\/uwyejMOnjAF4CMgN8+mPLkOGsslrEr3t0QxikibExDvua8sICk0PFg1+RcpPlOoEeS8oXcBMYVeW7GPOv0b\\\\\\\/l8yf6FvpWVlzFwmYffKgouJf0J+UA2wwTYLYq6JKfw9y8pHc2vZNIsWlMTGkhtnN3ePnHFvLL45dLG6vxbyYKC9mdoy8it\\\\\\\/pxfiVJY62e7V1EzzhtV38jFG4n75oMObs3p+C4X5J3k7NK3Zv\\\\\\\/xKDBBvy+Bh0e8x9vR9n2hFPwWCfCrU7XfONna\\\\\\\/trbYzd+BedneVff2Tu3sS+MHtxyD20XefF+RhkDXfoMS9yY3hmuCGTE2rcKSVkPhBBhYJ8RJURKzk52FQepu9ZAxGG6iDfPRmJPrpyfat2Q+QIAm0zUIbjelU4h5I+6JpQLA0Yu6VgcajQEhh4yHZ2eB1A210qXWdWmLXyHq7DcU1wswQUHrkXhmqJsQYnt6N3w\\\\\\\/A7VAnuG2zYRGsIsUv3BoDMoSTnuPjH7ovSQdT2hGs7tofLnUJWwMyfdIllIKwwff09SbcQQYkPtkPvtn9GuBSRXu5ua6QWi2I4TPXcc1ASraE+D+ZGfH3Ar+LTm76qqjdsYJssKApGCsaYmDHnx6GPB1GN+EcFk4WPRf+cVC9qbNGK+rXwkD\\\\\\\/nkjxU\\\\\\\/c3EUyz78Rt3coVxhDVX9E4u6cJbka8Ul76+enm5jZ+MsMeec9DIfz\\\\\\\/nprbQ+NeCVEqVASbuNgARXi9pv2ivGe5Y+fyIYEYHdXdtlJ3hEHt9AFfrzQ2UvYMrVuLURWAW6Guut85imFWlORXYJTmXl51H2epJpqQ3O+XqlIo\\\\\\\/UPe9SCffXffQ4+RuoUWODTt35P4i4C7XVynJVAtM6Wps7J2ewMnQgJ3Js6lacE9lK8GD3IVt1VGECljsihaj+GYzEtCogB1gVV5dlxvQqfmULvxQi+DWNxyNEadZ+B7GklS3wTgKciOAdNwhRHQ0dvJHYd3yfRJimSiLmEMwVCWvj7L+80EhUpP8EpXwUWl1RSgyAASP\\\\\\\/lb\\\\\\\/eWYwklD7xvUKMC6xpbOAL6YW8HWUonb69SFiyDait+rZqh1BcA7hunsLNY7SSyBFp7bFmZw89\\\\\\\/ZpyvlizdeAlAWk0kJ56rCVVGjJwdUmeqH0cqjH6GjsCrBp6G0uy4wir3mHz4JpP0EbZcG8Cl6GCY7GuveBrL03ePM7Jzcd7Ity3Cr3xsSGsLg7oN4WS3F+QiMfOoNNd6WpJMn6Ji04qRoTfKrJT44AUzeq6WJv7eeXN5sM89oZtRfz0thRUnKkDqvxDKa\\\\\\\/GnXGOqmz28YbDhQHW7pa3JlEixsR22AWdkQ\\\\\\\/aCnJy1WEn7dOqDvHfLw32hMtwsF8AlIU3z9R9GGT9y1T0plRsL70xOFuGg\\\\\\\/oolVr5knjS8OoF6\\\\\\\/ojccMlgfLkGtiGqFx\\\\\\\/tist40NlGBh9MK7w36CJehWKSsXhmjIkyWObB+8fKunEzAULjCcl5\\\\\\\/Y\\\\\\\/IY0yLQMlP2a+MgKNEP4vWaXeurbHY6cxxncGwryfu3\\\\\\\/mNjnC4cFGfQb2a4nUm77Vdn+W6Ab3hrVwnIIQyvRre3fjM0c2kRu+xvYZa4pTl0O3DXQihM\\\\\\\/ctS991sOSUGjQYzbBZUAZz5uZaZ8B3jFFvV5R6gPtx27jAejXaEvtnPRhpMa+tsRgSX9QRSuz5COj5Dmbgr341ibplhImqerF5enFwkS9\\\\\\\/cg0gGvnWpySFU5Xwyq2SZ9vTZh3tIebrtXFhs4L3DBV\\\\\\\/1DEAbT+kmEE\\\\\\\/ce9wa3O71es5yUSULB06U6f2Os\\\\\\\/vsq4g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Iu+\\\\\\\/VZf3UzWmvG4tJtsQpA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d56b770b0cbb461e31ebca9cc4f1084ddfe2ce04e0738aea679656e633d3fa5d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460278 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2761" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GFpeZYtENyNxrD-jB32wwyXAfcRBSKpP7jpiLdzGmLQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2761" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--Yec7u2s5EaXRCCWuRs3Q549ycLr8VCnk-I8TDlUprs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2745" title="Identification of QTLs Associated with Virulence Related Traits and Drug Resistance in Cryptococcus neoformans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2773" title="A Comparative Analysis of 5-Azacytidine- and Zebularine-Induced DNA Demethylation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460278" id="node460278" data-pisa="ggg;6/9/2761" data-pisa-master="ggg;g3.116.032409" data-apath="/ggg/6/9/2761.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Covariance Between Genotypic Effects and its Use for Genomic Inference in Half-Sib Families</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dörte Wittenburg</span>, <span class="highwire-citation-author" data-delta="1">Friedrich Teuscher</span>, <span class="highwire-citation-author" data-delta="2">Jan Klosa</span> and <span class="highwire-citation-author" data-delta="3">Norbert Reinsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2761-2772;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032409 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460278" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dörte Wittenburg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%B6rte%2BWittenburg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wittenburg%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AD%25C3%25B6rte%2BWittenburg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wittenburg@fbn-dummerstorf.de" class="" data-icon-position="" data-hide-link-title="0">wittenburg@fbn-dummerstorf.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Friedrich Teuscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Friedrich%2BTeuscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teuscher%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFriedrich%2BTeuscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jan Klosa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BKlosa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Klosa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJan%2BKlosa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Norbert Reinsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norbert%2BReinsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reinsch%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANorbert%2BReinsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/9/2761" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460278" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2761.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460278" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2761.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460278" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2761.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460278" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/9/2761.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460278" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In livestock, current statistical approaches utilize extensive molecular data, <em>e.g.</em>, single nucleotide polymorphisms (SNPs), to improve the genetic evaluation of individuals. The number of model parameters increases with the number of SNPs, so the multicollinearity between covariates can affect the results obtained using whole genome regression methods. In this study, dependencies between SNPs due to linkage and linkage disequilibrium among the chromosome segments were explicitly considered in methods used to estimate the effects of SNPs. The population structure affects the extent of such dependencies, so the covariance among SNP genotypes was derived for half-sib families, which are typical in livestock populations. Conditional on the SNP haplotypes of the common parent (sire), the theoretical covariance was determined using the haplotype frequencies of the population from which the individual parent (dam) was derived. The resulting covariance matrix was included in a statistical model for a trait of interest, and this covariance matrix was then used to specify prior assumptions for SNP effects in a Bayesian framework. The approach was applied to one family in simulated scenarios (few and many quantitative trait loci) and using semireal data obtained from dairy cattle to identify genome segments that affect performance traits, as well as to investigate the impact on predictive ability. Compared with a method that does not explicitly consider any of the relationship among predictor variables, the accuracy of genetic value prediction was improved by 10–22%. The results show that the inclusion of dependence is particularly important for genomic inference based on small sample sizes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/autoregressive-prior" class="hw-term hw-article-keyword hw-article-keyword-autoregressive-prior" rel="nofollow">autoregressive prior</a></li><li class="kwd"><a href="/keyword/bayesian-statistics" class="hw-term hw-article-keyword hw-article-keyword-bayesian-statistics" rel="nofollow">Bayesian statistics</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/recombination-rate" class="hw-term hw-article-keyword hw-article-keyword-recombination-rate" rel="nofollow">recombination rate</a></li><li class="kwd"><a href="/keyword/snp-effect" class="hw-term hw-article-keyword hw-article-keyword-snp-effect" rel="nofollow">SNP effect</a></li></ul><p id="p-3">In whole-genome regression analyses, it is often the case that the number of genomic markers, <em>p</em>, exceeds that of the observations, <em>n</em>. Moreover, linkage and linkage disequilibrium (LD) between loci adds a second source of dependency among the predictors. The number of genomic markers such as single nucleotide polymorphisms (SNPs) is still growing, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-1.gif"/></noscript></span></span>26 million SNPs have been identified in the whole-genome sequences of cattle (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> 2014</a>). Genomic prediction works reasonably well when based on a huge number of explanatory variables (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gianola 2013</a>), but the high dependency among predictors, which is often called multicollinearity, may lead to the incorrect genomic inference of marker effects because the standard error of the estimated effects is likely to be high.</p><p id="p-4">For “<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-2.gif"/></noscript></span></span>” problems, various methods are available that implicitly consider dependencies by selecting relevant predictors and/or shrinking the effect sizes (for a thorough review, see <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">de los Campos <em>et al.</em> 2013</a>). Bayesian (<em>e.g.</em>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Habier <em>et al.</em> 2011</a>) and penalized (<em>e.g.</em>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gianola <em>et al.</em> 2006</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Piepho 2009</a>) methods are the most common choices for genomic prediction. However, explicitly exploiting dependencies, especially those due to the proximity of SNPs, relies on the appropriate order of loci. Using the order of SNPs, and clustering them according to their adjacency combined with the Group Lasso method can obtain better performance than other penalized approaches, where clusters that contain causal variants may be identified with more confidence (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dehman <em>et al.</em> 2015</a>). Alternatively, haplotype-based approaches (<em>e.g.</em>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Calus <em>et al.</em> 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cuyabano <em>et al.</em> 2014</a>) exploit the connections between SNPs, which may improve the accuracy of genetic value prediction. Associations throughout the genome can also be modeled using a first-order antedependence correlation structure (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Yang and Tempelman 2012</a>). A penalty term placed on successive differences between the coefficients is employed to consider the natural order of effects, thereby yielding estimates with smooth transitions. In addition, sparsity in terms of nonzero effect estimates can be achieved by penalizing the <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-3.gif"/></noscript></span></span> norm of differences in the fused lasso approach (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Tibshirani <em>et al.</em> 2005</a>). In a Bayesian smoothing framework, random-walk priors were suggested by <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fahrmeir and Lang (2001)</a>, which also allow nonequal distances between predictors. In particular, for genetic applications, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gianola <em>et al.</em> (2003)</a> described an autoregressive process with variable distances between markers in a mixed-model formulation. Incorporating the dependencies among SNPs can improve the outcomes of genomic evaluations (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Yang and Tempelman 2012</a>), but the assumed pattern of the covariance structure has been based on general assumptions in previous studies.</p><p id="p-5">The objective of the present study was to extend the Bayesian approach by using an autoregressive prior for marker effects, and to determine the autocorrelation parameter explicitly according to genetic theory. The population structure influences the extent of associations. Family stratification leads to different levels of LD among families, and may result in a biased population-wide measure of LD. Thus, dependencies caused by LD were theoretically derived for a single half-sib family that is part of a typical livestock population (<em>e.g.</em>, dairy cattle). This required the haplotypes and recombination rates of the common parent (sire) as well as the LD of gametes in the population from which the individual parent (dam) was derived. The obtained covariance structure was then integrated into a statistical model for genomic evaluation. Genome segments with significant impacts on a quantitative trait were inferred, and the precision of the parameter estimates as well as the accuracy of genomic prediction were evaluated. This report ends with a discussion of partial successes, drawbacks, and further options for considering dependencies.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Statistical model</h3><p id="p-6">To study the genetic effects captured by SNPs, each with two alleles, A and B, the following whole genome regression model is fitted to a trait <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-4.gif"/></noscript></span></span>,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>The design matrix <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-6.gif"/></noscript></span></span> contains the genotype codes at locus <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-7.gif"/></noscript></span></span> for individual <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-8.gif"/></noscript></span></span>, where 1 and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-9.gif"/></noscript></span></span> indicate homozygous genotypes AA and BB, respectively, and the heterozygote is coded as 0. The vector <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-10.gif"/></noscript></span></span> contains the additive genotype effect at each SNP (<em>i.e.</em>, half the difference between homozygotes). Furthermore, <em>μ</em> denotes the overall mean, and <strong>1</strong> is a vector of <em>n</em> ones. The residuals are assumed to be independent and normally distributed, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-11.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-12.gif"/></noscript></span></span>. For convenience, other effects are omitted.</p></div><div id="sec-3" class="subsection"><h3>Covariance between SNP genotypes</h3><p id="p-7">In this section, model (1) is extended to consider the dependencies among predictor variables in <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-13.gif"/></noscript></span></span> due to linkage and LD specifically for a paternal half-sib family design. The covariance between genotype codes can be derived theoretically for each pair of SNPs. The derivation is based on <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bonk <em>et al.</em> (2016)</a>, who deduced the covariance between SNP genotypes coding for additive and/or dominance effects in a general mating, where the haplotypes of both parents are available. In the main result for the additive effects of SNP alleles, the covariance can be split into paternal (<em>s</em>) and maternal (<em>d</em>) contributions because the alleles are inherited independently. In extension to <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bonk <em>et al.</em> (2016)</a>, the maternal contribution must be generalized. It is assumed that a dam is drawn randomly from the population. Half-sibs have a common sire, so the covariance between loci <em>j</em> and <em>k</em> is determined according to the sire’s diplotype <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-14.gif"/></noscript></span></span>,<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-17.gif"/></noscript></span></span> take a value of <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-18.gif"/></noscript></span></span> if the A allele was inherited, but <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-19.gif"/></noscript></span></span> otherwise, and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-20.gif"/></noscript></span></span>.</p><p id="p-8">To determine the paternal contribution to the covariance in <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a>, three different types of sire diplotypes are distinguished (the results were taken from <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bonk <em>et al.</em> 2016</a>).</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-9">Double homozygous sire (haplotypes AA and AA): all half-sibs inherit the same paternal haplotype AA, and the paternal covariance is zero.</p></li><li id="list-item-2"><p id="p-10">The sire is heterozygous at one locus (haplotypes AA and AB): two haplotypes AA and AB can be observed among half-sibs, where each is equally frequent. The paternal covariance is also zero.</p></li><li id="list-item-3"><p id="p-11">Double heterozygous sire (haplotypes AA and BB or AB and BA): all possible haplotypes appear among daughters with a probability that depends on the recombination rate <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-21.gif"/></noscript></span></span>. Thus, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-22.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-23.gif"/></noscript></span></span>. If <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-24.gif"/></noscript></span></span>, then <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-25.gif"/></noscript></span></span>.</p></li></ol><p id="p-12">Using the allele frequency, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-26.gif"/></noscript></span></span>, and haplotype frequencies of the maternal gametes, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-27.gif"/></noscript></span></span>, the second part of <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a> is:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-28.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-29.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-30.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-31.gif"/></noscript></span></span>Combining the terms yields <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-32.gif"/></noscript></span></span>, the LD of maternal gametes. If <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-33.gif"/></noscript></span></span>, then <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-34.gif"/></noscript></span></span>.</p><p id="p-13">In summary, the covariance between SNP genotypes among half-sibs can be split into a linkage part contributed by the sire and an LD part added by the mother. The covariance matrix <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-35.gif"/></noscript></span></span> is set up with the following elements:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-36.gif"/></noscript></span></span>The linkage phase of sire, the corresponding recombination rate (<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-37.gif"/></noscript></span></span>), and the LD (<span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-38.gif"/></noscript></span></span>) of maternal gametes are assumed to be known. If they are not available, these population parameters may be estimated, such as using the maximum likelihood approach (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gomez-Raya <em>et al.</em> 2013</a>). Now it is necessary to determine whether genomic evaluations can be improved by knowing this covariance structure.</p></div><div id="sec-4" class="subsection"><h3>Specification of prior assumptions</h3><p id="p-14">To estimate the unknown parameters of model (1), a Bayesian shrinkage approach is employed. The matrix <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-39.gif"/></noscript></span></span> can be incorporated as a scale matrix during the specification of the prior assumptions. Thus, a hierarchical structure is defined for model (1):<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-40.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-41.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-42.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-43.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-44.gif"/></noscript></span></span> denotes the inverse <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-45.gif"/></noscript></span></span> distribution with <em>ν</em> degrees of freedom and scaling parameter <em>S</em>. Let <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-46.gif"/></noscript></span></span> be a covariance matrix that depends on the elements of <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-47.gif"/></noscript></span></span>.</p><p>The SNP effects can be estimated as the mean of their posterior distribution, which can be obtained by Gibbs sampling from the conditional distribution (<em>e.g.</em>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Sorensen and Gianola 2002</a>)<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-48.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>Several specifications of <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-49.gif"/></noscript></span></span> are suitable, which differ in terms of the dynamics of their regularization parameters, as follows.</p><ul class="list-simple " id="list-2"><li id="list-item-4"><p id="p-16">(P1) Uncorrelated prior <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-50.gif"/></noscript></span></span>. An inverse prior is stipulated for the shrinkage parameters <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-51.gif"/></noscript></span></span>, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-52.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-53.gif"/></noscript></span></span>. Hence, the posterior distribution is <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-54.gif"/></noscript></span></span>. This approach is similar to that proposed by <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Xu (2003)</a>, which represents a baseline model.</p></li><li id="list-item-5"><p id="p-17">(P2) Correlated prior <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-55.gif"/></noscript></span></span>. A flat prior is used for the variance component, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-56.gif"/></noscript></span></span>, which is similar to that proposed by <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> (1994)</a>. This parameter is distributed <em>a posteriori</em> as <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-57.gif"/></noscript></span></span>.</p></li><li id="list-item-6"><p id="p-18">(P3) Correlated and adaptive prior <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-58.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-59.gif"/></noscript></span></span>. The prior of the regularization parameters is specified as <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-60.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-61.gif"/></noscript></span></span>. Therefore, the pattern of the posterior density <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-62.gif"/></noscript></span></span> is unknown, but it is possible to employ a hybrid strategy, where a Metropolis step replaces the Gibbs step if sampling a parameter is impossible (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Tierney 1994</a>). For each <em>γ</em>, a single Metropolis iteration is used. A log-normal density is applied to the <em>γ</em>s as the proposal density <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-63.gif"/></noscript></span></span> to obtain a random walk chain. By single-site updating during each iteration <em>t</em>, the <em>γ</em>s are individually drawn from the proposal distribution:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-64.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-65.gif"/></noscript></span></span></p></li></ul><p>The tuning parameter <em>ε</em> is set to one. (The choice of <em>ε</em> is discussed below.) Let <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-66.gif"/></noscript></span></span> denote the vector of <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-67.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-68.gif"/></noscript></span></span>, where the <em>j</em>th component is replaced by the current proposed value, and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-69.gif"/></noscript></span></span> refers to the vector of the samples obtained from the last iteration. The ratio <em>R</em> is obtained as (see <em>Appendix</em>):<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-70.gif"/></noscript></span></span>with <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-71.gif"/></noscript></span></span> using the Hadamard product (#). The acceptance ratio is then determined as <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-72.gif"/></noscript></span></span>. The proposed value is accepted, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-73.gif"/></noscript></span></span>, if a random sample from a uniform distribution is lower than <em>α</em>; otherwise, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-74.gif"/></noscript></span></span>. The idea is related to the weighting of variables: with values starting at one, the proposed values of <em>γ</em> move slowly away from the initial estimate toward zero if there is evidence of a null effect, or they increase for nonzero SNP effects. Finally, after <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-75.gif"/></noscript></span></span> Metropolis steps, the vector <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-76.gif"/></noscript></span></span> is sampled from the conditional distribution (3).</p><ul class="list-simple " id="list-3"><li id="list-item-7"><p id="p-19">(P4) Correlated and adaptive prior <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-77.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-78.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-79.gif"/></noscript></span></span>. The prior of the shrinkage parameters is assumed to be <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-80.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-81.gif"/></noscript></span></span>. The posterior density is then recognized as the kernel of a scaled <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-82.gif"/></noscript></span></span> distribution with <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-83.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-84.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-85.gif"/></noscript></span></span> (see <em>Appendix</em>).</p></li></ul><p id="p-20">If covariances between SNPs are not considered, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-86.gif"/></noscript></span></span>, the prior P2 is similar to a ridge-regression-type of model, while P3 and P4 are similar to, <em>e.g.</em>, BayesA-type of model (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Meuwissen <em>et al.</em> 2001</a>). In P3 and P4, two different types of modified Cholesky decompositions of the scale matrix <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-87.gif"/></noscript></span></span> are incorporated (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pourahmadi 2007</a>). For P3, where the correlations between SNPs are known and stationary, the covariances between SNPs are affected by the <em>γ</em>s which are sampled during Markov chain Monte Carlo (MCMC) simulations. Shrinkage based on P4 is related to differences in the effects of adjacent markers because the <em>γ</em>s influence the entries of the lower triangular matrix <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-88.gif"/></noscript></span></span>.</p><p id="p-21">Furthermore, the overall mean and residual variance component are distributed <em>a posteriori</em> as<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-89.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-90.gif"/></noscript></span></span>The columns of <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-91.gif"/></noscript></span></span> are centered to favor mixing of the MCMC algorithm (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Stranden and Christensen 2011</a>).</p></div><div id="sec-5" class="subsection"><h3>Criteria for evaluation</h3><p id="p-22">The accuracy of genetic value prediction was verified by cross-validation. As a measure of accuracy, the mean correlation was calculated between the estimated and simulated genetic values in test data sets, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-92.gif"/></noscript></span></span>. In addition, to compare the predictive ability of the chosen priors, the mean true genetic value (TGV) was calculated for individuals, which were selected based on their estimated genetic value (EGV) according to a given fraction of selection (<em>r</em>). A range of <em>r</em> values between 0.05 and 1 was considered for evaluation. It is expected that the different methods would yield different rankings for the EGV. In general, the maximum mean TGV is obtained when only a few individuals are selected as parents of future offspring, and it approaches zero if more individuals are selected. Thus, the performance of a method can be determined by comparing the relationship between the mean TGV and <em>r</em>.</p><p id="p-23">The precision of the estimates of the residual and genetic variance component was determined as the mean squared error (MSE) based on repeated simulations:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-93.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-94.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-95.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-96.gif"/></noscript></span></span> are the estimated residual and genetic variance in the <em>l</em>th training block, respectively, <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-97.gif"/></noscript></span></span>. Given the estimated marker effects, the genetic variance was estimated as <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-98.gif"/></noscript></span></span>. The true genetic variance was calculated based on simulated marker effects as <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-99.gif"/></noscript></span></span> (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bonk <em>et al.</em> 2016</a>). This formula considers the contribution of LD to the genetic variance (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gianola <em>et al.</em> 2013</a>). Furthermore, the precision of the estimated effects at selected key SNPs was verified by the SD of traced samples based on one MCMC run. The SNPs at the five simulated quantitative trait loci (QTL) with largest effects and, for each of the two largest QTL, two SNPs being in high or low LD with the QTL, and within a window of 20 SNPs to both sides were selected as key SNPs.</p><p id="p-24">An appropriate measure of significance was required to evaluate the suitability of the suggested priors for understanding the genetic architecture, particularly relevant genomic regions. First, because the SNP effects were correlated, segment effects <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-100.gif"/></noscript></span></span> were calculated for a sliding window with an arbitrary width, which covered 11 SNPs, in the <em>l</em>th sampling round. These effects resemble haplotype effects. Second, the posterior probability of being positive was obtained from the Gibbs samples:<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-101.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-102.gif"/></noscript></span></span> denotes the indicator function, and <em>N</em> the number of Gibbs samples after the burn-in period. If <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-103.gif"/></noscript></span></span> (positive effect size), or <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-104.gif"/></noscript></span></span> (negative effect size), the segment was declared to have a nonzero effect on <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-105.gif"/></noscript></span></span>. This quantity represents a measure of evidence, which is a Bayesian analog of the <em>P</em>-value, in a similar manner to that described by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">De Braganca Pereira and Stern (1999)</a>.</p></div><div id="sec-6" class="subsection"><h3>Data</h3><p id="p-25">Two sets of data were explored. First, simulated data were used to evaluate the performance of the Bayesian approach depending on the different prior choices. Second, real genotype data were used to study the pattern of covariance between SNPs in a real half-sib family and for a particular chromosome. A real phenotype was not analyzed because the present study focused on the impact of the covariance matrix on the outcomes of genomic evaluations. Thus, challenges related to real observations (other nuisance effects or uncertainty about genetic effects on the selected chromosome) were eliminated completely. Hence, a phenotype was simulated based on the real genotypes to investigate the feasibility of the method in a general manner.</p><div id="sec-7" class="subsection"><h4>Simulated data:</h4><p id="p-26">The genomic data were simulated using a synthetic and simplified approach, but the structure obtained for the dependencies resembled a realistic setting. Further details are provided in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS1.pdf">File S1</a>. In total, 500 marker genotypes for 10,000 progeny were simulated on a chromosome segment with a length of 12 cM, but only the loci at which the sire was heterozygous were considered in further analyses (<span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-106.gif"/></noscript></span></span>). The SNP alleles were recoded, so the sire haplotypes were AA/BB regardless of the allele frequencies. The coding of alleles only affected the sign of the covariance and not the estimated effect size.</p><p id="p-27">To simulate phenotypic observations, the effects of either five or 50 QTL were drawn randomly from a gamma distribution with shape parameter <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-107.gif"/></noscript></span></span>, and scale parameter <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-108.gif"/></noscript></span></span>. The sign was sampled with equal likelihood. In the five-QTL scenario, the second largest QTL was placed adjacent to the largest QTL, with four SNPs between them to complicate its detection. A residual error was added, and the resulting phenotype was scaled to have a variance of one. Finally, the simulated residual variance component <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-109.gif"/></noscript></span></span> was 0.500, and, considering the formula for additive genetic variance given above, the simulated additive genetic variance <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-110.gif"/></noscript></span></span> was 0.487 in the five-QTL scenario, and 0.489 in the 50-QTL scenario. The QTL were taken from the SNP set, and they remained in the data.</p><p id="p-28">The accuracy of the predicted genetic value, and the estimated variance components was evaluated using a <em>B</em>-fold cross-validation. For that, the complete data set (<span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-111.gif"/></noscript></span></span>) was split into successive blocks with a training set size of <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-112.gif"/></noscript></span></span> (small sample size, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-113.gif"/></noscript></span></span> repetitions) or <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-114.gif"/></noscript></span></span> (medium sample size, <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-115.gif"/></noscript></span></span> repetitions).</p></div><div id="sec-8" class="subsection"><h4>Semireal data:</h4><p id="p-29">The data set comprised a single half-sib family of Holstein-Friesian cows (<span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-116.gif"/></noscript></span></span>), which were initially genotyped with a 50K SNP chip, where the complete data set was described by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wittenburg <em>et al.</em> (2013)</a>. The sire was phased based on the daughter genotypes using the R package <em>hsphase</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ferdosi <em>et al.</em> 2014</a>). For convenience, only <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-117.gif"/></noscript></span></span> SNPs at which the sire was heterozygous were selected from BTA1. The SNP alleles were recoded corresponding to the sire haplotypes AA/BB. The paternal recombination rate and LD of maternal gametes were estimated by numerical maximization (NM) of the log-likelihood function using the R function <em>optim</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">R Core Team 2014</a>) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS1.pdf">File S1</a>). It was found that 407 of the 903 eigenvalues of <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-118.gif"/></noscript></span></span> were negative, so the bending algorithm proposed by <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jorjani <em>et al.</em> (2003)</a> was employed to obtain a positive definite approximation of the covariance matrix. A phenotype was simulated based on five QTL (<span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-119.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-120.gif"/></noscript></span></span>), as described above.</p><p id="p-30">The data are provided as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS2.zip">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS3.zip">File S3</a>. The physical order of the SNPs on BTA1 followed the Btau4.2 annotation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS4.txt">File S4</a>).</p></div></div><div id="sec-9" class="subsection"><h3>MCMC computing</h3><p id="p-31">(Metropolis-within-) Gibbs sampling algorithms were implemented in Fortran 90 embedding LAPACK 3.5.0 (<a href="http://www.netlib.org/lapack">www.netlib.org/lapack</a>) and module <em>random</em> 1.13 (<a href="http://jblevins.org/mirror/amiller">jblevins.org/mirror/amiller</a>). The program ran on a 2.93 GHz multi-user system. A single chain comprising 50,000 sampling rounds was executed, and 20,000 iterations were omitted as the burn-in. The R package <em>coda</em> was used for MCMC diagnostics. In particular, the effective sample size (ESS) and Heidelberger and Welch’s test, which tests the null hypothesis that the samples were drawn from a stationary distribution, were employed to determine the convergence of a Markov chain.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-32">The authors state that all data necessary for confirming the conclusions presented in the article are given in the Supplemental Material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS2.zip">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS3.zip">File S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS4.zip">File S4</a>).</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Simulated data</h3><p id="p-33">The diagonal elements of the theoretical covariance matrix <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-121.gif"/></noscript></span></span> ranged from 0.473 to 0.500, with a mean value of 0.492. The off-diagonal elements varied from 0.174 to 0.331 around a mean of 0.231. The covariance decreased gradually with increasing distance between the SNPs. After conversion into correlations, the off-diagonal entries ranged from 0.352 to 0.667, with a mean value of 0.470. As shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, the theoretical and observed correlations were compared for a randomly selected SNP in a window of 200 SNPs, where the theoretical correlation decreased rapidly from the diagonal to off-diagonal elements, but this agreed with the values observed based on the genotypes. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> shows the paternal contribution (<em>i.e.</em>, linkage) to the covariance between SNPs, as discussed later.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Theoretical vs. empirical correlation for a randomly selected SNP based on the simulated genotypes. (B) Contribution of linkage (paternal part) to the covariance between SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-208432816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Theoretical &lt;em&gt;vs.&lt;/em&gt; empirical correlation for a randomly selected SNP based on the simulated genotypes. (B) Contribution of linkage (paternal part) to the covariance between SNPs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2761/F1.medium.gif" width="305" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/9/2761/F1.medium.gif" width="305" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469006" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-34" class="first-child">(A) Theoretical <em>vs.</em> empirical correlation for a randomly selected SNP based on the simulated genotypes. (B) Contribution of linkage (paternal part) to the covariance between SNPs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">MCMC diagnostics indicated the convergence of the MCMC algorithm for all of the proposed priors. ESS at key SNP effects was rather high, <em>i.e.</em>, &gt; 10,309, and &gt; 30,000 for a few exceptions; these values hardly differed between SNPs being the QTL or not. Heidelberger and Welch’s test was generally passed. The test failed only at the key SNP with the smallest simulated effect in the five-QTL scenario when the prior P4 was selected. Furthermore, for the hybrid strategy with prior P3, the acceptance rate <em>α</em> ranged from 0.54 to 0.79 (from 0.55 to 0.80) in the five-QTL (50-QTL) scenario.</p><p id="p-36">For all of the selected priors, and with five and 50 simulated QTL, the estimated SNP effects agreed well with the simulated effects when the complete data set (<span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-122.gif"/></noscript></span></span>) was used. Most of the shrinkage effects were observed with the correlated prior P2. Many spurious effects were estimated with small effect sizes using all of the correlated prior choices. As expected, the empirical SD of key SNP effects was lowest when based on the complete data set, and it increased as the sample size decreased (<em>e.g.</em>, see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A–C</a> for the five-QTL scenario). In most cases, the SD of SNP effects at the QTL was lowest with the correlated prior P2, irrespective of the sample size. At non-QTL, smallest SD was obtained with P2 only for the small sample size. This was attributed to the very high shrinkage due to the prior P2.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation with five QTL. SD of estimated effects at key SNPs for different sample sizes based on one MCMC run:  (A),  (B),  (C). (D) Mean of TGV of individuals that were selected by their EGV based on 100-fold cross-validation (size of training set )." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-208432816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation with five QTL. SD of estimated effects at key SNPs for different sample sizes based on one MCMC run: &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-123&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (A), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-124&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (B), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-125&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (C). (D) Mean of TGV of individuals that were selected by their EGV based on 100-fold cross-validation (size of training set &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-126&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2761/F2.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/9/2761/F2.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469174" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">Simulation with five QTL. SD of estimated effects at key SNPs for different sample sizes based on one MCMC run: <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F2/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F2/embed/mml-math-123.gif"/></noscript></span></span> (A), <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F2/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F2/embed/mml-math-124.gif"/></noscript></span></span> (B), <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F2/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F2/embed/mml-math-125.gif"/></noscript></span></span> (C). (D) Mean of TGV of individuals that were selected by their EGV based on 100-fold cross-validation (size of training set <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F2/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F2/embed/mml-math-126.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">For the five-QTL scenario and based on one MCMC run, few significant chromosome segments were falsely detected when one of the correlated priors was selected. However, for small sample sizes, repeated simulations showed that using the uncorrelated prior P1 and correlated prior P2, only one significant segment at the end of the chromosome was identified correctly (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The relevant region was smaller with the uncorrelated prior assumption. No significant segments were found with priors P3 and P4. For the 50-QTL scenario and using all priors, hardly any chromosome segments were detected due to the high level of uncertainty. For a medium sample size, the identification of significant segments was generally improved, and it even reached 100% for the largest QTL with all of the priors (results not shown).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation with five QTL, , and 100 repetitions. Detection of nonzero segment effects using the uncorrelated prior P1 (A) and correlated prior P2 (B). Gray dots indicate the simulated QTL positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-208432816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation with five QTL, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-127&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and 100 repetitions. Detection of nonzero segment effects using the uncorrelated prior P1 (A) and correlated prior P2 (B). Gray dots indicate the simulated QTL positions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2761/F3.medium.gif" width="224" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/9/2761/F3.medium.gif" width="224" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469105" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Simulation with five QTL, <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F3/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F3/embed/mml-math-127.gif"/></noscript></span></span>, and 100 repetitions. Detection of nonzero segment effects using the uncorrelated prior P1 (A) and correlated prior P2 (B). Gray dots indicate the simulated QTL positions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">The estimated variance components differed only slightly among the four methods for the complete data set, which were estimated to range from 0.475 to 0.497 for <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-128.gif"/></noscript></span></span>, and from 0.503 to 0.507 for <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-129.gif"/></noscript></span></span> in the five-QTL scenario (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In the 50-QTL scenario, the estimates ranged from 0.499 to 0.522 for <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-130.gif"/></noscript></span></span>, and from 0.486 to 0.488 for <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-131.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The MSE of the estimated variance components showed that the precision was greatest with the correlated prior P2 irrespective of the number of QTL when the sample size was small, as well as with a medium sample size and 50 QTL (see <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469219/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469219/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Estimated variance components based on one MCMC run and <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/T1/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/T1/embed/mml-math-132.gif"/></noscript></span></span> observations</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469072/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469072/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean squared error (MSE) of the estimated variance components</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469146/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469146/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">MSE of the estimated variance components</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">The accuracy of the genetic value predictions was highest for the correlated prior P2 only with <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-153.gif"/></noscript></span></span>; otherwise, the highest accuracy was obtained using the uncorrelated prior P1. The decrease in the mean TGV according to the fraction (<em>r</em>) of selected candidates gave mixed results (<em>e.g.</em>, see <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2D</a>), where no method performed especially well with five QTL. The mean TGV was highest with the correlated prior P2 only when <em>r</em> was between about 30% and 70% in the five-QTL scenario, but this prior was generally better with 50 simulated QTL. The adaptive priors P3 and P4 generally performed worse in terms of all the evaluation criteria.</p><p id="p-45">Additional figures showing the results are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS1.pdf">File S1</a>. Moreover, as part of this material, a simple simulation study was conducted to determine the shape of the theoretical covariance according to the recombination rate between a pair of SNPs. The average theoretical covariance agreed well with the empirical covariance obtained based on the progeny genotypes.</p></div><div id="sec-13" class="subsection"><h3>Semireal data</h3><p id="p-46">There was a good agreement between the theoretical and empirical correlations between SNP genotypes. The pattern of the correlations is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS1.pdf">File S1</a>. A gradual decrease in the entries was found with increasing distance between the SNPs. The off-diagonal values ranged from <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-154.gif"/></noscript></span></span> to 1.000, with a mean value of 0.219. The low minimum value was due to an extreme estimate of the LD, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-155.gif"/></noscript></span></span> between SNP 99 and 619, where the corresponding covariance was <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-156.gif"/></noscript></span></span>. If the distance between SNPs is rather large, then an extreme covariance indicates a potential error in the marker map. This hypothesis is supported by the values in the 99th row of <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-157.gif"/></noscript></span></span>, which is the matrix of the pairwise LD values. It was expected that the maximum at around position 99 would decrease more or less smoothly on both sides, but this was not the case for this SNP, unlike SNP 619 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated maternal LD with two selected SNPs on BTA1 for the real genotypes. The vertical line refers to the SNP with which pairwise LD was calculated. Smoothing via B-splines visualizes the trend of the data (red curves). (A) LD pattern shows a potential error in the marker map. (B) Maximum LD was observed around the reference SNP indicating correct positioning." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-208432816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated maternal LD with two selected SNPs on BTA1 for the real genotypes. The vertical line refers to the SNP with which pairwise LD was calculated. Smoothing via B-splines visualizes the trend of the data (red curves). (A) LD pattern shows a potential error in the marker map. (B) Maximum LD was observed around the reference SNP indicating correct positioning.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2761/F4.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/9/2761/F4.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469080" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-47" class="first-child">Estimated maternal LD with two selected SNPs on BTA1 for the real genotypes. The vertical line refers to the SNP with which pairwise LD was calculated. Smoothing via B-splines visualizes the trend of the data (red curves). (A) LD pattern shows a potential error in the marker map. (B) Maximum LD was observed around the reference SNP indicating correct positioning.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">Using MCMC computing, ESS at key SNP effects varied between 324 and 39,910, and was lowest with prior P2. However, ESS differed slightly between SNPs at the QTL and non-QTL. Heidelberger and Welch’s test was passed by all of the priors, which indicated that the Markov chain converged to a stationary distribution. Furthermore, the acceptance rate <em>α</em> ranged from 0.13 to 0.25 with prior P3.</p><p id="p-49">The estimated SNP effects were far from their simulated effect sizes, which was due mainly to the small sample size causing a high amount of uncertainty. Using the correlated prior P2, which was the approach with the highest degree of shrinkage, a significant segment at the end of the chromosome was identified correctly. The variance components were estimated as <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-158.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-159.gif"/></noscript></span></span>. The other prior choices yielded variance component estimates &gt; 2 × these values.</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-50">Our theoretical investigations have shown that the covariance between SNPs depends on the genetic distance and the maternal LD between them. In this study, the covariance matrix derived for a single half-sib family was employed in genomic evaluations of a simulated phenotype with heritability of about 50%. The results depended greatly on the sample size, but the inclusion of the covariance matrix was clearly beneficial for small sample sizes (<span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-160.gif"/></noscript></span></span>) in terms of both the MSE of the additive and residual variance components as well as the accuracy of the genetic value predictions, although there were no advantages with larger sample sizes (<span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-161.gif"/></noscript></span></span>). These variable outcomes may have several explanations, which are related to the nature of covariance in the selected population design and the different prior assumptions.</p><div id="sec-15" class="subsection"><h3>Covariance matrix</h3><p id="p-51">Previous studies have suggested that some covariance or correlation structure for marker effects should be included in regression models to estimate their effects. Candidates selected from covariance structures have proved useful in other fields, such as time series analysis, <em>i.e.</em>, equally spaced autoregressive, Toeplitz and Gaussian decay (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gianola <em>et al.</em> 2003</a>), and first-order antedependence (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Yang and Tempelman 2012</a>). All of these methods assume some decay of the correlations with increasing distance between the markers, thereby considering the fact that recombination becomes more frequent when markers are more distant. However, in contrast to these previous methods, our derivation makes explicit use of well-established genetic arguments, and it is based on the recombination frequencies, Haldane’s underlying mapping function (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Bonk <em>et al.</em> 2016</a>), and the population-wide pairwise LD. Basically, the covariance between SNPs is determined as the sum of a paternal and maternal part. Assuming many half-sibs, the paternal part is considered for coinheritance (linkage), whereas the maternal part models the population-wide LD. The sign and size of the correlations are known <em>a priori</em> (with the exception of the second adaptive method, P4) in contrast to the antedependence model proposed by <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Yang and Tempelman (2012)</a>. In the latter model, the sign and size of the regression for a marker effect on that of its predecessor are determined during each iteration by sampling from a normal distribution. Both adaptive versions allow for some variation in the covariances because each sampled gamma parameter affects both the diagonal and off-diagonal elements of the inverse covariance matrix. In non-QTL regions, small <em>γ</em>s are expected, whereas large <em>γ</em>-values are related to QTL regions. This was also observed in our data analyses (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS1.pdf">File S1</a>). Therefore, such a covariance structure may be classified as nonstationary in a similar manner to the antedependence model because the covariances depend on the distances between markers and on the marker positions, <em>i.e.</em>, their proximity to QTL.</p><p id="p-52">Unlike haplotype-based approaches (<em>e.g.</em>, <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Calus <em>et al.</em> 2008</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Cuyabano <em>et al.</em> 2014</a>), which require that all of the individuals are phased, only the sire’s haplotypes need to be known when using the proposed method. In the present study, we focused on a half-sib family structure, but other population structures are also relevant for livestock, <em>e.g.</em>, full-sib families often occur in chickens. Gametes of full-sibs are derived from a single sire and dam, and therefore the covariance between SNPs is restricted to the linkage part. (Paternal half-sibs represent a special case of a population with full-sib family structure, and the maternal gametes can be seen as random samples of a population with corresponding LD.) Then, the haplotypes of both parents should be considered when setting up the covariance matrix, as shown by <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Bonk <em>et al.</em> (2016)</a>. A typical livestock population comprises a mixture of families, so population-specific effects may be required instead of family-specific effects. The extension of the covariance matrix to a multiple-family study will be investigated in future research. The population covariance matrix could be set up as a weighted average of family-specific covariance terms, where the weight depends on the family size relative to the total sample size. Family sizes are often much smaller than 100, especially for full-sibs, so including the covariance between SNPs in genomic evaluations is expected to be beneficial, but this should be confirmed in a subsequent study.</p><p id="p-53">All of the parameters required for the covariance matrix <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-162.gif"/></noscript></span></span> were available for the simulated data. However, the recombination rate and LD of the maternal gametes had to be estimated from the progeny genotypes for the real data. In this study, estimates were obtained by NM of the log-likelihood function. Alternatively, <em>θ</em> and <em>D</em> could be estimated by the expectation-maximization (EM) algorithm (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gomez-Raya <em>et al.</em> 2013</a>). However, this iterative approach is more time consuming than the iterations required for NM. There was good agreement between the theoretical and empirical covariances obtained from the progeny genotypes, but the covariance matrix was indefinite for the real data genotypes (and positive definite for the simulated data), and thus it was forced to be positive definite by bending. This was probably caused by errors in the estimation of the required parameters, which could be estimated more precisely using larger families.</p><p id="p-54">When the alleles of the sire haplotypes were coded according to the observed minor allele frequency in the sample, both of the sire diplotypes AA/BB and AB/BA appeared in the double heterozygous case. The covariance matrix had then a block structure, which represented the nonrecombinant genome segments.</p></div><div id="sec-16" class="subsection"><h3>Prior assumptions</h3><p id="p-55">In this study, each of the correlated priors incorporated a dense covariance matrix, and its inverse is also dense. According to <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lang <em>et al.</em> (2002)</a>, locally adaptive random-walk priors penalize the difference between successive effects, thereby allowing local regularization to avoid oversmoothing. The inverse <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-163.gif"/></noscript></span></span>, which is also called the precision matrix, could be adjusted to have a banded structure to allow better local adaptivity in genomic evaluations. As an option, sparseness can be achieved by eliminating the “noise”. In the present study, the paternal contribution (<em>i.e.</em>, linkage) to the covariance matrix was more important than the maternal part (<em>i.e.</em>, population LD), where the interquartile range of <em>D</em> was zero in the simulations (mean <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-164.gif"/></noscript></span></span>) and 0.064 in the real genotypes (mean <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-165.gif"/></noscript></span></span>). Thus, an approximation of <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-166.gif"/></noscript></span></span> retained only the paternal part <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-167.gif"/></noscript></span></span>. <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a> shows the pattern for a randomly selected SNP. This simple structure can be inverted theoretically by exploiting, for instance, <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-168.gif"/></noscript></span></span>, based on Haldane’s mapping function. The dependence throughout the genome represents an autoregressive process, so the inverse covariance matrix depends only on the predecessor and successor, thereby yielding a three-band matrix that fully considers the unequal distances between SNPs. This type of sparse structure has also been derived and implemented using a Gibbs sampling strategy for a backcrossed population by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Reichelt <em>et al.</em> (2015)</a>. Using this structure, a more local impact of the regularization parameters is obtained using the adaptive priors P3 and P4, and the error propagation caused by numerical imprecisions along the chromosome appears to be reduced. The influence of this structure on the performance of the correlated prior selections is shown in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>, and in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS1.pdf">File S1</a>. In particular, the MSE was smallest using the adaptive priors P3 and P4. Furthermore, the identification of significant chromosome segments was improved, and slightly superior to the uncorrelated prior P1 (<em>e.g.</em>, see <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> for the five-QTL scenario). During the analysis of real data, this approximation has an additional positive impact because only the recombination rate needs to be estimated. Unlike classical linkage analysis or linkage/LD analysis (<em>e.g.</em>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Meuwissen <em>et al.</em> 2002</a>), which is applied to a marker bracket instead of all markers simultaneously, this restriction to the linkage part, works well, even though the parental origin of the SNP alleles has not been identified.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469231/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469231/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">MSE of the estimated variance components when the inverse covariance matrix was sparse</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469181/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469181/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">MSE of the estimated variance components when the inverse covariance matrix was sparse</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation with five QTL, , and 100 repetitions. Detection of nonzero segment effects using the sparse inverse covariance matrix and uncorrelated prior P1 (A), correlated prior P2 (B), adaptive prior P3 (C), and adaptive prior P4 (D). Gray dots indicate the simulated QTL positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-208432816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Simulation with five QTL, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-183&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and 100 repetitions. Detection of nonzero segment effects using the &lt;strong&gt;sparse&lt;/strong&gt; inverse covariance matrix and uncorrelated prior P1 (A), correlated prior P2 (B), adaptive prior P3 (C), and adaptive prior P4 (D). Gray dots indicate the simulated QTL positions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2761/F5.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/9/2761/F5.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2761/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469251" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-58" class="first-child">Simulation with five QTL, <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F5/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/F5/embed/mml-math-183.gif"/></noscript></span></span>, and 100 repetitions. Detection of nonzero segment effects using the <strong>sparse</strong> inverse covariance matrix and uncorrelated prior P1 (A), correlated prior P2 (B), adaptive prior P3 (C), and adaptive prior P4 (D). Gray dots indicate the simulated QTL positions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">A second option for approximation uses the modified Cholesky decomposition of the inverse matrix, <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-184.gif"/></noscript></span></span>, which is suitable for P4. <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wu and Pourahmadi (2003)</a> derived a smoothing algorithm for <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-185.gif"/></noscript></span></span> along its subdiagonals for longitudinal data and <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-186.gif"/></noscript></span></span>. Future studies should investigate whether this banded structure can also be derived for <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-187.gif"/></noscript></span></span>. The sparse structure of the inverse matrix is also a consequence of the antedependence specification proposed by <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Yang and Tempelman (2012)</a>. Exploiting a sparse structure in the MCMC algorithm will also improve the computational speed (<em>e.g.</em>, for <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-188.gif"/></noscript></span></span>, the current implementation required on average 15 min based on prior P1, and 48 min based on prior P4.)</p><p id="p-60">The prior covariance matrix used in P3 is similar to that described by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">George and McCulloch (1993)</a> in terms of the dynamic regularization parameters (<em>γ</em>s), although it is implemented as a variable selection approach. In the present study, variable selection was not executed because it was assumed that each marker contributes to the genetic variation, at least indirectly through LD. However, this strategy led to poor mixing in the MCMC algorithm compared with the adaptive prior P4 using semireal data (results not shown). The trace plots contained repetitions of effect samples with low variability, followed by samples with very high variability, which was not observed to the same extent for the simulated data with small sample sizes. The correlated prior P2 was the most robust prior choice for both the simulations and semireal data because it could detect significant chromosome segments and estimate the variance components with the smallest MSE. To facilitate realistic genome-wide evaluations, we recommend specifying chromosome-wise priors, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-189.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-190.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-191.gif"/></noscript></span></span>. Genome-wide shrinkage, which employs a single regularization parameter <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-192.gif"/></noscript></span></span>, may be too strict if only a few QTL are present.</p><p id="p-61">Bayesian approaches require the specification of prior distributions but the use of hyper-parameters was minimized in the present study. The sensitivity to the selected priors has been evaluated and discussed previously, <em>e.g.</em>, by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Knürr <em>et al.</em> (2013)</a> and <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gianola (2013)</a>. In the adaptive prior P3, the tuning variance was set to <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-193.gif"/></noscript></span></span>, which made this approach sensitive, and it compromised the repeatability of the experiments. For example, for different choices of <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-194.gif"/></noscript></span></span>, a varying acceptance rate <em>α</em> was obtained, where <em>α</em> was higher when <em>ε</em> was smaller. However, the impact on a single-effect estimate was negligible for both the simulated and semireal data, whereas the estimated variance components <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-195.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-196.gif"/></noscript></span></span> differed in terms of the second decimal place (results not shown). With <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-197.gif"/></noscript></span></span>, the range of <em>α</em> roughly agreed with the rule of thumb given by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Besag <em>et al.</em> (1995)</a> who showed that an acceptance rate between about 30% and 70% was often satisfactory. Alternatively, the tuning variance may be adjusted during the burn-in phase to obtain an intermediate <em>α</em> (25–50%) in a similar manner to <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Yang <em>et al.</em> (2015)</a>.</p></div><div id="sec-17" class="subsection"><h3>Sample size</h3><p id="p-62">The sample size had a major effect on the outcome. For small sample sizes (<span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-198.gif"/></noscript></span></span>), the effect estimates were poor, and a high EGV was obtained even for those individuals with low TGV. Thus, the curves of the mean TGVs (<em>e.g.</em>, in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2D</a> based on five simulated QTL) started below 0.900 at a selection rate <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-199.gif"/></noscript></span></span>, although the 95% quantile for the TGV was 1.106. The shape of the curve based on the correlated prior choice P2 was unusual, where it decreased very slightly until <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-200.gif"/></noscript></span></span>, and it then declined rapidly, which may be explained by the following two reasons. First, the SNP effects were greatly reduced toward zero due to the major effect of the shrinkage parameter <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-201.gif"/></noscript></span></span>. Second, the sire was heterozygous at all of the loci considered (with A alleles on one strand and B alleles on the other), so there were only few recombinant offspring, and the distribution of EGV was bimodal (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS1.pdf">File S1</a>). The variation around the two modes was due to variation in the maternally inherited gametes. Thus, the individuals selected with the highest EGV comprised a mixture of high/medium/low-TGV individuals, and the curve obtained for the mean TGVs decayed slowly. The curve approached the population mean rapidly above <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-202.gif"/></noscript></span></span>. This outcome was also observed, but to a much greater extent, in simulations with 50 QTL as well as when combined with the sparse inverse covariance matrix (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1/FileS1.pdf">File S1</a>). This pattern was not detected with a medium sample size (<span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-203.gif"/></noscript></span></span>), probably because of the improved estimation of the parameters, and the occurrence of more recombinant offspring. This phenomenon is unlikely to occur if more than one chromosome is investigated, thereby causing greater variation in the paternal gametes.</p></div><div id="sec-18" class="subsection"><h3>Conclusion</h3><p id="p-63">In this study, the dependence between SNP genotypes was derived theoretically from the genetic parameters for a half-sib family. Integrating this information into genomic evaluations improved the estimates of the variance components and the genetic value predictions when the sample size was small (<span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-204.gif"/></noscript></span></span>). Thus, in small populations, for which the parameter estimates are typically affected by high uncertainty when the number of predictors is large, additional information about the population structure can increase the precision, thereby following the general Bayesian principle. The proposed correlated prior choices could potentially obtain better overall performance if a locally adaptive approximation of dependence is employed.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-66">Special thanks are given to our colleagues at the Leibniz Institute for Farm Animal Biology (FBN, Dummerstorf, Germany), S. Bonk and M. Reichelt, who contributed to discussions during the project. We also thank the anonymous reviewers for their helpful comments. The real genotype data were obtained in the Fugato-plus project “BovIBI” funded by the German Federal Ministry of Education and Research (BMBF). The publication of this article was funded by the Open Access fund of the Leibniz Association and the FBN.</p><p id="p-67">Author contributions: D.W. developed the theory, implemented the statistical methods, performed the analysis, and wrote the manuscript. F.T. contributed to the theoretical investigations of the covariance between markers, and the estimation of the required parameters. J.K. developed the simulation design. N.R. raised the initial question and contributed to the discussions. All of the authors have read and approved the final manuscript. The authors declare that they have no competing interests.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-19"><h3><em>Prior P3</em></h3><p id="p-64">The target distribution <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-205.gif"/></noscript></span></span> is obtained from<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-206.gif"/></noscript></span></span>For convenience, it is <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-207.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-208.gif"/></noscript></span></span>. Hence,<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-209.gif"/></noscript></span></span>with <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-210.gif"/></noscript></span></span> employing the Hadamard product (#). Thus,<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-211.gif"/></noscript></span></span>The proposal density is lognormal, <em>i.e.</em>,<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-212.gif"/></noscript></span></span>Except the last ratio, this function is symmetric in <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-213.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-214.gif"/></noscript></span></span>. Let <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-215.gif"/></noscript></span></span> denote the vector of <em>τ</em>s, where the <em>j</em>th component is replaced by the current proposed value, and <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-216.gif"/></noscript></span></span> refers to the vector of samples from the last iteration. Finally, the ratio <em>R</em> is</p><span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-217.gif"/></noscript></span></span></div><div class="section" id="sec-20"><h3><em>Prior P4</em></h3><p id="p-65">The posterior density of regularization parameters is derived as<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-218.gif"/></noscript></span></span>Set <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-219.gif"/></noscript></span></span>, then<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-220.gif"/></noscript></span></span>Thus, the <em>γ</em>s are conditionally independent, each distributed as scaled <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2761/embed/mml-math-221.gif"/></noscript></span></span>.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-68">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032409/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-69"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-03-19"><span class="received-label">Received </span>March 19, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-06-22"><span class="accepted-label">Accepted </span>June 22, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Wittenburg <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.1" data-doi="10.1214/ss/1177010123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Besag</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higdon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mengersen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayesian computation and stochastic systems.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D10%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1177010123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/ss/1177010123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RG25500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.2" data-doi="10.1186/s12711-015-0181-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reichelt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teuscher</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segelke</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinsch</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mendelian sampling covariability of marker effects and genetic values.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0181-x%26rft_id%253Dinfo%253Apmid%252F26743767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0181-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26743767&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.3" data-doi="10.1534/genetics.107.080838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of genomic selection using different methods to define haplotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">553</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%2BL.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D553%26rft.epage%253D561%26rft.atitle%253DAccuracy%2Bof%2BGenomic%2BSelection%2BUsing%2BDifferent%2BMethods%2Bto%2BDefine%2BHaplotypes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080838%26rft_id%253Dinfo%253Apmid%252F18202394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.4" data-doi="10.1186/1471-2164-15-1171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuyabano</span>  <span class="cit-name-given-names">B. C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction of genetic merit using LD-based haplotypes in the Nordic Holstein population.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1171%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1171%26rft_id%253Dinfo%253Apmid%252F25539631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25539631&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.5" data-doi="10.1038/ng.3034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capitan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pausch</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stothard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Binsbergen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">858</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D858%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3034%26rft_id%253Dinfo%253Apmid%252F25017103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25017103&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.6" data-doi="10.3390/e1040099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Braganca Pereira</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Evidence and credibility: full Bayesian significance test for precise hypotheses.</span> <abbr class="cit-jnl-abbrev">Entropy (Basel)</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">99</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEntropy%2B%2528Basel%2529%26rft.volume%253D1%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fe1040099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/e1040099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.7" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.8" data-doi="10.1186/s12859-015-0556-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dehman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambroise</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuvial</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Performance of a blockwise approach in variable selection using linkage disequilibrium information.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0556-6%26rft_id%253Dinfo%253Apmid%252F25951947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-015-0556-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25951947&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fahrmeir</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Bayesian inference for generalized additive mixed models based on Markov random field priors.</span> <abbr class="cit-jnl-abbrev">Appl. Stat.</abbr> <span class="cit-vol">50</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">201</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BStat.%26rft.volume%253D50%26rft.spage%253D201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.10" data-doi="10.1186/1471-2105-15-172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferdosi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinghorn</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gondro</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">hsphase: an R package for pedigree reconstruction, detection of recombination events, phasing and imputation of half-sib family groups.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-172%26rft_id%253Dinfo%253Apmid%252F24906803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24906803&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.11" data-doi="10.2307/2290777"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">E. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Variable selection via Gibbs sampling.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">88</span>(<span class="cit-issue">423</span>): <span class="cit-fpage">881</span>–<span class="cit-lpage">889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D88%26rft.spage%253D881%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2290777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2290777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LT93200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.12" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: the Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">On marker-assisted prediction of genetic value: beyond the ridge.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D347%26rft.epage%253D365%26rft.atitle%253DOn%2BMarker-Assisted%2BPrediction%2Bof%2BGenetic%2BValue%253A%2BBeyond%2Bthe%2BRidge%26rft_id%253Dinfo%253Apmid%252F12586721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.14" data-doi="10.1534/genetics.105.049510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic-assisted prediction of genetic value with semiparametric procedures.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1761</span>–<span class="cit-lpage">1776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049510%26rft_id%253Dinfo%253Apmid%252F16648593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE3NjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.15" data-doi="10.1007/s00122-013-2064-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hospital</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verrier</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Contribution of an additive locus to genetic variance when inheritance is multi-factorial with implications on interpretation of GWAS.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1457</span>–<span class="cit-lpage">1472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D1457%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2064-2%26rft_id%253Dinfo%253Apmid%252F23508282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2064-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23508282&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.16" data-doi="10.1186/2049-1891-4-30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Raya</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rauw</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Haplotype phasing after joint estimation of recombination and linkage disequilibrium in breeding populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci. Biotechnol.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%2BBiotechnol.%26rft.volume%253D4%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2049-1891-4-30%26rft_id%253Dinfo%253Apmid%252F23916349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2049-1891-4-30&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23916349&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.17" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.18" data-doi="10.3168/jds.S0022-0302(03)73646-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jorjani</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klei</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emanuelson</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A simple method for weighted bending of genetic (co)variance matrices.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">86</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">677</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DJorjani%26rft.auinit1%253DH.%26rft.volume%253D86%26rft.issue%253D2%26rft.spage%253D677%26rft.epage%253D679%26rft.atitle%253DA%2BSimple%2BMethod%2Bfor%2BWeighted%2BBending%2Bof%2BGenetic%2B%2528Co%2529variance%2BMatrices%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252803%252973646-7%26rft_id%253Dinfo%253Apmid%252F12647974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(03)73646-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12647974&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.19" data-doi="10.1186/1297-9686-45-24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knürr</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Läärä</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Impact of prior specifications in a shrinkage-inducing Bayesian model for quantitative trait mapping and genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-24%26rft_id%253Dinfo%253Apmid%252F23834140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23834140&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.20" data-doi="10.1007/s001800200121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fronk</span>  <span class="cit-name-given-names">E.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahrmeir</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Function estimation with locally adaptive dynamic models.</span> <abbr class="cit-jnl-abbrev">Comput. Stat.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">479</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStat.%26rft.volume%253D17%26rft.spage%253D479%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001800200121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001800200121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsaker</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Fine mapping of a quantitative trait locus for twinning rate using combined linkage and linkage disequilibrium mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">373</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D161%26rft.issue%253D1%26rft.spage%253D373%26rft.epage%253D379%26rft.atitle%253DFine%2BMapping%2Bof%2Ba%2BQuantitative%2BTrait%2BLocus%2Bfor%2BTwinning%2BRate%2BUsing%2BCombined%2BLinkage%2Band%2BLinkage%2BDisequilibrium%2BMapping%26rft_id%253Dinfo%253Apmid%252F12019251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYxLzEvMzczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.23" data-doi="10.2135/cropsci2008.10.0595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ridge regression and extensions for genomewide selection in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D1165%26rft.epage%253D1176%26rft.atitle%253DRidge%2BRegression%2Band%2BExtensions%2Bfor%2BGenomewide%2BSelection%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.10.0595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.10.0595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000268109800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.24" data-doi="10.1093/biomet/asm073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pourahmadi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Cholesky decompositions and estimation of a covariance matrix: orthogonality of variance–correlation parameters.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">94</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1006</span>–<span class="cit-lpage">1013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPourahmadi%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D4%26rft.spage%253D1006%26rft.epage%253D1013%26rft.atitle%253DCholesky%2BDecompositions%2Band%2BEstimation%2Bof%2BA%2BCovariance%2BMatrix%253A%2BOrthogonality%2Bof%2BVariance%2BCorrelation%2BParameters%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252Fasm073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjk6Ijk0LzQvMTAwNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yNzYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-web" id="cit-6.9.2761.25"><div class="cit-metadata unstructured">R Core Team, 2014 <em>R: a language and environment for statistical computing</em>. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>. Accessed: July 4, 2014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.9.2761.26"><div class="cit-metadata unstructured">Reichelt, M., M. Mayer, F. Teuscher, and N. Reinsch, 2015 Bayesian modelling of marker effects in backcross experiments by means of their covariance matrix. Abstract Volume of the 61st Biometrical Colloquium, Biometrics and Communication: From Statistical Theory to Perception in the Public, Dortmund, Germany, p. 124.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2761.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source"><em>Likelihood</em>, <em>Bayesian and MCMC Methods in Quantitative Genetics</em></span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.28" data-doi="10.1186/1297-9686-43-25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stranden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Allele coding in genomic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D43%26rft.spage%253D25%26rft.epage%253D25%26rft.atitle%253DAllele%2Bcoding%2Bin%2Bgenomic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-25%26rft_id%253Dinfo%253Apmid%252F21703021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21703021&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.29" data-doi="10.1111/j.1467-9868.2005.00490.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saunders</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosset</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Sparsity and smoothness via the fused lasso.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">67</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">91</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D67%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00490.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00490.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.30" data-doi="10.1214/aos/1176325750"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tierney</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Markov chains for exploring posterior distributions.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1701</span>–<span class="cit-lpage">1728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D22%26rft.spage%253D1701%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176325750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176325750&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994QX01100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.31" data-doi="10.1051/gse:19940202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutledge</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Bayesian analysis of mixed linear models via Gibbs sampling with an application to litter size in Iberian pigs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">91</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D26%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19940202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19940202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NL04000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.32" data-doi="10.3168/jds.2012-5635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittenburg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melzer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willmitzer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesting</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Milk metabolites and their genetic variability.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">2557</span>–<span class="cit-lpage">2569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D96%26rft.spage%253D2557%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2012-5635%26rft_id%253Dinfo%253Apmid%252F23403187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2012-5635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23403187&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.33" data-doi="10.1093/biomet/90.4.831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pourahmadi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Nonparametric estimation of large covariance matrices of longitudinal data.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">90</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">831</span>–<span class="cit-lpage">844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DWu%26rft.auinit1%253DW.%2BB.%26rft.volume%253D90%26rft.issue%253D4%26rft.spage%253D831%26rft.epage%253D844%26rft.atitle%253DNonparametric%2Bestimation%2Bof%2Blarge%2Bcovariance%2Bmatrices%2Bof%2Blongitudinal%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F90.4.831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjg6IjkwLzQvODMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating polygenic effects using markers of the entire genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">789</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D163%26rft.issue%253D2%26rft.spage%253D789%26rft.epage%253D801%26rft.atitle%253DEstimating%2BPolygenic%2BEffects%2BUsing%2BMarkers%2Bof%2Bthe%2BEntire%2BGenome%26rft_id%253Dinfo%253Apmid%252F12618414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzIvNzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.35" data-doi="10.1534/genetics.111.131540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Bayesian antedependence model for whole genome prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1491</span>–<span class="cit-lpage">1501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131540%26rft_id%253Dinfo%253Apmid%252F22135352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0OTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2761.36" data-doi="10.1186/s12711-015-0092-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Improving the computational efficiency of fully Bayes inference and assessing the effect of misspecification of hyperparameters in whole-genome prediction models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0092-x%26rft_id%253Dinfo%253Apmid%252F25885894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0092-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25885894&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/9/2761.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2745" title="Identification of QTLs Associated with Virulence Related Traits and Drug Resistance in Cryptococcus neoformans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2773" title="A Comparative Analysis of 5-Azacytidine- and Zebularine-Induced DNA Demethylation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/9"><span>Volume 6 Issue 9, September 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/9.cover-source.jpg?itok=nund1hRf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2761" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WhGWBjBhKhGDM-kwgngFj0_Awvf48CfXmFdUDSyke3Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/9/2761" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/9/2761" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/9/2761" class="active" data-icon-position="" data-hide-link-title="0">Covariance Between Genotypic Effects and its Use for Genomic Inference in Half-Sib Families</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460278" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Qek8grJGYAjErneqpPs6oMMLCS1pufq0m-tDwtfWvYw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2761.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/9/2761" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-agJ0nFPwOifTijF72HtVE2LXScVDsBpe2VPFz-5kfg0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/9/2761" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2761.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460278" id="node460278--2" data-pisa="ggg;6/9/2761" data-pisa-master="ggg;g3.116.032409" data-apath="/ggg/6/9/2761.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Covariance Between Genotypic Effects and its Use for Genomic Inference in Half-Sib Families</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dörte Wittenburg</span>, <span class="highwire-citation-author" data-delta="1">Friedrich Teuscher</span>, <span class="highwire-citation-author" data-delta="2">Jan Klosa</span> and <span class="highwire-citation-author" data-delta="3">Norbert Reinsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2761-2772;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032409 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460278--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dörte Wittenburg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%B6rte%2BWittenburg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wittenburg%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AD%25C3%25B6rte%2BWittenburg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wittenburg@fbn-dummerstorf.de" class="" data-icon-position="" data-hide-link-title="0">wittenburg@fbn-dummerstorf.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Friedrich Teuscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Friedrich%2BTeuscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teuscher%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFriedrich%2BTeuscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jan Klosa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BKlosa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Klosa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJan%2BKlosa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Norbert Reinsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norbert%2BReinsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reinsch%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANorbert%2BReinsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/9/2761" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/9/2761&amp;title=Covariance%20Between%20Genotypic%20Effects%20and%20its%20Use%20for%20Genomic%20Inference%20in%20Half-Sib%20Families" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/9/2761&amp;title=Covariance%20Between%20Genotypic%20Effects%20and%20its%20Use%20for%20Genomic%20Inference%20in%20Half-Sib%20Families" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/9/2761&amp;title=Covariance%20Between%20Genotypic%20Effects%20and%20its%20Use%20for%20Genomic%20Inference%20in%20Half-Sib%20Families" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/9/2761&amp;text=Covariance%20Between%20Genotypic%20Effects%20and%20its%20Use%20for%20Genomic%20Inference%20in%20Half-Sib%20Families" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/9/2761&amp;title=Covariance%20Between%20Genotypic%20Effects%20and%20its%20Use%20for%20Genomic%20Inference%20in%20Half-Sib%20Families" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/9/2761&amp;t=Covariance%20Between%20Genotypic%20Effects%20and%20its%20Use%20for%20Genomic%20Inference%20in%20Half-Sib%20Families" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/9/2761&amp;title=Covariance%20Between%20Genotypic%20Effects%20and%20its%20Use%20for%20Genomic%20Inference%20in%20Half-Sib%20Families" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/9/2761&amp;title=Covariance%20Between%20Genotypic%20Effects%20and%20its%20Use%20for%20Genomic%20Inference%20in%20Half-Sib%20Families" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460278/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460278" id="node460278--3" data-pisa="ggg;6/9/2761" data-pisa-master="ggg;g3.116.032409" data-apath="/ggg/6/9/2761.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Covariance Between Genotypic Effects and its Use for Genomic Inference in Half-Sib Families</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dörte Wittenburg</span>, <span class="highwire-citation-author" data-delta="1">Friedrich Teuscher</span>, <span class="highwire-citation-author" data-delta="2">Jan Klosa</span> and <span class="highwire-citation-author" data-delta="3">Norbert Reinsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2761-2772;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032409 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460278--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dörte Wittenburg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%B6rte%2BWittenburg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wittenburg%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AD%25C3%25B6rte%2BWittenburg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wittenburg@fbn-dummerstorf.de" class="" data-icon-position="" data-hide-link-title="0">wittenburg@fbn-dummerstorf.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Friedrich Teuscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Friedrich%2BTeuscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teuscher%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFriedrich%2BTeuscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jan Klosa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BKlosa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Klosa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJan%2BKlosa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Norbert Reinsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Leibniz Institute for Farm Animal Biology, Institute of Genetics and Biometry, 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norbert%2BReinsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reinsch%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANorbert%2BReinsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2761&amp;rft.epage=2772&amp;rft.atitle=Covariance%20Between%20Genotypic%20Effects%20and%20its%20Use%20for%20Genomic%20Inference%20in%20Half-Sib%20Families&amp;rft.volume=6&amp;rft.issue=9&amp;rft.date=2016-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wittenburg%2C+D%C3%B6rte&amp;rft.au=Teuscher%2C+Friedrich&amp;rft.au=Klosa%2C+Jan&amp;rft.au=Reinsch%2C+Norbert"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460278/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460278/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460278/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460278/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460278/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460278/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460278/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460278/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460278/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460278/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460278/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460278/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/9/2761" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/9/2761.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/9/2761.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/9/2761.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/9/2761.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
