<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/9/2799" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/9/2799.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/9/2799.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/9/2799.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/9/2799/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/9/2799.atom" />
<meta name="HW.pisa" content="ggg;6/9/2799" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Canopy Temperature and Vegetation Indices from High-Throughput Phenotyping Improve Accuracy of Pedigree and Genomic Selection for Grain Yield in Wheat" />
<meta name="DC.Identifier" content="10.1534/g3.116.032888" />
<meta name="DC.Date" content="2016-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Rutkoski et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection can be applied prior to phenotyping, enabling shorter breeding cycles and greater rates of genetic gain relative to phenotypic selection. Traits measured using high-throughput phenotyping based on proximal or remote sensing could be useful for improving pedigree and genomic prediction model accuracies for traits not yet possible to phenotype directly. We tested if using aerial measurements of canopy temperature, and green and red normalized difference vegetation index as secondary traits in pedigree and genomic best linear unbiased prediction models could increase accuracy for grain yield in wheat, Triticum aestivum L., using 557 lines in five environments. Secondary traits on training and test sets, and grain yield on the training set were modeled as multivariate, and compared to univariate models with grain yield on the training set only. Cross validation accuracies were estimated within and across-environment, with and without replication, and with and without correcting for days to heading. We observed that, within environment, with unreplicated secondary trait data, and without correcting for days to heading, secondary traits increased accuracies for grain yield by 56% in pedigree, and 70% in genomic prediction models, on average. Secondary traits increased accuracy slightly more when replicated, and considerably less when models corrected for days to heading. In across-environment prediction, trends were similar but less consistent. These results show that secondary traits measured in high-throughput could be used in pedigree and genomic prediction to improve accuracy. This approach could improve selection in wheat during early stages if validated in early-generation breeding plots." />
<meta name="DC.Contributor" content="Jessica Rutkoski" />
<meta name="DC.Contributor" content="Jesse Poland" />
<meta name="DC.Contributor" content="Suchismita Mondal" />
<meta name="DC.Contributor" content="Enrique Autrique" />
<meta name="DC.Contributor" content="Lorena González Pérez" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Matthew Reynolds" />
<meta name="DC.Contributor" content="Ravi Singh" />
<meta name="article:published_time" content="2016-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Canopy Temperature and Vegetation Indices from High-Throughput Phenotyping Improve Accuracy of Pedigree and Genomic Selection for Grain Yield in Wheat" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection can be applied prior to phenotyping, enabling shorter breeding cycles and greater rates of genetic gain relative to phenotypic selection. Traits measured using high-throughput phenotyping based on proximal or remote sensing could be useful for improving pedigree and genomic prediction model accuracies for traits not yet possible to phenotype directly. We tested if using aerial measurements of canopy temperature, and green and red normalized difference vegetation index as secondary traits in pedigree and genomic best linear unbiased prediction models could increase accuracy for grain yield in wheat, &lt;i&gt;Triticum aestivum&lt;/i&gt; L., using 557 lines in five environments. Secondary traits on training and test sets, and grain yield on the training set were modeled as multivariate, and compared to univariate models with grain yield on the training set only. Cross validation accuracies were estimated within and across-environment, with and without replication, and with and without correcting for days to heading. We observed that, within environment, with unreplicated secondary trait data, and without correcting for days to heading, secondary traits increased accuracies for grain yield by 56% in pedigree, and 70% in genomic prediction models, on average. Secondary traits increased accuracy slightly more when replicated, and considerably less when models corrected for days to heading. In across-environment prediction, trends were similar but less consistent. These results show that secondary traits measured in high-throughput could be used in pedigree and genomic prediction to improve accuracy. This approach could improve selection in wheat during early stages if validated in early-generation breeding plots.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/09/01" />
<meta name="citation_mjid" content="ggg;6/9/2799" />
<meta name="citation_id" content="6/9/2799" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/9/2799" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/9/2799.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/9/2799.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/9/2799.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.032888" />
<meta name="citation_pmid" content="27402362" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2799" />
<meta name="citation_lastpage" content="2808" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jessica Rutkoski" />
<meta name="citation_author_institution" content="International Programs, College of Agriculture and Life Sciences, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrated Plant Sciences, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico" />
<meta name="citation_author_email" content="jer263@cornell.edu" />
<meta name="citation_author" content="Jesse Poland" />
<meta name="citation_author_institution" content="Department of Plant Pathology, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author" content="Suchismita Mondal" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico" />
<meta name="citation_author" content="Enrique Autrique" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico" />
<meta name="citation_author" content="Lorena González Pérez" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico" />
<meta name="citation_author" content="Matthew Reynolds" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico" />
<meta name="citation_author" content="Ravi Singh" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. A. Babar;citation_author=M. P. Reynolds;citation_author=M. Van Ginkel;citation_author=A. R. Klatt;citation_author=W. R. Raun;citation_title=Spectral Reflectance to Estimate Genetic Variation for In-Season Biomass, Leaf Chlorophyll, and Canopy Temperature in Wheat;citation_pages=1046-1057;citation_volume=46;citation_year=2006;citation_issue=3;citation_doi=10.2135/cropsci2005.0211" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. P. L. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods.;citation_pages=26-26;citation_volume=43;citation_year=2011;citation_pmid=21729282;citation_doi=10.1186/1297-9686-43-26" />
<meta name="citation_reference" content="citation_volume=6" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=J. B. Endelman;citation_author=J.-L. Jannink;citation_title=Shrinkage estimation of the realized relationship matrix. G3 Genes Genome.;citation_pages=1405-1413;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Remote Sens.;citation_author=A. A. Gitelson;citation_author=M. N. Merzlyak;citation_title=Remote estimation of chlorophyll content in higher plant leaves.;citation_pages=2691-2697;citation_volume=18;citation_year=1997;citation_doi=10.1080/014311697217558" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: A high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with genomic selection: gain per unit time and cost.;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_volume=345;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-trait genomic selection methods increase genetic value prediction accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_title=Genomic selection in plant breeding: knowledge and prospects.;citation_pages=77-123;citation_volume=110;citation_year=2011;citation_doi=10.1016/B978-0-12-385531-2.00002-5" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=S. Mondal;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=Z. Kehel;citation_author=E. Autrique;citation_title=Characterization of heat- and drought-stress tolerance in high-yielding spring wheat.;citation_pages=1552-1562;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.10.0709" />
<meta name="citation_reference" content="citation_journal_title=F. Crop. Res.;citation_author=A. Pask;citation_author=A. K. Joshi;citation_author=Y. Manès;citation_author=I. Sharma;citation_author=R. Chatrath;citation_title=A wheat phenotyping network to incorporate physiological traits for climate change in south asia.;citation_pages=156-167;citation_volume=168;citation_year=2014;citation_doi=10.1016/j.fcr.2014.07.004" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. S. Pinto;citation_author=M. P. Reynolds;citation_author=K. L. Mathews;citation_author=C. L. McIntyre;citation_author=J. J. Olivares-Villegas;citation_title=Heat and drought adaptive QTL in a wheat population designed to minimize confounding agronomic effects.;citation_pages=1001-1021;citation_volume=121;citation_year=2010;citation_issue=6;citation_pmid=20523964;citation_doi=10.1007/s00122-010-1351-4" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod. Sci.;citation_author=J. E. Pryce;citation_author=H. D. Daetwyler;citation_title=Designing dairy cattle breeding schemes under genomic selection : a review of international research.;citation_pages=107-114;citation_volume=52;citation_year=2012;citation_doi=10.1071/AN11098" />
<meta name="citation_reference" content="citation_journal_title=Animal;citation_author=M. Pszczola;citation_author=R. F. Veerkamp;citation_author=Y. de Haas;citation_author=E. Wall;citation_author=T. Strabel;citation_title=Eff-ect of predictor traits on accuracy of genomic breeding values for feed intake based on a limited cow reference population.;citation_pages=1759-1768;citation_volume=7;citation_year=2013;citation_pmid=23915541;citation_doi=10.1017/S175173111300150X" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=M. Reynolds;citation_author=P. Langridge;citation_title=Physiological breeding.;citation_pages=162-171;citation_volume=31;citation_year=2016;citation_pmid=27161822;citation_doi=10.1016/j.pbi.2016.04.005" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=L. R. Schaeffer;citation_title=Sire and Cow Evaluation Under Multiple Trait Models;citation_pages=1567-1580;citation_volume=67;citation_year=1984;citation_issue=7;citation_doi=10.3168/jds.S0022-0302(84)81479-4" />
<meta name="citation_reference" content="citation_journal_title=Rationale. Agron. J.;citation_author=C. L. Wiegand;citation_author=A. J. Richardson;citation_title=Use of spectral vegetation indices to infer leaf area, evapotranspiration and yield: I.;citation_pages=623-629;citation_volume=82;citation_year=1990;citation_doi=10.2134/agronj1990.00021962008200030037x" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=C. L. Wiegand;citation_author=A. J. Richardson;citation_title=Use of Spectral Vegetation Indices to Infer Leaf Area, Evapotranspiration and Yield: II. Results;citation_pages=630-636;citation_volume=82;citation_year=1990;citation_issue=3;citation_doi=10.2134/agronj1990.00021962008200030038x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. K. Wong;citation_author=R. Bernardo;citation_title=Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations.;citation_pages=815-824;citation_volume=116;citation_year=2008;citation_issue=6;citation_pmid=18219476;citation_doi=10.1007/s00122-008-0715-5" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Canopy Temperature and Vegetation Indices from High-Throughput Phenotyping Improve Accuracy of Pedigree and Genomic Selection for Grain Yield in Wheat" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection can be applied prior to phenotyping, enabling shorter breeding cycles and greater rates of genetic gain relative to phenotypic selection. Traits measured using high-throughput phenotyping based on proximal or remote sensing could be useful for improving pedigree and genomic prediction model accuracies for traits not yet possible to phenotype directly. We tested if using aerial measurements of canopy temperature, and green and red normalized difference vegetation index as secondary traits in pedigree and genomic best linear unbiased prediction models could increase accuracy for grain yield in wheat, Triticum aestivum L., using 557 lines in five environments. Secondary traits on training and test sets, and grain yield on the training set were modeled as multivariate, and compared to univariate models with grain yield on the training set only. Cross validation accuracies were estimated within and across-environment, with and without replication, and with and without correcting for days to heading. We observed that, within environment, with unreplicated secondary trait data, and without correcting for days to heading, secondary traits increased accuracies for grain yield by 56% in pedigree, and 70% in genomic prediction models, on average. Secondary traits increased accuracy slightly more when replicated, and considerably less when models corrected for days to heading. In across-environment prediction, trends were similar but less consistent. These results show that secondary traits measured in high-throughput could be used in pedigree and genomic prediction to improve accuracy. This approach could improve selection in wheat during early stages if validated in early-generation breeding plots." />
<meta name="og-title" property="og:title" content="Canopy Temperature and Vegetation Indices from High-Throughput Phenotyping Improve Accuracy of Pedigree and Genomic Selection for Grain Yield in Wheat" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/9/2799" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection can be applied prior to phenotyping, enabling shorter breeding cycles and greater rates of genetic gain relative to phenotypic selection. Traits measured using high-throughput phenotyping based on proximal or remote sensing could be useful for improving pedigree and genomic prediction model accuracies for traits not yet possible to phenotype directly. We tested if using aerial measurements of canopy temperature, and green and red normalized difference vegetation index as secondary traits in pedigree and genomic best linear unbiased prediction models could increase accuracy for grain yield in wheat, Triticum aestivum L., using 557 lines in five environments. Secondary traits on training and test sets, and grain yield on the training set were modeled as multivariate, and compared to univariate models with grain yield on the training set only. Cross validation accuracies were estimated within and across-environment, with and without replication, and with and without correcting for days to heading. We observed that, within environment, with unreplicated secondary trait data, and without correcting for days to heading, secondary traits increased accuracies for grain yield by 56% in pedigree, and 70% in genomic prediction models, on average. Secondary traits increased accuracy slightly more when replicated, and considerably less when models corrected for days to heading. In across-environment prediction, trends were similar but less consistent. These results show that secondary traits measured in high-throughput could be used in pedigree and genomic prediction to improve accuracy. This approach could improve selection in wheat during early stages if validated in early-generation breeding plots." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
    <title>Canopy Temperature and Vegetation Indices from High-Throughput Phenotyping Improve Accuracy of Pedigree and Genomic Selection for Grain Yield in Wheat | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ToqeZI7u_kdOp1x0tthMWKum6hD-9S5DOFNOGR06ObY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460286","apath":"\/ggg\/6\/9\/2799.atom","pisa":"ggg;6\/9\/2799","ac":{"\/ggg\/6\/9\/2799.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/9\/2799.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2799"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2799"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/9\/2799.full":true,"\/content\/6\/9\/2799":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228gzPKRMAwCqvQFNd3MZOhLGIbnuhId9lOjEaTa9BOrUlJlH+eW+3J0eF56aFiCOOT83l\\\\\\\/w+dWFcoCTLeW7CubCiWzE\\\\\\\/D16nJ3DM9lGbRAsMLTZMd7BGzjZweWd1cc2D0BVdyHr8X+XtzB71hQ+TK3FMYO0ezJ8nVlcBN03SAd7OWqWwjOONUj0wjkA+eBWdouAbqB7xp0bz81CLS\\\\\\\/J0TGOswTke0qvkZ5y8cwAOLn4zGnc7qQ47N1IwJQvH18qgSMupFMcdXU0Nc+WPcZSWjz\\\\\\\/J94cRCUt0yIQPW33Q+bZ3BFZ92jOpTL+Dc1uOcukCr33Ebze0N\\\\\\\/fYfVVLtT5\\\\\\\/Gx6e1JsKJfN+tjiciljR3ZREtL5bH250F2Se02686Gqctu7UzcEwirXOBkrj4ecCq2qJhra2W3iSzlQ6DPc3gKMS5EfDFQDv1H\\\\\\\/kFTd5k\\\\\\\/RzmCN3GFX23kQKZd+YhGLHiFMfZRrlIlMJhzacl5YbQrr0cF2Kn+3DNWZG+VO038UBl6oH6h06S7MWs18WFvrtLqnDufFFfA93nInTsfjlTt0vWaewkRuEsSe30vhlBnQSrC31CaE198zHO643yIfPXODuUt8KPu4BEqg9fbWTIQltMWrt\\\\\\\/aa0IWbZsGjbPdFb88HHzndj2sDcDNZ\\\\\\\/wV2wcdQnUDZjEpqLPo10VI1MhrACRubmNwI+7l0FLuqhugHjxorkhjy0zrxxeNKqw7ifSXoANvgcqvYn3y6ZYoLjX\\\\\\\/s8mDeMdIOyHnfVcQhEfkjh7HvTwtXX8iq0L6Qnoj+qyj4bS0eFMLxZWg0bRCLv8F\\\\\\\/WOeV3qyPZ+7LplpZKMvmywO\\\\\\\/l0Zu8UDvPpf5etECZdM7Cb+\\\\\\\/VEyxQy1s\\\\\\\/d46oZDdEqAHrt1KSq\\\\\\\/eYgcKq7ZbpEIrqZYBlf6kVmvZJrjTIvVrIz9P9zHZ3udOOjQcQ+YGdCA9qBwnxygO7gVb7OLzVHJiEVj8JQRjtPnGLQyVgUqeXGs99B0uL5G3oUvynzH9vdWpGr8bdNpRmw\\\\\\\/1y2lFkcEu7LMdnACTZzBJaxz8ag13sYlCvybWqZM01BCZPPTVqmTjuc4CnebJXx0PdmGF\\\\\\\/XKtGbfMJyqezIKRNCruIYphBd4ZcOH57qrguhnZ5ylnX1yFTXuXRU2+OGsDewEoLMxO9RM0fdut2b0uJWVAIc9FX7d7WyeSwSYZzlVCWkLEh8NFyxjA+t2fYP25QaFmSyM9gAWoguCej9eveEFzI8V0yhm7pRdb2ESTwX6N86c9QlKZ4T3zvCbI7kIGcTguar\\\u0022,\\\u0022iv\\\u0022:\\\u0022qeGcV8EVGNavSqNAeMIhtw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221f1f7b4102ab6878f1e651d29d5d68ffc08b963b622b047525eefb92e623b2eb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222h+Y\\\\\\\/KsSNXJp\\\\\\\/hvWRu+Wc8dF+B0f\\\\\\\/Fe6gbj3BYX\\\\\\\/zIcr1ymBt4EeEWLqBSp0kqBk+9CTNDkjBRb2rTwRnnk0npB3K7Ra+hNYNqs\\\\\\\/bXsfOU8h4zyH62cwVi8528ui8HKJDtFjPLT8CvHPl9uRxsNBNgv\\\\\\\/OUF7BU218YzSSCacwlV1rzyhBgnzg8jLJWxbjgS\\\\\\\/oGUt9UOiBsuSdowTlswzlgQdBTsL\\\\\\\/VNt3ZwWrApg2fP7nBa0RGaOHz2KjYVSXauA3pamlAxPVjcW1LZgWlMLWN2\\\\\\\/K\\\\\\\/dILbd7t3M7EBR0vDzsdh+AAlTewKSCdaUZ542Uk\\\\\\\/ay30woAgRWJSIFCQzyTCUQp6UqbO7Vgi827n+dajk+4JgdEbx3G+aoOJTWwB\\\\\\\/HQ6hUVdhEdySFZnOHvyeZJOhsUXCmrVyo2v8vE4\\\\\\\/rk3bEOTO9H+nBmzQOlCoEUVF7ZWSp2fGVprEGlWmIXIXSlsi8N4bA2VQiPPwc97RRRC8qEgaRL53FD3kqcVmnfZa2Ag23KGarsCPD9ZDf5lJxP5qrbrCMhuAKZUU53xsUDV72EX8KNb5WKeu6QGVDXQcM1CzrON6nsEgxlpgq60wWsdQHk25S4n0wDV33Doz2rUPGCFmTpPwQhEOcZ+GyrdwPr7grPio8IoVReN2Ubsp5TWqjqr8jNKTTijVbQyOoibh2La+C\\\\\\\/g\\\\\\\/c9nrIM3ROkGRrX7f1\\\\\\\/yX66VCiti2y4mWwEoh05nsy0Lk+WeRjn3Ke0arpaE7F5QF7db67eCBb9xc7uypmB8amM+aYWrnvhrOkPc+AXoFgZ+XeJa\\\\\\\/BxbVPxxPzcI9fkOzIWqafjth+z4WwEkkrQntIIQDxRxLFO\\\\\\\/kzAD4U5Z73Hpv73j\\\\\\\/libk+8Ty2nmn1gABUGJw4UfxjlvrC1yUoxaJI\\\\\\\/XwqrPwFXUk0yfbjF9fcfWVsh2nuUD09wZ2+KKrN\\\\\\\/1oqhTDnqNK1g1XdsFpm8YJVxHjnMPm\\\\\\\/euj1V25vboHpNTW6kO+bLTLFdscfygAyQZ1CTsow7U2YdHz2O4xUENq7ZMJGkRjUC8iUZPRu1S+HcVpqrJikaE6JO0U2Ty3myDK6\\\\\\\/Lz84\\\\\\\/2J1PZki8A7XpVrLx6VCJrFqDfuCOw9OM2WENXfyte6U779GG\\\\\\\/hJt1hgtEOMdWuF96pkd3jKREdIimFaRj52bS6j4C3fFwsl2Ku8gc9vdqSTDSfvwBuNgakTcEGlSYXuW25svUUmEgDqg3f6gplbd5iXV1HtWvO51rmyBeaz6mWLePgSxLxZIZasqIbjAvK8bpNcrtoAzuu+X9tBm26WBl\\\\\\\/ApcwFJJuBJ5H\\\\\\\/2yziEy6rFLyiy6tYGuQJYtCyTIXZHUohmv1YDuT6w0SEcqHImdtlX0+uXY5woCG1HIdsSbmQ\\\\\\\/Ek\\\\\\\/f3Lnj9sOqVENnECnoslOzTQ24cycqO3cW67pNocmwIgJlsCPWsuRb2IyLbx4Q2yrbuyTUrKVNtTjqCLOcX8mSDrAJQz0PkUYYv93R2RNHEufZ4sHWFFIXOfp+GF5u0pvg8mAkzHQarFm9JlahzqJd\\\\\\\/N6MRyI75OYP24Jew1ZlkzlToMyOKesUv9Bic3fg6xzdnTqRFqAFzL02FY\\\\\\\/WS\\\\\\\/E4tV3D82dbSFrVoaev1Fe2Qfzzvjldjk8HP2vVChykyPrt1xs\\\\\\\/auXLOdHDPSlGIHZr3sWEp2qs3IYIw3iO6jivzioHM+ec6SJBT7a6CB\\\\\\\/zUkLGi+mlUntCvPyxS09latf4qFVBLPtK0G5Ur99l\\\\\\\/NBTF0BUZIHxUqVbcMRMKwToajLBYUfk+QnCgWFAU2d8aS8QhPtrkDf8CfFln4gyaB0IAtH3OmrFCEyHhJ37ef1o32tfYCKK2VPn8gE3\\\\\\\/0G9SBArZ05l9NjH0V8+4rKwS+\\\\\\\/szyfbVBdNifmMLTkheI36ZcOoJsG8nvUQHarCToF5WOeL5OfGDXPL7dRcnO01cs\\\\\\\/fEIpMqWxBnt5icR32bBqwbeNEEj\\\\\\\/S8yRshGAm82OgjX6GRUFvJQQHsjANut6C99zHBmth5fee8L5LmKZRHv0Vy8hxmQ2+VLcFY9kV05aVL2OHNX2D9wl27bRT07bOhPOoXYnNENHkLzFtXliTqHLGzvhA6ueqbygTJl+77LigfIsneHay8QybxM6vsjF+5oSZ3nozgucO2nReLhuaCIDWpxoHuVwbyz4OFdabBv2omr\\\\\\\/weuCiBTmtXDL+ES67E0UwbFzansK5\\\\\\\/eNIqGIvhDl7rQPW0BLl+pQ0CheNn\\\\\\\/u4js3jzGBpfrtXtW0pNPXkhmYNyc9sPNua8nlrScmlQbACrLl7q04ilhznGXLh1S91cxIee+eZRd40ikuXtZ8MfM\\\\\\\/q1wmF9NNE2GWUZDjbiW6OBGVDT3b+DMtzyJtJXj5ebGugVNdSY8A+\\\\\\\/UwarTtiM625oS2WH5zSP+oMDOEcyhccoOGGImTaDvnpSTFVZrLPSvqGuM9K\\\\\\\/ctxyEeXZUgOql5DQxJ5Xz+ALg6lkttgOuPz2XE0VmMH9I9sZRX1VHZ5wd3m+bWlG2OVAccj0pqJ\\\\\\\/W5mr\\\\\\\/BmqkHIj69yVLG8TIcGHqJBt00E2MPhmn1Iffk15Gs2TAf9AlDsCnWEQ3BdAr6RUCb8ftjh\\\\\\\/LWxqUfesax5z+wSx0sEJ0YM8UYFbNMuxzPOwuR\\\\\\\/Tw1Dq3xu8lXQfutx7paL+MAfGOoisEaDePap64bOSZUE92M5yZkDu9eQwaEIPs20zD0nPZuXSjZ2yQLk7Ls\\\\\\\/\\\\\\\/aEkTDxGzcv91MSu0gQsAwEF7Bd8CyhgDu9NRIZzhIH4GEz3P78i2UxQQQFqfytwxIlQtpvDeaXI3jPlup39Qfs5JDLitv28aBn0eWN8Wg49ZJgO6IM9W7tNyrL3XNW403ZIfGrspFZ3\\\\\\\/00e0peDC99FEkE2R8pSNnJ6pbpIq9ChLGndURRUZgsX8gFyaocxURp8hQE1UH\\\\\\\/kpA55wxWlacj1TC91ReobHadIxTIYz+rYwH0HgdU\\\\\\\/IumXr2VYCKNdzpaA7zchI13MQa6hreYIzggY9MttiL7oFGdNAVBbprHu3mvZCdvrFRUx2cAM88drwbcVb+Zk0SJTgbZjmqX9zG+m\\\\\\\/bRVh+GnK5rj3In\\\\\\\/IJs+Mpj0dUmWjFVKBCtkb4FzfPl4r7SR02hCeE6bZIth1K3oqg23IXJnubwErAKnjOlfaflTR9DU0cgh3hFbW\\\\\\\/WnxmF4iU1GhpIb6HoRyqDDrE\\\\\\\/nSGJWtBopVBUHAk53GQCq5UYH4Mqp1u0dofa95nCxs3yhjTgaCKHNDfufy9J07ME5LVoTAkI0TQh2d6u5oNpXpM6nKxYSM7xbDovoT4JJ+6tglNQfSDbURcFl5Mxgi4BYnOCq9fTBbRSgtXxNfaZWd1ArVp1y\\\\\\\/pCYTrwt8\\\\\\\/jK0vaI4hgoO+QVHBXiz66FIZG\\\\\\\/yxAtnrFx5g4RK7lnGkFAUBlbvbpO79EViVFn4dfAT1TuGh6EITb5J4Ib9n7ESfwa6gD7waRU5JFJyM1zCQWvpRGZYX3u23nYtAOG\\\\\\\/oAcXD4wCk9elj9QIApouccVRDgQuGhSHT1BEUchHzkx\\\\\\\/\\\\\\\/J5vgzK3IfPqkYBfaWXW+aw5Bp3oc3fUh1o3qcwcsA05C8fSHpqMCV8f80KFSV6L9SF1pBzXQElkdkkv4P3vbMpKADaw9mltEUGABuBkr6jJYBGexC5Izite0R1Pogk6YQ5bGPSda2TkCamPu\\\\\\\/1GNTFyi41sjuirMOlHXJ3y4TIOGRyp+bxgBv7r0bEoeiMgmoKt\\\\\\\/NUMtM2\\\\\\\/VJmdg4EV9dCjJ6jn8dqDwqNEh0udnSD1qKI10Radp5tr\\\\\\\/bqqRfbpyI2pbKiAlKTHXJkq1DKXKkur9Gxl9I6fxw7ChGkK7CM9Q5wnRl5O\\\\\\\/JyfSudVWDgfhBBZ2I\\\\\\\/6MbnVKw5TWik2gqMFkvFfx95VVFg9mPNfi93e74ih5dhh6aPc2O0CJfjnpy4Rv0NpEoS83eKlcy1B68WPKmrWAn2mZWIHQ1KWl6FWbUNbLm7q+Xx76EvMERMZNadETpVggvZsEICGCOvXT1E5u4I13q6iHEo5XB3H96jeTlt6d1BvHyI556QbTWzvDvD1qw9cjQFnL7ZnEh78c40ZhgF4NOwC+BOwrWedQZHlmOZ+uDWCfYZgOh+im745IKfBUlq3l5sR1bxNgos6WJnPHhiT4sZv1XfvmtvdrMJ4qAc4bRbm\\\\\\\/8XIu\\\\\\\/\\\\\\\/qY7YN+5TPaKUBEwsvtzaPWoiVYJjHY62hDEKQ+Cw6Q5vrIMbIcoceWce+3IM8Qb0rCMNIIjxvrwQqsUi0FMCiDFZm9y18BnNHaeNQBY3RreXIbrxye1mKt5tJnAK+UF3mL1+7LCQoN8fbHicjgszfi+49c0yUVrpC5uprzSLRfvD1k9tX25rK\\\\\\\/gULeBTqH1wypbpO4zbTMhZA0wxV1UZCbb+uCZCpWl7rw2JG2bfIaB7Sx+rv\\\\\\\/fmf92CWYaTcNXX+FW6Tl+cTQi84wwXBEcJZtBX00iQlU9TeLXNu5eFam9PZ6lzCysHeM31tsDJVwoIE66xYKOcE8OqPOStzIo89OV1kw8FYT+qs3UMHngWnlKaji7yALmfIF2lOh5N15ZEBrbqd2bst1a85CM4MMPmqL73w5k9OSSoFUhrbGYzIkdFdRzl4wd5\\\\\\\/AAZXqJCED+dXTqzg8EcWBC+YzqWz937vvzGV+PH\\\\\\\/Nx9gdlN\\\\\\\/3zb81O8jyy5Uh\\\\\\\/G+5wmodJfbpv8EkX7T8gVyHFXYxIpKm80wLyuqX0+wsvcWEjRDHoZn2z1AyfeLoMrWSTwXnW0FAqz1pCq3bcRSh9BvSxoi4pydvuWxyiVtib\\\\\\\/h+9F\\\\\\\/l0zcs9eY\\\\\\\/PQ7anVmJ3LlKkGEuuzEDk4YpMcNi1LMxCVSdwM\\\\\\\/H5CFbNw48zpkSEWHQJ1ns+PEI7xPISqx0Uoh63zmv6Gpu5Ic4OrbqMFumMLldwUG0XL8aYbL7tAIyQ9lEhClt+AhJBY\\\\\\\/wlVifn9OOthN\\\\\\\/X5UMWM4brtZGNoVRDebEzU5UdRcriE4B+zNa\\\\\\\/H7prbsBs3Mn8qoDjcAzbdd+WKx86lZNVv0q5eFl09qxxN\\\\\\\/y433iccOQ+tckj+SancwynPFDWj+MaOStHgpbPFzPberXBgFRAQJMxrb\\\\\\\/\\\\\\\/AdyW9uxWPubEwwFnuF6j\\\\\\\/rYRJlmU99VC24bCOgXO3naJ1H1xgcyEuus26t6REmEIjpt7mgYwwJbzXm7GrUhoAma6wnhUlBg9ZJnm3TLs1fVQu\\\\\\\/s6YLeawgRX+\\\\\\\/NWbBHM6bkWtSLpXd0lmRFqQ5i25n5WPvMgOyko5Ni1CtfGlaHCGfLpwKbJW+cOQsaHifuhW0a32mHneLRPSmhHqTWy70ds4tE2hSbcaHt7obazZmWd64jlDlRyz7RIct60MvDH\\\\\\\/uf3+fPCtqKNbHwyb6ni6\\\\\\\/iuR8KPrh8Od6IQiB68+dDES18qNGT5r8KI8o1vt7Di1k8YBGsV65Ge3v5ikpcEklQKjNcUTnJwbT3UVAZG3NRbM+Zb0zkpCymZKYgqDlQXRB47WxlcInCsvuF2OMqvY5W0dc59iPP\\\\\\\/vBQa4iGC3HVuT+o9PXeVTHSB+WQxfAO3qYJhfPlJJaKvIA8En3ys6er7vj1lhEGY5w4cpcRJ94QIwMHg6HBCWm7k1u+k\\\\\\\/Q8i7aEz7bNJ4kTqAniFQPFUDmVX0UabOQ5uE7sM8RcYRa9qhQisCl\\\\\\\/gMh2sbS00kTZHesNpHv9h\\\\\\\/zfzFAFveow1a\\\\\\\/GhGvR8rlu9FhkrdGA1Lu3ZC8TFLdTwlZmev8k3in42\\\\\\\/tSH\\\\\\\/GAw+Lk84NOGLU+vdd9IIGdedVGFWeSTsG5BAm491l\\\\\\\/MxctwxFRmhpjzF+UpCKbIbN3n5BmBQaf4leTaxU\\\\\\\/cnaDkZh\\\\\\\/56e\\\\\\\/4Ky5XoOq3FhOB\\\\\\\/ET5IzE6HGoELGV4MEfPKYn6g9qRifdUlOc0RDxPr2Vh8JG39QTHf0pinSEIWoEW1eqz26I1kbCxDwt4QiLQyQV\\\\\\\/muY962tFjTva0cyoNEoIPGBip7guDA6Nfvi2zaxFyKo2F702T1EHHqv\\\\\\\/P4aTv2\\\\\\\/y31gUYQFh9yt7e13vn5HeQ8gK\\\\\\\/fTwjrDbK5VOzYRiRseuu5UTOtbRXzINi86DrIpoBsJ7aMLhxzpWLq+ROWL+cOPAbneNLL9E590Ty+KmK0TFJwUXUw0ULiIOlsxkEuBNstwDjDBbt3x5MfNAb4VD3K5SDJuUx9DvuwDSCiscGDozVMHaax9mRcjG\\\\\\\/d239qgC1y+z7YMY6Fy3WrDW\\\\\\\/YqQ5uQnUmzYwL7IR0+skKvLIcVBPleu4JsHpZkpRP7iajEu\\\\\\\/SmpA5noDLLnU9UGZCTsDPoGsVVRe+9ciEMk2XfpewwJo9Ixx7GNncxO3lEfu6vAgaZgfNuPkVorj1W7ODppbs9hkfqKNoi9PeNBGsVWoEIoQ4Vqt7+p0WSxHk20BJxuYPmGZpJ+MJj3yrz5sSA2RHlVY+67rGGbGkH+jF+OTJee\\\\\\\/xlA1mmiNEIc5oDapAa5IjwlRPrxJJavWDpJmqvULdXmQDwk5YR7JhLuGjhjndq8DFnoia\\\\\\\/tKKpJjNMbWqzgMWoM5DhKJaDfM2LZ7ZEYSA53kUZhhhfYsi+nZ8taRYjqL25lHdg+Qjn9sTus9OzKgf19Yo5FlOQcBo3G+uaRDXf7yFQ9NUFQGMYQtO2HDe0pUiEalu5DzMIZ+18DF9hH0QoZmfdB51LkOd7rWBhYoQ8IfS5IICDH6O9g0UUg02LZhfxgbjPHXd49ggQixO4njhu+LaF26laNyyBP5c3KrD2NzWdCOGaEwANDntwyPdMcidYuEGpfDjSS+9S8Jq5cmcLrdsnlexY+gPRqouIS6x2LWrXLhvesT+iF3blxMBTkVNE1wepb+r6ikPf1XMcuWAC1+P4aM++09f2GxlS1QClOuHaTnqZ6bOgmoXXdXo1OsGPgjYKEnAlSo6cYkOytoLmnw1\\\\\\\/1fLDOLQ5PlSSS5L1ak9lu\\\\\\\/T2GJ+S0SbNnAILyS1uw7+n4gp0e6KCCRcA0gVWPSCrROW04mqVIZzXoJPwM30TVkQMRJrsW512Shm47Wg30SddBsluz6IM\\\\\\\/l7\\\\\\\/THHL6EUg2Ax3Yu\\\\\\\/dhOeOnG6aZuJlgL57vjgNeIPLc0sIWDsBAIf7bJPlkqWLWMuonzbN6iHK3JszNwFuByshQUNmm85J3iT3Q8Im7unig0FxxQyo5lyR2VF1S3H\\\\\\\/469ZKgOkid87TmQihZNOraQw5\\\\\\\/U57gyM4tdsIY091cew9IfK9e2GJKnwTiEsPZf3dUA6X1\\\\\\\/UQWDwc+7sioNclIDudktEyVZp+vTs8qREwaYuxZGx2tc2OW4XZP6CzNDyOs3IFgHO6w5CnYAFT3iVcJUXHY9po\\\\\\\/MvCGVyIBLoNQDJXVq0+hbuVKSGHQXefMRlt1iRLyO\\\\\\\/TJcYwJ5QrUxjR3CLTz4snZls+7Ifo9RNcPHiGMIiNsfcIJ4eUDTRWON5UZ5TIz53Zqx1FF3z9p7Wd0zrffuTq02Nq+CVJrV2x3ynORFCzy0stZY7M4dfFEgA33Q\\\\\\\/7SWmKdG+96Dr3fYl8aEfQZqKvaQf1YGz320JuPhSGAiFDNykEZwaCBavz7quntJvlXWKiKXNRVpgdcUB01QylcOCcK2kcOLgT4z6JUi7jP8lvOKMeZnswjLBfUJOZPnQJGhQMuMu2ZZdZsjM23mUAv2iSSdcFcQLBl8uzsuGsluFi41+3l39wSKnFsZCpVhBrBsgou4ZTkv9b84XS2aj8rwjeKD\\\\\\\/46DfgRww1LkvKoaHIRdNhLeSbr50BwFaTpszhvb4dS89qoucfaRwDUxLelf8adDJHBEGb0GTou3xODE4PU7ja4p2eU1csZVRHKdq6CJEGwdwnoUd8q+NpDuWjEbzvt2\\\\\\\/uTPq2oPYzKLFUmq7wk0gIXfnSZqE9z4M+t4ZJS8k+DKNYw5YUVsnlTt+59Woj1gatwFjohVG05g1KC7QWiNV5XOanzuunwqZWs9O0v9c+4FIx0CccfO3HZ5m7arlRugHq8uZvKzaJNOX1WUk4ITBWE3yI\\\\\\\/OjmPx8NpIXDDzCdUwXe7nB472Qa1vyBs4d3q2xBntORwm5U2ouoJxBTsc3H6bgxXXXJmvsscTIOqEqe6kkq1eqVkxd\\\\\\\/94dnjm+zMmhTU5ev0pkOWCijYjIrwLfN1OVxCRHaN7dHJ5dqBOGyqptAaK676Ejgrd463DRmUDTtiE1RDhLosO\\\\\\\/znLTl4EiY7Y9M\\\\\\\/kg\\\\\\\/UuqdZfTEW2nJEQI38SVlmkeneGwCc2uJmPCWScRvS+JQwM8WTgzHUPjM0su0mTNZn+Q6QcTzNhR0lX0z8DaMiz5FicSlBu1yE3k2EP2P43jvw2EYxqpEsGJ0vKjsZcBTAKozm4gdRpmZQxilkdH+TqIBUnJEAYGZkFObEbc7VqTCR21D4iLV4Ow6tGSOIkjxJTNmlUWloTpFItONWGiprEalqpoQC1wQsiGABqj5fe5KHS8bp1iAoO9qKZ35qmpOSvSRiuDwpMUIttaKB9MZrZ00LYJpvJbPGaK26NYBvvbgmycda4xuXAvblf421+tXDNq2SwyLgY826TDtbM+DWwgpFhftHqcpGF4s9oRbMOn+i0fzkfwsKrKWligeTNEEvnrw2RALiAljWGmuGk6Rc2XYfXLki7VZbJd8R9c\\\\\\\/ei1IMm1sz1Um8hc+Ze0pKr8uHYoYHD6jrPGiQpdpKxACQbBTBBHgfeb53qPNzdV6RCZdUrF0mQabbHBX0ZQzhSsce4wtpdBVVkXFsyrZ8YYP9Eo1rgHeHa3Sxs4x4SwRuTf1Hn478U3Mg0MIh5UNR7Ttq2nZhwuD8DDcAcPlvRbz8N\\\\\\\/CnKf1m7+hX\\\\\\\/IoPr8JIf8x9yRdBsg5DU8V70p31H7qQd7Y1szpCQFj8RmC4hurJm1tg9DdmLvQ2EuFZlkjs+FRTgLVcL9DHzVmsp3xnqi5038wUI9wDKrf6XApSj8r81sUWD3szd7ahaVIfTbanQfCr5pzRVCQRwk7kyLDoKIlqFOT0bZWSeCg50cvIaQFYYnWbE4a0qfvSoNt2AQcNabxlhPiwjur1pt2et\\\\\\\/PaJ76l49C3fP5hGO45gog9W0dt9dYDPX2k3wzPLAJxnFXBQKxDWI9AtPjM2g5Yo6ffVvIHN41VnbnzEsLmg9AyWg3Rv\\\\\\\/hcwe3bUjHG5p1gMZMZJarc4SztdGybHKnF9Cpc\\\\\\\/nl3Y9g0L6HKg2O7w2uzP2jIBXm7xrk2hgxl3We6osl+VA606CP85qi3yyY9YCv0sfx9Mlx8TPy7\\\\\\\/8nV7RbBlpoz5tlXSzXerlUBSFsGIRs7bO+45lYbBEPBdWuordLJLplXqGHsKg\\\\\\\/7W8D8lIe2BB5HTLJCW7IfryVt88SEzEbPbOg5zIM7oaiWPrudws9Qq8i3lHpWnuCnef9v1760+pvD01WGKBOrXT3CbGfOSaQ6fXfpZkcO5rtzrGAM9J8KufT0EldSLLLKvfR3dYTZ6ijyXQYkaeqxUn3Ox8umH2z\\\\\\\/sIZiQkMEqVb57ewYXxBK6l2uI8Zb10O0428p4n3a18ouzBDTNo1kwnGXQnRDJudzXDTeGuTIN1jZzTYYHF3zzA8GBH\\\\\\\/1+pa\\\\\\\/ExrcwgEd6ZFNnP8Yb1f0trE7yDQdHBkeWqt4pu\\\\\\\/O2HaeHW4YvcY22SRlu2aR3v7OxYqTvCMZNBxbcP64hXduO8HWV6TSnrOut\\\\\\\/4L0ELOb2tQEmApfibkdfxlohA4Y\\\\\\\/P0oMFBIlpSInSMJmOBCQvOCRZc1wR9tFVWvC+PL8cu0LjDSXlBF39FjxY6HSZse9lCo2kTkx21fxOgGtcwc+Sl7\\\\\\\/YfkW+IyscSI9E8i3Q+2XfkkreDCGL+KizpH4DCTkSmbS136aE2xzM9Px6GMUE1dzNbZM38qn1DHrKkCaxYwflVV7aIIpRRUSBWxt10o+wXaPJrIL74KmgcuWvvNlLtsVHUR0EYQUofTYHbDq3d9yFtSSxm4EWav7WtWzQIOL46wxaURQWeArxnvR6sQ0qOzjfa4UM9Dd8MYtLjVCGZRBT+\\\\\\\/TTQNl8JToE2nTY4TvjfqfUzYnCfn8O2egtq\\\\\\\/kRqQ5P3JrQkDfdYYZvZSURQSGZ2kiUvplg0Dw5Syt601\\\\\\\/nL2T+2M1G64rRzN+jUeGJa2lYIJz9vBJptVFQ3PrSH2pts16ZgpHXptn986ub29ZBH1m4keHSjZpido2muQPdJ7YGsI+udzWJOWz363Wn4p1+siRz+pyMZYWAnCa8uYos47V1Rw08Jz\\\\\\\/fQ+HH8FlRoEdvzuaG\\\\\\\/rR5QVILJoWd07Z3+3jdATkIrdSAxXl1MkW66B62ae\\\\\\\/0v7wWzPpq+Zb+WNCdGbEwquLryDA7GhpA8xJJrLYgxUsvj0I08IhSLIJPLQQ7xEnLeSy4C09AtAfisGaw4QH8J+VaNfX29XOPnXlYumaX2sDdgI+27P4jOQbD4mPQes1EtKKU6usmPKzGoGL5\\\\\\\/+4XPdN6B0dXrAFvbaIpMwjrsVSq\\\\\\\/cgTVqqKm33PtyZiw7m2PnY2p\\\\\\\/nylRzLdpljIDRdcSZ\\\\\\\/S0LXxsaAc4UsmDSnToEpXr8Bd0Dl6m36UOVlzZL10+Z5HB84YKOs2H1irwh9c8Bj3j0qwnxcy0mjBYRW9GHxpF9NuzAFKXT4v4f7hvW\\\\\\\/qEuYiSs+lLlQhI4JoiMY+2rfbtN1xzfcJ7bKqyp9xDwQ12Bvlhf2RlE3CLBJhRWNTBApxNEnnvSJgpCeDNh9HuQoLaNoQ8sP1wXTgu348\\\\\\\/I6cp5PBpy4C1EWIdczFdwpciFo8dBrlJOtkBUF9iDV78mLIa8CCA1QgSGLxS8bOO2+ev3IOVhvjMaHkaMRhuhcQukkIDhO7ic\\\\\\\/GhmqomL7ma0o2SjoyLZ3552gKzA\\\\\\\/zpzcN8B8hdivYI7wppmQOAoWCb0ySxu3Zf3IqkYfGprHsB8Wi5+RBdAZBdxyL5LDDCmiT5hv9BboS\\\\\\\/k\\\\\\\/CcfS3LAMSVda678CwpslDIJh\\\\\\\/WVL1WtSYHcGvZuVrwiuTB9Pz5BhaRbM\\\\\\\/Z+N0LI5Ju9JnAdonhmvXZ8SVXKN4kQyK3JW0aeOd7A\\\\\\\/+fSZ+qP\\\\\\\/oAaeAyD2CT24eWwp\\\\\\\/WsP8UQmDh\\\\\\\/xX\\\\\\\/Uf35omPQL+Jatj9\\\\\\\/8ONlcF75wlyv2NSE4Dii2awnu\\\\\\\/eGFEaKWjYCWVXO1jai4gIVy0zulGk478e5+Y\\\\\\\/8nrHO\\\\\\\/WS2BTjWGxCAFR6wgbYiobdAVYfWzMF45dVjaV4NJHsxF3abpVxfVCTrvG9BaQn0KnTkCkZRYr94xBwr3jmeIaM\\\\\\\/mg\\\\\\\/P4KYCbDA4UoZiT72uuIkohOYpbIahwJYC7IJU9LAJlDP8RUYz6u3lC\\\\\\\/JJ9nN0ytsLJ4vSCnWqp2GHvpVejwkmcasg+dicHIxxP8iEAGQOXfNqPyFydV6Tjbm5TB6YwGolmSz9G4n2mDyfFJxnBVtHlPjscfL0Pkc3X22jQ6Uk6hE6VkSAjIZkd9ct\\\\\\\/KEOInZgHB2NMMOn56i3mwn+iofFJyeC9QAyViDmKQZPdriwy47NDVvAcXi9Oqp42g0PWA0k0JE\\\\\\\/JWAOdusCQUdGSI7yw+rjwCKI2j\\\\\\\/SyuIih50RBGuRip1P2b9KBc5ULjjHxjFygxoHvJNtCEW6rE5rhOvD3T2rtuxdJ0\\\\\\\/xC5DCNNGfJM3cojf8nRZS8MaGCQaImIdF2e0GXS4Oa\\\\\\\/BYQpnzOJhlejpr5SkiEtrEsz5363HZnSzVR8rTIrXiPduOzHU\\\\\\\/lECxS1G8XBzn20Glp7bWTqIBzNhqBi8uAFiEslaitAgmmW3PvKR8JO0LYouo0wecl0atQCFZMAi3zjWHqaXYF8EjG4hPDKwq+Wm0gDleiZYu0G\\\\\\\/1I9KqL\\\\\\\/To8DwIy267mDcWC8blOq0pZK3HWCDhaPqeRvlkqaHvCXUAPoIZjXSSaG\\\\\\\/Cq9gSzTtJgmB61LXwPXwJD0quoOQqnZ+1TUNRd+GXuLco1daBGGPkJkct2Iv3lkb2lxQoSeey3DwsVyz8vZ\\\\\\\/NagXUIomyREgO2t5hdQir10RJn1GPayNiM6yGU\\\\\\\/dNEu2fcLp8owuZm+QlRg7R+wKyMXpxdfZ3JoooDvt1bpQPbFoysCcGKPNkSvMSbPPvQ1TRIETTS96+dKBBCk6qsLeRSri6\\\\\\\/FC\\\\\\\/k6jQ25zCof2OsWUgkL\\\\\\\/e0S8LMNiGHbtaWLC4ZqLv7EEB3ZWGm7TMuxrFtiXdCwhpJVr2MjNOddh4Uu6qEcEloXC320CdyCoC9Py3DBE\\\\\\\/Ez3jjgVaA00vUpaTrXOWS8I\\\\\\\/dPplZaBc6lWo74kFTZVu3RNwj97csUrqZLnVZ\\\\\\\/uN02mjGidj80i8WmRLfwzSkX9WwX1U8FDaEnmZZB6ARuO1VC2zfXJ9ENbXICVMBMtYwKbgq5cI5q+bXU5AHtrSQyUgct7PuZse6Qwwg2ZGSKPKbpW8jUjFmiySQ69wLNrRIsZcMNrcLi6Jfus3HMbr1ucYxYxBF88ODbh0gD8LozfYuERtXA7h18sebtbenrjRiWPxpVz90aAYjuYhFdp9jy\\\\\\\/DR8M5aDforW6j2c4twlgXi2DcnfXNMwXPJn7piL0dOIDDiJ+k5edtnkcDm6dAykxbdzdVwK2iqIqLln1IjBXrn1t+AeEHOIEj1GS80ubycvLtNCeAp58xt8DZR2vhK+xFKuQzwKdT6lICTtwCaF2vWZQo9tTGCcFLDS7LtSaUcRSO5ZMFRnL2bfwDqK\\\\\\\/cKtRCfRJoHSctI+L+pSwPGikwP5ljwd6GK0gayCLZTNB6+jf8gfBXMUphjjnYCi1i2UKVU1UDMBoYqby1lOZOhvgWbWlP5Y38BcnRpTgBGtqBo+0oy7RE6xSdOkGwxsS0iFFItjbQLQ+jEQbxFJEYEEvvexEqvJzuquoJzbR6c30iNL8Qg+N2TiRnSXzEZHdOx7+zHQEVWQATWtl9W6fMYlCwgygczDTCZJ2sAOH8W4EOAlR2OU\\\\\\\/QqWFr4BdBPOkvoWZI1DuF+eOuVYXnuFxVLFm6jCSYQo28KzTgC9K+bW6BBOWDMIgPxrKXDEpDsEt7izJaHpkBTKja4MLr+bdIiP4xgByphxkcC3eJ7Tffc4Kc3cdsnigEIx3TjcTtbDoSd7RUTPnJXfRW2XoEeMsM6P\\\\\\\/U7XGeL8fTlR6mDyTTbjJESMLwqKnneRs7ka5exA0CgN5dKmf+I2FDOX6kAgQY08XRdqyC3aS6dMtRam5b6o+6m1bIAM9vz8taaVPyHdp\\\\\\\/mYV2qFn2AGzAO9Ph2UGDWgZUcq7YM9RSTfXQktxRE0wupMBuQh2DamEG\\\\\\\/kVBxIcq2CMfT3xF3i1yfAJeT1jNuR76xi07ZEfEWeijVweCtmQZQch4knxNbO8TTQPCBWfjR+KmFqnPxfFFPZoGo5VpXtz+PIRlLzgCvPv0uA7\\\\\\\/iWGcG5gi13NEGoTzg3gUZv+F\\\\\\\/p2VTo5X\\\\\\\/tJOy7KRlSTjVjLADRiB1NrGYpZsxn8RSNfI0w81iFZ0exdiAqOex8pEpnyC8luf6OORbnbWntCmMtBsVoFqytd+Q6EznDOEwPRDcjn0QOuKSjV\\\\\\\/x9a5UhiidB6JQLkMiHz9p2mq4nrO0u2XklRq6HLmfU7lo54s11L2ANmvDjFZp47g0RvW43pYbJr4vEVomRH4JnAQ\\\\\\\/MDE0rSv1p2LeRjpB8sUkGRYSg73+dZFW5kRRi3f5bZQ5akEbzO\\\\\\\/B9BdMvthhVoTZW32Z\\\\\\\/45WkcirUxNvrwmdOBARYZVIpslo8eLhp\\\\\\\/rOjt6fhUBD5M7QVzDL4e45Bh6L5ubNOwUSIiLrlT1hJH6uQ8UNnLov4IfIXaXuRRoKjdiYOsGX4\\\\\\\/Ja8LCYSHJawR2agBcvZyal8jBdVZANdZ4GFwb8mZEZNtKzK4aCa6CDvqT\\\\\\\/Q9jyY6flVzGgz+O3DXSfmxq2GV6qzK3b0jO4lE+zBKT3BuhumVnCs4wVgdbngYYvy\\\\\\\/OLtwRsLEIUCoY1twrzPj0RtdkGG6DNVsxEd2\\\\\\\/o2mj\\\\\\\/cQiicRITqTfRiQH55VbdlrvW6Q8qihU51zbVUyzQO5LGFd\\\\\\\/SCTYnXRLyT0ObNAeqVy6Qr+SQqG86UW4Z32CShoYGKjVD\\\\\\\/eCIAHkTYfRTuMf3w0RVnZ6liFhRby0GT5E7zl4Oa9apkAVInlrY+bwg\\\\\\\/MeXUSTCwetF0tWuaKcMVSqThHoi2OM41GyC2N3TgWR9VoLpMGpRhMTu8vqyx79kWZI1IiLvWhsb4nbmvvkUnIt2LVJUMWrXObyFPl4CwwmB7uKHLMp9I1eCJ9kx6Pwaqk7CPoeVU\\\\\\\/rRVi+CxUPvDIUlRaeawU1uDgNA5z7kACavy7TI74mDDR4MeuhhYjCjaFLj2gQV2KrmHk0KUNj8KvoBn8qsWJV5wu1BpjUVexXJ9qc6sM+nqNSGHs5tTkM+rSPrWYzbkNkD8XJ7DeT2IiinEFmnkS4YB0RD+J8J11RHOkkgQqMtPak0QiOkD3U548sunsDo5z7ldiHt6iUlZYPr861N\\\\\\\/BIXbSoom80fRkrIIFXstaJztIw5TcgfliZPi05Zy3oWnZwSAnS95RUdk4xaoWb9xF2\\\\\\\/y50CAgXNmBSIdR02gFfZ6TioQGAoS9RxQYK7L6E2C+Nt4ltPcId7D+hNubOD0LkaX9sUEeXy1TolS+bBouP3IImxeRLmXOwBfe1+750ns4i2gKvoE9ID4FHRxM54CYY6ce0Zt8vtorYFtV8nXuDNeWRcwu6xFcXkoJ+3bXvNx8UwyXJCuC9nESMO2Codz\\\\\\\/NNPmYFJfRHxDIEHEEgCkZnN6CqzPGhwg3wxozWHH7+Jl3qg4c4\\\\\\\/l4b4qsVVVJVd8pEnv3w7o+wE58xJCQV6tLo1qWU3sX24cRYa4DSOhSDAymcXEqE6BnISgsfFVML4PUNTBI5Vxbr3nmHWQmnEyoPu6X+bSNUMD8Lmp8fHmvHs5n3FDkgoc\\\\\\\/YBK4ky+CkMdrcmxVCLEVRk+KxnOoDCm\\\\\\\/GWNK50jwdkgiiXV7S0q9nVaFHNuZ0\\\\\\\/9XwEx+bI+\\\\\\\/hFrn8tfPdmcBPDRaEycvn6taHBoHB5EGIuP5Jpx+7ANWbapaSoZufLfr5LWjmvzD4z3xwrayf6HePOVMrdWwMEsfhQVo7p4Dmt5\\\\\\\/W0OEulAgLptGioJ8qEt6BmxOmLy9K70lwuMiMtuZfngoi5O7CFqRuooHYHHVVZFkds3N1sH3fx\\\\\\\/HOKjvI7Hn\\\\\\\/OTQjVbpOstm9MI9Bv1HoTl\\\\\\\/eon4VELVSft3sIzaHR6Nje\\\\\\\/xLSWZ5Wj6Z2doX\\\\\\\/8o7wH4dQ3QtZZhOshqfc3PmSOGWEdU0HKFp4cmmDlVmW5xU5XFnf+mtGr6coDMlQooqB0KuhuUHKmEbuFjjb4b+T07Q1RCF5EMlV5r0+VgLydz8XZx3IfAEgRpH\\\\\\\/AFfsdchLUIlTXF3e\\\\\\\/O6khyEpJ7rM\\\\\\\/19wMcXR0+MQQqbq2f7tA6K7LDq4am+N7PiZxcGgbBK+j54kVT7sBDqydzuONIDw5VVgkySuQgW6nl8ljtR+KAkFlx6u1Fqaene8+S6zEYauLMz6U4ycz8pQglaO+c8z9xm68rUbOPQwg85Bo8Ryn0eqlNl4QDsm\\\\\\\/ljDSUffVrfnSKp9t6uMafPZbPZyw8glCiOJ\\\\\\\/CtXAwX8fJGVyhD8wLE4Wa9pPorkYjA\\\\\\\/bvH\\\\\\\/8RhTDNixstySzBtXvnWxjdvjZ06EptSia5mPDLY6mMHZg3M51M4za+\\\\\\\/dePVFO7OqWIMuV+2g3CdyMHY1vCllkQx7P1b\\\\\\\/oQ9VOWqpKfvtgTGH4eGq1OyoE+Vcm+EVOJLNYjacTDsyyfgdlR3MUed7iAjfaH7ZpRqpYlDfDTnpORLmHExkiact\\\\\\\/ye0n0oMYTCKAkyEfhJUHqTZw0TmgVvRJazbcuio38kqjPwzNkWdQCf2s2rzCut5eITecrToRNIRdjt27LupvdLGF1CTNfLQHr1NGUkGnV5SlexQtQ48luG\\\\\\\/5aNMOCWPuj3juEeIN896MuNXmvJDeMGn5mYtdWGprmtImcgyCgK8E4\\\\\\\/6HKYtnTO3gOSQfRPOQ9mwml3twwAEaf3WA143Ss4l6Lq93g2z+Fx2iCyvSEj97FpvjUQ6r0LeeHriLv8LBLcDGxorj2sNlSPifKM1vVBuc94Cd44XncyWF\\\\\\\/ZSrIiin5kAdV6W+tAI1kZ\\\\\\\/kk8hoEG0ufn7hR+5axpFsSyDHzFHkJWwEFHCdGe2W3INogNjDIQrKHVRfc5AfIyinrBvX66TJUlfA4PdNsF0m2Zmn9Eefx\\\\\\\/xlXq2nLeceqMAmZZWLkf7om22c5WuGMGlBFKkdK+X4HfSNmP4TudEw0mS2AmqbBJH5cB0u5++Tyz4p6WIOWFkllaVt8UDrBJEkqEXEmqBzDY1WzDB5v8Y\\\\\\\/PtaNdW7noD3Mdc+mhT+iB0GEKqcjKxpy39ob\\\\\\\/lao7DejY6iRx4IOAnjEEVmpKtTPUN841FjXnOiWvCjWsBPImb32jTANFXSWC2DOBe+vTeh8jJ+FyPQE1apqfQ5l8ixL2HiQrOxAVik0c3mnpIi4RFhmTRpEhuUSmUTEaRIGdfPohgc8SPx2bjRh2Hhe7REEsJH\\\\\\\/9GG3jILMtWiH9gyVEvt8TLXny3rN5qH1hEzP5fiLjxikifnHT+ykkCMXbOF2km2buZabbP6M+BVLvazs3giIvFvZQmTj9stUl5wJc8JSla9MJmP00gcv\\\\\\\/gq+dmjhujsusABDAMnvHw2z1z40fv0O1U8R8kzRf+qF2TATs+zUfqhlnW0Q0nJfQ2IwPOTTGQCSBxXa8FqCGtFrRSE1znkKqdBL6nHBs1GK5461f5G4sqsQ3RvwYz6oVV76unM2ldvu\\\\\\\/h014bsZEn4DNVFYlMOPQd9kOTcwK9opFgftbALy979q5YxMxK3hM2vUdJSoz+ZBg7I96Bknsm00yEAjvERm95Rw2oQFK+0dkOEVbChE1Prk+Um2NBZ76gAB70\\\\\\\/usn+EFmhchKPtck8whGo1WhO7xqH\\\\\\\/DYqS\\\\\\\/5s7UWwjhresml\\\\\\\/C55ylQ52w1CrBC79C5\\\\\\\/oYmsFJ0J\\\\\\\/GC4s+z8q367LG+iKlBoBmV4kK\\\\\\\/avD+e3vCCiEw\\\\\\\/Q7y9LM\\\\\\\/2q8x6wYRyM5pdfbmUYqLFVlNK7QbEWt3Ua1SftQ8MriHAAH1UB48YXvl6e2Euz8D2V1udYLxxv2YXVxlxKrY3VZuA4FRbfDnZOx4EkobBbQLB4V9ijYpmNbGUWkiRJMLiN2ZbgaAnY9XsmPfAKA0xR+Id294lNFc8V9hutM+Vci8FUURO3wvrKvOOl4NScs3cUi7BrW1rVCULsqlAiFtC9fy7apdKga8vN6QZHwRxf7vlSgVMhS\\\\\\\/d9v9vVJsbQtGSEnfOlczWe5Uwku2iEbGbLT3CeguFnXfqhNs7bXDmM6nBinEL0V4do7hSmcNRwnkfjcyQRN3Szt8zsHLEm1iOJHBxIhkykNROd69961OHXqcBhenqOhx1NLDmRzqQrUN9SaVFlTrbh7WMKUjww5zS5v2mGk1rqpkR9JrXZnQwSaL7GT\\\\\\\/lHW4um5OsL4X592IcHCCaO69WKJ5zBrOgeRDYmCrT+9Xh9yo4Ll1bA5cmF7ueg\\\\\\\/+emeYQeQ6e65tQBbtQK3AyXYOy+glPafQ+T0ZGAtWwbu7xgWXp2Vxc\\\\\\\/RMcwmJq8BqSzuf6XDFVfsZZaJ1gGWejut4geF2WyQd8f5VURJNGbpl5Y1s1eaRj36vC43BrDP1Z1\\\\\\\/aeIEBjTjFieqZcyG8x+NmAQcdF8kpPf0xWqhHCRA29R4G\\\\\\\/27GNIfn1Hl4bIZB3PqWmYNl3ydJSebOa+o3g7MNIHEo2uosoZiXfGZRFdgQW04qRdbKVjXxhuujy6VQh1hOI7VR9FO0vUcAoT1PfNiRMIQw1b5z0PzYNt9hKOjCCnILBntkg9HLqCLdW9qHH30C\\\\\\\/vVeOQ2JUlsv3mbWZB0nYiesv0m2At7gTFK7IyVU8YQMJnjaEY4rYZtUgMIobbsCnelVouEgmGEu22jhsoDKbGjG2Os7+yI3++0wuk1apEfNDJ1QDUmEI\\\\\\\/jiijVOgmN0lePAy3g6BohY+9B2t+WndnKSQA6q04Xv+uvmPfHaaTNpLxW4XRPyQZCC3T6eIcPORbJ08O12z7xnBfrgGpoG1pvmOZruQpVeFcwsqruLaduXyV5PY8psqpfh8w76fWSqcrMYlh5JbvoiYAClrEhlXAZSOjYDoNoWWS1TRzswTQpE6puWgYLpwhzyxYPlv8Y\\\\\\\/+KNCUrew+5fF9ohPcRwUEvoDnNVF4dosQwngOOTzi+HaGJB9zQX\\\\\\\/XKivKztSC3Tb+U7vdlhr7Kci+W0YOOsovBA6kgFIx87wrCa5XLe7rcWFmihDu44xldhQpP6zSoC+6CMekzAAJICMNBq4Ub5I3wpgkdHU2St+8s387XXhXb\\\\\\\/DMGe3Q\\\\\\\/BH029aywxrBdHj6vfw3PNLxlg48D2bI8Eq1e0MXblH9h+sa3HRFpwiQ12hsi5NxbpL76IaYw+sSmWXqf1K\\\\\\\/gIbsPLA5YIgbz6VofzO7syugPbaxZuq0oFFEjJmnZlnyK4Sr8UjOm0DXeCzR8yI7srLWHmRoxfH7LV9Y3cvcBbSPRVaZYvP7eHOU5PTSJgROCpgLantk5YgNgUS9WGDYPYdq+yHXrioYFRA+pNTr+\\\\\\\/AeDx01WAKjSmM79RC22hEFSrth48Ciw48C1EiYJPpHEGJh72uktDCr9L38XMEv2Jr66SUZtKhpNxzNgT0bny40mZfK\\\\\\\/sKlSTCq1762VuQi8HzyXxVWGIoE9QEQqDR3sSRcvh\\\\\\\/D\\\\\\\/idcDoW1lrwjbzVxkOpwO8+Bz3hIibbVM+afAYL3UI5KiiINfa0e29sLymOSvQwFQaIKexsOwSrIaABhD0N7ngu1rXHL5eBRqrgIIx113EgdDWB0isH\\\\\\\/h2vcURBqjRRxX\\\\\\\/3eLCA6o45SS8OZ7tO5xITmE8BRVgxBDivD60QBI1XXpwIPeKkPYfrmvk8xTWEW4i9G25b6qKEnJ1xm8hJOj4fwD0M+0tvcuuTwVKQXb0AZje+4+iEnBI5S1v2VK3KW1NTk28TZCdCh5EtH8jEnjpbUyaXuRRIZDVy1XTdAhmq52Wavn49CGi6FmnLSR1OizL9Tr5LEuli9fUr6I7m6dRJAvvVyy1JNuNKchq8WJ3572f0cobQUEULYyaYlt0aaHDmNt36UdqnuwH7HNnhSbhGOc6OkoGU+cZAqSxzx\\\\\\\/TH3lfl5vv3uJG8OU\\\\\\\/2v2CePlpXmLN+0T7zG3ZItdh1iMkOrrBtXNFD1zMZj0Q70aTAYh83Uf6npBNbUrlY1MF+wbuhtISSUPT79XlNzMXMRWMwT1yMKU6TRo6YeUaaEu4Wlz3qblSOY6y39VpNc4kmgUV5kSU9x5LN9rx5GCyIAU7X9g9y31JqrA+ErIa9MPovE4rC8KB1m8UBQYUUeLVcld8ypreISF951FXYZipPS1sKOb1LKTh\\\\\\\/hJa9c2kFKH17NgDtO\\\\\\\/WxdMS7\\\\\\\/nq038V0NVnxyD2V+R0B4jr7RLfJ7WE1jvxSdJ4L9LoOqpw72GJgPUFrlrWqaE2YXEZi80udIwCWge3vhd884tdd95j\\\\\\\/cNkB8MJ1GUG+AI68DqugsAoTs82NMx7KsMlI24rsV\\\\\\\/nFGpvC680obf+NHoQ5s9\\\\\\\/7ezA5AmOgYTkN7R8+vaWR347aBGLtSQAvN7cMB3gsaJUs65WMQjl5uKdwmaFcy8jiKiZLhNtbZbpA5Prw7WggCV5XRe9sdOe3rsEYFP+\\\\\\\/JWCXwVQNqGKMxL6uT99K2ihGj+bcRNuBzZ83d2QS\\\\\\\/7cmLWZnevZFL4o5iZSzOKsa2a09FcC73HMPPflOwFu+PL1SDXddKJAzpIDydtZ3iTHmohXO6jX55LwzWNh9wEdiAe2aE1SfcwVN\\\\\\\/HRp58+Vq9mRODXiwxbSqbpLY+I1EN7rNVShcHyhqdAjkmeLIvXfSEb5uuot02rM\\\\\\\/5cXqPTEq3\\\\\\\/zTpN\\\\\\\/nTatNqFRBz3aYbn0Riy6y2cAtC9Bh2tla3hLxlW72ZqKm1arZZ4Y\\\\\\\/OYsP8nNFFTv6kT7rvz7iZiioKaWMhNDObqn+Gjkf5WXMNbgeG8gmXNJyJWOAk93vBSn2L96Y\\\\\\\/pcGGq7zQcTbFZIgs+zfC0Yv0GXHFhQQCfHRNJd2nfdQ+ixGf+nx1B5Pj8FiCO8vyRG\\\\\\\/RwSbb4uDCg6y6Hl+Lq6Sr1tOPxqVvgwgbG35\\\\\\\/V68RNK0INTiIgiTLJURMs0lG2kPqhsvxd\\\\\\\/rgACM4RaiHbpcGUJEk1dzLLFOPxhAcrzh7\\\\\\\/OX\\\\\\\/USMy4CzxZ8rtfKwwfFaxy2W3VJLQdNPrf\\\\\\\/pUlMni4K+JeAW7P+TqeA25j1zozvGp2UNmQIM1ky1uqqDvz8V\\\\\\\/oCRfHs5PF7PJ9Ou+4BbvY3htVTc8bdg6kbZ7nLZmqqeCOc+dVnEMxFdyLk\\\\\\\/cVt7ED+4AXp\\\\\\\/ENWDyEiBNdg4V6xWY9wovXpxsUUEQNoOJhqIUayFs\\\\\\\/gMYC7XvyY8+hVsD9KiZuqcIODuR8ri0IAtuVMe37kHDYVv\\\\\\\/vcr\\\\\\\/ykmn1mmWsNc8avj1FvXhqOYdhYumFLu\\\\\\\/XOjP1wNU3XC63FnUXNqAlDVdn0\\\\\\\/lu8hRiuGQQq9SkQV4bWHuN7uOCMpZGrItGWk+oxPRyK1gbd56l3qmTSoSUU3TzxKEp3ajVM5hBp8zdlVCz6eZoHiRwn05MAoNIoyxIFgN0fQsSsTq16+HUN0LhAWKU6p3B\\\\\\\/GraHRu12dGQEWQDb6aZq8M3dAPg\\\\\\\/CkpQ\\\\\\\/eV2gdCZouuBq3iI2LYEaemEQBQ6BNAaj+LPCVrS0lvXBrLIEGsm7bybldOMhXOvuf4+E6aIhrvVb0TE+LGBSzBL15nZBCo8EGShU13183IhzKxBd2J8gCSlZmfb0iwk7Nj\\\\\\\/q7bzYYVWWRf2FPk4\\\\\\\/8evVR8Kd37XjWdn7V9PfKhdTAfv2mLoJm9000Pcle6Zr834ycsdzCEMzJEisT7RsCbsMIIiXn\\\\\\\/8yn2nAxPKcg1axG0sf9Q9jcUhfK+DDPxPHY3\\\\\\\/jwbpXYq\\\\\\\/016H0\\\\\\\/Zp4e1B+KkDyGm5M0FVlMcng\\\\\\\/89k4f1OLkCrlfqYDT1tC6XwPj\\\\\\\/zZxvH3ZJ9rGgeU5ZiCLVy+G\\\\\\\/+853V\\\\\\\/rfqiTo2SMtLvUX9IahccLjabFHG5OHmqr\\\\\\\/EYUrg0XDvZSzNL9wXS04Bf6E8IIcmxkD8UwYd1SI6b7AXJZEe7wpg9e97HAAPxTj7CnsJpO5J3gcSH7OKZVnjYYgkwr9p5BZnY3Azw3JIfiB2c24FMsk0L4B8sRH6il9pKWmk5am2GrABrqtf8LqgvOIXXnyyqpAKvlrJq3AHuozRefb5yS8vtdOyoqNKwhTbzby+LbWtKSiv+2IBrFotpzmRqQul9wgsGPcChSqf\\\\\\\/TNzlPtqY4AZ3XT1JDIX8oYlPyz2we3R69NXnmiQ1kNFh1D54Z1wxQMh7EFeKKbTnO48w8fUhB3i9x4zyK8otj6rkLrF\\\\\\\/giiD1oCkB+1\\\\\\\/KwW53RY7lS\\\\\\\/yQoA\\\\\\\/zywHVPsaxZBTltvkWfQHllfbtgzu96UZ0+tcEwoBx3v5ZaZ\\\\\\\/8DpPx2C+7tMya5STrTK4f\\\\\\\/cnxbO5UwvTstAcSysAWjuj6gdZgpXY876XIla3cwtYnh3l15Eu3xC04yHXY4p0QnFCXVhCMPvGwkprUUQVkhoQTtKrpch7NIkYfjqbH\\\\\\\/YuICIk1xe8zLRjHwf1cKxbF+JnCqLxF06vfg\\\\\\\/Qw0YgWfENngTrgxwBod3TohKGWfBkxkll5tBRFRKPRZmGnTi14wSZVpoBesIC5hES41QTvT\\\\\\\/nV7MtvUdDcW9PCq6VzVzka\\\\\\\/WeCWYLKc5oL0LPDdojDfSzb7sxxzOcTO4apWJt5LWQqwst5S2V+5lkZLv9WQv4X3gosCGAzPJgy1TyHyIxqZoSLFLCkPU2o1iHsDXElo5CQglDFRRhftGW0HhdwLRDPMX26+s0lIRbM3ri2+ZVdjCki5gZtpuofJMv9sTNqfScW9uyUiUMLmgcCDT\\\\\\\/T4Mq35NH7H2eDVjn7DT5DdHtaCxGNCSBZaFs99edJiqHjxbR9krMgdoeKdngkdo+fvGeNRBRjpXxn51b+cAXShLUa2hVFQxYq2l2c2ysyEuhSSt7vshTbU0OSXccd13xO4C1xoGDAp4mJIdrgaPT7D01coNB6vGudXhd6k8Woq7Fif5A702dwQ0+Hkl+d46VMgSdzLG2UB5uVJoZsFr+mbxPV4AqTqnBHDJ381j8sQ39K6T5IEx8nmT0WuPKaZFo5JR12ZBQPewBPTC1GDzikb55YylBAvO16qzAW3113TCYHQIhwiKqNJf8K2RgImJnqS3tHp+IiF8o8xLBVYqBFUJCue28ASC5iUQfs5SeYyeK+WR\\\\\\\/XI5doY4KcSP25O+sDQsSoqsGKTRZw1NaIVCtLSwiuFPhhe\\\\\\\/vRYbP+Cl0uoUA7QaSnDhk8U33Wgeppt3DdwQaxlV+mhKqu6ZUn6VuDiY02BtFroVWKKqfJHLxRDLCfK1pr4+1z8AuYqy+V4tqE\\\\\\\/YzJz7a7V5TjpqwxdwSP1jooOhx5m9vq4uO3bm46eOGBkO+Q8yUVLriYVkF3o1qKOZBrGNs1IzfumRuw9R9Ud6abOxGAO\\\\\\\/LTd8YuF7KasbrzbVNTsufCXwnl7A20f17s60PA8el1qkWR6fJ7BQwMVRD3nrV+OhPj7MuGEG8Up1i0SnFSTvWGhnujZZJXYOzNtoQwbubazq8Hi8sOMoAv8oBEUs6WbCSoifNaB8RqBiybWiEB6mU0LLQV6rxXFuZkaAoLoOnNdF3f8v9kq1COtz9YZ5gHlArTg\\\\\\\/qqkmta5ohuo3cLyJpkaPsBCim8bkwMEdSxoqEZAkpSQtgZ+8BRVwAzLKV0tLudIoZWsEA0d5GW6qm0zh9lCMWZMrv9au06o9PON3FEKRxSgqM6DFD1eZdtHrg98UMIO5gEzO2xfU\\\\\\\/orY5H6l8c3dQtyDPg4lnvmkSGBKeEA4ca0bkxhtL8TTYM71kQ+yypim0SG68lKgZ7wVWxFenjxdUsA\\\\\\\/c02UZL2siAI5WyByW1dAszSuon8aBQE8wIiOQS1v+a7uDYx3RebLsgP70XYZ1b9GAzMtKXUqnpbewL9QBwoK0ZhEkPl2RbVlmHxxdPu+4kDQJ0WMxdibvXYsIXR65hfR9zDJudUTK2SjlzGV6QPFZKlMQdC6hCQ+uJrOoAjMs3PROKBQ6g5vNTeIMVPMsM9rYF\\\\\\\/J0KNcJORBYSeqa2Acf6w6eU+Xyf4MnCi+MTMTvtvpTEpeVluH1xUnpcmrWzS687j4pmWgtz7B1xa\\\\\\\/yf7P5FWz4U27AnACbWuP1M1bwik6AZKuLA+owX8GZdUtMU7purPk983j1tHUWXYUEO0u9v8gQ\\\\\\\/Vnz4ZBFrgb7iuioRAu0bXXRDV3VsVHoQQZX2SuCkDfkbIrpXPXozj3Ouo\\\\\\\/3FNMckILlHubWERakm8rrBKRVdhUdSe82UJUD\\\\\\\/IIvHlvqNaqgTpqYoInT5c8C8ewhS9SEl\\\\\\\/t0ruEPXh0FwXaq+FUwDdDoxKvG\\\\\\\/ItMQxoGxNFSV\\\\\\\/wNrUroXgejH41Jp3wt3fddgCGXweV682F3Xru4Vh5Eu1A19m\\\\\\\/7DZKKk1nU+YFAaG+mCOf++hB22Fc9O\\\\\\\/kCEP1WaWX7HiT1AWvpvVKWDq+SpKC95a5zG\\\\\\\/QK5yNTZwExA4NkMpyzYsWmXyxpAwIPUHFP4Aq1OOHhiX8Zia4wxdyn4xcDoNPnK4VFDEHBcjurjCQL+OG7Z6R\\\\\\\/vYiYa6jFKfIYpTeEU2Xy\\\\\\\/Foy+88eN+J92kzdSBf81Zjy8YEHm\\\\\\\/WQuoKByzZNyyDfnD0HwmAY4vvcXbdrsAYB44tW7nJ9qma9IkQm2OTBCOXfJCvTpKC9NmsKMCfa6LlxfY\\\\\\\/apnopLpuFzaK5fuYzhpoVGGXkKOhfeD3x+g9Nn\\\\\\\/VrkpBgLaMwJltBmRCm5OaQr1AXAp0BmTx4tjmnhUGGUffwFLvFYlCG0GkhHO6++n1XS4sAvGq4t0A2TaBOmGGBv2iXFfT2RrD1NZOI\\\\\\\/OZmKdlzVEJQN7+o\\\\\\\/cuJ0SmQ8V+4CQi2RGRKWIidHw8vm+CyJDyU6a28EPzoMtKLOOWRAbPSW3fCLIE\\\\\\\/xjUCixFTTe1GBF\\\\\\\/Y2\\\\\\\/xVeHKT6M8xqmZvasKkiql7jPchNO3TfEg7tlaCSdraYKnoh2vrzY4\\\\\\\/q3A\\\\\\\/GFuT8hPkaffpnet2zGuFFbQyYnBDr8l3HTziZ5rWOuUYVmgEhMNBb8xv404ttkEE7ZM\\\\\\\/r1BlzkGnE+uqArX1u\\\\\\\/4iKhLRWOUdTP+w\\\\\\\/H83e\\\\\\\/sLjbCd1A4q8j2y47sg0VX3iQNuSkuL8wkIdbqVyYRZuxW25zAZFW1I34oavLnxbxl4O+LClfP4HYdexEtV1MAN5SjMrdQjW2UB6QsxDhKSq9YpvWHuvsqgW8XV+kEcoeot2Gg\\\\\\\/GTCb9HafFyj1Vb8gUsmNrr1tOOn4fSgeBH90Ack9fCCziwJybyzvE0dtFcz+pSNFjeYfU2j45h+cy\\\\\\\/FrtWnmYcWy\\\\\\\/+6gBHbphCcAQNYezgouu08InpAGCYiZUCBSaEztgIIpig84Ik3BA07iKI9xuZnGGS0slurNEyxW\\\\\\\/b3Qo3lf1d1y1Tpws9h95ufUFkfuKPnuQrBL6rnTBigfC5K0N+cIC5w4oy26P1PjshCo0zmm\\\\\\\/e971\\\\\\\/819PsnGCYB4KRDmLGknjbaYxa1Ou\\\\\\\/myzepI3YJCuQZ9cuvKBe\\\\\\\/Nt95kbeq71WdZJ14e2xXwJjvsdTc+4LKkG4\\\\\\\/CPXM7aEF0N2JTF+r58vFCxmEJRR3gH4vECsr8jwTYb8dl86SPaJ08+jSS9iM0uo4dupvNgCqYVAwo7b+ch+cAigkydUphXBerWtsVQiUQi1tgSgrbcHoxXQ6cJLEGzMT4hkkPcYbhPMQHE\\\\\\\/WJhIGUQFC4lrP6mS6KNpu\\\\\\\/eEVPOWaqUwQxPK+MmLkrgBNzJyApnDpDs6dhDYeS\\\\\\\/EpbfIYTGRSeDNRZrrLBtey2P\\\\\\\/qub5gyTpO8tPtKj2myt3gXMU7JBk+tN5mxmikdwBrRcdCPRmMdzkyFtQ5e2zoy7bm1O2Ca4SiQlFf37tGU8VBfQcF827F4Ucvlq7BcelT40I8\\\\\\\/6kO2O5I7KHnxsbRDpwsoVm47jUVnLO1gyY2QAXUpXb98uRoyX2MXDo+vRD3KmJh15ruz0ZvbjQc4RdagmIOsl4Y2whNwRpJd\\\\\\\/4kMjvwm4Tw0ODAUY12jcAjRlEwazpW+bK6GazbOn2U3r5cAsmwmWvzKnmvZ2ONgaHWIYAZ5HlZWm\\\\\\\/3VxBtm4PYLNUoIuLqnsHlfUoQedsnzWEGT9ojiAy82Rr0Ir2HcTsUYNxjH8arsCI99IFCMQjarGTtzaANrMRsUYabgct8Lk19vQylHAmfh5nr46EJ7A5OV80j2o9q3AhEIEwIyG4Lz17TTdH89oSBFtqAzpgaZcqOoLYczSYY\\\\\\\/hhPu258VKG6RbI2qsKER+nrwPfQvyq18OVKUowslUEcx6JKKWldvs6pqF66i0sHvi1bHmW2Nh5y4sUKUpN9kLdkEF8Ux9NUsdlzjTzHwpQhZqURhFqY+rJgMjTDlDjb9dOZJ9i5j1zOFtai7A\\\\\\\/m83XeNMBVQtlmat7bAJNs4hdFKCfHUy0vdk8WI17lg\\\\\\\/oFj2ZkV7x7n82i77pTdA78TEJwGZWLnxaRTvv4zC2deLdM0hlLwn9COfQ\\\\\\\/Dp72kD31NIcaJUsz2HRRAQMY3ox39eJTqfuz\\\\\\\/iz3RFZMk4oT\\\\\\\/pTqEfcbPulJNry2b5u0fOGKdKrN4PtTxC5M5fQZ+PT4pcorp5HkvDSDfZ6oXoCxQWmRGOG6vWZdlmBfrLfkhDMkswjhJ\\\\\\\/UdcuM4Lvzf7qNYV59hFVbqL6brqoPDcMPYjCEh2+IVn0Me3vrDFSiKtqhnjQsA51lFcMms45hlCjzGfIhTkZy8VrdiiSnqMO\\\\\\\/Zra3K+QgD3BofOISnOAY7b6m3QtuRh5GTpqMvFAHbscz\\\\\\\/qKB5li\\\\\\\/5F7zfNfwi8BlD0AbpYaPFX9sGR3vkw7bTfF3J+Qti7Wm2ghj0s5H0kZV36NsJpPi35JU4ih+R6o1rEHECXKYaQVZJtwwfNc3ZEPx1WhoOeK9h+rLHEAno\\\\\\\/yhitWy7861V2+Zq8OskTvAr\\\\\\\/wNGwtHshSgR43Bt66bAe+een5hivgQYOLse01+r74q0vsBdkUKjIjKS4bPaR1eE8v4KCaBwNKFj16vvdWcL4PPcKDueyDjfs5IyBsfXW+X62TcBov54aP50MDMo9DAgl07Om+IRAI4BUtXPyvHMKxmsrCo+0qdofQ271lHZ032ONmik5\\\\\\\/sElGj1yWiTWZAxGAk2QGhFWwQhVA84nLFKXj6zFAuff1K4rRvyXHrfCmF2lppygAg6qk\\\\\\\/FhqRTSVdX8FSaNiGcjUsfc4nWb0wtG1qRws2c4\\\\\\\/O9ZrXZCEmTMCsmT83IxUb936PrDGPvWG1YAaJrZPhmgi\\\\\\\/ut3gtLDkBPd6PahWCcx7zShgJ0+MKQb6DtCTgK\\\\\\\/ZNVUX5CQ+iMygdLRQ5NTpAXRaLR4iEru9x4Dbt4Xgi1wCzuY5w3ketMPi4qFc78EtlsBE1Tx3rchCPhj+56uPOR3rRvWZD4SOPNRCbY7ZHP0gXeQ8bq5YWMTQi6ADmcXorm9go5EXgYY537IT0iZYyvmuBdvPxDoDe3rqd4SLNW\\\\\\\/5\\\\\\\/hh7mf2zeMb7B7cQ\\\\\\\/YZSnjCiCVf3HA3Jw9m6ksRuIS2zUDHrIadjNbnB3H5Xlu3G\\\\\\\/K2ekOiVud2VIu36UxArUpqW62CViRySyp0uizj9D\\\\\\\/9\\\\\\\/2UsNLOLIC0gjqpBkyYuZc0cEYsA8jMj5RIjqKpHTxHg4rIMtgJG9++YfSFCURbgnnlpHWGyWZO97rGIaqY87cZJEFwNsOzWnBE59QZ2dWenNstWga\\\\\\\/dY1ToIDDI9nyB8Oj5Kf2bTxbVcifqaTKlNHQS3cr50xV+XOHma33kFEYkgeDJ+vAoWsxIhkXaIYWcD2dNJry\\\\\\\/bbwNLKBGt6hpPB8trSFnB0Kj3PtnSBsO6UVl12Vpza+YMLBN2NUAz8a2JYmRn+HYuHLmyfy9jCC8ARmIVZSbT2RTTEIA\\\\\\\/E5BMswQFBi\\\\\\\/3RBgWLXBDkQoqjM5XNfyQNNIR\\\\\\\/wrVY3uMx8WTGqAp5swyGuayvOKhyXcH43Yk2wMDh43Z5ucOG26Q9AaFxeXDJ8jGyg35Z0fq8puzcIn898jPUaLN6OJthGwiUbWUsdlSbswqhCUuJk6IJilbMSpUoxBymqrLZtKPxleDjRSzSdb3QuD8cHJ7d6P\\\\\\\/mq+a+bGw1dUKE0AH0YmBU+5fVQeHVqOEXmKT9wNEYUciXbKiZub0WFOmwoWO9Rxz5jXgJP6DinAzjGEmjYICvlcIRkXzUnrA1FXxFPPbMzX6ZvNjaygZOxcyKrrNSdkIrAEQ05832fHPQqwnpleDyZDYRQq4X1F4owZG1QF5\\\\\\\/hcPtKi+Ga94Gkf6iJX4kMUckygM9qthmD65iIQsJluNidnNOIrjkltGtPwKRCxWU8lMytQ79d+ceZQEo1eS7xMQbKbY8a6p1S5CM0mAlApgQpuhqShmqyPlO4arJIw27gH7hp8692vZTJbVBxW5+CBDFxiA\\\\\\\/Xj1+awwWHzQ7l4wDWTZbe7\\\\\\\/XO9Mdki2w4+WEUuwhl4\\\\\\\/0HFOdDd\\\\\\\/KWv1nMk2psIY8qcF\\\\\\\/Q\\\\\\\/ISbKMy9KL3Zxk1A\\\\\\\/dUOGkgBGwN4tUHhA7mWzt8ayyovmwCSRZ\\\\\\\/1F61onXjRA46X3xdZv1wsaEGly4NRXdRP6U86XcsqJW4gcZO+in+hqoKMQxa6hnrAfmzGEY6GAiLUPlqBq8BeFmeKUJPvDszP9DaT20gz6TlqTb3uij0Vgp57E3VizBfCYkwlsIkFI1UfylP\\\\\\\/06Z1uzYTQJjJvh77ga5QW\\\\\\\/bmomLR4OqaPoPQhI4BS9zP5BwPc1zCNGBViUm0DwAZ0tn9B53st9wkcXcxq7l4XLyhJN09oVNzuHQu63J4FksXQBNmEqruY\\\\\\\/2RTNPdHT4qZhh3IW+ZKJOA5GBVM\\\\\\\/GOWXQwHIb87GP7pvEn3cCtpBRiMpTlxKGoPF5iVC9ajRPLKftjpC9mfjhwR4hiKWquoycfnmw\\\\\\\/bhcIH4\\\\\\\/TB\\\\\\\/oJGAVpKcjfdOOLXSsUDbEca+dJ5eM3NPohpU2g83cvvtulfZD4eJDs7Ysu4jh6PAOCIJz7ldfzFYcIUhB1DVSg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022VLLKf\\\\\\\/yvjcPj2TFpkKqM4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002239daebcca98ccdf4d5cc5f318f19f73052f8015b55d119193224307a30690306\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022W\\\\\\\/AB1kEeWG75wABEd85K\\\\\\\/dTOUO8TF1ee867\\\\\\\/1mzEojzPQWJCy4uizQ7LuBCZVf6zR6JqmaC7M6vEk2O2gQhJcEwk2ri5gJmve0M4FxKf0WpmKRX9TCcJtF5QtEjULBmyrURmYTjLK4h1pv54IjD63D66GWwaRXmlym6Qzu0JueRqxQsf5D1A2p52DJZCvGw0p\\\\\\\/dV7QWJiVoMqtN56YGp1KqQhNPvhOzU9WT64c1huVHY+z1qjQsmY\\\\\\\/QbbOqOoCkRImsUA\\\\\\\/DDWFkraOzP+Um0CRiVKOM9hnnEopzBY8l2gffUAaS0ymHLCDX5pGj3E5VKE7ZekgBsWWiwhz9uaJm3lkM+ObAKV4d5vORDH6MRQG6zoNsdqA89QVEGuJIQbQszIfkuNwmpcgb\\\\\\\/MCDfnHC35HK\\\\\\\/RrlcUpzjcF5wTeA8XOm0HPJSauGDEQzgac3NhxHgSr3ZXdODmFaqLKbGsSv1eW+X+HCrg+pgJI4Af2DvMfuspZWuLKmGGCd9D0PMvRHb1yw2u\\\\\\\/66VtgkP8mmRu\\\\\\\/YDSAtQEa0y+kXFoyOfaVxeUE55z04BVpZPwMWGLbeM6qaCp2aE6twWvTd4Kmtt6Aayhpx6+2N6T31Gvi4YGq30\\\\\\\/QGT8Q5+PuI7PHvVY84M2y6dBKB04qumOn\\\\\\\/r+5P2bWF3YpS4JJaHUh2gPeeKdBgBMa33UNjnR0OH4MK4k4adu2\\\\\\\/4sy9cT0yo0PP3ayEFRJ577SAQT85pYj8cVxIikLH3MfIJ8HlOtpk4QIwheKd35KR0Q5lRAxF02iu+p3sXbwavB5+XLcFNtIlWqi\\\\\\\/YRU41Jfiq67Pkk6qy4HfpTxHF9WBiH32NeBJzm9SmWHw2O3wrRboCb4mVx48kL8l4dEGKTM4jiekHR1vDseAzdbmeI5a4PfIyPrOQytsVImyMQRiYXfBZcGuQS+uJcYo23zC\\\\\\\/xEmOUbfYqkqAH+sEGqGvDTM+rSizIzM1pkITNMNifYZQs3QArYlkwXpxsBUTRz+NcLnmm2EBNDeqzPtA35m5AQKbLHdP0URyBO33mouXsDqkYwvyxSypNDVKwj575XuYwLOA8Z6hZ9rq7iHyoTvFtiKV9Yb6CyRxLwspv2aDR+8w0s6jl7DvyJgOtyTcdiJoNrm7Gfc6e2hqzNGnT5+KSJPJacmfL99WpIMOx5RuxjihPk9TnNWQzm5BoaHlmtp3JcwtFLrLFKG6nC8LMcDglGIxxjTOg9PNhm\\\\\\\/Jmy1MQqz8x3uu5PBqw\\\\\\\/e0+t+h6Y51E7RYseFnm+GZyON4Q6JGKM03q50Xfaqx3MCq2JZV1wqTmJJWGrZnz6PmnoITUxpmOmBfCp7J\\\\\\\/O8Me2MAkoTqxX2S6qqcYur5DbmK1kMAHCvTOuEdOMwVrf7T1hKGQ\\\\\\\/M64LoFVxrB91D2bvumlaMey+Dm\\\\\\\/IyzNc9Su2u83Uiw8eu1XYFnh807eoF\\\\\\\/eQ9SV3iHH9s+mZ2TLTQTpWX3FMCDRmlYIX4DT5xCtyMSxeNlEZxPK6A0uK4r6+HKNpM3RHYhRjufiTjOepoMIRWQ9LfehakCcpcn6fyxXNG+2g7ZOUfFWJcrH+rTk\\\\\\\/NToNMaCJXrWc1KPNmBU1ZYRETRwwbuUkVbOoLSc2\\\\\\\/jxK8u8BwmpBuJ5AJm0tTNKKqQeVGT5elnVuqkCiU7+2+7C\\\\\\\/Dt5Pw\\\\\\\/u\\\\\\\/veQylGL6OgB+BoLxFPmXizDzIO5u5ffwooI5gwTf1gmtgeG7vqV4iiaJ859RNSJm56m8r86v31z3DPbI8oAg4Sg6khvfuCongHGH7PMTIkhjN9mEnsuATzXnpRjZ6GjbjF7KUzy7c1gqCYUq4tikojRg+MU8d553n5Ld80hAZolod5+FDMop6eEWYsbEGBxR60+MOfYIK1MdZNEFTW2iXJkf75WjNZX8Se+jxYCvfRAaSgYhxUKqk0OkJI\\\\\\\/mOwQL\\\\\\\/2Qn7iZQhenDaV\\\\\\\/To+eGFGtYpYAHUQzgy1awP7ocRAd11c3OE\\\\\\\/pV5zEPe0HdYAKRH0yZnKAGqRqFcZw0VC7WIZg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022iav8jEbMi6trxm2RR8Ln5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229ca4196d9d02fb72dfc4dd74a924b9857ba5c338c7ae9df9b2aa4908ac46f463\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460286 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2799" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yBvl1Wbyj2hBOw5K1MAEt8AYDUi6ft5HuKwKV2zCJxI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2799" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2D_wpS72yBEa4_8U-x0iV3gE4NglyBimwQmF_GDU_Kg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2791" title="Whole Genome Comparison of Thermus sp. NMX2.A1 Reveals Principal Carbon Metabolism Differences with Closest Relation Thermus scotoductus SA-01" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2809" title="A Split-Ubiquitin Based Strategy Selecting for Protein Complex-Interfering Mutations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460286" id="node460286" data-pisa="ggg;6/9/2799" data-pisa-master="ggg;g3.116.032888" data-apath="/ggg/6/9/2799.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Canopy Temperature and Vegetation Indices from High-Throughput Phenotyping Improve Accuracy of Pedigree and Genomic Selection for Grain Yield in Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jessica Rutkoski</span>, <span class="highwire-citation-author" data-delta="1">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="2">Suchismita Mondal</span>, <span class="highwire-citation-author" data-delta="3">Enrique Autrique</span>, <span class="highwire-citation-author" data-delta="4">Lorena González Pérez</span>, <span class="highwire-citation-author" data-delta="5">José Crossa</span>, <span class="highwire-citation-author" data-delta="6">Matthew Reynolds</span> and <span class="highwire-citation-author" data-delta="7">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2799-2808;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032888 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460286" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Programs, College of Agriculture and Life Sciences, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding and Genetics Section, School of Integrated Plant Sciences, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jer263@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">jer263@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Pathology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Suchismita Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suchismita%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuchismita%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Enrique Autrique </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Enrique%2BAutrique%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Autrique%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEnrique%2BAutrique%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lorena González Pérez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorena%2BGonz%C3%A1lez%2BP%C3%A9rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALorena%2BGonz%25C3%25A1lez%2BP%25C3%25A9rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Matthew Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/9/2799" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460286" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2799.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460286" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2799.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460286" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2799.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460286" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/9/2799.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460286" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection can be applied prior to phenotyping, enabling shorter breeding cycles and greater rates of genetic gain relative to phenotypic selection. Traits measured using high-throughput phenotyping based on proximal or remote sensing could be useful for improving pedigree and genomic prediction model accuracies for traits not yet possible to phenotype directly. We tested if using aerial measurements of canopy temperature, and green and red normalized difference vegetation index as secondary traits in pedigree and genomic best linear unbiased prediction models could increase accuracy for grain yield in wheat, <em>Triticum aestivum</em> L., using 557 lines in five environments. Secondary traits on training and test sets, and grain yield on the training set were modeled as multivariate, and compared to univariate models with grain yield on the training set only. Cross validation accuracies were estimated within and across-environment, with and without replication, and with and without correcting for days to heading. We observed that, within environment, with unreplicated secondary trait data, and without correcting for days to heading, secondary traits increased accuracies for grain yield by 56% in pedigree, and 70% in genomic prediction models, on average. Secondary traits increased accuracy slightly more when replicated, and considerably less when models corrected for days to heading. In across-environment prediction, trends were similar but less consistent. These results show that secondary traits measured in high-throughput could be used in pedigree and genomic prediction to improve accuracy. This approach could improve selection in wheat during early stages if validated in early-generation breeding plots.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/secondary-traits-genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-secondary-traits-in-genomic-selection" rel="nofollow">Secondary traits in genomic selection</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/multivariate-analysis" class="hw-term hw-article-keyword hw-article-keyword-multivariate-analysis" rel="nofollow">multivariate analysis</a></li><li class="kwd"><a href="/keyword/selection-index" class="hw-term hw-article-keyword hw-article-keyword-selection-index" rel="nofollow">selection index</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">Genomic selection (GS) and high-throughput phenotyping (HTP) have great potential to increase the efficiency of wheat, <em>Triticum aestivum</em> L., breeding programs. GS is the use of markers covering the whole genome for selection. With GS, reviewed by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Lorenz <em>et al.</em> (2011)</a>, a training set that has been phenotyped and genotyped is used to calibrate a prediction model, which is then used predict the breeding values of a ‘test set’ of genotyped selection candidates. This enables indirect selection for quantitative traits prior to phenotyping. Genomic selection has already been implemented in dairy cattle breeding to increase rates of genetic gain (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Pryce and Daetwyler 2012</a>), and simulation studies have demonstrated that GS can increase rates of genetic gain in crop plants (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bernardo and Yu 2007</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Wong and Bernardo 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Heffner <em>et al.</em> 2010</a>). In contrast to GS, HTP is the use of remote and proximal sensing to measure a large number of phenotypes across time and space at low cost and with less labor intensity. If traits measured using HTP are correlated with those of economic importance, HTP data could be used to dramatically increase both the selection accuracy and intensity.</p><p id="p-4">In wheat, canopy temperature (CT) and normalized difference vegetation index (NDVI) are associated with components of grain yield (GY) and can be routinely measured using HTP platforms (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Reynolds and Langridge 2016</a>). Canopy temperature is an indicator of evaporative cooling from the canopy surface. Cooler CT is associated with greater stomatal conductance and increased gas exchange rate under irrigated conditions and better hydration status under drought (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Pinto <em>et al.</em> 2010</a>). Normalized difference vegetation index is an indicator of canopy size and greenness, and is referred to as RNDVI when based on the difference between near-infrared and red reflectance (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Wiegand and Richardson 1990a</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">1990b</a>), or GNDVI when based on the difference between near-infrared and green light reflectance (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gitelson and Merzlyak 1997</a>). While RNDVI is only sensitive to low chlorophyll-a concentration, GNDVI is sensitive to a wide range of chlorophyll concentrations (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Gitelson and Merzlyak 1997</a>). Larger NDVI values are associated with greater biomass accumulation and a faster growth rate when measured during the vegetative (VEG) phase, and a longer grain filling (GF) duration and delayed leaf senescence when measured during the GF phase (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Babar <em>et al.</em> 2006</a>). Measurements of CT, GNDVI, and RNDVI taken at different growth stages can be considered separate traits, though they generally have a high correlation and can be expected to share some common physiological (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Babar <em>et al.</em> 2006</a>) and genetic (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Pinto <em>et al.</em> 2010</a>) bases.</p><p id="p-5">In plant and animal breeding, indirect selection for traits that are expensive or difficult to measure using correlated secondary traits is common. Examples from wheat breeding include selection for reduced plant height to improve harvest index and lodging resistance, and selection for higher protein to improve quality. Selection on secondary traits is advantageous when the secondary trait is highly heritable, highly genetically correlated with the target trait, and it is inexpensive to measure relative to the target trait. One of the challenges of using secondary traits for indirect selection is that the relative utility of secondary traits is situation dependent. Incorporating secondary traits in multivariate pedigree or genomic prediction models partially overcomes this problem because genetic covariances between traits are estimated using a model training set that is representative of the selection candidates and evaluated in the target environment(s). When used in multivariate pedigree or genomic prediction models, secondary traits have been found to improve prediction accuracy and reduce bias compared to univariate models, especially when secondary traits are measured on both the model training population and the selection candidates (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Calus and Veerkamp 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Jia and Jannink 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Pszczola <em>et al.</em> 2013</a>). Evidence also suggests that secondary traits, like genome-wide markers, can capture the Mendelian sampling term, and may reduce the advantage of genomic prediction over pedigree prediction (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Pszczola <em>et al.</em> 2013</a>).</p><p id="p-6">In wheat breeding, prior to GY testing in large replicated plots, a large number of lines are grown in small plots in the field for both seed increase and visual selection. Because measurements of GY on small plots are not meaningful, accurate predictions of GY at this stage would enable an increase in the rate of genetic gain. These predictions could be generated using all available data, including correlated traits and pedigree or genome-wide markers in multivariate mixed models. By including correlated traits observed on selection candidates, both pedigree and genomic selection accuracies could be improved. Because breeding lines must be grown in the field for seed increase prior to GY testing, there would be only a marginal additional cost of HTP, which is quite low, at least in the case of the CIMMYT wheat breeding program.</p><p id="p-7">Canopy temperature, GNDVI, and RNDVI could be excellent secondary traits for pedigree and genomic prediction of GY in wheat because of their high heritabilities and genetic correlations with GY, and because they can be measured remotely on large numbers of selection candidates, possibly during the seed increase generation prior to GY testing. The idea of using CT, GNDVI, or RNDVI to indirectly select for GY is not new; however, an evaluation of these traits for their potential to improve genomic and pedigree prediction accuracies for GY has not yet been done in wheat or other cereals. The objective of this study was to determine if CT, GNDVI, and RNDVI measured using an aerial HTP platform during VEG and GF stages could improve accuracies for GY in wheat when used as secondary traits in multivariate pedigree and genomic prediction models.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Phenotyping</h3><p id="p-8">A total of 1092 inbred breeding lines were grouped into 39 GY trials, and grown during the 2013–2014 crop season at the Norman E. Borlaug Research station in Ciudad Obregon, Sonora, Mexico. Each trial consisted of 28 breeding lines and two checks, and was arranged in an alpha lattice design consisting of three replicates and six blocks. The trials were grown in each of the following environments, ‘early heat’, bed sowing 30 d before the optimal plating date with optimal flood irrigation; ‘optimal’, bed sowing at the optimum planting date with optimal flood irrigation; ‘drought’, bed sowing at the optimum planting date with reduced food irrigation; ‘severe drought’, flat sowing at the optimum plating date with minimal drip irrigation; and ‘late heat’; bed sowing 90 d after the optimal plating date with optimal flood irrigation. Trials in early heat were sown in October. Optimal, drought, and severe drought trials were sown in mid-November, and late heat trials were sown during the last week of February. During the crop season, the optimally irrigated environments received 500 mm of water, and drought and severe drought environments received 250 and 180 mm of irrigation, respectively.</p><p id="p-9">Days to heading (DTHD) was recorded as the number of days from germination until 50% of spike emergence in each plot, and was recorded in the first replicate of each trial. Lodging, which was observed only in the optimal environment, was recorded visually on a zero to five scale. GY was the total plot GY measured after maturity.</p><p id="p-10">High-throughput phenotypic data were collected with a thermal (Model A600 FLIR Infrared camera, Wilsonville, OR) and hyperspectral camera (A-series, Micro-Hyperspec VNIR, Headwall photonics Fitchburg, MA) mounted to a manned aircraft. Data were collected around solar noon time on each date, aligning the aircraft to the solar azimuth for data acquisition. Images of the experimental fields were obtained, and formatted to tabular data by calculating the mean value of the pixels inside the center of each individual trial plot represented as a polygon area on a map.</p><p id="p-11">The 38 cm per pixel CT data were corrected with a linear calibration using parameters calculated based on previous field and camera measurements. For each flight, the individual <strong>i</strong>mages were used to compose a unique mosaic that were then manually georeferenced. The original image data were stored in Kelvin units × 100, and pixel values were converted to Celsius degrees according to the formula (Pixel value)/100 – 273.15.</p><p id="p-12">The 30 cm per pixel hyperspectral data were calibrated radiometrically with coefficients provided by the Laboratory for Research Methods in Quantitative Remote Sensing of the Consejo Superior de Investigaciones Científicas (QuantaLab, IAS-CSIC, Spain) derived with a calibrated uniform light source. Dark frame subtraction was also performed to reduce the noise of the sensor. Correction to decrease the effects of the atmospheric conditions in the images was performed by modeling irradiance based on sun-photometer field measurements (Microtops II, Solar Light Company, Glenside, PA). The images were orthorectified and coarsely georeferenced based on the built-in Inertial Navigation System (INS). For data extraction, images were aligned manually whenever images did not overlay the plot polygons due to INS inaccuracy.</p><p id="p-13">Data from CT, RNDVI, and GNDVI were grouped into VEG and GF stages (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) as in <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Pask <em>et al.</em> (2014)</a> and <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Mondal <em>et al.</em> (2015)</a>. There were two to five measurement dates per growth stage for all environments, except late heat, where only one measurement date coincided with the VEG stage. Canopy temperature, GNDVI, and RNDVI were then renamed as CT-VEG, CT-GF, GNDVI-GF, GNDVI-VEG, RNDVI-GF, and RNDVI-VEG according to the growth stage classification.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469106/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469106/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Classification of secondary trait measurement dates into vegetative and grain filling stages</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Phenotypic data quality control</h3><p id="p-15">Within each trial within environment, we calculated repeatability for CT, GNDVI, and RNDVI measured on individual dates, and GY. This within-date repeatability, based on the phenotypic data, was calculated as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-1.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-2.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-3.gif"/></noscript></span></span> are the genetic, replicate, and residual variances, respectively, and <em>nrep</em> is the number of replicates of the breeding lines in the trial (<em>nrep</em> = 3). The variance components were estimated by fitting the model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>y<sub>ij</sub></em> is the phenotype, µ is the mean, <em>g<sub>i</sub></em> is the random effect of genotype assuming identical and independently distributed (iid) <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-5.gif"/></noscript></span></span>), <em>r<sub>j</sub></em> is the random effect of replicate with iid <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-6.gif"/></noscript></span></span>, and <em>ε<sub>ij</sub></em> is the residual with iid <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-7.gif"/></noscript></span></span>. Significant outliers (<em>p</em>-value &lt; 0.001) were identified using Studentized residuals. If outliers were found, they were removed and repeatability was recalculated.</p><p id="p-16">Similarly, whenever there were multiple measurements across dates we calculated overall repeatability across time and space within each trial for CT-VEG, CT-GF, GNDVI-VEG, GNDVI-GF, RNDVI-VEG, and RNDVI-GF. Overall repeatability for each trait within trial within environment was calculated as <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-8.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-9.gif"/></noscript></span></span> are the genetic, the genotype-by-date interaction, and residual variances respectively, <em>ndate</em> is the number of phenotype dates for the trait, which ranged from two to five (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), <em>nrep</em> is the number of replicates in the trial (<em>nrep</em> = 3). Variance components were estimated by fitting a mixed model similar to (1) but adding the random effects of the dates and the interactions date × environment as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <em>y<sub>ijk</sub></em> is the phenotype, µ is the mean, <em>g<sub>i</sub></em> is the random effect of genotype with iid <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-11.gif"/></noscript></span></span>), <em>d<sub>j</sub></em> is the random effect of date with iid <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-12.gif"/></noscript></span></span>, <em>r<sub>k(j)</sub></em> is the random effect of replicate nested within date with iid <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-13.gif"/></noscript></span></span>, <em>gd<sub>i,j</sub></em> is the random effect of genotype-by-date interaction with iid <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-14.gif"/></noscript></span></span>, and <em>ε<sub>ijk</sub></em> is the residual with iid <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-15.gif"/></noscript></span></span>. If outliers were detected, they were removed, and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-16.gif"/></noscript></span></span> was recalculated. We also calculated <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-17.gif"/></noscript></span></span>, excluding individual dates with <em>r<sup>2</sup></em> &lt; 0.01. Individual low-repeatability dates were then removed if doing so improved <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-18.gif"/></noscript></span></span>. Lastly, we removed trials where any of the traits had <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-19.gif"/></noscript></span></span> &lt; 0.01. Note that, for model (2), we have assumed a very restrictive model that assumes that the dates are identically independent distributed. The resulting dataset (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032888/-/DC1/FileS1.zip">File S1</a>) contained 616 lines.</p></div><div id="sec-4" class="subsection"><h3>Genetic value estimation</h3><p id="p-17">To estimate genetic values for traits measured across multiple dates for subsequent use in prediction modeling, for each environment, we estimated best linear unbiased estimates (BLUEs) of the breeding lines by fitting the mixed model:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-20.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <em>y<sub>ijklm</sub></em> is the phenotype, µ is the mean, <em>g<sub>i</sub></em> is the fixed effect of genotype (BLUE), <em>t<sub>j</sub></em> is the random effect of trial with iid <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-21.gif"/></noscript></span></span>, <em>d<sub>k</sub></em> is the random effect of trait measurement date with iid <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-22.gif"/></noscript></span></span>, <em>r<sub>l(jk)</sub></em> is the random effect of replicate within trial and date with iid <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-23.gif"/></noscript></span></span>, <em>b<sub>m(jkl)</sub></em> is the random effect of incomplete block within trial, date, and replicate with iid <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-24.gif"/></noscript></span></span>, and <em>ε<sub>ijklm</sub></em> is the residual with iid <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-25.gif"/></noscript></span></span>. For traits that were measured only once (<em>e.g.</em>, GY), the random effect of date was excluded from model (3). For DTHD, which was measured only once, and on one replicate, the date, replicate, and block effects were excluded from model (3). To enable us to simulate a scenario where the individuals in the test set are not grown in a replicated incomplete block design, as is the case in wheat breeding prior to the yield testing phase, we removed the effects of replicate and incomplete block from model (3), and used this model to estimate BLUEs on a per replicate basis.</p><p id="p-18">To enable across-environment prediction model training, BLUEs were also estimated for each trait across all environments except one which was designated as the validation (testing) environment. To estimate these BLUEs, a random effect for environment was included in model (3), all effects except for the genotype effect were nested within environment, and data from all environments except the validation environment were included in <em>y</em>.</p><p id="p-19">For prediction model validation, best linear unbiased predictors (BLUPs) of the breeding lines were calculated for GY within each environment by fitting the mixed model:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <em>y<sub>ijkl</sub></em> is the phenotype, µ is the mean, <em>g<sub>i</sub></em> is the random effect of genotype with iid <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-27.gif"/></noscript></span></span>, <em>t<sub>j</sub></em> is the random effect of trial with iid <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-28.gif"/></noscript></span></span>, <em>r<sub>k(j)</sub></em> is the random effect of replicate within trial with iid <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-29.gif"/></noscript></span></span>, <em>b<sub>l(jk)</sub></em> is the random effect of incomplete block within trial and replicate with iid <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-30.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-31.gif"/></noscript></span></span> is the fixed effect covariate for lodging (fit only in the optimal environment where lodging was observed) for the <em>i</em>th genotype in the <em>j</em>th trial, <em>k</em>th replicate and <em>l</em>th block, and <em>ε<sub>ijkl</sub></em> is the residual with iid <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-32.gif"/></noscript></span></span>. In order to validate prediction models that included a covariate for DTHD, BLUPs for GY corrected for DTHD were calculated by including DTHD as a fixed effect for the <em>i</em>th genotype in the <em>j</em>th trial in model (4).</p></div><div id="sec-5" class="subsection"><h3>Genotyping</h3><p id="p-20">Genotyping-by-sequencing (GBS) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Elshire <em>et al.</em> 2011</a>) was used for genome-wide genotyping. All 1092 lines selected for this analysis were included, along with a larger set of 19,965 lines, and were sequenced at 192-plexing on Illumina HiSeq2000 or HiSeq2500 with 1 × 100 bp reads. Single nucleotide polymorphisms (SNPs) were called across all lines using the TASSEL GBS pipeline (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Glaubitz <em>et al.</em> 2014</a>) anchored to the genome assembly of Chinese Spring (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">International Wheat Genome Sequencing Consortium 2014</a>).</p><p id="p-21">SNP calls for the set of 1092 breeding lines were extracted, and then markers were filtered so that percent missing data and percent heterozygosity per marker did not exceed 80% and 20%, respectively. Next, lines with &gt; 80% missing marker data were removed, and markers were recoded as –1, 0, and 1, corresponding to homozygous for the minor allele, heterozygous, and homozygous for the major allele respectively. Lines that did not overlap with the set of 616 lines with quality phenotypic data for all traits in all environments were removed. Next, markers with a minor allele frequency &lt; 0.01, and &gt; 80% missing data were removed, and missing data were imputed with the marker mean. A total of 12,083 SNPs scored on 557 individuals was used for subsequent analysis.</p></div><div id="sec-6" class="subsection"><h3>Pedigree and genomic relationship matrices</h3><p id="p-22">For the set of 557 lines with quality phenotypic and genotypic data, the genomic relationship matrix (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032888/-/DC1/FileS2.zip">File S2</a>) was estimated according to equation 15 in <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Endelman and Jannink (2012</a>). For the same set of lines, the pedigree relationship matrix (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032888/-/DC1/FileS3.zip">File S3</a>) was estimated as 2× the coefficient of parentage.</p></div><div id="sec-7" class="subsection"><h3>Square roots of heritabililites and genetic correlations</h3><p id="p-23">For traits measured across multiple time points, square root of broad sense heritability (phenotypic selection accuracy) on a single plot basis, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-33.gif"/></noscript></span></span>, and on a line mean basis, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-34.gif"/></noscript></span></span>, for each trait within environment were calculated as <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-35.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-36.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-37.gif"/></noscript></span></span> is the genetic variance, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-38.gif"/></noscript></span></span> is the error variance, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-39.gif"/></noscript></span></span> is the genotype-by-date interaction variance, <em>ndate</em> is the number of measurement dates for the trait, and <em>nrep</em> is the number of replications (<em>nrep</em> = 3). A random effect of genotype-by-date interaction effect was added to model (3) for variance component estimation. Square roots of heritabilities were also estimated for all traits corrected for DTHD by adding a fixed effect covariate for DTHD to the model. For traits that were measured at one point in time (such as GY), <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-40.gif"/></noscript></span></span>and, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-41.gif"/></noscript></span></span>, were calculated as <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-43.gif"/></noscript></span></span>, and for variance component estimation, the effect of measurement date and genotype-by-date interaction was removed from the model.</p><p id="p-24">Genetic correlations between traits including CT-VEG, CT-GF, GNDVI-VEG, GNDVI-GF, RNDVI-VEG, RNDVI-GF, and GY were calculated based on estimates of variances and covariances from a multivariate Gaussian mixed model:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-44.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <em>n</em> is the number of traits, y<sub>1</sub> is a vector of BLUEs for trait one, y<sub>n</sub> is a vector of BLUEs for trait <em>n</em>, where BLUEs were estimated using model (3), <strong>X</strong> is the fixed effects design matrix, which is the same for each trait, <strong>u</strong><sub>1</sub> is a vector of fixed effects for trait one, <strong>u</strong><sub>n</sub> is a vector of fixed effects for trait n, <strong>Z</strong> is the random effects design matrix, which is the same for each trait, <strong>a</strong><sub>1</sub> is a vector of fixed effects for trait one, <strong>a</strong><sub>n</sub> is a vector of fixed effects for trait n, ε<sub>1</sub> is a vector of residuals of trait one, and <strong>ε<sub>n</sub></strong> is a vector of residuals for trait n. Variance components were estimated assuming <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-45.gif"/></noscript></span></span>, and again assuming <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-46.gif"/></noscript></span></span>, where <strong>A</strong> is the pedigree relationship matrix, <strong>G</strong> is the genomic relationship matrix, and <strong>H</strong> is the variance-covariance matrix for the breeding values of the traits. In both models, <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-47.gif"/></noscript></span></span>, where <strong>I</strong> is an identity matrix, and <strong>R</strong> is residual variance-covariance matrix between the traits. The covariance matrices <strong>H</strong> and <strong>R</strong> were assumed unstructured. For each relationship matrix, <strong>A</strong> and <strong>G</strong>, correlations were estimated twice, once including a fixed effect covariate for lodging in the optimal environment, and once including a fixed effect covariate for DTHD in all environments and lodging in the optimal environment. To estimate the genetic correlations between DTHD and GY in each environment, and with each relationship matrix, we used bivariate models (model (5), with <em>n</em> = 2 traits).</p></div><div id="sec-8" class="subsection"><h3>Prediction models and validation</h3><p id="p-25">After quality control, complete data on 557 breeding lines remained, and was used for prediction modeling. Accuracies from a multivariate mixed model incorporating GY on the training set only, and secondary traits on both the training and test sets (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) were compared to those of a univariate mixed model, which does not incorporate secondary traits. The univariate prediction model was:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-48.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <em>y<sub>i</sub></em> are the BLUEs of the breeding lines from model (3), µ is the mean, <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-49.gif"/></noscript></span></span>is a fixed effect covariate for lodging fit only in the optimal environment, <em>g<sub>i</sub></em> is a random effect of genotype with <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-50.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-51.gif"/></noscript></span></span>, and <em>ε<sub>i</sub></em> is the residual error with iid <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-52.gif"/></noscript></span></span>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2799/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Data used for univariate and multivariate prediction modeling. Each box indicates the presence or absence of phenotypic data for a particular trait on either the training or test set. Presence and absence of phenotypic data are indicated by black stripes and solid gray, respectively. The trait and population of interest for prediction is marked with a black asterisk. The traits include CT during the grain filling phase (CT-GF), CT during vegetative phase (CT-VEG), GNDVI during the grain filling phase (GNDVI-GF), GNDVI during the vegetative phase (GNDVI-VEG), RNDVI during the grain filling phase (RNDVI-GF), RNDVI during the vegetative phase (RNDVI-VEG), and grain yield (GY)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1215542205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Data used for univariate and multivariate prediction modeling. Each box indicates the presence or absence of phenotypic data for a particular trait on either the training or test set. Presence and absence of phenotypic data are indicated by black stripes and solid gray, respectively. The trait and population of interest for prediction is marked with a black asterisk. The traits include CT during the grain filling phase (CT-GF), CT during vegetative phase (CT-VEG), GNDVI during the grain filling phase (GNDVI-GF), GNDVI during the vegetative phase (GNDVI-VEG), RNDVI during the grain filling phase (RNDVI-GF), RNDVI during the vegetative phase (RNDVI-VEG), and grain yield (GY).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2799/F1.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/9/2799/F1.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2799/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2799/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469148" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Data used for univariate and multivariate prediction modeling. Each box indicates the presence or absence of phenotypic data for a particular trait on either the training or test set. Presence and absence of phenotypic data are indicated by black stripes and solid gray, respectively. The trait and population of interest for prediction is marked with a black asterisk. The traits include CT during the grain filling phase (CT-GF), CT during vegetative phase (CT-VEG), GNDVI during the grain filling phase (GNDVI-GF), GNDVI during the vegetative phase (GNDVI-VEG), RNDVI during the grain filling phase (RNDVI-GF), RNDVI during the vegetative phase (RNDVI-VEG), and grain yield (GY).</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">The multivariate prediction models were as described in model (5), and assumed either <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-53.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-54.gif"/></noscript></span></span>. Prediction models included a covariate for lodging in the optimal environment. Whenever training and test set secondary trait data were not based on the same number of observations, leading to heterogeneous residual variances, weights based on the number of observations were included in the diagonal of <strong>R</strong>.</p><p id="p-28">Accuracies were estimated using fivefold cross validation with the same fold assignment for all analyses. For each fold, the predictive ability was calculated as the Pearson’s correlation between predictions for GY and the BLUPs for GY from model (4). Prediction accuracy, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-55.gif"/></noscript></span></span>, was calculated as <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-56.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-57.gif"/></noscript></span></span> is the mean predicative ability across folds. SE of the prediction accuracy (<span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-58.gif"/></noscript></span></span>) was calculated as <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-59.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-60.gif"/></noscript></span></span> is the SD of the predictive ability. The average accuracy across the five environments was calculated as the mean of the prediction accuracies (with one accuracy per environment). The SE of the average accuracy was calculated as the SD of the accuracies divided by <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-61.gif"/></noscript></span></span>. Cross validation was conducted within each environment, and across-environment where training and validation set data were from different environments. To perform the latter, for each environment, we performed fivefold cross validation but with data from the environment of interest left out of the training set.</p><p id="p-29">Multivariate prediction model accuracies were estimated once where BLUEs of secondary traits on the test set included data from all three replicates, and once where BLUEs of secondary traits on the test set were estimated using data from the first replicate only. Accuracies were also estimated including a covariate for DTHD in the prediction models.</p><p id="p-30">To estimate the prediction accuracy for the case where pedigree or genomic relationship information is not available, but phenotypic data for GY is available on the training set and phenotypic data for secondary traits is available on all individuals, we fit the same set of within-environment multivariate prediction models as before, except with <strong>y<sub>1</sub></strong> and <strong>y</strong><sub>n</sub> equal to the vectors of BLUEs of the individuals for traits 1 and <em>n</em>, respectively, estimated on a per-replicate basis. To correct for the replicate effect, a random effect of replicate nested within trait was included in the multivariate prediction model, and individuals were assumed unrelated with <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-62.gif"/></noscript></span></span>, where <strong>I</strong> is an identity matrix. The predictions resulting from this model were of total genetic values, whereas the predictions from pedigree and genomic selection models were of breeding values.</p></div><div id="sec-9" class="subsection"><h3>Factors affecting accuracy gained from using secondary traits</h3><p id="p-31">All genomic and pedigree prediction model accuracies estimated from cross validation within environment were analyzed to assess four factors affecting the accuracy gained from using secondary traits in pedigree and genomic prediction: i. the relationship matrix used in prediction modeling, ii. the environment, iii. the mean magnitude of the genetic correlation,<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-63.gif"/></noscript></span></span>, between secondary traits and GY, and vi. the mean square root of the heritability of secondary traits, <em><span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-64.gif"/></noscript></span></span></em>, on either a line mean or single plot basis, depending upon whether secondary traits were replicated or not replicated in the prediction model. To estimate the effects and significance of these factors, we fit the linear regression model<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-65.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <em>y<sub>i</sub></em> is the multivariate accuracy minus the univariate accuracy for the corresponding univariate model, µ is the mean, <em>r<sub>i</sub></em> is the effect of relationship matrix used in the prediction model, <em>m<sub>ij</sub></em> is the effect of environment, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-66.gif"/></noscript></span></span> is the effect of <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-67.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-68.gif"/></noscript></span></span>is the effect of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-69.gif"/></noscript></span></span> between secondary traits and GY. The reference levels for relationship matrix and environment were pedigree and optimal, respectively.</p></div><div id="sec-10" class="subsection"><h3>Software</h3><p id="p-32">For high-throughput phenotyping, the software ArcMap (ESRI, Redlands, CA) was used to convert images to tabular data, and for manual georeferencing. ImapQ (Alava Ingenieros, Madrid, Spain) was used to correct the CT data with a linear calibration. Autopano Giga (Kolor SARL, France) was used to construct mosaics of multiple images. ENVI software (Excelis VIS, Boulder, CO) was used to convert pixel values to Celsius degrees. HyproQ (Alava Ingenieros, Madrid, Spain) was used to process the hyperspectral data.</p><p id="p-33">The coefficient of parentage was estimated using the ‘browse’ application in the International Crop Information System software package. All other analyses were done in the R programming language and software environment (<a href="http://www.r-project.org">http://www.r-project.org</a>). The package ‘ASReml-R’ (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gilmour <em>et al.</em> 2009</a>) was used to fit univariate mixed models for square root of heritability and repeatability estimation, and to fit multivariate prediction models. The package ‘rrBLUP’ (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Endelman 2011</a>) was used to calculate G. The R package ‘EMMREML’ (<a href="http://cran.r-project.org/package=EMMREML">http://cran.r-project.org/package=EMMREML</a>) was used to fit univaraite genomic prediction models. The regression model assessing factors affecting accuracy was fit using the R package ‘lm’.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-34">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the main body and supplemental information of the article.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Square roots of broad sense heritabilities and genetic correlations</h3><p id="p-35">Values of <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-70.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-71.gif"/></noscript></span></span>(<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) were generally high. Values of <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-72.gif"/></noscript></span></span>were on average 10% higher than their corresponding <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-73.gif"/></noscript></span></span>values. On average, <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-74.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-75.gif"/></noscript></span></span>values were 13% higher when DTHD was not corrected for. Values of <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-76.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-77.gif"/></noscript></span></span>were highest in early heat, and lowest in severe drought.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469055/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469055/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Square root of the broad sense heritabilities on a line mean basis, and on a single plot basis</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">In general, genetic correlations estimated with <strong>A</strong> and <strong>G</strong> were similar; thus, the genetic correlations reported (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032888/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032888/-/DC1/FigureS2.pdf">Figure S2</a>) are an average across estimates using A and G. Correlations ≥|0.09| are significant at the 0.05 level of significance. Genetic correlations between GY and secondary traits (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) ranged in magnitude according to the environment and correction for DTHD. Both with and without correcting for DTHD, genetic correlations were largest in magnitude in early heat and late heat. Secondary trait genetic correlations with GY also ranged in direction depending on the environment. For example, genetic correlation between GNDVI and GY was negative in severe drought, and positive in optimal. Genetic correlations between CT and GY were consistently negative in all environments. Correcting for DTHD almost always reduced the genetic correlations between secondary traits and GY, except for in late heat and optimal. The genetic correlations between GY and DTHD in optimal, drought, severe drought, late heat, and early heat were 0.17, –0.49, –0.71, –0.15, and 0.72, respectively. Genetic correlations between secondary traits varied depending upon the environment (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032888/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032888/-/DC1/FigureS2.pdf">Figure S2</a>). Drought and severe drought showed a similar pattern of genetic correlations, which was different from that of optimal, late heat, and early heat.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469145/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469145/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Genetic correlations between secondary traits and grain yield</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Effect of secondary traits on accuracy within environment</h3><p id="p-39">Multivariate pedigree and genomic prediction models that incorporated secondary trait data on training and test sets were consistently, and often substantially, more accurate than their corresponding univariate prediction models for GY within environment (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), demonstrating that secondary traits from HTP can improve pedigree and genomic prediction accuracy. No improvement in accuracy was observed when secondary traits were observed only on the training set (data not shown). The improvement in accuracy due to secondary traits depended on the replication of secondary traits in the test set, and whether we corrected for DTHD.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2799/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Univariate and multivariate prediction accuracies within environment, with and without correcting for days to heading. Within environment prediction accuracies from models using pedigree (A) and genomic (G) relationship are shown for each of the five environments, and for the average accuracy across all environments. Yellow, accuracies from univariate models (UV); red, accuracies from multivariate models (MV) where secondary trait data were from one replicate; blue, accuracies from MV models where secondary trait data were from three replicates. The first row of bar plots shows accuracies without correcting for days to heading (uncorrected), and the second row of bar plots shows accuracies with correcting for days to heading (corrected). SE is shown with error bars, and accuracy values are printed above each bar." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1215542205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Univariate and multivariate prediction accuracies within environment, with and without correcting for days to heading. Within environment prediction accuracies from models using pedigree (&lt;strong&gt;A&lt;/strong&gt;) and genomic (&lt;strong&gt;G&lt;/strong&gt;) relationship are shown for each of the five environments, and for the average accuracy across all environments. Yellow, accuracies from univariate models (UV); red, accuracies from multivariate models (MV) where secondary trait data were from one replicate; blue, accuracies from MV models where secondary trait data were from three replicates. The first row of bar plots shows accuracies without correcting for days to heading (uncorrected), and the second row of bar plots shows accuracies with correcting for days to heading (corrected). SE is shown with error bars, and accuracy values are printed above each bar.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2799/F2.medium.gif" width="440" height="293"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/9/2799/F2.medium.gif" width="440" height="293"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2799/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2799/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469159" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-40" class="first-child">Univariate and multivariate prediction accuracies within environment, with and without correcting for days to heading. Within environment prediction accuracies from models using pedigree (<strong>A</strong>) and genomic (<strong>G</strong>) relationship are shown for each of the five environments, and for the average accuracy across all environments. Yellow, accuracies from univariate models (UV); red, accuracies from multivariate models (MV) where secondary trait data were from one replicate; blue, accuracies from MV models where secondary trait data were from three replicates. The first row of bar plots shows accuracies without correcting for days to heading (uncorrected), and the second row of bar plots shows accuracies with correcting for days to heading (corrected). SE is shown with error bars, and accuracy values are printed above each bar.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">On average, without correcting for DTHD, secondary traits measured on all three replicates in the test set led to a 67% improvement in pedigree prediction accuracy, and an 85% improvement in genomic prediction accuracy. When we used only one replicate of secondary trait data on the test set, secondary traits lead to a 56% increase in pedigree prediction accuracy, and a 70% increase in genomic prediction accuracy.</p><p id="p-42">When correcting for DTHD, on average, secondary traits improved pedigree prediction accuracy 37%, and genomic prediction accuracy 48%, when secondary trait data on the test set was replicated. When only one replicate of secondary trait data were used on the test set, secondary traits increased pedigree prediction accuracy 17%, and genomic prediction accuracy 24%.</p><p id="p-43">Overall, we observed consistent trends in accuracy. Multivariate prediction models with replicated secondary trait data on the test set were most accurate, followed by multivariate prediction models with unreplicated secondary trait data on the test set, and univariate prediction models. Also, secondary traits had a slightly larger impact on accuracy in the genomic prediction models compared to the pedigree prediction models, and correcting for DTHD consistently reduced accuracies and led to smaller differences in accuracy between the multivariate and univariate models. The largest impact of correcting for DTHD was observed in drought. In drought, without correcting for DTHD, secondary traits lead to the largest percent increase in accuracy compared to all other environments. In contrast, when correcting for DTHD in drought, secondary traits led to no improvement or a decrease in accuracy. The smallest impact of correcting for DTHD was observed in late heat, where the multivariate and univariate prediction accuracies were largely unaffected.</p></div><div id="sec-15" class="subsection"><h3>Effect of secondary traits on across-environment accuracy</h3><p id="p-44">Without correcting for DTHD, across-environment multivariate pedigree and genomic prediction models incorporating secondary trait data on training and test sets were consistently, and substantially, more accurate than their corresponding univariate prediction models (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). On average, secondary traits improved pedigree prediction accuracy 114%, and genomic prediction accuracy 160%, when secondary trait data on the test set was replicated. When only one replicate of secondary trait data on the test set was used, secondary traits improved pedigree prediction accuracy 71%, and genomic prediction accuracy 107%.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2799/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Univariate and multivariate prediction accuracies across-environment, with and without correcting for days to heading. Across-environment prediction accuracies from models using pedigree (A) and genomic (G) relationship are shown for each of the five environments, and for the average accuracy across all environments. For each environment, the training set used in prediction contained data from all environments except the environment of interest. Yellow, accuracies from univariate models (UV); red, accuracies from multivariate models (MV) where secondary trait data were from one replicate; blue, accuracies from MV models where secondary trait data were from three replicates. The first row of bar plots shows accuracies without correcting for days to heading (uncorrected), and the second row of bar plots shows accuracies with correcting for days to heading (corrected). SE is shown with error bars, and accuracy values are printed above each bar." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1215542205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Univariate and multivariate prediction accuracies across-environment, with and without correcting for days to heading. Across-environment prediction accuracies from models using pedigree (&lt;strong&gt;A&lt;/strong&gt;) and genomic (&lt;strong&gt;G&lt;/strong&gt;) relationship are shown for each of the five environments, and for the average accuracy across all environments. For each environment, the training set used in prediction contained data from all environments except the environment of interest. Yellow, accuracies from univariate models (UV); red, accuracies from multivariate models (MV) where secondary trait data were from one replicate; blue, accuracies from MV models where secondary trait data were from three replicates. The first row of bar plots shows accuracies without correcting for days to heading (uncorrected), and the second row of bar plots shows accuracies with correcting for days to heading (corrected). SE is shown with error bars, and accuracy values are printed above each bar.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2799/F3.medium.gif" width="440" height="293"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/9/2799/F3.medium.gif" width="440" height="293"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2799/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2799/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469061" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">Univariate and multivariate prediction accuracies across-environment, with and without correcting for days to heading. Across-environment prediction accuracies from models using pedigree (<strong>A</strong>) and genomic (<strong>G</strong>) relationship are shown for each of the five environments, and for the average accuracy across all environments. For each environment, the training set used in prediction contained data from all environments except the environment of interest. Yellow, accuracies from univariate models (UV); red, accuracies from multivariate models (MV) where secondary trait data were from one replicate; blue, accuracies from MV models where secondary trait data were from three replicates. The first row of bar plots shows accuracies without correcting for days to heading (uncorrected), and the second row of bar plots shows accuracies with correcting for days to heading (corrected). SE is shown with error bars, and accuracy values are printed above each bar.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">When correcting for DTHD, across-environment multivariate pedigree and genomic prediction models were frequently about as accurate as, or less accurate than, their corresponding univariate prediction models (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). On average, when secondary trait data on the test set was replicated, secondary traits improved pedigree prediction accuracy 13%, and genomic prediction accuracy 22%. When secondary trait data on the test set was not replicated, on average secondary traits did not increase pedigree prediction accuracy, and increased genomic prediction accuracy 4%.</p><p id="p-47">The overall trends for across-environment prediction were that multivariate models were more accurate than univariate models only without correcting for DTHD, secondary traits lead to greater accuracies when secondary trait data were replicated <em>vs.</em> not replicated in the test set, and correcting for DTHD increased the univariate prediction accuracies and reduced the gain in accuracy from using secondary traits.</p></div><div id="sec-16" class="subsection"><h3>Prediction accuracies from models assuming lines were unrelated</h3><p id="p-48">We found that, on average, when predictor trait data were not replicated on the test set, and when correcting for DTHD, multivariate prediction models that assumed lines were unrelated were either equally accurate or slightly less accurate than genomic and pedigree predictions incorporating secondary traits (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032888/-/DC1/TableS1.pdf">Table S1</a>). When secondary trait data were replicated on both training and tests sets, and prediction models did not correct for DTHD, multivariate prediction models that assumed lines were unrelated were actually slightly more accurate on average compared to the multivariate pedigree and genomic prediction models. This indicates that, without estimates of relationship, secondary trait data can be useful for predicting total genetic values of GY.</p></div><div id="sec-17" class="subsection"><h3>Factors affecting accuracy gained from using secondary traits</h3><p id="p-49">The regression model that we fit to examine factors affecting the accuracy gained from using secondary traits in pedigree and genomic prediction was significant (<em>P</em> &lt;&lt; 0.01) with an adjusted <em>r<sup>2</sup></em> = 0.78. Based on this model, we found that the gain in accuracy from using secondary traits was significantly associated with a) <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-78.gif"/></noscript></span></span> between secondary traits and GY (<em>P</em> &lt; 0.05), and b) <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-79.gif"/></noscript></span></span> of secondary traits (<em>P</em> &lt;&lt; 0.01) (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). We also found that compared to the optimal environment, the accuracy gained from using secondary traits was significantly greater in late heat, and early heat (<em>P</em> &lt;&lt; 0.01).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469197/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469197/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Regression model coefficients explaining the accuracy gained from secondary traits</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-53">This study found that, in wheat, GNDVI, RNDVI, and CT measured across the crop growth cycle using an aerial HTP platform can substantially improve genomic and pedigree prediction model accuracies for GY when observed on training and test sets and used as secondary traits in multivariate models. Our prediction-model-based approach is an improvement upon conventional indirect selection based on secondary traits for two reasons. First, by using a prediction model, we take advantage of secondary trait data on the selection candidates <em>per se</em>, and GY data from selection candidates’ relatives. Second, by using a model training set evaluated for all traits in the target set of environments, secondary traits are appropriately weighted, depending in part upon the magnitude and the direction of their genetic correlations with GY. Because a quality control procedure was used prior to prediction modeling, an important assumption of this study is that there are no major technical errors in the HTP data collection that would give rise to near zero repeatability for trait measurements. We expect this assumption to hold true as HTP data collection and imaging processing pipelines continue to improve, minimizing technical errors.</p><p id="p-54">There was no improvement in accuracy when secondary traits were observed on the training set only, which is consistent with a similar study that evaluated secondary traits for feed intake in cattle (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Pszczola <em>et al.</em> 2013</a>). This may have been due to the relatively high heritabilities of GY in this study. Based on simulation, including secondary traits only on the training set can improve accuracy when the heritability of the trait of interest is low, and the heritabilities of the secondary traits are high (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Jia and Jannink 2012</a>). Thus, in the current study, heritabilities of GY were sufficiently high to make including secondary traits only on the training set ineffective for improving accuracy.</p><p id="p-55">Due to genotype-by-environment interaction, within-environment prediction accuracies were always higher than across-environment prediction accuracies. Thus, comparing the percent increases in accuracy due to secondary traits in across- <em>vs.</em> within-environment prediction may be misleading. Based on percent increase in accuracy, without correcting for DTHD, secondary traits appeared to be much more beneficial for prediction across environment than for prediction within environment, when in fact the numerical increases in accuracy due to secondary traits in across-environment prediction were lower than those of within-environment prediction.</p><p id="p-56">Correcting for DTHD improved univariate prediction accuracies across-environment, which suggests that the genotype-by-environment variance for GY corrected for DTHD is lower than that of uncorrected GY. For both within- and across-environment prediction, correcting for DTHD reduced the genetic correlations between GY and the secondary traits, which in turn reduced the accuracy gained from including secondary trait data on the test set. In plant breeding, GY is sometimes corrected for DTHD or another phenological trait in order to avoid indirect response to selection for that trait. To take full advantage of secondary trait data while avoiding indirect selection on a phenological trait, a better approach may be to include data on the phenological trait in a multivariate prediction model along with any available secondary traits, and then use the multivariate BLUPs to calculate a selection index with GY and the phenological trait weighted appropriately.</p><p id="p-57">If pedigree or genomic relationship information are not available, total genetic value for GY can be predicted using secondary trait data, assuming there is a population of lines phenotyped for both GY and secondary traits across multiple replicates that can be used to train a multivariate prediction model. Prediction accuracies for GY genetic value estimated in this way were frequently very similar to prediction accuracies for GY breeding values estimated using multivariate genomic and pedigree prediction models. If we assume that all the genetic variance is additive, this suggests that multivariate pedigree and genomic prediction accuracies were often driven primarily by secondary trait information on the individuals <em>per se</em>. However, to ensure the best estimates of breeding values, utilizing pedigree or genomic relationship information is recommended.</p><p id="p-58">The accuracy gained from using secondary traits was associated with <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-84.gif"/></noscript></span></span> between secondary traits and GY, and <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2799/embed/mml-math-85.gif"/></noscript></span></span> of secondary traits, which is in agreement with selection index theory. The significance of the effects of late heat and early heat on the accuracy gained from secondary traits is difficult to explain. According to <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Schaeffer (1984)</a>, the greater the absolute difference between the residual and genetic correlations between traits, the greater the percent reduction in the prediction error variance, and therefore greater accuracy, in multivariate BLUP. However, after examining the absolute differences between the residual and genetic correlations in each environment, we still could not explain the significant environment effects. This indicates that the gain in accuracy due to a set of secondary traits may be difficult to predict based only on heritabilities and correlations, and cross-validation in the environments of interest will be required to assess the gain in accuracy from using secondary traits in pedigree and genomic selection.</p><div id="sec-19" class="subsection"><h3>Conclusion</h3><p id="p-59">We have shown that, in wheat, GY secondary traits measured using an aerial HTP platform can be used to increase pedigree and genomic prediction model accuracies for GY when observed on both the model training and test sets. These prediction models could be useful for making selections in generations where GY cannot be measured accurately but secondary traits can. In crop breeding, there are instances where selection candidates can be phenotyped for some traits, but, due to insufficient seed, high cost of phenotyping, severe weather events, and/or long juvenile phase duration, not all traits of economic importance can be phenotyped. In these cases, traits correlated with the economically important traits could be used in pedigree and genomic prediction models to improve selection accuracy without delaying selection. This approach appears promising in wheat and should be investigated in other crops especially as HTP become more accessible.</p><p id="p-60">Additional research is needed before secondary traits from HTP can be used routinely in wheat breeding to increase pedigree and genomic selection accuracy for GY when GY cannot be measured directly on the selection candidates. First, in this study, we used a simple repeatability model for GNDVI, RNDVI, and CT measurements within a growth stage; however, time series models for measurements of these traits taken across time need to be compared to identify the best model for each trait. This could improve the phenotypic selection accuracy of GNDVI, RNDVI, and CT, potentially leading to better prediction accuracies when these traits are used in pedigree and genomic selection. To help address this need, we are currently working on evaluating repeatability, multivariate, and random regression models for GNDVI, RNDVI, and CT. Second, this study used secondary trait data from large plot sizes suitable for measuring GY, but, for selection, it will be useful to use secondary traits from plots that are of small size due to limited seed availability. Thus, the utility of secondary traits measured using HTP in pedigree and genomic prediction models need to be assessed for the case where the test set individuals are sown in a small plots. Lastly, HTP data analysis pipelines that lead to better data quality and faster speed of processing should continue to be developed.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-61">This research was supported by funds from the CGIAR research program on wheat, “Genomic Selection: The next frontier for rapid gains in maize and wheat improvement” funded by the Bill and Melinda Gates Foundation (BMFG), “Durable Rust Resistance in Wheat” funded by BMFG and the United Kingdom Department for International Development (DIFID), the “Cereal Systems Initiative for South Asia” funded by BMFG and the United States Agency for International Development (USAID), the USAID “Feed the Future Initiative” (USAID Cooperative Agreement No. AID-OAA-A-13-0005), and the National Science Foundation (IOS-1238187).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-62">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032888/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032888/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-63"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-06-06"><span class="received-label">Received </span>June 6, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-06-29"><span class="accepted-label">Accepted </span>June 29, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Rutkoski <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.1" data-doi="10.2135/cropsci2005.0211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Babar</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Ginkel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klatt</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raun</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Spectral reflectance to estimate genetic variation for in-season biomass, leaf chlorophyll, and canopy temperature in wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1046</span>–<span class="cit-lpage">1057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBabar%26rft.auinit1%253DM.%2BA.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D1046%26rft.epage%253D1057%26rft.atitle%253DSpectral%2BReflectance%2Bto%2BEstimate%2BGenetic%2BVariation%2Bfor%2BIn-Season%2BBiomass%252C%2BLeaf%2BChlorophyll%252C%2Band%2BCanopy%2BTemperature%2Bin%2BWheat%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.0211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.0211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000237375900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.2" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genome-wide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.3" data-doi="10.1186/1297-9686-43-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D43%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DAccuracy%2Bof%2Bmulti-trait%2Bgenomic%2Bselection%2Busing%2Bdifferent%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-26%26rft_id%253Dinfo%253Apmid%252F21729282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729282&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2799.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2799.4"><div class="cit-metadata"><cite>Elshire, R. J., J. C. Glaubitz, Q. Sun, J. A. Poland, K. Kawamoto <em>et al.</em>, 2011 A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species. PLoS One <span class="cit-vol">6</span>: e19379.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.5" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Shrinkage estimation of the realized relationship matrix. G3 Genes Genome.</span> <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D2%26rft.spage%253D1405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-other" id="cit-6.9.2799.7"><div class="cit-metadata unstructured">Gilmour A. R., B. J. Gogel, B. R. Cullis, and R. Thompson, 2009 ASReml user guide release 3.0. VSN International, Hemel Hempstead, UK</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.8" data-doi="10.1080/014311697217558"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gitelson</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merzlyak</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Remote estimation of chlorophyll content in higher plant leaves.</span> <abbr class="cit-jnl-abbrev">Int. J. Remote Sens.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2691</span>–<span class="cit-lpage">2697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BRemote%2BSens.%26rft.volume%253D18%26rft.spage%253D2691%26rft_id%253Dinfo%253Adoi%252F10.1080%252F014311697217558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/014311697217558&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.9" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: A high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2799.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.10" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.11"><div class="cit-metadata"><cite>International Wheat Genome Sequencing Consortium (IWGSC), <span class="cit-pub-date">2014</span> A chromosome-based draft sequence of the hexaploid bread wheat (<em>Triticum aestivum</em>) genome. Science <span class="cit-vol">345</span>: 1251788.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.12" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-trait genomic selection methods increase genetic value prediction accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjc5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.13" data-doi="10.1016/B978-0-12-385531-2.00002-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in plant breeding: knowledge and prospects.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D110%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-385531-2.00002-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-385531-2.00002-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.14" data-doi="10.2135/cropsci2014.10.0709"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kehel</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrique</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Characterization of heat- and drought-stress tolerance in high-yielding spring wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1552</span>–<span class="cit-lpage">1562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1552%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.10.0709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.10.0709&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.15" data-doi="10.1016/j.fcr.2014.07.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pask</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manès</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatrath</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A wheat phenotyping network to incorporate physiological traits for climate change in south asia.</span> <abbr class="cit-jnl-abbrev">F. Crop. Res.</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF.%2BCrop.%2BRes.%26rft.volume%253D168%26rft.spage%253D156%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2014.07.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2014.07.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.16" data-doi="10.1007/s00122-010-1351-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathews</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivares-Villegas</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Heat and drought adaptive QTL in a wheat population designed to minimize confounding agronomic effects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">1001</span>–<span class="cit-lpage">1021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DPinto%26rft.auinit1%253DR.%2BS.%26rft.volume%253D121%26rft.issue%253D6%26rft.spage%253D1001%26rft.epage%253D1021%26rft.atitle%253DHeat%2Band%2Bdrought%2Badaptive%2BQTL%2Bin%2Ba%2Bwheat%2Bpopulation%2Bdesigned%2Bto%2Bminimize%2Bconfounding%2Bagronomic%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1351-4%26rft_id%253Dinfo%253Apmid%252F20523964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1351-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20523964&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2799.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281794100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.17" data-doi="10.1071/AN11098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Designing dairy cattle breeding schemes under genomic selection : a review of international research.</span> <abbr class="cit-jnl-abbrev">Anim. Prod. Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%2BSci.%26rft.volume%253D52%26rft.spage%253D107%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAN11098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AN11098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000301064800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.18" data-doi="10.1017/S175173111300150X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pszczola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Haas</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strabel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Eff-ect of predictor traits on accuracy of genomic breeding values for feed intake based on a limited cow reference population.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%26rft.volume%253D7%26rft.spage%253D1759%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS175173111300150X%26rft_id%253Dinfo%253Apmid%252F23915541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S175173111300150X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23915541&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2799.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.19" data-doi="10.1016/j.pbi.2016.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Physiological breeding.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">162</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D31%26rft.spage%253D162%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2016.04.005%26rft_id%253Dinfo%253Apmid%252F27161822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2016.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27161822&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2799.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.20" data-doi="10.3168/jds.S0022-0302(84)81479-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Sire and cow evaluation under multiple trait models.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DSchaeffer%26rft.auinit1%253DL.%2BR.%26rft.volume%253D67%26rft.issue%253D7%26rft.spage%253D1567%26rft.epage%253D1580%26rft.atitle%253DSire%2Band%2BCow%2BEvaluation%2BUnder%2BMultiple%2BTrait%2BModels%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252884%252981479-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(84)81479-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.21" data-doi="10.2134/agronj1990.00021962008200030037x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiegand</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span><span class="cit-comment">a</span> <span class="cit-article-title">Use of spectral vegetation indices to infer leaf area, evapotranspiration and yield: I.</span> <abbr class="cit-jnl-abbrev">Rationale. Agron. J.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRationale.%2BAgron.%2BJ.%26rft.volume%253D82%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1990.00021962008200030037x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1990.00021962008200030037x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.22" data-doi="10.2134/agronj1990.00021962008200030038x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiegand</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span><span class="cit-comment">b</span> <span class="cit-article-title">Use of spectral vegetation indices to infer leaf area, evapotranspiration and yield: II. Results.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">630</span>–<span class="cit-lpage">636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DWiegand%26rft.auinit1%253DC.%2BL.%26rft.volume%253D82%26rft.issue%253D3%26rft.spage%253D630%26rft.epage%253D636%26rft.atitle%253DUse%2Bof%2BSpectral%2BVegetation%2BIndices%2Bto%2BInfer%2BLeaf%2BArea%252C%2BEvapotranspiration%2Band%2BYield%253A%2BII.%2BResults%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1990.00021962008200030038x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1990.00021962008200030038x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2799.23" data-doi="10.1007/s00122-008-0715-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">815</span>–<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWong%26rft.auinit1%253DC.%2BK.%26rft.volume%253D116%26rft.issue%253D6%26rft.spage%253D815%26rft.epage%253D824%26rft.atitle%253DGenomewide%2Bselection%2Bin%2Boil%2Bpalm%253A%2Bincreasing%2Bselection%2Bgain%2Bper%2Bunit%2Btime%2Band%2Bcost%2Bwith%2Bsmall%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0715-5%26rft_id%253Dinfo%253Apmid%252F18219476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0715-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18219476&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2799.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254262600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/9/2799.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2791" title="Whole Genome Comparison of Thermus sp. NMX2.A1 Reveals Principal Carbon Metabolism Differences with Closest Relation Thermus scotoductus SA-01" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2809" title="A Split-Ubiquitin Based Strategy Selecting for Protein Complex-Interfering Mutations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/9"><span>Volume 6 Issue 9, September 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/9.cover-source.jpg?itok=nund1hRf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2799.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5jg5IRzx4cXUNIdonjgigY2m_iK5RN_S3d2j8Hotc8o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/9/2799" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/9/2799" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/9/2799" class="active" data-icon-position="" data-hide-link-title="0">Canopy Temperature and Vegetation Indices from High-Throughput Phenotyping Improve Accuracy of Pedigree and Genomic Selection for Grain Yield in Wheat</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460286" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2bZGWU1aJuhUsy07PHpa3Fj7G5ashoonVKLqhZXYhF8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2799.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/9/2799" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-nlrPB_76DZ9JD0aY5zTnFkatp4jJk7JBHWQHyqaynN8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/9/2799" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2799.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460286" id="node460286--2" data-pisa="ggg;6/9/2799" data-pisa-master="ggg;g3.116.032888" data-apath="/ggg/6/9/2799.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Canopy Temperature and Vegetation Indices from High-Throughput Phenotyping Improve Accuracy of Pedigree and Genomic Selection for Grain Yield in Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jessica Rutkoski</span>, <span class="highwire-citation-author" data-delta="1">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="2">Suchismita Mondal</span>, <span class="highwire-citation-author" data-delta="3">Enrique Autrique</span>, <span class="highwire-citation-author" data-delta="4">Lorena González Pérez</span>, <span class="highwire-citation-author" data-delta="5">José Crossa</span>, <span class="highwire-citation-author" data-delta="6">Matthew Reynolds</span> and <span class="highwire-citation-author" data-delta="7">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2799-2808;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032888 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460286--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Programs, College of Agriculture and Life Sciences, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding and Genetics Section, School of Integrated Plant Sciences, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jer263@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">jer263@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Pathology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Suchismita Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suchismita%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuchismita%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Enrique Autrique </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Enrique%2BAutrique%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Autrique%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEnrique%2BAutrique%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lorena González Pérez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorena%2BGonz%C3%A1lez%2BP%C3%A9rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALorena%2BGonz%25C3%25A1lez%2BP%25C3%25A9rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Matthew Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/9/2799" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/9/2799&amp;title=Canopy%20Temperature%20and%20Vegetation%20Indices%20from%20High-Throughput%20Phenotyping%20Improve%20Accuracy%20of%20Pedigree%20and%20Genomic%20Selection%20for%20Grain%20Yield%20in%20Wheat" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/9/2799&amp;title=Canopy%20Temperature%20and%20Vegetation%20Indices%20from%20High-Throughput%20Phenotyping%20Improve%20Accuracy%20of%20Pedigree%20and%20Genomic%20Selection%20for%20Grain%20Yield%20in%20Wheat" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/9/2799&amp;title=Canopy%20Temperature%20and%20Vegetation%20Indices%20from%20High-Throughput%20Phenotyping%20Improve%20Accuracy%20of%20Pedigree%20and%20Genomic%20Selection%20for%20Grain%20Yield%20in%20Wheat" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/9/2799&amp;text=Canopy%20Temperature%20and%20Vegetation%20Indices%20from%20High-Throughput%20Phenotyping%20Improve%20Accuracy%20of%20Pedigree%20and%20Genomic%20Selection%20for%20Grain%20Yield%20in%20Wheat" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/9/2799&amp;title=Canopy%20Temperature%20and%20Vegetation%20Indices%20from%20High-Throughput%20Phenotyping%20Improve%20Accuracy%20of%20Pedigree%20and%20Genomic%20Selection%20for%20Grain%20Yield%20in%20Wheat" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/9/2799&amp;t=Canopy%20Temperature%20and%20Vegetation%20Indices%20from%20High-Throughput%20Phenotyping%20Improve%20Accuracy%20of%20Pedigree%20and%20Genomic%20Selection%20for%20Grain%20Yield%20in%20Wheat" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/9/2799&amp;title=Canopy%20Temperature%20and%20Vegetation%20Indices%20from%20High-Throughput%20Phenotyping%20Improve%20Accuracy%20of%20Pedigree%20and%20Genomic%20Selection%20for%20Grain%20Yield%20in%20Wheat" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/9/2799&amp;title=Canopy%20Temperature%20and%20Vegetation%20Indices%20from%20High-Throughput%20Phenotyping%20Improve%20Accuracy%20of%20Pedigree%20and%20Genomic%20Selection%20for%20Grain%20Yield%20in%20Wheat" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460286/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460286" id="node460286--3" data-pisa="ggg;6/9/2799" data-pisa-master="ggg;g3.116.032888" data-apath="/ggg/6/9/2799.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Canopy Temperature and Vegetation Indices from High-Throughput Phenotyping Improve Accuracy of Pedigree and Genomic Selection for Grain Yield in Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jessica Rutkoski</span>, <span class="highwire-citation-author" data-delta="1">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="2">Suchismita Mondal</span>, <span class="highwire-citation-author" data-delta="3">Enrique Autrique</span>, <span class="highwire-citation-author" data-delta="4">Lorena González Pérez</span>, <span class="highwire-citation-author" data-delta="5">José Crossa</span>, <span class="highwire-citation-author" data-delta="6">Matthew Reynolds</span> and <span class="highwire-citation-author" data-delta="7">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2799-2808;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032888 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460286--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Programs, College of Agriculture and Life Sciences, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding and Genetics Section, School of Integrated Plant Sciences, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jer263@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">jer263@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Pathology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Suchismita Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suchismita%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuchismita%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Enrique Autrique </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Enrique%2BAutrique%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Autrique%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEnrique%2BAutrique%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lorena González Pérez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorena%2BGonz%C3%A1lez%2BP%C3%A9rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALorena%2BGonz%25C3%25A1lez%2BP%25C3%25A9rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Matthew Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de Mexico, 06600, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2799&amp;rft.epage=2808&amp;rft.atitle=Canopy%20Temperature%20and%20Vegetation%20Indices%20from%20High-Throughput%20Phenotyping%20Improve%20Accuracy%20of%20Pedigree%20and%20Genomic%20Selection%20for%20Grain%20Yield%20in%20Wheat&amp;rft.volume=6&amp;rft.issue=9&amp;rft.date=2016-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Rutkoski%2C+Jessica&amp;rft.au=Poland%2C+Jesse&amp;rft.au=Mondal%2C+Suchismita&amp;rft.au=Autrique%2C+Enrique&amp;rft.au=P%C3%A9rez%2C+Lorena+Gonz%C3%A1lez&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Reynolds%2C+Matthew&amp;rft.au=Singh%2C+Ravi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460286/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460286/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460286/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460286/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460286/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460286/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460286/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460286/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460286/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460286/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460286/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460286/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/9/2799" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/9/2799.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/9/2799.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/9/2799.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/9/2799.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
