<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/9/2881" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/9/2881.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/9/2881.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/9/2881.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/9/2881.atom" />
<meta name="HW.pisa" content="ggg;6/9/2881" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants" />
<meta name="DC.Identifier" content="10.1534/g3.116.030338" />
<meta name="DC.Date" content="2016-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Nelson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Transcriptomic analyses from across eukaryotes indicate that most of the genome is transcribed at some point in the developmental trajectory of an organism. One class of these transcripts is termed long intergenic noncoding RNAs (lincRNAs). Recently, attention has focused on understanding the evolutionary dynamics of lincRNAs, particularly their conservation within genomes. Here, we take a comparative genomic and phylogenetic approach to uncover factors influencing lincRNA emergence and persistence in the plant family Brassicaceae, to which Arabidopsis thaliana belongs. We searched 10 genomes across the family for evidence of &gt; 5000 lincRNA loci from A. thaliana . From loci conserved in the genomes of multiple species, we built alignments and inferred phylogeny. We then used gene tree/species tree reconciliation to examine the duplication history and timing of emergence of these loci. Emergence of lincRNA loci appears to be linked to local duplication events, but, surprisingly, not whole genome duplication events (WGD), or transposable elements. Interestingly, WGD events are associated with the loss of loci for species having undergone relatively recent polyploidy. Lastly, we identify 1180 loci of the 6480 previously annotated A. thaliana lincRNAs (18%) with elevated levels of conservation. These conserved lincRNAs show higher expression, and are enriched for stress-responsiveness and cis -regulatory motifs known as conserved noncoding sequences (CNSs). These data highlight potential functional pathways and suggest that CNSs may regulate neighboring genes at both the genomic and transcriptomic level. In sum, we provide insight into processes that may influence lincRNA diversification by providing an evolutionary context for previously annotated lincRNAs." />
<meta name="DC.Contributor" content="Andrew D. L. Nelson" />
<meta name="DC.Contributor" content="Evan S. Forsythe" />
<meta name="DC.Contributor" content="Upendra K. Devisetty" />
<meta name="DC.Contributor" content="David S. Clausen" />
<meta name="DC.Contributor" content="Asher K. Haug-Batzell" />
<meta name="DC.Contributor" content="Ari M. R. Meldrum" />
<meta name="DC.Contributor" content="Michael R. Frank" />
<meta name="DC.Contributor" content="Eric Lyons" />
<meta name="DC.Contributor" content="Mark A. Beilstein" />
<meta name="article:published_time" content="2016-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Transcriptomic analyses from across eukaryotes indicate that most of the genome is transcribed at some point in the developmental trajectory of an organism. One class of these transcripts is termed long intergenic noncoding RNAs (lincRNAs). Recently, attention has focused on understanding the evolutionary dynamics of lincRNAs, particularly their conservation within genomes. Here, we take a comparative genomic and phylogenetic approach to uncover factors influencing lincRNA emergence and persistence in the plant family Brassicaceae, to which &lt;i&gt;Arabidopsis thaliana&lt;/i&gt; belongs. We searched 10 genomes across the family for evidence of &amp;gt;&amp;nbsp;5000 lincRNA loci from &lt;i&gt;A. thaliana&lt;/i&gt;. From loci conserved in the genomes of multiple species, we built alignments and inferred phylogeny. We then used gene tree/species tree reconciliation to examine the duplication history and timing of emergence of these loci. Emergence of lincRNA loci appears to be linked to local duplication events, but, surprisingly, not whole genome duplication events (WGD), or transposable elements. Interestingly, WGD events are associated with the loss of loci for species having undergone relatively recent polyploidy. Lastly, we identify 1180 loci of the 6480 previously annotated &lt;i&gt;A. thaliana&lt;/i&gt; lincRNAs (18%) with elevated levels of conservation. These conserved lincRNAs show higher expression, and are enriched for stress-responsiveness and &lt;i&gt;cis&lt;/i&gt;-regulatory motifs known as conserved noncoding sequences (CNSs). These data highlight potential functional pathways and suggest that CNSs may regulate neighboring genes at both the genomic and transcriptomic level. In sum, we provide insight into processes that may influence lincRNA diversification by providing an evolutionary context for previously annotated lincRNAs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/09/01" />
<meta name="citation_mjid" content="ggg;6/9/2881" />
<meta name="citation_id" content="6/9/2881" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/9/2881" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/9/2881.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/9/2881.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/9/2881.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.030338" />
<meta name="citation_pmid" content="27440919" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2881" />
<meta name="citation_lastpage" content="2891" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andrew D. L. Nelson" />
<meta name="citation_author_institution" content="School of Plant Sciences, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author_email" content="andrewnelson@email.arizona.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9896-1739" />
<meta name="citation_author" content="Evan S. Forsythe" />
<meta name="citation_author_institution" content="School of Plant Sciences, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author" content="Upendra K. Devisetty" />
<meta name="citation_author_institution" content="Department of Forest Ecosystems and Society, Oregon State University, Corvalis, Oregon 97331" />
<meta name="citation_author" content="David S. Clausen" />
<meta name="citation_author_institution" content="Department of Statistics, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Asher K. Haug-Batzell" />
<meta name="citation_author_institution" content="Genetics Graduate Interdisciplinary Group, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author" content="Ari M. R. Meldrum" />
<meta name="citation_author_institution" content="School of Plant Sciences, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author" content="Michael R. Frank" />
<meta name="citation_author_institution" content="School of Plant Sciences, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author" content="Eric Lyons" />
<meta name="citation_author_institution" content="School of Plant Sciences, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author_institution" content="Genetics Graduate Interdisciplinary Group, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author" content="Mark A. Beilstein" />
<meta name="citation_author_institution" content="School of Plant Sciences, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author_email" content="andrewnelson@email.arizona.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3392-1389" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. Adrian;citation_author=S. Farrona;citation_author=J. J. Reimer;citation_author=M. C. Albani;citation_author=G. Coupland;citation_title=cis-regulatory elements and chromatin state coordinately control temporal and spatial expression of FLOWERING LOCUS T in Arabidopsis.;citation_volume=22;citation_year=2010;citation_issue=5;citation_pmid=20472817;citation_doi=10.1105/tpc.110.074682" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=U. Arnason;citation_author=J. A. Adegoke;citation_author=A. Gullberg;citation_author=E. H. Harley;citation_author=A. Janke;citation_title=Mitogenomic relationships of placental mammals and molecular estimates of their divergences.;citation_pages=37-51;citation_volume=421;citation_year=2008;citation_issue=1–2;citation_pmid=18590805;citation_doi=10.1016/j.gene.2008.05.024" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=M. A. Beilstein;citation_author=I. A. Al-Shehbaz;citation_author=E. A. Kellogg;citation_title=Brassicaceae phylogeny and trichome evolution;citation_pages=607-619;citation_volume=93;citation_year=2006;citation_issue=4;citation_pmid=21646222;citation_doi=10.3732/ajb.93.4.607" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. A. Beilstein;citation_author=N. S. Nagalingum;citation_author=M. D. Clements;citation_author=S. R. Manchester;citation_author=S. Mathews;citation_title=Dated molecular phylogenies indicate a Miocene origin for Arabidopsis thaliana.;citation_volume=107;citation_year=2010;citation_issue=43;citation_pmid=20921408;citation_doi=10.1073/pnas.0909766107;citation_georef=2013040705" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=E. H. Blackburn;citation_author=K. Collins;citation_title=Telomerase: an RNP enzyme synthesizes DNA.;citation_volume=3;citation_year=2011;citation_issue=5;citation_pmid=20660025;citation_doi=10.1101/cshperspect.a003558" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=M. N. Cabili;citation_author=C. Trapnell;citation_author=L. Goff;citation_author=M. Koziol;citation_author=B. Tazon-Vega;citation_title=Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses.;citation_volume=25;citation_year=2011;citation_issue=18;citation_pmid=21890647;citation_doi=10.1101/gad.17446611" />
<meta name="citation_reference" content="citation_journal_title=Nature Comm.;citation_author=L. Castaings;citation_author=S. Bergonzi;citation_author=M C. Albani;citation_author=U. Kemi;citation_author=O. Savolainen;citation_author=G. Coupland;citation_title=Evolutionary conservation of cold-induced antisense RNAs of FLOWERING LOCUS C in Arabidopsis thaliana perennial relatives.;citation_pages=4457;citation_volume=5;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J.-L. Chen;citation_author=M. A. Blasco;citation_author=C. W. Greider;citation_title=Secondary structure of vertebrate telomerase RNA.;citation_pages=503-514;citation_volume=100;citation_year=2000;citation_issue=5;citation_pmid=10721988;citation_doi=10.1016/S0092-8674(00)80687-X" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. Cheng;citation_author=E. van den Bergh;citation_author=P. Zeng;citation_author=X. Zhong;citation_author=J. Xu;citation_title=The Tarenaya hassleriana genome provides insight into reproductive trait and genome evolution of crucifers.;citation_volume=25;citation_year=2013;citation_issue=8;citation_pmid=23983221;citation_doi=10.1105/tpc.113.113480" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=X. Dai;citation_author=P X. Zhao;citation_title=psRNATarget: a plant small RNA target analysis server.;citation_volume=39;citation_year=2011;citation_issue=Web Server issue;citation_pmid=21622958;citation_doi=10.1093/nar/gkr319" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=T. Derrien;citation_author=R. Johnson;citation_author=G. Bussotti;citation_author=A. Tanzer;citation_author=S. Djebali;citation_title=The GENCODE v7 catalog of human long noncoding RNAs: Analysis of their gene structure, evolution, and expression;citation_pages=1775-1789;citation_volume=22;citation_year=2012;citation_issue=9;citation_pmid=22955988;citation_doi=10.1101/gr.132159.111" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Djebali;citation_author=A. M. Carrie A Davis;citation_author=A. Dobin;citation_author=T. Lassmann;citation_author=A. Mortazavi;citation_title=Landscape of transcription in human cells.;citation_pages=101-108;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955620;citation_doi=10.1038/nature11233" />
<meta name="citation_reference" content="citation_journal_title=J. Comp. Biol.;citation_author=D. Durand;citation_author=B V. Halldórsson;citation_author=B. Vernot;citation_title=A hybrid micro-macroevolutionary approach to gene tree reconstruction.;citation_pages=320-35;citation_volume=13;citation_year=2006;citation_issue=2;citation_doi=10.1089/cmb.2006.13.320" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. Duret;citation_author=C. Chureau;citation_author=S. Samain;citation_author=J. Weissenbach;citation_author=P. Avner;citation_title=The xist RNA gene evolved in eutherians by pseudogenization of a protein-coding gene.;citation_pages=1653-1655;citation_volume=312;citation_year=2006;citation_issue=5780;citation_pmid=16778056;citation_doi=10.1126/science.1126316" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. A. Elisaphenko;citation_author=N. N. Kolesnikov;citation_author=A. I. Shevchenko;citation_author=I. B. Rogozin;citation_author=T. B. Nesterova;citation_title=A dual origin of the Xist gene from a protein-coding gene and a set of transposable elements.;citation_pages=e2521-e2521;citation_volume=3;citation_year=2008;citation_issue=6;citation_pmid=18575625;citation_doi=10.1371/journal.pone.0002521" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=M. Freeling;citation_author=S. Subramaniam;citation_title=Conserved noncoding sequences (CNSs) in higher plants.;citation_pages=126-132;citation_volume=12;citation_year=2009;citation_issue=2;citation_pmid=19249238;citation_doi=10.1016/j.pbi.2009.01.005" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=L. Goodstadt;citation_author=C. P. Ponting;citation_title=Phylogenetic reconstruction of orthology, paralogy, and conserved synteny for dog and human.;citation_pages=e133-e133;citation_volume=2;citation_year=2006;citation_issue=9;citation_pmid=17009864;citation_doi=10.1371/journal.pcbi.0020133" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=J. Gosai;citation_author=S. W. Foley;citation_author=D. Wang;citation_author=I. M. Silverman;citation_author=N. Selamoglu;citation_title=Global analysis of the RNA-protein interaction and RNA secondary structure landscapes of the Arabidopsis nucleus.;citation_pages=376-388;citation_volume=57;citation_year=2015;citation_issue=2;citation_pmid=25557549;citation_doi=10.1016/j.molcel.2014.12.004" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Med.;citation_author=T. Gutschner;citation_author=M. Hämmerle;citation_author=S. Diederichs;citation_title=MALAT1–a paradigm for long noncoding RNA function in cancer.;citation_pages=791-801;citation_volume=91;citation_year=2013;citation_issue=7;citation_pmid=23529762;citation_doi=10.1007/s00109-013-1028-y" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=J. C. Hall;citation_author=K. J. Sytsma;citation_author=H. H. Iltis;citation_title=Phylogeny of Capparaceae and Brassicaceae based on chloroplast sequence data;citation_pages=1826-1842;citation_volume=89;citation_year=2002;citation_issue=11;citation_pmid=21665611;citation_doi=10.3732/ajb.89.11.1826" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. B. Hansen;citation_author=T. I. Jensen;citation_author=B. H. Clausen;citation_author=J. B. Bramsen;citation_author=B. Finsen;citation_title=Natural RNA circles function as efficient microRNA sponges.;citation_pages=384-388;citation_volume=495;citation_year=2013;citation_issue=7441;citation_pmid=23446346;citation_doi=10.1038/nature11993" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Haudry;citation_author=A. E. Platts;citation_author=E. Vello;citation_author=D. R. Hoen;citation_author=M. Leclercq;citation_title=An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions.;citation_pages=891-898;citation_volume=45;citation_year=2013;citation_issue=8;citation_pmid=23817568;citation_doi=10.1038/ng.2684" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=H. Hezroni;citation_author=D. Koppstein;citation_author=M. G. Schwartz;citation_author=A. Avrutin;citation_author=D. P. Bartel;citation_title=Principles of long noncoding RNA evolution derived from direct comparison of transcriptomes in 17 species.;citation_pages=1110-1122;citation_volume=11;citation_year=2015;citation_issue=7;citation_pmid=25959816;citation_doi=10.1016/j.celrep.2015.04.023" />
<meta name="citation_reference" content="citation_title=Plant Genome Diversity Volume 2, Physical Structure, Behaviour and Evolution of Plant Genomes;citation_title=The incidence of polyploidy in natural plant populations: major patterns and evolutionary processes;citation_author=B. C. Husband;citation_author=S. J. Baldwin;citation_author=J. Suda;citation_pages=255-276;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Johnson;citation_author=I. Zaretskaya;citation_author=Y. Raytselis;citation_author=Y. Merezhuk;citation_author=S. McGinnis;citation_author=T L. Madden;citation_title=NCBI BLAST: a better web interface.;citation_volume=36;citation_year=2008;citation_issue=Web Server issue;citation_pmid=18440982;citation_doi=10.1093/nar/gkn201" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. Kagale;citation_author=S. J. Robinson;citation_author=J. Nixon;citation_author=R. Xiao;citation_author=T. Huebert;citation_title=Polyploid evolution of the Brassicaceae during the Cenozoic era.;citation_volume=26;citation_year=2014;citation_issue=7;citation_pmid=25035408;citation_doi=10.1105/tpc.114.126391" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Kapusta;citation_author=Z. Kronenberg;citation_author=V. J. Lynch;citation_author=X. Zhuo;citation_author=L. Ramsay;citation_title=Transposable elements are major contributors to the origin, diversification, and regulation of vertebrate long noncoding RNAs.;citation_pages=e1003470;citation_volume=9;citation_year=2013;citation_issue=4;citation_pmid=23637635;citation_doi=10.1371/journal.pgen.1003470" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Katoh;citation_author=D. M. Standley;citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability.;citation_volume=30;citation_year=2013;citation_issue=4;citation_pmid=23329690;citation_doi=10.1093/molbev/mst010" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. R. Kelley;citation_author=D. G. Hendrickson;citation_author=D. Tenen;citation_author=J. L. Rinn;citation_author=E. S. Lander;citation_title=Transposable elements modulate human RNA abundance and splicing via specific RNA-protein interactions.;citation_pages=537;citation_volume=15;citation_year=2014;citation_issue=12;citation_pmid=25572935;citation_doi=10.1186/s13059-014-0537-5" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=D. Koenig;citation_author=D. Weigel;citation_title=Beyond the thale: comparative genomics and genetics of Arabidopsis relatives.;citation_pages=285-298;citation_volume=16;citation_year=2015;citation_issue=5;citation_pmid=25854181;citation_doi=10.1038/nrg3883" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=M. Kretz;citation_author=D. E. Webster;citation_author=R. J. Flockhart;citation_author=C. S. Lee;citation_author=A. Zehnder;citation_title=Suppression of progenitor differentiation requires the long noncoding RNA ANCR.;citation_volume=26;citation_year=2012;citation_issue=4;citation_pmid=22302877;citation_doi=10.1101/gad.182121.111" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. T. Y. Lam;citation_author=H. Cho;citation_author=H. P. Lesch;citation_author=D. Gosselin;citation_author=S. Heinz;citation_title=Rev-Erbs repress macrophage gene expression by inhibiting enhancer-directed transcription.;citation_pages=511-515;citation_volume=498;citation_year=2013;citation_issue=7455;citation_pmid=23728303;citation_doi=10.1038/nature12209" />
<meta name="citation_reference" content="citation_journal_title=Trends Biochem. Sci.;citation_author=M. T. Y. Lam;citation_author=W. Li;citation_author=M. G. Rosenfeld;citation_author=C. K. Glass;citation_title=Enhancer RNAs and regulated transcriptional programs.;citation_pages=170-182;citation_volume=39;citation_year=2014;citation_issue=4;citation_pmid=24674738;citation_doi=10.1016/j.tibs.2014.02.007" />
<meta name="citation_reference" content="citation_journal_title=EMBO Rep.;citation_author=M. Lapidot;citation_author=Y. Pilpel;citation_title=Genome-wide natural antisense transcription: coupling its regulation to its different regulatory mechanisms.;citation_volume=7;citation_year=2006;citation_issue=12;citation_pmid=17139297;citation_doi=10.1038/sj.embor.7400857" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=L. Li;citation_author=S. R. Eichten;citation_author=R. Shimizu;citation_author=K. Petsch;citation_author=C.-T. Yeh;citation_title=Genome-wide discovery and characterization of maize long non-coding RNAs.;citation_pages=R40;citation_volume=15;citation_year=2014;citation_issue=2;citation_pmid=24576388;citation_doi=10.1186/gb-2014-15-2-r40" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. Liu;citation_author=C. Jung;citation_author=J. Xu;citation_author=H. Wang;citation_author=S. Deng;citation_title=Genome-wide analysis uncovers regulation of long intergenic noncoding RNAs in Arabidopsis.;citation_volume=24;citation_year=2012;citation_issue=11;citation_pmid=23136377;citation_doi=10.1105/tpc.112.102855" />
<meta name="citation_reference" content="citation_journal_abbrev=Algorithms Mol Biol;citation_author=R. Lorenz;citation_author=S. H. Bernhart;citation_author=C. Höner;citation_author=Z. Siederdissen;citation_author=H. Tafer;citation_title=ViennaRNA Package 2.0.;citation_pages=26-26;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=22115189;citation_doi=10.1186/1748-7188-6-26" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=E. Lyons;citation_author=B. Pedersen;citation_author=J. Kane;citation_author=M. Alam;citation_author=R. Ming;citation_title=Finding and comparing syntenic regions among Arabidopsis and the outgroups papaya, poplar, and grape: CoGe with Rosids.;citation_volume=148;citation_year=2008;citation_issue=4;citation_pmid=18952863;citation_doi=10.1104/pp.108.124867" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=S. Magallón;citation_author=S. Gómez-Acevedo;citation_author=L. L. Sánchez-Reyes;citation_author=T. Hernández-Hernández;citation_title=A metacalibrated time-tree documents the early rise of flowering plant phylogenetic diversity.;citation_pages=437-453;citation_volume=207;citation_year=2015;citation_issue=2;citation_pmid=25615647;citation_doi=10.1111/nph.13264" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=D. Managadze;citation_author=I. B. Rogozin;citation_author=D. Chernikova;citation_author=S. A. Shabalina;citation_author=E. V. Koonin;citation_title=Negative correlation between expression level and evolutionary rate of long intergenic noncoding RNAs.;citation_volume=3;citation_year=2011;citation_issue=1;citation_pmid=22071789;citation_doi=10.1093/gbe/evr116" />
<meta name="citation_reference" content="citation_journal_title=Ann. N. Y. Acad. Sci.;citation_author=G. D. Moghe;citation_author=S.-H. Shiu;citation_title=The causes and molecular consequences of polyploidy in flowering plants.;citation_pages=16-34;citation_volume=1320;citation_year=2014;citation_issue=July;citation_pmid=24903334;citation_doi=10.1111/nyas.12466" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=F. Murat;citation_author=Y. Van de Peer;citation_author=J. Salse;citation_title=Decoding plant and animal genome plasticity from differential paleo-evolutionary patterns and processes.;citation_volume=4;citation_year=2012;citation_issue=9;citation_pmid=22833223;citation_doi=10.1093/gbe/evs066" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. P. Nawrocki;citation_author=S. W. Burge;citation_author=A. Bateman;citation_author=J. Daub;citation_author=R. Y. Eberhardt;citation_title=Rfam 12.0: updates to the RNA families database.;citation_volume=43;citation_year=2015;citation_issue=Database issue;citation_pmid=25392425;citation_doi=10.1093/nar/gku1063" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Necsulea;citation_author=M. Soumillon;citation_author=M. Warnefors;citation_author=A. Liechti;citation_author=T. Daish;citation_title=The evolution of lncRNA repertoires and expression patterns in tetrapods.;citation_pages=635-640;citation_volume=505;citation_year=2014;citation_issue=7485;citation_pmid=24463510;citation_doi=10.1038/nature12943" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=A. D. L. Nelson;citation_author=D. E. Shippen;citation_title=Evolution of TERT-interacting lncRNAs: expanding the regulatory landscape of telomerase.;citation_pages=1-6;citation_volume=6;citation_year=2015;citation_issue=September;citation_pmid=25674101;citation_doi=10.3389/fgene.2015.00001" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. A. Pennacchio;citation_author=G. G. Loots;citation_author=M. A. Nobrega;citation_author=I. Ovcharenko;citation_title=Predicting tissue-specific enhancers in the human genome.;citation_volume=17;citation_year=2007;citation_issue=2;citation_pmid=17210927;citation_doi=10.1101/gr.5972507" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=R. C. Pink;citation_author=K. Wicks;citation_author=D. P. Caley;citation_author=E. K. Punch;citation_author=L. Jacobs;citation_title=Pseudogenes: pseudo-functional or key regulators in health and disease?;citation_volume=17;citation_year=2011;citation_issue=5;citation_pmid=21398401;citation_doi=10.1261/rna.2658311" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Ponjavic;citation_author=C. P. Ponting;citation_author=G. Lunter;citation_title=Functionality or transcriptional noise? Evidence for selection within long noncoding RNAs.;citation_volume=17;citation_year=2007;citation_issue=5;citation_pmid=17387145;citation_doi=10.1101/gr.6036807" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=D. B. Pontier;citation_author=J. Gribnau;citation_title=Xist regulation and function explored.;citation_pages=223-236;citation_volume=130;citation_year=2011;citation_issue=2;citation_pmid=21626138;citation_doi=10.1007/s00439-011-1008-7" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. P. Ponting;citation_author=P. L. Oliver;citation_author=W. Reik;citation_title=Evolution and functions of long noncoding RNAs.;citation_pages=629-641;citation_volume=136;citation_year=2009;citation_issue=4;citation_pmid=19239885;citation_doi=10.1016/j.cell.2009.02.006" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Xiaodong. Qi;citation_author=Y. Li;citation_author=S. Honda;citation_author=S. Hoffmann;citation_author=M. Marz;citation_title=The common ancestral core of vertebrate and fungal telomerase RNAs.;citation_volume=41;citation_year=2013;citation_issue=1;citation_pmid=23093598;citation_doi=10.1093/nar/gks980" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol.;citation_author=Xin. Qi;citation_author=S. Xie;citation_author=Y. Liu;citation_author=F. Yi;citation_author=J. Yu;citation_title=Genome-wide annotation of genes and noncoding RNAs of foxtail millet in response to simulated drought stress by deep sequencing.;citation_pages=459-473;citation_volume=83;citation_year=2013;citation_issue=4–5;citation_pmid=23860794;citation_doi=10.1007/s11103-013-0104-6" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_issue=6;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Biochimie;citation_author=A. Romito;citation_author=C. Rougeulle;citation_title=Origin and evolution of the long non-coding genes in the X-inactivation center.;citation_pages=1935-1942;citation_volume=93;citation_year=2011;citation_issue=11;citation_pmid=21820484;citation_doi=10.1016/j.biochi.2011.07.009" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=P. Shuai;citation_author=D. Liang;citation_author=S. Tang;citation_author=Z. Zhang;citation_author=C.-Y. Ye;citation_title=Genome-wide identification and functional prediction of novel and drought-responsive lincRNAs in Populus trichocarpa.;citation_volume=65;citation_year=2014;citation_issue=17;citation_pmid=24948679;citation_doi=10.1093/jxb/eru256" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. S. Silvio;citation_author=M. Morgante;citation_author=D. Tomes;citation_author=X. Niu;citation_author=R. M. Kevin A Fengler;citation_title=Conserved noncoding genomic sequences associated with a flowering-time quantitative trait locus in maize.;citation_volume=104;citation_year=2007;citation_issue=27;citation_pmid=17595297;citation_doi=10.1073/pnas.0704145104" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML Version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_issue=9;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. A. Stamatoyannopoulos;citation_author=M. Snyder;citation_author=R. Hardison;citation_author=B. Ren;citation_author=T. Gingeras;citation_title=An encyclopedia of mouse DNA elements (Mouse ENCODE).;citation_pages=418-418;citation_volume=13;citation_year=2012;citation_issue=8;citation_pmid=22889292;citation_doi=10.1186/gb-2012-13-8-418" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Tang;citation_author=M. R. Woodhouse;citation_author=F. Cheng;citation_author=J. C. Schnable;citation_author=B. S. Pedersen;citation_title=Altered patterns of fractionation and exon deletions in Brassica rapa support a two-step model of paleohexaploidy.;citation_volume=190;citation_year=2012;citation_issue=4;citation_pmid=22308264;citation_doi=10.1534/genetics.111.137349" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=I. Ulitsky;citation_author=D. P. Bartel;citation_title=lincRNAs: genomics, evolution, and mechanisms.;citation_pages=26-46;citation_volume=154;citation_year=2013;citation_issue=1;citation_pmid=23827673;citation_doi=10.1016/j.cell.2013.06.020" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=I. Ulitsky;citation_author=A. Shkumatava;citation_author=C. H. Jan;citation_author=H. Sive;citation_author=D. P. Bartel;citation_title=Conserved function of lincRNAs in vertebrate embryonic development despite rapid sequence evolution.;citation_pages=1537-1550;citation_volume=147;citation_year=2011;citation_issue=7;citation_pmid=22196729;citation_doi=10.1016/j.cell.2011.11.055" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. J. Volders;citation_author=K. Helsens;citation_author=X. Wang;citation_author=B. Menten;citation_author=L. Martens;citation_title=LNCipedia: a database for annotated human incRNA transcript sequences and structures.;citation_pages=246-251;citation_volume=41;citation_year=2013;citation_doi=10.1093/nar/gks915" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=K. C. Wang;citation_author=H. Y. Chang;citation_title=Molecular mechanisms of long noncoding RNAs.;citation_pages=904-914;citation_volume=43;citation_year=2011;citation_issue=6;citation_pmid=21925379;citation_doi=10.1016/j.molcel.2011.08.018" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=S. Washietl;citation_author=S. Findeiss;citation_author=S. A. Müller;citation_author=S. Kalkhof;citation_author=M. von Bergen;citation_title=RNAcode: robust discrimination of coding and noncoding regions in comparative sequence data.;citation_volume=17;citation_year=2011;citation_issue=4;citation_pmid=21357752;citation_doi=10.1261/rna.2536111" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Washietl;citation_author=M. Kellis;citation_author=M. Garber;citation_title=Evolutionary dynamics and tissue specificity of human long noncoding rnas in six mammals.;citation_volume=24;citation_year=2014;citation_issue=4;citation_pmid=24429298;citation_doi=10.1101/gr.165035.113" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="twitter:description" content="Transcriptomic analyses from across eukaryotes indicate that most of the genome is transcribed at some point in the developmental trajectory of an organism. One class of these transcripts is termed long intergenic noncoding RNAs (lincRNAs). Recently, attention has focused on understanding the evolutionary dynamics of lincRNAs, particularly their conservation within genomes. Here, we take a comparative genomic and phylogenetic approach to uncover factors influencing lincRNA emergence and persistence in the plant family Brassicaceae, to which Arabidopsis thaliana belongs. We searched 10 genomes across the family for evidence of &gt; 5000 lincRNA loci from A. thaliana . From loci conserved in the genomes of multiple species, we built alignments and inferred phylogeny. We then used gene tree/species tree reconciliation to examine the duplication history and timing of emergence of these loci. Emergence of lincRNA loci appears to be linked to local duplication events, but, surprisingly, not whole genome duplication events (WGD), or transposable elements. Interestingly, WGD events are associated with the loss of loci for species having undergone relatively recent polyploidy. Lastly, we identify 1180 loci of the 6480 previously annotated A. thaliana lincRNAs (18%) with elevated levels of conservation. These conserved lincRNAs show higher expression, and are enriched for stress-responsiveness and cis -regulatory motifs known as conserved noncoding sequences (CNSs). These data highlight potential functional pathways and suggest that CNSs may regulate neighboring genes at both the genomic and transcriptomic level. In sum, we provide insight into processes that may influence lincRNA diversification by providing an evolutionary context for previously annotated lincRNAs." />
<meta name="og-title" property="og:title" content="A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/9/2881" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Transcriptomic analyses from across eukaryotes indicate that most of the genome is transcribed at some point in the developmental trajectory of an organism. One class of these transcripts is termed long intergenic noncoding RNAs (lincRNAs). Recently, attention has focused on understanding the evolutionary dynamics of lincRNAs, particularly their conservation within genomes. Here, we take a comparative genomic and phylogenetic approach to uncover factors influencing lincRNA emergence and persistence in the plant family Brassicaceae, to which Arabidopsis thaliana belongs. We searched 10 genomes across the family for evidence of &gt; 5000 lincRNA loci from A. thaliana . From loci conserved in the genomes of multiple species, we built alignments and inferred phylogeny. We then used gene tree/species tree reconciliation to examine the duplication history and timing of emergence of these loci. Emergence of lincRNA loci appears to be linked to local duplication events, but, surprisingly, not whole genome duplication events (WGD), or transposable elements. Interestingly, WGD events are associated with the loss of loci for species having undergone relatively recent polyploidy. Lastly, we identify 1180 loci of the 6480 previously annotated A. thaliana lincRNAs (18%) with elevated levels of conservation. These conserved lincRNAs show higher expression, and are enriched for stress-responsiveness and cis -regulatory motifs known as conserved noncoding sequences (CNSs). These data highlight potential functional pathways and suggest that CNSs may regulate neighboring genes at both the genomic and transcriptomic level. In sum, we provide insight into processes that may influence lincRNA diversification by providing an evolutionary context for previously annotated lincRNAs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
    <title>A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fjrwzdxF3TA856gOFTyu-4OID0FUX59yne7UKQEnX2A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460295","apath":"\/ggg\/6\/9\/2881.atom","pisa":"ggg;6\/9\/2881","ac":{"\/ggg\/6\/9\/2881.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/9\/2881.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2881"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2881"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/9\/2881.abstract":true,"\/content\/6\/9\/2881":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022P1uv8p1BTFMAvEvSwjGKLd\\\\\\\/preIRNm+Kv12+4bn5eLgGxu3xgkRjoZZOcTf1NIKvZKZrP3gHoZZVG7dJLGoB6ZGSi557tZINJnlWPDncm3dh07VgV9GiJaPI+Nz3umdLwSh0ZDdasR1TnB3BLPTjcR1eZ\\\\\\\/WGY5IaarGPI7zfLQE0zu\\\\\\\/5V\\\\\\\/Yc60yWkap1WOJx+jhSjLrVCGS+DsQ8kq5Uvprw4x5mX2awEhVWI2KYpYKrheJLJQNrLrxI0TOgmhjPtxl70wppjf0uxZy7yIuRz8l\\\\\\\/g6IqoJMO1Osclpy2ZI2TuFtynrVM6h16Dvmfdv\\\\\\\/L\\\\\\\/PvUSkluVtogclNMjVVzve10PyfvykQOHb3QGjipeb2agpv3MfjCfolBXpYeC6eBp2M6qdFReXnH1\\\\\\\/Mu6mdG3D89awK4zTsw0p\\\\\\\/tmdpYrI8Qlw3DH278tJfGE5NJkJ9+AMA1YKXd8JRzBija3mGVsQQaUAd+NiOETi1Ie3nAHdnVJVSdDQzwz07ftyIpAfg9xCkHC1gQhTHgbPia9N+brRy6COKmjlFQ6N\\\\\\\/xCCEt44lmYA5YkJeIMx9dTljlATs3gJWgMFVU\\\\\\\/V4ycr8V7vSyFFncZroLmH3MvNvC1SAB8HKeylAC6KEZv2uc4X4TdQ+HSi\\\\\\\/vxcRsaH9cUXp37GXqp9fzO4V9kShm5wawGnFtRbCkAyy1dy8k1uoAmWsXY3NgZ+K1p0qEqGFp08o8OHb5vTJtAW7dUefLeV1ieuJJ00zhxg7dsgZf57LNvt20MeZFqtwjUHf7SAwg9p7YAlsQGDGUjGVRHiwB+Kr0kd+SYfpVmFN0BuMRX7bUKq5HCohIxnA0egJ8roIGcCPVjm21TghRvDSc7zNh0nET3S27z11SvR34nErjiHFMyGBhCKr834IJM4oZAo9hqdmhgIw\\\\\\\/VhSKaUNTyD2Ff3Lw5cPrF7Z6LBfTAkcnnMEBDIMS1WXp6PE5caiLIqzY+oiSEq4s3XwFG6KyK+pgqmY5gGk0mTk4VgBwBOjf7GbJv0DiuzCNBcOH85dBqPgQj9vRXeZOIgfTAZdugm1KpsR947ZFqU2iDbHU51Bo906gRVGE1oCmcJjgIYnpbAfEZFa+La4nepvALhBdLBqulQ3HXHciYqTwOanQd+XRO3+YhMSTtS82r6LKbQ04RlWIzm+gadN2Dppfi6viGeh5wn2awJvoUx+FpeEfgjhlsUB5I33zl8cJCkLC6ttV8Dn3OxVIGMzV4Z+m8hiHSlm0ZvOJObCdaBUVeg3ZGTEFT4Ym2nufnbWw\\\u0022,\\\u0022iv\\\u0022:\\\u002298K7SulyLF0iTTKz8LVpQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d2ddc7521661cd7c8c1b79d148d48d0ce98a1fc2ead199a519d77c0c4a3e5527\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rSNdpsVTb1eB7RYLasz7UHue9U87N239OlBH3WsGULfctPCQzWr9njZWutI0ne8CKPLdrU7AT0ZLl1Pq9JxzhBrD4afujOfRa\\\\\\\/QXbh0WiFD32tgKMxXYBxl6j7dRE\\\\\\\/jcSdTR2oiHFuQq4gqkuSvGe1EkZ0ucRBJis\\\\\\\/O6DY7\\\\\\\/jBEzR4tlKyJFI1LP4\\\\\\\/hkMIK3tKawWPw\\\\\\\/hX9DdtToNfRRMR3hd5th\\\\\\\/DZzmgFhufI\\\\\\\/ntGyXJd69QbiCINAq2sYWEgQ9H4S1yZOc2xnCW19\\\\\\\/9aI1SLc\\\\\\\/FxQm73PbelJ5wIUHgvDuTyK52hPbFFkYxWOss1zvkhd5ENjKOhASDW4Un9WWLejOhM0W1iKPuMXRnHOKJRPUYGZ7ZerdKQfk\\\\\\\/aKD5047lAcIptG8Y1OqK9toM2T284SmUNohMhxK++GL7FSY8Bwt2bRaSYR6QFPAtuhXopJVR4T89A57H+mt00Hnpn54A677t2cipZ5LYANerDfYnn7+BWZrFVpaoaoTSisXiu5NJB\\\\\\\/aICE\\\\\\\/r3bo9AV24qiocBoqjRs\\\\\\\/5TApWTIQ9R7qhZWEVObWByub19yW3YGiRk6UH80la1lIVBSEZSRKAtPL3YojuxGSk0BcX\\\\\\\/ggAI1li5SxXTjZ9h39lfJpwx3hFchXsoxt+UaEyPPdX6xUKlhNVp7X15B4CeHXLszIKKEEey5kXJC\\\\\\\/N9e9bW8EAcAUN0dMpFYsKgUXwHUnqhrB0HUqD0ndOIgNlGyYLo12VO8PY5KLtqRjujV\\\\\\\/soOA0ZTLFduT6C+Y3peUHLV6TiuJzDtrzO0Mv2eoU1A0dLTxnZJcml+9ztsh0TMnIU4plviW6xRhmfnuUMq2apxnrvVkxhV5+uobfYcFsrSdzZPsF\\\\\\\/9t4UomRztTClbMJsJz8WmgEOt7l55OYihrpYm4lcWkIofMss1x1umiQO9lLP8mcPDYoBr06GC+jPgiLXPI3IUcwGQ78pXdwRq4DaPmupKwzwIYb1Tv8sHiYVrxkmW577\\\\\\\/zjQSEulZ7xv+DRSUk94fLYmFcOvfCcsP\\\\\\\/DnhfRvHixAaHD3IBMhzH4vmP9J94Ju49ROTSizkIY\\\\\\\/xSiQEVygoDChtM6iT2npeZ5Z0nhC\\\\\\\/assN4f8oefrGCw5FyHDfDui\\\\\\\/CKm4X4k6bwlzYyjnUTIbuzQiGu7\\\\\\\/6hRGtMjozCoxtEmkxZh7Pj8Ymanu\\\\\\\/zXKja38AxofVQuVEWEiF2GUEI\\\\\\\/R6MpJ7CYc14k6HzjLTLkeSU3K+Ei5uTFJa2hs24v7Okpld05bDYE1SMKXpoMSHYpASlXGBLCckyDKfInWBCCh4Tx0dMsr+dI+JD4rXPgs4WGkxUHDjg\\\\\\\/\\\\\\\/JALcggJMjBt2jrq0nfy6BzfcbdDfvgToA7XjjxEuoVihV\\\\\\\/I27np0oyOhcHG3GkekHVl6e3bkkcTYKc1QqXjUYwnC+dXe\\\\\\\/Nrqfck1l7oNxGTJOIMHmGnop3qlhVAlZCeauzkW4NM1Vsl7k+FCXBEly9M3+2CGIRiGeB7jpIdbc4lrrsxAaQEyw6+N4Yp8azJEwiL0BraFFJIOm6mRyJX\\\\\\\/2mJrreDxT4yy\\\\\\\/073j0PhX7jsML16vM5pZH3tKIx47tF2R568ZKjescC9Fbvxpipzd24J5nppaW04Y+Z72F3UsgXlC\\\\\\\/fPaONXKRlRqywCRpm7vxMxMvdHPHTrgKzqJ1PrccdIWuOpmpsmH1jArNNdYfm0mK0ZQ5Dx3s7oF7\\\\\\\/SmFYVzg6Wgaz7wiIjhPrLjIsBuIogqPpxStt62OamRWyxNZ9DaWUusPWLIKf3sCfCuqO60W4jXLg1v7K6f200V0BEQF68\\\\\\\/b8JGtmWCBMo2jc+nkoMjUqp3bG8Acsp2RxxX2Lo5nIoPeLRjwbQjeQ\\\\\\\/njOvHyCiP3pkQ54a3biQWSwP3AKkAcubQbkrFE4ULpdSBBxZoEGwPQI3SZ8m3H7NVS\\\\\\\/s2sw21xvkgBKEM3UkBnKe6PM1TBddl5gdCg+W+bpyMQz1SidjVLabdBu4IQEK8wHRqxX557C9kPIkDiGoYetnP7jJ9BcX3O5TnU64HKw6HRF0ssLtw4HhLU0tGXRGFE7UTacFWL033+Pv5bTdH8Mm1T+NwyURwqQRznYtlA6Du5wj1rGibnak51dVSTF774KUQyIQX\\\\\\\/Hwzr9mob5+cUKrQ6lfeW\\\\\\\/e8UGjn92GhOx6uYOlZoIT3oCERMi2MAmoDEPaJOimiOCI0bxYF7mz4qCkpMJiZVKb9JmUwcqtM7aKQvGvpbbVj179sG8sMKpH1KCRUqvPDA6Wh9Dal9H0jojqKJum1vlaoQwNHOADx+rnGdot994M8yDgfIpOY+Quds5jhl1yGKlVt0xcGK2FsoJpnyIYuSw6liAbpUb3+2RXaC9LvhQg4RTrWyYocL4OC9NDA7f\\\\\\\/gCGdHYFfgjOE62NF\\\\\\\/nvhfYECSeMvqa4BdkCXeNu9j64mZ0JbbqXzq1boBUaxutzmYUQXAtf6EExhFhRsMJeCje9mX3D78\\\\\\\/q+UjAkW28i1p6CrbeVfPoaZeuGxb\\\\\\\/Cv1EeRs4b92GnTrMQ4ZSFRLKdvubISssuvsMd0fOy9\\\\\\\/OXuirKx\\\\\\\/DkW9bkye3NmnmsD+jBHW750oKREgiqRYKE+hsboyjXW+1anhyvqpmB\\\\\\\/6UPfW\\\\\\\/1GJdslYnO7Ph2iykXmnoyz6SD85nhd6grqlmYVnMBAPG7HqyeAMPHd9joV9S0Pwf4cX02ELVjPdZMZ54m7K9lk8x\\\\\\\/NTq6cM\\\\\\\/TeF64BsoJrNpUxZqVUjlb0SuS27YN4U1a2ET5flg\\\\\\\/iC39YPv16n9Qm3oHUAXdRiQYSWIPBUXd3+0r7o4C6xzYhmEsc4PohunPLjMUuN15K70XDeImTN9dD+jdt8w+uue7LtrG89d0vkj\\\\\\\/heTyToqB2PrUwBRO+tEY3al53cMnXZcipQDZJYofaYwQ9NmXwes0cCWNoALB2dtT7a99d8wnpN4XADRSwgKfE1ficVRMs\\\\\\\/oEo2ESAPAb5Yp9THS22k6syQK+ckOmUdCR+fK8KnXKWfsAprhUyHxhBCluJFn06gPrjrhBWQekRpMBWtkrq0H6dNsIUof7K7boga6u+xozCultschoh7Lk0MtQ8N+4IRlDuGvMfi4T\\\\\\\/ls6knPe8Gk7OCLq4+P9kB3EuK59IVZp6EwEhsImEIsJDAPagCtz5Rwj1565kTdtGa6deli0nl9qLWjG7B+qA1kf\\\\\\\/YCR1zeVhkcRdH7inqkDqSYBuUjB2Z5\\\\\\\/KgARxBmY3wdBTDaIuwvvsZzNbMm3KZEofTeTmM2uRaZmYC7HJijLomcDeCv5Kvh9Z7PLcn\\\\\\\/XmGo2CZ3G6kBZX0rLjtdBKdUvxLdt5cVaxe4why3IHeoe6BlsDMYeciWfFaHC2kG+iCZLLREt5QcGLBEhK3OssyPb9DV3lCzFzINRBMAdobiRe73TyTa8ls\\\\\\\/BAp6g0dVQtPxlu6H9kNZl1872Me8r+ynvNJAl3RzfhmUuNKcsBQodnk5SGF\\\\\\\/yYHPBUVjHMdCg17BngO6lKBFzI+n0QqFRRHYamRZaPehuxC0w66wzNygqOqUh3MSyM\\\\\\\/00wfEIbC6k7jMfAofeHU8RXw8VpwMelP4hs\\\\\\\/mEJk211NDU+DhKkGu8pcjiPWJ7w+yWx8GurtRuiuB9yGtYLX\\\\\\\/IqM39vunWSVnwJp7JjnRRHqROoGxyrgmrchWBduUAOi+cupDfOZl5\\\\\\\/nxmICXGJ4cvFU86jNvJpknuEDa2zHC1pdqjeFd0m8TkM6CyzZiguPjaeq4zrReL7+iO7haV7mrp69+G7i3N4RSYkChJjzWEvzg7Fo4tanlgzA0xyGYCySrEyRGoRnwtb2yqMpuDuX8ENZzrINxCzRCl2BjsakPh2duo9u5O3Kl777+0AA9I0ik2qVQrPjSR8FgpeqHvtggeAcv43KskpAi06lNR\\\\\\\/SOhA82Usl0yW0FXYnJTApsdTi3OJ9AEd5oUJStGohcOOQF2WjcvLhpfNdGGA6Jsf6tW\\\\\\\/zxRNbGz6OQMAOMJYUwuDIR+0T0jT3cxjv7jD6Ij+pBDdbLt3r9LgAFIMXOMlRVOx5IIGstQhBC2sXNw+4iv+ANS9fk9J5soNpMacuCi4UvJaEEW3fT7EmihYaaWJBZOyKlQ79oQeyxvSuPABih1XlCuM+ijeFDSKxdSfoZJvX9mCN+1Ug9aZy0Boux8LegoPw\\\\\\\/r+rqmHyoT573RVlWXxuw2wayuzlMamcBDb3ZAb3M8R2o7nbNsXw33NsdIpIwMQ1ylMjcP8i3TryIp2V1bjrAGoEZ4Tpb1lA9s6ePy\\\\\\\/hgqp9++aAcNYm+LM6+HyCPXYKnDCc+mGLt6kwrlI31dLKhrJvCn9XnmH6faKSTAqHLXOfLLV\\\\\\\/QA7KmfAK0PSZ5OJeJpS07VEuUwt7IB5uZnFjV4zSnQRPbkT+0f1fd7gsKHt86cicnUldJw0D8lJwfHF8yKERDlXWT5OoIxs6BPIIIsHsKUc02JGU2ZLIsco\\\\\\\/aXW4QpyhES2ghn6mTcoBrTzEtV2AJ5bbXkEjWUrK0JnPh94CDUF52OwurWCZ1tNLGI+7WKC2nZID1fHyCECIuGlJGMqVsKfbHaPCYB0PHL18p\\\\\\\/d8zqWYINL5KBac7gDa2p34r0jm+pTH1fYxj0I3xYe3lyoWhzpVv9OHFCzjhXXxPMqO9HuwEldXYhOJbTWJv9QVWubHNhoPrEWw6hGEBe6DJWKI\\\\\\\/6qiQWdfmG6KrzH84rQCXU2eIEKzpinQ2O+gFO35rX4sYARmobIf41z+CPofGEGGTwQEiNk7SYaFsEYmCcMkgMHL2rt0TMv4bGxUHRNBBuG8g37UCCTxJLHIuyRhIcKb3KBUGT4RmXSQJGaHi+NzrDj+aHZFAJ9D9L42F18mUVZxmHnDFOxmhdldOJcGzg\\\\\\\/l0KG05YXMe7pxnFWJapzXK478DdNuhYaNmK1gfnFgVVsAet6Ov78FIm5Ybi3qjRjBB5z1l+yhGnW7fraCeZV1dfMrkqBIiaD2GcrnbVZeyQG3R4RmSjdf2ygqT4GJpItjRzvp1\\\\\\\/wuCe5+pQPImv\\\\\\\/ZYPDAsv1mKsV7yriuoAZ8Nqio5YSZQy7H1daYvqZC\\\\\\\/tuC53y9t882JdykhjbeI\\\\\\\/sH3d6VwOj2U42UB8\\\\\\\/qfzkCUtRps38lVFJJ2sSjgaEjJviQ\\\\\\\/Z7Nkas9cOJ4v9mu2DW4q30TpzoJKETOXolNiNS7pYbb18s77A7ewliaXJ\\\\\\\/bNjbdtSKuy7QOhrW3qbH97HccZUamm8F2eEQ0pKBSL8Pt4wn8ZHheFkfZeCF0PWLsMo7J6tsCePHRhKGsewnOYq6XJ2YkUz05+R7NiBEPSFDTMAdd2fQMiMrO8rJu\\\\\\\/uOcEBx+jiX9HuZBQkmdpaAy562gr8GfOYKg4GWzFhfVwJu0kqF7guEsD3yWxzrtW\\\\\\\/VJa0+2ZDLpwXO3DYbX8Zu+Yy4I+8tS5Y53gIIYTkkkuZ1Tr5FjZguzrNaxL55uIfskzWJPKr\\\\\\\/\\\\\\\/YrZ3ZaoFQS3eLMVHqQUoPfTPWn1\\\\\\\/glwjvPJyFgPIyQGwqZhRCvDWDj8pBcYutbJBWNQr9Os9ANLw+sQ8jG+wdP2CjDRiJqybu0f612Kr31UDiBy0hD2IvgCIDgruWZMpOSOX0JeSQ3VBEz+o9R1EVg\\\\\\\/Vl6++sXM8TsJsLuhg16dQoTGJ7ZrEaTSQrNn6qjqGTPdB3zBY11aKb0x3XOyz0xTjRtGLBmV3UFVMaUESkPlCsrE1mHU9zSJn\\\\\\\/BYdz\\\\\\\/6WoxXKzX0Oc9sgzG4pziQgVpbCepQq9wx+Eo+3quP2bKoEy5hFy2n9TEwAdi3c3rVvUHIWkOX6uD+3JMuFhaYxi16l8o3FirczDf83PnAKZhgbdCFldXcnGRsrol6qD6c1+tmo6G1f92Eazjoy8tr8NHoAt3Wse1ADWsLGGlzURe5MBH8Cb\\\\\\\/cGgi5YhI6SgkwhZyawyRFYnshfU\\\\\\\/k9FFg3fg9jdngz2q3xgZBju6UzorraXkalSZ8VLZJCUeXT\\\\\\\/jerjeOcsuebmZTIacx47SFyB8FfXaJh51AqRtrje8F6Xe7\\\\\\\/HkjR4PAsdrioM1sNLHLhAT3MF6xpNxvKutKSvVhmxBip9QKc95Ii8IF3FjQ\\\\\\\/wuFZibM6Qmnoydk5zmiiZF6L9rqrZ4xRAjkHArxDePzetkOBt82In1uqCG3LtRIdmkrWcs5+dtjCqOW10fQSR1s49rxMjgtH\\\\\\\/7R5X2O5xzhdUQoKKtPMOWPaVfjVeuX5tIYqg\\\\\\\/dWb6Zj6laEAoWse1K+OW9sJhCnoDhZgPa3lu5u7cdqkFCIb4Qwt7uJ32pIRRqfCifg282VKaFF7U8X2Ql2P\\\\\\\/mCn+G1VGLWxo49VY8utqTDD7QBFN+ZT8poP7vP3DA0goPZqFXLl\\\\\\\/rqZHortJ4MSOU0KCAN9gEx+SBYJ36K+O0DPiZ1tiaCdNC4qVcgtikiyDLb11ZO5rqFm\\\\\\\/jRGoDdFF37p2mZHGo++giob5CJEJPcM5o+bB6iTB6NYWjJbigGNGFsqkVVw2eI98PaWkxaQUF+7\\\\\\\/kYxRq4MOhrycoVwcNEV6WY9mbttlqdTO9dK3JW2wSTSOH+hXhQS7T4F13Nav4ADSEvKejc8oojFqfKbIyk80tT8QBVkmT5auOOLhOH8tkYzvk+46s2DkXXYxVh0Wn8QchDI1V8lo5IhwYfYEwAFgZRud+vcDYmOE6JUmMhb47tGeshC72XdSYHbp7r+\\\\\\\/nN1dvZ++eSqQm3HyxyZYVctOq3kcL7PjOSuFsy8F+kW3ZHGK1qkqqZ9iKnh2kxfre8r\\\\\\\/FXQbvEuzZY8RYrZYEFCpxbhDxWMC\\\\\\\/jrPwSsSoPxbEIPD5amnfTQSi\\\\\\\/Mdwy11el2iVMrY4emikjKqiKAp5Z4leymmhKRsY7uw5jezEMcfYuM9bpIQaOEawt8Uax9VPKfVcbEF30nVyHra6l7Ybe2VCNocsUu76CM2Bgaly43eyFMCV9R4wFx\\\\\\\/t5Ly5MOblgZ9X4cVezi53tRj0BfPowkScBgpqQRrOe5s3c9GgX0D9NroWA3R\\\\\\\/WdaO6oESprYXq25MjOhAcvlVrGSNPujEeQ8Tacy+nQioZ4KjV4xXCcFxGGkHcY2UlZ\\\\\\\/pf9Ie8E6xgR2dArxhqzN1VEHNE832n4tRhu\\\\\\\/PxuWP11wlkyzkpD24qnjdFnTLaRHH+5y2giCHbgRon7u0dHVk8JpJx8LBf3foBzpOKB9cxRtiiQmu41vAcRguypGUcVe5HTc3QyGNIFzokwDfEP8JjoePFUEE3rbRELFIWgayTjhqinQLPBQ2QvKVEyLpchlPX9uXyrh2VROJmagtgNA6eCPa7EU\\\\\\\/TQ2HF\\\\\\\/RjvhN4qtMgZla4a4hF2vHbOZsgAw7i5lLzzosVzUu8GxE2sMjM4hd4JZKHrc9HQXrCXX\\\\\\\/dcd4ryJPH7yBxhaE5qhZnlCqnTdk+WvLqNnOkDCg9asKrsauFDBr1oD4AtAbh7AFDBmghbgB\\\\\\\/t8X\\\\\\\/ebY+WN7Z6VC\\\\\\\/uzb1\\\\\\\/KtWEyzmxmsqB45DV+a4IBG0MWVDYN606ZYsTW71pXVOpuscPSkWWWYWauSwPdF4z8qKtANJ5Rxt+ZJ7aaorEGDmUSRuPa22a6KusANfhyM36GXPSEsXC+hbRC38ATtctSD78x8sm\\\\\\\/SfSwPMm5mJW0nGIB7PIxLc3JyAsD5qehglQtfkbVs3CbgB37lJbuOBFkzdpZR6UXaWMBAZbbOP1RAPkKhqCnTHefm5tEamsLQx15IYJmjyswEeXFXlst2cI5yJZY0lqHK9Re7goPYlukQKoh5SCdwWfmfF0NbCt42eqVngA83gDgCNoFMj4shSduChJ01Hya2vkzLoIn2sLhjz3TLykYdr7FJktIPMFU86Kw324QETl559RVlmrNDpZ1PCqkoHLQb6gZ8DMm1fHB455rjbyJz+Bxr9ouSubWUKxn8bGDMSRj3egq\\\\\\\/iv5oYpSaV+fmDMJDv5TFOv2x9oVyA5BQ29XJM0oxfZ4eWRQc\\\\\\\/H6HwkJ8d7cWZ9IYSyHvBzNdH\\\\\\\/7\\\\\\\/2+9UP6gXWeao6OJ2rZmWp6zIArPXIJQjslxGGEZjS0\\\\\\\/x5850SAvajfY8bec6Wi+jBe0DYotJFh5v9aDBkgs41vbXLcaxULM9eVDzndwZLV1tOvN4K5YrCJYVvyu6BInr9FymZxsnJOieziJ3pG\\\\\\\/pV3dYhcuT4944dwTIjrSloFGnf1CQ\\\\\\\/4evoXU+ubEYMmXqTpJCktkZ1rQ+eOG69Z9N+OiahbnciErhEkz6eq4GsmdLWampv4dRe0+oV6GSPA9Igx7trIFtpo10Cfct\\\\\\\/Mn1uhxlos5eyIltzTEfH0Tz07JvL+JBXPFlBAlGuWwaXvu7js7oTJkrl9J2st7HDbuDxJxuQaZSrSasrVrXWXlXJ5gr31jnnC9tw4I9+5hu+6Swvfv4CzhP\\\\\\\/SA9c3gev9si\\\\\\\/RGliISh\\\\\\\/o\\\\\\\/D2qEEK7w7APQqfYOC6+2DYqamFmq9Hib3HYTShEA6BqNghUKkXnDgpjpWd7z5M0EUyBIpigQzmrwBM0oOnU+pnPwb9J\\\\\\\/uO9FbE3+rbqGkkxSLOVgzY0hit\\\\\\\/1zNKoh816vs31GzykCRW8jHYpyKdnnDO6Rq0CUBZwdKV3zeP+vMv7GkBe0Omepn7UKH1q2\\\\\\\/71rXkAA3Yg6oVCo5K2Xs68k\\\\\\\/+2eLhTPY7f+cJbxZEJNoBZUm1tpX5Gs7JIDiAWlvsJVCNV+JBp\\\\\\\/wFa48Xta4CFOtGRnrZlfnigeNeaUpTJUrdcGCX6UDNOI0vM3TN3o5KvwOFMgrsYCfEsz53XFJO2W8kHZitvYIbBcKtFoEb4X\\\\\\\/cLKrcsvG3IYqEDHRPSjLH9BSy2m2ILQw8d6FF2p+tNqsIEReZ4W4DbsGnbUY4YxfHok5ukomLgqja7VxRdlIK4AuLD\\\\\\\/qVb4QFMHUNAMySdfXR\\\\\\\/GQ4\\\\\\\/0tcfKogABvmI7mYesAaYPgeb1Z3CrujAPvvolk8kt1Fp8KX9vdy8EPqZlNtBZTbivjWrUOHroyjoX7gAP94LYaIUFeGOQsVZBFtPUsON9nniz6swonRRUGSptPRJvy5mkPBrkuhYjXVSpNw+q\\\\\\\/NQsjsoPVXSZ+iLnxAUFK9nw1UjOStF36XwEyEbjlhsDF39yjHFTPSJbjRvD169KTicDQSh31Hvtl15j3spNXaer8uJ20JSKUKdAcBMqVrCgYw4VxJoCpSioCLLrM+KIzkzA3RDwNVcTuMaQGJ476xv5mduR109eLW7RM8LplcsoxfnFztRSet58DTjezXmqJc5N\\\\\\\/8kTCU0mRH1fJ3P2SgCObMFhEVrHxobvD2j7zj3KSnfSffgWao+SGkSwny6c\\\\\\\/gt47mKrsRjzoKHZLct31Lq3OYCOHk+hymYW67vD4i0+71nnclgr\\\\\\\/StvDkSvOBmLwqabiHkiY0M+n+FH9OXRGE8sORRPjQUyxUWmUG8855Z\\\\\\\/tqcZzfhyQYgRpXNkmJXp\\\\\\\/UbrDWfGzp23lx6VDtr9xBjLLjVuFWnXp6EGYlOwq0rxTVoPL5hlPEpqkGeUbu\\\\\\\/YEylYhM7Uf\\\\\\\/0y64OSiDaXg5ewodBXZpCbh\\\\\\\/ijLYgyRO8AJvBhq+AUuAmCEx+uU1yvs30o85NQChj2erOpHlZ8OQ2SkcwaRqhpzvJNijQNCcnpC\\\\\\\/vcdp+IoH452Q6OVCmhTc7lIQTixCZAMI4eXHbUe7dm8B2T0+92PkugCXdz7KvJl3m8+L3lj4JdMq8xkyYsvEWrTH+KZYfonDUQk5xqJNbzAZ79t9ozE7PCJ\\\\\\\/aiLAc17HVy70FjLybfKQXJFY4BecV3dJW5lHw5wYE+yw+SsOXHKs1HxzYALRkBYQgUxijMlGo8cYOky8UMkGXzJyU20T1x9x\\\\\\\/BlR33fbZAQtoV6ZFtNmYnT7a90+Ix1FxsnmX8Kha6\\\\\\\/7oUaqMMZr3S9BbHB1mJvopGstzFOTYENvwJRyEZh89KUdk\\\\\\\/HPf0Ib531DQogDA8FKQ57EABidUfVeYDMNS2Oz0zDYpZ5\\\\\\\/5n01cxnwAyJB1tHcH\\\\\\\/6vjS7KwtEzlUazG6xIIC\\\\\\\/5006pGCylsYfGZlr1ze1UbAHlFs9G6HVNtpg\\\\\\\/F+WSMBopvScUFkwsa6Ryc53wBk0lpqWOc4rr225zV5GipM0+a1yyampm\\\\\\\/MNgnPvUOuVsQ7baH8z4ICBkuC2UvvPfaWGHKeFIGqxUq0\\\\\\\/OPJyanFuXfnZ6zfdP8i5nNY7ir7cPZA8TNc1zISZ+HKvMUENDxgr49IvQUDDIb07y9DH5SpgnGb2tWNhY00ajYlnl7bZW0+w29N5I\\\\\\\/cmYzpvA+V8jLn0VS+LuJ1Zfpnii6sLtHRl7Sx7HrE6KZqU85\\\\\\\/zvf6jWwCv0tOIwPtYojFX0XZ2qflaMvRNVDj6alDNCBLcq9\\\\\\\/I59NjZUNrS9PqDAK1lPtBI4xjS3ASxFHRdxUMjYW8vQCl1qMURTc4qqEq9c4WB\\\\\\\/SCx3eHL8pg0wo\\\\\\\/cQ7pbtaZSnGTqFccLMUzwL31GjFvXaaJDGNFvzXZMpHM68LhV8bqcZVlj6pnvfl8Hy6JM4Rj3YieyVUYvEt8As+Vw9lixZajfA\\\\\\\/PgXMU7E4za\\\\\\\/V9oVzBCfz8+FiUN4ToZAy7byhUc0LvG3FNA\\\\\\\/HwYkJu3wnRB5KUSvw756I5JK7WrL7IhcwtPgjyLvlj9dWgkFRpcYZm3vVHlC2mok5lKMakQl+92\\\\\\\/3VbRiuPGBOmpHMnbFSxdxeIole9AjTgcvU19QMzQOrutR4TPgAl5TloBNs1wsYThnui4L0pLalXOmfzC75lRmtPJPL8ZrCvrcgbUc4ZnbA862xGyRCRLdpHAdJ6EFK5X9LaExX0pFiB72TVzhr32He+ydJN2eY+MHdRD8ugF2AwtC+\\\\\\\/Pd+2O+hVGOkENC10YONNSC4+VdDZJZjNSWlSAVw3lwOyRmxkfhsrXHlQQILKY01n0VK7VyzKZvmRn9MBN+s9M9edMpqpQzxx16Z+ckbRc2bW77tjm7J7JZSI+yBaoyMnnrNaDt8M6gzhSd2H3KZC3tJs1n132+sNcvfCahuCvtfgjQ36rId5g2Nqrdu9mBfbNfU73qxA4cSrH70tkJa1wp8odR4XE10VkkpHTSKWcXFQAL54qoxkGQEA\\\\\\\/c1V7yJ3HZHU6aXVgqAbFluUyZor2pdy+hbW+Qj+HrGXYv2Cg+ogCeyBPUEvHvnWd+C9wtiXSFc73sMwhZEeQzL8P3jLL1dh405+kDy7xP5DSyRc0NjFI8l8xxnmZMhto7aWnxXq7Ckag1rEPFmnysd9n+s0Rf5ZIoIwbbs6mm3tSsgxr\\\\\\\/8Je5nN1FUqVQgqP8ChFFL1AMbaud7eWD0LC+6ABJkXBzvc061dWEZp3M\\\\\\\/QAycXwYBjVlmOaz5x5\\\\\\\/ZifpiVvAakPMxDcy\\\\\\\/uxm5DviQyBCBtmPafksHkH9WoybMVJpVEeudgrF2MegFhrxX1mTsqyC3e56+V8Ngxzh\\\\\\\/qcL2ZCty8I5hMg\\\\\\\/pUgNX1\\\\\\\/v9A0kOKUH8IyoV27OKt+XKguCM6KUfW4uauFXxRy0hO89pqE4ys1UZDXnJ0DBjj2LHl4taUK7KaRoEdJb+xoixSFOQ5aAiGQLGXJlQrH2LDsBemFyfNpDlWbQjzD57obzHb8F4Gu5oKZIEGs00QxIOR3JCoBxcqJh6TpM1KRPFxyM\\\\\\\/PinH5LrPCt4n9HeQpGHaLx6G2\\\\\\\/23bt09EMTT68uFpFvXuOgmU1LH3pAYnHY4oxR7RDCn+JDfZ25L+CWuTG0xR9l5ozSOhRECYXH1Q8R1h9zBEw2OLQC++0tZo4yu8SyyoLr6SPGeUtgUrQoaPZN2pBiJyo09l0zlukeN4IpKOTyLN0mP3IHP1klFaA2BcKSGz3WsQ0ErfLO87uwYhJs8PI39AKuGuy8F2VU04odJEHMiX2+HxbbKHaZcIKfQwy7ea9pxE4F4EVenQISpghHAOR0cCjp9aXfMiVr6+pLQygW8ubxLul3DdjjPv449MT+bRn9wU8K0bKtpjtmewrEzVKCYErw9YJeyq\\\\\\\/ufiJsrpv1Z1EKtbqL9ao6J6qwUbm3oYQ+1T+muuhsSQb8qGctYAz+m2vOLrhl9sdoC\\\\\\\/36MRhwiJhUEn7F89i+o1IxKKIHuCFzqRAqcac5jDymN9L8TfsIMrRTQz84F9Hz0T1FfKf+lLXRLTyO6OYN1yeMB5vxVQ6GseBUE8iYTcohoRlqnh2ZSKkxlCOyQlTpNp0RBtnw2uvQ8VhZ3XaZu2ktlBMHXdNT\\\\\\\/LKuERURZwE\\\\\\\/FOqSkzlGq1xLOz+CJiZP6DWoGQBjYaOQIAcp8cXI+UU14WLcPtTywNSOnc472vwDpz8\\\\\\\/cCYjpU4wUq6ahPh5cD4iI6BjJtEEMdhQTORJLKCTJMFWluZ5RSVZXZh+ZR4AYDVydzvjg8TwdCNIoOzwoiZ0B3E6WLHQ18H0vm8b1MUP7tfXCyp2ojMs2nH6QUPn213VfeCAa7XOq4\\\\\\\/3eA1+MPIO9+xhjmyJT2L\\\\\\\/DTJ0qzB4MJFiNnLhZXp4YjUEGNWTsty4UWNjoTDsvpvHgklEJn+6CPUFgj84FrodkjD7E55CzLO6JNwvhw96bM6g+A\\\\\\\/rF7Ba26lC\\\\\\\/Z1Md6pM+aWNdK5JPTszicssKrgTm9OGcjfOwjZ7AQpqW2j52XODDVuPVpN61oip82M5EXPbejGTc4C1X1ICaClGtekysZopS+JuCSHzyk0UTdXroFf3pOS8sUCg8UxhAU3kpUN\\\\\\\/nZIzdYqWhKNXFPyTjfkGsQTm9b0B9EbtTIjF7tLu2UkrscmUiKwwVCZsOC49CyNFAQe87QUrf3xaNw2UkkQEfD5ELVj5aTNmfKV\\\\\\\/HF469p\\\\\\\/Z3qICns8IXIPVbUYWI3b6xTHpjhhSZ\\\\\\\/7hSHjkJIKftzGDQmgXDKM0b58MKORWeqU8TX3Q71HY+AtY1atZcyqVsJdSoQzycTwdyoa8bQhUBJNHgH7fbMgLxVyd1d3+kgXyWZ8awTAHMoBkvv+xCRiJJF\\\\\\\/Wg\\\\\\\/n8yoS9Lrd6aeNJVC1kaS4svpfVJunR8pT9RgH3JZCZKJfmQPoxuXe3HeZA2oCs4Lcg86S7hjq3Idfq\\\\\\\/IdjpQ\\\\\\\/76vIBjsCVkkj0JI4MOTmzMHu7JMJgClSfUFzNmUUZIDLBSqrbkFD7vlnH\\\\\\\/Ze3oIq4OvpHEReC0NSrCXkD+tnB19a4wtXugceiFkoKq6WIpa6OGNHBGcjnY2Tvi4B7xAErJKu5IXw2EgKwsuvzaBnCx0zFwl6yl3CjA65mkSyKwWy1MS3wMDQ2knnybb1FuZwI1JuQ+SKHHl1Iek+SM7ws\\\\\\\/iYMxd8IXqix+mOniYRScMo9WfaVwLAsjI3sJ6i0FRfumvpbU0FAA6fNP1w88i+Tl7s00JWfXNu0\\\\\\\/G95W1hWlXMvp8NjX1bi2W8QjG5yeqagY0FjXZyBnfbnRFlDx4yE8TEilhBt9pP7138hSHq2MhXGPuJxfnzcWGdF1AoJFXshcmLirIubAjmoM2Z17TBkk8T07YFMVKr4lgO1W9RaqyQ\\\\\\\/MMH0w87ccx35SuSDJLLTe+bDunHIv+IQDQANcVLkRuzPCENOeZ5RHxdY0XZar6tpetqUEorFL+SgOTmq0eAAhN7TDlFfWD62usnnm1LeV5i70Zl4xJk12dSEYRIvL\\\\\\\/9xk7+frzJ65Q+lioe2J8Vap1PLUiwpont4ysN8hP4sygBPD1E0v7pYyG1PUc2TIWjUiR9aMn06\\\\\\\/yLukbMDKDk6nBGA39aQfBqZhxqVcOo9+y\\\\\\\/WIOilg5rfMpIjmsEcSFWyWnpYj7UrMjqgtimBlDGSFJdq2kYtiWvPMH3kmzsglDmljkUHsqlxrMn3CyC2zrFw9wIJPyIi+34p\\\\\\\/NmDBKkUKUZSyBQ0ceWSWL6mEygjpLEj3\\\\\\\/0nAQjqH3CBl3RguiH2W5FKXT8+BdSqaUUs2MNY7nASzxFyIWcARrPUhQQP3nagUn\\\\\\\/W6+yrwmx6CZXSnMz+HmV6S+d0hMKWROrvKTZgDQnr1pzItjUs3xf4RsNQt9A9fKqULxx\\\\\\\/KhQq9ocz0f768O5\\\\\\\/uF5CbRhcfSsG7+y25em8oK8bU8bNaTEsQVu5RgAeGelhyfD9ZqmGDqY3domkQqj5+H79NdMPH1s1BoU7a7DOJjuDtxjXr88ozmazVE\\\\\\\/9ilBbCxBGUDM+j9owKrBx1DzmE3PGVkqz3TsHTGEthuRtkR3Sm\\\\\\\/taCDO4UScEmO2mwG1nHcKuqYDE7XwCSbi5hhuP+LaJm4qI\\\\\\\/MODUzxygIOyVkZr32eO+40HF+cts0Jc\\\\\\\/hNUexkHIyzg\\\\\\\/g8kPTtorhmF85L+xC9PCAN345mwCP0bjNU5AaBaqa1kCXS\\\\\\\/L81pstzoXpg6oGO3Aix+6lAHdaW9lh\\\\\\\/EdA036lQ2ZNofKMqPQVTWazTR3qnNK3N7GAxCHM+6eMqF1dYLW8mnGSynWhvINWMELo4Y+BPB3nZjN9VaEViNwr7SXubUPae5CnMqddy\\\\\\\/X+9c2Q1FN7M5NE1cgD1LNEKdnyz6E8PGiEbysGoi0kU3QHgGQ0E\\\\\\\/yH4UtmsswC0XiAu0miHIVDvV6y9d90wlw30CQWaTUHLP9Iu+nZxnLsOpjBXRaIjSGfnrxzMIdtKenPAe8wpwvg\\\\\\\/8Gc9p8xLbSJL353nmFpSmWmYzfN5ZrQgXyGlwiJtQAuuZEMgF++Ch0VveL2Jj1XujiZxAVn4d7Ff2idk8FTN70VWYDZb1UeLci9Ufpze3e5LRr3qX6JgUaMvCKPvUH0OF\\\\\\\/ToesYJbm+Eu+5\\\\\\\/+U4uT8Pm5AnNtS5w7pVFL3mKhIuLR4CZgu2a3B710SfO1y5JYzv\\\\\\\/uezqbUgMMRe6TIFFlcYn2SgtRsMHMyBz41FE2n7e6rkLFvVU328L35jfqCwE5ByyjjOvXNrG+PITzR5QQK5\\\\\\\/1CPUNdjTgYQxUZ4EQkhy57SGYqrOPtf8Q21dYmWKJbV37X51B6AdHzGJGDK7T6Juzx18WXTOPsMcbKtfFYQkrIGJmGjPH\\\\\\\/B8zIbEUpcGU1FUgN\\\\\\\/Db2H54LrUqe77jQdX0vA6Hrfesgx9CRALs\\\\\\\/FpMW829rKmdoU0tE\\\\\\\/QURDqAEoCINu+fLaL2ON+rRvRzsikg3RzhuNXqYv3AmsQ\\\\\\\/HRt\\\\\\\/FdLunEVsSYvxcPl3jlgAQlgpSvw49ZYKK3RXxxFXMIa1Qnv598Qe1ctAjbV56RWnXTTkEnFBpwFJ6SSRcZvbEO67tiZgq3yWJ9hOTvqMvXf0srb\\\\\\\/RGWmzMLqwKykMBtK9XaYKlXHq3Zz8beXqnH4ufJrDHT8QGNn\\\\\\\/HEQ7U0yKdd6cmCsxuQOS8pfTGb5W+qPRv9ZUSeY9Asc33UwmS+3+mlgeKUVuK3KMxFd8sEtW3DbZfjJ+EE5iQzFZFVAXyWsVu+CXEkYT9qtmp5iU2tK6xisBnLadYVlQE+FN00KjSxoWvnaATK74wFf\\\\\\\/GYVb38lcc1rYsrmt7ojCO8F2zgHFjmZrwR15J1wTb\\\\\\\/TfZ9YU1XRsNpd4j+bDvK2LsbAMEJMgTJmzxZfp9KKPLluqOGLR+IHa+LmbOr8XQX2nCYr9e04DEBu2K8VPgKAk2AhYDeuiDAHFV0Z3Tnf1Royo\\\\\\\/rJ5giArq01azTd6UJ9Jeq2xJH69fSeNtieuSL+CRCAeTYEfYOLzLS185hCEBIOxLH3NkRU0osKBnCgLHqrx8HYgbBtRnhHkVgU1A58GEpzqKxYRXvUTElAhB1fV4e9\\\\\\\/QUlTS3RGiDTecRb31mkHEBCeW5pYvtH5xpgFmsqi8hes1qi38uy9hPflFE1VxnAH\\\\\\\/9IgY\\\\\\\/wy5Di5gnYfW\\\\\\\/1RHoQ2V6FujWPcDs9uRdJcGGM4b0uzq+1rd1LIPa26s\\\\\\\/HkVp0saRi1gsm\\\\\\\/UpUHQHYusA1DzKVe+MGHy0AzWnY7a9aJqYY\\\\\\\/W5QZiRPucGb\\\\\\\/26DYZu0tuS3FPahZQk7EhbRP9nKiheo1wFUpn9+LntV6E0CSWJ\\\\\\\/CS\\\\\\\/6dy\\\\\\\/+diW+fb1ufhgaITssHzMQIweiqyDqXmU5Z+oAJc1nQkPkri3fhPSnPhUzRn6eDhwBhDfc2kA6LXTjmXdYhJgF+SApS5NC3DwdJbGCQMKtKnMJA1W3xOH2jy1\\\\\\\/OOrrN\\\\\\\/vDdt2o+pM84VuAVFAS58\\\\\\\/Rmji46OIG1QfUaGe105DE\\\\\\\/78dPZZuSovBiv6iSe+L56IYggXKDsBhXWF2biByVpx3KEs57x9YFeizwJ2EszFyV01rwU5+W\\\\\\\/LcGxX6aRjYdbh52Aenl4lCflHIeELmqP4hn5SEwvtYooMZ95dl+jrCE+7c3Fs\\\\\\\/JvfSTt2lUxMBX02kwM+m\\\\\\\/MOy2m1IVlmpyrDxhcrQesstTrNEPal4DRyzLFFvo+locnfQh\\\\\\\/FMjk6k1vBccMLSj433\\\\\\\/dW\\\\\\\/NNqdFhwusMMIOv1x3LN9rqdKviPfW7rneCg977nwqkntQ3EEqpXnQiL8ot3k83y7lDAd4926fuhUwgzrUZVAamLeoFdeHrrbx997BiS0CLrtC9ZPJTIc9XBX1rd2vpyMX853S\\\\\\\/y3EMUJRone59+uzmDYSxu6VgHHzjIoEPKGqWQnprOjIpkIWZi+Hfa6ZvxG3lNqIM7L5\\\\\\\/jaYpbkJyZ0uyHnr0tqOHI\\\\\\\/1YACSyUAKK2244JOt\\\\\\\/dv28yoZkcD+JFSrYg029zIznSQh7MCnPnJtfJ8egQjR9SS6wy1g8QgolO+\\\\\\\/KSvvFvK1B9OatN5+NVanvnOQX3U\\\\\\\/m+QWMqGyYswDQbxiy8kZUYETMmjFZuZ4Yux8Mr+fs+SIOoVJqwVoUJHkUuzegw7tbhRwf7aOPCHuLoE4vWQBNl6lyZQZ2jFCCHuerGaeWLI7k16porVUgxA4Ja6RHTtYBVd6Sv7fcksPj6pPFmTDL9yKankS59A2VlbtRk83YKzD4ZplkAU10PveF3a6DFdvT\\\\\\\/nHx2992aWIu4eZ45iYLn9sG1pbZ2zv\\\\\\\/GOL5afz+PjYCrLvJlGG+fbcsDrmvghcK1bINRZS+Q4QWCxOAH4XjzVghWjxsFBPyxvpJ657uXp\\\\\\\/784wbJZnCMpY+CL4CLxZ4s2MP\\\\\\\/GSV\\\\\\\/surnVXeFVIevI1DEtgMTrjQZCisK+jjGveKNwpw\\\\\\\/h1cwsVr3w6BztJ\\\\\\\/k7erGFtQxmRCocusibGMAmG5PHQ1++RnJCDLB0qFd5wNwe\\\\\\\/iVmmEoGHnGoginaVtOkIIeo39QUScaqK1NlAEf2tlezCmSa57rnoq1h53BseamFzpvDUVjZnmW\\\\\\\/pkTILr9MuPmtBSiXLz2DSRbN2eC77cgssiLw3Nje9jytfM6eAWXURXLDoaRiN9tTKoAR2pGaMoDWIrRdS3pj60hJvX\\\\\\\/Fj\\\\\\\/\\\\\\\/X6LbPmVbr5vSmoo969Bxb+AbhDh0VQXs7s4oMa+rFbaG+hsEzxQ2MXGCMZ9+JPrNvPPHmRycz9xESzZD6vB76KNwUjSpL7CjKJgWvBSEslYhjHPYUUTciPO26xxThzKFkEsHXStF0yWnj2mlVYNElZkpW04zPLA+x9cqFL5Qe7B4H039j7VMUN4olTD1tIeZIVjBoTDwPxVAQogGcWaRoVOflagkCR9usEkZkQWntpO279Mytq2lkIZzBm6Uoi3aAQ8Hb6BqTN8l0anCGd\\\\\\\/41EWqzv1Zd\\\\\\\/5d4C+rdNQ5bu\\\\\\\/4SVsfBXHS243yXYXelw4AFWEuV53pncbT+AZOYXQgCuPFSCEAVDPXDOxfJlC5eO5qQl7l+9kjr1RjIA7NzJySkR7PucR7ZxKaJVFIb\\\\\\\/QigaD7nYZtuyM8oQ0m+HeS3aU8V8kKJx4fAWhyP4idYhyqRdgiwlN1GJeFxoAyTyv6ZpQF3E0JGmAzllkdvmuJyTA7rgxqW8Ajhe0+C2B0ExchS7GUAVfa7d\\\\\\\/AAH4M7ujzfySwg9sZ7Bvt8vkO8daZIu2293NKM5XESiTH6I43F7Z4DE12eAtYtm6JVuq+gw9jGP93546GMYObx7NXLpdC8MccdY69rthLejn6fZowacq4\\\\\\\/L3XXtf4jS0CIbEX0UpOZX6XBZqLZb1ECI+LjTo8u1IYc9yqMNz2NlEaxH2mFtZUKDbL9NlXmP80p1cypMEgvTo9EcnNaJMl0fd2EQllmP7hQ6QjP3XzvsK2SKxgB1TYOb6crslBMel84oYJObbQoQc9HHm7FLqjJUIFHm4\\\\\\\/TH72\\\\\\\/Zv97BISqXWGzHvZcD21Kx6S2NA3Fv2OzYaJzmV3fl6mJVxDYwJsrD3oKVgvd0TNLwLeDRsuzpZhanunsTNcekDmbRErd36WHCOeK4l9xYHfrYr6E8eCV3L3aX36qKaBA7tZoPSKOSPSsSkDNQhV8anFxgqGNZ9K6GIVS4mjqgOdNFsdzn+c4ykrAvYiYMZTur1qSVDqI5XOkBRTUOP7S3DkmqfJmq\\\\\\\/\\\\\\\/hAQ2D\\\\\\\/sry0d4G28H5VYwZ6cu\\\\\\\/kWF5MWV9HEMv7Gm4uIPdMThJd2JB3Dca9yH3CeR4FqcAdUQPoxBx9ONwfI8uK1sTyL3pAu90o+GWT4XdLUm8HUJExJMzv8jqFr1tgKb8mJn3QY8xUg9HEmaPn7vLNQmihHURmUbPQPfGXafLvY0rmuicBv5u8TQdIPyQ3qNdWyAv+30pNl16Zsq4gUMIOYM5wOmTQ20g8FzCorpsxJu8McmTlx\\\\\\\/rP8TVWl45LsLQeQR2OnejXGG8E718fK2pqjbKOn+9LDrqN9OPEyHEI++dqEFiwgehhh4zy9gy59ibL1pvaSz2qjwbgxdcAVyLC4Lw0qPlKi9CltJNzLG+qKHZ09WXjsnWwzytcmll+Hej6tsbdY+VBoXnzJNQQi\\\\\\\/rQeax4Qj2iw1aKTlCdDZhJx4tbfsjc58Bi+xduEuLfqK7ojAYJFHHQ3RgifhCP8UcUCzkmmKH3ed7XWBX68YH1W3SHeYtywO3kdwpmnoT0oVPozKAvpdZ\\\\\\\/lmVtC7MXwVUfrhvlCAvgTvrxLu9j44lIfEGULxxzMS+XJHOHGRETnveym8WZZgjkp40wFJqlC2V5w6D4N8w8c1+yqCyxTOHxDI1VpoObCQLXRM2RbcGOP96ccytS4uizh5g5M9c8FwpvaD7KEfOQuBTlTmZOBQ7zesgbG0vQLgXb1uPlSXThfeuNe2VgoZ2RSMsiPAa4b5+xFGMg\\\\\\\/9EPtXpgqKOByfYuQ7n07GeqouVvsR\\\\\\\/FuOExD\\\\\\\/eXksCQh1LqRMvnlgrk1N8i3zqjze3TQaLubof6v+KROX4EaSYBDD6jcB58NFhh6Dl+FoE3PDQG6\\\\\\\/0qM1W+PAwNjiw7\\\\\\\/8HsFatVCddjS5SXL2dVu\\\\\\\/\\\\\\\/iVd5WNJVXTzb7o37kGmgRlbIpOlEi3\\\\\\\/wACStLDPjCa2B2s9ekVmG\\\\\\\/\\\\\\\/mRC178y2csZ153FlRzHccXYc2137H12ZXqRZDsJWdLArAGrEifRuXUHRpR3w8ne9+\\\\\\\/pvOXlCKxjqY4dIZj+VrHhoYdXjep\\\\\\\/Nt\\\\\\\/OMWv2W4xguqnXaG6\\\\\\\/U8tn5pImAG3hTw9FxMgmYtkD\\\\\\\/q\\\\\\\/UZRGKUcg8dBXNWQuZPz1B34Aqb08ZKrwk5URnLcIDDRyJc\\\\\\\/lZhncTBDzvtYz8zaii7uEBzS2kY6yDpBlCfNk4AP7ECvQwaasyClkQ1OihjinN6IUTdfmLuGcu0R0cY0cuDTD\\\\\\\/3M3h8GIzkZfkSOQiWb+yNn0NlT8qSrGXNtQn+HlkJzaOGkgPgIpP+BGLHk9EOsH6jioUqMmfiwgS71T9ecMhneFnLLw46vfYRQcKc86ZvkcPztQmfbUYQLMwmdtoVdo2WpSuyn0bSzEH6UKCSB3tn\\\\\\\/dcBzjwxDa1Tt8rb8N7wcG1VUXjlJEJA6qlujuTuTycfu+DBbZIoiUncNlp9WqHIN7675Sb4JWtNC3SWIvZuQEw9WI98aId8jL6C0aHL6Gb7ujeEms1tume\\\\\\\/q8\\\\\\\/OP\\\\\\\/M1lSGCYEiu7bOrUtURT9NocPX9Lkvd2JezYpInlq2MDceAc5B1VYnqEbMJvGHtq7U\\\\\\\/ARLy6s6KxhqJnLV4Nzdsas3yL6UNZbUG52FvW0ElOYB9IfL1eTRstcFmipKorQ04IwKDnCvx9F6P8034iK\\\\\\\/8ih+fp1kx+w2M+tmE9yZFEs6+u2a1HW4b2JHdoMVjiZvscYXMH0qW+AzRzH+dH3DGfQ1kVz9ZWVLGrT7g+Y+eLRxrdZPnV40CyTYcKtxEswGQ0uEc14Iu17Xg4KjCEDj7+iWaEijORewZwOCY0PJZGFpY3tJh\\\\\\\/GIliq6OBkoGr1+5as2A3OCtKqXmBmgHSkZ1MAawz+HgllZfcLW+UGLRBIAunCCJDbl+WU5Yjs\\\\\\\/vfV7IudlIadHNaodsF2gEGCiFEoN5jnuYsY2GN9rlrlP7BfRKczM54HOv+Ddz901+cGsUvL3zxpDVqxMghlZysLwMFbbLBNrptUfSvObHJs+EX99lrJfc\\\\\\\/vnsvWdzqNk\\\\\\\/2r88+Yn2e4djZV+CwizBW8HomZgeZBx6nJxIOoWMa421f7XIXwyZkflZ4DHHQLfFp7jrF81xu4BI+0d2DNeyRFE5u16cqd9ORrLplBPysjrwLPP7i8Km9P9CKcHIWxW3qePSdrx7ZmYGqZY5iPCEeeRYMgzjRQ6UWrj1GbCCNZTnak+q\\\\\\\/cIMwa+2WLi1sEweAfTfvBp8eRK\\\\\\\/aVqufqe9x2o7SvBav4zk8lqHBzLqrhsWwEHEJBL1wOdNCJu85TBEECLrko25jqna6InOaVnqUJLQfWEGLHdndADlxd5+DdIcBo4wgb5\\\\\\\/wwjxyV3LhN27t9\\\\\\\/z2Nekxw8Y0o+H9rb0DehRYK9ZMBMVpsydVXAtiYJi2ZQGZwAyy6w51U+ES479vLX2ZX6QyA700VXYug+3BTEC\\\\\\\/tjyqCFH3PYNi9LJAIODxvvFirXmDgfUm+TztC6qoceQ\\\\\\\/wtqRSokynTdratut7S7k+luywPIULfuf\\\\\\\/mvBzvLDbOxWxeT9MFRbWxE8uC7LrTzsdTh8SSz5mi+kGjqKH+FGKX8t8nsuv40zWBQe+Iu0RkXd9ovpSylFiN7XYE5Ary\\\\\\\/zLJxDgwXe\\\\\\\/eKNA7I2emnDeLnHU9RPs6CBk85qskLA+pLCBjKt\\\\\\\/VxaBk6C\\\\\\\/\\\\\\\/0InRHUuB9HM8g0BygR2YHzQ+f213MFklONekfAUmnC\\\\\\\/qVrTExZa6gjn6Nq5MG0IRFqKy0IsWg1r\\\\\\\/k9ZHDzTlPvoJLkvqoNALcVtYdVyn92QT0Jbsjhh5dpFxstyaeccFkD6c1tMn4TcasDigkOVq7l\\\\\\\/XmxFhwB6CWfdY6t4rE65qyC9z864Z\\\\\\\/vBMeSADczjysJpwWiZ7EbFQ+L0jylsBMAOJHsPIv1vlGgkzb\\\\\\\/\\\\\\\/A4nU7ArFX8BKtv\\\\\\\/HPBWTcoHKnXm2n925d6LQ0zbS5nvjVJg2rfNBEUIBcO53k\\\\\\\/gX2DKHhIOsH\\\\\\\/g2Y1JPanI0crtULHgzE\\\\\\\/b\\\\\\\/q1WkWCYhmRbvsSVj5YOM\\\\\\\/xWmOvpkpFqvwGZ+2hRH7xmhfCAevYFqyNgaGPnMg2Ldns\\\\\\\/cJ63WW5TtYn37i9iIPMM5qMQ1bIl3JI3SqYqT30PiVPT3QY1ZKW\\\\\\\/5lIf7HE8qQQDO3wGXtH583JVXVIVYQPiTQV1au3DQrVtdEMeYJSds1mKfhmPe6\\\\\\\/punKFjMqpv5b+pa8GxchJ5vVP64Edwx\\\\\\\/2dP1udh+eJZxfnsJHHBGSsgXEOF8bm98Ry5jvK7O\\\\\\\/yJVWcU2w8HdCEoVQ\\\\\\\/tKmIiMJmLcK5fy1tzM20lm9QvCIHOEccpF4ASRok+qIFDjX7+0SxpG5G8BRga1rmUPUMzv1TfO8RFpYtQWwMkDdn\\\\\\\/rWsi0lWsAmItL\\\\\\\/XS3fZI9915ko\\\\\\\/dptNczhpUJsfQ2s9P9rxCm2nM00MQvZfHhgKmRdfg4c8RnTqJXqVfTnvy32uNw1kAEVBeSFiUsADhve4aY4FN7aZi653gdNwJXoXURKSZ7suzhvPLn6P0r62\\\\\\\/0bxKwX0ohlPsBLMSjEwwjij5GcCWwRi8xjRJBKDo4n6g28eXI8ats2ZCw+OJRxqFhC5cGSez4PkPn0I2RIybNAuHvWOi1SOLes6Pqy\\\\\\\/215utHx90C\\\\\\\/Rnp31BFjc4qbuinaqNJCb3a7Zc9q8+7r8mZUlch\\\\\\\/Uvp7efN0n6rMqGo5sYft9B4GhgqAgOc4C+rAP0fWnkE\\\\\\\/4DdmEqC7S0JKjw6ttLrd5j4nIFEjr5NO+elhOuveF1+gkxyW+Tu1j7QeWDJQXooShA7TfLVPBynHGo15jvqJH8wtkCefht2vjh6fnG+N\\\\\\\/NzJodHZtWoXNMHaY19XHYjwTM7lejZsYxwNVcfVBd1FDXrhWlnnFb4hsQzrLC5My2xRU+blkPDGdyzLmNykoPSXKjVDKoR691wcn+3WfRahrfEKhmUYZnZtP6hOhJCq3ELPu8aZeqXXV5wKPAMnsoSTWTno4H5XAinjb9WlJXbwbdp+yCIAj4Ec9+QtmNlr2zq3RhCe4LqSK3RmEamH2o8CZVzt\\\\\\\/agT1OyOg3488W3ufrFxbbxWwre5oxZaVL+EXih9e1OL0kMJIGC4GiQqKWiD\\\\\\\/eHSddCtS4Pz8ZVLn\\\\\\\/w7PjZ+xCwOQpnbbALxVqoOxr5NQOIQSdlOdeRxDg+QNYpA0Mh3UI+8TO+AZla10QLBQPosnhV9bQ5AyQfhq7+VGdwbgAPVIw\\\\\\\/Z90kx47nYaLI+J30ja\\\\\\\/lJVCw8Z7AWhVrx26g4vth+sDVk5A+CJ0IAkWd5SNPZN9hk4JOzH9bGeucDe3odwLlvityXuehii5lY2j\\\\\\\/RvbdJzZRN9zfU+tOxcZV3DtBup2uWGM0pE8J9j3kNcKyuTdeGMO6\\\\\\\/R0\\\\\\\/o3fIbU1jSaidIfVHrWNo3OrYP\\\\\\\/2gxIP0q5s\\\\\\\/LGfw5WBUVju59w3bJ16Xbdb4XEiI0daHxkYaoElleguMRChTTeb74nJTuNPnP0U2b9m+FRYzxWys7QwLRmDoYG+n7v9BEnWPocy5jlEFG1NSROGJHhJn7FVZuTN07iqrK6yMSJvIXGjcuEmBJ9vtS7UdemX1KFJnWiYH4KpvEsYqlZKmD0YBWvWcrflp5ddVAWFFzYPHnI4pnaO86y0TXCTu\\\\\\\/UDxBH1T0T8i+zfbDFI4z2B8b6qGyuXLOUsgwchIsdtxBi1z7ggYRkspFGOLWK6gGw\\\\\\\/zlmpxZl0Y8znqyewlUGh7jxhwOBtK4mrud3OnvqCtRgC1nlaKVvihTITq05NCcZiA8zOnZwInxTPIUJYhdvvNJIsCUsp7yRFsCJLWEaZisUQmZSXwTtVgusxFnHnGe2aILmeja+xW57DVu4HKF2T9Vufr9rGxCtmdQaTnaWDffjs\\\\\\\/SzbScWxmiOojGrj4hR563Fi4Fr0lDkX93aTEV1+SaPga8\\\\\\\/r4IVfGlHqytJETJ0gBR3aE65zt0c3VXRhi0Dl\\\\\\\/sRGPAGVunJ4na0hWHk5WM+hiZOhYoJXqVifEktb\\\\\\\/Oe5WeGj8Y027UKx7U6JwUHYwcJ7qQ7A2+CNVi2jras+wvVtYigbRG5gDn0YLcEUUEthdKVR+CTNU0aCN+rsG7T9jaY8ueXjxl2Fiv+QTHWbogIA908Zno0gXIU3O34mGuaCC748HzdpEzh+kqt4GdERIE5ecxiA7dCEf1aYKknpmEqEM\\\\\\\/7MWYuj1XWO3SElu05O7bf1QD7GxkyCgn9V64p+3uGf2lqH2+i41GVuYbBVwxExL1fNEpZ6DpWiDh6MNYk14QLndo3tz0m\\\\\\\/LWM5ZaMldJY4wv+Wpnz50qPF9wUB5MlTlvdxQUHzuE\\\\\\\/ZbCGf8HlSGu4gZK2OIEfyuXthYpIFyWBLgC3T8ictCn1SpZoqNgh9a1o5OzKE2jpGfmCfx2GI7X5CppEYXBVCwdcRi4ZW01bjXHIwjQBrLdL8Qt5f13\\\\\\\/BkDWBtbbb15fc0Y2z3yQFnsHtwX03MF2E2iy+N5kL\\\\\\\/\\\\\\\/gyD5pUvh7vxIRWKqlaPfsd\\\\\\\/ybiL\\\\\\\/zcRLqnSkONS1lDdmWpuVp8mqRJXj9DPyIOa6G+mNR2FUVBx4Vc2yOS59wVBas\\\\\\\/7HVHvax72s91qXC7sJf3xlHQV3n3BpB8jPxFA8LKH8BvkAcM0FZvn1Pn80ZAkPndHZr5f0EWQL\\\\\\\/WfWUKeAsoIPOM5bGJtAEUmMLdDCv49qHk0VTh3DkZMIIdN0Vt2tgwAzj41RCD65e4q2+K2D3zGno\\\\\\\/5i5qni6TGh7+PBeITkf73oMqzIFFxD2pRekJ\\\\\\\/ic1y58UlnqWjcmGiy6sCrLFS294I3qI+eezbfetn1lRAQ4GwiKxBYiGZct6wvA2sLE1V5dkVoFNOorxG4gbUmzWWvkzHqrN95Df9Y1C+kFD4fMNCL8urhUqzL600AYCpe8ABJzWl2hRI2h7e3Lh+L2GL+4qRmAMFcGkHYSTxqFkS4Br7CjZSCcV2XFquGD0odxaTosg2aI6AW4jVGPVdNMEiV26XaSjS8TlakH+yjXUbQTv5551WhIEbju4U8orPb1jXk2GYzyaSq+WoLaUIEXHnY63YApAn0Ye71+bDFrJ+sdbzpoQ5FuUQTqdRtvp9efvthmQmC4HSBKL2HDin1ep9hnnYRo9vd+h016WPBGUWrSBcUwa+bfJYRe2gsn1CEtdcWWmGefuPKj8qwsjK9U9vOYqYfL4Qo9FRcEtYQGj0goZzxV3MaydNwHkrc8HhBrMIiB6vhqT7CrT3OmRLPiC\\\\\\\/Oaw8Z1tTRWlhYuEXb06tdrEGDeZ7\\\\\\\/uVpxWb94ca3jqb20CTL1mmfOEd7ldrCm7xPwck00NvXaWg4HC6usnMTECC7jb2aJcChJBh9u9geUkUxunRnkE+uxFxvjtOQitSgBbR94TOCIR6Dc4+OsPxudheMorZXUiGsuhrUFOztxUhyA0Bt82SY1Pwqpk0fB5J1EBe3WRrvtFC+qGQkvUgzXso1Kwo5q9jtKkxyKNVzJtfc3dWMvnPlgPKk3iEacX9vsWdyVInYEjlayfTczYrphnXYgw257VvzNXoSLZ0ZXqFAtfuBn6hu3FJ1SqltPUR5mMgCZn\\\\\\\/xBG1TwgzcXM6bFlC\\\\\\\/+LF+n1\\\\\\\/mk\\\\\\\/Cm\\\\\\\/rZgyPhXtwV5Ms41XVmgkrbqwTkf3lk4r+M6eYR16EeXq\\\\\\\/rWGMTEC4G4lsoPMyNW5\\\\\\\/DM\\\\\\\/60GeXheudxRki4kU\\\\\\\/\\\\\\\/aT0xct5+108bY7Xd9yiXr6UmHbaxNPTAjpqqqCPjbGxsIC56Vq5wr6D4Vf+Z4QYq3+yb61FIzi88MPzuvkyZhACugc1T0P+PDi0Gem0Pw3badhVxuQAOfd4VSy+A89wn\\\\\\\/sg77GTo+0eZsqn3RctkE91AnwWwWTAoOmmcUiHG6nowMP\\\\\\\/Bndn+EDFyCxc3F+gBCf6QOQKFp0PaIYuc2xpi64mgKjXjBrabdtRy46tev8jxYAh0mN6bHaS0jPDZGmnlDP6Q29D5z7RctL0dblzMJtCwRz+7sA4QHU6E42LP3DSZS\\\\\\\/7cS7taW74PGWrFcGkekGTIG4K7LLvprA++Z++6oTZ0QU0c7oGYYHiOsqODPwZGE5yyGIO3J1bsVdjGNHPpcJrbCNseRZmboQCkcDFsoJn+drEqRxqXSpprle0RerJJ+0oLRaMffM4mfLC4MASzYRXNYOyv2dnbtBB+L4eGm6VOc6IGAoVA1sjJdvqijBvvEsRsNYE6Yt2r1pkRBzgu+rglxdzO9V3\\\\\\\/fy7hSQeZcPZ1yUuHWCK1Kw+URDLge9i+sSfHuzCqWqxu0i+ToL4LrcWWkuTLdPQnhh+UTlltYRaGh6dihb8UCAwzEctMUp7pOX63adgCXXf0JNbeK4YjxkcYO15fwDqfwg8EhpvVM8CBNItzhYIW6XN7HulvqHchwb69kpGUXXPPIFvBHQRVqPGG0rj\\\\\\\/VE3al6lNeGX+a8na\\\\\\\/\\\\\\\/EVktL6TQzxM5yKrZSSqfoTDk6JmMcxxOT100g65ZowIoK1flWltix2wAjdBfNJLHYbd5seZmSU5ecY2Y8O\\\\\\\/H1I0w020S7BREzpMHEY02bLXJ2Qsq5XisORzTa6xeUjHEoC6wFlTIOfdOBYs\\\\\\\/o01zg8OChxPWrsFTUvTVlUCeKtMa465xTOe\\\\\\\/NPpC+dmgf\\\\\\\/N9R5UooevbR9ExXzC4vBuEszm9010fXcWp1LEpdS51mtAj6Tnyf9AZYTm6pdu\\\\\\\/7tNIH5xSe6eakVZqEYNSqUv7Sqttbq6nv4DNL6Gqd4aTMjGCN0StZEI2Zpaxu4Oa33TmwRI3EFKjDqYJLJWt9zLEEIU7yHAcbJGjKm\\\\\\\/s2yuZj4+Swh5nM6jyE2M1yZw7Rm1b5VM1ZQu9HL7\\\\\\\/ZcFKwhnYw2Lik33Bz7T5EgczzFQtFftLu0vsES8qt7\\\\\\\/nXhC8UejhFdwinmyP390eVc\\\\\\\/1ECQW3mntxMkk+yh11iTBn8Sgw9SNFTs3ch9rTCN0mm00Mx454MHgOE9L3A8tHQRAvQgMBEQxp81J60Q3pKj15utsUtI2T\\\\\\\/WquPYYeA5PVsnq5TzlGMVGWv\\\\\\\/+BVfQZeVKlk1kY0BsQABspL49VAejvX7F7KpH+ibP\\\\\\\/VQNSfpLaR5FyqBmS3aRgAM9T2P+rQ1+kW5eM+Q3txVqHHRvNCOobqHdFQ+UwBI\\\\\\\/mRhJwS4shB8IT5CeA7m1jsTrIcVf8qQuznhlfUKJV6XUyGoypROZ2H4XOjd5cQ1R3CGOz\\\\\\\/HqWpvifAv\\\\\\\/4br7wgOOpi+SBRgl8FiEYF\\\\\\\/ASN1QsqDMGRbjbH1VglRfT8s4h5SrEQH\\\\\\\/nZPs0Nm\\\\\\\/9W1mr+3rJclxrv640v+6GviDCY8dU0bob6P+qMMaNvseHm3\\\\\\\/HCi+pZQjYlnLyyCMJOZwWZkWU1qLOjBQqANU5oi7d\\\\\\\/nLpCxUHt8yiR3NrKRxC882nsm0Gx6eitkrUQp0PQ1zAYpsabXYRJX5sd9JiDZDwfOsnE1aTdcpPBpAxPw0mfAzntCm0BrQ2v65M6t9KImlBqfqtKtzoe0pu8mH810j\\\\\\\/z4xfb3RwwTZW2LnM2DgpAjtHOfNfKi\\\\\\\/zlLpn3WoVzYVt+oLTjDliLD\\\\\\\/OCDO24\\\\\\\/HcK9JafwBrAU0p1naDFLpjf3V0Pg86zGCInIFVWTz4pqN\\\\\\\/37mlLFsius9jE3Q7I1RWVDIdQuqR3bBnFOSHT6xI0njFX0Mpq77xGuDUT9GALIZkhNDKqHzIp7FYAz6C6e9U6E2w\\\\\\\/h0QbLm2SJKIXMuMBCd6o8c91LzRvvhjSqPOCSV05hMO5pJ+pPfp\\\\\\\/Fgzh2zKFT3nKD5A0WmstPH3qFMvSN09ANDRQzCVvvXf01p+VjMUQbHiVUMcN1IRv0xD1jNxzDRgJROpFlnKAKsBxpZBcbpxbZRVbGPSZk62USi5NCLaNWePDsW9hMPWQF4XHGCaNF4j4YyF4mreSCkemY5UFfEEy4BHmBRGIN+uPeEOznvY0XN8EKAh5GhM0UVetvNvzP+T0b1qRFJyfoz+BcGLkprMwz6Ee2+1bdYHA1pSmWURnw6mk2qzh1\\\\\\\/YfHUzylPM+9X1ttVgK7dHb0UQxWJBnA9OI8Zm+CbKKWUm3379lRnqn8Li7LEkOkyxBOSKmFHzd7T1A91tHUY6HvnzXJtFb75gEIwqWg8u0oL6xasica3YUmPdxA4U9Fk2QiMbYGX8clAr+sGkuHRuc0yCal8v2ZhABrperE9KAGG2d7LvCeU0kh7MKtF+YDfJG5uALiIJGuS0MXEaNS4mK6v4AkJtyfV6nG6TO6iDPJgFS7896ArVc1YUMvGi+W6lxwdDdlATYr3cwDQsh9AxRG26RyjspcdOJMTuslGH70WG1xqpMH7P7dR2OhK3gqsCZE3p\\\\\\\/JnQesThKrrNHw0oKsai1uIWvKO\\\\\\\/toKZLq8S0cEKhmZS0dlCxcY4+lLkUb0ylo0mB0ukWLTElKH9lTbJndJxGpOcTbVKNtCkZjE3AAxdXP1DsiUX0nPIxr2Na9UypHSzL91mI\\\\\\\/3hAvFDahn\\\\\\\/oqqbEXhzcjFr5dKbCLvEBuP1V\\\\\\\/MAlKRtgtx3LouUEfVE92U4MF2xAbb3XscGdnqkqHauUwJNLkF6K7nIY7D7jHdsJIrR9\\\\\\\/VNjt3u2XA+bpK3O+EX8v+HHsp7qzPCwoFMAhA0KTOofpVwUqGdR\\\\\\\/Y3gfiZwRudjjsQtUgHO9CqmOOKbb2uLNMfpA6HEDqfdoLRiZUH6g\\\\\\\/c35\\\\\\\/mvP1mei7mkc2SXIFkhZ5AJAZs4LRT0gplufbep9cj235uDLg28PbLyuT6TNqG8i+90s2heFAGVVswI09kp7PBDmTi6P0xqd4qpCe5\\\\\\\/aHKmb52oHaamzNLDRkN3JDQ5jayyDYLISl0xnyr8z\\\\\\\/P1znjyLFzW3v6taOQkaW2MQ6NmWlPCx4fjC4sIYX+L3sHlNPKOhmwVn99MVwctd\\\\\\\/lQsCIGpFRv2qW0e7Y\\\u0022,\\\u0022iv\\\u0022:\\\u0022bzMbDnhm3r0bYk9rr4kwfQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222b052f5908e670f91f1ee9f1270ddc39908330a82ecb3a236df12d20779c8f89\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kU8CObsyVzFjncxwk6yVwWKWL3U\\\\\\\/e\\\\\\\/dYa+ilcVJH3GR4JJN+ILS1yXwIO7qzGMGBH5kSxZb9BYnWlNYqyKy63zst1X6iQfo49G\\\\\\\/RWXICMykWM7weyCLVRM9A7y8w8O7+xF3qru7mlu6k1DkWjRMheFvAnhkvETrdMVX1WxNFtR7KK9s9cbDoisLnJyh6yHf9ernLwfGGey\\\\\\\/HDqYRiVD5mzbBQNf6WWjgdPfLPrZtsh3fj0zprPVsd0JnMLPLVlVnGogZrqO\\\\\\\/cyUiWGLa6WOiRjKrY4cToWRjv6Uvlt1A2oY5cJXsHk9M5\\\\\\\/i6BL9x9sZ4vO9IpZWDaRlfrunbzfLrb\\\\\\\/4ycS7e8E6jY3bDm6me5PXOVJmRdZeT1RsUQJp1qbITAqFQ8Mb4XVG4TGC9\\\\\\\/l2oeEyglZNd7TKfxx5ElbtvkBBuPb\\\\\\\/J1BwVzxCnZDBqiJoXnnXJG26od+9xPcOgyPmAi4zUntigxLjBe683zk5GRuA8EfoMr9qQw3iR2pt3TXhS6aL5cTZe\\\\\\\/EjNgrYXzf3+p9c58pr4X7I4mXWB6R9azhO7SaKmbmoMtFK2FWaY01cgnFwL6EQjBiQmFyt8lAXUMRc3ouYYzzY4yTaOJY3Ym557ZcgpiKENqWZJcWny0cHOrVqHFfM7cEHvX5Mba42KoqbB7dP7D0nV71YHkf8+t4GnOIwK2U3UfuBVYYjvhDti9wrHdHNpiIaOymCVSlMmqJ4I7G+AujD08y\\\\\\\/maHnAXfDpHHrwi58iJV\\\\\\\/kiFxYBxFd14zmdZPmeo+J7VdOYB1BF4kRcDEIi3HaEZs5QirwLKOFuaPMtgTvc8tyoNOrVQh1pIPUAtRza+4jvJ7q5gcGsxKEbeXdb0ie2O9jBWBG7JGWv5zMLRCvM1Uy4qqfXBAFpOFENs6UQP576cwcXRyfrYlWt6mvI\\\\\\\/6UYLLuA8f2bpVbpebxFLSNc0Ch1E861xV1BGZZfAEOIbb8V7KIhjeTWRDZYFWX3wFr8340ApJ\\\\\\\/opnVzhgUA3CxqXsMI7yrwaAz3cy8aBcM2NtBINUSYhIRdhMuGMOyubkk4ZFQyV6iWOSuXe0jqJny08J4zUjZATWMJw1C84U+qKtSvInf4sju5gAw61Y6evWk7foC8R03l2turyewAj2TggliEzbzYyNfVOdqzvUzDsG7gmvjZ5ef0gexyM41o+5IB1F2OJxrPgRvdYV4RPWBjGCFm6yZq8vznS5oKqo0OO\\\\\\\/3fWyCG+Eg0xHV+RGg1RqRXyLdt0+8aOLAgw2MTCv6FRm3O5dIp\\\\\\\/WmMHXXjPeSPEOy5\\\\\\\/yzYa0\\\\\\\/dyrbEZSW1NOwNuK3nAc2ikzioLyu3iuqPitsTeM6V86NJzXcWoVdIhnqrlcXkDM6WLC0RNgLUzLIn4MBIvwZHY7Hh1aXHDM0J6UCnsGLgAUKHzscIGy1DNpHpGK2BXHdrXGwT\\\\\\\/8T8P+85A+nU73SEEj1ceYQmSeUtTkfxIhjNTkZQNr9yYo1MCw5kqgNFtw67jq02cuSXnQVjMwUu44Kw4H9NZm6yjyPJmn4vFGvP5u+CMJl2\\\\\\\/2mu9FuA4j65xYHizCBwM1xWvnUqLBhflUbWKNPVnU5aN5ppP1R9OQnGPIdjyhUIRLU+DU4UN+A7nX91uwCSQYutKi5R5G+ED7nL7CL0ixzPBUBtLPOkFNINPocbQDbXXVnejM+NWu7qxrE3LnXjLcymW8NlOYe1kJug\\\\\\\/bXNx2HYy+kP14\\\\\\\/uaoBPqwxAZBVJy3mHVEYJ\\\\\\\/Q+OFnqGnt8p\\\\\\\/Co2qIlG\\\\\\\/J1Tept9FEE8BFNwoIrbQddIZL2BTnhyIHzaxVt4Q1FuIuMft+B4lFEuhvhLHJXsG2BsOaFdLM4+8kb7ypZWvPSgbAhbyLkF8aJXZ+jShgLAmISKkRiPM2RcdYFbUlbEfE8VQdsWj0Gqt35kWyujggtZnG7oKUl+h8gRLjp4f75jP47AzYlk4IfSWZleYwnSGVIxttda36RWBjJxtWpwt9thLpVH7mmUYgNhLFfbewQjGmkGPQWEzO2PlOEROPWYTz+WMJpqP0=\\\u0022,\\\u0022iv\\\u0022:\\\u0022bN65KsdY2+DdIw59QvCTEg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002215255d9b06668befa77a13af3960b5828acce1ce0d9ab4ea185e37955a47d7bc\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460295 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2881" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SxHSuHjiAQuyzSh6xFiBbSyUO5OW3TxTSfbKIMJAiyM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2881" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XUoQDtyC8ll-pP0RJU5nYCd-WELLNmjuPloVNYNRGOc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2867" title="Genome-Wide Divergence in the West-African Malaria Vector Anopheles melas" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2893" title="Identification of Genes in Candida glabrata Conferring Altered Responses to Caspofungin, a Cell Wall Synthesis Inhibitor" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460295" id="node460295" data-pisa="ggg;6/9/2881" data-pisa-master="ggg;g3.116.030338" data-apath="/ggg/6/9/2881.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9896-1739" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew D. L. Nelson</span>, <span class="highwire-citation-author" data-delta="1">Evan S. Forsythe</span>, <span class="highwire-citation-author" data-delta="2">Upendra K. Devisetty</span>, <span class="highwire-citation-author" data-delta="3">David S. Clausen</span>, <span class="highwire-citation-author" data-delta="4">Asher K. Haug-Batzell</span>, <span class="highwire-citation-author" data-delta="5">Ari M. R. Meldrum</span>, <span class="highwire-citation-author" data-delta="6">Michael R. Frank</span>, <span class="highwire-citation-author" data-delta="7">Eric Lyons</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-3392-1389" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark A. Beilstein</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2881-2891;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030338 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460295" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew D. L. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.%2BL.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BD.%2BL.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9896-1739" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew D. L. Nelson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andrewnelson@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">andrewnelson@email.arizona.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Evan S. Forsythe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BS.%2BForsythe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forsythe%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BS.%2BForsythe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Upendra K. Devisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Ecosystems and Society, Oregon State University, Corvalis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Upendra%2BK.%2BDevisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devisetty%20UK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUpendra%2BK.%2BDevisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David S. Clausen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BClausen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clausen%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BS.%2BClausen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Asher K. Haug-Batzell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics Graduate Interdisciplinary Group, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asher%2BK.%2BHaug-Batzell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haug-Batzell%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAsher%2BK.%2BHaug-Batzell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ari M. R. Meldrum </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ari%2BM.%2BR.%2BMeldrum%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meldrum%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAri%2BM.%2BR.%2BMeldrum%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael R. Frank </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BFrank%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Frank%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BFrank%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Eric Lyons </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics Graduate Interdisciplinary Group, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BLyons%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lyons%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BLyons%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Mark A. Beilstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BBeilstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beilstein%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BA.%2BBeilstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3392-1389" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark A. Beilstein</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andrewnelson@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">andrewnelson@email.arizona.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/9/2881" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460295" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2881.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460295" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2881.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460295" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2881.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460295" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/9/2881.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460295" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Transcriptomic analyses from across eukaryotes indicate that most of the genome is transcribed at some point in the developmental trajectory of an organism. One class of these transcripts is termed long intergenic noncoding RNAs (lincRNAs). Recently, attention has focused on understanding the evolutionary dynamics of lincRNAs, particularly their conservation within genomes. Here, we take a comparative genomic and phylogenetic approach to uncover factors influencing lincRNA emergence and persistence in the plant family Brassicaceae, to which <em>Arabidopsis thaliana</em> belongs. We searched 10 genomes across the family for evidence of &gt; 5000 lincRNA loci from <em>A. thaliana</em>. From loci conserved in the genomes of multiple species, we built alignments and inferred phylogeny. We then used gene tree/species tree reconciliation to examine the duplication history and timing of emergence of these loci. Emergence of lincRNA loci appears to be linked to local duplication events, but, surprisingly, not whole genome duplication events (WGD), or transposable elements. Interestingly, WGD events are associated with the loss of loci for species having undergone relatively recent polyploidy. Lastly, we identify 1180 loci of the 6480 previously annotated <em>A. thaliana</em> lincRNAs (18%) with elevated levels of conservation. These conserved lincRNAs show higher expression, and are enriched for stress-responsiveness and <em>cis</em>-regulatory motifs known as conserved noncoding sequences (CNSs). These data highlight potential functional pathways and suggest that CNSs may regulate neighboring genes at both the genomic and transcriptomic level. In sum, we provide insight into processes that may influence lincRNA diversification by providing an evolutionary context for previously annotated lincRNAs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/lincrna" class="hw-term hw-article-keyword hw-article-keyword-lincrna" rel="nofollow">lincRNA</a></li><li class="kwd"><a href="/keyword/brassicaceae" class="hw-term hw-article-keyword hw-article-keyword-brassicaceae" rel="nofollow">Brassicaceae</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">evolution</a></li><li class="kwd"><a href="/keyword/transcriptomics" class="hw-term hw-article-keyword hw-article-keyword-transcriptomics" rel="nofollow">transcriptomics</a></li></ul><p id="p-3">Long noncoding RNAs (lncRNAs) are defined as transcripts that are &gt; 200 nt in length but are not predicted to encode polypeptides of &gt; 100 amino acids (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> 2012</a>). Reported lncRNA repertoires in mammals vary, but are commonly in the thousands to tens of thousands of transcripts, accounting for ∼90% of the genome (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Derrien <em>et al.</em> 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cabili <em>et al.</em> 2011</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Stamatoyannopoulos <em>et al.</em> 2012</a>). The biological roles of a few lncRNAs, such as the telomerase RNA (TER), COOLAIR, Xist, and MALAT1 are well characterized (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blackburn and Collins 2011</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pontier and Gribnau 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gutschner <em>et al.</em> 2013</a>). These RNAs function in genome maintenance, chromosome silencing, stress response, and alternative splicing, respectively. Despite these key examples and the prevalence of lncRNAs within genomes, functional data for the majority of lncRNAs are lacking.</p><p id="p-4">Much of what we know about lncRNAs is derived from extensive next-generation sequencing in mammalian systems. On average, mammalian lncRNAs are transcribed at ∼10-fold lower levels than protein-coding genes (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cabili <em>et al.</em> 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Managadze <em>et al.</em> 2011</a>). In addition, a majority of lncRNAs in mice and humans are tissue specific, with many lncRNAs restricted to the brain, liver, or testes (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Necsulea <em>et al.</em> 2014</a>). LncRNAs are processed similarly to mRNAs: they are transcribed predominantly by Pol II, capped, polyadenylated, and composed of multiple exons (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ponting <em>et al.</em> 2009</a>). Moreover, lncRNA loci exhibit epigenetic marks associated with active chromatin (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Cabili <em>et al.</em> 2011</a>).</p><p id="p-5">lncRNAs are often categorized based on the genomic context from which they are transcribed. Some lncRNAs are embedded within, or overlap with, protein-coding genes (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Ponting <em>et al.</em> 2009</a>). These lncRNAs are further classified into different categories based on directionality of overlap, and the degree to which transcription varies from the related protein-coding gene. Overlapping lncRNAs can serve as key regulators of the genes to which they are linked (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wang and Chang 2011</a>). For example, a subset of lncRNAs that overlap a protein-coding gene in the antisense direction function as <em>cis</em>-natural antisense transcripts (<em>cis</em>-NATs) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lapidot and Pilpel 2006</a>). A specific subgroup of lncRNAs originate in intergenic regions, and are referred to as long intergenic noncoding RNAs (lincRNAs). LincRNAs are autonomous transcriptional units, in that their transcription does not appear to be dependent on that of adjacent genes (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Cabili <em>et al.</em> 2011</a>), and thus these molecules may function in molecular pathways independent of neighboring genes (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Ulitsky and Bartel 2013</a>). Categorizing lincRNAs based on functional characteristics remains a challenge. We will focus specifically on the intergenic class of lncRNAs in this manuscript.</p><p id="p-6">Recent comparative analyses in mammals have demonstrated that lncRNA populations display poor genomic and transcriptomic conservation relative to protein-coding genes (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Necsulea <em>et al.</em> 2014</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hezroni <em>et al.</em> 2015</a>). Lack of conservation is derived in part from relaxation of constraint on nucleotide evolution (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ponjavic <em>et al.</em> 2007</a>). A relatively large proportion of lncRNAs are species-specific (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hezroni <em>et al.</em> 2015</a>), suggesting lack of constraint on nucleotide evolution is not the only factor leading to diversification. However, the factors affecting the emergence of new lncRNAs are not well understood.</p><p id="p-7">While the origins of most lncRNAs are unknown, three scenarios have been proposed for emergence of new lncRNA loci (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Ulitsky and Bartel 2013</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Ponting <em>et al.</em> 2009</a>): pseudogenization, gene duplication, or <em>de novo</em> transcription from a previously silent locus. Although they make up a small portion of the overall number of mammalian lncRNAs, there is ample evidence for the role of pseudogenization in the emergence of lncRNAs (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Ulitsky and Bartel 2013</a>). Pseudogenized loci often remain transcriptionally active, albeit at lower levels, and are, by definition, noncoding (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Pink <em>et al.</em> 2011</a>). The role of gene duplication in lncRNA emergence is less clear. Most lncRNAs appear to be single copy in vertebrates, but these inferences are based on presence or absence of similar sequences among related species (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Ulitsky <em>et al.</em> 2011</a>), rather than using a phylogenetic approach to infer duplication history. Most lncRNAs appear to emerge <em>de novo</em>, and transposable elements (TEs) may play a key role in this emergence. Compared with protein-coding genes, TE-derived repetitive sequences are more prevalent in mammalian lncRNAs; they account for 30% of total lncRNA sequence in humans (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kapusta <em>et al.</em> 2013</a>). While there is evidence to suggest that TEs contribute to sequence diversification of lncRNA loci, it is unclear if TEs drive the emergence of novel lncRNAs.</p><p id="p-8">A subset of lncRNAs display lower rates of evolution, presumably due to conservation of function. Examples of conservation of synteny, sequence, structure, or gene organization are seen in the lncRNAs TER, Xist, and COOLAIR (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Wang and Chang 2011</a>; <a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">Ulitsky and Bartel 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Castaings <em>et al.</em> 2014</a>). The telomerase RNA, TER, an essential lncRNA that participates in genome maintenance, displays conservation of sequence and synteny within major eukaryotic clades, and major structural elements tied to function are conserved among fungi, ciliates, and vertebrates (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Xiaodong Qi <em>et al.</em> 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2000</a>). Xist is a eutherian lncRNA that is responsible for X-chromosome inactivation. A lncRNA with overall poor sequence conservation, <em>Xist</em> loci are conserved syntenically in eutherians in functional repeat units (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Elisaphenko <em>et al.</em> 2008</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Duret <em>et al.</em> 2006</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Romito and Rougeulle 2011</a>). COOLAIR is a lncRNA involved in regulating flowering in response to temperature in the plant family Brassicaceae (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Castaings <em>et al.</em> 2014</a>). COOLAIR is syntenic within sampled Brassicaceae, and functionally important domains are conserved. Thus, as with protein-coding genes, function likely constrains sequence and positional evolution for a subset of lncRNAs.</p><p id="p-9">In plants, lincRNA datasets have been inferred from transcriptome data for <em>Arabidopsis thaliana</em>, <em>Populus trichocarpa</em>, and <em>Zea mays</em>, among others (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> 2012</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Shuai <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2014</a>). The most comprehensive lincRNA annotation exists for <em>A. thaliana</em>, where a detailed analysis of 200 tiling arrays and numerous RNA-seq datasets uncovered 13,230 intergenic transcripts, of which 6480 were classified as lincRNAs (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> 2012</a>). Similar to their mammalian counterparts, <em>A. thaliana</em> lincRNAs (AtlincRNAs) are processed like mRNAs, expressed at low levels, and a subset display tissue-specificity. Homology searches in poplar and grape yielded hits for &lt; 1% of AtlincRNAs, suggesting they may be conserved at lower rates than mammals. In plants, genomes separated by ≥ 100 million yr of evolution [for example, Arabidopsis and poplar diverged ∼100 million yr ago (Mya) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Magallón <em>et al.</em> 2015</a>)], appear unlikely to yield comparative data useful for distinguishing between conserved and species-specific lincRNAs. Fortunately, <em>A. thaliana</em> is a member of the plant family Brassicaceae, which arose ∼54 Mya (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beilstein <em>et al.</em> 2010</a>), and for which a wealth of genomic and transcriptomic data are publicly available. As a result, the family is ideal for evolutionary comparisons, and thus provides a framework to infer factors influencing lnc/lincRNA diversification more broadly.</p><p id="p-10">We present an evolutionary and comparative genomic analysis of &gt; 5000 lincRNAs in <em>A. thaliana</em> and its relatives within Brassicaceae spanning 54 million yr of divergence. For our comparative analyses, we used genome data from 10 species within the Brassicaceae plus <em>Tarenaya hassleriana</em>, a member of the sister lineage Cleomaceae (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beilstein <em>et al.</em> 2006</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cheng <em>et al.</em> 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hall <em>et al.</em> 2002</a>). Other studies have used lnc/lincRNAs as characters projected at the tips of an organismal tree (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Necsulea <em>et al.</em> 2014</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hezroni <em>et al.</em> 2015</a>). While trees used in this way are powerful tools for inferring evolutionary patterns, here we take an explicitly phylogenetic approach to understand the dynamics of lincRNA evolution. Using sequence similarity, we reconstructed families of homologous lincRNA loci, aligned the constituent sequences, built gene trees, and used gene tree/species tree reconciliation to infer evolutionary processes. The advantage of this method is that it allows us to investigate factors affecting lincRNA emergence and decay. Our results indicate that small-scale duplication events impact lincRNA emergence more than whole genome duplication (WGD) events or activity of TEs. WGD events appear to have propelled the loss of putative lincRNA loci relative to protein-coding genes. In addition, we identified a subset of AtlincRNAs that are conserved across the sampled Brassicaceae genomes. These conserved AtlincRNAs are more likely to be stress-responsive and enriched for <em>cis</em>-regulatory elements, suggestive of both a function, and a reason for conservation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation for identification, clustering, and phylogenetic analysis of AtlincRNAs and their homologous loci. (A) Species analyzed within Brassicaceae. A chronogram of the Brassicaceae species, and outgroup T. hassleriana, used in this study. Lineages&#xA0;I and II are indicated in red. Number of homologous AtlincRNA loci detected in each species shown. (B) General scheme for identifying AtlincRNA sequence homologs in other species. The Liu et al. (2012) lincRNA dataset (dark green box denoted by &#x201C;Q&#x201D;) were used as the query in a reciprocal BLAST of genomes (round colored circles). Overlap between identified sequence homologs and known gene datasets (colored triangles) was determined for annotation purposes. Homologous sequences, along with available annotations, were extracted and aligned. Colored lines represent sequences, and are color coded to match the genomes from which they were extracted. (C) Phylogenetic inferences of conserved AtlincRNA families. Currently accepted species relationships, with two lineages, indicated (center). A red asterisk represents the last common ancestor of lineage I and II species. Aligned AtlincRNA families were filtered according to the conservation criteria shown. The number of conserved AtlincRNA families (all species combined) passing through each phylogenetic analysis step is listed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-271616370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic representation for identification, clustering, and phylogenetic analysis of AtlincRNAs and their homologous loci. (A) Species analyzed within Brassicaceae. A chronogram of the Brassicaceae species, and outgroup &lt;em&gt;T. hassleriana&lt;/em&gt;, used in this study. Lineages&#xA0;I and II are indicated in red. Number of homologous AtlincRNA loci detected in each species shown. (B) General scheme for identifying AtlincRNA sequence homologs in other species. The Liu &lt;em&gt;et al.&lt;/em&gt; (2012) lincRNA dataset (dark green box denoted by &#x201C;Q&#x201D;) were used as the query in a reciprocal BLAST of genomes (round colored circles). Overlap between identified sequence homologs and known gene datasets (colored triangles) was determined for annotation purposes. Homologous sequences, along with available annotations, were extracted and aligned. Colored lines represent sequences, and are color coded to match the genomes from which they were extracted. (C) Phylogenetic inferences of conserved AtlincRNA families. Currently accepted species relationships, with two lineages, indicated (center). A red asterisk represents the last common ancestor of lineage I and II species. Aligned AtlincRNA families were filtered according to the conservation criteria shown. The number of conserved AtlincRNA families (all species combined) passing through each phylogenetic analysis step is listed.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2881/F1.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/9/2881/F1.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469189" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Schematic representation for identification, clustering, and phylogenetic analysis of AtlincRNAs and their homologous loci. (A) Species analyzed within Brassicaceae. A chronogram of the Brassicaceae species, and outgroup <em>T. hassleriana</em>, used in this study. Lineages I and II are indicated in red. Number of homologous AtlincRNA loci detected in each species shown. (B) General scheme for identifying AtlincRNA sequence homologs in other species. The <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> (2012)</a> lincRNA dataset (dark green box denoted by “Q”) were used as the query in a reciprocal BLAST of genomes (round colored circles). Overlap between identified sequence homologs and known gene datasets (colored triangles) was determined for annotation purposes. Homologous sequences, along with available annotations, were extracted and aligned. Colored lines represent sequences, and are color coded to match the genomes from which they were extracted. (C) Phylogenetic inferences of conserved AtlincRNA families. Currently accepted species relationships, with two lineages, indicated (center). A red asterisk represents the last common ancestor of lineage I and II species. Aligned AtlincRNA families were filtered according to the conservation criteria shown. The number of conserved AtlincRNA families (all species combined) passing through each phylogenetic analysis step is listed.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Identification of orthologous AtlincRNA loci in Brassicaceae</h3><p id="p-12">AtlincRNAs were used as a query in a BLAST (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1990</a>) against the genomes of 10 Brassicaceae and one outgroup (<em>T. hassleriana</em>), using the following parameters: (penalty –2, reward 1, gapopen 5, gapextend 2, wordsize 8, evalue 1e–20). All genomes are listed in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS1.xlsx">File S1</a>. Close hits (those closer together than the original size of the query lncRNA) were merged, top blast hits from each species designated, and then FASTA sequences extracted for each hit. The adjacent protein-coding genes on either side of the lincRNA (or a 5 kb region if protein-coding genes were lacking) were used in a separate series of reciprocal BLASTs to determine if the top lincRNA hits from each species were syntenic as well as sequence similar. Only the top BLAST hit for each lincRNA in each genome was analyzed for synteny. In addition, the top BLAST hit was used as query in a separate reciprocal BLAST against the original query genome to determine reciprocity. Only top BLAST hits that were syntenic and reciprocal were denoted as sequence homologs. Each sequence name includes subject species name, query lncRNA name, and species, followed by a unique identifier. Any hits that overlapped with a known gene had that gene ID appended to their ID. Sequences for each hit were extracted from the appropriate genome and clustered together into a family with an ID corresponding to the query. Similar parameters were used with a dataset of 10,000 human lncRNAs from the LNCipedia.org (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Volders <em>et al.</em> 2013</a>) dataset (version 3.1) for identifying orthologous loci in the genomes of chimp, orangutan, and mouse. Alignments were performed using MAFFT (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Katoh and Standley 2013</a>) from the command-line using standard parameters. These alignments were used for downstream phylogenetic analyses. RNAcode (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Washietl <em>et al.</em> 2011</a>) was performed using standard parameters on alignments that contained at least four taxa. All lincRNA families identified with a query lincRNA containing a small ORF were removed from our analysis.</p></div><div id="sec-3" class="subsection"><h3>Calculating transposable element content in lncRNAs</h3><p id="p-13">To determine TE content in lncRNAs, we masked the Arabidopsis genome using RepeatMasker (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Smit <em>et al.</em> 2015</a>). The Arabidopsis repeat database was acquired from RepBase (Genetic Information Research Institute). In addition to sequences present in RepBase, we added sequence for known transposable elements found in the TAIR10 annotation. RepeatMasker, and all dependencies were run according to parameters previously used in mammals (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Kapusta <em>et al.</em> 2013</a>). Exonic and intronic sequence was used for calculating TE overlap with protein-coding and lincRNA loci, with the exception of 5′ and 3′ UTRs for protein-coding genes.</p></div><div id="sec-4" class="subsection"><h3>Conservation of expression and structure</h3><p id="p-14">Correlating conservation to RNA-seq FPKM (fragments per kilobase of transcript per million mapped reads) values was performed using values reported by <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> (2012)</a>. <em>A. thaliana</em> lincRNAs were binned according to the phylogenetic depth to which they were conserved in the family. Minimum free energies were calculated for each of these lincRNAs using RNAfold (Vienna Package 2.0; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lorenz <em>et al.</em> 2011</a>) in high-throughput. As these lincRNAs varied in length substantially, for direct comparison, an average MFE was calculated by dividing the MFE by the length of the lncRNA.</p></div><div id="sec-5" class="subsection"><h3>Inferring lincRNA loss and decay</h3><p id="p-15">For the conserved AtlincRNA families with missing loci in another species, reciprocal BLASTN was rerun on this species’ genome using a less stringent 1e–5 cutoff value. Additionally, BLASTN was performed in the same genome using protein-coding genes adjacent to the AtlincRNA using the 1e–5 value. This lower E-value was used to account for potential decay of adjacent protein-coding genes, such as in the mesopolyploid species. Genomic coordinates for all lincRNA BLAST returns at the 10<sup>−5</sup> threshold were compared to the coordinates for returns of the BLAST of the <em>A. thaliana</em> adjacent protein-coding genes. For a lincRNA BLAST return to be considered a homologous locus undergoing sequence decay, the two regions must fall within 10 kb of one another on the same chromosome (or greater if the nearest protein-coding gene was further away from the AtlincRNA). BLAST returns that did not meet this criteria, or absence of any lincRNA BLAST return at the lower threshold were considered loss events. Loss was confirmed using the comparative genomics platform CoGe (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lyons <em>et al.</em> 2008</a>; <a href="https://genomevolution.org/CoGe/)">https://genomevolution.org/CoGe/)</a>. Loss <em>vs.</em> decay of protein-coding genes was inferred in a similar manner.</p></div><div id="sec-6" class="subsection"><h3>Inferring dating of duplication events</h3><p id="p-16">Maximum likelihood phylogenetic trees were inferred from each nucleic acid alignment with RAxML version 7.2.8 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Stamatakis 2014</a>) using a general time reversible (GTR) model with gamma distributed rate heterogeneity. Support values were calculated from 100 bootstrap replicates. The topology of each gene tree was reconciled to the known species topology using Notung version 2.6 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Durand <em>et al.</em> 2006</a>). Trees were rooted in Notung using the <em>root</em> function, which roots each along the branch that provides the most congruence with the species tree. The <em>rearrange</em> function was used to rearrange poorly supported (&lt;70% bootstrap support) relationships to reflect the species topology. Inferred duplication information was extracted from Notung output <em>info</em> files, and <em>png</em> files were generated for visual inspection and downstream analysis.</p></div><div id="sec-7" class="subsection"><h3>Inferring characteristics that correlate with conservation</h3><p id="p-17">To identify AtlincRNA families with miRNA binding motifs, we ran all the sequences from each family through the miRNA prediction software psRNATarget (<a href="http://plantgrn.noble.org/psRNATarget/">http://plantgrn.noble.org/psRNATarget/</a>) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dai and Zhao 2011</a>) using only miRNAs identified in Brassicaceae. For psRNATarget, more stringent cut-off threshold of 2.0 was used, with the length for complementarity scoring set at 20 nt, with the flanking region around the target set at 17 bp upstream and 13 bp downstream (standard settings). Stress-responsive lincRNAs were identified from the <a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> (2012)</a> dataset. The genomic locations of conserved noncoding regions were obtained from <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Haudry <em>et al.</em> (2013)</a>. Bedtools overlap was used to determine if lincRNA loci overlapped with these CNS (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Quinlan and Hall 2010</a>).</p></div><div id="sec-8" class="subsection"><h3>Statistical analyses</h3><p id="p-18">Fisher’s exact test was used when comparing the observed numbers of identified lincRNAs in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. In each case, the observed lincRNAs in the indicated species (either <em>Leavenworthia alabamica</em>, <em>Brassica rapa</em>, or <em>Brassica oleracea</em>) were compared to an expected value based on the number of observed instances in equally or more divergent species (the average of <em>Eutrema salsugineum</em> and <em>Schrenkiella parvula</em> were used in both cases). When comparing the correlation between conservation of AtlincRNAs <em>vs.</em> expression of that locus in <em>A. thaliana</em>, we used a linear regression analysis to identify the significance, and established a Pearson’s correlation coefficient. A score test was performed with a Bonferroni multiple comparison correction for the lincRNA loci loss and decay analysis. Score intervals and score tests are reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS2.xlsx">File S2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of conserved homologous lincRNA loci in select mammals and Brassicaceae. (A) Percent of human (H. sap) lncRNA homologs identified in close relatives. Percentage of recovered loci are shown next to each bar. The accepted organismal phylogeny and estimated times of divergence for these three species was derived from Arnason et al. (2008) and is shown to the left to allow for direct comparisons. (B) Percentage of homologous loci recovered for AtlincRNAs (green), genic (yellow) and intergenic (blue) loci using a similar search protocol as that shown for human lncRNAs. Species&#x2019; names are abbreviations of those shown in Figure 1A. Percentage is shown next to each bar. Divergence times and species phylogeny was obtained from Beilstein et al. (2010)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-271616370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of conserved homologous lincRNA loci in select mammals and Brassicaceae. (A) Percent of human (&lt;em&gt;H. sap&lt;/em&gt;) lncRNA homologs identified in close relatives. Percentage of recovered loci are shown next to each bar. The accepted organismal phylogeny and estimated times of divergence for these three species was derived from Arnason &lt;em&gt;et al.&lt;/em&gt; (2008) and is shown to the left to allow for direct comparisons. (B) Percentage of homologous loci recovered for AtlincRNAs (green), genic (yellow) and intergenic (blue) loci using a similar search protocol as that shown for human lncRNAs. Species&#x2019; names are abbreviations of those shown in Figure 1A. Percentage is shown next to each bar. Divergence times and species phylogeny was obtained from Beilstein &lt;em&gt;et al.&lt;/em&gt; (2010).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2881/F2.medium.gif" width="260" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/9/2881/F2.medium.gif" width="260" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469026" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Comparison of conserved homologous lincRNA loci in select mammals and Brassicaceae. (A) Percent of human (<em>H. sap</em>) lncRNA homologs identified in close relatives. Percentage of recovered loci are shown next to each bar. The accepted organismal phylogeny and estimated times of divergence for these three species was derived from <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arnason <em>et al.</em> (2008)</a> and is shown to the left to allow for direct comparisons. (B) Percentage of homologous loci recovered for AtlincRNAs (green), genic (yellow) and intergenic (blue) loci using a similar search protocol as that shown for human lncRNAs. Species’ names are abbreviations of those shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>. Percentage is shown next to each bar. Divergence times and species phylogeny was obtained from <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Beilstein <em>et al.</em> (2010)</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-20">The datasets used in this study were acquired from publically available resources and are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS1.xlsx">File S1</a>. Additional information pertaining to analyses is available upon request.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>AtlincRNA loci are conserved at an intermediate level when compared to protein-coding genes or intergenic regions</h3><p id="p-21">To characterize the evolution of plant lincRNAs, we focused on a recently published dataset of ∼6500 Arabidopsis lincRNAs (<a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> 2012</a>). Because there is little evidence of AtlincRNA loci at the genomic level in poplar and grape (<a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> 2012</a>), we restricted our search for sequence homologs to more recently diverged taxa. Similar sequences were identified using a reciprocal BLASTN approach (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Johnson <em>et al.</em> 2008</a>), sampling from the genomes of 10 Brassicaceae species plus <em>T. hassleriana</em> (Cleomaceae) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). In each of the plant genomes examined, we determined that an E-value cutoff of 1e–20 recovered similar sequences that were most often syntenic and returned the AtlincRNA query in reciprocal BLASTN searches. Herein, we refer to all loci that meet these three criteria as homologous. We then asked whether this E-value cutoff returned homologous sequences from other well-characterized lncRNA datasets as a means of further validating its use to recover homologous sequences in Brassicaceae. Using a random set of 10,000 human lncRNAs as query, we searched the <em>Pongo abelii</em> (orangutan) and <em>Mus musculus</em> (mouse) genomes, and identified homologous loci for 96% and 20% of the human lncRNAs in orangutan and mouse, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>), which is similar to previously reported percentages of 81% and 19%, based on genomic and transcriptomic approaches (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Necsulea <em>et al.</em> 2014</a>).</p><p id="p-22">The Brassicaceae taxa sampled span a range of divergence dates with <em>A. thaliana</em> of ∼13 Mya to ∼65 Mya (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Beilstein <em>et al.</em> 2010</a>). Using a set of 5362 unique AtlincRNAs as query in a BLAST of Brassicaceae genomes (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, A and B</a>), we found that the percentage of AtlincRNAs for which sequence homologs could be identified decreased as divergence date (phylogenetic distance) increased: <em>e.g.</em>, 23% for <em>Capsella rubella</em> (1233/5362; ∼18 Mya), 19% in <em>S. parvula</em> (1057/5362; ∼42 Mya), and 3% in <em>Aethionema arabicum</em> (186/5362; ∼54 Mya) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). Species of equal phylogenetic distance to <em>A. thaliana</em> differed in the number of AtlncRNA homologs recovered. For example, all lineage II species diverged from Arabidopsis ∼42 Mya, but we recovered homologs for only 16% of AtlncRNAs in <em>Brassica rapa</em>, whereas <em>S. parvula</em> and <em>E. salsugineum</em> harbored 19% and 20%, respectively (green bars, <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>; <em>P</em>-value &lt; 0.001). Using the same BLASTN parameters as those in our search for lincRNA homologs, sequence homologs were identified in the <em>A. arabicum</em> genome for 67% of a set of ∼10,038 <em>A. thaliana</em> protein-coding genes, and for 1% of 14,426 intergenic regions (blue and yellow bars, <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). Thus, AtlincRNA loci are conserved at an intermediate level in comparison to protein-coding genes and intergenic regions.</p></div><div id="sec-12" class="subsection"><h3>Around 22% of AtlincRNA loci were present in the common ancestor of lineage I and II species ∼42 Mya</h3><p id="p-23">Following homolog identification in Brassicaceae genomes, we clustered the reciprocal BLASTN results from each pairwise AtlincRNA query into sequence families (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). Each family contained the original query AtlincRNA, as well as homologous sequences from each subject genome that matched our criteria of synteny, reciprocity, and sequence similarity (E-value cutoff = 1e–20). From these families, we developed a more rarefied dataset of families composed of sequence homologs from a minimum of four species distributed between lineages I and II (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>). For example, in addition to the AtlincRNA query, all families were required to include sequences representing three other species, at least one of which had to be <em>B. rapa</em>, <em>B. oleraceae</em>, <em>S. parvula</em>, or <em>E. salsugineum</em> (representing Lineage II). Within the AtlincRNA dataset, 1271 loci (23%) met this criterion, and were grouped into unique families (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS1.xlsx">File S1</a>). We did not permit returned homologs to be included in &gt; 1 lincRNA family, and thus all families are unique. We refer to these families as conserved since these loci emerged, at minimum, in the most recent common ancestor of the two lineages ∼42 Mya (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Beilstein <em>et al.</em> 2010</a>).</p><p id="p-24">We hypothesized that the observed sequence conservation of some AtlincRNA families could be due to the presence of short ORFs (&lt; 100 aa), violating an important condition of inclusion as a putative lincRNA locus. To address this concern, these conserved families were screened for protein-coding potential via RNAcode (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Washietl <em>et al.</em> 2011</a>), and similarity to known noncoding RNAs, using the rFAM database (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nawrocki <em>et al.</em> 2015</a>). RNAcode analyzes multi-sequence alignments for nucleotide substitutions or frameshifts that would maintain an ORF across multiple species. We found statistically significant evidence for a conserved ORF (<em>P</em>-value &lt; 0.001; RNAcode) in 90 AtlincRNAs (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1C</a>). These families were excluded from further analysis but are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS1.xlsx">File S1</a>. Moreover, we found 42 (3.5%) of the conserved set of AtlincRNA families contained sequences with significant similarity to a known noncoding RNA (<em>i.e.</em>, spliceosomal and snoRNAs; <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1C</a>; listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS1.xlsx">File S1</a>). Indeed, 14 of the 51 AtlincRNAs with a sequence homolog in <em>T. hassleriana</em>, and nine of 17 AtlincRNAs with a sequence homolog in <em>Carica papaya</em> contain known noncoding RNA elements, explaining much of the sequence conservation seen in these more divergent genomes. In total, we identified 1180 (22%) conserved AtlincRNAs, for which sequence conservation was independent of coding potential (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS1.xlsx">File S1</a>). Of the original query AtlincRNAs used to build these conserved families, 93 have homologous sequences in all the Brassicaceae genomes we tested, and therefore represent an even more conserved dataset (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS1.xlsx">File S1</a>). In sum, we define here a class of conserved genomic regions that have been annotated as lincRNAs in <em>A. thaliana</em>. Whether these loci are conserved due to lincRNA function remains an open question.</p></div><div id="sec-13" class="subsection"><h3>Overlap with CNS, transcription levels, and stress-responsiveness all correlate positively with conservation of AtlincRNA loci</h3><p id="p-25">We next attempted to understand factors influencing sequence conservation within the class of conserved AtlincRNA families. In general, protein-coding genes are more conserved than lincRNAs, both at the sequence level, and in regard to synteny (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Goodstadt and Ponting 2006</a>). However, intergenic regions can harbor important regulatory elements for protein-coding genes, and therefore display evidence of selective constraint. A recent comparative genomic analysis of sites under selection in Brassicaceae demonstrated that the percentage of sites in a genome under selection increases with proximity to the translation start site of the nearest protein-coding gene (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Haudry <em>et al.</em> 2013</a>). To address whether proximity of an AtlincRNA locus to a protein-coding gene might explain conservation of the latter, we tested for a positive correlation between proximity to a protein-coding gene, both up and downstream and on either strand, and sequence conservation (identification of a sequence homolog) for the AtlincRNA dataset. We detected no significant correlation between conservation of an AtlincRNA locus and its proximity to a known gene (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-26"><a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Haudry <em>et al.</em> (2013)</a> further identified a suite of 90,104 <em>A. thaliana</em> noncoding genomic regions (conserved noncoding sequences, or CNS) that showed a reduced substitution rate over contiguous regions. CNS have been identified in a variety of eukaryotes and are believed to be broadly important for gene regulation (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Freeling and Subramaniam 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adrian <em>et al.</em> 2010</a>). In <em>A. thaliana</em>, these elements are typically short (on average 36 bp in length) and predominantly reside adjacent to (within 500 bp), or within (<em>i.e.</em>, untranslated regions and introns) genes. However, a subset (22%) of the identified CNS reside in intergenic space. Thus, we searched for overlap between these previously defined CNS and AtlincRNA loci. We detected a significant enrichment in overlap between intergenic CNS and the conserved AtlincRNA dataset (941/1180, or 80%) <em>vs.</em> the nonconserved AtlincRNA dataset (996/4082, or 24%; <em>P</em>-value &lt; 0.001; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). In sum, the presence of a CNS, but not proximity to a protein-coding gene, strongly correlates with genomic conservation of AtlincRNA loci.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Features enriched in conserved AtlincRNAs. (A) Percent of AtlincRNA loci overlapping with conserved noncoding sequence defined by Haudry et al. (2013). Conserved AtlincRNA loci are defined by having sequence homologs in &#x2265;&#xA0;four species, with at least one species in the opposite lineage (i.e., Lineage&#xA0;II). Nonconserved AtlincRNAs are those with " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-271616370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Features enriched in conserved AtlincRNAs. (A) Percent of AtlincRNA loci overlapping with conserved noncoding sequence defined by Haudry &lt;em&gt;et al.&lt;/em&gt; (2013). Conserved AtlincRNA loci are defined by having sequence homologs in &#x2265;&#xA0;four species, with at least one species in the opposite lineage (&lt;em&gt;i.e.&lt;/em&gt;, Lineage&#xA0;II). Nonconserved AtlincRNAs are those with &lt;&#xA0;four sequence homologs. **&#xA0;&lt;em&gt;P&lt;/em&gt;-value&#xA0;&lt;&#xA0;0.001. (B) Box and whiskers plot of expression values for AtlincRNA families with homologous loci identified for increasingly divergent species. Expression is denoted as the average FPKM (fragment per kilobase of exon per million fragments mapped) values across four different tissues along a logarithmic scale [flowers, leaves, siliques, root; values from Liu &lt;em&gt;et al.&lt;/em&gt; (2012)]. Transcription data were available for 2666 AtlincRNAs. The number of families with representatives at each divergence time-point is listed. Divergence times correspond to those shown in Figure 1A. A Pearson&#x2019;s Correlation Coefficient was calculated (CC, top left). A linear regression analysis was performed to determine the statistical significance of this coefficient. (C) Percent of all nonconserved (orange) or conserved AtlincRNA (blue) families with miRNA binding motifs. (D) Percent of stress-responsive AtlincRNAs out of total number of AtlincRNAs conserved to each node (nodes indicates by divergence dates shown along &lt;em&gt;x&lt;/em&gt;-axis). Actual number of stress-responsive AtlincRNAs shown above each bar. Where shown, *** indicates &lt;em&gt;P&lt;/em&gt;-value&#xA0;&lt;&#xA0;0.0001 relative to the &lt;em&gt;A. thaliana&lt;/em&gt;-specific lincRNAs (node&#xA0;1).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2881/F3.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/9/2881/F3.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469154" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Features enriched in conserved AtlincRNAs. (A) Percent of AtlincRNA loci overlapping with conserved noncoding sequence defined by <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Haudry <em>et al.</em> (2013)</a>. Conserved AtlincRNA loci are defined by having sequence homologs in ≥ four species, with at least one species in the opposite lineage (<em>i.e.</em>, Lineage II). Nonconserved AtlincRNAs are those with &lt; four sequence homologs. ** <em>P</em>-value &lt; 0.001. (B) Box and whiskers plot of expression values for AtlincRNA families with homologous loci identified for increasingly divergent species. Expression is denoted as the average FPKM (fragment per kilobase of exon per million fragments mapped) values across four different tissues along a logarithmic scale [flowers, leaves, siliques, root; values from <a id="xref-ref-37-9" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> (2012)</a>]. Transcription data were available for 2666 AtlincRNAs. The number of families with representatives at each divergence time-point is listed. Divergence times correspond to those shown in <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1A</a>. A Pearson’s Correlation Coefficient was calculated (CC, top left). A linear regression analysis was performed to determine the statistical significance of this coefficient. (C) Percent of all nonconserved (orange) or conserved AtlincRNA (blue) families with miRNA binding motifs. (D) Percent of stress-responsive AtlincRNAs out of total number of AtlincRNAs conserved to each node (nodes indicates by divergence dates shown along <em>x</em>-axis). Actual number of stress-responsive AtlincRNAs shown above each bar. Where shown, *** indicates <em>P</em>-value &lt; 0.0001 relative to the <em>A. thaliana</em>-specific lincRNAs (node 1).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Given the positive correlation between conservation of lincRNA loci and their expression and structure in vertebrates (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Necsulea <em>et al.</em> 2014</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Managadze <em>et al.</em> 2011</a>), we tested if this paradigm also characterized genomically conserved AtlincRNAs. We inferred the most recent common ancestor in which the AtlincRNA locus was present based on our genomic comparisons. We used the divergence date of nodes in the tree to ask whether the expression level (FPKM) or structural complexity (Minimum Free Energy, MFE) of the AtlincRNA correlates with age of its emergence within the family (<em>i.e.</em>, most recent common ancestor [node] where the locus was likely present) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). We found that AtlincRNAs with an orthologous locus detectable in <em>A. arabicum</em>, and thus for which the ancestor of all extant Brassicaceae is inferred to have had a copy (54 Mya), were on average expressed at a higher level in <em>A. thaliana</em> than the average value for the population of <em>A. thaliana</em>-specific lincRNAs (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). In fact, expression in <em>A. thaliana</em> is positively correlated with sequence conservation across Brassicaceae (Correlation Coefficient of 0.136, <em>P</em>-value &lt; 0.0001). In contrast, we found no correlation between age of emergence of an AtlincRNA locus and MFE, as determined by RNAfold (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Lorenz <em>et al.</em> 2011</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FigureS2.pdf">Figure S2</a>).</p><p id="p-29">Another possible explanation for sequence conservation is conservation of function. While it is difficult to infer function of a lincRNA from sequence alone, there are categories that are more amenable to functional prediction, such as natural antisense transcripts and microRNA sponges. The lincRNAs in our dataset do not overlap known genes (in either direction), and therefore cannot be antisense transcriptional regulators. However, given the recent reports of lncRNAs acting as molecular sponges of miRNAs (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kretz <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hansen <em>et al.</em> 2013</a>), we assessed the potential of conserved AtlincRNA loci to bind miRNAs. We searched for miRNA binding sites using the Brassicaceae miRNA dataset in psRNATarget (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Dai and Zhao 2011</a>). AtlincRNA families with putative miRNA binding sites make up 4% of the overall lincRNA population. We observed a modest enrichment in miRNA binding sites in the conserved AtlincRNA dataset (5% <em>vs.</em> 4%; <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>; a list of lincRNAs with miRNA binding sites is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS1.xlsx">File S1</a>). Interestingly, in 9 out of the 59 AtlincRNAs that harbored miRNA binding sites, the sequence of the motif was conserved at the same locus in all Brassicaceae, potentially representing a deeply conserved lincRNA regulatory pathway.</p><p id="p-30">Finally, we asked if the conserved AtlincRNAs were over-represented in the stress-responsive lincRNA dataset produced by <a id="xref-ref-37-10" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> (2012)</a>. In the dataset of 5270 AtlincRNAs that we examined, 969 were differentially expressed in response to at least one of four environmental stresses (abscisic acid, cold, drought, and salt) (<a id="xref-ref-37-11" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> 2012</a>). We determined whether the proportion of stress-responsive lincRNAs increased with the inferred age of emergence. More specifically, for all 5270 AtlincRNAs, we determined the species with the deepest coalescent point in the organismal tree from which a sequence homolog was retrieved, and then, for each point, we calculated the percentage of lincRNAs classified as stress responsive. For example, 1736 lincRNAs are <em>A. thaliana</em>-specific, and 147 of these (8.5%) were stress-responsive (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>). For the 1785 lincRNAs that coalesce at the node uniting <em>A. thaliana</em> and <em>A. lyrata</em>, 339 were stress-responsive (19.0%, <em>P</em>-value &lt; 0.0001; Fisher’s exact test relative to stress responsive <em>A. thaliana</em>-specific lincRNAs); 106 lincRNAs coalesce at the node uniting <em>A. thaliana</em> and <em>A. arabicum</em>, 32 stress responsive (30.0%, <em>P</em>-value &lt; 0.0001). And, for the 25 lincRNAs that coalesce at the node uniting <em>A. thaliana</em> and <em>T. hassleriana</em>, 12 were stress-responsive (49.0%, <em>P</em>-value &lt; 0.0001) (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3D</a>). Thus, conservation of an AtlincRNA locus correlates with its propensity to be differentially regulated in response to environmental stress in <em>A. thaliana</em>. A list of these lincRNAs is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS1.xlsx">File S1</a>.</p></div><div id="sec-14" class="subsection"><h3>Gene/whole genome duplication, but not transposable elements, influence lincRNA diversification</h3><p id="p-31">We next identified genomic factors that might be driving emergence or loss of AtlincRNA homologs in Brassicaceae genomes. Recent findings in vertebrates suggest a role for TEs in lncRNA diversification (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Kapusta <em>et al.</em> 2013</a>). The AtlincRNA dataset of <a id="xref-ref-37-12" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> (2012)</a> excluded sequences with fragments of TEs, precluding comparison with results in vertebrates. To remedy this issue, and to explore the potential role of TEs in lincRNA diversification, we reanalyzed the AtlincRNA dataset, including the intergenic transcripts previously shown to contain repetitive elements, using the same filtering parameters used in the vertebrate study (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Kapusta <em>et al.</em> 2013</a>). This yielded ∼12,000 putative AtlincRNAs, 45% of which contained at least 10 nt of TE DNA (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS2.xlsx">File S2</a>). Similar to vertebrates, AtlincRNA loci contained significantly more TE content than protein-coding loci (<em>P</em> &lt; 0.01; Fisher’s exact test). However, the percentage of AtlincRNA loci containing at least 10 nt of a TE was significantly less than that reported for vertebrate lncRNAs (<em>P</em> &lt; 0.01; Fisher’s exact test) (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Kapusta <em>et al.</em> 2013</a>). We also asked if AtlincRNA emergence correlated with the presence of a TE, either within the lincRNA, or in the region upstream or downstream of the lincRNA (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). A small percentage (0.2%) of the species-specific AtlincRNA loci (<em>i.e.</em>, appear to have emerged since <em>A. thaliana</em> and <em>A. lyrata</em> diverged ∼13 Mya) contained at least 10 nt of TE DNA. A larger percentage (13.2%) of the species-specific AtlincRNAs were within 500 bp of TE DNA. None of the conserved AtlincRNAs contained, nor were they within 100 bp of, any TEs. Thus, for the loci encoding AtlincRNAs in Brassicaceae, we find little evidence to indicate that transposable element activity promotes the emergence of new lincRNAs in the genome. However, as seen in vertebrates, adjacent TEs may be driving expression of these species-specific lincRNA loci (<a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Kapusta <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kelley <em>et al.</em> 2014</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transposable element (TE) content in AtlincRNAs. (A) Percent of lincRNAs and coding sequences from A. thaliana that overlap with &#x2265;&#xA0;10&#xA0;bp of a TE as determined by RepeatMasker. Actual percent shown above each bar. (B) Connection between TEs and AtlincRNA emergence. AtlincRNAs were binned based on when they are believed to have emerged [shown on x-axis in millions of years ago (Mya)]. TE content, either within or adjacent to the lincRNA, was determined for AtlincRNAs within each bin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-271616370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Transposable element (TE) content in AtlincRNAs. (A) Percent of lincRNAs and coding sequences from &lt;em&gt;A. thaliana&lt;/em&gt; that overlap with &#x2265;&#xA0;10&#xA0;bp of a TE as determined by RepeatMasker. Actual percent shown above each bar. (B) Connection between TEs and AtlincRNA emergence. AtlincRNAs were binned based on when they are believed to have emerged [shown on &lt;em&gt;x&lt;/em&gt;-axis in millions of years ago (Mya)]. TE content, either within or adjacent to the lincRNA, was determined for AtlincRNAs within each bin.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2881/F4.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/9/2881/F4.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469133" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">Transposable element (TE) content in AtlincRNAs. (A) Percent of lincRNAs and coding sequences from <em>A. thaliana</em> that overlap with ≥ 10 bp of a TE as determined by RepeatMasker. Actual percent shown above each bar. (B) Connection between TEs and AtlincRNA emergence. AtlincRNAs were binned based on when they are believed to have emerged [shown on <em>x</em>-axis in millions of years ago (Mya)]. TE content, either within or adjacent to the lincRNA, was determined for AtlincRNAs within each bin.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Given the prevalence of gene duplicates in plant genomes, and in Brassicaceae specifically (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Koenig and Weigel 2015</a>), we asked whether duplication events, either in the form of WGD or local duplication, might be a mechanism for lincRNA emergence. To investigate the impact of duplication events on lincRNA evolution, we inferred the most likely gene tree, and estimated branch support using maximum likelihood bootstrap for each of the 1180 conserved AtlincRNA families, and then employed Notung (2.0) (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Durand <em>et al.</em> 2006</a>) to determine the duplication history. Notung reconciles topological incongruence between gene trees and the accepted organismal tree, using incongruence to infer duplications and losses. We analyzed duplication in all conserved AtlincRNA families but omitted highly duplicated families (&gt; 3 duplication events along the backbone of the tree) from downstream analysis. Of the remaining 1005 conserved families, 296 (29%) showed evidence of at least one, but sometimes multiple duplication events along the branch leading to <em>A. thaliana</em> (blue line, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>; lincRNA IDs listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS2.xlsx">File S2</a>), indicating that numerous AtlincRNAs are likely the product of relatively recent duplication events. In general, duplications were relatively evenly distributed along the backbone nodes leading to <em>A. thaliana</em>, although 106 families (26%) experienced a duplication event along the branch uniting lineage I and lineage II (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Due to the low number of identifiable homologs in <em>A. arabicum</em> and <em>T. hassleriana</em>, we recovered only a few duplication events that trace back to the deepest nodes in our tree. It should be noted that no duplication event coincided with the insertion of a TE, either within or adjacent to the lincRNA locus. In sum, gene duplication appears to have played a role in the evolution of approximately one-third of conserved AtlincRNAs. Moreover, the gene duplication events driving lincRNA evolution do not appear to be due to the activity of transposable elements.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inferred timing of duplication and duplication dependence in the conserved AtlincRNA families. Timing of duplications within A. thaliana conserved lincRNA families. The data represent duplications that occur along the backbone leading to the AtlincRNA (blue bar). Duplications are shown per node, with approximate divergence times (Mya) shown in red. The number of duplication events per node are shown in the green circles. The number of AtlincRNA families with duplications per node are shown in purple circles. Some families contained a duplication at multiple nodes and therefore were counted multiple times. Overall, 296 AtlincRNA loci showed evidence of a duplication event at least once, but in some cases multiple times. The total number of duplications are shown below." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-271616370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Inferred timing of duplication and duplication dependence in the conserved AtlincRNA families. Timing of duplications within &lt;em&gt;A. thaliana&lt;/em&gt; conserved lincRNA families. The data represent duplications that occur along the backbone leading to the AtlincRNA (blue bar). Duplications are shown per node, with approximate divergence times (Mya) shown in red. The number of duplication events per node are shown in the green circles. The number of AtlincRNA families with duplications per node are shown in purple circles. Some families contained a duplication at multiple nodes and therefore were counted multiple times. Overall, 296 AtlincRNA loci showed evidence of a duplication event at least once, but in some cases multiple times. The total number of duplications are shown below.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2881/F5.medium.gif" width="404" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/9/2881/F5.medium.gif" width="404" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469044" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-34" class="first-child">Inferred timing of duplication and duplication dependence in the conserved AtlincRNA families. Timing of duplications within <em>A. thaliana</em> conserved lincRNA families. The data represent duplications that occur along the backbone leading to the AtlincRNA (blue bar). Duplications are shown per node, with approximate divergence times (Mya) shown in red. The number of duplication events per node are shown in the green circles. The number of AtlincRNA families with duplications per node are shown in purple circles. Some families contained a duplication at multiple nodes and therefore were counted multiple times. Overall, 296 AtlincRNA loci showed evidence of a duplication event at least once, but in some cases multiple times. The total number of duplications are shown below.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Conserved AtlincRNA families are required to include a representative from both lineages I and II, indicating that the locus was present in the common ancestor of species in these groups (red asterisk, <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1C</a>). Therefore, the lack of an AtlincRNA-like homolog from a species in these lineages suggests that either: 1) the locus was purged from the genome, or 2) it has accumulated sufficient nucleotide divergence to prevent identification at our BLASTN cutoff value. We refer to these alternatives as lincRNA locus loss or decay, respectively. To infer rates of loss and decay, we repeated the reciprocal BLASTN search using a less stringent E-value cutoff (1e–5) (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). For sequence variable loci (<em>i.e.</em>, recovered between 1e–20 and 1e–5), we determined whether they shared synteny with the AtlincRNA query, and, if so, classified them as decay events in that species. Alternatively, if we failed to recover additional BLASTN hits at lower stringency, or the recovered sequences were in different genomic locations than the AtlincRNA query, they were classified as loss events (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence loss and decay events in the conserved AtlincRNA families. (A) Strategy for inferring sequence decay or loss for the absent loci in the conserved AtlincRNA families using a less stringent BLASTN cutoff (1e&#x2013;5) and synteny. (B) Bar graph of the percent (out of total 1023) of lincRNA loci experiencing sequence decay in the species listed. Pairwise comparisons of the proportion of lost or decayed loci were performed between all species using a score test for a difference of binomial proportions. Species that, after a Bonferroni correction, were not significantly different from one another were grouped. (C) Bar graph of the percent (out of total 1023) of lincRNA loci experiencing loss in the species listed. Raw numbers are shown in File S2. Light blue bars depict the level of loss and decay observed for protein-coding loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-271616370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sequence loss and decay events in the conserved AtlincRNA families. (A) Strategy for inferring sequence decay or loss for the absent loci in the conserved AtlincRNA families using a less stringent BLASTN cutoff (1e&#x2013;5) and synteny. (B) Bar graph of the percent (out of total 1023) of lincRNA loci experiencing sequence decay in the species listed. Pairwise comparisons of the proportion of lost or decayed loci were performed between all species using a score test for a difference of binomial proportions. Species that, after a Bonferroni correction, were not significantly different from one another were grouped. (C) Bar graph of the percent (out of total 1023) of lincRNA loci experiencing loss in the species listed. Raw numbers are shown in File S2. Light blue bars depict the level of loss and decay observed for protein-coding loci.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2881/F6.medium.gif" width="440" height="376"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/9/2881/F6.medium.gif" width="440" height="376"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2881/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469240" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-36" class="first-child">Sequence loss and decay events in the conserved AtlincRNA families. (A) Strategy for inferring sequence decay or loss for the absent loci in the conserved AtlincRNA families using a less stringent BLASTN cutoff (1e–5) and synteny. (B) Bar graph of the percent (out of total 1023) of lincRNA loci experiencing sequence decay in the species listed. Pairwise comparisons of the proportion of lost or decayed loci were performed between all species using a score test for a difference of binomial proportions. Species that, after a Bonferroni correction, were not significantly different from one another were grouped. (C) Bar graph of the percent (out of total 1023) of lincRNA loci experiencing loss in the species listed. Raw numbers are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS2.xlsx">File S2</a>. Light blue bars depict the level of loss and decay observed for protein-coding loci.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">We identified the number of loci that were lost or decayed and performed pairwise comparisons between all species. Based on these comparisons, we identified three distinct groups of species (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6, B and C</a>). The percent of decayed loci was similar for the group containing <em>S. parvula</em>, <em>B. oleracea</em>, <em>B. rapa</em>, and <em>L. alabamica</em> (17.0–19.6%), while a lower percent of decay characterized the group containing <em>Capsella grandiflora</em>, <em>C. rubella</em>, and <em>E. salsugineum</em> (13.0–14.8%; <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS2.xlsx">File S2</a>). The percent of decayed loci was significantly different between these two groups (<em>P</em> &lt; 0.01 based on pairwise comparisons using a Bonferroni multiple comparison correction; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS2.xlsx">File S2</a>). <em>A. lyrata</em> experienced significantly less decay than either of these two groups (1%; <em>P</em> &lt; 0.0001; Bonferroni multiple comparison correction). The groupings identified by pairwise comparisons for loci experiencing loss were composed of different species. The pairwise differences between the mesopolyploids <em>B. oleracea</em>, <em>B. rapa</em>, and <em>L. alabamica</em> were insignificant and ranged from 11.8% to 13.0% (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>). In relation to each other, similar percentages of loss (2.8–4.0%) were detected in the nonpolyploid species <em>C. grandiflora</em>, <em>C. rubella</em>, <em>E. salsugineum</em>, and <em>S. parvula</em>. The percent of lost loci was significantly different between these two groups (<em>P</em> &lt; 0.0001; Bonferroni multiple comparison correction). Loss of lincRNA loci was rare in <em>A. lyrata</em> (0.8%). Interestingly, the species for which we observed a greater than expected increase in lincRNA loss have experienced a recent WGD (mesopolyploidization event) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kagale <em>et al.</em> 2014</a>), suggesting the two may be correlated.</p><p id="p-38">We also examined a randomized set of 10,611 <em>A. thaliana</em> protein-coding loci for loss and decay. We detected significantly elevated levels of loss and decay in <em>B. oleraceae</em> compared with other species, while <em>A. lyrata</em> showed significantly lower levels (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1/FileS2.xlsx">File S2</a>). Thus, for protein-coding genes, we did not detect groups of species with similar levels of loss and decay that correlated with WGD or phylogenetic position.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><div id="sec-16" class="subsection"><h3>A subset of AtlincRNAs are conserved across Brassicaceae and may be cis-regulatory RNAs</h3><p id="p-39">The evolution of AtlincRNA loci is broadly similar to that seen in vertebrate systems, wherein sequence conservation is inversely proportional to timing of divergence. The percent of lncRNAs found to be homologous between humans and mice, which diverged ∼90 Mya, ranges from 19% to 38% (<a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Necsulea <em>et al.</em> 2014</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Washietl <em>et al.</em> 2014</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Hezroni <em>et al.</em> 2015</a>). Of the human lncRNA dataset used by <a id="xref-ref-45-7" class="xref-bibr" href="#ref-45">Necsulea <em>et al.</em> (2014)</a>, ∼3% can be identified in chicken, which diverged from humans &gt; 300 Mya. In our analysis, a large percentage of the AtlincRNAs appear to be either species- or genus-specific, thus explaining previous reports on the extremely low (&lt; 1%) recovery of sequence homologs between more distantly related species [<em>i.e.</em>, <em>Arabidopsis</em> and poplar (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Shuai <em>et al.</em> 2014</a>; <a id="xref-ref-37-13" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> 2012</a>)]. Despite this variation, we identified a group of 1180 lincRNA loci with sequence homologs in both lineage I and II of the family. Within this conserved set, sequence homologs were detected in all tested Brassicaceae genomes for 93 AtlincRNAs, dating the origin of these loci to at least 54 Mya. Thus, our comparative genomic analysis serves as an additional filter in the identification of conserved AtlincRNA loci. The implications of this sequence conservation are unclear, but could be due to conservation of lincRNA function.</p><p id="p-40">We identified several factors that may best explain the genomic conservation we observed for 22% of the AtlincRNA loci. Features that were enriched within the conserved AtlincRNA dataset include higher overall expression, stress-responsiveness, and overlap with previously identified intergenic conserved noncoding sequences (CNS). The propensity of more deeply conserved AtlincRNAs to overlap with CNS is particularly interesting because these DNA elements are predicted to be <em>cis</em>-acting transcriptional regulators (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Freeling and Subramaniam 2009</a>). Examples of this regulation include a CNS referred to as <em>Vgt1</em> that is associated with flowering time in the grasses (Salvi <em>et al.</em> 2007). Here we note overlap between the AtlincRNA dataset of <a id="xref-ref-37-14" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> (2012)</a> with the CNS from <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Haudry <em>et al.</em> (2013)</a>, to our knowledge providing significant evidence of CNS transcription for the first time. Given the size difference between CNS (∼36 bp) and lincRNAs (&gt;200 nts), overlap with a CNS is not enough to explain the retention we see in the more conserved AtlincRNA dataset, suggesting sequence conservation is driven by additional factors. More importantly, transcription of CNS as lincRNAs suggests that these regions might regulate gene expression at the RNA level as well.</p><p id="p-41">Vertebrate enhancer regions are important <em>cis</em>-regulatory elements that show signatures of selection and, in some cases, control cell- and tissue-specific expression profiles. Some have argued that plant CNSs are functional analogs of vertebrate enhancers (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Freeling and Subramaniam 2009</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pennacchio <em>et al.</em> 2007</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lam <em>et al.</em> 2014</a>). Large-scale transcriptomic analyses indicate that many of these enhancer regions are transcriptionally active (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Djebali <em>et al.</em> 2012</a>). When transcribed, they are termed enhancer RNAs (eRNAs), and data support a model in which the presence of the eRNA, and not just transcription of the enhancer region, regulates expression of adjacent genes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lam <em>et al.</em> 2013</a>). While evidence of transcription of Arabidopsis CNS is not sufficient to demonstrate that CNS-overlapping lincRNAs are enhancer RNAs, this result suggests that further study is warranted.</p></div><div id="sec-17" class="subsection"><h3>Genome dynamics are driving diversification of lincRNA-encoding loci</h3><p id="p-42">The burgeoning interest in lncRNAs and the observation that a large set of them are species specific, have propelled studies focused on identifying factors influencing their diversification. Transposable elements are implicated in diversification of lincRNA populations in vertebrates (<a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">Kapusta <em>et al.</em> 2013</a>). Similar to vertebrate analyses, AtlincRNAs contain more TE content than protein-coding genes. Nevertheless, species-specific AtlincRNAs were no more likely to contain a TE than AtlincRNAs with sequence homologs in other Brassicaceae, suggesting that transposable element activity is not driving species-specific AtlincRNA emergence. However, we noticed an increase in the number of species-specific lincRNA loci within 500 bp of a TE compared with AtlincRNA loci for which similar sequences were identified in the genomes of other Brassicaceae. Thus, it may be that TEs are acting as <em>cis</em>-regulatory elements, facilitating transcription of these lincRNA loci, similar to observations in humans and other vertebrates (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kelley <em>et al.</em> 2014</a>; <a id="xref-ref-28-8" class="xref-bibr" href="#ref-28">Kapusta <em>et al.</em> 2013</a>). Brassicaceae genomes are relatively depauperate in TEs when compared with genomes in grasses or other plant families (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Murat <em>et al.</em> 2012</a>), whose TE content is more similar to that in vertebrates. Hence, it is possible that the lack of influence exerted by TEs we observed in Brassicaceae may not be representative of other groups of plants. Add to this the observation that lincRNAs in grasses are less conserved genomically than are those in Brassicaceae (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2014</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Xin Qi <em>et al.</em> 2013</a>), and a reasonable hypothesis moving forward is that lincRNA diversification driven by TEs depends on their abundance and level of activity in the genome.</p><p id="p-43">WGD, given its prevalence in plants, presents another likely mechanism for emergence of lincRNAs (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Husband <em>et al.</em> 2013</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Moghe and Shiu 2014</a>). All Brassicaceae share a WGD termed the α duplication, and, if this event precipitated the emergence of lincRNAs in the group, it could explain the inability to find AtlincRNA sequence homologs in <em>T. hassleriana</em> (Cleomaceae) and <em>C. papaya</em> (Caricaceae), whose divergences predate the WGD (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Beilstein <em>et al.</em> 2010</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cheng <em>et al.</em> 2013</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Koenig and Weigel 2015</a>). However, our analyses of gene duplication did not recover an overrepresentation of lncRNAs with duplications along the same branch that the α WGD occurred. Instead, lincRNA duplication events associated with AtlincRNAs were fairly evenly distributed along the backbone leading to <em>A. thaliana</em>. The lack of a correlation between known WGD and lincRNA emergence implies that the duplications we detected are local rather than global events. In contrast to emergence, we found that WGD events correlate with an accelerated loss of lincRNA loci. This is consistent with observations for <em>B. rapa</em> protein-coding loci that indicate deletions, and not point mutations, make up the bulk of the gene fractionation that has occurred postpolyploidization (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Tang <em>et al.</em> 2012</a>). In sum, our data suggest that recent WGD may contribute to variability in the persistence of putative lincRNAs among species by increasing the rate of their deletion, likely due to fractionation post polyploidy. Thus, the α WGD event that defines Brassicaceae may have led to a dramatic decline in the ancestral lincRNA population, resulting in very few lincRNAs with conserved loci throughout the family (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Nelson and Shippen 2015</a>).</p><p id="p-44">A significant caveat to these analyses is that conservation of a lincRNA-encoding locus does not imply expression, and thus it is not clear if expression is conserved across the family. However, even with these limitations, comparative genomic approaches can still be informative in systems with minimal transcriptomic data. Due to their above average sequence conservation, the conserved lincRNA dataset described here represents an excellent starting point for functional analysis. For example, several of the conserved and stress-responsive AtlincRNAs we identify here were recently shown to be protein-bound and nuclear localized, providing further evidence that signatures of conservation may underlie conservation of function across Brassicaceae for well conserved AtlincRNAs (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gosai <em>et al.</em> 2015</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank the PaBeBaMo discussion group at the University of Arizona, and in particular David Baltrus, Rebecca Mosher, and Ravi Palanivelu, for critical comments on this work. We are very grateful for the support of the cloud computing resources at the iPlant Consortium. We also thank the Statistical Consulting Laboratory at the University of Arizona for advice regarding appropriate statistical analyses for this work. This work was supported by National Science Foundation Molecular and Cellular Biosciences grant #1409251 to M.A.B. and Plant Genome Research Program grant # 144490 to M.A.B and E.L.</p><p id="p-46">Author contributions: A.D.L.N. and M.A.B. conceived the project. A.D.L.N., E.S.F., A.M.R.M., and M.R.F. performed data collection. A.D.L.N., E.S.F., U.K.D., D.S.C., and A.K.H. performed data analysis. A.D.L.N., E.S.F., U.K.D., A.K.H., E.L., and M.A.B. edited the manuscript. A.D.L.N., E.S.F., and M.A.B. wrote the manuscript. All authors read and approved the final manuscript. The authors declare that they have no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-47">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030338/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-48"><em>Communicating editor: T. Slotte</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-04-19"><span class="received-label">Received </span>April 19, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-07-09"><span class="accepted-label">Accepted </span>July 9, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Nelson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.1" data-doi="10.1105/tpc.110.074682"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adrian</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrona</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reimer</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albani</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>cis</em>-regulatory elements and chromatin state coordinately control temporal and spatial expression of <em>FLOWERING LOCUS T</em> in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1425</span>–<span class="cit-lpage">1440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.074682%26rft_id%253Dinfo%253Apmid%252F20472817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzUvMTQyNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yODgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.2" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.3" data-doi="10.1016/j.gene.2008.05.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnason</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adegoke</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gullberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harley</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mitogenomic relationships of placental mammals and molecular estimates of their divergences.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">421</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">37</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DArnason%26rft.auinit1%253DU.%26rft.volume%253D421%26rft.issue%253D1-2%26rft.spage%253D37%26rft.epage%253D51%26rft.atitle%253DMitogenomic%2Brelationships%2Bof%2Bplacental%2Bmammals%2Band%2Bmolecular%2Bestimates%2Bof%2Btheir%2Bdivergences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2008.05.024%26rft_id%253Dinfo%253Apmid%252F18590805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2008.05.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18590805&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.4" data-doi="10.3732/ajb.93.4.607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beilstein</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Al-Shehbaz</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellogg</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Brassicaceae phylogeny and trichome evolution.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">93</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">607</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DBeilstein%26rft.auinit1%253DM.%2BA.%26rft.volume%253D93%26rft.issue%253D4%26rft.spage%253D607%26rft.epage%253D619%26rft.atitle%253DBrassicaceae%2Bphylogeny%2Band%2Btrichome%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.93.4.607%26rft_id%253Dinfo%253Apmid%252F21646222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6IjkzLzQvNjA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.5" data-doi="10.1073/pnas.0909766107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beilstein</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagalingum</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manchester</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathews</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dated molecular phylogenies indicate a Miocene origin for <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>(<span class="cit-issue">43</span>): <span class="cit-fpage">18724</span>–<span class="cit-lpage">18728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0909766107%26rft_id%253Dinfo%253Apmid%252F20921408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQzLzE4NzI0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.6" data-doi="10.1101/cshperspect.a003558"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blackburn</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Telomerase: an RNP enzyme synthesizes DNA.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">a003558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a003558%26rft_id%253Dinfo%253Apmid%252F20660025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiIzLzUvYTAwMzU1OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yODgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.7" data-doi="10.1101/gad.17446611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabili</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koziol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tazon-Vega</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">18</span>): <span class="cit-fpage">1915</span>–<span class="cit-lpage">1927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.17446611%26rft_id%253Dinfo%253Apmid%252F21890647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI1LzE4LzE5MTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjg4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castaings</span>  <span class="cit-name-given-names">L</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergonzi</span>  <span class="cit-name-given-names">S</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albani</span>  <span class="cit-name-given-names">M C</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemi</span>  <span class="cit-name-given-names">U</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">G</span></span></li></ol><cite>. <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolutionary conservation of cold-induced antisense RNAs of <em>FLOWERING LOCUS C</em> in <em>Arabidopsis thaliana</em> perennial relatives.</span> <abbr class="cit-jnl-abbrev">Nature Comm.</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">4457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BComm.%26rft.volume%253D5%26rft.spage%253D4457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.9" data-doi="10.1016/S0092-8674(00)80687-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blasco</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greider</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Secondary structure of vertebrate telomerase RNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">100</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">503</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChen%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D100%26rft.issue%253D5%26rft.spage%253D503%26rft.epage%253D514%26rft.atitle%253DSecondary%2Bstructure%2Bof%2Bvertebrate%2Btelomerase%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980687-X%26rft_id%253Dinfo%253Apmid%252F10721988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80687-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10721988&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085700000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.10" data-doi="10.1105/tpc.113.113480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Bergh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The <em>Tarenaya hassleriana</em> genome provides insight into reproductive trait and genome evolution of crucifers.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">2813</span>–<span class="cit-lpage">2830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.113480%26rft_id%253Dinfo%253Apmid%252F23983221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMjgxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yODgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.11" data-doi="10.1093/nar/gkr319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">X</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">P X</span></span></li></ol><cite>. <span class="cit-pub-date">2011</span> <span class="cit-article-title">psRNATarget: a plant small RNA target analysis server.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>(<span class="cit-issue">Web Server issue</span>):<span class="cit-fpage">W155</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr319%26rft_id%253Dinfo%253Apmid%252F21622958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOS9zdXBwbF8yL1cxNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjg4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.12" data-doi="10.1101/gr.132159.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Derrien</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussotti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanzer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Djebali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1775</span>–<span class="cit-lpage">1789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DDerrien%26rft.auinit1%253DT.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1775%26rft.epage%253D1789%26rft.atitle%253DThe%2BGENCODE%2Bv7%2Bcatalog%2Bof%2Bhuman%2Blong%2Bnoncoding%2BRNAs%253A%2BAnalysis%2Bof%2Btheir%2Bgene%2Bstructure%252C%2Bevolution%252C%2Band%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.132159.111%26rft_id%253Dinfo%253Apmid%252F22955988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTc3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yODgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.13" data-doi="10.1038/nature11233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Djebali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrie A Davis</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lassmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Landscape of transcription in human cells.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>(<span class="cit-issue">7414</span>): <span class="cit-fpage">101</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDjebali%26rft.auinit1%253DS.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D101%26rft.epage%253D108%26rft.atitle%253DLandscape%2Bof%2Btranscription%2Bin%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11233%26rft_id%253Dinfo%253Apmid%252F22955620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955620&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.14" data-doi="10.1089/cmb.2006.13.320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">D</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halldórsson</span>  <span class="cit-name-given-names">B V</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vernot</span>  <span class="cit-name-given-names">B</span></span></li></ol><cite>. <span class="cit-pub-date">2006</span> <span class="cit-article-title">A hybrid micro-macroevolutionary approach to gene tree reconstruction.</span> <abbr class="cit-jnl-abbrev">J. Comp. Biol.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">320</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComp.%2BBiol.%26rft.volume%253D13%26rft.spage%253D320%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2006.13.320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2006.13.320&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.15" data-doi="10.1126/science.1126316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chureau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samain</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissenbach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avner</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The Xist RNA gene evolved in eutherians by pseudogenization of a protein-coding gene.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">312</span>(<span class="cit-issue">5780</span>): <span class="cit-fpage">1653</span>–<span class="cit-lpage">1655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D312%26rft.issue%253D5780%26rft.spage%253D1653%26rft.epage%253D1655%26rft.atitle%253DThe%2Bxist%2BRNA%2Bgene%2Bevolved%2Bin%2Beutherians%2Bby%2Bpseudogenization%2Bof%2Ba%2Bprotein-coding%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1126316%26rft_id%253Dinfo%253Apmid%252F16778056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTIvNTc4MC8xNjUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.16" data-doi="10.1371/journal.pone.0002521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elisaphenko</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolesnikov</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogozin</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesterova</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A dual origin of the Xist gene from a protein-coding gene and a set of transposable elements.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e2521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElisaphenko%26rft.auinit1%253DE.%2BA.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253De2521%26rft.epage%253De2521%26rft.atitle%253DA%2Bdual%2Borigin%2Bof%2Bthe%2BXist%2Bgene%2Bfrom%2Ba%2Bprotein-coding%2Bgene%2Band%2Ba%2Bset%2Bof%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0002521%26rft_id%253Dinfo%253Apmid%252F18575625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0002521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18575625&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.17" data-doi="10.1016/j.pbi.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subramaniam</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Conserved noncoding sequences (CNSs) in higher plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">126</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DFreeling%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D126%26rft.epage%253D132%26rft.atitle%253DConserved%2Bnoncoding%2Bsequences%2B%2528CNSs%2529%2Bin%2Bhigher%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19249238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19249238&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265115400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.18" data-doi="10.1371/journal.pcbi.0020133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">C. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Phylogenetic reconstruction of orthology, paralogy, and conserved synteny for dog and human.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">e133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DGoodstadt%26rft.auinit1%253DL.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De133%26rft.epage%253De133%26rft.atitle%253DPhylogenetic%2Breconstruction%2Bof%2Borthology%252C%2Bparalogy%252C%2Band%2Bconserved%2Bsynteny%2Bfor%2Bdog%2Band%2Bhuman.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0020133%26rft_id%253Dinfo%253Apmid%252F17009864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0020133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17009864&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.19" data-doi="10.1016/j.molcel.2014.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gosai</span>  <span class="cit-name-given-names">J.</span>  <span class="cit-name-suffix">Sr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selamoglu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Global analysis of the RNA-protein interaction and RNA secondary structure landscapes of the <em>Arabidopsis</em> nucleus.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">57</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">376</span>–<span class="cit-lpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D57%26rft.spage%253D376%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.12.004%26rft_id%253Dinfo%253Apmid%252F25557549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25557549&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.20" data-doi="10.1007/s00109-013-1028-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutschner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hämmerle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diederichs</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MALAT1–a paradigm for long noncoding RNA function in cancer.</span> <abbr class="cit-jnl-abbrev">J. Mol. Med.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">791</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BMed.%26rft.volume%253D91%26rft.spage%253D791%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00109-013-1028-y%26rft_id%253Dinfo%253Apmid%252F23529762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00109-013-1028-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23529762&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.21" data-doi="10.3732/ajb.89.11.1826"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sytsma</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iltis</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Phylogeny of Capparaceae and Brassicaceae based on chloroplast sequence data.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">89</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1826</span>–<span class="cit-lpage">1842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DHall%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D89%26rft.issue%253D11%26rft.spage%253D1826%26rft.epage%253D1842%26rft.atitle%253DPhylogeny%2Bof%2BCapparaceae%2Band%2BBrassicaceae%2Bbased%2Bon%2Bchloroplast%2Bsequence%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.89.11.1826%26rft_id%253Dinfo%253Apmid%252F21665611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xMS8xODI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.22" data-doi="10.1038/nature11993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clausen</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bramsen</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Natural RNA circles function as efficient microRNA sponges.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>(<span class="cit-issue">7441</span>): <span class="cit-fpage">384</span>–<span class="cit-lpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHansen%26rft.volume%253D495%26rft.spage%253D384%26rft.atitle%253DNatural%2BRNA%2Bcircles%2Bfunction%2Bas%2Befficient%2BmicroRNA%2Bsponges.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11993%26rft_id%253Dinfo%253Apmid%252F23446346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11993&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23446346&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316650500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.23" data-doi="10.1038/ng.2684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haudry</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vello</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoen</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leclercq</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">891</span>–<span class="cit-lpage">898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D891%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2684%26rft_id%253Dinfo%253Apmid%252F23817568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23817568&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.24" data-doi="10.1016/j.celrep.2015.04.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hezroni</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koppstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avrutin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Principles of long noncoding RNA evolution derived from direct comparison of transcriptomes in 17 species.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1110</span>–<span class="cit-lpage">1122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D11%26rft.spage%253D1110%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.04.023%26rft_id%253Dinfo%253Apmid%252F25959816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.04.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25959816&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2881.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Husband</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldwin</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suda</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The incidence of polyploidy in natural plant populations: major patterns and evolutionary processes</span>, pp. <span class="cit-fpage">255</span>–<span class="cit-lpage">276</span> in <span class="cit-source"><em>Plant Genome Diversity Volume 2</em>, <em>Physical Structure</em>, <em>Behaviour and Evolution of Plant Genomes</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Greilhuber</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Dolezel</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Jaroslav</span>  <span class="cit-name-given-names">J. F.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.26" data-doi="10.1093/nar/gkn201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">M</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaretskaya</span>  <span class="cit-name-given-names">I</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raytselis</span>  <span class="cit-name-given-names">Y</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merezhuk</span>  <span class="cit-name-given-names">Y</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGinnis</span>  <span class="cit-name-given-names">S</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T L</span></span></li></ol><cite>. <span class="cit-pub-date">2008</span> <span class="cit-article-title">NCBI BLAST: a better web interface.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>(<span class="cit-issue">Web Server issue</span>):<span class="cit-fpage">W5</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn201%26rft_id%253Dinfo%253Apmid%252F18440982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEzOiIzNi9zdXBwbF8yL1c1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.27" data-doi="10.1105/tpc.114.126391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kagale</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nixon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huebert</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Polyploid evolution of the Brassicaceae during the Cenozoic era.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">2777</span>–<span class="cit-lpage">2791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.126391%26rft_id%253Dinfo%253Apmid%252F25035408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMjc3NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yODgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.28" data-doi="10.1371/journal.pgen.1003470"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kapusta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kronenberg</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transposable elements are major contributors to the origin, diversification, and regulation of vertebrate long noncoding RNAs.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e1003470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003470%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003470%26rft_id%253Dinfo%253Apmid%252F23637635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23637635&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.29" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzQvNzcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.30" data-doi="10.1186/s13059-014-0537-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrickson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transposable elements modulate human RNA abundance and splicing via specific RNA-protein interactions.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D537%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0537-5%26rft_id%253Dinfo%253Apmid%252F25572935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0537-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25572935&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.31" data-doi="10.1038/nrg3883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koenig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Beyond the thale: comparative genomics and genetics of <em>Arabidopsis</em> relatives.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">285</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3883%26rft_id%253Dinfo%253Apmid%252F25854181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25854181&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.32" data-doi="10.1101/gad.182121.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kretz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flockhart</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zehnder</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Suppression of progenitor differentiation requires the long noncoding RNA ANCR.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">338</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.182121.111%26rft_id%253Dinfo%253Apmid%252F22302877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjYvNC8zMzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjg4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.33" data-doi="10.1038/nature12209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">M. T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesch</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gosselin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heinz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rev-Erbs repress macrophage gene expression by inhibiting enhancer-directed transcription.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">498</span>(<span class="cit-issue">7455</span>): <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D498%26rft.spage%253D511%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12209%26rft_id%253Dinfo%253Apmid%252F23728303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23728303&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320929400058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.34" data-doi="10.1016/j.tibs.2014.02.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">M. T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenfeld</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glass</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Enhancer RNAs and regulated transcriptional programs.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">39</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">170</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiochem.%2BSci.%26rft.volume%253D39%26rft.spage%253D170%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2014.02.007%26rft_id%253Dinfo%253Apmid%252F24674738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2014.02.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24674738&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335426200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.35" data-doi="10.1038/sj.embor.7400857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lapidot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilpel</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide natural antisense transcription: coupling its regulation to its different regulatory mechanisms.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1216</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.embor.7400857%26rft_id%253Dinfo%253Apmid%252F17139297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6OToiNy8xMi8xMjE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.36" data-doi="10.1186/gb-2014-15-2-r40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petsch</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide discovery and characterization of maize long non-coding RNAs.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">R40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR40%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-2-r40%26rft_id%253Dinfo%253Apmid%252F24576388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-2-r40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24576388&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.37" data-doi="10.1105/tpc.112.102855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide analysis uncovers regulation of long intergenic noncoding RNAs in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">4333</span>–<span class="cit-lpage">4345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.112.102855%26rft_id%253Dinfo%253Apmid%252F23136377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMS80MzMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.38" data-doi="10.1186/1748-7188-6-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernhart</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Höner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siederdissen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tafer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ViennaRNA Package 2.0.</span> <abbr class="cit-jnl-abbrev">Algorithms Mol. Biol.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAlgorithms%2BMol%2BBiol%26rft.aulast%253DLorenz%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DViennaRNA%2BPackage%2B2.0.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1748-7188-6-26%26rft_id%253Dinfo%253Apmid%252F22115189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1748-7188-6-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22115189&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.39" data-doi="10.1104/pp.108.124867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ming</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Finding and comparing syntenic regions among <em>Arabidopsis</em> and the outgroups papaya, poplar, and grape: CoGe with Rosids.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">148</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1772</span>–<span class="cit-lpage">1781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.108.124867%26rft_id%253Dinfo%253Apmid%252F18952863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQ4LzQvMTc3MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yODgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.40" data-doi="10.1111/nph.13264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magallón</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gómez-Acevedo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sánchez-Reyes</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Hernández</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A metacalibrated time-tree documents the early rise of flowering plant phylogenetic diversity.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">207</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">437</span>–<span class="cit-lpage">453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D207%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13264%26rft_id%253Dinfo%253Apmid%252F25615647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13264&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25615647&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.41" data-doi="10.1093/gbe/evr116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Managadze</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogozin</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chernikova</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalina</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Negative correlation between expression level and evolutionary rate of long intergenic noncoding RNAs.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1390</span>–<span class="cit-lpage">1404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr116%26rft_id%253Dinfo%253Apmid%252F22071789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjMvMC8xMzkwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.42" data-doi="10.1111/nyas.12466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moghe</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">S.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The causes and molecular consequences of polyploidy in flowering plants.</span> <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">1320</span>(<span class="cit-issue">July</span>): <span class="cit-fpage">16</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.volume%253D1320%26rft.spage%253D16%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnyas.12466%26rft_id%253Dinfo%253Apmid%252F24903334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nyas.12466&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24903334&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.43" data-doi="10.1093/gbe/evs066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salse</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Decoding plant and animal genome plasticity from differential paleo-evolutionary patterns and processes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">917</span>–<span class="cit-lpage">928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs066%26rft_id%253Dinfo%253Apmid%252F22833223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjQvOS85MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjg4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.44" data-doi="10.1093/nar/gku1063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nawrocki</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burge</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daub</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rfam 12.0: updates to the RNA families database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D130</span>–<span class="cit-lpage">D137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1063%26rft_id%253Dinfo%253Apmid%252F25392425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0My9EMS9EMTMwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.45" data-doi="10.1038/nature12943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Necsulea</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soumillon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warnefors</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liechti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daish</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The evolution of lncRNA repertoires and expression patterns in tetrapods.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">505</span>(<span class="cit-issue">7485</span>): <span class="cit-fpage">635</span>–<span class="cit-lpage">640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D505%26rft.spage%253D635%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12943%26rft_id%253Dinfo%253Apmid%252F24463510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12943&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24463510&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330321000031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.46" data-doi="10.3389/fgene.2015.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">A. D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shippen</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolution of TERT-interacting lncRNAs: expanding the regulatory landscape of telomerase.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">September</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2015.00001%26rft_id%253Dinfo%253Apmid%252F25674101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2015.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25674101&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.47" data-doi="10.1101/gr.5972507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pennacchio</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loots</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nobrega</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovcharenko</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Predicting tissue-specific enhancers in the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">201</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5972507%26rft_id%253Dinfo%253Apmid%252F17210927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMjAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.48" data-doi="10.1261/rna.2658311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pink</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wicks</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caley</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Punch</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Pseudogenes: pseudo-functional or key regulators in health and disease?</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">792</span>–<span class="cit-lpage">798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2658311%26rft_id%253Dinfo%253Apmid%252F21398401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE3LzUvNzkyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.49" data-doi="10.1101/gr.6036807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ponjavic</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lunter</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Functionality or transcriptional noise? Evidence for selection within long noncoding RNAs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">556</span>–<span class="cit-lpage">565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6036807%26rft_id%253Dinfo%253Apmid%252F17387145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzUvNTU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.50" data-doi="10.1007/s00439-011-1008-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pontier</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gribnau</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Xist regulation and function eXplored.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">130</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">223</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DPontier%26rft.auinit1%253DD.%2BB.%26rft.volume%253D130%26rft.issue%253D2%26rft.spage%253D223%26rft.epage%253D236%26rft.atitle%253DXist%2Bregulation%2Band%2Bfunction%2Bexplored.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-011-1008-7%26rft_id%253Dinfo%253Apmid%252F21626138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-011-1008-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21626138&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.51" data-doi="10.1016/j.cell.2009.02.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reik</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution and functions of long noncoding RNAs.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">136</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">629</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPonting%26rft.auinit1%253DC.%2BP.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D641%26rft.atitle%253DEvolution%2Band%2Bfunctions%2Bof%2Blong%2Bnoncoding%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.02.006%26rft_id%253Dinfo%253Apmid%252F19239885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19239885&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263688200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.52" data-doi="10.1093/nar/gks980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">Xiaodong</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Honda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The common ancestral core of vertebrate and fungal telomerase RNAs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">450</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks980%26rft_id%253Dinfo%253Apmid%252F23093598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjQxLzEvNDUwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.53" data-doi="10.1007/s11103-013-0104-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">Xin</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide annotation of genes and noncoding RNAs of foxtail millet in response to simulated drought stress by deep sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">83</span>(<span class="cit-issue">4–5</span>): <span class="cit-fpage">459</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%26rft.volume%253D83%26rft.spage%253D459%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-013-0104-6%26rft_id%253Dinfo%253Apmid%252F23860794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-013-0104-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23860794&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325116600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.54" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yODgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.55" data-doi="10.1016/j.biochi.2011.07.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romito</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rougeulle</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Origin and evolution of the long non-coding genes in the X-inactivation center.</span> <abbr class="cit-jnl-abbrev">Biochimie</abbr> <span class="cit-vol">93</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1935</span>–<span class="cit-lpage">1942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimie%26rft.volume%253D93%26rft.spage%253D1935%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biochi.2011.07.009%26rft_id%253Dinfo%253Apmid%252F21820484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biochi.2011.07.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21820484&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.56" data-doi="10.1093/jxb/eru256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shuai</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide identification and functional prediction of novel and drought-responsive lincRNAs in <em>Populus trichocarpa</em>.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">65</span>(<span class="cit-issue">17</span>): <span class="cit-fpage">4975</span>–<span class="cit-lpage">4983</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Feru256%26rft_id%253Dinfo%253Apmid%252F24948679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjEwOiI2NS8xNy80OTc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.9.2881.57" data-doi="10.1073/pnas.0704145104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silvio</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgante</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomes</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kevin A Fengler</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Conserved noncoding genomic sequences associated with a flowering-time quantitative trait locus in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>(<span class="cit-issue">27</span>): <span class="cit-fpage">11376</span>–<span class="cit-lpage">11381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0704145104%26rft_id%253Dinfo%253Apmid%252F17595297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzI3LzExMzc2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-web" id="cit-6.9.2881.58"><div class="cit-metadata unstructured">Smit, A. F. A., R. Hubley, and P. Green, 2015 RepeatMasker Open-4.0. Available at: <a href="http://repeatmasker.org">http://repeatmasker.org</a>. Accessed: Month March, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.59" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML Version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIzMC85LzEzMTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjg4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.60" data-doi="10.1186/gb-2012-13-8-418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatoyannopoulos</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardison</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gingeras</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An encyclopedia of mouse DNA elements (Mouse ENCODE).</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DStamatoyannopoulos%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D418%26rft.epage%253D418%26rft.atitle%253DAn%2Bencyclopedia%2Bof%2Bmouse%2BDNA%2Belements%2B%2528Mouse%2BENCODE%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-8-418%26rft_id%253Dinfo%253Apmid%252F22889292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-8-418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22889292&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.61" data-doi="10.1534/genetics.111.137349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodhouse</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Altered patterns of fractionation and exon deletions in <em>Brassica rapa</em> support a two-step model of paleohexaploidy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1563</span>–<span class="cit-lpage">1574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137349%26rft_id%253Dinfo%253Apmid%252F22308264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjg4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.62" data-doi="10.1016/j.cell.2013.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulitsky</span>  <span class="cit-name-given-names">I</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">lincRNAs: genomics, evolution, and mechanisms.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">154</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">26</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.06.020%26rft_id%253Dinfo%253Apmid%252F23827673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23827673&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321327900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.63" data-doi="10.1016/j.cell.2011.11.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulitsky</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shkumatava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jan</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sive</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Conserved function of lincRNAs in vertebrate embryonic development despite rapid sequence evolution.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1537</span>–<span class="cit-lpage">1550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DUlitsky%26rft.auinit1%253DI.%26rft.volume%253D147%26rft.issue%253D7%26rft.spage%253D1537%26rft.epage%253D1550%26rft.atitle%253DConserved%2Bfunction%2Bof%2BlincRNAs%2Bin%2Bvertebrate%2Bembryonic%2Bdevelopment%2Bdespite%2Brapid%2Bsequence%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.11.055%26rft_id%253Dinfo%253Apmid%252F22196729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.11.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22196729&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298403400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.64" data-doi="10.1093/nar/gks915"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volders</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helsens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menten</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">LNCipedia: a database for annotated human incRNA transcript sequences and structures.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>(<span class="cit-supplement">D1</span>): <span class="cit-fpage">246</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D41%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks915&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.65" data-doi="10.1016/j.molcel.2011.08.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">H. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular mechanisms of long noncoding RNAs.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">904</span>–<span class="cit-lpage">914</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DWang%26rft.auinit1%253DK.%2BC.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D904%26rft.epage%253D914%26rft.atitle%253DMolecular%2Bmechanisms%2Bof%2Blong%2Bnoncoding%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.08.018%26rft_id%253Dinfo%253Apmid%252F21925379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.08.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21925379&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295309800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.66" data-doi="10.1261/rna.2536111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Washietl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Findeiss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalkhof</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Bergen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNAcode: robust discrimination of coding and noncoding regions in comparative sequence data.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">578</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2536111%26rft_id%253Dinfo%253Apmid%252F21357752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE3LzQvNTc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2881.67" data-doi="10.1101/gr.165035.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Washietl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garber</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolutionary dynamics and tissue specificity of human long noncoding rnas in six mammals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">616</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.165035.113%26rft_id%253Dinfo%253Apmid%252F24429298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI0LzQvNjE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/9/2881.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2867" title="Genome-Wide Divergence in the West-African Malaria Vector Anopheles melas" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2893" title="Identification of Genes in Candida glabrata Conferring Altered Responses to Caspofungin, a Cell Wall Synthesis Inhibitor" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/9"><span>Volume 6 Issue 9, September 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/9.cover-source.jpg?itok=nund1hRf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2881.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HrUtu8yQ95ykP1k4U4y0SU16mNjj7KuCOJtAQCA0qDs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/9/2881" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/9/2881" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/9/2881" class="active" data-icon-position="" data-hide-link-title="0">A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460295" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XzbJLhEELjjxPuRIta-DLl1I2TGSsGV9hNZkA_M5_QQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2881.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/9/2881" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Zw0KwG27uYz2J_-U3ng-AveKai_ofcMzFdXYK7MGo-Q" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/9/2881" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2881.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460295" id="node460295--2" data-pisa="ggg;6/9/2881" data-pisa-master="ggg;g3.116.030338" data-apath="/ggg/6/9/2881.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9896-1739" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew D. L. Nelson</span>, <span class="highwire-citation-author" data-delta="1">Evan S. Forsythe</span>, <span class="highwire-citation-author" data-delta="2">Upendra K. Devisetty</span>, <span class="highwire-citation-author" data-delta="3">David S. Clausen</span>, <span class="highwire-citation-author" data-delta="4">Asher K. Haug-Batzell</span>, <span class="highwire-citation-author" data-delta="5">Ari M. R. Meldrum</span>, <span class="highwire-citation-author" data-delta="6">Michael R. Frank</span>, <span class="highwire-citation-author" data-delta="7">Eric Lyons</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-3392-1389" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark A. Beilstein</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2881-2891;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030338 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460295--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew D. L. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.%2BL.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BD.%2BL.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9896-1739" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew D. L. Nelson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andrewnelson@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">andrewnelson@email.arizona.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Evan S. Forsythe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BS.%2BForsythe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forsythe%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BS.%2BForsythe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Upendra K. Devisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Ecosystems and Society, Oregon State University, Corvalis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Upendra%2BK.%2BDevisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devisetty%20UK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUpendra%2BK.%2BDevisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David S. Clausen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BClausen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clausen%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BS.%2BClausen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Asher K. Haug-Batzell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics Graduate Interdisciplinary Group, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asher%2BK.%2BHaug-Batzell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haug-Batzell%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAsher%2BK.%2BHaug-Batzell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ari M. R. Meldrum </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ari%2BM.%2BR.%2BMeldrum%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meldrum%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAri%2BM.%2BR.%2BMeldrum%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael R. Frank </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BFrank%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Frank%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BFrank%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Eric Lyons </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics Graduate Interdisciplinary Group, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BLyons%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lyons%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BLyons%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Mark A. Beilstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BBeilstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beilstein%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BA.%2BBeilstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3392-1389" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark A. Beilstein</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andrewnelson@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">andrewnelson@email.arizona.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/9/2881" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/9/2881&amp;title=A%20Genomic%20Analysis%20of%20Factors%20Driving%20lincRNA%20Diversification%3A%20Lessons%20from%20Plants" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/9/2881&amp;title=A%20Genomic%20Analysis%20of%20Factors%20Driving%20lincRNA%20Diversification%3A%20Lessons%20from%20Plants" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/9/2881&amp;title=A%20Genomic%20Analysis%20of%20Factors%20Driving%20lincRNA%20Diversification%3A%20Lessons%20from%20Plants" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/9/2881&amp;text=A%20Genomic%20Analysis%20of%20Factors%20Driving%20lincRNA%20Diversification%3A%20Lessons%20from%20Plants" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/9/2881&amp;title=A%20Genomic%20Analysis%20of%20Factors%20Driving%20lincRNA%20Diversification%3A%20Lessons%20from%20Plants" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/9/2881&amp;t=A%20Genomic%20Analysis%20of%20Factors%20Driving%20lincRNA%20Diversification%3A%20Lessons%20from%20Plants" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/9/2881&amp;title=A%20Genomic%20Analysis%20of%20Factors%20Driving%20lincRNA%20Diversification%3A%20Lessons%20from%20Plants" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/9/2881&amp;title=A%20Genomic%20Analysis%20of%20Factors%20Driving%20lincRNA%20Diversification%3A%20Lessons%20from%20Plants" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460295/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460295" id="node460295--3" data-pisa="ggg;6/9/2881" data-pisa-master="ggg;g3.116.030338" data-apath="/ggg/6/9/2881.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9896-1739" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew D. L. Nelson</span>, <span class="highwire-citation-author" data-delta="1">Evan S. Forsythe</span>, <span class="highwire-citation-author" data-delta="2">Upendra K. Devisetty</span>, <span class="highwire-citation-author" data-delta="3">David S. Clausen</span>, <span class="highwire-citation-author" data-delta="4">Asher K. Haug-Batzell</span>, <span class="highwire-citation-author" data-delta="5">Ari M. R. Meldrum</span>, <span class="highwire-citation-author" data-delta="6">Michael R. Frank</span>, <span class="highwire-citation-author" data-delta="7">Eric Lyons</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-3392-1389" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark A. Beilstein</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2881-2891;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030338 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460295--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew D. L. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.%2BL.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BD.%2BL.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9896-1739" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew D. L. Nelson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andrewnelson@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">andrewnelson@email.arizona.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Evan S. Forsythe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BS.%2BForsythe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forsythe%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BS.%2BForsythe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Upendra K. Devisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Ecosystems and Society, Oregon State University, Corvalis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Upendra%2BK.%2BDevisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devisetty%20UK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUpendra%2BK.%2BDevisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David S. Clausen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BClausen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clausen%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BS.%2BClausen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Asher K. Haug-Batzell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics Graduate Interdisciplinary Group, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asher%2BK.%2BHaug-Batzell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haug-Batzell%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAsher%2BK.%2BHaug-Batzell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ari M. R. Meldrum </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ari%2BM.%2BR.%2BMeldrum%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meldrum%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAri%2BM.%2BR.%2BMeldrum%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael R. Frank </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BFrank%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Frank%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BFrank%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Eric Lyons </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics Graduate Interdisciplinary Group, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BLyons%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lyons%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BLyons%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Mark A. Beilstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BBeilstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beilstein%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BA.%2BBeilstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3392-1389" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark A. Beilstein</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andrewnelson@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">andrewnelson@email.arizona.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2881&amp;rft.epage=2891&amp;rft.atitle=A%20Genomic%20Analysis%20of%20Factors%20Driving%20lincRNA%20Diversification%3A%20Lessons%20from%20Plants&amp;rft.volume=6&amp;rft.issue=9&amp;rft.date=2016-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Nelson%2C+Andrew+D.+L.&amp;rft.au=Forsythe%2C+Evan+S.&amp;rft.au=Devisetty%2C+Upendra+K.&amp;rft.au=Clausen%2C+David+S.&amp;rft.au=Haug-Batzell%2C+Asher+K.&amp;rft.au=Meldrum%2C+Ari+M.+R.&amp;rft.au=Frank%2C+Michael+R.&amp;rft.au=Lyons%2C+Eric&amp;rft.au=Beilstein%2C+Mark+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460295/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460295/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460295/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460295/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460295/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460295/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460295/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460295/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460295/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460295/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460295/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460295/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/9/2881" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/9/2881.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/9/2881.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/9/2881.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/9/2881.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
