<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/9/2919" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/9/2919.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/9/2919.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/9/2919.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/9/2919/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/9/2919.atom" />
<meta name="HW.pisa" content="ggg;6/9/2919" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Optimizing Training Population Data and Validation of Genomic Selection for Economic Traits in Soft Winter Wheat" />
<meta name="DC.Identifier" content="10.1534/g3.116.032532" />
<meta name="DC.Date" content="2016-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Hoffstetter et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection (GS) is a breeding tool that estimates breeding values (GEBVs) of individuals based solely on marker data by using a model built using phenotypic and marker data from a training population (TP). The effectiveness of GS increases as the correlation of GEBVs and phenotypes (accuracy) increases. Using phenotypic and genotypic data from a TP of 470 soft winter wheat lines, we assessed the accuracy of GS for grain yield, Fusarium Head Blight (FHB) resistance, softness equivalence (SE), and flour yield (FY). Four TP data sampling schemes were tested: (1) use all TP data, (2) use subsets of TP lines with low genotype-by-environment interaction, (3) use subsets of markers significantly associated with quantitative trait loci (QTL), and (4) a combination of 2 and 3. We also correlated the phenotypes of relatives of the TP to their GEBVs calculated from TP data. The GS accuracy within the TP using all TP data ranged from 0.35 (FHB) to 0.62 (FY). On average, the accuracy of GS from using subsets of data increased by 54% relative to using all TP data. Using subsets of markers selected for significant association with the target trait had the greatest impact on GS accuracy. Between-environment prediction accuracy was also increased by using data subsets. The accuracy of GS when predicting the phenotypes of TP relatives ranged from 0.00 to 0.85. These results suggest that GS could be useful for these traits and GS accuracy can be greatly improved by using subsets of TP data." />
<meta name="DC.Contributor" content="Amber Hoffstetter" />
<meta name="DC.Contributor" content="Antonio Cabrera" />
<meta name="DC.Contributor" content="Mao Huang" />
<meta name="DC.Contributor" content="Clay Sneller" />
<meta name="article:published_time" content="2016-09-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Optimizing Training Population Data and Validation of Genomic Selection for Economic Traits in Soft Winter Wheat" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection (GS) is a breeding tool that estimates breeding values (GEBVs) of individuals based solely on marker data by using a model built using phenotypic and marker data from a training population (TP). The effectiveness of GS increases as the correlation of GEBVs and phenotypes (accuracy) increases. Using phenotypic and genotypic data from a TP of 470 soft winter wheat lines, we assessed the accuracy of GS for grain yield, &lt;i&gt;Fusarium&lt;/i&gt; Head Blight (FHB) resistance, softness equivalence (SE), and flour yield (FY). Four TP data sampling schemes were tested: (1) use all TP data, (2) use subsets of TP lines with low genotype-by-environment interaction, (3) use subsets of markers significantly associated with quantitative trait loci (QTL), and (4) a combination of 2 and 3. We also correlated the phenotypes of relatives of the TP to their GEBVs calculated from TP data. The GS accuracy within the TP using all TP data ranged from 0.35 (FHB) to 0.62 (FY). On average, the accuracy of GS from using subsets of data increased by 54% relative to using all TP data. Using subsets of markers selected for significant association with the target trait had the greatest impact on GS accuracy. Between-environment prediction accuracy was also increased by using data subsets. The accuracy of GS when predicting the phenotypes of TP relatives ranged from 0.00 to 0.85. These results suggest that GS could be useful for these traits and GS accuracy can be greatly improved by using subsets of TP data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/09/01" />
<meta name="citation_mjid" content="ggg;6/9/2919" />
<meta name="citation_id" content="6/9/2919" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/9/2919" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/9/2919.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/9/2919.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/9/2919.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.032532" />
<meta name="citation_pmid" content="27440921" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2919" />
<meta name="citation_lastpage" content="2928" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Amber Hoffstetter" />
<meta name="citation_author_institution" content="Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691" />
<meta name="citation_author" content="Antonio Cabrera" />
<meta name="citation_author_institution" content="Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691" />
<meta name="citation_author" content="Mao Huang" />
<meta name="citation_author_institution" content="Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691" />
<meta name="citation_author" content="Clay Sneller" />
<meta name="citation_author_institution" content="Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691" />
<meta name="citation_author_email" content="sneller.5@osu.edu" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=R. Abdollahi-Arpanahi;citation_author=A. Nejati-Javaremi;citation_author=A. Pakdel;citation_author=M. Moradi-Shahrbabak;citation_author=G. Morota;citation_title=Effect of allele frequencies, effect sizes and number of markers on prediction of quantitative traits in chickens.;citation_pages=123-133;citation_volume=131;citation_year=2014;citation_pmid=24397350;citation_doi=10.1111/jbg.12075" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=F. G. Asoro;citation_author=M. A. Newell;citation_author=W. D. Beavis;citation_author=M. P. Scott;citation_author=J.-L. Jannink;citation_title=Accuracy and training population design for genomic selection on quantitative traits in elite North American oats.;citation_pages=132-144;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.02.0007" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Genomewide Selection for Rapid Introgression of Exotic Germplasm in Maize;citation_pages=419-425;citation_volume=49;citation_year=2009;citation_issue=2;citation_doi=10.2135/cropsci2008.08.0452" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. Cabrera;citation_author=E. Souza;citation_author=M. Guttieri;citation_author=A. Sturbaum;citation_author=A. Hoffstetter;citation_title=Genetic diversity, linkage disequilibrium, and genome evolution in soft winter wheat.;citation_pages=2433-2448;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.09.0601" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Cabrera;citation_author=M. Guttieri;citation_author=N. Smith;citation_author=E. Souza;citation_author=A. Sturbaum;citation_title=Identification of milling and baking quality QTL in multiple soft wheat mapping populations.;citation_pages=2227-2242;citation_volume=128;citation_year=2015;citation_pmid=26188588;citation_doi=10.1007/s00122-015-2580-3" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Accuracy of genomewide selection for different traits with constant population size, heritability, and number of markers.;citation_author=E. Combs;citation_author=R. Bernardo;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. l. Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Perez;citation_author=J. Hickey;citation_author=J. Burgueno;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R Package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Genomic selection.;citation_pages=323-330;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076469;citation_doi=10.1111/j.1439-0388.2007.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=J.-L. Jannink;citation_author=H. Iwata;citation_author=E. Souza;citation_author=M. E. Sorrells;citation_title=Genomic Selection Accuracy for Grain Quality Traits in Biparental Wheat Populations.;citation_pages=2597-2606;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2011.05.0253" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. L. Heffner;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome.2010.12.0029" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. Hoffstetter;citation_author=A. Cabrera;citation_author=C. Sneller;citation_title=Identifying quantitative trait loci for economic traits in an elite soft red winter wheat population.;citation_pages=547-558;citation_volume=56;citation_year=2016;citation_issue=2;citation_doi=10.2135/cropsci2015.06.0332" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_title=Genomic selection for wheat traits and trait stability;citation_author=M. Huang;citation_author=A. Cabrera;citation_author=C. Sneller;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Isidro;citation_author=J.-L. Jannink;citation_author=D. Akdemir;citation_author=J. Poland;citation_author=N. Heslot;citation_title=Training set optimization under population structure in genomic selection.;citation_pages=145-158;citation_volume=128;citation_year=2015;citation_pmid=25367380;citation_doi=10.1007/s00122-014-2418-4" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J. L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomes Genetics;citation_author=B. Lado;citation_author=I. Matus;citation_author=A. Rodriguez;citation_author=L. Inostroza;citation_author=J. Poland;citation_title=Increased genomic prediction accuracy in wheat breeding through spatial adjustment of field trial data;citation_pages=2105-2114;citation_volume=3;citation_year=2013;citation_pmid=24082033" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=A. Liaw;citation_author=M. Wiener;citation_title=Classification and regression by randomForest.;citation_pages=18-22;citation_volume=2;citation_year=2002;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Liu;citation_author=Z. A. Abate;citation_author=H. Lu;citation_author=T. Musket;citation_author=G. L. Davis;citation_title=QTL associated with Fusarium head blight resistance in the soft red winter wheat Ernie.;citation_pages=417-427;citation_volume=115;citation_year=2007;citation_issue=3;citation_pmid=17569028;citation_doi=10.1007/s00122-007-0577-2" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=S. Y. Liu;citation_author=M. D. Christopher;citation_author=C. A. Griffey;citation_author=M. D. Hall;citation_author=P. G. Gundrum;citation_title=Molecular characterization of resistance to Fusarium Head Blight in U.S. soft red winter wheat breeding line VA00W–38.;citation_pages=2283-2292;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2012.03.0144" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J. L. Jannink;citation_title=Potential and optimization of genomic selection for Fusarium Head Blight resistance in six-row barley.;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=G. Moser;citation_author=M. S. Khatkar;citation_author=B. J. Hayes;citation_author=H. W. Raadsma;citation_title=Accuracy of direct genomic values in Holstein bulls and cows using subsets of SNP markers.;citation_pages=37-37;citation_volume=42;citation_year=2010;citation_pmid=20950478;citation_doi=10.1186/1297-9686-42-37" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Perez;citation_author=G. de los Campos;citation_author=J. Crossa;citation_author=D. Gianola;citation_title=Genomic-enabled prediction based on molecular markers and pedigree using the bayesian linear regression package in R.;citation_pages=106-116;citation_volume=3;citation_year=2010;citation_pmid=21566722;citation_doi=10.3835/plantgenome2010.04.0005" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Rutkoski;citation_author=J. Benson;citation_author=Y. Jia;citation_author=G. Brown-Guedira;citation_author=J. L. Jannink;citation_title=Evaluation of genomic prediction methods for Fusarium Head blight resistance in wheat.;citation_pages=51-61;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.02.0001" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. E. Rutkoski;citation_author=J. A. Poland;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=S. Bhavani;citation_title=Genomic selection for quantitative adult plant stem rust resistance in wheat.;citation_pages=1-10;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. H. Sallam;citation_author=J. B. Endelman;citation_author=J.-L. Jannink;citation_author=K. P. Smith;citation_title=Assessing genomic selection prediction accuracy in a dynamic barley breeding population.;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.05.0020" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=T. Schulz-Streeck;citation_author=J. O. Ogutu;citation_author=A. Gordillo;citation_author=Z. Karaman;citation_author=C. Knaak;citation_title=Genomic selection allowing for marker-by-environment interactions.;citation_pages=532-538;citation_volume=132;citation_year=2013;citation_doi=10.1111/pbr.12105" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Smith;citation_author=M. Guttieri;citation_author=E. Souza;citation_author=J. Shoots;citation_author=M. Sorrells;citation_title=Identification and validation of QTL for grain quality traits in a cross of soft wheat cultivars Pioneer Brand 25R26 and Foster.;citation_pages=1424-1436;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2010.04.0193" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. I. Vazquez;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=G. de los Campos;citation_author=D. Gianola;citation_title=Predictive ability of subsets of single nucleotide polymorphisms with and without parent average in US Holsteins.;citation_pages=5942-5949;citation_volume=93;citation_year=2010;citation_issue=12;citation_pmid=21094768;citation_doi=10.3168/jds.2010-3335" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=K. A. Weigel;citation_author=G. de los Campos;citation_author=O. González-Recio;citation_author=H. Naya;citation_author=X. L. Wu;citation_title=Predictive ability of direct genomic values for lifetime net merit of Holstein sires using selected subsets of single nucleotide polymorphism markers;citation_pages=5248-5257;citation_volume=92;citation_year=2009;citation_issue=10;citation_pmid=19762843;citation_doi=10.3168/jds.2009-2092" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Optimizing Training Population Data and Validation of Genomic Selection for Economic Traits in Soft Winter Wheat" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection (GS) is a breeding tool that estimates breeding values (GEBVs) of individuals based solely on marker data by using a model built using phenotypic and marker data from a training population (TP). The effectiveness of GS increases as the correlation of GEBVs and phenotypes (accuracy) increases. Using phenotypic and genotypic data from a TP of 470 soft winter wheat lines, we assessed the accuracy of GS for grain yield, Fusarium Head Blight (FHB) resistance, softness equivalence (SE), and flour yield (FY). Four TP data sampling schemes were tested: (1) use all TP data, (2) use subsets of TP lines with low genotype-by-environment interaction, (3) use subsets of markers significantly associated with quantitative trait loci (QTL), and (4) a combination of 2 and 3. We also correlated the phenotypes of relatives of the TP to their GEBVs calculated from TP data. The GS accuracy within the TP using all TP data ranged from 0.35 (FHB) to 0.62 (FY). On average, the accuracy of GS from using subsets of data increased by 54% relative to using all TP data. Using subsets of markers selected for significant association with the target trait had the greatest impact on GS accuracy. Between-environment prediction accuracy was also increased by using data subsets. The accuracy of GS when predicting the phenotypes of TP relatives ranged from 0.00 to 0.85. These results suggest that GS could be useful for these traits and GS accuracy can be greatly improved by using subsets of TP data." />
<meta name="og-title" property="og:title" content="Optimizing Training Population Data and Validation of Genomic Selection for Economic Traits in Soft Winter Wheat" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/9/2919" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection (GS) is a breeding tool that estimates breeding values (GEBVs) of individuals based solely on marker data by using a model built using phenotypic and marker data from a training population (TP). The effectiveness of GS increases as the correlation of GEBVs and phenotypes (accuracy) increases. Using phenotypic and genotypic data from a TP of 470 soft winter wheat lines, we assessed the accuracy of GS for grain yield, Fusarium Head Blight (FHB) resistance, softness equivalence (SE), and flour yield (FY). Four TP data sampling schemes were tested: (1) use all TP data, (2) use subsets of TP lines with low genotype-by-environment interaction, (3) use subsets of markers significantly associated with quantitative trait loci (QTL), and (4) a combination of 2 and 3. We also correlated the phenotypes of relatives of the TP to their GEBVs calculated from TP data. The GS accuracy within the TP using all TP data ranged from 0.35 (FHB) to 0.62 (FY). On average, the accuracy of GS from using subsets of data increased by 54% relative to using all TP data. Using subsets of markers selected for significant association with the target trait had the greatest impact on GS accuracy. Between-environment prediction accuracy was also increased by using data subsets. The accuracy of GS when predicting the phenotypes of TP relatives ranged from 0.00 to 0.85. These results suggest that GS could be useful for these traits and GS accuracy can be greatly improved by using subsets of TP data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
    <title>Optimizing Training Population Data and Validation of Genomic Selection for Economic Traits in Soft Winter Wheat | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"S9Xh9n9M49X8CjqaF2o4uvrxFi4cTVlgnIUjOY7jfmA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460292","apath":"\/ggg\/6\/9\/2919.atom","pisa":"ggg;6\/9\/2919","ac":{"\/ggg\/6\/9\/2919.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/9\/2919.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2919"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2919"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/9\/2919":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HgysOm3Pfso1PK1BXqpfdRwAHMQyTWGKB0RzQex84J6hzumBw1SKvQGpMLys0\\\\\\\/ckDr+7tZSONWd+QJDmSWHqwzDiBPDaBPDPUJt9bcsT9fEIeekoclpKIZUwWs\\\\\\\/wZ+ChvwFpePfLu4qPWxTKiJjySZmvg3EExJ9PmUXD0MAzUCha1fhjZrYAJBrW79YqhseECVLmzOcvHG2hLedDgEVg1uu7omnQfGmh71SsWK8G+GbLGFvyTxHPkNY\\\\\\\/vN6h5vp\\\\\\\/239xv4kjF08mn4ehhRmHZHcQbj6mNpf4i287mW\\\\\\\/x8EXtcmfTQTHr6sPnaB9ANGl5OiBo\\\\\\\/H5tRGrSpf5fTy+MN4v6vPHZHga7P36Sky3uxAsHXbEFdvQQlxPQcLMzyj6rmBAiaERYO+7hNekOwPdEGl2dgKz08riV413km0mYCGy1iLCFtUrrUxhHhmwl+7L5DcwIoQWXojd7j2tzy3RSGyp6i1H3E3uv0\\\\\\\/1vHzVbj5Ldi2n4CBtcUPz\\\\\\\/p4DBMuYxjq+3QOPUnWPoHuQxho9l4py7DfD5M7Wlf5DLg2\\\\\\\/2seDGOI9MP2F1WZtMOHKhe8I7zNdHVS+vd0qkBfDqwfM8ca2QyY+LPR2yk+QNlqWvCAUP\\\\\\\/dgln0waYXp1ACr9AqPsq1WJhmYhTwSEhl7qvtLc8BvtGjhRjEpHNkY8OEwlEMKH1vkHH9ahHwr8s29tE4LHr4oHqoq1yIMbKLzzcDeQ684tuXoTXRbjfrQtjwlE\\\\\\\/y3UzoLTWskMvyM9ocbVBrw9fAXmKUb17BofFGcE9JEU9bLibjsJduuHj3bwFi40rFP4uznbSnMass6NpPWEbPdmW0nhfiAJRHuDwRTz\\\\\\\/ARvpIq7FzfLMT6+iVTesgSb\\\\\\\/pMNahQpQF7PPJOxS1UrstjVPSA633WjtyTxg3wcCtOhCuENKgD9G99Sth+yBgSKv6GeRXaPSlTjzUy1wPsI1STzEBo2qKaYKHxgazh5GahZjMsZ80L4SgI5G+Mq8yiubiB3sT0KIxaXKyPQUaq\\\\\\\/wxG2vXYiVx0NeP8Ucva4ZHmiZP9RRBxQeNVmgMNyyHR1IRfmgtNnrIEPn\\\\\\\/mBNI\\\\\\\/9eIueJ1SI82o0sJ7\\\\\\\/+DeAUjR28xmVrNlLYxfq\\\\\\\/P08E6amC9H7Gea4XTpAdk5EPLQ08crVW4eLT6+21O0gKCrqEbB44iJqSX8Ei8FzPIrI0l0DOKioQNVvLxGcU0ADr1P+vO6XhDUWjzBGT4Kh3904xBrmhhuVwd\\\\\\\/3ZI1ObpOaB+Q3XGIwGLGBk4c+635r8YSaEJf4\\\u0022,\\\u0022iv\\\u0022:\\\u0022R0xF1qpL0VTCxP9E8GBQZA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022da506373819d55ed54753e14d78b381353e190170c048e1ebd28d8134e3a717c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225nk7EkNba7Uu1UlwAf9EetCItCHfkFUTrb8qUuJgo5VU\\\\\\\/7vulOWOJGojB3bfAP97AZRcILZtBeg8M1hs5MyeVZKsCDYZgpf714d8dRbYJFs6F\\\\\\\/5Kjf8ul2mjUIAE0\\\\\\\/zaO2B8mGMMqURPeXBVxfWq17wnrs8vLMqFyMUBPQ9ZxqLglgIqDQycAX4Rxwoc4bXm1NOZPW5iUNHObZy0pm7nwYz4fcbNrVvoeshhRShcdrzfKUNI5Z+ZIJRhhp5ki5Gbr8P1JBCyFESXuPsGJD5Q5Nu2x5jQY1Nfn834c+Ho9jtEEcG3g3nEmlN8Dd45VZ8lfxHxcQCwZlARwXtirDTvOJDCkUqPasMJGw97M\\\\\\\/YGTDPeC0DiryxrUUcm+c\\\\\\\/D8goQSzAtTGOQjl8kohomCNJPK6GqJ1fPbRt5Sh1GzUhQ9MONrccwl3HyvvoUvZ\\\\\\\/sTbqFHqFh7j6YDO5POwtBaZl1WIWMCEx01t9thI0HKflFMMf8B4QgR2Vp8x6kFC9mq0zt2rqciu9Bvha7iPp9oyn9E8XdwlY\\\\\\\/IFzyGXvIFrKwSVdyUKqUAwoTKF75KcKqUiJwDFTVcJWHbqF5XVCXGOowkJ5QpGjOefXMJVG9dXMDpZEMQwIpdnPWPvYGeXAEZ2SCLQQl1xCkKorb5Uw7nqcWXE4Py4MTqS6Gyb5fFh5PFcttbW5435InlqP\\\\\\\/bVQPSbGdLH383gMZJWtU\\\\\\\/2GQiFoacNdM21ajyn0XCWcckReH63ub1IoqxTJtVmRLZPHxdKUtsbvopcP5UD5RYGYt+wOyb4GDi3TIa\\\\\\\/nKBD8e20ZvNO5Bt2ptIbOjC2R5tD7YP+MStH+vXkqS+5cE77isp9umu\\\\\\\/SXgPp7u\\\\\\\/uhtaEbNFqLde2ObjsTfteUih\\\\\\\/K1LWXWv6Lc1Jw\\\\\\\/cFQeqH0Abik5hPPb\\\\\\\/WpcCEFF8Kpw+gR3+eoiwVwnzx9lIOsrNfW971fkVML++15RAySkMADYtXUp0JPWGCcmnVhBPl5Y3Tp0C4ZZalbbn6FEApbWQhokf1js+nLH8LE01Sbe64aczv13I2SeU35cDkDVTInvLsic74oKLzfDGaxHYkt6cgBa3rMFzEUFwFtYPNOFYc01MMnfoPp5\\\\\\\/W1qXKBcwoscuByIuga8mzC9iReOYiWNqGeeQ1hLCHaA5m55aHw\\\\\\\/BHrk53aqtPQs73OiuU657G+nk0vilzWrhNUX0JqR\\\\\\\/d0DkO2fWvi71RrBGtUtP9xPprVjsZXc6WnAJSuX53h9tTQLoM2X0GNoTuElhE97pvtDDaKnzRq7WXmuQqE6RBU567jxTcG\\\\\\\/Tz0GrCyRSRV8Hf9dDbCFkl7gEdzOJy+9BNhghItX8yJHfLVDmx0XXYKtbf2+wnt0bNC2cBICpEcmBqa216oukGdL8fydL\\\\\\\/mNO68FjSgkEP8v6fqGM3tDho5fvbS+k\\\\\\\/djkbHi3hPLU8mDUFJvqfNDaIvpv6Ynpskl279MZshoUUXsox5dG\\\\\\\/Zix\\\\\\\/kdTTOPhpTb\\\\\\\/pA49VMkLriHYA9yOVj2PU0s54I2dX0c2eKobhROpoACkPyUmGcRdH0rOXMnxVnlhmpEYWxE746zmyMQDWgZuKuEjf50b7o7iBZxGbzrRjOHD6f\\\\\\\/Ir99S1Zma3GVgJdMyGjJzn39TjdeFs\\\\\\\/uLw+Cq5PX3Rqr5cVBxyZgf0fhmyUCTnSb1P\\\\\\\/X1uH2iJpFYxnfi5I7i9JfJqpus9uQv+9Mp9jOHKsU\\\\\\\/eQAK1m53ec7LKVwOrRXButRc72ugQI\\\\\\\/OukheVW2R8ri4PsNq+1v5c9fI+5gBI7KMVHzdgMPG6v3LFfblnUTdmvsUoWfXjC0\\\\\\\/khgIKYD+VaiZJKpHIzWom1BWDw2H1efMhGU18GKYn6njx3gHFKCWkB\\\\\\\/Ore752RorVe5Zoy+ic2ntBDDWdWsQVU9o6A0pkvy0uiavwa07MjMHp8qEOxU+SXSJMuB\\\\\\\/ClsEWsyL3SZMpbPwGSQyPF2YwbWSBHLAeoXgYgaq\\\\\\\/1hM3GRqA8byjvdgTNBxVfO1SjXLr6fA3\\\\\\\/tAAkLSklJ5hcWH9KPiTgb+3I6ZGU84VtKovEuuV720tdy0bYoJYXcRiEhiB2mgk8yMw7fn9F9jRwEcLSEPX8MWsGZN5eAesSx871PrKc1zNQTvPGgbKOuCEoD6gzB\\\\\\\/L4WRtpTVAWBrbJzSgX\\\\\\\/jgY+N2N33YWJ\\\\\\\/A7vODKRBYHnljLW72nprHprEf9wyUnp+qP6mEfxSVs0wdOWWe1uMfb+cXP15qoWh15tUPgGGbNMm0PJXx5opHYjliQqzy1RpzAYMJsA1y432Sw8CgUT52cugbharc6QAsnfy3DFIbgryKgU6AeHjgtbj+ElQMIIj4aPX31kCmZm2Gw5pHLKVJy9ikuURLivxUjx5HJZvkfb5OmVDqOlUXmtKLQirqGWN69w08fDEK8wjB9pR7fo3Nv0act31QKo6PAdKn913UAEe54jQR8U3fB5DdByJZsmC\\\\\\\/1Z36aTDtXWUIqS1kP\\\\\\\/5lRLTsZCTT4FwPkoLplMkM6t8qN\\\\\\\/TGA+7NKyQWAEu3DHYbflgl087NAK6kFmg+GQhxpXvnxEynGxzSMLcf\\\\\\\/aOAYTlHHIFQqVJ4b9rykuG7vKmm+Z7RRr4R3HB2aOvTT4TEGXMxdwewg+tgvFbyN\\\\\\\/bIXb3CtzxDan+85HYUi1aHmNGWnvFoqVeMLRQiprte\\\\\\\/LS4Njl9Ox2uw6GgIwY0S8pGRzpY+pO8IZMRSNY\\\\\\\/5gjgzBWtP1PXBgZkZzvd1rGYf\\\\\\\/W7nqfSuIqD6WmxAGNI0kuJZCrjzbUYh1GOsQJgF0Y\\\\\\\/IbQkSPjyfmA5CD+VqNW50mtv3QSwp7Vm2EoFAU5c9iQmya9sBDW9sTlFIlDXOHhlMHaaNg2DJlq6TKBu\\\\\\\/oJG0qyCPhQAMVuPCs1giUKQEcEwwj74KOsNifTAh55XGTWYLzbMYi3DnOFAE2uO3xMJM8LRF7h5Tc278vMHb6qRJ9ET5lLHqJww0+kkMBDfs3coTdb5+9RcoqlZXrw6NsOoG\\\\\\\/h67H\\\\\\\/0wIKP1BPYZyjOhvT4LZr4H7l5Rt\\\\\\\/hA8AAfVDfrHFjQXjVBtq\\\\\\\/9kOnKg81vC\\\\\\\/8X9\\\\\\\/xVXEOOWDy5ZOIOOM5wlbdEuouViEBYHwguvyq0m0zSb+E0w3kIzhMvHWiqtphVi7LyWfp6UpKk2T5HP8pAb9NTT4NyDAM2+X8hVGTimOpnfmYe6RorLjD2sudN9RJuGimikJHIq7oUAhxCdxt6eY8\\\\\\\/MEYW1zVGWCoHiqMUfHmc2wqtCp3YnSFdah8sfbNjN1Fsbi6yvJS6ecri9FuDGARE48HwJBJahJu+7zTtzb7jqyiAW3s9ucKmbJbrTS+3t2LRmdcCyJAqL66rFg\\\\\\\/FIhsZ7dT5EVYsSvKlvuRiQlrP5JKaZOwqknTR4RyFGjQu+Ve7\\\\\\\/2BIol2B+oCIGxj3xv2DZ9gtcmRqlHEPYOQh+Q2Uf+VJnEDLAPa45CVKdwHtI85AkpmupFfREBnBW8AWkmhDoOMWd7nlAx5EVxa8zeJZsdhHhkXejCq0QxPRhsn5gp3Rg\\\\\\\/qbRtlGSwev80Je71dpPyx+HWUQgDLlADacaJ1IZuAUeB93PhnSecvtDp\\\\\\\/l1\\\\\\\/bEsrGMUA4MBICPKVEMy2TuW9FzkIgCkZVyzneXYgOuDiFNETLxKWBfrmtPK1MfA7Z70zE0q3qaRjAtEQXlABBv1jIHlvtofUA\\\\\\\/DgFNhTedE44P8t3cc\\\\\\\/weBn9CD\\\\\\\/z+od+TygQCIcUxfM912GCAP57q8gVdjLun+MNGj3Ey3LFzMdZEfPkQQPrIFggJmJ\\\\\\\/NovuZm8Oo6gJeU8yevQs2ZyACyKOiXY3ov0KPXZ1hEyuxbdWnEaurDBxHVHzw\\\\\\\/APSAFuiYOe64gzeGwDOQMeQ1qvuH\\\\\\\/0\\\\\\\/2V3bQWHGPxb1UTEvG4Qor34Ek7AMBsa8OTyqMZFHyCPeW\\\\\\\/3KzIufruqslXDb+L3BmP3rOZQvXNe8\\\\\\\/FSvTM5ZTLApPDYvEH5tBCLrTcUCwYupwOPE1ZAGIEHe2UxVgLjFTAeGpSAvEdeWQuq43TxLEYrylY9rOzr68p9YOhH68ZhyfemAwuEdY9Alh+uVnzDbz9926mxpbPVjNQp13yXs+NhnMbBEMJg2DNQVgAfaaZF+qttW\\\\\\\/VmJHj2sbpvRLE+iSyxsrIKlTCn+ZgUqVOW7bzFll+9QKpTL9yCrqs4fUyuTbFp3v3KIkuraHiO1jd6dvhudnieDoBt8LqGMuVGsNtp6QSh9gr0v8owg883GrxDql3R+x4W8ncwcrlolFnRoFF6i1TDj9JR6nXgeGFnBmmcOw6hwBNN7apuH\\\\\\\/Sv8r2WlcAeXYf0+4hXpuUmxVI5LF1mucCSg9JAUzEeR22f4UlenIJkXytXK1Yo72K0x\\\\\\\/dT1s95FHbk0YcoI8QWA0R96WrIxfMpA09e+AmIDUd+6NiOXr8gnmVkbZJFkVZgox4yUGPgMAoV+H1m78XzxsCloL21k7cwao247399NRrllxM95795STYRvvB4AEzAnungltS6\\\\\\\/08MtAGy\\\\\\\/KFn1g8CD99n0pCeC1ds9cIbw82LUSQs0EdOV4yEI0vaaXwb6hbqVygcEmL8kmWFFhKSaOZp1RWVlnFd9DksjV2LBJooBgLMNusFKSbIW\\\\\\\/j4cpp6T83u93o92q7nc6R++cKN5rnKsyz+7fPi1OP5rBo0OabsIHlw+rmumkL3xY2GG5b7BG4iagVs7y5iYPPRn6CkVealahEGLd81NW\\\\\\\/7P6Uf9CX\\\\\\\/RvEQjx9AqXNPVQBL69P8G9msQT9oRpPVPW0r3CT9Krd5QFu+hs9aPNyIVHsEv0ykhEDIthW5Bc\\\\\\\/AaHPy19cZrqIj77lofFSeqx79Ql5W2eLCCwguU0dUUW\\\\\\\/Y3sIObwYVLqc1hMQs0T1uEDkG4DCUtLPMvN6h3TjSaL3YjNz5\\\\\\\/v4I6apBq\\\\\\\/UBydRamiwcy1wSO2eNbdGVQZokKOpMQUXgpHsLf6DOLNwhD7Zi0lRhBxn\\\\\\\/5Ztd1GJ7d4FpUmh+YR8PY9rKjSW6mCa6G\\\\\\\/EIQ1zylYizi21PwLV\\\\\\\/pgbkGSuMCUZoHXrYUHpY2F\\\\\\\/FsGUELAHhN8QQDtQDQ56IHqNSbEcD09Na9iRdhofSyDyGGqytOMuosbFJOdibzULZuIGtV9TilRCsP34Fpr62+2hSR8mlUbhzBx+j5Ka93OH8SdeIKJPdFcs6ftiVr8GY4LtCwRMu\\\\\\\/bQUXKSV4usnw9t\\\\\\\/zvq2L9qZ34Tn8wUpjqZ+8GBSo7c4seEPkSM13kR\\\\\\\/x05YCvw5ZsmsZtJ6zRrcztu28DHhZ85dV1R9\\\\\\\/anapfdlEiUfAbO1L7VdsSjqmPUaVXWZ0R+3uzsLAoufRZ7lIHPrnv40vIVOB9Y6seFkk5wi6owUA4O32ZMjmqWCK8ySmyfgoRd6xc\\\\\\\/ARk6\\\\\\\/3t6ETjLTa83USpPvDprIBS9KXVW\\\\\\\/rCSa5Q4y1+BER2YBCrhk+uQoD69dGsfhXci37x\\\\\\\/brXNm9TFo15q2+fD9mUP8\\\\\\\/vmbLx32IvOdEcD0q9J3ueenwOvzoLav+cJM83Gqzu2EKexXTw9tat6PAvMGQ16+XcI3XUE\\\\\\\/BiUsp55\\\\\\\/+mGtKjhlf84YWD9G4fXaiS6ANxpAIMXd6xvt0TTAIuq+A1GllMc14DkSvznkHNjyCiFsDCcifBi7TzG0tpBXMz1eURo4T9JOLZSGk6fWSqmB8gWUECcDdYcOgXTVRtF+JjelhX5q90r4b2tEwnn4Qv5PbSYvZrEEhgA\\\\\\\/tseqes7jxBbY2IyYQek1L12737Aw2VwC6auAuz3omgYy11iLWugQqLhIgTZX99RnDgECuW7+TmP6xXQdT6M7gC9c7vgqDulC\\\\\\\/fpv43qMEr3p\\\\\\\/gzV6IbrzhVo0kgj89vRkPRT34Rh09ObpDWKrXNuVSxuigjNmWzFuUwx+GfBNbc7T3irf4XBqUWJId5SsiV+BKu6XLs5TckeomY5ahgskPeBJmZlAAb+wD\\\\\\\/NuCrRXADLqT12hmkg5x0CHtGAt934rr8o6N8EG0bd9GTMib7lVHIS995nNTd3ymywXGZfM93Pk3DaXCo04yI\\\\\\\/itpjCKDxWOM7cxqS5mosGxdrU6ArdjQoyPxks3oNJmz+qQ0s6W0RU5rbb0hN48YAckqE7tRHm0INJXW4ZH0DIs9VmhDTsSHS8EXpIMHMWEZw3r5cf\\\\\\\/3yb8TYSSckBjlJz+1vtsYnNETHkrVxjMMOajebsfm\\\\\\\/J8x5uWmOT55uQHoWCKgQ\\\\\\\/GexWRdBIqQUeU1e4ZKsA1uEGhaCj0y6YssRdwwy3qHOo80BpcynbLLzeKqE\\\\\\\/D9qMOonYSdohBJ\\\\\\\/hzdr8LbCE1oHle1jTN50T9xmN2JhV1EWxtLl+K\\\\\\\/gd+qHthY8HkNyauv0ntX7StDpW+hyZJjp3F4KEb1zxEjSquxh72vXNkX3k9b+4pc16kLCD4DzdCP6jspCgcJ\\\\\\\/g0IPlZ17njuQooBw+FiZc8b90WrpDUWZqFzZN32x7k4ImGcjKV4+WhM2gU+BqZ+gkqj9OI0cXQ3cdgFB7v7n7QjqwKLbeLI1WhLGG12XwVeY+gAsdJnNlFzIpeO0Eud\\\\\\\/aAjZVg\\\\\\\/bZ9fOWbYdoT5fEwL9TzxsaDBfYzfacQtcDyz9OAme5Q0yBtLePdXQMws604oj7WMrK5WDov8t9YgLY7oQvA5rGyNedral8CQFyuhtlzLN4p7HzXRZnL6sycnFP8dk6w7c3106lBxe8lb\\\\\\\/zHvU3flwhpTmPslZUWBWu\\\\\\\/9nNELyk3Ytd+kBGYdwdkt6j3dUgDMb313LK6FvX\\\\\\\/nrM9osY\\\\\\\/T+S4cyZcKeyOnmm\\\\\\\/pyQtQq2yX0ZJ0bvz9Er1Tl8662COMMXkTAgiJmagUtyfLibQtb3Re\\\\\\\/83jtzfxUfywKa9mWh73OgSWCNw8\\\\\\\/9vp3JjDTPeT0LAFwCrtfnEv7xCsJAoxUppA18NBPwO\\\\\\\/yVVs6PYuRlv3xbHKWyzNMT\\\\\\\/ZE40NmiZBzO8w5NPlz+DdD7erZZrMkdCs5+pj22mp2J43Lb+QkrxRmzTdMNDtcmq9F4+xkIhcLaCW86n9p0+obQMWg+aoLQgt1fKBmjWzwo0VQ3ySucCJ3ualjkGxJLc5XyERBJoTULWHr6N4oz8hktLi+r6AauqzkMIl62NATe0XZCMzPfKN8hTNUxcipMFuFA5O0w\\\\\\\/zAyq7cbtxwvb7OeCgCmLdJvcVHHVgdTpPQ6SJ5aPjkIqpukBZ9tvCwV\\\\\\\/fz8\\\\\\\/52YjiILoeMz\\\\\\\/crsXPZPYmiWILvJHzU5f\\\\\\\/DE7NVAsIfzWYBRqJPPZcqoaLpbMf+82NXxTTydo22PmaN9S3w7QmWO0aWMck69bDGAJ327mcspFDkKhBmz2P6BTqUaeke9P8slqJRviFLgezbwnAdhC9Y6pAmteroOPceTwyptqdutS5AfATyrmvq110E1FQ+USh62nxhpoxlXvuN7q5fdUVpSFImJWxtFT1t+8G00TUeuS7e11pv8zrqxKHsVba1RoxxH9HBtS\\\\\\\/mPNhnmkYEjg9amU938HCtnJ2r8Kp02qwB5Q6AJP1CfGK6\\\\\\\/CcR7DpMW9KDbxXZWWmANNTKBL0e31Axwp1oGpk2zjk3fmjfus47fB22L8+o3gjJMiOlnGCICuR149lxX9a0Heyz1Uy1mz8w84Cvae59DZeDIj6UlKoHXoy8DutE1BgMXL4c7LC2p80eDV9kREibJ3\\\\\\\/caBPs5EnbNfssv5iKxvuDotjCJ5Qhf8c9HIc7EtQkHjWqW5efrGihixI4ll0VOHLDQa9Xy9iVIUlXOZQ1BKzhhFmz7K9\\\\\\\/+giUM1IoUVBHw30yKTPsXmjgfy4CMFqPNfNEGl3qC+pTefU1zx1QHyr5wjKXifvdB2sOlEtDjWqSlDEzfrWaLmmJQAbtpH0iZp1f5Tnzmd45MHqVcSj7q6vgBI9nrLK9TayZCSM37oqO503q2+ib\\\\\\\/HOz6X1rvPF\\\\\\\/YCSVN0oxM55y4Ro3TRXHT\\\\\\\/NdyGJV0Jqr62luh9+DSB3fV02cOztPxQUGoy4Y5ZTWBvlDws6hjO7ylq7zOFX+0G9H+\\\\\\\/lC7UOzRBzQbqkUFJ92enLfln6fRNeizLxBKM5kjr95NRoORdOjikDG7KHK64vsaSTcpVQ9vZUUoj7rCmpSCxtJVQCqXAzZocA93UtLtONLMKweCEWC98NfumY1WCg7o9AwW4MqrD2Hn1g8aP9PzhjVAhDrhilr+ZIZiViPiCBkwEez0JU4ukqAlvSGO8TXALuMtlI5uUWSSilN1dX2bigk+XCiunCsWvxfclHG39enWE3OG7lOvQwWtWMYr9bHgZdYoWc3D3HiK5x4JRYuDC3ElxhmeTbyPHQFFc\\\\\\\/6B8pNMvHemet6sp2Qpxttz\\\\\\\/b3MxRmhWEVYivJt6A0glHRbGo7hhUdjXUd2JWasjXKkF+z1DTXFooBfWndez3JxK+SHeFSunxsSLz6RWMbP2L6gvqG7ZQN9xlmBiaySdOJLtClcpQitd9lhQT2U9eAtMP43GpDh9kExoucHPzy4RVeN8+QQDGVS96+PrCr9xhSm6E87\\\\\\\/iNfHQmxFi9hKge5ESmo1wrXzjAt66iT9s6PFIIYoUP5plXVeLQtLabcq3kPwn5drpdn+AfXSJrbPXfhcN\\\\\\\/yKqdNKi3ngO5JdYXJ6p5IX0djNg00Bd0HeP1q9WcI6q2VQJ75PG5tPnj6tA+M6YZ1dwj7NbQFBTXdZ6NOixHfCNkRCPc3lk1b3LECeN8Xn8sbARE0I+mb+ZbOC7uZo3c42GlItK2bN9jYjF6HwTAGRpEAk0T6IXp\\\\\\\/NlKbzmQaIwt8VyYoy08qr8wr8Yb3sun+G2GU\\\\\\\/NF83qbHUMcp58rwhge13\\\\\\\/b\\\\\\\/2VuwA+HiHDgiGg5fpjWSs0AcYuhD1I6IkDkPUyMY0\\\\\\\/vfhVRJinjrNRZcg4Q2d3M28OyHoW7tRzn7YJDV1teDrBFYb9SUGUXTjzNiqe0fjCGkJx9cSclGPl1gNqAzbTVRS8wnpoo45KK34MDK6EmohNNgbEAZ8rCNe41eq7nAbVQWwR6WMzbXR5f2qzA2wq883qVQf4rSMeaKCVr5IQ4qKfucsa22trEzA+TO4XSrwvUhsgBRrK3uCqwAxAG+rGjorK0lsQWBzo0elr9ne8DtB+Q6JdBQrLodUsOEdoaawjNe2H8GwLzWdb81r3W9+SmcSTDMhk3ML9v8OeTHemfp1MDBQmBILYqpizbUmgYKyreOLx1nq5karDwqKuov3YCvuuaXQWNMLDkTclPHEjAz8Lj4Vcsy0kQ5W5GxvXUpemsyJr3fCW6PtNKA5ZGVzlccjDCdXHqt7AZ5tsgZK2V5hXDLMHfH3fE6q8CTWKBnbJ4pJDUnH7+p00Yq79k8jWh2yx1MQ6iD9SUQm8Qi5J+eoFftbbbPc6+4GSAmsB6ShtPTws+5R3YXu1+erTaB72WMxgfzNBc6tjrSDsaRJYSLHpzGY7I+JcKCUdMSeRAjHvYEQXKSCzy8YrQiBzuAt4n0LUNKprMFgCBVayeVtkFiJslmx5qEChK0DwppaZXSnWf1GKWEFQa7WwsKETAgql8kJ9eGp9DIwpb8ZkbIECaGPHaRarwi+tCk1+u5D5C50mTd5mklF3PSWx+Fjxb8HSe0mUfUh5K5cgtmuGTp3weu\\\\\\\/owanUFRahAClK1Nc+QvNqkgYtgCWKenj8OdoKAGLoiDtEd0mg3zAXr4hpiBlTJNeAZYr5sY\\\\\\\/vsNBWz3ZRxibzTG69m4EahIHBHO\\\\\\\/sG1j5d6ww4UE02uxQxYGbTS0c8DenK3re4EuF+enxP8LWQ\\\\\\\/YKAlk+kfRZx9uS5hyIAlSon\\\\\\\/PNrtVhFtP60OCXbHj\\\\\\\/8lVeth+HGq1WoX1QzMFxlRzUDKAsZ7PIDxJvZkYVejq2QaHl46AGmv3Qdp2wWP2p3rHjuOfWoAHmVW\\\\\\\/QhC+9XACgn5EEI+bcHS46BhkbjX5mfWUhcqVqgrUiBrmPTC8IOgDmR0smGrddhDDi+3kSnVd0+ktdJYdn5E2+5VqjncXADMoCljxrgFBsT\\\\\\\/eAGpMqriq7coEHZ7rWkOM2JNUb+Ux7\\\\\\\/RbZzKwI97FJbmvNQM299e\\\\\\\/ob1XBDHwc6qJUMH+SHo0UzaoahtQJprgqzXU56syQayhCtBsyM4v9KFf0AZO3AuLgc0GAgUgPofWspYQ6AAc8LfgZTXzzqmFQ89YFWYSGV2uuHq5iPt\\\\\\\/esIc9Vy2mlJ94ostdL8LmW1NuphIKA8re5KBPbGfMgxA7NdKZWoS0nOzkRjOZ4hLpjMEVr2\\\\\\\/4KgNVR0cvoR5ntnvvDZ8bSHz9ojltu1x0xMrpbf1WutpRUfLFYa+gZecEE2Z8mPYXmcLQiFL6ptUN6ikNb64MLCpOduR+cch6luoC1YEQxiGt5j8TDEwX2r4Hv6QQnJiKSWQiASxVaMvIJwgoH8736IRNBkZYU9qx7lIqotTm47JZk9npolgi77wWNWT6k33AOjkJs8PThAvCRj2Lj0JclVWE1ixwryA0WCnepGmAs32UP4PUfv+aLsgDZNudfvFQd2mDhFc36qb16B2DXW598By6+V47nmK2CC9CDoOC0BG8CE95ZDowQH1NEspoYjbT\\\\\\\/E\\\\\\\/ZBFB9Cs7W00Pf24XemhoO4Ktcs52BHEbYI\\\\\\\/mPkIYszVE8zIWTU2J8HH9qx9hFBRwU1CX4LWraaLRtHN0DdsXe5wOHXgmuQqfhdVHauCYI\\\\\\\/cOgjT4izEPxIGUUdGzmUPl66VG9t+pxC2s\\\\\\\/dV+9tCn559RM7Vygqb5R0ughebV0UOZ0WnWXXIZlG2n6wtN+K4P1OKmvwl2mGpW8VwuU6Qud6vTlVMVutVkDywUPe+zHrkzATyYTh0l894eAepyi7KEJt5oaR+e4wW2mSgl8JJQQ\\\\\\\/mPmY++\\\\\\\/qh\\\\\\\/ys4crdsZ3HhUCVmwKGc6OVO3MOyJUe+PLq1sFQbTgypElVdEn8mPrhwFSJgxAdcgdWJrurlERamtzhmCG79Wd5bgNPSOcjKgMBqm4YC7L2wHVvmeVojB32wM\\\\\\\/SVnzSjE049oaQ23xMilJzu7xdGBlzd+FJGWQwYThpZozou\\\\\\\/9pyL0DmxvGDtc+51BJKsirwXGuoGXsGeM\\\\\\\/RAvyQCaoThHcEBlQcZyaYtzuiDhQI+oKfUXBsUJit4KvTouAKInrbdie4DnrKR10ZrDUfV5RC\\\\\\\/zwkxROh04PmuBxy3prQbJ2o5g+lELpJBDyNvAUXmhmNGQatu3hYh6CoQ5ZUiatDq05gu8qvx1gCmd3owlHqTmUZixmV+5Aq7E\\\\\\\/WY7EFDTqFuZfDCgWrIZ2vkF\\\\\\\/wi58UPxCPS5ZyQ9GEPp2gIvOnVSnFlHhdX9RN6Wk\\\\\\\/5VFnp42ipm4iHtZZNEAOiNv9By+raarcQL8neXhphB1cuDivFO6PQjm+3e5Dt6OooMHaSpIBcLPOhfddnS6SPzqPFYdVAmcxrd4meouktSJ\\\\\\\/D7SHZNNY1mQ38NXn4MQ9MIslch\\\\\\\/rLXj2xtCIFVLN6eGLth7xrbzfvcxPOtiTlMCSq2pL5jzZL25ZMRIrX\\\\\\\/yDe9Oeh7nYwPiwGrWHmPfMq7mzcNrWFMZnc6zxEDVhvS4xdVYigL22nsmOnrXQQEmEICJbES4XENJluoUblxDgQMyxDbccN8RhSuT5gPYSK+hJ1aIFuqKygmeN4nDc4WjICJLx7QtS7YbN+B4Gl8Trmue9GGWoD2ikGpWqKmCAlY0f9byFkvjlqmtG1rD5L0uoXDhmw7OyIWktLwpcvryw4TbJwJFV1XIp8XyXr6hZqpVIYy\\\\\\\/8inP2lNBhkW+xANB81MsurI0IeQhyvEZchYEYvovJp28R\\\\\\\/pvuYY+EblPPDzJLr1FRO3JjK0+9dzUaYeLHXPYQPayZyCIMjUHhxoqx0KWMFcd6hbYSZ4wEoQ1qVYAMMae\\\\\\\/s7zC4qTp0qVgWGgdoRSRe3cdyCkHDPG5\\\\\\\/lmnelPQrGHoeUKQKCM0\\\\\\\/NdCI1cUOiT6aLoKi\\\\\\\/Z+t7B687bx1LN2W2D9t6gWcfZhL0KPnkduJZev4st\\\\\\\/KpN86nRMmPGNT+ZmEiNXKz7W20GdIWqu9eXDXvpXcA7yO46Pgo1p\\\\\\\/J3kMd\\\\\\\/jjJ57XTibHDZa2Zu1+SyZonD3GPNmSGSVrxZEWHlomkk+jiCbYrFuLEixUgCg7oIgb7yAFwpE68IbVeuGLjbPJZlkPJlJs3Ckkt94kcaZPXjP63Sp8oLin4HTunrFfzUb6HYgnaxRmIU3qAejNNR2XCeL1Cnp6LtFdtinlGtZWE+WCvRTrPEaPQ6z121o+wx5paIdVmFgNPkhHEbJFu11AUO09ZfROZ1JfexHCSdWkyZj75GPn8BxteBudJDbEx4UCzVWnCsXjPIGUi25Z9tWujCh2XfFsqeDUEu7H1y19oRU2NV6oDJTsPi4Xa3LYOdjiBWHsc+Frvv0Q7Dmwyw4Fg7NCREt9XLlhR2qRU4QEJC9p8UhpgU4Z401Pvjz7xVEVIaVk\\\\\\\/aPHcYASKhUtgG366Sl74lyTt5AXqNy7a0iOYk7zMR2WTPZO3urSOb6\\\\\\\/A\\\\\\\/LLTxOwESyMjgcUQo74cUuCX5J9M\\\\\\\/OwAzdCmhDGllR6AfNUPgyReiw6HoRJo+hs4J7QcRGm6jJx6m08G2YA\\\\\\\/ygPCrSu81vA39JCPqCwtfgKjHyZXY5N02eLliSfmAJ4TbzWmP23ymg744au\\\\\\\/aVg2e2CWmSANZUnZXbUj4nA0GbPoDjRLJKb5dFys7vi42HKTyS7ry92f8tG8QTq0EWARXcKFyYd3ZV+IcbIUBYFXaLx4mDIZ3sbcJHvmCS\\\\\\\/ltrVZ3K4brPvYtx+wS7Eyqg6R47r5Fz\\\\\\\/5Nx0ilTWSUUWU\\\\\\\/t9y+dNsPeWet0Apdvh9X\\\\\\\/jaqYch6D8q9j0QRtQ3vqbGGDo6OxxU0t4fCNXVFkLSkc1GK4z2M6zjruVmAP7Ay4o9xTCsvaEsf3YGfFKsfF9IT0GoESt245z3H2KmRh6PKjgx7rLsFvCB+lZm\\\\\\\/rW3ojfU1mVWrCHTctQR9cfgXUtBDs+0bQUiq2VdbXs\\\\\\\/C98568kwkRhPVizZjg4fpGvxjujEKpnAmGMLqvVjF9U79g+6i6\\\\\\\/Cgn\\\\\\\/FNTejNLfg5nm6aMjzfLNzKxLwI4SE0ZHEfUXcr5WHY79m6TYx6Us+1jQa0wYTaxvix1Dx3Rpzov\\\\\\\/6ppMqxh1I0pLAHVSqx9MuouHmYeev2eedDSYRmoOzzc7oHrjKNbIuoh7uHb9F2Tm4U\\\\\\\/mO8kJ2+tnGAoy9CKBaAIwnjmjhfS22T9rL8T4RJzca4KNXZaHJywKrA5wGIPDIaLpxmJQEx53jJP8aH5aWIU5pXclNSRXZibPXW\\\\\\\/Wg\\\\\\\/SyE7E+1FFlYg2pdt56c7CgtpUc0R4Y62b5tUnn\\\\\\\/KqoHQOqLX5bNxYRJC8TQvqrm8R6nlFntdUqCFLyitot+8Z7Fi\\\\\\\/MLN7rllFu4ovBG+yEbCIYrgXdavdQsz9+EtJu7rFj2RxrSwGz4+gYEEcxHyEvp\\\\\\\/CfjeY8+7+qGMv6EGT6X3PZZdbj2J9ANLo5dVlORCYrMt014heZRZv1+nHcxsRC5z\\\\\\\/AV9XOyh3DzGSwphXYj0FuuiopplXE9daR9Yz7SB0X3U5MGhXCRYcBPkZaijwIsMLuRHjBvWXjsSjWXW81h8eH97KhGYIUg3+A04N6anPt8GSrMR1dOOCoKuM\\\\\\\/2XpkUq4FHxFUhfu7p7Iq27wwVr7dr\\\\\\\/UQWblEkcl1xMVlSF0Ks+XGoJ4508nJ9Jt4KwmfzGQxwWVJ98PhjY8giuKOZKEI2h0W5WaxgpnF+6KG4Bv1lztb3wa+JnVQ0u02MMZGykaUvP\\\\\\\/TrH52DcG82rIKc9+benmUbbo\\\\\\\/dY67SqI3ZanwcpqjwuFn17v297ypkawj7tsN7vvOVbqhNjxrRIzcRbzhX0uckmmKmOS06FRHN0gko+IS8V23y+ZZluPrQasa6EwgxUdoLHimI\\\\\\\/8YfBDhRSDRFm3b9IbzPgwHDuz8sB8jkfHWva3v6avduOzddyjmoIEs40Ak3xHiG1FUY7+Hh8cdDP+lGUyZJzQHQ9zK0bPyy0nb+21GGfWoxIN5mKLDd2XX3TJCP190yXsovEWST4cDp3Zlg3zi2YzP9exzlGGB5EOXE3cH2dkPrlNZPaJkDu7or\\\\\\\/d0qAPzXaCEICWIWd94V+OSPrNakXwLkutqcGnNCYdtN81dwz6xBm2ipixJD4q9O3NbA0eut1XwppXZGhK6Avkp4x0tvf\\\\\\\/fKBVpuAwf1e6egV0qW0MpgZC6aZ+Xrb+AvnD9m44TRrV5STUxjhyukFkIZsbjdkYQVFzVSBibeuYkPeMmnP4Jv8IlgU8wxXIV1k0YzqR0D7SeM9Qyu4FBype3HxrJZWbX\\\\\\\/rLMjsEimzw4oMx\\\\\\\/fZfHuSkFN\\\\\\\/AVuEy1DUpkfQxizWQ0Opklz5KbPZR2qT7kL0FatZqki9r+TKeL\\\\\\\/wFoPheIjtlixtJfsP\\\\\\\/KBkCbQUGaw5Nl0pfOWiBf5lE43wDhcwYPjBeB7qV3hGRVS2+Rw7ESnt1JzCeAv45jSNHV\\\\\\\/nJSbly5LnS5JgGTEI+SUg9tm40Zl3NlmbP\\\\\\\/TC9\\\\\\\/7dfDU9mHu7PXm8EnniWYg+SIakJ5D+jnF6FDJr6yO+ohQmfj4Ijm+B23KQVchZT\\\\\\\/K5IRhDskc5Rdm0MNTfgMUm6xjhYasapms483BVzWCPehguq253S0Ll2NRpR0ZxXD6vG1P6dWa1CdvdEvq8iV3Ky96PC93rlnz9rDa2\\\\\\\/rSTDZj8L215\\\\\\\/gqfp0oJdTr9ri\\\\\\\/hTYGlFWIU2+fqBIPvu\\\\\\\/ot7SwsCDwholiY3SOapc8h1fCBBeFHPZiD72iPQBtbPb65\\\\\\\/QxBk0N\\\\\\\/dXmXPTvTcMK2hS2VW3k65KQRajMkPqbwkBPyYk+qydFB2s5yUNDCJLahTbo5H9MtZZ4l9bPSFN1kqz0dHOeheo2mjqcufLtb16KXm3XRnwXkmMaYxevRogRcKCXBBX67dQA9CnXYse9DAGf7qIdPM0cUUzkduyQUcdbi+hMxQwIxKr++oeW8mX1qYuxLgnKN\\\\\\\/piVBpsVmki66avAmYTphMDnoQFPWCppUNKG\\\\\\\/nwmPyjx4kjZfVf\\\\\\\/YP6Jo\\\\\\\/3Lcx0XiEf0GleX9kFYwLg7h\\\\\\\/rA2O\\\\\\\/PIZBZgA4OU1VabsoGk1Kc2M9VOcX0xR8u4PMgoXbMLS6+5Tbl2aaYXkzBTzFKiVaW5hfbKQHbqI6t98U+Md27rpemw4\\\\\\\/l7U2c3N+sgLdMRE0asciiEd18wPrrAIWTDVms7DWzWV198thU45hwM4t3senYSjXK\\\\\\\/SkO30j5\\\\\\\/f3iELo+zoEvmK1wp+v91yUkD43F4stzJIjkSZOgIiczNFt+hcCZGNesjuayPrz1azlJQoF8NSp9RP4Prtkwv84tZUjdjHIjvkSt78we+3mnG\\\\\\\/bwimmOKaeED02n3TxAfxb5HDyUaUpRF1Nu6kNqGrUkVms89tvBWn5jwKDQEWbxq5enGXxk7yumePUDOSXxA1T0KH+h3PFc\\\\\\\/IMqAVz+ZKNkPyWZOgAZ71Kt7PY3lXbPBwRONVTknj1J3LLec9gY1jVenJa+kiJ36QYZNQIH62479+Bdw60SvYRFmTX1yMtSbJ3JBUKQq0t20sVLZLJPUHkKUouMiXs618CEGvG7pLwitM3Qc2dKm4sXzyygMyPYxyTCUIXnZNzka\\\\\\\/ZSer0s0lbzSX2W3f4dbaax7FT4gcVuhyVEwdDYtOjQdpXsUgRU5S\\\\\\\/xZ0nk9jqCVLQf9eFtfaXmfRtzaKzj7I5hsbgp4Dk4tjUcuEaBcV3R6L+n7rF\\\\\\\/y6E9ThDngk4DXKey6IT6zlViNND\\\\\\\/LhlgOYx9S\\\\\\\/zpp0pqNbNEqMhFdLpZY0hp506O9DFV64nkf3VcGCfX9fUSCCcHQfotXiDg1SkcueTVF6Vufig+Hqqj4M\\\\\\\/BEUSo7iaESaPxFA2IYs0HWhixh20avxxyqz1KVOs58IA2jkR5CuTJ8+P2hangtD8xeqsq6dXtaz1\\\\\\\/0ZIVMknpTXIQsjRsq0gvqr+b1xk6DPh+j3ry+GORWJs\\\\\\\/+MvCyg186GraiFo09XU357Ok7TZxjlw1Wfang5VBNPi66ZqDKD\\\\\\\/K1gmppVuCTqzPx3SqJqAd3MogRCt0yiOhAza6Rs5S5DZqM3LaT4\\\\\\\/Mmr0npq9GtMsX9vKaldSN9DS3pb2kN7SlNArH8X8eoBy4nGSJ9adLpPrAwXR+PwcN3KMLh5cW8u3DxkjA5kxcxZSQIf\\\\\\\/1XAT1aIQakW5zB+ofCJWhi3m2uKBZJSTjQdi47bQ27KIdEOe+UCX9PHf1cQbRbP\\\\\\\/z4tgzFwPCUPHAZ3ke24idrFpWhOUL1C\\\\\\\/1FYU7ZqkA9aDicAp6DtlEk6aYGIbADfpuPNduXO6VvIDt1t\\\\\\\/1SnEGHe7k7f0dWtTQaha\\\\\\\/028arVyBaYrwT939ApksUK9pA8Am1h+thF6VAu\\\\\\\/Rs5G9ywz93PMx3AnueZMvuDCzGtkcydscUrHMHueGNcxnzMESCTmWRHYkb1ZByIs4QQzMhCZpDOd9FLALtweEnN5m\\\\\\\/WrvLIxIbudnzTDQO3vIIM9cdDAcape\\\\\\\/BoY\\\\\\\/RbH6LTpxap9TOtRKNJ8shjuvRDtFUIWz\\\\\\\/XSaJYkkqejdaimPhPvLQplEqQhpEBsPFoZOorRlWUwc61tletBqhDWG6Yt68gnWrTwon4sWqoN4ZU5HtYa16+I34vYoeXzWPEdFcWUSbFH+ehD+5axO8bdLBQr+TzFjlrtB66odVQqoDGKwDW57lM79Gbw5dq5AOb4K7cHtmGK1Y8Y8dkNqiYdwcMout2hFyOEjPQ3aQ51LFLSb+9TyI+WMFaigZLxa\\\\\\\/gQJfLOb5qu\\\\\\\/G9oQFTT1PlBpVoUMBByj3A8ZdNmFDsV8DLW5SAUsC+ScXJxCnbdEjU4YXoj2UKPwfJ698R+WrvtXC+7zSj8xr2OdesQkAQ\\\\\\\/te\\\\\\\/mlrP6riaKoc6\\\\\\\/fzrYO5juE3wg\\\\\\\/GpjeHooOcd7AN6hBc1fBphWjpMgao8Boab7OooQPW3AlwZ7Yd\\\\\\\/BBwbG+WSM30TKC7VCuZkuhMbkLtPEn2Pp26HVAaIgQNXq35KrA+vtIQZ4rqYr1rc7Wn0rf\\\\\\\/MlueD2wuPl370NEelJZeye1LX+qvsnE0FIv3LWGL45vr8xRQ89Iegv95IGJ7AwsygmgcMw2KySq3qQ6tDFSo9PZGmhq645OxFIP0zAmajTdn182jdK6pOCUQU8SW2U\\\\\\\/DPzymVf1ClEh6DiMNw2nnE3Qo2B9rZqbTvAGRKZamkz+zUMapEQ95fd2\\\\\\\/nTLPONsOofe92u+c7wCiLeSIT\\\\\\\/RZW9WKgjst364xQlIo4AUD6K6XzpBDJxST+hyHEyY9YsUmE9NJbusSCcbcKoGlirq29cg8lIiGyAYOd92EkYKa\\\\\\\/rtiAomxL0lioJxQYjzsVe7I\\\\\\\/DxBzTiXIkZQgZZ+Ft2cK9+DUCvPb6HQja\\\\\\\/pWthQogitSXRrSnqB8yYq5klJC\\\\\\\/sD7AHZO+u\\\\\\\/ucumCKOOWgn1YZMhsAbxYICaJHzDqa9rrWoOjEZv1zhfLUX0zPNWk4RdWDw6oLCI7WNN+f36pVLMhe\\\\\\\/Gz2qD8SzT4PLCmwBxPhwYgWYAH46is4300QUP0AnFb0nGS8nymquMAwMvEhHpTdniWzuh63wlcM7y0CrhdZrfY0LjtGBQuq45bjws3SFya47j0wiLVAgREIEl2KWHbQs9aeNCzOA1lZduUUFCHuxzyXaD28XHw9EAzMDTVUATMVd4nYRY3wIxr+80ujquI1c269vQh3dykf+Pzfmr8Y3lx5msZAnga54n2UPkr\\\\\\\/I1E0hRPhcr4+9uFIZ57g7NPuDTihHL+F9pF+f11pg56vrgDVf1MSiSmnvbtryH\\\\\\\/yLt3JUPvFmQY2bxjxrEVgNB5KeauKhRznhjtDW6gFDpiPyOuIKag9tOQ01VQElnjx+Wl8vXeccr0WaIC0hY+0hZBxMXYfj7YNCbj\\\\\\\/dM1yRA87DmqDSu6IVDnPjFWMkPkCEBmVzvDNc0q5OSbhAIByqPwDevR7HnO6HWrLi20pg2CQiWT\\\\\\\/qUenC1Ig5qFD3M4jLD6z7FRC5b420wDMRxIupjI0ePjdw9pTKrCUD9gXrvMDpxY73s2edoRxJr91BUA7PNThkBUEljIi2smYdepMmL7AjuPgruLPWLD0lNVh6vf1MesKXhX9ZgZCP+\\\\\\\/LiDCfRsO3+MAK\\\\\\\/WQqWzWMP+3jAdYsQfJjJA2yi\\\\\\\/1Yqmn5NcAtz6bCrFmY2XM0ONvsqEX+3LOPnZT7MParO8mYPB5nEhONxODqX2yVJjzAI39g1FGO4d4IIFWg4jfIJTdGLh4+0jOQk57DBYo3lK4+1H9Q7HkiVXuWrcDfNRmffIDaYONmY\\\\\\\/LIiDdgyDNShZ12fEREjzSVC+hqOZMjB9tSFKaNH1wkY12Lrt6jtB6jsu33f3cbBfqdUGqrkW9KAH\\\\\\\/NxO3Q7rAMmfSeY4QIqiTINsfn0FMinz+pJRp3YQCsRD2p4afC1tsdomFGmFHGxeRaIxt7ZnJl2d\\\\\\\/VkJNkkQKw5wcZQM4qgQBU3RURDAjcYdq4nztAk21x0bpVE0OgpQXbTBD3cAgkO20PSilQjyIP1clqyOqyKp8rUoGhqMqpnbFnX\\\\\\\/F2fkib3FPOOAXjFGMF\\\\\\\/ebLRarpTVJBgFLfPbJqXUeUahvqvieDwHj6ivF09YvJLyAdANcrpcdUBwEbUwO1Zp0xS0p3DANYeQyKJaYCaHRANOMJo2zjT1RJOR\\\\\\\/DvPF28G2xn\\\\\\\/VJ61t13VEBCvqF0VLNBNMWsKH0WU94ZNeU6sqq\\\\\\\/+xmnbkTfqRWw4bC\\\\\\\/4V8CyaNVIfgUR7ILM6+JDyQi7tCg81DhrYhNHfmKp\\\\\\\/LsYEvQAIXTpHRDWQqDgUBzdTQW2zbyGPAqiD1dMGPmFp1W\\\\\\\/gDCVojWY156MXT1KB2Ja4gOPpjWR8gQ1W+BKPnb+967Q39Q9n4Iz2+UYWsu59arfy7xU9TEZz46WYElnDlJrcuOfVAp1suuC2ZY8MH1PlcDaV8c1BWvJJbhqEWat6UaJrT6YjIQS7e0aO7aeUjn7CfbDnjE7Vxktq56+qFAWIq14R3ikOXMWEAd3\\\\\\\/CgmCT8QlUIWMlxauBOthFNNoe1RXyvFQTjPDI3uJyI+Hhpqm\\\\\\\/mBrho8Ubt0nWbktq5kk5tRNDCMjbs6EquGckoAJFdjJ5qfq+IflvjXiAh0jRLuVGFALSJHrFe\\\\\\\/Ph6P9wzNt4e35pPeKcbE5m4PBfOt2L1YIuUXQb\\\\\\\/nHr1mSNsEVVWQ8lVbIZZbBzB93IkFPKcmI73pmzRM1W75hCMhbr\\\\\\\/Epm7kBxOYiQq5XW6qHIB+ijz+ePr6e8gVajl2JldwmC3ysWaY\\\\\\\/VxF5XFgAMJhyDxRdzYFza9kLrkyXRV8A+sYKEyPvQCD\\\\\\\/aBKzu2LIZPWvRpZbpvaLoR8zAJ7nuHci\\\\\\\/CWknYYYM1Dxp62XEXMqCqQT9DzLvQPdmhgbRqPFTSAP9l7oZfNkyEKjd6SA8XVuEArTrDwi33UUBjwnuFy05UeJsJIywqXAxKPO9AJNQYjj1gctnQUbS+5dHm7eByrBOYbxtvHtmcelBbzdw\\\\\\\/O9hfNxyeqBq2SKUs0hW\\\\\\\/WRn+A23tgT5\\\\\\\/88mM1TbDUTqyH6ksq57mfqjCtbinAghk2nrEq1GStq1pXbtbxV0SoR6QSMLcUJ5BK+22XJA49Mjo0YjTuxQLV9tf6SFFvf40m0rxWTkDdYH8bE0sx26A473L+FGBzLgVDsSnYE1QQ5JQRTKP29AJjjYYEov2XWYNPUnmkVXxuw67Z6wi38bNuab8kJ90KkJs9KwEjaSqFBscZn\\\\\\\/4wt1v91lx5KnU2hjIylgLueBXQm+9SBYtXAK1OMPcD0KXGSkHvDtyAknsRj+ru3RwJmfLnKhHcf3cYaGZduHRqZwlFxbDisKsdITvdhR9OLWMEoHaSpGKLXOesNzmBEbue+HKVV5lrHe4I9GHPpKsp85PNEP19+vJDNw8NtmP55aPTtCa93KjrT8vWGvZcaxrsBOu8b5M5Qt0HObgR7\\\\\\\/28YMx9gFuNhcY5wguJGn0xJeQ8TnthzgfqBJfvJq3nOfoSFXGOWzcY2hXEZbboOBpcAIlqVccJDyoCc1+9T1Siheh5Khe4JUBqTnO8QhthZeJ8N6uOyYjZSvNWDhEr6INZQxbbYV9qeId02EKbpDVlZSLX29gQW4+DvZaWymZHJcekQDwvJhpBZG9GNqA+twBKfEDkbky6nYUWxv1p+9KkN6uCwuOVmYXlym4ZUnxS\\\\\\\/Te6Mz2T2q60G8Qlem3ZqQf8hdCeDsEHzU9h4acuAcQkkYvgjR03QNc0hyWjrfAQz\\\\\\\/1Uu+Nl4twd+uYgEO1VDw+BckSDaWzL55PUOODooQrSV0UmooQqbgrRShH1Wet5TRxl\\\\\\\/GZr4wa1Fl76k0E0bIwN3U6afv3XW3wfA3u3wp2hWy72p6pRnBbYfrzhLZfJnQd1K3Per\\\\\\\/2Vzs6UNsfrdAYrbjRCd\\\\\\\/11uomFgvyENOJd1IYSwnqrE9WHKp10x+TZnZgvVGRiDgsFy+xhVXu4+AI1tJk4uzbVE36pSuOgQkbaON36F6dMvoG6ylBSJ\\\\\\\/hIpiIxXVB36ONKi8Jc56r5PerTYJTSjQXqhf+4paQYYxB2LTUmYMqJjIAVHCIL6gOYmKKbN2dIHsRvn6SHnUzRyG+qRxXZxl7Gacms8PYSrdEquZYhT5X3gVS8iy4pGdeq581\\\\\\\/dHhKh22kK9H2yyvj8\\\\\\\/fKmEMx7qM8O9i+b6ZTBy2rj4SkOeNhBjJe3tyajPY7InMeqJBUZoQCS1zLcM1JJap9v4hseOpS8ZXaHHW33a0PKiSU2DbVPbRW9OdD\\\\\\\/U91oBkeIqs4OS7nNAMfkKVbvZAXFS5RziPMUcy8VihcNAqqw3Q4THUAry6v3aqcB1r\\\\\\\/KwlzT0KixKCObt9no815je2g7RHrtSxqVJQSoSP9Gi+Wf2bN10SWPOPEJrRuYl6lJ08cu0AjR\\\\\\\/zAfoaRxWxyC35Xh2fucnE5pWF2EEyPgMCmwMU1u9a+2DWujRnsV70mZ2J41j9JLQM9UFtJA9tFDPq8nH912AY\\\\\\\/3DHZL3TK9WW1uwHnvYr7At18Igqh\\\\\\\/EAbyXrPqbhHqUwIKZTRdCDDbwQZQ2615RUZwwV3pFUPfcvfUCgfv9epwHH4kOka2LUUh1kpLxUnVQjKNbTYjHSNKCrRg+LBkWSIkNjiICU2JvGIYZjVz71uiMdhPtb1EXrDn+0u9uwFl4oUQL4kg66RJ38qnHVXDIw4vzEO00JQbgWoCFg+q+dnQjylJiwA9uCBwrrHgfb7KATQKT3D2651RXg\\\\\\\/iidaF8D9FSh1JL8LLJtspDOodyEIIWV77dMr38jLBgA79SK6fRZvHVgADgBQlADUzmtD801hggME1uw0iJGRFRp44lnI5n7iok1gJolfRRptnTm336oUzfDXoH9I2Xa8Tg3UOW\\\\\\\/G6Yqt8mnb4hwgsLYmv0HKXDWe748\\\\\\\/crZXE5hhgUpUOfEMyYVbMg21zzU+ZjjfMwLSg9zcjgdoJ7NcVGi\\\\\\\/\\\\\\\/J6OLVQ2HfUs3QVVwKCkjw0976VjpzK\\\\\\\/AnwYO12BLx86uHBBoFNajA9DYqStV5DsI4qQvk\\\\\\\/I+XI27EMDkfbxJGrqGApRcbUCZa\\\\\\\/2wyPRcUf6uoRrlddvNBItYr\\\\\\\/DaKKWHUgurQntBvKfGQT4L7ZsSOLbDX6mIJMjgzyuam2NCzEwjVQI6UuWq1jGRVeVZyKnwUUBhy5bkEqVGVLdb9AuZVbVC7iDeUcpGTrHpe8Hv\\\\\\\/4fGXPTe3306meemsvQbn2vhnkDtjZJJ6ibMzuWqj3FH5L3Zdy3mwKNXQtn\\\\\\\/0wLULtH7mf3vUlDHYW1doW8pfA7PAF\\\\\\\/iheyCCh3ubbqkZkMiLLub\\\\\\\/erlWZORvheJlraWhQMqYoGX6\\\\\\\/5XPaucHJKX28aQZn3d2CQsnx5ZUCfIUyEkykRcjjjlFQEYobumMW8clzwbcQDjGATDJl0Hoj4rvryKZ\\\\\\\/equOBsGUnGx43T1RYVYzZx99vz4Git0xF1UCMeYbFEcA9v4+HYRdMBGQpzmsS2mxps8mak5Tl3aEax9jurRQwnzFPvXR79akAoNf1Dlykmo2rqCoa2d\\\\\\\/EMoanAAu6MjdWMpMI0nNV02gSRzhUCTjEhs+oFhLiIgJnliG4Ni2laR0wl9FXQmQvdaxO63WiAtyl6Gw+Ue0mhLBA5L41rIem97vDEEJ6fHlySXc1m6Ob7lKyIiwOJrk9Rpvh8h70Z4mYKFvztiPc8ES7J2CseJJAM7dQCdk\\\\\\\/zevTd2+HIY91i7KpDIfqmoZB7oClXKIXvn0zdSeb71KvgRO6S3ZP4HJx+phrDm+KTgmQJw1q+D2J5nQlG6u2BRRKsIPFd4\\\\\\\/6Q1C+ca+z+Qe7tIUuHCm095ajnTUn+D7ZhmBG0RVWOYXGC+4zL34ksctusLNjiMlPAoZaLUR\\\\\\\/Rp9XAmcHy4INFo841l440rLlm6vLXoEG0rUi9oRa\\\\\\\/vAOq6Yw9BNVL2xSvx3h+qvB7DNE7b5guCin4CbRDmm7stCmqEDqsR+Q3lwl+qbo6KV6VS8AgsQJ5eHg0o1L54GNiXfWXAmDnM5v0Q2UxVMfvxYgeoT6MQhn4W+BaQctfKzQK3ibZYmu5X1kOva4sxlLrLiOYLoHJCiisZ\\\\\\\/e1xj5UP1E42rAJCzg0OsJfSHjOsgACVhWHZAmtKgD5nPjXXwwvTcXMUzEHBtXewilJ0hMcBHM6VR18am7K4svwT8H91PMhbvart35mDNyv3qQ3TLLhd1muz0Rhwu1W8Qf2NpyEHeDEOrawKq0Ro59Qf\\\\\\\/CeGzzMOelEYtLLjPwaQr9IaEf0wZrmatWbs2LYNtuW8qx4ANMTYmCXcFgYst1eyz8xk4NdQrXXa+mK\\\\\\\/D+x8wdG6jKznyxY28Ovh8++Pg25T9LZiDX1N00nDgqgRmmuJYfw7tWjBR3g9OxjPHF9pWhgvLTfe0ELE82JnzTJzMmc1cBZsMHtGu1VGEQOO3MXvYVnhVV50okL+kYRTJO5L5aZM0u\\\\\\\/SBb1m1BCnpE06boKXKBQskykVpQdRq+MY22HhCrFVmQQiouqhMGRf7xYxtHL00\\\\\\\/JuYuvHfYoiIN5Qs1xhy0zGBGLgfXG3f6OD8eCGCC6z3M7jddHLQSCtZXIPsH5xPk\\\\\\\/bUL6d6Jz4ZA+tKbav9yhU7waGH83NLF7P8qFoXQEnE6HrUucJhc8iIzCII5Mk4hGp2WSNdVnMZJ\\\\\\\/F55xjjbaCbFyiTyRW1xPxTwRdymPGQnuV8uySacincqvfax0xm9LJ4YkZfuaRR8z41z8162eSbVgwLw+UAmjonUoLClGW8O8H3ngCZulb6uZ5hLWnFuL+99D0mu+jkOisYMB5wufBlHMIHWnehlWYGCIEkEp5dtK2vBypQsGN479mBMy+sUrguyqWMNxd03D331hai7\\\\\\\/0hzz9j1CTEvEw60SMBo+R0Uxx9x3P5xbewPlmnfbljxMVVQ6ptB8QrWaE\\\\\\\/fAqkcdL6kYXYsMOmC6Dp9rI9W6coqktaN3Z7htyD\\\\\\\/qkWqC5pVwj0t4veKqcDcyQRr+JxXOXJEfNTY8jYTEYIUjRKq8lSCFTNrb2ugsMfZvMiNpCN8e2\\\\\\\/Z2DOidAuSRWJhFTpa9MYzTQmAoog8QOh71ta9YlAXuVR7YI+3CeLEeZ\\\\\\\/2O48mWLVtNSflZh+LF7M5IoFbYTMQj+nvUFtFEZf9nFO\\\\\\\/NpvBq9dE9NaXtonoLModizYq1DBI+nxHimJDLNbdDR7lS1kmwDTly7+n3R+IAQ5CFcRNiTJ8dPPxKcNcYBqXA9HE2Z3RENIv7ERm9cpVqAWW8adOzqdO2ExAZeKFDy9jwZmI38NyWFuBweL\\\\\\\/EVq5Z3aZOpoK81\\\\\\\/lhzOWLq9W0NF2vV6ghRkkEWtx8aqFVgcAYE5ICqiEWqXnnpJVQtRuTPHZ5ffnLRU9Jw5oWss\\\\\\\/iE8KQMFTzpn9JfJAGS1Ezc+PW3VxEEV0kMtJ8w4jRxWdXjZTktsMW+BBsKgmDI550C2pRQ1KCblOma2iHCfgCJ1l6hi7AUIUhEb4waQmTt9RoXxgKDoMoZgngciRsl9iS9fi0Q9q6wv2Gu3FcuVeex0yjE4+BQjaVPK4Vic5eKEKr\\\\\\\/R4B17PqZz8LAefJbAUsNzDqNSy2el15iaY6bAXzMc6K7DPkexjlSBWkyXOCYcVDmv\\\\\\\/Gsc7SjhtF3Tn3qr5p7ZvtwpK81CxST6n97RcWtXbMAiYMMqJJsgF0hCCwgSUmwukGPjfTyEq4nyc49sHEiYehPWNOdhiGVbZ86BrQjUnXWnIXAvtxrgbyWq3LxGMqkchWCSx+fWUNHGKpT\\\\\\\/debObExNlk7rjS8vO\\\\\\\/jRu9hYsDhclHnb8eGtzB5nNUagA0bzErWEBpPbibMrhuS38wJTDwdLj4g2D19obLOlrxf7jmFMzQ58yp1Ou0gjWOEQEC3Mj5rmbAKVnhmOd0Gubb4\\\\\\\/0\\\\\\\/YSofTtmj76Ikm+9MiShrUk\\\\\\\/ekiSv2W3ScThaLb6lT9H6ntZIWWrgKoJVa0yhebmEkzLD9ILUPQW0RSEyvNoSph3OsQ8y3cwZje1elA1PeN13luwXmaHc5J1O\\\\\\\/PS4S5temHveCHvGsh+Kau2t5xeHGDY7r+QwosswO6dhBbrK7pQAbuaPcQP\\\\\\\/ff\\\\\\\/TndKnpz+yNJJPx28+FfHOt86vugoPHOTRT\\\\\\\/qC5HPSL3Iox4OUXw1EWvpzw6clgNwqw518fYsGohuPWbpcARxzy0cQgYv6R8zBk+3wawXiOfpn+Ds1lTiFUCcGE9Xits7WL0n3erHARdcQidIwzr4hZwRCmteTy8d2XJI3ins9AN4Lu6oJJuf88qAocUWQj41kBhnOGkzuVhR68Hek4TlezKh4jSFLduNRLgV+2hd7wbZJciYofJt7Of0uYO0o68Hpu5Z8n6O69Rz9gA8G1mXU4tJVQQjyv2dV8NCTstgO3xPtsdHJBTymYUUbQ37LwBi86Fm6l8A75sVXT2L1HkEvE2Hu6N1Ane8gAoe7TW0KpIoP0\\\\\\\/n2fJX7EJOlzZaPm2jllbSEaKN\\\\\\\/JVklj6Vouxn4vQbvprWbC4mn6Z53DbOogFTpm0rzAy3TwCSdW5Yvc2KwiErwIq\\\\\\\/URB+xgF1Q\\\\\\\/amiqZ9Wlmwk+XLWH\\\\\\\/EQU36P4cMHju8oE73vd85rcsxG7kb9oTH2ldK7YTZeccLKkNVqhxvmmbuSkiBgKsiMk+9278B9TGPWyjin1pb5xLliuQNbF4QUWw1DuVT+MsG551Lk3opdhi7kRE70NYFjZrPXpym1MY8aS9FIWrCBnboNXAhKAtCDPBseVI4vuEKkBYLvxj+mu8\\\\\\\/MXTA1Pixm6GWvDknS6RRzizw+pdNTfC7q0VOCx2sSiMKqTU2RFqppnG0m1RIsgokaB0V7NhLbHks8uvP6gbYgg79DAVhoghFOLXQQ+SoeScahnHz35G+VueWN3l1bkP8bJ+2f+bXsnkkBY+jBB4w+S+YDJhijyAYlRIq1jxZkQMFzZ6g\\\\\\\/ZJ86K3nLI7lEyam0a8XF3wMG22KfugFMxkrv85CkP20vBMEmwN9PWKkPlz8IbojC0rs7PtEXDInekA3tbxpzfDdIfJYPZ7LtBEzjCcBUHp1ZIladpmN9itVGiISbi9be3rVH0Fg\\\\\\\/l0Wytb7YaTwBtlBu1Nt7g8cmiDWqeYnQrvxK5K4pZybR8QhBMd27MJhjOvPibeDkdMbYv2qLNaaCnxtfeZ03qkl2Vp8LG6fJVz6EhIjvE3W1TDxYmtg9crnlqmaRArNcVrcWlA2R7NsJhtzX7P1t4RUGenCZMAcl7rYQMEMGr7tVD9fHQv\\\\\\\/ymj4HWX2BSQmKne4UtESupvboYv4pp5suD5TRwEQKxt04Y34oacSHjbe77ompqGow1CoLRw83xOAQDfKzp9iFWZOGch75duMmOFwOda+af\\\\\\\/kOjQcvK+IhlkkoX3ZdlheEfehpL1fHebEM+QAlDyWvEXal0k93thhQOmRcUGT5HAg5\\\\\\\/0DiK5TaMpnCu6N4wIIUIIVi+xm3NivldyqcwttpxFApbPSvOvqkzimUOfkW5b3ZJf1QNNjt\\\\\\\/qTo2uuyP2s5JmkkRK27B6aUkHX0dCoSZX0brzqFUa383G0d3Zfc+junNuIXxT8hnbc6dYQqG5MN2rP1Ru2E3DHmE8GFnBMWyqqU2S6mYct9hTkIKqe4G3I9iU3tTYT9u\\\\\\\/3qIN35pVUwccYHO9n9sRQT2B7\\\\\\\/7QWA853I9MFP6zqqVvj7tSX3ZcBXm0j2t1tioV6zedSfszaedQOS3kZAe4M6WpWvfkSLLrlwL5APJMGl6jj8Rs43fOghA77k\\\\\\\/AqTmnwEKt8EVpX+VtOmK2idzrYjK0+6UvWl6UZmHzd8ve\\\\\\\/kJcy3jnhJV4wEOXHuMvlj5XBZmn8Q9e8K6hQIHbGFDa5L3gFEMmJte0AKCALAGRRfVgF2DmOOr1HiahCCQ+ZEDuiKmYrkpXwCYAFLul8YBv2rFGpw\\\\\\\/SbQxh4Yp5hp8o\\\\\\\/9Wat5JHxxoCPagepahqNa9hNhHisQhhF1i\\\\\\\/n01tzMKGpMtYut4hKyqxtOvr7lcn3xoRoiy\\\\\\\/\\\\\\\/BkCwxTwPzw6Tm0HpwIpS4Hn5A6NeukNsEaWZE5oWKUwAt0PqTBLlsK5uZX8fr9qTdZ7tegthfgK2V4ngTdW9OT0USPl6YP7MRK2f4Udm6Eg72QnMWTzxj6UraeRW4X4POuwQLX8AXHslEOR2R7+S+GdqOF+2SXPXeonZAkoz2\\\\\\\/UqyZWd1IQ1SEtz7U+ZxKQ3oEeVYGACruTuC+D8CHlulLgiP6KOgQfri+kwld5ju8ilx4cBusT5kxHvMcZcmFO7RG3CtA0E1MGTQTlgBXr9y5QFO6tJuQGlXuS3CDWRfZj\\\\\\\/zqwQCRez+iWWAxiCOd\\\\\\\/NyM8I8zoS5zv1RQxxaZZ9s2LSCQHSuMHYrwoFdQ9\\\\\\\/FBMQg\\\\\\\/npIs\\\\\\\/VIcdzPXR47hntIQMGv4zkacZ+qdKf\\\\\\\/7xpF8CEweygJ4HlR3Vj7wf2BP8NVP72Z1VvNPOyiOws13kzGddFv61sZzm4RAl0s0pW5wwuEwyNNgyeovJ9SJESyGXvLmxv3DJwlKXXThizfsvah+AkHUBeWJTflNNCM2QyepySDaD4pxfVH08FjVaymdsdxQqbcxFhjbSGAWNk+8stv5LfWx2nhWYQDYeIlzzb7+OEwn8n4ibOb8T5hGe9Lwrc1QY2XOZYixA5MCRIifSsNKMiswzgzqF+HDjWDRzPhHV8v2ezemKUyI80fRrx3ciKfOTwaKJgBis0Ya3W6roi0B8l9giUbyrTx8AsjRkevqjiIjfHthbkGTDigJkoATdMT4Of\\\\\\\/6aDSWFqnmrcRatuzkUzkQgnyGGOOga\\\\\\\/o1hDMy5V2pavaK7Xho7v53plhcF7JCDQ8p56xuT8AQ3lI6fdC8mENM+ZevDxebsb0dM\\\\\\\/uKKIzVT8sUNUURI0DV\\\\\\\/0ANBJ7m3qntKzfZn9OaWrp866z\\\\\\\/48Tnw8FIJQS7M3uvHkqjozuqBLM01dwo1fNz+CSZGU9T+vydIaoNrvKSJvehDAewG1JUfUneCUolpmmsd9arm+k9LXMPDPWxUalsmZITbAzR6+DFlM\\\\\\\/hbiGbxh+64vITSR5o5tmqX3U7C0SKcnF4ejyVJs7wquRtXSX+bWr94ZMHnobSoWuBIWeYJ6eUzrbtLx5oyE4xV0sVokjiGvbF\\\\\\\/oZybFGHbCShRAEIJXYwlIafedCUnz4POyXEaENcCF1O80Zyv1J2\\\\\\\/Psw4zDub2suj\\\\\\\/xIAtgqmGFFbXcE0YR4n8JanjQXt+N63vhQx5WpAw0WzBI8SCyma4JTi4VVJNYGJlJTdIx0ieugNJ+NdICklmJgRgSRbIOtix70ZMrGhxuGDO6nJu7oRALnpdW23GTqEMLI30fzmpxcHvAXOf1LBfhaVB\\\\\\\/NhVP0ZdofrRtjVadBAhtGTgSpX6rs2ZtaD1p+BSshXGNSIXpYNbak5liDThMoY\\\\\\\/bbQ1pJfatiZhHea8H3SPB6+mAfoJI0zGHc1TeJ57brxEcjm\\\\\\\/BnX+3Xy\\\\\\\/L4rCLU7vVYg8Sv\\\\\\\/O265QSZkpETVyzhJxisGhwvaG5dqfOzBH2R\\\\\\\/EZn4gxQl9yXD6ejx1AU9XGjZHhhc2PfORePNbJ6P6Rws\\\\\\\/MsFXJ6oz8xGRKiz0aOYbxx0KpGsgMrNXeg9rZNv3IBjKaUcqxbgF4+jIOscXXQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022obqoA+obZ4cHehMr9hLICw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224287d502ad19295c1f43a2edb8797f5daa0428c65571a17bfd3ad25dd5fff707\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XRZJMRobVckuPMyU6d6Dn\\\\\\\/ExXxS66yKVw7Mwqiva4BvAltLgN9yltYs\\\\\\\/NslEZBA\\\\\\\/n+AyXd4m59y1vCbovxQmh9I\\\\\\\/1RW9XADNm4Mgy\\\\\\\/edJdmrqJQVAROtPiU6ZH2DVsI7dvhb30fHJV1OO88V8uBRQaS1AHLQjjGe0N7oALQfUhBAr9r4v50xkEO1SB+XSbDFLBe9EEuhjCaWN9BsioOq49xVSgWW0Y4E1LArVWfreScedeXiV6SeUMj9mYPJuTYdBngEzrAaYfp9C2ZHBcof+ZoIHQ1+jFT3b4jplmmkIKBjX43RKO4vulAZw5WVpXPJteihjMFnArBV9BrpklXRnG+iR60wyR4Hyem7PwB0\\\\\\\/aqzcuo90UbhAE\\\\\\\/BCHuWj+I+2lsN\\\\\\\/K3Zl7xT51qcbJ1tVhcy54Db\\\\\\\/RlnDKoaEuem7Pxs7D6IBDgyJEdxFqpX1wb5YF8l5+9mWwkoo4dHkxB92XYPxhcr8\\\\\\\/6ZpRJjyEA7w0MDsEvYUyuLmgyVmhytTQ595JspxuCoCcttkZ5N+yzSyR3BM8cVpu41i3330WrJ4q\\\\\\\/J6MEWeHuIgLMNvjFFHA3p\\\\\\\/dxstwTpDMkqeROBMB9K1HnwrSHMQS3dIajMdnlHj3hFNi7DKFU5T6xeD+WF3bZfqw4oFOsY9cqtHCHALFJXupe5JcINNav65vB+Y6jogY554VZD3YOzO0aZ7EFH7b+1TgHyWfGvrXoZtHO5jLHQqv5xGJ2UQoZBuLHhcUpNpBMINyW7lonWWmBJ9ZR1UdXI4xJIYavM0QODI62o+Ep17sB92CeYdb8OxM5WygUyl7ccpBlvQqdCLQ26UZ\\\\\\\/UtcZK\\\\\\\/auSKAb+iT1eAzgQPrhrD+uXHqnm4NDR0YP7\\\\\\\/nQEYiRMEAM+A8XdueNUl4NOxaqD47t+5cpl9Q2LJmi1f2eTmoLfrBr5aI5ibO6qrEu4x6u5xRZSY78PBeuBXN0r4hya5UC\\\\\\\/StEFggqPMs7PVNjtsLg57CpHM9zkkQs+9EVzPto20zFiVmu1ftlXEgbt1R4oLpkVK0p4x5Ps6hz8pbi5SqKm+TrBfzLZD\\\\\\\/ggx9dThfKjUiuphF2B5hMWn8xZVQ2M7ASxL5s2EQYUFkGq847R51pmsFSr+uudzXZQJzi3HhhkVGpBNd25vPPvlqdr5xZ9tmE38nj\\\\\\\/GzfWfjsJTa1EgVh\\\\\\\/o6DQbV8SyBWzIgYj+L2OOQnpytL\\\\\\\/1E9zAowaTPSVTyd3ZL57jDbc9UsaVNgqvynee7H2yoBRNYQyNfp7CCd+GJFpYXMmK9ThEgPn6eeidgrAjAhlbV4nMA18ZNnDF40ZwEYG6bXmHMVQt5XirMl8EkVzSM5mN4vsSCH5DsLuTY2WimbAFZHEuCBcxe8DlAsX85SCWmMxMXUwjKdudSyCNcSCMiQHLtGuQzEpaHSMgv9if9IhCM5Hs8HIP6949hVvX5vYRsh2W8y1Qux2vRutsFu0gIv75LjcCd6q7u6HWxM6+yXUfoQdTF3q7RSulk9LpAn2LzwX4h5eCnC2GhBmUuAZ4UFJ9NhjaymGuHxTqjh+V4Pg9fLfa2AbDP8hADgYqUtDomc8Yz1Cp0sXMCfTv\\\\\\\/K+73VY+DYsj0lUDPeVyx6LZAu3nINvDhrmWEj65TZgPQPvet7XK0VqCUevBxxmTZVrWtPPvkQRjUpntGDbaVz0BOVtTx0Dy9rSpMbB8iecH15ZnDlO1UGfhJTGoToo9Y59I1ojM4fbuNzvgKzOPLoC4qjFkDwt+J35+lkEq5W6W4kUJEr75gSF5YM3G7SVtrc65gXAXmwGV+TbMBDpLs6RNgaAy9kjDaN0Mt2s9xYxRiwVCxN8p3FZssjY9+fpJrmtEqkIm0kc\\\\\\\/Z9eOkcf\\\\\\\/DMqV7rgX1vpCeDYwYvBZ0+xy8AU9Yot8\\\\\\\/avGfo\\\\\\\/IstqBBE3xqJT92oVoFyqgnN9P8IabEwfXoISaDwYW2frqwZcnCda\\\\\\\/Mbd8XmmZs8I7h0gyQuwZUpc868cET2KKIQMXMUpkWXDLM8D0RF+UcyJtw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZBYfRBKgvLcEvwGM92N7DA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227132d60572713920ee28ba28a6f0893898ba9104401fb8d684c89cfc6c30c8bb\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460292 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2919" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-U2QFGOqnwadWBCEiJ607evUMhX9wrs9-YSdkPDeCUQM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2919" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0fQYfjSY-tqEoBxyb4aFUB2c9gChlSUR23Z_Lkbs1us" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2909" title="From Genotype to Phenotype: Nonsense Variants in SLC13A1 Are Associated with Decreased Serum Sulfate and Increased Serum Aminotransferases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2929" title="Genome-Wide Association Mapping for Female Infertility in Inbred Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460292" id="node460292" data-pisa="ggg;6/9/2919" data-pisa-master="ggg;g3.116.032532" data-apath="/ggg/6/9/2919.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Training Population Data and Validation of Genomic Selection for Economic Traits in Soft Winter Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amber Hoffstetter</span>, <span class="highwire-citation-author" data-delta="1">Antonio Cabrera</span>, <span class="highwire-citation-author" data-delta="2">Mao Huang</span> and <span class="highwire-citation-author" data-delta="3">Clay Sneller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2919-2928;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032532 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460292" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amber Hoffstetter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amber%2BHoffstetter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffstetter%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmber%2BHoffstetter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Antonio Cabrera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BCabrera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabrera%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonio%2BCabrera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mao Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mao%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMao%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Clay Sneller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clay%2BSneller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sneller%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClay%2BSneller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sneller.5@osu.edu" class="" data-icon-position="" data-hide-link-title="0">sneller.5@osu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/9/2919" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460292" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2919.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460292" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2919.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460292" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2919.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460292" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/9/2919.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460292" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection (GS) is a breeding tool that estimates breeding values (GEBVs) of individuals based solely on marker data by using a model built using phenotypic and marker data from a training population (TP). The effectiveness of GS increases as the correlation of GEBVs and phenotypes (accuracy) increases. Using phenotypic and genotypic data from a TP of 470 soft winter wheat lines, we assessed the accuracy of GS for grain yield, <em>Fusarium</em> Head Blight (FHB) resistance, softness equivalence (SE), and flour yield (FY). Four TP data sampling schemes were tested: (1) use all TP data, (2) use subsets of TP lines with low genotype-by-environment interaction, (3) use subsets of markers significantly associated with quantitative trait loci (QTL), and (4) a combination of 2 and 3. We also correlated the phenotypes of relatives of the TP to their GEBVs calculated from TP data. The GS accuracy within the TP using all TP data ranged from 0.35 (FHB) to 0.62 (FY). On average, the accuracy of GS from using subsets of data increased by 54% relative to using all TP data. Using subsets of markers selected for significant association with the target trait had the greatest impact on GS accuracy. Between-environment prediction accuracy was also increased by using data subsets. The accuracy of GS when predicting the phenotypes of TP relatives ranged from 0.00 to 0.85. These results suggest that GS could be useful for these traits and GS accuracy can be greatly improved by using subsets of TP data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/soft-wheat" class="hw-term hw-article-keyword hw-article-keyword-soft-wheat" rel="nofollow">soft wheat</a></li><li class="kwd"><a href="/keyword/breeding-values" class="hw-term hw-article-keyword hw-article-keyword-breeding-values" rel="nofollow">breeding values</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">The improvement of high-throughput marker technology and the ability to generate inexpensive and abundant marker data can allow the use of molecular markers to reshape breeding and aid breeders in making faster genetic gains than using only phenotypic data. Traits of great economic importance in soft red winter wheat include grain yield, resistance to <em>Fusarium</em> Head Blight (FHB, causal agent <em>Fusarium graminearum</em>), and various quality traits. While QTL have been identified for these traits in soft red winter wheat (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cabrera <em>et al.</em> 2015</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> 2016</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Liu <em>et al.</em> 2007</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Smith <em>et al.</em> 2011</a>) most have small effects suggesting that marker-assisted selection (MAS) for individual QTL will likely be ineffective.</p><p id="p-4">An alternative to MAS is genomic selection (GS), which is a marker-based breeding tool that estimates all marker effects simultaneously and uses all markers to model the genomic estimated breeding value (GEBV) of individuals (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Meuwissen <em>et al.</em> 2001</a>). Genomic selection uses a training population (TP) that has been genotyped and phenotyped to develop a model that can be used to calculate the GEBVs of additional related individuals based solely on their genotype (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jannink <em>et al.</em> 2010</a>). Individuals with desirable GEBVs can be used as parents to initiate a new cycle of breeding and to enter phenotyping trials. Genomic selection has been used in animal breeding to select sires and dams based on their GEBVs and is implemented routinely.</p><p id="p-5">The use of GS in plant breeding has lagged behind its use in animal breeding in part because the commercial value in animal breeding is an individual’s breeding value whereas in crops the value is the individual’s performance <em>per se</em>. In addition GS requires genomewide, high-throughput, low cost genotyping of 1000s of individuals and this technology has been lacking for many crops. In wheat, studies have shown that GS can accurately model quantitative traits. In a soft red winter wheat population, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Heffner <em>et al.</em> (2011b)</a> found GS accuracy for grain yield was 0.20 while the accuracy of predicting flour quality traits and heading date was high (0.66 to 0.76). <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Huang <em>et al.</em> (2016)</a> also studied GS accuracy in soft red winter wheat for agronomic and quality traits and reported GS accuracy ranging from 0.33 to 0.75. Other wheat studies have found prediction accuracy for grain yield ranging from 0.32 to 0.64 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Poland <em>et al.</em> 2012</a>). Others are working to determine the use of GS at improving disease resistance in wheat. The GS accuracy for predicting adult stem rust in wheat was 0.61 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rutkoski <em>et al.</em> 2014</a>) while the accuracy for FHB incidence and severity in wheat was 0.56 and 0.64, respectively (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rutkoski <em>et al.</em> 2012</a>).</p><p id="p-6">Research is being conducted to determine the optimum number of markers and lines in the TP to estimate GEBVs. <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo and Yu (2007)</a> used simulations and found increasing the number of markers increased accuracy when heritability was high, but not when trait heritability was low. Empirical studies have also shown that increasing the number of markers can increase prediction accuracy. In both biparental and multiparental wheat populations, the highest prediction accuracy was found using the largest marker set (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Asoro <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Heffner <em>et al.</em> 2011a, b</a>) though <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lorenz <em>et al.</em> (2012)</a> reported that increasing the number of markers did not result in increased accuracy. Most empirical studies also show that increasing the TP size improves prediction accuracy (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Heffner <em>et al.</em> 2011a</a>,<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">b</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lorenz <em>et al.</em> 2012</a>).</p><p id="p-7">These studies have investigated the effect on GS accuracy of modifying the TP by randomly deleting (or adding) lines and markers while little research has been done on optimizing GS accuracy by systematically selecting or eliminating TP lines or markers. <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Isidro <em>et al.</em> (2015)</a> optimized a TP of 1127 soft winter wheat lines by using three methods to eliminate TP lines. Their results showed that by using a systematic approach a small population can be created that produces a GS accuracy that is similar to that obtained by using a large population.</p><p id="p-8">There has been research in animals comparing GS accuracy using high- and low-density SNP chips in GS as the cost of the high-density chips can limit population size (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Weigel <em>et al.</em>, 2009</a>). <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Weigel <em>et al.</em> (2009)</a> used a population of Holstein bulls and reduced the number of SNPs from 32,518 to subsets of 300 to 2000 SNPs based on the absolute value of the allele effect or even spacing along the chromosome. Using the markers subset based on allele effects provided a higher accuracy than evenly spacing markers; however both produced somewhat lower accuracy than using all markers. <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Vazquez <em>et al.</em> (2010)</a> used a similar approach and got similar results. <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Moser <em>et al.</em> (2010)</a> reported that small subsets of markers selected based on allele effects gave a similar accuracy to that obtained using all markers. In a population of chickens, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdollahi-Arpanahi <em>et al.</em> (2014)</a> partitioned 350,000 markers into five groups based on the absolute value of their allele effect. Across all three traits the group of SNPs with the greatest allele effects provided comparable or higher accuracies than using all 350,000 SNPs. <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Schulz-Streeck <em>et al.</em> (2013)</a> worked with two maize populations and formed subsets of markers based on consistency of marker effects over environments. They reported that even subsets with 50 markers could provide similar accuracy to that obtained using the full set of 768 markers.</p><p id="p-9">A few studies have been conducted to determine the prediction accuracy of GS models built using the TP data to predict the phenotype of lines from independent data sets. <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Lorenz <em>et al.</em> (2012)</a> evaluated the accuracy of GS models built using data from one or two populations to predict the third population for FHB resistance and deoxynivalenol concentration in barley. When using one population to predict another, the prediction accuracies ranged from 0.47 to 0.77 for FHB and from 0.41 to 0.58 for deoxynivalenol concentration. The combination of two populations to predict the third population resulted in the highest prediction accuracies for both traits (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Lorenz <em>et al.</em> 2012</a>). Little research has been published using phenotypic data from one environment to predict phenotypes in another environment.</p><p id="p-10">Genomic selection will have the greatest impact on plant breeding when the TP has been optimized to produce the greatest GS accuracy and when phenotypic data from one set of environments can predict performance in a different set of environments. Our objectives were to (1) determine the prediction accuracy of different GS models for four economically important traits of SRWW, (2) determine the ability of the GS model built on TP data to predict the phenotype of different populations of SRWW with varying relationships to the TP, (3) determine the effect of using subsets of TP lines and markers on GS accuracy, and (4) assess the accuracy of a GS model built on data from one environment to predict performance in a different environment.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-11">Three populations were used in this study. The first population is the TP and was previously described by <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> (2016)</a>. The TP consisted of 470 F<sub>4</sub>-derived lines from The Ohio State University SRWW breeding program and is derived from 47 biparental crosses involving 23 parental lines that were selected for desirable GY, FHB resistance, quality, and pedigree diversity (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032532/-/DC1/TableS1.pdf">Table S1</a>). Seed was available for 22 of the 23 parental lines and these 22 lines comprise the parental population (PP) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032532/-/DC1/TableS2.pdf">Table S2</a>). The third population consisted of 94 F<sub>4</sub>-derived lines from The Ohio State University SRWW breeding program and is called the validation population (VP) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032532/-/DC1/TableS3.pdf">Table S3</a>). Of the 94 VP lines, 85 share some pedigree relationship to the TP: these 85 are termed the VP2 set. Seventeen VP lines were half sibs with some of the TP lines and these 17 are termed the VP3 set.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data</h3><p id="p-12">Phenotypic data for the TP were previously described by <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> (2016)</a>. The TP was evaluated in the 2009–2010 and 2010–2011 growing seasons. For GY, lines were grown in an augmented design with one or two replications at three locations: the Northwest Agricultural Research Station near Custar, Ohio (N 41° 16.8’ W 83° 50.4’), the North Central Agricultural Research Station near Fremont, Ohio (N 41° 21’ W 83° 7.2), and the Ohio Agriculture Research and Development Center near Wooster, Ohio (N 40<strong>°</strong> 46.2’ W 81° 55.8). Grain yield was obtained using a plot combine and seed weight per plot was adjusted to 13% moisture. The FHB index ratings were collected from an inoculated and misted nursery during June 2010 and 2011 in Wooster, Ohio from a randomized complete block design of single row plots with three replicates each year. Index was determined by visually estimating the percentage of symptomatic spikelets in a sample of spikes at about Feekes stage 11, which was about 21 d after heading (Feekes 10.3). A sample consisted of a handful of spikes (∼15 heads per handful) and three areas per plot were sampled. The SE and FY traits were analyzed using the methods of <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Smith <em>et al.</em> (2011)</a> on grain samples taken from one replicate from the Custar and Fremont Ohio locations in 2010 and 2011. For GS for FHB, SE and FY we used genotype means over all environments.</p><p id="p-13">Phenotypic evaluation of the PP was conducted during 2010–2011 and 2011–2012 growing seasons. The PP lines were included in part of a larger trial that was grown in an augmented design with either one or two replications at the same three locations as the TP (Custar, Freemont, and Wooster, Ohio). In 2010–2011 there was one replicate at each location and in 2011–2012 there were two replicates in Custar and one replicate in Wooster, Ohio. The augmented design consisted of five blocks per replication each with 38 unique lines and two checks (‘Malabar’ and ‘Pioneer 25R47’). Planting and harvest dates as well as nitrogen application are the same as described by <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> (2016)</a> for the TP. Heading date, height, FHB, SE, and FY data were collected as described for the TP. The VP lines were phenotyped during the 2011–2012 growing season in Custar and Wooster, Ohio using an augmented design with one replicate in each environment. The design consisted of three blocks each with 37 unique lines and three checks per block (Pioneer 25R47, ‘Bromfield’, and Malabar). Planting and harvest dates as well as fertilizer applications were the same as described by <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> (2016)</a>. Height, HD, and FHB were also recorded as described for the TP. The SE and FY data were analyzed on grain samples from the trial in Wooster, Ohio.</p><p id="p-14">Best linear unbiased predictions (BLUPs) of line effects were obtained as described by <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> (2016)</a>. For grain yield, BLUPs were obtained over all environments (GYA) and just the Wooster (GYW) environments for all populations. BLUPs were also obtained over the combined data of Custar and Fremont (GYN) environments for the TP only. The BLUPs for FHB, FY, and SE were obtained over all environments for all populations. To normalize the TP FHB data, a square root transformation was conducted and BLUPs of these values were obtained for all lines. Subsets of TP lines with low genotype-by-environment (GEI) variance and high heritability, as described by <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> (2016)</a>, were also used for each trait. Within each trait, the GEI variance for each line was plotted from lowest to highest and the “elbow” of the curve was visually estimated and then further identified by evaluating the difference between the GEI variance for consecutive lines in the data file: the elbow occurs where the slope of the two lines changes and that occurs where the difference between consecutive lines increases. Lines having a GEI variance greater than the elbow point were eliminated from the analysis. This created a subset of lines for GYA, GYW, and GYN containing 400 lines, a subset for FHB containing 440 lines, and a subset for quality traits containing 447 lines.</p></div><div id="sec-4" class="subsection"><h3>Genotypic data</h3><p id="p-15">DNA for genotyping was extracted from a sample of lypholized leaves collected from two 2-wk-old seedlings per line using a 96-well format (DNAeasy 96 Plant Kit, Qiagen Group, Valencia, CA). All three populations were genotyped using genotyping-by-sequencing technology by Triticarte Pty Ltd Yarralumla, ACT, Australia (<a href="http://www.triticarte.com.au">http://www.triticarte.com.au</a>). The genotyping produced 4858 biallelic SNP markers and 28,311 dominant DaRT markers that were scored in the TP. For most of the GS work we only used the 4858 SNPs. Of the 4858 SNPs, 2442 SNP markers were scored in common between the PP, VP, and the TP. When using the TP data to develop GEBVs for the VP and PP lines, we only used the 2442 common markers. All markers used in the analysis had a call rate of at least 70%. Missing data in the TP were imputed based on the average marker score of lines from the same cross. Missing values were imputed as 1 if the average genotype score for other lines from that family was greater than 0 or as −1 if the average was &lt;0. Missing data in the PP and VP populations were imputed as the common allele.</p><p id="p-16">Unique subsets of the 4858 SNPs and the 28,311 DaRT markers from the TP were made for each trait using results from an association analysis (<a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> 2016</a>) that was conducted using the subsets of TP lines with low GEI for each trait. Marker set 1 was the same for all traits and consisted of all the 33,169 DaRT and SNP markers scored in the TP. Marker subset 2 varied by trait and contained markers that were significantly associated with the trait at <em>P</em> &lt; 0.05. For GY, marker subset 2 contained 2902 markers with a <em>P</em> &lt; 0.05 for GYW or GYN: the correlation of marker effects for GYN and GYW was <em>r</em> = 0.64 (<a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> 2016</a>). Marker subset 3 varies by trait and for grain yield it included markers from subset 2 but excluded markers with a high residual variance. The residual variance of markers ranged from −1.42 to 1.69. Huber weights were used to remove markers with a high residual variance using the ‘MASS’ package and rlm of R.3.0.1. The criteria for marker subset 3 for FHB and quality differ from grain yield because marker effects were estimated over environments for these traits and not individual environments as for grain yield (<em>e.g.</em>, GYN and GYW). Marker subset 4 varied by trait and included markers associated with each trait at a <em>P</em> &lt; 0.01. For GY, this subset included 604 markers with a <em>P</em> &lt; 0.01 for GYN or GYW. Marker subset 5 varied by trait and contained markers with a <em>P</em> &lt; 0.005 for each trait. Finally, marker subset 6 was created for grain yield only and included 362 markers with <em>P</em> &lt; 0.01, a low residual variance, and the average of the absolute value of allele effects for GYN and GYW greater than 30 kg hectare<sup>-1</sup>. The number of markers in each set for each trait is shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469234/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469234/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Accuracy of genomic selection for grain yield (all environments, GYA; Wooster Ohio, GYW; Northwest Ohio, GYN), <em>Fusarium</em> Head Blight resistance (FHB), softness equivalence (SE), and flour yield (FY) using either all TP data (470 lines and 33,169 markers) or subsets of lines (<em>n</em> &lt; 470) chosen for low genotype-by-environment interactions, and subsets of marker data (M2–M6) chosen based on significance criteria (see <em>Materials and Methods</em>)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Population relatedness</h3><p id="p-18">The degree of relationship between the TP and the VP was calculated using the 2422 SNP markers scored in both the TP and VP. Relationship was calculated as a simple matching coefficient which is equal to the probability that alleles randomly sampled from two individuals are identical by state. The simple matching coefficient was calculated using PROC IML in SAS 9.1. The relationship between the VP and the TP was visualized using a principal component analysis (PCA) conducted in R 3.0.2 using the function ‘eigen’. The scores of the first and second principal components were plotted to create a PCA graph to show how the VP lines grouped with the TP lines.</p></div><div id="sec-6" class="subsection"><h3>Genomic selection</h3><p id="p-19">Three GS models were used in the TP with the 4858 SNP markers: Ridge-Regression BLUP (RRBLUP), Bayesian LASSO, and Random Forest. For each model ten-fold cross-validation was used to estimate GEBVs and the accuracy of GS. For cross -validation the TP lines in the prediction set were randomly selected without replacement allowing GEBVs of all 470 lines to be predicted within one cycle. Each model was implemented in R 3.0.2 (R Development Core Team 2011) using a Windows 2008 64-bit virtual machine with a 3.46 GHz Intel(R) Xenon(R) CPU processor and 32.0 GB of RAM.</p><p id="p-20">RRBLUP was conducted using the ‘rrBLUP’ package 4.2 in R (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Endelman 2011</a>). The function mixed.solve was used, which uses a mixed model in the form of:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-1.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-2.gif"/></noscript></span></span>where <strong>X</strong> is the design matrix for the fixed effects <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-3.gif"/></noscript></span></span>, <strong>Z</strong> is the design matrix for the random effects <em>u</em>, <em>u</em> are the random effects, <strong>K</strong> is a positive semidefinite matrix that accounts for the relatedness among individuals based on markers, and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-4.gif"/></noscript></span></span> is the additive variance (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Endelman 2011</a>). The residuals are normal with a constant variance and variance components were estimated by REML using the spectral decomposition algorithm of <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kang <em>et al.</em> (2008)</a>. A marker-based formulation was used to estimate GEBVs for each marker, which were then multiplied by the marker matrix to give the resulting GEBVs of the individuals in the prediction set. Bayesian LASSO predictions were performed using the ‘BGLR’ package for R, version 1.0.4, and hyper-parameters chosen based on the guidelines of <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Perez <em>et al.</em> (2010)</a>. The residual variance was calculated using <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-5.gif"/></noscript></span></span> = 3 and the equation:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-6.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-7.gif"/></noscript></span></span> was chosen to reflect the expectation of the model’s residual variance. To calculate the variance of the infinitesimal effect of the <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-8.gif"/></noscript></span></span> = 3 and the equation:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-9.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-10.gif"/></noscript></span></span> is the average of the diagonal value of <strong>A</strong>, the kinship matrix, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-11.gif"/></noscript></span></span> is the prior expectation of <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-12.gif"/></noscript></span></span>. Finally, lambda was calculated with the following equation:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-13.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-14.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-15.gif"/></noscript></span></span> is the sum of squares (over markers) of the average genotype (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Perez <em>et al.</em> 2010</a>). Random Forest predictions were made using the ‘randomForest’ package version 4.6-10 in R (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Liaw and Wiener 2002</a>). The RRBLUP and Bayesian LASSO models were conducted using 1500 cycles while the Random Forest model was conducted using only 500 cycles due to computational intensity. All six traits in the TP were analyzed (GYA, GYW, GYN, FHB index, SE, and FY) and FHB index was also transformed using a square root transformation to bring the data into normalcy (FHB_SQRT).</p><p id="p-21">The accuracy of a GS model was determined using a Pearson’s correlation between the GEBVs of all lines and their phenotypic BLUP. The relative efficiency per cycle of GS as compared to a cycle of phenotypic selection (RE<sub>c</sub>) was calculated as <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-16.gif"/></noscript></span></span> where <em>r</em> is the accuracy of the model and <em>H</em> is the heritability of the trait (<a id="xref-ref-16-10" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> 2016</a>). Relative efficiency of GS per year (RE<sub>y</sub>) was determined by multiplying the RE<sub>c</sub> by the ratio of years to complete one cycle of phenotypic selection <em>vs.</em> years to complete one cycle of GS for each trait. In our program it is estimated to take 7 yr for one cycle of phenotypic selection for grain yield and 5 yr for one cycle for FHB, FY, and SE. One cycle of GS can be completed in 1 yr in winter wheat. Finally, proc.time was used to determine the length of time to complete 300 cycles of 10-fold cross-validation for each GS model.</p><p id="p-22">The six marker subsets described above were used to create GS models for each trait using RRBLUP. We then used 10-fold CV to determine the accuracy for GS for each trait and marker subset. Marker subsets were used to build GS models using the TP data and then determine the accuracy of GS for each marker subset/trait combination using either all 470 TP lines or the subsets of lines with low GEI.</p><p id="p-23">A model was built using the 2442 SNPs that were scored in common in the TP, PP, and VP and used to calculate the GEBVs of the VP and PP lines. These GEBVs were then correlated to the phenotypes of the PP and VP lines. Phenotypic information from the TP together with markers scored in common between the TP and PP and RRBLUP were used to calculate the GEBVs for the 21 PP lines. The TP is composed of progeny of the PP lines therefore the true breeding value (TBV) of each of the PP lines could be calculated to assess the ability of the GS model to predict the TBVs. The TBV of the ith PP line was calculated for each trait using the equation:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-17.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-18.gif"/></noscript></span></span> is the mean of the offspring of the ith parent, and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-19.gif"/></noscript></span></span> is the mean of the TP. Both the phenotype and TBV of the PP lines were then correlated to their GEBV using unweighted and weighted correlations. A weighted correlation was used to account for the fact that some PP lines contribute more parentage to the TP than others. The weighted correlation used the proportion of TP parentage from each PP line as a weighting factor and was calculated using the equation:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-20.gif"/></noscript></span></span>where <em>w</em> is the weight of each parent, <em>x</em> is the GEBV for each PP, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-21.gif"/></noscript></span></span> is the mean GEBV of all PPs, <em>y</em> is the phenotype or TBV of each PP, and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9/2919/embed/mml-math-22.gif"/></noscript></span></span> is the mean phenotype or TBV of all PPs.</p><p id="p-24">The RRBLUP model was retrained using the TP phenotypic and data from markers that were scored in both the TP and the VP to obtain GEBVs of each VP line using the same method as described for the PP population. The prediction accuracy was determined using a Pearson’s correlation between the GEBVs of the VP lines and their phenotypic BLUPs. Accuracy was determined using the entire VP data set, as well as for the VP2 and VP3 subsets.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-25">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Phenotypic data</h3><p id="p-26">Phenotypic data from the TP were previously described by <a id="xref-ref-16-11" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> (2016)</a> and are briefly recapped here for convenience. For GY, each of the environments (location/year combination) produced a wide range of values for each trait. The GEI pattern among the six test sites showed that the two Wooster environments clustered together and provided different results than the Northwest Ohio (Custar and Fremont) environments (<a id="xref-ref-16-12" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> 2016</a>). The entry-mean heritability for yield over all environments (GYA) was 0.60, in the Wooster environments (GYW) was 0.69, and in the Northwest environments (GYN) was 0.51. Heritability was 0.59, 0.85, and 0.92 for FHB, FY, and SE, respectively. For all traits there were lines in the TP, PP, and VP with numerically superior phenotypes to the elite check for each trait. Thus like the TP, the PP and VP can be considered elite, adapted soft red winter wheat populations.</p></div><div id="sec-10" class="subsection"><h3>Training population (TP) cross-validation</h3><p id="p-27">Using only RRBLUP we obtained nearly identical accuracy using all 33,169 markers or just the 4858 SNPs. We decided to use just the 4858 SNPs to compare the accuracy of GS with different models as some models can take far too much time with 33,169. Using 10-fold cross-validation the RRBLUP and Random Forest models had the highest accuracies for all traits and produced nearly identical results (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The Bayesian LASSO model had the lowest accuracy for all traits with the exception of GYW. Excluding the Bayesian LASSO results, GS accuracy ranged from 0.34 to 0.63. Accuracies for GYA and GYN were significantly lower than for GYW. We determined which model was computationally faster for estimating GEBVs in the TP. When using all TP lines, 4858 markers, 300 iterations, and 10-fold CV, RRBLUP was fastest completing the process in 5.6 hr, which was 28.8 hr less than Random Forest and 80 hr less than Bayesian LASSO. Only results from using RRBLUP will be discussed further.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469244/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469244/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genomic selection accuracy (<em>r</em>), standard deviation of accuracy (σ), relative efficiency per cycle (RE<sub>c</sub>), and relative efficiency per year (RE<sub>y</sub>) of three genomic selection models using a TP of 470 wheat lines and 10-fold cross-validation for four traits: grain yield over all six environments (GYA), grain yield at Northwest Ohio environments (4 environments, GYN), grain yield at Wooster, Ohio (2 environments, GYW), flour yield (FY), softness equivalence (SE), and <em>Fusarium</em> Head Blight index (FHBI)</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">The relative efficiency of GS <em>vs.</em> phenotypic selection per cycle (RE<sub>c</sub>) and per year (RE<sub>y</sub>) was calculated. All assessments of relative efficiency showed GS to be more efficient than phenotypic selection. The greatest RE<sub>y</sub> for any grain yield trait was found for GYW (4.8), which was the grain yield trait with the highest accuracy and the highest heritability.</p></div><div id="sec-11" class="subsection"><h3>Using TP data to predict the value of other populations</h3><p id="p-33">We assessed the ability of GS to predict the phenotypes of related lines that were phenotyped in independent environments. We used RRBLUP and all TP data to calculate the GEBVs of the PP and VP lines and estimated accuracy by correlating those GEBVs with the phenotypes of the PP and VP lines using both unweighted and weighted correlations. The RRBLUP model was chosen because it was more efficient (lower computational time and equal or higher accuracies) than Random Forest or Bayesian LASSO. The accuracy of GS models based on the TP data to predict the PP phenotypes was considerably greater for all traits when we used a weighted correlation than an unweighted correlation (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The accuracy of GS ranged from −0.23 to 0.85 for weighted correlations. Accuracy was low for GYA and GYN but was 0.67 for GYW using the weighted correlation. On average, the GS accuracy for predicting the TBV of the PP was similar or lower than the accuracy for the PP phenotypes (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). As with the phenotypes, accuracy was generally higher when using the weighted correlation than the unweighted correlation. Interestingly, the accuracy for GYA and GYN increased greatly when using TBV <em>vs.</em> phenotypes while the accuracy for SE and FHB was greatly reduced. The difference between the mean phenotype of the top three (T3) and bottom three (B3) PP lines showed that the T3 group was always superior to the mean of the B3 group for all traits and this difference was significant at <em>P</em> &lt; 0.1 for GYW, SE, and FHB.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469150/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469150/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Genomic selection accuracy obtained using data from the TP to calculate genomic estimated breeding values (GEBVs) for lines in the Parental (PP) and Validation (VP) populations and correlating those GEBVs to the phenotypes of the PP and VP lines</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">In general all accuracies from using TP data to predict the GEBVs of the VP lines were low and all were lower than those observed when using the PP phenotypes and weighted correlations (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Assessing accuracy using data from just 85 VP lines that were related to the TP and PP did not impact GS accuracy. In the VP, only SE and FHB were moderately predicted by the TP data with accuracies ranging from 0.22 to 0.33. There was no significant difference between the mean phenotype of the top three (T3) and bottom three (B3) VP lines as ranked by their GEBVs for any trait either (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The T3 group was superior to the B3 group for GYN, FY, SE, and FHB and this difference was significant for FY and FHB at <em>P</em> &lt; 0.1.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469223/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469223/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Average phenotype of the top three (T3) and bottom three (B3) of the 21 parental lines (PP) and validation population lines (VP) as ranked based on their genomic estimated breeding values (GEBVs) that were predicted using data from the training population (TP) and the Ridge-Regression BLUP model</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">The results of the simple matching coefficient indicate the VP and the TP populations are related. The average genetic similarity within each population alone was 0.69, while the average genetic similarity between the VP and TP was 0.68. The PCA also shows a relationship between the VP lines and the TP lines (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2919/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component analysis of the 470 wheat lines of the training population (TP) and the 94 lines of the validation population (VP) using data from markers scored in both populations. Five groups of TP lines were defined by Cabrera et al. 2014 and are identified as groups 1 to 5. The VP are identified by the &#x201C;+&#x201D; and the &#x201C;*&#x201D; where the &#x201C;+&#x201D; represents the 85 lines with varying relationship to the TP and the &#x201C;*&#x201D; represent the 17 VP lines that are half sibs of the TP." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1154872945" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Principal component analysis of the 470 wheat lines of the training population (TP) and the 94 lines of the validation population (VP) using data from markers scored in both populations. Five groups of TP lines were defined by Cabrera &lt;em&gt;et al.&lt;/em&gt; 2014 and are identified as groups 1 to 5. The VP are identified by the &#x201C;+&#x201D; and the &#x201C;*&#x201D; where the &#x201C;+&#x201D; represents the 85 lines with varying relationship to the TP and the &#x201C;*&#x201D; represent the 17 VP lines that are half sibs of the TP.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2919/F1.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/9/2919/F1.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2919/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2919/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469130" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-39" class="first-child">Principal component analysis of the 470 wheat lines of the training population (TP) and the 94 lines of the validation population (VP) using data from markers scored in both populations. Five groups of TP lines were defined by <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cabrera <em>et al.</em> 2014</a> and are identified as groups 1 to 5. The VP are identified by the “+” and the “*” where the “+” represents the 85 lines with varying relationship to the TP and the “*” represent the 17 VP lines that are half sibs of the TP.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Accuracy of GS using subsets of TP data and cross-validation</h3><p id="p-40">The accuracy of the GS model was determined using data subsets derived from the TP by (1) removing TP lines with high GEI from the TP for each trait and (2) selecting markers based on results from an association analysis conducted using the TP data (<a id="xref-ref-16-13" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> 2016</a>). In this analysis we used all 33,169 markers as they were used by <a id="xref-ref-16-14" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> (2016)</a> in an association analysis in this population. Relative to using all TP data, the accuracy of GS for grain yield decreased 23% when using data from all markers and a subset of 400 lines with low GEI, though using only lines with low GEI had little effect on accuracy for the other traits (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-41">The GS accuracy was increased when using data from all TP lines and subsets of significant markers for all traits except GYN (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Excluding GYN, GS accuracy for the marker data subsets increased relative to using all markers by 41% (FY) to 76% (FHB) with the largest gains for all traits occurring for marker subset 2. The other smaller marker subsets gave similar accuracies as marker subset 2. Marker subset 5 had the fewest markers for all traits with, on average, &lt;1% of the markers in the full set, yet provided very similar accuracies to those of marker subset 2, and provided superior accuracy compared to the full set of markers for all traits (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). For example, marker subset 5 for FHB had just 134 markers and had an accuracy (0.58) that was similar to that of marker subset 2 (0.64) with 1672 markers and superior to the accuracy using the full set of 33,169 markers (0.35). We also evaluated GS accuracy using all nonsignificant markers (<em>e.g.</em>, all markers minus the subset 2 markers) for each trait. These subsets of nonsignificant markers produced within-environment GS accuracies that were on average 20% lower than the accuracy obtained using all marker data, and 46% less accurate than using marker subset 2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032532/-/DC1/TableS4.pdf">Table S4</a>).</p><p id="p-42">There was a positive interaction between using subsets of TP lines and subsets of markers for GYA, GYN, and FHB where the greatest accuracies were obtained using the combination of a subset of lines with low GEI and marker subset 2: these accuracies were 75% (GYA), 113% (GYN), and 121% (FHB) greater than the accuracy obtained using all TP data (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). For GYW, using just the subset of low GEI lines decreased accuracy, though using the subset of lines and the reduced marker sets resulted in accuracies that were similar to those obtained using all data (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). The combination of subsets of low GEI lines and marker subsets did not increase GS accuracy for SE and FY above that obtained using just the reduced marker subsets.</p></div><div id="sec-13" class="subsection"><h3>Using subsets of TP data to predict phenotypes between environments</h3><p id="p-43">Phenotypic data for grain yield in the TP from only one set of environments (<em>e.g.</em>, GYW data or GYN data) were used to build a GS model that was then used to predict the phenotype of the TP lines in the other set of environments. The accuracies of these between-environment predictions were low and negative when using all TP data (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). Positive GS accuracies were obtained using the data from the subset of 400 lines with low GEI and all markers (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>), and was even further increased by using marker subsets (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). The highest between-environment accuracy (<em>r</em> = 0.37) was found using the 400 low GEI lines and marker subset 6. This set contained 362 markers that had low residual variance, were significant for GYW or GYN, and the absolute value of allele effects for GYW or GYN were greater than 30 kg hectare<sup>-1</sup>. Similar, positive accuracies for between-environment predictions for grain yield were also found with the other marker subsets. The results were similar when using either GYW or GYN to train the model and predicting the other environment.</p><p id="p-44">The TP phenotypic data for FHB, FY, or SE collected from 1 yr (2010 or 2011) were used to build a GS model that was used to predict the phenotype of the TP lines in the other year. When using all marker data the GS accuracy for between-year was low for FHB (range 0.13 to 0.16) and moderate for SE and FY (range = 0.24 to 0.59) when using either all TP lines or the subsets of lines with low GEI (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>). The GS accuracies remained low for FHB when using the marker subsets and 2010 data to build the model. The GS accuracies for FHB increased though when using the reduced marker subsets and 2011 phenotypic data to build the model: the accuracy using marker subset 5 and lines with low GEI, 0.42, was 180% greater than using all TP data (0.15).</p><p id="p-45">For SE and FY, removing lines with high GEI did not increase prediction accuracy between years regardless of marker subset (<a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1</a>). Using reduced marker subsets had little impact on between-year accuracy when using 2011 data to train the model to predict 2010 FY phenotypes, and produced a modest improvement for predicting 2010 SE phenotypes. Using the reduced marker subsets with 2010 TP data increased the ability to predict the 2011 phenotypes as GS accuracy increased by an average of 70% for SE and 26% for FY when compared to using all TP data.</p><p id="p-46">We also evaluated GS accuracy using all nonsignificant markers (<em>e.g.</em>, all markers minus the subset 2 markers) for each trait. These subsets of nonsignificant markers produced between-environment GS accuracies that were on average 15% lower than the accuracy obtained using all marker data, and 33% less accurate than using marker subset 2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032532/-/DC1/TableS4.pdf">Table S4</a>).</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><div id="sec-15" class="subsection"><h3>Cross-validation of GS using all TP data</h3><p id="p-47">The TP, PP, and VP all consisted of elite inbred soft red winter wheat breeding lines with adaptation to Ohio. The populations showed a range of phenotypes, with some lines in each population outperforming the best checks for each trait. The TP data were previously used to detect QTL for GY, FHB resistance, SE, and FY (<a id="xref-ref-16-15" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> 2016</a>) though none of the QTL identified for these traits had a large effect suggesting the population would be better suited for GS for these traits.</p><p id="p-48">For all traits except GYW, the RRBLUP and Random Forest GS models produced similar GS prediction accuracy while the Bayesian LASSO model produced the lowest prediction accuracy (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). This is consistent with results from other studies which have found that shrinking all marker effects equally toward 0 and assuming they have a common variance produces similar or superior results to those models that do not impose this restriction such as Bayes A and Bayes B (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Heffner <em>et al.</em> 2011a</a>,<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">b</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Lorenz <em>et al.</em> 2012</a>). The results of this study also found the RRBLUP model to be computationally much faster than the other two models, therefore it was chosen as the model for all further data analysis. Further discussion will be restricted to the RRBLUP results.</p><p id="p-49">The cross-validation accuracies from the GS models built using all TP data indicate that GS could be a useful tool within this population for these economically important traits. For grain yield the highest accuracy was found using data from the Wooster, Ohio (GYW) environment alone (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). During the 2010–2011 growing season, some plots at the northern Ohio locations suffered from water damage (<a id="xref-ref-16-16" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> 2016</a>). This resulted in a lower entry-mean heritability for GYN (<em>H</em> = 0.51) compared to GYW (<em>H</em> = 0.69) and likely explains why accuracy for GYN was lower than GYW. The accuracies for grain yield using all TP lines and marker data ranged from 0.41 to 0.57 (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>) and are generally similar to, and often higher than, the accuracy for grain yield reported by others. In a soft winter wheat multifamily population, <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Heffner <em>et al.</em> (2011b)</a> reported accuracy for grain yield of 0.20 using RRBLUP as the prediction model. The prediction accuracy of RRBLUP for grain yield was 0.36 in another soft winter wheat population (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Heslot <em>et al.</em> 2012</a>). <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Combs and Bernardo (2013)</a> reported a prediction accuracy of 0.10 for grain yield using the RRBLUP model in a wheat population. Prediction accuracies of 0.30 to 0.32 for grain yield were reported in a population of wheat lines from Chile, CIMMYT, and Uruguay using RRBLUP (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lado <em>et al.</em> 2013</a>). Using two populations of wheat, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> (2014)</a> found the accuracy of GS for grain yield using RRBLUP ranged from 0.29 to 0.67 in a population of 306 lines, while in a population of 599 lines the accuracy was 0.42 to 0.45.</p><p id="p-50">The GS accuracy for FHB using all TP data was 0.35, which is lower than most other reports for FHB resistance. In barley, the accuracy of GS for predicting the percentage of FHB infected kernels ranged from 0.41 to 0.67 (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Lorenz <em>et al.</em> 2012</a>) and for predicting FHB severity the accuracy was 0.30 to 0.39 using RRBLUP (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sallam <em>et al.</em> 2015</a>). In soft winter wheat, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Rutkoski <em>et al.</em> (2012)</a> reported a prediction accuracy of 0.56 and 0.64 for FHB incidence and severity, respectively. The low accuracy of this study could be because there was low disease pressure in 2011 (<a id="xref-ref-16-17" class="xref-bibr" href="#ref-16">Hoffstetter <em>et al.</em> 2016</a>) even though the heritability for FHB in this study was moderate (H = 0.69). The accuracies of FY and SE with RRBLUP were 0.62 and 0.51, respectively (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Heffner <em>et al.</em> (2011b)</a> reported a GS accuracy of 0.76 and 0.66 for FY and SE, respectively, in a multifamily soft wheat population using RRBLUP. The accuracy for FY and SE in a biparental population was 0.57 and 0.37, respectively (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Heffner <em>et al.</em> 2011a</a>). The accuracies reported for FY and SE in our study are in these ranges.</p><p id="p-51">The relative efficiency per cycle (RE<sub>c</sub>) of GS across traits mostly reflected the GS accuracy. The RE<sub>c</sub> for SE and FY in this study (0.53 and 0.68, <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>) are comparable to those reported by <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Heffner <em>et al.</em> (2011a)</a> in two biparental populations (0.29 to 0.70). Over all traits the RE<sub>c</sub> values show that a cycle of GS is not as efficient as a cycle of phenotypic selection. However, the real advantage of GS is it can shorten the duration of a breeding cycle and improve gain per year (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Goddard and Hayes 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Heffner <em>et al.</em> 2009</a>, 2011b; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Jannink <em>et al.</em> 2010</a>). In this population the relative efficiency of GS on per year basis (RE<sub>y</sub>) ranged from 2.4 (FHB) to 4.8 (GYW, <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>) which indicates that GS can be an efficient method for making genetic gains for these four traits by decreasing the breeding cycle time to 1 yr.</p></div><div id="sec-16" class="subsection"><h3>Validating the TP GS model in related populations</h3><p id="p-52">The ability of the GS models built using all of the TP data to predict the phenotypes of other related lines varied by trait and population. The best results were obtained when predicting the phenotype of the PP lines as these lines were more related to the TP than were the VP lines. For most traits in the PP population, higher prediction accuracy was found using a weighted correlation than an unweighted correlation (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). The reason the weighted correlation provides a higher accuracy then the unweighted correlations can be attributed to the fact that each PP line did not contribute equally to the parentage of the TP and weighting the correlation accounts for this. The models were less predictive of the TBV of the PP lines than it was of the phenotype of the PP lines. Because the TBV is estimated based on the performance of the progeny, the TBV of PP lines with few offspring in the TP is probably poorly estimated. In addition, the TBV of an individual should be estimated by mating the individual to a large sample of random individuals from the population. This was not the mating scheme used in this study, therefore it is likely that all estimated TBVs in this study have considerable bias. Overall the GS accuracies found for the PP range from −0.15 to 0.85 across all traits and analyses and shows that the performance of the PP lines can be estimated using GS models built using progeny (TP) data. To date, there is little research on using the progeny to predict the performance of parental lines. <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Sallam <em>et al.</em> (2015)</a> used the parental lines to predict the performance of the progeny in a barley population. Using RRBLUP the accuracy for grain yield was 0.57. The accuracies found for GYW (0.57 and 0.67, <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>) in this study are very close to those reported by <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Sallam <em>et al.</em> (2015)</a>.</p><p id="p-53">The GS model built using the TP data were less successful at predicting the performance of the VP lines than the PP lines even though the VP and TP appeared quite related based on marker similarity. This marker-based similarity between the VP and TP occurred despite having only three immediate parents in common. The poor predictive performance for the VP sets may also be due to limited phenotyping of the VP as all phenotypes, except for FHB, were obtained from unreplicated trails. Also the VP lines were evaluated in a different year from the TP and therefore GEI may be affecting the ability to predict the performance of lines tested in another environment. Collectively, the results from the PP and VP show a GS model built using the TP data can predict the phenotypes of a highly related population, even when the phenotypes were obtained from different environments: the degree of this accuracy though varied by trait.</p></div><div id="sec-17" class="subsection"><h3>Accuracy of GS when using subsets of TP data</h3><p id="p-54">Most studies and simulations show that increasing the size of the TP and the number of markers increases the accuracy of GS (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernardo 2009</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>; <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Heffner <em>et al.</em> 2011b</a>; <a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Lorenz <em>et al.</em> 2012</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Poland <em>et al.</em> 2012</a>). In contrast, we show that the accuracy of GS can be greatly improved by systematically reducing the number of lines and markers in the TP data set. Compared to using all lines, removing TP lines with high GEI from the TP either reduced, or did not impact, prediction accuracy when we used all markers. When using all TP lines and systematically reducing the number of markers in the model to include only those markers significantly associated with the trait at <em>P</em> &lt; 0.05 (marker subset 2), the prediction accuracy was increased for all traits except GYN (<a id="xref-table-wrap-1-12" class="xref-table" href="#T1">Table 1</a>). Depending on the trait, marker subset 2 contained just 4.7–8.7% of all the markers. The gain in GS accuracy from using marker subset 2 with all 470 lines ranged from 39% to 83% (excluding GYN) compared to using all markers. In addition marker subset 5 had &lt;1% of the TP markers for any trait and produced GS accuracies that were similar to those of marker subset 2 and that were superior to using all markers for all traits except GYN.</p><p id="p-55">Therefore using only markers with statistical evidence of being associated with QTL increased the accuracy of GS for all traits except GYN regardless of the number of lines in the TP. Others have shown that selecting a greatly reduced subset of markers based on effects (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abdollahi-Arpanahi <em>et al.</em> 2014</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Moser <em>et al.</em> 2010</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Vazquez <em>et al.</em> 2010</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Weigel <em>et al.</em> 2009</a>) or consistency (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Schulz-Streeck <em>et al.</em> 2013)</a> produced GS accuracies that were similar to those obtained using all markers. Our results are unique as markers were selected based on significance and the subsets often produced increased accuracy relative to using all markers. The fact that marker subset 2 provides the highest prediction accuracies for most traits could be attributed to this marker set including enough markers to represent the genome while only including those with evidence of being linked to QTL controlling the trait. It should be noted that similar GS accuracies were obtained for all traits with marker subset 5 which were likely too small to cover the wheat genome, yet must be covering the most relevant regions of the genome. It is easy to understand why a model using only the significant markers is more predictive than a model using all markers as the effects associated with the selected markers are larger and more likely reflect the effects of real QTL than markers that are not significant. Including markers that are not significantly associated with a QTL lowered the accuracy perhaps because the effects associated with these regions are poorly estimated and thus these markers primarily add error or noise to the data set. In addition RRBLUP distributes variation equally over all markers and equally shrinks all marker effects toward zero (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Meuwissen <em>et al.</em> 2001</a>). By removing markers not statistically associated with a trait, perhaps the model can more accurately estimate marker effects and there is less shrinkage of true marker effects.</p></div><div id="sec-18" class="subsection"><h3>GS accuracy between environments</h3><p id="p-56">Using data from one set of environments to predict the performance in independent environments is the crux of plant breeding. The GS prediction accuracy between environments for grain yield was very low (<a id="xref-table-wrap-1-13" class="xref-table" href="#T1">Table 1</a>) when using all markers and lines. However, the accuracy of between-environment predictions for grain yield was increased considerably when using the subset of lines with low GEI and subsets of markers associated with grain yield in either environment. The highest prediction accuracy for grain yield between environments (<em>r</em> = 0.37) was found using the subset of 400 lines with low GEI and marker subset 6 (362 markers significant at <em>P</em> &lt; 0.01 and with marker effects whose absolute value was &gt;30 kg hectare<sup>-1</sup>, <a id="xref-table-wrap-1-14" class="xref-table" href="#T1">Table 1</a>) and was greater than the phenotypic correlation of GYW and GYN (<em>r</em> = 0.20). The majority (99%) of the markers in subset 6 are not in linkage disequilibrium (LD) with one another suggesting these markers are likely mostly tagging independent QTL.</p><p id="p-57">As with grain yield, the between-environment prediction accuracies for FHB, SE, and FY were lower than when using data from both years to build the model and it varied depending on which year’s data were used to build the model. For these traits using a subset of lines with low GEI had little impact on improving accuracy (<a id="xref-table-wrap-1-15" class="xref-table" href="#T1">Table 1</a>). Using subsets of markers did improve the between-environment accuracy especially when using 2010 data to build the model to predict 2011 FY or SE phenotypes and using 2011 data to predict 2010 FHB phenotypes (<a id="xref-table-wrap-1-16" class="xref-table" href="#T1">Table 1</a>). Thus, as for grain yield, the ability of the data from one environment to predict the other, and the impact of using subsets of TP data, varied by which environment’s data were used to build the prediction model. For every prediction scenario the greatest GS accuracy was attained by using either a subset of lines and/or a subset of markers (<a id="xref-table-wrap-1-17" class="xref-table" href="#T1">Table 1</a>). The greatest GS accuracy between environments for FHB (<em>r</em> = 0.42) was greater than the correlation of FHB phenotypes between environments (<em>r</em> = 0.32). However, for SE the highest accuracy between environments was lower than the correlation of phenotypes between environments (<em>r</em> = 0.81) and for FY was equal to the correlation between environments (<em>r</em> = 0.72).</p><p id="p-58">For grain yield, removing lines with high GEI likely allowed the model to more accurately estimate genetic effects and GEBVs. A line’s mean phenotype within an environment is due in part to its genetic effect and in part to the GEI of that line with the environment: the calculated GEI is due in part to true GEI but also in part to experimental error that may be confounded with that line and that environment. It is interesting that the accuracy of GYA and GYW decreased (relative to using all TP data) when using a subset of lines, but increased when using subsets of markers. In contrast, the GS accuracy for GYN and the between-environment GS accuracies only increased when using both the subset of lines and subsets of markers. This could be due to the error (noise) present in the phenotypic data for GYN, likely caused by the water damage to these plots. Most of the 70 lines with high GEI variance that were removed had negative GEI with the GYN environment and positive GEI with Wooster. This likely led to underestimating the true yield potential of these lines in the GYN environments while data from the Wooster environments were more indicative of their true genetic value. Including these high GEI lines in any analysis adds noise to the GYN data set and that reduces our ability to predict GYN, or the ability of GYN data to predict grain yield elsewhere. Thus GS accuracy is low for GYN and between-environment predictions when using data from all TP lines even with the reduced marker subsets. This is not a problem when using GYW or GYA data (GYW is a component of GYA) as the GYW phenotype data likely more accurately reflected true genetic values than did the GYN phenotypic data. Prediction accuracy for GYA and GYW increased using just subsets of markers as the markers were selected for significance for GYW or GYN, which means they were discerning genetic effects from noise.</p><p id="p-59">The GS accuracy and RE<sub>y</sub> observed for these traits in the TP indicate that GS could be very useful for improving these traits. This conclusion is further supported by the ability of the TP data and GS model to predict the performance of the parents of the TP. For all traits, the accuracy of GS can be greatly increased by using subsets of markers that are significant for the trait. The increase can be even greater when also using a subset of lines with low GEI for some traits. Increases of GS accuracy within the TP using cross-validation of up to 131% were attained by using data subsets. Even using marker subsets containing &lt;7% of the total markers produced greater accuracies than using all markers. The increase in accuracy from using subsets of TP lines and/or markers is similar to what other studies have shown one would expect by increasing the TP size and number of markers (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>; <a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Heffner <em>et al.</em> 2011a</a>,<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">b</a>; <a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Lorenz <em>et al.</em> 2012</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Poland <em>et al.</em> 2012</a>). Perhaps even more important is the finding that predicting the performance of lines in one set of environments by using only TP data from a second set of independent environments was significantly improved by using subsets of lines and markers.</p><p id="p-60">It is now relatively easy and economical to generate lots of marker data using genotyping-by-sequence (GBS). However, our results suggest that a small subset of selected markers can provide more accurate GS predictions than a large number of markers. It is important to note than one must have a large number of markers and TP lines to begin with to have accurate estimates of genetic effects and genome coverage. From the large set of markers you can systematically select lines and markers to form a small and more informative subset that produces the greatest accuracy for use when implementing GS. These small marker subsets may allow the use of alternative genotyping platforms that are less expensive and more repeatable than GBS in future cycles of GS initiated from the TP. In plant breeding it is difficult to add more lines to a TP to increase the accuracy of GS as this greatly increases phenotyping resources. The TP size must allow for efficient phenotyping over multiple environments in order to accurately estimate GEI and marker effects. This study removed 5 to 17% of the TP lines based on GEI: the remaining lines must form a TP of sufficient size to provide robust estimates of genetic effects. These results suggest that starting with a large number of lines and markers in the TP allows one to select for the most informative ones to use in generating GS models with improved accuracy that can then be used when implementing GS.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-61">The authors would like to thank Dr. David Francis for helping with the statistical analysis and Dr. Pierce Paul for FHB inoculations. Salaries and research support were provided in part by State and Federal funds appropriated to the Ohio Agricultural Research and Development Center, The Ohio State University. This material is based upon work supported by the National Institute of Food and Agriculture, U.S. Department of Agriculture (USDA), under agreement no. MIN-13-G19, and the Triticeae Coordinated Agricultural Project (2011-68002-30029) of the USDA National Institute of Food and Agriculture. This research complies with the current laws of the United States. The authors of this study declare that there is no conflict of interest for this study.</p><p id="p-62">Author contributions: A.H. collected and analyzed data and wrote the manuscript. A.C. collected samples and extracted DNA and reviewed the manuscript. M.H. contributed to data analysis and manuscript review. C.S. supervised the study and edited the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-63">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032532/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032532/-/DC1</a></p></li><li class="fn" id="fn-10"><p id="p-64"><em>Communicating editor: M. Warburton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-06-15"><span class="received-label">Received </span>June 15, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-07-13"><span class="accepted-label">Accepted </span>July 13, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Hoffstetter <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.1" data-doi="10.1111/jbg.12075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdollahi-Arpanahi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nejati-Javaremi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pakdel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moradi-Shahrbabak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Effect of allele frequencies, effect sizes and number of markers on prediction of quantitative traits in chickens.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D131%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12075%26rft_id%253Dinfo%253Apmid%252F24397350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24397350&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.2" data-doi="10.3835/plantgenome2011.02.0007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy and training population design for genomic selection on quantitative traits in elite North American oats.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.02.0007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.02.0007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.3" data-doi="10.2135/cropsci2008.08.0452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomewide selection for rapid introgression of exotic germplasm in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D49%26rft.issue%253D2%26rft.spage%253D419%26rft.epage%253D425%26rft.atitle%253DGenomewide%2BSelection%2Bfor%2BRapid%2BIntrogression%2Bof%2BExotic%2BGermplasm%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000264654000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.4" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genomewide selection for quantitative traits in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.5" data-doi="10.2135/cropsci2013.09.0601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabrera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttieri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sturbaum</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffstetter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic diversity, linkage disequilibrium, and genome evolution in soft winter wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">2433</span>–<span class="cit-lpage">2448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D2433%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.09.0601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.09.0601&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.6" data-doi="10.1007/s00122-015-2580-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabrera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttieri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sturbaum</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of milling and baking quality QTL in multiple soft wheat mapping populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">2227</span>–<span class="cit-lpage">2242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D2227%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2580-3%26rft_id%253Dinfo%253Apmid%252F26188588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2580-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26188588&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Combs</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accuracy of genomewide selection for different traits with constant population size, heritability, and number of markers.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">6</span>:.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.8" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">G. l.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI5MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.9" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.10" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R Package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.11" data-doi="10.1111/j.1439-0388.2007.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DGenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00702.x%26rft_id%253Dinfo%253Apmid%252F18076469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076469&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.12" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> Genomic selection for crop improvement. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.13" data-doi="10.2135/cropsci2011.05.0253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic Selection Accuracy for Grain Quality Traits in Biparental Wheat Populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2597</span>–<span class="cit-lpage">2606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D2597%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.05.0253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.05.0253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000295839200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.14" data-doi="10.3835/plantgenome.2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome.2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome.2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.15" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.16" data-doi="10.2135/cropsci2015.06.0332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffstetter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabrera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sneller</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identifying quantitative trait loci for economic traits in an elite soft red winter wheat population.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">547</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D547%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.06.0332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.06.0332&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabrera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sneller</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic selection for wheat traits and trait stability</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> (<span class="cit-comment">in press</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.18" data-doi="10.1007/s00122-014-2418-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isidro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Training set optimization under population structure in genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2418-4%26rft_id%253Dinfo%253Apmid%252F25367380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2418-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25367380&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.19" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo3OiI5LzIvMTY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI5MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.20" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjkxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lado</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matus</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inostroza</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Increased genomic prediction accuracy in wheat breeding through spatial adjustment of field trial data</span>. <abbr class="cit-jnl-abbrev">Genes Genomes Genetics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2105</span>–<span class="cit-lpage">2114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomes%2BGenetics%26rft.volume%253D3%26rft.spage%253D2105%26rft_id%253Dinfo%253Apmid%252F24082033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24082033&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liaw</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiener</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Classification and regression by randomForest.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">18</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D2%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.23" data-doi="10.1007/s00122-007-0577-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abate</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Musket</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">QTL associated with Fusarium head blight resistance in the soft red winter wheat Ernie.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLiu%26rft.auinit1%253DS.%26rft.volume%253D115%26rft.issue%253D3%26rft.spage%253D417%26rft.epage%253D427%26rft.atitle%253DQTL%2Bassociated%2Bwith%2BFusarium%2Bhead%2Bblight%2Bresistance%2Bin%2Bthe%2Bsoft%2Bred%2Bwinter%2Bwheat%2BErnie.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0577-2%26rft_id%253Dinfo%253Apmid%252F17569028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0577-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17569028&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248137500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.24" data-doi="10.2135/cropsci2012.03.0144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christopher</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffey</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundrum</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Molecular characterization of resistance to Fusarium Head Blight in U.S. soft red winter wheat breeding line VA00W–38.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">2283</span>–<span class="cit-lpage">2292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D2283%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.03.0144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.03.0144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.25" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and optimization of genomic selection for Fusarium Head Blight resistance in six-row barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjkxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.27" data-doi="10.1186/1297-9686-42-37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accuracy of direct genomic values in Holstein bulls and cows using subsets of SNPs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMoser%26rft.auinit1%253DG.%26rft.volume%253D42%26rft.spage%253D37%26rft.epage%253D37%26rft.atitle%253DAccuracy%2Bof%2Bdirect%2Bgenomic%2Bvalues%2Bin%2BHolstein%2Bbulls%2Band%2Bcows%2Busing%2Bsubsets%2Bof%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-37%26rft_id%253Dinfo%253Apmid%252F20950478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20950478&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.28" data-doi="10.3835/plantgenome2010.04.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic-enabled prediction based on molecular markers and pedigree using the bayesian linear regression package in R.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D3%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.04.0005%26rft_id%253Dinfo%253Apmid%252F21566722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.04.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21566722&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.29" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.30" data-doi="10.3835/plantgenome2012.02.0001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown-Guedira</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evaluation of genomic prediction methods for Fusarium Head blight resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.02.0001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.02.0001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection for quantitative adult plant stem rust resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D7%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.32" data-doi="10.3835/plantgenome2014.05.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sallam</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessing genomic selection prediction accuracy in a dynamic barley breeding population.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">: .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.05.0020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.33" data-doi="10.1111/pbr.12105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordillo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karaman</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection allowing for marker-by-environment interactions.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D132%26rft.spage%253D532%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.34" data-doi="10.2135/cropsci2010.04.0193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttieri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoots</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification and validation of QTL for grain quality traits in a cross of soft wheat cultivars Pioneer Brand 25R26 and Foster.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1424</span>–<span class="cit-lpage">1436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D1424%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2010.04.0193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2010.04.0193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.35" data-doi="10.3168/jds.2010-3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predictive ability of subsets of single nucleotide polymorphic markers with and without parent average in US Holsteins.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5942</span>–<span class="cit-lpage">5949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVazquez%26rft.auinit1%253DA.%2BI.%26rft.volume%253D93%26rft.issue%253D12%26rft.spage%253D5942%26rft.epage%253D5949%26rft.atitle%253DPredictive%2Bability%2Bof%2Bsubsets%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bwith%2Band%2Bwithout%2Bparent%2Baverage%2Bin%2BUS%2BHolsteins.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3335%26rft_id%253Dinfo%253Apmid%252F21094768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21094768&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2919.36" data-doi="10.3168/jds.2009-2092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Recio</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predictive ability of direct genomic values for lifetime net merit of Holstein sires using selected subsets of single nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">5248</span>–<span class="cit-lpage">5257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DWeigel%26rft.auinit1%253DK.%2BA.%26rft.volume%253D92%26rft.issue%253D10%26rft.spage%253D5248%26rft.epage%253D5257%26rft.atitle%253DPredictive%2Bability%2Bof%2Bdirect%2Bgenomic%2Bvalues%2Bfor%2Blifetime%2Bnet%2Bmerit%2Bof%2BHolstein%2Bsires%2Busing%2Bselected%2Bsubsets%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bmarkers%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2092%26rft_id%253Dinfo%253Apmid%252F19762843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19762843&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/9/2919.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2909" title="From Genotype to Phenotype: Nonsense Variants in SLC13A1 Are Associated with Decreased Serum Sulfate and Increased Serum Aminotransferases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2929" title="Genome-Wide Association Mapping for Female Infertility in Inbred Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/9"><span>Volume 6 Issue 9, September 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/9.cover-source.jpg?itok=nund1hRf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2919" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KcvdQvH86cVzEnM-WzqlHkxhjZxL8BkYIREwEjOm-jQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/9/2919" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/9/2919" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/9/2919" class="active" data-icon-position="" data-hide-link-title="0">Optimizing Training Population Data and Validation of Genomic Selection for Economic Traits in Soft Winter Wheat</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460292" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-OAljBxoKNhju4Du9rhT4QIFzqwCon66KKjzF_WYeiDE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2919.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/9/2919" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-rVcoB9HN5-qKCbi0iGqjOkJsECbsFcF_JxUijn7ItG0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/9/2919" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2919.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460292" id="node460292--2" data-pisa="ggg;6/9/2919" data-pisa-master="ggg;g3.116.032532" data-apath="/ggg/6/9/2919.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Training Population Data and Validation of Genomic Selection for Economic Traits in Soft Winter Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amber Hoffstetter</span>, <span class="highwire-citation-author" data-delta="1">Antonio Cabrera</span>, <span class="highwire-citation-author" data-delta="2">Mao Huang</span> and <span class="highwire-citation-author" data-delta="3">Clay Sneller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2919-2928;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032532 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460292--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amber Hoffstetter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amber%2BHoffstetter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffstetter%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmber%2BHoffstetter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Antonio Cabrera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BCabrera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabrera%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonio%2BCabrera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mao Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mao%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMao%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Clay Sneller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clay%2BSneller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sneller%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClay%2BSneller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sneller.5@osu.edu" class="" data-icon-position="" data-hide-link-title="0">sneller.5@osu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/9/2919" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/9/2919&amp;title=Optimizing%20Training%20Population%20Data%20and%20Validation%20of%20Genomic%20Selection%20for%20Economic%20Traits%20in%20Soft%20Winter%20Wheat" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/9/2919&amp;title=Optimizing%20Training%20Population%20Data%20and%20Validation%20of%20Genomic%20Selection%20for%20Economic%20Traits%20in%20Soft%20Winter%20Wheat" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/9/2919&amp;title=Optimizing%20Training%20Population%20Data%20and%20Validation%20of%20Genomic%20Selection%20for%20Economic%20Traits%20in%20Soft%20Winter%20Wheat" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/9/2919&amp;text=Optimizing%20Training%20Population%20Data%20and%20Validation%20of%20Genomic%20Selection%20for%20Economic%20Traits%20in%20Soft%20Winter%20Wheat" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/9/2919&amp;title=Optimizing%20Training%20Population%20Data%20and%20Validation%20of%20Genomic%20Selection%20for%20Economic%20Traits%20in%20Soft%20Winter%20Wheat" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/9/2919&amp;t=Optimizing%20Training%20Population%20Data%20and%20Validation%20of%20Genomic%20Selection%20for%20Economic%20Traits%20in%20Soft%20Winter%20Wheat" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/9/2919&amp;title=Optimizing%20Training%20Population%20Data%20and%20Validation%20of%20Genomic%20Selection%20for%20Economic%20Traits%20in%20Soft%20Winter%20Wheat" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/9/2919&amp;title=Optimizing%20Training%20Population%20Data%20and%20Validation%20of%20Genomic%20Selection%20for%20Economic%20Traits%20in%20Soft%20Winter%20Wheat" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460292/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460292" id="node460292--3" data-pisa="ggg;6/9/2919" data-pisa-master="ggg;g3.116.032532" data-apath="/ggg/6/9/2919.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Training Population Data and Validation of Genomic Selection for Economic Traits in Soft Winter Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amber Hoffstetter</span>, <span class="highwire-citation-author" data-delta="1">Antonio Cabrera</span>, <span class="highwire-citation-author" data-delta="2">Mao Huang</span> and <span class="highwire-citation-author" data-delta="3">Clay Sneller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2919-2928;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032532 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460292--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amber Hoffstetter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amber%2BHoffstetter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffstetter%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmber%2BHoffstetter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Antonio Cabrera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BCabrera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabrera%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonio%2BCabrera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mao Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mao%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMao%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Clay Sneller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Horticulture and Crop Science, The Ohio State University Ohio Agriculture Research and Development Center, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clay%2BSneller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sneller%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClay%2BSneller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sneller.5@osu.edu" class="" data-icon-position="" data-hide-link-title="0">sneller.5@osu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2919&amp;rft.epage=2928&amp;rft.atitle=Optimizing%20Training%20Population%20Data%20and%20Validation%20of%20Genomic%20Selection%20for%20Economic%20Traits%20in%20Soft%20Winter%20Wheat&amp;rft.volume=6&amp;rft.issue=9&amp;rft.date=2016-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hoffstetter%2C+Amber&amp;rft.au=Cabrera%2C+Antonio&amp;rft.au=Huang%2C+Mao&amp;rft.au=Sneller%2C+Clay"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460292/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460292/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460292/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460292/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460292/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460292/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460292/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460292/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460292/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460292/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460292/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460292/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/9/2919" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/9/2919.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/9/2919.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/9/2919.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/9/2919.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
