<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/9/2693" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/9/2693.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/9/2693.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/9/2693.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/9/2693.atom" />
<meta name="HW.pisa" content="ggg;6/9/2693" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Signatures of North American Soybean Improvement Inform Diversity Enrichment Strategies and Clarify the Impact of Hybridization" />
<meta name="DC.Identifier" content="10.1534/g3.116.029215" />
<meta name="DC.Date" content="2016-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Vaughn and Li. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Crop improvement represents a long-running experiment in artificial selection on a complex trait, namely yield. How such selection relates to natural populations is unclear, but the analysis of domesticated populations could offer insights into the relative role of selection, drift, and recombination in all species facing major shifts in selective regimes. Because of the extreme autogamy exhibited by soybean ( Glycine max ), many “immortalized” genotypes of elite varieties spanning the last century have been preserved and characterized using ∼50,000 single nucleotide polymorphic (SNP) markers. Also due to autogamy, the history of North American soybean breeding can be roughly divided into pre- and posthybridization eras, allowing for direct interrogation of the role of recombination in improvement and selection. Here, we report on genome-wide characterization of the structure and history of North American soybean populations and the signature of selection in these populations. Supporting previous work, we find that maturity defines population structure. Though the diversity of North American ancestors is comparable to available landraces, prehybridization line selections resulted in a clonal structure that dominated early breeding and explains many of the reductions in diversity found in the initial generations of soybean hybridization. The rate of allele frequency change does not deviate sharply from neutral expectation, yet some regions bare hallmarks of strong selection, suggesting a highly variable range of selection strengths biased toward weak effects. We also discuss the importance of haplotypes as units of analysis when complex traits fall under novel selection regimes." />
<meta name="DC.Contributor" content="Justin N. Vaughn" />
<meta name="DC.Contributor" content="Zenglu Li" />
<meta name="article:published_time" content="2016-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomic Signatures of North American Soybean Improvement Inform Diversity Enrichment Strategies and Clarify the Impact of Hybridization" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Crop improvement represents a long-running experiment in artificial selection on a complex trait, namely yield. How such selection relates to natural populations is unclear, but the analysis of domesticated populations could offer insights into the relative role of selection, drift, and recombination in all species facing major shifts in selective regimes. Because of the extreme autogamy exhibited by soybean (&lt;i&gt;Glycine max&lt;/i&gt;), many “immortalized” genotypes of elite varieties spanning the last century have been preserved and characterized using ∼50,000 single nucleotide polymorphic (SNP) markers. Also due to autogamy, the history of North American soybean breeding can be roughly divided into pre- and posthybridization eras, allowing for direct interrogation of the role of recombination in improvement and selection. Here, we report on genome-wide characterization of the structure and history of North American soybean populations and the signature of selection in these populations. Supporting previous work, we find that maturity defines population structure. Though the diversity of North American ancestors is comparable to available landraces, prehybridization line selections resulted in a clonal structure that dominated early breeding and explains many of the reductions in diversity found in the initial generations of soybean hybridization. The rate of allele frequency change does not deviate sharply from neutral expectation, yet some regions bare hallmarks of strong selection, suggesting a highly variable range of selection strengths biased toward weak effects. We also discuss the importance of haplotypes as units of analysis when complex traits fall under novel selection regimes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/09/01" />
<meta name="citation_mjid" content="ggg;6/9/2693" />
<meta name="citation_id" content="6/9/2693" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/9/2693" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/9/2693.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/9/2693.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/9/2693.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.029215" />
<meta name="citation_pmid" content="27402364" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2693" />
<meta name="citation_lastpage" content="2705" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Justin N. Vaughn" />
<meta name="citation_author_institution" content="Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_institution" content="Department of Crop and Soil Science, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Zenglu Li" />
<meta name="citation_author_institution" content="Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_institution" content="Department of Crop and Soil Science, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_email" content="zli@uga.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. M. Akey;citation_author=G. Zhang;citation_author=K. Zhang;citation_author=L. Jin;citation_author=M. D. Shriver;citation_title=Interrogating a High-Density SNP Map for Signatures of Natural Selection;citation_pages=1805-1814;citation_volume=12;citation_year=2002;citation_issue=12;citation_pmid=12466284;citation_doi=10.1101/gr.631202" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=C. Bank;citation_author=G. B. Ewing;citation_author=A. Ferrer-Admettla;citation_author=M. Foll;citation_author=J. D. Jensen;citation_title=Thinking too positive? Revisiting current methods of population genetic selection inference.;citation_pages=540-546;citation_volume=30;citation_year=2014;citation_pmid=25438719;citation_doi=10.1016/j.tig.2014.09.010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Beissinger;citation_author=C. N. Hirsch;citation_author=B. Vaillancourt;citation_author=S. Deshpande;citation_author=K. Barry;citation_title=A genome-wide scan for evidence of selection in a maize population under long-term artificial selection for ear number.;citation_volume=196;citation_year=2014;citation_pmid=24381334;citation_doi=10.1534/genetics.113.160655" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. P. Bollback;citation_author=T. L. York;citation_author=R. Nielsen;citation_title=Estimation of 2Nes From Temporal Allele Frequency Data;citation_pages=497-502;citation_volume=179;citation_year=2008;citation_issue=1;citation_pmid=18493066;citation_doi=10.1534/genetics.107.085019" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_title=Biological Resources and Migration;citation_title=Biogeography, local adaptation, Vavilov, and genetic diversity in soybean;citation_author=T. Carter;citation_author=T. Hymowitz;citation_author=R. Nelson;citation_pages=47-59;citation_year=2004a" />
<meta name="citation_reference" content="citation_title=Soybeans: Improvement, Production, and Uses;citation_title=Genetic diversity in soybean;citation_author=T. E. Carter;citation_author=R. L. Nelson;citation_author=C. H. Sneller;citation_author=Z. Cui;citation_pages=303-416;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. F. Doebley;citation_author=B. S. Gaut;citation_author=B. D. Smith;citation_title=The molecular genetics of crop domestication.;citation_pages=1309-1321;citation_volume=127;citation_year=2006;citation_issue=7;citation_pmid=17190597;citation_doi=10.1016/j.cell.2006.12.006" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. Falconer;citation_author=T. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. F. Feder;citation_author=S. Kryazhimskiy;citation_author=J. B. Plotkin;citation_title=Identifying signatures of selection in genetic time series.;citation_volume=196;citation_year=2014;citation_pmid=24318534;citation_doi=10.1534/genetics.113.158220" />
<meta name="citation_reference" content="citation_title=The Genetical Theory of Natural Selection: A Complete Variorum Edition;citation_author=R. A. Fisher;citation_year=1930" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. Foll;citation_author=H. Shim;citation_author=J. D. Jensen;citation_title=WFABC: a Wright–Fisher ABC-based approach for inferring effective population sizes and selection coefficients from time-sampled data.;citation_pages=87-98;citation_volume=15;citation_year=2015;citation_pmid=24834845;citation_doi=10.1111/1755-0998.12280" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=Z. Gizlice;citation_author=T. E. Carter;citation_author=J. W. Burton;citation_title=Genetic Diversity in North American Soybean: I. Multivariate Analysis of Founding Stock and Relation to Coefficient of Parentage;citation_pages=614-620;citation_volume=33;citation_year=1993;citation_issue=3;citation_doi=10.2135/cropsci1993.0011183X003300030038x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=Z. Gizlice;citation_author=T. Carter;citation_author=J. Burton;citation_title=Genetic Base for North American Public Soybean Cultivars Released between 1947 and 1988;citation_pages=1143-1151;citation_volume=34;citation_year=1994;citation_issue=5;citation_doi=10.2135/cropsci1994.0011183X003400050001x" />
<meta name="citation_reference" content="citation_title=Population Genetics;citation_author=M. Hamilton;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=Y. Han;citation_author=X. Zhao;citation_author=D. Liu;citation_author=Y. Li;citation_author=D. A. Lightfoot;citation_title=Domestication footprints anchor genomic regions of agronomic importance in soybeans.;citation_pages=871-884;citation_volume=209;citation_year=2016;citation_pmid=26479264;citation_doi=10.1111/nph.13626" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. N. Hirsch;citation_author=S. A. Flint-Garcia;citation_author=T. M. Beissinger;citation_author=S. R. Eichten;citation_author=S. Deshpande;citation_title=Insights into the effects of long-term artificial selection on seed size in maize.;citation_volume=198;citation_year=2014;citation_pmid=25037958;citation_doi=10.1534/genetics.114.167155" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T.-Y. J. Hui;citation_author=A. Burt;citation_title=Estimating effective population size from temporally spaced samples with a novel, efficient maximum-likelihood algorithm.;citation_volume=200;citation_year=2015;citation_pmid=25747459;citation_doi=10.1534/genetics.115.174904" />
<meta name="citation_reference" content="citation_journal_title=Econ. Bot.;citation_author=T. Hymowitz;citation_author=J. R. Harlan;citation_title=Introduction of soybean to North America by Samuel Bowen in 1765.;citation_pages=371-379;citation_volume=37;citation_year=1983;citation_doi=10.1007/BF02904196" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. L. Hyten;citation_author=Q. Song;citation_author=Y. Zhu;citation_author=I.-Y. Choi;citation_author=R. L. Nelson;citation_title=Impacts of genetic bottlenecks on soybean genome diversity.;citation_volume=103;citation_year=2006;citation_pmid=17068128;citation_doi=10.1073/pnas.0604379103" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=I. Imai;citation_author=J. A. Kimball;citation_author=B. Conway;citation_author=K. M. Yeater;citation_author=S. R. McCouch;citation_title=Validation of yield-enhancing quantitative trait loci from a low-yielding wild ancestor of rice.;citation_pages=101-120;citation_volume=32;citation_year=2013;citation_doi=10.1007/s11032-013-9855-7" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=J.-L. Jannink;citation_author=M. C. Bink;citation_author=R. C. Jansen;citation_title=Using complex plant pedigrees to map valuable genes.;citation_pages=337-342;citation_volume=6;citation_year=2001;citation_issue=8;citation_pmid=11495765;citation_doi=10.1016/S1360-1385(01)02017-9" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. Jiang;citation_author=H. Nan;citation_author=Y. Gao;citation_author=L. Tang;citation_author=Y. Yue;citation_title=Allelic combinations of soybean maturity loci E1, E2, E3 and E4 result in diversity of maturity and adaptation to different latitudes.;citation_pages=e106042;citation_volume=9;citation_year=2014;citation_pmid=25162675;citation_doi=10.1371/journal.pone.0106042" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=M. J. Kovach;citation_author=S. R. McCouch;citation_title=Leveraging natural diversity: back through the bottleneck.;citation_pages=193-200;citation_volume=11;citation_year=2008;citation_issue=2;citation_pmid=18313975;citation_doi=10.1016/j.pbi.2007.12.006" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H.-M. Lam;citation_author=X. Xu;citation_author=X. Liu;citation_author=W. Chen;citation_author=G. Yang;citation_title=Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection.;citation_pages=1053-1059;citation_volume=42;citation_year=2010;citation_issue=12;citation_pmid=21076406;citation_doi=10.1038/ng.715" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Lewontin;citation_author=J. Krakauer;citation_title=DISTRIBUTION OF GENE FREQUENCY AS A TEST OF THE THEORY OF THE SELECTIVE NEUTRALITY OF POLYMORPHISMS;citation_pages=175-195;citation_volume=74;citation_year=1973;citation_issue=1;citation_pmid=4711903" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. Lorenzen;citation_author=S.-F. Lin;citation_author=R. Shoemaker;citation_title=Soybean pedigree analysis using map-based molecular markers: recombination during cultivar development.;citation_pages=1251-1260;citation_volume=93;citation_year=1996;citation_pmid=24162537;citation_doi=10.1007/s001220050363" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. Maynard-Smith;citation_author=J. Haigh;citation_title=The hitch-hiking effect of a favourable gene.;citation_pages=23-35;citation_volume=23;citation_year=1974;citation_issue=1;citation_pmid=4407212;citation_doi=10.1017/S0016672300014634" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=R. S. Meyer;citation_author=M. D. Purugganan;citation_title=Evolution of crop species: genetics of domestication and diversification.;citation_pages=840-852;citation_volume=14;citation_year=2013;citation_pmid=24240513;citation_doi=10.1038/nrg3605" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=P. L. Morrell;citation_author=E. S. Buckler;citation_author=J. Ross-Ibarra;citation_title=Crop genomics: advances and applications.;citation_pages=85-96;citation_volume=13;citation_year=2012" />
<meta name="citation_reference" content="citation_title=Soybean: Improvement, Production, and Uses;citation_title=Origin, US history and development, and world distribution;citation_author=A. Probst;citation_author=R. Judd;citation_pages=1-15;citation_year=1973" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Ross-Ibarra;citation_author=P. L. Morrell;citation_author=B. S. Gaut;citation_title=Plant domestication, a unique opportunity to identify the genetic basis of adaptation.;citation_volume=104;citation_year=2007;citation_pmid=17494757;citation_doi=10.1073/pnas.0700643104" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=D. E. Reich;citation_author=J. M. Higgins;citation_author=H. Z. Levine;citation_author=D. J. Richter;citation_title=Detecting recent positive selection in the human genome from haplotype structure.;citation_pages=832-837;citation_volume=419;citation_year=2002;citation_issue=6909;citation_pmid=12397357;citation_doi=10.1038/nature01140" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schmutz;citation_author=S. B. Cannon;citation_author=J. Schlueter;citation_author=J. Ma;citation_author=T. Mitros;citation_title=Genome sequence of the palaeopolyploid soybean.;citation_pages=178-183;citation_volume=463;citation_year=2010;citation_issue=7278;citation_pmid=20075913;citation_doi=10.1038/nature08670" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=H. Sonah;citation_author=L. O’Donoughue;citation_author=E. Cober;citation_author=I. Rajcan;citation_author=F. Belzile;citation_title=Identification of loci governing eight agronomic traits using a GBS-GWAS approach and validation by QTL mapping in soya bean.;citation_pages=211-221;citation_volume=13;citation_year=2015;citation_pmid=25213593;citation_doi=10.1111/pbi.12249" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Q. Song;citation_author=D. L. Hyten;citation_author=G. Jia;citation_author=C. V. Quigley;citation_author=E. W. Fickus;citation_title=Development and evaluation of SoySNP50K, a high-density genotyping array for soybean.;citation_pages=e54985;citation_volume=8;citation_year=2013;citation_pmid=23372807;citation_doi=10.1371/journal.pone.0054985" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=Y. Tsubokura;citation_author=S. Watanabe;citation_author=Z. Xia;citation_author=H. Kanamori;citation_author=H. Yamagata;citation_title=Natural variation in the genes responsible for maturity loci E1, E2, E3 and E4 in soybean.;citation_volume=113;citation_year=2014;citation_pmid=24284817;citation_doi=10.1093/aob/mct269" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. van Heerwaarden;citation_author=M. B. Hufford;citation_author=J. Ross-Ibarra;citation_title=Historical genomics of North American maize.;citation_volume=109;citation_year=2012;citation_pmid=22802642;citation_doi=10.1073/pnas.1209275109" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. N. Vaughn;citation_author=R. L. Nelson;citation_author=Q. Song;citation_author=P. B. Cregan;citation_author=Z. Li;citation_title=The genetic architecture of seed composition in soybean is refined by genome-wide association scans across multiple populations.;citation_volume=4;citation_year=2014;citation_pmid=25246241;citation_doi=10.1534/g3.114.013433" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=J. J. Vitti;citation_author=S. R. Grossman;citation_author=P. C. Sabeti;citation_title=Detecting natural selection in genomic data.;citation_pages=97-120;citation_volume=47;citation_year=2013;citation_pmid=24274750;citation_doi=10.1146/annurev-genet-111212-133526" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Z. Wen;citation_author=J. F. Boyse;citation_author=Q. Song;citation_author=P. B. Cregan;citation_author=D. Wang;citation_title=Genomic consequences of selection and genome-wide association mapping in soybean.;citation_pages=671;citation_volume=16;citation_year=2015;citation_pmid=26334313;citation_doi=10.1186/s12864-015-1872-y" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=S. Zhao;citation_author=F. Zheng;citation_author=W. He;citation_author=H. Wu;citation_author=S. Pan;citation_title=Impacts of nucleotide fixation during soybean domestication and improvement.;citation_pages=81;citation_volume=15;citation_year=2015;citation_pmid=25849896;citation_doi=10.1186/s12870-015-0463-z" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=Z. Zhou;citation_author=Y. Jiang;citation_author=Z. Wang;citation_author=Z. Gou;citation_author=J. Lyu;citation_title=Resequencing 302 wild and cultivated accessions identifies genes related to domestication and improvement in soybean.;citation_pages=408-414;citation_volume=33;citation_year=2015;citation_pmid=25643055;citation_doi=10.1038/nbt.3096" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Signatures of North American Soybean Improvement Inform Diversity Enrichment Strategies and Clarify the Impact of Hybridization" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
<meta name="twitter:description" content="Crop improvement represents a long-running experiment in artificial selection on a complex trait, namely yield. How such selection relates to natural populations is unclear, but the analysis of domesticated populations could offer insights into the relative role of selection, drift, and recombination in all species facing major shifts in selective regimes. Because of the extreme autogamy exhibited by soybean ( Glycine max ), many “immortalized” genotypes of elite varieties spanning the last century have been preserved and characterized using ∼50,000 single nucleotide polymorphic (SNP) markers. Also due to autogamy, the history of North American soybean breeding can be roughly divided into pre- and posthybridization eras, allowing for direct interrogation of the role of recombination in improvement and selection. Here, we report on genome-wide characterization of the structure and history of North American soybean populations and the signature of selection in these populations. Supporting previous work, we find that maturity defines population structure. Though the diversity of North American ancestors is comparable to available landraces, prehybridization line selections resulted in a clonal structure that dominated early breeding and explains many of the reductions in diversity found in the initial generations of soybean hybridization. The rate of allele frequency change does not deviate sharply from neutral expectation, yet some regions bare hallmarks of strong selection, suggesting a highly variable range of selection strengths biased toward weak effects. We also discuss the importance of haplotypes as units of analysis when complex traits fall under novel selection regimes." />
<meta name="og-title" property="og:title" content="Genomic Signatures of North American Soybean Improvement Inform Diversity Enrichment Strategies and Clarify the Impact of Hybridization" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/9/2693" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Crop improvement represents a long-running experiment in artificial selection on a complex trait, namely yield. How such selection relates to natural populations is unclear, but the analysis of domesticated populations could offer insights into the relative role of selection, drift, and recombination in all species facing major shifts in selective regimes. Because of the extreme autogamy exhibited by soybean ( Glycine max ), many “immortalized” genotypes of elite varieties spanning the last century have been preserved and characterized using ∼50,000 single nucleotide polymorphic (SNP) markers. Also due to autogamy, the history of North American soybean breeding can be roughly divided into pre- and posthybridization eras, allowing for direct interrogation of the role of recombination in improvement and selection. Here, we report on genome-wide characterization of the structure and history of North American soybean populations and the signature of selection in these populations. Supporting previous work, we find that maturity defines population structure. Though the diversity of North American ancestors is comparable to available landraces, prehybridization line selections resulted in a clonal structure that dominated early breeding and explains many of the reductions in diversity found in the initial generations of soybean hybridization. The rate of allele frequency change does not deviate sharply from neutral expectation, yet some regions bare hallmarks of strong selection, suggesting a highly variable range of selection strengths biased toward weak effects. We also discuss the importance of haplotypes as units of analysis when complex traits fall under novel selection regimes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/9.cover-source.jpg" />
    <title>Genomic Signatures of North American Soybean Improvement Inform Diversity Enrichment Strategies and Clarify the Impact of Hybridization | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__34Arr8QetTzezDWW4Etyv7gbgH-x-ZTNo3yO15Q-gHU__Mk-lyyG5WoK4SRy56eUq5eomxN7OGXjpDv-Ob1a1Q1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"tbkKFccjiG2YsodEPq6se_5BS1T2YzsuLNRaWHr7dOY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460291","apath":"\/ggg\/6\/9\/2693.atom","pisa":"ggg;6\/9\/2693","ac":{"\/ggg\/6\/9\/2693.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/9\/2693.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2693"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/9\/2693"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/9\/2693.abstract":true,"\/content\/6\/9\/2693":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227O2rn9pb7Wurz1KixdffC9\\\\\\\/m6JJhGSQ\\\\\\\/sMCe8+zseRIR1kLAnJM7fD\\\\\\\/7gCuWg\\\\\\\/1JzwEneyibGNNSJxPGdmwqErICgL8cwlHjpdsbF0WomWWyq798bQU26lNwSuLQgrzgEjFtZuFlWD1V0U4rxuSuLUcpnqIC\\\\\\\/Oh6DM7yNjy\\\\\\\/d+69v47L9u9tYMBPzB1+upLh94HSI1vaVV9UySrhJZTvJZ+9zsn8rrLyQ6U7SM2SCki1vharV0hYaBbwMvQkwoFXyRYxOdbnS66fGI9cBLIksc7RXQty0\\\\\\\/6O7+1TyqfPXycly7hGzwppac38Ns5iqpnsVrllCsvhUe9J99SjMkNnamPTHxh25+UQjEzF6t7QHBRxcHysRTGrDy1lj2prjCa8+0+reamgCH+2q1gp6JBwSCAWR3cwATA5JtrfbNT7itaDN3d9nG6IQmLb4w3eBRlDvgiLOQCkSn0jb94yMF5YjvwHKSo3tyXRFOyhDvt6SvXMDwWLMnkpuhV0UjcIGS5c4rImcX9eqOew9Xd0RX1AhkHUE7bGzVu8YSDrnClzNrNDee5kMCjUMVn5rJrD5sdQ\\\\\\\/YgNLP8JvJJXipsR2O6NVT1rS7nbf5wJebXTFBljLomCIVQLTOBBh+P0h578B7lTZYhHgMFm1snGxyr56VtsnLnybom847UiHvWAZ5JrGlVGVQ6A\\\\\\\/XEilbhDa3nmayFwce8LCLiMkM8zKxKlB2TkNGy+cIcQJFtdELoBRfLqI3DRrOSGnns7wmhyJ5pyxwsfdBFsrG4OqpZLFpcgxsuN\\\\\\\/ybfldsfvm2jgykMZkxkI5eGa2sKYrKap76\\\\\\\/OI\\\\\\\/Q5CzkYzP9u6i+fb54ZO5VK0lx3IU3F5PGpLzmGab6JEag3JNXWbrXvSyM0GvPD+qVDpBalf\\\\\\\/RhuIGXP8VatJg5SJeT707Oaj929zxJVE71SRXniRFtW+59pZoHU\\\\\\\/RngrLhHp2SyTQuDCmZVUKboP6UCeqYQ\\\\\\\/iQu8dsly+hXDAbybdgTohTdPvdw2zkGH13CtZSMfBa8b85uAufG0NL1M5vPXLzwMXMVFOACko\\\\\\\/dkk6Le6aw1MEIooQzi+HjJZXcyl20PrJnAI77iChmiDqIConIRvb5kcFZIcChGkddRO\\\\\\\/ACotpoBiLm3QB5HhHV0do0WKYFH3hxgJ+JnIgrTH+iyRiYbKYk+GlTf1ZMMKOtt\\\\\\\/h6G2XSEA+zsttB2TMrEZrwUDGepAUFs+eCdHMk+gxp9wNbC3arx53ebfO5gnSLACuCcMOUNzZHWn8OUs4N4SqoAP6nx\\\u0022,\\\u0022iv\\\u0022:\\\u0022BTnBidhOKGVlMkAM+Jw6xA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226bdb7e45dc5b3a5668d33ca77ad7c83a963a13abc99416b4d4b8dd32a5ba0d1f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Fle8kAh2DelSr1zJ3+oXqpaQb4e8wgzrvqCPPADM9esOYKYBm64HnPKUJqQE1p\\\\\\\/3sSzliWGVxQkAh4fUtFZumuAVk4yjOGfucYy7TSBEXiZHP1pWYXURohU4AW9pODLtkNhUxfzSbIuj3BLCeFmRTwJzK5zsvuvU8CduBDnuJ3IkNW87nuJUWyG2koN5HK6bI8sYzRCYheAw4Bf1+vOci2Zp2dtNF9aC4mnS2P3JRC+B8wDBXRfZi7CasO2\\\\\\\/sd8R24AMqxbFcjXd5hhkF+r14ZpAFw3rsRjD7e7Y0wsplAWddqT5RkMEgZKfl3Sjvrv+wRKcN2kMrppY6KH0\\\\\\\/L2cyLbhRlawzzpJGxjLivmeU3di9tQfnw3z5Qlt5xkX7+jnDsARVeTWVEFiQenukDuZUhm8jMzYMF8k9r3KFDgjo1aLZva8rHl\\\\\\\/91dA41REpjMc4ovqQFaG3xU+Az7\\\\\\\/0dgR2UbvqYQ+xytph5+Cb893+aynPHSSXlixxwHBLL29gGZKVM9PlvSfpVffBdWqAMZkvhod1vf8RbzjCSzjkNtj7p+4fuKU3t\\\\\\\/2mnRQhgju5ej83KvZ\\\\\\\/UyBKX3uQUMXn2m1YwaeYOOQvPTnFYR80iQNyoCKu6ibVIsUqIM9l2R0bJoORRNbGFAR+y55mO7cLviI\\\\\\\/hHMsKEwqfdD3FtKAQ\\\\\\\/HZisoX1yyBEDEz6TS09V6q2d9P++XCBDQCiONaBItALsGXjWjO7nGps8cvc6o3MkC78dkyMpW9FSSVfBTLnhYaTN9I62vYylDdQJj9klOY9uY4mJOiS+6Wnvumq81ItNOBu\\\\\\\/oT6x\\\\\\\/1tHuILGOC6QpdUdgWd2RtDf+rRpl2y4xD9Bx94tQWhgtQWw6vEtCV35TIXMnfkVhX7UHbAfKLgiDg9IhksMzOgN0lAI7nG89RH3MmJSOl2jDhaT+KVzmprWbqHrn9g1IXWBqe2SjdbNQ\\\\\\\/FiKn8cFVvYFPY+VEWHNA\\\\\\\/Z2MPs6441IDn0ktm2A1PA\\\\\\\/8AiSWr9dsqhfI1XaE\\\\\\\/va9hJYMsYDZFIePnceZiUdmml8VidavnJj3M47hYqDGTxLMznP\\\\\\\/uWWcgRR+V5fuS8OSuNMhAwaoibNOG2O7sqoxpTWzAfEnDpQkW5jiTT8zJsiMTyOdyOnRnL\\\\\\\/KzNjMYmPYTyGMrLfmQlgj+mGXXAqnsX+yh6aj+TMxkR10cq1BcPxgUvjMB3lzSkLo7Gq36CuSnkPOwWvJLjnNBiFOuUoWjNAGOHseoKG0yCDyBLm1OxNTLNzd5xvVskye3OMT6X9KzPhAmzbfwMk+qNmP4siAGkAQL6uF451hSQJxvflIk32b2rN6+RqPTCTVljLAxHjQLYleGozzpA+7LMPCFxRjcuYBOD9KIbaIBMpFT1jYfl9aH53HhZV4IAyQicSrLoJKPJrDk150JJe4m2SCE832F\\\\\\\/jwS3pG5cJXEoZQYUaAglQGxxi4og9ZZPxtFxAIKPU0evph6Wx+PuzTOeef3LXQGFcFMMtahMUe+ToR+zWqSQIlBMi4HW77uvZSOC7R4JpRgBzJdnHWiytb9J+l\\\\\\\/yicFDeeIH9t7iuxMV5pdpZ7AtrrBJsKAm8PqKCJA94ihhTWNaytM7Kkba2usSaBJBH3qMDlTRnzfbjD1KcYSA4L6C4AqSR4c1qqgAhENXJlsrr\\\\\\\/8Awi1UApxFZZiT5kwRMycKQUB4hqrzPunNXuLWktdQzqlOFG9zE5EXMjMNcwm8x9Dfv7vtwPSNAxD2mlHpxnEHW7caVwqGUzZhpHCW2\\\\\\\/pqz0LC8P0WjcpYUmaRfBr0bIwtlhYJwwvwQIqNOKmhp1T+LxUMOLuCdEqY3\\\\\\\/DAU2cJ+22p4KXoLIzZGchUapeQpEvKXk9s373DaN5l4tq42ikmi23i5p4GfBo3o+oOcazLUAVFj1skgOJYWlxcOcgI6LzclSxbsADhwfmerJfzWwXupESzAZ0cuvLh+g8IKLrb8S7ZZZZFTU1hIW0DmEwTJpAwbNdjiRUyJCxWAs9XsfAO3NrEWvzx6t53J47pFTdmNU5igJ4FFoUAyQ2zZFNIyNkM6Zoz\\\\\\\/8KVV5SNulEtdsC5g\\\\\\\/9V54xVmHCzAVSOXDNOCnzGRH3h\\\\\\\/Y7rel7wUTXNxg+N8R+V6j7ePinxmKACMLa06dSMSSuqKlEG\\\\\\\/suAoMHDAhcPTQ6mxEBsewL2J4wPLVTrq5czCJUc9Telys8wG7RQuC+ss9mOC13d1LXxl2ishleddrBYCEwgiSPeNtdJMVuqvPglx7VNGXYVrYnd0lwu+9f+IqkYesYRXj4gnCoijvmn5p2PB46Q0XvmtO+vBVIOHEGQtOW+n0tHNX7mBTX68vq0VivPz8g8Mlj2Wx+JauiZJ8NmFfOrB0p9CnQ0qbQn9JjP7oMipqmKYhnA0qdpsH3ykHv5demjywjd\\\\\\\/705M\\\\\\\/WYVJDAUlRSSW9srspSNsivY1yNS2s3D+eRFVyb+y6GUeUiwwhnuSly8bhkEgJ\\\\\\\/+GdcP7ywR31coNDjXSsM3nHnBHE2e1LiE8jNpl7F2+CkesSEethUNktZw\\\\\\\/r\\\\\\\/FpwgJ8PJ9bO9Brx7ofy2mfBTNUNGaCyQ7XFyYX2LOn9SU5R\\\\\\\/zG4J9TULFg+9MvFwsF5eSM5lZST2gBZPWQ+QaXPUM9VaGw4NtJnJ8EUyWpTFAqrdrEF2m46qiQdb3u01p+EHzyzGaJ2DDA8YcfTMyfKNs2cMP6vjAJ\\\\\\\/USoPKBrb2DdmcWiW22yMwK2mZ8swcGgdORSAKlUfnRgMMpZouE\\\\\\\/2tD\\\\\\\/qx8sSUd+o7kCYPUZgzqjnS+ZXL22fhqSGxrcpLv9Velxjn2j68aabFtR4VfdWlBjKMSslQDxlqhuTZZQ3JCexcB3Wp8JWAsWT9lxaE0jmW8AXudGYyRGUdG+ED5wOgy7pg3ZR495tgXilOH5zYWgnbO44A\\\\\\\/jJAPOa2Llgl4\\\\\\\/3TMTBUD88NxYwcuAGHYUJiQhavmsxTvzeeYkEQpBGYyRRvXqtmdlqfKf+WaGlBzq76GnGX95ANfa4WxpbK5bfbvl9r1wh3EddcGckFybM\\\\\\\/PUO6hDBx\\\\\\\/+Qx+R8sPuh4ZwH4bCpOSwikAE\\\\\\\/jr+IHIdzxNclbyl7jraL4UhLQ6b43oNr950thQVr6x3rlrXZlu54z+Y1n27BiyJQQs+Zpndu5ML3OW8h0Q1Zo1NwVHl4qDeBTDJOUmlgOcS84ocEeli96hknrg2usy3HHM+XhAbHsb\\\\\\\/LjSNGR34yf5zDWtrVlB6bs5QlXd22eg3M3GkygKzcenJJUdw1n3IdJk2w4p8naFrWNPOgbfAOJWxjFa8LXbk9fKEUh9ERYPD4okINTDMOhg885qBbcP+RJKoHtFwlrr85hOSg6kGdprDIONpXkndLpeH3CFFRSSuH\\\\\\\/LJ6RGeG7F8\\\\\\\/LSWY0q7OfqrymLVZj196k7l6gevgBFpy5XYPCrGQ7S2PTPd\\\\\\\/b\\\\\\\/ERY5vmpdUjL21PRrfsPZKEke\\\\\\\/\\\\\\\/KsQ827Kg1Guq2CkMtSrMt3ug2kPmsjjn6br8d\\\\\\\/FATX\\\\\\\/Il+AT1BzB4Uv1igXNnfagn0fIG16DQvS1dlGbC8E4NGAEMsvWZNm9JhCcKYFijKkVGeH6V8dU0Qr+Dvhmp\\\\\\\/UNuaYSgGzfmb2A6YEGD3bScwTVO7wfVUrggYdNA\\\\\\\/gp1cj2Yy3ZJrnWw4kpt33uG0XpJlbr5FIOOyJ3iaX6DKVXnRUnNxByjWTmwKmXTbFqKA5HOph3NUhUU7XwQYuQGHuMGAya+ie1tkJsykchnSJ4fCC8Hrej+CaXDL+yBU7xRLUQe3f9EoLs1YGP7a0IfjA988dvELfnUAFbbvlNOhwXKbK++QXm1pChWeS4yjPbR2qjhyfNyj8mDfmCRvugRrZ8J0noHoUnOAnTsWjlagdV4CQ\\\\\\\/5MWqYr2E74ZxDrmQmVdsNicnA8AS3rAZzC5ow9fn0f8RKXZ\\\\\\\/tcTpdKLLbNEyhHKFsbf+f3chbma\\\\\\\/k785AwYzr5hgkz5hUSqZ3+Pd3ikDYg836xgoAE2Jc3lsPUfIscwh\\\\\\\/X+EWuBAt9i0xRjWDWWVBtsG9An359uIfimqhM1VNJ7hScljpwsRU2ICaNNdDzSg9RyWkuUW3jndGEKxBu6FNpZNdxNqv9Frobb85oy8qtqbcGcYalmL+JOCxOFtPpkIt9HVjK+hUcksYTGBsSb3tb3gnPxowPnXK5UXdqRlXcoPDpqiHAI57W92n\\\\\\\/Uh9feXin\\\\\\\/mIX\\\\\\\/f6d9KoS725FKzZAGleC7ffH\\\\\\\/RJojBHrXGOCKTxImJuDsYs7KFQmnyDM6zHzpfeFIOW4Br+JpioOJxgUVq7A4T9aMaqmiBkekUh+x9dgbTP4tT7ctxXEeMFkr0rXR0qSqB5KNB4lK9UKsrkTVJKCBC8wN8FGLexJ7ODjBvMa+ktYFVhJy3zFgcLUS55Ehy3yPB16zBsT7oOyHFWrGC9Fbq3qY1dDZKuSikz1D2IwePkFnOKzDeQhY72KmhBcZALqIWKfUDvMpEuonQApWrz1SEudMci7WzE4pFTxpgFyolTeUNt2gxOQOq2uDG0huuBPI5q8INIV6CJwlPim0ZTBCAN8SaiEc5xxfM28O6hVWrpvBi2tjiT6GHe4\\\\\\\/hqjcJp3TVKtioudallgTP7CXWD3zyuH2kqq7T0Mm6yhw2tBxbC7XbqgsGac5UyWn5oD2WfJQUNfVXO\\\\\\\/1GE0ZU3zrV7yBpk+qT8ySD6ojDxJIqAWnRXaGqkFl4+YkEamVFWkaG8Bz0dU9YVNy1M+8GeF0KgCZL5J\\\\\\\/YGp\\\\\\\/V4bllGiTKXbGiKdF3yYNGCeqABOqjkGku1UHA9mj5FspT9zjFFKzvg7FzWRwA9pEJqxRZ6Unf3yuzLXT3NU+CLCdfeXbdzGb11K3lrqIaK6e\\\\\\\/TsUM+w3iuhUMDhzfitaqK\\\\\\\/7d\\\\\\\/sImOJLzH+NxvxVyiLMBPg1vM15WNndKhHvdN0\\\\\\\/wyCfEK2BsLxP2HSKfOK+97N4PE6dw5CTDbfpr0bPhhxtUJ4UuxE1tX1dzEHx+db7V0VMipVcI\\\\\\\/e94fiXOs3LhstwIr+y+Mm\\\\\\\/p0vhstHqGkIITj8NMchwA6TD0yFgPGAkyXc\\\\\\\/vNSXToQrpc5uRSp5Y8RHk4toqko9N3mV+A\\\\\\\/9T3\\\\\\\/pZ9pxdKZtjPhBG0icEp5T5ZF2+BXpdwnbiQq42vNH7uXThRQB2tWt8YiRzUSstNnB3JT\\\\\\\/CTXe8o76grfTy0keW5inChp11X34m4UGRaVfuIBUjVebYha7Fl6FGw0rhbtdCb2vsJAue6sI6mV13oPUHwYjbvnhPvJ4iRwfXhUzSkA88wi2Y8f9cfPtTjmTpTpTQKLosLbZlj5A\\\\\\\/rVUB\\\\\\\/zmcxNWVy5fpAVIEjOSKIjLO6YT5BBviPTqoe9uNRpO1RO23rHqsI+ECGTW583KXVrRm\\\\\\\/ccOM5EHc2i5lNsERQXXBLmFclFJlME+1XeAIwwTx6fOAfPiiRw8v5SvZtWbujtShVD0gYYdZITec7cMuYQRGsuLMmkG4NQ3SCp629CdPmVvPcSthgHKVkuRaMNUGyex19rUvlb33qwJ2\\\\\\\/4Eh5Kua\\\\\\\/+LS3U\\\\\\\/qaEFRU9+Q5pMLfzHkW8uejEBoGig7JBENKprIEITIaPu3ycxZWGaz\\\\\\\/JN5OoB8KNofSay8xbzrTNxsjpaMtCSROFXRsRxGiCVMI808fy\\\\\\\/trWlnqmrJ09MjuCfg0RhwOxinkk+EEkqXAnA2qXQqtU7sEch1UpqxAn1RrldR6K8qSqCfBlpPT7tyLpgzNPIvkQhAGcsvsS+gdvg9Fv\\\\\\\/qk10Z0xAtCbwYv0rTgd4qdGm13KXes\\\\\\\/BGnxbCo9896MVrKqt7On5gAGT1QNJsrQM8kipSOzZUT4X7QobkTGJP6I\\\\\\\/bEEpN\\\\\\\/M\\\\\\\/y58QmargDwPAAiq+QS\\\\\\\/ZOjNgJXFP3hun7A+Ray1OV3b0lDWvBywGxPcpDbG9fDry3yw5SQnX6a9LTt2gFHyT697ybnAO7a9Agg45w5B84dk\\\\\\\/xrHyCAHMtA+RLXnudjkuAoo3bwO5A0WRs2PkxzyPHYF23cUFrXCM\\\\\\\/J179MpkiJILc7TQl0BSiGwhAOpSUaoiz2Ce9jKNFnQ2lzr5qc1s4dP18N\\\\\\\/QYojR1V89AqAk9SbYX9vvpmysdS0fA5704hu20G3cVTUvDSxDl6PuEWjuJsBDmq5pAmGJcsxS9QIbMZgA2KF6WKIIxbEn8cMpjcz0bNUzucnJrHYso1xgMGlNqGFkhJ7gaE+dXhZo7ocvoKa5nyam\\\\\\\/PfuMVdyPNBSa+546i9yFYBeMeJ+ZfsXzBkllBHOCOHpsOKRccSJXp43Vdh1mwlLzLojb\\\\\\\/ue3cFhT4nklK7JCd7qMwEztJ\\\\\\\/Ybyi173kieRT73UF3N65Rev7BA2OXfA8Q\\\\\\\/9eTMoXINMR+mWyHUPLueJ8BOjA1\\\\\\\/rY9k0RgWrOUiIii+p17KheUU0Xe2sSf0DqH0MB+R8bPq28eD5mRALMahHjUSIZDc1emQ90NGGOD+UoZ7Znw3Zzd2QDg0Dyb+pU89xT1qSNIqfd8uBpaUaS6IdUON1LOMOb+XEu84R5q6VxiHMCZS+ayXVziAR7VrAlH2LnC1DvoF780KzgKnpXpX9joHide99QSvfY2XXZQMTAWRZGI76J8hteA5kJxD+odZPdIkEswMHI7IowFGRi\\\\\\\/IezDth6TG\\\\\\\/nbtiFBvdWXZse2z5oOpOkGHdmgYBB2HHIcXSWvU+25n8rSHpE3rpYaIDFmQrjMc1stR9edIuv\\\\\\\/WridpX2tnBABQOHUns30IFAzMX482nVri13kTk\\\\\\\/x30tBaijC4tG7npYoRoSNGwwQ+57c+l7zMwgk6ZkoyZJql88K5D7mpxWezLDeit2Y20F1M1m\\\\\\\/LfHifQh0tLb52gSjblOgv4mPlTDJ\\\\\\\/NaoRZ1GrfLVG8HTQGXWLBdV2a3ZSY6uDQ2ZT7+ADRQYmiqU0fZE7VrFkRefu86mhNNLvNh1npuDS5co4\\\\\\\/waQgD+qR46lb+669M7THrsWGiOxXUnsr87SFtAf\\\\\\\/VtlrbTOsrCZNTmH5ez47dyl2ftl+PLpo8PI5XHQAscl0L+PbRce1uXzevrrTcRu6wMLqy\\\\\\\/8ZrJpzXDVXUV97d3DBF\\\\\\\/KEzdFyu+wzqqJpmWQQYoKv39WpMH7IC0ZWt3kr\\\\\\\/HSREOrhFvjReMQhRU79KFvBxQKUsyMFohDBzNWHBRQldz+\\\\\\\/Xqx5sL2xR599Ma7sXdQ17g6j4S2HZ7yga620z1WaaGOnO+s8eHGHGRYSMeIFNqG7veGWBvO0X+VJXVw31jwokIgkvCtW4xZH6eukN5Qq7fVHNKoHvKPHYCx6fI3PZgnYuF6lx7bSLWP+fuknflZEp7sLJypEuYS\\\\\\\/lswLVd4FyeemzjMNbRSLkxZMdWMO2q1de52O9gv2w3Bg4JWvccH4s4JhKPDasPv11YzaiHDhQ8x+i0L0wv60mVRryFlOLCte+S3aRmNmJEI8aeZByYuii06QIJjpMdy065BiRgfsodUw6PSM3X9nAwcyRJd8+FC25Sc6fEaYKSyBVYhtn6EmluwlgPyXjSrjFkfYJ5DBHajlgN3OIO129sUuCK5SH51vbpvv3JminQFR9h3tUoiAe7dlmb9Vgt0gneKE38LwU2hsns6upKT6mSy6jJ26Qjt2WmAqhJIFf7JkM9NhVOfqBsJVG6d8dJZUsQS9+jjx039mYBe972n5cu\\\\\\\/+JBHqBZ\\\\\\\/9k5kCAbv5lhNElIEyhS4fCUmsd6h2OyjQ6lP24C2qviW+r8xoyC15F84prtylkk\\\\\\\/UNwZwxIve6I6V0PL0hbAwDZ09ZZ\\\\\\\/Z7gSUp5aC0fABIMzrHJclhRVxey\\\\\\\/6kwp\\\\\\\/k\\\\\\\/JJgA1lErJeXHnzSCeuRCtvrjJMNmZhW2l+hG548mXzuF\\\\\\\/XLM0FWge\\\\\\\/qGbCAat1cXfteiPdOSNStiGp+3S6zLK\\\\\\\/v4\\\\\\\/n3wgHZJsKtd1DyF2gn+HNXidaFcvbBHvzk+iFqrk9Kk5djE8oi1ZwYW1nvgnIuLknTZOB\\\\\\\/CqwAWIfHVXdFRTX3ww2MnaUZu5KLH422yfIba9X6Z630xqEj3wIi90nt\\\\\\\/t256Kh4os1Tpi3n01rkc+mV6kObcaGpzj6P6986jzySSGl1UJdg\\\\\\\/Cz1CqP58GGg3\\\\\\\/tlOs3Q7sLwTDwVoFQJKgDCffVsAfU7l2b+VF6F\\\\\\\/DcUTkqJNIAjj781KROmrJ4\\\\\\\/cXWf1yJTcanYDi6KHUWh7A\\\\\\\/UkpzDjytO2K0\\\\\\\/a6DqeJZGZg7IPbnTeG0xVlwm4EZHChFfW\\\\\\\/nJr40t+fHvyc62KFL+aVrm0C4MB4i6m8TwVqHg0eG1l25jPod6WD8mfi4xkiwxXBy5Vbamsq2P+ZXt5FKMw9GTBz4DskmkszbY+1zvn0g8fjTKySiOQoYECKhRjRRwo3F2TvB9bhqnjxTWRUtbe3NZmFTBZHYc6Rx70K930fLLXrZS1vDC9kUWHtZXAXuxc0y+0r5OXB3DFRm2L477K7QD2WEuq3f48cS7fiLp4KIb\\\\\\\/+3dljc+lm\\\\\\\/tAYfaaeylFt9j55nSVoSHSmeL8qy6aaaG0bMp8vA8bnzCA714iwWymDBHx4x7nsIdYjX6tKTAiO219WxEpVIjQkR3pcZzkd8PTjQ2+YXoLoK3jrGJfra9osW1Rte6k7jqz2An44NZO0McwUKnsStcBju2L1Ygb4R49zQ3ODM2q4A08vk+RGhrOph7CyDiP1dEBvT5e2m5u4i3t05gtNZfZjwLQLRKVSbsn6MYLiDlP0Ed7MsC95itYMID9nuIGh8tjl8AhAMrof74x600PaULx54aZMWTqQEbYB\\\\\\\/VBwMoFNw+fXw7sXBpzI1jM6CPzNFu+zFeen65w0Pzwq6c+ZAv7SEO1QY0swt+sYFOHr0uz+1k42snEn81zlYg33fLteDIKxW0bAPmCDhuZCXAi4IRJ3eXbagYg3U+Nf198P0IwT2UpQqwXYpH5zvwX3mFZRire9uz27v+Zyd3VvkztyUL5Ug6FcrRpRWvnYTkDTu8yZrXgoNV5ZmPZYvlft9AgyCNqfeLGR\\\\\\\/RUqWmcvI6uCG2AjdQiL7DzdcGCX2\\\\\\\/sgDuwYNsSJjLUh8EAw8AHFU2GTuFhg3nqSKN5UwWMpU6Uju9zoHeXd7OwMZxK2vk1dm+aTi+mlhsQ5hKPdGTmoVqeO\\\\\\\/X9x27haMwBVGDsILRF7w\\\\\\\/eM4eFCgCe0tC51HFqGaFo91mUomqcrnMZTwmOp3vMTKwDXaMT3KvaP5vNmYHZD2u8qK2pKwmdINMxZcPRv42Oer1v\\\\\\\/XmSJ\\\\\\\/8e1llJpoboDAqBwntN0irOCY2pnCBqkzAj0XVXza9k3SuoLMoAqnNaJC3\\\\\\\/cBhGKJNzPLUPAURUD2073jhPm\\\\\\\/Ip8Ny5BPSW974JOoF4C1axWTlEVFNcEqkNgC7MGpsx7w+J3tLxQG\\\\\\\/HXK6vFoCmfE9YZMKeRGx8VcObJ5exv\\\\\\\/Rno9tjlV9Zzywpwvs6kbTrsS73Jj3BjLFdpg2GgMqOWhIsOwbgzYJlLnusQdhju6N2LF8uhLVwlAZy+qVZfFiVnhqvzmv\\\\\\\/3D3SfFrq7pFjjuvZ5tVni2wnZsCqXz4CTVRP08WO3FGCiR1qKq0RqhzRQVxesMf5TGl3sQaWECDBjZkNP7VYMDPN6axJpdUacQjomHZprw6qMYaWM2wIyboqsSQ8t\\\\\\\/S7ABCx9J06vtCinMviFVRw9GL54SgXp+LFmpi7F+ckt4iyVHG2AULOkc1qydM43PS788RYWuQbsrSGSU6UJOHPcUKYFxn5+1h\\\\\\\/ssOL7KvokgdZSiUInpadRgaVcY0tYPTvkawDrA4ZDwL4LalnpKUSsu7vLB5NslaflMbFFNJtTf7xS14nPAyq+FRxRW\\\\\\\/79MFZGnTipOraq\\\\\\\/BAnuT99ic8uNcMLehQICNrzkBW4WVjsZfD6nIdMreEUk4zP3EFHzUqlhv3pzEp+Ewsho8QpdxrzR52ApLPbQPwhsSehzn9\\\\\\\/SgV3FjbpZFbFlqKBVEhC0h0jcUfNuYwoAC3D3T+j3PWG4IuJP\\\\\\\/bCdVyrEHhe9DMAksyfyROnTPdigncZvvgUXg16p14aj99z4PcbAML9LzKTY\\\\\\\/yTEaYqFonGDnnDr2k30iGboq1WNYOf56fMB4JoEUntHCLg9HqDBNb2k\\\\\\\/Nbb\\\\\\\/vV+XUlBdWhMy6Lzk2kRmiTsgRZbBGIEc803JL7lPR5aklQOp9FYaNy7REaSChhC7lAxsOwWDc2eSznodT+kSgvt+lVm3hAHOyqzTk4ysVDQTw3z3X7yikYDEx5RsfLhGZv7UIK91IiT0Iv5c+UVsQZcVonppNRVrfdXfQnYZoEfDMTOm56g3rD3bDBcThXSYN8YU\\\\\\\/9FQ+ERVTDNrsh8oTQoIf9A75+0DjEr12h+tgSHZIIYQN1taHnIYmhkiVxSCXr2nNyZ+tql8Bnu6bLBqIwPCYA0zvXkeD4NKcsaSm67MN7dDCrQHrgvxg0EYLvp\\\\\\\/uJd\\\\\\\/gqWmIRp2xL0JfOsx6r8l\\\\\\\/VE6r8UihIUvAsjBeV2wRO9tMH24LWHBewqRAmaq2PhfiAtPB0ChxeooIVIg5wpMmNMPVsYW0TsmLLCk630rdZWon782ZyJqaiNgcA7A7DrfU3gWihlERO8eqKp\\\\\\\/Fao7\\\\\\\/6JV7us9TabR5VMhNL0zECkaTJy\\\\\\\/sIreKSoEro7rHVnsPcgX+C9HILO5AIJj35N0eSsrtWhGCuPMxyMpasohr77p\\\\\\\/fhoEGwJvoNR9KWDbtJbL2eevrAiRnIWXSKpm7umNeaij8Id7T3wyMWvdDNifJLEksrdb6IYieng47xLhR74jEKpBvIcWtAITTNfEhoug6R5fX9qIxZAL8qA3Y3szvbHs4ZZd19V9cemZ5SSWiBHmBljmhGexE5H4ZmfpAQYapv59t60L68AfD3LMolsGihXpOblw2FM+XA6kp19Gp91EfbsvxsQVLjJoFK0T32LkakOYTcjlRuxh33CJBTTwlSPCilWb1BlO2G\\\\\\\/nchdtOwE1skAZlhG9Wam7WfZ0xWN+ZJtuoObTp5PaUALQGFFHKZT3Dfxx5C5QjidXpYbbKbgZ\\\\\\\/7wBZUmnTRm7YBVI75oskK0\\\\\\\/Kz0xbSdbOOAdYInUoAmAgLNvcEyfKRXTCtFIcppR8\\\\\\\/81zIYZp\\\\\\\/5j8Yu2TZl7YRP9tXmrH4LZkQ03bMGnc5ttNRlxvOF5RIwAXYZ+3bdxz40UDPzXOQPCHNEIjifyBV0wqwBssgrVMjYXKKpCsQFGGOxwXg8MoVyAaSeqNAxNQOWtU0NO0hNIiAv\\\\\\\/SD3sm5noqt\\\\\\\/DV4QmK\\\\\\\/cwwGGieeB+Xszk0AgwieliPhMqQvIgGofKLjjuIIJeoY34dW78t+wZVWca6dokke21ck4SwxpmXoLLKjvb+rwcgiiKioYuoSgWWBA8nN1J5Fj+rteujg7xInyBBhO+efFhxA1bgRGCDjGTCXeEWJdtV2R11raYZ82MgU2UB5AbgNhpTOSljI1XdzgxGrDjdNpKBH7LWI\\\\\\\/5y7aqu7gOz6Bzs7khUdU5xt9EODxJdLuzsd2o20S\\\\\\\/uoWax0ldmuUjQsUljEiZUTpaDGcuUPeTQw0ohdWNsp3+EKR7OaGc4soTRVKer2xYyma4gUiESNProL5ElB+cq2y7GPGUfCFT7lczTtwg4fYRTYQprKoOqRcDdmoSwhJhi34sylGgqflyY0eBgx0jalKQ8AwRGnRJogaM3xDnYJPqb+zMNQw1a2GvltxAKhNs5n7DveA4JwVtkp+aIu44psJpfWB0Km+Wx9LIoFgk9DoalCBCRPnte94f8AIvQB6GEfQnlH2W6L2MatvX3t7FU2kWOZPZEgFEtn9Hkq3NuB+c4w7iB\\\\\\\/+WL1qGNjbyXop4ZQs\\\\\\\/JNq2seIVaBBPdEt\\\\\\\/Q1GNm8UAISWWowEwpWCQ4iyRR4dFoxG8XWGPl8wJZe9NwyNDkRm2AKS3QzrkdnqaAdvmsoQHBDdrZqJ8mIoTFz2+xyVPCWbPSJ\\\\\\\/WF1J7lUz66VUDt3J6l\\\\\\\/yI+6WTk\\\\\\\/p1Xc3tVeqWeGR\\\\\\\/HeAqkbqO25ev2eHGXhB\\\\\\\/ZjOsBDhfSdvFza5Wo8s39OgPunnACfHsiw\\\\\\\/NNc6GcnGDt8QJ6tWopiTp7P+olozFuY1DbJzDLk6LteoGY\\\\\\\/3Q0gMdZS6LnzdQuFU5gZQzyToqs9yihg4e8hOgnanik+v1vFsRAPMLLpMnM5437UqLJUJj3Q8uRl10hAfXsJ+o+8JXCt+8WN+TetjTztcEXi8gIfIW4ecpn\\\\\\\/i6kdxw914M0\\\\\\\/RoDNaewrfHr69359J3pND+GvtmzDW4fgl\\\\\\\/DzqttAyZ7Ir\\\\\\\/4YiL9kJLFSK662kuv1OBx5GxLNN27XyoR5YFjvYOCdgH4eL+H\\\\\\\/u9HtaY68iH8KUmg67gXcLjSZo9+Mhxtlf0KBPHL5ZVQ20nx6nps5yKjO+xvxVRlIcRpi07vr\\\\\\\/uL4hJ9Vc3+uH93hsElizyNdF3l2\\\\\\\/OwDI8QtjtItWMAky5LPfMcnpABB8\\\\\\\/4KeE5piWZzEJB3wPHtnUKKQBrAU0iSbrQiXS90ny0hTiVoy7B8UEwiCfmCFyngQiCjR\\\\\\\/zi\\\\\\\/RLI7IfvE2POtfffsdvqNtsuJl4wOwLQR5LCiFWXFvbiSt7ZbJvmj943ZChC12JYlzEc7g8aVSut4E9uI4metBUdBFg8xvhYfvuCGSJA8SQUiJm1Z\\\\\\\/D5yuRmt0SdAF0R3mY4C1RhOmC6zxzmih3cq\\\\\\\/EIQ6bCvbQXt7gTKvSZI1FgPnRb+Naq8MvBxOmrpJywATJB5UAaBTkYVwFmkxWP9QV2dY0ZX+avZGPpX6RJzqEfdGbZtki4VTc8umTzrnHVxTjaoMDuUpEG9MV306ECnWr1YUbvOtZMQT4QS2FXolrTt1zlkR61x\\\\\\\/QZYpHAsJHTpvhzELITJoRS3KCGl0jebkOpDDiQTHQzJnE84DmjSqYfqkMOdhIvOl3J8pAyYRF6cdqfI2jmbQB0BtBh8VbXJC32b7KaAtFPdSGZqX2ddtmAwMB\\\\\\\/9J6EEF3rRPoVI8SbrZVPucveP8NHK+hBWkaC1kE8pFG+6JGLsGCi5TCzE2MF1XFJePUVp7uYRrg1N+D6FXVpDTxVffqWQZTO4hKIqYEqfbFezyFb51CzDYJTv2FFM15IokBSUEBNebmmm4vxF2mOS1+IktWdV5ZLaKVa2gkiX3\\\\\\\/nU57pH5AmI4skcsQRoDBN6gGlVLK8vrbZrBBVDlt2Rju1Kijsqz1+LH2LXPSawz3pPBd2nvq\\\\\\\/BcXMdR74\\\\\\\/FA35sBb8WJUSObjW958REEyH4gvmFcD+9NNe5rINX2RKBqFW0xZ\\\\\\\/Fnzmfzrrds4ZWeFvdTP1c897CQZgldZJydCq7+S6Qg8yW0R1tOpShDnwaKlM6\\\\\\\/zR+saI8aLWN5HlCU43uJewM2o4Vy9sAndgCgAOUA94p5iEgNhPyBGbYmBo5M1lucFD1BR+dOup8YsNcq7qT2Uje7VR5nrg5lK5q6s4vRjcaO8kBOCjJeqDwYClT4QBba06wsl0yNnr7+dJfj9blxpJpnFA50LszsvyUIl0EtJN24ZaHYlz4okPzWDFbJd8qJj\\\\\\\/jgjxazng7Pq+wXHwK2y+yFNFitQuL5r2XEKbVgjjHXARMrdfsLlSn9BHQRb2e2ZHJNJlsIrP8\\\\\\\/\\\\\\\/2uyBopmJI5zh6\\\\\\\/6U1qwziKM\\\\\\\/4z+y0U38bJHiwLfiGefr8Cu0vp+RkWR9WOB1KqG+kuKCVi6+P8YTkEgPW4IgLo1vkd2Vl0bpu3Hznfg++5oyIhWIN0E8hm7AQk5N+5Di9ayBTKMoefBzmo8UztPFwLywMXZ7tmT0VTQ3CrM4+WHCzjdzblIOnlV8v6gHyMx0AOqvDHAfrbV0yUisukjG8rLixzYXQDtzJXlwLhZoq5jalxirCkN2pibshZ3Zp2fl92ZyDr4LjEmzTRGqQuq3J\\\\\\\/nQ0JGADn8SJw883B+4ZrA4hXTvG3C+BRK77xRZDS1wvA3pyy7vEG\\\\\\\/55aGvJsAL4jRCodWC7a+J2CyA28l3DoC+MDXfRXsOXj2EeyhbNMB+2Xo57BZXq51taX0j+e\\\\\\\/vLutYJm3WhPxvtiLysRQURELqS275n6FYM7AdQHwd+o1PKFbvcqDYiMkW1RgX4U6kxhtC8pxDq\\\\\\\/0u4qgFZk\\\\\\\/iKSXPmRfMr\\\\\\\/a91U2ND4d0NmD\\\\\\\/JCNO8ypUJ8pyXF\\\\\\\/Km11dKqx9adAQ\\\\\\\/NJobMWpj6dEqHacmZinICuClz01eseh\\\\\\\/w5sD5xdpKMsxYjz9a204UOcXj1GyGIJC5jBmThIPDN+1dLN7O5RHD+virVxKyL1F9WGsRUIW+4MeDfE4i\\\\\\\/d6Vk3C0ex2m1LzkkDeXLWqb4\\\\\\\/txDdSMKzYpAU++yn4cVPklqCkc969+c7pYPJwkFTLD6TVbpkKUDQJyIe4GmINLuZQbNGUZwBR62Zage015kUCnsWFLsYlJXoOAths\\\\\\\/7R+xaya8k0F+fdajiXhPYc7VORzSYooppIq8TOYfOdElV5MaIDwSWOdWlZuXcSy+Bk4hXGs8F3av29G4EoIAOD8rOfS28IqKB3WqSvFiKpUjqclfi+oDdZXa7V55mP8X\\\\\\\/EjgoIsIVQNCWyig70Yr+iidvEys\\\\\\\/Y3meI8tsd1036Honmf4fXt4D6nTJbG7Vfgn2owysO3ERjhRpbX+VLOE6VNg9iVPoactGSYey\\\\\\\/i\\\\\\\/Hps9k5bKMqUrcc7FLq0W\\\\\\\/vyvTbKOuuemoptuwCI6gAiOuYeLiYXfXEo7ZD47Vu\\\\\\\/Uxdhji5EkH98EcWI6W\\\\\\\/zR7VMG7KjRLK+bEPIwAvw7feJ9a1C+6vPB5fQ2FrDuyYVBZ93scb3YvF18M6hGGZWK2xg1OBezUV71ONM4cXj3t1A5OHhXlqrE8A86yB0ADJtwYEoHf9IykhFn0cJY2TDZa+tRSefJ5Y1Dus3nhtqQCHSaRvMpuigXf+jgFrRNBLKf\\\\\\\/+yvb9xyrMOqAxdirAHEkD1JpE5zievwdJq+qj8sjHsS3LXywxEDgtMNcvyFbNLmFtQ1TjA6KogWUp7496yTfWf5J+QSXOt4o8P3OfWK00DcoB9sPOqNJTqHukaAE+QumKntmIyWnvA66VVzdqISgCaJOf254BNhITzKLEtpWjJEm1ZDdHLMySIYLbQI3Umvwz1SF2uxXZIwFIuJWviPK2roOuCFOGzRmkQWsRtZIZS6qTRKAFIJfpCZuR\\\\\\\/EsrVkCKsGB4kjSLGKKIVUyzH\\\\\\\/wIeiPpNC3ZHPTysftYIANII2FUR1ZcG7HZjSAQzRudc5w6NMPAHthfZ\\\\\\\/1AF8PUJ7WThEWAJxmLNHeNyMcIfRnuWD9Rjh0kqjaJEkMagLj0BQ5JzWL+rTygV3QQCToqH4FznpnPA7Hj4M7kPvwvoRXsNysal2iBGo1eDBL42VQYzC5g3FmXUvrNK6Ohrprdlo5\\\\\\\/+OeMqcoFkP9QtvjMGf5gt1czLHRNXUD1ccRL0S\\\\\\\/kROmh5ASNCU\\\\\\\/GXAB9yvvQJC0O5HS6muQ3RGqlMEjNhCc02NEJShn21H6mEInqXwFtNNEJq8QLmHw2N6zCUefWJSYIOQUYe4U9gxDO63k1qI9Yah8zfNRnJVFp72dceiaEULoBXx5+1FFqVgy44S3K4z88W\\\\\\\/BE5CM3HC+QyipujKaA6ohd\\\\\\\/T7dsnWUzV0M5uQKcqPpiYpNclyj6q2kPWoBVBQHKwpRm1N0TUFjNUvHoUT9idd7OEKBEtEDCVf7po95\\\\\\\/OCp563vhe9kWCPkmvm+VG2UcJtLmkQNfAlXWQLNnrDvGfJQj9jVQ47\\\\\\\/FlxfczcBPBWeG\\\\\\\/sltOjakiGXVIhp+ErO\\\\\\\/vA857k6Lb4FPlTM7iJ0pzzTLVRUdgOkNhOh\\\\\\\/vl3x9GQardIYlsd4IiO0HCGqwo4WPdwoonLr8qDRwEU5vOIyi8HIMtF55embiJkG+gMdtM0EH8zvUwCSjPdJt4wbSQ9mWBvYuSGA9plxWTRbOCZBVpHZbv1kJKujTie5Fnf9V1hGHdWIpSsqKJUcDWxZ8I\\\\\\\/b5ZKxSu0Uh\\\\\\\/sDyYECO\\\\\\\/0Rd+myyaG8JAEugRFupS99RZnwhkvMRaxJAr\\\\\\\/z3Q\\\\\\\/sXYGzsPRKHJybadx2OGP3+CjQaUqKsuKu3K1E370pa5h5++AGCVu3WL+\\\\\\\/WmqB0ZrI4D+sZ0mdBk17\\\\\\\/9r9cBsq3sFg24mwJmCzJE9MXMGCwYfli8d1tKbVktU+47Gve3HT3kfk6YzRRAch2YQ3wArWZrWgvUIqYgwLx9+sdFk2dbAmniFUBZ7286LOxtGOelW4P\\\\\\\/NOT2mWhf8iElF7hvv77yp3No6JZfplQXoilPYcZfEt1rg\\\\\\\/VsGCR\\\\\\\/F9Ca9xCRQTRvS\\\\\\\/nlX6wrKrO7L9ej7ShTa+UN\\\\\\\/mkUxOTG6jBcYxKZtyDDDPe51bjlNBtWqZ9l4wE9a3SGa5G76jeqIT+jo56OavRUD6aLvcLfJlrLsGgM5Fk64qhf5zZmi7q6KzPuEpWQAV9EF+jrxRo32l4vpSZ7P+Sh61u\\\\\\\/7FojLD\\\\\\\/LHSGmLJja\\\\\\\/\\\\\\\/Cz2NAyd\\\\\\\/tH\\\\\\\/Y6zabIa3SzAXx2WnOgYBce4OOPoAc1gKZIkwRwgxMtOWOcGSiHWMUFCTGqg+JWq4el2T7FEgwcxZ7Ap2DRk038OzeQYDzOfop8vcwM\\\\\\\/qVOAKoLqNU7D9KbKksSp5W+S1buX\\\\\\\/QR1AWte0Ma0XEEMXm4FhhaD62+Y3NQ9e7msEsMuHVskoan+Mgvi4CmhYuhvndpEQu+EpFTm2v1wP1MVZ12MG6Ntm+hzKfXD8SQ\\\\\\\/n9EGbE49rzg7OOfSei31HBCIpM5X8hEUrrbxqRuzhIAJwaRPCBHwomSQE6EDEjTAjA7jH7wIejYe99ph3qYBu6sS+B9oi95jlkTQg13uXrcdqB79AAaCzOQlB4J7hYbHHmQRlDelD\\\\\\\/l3JrkA7bwzmdQbAJHrAPBhFpFhAyyHjdbw9nFohvRKFkB8D+gZedA6R7J+jSvFI3wK9FN\\\\\\\/To6T8cGKstpinPC1h8IEb+BOZcj76ksUs6ABOXBADik3LAdq8aoeXa+3KAmf5pQteN8vXH6j6j1\\\\\\\/zVlvyp4pRsSMnCCiHdviXTsyUN3MM0VzBFxNeHmb\\\\\\\/s96ktMsg8TFKQW0dlhTq5SPfVxDNs8XxnEl4RyMQnUEsDuXQ5D97AuZbPO7GquH0oJBgZ6GO418npizP+AsCCeAb6kemudyLa62qDLxG3R+FivADjnBcEkRs5ITsN91AKUxluLZCEcs\\\\\\\/M6ebixnvSYmhpPU3MmkFiuW4yDFE6K1xqOJ\\\\\\\/vub6ijfkCmtS198mmwWe4AM19cGSFiwiFpDA++POn+XdxNSxOEuh1yB7FTgpvWJGuCdZOYrUfKz+b8xvGuANqkARVYihwU+dBO5GaCwWQCrt+8kAf+2w6F1bKK8WdTtqCmmF7lUPECh\\\\\\\/+3RHQnIA\\\\\\\/za4TgI6ZbKLVs18f6ojAE9w5qMs6hQ0v6De+EUReEAh3FGRLaiE7sPhVlivdXSM9hPQXLRejlxwNcp4NcLok0PPIZ0KrpWwFFx7K7swfHl2+jDuabgH+ctdr+2QkQSslhYzELcoG\\\\\\\/64MU0AQmo4H2lmCpTnsrPu9+DdAb33rzy+eaLrC4DdKveX05vBiVejirHdrBG4X67fmOdTsGtggiF7Vvea06AGse38xgHQXyxdWGwoyKMy9DwxiHEESTKMmI13mbQEw5ysxoPpV\\\\\\\/Uh6oM4kqNAJP8cBGmgIhZ5S8X157tYserCQlT0hzfuLvD3CCqanILy6viAoVrza\\\\\\\/H7OljLDzYeRKRPmX94W3Ipb\\\\\\\/Rmk0a2f0+Ho2bUCXiGQM\\\\\\\/fcueIfKoYB5hL5cSbzexSlGQIjID47D6UT2Id7J\\\\\\\/Widx1QLxUFmurtNha0T4rEsEDcuBnd6p0glmI1q5uU1q7yEBmdzP7M+nKDEHRSimHKGxLm7gRJwuhVQRUDGK5bPhzM9Sofpy8v8mYd1d9Nllms9OMVAHknKTmSOfhzMrhxEVRSBqBYSh5Nv40gp57wVMgony+RJYAo3ymfMq5danDNTZrs9Glqrg82EQziVvew1uUN9G9xEWedXWFR06oAvcv0BXau2fBHMe5r\\\\\\\/4X1gM4izHwXNUF8yzxVW7lgcgQO48EePqwXZ\\\\\\\/vaK78Fmiwu8YslqazHjtHdcuIaLl\\\\\\\/+\\\\\\\/LdloTDHqcVQrzIIwYKTlnBa7+C9EA4\\\\\\\/9gRQQAACQgiwDMG1J2uxefMYkTGPNYKzXg4wGHa50ZHNuVDKAJYvvft2UbOp7rSTy0MP3WlOlcHBZEwZwi6lx4EY6m1JAiL3PGdORB0WFzBNeW3ylGKSzOqde0qSR9zg2K6Jdhz2qJfzrlyy9t5AjEpSbIIy133tZsrjIjvOkosqWKFPI4IyO1Axd2q6W+znKi6+JPIqFWdkLHyEzx\\\\\\\/X85GX1lxswbnpnfO3kxkTvFoW096ffs7TpBzpJe\\\\\\\/yhrzz0MS9UPdu7GZpZrF2eH3YFJPODjhBqtA\\\\\\\/+Jj0yb4VAPPmyt0cXLQJ81gS0+sW38WOGnPJ8xbf9Xl8pehcZW5e5NTJzT8CbdrWvrwPwuPQPryXta7ytPicqparwvMXfJpIeI6QSU+jMl6qq1xA3UBv\\\\\\\/SwGOxWVIB5ljULBHXbFNiBXqDJaMGxvyCKqLlHMsAXREn0zwet+mzk8Lp8duAXYQcBRJ+a+4SRuEJuqBINjJAbMvhXnqmJX2sCwfDpvj1wddn3qe4IdDRjQVM8e1O\\\\\\\/RaziR6MDj62+hWBsu0+bSzl4DENDwcq\\\\\\\/KToH8oPaPW+M0MQ0YkOziEAwWA0CtRlIcBYpuCMygfF5UE2u\\\\\\\/hhuv1ShViUGdf3gha46aaZrkBtqlP0gJVJNQ7HXR0M5tzLaYEPMs7iBfRfeiesoxMSS7IonbS9+vAckDsylOJH6JXTtFwutPOyH4W+6KtrlLkg8U1BIUOMteqtQ1df72KyJIquBhbngwzUDSzTTAv+06ilNp6KUyX6Uw0Ze\\\\\\\/bUZl8P\\\\\\\/qZnnyUrxFzGj4VtqCK0ufSPJTnD6FuF5QvhvkBnzuyW+k4Ak2LIEz2CBE6djPb27ZW22qUm1Ya+zuuK4983GmX4h7zhS5\\\\\\\/6wW4lIK8xhFcuBz931O2xoC1DLsKir57Xq+Dcb4QqB7tFcvYQG7FjVmtxlssTMcwmwPkHNO21KbifxYsunZ3wLVhrLfjGX\\\\\\\/aN342L6NXdLpgi0fHQvAAs6n0m4IwGc2jbNfbxwVlSl\\\\\\\/ZZ76hG3ydi2nj0MkwqF+T4az5wI1x2RZXqH2805\\\\\\\/MkmGxpDUf4xYf0bqxrydQgha228oHUZCB6utkkFMWoDRfz2c+T\\\\\\\/kveNQM0oSBxSpLvtguRuSCxwlMj\\\\\\\/gJ7MeWdgCeq9fHS0peUNHh4uNIF5hd0rrDvNF6zdxTKyyII+PorwTTz7I\\\\\\\/o0qnk2NPpjyxBcSIjxWt9V69MqtAkGt4vw7Nmug1oRZZE+PHzfqHtnOz8HtTPYKssNke\\\\\\\/nLQaHfgFfBT5wgqUPyzCbKlx4XTJr3nw5dqx8JIWKtDoGK\\\\\\\/xdeIQM3Tgn68RUiIhNvtad6LcCla7QrB7uFKHu75FsqZWT8w1zc\\\\\\\/\\\\\\\/OiprhxdvBCA0qHZjcFX4Gk1fPHypgr82tC4PQ1Y+giOi5gCuYQk6qPQr2uaJBNK2zu90Gpy+OEAMm8zriOe7IH3cJsb\\\\\\\/U\\\\\\\/mowwQGp4b+KLtivBkh7pSNa6RkBUt0TLk5qYC4SfhVk+UZNaVoOvoX7th0dlXNh5E2ylb9fmyvJ2HmQObfAcToOaVjZouhh45hJWQYI7en4IwVI2bnJZiogKJ8kPTrWjmInB1U8nlp+ExWY\\\\\\\/IJp6g1\\\\\\\/pXrc6K2X1CZsX7027Z7rH6cAH0rwTPw1SIjLQSuddbdu\\\\\\\/K0GPVytgG4zpb4R0GsKyAwKOvPcn3FuJgn2Zn91ZUfS6sMfKUx4zAXmagzTbhnCEvEgaPYpWsoMWHgTzwqa+tEXjHfEJS41ITm3yxk0UZegbm3IMn+xAmD5eafFEH3jvUL25MMhzfjV0\\\\\\\/jDUoWhVqRrEQ68jp3Gecz\\\\\\\/8bOllhVzFgmn8tTMwVWUQW50ecKDaafoYODtuacd\\\\\\\/tOtJnsn+NoHdQTU9PihQYgXJR2930ApO1uJx5ZnwhMt\\\\\\\/3v9c5+yBKIZuYWdlScSkmkhfar8OKCG4xF56s58rxMx3m03jR9C3Zf6BWFSlVO1W4wpML8OjPyICsOL44immhjp4VFCzOSdvNhixmKMkUtQu+qypSZaX6ANkLrCUq9Skq7gZdhI+kAE4GztKWDKnIB+rzl2n0NcywviN9FuyLJCa1yv2B\\\\\\\/Wk+CD71qnsBl17is9ZE6XbRbCUIME7FDWYUVcapv48F+QAw9YAv6GYaTiUOVBkDbLgpkOk+zXdjloOvMQNhxA6UXBp0VJ5wgmz2NcNAQBRGDAbjX0w7VR+Rnoss4R\\\\\\\/yepgWIHH\\\\\\\/j+B3TiFM3SBsAs4HFm2X\\\\\\\/wCORixAP3rhKxT8UdfIl5rqXZrfBHTDTHSxipdzWFVuj8MLcCp7oDqawl5gEs1+OxMDqhD7Y32TYWGXRF+YmY6a8IffMFJo7nr\\\\\\\/sK8mYj0J9JMhxKiPDn7AwI3705W+uIsYJOSzRQ5eAz2R71F\\\\\\\/Z644ZeXlj8xzCELl1mFswWFx8cRf\\\\\\\/FSP3ChtD1APRffT6W\\\\\\\/cFsn\\\\\\\/rVlqAqV+VL4761NEtIqPeLJAouWOVsLHNbvCgh\\\\\\\/NZ3nLoduaFFFFw7ywBWL0iFs5kBxGeUtGfYnKmqApLWsfTq1unQiMAxmJecw+e+3a0MbJLpX\\\\\\\/gR2KtBKzN1xnCKEF8bNvNQLJ9hii\\\\\\\/h39\\\\\\\/WSSryxaVgDSGsh3P5AqD0E3TIFuy+XPUn2N\\\\\\\/u4XfLT8VG+6caKgdb8iLX9BIHa9snmA64MYv5UdSCGaVqPrulEvJjpHUD1ZaPbJ8afMgu9OOTx264UC+fIQEtYvZ1PSZFsyignL8HwnvhTXZTK6YhilzHwbdfiUshcoPSfHy1xdeCidcSXlwktoVhcZQbXz7VZc+teetdykEqq14Yhjraf3bpFyynPcVzWWtBfEPsd\\\\\\\/t05pAfsfkfzdjmFAJQMdRm2M2QgdVodqUMTght2V\\\\\\\/AyAZ\\\\\\\/LQBaPx6YSNo\\\\\\\/chlPy91ALnvRo1UtVLQDBibItAOQTVucXj79GAZ060WPhrWFCQwa+9BxZtIcsnoK9eRTU1i23hDdYDps2k85o32Ah0IU8oUZ8GD2mNQx7V8rBvUXCc5j0Z5QZGjKM9hv4eq6xlAccqB0aNEyuaLryneMxmYliptJkoNT\\\\\\\/lLjQtAoa2HuKfH0umd9g1Yp9mrh2tBJL5g2ii6CEqZINTGGwHM1D2JUn+m+L8MZsRYOK0\\\\\\\/SPFh1vRhiBCz5NvynFZbI8vDXNIk\\\\\\\/MkPewR0Pzf4UjOUsJqSIxSJjWi2Ym7N5c3y2QqeSJz8J3gw3+VReN3wU\\\\\\\/TRiaAe3K0jQhmFxuIEG8oh37a8EU5s1JXVd66M0g2SE1R9ompRDwIT5EM0n3wzyVZc+dATL\\\\\\\/vGjH4drMtCvxjSYNcrXqJ\\\\\\\/dhVT6msrfgv7unY43ktQNu2ILP69he+f\\\\\\\/bgkDkZZJ1eoJHIvJ4svOKyBMlsFVExOwWwcbak42eNKQp4V3Dp5\\\\\\\/JPUBjD6XQNYUXsY717Oy8V6BWyzpr2q7EI0MUhbQzqVf7LlujFBqJEk06MrPGTZwMuv0UuK89vQ6+uijLwAmFX5EvMNf8spbVq8xMWJHirUhMotLBwGA5y7QN8hff6vW29fYh0k1yKK7a73dDUWIzQZUJ+wyPZG4Ej13SljbpAs2zF0wf5po\\\\\\\/IYCwq\\\\\\\/o8jaxj9N38gP0FqWXwtPAeYgR7B\\\\\\\/mvrJstTIhxa0KgWqfaHsB3X\\\\\\\/88M00cmIoy4QfdrwfAUMtV3Kx6wRhBU8Ih+J+75A\\\\\\\/XiNkeBoqZJvDODyQqxTCLOqZGUc38fKwpJIJm\\\\\\\/b5HfGGzW4bneH\\\\\\\/jVizmwJMW4YIw1ile+OoQW3Vd009JxHunl8ldsopjussfCiyH3FPTQ66s2IuLoe9E28G1tECrCxv5vx7AZb2ic9NhEdnwbfh9U3u5MB5Pqg9g9w7X8FmiZpYrxtcsK2BlkgP+nPuN84n91d5uf7s4aroxfKRUFfY7dCNbLpZJnswxkgm2PmVXJux\\\\\\\/5novy7hFpomeNN0JkM8D8h9ZMc\\\\\\\/dCPQLqusf1kKRa7pShqdSEwttK7BsqRAaV2l5D+4osK\\\\\\\/h76j8Tle1NEYph74Fsan2HG8Py9Thz3IteI4ehGyLhQUj9kxJMU6XhE773f7SaQhewIuOICG5PEuJg1+cVHFwvJb52ciXPMhpfevn6OHnEZ8EuhOoENNftPdUIEtnAQlBJyEopP1oPQZB83g47GgOtsPOyrqhmAiUH5H5mZVrZb23nRCTpuawp+HvHPKpH4UAsLyTkQVz8M+86pd898Q2Fyy1CG9TrF2bY9+AE7Q2+lzkg0r+Osyo0kq0ChHY61OqE4z8zz8tZefiJPtlXlssPdvh24CN+rEQcp0W+LTLJ0BkeOGmso8kE1iK8ilIp95Ob6dCKPhayN45mG5gMezabriYfsvNwGUptsPz8xWRBiW+UW0gmKZ4SSC5CC5MDhGFj47au\\\\\\\/EJm8Vnx6ppItMmvKvyqUHeaoDsRCMKT19\\\\\\\/szIXKWvtWRdzi5h5nn\\\\\\\/\\\\\\\/SRT\\\\\\\/IHXafhTLU8bkcU3r8MR\\\\\\\/3czIB5atmfwGZ\\\\\\\/ry6zc6eDYoHGShTwn6ipqAhX0LRoAPgpKO9tLZ96\\\\\\\/HWPl13z\\\\\\\/IHs66rr8X3bLqCPEkktZkRWoAImG7dlxdAHl4HPe8LxDoLRx63B37vcy4\\\\\\\/LG8Bq0ou2G8vnJX01NOoARhbwpkQKwAXtyNtr06H9weUdcW00zGkwN6L+T9TDgfv0FPl3RjPvNjn72DHGORm0DQMxuvBU66VbPotW4YMzKMm3zy1HI3XVbWU92FQJF8QA2zWpHw\\\\\\\/fwZ2vab8woS38KNbmNgv9CPo+rR8OzMg2AdUaafkdmmpx8J8rmjfDd5bB\\\\\\\/FEZ9IhrWF6\\\\\\\/0zNw6xcfHeqsmPuvuC1wiKGtqd+uezcY1z2v9sb5Bs87bq72oRchlJCbRONlshZElGILDNzHg7u31da9juiLhF6kR078PcgAsjjoHNPx1KEoSZJ8q+d4oWy9mOiQVQQk8EkT3osHC231Wmk6ieSRKxu4gd4qcrWN0V8H6HtFI2YPf1V8Iv7ThW9jOXTfnWQcI0JqegEs2tiYMmAwgunhKo+4CSJh7Lw7SQQvDRd9PydfB3XfWVwHMnjGrWGEGmjjEEvAOSmfW8w1N+l6yddsdQpWyV3UHMH0JY96gkEN0Bn\\\\\\\/+qLlPaHEQuoRHPRa0QFBgWJZui+owyWZkJ34Jn22NaIeNpGazqTgshmOqMZNIWUT+xDJ31sSpThIXB6iBmmXWDjvwIlugE4os8QKETVMmBUWX9JYn\\\\\\\/v6jE9ivz3arjgXCJ0sRd22hShIJaaxNQT18XLEedBCOl9bmhBRsUl4\\\\\\\/fuTbaH0n1JDa49Vm80Fcaaof2sbHpyTYbsbY664Qn6HoHxdK20KqOc3irezKPEdwUGsSvb2d4fldzcsZIJw0jRQvIHGdcvMmPUOfQG4nqTu5uIxukhUPV6B0WY1HsonerzDNnAsJD54JWMhJyrJQgsqSOQBprb3hXHEkcD6Fen3m4dUWmwJF5Jpjq21vsHW9sIjxD2MgGAoI4vjWgTknRYVJfEMhgoCz6\\\\\\\/CmSKM+dlHZUN+vpe7Lq3bmIJ5k3hOCBQFsgVhX1cghBaBUSY6leZq1h8tB52n1OVbv2IXR5RKI0aQBCzg+zMVnI8E3PBC0NaqShRueUylS9tVxpxGQkbj1NHHFOPADBdo4jbatLwTdqIALfmWkdnY5ukXJc0ljH0BrJrTf5+e+L8\\\\\\\/eOQoVxS8vlkPOuv1rLdSMazpCV7H1tt2ymuOuY2+XuIgWSdY9FfO8A6LMnMWmXtQZMwV84uPEgicBkLB9\\\\\\\/S17xNWNkwwilfMf+atjlib1uZV0i3bUeAleXuZ9D\\\\\\\/Ahm5budvwI+4fbCVjbycUPB2FiE5sg6D1o0n2FaoQg3\\\\\\\/aOZtf\\\\\\\/9i4evsxh6LogSIRKHep7N3cujkpZAmmCI6e2PMN5xIPMWTEOwhkXjAoo9ToLFeLK7rxnJpD04yuk9II9FSAo9bn+Nd0Et1TwXOOiykpUv0KvGY1GN+TVTj0rqpkwEs5NKuBQEWdE89fnxNrrepYgIP2lM5aVieqcQf8Np+a5tbtonKPOnNNS+H4YVkCVmZm\\\\\\\/LnZojwggTeKZwX96XORT9GU8FIbVIvtEdBSXBzzr29V8jzLQ4ImEVUNa9khQsgShBobjU3t95HjfzwhYZHNPxe\\\\\\\/IT4DDYDWnBVXrlyXr\\\\\\\/x75tk88I1z+pLBKQNj6jpng9BUtNLRH6wxvCknRhltJQW68l3na0w4tAezMJD8r7HVnPiO7YkiMRlr+G\\\\\\\/fs9QlzCtz+exWVKkcirwc7b+dfV\\\\\\\/GmUS5t739iLxVu0imi50gdLez0sqPtz2zeTHNrbXjWNhscjMgYB7VVzFDlnfxKamjtrwZZhsIqHG11RZHOaEo1k7qAEH4LejNvLGZAr83LKAdxCnjmsx+RKBVuw5mj7kDgYlxIIQ1hHYhKIuOxIvKDk5wqe6lSDKu23IguZsFcZevS\\\\\\\/7GFnRHJ8dphbO660PnZ9+8O8TJtqvSgYcJFkUX5yByrjb1hBatHFjXTZtQX9HU00twH0s8idlZA0LL4ykEPm+IJtSXrBa7mpY4beJSbzm\\\\\\\/56RCCw9I54ajJ3T1f1iTUjCr3AJByu1ujg2yca764ufE7sv2pKax3LF1HTEJr8h5VuHaKVuYCHnoXIqBdN3UH8\\\\\\\/bvLnBsh769EnkC1PCy7AgC1suwFwWtcGlCOlFBj5bCu0bJPWcYSqjjHTO5LFwFp8N3IrEEt8+J4GUKYxpUo6B0K0YIFDJ9rgCjUH0ogjwIN\\\\\\\/XuMGJXq327DAWfXxAzMmLF3dkn13\\\\\\\/sG3m\\\\\\\/gmi5e56NlQcrInhC+AeWbg2qqaLANK3lkF7\\\\\\\/\\\\\\\/jDCKpwv5F6n1dfrOSMaWAhguTRlxnASkvwczfkkjUIU3B1bw8EH7oswhYDCmXp\\\\\\\/nhyNwOlceiWdwCK8QZgXw3fw81dDUwR58CYv0hP568VIflbmgtAF4nd8Xu+4oUnuV++qwHNFqxQTR6fXgpqA00pFusmgGYpWhhC+3XogzReWSNQLwZApp4SsvF+BeAbX+jy5ufaaJWC+424XgcdgsgTrzuDrEmgwxlcfIV5wFJHIdMYKRq5uVLqRyQrGhP4fnhPnjl\\\\\\\/\\\\\\\/G+N85E1UgJABcxP4qBg4Pw0E8kvBXY6VNAHdgjDIMMTC54\\\\\\\/x6RtKyifkaqcmSXALWRMV\\\\\\\/c8SMKyL1kAX4XBfJszrD5FetYWngOMiMu9cpnv1B372eWbCV8rkaRc8ILSI7tri1eYJMrWWBhNbOQBUsGlhPXwhnkg78mZ5Czq4zaUe+X7StMu3Kl5W0UukAS5H45pCvopBcv6GkovP9i3I0K7Llh4dx0rdCcshM77vzdod6OiE+QOD9HWdOys9+vgE7cGVd\\\\\\\/iC4nzN\\\\\\\/\\\\\\\/8noF+yd0nL2jW64L1C7HEwy\\\\\\\/D6nGTDm9RNzwyNTG5SQTjMQ7lQpAvg6EKuPGdUoUr0u+gcUYYr0nJmv8J7gf6wXQOuhALZ2fACW4JrXPuE1ulJUrLfY5+efbUBLBNQnD2aCmtPE2AmZCF7ePvTyr72uxj+dJQy+de\\\\\\\/aArXGbjadU\\\\\\\/mQz93vw9RfVWJB1Nh5yPyv2CIWosksP\\\\\\\/0Yz4SVMZrntKENtEBvlHUVCORyWntXBZWO\\\\\\\/PcfsM42pBrdPZd+Ncncf2Y9nZZZ0mjZA03iL+R5i91tXZXEHlf6ydTpM1J98fbuvO4miJ9mIg9M3v6b64F2bdylU2mCGeyctJWSqD2\\\\\\\/osxjYYshzvNBafw6w3aDrEnbsTqFmDZKJAYCiSzqlWEbrAFF2zg36N\\\\\\\/+xg6Uf2R6blWL\\\\\\\/aZ7ZGW5sRbkQlgJKNdinllTFhHdks+9V5OhqoVFB\\\\\\\/28oSYQb2V5Xnw4sinj8qQ2D9dzsgqW\\\\\\\/xlQv0tm3ATec9rf2Oa+nJgvtV21KFC9DWdMdQWNJPkTyNoBcPi822lZZfzlh5vA0bz7oF5N8gziDj+IRoh8PR6TXl6TSK4EkL2C4OYwzRSoJ\\\\\\\/JDWZaSCv0x5KgkOqO+KzF1qko0mRdQDocHdQDbnHTZmdHwrYhceVdfJ8gzLbvMVWYTVWYA0ywJNlvLGAbc6BEtKvQIwOlMb0sbu4HkSvWZQvhu+AA6hPrPfdUUbWJEWQNIOL6jNDM70CKpc8XCR3wdipI6iI4Niq5tt6Ect0jQ+BTuH5hALm44HbejCxaWe7IZ9pJzkKattm3MVbBnTUHFLcKptgUzpNQnF9P8NSyGgA1fsai3qmB82ILXlNwdhTUtIGvdeW5oyshYehnxqj\\\\\\\/AVCD\\\\\\\/wY8ZWHlXYpquMQmz9oHlrngMgL6XalcDMDrZUSSYaSLkrH9313Y9QQyGph+Fh892FZsRla1eEtt3vXyWQ6ngIFzbhV5\\\\\\\/0LrdLZLT\\\\\\\/VjgOAZv+0u4joWumvNOg2mvB3cNEdCGfPQnk83TKcHtLalABXy8g8SZYqw36aykGnj+X2ebTGjGP48OeS4ieuf7xzHZwdbINP+ucNrxL0gblbZxhSNDij61tZBFq2fRKnjMeM8K2CG9OjkxVvfWT+1P7hfgwout4HgH0UR1\\\\\\\/HhT6Pl41E7Z4e7EueffMAEp+JwZ0t0wJ9xx4KsFEFYsTjD7f8WvH+R1lcUq215zt+tO61ckcpzsPmyRfap\\\\\\\/ZwRQCftl91lfHxM3c4hErrNaJdgNWQ06HEr4lpUzmip985ReGYENqB78Y\\\\\\\/8ws+m18ugzALe3ME5XrtU2AvASQog+PLjUMG0dqWwxIK3QgVNy6CUnbD05ugWvvS67Teub3k2BLaoxAeIYv5fDdzqAIGC7P+3Dz0PM0dkruiMBaMIrgEYdDF\\\\\\\/rQP8enDPdMqU1X\\\\\\\/C7ULxOm2cehlHjMyg4LvgMKu\\\\\\\/+jabQ\\\\\\\/\\\\\\\/lcRpjsxG+DDb05LIrAbI\\\\\\\/jlwm1nU6bNLK5Hr+6vwhN\\\\\\\/TQYBPWlv3tlZBPcjMEdoYknlULznJ9n0QneQMKW4I94C2H2EeZCD2FRt3ekXOeZA\\\\\\\/YAxy0KThu\\\\\\\/XJbE6oHVSjcHmyIEwm1GuKBKVpnJ0DHaOZ6E9x6C5c6y1O9euv9lybzON1belSIvErGZ\\\\\\\/5ezgDsrd6R74QGKKhkatYEI8ZJzOLgwgQsuyJKPFudivLxHLE5MKSorrK\\\\\\\/2VorJioc4z7Zk7ws88sJMfi\\\\\\\/hNlJWjxrryu4NjJpTApYiWs3atjSmOa\\\\\\\/E\\\\\\\/81oXTt08A2Lwr8BOuDgl4RW0xxtOgk+lpqZBj7gikLyUz4RIgihG1yEQYUcXe4joLkiZkieLhz6uk4MUjw6sXY6I7hahlTSPDXgvwML2DyYnN9G14s0OBe\\\\\\\/KSxhzvUgQ7kSVscVYmCTwZBCZfER7TD\\\\\\\/F2TX7LSN7qmMMewiuLtWlC1JQVqlp6TT+X2ej2MbUoEgIVeQZCFyXPQG3ytS5qfmdUWCVKxcs+FpUu0M9s2xt\\\\\\\/ux+kkUNu4VSwgapj\\\\\\\/JCl46M6YvePydhCMjxmckpuvWtkoS06kxrywoKHWY1adkd8\\\\\\\/rr2GGQu9+1ao9x6j4k0lpDtZ4\\\\\\\/cgbMtpyMFXriyTFFF2u4DY0+W11i\\\\\\\/SzfymgOPSgPATrR1kzcwKV9tgSPDhBRMTVL8t3r0XQK36MKa7+5f9lLFQeouOnWFZd3HvNmjgEH2J5oEmXBRY5yXbHHfT2GOfXugdtDj90exgnV1rF56cpSrkrl4spzPxun4NrWwo6r4oJCSm+RJyiWIKXsPcaKKSWcEZ7IjqsUXck5Z5vtFrYa0A+BHzGNtk+KllhVu4kJTqaYnKEre58QoBouahQoZXvCG5OUxQDzjS3AbNCe354g\\\\\\\/\\\\\\\/q+XQBxpXnphEEWCdSo+sLRABgYUZ+nlKd5+eLT22OIqtOGqS55TqT7vi9EqN9U5G5BzNHZ5+N+OWTgF7MJbmC5DMzpwze+RQkPyI16hf4UxnWnGPdJBc0x0HiigJ5ZZISKqhQ2ZgLG5edZHm4OXjxGH0umGTtaWuDTkyanjSdXlW50Zu7a4MGhxrcuQcYSIYGu9ec3hIfN9phqZv2GNWLIACadIPory+ZEuICjAoQCJQvFBNc5I6+wvBL9XevzqvAOOdjuKDYoXvADBjMWeQJWDGr268Xpt+uX20sqfTrbgcsh6TjEkkxyvDONhIu8LpGvruUwbH+l+LoGPi3V2fD3JbOrsXepgp\\\u0022,\\\u0022iv\\\u0022:\\\u002296S1MgLfkxbWKnYgzu2GXw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e16141416e54b63ca537f0b03ab3b90bdbd0154a2b3f3cac766bad3d308fb823\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XMSkCneH2T9IqWSdZCNWD01vvbp5JMeVdLwJDkcm1DeohM2TRYzrFOos4MJPFv\\\\\\\/ne\\\\\\\/C2ZR7pJOcOlusMoWHBLi4tD+JF99dTa+PSBokCnYLVuZ\\\\\\\/xE7TUY3KmMDQ209EUopv\\\\\\\/h40p1nWRFpPWYsqyiZLQk4xeoDJl03mwaImrwN+JUCj0p3WyW1mwgnqja266OeMfEy9CbsIrIF4iU82OECINgrgTbsZEoaiObjdfceIhxPlafm0WIh7dDE2JqHYN+RgQd7DCAVrcQp3SREn9xthJcV71DIMeVRcn4FuenRm7oPbH7RYW69FRN6Jr60o0Hfhfd+6bywYIfDmtiKkQssy4vESMzXcNFnSCzmpJup0I60ZAWTsKr5yg9ZuYFx\\\\\\\/jnujWHkiZh7mBgxncGvTQt0pk6h6N40IbQzTkjoRvGZaKNoaKAuw4eVxYd5u9Ot7TGp3Qz2EXFrBgakeC6mbuLNozLkOGjLmV5N1I+Gr\\\\\\\/Ga3CFVVxJtVjEVFgpZJJKKWIdVZkFH\\\\\\\/qR5RtJ\\\\\\\/QTERYEpP84HkHPW11BFY7LyPgo2GbRVgRcsWj85L7MwnD5LVTyHAgoTZ1m1Vb76KC1QjMdArzh5IMgBAkmCLiOhCtuznSU0pzxHs3e+4k00FelewTYC+efHyKtD0PtbShWatpRSndF+QVGoUnbznm1A8DJaKFYoBLwC+Apqt6H+9Uk5yjjfBjX76OnfRYpujSRPGO9Acy45zA1n1Fi6FujJIDtJcLOytBrdnvVoBqqEATvn1yZ7lK6n1O\\\\\\\/hQQA861S9PMA9+gNZvmBWV+CX4tNIp0yv2PuVRwI9FGPwy2ZLqydhUdNtKNhvWV7xHpjKniJLufsSbMoKHXotEShLpYLdobZzNC5xHdKW2u\\\\\\\/ed7zifKwCMgB2vOJlAnXyzVFCZ3euL\\\\\\\/urJAkNRQNR\\\\\\\/IA2sntgpEmrYtLsM5kaFwPQYzs3s5auzZRta8t26UZb3+4oVrA0lqlD3jNFAqcUTVWml\\\\\\\/UwHYUA4irFTc0gBMy4l8Bqva14HKHYtFuwIo70VZhSfJBEHdC\\\\\\\/0Wioz5EyzzhWkAmlUxiP\\\\\\\/LBQawCNhf6hVKwiyblCVU2xYuUHtwKGZLg3rHrQ8+iXSWcxPccRYa3gevVKlyUEhjIvx4lsQ\\\\\\\/SLhFSc+YrrWiGEjdkgMwTIeghwIWAptfe3visbk5kGsjQUGehbjwS\\\\\\\/kLvyKAKDMk74cZWyCvlIGQx7ihg1uPOn6wYvTBrYx+gn3ngueBqNc9GFGN9ADy+724g5U4Nk8UTS1zaLoRbqIStrSQL6tqfFO0YribllsDKKnDivGplMWtUaUeWlHjCH43CqhPv62k4Zeo01Abi8ZiHitfGYRQyaOA8+noHGC27cSaIwMKm6ulss1fOWXRQ73r2NMiJQGL5gDwBGN3gwRFOqSmhSzKd5bCIfSDlnu2xorIRn7bKfyDqZvIrRDKrqjOmDFoe0dJ53q\\\\\\\/Q+jDZWpWe2ZTsDkGpR+17uots5sUrgQBpoNOShn0hQduukLs2dUd0+ZVoPUSC7aRvydteWJp1IQvjO2rm\\\\\\\/gu2mpaTQdUWacy9HcVzkaG6IlnNWd0RIuCppD4GBetROeocCS945yhF56QFjRV6Il7IfudbifwbeRFOkPeJZlJG7iQ7MxFnzi74K\\\\\\\/Xrb00WsMU7\\\\\\\/fQaH2a2mE+9ZX0FkYOtIRT7y09MHRK5efhLD1r\\\\\\\/HQVTOlOwtrPgWE6FF73unpudb91LLqswmD1zXiFkL0\\\\\\\/YfDmNm2EGW\\\\\\\/kkeWgyOKTRyjd3cz87Y9NcAYpvuS4EaMMsAuBZNooI5E0NpLevIVdrSvsuK03wbrdI6fZ5PV8+3qGC8TKY+t3wneAsrcewO\\\\\\\/z8mneHYz+ZgSUvBeIumpYZKCBKOnCFwCyER713j+5cwP5FRhXXptVVSxB+BRuzUCzU4AMxzy8khZeMeRjGTtueWnxqZGRl0dOilHeldz0xX65U5m9qfrUeWjgmjnhyoH9R\\\\\\\/XBIpuk0ByDwKUHj+cXvxd35G0lCAiuUdeViEZzUWAPay2qZCZs=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ePx2pkNR3OYR6fgNxJEJRg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fe95002ddd505dd3cc579657b9a09c6dc283ae1dd577644f2dbc74a9b2b8b39c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460291 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2693" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XY2VDKNGZwb7rancXcf8WF52XqtPD69THSN1PHAQG0o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2693" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zKO-_QAkwbW9ch_p6PHtTLRzZHouHbzzcPRMtK1Y3cE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2687" title="An Inversion Disrupting FAM134B Is Associated with Sensory Neuropathy in the Border Collie Dog Breed" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2707" title="Microsporidia Intracellular Development Relies on Myc Interaction Network Transcription Factors in the Host" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460291" id="node460291" data-pisa="ggg;6/9/2693" data-pisa-master="ggg;g3.116.029215" data-apath="/ggg/6/9/2693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Signatures of North American Soybean Improvement Inform Diversity Enrichment Strategies and Clarify the Impact of Hybridization</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Justin N. Vaughn</span> and <span class="highwire-citation-author" data-delta="1">Zenglu Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2693-2705;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029215 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460291" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Justin N. Vaughn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop and Soil Science, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BN.%2BVaughn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaughn%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BN.%2BVaughn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zenglu Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop and Soil Science, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zenglu%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZenglu%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zli@uga.edu" class="" data-icon-position="" data-hide-link-title="0">zli@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/9/2693" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460291" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2693.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460291" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2693.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460291" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/9/2693.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460291" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/9/2693.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460291" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Crop improvement represents a long-running experiment in artificial selection on a complex trait, namely yield. How such selection relates to natural populations is unclear, but the analysis of domesticated populations could offer insights into the relative role of selection, drift, and recombination in all species facing major shifts in selective regimes. Because of the extreme autogamy exhibited by soybean (<em>Glycine max</em>), many “immortalized” genotypes of elite varieties spanning the last century have been preserved and characterized using ∼50,000 single nucleotide polymorphic (SNP) markers. Also due to autogamy, the history of North American soybean breeding can be roughly divided into pre- and posthybridization eras, allowing for direct interrogation of the role of recombination in improvement and selection. Here, we report on genome-wide characterization of the structure and history of North American soybean populations and the signature of selection in these populations. Supporting previous work, we find that maturity defines population structure. Though the diversity of North American ancestors is comparable to available landraces, prehybridization line selections resulted in a clonal structure that dominated early breeding and explains many of the reductions in diversity found in the initial generations of soybean hybridization. The rate of allele frequency change does not deviate sharply from neutral expectation, yet some regions bare hallmarks of strong selection, suggesting a highly variable range of selection strengths biased toward weak effects. We also discuss the importance of haplotypes as units of analysis when complex traits fall under novel selection regimes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/selection-standing-variation" class="hw-term hw-article-keyword hw-article-keyword-selection-on-standing-variation" rel="nofollow">selection on standing variation</a></li><li class="kwd"><a href="/keyword/haplotype-frequencies" class="hw-term hw-article-keyword hw-article-keyword-haplotype-frequencies" rel="nofollow">haplotype frequencies</a></li><li class="kwd"><a href="/keyword/detecting-selection" class="hw-term hw-article-keyword hw-article-keyword-detecting-selection" rel="nofollow">detecting selection</a></li><li class="kwd"><a href="/keyword/maturity-groups" class="hw-term hw-article-keyword hw-article-keyword-maturity-groups" rel="nofollow">maturity groups</a></li></ul><p id="p-3">The study of crop domestication inspired many of the earliest insights in evolutionary biology (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ross-Ibarra <em>et al.</em> 2007</a>). Domestication loci have historically been detected through the phenotypic characterization of crosses between wild and domesticated species (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Doebley <em>et al.</em> 2006</a>). The transition from a wild species to a domesticated one can involve selection on rare or even naturally deleterious alleles such as nonshattering fruit (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Meyer and Purugganan 2013</a>), although long-term improvement likely depends on genetic variants at intermediate frequencies in the progenitor population (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beissinger <em>et al.</em> 2014</a>). With the advent of genome-wide genotyping, researchers began to compare the genetic diversity of particular genomic regions within wild progenitor species with the diversity in their cultivated relatives, under the assumption that strong artificial selection would have fixed both the causal variants and the neutral polymorphisms surrounding those variants in the domesticate (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Morrell <em>et al.</em> 2012</a>), a process often referred to as a “selective sweep” (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Maynard-Smith and Haigh 1974</a>). Thus, without knowing the domestication phenotype, a list of domestication loci could be generated (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhao <em>et al.</em> 2015</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zhou <em>et al.</em> 2015</a>). Correlations between domestication analysis and genome-wide association studies are beginning to reveal the phenotypic consequences of these loci (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wen <em>et al.</em> 2015</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Zhou <em>et al.</em> 2015</a>). Even without knowing the mode of action underlying an allele’s benefit, identification of such loci will also likely enhance the efficacy of crop breeding and genomic selection (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Morrell <em>et al.</em> 2012</a>).</p><p id="p-4">The Huang-Huai Valley in China appears to be the center of origin of domestication for soybean (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Han <em>et al.</em> 2016</a>). This domestication event led to the creation of thousands of landraces, defined generally as unimproved varieties with nonvining growth form, reduced seed-shattering, and lighter seed coat. Starting in 1765, a small subset of these landraces was gradually introduced into North America (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hymowitz and Harlan 1983</a>). Because soybean is highly autogamous, its improvement has occurred in two major phases; line selection, which was limited by the availability of imported landraces, followed modern selection based on controlled crosses (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carter <em>et al.</em> 2004a</a>). There is no explicit record of the breadth of soybean germplasm that was available to early farmers in North America, but it appears that a substantial amount of the genetic diversity found in landraces was present in known North American soybean ancestors (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hyten <em>et al.</em> 2006</a>). In spite of this diversity, it is known through pedigree analysis that only a few of these ancestors contributed substantially to the breeding programs and thus to modern elite germplasm (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gizlice <em>et al.</em> 1994</a>). This biased contribution could have multiple causes, but the most obvious is that these lines were the dominant soybean varieties in production. This dominance was, in turn, the result of more than a century of line selection that, in effect, filtered the landrace germplasm into a handful of geographically structured colonies (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Carter <em>et al.</em> 2004a</a>). One aim of this research was to determine the degree to which this earlier colony structure manifested itself after the first breeding programs were established and controlled crosses became routine, referred below as the “posthybridization era.”</p><p id="p-5">The early phase of soybean development entailed the selection of lines that were optimal for particular growing regions. Desirable traits were likely to be quite similar in the prehybridization and posthybridization eras [although initially introduced as a forage crop, soybean was harvested primarily for its seed beginning in the 1860s (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Probst and Judd 1973</a>)]. Yet, given a wild outcrossing rate of &lt;1%, alleles favored by selection would rarely be able to recombine with beneficial alleles present in other introduced lines during the prehybridization phase (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Carter <em>et al.</em> 2004a</a>). Once crossing facilitated this recombination, the haplotypes containing the combination of beneficial alleles might displace the original haplotypes, thereby reducing diversity across that region of the genome. An additional aim of this research was to identify the ancestral alleles that, in part, were responsible for the early success of their donors in the prehybridization era and to evaluate the change in allele frequencies, diversity levels, and haplotype structure after hybridization became common practice.</p><p id="p-6">Two recent studies have compared elite soybean cultivars from North America with a sample of globally distributed landraces (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Wen <em>et al.</em> 2015</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Zhou <em>et al.</em> 2015</a>). In both of these studies, the elite lines used are primarily grown in the Midwest and Northern US. Both studies identified regions that either exhibit reduced diversity or aberrant population differentiation relative to the genome-wide average. <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Zhou <em>et al.</em> (2015)</a> also characterized F<sub>st</sub> values between US and Canadian varieties relative to four landrace subpopulations, noting that sharp distinctions in allele frequencies for known genes, such as E1, a maturity gene, and the T locus, which controls tawny <em>vs.</em> gray pubescence. <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Zhou <em>et al.</em> (2015)</a> also noted that many of these genomic regions, though very likely domestication or improvement loci, were not detected based on landrace-by-elite comparisons. Instead, detection required more detailed analyses across subpopulations. It also remains difficult to differentiate which regions identified in these studies were the result of selection or were the result of the narrow founding bottleneck that occurred for North American subpopulations.</p><p id="p-7">Nearly all crops have undergone a population bottleneck and consequent loss of genetic diversity relative to their wild progenitors (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kovach and McCouch 2008</a>). In soybean, genetic diversity across global germplasm is effectively half that of its wild progenitor (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hyten <em>et al.</em> 2006</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lam <em>et al.</em> 2010</a>). This bottleneck can be a consequence of the fact that only a small family or, in the extreme case, a single individual, possess the desired features for agricultural production. Such scenarios establish genome-wide linkage disequilibrium (LD) between neutral alleles and those alleles underlying desirable traits, and this LD will substantially complicate the interpretation of loci identified by solely comparing pre- with postbottleneck populations. Still, knowledge of these patterns will be important for informing diversity enrichment strategies, and careful characterization of the postfounding population structure should allow us to more effectively differentiate reductions in diversity related to the bottleneck from those related to selection that occurred later, during modern North American improvement.</p><p id="p-8">Many statistics designed to identify selection serve as a proxy for the rate of allele frequency change. For example, extended linkage disequilibrium (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sabeti <em>et al.</em> 2002</a>) is based on finding genomic regions in which selection has outpaced recombination’s ability to statistically decouple alleles at different loci. While this method and its variants are very powerful, the ability to monitor allele frequency changes through time still offers the most direct measure of selection, particularly on the time-scale of contemporary crop improvement (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bollback <em>et al.</em> 2008</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Feder <em>et al.</em> 2014</a>). Indeed, detection of selection using time-serial allele frequency changes should be more sensitive than other established methods to alleles segregating at high and intermediate frequencies prior to the application of artificial selection. Because soybean lines are effectively “immortalized genotypes,” many lines sampling the last century of soybean breeding have been genotyped using SoySNP50K Infinium Chips (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Song <em>et al.</em> 2013</a>), thus creating an experimental system that facilitates time-serial analysis.</p><p id="p-9">Researchers in maize have recently begun to explore the genome-wide effects of long-term selection in experimental populations. Generally, these experiments are based on selection for phenotypes that are components of yield such as seed-size (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hirsch <em>et al.</em> 2014</a>) and ear-number (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Beissinger <em>et al.</em> 2014</a>). These studies used pooled samples of progenitor and postselection populations and scanned the genome for extreme levels of population differentiation, or F<sub>st</sub>. Generally, these studies found that selection on standing variation was much more common than selection on new mutations and that selected alleles rarely reached fixation, potentially because the time-scale of the analysis was too short or that many of the favorable alleles exhibit some degree of dominance. The dominant mode of selection in agricultural populations remains an open question with many implications for crop improvement (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">van Heerwaarden <em>et al.</em> 2012</a>). The final aim of this work was to use soybean improvement over the last century to help more fully define modes of artificial selection, and to provide strategies for further genetic gains in soybean yield.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genotypic data</h3><p id="p-10">Genotyping of the USDA Soybean Germplasm Collection, where all lines in this study have been deposited, was previously performed using the SoySNP50K iSelect BeadChips (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Song <em>et al.</em> 2013</a>). The data were accessed from <a href="http://soybase.org/data_distribution/soybase_soy50K_snp_all_cultivars_and_snps.gz">http://soybase.org/data_distribution/soybase_soy50K_snp_all_cultivars_and_snps.gz</a> on February 6, 2014. Data were processed as described previously (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vaughn <em>et al.</em> 2014</a>). Physical distances described in this manuscript are based on genome assembly version Glyma.Wm82.a1 (Gmax1.01) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schmutz <em>et al.</em> 2010</a>); the distances are slightly shifted in version Glyma.Wm82.a2 (Gmax2.0). All marker information is given in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS4.txt">File S4</a>.</p></div><div id="sec-3" class="subsection"><h3>Population analysis</h3><p id="p-11">Identity-by-state (IBS) matrices were generated using the command-line implementation of TASSEL 4.0 (-distanceMatrix) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bradbury <em>et al.</em> 2007</a>). Population structure was assessed using ADMIXTURE (version linux-1.23) with <em>K</em> = 3 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> 2009</a>) [<em>K</em> was set to 3 based on the pattern observed when ordering lines based on maturity (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>)].</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population structure of modern public soybean varieties. Rows in the symmetrical IBS matrix are sorted by maturity group (right panel) and then by date-of-release as indicated in adjacent xy plots. Population assignments and admixture are shown in the far-right panel given a model of three populations (see Material and Methods). Colored boxes within the matrix indicate how the total set of accessions was divided into populations for further analysis. SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1448872788" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Population structure of modern public soybean varieties. Rows in the symmetrical IBS matrix are sorted by maturity group (right panel) and then by date-of-release as indicated in adjacent xy plots. Population assignments and admixture are shown in the far-right panel given a model of three populations (see &lt;em&gt;Material and Methods&lt;/em&gt;). Colored boxes within the matrix indicate how the total set of accessions was divided into populations for further analysis. SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2693/F1.medium.gif" width="440" height="352"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/9/2693/F1.medium.gif" width="440" height="352"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469016" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Population structure of modern public soybean varieties. Rows in the symmetrical IBS matrix are sorted by maturity group (right panel) and then by date-of-release as indicated in adjacent xy plots. Population assignments and admixture are shown in the far-right panel given a model of three populations (see <em>Material and Methods</em>). Colored boxes within the matrix indicate how the total set of accessions was divided into populations for further analysis. SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Fractional contribution by haplotype sharing</h3><p id="p-13">Because of the very low heterozygosity levels in soybean, each marker was determined to be most closely related to one of 29 major North American ancestors based on the ancestor with which it had the longest match length to the left and right of each marker. These 29 ancestors were chosen based on their percent contribution as assessed previously using pedigree analysis (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gizlice <em>et al.</em> 1994</a>), although some first progeny were used when an original ancestor was not available. The longest matching ancestor haplotype was required to be &gt;5 markers longer than the second longest haplotype match. A threshold of 5 was chosen after simulations indicated that between 1 and 8 markers were sufficient to identify identical-by-descent regions from the correct grandparents of recombinant inbred lines simulated using random sets of genotypes from the 29 North American ancestors (data not shown). More stringent length thresholds (&gt;8) began to sharply increase the number of markers within a line that were deemed ambiguous. The threshold was held constant across the genome since the physical positions of markers on the SoySNP50K chip were chosen to roughly approximate genetic distance: ∼0.1 cM per marker (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Song <em>et al.</em> 2013</a>). Heterozygous markers, which only occur ∼14 times per genotype on average, were not penalized as they are generally the result of technical genotyping errors [roughly 40% of heterozygotes are likely true homozygotes (Ben Stewart-Brown, personal communication)], or heterogeneity at a few loci within a single line. In such cases, the burden of differentiating haplotypes is passed to down/upstream markers. Each member of each population, MG 0-I, MG III-IV, or MG V+ (as described in <em>Results</em>), was assessed, and the number of markers traced to a particular ancestor was tallied for every ancestor and divided by the total markers. Markers for which the longest haplotype could not be determined or was ambiguous were not counted in the genome-wide average. Scripts used to perform key aspects of this analysis, among others associated with this study, are available as in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS3.zip">File S3</a>.</p></div><div id="sec-5" class="subsection"><h3>Diversity analysis</h3><p id="p-14">TASSEL 5.0 was used to calculate average pairwise difference, π, between haplotype windows for a given set of lines. A sliding window that was 50 markers wide was moved in 10 marker increments across the length of each chromosome. TASSEL options were as follows: -diversity, -diversitySlidingWin, -diversitySlidingWinStep 10, and -diversitySlidingWinSize 50. The units of π are the number of pairwise differences per marker averaged across the sliding window. The π measure for all 29 North American ancestors, π<em><sub>a</sub></em>, was compared with π for lines released prior to the 1970s within each population, π<em><sub>x</sub></em>, and log<sub>2</sub>(π<em><sub>x</sub></em> / π<em><sub>a</sub></em>) value was used to characterize changes in diversity. By normalizing by π<em><sub>a</sub></em>, this statistic allows comparison across loci of various starting diversity levels. The log<sub>2</sub> transformation is an extension of that rationale in that when two separate loci start at the same diversity level, a reduction to 25% the original diversity, for example, should appear twice as substantial as a reduction to 50%.</p></div><div id="sec-6" class="subsection"><h3>Effective population size, N<sub>e</sub>, based on temporal fluctuations in allele frequency</h3><p id="p-15">For each population, we grouped varieties released from the 1940s, 1950s, and 1960s and used these allele frequencies for generation 0. Only loci with major alleles that started at a frequency between 0.5 and 0.6 were used for the initial timepoints. This frequency filtering was done in order to avoid analyzing loci with alleles that commonly reappear after fixation as a result of statistical sampling. Varieties released in each additional decade (the 1970s, 1980s, 1990s, and 2000s) were grouped separately, and we assessed allele frequencies within these decades, treating them as sequential samples. The effective population size was estimated for each set of temporal data points using the R package NB (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hui and Burt 2015</a>) with the following parameters: “alleles=rep(2,<em>x</em>), sample.interval=c(0,15,10,10,10), profile.likelihood=FALSE, and bound=c(20,1000),” where <em>x</em> is the number of loci that had an initial allele frequency between 0.5 and 0.6. The generational intervals (“sample.interval”) were selected based on the effective number of generations we assumed to have occurred during each decade or combination of decades and the average midpoint of the years of release within each group.</p></div><div id="sec-7" class="subsection"><h3>Power analysis to detect selection</h3><p id="p-16">We evaluated the statistical power of four possible measures of selection based on the set of simulations defined in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. For each simulation, the population was held constant at 200 and allele frequencies were assessed at the generations used above: 0, 15, 25, 35, and 45. We assessed power using both the true population frequency and the frequency as estimated given the sample depth typical of our study (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). The power is reported as the percentage of selected loci that surpass the threshold giving a 5% false positive rate when both selected and neutral loci are combined in equal parts (neutral distributions were taken from the simulation with the same initial allele frequency of the beneficial allele under selection). Additionally, when assessing power, absolute values of all measures were used because we will not know the relationship between the polarity of an allele’s selection coefficient and its initial frequency in real data. Linkage disequilibrium between loci was not considered.</p><p id="p-17">For F<sub>st</sub> measures and linear regression, only the timepoints up to and including the first timepoint at which an allele reached fixation (&lt;5% or &gt;95%) were used. The slope of the linear regression was estimated using the Perl module Statistics::LineFit, reported as Δf. The logistic regression coefficient (reported as “logistic β” in <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>), was calculated, using the R command “glm(formula = y ~ x, data = d, family = binomial),” where <em>y</em> is the allele frequency at decade-by-decade time times used above, <em>x</em> is time, and <em>d</em> represents the dataframe. F<sub>st</sub> was calculated as the probability that two alleles sampled from a single timepoint are identical relative to the probability that two alleles sampled from the combined first and last timepoints are identical (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hamilton 2011</a>). The WFABC algorithm was used to directly estimate the selection coefficient (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Foll <em>et al.</em> 2015</a>). Default parameters were used for wfabc_1 and wfabc_2, except -ploidy was set to 1 because all lines are recombinant inbred lines. To avoid computational instability in the WFABC algorithm, N was held fixed at 200 - the actual size in the simulation - when initial frequency was 0.8 and true selection coefficient was 0.1.</p></div><div id="sec-8" class="subsection"><h3>Scans for selection</h3><p id="p-18">We used the N<sub>e</sub> estimates in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> for each population and simulated a distribution of Δf expected for a set of neutral alleles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FigureS3.tif">Figure S3</a>). Sequential timepoint samples were used as for N<sub>e</sub> estimation above, except that all loci with an initial major allele frequency &lt;0.95 were included, not just those between 0.5 and 0.6. The starting frequency distribution of simulated, neutral alleles was designed to reflect that of the actual markers in each population.</p><p id="p-19">Thresholds for putatively selected regions were determined based on the shape and variance of total distributions of Δf and log<sub>2</sub>(π<em><sub>x</sub></em> / π<em><sub>a</sub></em>). See <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> for population-specific thresholds.</p></div><div id="sec-9" class="subsection"><h3>Haplotype spectra analysis</h3><p id="p-20">All possible haplotypes within a 50 marker window were identified and their frequencies assessed across each chromosome in 10 marker increments as done for the diversity analysis above. Each unique haplotype in the pre-1970s sample was assigned a color in order of highest to lowest frequency (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Haplotypes in later samples were assigned the same color if a color had already been assigned in the earlier sample. Gray was used for additional haplotypes if more than seven haplotypes had already been observed in a given window.</p><p id="p-21">For each window, the top three largest rates of change (|Δf|) for individual markers were averaged and reported along with the most rapid change for a haplotype within the same window. The use of three alleles was a heuristic approach to balance a) the excessive noise of using only the single highest marker in a window and b) the dramatic reduction in sensitivity associated with taking the average across the entire window. Average pairwise length of shared haplotypes (<em>H</em>) was assessed using H-scan (version 1.3) developed by Philip Messer (<a href="https://dl.dropboxusercontent.com/u/77898333/H-scan.cpp">https://dl.dropboxusercontent.com/u/77898333/H-scan.cpp</a>).</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-22">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Maturity defines population structure in modern varieties</h3><p id="p-23">Our total dataset consisted of 579 soybean varieties released from &lt;1940 to 2009 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/TableS1.xls">Table S1</a>). Each line was genotyped using the SoySNP50K Infinium Chip as part of USDA’s effort to characterize the entire <em>Glycine</em> germplasm bank. The data are publicly available. Historical analysis in maize indicates that modern breeding for heterotic groups strongly differentiated North American populations (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">van Heerwaarden <em>et al.</em> 2012</a>). Soybeans are not released as hybrids, but they are very sensitive to photoperiod; therefore, flowering and maturation time are expected to be major variables in structuring the cultivated soybean population. We analyzed the IBS across all accessions in this study. When sorted based on maturity group (MG), three highly similar groups emerge (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The pattern is generally defined by the similarity of MG 0-I lines, III-IV lines, and V+ lines. Lines were subordered within each maturity group based on year-of-release, but there were no evident patterns associated with this parameter. Thus, lines have been comparably differentiated into three maturity-group populations since the very earliest stages of modern soybean breeding. This supports and expands prior results based on pedigree analysis that indicated very distinct ancestral contributions to modern Northern <em>vs.</em> Southern cultivars (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Gizlice <em>et al.</em> 1994</a>). In short, a small number of dominant lines quickly established their territories, the boundaries of which have been in place ever since. The exception appears to be the MG II lines, which, prior to ∼1970s, fall into the MG 0-I cluster. After the 1970s, these lines exhibit substantial admixture between the MG 0-I and MG III-IV groups. For this reason, in further analysis, MG II lines were removed, reducing the total genotypes from 579 to 487. We treated the three remaining clusters as independent populations, which will be referred to as “MG 0-I,” “MG III-IV,” and “MG V+” throughout the remainder of the manuscript (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>).</p></div><div id="sec-13" class="subsection"><h3>North American soybean ancestors have biased IBS between populations</h3><p id="p-24">As described above, it is known from pedigree analysis that Northern and Southern breeding lines are derived from distinct ancestry. Therefore, it is likely that populations 0-I, III-IV, and V+ each have very distinctive profiles of ancestry. To investigate this possibility, we calculated the average IBS of each population relative to each of the 29 major North American ancestors (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FigureS1.pdf">Figure S1</a>). Nearly every ancestor shows a significantly higher IBS with one or more populations over another (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FigureS1.pdf">Figure S1</a>). In <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, we emphasize the ancestors with the most substantial relatedness and/or differential relatedness to populations.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Biased population IBS with major ancestors. For each population, the IBS between each individual and the indicated ancestor is given as a boxplot. Pairwise IBS between each ancestor is shown at the top as a heatmap matrix in which black represents an IBS of 1, or perfect identity. The &#x201C;other&#x201D; category represents combined results from all other ancestors in Figure S1. Maturity group of the ancestor is indicated in brackets below name. PI548445 is shown independently because it in an outlier for its low IBS relative to all other ancestors. IBS, identity-by-state." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1448872788" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Biased population IBS with major ancestors. For each population, the IBS between each individual and the indicated ancestor is given as a boxplot. Pairwise IBS between each ancestor is shown at the top as a heatmap matrix in which black represents an IBS of 1, or perfect identity. The &#x201C;other&#x201D; category represents combined results from all other ancestors in Figure S1. Maturity group of the ancestor is indicated in brackets below name. PI548445 is shown independently because it in an outlier for its low IBS relative to all other ancestors. IBS, identity-by-state.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2693/F2.medium.gif" width="440" height="320"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/9/2693/F2.medium.gif" width="440" height="320"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469209" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Biased population IBS with major ancestors. For each population, the IBS between each individual and the indicated ancestor is given as a boxplot. Pairwise IBS between each ancestor is shown at the top as a heatmap matrix in which black represents an IBS of 1, or perfect identity. The “other” category represents combined results from all other ancestors in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FigureS1.pdf">Figure S1</a>. Maturity group of the ancestor is indicated in brackets below name. PI548445 is shown independently because it in an outlier for its low IBS relative to all other ancestors. IBS, identity-by-state.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Distinct landraces clearly have a differential relatedness to the three elite populations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Generally, these results compliment prior work based on pedigree analysis (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Gizlice <em>et al.</em> 1994</a>). Key ancestors such as Lincoln, Mandarin (Ottawa), CNS, and S-100 show significantly higher IBS to specific populations, and these lines are genetically distinct from one another, suggesting distinct genetic contributions. Identity to population 0-I is dominated by Mandarin (Ottawa) and Capital, although Capital does have fairly high IBS with Mandarin (Ottawa). In addition, Lincoln also has a higher than average IBS with this population. Population MG III-IV is highly related to Lincoln, whereas population MG V+ appears to have three major ancestors: CNS, Roanoke, and S-100. S-100 is unique in that it appears to make similarly high contributions to all populations, perhaps because of its similarity to Lincoln.</p><p id="p-27">The relationship of elite population structure to the North American soybean ancestors has implications for which genomic regions experienced reduced diversity in the earliest lines developed from hybridization breeding. Diversity within those early populations could be reduced because of selection or because there was reduced diversity among the lines that served as parents in the vast majority of crosses resulting in those lines (see <em>Introduction</em>). To achieve a more refined measurement of the degree to which an ancestral line contributed to a population, we compared the haplotypes of elite lines within each population released prior to the 1970s to the haplotypes of all 29 major North American ancestors (see <em>Materials and Methods</em>).</p><p id="p-28">The results of this haplotype analysis (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) generally support our IBS analysis (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FigureS1.pdf">Figure S1</a>), but they also clarify situations in which one ancestor shares substantial haplotype structure with another ancestor but did not contribute to the same population, as in the case of S-100 [Jackson (PI 548657) was excluded from this haplotype analysis because it was introduced in the 1950s and is 85% identical to Roanoke (PI 548485) (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>)]. In addition, the results are strikingly similar to those seen in prior pedigree-based studies (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gizlice <em>et al.</em> 1993</a>), yet, while those studies only emphasized Northern <em>vs.</em> Southern germplasm, we have further subdivided the Northern germplasm. The differential contribution of Mandarin (Ottawa) suggests a major cause for the early and stable distinction between these MG0-I and MGIII-IV populations (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The ancestry for the MGIII-IV is generally the most homogenous; the single cultivar Lincoln has a fractional contribution of nearly 0.3. It is also interesting that, though Peking has a very low IBS (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>), it has &gt;0.03 fractional contribution to the MGV+ population (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). This finding is consistent with the historical use of Peking as an early donor of soybean cyst nematode (SCN) resistance in Southern but not Northern varieties (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carter <em>et al.</em> 2004b</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469138/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469138/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Fractional contribution based on haplotype sharing for major North American ancestors relative to varieties released prior to 1970 within each population</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">In the remainder of the manuscript, we treat ancestors with a fraction contribution &gt;0.03 as “founding ancestors” of a particular population. This distinction is in contrast to the group of all 29 major North American ancestors used above.</p></div><div id="sec-14" class="subsection"><h3>Reduced diversity in founding ancestors dictates regions of reduced diversity in posthybridization cultivars</h3><p id="p-31">With regard to the biased contribution of different ancestors, we attempted to determine if a region of reduced diversity in a population resulted from the pool of founding ancestors being fixed at that locus or from a single ancestral haplotype that was rapidly fixed in early stages of regional breeding. For this analysis, genotyping information for each population was divided into windows that were 50 markers long. For each of these windows, we assessed the log<sub>2</sub> ratio of genetic diversity (π) of the earliest stages of the posthybridization era (pre-1970s), π<em><sub>x</sub></em>, to that of all 29 North American ancestors, π<em><sub>a</sub></em>. Generally, this diversity measure varies between 2 and −2. Values were rarely &gt;2, thus, diversity was rarely substantially increased (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS1.zip">File S1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between reduced diversity and founding ancestors of pre-1970s varieties within each population. Only chromosome 10 is shown as a representative example; similar plots for all chromosomes are available as supplemental material (File S1). (A) Log2 ratio of mean pairwise difference of a given population (&#x3C0;x) relative to all 29 North American ancestors (&#x3C0;a) is given for each window of 50 markers along the chromosome. Markers are numbered 1 through the last marker on the chromosome in order of physical position (File S4). (B) The identity of each marker relative to the given ancestor is depicted for each of the pre-1970s lines from populations MG 0-I and MG V+. Each individual is shown as a row and, therefore, appears as many times as there are ancestors. A locus is colored if it is identical to the ancestor. White space indicates a mismatch. Heterozygous markers, though rare, are colored gray. Gray vertical lines spanning the figure indicate regions of reduced diversity. Position of maturity gene, e2, is labeled. The trough at position 1074 is off scale for all populations (see Table 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1448872788" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between reduced diversity and founding ancestors of pre-1970s varieties within each population. Only chromosome 10 is shown as a representative example; similar plots for all chromosomes are available as supplemental material (File S1). (A) Log&lt;sub&gt;2&lt;/sub&gt; ratio of mean pairwise difference of a given population (&#x3C0;&lt;em&gt;&lt;sub&gt;x&lt;/sub&gt;&lt;/em&gt;) relative to all 29 North American ancestors (&#x3C0;&lt;em&gt;&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt;) is given for each window of 50 markers along the chromosome. Markers are numbered 1 through the last marker on the chromosome in order of physical position (File S4). (B) The identity of each marker relative to the given ancestor is depicted for each of the pre-1970s lines from populations MG 0-I and MG V+. Each individual is shown as a row and, therefore, appears as many times as there are ancestors. A locus is colored if it is identical to the ancestor. White space indicates a mismatch. Heterozygous markers, though rare, are colored gray. Gray vertical lines spanning the figure indicate regions of reduced diversity. Position of maturity gene, &lt;em&gt;e2&lt;/em&gt;, is labeled. The trough at position 1074 is off scale for all populations (see Table 2).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2693/F3.medium.gif" width="394" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/9/2693/F3.medium.gif" width="394" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469168" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Relationship between reduced diversity and founding ancestors of pre-1970s varieties within each population. Only chromosome 10 is shown as a representative example; similar plots for all chromosomes are available as supplemental material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS1.zip">File S1</a>). (A) Log<sub>2</sub> ratio of mean pairwise difference of a given population (π<em><sub>x</sub></em>) relative to all 29 North American ancestors (π<em><sub>a</sub></em>) is given for each window of 50 markers along the chromosome. Markers are numbered 1 through the last marker on the chromosome in order of physical position (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS4.txt">File S4</a>). (B) The identity of each marker relative to the given ancestor is depicted for each of the pre-1970s lines from populations MG 0-I and MG V+. Each individual is shown as a row and, therefore, appears as many times as there are ancestors. A locus is colored if it is identical to the ancestor. White space indicates a mismatch. Heterozygous markers, though rare, are colored gray. Gray vertical lines spanning the figure indicate regions of reduced diversity. Position of maturity gene, <em>e2</em>, is labeled. The trough at position 1074 is off scale for all populations (see <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Using chromosome 10 as an example (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), there are three regions of reduced diversity. The first region exhibits reductions in all three populations (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). The second region is unique to MG 0-I. The third, minor trough is unique to MG V+. <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a> depicts the identity of each marker of each individual variety in the pre-1970s populations relative to their respective set of founding ancestors. In regions of reduced diversity, we would expect homogeneity across all lines, yet the identity with ancestors should be different if diversity was present in that region among the founding ancestors. In other words, strong vertical patterning is indicative of reduced diversity in a population; if the pattern is solid across all ancestors, then the haplotype was fixed in most of the early crosses. Alternatively, if diversity was present, then early selection may have caused the reduction in diversity. It should also be noted that some regions will have reduced diversity in a population but will not show a particularly low log<sub>2</sub> diversity ratio. This result is due to there being little or no diversity among all 29 North American ancestors.</p><p id="p-34"><a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> summarizes the results of this analysis, and all graphical files comparable to <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> are available as supplemental material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS1.zip">File S1</a>). In nearly all cases, a single haplotype appears in all or most of the founding ancestors at regions of reduced diversity. Because there is variation for these regions across the total set of ancestors (otherwise the log2 diversity ratio would be ∼0), these loci could possibly represent regions that are important to their specific regions of cultivation. The most obvious of these would be genomic intervals containing genes related to flowering time. Indeed, all four of the major effect maturity genes, <em>e1–e4</em>, are present at a threshold of &lt;2.3 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Interestingly, for all low diversity genomic regions proximal to maturity genes, founder column is less than 100% in <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> (and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS1.zip">File S1</a>); therefore, the ancestors were segregating for haplotypes that were eventually fixed by 1970. In the MG 0-I population, the ancestral haplotype common to Lincoln and Capital was lost in favor of the more common alternative haplotype (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). The alternative haplotype is found in the majority of MG 0-I ancestors such as Mandarin (Ottawa), Manitoba Brown, and Flambeau. Interestingly, Lincoln is a maturity group III line, but Capital, a maturity group 0 line, shares the Lincoln haplotype. Moreover, while Richland and Mukden are maturity group II, they have the Mandarin (Ottawa) haplotype. It is known that maturity genes interact in complex ways, particularly in early maturity groups (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tsubokura <em>et al.</em> 2014</a>), but it is clear that the Mandarin (Ottawa) haplotype for <em>e2</em> was favored by early breeders.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469198/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469198/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Characterization of regions of reduced diversity in founding ancestors of each population</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">It is rare for any population to completely lose diversity across a 50 bp window (values &lt;−8 in <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Extreme diversity ratio (&lt;−8) can be the result of there being very little diversity in North American ancestors and that diversity being lost. This appears to be the case for the extreme chromosome 10 values shared by all populations (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>), although clearly there are exceptions. It is also rare for all three populations to have the same region of reduced diversity (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>, but more easily seen in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS1.zip">File S1</a>).</p></div><div id="sec-15" class="subsection"><h3>Effective population size correlates with number of major founding ancestors</h3><p id="p-41">Because populations are finite, evolution occurs due to genetic drift regardless of selection. The effective population size, N<sub>e</sub>, is the population size that, if the loci comprising the sample were neutral and individuals were randomly mating, would produce the observed dispersion of allele frequencies from one generation to the next. Generally, estimates of N<sub>e</sub> are important in generating a null expectation of allele frequency change. Starting with posthybridization lines released prior to 1970 and ending with lines released in the 2000s, we tracked the decade by decade change of allele frequencies within each population. We used a likelihood approach based on the variance in allele frequency change to predict N<sub>e</sub> (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hui and Burt 2015</a>). Because of the likelihood framework, the approach rigorously accounts for experimental sampling as well. Still, biased sampling could pose a problem to this analysis and those that follow. An examination of principal component (PC) plots based on genome-wide marker data within each population indicates that sampling was generally random (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FigureS2.pdf">Figure S2</a>), although there does appear to be mild population differentiation when moving from pre-1970s populations to 2000s populations (bottom panels in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FigureS2.pdf">Figure S2</a>).</p><p id="p-42">N<sub>e</sub> for each population is given in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>. Population MG V+ has the largest N<sub>e</sub>, followed by 0-I and then III-IV. Inbreeding reduces N<sub>e</sub> because, in effect, it is structuring one population into several subpopulations due to the nonrandom mating of genotypes. The use of a single genotype as a parent within a population over time is one form of inbreeding. In turn, our N<sub>e</sub> estimates correlate with the homogeneity of parentage within each population: MG V+ has three major ancestors, MG 0-I has two, and MG III-IV has only one (Lincoln) (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469144/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469144/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Effective population size (Ne) estimates for populations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Selection is typically on standing variation, but “haplotype sneaks” are also common</h3><p id="p-45">The ability to estimate selection coefficients from time-serial data are an area of active research (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bank <em>et al.</em> 2014</a>). We compared one such published algorithm, WFABC (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Foll <em>et al.</em> 2015</a>), with the commonly used F<sub>st</sub> statistic, as well as two simple statistics based on ether linear or logistic regression. We simulated the allele frequency trajectory based on the number of generations in our sample and a population size of 200, a value comparable to relevant N<sub>e</sub> estimates (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). The range of initial frequencies and selection coefficients are given in <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469152/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469152/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Power of assorted statistics in the detection of selection at a 5% false positive rate for selection coefficients 0.02, 0.05, 0.1, and initial favored allele frequencies of 0.2, 0.5, and 0.8</span><div class="sb-div caption-clear"></div></div></div><p id="p-49">While the mean WFABC prediction of selection coefficients was fairly accurate (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS5.zip">File S5</a>), the variance in estimates resulted in poor power to differentiate selected loci at a 5% false positive rate (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>). F<sub>st</sub> was most powerful when selected alleles started at low frequencies, but was very limited when the selected allele started at high frequencies. This limitation is because F<sub>st</sub> can only take advantage of end points in the allele trajectory, whereas the other three estimates use intermediate points up to fixation. The two simplest measures, Δf and the logistic coefficient, behaved similarly although the logistic regression performed inferiorly at low initial frequencies and superiorly at high frequencies. Statistical sampling reduced the power of all methods, as expected, although the logistic coefficient was less affected. Because we are uncertain as to the distribution of initial frequencies of selected alleles in our real data, we used Δf for the remainder of the analyses and the manuscript.</p><p id="p-50">Though it is often rightly assumed that in natural populations the vast majority of polymorphisms should behave neutrally, this will not always be the case in breeding populations, which have high linkage disequilibrium (LD) and have been under intensive selective pressure. Indeed, we know that variation in allele frequencies is increased by selection in the populations used herein, although we cannot be sure of the magnitude. If selection has, in fact, driven many alleles to higher frequency, the observation that few values extend beyond the distribution under neutral expectation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FigureS3.tif">Figure S3</a>) indicates that selected loci are likely of small effect and, after including the markers in high LD with selected loci, comparable in number to neutral loci. Given the difficulty in devising an appropriate null distribution (see <em>Discussion</em>), the loci with the strongest signatures of change are the best candidates for regions under strongest selection (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akey <em>et al.</em> 2002</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Beissinger <em>et al.</em> 2014</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hirsch <em>et al.</em> 2014</a>). Using population specific thresholds, we established a set of putatively selected regions based on Δf, and on the reduction in diversity between the pre-1970 lines in a population and those released in the 2000s (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469229/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469229/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Counts and relative frequencies of selection modes for sliding window analysis</span><div class="sb-div caption-clear"></div></div></div><p id="p-53">Strong selection that acts on a new or introduced allele, a hard selective sweep, will produce a signature of rapid allele frequency change and reduced diversity around the allele due to the fixation of nearby neutral alleles in linkage. Soft sweeps, on the other hand, will exhibit a substantial change in allele frequencies without this extreme change in diversity. Thus, though they are related, the two measures used above—change in individual allele frequency and reduction in diversity across a window of loci—are potentially sensitive to distinct possibilities in the evolutionary history of a genomic region.</p><p id="p-54">We performed a diversity analysis in the manner described above except that the lines released between 1940 and 1970 within a given population were treated as the ancestor population, and the lines released after year 2000 were compared with these populations. Individual markers with the top three highest Δf within each diversity window were averaged and plotted by the relative diversity within a window (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). We defined categories shown in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a> using thresholds based on the shape of total distributions and the estimated N<sub>e</sub> (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). As is common in genome-wide characterization of sweeps, there is rarely enough data to differentiate selection on a new mutation from selection on very rare standing variation. Our approach is also susceptible to confusing these two conceptually distinct scenarios, although we feel the operational definitions used herein give an accurate impression of which modes of selection are most common.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Modes of selection acting within and across each population. Each point represents the results for a window of 50 markers incremented by 10 markers along the chromosome. The diversity of lines released in the 2000s relative to lines released prior to 1970 is plotting on the y-axis for each population. The average of the top three absolute values of &#x394;f for each window is plotted on the x-axis. Categories are color coded based on the thresholds defined in Table 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1448872788" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Modes of selection acting within and across each population. Each point represents the results for a window of 50 markers incremented by 10 markers along the chromosome. The diversity of lines released in the 2000s relative to lines released prior to 1970 is plotting on the y-axis for each population. The average of the top three absolute values of &#x394;f for each window is plotted on the x-axis. Categories are color coded based on the thresholds defined in Table 5.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2693/F4.medium.gif" width="440" height="143"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/9/2693/F4.medium.gif" width="440" height="143"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468998" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-55" class="first-child">Modes of selection acting within and across each population. Each point represents the results for a window of 50 markers incremented by 10 markers along the chromosome. The diversity of lines released in the 2000s relative to lines released prior to 1970 is plotting on the y-axis for each population. The average of the top three absolute values of Δf for each window is plotted on the x-axis. Categories are color coded based on the thresholds defined in <a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">MG 0-I had the most dynamic range of values, although all populations exhibited comparable relationships between hard and soft sweeps (<a id="xref-table-wrap-5-5" class="xref-table" href="#T5">Table 5</a>). The enrichment of soft sweeps relative to hard sweeps is expected from breeding populations, which are known to have rapid response to selection due to the presence of standing variation (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Falconer and Mackay 1996</a>), and the finding corroborates with other studies on long-term selection projects (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Beissinger <em>et al.</em> 2014</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Hirsch <em>et al.</em> 2014</a>) as well as historical genomic analysis (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">van Heerwaarden <em>et al.</em> 2012</a>). It should be noted that some soft sweeps may be incomplete sweeps that simply have not risen to high enough frequencies to dramatically change diversity; indeed, in certain contexts incomplete sweeps may actually increase diversity before decreasing it.</p><p id="p-57">Interestingly, many regions exhibited a substantial reduction in diversity while not showing a resultant increase in Δf (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). This signal persisted at a similar scale when low diversity ancestral regions (π<em><sub>a</sub></em> &lt; 0.1) were removed (data not shown) (generally, there was only a very weak correlation between reduction in diversity and the ancestral diversity level). This result indicates that a genomic region has become fixed without a substantial change in constituent allele frequencies. We refer to these windows as “haplotype sneaks” for the remainder of the manuscript (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-5-6" class="xref-table" href="#T5">Table 5</a>). Three possible origins of haplotype sneaks are: 1) an initially high frequency haplotype goes to fixation, 2) a rare untyped variant becomes a target of strong selection, or 3) recombination creates a rare haplotype that sweeps through the population.</p><p id="p-58">To differentiate Type 1 from Type 2 or 3 above, we tabulated all possible 50-marker-long haplotypes (in 10 marker increments) across the genome for each population (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). We tracked the change in these haplotypes from the pre-1970s sample to the 2000s sample. In addition, we overlaid pairwise diversity measures, maximum allele and haplotype changes within a window, and average length of shared haplotypes within the samples. Entire plots for all chromosomes for all populations are available as supplemental material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS2.zip">File S2</a>). Across all populations, we found 15 examples of Type 2|3 regions, although 73% occurred in the MG 0-I population. As an example, a ∼500 marker-wide region from chromosome 11 of MG 0-I has three subregions, with haplotypes that changed more rapidly than any of the underlying markers (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). This region also shows many of the hallmarks of a hard-sweep, except that there are no dramatic changes of any single marker frequency. As emphasized above, such signatures may be indicative of selection on a recombinant <em>or</em> on a rare, untyped variant that has become selectively advantageous.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Haplotype spectra associated with selection. Top two panels within each subfigure show the frequency of each 50 marker haplotype, in 10 marker increments, along a chromosome for pre-1970s and 2000s samples in a given population. Markers are numbered 1 through the last marker on the chromosome in order of physical position (File S4). Haplotypes are ordered in the top panel, purple to crimson, based on frequency in pre-1970s sample. All additional haplotypes are colored gray. The same haplotype in both samples, pre-1970s and 2000s, will have the same color, excepting gray haplotypes. Note that shared colors left to right along the chromosomes could represent unlinked haplotypes although similar frequencies (and frequency changes) generally suggest linkage. The third panel shows average pairwise difference (&#x3C0;) for 50 marker windows; lines are colored based on indicated samples. In the fourth panel, blue lines indicate the frequency change of the most rapidly changing haplotype within a window. Orange lines indicate the average change of the three most rapidly changing alleles in a window. Haplotype change is based on the first (pre-1970s) and last (2000s) samples, whereas allele change is based on all sampled decade groupings (see Materials and Methods). The last panel depicts the average shared haplotype length (H) around a marker for pre-1970s (purple) and 2000s (yellow) samples. Gray dotted lines throughout indicate the median of values in a panel, while red lines, when present, indicate genome-wide thresholds described in Table 5 and shown in Figure 4. See D for annotation of figure elements. (A) Section of chromosome 11 from population MG 0-I. (B) Section of chromosome 5 from MG III-IV. (C) Section of chromosome 19 from MG V+. (D) Entire chromosome 1 from MG 0-I." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1448872788" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Haplotype spectra associated with selection. Top two panels within each subfigure show the frequency of each 50 marker haplotype, in 10 marker increments, along a chromosome for pre-1970s and 2000s samples in a given population. Markers are numbered 1 through the last marker on the chromosome in order of physical position (File S4). Haplotypes are ordered in the top panel, purple to crimson, based on frequency in pre-1970s sample. All additional haplotypes are colored gray. The same haplotype in both samples, pre-1970s and 2000s, will have the same color, excepting gray haplotypes. Note that shared colors left to right along the chromosomes could represent unlinked haplotypes although similar frequencies (and frequency changes) generally suggest linkage. The third panel shows average pairwise difference (&#x3C0;) for 50 marker windows; lines are colored based on indicated samples. In the fourth panel, blue lines indicate the frequency change of the most rapidly changing haplotype within a window. Orange lines indicate the average change of the three most rapidly changing alleles in a window. Haplotype change is based on the first (pre-1970s) and last (2000s) samples, whereas allele change is based on all sampled decade groupings (see &lt;em&gt;Materials and Methods&lt;/em&gt;). The last panel depicts the average shared haplotype length (H) around a marker for pre-1970s (purple) and 2000s (yellow) samples. Gray dotted lines throughout indicate the median of values in a panel, while red lines, when present, indicate genome-wide thresholds described in Table 5 and shown in Figure 4. See &lt;em&gt;D&lt;/em&gt; for annotation of figure elements. (A) Section of chromosome 11 from population MG 0-I. (B) Section of chromosome 5 from MG III-IV. (C) Section of chromosome 19 from MG V+. (D) Entire chromosome 1 from MG 0-I.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/9/2693/F5.medium.gif" width="295" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/9/2693/F5.medium.gif" width="295" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/9/2693/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469092" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-59" class="first-child">Haplotype spectra associated with selection. Top two panels within each subfigure show the frequency of each 50 marker haplotype, in 10 marker increments, along a chromosome for pre-1970s and 2000s samples in a given population. Markers are numbered 1 through the last marker on the chromosome in order of physical position (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS4.txt">File S4</a>). Haplotypes are ordered in the top panel, purple to crimson, based on frequency in pre-1970s sample. All additional haplotypes are colored gray. The same haplotype in both samples, pre-1970s and 2000s, will have the same color, excepting gray haplotypes. Note that shared colors left to right along the chromosomes could represent unlinked haplotypes although similar frequencies (and frequency changes) generally suggest linkage. The third panel shows average pairwise difference (π) for 50 marker windows; lines are colored based on indicated samples. In the fourth panel, blue lines indicate the frequency change of the most rapidly changing haplotype within a window. Orange lines indicate the average change of the three most rapidly changing alleles in a window. Haplotype change is based on the first (pre-1970s) and last (2000s) samples, whereas allele change is based on all sampled decade groupings (see <em>Materials and Methods</em>). The last panel depicts the average shared haplotype length (H) around a marker for pre-1970s (purple) and 2000s (yellow) samples. Gray dotted lines throughout indicate the median of values in a panel, while red lines, when present, indicate genome-wide thresholds described in <a id="xref-table-wrap-5-7" class="xref-table" href="#T5">Table 5</a> and shown in <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>. See <em>D</em> for annotation of figure elements. (A) Section of chromosome 11 from population MG 0-I. (B) Section of chromosome 5 from MG III-IV. (C) Section of chromosome 19 from MG V+. (D) Entire chromosome 1 from MG 0-I.</p><div class="sb-div caption-clear"></div></div></div><p id="p-60">Across all populations and genomic windows, the ratio of greatest-haplotype-change to greatest-individual-allele-change for haplotype sneaks is 18% greater than neutral windows as defined in <a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a> (p-value = &lt;10<sup>−6</sup>). The inclusion of neutral regions or regions selected for single alleles will reduce this ratio. Thus, the median value of 0.86 among haplotype sneaks, though higher than neutral, does not exceed 1. Moreover, haplotype change was assessed based on pre-1970s samples relative to 2000s samples, whereas Δf is only evaluated prior to fixation across all decades and, therefore, can have a higher value even if total change is the same. Still, composite graphs across all populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS2.zip">File S2</a>) rarely reveal examples, such as <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>, of a haplotype sneak being associated with a striking change in any haplotype frequency. More commonly, haplotype sneaks are simply related to the fixation of large chromosomal regions within a population over time.</p><p id="p-61">Using haplotype spectrum plots, we also interrogated soft sweeps, which were the most common mode of selection detected in our data (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-5-8" class="xref-table" href="#T5">Table 5</a>). For example, an allele on chromosome 5 in population MG III-IV changes rapidly, but there are effectively no other signals indicating selection on this region (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a>). This is suggestive of selection on standing variation, in that the locus under selection had already extensively recombined with other linked segregating loci prior to the change in selective regime. Indeed, marker-level analysis indicates that three distinct haplotypes, all containing the most rapidly changing markers, rose at comparable rates (not shown).</p><p id="p-62">Haplotype spectra plots from MG V+ population (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS2.zip">File S2</a>) show that large genomic tracks can have increases in π and decreases in H relative to pre-1970s cultivars, as was suggested by <a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a>. When averaged across the genome, H was somewhat reduced in both MG 0-I and MG V+, from a median value of 107–86 and 88–68, respectively. Thus, the reductions in H suggest that LD associated with the early population structure in the pre-1970s sample was steadily lost as a result of germplasm sharing and hybridization across breeding programs throughout the second half of the 20th century. The median genome-wide log<sub>2</sub> diversity ratios were slight: –0.05 and –0.01 in MG 0-I and V+ populations, respectively, likely reflecting a balance between increased diversity related to recombination and reduced diversity related to selection. In contrast, the MG III-IV population exhibited a genome-wide increase in H from 77 to 112 in pre-1970s populations relative to 2000s, and a sharper reduction in diversity with a median log<sub>2</sub> ratio of –0.19.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><div id="sec-18" class="subsection"><h3>Can we utilize selected alleles across populations?</h3><p id="p-63">The genetic diversity in modern cultivars is fairly representative of that found in North American ancestors (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Hyten <em>et al.</em> 2006</a>). This finding is correct based on diversity across all North American elite cultivars, yet the population as a whole is also highly structured (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). In this regard, the effective diversity available to regional breeding programs is much narrower. In addition, the fractional contribution of ancestors within each population is very uneven (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>). Still, after initial population structure was established, diversity has generally been maintained (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS2.zip">File S2</a>), although in many cases selection has reduced the diversity locally (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5, A–D</a>). It remains a standing challenge in the breeding community as to how to retain beneficial alleles already present in breeding germplasm while also exploring the utility of exotic material in elite backgrounds. A major method to address this goal involves developing near-isogenic lines (NILs) that introduce genes from exotic material (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Imai <em>et al.</em> 2013</a>). Since this approach is very laborious and costly, we hope our results will help breeders to more effectively choose regions that might benefit from the introduction of exotic alleles and to identify parents that would deliver those alleles.</p><p id="p-64">Anecdotal evidence among soybean breeders has long held that elite Northern by Southern crosses rarely result in agronomically valuable progeny. Our population structure analysis supports these observations, in that admixed lines of MG V+ and either MG 0-I or MG III-IV are rare (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). This agronomic “incompatibility” is to a large degree explained by the low probability of accumulating the appropriate maturity genes together with a transgressive (or even average) segregant for yield (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jiang <em>et al.</em> 2014</a>). Epistatic effects may also play a role with regard to simpler traits such as lodging that can have a substantial impact on yield. Our results in identifying regions under breeding selection suggest an experimental program for using marker-assisted introgression in order to test transferable yield alleles across these populations. We have produced a list based on whether an allele’s frequency change in one population exceeds thresholds given in <a id="xref-table-wrap-5-9" class="xref-table" href="#T5">Table 5</a> and if the same allele is fixed in the opposite direction of selection in at least one other population (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). While some of these alleles were fixed early in breeding (Δ<em>f</em> = ∼0, final freq. = ∼0 or ∼1 in <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>), most were still segregating after the 1970s. Those that were fixed prior to our sampling may represent regions that were responsible for founder success. Alternatively, they may have simply been lost by chance; in which case, they could potentially be reintroduced through marker-assisted selection to improve the recipient population. Whatever the effect on yield, the result of such an introgression would be interesting: either the effect is positive and the allele gives a generic yield benefit, or the effect is negative or neutral and the allele was selected in a distinct population because it is specifically advantageous to a particular environment or genetic background.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469070/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469070/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Tagging markers for haplotype blocks putatively selected in one population but fixed in the opposite direction in another</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Detecting selection within individual populations</h3><p id="p-70">Detection of selection is complicated by the fact that neutral allele frequencies also change over time in finite populations. Advantageous alleles are expected to change frequency more rapidly over many generations than neutral alleles. Though we can assess this rate of allele frequency change in this study, it remains a challenge to determine if the extreme values that we observe in allele frequency change could be predicted by genetic drift alone. The distribution of a given statistic under neutral expectation can be solved or simulated using the estimated effective population size (N<sub>e</sub>). As described in the <em>Results</em>, it is likely inappropriate to assume that the majority of loci are behaving in a neutral fashion in crop populations under active selection for a complex trait. This is not only because of selection, but because of the high LD present in these populations. Therefore, establishing an appropriate null model remains extremely challenging. Even in an experimental population with known demographics, selection and other modes of nonrandom mating can result in a twofold reduction in N<sub>e</sub> estimates calculated directly from demographic parameters (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Beissinger <em>et al.</em> 2014</a>).</p><p id="p-71">There are numerous statistics available for detecting selection. These different statistics are sensitive to different evolutionary time-scales and to how the data are structured (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Vitti <em>et al.</em> 2013</a>). For example, F<sub>st</sub> values are often used to detect selection since neutral loci will estimate a single F<sub>st</sub> value indicative of population divergence and migration, while selected alleles will have a variety of F<sub>st</sub> values that are greater or less than the neutral F<sub>st</sub>, depending on the type of selection (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lewontin and Krakauer 1973</a>). In modern incarnations, the neutral F<sub>st</sub> estimate and its variance are derived from a genome-wide set of markers (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akey <em>et al.</em> 2002</a>). While the statistic has shown some efficacy, F<sub>st</sub> is generally measured between two populations or two timepoints. Recently, the genotyping of experimental populations has generated interest in algorithms designed to exploit data from samples taken at multiple timepoints for a single population (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bank <em>et al.</em> 2014</a>). Herein, we used a simple regression approach to estimate allele frequency change prior to fixation as a function of time, Δ<em>f</em>. In simulations, this simplistic approach had more power than F<sub>st</sub> (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>) and both statistics are more powerful than attempts to directly estimate the selection coefficient. The time-serial approach also appears to be more sensitive to selection on standing variation than average shared haplotype length (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5B</a>). Though they are both dependent on the change in allele frequency prior to fixation, the advantage of Δf over F<sub>st</sub> is related to fact that F<sub>st</sub> is ignoring the time-scale over which the change occurs. In simulations, both measures generally had a consistent neutral distribution regardless of initial allele frequency (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS5.zip">File S5</a>).</p><p id="p-72">It has long been appreciated that simultaneous selection at multiple loci can dramatically affect selection efficacy (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fisher 1930</a>). In the context of artificial selection acting on standing variation in a complex trait, it may be quite common for two or more loci with substantial independent effects to be in tight linkage, and these could either be in negative or positive phase. If in negative phase with comparable effects, the Hill–Robertson effect will dominate (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hill and Robertson 1966</a>). If the key recombinant occurs prior to drift removing one of the beneficial alleles, the rare haplotype will rapidly rise in frequency. Such a scenario may explain the observed excess of recombined parental haplotypes in the progeny of some breeding pedigrees (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lorenzen <em>et al.</em> 1996</a>). Importantly, this change may not have nearly as comparable an effect on the underlying allele frequencies, although they too should rise. We considered if selection on such key recombinations was one of the underlying causes of the haplotype sneaks that we observed (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-5-11" class="xref-table" href="#T5">Table 5</a>).</p><p id="p-73"><a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5D</a> depicts what might at first appear to be a classical hard sweep on chromosome 1 in the MG 0-I population; diversity is substantially reduced, allele frequencies show a rapid change, and, as should follow diversity reductions, the average shared-haplotype length for that region becomes longer. The haplotype spectra indicate that a haplotype initially present at low frequencies rose to ∼78% by the 2000s. Again, this suggests a hard sweep. Yet, what is less explicable is that the purple haplotype, which was common in the pre-1970s, also rose rapidly to near fixation by the 2000s. Indeed, the most rapidly changing marker falls within this region. We hypothesize that the changes in these two regions were not coincidental, and that the purple haplotype had initially been linked to a deleterious region that was inhibiting its rise to fixation. Marker-level analysis across this region indicated that the gray haplotype in the 2000s sample was in fact a recombinant between the green and purple haplotypes that appeared very early in MG 0-I breeding (data not shown). Unfortunately, the green/gray haplotype only appeared in this early sample in linkage with the purple haplotype, and so we cannot rule out that selection was simply on a rare allele that was linked to the purple haplotype by chance (and that the position of peak Δf was also due to chance fluctuation in the unlinked purple haplotype frequency). As emphasized by this example, even signatures of “hard sweeps” can have multiple interpretations when selection is acting on numerous QTL simultaneously. High coverage resequencing can be used to identify and characterize rare variants in a sample and will allow more precise definition of such sweeps. Still, even with resequencing data, our results indicate that the composite perspective of both haplotype and single-marker analyses could be critical in interpreting the results of artificial and natural selection research. The further integration of these methods with time-series models should be a fruitful aim for future studies.</p><p id="p-74">As made clear by this study and many others, it can be very difficult to define an appropriate null distribution for selection. As it eliminates the need to estimate a null distribution from the tested data, perhaps the most useful methodology for understanding selection in crop improvement will involve analyzing genome-wide marker data for hundreds of members of known breeding pedigrees in which resultant progeny were the product of selection (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sebastian <em>et al.</em> 1995</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jannink <em>et al.</em> 2001</a>). Many crop communities possess “immortalized” genotypes of milestone cultivars and derived lines. We expect these datasets to be enlightening not only for applied breeding but for evolutionary biology as well.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-75">We thank Roger Boerma and Tommy Carter for advice on the manuscript. We also thank Emily Jackson and Tatyana Nienow for help in curating <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1/FileS1.zip">File S1</a> and <a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>. We wish to thank the three anonymous reviewers of this manuscript for their insights and suggestions. Funding for this research was provided by the United Soybean Board.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-19"><p id="p-76">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029215/-/DC1</a></p></li><li class="fn" id="fn-20"><p id="p-77"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-04-11"><span class="received-label">Received </span>April 11, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-06-14"><span class="accepted-label">Accepted </span>June 14, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Vaughn and Li</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.1" data-doi="10.1101/gr.631202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shriver</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Interrogating a high-density SNP map for signatures of natural selection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1805</span>–<span class="cit-lpage">1814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DAkey%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D12%26rft.issue%253D12%26rft.spage%253D1805%26rft.epage%253D1814%26rft.atitle%253DInterrogating%2Ba%2BHigh-Density%2BSNP%2BMap%2Bfor%2BSignatures%2Bof%2BNatural%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.631202%26rft_id%253Dinfo%253Apmid%252F12466284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMi8xODA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.2" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.3" data-doi="10.1016/j.tig.2014.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bank</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrer-Admettla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Thinking too positive? Revisiting current methods of population genetic selection inference.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D540%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.09.010%26rft_id%253Dinfo%253Apmid%252F25438719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25438719&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.4" data-doi="10.1534/genetics.113.160655"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A genome-wide scan for evidence of selection in a maize population under long-term artificial selection for ear number.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160655%26rft_id%253Dinfo%253Apmid%252F24381334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.5" data-doi="10.1534/genetics.107.085019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bollback</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">York</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimation of 2Nes from temporal allele frequency data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBollback%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D179%26rft.issue%253D1%26rft.spage%253D497%26rft.epage%253D502%26rft.atitle%253DEstimation%2Bof%2B2Nes%2BFrom%2BTemporal%2BAllele%2BFrequency%2BData%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085019%26rft_id%253Dinfo%253Apmid%252F18493066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc5LzEvNDk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.6" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTkvMjYzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2693.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hymowitz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004a</span> <span class="cit-article-title">Biogeography, local adaptation, Vavilov, and genetic diversity in soybean</span>, pp. <span class="cit-fpage">47</span>–<span class="cit-lpage">59</span> in <span class="cit-source"><em>Biological Resources and Migration</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Werner</span>  <span class="cit-name-given-names">D.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2693.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sneller</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span><span class="cit-comment">b</span> <span class="cit-article-title">Genetic diversity in soybean</span>, pp. <span class="cit-fpage">303</span>–<span class="cit-lpage">416</span> in <span class="cit-source"><em>Soybeans: Improvement</em>, <em>Production</em>, <em>and Uses</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Boerma</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Specht</span>  <span class="cit-name-given-names">J. E.</span></span>. <span class="cit-publ-name">American Society of Agronomy</span>, <span class="cit-publ-loc">Madison</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.9" data-doi="10.1016/j.cell.2006.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The molecular genetics of crop domestication.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1309</span>–<span class="cit-lpage">1321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDoebley%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D127%26rft.issue%253D7%26rft.spage%253D1309%26rft.epage%253D1321%26rft.atitle%253DThe%2Bmolecular%2Bgenetics%2Bof%2Bcrop%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.12.006%26rft_id%253Dinfo%253Apmid%252F17190597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17190597&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243690000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2693.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source"><em>Introduction to Quantitative Genetics</em></span>, <span class="cit-publ-name">Benjamin Cummings</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.11" data-doi="10.1534/genetics.113.158220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kryazhimskiy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identifying signatures of selection in genetic time series.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158220%26rft_id%253Dinfo%253Apmid%252F24318534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzIvNTA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2693.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-source"><em>The Genetical Theory of Natural Selection: A Complete Variorum Edition</em></span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.13" data-doi="10.1111/1755-0998.12280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">WFABC: a Wright–Fisher ABC-based approach for inferring effective population sizes and selection coefficients from time-sampled data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12280%26rft_id%253Dinfo%253Apmid%252F24834845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24834845&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.14" data-doi="10.2135/cropsci1993.0011183X003300030038x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gizlice</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Genetic diversity in North American soybean: I. Multivariate analysis of founding stock and relation to coefficient of parentage.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">614</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DGizlice%26rft.auinit1%253DZ.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D614%26rft.epage%253D620%26rft.atitle%253DGenetic%2BDiversity%2Bin%2BNorth%2BAmerican%2BSoybean%253A%2BI.%2BMultivariate%2BAnalysis%2Bof%2BFounding%2BStock%2Band%2BRelation%2Bto%2BCoefficient%2Bof%2BParentage%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1993.0011183X003300030038x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1993.0011183X003300030038x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LM11100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.15" data-doi="10.2135/cropsci1994.0011183X003400050001x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gizlice</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic base for North American public soybean cultivars released between 1947 and 1988.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DGizlice%26rft.auinit1%253DZ.%26rft.volume%253D34%26rft.issue%253D5%26rft.spage%253D1143%26rft.epage%253D1151%26rft.atitle%253DGenetic%2BBase%2Bfor%2BNorth%2BAmerican%2BPublic%2BSoybean%2BCultivars%2BReleased%2Bbetween%2B1947%2Band%2B1988%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1994.0011183X003400050001x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1994.0011183X003400050001x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PF01600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2693.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source"><em>Population Genetics</em></span>, <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.17" data-doi="10.1111/nph.13626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lightfoot</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Domestication footprints anchor genomic regions of agronomic importance in soybeans.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">871</span>–<span class="cit-lpage">884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D209%26rft.spage%253D871%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13626%26rft_id%253Dinfo%253Apmid%252F26479264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26479264&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.18" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.19" data-doi="10.1534/genetics.114.167155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Insights into the effects of long-term artificial selection on seed size in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.167155%26rft_id%253Dinfo%253Apmid%252F25037958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvNDA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.20" data-doi="10.1534/genetics.115.174904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hui</span>  <span class="cit-name-given-names">T.-Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Estimating effective population size from temporally spaced samples with a novel, efficient maximum-likelihood algorithm.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.174904%26rft_id%253Dinfo%253Apmid%252F25747459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzEvMjg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.21" data-doi="10.1007/BF02904196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hymowitz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harlan</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Introduction of soybean to North America by Samuel Bowen in 1765.</span> <abbr class="cit-jnl-abbrev">Econ. Bot.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcon.%2BBot.%26rft.volume%253D37%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02904196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02904196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983RQ17700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.22" data-doi="10.1073/pnas.0604379103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">I.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Impacts of genetic bottlenecks on soybean genome diversity.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">16666</span>–<span class="cit-lpage">16671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0604379103%26rft_id%253Dinfo%253Apmid%252F17068128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzQ1LzE2NjY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.23" data-doi="10.1007/s11032-013-9855-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Imai</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimball</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conway</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeater</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Validation of yield-enhancing quantitative trait loci from a low-yielding wild ancestor of rice.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D32%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-013-9855-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-013-9855-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.24" data-doi="10.1016/S1360-1385(01)02017-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Using complex plant pedigrees to map valuable genes.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DJannink%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D337%26rft.epage%253D342%26rft.atitle%253DUsing%2Bcomplex%2Bplant%2Bpedigrees%2Bto%2Bmap%2Bvaluable%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1360-1385%252801%252902017-9%26rft_id%253Dinfo%253Apmid%252F11495765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1360-1385(01)02017-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11495765&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170479900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.25" data-doi="10.1371/journal.pone.0106042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Allelic combinations of soybean maturity loci E1, E2, E3 and E4 result in diversity of maturity and adaptation to different latitudes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e106042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De106042%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0106042%26rft_id%253Dinfo%253Apmid%252F25162675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0106042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25162675&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.26" data-doi="10.1016/j.pbi.2007.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kovach</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Leveraging natural diversity: back through the bottleneck.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DKovach%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D193%26rft.epage%253D200%26rft.atitle%253DLeveraging%2Bnatural%2Bdiversity%253A%2Bback%2Bthrough%2Bthe%2Bbottleneck.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2007.12.006%26rft_id%253Dinfo%253Apmid%252F18313975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2007.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18313975&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255623300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.27" data-doi="10.1038/ng.715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">H.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1053</span>–<span class="cit-lpage">1059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLam%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D12%26rft.spage%253D1053%26rft.epage%253D1059%26rft.atitle%253DResequencing%2Bof%2B31%2Bwild%2Band%2Bcultivated%2Bsoybean%2Bgenomes%2Bidentifies%2Bpatterns%2Bof%2Bgenetic%2Bdiversity%2Band%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.715%26rft_id%253Dinfo%253Apmid%252F21076406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076406&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krakauer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Distribution of gene frequency as a test of the theory of the selective neutrality of polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D195%26rft.atitle%253DDISTRIBUTION%2BOF%2BGENE%2BFREQUENCY%2BAS%2BA%2BTEST%2BOF%2BTHE%2BTHEORY%2BOF%2BTHE%2BSELECTIVE%2BNEUTRALITY%2BOF%2BPOLYMORPHISMS%26rft_id%253Dinfo%253Apmid%252F4711903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzQvMS8xNzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzkvMjY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.29" data-doi="10.1007/s001220050363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Soybean pedigree analysis using map-based molecular markers: recombination during cultivar development.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D93%26rft.spage%253D1251%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050363%26rft_id%253Dinfo%253Apmid%252F24162537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24162537&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.30" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maynard-Smith</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haigh</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The hitch-hiking effect of a favourable gene.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.31" data-doi="10.1038/nrg3605"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purugganan</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolution of crop species: genetics of domestication and diversification.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">840</span>–<span class="cit-lpage">852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D840%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3605%26rft_id%253Dinfo%253Apmid%252F24240513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3605&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24240513&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Crop genomics: advances and applications.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D13%26rft.spage%253D85%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-6.9.2693.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Probst</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Judd</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Origin, US history and development, and world distribution</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span> in <span class="cit-source"><em>Soybean: Improvement</em>, <em>Production</em>, <em>and Uses</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">B.</span></span>. <span class="cit-publ-name">American Society of Agronomy</span>, <span class="cit-publ-loc">Madison</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.34" data-doi="10.1073/pnas.0700643104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Plant domestication, a unique opportunity to identify the genetic basis of adaptation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8641</span>–<span class="cit-lpage">8648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0700643104%26rft_id%253Dinfo%253Apmid%252F17494757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxNjoiMTA0L3N1cHBsXzEvODY0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.35" data-doi="10.1038/nature01140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">H. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Detecting recent positive selection in the human genome from haplotype structure.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">419</span>: <span class="cit-fpage">832</span>–<span class="cit-lpage">837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D419%26rft.issue%253D6909%26rft.spage%253D832%26rft.epage%253D837%26rft.atitle%253DDetecting%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome%2Bfrom%2Bhaplotype%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01140%26rft_id%253Dinfo%253Apmid%252F12397357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12397357&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178769800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.36" data-doi="10.1038/nature08670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlueter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitros</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the palaeopolyploid soybean.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchmutz%26rft.auinit1%253DJ.%26rft.volume%253D463%26rft.issue%253D7278%26rft.spage%253D178%26rft.epage%253D183%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpalaeopolyploid%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08670%26rft_id%253Dinfo%253Apmid%252F20075913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20075913&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273582700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-6.9.2693.37"><div class="cit-metadata"><cite><span class="cit-comment">Sebastian, S. A., S. V. Tingey, and M. K. Hanafey, 1995 Method to identify genetic markers that are linked to agronomically important genes. US Patent 5437697. Available at: <a href="http://www.google.co.uk/patents/US5437697">http://www.google.co.uk/patents/US5437697</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.9.2693.38" data-doi="10.1111/pbi.12249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donoughue</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cober</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajcan</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belzile</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of loci governing eight agronomic traits using a GBS-GWAS approach and validation by QTL mapping in soya bean.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D13%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12249%26rft_id%253Dinfo%253Apmid%252F25213593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25213593&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.39" data-doi="10.1371/journal.pone.0054985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quigley</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Development and evaluation of SoySNP50K, a high-density genotyping array for soybean.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e54985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De54985%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054985%26rft_id%253Dinfo%253Apmid%252F23372807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372807&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.40" data-doi="10.1093/aob/mct269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubokura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanamori</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamagata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural variation in the genes responsible for maturity loci E1, E2, E3 and E4 in soybean.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmct269%26rft_id%253Dinfo%253Apmid%252F24284817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjk6IjExMy8zLzQyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.41" data-doi="10.1073/pnas.1209275109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Historical genomics of North American maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">12420</span>–<span class="cit-lpage">12425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1209275109%26rft_id%253Dinfo%253Apmid%252F22802642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzMxLzEyNDIwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi85LzI2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.42" data-doi="10.1534/g3.114.013433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genetic architecture of seed composition in soybean is refined by genome-wide association scans across multiple populations.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2283</span>–<span class="cit-lpage">2294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013433%26rft_id%253Dinfo%253Apmid%252F25246241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTEvMjI4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOS8yNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.43" data-doi="10.1146/annurev-genet-111212-133526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitti</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossman</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Detecting natural selection in genomic data.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft.volume%253D47%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-111212-133526%26rft_id%253Dinfo%253Apmid%252F24274750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-111212-133526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24274750&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329816000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.44" data-doi="10.1186/s12864-015-1872-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyse</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic consequences of selection and genome-wide association mapping in soybean.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D671%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1872-y%26rft_id%253Dinfo%253Apmid%252F26334313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1872-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26334313&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.45" data-doi="10.1186/s12870-015-0463-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Impacts of nucleotide fixation during soybean domestication and improvement.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D15%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-015-0463-z%26rft_id%253Dinfo%253Apmid%252F25849896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-015-0463-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25849896&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.9.2693.46" data-doi="10.1038/nbt.3096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Resequencing 302 wild and cultivated accessions identifies genes related to domestication and improvement in soybean.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">408</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D408%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3096%26rft_id%253Dinfo%253Apmid%252F25643055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25643055&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F9%2F2693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/9/2693.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/9/2687" title="An Inversion Disrupting FAM134B Is Associated with Sensory Neuropathy in the Border Collie Dog Breed" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/9/2707" title="Microsporidia Intracellular Development Relies on Myc Interaction Network Transcription Factors in the Host" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/9"><span>Volume 6 Issue 9, September 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/9.cover-source.jpg?itok=nund1hRf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/9/2693.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-een4erBPUAjF34Jiwk1Gy0T9RPa-UgEEVUOasMUU4NU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/9/2693" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/9/2693" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/9/2693" class="active" data-icon-position="" data-hide-link-title="0">Genomic Signatures of North American Soybean Improvement Inform Diversity Enrichment Strategies and Clarify the Impact of Hybridization</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460291" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-HlkkTL3qjjP39MlY3r1Eq9QNCr3lBJd0-1qiHmeOFMo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2693.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/9/2693" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-8or_PkPoouN1HMzdXO6ppF1nlXYhs5c0fNZrwPoFzdw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/9/2693" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/9/2693.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460291" id="node460291--2" data-pisa="ggg;6/9/2693" data-pisa-master="ggg;g3.116.029215" data-apath="/ggg/6/9/2693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Signatures of North American Soybean Improvement Inform Diversity Enrichment Strategies and Clarify the Impact of Hybridization</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Justin N. Vaughn</span> and <span class="highwire-citation-author" data-delta="1">Zenglu Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2693-2705;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029215 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460291--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Justin N. Vaughn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop and Soil Science, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BN.%2BVaughn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaughn%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BN.%2BVaughn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zenglu Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop and Soil Science, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zenglu%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZenglu%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zli@uga.edu" class="" data-icon-position="" data-hide-link-title="0">zli@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/9/2693" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/9/2693&amp;title=Genomic%20Signatures%20of%20North%20American%20Soybean%20Improvement%20Inform%20Diversity%20Enrichment%20Strategies%20and%20Clarify%20the%20Impact%20of%20Hybridization" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/9/2693&amp;title=Genomic%20Signatures%20of%20North%20American%20Soybean%20Improvement%20Inform%20Diversity%20Enrichment%20Strategies%20and%20Clarify%20the%20Impact%20of%20Hybridization" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/9/2693&amp;title=Genomic%20Signatures%20of%20North%20American%20Soybean%20Improvement%20Inform%20Diversity%20Enrichment%20Strategies%20and%20Clarify%20the%20Impact%20of%20Hybridization" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/9/2693&amp;text=Genomic%20Signatures%20of%20North%20American%20Soybean%20Improvement%20Inform%20Diversity%20Enrichment%20Strategies%20and%20Clarify%20the%20Impact%20of%20Hybridization" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/9/2693&amp;title=Genomic%20Signatures%20of%20North%20American%20Soybean%20Improvement%20Inform%20Diversity%20Enrichment%20Strategies%20and%20Clarify%20the%20Impact%20of%20Hybridization" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/9/2693&amp;t=Genomic%20Signatures%20of%20North%20American%20Soybean%20Improvement%20Inform%20Diversity%20Enrichment%20Strategies%20and%20Clarify%20the%20Impact%20of%20Hybridization" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/9/2693&amp;title=Genomic%20Signatures%20of%20North%20American%20Soybean%20Improvement%20Inform%20Diversity%20Enrichment%20Strategies%20and%20Clarify%20the%20Impact%20of%20Hybridization" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/9/2693&amp;title=Genomic%20Signatures%20of%20North%20American%20Soybean%20Improvement%20Inform%20Diversity%20Enrichment%20Strategies%20and%20Clarify%20the%20Impact%20of%20Hybridization" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460291/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460291" id="node460291--3" data-pisa="ggg;6/9/2693" data-pisa-master="ggg;g3.116.029215" data-apath="/ggg/6/9/2693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Signatures of North American Soybean Improvement Inform Diversity Enrichment Strategies and Clarify the Impact of Hybridization</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Justin N. Vaughn</span> and <span class="highwire-citation-author" data-delta="1">Zenglu Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2693-2705;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029215 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460291--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Justin N. Vaughn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop and Soil Science, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BN.%2BVaughn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaughn%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BN.%2BVaughn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zenglu Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop and Soil Science, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zenglu%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZenglu%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zli@uga.edu" class="" data-icon-position="" data-hide-link-title="0">zli@uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2693&amp;rft.epage=2705&amp;rft.atitle=Genomic%20Signatures%20of%20North%20American%20Soybean%20Improvement%20Inform%20Diversity%20Enrichment%20Strategies%20and%20Clarify%20the%20Impact%20of%20Hybridization&amp;rft.volume=6&amp;rft.issue=9&amp;rft.date=2016-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Vaughn%2C+Justin+N.&amp;rft.au=Li%2C+Zenglu"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460291/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460291/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460291/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460291/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460291/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460291/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460291/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460291/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460291/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460291/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460291/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460291/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/9/2693" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/9/2693.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/9/2693.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/9/2693.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/9/2693.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
