<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/9/2963" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/9/2963.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/9/2963.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/9/2963.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/9/2963/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/9/2963.atom" />
<meta name="HW.pisa" content="ggg;9/9/2963" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (Avena sativa L.)" />
<meta name="DC.Identifier" content="10.1534/g3.119.400228" />
<meta name="DC.Date" content="2019-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Carlson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Oat ( Avena sativa L.) has a high concentration of oils, comprised primarily of healthful unsaturated oleic and linoleic fatty acids. To accelerate oat plant breeding efforts, we sought to identify loci associated with variation in fatty acid composition, defined as the types and quantities of fatty acids. We genotyped a panel of 500 oat cultivars with genotyping-by-sequencing and measured the concentrations of ten fatty acids in these oat cultivars grown in two environments. Measurements of individual fatty acids were highly correlated across samples, consistent with fatty acids participating in shared biosynthetic pathways. We leveraged these phenotypic correlations in two multivariate genome-wide association study (GWAS) approaches. In the first analysis, we fitted a multivariate linear mixed model for all ten fatty acids simultaneously while accounting for population structure and relatedness among cultivars. In the second, we performed a univariate association test for each principal component (PC) derived from a singular value decomposition of the phenotypic data matrix. To aid interpretation of results from the multivariate analyses, we also conducted univariate association tests for each trait. The multivariate mixed model approach yielded 148 genome-wide significant single-nucleotide polymorphisms (SNPs) at a 10% false-discovery rate, compared to 129 and 73 significant SNPs in the PC and univariate analyses, respectively. Thus, explicit modeling of the correlation structure between fatty acids in a multivariate framework enabled identification of loci associated with variation in seed fatty acid concentration that were not detected in the univariate analyses. Ultimately, a detailed characterization of the loci underlying fatty acid variation can be used to enhance the nutritional profile of oats through breeding." />
<meta name="DC.Contributor" content="Maryn O. Carlson" />
<meta name="DC.Contributor" content="Gracia Montilla-Bascon" />
<meta name="DC.Contributor" content="Owen A. Hoekenga" />
<meta name="DC.Contributor" content="Nicholas A. Tinker" />
<meta name="DC.Contributor" content="Jesse Poland" />
<meta name="DC.Contributor" content="Matheus Baseggio" />
<meta name="DC.Contributor" content="Mark E. Sorrells" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="DC.Contributor" content="Michael A. Gore" />
<meta name="DC.Contributor" content="Trevor H. Yeats" />
<meta name="article:published_time" content="2019-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (Avena sativa L.)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Oat (&lt;i&gt;Avena sativa&lt;/i&gt; L.) has a high concentration of oils, comprised primarily of healthful unsaturated oleic and linoleic fatty acids. To accelerate oat plant breeding efforts, we sought to identify loci associated with variation in fatty acid composition, defined as the types and quantities of fatty acids. We genotyped a panel of 500 oat cultivars with genotyping-by-sequencing and measured the concentrations of ten fatty acids in these oat cultivars grown in two environments. Measurements of individual fatty acids were highly correlated across samples, consistent with fatty acids participating in shared biosynthetic pathways. We leveraged these phenotypic correlations in two multivariate genome-wide association study (GWAS) approaches. In the first analysis, we fitted a multivariate linear mixed model for all ten fatty acids simultaneously while accounting for population structure and relatedness among cultivars. In the second, we performed a univariate association test for each principal component (PC) derived from a singular value decomposition of the phenotypic data matrix. To aid interpretation of results from the multivariate analyses, we also conducted univariate association tests for each trait. The multivariate mixed model approach yielded 148 genome-wide significant single-nucleotide polymorphisms (SNPs) at a 10% false-discovery rate, compared to 129 and 73 significant SNPs in the PC and univariate analyses, respectively. Thus, explicit modeling of the correlation structure between fatty acids in a multivariate framework enabled identification of loci associated with variation in seed fatty acid concentration that were not detected in the univariate analyses. Ultimately, a detailed characterization of the loci underlying fatty acid variation can be used to enhance the nutritional profile of oats through breeding.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/09/01" />
<meta name="citation_mjid" content="ggg;9/9/2963" />
<meta name="citation_id" content="9/9/2963" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/9/2963" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/9/2963.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/9/2963.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/9/2963.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400228" />
<meta name="citation_pmid" content="31296616" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2963" />
<meta name="citation_lastpage" content="2975" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Maryn O. Carlson" />
<meta name="citation_author_institution" content="Plant Breeding &amp; Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853" />
<meta name="citation_author_email" content="mocarlson@uchicago.edu" />
<meta name="citation_author_email" content="thy6@cornell.edu" />
<meta name="citation_author" content="Gracia Montilla-Bascon" />
<meta name="citation_author_institution" content="CSIC, Institute for Sustainable Agriculture, Córdoba, Spain" />
<meta name="citation_author" content="Owen A. Hoekenga" />
<meta name="citation_author_institution" content="Cayuga Genetics Consulting Group LLC, Ithaca, NY 14850" />
<meta name="citation_author" content="Nicholas A. Tinker" />
<meta name="citation_author_institution" content="Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ottawa, ON, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2452-4779" />
<meta name="citation_author" content="Jesse Poland" />
<meta name="citation_author_institution" content="Wheat Genetics Resource Center, Department of Plant Pathology, Kansas State University, Manhattan, KS 66506, and" />
<meta name="citation_author" content="Matheus Baseggio" />
<meta name="citation_author_institution" content="Plant Breeding &amp; Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853" />
<meta name="citation_author" content="Mark E. Sorrells" />
<meta name="citation_author_institution" content="Plant Breeding &amp; Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="Plant Breeding &amp; Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853" />
<meta name="citation_author_institution" content="R.W. Holley Center for Agriculture and Health, US Department of Agriculture, Agricultural Research Service, Ithaca, NY 14853" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4849-628X" />
<meta name="citation_author" content="Michael A. Gore" />
<meta name="citation_author_institution" content="Plant Breeding &amp; Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6896-8024" />
<meta name="citation_author" content="Trevor H. Yeats" />
<meta name="citation_author_institution" content="Plant Breeding &amp; Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853" />
<meta name="citation_author_email" content="mocarlson@uchicago.edu" />
<meta name="citation_author_email" content="thy6@cornell.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6287-952X" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=J. Aitchison;citation_title=Principal component analysis of compositional data;citation_pages=57-65;citation_volume=70;citation_year=1983;citation_issue=1;citation_doi=10.1093/biomet/70.1.57" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=H. Aschard;citation_author=B. J. Vilhjálmsson;citation_author=N. Greliche;citation_author=P.-E. Morange;citation_author=D.-A. Trégouët;citation_title=Maximizing the Power of Principal-Component Analysis of Correlated Phenotypes in Genome-wide Association Studies.;citation_pages=662-676;citation_volume=94;citation_year=2014;citation_pmid=24746957;citation_doi=10.1016/j.ajhg.2014.03.016" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=F. G. Asoro;citation_author=M. A. Newell;citation_author=W. D. Beavis;citation_author=M. P. Scott;citation_author=N. A. Tinker;citation_title=Genomic, Marker-Assisted, and Pedigree-BLUP Selection Methods for β-Glucan Concentration in Elite Oat.;citation_pages=1894;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.09.0526" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=A. Banaś;citation_author=H. Debski;citation_author=W. Banaś;citation_author=W. K. Heneen;citation_author=A. Dahlqvist;citation_title=Lipids in grain tissues of oat (Avena sativa): differences in content, time of deposition, and fatty acid composition.;citation_volume=58;citation_year=2007;citation_pmid=17586606;citation_doi=10.1093/jxb/erm125" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotech. J.;citation_author=W. A. Bekele;citation_author=C. P. Wight;citation_author=S. Chao;citation_author=C. J. Howarth;citation_author=N. A. Tinker;citation_title=Haplotype-based genotyping-by-sequencing in oat genome research.;citation_pages=1452-1463;citation_volume=16;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Soc. Ser. B Methodol.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J. R. Stat.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_title=ASReml-R reference manual. The State of Queensland, Department of Primary Industries and Fisheries, Brisbane.;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=European journal of nutrition;citation_journal_abbrev=Eur J Nutr;citation_author=M. S. Butt;citation_author=M. Tahir-Nadeem;citation_author=M. K. I. Khan;citation_author=R. Shabir;citation_author=M. S. Butt;citation_title=Oat: unique among the cereals.;citation_pages=68-79;citation_volume=47;citation_year=2008;citation_issue=2;citation_pmid=18301937;citation_doi=10.1007/s00394-008-0698-7" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=C. T. Butts;citation_title=A statnet Tutorial.;citation_pages=1-27;citation_volume=24;citation_year=2008;citation_issue=9;citation_pmid=18612375" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. B. Cahoon;citation_author=Y. Lindqvist;citation_author=G. Schneider;citation_author=J. Shanklin;citation_title=Redesign of soluble fatty acid desaturases from plants for altered substrate specificity and double bond position;citation_pages=4872-4877;citation_volume=94;citation_year=1997;citation_issue=10;citation_pmid=9144157;citation_doi=10.1073/pnas.94.10.4872" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=E. B. Cahoon;citation_author=S. Shah;citation_author=J. Shanklin;citation_author=J. Browse;citation_title=A Determinant of Substrate Specificity Predicted from the Acyl-Acyl Carrier Protein Desaturase of Developing Cat's Claw Seed;citation_pages=593-598;citation_volume=117;citation_year=1998;citation_issue=2;citation_pmid=9625712;citation_doi=10.1104/pp.117.2.593" />
<meta name="citation_reference" content="citation_author=A. S. Chaffin;citation_author=Y.-F. Huang;citation_author=S. Smith;citation_author=W. A. Bekele;citation_author=E. Babiker;citation_year=2016;citation_doi=10.3835/plantgenome2015.10.0102" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. P. Cook;citation_author=M. D. McMullen;citation_author=J. B. Holland;citation_author=F. Tian;citation_author=P. Bradbury;citation_title=Genetic Architecture of Maize Kernel Composition in the Nested Association Mapping and Inbred Association Panels.;citation_volume=158;citation_year=2012;citation_pmid=22135431;citation_doi=10.1104/pp.111.185033" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=K. Esvelt Klos;citation_author=Y.-F. Huang;citation_author=W. A. Bekele;citation_author=D. E. Obert;citation_author=E. Babiker;citation_title=Population Genomics Related to Adaptation in Elite Oat Germplasm.;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2015.10.0103" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=K. Gacek;citation_author=P. E. Bayer;citation_author=I. Bartkowiak-Broda;citation_author=L. Szala;citation_author=J. Bocianowski;citation_title=Genome-Wide Association Study of Genetic Control of Seed Fatty Acid Biosynthesis in Brassica napus.;citation_pages=2062;citation_volume=7;citation_year=2017;citation_doi=10.3389/fpls.2016.02062" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Gieger;citation_author=L. Geistlinger;citation_author=E. Altmaier;citation_author=M. H. de Angelis;citation_author=F. Kronenberg;citation_title=Genetics meets metabolomics: a genome-wide association study of metabolite profiles in human serum.;citation_pages=e1000282-e1000282;citation_volume=4;citation_year=2008;citation_issue=11;citation_pmid=19043545;citation_doi=10.1371/journal.pgen.1000282" />
<meta name="citation_reference" content="citation_title=ASReml user guide release 3.0;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Food Funct.;citation_author=M. M.-L. Grundy;citation_author=A. Fardet;citation_author=S. M. Tosh;citation_author=G. T. Rich;citation_author=P. J. Wilde;citation_title=Processing of oat: the impact on oat’s cholesterol lowering effect.;citation_pages=1328-1343;citation_volume=9;citation_year=2018;citation_doi=10.1039/C7FO02006F" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Gusev;citation_author=A. Ko;citation_author=H. Shi;citation_author=G. Bhatia;citation_author=W. Chung;citation_title=Integrative approaches for large-scale transcriptome-wide association studies.;citation_pages=245-252;citation_volume=48;citation_year=2016;citation_pmid=26854917;citation_doi=10.1038/ng.3506" />
<meta name="citation_reference" content="citation_journal_title=J. Oleo Sci.;citation_author=N. B. Halima;citation_author=R. B. Saad;citation_author=B. Khemakhem;citation_author=I. Fendri;citation_author=S. Abdelkafi;citation_title=Oat (Avena sativa L.): Oil and Nutriment Compounds Valorization for Potential Use in Industrial Applications.;citation_pages=915-932;citation_volume=64;citation_year=2015;citation_doi=10.5650/jos.ess15074" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=B. T. Hizbai;citation_author=K. M. Gardner;citation_author=C. P. Wight;citation_author=R. K. Dhanda;citation_author=S. J. Molnar;citation_title=Quantitative Trait Loci Affecting Oil Content, Oil Composition, and Other Agronomically Important Traits in Oat.;citation_pages=164;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.07.0015" />
<meta name="citation_reference" content="citation_title=Plant Breeding Reviews;citation_title=Estimating and Interpreting Heritability for Plant Breeding: An Update;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=H.-Y. Hung;citation_author=C. Browne;citation_author=K. Guill;citation_author=N. Coles;citation_author=M. Eller;citation_title=The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population.;citation_pages=490-499;citation_volume=108;citation_year=2012;citation_issue=5;citation_pmid=22027895;citation_doi=10.1038/hdy.2011.103" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=E. N. Jellen;citation_author=B. S. Gill;citation_author=T. S. Cox;citation_title=Genomic in situ hybridization differentiates between A/D- and C-genome chromatin and detects intergenomic translocations in polyploid oat species (genus Avena).;citation_pages=613-618;citation_volume=37;citation_year=1994;citation_issue=4;citation_pmid=18470105;citation_doi=10.1139/g94-087" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. F. Kianian;citation_author=M. A. Egli;citation_author=R. L. Phillips;citation_author=H. W. Rines;citation_author=D. A. Somers;citation_title=Association of a major groat oil content QTL and an acetyl-CoA carboxylase gene in oat.;citation_pages=884-894;citation_volume=98;citation_year=1999;citation_doi=10.1007/s001220051147" />
<meta name="citation_reference" content="citation_journal_title=Commun. Stat. Appl. Methods;citation_author=S. Kim;citation_title=ppcor: An R Package for a Fast Calculation to Semi-partial Correlation Coefficients.;citation_pages=665-674;citation_volume=22;citation_year=2015;citation_pmid=26688802;citation_doi=10.5351/CSAM.2015.22.6.665" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Korte;citation_author=B. J. Vilhjálmsson;citation_author=V. Segura;citation_author=A. Platt;citation_author=Q. Long;citation_title=A mixed-model approach for genome-wide association studies of correlated traits in structured populations.;citation_pages=1066-1071;citation_volume=44;citation_year=2012;citation_issue=9;citation_pmid=22902788;citation_doi=10.1038/ng.2376" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=J. Krumsiek;citation_author=K. Suhre;citation_author=T. Illig;citation_author=J. Adamski;citation_author=F. J. Theis;citation_title=Gaussian graphical modeling reconstructs pathway reactions from high-throughput metabolomics data.;citation_pages=21-21;citation_volume=5;citation_year=2011;citation_pmid=21281499;citation_doi=10.1186/1752-0509-5-21" />
<meta name="citation_reference" content="citation_title=Applied linear regression models.;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Journal of Agricultural and Food Chemistry;citation_journal_abbrev=Journal of Agricultural and Food Chemistry;citation_author=S. Leonova;citation_author=T. Shelenga;citation_author=M. Hamberg;citation_author=A. V. Konarev;citation_author=I. Loskutov;citation_title=Analysis of oil composition in cultivars and wild species of oat (Avena sp.).;citation_pages=7983-7991;citation_volume=56;citation_year=2008;citation_issue=17;citation_pmid=18707115;citation_doi=10.1021/jf800761c" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. S. Ludwig;citation_author=W. C. Willett;citation_author=J. S. Volek;citation_author=M. L. Neuhouser;citation_title=Dietary fat: From foe to friend?;citation_pages=764-770;citation_volume=362;citation_year=2018;citation_issue=6416;citation_pmid=30442800;citation_doi=10.1126/science.aau2096" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. Li;citation_author=Z. Peng;citation_author=X. Yang;citation_author=W. Wang;citation_author=J. Fu;citation_title=Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels.;citation_pages=43-50;citation_volume=45;citation_year=2013;citation_pmid=23242369;citation_doi=10.1038/ng.2484" />
<meta name="citation_reference" content="citation_journal_title=THE ARABIDOPSIS BOOK;citation_author=Y. Li-Beisson;citation_author=B. Shorrosh;citation_author=F. Beisson;citation_author=M. X. Andersson;citation_author=V. Arondel;citation_pages=0161e;citation_volume=11;citation_year=2013;citation_pmid=23505340;citation_doi=10.1199/tab.0161" />
<meta name="citation_reference" content="citation_author=A. E. Lipka;citation_author=M. A. Gore;citation_author=M. Magallanes-Lundback;citation_author=A. Mesberg;citation_author=H. Lin;citation_year=2013;citation_pmid=23733887;citation_doi=10.1534/g3.113.006148" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=A. E. Lipka;citation_author=C. B. Kandianis;citation_author=M. E. Hudson;citation_author=J. Yu;citation_author=J. Drnevich;citation_title=From association to prediction: statistical methods for the dissection and selection of complex traits in plants.;citation_pages=110-118;citation_volume=24;citation_year=2015;citation_pmid=25795170;citation_doi=10.1016/j.pbi.2015.02.010" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. E. Lipka;citation_author=F. Tian;citation_author=Q. Wang;citation_author=J. Peiffer;citation_author=M. Li;citation_title=GAPIT: genome association and prediction integrated tool.;citation_volume=28;citation_year=2012;citation_pmid=22796960;citation_doi=10.1093/bioinformatics/bts444" />
<meta name="citation_reference" content="citation_journal_title=Nutr. Rev.;citation_author=M. Meydani;citation_title=Potential health benefits of avenanthramides of oats.;citation_volume=67;citation_year=2009;citation_pmid=19941618;citation_doi=10.1111/j.1753-4887.2009.00256.x" />
<meta name="citation_reference" content="citation_journal_title=J. Infrared Spectrosc.;citation_author=G. Montilla-Bascón;citation_author=P. R. Armstrong;citation_author=R. Han;citation_author=M. Sorrells;citation_title=Quantification of betaglucans, lipid and protein contents in whole oat groats (Avena sativa L.) using near infrared reflectance spectroscopy.;citation_pages=172-179;citation_volume=25;citation_year=2017;citation_doi=10.1177/0967033517709615" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. A. Newell;citation_author=F. G. Asoro;citation_author=M. P. Scott;citation_author=P. J. White;citation_author=W. D. Beavis;citation_title=Genome-wide association study for oat (Avena sativa L.) beta-glucan concentration using germplasm of worldwide origin.;citation_pages=1687-1696;citation_volume=125;citation_year=2012;citation_issue=8;citation_pmid=22865125;citation_doi=10.1007/s00122-012-1945-0" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. F. O’Reilly;citation_author=C. J. Hoggart;citation_author=Y. Pomyen;citation_author=F. C. F. Calboli;citation_author=P. Elliott;citation_title=MultiPhen: joint model of multiple phenotypes can increase discovery in GWAS.;citation_pages=e34861-e34861;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22567092;citation_doi=10.1371/journal.pone.0034861" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. A. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-Sequencing for Plant Breeding and Genetics.;citation_pages=92-102;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. K. Pritchard;citation_author=M. Przeworski;citation_title=Linkage disequilibrium in humans: models and data.;citation_pages=1-14;citation_volume=69;citation_year=2001;citation_issue=1;citation_pmid=11410837;citation_doi=10.1086/321275" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=D. Laloë;citation_author=S. Nicolas;citation_author=T. Altmann;citation_author=D. Brunel;citation_title=Maximizing the Reliability of Genomic Selection by Optimizing the Calibration Set of Reference Individuals: Comparison of Methods in Two Diverse Groups of Maize Inbreds (Zea mays L.).;citation_volume=192;citation_year=2012;citation_pmid=22865733;citation_doi=10.1534/genetics.112.141473" />
<meta name="citation_reference" content="citation_journal_title=Cereal Chem.;citation_author=M. Saastamoinen;citation_author=J. Kumpulainen;citation_author=S. Nummela;citation_title=Genetic and Environmental Variation in Oil Content and Fatty Acid Composition of Oats.;citation_pages=296-300;citation_volume=66;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Santantonio;citation_author=J.-L. Jannink;citation_author=M. Sorrells;citation_title=Homeologous Epistasis in Wheat: The Search for an Immortal Hybrid.;citation_volume=211;citation_year=2019;citation_pmid=30679260;citation_doi=10.1534/genetics.118.301851" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=G. Schwarz;citation_title=Estimating the Dimension of a Model.;citation_pages=461-464;citation_volume=6;citation_year=1978;citation_doi=10.1214/aos/1176344136" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=R. J. Snowdon;citation_author=W. Friedt;citation_title=Molecular markers in Brassica oilseed breeding: current status and future possibilities.;citation_pages=1-8;citation_volume=123;citation_year=2004;citation_doi=10.1111/j.1439-0523.2003.00968.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Stacklies;citation_author=H. Redestig;citation_author=M. Scholz;citation_author=D. Walther;citation_author=J. Selbig;citation_title=pcaMethods—a bioconductor package providing PCA methods for incomplete data.;citation_volume=23;citation_year=2007;citation_pmid=17344241;citation_doi=10.1093/bioinformatics/btm069" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Stephens;citation_title=A Unified Framework for Association Analysis with Multiple Related Phenotypes.;citation_pages=e65245;citation_volume=8;citation_year=2013;citation_doi=10.1371/journal.pone.0065245" />
<meta name="citation_reference" content="citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=G. Sun;citation_author=C. Zhu;citation_author=M. H. Kramer;citation_author=S.-S. Yang;citation_author=W. Song;citation_title=Variation explained in mixed-model association mapping.;citation_pages=333-340;citation_volume=105;citation_year=2010;citation_issue=4;citation_pmid=20145669;citation_doi=10.1038/hdy.2010.11" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=F. G. Sunstrum;citation_author=W. A. Bekele;citation_author=C. P. Wight;citation_author=W. Yan;citation_author=Y. Chen;citation_title=A genetic linkage map in southern-by-spring oat identifies multiple quantitative trait loci for adaptation and rust resistance.;citation_pages=82-94;citation_volume=138;citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=P. Tanhuanpää;citation_author=O. Manninen;citation_author=A. Beattie;citation_author=P. Eckstein;citation_author=G. Scoles;citation_title=An updated doubled haploid oat linkage map and QTL mapping of agronomic and grain quality traits from Canadian field trials.;citation_pages=289-301;citation_volume=55;citation_year=2012;citation_issue=4;citation_pmid=22443510;citation_doi=10.1139/g2012-017" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_title=Oats: Chemistry and Technology;citation_author=J. Valentine;citation_author=A. A. Cowan;citation_author=A. H. Marshall;citation_title=Oat Breeding;citation_year=2011;citation_doi=10.1094/9781891127649.002" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=J. M. VanLiere;citation_author=N. A. Rosenberg;citation_title=Mathematical properties of the r2 measure of linkage disequilibrium.;citation_pages=130-137;citation_volume=74;citation_year=2008;citation_issue=1;citation_pmid=18572214;citation_doi=10.1016/j.tpb.2008.05.006" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=Q. Yang;citation_author=H. Wu;citation_author=C.-Y. Guo;citation_author=C. S. Fox;citation_title=Analyze multivariate phenotypes in genetic association studies by combining univariate association tests.;citation_pages=444-454;citation_volume=34;citation_year=2010;citation_issue=5;citation_pmid=20583287;citation_doi=10.1002/gepi.20497" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=J. Zhang;citation_author=X. Wang;citation_author=Y. Lu;citation_author=S. J. Bhusal;citation_author=Q. Song;citation_title=Genome-wide Scan for Seed Composition Provides Insights into Soybean Quality Improvement and the Impacts of Domestication and Breeding.;citation_pages=460-472;citation_volume=11;citation_year=2018;citation_doi=10.1016/j.molp.2017.12.016" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Oil Chem. Soc.;citation_author=M. Zhou;citation_author=K. Robards;citation_author=M. Glennie-Holmes;citation_author=S. Helliwell;citation_title=Oat lipids.;citation_pages=159-169;citation_volume=76;citation_year=1999;citation_doi=10.1007/s11746-999-0213-1" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=X. Zhou;citation_author=M. Stephens;citation_title=Efficient multivariate linear mixed model algorithms for genome-wide association studies.;citation_pages=407-409;citation_volume=11;citation_year=2014;citation_pmid=24531419;citation_doi=10.1038/nmeth.2848" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Zhou;citation_author=M. Stephens;citation_title=Genome-wide efficient mixed-model analysis for association studies.;citation_pages=821-824;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706312;citation_doi=10.1038/ng.2310" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. Zhu;citation_author=B. G. Rossnagel;citation_author=H. F. Kaeppler;citation_title=Genetic Analysis of Quantitative Trait Loci for Groat Protein and Oil Content in Oat;citation_pages=254-260;citation_volume=44;citation_year=2004;citation_issue=1;citation_doi=10.2135/cropsci2004.0254" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (Avena sativa L.)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="twitter:description" content="Oat ( Avena sativa L.) has a high concentration of oils, comprised primarily of healthful unsaturated oleic and linoleic fatty acids. To accelerate oat plant breeding efforts, we sought to identify loci associated with variation in fatty acid composition, defined as the types and quantities of fatty acids. We genotyped a panel of 500 oat cultivars with genotyping-by-sequencing and measured the concentrations of ten fatty acids in these oat cultivars grown in two environments. Measurements of individual fatty acids were highly correlated across samples, consistent with fatty acids participating in shared biosynthetic pathways. We leveraged these phenotypic correlations in two multivariate genome-wide association study (GWAS) approaches. In the first analysis, we fitted a multivariate linear mixed model for all ten fatty acids simultaneously while accounting for population structure and relatedness among cultivars. In the second, we performed a univariate association test for each principal component (PC) derived from a singular value decomposition of the phenotypic data matrix. To aid interpretation of results from the multivariate analyses, we also conducted univariate association tests for each trait. The multivariate mixed model approach yielded 148 genome-wide significant single-nucleotide polymorphisms (SNPs) at a 10% false-discovery rate, compared to 129 and 73 significant SNPs in the PC and univariate analyses, respectively. Thus, explicit modeling of the correlation structure between fatty acids in a multivariate framework enabled identification of loci associated with variation in seed fatty acid concentration that were not detected in the univariate analyses. Ultimately, a detailed characterization of the loci underlying fatty acid variation can be used to enhance the nutritional profile of oats through breeding." />
<meta name="og-title" property="og:title" content="Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (Avena sativa L.)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/9/2963" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Oat ( Avena sativa L.) has a high concentration of oils, comprised primarily of healthful unsaturated oleic and linoleic fatty acids. To accelerate oat plant breeding efforts, we sought to identify loci associated with variation in fatty acid composition, defined as the types and quantities of fatty acids. We genotyped a panel of 500 oat cultivars with genotyping-by-sequencing and measured the concentrations of ten fatty acids in these oat cultivars grown in two environments. Measurements of individual fatty acids were highly correlated across samples, consistent with fatty acids participating in shared biosynthetic pathways. We leveraged these phenotypic correlations in two multivariate genome-wide association study (GWAS) approaches. In the first analysis, we fitted a multivariate linear mixed model for all ten fatty acids simultaneously while accounting for population structure and relatedness among cultivars. In the second, we performed a univariate association test for each principal component (PC) derived from a singular value decomposition of the phenotypic data matrix. To aid interpretation of results from the multivariate analyses, we also conducted univariate association tests for each trait. The multivariate mixed model approach yielded 148 genome-wide significant single-nucleotide polymorphisms (SNPs) at a 10% false-discovery rate, compared to 129 and 73 significant SNPs in the PC and univariate analyses, respectively. Thus, explicit modeling of the correlation structure between fatty acids in a multivariate framework enabled identification of loci associated with variation in seed fatty acid concentration that were not detected in the univariate analyses. Ultimately, a detailed characterization of the loci underlying fatty acid variation can be used to enhance the nutritional profile of oats through breeding." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
    <title>Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (Avena sativa L.) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VaHUYwpw19gBccwijzMwU8pDQC1yAyj1USHLAENI5Ys","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494061","apath":"\/ggg\/9\/9\/2963.atom","pisa":"ggg;9\/9\/2963","ac":{"\/ggg\/9\/9\/2963.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/9\/2963.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2963"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2963"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/9\/2963":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pLWMC24QVyLPfMwLKaZyOuESko2xAeu+nsz6H48qLWes0uBy\\\\\\\/F26UoWnaOFKwTyhP4c5GGUumSEkuZOcPwVIv0fjKWgDjI8fitsdv6DvvWpTXFW+bCGPyoK0MOOHaDqJ4JpqYXCLLLMbL9Kxg5g0ahKSyBnTlIQDlXFbtrtWWHYYAdwkP0NN74SVRaky6103hdOG+v+Mr7q69WvMPzHmFk3tVFQIj65JOiQD+M11TeSHB6xpKkVzRvwGYtx9rzuz4EacXpOGnV4kbtf7DkFbWgud5V+krIlvuQ4YDgbhJAXriQRBK7Il62V7xNiPkBNisKBDFQuHtOl1+cNWtkWy41U1EJ+4hWEUFxcSRf6HlvT75QdqJ1QPywCtV\\\\\\\/z4bI28Oz81BcYwUEH4mXYgBuj2wSmRATxT7puPyr3+xTH5KCrIEvj4r0QL\\\\\\\/+TwWJpOCtDH6a12104SCGErThV9E08zt9UUDxLzMCa+YYNLUbZPodoZtypJrtkPSDeunrrn7JlSHJpO2sW\\\\\\\/ijTmeW8oJbEJRJVxuiBkF8ziTEzP+Fd5dS+MRDdZ2b48fAd+HSZAA9UW9HARwWJg4yGsaYVxCVLMCmtj97loYwfO1YiqY0jsFEgIVb9JbDyXrYCjb+TU6Kh569WmvpLsKac2yvAlkpfQmncPwvCEapN+xDlnRoinYHHkI5IiweTb+TQE6FaW9bwB2zMT1khy7m3iPz7wrbaMnx9GuYbxTJs\\\\\\\/ZCHcWoK+WdUk8L2tZT6ycE6DJlP31+myLbPwjy0wPazMYOlYnygB8qI\\\\\\\/UyipAspdy07b5KccGeFmCnYux\\\\\\\/\\\\\\\/hzVkpsOGR8LbCf3F1biJ+yvKkFY9iW5xDSiIdxm3wpONgg7QVnCo3u\\\\\\\/ikQHEPY1Xmwt0DRc\\\\\\\/xCbPGN3VhzQ8XbB0wYvAKx3YFnC\\\\\\\/dTHo3uu8rAH+F4KY9tLE2c69q6FvnXah6L7tmyKg\\\\\\\/bm4PRUISvAvoowgEoXtDrBqcz3cgnR1xxdoKWf4KnPWuK8oho2RhShO4vhuBqRue53kywwvLZi6BIxp9paKEgv97UsXg+dIY7mozu4nUkW1+d4VL3Xh3wSDGWno5JuvydUNi5ERAWrZPzBc1ShVyOef2lXk1GkKATH8a8s4GluCEVSMDXVnhww0knE5dU5coF07cgXJRVYZyON30yygzd7Zra+pDPVyAWhTmGkg6wDLrB6E6HyFtkQI6ZQXSznF1Zts\\\\\\\/VXH47FyS\\\\\\\/2XIYZs9zFAC7\\\\\\\/ngM5nqkV4GkbCzYfbvRexsaDnED0MLbvZ\\\\\\\/9YqR\\\u0022,\\\u0022iv\\\u0022:\\\u0022S2kYEoEzpw+kRfn1ImaahA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229d8c1d4e2f897e2767fb5955637c43f979ab47afb1850f738294efddc34397da\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+3xtEPWpaajEK0V4SWOx4bRkMkHMQejN7AqXNXUp8OPBlOEl7NF58N3uZBybY67YkBxZRgoThiOG3TLc6PxOiOXFw+nS3mLUIFq9KXVGDsGNzfNc4FeCgWjZ8tOhifLt2OgeuyEfIRlNl5BxP3YtfKYS8+X1i2ZUAqXa9Liij3JQ0XxvQXOKAN52FIf5WI6xIMGAW+Y9p09Er2tecbbWHiFHY7VVdAZH+u7G5BNf08dGHSWNxYK3Zg2ZlbqTz0H4\\\\\\\/K2HpYtJduYsG7bOsuKTF18QW94SDWvFQwNpV62yJOs\\\\\\\/4B\\\\\\\/zIIlPIN4+nFgvVIynusYKHmaADb1FQHpcMgSxgIBtpqv4J1QIsgNBkMYigqxcdKcKE7X2xF0vmDrESkz7gF0s7ygSfSCVWYguJwNIx8zT5eO4b6WqYfmp9MiomqtY6WB1ZKRttoufbCtSRz7ttsphDa25ATu4HBEG7rQ\\\\\\\/3lpBr1MV9X3a2K0MfQaDvMwRTHk3\\\\\\\/H1qoLN2CXC9Oj3VHgeKb4G5AdyYJXJAIY8aJOfYDE7NyhYwxEAyEd9ILxVnLyBcf8krimH\\\\\\\/qd0SGbOZ9iMFmkeM5p8wIj\\\\\\\/MNwzOnxbtcM0+Nu9WrqZdlbhBaDuLgodzNlXdvy9o08QvTJN\\\\\\\/bjqFUV\\\\\\\/6PSC5dYVQTU3rIMRFcaRezPczaRnzfLFNaBs59V2KugdMJjdSfm1whPOKa\\\\\\\/1ZlvJ2js\\\\\\\/cMfjbmM3LNFNzwIR\\\\\\\/xWu1qEIuqie2iAHHJ\\\\\\\/pj6YO33dLPgN\\\\\\\/33Rw1k6wMnIMzxnEG0rkEHGXK7siQXn8A19AH4t6LsA2cf9Zl58RrDHU2rflhVhzMj2R0TMAnwf7PPdOs0LWHi5jCVImTPeoo+nPJXGtbyoSGMEbBGW\\\\\\\/nsGcwLT0ixbJKR7mNgQIgLgkqcGORqVILVdkSptn5FqaJWMATdWvs0NNH9IIaedEE6WK+vh55IduQYBEJPOR6KzB6Yiwcc9MF1p+rcgxNuNO9sEec717hkpGsUlq9cwKl4uy7PFOahjhOrbGbh74XBS1tV8bwJik+qGHLb2ttJ+tW65UuMJrdo8FKt\\\\\\\/ZqQSn8NUpC\\\\\\\/jaslwbImLJE95R5AgWjI5oO+4owVcip50Qs9Dwdduy6CdfVsfJDbYC\\\\\\\/FdbSzeFNkaHutb+VX7uYsMYgd8mcNiPsfyO+uV72zuV2NBsFYR74eQa572y4uFV6CPhXGaxEYp9REIk91MD3jW6BwOY7ckxS7rHgqtKRba\\\\\\\/JYVVtKolcNvAaYsbnYKUTYsq8OHEMcyaH54pA0xA1otsakCllRWcPR\\\\\\\/PW8Njy0eu1yRlcJ9+bQn2hlfwa5fPajn7Pm9gSDi68PbdcbOLHHM87\\\\\\\/sXnKJvAeVL7ILN2pzbydkGNSCXq6\\\\\\\/2HYi+RCJrGDLNKE3nqbB3KyWFv5JQU1d4S0IPKqpXG9rmK\\\\\\\/t1Z1oiFBUk6ab7T5RSOHfANT\\\\\\\/70gT0jlrJGs7+zmj+P5h\\\\\\\/c+7CzIR6ghs16gjZIKWlBBaEt+Cj1k48iUjsD0\\\\\\\/9TXi8bbUlHOF6mhrU6dD4ZU\\\\\\\/7jMjj\\\\\\\/cU\\\\\\\/2ylU0Ipe4FHUMVM2YMUPL+TqDnkf2Y3HLo4eu8OGWsI9FeUvlJS6+Eo4fmCoQ74pVZqpsJcoMeQ7D2IKSymnLpWEkX1YwWCXc3BlA1mY2ouObpwr9kheOZQE1vLyVTnjmRn1qe+LUCyu0UH3HgTg1v6hFkMHq7ibd5ZsZPgCa8v1nw20gRRrHmHLBTOBHouHhJ4pOK6yk6TJREX668z3S0Z7x\\\\\\\/RBOf5SujzEx1tBxUW6w\\\\\\\/irHaRfTfXtuUwN7uqdlnIUINaekS2XkOu9Mnp08Ga3p0zpnL3ScBLZJmMiPZV4GKnLWA03xjYY+s\\\\\\\/iRG0SEzfDSXrUZ0qgIZFTEXt3tagmitcAUCQ8SgRynXXyyahgIo1klOT7qoZ9U\\\\\\\/y8pRnNtmZyWxdsLb68QrUu\\\\\\\/ReHTVcgS2eVSbNtsFZR5L4ZyNHrzjDIURP2bxcfqs6\\\\\\\/TdXvePhBaBk1ca\\\\\\\/PmMbmHuz70XhBYo6p5uHam+8+Uew1NlXwr2xq+nsU5bwMr58o23MDTjUUYzH6q9awww\\\\\\\/BDR8Ji6IhIDTef2X9eq\\\\\\\/Pw5OV3UF3VgSw42mo9jM5tjjhd26YvNyayzoqmUiKV1ihdhEV00N3YK2X3AGjzANAzJJ9Rdux\\\\\\\/tCD3IBQIHPq+Xwy0JLZY+lPq3A\\\\\\\/efzESp3JKhLVjlltJC2r13h7Dy59DepU42NRiFRjocDXKmmXVgNS8H+70oHdO+wfJh+3+IGo3v9KqqqoMR\\\\\\\/QJNyCSkXeNZb+nolmNZ0FYEo7ysw5tWVdSkwXeO73OiCyN+vTGB6iIVRYdfAAIu6no06vw3r7ODdZNfOriR2vDNtXOLPWTwFZH2XPGu9imM8HfbcK5OWEDV+ALXZEXi7t5m9NOyYvnbPomg9yGammGgrPsKt22DSzSbwVvD9ybrKlLbTLDChpZp+NQOFwUd7AMyWm54TLyN2ELvthQrOFg86fUaXjJfl6nqtIJYzEt4BgmsB4YspoKHK3TdvJ5ChExCJJ9eOqsp5wanoWM7Rd253+X5LTq+uWsKPU4\\\\\\\/u5dxIT+WdUPXNbl8lVhO4YnIn7Y\\\\\\\/aKjZqUuHrnBU0MTmE3yNWmlOASqugbNEx790vokVyVKgBq1WodSOVXgqmz6G13kp01VhgbOnqCRTe4W5PkHUpJlDvDnhtz+8LLflpGjI679DCkRZ2CPleNjsLQDsIfAJt48jn4lcWDWDKm1DU\\\\\\\/PIBSxlKEra3WEaoPfLzPMexUI4RRvjQq\\\\\\\/DQTO+tTTgni1\\\\\\\/b\\\\\\\/1SKwbbbFtYblBotaByrzn62kCWUMWq7U4XBa36uEiTXFwV6yvWMB0bpnECdWSKpyknTxmjweKEfqEg4pNzNDi1kVlYwt1LzO+D8TS4465wgS\\\\\\\/H6E74sTkoSTps2pnM8+ZAsxocbYGVV9hx5k8MgajHwvj9pZhRLpeiJ6QescaWZIqHGf\\\\\\\/PDRTp9m8YKGw6lLI3QowmBCUo9H69vWOCv6+6yZQAV76SmipVgew6QSj+JLfZFSwqrT3T9QvR5B6l9p3fvycSrJzGTrZacXuLKSJ2CBXA15NbNMc0uNi\\\\\\\/5h1279h17jSsI5whXmm64c\\\\\\\/XgNbZ4L9UDBkSrKSgDnQpLhtLfVQ9l5Y4oWtZyfm+y5Bk8qbctVp4Kwz5tnXjrKaWNB0HCWjn1QQL6GVFuu4cO1vsG\\\\\\\/35qIa5VoT\\\\\\\/NRVXTvKIjESS9qk9mDQHgr4iif9dNvG698ExLgOx\\\\\\\/0XpvJcN0DnDXyZX615ay9IsAt+e+rdxUDiu1Vh6LFAARasj0IDBE8bBZFQA1JJF3v+i7\\\\\\\/XTDkRrOSCTnO73TA\\\\\\\/EPkYg3ArF0qP0tNZDRVwlzJk7k+vDtyGDvg9cjBQmtOBf0qsbhMJsdL8osKGhFJzMTcQEpm5VQyPwoHUABD3TOMMBH4GnVR0r6LmmfdUkonRaUoAyXse1282NfgJTqE\\\\\\\/uU8pcW6a\\\\\\\/An7ibaJIiBIOcPyItoV5nhvq3U+3mBm+Lru0TjCzBlzxsIww7XaxxuiGKhanQfefDSVXY5+Zz9dUnyOCN\\\\\\\/K0R48FsSFmX22wtQbpA4jQRgd4fgJr6KgxpkZOGnV8h0f5fctHsZE3QydqArm+sDIRjuIr0J8z2TBganX4RuUvhGqUeE7zwY+kBk9wy5FsVfhcXWbw\\\\\\\/OwTBXvxxngI7TQsggmdRw8+8gCbrh5B8JDmnBLBnUeqtKPTAtnxkUB4hldx5WFeOMt8tc4mRXR5i6fGBqlZNVaGlxTOhASNXLmOK0BWC9967DPU1bY0m8ddUl6U0F06lCzdQyspXwidhTS3BpBk0hz90a5BMlxXRdQP+nS9DenqIyig7nyZUKCE8x8N65aFD1uHd9siwRSqJsVyxD39plLScvNdp986seiCHpNPZSLe6Ru8q0fm3RUU\\\\\\\/dFIPVrHojk7niStzRF9U\\\\\\\/26aBkZDNCDjz5uzuaBP5N1MK0SkqfEVCPOy3hgko3SGoE\\\\\\\/T4fXlviCMbVUOTSdQ830XpGzOEszbRlZSU9FxXjjBN0yNi5bFkEurgtvCjZkfnIDXLYoSVThqVJMB64\\\\\\\/Uo63F5iceLTeNJ6\\\\\\\/03aXIe1TMLlYyga7f+OlQPv7+AmWM60WcMg61jlWF0SEeDGFhrUHrOdipAo61+lSsjjE4gNLLpREea0r7ciYOnX4pLxpbiaZcpXVM1x9NabWYvcOIMUIvzAr09arAljNk+auKBRdAQr6qwaQ5VNNW8I\\\\\\\/i2ai6TSX0e7trgwcSocuARKdOVA76QPcIV8+7ElnGJ73tJ75EhqNJekRMq9ydW6izhWijb2SNBC1bwn4quRGk3bGs9nm\\\\\\\/C3UU\\\\\\\/rsuTrN45mFq4agIA39ikQGKJJgKXOjOFnTkDzG8XMjp3NFeHnqzAmFkqMqMMqpamMnbr+HAhSrAECE49jchY\\\\\\\/tme71CvcMjqWhO6XrrLvTKXm1lAMMFwb+0\\\\\\\/TAYWmx\\\\\\\/SQBxWZEd7Re5DZ7rFebK7j3eAOljvE4H1i99UtjaHcQ1qhinomz5DJQl5MYeAEGJ7C62l1s9og2TDxS2PQIxOBuVEh1uYi5d8yBmCrB174e7Y6ok4X8AT5Y08fTQ2sOn9uUi0Qy+wwJ8h\\\\\\\/1CjetwyzaQpTrDBympLHu1cAnOZMkLWnFitNhuBAwfpo\\\\\\\/WnF9Vp66iA7U86s6q4EUAIPioL5MFWFkiGu3ifSDEEhHDarNDbnggiMipoAfb+pgj2mlZ63mnAFMzHjureDt9MoeJW01nBywb74MHI5kccu0ZLZ7Yu4gxNGuNwJOTsd4GT8QSk7b3JvrRvVXj9Yoj8sIz0LR9qhb8L7lFL7u+st2BHd1Dk3qoDJ+ztim5BEcvxsectADFHoeozqpbOMEiWlEvPsNEPlzX6d+ZwhwxVpRCRp+e49XCSWDPulj8e44q03v1Hf\\\\\\\/gVUHz2s3ZLZKfqwtIaW8ZYP1Kzh\\\\\\\/4UdoQfUX0TaPnDxcBEnCgtJeR8T\\\\\\\/sCtcmEm2cu9jciNZzoKiqL3dvuirzKaZB+PGpRQfo4oVzcGVOLHovzrGPPN5+VXqkwTawvrMvKe\\\\\\\/AEQnfgWgabbcB2B60FKne48OIs49SYWDjL8autCZB3w2ye476Y\\\\\\\/c3engJrXQLyjf5M6oAAXBwuWLcSLP8Ie8ahUem3eygmrutttaW\\\\\\\/bmTIwveUi65S3QvGK+L2FV0UDWKuyx+rh4sw4lXFl9ujOGmeVXr\\\\\\\/RCLivLH55SdBqPrldEcicJjhD0M4hChbmIBgRO9VdsF4mKR1HcrgOTK6Y0tXqToadwDTESAR+vaAgJmOH4VemO\\\\\\\/bkjKD8q3MtlGQgg30doClYJIILqiUDu5wQDOS9dwWfFAWKPmzI2ufJa8XWwJ24S\\\\\\\/bLBOWFycR3iUrlclOuzVHF06WxHUfWQsBeFyhBP2ijnuia\\\\\\\/0ul3g6b3\\\\\\\/23QpasXKgQNqTn0y8FAV1aP1x7\\\\\\\/ZX8QRIflQi9EslLCegSDX1W\\\\\\\/s\\\\\\\/+hMdBO3ZGQXNSOMh2DFAG9II7fbcd+l6h9HcyUBU+9X1PB2EjFnbHzagX1xnNVoBvkQQvxITpXYthAFYKhz4QPsO3S7OYu1RsGgGpwMO8gZVwXkHl\\\\\\\/uFxQ6OaNCg\\\\\\\/VtCFsbvbBfDEhEGmKEUTLgWqnFCDC8Z5ryiH4tnbTsCXpox8VKfcW87bgIHLewnpw6mreFHcmHyqylWb5NSKaVelV5r1wqpBriDhn150E8VExFES45A+CWThx6eTj3I6xZv\\\\\\\/PWgoDKKxd4yNexyiGapWvUQByFSd\\\\\\\/k\\\\\\\/uDBmmJj+UzU8oyvucHFDJ4qml4ITaZ2f6G76+FfJjSp8+mDjbtdWvmWx2uBvGzppcbyhszcjiL8wzYu1gbpEmLZdz+096i0fFqwsO39uxcjlGTWFPXem2d8KqjB9W0dLttiC7p9R8JuLLHCNnwNz3VVZTOy1QitJbWcJrLVPntr79BzqI0h8BRRdT2PNPRr9uzltgIkKE99W1cNR8zcnCElf2iBQpxE7Q8s\\\\\\\/+lGUZj\\\\\\\/HNJCE06Z0kIAMimfEnF0CPiE5z+H9T4vtSTGKre1nfsstAyo0jA+4qFZ7szSq2adypsRMr8PGrTmG7Q3oXzrurn6OR47a8Jt0liK7SgcjvHo+GcKJCG\\\\\\\/1GocGlWf6JqfAzUUSUzuDXeUAkn9SVnl9x46cAUmmu0iPd6ez+gGT9y3\\\\\\\/TnmXnYYjFJXRcxBPu1KigqDC8EmCs1edCNokZl8OkDyEfDkruqjG34aZWztLDbCbFaJy7btZ4H7uTZm92Z8STN+jAkIwWpNUnM28AdHNwhN6pOTKqRoKWVvDrhkyFcVibhGvbM+WKazJ+yMGWDiUuja4Y98e57RtsqhaGVZEhsbqyiSy7jIZZAlHYT3W116M33M\\\\\\\/j67Jk\\\\\\\/1iaRAoJ\\\\\\\/aEn+2VcxGoVA\\\\\\\/5lB46M8KA2B1Ria0ZsexE9V1PAjW0OkTQCQy8oA7s5TY\\\\\\\/YnHZFTe\\\\\\\/MwKgbJ4OdsbNkHx5D1gQ2WJZVwn1GpeIVkcVDR63HN46aCfm5quKF8\\\\\\\/KJHtlkzOfIGWd19qFhNN+1RNRBI17Xx66u4\\\\\\\/\\\\\\\/HDXhaB4Orkr2GS31LXrylyZgUsjjUTLDkriXvwiD7EHZ+k8mgKRpWTYE4f51NEV+I9nZQ8ZjmRgRXSDK4\\\\\\\/P7Q20WKoluziULtrJCn9+tE16UI6tzIENg15rICXGclBIX6hz5r+q+81URKS9RqLPJCEmplYKQ1mU0IkrwIqD3RwMqkSOs1jmbdkcW+kA5KYH+enMhMOzHGeuaCkCSuW6JF9oT2BD7w9c9\\\\\\\/DC\\\\\\\/V6oKi84LeuSKaqswy5+IcMKZ+930fKi5YR8cqlhy12PmjXHvI9aIyLR5nCspFzoTwp3caSKjLgQj9IUWsZmMnme3h2X9D4E2l0\\\\\\\/DnyenywfIlHA4feAnQxnCLGcun69uVFpHJ8iyX\\\\\\\/L+lebn1UwleLnBvWKmu+rqYpB3lhSDCYR21LFqA4nU3eaw3AhXxfIjuVBavo06ihkUrvoFxkUw6gvopb3SPNgx15gLPxfeLFZ\\\\\\\/bc4kn4ZdJJAjOPvJtwj+QW8LW+vJ\\\\\\\/IPq1FAK7CdSmriqQBVdlScwjIkj3sILyoZ+OsEUFI9CqmJWT\\\\\\\/WTNc9xVpDjdoG5P08+bRkNGqvKJBwNNHIlVIUCRJVybxFJpqhUjSoQlF2qAc5z6WpxZGL1yZlCK6VM7sRpvaVFOwEj8aE2vKIAJ\\\\\\\/JvhTVn6PIivQi30jU5eQr5\\\\\\\/GeMFfDO7B793GTfdASrV\\\\\\\/vsjWxYm4Ds6SM48szV69HNebhRwNbDYNJDCZHm1kgPPo\\\\\\\/jJzT9+Fli\\\\\\\/0gbugFL4M8paeh4lpzIgjboIKVUIieH2dE13xkYnw3hwXK4cb8xkmY2MMMJLWNFLUiL63M2CDPgG3k8wgS1IBH34dEksDOF60XIJOfsib2EbQgibB27zTMDSH3G3ll+HZJWN4tPZi7atG370NB\\\\\\\/j8q45b5rhgRaw1mRKGRqnFB5EBKNmgL9+J87g1MVJXyqJrXwYeGuBHbaTioR4SoLBAwHV1dmZjTrMKyHCLjMdN9wJUTByMoPoD3uwO8AHslP3vqnZXNp\\\\\\\/cQ4Cj6BgjFE2irMDMP7rSn5gzHUtnXCLvWLOa0Ttc90Mf5juv1MRN+rVP5AgsmT7di3cNgYsTonRI4mkezQbFe7pRkE\\\\\\\/0h9Krtb487KSM29Ep3y\\\\\\\/N84bi3YV6ebNXWghpQJaM9wE+6ao8O1r9zNK+oHWpOGwa7C1DZaqUFW7ONq77yRg0qoAuJ1NSJkFTT44KWmMmIuopEzXzMxodgeWMIMGFCr6VUolRNcjM4IrxUwRCxYXa\\\\\\\/MyOF9UWVaqC61Fmh75s6OIasTRrhyCdG3j1ku0dcuIY8xiYeHpLCO+wpax7x1GftCGR17e0JztpGwLXCMda7yBNzSUkRnFSQhI2ZddCAX8rGPkuRf6Jd5kkVwY+pfzq9ccg\\\\\\\/MTWPhETTrPDp5ywdk22C+Z2b40nmGi+pyd0G3sCVBSb4veJhrhPVnsUNGcSP9FHWg+aYL3cwdT2OhE+rgC3uP27prhoyFGHl84NiQ3IcT\\\\\\\/fgvghBtbBSSfE\\\\\\\/8tiz+nRB8jUhCOBQwvgpKI0SY1w0DsE5lfdkkhU0s\\\\\\\/J9iCv28FYpbANnVL1v7RlQDQPkDat52qAJi+KAlKUaa5YKB8VYqwPGOnTNUMbKMwn7zlj5BUBGXzEzUctuJs7LF9MEbmdYpRcg1pyPSZx0I6AM1dJh0KY4koXkJlE5Wg55lV467lUZxYLtG4WniuvFXlKf8jB28Lz7ISYMllGor9ZXALF7nXslp91dbs5bI8FEldZisYG2NBgxawYrcf83VStiLdYifhNRlDLPDuTTgIFBA1XCxBG9IZ5ZVAgm2XbfQTg80Tz889Thh5O5G4gGaMJTHW3FkQry+58LGB0B+30vGBJYpn8YzkSRhteKo\\\\\\\/e2CjdGG\\\\\\\/Y1jtiA98OvGyA0SACygXy8hxJHBx3elpGrg4FVC4Ki0bC67PjxqY8LrAJaX43vpiX0D3E244BFz8HgrgazioYlgbLa2ZpvZ+qlS\\\\\\\/pMKbXnGaLKSKpEnyIZl+Ad+EUX+0tdsM0q6OxbIFWuj5bLOz0oZkWreg18ekvIrNYuFeYu5ksOzT0nugVDx2IaWZaeR32guJcgR7PwFMrzxQ3IRKfgKlUhns1FmGK1TbSC709nR851gHVw5nGeCU\\\\\\\/fmbPQOuH61zNYH7EQmCYha3kI5FCIKcfzJbA0jzWi1juvo1rh6oRrDRMwt1AgqINeJ6Vo8uQVJkjgfVp+xCO+SZHoyAwWVIadfoIvjhfrlfEcIDKwBQ2EJM6iazBxjtSv\\\\\\\/gAi+hYsqBkp\\\\\\\/oYQ9408f0bYb8hbx0cCpUMQHNCi9XTHvAIDCCf6T8mTk2I+sUvFMfmodXYveFFsO\\\\\\\/yWv9RILsnt09da7Bo2K4z27NhJ20z3Eq8y+MqOE6o8Xhg3FzCYMnEXx0hJF+XAbymRrnYgCv3GhW9z9TDub4N3uKTrM2JxPWN+WeaTNawYm1jvPxWo56v2Rwv9372wiWATfTJuDXeT\\\\\\\/YMwwokWNipW0YZwx96tdZCgaM9vc72+F3lO6pZ\\\\\\\/RPOXHZpSEr4umbNh\\\\\\\/GVR753ZUNNylEvIA4V79tU4GTtpzruRgyDlGP3JJdlcOr3rQ0DcRm4vXlVzoRMjGls1y0pLZCunR+T5c\\\\\\\/lMDHECVPxf3hdJ2HB3xFH0iThY6\\\\\\\/gGztyKRfiju1Zf1HllV7apFpIiiia+BjjyQ+0dO9kLorMtML0uiNIrn3SGjG0rPNhBpenKVzy5SiayH1r6dK27tSJNpuwDeqIGu3a7GvgtBBY+CWgD5JAdS579UmigpU3+IfCE2LGS29Ni7NnmxNfMlXBK9H5+5Hu\\\\\\\/fvXngVIh5ErSBif9hjZIYnWeZ2ry8MBAIrI+qYJJTuk8UnS74dkWnOh+gBQphtN3XWvalpxUJfSSkPuZsgP5vFtNCQQG16v182mVYA\\\\\\\/IuiH+RS\\\\\\\/QfAdlZdX1vgwzxfSgg3b42\\\\\\\/JfN+nL2dQCP9iNJf3uVs78CwZ90PgaxPoYFX3IdotjN\\\\\\\/JEAW0W42cYeXBa0V8cd4BqFEFnIwhyd+FWbj862wh3sp1849vScKJnv2U6hJYAMfRqJS49qtzay83QFbaJdD9irSlizsO2+iaYGWnCCyax3ZbmiQ6W8aYhngbIHKzOUfcHmZ9+pL3ghrRNGDvG4hNggh2MWCw9VMqzCg\\\\\\\/8oW7ujlRHQpOPxBHMFVPnN5EVh+4Z\\\\\\\/W+uzFDNG1ZYkVHgLJG4ww11kaaFhN8HiBI7WhIBxnI9XtFE\\\\\\\/J2KjE4YLf3Kty+uHHN8T1q5WwiMTx8heWXVjD7WTahLiNvi+jHZi22jJdOLPv2U2+EsMxvsmnZHkF5e3e\\\\\\\/24qSZ6vepSy7sXcfm2f3Pv5VrMXQS8XsvkE4QNnhzlOktrzWJFgqGuF1aLe8J\\\\\\\/as6IakQbSd+AoK2Ld5HgWevV+bskTut9A+rnbETPMNdWrVc9rTDyuVuyIHe\\\\\\\/s9VFbVqz\\\\\\\/uVGEN0nr+yUO1wsV5FaeEULZRssi49Vj9pZBr0j\\\\\\\/EGT03e1e9XTy+YVunb3AmCR+3PvhDNTB5GGHjLCHuoJMg8luGxKNxAr6C2qa4XKsuSY264Vb1o2U33dmrB32v\\\\\\\/3HsIw3JuAXflBIhfk0nQ2v2lYkwvIpFVTTVJ\\\\\\\/lRQiUtYniZuDx8NaXrLYfb15Rty5uBErKJbzOO9DUk5V53jsJJ7rhCbj2q7dxFx61UfdCZCDmK01Cz9njOoMG9VZ7yzRwawvDFzD9cmsibb\\\\\\\/FhKndLMstWnxLyYwD55xxPPxhQN7qVBwq0vF9aGFwlxwtt9hKYunPhskdsaY6hfyJm6gO\\\\\\\/CugBMGmafL79o+mV\\\\\\\/ZAFjMI6yjtII42Etkr28KQxABIlPLVVH1DURWVveLsJGi3+ut4vP8R1i7VsgPMSy3Rkudy30zCTv\\\\\\\/NoWt\\\\\\\/x3iuuV6em0VWsPBubxT9ZWf4yIRkP706aDSLXh0Jezi5C\\\\\\\/SudfFC8kaaDCUFbirJ578WtoC8i7JmF+cpJddVjM3uwrkSoish10xCDpX55HwKNmWRZnl9FcDRCA+u\\\\\\\/G1hcU5FUpnQq19+8NmYTCTDDwTgGj7f9qZ+YHJ\\\\\\\/yW3MwNV4nEaEw1uDKsj2nQ55e285JsXxkMSjTXpQWD\\\\\\\/Mg641cucOEztB5qHzdBB6ohnD7Zk66adcqantCTZIUaSsZAyiVBkZWoWuGUtBhkneZWoxDsitWretZP\\\\\\\/AphFsoFsHvgRYhP9JToRYlqf8wGRttovi5YQ\\\\\\\/L0ZIDFEcHM6wOIyjQ5wsIKphU\\\\\\\/LjyRo8EmAvJjTLfH1ik7Q38hyEyARy\\\\\\\/tsAj0rBDqRyWgfophUZR50CU6EUYrPwsyO81xnUHOk5h8HxqYe0p5OB2kpwZnfIEFrM3G\\\\\\\/cMZ9V+2uX9G5Ce6FjXMJX8Dyf7OdbCELa9HjX6QtgTjW8bZfKRfBITdSy4CHfm\\\\\\\/+mqw9oJvSBexcT9tNPWpRog+Qs+LB9aqKNNrdr9cdNGjDXQuZXbbkNkRNgqnrAhBJ5hl\\\\\\\/+ImK8sZu08ksOu7I5OXDvLETsvkjQY+qxDIllaZc4elHCW+Lr8q3DO1IQ\\\\\\\/mc09kwojO6Q5UZztYspFWMpJZyMciLxgsHRNbYEM38urLnt7MR2M+rAQm35bIIqyg0nCCtpoVFVxtkCczZ\\\\\\\/SQv99MYtm4zAaemH6oybABvH++e+d\\\\\\\/X1XZSzUZH8JSGO7sSSCahTOBgWvCpKfILbaO9HUa5WVFOTt1OJUP1Mu\\\\\\\/d1BvsuHXEM07OBMhf2\\\\\\\/srMLSWRz94BoCVHUEhraMfonOeTYNdYAnH0Z2ayGG69yV0Y2xlgyPqlauhSS9RJJq+\\\\\\\/J8Al12jHc6EcxdDrsfBgkLuNVQzvohPMqq7ai1VeM2H71NfUnS9JWyWgVT2NtBaHDiH+UFHINZHzLygRumezhneDCZozNkz+22ZoilkqyaEfUb5Ez5z1go1AVTwOYqGmJW3SQs\\\\\\\/FNjJZCv1kfLRxSsbM4O\\\\\\\/mQw+tbMZ+SQ3vgW1BSd9SmzUEumtHhFduOUC6d693uaLAuaOFq\\\\\\\/4JRFBUKjjEc3dFYmYzJvRDQY4UiUkzQnox+2A5RjAApcLYK5nHMARA8BfQo99xkq+gQ9l22CLB+KbCXzh+MP7R2h8aKjJ2TSq8IlVLMHzS6QsGikwiFqyUjMKxTc8ukFZInASJ1KDw4lSo4qFYRtML2FIwf1adthW6lAaA21VO4STu+p6pMpeV4pR0wUY+opW4QhnF4Ys8XbXHrAdbC3sf\\\\\\\/jlZ7x3dBBcB4fn7Co7tTgC2\\\\\\\/4FptKbS6EoQNcgM2PDCpjEpOzeN1jhenj6enH4fMmQstgJmhxr7HTQyT34hI6Ox\\\\\\\/8l6Wh+zpdns+hVdsjFzmRjKShYyhsIFfRTp1eQ9aLjSdtmnCFdFg7rSxeA4pAppsidTauWrsXCXcPajxPN6TL60OqCtk5TgD7opN8g6WsyAH\\\\\\\/j2fk2beGtUU\\\\\\\/aCm+IVVzVvrpdf8ueTAwCRhf3QDakVrDmvWWccBppjFp0DDCq1k3Ci4VMpt114IwlV7ssY1PDjiRNZMeAtK+Gd\\\\\\\/TFqyAGSYPr6HsAshKKH+v9\\\\\\\/F3QNSg57R6iWnSto3IU5ZgpYe4KFyMN4dIDklpJWqoPirry9Q1NqoIqsHMwwzCkEreQwTmMHUaXQU\\\\\\\/yhKGGUqMaCX4iU8Bqme3K9YV7bxxTpDETkUjrKCNDeEctl8tD\\\\\\\/DfGB7WXCSRLrlOTMjldJoBKIUKui2v7FvRpQSQEMg+UvhVg3ApNNEFDJlqV+OZZJRJuwdNb+jKONX\\\\\\\/g8ZTw7TfJU3ccJO6Vd9QaaEXeSJmqsU7F+64itdc9bkSc7cVDidNr6wegRjYsIpeZ1zfhaqsvDSgPBpJzHS5Knzsi6iKODZIAyZa5cn9Q0jYpG9n7FHwLChy+yR4GCtABMKWtukh1BU4\\\\\\\/afnHCFd0Xt1pczicee2JCf+Afwb+KZ4M7h2TTz2S0BHTSWNGcPfXaE5t7nziojLev\\\\\\\/GPlX8fKBSW+Hou1CsK\\\\\\\/sLCA9XVCprITuih4I3bzWosMoYv+BRzYpHje7mGlGex+S++n0oKrxkfF25OJqYMISLTX3fLjD2AmAsX5aw0BQ1+C1kfY+p7kLHh5xhiVuDVvOFnX4xVLHOIOnHxLjBb\\\\\\\/2LtKIDqkMEMmts42Y2MwZuglcGL5TX6ajafDoBVM5GvM38FZ1peHC2cvQ\\\\\\\/GIs53jfswLf5J\\\\\\\/L6p4JHNhWWT132RRH6SeZMwU\\\\\\\/HqnYQDWAtcTIoYX6D+24ZrqhDEMxdeulBBv08wfdudL4JMbCae6jjckkuvlPbuZhi3G8fnMJOO4nROSxynxvcG9ecOU+9GeYr\\\\\\\/SZXU+obrWOvqW6HNHLjV9+tm8xkdnjYs5zf4zHVwGra31qERhoUI\\\\\\\/cBmF+q6B8Fk9LNQGaej+tVJrfA54wynE31zCpgWGIa64Z2jlhbV7Sk3P6V2jSpQ6CIuvchWU4XzziYSKMuCPrzIUktXMOe0EzAkUyRTduWkC3onfU4JMazODEqbX1bLdLZG8vh6VKzO72hFP4RV\\\\\\\/iLehOyYnxbUqobocG2ynUAQ1sH2XpbJUfASX5SX8gSaKYLl92gZiJc3hdr6it\\\\\\\/nGhlOE+Bszg+HTcqbNQCpZzoURcK22O323J9xZZPDH3p5Nje6UXhAqLBtLjk9UWU7SSEySGZSs4IChztrDTxzzt3sjpZpff55mfWKA3PCwABd7mL0dcRqXoWMRx7aTy16OzG1Dt9GLTtToDpvQumI4RQb5lignM6Z92MOHHaJ\\\\\\\/MIgFVH2XDoU5iWW700lDLjSGQDNL4E+sJ4MK1aEAAo8BRIolI5T6b3R\\\\\\\/gRTTSu+YAbXdFp\\\\\\\/2afMm7JNZ+WR4VopOtdrX6to4Z6cR2CG0NIZJNXu6nWup\\\\\\\/hlZEOMW976tdYTOhimZ4MOzhbkWLpAkJqIDIEII3JsomIv5I8yrL++C7rBbrbuKta+PX4TDwfRqvNbpsG3GXdxXtn0WKkE+hkCirBhpPcBRs\\\\\\\/BPRyg\\\\\\\/6ZToQ3IAIVZfUCArRpfOAjGU4daD\\\\\\\/y40SZDrlmUtuCsrWRRLJ1lZxqzKfK5suc16NQmV3JVVOKoMvYvCp8ZIOBt4xC2TQc4gwazUt9WN7A0Z7b8Wxbmm5dcsvSdY1TyyKEXEs20pjKm+QDcieIjdDpKYSfOdr8th+0LPpZXfgT+QwvadDzhAxMWpnxazeGXYvS+kmLxr2LgnAZnky8oon6iHKNz45NdZ18D\\\\\\\/MJCQoIiofbs3YM96MTpDWuH09lRDkzylx7l+ppxeB+\\\\\\\/1Adfd6MuP4HAV72r\\\\\\\/lSpYTzWWpcTW0407eYolX0+LYughD7furgK\\\\\\\/pOM9snvqi7yURCNbRO20rWANm05wmjSrS+WteL3L\\\\\\\/\\\\\\\/gKn+cxgRuCk\\\\\\\/bNasLQSqt3RZfXAxo8Y+l8Hxeyd7ZWaK9rHh3ru0xzYg\\\\\\\/aSPgEn\\\\\\\/aKwoV\\\\\\\/m6r1va3B6vEHXHmv8cwfjfDdvDpDEuB0t7\\\\\\\/yHor+EZ+p0IUZV68wD69I2OzR5dYKAbBrEw97qtdheDG+yP9GVcTXwwtPn71IE6ZbPwxSvSDuAbB8SFNBe0kARW38Q0a6vQrKxUjF37\\\\\\\/q6npjCsy4pwdLiHFxeh0Fz9wGoUkd8RlT+ix6GaqSNSixGpDkFG+\\\\\\\/hCyiYe1BADH\\\\\\\/dh56QOHn2wxStZNujxvh\\\\\\\/+MC+S0AjnBr0XTaa2WYVtdBTVFBny+RUITMhKLdKMpoJgjc6n2xmvsTYYdYuX8a6jkfsb9ZrlFaMYSKfEY60qIqIDD+n\\\\\\\/rOE5Q5zvKxMw8c21Zmx9q5L+mxp6YVg6AWLgYlIpLTFuzc5WQEpwDtaohCpLWzoYHp2TUsRe6gvIGqAwQBiSs+7KyfIgn+XUMa3oBuq\\\\\\\/+00zb4okJKmXkpqHuyzTwyX4iO0xuPKG5AHqAHPgeBHU+4mfdMuaRDF0gZHPGgptBVz4Iwpa0+cDLZ5\\\\\\\/CtrtPgMCiUU71muASx1dCY7uytE5dv1s8ACirWIi\\\\\\\/9BV\\\\\\\/XJNh66Lw\\\\\\\/m3WqybrWV\\\\\\\/puXVzAIjII3YCPDP1O6ZBLL9eCdiEHsvl85GReoBUl+JxTjk2L3EO49cfU1aDa6Hb9e06DtnCzw+laH5lMHMs2ZacML8pjUXfkqFpmnyrAKpcfEbuTIbr2\\\\\\\/FCQpjVlWWKJMMcY7vghDo\\\\\\\/vVmM6zG9LtYM0DEWzUs0IQoXcJB6kLcGHVOTn+hesyCPAb9G4dJEXpwwm+\\\\\\\/F7BzOnk2iI9\\\\\\\/tmuj102DShke59vm4wVNnU2FDmRmVArovTGc1VanV20jIHwor1ddE4frsSVDj0os\\\\\\\/rFDU\\\\\\\/OuCJrCRonPubd7ETT0FRALcuGLDdGgX1mRE0v+50bxKGZpSYxmEq06sPzP8VCbqX4gBwOA1NqQ57agWfKpYPYqCohrbw1VDppV1+iLI6vERDCvlBq88ihoBfJ8VUoM5tY5KXYr191d+G0uJStXHaF6Qq5rjtzqYpZ76p786dejnlA9cn3rxU0qvjrAtC3VKBaycCnT5g8neEoM8fOIJDqyLKWID6OOgql7mzR2hGuwVmk\\\\\\\/OqXklv980uDPnWd1rHOXUvIwAQYeh5yfjU9ZZdB9pnRGUinNaQagJZKBlhLv0TVE0XHi5yH41sDA9m7D0fYO9tgqywQK7uAL4ybxAox8ApNxYBkGzJ82R\\\\\\\/G8omHFT4BYk4473XJJBXbkxq98tdOMSAoRzb2sO7L\\\\\\\/x4Pl4b8+WEzh3zc1tZ4paHQJErTmcI5MSb0STTnW8ZtnV7YjwdhpgWbv4b4j2U1On0znfF1XmfQWQ3ezMvoiNSaO1\\\\\\\/sokvGqoU3CFxkH82vT5NNymwqd\\\\\\\/grQLRTJGZ9pVhuZke6FBTHTxXbmT7NCSNubQFC7m2jqso+VYFhEzvkrPvrAavu919ip4YUsnGX7FU9+PQooBBY3z55P5GqgJ6FY7MC98e29FPsXYyO4mDh15E1Q89J+VFpcOUQDqCqsrJjJuOq3R7C5gC4G+FR\\\\\\\/3K4HF94IYAMi36qF4Aw+nCsb6cp+BY931H\\\\\\\/kT3iKcqFdozljcztkykq67Zk8GUwcXjxLVVwhIaXToTbJC\\\\\\\/ZvXs7NQl7iUYZGUeIHA0DWtmIJCrfLEWOfsCE4Ms55KpjTWr0DjTILCg\\\\\\\/hYA7YXUXIFwejAd02Ab8WoMml98OuBZ1QBIwW5qkj4q\\\\\\\/bW21VLmBlMUoYkDYgvEKXJsnzOT3BHJblnImc0hEgFCzVSOguGUzCidN3X1UXfL8hh7fMWwIo1KROLsxqoEd\\\\\\\/lnOcOYCuaDI3XMzDCN3yvvD5ac408dERvIOcB+Ixkg1ly6a6N3U33r5PjygELQKi6IkQLTIPxLXs\\\\\\\/5YjPzJEYiw6hKjMJGBrzFPhh44MBxzZ9BjO7uKe7lNuDpe7EPCOyXIDELt+ikn63PB9feCRXNnI88YsYNFE9iWyvKY0MLx1tR5YgLYJDZFzZBaTip9a222+3DdMhjeq785RV34gmsCFksDJ9BPbibD7NWdugfXnlEnVh+f1F9Q2tJX\\\\\\\/vmP\\\\\\\/lV8WLw77hb4HtqappqfQw9dQUtsIphgCcHL0ep4Ba+H4QkbGmckIHFopNxQnHTvku1HgqE5Gm1gCPrcwybsLzU18tJFh06\\\\\\\/XyLIER0+uZETATSqMk2xJNY80pKpI6WB0s6WRIxpkm2q9ljOfRFwTWj4gJJAvx1SQLqH\\\\\\\/AhKdHNCk2aCrGmwy+\\\\\\\/djHru\\\\\\\/SChq6Ijf\\\\\\\/sJRKh6ZssjxJ1VefaUFpf9M33YpO\\\\\\\/IBy2J98wO2l93fYm1ouG+CQGJJwQKS59\\\\\\\/93CQDAelIzush+Z\\\\\\\/+Q9UTTcTYltsnSF1mMMzSh2J9xtq5PlJQuxvUrWpj6vj3rGH7qZxN1nkdk4T3n4RbjhJbjK6tDGMPCbsdVPfRi21CRej\\\\\\\/JCqtPTqraIws7VeHmU5EJwPN7OXXG2LF3mooGVl88NVQYErGp9ZIrSw5UFgBjr1TRvIhKl5S7I1GPIKOmYZWiwGFhsC64nOY4mrVBanIacu7FwbVGPG5NlcnW4r0f6v8TnIk82nXLPVc450TQZ2ioxvH2+TB9so9MB9E\\\\\\\/tREQwtpUbSlS8Rk3QkR+0oxjMb0YCQGnW1qM427\\\\\\\/4LeYalfNVEKjhKMQ0tu2cfjO33FdOVUFb8eEJ\\\\\\\/Cqk9NqdWFY65ddR0aTx3YGU3sFPVaKfqk3Qg8ctZ6v5zBY7jc0onIGFAaqehlN+lXCq\\\\\\\/rVoPVGkodOALIYJBquo4QoehM+0Z9vMqeBkNZCCP3b5e+hCIiuqUdJHQoFwRlDUxNU+fK501rWlLVfP7LkfePHoQm8uGDNx50h655QCVALlqAmFVlmcoD18iT7x0qNXI2o7csioFII6sUf9WRUAi7hEahKjSqe6qh53qGysm2sLm+JyrUzJa8ZfcYV5Q\\\\\\\/AKGafAJ0nHstJvkIisRF6TmIshTVfr\\\\\\\/l390zmIKQuGgycoxPPDJSnWAZlel3OMHn6seoCSa1RgcPG233KTX6rwwSAmyk6oWJwfKwKnCcMJNHrNV7T86eNWGQZN146J\\\\\\\/YegNTmGfPi26O8drX3lEesBXbewNnZcJRBI652mNBHtaNZSRnqsxh3R8w\\\\\\\/\\\\\\\/9CkQyyurBGlGg+rwQq6\\\\\\\/bMkYVlkw\\\\\\\/Zwx+nlFFdlPDfziDXyTkOAcWygmXs53ymywn4SfbZI7lPBVPTTXBVuE9U0CBsAZw2re2+1rK1BJ7CL2wFJqPViiJ+psBnvLi\\\\\\\/gB5MkKH+q9N7IY1vsxZnP\\\\\\\/pd\\\\\\\/qX2A7xIeB\\\\\\\/PnMqaKFLkvo5SWFYCTh6rpfllkNS0RWJMxExWFL61qpu2R51rGzI0r2f4ZS35qQkwJEuEky6Ck387j6xo7GzjxmvTsdZuVi4fPzIWpgXXTL4gi+NAL\\\\\\\/L071SwX8RxwXD1vkcDFhWXHz2s19BXLza9nNqdOf260lF3wJmB\\\\\\\/MFB1EWuaP5eEgDqvHeWBG9tAg8LRKy1weRZ26EuJaxKTXb321MejHpodSegkL8aztxOnJIBFKMGTid+6VORGZWfZoaSh4fGLzkhUGA12prA9NQ1QXYegLz+5pWZ8XBHjHjDoOc63K0egPaTMjj5Bkl\\\\\\\/aJd2mlafPTyrUTpUA9EzVzUMIH8c6vPNC5GfLd8QmYX9e9nV+wQmVg5FlOrnMqfThqeaS3qZkWCM6KTIQbvuusOp0ULpT5zLwGFjoxy75s+zwiwTV0tnq0\\\\\\\/GvPBTkrOpwrRkLcK4mCNcoK1IwjcpTdYXD1dD3THxW895UmPq2K3yvURmcusATNShYzJe4q025porFvjlTvJbKl8l0o5YIV0XcAMRx0\\\\\\\/hvECB8QI1hPoDWVtxe5KnpVfftUkTOB4DJg23Gwus2eWRxOliR1hj19+U+dV0uyiWgtfRDtT9sqhNjruIXhh0Rn4n77fKHMesn6QV0S1kQyL6pLvOdVKZeHnrF0+gjWgDqdA\\\\\\\/NNRQY\\\\\\\/WyfJnEJrvOAms9UNuT+IbHzoXN6Iw9lWHkuN4N0X8suGnAZVcUsIQ5d2VBmB2caDyf5VGpErSuOrPbusB0pJY5mt1ZIjAlD+EDIU2i2s1nYmsNMfOshG3Tc2mBAOR+kB9XhYh45oSzPvPKUs\\\\\\\/knuxZPo+lg3tflt4PgquRBnvUm7qFRBTpGSI5+5EGpKDbfqRVrosDRFz2i0kVZzBLlhuZTyrTT1vJQLQJbxH237EvWM5fsAunTzgR5VSX815WgSHsr58Uojg8S+Z5z13FFSdnc8qHLDLn7nAz2eMnavX9OzxKAb49d1WCQQpJ5pzi\\\\\\\/86ADFixFwxfcwHtdArdzVReuhwyE06lO5qIqgFEBlo8SvCbr3ofz9iIVme42NA3f\\\\\\\/x7OwlEgznPZ7hY6os3AA9a+5+\\\\\\\/JmZApIz6c2HszgP5SakA9a8\\\\\\\/GbdSaKioy1hSMRUhwVOG8hjYD55tl9FOD3cBsz8ghAlYJgHBZSIEmDPgmv5IfWd3J1TfJNXCA6SS4Fsk84t1c3i07t2cEaMIq7SbBwL\\\\\\\/1VpLVCKt3QiA0xmfCSHfv69N\\\\\\\/qujX1ed1M1dtqDJbKdbFYXvMF2L6ua1DGtlBj1P8\\\\\\\/1HhnDgcKh+HVjNPwGQ\\\\\\\/dz2DjUtk6bbWHemTlJYp2D2HkcFoiujrtRVuckpl1KjXnoZu+kR7aRYQgDFWbodhe338MO8zBZlO9f9e7nf9Xae3yz5mp3r51SLTYcV5RLOv8UKbQGgYovT1R+3AIPSbUqQOfj75wfMGYOEo6jg97CLJE+bd9gMGD1RJgAEdaK+LmBwpTEd2OBGlECAInAfHX68gXet1dF1YIXsiGdkog8\\\\\\\/Zah1U7u+JGBDXyAx67NgKW484Q+IkQKIPRrun5K8c6gPQIB+A8P2MZolEVEySniQDZWbAsn0jUVLsDweH990zqaDfTv08rhCoeafyxzChXuOB3o\\\\\\\/B1MdKHcV2PA1J0Rfv8IY6fRwCHmQn4KFKc0fSRYQG2YmoJJ2CynTLdoARZNkFCG583PeWGPSbDOlpt6pFrolz9QVX+oYGeDOjwLf6WiaK3LbGfrpXYW0a2odCEtWKN9r0egvyYiaDzFSgYzOzT9Ev76ChNUfEMv92Xs2KTnCp9xzOiKfMGs8kirj3J3DnLeSLqFrXyPiyuQP96hWdWTRrmdw9aNGg+RbVEO3CY+FNYOHlNcZcUjmhcCc90xaxj9GkD8ul8exBGTgFxxz4h+qqW8oUq4gxdRmLSAW+ON8zUmpJXrwCqQ2ApxUNmaTntPa7dW2uCOQp6\\\\\\\/MVvqY\\\\\\\/xsdR2knyVEWi8od6B1FzkC4seUNMHHa6HEU7b4vehvB7l3i4ItaHkTVKurf099NyxEdihaGBYKseXTR+4g73M8DFRZHELNyAKAcRevkRUmtYr6qQqJ3DtBrmoiRuySGRoxtkLoP0+dWOyPEQMMdLw3pG5MpdCSQsg72g51TfSM1\\\\\\\/qwmTwzVOqEEyI1wHBRDVtv338T+g923VBOgta4f8\\\\\\\/TxJE9oq3FVVE1TOnhjNzZCAC7XFH0\\\\\\\/wfmWu8Yp8GmRxuy4W5E+1Rm8K96hl0y8lM+5ChRuDIq1j+\\\\\\\/9\\\\\\\/1lxbQm11vD78Kyko+44cZ7MwUzTGT6PfcTU4\\\\\\\/yEigwV4gqDZ\\\\\\\/L4GcOl52O0YZEjuu4lu89IvbScqgPwShtvy3LDCUFb1xk+Qfyu7mOzqY2obkQmDkMjav0Q0C0FsTLw32uUOKq9X8rOGpm2jEE1BJA9XZdw4pQcBztZXGAe+M3Vh5tiiAQsMlRG06YKH\\\\\\\/hzn\\\\\\\/6VZ2lOU6lQ5kzb67H4bhVVJeLY\\\\\\\/CL64f8\\\\\\\/upuYIiMlCJfZGN13VpeRORuVO11EPvGnwGKlTKtvb1HDdzbiY0iwSkz0KVqcNkYypCAXH3J4L3Ie0QS3pK9nihFAAY9cjgJINJ4CZO3Rpv0WGhFGdO9StF3WLIh1fQBQE6sv3+5p1tjAlFldfKjS0TC9LtxStEfmfFRLm4IvB0bju1fbaqc3Dc8vAh\\\\\\\/GtPg+32uMjwqsY\\\\\\\/M6ECoDhfpDsVZ2hFXfl6K\\\\\\\/qO\\\\\\\/yq1Xb5vB1ivrGLhuIlv705GQxzBOycrfzznlO7U5Kcie6LTdgw9NQ1m2H5pHfNKclnS77cng99LxniPiffo5\\\\\\\/K138j4Ldf5\\\\\\\/IQNywaD8ePuIwtfpPVmSQwOrysSjebxdHcGQPQC6U3oOcyFtY6\\\\\\\/xxN3BC7tMmwQrQJPLBbAJg+LqIGCoeQEx7m0d5ETjmZG3hCWC9U3suvBpoilecZJfU1S4n\\\\\\\/IcKDYcLleFGbD8HpljOx2jc2mnhnbhrswmVpTAVqvdGcJuVaKRUY9hNg4DBhk76Asjk8XzivJBNq4G2XH1xidbVUBYHJZz5iIJq3EisuWu5rF0\\\\\\\/\\\\\\\/NcYsynIu4+CYe2cbQpH9lAm7D4q0pQDAb8a0HbgJNw4AGiUXu3mYJB0SrgwCkQqslghNRrfhQRQvBxRy9ZaxVwrvBCa2OmPG6hrdlSqZwZGJkEzFwAVpCcos9xS+Y0QRlemp2HvGR0HsiEKYek9dMRhaM8l+It9j1v75qFaJYnMzvuzgnEEsTP4YhHN8TYtOuFuiLTqniC5fjjqJQfBm4IeHRclMBzZL86X9t0KrY1VNYi+qBpO4vpmnk+nZs5Ei4YS70O\\\\\\\/RvlIW3ALphoeqdtPbTk7cahgbBh80U0NqmhADqJq8mEmodhCSyu7ljaEFxBr2b7YnY5JpNB5OGWoD7EuB+tf5TGT6OXx2DZR\\\\\\\/JQ4Q45c\\\\\\\/a2vBmiS+FrbcbwhySJkrgpk9kHgdE4UUIi7s+ifVHAjz1su8nTwTWhYBHCbu\\\\\\\/c43JG0c7qtqYoSCJEE4XYZtLrrIlo340iOCNQ8E7YrlC09k9DrJBw2\\\\\\\/LpxBqND4XJ6\\\\\\\/uN+frOG2KviXSc0cHu86QWlOtDg22NRD8xXdWcODsz3pLe7vey0+XbaZLNAJ3PD5sgJ6hAMu6kVn+8Xn5UJD5IHwrn0FlQDD7teTekGdq7ZMT7Uia\\\\\\\/nUjVaZHqNh34RtpiAO\\\\\\\/dff11gPFRNI8wk56VBRCo9rGP5ObBzucvHzlJVeu9xmApodcW+JfE9fbTdsziMj598kBKiOdcVmEegd4fLKhNBp0uCLSKAq+wYhaVqkDWsHwZoiogNRQAsLh9RMex2nUmvWbaiioCrCWCNi0pCFeoN13wv8Kn1Z9CZVWYSUpRk0GRtCSydHODdDLoMbfSeYzXCs1c3N4NObmUp\\\\\\\/pc02qTZRIhaBFbu+CeGzKUNNfzBVKLqT0go5FOgcQhm4rQyMbZDI+E+A2Vq3bQswIA8Xq\\\\\\\/gP\\\\\\\/IOu81jEUwF+qNIS2MgTBzpHN158NjVC6LVAnSOHGhqYte5OIltgnJIyRDY4SzTTbjz1j27CJgwGfMJ31k\\\\\\\/GShCVGJ50ByB1x2Yo69R1egS5Z0lurNMF3VBsK6LYtm+Pz7srixfzsPP5x8tFKOuXXhZOEzwxMZofNe8Zt\\\\\\\/ayzMnQUf5g37NFuV5833d5fYUHebmSD1s5+K2VBT7qKnW8Am1qEtau6VAu1nCEvdmU9N6SYRUoD1s2hT4sEW6kQhh3haakmtmFGC2DvqeBGAEmnxzUm3Tj+Q2ZWAD7QFhz0Orz1Vg3ain3wca4lpD8Ttvs84UwZdn0p3A+s2xfkL6fA9mc9bUDKNrhpZPPfYjitryd7RY8p3cfscj6mIXRjArfISerwnwlg+ZIONwX4eURYRputBZQIRRyJZIy1yfZQFCn9BQoGc8UChY6H4WEBI0xGH+7YYc3MPU+MkBYKnk53EEiQxaFB2Ov9hg6UeUPpeLOCZBecVKgPd3uW19z8cngEDkZ6B5pnAjPhK+3adHWphwX8ZBU71TNRt5q\\\\\\\/QjF3yW3rr6tqHoNdm+17djH4ueq81HdCRa+Me7DwBuAHgq3ZWpTLkMZsd4zFNdNL1oudBgAOV6bk3X91FfmUWrIDOTG19mzIW5+MC55ueD1ZuUq6N5NiSjR1fEdVJeghVMvk074kCuK8h6LnIerQ7vpWeJDgg3Rka9hJ5VcDhsOtucNFVock9QdmqrxYGwokjXeDPL\\\\\\\/NZg\\\\\\\/QrbpN70BqfQf6AVv\\\\\\\/ub7i6ZB+2Cuigly1tUnS0qLpXb6cGG81DZEAPpk4sEjjvjdTVkt4erIgPXvlXqisUCetMzHQn+UTgYPp6vmTvzURYukh2M2GQfbRU4In8Vv4r\\\\\\\/IJl8rUG\\\\\\\/ldOanT+fee0bWH8wwTbrV3ivSDcur1HO8DfOPdnRnuk3xP1ZClkIbBb7AaAj6VnYGX+yLExR72FYYUt0BkJhczmXPEWRaPBpOZ9jOjm8IaFw+1KfoP\\\\\\\/1i9mH4oHakwHuMf6bsDNowStqINzBLPjxEbv1y6jsyhzxkpcagC7mbeurEZigwFgvJWWs5WToCEtyXmo\\\\\\\/zbtUQnA6m7C\\\\\\\/e3NPITfeLvdlgSm5RG4dxsm+SbHzHcBUlRNMU5yxRoMwJ\\\\\\\/GsT1mQpk1jdazAo6le+BIv4bQDdr1a1j8vhSLjS4TdZB4J32fHggEO2hnbenV7tA86mDLjGhJcITQXcp9pX8R30YmCTYpi6XLECg0aC0pgb85YpB8O4b352DB5RW3e8zzZceHuqe1HrEXrqa+3qFkjfxUISCs7wRtWl1uecMToYObN82c77bhvJpt\\\\\\\/OoZBmWzgtoVqB4DTFsrPY5VdWY7gXY284zwlKVzoIoFVBAns65vnR15vTo+HMmHrtIgXFadq8ouxKlosXpbFju+6hlwsLbz6xSezGJqJaNvXUFsovAib3SCgHE3Wtnuw\\\\\\\/o8pA9x3mbOHSFl+oSr9AIDxAw2b5gMRH4YAhAAxeixkV5gUDLA9O4DC73BG1UdCqn41Ua\\\\\\\/CModT2EgE5pSVbdhNIMaWUiscwID1uwnBlzi6HH5ZnuCkM2zfnv1XFlBs\\\\\\\/nJ0j41TEecbNI07fyBHcp92PLPSXeiXG1UZMwy28ygjsZc\\\\\\\/MY3vslN4QLbM2T2r9EAFjrmGkAOjNLz5ozJxwzaZfaUpButs+oCusfNcHq70dDfKr0Ya7s7VYOKs0DOQ071OQXzcAjfFHtsFi1q5CSjGVkrORdUoGCCUWv0SMKr5qOVbB\\\\\\\/bfyeBmXGTNhKcJQ0hlEIMrRZctqUd\\\\\\\/dKdmmdiJTz5hp+9Jh+tC4GBu7k1eOo9R5zLim\\\\\\\/tG1myJIWvtIqrfdiMVe8KF23STDuz3P9I+5IgM7v27eMUTOysRegIo75UtYlW5yY0mzBqQdpO10S3f\\\\\\\/1UjhLixI7GUCcxPTKOutfJk5ifpcuHWCgGFHCCDA+oOpNiTLU\\\\\\\/emMdQ3Uw4rfhM8cFK5bt82a6br3Fy5VzBBf6vAihfi\\\\\\\/xXauThF00Z\\\\\\\/Xv8VBT19Gm8r5YVp39L1OJdJ4KYjPDSE8Fl5EZ2nNS3fmGjumhslKi4JHNcktBnaSyD383f+1BagWHXTIhfulfrD27Dwu3HY5K16h7pviK7HlWChaujmrDrKwoMlmGJ1FDgMTUM6xLbDuTx\\\\\\\/bTO1pkNRO1yAHhQZhuIvXyH\\\\\\\/wO8yuM8i3i9LR4+Acy5frNbxi95A4A6Btqw8HmelqugTsYWpAEYsBbvRoaFhhJWhmwyboYK+FiBXYbpLM9ZPfzl1jE3rejFjExxRHcwcN4TLyi4IoVUSd+i4pnGXvTdDkejKKIUCELRi6Z71EVSbl78h8fiJe+2LXkK\\\\\\\/TI+I0Du0W4lg7lTySAGe2zChm5hMwNqCNRqqAp6Hns+w5\\\\\\\/N1LJ7YgJmHjfYB8aNsmKRxH9Uc5GUkZEfWYAjt4X03fTDWPvtqavY+ag9l+k1UAVO3fcZsxoK5Z2znr0LTqgCMOlggF5vnrcycyVf32g8DYfyPjG\\\\\\\/DKanZ2Vo6vr\\\\\\\/5bWpfQuPAVrtWvwCBvileGh7m1nCbmfcD6jfiFPsnJ9+3zrq2twjgBOKcnPJHaOsDcKPBluDKGrwErGsOsFPvCNevfjzcr4HCICqiPWbkTMyIN32Q5tFeiP89T5\\\\\\\/Pdzp2uFlWLQchojiQ7Vjfa+XMCZopnmfQzZORk0xWLlP3geFoHxdC51jcU1TiTFL5HHorju7uiYGk6gAKTPq4H1BtuyrZGO4GHBgeXUHobsUbemSxh0NCfhgZlPwIAhBzuEF6NveRUqDJUds3whEFuUjjzWe3pwTpVUINz5OWNx9q3wSMe8xdYn8JxKePUAOIdHKlVbSTL\\\\\\\/9b8yRMPWOmow2vYY8GaxhwXGqLzgL7SqsW1yjsixVmtD\\\\\\\/q8iyKEDXsy63G6ZGTfn31Z7zhJSSZ5Eco\\\\\\\/Chvk9q5eQis\\\\\\\/O45tZDwZQRQbi+aVGze4dLO996+b7TqiciJFrb7KQQNiuE4lFgoK6nkYMb5LQKyKmpgOv7VV4abAyIdQQbJo2tvYSg+Npm3LrhquOPF84xrdP7pxPhNuQRUnlKj5maYV\\\\\\\/g6uYw5XyjILQn5RynjGjdL+mgcbNQBnfJXZLDZgcXP82DxpYOs20OgPwhWxUfKcVmsM8pe11PcIRWsJ7a\\\\\\\/rft196T16TrzLH6UWIIsZsuYdO19oeob1RnsROhMaDAt81JA4x\\\\\\\/PksdqZ8aQdIGjTeB4JJa6f0wWehhAJoRsgmsA+WIVlkUsKaqL6zS1STDBc7NIWQ2TJgjg2ZKmHXGXlmW8cGzJUQjFStuqi7O3w8BcptROFYpvwk2rGsEaoRizEaisUYmOVdRREoK4YeRfUoEsTnUw458oq89bbKQOJfQZ7rBTa6eUFeM8jnUOiqbxDx55UQcSYPDYt8F5gYDROVMScvLXeuPDXSmq6yP31I3h8v8elUYXWOl3t4VFc7lBEJCdACvq7eko6nklNB7lL0biHJwGHrw3V60EJKX2Y4GHTqgQ2xiX8AEZ6yVZQkNk\\\\\\\/dnp0kVULz8E17Q3c5vIpfPg0oqOXmPKNUn\\\\\\\/omYMCLKaCNbsq3FULVX5ncC1jw\\\\\\\/\\\\\\\/lHBNXgwH0boh+SaueSxopuqruXnJa8EXT8bGkRmtM6Y9OsntSqNXjAu4g0RvRn68\\\\\\\/3UbulDkj4rnzlDC5+lh2P\\\\\\\/muaUQA5XVtYcxGMcL7qKmDkXXdHPpy7wahix79eSraIl3f8W5kcNWIa+0u5uLhyA7Eec\\\\\\\/Mq32u1XBMj1425H9TsvsyfPOrknTtrfY9NWgAM\\\\\\\/Mu5tdlExD7vbYH48yvnqmm3lBxNQIGbNUgjLi4G12ZGQ2dlA2KQKKqjHa0i+ZypSdvJbIHo7AX4SdE2W4S\\\\\\\/BMq9BRnX2r621Howtc2v8otKlozxvl\\\\\\\/b21YFRndiEuUV7R8Wbn6ACuWLUVfnnr4RcnBRWm38P+PJKJHXuJ85Lig2O\\\\\\\/NtcFMX9R\\\\\\\/\\\\\\\/p4QUZ7Up50YqEgHpzivjaY4ehyIEpHacZpmQjkurr2wE8WTKcHseZUEpD5KtrezQDbBTB5FqFcsOZi2CKJGmIxEopZe1CRnhrkawxAby6LYA4fg7RMqi4fe3bUDc\\\\\\\/QZ6ZAdJFfx9n6WqpFrC5NO11N7QHb9Ugd9GmmlhA+zBEW3mAn3Y7TyhnliTr3+mXE+U0\\\\\\\/e2p42UrjDUygOOAX4LWYl03aZoKEIzqXRZ\\\\\\\/Ao8CICF7NH7KK\\\\\\\/dQppfW06O7ABxtdofyJ1Bfv02QMzWaBoHEgKSVtCPHvh\\\\\\\/kFtAezwtZ6xeY5+DXRho363g45FtELukcl6cW5cPRrQkj9Bkzkk4l0xiLDDGm8UuAOqzbchW\\\\\\\/3v+AOXKEIf2EFxzA4gG0+SY7a1XtsoYETrf\\\\\\\/qeH23aXLlBRJHBctKze\\\\\\\/PMmxGjWbBKELuyaPmse+R2XQqQeE1d49I6eEVAAioEovAnXIU9IZpJ6pO92qXF7dQA4N3VEK\\\\\\\/4RIJeKP2ZwCqWTjf3OgJuKB24V1Td0HoFjq83kNjpLRW13YBSEAzQV+Dq7F0wt3zV6kgY7cOBGBLwr6jdtIO0kdndzM+K\\\\\\\/PtJQuUrEdfD+yWqnrB83Zarm2pqKO\\\\\\\/N2b8fXBpW\\\\\\\/rh8Gw488hXDdQpDG4YLxay2ZMFjCqlGbj40yssDx\\\\\\\/rkm\\\\\\\/tfxXuc4oMwrDWgsVFvtD80Czz3cXqOHQOT2IsqXTT32Roqd7smuaY+WO1n8GoG7Qd+0jAjaoM3P8SFrWYtjPPgl2E42HORdHhlXlM3b4FI8xKDFU6nqRDwBAqtlTgEp4\\\\\\\/5PT+3zFbw2mpD0I7VU2cjpvm13\\\\\\\/w90TR\\\\\\\/wp15p0oPjbtXr+G7SlfCg8bbcKtWDTmayJZaxZ1V1fOyZ32EwUgUmJ0yMwDtsy1fd6U8ORl9YgmIma7EGr8LGZEJPTVUhc7QJRIGlsbvAa9ag13gchCK1TcKruT7lD6TTgEDkl2tWO3H2U302EMjrDmD5kuk6aj5+r\\\\\\\/p9Mp1Ti9IhUEJms+Ws3kfbZ6Xq6oML1N+pNI6850XGYa65q3njtcOvbcK6+A5oRIcLKYtZ8jA6G4SMpkh5UvIN6zKqxCi2WuRsQ89wmk6VoL6kp\\\\\\\/s5IAM3EJGKq4bZqgeJZnj8dTGgor7zWRG7dBCkaaUj6qtBJDDuRzNfyJiPdvIrfRwneBFsFsQ355JbiZiep6YG54NYRROpk2wVAaxNERYpQzeLD9dYRTJAgrAFyv0D66Gz9qprrgcU5hzCC+GV\\\\\\\/JH4uwar8Fp3IOrHUCdpKIS3dGw6r5kGu4BRwJ6da6PnYgG6EZSvZdXM4yToizA5mnCbg21dwbfp+DZIjvpLtXgXmz+cqd8+o8z\\\\\\\/90Jn4laAUeeXVzXs6X9U2n4ajY0mwc\\\\\\\/MEZwBq6n8i8U2QqZFr9yYurF1r6R8g\\\\\\\/3Td9pEpJZLLYONcsafBiqEG9Hm6Lqo0lsQvR+VVOS\\\\\\\/GeX8h5Eq3wekhkyEh4xJY9bDou7OShxtY8nZYhU0F8yfHQIrsTIhzokOwVia6cb941czfx\\\\\\\/RIDKe3seURowOFpEI6tylzrQhBHFIaXyvArhs51K9RsMxv8tsLJVUkxfBL\\\\\\\/06XjsFQcvh0Bmef++QJJgJNVXVI9SzzNhm3+L4uNF4Bkm0a5qehKvKn\\\\\\\/DlWuGttsQXaaA6ojJxNwBw+2Q\\\\\\\/H+UIp4HLxqoqPbwZaqz7xYHeEw1BT+jfdJQNzkdlCdhzcDD2+u\\\\\\\/tmXGh2aqsfWWiugSufmcW9gWMH\\\\\\\/1zklveVWq+6N3fbHZWJkLVAXC4wR8TLaF5fZhHyoJetmrCJ63QB+JptboBb6zKZPulSnFDt4jKkD6Yd9J0PYccBf5gLqIZNjYYctgZLScN92VsLqlxXl0qHLuqtWJCHlaey77IrwJpX6WoUD1lyEnpw\\\\\\\/zWfw5pcHwfxoOWiZlUZpWyVW94HpiZ8gMBhHCfJ+w+dRuJu0FW+M41ipv\\\\\\\/M9th2Jl\\\\\\\/6PEjuJj9uIKic9t04NwGc6a7tIRs7Zgv\\\\\\\/VS2+Nipx6045cqvt4NRAoNO6kgubGSpazUIxrF2xWKqY8ey8fh4KgmcwOBKhnndZicK02LCX1okZjmA91+\\\\\\\/1UZ1bKNfXaS51JprQER3ujjM1o9mYNQzRbhtwolUn2xBGWA7p8vru2ZOsKiXhzeboHQQZ5ubCxZs9wpCkKv7yE4q1m9zJ2bd4cjegTVt6Pjue1i1f2\\\\\\\/MkiwWos\\\\\\\/VE0w05cjr53FKVNO8YMRIrkS\\\\\\\/M0iH3V6zXXoJGxjPUyMcEm2Umb5Ren0UPjrRHu+b+3rW8ltYexCSCVdk\\\\\\\/EFYvZjYJXhKmvyjFh\\\\\\\/szPSWZ+mBo3ldaGEgfTVdM\\\\\\\/4tj8jhfJhajCWNGvXGtatPvBH7iZ+ed5wYFVjxe48lHB19ooUKUrKsSSRAuMk0k8GZ+yzjKlLar4XE4\\\\\\\/nlgRJgxVZ2O0mKDSyHmY\\\\\\\/QCXhIsSl528RLSsfo+ZIsU7ZsKk67E6XGa2mPpJ\\\\\\\/A0tAQrS3Yho57YQzSTggZGDBzXhG1i6B9Sjc7dFvDRl7G++zzYhEBU0nYf5dHcucDKL7YnrPx1aarvzAqgM57FBuHpjR7hgyqyDgs2OUoWXWMz56FRGevU5J2iyOxQdoXgvX1JJsxpKP077+gIz9I5w9mN8i2ay0KLXGp2mByL10MwccaSNCnNsbeYtdL7Ej9BvEMfihtOfIUhgMf\\\\\\\/RcbbzkiV6KDVo\\\\\\\/mwRuBUWfDyXIbbUTNwmnUeTYe3xk62HGrNks5u3JPDyfLE97hiFTuDUTvHJd1OO+pdUuv1ccDSYcU74Oppe8YDoWgzQp0Lh1OVA5O6I66vETyvsAq6ltgCWlX2GlA1hxi2vx48uF9yHGAlsHOF4+1KaaLyQtsvGuS8JWJkQ\\\\\\\/RjmkxHy3\\\u0022,\\\u0022iv\\\u0022:\\\u00226KXcul4RkQkvanTogZoufw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229514b02b5a069f66e9798b6ba2a9a051116b8bfac21ddf84cff6836858a641cf\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VeR+o4mmnAeYVl3xEwzoqchgOkgdO7lLp2ac67+PIG8YNBVte9Kht2f7k532eLfZM\\\\\\\/GS4cNJ+L4mF65x83H\\\\\\\/JqxhKquhGw5Si4ODu6VLPO7+P0IRpVZKj2dvoQM48wqlEAi\\\\\\\/GQXwgwwvcd0R4CejeBG8brE2WNRmRqAo6cxWmyxWLRAb+oQ5PiQIwSQbE2Kwm+OQTtIBPJBUxBLT9B8mVILQsYZT9w\\\\\\\/nNrOr0WlvnDmdL2SSVQL664Akdv1OqCWtEalz\\\\\\\/ldTfB3K\\\\\\\/m78LxOyCDorBceTiPXw4nXXQCIb+VRUqglayhhdGhX98r31uzl1FP+3GZZ\\\\\\\/\\\\\\\/SbKsmveQTdOTXwHZFcYvyC47Rz9Fi+y1G6VU12RCJj4h4IpnNe\\\\\\\/Y3CFoZa7rECXE0EQOMf+z60gJ56g4eynhf2pwV\\\\\\\/1oSRdYI23dKvG34H2hJlQVgOMZGjrQz07qrX0oplhX1yYXIyGrbRquZteHQXbsJZzzmS6QRB6RwE9unT6mm3FS6OlDaRjabeP56KFHAj+YFs3rk3VC38nnmuzEEes4Pdk6AJa224J+jG8ZHw6vgcjf\\\\\\\/WhbynRKkfrauyQ3InZtaPjKh8v\\\\\\\/Cw1dtgomqcIm9X4Tmr24z7Nc8jDks2sHchgR0uFEOOC8Lo4uOBRXu92vtgNBy2cldgf6tV4Fy4u2b1zeEvlUYIOMFs+XNyfLcmiZmCrOTdZMy02Iz7ABB24mwNU0aW\\\\\\\/Hea632ZbHJ97vqwyrDReqLgHHLA+6nnVO13yeihl\\\\\\\/yVlVTHK2WVbNSwD3AoHVnBRvKcY698aDLHfFaYIwRUPoQORd6wx16fC2ofJTv82w2g9ZMDrsyr3Xcsh4eoX2sY3LxatE1++b2lDdeEZSKjp4tbTfjWXNS1EuBL3JiQZG6sg2k0owtwN3\\\\\\\/nVuVjC8VXo1Jx42LyE7KDdKkL7o2Bnt9qRdA+VFPbglRKGQp72vlL2RLec80M9HWWazSg1rHT555aCL0cCLBUjILx9gU8iaMnXLHn1oZAP8R6TK+oNVZyuF8dKwGk8k1wSyAoBGbs5Jw4hnliLnZUUeKZRFx6zvGtZcYuC6z+GbTmkDUtijCwU14Lovn0P+69VFzayXAZ2zESzQvNj42lq5ixPd2kJu8XJOfmK1v4\\\\\\\/H48v8NlZnaFLOH0W2do6oa\\\\\\\/vK3PCjMIE2x8fcBsUepZlacMpc3K7WJobA3D+ywFYkiqHfI4Hx\\\\\\\/7G3lrAOi2HpvZokOqqZjPB6RWpWwI7ubJdmhopNX253xyHkUDT\\\\\\\/UhMTPn4phN7E+VEDB8b5VZLlZxsEAhD7r8pEbYAyozUnqmLWGUbpLQvU1Q1H505eYoRTPfZQNJJ18YvMu5sHI7bFNH+tkZXalM\\\\\\\/qpurABZ5m6YanaqbJrKKYTfvLbrqbtEoy\\\\\\\/0bDnJxALRi+sYHQRtzYyKzlfPJeUou4wwFeFaIoSM855BsniIRkb0Sc90U1AzWparbkDBfthpxTttaAhmVSmFP+6BqHnYu+c0\\\\\\\/65+Bln7CHv3vxq0P0vxMQ6Fd0cN5npyR22Q9fLPIETsopAiGbYVSnFnWnyH9Hg\\\\\\\/i8Pb4DrUGXbEXRX8rbTHNyl3c2c6aPtoDTb+x3W1xClMoq9OWnpAt8l2\\\\\\\/JV9o5ES+bFj1rlz9oBuJxekedcmZgVWYVo4yCKAWXEX2sFBfBWCLEJU7O7UEH1AE6I3xWt1KeSDtEmqQFqamgV9TCnyvYltmcnv\\\\\\\/m3dMaIjs6zt7rZh9a9XmU9shZXaWBuxQ0rvygEY6T7Tefsd0YKbczMQLJtIrHZ9WUd8ealWjfMmhHcENdxdfpFHzZLxjQ1ub7uO4QSfwoo7J5NlvvOigCby2ppPdL6UZy6slA\\\\\\\/DB+eSWPbWLQMLMpK+9B8wYdvtgHQoTnXxQbdjAUeKB\\\\\\\/r3i3n5CtiqzoRujQUXTLpw4qAFDapUvh5bkpbTwKK1VsAp7yxFtygUY9X+aaml8vbmTxBFXDNzgwZfwqBKZPoD4UjYCHfP2raqE+NU+dFgiRt\\\\\\\/O0qLd+Z\\\\\\\/FEngz\\\\\\\/r\\\\\\\/7xrRe22KYrtzkZsYS8+8NQflH\\\\\\\/Z2U\\\u0022,\\\u0022iv\\\u0022:\\\u0022rS0Y5BhiZWOVEfE+nUno5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002216b6d92b62b1e905cad989edcf250a1a449778a09465a445057d57f07e3e672e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494061 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2963" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cpsujCWNq0fQRUbV7YmJgIBuvcZGBfNdHA-rh0TSK6o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2963" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HFXrJDtZ41Yq-crW4yYTIluVPox6zY2M-VUzVGXjZFs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2951" title="Combining Transcriptomics and Proteomics Reveals Potential Post-transcriptional Control of Gene Expression After Light Exposure in Metarhizium acridum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2977" title="FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494061" id="node494061" data-pisa="ggg;9/9/2963" data-pisa-master="ggg;g3.119.400228" data-apath="/ggg/9/9/2963.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (<em>Avena sativa</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Maryn O. Carlson</span>, <span class="highwire-citation-author" data-delta="1">Gracia Montilla-Bascon</span>, <span class="highwire-citation-author" data-delta="2">Owen A. Hoekenga</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2452-4779" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicholas A. Tinker</span>, <span class="highwire-citation-author" data-delta="4">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="5">Matheus Baseggio</span>, <span class="highwire-citation-author" data-delta="6">Mark E. Sorrells</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-6287-952X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Trevor H. Yeats</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2963-2975;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400228 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494061" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maryn O. Carlson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maryn%2BO.%2BCarlson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlson%20MO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaryn%2BO.%2BCarlson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mocarlson@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">mocarlson@uchicago.edu</a>
<a href="mailto:thy6@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">thy6@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gracia Montilla-Bascon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIC, Institute for Sustainable Agriculture, Córdoba, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gracia%2BMontilla-Bascon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montilla-Bascon%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGracia%2BMontilla-Bascon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Owen A. Hoekenga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cayuga Genetics Consulting Group LLC, Ithaca, NY 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Owen%2BA.%2BHoekenga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoekenga%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOwen%2BA.%2BHoekenga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicholas A. Tinker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ottawa, ON, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BA.%2BTinker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tinker%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BA.%2BTinker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2452-4779" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicholas A. Tinker</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Wheat Genetics Resource Center, Department of Plant Pathology, Kansas State University, Manhattan, KS 66506, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matheus Baseggio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matheus%2BBaseggio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baseggio%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatheus%2BBaseggio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mark E. Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BE.%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BE.%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div><div class='author-affiliation'><span class='nlm-sup'>††</span>R.W. Holley Center for Agriculture and Health, US Department of Agriculture, Agricultural Research Service, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Trevor H. Yeats </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Trevor%2BH.%2BYeats%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeats%20TH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATrevor%2BH.%2BYeats%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6287-952X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Trevor H. Yeats</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mocarlson@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">mocarlson@uchicago.edu</a>
<a href="mailto:thy6@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">thy6@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/9/2963" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494061" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494061/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2963.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494061" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494061/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2963.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494061" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494061/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/9/2963.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494061" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494061/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Oat (<em>Avena sativa</em> L.) has a high concentration of oils, comprised primarily of healthful unsaturated oleic and linoleic fatty acids. To accelerate oat plant breeding efforts, we sought to identify loci associated with variation in fatty acid composition, defined as the types and quantities of fatty acids. We genotyped a panel of 500 oat cultivars with genotyping-by-sequencing and measured the concentrations of ten fatty acids in these oat cultivars grown in two environments. Measurements of individual fatty acids were highly correlated across samples, consistent with fatty acids participating in shared biosynthetic pathways. We leveraged these phenotypic correlations in two multivariate genome-wide association study (GWAS) approaches. In the first analysis, we fitted a multivariate linear mixed model for all ten fatty acids simultaneously while accounting for population structure and relatedness among cultivars. In the second, we performed a univariate association test for each principal component (PC) derived from a singular value decomposition of the phenotypic data matrix. To aid interpretation of results from the multivariate analyses, we also conducted univariate association tests for each trait. The multivariate mixed model approach yielded 148 genome-wide significant single-nucleotide polymorphisms (SNPs) at a 10% false-discovery rate, compared to 129 and 73 significant SNPs in the PC and univariate analyses, respectively. Thus, explicit modeling of the correlation structure between fatty acids in a multivariate framework enabled identification of loci associated with variation in seed fatty acid concentration that were not detected in the univariate analyses. Ultimately, a detailed characterization of the loci underlying fatty acid variation can be used to enhance the nutritional profile of oats through breeding.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/fatty-acid-methyl-ester" class="hw-term hw-article-keyword hw-article-keyword-fatty-acid-methyl-ester" rel="nofollow">fatty acid methyl ester</a></li><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/multivariate-gwas" class="hw-term hw-article-keyword hw-article-keyword-multivariate-gwas" rel="nofollow">multivariate GWAS</a></li><li class="kwd"><a href="/keyword/oat" class="hw-term hw-article-keyword hw-article-keyword-oat" rel="nofollow">oat</a></li></ul><p id="p-4">Oat (<em>Avena sativa</em> L.) is a nutrient-rich human and animal food source. Recent studies have revealed the numerous beneficial effects of oat consumption on human health, from reduction in cardiovascular diseases risk (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grundy <em>et al.</em> 2018</a>) to cancer prevention (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Meydani 2009</a>). These positive health effects are likely due to oat’s unique nutritional profile, which differs markedly from that of other cereals, notably in the complement of essential amino acids, fatty acids, β-glucan, and phenolic compounds (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Butt <em>et al.</em> 2008</a>). In particular, lipids account for as much as 18% of the oat grain (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Halima <em>et al.</em> 2015</a>). Moreover, these lipids are predominantly composed of unsaturated fatty acids, rendering oats a healthful energy source in human and animal diets. In response to the growing awareness of oat’s health-promoting properties, nutritional quality has become a key target for oat breeders. Maintaining and/or optimizing lipid composition is an important component of these efforts (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Valentine <em>et al.</em> 2011</a>).</p><p id="p-5">To date, researchers have primarily investigated the genetic basis of variation in fatty acid composition by mapping quantitative trait loci (QTL) in biparental populations (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hizbai <em>et al.</em> 2012</a>). However, the use of such QTL in marker-assisted selection (MAS) is likely only effective if the parents in the mapping population exhibit trait variation and are closely related to lines in the relevant breeding populations (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Snowdon and Friedt 2004</a>). A genome-wide association study (GWAS) can mediate this limitation if the population in which the GWAS is conducted captures the genetic variation present in the target breeding population(s) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lipka <em>et al.</em> 2015</a>). Indeed, a GWAS can identify allelic diversity associated with trait variation in complex plant pedigrees when both genotypic and phenotypic data are available. In several agricultural crops, GWAS have provided insight into the genetic architecture of oil composition in crops such as maize (<em>Zea mays</em> L.; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cook <em>et al.</em> 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2013</a>), rapeseed (<em>Brassica napus</em> L.; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gacek <em>et al.</em> 2017</a>), and soybean (<em>Glycine max</em> (L.) Merr.; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhang <em>et al.</em> 2018</a>). To our knowledge, researchers have not yet conducted a GWAS of seed oil traits in oat though, small-scale surveys suggest substantial phenotypic diversity is present in existing oat germplasm (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Saastamoinen <em>et al.</em> 1989</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Leonova <em>et al.</em> 2008</a>). Thus, we aim to uncover the genetic basis of fatty acid composition in cultivated oat as a foundation for improvement of oat nutritional quality through genomics-assisted breeding.</p><p id="p-6">Historically, a GWAS considered a single-trait or multiple traits independently. To capitalize on the increasing quantity and complexity of phenotypic data, GWAS methods were developed to analyze multiple traits simultaneously. The primary benefit of multi-trait GWAS is that phenotypic correlations can increase statistical power to detect association signals relative to univariate methods (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">O’Reilly <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Korte <em>et al.</em> 2012</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Stephens 2013</a>, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zhou and Stephens 2014</a>). Several approaches leverage trait correlations in the univariate linear mixed model framework. When there is a direct relationship between two phenotypes, their ratios can be used as the phenotype in a univariate GWAS. This approach lowers variance and conceptually targets a mediator of the phenotypes, such as an enzyme that converts one metabolite to another (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gieger <em>et al.</em> 2008</a>). Another approach combines test statistics from univariate GWAS of each trait in order to detect genetic variants with pleiotropic effects (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Yang <em>et al.</em> 2010</a>). In addition, dimension reduction techniques, such as principal component analysis (PCA), can be used to derive transformed phenotypes as inputs for univariate GWAS (PC-GWAS). By combining signals across many traits, PC-GWAS captures genetic signal associated with both single trait and pleiotropic effects, resulting in increased statistical power in many instances (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aschard <em>et al.</em> 2014</a>). In addition, PCA provides a principled method to identify which linear combinations of traits to use in a GWAS. Finally, the association between a genetic variant and multiple traits can be directly modeled in a multivariate linear mixed model (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">O’Reilly <em>et al.</em> 2012</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Korte <em>et al.</em> 2012</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Zhou and Stephens 2014</a>). We refer to the multivariate mixed model approach as multi-GWAS in order to differentiate this method from PC-GWAS which ultimately relies on a univariate linear mixed model for testing genetic associations.</p><p id="p-7">Multivariate GWAS approaches are well-suited for genetically dissecting metabolic networks, because the abundances of precursors, intermediates, and products are likely correlated due to metabolic flux. Here, we present a GWAS of seed fatty acid content and composition in a diverse germplasm collection of cultivated oats. We compared single-trait, univariate GWAS on the abundance of ten fatty acids and their total with two multivariate GWAS methods: GWAS of principal components (PC-GWAS) and GWAS with a multivariate model that accounts for the abundance of all fatty acids simultaneously (multi-GWAS). These multivariate GWAS methods detected several novel loci associated with distinct effects on fatty acid composition and previously identified loci. Conservation of fatty acid metabolism across plant species (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li-Beisson <em>et al.</em> 2013</a>) allowed us to preliminarily interpret our GWAS results despite the lack of a reference genome sequence for hexaploid oat. Aside from providing new insights into the genetic control of fatty acid composition in oat germplasm, our results suggest that multivariate approaches will be useful for improving the power of GWAS of compositional or other highly correlated traits that share a common genetic basis.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inferred pathways of fatty acid synthesis and modification in oat seeds. Fatty acid abbreviations adhere to standard conventions, with chain length and degree of unsaturation (position[s] of double bond[s]) separated by a colon (for example, palmitic acid is denoted by 16:0). We detected two isomers of 18:1, 18:1(9) and another isomer, 18:1*, with unknown double-bond position (likely 18:1(11)). The 18:1(9) isomer was more abundant. Fatty acids up to 18 carbons in length are synthesized by a fatty acid synthase (FAS) complex. The nascent acyl chain is attached to acyl carrier protein (ACP) subunit of FAS and grows by two carbons per cycle through the action of distinct ketoacyl-ACP synthase (KAS) subunits of this complex (KASI and KASII are shown). Elongation is terminated either by a thioesterase that releases the fatty acid from ACP, or a double bond is introduced by an acyl-ACP desaturase (AAD) that typically acts with specificity for the &#x2206;9 position and preference for C18 substrate. Thus, the initial fatty acid produced by FAS results from competition between one or more thioesterase and AAD isoforms. Subsequent elongation to &#x2265; 20C is catalyzed by the fatty acid elongase complex, in which the ketoacyl-CoA synthase (KCS) subunit determines chain length. Further desaturation is catalyzed by additional fatty acid desaturases (FAD2 and FAD3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-286196959" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Inferred pathways of fatty acid synthesis and modification in oat seeds. Fatty acid abbreviations adhere to standard conventions, with chain length and degree of unsaturation (position[s] of double bond[s]) separated by a colon (for example, palmitic acid is denoted by 16:0). We detected two isomers of 18:1, 18:1(9) and another isomer, 18:1*, with unknown double-bond position (likely 18:1(11)). The 18:1(9) isomer was more abundant. Fatty acids up to 18 carbons in length are synthesized by a fatty acid synthase (FAS) complex. The nascent acyl chain is attached to acyl carrier protein (ACP) subunit of FAS and grows by two carbons per cycle through the action of distinct ketoacyl-ACP synthase (KAS) subunits of this complex (KASI and KASII are shown). Elongation is terminated either by a thioesterase that releases the fatty acid from ACP, or a double bond is introduced by an acyl-ACP desaturase (AAD) that typically acts with specificity for the &#x2206;9 position and preference for C18 substrate. Thus, the initial fatty acid produced by FAS results from competition between one or more thioesterase and AAD isoforms. Subsequent elongation to &#x2265; 20C is catalyzed by the fatty acid elongase complex, in which the ketoacyl-CoA synthase (KCS) subunit determines chain length. Further desaturation is catalyzed by additional fatty acid desaturases (FAD2 and FAD3).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2963/F1.medium.gif" width="440" height="268"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/9/2963/F1.medium.gif" width="440" height="268"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494070" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Inferred pathways of fatty acid synthesis and modification in oat seeds. Fatty acid abbreviations adhere to standard conventions, with chain length and degree of unsaturation (position[s] of double bond[s]) separated by a colon (for example, palmitic acid is denoted by 16:0). We detected two isomers of 18:1, 18:1(9) and another isomer, 18:1*, with unknown double-bond position (likely 18:1(11)). The 18:1(9) isomer was more abundant. Fatty acids up to 18 carbons in length are synthesized by a fatty acid synthase (FAS) complex. The nascent acyl chain is attached to acyl carrier protein (ACP) subunit of FAS and grows by two carbons per cycle through the action of distinct ketoacyl-ACP synthase (KAS) subunits of this complex (KASI and KASII are shown). Elongation is terminated either by a thioesterase that releases the fatty acid from ACP, or a double bond is introduced by an acyl-ACP desaturase (AAD) that typically acts with specificity for the ∆9 position and preference for C18 substrate. Thus, the initial fatty acid produced by FAS results from competition between one or more thioesterase and AAD isoforms. Subsequent elongation to ≥ 20C is catalyzed by the fatty acid elongase complex, in which the ketoacyl-CoA synthase (KCS) subunit determines chain length. Further desaturation is catalyzed by additional fatty acid desaturases (FAD2 and FAD3).</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Germplasm</h3><p id="p-9">We assembled a 1,012 line diversity panel for oat, consisting of 391 inbred lines from the AFRI Core collection (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Esvelt Klos <em>et al.</em> 2016</a>), 219 inbred lines from Iowa State University (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Newell <em>et al.</em> 2012</a>), and 402 lines from a selection experiment conducted at Iowa State University composed of 20 inbred parents and 382 Cycle 1 or Cycle 2 individuals (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Asoro <em>et al.</em> 2013</a>). In 2014, the diversity panel was grown in an augmented field design with one replicate in each of two locations, with distinct soil types, near Ithaca, NY in 2014. Seeds were harvested, dried, and mechanically dehulled (Codema LLC, Maple Grove, MN, USA), as previously described (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Montilla-Bascón <em>et al.</em> 2017</a>). After visual inspection, any residual hull material was manually removed with a razor blade. Using the CDmean method of <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rincent <em>et al.</em> (2012)</a>, which maximizes the genetic diversity relative to the larger source population, we selected a 500 line subset of the diversity panel for further analysis.</p></div><div id="sec-3" class="subsection"><h3>Genotyping</h3><p id="p-10">A single seed from each line was planted in a greenhouse in Ithaca, NY. After four weeks, leaf tissue was collected for genotyping. DNA extraction and library preparation for genotyping-by-sequencing (GBS) were performed at Kansas State University (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Poland and Rife 2012</a>). Briefly, we extracted genomic DNA with the CTAB method as previously described (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Esvelt Klos <em>et al.</em> 2016</a>) and constructed 96-plex GBS libraries using the restriction enzymes <em>Pst</em>I and <em>Msp</em>I (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Poland <em>et al.</em> 2012</a>). Each library was then sequenced on a single Illumina HiSeq 2000 lane by the Cornell University Biotechnology Resource Center Genomics Facility.</p><p id="p-11">Genotypes were called using the genotyping pipeline Haplotag, as this pipeline does not require a reference genome sequence (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Tinker <em>et al.</em> 2016</a>). Subsequently, single-nucleotide polymorphisms (SNPs) were mapped to a published consensus genetic linkage map generated from 15 bi-parental mapping populations (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chaffin <em>et al.</em> 2016</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bekele <em>et al.</em> 2018</a>). We filtered SNPs using the following criteria: 1) biallelic; 2) minor allele frequency (MAF) &gt; 2%; 3) site missingness &lt; 60%; and 4) site heterozygosity &lt; 10%. After initial SNP filtering, eight lines with more than 80% missing data and/or 10% heterozygous genotype calls were excluded from further analysis. To remove SNPs providing redundant information, we calculated Pearson’s correlation coefficient (<em>r</em>) for all pairwise combinations of SNPs, where missing data were coded as a fourth state. Only one SNP was retained in a group of SNPs with <em>r<sup>2</sup></em> = 1, leaving 29,320 SNPs. For the GWAS, we imputed missing data with the mean genotype.</p></div><div id="sec-4" class="subsection"><h3>Kinship and population structure</h3><p id="p-12">To minimize the influence of missing data on pairwise relationship estimates, we estimated the relatedness matrix utilizing a 15,228 SNP subset (of the 29K filtered SNP set) with low missing data (&lt;20%) with the <em>A.mat</em> function in the R package <em>rrBLUP</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Endelman 2011</a>). Prior to conducting principal component analysis (PCA) with EIGENSTRAT (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Price <em>et al.</em> 2006</a>), we also removed SNPs that were not anchored to the genetic linkage map, resulting in a set of 12,585 SNPs.</p></div><div id="sec-5" class="subsection"><h3>Fatty acid chemical analysis</h3><p id="p-13">Dehulled oat seeds were stored at -20° prior to gas chromatography-mass spectrometry (GC-MS), performed at the Proteomics and Metabolomics Facility at Colorado State University. Oat seeds (300 mg) were ground to a fine powder in a 5 mL polypropylene vial containing a tungsten bead. A 6:3:1 (by volume) solution of cold methyl tert-butyl ether, methanol, and water (3 mL) was added to the vial prior to one hour of shaking at 4°. Subsequently, 750 µL of water was added, the vial was vortexed again and centrifuged for 15 min at 4° to induce phase separation. The upper organic layer, containing non-polar lipids, was transferred to a new glass vial and stored at -80° until analysis.</p><p id="p-14">Thawed samples were centrifuged at 3,750 rpm at 4° for 10 min. The solvent was removed from 100 µL aliquots of each sample by nitrogen evaporation at room temperature. Once dried, 200 µL of toluene containing 1.25 mg/mL of internal standard (glyceryl triheptadecanoate), 200 µL of methanol, and 200 µL of 3N methanolic HCl, were added to the sample, and the mixture was incubated at 60° for 1 h. Then, 1 ml of hexane and 300 µL of water were added to the cooled sample. After brief vortexing, the sample was centrifuged at 2,000 rpm for 5 min at 4°.</p><p id="p-15">One microliter of the upper hexane layer containing the fatty acid methyl esters (FAMEs) was injected onto a TG-WaxMS column (30 m × 0.25 mm × 0.25 µm, Thermo) in a TRACE 1310 Gas Chromatograph (Thermo) coupled to a Thermo ISQ LT GC-MS. The injector temperature and split ratio were 260° and 30:1, respectively. The column was eluted with a constant flow of He carrier gas (1.2 mL min<sup>-1</sup>). The oven initial temperature was 200° and held for 1 min, then increased to 260° at 10° min<sup>-1</sup> and held for 3 min. Detection was completed under electron impact mode, with a scan range of 50-650 atomic mass units and scan rate of 5 scans s<sup>-1</sup>. Transfer line and source temperatures were 260° and 230°, respectively.</p><p id="p-16">Quality control samples, consisting of pooled experimental samples, were injected after every 20 samples. Batches of 100 samples were prepared and analyzed simultaneously. Ion source, inlet liner, and septa were either cleaned or replaced after analysis of 300 samples. Standard curves were established for 16:0, 18:0, 18:1(9), 18:2, and 18:3. Two standard FAME mix samples (Nucheck GLC-85 and Sigma 47080-U) were used to confirm the retention times. In the case of 18:1, two peaks were consistently observed. The major peak in all samples corresponded to 18:1(9) based on retention time and authentic standards. The minor peak we designate 18:1* since assignment of double bond position was not possible based on spectra or authentic standards (Figure S1), although 18:1* is most likely the methyl ester of <em>cis</em>-vaccenic acid, 18:1(11) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cahoon <em>et al.</em> 1998</a>). Data processing was completed with Chromeleon 7 software (Thermo).</p></div><div id="sec-6" class="subsection"><h3>Phenotypic data analysis</h3><p id="p-17">Ten fatty acids were measured as their respective FAMEs from the 500-line diversity panel (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> for fatty acid structures and abbreviations). We observed 19:0 and 15:0 in several samples, but as values of these compounds were largely below the limit of detection (LOD), we excluded 19:0 and 15:0 from analysis. Samples that had levels below the LOD (zero values) for 14:0 (n = 1), 16:1 (n = 2), 20:0 (n = 50), and 20:1 (n = 1) were replaced with a uniform random variable between zero and the LOD, where the minimum non-zero value was used a proxy for LOD (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lipka <em>et al.</em> 2013</a>). The ten individual FAME traits and total FAME were inspected for outliers in ASReml-R 3.0 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Butler <em>et al.</em> 2009</a>) by examination of the Studentized deleted residuals (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kutner <em>et al.</em> 2004</a>) from linear mixed models fitted with environment, block, line, and GC-MS batch as random effects and heading date as a fixed effect. To assess the effect of the two environments on FAME concentrations, we calculated pairwise Spearman’s rank correlations (ρ) and performed hierarchical clustering on all trait-environment combinations.</p><p id="p-18">Given that fatty acids comprise a chemical family, we also considered the relative contribution of each member to total concentration. Such a compositional analysis imposes a unit-sum constraint, <em>i.e.</em>, all proportions must sum to one, which, in turn, induces spurious negative correlations between variables (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aitchison 1983</a>). Therefore, to relieve the constraints of this sample space (the unit simplex), we employed Aitchison’s log-contrast transformation. Specifically, we divided each FAME by an arbitrarily selected compound (here, 18:1*) prior to calculating pairwise correlations (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wei and Simko 2017</a>).</p><p id="p-19">As the FAME concentrations were approximately normally distributed, we used the raw phenotype values to estimate a best linear unbiased predictor (BLUP) for each line in ASReml-R. Our BLUP model (<a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>) incorporated environmental and technical covariates, as well as variation in heading date (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gilmour <em>et al.</em> 2008</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2963/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2963/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>in which Y<em><sub>ijkl</sub></em> is a plot-level average; <em>µ</em> is the grand mean; genotype<em><sub>i</sub></em> is the effect of the <em>i</em>th genotype; env<em><sub>j</sub></em> is the effect of the <em>j</em>th environment; (genotype x env)<em><sub>ij</sub></em> is the effect of the <em>i</em>th genotype in the <em>j</em>th environment; heading(env)<em><sub>ij</sub></em> is the effect of days to heading of the <em>i</em>th genotype within the <em>j</em>th environment; block(env)<em><sub>jk</sub></em> is the effect of the <em>k</em>th block within the <em>j</em>th environment; batch<em><sub>l</sub></em> is the effect of the <em>l</em>th GC-MS batch; and, e<em><sub>ijkl</sub></em> is the random error term, assumed to follow a normal distribution with mean zero and variance <em>σ<sup>2</sup></em>. Only heading(env)<em><sub>ij</sub></em> was fitted as a (continuous) fixed effect; all other terms were fitted as random effects.</p><p id="p-20">Variance component estimates from the fitted model were used to calculate line-mean heritability for each trait (<em>h<sub>l</sub><sup>2</sup></em> as per <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Holland <em>et al.</em> (2002)</a> and <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hung <em>et al.</em> (2012)</a>). Standard errors of the line-mean heritabilities were estimated using the delta method (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Holland <em>et al.</em> 2002</a>).</p><p id="p-21">To identify axes of variation in the FAME data set, we performed PCA on the centered and scaled (to unit-variance) BLUPs using the R package <em>pcaMethods</em> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Stacklies <em>et al.</em> 2007</a>). We extracted the principal component (PC) loadings to determine the contribution of each compound to each PC.</p></div><div id="sec-7" class="subsection"><h3>Network analysis with a Gaussian graphical model (GGM)</h3><p id="p-22">To assess the relationships between BLUPs for the 11 FAME measures (ten individual, plus the total FAMEs), hereafter referred to as FAME BLUPs, we first estimated pairwise Pearson’s correlation coefficients (<em>r</em>) between each trait with the R package <em>Hmisc</em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Harrell 2018</a>). We next constructed a Gaussian graphical model (GGM). As GGMs reflect the conditional dependencies between variables, they are more likely to capture causality and precursor/product relationships in metabolic networks relative to standard correlation analyses (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Krumsiek <em>et al.</em> 2011</a>). We employed the R package <em>ppcor</em> for calculation of partial correlations and significance testing (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kim 2015</a>). An edge was drawn between two metabolites if the partial correlation was significant after applying a Bonferroni correction ((α = 0.05 / ((<em>m</em>(<em>m</em>-1)) / 2)), where <em>m</em> is equal to the number of metabolites (<em>m</em> = 10). We visualized both the <em>r</em> and <em>pr</em> networks with the R packages <em>network</em> and <em>ggplot2</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Butts 2008</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wickham 2016</a>).</p></div><div id="sec-8" class="subsection"><h3>Genome-wide association studies (GWAS)</h3><p id="p-23">We leveraged the correlations between FAME traits in a multivariate GWAS with 492 lines and 29,320 SNPs. Specifically, we fitted a multivariate linear mixed model, accounting for population structure and relatedness, with all ten FAME concentrations in GEMMA (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zhou and Stephens 2012</a>). To determine the optimal number of genetic principal components (PCs) based on the genotype matrix to include in the GWAS model, we used the Bayesian information criterion (BIC), comparing models with zero to five PCs (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Schwarz 1978</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lipka <em>et al.</em> 2012</a>). To minimize deviation from multivariate normality, we quantile transformed each phenotype to a standard normal distribution (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Stephens 2013</a>). Specifically, the raw BLUPs were rank normalized using the <em>qnorm</em> function of R and scaled to unit variance. To inform our multivariate analysis, we also conducted univariate association analyses on each of the non-transformed FAME BLUPs and the PCs derived from singular value decomposition of the FAME BLUPs data matrix (PC-GWAS, see <em>Phenotypic data analysis</em>). In each association test, we employed the univariate analog of our multivariate GWAS model in GEMMA. To account for multiple testing, we computed a Bonferroni correction (α = 0.05 / 29,320 = 1.7 × 10<sup>−6</sup>). Alternatively, we controlled the false-discovery rate (FDR) at 5% and 10% (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Benjamini and Hochberg 1995</a>) using the R package <em>qvalue</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Storey <em>et al.</em> 2019</a>).</p><p id="p-24">Pairwise LD between SNPs was estimated using squared allele frequency correlations (<em>r</em><sup>2</sup>), excluding missing and double heterozygous genotypes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pritchard and Przeworski 2001</a>). We defined the <em>significant SNP set</em> as the union of SNPs identified as significantly associated with a phenotype at a 10% FDR in any of the univariate or multivariate analyses. To remove likely false positives and focus on GWAS signals with the most statistical support, we further analyzed only those significant SNPs in LD (<em>r</em><sup>2</sup> &gt; 0.5) with at least two other significant SNPs.</p><p id="p-25">The amount of phenotypic variation explained by each SNP in the univariate analyses was approximated using a likelihood-ratio-based <em>R</em><sup>2</sup> statistic (<em>R</em><sup>2</sup><sub>LR</sub>), as defined by <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sun <em>et al.</em> (2010)</a> (Table S2). The <em>R</em><sup>2</sup><sub>LR</sub> value of models with or without a significant SNP was calculated using the maximum log-likelihood of the model of interest fitted in GEMMA and the maximum log-likelihood of an intercept-only model fitted with the <em>lm</em> function in R. For the SNPs detected to be significant in the multi-GWAS, we approximated the relative contribution of each trait to the multivariate signal by calculating <em>R</em><sup>2</sup><sub>LR</sub> for each SNP in each of the ten respective univariate models developed for the quantile-transformed FAME BLUPs (Table S3).</p><p id="p-26">To qualitatively summarize the effects of SNPs identified as statistically significant in the multi-GWAS on individual FAME levels, we compared the BLUP phenotypic means of the distinct homozygous classes (major and minor) and conducted a pairwise <em>t</em>-test (<em>pairwise.t.test</em> function in R) for each SNP. Phenotypes were mean centered to zero and standardized to unit variance prior to testing. We excluded heterozygous and imputed genotypes.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-27">All genotype and phenotype data are freely available from the Triticeae Toolbox/Oat database (<a href="https://triticeaetoolbox.org/oat/">https://triticeaetoolbox.org/oat/</a>). Raw GBS sequence reads are available as NCBI BioProject PRJNA555603. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7889747">https://doi.org/10.25387/g3.7889747</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Genotyping and population structure</h3><p id="p-28">We assembled a panel of 1,012 oat lines from three existing germplasm collections. This panel consists of inbred lines from around the world, some of which were selected based on variability of beta-glucan, including selections from a beta-glucan improvement program (see <em>Materials and Methods</em>). We genotyped this panel with genotyping-by-sequencing (GBS), a reduced-representation sequencing technique (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Poland and Rife 2012</a>). After genotype calling with the non-reference pipeline Haplotag (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Tinker <em>et al.</em> 2016</a>), we identified an optimal and diverse subset of 500 lines using the CDmean method of <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Rincent <em>et al.</em> (2012)</a>. Within this subset, we applied standard quality control filters, resulting in a data set of 29,320 SNPs and 492 individuals in the diversity panel. Of these 29,320 SNPs, 19,558 (67%) were anchored to the consensus genetic map (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chaffin <em>et al.</em> 2016</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bekele <em>et al.</em> 2018</a>). Principal component analysis (PCA) on a reduced set of 12,585 SNPs (see <em>Materials and Methods</em>) revealed minimal structure with respect to line origin (Figure S2).</p></div><div id="sec-12" class="subsection"><h3>Variation in seed fatty acid content</h3><p id="p-29">Oats are especially high in healthful mono- and di-unsaturated fatty acids, relative to other cereals. To investigate variation in nutritional quality in oat, we measured the levels of ten FAMEs derived from non-polar seed lipids in the 492-line diversity panel grown in two environments (referred to as ENV1 and ENV2). In both environments, 18:2, 18:1(9), and 16:0 were the most abundant FAME species (Figure S3C), in accordance with previous reports (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Banaś <em>et al.</em> 2007</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Leonova <em>et al.</em> 2008</a>). In particular, 18:2 and 18:1(9) accounted for &gt;70% of the total FAME content in all lines (Figure S3F).</p><p id="p-30">FAME concentrations were consistently higher in ENV1 compared to ENV2 (pairwise <em>t</em>-tests detected significant differences for all traits except 18:3). To assess the impact of this consistent environmental difference on the statistical relationships between traits with respect to environment, we calculated Spearman’s rank correlations between each trait-environment combination. Hierarchical clustering revealed grouping with respect to environment and to a lesser extent compound (Figure S3B). For example, the three FAMEs in highest abundance, 18:2, 18:1(9), and 16:0, clustered together in each environment, along with 20:1. In addition, we observed clustering between 20:0 and 18:0, and 16:1 and 18:1*, respectively.</p><p id="p-31">Having observed that trait-environment clusters were predominately defined by environment (and secondarily by compound), we were interested in assessing the differential effect of environment on FAME composition, defined as the percent contribution of each FAME to total concentration. We observed significantly higher proportions of 14:0, 18:0, 18:2, and 20:0, and lower proportions of 18:1(9), 18:3, and 20:1 in ENV1 as compared to ENV2 (Figure S3F; <em>p</em>-value of pairwise <em>t</em>-test &lt; 0.05). The proportions of other compounds, and the sum of the saturated FAMEs, were comparable across environments.</p><p id="p-32">Assessing the correlation structure in this compositional sample space necessitated relieving the unit-sum constraint imposed by the fact that proportions must sum to one. After applying the log-contrast transformation (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aitchison 1983</a>) to FAME concentrations, <em>i.e.</em>, taking the logarithm of each FAME divided by arbitrarily selected FAME (18:1*), trait rather than environment defined clustering (Figure S3D-E). Yet, we still observed positive correlations between all trait pairs, excluding the pairwise comparisons that featured 16:1.</p><p id="p-33">Consistent effects of environment on concentrations across lines corresponded to high line-mean heritabilities for all FAMEs, ranging from 0.67 to 0.91 (Table S1). Thus, we conclude that genetic determinants have a greater impact than environmental factors on variation in FAME concentrations between lines.</p></div><div id="sec-13" class="subsection"><h3>Correlation networks</h3><p id="p-34">Using data from the two environments, we calculated BLUPs of each FAME for the 492 lines in the diversity panel (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Consistent with our analysis of the raw phenotypes, we observed only positive correlations between FAME BLUPs (Figure S3), translating to a dense, highly connected network (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in seed fatty acid concentration in a diverse oat panel. A. Box plots of the ten FAME best-linear unbiased predictor (BLUP) distributions measured in an oat diversity panel (n = 492). Compounds are divided into three groups based on mean concentration, each plotted with distinct y-axes. The color of the box denotes the number of double bonds in the corresponding FAME. B. Overlaid histograms of unsaturated, saturated, and total FAME BLUPs derived from the independent FAME measurements. C. The cumulative contribution of each FAME to total concentration plotted on a log-transformed scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-286196959" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variation in seed fatty acid concentration in a diverse oat panel. A. Box plots of the ten FAME best-linear unbiased predictor (BLUP) distributions measured in an oat diversity panel (&lt;em&gt;n&lt;/em&gt; = 492). Compounds are divided into three groups based on mean concentration, each plotted with distinct y-axes. The color of the box denotes the number of double bonds in the corresponding FAME. B. Overlaid histograms of unsaturated, saturated, and total FAME BLUPs derived from the independent FAME measurements. C. The cumulative contribution of each FAME to total concentration plotted on a log-transformed scale.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2963/F2.medium.gif" width="440" height="211"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/9/2963/F2.medium.gif" width="440" height="211"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494192" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Variation in seed fatty acid concentration in a diverse oat panel. A. Box plots of the ten FAME best-linear unbiased predictor (BLUP) distributions measured in an oat diversity panel (<em>n</em> = 492). Compounds are divided into three groups based on mean concentration, each plotted with distinct y-axes. The color of the box denotes the number of double bonds in the corresponding FAME. B. Overlaid histograms of unsaturated, saturated, and total FAME BLUPs derived from the independent FAME measurements. C. The cumulative contribution of each FAME to total concentration plotted on a log-transformed scale.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fatty acid methyl ester (FAME) correlation networks. A. A network constructed from Pearson&#x2019;s correlations (r) between FAMEs. B. An analogous network constructed from pairwise partial correlations (pr) with edges corresponding to biosynthetic steps annotated (See Figure 1 for abbreviations). In both A and B, an edge was drawn between compounds if the pairwise r or pr value was significant given a threshold of &#x3B1; = 0.05, after applying a Bonferroni correction for multiple-testing. Edge width is proportional to the magnitude of the correlation, and edge color indicates a positive or negative correlation. Node size corresponds to the mean concentration (mg g-1), with compounds grouped into three categories: less than 1, 10, and 25 mg g-1. Node color differentiates saturated from unsaturated FAMEs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-286196959" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fatty acid methyl ester (FAME) correlation networks. A. A network constructed from Pearson&#x2019;s correlations (&lt;em&gt;r&lt;/em&gt;) between FAMEs. B. An analogous network constructed from pairwise partial correlations (&lt;em&gt;pr&lt;/em&gt;) with edges corresponding to biosynthetic steps annotated (See Figure 1 for abbreviations). In both A and B, an edge was drawn between compounds if the pairwise &lt;em&gt;r&lt;/em&gt; or &lt;em&gt;pr&lt;/em&gt; value was significant given a threshold of &#x3B1; = 0.05, after applying a Bonferroni correction for multiple-testing. Edge width is proportional to the magnitude of the correlation, and edge color indicates a positive or negative correlation. Node size corresponds to the mean concentration (mg g&lt;sup&gt;-1&lt;/sup&gt;), with compounds grouped into three categories: less than 1, 10, and 25 mg g&lt;sup&gt;-1&lt;/sup&gt;. Node color differentiates saturated from unsaturated FAMEs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2963/F3.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/9/2963/F3.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494145" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Fatty acid methyl ester (FAME) correlation networks. A. A network constructed from Pearson’s correlations (<em>r</em>) between FAMEs. B. An analogous network constructed from pairwise partial correlations (<em>pr</em>) with edges corresponding to biosynthetic steps annotated (See <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> for abbreviations). In both A and B, an edge was drawn between compounds if the pairwise <em>r</em> or <em>pr</em> value was significant given a threshold of α = 0.05, after applying a Bonferroni correction for multiple-testing. Edge width is proportional to the magnitude of the correlation, and edge color indicates a positive or negative correlation. Node size corresponds to the mean concentration (mg g<sup>-1</sup>), with compounds grouped into three categories: less than 1, 10, and 25 mg g<sup>-1</sup>. Node color differentiates saturated from unsaturated FAMEs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">As FAMEs participate in shared biochemical pathways, we hypothesized that many of the observed correlations reflected indirect rather than direct interactions. Our rationale was that metabolic networks are generally sparse due to the limited reactivity of organic compounds and stepwise nature of metabolic pathways. Specifically, fatty acid biosynthetic pathways exhibit this property (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Li-Beisson <em>et al.</em> 2013</a>). Therefore, akin to <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Krumsiek <em>et al.</em> (2011)</a>, we constructed a Gaussian graphical model (GGM) to account for the conditional dependencies between FAME BLUPs. We estimated partial correlations (<em>pr</em>s), defined as the correlation between the residuals of two compounds after accounting for the other <em>n</em>-2 compounds. As hypothesized, the <em>pr</em> distribution was shifted toward zero relative to its <em>r</em> counterpart (Figure S4). This translated to a far sparser network, such that every node was no longer connected to nearly every other node by an edge, and to the introduction of negative correlations (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Notably, the large positive correlations between compounds with a difference of two carbons, but the same number of double bonds (<em>e.g.</em>, 18:0 and 20:0), persisted in the <em>pr</em> network. While the significant pairwise <em>pr</em>s were not restricted to only precursor/product relationships in fatty acid biosynthesis, some of the strongest correlations in this network are between such pairs (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>).</p></div><div id="sec-14" class="subsection"><h3>Decomposition of FAME variance</h3><p id="p-38">To identify the contribution of each fatty acid to the major axes of variation in the phenotypic data set, we performed PCA on the centered and scaled BLUPs for the ten FAMEs (Figure S5). All FAMEs exhibited negative loadings along the first PC (explaining 56.8% of the total variance), which tracked with variation in total fatty acids (adjusted <em>R</em><sup>2</sup> of total with PC 1 = 0.952; Figure S5). Thus, most of the variation in individual FAME concentrations can be explained by variation in fatty acid totals. The second PC (explaining 15.2% of variance) primarily distinguished saturated from unsaturated FAMEs, whereas the third and fourth PCs differentiated 14:0 and 18:3 from all other FAMEs, respectively. While 20:1 contributed disproportionately to both the fifth and sixth PCs, these two PCs were less obviously interpretable.</p></div><div id="sec-15" class="subsection"><h3>Genetic mapping for fatty acids</h3><p id="p-39">We employed several mapping approaches to capitalize on the correlations between traits. First, we performed a multivariate GWAS of all ten FAME concentrations simultaneously (multi-GWAS). Second, we conducted GWAS with each PC derived from decomposition of the phenotypic data matrix (of dimension <em>n</em> x <em>m</em> = 492 lines × 10 traits) as phenotypes (PC-GWAS). Third, we implemented univariate tests of association for each FAME separately, as estimated by their BLUPs. To account for population structure, we included the first three PCs of the non-imputed genotype matrix as covariates in all of our GWAS models. These PCs were selected based on application of the BIC model selection procedure in the multi-GWAS, and explained 6.7, 4.3, and 3.2% of the total variance, respectively.</p><p id="p-40">In the multi-GWAS, a total of 25 SNPs were significant after accounting for multiple testing with a 5% Bonferroni significance threshold (<em>p</em>-value ≤ 1.7 × 10<sup>−6</sup>). We also considered SNPs passing the two less conservative significance thresholds of genome-wide (FDR of 5% and 10%). At these thresholds, 93 and 148 SNPs were detected, respectively.</p><p id="p-41">After applying a 10% FDR threshold independently in each association test, and combining results across all ten PCs, 129 SNPs were significantly associated with FAME variation in the PC-GWAS (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4C</a> and Figure S6). Approximately half of these 129 SNPs (66 SNPs) were represented in the multivariate 10% FDR significant set (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The majority of SNPs uniquely identified in PC-GWAS were associated with PC10 and not anchored to a linkage group. This PC explained only 0.4% of total variance across the ten FAMEs, with 18:1(9) weighted most heavily along this PC (Figure S5).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multivariate genome-wide association study (multi-GWAS). A. Negative log10 p-values from a multi-GWAS of ten FAME BLUPs plotted against genetic position in the consensus linkage map (Chaffin et al. 2016; Bekele et al. 2018). Dotted lines denote the three significance thresholds considered, with a Bonferroni-corrected threshold of 5% in red, and 5 and 10% false-discovery rate (FDR) thresholds in light and dark gray, respectively. Markers with a p-value passing the Bonferroni threshold are shown in red. B. A quantile-quantile plot of the multi-GWAS p-values. C. Venn diagram comparing significantly associated SNPs (at a 10% FDR) from combined univariate analyses of ten individual and total FAME untransformed BLUPs (Univariate), combined univariate analyses of ten principal components (PCs) of the ten FAME BLUP data matrix, and multi-GWAS of the ten FAME quantile-transformed BLUPs (Multivariate)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-286196959" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Multivariate genome-wide association study (multi-GWAS). A. Negative log&lt;sub&gt;10&lt;/sub&gt; &lt;em&gt;p&lt;/em&gt;-values from a multi-GWAS of ten FAME BLUPs plotted against genetic position in the consensus linkage map (Chaffin &lt;em&gt;et al.&lt;/em&gt; 2016; Bekele &lt;em&gt;et al.&lt;/em&gt; 2018). Dotted lines denote the three significance thresholds considered, with a Bonferroni-corrected threshold of 5% in red, and 5 and 10% false-discovery rate (FDR) thresholds in light and dark gray, respectively. Markers with a &lt;em&gt;p&lt;/em&gt;-value passing the Bonferroni threshold are shown in red. B. A quantile-quantile plot of the multi-GWAS &lt;em&gt;p&lt;/em&gt;-values. C. Venn diagram comparing significantly associated SNPs (at a 10% FDR) from combined univariate analyses of ten individual and total FAME untransformed BLUPs (Univariate), combined univariate analyses of ten principal components (PCs) of the ten FAME BLUP data matrix, and multi-GWAS of the ten FAME quantile-transformed BLUPs (Multivariate).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2963/F4.medium.gif" width="440" height="373"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/9/2963/F4.medium.gif" width="440" height="373"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494157" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Multivariate genome-wide association study (multi-GWAS). A. Negative log<sub>10</sub> <em>p</em>-values from a multi-GWAS of ten FAME BLUPs plotted against genetic position in the consensus linkage map (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Chaffin <em>et al.</em> 2016</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bekele <em>et al.</em> 2018</a>). Dotted lines denote the three significance thresholds considered, with a Bonferroni-corrected threshold of 5% in red, and 5 and 10% false-discovery rate (FDR) thresholds in light and dark gray, respectively. Markers with a <em>p</em>-value passing the Bonferroni threshold are shown in red. B. A quantile-quantile plot of the multi-GWAS <em>p</em>-values. C. Venn diagram comparing significantly associated SNPs (at a 10% FDR) from combined univariate analyses of ten individual and total FAME untransformed BLUPs (Univariate), combined univariate analyses of ten principal components (PCs) of the ten FAME BLUP data matrix, and multi-GWAS of the ten FAME quantile-transformed BLUPs (Multivariate).</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">In the univariate GWAS of the ten FAMEs and total FAME concentration, we identified 73 significantly associated SNPs at a 10% FDR – the smallest number of significant associations among the three approaches (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a> and Figure S7). Furthermore, the majority of these SNPs were identified by either multi- or PC-GWAS (47 SNPs). The uniquely identified SNPs in the univariate GWAS were primarily associated with less abundant fatty acids, <em>e.g.</em>, 18:0, 20:0, and 20:1. To provide a more apt comparison between the univariate and multivariate analysis, which employed quantile-transformed traits, we also performed the univariate GWAS with quantile-transformed traits. This analysis identified 62 SNPs at a 10% FDR (Figure S8). All but two of these SNPs were also found in GWAS of the non-quantile transformed BLUPs, suggesting that deviations from normality in the untransformed BLUPs had minimal influence on the results (Figure S8; Table S2).</p></div><div id="sec-16" class="subsection"><h3>Integration of GWAS results</h3><p id="p-44">To focus on GWAS signals with the most statistical support, we only considered significantly associated SNPs with <em>r</em><sup>2</sup> &gt; 0.5 with at least two other significant SNPs (at a 10% FDR threshold) across all analyses. This high confidence SNP set consisted of a 152 SNP subset of the 237 significant SNPs. The majority of SNPs in the high confidence SNP set (101 SNPs) spanned eight of the 21 consensus linkage groups, while 51 SNPs were unmapped (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). In the absence of a reference genome sequence, we used pairwise LD between significantly associated SNPs to putatively define the boundaries of independent signals (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). We further refined these boundaries such that SNPs within a signal interval had concordant estimated effects on the FAME abundances. Here, we assumed that if SNPs were in LD with the same causal locus they should exhibit correlated effects on the ten FAME traits. To interpret SNP effects, we inspected the mean difference between homozygous genotypic classes excluding lines with missing data or heterozygous genotypes (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). While this simple summary does not account for population structure and unequal relatedness, it provides qualitative insight into the effects of SNPs on individual FAMEs, which are obscured by multivariate GWAS.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage disequilibrium between markers associated with fatty acid methyl ester (FAME) variation identified in multivariate, principal component (PC), and univariate analyses. A. Linkage disequilibrium (LD), defined as the squared allele frequency correlation coefficient (r2), between markers identified as significantly associated with phenotypic variation at a 10% false-discovery rate (FDR) threshold in any of the multivariate, PC, or marginal analyses. (Here, we show only those significant markers that were in LD with at least two other significantly associated markers (see Materials and Methods). Single-nucleotide polymorphisms (SNPs) are ordered by hierarchical clustering, with abbreviated SNP name on the y-axis colored by linkage group assignment (see B) in the consensus genetic map (Chaffin et al. 2016; Bekele et al. 2018). Gray boxes denote missing values (see Materials and Methods). B. An incidence matrix of results from the multivariate, PC, and univariate association analyses. The x-axis mirrors that of A, with each vertical tract corresponding to the same SNP pictured in A. A colored rectangle indicates that the SNP was identified as significant at an FDR threshold of 10%, with the PC and Univariate tracks representing the union of significant SNPs across all ten and 11 traits, respectively. The color of the rectangle corresponds to the linkage group. A black dot along the gray track indicate that the SNP has a minor allele frequency &lt; 5%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-286196959" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage disequilibrium between markers associated with fatty acid methyl ester (FAME) variation identified in multivariate, principal component (PC), and univariate analyses. A. Linkage disequilibrium (LD), defined as the squared allele frequency correlation coefficient (&lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;), between markers identified as significantly associated with phenotypic variation at a 10% false-discovery rate (FDR) threshold in any of the multivariate, PC, or marginal analyses. (Here, we show only those significant markers that were in LD with at least two other significantly associated markers (see &lt;em&gt;Materials and Methods&lt;/em&gt;). Single-nucleotide polymorphisms (SNPs) are ordered by hierarchical clustering, with abbreviated SNP name on the y-axis colored by linkage group assignment (see B) in the consensus genetic map (Chaffin &lt;em&gt;et al.&lt;/em&gt; 2016; Bekele &lt;em&gt;et al.&lt;/em&gt; 2018). Gray boxes denote missing values (see &lt;em&gt;Materials and Methods&lt;/em&gt;). B. An incidence matrix of results from the multivariate, PC, and univariate association analyses. The x-axis mirrors that of A, with each vertical tract corresponding to the same SNP pictured in A. A colored rectangle indicates that the SNP was identified as significant at an FDR threshold of 10%, with the PC and Univariate tracks representing the union of significant SNPs across all ten and 11 traits, respectively. The color of the rectangle corresponds to the linkage group. A black dot along the gray track indicate that the SNP has a minor allele frequency &lt; 5%.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2963/F5.medium.gif" width="382" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/9/2963/F5.medium.gif" width="382" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494186" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-45" class="first-child">Linkage disequilibrium between markers associated with fatty acid methyl ester (FAME) variation identified in multivariate, principal component (PC), and univariate analyses. A. Linkage disequilibrium (LD), defined as the squared allele frequency correlation coefficient (<em>r<sup>2</sup></em>), between markers identified as significantly associated with phenotypic variation at a 10% false-discovery rate (FDR) threshold in any of the multivariate, PC, or marginal analyses. (Here, we show only those significant markers that were in LD with at least two other significantly associated markers (see <em>Materials and Methods</em>). Single-nucleotide polymorphisms (SNPs) are ordered by hierarchical clustering, with abbreviated SNP name on the y-axis colored by linkage group assignment (see B) in the consensus genetic map (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Chaffin <em>et al.</em> 2016</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Bekele <em>et al.</em> 2018</a>). Gray boxes denote missing values (see <em>Materials and Methods</em>). B. An incidence matrix of results from the multivariate, PC, and univariate association analyses. The x-axis mirrors that of A, with each vertical tract corresponding to the same SNP pictured in A. A colored rectangle indicates that the SNP was identified as significant at an FDR threshold of 10%, with the PC and Univariate tracks representing the union of significant SNPs across all ten and 11 traits, respectively. The color of the rectangle corresponds to the linkage group. A black dot along the gray track indicate that the SNP has a minor allele frequency &lt; 5%.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of multivariate, principal component (PC), and univariate genome-wide association study (GWAS) results. A. An incidence matrix of results from the multivariate, PC, and univariate GWAS. SNPs are ordered by hierarchical clustering of pairwise linkage disequilibrium (LD) estimates, as in Figure 5. A colored rectangle indicates that a SNP was significant in the given test at a 10% false-discovery rate (FDR) threshold, with color corresponding to a linkage group in the consensus genetic map (Chaffin et al. 2016; Bekele et al. 2018). Presence of a black dot indicates that the SNP has a minor allele frequency &lt; 5%. Labeled (a-f) line segments span the SNPs in six LD clusters, defined by visualization of the pairwise LD matrix (see Figure 5). B. Mean phenotypic differences between distinct homozygote classes at a significantly associated SNP within each of the six LD clusters (a-f) defined in A. Genotype is plotted on the x-axis, with the mean number of phenotypic standard deviations away from the mean on the y-axis. Each centered and unit-variance scaled FAME BLUP (including total) is plotted, with warm colors for saturated and cool colors for unsaturated FAMEs. Total is indicated by a black line. The bold lines indicate that the pairwise t-test between genotype class means was significant at &#x3B1; = 0.05 after correcting for multiple testing with a Bonferroni correction. To simplify the plots, only traits with significantly different means between genotypes are labeled. As lines with missing genotypes are excluded from these calculations, we present the genotype counts in the bottom right or left-hand corner of each plot." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-286196959" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of multivariate, principal component (PC), and univariate genome-wide association study (GWAS) results. A. An incidence matrix of results from the multivariate, PC, and univariate GWAS. SNPs are ordered by hierarchical clustering of pairwise linkage disequilibrium (LD) estimates, as in Figure 5. A colored rectangle indicates that a SNP was significant in the given test at a 10% false-discovery rate (FDR) threshold, with color corresponding to a linkage group in the consensus genetic map (Chaffin &lt;em&gt;et al.&lt;/em&gt; 2016; Bekele &lt;em&gt;et al.&lt;/em&gt; 2018). Presence of a black dot indicates that the SNP has a minor allele frequency &lt; 5%. Labeled (&lt;em&gt;a-f&lt;/em&gt;) line segments span the SNPs in six LD clusters, defined by visualization of the pairwise LD matrix (see Figure 5). B. Mean phenotypic differences between distinct homozygote classes at a significantly associated SNP within each of the six LD clusters (&lt;em&gt;a-f&lt;/em&gt;) defined in A. Genotype is plotted on the x-axis, with the mean number of phenotypic standard deviations away from the mean on the y-axis. Each centered and unit-variance scaled FAME BLUP (including total) is plotted, with warm colors for saturated and cool colors for unsaturated FAMEs. Total is indicated by a black line. The bold lines indicate that the pairwise &lt;em&gt;t&lt;/em&gt;-test between genotype class means was significant at &#x3B1; = 0.05 after correcting for multiple testing with a Bonferroni correction. To simplify the plots, only traits with significantly different means between genotypes are labeled. As lines with missing genotypes are excluded from these calculations, we present the genotype counts in the bottom right or left-hand corner of each plot.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2963/F6.medium.gif" width="403" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/9/2963/F6.medium.gif" width="403" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2963/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494100" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-46" class="first-child">Comparison of multivariate, principal component (PC), and univariate genome-wide association study (GWAS) results. A. An incidence matrix of results from the multivariate, PC, and univariate GWAS. SNPs are ordered by hierarchical clustering of pairwise linkage disequilibrium (LD) estimates, as in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>. A colored rectangle indicates that a SNP was significant in the given test at a 10% false-discovery rate (FDR) threshold, with color corresponding to a linkage group in the consensus genetic map (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Chaffin <em>et al.</em> 2016</a>; <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Bekele <em>et al.</em> 2018</a>). Presence of a black dot indicates that the SNP has a minor allele frequency &lt; 5%. Labeled (<em>a-f</em>) line segments span the SNPs in six LD clusters, defined by visualization of the pairwise LD matrix (see <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). B. Mean phenotypic differences between distinct homozygote classes at a significantly associated SNP within each of the six LD clusters (<em>a-f</em>) defined in A. Genotype is plotted on the x-axis, with the mean number of phenotypic standard deviations away from the mean on the y-axis. Each centered and unit-variance scaled FAME BLUP (including total) is plotted, with warm colors for saturated and cool colors for unsaturated FAMEs. Total is indicated by a black line. The bold lines indicate that the pairwise <em>t</em>-test between genotype class means was significant at α = 0.05 after correcting for multiple testing with a Bonferroni correction. To simplify the plots, only traits with significantly different means between genotypes are labeled. As lines with missing genotypes are excluded from these calculations, we present the genotype counts in the bottom right or left-hand corner of each plot.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">Using these criteria, we identified six main clusters, consisting of 124 SNPs in the high confidence SNP set (<em>a-f</em> in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Five of the clusters localized to three linkage groups: Mrg02, Mrg05, and Mrg01, while one cluster was not anchored to the linkage map (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). The remaining 28 markers formed smaller clusters that were either unanchored or primarily supported by SNPs with MAF &lt; 5% (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figures 5</a>, <a id="xref-fig-6-4" class="xref-fig" href="#F6">6</a>, and S9). One notable small cluster is anchored at 36.0 cM on Mrg12 by a SNP with a very small <em>p</em>-value (1.9 × 10<sup>−11</sup>). Although the two other SNPs in this cluster were unanchored, the overall peak SNP in the multi-GWAS (<em>P</em> = 1.1 × 10<sup>−33</sup>) was mapped to a nearby position (40.9 cM) on this linkage group but excluded from the high confidence SNP set by our <em>r</em><sup>2</sup> clustering criteria. This SNP and the LD cluster SNPs were associated with 14:0 and PC3 (which exhibits a large 14:0 loading, Figure S5B), suggesting at least one gene in this Mrg12 region underlies variation in seed 14:0 content.</p><p id="p-48">We observed two primary clusters in Mrg02, spanning 37.5 to 107.5 cM (cluster <em>a</em>) and 67.4 to 85.3 cM (cluster <em>b</em>), respectively. These large intervals are consistent with previous reported of long-range LD in this linkage group (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Chaffin <em>et al.</em> 2016</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Esvelt Klos <em>et al.</em> 2016</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Sunstrum <em>et al.</em> 2019</a>). We suspect that long-range LD, coupled with a causal locus in Mrg02, inflated test statistics for SNPs in Mrg02 resulting in the over representation of this linkage group in the high confidence SNP set. The peak SNPs from the two clusters (<em>a</em> and <em>b</em>) had markedly different minor allele frequencies (0.33 <em>vs.</em> 0.09; <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6B</a>, <em>a</em> and <em>b</em>). As such, the <em>r</em><sup>2</sup> for these two SNPs is constrained to a maximum value of <em>r</em><sup>2</sup> = 0.19, reflecting the sensitivity of <em>r</em><sup>2</sup> to differences in allele frequency (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">VanLiere and Rosenberg 2008</a>). Therefore, while weak LD between these SNPs (<em>r</em><sup>2</sup> = 0.14) resulted in assignment to different clusters by our criteria (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>), the two clusters are likely tagging the same locus, or different alleles at the same locus. Note that without a physical map we cannot identify signal intervals with certainty. Furthermore, SNPs within both clusters exhibit similar patterns of phenotypic associations. In both clusters, SNPs were associated with increases in low concentration 20:1 and the biosynthetically-related, and abundant 18:1(9) (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6B</a>, <em>a</em> and <em>b</em>). SNPs in cluster <em>a</em> were also associated with higher levels of 20:0, 18:2, 18:3, and 16:0, while SNPs in cluster <em>b</em> also exhibited association with higher levels of 18:1*. Notably, in the univariate GWAS, the SNPs within these two clusters were only associated with 20:1.</p><p id="p-49">Two clusters were also observed in Mrg05, mapping to non-overlapping intervals between 124.1 and 135.5 cM (cluster <em>c</em>) and 114.7 to 122.7 cM (cluster <em>e</em>). The minor alleles in cluster <em>c</em> were associated with significant decreases in most FAMEs as well as total concentration. All three approaches (univariate, PCs, and multivariate) detected significant SNPs in the interval associated with cluster <em>c</em>. In contrast, cluster <em>e</em> was only detected by the multi-GWAS. In this cluster, the minor allele was associated with higher total FAMEs and concentrations of the most abundant FAMEs, as well as, decreases in 18:3 and 16:1. The correlated estimated effects of SNPs in cluster <em>e</em> on 16:1 and 18:3 were not expected <em>a priori</em> based on known biochemical pathways (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>) or the observed correlation networks (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). To further probe the major drivers of the multi-GWAS signals, for each significant SNP in the multi-GWAS we calculated <em>R</em><sup>2</sup><sub>LR</sub> in the univariate models for each trait. The difference between <em>R</em><sup>2</sup><sub>LR</sub> with or without a given SNP included in the model, ∆<em>R</em><sup>2</sup><sub>LR</sub>, provides an approximation of phenotypic variance explained by that SNP. Thus, we used univariate ∆<em>R</em><sup>2</sup><sub>LR</sub> of each component trait as a means of qualitatively dissecting the multi-GWAS signal (Table S3). By this analysis, detection of cluster <em>e</em> SNPs associate with 18:3 variation, despite the fact that we failed to detect significant associations in the univariate GWAS. Thus, it appears that unanticipated correlations between 18:3 and other fatty acids, such as 16:1, underlie the enhanced sensitivity of the multi-GWAS model.</p><p id="p-50">The cluster identified on Mrg11 spanned 3.7 to 8.8 cM. This cluster (<em>f</em>) contained two SNPs identified in the univariate analyses of total and 18:1(9) FAMEs, and four SNPs uniquely identified in the multi-GWAS. The main effects associated with the peak SNP in this cluster were increased 16:1 and 18:1* and decreased 18:0 and 18:3 (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6B</a>). However, other SNPs in this cluster were associated with decreases in the majority of FAMEs, including total concentration (Figure S9). Thus, as with cluster <em>a</em> and <em>b</em>, the SNPs of cluster <em>f</em> may be tagging multiple alleles or loci and our interpretation is limited by the present lack of a genome sequence in hexaploid oat.</p><p id="p-51">The final cluster (<em>d</em>) consisted of 29 SNPs of low minor allele frequency (&lt; 0.05). The only mapped SNP in cluster <em>d</em> was located at 80.3 cM on Mrg03 of the consensus map and a significant association of this SNP was only detected in GWAS of the last PC. SNPs in this cluster were predominantly associated with higher levels of saturated fatty acids (16:0, 18:0, and 20:0).</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-52">We report the first GWAS of seed fatty acid composition in oat. Our results reveal several large effect loci underlying this important seed quality trait. At least four loci on four linkage groups contribute to variation in the content and composition of seed fatty acids in oat. Ongoing efforts to sequence the ∼12.5 Gb hexaploid oat genome will likely enable higher resolution mapping of these GWAS hits. On the other hand, inability to define distinct GWAS support intervals may, in part, reflect the complex genetics of oat germplasm. Specifically, translocations and other chromosomal rearrangements are common in oat germplasm (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jellen <em>et al.</em> 1994</a>), and these are thought to underlie distortions of LD within the consensus linkage groups as observed here and in other studies (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Chaffin <em>et al.</em> 2016</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Esvelt Klos <em>et al.</em> 2016</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Sunstrum <em>et al.</em> 2019</a>).</p><p id="p-53">Previous studies used linkage analysis in biparental populations to map QTL for fatty acid composition or oil amount in oat seeds. <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kianian <em>et al.</em> (1999)</a> mapped QTL for oil content in two RIL populations, “Kanota” x “Ogle” (KO) and “Kanota” x “Marion” (KM). In both the KO and KM populations, alleles of a gene encoding Acetyl-CoA Carboxylase, <em>ACCaseA</em>, were found to associate with a QTL accounting for up to 48% of phenotypic variance in these populations (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kianian <em>et al.</em> 1999</a>). This gene is likely to underlie seed oil content because its respective enzyme catalyzes the synthesis of Malonyl-CoA, the major substrate of fatty acid synthase. Colocalizing oil content QTL were later discovered in additional populations: “Ogle” x “MAM17-5” (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zhu <em>et al.</em> 2004</a>), “Aslak” x “Matilda” (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Tanhuanpää <em>et al.</em> 2012</a>), and “Dal” x “Exeter” (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hizbai <em>et al.</em> 2012</a>). <em>ACCaseA</em> is tightly linked with the RFLP marker <em>cdo665B</em>, which is placed at a position of 135.5 cM on Mrg05 of the consensus map of <a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Chaffin <em>et al.</em> (2016)</a>. Thus, these QTL, and their putative causal locus, <em>ACCaseA</em>, likely correspond to cluster c (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6</a>) in our current work.</p><p id="p-54">Clusters <em>a</em> and <em>b</em>, mapping to overlapping intervals on Mrg02 (<a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6</a>), may correspond to two oil content QTL discovered in the “Dal” x “Exeter” population, <em>oPt-11790</em> and <em>oPt-16384</em> (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Hizbai <em>et al.</em> 2012</a>). Another QTL identified in Hizbai <em>et al.</em> 2012, <em>oPT-17489</em>, colocalizes with cluster <em>f</em> on Mrg11 in our work (<a id="xref-fig-6-10" class="xref-fig" href="#F6">Figure 6</a>). Mechanistically, the effects associated with cluster <em>b</em> and <em>f</em> may be attributed to changes in acyl-ACP desaturase (AAD) activity. In the case of cluster <em>b</em>, an increase in AAD activity could lead to increases in both 18:1(9) and 18:1* as well the elongation of 18:1(9) to 20:1. The correlated change in 16:1 and 18:1* (assumed to be 18:1(11)), on the other hand, would be consistent with an altered substrate specificity for AAD. In plants, AAD typically has a strong preference for 18:0-ACP as its substrate, and the conversion of 16:0-ACP to 16:1-ACP is typically a minor side reaction (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cahoon <em>et al.</em> 1997</a>). In some cases, such as the seeds of cat’s claw (<em>Doxantha unguis-cati</em> L.), AAD isoforms with preference for 16:0-ACP as a substrate can lead to substantial production of 16:1(9) and its elongation product, 18:1(11) (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cahoon <em>et al.</em> 1998</a>). While these interpretations are speculative, they illustrate how the increased sensitivity of multivariate GWAS models can link relatively subtle perturbations of metabolic networks to their genetic and biochemical bases.</p><p id="p-55">One objective of this study was to identify loci associated with variation in 18:3 content, as these omega-3 fatty acids are a large determinant of oil quality. Increased consumption of omega-3 fatty acids is associated with improved cardiovascular health, although there is uncertainty regarding the benefits of consuming omega-3 fatty acids relative to omega-6 fatty acids, such as 18:2 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ludwig <em>et al.</em> 2018</a>). In oat seeds, the very low abundance of 18:3 relative to 18:2 in germplasm (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>) renders enhanced 18:3 content an unlikely breeding target for improving the healthfulness of oats. On the other hand, 18:3 is exceptionally prone to oxidation, resulting in rancidity. Therefore, decreased 18:3 content may improve postharvest stability (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zhou <em>et al.</em> 1999</a>). In univariate GWAS, we were unable to detect loci associated with 18:3 variation (<a id="xref-fig-6-11" class="xref-fig" href="#F6">Figure 6A</a>, Table S2). However, multi-GWAS that leveraged unexpected correlations between 18:3 and other fatty acids revealed a locus in Mrg05 (cluster <em>e</em>, <a id="xref-fig-6-12" class="xref-fig" href="#F6">Figure 6</a>) associated with substantial differences in 18:3 content relative to the more abundant fatty acids (<a id="xref-fig-6-13" class="xref-fig" href="#F6">Figure 6B</a>). Markers in cluster <em>e</em> may predict variation in 18:3 content associated with postharvest stability in oat breeding programs.</p><p id="p-56">For SNPs identified by GWAS to be useful for MAS, it is important that they exhibit relatively large effects on the trait(s) of interest. Unfortunately, the increased sensitivity of multivariate GWAS is accompanied by difficulty in accurately assigning effect sizes of a given SNP on each trait. While we have attempted to deconvolute the drivers of our multivariate GWAS signal using ∆<em>R</em><sup>2</sup><sub>LR</sub> statistics of univariate GWAS models (Supplemental Table 3), this still relies on quantile-normalized traits that obscure a SNP’s true effect size. Our best approximation of a given SNP’s contribution to each trait comes from ∆<em>R</em><sup>2</sup><sub>LR</sub> statistics for SNPs that were detected in univariate GWAS of non-transformed individual traits (Supplemental Table 2). By this measure, significantly associated SNPs explained approximately 2–16% of phenotypic variance within each of the 11 traits in our population. For total fatty acid content, significant SNPs explained approximately 3–7% of the phenotypic variance. The peak cluster <em>c</em> SNP, for example, has a ∆<em>R</em><sup>2</sup><sub>LR</sub> of 6.6% for GWAS of total fatty acid content in our population, and is likely a useful GBS marker for tagging <em>ACCaseA</em> alleles. We have also identified a new marker for total fatty acid content, avgbs2_90110.1.11, that has a similarly large effect size but appears to not be linked to <em>ACCaseA</em> (∆<em>R</em><sup>2</sup><sub>LR</sub> = 7.4%; Supplemental Table 2). This marker is presently unmapped and the allele that confers high fatty acid content has a frequency of just 0.025 in our population, suggesting that it marks a locus for oil content that is currently underutilized in oat breeding programs.</p><p id="p-57">In conclusion, we have demonstrated the utility of multivariate GWAS methods for identifying genetic associations with metabolites that share a biosynthetic pathway. By analyzing all ten fatty acids simultaneously with multi-trait GWAS methods, we were able to detect SNPs that were not detected by traditional, univariate GWAS. The increased sensitivity of multivariate GWAS may be particularly advantageous for polyploid species like oat, where the effect of a single locus may be buffered by the activity of its homeologs (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Santantonio <em>et al.</em> 2019</a>). On the other hand, the poorly characterized structural variation within oat haplotypes, in combination with long-range LD, currently presents a significant challenge for resolving GWAS signals to individual genes. The present study provides a foundation for future investigations of the genetic basis of fatty acid variation in oat. With anticipated improvements in genomic resources and the implementation of LD-independent approaches, such as transcriptome-wide association studies (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gusev <em>et al.</em> 2016</a>), we expect further resolution of the genetic basis of fatty acid composition.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">The authors wish to thank the Proteomics and Metabolomics Facility at Colorado State University for FAME analysis. This research was supported by PepsiCo Global R&amp;D, the USDA–ARS, USDA-NIFA-AFRI 2011-68002-30029 (Triticeae-CAP), USDA-NIFA-AFRI 2017-67007-25939 (Wheat-CAP), USDA-NIFA-AFRI 2017-67007-26502, and Hatch Project 149-447. Disclaimer: The views expressed in this article are those of the authors and do not necessarily reflect the position or policy of PepsiCo, Inc. Mention of a trademark or proprietary product does not constitute a guarantee or warranty of the product by the USDA and does not imply its approval to the exclusion of other products that may also be suitable. The USDA is an equal opportunity provider and employer.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> Present address: Committee on Genetics, Genomics, &amp; Systems Biology, University of Chicago, Chicago IL, 60637</p></li><li class="fn" id="fn-2"><p id="p-59">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7889747">https://doi.org/10.25387/g3.7889747</a>.</p></li><li class="fn" id="fn-3"><p id="p-60"><em>Communicating editor: A. Doust</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-31"><span class="received-label">Received </span>March 31, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-08"><span class="accepted-label">Accepted </span>July 8, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Carlson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.1" data-doi="10.1093/biomet/70.1.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aitchison</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Principal component analysis of compositional data.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/biomet/70.1.57</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DAITCHISON%26rft.auinit1%253DJ.%26rft.volume%253D70%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D65%26rft.atitle%253DPrincipal%2Bcomponent%2Banalysis%2Bof%2Bcompositional%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F70.1.57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/70.1.57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983QH66900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.2" data-doi="10.1016/j.ajhg.2014.03.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aschard</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Vilhjálmsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Greliche</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.-E.</span> <span class="cit-name-surname">Morange</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.-A.</span> <span class="cit-name-surname">Trégouët</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Maximizing the Power of Principal-Component Analysis of Correlated Phenotypes in Genome-wide Association Studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">662</span>–<span class="cit-lpage">676</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2014.03.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D94%26rft.spage%253D662%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.03.016%26rft_id%253Dinfo%253Apmid%252F24746957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.03.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24746957&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.3" data-doi="10.2135/cropsci2012.09.0526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>,  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Newell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. D.</span> <span class="cit-name-surname">Beavis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Scott</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Tinker</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic, Marker-Assisted, and Pedigree-BLUP Selection Methods for β-Glucan Concentration in Elite Oat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1894</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2012.09.0526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D1894%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.09.0526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.09.0526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.4" data-doi="10.1093/jxb/erm125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banaś</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Debski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Banaś</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. K.</span> <span class="cit-name-surname">Heneen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Dahlqvist</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Lipids in grain tissues of oat (Avena sativa): differences in content, time of deposition, and fatty acid composition.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">2463</span>–<span class="cit-lpage">2470</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erm125</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferm125%26rft_id%253Dinfo%253Apmid%252F17586606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erm125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586606&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250644400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bekele</span>,  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. P.</span> <span class="cit-name-surname">Wight</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Howarth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Tinker</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Haplotype-based genotyping-by-sequencing in oat genome research.</span> <abbr class="cit-jnl-abbrev">Plant Biotech. J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1452</span>–<span class="cit-lpage">1463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotech.%2BJ.%26rft.volume%253D16%26rft.spage%253D1452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hochberg</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J. R. Stat.</span> <abbr class="cit-jnl-abbrev">Soc. Ser. B Methodol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoc.%2BSer.%2BB%2BMethodol.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Cullis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gilmour</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gogel</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>ASReml-R reference manual</em>. The State of Queensland, Department of Primary Industries and Fisheries, Brisbane.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.8" data-doi="10.1007/s00394-008-0698-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butt</span>,  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tahir-Nadeem</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. K. I.</span> <span class="cit-name-surname">Khan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Shabir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Butt</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Oat: unique among the cereals.</span> <abbr class="cit-jnl-abbrev">Eur. J. Nutr.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00394-008-0698-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bnutrition%26rft.stitle%253DEur%2BJ%2BNutr%26rft.aulast%253DButt%26rft.auinit1%253DM.%2BS.%26rft.volume%253D47%26rft.issue%253D2%26rft.spage%253D68%26rft.epage%253D79%26rft.atitle%253DOat%253A%2Bunique%2Bamong%2Bthe%2Bcereals.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00394-008-0698-7%26rft_id%253Dinfo%253Apmid%252F18301937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00394-008-0698-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18301937&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254259000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butts</span>,  <span class="cit-name-given-names">C. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">network: a Package for Managing Relational Data in R.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DGoodreau%26rft.auinit1%253DS.%2BM.%26rft.volume%253D24%26rft.issue%253D9%26rft.spage%253D1%26rft.epage%253D27%26rft.atitle%253DA%2Bstatnet%2BTutorial.%26rft_id%253Dinfo%253Apmid%252F18612375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18612375&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.10" data-doi="10.1073/pnas.94.10.4872"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cahoon</span>,  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Lindqvist</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Schneider</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shanklin</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Redesign of soluble fatty acid desaturases from plants for altered substrate specificity and double bond position.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">4872</span>–<span class="cit-lpage">4877</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.94.10.4872</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCahoon%26rft.auinit1%253DE.%2BB.%26rft.volume%253D94%26rft.issue%253D10%26rft.spage%253D4872%26rft.epage%253D4877%26rft.atitle%253DRedesign%2Bof%2Bsoluble%2Bfatty%2Bacid%2Bdesaturases%2Bfrom%2Bplants%2Bfor%2Baltered%2Bsubstrate%2Bspecificity%2Band%2Bdouble%2Bbond%2Bposition%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.10.4872%26rft_id%253Dinfo%253Apmid%252F9144157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTAvNDg3MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yOTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.11" data-doi="10.1104/pp.117.2.593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cahoon</span>,  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Shah</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shanklin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Browse</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A Determinant of Substrate Specificity Predicted from the Acyl-Acyl Carrier Protein Desaturase of Developing Cat’s Claw Seed.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">598</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.117.2.593</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DCahoon%26rft.auinit1%253DE.%2BB.%26rft.volume%253D117%26rft.issue%253D2%26rft.spage%253D593%26rft.epage%253D598%26rft.atitle%253DA%2BDeterminant%2Bof%2BSubstrate%2BSpecificity%2BPredicted%2Bfrom%2Bthe%2BAcyl-Acyl%2BCarrier%2BProtein%2BDesaturase%2Bof%2BDeveloping%2BCat%2527s%2BClaw%2BSeed%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.117.2.593%26rft_id%253Dinfo%253Apmid%252F9625712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMTcvMi81OTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjk2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.12" data-doi="10.3835/plantgenome2015.10.0102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaffin</span>,  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-F.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Bekele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Babiker</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> A Consensus Map in Cultivated Hexaploid Oat Reveals Conserved Grass Synteny with Substantial Subgenome Rearrangement. Plant Genome 9<span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.10.0102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.10.0102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.10.0102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.13" data-doi="10.1104/pp.111.185033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>,  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">McMullen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Holland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Tian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bradbury</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic Architecture of Maize Kernel Composition in the Nested Association Mapping and Inbred Association Panels.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">824</span>–<span class="cit-lpage">834</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.111.185033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.185033%26rft_id%253Dinfo%253Apmid%252F22135431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTgvMi84MjQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjk2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.14" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.15" data-doi="10.3835/plantgenome2015.10.0103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esvelt Klos</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-F.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Bekele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Obert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Babiker</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Population Genomics Related to Adaptation in Elite Oat Germplasm.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.10.0103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.10.0103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.10.0103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.16" data-doi="10.3389/fpls.2016.02062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gacek</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. E.</span> <span class="cit-name-surname">Bayer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Bartkowiak-Broda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Szala</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bocianowski</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-Wide Association Study of Genetic Control of Seed Fatty Acid Biosynthesis in Brassica napus.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2062</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.02062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D2062%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2016.02062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2016.02062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.17" data-doi="10.1371/journal.pgen.1000282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gieger</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Geistlinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Altmaier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">de Angelis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Kronenberg</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetics Meets Metabolomics: A Genome-Wide Association Study of Metabolite Profiles in Human Serum.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000282</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000282</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGieger%26rft.auinit1%253DC.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253De1000282%26rft.epage%253De1000282%26rft.atitle%253DGenetics%2Bmeets%2Bmetabolomics%253A%2Ba%2Bgenome-wide%2Bassociation%2Bstudy%2Bof%2Bmetabolite%2Bprofiles%2Bin%2Bhuman%2Bserum.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000282%26rft_id%253Dinfo%253Apmid%252F19043545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19043545&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2963.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gogel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Cullis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Butler</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">ASReml user guide release 3.0</span>, <span class="cit-publ-name">VSN Int Ltd.</span>, <span class="cit-publ-loc">Hemel Hempstead, UK</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.19" data-doi="10.1039/C7FO02006F"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grundy</span>,  <span class="cit-name-given-names">M. M.-L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Fardet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Tosh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. T.</span> <span class="cit-name-surname">Rich</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Wilde</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Processing of oat: the impact on oat’s cholesterol lowering effect.</span> <abbr class="cit-jnl-abbrev">Food Funct.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1328</span>–<span class="cit-lpage">1343</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1039/C7FO02006F</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BFunct.%26rft.volume%253D9%26rft.spage%253D1328%26rft_id%253Dinfo%253Adoi%252F10.1039%252FC7FO02006F%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/C7FO02006F&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.20" data-doi="10.1038/ng.3506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Shi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bhatia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Chung</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Integrative approaches for large-scale transcriptome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3506</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3506%26rft_id%253Dinfo%253Apmid%252F26854917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26854917&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.21" data-doi="10.5650/jos.ess15074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halima</span>,  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. B.</span> <span class="cit-name-surname">Saad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Khemakhem</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Fendri</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Abdelkafi</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Oat (Avena sativa L.): Oil and Nutriment Compounds Valorization for Potential Use in Industrial Applications.</span> <abbr class="cit-jnl-abbrev">J. Oleo Sci.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">915</span>–<span class="cit-lpage">932</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5650/jos.ess15074</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BOleo%2BSci.%26rft.volume%253D64%26rft.spage%253D915%26rft_id%253Dinfo%253Adoi%252F10.5650%252Fjos.ess15074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5650/jos.ess15074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrell</span>,  <span class="cit-name-given-names">F. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-comment">Hmisc: Harrell Miscellaneous. R package version 4.1-1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.23" data-doi="10.3835/plantgenome2012.07.0015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hizbai</span>,  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Gardner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. P.</span> <span class="cit-name-surname">Wight</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Dhanda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Molnar</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative Trait Loci Affecting Oil Content, Oil Composition, and Other Agronomically Important Traits in Oat.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.07.0015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D5%26rft.spage%253D164%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.07.0015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.07.0015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2963.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. E.</span> <span class="cit-name-surname">Nyquist</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. T.</span> <span class="cit-name-surname">Cervantes‐Martínez</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Estimating and Interpreting Heritability for Plant Breeding: An Update</span>, pp. 9–112 in <span class="cit-source">Plant Breeding Reviews</span>, <span class="cit-publ-name">Wiley-Blackwell</span>, <span class="cit-publ-loc">Hoboken, NJ</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.25" data-doi="10.1038/hdy.2011.103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>,  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Browne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Guill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Coles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Eller</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">490</span>–<span class="cit-lpage">499</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2011.103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHung%26rft.auinit1%253DH.%2BY.%26rft.volume%253D108%26rft.issue%253D5%26rft.spage%253D490%26rft.epage%253D499%26rft.atitle%253DThe%2Brelationship%2Bbetween%2Bparental%2Bgenetic%2Bor%2Bphenotypic%2Bdivergence%2Band%2Bprogeny%2Bvariation%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.103%26rft_id%253Dinfo%253Apmid%252F22027895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22027895&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.26" data-doi="10.1139/g94-087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jellen</span>,  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. S.</span> <span class="cit-name-surname">Gill</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Cox</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genomic in situ hybridization differentiates between A/D- and C-genome chromatin and detects intergenomic translocations in polyploid oat species (genus Avena).</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g94-087</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DJellen%26rft.auinit1%253DE.%2BN.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D613%26rft.epage%253D618%26rft.atitle%253DGenomic%2Bin%2Bsitu%2Bhybridization%2Bdifferentiates%2Bbetween%2BA%252FD-%2Band%2BC-genome%2Bchromatin%2Band%2Bdetects%2Bintergenomic%2Btranslocations%2Bin%2Bpolyploid%2Boat%2Bspecies%2B%2528genus%2BAvena%2529.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg94-087%26rft_id%253Dinfo%253Apmid%252F18470105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g94-087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18470105&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.27" data-doi="10.1007/s001220051147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kianian</span>,  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Egli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Phillips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. W.</span> <span class="cit-name-surname">Rines</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Somers</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Association of a major groat oil content QTL and an acetyl-CoA carboxylase gene in oat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">884</span>–<span class="cit-lpage">894</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220051147</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D98%26rft.spage%253D884%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000080534000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.28" data-doi="10.5351/CSAM.2015.22.6.665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ppcor: An R Package for a Fast Calculation to Semi-partial Correlation Coefficients.</span> <abbr class="cit-jnl-abbrev">Commun. Stat. Appl. Methods</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">665</span>–<span class="cit-lpage">674</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5351/CSAM.2015.22.6.665</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommun.%2BStat.%2BAppl.%2BMethods%26rft.volume%253D22%26rft.spage%253D665%26rft_id%253Dinfo%253Adoi%252F10.5351%252FCSAM.2015.22.6.665%26rft_id%253Dinfo%253Apmid%252F26688802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5351/CSAM.2015.22.6.665&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26688802&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.29" data-doi="10.1038/ng.2376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Vilhjálmsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Segura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Platt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Long</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A mixed-model approach for genome-wide association studies of correlated traits in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1066</span>–<span class="cit-lpage">1071</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKorte%26rft.auinit1%253DA.%26rft.volume%253D44%26rft.issue%253D9%26rft.spage%253D1066%26rft.epage%253D1071%26rft.atitle%253DA%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bof%2Bcorrelated%2Btraits%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2376%26rft_id%253Dinfo%253Apmid%252F22902788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22902788&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.30" data-doi="10.1186/1752-0509-5-21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krumsiek</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Suhre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Illig</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Adamski</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Theis</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Gaussian graphical modeling reconstructs pathway reactions from high-throughput metabolomics data.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1752-0509-5-21</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DKrumsiek%26rft.auinit1%253DJ.%26rft.volume%253D5%26rft.spage%253D21%26rft.epage%253D21%26rft.atitle%253DGaussian%2Bgraphical%2Bmodeling%2Breconstructs%2Bpathway%2Breactions%2Bfrom%2Bhigh-throughput%2Bmetabolomics%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-5-21%26rft_id%253Dinfo%253Apmid%252F21281499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-5-21&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21281499&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2963.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kutner</span>,  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Nachtsheim</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Neter</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source"><em>Applied linear regression models</em>.</span> <span class="cit-publ-name">McGraw-Hill/Irwin</span>, <span class="cit-publ-loc">Boston, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.32" data-doi="10.1021/jf800761c"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leonova</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Shelenga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hamberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Konarev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Loskutov</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Analysis of Oil Composition in Cultivars and Wild Species of Oat (Avena sp.).</span> <abbr class="cit-jnl-abbrev">J. Agric. Food Chem.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">7983</span>–<span class="cit-lpage">7991</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/jf800761c</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.stitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.aulast%253DLeonova%26rft.auinit1%253DS.%26rft.volume%253D56%26rft.issue%253D17%26rft.spage%253D7983%26rft.epage%253D7991%26rft.atitle%253DAnalysis%2Bof%2Boil%2Bcomposition%2Bin%2Bcultivars%2Band%2Bwild%2Bspecies%2Bof%2Boat%2B%2528Avena%2Bsp.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjf800761c%26rft_id%253Dinfo%253Apmid%252F18707115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jf800761c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18707115&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258938000060&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.33" data-doi="10.1126/science.aau2096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ludwig</span>,  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. C.</span> <span class="cit-name-surname">Willett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Volek</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Neuhouser</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Dietary fat: From foe to friend?</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">362</span>: <span class="cit-fpage">764</span>–<span class="cit-lpage">770</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aau2096</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLudwig%26rft.auinit1%253DD.%2BS.%26rft.volume%253D362%26rft.issue%253D6416%26rft.spage%253D764%26rft.epage%253D770%26rft.atitle%253DDietary%2Bfat%253A%2BFrom%2Bfoe%2Bto%2Bfriend%253F%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aau2096%26rft_id%253Dinfo%253Apmid%252F30442800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjIvNjQxNi83NjQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjk2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.34" data-doi="10.1038/ng.2484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Peng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">50</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2484</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2484%26rft_id%253Dinfo%253Apmid%252F23242369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23242369&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.35" data-doi="10.1199/tab.0161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li-Beisson</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Shorrosh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Beisson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. X.</span> <span class="cit-name-surname">Andersson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Arondel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-comment">Acyl-Lipid Metabolism. Arabidopsis Book. 8: e0161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1199/tab.0161</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTHE%2BARABIDOPSIS%2BBOOK%26rft.volume%253D11%26rft.spage%253D0161e%26rft_id%253Dinfo%253Adoi%252F10.1199%252Ftab.0161%26rft_id%253Dinfo%253Apmid%252F23505340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1199/tab.0161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23505340&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.36" data-doi="10.1534/g3.113.006148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>,  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Gore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Magallanes-Lundback</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mesberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> Genome-Wide Association Study and Pathway-Level Analysis of Tocochromanol Levels in Maize Grain. G3 Genes Genomes Genet. 3: 1287–1299.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.006148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.006148%26rft_id%253Dinfo%253Apmid%252F23733887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjMvOC8xMjg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.37" data-doi="10.1016/j.pbi.2015.02.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>,  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Kandianis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Hudson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Drnevich</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">From association to prediction: statistical methods for the dissection and selection of complex traits in plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2015.02.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D24%26rft.spage%253D110%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2015.02.010%26rft_id%253Dinfo%253Apmid%252F25795170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2015.02.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25795170&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.38" data-doi="10.1093/bioinformatics/bts444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>,  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Tian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Peiffer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GAPIT: genome association and prediction integrated tool.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2397</span>–<span class="cit-lpage">2399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts444</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts444%26rft_id%253Dinfo%253Apmid%252F22796960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22796960&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308532300066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.39" data-doi="10.1111/j.1753-4887.2009.00256.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meydani</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Potential health benefits of avenanthramides of oats.</span> <abbr class="cit-jnl-abbrev">Nutr. Rev.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">735</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1753-4887.2009.00256.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNutr.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1753-4887.2009.00256.x%26rft_id%253Dinfo%253Apmid%252F19941618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1753-4887.2009.00256.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19941618&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.40" data-doi="10.1177/0967033517709615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montilla-Bascón</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Armstrong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Han</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Quantification of betaglucans, lipid and protein contents in whole oat groats (Avena sativa L.) using near infrared reflectance spectroscopy.</span> <abbr class="cit-jnl-abbrev">J. Infrared Spectrosc.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1177/0967033517709615</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInfrared%2BSpectrosc.%26rft.volume%253D25%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0967033517709615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/0967033517709615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.41" data-doi="10.1007/s00122-012-1945-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newell</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. G.</span> <span class="cit-name-surname">Asoro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Scott</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">White</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. D.</span> <span class="cit-name-surname">Beavis</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association study for oat (Avena sativa L.) beta-glucan concentration using germplasm of worldwide origin.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1687</span>–<span class="cit-lpage">1696</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1945-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DNewell%26rft.auinit1%253DM.%2BA.%26rft.volume%253D125%26rft.issue%253D8%26rft.spage%253D1687%26rft.epage%253D1696%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bfor%2Boat%2B%2528Avena%2Bsativa%2BL.%2529%2Bbeta-glucan%2Bconcentration%2Busing%2Bgermplasm%2Bof%2Bworldwide%2Borigin.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1945-0%26rft_id%253Dinfo%253Apmid%252F22865125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1945-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22865125&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.42" data-doi="10.1371/journal.pone.0034861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Reilly</span>,  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Hoggart</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Pomyen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. C. F.</span> <span class="cit-name-surname">Calboli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Elliott</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MultiPhen: Joint Model of Multiple Phenotypes Can Increase Discovery in GWAS.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e34861</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0034861</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DO%2527Reilly%26rft.auinit1%253DP.%2BF.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De34861%26rft.epage%253De34861%26rft.atitle%253DMultiPhen%253A%2Bjoint%2Bmodel%2Bof%2Bmultiple%2Bphenotypes%2Bcan%2Bincrease%2Bdiscovery%2Bin%2BGWAS.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0034861%26rft_id%253Dinfo%253Apmid%252F22567092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0034861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22567092&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.43" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>,  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of High-Density Genetic Maps for Barley and Wheat Using a Novel Two-Enzyme Genotyping-by-Sequencing Approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e32253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0032253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.44" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>,  <span class="cit-name-given-names">J. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. W.</span> <span class="cit-name-surname">Rife</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-Sequencing for Plant Breeding and Genetics.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.05.0005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.45" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>,  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Patterson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Plenge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Weinblatt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Shadick</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.46" data-doi="10.1086/321275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J. K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Przeworski</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Linkage Disequilibrium in Humans: Models and Data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/321275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D14%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bin%2Bhumans%253A%2Bmodels%2Band%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321275%26rft_id%253Dinfo%253Apmid%252F11410837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11410837&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170108100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.47" data-doi="10.1534/genetics.112.141473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Laloë</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nicolas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Altmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Brunel</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maximizing the Reliability of Genomic Selection by Optimizing the Calibration Set of Reference Individuals: Comparison of Methods in Two Diverse Groups of Maize Inbreds (Zea mays L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">728</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.141473</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141473%26rft_id%253Dinfo%253Apmid%252F22865733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saastamoinen</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kumpulainen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nummela</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Genetic and Environmental Variation in Oil Content and Fatty Acid Composition of Oats.</span> <abbr class="cit-jnl-abbrev">Cereal Chem.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">296</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCereal%2BChem.%26rft.volume%253D66%26rft.spage%253D296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.49" data-doi="10.1534/genetics.118.301851"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santantonio</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Homeologous Epistasis in Wheat: The Search for an Immortal Hybrid.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301851</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301851%26rft_id%253Dinfo%253Apmid%252F30679260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIxMS8zLzExMDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjk2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.50" data-doi="10.1214/aos/1176344136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Estimating the Dimension of a Model.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aos/1176344136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D6%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978EQ63300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.51" data-doi="10.1111/j.1439-0523.2003.00968.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snowdon</span>,  <span class="cit-name-given-names">R. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Friedt</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Molecular markers in Brassica oilseed breeding: current status and future possibilities.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0523.2003.00968.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D123%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2003.00968.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2003.00968.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000189142400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.52" data-doi="10.1093/bioinformatics/btm069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stacklies</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Redestig</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Scholz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Walther</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Selbig</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">pcaMethods—a bioconductor package providing PCA methods for incomplete data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1164</span>–<span class="cit-lpage">1167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm069</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm069%26rft_id%253Dinfo%253Apmid%252F17344241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17344241&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246773300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.53" data-doi="10.1371/journal.pone.0065245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Unified Framework for Association Analysis with Multiple Related Phenotypes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e65245</span> (erratum: PLoS One 14: e0213951).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0065245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De65245%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0065245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0065245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>,  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Bass</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Dabney</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Robinson</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-comment"><em>qvalue: Q-value estimation for false discovery rate control</em>. R package version 2.14.1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.55" data-doi="10.1038/hdy.2010.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Kramer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.-S.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variation explained in mixed-model association mapping.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">340</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2010.11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DSun%26rft.auinit1%253DG.%26rft.volume%253D105%26rft.issue%253D4%26rft.spage%253D333%26rft.epage%253D340%26rft.atitle%253DVariation%2Bexplained%2Bin%2Bmixed-model%2Bassociation%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.11%26rft_id%253Dinfo%253Apmid%252F20145669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20145669&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sunstrum</span>,  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Bekele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. P.</span> <span class="cit-name-surname">Wight</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Yan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">A genetic linkage map in southern-by-spring oat identifies multiple quantitative trait loci for adaptation and rust resistance.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D138%26rft.spage%253D82%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.57" data-doi="10.1139/g2012-017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanhuanpää</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Manninen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Beattie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Eckstein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Scoles</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An updated doubled haploid oat linkage map and QTL mapping of agronomic and grain quality traits from Canadian field trials.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">301</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g2012-017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DTanhuanpaa%26rft.auinit1%253DP.%26rft.volume%253D55%26rft.issue%253D4%26rft.spage%253D289%26rft.epage%253D301%26rft.atitle%253DAn%2Bupdated%2Bdoubled%2Bhaploid%2Boat%2Blinkage%2Bmap%2Band%2BQTL%2Bmapping%2Bof%2Bagronomic%2Band%2Bgrain%2Bquality%2Btraits%2Bfrom%2BCanadian%2Bfield%2Btrials.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg2012-017%26rft_id%253Dinfo%253Apmid%252F22443510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g2012-017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22443510&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>,  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Bekele</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hattori</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis. G3 Genes Genomes Genet. 6: 857–863.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2963.59" data-doi="10.1094/9781891127649.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valentine</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Cowan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Marshall</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Oat Breeding</span>, pp. <span class="cit-fpage">11</span>–<span class="cit-lpage">30</span> in <span class="cit-source">Oats: Chemistry and Technology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Webster</span>,  <span class="cit-name-given-names">F. H.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Wood</span></span>. <span class="cit-publ-name">AACC International, Inc.</span>, <span class="cit-publ-loc">St. Paul, Minnesota</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/9781891127649.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOats%253A%2BChemistry%2Band%2BTechnology%26rft_id%253Dinfo%253Adoi%252F10.1094%252F9781891127649.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/9781891127649.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.60" data-doi="10.1016/j.tpb.2008.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanLiere</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Rosenberg</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mathematical properties of the r2 measure of linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tpb.2008.05.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DVanliere%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D130%26rft.epage%253D137%26rft.atitle%253DMathematical%2Bproperties%2Bof%2Bthe%2Br2%2Bmeasure%2Bof%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2008.05.006%26rft_id%253Dinfo%253Apmid%252F18572214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2008.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18572214&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257912400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Simko</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-comment">R package “corrplot”: Visualization of a Correlation Matrix. R package version 0.84</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2963.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-comment"><em>ggplot2: Elegant Graphics for Data Analysis</em>. Springer-Verlag, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.63" data-doi="10.1002/gepi.20497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-Y.</span> <span class="cit-name-surname">Guo</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Fox</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analyze multivariate phenotypes in genetic association studies by combining univariate association tests.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20497</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DYang%26rft.auinit1%253DQ.%26rft.volume%253D34%26rft.issue%253D5%26rft.spage%253D444%26rft.epage%253D454%26rft.atitle%253DAnalyze%2Bmultivariate%2Bphenotypes%2Bin%2Bgenetic%2Bassociation%2Bstudies%2Bby%2Bcombining%2Bunivariate%2Bassociation%2Btests.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20497%26rft_id%253Dinfo%253Apmid%252F20583287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20583287&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280349600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.64" data-doi="10.1016/j.molp.2017.12.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Bhusal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome-wide Scan for Seed Composition Provides Insights into Soybean Quality Improvement and the Impacts of Domestication and Breeding.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">460</span>–<span class="cit-lpage">472</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molp.2017.12.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft.volume%253D11%26rft.spage%253D460%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molp.2017.12.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molp.2017.12.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.65" data-doi="10.1007/s11746-999-0213-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Robards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Glennie-Holmes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Helliwell</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Oat lipids.</span> <abbr class="cit-jnl-abbrev">J. Am. Oil Chem. Soc.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">169</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11746-999-0213-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BOil%2BChem.%2BSoc.%26rft.volume%253D76%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11746-999-0213-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11746-999-0213-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000079034700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.66" data-doi="10.1038/nmeth.2848"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">X.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient multivariate linear mixed model algorithms for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">409</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2848</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D407%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2848%26rft_id%253Dinfo%253Apmid%252F24531419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2848&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24531419&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.67" data-doi="10.1038/ng.2310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">X.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide efficient mixed-model analysis for association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D821%26rft.epage%253D824%26rft.atitle%253DGenome-wide%2Befficient%2Bmixed-model%2Banalysis%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2310%26rft_id%253Dinfo%253Apmid%252F22706312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706312&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2963.68" data-doi="10.2135/cropsci2004.0254"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. G.</span> <span class="cit-name-surname">Rossnagel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. F.</span> <span class="cit-name-surname">Kaeppler</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic Analysis of Quantitative Trait Loci for Groat Protein and Oil Content in Oat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">254</span>–<span class="cit-lpage">260</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2004.2540</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DZhu%26rft.auinit1%253DS.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D254%26rft.epage%253D260%26rft.atitle%253DGenetic%2BAnalysis%2Bof%2BQuantitative%2BTrait%2BLoci%2Bfor%2BGroat%2BProtein%2Band%2BOil%2BContent%2Bin%2BOat%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0254&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187633900035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/9/2963.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2951" title="Combining Transcriptomics and Proteomics Reveals Potential Post-transcriptional Control of Gene Expression After Light Exposure in Metarhizium acridum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2977" title="FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/9"><span>Volume 9 Issue 9, September 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/9.cover-source.jpg?itok=4kZTDz_V" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2963" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cqa8fssd7Tgjthz3lQ4Ga4ZGPprfZf_CqefTMuHs3cw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/9/2963" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/9/2963" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/9/2963" class="active" data-icon-position="" data-hide-link-title="0">Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (Avena sativa L.)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494061" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2Kw8mBsuaZdvXjM0A5iBt04iau1QCh33N3jQ6LrDDB0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2963.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/9/2963" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-2oRPnv_HKa3EWV6wJFhjYlYXMjHli4ZQ7Nkf-pSVBkA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/9/2963" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2963.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494061" id="node494061--2" data-pisa="ggg;9/9/2963" data-pisa-master="ggg;g3.119.400228" data-apath="/ggg/9/9/2963.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (<em>Avena sativa</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Maryn O. Carlson</span>, <span class="highwire-citation-author" data-delta="1">Gracia Montilla-Bascon</span>, <span class="highwire-citation-author" data-delta="2">Owen A. Hoekenga</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2452-4779" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicholas A. Tinker</span>, <span class="highwire-citation-author" data-delta="4">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="5">Matheus Baseggio</span>, <span class="highwire-citation-author" data-delta="6">Mark E. Sorrells</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-6287-952X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Trevor H. Yeats</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2963-2975;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400228 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494061--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maryn O. Carlson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maryn%2BO.%2BCarlson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlson%20MO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaryn%2BO.%2BCarlson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mocarlson@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">mocarlson@uchicago.edu</a>
<a href="mailto:thy6@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">thy6@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gracia Montilla-Bascon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIC, Institute for Sustainable Agriculture, Córdoba, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gracia%2BMontilla-Bascon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montilla-Bascon%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGracia%2BMontilla-Bascon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Owen A. Hoekenga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cayuga Genetics Consulting Group LLC, Ithaca, NY 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Owen%2BA.%2BHoekenga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoekenga%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOwen%2BA.%2BHoekenga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicholas A. Tinker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ottawa, ON, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BA.%2BTinker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tinker%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BA.%2BTinker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2452-4779" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicholas A. Tinker</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Wheat Genetics Resource Center, Department of Plant Pathology, Kansas State University, Manhattan, KS 66506, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matheus Baseggio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matheus%2BBaseggio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baseggio%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatheus%2BBaseggio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mark E. Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BE.%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BE.%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div><div class='author-affiliation'><span class='nlm-sup'>††</span>R.W. Holley Center for Agriculture and Health, US Department of Agriculture, Agricultural Research Service, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Trevor H. Yeats </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Trevor%2BH.%2BYeats%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeats%20TH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATrevor%2BH.%2BYeats%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6287-952X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Trevor H. Yeats</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mocarlson@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">mocarlson@uchicago.edu</a>
<a href="mailto:thy6@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">thy6@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/9/2963" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/9/2963&amp;title=Multivariate%20Genome-Wide%20Association%20Analyses%20Reveal%20the%20Genetic%20Basis%20of%20Seed%20Fatty%20Acid%20Composition%20in%20Oat%20%28Avena%20sativa%20L.%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/9/2963&amp;title=Multivariate%20Genome-Wide%20Association%20Analyses%20Reveal%20the%20Genetic%20Basis%20of%20Seed%20Fatty%20Acid%20Composition%20in%20Oat%20%28Avena%20sativa%20L.%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/9/2963&amp;title=Multivariate%20Genome-Wide%20Association%20Analyses%20Reveal%20the%20Genetic%20Basis%20of%20Seed%20Fatty%20Acid%20Composition%20in%20Oat%20%28Avena%20sativa%20L.%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/9/2963&amp;text=Multivariate%20Genome-Wide%20Association%20Analyses%20Reveal%20the%20Genetic%20Basis%20of%20Seed%20Fatty%20Acid%20Composition%20in%20Oat%20%28Avena%20sativa%20L.%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/9/2963&amp;title=Multivariate%20Genome-Wide%20Association%20Analyses%20Reveal%20the%20Genetic%20Basis%20of%20Seed%20Fatty%20Acid%20Composition%20in%20Oat%20%28Avena%20sativa%20L.%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/9/2963&amp;t=Multivariate%20Genome-Wide%20Association%20Analyses%20Reveal%20the%20Genetic%20Basis%20of%20Seed%20Fatty%20Acid%20Composition%20in%20Oat%20%28Avena%20sativa%20L.%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/9/2963&amp;title=Multivariate%20Genome-Wide%20Association%20Analyses%20Reveal%20the%20Genetic%20Basis%20of%20Seed%20Fatty%20Acid%20Composition%20in%20Oat%20%28Avena%20sativa%20L.%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/9/2963&amp;title=Multivariate%20Genome-Wide%20Association%20Analyses%20Reveal%20the%20Genetic%20Basis%20of%20Seed%20Fatty%20Acid%20Composition%20in%20Oat%20%28Avena%20sativa%20L.%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494061/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494061" id="node494061--3" data-pisa="ggg;9/9/2963" data-pisa-master="ggg;g3.119.400228" data-apath="/ggg/9/9/2963.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (<em>Avena sativa</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Maryn O. Carlson</span>, <span class="highwire-citation-author" data-delta="1">Gracia Montilla-Bascon</span>, <span class="highwire-citation-author" data-delta="2">Owen A. Hoekenga</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2452-4779" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicholas A. Tinker</span>, <span class="highwire-citation-author" data-delta="4">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="5">Matheus Baseggio</span>, <span class="highwire-citation-author" data-delta="6">Mark E. Sorrells</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-6287-952X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Trevor H. Yeats</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2963-2975;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400228 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494061--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maryn O. Carlson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maryn%2BO.%2BCarlson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlson%20MO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaryn%2BO.%2BCarlson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mocarlson@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">mocarlson@uchicago.edu</a>
<a href="mailto:thy6@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">thy6@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gracia Montilla-Bascon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIC, Institute for Sustainable Agriculture, Córdoba, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gracia%2BMontilla-Bascon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montilla-Bascon%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGracia%2BMontilla-Bascon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Owen A. Hoekenga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cayuga Genetics Consulting Group LLC, Ithaca, NY 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Owen%2BA.%2BHoekenga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoekenga%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOwen%2BA.%2BHoekenga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicholas A. Tinker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ottawa, ON, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BA.%2BTinker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tinker%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BA.%2BTinker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2452-4779" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicholas A. Tinker</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Wheat Genetics Resource Center, Department of Plant Pathology, Kansas State University, Manhattan, KS 66506, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matheus Baseggio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matheus%2BBaseggio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baseggio%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatheus%2BBaseggio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mark E. Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BE.%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BE.%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div><div class='author-affiliation'><span class='nlm-sup'>††</span>R.W. Holley Center for Agriculture and Health, US Department of Agriculture, Agricultural Research Service, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Trevor H. Yeats </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding & Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Trevor%2BH.%2BYeats%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeats%20TH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATrevor%2BH.%2BYeats%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6287-952X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Trevor H. Yeats</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mocarlson@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">mocarlson@uchicago.edu</a>
<a href="mailto:thy6@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">thy6@cornell.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2963&amp;rft.epage=2975&amp;rft.atitle=Multivariate%20Genome-Wide%20Association%20Analyses%20Reveal%20the%20Genetic%20Basis%20of%20Seed%20Fatty%20Acid%20Composition%20in%20Oat%20%28Avena%20sativa%20L.%29&amp;rft.volume=9&amp;rft.issue=9&amp;rft.date=2019-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Carlson%2C+Maryn+O.&amp;rft.au=Montilla-Bascon%2C+Gracia&amp;rft.au=Hoekenga%2C+Owen+A.&amp;rft.au=Tinker%2C+Nicholas+A.&amp;rft.au=Poland%2C+Jesse&amp;rft.au=Baseggio%2C+Matheus&amp;rft.au=Sorrells%2C+Mark+E.&amp;rft.au=Jannink%2C+Jean-Luc&amp;rft.au=Gore%2C+Michael+A.&amp;rft.au=Yeats%2C+Trevor+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494061/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494061/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494061/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494061/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494061/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494061/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494061/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494061/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494061/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494061/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494061/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494061/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/9/2963" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/9/2963.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/9/2963.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/9/2963.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
