<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/9/2835" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/9/2835.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/9/2835.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/9/2835.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/9/2835.atom" />
<meta name="HW.pisa" content="ggg;9/9/2835" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Single-Gene Deletions Contributing to Loss of Heterozygosity in Saccharomyces cerevisiae: Genome-Wide Screens and Reproducibility" />
<meta name="DC.Identifier" content="10.1534/g3.119.400429" />
<meta name="DC.Date" content="2019-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Hoffert, Strome. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Loss of heterozygosity (LOH) is a phenomenon commonly observed in cancers; the loss of chromosomal regions can be both causal and indicative of underlying genome instability. Yeast has long been used as a model organism to study genetic mechanisms difficult to study in mammalian cells. Studying gene deletions leading to increased LOH in yeast aids our understanding of the processes involved, and guides exploration into the etiology of LOH in cancers. Yet, before in-depth mechanistic studies can occur, candidate genes of interest must be identified. Utilizing the heterozygous Saccharomyces cerevisiae deletion collection (≈ 6500 strains), 217 genes whose disruption leads to increased LOH events at the endogenously heterozygous mating type locus were identified. Our investigation to refine this list of genes to candidates with the most definite impact on LOH includes: secondary testing for LOH impact at an additional locus, gene ontology analysis to determine common gene characteristics, and positional gene enrichment studies to identify chromosomal regions important in LOH events. Further, we conducted extensive comparisons of our data to screens with similar, but distinct methodologies, to further distinguish genes that are more likely to be true contributors to instability due to their reproducibility, and not just identified due to the stochastic nature of LOH. Finally, we selected nine candidate genes and quantitatively measured their impact on LOH as a benchmark for the impact of genes identified in our study. Our data add to the existing body of work and strengthen the evidence of single-gene knockdowns contributing to genome instability." />
<meta name="DC.Contributor" content="Kellyn M. Hoffert" />
<meta name="DC.Contributor" content="Erin D. Strome" />
<meta name="article:published_time" content="2019-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Single-Gene Deletions Contributing to Loss of Heterozygosity in Saccharomyces cerevisiae: Genome-Wide Screens and Reproducibility" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Loss of heterozygosity (LOH) is a phenomenon commonly observed in cancers; the loss of chromosomal regions can be both causal and indicative of underlying genome instability. Yeast has long been used as a model organism to study genetic mechanisms difficult to study in mammalian cells. Studying gene deletions leading to increased LOH in yeast aids our understanding of the processes involved, and guides exploration into the etiology of LOH in cancers. Yet, before in-depth mechanistic studies can occur, candidate genes of interest must be identified. Utilizing the heterozygous &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; deletion collection (≈ 6500 strains), 217 genes whose disruption leads to increased LOH events at the endogenously heterozygous mating type locus were identified. Our investigation to refine this list of genes to candidates with the most definite impact on LOH includes: secondary testing for LOH impact at an additional locus, gene ontology analysis to determine common gene characteristics, and positional gene enrichment studies to identify chromosomal regions important in LOH events. Further, we conducted extensive comparisons of our data to screens with similar, but distinct methodologies, to further distinguish genes that are more likely to be true contributors to instability due to their reproducibility, and not just identified due to the stochastic nature of LOH. Finally, we selected nine candidate genes and quantitatively measured their impact on LOH as a benchmark for the impact of genes identified in our study. Our data add to the existing body of work and strengthen the evidence of single-gene knockdowns contributing to genome instability.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/09/01" />
<meta name="citation_mjid" content="ggg;9/9/2835" />
<meta name="citation_id" content="9/9/2835" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/9/2835" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/9/2835.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/9/2835.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/9/2835.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400429" />
<meta name="citation_pmid" content="31270132" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2835" />
<meta name="citation_lastpage" content="2850" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kellyn M. Hoffert" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Northern Kentucky University, Highland Heights, Kentucky, 41099" />
<meta name="citation_author" content="Erin D. Strome" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Northern Kentucky University, Highland Heights, Kentucky, 41099" />
<meta name="citation_author_email" content="stromee1@nku.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6334-9619" />
<meta name="citation_reference" content="citation_journal_title=Mol. Oncol.;citation_author=T. M. A. Abdel-Fatah;citation_author=R. Russell;citation_author=N. Albarakati;citation_author=D. J. Maloney;citation_author=D. Dorjsuren;citation_title=Genomic and protein expression analysis reveals flap endonuclease 1 (FEN1) as a key biomarker in breast and ovarian cancer.;citation_pages=1326-1338;citation_volume=8;citation_year=2014;citation_pmid=24880630;citation_doi=10.1016/j.molonc.2014.04.009" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Alimonti;citation_author=A. Carracedo;citation_author=J. G. Clohessy;citation_author=L. C. Trotman;citation_author=C. Nardella;citation_title=Subtle variations in Pten dose determine cancer susceptibility.;citation_pages=454-458;citation_volume=42;citation_year=2010;citation_issue=5;citation_pmid=20400965;citation_doi=10.1038/ng.556" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Amit;citation_author=S. J. Weisberg;citation_author=M. Nadler-Holly;citation_author=E. A. McCormack;citation_author=E. Feldmesser;citation_title=Equivalent mutations in the eight subunits of the chaperonin CCT produce dramatically different cellular and gene expression phenotypes.;citation_pages=532-543;citation_volume=401;citation_year=2010;citation_issue=3;citation_pmid=20600117;citation_doi=10.1016/j.jmb.2010.06.037" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. P. Andersen;citation_author=Z. W. Nelson;citation_author=E. D. Hetrick;citation_author=D. E. Gottschling;citation_title=A Genetic Screen for Increased Loss of Heterozygosity in Saccharomyces cerevisiae.;citation_volume=179;citation_year=2008;citation_pmid=18562670;citation_doi=10.1534/genetics.108.089250" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Bandhu;citation_author=J. Kang;citation_author=K. Fukunaga;citation_author=G. Goto;citation_author=K. Sugimoto;citation_title=Ddc2 mediates Mec1 activation through a Ddc1- or Dpb11-independent mechanism.;citation_pages=e1004136;citation_volume=10;citation_year=2014;citation_pmid=24586187;citation_doi=10.1371/journal.pgen.1004136" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Soc. Trans.;citation_author=J. D. Beggs;citation_title=Lsm proteins and RNA processing.;citation_volume=33;citation_year=2005;citation_pmid=15916535;citation_doi=10.1042/BST0330433" />
<meta name="citation_reference" content="citation_journal_title=Soc. Ser. B Methodol.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. Ben-Shitrit;citation_author=N. Yosef;citation_author=K. Shemesh;citation_author=R. Sharan;citation_author=E. Ruppin;citation_title=Systematic identification of gene annotation errors in the widely used yeast mutation collections.;citation_pages=373-378;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22306811;citation_doi=10.1038/nmeth.1890" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Berger;citation_author=A. G. Knudson;citation_author=P. P. Pandolfi;citation_title=A continuum model for tumour suppression.;citation_pages=163-169;citation_volume=476;citation_year=2011;citation_issue=7359;citation_pmid=21833082;citation_doi=10.1038/nature10275" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Botstein;citation_author=G. R. Fink;citation_title=Yeast: an experimental organism for 21st Century biology.;citation_volume=189;citation_year=2011;citation_pmid=22084421;citation_doi=10.1534/genetics.111.130765" />
<meta name="citation_reference" content="citation_journal_title=Transl. Oncol.;citation_author=W. H. Choi;citation_author=S. Lee;citation_author=S. Cho;citation_title=Microsatellite alterations and protein expression of 5 major tumor suppressor genes in gastric adenocarcinomas.;citation_pages=43-55;citation_volume=11;citation_year=2018a;citation_doi=10.1016/j.tranon.2017.10.007" />
<meta name="citation_reference" content="citation_journal_title=Oncogenesis;citation_author=C. Choi;citation_author=N. Thi Thao Tran;citation_author=T. Van Ngu;citation_author=S. W. Park;citation_author=M. S. Song;citation_title=Promotion of tumor progression and cancer stemness by MUC15 in thyroid cancer via the GPCR/ERK and integrin-FAK signaling pathways.;citation_pages=85;citation_volume=7;citation_year=2018b;citation_doi=10.1038/s41389-018-0094-y" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=J. S. Choy;citation_author=E. O’Toole;citation_author=B. M. Schuster;citation_author=M. J. Crisp;citation_author=T. S. Karpova;citation_title=Genome-wide haploinsufficiency screen reveals a novel role for γ-TuSC in spindle organization and genome stability.;citation_volume=24;citation_year=2013;citation_pmid=23825022;citation_doi=10.1091/mbc.E12-12-0902" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=K. R. Christie;citation_author=E. L. Hong;citation_author=J. M. Cherry;citation_title=Functional annotations for the Saccharomyces cerevisiae genome: the knowns and the known unknowns.;citation_pages=286-294;citation_volume=17;citation_year=2009;citation_issue=7;citation_pmid=19577472;citation_doi=10.1016/j.tim.2009.04.005" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. A. Cohen;citation_author=R. D. Mitra;citation_author=J. D. Hughes;citation_author=G. M. Church;citation_title=A computational analysis of whole-genome expression data reveals chromosomal domains of gene expression.;citation_pages=183-186;citation_volume=26;citation_year=2000;citation_issue=2;citation_pmid=11017073;citation_doi=10.1038/79896" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=G. J. Cost;citation_author=J. D. Boeke;citation_title=A useful colony colour phenotype associated with the yeast selectable/counter-selectable marker MET15.;citation_pages=939-941;citation_volume=12;citation_year=1996;citation_issue=10;citation_pmid=8873447;citation_doi=10.1002/(SICI)1097-0061(199608)12:10&lt;939::AID-YEA988&gt;3.0.CO;2-L" />
<meta name="citation_reference" content="citation_journal_title=Oncol. Rep.;citation_author=B. Dai;citation_author=P. Zhang;citation_author=Y. Zhang;citation_author=C. Pan;citation_author=G. Meng;citation_title=RNaseH2A is involved in human gliomagenesis through the regulation of cell proliferation and apoptosis.;citation_pages=173-180;citation_volume=36;citation_year=2016;citation_doi=10.3892/or.2016.4802" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. De Preter;citation_author=R. Barriot;citation_author=F. Speleman;citation_author=J. Vandesompele;citation_author=Y. Moreau;citation_title=Positional gene enrichment analysis of gene sets for high-resolution identification of overrepresented chromosomal regions.;citation_volume=36;citation_year=2008;citation_pmid=18346969;citation_doi=10.1093/nar/gkn114" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Deutschbauer;citation_author=D. F. Jaramillo;citation_author=M. Proctor;citation_author=J. Kumm;citation_author=M. E. Hillenmeyer;citation_title=Mechanisms of Haploinsufficiency Revealed by Genome-Wide Profiling in Yeast.;citation_volume=169;citation_year=2005;citation_pmid=15716499;citation_doi=10.1534/genetics.104.036871" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=Z. Dragovic;citation_author=S. A. Broadley;citation_author=Y. Shomura;citation_author=A. Bracher;citation_author=F. U. Hartl;citation_title=Molecular chaperones of the Hsp110 family act as nucleotide exchange factors of Hsp70s;citation_pages=2519-2528;citation_volume=25;citation_year=2006;citation_issue=11;citation_pmid=16688212;citation_doi=10.1038/sj.emboj.7601138" />
<meta name="citation_reference" content="citation_journal_title=Cell Metab.;citation_author=D. J. Drucker;citation_title=Never Waste a Good Crisis: Confronting Reproducibility in Translational Research.;citation_pages=348-360;citation_volume=24;citation_year=2016;citation_pmid=27626191;citation_doi=10.1016/j.cmet.2016.08.006" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. Duntze;citation_author=V. MacKay;citation_author=T. R. Manney;citation_title=Saccharomyces cerevisiae: A Diffusible Sex Factor;citation_pages=1472-1473;citation_volume=168;citation_year=1970;citation_issue=3938;citation_pmid=5445939;citation_doi=10.1126/science.168.3938.1472" />
<meta name="citation_reference" content="citation_journal_title=Cell stress &amp; chaperones;citation_journal_abbrev=Cell Stress Chaperones;citation_author=D. P. Easton;citation_author=Y. Kaneko;citation_author=J. R. Subjeck;citation_title=The hsp110 and Grp1 70 stress proteins: newly recognized relatives of the Hsp70s.;citation_pages=276-290;citation_volume=5;citation_year=2000;citation_issue=4;citation_pmid=11048651;citation_doi=10.1379/1466-1268(2000)005&lt;0276:THAGSP&gt;2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=U. Eser;citation_author=D. Chandler-Brown;citation_author=F. Ay;citation_author=A. F. Straight;citation_author=Z. Duan;citation_title=Form and function of topologically associating genomic domains in budding yeast.;citation_volume=114;citation_year=2017;citation_pmid=28348222;citation_doi=10.1073/pnas.1612256114" />
<meta name="citation_reference" content="citation_journal_title=Cell cycle (Georgetown, Tex.);citation_journal_abbrev=Cell Cycle;citation_author=M. Fasullo;citation_author=M. Sun;citation_title=The Saccharomyces cerevisiae checkpoint genes RAD9, CHK1 and PDS1 are required for elevated homologous recombination in a mec1 (ATR) hypomorphic mutant.;citation_pages=2418-2426;citation_volume=7;citation_year=2008;citation_issue=15;citation_pmid=18677117;citation_doi=10.4161/cc.6411" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Gao;citation_author=Y. Su;citation_author=J. Koeman;citation_author=E. Haak;citation_author=K. Dykema;citation_title=Chromosome instability drives phenotypic switching to metastasis.;citation_volume=113;citation_year=2016;citation_pmid=27930335;citation_doi=10.1073/pnas.1618215113" />
<meta name="citation_reference" content="citation_journal_title=Lupus;citation_journal_abbrev=Lupus;citation_author=M. Gayed;citation_author=S. Bernatsky;citation_author=R. Ramsey-Goldman;citation_author=A. Clarke;citation_author=C. Gordon;citation_title=Lupus and cancer;citation_pages=479-485;citation_volume=18;citation_year=2009;citation_issue=6;citation_pmid=19395448;citation_doi=10.1177/0961203309102556" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. Giaever;citation_author=C. Nislow;citation_title=The Yeast Deletion Collection: A Decade of Functional Genomics;citation_pages=451-465;citation_volume=197;citation_year=2014;citation_issue=2;citation_pmid=24939991;citation_doi=10.1534/genetics.114.161620" />
<meta name="citation_reference" content="citation_journal_title=Cell Div.;citation_author=M. Giam;citation_author=G. Rancati;citation_title=Aneuploidy and chromosomal instability in cancer: a jackpot to chaos.;citation_pages=3;citation_volume=10;citation_year=2015;citation_pmid=26015801;citation_doi=10.1186/s13008-015-0009-7" />
<meta name="citation_reference" content="citation_journal_title=Exp. Cell Res.;citation_author=S. T. Guest;citation_author=Z. R. Kratche;citation_author=A. Bollig-Fischer;citation_author=R. Haddad;citation_author=S. P. Ethier;citation_title=Two members of the TRiC chaperonin complex, CCT2 and TCP1 are essential for survival of breast cancer cells and are linked to driving oncogenes.;citation_pages=223-235;citation_volume=332;citation_year=2015;citation_pmid=25704758;citation_doi=10.1016/j.yexcr.2015.02.005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. E. Haber;citation_title=Mating-Type Genes and MAT Switching in Saccharomyces cerevisiae;citation_pages=33-64;citation_volume=191;citation_year=2012;citation_issue=1;citation_pmid=22555442;citation_doi=10.1534/genetics.111.134577" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. C. Harrison;citation_author=J. E. Haber;citation_title=Surviving the breakup: the DNA damage checkpoint.;citation_pages=209-235;citation_volume=40;citation_year=2006;citation_pmid=16805667;citation_doi=10.1146/annurev.genet.40.051206.105231" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=W. He;citation_author=R. Parker;citation_title=Functions of Lsm proteins in mRNA degradation and splicing.;citation_pages=346-350;citation_volume=12;citation_year=2000;citation_issue=3;citation_pmid=10801455;citation_doi=10.1016/S0955-0674(00)00098-3" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Hennecke;citation_author=J. Beck;citation_author=K. Bornemann-Kolatzki;citation_author=S. Neumann;citation_author=H. Murua Escobar;citation_title=Prevalence of the Prefoldin Subunit 5 Gene Deletion in Canine Mammary Tumors.;citation_pages=e0131280;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0131280" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=I. Herskowitz;citation_title=MAP kinase pathways in yeast: for mating and more.;citation_pages=187-197;citation_volume=80;citation_year=1995;citation_issue=2;citation_pmid=7834739;citation_doi=10.1016/0092-8674(95)90402-6" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. Hieter;citation_author=C. Mann;citation_author=M. Snyder;citation_author=R. W. Davis;citation_title=Mitotic stability of yeast chromosomes: a colony color assay that measures nondisjunction and chromosome loss.;citation_pages=381-392;citation_volume=40;citation_year=1985;citation_issue=2;citation_pmid=3967296;citation_doi=10.1016/0092-8674(85)90152-7" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=T. R. Hughes;citation_author=C. J. Roberts;citation_author=H. Dai;citation_author=A. R. Jones;citation_author=M. R. Meyer;citation_title=Widespread aneuploidy revealed by DNA microarray expression profiling.;citation_pages=333-337;citation_volume=25;citation_year=2000;citation_issue=3;citation_pmid=10888885;citation_doi=10.1038/77116" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Kacser;citation_author=J. A. Burns;citation_title=THE MOLECULAR BASIS OF DOMINANCE;citation_pages=639-666;citation_volume=97;citation_year=1981;citation_issue=3-4;citation_pmid=7297851" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. G. Knudson;citation_title=Antioncogenes and human cancer.;citation_pages=10914-10921;citation_volume=90;citation_year=1993;citation_issue=23;citation_pmid=7902574;citation_doi=10.1073/pnas.90.23.10914" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. G. Knudson;citation_title=Mutation and Cancer: Statistical Study of Retinoblastoma;citation_pages=820-823;citation_volume=68;citation_year=1971;citation_issue=4;citation_pmid=5279523;citation_doi=10.1073/pnas.68.4.820" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=S. Kruglyak;citation_author=H. Tang;citation_title=Regulation of adjacent yeast genes.;citation_pages=109-111;citation_volume=16;citation_year=2000;citation_issue=3;citation_pmid=10689350;citation_doi=10.1016/S0168-9525(99)01941-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Lehner;citation_author=M. M. Stone;citation_author=R. A. Farber;citation_author=T. D. Petes;citation_title=Ninety-six haploid yeast strains with individual disruptions of open reading frames between YOR097C and YOR192C, constructed for the Saccharomyces genome deletion project, have an additional mutation in the mismatch repair gene MSH3.;citation_volume=177;citation_year=2007;citation_pmid=17947417;citation_doi=10.1534/genetics.107.079368" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Lengauer;citation_author=K. W. Kinzler;citation_author=B. Vogelstein;citation_title=Genetic instabilities in human cancers.;citation_pages=643-649;citation_volume=396;citation_year=1998;citation_issue=6712;citation_pmid=9872311;citation_doi=10.1038/25292" />
<meta name="citation_reference" content="citation_journal_title=BMC cancer [electronic resource];citation_journal_abbrev=BMC Cancer;citation_author=P. E. Leone;citation_author=M. Mendiola;citation_author=J. Alonso;citation_author=C. Paz-y-Miño;citation_author=A. Pestaña;citation_title=Implications of a RAD54L polymorphism (2290C/T) in human meningiomas as a risk factor and/or a genetic marker.;citation_pages=6-6;citation_volume=3;citation_year=2003;citation_pmid=12614485;citation_doi=10.1186/1471-2407-3-6" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=A. J. Levine;citation_title=The Tumor Suppressor Genes;citation_pages=623-651;citation_volume=62;citation_year=1993;citation_issue=1;citation_pmid=8394683;citation_doi=10.1146/annurev.bi.62.070193.003203" />
<meta name="citation_reference" content="citation_journal_title=Cell Commun. Signal. CCS;citation_author=M. Li;citation_author=C. Jin;citation_author=M. Xu;citation_author=L. Zhou;citation_author=D. Li;citation_title=Bifunctional enzyme ATIC promotes propagation of hepatocellular carcinoma by regulating AMPK-mTOR-S6 K1 signaling.;citation_pages=52;citation_volume=15;citation_year=2017a" />
<meta name="citation_reference" content="citation_journal_title=Cancer Lett.;citation_author=Y. Li;citation_author=N. Sun;citation_author=Z. Lu;citation_author=S. Sun;citation_author=J. Huang;citation_title=Prognostic alternative mRNA splicing signature in non-small cell lung cancer.;citation_pages=40-51;citation_volume=393;citation_year=2017b;citation_doi=10.1016/j.canlet.2017.02.016" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. A. Loeb;citation_author=K. R. Loeb;citation_author=J. P. Anderson;citation_title=Multiple mutations and cancer.;citation_volume=100;citation_year=2003;citation_pmid=12552134;citation_doi=10.1073/pnas.0334858100" />
<meta name="citation_reference" content="citation_journal_title=Trends in Pharmacological Sciences (Regular ed.);citation_journal_abbrev=Trends in Pharmacological Sciences (Regular ed.);citation_author=W. H. Mager;citation_author=J. Winderickx;citation_title=Yeast as a model for medical and medicinal research.;citation_pages=265-273;citation_volume=26;citation_year=2005;citation_issue=5;citation_pmid=15860374;citation_doi=10.1016/j.tips.2005.03.004" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=M. Manente;citation_author=M. Ghislain;citation_title=The lipid-translocating exporter family and membrane phospholipid homeostasis in yeast.;citation_volume=9;citation_year=2009;citation_pmid=19416366;citation_doi=10.1111/j.1567-1364.2009.00513.x" />
<meta name="citation_reference" content="citation_journal_title=Cancer Res.;citation_author=J. M. Mason;citation_author=H. L. Logan;citation_author=B. Budke;citation_author=M. Wu;citation_author=M. Pawlowski;citation_title=The RAD51-stimulatory compound RS-1 can exploit the RAD51 overexpression that exists in cancer cells and tumors.;citation_volume=74;citation_year=2014;citation_pmid=24753542;citation_doi=10.1158/0008-5472.CAN-13-3220" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=B. Ono;citation_author=N. Ishii;citation_author=S. Fujino;citation_author=I. Aoyama;citation_title=Role of hydrosulfide ions (HS-) in methylmercury resistance in Saccharomyces cerevisiae.;citation_pages=3183-3186;citation_volume=57;citation_year=1991;citation_issue=11;citation_pmid=1781681" />
<meta name="citation_reference" content="citation_journal_title=Carcinogenesis;citation_author=S. R. Payne;citation_author=C. J. Kemp;citation_title=Tumor suppressor genetics.;citation_volume=26;citation_year=2005;citation_pmid=16150895;citation_doi=10.1093/carcin/bgi223" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=O. Protchenko;citation_author=M. Shakoury-Elizeh;citation_author=P. Keane;citation_author=J. Storey;citation_author=R. Androphy;citation_title=Role of PUG1 in Inducible Porphyrin and Heme Transport in Saccharomyces cerevisiae.;citation_volume=7;citation_year=2008;citation_pmid=18326586;citation_doi=10.1128/EC.00414-07" />
<meta name="citation_reference" content="citation_journal_title=Pathol. - Res. Pract.;citation_author=X. Qiu;citation_author=X. He;citation_author=Q. Huang;citation_author=X. Liu;citation_author=G. Sun;citation_title=Overexpression of CCT8 and its significance for tumor cell proliferation, migration and invasion in glioma.;citation_pages=717-725.5;citation_volume=211;citation_year=2015;citation_doi=10.1016/j.prp.2015.04.012" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Rivals;citation_author=L. Personnaz;citation_author=L. Taing;citation_author=M.-C. Potier;citation_title=Enrichment or depletion of a GO category within a class of genes: which test?;citation_volume=23;citation_year=2007;citation_pmid=17182697;citation_doi=10.1093/bioinformatics/btl633" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Sabatti;citation_author=S. Service;citation_author=N. Freimer;citation_title=False Discovery Rate in Linkage and Association Genome Screens for Complex Disorders;citation_pages=829-833;citation_volume=164;citation_year=2003;citation_issue=2;citation_pmid=12807801" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=C. Saveanu;citation_author=M. Fromont-Racine;citation_author=A. Harington;citation_author=F. Ricard;citation_author=A. Namane;citation_title=Identification of 12 new yeast mitochondrial ribosomal proteins including 6 that have no prokaryotic homologues.;citation_volume=276;citation_year=2001;citation_pmid=11278769;citation_doi=10.1074/jbc.M010864200" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=T. Schmidlin;citation_author=K. Matt;citation_author=B. A. Kudlow;citation_author=V. MacKay;citation_author=L. Daniel;citation_title=Single‐gene deletions that restore mating competence to diploid yeast.;citation_pages=276-286;citation_volume=8;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=L. Shaner;citation_author=H. Wegele;citation_author=J. Buchner;citation_author=K. A. Morano;citation_title=The yeast Hsp110 Sse1 functionally interacts with the Hsp70 chaperones Ssa and Ssb.;citation_volume=280;citation_year=2005;citation_pmid=16221677;citation_doi=10.1074/jbc.M503614200" />
<meta name="citation_reference" content="citation_journal_title=Arthritis Res. Ther.;citation_author=L. Song;citation_author=Y. Wang;citation_author=J. Zhang;citation_author=N. Song;citation_author=X. Xu;citation_title=The risks of cancer development in systemic lupus erythematosus (SLE) patients: a systematic review and meta-analysis.;citation_pages=270;citation_volume=20;citation_year=2018;citation_doi=10.1186/s13075-018-1760-3" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. C. Stirling;citation_author=M. S. Bloom;citation_author=T. Solanki-Patil;citation_author=S. Smith;citation_author=P. Sipahimalani;citation_title=The complete spectrum of yeast chromosome instability genes identifies candidate CIN cancer genes and functional roles for ASTRA complex components.;citation_pages=e1002057-e1002057;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21552543;citation_doi=10.1371/journal.pgen.1002057" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. D. Strome;citation_author=X. Wu;citation_author=M. Kimmel;citation_author=S. E. Plon;citation_title=Heterozygous Screen in Saccharomyces cerevisiae Identifies Dosage-Sensitive Genes That Affect Chromosome Stability.;citation_volume=178;citation_year=2008;citation_pmid=18245329;citation_doi=10.1534/genetics.107.084103" />
<meta name="citation_reference" content="citation_journal_title=Molecular medicine (Cambridge, Mass.);citation_journal_abbrev=Mol Med;citation_author=M. Sun;citation_author=L. Ma;citation_author=L. Xu;citation_author=J. Li;citation_author=W. Zhang;citation_title=A human novel gene DERPC on 16q22.1 inhibits prostate tumor cell growth and its expression is decreased in prostate and renal tumors.;citation_pages=655-663;citation_volume=8;citation_year=2002;citation_issue=10;citation_pmid=12477976;citation_doi=10.1007/BF03402176" />
<meta name="citation_reference" content="citation_journal_title=Cancer Prev. Res. (Phila.);citation_author=A. Taguchi;citation_author=J.-H. Rho;citation_author=Q. Yan;citation_author=Y. Zhang;citation_author=Y. Zhao;citation_title=MAPRE1 as a plasma biomarker for early-stage colorectal cancer and adenomas.;citation_volume=8;citation_year=2015;citation_pmid=26342024;citation_doi=10.1158/1940-6207.CAPR-15-0077" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=I. The;citation_author=S. Ruijtenberg;citation_author=B. P. Bouchet;citation_author=A. Cristobal;citation_author=M. B. W. Prinsen;citation_title=Rb and FZR1/Cdh1 determine CDK4/6-cyclin D requirement in C. elegans and human cancer cells.;citation_pages=5906;citation_volume=6;citation_year=2015;citation_pmid=25562820;citation_doi=10.1038/ncomms6906" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in oncology;citation_journal_abbrev=Curr Opin Oncol;citation_author=S. Thiagalingam;citation_author=R. L. Foy;citation_author=K. Cheng;citation_author=H. J. Lee;citation_author=A. Thiagalingam;citation_title=Loss of heterozygosity as a predictor to map tumor suppressor genes in cancer: molecular basis of its occurrence.;citation_pages=65-72;citation_volume=14;citation_year=2002;citation_issue=1;citation_pmid=11790983;citation_doi=10.1097/00001622-200201000-00012" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=L. C. Trotman;citation_author=M. Niki;citation_author=Z. A. Dotan;citation_author=J. A. Koutcher;citation_author=A. D. Cristofano;citation_title=Pten Dose Dictates Cancer Progression in the Prostate.;citation_pages=e59;citation_volume=1;citation_year=2003;citation_pmid=14691534;citation_doi=10.1371/journal.pbio.0000059" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=R. A. Veitia;citation_title=Exploring the etiology of haploinsufficiency.;citation_pages=175-184;citation_volume=24;citation_year=2002;citation_issue=2;citation_pmid=11835282;citation_doi=10.1002/bies.10023" />
<meta name="citation_reference" content="citation_journal_title=Radiol. Oncol.;citation_author=T. Vladusic;citation_author=R. Hrascan;citation_author=N. Pecina-Slaus;citation_author=I. Vrhovac;citation_author=M. Gamulin;citation_title=Loss of heterozygosity of CDKN2A (p16INK4a) and RB1 tumor suppressor genes in testicular germ cell tumors.;citation_pages=168-173;citation_volume=44;citation_year=2010;citation_pmid=22933911;citation_doi=10.2478/v10019-010-0035-7" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=K. M. Yamada;citation_author=A. Hall;citation_title=Reproducibility and cell biology.;citation_volume=209;citation_year=2015;citation_pmid=25918221;citation_doi=10.1083/jcb.201503036" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. W. Y. Yuen;citation_author=C. D. Warren;citation_author=O. Chen;citation_author=T. Kwok;citation_author=P. Hieter;citation_title=Systematic genome instability screens in yeast and their potential relevance to cancer.;citation_volume=104;citation_year=2007;citation_pmid=17360454;citation_doi=10.1073/pnas.0610642104" />
<meta name="citation_reference" content="citation_journal_title=Oncotarget;citation_author=C. Wang;citation_author=J.-F. Chang;citation_author=H. Yan;citation_author=D.-L. Wang;citation_author=Y. Liu;citation_title=A conserved RAD6-MDM2 ubiquitin ligase machinery targets histone chaperone ASF1A in tumorigenesis.;citation_pages=29599-29613;citation_volume=6;citation_year=2015;citation_doi=10.18632/oncotarget.5011" />
<meta name="citation_reference" content="citation_journal_title=Microbial &amp; comparative genomics;citation_journal_abbrev=Microb Comp Genomics;citation_author=X. Zhang;citation_author=T. F. Smith;citation_title=Yeast &amp;quot;operons&amp;quot;.;citation_pages=133-140;citation_volume=3;citation_year=1998;citation_issue=2;citation_pmid=9697097;citation_doi=10.1089/omi.1.1998.3.133" />
<meta name="citation_reference" content="citation_journal_title=Mathematical biosciences;citation_journal_abbrev=Math Biosci;citation_author=Q. Zheng;citation_title=A note on plating efficiency in fluctuation experiments.;citation_pages=150-153;citation_volume=216;citation_year=2008;citation_issue=2;citation_pmid=18822300;citation_doi=10.1016/j.mbs.2008.09.002" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=Q. Zheng;citation_title=Comparing mutation rates under the Luria-Delbrück protocol.;citation_pages=351-359;citation_volume=144;citation_year=2016;citation_doi=10.1007/s10709-016-9904-3" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res.;citation_author=Q. Zheng;citation_title=Methods for comparing mutation rates using fluctuation assay data.;citation_pages=20-22;citation_volume=777;citation_year=2015;citation_pmid=25912079;citation_doi=10.1016/j.mrfmmm.2015.04.002" />
<meta name="citation_reference" content="citation_journal_title=Mathematical biosciences;citation_journal_abbrev=Math Biosci;citation_author=Q. Zheng;citation_title=Statistical and algorithmic methods for fluctuation analysis with SALVADOR as an implementation.;citation_pages=237-252;citation_volume=176;citation_year=2002;citation_issue=2;citation_pmid=11916511;citation_doi=10.1016/S0025-5564(02)00087-1" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Zhu;citation_author=D. Heinecke;citation_author=W. A. Mulla;citation_author=W. D. Bradford;citation_author=B. Rubinstein;citation_title=Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity.;citation_volume=5;citation_year=2015;citation_pmid=25823586;citation_doi=10.1534/g3.115.017913" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Single-Gene Deletions Contributing to Loss of Heterozygosity in Saccharomyces cerevisiae: Genome-Wide Screens and Reproducibility" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="twitter:description" content="Loss of heterozygosity (LOH) is a phenomenon commonly observed in cancers; the loss of chromosomal regions can be both causal and indicative of underlying genome instability. Yeast has long been used as a model organism to study genetic mechanisms difficult to study in mammalian cells. Studying gene deletions leading to increased LOH in yeast aids our understanding of the processes involved, and guides exploration into the etiology of LOH in cancers. Yet, before in-depth mechanistic studies can occur, candidate genes of interest must be identified. Utilizing the heterozygous Saccharomyces cerevisiae deletion collection (≈ 6500 strains), 217 genes whose disruption leads to increased LOH events at the endogenously heterozygous mating type locus were identified. Our investigation to refine this list of genes to candidates with the most definite impact on LOH includes: secondary testing for LOH impact at an additional locus, gene ontology analysis to determine common gene characteristics, and positional gene enrichment studies to identify chromosomal regions important in LOH events. Further, we conducted extensive comparisons of our data to screens with similar, but distinct methodologies, to further distinguish genes that are more likely to be true contributors to instability due to their reproducibility, and not just identified due to the stochastic nature of LOH. Finally, we selected nine candidate genes and quantitatively measured their impact on LOH as a benchmark for the impact of genes identified in our study. Our data add to the existing body of work and strengthen the evidence of single-gene knockdowns contributing to genome instability." />
<meta name="og-title" property="og:title" content="Single-Gene Deletions Contributing to Loss of Heterozygosity in Saccharomyces cerevisiae: Genome-Wide Screens and Reproducibility" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/9/2835" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Loss of heterozygosity (LOH) is a phenomenon commonly observed in cancers; the loss of chromosomal regions can be both causal and indicative of underlying genome instability. Yeast has long been used as a model organism to study genetic mechanisms difficult to study in mammalian cells. Studying gene deletions leading to increased LOH in yeast aids our understanding of the processes involved, and guides exploration into the etiology of LOH in cancers. Yet, before in-depth mechanistic studies can occur, candidate genes of interest must be identified. Utilizing the heterozygous Saccharomyces cerevisiae deletion collection (≈ 6500 strains), 217 genes whose disruption leads to increased LOH events at the endogenously heterozygous mating type locus were identified. Our investigation to refine this list of genes to candidates with the most definite impact on LOH includes: secondary testing for LOH impact at an additional locus, gene ontology analysis to determine common gene characteristics, and positional gene enrichment studies to identify chromosomal regions important in LOH events. Further, we conducted extensive comparisons of our data to screens with similar, but distinct methodologies, to further distinguish genes that are more likely to be true contributors to instability due to their reproducibility, and not just identified due to the stochastic nature of LOH. Finally, we selected nine candidate genes and quantitatively measured their impact on LOH as a benchmark for the impact of genes identified in our study. Our data add to the existing body of work and strengthen the evidence of single-gene knockdowns contributing to genome instability." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
    <title>Single-Gene Deletions Contributing to Loss of Heterozygosity in Saccharomyces cerevisiae: Genome-Wide Screens and Reproducibility | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Gub53aRLVuOG_y8gXlOKzRcqf24xl2mkiqovwdLDqhQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494056","apath":"\/ggg\/9\/9\/2835.atom","pisa":"ggg;9\/9\/2835","ac":{"\/ggg\/9\/9\/2835.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/9\/2835.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2835"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2835"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/9\/2835":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Hp2oKXs+Nli3lch6X1cQ2Q28pggbFU9qrg7AeAIqRTAnhFwNW6KThkAIBdfodjCUHwrzfOhuC3xoL3LYxqDdR4xBXmeW1KQU5Il2OHXCCh1YV5g1IvQBEmQUDCWiBVO0ns7UABtPDynFfTEsPRtveBDYQMBSpT4pvJvDfLjaL3BXRkaiYzLlA1b4RipnAT6PA4nQq1PdZiuODvfD7o1AG\\\\\\\/jufV3BnxwW9rR\\\\\\\/7Xpkqd5u\\\\\\\/f3U4tZxbuHXwxUEc4T97vfJ+IHPtaNHC2vCY9B4OzAk1a0cJOP3gpHaBXd3iU9e9Uo06Ovmm5LYL1B4PidQkIgvzw4TMcHioFlfSooT92idE8LS0UgvkC+QPAjuVZIyLZjr9ZZ\\\\\\\/Bz5GwKZU2aQ9NMRJu5ponlta4mQA6RKuzTb9gmgZkizjxzbWiX\\\\\\\/p5ywHVl4SPT0cuaMfWJaJfbQRKAhnX4rUESDfueZUVduxO6z8mLpEOVpz9N3bNJb00bhd3OTD\\\\\\\/rzeN9MQpIhWl76KJ7PLQt+IISiLFpjw1vEnDyGslGJG0weEanq78jg+B1tE1qdmlHIYTWEN9QmSur6Dy3YFVw31oegseo1sl2Pvsm1T42qTOlcTs8iYK0bF6UYJgHpI7DGUU6g1Y6A5Vz+U9lJlp0f+umVvzstWOUNke0GDpQWf4Y1qy1knqlkrMpnQy\\\\\\\/bcivB0naAhi1ENxPChmOixWn4GqHzQUFYIb6zMUoT4cZ413zClnMkMgjcI86IZaQ1YAfuNJBcgnniAqSRYv1VRVhNTzjaPnPE7PxZh0TEyUwSc5NX1nHYhL5LihWqacaFpReiyg2Oh1fMuburOXzeW9nIm6k21THOtqFxZABkGQaLWZoRvZpsrK5tybaDHGwDTESETs+taS8sszOducZM\\\\\\\/oUZYLN4MIAKabpdxI2zks5YNfDlP2OCQM9pc4p7WVvpcpwcdTmack\\\\\\\/INfN\\\\\\\/EEq+HE3EdzOiSkP\\\\\\\/c4Ad+9P56S0c2S7O4yApdfKD+Cus96MfleHK2E8G8aOnB77tyKdbqIvGIO04GIk+FGVlzpk6li8oV+pAv3k+5TBSYPvLt2STBgV4bPtkZ8i1u\\\\\\\/BwxYk6zyJ2pP3cn54Oo3Qwe9spSr1axOojaxll3GAbN7swfpzwHtBg8q5nZ3IBfO8EVEWQ8XyxiFGbVespqJtZ6OYgBt0dM9MwiI6zaegKD8WBlW+RwcUH7eMLjyyF7h89tLHAVxikyxiwoG5rVwnk5YffoRKFOmBwczeGgi4yBPBlMqv1+M5Cc2pnIBetePEZl\\\u0022,\\\u0022iv\\\u0022:\\\u0022fxFB1cElKduiSIA3RnfP2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226791268bd06c777c1030727a789aa81923e7855c35ae3d1a989940b5f346d86f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+7WxLVRoqpRk+lHn1Pm6gupgO49O7RrE+T5ptIgHHyuA6avrRI86fdwyPAPNqq+PX6PiNpNAqMYxOd5BMDMp6RQ4fgZHKPAk85IpcRr8VDbgbTJpgf8EYKPtItcax5uul2Cqo9prc5SOEcGQAh4Op+Ad9+bQNnd511aV+gAipj3aTKWGVxgt\\\\\\\/nMtuPGfU80WlGNE0ncNecoRLCZvhl8U1Ril7Brh2SYPiVqRZ\\\\\\\/lIeaVF1iJ1DDIWpRP8lqzn6uU5YhFg1GjUHjrq6F9aMm+w6vZBnj6Oe2EnB4NI8SAcx0\\\\\\\/O+2GyILyn2Wc9JWLOp6NxvwV\\\\\\\/76BfqlzFUA9YsJklQdKnoC3EVguMDVp40nn\\\\\\\/j8f6SGxdXyPq76EjNVqdk4RxPyMqG0m4i45CrzXK9GRBVQA\\\\\\\/hiKtM1lDDjlk5\\\\\\\/6mKSlQSOKixdXOASPo4Cd0Jvt01x7AWgZGx0r\\\\\\\/H0tVQ9J19sr\\\\\\\/AmkpwoE4ySCz6siZNjfrDY5W8EZlA41Huj0r8mJI1lZko64Er4fOhzlWX4BTqQSVcI6HTLexvrru123utRLrtSFBV+TfybauKAOGORoSguqJbhSqr6vbuGX\\\\\\\/Xr8JDEm6Mf+CJRU1CeSjucFx11btrmUSi66tIvVh57JJcPl\\\\\\\/7UYcLAnYDvhFTo9t\\\\\\\/uaUq6kKxLPGf6Q21bEIpwhLWol7ZsKKlpZrdgfDOyNm4yzmgUUXsN29JJGbbQjnbgiwE0I7kdJvTuEQ4nGGmWHjKct1Z0v0rNwYPndoPvQbkmOAjwXCOSDU8B3w1viaIwMahitsFx72+i49NbFc0Cn0X92By9w\\\\\\\/zJyK3yix5LMsCweZcUCv18pvxuYTiLtBE++c1CuHXEcCj9JONWvERGmU7NebJi9ZV7sHEAOPiFzVd\\\\\\\/Q10zmHHYnbd0SRzhb\\\\\\\/fnpWGW0d4jkYIVarqLKv1VobZVHAUJ1lZ2CUJE1Nc\\\\\\\/ec6DiOlNH9N2uwVlsAYmFiXBnXj59987r6NVL7ExnonStdDXPGf9dI306kjBWdU2oTlgufVCJRgne7nkR2ndOtATErrTAAuj48Uj5DdA+h3IO3HMsKxqS8A21yRXMrDjbiARkDQSwGWeDURRyHfs8IQCl\\\\\\\/pyeZ1w28C17l6w490W5Ki3QW\\\\\\\/Vc3NM+0Gv1zkhPyNz3H4tgNpMFJTrBMVVYbKtOh+ADh2xBRbA80YO6zkP92w6GjYCamx2aTeUblbgYJ\\\\\\\/w\\\\\\\/IgEcRqfKKH+5gtDsN7kwLm3SJay64wOqyiQ0nX6IM3PuAN9i3lyYora6+fWdVJUk8MCBwYgO\\\\\\\/Cvt9zieG9XH1Fs1qIqG8v9ezoAdcNqX\\\\\\\/VKmqxJ4FUBmwzud5nUsD0JjU1C18K84djBksWb7PMBy93rDqpy+Lqke87zU4CeDMkD6s2YUsmbRJcSbaHU7W+XouCGHiKJqmGOAYithV5UF63gZIBcYno1wdFjFDrynz7vu9C9AJcs8PK5EAsH2SGdxNJ84++V7uXguiZb6w0+9ZQg6CU1KMvjzIarvgNcV7jWZOEdWbPRylXZFlgFngo2xfIw9MizhzcFJsxwev\\\\\\\/M65qTMRdFShg+SwI+JB5RJ42FrdSxlMYSIEMexRe4Hm86mZmZ7c0OOZYmmXrnLgqs\\\\\\\/uzsmUdwJBq5vpyhJFagAX32pM4iUH7wgBfLqW7ooiQ5UfN0wAuG2MjzRHMBp96uEegG1vOgArlA1MRkilxBNk90fBoZ5LFsDxkoa3GYlgdGnVARsOrz6QX73WbHctW2h9bZ5uUscPIacctlZGUvIGVS7OZ+e5P7YYFe60u5ta9hfV9u0LO2De\\\\\\\/E7lgO+ea6iwmikXGa8Fwqz2oUhZC268J8fMlLxa5vhJ385iIZwdo5tYTfUKyZrs5ZueUAQj5N3kRGQe6B1o0U0MYQGbLhSOTnDt3n6T9Qr27WtIIss7KeXRqBqpGc9fOq+TzljzRKs86TYV6yrxSHImhz34XUCACtvhyVxUV0MKekIrShsEy\\\\\\\/zv\\\\\\\/VikD0ekISv7CsTClqk7w\\\\\\\/+VmjvEHYP1\\\\\\\/3MVglElDK8PPcKueHWDVMOACxYkpMOpZ4EP3mODSUvAJAcSGPc6pK0q+OrZI09EL2NwyFLUfXTvawD9EPFMxYHd6M48zfmGZDKy0rlBrkoYwwh8Tn+LKrMeKB79SA3ICrtqW9xQ4JfQO3UlBBZGoKMttVfG6NEHEhLUdbCI0U0V+lb6vTRxvoUnoh\\\\\\\/IENLJ6Q\\\\\\\/68XH1Sa4\\\\\\\/ukzix+QZfDKuQ8wKKO\\\\\\\/HCw0BjEDU0rVb3PGJVIV3DkF7VWBjogKjQUHX7mSN1G5E6EjHkQnP8ma6KQ8dYf6r33YcSfWQi9AeVoIVNPEhFGjy3gKVKfmAewgfbuJCwqg2LegI6XL\\\\\\\/xjQnPk5FKprjaMLaTQnRdRalXHkMmUkh0lrXP4j1eSKlQdR76ue9nVzZLSq755WUK5+HVCZpteqQfMZLrVPldPyhjF4m\\\\\\\/cIfiUmFLPP8hBuOxOpST9aZJVRETUb0Ohmujum0VMEONYbnHG\\\\\\\/RFZ90+G1ymkRP3HYGWWjSAzqSoGlRMrrh143ci1ui0eVZkoc76Qhqznp7vddTyL6\\\\\\\/W6o7EKJFhwT+pduKv\\\\\\\/nbQfFR3ElS5nrTTzDiQyRLhqgpu9xxCPNZHRWpYMJqcSpUlhBvu5ITDq47AbgotDGa4CUI5BCGA814P+7J5Rx6x7KehgwLOpcGeuOcXRzQtmE9c6rSSq8rgoPWUc0mM\\\\\\\/OKjadcP5xb7zoU\\\\\\\/xnQOvsgf6P3YHE6z+ec+lUPnonVaJr2Bn5869DHMfNJQ33gZqPx3hluk2KUb7Ge9gt5oBRud21or5EjObRmEYpdFWozE5\\\\\\\/23LnCnFy06HM0BTKd8FtrDHF67dwyivZDmJC9\\\\\\\/X+dI1IKE0SVFqiNiEELSJ1MvZjrlsfDn72D4zxQcmhOaXnaU4jGYomSVKAEHDEGbQAkHxjgTkgC3iKOJgIB86cdUGbMrK40+YD2T\\\\\\\/cQDRtBD6ShFWw19XVPsUQjJUJQsEzC6\\\\\\\/+3GRh02ZgXGw7y2l+mwQR1pNHdEkAXQ8lB4ZsG8nPs57wvdepcYyuOMQMzwZSuddFn3YGncrhD1pUMMTI\\\\\\\/V3y4crNNEpQ2Q4NG46pcq8HLsiFQrDvfn0mU8uHJ4TQFuCANtgn4XjkCziQosiEXsqsquCaqK09P6W15LsQ\\\\\\\/kU4HATeNg2b4fcWXLY0tf\\\\\\\/qr6aS2akpdPxhM8bqhi5VW8KUTAYRpbldAcveD1V6uPVLiwqBxSXir7xe1cERXlvzgy6cFdvfH\\\\\\\/Ff+O3KIGhs0elqnhNHbV7XlANOrv8mgHGSheiswmUDy0XGDaeYdZpP6P71OvPm9HmPdFIF\\\\\\\/nJLE0Lrl+nWvtnDcS4Jms3LFgq5hjmJbSNlBPAPKWnvfJsaaJjtGprDUbi0Vov37mIqcg8Xag28uEEHD4VoMbqXa3dmAVijVIUY0ssnIYAJ3sQ4womV4cE+Xo\\\\\\\/YNUjKcfb3V21IH3rJhXiCBeADBK+MSqj4HzCt3bh5nrEGsTp8HxdFY8EhtWEo\\\\\\\/AYsv5QaTHkEi\\\\\\\/OgISmgcNaD8YXk0ncH5+7O1LypUyyGvXYcrbxjXiKuKVd\\\\\\\/\\\\\\\/oiCj2IrSOyZJjmJ6R4f3H2DjBhPkPI8rZ+HPF\\\\\\\/j+eJ1zoN5kBx9276Wq+jXE0nNuWGVd\\\\\\\/\\\\\\\/iCReUOp7Ibs7gs89X\\\\\\\/VGT0l6GQVuA81vTROePtuYw2xEk6gIibSIPSV+tcntn0Sr\\\\\\\/GpVwHSah\\\\\\\/4GGEGO\\\\\\\/PSx9vU6njAu8x5wqN2J0eSoYfNn8skZtHmUedhlSoD+p6bo8712PTIMEyicpw9Bqi48U1QNinfyFI5xAxB0IXQopeKA0lx4kpAhA+vsaASsk8I43KoeHzDsecORxhoPaU0et\\\\\\\/MaB0IFwBFflmmDqn7gw0FVJ7o5SbNZ\\\\\\\/YvpkuHGnu9u8twXZQjkC3OuBLHxJ\\\\\\\/vO0lvrQiL5VhrZhYO\\\\\\\/ghb+9n9Lyy2LtJQ19ImnweS6Yr+g6nxmGDlNM1PgVsk9jz9zXoWpgDS3q5Q\\\\\\\/V2r4NXrVs3Jt++jVZLGw6niKem2YzgBSVXnBgxtOP0sNJpe3N6QrbY2dUJiM7rHPis8ncY6l1E\\\\\\\/qe0nv5FL2ERu5zYt5ZgtelaYJggyWcR+HwAOViPhNABRt50ZgHBgT\\\\\\\/1pPMMG5k+wT6xNh2w1PIC7t4gOaJM5bfbjabfxiOWCYT5pqUKaRS+DZpqOOwwHMa5gKxoOqRGy6bslMmPzz7926r9nxoF1Q\\\\\\\/dZksxZPfTOQ93NnCP9Rq9dzST+HTfJECK5p6zr7XBJqp8lkHTIWgqVqii9cVf3GgunEQyo\\\\\\\/Ek0bY\\\\\\\/eGrtzBfF3JACAKH3WJQpy7PW3AbP3YN5aQvqBngNrmIvNJO0ASNuGa3p2MJQjg5aQyeKLazsnmRiYfhc6N76\\\\\\\/hM9rzMD0UFDE9dQFVNIeWe3Gg3g9a2jrZPcWTz7qWWp+IX1pcd2L093pNwdjYDJYwfpFZ+t3KJevtXyZNUttB4v7hbteHkh0HzN9o6gZ7k7kMHdKTA6OorMS6dJyfAAh7iAiJdqWh1QL4Ptwz2mpfgA5iCNQ6VHWZx8iODtg3xJMB5um\\\\\\\/vegbMAuxwUakDSTL1haZAwmX8hBBFK8BLnm9vLg9L+x41pmIPG5MjTHibuQyCAmN\\\\\\\/FGUCJtRyS9lVmu\\\\\\\/5pXsNcbFBTe6Sy18pc+PWYzm77Ijd1utgJMJk058ZsoyOXNu9vK+8JBMU+A3WASgd\\\\\\\/\\\\\\\/9ObirqV6iyMR+cw\\\\\\\/R39mrwkWXQ\\\\\\\/MWLM5AycwmWekDcVIY1Fj2FmHSnQ\\\\\\\/up06Dq+JO+8T2qBBDFFhh\\\\\\\/0gN8VR1NFcN6SwWR831DASkZrq0rHWrDhubE52CDm45IGNn1bULr7k8VPL8NJnGAtbcdxl33RLV8s\\\\\\\/2tZibgZYu8pZVDbxLjquklgyUhIx9k9pUvB9+xRkmW7tj6doNCbFNr7n1B6LYKq0IvX9DkupoPmQPBA2JXzozXbk9qkOm5fpxgj4L41nTn3UZlrp+ZtzpIczXEMKbxhyYqUL60iN2SQVkICAYYFIl6TjzSUrQVS4xzPlGTSqp8Lb5hF2rsVyNSt5siaN8Hu2M4MWP1SGb3Gv3z807rwH86aRi3B\\\\\\\/aCG1ipQZBcljjRahVYy37bt2pM3HC6Ad0UETXwRByGM\\\\\\\/ipQffJ54x4FfIOZVVNU7JGyGPsKXgpQI+YqP1h3EOIyo9utY4V1z0pu3PEo\\\\\\\/RzIMSdP6JxqwBWAcPsVNKs0qGnMLWdn9TO+H4E6GWeUgTyS2zossQ+mzL2gX2U6\\\\\\\/43FvA4lHVubhUif\\\\\\\/2jX1oNah3hNQmz7M+bJFphgrampt21D9vAkQ6aNRZnvvrmjuihJfWgNwsrUXUWFAVHzlHM9pVg3QR5T\\\\\\\/vDgcKPYBegLNpP8XX7u3GQBxG7o\\\\\\\/EKvapjYb5l3KfQisnLXGZZI95VNWN4MQtpJN8GMcfRub7K2E0ogTtArQ\\\\\\\/Oehyo6AKVGRgM98160oAWZrScY03OZ68tbfyxnV4e11uHTzLPrk2fR\\\\\\\/71tp7DlbXJjc85oEWU+qXDtzT\\\\\\\/9qDP8R66XgtHTtT24Pkk96FtTI5GzrAOOh\\\\\\\/EhqgaSdOF2SQ43cmhI9cv3AsmySPDXi2Cv9cShN2AjMNalHTa9SyCPR5C\\\\\\\/Kov7A5VYlLh6g7g4nV+8x7rIcBTVCRcYyKyWdr4auil6em6nGrGRiLTo0rGgxpJtPBlbFGYiwrEksBb53SjAyJjlKCZFkaPq942pUC88xHSjSK2uIK18BsPdQBzVsDgDdroHMlcq0swmtPZ7yNHjmuEQWSDnYXRh9zinBva2hCj1FVJINxkoZMi7Zzrv35Xv+4K9fhquAoTqNNvC168R6NSJBDJ85MLcRXkATqMt8hoRGnPjPm1v\\\\\\\/PuLCn1s66m2omH8Uhjy5ntczmuz4v\\\\\\\/t70F1wdWKM\\\\\\\/uNXlqrGjLCD4tMKdjQORlAfSPZUGs0qDEixgH8jbzZp8mXQIO1oebApURPJBvdKKIQFD3ZamWvZy5yPqMC2LSVB+Zc2QIrJZFcoqbVM72ZO\\\\\\\/XxRyPD0eMCW0qWpL4aTD5h\\\\\\\/bTuj27j3FRnGWUyzFSa1EeY8+EXHMYMUYmSt7XLRfOqSEJOt9qIMyqt4LH0OGuEHEoJV2zBSlNYOmyr8XnyIK4h10LUiCvMMmFWSv1H6SVSa7avpb6\\\\\\\/cmTcKZTdj9W3WgVwjHs83aOtEt5\\\\\\\/ArFR42YmQbMwLUJKreEnHoA8smYGOC0oG1SMkUbghdAC3MxYRoCHr9C5a9JP1oDdAXgNH9ARHVSoN49F0YH1ZQeB259VLf+KDQd4hT+Aar3idrNd7npAfZrqTM4PNuJyfy3WZTaIIOOMT2yAJhmkgLytUUYqwWboEOcPCO5EeFrBvNqltMcGRtYaXQnOhOdlIo5bp8UbceIDtd3nhQ0NSprA3d8ljmzK6etYCy9D3yB7PczvlR9INAMnuUyi\\\\\\\/zZzP4a21XWxfU4BG3EUWkBiKgoRiRINgwsPq1MddedAP7o7OeZlKHtohLVFyOIZpe\\\\\\\/lvKJmbnCOb70Qnf+VbzoEPaNd4NmTwYu4jORMJ\\\\\\\/\\\\\\\/BGIb8HrQovlmJBRYv+xhA23c8Zg65ciw23QZeU06SHJYjlh1vY10uo63dND80FUah5uHeKEgyc8D8c+e3qte7\\\\\\\/f97c+4enIKYZGL0YqsaE8URE6Z4cFWT22UluXB3zJ+GiXFA3M8sXSIjQ6h\\\\\\\/2nF4\\\\\\\/2xw57AoH2TUDrNEc3Z2d3emLNxOz93Ggp7yH22wNQNLynxCALDH2Y+czixEbIw50q\\\\\\\/1JCDmmHHNN8pOzP9Nnw9qVXBCsNdrSTy2bRD4Qs3mjiqxy4P0WZPkccGpCAHjzSXF7Kc3l+dBdM3l+K3tDkitnKWkM1cOnl15c353bIALYFE+hBhqXQJ+9gSMHPW7gEsxsVxvrBHqevg2M8WvsCTfmi7kmS\\\\\\\/acf4f4ncUmX8WrvP4CDMzcgbq9RwlgYiwzQJOC8gCDp9XzqZoIynSyZ8ouJi8exsgADO+eACbSJX371aQlvDpo32nT81mW+9hJRl8lgcIqTwTx4hZV263+Y1Q5pA64ObeUl5LkV2MVcw6iFRpLSZvnt6pOQmcxhgv3mX3jePRVexUL5FrPDKTttsjMgfBA\\\\\\\/c56TjQj833d5d7x4HwKrxBHewooni7CRVO21gcWX7DqcbhtcWCOlDpuxu2dm+sf8OtGpmMVGe4stpkzuznsjZMNQQsAaE0Zbdl5V32WCpKrvInKLE9Udo4EKCjueHNnbj\\\\\\\/lFv31xri20S2G4Bk114WhHxo+Mc75yxbCkXsY90gOUbftpHADmTeKNRfFDggxL9yqFoa9GdkxlBWhZBKCfH4UKY57hjn2vVU8MaubJ3DQPUM5+0TIqyGuwOWuq9Cu5v\\\\\\\/uqwKNeC8Nbsf7rmnH3ndheFL73Nbd2BxAeq46zbS+BS2\\\\\\\/Tszf+irNenDR1XethbsGrCKWyOxayghNFR1Ss+KxtXz\\\\\\\/Enkx5EDJWHDHuqP8gwy4LaTMP+FoEBd8vaPrQ5q0WN+1Ce2Ij5Ho+Vtoy46fwDavf2llYyV7yM9F4gbGyztZvKQ3ugfAm96IsKtDsStafQf\\\\\\\/J3+EViYuQer+KAFDXlIA9\\\\\\\/cum8yoQVA0TapiYBE+dTur3WHn4izBSyDH73ch+UcRRcMENiqQzJ7cpOLvW9sB1pGtWeSuu3hFug9RuOje8qA6wmUGIdVp3khy\\\\\\\/qLaHdJazdniQQaVehYpXstMkgnlXwjzjJOvTKtwXWJAtMm78jpHDySLtai8YqKQhbIyrLkUTPaXWO8LO\\\\\\\/Ue3Wx54KM9ICo5oCffBaqehnWvzOfgHl17kinXWdOSE7OZuLAwTCMZv8\\\\\\\/G8tj2efrdgfz9LVQYwYWQ4OemSN2q9qPU3aOiZ3X0IeBl4MPuphxoQB1xJfUeiFhUGFM9xjftKIMIAHu41uZbptxQBaGgfNQUSfXrs20neF8WF0UISbDnbPHC3Z8H3t2FAfGFfPm\\\\\\\/ourP24ZtuA8yrhIMVVqEVWQ1MeL\\\\\\\/EtZ0dR4Rdq92tyxNibdXju62fLgCVsdeXoVKM3DqTD01eAfzK7w5l06VpBqQ8y9b3KaUxZcuRFyUDAZmlQfXxwcgvvlGgyUQMPKmjoNw+baeoxw6l5gqUqFtNT\\\\\\\/1Qm0pD2owxhsbgqCtbGlsQPotu11N35S1rRAPivuIXO7ocpKP2IFtvVgIQwbZ9X+e\\\\\\\/GI2loW6WP\\\\\\\/cI5VkPsxrTXU628dfyLBKhtkUyQuPpPutADRB1vTTcHumdMXf3INHoyVS9i\\\\\\\/hEHP61ixUFx0dLrIZiYZ7uhf+Pna5R2kCzO5YQ6wntjX7kXoM5GegJNXN65KsCFoEgaytzvNG9zYYRFH5EgDduuilFkgUjh4DI1GrFlN5\\\\\\\/riM4at46YxKK4yAR4OrYfE8BoCWtWS6yySfMxb9oMDB390sVyxTWycF+fPOSUO\\\\\\\/vPzWKIOap9utetpKEUsKaXMfpGsaiWnO1SVRxyd3v0jUezptKMgZuVWUKGtOpTL1AVoZqqf8qyr6w\\\\\\\/0iGSHQ+MVnrypL5QomXXhiqEgIV6aEbOFA995Cnl76hsILU1wXsTKZh0megiWwuitQA1TCM5rkmmEiT5kuLX7McojsbnouA1X3s3o36rzvjqOBrFjwAUz1wG\\\\\\\/05qll6dTIDRNB1R8ouuE+G6EMRzDuOg8KfFLCgVVKOcCJKgHwPim8bX+Bo1yCmgu0JWZemR8gLG1tLy9KQHIbpR6\\\\\\\/zAXFha1T6wLfctjJgQTUF44+2MSgDgCBw+Qtfv1q0xMuLM1lR5rBEkO2jVVMuPfmVjD7dNZLd0QmZLX5kxgFqYPkWrWdhVcFDyKgLg5hbBg1whYAeJs9RBYDeSmEHYWpmRwaPqLarrOKNnUnQqoh54ZG5S9aMdcdlYSpWAhhaArBYSxzuP9iycPsjOk+EVvh+xtUeQ5XQHhvYHF6I4+VLNXuRK2giQkpjM3awM+croBRj15zLaQ9PJWZ\\\\\\\/kgykNeYLWl2gcGt0JxnbfWBGtLyNWNV088\\\\\\\/bYNNz+ZM3hhwUsS9x8Gd05A09znCvhCln0QoJT6MgqnzoZHu6XXA3i8\\\\\\\/IPucfB6TB0FiXDc2NBsQ8fuXrFhJIFJi11Unre5KJfLVhhLMCp357gFI6G3SV7YdOICgoxVEmtoXDeHAdZJIWen3k9iCRnfYitGNTjzSXybtOVHmVRe2O3tKtztGGtTp8hTylScAydaU\\\\\\\/kNxfhZ61TL4zAfIlPLkMevc3QDUD4LlUttcnyqu+PgEXFe5EdkcTO+L4PkPOOeLMtK4Tcdb9\\\\\\\/jvNPY5cQVBcfJaGS7O4iIPqoiF\\\\\\\/ndPmu9xnbMALYFPuSXSFOmxDxmt48a7qi+MTZvs+HUzG8CeSAdRWBRF\\\\\\\/eDX5hHTKPPwHYe6\\\\\\\/bjjpoIEBCBN4g2tjmQNBlfbI5Nbyfd5SXsSlIgq2NPSCmXHdCt6oMKuf9jChma5oklYWZLMPhAaqB1RN+5MUYaUdA\\\\\\\/bxz6FnPeySHWaN5gC4DJzVDGdhKxTcBxDwrPEXuU8UieY4gqAJSauiqeacl0BLRBReh6Ph6bWrcuWOEJMBuO0EXMC+jzuNkyAUolNy9\\\\\\\/MVH7jgIkSCA14lBWxshi9fuIq3PswHf+wni8lBKGHuXUTYtVLyqZ\\\\\\\/wgoTk9mE0MjsAv+EJVuldDPPtdNeYLj5l0UGU8Q\\\\\\\/F4eWedowl0rZuILbW4cF4T2gZ\\\\\\\/Fbg1GFi+WFZ12aMCn4YuMdYfd5BImjeKNnSOV18JX4h8C1w+uVdhLIsowxqQy5VmPLha6Nr4AXfgffriXc0yhr3pzBmKLV6Mpl+X0K69EBN172gUjCo0c+iyGDXR2pm256N8I5ZGDf7qBy0A60mhr2lMBIDl9\\\\\\\/BvgUcFqDzBTD\\\\\\\/q6NQioNtqv+dMtJ3S4RaTnN+yH3dqx0HXfc8EhBsBSHY0y0f3dPgLp6DBJuzosDnak8a\\\\\\\/3Z9z3N+E1f6Gy9yjWEZUKcniJph1fUmnhhi27Umoi0D5qc+8kB+MgDUpvAA7ToidcQFayxR\\\\\\\/tKGYXE9f47dFmxtYONoItekCJfG0\\\\\\\/L9wbLge8HiqzfdHtxxoLOS1kz1Emf78QXrSFjdkVvPIej+\\\\\\\/0D0lZSkx362bU\\\\\\\/GKCILBrCMpmvp6nsjQmFIU6xFOSyGtirnAS9notTaS2yFeUoA1p5gObb1FcjlRffG5BCk2bZFoQt5aPMU6\\\\\\\/R++YWEpqs2LgsEaceNLPrMemJszs0Q6IS6RgLqu+7tTR05zq9kJWY2BbiI\\\\\\\/mutcwKyUPzGPeZxzIqO3NT\\\\\\\/+NYl1uhy30zqUIFx9SGpHPMs3\\\\\\\/gcOb8sPCORVdUvN+WpnEZlPSWMKjy08UkYkYylw88+ifvxwqFlVr8mnl5OGpLMcAvzzFUZacGtJZ0lSFS6CjGDODMecwKQQSSNCG41zXUA4uRYCtfZb2DMdtONPKEAgTO\\\\\\\/MRkdqxqbJQTrxcKU6+WbJVaqp6EkawHmJRtGNEmYSwZVNH+drLvaU1Pda1MCe645sweLb6hJ96CCADMrWz\\\\\\\/VqYcsGifL7iaP0s1o6Oc8qyb+b5xwe8Iu9cpdn1Vts2beawXfeA7dkN0Rp9ZhVKoHky+T49ZkIs8M+W+q+78tlwXTDEYsoes\\\\\\\/3yMyiSD6yBBaP3umYUvY1L5uWkwtmLKxzsHzu7vWgCxdvnXagBJlrIMGf8ThEJxQpSDOnyqIwdh67TfMGlrj99P\\\\\\\/ttUEX2ZH5eZYPGtUUsdc+CrSP8G0fJd5y1J8uicUUKcBILIOyqTPe212+NIZOrZuh\\\\\\\/4tlS8HvEBAXgTkKj57XpkYZFmRid6up2Ot9AaXmkKRDJK61PvwVOMYh2zawDGjNQa02Q3Puy9je6zEOdJVmfLb8Bt2WuRPhTADIZN5uo5ElClPw3nmmHAf1nQNe4oDiFAuTcw\\\\\\\/2EtiOZJIIdBR8njncq8P0FjRukp6kKudXg4GSSGUGMJVaA6stkuAoPUZ9JlZe70T9+N5TjBPCTg+j6heVy11MwjIZ7u\\\\\\\/SotYoAqhe00y8qHfjBDyDeuMoSjr8ds2aKHYp2nWy\\\\\\\/YxYrqFlT0d9KIMGWEmd+k87g15T6LlBhxVJmtc9QPFrYW9hUAz\\\\\\\/JBh4V5ierhMaqMq+z29mRzd8z8FE6UcWx0mDaHlupANQYcPUEnBo7ZfBWaPDipxJkf\\\\\\\/aaTeStgbXsFsuQyH6GjmRrC7bdeIIojzaVVo1WA9MzSGMW9IYBmcxv\\\\\\\/q+I1YuyfHlD27YQt6lrrYYHEnbpjml+rpShlwQCeqQFUPQe+UqjfUOMx+oDzMyU9MJJ8QRjej4zy0eeQTMJzH8jlX56LsthOfdoIjXJyxkUkM1tAis6yUU66pdp+TIlNuKMCzQ9rUEViRQoodjhNnj2F0L84lAFigOXi3K2ieiotVgU7GLsFqlipwOVThun3BtHhDsXjayrKUt+Mkx6+BBQOQ7FJAwFByaHsoTwQ3SH7OT46nVSJB0G045A8D438hPey9gjWNfRhse\\\\\\\/V3wU0yxqFb32tjV\\\\\\\/PkTEFKe3jbxRUGRvpFtYW4STjFJtZbnr6Op6CStlQvTk7UM2EgPx6ahL+3mXl755fDh+PzPGzqg0sxBCP4EP+Q20bOlIVA3qELopAgWgidv7\\\\\\\/AAbF++PO\\\\\\\/p34Jq9Pegvi39W8ORgpPPT0vWypeNWDVfVsD4nErUwJNSXtKAh8tUMs43unIfn9Ylhu4vj2iXjwtIloe7W7qyD1OsK97Yxb\\\\\\\/VS+dLmLXicW3hxAjSgoQh0RPkmWIkJkCI3mE1UjVANXZe5t705ULMscNx+9+Vi4bolv8ZIbgE54uO\\\\\\\/8p+ZhW7nNdg8iHJjmm1o4RqyF\\\\\\\/MO9V2qv17cwX+uyXW+CdNUhyHbCnt2uAxjpIT1PPh7cgtNTIAHDNwIzbtGySKW9KYMv+ARwe3EwlrwJQYLkaMM7BcYRyy85jlER801ibBIt6bkmrpVRWeq7YzI37rKwPd39hCemsI0wOc9w5uwWGZNFBIWyWyYctpWLJFFXxboMhF1k\\\\\\\/L8hclIQPjAXw7P6WhZBj+n8PNnuHp3l4CULdebVfWsltDS1ZRAE2XnMg\\\\\\\/ajnlGK\\\\\\\/w\\\\\\\/t9XoYZxNEdZzy5rsRqRk+6YJTsb3t9+Pn3KrHuR+Isey5cIedcrd\\\\\\\/VGdjB+gP6g\\\\\\\/a8UvVGHmx3mu2TimexLapwJ0aBu41gcGf7BfMHOLiB5UfahxzqbwIOsM95nYv8aaDUKcOZnZ093RmDkokFVoiZqIPpkgjmP9fl6Bb\\\\\\\/2XY4alIox1kuFMzU68AZUfPv06iQCiWhgwNLUlDTnJg1Njqf0zrVQs2CBuy2CY6a8gryZ4JCcDco2eZdGYUmP5hly+485BZ\\\\\\\/7PhhZnU2bGi5ktmipt4VT9Sz2JPlqS6Nf2q0S3JyOxXaVItCK3clN0Dg\\\\\\\/BGUyA8TP3aRahlh5zFqvNZybB2YBxdXbpn7A50dPAY4vRcB6GeOyWLaBkDDO\\\\\\\/aZ+53bq0ZDh\\\\\\\/Rv40N00EekBs1LXgpODYEQmMeL8dCwEXkBr6V00VuvZXbK2aszySGHhsHbz51KkwSJBWemnYWSq5pU7xwnOavzk7Hlikq1qv1TMyDwmDM\\\\\\\/+AzToE8ZdmHuvVklvg76JpevJgH+Vi2aSzzGuMMH5Hh+PwpvKoRHFhidPUzAf8cihgRXnraKTTHvsaExH3HZZp\\\\\\\/7fULKiv4lA8KILan7gS0dl+0rQ4vGJvv4s96ZJK+wthYcwpw0keJosgR8I7YtCZXxyrEiTmGQ1lBPAo5nHbheK25OBpXCcs8AtSSNLJLLo2CogaTCi1RJ17fOH+P+mLdDdVFHVcBYXwJZWcsUEK0LXICNZggATykjY2DDs3GYKDojljLcLmT66CQ4\\\\\\\/lBRxfvRd3eTu0SwZCJbV4Oh2ieZPFBhKxOQaonfYo51Y5iOcM0PPxOMNWeccj9LnLfudq2JGHfhG4CSKSq1k04jc7pWt82es0neENmRerpbFNqAUPL1qtr8HozKMe6uYTSaW7RjNw696jPGRPlmsSzPewvLLTm\\\\\\\/018sGHoirr0vlbtvYT5bo5q\\\\\\\/Xsyn22v0UTVLVI0QhKyff+KTKjx3VtgpPb2u2sE4IkOVGGBr3e\\\\\\\/sMuk3XbQrLoyXuvDlfU+WNOWkyc4\\\\\\\/HElPiYmRMW8BwYupsxzqhFWY6RurCSLYnx2zTrUzySrBDymR136ZPOgxe3uErdGQMZuOdqT2BsHCx46\\\\\\\/tBV49L4cZMo2Cs36drSaFArgjSxjJw9OfwN\\\\\\\/Q3eX+t7imFyySh8RMmIgp6xsF30d6p2OShS3ANv6eQUNPhnYLKAQQWmn7C5hmw7YVLvpfks+cSTnj0qgOOXi146ewSGgsP5Sg\\\\\\\/TMbNoJ9NQb8dOUUx4XoP98\\\\\\\/cQINFjZr0+amEXeCMGdLEfG59zbhf7Kyvx1Tn0K4OLzlpW6dQuPTs1KS67WNNPvzsCO0+F0DHmp9wLE7XOV29s9DiqblBmYFzDG6WHcSFkUDjiN+S20gikLSsjtPMKMBB0fBxlXcb96wki6+Z3I6a2p6jLVgsD0Szlf\\\\\\\/qvdgWgUKmjkvEL36lHAb5m8dnF9+vdcG453GPFewPuCw6JGInoBTAlhPdJToAiHl7+vEAshue9dP0T3+NtaEDuD3NFNvVQLXAnPYdkn7LE9EUEGr4dIlZI7xKywES75C9YNH+dAbQWC3c612GuWzBRLZpSQHypalW3cUTEI5UWsh1qMw4Il6tQxeuRbdk6PzZrjurnjz6jkyeHNQYlQWKgPzZSePBc11\\\\\\\/vUuQIWIb\\\\\\\/hZOQ0mWWJ8fTG+6axJ8sbGyC3IdBnF4B0uKxAqfgqS9JjiB0ZgiIKI0uJAddPliUoAMRnLfdQeJt5q6yehO2b+Hxm8idUQ6GgqTHUckk+Kn1o80S0l6EduclhoV7PofkqYD37vhEJbZV6a6difuzdfAVGdwCjqSzIEn7aLYmO4Q+T5EhfKcwlLDmhPzQuATC\\\\\\\/73UpEiegsRMNg6E\\\\\\\/DZd1faRZ7U3Y59klYf7iiyqt5P3+X2mhcZDHvnckNLlWv6siS5OCRfUddNsWDCfyDPQ5F+qjqdECmz4eWUxKKoEICdjmQOnYBvlyOGNw5Glbq4BPeu7bx3tRQsl1HTmkDikGzATXVGDL9JXzBqKj1BOdIi2N3stY\\\\\\\/dp0p5tXIa0pow2hDMHHpm2AtiajnYQ\\\\\\\/Edg3dYYQd0LvFmmA1zwvIaiJzWNIQlGDU2OSqAiD8fE8nudDTZGjTVmMz5pzmZ9uZeh2rRwha3FD2OA6IzbLfY0oxJoIpGEUofdwETIOXNLm20hYoT2CbwUZbAdE7x59znb8keImvZ\\\\\\\/ww+MIjOmplH1r+dfqip7K6cfa1D2gDqMnWqPj99fAEdGfREgalJkTo+ZIWZjcAoCp3\\\\\\\/MqY0hHnhd8bryCHCBf2XZOwnQoSljrVpiGQ2sGD0nxMShaV\\\\\\\/wM6CznVXUASBTfz4wi8HaED4Yl7Z6XK4umpBv48WotDXJ5c24jpUJE+nNvTaKlGdS6UFAb5LPOxejl8GrEuqO4ZcX17Dxa7pB4duOxxGthizXzbv6CDJsZcomJ7jWOKSZ16gm5pt3Pl61c6Ugu0JUFs8L3pJxo39ORNyw9kKBb38lpf63DULDjWanO02s7oa+NyZVORIPkDjNryzcbkMkuLpitIZ3jqqZa4hPXqJY6\\\\\\\/Q+0C\\\\\\\/t+2SwDPZDHOq4X7wlr04k0WWp+nSDcDwtMPiODDuixpn3\\\\\\\/Cn\\\\\\\/Pjvdq8u3xnXB8w5UHuyWenyIAGhIbT3EwPlqcxf+GZS3XxVFeQJGO3i1VZTF\\\\\\\/D3H8tgMnq6bMWMoLUbJHeoSvjhzpqD+sVVZHjWUZXAS3dEZKHwyGwZ2CQtXwdkcaZ6EGS7Dwn6YRIs6a0IuEP5GXRTetKZUQPsjH6O\\\\\\\/04ej4tX8tNGjShmb0S2pIT7seog8Urg1vvlUxQIeupFZLKO5uPf8Lmi2IbfbQpYuR7z2i2cVSdUVfszpvez5hAZIifYMQzlOO091uiCgkD\\\\\\\/NkQ45XDUW2tabgbF+BcFsHQMRquOX3Lx68kymF7r4ZIU4GyDo9nohxU1fDq6556IpvFHcH4d8ZuBdtXG\\\\\\\/n19pdzMdcr3s9qxmxGAUhvniSwSrsBZ4XIgB+X5AL62kyX5jz69VNispI46VvROYr\\\\\\\/OKFtuLp99xQIg79DxX+p9Cam2cvwR6IwU1+BYooQo+WlFUee1MJ86SrGoA4WdDmwDZAdExiBHacH6X0VuVXCdRgLYcJzzsWg6EMDUBqZAwhuLotDRWsO4VZm0HPgsNxDaq3S51yM2vdFH\\\\\\\/r9wxgQVdWpW3Zv+ycdjCc+OVeyYoU\\\\\\\/Ln8347z0uL2X+WjDLXqM7f5SIa21e+RSZ6ZBM8Cu9sBJmCWD68PqIKQiZ+BXAblJp8pB+uFr\\\\\\\/7HY795n75\\\\\\\/ns\\\\\\\/hp1\\\\\\\/ckEpnX7XXp+qo0RdzbgJ1EqV6SmGPc5lN24jcNuQdsINGgEq53UbGWmrfHi5tHjLW1IEQB9VUiDF7H1K5FnDV8VNG+8jUfTDCvIJ1l62sDaSO4a89Y6kSVMEFpwfLC5TrdvXAyWfg5TgRCk98gMH2OCG1G4IJy5pAso9RybDToAGVAAf6EAAEw1A\\\\\\\/qN\\\\\\\/BQD\\\\\\\/erNgLMfYqs+MeP1BMEZcFmAmChMV3o+ui533Nj+kKuJPeUl3WWsA7Cxd6ejWwZ5csKLHcqhmta6vg5u2e4MWoVaaQDnwgVapwWkiQH8QBA6Frife6zUFLMl2yRhlgsWh0CG\\\\\\\/ynQFgQGqUAW7\\\\\\\/9kMKDU2fanzTVirFSGXM72GtpCspZhWwe1oZrzSxTN4vFssNW99+abkHdweVbo+kBJumRMQ1XmZEV1Dt0lG+BEA\\\\\\\/8zeYmjiyRQ1ZKYQAXU27CoJugpimMI8pu2a5ktXHRVjddKFrikqWolp03fQ+JVzL+WxVpdQRTlJoho9dWQk\\\\\\\/ME4KHxd2rnaPBHNo1D1JQPTUIPxwUFlOn50XGixwIfy0n36zHri6KnqmAIQfxb0T9YIo2nzQVtIn+2wYIZ3spFisaXUz0pZCyYfrJHXY3FgM9kFZS15ouyHW1jtI52DREzUz7dINc5sLAAoxZbG08m9y7jHSIYQHMvZqCnW8Ovt+dbDPuxo4oJ02wCa5tO31E+iOIeTz8xAh5kLKbKw2wdU5kD8H9ULgIv9l\\\\\\\/AabPfUxQQGyTvOo18DwKVzkQV6bxY+j+SiLZGXXZtB8kJFpRbNkysP7mocodrUyZdKNsYnhY+I6kdmdxy75mNeFJYEAy5RRQz0xfQ6HZhQl7GAT9aLW6zN9d21C9jJ171OmsZor+Uc4pzxOeZKvVH\\\\\\\/8yYHTAtoWGUsOQbkyJyRGBKvJlOZZFWf74FBx+mKvSFifOxRNWkxfqjOj0gZ9SMRvtQJGbOF81qx1fq3jihufXuarOm9OCoJdNqap8D6j+phNVshJ+99TlMUJw0GXVOR\\\\\\\/DfuefB9CdF5pXBnM5qjdQWqzfI2QKSuuf9MAYOKvBgibdoCoqoTFBaw31qBSC1L8mpOGRpkdLK5bgVmZHiHj3QljXM3W92CJ\\\\\\\/VAZA6n1wJw3KsBY+gTKsVlusOW3aAksbtbIAEr+gaEAiEmyCBtdsJtZiLSSZQaFz+hWpYtgAwhuh\\\\\\\/2Ldv+20dnWi6RIVEatQdzVlVY5d4AHYjgZyQeCOKKwipaBu2fJ1kTN3UQ2t0e5WMRjoiF1Xp\\\\\\\/7uh8nLVwyn+p+AEC+7QhoUHTAtlOqQT\\\\\\\/yoleA7udaM1em2mINC\\\\\\\/rOE1bcLUrJ5yYLnjME+o8+jFIk5LkXQlqRNqC4a0XyCi7yBrQEsInjDbze2KwV4GXQ4E+LL366evEVLyHuQEWCssxi6SrlbMQVMztoQ+pyRVtI8ix3j7aWmVd+MVldxaOLGp1vGu2G\\\\\\\/WIxOxnxhQMwcdSsVlH92rYMK+\\\\\\\/GrovS+ES40i02y\\\\\\\/0uN09IXxDk2\\\\\\\/7ExDoMQfn1q+3UVMCFUGDhuyV5KUGn0JLwPISrNaLzFRteQnGjsTxp4GzoHDGTLTFyddQqyyKhYdHqQgBqV2MT+6J9BGYFoPJehB+16RkC+m\\\\\\\/OgmS+MpWZCousBh3DBJQSAQ6m7qmFn6Yf7mi+tW9+tw29jbRSLgvjMOzPqY507kLZCyChkRBtfB9Kqdj2Vqms36ZOz94IXiYyikHgc1AwmplNRX0dzc66VqtoGuXcr0qA45JlKT8u8j2qt6WUGMfUxIGXdr4MzR4loes1CO3zdHg0gqi3d7eWg3Y5MQ78acpSMMWBm7amEml2c\\\\\\\/rcfK78mN+XjImLe8U\\\\\\\/ixjFIcv8X3aZeQmO0ta8lUmHRvrH6gjgHUn0y8ROoNuXFnpBJYRRVrDhUV63eqvISqul+zGBYfmWGfuu6XdjLO+RWv+KbHNYA3vrb5mygNfXA40wP48oiR9sQbmguxrgsuR0zTt7fLGFHn7ms1rhu5g4Y61UuyWwDuMHnRQLjw7V8eBTOcgPTU6fFnnKpHqbVhxg+YFWI2fInfgjUJp5xaane5kHK371MVIcPDcGO4gkOMpKQiDqdux6P3LiFwd4yXy0XPunVnFcRh8gOj9QRBiYxYbimE0ZPdlyOUoKR8IoTUzXTJQtrmyu\\\\\\\/fRRFYeUa5eXbhCvPY8zzIwWVPsSD3vnI8YusrPpvg6xvoozKcZoR6ep0T6luSb2ZAxv6+3QxItm9VI\\\\\\\/N1kIU3W6nva8vU4CFy5HHezZo9+PRGxuwGBPUzMAeY70\\\\\\\/h0XCyXnBdKpoP3ZueDD57Ir2J+aXeD\\\\\\\/HGz\\\\\\\/c2FQNCLQ0UXYcu5Z3ZAvTi4IXyhMr4r4mQKwkZt\\\\\\\/AXgk+yKh2xmT7L4v2OqGwU6zuBaWj8Teblbhr06MVEZA8pJtOVKnnlc39hIw5hcG82EFPwEqEXD\\\\\\\/T+FguykMk5pPKvQqx2xm6IQWkBW7usw84LmzWAIrd4vDmfs4qWwUaVU2iMiJX4M6L8J5x9EtkyXuDdwKw4ohD1QDgeuVre63Pi3SPBiwYdG65UrZSW93OTh2u63OV2XecNEUBe05XFplrSkFwCDUFYevNEawYBChqpTsdAnBY9ZuQAlK6bcZeJrd9+sSxWFSZTe5fSjgw0GPzct6Cby6s4HADOC\\\\\\\/pnFfon8+2cTRtaX5rux0sWtA9pvXH\\\\\\\/YD71QOJxRz5YywwJJd3gFLaoypvvRmh6EwOGeRU+Io66FAH2MW5h84G3fZbZFOt2KqIS4ERH7pO4KAvq6rzQBX9oJqyPj\\\\\\\/y\\\\\\\/aXeYP6j2yLUoR5kp7I\\\\\\\/++WFwlvvwGpWjiR0SDH88fUuCZ76l6oFeYHCqekEGWyYuAC4tO5lZnTv2Cb519F8bKQmTCkrvgSARgDnUYqKdLV1E72\\\\\\\/ByCJtoWFlbwLN4PN2ZHl+gCbRPF2cCjC3DIYN30Ws9trae9pg7STYBy4XIwdCJ9DlB7My89Tlt0k1k2J7n5y6eppa2V1Oje9zxYbdmGFJyp7ttY\\\\\\\/1pMjeD+3Cit8GkDkbzmn1clgimHRnDy6ZXZ\\\\\\\/x86PFXR1USCF7XacF3QJZnz3isbxmmJKoMq3EN\\\\\\\/tU79nbMbpbFiiAy\\\\\\\/JsabIc+pwZ\\\\\\\/AEC9fqsqk6bIfpW2viIzBEgEbAaAzNwWG4PREkGkKcEMXovdvsDevO28mhL2He3EiO\\\\\\\/lz8saLo7yHVFqDV96h2Md5eIPu7sQcftNMU7ahRgmwbsstSwn4WMIKSax7Hxg1xbslvO6KDi4v95t4tSPBx6w6Ck560QYTxQTYUwSvdZ+Fo8hRHMa9YPNHBwlhi9q\\\\\\\/zA63Igez7Vo6i4yO0EeftFcnB1CqZxeoUg0m7OeNSnOssaefovIp\\\\\\\/FYUdySeKn7+7Qch2oAolKabMU8RnE2BZkc2CLY5bKEL6S0JP81VtX3S4qLp3e1IGkjIZS\\\\\\\/OWWmLbbpQ0ZD8XOODzR7NlHY6GoeIx3Hs1Abc6nAnZ1uOm8qzGtkEawb8zatr4Njks0x+noztATeSPmI7XsAe0h59CpmgO5GXqd8+0S5pc7xZHGNINUYtyOnauDZEvLqH\\\\\\\/16AAyie+j\\\\\\\/NtlyIlLLns7\\\\\\\/APawsjTMLJTO\\\\\\\/Fy94wTu50yHpx2tK4HMcHHgaoAhIX7LMOUWqE\\\\\\\/ZNAx8gSbL9hY\\\\\\\/2h8o72sbq\\\\\\\/nzzVsAgbbjlQ4sC2594w9kP2a+7UVXW484fWtbKQZjsvA7HmgxiQPjZOekHqOd3XaKd22nClz73MJcmKjalY8y3+4T0VBQGsnXYtaE8glV03DXE0YlaDm8mRIrVmxHll5k8AgxmKJOQ1+Ri1afxWPCoFbBEFTAln5diYhIhrgaXQI3prxaomqlVwiBu9tJ+YOmClZBX75J1LT8cJ8PmRGheKJdzLfWcNu4FqmGu+hvLy\\\\\\\/J4zqIZOKnK0\\\\\\\/gdVULgCPGh70vCHh3MQY9nkO9YG\\\\\\\/CuT2OWqe7SSwr61CKWU3RZ3nrJy0eIfF6BNx4G\\\\\\\/lFYdkHMtmv2Olb+CvUESInoZlK\\\\\\\/HJNuq7WSt+f6ejwFujh5U6yMEiZtNwu4+9chv8KKTx17KOlDRHpsSmBjMkgt7fY8lkZtWisLTBMroZDm37oibo4RiVEbZ67LpG28SfEocGl8sC8VkhKadtBrGDxDPMqzrsBF03SLrR82mVgcZgc4K96x6seUPPD9JJ26N3BaP972YA0BWc3jrrYblVDveoPOZZ9RVxnhf5AQc0UN1sE4WlChGlJWKG16Uu2w\\\\\\\/dYwLRi2CkjNu4A9OzOQBrCd2AND0LHHWAbetkqkUYI7+U1HG+GT1amqftLc2r+eSB2JMvfwT7KOTd3vdOSNI2mgZ+VEBItwTLXTWVlnr4M85\\\\\\\/FmKV9HROqW20SwbldCdA09Mj76jfO6akVI27keAFBrX1gpkBVJTEDWEJKxjXI\\\\\\\/CNR6K+aGqE6YCylNaqOfeUrU5bFB2uEFLp7VJMKvLpvP\\\\\\\/N6ubJAAQTxt8TOgldDc0NsAKWcY4DhvR7ai25rDMVsUeW5f0Ox3+27iQ5DYmEniIbSMhaRAL6J6qS7A91W3tpjBq1ZiGkY+LkTtcQn\\\\\\\/Yys3kQofB6XXd8chzWRX40sLKNGJ2DhJjVkQRw34SFE\\\\\\\/j6R4Ys8uXIx3rlZUFQWIVfEYyg6zDaGZFSy6lB6DQ0YzyQpy0lVZgxuvGEt9N8iRZUimvqaFhki8CLLFBnBlA3XZttVZYaG8cvgpn0i69ZQRkeKitIJTv7\\\\\\\/fGBE7hys0fDrr2O74U+G3bJMA+jP3ZOyO9NPPgZm+psiG5rynN9YSThoo7QrQWXX1X8NsEg0DKkxoWd6UGtswwfXc\\\\\\\/NJhfBRJ+8U0gpc0YzcsJX9SHJMFB1bN+5+gu2GAI4VWGlRlnkAbUM7pOBojlMBi5J0L\\\\\\\/TsQOqvoZgpGinOKD8TYT6HzAfNY2Uiv0+\\\\\\\/uK+OnJ3bVSHTmCEfCwnhYEVI9SkuUDYNreMBwOATLeglRzf\\\\\\\/1\\\\\\\/OSw6vX8LkeehvnKtVWybpZ0hlo8eSVtgEpUbdF38VugGGZmXEh29rkHCzPdzZkBADFJ6ejLQZUuE7I3hgxvcKBQIonmL43rh37BAJABGw5xxYoupkRaTTGtkfO57tAcRLzkodNr9rLolGAbUEdNdqCyOzzD3uZaZUduvC7FJsS9O6bPOIIQ4CwEbYoUIkRI74ZcKYKLrEHQYXUeIn68EcewXAsQk1w4zX4hg4Pmgk38L8DlAonS3xJYz52mUKrnvU7YMiF\\\\\\\/0MfdeRegUrS3r1MBv9p+CWTJbn6HbznN8ZAko4mOvY7CbSi+ze3jYpzELg1S57qGD8koz3hmeBhp58LvdiK\\\\\\\/1DaP\\\\\\\/43RynTPJMEWB3\\\\\\\/NsNELKxKoN4PfcAod2imskVQrB5vMEVVXJx5cjd4Qf69gYdN1Kno3OdB9\\\\\\\/ZohNb3bCuBHb+uiLyJeDXJ0ob2O0tBmK\\\\\\\/k83yuXXJL4OF4CodYufcvnwT+Ml0XaOxJ1Th7PLTulM47fNLflUrf4bD4wVdJcr12dAb47SAxdNPJTgj1AgeBAk2lG9D36A7PYCTxJe5HyVXtNMXKCX1Cg9gadoRXOlFq8RNsyCT+frKx1SQrkhJfTFlxh7Jsf1kgc8VUkjBMA8UwEMZnf+jUm\\\\\\\/rthv+4j03WRxMUVVI1gBBsGN3YTgo1BuN2Snf4UpAa18XE+Vuvch8EUGtDYS7k\\\\\\\/VZjouzAXxGpGjEBU5Z19fyjjmX4p\\\\\\\/3g6TGSHFkSXSYIGoVFp+nrNXOQI4UjJCXSpX2mxHZ+KTby29egd3rhuYXYa8y8ylLsq+47H8lUD1QAdYiDRBDoEpbZk2eiB23cZ\\\\\\\/M2s+F2PdZ0oHJeJ9VOQCEpLO5uOdzXR28LVG4ktrMfhGy2secVuGsnALcUzhqKvC7KpAjVTbGuU2TV0Qs549mxri09b3EIhWy1m4wnP\\\\\\\/oSsG1zqPm0kamH5CdIh0NFjx66ydp4pFcDdBBhEvqyY4DQoDjrGsYI3sWyXyF\\\\\\\/hO6oz\\\\\\\/v0QpcBu8gILFI2bvqd8GMlR2btIZxjeX1X10wfIzRkZxiggyEVQY87S0K+mRXEu08IcdUi35dEy2iPIDLHZDsMNjXeqEcVjbHMhcLd\\\\\\\/A2Yu+mjtnTphMqXtf4VgxIExQXNvjrXFN\\\\\\\/IpfNJ+GS+y+a9\\\\\\\/7yakIeaCFsiMqcKKKvGIxBLvlYhy1XP2SSIgVKZ0WkMCgWJAa6O7\\\\\\\/J28ajfrZgTWhKO\\\\\\\/Goo4c5RWK2zyQmibbKXnoXNRa6JHWCTj7+veicu+ChqE4npLMg\\\\\\\/Wh4MKH7Y7Y5mHI5Our7jczv4NEuuRXqBHFSpi\\\\\\\/zVupHFJtv3BSmYYNo8jEFS5zAW0UCH9og++CL22RO1KK0cmj4N9vu+ltKTB3vkO4P7roEcT76yUrBpVykfX6le+wBIUZp0HItX9SDiNhFojyOsQfE74GZljQM49Crrin520TxmQdk3HD3x418tp8uY4e0jJvaoT6F1bCNDJhIA79XTR5otQ06LOWSvp4KUArtubZfrqWVNRp0KZyRkCfZCChnkxJ\\\\\\\/uSJVpcoYeN6fRYqkW36ZljxSR5rW7jB6mXP+jqH+83+K2mNgGcH\\\\\\\/zrNAX8tMcUyfvzjKhoIb3Bion\\\\\\\/+lcfFa45OItD98rRI6pa7+5OGlrRnOSGZLhlhODZYfChFjlGC+Ar4xk26J1T0K4cad8cd2TwiY0EsLwGYUzbQtx1Vx9CRW3F0nGj6427VMlLgZcOs21LuaXtkowizTyjJJ2YKoKFX+kjWqRfnwLafoJQvSZn92bK8R+WQO8xzD\\\\\\\/QuQ2ABKEbVuuQKg66NW9\\\\\\\/6Ip1KyNYQfJ+JH\\\\\\\/LsTsH8I95Lv+Li6iQvJMJVGJqy40XSkNpkojZpVf9dsS+F\\\\\\\/EP+0V9E\\\\\\\/3d5OcJvlDk+FvEjUxZCiBJc69Mt2RklcEVJrqACmpPD6mQf2W0CJutK\\\\\\\/mBk7PSHtpP84us0JeqKuNHn+IE9eTBNfm0vcFfFMUAltB9OdPvczCb8UqhiCuYyzd1TG5LujMejeMgQ37CqeBBfseefiSml1kNnIfz2zJWb2UsIEpDCzd1V8r7BJG7b\\\\\\\/ul1R3CVXZ2y83icH56J37f4kSr1PXLn5rZlt7agsf+oeQMMMVGLoNNoVvSb9SV8e8JdcJiSXCyfZFkI+pxNQFrGRVovDwGbdVCDtS9r4aCUHivnFKBgyCryXXHqu8xrbDCoQrf90GOVLL0Q8FkpLVO3cbwiEqNC\\\\\\\/0XWwiOaPftqqmnGra6htpIIpGurpnwBvOoyHCpcArltrktFcQNIHlMmwqIjbSraHbQ6KCYMa3bMROimU26w093nT0bc7IVJ3chqnws9iAuse+Ayxb0pOUphop+qKX6o86aNpSbcwZQxt0DlypC30WMBR0ndTtjG95CGMWGPeSgRJxpJN\\\\\\\/03u51VXS\\\\\\\/pfOUx5oc7IxJ\\\\\\\/B3uumOWPv4Ft0IwuPsBax3sSKwg8WCwLrYy+l2rE5+yUiwkQSm5NqqyB\\\\\\\/l3LvOuF3896HRO2DQZRuWRzhx6NndFwhsCF2M4DLkpcNdC\\\\\\\/\\\\\\\/m+TQbNb0\\\\\\\/bW1spfnTamhUAR6ATEpzhCkpMpaeiWL6dnXe9bXo25M8b8lbWbXYDhqc6tWld5DHIW2V5v49pSKbD8gCkcAiyICAIEJFkFKy8XPJD4x00PQ5la8QgcjEwFAhK3hAAZk9Cd2fWSu0wmJ4hxnGyASZqI\\\\\\\/3C2HyqrFZEKkNtXlq+EkLC4QTcKXc9io9CogqZsOo2Lew0FydDZbk9uUcMMGlV1Gv04U3pkVprqiwM+rpVmg0XNtlseP7UN0R9SCU4B5VEle1In9dZ9SGjHzEJR3f5CLDE7tkAipYGBvSOZS0bqkFhtdWKhUYjhbWX9zOCAsiyV\\\\\\\/QIVBqzZhu5IMt2\\\\\\\/nI229ESw2G0PYNsNTlAv7qoMl4Yq+oJ8Kh8xRZfIfpC2Funx2ozE\\\\\\\/XYfvzIsL7xa8A42aVRnI8\\\\\\\/YXbH3Lresvv9BBJF0O4nNKTsklQGGoYPb8x4vB0IYfSjKWQ7fyXdo\\\\\\\/VKCWhjbGyce3WfIfPxvbO12Z1UKoMd4PW4T8hKX\\\\\\\/6TRfomJvsUNH\\\\\\\/xPX0tmju8YRbX1sI+fUIY3dYamq4Lqa18hHC2B+9GVqRyv0kbDwWCUh288HxsuQvouW6Xy4sGfSmITteXar78v3Q6YqcPraiN7dPs6NDPmDl5KiI87CY6QjP2GcFz10Daq+Qgj9aFws0nnDVYZYRZTWyBZ5dAkHKcRrQXwQ4dsrb5+vIj0Dcd4h8BIB6AL\\\\\\\/xm0prwU0GEHp2RdEiuuBfTmU60k4on+4HMvaVGrkCEvRsOMs3z2BfV4V03EKYNffuuEFIDoAGV3KyqoLCobJ05nS7VP2+DV\\\\\\\/A2+wR\\\\\\\/J1V2HZAmAgq+RJvq6BYsVVhWrntAU2SOkwwch1c7fmWtgbiakc3rHtVQLRwh6r2SadRAKQvm43J+16KSU4Z4VBLPPmDlkqbvnpOnD1aHnaRqOc7Qa4kkiB3NU4vu4v+rEcqKy0qQXJ1kuq8U5wS4JmQ8Qvv8MZgBJf5mIOQDXhCdQvj3aWDRoPD5v386p45c+xe6HQ2xdWz6P+3DhSkdmYOPVsaFEQlQoXzY14Dujm5JOywwB7BNC3OblUyAPucpbfd5meFFRUzhpgbSpw1mMeZlNREb5AlwdoiT\\\\\\\/g+oc\\\\\\\/e8NgSFtoJu7LJsyRrKx3t9S1+9IxGlAu0BslkitfytFQL\\\\\\\/aOH11JPNsSvUOJyb3qUsXDn06wdfVcjcYkyLB5omqTLACeSQUrnCxMpJaPLRa\\\\\\\/GIKid4UvfFqSJOebiKKT5zHBZLXBT7NQsGgjjAqoa08AwypjuBvbBwE0FwC07B+sfBdM0lcVy0uZdIz+si+hOo4XaiG5hbdqgqZEJT+yRvhhrLsTBowFZWGGlRlDvItPEY8rb0lLIgcR7We1kug5R+I9C5WvzvYyoPoMq7MCIfD6Zy6es88W2iDRkSGxeEoEHlmuiJe7t9bDh5fw7+PkAnQwbDFu9RgxUCZiYz0M59TIl2vyzvGVU+yIwh1LL14pI9A\\\\\\\/kvGC2TzILhSVL6p\\\\\\\/fGBZYdmAgM+L\\\\\\\/QnCOQwIOXvoE8TEMwDFWUo\\\\\\\/iu\\\\\\\/jqeBgFBQL6l7SJfHVwENL6vljwuejym2CI7HCnM+QW3cDYE2g2t75CqlCDUGd5Wxqx3ZgbUkff5ii7rBpPcyE74tb7f0gdZEvmbZ\\\\\\\/mcT6YUYyXb4ptR+smQupoSabojkKSTb1iEmhL1SBaJjhqIpXyOt4bcvch575Ca6oCZvewKha67CA5rDmZPOTKAXZ0QyYUDcVt\\\\\\\/FTBe9OUL\\\\\\\/3NxkQpl5M9eS9tMfaC+DPJ17aNr+szXcm1FX5rmTWADiZh+tl1S8Ui0FCEH+PXFsN2QiaGFaVw2KQBhdEeb0TyXdKT39yacg6XS1xJi0nGC9sdU2\\\\\\\/duNOOcafOWXlanAF0Q3vyOoizHmeeEvgj8rgGdE3mtlfQ0R1+8QC5uArUkUqpsqQGH5G9BRT0EEpphgbwTgx7cwERkN+j3MGldpnboOBdaqz6vHcrnUoeR+WbpV1mR9gO9jJrCcgOLnIPqpyMWHQYjEQr\\\\\\\/FqmdYQni6Z6MW79D6Z8F9TXdCCwL+2fbg+OrrYd+dBfChNtQ2Mz1ptPt1EDCf\\\\\\\/vWTPaYlIrQPus6fFi2qMUTVie68agL7XN8R1ZN6Sf1GX0aeY0CCfc\\\\\\\/sC+UDs7vTpaF9yzupQ2jmqKZxd0BK4mFz1mOAt0eRBHsLm0vyUfavTVATuL2SbIM\\\\\\\/MDo0R6rrZK3sVPcHYY+TrppzlMde8bIk5+1\\\\\\\/OuJjJGdMSy4mHj9Tsqnepgvo32o8YqUFddVcbpXekSEf2bexxDG+TTBG90AGbeooi5dnPxdrdzySYPYquTt95yDwTi8+oPVk1I1FxEwRjModKCbWNT5vH8nzeavbQqLNomKJQcIZ6a9HQr2K9QfRdLKWiLnzM3s9ReOpPoVJuylORpyHC0H1YMcEO972ahp8XIy7mZK7KYVPcZfSV+xNbKMO0cixORtG1w\\\\\\\/LYdeFpYXb6k0S5GWUYZLp4oaoogUSCzccdxRBdyK\\\\\\\/wilr1WVacRG4SD9v1fegUumEmLWX0Ku9CbHa98bfJ9z3CUUFgTy\\\\\\\/QfO38aL5hvi4tPum4cCbCkA2uhoEqzjINk4eac1PcJLP3CzkvQoOoR3huoBBNUd\\\\\\\/GvAKFbd71BUV9hX2Eh2Q+oe9eFuKbnq7GM9H0qz42njasY0c+LhftlzgWv5ns\\\\\\\/fSGwmvd\\\\\\\/5ZChYe6C6sYudIoGTbyw\\\\\\\/rRLPgqySOBZ+yEOFjnDpIP+BvfDysx+P+p0Hs1yfPsMH3qguhNeS\\\\\\\/4TCtzJcfMewLrZ30cM9DGBwWL8Y55wfRYCiD3Uk1EORMfN8U\\\\\\\/SuGy2mLxzcJOMi+2VAg2cbKqizHhldEcb2t8Sq7DjnzO5\\\\\\\/wYHnmJGR4lsmCEYGGWYwGgg\\\\\\\/r+LN\\\\\\\/yBLBDUoCalqvRT9sl884a9pQcl73WQ3mvEtnno97XiOb+M\\\\\\\/TxB+2\\\\\\\/KupxjdqyQd8zVFpzXMxji2dTMeDtrZsBg+Ykb\\\\\\\/oH19\\\\\\\/vvb15tBNqCznxAqO+dqUVm15OPj2gJTv39DBDysC+qx18UNZUDOUrMxpN4lNJcVDxQhemPZyaoZrxUg4p8l1O8p8aZJ9Igd6RjrJfXAinERIpAtEF1g6Doh3y\\\\\\\/y0dBisbdjpk9p8WYj9DWcaIT9JmFJzBmBI2jk+r\\\\\\\/Om4MuPmeuemVl6HU1LjvmcPURLCbNngED6S2teTpM4L7+yNv7OziYFzRxrkGv\\\\\\\/VeKWtMYU3IihPXc7gm\\\\\\\/FRYwyOWsmwaqNU4diDmhb40jJsFV8OJzTbDdSH84cAIT7dYN0gtZr72p9Dg5iieRCba2sRUcA3cmT5BATdMDLstnWbueeCpvt+6BbSHk5TqrB7eR8hFsD1cSDIz3Qm5gTgheaOoKHTFTVSY9TDe5U6yYRvREbUC1wwTg466IQXGw0zMTcw25Swo6duoY1NpZWx8a7PvN+aKzn6fKNLimC7sPDWTXR9M\\\\\\\/eRm36DW9Ti1Ludq64qqJKeQhmX265jqizndaK\\\\\\\/GsyOxgMWMKJbcuuXDQ1jAjZRZkzRHsw9BYZOp588oIy9t5HDjNtjTuCUG7FWOO0A07lApP\\\\\\\/a9cpVYFIUsSecKnt2u26t8ZyDTfgTSFyjo5HBNbO2\\\\\\\/PwVILwz3lgTjASUsMYx8sDkrzOFFtoKFSSCDuxrX\\\\\\\/eVf5SdEqyIYLmT2ZrGksTaoD6bkc8JsD66UeLYyuv7b\\\\\\\/8ZWZhTgxbrkaahM8jM00W9l0cKq\\\\\\\/WPO+I8T0EKGYuk0lVoMVzoqNjBToIyk2MfwAo4zn1LTxjBCULNZxWyINulsD04M0CcbAmEVp6xs3WXz6NP1UlPdRfaqe9ymJrdeieC9YDa4DGkYjCcSQta7xu+JWKpmwtcAvsnMWN+tKQ2okZgobUzofZbW5SstNcxypoBzuEKsAAov71TMmrDiqA2x1crhkxhlxRY5yGzc7M+ixwJOD50Fr7RdBqj\\\\\\\/3NGGz+lZvhXpC8EsZvUoqSEDuhLvQVyz05tGhrgkgTBVtIW9OvPGp98I8BSQKCZiwosY96bT\\\\\\\/6VDNImUthhwMG3cjVCoSBv\\\\\\\/Uy+q7xVgsGosKj6IuU8kSZT3vLMd\\\\\\\/iFpGrUYL7XX+3U22lmIV9f1ValTSQTHLy047djp1aFBKCKUHbhFjuVPjQloIDntwfaJDlZ9wV1jNMfHdLYrz1JrbXBjlZKKe+m24CrpfnrtihBfQQ5aMArZ2JT7kEp0d0j1l3KR9GQMj9rh7oApFKxlDenqizJHRHsbcuLtCJ198FRajhICKBngYRJmCXpAcC\\\\\\\/cDNGpNlTJWud3Nt4YZDm\\\\\\\/ppMJUz1dtU1nul7z0RtVZ\\\\\\\/PpzyBNTzonwOJyivPs3mFPVaztWUfNzyJ\\\\\\\/4YB4B5bwiuCYuEh\\\\\\\/pOdYTYYpTk3vQ70cySjYNtOmSfMQaMH78xTwRU2hPQWJI1gSO6KWkkZImHM0C\\\\\\\/bQJQLR6h0pJOA2zweYe7XNaLGJcOz8SOh4SpW6Or9JlTyJTMrA5b1+hbUnkkyNjee87BsjJ6nJ49n54KtdQYJFIpZOLDS4szvgoZDabEZh4EeLcxw7VOSwjH7VPZ+lTcmIqaQcncQVEtXPKg1OK8eoCMg5LKUSkTL+4BxQI2saPipLt2nb9VSShmcNrPIx5X9xipTrxMybjVpk\\\\\\\/bYwJdIQW4ICFKdmL+n3W82KSBGdvYA+ABsMn9tn3epyPlu1tFnbyDgtQg8fyzQoKlL\\\\\\\/HzMKyBPe1UAJeZxp\\\\\\\/a8fZCLJQzjseeacrkCRfDc\\\\\\\/tDJiZTYHxMfFV52206sgiEQw6nLLL8GMbAfYp+dG7GQFaRvJ95HrPO8DKOEbWxnE58h8\\\\\\\/qUoOm5kMD9S+Pg9YDhLshm2UK6dl\\\\\\\/YAAMS82KfabmvKSftcQon\\\\\\\/3ZPFcWt8JRifW79YAjA2+66c0tBQWibOiEk1cDycgGZDrHouda39rItNns2k0WIbLQuM2BYlEDOQ4P1OtIF9FnE0BR6S8+QJz\\\\\\\/tARx3+\\\\\\\/1h+lABihPG9yRRwmnTwnBR9lOOdLItty6sLKudMZiKT51dYWqvuCLJWpM72y+9LwvwfXez0oMTkRUzL4RiM1\\\\\\\/PzEGvRXYeNAcb9LcLQVwBE8c\\\\\\\/UobUFH95AW1XYADw5emR2kczfTu58NZNgmiD8kPBO4b6rrul4BF2o+qGtoEkGIfHaNB+9cJ5zGv+uClFYdDoDrdkATNXAlbdGhEUxw62\\\u0022,\\\u0022iv\\\u0022:\\\u00226JGUKxi2NN+5w7O7uPM3rw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221bbd1c3793894fecca1cdb196b97301f63c10ce744ee7e77400ea67efef32f37\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222mf5AxpRx2Ik+J61sD3RFvmkL6VE+bYV\\\\\\\/GWAcJ1ILovXZ\\\\\\\/j24o9dJTaSWP2WcqcnerCK6UsihVpNjRjrOxjwN8Nnh\\\\\\\/evQeV7OvoV6iDVjCbfgVfnjS82sl9tIwYnVaqyMvdIRNTaxjfIezwD02xh32kdJCjfOIz62xBsEjByJEyH7RUP41HKqCABCF6DZ\\\\\\\/j4yyjTsUmZxNSE2rz2EmnFGsR2W+ICvsG5\\\\\\\/fAIhw3X2gpgJdhJf7d5fFUCFT3WKXSDzNsrweJ8t+KOrOtznRkZgcX9HAjvaFr7K7TNVgVtlygkdk4D+gFvzQ3Tn+5nU8PODPPGiCsD1Tx8b4bSAm5JoYsbyUX66j4GRqFr9CfyG1MwZ7i5+A0wYHneQk4SX6yV4L+5ZSXVnmPKBWDIUAW6xjcz+ZTz1JNJiyNRy96ooxb49F3HxLHJh4XWI9Uv2m4TRyXTmmUTulNABWyShEtC9f+emifvoK2HTtsCAEQlj8GNhRXZbgnFWthGazeNOFv1LEmOaCre8F+\\\\\\\/LNyvznrHDQbnzrsv9Uzw0v9HadhLmw4ThCiwLgkfiPsV1jcmpqnM6sMzraI+Q4oZi5zVb5YXeworLNTaoG27cCFwuc7P3gRH3jq3J8IY1+bzg7qgLheAk89BFjdQlG\\\\\\\/O5jC5IwNQ+T\\\\\\\/JTPE45j+eQK0mBG\\\\\\\/BJszwolEftR7PLfO9TzdA\\\\\\\/3ijRoyxEl4GRi6a3wSdA6ysCtbHKZZxddCu0g5PTrGjQi8SRFXu4Yn68vmwK5neKFvGIpFZYYB4OP4CdU4k5zbYIhLS\\\\\\\/Eveax7ZdDYHu9GvJbTDKWv2dy+yU\\\\\\\/32T34dfR6B2YaONufO\\\\\\\/4\\\\\\\/FhuqSYT049kJgSkuVQP3EobAR\\\\\\\/V6B6dvg\\\\\\\/cUZP5FH1LehDieccrA2TizpHKubKoroM8Wx1YpEYUSv7jZJc4tJRLBEhfQQTbbmrFRBEJCWQpQhdSmJopbi2yukGX5G36OlEXtSz6Ti+vfv8cCTpYdIAfwjfkAQX3rz0w1TEEBl0ZsjMszlCo9fOz9QeqdUUPNW\\\\\\\/\\\\\\\/ORDtb8jYocXo9iAhAfA8GkAaCm5opbM+zRIJEdjSAJ1kDCeqPo5KJ5gq21HmGcZHwJsLJInhgJa3m1RGqZI7+CpvQQKWygtFykhJ5Ua9H1M6doFTOq2SZKaPTIFkIhgifgO5BUQc+7AbdpT5bZ65wP9HHeg25efE7U+w2Y+bd4bJ5JeYMKnEEuEjzkZf1Xg7EzoNtXJtCuOVkiqrqFU9xwM5vF\\\\\\\/Ui89grxFzOXwOTXitz2z+lmdlYaiUUavlROXBmac8T6NXyTsw+qZ+VXcFl3YGZYtKBcIR59XEKCKUyRdA5PcRtZ2xac7oJouza3UVvSJs3cqXNAzeuZBwb1E4aj888nf75hc2AT+I4aKLnll8yNOcsq31UlWDYNLAKOGhH6vods3a+F3pJTkJBqFehy8AGhjJAtm9yQeuMSrlAwI\\\\\\\/K1zTQRxPN3nOlkP0UAYz2AgRMg479UtrHpOcWZMhEZpx6i1R8q9+djNnFZAZyjCxUp+Ckp01Q+VWH9PoxsKXt6QUKmsHjxsRsYJ0Irb53yum3uo0jdjIaKsf+Rehrr6m1VOQ7qghgM9YUCxBiWKfLFz1yZp9JEdZVruh+8SMdWm9ahcnFDMwre9uF\\\\\\\/VyiK9Bpj2hleR\\\\\\\/qMkGJOcqZzmCWyXXbv\\\\\\\/eJD1byIIYNG196CZOLGsMj1aIUsyzC61WxINP66PHxXFQx239HQ76D9mUA9xvbfOPdHR7PRKzsBmNFE33mf4bAkk50sq69zc15Rwg5ZjJyXNMkWGLsRFk2cPucMH2+WyMQWWSzuiPQk1glXOSC08A15Cb9p5rx+JLdXbEpLGjJHgfoVIjaldtotuML3nFkMU04FBhfMFTOgn8vy+Ck5gkQYTqUFjLB4q8kADMQVjCeuyT83GbQ+GObCj8j+48BYG6UU92RoSG31hsWMLO+aMLne0MLxNjUOoF9fTw2GW+RWbTq0ZyK6UALqF\\\\\\\/ipwnybPRRtMbSRYlNtoQpwOkf33R1B5U4Zw\\\\\\\/aiAhTsDi51D9AR\\\u0022,\\\u0022iv\\\u0022:\\\u0022U\\\\\\\/fugUPG3mfLKMZJFEBJRw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002248be63956dc68272aa9feb8b9d3501898ae9c24915baf00b00b84f9faa4146ce\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494056 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2835" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jj1UpAoR9uoPAqYEk7AJx20FZbmfOWgV2r_YcwWdHAc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2835" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eMDanrQ7UGL8PFSSGEvxnoHcOtdalncG5A80mMamciQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2823" title="GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2851" title="Developing a High-Quality Linkage Map for the Atlantic Killifish Fundulus heteroclitus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494056" id="node494056" data-pisa="ggg;9/9/2835" data-pisa-master="ggg;g3.119.400429" data-apath="/ggg/9/9/2835.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Gene Deletions Contributing to Loss of Heterozygosity in <em>Saccharomyces cerevisiae</em>: Genome-Wide Screens and Reproducibility</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kellyn M. Hoffert</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6334-9619" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erin D. Strome</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2835-2850;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400429 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494056" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kellyn M. Hoffert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Northern Kentucky University, Highland Heights, Kentucky, 41099</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kellyn%2BM.%2BHoffert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffert%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKellyn%2BM.%2BHoffert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Erin D. Strome </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Northern Kentucky University, Highland Heights, Kentucky, 41099</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BD.%2BStrome%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strome%20ED&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErin%2BD.%2BStrome%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6334-9619" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erin D. Strome</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stromee1@nku.edu" class="" data-icon-position="" data-hide-link-title="0">stromee1@nku.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/9/2835" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494056" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494056/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2835.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494056" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494056/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2835.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494056" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494056/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/9/2835.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494056" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494056/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Loss of heterozygosity (LOH) is a phenomenon commonly observed in cancers; the loss of chromosomal regions can be both causal and indicative of underlying genome instability. Yeast has long been used as a model organism to study genetic mechanisms difficult to study in mammalian cells. Studying gene deletions leading to increased LOH in yeast aids our understanding of the processes involved, and guides exploration into the etiology of LOH in cancers. Yet, before in-depth mechanistic studies can occur, candidate genes of interest must be identified. Utilizing the heterozygous <em>Saccharomyces cerevisiae</em> deletion collection (≈ 6500 strains), 217 genes whose disruption leads to increased LOH events at the endogenously heterozygous mating type locus were identified. Our investigation to refine this list of genes to candidates with the most definite impact on LOH includes: secondary testing for LOH impact at an additional locus, gene ontology analysis to determine common gene characteristics, and positional gene enrichment studies to identify chromosomal regions important in LOH events. Further, we conducted extensive comparisons of our data to screens with similar, but distinct methodologies, to further distinguish genes that are more likely to be true contributors to instability due to their reproducibility, and not just identified due to the stochastic nature of LOH. Finally, we selected nine candidate genes and quantitatively measured their impact on LOH as a benchmark for the impact of genes identified in our study. Our data add to the existing body of work and strengthen the evidence of single-gene knockdowns contributing to genome instability.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/s-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-s-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">S. cerevisiae</em></a></li><li class="kwd"><a href="/keyword/loss-heterozygosity" class="hw-term hw-article-keyword hw-article-keyword-loss-of-heterozygosity" rel="nofollow">loss of heterozygosity</a></li><li class="kwd"><a href="/keyword/mat" class="hw-term hw-article-keyword hw-article-keyword-mat" rel="nofollow">MAT</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/met15" class="hw-term hw-article-keyword hw-article-keyword-met15" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">MET15</em></a></li><li class="kwd"><a href="/keyword/screen-comparison" class="hw-term hw-article-keyword hw-article-keyword-screen-comparison" rel="nofollow">screen comparison</a></li></ul><p id="p-3">Genome instability underlies a multitude of changes observed during tumorigenesis. The accumulation of mutations, both as drivers and as a result of tumor formation, give important insight into cancer progression (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Loeb <em>et al.</em> 2003</a>). For most genes conferring a protective effect against tumorigenesis, loss of function alterations need to occur in both alleles before the development of cancer phenotypes (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kacser and Burns 1981</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Knudson 1993</a>). However, some genes have been discovered to impart haploinsufficient effects whereby a mutation event in only one allele leads to an abnormal cellular phenotype (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Veitia 2002</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Trotman <em>et al.</em> 2003</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alimonti <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Berger <em>et al.</em> 2011</a>). In most cancers, the accumulation of mutations leading to tumorigenesis does not happen only at the single nucleotide level, but with large chromosomal gains or losses (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lengauer <em>et al.</em> 1998</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Loeb <em>et al.</em> 2003</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Giam and Rancati 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gao <em>et al.</em> 2016</a>). Loss of heterozygosity (LOH) events are one such type of genome instability implicated in tumorigenesis and can arise from a myriad of underlying mechanisms, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Knudson 1971</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Vladusic <em>et al.</em> 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Choi <em>et al.</em> 2018a</a>) for additional review: (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Levine 1993</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Thiagalingam <em>et al.</em> 2002</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Payne and Kemp 2005</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representative LOH events impacting the MAT locus that can result in diploids exhibiting haploid mating behavior. Due to the co-repressible nature of the MAT locus, both MATa and MAT&#x3B1; alleles must be present and active in order to suppress the production of mating pheromones and receptors, leading to the non-mating diploid phenotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485245804" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Representative LOH events impacting the MAT locus that can result in diploids exhibiting haploid mating behavior. Due to the co-repressible nature of the MAT locus, both &lt;em&gt;MATa&lt;/em&gt; and &lt;em&gt;MAT&#x3B1;&lt;/em&gt; alleles must be present and active in order to suppress the production of mating pheromones and receptors, leading to the non-mating diploid phenotype.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2835/F1.medium.gif" width="440" height="354"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/9/2835/F1.medium.gif" width="440" height="354"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494073" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-4" class="first-child">Representative LOH events impacting the MAT locus that can result in diploids exhibiting haploid mating behavior. Due to the co-repressible nature of the MAT locus, both <em>MATa</em> and <em>MATα</em> alleles must be present and active in order to suppress the production of mating pheromones and receptors, leading to the non-mating diploid phenotype.</p><div class="sb-div caption-clear"></div></div></div><p id="p-5">Identifying driver mutations responsible for LOH events has proven a difficult task in mammalian cells; by the time tumors are large enough to be detected and examined, tens if not hundreds to thousands of mutations have occurred. Utilizing <em>Saccharomyces cerevisiae</em> for studying genomic instability mechanisms, including LOH, has long been an invaluable resource in furthering our understanding of molecular underpinnings (for review see (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Mager and Winderickx 2005</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Botstein and Fink 2011</a>)). Much of the knowledge gained through yeast can then be applied to higher eukaryotic organisms.</p><p id="p-6">With the creation of the <em>S. cerevisiae</em> gene knockout collections, large scale surveys for genes implicated in a particular phenotype can be systematically conducted (for review (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Giaever and Nislow 2014</a>)). The 6,477 strains included in the heterozygous deletion collection (hereafter referred to as SCDChet) target both essential and non-essential genes. In this screening, we use SCDChet to systematically screen for genes with a haploinsufficient effect resulting in increased LOH events. We first utilize the mating type (MAT) locus, on chromosome III, which is endogenously heterozygous, <em>MATa</em>/<em>MATα</em>, in diploid yeast cells. The non-mating diploid phenotype results from a co-repressive mechanism driven by the <em>MATa</em> and <em>MATα</em> alleles (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Duntze <em>et al.</em> 1970</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Herskowitz 1995</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Schmidlin <em>et al.</em> 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Haber 2012</a>). When an LOH event occurs at this locus, the co-repressible mechanism is no longer active to prevent mating, and the diploid cell mates as if it were haploid. As a secondary assay to confirm increases in LOH events, mutants identified as top-hits for LOH at the MAT locus were tested for increases in LOH at the separate <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> locus on chromosome XII. The SCDChet is heterozygous at <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000004294/overview">met15</a>Δ/<a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em>) and a color-identifying sectoring assay allows LOH at this locus to be readily measured (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Cost and Boeke 1996</a>).</p><p id="p-7">The MAT and <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> loci have been exploited as markers in other LOH screens interested in identifying heterozygous and/or homozygous knockouts that increase genome instability (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Yuen <em>et al.</em> 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Andersen <em>et al.</em> 2008</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Choy <em>et al.</em> 2013</a>). Our goals were therefore twofold: first to identify genes with haploinsufficiency impacts on genome stability, and second, in performing a variety of follow-up analyses, to identify those top-hits with the most potential to be cancer susceptibility genes. These follow-up analyses include gene ontology characterizations, positional gene enrichment identification, and multiple comparisons to other screens interested in identifying genes with impacts on genome stability, to determine the enrichment and reproducibility of gene identification. Thereby not only giving insight into candidate cancer susceptibility genes that have a haploinsufficient impact, but also deepening our understanding of the differences in results due to variations in screening set-ups and the stochastic nature of LOH events. By extensively documenting the methodology used, as well as comparing datasets across multiple screens, we aim to sort through much of the noise of screen results and avoid the far too common crisis of reproducibility (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yamada and Hall 2015</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Drucker 2016</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Creation of mating tester haploids</h3><p id="p-8">One <em>MATa</em> haploid <em>(MATa</em>: <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">his3</a>-∆200</em>, <em><a href="http://www.yeastgenome.org/locus/S000002414/overview">trp1</a>-∆1</em>, CF:[<em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>: <a href="http://www.yeastgenome.org/locus/S000002414/overview">TRP1</a> <a href="http://www.yeastgenome.org/locus/S000006779/overview">SUP11</a> <a href="http://www.yeastgenome.org/locus/S000006466/overview">CEN4</a> D8B], <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a>-3</em>, <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">lys2</a>-801</em>, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>-52</em>, <em><a href="http://www.yeastgenome.org/locus/S000005654/overview">ade2</a>-101)</em> and one <em>MATα</em> haploid (<em>MATα</em>: <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">his3</a>-∆200</em>, <em><a href="http://www.yeastgenome.org/locus/S000002414/overview">trp1</a>-∆1</em>, CF:[<em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>: <a href="http://www.yeastgenome.org/locus/S000002414/overview">TRP1</a> <a href="http://www.yeastgenome.org/locus/S000006779/overview">SUP11</a> <a href="http://www.yeastgenome.org/locus/S000006466/overview">CEN4</a> D8B], <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a>-3</em>, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>-52</em>, <em><a href="http://www.yeastgenome.org/locus/S000005654/overview">ade2</a>-101)</em> were transformed using lithium acetate transformation with a HIS3MX cassette, carrying the <em>Schizosaccharomyces pombe HIS5</em> gene, to replace the wildtype <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">CAN1</a></em> locus. This resulted in the production of haploid strains EDS585<em>a</em> (<em>MATa</em>: <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">his3</a>-∆200</em>, <em><a href="http://www.yeastgenome.org/locus/S000002414/overview">trp1</a>-∆1</em>, CF:[<em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>: <a href="http://www.yeastgenome.org/locus/S000002414/overview">TRP1</a> <a href="http://www.yeastgenome.org/locus/S000006779/overview">SUP11</a> <a href="http://www.yeastgenome.org/locus/S000006466/overview">CEN4</a> D8B], <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a>-3</em>, <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">lys2</a>-801</em>, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>-52</em>, <em><a href="http://www.yeastgenome.org/locus/S000005654/overview">ade2</a>-101</em>, <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">can1</a></em>::<em>HIS5)</em> and EDS588<em>α (MATα</em>: <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">his3</a>-∆200</em>, <em><a href="http://www.yeastgenome.org/locus/S000002414/overview">trp1</a>-∆1</em>, CF:[<em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>: <a href="http://www.yeastgenome.org/locus/S000002414/overview">TRP1</a> <a href="http://www.yeastgenome.org/locus/S000006779/overview">SUP11</a> <a href="http://www.yeastgenome.org/locus/S000006466/overview">CEN4</a> D8B], <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a>-3</em>, <a href="http://www.yeastgenome.org/locus/S000000319/overview">LYS2</a>, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>-52</em>, <em><a href="http://www.yeastgenome.org/locus/S000005654/overview">ade2</a>-101</em>, <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">can1</a></em>::<em>HIS5)</em> hereafter referred to as 585<em>a</em> and 588<em>α</em>.</p></div><div id="sec-3" class="subsection"><h3>Heterozygous deletion collection (SCDChet) screen for LOH at the MAT locus</h3><p id="p-9">The heterozygous deletion collection (6,477 strains) was purchased from Open Biosystems (YSC1055). Three initial copies of the collection were made (200 μL YPD + G418 (0.2 mg/mL)); 10 μL of cells were transferred to a 96-well plate, grown overnight at 30°, 60 µL of 67% glycerol was added and plates were kept were frozen at -80°. Haploid strains <em>585a</em> and <em>588α</em> were struck from freeze-down stocks and grown for 3 days. One colony from each strain was inoculated individually into 100 mL SC-His and grown overnight on a shaker. On the same day, deletion collection plates were thawed and new copies were made of each plate grown in 200 μL YPD + G418 (0.2 mg/mL) with 20 μL starting inoculum of deletion collection cells. These copies were incubated overnight at 30° to allow LOH events to occur. The deletion collection cells were mixed with either <em>585a</em> or <em>588α</em> haploid cells (200 μL YPD, 10 μL SCDChet cells, 10 μL haploid cells) and grown for 24 hr at 30° to allow mating events to occur. The cells were then spun down at 2,500 rpm for 4 min, the supernatant was removed, and the cells were washed with 200 μL ddH<sub>2</sub>O. The cells were again spun down at 2,500 rpm, the supernatant was removed, and the cells were resuspended in 200 μL ddH<sub>2</sub>O. Resuspended cells were pinned to SC-His + G418 (0.4 mg/mL) in duplicate and incubated for 48 hr at 30° before colony counts were taken. A scoring system was utilized to record colony count ranges. Pinned spots with no colonies were recorded as “0,” 1-9 colonies were scored as “+,” 10-19 as “++,” 20-29 as “+++,” and any colony counts above 30 were scored as “++++” (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2C</a>). Two complete trials for each haploid mater were conducted, utilizing different copies of the SCDChet collection for each trial.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Theory and set-up of LOH screen at the MAT locus. A) Example of a possible triploid cell formation mechanism that would allow for growth under double selection conditions. The LOH event could occur through a variety of mechanisms, all leading to the ability of the diploid cell to mate. B) Model of the MAT locus LOH screening methodology. C) The scoring system of colony counts utilized in the screening. Strains demonstrating a minimum score of &#x201C;+++&#x201D; in all four replicates for mating with a particular haploid mating type were included as a top-hit." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485245804" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Theory and set-up of LOH screen at the MAT locus. A) Example of a possible triploid cell formation mechanism that would allow for growth under double selection conditions. The LOH event could occur through a variety of mechanisms, all leading to the ability of the diploid cell to mate. B) Model of the MAT locus LOH screening methodology. C) The scoring system of colony counts utilized in the screening. Strains demonstrating a minimum score of &#x201C;+++&#x201D; in all four replicates for mating with a particular haploid mating type were included as a top-hit.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2835/F2.medium.gif" width="366" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/9/2835/F2.medium.gif" width="366" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494183" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-10" class="first-child">Theory and set-up of LOH screen at the MAT locus. A) Example of a possible triploid cell formation mechanism that would allow for growth under double selection conditions. The LOH event could occur through a variety of mechanisms, all leading to the ability of the diploid cell to mate. B) Model of the MAT locus LOH screening methodology. C) The scoring system of colony counts utilized in the screening. Strains demonstrating a minimum score of “+++” in all four replicates for mating with a particular haploid mating type were included as a top-hit.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">The methodology used in this screening was verified through a series of control experiments. <em>585a</em> and <em>588α</em> were put through the screening protocol as described above, but without SCDChet cells introduced to the samples. Haploid tester strains were grown in duplicate and 20 µL samples were placed into 96-well plates. None of the 192 wells per haploid pinned onto SC-His + G418 (0.4 mg/mL) contained colonies, indicating that false positives of the haploid testers growing alone on double selection plates are unlikely. To screen for false positives originating from SCDChet cells, five SCDChet plates were selected at random, and put through the screening protocol above without the addition of a haploid tester. Of the 413 unique strains pinned, 23 samples contained colonies. Fourteen of these strains were identified as producing lawn growth, not single colonies, interpreted as their starting with a genotype conferring an ability to grow on SC-His + G418 (0.4 mg/mL) without needing to mate with <em>585a</em> or <em>588α</em>. (When assayed again in the full screen, these 14 wells continued to show lawn growth when incubated with either the <em>585a</em> or <em>588α</em> strains.) Therefore, these wells were excluded as top-hits, as well as any other wells that showed lawn growth, after incubation with both <em>585a</em> and <em>588α</em>. The wells identified with this characteristic can be found in Table S1. Of the remaining nine wells that contained colonies, not one contained a background level of growth to score above a “++” on the scale established. This indicates a low level of false positives – approximately 2% - if classifying any amount of growth as a positive hit. This then informed our decision on a threshold for ‘top-hits’ from our screen (as discussed later), ensuring these types of events are not enough to warrant categorization as a ‘top-hit’ to mark a gene as one of interest. <em>585a</em> and <em>588α</em> were also mated with control haploids containing KANMX cassettes to determine false negative rates. Two different control <em>MATa</em> and two different control <em>MATα</em> strains, with KANMX cassettes inserted at different locations in the genome, were utilized. All conditions were kept the same as the screening described above, but instead of inoculating SCDChet cells, the KANMX-containing haploid controls were inoculated into 100mL YPD + G418 (0.2mg/mL). Opposite mating types were paired. All pairings, 32 samples in total, resulted in uncountable lawn growth or colony counts greater than thirty (++++ score). This verifies strain mating occurs in the incubation time allotted, scoring at the top level of growth, and giving no indication of false negatives.</p></div><div id="sec-4" class="subsection"><h3>Secondary screen for LOH at MET15</h3><p id="p-12">Methodology was adapted from previous screenings (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ono <em>et al.</em> 1991</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Cost and Boeke 1996</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Andersen <em>et al.</em> 2008</a>); briefly, the 217 top-hits identified in the MAT locus LOH assay were additionally evaluated to determine if LOH was also increased at a secondary locus. When cells lacking functional <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> are placed on plates containing lead (Pb<sup>2+</sup>), the excess sulfur precipitates as dark-colored lead (II) sulfide (PbS) (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Cost and Boeke 1996</a>). Cells must contain at least one functional copy of <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> to remain white in appearance; cells that have undergone LOH at <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> and no longer maintain a functional copy will appear as a black/brown sector. Two microliters of each SCDChet top-hit were inoculated into 200µL YPD and incubated for three days at 30° to allow LOH events to occur. Cells were then pinned to plates in triplicate containing 0.7mg/mL lead nitrate (for full plate recipe see Guide to Yeast Genetics and Molecular Cell Biology, Elsevier 2002), and allowed to grow at 30° for four days. Plates were then placed at 4° for 24 hr to aid in color development before sectors were counted. As with other published methodologies (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Ono <em>et al.</em> 1991</a>; <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Cost and Boeke 1996</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Andersen <em>et al.</em> 2008</a>), pinned samples grew as patches, not individual colonies, and sectors are counted as the total number of dark growth regions within the larger patch. The screen was run twice (two biological replicates); strains were scored as positive for LOH at <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> if an average of more than 13 sectors appeared in each replicate across the 6 replicates when analyzed at 24x magnification. Eight mutants (<em><a href="http://www.yeastgenome.org/locus/S000000071/overview">KIN3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET17</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001388/overview">STH1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005326/overview">MVD1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004949/overview">HRB1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005352/overview">BSC5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002235/overview">VAM6</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000001781/overview">YKR073C</a></em>) were unable to be analyzed for sectoring due to presenting as entirely brown colonies from the initial pinning to the lead-containing plate.</p><p id="p-13">The methodology used in this screening was verified through a series of control experiments. To determine the baseline rate for LOH at <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> for the SCDChet collection, we utilized two biological replicates of a plate chosen from the SCDChet collection that contains no strains identified as increasing LOH at MAT in our primary screening assay. In an attempt to ensure that LOH events that happened earlier, and thus resulted in a larger sector portion, were not unduly overrepresented in our results due to increased visibility, we utilized a dissecting microscope for analysis of sectors. Sectors were counted at 24x magnification and averaged across all 94 strains on the plate for 3 technical replicates. Analysis at 24x magnification revealed a high level of sectoring in these strains with the average at 11 sectors. Previous screens utilizing the <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> locus and analyzing sector appearance did so without magnification and set their threshold for classification as a hit at 2 sectors (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Andersen <em>et al.</em> 2008</a>). We therefore set our top-hits threshold level at +2 over the average seen in this control set of analyzed strains. The top-hit threshold was therefore set at having 13 or more sectors as indicating an increase in LOH over the baseline.</p><p id="p-14">It has been previously documented (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bae <em>et al.</em> 2017</a>) that the SCDChet strain that contains the knockout for <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">YLR303W</a>/<a href="http://www.yeastgenome.org/locus/S000004294/overview">MET17</a></em> (alias of <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em>) is genotyped as <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">met15</a>∆</em>::<em>kanMX/<a href="http://www.yeastgenome.org/locus/S000004294/overview">met15</a>∆</em>. Due to its lack of functioning <a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a>, all cells of this strain plated on Pb<sup>2+</sup>-containing plates should appear entirely black/brown. This phenotype was confirmed in all replicates (two biological, three technical) where this strain was subjected to the screening protocol. This strain serves as a positive control for this secondary screening.</p></div><div id="sec-5" class="subsection"><h3>Fluctuation analysis benchmarking of top candidate gene deletions</h3><p id="p-15">The parental strain used to create SCDChet was ordered from Open Biosystems (BY4743). This strain underwent lithium acetate transformation with a <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">can1</a></em>::<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> cassette in order to render the <a href="http://www.yeastgenome.org/locus/S000000789/overview">CAN1</a> locus heterozygous. Mutant strains containing the heterozygous knockouts of <em><a href="http://www.yeastgenome.org/locus/S000000004/overview">SSA1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003647/overview">CCT7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000566/overview">MRC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003874/overview">RSM7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000337/overview">HSL7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000373/overview">SSE2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000987/overview">PUG1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005891/overview">YOR364W</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000340/overview">MEC1</a></em> were grown from SCDChet and subjected to the same <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">can1</a></em>::<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> switch out via lithium acetate transformation. Strains were then struck for individual colonies and grown at 30° until they reached ∼3mm, allowing LOH events to occur. Twenty-four individual colonies per strain were then resuspended in water, assessed for optical density, and the 15 colonies with the most similar size, as read by absorbance at 562nm greater than 0.5, were used for analysis. The rate of LOH events resulting in a change to the CAN locus were determined by plating dilutions on non-selective (YPD for population size) and SC-Arg<sup>-</sup> plus canavanine (60 μg/mL) plates (for LOH events). Plates were grown for 3-5 days at 30°, followed by colony counting. Fluctuation analysis for LOH rate with 95% confidence intervals (CI) were calculated utilizing the R advanced calculation package Salvador (rSalvador) (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Zheng 2002</a>, <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">2008</a>, <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">2016</a>). The LOH rates and confidence intervals were measured for two biological replicates for each strain.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-16">Strains are available upon request. Table S1. Table of complete MAT locus LOH screen data. Table S2. Top hits identified in MAT locus LOH screen. Table S3. Sector counts of the 217 SCDChet gene deletions analyzed in secondary <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> screen. Table S4. GO Slim Mapper results for the 217 identified genes in the MAT locus screen. Table S5. GO Slim Mapper results for the 100 identified genes in the <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> secondary screen. Table S6. GO Slim Mapper and GO Term Finder results for the 91 genes identified in two or more related independent studies. Table S7. Positional Gene Enrichment analysis of the 217 MAT Top Hits, 100 MAT + <a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a> Top hits, and 91 genes identified in 2+ independent studies. Table S8. Human homologs and their associated cancer types for genes identified by the MAT locus screen. Table S9. Human homologs and their associated cancer types for genes identified by multiple independent studies. Table S10. Positional Gene Enrichment analysis of genes identified by similar independent studies. Figure S1. Visual representation of Positional Gene Enrichment data for enriched region on chromosome II. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.8325992">https://doi.org/10.25387/g3.8325992</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>MAT locus screen for genes with haploinsufficiency effects on LOH</h3><p id="p-17">To identify genes that, when heterozygously mutated, resulted in increased incidence of LOH at the <em>MAT</em> locus, the SCDChet collection was screened. SCDChet cells were grown in 96-well plates and paired separately with <em>MATa</em> and <em>MATα</em> haploid tester strains. When pinned onto double selection media, only cells that had mated and formed triploids grew (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">2B</a>). The entire collection was screened twice (biological replicates) with technical replicates performed each time. Furthermore, this was done with both <em>MATa</em> and <em>MATα</em> haploid maters resulting in 8 data points for each SCDChet strain. Each trial stamped on SC-His + G418 plates was counted independently for growth and assigned a score of 0, +, ++, +++, or ++++ (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). The scores of all four replicates were summed for an SCDChet strain with given haploid pairing; any combination of “+” scores adding to 12 or higher were further analyzed to be considered as a ‘top-hit’. For comparison, across the whole SCDChet collection, the average amount of growth seen on the double-selection plates scored as a single “+” for any one spot, and a summed plus score of three across the four replicates. Further, any pairing that resulted in a score of twelve, but contained a replicate that scored “++” or below was removed from top-hit consideration due to the possibility of it being a false positive. Strains that were annotated in the SCDChet database provided by Open Biosystems as having a phenotype related to mating were excluded from consideration as a top-hit, as their ability to mate did not represent an LOH event, but an inherent characteristic of that particular mutant. This screening mechanism resulted in 217 heterozygous gene mutations, approximately 3.4% of the genome, being scored as top-hits (Table S2). For listing of all scores of all strains from this study, see Table S1.</p></div><div id="sec-9" class="subsection"><h3>Secondary screen for haploinsufficiency effects on LOH at an alternate Locus (MET15)</h3><p id="p-18">To further understand the extent of the impact of these 217 top-hit heterozygous gene mutations on LOH events, these strains were screened for their LOH impacts at an alternate secondary locus; the <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> gene on chromosome XII. Each identified top-hit SCDChet strain was grown up in a 96-well plate for three days, pinned to lead-containing plates, then grown up into a patch over four days until color developed. When pinned onto plates containing Pb<sup>2+</sup>, cells within a patch that have undergone an LOH event will appear as black/brown sectors. All 217 top-hit SCDChet strains were tested twice (biological replicates) and stamped onto lead-containing plates in triplicate each time (technical replicates), resulting in 6 sector counts for each mutant strain. One hundred of our initial 217 top-hit heterozygous gene deletions induced increased LOH events at the secondary <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> locus. This indicates that ∼48% of our intiallially identified heterozygous gene mutaitons have large reproducible effects on LOH at multiple loci, representing 1.5% of the genome. Eight of the initial 217 strains were unable to have sectors counted for LOH at <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> due to presenting a brown phenotype throughout the entire patch. See Table S3 for <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> sectoring data for each tested strain.</p></div><div id="sec-10" class="subsection"><h3>Multiple screen comparison to identify reproduced results from independent studies assessing LOH and/or haploinsufficiency induced genome instability</h3><p id="p-19">To further expand our analysis of the reproducibility of gene identification, we compared our results to previously reported screens that asked similar questions about genes contributing to genome instability. Screens were selected for comparison based on a) utilizing heterozygous knockouts to screen for genome instability, b) screening homozygous knockouts specifically for <em>LOH</em>, or c) a combination of the two (for a visual representation of screen selection, see <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiple independent screen selection and gene lists comparison. A) Data sets were compared across screens that either assayed heterozygous deletions, assayed for LOH events, or both. Screens looking at haploinsufficiency and its connection to genome instability, or utilizing homozygous knockouts to understand LOH mechanisms were chosen as they provide the most relevant data sets for comparison. B) Gene lists from the six screens selected for comparison were mapped for their overlapping top-hits. Gene deletions identified in two independent studies are shown in burgundy, whereas genes appearing as hits in three independent studies are shown in pink. If a particular gene deletion reproduced in multiple screens within the same publication, the solid color was changed to a striped pattern. Only screens performed in a diploid system were considered when determining the number of screens a gene reproduced in." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485245804" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Multiple independent screen selection and gene lists comparison. A) Data sets were compared across screens that either assayed heterozygous deletions, assayed for LOH events, or both. Screens looking at haploinsufficiency and its connection to genome instability, or utilizing homozygous knockouts to understand LOH mechanisms were chosen as they provide the most relevant data sets for comparison. B) Gene lists from the six screens selected for comparison were mapped for their overlapping top-hits. Gene deletions identified in two independent studies are shown in burgundy, whereas genes appearing as hits in three independent studies are shown in pink. If a particular gene deletion reproduced in multiple screens within the same publication, the solid color was changed to a striped pattern. Only screens performed in a diploid system were considered when determining the number of screens a gene reproduced in.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2835/F3.medium.gif" width="387" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/9/2835/F3.medium.gif" width="387" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494083" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-20" class="first-child">Multiple independent screen selection and gene lists comparison. A) Data sets were compared across screens that either assayed heterozygous deletions, assayed for LOH events, or both. Screens looking at haploinsufficiency and its connection to genome instability, or utilizing homozygous knockouts to understand LOH mechanisms were chosen as they provide the most relevant data sets for comparison. B) Gene lists from the six screens selected for comparison were mapped for their overlapping top-hits. Gene deletions identified in two independent studies are shown in burgundy, whereas genes appearing as hits in three independent studies are shown in pink. If a particular gene deletion reproduced in multiple screens within the same publication, the solid color was changed to a striped pattern. Only screens performed in a diploid system were considered when determining the number of screens a gene reproduced in.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-11" class="subsection"><h4>Choy et al:</h4><p id="p-21">The greatest number of overlapping hits, 26, were seen when our results from the MAT locus screen were compared to a screen that most mirrored our own, having utilized the SCDChet to screen for LOH at the MAT locus, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>, (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Choy <em>et al.</em> 2013</a>). This overlap shows statistical significance at the level of <em>P</em> = 3.995 × 10<sup>−5</sup> when a hypergeometric probability was calculated using a normal approximation, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. (Ten of the overlapping genes, <em><a href="http://www.yeastgenome.org/locus/S000000355/overview">APD1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000373/overview">SSE2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000476/overview">HSM3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000589/overview">FRM2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002440/overview">MRH1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000915/overview">TMN3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000987/overview">PUG1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000006438/overview">RPL36B</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003647/overview">CCT7</a></em>, also reproduced in our secondary screen for increasing LOH at <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em>). This screen arrayed four samples from each well of the SCDChet onto solid YPD media with either a <em>MATa</em> or <em>MATα</em> haploid tester to allow for mating. The mating results for each pinned position were then summed to result in a score of 0-4 for each. The entire screen was performed in triplicate, strains that had a score ≥ 2 SD above the mean were included as top-hits.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494096/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494096/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Multiple Screen Comparison Gene Identification Overlap. A hypergeometric probability was calculated using a normal approximation using the webtool <a href="http://nemates.org/MA/progs/overlap_stats.html">http://nemates.org/MA/progs/overlap_stats.html</a>. The same tool was used to calculate a representation factor. A representation factor is calculated as the number of genes in common between two studies divided by the number of expected genes. The number of expected genes is estimated as the number of genes in the first study times the number of genes in the second study which is then divided by the total number of genes that were screened. Representation factors greater than 1 indicate more overlap than expected, representation factors less than 1 indicate less overlap than expected</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h4>Strome et al:</h4><p id="p-23">Previously an independent screen was conducted using random insertional mutagenesis generated heterozygous knockouts, to identify gene mutations involved in genome instability, specifically chromosome transmission fidelity (CTF) (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Strome <em>et al.</em> 2008</a>). Heterozygous knockouts containing a chromosome fragment (CF) allowed for visual identification of fragment loss due to insertional mutagenesis induced genome instability (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hieter <em>et al.</em> 1985</a>). The SCDChet was not used in this screening. Of the 164 hits identified, 4 of them (2.4% of their 164) overlap with genes we have identified for increasing MAT LOH, representing 1.8% of our dataset, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>. Two of the identified gene deletions – <em><a href="http://www.yeastgenome.org/locus/S000002498/overview">RLI1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000628/overview">BPH1</a></em> – also increased LOH at <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em>.</p></div><div id="sec-13" class="subsection"><h4>Andersen et al:</h4><p id="p-24">Screens that assayed for LOH events even if not utilizing heterozygous deletions can still provide valuable insight. This study utilized the homozygous deletion collection (SCDChom) and looked at LOH events at three different loci. Their primary screen utilized the intrinsic heterozygosity at the <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> locus in SCDChom to measure LOH. To further examine genes of interest identified in their initial screen, they constructed a strain with Many Heterozygous Markers (MHM) on chromosomes III, IV, and XII to understand the extent of the events at various loci. From the initial <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> screening in SCDChom, they identified 132 gene deletions that resulted in increased LOH. They were able to successfully recreate 114 of these knockouts in their MHM background, which they screened again for <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> loss at elevated rates. Of the 114 MHM knockouts, 61 of them again demonstrated elevated LOH at <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em>. These 61 knockout strains were then examined further for the extent of their LOH activity at two additional loci, <em><a href="http://www.yeastgenome.org/locus/S000002910/overview">SAM2</a></em> and MAT. Additional assays identified 26 of these genes with effects increasing LOH events across three independent loci. In comparing all genes identified as increasing LOH from their screening data, we observe three overlaps with our gene list, <em><a href="http://www.yeastgenome.org/locus/S000002171/overview">SLX5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001352/overview">ICE2</a></em>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>. A representation factor (Rf) calculated for this overlap indicates a value greater than 1, (Rf = 1.4), indicating more overlap than expected, although not at a p-value &lt;0.05, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Our secondary <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> screen also identified <em><a href="http://www.yeastgenome.org/locus/S000002171/overview">SLX5</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001352/overview">ICE2</a></em> knockdowns as contributing to increased <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> LOH. These three genes represent a 4.9% overlap in their dataset and 1.7% of our data (when corrected to remove essential genes not identifiable in their screening method).</p></div><div id="sec-14" class="subsection"><h4>Yuen et al:</h4><p id="p-25">Another screen looking at LOH events due to homozygous mutations once again utilized SCDChom but examined the presence of the bi-mater (BiM) phenotype (among other genome instability assays) (<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Yuen <em>et al.</em> 2007</a>). The BiM assay measures LOH events at the MAT locus that allow for mating with haploid testers of both mating types. When comparing top-hits between our screens, 5 genes overlap as having elevated levels of LOH, <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>. A representation factor (Rf) calculated for this overlap indicates a value greater than 1, (Rf = 1.2), indicating more overlap than expected, although not at a p-value &lt;0.05, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. This represents 2.8% of top-hit data from our screen (corrected to remove essential genes not identifiable with the SCDChom), and 4.1% of overlap from their list of identified genes that lead to a BiM phenotype. Two genes identified in both screens – <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">IML3</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005599/overview">SGO1</a></em> – reproduced in our secondary <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> screening.</p></div><div id="sec-15" class="subsection"><h4>Schmidlin et al:</h4><p id="p-26">A third screen utilizing SCDChom again analyzed mating capabilities with haploid testers to examine LOH (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Schmidlin <em>et al.</em> 2007</a>). Mating pairs of SCDChom cells and a haploid tester were pinned to plates of minimal medium that allowed for triploid selection; growth of four or more colonies in a sample was considered a positive hit. One hundred homozygous gene deletions were identified in this initial screen, and seven of those overlap with top-hits found in our screening, <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>. A representation factor (Rf) calculated for this overlap indicates a value greater than 1, (Rf = 2.0), indicating more overlap than expected, although at a p-value = 0.06, <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. Only including our non-essential hits, this represents 3.9% of our dataset, and 7% of the strains initially identified in their screening. Eighty-nine homozygous deletions were then remade by mating the corresponding deletion collection haploid strains to make new homozygous diploids in the same background. In assays to confirm the mating phenotype, six reconfirmed. One of the six genes they identified overlapped with one of our hits – <em><a href="http://www.yeastgenome.org/locus/S000001444/overview">IST3</a>. <a href="http://www.yeastgenome.org/locus/S000001444/overview">IST3</a></em>, as well as <em><a href="http://www.yeastgenome.org/locus/S000003874/overview">RSM7</a></em> (one of the genes identified in their initial screen), were identified in our secondary screen as increasing the rate of LOH at <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em>.</p><p id="p-27">While the stochastic nature of LOH events, as well as the differences in the instability phenotypes being assayed, contribute to the limited overlap between screens, genes that consistently appear as top-hits in screens interested in similar instability mechanisms provide interesting avenues for further investigation. <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a> shows the results of the comparison of each of these individual screens to each other and the 91 genes found minimally in two independent publications, seven of which were identified in three studies. Furthermore in the current climate of questions surrounding reproducibility we are pleased to see a significant level of overlap between our screen results and the Choy <em>et al.</em> screening for MAT locus LOH with SCDChet strains; approximately 11.9% of our top-hit genes were identified in their results and this represents 7.8% of their dataset.</p></div></div><div id="sec-16" class="subsection"><h3>Gene ontology analysis for identified enrichment categories</h3><p id="p-28">To achieve a primary understanding of the functions carried out by members of our first list of top-hits, we analyzed this 217-member <em>MAT</em> locus LOH gene list using two Gene Ontology (GO) tools: <em>Saccharomyces</em> Genome Database (SGD) GO Slim Mapper and GO Term Finder. GO Slim Mapper provides an overview of broader parent terms that a gene can be mapped to, selected by SGD curators, and does not automatically generate enrichment p-values based on its grouping of genes into categories. Fisher’s exact test, with the Benjamini-Hochberg correction for false discovery rates (FDR) (Q = 0.05) were selected and applied based on their documented use for ontology analysis (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Benjamini and Hochberg 1995</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sabatti <em>et al.</em> 2003</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rivals <em>et al.</em> 2007</a>). Significant results from SGD GO Slim Mapper (p-value &lt; 0.05, and <em>P</em> &lt; Benjamini-Hochburg critical value) are summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2A</a>. Four ontology varieties of SGD GO Slim Mapper were utilized: Cellular Component, Molecular Function, Biological Process, and Macromolecular Complex. Alternatively, SGD GO Term Finder selects the most granular term for each gene within a query, providing as detailed of an analysis as currently available based on the literature (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Christie <em>et al.</em> 2009</a>). GO Term Finder uses a binomial distribution to calculate p-values corrected for multiple comparison analysis, and there are three varieties of annotations: Cellular Component, Molecular Function, and Biological Process. Many of the categories found in GO Slim Mapper Macromolecular Complex are absorbed into the more specific GO Term Finder Cellular Component, but GO Term Finder and GO Slim Mapper analysis were kept separate due to the nature of the algorithms and the statistical analysis of each dataset.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494112/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494112/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Go Slim Mapper and Go Term Finder Results. A. 217 MAT Top Hits List Significantly enriched gene ontology categories identified with SGD Slim Mapper and SGD Term Finder tools are shown. SGD Term Finder reported multiple comparison corrected p-values &lt; 0.05 are shown. P-values from SGD Slim Mapper were corrected with the Benjamini-Hochberg critical value. For SGD Slim Mapper, all significantly enriched categories with a uncorrected p-value &lt; their Benjamini-Hochberg critical value (Q = 0.05) are shown. Gene Ontology Identification numbers (GOID) and names of the genes that represent the enrichment are included</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">The SGD Slim Mapper Gene Ontology tool reported two significantly enriched groups after FDR correction. For the Molecular Function ontology, <em>Molecular Function Unknown</em> (p<sub>adj</sub> = 4.997x10<sup>−4</sup>) was identified as enriched with 88 genes from the 217 top-hits list lacking specific information on their molecular function. Within the Biological Process ontology, <em>Chromosome Segregation</em> (p<sub>adj</sub> = 4.605x10<sup>−4</sup>) was found to be overrepresented with 17 genes classified in this group. To determine if a further understanding of the relationships between the top-hits identified in this screen could be found using more explicit ontology terms, GO Term Finder results were analyzed. The only ontology category that was found to be enriched through GO Term Finder was Cellular Component – <em>Chaperonin-containing T-complex</em> (CCT-complex) (p<sub>adj</sub> = 0.0489). Four genes that are part of this complex were identified in our screening, <em><a href="http://www.yeastgenome.org/locus/S000001404/overview">CCT2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003647/overview">CCT7</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003545/overview">CCT8</a></em>, three core subunits of the complex, as well as <em><a href="http://www.yeastgenome.org/locus/S000000004/overview">SSA1</a></em>, an ATPase that associates with the core subunits. A complete list of GO Slim Mapper annotations, adjusted p-values, and genes annotated to each term can be found in Table S4.</p><p id="p-30">With the goal of identifiying additional ontologies of interest, SGD GO Slim Mapper and GO Term Finder were also applied to the narrowed list of 100 genes identified as increasing LOH events at two loci. <em>Molecular Function Unknown</em> (<em>P</em> = 2.328x10<sup>−4</sup>) again reproduced as being significantly enriched in this dataset with 47 genes from 100 top-hit list represented, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2B</a>. For the list of all ontologies for the 100 gene list and adjusted p-values of representation see Table S5.</p><p id="p-31">To determine if multiple screen comparisons for repetition of gene identification was likely to lead to ontologies worthy of further pursuit, the 91 genes that were identified in at least two of the independent studies previously discussed, were also analyzed with SGD GO Slim Mapper and GO Term Finder to determine category enrichment. For GO Term Finder Biological Process, 82 categories were annotated as significant (<em>P</em> &lt; 0.05), the four categories with largest enrichment are <em>DNA Metabolic Process</em> (p<sub>adj</sub> = 4.63x10<sup>−13</sup>), <em>Cellular Response to DNA Damage Stimulus</em> (p<sub>adj</sub> = 9.12x10<sup>−15</sup>), <em>Cellular Response to Stress</em> (p<sub>adj</sub> = 3.75x10<sup>−13</sup>), and <em>DNA Repair</em> (p<sub>adj</sub> = 1.19x10<sup>−12</sup>). GO Slim Mapper identified 14 Biological Process categories (<em>P</em> &lt; 0.05), the four with most significant enrichment are <em>Organelle Fission</em> (p<sub>adj</sub> = 4.79x10<sup>−11</sup>), <em>Cellular Response to DNA Damage Stimulus</em> (p<sub>adj</sub> = 3.44x10<sup>−16</sup>), <em>Mitotic Cell Cycle</em> (p<sub>adj</sub> = 1.04x10<sup>−10</sup>), and <em>DNA Repair</em> (p<sub>adj</sub> = 1.30x10<sup>−14</sup>). Thirty-five categories were enriched for GO Term Finder Cellular Component (<em>P</em> &lt; 0.05); the four categories with largest enrichment are <em>Chromosome</em> (p<sub>adj</sub> = 3.25x10<sup>−12</sup>), <em>Chromosomal Part</em> (p<sub>adj</sub> = 4.23x10<sup>−12</sup>), <em>Nuclear Chromosome</em> (p<sub>adj</sub> = 6.29x10<sup>−8</sup>), and <em>Nucleus (</em>p<sub>adj</sub> = 2.41x10<sup>−8</sup>). GO Slim Mapper identified two Cellular Component categories (<em>P</em> &lt; 0.05), <em>Chromosome</em> (p<sub>adj</sub> = 3.42x10<sup>−9</sup>), and <em>Nucleus (</em>p<sub>adj</sub> = 4.16x10<sup>−9</sup>). <em>DNA-Dependent ATPase Activity</em> (p<sub>adj</sub> = 0.000228), <em>DNA Binding</em> (p<sub>adj</sub> = 0.00224), <em>G-quadruplex DNA Binding</em> (p<sub>adj</sub> = 0.00416) and <em>Exonuclease Activity</em> (p<sub>adj</sub> = 0.0290) were the significant categories identified using GO Term Finder Molecular Function. While GO Slim Mapper also picked up <em>DNA Binding</em> (p<sub>adj</sub> = 7.41x10-<sup>05</sup>) and <em>ATPase Activity</em> (p<sub>adj</sub> = 0.000808) within the Molecular Function ontology. GO Slim Mapper additionaly found 5 enriched groups in the Macromolecular Complex category (<em>P</em> &lt; 0.05), <em>Chromosome</em>, <em>Centromeric Region</em> (p<sub>adj</sub> = 1.16x10-<sup>06</sup>), <em>Kinetochore</em> (p<sub>adj</sub> = 0.000135), <em>SUMO-Targeted Ubiquitin Ligase Complex</em> (p<sub>adj</sub> = 0.00019), <em>Condensed Nuclear Chromosome Outer Kinetochore</em> (p<sub>adj</sub> = 0.00019) and <em>Condensed Nuclear Chromosome Kinetochore</em> (p<sub>adj</sub> = 0.000205). For a complete list of GO results from the 91 gene top-hit list of overlaps from the multiple screen comparison, see Table S6.</p></div><div id="sec-17" class="subsection"><h3>Analysis of positional gene enrichment for chromosomal regions of interest</h3><p id="p-32">To investigate if our screens identified any enriched chromosomal regions, which might be indicative of neighborhoods in the genome that contribute to LOH events, we mapped the location of each gene identified as a top-hit using Positional Gene Enrichment analysis (PGE) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">De Preter <em>et al.</em> 2008</a>). Genes, which when mutated, identified as top-hits causing increased LOH were dispersed throughout all 16 chromosomes. However, utilizing PGE to assess our different top-hit gene lists allowed us to identify locations with significant enrichment. Analysis of our MAT 217 gene top-hits list identified 41 total locations with significant enrichment, further refinement revealed seven clusters of genes, located on six different chromosomes, with three or more ORFs constituting the enrichment and with multiple comparison adjusted p-values &lt; 0.01 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). When we analyzed the narrowed list of 100 mutants that increase LOH at both assayed loci (hereafter referenced as the MAT + <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> dataset), 27 total locations were identified significant enrichment, further refinement revealed nine clusters of genes, located on six different chromosomes, with three or more ORFs constituting the enrichment and with multiple comparison adjusted p-values &lt; 0.01. Analysis of our multiple screen comparison 91 gene top-hits list identified 22 total locations with significant enrichment, further refinement revealed two clusters of genes, located on two different chromosomes, with three or more ORFs constituting the enrichment and with multiple comparison adjusted p-values &lt; 0.01. For a list of all enriched chromosomal locations, see Table S7.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Positional Gene Enrichment (PGE) analysis for enriched chromosome regions. Genes highlighted in green were identified by the MAT screen; genes highlighted in orange were identified by the MAT and MET15 screens; genes highlighted in burgundy were identified in two independent studies; genes highlighted in pink were identified by three independent studies. The key denoting color labels remains the same throughout all parts of the figure. A) Enriched regions of chromosome II from PGE analysis on MAT locus screen top-hits (padj = 2.74 &#xD7; 10&#x2212;4), MAT + MET15 top-hits (padj = 9.69 &#xD7; 10&#x2212;6), and Multiple Screen Comparison top-hits (six screens compared in Fig. 3), (BP region 442918-575991) (padj = P = 0.016). B) Enriched regions of chromosome V from PGE analysis on MAT locus screen top-hits (padj = 2.31 &#xD7; 10&#x2212;4), MAT + MET15 top-hits (padj = 9.69 &#xD7; 10&#x2212;6), and Multiple Screen Comparison top-hits (padj = 2.03x10&#x2212;5). C) Enriched regions of chromosome VII from PGE analysis on MAT locus screen top-hits (padj = 0.00358). No enriched regions of 3 or more identified genes were enriched on this chromosome when the MAT + MET15 or Multiple Screen Comparison datasets were analyzed. D) Enriched regions of chromosome IX from PGE analysis on MAT locus screen top-hits (padj = 4.40 &#xD7; 10&#x2212;4), MAT + MET15 top-hits (padj = 0.00194), and Multiple Screen Comparison top-hits (padj = 0.0299). E) Enriched regions of chromosome X from PGE analysis on MAT locus screen top-hits (padj = 0.00901) and Multiple Screen Comparison top-hits (padj = 4.18x10&#x2212;4). No regions containing three or more identified genes were enriched when our MAT locus dataset was run. F) Enriched region of chromosome XI from PGE analysis on MAT locus screen top-hits (padj = 0.00751) and the Multiple Screen Comparison top-hits (padj = 0.013). G) Enriched region of chromosome XV from PGE analysis on MAT locus screen top-hits (padj = 0.00288), and MAT + MET15 top-hits (padj = 0.00332). No enriched regions containing more than three identified genes are found on chromosome XV for the Multiple Screen Comparison analysis. H) Enriched region of chromosome XVI from PGE analysis on Multiple Screen Comparison dataset (padj = 0.046)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485245804" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Positional Gene Enrichment (PGE) analysis for enriched chromosome regions. Genes highlighted in green were identified by the MAT screen; genes highlighted in orange were identified by the MAT and MET15 screens; genes highlighted in burgundy were identified in two independent studies; genes highlighted in pink were identified by three independent studies. The key denoting color labels remains the same throughout all parts of the figure. A) Enriched regions of chromosome II from PGE analysis on MAT locus screen top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 2.74 &#xD7; 10&lt;sup&gt;&#x2212;4&lt;/sup&gt;), MAT + &lt;em&gt;MET15&lt;/em&gt; top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 9.69 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt;), and Multiple Screen Comparison top-hits (six screens compared in Fig. 3), (BP region 442918-575991) (p&lt;sub&gt;adj&lt;/sub&gt; = &lt;em&gt;P&lt;/em&gt; = 0.016). B) Enriched regions of chromosome V from PGE analysis on MAT locus screen top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 2.31 &#xD7; 10&lt;sup&gt;&#x2212;4&lt;/sup&gt;), MAT + &lt;em&gt;MET15&lt;/em&gt; top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 9.69 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt;), and Multiple Screen Comparison top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 2.03x10&lt;sup&gt;&#x2212;5&lt;/sup&gt;). C) Enriched regions of chromosome VII from PGE analysis on MAT locus screen top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 0.00358). No enriched regions of 3 or more identified genes were enriched on this chromosome when the MAT + &lt;em&gt;MET15&lt;/em&gt; or Multiple Screen Comparison datasets were analyzed. D) Enriched regions of chromosome IX from PGE analysis on MAT locus screen top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 4.40 &#xD7; 10&lt;sup&gt;&#x2212;4&lt;/sup&gt;), MAT + &lt;em&gt;MET15&lt;/em&gt; top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 0.00194), and Multiple Screen Comparison top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 0.0299). E) Enriched regions of chromosome X from PGE analysis on MAT locus screen top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 0.00901) and Multiple Screen Comparison top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 4.18x10&lt;sup&gt;&#x2212;4&lt;/sup&gt;). No regions containing three or more identified genes were enriched when our MAT locus dataset was run. F) Enriched region of chromosome XI from PGE analysis on MAT locus screen top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 0.00751) and the Multiple Screen Comparison top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 0.013). G) Enriched region of chromosome XV from PGE analysis on MAT locus screen top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 0.00288), and MAT + &lt;em&gt;MET15&lt;/em&gt; top-hits (p&lt;sub&gt;adj&lt;/sub&gt; = 0.00332). No enriched regions containing more than three identified genes are found on chromosome XV for the Multiple Screen Comparison analysis. H) Enriched region of chromosome XVI from PGE analysis on Multiple Screen Comparison dataset (p&lt;sub&gt;adj&lt;/sub&gt; = 0.046).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2835/F4.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/9/2835/F4.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494105" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Positional Gene Enrichment (PGE) analysis for enriched chromosome regions. Genes highlighted in green were identified by the MAT screen; genes highlighted in orange were identified by the MAT and MET15 screens; genes highlighted in burgundy were identified in two independent studies; genes highlighted in pink were identified by three independent studies. The key denoting color labels remains the same throughout all parts of the figure. A) Enriched regions of chromosome II from PGE analysis on MAT locus screen top-hits (p<sub>adj</sub> = 2.74 × 10<sup>−4</sup>), MAT + <em>MET15</em> top-hits (p<sub>adj</sub> = 9.69 × 10<sup>−6</sup>), and Multiple Screen Comparison top-hits (six screens compared in <a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3</a>), (BP region 442918-575991) (p<sub>adj</sub> = <em>P</em> = 0.016). B) Enriched regions of chromosome V from PGE analysis on MAT locus screen top-hits (p<sub>adj</sub> = 2.31 × 10<sup>−4</sup>), MAT + <em>MET15</em> top-hits (p<sub>adj</sub> = 9.69 × 10<sup>−6</sup>), and Multiple Screen Comparison top-hits (p<sub>adj</sub> = 2.03x10<sup>−5</sup>). C) Enriched regions of chromosome VII from PGE analysis on MAT locus screen top-hits (p<sub>adj</sub> = 0.00358). No enriched regions of 3 or more identified genes were enriched on this chromosome when the MAT + <em>MET15</em> or Multiple Screen Comparison datasets were analyzed. D) Enriched regions of chromosome IX from PGE analysis on MAT locus screen top-hits (p<sub>adj</sub> = 4.40 × 10<sup>−4</sup>), MAT + <em>MET15</em> top-hits (p<sub>adj</sub> = 0.00194), and Multiple Screen Comparison top-hits (p<sub>adj</sub> = 0.0299). E) Enriched regions of chromosome X from PGE analysis on MAT locus screen top-hits (p<sub>adj</sub> = 0.00901) and Multiple Screen Comparison top-hits (p<sub>adj</sub> = 4.18x10<sup>−4</sup>). No regions containing three or more identified genes were enriched when our MAT locus dataset was run. F) Enriched region of chromosome XI from PGE analysis on MAT locus screen top-hits (p<sub>adj</sub> = 0.00751) and the Multiple Screen Comparison top-hits (p<sub>adj</sub> = 0.013). G) Enriched region of chromosome XV from PGE analysis on MAT locus screen top-hits (p<sub>adj</sub> = 0.00288), and MAT + <em>MET15</em> top-hits (p<sub>adj</sub> = 0.00332). No enriched regions containing more than three identified genes are found on chromosome XV for the Multiple Screen Comparison analysis. H) Enriched region of chromosome XVI from PGE analysis on Multiple Screen Comparison dataset (p<sub>adj</sub> = 0.046).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-18" class="subsection"><h4>Chromosome II:</h4><p id="p-34">A large region of significance was identified comprised of 20 ORFs found on the right arm of chromosome II (Base Pair (BP) region 501798-545972) (p<sub>adj</sub> = 2.75 × 10<sup>−4</sup>), when the 217 MAT locus LOH top-hits list was evaluated, as shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>. Seven of the ORFs found in this region were identified by our MAT locus LOH screen – <em><a href="http://www.yeastgenome.org/locus/S000000337/overview">HSL7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000339/overview">CKS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000341/overview">YBR137W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000343/overview">ATG42</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000345/overview">BMT2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000348/overview">YBR144C</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000355/overview">APD1</a></em>. When the MAT + <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> data set was subjected to PGE analysis, a more defined region of chromosome II (a subsection of the region mentioned above) was found to be more significantly enriched (BP region 504848-545972) (p<sub>adj</sub> = 9.69 × 10<sup>−6</sup>) <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>. Six of the seven previously mentioned gene deletions in this region, again appear in a now refined territory comprised of 18 ORFs. We next refined our PGE investigation to all top-hits found in our multiple screen analysis, aiming to better identify chromosomal regions important in LOH events. The region of chromosome II identified above is expanded (BP region 442918-575991) (p<sub>adj</sub> = 0.016) and contains 5 ORF hits (<em><a href="http://www.yeastgenome.org/locus/S000000303/overview">YBR099C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">IML3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000337/overview">HSL7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000355/overview">APD1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000373/overview">SSE2</a></em>) in a 79 ORF region.</p></div><div id="sec-19" class="subsection"><h4>Chromosome V:</h4><p id="p-35">When the 217 top-hits from the initial screen of increasing LOH at the MAT locus were analyzed, a 24-gene region containing eight top-hit ORFs was found to be enriched on chromosome V (BP region 375211-424307) (p<sub>adj</sub> = 2.31 x10<sup>−4</sup>). These top-hit genes are <em><a href="http://www.yeastgenome.org/locus/S000000911/overview">FLO8</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000915/overview">TMN3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000919/overview">RPL23B</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000926/overview">DSE1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000931/overview">SAK1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000932/overview">COM2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000933/overview">RPS26B</a></em> <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>. The genes <em><a href="http://www.yeastgenome.org/locus/S000000931/overview">SAK1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000932/overview">COM2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000933/overview">RPS26B</a></em> are immediately adjacent to one another and are identified as a further enriched cluster with a p-value of 9.57 × 10<sup>−4</sup>. After narrowing our MAT LOH top-hits list to 100 genes that increase LOH at both MAT and <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em>, one larger region (BP 387228-560360) with 11 hits across a 100-ORF span (p<sub>adj</sub> = 9.69x10<sup>−6</sup>) and two further enriched subsections of this region (BP region 387228-438340) and (BP region 387228-397649) were identified. The downstream-shifted 30-gene region of chromosome V identified as enriched (BP region 387,228-438,340) (p<sub>adj</sub> = 9.69 × 10<sup>−6</sup>) is comprised of seven top-hits (<em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000915/overview">TMN3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000919/overview">RPL23B</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000926/overview">DSE1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000931/overview">SAK1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000932/overview">COM2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000937/overview">YER135C</a></em>) in the region of 30 genes, while the smaller region is comprised of three ORFs in a six ORF span (p<sub>adj</sub> = 2.57x10<sup>−4</sup>) <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>. Running the PGE analysis with the overlapping 91 genes from the six compared screens presented a further narrowed enriched region (BP region 387228-396168) (p<sub>adj</sub> = 2.03x10<sup>−5</sup>), with 3 repeated ORFs (<em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000915/overview">TMN3</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000918/overview">SLX8</a></em>) in a 5 ORF region (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>). Notably, <em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000915/overview">TMN3</a></em> were identified by two independent studies (this study and Choy <em>et al.</em>), and <em><a href="http://www.yeastgenome.org/locus/S000000918/overview">SLX8</a></em> was identified by three independent studies (Schmidlin <em>et al.</em>, Andersen <em>et al.</em>, and Yuen <em>et al.</em>).</p></div><div id="sec-20" class="subsection"><h4>Chromosome VII:</h4><p id="p-36">An enriched region of significance was identified comprised of 23 ORFs on chromosome VII (BP region 67598-98589) (p<sub>adj</sub> = 3.57 × 10<sup>−3</sup>), when the 217 MAT locus LOH top-hits list was evaluated, as shown in <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4C</a>. Five of the ORFs (<em><a href="http://www.yeastgenome.org/locus/S000003197/overview">SHE10</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003195/overview">MTC3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003186/overview">YGL218W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003184/overview">KIP3</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003176/overview">SIP2</a></em>) found in this region were identified by our MAT locus LOH screen. When the narrowed list of 100 mutants that increase LOH at both MAT + <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> were subjected to PGE analysis this region is no longer identified as being enriched, and further does not recur in PGE evaluation of the 91 overlap list from the multiple screen comparison.</p></div><div id="sec-21" class="subsection"><h4>Chromosome IX:</h4><p id="p-37">An enriched region was identified (BP region 255113 – 270572) when the 217 MAT locus LOH top-hits list was evaluated. This region contains five ORFs (<em><a href="http://www.yeastgenome.org/locus/S000001315/overview">GPP1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001313/overview">MMF1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001312/overview">PCL7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001310/overview">NEO1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000001308/overview">MET30</a></em>) identified in a 10 ORF region (p<sub>adj</sub> = 4.40 × 10<sup>−4</sup>) (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4D</a>). Two enriched regions were identified when PGE was run with the 100 top-hits that appeared in both our MAT + <a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a> screens. The larger region runs from 193592-264891bp and had five hits (<em><a href="http://www.yeastgenome.org/locus/S000001352/overview">ICE2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001349/overview">AIM19</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001322/overview">YIL060W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001313/overview">MMF1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000001310/overview">NEO1</a></em>) in a 49 ORF region. A smaller subsection was identified as the second hit, this region is 18,502bp long and contains 14 ORFs of which 3 were found in our top-hits (BP region 246389-264891) (p<sub>adj</sub> = 2.72 × 10<sup>−3</sup>) (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4D</a>). A 13 ORF region of chromosome IX (BP region 83302-100501) contains two genes identified by at least two of the six compared studies (p<sub>adj</sub> = 0.0299). One of the core CCT-complex genes, <em><a href="http://www.yeastgenome.org/locus/S000001404/overview">CCT2</a></em>, is found in this region, and was identified by our study as well as Choy <em>et al.</em> Additionally, <em><a href="http://www.yeastgenome.org/locus/S000001394/overview">CSM2</a></em> was identified by this study, and the primary screening conducted by Schmidlin <em>et al.</em> (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4D</a>).</p></div><div id="sec-22" class="subsection"><h4>Chromosome X:</h4><p id="p-38">No enriched regions containing more than 2 ORFs are found when running the 217 MAT locus genes alone. The MAT + <a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a> 100 gene analysis identifies three ORFs (<em><a href="http://www.yeastgenome.org/locus/S000003546/overview">YJL009W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003541/overview">SYS1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003770/overview">SPC1</a></em>) in a 23 ORF region (BP region 419849-458354) (p<sub>adj</sub> = 9.01 × 10<sup>−3</sup>) (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4E</a>). A second enriched locus of interest is identified on chromosome X when the overlapping gene list from the multiple screen comparison is analyzed. This region is from 491074-517506bp and contains 3 top hits (<em><a href="http://www.yeastgenome.org/locus/S000003793/overview">CPR7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003795/overview">PET191</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003804/overview">POL32</a></em>) in a 12 ORF region (p<sub>adj</sub> = 4.18x10<sup>−4</sup>).</p></div><div id="sec-23" class="subsection"><h4>Chromosome XI:</h4><p id="p-39">Three ORFs (<em><a href="http://www.yeastgenome.org/locus/S000001781/overview">YKR073C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001784/overview">ECM4</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000001786/overview">YKR078W</a></em>) found in the MAT screen are identified in a 8 ORF region on chromosome XI (BP region 577829-586351) (p<sub>adj</sub> = 7.51 × 10<sup>−3</sup>) (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4F</a>). This region is not found in our MAT + <a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a> list. However, an overlapping region is found when the multiple screen comparison list is studied (BP region 584594-595940) (p<sub>adj</sub> = 0.013) and contains 2 ORFs, <a href="http://www.yeastgenome.org/locus/S000001786/overview">YKR078W</a> and <a href="http://www.yeastgenome.org/locus/S000001790/overview">NUP133</a>, in a 5 ORF region.</p></div><div id="sec-24" class="subsection"><h4>Chromosome XV:</h4><p id="p-40">A large region is returned from PGE analysis of the MAT top-hits list (BP region 69376-139045) (p<sub>adj</sub> = 2.88x10<sup>−3</sup>) with seven (<em><a href="http://www.yeastgenome.org/locus/S000005495/overview">MED7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005494/overview">YOL134C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005483/overview">HRP1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005475/overview">PAP2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005465/overview">WSC3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005464/overview">NDJ1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005456/overview">COQ3</a></em>) of the 46 ORFs in the region being found in that screen. A subsection of this region (BP region 69376-115808) (p<sub>adj</sub> = 3.32 × 10<sup>−3</sup>) with four ORFs (<em><a href="http://www.yeastgenome.org/locus/S000005495/overview">MED7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005483/overview">HRP1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005475/overview">PAP2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005465/overview">WSC3</a></em>) out of a 33 ORF section is identified from the 100 gene MAT + <a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a> list (<a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4G</a>). Analysis of the 91 overlap list from the multiple screen comparison did not return any enriched regions on this chromosome.</p></div><div id="sec-25" class="subsection"><h4>Chromosome XVI:</h4><p id="p-41">No enriched regions are identified from the 217 MAT locus gene list for chromosome XVI. However, when the 100 top-hits gene list from the MAT + <a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a> screens is analyzed for chromosomal enrichment four ORFs in an 83 ORF region are found (BP region 600646-728613) (p<sub>adj</sub> = 0.041). Two (<em><a href="http://www.yeastgenome.org/locus/S000006438/overview">RPL36B</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006186/overview">DIP5</a></em>) out of a 20 ORF region are found in a separate region of chromosome XVI (BP region 41043-76239) (p<sub>adj</sub> = 0.046) and two other ORFs (<em><a href="http://www.yeastgenome.org/locus/S000006115/overview">DDC1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006122/overview">YIG1</a></em>) out of a 9 ORF region (BP region 169769-181114) (p<sub>adj</sub> = 0.019) are also identified when the list of 91 genes that were found in at least two separate screens are analyzed (<a id="xref-fig-4-14" class="xref-fig" href="#F4">Figure 4H</a>). Within these regions several genes from heterozygous LOH/genome stability screens are found, <em><a href="http://www.yeastgenome.org/locus/S000006186/overview">DIP5</a></em> was identified by both <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Strome <em>et al.</em> (2008)</a> and Choy et.al., and <em><a href="http://www.yeastgenome.org/locus/S000006438/overview">RPL36B</a></em> was identified by Choy <em>et al.</em> and this study.</p></div></div><div id="sec-26" class="subsection"><h3>Fluctuation analysis of LOH candidate genes selected from data narrowing criteria</h3><p id="p-42">In an effort to determine the level of LOH induced by heterozygous gene mutations of interest we created strains capable of a quantitative fluctuation analysis assessment. Nine genes, <em><a href="http://www.yeastgenome.org/locus/S000000004/overview">SSA1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003647/overview">CCT7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000566/overview">MRC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003874/overview">RSM7</a></em>, <em>HS7</em>, <em><a href="http://www.yeastgenome.org/locus/S000000373/overview">SSE2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000987/overview">PUG1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005891/overview">YOR364W</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000340/overview">MEC1</a></em>, were chosen for this assessment based on appearing within our screen and at least one secondary analyses discussed above (additional description of gene functions’ can be found in the discussion section below); the LOH rates of strains heterozygously mutated for these genes were then benchmarked using fluctuation analysis at the <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">CAN1</a></em> locus. <em><a href="http://www.yeastgenome.org/locus/S000000987/overview">PUG1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005891/overview">YOR364W</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003874/overview">RSM7</a></em> were chosen because heterozygous mutations in these genes results in the highest LOH event scores in both our MAT and <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> assays. Futhermore <em><a href="http://www.yeastgenome.org/locus/S000005891/overview">YOR364W</a></em> is categorized as a dubious open reading frame (ORF) indicating lack of clarity on if a protein/functional product is produced, and if so, an unknown function for that product. As such, <em><a href="http://www.yeastgenome.org/locus/S000005891/overview">YOR364W</a></em> lacks characterization and falls in the <em>Unknown</em> categories for all three main GO terms: Molecular Function, Biological Process, and Cellular Component. <em><a href="http://www.yeastgenome.org/locus/S000000566/overview">MRC1</a></em> was selected because it was indentified in three of the studies we analyzed, while <em><a href="http://www.yeastgenome.org/locus/S000000373/overview">SSE2</a></em> was chosen because it was identified in two of the separate published studies (this study and Choy <em>et al.</em> that showed significant overlap) we analyzed for increasing LOH. To learn more about one of the clustered regions identified through PGE we selected three genes from the chromosomal II array, <em><a href="http://www.yeastgenome.org/locus/S000000340/overview">MEC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000373/overview">SSE2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000337/overview">HSL7</a></em>. <em><a href="http://www.yeastgenome.org/locus/S000003647/overview">CCT7</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000004/overview">SSA1</a></em> were included in these analyses because they are part of the overrepresented gene ontology category of the CCT-complex, identified by both our study and Choy <em>et al.</em> Finally <em><a href="http://www.yeastgenome.org/locus/S000003647/overview">CCT7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000373/overview">SSE2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000566/overview">MRC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003874/overview">RSM7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000337/overview">HSL7</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000004/overview">SSA1</a></em>, all have identified human homologs (see discussion) which could make their further investigation more relevant to LOH events in cancers. The SCDChet strains containing heterozygous mutations in <em><a href="http://www.yeastgenome.org/locus/S000000004/overview">SSA1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003647/overview">CCT7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000566/overview">MRC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003874/overview">RSM7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000337/overview">HSL7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000373/overview">SSE2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000987/overview">PUG1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005891/overview">YOR364W</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000340/overview">MEC1</a></em>, as well as the parental strain used to construct SCDChet – BY4743 – were transformed with a <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">can1</a></em>::<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> cassette to render their <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">CAN1</a></em> locus heterozygous. Four independent fluctuation analysis experiments for the parental strain BY4743 were conducted to estimate the baseline LOH rate and 95% confidence intervals (see <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) using the rSalvador package (<a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Zheng 2002</a>). The nine heterozygous gene deletion strains tested all showed significant increased LOH rates with 7- to 31-fold increases over the parental LOH rate (see <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). These increased rates demonstrate that all of our secondary analyses were successful in narrowing candidates of interest whose single-gene deletion leads to a significant increase in LOH.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LOH rates at the CAN1 locus due to nine separate heterozygous gene mutations. The data shown represents a combination of a minimum of two independent experiments. The black circle depicts the mean LOH rate with the tails showing the experimental 95% CIs. Non-overlapping 95% CIs, to the wildtype BY4743 strain, are considered significantly different as the 95% CI overlap method mimics a two-tailed, two-population t-test at the conventional P &lt; 0.05 level with an improvement in type I error rate and statistical power when compared to a t-test, which has been found unsuitable for FA data analysis (Zheng 2015)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485245804" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;LOH rates at the &lt;em&gt;CAN1&lt;/em&gt; locus due to nine separate heterozygous gene mutations. The data shown represents a combination of a minimum of two independent experiments. The black circle depicts the mean LOH rate with the tails showing the experimental 95% CIs. Non-overlapping 95% CIs, to the wildtype BY4743 strain, are considered significantly different as the 95% CI overlap method mimics a two-tailed, two-population &lt;em&gt;t&lt;/em&gt;-test at the conventional &lt;em&gt;P&lt;/em&gt; &lt; 0.05 level with an improvement in type I error rate and statistical power when compared to a &lt;em&gt;t&lt;/em&gt;-test, which has been found unsuitable for FA data analysis (Zheng 2015).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2835/F5.medium.gif" width="366" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/9/2835/F5.medium.gif" width="366" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2835/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494144" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-43" class="first-child">LOH rates at the <em>CAN1</em> locus due to nine separate heterozygous gene mutations. The data shown represents a combination of a minimum of two independent experiments. The black circle depicts the mean LOH rate with the tails showing the experimental 95% CIs. Non-overlapping 95% CIs, to the wildtype BY4743 strain, are considered significantly different as the 95% CI overlap method mimics a two-tailed, two-population <em>t</em>-test at the conventional <em>P</em> &lt; 0.05 level with an improvement in type I error rate and statistical power when compared to a <em>t</em>-test, which has been found unsuitable for FA data analysis (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Zheng 2015</a>).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-27"><h2 class="">Discussion</h2><p id="p-44">Through the work presented here we sought to identify heterozygous gene mutations with haploinsufficient impacts on LOH, as the homologs of these genes are particularly interesting targets for study since loss/mutation to only one allele could induce LOH-based cancer phenotypes. Multiple refinement strategies were employed to attempt to identify those genes with the greatest potential as candidate cancer susceptibility genes for futher study.</p><div id="sec-28" class="subsection"><h3>Human homolog identification and cancer associations</h3><p id="p-45">Of the 217 genes identified as top-hits in the primary MAT locus LOH screening, we were able to identify 127 with a known human homolog (58.5%) utilizing YeastMine (<a href="https://yeastmine.yeastgenome.org">https://yeastmine.yeastgenome.org</a>) and NCBI Homologene (<a href="https://www.ncbi.nlm.nih.gov/homologene">https://www.ncbi.nlm.nih.gov/homologene</a>). Literature searches on these 127 genes identified 86 with a known association with cancer incidence (40% of the top-hits list, 68% of the “genes with human homologs” list) (see Table S8 for all top-hits with known human homologs). When the smaller list of mutants that increased LOH at both MAT + <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> was examined for human homologs in the same manner as above, 58 of the 100 genes (58%) were identified as having a human homolog, with 36 of the 58 human homologs with a known association to cancer (62%).</p><p id="p-46">By again performing data comparisons to multiple independent studies, we are able to collect more information about genes with human homologs and those with known associations to cancers. Sixty-seven out of the 91 genes identified in at least two screens, have a known human homolog (74%). Including the genes identified in our screen (discussed above), a search of the current literature revealed that 54 of those 67 genes have an association with cancer (69%) (Table S9) (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Sun <em>et al.</em> 2002</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Leone <em>et al.</em> 2003</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Mason <em>et al.</em> 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdel-Fatah <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hennecke <em>et al.</em> 2015</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">The <em>et al.</em> 2015</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wang <em>et al.</em> 2015</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Taguchi <em>et al.</em> 2015</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dai <em>et al.</em> 2016</a>). This serves as a positive control that performing these screens, and the multiple screen analyses, identifies genes, both with known human homologs that can be investigated for impacts on cancer development, as well as those that have already been linked to roles in cancer progression. The list of genes, with known human homologs not already associated with cancer phenotypes, are prime candidates as novel cancer susceptibility genes for future studies (as an example see discussion of <em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em> below).</p></div><div id="sec-29" class="subsection"><h3>Multiple screen comparisons for reproducibility of gene identification</h3><p id="p-47">In addition to conducting new screens to search for additional candidates, pooling the data from multiple, related screens, serves as a further layer to test reproducibility and enables identification of candidate genes that are likely contributors to genome instability.</p><p id="p-48"><em><a href="http://www.yeastgenome.org/locus/S000000373/overview">SSE2</a></em> encodes a heat shock protein 70 (HSP70) family member which functions as a nucleotide exchange factor for cytosolic Hsp70s during protein refolding (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Easton <em>et al.</em> 2000</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Shaner <em>et al.</em> 2005</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Dragovic <em>et al.</em> 2006</a>). This gene was identified in our primary MAT screen, reproduced in our secondary LOH screen at <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em>, was also identified by Choy <em>et al.</em> as contributing to LOH at MAT in a haploinsuffient manner, and is found in an enriched region of chromosome II, and has a human homolog (<em>HSPH1)</em> with a known cancer association. The discovered relationship of heterozygous loss of this gene to induction of LOH events in yeast could provide an avenue for further mechanistics studies in this model system, a pathway for study design in mammalian cells, and a justification to screen for <em>HSPH1</em> alterations in additional cancer types.</p><p id="p-49">Similarly, <em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em> was identified in our MAT and <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> datasets (with one of the highest levels of LOH events observed in each), was also identified by Choy <em>et al.</em>, is found in an enriched region of chromosome V, and has a human homolog (<em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em>), however there is no current direct cancer association. This gene encodes part of a protein complex involved in mRNA splicing, processing body assembly, and decay (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">He and Parker 2000</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beggs 2005</a>). First characterized in systemic lupus erythemathosus patients, this protein is an intriguing prospect for further study, as SLE is a heterogeneous disorder, linked to increased incidence of many cancer types, however the data are inconsistent (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gayed <em>et al.</em> 2009</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Song <em>et al.</em> 2018</a>). Direct investigation into <em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em> haploinsufficiency in a yeast model might allow investigation into the role of LOH events in disease progression.</p><p id="p-50">Additionally, the seven genes found in three overlapping, related screens – <em><a href="http://www.yeastgenome.org/locus/S000000521/overview">DCC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000918/overview">SLX8</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002924/overview">EMI2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000566/overview">MRC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002726/overview">MCM21</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001073/overview">RRM3</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005670/overview">ELG1</a></em> – qualify as strong candidates for further investigation due to consistently contributing to instability via LOH mechanisms under a variety of experimental conditions. Six of these genes are categorized in the <em>DNA Metabolic Processes</em> gene ontology and are further linked to chromosome organization (<em><a href="http://www.yeastgenome.org/locus/S000000521/overview">DCC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000918/overview">SLX8</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000566/overview">MRC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002726/overview">MCM21</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001073/overview">RRM3</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005670/overview">ELG1</a>)</em>. With known human homologs and cancer associations for all but <em><a href="http://www.yeastgenome.org/locus/S000002726/overview">MCM21</a></em>, study of the mechanisms of association with increased LOH events could yield more information for inclusion in epidemiological studies as well as pathway information for targeted treatments.</p><p id="p-51">Additional genes that induced the highest levels of LOH in our screens and showed statistically significant increases in LOH via fluctuation analysis are <em><a href="http://www.yeastgenome.org/locus/S000003874/overview">RSM7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000987/overview">PUG1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005891/overview">YOR364W</a></em>. <em><a href="http://www.yeastgenome.org/locus/S000003874/overview">RSM7</a></em> encodes a small subunit mitochondrial ribosomal protein (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Saveanu <em>et al.</em> 2001</a>) and mutations in this gene were also picked up by Schmidlin <em>et al.</em> as increasing LOH events. Little else has been published about this gene and it is intriguing to consider how defects in a mitochondrial ribosomal protein induce nuclear genome instability. Further, while a human homolog has been identified, to date there has been no association with cancer making it a novel candidate cancer predisposition gene identified here. <em><a href="http://www.yeastgenome.org/locus/S000000987/overview">PUG1</a></em> encodes a protoporphyrin uptake protein in the plasma membrane also involved in heme transport (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Protchenko <em>et al.</em> 2008</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Manente and Ghislain 2009</a>). Choy <em>et al.</em> identified haploinsufficiency effects of loss of this gene on chromosome maintenance and Zhu <em>et al.</em> have shown increased colony sectoring in a CTF assay using the SCDChet (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Zhu <em>et al.</em> 2015</a>). Again, little additional information has been published on this gene. <a href="http://www.yeastgenome.org/locus/S000005891/overview">YOR364W</a>, however, has had no direct work published about it, and is therefore another intriguing candidate. This ORF is still considered “dubious” and “unlikely to encode a functional protein” (<a href="https://www.yeastgenome.org/locus/S000005891">https://www.yeastgenome.org/locus/S000005891</a>). We however saw high levels of LOH induction in both our MAT locus and <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> locus screens as well as 14-fold increase in LOH at the <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">CAN1</a></em> locus via fluctuation analysis.</p></div><div id="sec-30" class="subsection"><h3>Gene ontology enrichment</h3><p id="p-52">Further indication that our screen results are pertinent to the identification of human cancer-relevant gene mutations comes from our gene ontology analyses. On top of identifying many ontologies with clear cancer relevance such as <em>Chromosome Segregation</em>, <em>Mitotic Cell Cycle</em>, and <em>DNA Repair</em>, our GO analysis identified other ontologies which may hold relevance, but require more analysis of the members for candidate cancer susceptibility genes. For example, one of the top GO Term Finder ontology hits identified the <em>CCT-complex</em>; conserved from yeast to humans, some components of this complex have known cancer impacts. In yeast, systematic identical null mutations in each subunit of the eight core CCT complex proteins revealed varying phenotypic effects, indicating the possibility of secondary roles that individual subunits play in addition to cytoskeletal subunit folding as part of the CCT-complex (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amit <em>et al.</em> 2010</a>). Secondary cellular roles of these subunits are additionally supported in research linking many, but not all, of the eight subunits to different cancer types. For instance, the homolog of one of our identified top-hits, <em><a href="http://www.yeastgenome.org/locus/S000001404/overview">CCT2</a></em>, along with <em><a href="http://www.yeastgenome.org/locus/S000002620/overview">TCP1</a></em>, has recently been linked as a driver mutation in breast tumor formation (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Guest <em>et al.</em> 2015</a>). As well, change in expression of <em><a href="http://www.yeastgenome.org/locus/S000003545/overview">CCT8</a></em> has been linked to poor prognosis in glioma patients (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Qiu <em>et al.</em> 2015</a>). Studying the proteins of this complex could lead to more mechanistic insight on their secondary roles as well as determination of if other members of the complex play a role in cancer development. Furthermore, because members of the CCT-complex are essential genes, utilizing the SCDChet allowed for identification of this complex where it could not have been assessed using homozygous deletions; therefore, allowing for a more comprehensive look at genes contributing to genome instability through LOH events.</p></div><div id="sec-31" class="subsection"><h3>Chromosome location enrichment clustering of genes involved in LOH</h3><p id="p-53">In the search for additional information that might aid in generating a better understanding of LOH we investigated the chromosome location of hits within our screen results alone, and further compared across multiple screens. These searches hold the potential to identify chromosomal regions and genes important in LOH events as well as to rule out genes that might have been identified due to artifacts in strain construction. On the one hand there are acknowledged errors, advantages, and disadvantages, for using a particular deletion collection (aneuploid strains, secondary mutations, incorrect genotyping) as well as screen-specific issues of strain reconstruction due to the nature of certain mutations (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hughes <em>et al.</em> 2000</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Giaever <em>et al.</em> 2002</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Deutschbauer <em>et al.</em> 2005</a>; <a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">Yuen <em>et al.</em> 2007</a>; <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Schmidlin <em>et al.</em> 2007</a>; <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Andersen <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ben-Shitrit <em>et al.</em> 2012</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Giaever and Nislow 2014</a>). These known inconsistencies however do not negate the importance of the <em>S. cerevisiae</em> deletion collections as a tool for understanding genome instability, and the ability to apply knowledge to the study of higher eukaryotes. Conversely, enriched regions may contain regulatory sequences or identify the presence of a particularly important gene in the vicinity, and may point us to candidates with the most potential for further investigation. Because there is not conclusive evidence in all cases to confirm if a gene knockout is contributing to LOH due to regional effects, or via an independent mechanism, all possibilities need to be considered. We take the identified region of chromosome II as an example to discuss these models.</p><div id="sec-32" class="subsection"><h4>Chromosome II:</h4><p id="p-54">The large region of significance identified on the right arm of chromosome II, included seven MAT locus LOH screen identified genes within a 20 ORF region – <em><a href="http://www.yeastgenome.org/locus/S000000337/overview">HSL7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000339/overview">CKS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000341/overview">YBR137W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000343/overview">ATG42</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000345/overview">BMT2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000348/overview">YBR144C</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000355/overview">APD1</a></em> and expanded to 19 identified hits in a region of 49 ORFs when run with the 91 gene multiple screen comparison top-hits list. Several possibilities exist for why this region was identified as an enriched locus for increasing LOH events.</p><p id="p-55">The first possibility is that all/most of the genes in this neighborhood have independent impacts on LOH occurrences. While a few occasions of clustering of yeast genes with similar functions have been identified (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Zhang and Smith 1998</a>), it is possible that due to the large variety of gene functions that could be perturbed and lead to LOH that this region was not previously identified as one of these such clusters. Evolutionarily this clustering may have come into existance due to co-regulation of these genes for their involvement in genome stability maintenance.</p><p id="p-56">A second possibility is that neighboring gene effects are driving LOH events through a single driver gene in the locus. For example, near the center of this region lies the gene <em><a href="http://www.yeastgenome.org/locus/S000000340/overview">MEC1</a></em>. Orthologous to human ATR, <em><a href="http://www.yeastgenome.org/locus/S000000340/overview">MEC1</a></em> is a critical mitotic checkpoint gene that plays an important role in responding to DNA damage as the cell navigates through the cell cycle (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Harrison and Haber 2006</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bandhu <em>et al.</em> 2014</a>). Further, mutations in <em><a href="http://www.yeastgenome.org/locus/S000000340/overview">MEC1</a></em> have previously been shown to increase mitotic recombination events (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fasullo and Sun 2008</a>) and decrease chromosome maintenance events (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Choy <em>et al.</em> 2013</a>), both of which could lead to increases in LOH; and has been identified in at least two previous screens looking for genes with impacts on genome stability (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Stirling <em>et al.</em> 2011</a>; <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Choy <em>et al.</em> 2013</a>). <em><a href="http://www.yeastgenome.org/locus/S000000340/overview">MEC1</a></em> was identified as a top-hit by Choy <em>et al.</em> indicating heterozygous loss of this locus can increase LOH events. As an essential gene this ORF would not be identifiable through screens utilizing the homozygous deletion collection. Other groups have reported neighborhood effects of KANMX insertions, however these are generally limited to adjacent genes within 600bp upstream or downstream of the driver locus. Ben-Shitrit <em>et al.</em> published a mechanism to predict such neighboring gene effects, however implementation requires a known protein-protein interaction network with anchoring proteins for the phenotype being measured (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Ben-Shitrit <em>et al.</em> 2012</a>). Since LOH events are caused by widely variable mechanisms via genes with widely variable functions we were unable to utilize their algorithm in this situation. Further study of this region to determine if all the identified ORFs are presenting as top-hits due to their proximity to a particular gene, like <em><a href="http://www.yeastgenome.org/locus/S000000340/overview">MEC1</a></em>, or if they are contributing to genome instability through neighborhood-independent mechanisms could be accomplished through additional studies. This could take the form of individual knockout of every gene separately in a new strain, LOH rate quantification (such as via fluctuation analysis), and complementation assays, or through separate testing for expression levels of all of the genes in the region in each of the individual SCDChet strains representing the genes across this chromosomal region. Examining the extent of individual neighborhood effects in these clusters is a future direction that falls outside the scope of this study.</p><p id="p-57">A third model is that the chromosomal region itself, potentially through TF binding, histone modification, replication firing, or three-dimensional architecture, may play roles in multiple loci being identified. Several studies have shown that a gene’s expression level tends to be similar to that of its neighbors on a chromosome (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Zhang and Smith 1998</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kruglyak and Tang 2000</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cohen <em>et al.</em> 2000</a>). If these are dosage sensitive genes, this might contribute to them having haploinsufficiency effects, which might account for the chromosomal region’s identification. To assess this possibility for the chromosome II region we searched for topologically associating domains (TAD) within a high-throughput chromosome conformation capture (Hi-C) dataset (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Eser <em>et al.</em> 2017</a>). A 130kb TAD was found to stretch across the region we identified (spanning from ∼450-580kb on Chr II), however the authors of that study report that the TAD-like domains they found were more strongly correlated with replication timing than with transcription.</p><p id="p-58">A fourth model is that chromosomal regions are being identified as involved in increased LOH as a result of an artifact of how the knockouts were generated. To attempt to assess the possibility of region identification due to strain artifacts we have done two analyses. We started by running PGE with all 898 genes from the multiple screen comparisons (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">De Preter <em>et al.</em> 2008</a>). The logic here is that if large regions are identified without individual genes being replicated this could tell us about the possibility for artifacts. Further, in running PGE in this manner we made note of which screen identified each gene, Table S10. We then further categorized each gene identified to the screen set-up and screens utilizing the same starting strains (<em>i.e.</em>, SCDChet <em>vs.</em> SCDChom <em>vs.</em> non-SCDC strains) were combined to determine if strain creation artifacts could be at play. We found that for the region in chromosome II, nearly all of the hits (16/19) were found from screens that utilized SCDChet, Figure S1. This could support the theory that something about these particular strains and the way they were made is leading to identification of this locus, but could also indicate a region high in genes with haploinsufficiency effects or essential genes not able to be identified by SCDChom screens (further substantiated by the fact that the other screen that identified two genes in this region did not use either deletion collection but completed insertional mutagenesis to assay for heterozygous effects (<a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Strome <em>et al.</em> 2008</a>)). We therefore moved to a second analysis to assess how strains were created as part of making the <em>S. cerevisiae</em> deletion collections. Since past artifacts have been identified when a set of strains were all created in the same lab (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lehner <em>et al.</em> 2007</a>) we wanted to gauge if this chromosomal region of knockouts met this criteria. Information on strain creation from the Standford yeast deletion project website (<a href="http://www-sequence.stanford.edu/group/yeast_deletion_project/overlapping.html">http://www-sequence.stanford.edu/group/yeast_deletion_project/overlapping.html</a>) identifies “lab 11” as having created this entire block of gene mutations, ending just where our identified region ends (at ORF <a href="http://www.yeastgenome.org/locus/S000000376/overview">YBR172C</a>), however starting farther up (having created strains starting at <a href="http://www.yeastgenome.org/locus/S000000284/overview">YBR080C</a>, our identified region starts at <a href="http://www.yeastgenome.org/locus/S000000331/overview">YBR127C</a>). This unfortunately both supports and conflicts with the theory of artifact-induced LOH events in all strains made by this group. Supporting evidence is that this group did make all the gene knockout strains we identified. Conflictingly, however we would have expected to have identified all genes/most genes in the chromosomal region that encompasses all of the strains they created, not approximately half.</p><p id="p-59">These models could apply to all of the PGE identified enriched regions and further investigation into all of these loci are warranted, but are outside of the scope of this mutant screen report.</p></div></div><div id="sec-33" class="subsection"><h3>Essential genes</h3><p id="p-60">As less frequently evaluated loci, not assayed in haploid studies or studies of the homozygous deletion collection that are more frequently conducted, the cohort of essential genes from our screen are interesting targets for further study. 37 essential genes were identified in our primary screen <em><a href="http://www.yeastgenome.org/locus/S000002255/overview">RPN6</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000028421/overview">HSK3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005261/overview">PFS2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002498/overview">RLI1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002711/overview">RSC3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000961/overview">BUR6</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003081/overview">SLD3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001114/overview">ERG7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001516/overview">TTI1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005483/overview">HRP1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002213/overview">PSA1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002325/overview">FAP7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002444/overview">KRS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002781/overview">FRQ1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001859/overview">MOB2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003546/overview">YJL009W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003647/overview">CCT7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004056/overview">SPC3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005495/overview">MED7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005788/overview">GPN2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000006391/overview">RPO26</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002173/overview">TSC13</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002186/overview">MPS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001308/overview">MET30</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001388/overview">STH1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001404/overview">CCT2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003545/overview">CCT8</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004550/overview">TUB1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005326/overview">MVD1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005494/overview">YOL134C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005621/overview">RKI1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005733/overview">RET1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000119/overview">MCM2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001886/overview">SMC1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001310/overview">NEO1</a></em>. Of these essential genes one, <em><a href="http://www.yeastgenome.org/locus/S000003647/overview">CCT7</a></em>, was chosen for strain recreation to allow quantitative assesement of LOH rate due to heterozygous mutation. The significant 27-fold increase in LOH due to loss of this gene demonstrates its haploinsufficient impact on genome stability. Among this group, 32, have previously identified human homologs of which 23 have a known cancer association. Once again, analyses of these genes/proteins/pathways in a yeast system may help uncover additional mechanistic information important in understanding their roles in cancer development. Further, the nine genes (<em><a href="http://www.yeastgenome.org/locus/S000000914/overview">LSM4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000961/overview">BUR6</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001114/overview">ERG7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002444/overview">KRS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004056/overview">SPC3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005788/overview">GPN2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002173/overview">TSC13</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000323/overview">MUD1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000001310/overview">NEO1</a></em>) with known human homologs not already associated with cancer phenotypes are prime candidates as novel cancer susceptibility genes as they, like their yeast counterparts, may be able to induce haploinsufficient effects, not abiding the two-hit hypothesis, and therefore being more impactful gene mutations.</p></div><div id="sec-34" class="subsection"><h3>Conclusions</h3><p id="p-61">Analyzing new screen data independently and then in conjunction with gene lists from relevant published works, as well as performing analyses beyond individual genes by looking at such items as ontology representation and positional gene enrichment, can increase the power of a study to identify genes of most importance. The studies a group might choose for comparison could be selected by literature search identification of relevant screens. We propose that this methodology of candidate narrowing allows the community to wade through the noise of data and focus on chromosomal deletions that play important roles in the phenotype or pathway of interest.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. Research was supported by an Institutional Development Award (IDeA) from the National Institute of General Medical Sciences of the National Institutes of Health: P20GM1234 and an NIGMS R15 Area Award: 1R15GM109269-01A1.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-63">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.8325992">https://doi.org/10.25387/g3.8325992</a>.</p></li><li class="fn" id="fn-3"><p id="p-64"><em>Communicating editor: G. Brown</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-13"><span class="received-label">Received </span>December 13, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-06-28"><span class="accepted-label">Accepted </span>June 28, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Hoffert, Strome</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.1" data-doi="10.1016/j.molonc.2014.04.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdel-Fatah</span>,  <span class="cit-name-given-names">T. M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Russell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Albarakati</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Maloney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dorjsuren</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic and protein expression analysis reveals flap endonuclease 1 (FEN1) as a key biomarker in breast and ovarian cancer.</span> <abbr class="cit-jnl-abbrev">Mol. Oncol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1326</span>–<span class="cit-lpage">1338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BOncol.%26rft.volume%253D8%26rft.spage%253D1326%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molonc.2014.04.009%26rft_id%253Dinfo%253Apmid%252F24880630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molonc.2014.04.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24880630&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.2" data-doi="10.1038/ng.556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alimonti</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Carracedo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Clohessy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Trotman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Nardella</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Subtle variations in Pten dose determine cancer susceptibility.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">454</span>–<span class="cit-lpage">458</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.556</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAlimonti%26rft.auinit1%253DA.%26rft.volume%253D42%26rft.issue%253D5%26rft.spage%253D454%26rft.epage%253D458%26rft.atitle%253DSubtle%2Bvariations%2Bin%2BPten%2Bdose%2Bdetermine%2Bcancer%2Bsusceptibility.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.556%26rft_id%253Dinfo%253Apmid%252F20400965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20400965&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277179500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.3" data-doi="10.1016/j.jmb.2010.06.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amit</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Weisberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nadler-Holly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">McCormack</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Feldmesser</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Equivalent Mutations in the Eight Subunits of the Chaperonin CCT Produce Dramatically Different Cellular and Gene Expression Phenotypes.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">401</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">543</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmb.2010.06.037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAmit%26rft.auinit1%253DM.%26rft.volume%253D401%26rft.issue%253D3%26rft.spage%253D532%26rft.epage%253D543%26rft.atitle%253DEquivalent%2Bmutations%2Bin%2Bthe%2Beight%2Bsubunits%2Bof%2Bthe%2Bchaperonin%2BCCT%2Bproduce%2Bdramatically%2Bdifferent%2Bcellular%2Band%2Bgene%2Bexpression%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2010.06.037%26rft_id%253Dinfo%253Apmid%252F20600117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2010.06.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20600117&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.4" data-doi="10.1534/genetics.108.089250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>,  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. W.</span> <span class="cit-name-surname">Nelson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. D.</span> <span class="cit-name-surname">Hetrick</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Gottschling</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A Genetic Screen for Increased Loss of Heterozygosity in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.089250</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089250%26rft_id%253Dinfo%253Apmid%252F18562670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzExNzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bae</span>,  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Seberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. P.</span> <span class="cit-name-surname">Carroll</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Swanson</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Identification of Genes in Saccharomyces cerevisiae that Are Haploinsufficient for Overcoming Amino Acid Starvation. G3(Bethesda) 7: 1061–1084.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.6" data-doi="10.1371/journal.pgen.1004136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bandhu</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Fukunaga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Goto</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sugimoto</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ddc2 mediates Mec1 activation through a Ddc1- or Dpb11-independent mechanism.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e1004136</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004136%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004136%26rft_id%253Dinfo%253Apmid%252F24586187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586187&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.7" data-doi="10.1042/BST0330433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beggs</span>,  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Lsm proteins and RNA processing.</span> <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">438</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1042/BST0330433</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSoc.%2BTrans.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBST0330433%26rft_id%253Dinfo%253Apmid%252F15916535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoicHBiaW9zdCI7czo1OiJyZXNpZCI7czo4OiIzMy8zLzQzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yODM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hochberg</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat.</span> <abbr class="cit-jnl-abbrev">Soc. Ser. B Methodol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoc.%2BSer.%2BB%2BMethodol.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.9" data-doi="10.1038/nmeth.1890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben-Shitrit</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Yosef</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Shemesh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Sharan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Ruppin</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Systematic identification of gene annotation errors in the widely used yeast mutation collections.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">378</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1890</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBen-Shitrit%26rft.auinit1%253DT.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D373%26rft.epage%253D378%26rft.atitle%253DSystematic%2Bidentification%2Bof%2Bgene%2Bannotation%2Berrors%2Bin%2Bthe%2Bwidely%2Bused%2Byeast%2Bmutation%2Bcollections.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1890%26rft_id%253Dinfo%253Apmid%252F22306811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22306811&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.10" data-doi="10.1038/nature10275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>,  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Knudson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. P.</span> <span class="cit-name-surname">Pandolfi</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A continuum model for tumour suppression.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">476</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">169</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBerger%26rft.auinit1%253DA.%2BH.%26rft.volume%253D476%26rft.issue%253D7359%26rft.spage%253D163%26rft.epage%253D169%26rft.atitle%253DA%2Bcontinuum%2Bmodel%2Bfor%2Btumour%2Bsuppression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10275%26rft_id%253Dinfo%253Apmid%252F21833082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21833082&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293731900027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.11" data-doi="10.1534/genetics.111.130765"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Fink</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Yeast: an experimental organism for 21st Century biology.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">704</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.130765</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.130765%26rft_id%253Dinfo%253Apmid%252F22084421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzMvNjk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.12" data-doi="10.1016/j.tranon.2017.10.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lee</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Cho</span></span></li></ol><cite>, <span class="cit-pub-date">2018a</span> <span class="cit-article-title">Microsatellite alterations and protein expression of 5 major tumor suppressor genes in gastric adenocarcinomas.</span> <abbr class="cit-jnl-abbrev">Transl. Oncol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tranon.2017.10.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransl.%2BOncol.%26rft.volume%253D11%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tranon.2017.10.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tranon.2017.10.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2835.13" data-doi="10.1038/s41389-018-0094-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Thi Thao Tran</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Van Ngu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. W.</span> <span class="cit-name-surname">Park</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018b</span> <span class="cit-article-title">Promotion of tumor progression and cancer stemness by MUC15 in thyroid cancer via the GPCR/ERK and integrin-FAK signaling pathways.</span> <abbr class="cit-jnl-abbrev">Oncogenesis</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41389-018-0094-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogenesis%26rft.volume%253D7%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41389-018-0094-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41389-018-0094-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.14" data-doi="10.1091/mbc.E12-12-0902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choy</span>,  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">O’Toole</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Schuster</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Crisp</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Karpova</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide haploinsufficiency screen reveals a novel role for γ-TuSC in spindle organization and genome stability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2753</span>–<span class="cit-lpage">2763</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1091/mbc.e12-12-0902</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E12-12-0902%26rft_id%253Dinfo%253Apmid%252F23825022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xNy8yNzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.15" data-doi="10.1016/j.tim.2009.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christie</span>,  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Hong</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Cherry</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Functional annotations for the Saccharomyces cerevisiae genome: the knowns and the known unknowns.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tim.2009.04.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DChristie%26rft.auinit1%253DK.%2BR.%26rft.volume%253D17%26rft.issue%253D7%26rft.spage%253D286%26rft.epage%253D294%26rft.atitle%253DFunctional%2Bannotations%2Bfor%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome%253A%2Bthe%2Bknowns%2Band%2Bthe%2Bknown%2Bunknowns.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2009.04.005%26rft_id%253Dinfo%253Apmid%252F19577472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2009.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19577472&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268616600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.16" data-doi="10.1038/79896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>,  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Mitra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Hughes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Church</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A computational analysis of whole-genome expression data reveals chromosomal domains of gene expression.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/79896</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCohen%26rft.auinit1%253DB.%2BA.%26rft.volume%253D26%26rft.issue%253D2%26rft.spage%253D183%26rft.epage%253D186%26rft.atitle%253DA%2Bcomputational%2Banalysis%2Bof%2Bwhole-genome%2Bexpression%2Bdata%2Breveals%2Bchromosomal%2Bdomains%2Bof%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F79896%26rft_id%253Dinfo%253Apmid%252F11017073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/79896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11017073&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089638800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.17" data-doi="10.1002/(SICI)1097-0061(199608)12:10&lt;939::AID-YEA988&gt;3.0.CO;2-L"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cost</span>,  <span class="cit-name-given-names">G. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Boeke</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A useful colony colour phenotype associated with the yeast selectable/counter-selectable marker MET15.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">939</span>–<span class="cit-lpage">941</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/(SICI)1097-0061(199608)12:10&lt;939::AID-YEA988&gt;3.0.CO;2-L</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DCost%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D939%26rft.epage%253D941%26rft.atitle%253DA%2Buseful%2Bcolony%2Bcolour%2Bphenotype%2Bassociated%2Bwith%2Bthe%2Byeast%2Bselectable%252Fcounter-selectable%2Bmarker%2BMET15.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199608%252912%253A10%253C939%253A%253AAID-YEA988%253E3.0.CO%253B2-L%26rft_id%253Dinfo%253Apmid%252F8873447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199608)12:10&lt;939::AID-YEA988&gt;3.0.CO;2-L&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8873447&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VE32100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.18" data-doi="10.3892/or.2016.4802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Pan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Meng</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">RNaseH2A is involved in human gliomagenesis through the regulation of cell proliferation and apoptosis.</span> <abbr class="cit-jnl-abbrev">Oncol. Rep.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3892/or.2016.4802</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncol.%2BRep.%26rft.volume%253D36%26rft.spage%253D173%26rft_id%253Dinfo%253Adoi%252F10.3892%252For.2016.4802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3892/or.2016.4802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.19" data-doi="10.1093/nar/gkn114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Preter</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Barriot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Speleman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Vandesompele</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Moreau</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Positional gene enrichment analysis of gene sets for high-resolution identification of overrepresented chromosomal regions.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-elocation-id">e43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkn114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn114%26rft_id%253Dinfo%253Apmid%252F18346969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18346969&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.20" data-doi="10.1534/genetics.104.036871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deutschbauer</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Jaramillo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Proctor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kumm</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Hillenmeyer</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of Haploinsufficiency Revealed by Genome-Wide Profiling in Yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1915</span>–<span class="cit-lpage">1925</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.036871</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036871%26rft_id%253Dinfo%253Apmid%252F15716499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzE5MTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.21" data-doi="10.1038/sj.emboj.7601138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dragovic</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Broadley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Shomura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bracher</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. U.</span> <span class="cit-name-surname">Hartl</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Molecular chaperones of the Hsp110 family act as nucleotide exchange factors of Hsp70s.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2519</span>–<span class="cit-lpage">2528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.emboj.7601138</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DDragovic%26rft.auinit1%253DZ.%26rft.volume%253D25%26rft.issue%253D11%26rft.spage%253D2519%26rft.epage%253D2528%26rft.atitle%253DMolecular%2Bchaperones%2Bof%2Bthe%2BHsp110%2Bfamily%2Bact%2Bas%2Bnucleotide%2Bexchange%2Bfactors%2Bof%2BHsp70s%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601138%26rft_id%253Dinfo%253Apmid%252F16688212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTEvMjUxOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yODM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.22" data-doi="10.1016/j.cmet.2016.08.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drucker</span>,  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Never Waste a Good Crisis: Confronting Reproducibility in Translational Research.</span> <abbr class="cit-jnl-abbrev">Cell Metab.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cmet.2016.08.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BMetab.%26rft.volume%253D24%26rft.spage%253D348%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmet.2016.08.006%26rft_id%253Dinfo%253Apmid%252F27626191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cmet.2016.08.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27626191&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.23" data-doi="10.1126/science.168.3938.1472"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duntze</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">MacKay</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. R.</span> <span class="cit-name-surname">Manney</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Saccharomyces cerevisiae: A Diffusible Sex Factor.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1472</span>–<span class="cit-lpage">1473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.168.3938.1472</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDuntze%26rft.auinit1%253DW.%26rft.volume%253D168%26rft.issue%253D3938%26rft.spage%253D1472%26rft.epage%253D1473%26rft.atitle%253DSaccharomyces%2Bcerevisiae%253A%2BA%2BDiffusible%2BSex%2BFactor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.168.3938.1472%26rft_id%253Dinfo%253Apmid%252F5445939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIxNjgvMzkzOC8xNDcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.24" data-doi="10.1379/1466-1268(2000)005&lt;0276:THAGSP&gt;2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Easton</span>,  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kaneko</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Subjeck</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The Hsp110 and Grp170 stress proteins: newly recognized relatives of the Hsp70s.</span> <abbr class="cit-jnl-abbrev">Cell Stress Chaperones</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">290</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1379/1466-1268(2000)005&lt;0276:THAGSP&gt;2.0.CO;2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bstress%2B%2526%2Bchaperones%26rft.stitle%253DCell%2BStress%2BChaperones%26rft.aulast%253DEaston%26rft.auinit1%253DD.%2BP.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D276%26rft.epage%253D290%26rft.atitle%253DThe%2Bhsp110%2Band%2BGrp1%2B70%2Bstress%2Bproteins%253A%2Bnewly%2Brecognized%2Brelatives%2Bof%2Bthe%2BHsp70s.%26rft_id%253Dinfo%253Adoi%252F10.1379%252F1466-1268%25282000%2529005%253C0276%253ATHAGSP%253E2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11048651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1379/1466-1268(2000)005&lt;0276:THAGSP&gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11048651&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089737100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.25" data-doi="10.1073/pnas.1612256114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eser</span>,  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Chandler-Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Ay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. F.</span> <span class="cit-name-surname">Straight</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Duan</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Form and function of topologically associating genomic domains in budding yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">E3061</span>–<span class="cit-lpage">E3070</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1612256114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1612256114%26rft_id%253Dinfo%253Apmid%252F28348222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzE1L0UzMDYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.26" data-doi="10.4161/cc.6411"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fasullo</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sun</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Saccharomyces cerevisiae checkpoint genes RAD9, CHK1 and PDS1 are required for elevated homologous recombination in a mec1 (ATR) hypomorphic mutant.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2418</span>–<span class="cit-lpage">2426</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/cc.6411</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DFasullo%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D15%26rft.spage%253D2418%26rft.epage%253D2426%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2Bcheckpoint%2Bgenes%2BRAD9%252C%2BCHK1%2Band%2BPDS1%2Bare%2Brequired%2Bfor%2Belevated%2Bhomologous%2Brecombination%2Bin%2Ba%2Bmec1%2B%2528ATR%2529%2Bhypomorphic%2Bmutant.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.6411%26rft_id%253Dinfo%253Apmid%252F18677117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.6411&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18677117&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258829800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.27" data-doi="10.1073/pnas.1618215113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Su</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Koeman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Haak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Dykema</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Chromosome instability drives phenotypic switching to metastasis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">14793</span>–<span class="cit-lpage">14798</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1618215113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1618215113%26rft_id%253Dinfo%253Apmid%252F27930335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzUxLzE0NzkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.28" data-doi="10.1177/0961203309102556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gayed</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bernatsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ramsey-Goldman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Clarke</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gordon</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Lupus and cancer.</span> <abbr class="cit-jnl-abbrev">Lupus</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1177/0961203309102556</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLupus%26rft.stitle%253DLupus%26rft.aulast%253DGayed%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D6%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DLupus%2Band%2Bcancer%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0961203309102556%26rft_id%253Dinfo%253Apmid%252F19395448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/0961203309102556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19395448&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266265000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.29" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Chu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ni</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Connelly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Riles</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the Saccharomyces cerevisiae genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature00935</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.30" data-doi="10.1534/genetics.114.161620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Nislow</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The yeast deletion collection: a decade of functional genomics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">465</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.161620</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D197%26rft.issue%253D2%26rft.spage%253D451%26rft.epage%253D465%26rft.atitle%253DThe%2BYeast%2BDeletion%2BCollection%253A%2BA%2BDecade%2Bof%2BFunctional%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161620%26rft_id%253Dinfo%253Apmid%252F24939991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNDUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.31" data-doi="10.1186/s13008-015-0009-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giam</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Rancati</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Aneuploidy and chromosomal instability in cancer: a jackpot to chaos.</span> <abbr class="cit-jnl-abbrev">Cell Div.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13008-015-0009-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BDiv.%26rft.volume%253D10%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13008-015-0009-7%26rft_id%253Dinfo%253Apmid%252F26015801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13008-015-0009-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26015801&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.32" data-doi="10.1016/j.yexcr.2015.02.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guest</span>,  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. R.</span> <span class="cit-name-surname">Kratche</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bollig-Fischer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Haddad</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Ethier</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Two members of the TRiC chaperonin complex, CCT2 and TCP1 are essential for survival of breast cancer cells and are linked to driving oncogenes.</span> <abbr class="cit-jnl-abbrev">Exp. Cell Res.</abbr> <span class="cit-vol">332</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">235</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.yexcr.2015.02.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExp.%2BCell%2BRes.%26rft.volume%253D332%26rft.spage%253D223%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.yexcr.2015.02.005%26rft_id%253Dinfo%253Apmid%252F25704758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.yexcr.2015.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25704758&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.33" data-doi="10.1534/genetics.111.134577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>,  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mating-type genes and MAT switching in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.134577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHaber%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D191%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D64%26rft.atitle%253DMating-Type%2BGenes%2Band%2BMAT%2BSwitching%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.134577%26rft_id%253Dinfo%253Apmid%252F22555442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTkxLzEvMzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.34" data-doi="10.1146/annurev.genet.40.051206.105231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">J. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Haber</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Surviving the breakup: the DNA damage checkpoint.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">235</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.40.051206.105231</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHopwood%26rft.auinit1%253DD.%2BA.%26rft.volume%253D40%26rft.spage%253D209%26rft.epage%253D235%26rft.atitle%253DSurviving%2Bthe%2Bbreakup%253A%2Bthe%2BDNA%2Bdamage%2Bcheckpoint.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.40.051206.105231%26rft_id%253Dinfo%253Apmid%252F16805667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.40.051206.105231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16805667&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243038200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.35" data-doi="10.1016/S0955-0674(00)00098-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Parker</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Functions of Lsm proteins in mRNA degradation and splicing.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0955-0674(00)00098-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DHe%26rft.auinit1%253DW.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253D346%26rft.epage%253D350%26rft.atitle%253DFunctions%2Bof%2BLsm%2Bproteins%2Bin%2BmRNA%2Bdegradation%2Band%2Bsplicing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0955-0674%252800%252900098-3%26rft_id%253Dinfo%253Apmid%252F10801455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0955-0674(00)00098-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10801455&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087138200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.36" data-doi="10.1371/journal.pone.0131280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hennecke</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Beck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bornemann-Kolatzki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Neumann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Murua Escobar</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Prevalence of the Prefoldin Subunit 5 Gene Deletion in Canine Mammary Tumors.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0131280</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0131280</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0131280%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0131280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0131280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.37" data-doi="10.1016/0092-8674(95)90402-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herskowitz</span>,  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">MAP kinase pathways in yeast: For mating and more.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(95)90402-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHerskowitz%26rft.auinit1%253DI.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D187%26rft.epage%253D197%26rft.atitle%253DMAP%2Bkinase%2Bpathways%2Bin%2Byeast%253A%2Bfor%2Bmating%2Band%2Bmore.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252895%252990402-6%26rft_id%253Dinfo%253Apmid%252F7834739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(95)90402-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7834739&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QD92400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.38" data-doi="10.1016/0092-8674(85)90152-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Mann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Snyder</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Davis</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Mitotic stability of yeast chromosomes: a colony color assay that measures nondisjunction and chromosome loss.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(85)90152-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHieter%26rft.auinit1%253DP.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D381%26rft.epage%253D392%26rft.atitle%253DMitotic%2Bstability%2Bof%2Byeast%2Bchromosomes%253A%2Ba%2Bcolony%2Bcolor%2Bassay%2Bthat%2Bmeasures%2Bnondisjunction%2Band%2Bchromosome%2Bloss.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252885%252990152-7%26rft_id%253Dinfo%253Apmid%252F3967296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(85)90152-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3967296&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AFE4800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.39" data-doi="10.1038/77116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>,  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Roberts</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Dai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Meyer</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Widespread aneuploidy revealed by DNA microarray expression profiling.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/77116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHughes%26rft.auinit1%253DT.%2BR.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D333%26rft.epage%253D337%26rft.atitle%253DWidespread%2Baneuploidy%2Brevealed%2Bby%2BDNA%2Bmicroarray%2Bexpression%2Bprofiling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F77116%26rft_id%253Dinfo%253Apmid%252F10888885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/77116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10888885&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087920900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kacser</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Burns</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">The Molecular Basis of Dominance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKacser%26rft.auinit1%253DH.%26rft.volume%253D97%26rft.issue%253D3-4%26rft.spage%253D639%26rft.epage%253D666%26rft.atitle%253DTHE%2BMOLECULAR%2BBASIS%2BOF%2BDOMINANCE%26rft_id%253Dinfo%253Apmid%252F7297851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6Ijk3LzMtNC82MzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.41" data-doi="10.1073/pnas.90.23.10914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knudson</span>,  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Antioncogenes and human cancer.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">10914</span>–<span class="cit-lpage">10921</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.90.23.10914</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKnudson%26rft.auinit1%253DA.%2BG.%26rft.volume%253D90%26rft.issue%253D23%26rft.spage%253D10914%26rft.epage%253D10921%26rft.atitle%253DAntioncogenes%2Band%2Bhuman%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.23.10914%26rft_id%253Dinfo%253Apmid%252F7902574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTAvMjMvMTA5MTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.42" data-doi="10.1073/pnas.68.4.820"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knudson</span>,  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Mutation and Cancer: Statistical Study of Retinoblastoma.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">820</span>–<span class="cit-lpage">823</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.68.4.820</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKnudson%26rft.auinit1%253DA.%2BG.%26rft.volume%253D68%26rft.issue%253D4%26rft.spage%253D820%26rft.epage%253D823%26rft.atitle%253DMutation%2Band%2BCancer%253A%2BStatistical%2BStudy%2Bof%2BRetinoblastoma%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.68.4.820%26rft_id%253Dinfo%253Apmid%252F5279523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI2OC80LzgyMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yODM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.43" data-doi="10.1016/S0168-9525(99)01941-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tang</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Regulation of adjacent yeast genes.</span> <abbr class="cit-jnl-abbrev">Trends Genet. TIG</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0168-9525(99)01941-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D109%26rft.epage%253D111%26rft.atitle%253DRegulation%2Bof%2Badjacent%2Byeast%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252899%252901941-1%26rft_id%253Dinfo%253Apmid%252F10689350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(99)01941-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10689350&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085568600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.44" data-doi="10.1534/genetics.107.079368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Stone</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Farber</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. D.</span> <span class="cit-name-surname">Petes</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Ninety-six haploid yeast strains with individual disruptions of open reading frames between YOR097C and YOR192C, constructed for the Saccharomyces genome deletion project, have an additional mutation in the mismatch repair gene MSH3.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1951</span>–<span class="cit-lpage">1953</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.079368</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.079368%26rft_id%253Dinfo%253Apmid%252F17947417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE5NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.45" data-doi="10.1038/25292"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lengauer</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Kinzler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Vogelstein</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genetic instabilities in human cancers.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">396</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">649</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/25292</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLengauer%26rft.auinit1%253DC.%26rft.volume%253D396%26rft.issue%253D6712%26rft.spage%253D643%26rft.epage%253D649%26rft.atitle%253DGenetic%2Binstabilities%2Bin%2Bhuman%2Bcancers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F25292%26rft_id%253Dinfo%253Apmid%252F9872311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/25292&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9872311&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077694200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.46" data-doi="10.1186/1471-2407-3-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leone</span>,  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mendiola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Alonso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Paz-y-Miño</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Pestaña</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Implications of a RAD54L polymorphism (2290C/T) in human meningiomas as a risk factor and/or a genetic marker.</span> <abbr class="cit-jnl-abbrev">BMC Cancer</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2407-3-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bcancer%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BCancer%26rft.aulast%253DLeone%26rft.auinit1%253DP.%2BE.%26rft.volume%253D3%26rft.spage%253D6%26rft.epage%253D6%26rft.atitle%253DImplications%2Bof%2Ba%2BRAD54L%2Bpolymorphism%2B%25282290C%252FT%2529%2Bin%2Bhuman%2Bmeningiomas%2Bas%2Ba%2Brisk%2Bfactor%2Band%252For%2Ba%2Bgenetic%2Bmarker.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2407-3-6%26rft_id%253Dinfo%253Apmid%252F12614485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2407-3-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12614485&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.47" data-doi="10.1146/annurev.bi.62.070193.003203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>,  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The Tumor Suppressor Genes.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">651</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.bi.62.070193.003203</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DLevine%26rft.auinit1%253DA.%26rft.volume%253D62%26rft.issue%253D1%26rft.spage%253D623%26rft.epage%253D651%26rft.atitle%253DThe%2BTumor%2BSuppressor%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.bi.62.070193.003203%26rft_id%253Dinfo%253Apmid%252F8394683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.bi.62.070193.003203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8394683&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LK68700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2835.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Jin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017a</span> <span class="cit-article-title">Bifunctional enzyme ATIC promotes propagation of hepatocellular carcinoma by regulating AMPK-mTOR-S6 K1 signaling.</span> <abbr class="cit-jnl-abbrev">Cell Commun. Signal. CCS</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BCommun.%2BSignal.%2BCCS%26rft.volume%253D15%26rft.spage%253D52%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2835.49" data-doi="10.1016/j.canlet.2017.02.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017b</span> <span class="cit-article-title">Prognostic alternative mRNA splicing signature in non-small cell lung cancer.</span> <abbr class="cit-jnl-abbrev">Cancer Lett.</abbr> <span class="cit-vol">393</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.canlet.2017.02.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BLett.%26rft.volume%253D393%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.canlet.2017.02.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.canlet.2017.02.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.50" data-doi="10.1073/pnas.0334858100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loeb</span>,  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Loeb</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Anderson</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Multiple mutations and cancer.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">776</span>–<span class="cit-lpage">781</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0334858100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0334858100%26rft_id%253Dinfo%253Apmid%252F12552134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDAvMy83NzYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.51" data-doi="10.1016/j.tips.2005.03.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mager</span>,  <span class="cit-name-given-names">W. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Winderickx</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Yeast as a model for medical and medicinal research.</span> <abbr class="cit-jnl-abbrev">Trends Pharmacol. Sci.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">273</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tips.2005.03.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BPharmacological%2BSciences%2B%2528Regular%2Bed.%2529%26rft.stitle%253DTrends%2Bin%2BPharmacological%2BSciences%2B%2528Regular%2Bed.%2529%26rft.aulast%253DMager%26rft.auinit1%253DW.%2BH.%26rft.volume%253D26%26rft.issue%253D5%26rft.spage%253D265%26rft.epage%253D273%26rft.atitle%253DYeast%2Bas%2Ba%2Bmodel%2Bfor%2Bmedical%2Band%2Bmedicinal%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tips.2005.03.004%26rft_id%253Dinfo%253Apmid%252F15860374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tips.2005.03.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15860374&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229466700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.52" data-doi="10.1111/j.1567-1364.2009.00513.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manente</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ghislain</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The lipid-translocating exporter family and membrane phospholipid homeostasis in yeast.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">687</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1567-1364.2009.00513.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00513.x%26rft_id%253Dinfo%253Apmid%252F19416366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2009.00513.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19416366&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267751800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.53" data-doi="10.1158/0008-5472.CAN-13-3220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. L.</span> <span class="cit-name-surname">Logan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Budke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pawlowski</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The RAD51-stimulatory compound RS-1 can exploit the RAD51 overexpression that exists in cancer cells and tumors.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">3546</span>–<span class="cit-lpage">3555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1158/0008-5472.CAN-13-3220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-13-3220%26rft_id%253Dinfo%253Apmid%252F24753542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI3NC8xMy8zNTQ2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ono</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ishii</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Fujino</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Aoyama</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Role of hydrosulfide ions (HS-) in methylmercury resistance in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">3183</span>–<span class="cit-lpage">3186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DOno%26rft.auinit1%253DB%26rft.volume%253D57%26rft.issue%253D11%26rft.spage%253D3183%26rft.epage%253D3186%26rft.atitle%253DRole%2Bof%2Bhydrosulfide%2Bions%2B%2528HS-%2529%2Bin%2Bmethylmercury%2Bresistance%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F1781681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI1Ny8xMS8zMTgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.55" data-doi="10.1093/carcin/bgi223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>,  <span class="cit-name-given-names">S. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Kemp</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Tumor suppressor genetics.</span> <abbr class="cit-jnl-abbrev">Carcinogenesis</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2031</span>–<span class="cit-lpage">2045</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/carcin/bgi223</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCarcinogenesis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcarcin%252Fbgi223%26rft_id%253Dinfo%253Apmid%252F16150895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/carcin/bgi223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16150895&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233415600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.56" data-doi="10.1128/EC.00414-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Protchenko</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Shakoury-Elizeh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Keane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Storey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Androphy</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Role of PUG1 in Inducible Porphyrin and Heme Transport in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">871</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.00414-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00414-07%26rft_id%253Dinfo%253Apmid%252F18326586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI3LzUvODU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.57" data-doi="10.1016/j.prp.2015.04.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">He</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sun</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Overexpression of CCT8 and its significance for tumor cell proliferation, migration and invasion in glioma.</span> <abbr class="cit-jnl-abbrev">Pathol. - Res. Pract.</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">725.5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.prp.2015.04.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPathol.%2B-%2BRes.%2BPract.%26rft.volume%253D211%26rft.spage%253D717%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.prp.2015.04.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.prp.2015.04.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.58" data-doi="10.1093/bioinformatics/btl633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rivals</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Personnaz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Taing</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.-C.</span> <span class="cit-name-surname">Potier</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Enrichment or depletion of a GO category within a class of genes: which test?</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">407</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btl633</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl633%26rft_id%253Dinfo%253Apmid%252F17182697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17182697&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244259200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabatti</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Service</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Freimer</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">False discovery rate in linkage and association genome screens for complex disorders.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">833</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSabatti%26rft.auinit1%253DC.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D829%26rft.epage%253D833%26rft.atitle%253DFalse%2BDiscovery%2BRate%2Bin%2BLinkage%2Band%2BAssociation%2BGenome%2BScreens%2Bfor%2BComplex%2BDisorders%26rft_id%253Dinfo%253Apmid%252F12807801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvODI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.60" data-doi="10.1074/jbc.M010864200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saveanu</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Fromont-Racine</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Harington</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Ricard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Namane</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Identification of 12 new yeast mitochondrial ribosomal proteins including 6 that have no prokaryotic homologues.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">15861</span>–<span class="cit-lpage">15867</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M010864200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M010864200%26rft_id%253Dinfo%253Apmid%252F11278769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzYvMTkvMTU4NjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidlin</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Matt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Kudlow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">MacKay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Daniel</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Single‐gene deletions that restore mating competence to diploid yeast.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft.volume%253D8%26rft.spage%253D276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.62" data-doi="10.1074/jbc.M503614200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaner</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wegele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Buchner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Morano</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The yeast Hsp110 Sse1 functionally interacts with the Hsp70 chaperones Ssa and Ssb.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">41262</span>–<span class="cit-lpage">41269</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M503614200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M503614200%26rft_id%253Dinfo%253Apmid%252F16221677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvNTAvNDEyNjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.63" data-doi="10.1186/s13075-018-1760-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The risks of cancer development in systemic lupus erythematosus (SLE) patients: a systematic review and meta-analysis.</span> <abbr class="cit-jnl-abbrev">Arthritis Res. Ther.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">270</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13075-018-1760-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArthritis%2BRes.%2BTher.%26rft.volume%253D20%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13075-018-1760-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13075-018-1760-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.64" data-doi="10.1371/journal.pgen.1002057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stirling</span>,  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Bloom</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Solanki-Patil</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sipahimalani</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Complete Spectrum of Yeast Chromosome Instability Genes Identifies Candidate CIN Cancer Genes and Functional Roles for ASTRA Complex Components.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1002057</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DStirling%26rft.auinit1%253DP.%2BC.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1002057%26rft.epage%253De1002057%26rft.atitle%253DThe%2Bcomplete%2Bspectrum%2Bof%2Byeast%2Bchromosome%2Binstability%2Bgenes%2Bidentifies%2Bcandidate%2BCIN%2Bcancer%2Bgenes%2Band%2Bfunctional%2Broles%2Bfor%2BASTRA%2Bcomplex%2Bcomponents.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002057%26rft_id%253Dinfo%253Apmid%252F21552543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552543&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-2" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.65" data-doi="10.1534/genetics.107.084103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strome</span>,  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kimmel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Plon</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Heterozygous Screen in Saccharomyces cerevisiae Identifies Dosage-Sensitive Genes That Affect Chromosome Stability.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1207</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.084103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084103%26rft_id%253Dinfo%253Apmid%252F18245329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzExOTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.66" data-doi="10.1007/BF03402176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A human novel gene DERPC on 16q22.1 inhibits prostate tumor cell growth and its expression is decreased in prostate and renal tumors.</span> <abbr class="cit-jnl-abbrev">Mol. Med.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">655</span>–<span class="cit-lpage">663</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF03402176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmedicine%2B%2528Cambridge%252C%2BMass.%2529%26rft.stitle%253DMol%2BMed%26rft.aulast%253DSun%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D655%26rft.epage%253D663%26rft.atitle%253DA%2Bhuman%2Bnovel%2Bgene%2BDERPC%2Bon%2B16q22.1%2Binhibits%2Bprostate%2Btumor%2Bcell%2Bgrowth%2Band%2Bits%2Bexpression%2Bis%2Bdecreased%2Bin%2Bprostate%2Band%2Brenal%2Btumors.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF03402176%26rft_id%253Dinfo%253Apmid%252F12477976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF03402176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12477976&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.67" data-doi="10.1158/1940-6207.CAPR-15-0077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taguchi</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-H.</span> <span class="cit-name-surname">Rho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Yan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">MAPRE1 as a plasma biomarker for early-stage colorectal cancer and adenomas.</span> <abbr class="cit-jnl-abbrev">Cancer Prev. Res. (Phila.)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1158/1940-6207.CAPR-15-0077</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BPrev.%2BRes.%2B%2528Phila.%2529%26rft_id%253Dinfo%253Adoi%252F10.1158%252F1940-6207.CAPR-15-0077%26rft_id%253Dinfo%253Apmid%252F26342024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImNhbnByZXZyZXMiO3M6NToicmVzaWQiO3M6OToiOC8xMS8xMTEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.68" data-doi="10.1038/ncomms6906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">The</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ruijtenberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">Bouchet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cristobal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. B. W.</span> <span class="cit-name-surname">Prinsen</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rb and FZR1/Cdh1 determine CDK4/6-cyclin D requirement in C. elegans and human cancer cells.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">5906</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms6906</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D5906%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6906%26rft_id%253Dinfo%253Apmid%252F25562820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25562820&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.69" data-doi="10.1097/00001622-200201000-00012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thiagalingam</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Foy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Thiagalingam</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Loss of heterozygosity as a predictor to map tumor suppressor genes in cancer: molecular basis of its occurrence.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Oncol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1097/00001622-200201000-00012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Boncology%26rft.stitle%253DCurr%2BOpin%2BOncol%26rft.aulast%253DThiagalingam%26rft.auinit1%253DS.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D65%26rft.epage%253D72%26rft.atitle%253DLoss%2Bof%2Bheterozygosity%2Bas%2Ba%2Bpredictor%2Bto%2Bmap%2Btumor%2Bsuppressor%2Bgenes%2Bin%2Bcancer%253A%2Bmolecular%2Bbasis%2Bof%2Bits%2Boccurrence.%26rft_id%253Dinfo%253Adoi%252F10.1097%252F00001622-200201000-00012%26rft_id%253Dinfo%253Apmid%252F11790983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/00001622-200201000-00012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11790983&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173076100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.70" data-doi="10.1371/journal.pbio.0000059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trotman</span>,  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Niki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. A.</span> <span class="cit-name-surname">Dotan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Koutcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Cristofano</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Pten Dose Dictates Cancer Progression in the Prostate.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-elocation-id">e59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0000059</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D1%26rft.spage%253De59%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0000059%26rft_id%253Dinfo%253Apmid%252F14691534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0000059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14691534&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.71" data-doi="10.1002/bies.10023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>,  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Exploring the etiology of haploinsufficiency.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">184</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.10023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DVeitia%26rft.auinit1%253DR.%2BA.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D175%26rft.epage%253D184%26rft.atitle%253DExploring%2Bthe%2Betiology%2Bof%2Bhaploinsufficiency.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.10023%26rft_id%253Dinfo%253Apmid%252F11835282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.10023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11835282&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173713600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.72" data-doi="10.2478/v10019-010-0035-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vladusic</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hrascan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Pecina-Slaus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Vrhovac</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gamulin</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Loss of heterozygosity of CDKN2A (p16INK4a) and RB1 tumor suppressor genes in testicular germ cell tumors.</span> <abbr class="cit-jnl-abbrev">Radiol. Oncol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">168</span>–<span class="cit-lpage">173</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2478/v10019-010-0035-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRadiol.%2BOncol.%26rft.volume%253D44%26rft.spage%253D168%26rft_id%253Dinfo%253Adoi%252F10.2478%252Fv10019-010-0035-7%26rft_id%253Dinfo%253Apmid%252F22933911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2478/v10019-010-0035-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22933911&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.73" data-doi="10.1083/jcb.201503036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>,  <span class="cit-name-given-names">K. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hall</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reproducibility and cell biology.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">193</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1083/jcb.201503036</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201503036%26rft_id%253Dinfo%253Apmid%252F25918221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIwOS8yLzE5MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yODM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.74" data-doi="10.1073/pnas.0610642104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K. W. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Warren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kwok</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hieter</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Systematic genome instability screens in yeast and their potential relevance to cancer.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">3925</span>–<span class="cit-lpage">3930</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0610642104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0610642104%26rft_id%253Dinfo%253Apmid%252F17360454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzEwLzM5MjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.75" data-doi="10.18632/oncotarget.5011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-F.</span> <span class="cit-name-surname">Chang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Yan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.-L.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A conserved RAD6-MDM2 ubiquitin ligase machinery targets histone chaperone ASF1A in tumorigenesis.</span> <abbr class="cit-jnl-abbrev">Oncotarget</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">29599</span>–<span class="cit-lpage">29613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncotarget%26rft.volume%253D6%26rft.spage%253D29599%26rft_id%253Dinfo%253Adoi%252F10.18632%252Foncotarget.5011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18632/oncotarget.5011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.76" data-doi="10.1089/omi.1.1998.3.133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Yeast “Operons.”</span>. <abbr class="cit-jnl-abbrev">Microb. Comp. Genomics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/omi.1.1998.3.133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobial%2B%2526%2Bcomparative%2Bgenomics%26rft.stitle%253DMicrob%2BComp%2BGenomics%26rft.aulast%253DZhang%26rft.auinit1%253DX.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D133%26rft.epage%253D140%26rft.atitle%253DYeast%2B%2526quot%253Boperons%2526quot%253B.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fomi.1.1998.3.133%26rft_id%253Dinfo%253Apmid%252F9697097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/omi.1.1998.3.133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9697097&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.77" data-doi="10.1016/j.mbs.2008.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A note on plating efficiency in fluctuation experiments.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">153</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mbs.2008.09.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bbiosciences%26rft.stitle%253DMath%2BBiosci%26rft.aulast%253DZheng%26rft.auinit1%253DQ.%26rft.volume%253D216%26rft.issue%253D2%26rft.spage%253D150%26rft.epage%253D153%26rft.atitle%253DA%2Bnote%2Bon%2Bplating%2Befficiency%2Bin%2Bfluctuation%2Bexperiments.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mbs.2008.09.002%26rft_id%253Dinfo%253Apmid%252F18822300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mbs.2008.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18822300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.78" data-doi="10.1007/s10709-016-9904-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparing mutation rates under the Luria-Delbrück protocol.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-016-9904-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D144%26rft.spage%253D351%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-016-9904-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-016-9904-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.79" data-doi="10.1016/j.mrfmmm.2015.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Methods for comparing mutation rates using fluctuation assay data.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">777</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mrfmmm.2015.04.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%26rft.volume%253D777%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mrfmmm.2015.04.002%26rft_id%253Dinfo%253Apmid%252F25912079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mrfmmm.2015.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25912079&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.80" data-doi="10.1016/S0025-5564(02)00087-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Statistical and algorithmic methods for fluctuation analysis with SALVADOR as an implementation.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0025-5564(02)00087-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bbiosciences%26rft.stitle%253DMath%2BBiosci%26rft.aulast%253DZheng%26rft.auinit1%253DQ.%26rft.volume%253D176%26rft.issue%253D2%26rft.spage%253D237%26rft.epage%253D252%26rft.atitle%253DStatistical%2Band%2Balgorithmic%2Bmethods%2Bfor%2Bfluctuation%2Banalysis%2Bwith%2BSALVADOR%2Bas%2Ban%2Bimplementation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0025-5564%252802%252900087-1%26rft_id%253Dinfo%253Apmid%252F11916511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0025-5564(02)00087-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11916511&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175028400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2835.81" data-doi="10.1534/g3.115.017913"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Heinecke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Mulla</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. D.</span> <span class="cit-name-surname">Bradford</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Rubinstein</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1043</span>–<span class="cit-lpage">1056</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.017913</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.017913%26rft_id%253Dinfo%253Apmid%252F25823586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjUvNi8xMDQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/9/2835.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2823" title="GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2851" title="Developing a High-Quality Linkage Map for the Atlantic Killifish Fundulus heteroclitus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/9"><span>Volume 9 Issue 9, September 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/9.cover-source.jpg?itok=4kZTDz_V" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2835" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gnb1ovdM7PAIIbFYt97sEbHL3A8DsOeJOMfL-eUpoJE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/9/2835" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/9/2835" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/9/2835" class="active" data-icon-position="" data-hide-link-title="0">Single-Gene Deletions Contributing to Loss of Heterozygosity in Saccharomyces cerevisiae: Genome-Wide Screens and Reproducibility</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494056" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-tjzT-ZxkzH1I6BoIOyh-mtxSoG7RZ4w-UILpUazAov8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2835.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/9/2835" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-KaLFJLCNTTertzJu1n-HgO7BUomqzjDE8VwtrsO1XMY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/9/2835" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2835.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494056" id="node494056--2" data-pisa="ggg;9/9/2835" data-pisa-master="ggg;g3.119.400429" data-apath="/ggg/9/9/2835.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Gene Deletions Contributing to Loss of Heterozygosity in <em>Saccharomyces cerevisiae</em>: Genome-Wide Screens and Reproducibility</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kellyn M. Hoffert</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6334-9619" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erin D. Strome</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2835-2850;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400429 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494056--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kellyn M. Hoffert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Northern Kentucky University, Highland Heights, Kentucky, 41099</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kellyn%2BM.%2BHoffert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffert%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKellyn%2BM.%2BHoffert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Erin D. Strome </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Northern Kentucky University, Highland Heights, Kentucky, 41099</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BD.%2BStrome%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strome%20ED&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErin%2BD.%2BStrome%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6334-9619" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erin D. Strome</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stromee1@nku.edu" class="" data-icon-position="" data-hide-link-title="0">stromee1@nku.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/9/2835" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/9/2835&amp;title=Single-Gene%20Deletions%20Contributing%20to%20Loss%20of%20Heterozygosity%20in%20Saccharomyces%20cerevisiae%3A%20Genome-Wide%20Screens%20and%20Reproducibility" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/9/2835&amp;title=Single-Gene%20Deletions%20Contributing%20to%20Loss%20of%20Heterozygosity%20in%20Saccharomyces%20cerevisiae%3A%20Genome-Wide%20Screens%20and%20Reproducibility" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/9/2835&amp;title=Single-Gene%20Deletions%20Contributing%20to%20Loss%20of%20Heterozygosity%20in%20Saccharomyces%20cerevisiae%3A%20Genome-Wide%20Screens%20and%20Reproducibility" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/9/2835&amp;text=Single-Gene%20Deletions%20Contributing%20to%20Loss%20of%20Heterozygosity%20in%20Saccharomyces%20cerevisiae%3A%20Genome-Wide%20Screens%20and%20Reproducibility" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/9/2835&amp;title=Single-Gene%20Deletions%20Contributing%20to%20Loss%20of%20Heterozygosity%20in%20Saccharomyces%20cerevisiae%3A%20Genome-Wide%20Screens%20and%20Reproducibility" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/9/2835&amp;t=Single-Gene%20Deletions%20Contributing%20to%20Loss%20of%20Heterozygosity%20in%20Saccharomyces%20cerevisiae%3A%20Genome-Wide%20Screens%20and%20Reproducibility" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/9/2835&amp;title=Single-Gene%20Deletions%20Contributing%20to%20Loss%20of%20Heterozygosity%20in%20Saccharomyces%20cerevisiae%3A%20Genome-Wide%20Screens%20and%20Reproducibility" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/9/2835&amp;title=Single-Gene%20Deletions%20Contributing%20to%20Loss%20of%20Heterozygosity%20in%20Saccharomyces%20cerevisiae%3A%20Genome-Wide%20Screens%20and%20Reproducibility" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494056/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494056" id="node494056--3" data-pisa="ggg;9/9/2835" data-pisa-master="ggg;g3.119.400429" data-apath="/ggg/9/9/2835.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Gene Deletions Contributing to Loss of Heterozygosity in <em>Saccharomyces cerevisiae</em>: Genome-Wide Screens and Reproducibility</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kellyn M. Hoffert</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6334-9619" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erin D. Strome</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2835-2850;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400429 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494056--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kellyn M. Hoffert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Northern Kentucky University, Highland Heights, Kentucky, 41099</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kellyn%2BM.%2BHoffert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffert%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKellyn%2BM.%2BHoffert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Erin D. Strome </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Northern Kentucky University, Highland Heights, Kentucky, 41099</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BD.%2BStrome%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strome%20ED&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErin%2BD.%2BStrome%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6334-9619" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erin D. Strome</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stromee1@nku.edu" class="" data-icon-position="" data-hide-link-title="0">stromee1@nku.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2835&amp;rft.epage=2850&amp;rft.atitle=Single-Gene%20Deletions%20Contributing%20to%20Loss%20of%20Heterozygosity%20in%20Saccharomyces%20cerevisiae%3A%20Genome-Wide%20Screens%20and%20Reproducibility&amp;rft.volume=9&amp;rft.issue=9&amp;rft.date=2019-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hoffert%2C+Kellyn+M.&amp;rft.au=Strome%2C+Erin+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494056/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494056/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494056/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494056/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494056/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494056/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494056/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494056/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494056/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494056/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494056/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494056/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/9/2835" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-27" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/9/2835.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/9/2835.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/9/2835.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
